
<html lang="en"     class="pb-page"  data-request-id="ebdc398f-2d60-435b-a77a-32745d409757"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00562;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-20;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function" /></meta><meta name="dc.Creator" content="Vincent  Fagan" /></meta><meta name="dc.Creator" content="Catrine  Johansson" /></meta><meta name="dc.Creator" content="Carina  Gileadi" /></meta><meta name="dc.Creator" content="Octovia  Monteiro" /></meta><meta name="dc.Creator" content="James E.  Dunford" /></meta><meta name="dc.Creator" content="Reshma  Nibhani" /></meta><meta name="dc.Creator" content="Martin  Philpott" /></meta><meta name="dc.Creator" content="Jessica  Malzahn" /></meta><meta name="dc.Creator" content="Graham  Wells" /></meta><meta name="dc.Creator" content="Ruth  Faram" /></meta><meta name="dc.Creator" content="Adam P.  Cribbs" /></meta><meta name="dc.Creator" content="Nadia  Halidi" /></meta><meta name="dc.Creator" content="Fengling  Li" /></meta><meta name="dc.Creator" content="Irene  Chau" /></meta><meta name="dc.Creator" content="Holger  Greschik" /></meta><meta name="dc.Creator" content="Srikannathasan  Velupillai" /></meta><meta name="dc.Creator" content="Abdellah  Allali-Hassani" /></meta><meta name="dc.Creator" content="James  Bennett" /></meta><meta name="dc.Creator" content="Thomas  Christott" /></meta><meta name="dc.Creator" content="Charline  Giroud" /></meta><meta name="dc.Creator" content="Andrew M.  Lewis" /></meta><meta name="dc.Creator" content="Kilian V. M.  Huber" /></meta><meta name="dc.Creator" content="Nick  Athanasou" /></meta><meta name="dc.Creator" content="Chas  Bountra" /></meta><meta name="dc.Creator" content="Manfred  Jung" /></meta><meta name="dc.Creator" content="Roland  Schüle" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Cheryl  Arrowsmith" /></meta><meta name="dc.Creator" content="Yan  Xiong" /></meta><meta name="dc.Creator" content="Jian  Jin" /></meta><meta name="dc.Creator" content="Oleg  Fedorov" /></meta><meta name="dc.Creator" content="Gillian  Farnie" /></meta><meta name="dc.Creator" content="Paul E.  Brennan" /></meta><meta name="dc.Creator" content="Udo  Oppermann" /></meta><meta name="dc.Description" content="Modifications of histone tails, including lysine/arginine methylation, provide the basis of a “chromatin or histone code”. Proteins that contain “reader” domains can bind to these modifications and..." /></meta><meta name="Description" content="Modifications of histone tails, including lysine/arginine methylation, provide the basis of a “chromatin or histone code”. Proteins that contain “reader” domains can bind to these modifications and..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 24, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00562" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00562" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00562" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00562" /></link>
        
    
    

<title>A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00562" /></meta><meta property="og:title" content="A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0012.jpeg" /></meta><meta property="og:description" content="Modifications of histone tails, including lysine/arginine methylation, provide the basis of a “chromatin or histone code”. Proteins that contain “reader” domains can bind to these modifications and form specific effector complexes, which ultimately mediate chromatin function. The spindlin1 (SPIN1) protein contains three Tudor methyllysine/arginine reader domains and was identified as a putative oncogene and transcriptional coactivator. Here we report a SPIN1 chemical probe inhibitor with low nanomolar in vitro activity, exquisite selectivity on a panel of methyl reader and writer proteins, and with submicromolar cellular activity. X-ray crystallography showed that this Tudor domain chemical probe simultaneously engages Tudor domains 1 and 2 via a bidentate binding mode. Small molecule inhibition and siRNA knockdown of SPIN1, as well as chemoproteomic studies, identified genes which are transcriptionally regulated by SPIN1 in squamous cell carcinoma and suggest that SPIN1 may have a role in cancer related inflammation and/or cancer metastasis." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00562"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00562">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00562&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00562&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00562&amp;href=/doi/10.1021/acs.jmedchem.9b00562" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 9008-9025</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00522" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00294" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Vincent Fagan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vincent Fagan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vincent++Fagan">Vincent Fagan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Catrine Johansson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Catrine Johansson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Catrine++Johansson">Catrine Johansson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carina Gileadi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carina Gileadi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carina++Gileadi">Carina Gileadi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Octovia Monteiro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Octovia Monteiro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Octovia++Monteiro">Octovia Monteiro</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James E. Dunford</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James E. Dunford</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+E.++Dunford">James E. Dunford</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Reshma Nibhani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Reshma Nibhani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Reshma++Nibhani">Reshma Nibhani</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Philpott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Philpott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Philpott">Martin Philpott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jessica Malzahn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jessica Malzahn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jessica++Malzahn">Jessica Malzahn</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Graham Wells</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Graham Wells</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Graham++Wells">Graham Wells</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ruth Faram</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ruth Faram</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ruth++Faram">Ruth Faram</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adam P. Cribbs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adam P. Cribbs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adam+P.++Cribbs">Adam P. Cribbs</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nadia Halidi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nadia Halidi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nadia++Halidi">Nadia Halidi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fengling Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fengling Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fengling++Li">Fengling Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Irene Chau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Irene Chau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Irene++Chau">Irene Chau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Holger Greschik</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Holger Greschik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Urology, Center for Clinical Research, Medical Center, Signalling Research Centres BIOSS and CIBSS, University of Freiburg, D-79106 Freiburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Holger++Greschik">Holger Greschik</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Srikannathasan Velupillai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Srikannathasan Velupillai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Srikannathasan++Velupillai">Srikannathasan Velupillai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Abdellah Allali-Hassani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Abdellah Allali-Hassani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Abdellah++Allali-Hassani">Abdellah Allali-Hassani</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James Bennett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James Bennett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James++Bennett">James Bennett</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Christott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Christott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Christott">Thomas Christott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charline Giroud</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charline Giroud</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charline++Giroud">Charline Giroud</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew M. Lewis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew M. Lewis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+M.++Lewis">Andrew M. Lewis</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6447-354X" title="Orcid link">http://orcid.org/0000-0002-6447-354X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kilian V. M. Huber</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kilian V. M. Huber</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kilian+V.+M.++Huber">Kilian V. M. Huber</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1103-5300" title="Orcid link">http://orcid.org/0000-0002-1103-5300</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nick Athanasou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nick Athanasou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nick++Athanasou">Nick Athanasou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chas Bountra</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chas Bountra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chas++Bountra">Chas Bountra</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Manfred Jung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Manfred Jung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">FRIAS—Freiburg Institute of Advanced Studies, University of Freiburg, 79104 Freiburg, Germany</div><div class="loa-info-affiliations-info">Institute of Pharmaceutical Sciences, University of Freiburg, Albertstraße 25, 79104 Freiburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Manfred++Jung">Manfred Jung</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6361-7716" title="Orcid link">http://orcid.org/0000-0002-6361-7716</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Roland Schüle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roland Schüle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Urology, Center for Clinical Research, Medical Center, Signalling Research Centres BIOSS and CIBSS, University of Freiburg, D-79106 Freiburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roland++Sch%C3%BCle">Roland Schüle</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masoud Vedadi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masoud Vedadi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheryl Arrowsmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheryl Arrowsmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl++Arrowsmith">Cheryl Arrowsmith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yan Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Xiong">Yan Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jian Jin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Jin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mount Sinai Center for Therapeutics Discovery, Departments of Pharmacological Sciences and Oncological Sciences, Tisch Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Jin">Jian Jin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oleg Fedorov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oleg Fedorov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oleg++Fedorov">Oleg Fedorov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gillian Farnie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gillian Farnie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gillian++Farnie">Gillian Farnie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul E. Brennan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul E. Brennan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Target Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div><div class="loa-info-affiliations-info">Alzheimer’s Research UK Oxford Drug Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</div></div><span class="conrtib-corresp"><strong>*</strong>For P.E.B.: E-mail, <a href="/cdn-cgi/l/email-protection#4e3e2f3b22602c3c2b20202f200e3d292d602136602f2d603b25"><span class="__cf_email__" data-cfemail="136372667f3d7161767d7d727d536074703d7c6b3d72703d6678">[email protected]</span></a>. Phone, +44 1865 612932.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul+E.++Brennan">Paul E. Brennan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8950-7646" title="Orcid link">http://orcid.org/0000-0002-8950-7646</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Udo Oppermann</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Udo Oppermann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</div><div class="loa-info-affiliations-info">Botnar Research Centre, Nuffield Department of Orthopaedics, Rheumatology and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</div><div class="loa-info-affiliations-info">FRIAS—Freiburg Institute of Advanced Studies, University of Freiburg, 79104 Freiburg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>For U.O.: E-mail, <a href="/cdn-cgi/l/email-protection#780d1c17561708081d0a15191616380b1f1b56170056191b560d13"><span class="__cf_email__" data-cfemail="89fcede6a7e6f9f9ecfbe4e8e7e7c9faeeeaa7e6f1a7e8eaa7fce2">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Udo++Oppermann">Udo Oppermann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9984-5342" title="Orcid link">http://orcid.org/0000-0001-9984-5342</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00562&amp;href=/doi/10.1021%2Facs.jmedchem.9b00562" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 9008–9025</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 24, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>1 April 2019</li><li><span class="item_label"><b>Published</b> online</span>24 September 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 October 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00562" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00562</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9008%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DVincent%2BFagan%252C%2BCatrine%2BJohansson%252C%2BCarina%2BGileadi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D20%26contentID%3Dacs.jmedchem.9b00562%26title%3DA%2BChemical%2BProbe%2Bfor%2BTudor%2BDomain%2BProtein%2BSpindlin1%2Bto%2BInvestigate%2BChromatin%2BFunction%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9025%26publicationDate%3DOctober%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00562"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1710</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00562" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Chemical Probe for Tudor Domain Protein Spindlin1 to Investigate Chromatin Function&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Vincent&quot;,&quot;last_name&quot;:&quot;Fagan&quot;},{&quot;first_name&quot;:&quot;Catrine&quot;,&quot;last_name&quot;:&quot;Johansson&quot;},{&quot;first_name&quot;:&quot;Carina&quot;,&quot;last_name&quot;:&quot;Gileadi&quot;},{&quot;first_name&quot;:&quot;Octovia&quot;,&quot;last_name&quot;:&quot;Monteiro&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;E. Dunford&quot;},{&quot;first_name&quot;:&quot;Reshma&quot;,&quot;last_name&quot;:&quot;Nibhani&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Philpott&quot;},{&quot;first_name&quot;:&quot;Jessica&quot;,&quot;last_name&quot;:&quot;Malzahn&quot;},{&quot;first_name&quot;:&quot;Graham&quot;,&quot;last_name&quot;:&quot;Wells&quot;},{&quot;first_name&quot;:&quot;Ruth&quot;,&quot;last_name&quot;:&quot;Faram&quot;},{&quot;first_name&quot;:&quot;Adam&quot;,&quot;last_name&quot;:&quot;P. Cribbs&quot;},{&quot;first_name&quot;:&quot;Nadia&quot;,&quot;last_name&quot;:&quot;Halidi&quot;},{&quot;first_name&quot;:&quot;Fengling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Irene&quot;,&quot;last_name&quot;:&quot;Chau&quot;},{&quot;first_name&quot;:&quot;Holger&quot;,&quot;last_name&quot;:&quot;Greschik&quot;},{&quot;first_name&quot;:&quot;Srikannathasan&quot;,&quot;last_name&quot;:&quot;Velupillai&quot;},{&quot;first_name&quot;:&quot;Abdellah&quot;,&quot;last_name&quot;:&quot;Allali-Hassani&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;Bennett&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Christott&quot;},{&quot;first_name&quot;:&quot;Charline&quot;,&quot;last_name&quot;:&quot;Giroud&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;M. Lewis&quot;},{&quot;first_name&quot;:&quot;Kilian&quot;,&quot;last_name&quot;:&quot;V. M. Huber&quot;},{&quot;first_name&quot;:&quot;Nick&quot;,&quot;last_name&quot;:&quot;Athanasou&quot;},{&quot;first_name&quot;:&quot;Chas&quot;,&quot;last_name&quot;:&quot;Bountra&quot;},{&quot;first_name&quot;:&quot;Manfred&quot;,&quot;last_name&quot;:&quot;Jung&quot;},{&quot;first_name&quot;:&quot;Roland&quot;,&quot;last_name&quot;:&quot;Schüle&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Jin&quot;},{&quot;first_name&quot;:&quot;Oleg&quot;,&quot;last_name&quot;:&quot;Fedorov&quot;},{&quot;first_name&quot;:&quot;Gillian&quot;,&quot;last_name&quot;:&quot;Farnie&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;E. Brennan&quot;},{&quot;first_name&quot;:&quot;Udo&quot;,&quot;last_name&quot;:&quot;Oppermann&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;9008-9025&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00562&quot;},&quot;abstract&quot;:&quot;Modifications of histone tails, including lysine/arginine methylation, provide the basis of a “chromatin or histone code”. Proteins that contain “reader” domains can bind to these modifications and form specific effector complexes, which ultimately mediate chromatin function. The spindlin1 (SPIN1) protein contains three Tudor methyllysine/arginine reader domains and was identified as a putative oncogene and transcriptional coactivator. Here we report a SPIN1 chemical probe inhibitor with low nanomolar in vitro activity, exquisite selectivity on a panel of methyl reader and writer proteins, and with submicromolar cellular activity. X-ray crystallography showed that this Tudor domain chemical probe simultaneously engages Tudor domains 1 and 2 via a bidentate binding mode. Small molecule inhibition and siRNA knockdown of SPIN1, as well as chemoproteomic studies, identified genes which are transcriptionally regulated by SPIN1 in squamous cell carcinoma and suggest that SPIN1 may have a role in cancer related infl&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00562&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00562" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00562&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00562" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00562&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00562" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00562&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00562&amp;href=/doi/10.1021/acs.jmedchem.9b00562" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00562" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00562" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00562%26sid%3Dliteratum%253Aachs%26pmid%3D31550156%26genre%3Darticle%26aulast%3DFagan%26date%3D2019%26atitle%3DA%2BChemical%2BProbe%2Bfor%2BTudor%2BDomain%2BProtein%2BSpindlin1%2Bto%2BInvestigate%2BChromatin%2BFunction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D20%26spage%3D9008%26epage%3D9025%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/jmcmar.2019.62.issue-20/20191024/jmcmar.2019.62.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Modifications of histone tails, including lysine/arginine methylation, provide the basis of a “chromatin or histone code”. Proteins that contain “reader” domains can bind to these modifications and form specific effector complexes, which ultimately mediate chromatin function. The spindlin1 (SPIN1) protein contains three Tudor methyllysine/arginine reader domains and was identified as a putative oncogene and transcriptional coactivator. Here we report a SPIN1 chemical probe inhibitor with low nanomolar in vitro activity, exquisite selectivity on a panel of methyl reader and writer proteins, and with submicromolar cellular activity. X-ray crystallography showed that this Tudor domain chemical probe simultaneously engages Tudor domains 1 and 2 via a bidentate binding mode. Small molecule inhibition and siRNA knockdown of SPIN1, as well as chemoproteomic studies, identified genes which are transcriptionally regulated by SPIN1 in squamous cell carcinoma and suggest that SPIN1 may have a role in cancer related inflammation and/or cancer metastasis.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20422" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20422" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetic methyllysine and arginine reader proteins play a crucial role in many biological processes. They bind to posttranslational methylation marks on histone tails, thus interpreting and conveying the “histone or chromatin code” downstream.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> This large family of proteins consists of Chromo, PHD, Tudor, MBT, and PWWP domains.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Within the Tudor domains, there are five human spindlin proteins (SPIN1, -2A, -2B, -3, and -4) and each contain three Tudor methyl reader domains.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8 ref9">(4−9)</a> Spindlin1 (SPIN1) was initially identified in mouse oocytes<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> but has now been implicated in many cancers.<a onclick="showRef(event, 'ref7 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref7 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18">(7,10−18)</a> SPIN1 binds to trimethylated lysine 4 of histone 3 (H3K4me3) via the “aromatic cage” of Tudor domain 2.<a onclick="showRef(event, 'ref17 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref19 ref20">(17,19,20)</a> It can also simultaneously bind to the asymmetrically dimethylated arginine mark H3R8me2a via domain 1,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> which makes SPIN1 a bivalent methyllysine and methylarginine reader. SPIN1 also binds to H4K20me3 via domain 2, but little is known about this interaction.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Binding to these epigenetic marks at gene promoter regions leads to transcriptional activation of ribosomal proteins<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and promotes liposarcoma, ovarian, colon, breast, and nonsmall-cell lung cancers by activation of the GDNF-RET-MAZ,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Wnt/β-catenin,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> or PI3K/Akt<a onclick="showRef(event, 'ref11 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref11 ref15 ref16">(11,15,16)</a> pathways, respectively. SPIN1 has also been shown to facilitate the inactivation of p53 through interaction with the ribosomal protein uL18.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Recently, the C11ORF84 protein was identified that attenuates SPIN1 coactivator activity.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Knockdown of SPIN1 resulted in cancer cell and xenograft tumor growth inhibition, which suggests that small molecule inhibition of SPIN1 may be a viable approach for the treatment of certain cancers.<a onclick="showRef(event, 'ref7 ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref7 ref11 ref12 ref13 ref14 ref15">(7,11−15)</a> Therefore, potent and selective SPIN1 inhibitors are needed to aid with the further elucidation of SPIN1 biology. Micromolar inhibitors were identified using in silico methods,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and bidentate inhibitors that interact with Tudor domains 1 and 2 were reported.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">However, with <i>K</i><sub>D</sub> values above 3 μM, these inhibitors have considerably weaker binding affinity than that required for chemical probe inhibitors (in vitro <i>K</i><sub>D</sub> < 100 nM).<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> We recently identified the possible SPIN1 hit molecule A-366 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, IC<sub>50</sub> 186 nM).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> However, this molecule is a known chemical probe for the methyltransferase G9a (IC<sub>50</sub> 3 nM).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Using A-366 as a starting point, we now set out to develop SPIN1 inhibitors and we here report the biochemical, structural, and cellular characterization that led to a potent and selective SPIN1 chemical probe.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structures of A-366 with G9a and SPIN1. (A) Structure of A-366 (cyan) in complex with G9a (pink) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NVQ">4NVQ</a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (B) Structure of A-366 (cyan) in complex with SPIN1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Y">6I8Y</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We initially used X-ray crystallography to compare the binding modes of A-366 in complex with SPIN1 and G9a. Binding of A-366 to G9a is largely dominated by polar contacts (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> while in SPIN1 two molecules of A-366 bind, one in each aromatic cage of domain 1 and domain 2 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). In domain 1, binding is largely driven by π-stacking interactions with residues Trp62, Trp72, Tyr91, Phe94, and Tyr98 and in domain 2 via contacts with residues His139, Phe141, Trp151, Tyr177, and Tyr179, as well as polar interactions with Asp184. For inhibitor design we focused on domain 2 of SPIN1 because binding of the H3<sub>1–10</sub>K4me3-R8me2a peptide to SPIN1 is largely driven by the interaction of K4me3 with domain 2.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Compared to G9a, domain 2 of SPIN1 can accommodate larger substituents. Therefore, the chemical synthesis of A-366<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> was modified to allow incorporation of larger alkyl groups at R<sup>1</sup> and was optimized to allow efficient incorporation of <i>tert</i>-amines at R<sup>2</sup> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, series 1 and 2).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of SPIN1 Inhibitors Series 1, 2, and 3<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>General conditions: (a) 1,4-dibromobutane, KOH, TBAB, PhMe, 0 °C-reflux, 3 h, 92%; (b) NaOH, MeI, DMSO, rt, 1 h 98%; (c) HNO<sub>3</sub>, Ac<sub>2</sub>O, AcOH, −5 °C, 1 h, 87–92%; (d) Pd/C, H<sub>2</sub>, 9:1 EtOAc:EtOH, rt, 30–40 min, 76–86%; (e) alkyl bromide, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 18 h, 92–97%; (f) 2° amine, KI, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 48 h; (g) C<sub>12</sub>H<sub>25</sub>SH, KOtBu, DMF, 0–50 °C, 2 h, 79–86%; (h) alkyl halide, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 18 h; (i) Zn, AcOH, 110 °C, 2 h, 5–82%.</p></p></figure><div class="NLM_p">Commercially available compound <b>1</b> was dialkylated using the procedure of Barbasiewicz et al., with slight modification, to incorporate a spirocycle (compound <b>2</b>), which in later analogues was replaced with <i>gem</i>-dimethyls (compound <b>3</b>). A selective nitration was achieved at low temperature, which was followed by debenzylation by hydrogenolysis to give the nitrophenol <b>6</b>. The solvent mixture (9:1 EtOAc:EtOH) was decisive in determining the outcome of this reaction, with most other solvent mixtures leading to significant reduction of the nitro group. Nitrophenol <b>6</b> was alkylated to give compound <b>7</b>, and the chloride was displaced with pyrrolidine to give compound <b>8</b>. The demethylation of <b>8</b> was achieved by combining aspects of the procedures reported by Chae et al. and Magano et al., which involved nucleophilic displacement of the phenoxide by an in situ generated thiolate. Demethylation via a Lewis acid (e.g., BBr<sub>3</sub>) led to mixtures of phenols <b>6</b> and <b>9</b>. Alkylation of phenol <b>9</b> with bulkier alkyl halides was followed by nitro group reduction with zinc in acetic acid. The resultant aniline underwent in situ ring closure to give a series of cyclic amidines possessing various alkyl groups at R<sup>1</sup>. For the preparation of series 2, the chloride of compound <b>7</b> was displaced with various secondary amines, followed by reductive ring closure.</div><div class="NLM_p">An AlphaScreen assay was employed to assess the SPIN1 binding of the compounds. Compound binding was determined by the displacement of a trimethylated biotinylated histone 3 peptide (ARTK(me3)QTARKSTGGKAPRK-biotin) from a His tagged SPIN1 protein using an AlphaScreen histidine (nickel chelate) detection kit. The scintillation proximity assay (SPA) reported by Sweis et al. was used to evaluate G9a inhibition.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In all cases, the incorporation of larger alkyl groups at R<sup>1</sup> resulted in higher IC<sub>50</sub> values for both SPIN1 and G9a (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compound <b>10a</b>–<b>c</b>). However, a methylcyclopropyl group at R<sup>1</sup> (compound <b>10c</b>) resulted in a large increase of the IC<sub>50</sub> of G9a but without significant impact on the inhibition of SPIN1, therefore imparted SPIN1 selectivity. Comparison of the IC<sub>50</sub> of A-366 to that of compound <b>11a</b> suggested that the expansion of the spirocyle did not have a significant impact on the selectivity toward SPIN1. A number of tertiary amines were incorporated at R<sup>2</sup>. The use of any acyclic or six-membered cyclic amine resulted in a dramatic loss of SPIN1 inhibition. Substituted pyrrolidines were investigated but did not give a significant improvement in SPIN1 inhibition. However, the incorporation of isoindolines at R<sup>2</sup> did increase the SPIN1 inhibition, as well as dramatically decreasing the G9a inhibition (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>11b</b>). Substitution at the isoindoline 4-position (compound <b>11c</b>) led to an approximate 10-fold decrease of the SPIN1 inhibition, whereas substitution at the 5-position (compound <b>11d</b>) was well tolerated. The optimum substituents identified in series 1 and 2 were incorporated into series 3 (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). To prepare series 3, general conditions (d), (e), (f), (g), and (i) were employed, but the order of cleavage of the benzyl- and methyl-arylethers was reversed. Series 3 contained potent SPIN1 inhibitors (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compounds <b>22</b>, <b>23</b>, and <b>25</b>). Compound <b>25</b>, which had <i>gem</i>-dimethyls in place of a spirocycle, was equipotent as a SPIN1 inhibitor but showed no inhibition of G9a, which indicated that the spirocycle was not required. Compounds possessing butyl chains had dramatically lower potency and were therefore used as inactive controls (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compounds <b>24</b> and <b>26</b>). The inactive control compound <b>26</b> had a 19-fold decrease in potency compared to its corresponding active compound <b>25</b>. Unexpectedly, the use of a slightly longer SPIN1 construct resulted in a higher IC<sub>50</sub> for A-366 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) than that previously reported,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> which was explored later.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Compound IC<sub>50</sub> on SPIN1 (AlphaScreen assay) and G9a (Scintillation Proximity Assay)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">AlphaScreen SPIN1<sub>26–262</sub> IC<sub>50</sub> [μM] (Hill slope, SD)</th><th class="colsep0 rowsep0" align="center">G9a IC<sub>50</sub> [μM] (Hill slope)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="center">2.15</td><td class="colsep0 rowsep0" align="center">0.12 (0.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10b</b></td><td class="colsep0 rowsep0" align="center">8.91</td><td class="colsep0 rowsep0" align="center">0.9 (0.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10c</b></td><td class="colsep0 rowsep0" align="center">1.12 (1.0, 0.30)</td><td class="colsep0 rowsep0" align="center">3.1 (0.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="center">0.76</td><td class="colsep0 rowsep0" align="center">0.006 (0.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="center">0.66 (0.8, 0.46)</td><td class="colsep0 rowsep0" align="center">10 (0.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11c</b></td><td class="colsep0 rowsep0" align="center">6.65</td><td class="colsep0 rowsep0" align="center">not determined</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11d</b></td><td class="colsep0 rowsep0" align="center">0.60</td><td class="colsep0 rowsep0" align="center">not determined</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="center">0.34 (1.0, 0.21)</td><td class="colsep0 rowsep0" align="center">not determined</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="center">0.39 (1.0, 0.25)</td><td class="colsep0 rowsep0" align="center">no inhibition</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="center">6.22 (0.8, 3.20)</td><td class="colsep0 rowsep0" align="center">no inhibition</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="center">0.36 (0.8, 0.06)</td><td class="colsep0 rowsep0" align="center">no inhibition</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="center">6.92 (0.7, 0.88)</td><td class="colsep0 rowsep0" align="center">no inhibition</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>A-366</b></td><td class="colsep0 rowsep0" align="center">1.50 (0.9, 1.00)</td><td class="colsep0 rowsep0" align="center">0.003<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Sweis et al.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></p></div></div></div><div class="NLM_p">The crystal structure of compound <b>23</b> in complex with SPIN1 (Supporting Information (SI), <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Figure S1</a>) suggested that the improved potency of isoindoline-containing analogues may be attributed to additional π-stacking interactions between the isoindoline moiety and residues Phe141, Trp151, Tyr170, and Tyr177 in the aromatic cage.</div><div class="NLM_p">With potent compounds in hand, cellular target engagement was assessed using a nanoBRET assay.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> A halotag-histone 3.3 and an N-terminal Nanoluc-SPIN1 fusion protein was overexpressed in U2OS cells and treated with compounds <b>23</b>–<b>26</b> at a concentration of 1 μM. A significant reduction of the nanoBRET signal was observed for compounds <b>23</b> and <b>25</b> but not for the inactive <b>24</b> and <b>26</b>, which confirmed that the inhibitors indeed showed cellular target engagement (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). However, at elevated concentrations (>3 μM), increased cell death was observed (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Figure S6</a>). Both the active and inactive compounds displayed this toxicity, implying off-target effects. Despite significant effort, the toxicity issues relating to this series could not be resolved.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. NanoBRET assay. Full-length wild-type SPIN1 and histone 3.3 nanobret assay showing in cell target engagement for compound <b>23</b> and <b>25</b>. Graph shows fold-change in milliBRET inits (mBU) after 24 h treatment with compounds at 1 μM. Mean ± SD, <i>n</i> = 3 independent experiments, <i>n</i> ≥ 3 internal replicates. One-Way ANOVA with Dunnett’s posthoc multiple comparison test, ** <i>p</i> < 0.005.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Recently, bidentate inhibitors were reported that engaged both domains 1 and 2 of SPIN1, and although they had moderate activity, they showed a good degree of selectivity.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Therefore, in efforts to increase the SPIN1 selectivity and binding affinity, and to overcome the toxicity issues, bidentate inhibitors were designed. We aimed to link compound <b>25</b> to the moiety reported by Bae et al.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> which binds to domain 1 of SPIN1 (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, highlighted in blue). The initial use of simple alkyl chain linkers resulted in highly flexible molecules, which did not engage with two SPIN1 domains. Therefore, the Huisgen copper-catalyzed azide–alkyne cycloaddition (CuAAC) reaction was utilized because it resulted in more rigid and hydrophilic triazole-containing linkers. It also allowed the linker length to be varied easily. Alkyne-functionalized isoindolines <b>27</b> and <b>28</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) were prepared by <i>O</i>-alkylation of the corresponding <i>N</i>-Boc protected phenol, followed by Boc deprotection. Alkylation at the 5-position of these isoindolines was chosen because earlier structure–activity relationship studies revealed that substitution at this position was well tolerated (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>11c</b>,<b>d</b>). Using general conditions (f), isoindoline <b>27</b> and <b>28</b> were coupled to <b>19</b> and <b>21</b> to give compounds <b>29</b>–<b>31</b>, which were subjected to the CuAAC reaction (general conditions (j)) with azide-functionalized <b>32</b> and <b>33</b>, using copper powder as catalyst (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Reductive cyclization using general conditions (i) afforded compounds <b>34</b>–<b>38</b>. Azide-functionalized <b>32</b> and <b>33</b> were prepared by amide coupling of commercially available 4-(2-(pyrrolidin-1-yl)ethyl)piperidine with the corresponding azido carboxylic acids. The bidentate compounds <b>34</b>–<b>38</b>, as well as compound <b>25</b>, were evaluated as SPIN1 inhibitors using a range of biophysical assays (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Three of these compounds displayed a large increase in potency compared to <b>25</b>. Compound <b>35</b> was chosen as a chemical probe candidate because it displayed marginally better results in three of the four assays. On the basis of the isothermal titration calorimetry (ITC) alone, there was little to distinguish between compounds <b>35</b>–<b>37</b>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Bidentate SPIN1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>General conditions: (f) 2° amine, KI, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 48 h; (j) Cu, DMF, 60 °C, 2 h; (i) Zn, AcOH, 110 °C, 2 h, 5–82%.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biophysical Assays of Bidentate SPIN1 Inhibitors<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">AlphaScreen SPIN1<sub>26–262</sub> IC<sub>50</sub> [nM] (Hill slope, SD)</th><th class="colsep0 rowsep0" align="center">Octet BLI SPIN1<sub>21–262</sub><i>K</i><sub>D</sub> [nM]</th><th class="colsep0 rowsep0" align="center">ITC SPIN1<sub>49–262</sub><i>K</i><sub>D</sub> [nM]</th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>m</sub> shift SPIN1<sub>26–262</sub>Δ<i>T</i><sub>m</sub> [°C]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="center">362 (0.8, 0.06)</td><td class="colsep0 rowsep0" align="center">94 ± 0.015</td><td class="colsep0 rowsep0" align="center">15.4</td><td class="colsep0 rowsep0" align="center">6.8 ± 0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="center">152 (1.4, 0.12)</td><td class="colsep0 rowsep0" align="center">400 ± 0.020</td><td class="colsep0 rowsep0" align="center">60</td><td class="colsep0 rowsep0" align="center">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="center">33 (2.3, 0.01)</td><td class="colsep0 rowsep0" align="center">55 ± 0.008</td><td class="colsep0 rowsep0" align="center">9.9</td><td class="colsep0 rowsep0" align="center">13.2 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="center">48 (1.4, 0.03)</td><td class="colsep0 rowsep0" align="center">67 ± 0.002</td><td class="colsep0 rowsep0" align="center">6.6</td><td class="colsep0 rowsep0" align="center">12.8 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="center">59 (1.8, 0.04)</td><td class="colsep0 rowsep0" align="center">72 ± 0.012</td><td class="colsep0 rowsep0" align="center">10.1</td><td class="colsep0 rowsep0" align="center">11.0 ± 0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="center">3388 (0.7, 1.24)</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">1300</td><td class="colsep0 rowsep0" align="center">1.0 ± 0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Abbreviations; BLI, biolayer interferometry; ITC, isothermal titration calorimetry; nd, not determined.</p></div></div></div><div class="NLM_p">Compound <b>35</b> has a <i>K</i><sub>D</sub> < 10 nM (ITC) and displaces a histone peptide with an IC<sub>50</sub> of 33 nM, making it a potent SPIN1 inhibitor.</div><div class="NLM_p">Compound <b>38</b> was prepared as an inactive control. ITC showed that <b>38</b> has approximately 130 times less binding affinity than <b>35</b> and thus compounds <b>35</b> and <b>38</b> represent good candidates for an active and inactive chemical probe pairing (CPP).</div><div class="NLM_p">The crystal structure of <b>35</b> in complex with SPIN1 confirmed a bidentate binding mode, where the ethylpyrrolidine moiety indeed occupies the aromatic cage of domain 1, while binding of the isoindoline to domain 2 is retained (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of SPIN1<sub>49–262</sub> in complex with compound <b>35</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8B">6I8B</a>). (left) Electrostatic surface potential of SPIN1 with <b>35</b>. (right) Interactions of <b>35</b> (cyan) with SPIN1 (domains 1, 2, and 3 are shown in magenta, green, and yellow respectively).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There are five distinct human SPIN family members which form a distinct branch of the Tudor domain phylogenetic tree (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The selectivity of <b>35</b> was assessed by ITC on four SPIN subfamily members (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). SPIN2A was omitted due to its similarity to SPIN2B and its low endogenous expression. The results showed that compound <b>35</b> binds with varying affinity across the subfamily (<i>K</i><sub>D</sub> 10–130 nM). This suggests a similar binding mode among the SPIN proteins and is in line with their overall structures and peptide binding features (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Figure S2</a>).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Human tudor domain phylogenetic tree with SPIN subfamily highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Average ITC <i>K</i><sub>D</sub> (nM) of Compound <b>35</b> and A-366 on Human SPIN Subfamily Members<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">SPIN1<sub>49–262</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SPIN1<sub>26–262</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SPIN2B<sub>45–258</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SPIN3<sub>27–258</sub> (nM)</th><th class="colsep0 rowsep0" align="center">SPIN4<sub>36–249</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="center">9.9</td><td class="colsep0 rowsep0" align="center">111.1</td><td class="colsep0 rowsep0" align="center">46.1</td><td class="colsep0 rowsep0" align="center">131.1</td><td class="colsep0 rowsep0" align="center">18.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>A-366</b></td><td class="colsep0 rowsep0" align="center">180</td><td class="colsep0 rowsep0" align="center">393</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">nd</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup><sup>a</sup></sup><p class="last">Stoichiometry of binding (<i>n</i>) ranges were determined as <i>n</i> = 1 ± 0.1–0.2.</p></div></div></div><div class="NLM_p">We observed a discrepancy when comparing IC<sub>50</sub> data for various SPIN1 constructs and A-366 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) against previous work.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Whereas the latter study employed a SPIN1<sub>49–262</sub> construct yielding an IC<sub>50</sub> of 186 nM, in this work (using a SPIN1<sub>26–262</sub> construct), we observe an IC<sub>50</sub> of 1.5 μM, indicating an involvement of the N-terminus on binding and/or activity. We investigated this ligand behavior further by determining <i>K</i><sub>D</sub> values for compound <b>35</b> and A-366 against the two SPIN1 constructs with varied N-termini (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) by using ITC. The use of a shorter SPIN1 construct (residues 49–262) resulted in a 10–15-fold lower <i>K</i><sub>D</sub> for <b>35</b> compared to the longer construct with residues 26–262, whereas A-366 binding is affected by a factor of 2. These data are in agreement with the previous study and indeed indicate a modulatory role of the N-terminus in SPIN1. The increased Hill slope (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) observed for <b>35</b> in the Alpha-screen, indicating a non 1:1 binding stoichiometry, is not oberserved in ITC, suggesting possible technical artifacts in the AlphaScreen. In a cellular context, full-length SPIN1 shows efficient target engagement when using the nanoBRET assay against compound <b>35</b>, displaying an EC<sub>50</sub> of 270 nM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) and indicating cellular potency well within the acceptable range for a chemical probe.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> The inactive control compound <b>38</b> displayed no inhibition and crucially, no unspecific toxicity was observed for <b>35</b> or <b>38</b>.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. NanoBRET assay results for compound <b>35</b> and <b>38</b> using full length wild-type SPIN1 with histone 3.3. Graph shows milliBRET units (mBU) after 24 h treatment with 0.05–15 μM of compound <b>35</b> and <b>38</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>35</b> and <b>38</b> were screened against methyl-Lys/Arg binding domains (including Chromo, PHD, Tudor, and MBT domains) using a SYPRO Orange thermal shift assay (differential scanning fluorimetry (DSF)).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> In agreement with the ITC data, compound <b>35</b> induced a large shift in the thermal stability of all SPIN proteins assessed (Δ<i>T</i><sub>m</sub> 6.5–14.1 °C, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). No other significant thermal shift was observed for compound <b>35</b> or <b>38</b>.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (left) Heatmap of selected methyl-Lys/Arg reader domains using DSF (thermal shift assay). Scale shows Δ<i>T</i> °C. (right) Percentage enzyme activity of 33 methyltransferases after compound treatment at 10 and 50 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>35</b> and <b>38</b> were also tested against 33 protein, DNA, or RNA methyltransferases using established enzymatic SPAs as described previously.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The compounds were initially tested at high concentration (10 and 50 μM, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) and where inhibition occurred, IC<sub>50</sub> values were determined (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Table S4</a>). Compound <b>35</b> displayed at least 300-fold selectivity over all methyltransferases tested.</div><div class="NLM_p">In addition, a fluorescence polarization assay was used to test compound <b>35</b> and <b>38</b> against the malignant brain tumor methyl readers, L3MBTL1 and L3MBTL3. The compounds showed no significant inhibition and compound <b>35</b> had approximately 250-fold higher potency toward SPIN1 than toward these MBT methyl readers (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p">Chemoproteomics was employed to identify cellular interacting partners of compound <b>35</b>.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Compound <b>41</b> was prepared by subjecting isoindoline <b>39</b> and chloride <b>19</b> to general conditions (f) and (i) to give the benzyl protected <b>40</b>. Hydrogenolysis, followed by alkylation with 4-(Boc-amino)butyl bromide, and Boc deprotection, gave the amino functionalized <b>41</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The affinity probe <b>41</b> was coupled to NHS-activated sepharose beads and incubated with SCC040 cell lysates in the presence of compound <b>35</b> or <b>38</b>. Label-free quantification (LFQ) protein mass spectrometry was used to identify compound-specific interactors. The SCC040 head and neck squamous cell carcinoma (HNSCC) cell line was chosen because SPIN1 expression in squamous cell carcinoma tumors was strongly correlated to smooth muscle actin and galectin-1, which may contribute to cell invasion and proliferation.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Affinity Probe <b>41</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>General conditions (f) 2° amine, KI, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 48 h; (i) Zn, AcOH, 110 °C, 2 h, 63% over 2 steps; (k) Pd/C, H<sub>2</sub>, MeOH, rt, 18 h; 4-(Boc-amino)butyl bromide, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 48 h; TFA, DCM, rt, 2 h, 12% overall.</p></p></figure><div class="NLM_p">Importantly, the only displaced proteins by <b>35</b> were SPIN1, SPIN4, and C11ORF84 (SPIN.DOC) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv" class="ext-link">Supporting Information, data</a>). The recent study by Bae et al. identified SPIN.DOC to interact with SPIN1 by masking Tudor domain 2 and impeding its ability to bind to H3K4me3.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In the absence of structural data showing the exact interaction sites, we speculate that the SPIN1 domain 2 ligand pocket is not directly involved or is only partially interacting with SPIN.DOC because both proteins were pulled down by affinity probe <b>41</b>. This compound presumably interacts with Tudor domain 2, further corroborated by effective displacement with compound <b>35</b>. Because SPIN.DOC was shown to bind directly to all SPIN members,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> it is likely that the affinity probe <b>41</b> can pulldown SPIN.DOC as a complex with the SPINs and that compound <b>35</b> competes only with the interaction of SPIN1, or SPIN4, and the affinity probe. The results obtained here warrant further investigation.</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Chemoproteomic analyses using compound <b>35</b> (top) or <b>38</b> (bottom) as competitors in human squamous carcinoma cell line SCC040 lysates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Taken together, all the selectivity data demonstrates that compound <b>35</b> has excellent in vitro specificity, including in a cellular context.</div><div class="NLM_p">Proliferation, cell cycle, and apoptosis behavior of SCC040 cells were assessed following treatment with compound <b>35</b> and <b>38</b> (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Figure S7</a>). A moderate decrease in proliferation and increase in apoptosis was observed for compound <b>35</b> but not <b>38</b>. Although this response was significant, it was not as dramatic as we expected. Therefore, to gain a greater understanding of the role of SPIN1 in SCC040 cells, RNAseq analysis was performed (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Figure S8–S11</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv" class="ext-link">data</a>). RNA was isolated after treatment with compound <b>35</b> or <b>38</b> (1 μM for 5 or 7 days), or with SPIN1 specific siRNA (2 or 4 days).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> While treatment with <b>38</b> elicited very few transcriptional alterations, analysis of the induced changes that were modulated by both compound <b>35</b> and the siRNA (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv" class="ext-link">Supporting Information data</a>) identified 50 genes. The overlap of genes modulated by compound <b>35</b> and <b>38</b> was minimal, which indicates that the transcriptional signature induced by compound <b>35</b> can be attributed to on-target effects.</div><figure id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Heatmap of overlapping genes modulated by compound <b>35</b>, <b>38</b>, and siRNA knockdown in SCC040 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">When the transcriptional modulation induced by compound <b>35</b> is compared to that of siRNA SPIN1 knockdown, in many cases the genes appear to be modulated in opposing directions. However, because several genes were upregulated at the first time point but were downregulated at the second (S100A8, S100A9, CLDN8, SLPI, S100P), it is clear that the transcriptional modulation is a dynamic process, and therefore, at a given time point, the siRNA knockdown data may not be comparable to the compound inhibition data at that time point.</div><div class="NLM_p">The identification of six members of the serine protease inhibitor superfamily (SERPINB4, SERPINB3, SERPINB13, SLPI, SPINK5, PI3)<a onclick="showRef(event, 'ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40">(33−40)</a> and three members of the S100 proteins (S100P, S100A8, S100A9),<a onclick="showRef(event, 'ref36 ref41 ref42 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref36 ref41 ref42 ref43 ref44 ref45">(36,41−45)</a> besides the chemokine (CCL5) and an immune checkpoint inhibitor (NT5E (CD73)), suggests that SPIN1 may have a role in the activation of inflammatory responses. The recent report by Bae et al. showed that treatment with the SPIN1 inhibitor EML631 (<i>K</i><sub>D</sub> 4 μM) resulted in decreased expression of the pro-inflammatory cytokine IL1β,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> as well as BST2, which has been shown to activate the NF-κB inflammatory pathway in nasopharyngeal cancer.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Inflammation within the tumor microenvironment (cancer-related inflammation (CRI)) is now known to be a significant contributing factor in the development and progression of cancer, particularly in tumor invasion and metastasis.<a onclick="showRef(event, 'ref44 ref48'); return false;" href="javascript:void(0);" class="ref ref44 ref48">(44,48)</a></div><div class="NLM_p">Of the 50 genes identified here, a number have previously been reported to be differentially expressed (GBP2,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> DUOX2,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> IGFBP3<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a>) or were associated with a more invasive and metastatic phenotype (AKR1B10,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> FABP5,<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> HSPB1,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> KRT1,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> S100A9,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> CNFN,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> ID2,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> LCN2 (NGAL),<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> IGFBP2,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> IFIT1,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> CCL5<a onclick="showRef(event, 'ref48 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref48 ref59 ref60">(48,59,60)</a>) in squamous cell carcinomas (SCC).</div><div class="NLM_p">Importantly, compound <b>35</b> seems to modulate these genes in a direction which opposes that identified in these previously reported studies, thus may be of therapeutic value.</div><div class="NLM_p">To aid with the further development of compound <b>35</b> as a potential in vivo candidate, in vitro permeability, stability, and aqueous solubility profiles were assessed. The intrinsic clearance, half-life, and permeability were all within acceptable ranges, and the compound showed no efflux (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Compound <b>35</b> is highly soluble, with an aqueous solubility of >100 μM (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Table S11</a>). Therefore, despite its large molecular weight, compound <b>35</b> is suitable for further development.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. In Vitro Permeability and Stability of Compound <b>35</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="3" align="center" char=".">Caco-2 permeability</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">stability in liver microsomes</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">AB <i>P</i><sub>app</sub> (10<sup>–6</sup> cms<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">BA <i>P</i><sub>app</sub> (10<sup>–6</sup> cms<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char=".">efflux ratio (BA/AB)</th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>int</sub> (μL/min/mg protein)</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">6.59</td><td class="colsep0 rowsep0" align="char" char=".">6.07</td><td class="colsep0 rowsep0" align="char" char=".">0.92</td><td class="colsep0 rowsep0" align="char" char=".">20.3<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char=".">68.4<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">37.4<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">37.1<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Human.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Mouse</p></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56827" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56827" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have developed a potent, selective, and cell-active chemical probe of the SPIN subfamily of Tudor domains. The selectivity of compound <b>35</b> and <b>38</b> has been extensively characterized and the value of this CPP as a tool to study the biology of SPIN proteins was demonstrated by analysis of the induced transcriptional changes. This analysis highlighted a significant number of genes that have previously been reported to be associated with CRI and/or cancer metastasis. Taken together, this data suggest that SPIN1 may play a role in CRI and/or the metastatic potential of tumors, and warrants further investigation. The availability of a potent and cell active inhibitor besides an inactive control compound will facilitate and expedite the future evaluation of SPIN1. For future work, we have named compound <b>35</b> VinSpinIn (Vinnie’s Spindlin Inhibitor) and <b>38</b> VinSpinIC (Vinnie’s Spindlin Inactive Control). For a limited period, this CPP, along with other highly characterized chemical probes, are freely available to the scientific community via online request from <a href="https://www.thesgc.org/chemical-probes" class="extLink">https://www.thesgc.org/chemical-probes</a>.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82537" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82537" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry General Materials and Methods</h3><div class="NLM_p">All solvents and reagents were purchased from commercial sources and used without further purification. Chemical reactions were carried out in anhydrous solvents, which were purchased from Acros Organics. Thin layer chromatography (TLC) was carried out using aluminum plates coated with 60 F254 silica gel and were visualized using UV light (254 or 365 nm) unless otherwise stated. Automated flash chromatography was carried out on a Biotage Isolera One flash column chromatography system (LPLC), using Biotage SNAP KP-Sil cartridges (unless otherwise state). All end compounds were purified to ≥95% purity, as indicated by analytical reverse phase high performance liquid chromatography (RP-HPLC) using UV and ELSD detection as well as <sup>1</sup>H NMR and <sup>13</sup>C NMR. Analytical HPLC and low resolution mass spectrometry was carried out on a waters LCMS system (Waters 2767 sample manager, Waters SFO System Fluidics organizer, Waters 2545 binary gradient module, 2 × Waters 515 HPLC pumps, Waters 2998 photodiode array detector, Waters 2424 ELS detector, Waters SQ detector 2). Analytical separation was carried out on a Kinetex 5 μM EVO C18 column (100 mm × 3.0 mm, 100 A) using a flow rate of 2 mL/min in a 3 min gradient elution. The mobile phase was a mixture of 93% water, 5% acetonitrile, and 2% of 0.5 M ammonium acetate adjusted to pH 6 with glacial acetic acid (solvent A) and 18% water, 80% acetonitrile, and 2% of 0.5 M ammonium acetate adjusted to pH 6 with glacial acetic acid (solvent B). Gradient elution was as follows: 95:5 (A/B) 0.35 min, 95:5 (A/B) to 5:95 (A/B) over 1 min, 5:95 (A/B) over 0.75 min, and then reversion back to 95:5 (A/B) over 0.1 min and 95:5 (A/B) over 0.8 min.</div><div class="NLM_p">NMR spectra were recorded using a Bruker Avance 400 MHz spectrometer using the indicated deuterated solvent. Chemical shifts (δ) are quoted in parts per million (ppm) and referenced to the residual solvent peak. Multiplicities are denoted as: s, singlet; d, doublet; t, triplet; and q, quartet, and derivatives thereof. Coupling constants are recorded in Hz and rounded to the nearest 0.1 Hz. Two-dimensional NMR experiments (COSY, HSQC) were used to aid the assignment of <sup>1</sup>H and <sup>13</sup>C spectra. Atom numbering in structures is purely for the purposes of assignment and does not reflect any numbering conventions.</div><div class="NLM_p last">All final compounds were examined by the SwissADME substructure matcher (<a href="http://swissadme.ch" class="extLink">http://swissadme.ch</a>) to confirm that they did not contain known classes of pan-assay interference motifs (PAINS).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Conditions (a): Synthesis of 1-(4-(Benzyloxy)-3-methoxyphenyl)cyclopentane-1-carbonitrile (<b>2</b>)</h3><div class="NLM_p last">The procedure of Barbasiewicz et al. was used with slight modification.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Potassium hydroxide (8.44 g, 150 mmol, 12 equiv) and water (2.8 mL) were added to a stirring solution of 2-(4-(benzyloxy)-3-methoxyphenyl)acetonitrile <b>1</b> (3.17 g, 12.51 mmol, 1 equiv) and 1,4-dibromobutane (2.99 mL, 25.03 mmol, 2.0 equiv) in toluene (100 mL). The mixture was cooled to 0 °C, and tetrabutylammonium bromide (60.5 mg, 0.188 mmol, 0.015 equiv) was added. The mixture was stirred at 0 °C for approximately 15 min, then heated to 80 °C with slow stirring (50 rpm) until most of the KOH had dissolved (approximately 1 h). The mixture was heated at reflux and stirred vigorously for a further 2 h. Water (75 mL) was added to the cooled reaction mixture. The organic phase was separated, washed with water (3 × 50 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), evaporated, and the residue was purified by chromatography using a gradient of 0–40% of [1:1 DCM:EtOAc] in cyclohexane to give compound <b>2</b> (3.545 g, 11.53 mmol, 92%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.23–7.16 (m, 4H), 6.82 (s, 2H), 4.13 (t, <i>J</i> = 6.3 Hz, 2H), 3.96 (s, 4H), 3.81 (d, <i>J</i> = 6.9 Hz, 2H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.16–1.88 (m, 10H), 1.32–1.22 (m, 1H), 0.62–0.56 (m, 2H), 0.35–0.30 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 149.74, 147.76, 136.95, 132.76 (4 × C), 128.59, 127.91, 127.23 (3 × CH), 124.58 (C), 118.03, 113.89, 110.28 (3 × CH), 71.06 (CH<sub>2</sub>), 56.18 (CH<sub>3</sub>), 47.39 (C), 40.37, 24.11 (2 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.80 min. MS (ESI+): <i>m</i>/<i>z</i> 325.2 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Conditions (b): Synthesis of 2-(4-(Benzyloxy)-3-methoxyphenyl)-2-methylpropanenitrile (<b>3</b>)</h3><div class="NLM_p last">A solution of the 2-(4-(benzyloxy)-3-methoxyphenyl)acetonitrile <b>1</b> (1.387 g, 5.48 mmol, 1 equiv) and sodium hydroxide (aq 50 wt %, 1.752 g, 21.90 mmol, 4 equiv) in DMSO (10 mL) were cooled to 0 °C. Iodomethane (1.36 mL, 21.90 mmol, 4 equiv) was added dropwise, and the mixture was stirred at room temperature for 1 h after the addition. The mixture was diluted with water (100 mL), and the product was extracted with toluene (3 × 30 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and evaporated to give compound <b>3</b> (1.508 g, 5.36 mmol, 98%) as a yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.45–7.42 (m, 2H), 7.40–7.34 (m, 2H), 7.33–7.29 (m, 1H), 7.00 (d, <i>J</i> = 2.2 Hz, 1H), 6.92 (dd, <i>J</i> = 8.4, 2.2 Hz, 1H), 6.87 (d, <i>J</i> = 8.4 Hz, 1H), 5.15 (s, 2H), 3.92 (s, 3H), 1.70 (s, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 149.76, 147.73, 136.93, 134.49 (4 × C), 128.59, 127.92, 127.24 (3 × CH), 124.71 (C), 117.07, 113.94, 109.43 (3 × CH), 71.06 (CH<sub>2</sub>), 56.19 (CH<sub>3</sub>), 36.76 (C), 29.25 (CH<sub>3</sub>). HPLC: <i>t</i><sub>R</sub> 1.80 min. MS (ESI+): <i>m</i>/<i>z</i> 299.6 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Conditions (c)</h3><div class="NLM_p last">The procedure of Sweis et al. was used with slight modification.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> A solution of <b>2</b> or <b>3</b> (7.0 mmol, 1 equiv) in 1:1 acetic acid:acetic anhydride (5 mL) was added dropwise to a solution of glacial acetic acid (5 mL), acetic anhydride (5 mL), and nitric acid (70%, 5 mL), maintaining the internal temperature of the reaction between 0 and −5 °C. The mixture was stirred at 0 °C for approximately 30 min after the addition. The reaction mixture was poured over crushed ice and stirred vigorously for approximately 10 min. The product was isolated by filtration and rinsed thoroughly with cold water.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 1-(4-(Benzyloxy)-5-methoxy-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>4</b>)</h3><div class="NLM_p last">Prepared using general conditions (c) from compound <b>2</b>; (95%) pale-yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.50 (s, 1H), 7.45–7.32 (m, 5H), 6.99 (s, 1H), 5.18 (s, 2H), 3.97 (s, 3H), 2.75–2.64 (m, 2H), 2.10–1.98 (m, 4H), 1.98–1.81 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.62, 147.46, 142.50, 135.38 (4 × C), 128.83, 128.53, 127.54 (3 × CH), 127.49, 122.81 (2 × C), 111.15, 110.92 (2 × CH), 71.37 (CH<sub>2</sub>), 56.47 (CH<sub>3</sub>), 45.43 (C), 38.95, 23.84 (2 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.87 min. MS (ESI+): <i>m</i>/<i>z</i> 370.6 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 2-(4-(Benzyloxy)-5-methoxy-2-nitrophenyl)-2-methylpropanenitrile (<b>5</b>)</h3><div class="NLM_p last">Prepared using general conditions (c) from compound <b>3</b>; (87%) pale-yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.45–7.31 (m, 6H), 7.06 (s, 1H), 5.17 (s, 2H), 3.98 (s, 3H), 1.89 (s, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.54, 147.40, 142.23, 135.36 (4 × C), 128.82, 128.52 (2 × CH), 127.77 (C), 127.53 (CH), 123.00 (C), 111.08, 110.19 (2 × CH), 71.39 (CH<sub>2</sub>), 56.47 (CH<sub>3</sub>), 36.44 (C), 28.36 (CH<sub>3</sub>). HPLC: <i>t</i><sub>R</sub> 1.71 min. MS (ESI+): <i>m</i>/<i>z</i> 327.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Conditions (d)</h3><div class="NLM_p last">Palladium (10% on carbon (50% wet), 0.2 wt eq) was added to a solution of Benzyl aryl ether <b>4</b>, <b>14</b>, or <b>15</b> (2.0 mmol, 1 equiv) in 9:1 EtOAc:EtOH (50 mL). The system was flushed with N<sub>2</sub>, followed by H<sub>2</sub>, and the mixture was stirred at room temperature under H<sub>2</sub>. The reaction was monitored closely by TLC and was complete in approximately 30 min. The catalyst was removed by filtration through a bed of Celite, and the solvent was evaporated. The residue was purified by automated flash chromatography using a gradient elution of 0–10% MeOH in DCM.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 1-(4-Hydroxy-5-methoxy-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>6</b>)</h3><div class="NLM_p last">Prepared using general conditions (d) from benzyl aryl ether <b>4</b>; (84%) pale-yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.46 (s, 1H), 6.96 (s, 1H), 5.86 (bs, 1H), 4.00 (s, 3H), 2.74–2.63 (m, 2H), 2.10–1.96 (m, 4H), 1.96–1.82 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 149.34, 145.26, 143.38, 126.20, 122.88 (5 × C), 112.56, 109.99 (2 × CH), 56.44 (CH<sub>3</sub>), 45.38 (C), 38.95, 23.80 (2 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.41 min. MS (ESI+): <i>m</i>/<i>z</i> 263.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Conditions (e)</h3><div class="NLM_p last">Phenol <b>6</b>, <b>12</b>, <b>13</b>, <b>16</b>, or <b>17</b> (3.0 mmol, 1 equiv), potassium carbonate (2 equiv), and the required alkyl bromide (1.5 equiv) (1-bromo-3-chloropropane for <b>7</b>, <b>18</b>, and <b>19</b>; (bromomethyl)cyclopropane for <b>14</b> and <b>15</b>; 1-bromo-4-chlorobutane for <b>20</b> and <b>21</b>) were heated in ACN (20 mL) at 80 °C overnight. The cooled reaction mixture was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 50–100% DCM in cyclohexane.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 1-(4-(3-Chloropropoxy)-5-methoxy-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>7</b>)</h3><div class="NLM_p last">Prepared using general conditions (e) from phenol <b>6</b>; (97%) yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.49 (s, 1H), 7.00 (s, 1H), 4.24 (t, <i>J</i> = 5.9 Hz, 2H), 3.97 (s, 3H), 3.78 (t, <i>J</i> = 6.2 Hz, 2H), 2.77–2.66 (m, 2H), 2.36–2.28 (m, 2H), 2.12–1.98 (m, 4H), 1.98–1.84 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 150.52, 145.67, 140.67, 125.56, 120.87 (5 × C), 108.95, 108.68 (2 × C), 64.03 (CH<sub>2</sub>), 54.49 (CH<sub>3</sub>), 43.49 (C), 39.21, 37.04, 29.92, 21.92 (4 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.73 min. MS (ESI+): <i>m</i>/<i>z</i> 339.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> General Conditions (f)</h3><div class="NLM_p last">Alkyl chloride <b>7</b> or <b>18</b>–<b>21</b> (0.25 mmol, 1 equiv), potassium iodide (2 equiv), potassium carbonate (3 equiv), and the required secondary amine (1.5 equiv) were heated in ACN at 80 °C (5 mL) for 48 h. The cooled reaction mixture was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 0–5% MeOH in DCM.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 1-(5-Methoxy-2-nitro-4-(3-(pyrrolidin-1-yl)propoxy)phenyl)cyclopentane-1-carbonitrile (<b>8</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) from alkyl chloride <b>7</b>; (91%) yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.50 (s, 1H), 6.96 (s, 1H), 4.15 (t, <i>J</i> = 6.6 Hz, 2H), 3.95 (s, 3H), 2.75–2.65 (m, 2H), 2.62 (t, <i>J</i> = 7.2 Hz, 2H), 2.55–2.48 (m, 4H), 2.10–1.97 (m, 6H), 1.97–1.83 (m, 2H), 1.83–1.74 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.30, 148.00, 142.69, 126.89, 122.87 (5 × C), 110.82, 110.39 (2 × CH), 68.03 (CH<sub>2</sub>), 56.45 (CH<sub>3</sub>), 54.22, 52.65 (2 × CH<sub>2</sub>), 45.39 (C), 38.94, 28.44, 23.83, 23.48 (4 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.33 min. MS (ESI+): <i>m</i>/<i>z</i> 374.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Conditions (g)</h3><div class="NLM_p last">Aspects of the procedures of Chae et al. and Magano et al. were used with modifications.<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62,63)</a> Dodecane-1-thiol (2.4 equiv) was added to a solution of potassium <i>tert</i>-butoxide (2.5 equiv) in dry DMF (40 mL) under N<sub>2</sub>, which resulted in a heavy white precipitate. The mixture was cooled in an ice bath, and a solution of methyl aryl ether <b>4</b>, <b>5</b>, or <b>8</b> (5.0 mmol, 1 equiv) in dry DMF (20 mL) was added dropwise, maintaining the temperature at 0 °C. After the addition, the mixture was heated to 50 °C for approximately 2 h and monitored by TLC. Solid sodium hydrogen sulfate (4 equiv) was added to the cooled reaction, and the mixture was stirred vigorously until a color change, from orange to pale yellow, occurred. The solvent was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 50–100% DCM in cyclohexane as eluent.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 1-(5-Hydroxy-2-nitro-4-(3-(pyrrolidin-1-yl)propoxy)phenyl)cyclopentane-1-carbonitrile (<b>9</b>)</h3><div class="NLM_p last">Prepared using general conditions (g) from methyl aryl ether <b>8</b> but was purified using a Biotage SNAP KP-NH column, with a gradient elution of 0–10% MeOH in DCM; (88%) yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.78 (s, 1H), 6.81 (s,1H), 4.01 (t, <i>J</i> = 5.0 Hz, 2H), 3.10–2.98 (m, 6H), 2.70–2.58 (m, 2H), 2.11–1.89 (m, 10 H), 1.89–1.74 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.33, 146.70, 135.96, 133.40, 123.53 (5 × C), 120.68, 118.32 (2 × CH), 72.75, 54.55, 53.79 (3 × CH<sub>2</sub>), 45.55 (C), 38.88, 26.65, 23.77, 23.21 (4 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.25 min. MS (ESI+): <i>m</i>/<i>z</i> 360.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Conditions (h)</h3><div class="NLM_p last">Phenol <b>9</b> (0.25 mmol, 1 equiv), potassium carbonate (2 equiv), and (bromomethyl)cyclopropane (1.5 equiv) were heated in DMF (5 mL) at 50 °C overnight. The cooled reaction mixture was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 0–5% MeOH in DCM. The product was immediately subjected to general conditions (i).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Conditions (i)</h3><div class="NLM_p last">The procedure of Sweis et al. was used with modifications.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Zinc powder (10 equiv) was added to the nitroarene (0.25 mmol, 1 equiv), and the mixture was stirred vigorously in glacial acetic acid (5 mL) at 110 °C for 2 h. The acetic acid was evaporated, and the residue was dissolved in DCM (2 mL). Solid potassium carbonate (10 equiv) was added, and the mixture was stirred vigorously until CO<sub>2</sub> was no longer liberated. The mixture was directly loaded onto a Biotage SNAP KP-NH column, and the product was eluted with an eluent of 0–10% MeOH in DCM.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 5′-Ethoxy-6′-(3-(pyrrolidin-1-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>10a</b>)</h3><div class="NLM_p last">Prepared using general conditions (h) and (i) from phenol <b>9</b>; (6%) colorless oil that solidifies on standing. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.82 (s, 1H), 6.79 (s, 1H), 4.10–4–03 (m, 4H), 2.64 (t, <i>J</i> = 8.0, 2H), 2.54–2.51 (m, 4H), 2.07–1.93 (m, 10H), 1.82–1.75 (m, 4H), 1.39 (t, <i>J</i> = 6.0, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.38, 149.69, 148.29, 144.17, 134.85 (5 × C), 110.83, 103.42 (2 × CH), 67.87, 66.88 (2 × CH<sub>2</sub>), 58.61 (C), 54.27, 53.30, 38.13, 28.96, 26.63, 23.46 (6 × CH<sub>2</sub>), 15.31 (CH<sub>3</sub>). HPLC: 97%, <i>t</i><sub>R</sub> 1.15 min. MS (ESI+): <i>m</i>/<i>z</i> 358.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 5′-Isopropoxy-6′-(3-(pyrrolidin-1-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>10b</b>)</h3><div class="NLM_p last">Prepared using general conditions (h) and (i) from phenol <b>9</b>; (18%) colorless oil that solidifies on standing. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.79 (s, 1H), 6.77 (s, 1H), 4.32–4.26 (m, 1H), 4.03 (t, <i>J</i> = 6.3 Hz, 2H), 2.64 (t, <i>J</i> = 8.0, 2H), 2.54–2.49 (m, 4H), 2.06–1.86 (m, 10H), 1.81–1.74 (m, 4H), 1.29 (d, J = 6.2, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.57, 151.02, 149.50, 142.47, 134.94 (5 × C), 114.75, 103.15, 74.07 (3 × CH), 67.53 (CH<sub>2</sub>), 58.52 (C), 54.31, 53.39, 38.13, 29.07, 26.60, 23.47 (6 × CH<sub>2</sub>), 22.44 (CH<sub>3</sub>). HPLC: 99%, <i>t</i><sub>R</sub> 1.13 min. MS (ESI+): <i>m</i>/<i>z</i> 372.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 5′-(Cyclopropylmethoxy)-6′-(3-(pyrrolidin-1-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>10c</b>)</h3><div class="NLM_p last">Prepared using general conditions (h) and (i) from phenol <b>9</b>; (6%) colorless oil that solidifies on standing. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.81 (s, 1H), 6.79 (s, 1H), 4.05 (t, <i>J</i> = 6.4 Hz, 2H), 3.79 (d, <i>J</i> = 7.0 Hz, 2H), 2.64 (t, <i>J</i> = 7.6, 2H), 2.55–2.47 (m, 2H), 2.10–1.88 (m, 10H), 1.83–1.72 (m, 4H), 1.31–1.19 (m, 1H), 0.61–0.54 (m, 2H), 0.33–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.50, 150.08, 149.18, 144.05, 135.05 (5 × C), 112.13, 103.44 (2 × CH), 76.68, 67.81 (2 × CH<sub>2</sub>), 58.62 (C), 54.30, 53.36, 38.10, 29.06, 26.61, 23.48 (6 × CH<sub>2</sub>), 10.86 (C), 3.13 (CH<sub>2</sub>). HPLC: 100%, <i>t</i><sub>R</sub> 1.13 min. MS (ESI+): <i>m</i>/<i>z</i> 384.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 5′-Methoxy-6′-(3-(pyrrolidin-1-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>11a</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>7</b>; (40%) colorless oil that solidifies on standing. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.80 (s, 1H), 6.75 (s, 1H), 4.07 (t, <i>J</i> = 7.6, 2H), 3.38 (s, 3H), 2.62 (t, <i>J</i> = 8.0, 2H), 2.53–2.48 (m, 4H), 2.07–1.90 (m, 10H), 1.80–1.73 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.53, 148.76, 148.25, 144.99, 134.75 (5 × C), 107.79, 103.11 (2 × CH), 67.84 (CH<sub>2</sub>), 58.75 (C), 57.70 (CH<sub>3</sub>), 54.25, 53.20, 38.11, 28.91, 26.68, 23.46 (6 × CH<sub>2</sub>). HPLC: 99%, <i>t</i><sub>R</sub> 1.05 min. MS (ESI+): <i>m</i>/<i>z</i> 344.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 6′-(3-(Isoindolin-2-yl)propoxy)-5′-methoxyspiro[cyclopentane-1,3′-indol]-2′-amine (<b>11b</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>7</b>; (29%) colorless oil that solidifies on standing. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.22–7.14 (m, 5H), 6.84 (s, 1H), 6.77 (s, 1H), 4.15 (t, <i>J</i> = 6.6 Hz, 2H), 3.95 (s, 4H), 3.85 (s, 3H), 2.93 (t, <i>J</i> = 7.3 Hz, 2H), 2.17–1.90 (m, 10H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.30, 148.76, 148.25, 145.11, 140.17, 134.90 (6 × C), 126.64, 122.26, 107.72, 103.30 (4 × CH), 67.61, 59.20 (CH<sub>2</sub>), 58.78 (C), 57.68 (CH<sub>3</sub>), 52.90, 38.08, 28.87, 26.69 (CH<sub>2</sub>). HPLC: 96%, <i>t</i><sub>R</sub> 1.19 min. MS (ESI+): <i>m</i>/<i>z</i> 392.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 5′-Methoxy-6′-(3-(4-methoxyisoindolin-2-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>11c</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>7</b>; (47%) colorless oil that solidifies on standing. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.16 (t, <i>J</i> = 7.8 Hz, 1H), 6.84 (s, 1H), 6.81 (d, <i>J</i> = 7.4 Hz, 1H) 6.77 (s, 1H), 6.69 (d, <i>J</i> = 8.1 Hz, 1H) 4.15 (t, <i>J</i> = 6.5 Hz, 2H), 3.97 (s, 2H), 3.94 (s, 2H), 3.85 (s, 3H), 3.81 (s, 3H), 2.92 (t, <i>J</i> = 7.3 Hz, 2H), 2.15–1.90 (m, 10H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.24, 154.56, 148.80, 148.26, 145.12, 142.10, 134.88 (7 × C), 128.25 (CH), 127.84 (C), 114.74, 108.53, 107.79 103.33 (4 × CH), 67.59, 59.60 (2 × CH<sub>2</sub>), 58.77 (C), 57.71 (CH<sub>3</sub>), 56.46 (CH<sub>2</sub>), 55.25 (CH<sub>3</sub>), 51.97, 38.07, 28.90, 26.69 (4 × CH<sub>2</sub>). HPLC: 98%, <i>t</i><sub>R</sub> 1.33 min. MS (ESI+): <i>m</i>/<i>z</i> 422.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 5′-Methoxy-6′-(3-(5-methoxyisoindolin-2-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>11d</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>7</b>; (24%) colorless oil that solidifies on standing. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.08 (d, <i>J</i> = 8.1 Hz, 1H), 6.84 (s, 1H), 6.77–6.72 (m, 3H), 4.15 (t, <i>J</i> = 6.6 Hz, 2H), 3.92 (s, 2H), 3.89 (s, 2H), 3.85 (s, 3H), 3.78 (s, 3H), 2.91 (t, <i>J</i> = 7.3 Hz, 2H), 2.14–1.93 (m, 10H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.37, 158.96 148.74, 148.23, 145.07, 141.64, 134.85, 132.23 (8 × C), 122.87, 112.52, 107.97, 107.70 103.24 (5 × CH), 67.60, 59.40 (2 × CH<sub>2</sub>), 58.77 (C), 58.59 (CH<sub>2</sub>), 57.67, 55.47 (2 × CH<sub>3</sub>), 52.94, 38.09, 28.85, 26.69 (4 × CH<sub>2</sub>). HPLC: 98%, <i>t</i><sub>R</sub> 1.25 min. MS (ESI+): <i>m</i>/<i>z</i> 422.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_99" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 1-(4-(Benzyloxy)-5-hydroxy-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>12</b>)</h3><div class="NLM_p last">Prepared using general conditions (g) from methyl aryl ether <b>4</b>; (86%) yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.59 (s, 1H), 7.47–7.38 (m, 5H), 7.09 (s, 1H), 6.09 (s, 1H), 5.18 (s, 2H), 2.73–2.63 (m, 2H), 2.09–1.97 (m, 4H), 1.94–1.81 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 149.53, 144.63, 142.14, 134.57 (4 × C), 129.11, 129.01 (2 × CH), 128.72 (C), 128.16 (CH), 122.66 (C), 114.47, 110.30 (2 × CH), 71.88 (CH<sub>2</sub>), 44.97 (C), 39.03, 23.75 (2 × CH<sub>2</sub>). HPLC: 98%, <i>t</i><sub>R</sub> 1.64 min. MS (ESI+): <i>m</i>/<i>z</i> 339.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 2-(4-(Benzyloxy)-5-hydroxy-2-nitrophenyl)-2-methylpropanenitrile (<b>13</b>)</h3><div class="NLM_p last">Prepared using general conditions (g) from methyl aryl ether <b>5</b>; (79%) yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.51 (s, 1H), 7.47–7.36 (m, 5H), 7.12 (s, 1H), 5.17 (s, 2H), 1.87 (s, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 149.61, 144.65, 141.89, 134.61, 129.24 (5 × C), 129.11, 129.01, 128.17 (3 × CH), 122.60 (C), 113.50, 110.28 (2 × CH), 71.87 (CH<sub>2</sub>), 35.43 (C), 28.83 (CH<sub>3</sub>). HPLC: <i>t</i><sub>R</sub> 1.63 min. MS (ESI+): <i>m</i>/<i>z</i> 330.6 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 1-(4-(Benzyloxy)-5-(cyclopropylmethoxy)-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>14</b>)</h3><div class="NLM_p last">Prepared using general conditions (e) from phenol <b>12</b>; (99%) yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.51 (s, 1H),7.46–7.32 (m, 5H), 6.99 (s, 1H), 5.19 (s, 2H), 3.97 (d, <i>J</i> = 6.9 Hz, 2H), 2.73–2.61 (m, 2H), 2.10–1.96 (m, 4H), 1.96–1.81 (m, 2H), 1.37–1.23 (m, 1H), 0.71–0.67 (m, 2H), 0.42–0.38 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.38, 147.88, 142.54, 135.72 (4 × C), 128.75, 128.37 (2 × CH), 127.56 (C), 127.34 (CH), 122.85 (C), 113.23, 112.04 (2 × CH), 74.57, 71.45 (2 × CH), 45.36 (C), 38.94, 23.83 (2 × CH<sub>2</sub>), 10.14 (CH<sub>3</sub>), 3.48 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 2.03 min. MS (ESI+): <i>m</i>/<i>z</i> 410.7 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 2-(4-(Benzyloxy)-5-(cyclopropylmethoxy)-2-nitrophenyl)-2-methylpropanenitrile (<b>15</b>)</h3><div class="NLM_p last">Prepared using general conditions (e) from phenol <b>13</b>; (92%) yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.46–7.31 (m, 6H), 7.07 (s, 1H), 5.18 (s, 2H), 3.98 (d, <i>J</i> = 6.9 Hz, 2H), 1.87 (s, 6H), 1.37–1.28 (m, 1H), 0.71–0.66 (m, 2H), 0.42–0.378 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.30, 147.81, 142.26, 135.71 (4 × C), 128.73, 128.36 (2 × CH), 127.86 (C), 127.33 (CH), 123.04 (C), 112.46, 111.98 (2 × CH), 74.55, 71.47 (2 × CH<sub>2</sub>), 36.34 (C), 28.39 (CH<sub>3</sub>), 10.12 (CH), 3.46 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.84 min. MS (ESI+): <i>m</i>/<i>z</i> 367.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 1-(5-(Cyclopropylmethoxy)-4-hydroxy-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>16</b>)</h3><div class="NLM_p last">Prepared using general conditions (d) from benzyl aryl ether <b>14</b>; (76%) pale-yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.47 (s, 1H), 6.92 (s, 1H), 3.96 (d, <i>J</i> = 7.2 Hz, 2H), 2.72–2.59 (m, 2H), 2.09–1.94 (m, 4H), 1.94–1.81 (m, 2H), 1.36–1.24 (m, 1H), 0.76–0.68 (m, 2H), 0.43–0.37 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 148.90, 145.45, 143.20, 126.11, 122.91 (5 × C), 112.51, 111.00 (2 × CH), 74.67 (CH<sub>2</sub>), 45.36 (C), 38.95, 23.78 (2 × CH<sub>2</sub>), 10.00 (CH), 3.54 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.67 min. MS (ESI+): <i>m</i>/<i>z</i> 320.5 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 1-(5-(Cyclopropylmethoxy)-4-hydroxy-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>17</b>)</h3><div class="NLM_p last">Prepared using general conditions (d) from benzyl aryl ether <b>15</b>; (95%) pale-yellow oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.36 (s, 1H), 7.02 (s, 1H), 3.98 (d, <i>J</i> = 7.2 Hz, 2H), 1.86 (s, 6H), 1.36–1.24 (m, 1H), 0.74–0.70 (m, 2H), 0.42–0.39 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.20, 148.01, 142.60, 127.54, 122.82 (5 × C), 113.00, 111.38 (2 × CH), 74.40, 66.14 (2 × CH<sub>2</sub>), 45.34 (C), 41.16, 38.96, 31.92, 23.82 (4 × CH<sup>2</sup>), 10.10 (C), 3.40 (CH<sup>2</sup>). HPLC: <i>t</i><sub>R</sub> 1.88 min. MS (ESI+): <i>m</i>/<i>z</i> 379.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 1-(4-(3-Chloropropoxy)-5-(cyclopropylmethoxy)-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>18</b>)</h3><div class="NLM_p last">Prepared using general conditions (e) from phenol <b>16</b>; (92%) yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.48 (s, 1H), 6.97 (s, 1H), 4.21 (t, <i>J</i> = 5.9 Hz, 2H), 3.93 (d, <i>J</i> = 6.9 Hz, 2H), 3.77 (t, <i>J</i> = 6.2 Hz, 2H), 2.74–2.62 (m, 2H), 2.33–2.27 (m, 2H), 2.10–1.96 (m, 4H), 1.95–1.82 (m, 2H), 1.34–1.24 (m, 1H), 0.70–0.66 (m, 2H), 0.41–0.37 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.20, 148.01, 142.60, 127.54, 122.82 (5 × C), 113.00, 111.38 (2 × CH), 74.40, 66.14 (2 × CH<sub>2</sub>), 45.34 (C), 41.16, 38.96, 31.92, 23.82 (4 × CH<sub>2</sub>), 10.10 (C), 3.40 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.88 min. MS (ESI+): <i>m</i>/<i>z</i> 379.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 2-(4-(3-Chloropropoxy)-5-(cyclopropylmethoxy)-2-nitrophenyl)-2-methylpropanenitrile (<b>19</b>)</h3><div class="NLM_p last">Prepared using general conditions (e) from phenol <b>17</b>; (99%) yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.37 (s, 1H), 7.03 (s, 1H), 4.20 (t, <i>J</i> = 5.8 Hz, 2H), 3.94 (d, <i>J</i> = 6.8 Hz, 2H), 3.77 (t, <i>J</i> = 6.2 Hz, 2H), 2.32–2.26 (m, 2H), 1.86 (s, 6H), 1.34–1.23 (m, 1H), 0.69–0.65 (m, 2H), 0.40–0.36 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.15, 147.94, 142.32, 127.85, 122.97 (5 × C), 112.18, 111.31 (2 × CH), 74.38, 66.15, 41.18 (3 × CH<sub>2</sub>), 36.20 (C), 31.92 (CH<sub>2</sub>), 28.45 (CH<sub>3</sub>), 10.08 (CH), 3.38 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.88 min. MS (ESI+): <i>m</i>/<i>z</i> 370.6 [M + NH<sub>4</sub>]<sup>+</sup>.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 1-(4-(4-Chlorobutoxy)-5-(cyclopropylmethoxy)-2-nitrophenyl)cyclopentane-1-carbonitrile (<b>20</b>)</h3><div class="NLM_p last">Prepared using general conditions (e) from phenol <b>16</b>; (87%) yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.44 (s, 1H), 6.96 (s, 1H), 4.10 (t, <i>J</i> = 5.7 Hz, 2H), 3.93 (d, <i>J</i> = 6.9 Hz, 2H), 3.67 (t, <i>J</i> = 6.2 Hz, 2H), 2.72–2.64 (m, 2H), 2.09–1.96 (m, 8H), 1.96–1.79 (m, 2H), 1.36–1.25 (m, 1H), 0.71–0.66 (m, 2H), 0.41–0.37 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.12, 148.16, 142.62, 127.28, 122.84 (5 × C), 112.93, 110.96 (2 × CH), 74.43, 68.95 (2 × CH<sub>2</sub>), 45.34 (C), 44.59, 38.95, 29.41, 26.92, 26.31, 23.82 (6 × CH<sub>2</sub>), 10.11 (CH), 3.39 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.92 min. MS (ESI+): <i>m</i>/<i>z</i> 393.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 2-(4-(4-Chlorobutoxy)-5-(cyclopropylmethoxy)-2-nitrophenyl)-2-methylpropanenitrile (<b>21</b>)</h3><div class="NLM_p last">Prepared using general conditions (e) from phenol <b>17</b>; (100%) yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33 (s, 1H), 7.03 (s, 1H), 4.09 (t, <i>J</i> = 5.8 Hz, 2H), 3.94 (d, <i>J</i> = 6.9 Hz, 2H), 3.66 (t, <i>J</i> = 6.3 Hz, 1H), 2.03–1.98 (m, 4H), 1.87 (s, 6H), 1.35–1.25 (m, 1H), 0.71–0.65 (m, 2H), 0.41–0.36 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 152.04, 148.09, 142.33, 127.57, 123.02 (5 × C), 112.14, 110.89 (2 × CH), 74.41, 68.96, 44.60 (3 × CH<sub>2</sub>), 36.26 (C), 29.40 (CH<sub>2</sub>), 28.43 (CH<sub>3</sub>), 26.30 (CH<sub>2</sub>), 10.09 (CH), 3.38 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.93 min. MS (ESI+): <i>m</i>/<i>z</i> 384.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 5′-(Cyclopropylmethoxy)-6′-(3-(5-methoxyisoindolin-2-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>22</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>18</b>; (42%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.08 (d, <i>J</i> = 8.1 Hz, 1H), 6.81 (s, 1H), 6.80 (s, 1H), 4.11 (t, <i>J</i> = 6.4 Hz, 2H), 3.92 (s, 2H), 3.89 (s, 2H), 3.80 (d, <i>J</i> = 6.9 Hz, 2H), 3.78 (s, 3H), 2.92 (t, <i>J</i> = 7.3 Hz, 2H), 2.12–1.90 (m, 10H), 1.31–1.21 (m, 1H), 0.60–0.56 (m, 2H), 0.33–0.30 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.61, 158.96, 150.02, 148.75, 144.07, 141.62, 134.90, 132.20 (8 × C), 122.86, 112.51, 112.04, 107.97, 103.30 (5 × CH), 76.63, 67.54, 59.40, 58.59 (4 × CH<sub>2</sub>), 58.58 (C), 55.47 (CH<sub>3</sub>), 53.01, 38.16, 28.98, 26.60 (4 × CH<sub>2</sub>), 10.87 (CH), 3.16 (CH<sub>2</sub>). HPLC: 98%, <i>t</i><sub>R</sub> 1.39 min. MS (ESI+): <i>m</i>/<i>z</i> 462.8 [M + H]<sup>+</sup>.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 5′-(Cyclopropylmethoxy)-6′-(3-(isoindolin-2-yl)propoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>23</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>18</b>; (45%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.23–7.16 (m, 4H), 6.82 (s, 2H), 4.13 (t, <i>J</i> = 6.3 Hz, 2H), 3.96 (s, 4H), 3.81 (d, <i>J</i> = 6.9 Hz, 2H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.16–1.88 (m, 10H), 1.32–1.22 (m, 1H), 0.62–0.56 (m, 2H), 0.35–0.30 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.36, 150.04, 149.09, 144.16, 140.16, 135.14 (6 × C), 126.65, 122.27, 112.02, 103.56 (4 × CH), 76.64, 67.55, 59.21 (3 × CH<sub>2</sub>), 58.64 (C), 52.99, 38.09, 29.00, 26.62 (4 × CH<sub>2</sub>), 10.87 (CH), 3.16 (CH<sub>2</sub>). HPLC: 100%, <i>t</i><sub>R</sub> 1.40 min. MS (ESI+): <i>m</i>/<i>z</i> 432.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 5′-(Cyclopropylmethoxy)-6′-(4-(isoindolin-2-yl)butoxy)spiro[cyclopentane-1,3′-indol]-2′-amine (<b>24</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>20</b>; (44%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.21–7.16 (m, 4H), 6.82 (s, 1H), 6.80 (s, 1H), 4.05 (t, <i>J</i> = 6.4 Hz, 2H), 3.93 (s, 4H), 3.81 (d, <i>J</i> = 7.0 Hz, 2H), 2.79 (t, <i>J</i> = 7.3 Hz, 2H), 2.10–1.88 (m, 10H), 1.82–1.74 (m, 2H), 1.33–1.22 (m, 1H), 0.61–0.54 (m, 2H), 0.35–0.30 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.49, 150.10, 149.06, 144.09, 140.21, 140.21, 134.97 (7 × C), 126.62, 122.25, 112.08, 103.36 (4 × CH), 76.63, 69.07, 59.11 (3 × CH<sub>2</sub>), 58.64 (C), 55.84, 38.11, 27.28, 26.61, 25.61 (5 × CH<sub>2</sub>), 10.88 (CH), 3.16 (CH<sub>2</sub>). HPLC: 100%, <i>t</i><sub>R</sub> 1.40 min. MS (ESI+): <i>m</i>/<i>z</i> 446.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 5-(Cyclopropylmethoxy)-6-(3-(isoindolin-2-yl)propoxy)-3,3-dimethyl-3<i>H</i>-indol-2-amine (<b>25</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>19</b>; (42%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.22–7.16 (m, 4 H), 6.84 (s, 1H), 6.79 (s, 1H), 4.13 (t, <i>J</i> = 6.3 Hz, 2H), 3.96 (s, 4H), 3.82 (d, <i>J</i> = 6.9 Hz, 2H), 2.95 (t, <i>J</i> = 7.4, 2H), 2.15–2.07 (m, 2H), 1.33–1.23 (m, 7H), 0.63–0.55 (m, 2H), 0.36–0.29 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.11, 150.11, 147.80, 144.45, 140.14, 133.12 (6 × C), 126.66, 122.27, 111.42, 103.69 (4 × CH), 76.31, 67.61, 59.20, 52.96 (4 × CH<sub>2</sub>), 48.73 (C), 28.96 (CH<sub>2</sub>), 25.08 (CH<sub>3</sub>), 10.81 (CH), 3.16 (CH<sub>2</sub>). HPLC: 99%, <i>t</i><sub>R</sub> 1.34 min. MS (ESI+): <i>m</i>/<i>z</i> 406.7 [M + H]<sup>+</sup>.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 5-(Cyclopropylmethoxy)-6-(4-(isoindolin-2-yl)butoxy)-3,3-dimethyl-3<i>H</i>-indol-2-amine (<b>26</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>21</b>. (39%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.21–7.16 (m, 4H), 6.83 (s, 1H), 6.79 (s, 1H), 4.06 (t, <i>J</i> = 6.4 Hz, 2H), 3.93 (s, 4H), 3.82 (d, <i>J</i> = 6.9 Hz, 2H), 2.79 (t, <i>J</i> = 7.4 Hz, 2H), 1.98–1.87 (m, 2H), 1.84–1.75 (m, 2H), 1.32–1.21 (m, 7H), 0.61–0.54 (m, 2H), 0.37–0.28 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.22, 150.13, 148.57, 144.25, 140.19, 133.28 (6 × C), 126.61, 122.24, 111.47, 103.68 (4 × CH), 76.33, 69.10, 59.10, 55.84 (4 × CH<sub>2</sub>), 48.77 (C), 27.27, 25.59 (2 × CH<sub>2</sub>), 25.09 (CH<sub>3</sub>), 10.83 (CH), 3.16 (CH<sub>2</sub>). HPLC: 95%, <i>t</i><sub>R</sub> 1.34 min. MS (ESI+): <i>m</i>/<i>z</i> 420.8 [M + H]<sup>+</sup>.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 5-(Prop-2-yn-1-yloxy)isoindoline (<b>27</b>)</h3><div class="NLM_p last">Propargyl bromide (80 wt % in toluene, 0.30 mL, 2.81 mmol, 1.1 equiv) was added to a stirring mixture of <i>tert</i>-butyl 5-hydroxyisoindoline-2-carboxylate (0.600 g, 2.55 mmol, 1 equiv) and potassium carbonate (0.705 g, 5.10 mmol, 2 equiv) in ACN (20 mL), and the mixture was stirred at 40 °C overnight. The solvent was evaporated and the residue was purified by automated flash chromatography using a gradient elution of 0–5% MeOH in DCM (and using Seebach’s TLC stain to visualize the product) to give a colorless oil (0.630 g, 2.30 mmol, 90%). The oil was dissolved in dry DCM (20 mL), and trifluoroacetic acid (TFA, 2 mL) was added. The solution was stirred at room temperature for approximately 2 h. The solvent was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 0–10% of [7 N NH<sub>3</sub> in MeOH] in DCM (and using ninhydrin TLC stain to visualize the product) to give compound <b>27</b> (0.413 g, 2.38 mmol, 93% (84% overall)) as a brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.16 (d, <i>J</i> = 8.2 Hz, 1H), 6.90–6.82 (m, 2H), 4.69 (d, <i>J</i> = 2.4 Hz, 2H), 4.23 (s, 2H), 4.20 (s, 2H), 2.51 (t, <i>J</i> = 2.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.02, 143.17, 134.49 (3 × C), 122.90, 113.85, 109.01 (3 × CH), 78.66 (C), 75.45 (CH), 56.12, 53.04, 52.29 (3 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 0.71 min. MS (ESI+): <i>m</i>/<i>z</i> 174.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 5-(But-3-yn-1-yloxy)isoindoline (<b>28</b>)</h3><div class="NLM_p last">But-3-yn-1-ol (0.39 mL, 5.10 mmol, 1.2 equiv) was added to a solution of <i>tert</i>-butyl 5-hydroxyisoindoline-2-carboxylate (1.000 g, 4.25 mmol, 1 equiv) and triphenylphosphine (1.115 g, 4.25 mmol, 1 equiv) in dry THF (20 mL) under N<sub>2</sub>. The solution was cooled to 0 °C, and diethyl azodicarboxylate (DEAD, 0.669 mL, 4.25 mmol, 1 equiv) was added. The solution was heated at 60 °C for 5 h. The solvent was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 50–100% DCM in cyclohexane (and using Seebach’s TLC stain to visualize the product) to a colorless oil (0.509 g, 1.77 mmol, 42%). The oil was dissolved in dry DCM (20 mL), and trifluoroacetic acid (TFA, 2 mL) was added. The solution was stirred at room temperature for approximately 2 h. The solvent was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 0–10% of [7 N NH<sub>3</sub> in MeOH] in DCM (and using ninhydrin TLC stain to visualize the product) to give compound <b>28</b> (0.302 g, 1.61 mmol, 91% (38% overall)) as a brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.13 (d, <i>J</i> = 8.2 Hz, 1H), 6.83–6.72 (m, 2H), 4.20 (s, 2H), 4.18 (s, 2H), 4.08 (t, <i>J</i> = 7.0 Hz, 2H), 2.67 (td, <i>J</i> = 7.0, 2.7 Hz, 2H), 2.04 (t, <i>J</i> = 2.7 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.87, 143.29, 134.07 (3 × C), 122.90, 113.57, 108.70 (3 × CH), 80.46 (C), 69.87 (CH), 66.33, 53.06, 52.31, 19.56 (4 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.00 min. MS (ESI+): <i>m</i>/<i>z</i> 188.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 2-(5-(Cyclopropylmethoxy)-2-nitro-4-(3-(5-(prop-2-yn-1-yloxy)isoindolin-2-yl)propoxy)phenyl)-2-methylpropanenitrile (<b>29</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) from alkyl chloride <b>19</b> and alkyne functionalized isoindoline <b>27</b>; (96%) brown oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.37 (s, 1H), 7.12 (d, <i>J</i> = 8.0 Hz, 1H), 7.04 (s, 1H), 6.85–6.80 (m, 2H), 4.67 (d, <i>J</i> = 2.4 Hz, 2H), 4.19 (t, <i>J</i> = 6.4 Hz, 2H), 4.00–3.90 (m, 6H), 2.95 (t, <i>J</i> = 6.9 Hz, 2H), 2.50 (t, <i>J</i> = 2.4 Hz, 1H), 2.19–2.09 (m, 2H), 1.87 (s, 6H), 1.36–1.25 (m, 1H), 0.74–0.62 (m, 2H), 0.46–0.34 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.05, 151.91, 148.33, 142.46, 141.03, 132.57, 127.30, 123.09 (8 × C), 123.00, 113.85, 112.48, 110.96, 109.25 (5 × CH), 78.62 (C), 75.46 (CH), 74.47, 67.66, 59.26, 58.51, 56.13, 52.46 (6 × CH<sub>2</sub>), 36.34 (C), 28.39 (CH<sub>3</sub>), 28.21 (CH<sub>2</sub>), 10.14 (CH), 3.42 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.75 min. MS (ESI+): <i>m</i>/<i>z</i> 490.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 2-(4-(3-(5-(But-3-yn-1-yloxy)isoindolin-2-yl)propoxy)-5-(cyclopropylmethoxy)-2-nitrophenyl)-2-methylpropanenitrile (<b>30</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) from alkyl chloride <b>19</b> and alkyne functionalized isoindoline <b>28</b>; (98%) brown oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.38 (s, 1H), 7.10 (d, <i>J</i> = 8.0 Hz, 1H), 7.03 (s, 1H), 6.79–6.74 (m, 2H), 4.19 (t, <i>J</i> = 6.4 Hz, 2H), 4.07 (t, <i>J</i> = 7.1 Hz, 2H), 3.98–3.90 (m, 6H), 2.94 (t, <i>J</i> = 7.0 Hz, 2H), 2.66 (td, <i>J</i> = 7.0, 2.7 Hz, 2H), 2.17–2.11 (m, 2H), 2.03 (t, <i>J</i> = 2.7 Hz, 1H), 1.87 (s, 6H), 1.36–1.25 (m, 1H), 0.70–0.65 (m, 2H), 0.42–0.38 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 157.92, 151.92, 148.33, 142.45, 141.10, 132.11, 127.30, 123.08 (7 × C), 123.01, 113.61, 112.46, 110.96, 108.97 (5 × CH), 80.44 (C), 74.47 (CH<sub>2</sub>), 69.87 (CH), 67.67, 66.35, 59.26, 58.51, 52.47 (5 × CH<sub>2</sub>), 36.30 (C), 28.40 (CH<sub>3</sub>), 28.21, 19.54 (CH<sub>2</sub>), 10.14 (CH), 3.42 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.77 min. MS (ESI+): <i>m</i>/<i>z</i> 504.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 2-(5-(Cyclopropylmethoxy)-2-nitro-4-(4-(5-(prop-2-yn-1-yloxy)isoindolin-2-yl)butoxy)phenyl)-2-methylpropanenitrile (<b>31</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) from alkyl chloride <b>21</b> and alkyne functionalized isoindoline <b>27</b>; (52%) brown oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.34 (s, 1H), 7.11 (d, <i>J</i> = 8.0 Hz, 1H), 7.03 (s, 1H), 6.86–6.78 (m, 2H), 4.67 (d, <i>J</i> = 2.4 Hz, 2H), 410 (t, <i>J</i> = 6.4 Hz, 2 H), 3.98–3.85 (m, 6H), 2.81 (t, <i>J</i> = 7.2 Hz, 2H), 2.50 (t, <i>J</i> = 2.4 Hz, 1H), 2.0–1.94 (m, 2H), 1.87 (s, 6H), 1.82–1.74 (m, 2H), 1.36–1.26 (m, 1H), 0.70–0.63 (m, 2H), 0.42–0.38 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 156.94, 151.94, 148.31, 142.40, 141.40, 132.95, 127.23, 123.08 (8 × C), 122.95, 113.66, 112.30, 110.66, 109.23 (5 × CH), 78.95 (C), 75.44 (CH), 74.44, 69.36, 59.20, 58.44, 56.11, 55.59 (6 × CH<sub>2</sub>), 36.25 (C), 28.44 (CH<sub>3</sub>), 26.70, 25.21 (2 × CH<sub>2</sub>), 10.15 (CH), 3.44 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.75 min. MS (ESI+): <i>m</i>/<i>z</i> 504.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 2-Azido-1-(4-(2-(pyrrolidin-1-yl)ethyl)piperidin-1-yl)ethan-1-one (<b>32</b>)</h3><div class="NLM_p last">2-Azidoacetic acid (0.23 mL, 3.03 mmol, 1.3 equiv) was added to a stirring solution of <i>p</i>-toluenesulfonyl chloride (0.489 g, 2.57 mmol, 1.1 equiv) and diisopropylaminomethyl-polystyrene resin (3 mmol/g, 2.333 g, 3 equiv) in dry DCM (20 mL), and the mixture was stirred a room temperature for 1 h. 4-(2-(Pyrrolidin-1-yl)ethyl)piperidine (0.425 g, 2.33 mmol, 1 equiv) was added, and the mixture was stirred for a further 1 h. The mixture was filtered, and the filtrate was washed with Na<sub>2</sub>CO<sub>3</sub> (1 M, 50 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and evaporated to give compound <b>32</b> (0.5217 g, 1.966 mmol, 84%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.55 (d, <i>J</i> = 13.3 Hz, 1H), 3.92 (s, 1H), 3.61 (d, <i>J</i> = 13.5 Hz, 1H), 3.02 (td, <i>J</i> = 13.0, 2.8 Hz, 0H), 2.62 (td, <i>J</i> = 12.9, 2.9 Hz, 1H), 2.55–2.40 (m, 6H), 1.84–1.71 (m, 6H), 1.65–1.45 (m, 3H),1.22–1.08 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.27 (C), 54.26, 53.77, 50.78, 45.39, 42.49, 35.36 (6 × CH<sub>2</sub>), 34.29 (CH), 32.75, 31.82, 23.40 (3 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 0.67 min. MS (ESI+): <i>m</i>/<i>z</i> 266.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 3-Azido-1-(4-(2-(pyrrolidin-1-yl)ethyl)piperidin-1-yl)propan-1-one (<b>33</b>)</h3><div class="NLM_p last">3-Azidopropanoic acid (0.24 mL, 2.60 mmol, 1.3 equiv) was added to a stirring solution of <i>p</i>-toluenesulfonyl chloride (0.418 g, 2.20 mmol, 1.1 equiv) and resin-bound diisopropylaminomethyl-polystyrene (3 mmol/g, 2.000 g 3 equiv) in dry DCM (20 mL), and the mixture was stirred a room temperature for 1 h. 4-(2-(Pyrrolidin-1-yl)ethyl)piperidine (0.365 g, 2.00 mmol, 1 equiv) was added, and the mixture was stirred for a further 1 h. The mixture was filtered, and the filtrate was washed with Na<sub>2</sub>CO<sub>3</sub> (1 M, 50 mL), dried (Na<sub>2</sub>SO<sub>4</sub>), and evaporated to give crude compound <b>33</b> (89% but with a purity of approximately 90%) as a yellow solid. Compound <b>33</b> was unstable and following the workup, approximately 10% of an alkene byproduct had formed (presumably caused by elimination of the azide during evaporation) and therefore was not characterized but was used immediately as a crude mixture in the proceeding click reaction.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> General Conditions (j)</h3><div class="NLM_p last">Copper powder (<425 μm, 25 equiv), azide <b>32</b> or <b>33</b> (2 equiv), and alkyne <b>29</b>, <b>30</b>, or <b>31</b> (0.20 mmol, 1 equiv) were stirred vigorously in DMF (5 mL) at 60 °C, under N<sub>2</sub>, until all the alkyne starting material had been consumed (approximately 2 h). The cooled reaction mixture was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 0–10% of [7 N NH<sub>3</sub> in MeOH] in DCM.</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 2-(4-(((2-(3-((2-Amino-5-(cyclopropylmethoxy)-3,3-dimethyl-3<i>H</i>-indol-6-yl)oxy)propyl)isoindolin-5-yl)oxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)-1-(4-(2-(pyrrolidin-1-yl)ethyl)piperidin-1-yl)ethan-1-one (<b>34</b>)</h3><div class="NLM_p last">Prepared using general conditions (j) and (i) from alkyne <b>29</b> and azide <b>32</b>; (17%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.78 (s, 1H), 7.08 (d, <i>J</i> = 8.1 Hz, 1H), 6.85–6.76 (m, 4H), 5.26–5.16 (m, 4H), 4.51 (d, <i>J</i> = 13.4 Hz, 1H), 4.11 (t, <i>J</i> = 6.2 Hz, 2H), 3.91–3.78 (m, 7H), 3.11 (t, <i>J</i> = 6.5 Hz, 1H), 2.91 (t, <i>J</i> = 7.3 Hz, 2H), 2.64 (t, <i>J</i> = 13.0 Hz, 1H), 2.55–2.45 (m, 6H), 2.11–2.05 (m, 2H), 1.84–1.72 (m, 6H), 1.68–1.54 (m, 1H), 1.54–1.45 (m, 2H), 1.36–1.21 (m, 7H), 1.21–1.08 (m, 2H), 0.64–0.56 (m, 2H), 0.34–0.29 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.37, 162.92, 157.57, 150.37, 147.91, 144.89, 144.55, 141.67, 133.39, 132.81 (10 × C), 124.43, 122.97, 113.57, 111.46, 109.08, 102.28 (6 × CH), 76.23, 67.58, 62.32, 59.34, 58.55, 54.15, 53.64, 52.84, 51.05 (9 × CH<sub>2</sub>), 48.26 (C), 45.67, 42.75, 35.03 (3 × CH<sub>2</sub>), 34.17 (CH), 32.62, 31.68, 28.85 (3 × CH<sub>2</sub>), 25.09 (CH<sub>3</sub>), 23.39 (CH<sub>2</sub>), 10.75 (CH), 3.17 (CH2). HPLC: 98%, <i>t</i><sub>R</sub> 1.20 min. MS (ESI+): <i>m</i>/<i>z</i> 726.0 [M + H]<sup>+</sup>, 363.8 [M + 2H]<sup>2+</sup>.</div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 2-(4-(2-((2-(3-((2-Amino-5-(cyclopropylmethoxy)-3,3-dimethyl-3<i>H</i>-indol-6-yl)oxy)propyl)isoindolin-5-yl)oxy)ethyl)-1<i>H</i>-1,2,3-triazol-1-yl)-1-(4-(2-(pyrrolidin-1-yl)ethyl)piperidin-1-yl)ethan-1-one (<b>35</b>)</h3><div class="NLM_p last">Prepared using general conditions (j) and (i) from alkyne <b>30</b> and azide <b>32</b>; (47%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.62 (s, 1H), 7.06 (d, <i>J</i> = 6.5 Hz, 1H), 6.82 (s, 1H), 6.78–6.70 (m, 3H), 5.24–5.14 (m, 2H), 4.51 (d, <i>J</i> = 13.4 Hz, 1H), 4.22 (t, <i>J</i> = 6.5 Hz, 2H), 4.11 (t, <i>J</i> = 6.3 Hz, 2H), 3.93–3.76 (m, 7H), 3.21 (t, <i>J</i> = 6.5 Hz, 2H), 3.09 (td, <i>J</i> = 13.2, 2.5 Hz, 1H), 2.91 (t, <i>J</i> = 7.3 Hz, 2H), 2.63 (td, <i>J</i> = 12.9, 2.6 Hz, 1H), 2.53–2.41 (m, 6H), 2.13–2.03 (m, 2H), 1.84–1.70 (m, 6H), 1.66–1.53 (m, 1H), 1.52–1.42 (m, 2H), 1.35–1.21 (m, 7H), 1.19–1.04 (m, 2H), 0.62–0.54 (m, 2H), 0.36–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.3, 163.2, 158.0, 150.0, 148.3, 144.9, 144.2, 141.7, 133.3, 132.5, (10 × C), 123.5, 122.9, 113.2, 111.4, 108.8, 103.6 (6 × CH), 76.3, 67.6, 67.0, 59.4, 58.6, 54.3, 53.7, 53.0, 51.1 (9 × CH<sub>2</sub>), 48.7 (C), 45.7, 42.7, 35.3 (3 × CH<sub>2</sub>), 34.2 (CH), 32.6, 31.7, 29.0, 26.4 (4 × CH<sub>2</sub>), 25.1 (CH<sub>3</sub>), 23.4 (CH<sub>2</sub>), 10.8 (CH), 3.2 (CH<sub>2</sub>). HPLC: 100%, <i>t</i><sub>R</sub> 1.21 min. MS (ESI+): <i>m</i>/<i>z</i> 740.1 [M + H]<sup>+</sup>, 370.9 [M + 2H]<sup>2+</sup>.</div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 3-(4-(((2-(3-((2-Amino-5-(cyclopropylmethoxy)-3,3-dimethyl-3<i>H</i>-indol-6-yl)oxy)propyl)isoindolin-5-yl)oxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)-1-(4-(2-(pyrrolidin-1-yl)ethyl)piperidin-1-yl)propan-1-one (<b>36</b>)</h3><div class="NLM_p last">Prepared using general conditions (j) and (i) from alkyne <b>29</b> and azide <b>33</b>; (31%) while solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.76 (s, 1H), 7.08 (d, <i>J</i> = 8.2 Hz, 1H), 6.86–6.75 (m, 4H), 5.15 (s, 2H), 4.70 (t, <i>J</i> = 6.3 Hz, 2H), 4.51 (d, <i>J</i> = 13.3 Hz, 2H), 4.11 (t, <i>J</i> = 6.3 Hz, 2H), 3.91 (s, 2H), 3.88 (s, 2H), 3.81 (d, <i>J</i> = 6.9 Hz, 2H), 3.71 (d, <i>J</i> = 13.4 Hz, 2H), 2.97–2.87 (m, 5H), 2.58–2.41 (m, 7H), 2.13–2.03 (m, 2H), 1.82–1.66 (m, 6H), 1.60–1.42 (m, 3H), 1.35–1.22 (m, 7H), 1.14–0.98 (m, 2H), 0.61–0.54 (m, 2H), 0.34–0.28 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.26, 167.35, 157.60, 150.05, 148.39, 144.28, 143.84, 141.70, 133.31, 132.82 (10 × C), 124.25, 122.94, 113.49, 111.43, 108.99, 103.75 (6 × CH), 76.32, 67.57, 62.24, 59.37, 58.57, 54.24, 53.79, 52.97 (8 × CH<sub>2</sub>), 48.73 (C), 46.16, 45.58, 42.16, 35.35 (4 × CH<sub>2</sub>), 34.32 (CH), 33.56, 32.65, 31.86, 28.96 (4 × CH<sub>2</sub>), 25.07 (CH<sub>3</sub>), 23.39 (CH<sub>2</sub>), 10.82 (CH), 3.16 (CH<sub>2</sub>). HPLC: 99%, <i>t</i><sub>R</sub> 1.32 min. MS (ESI+): <i>m</i>/<i>z</i> 740.1 [M + H]<sup>+</sup>, 370.9 [M + 2H]<sup>2+</sup>.</div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 3-(4-(2-((2-(3-((2-Amino-5-(cyclopropylmethoxy)-3,3-dimethyl-3<i>H</i>-indol-6-yl)oxy)propyl)isoindolin-5-yl)oxy)ethyl)-1<i>H</i>-1,2,3-triazol-1-yl)-1-(4-(2-(pyrrolidin-1-yl)ethyl)piperidin-1-yl)propan-1-one (37)</h3><div class="NLM_p last">Prepared using general conditions (j) and (i) from alkyne <b>30</b> and azide <b>33</b>; (14%) while solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.58 (s, 1H), 7.07 (d, <i>J</i> = 8.2 Hz, 1H), 6.83 (s, 1H), 7.06–6.83 (m, 3H), 4.67 (t, <i>J</i> = 6.5 Hz, 2H), 4.52 (d, <i>J</i> = 12.9 Hz, 1H), 4.20 (t, <i>J</i> = 6.6 Hz, 2H), 4.12 (t, <i>J</i> = 6.3 Hz, 2H), 3.91 (s, 2H), 3.88 (s, 2H), 3.81 (d, <i>J</i> = 7.0 Hz, 2H), 3.72 (d, <i>J</i> = 14.0 Hz, 1H), 3.17 (t, <i>J</i> = 6.6 Hz, 2H), 2.97–2.88 (m, 5H), 2.56–2.44 (m, 7H), 2.12–2.05 (m, 2H), 1.84–1.66 (m, 6H), 1.60–1.43 (m, 3H), 1.35–1.20 (m, 7H), 1.11–0.98 (m, 2H), 0.62–0.54 (m, 2H), 0.36–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.10, 167.51, 158.00, 150.09, 147.97, 144.41, 144.28, 141.67, 133.19, 132.48 (10 × C), 123.29, 122.92, 113.28, 111.41, 108.78, 103.73 (6 × CH), 76.31, 67.59, 67.10, 59.37, 58.59, 54.22, 53.77, 52.98 (8 × CH<sub>2</sub>), 48.72 (C), 46.02, 45.58, 42.13, 35.18 (4 × CH<sub>2</sub>), 34.18 (CH), 33.58, 32.64, 31.86, 28.94, 26.17 (5 × CH<sub>2</sub>), 25.07 (CH<sub>3</sub>), 23.39 (CH<sub>2</sub>), 10.81 (CH), 3.16 (CH<sub>2</sub>). HPLC: 98%, <i>t</i><sub>R</sub> 1.21 min. MS (ESI+): <i>m</i>/<i>z</i> 754.1 [M + H]<sup>+</sup>, 377.9 [M + 2H]<sup>2+</sup>.</div></div><div id="sec4_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 2-(4-(((2-(4-((2-Amino-5-(cyclopropylmethoxy)-3,3-dimethyl-3<i>H</i>-indol-6-yl)oxy)butyl)isoindolin-5-yl)oxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)-1-(4-(2-(pyrrolidin-1-yl)ethyl)piperidin-1-yl)ethan-1-one (<b>38</b>)</h3><div class="NLM_p last">Prepared using general conditions (j) and (i) from alkyne <b>31</b> and azide <b>32</b>; (54%) white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.78 (s, 1H), 7.07 (d, <i>J</i> = 8.0 Hz, 1H), 6.83–6.76 (m, 4H), 5.26–5.16 (m, 4H), 4.51 (d, <i>J</i> = 13.4 Hz, 2H), 4.05 (t, <i>J</i> = 6.3 Hz, 2H), 3.91–3.77 (m, 7 H) 3.11 (td, <i>J</i> = 13.0, 2.8 Hz, 1H), 2.76 (t, <i>J</i> = 7.3 Hz, 2H), 2.65 (td, <i>J</i> = 13.0, 3.0 Hz, 1H), 2.56–2.43 (m, 6H), 1.96–1.86 (m, 2H), 1.86–1.70 (m, 8 H), 1.67–1.54 (m, 1H), 1.54–1.45 (m, 2H), 1.36–1.22 (m, 7 H), 1.22–1.07 (m, 2H), 0.62–0.54 (m, 2H), 0.36–0.27 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.20, 162.93, 157.53, 150.11, 147.92, 144.50, 144.33, 141.72, 133.03, 132.87 (10 × C), 124.46, 122.93, 113.46, 111.42, 109.09, 103.51 (6 × CH), 76.28, 69.09, 62.29, 59.24, 58.48, 55.83, 54.24, 53.70, 51.05 (9 × CH<sub>2</sub>), 48.72 (C), 45.66, 42.76, 35.19 (3 × CH<sub>2</sub>), 34.17 (CH), 32.62, 31.69, 27.25, 25.52 (4 × CH<sub>2</sub>), 25.07 (CH<sub>3</sub>), 23.40 (CH<sub>2</sub>), 10.82 (CH), 3.17 (CH<sub>2</sub>). HPLC: 99%, <i>t</i><sub>R</sub> 1.41 min. MS (ESI+): <i>m</i>/<i>z</i> 740.1 [M + H]<sup>+</sup>, 370.8 [M + 2H]<sup>2+</sup>.</div></div><div id="sec4_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 5-(Benzyloxy)isoindoline (<b>39</b>)</h3><div class="NLM_p last">Prepared by <i>O</i>-benzylation of <i>tert</i>-butyl 5-hydroxyisoindoline-2-carboxylate (0.590 g, 2.51 mmol, 1 equiv) using condition (e). The resultant oil was dissolved in dry DCM (20 mL), and trifluoroacetic acid (TFA, 2 mL) was added. The solution was stirred at room temperature for approximately 2 h. The solvent was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 0–10% of [7 N NH<sub>3</sub> in MeOH] in DCM (and using ninhydrin TLC stain to visualize the product) to give compound <b>39</b> (0.500 g, 2.22 mmol, 89%) as a brown solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.47–7.29 (m, 5H), 7.14 (d, <i>J</i> = 8.1 Hz, 1H), 6.89–6.82 (m, 2H), 5.06 (s, 2H), 4.20 (s, 2H), 4.18 (s, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 158.34, 143.15, 137.15, 133.77 (4 × C), 128.61, 128.58, 127.95, 127.45, 122.92, 113.82, 108.87 (6 × CH), 70.31, 53.05, 52.29 (3 × CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.28 min. MS (ESI+): <i>m</i>/<i>z</i> 226.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 6-(3-(5-(Benzyloxy)isoindolin-2-yl)propoxy)-5-(cyclopropylmethoxy)-3,3-dimethyl-3<i>H</i>-indol-2-amine (<b>40</b>)</h3><div class="NLM_p last">Prepared using general conditions (f) and (i) from alkyl chloride <b>19</b> and isoindoline <b>39</b>. (63%) while solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.44–7.28 (m, 5H), 7.10 (d, <i>J</i> = 8.0 Hz, 1H), 6.86–6.79 (m, 4H), 5.04 (s, 2H), 4.12 (t, <i>J</i> = 6.2 Hz, 2H), 3.97 (s, 2H), 3.95 (s, 2H), 3.81 (d, <i>J</i> = 6.9 Hz, 2H), 2.97 (t, <i>J</i> = 7.4 Hz, 2H), 2.17–2.08 (m, 2H), 1.45 (s, 6H), 1.32–1.22 (m, 1H), 0.63–0.57 (m, 2H), 0.36–0.31 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 178.80, 158.28, 149.68, 148.70, 143.87, 141.78, 136.70, 134.09, 131.96 (9 × C), 128.58, 127.92, 127.44, 123.00, 113.76, 110.99, 109.07, 104.32 (8 × CH), 75.96, 70.27, 67.63, 59.34, 58.59, 52.90, 48.37 (C), 28.58 (CH<sub>2</sub>), 25.05 (CH<sub>3</sub>), 10.66 (CH), 3.21 (CH<sub>2</sub>). HPLC: <i>t</i><sub>R</sub> 1.50 min. MS (ESI+): <i>m</i>/<i>z</i> 512.4 [M + H]<sup>+</sup>.</div></div><div id="sec4_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Conditions (k); Synthesis of 6-(3-(5-(4-Aminobutoxy)isoindolin-2-yl)propoxy)-5-(cyclopropylmethoxy)-3,3-dimethyl-3<i>H</i>-indol-2-amine (<b>41</b>)</h3><div class="NLM_p last">Palladium (10% on carbon (50% wet), 0.2 weight equiv) was added to a solution of benzyl aryl ether <b>40</b> (0.128 g, 0.25 mmol, 1 equiv) in MeOH (50 mL). The system was flushed with N<sub>2</sub>, followed by H<sub>2</sub>, and the mixture was stirred at room temperature under H<sub>2</sub> for 18 h. The catalyst was removed by filtration through a bed of Celite, and the solvent was evaporated to give the debenzylated product, which was used without further purification. Potassium carbonate (69 mg, 0.50 mmol, 2 equiv) and 4-(boc-amino)butyl bromide (71 mg, 0.28 mmol, 1.1 equiv) was added to the debenzylated product, and the mixture was stirred in DMF (5 mL) at room temperature, for 48 h. The reaction mixture was evaporated. and the residue was directly loaded onto a Biotage SNAP KP-NH column. The product was eluted with an eluent of 0–10% MeOH in DCM. The resultant white solid was dissolved in dry DCM (20 mL), and trifluoroacetic acid (TFA, 2 mL) was added. The solution was stirred at room temperature for approximately 2 h. The solvent was evaporated, and the residue was purified by automated flash chromatography using a gradient elution of 0–10% of [7 N NH<sub>3</sub> in MeOH] in DCM (and using ninhydrin TLC stain to visualize the product) to give compound <b>41</b> (12 mg, 0.02 mmol, 12%) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.07 (d, <i>J</i> = 8.2 Hz, 1H), 6.83 (s, 1H), 6.80–6.69 (m, 3H), 4.12 (t, <i>J</i> = 6.3 Hz, 2H), 3.98–3.84 (m, 6H), 3.81 (d, <i>J</i> = 6.9 Hz, 2H), 2.92 (t, <i>J</i> = 7.3 Hz, 3H), 2.77 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.20, 158.34, 150.09, 147.83, 144.34, 141.59, 133.05, 132.13 (8 × C), 122.88, 113.22, 111.41, 108.65, 103.56 (5 × CH), 76.33, 68.00, 67.55, 59.37, 58.61, 52.96 (6 × CH<sub>2</sub>), 48.70 (C), 41.88, 30.20, 28.92, 26.70 (4 × CH<sub>2</sub>), 25.08 (CH<sub>3</sub>), 10.81 (CH), 3.18 (CH2). HPLC: <i>t</i><sub>R</sub> 1.15 min. MS (ESI+): <i>m</i>/<i>z</i> 493.6 [M + H]<sup>+</sup>.</div></div><div id="sec4_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Construct Design, Cloning, Expression, and Purification of Human SPINDLIN Proteins</h3><div class="NLM_p last">A plasmid encoding full length human SPIN1 was obtained from Source Bioscience (IOH9972-pDEST26) and used as a template to clone SPIN1<sub>M26–S262</sub> and SPIN1<sub>P49–S262</sub> into the pNIC-CTHF vector with a TEV (tobacco etch virus) cleavable C-terminal His<sub>6</sub> tag. SPIN1<sub>G21-S262</sub> was cloned into the pNIC-Bio2 vector with a TEV cleavable N-terminal His<sub>10</sub> tag and a C-terminal biotinylation sequence. SPIN2B<sub>P45–S258</sub>, SPIN3<sub>M27-S258</sub>, and SPIN4<sub>T36-P249</sub> were cloned into the pNIC vector with a TEV cleavable N-terminal His<sub>6</sub> tag using templates obtained from the Mammalian Gene Collection and Source Bioscience (SPIN2B: cDNA clone IMAGE id: 6729986; SPIN3 IMAGENE: IRCBp5005F0211Q and SPIN4 IMAGE id 40032302). For crystallization studies, SPIN2B<sub>P45–S258</sub> and SPIN3<sub>P45–S258</sub> were also cloned into the pNIC-CTHF vector. The recombinant proteins were expressed in a phage-resistant derivative of <i>Escherichia coli</i> strain BL21(DE3) carrying the pRARE2 plasmid for rare codon expression. Cells were grown at 37 °C in Terrific Broth supplemented with 50 μg/mL kanamycin and 34 μg/mL chloramphenicol, until the culture reached an OD<sub>600</sub> of 2.0. The temperature was decreased to 18 °C and protein expression induced with 0.1 mM IPTG (isopropyl β-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside) overnight. Cells were collected by centrifugation and frozen at −80 °C. Biotinylated SPIN1<sub>G21-S262</sub> was expressed similarly in a BL21(DE3)-R3-pRARE2-BirA strain as described in ref <a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a>.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> For purification, cells were resuspended in 50 mM Hepes pH 7.5, 500 mM NaCl, 10 mM imidazole, 5% glycerol, 0.5 mM TCEP, and a protease inhibitor cocktail (Sigma), and lysed by sonication. The cell lysate was clarified by centrifugation and the proteins purified by nickel-affinity chromatography (GE Healthcare) using a stepwise gradient of imidazole. The histidine tag was removed by incubating SPIN1<sub>P49–S262</sub>, SPIN1<sub>G21-S262</sub>, SPIN2B<sub>P45–S258</sub>, SPIN3<sub>P45–S258</sub>, and SPIN4<sub>T36-P249</sub> with TEV protease at 4 °C overnight, and this was followed by size exclusion chromatography (Superdex 75 or Superdex 200, GE Healthcare) in 10 mM Hepes pH 7.5, 500 mM NaCl, 5% glycerol, and 0.5 mM TCEP. The TEV protease and the uncleaved proteins were removed by nickel-affinity chromatography and concentrated using an Amicon centrifugal filtration unit. Mass of purified proteins was verified by electrospray ionization time-of-flight mass spectrometry (ESI-TOF-TOF: Agilent LC/MSD).</div></div><div id="sec4_54_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Crystallization and X-ray Data Collection</h3><div class="NLM_p last">Proteins used for crystallization studies had the histidine tag removed, and all proteins were crystallized at 4 °C using the sitting drop vapor diffusion method. SPIN1<sub>Pro49-Ser262</sub> was incubated with 4.5 mM A-366 at 31 mg/mL, and crystals were obtained in a drop containing 100 nL of protein compound mixture, 30 nL of precipitant consisting of 0.1 M Bis-Tris pH 6.5, 0.2 M CaCl<sub>2</sub>, 45% MPD, and 20 nL of seeds obtained from crystals grown in 0.1 M BisTris pH 5.5, 0.1 M MgCl<sub>2</sub>, and 16% PEG3350. The crystal was cryoprotected in precipitant solution supplemented with 25% ethylene glycol before it was flash cooled in liquid nitrogen. SPIN1<sub>Pro49-Ser262</sub> was incubated with 1.5 mM of either compound <b>23</b> or compound <b>35</b> using 39 mg/mL protein. Crystals appeared in drops consisting of 75 nL of protein compound mixture and 75 nL of precipitant consisting of 55–60% MPD and 0.1 M SPG (succinate, phosphate, and glycine) buffer pH 5.5–6.0. SPIN2B<sub>P45–S258</sub> was crystallized using 23 mg/mL protein preincubated with 3-fold molar excess of a ARTK(me3)QTAR(me2a)KS peptide. Crystals appeared in drops consisting of 100 nL of SPIN2B/peptide and 50 nL of precipitant composed of 0.1 M Bis-Tris buffer pH 5.5 and 25% PEG3350. SPIN3<sub>P45–S258</sub> was crystallized in a drop consisting of 75 nL of 24 mg/mL protein, 55 nL of precipitant composed of of 0.1 M Tris buffer pH 8.4 and 38% PEG3350, and 20 nL of seeds prepared from crystals grown in the same condition. SPIN4<sub>T36-P249</sub> was crystallized using 29 mg/mL protein preincubated with 1.7 mM ARTK(Me3)QTARKS peptide. Crystals were obtained in drops consisting of 50 nL of protein/peptide, 80 nL of precipitant compost of 30% PEG3350 and 0.1 M HEPES pH 7.0 and 20 nL of seeds prepared from crystals grown in the same condition. Crystals of SPIN2B, SPIN3, and SPIN4 were cryoprotected in precipitant solution supplemented with 25% ethylene glycol before they were flash cooled in liquid nitrogen. All data were collected on beamlines I02, I03, and I24 at the Diamond Light source UK.</div></div><div id="sec4_54_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Structure Determination</h3><div class="NLM_p last">All data sets were processed, scaled, and merged at the Diamond light source using Xia2, autoPROC,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> XDS,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> POINTLESS,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> AIMLESS,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> and CCP4.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Electron density maps were obtained by molecular replacement using PHASER with previously determined structures of SPIN1 as a search model (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4H75">4H75</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MZG">4MZG</a>). The complex structure of SPIN1 with A-366 was solved to 1.52 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Y">6I8Y</a>), with compound <b>23</b> to 1.58 Å (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L">6I8L</a>) and with compound <b>35</b> to 1.76 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8B">6I8B</a>). SPIN2B was solved to 1.7 Å (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LUG">5LUG</a>), SPIN3 to 2.0 Å (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A1H">5A1H</a>), and SPIN4 to 1.86 Å resolution. All structures were refined in an iterative process combining REFMAC and PHENIX<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> with electron density map inspections and model improvement in COOT<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> and terminated when there were no significant changes in the <i>R</i><sub>work</sub> and <i>R</i><sub>free</sub> values, and inspection of the electron density map suggested that no further corrections or additions were justified. Structural analysis and figures were performed with PyMOL (<a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>).</div></div><div id="sec4_54_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> AlphaScreen Assay</h3><div class="NLM_p last">Compound binding was determined by the displacement of a trimethylated biotinylated histone H3 peptide from a Hexa His tagged SPIN1 Tudor domain protein using an AlphaScreen histidine (Nickel Chelate) detection kit. Compound was dispensed from DMSO stocks into assay plates (ProxiPlate-384 Plus, PerkinElmer) using an Echo 525 liquid handler (Labcyte). Assays were performed at 25 °C in a volume of 20 μL in a buffer that was 25 mM HEPES pH 7.5, 100 mM NaCl, 0.1% BSA, and 0.05% CHAPS with 12.5 nM peptide (ARTK(me3)QTARKSTGGKAPRK-biotin), 12.5 nM SPIN1, and 0.2% DMSO. Peptide and protein (SPIN1<sub>M26–S262</sub> C-terminal His<sub>6</sub> tag) were premixed, added to the compound plate and incubated for 30 min after which AlphaScreen beads were added to a final concentration of 2.5 μg/mL. Following a primary assay, a counterscreen with a biotinylated hexa-histidyl peptide (biotin-HHHHHH) instead of the SPIN1–histone peptide pair was performed to eliminate compounds that interfere with the assay chemistry using identical assay conditions. Assay plates were incubated for 60 min then luminescence was measured using a Pherastar FS/FSX (BMG).</div></div><div id="sec4_54_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Bio-Layer Interferometry (BLI) Assay</h3><div class="NLM_p last">The direct interaction between small molecules and SPIN1 was measured using a BLI Octet RED384 (FortéBio Inc., Menlo Park, CA). Superstreptavidin biosensors (FortéBio Inc.) were coated in a solution containing 50 μg/mL of biotinylated protein (SPIN1<sub>G21-S262</sub> C-terminal biotinylation) for 5 min at 25 °C. Compound was dispensed from DMSO stocks into plates using an Echo 525 liquid handler (Labcyte). Assays were performed at 25 °C in a volume of 60 μL in a buffer that was 25 mM HEPES pH 7.5, 150 mM NaCl, and 0.05% TWEEN 20. A duplicate set of sensors without protein was used as a background binding control. Association of samples (5000, 2500, 1250, 625, 312.5, 156.25, and 78.125 nM) to coated and uncoated reference sensors was measured over 120 s and dissociation over 240 s. Data analysis on the FortéBio Octet RED96 instrument was performed using a double reference subtraction (sample and sensor references) in the FortéBio data analysis software, which accounts for nonspecific binding, background, and signal drift and minimizes well-based and sensor variability, while <i>K</i><sub>d</sub> values were fitted using a 1:1 model.</div></div><div id="sec4_54_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Thermal Shift Assay (TSA)/Differential Scanning Fluorimetry (DSF)</h3><div class="NLM_p last">TSA/DSF experiments for UHRF1, 53BP1, TDRD3, SND1, SETDB1, and SGF29 were carried out as reported by Neisen et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> All other TSA/DSF experiments were performed on a Lightcycler 480 (Roche Molecular Systems). Compounds were dispensed into 384-well PCR plates (20 μL assay volume) to a final concentration of 20 μM. Purified proteins were diluted to 2 μM in buffer (20 mM HEPES, 500 mM, pH 7.5). SYPRO orange dye was added to a 1:1000 dilution from purchased stock (LifeTechnologies). Proteins were melted over a gradient from 20 to 95 °C with three acquisitions per cycle (0.19 °C/s). The shift in unfolding (D<i>T</i><sub>m</sub>) was calculated as the difference between the mean melting temperature (<i>T</i><sub>m</sub>) for DMSO reference wells (<i>n</i> = 16 per protein) and <i>T</i><sub>m</sub> for the compound wells (duplicates for each compound). <i>T</i><sub>m</sub> was estimated by fitting a sigmoidal Boltzmann equation to the fluorescence intensity <i>I</i>(<i>T</i>) over the unfolding process:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_m001.gif" alt="" id="_i73" /></img></span>where <i>I</i><sub>onset</sub> is the intensity before the onset of unfolding, <i>I</i><sub>end</sub> is the intensity at the end point of unfolding, and <i>S</i> is the slope of the curve.</div></div><div id="sec4_54_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">Purified protein was diluted (see SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Table S5</a>) in ITC buffer (100 mM NaCl, 20 mM Tris, pH 8.0) and then dialyzed against a 1000 times excess of ITC buffer at 4 °C overnight using SnakeSkin dialysis tubing (Thermo Scientific) with a 3.5 kDa molecular weight cut off. The dialyzed sample was then centrifuged at 14000 rpm in a standard chilled benchtop centrifuge at 4 °C to remove aggregate. Protein concentration (see SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Table S5</a>) was then verified via absorption at 280 nm using a NanoDrop ND1000 spectrophotometer. Compounds were diluted in ITC buffer (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">Table S5</a>). ITC was performed on a NanoITC standard volume instrument (TA Instruments) using direct titration (protein in cell, compound in syringe) at 20 °C with an initial injection of 3.7 μL followed by 30 injections of 7.96 μL at a stir rate of 350 rpm. Data were analyzed using an independent fit model with the NanoAnalyse software (TA Instruments, version 3.4.0).</div></div><div id="sec4_54_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Development of SPIN1 Nanobret Assay</h3><div class="NLM_p last">U20S cell (2.8 × 105) were plated in each well of a 6-well plate. After 6 h, cells were cotransfected with C-terminal HaloTag-Histone 3.3 (NM_002107) and an N-terminal NanoLuciferase fusion of full length SPIN1 at a 1:500 (NanoLuc to HaloTag) ratio respectively with FuGENE HD transfection regent. Sixteen hours post-transfection, cells were collected, washed with PBS, and exchanged into media containing phenol red-free DMEM and 4% FBS in the absence (control sample) or the presence (experimental sample) of 100 nM NanoBRET 618 fluorescent ligand (Promega). Cells were then replated in a 384-well assay white plate (Greiner no. 3570) at 2.7 × 103 cells per well. Compounds were then added directly to media at final concentrations of 0–30 μM or an equivalent amount of DMSO as a vehicle control, and the plates were incubated for 24 h at 37 °C in the presence of 5% CO<sub>2</sub>. NanoBRET Nano-Glo substrate (Promega) was added to both control and experimental samples at a final concentration of 10 μM. Readings were performed within 10 min using a ClarioSTAR (BMG labtech). A corrected BRET ratio was calculated and is defined as the ratio of the emission at 610 nm/460 nm for experimental samples minus the emission at 610 nm/460 nm for control samples (without NanoBRET fluorescent ligand). BRET ratios are expressed as milliBRET units (mBU), where 1 mBU corresponds to the corrected BRET ratio multiplied by 1000. The assay was further validated by domain specific site directed mutagenesis (Y170A) ablating peptide and ligand binding.</div></div><div id="sec4_54_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Cell Line Screening-Proliferation Assays, Cell Cycle, and Apoptosis</h3><div class="NLM_p last">SCC040 cells were seeded into 96-well (proliferation) or 6 well plates (cell cycle and apoptosis) at a density of 500 and 15000 cells/well, respectively, in DMEM medium (supplemented with 10% FCS, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, 100 units/mL penicillin, 100 μg/mL streptomycin, and 1× NEAA) in the presence of 1, 2, and 4 μM compound <b>35</b>, <b>38</b>, or DMSO control. To assess proliferation, cells were measured for confluence using brightfield imaging on the Celigo (Nexcelome) and direct cell counting of cell nuclei within each well 1 h by incubation of Hoechst 33342 at 10 μg/mL at day 4 and day 6 after seeding. To assess cell cycle parameters, cells were trypsinized, pelleted, and resuspended in 0.5 mL of SSC040 media with 10 μg/mL Hoechst 33342 for 45 min at 37 °C, 5% CO<sub>2</sub>. Cells were then analyzed on a BD LSRFortessa and cell cycle measured using FlowJoV10. To assess apoptosis cells the Dead Cell Apoptosis Kit with Annexin V Alexa Fluor 488 and PI (Invitrogen) kit was used. In brief, cells were trypsinized, washed in PBS, pelleted, and resuspended in 0.1 mL 1× annexin binding buffer. Then 1 μL of 100 μL/mL propidium iodide (PI) and 5 μL of Alexa Fluor 488 annexin V was added per sample and incubated at room temperature for 15 min in the dark. Then 0.4 mL of annexin binding buffer was then added to each sample, and cells were then analyzed on a BD LSRFortessa and apoptosis measured using FlowJoV10.</div></div><div id="sec4_54_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Chemoproteomic: Preparation of Affinity Resin</h3><div class="NLM_p last">Amine functionalized compound <b>41</b> was coupled to NHS-activated Sepharose 4 fast flow beads (GE Healthcare).<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Then 100 μL of bead slurry (50% in 2-propanol) were used for each pull-down experiment. Beads were washed with DMSO (500 μL), collected by centrifugation (3 min, 60<i>g</i>), and the supernatant removed. After three wash cycles, the beads were resuspended in DMSO (50 μL), to which the amine (0.025 μmol) and triethylamine (0.75 μL) were added. The beads were incubated at room temperature for 16 h and depletion of free amine from the supernatant determined by LC-MS analysis. Ethanolamine (2.5 μL) was then added to block any unreacted NHS sites and the beads incubated for a further 16 h. Derivatized beads were then washed with DMSO (3 × 500 μL) and buffer A (50 mM Tris pH 7.5, 0.8% v/v NP-40, 5% v/v glycerol, 1.5 mM MgCl<sub>2</sub>, 100 mM NaCl, 25 mM NaF, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 1 mM PMSF, 1 mM DTT, 10 μg/mL TLCK, 1 μg/mL leupeptin, 1 μg/mL aprotinin, and 1 μg/mL soy bean trypsin) (3 × 1 mL). Resin was incubated with cell lysates (2 mg of protein per pulldown at 6 mg/mL) that had been pretreated with either compound <b>35</b>, <b>38</b> (20 μM), or DMSO control for 30 min at 4 °C. Beads and treated lysates were incubated for 2 h at 4 °C before being washed with buffer A (5 mL) proteins were eluted by addition of 80 μL of 2× sample buffer (65.8 mM Tris-HCl pH 6.8, 26.3% (w/v) glycerol, 2.1% SDS, 0.01% bromophenol blue, 50 mM DTT). Before MS analysis, SPIN1 pull-down was verified by Western Blotting using primary antibody ab118784 (Abcam UK, rabbit) at 1:1000.</div></div><div id="sec4_54_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> MS Sample Preparation</h3><div class="NLM_p last">Samples were reduced with DTT (10 mM final concentration) for 30 min at room temperature, alkylated with iodoacetamide (55 mM final concentration) for 30 min at room temperature, diluted with TEAB, and incubated with trypsin (6 μL, 0.2 mg/mL) overnight at 37 °C. The digests were then desalted using SEPAC lite columns (Waters), eluted with 69% v/v MeCN, 0.1% v/v FA in H<sub>2</sub>O (1 mL),and dried in vacuo. Dried peptides were stored at −20 °C before resuspension in 2% V/V MeCN, 0.1% v/v FA in H<sub>2</sub>O (20 μL) for LC-MS/MS analysis. Mass spectrometry data was acquired at the Discovery Proteomics Facility (University of Oxford). Digested samples were analyzed by nano-UPLC–MS/MS using a Dionex Ultimate 3000 nano UPLC with EASY spray column (75 μm × 500 mm, 2 μm particle size, Thermo Scientific) with a 60 min gradient of 0.1% (v/v) formic acid in 5% (v/v) DMSO to 0.1% (v/v) formic acid with 35% (v/v) acetonitrile in 5% (v/v) DMSO at a flow rate of approximately 250 nL/min (600 bar per 40 °C column temperature). Mass spectrometry data was acquired with an Orbitrap Q exactive high field (HF) instrument (ThermoFisher). Survey scans were acquired at a resolution of 60000 at 400 <i>m</i>/<i>z</i>,and the 20 most abundant precursors were selected for CID fragmentation.</div></div><div id="sec4_54_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Data Analysis</h3><div class="NLM_p">Raw data was processed using MaxQuant version 1.5.0.253 and the reference complete human proteome FASTA file (UniProt). Label Free Quantification (LFQ) and Match Between Runs were selected; replicates were collated into parameter groups to ensure matching between replicates only. Cysteine carbamidomethylation was selected as a fixed modification and methionine oxidation as a variable modification. Default settings for identification and quantification were used. Specifically, a minimum peptide length of 7, a maximum of two missed cleavage sites, and a maximum of three labeled amino acids per peptide were employed. Peptides and proteins were identified utilizing a 0.01 false discovery rate, with “unique and razor peptides” mode selected for both identification and quantification of proteins (razor peptides are uniquely assigned to protein groups and not to individual proteins). At least two razor + unique peptides were required for valid quantification. Processed data was further analyzed using Perseus version 1.5.0.9 and Microsoft Excel 2010. Peptides categorized by MaxQuant as “potential contaminants”, “only identified by site” or “reverse” were filtered, and the LFQ intensities transformed by log2. Experimental replicates were grouped, and two valid LFQ values were required in at least one experimental group. Missing values were imputed using default settings, and the data distribution visually inspected to ensure that a normal distribution was maintained. Statistically significant competition was determined through the application of P2 tests, using a permutation-based FDR of 0.05 and an S0 of 2, and data were visualized in volcano plots.</div><div class="NLM_p last">The accompanying SI Data Excel file contains a summary of the chemoproteomic data.</div></div><div id="sec4_54_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> Transcriptional Responses to SPIN1 Inhibitor and siRNA Knockdown in Squamous Cell Carcinoma Cell Lines</h3><div class="NLM_p last">SSC040 cells were seeded into wells of a 48 well plate at a density of 10000 cells/well in DMEM medium (supplemented with 10% FCS, 2 mM <span class="smallcaps smallerCapital">l</span>-Glutamine, 100 units/mL penicillin, 100 μg/mL streptomycin, 1× NEAA). Compounds (<b>35</b> and <b>38</b> or DMSO control) or siRNA were added 24 h later. RNA was harvested using Trizol reagent 2 and 4 days after addition of siRNA or 5 and 7 days after addition of compounds. RNA was purified using a Direct-zol RNA miniprep kit (Zymo Research). Experiments were conducted in two biological replicates, with <i>n</i> = 3 technical replicates per condition.</div></div><div id="sec4_54_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> RNaseq, Library Preparation, and Sequencing</h3><div class="NLM_p last">RNA concentration derived from inhibitor experiments (<i>n</i> = 3, technical replicates) was normalized to 100 ng/50 μL, and Illumina libraries were prepared using a NEBNext Poly(A) mRNA magnetic isolation module in conjunction with a NEBNext Ultra II directional RNA library prep kit (New England Biolabs), according to the manufacturer’s instructions. Libraries were quantitated using the Agilent high sensitivity D1000 screen tape system on a 2200 Tapestation (Agilent), pooled at equimolar concentration, denatured, and sequenced on a NextSeq 500 platform (Illumina).</div></div><div id="sec4_54_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> NGS Data Processing and Analysis</h3><div class="NLM_p last">Sequencing reads from the RNaseq experiment were aligned to the human genome (hg38) using HISAT<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> with default parameters. The program featureCounts<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> was used to assign mapped reads to genes with annotation (gtf file Ensembl87). Differential gene expression analysis was performed using DESeq2<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> with an FDR-adjusted <i>p</i> value <0.05 and log2 fold change >1. Functional analysis of differentially expressed genes was performed using Reactome.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></div></div><div id="sec4_54_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> Statistical Methods and Data Analysis</h3><div class="NLM_p last">All statistical analyses and data fitting were performed using GraphPad Prism software version 7. Data were analyzed using two-sided <i>t</i> tests or One-Way ANOVA, unless otherwise stated, and as indicated in the figure legends with a <i>p</i>-value <0.05. Statistical analysis of the sequencing data, data collection,and structure refinement was automatically performed in all of the programs used as noted above. The number of sampled units (<i>n</i>) as well as the number of technical replicates is indicated in the figure legends. The <a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv" class="ext-link">Supporting Information contains the RNaseq data</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00562" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00562" class="ext-link">10.1021/acs.jmedchem.9b00562</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">X-ray crystallographic data; analysis of ligand binding and sequence alignment of SPIN family proteins; biophysical and biochemical assays results; cellular assay results; browser track for selected genomic regions; results of in vitro stability, permeability and solubility assays; NMR spectra (with assignments) and HPLC chromatograms of end compounds; chemical structures; chemoproteomic data summary;  (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings, RNAseq results and reactome analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf">jm9b00562_si_001.pdf (12.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv">jm9b00562_si_002.csv (1.36 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank: A-366 in complex with SPIN1 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Y">6I8Y</a>; compound <b>23</b> in complex with SPIN1 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L">6I8L</a>; compound <b>35</b> in complex with SPIN1 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8B">6I8B</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00562" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56880" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56880" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul E. Brennan</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Alzheimer’s
Research UK Oxford Drug Discovery Institute, Nuffield Department of
Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8950-7646" title="Orcid link">http://orcid.org/0000-0002-8950-7646</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#6e1e0f1b02400c1c0b00000f002e1d090d400116400f0d401b05"><span class="__cf_email__" data-cfemail="522233273e7c3020373c3c333c122135317c3d2a7c33317c2739">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Udo Oppermann</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">FRIAS—Freiburg
Institute of Advanced Studies, University
of Freiburg, 79104 Freiburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9984-5342" title="Orcid link">http://orcid.org/0000-0001-9984-5342</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2c59484302435c5c495e414d42426c5f4b4f024354024d4f025947"><span class="__cf_email__" data-cfemail="4431202b6a2b3434213629252a2a043723276a2b3c6a25276a312f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vincent Fagan</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Catrine Johansson</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carina Gileadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Octovia Monteiro</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James E. Dunford</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Reshma Nibhani</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Philpott</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jessica Malzahn</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Graham Wells</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruth Faram</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adam P. Cribbs</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nadia Halidi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fengling Li</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Irene Chau</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Holger Greschik</span> - <span class="hlFld-Affiliation affiliation">Department
of Urology, Center for Clinical Research, Medical Center, Signalling
Research Centres BIOSS and CIBSS, University
of Freiburg, D-79106 Freiburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Srikannathasan Velupillai</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdellah Allali-Hassani</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James Bennett</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Christott</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charline Giroud</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew M. Lewis</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6447-354X" title="Orcid link">http://orcid.org/0000-0002-6447-354X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kilian V. M. Huber</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1103-5300" title="Orcid link">http://orcid.org/0000-0002-1103-5300</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nick Athanasou</span> - <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chas Bountra</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manfred Jung</span> - <span class="hlFld-Affiliation affiliation">FRIAS—Freiburg
Institute of Advanced Studies, University
of Freiburg, 79104 Freiburg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute of Pharmaceutical Sciences, University of Freiburg, Albertstraße 25, 79104 Freiburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6361-7716" title="Orcid link">http://orcid.org/0000-0002-6361-7716</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roland Schüle</span> - <span class="hlFld-Affiliation affiliation">Department
of Urology, Center for Clinical Research, Medical Center, Signalling
Research Centres BIOSS and CIBSS, University
of Freiburg, D-79106 Freiburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Xiong</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Jin</span> - <span class="hlFld-Affiliation affiliation">Mount
Sinai Center for Therapeutics Discovery, Departments of Pharmacological
Sciences and Oncological Sciences, Tisch
Cancer Institute, Icahn School of Medicine at Mount Sinai, New York, New York 10029, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2387-3862" title="Orcid link">http://orcid.org/0000-0002-2387-3862</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oleg Fedorov</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Target
Discovery Institute, Nuffield Department of Medicine, University of Oxford, OX3 7FZ Oxford, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gillian Farnie</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Nuffield Department of Medicine, University of Oxford, OX3 7DQ Oxford, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Centre, Nuffield Department of Orthopaedics, Rheumatology
and Musculoskeletal Sciences, NIHR Bio-medical Research Centre, University of Oxford, Oxford OX3 7LD, U.K.</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>V.F. and C.J. contributed equally. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i87">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67577" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67577" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The work was supported by Cancer Research UK (C41580/A23900), the LEAN (Leducq Epigenetics Atherosclerosis Network) by the Leducq Foundation, Oxford NIHR Biomedical Research Centre, Myeloma UK, Arthritis Research UK (20522), and Alzheimer’s Research UK (ARUK-2018DDI-OX). The SGC (charity no. 1097737) receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA Darmstadt Germany, MSD, Novartis Pharma AG, Ontario Ministry of Economic Development and Innovation, Pfizer, Saõ Paulo Research Foundation-FAPESP, Takeda, and Wellcome [106169/ZZ14/Z]. Funding was received from People Programme (Marie Curie Actions) of the European Union’s Seventh Framework Programme (FP7/2007-2013) under REA grant agreement no. [609305]. R.S. and M.J. thank the Deutsche Forschungsgemeinschaft (DFG, CRC992) for funding. This work was further supported by grants from the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) [SFB 850,992,138, Project-ID 403222702, Schu688/15-1] as well as the Signalling Research Centres BIOSS and CIBSS [CIBSS - EXC-2189, Project ID 390939984]. The views expressed are the author(s) and not necessarily those of the NHS, NIHR, or Department of Health.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">SPIN1</td><td class="NLM_def"><p class="first last">spindlin1</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">SPA</td><td class="NLM_def"><p class="first last">scintillation proximity assay</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">CPP</td><td class="NLM_def"><p class="first last">chemical probe pairing</p></td></tr><tr><td class="NLM_term">HNSCC</td><td class="NLM_def"><p class="first last">head and neck squamous cell carcinoma</p></td></tr><tr><td class="NLM_term">SCC</td><td class="NLM_def"><p class="first last">squamous cell carcinoma</p></td></tr><tr><td class="NLM_term">CRI</td><td class="NLM_def"><p class="first last">cancer-related inflammation</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64167" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64167" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 76 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">Translating the histone code</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span> (<span class="NLM_issue">5532</span>),  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&issue=5532&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lje3ErjLv8wwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26issue%3D5532%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Reading the combinatorial histone language</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=564-574&issue=3&author=Z.+Suauthor=J.+M.+Denu&title=Reading+the+combinatorial+histone+language&doi=10.1021%2Facschembio.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Reading the Combinatorial Histone Language</span></div><div class="casAuthors">Su, Zhangli; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">564-574</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histones are subject to frequent combinatorial post-translational modifications (PTMs), forming a complex chem. "language" that is interpreted by PTM-specific histone-interacting protein modules (reader domains).  These specific interactions are thought to instruct gene expression and downstream biol. functions.  While the majority of studies have focused on individual modifications, the authors' current understanding of the combinatorial PTM patterns on histones is starting to emerge, benefiting from the convergence of multiple technologies.  Here, the authors review the key tech. advances and progress on discovery and characterization of combinatorial histone PTM patterns.  The authors focus on the interactions between reader domains and combinatorial PTMs, which is essential for understanding the mechanism and biol. meaning of establishing and interpreting information embedded in histone PTM patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHpmgsQnFL7rVg90H21EOLACvtfcHk0lh94JLMl5zQag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK&md5=819be9f59c1c1f0ae1ee09665b2d51da</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DReading%2520the%2520combinatorial%2520histone%2520language%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D3%26spage%3D564%26epage%3D574%26doi%3D10.1021%2Facschembio.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teske, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadden, M. K.</span></span> <span> </span><span class="NLM_article-title">Methyllysine binding domains: structural insight and small molecule probe development</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">14</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.ejmech.2017.04.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28478342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlygsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2017&pages=14-35&author=K.+A.+Teskeauthor=M.+K.+Hadden&title=Methyllysine+binding+domains%3A+structural+insight+and+small+molecule+probe+development&doi=10.1016%2Fj.ejmech.2017.04.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Methyllysine binding domains: Structural insight and small molecule probe development</span></div><div class="casAuthors">Teske, Kelly A.; Hadden, M. Kyle</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14-35</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  A frequent posttranslational modification that regulates gene expression is the mono-, di-, and/or tri- methylation of lysine residues on the histone tails of chromatin.  The recognition of methylated lysine marks is facilitated by specific reader proteins that contain a methyllysine binding domain.  This class of reader proteins has emerged as a focus of epigenetic research due to its crucial role in gene regulation, oncogenesis and other disease pathways.  The design and synthesis of small mols. that target these domains and disrupt reader/histone protein-protein interactions have demonstrated the druggability of methyllysine binding pockets and provided preliminary evidence that their disruption holds therapeutic potential.  In this review, the authors detail the structures of methyllysine binding domains, highlight the primary roles of these reader proteins in both normal and disease states, and describe the current status of small mol. development against these emerging epigenetic regulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8T6OS1rcnRLVg90H21EOLACvtfcHk0lj2en-dex5JFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlygsLk%253D&md5=8017ecb111b7b8b05f87b08a90853bec</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.047%26sid%3Dliteratum%253Aachs%26aulast%3DTeske%26aufirst%3DK.%2BA.%26aulast%3DHadden%26aufirst%3DM.%2BK.%26atitle%3DMethyllysine%2520binding%2520domains%253A%2520structural%2520insight%2520and%2520small%2520molecule%2520probe%2520development%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D136%26spage%3D14%26epage%3D35%26doi%3D10.1016%2Fj.ejmech.2017.04.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, B. B.</span></span> <span> </span><span class="NLM_article-title">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>124</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">503</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=9053325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=1997&pages=493-503&issue=2&author=B.+Ohauthor=S.+Y.+Hwangauthor=D.+Solterauthor=B.+B.+Knowles&title=Spindlin%2C+a+major+maternal+transcript+expressed+in+the+mouse+during+the+transition+from+oocyte+to+embryo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span></div><div class="casAuthors">Oh, Bermseok; Hwang, Sue-Yun; Solter, Davor; Knowles, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">493-503</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists</span>)
        </div><div class="casAbstract">Timely translation of maternal transcriptions and post-translational modification of their gene products control the initial development of preimplantation-stage embryos.  We have isolated and characterized a gene encoding a stage-specific embryonic protein.  This novel gene, spindlin (Spin), is an abundant maternal transcript present in the unfertilized egg and 2-cell, but not 8-cell, stage embryo.  Spin exhibits high homol. to a multicopy gene, Y-linked spermiogenesis-specific transcript (Ssty), and together they form a new gene family expressed during gametogenesis.  We find that spindlin assocs. with the meiotic spindle and is modified by phosphorylation in a cell-cycle-dependent fashion.  Furthermore, it comigrates with the previously described 30 × 103 Mr metaphase complex which is posttranslationally modified during the first mitotic cell cycle.  Our data suggest that spindlin plays a role in cell-cycle regulation during the transition from gamete to embryo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBqamj85CalbVg90H21EOLACvtfcHk0lj2en-dex5JFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D&md5=12c37613f401dedd7d68597ef3e63b0b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DB.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DSolter%26aufirst%3DD.%26aulast%3DKnowles%26aufirst%3DB.%2BB.%26atitle%3DSpindlin%252C%2520a%2520major%2520maternal%2520transcript%2520expressed%2520in%2520the%2520mouse%2520during%2520the%2520transition%2520from%2520oocyte%2520to%2520embryo%26jtitle%3DDevelopment%26date%3D1997%26volume%3D124%26issue%3D2%26spage%3D493%26epage%3D503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Developing spindlin1 small-molecule inhibitors by using protein microarrays</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnchembio.2377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28504676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=750-756&issue=7&author=N.+Baeauthor=M.+Vivianoauthor=X.+Suauthor=J.+Lvauthor=D.+Chengauthor=C.+Sagumauthor=S.+Castellanoauthor=X.+Baiauthor=C.+Johnsonauthor=M.+I.+Khalilauthor=J.+Shenauthor=K.+Chenauthor=H.+Liauthor=G.+Sbardellaauthor=M.+T.+Bedford&title=Developing+spindlin1+small-molecule+inhibitors+by+using+protein+microarrays&doi=10.1038%2Fnchembio.2377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span></div><div class="casAuthors">Bae, Narkhyun; Viviano, Monica; Su, Xiaonan; Lv, Jie; Cheng, Donghang; Sagum, Cari; Castellano, Sabrina; Bai, Xue; Johnson, Claire; Khalil, Mahmoud Ibrahim; Shen, Jianjun; Chen, Kaifu; Li, Haitao; Sbardella, Gianluca; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of inhibitors of methyl- and acetyl-binding domains has provided evidence for the 'druggability' of epigenetic effector mols.  The small-mol. probe, UNC1215, prevents methyl-dependent protein-protein interactions by engaging the arom. cage of MBT domains and, with lower affinity, Tudor domains.  Here, using a library of tagged UNC1215 analogs, we screened a protein-domain microarray of human methyllysine effector mols. to rapidly detect compds. with new binding profiles with either increased or decreased specificity.  Using this approach, we identified a compd. (EML405) that acquired a novel interaction with Tudor-domain-contg. protein Spindlin1 (SPIN1).  Structural studies facilitated the rational synthesis of SPIN1 inhibitors with increased selectivity (EML631-633), which engaged SPIN1 in cells, blocked its ability to 'read' H3K4me3 marks, and inhibited its transcriptional-coactivator activity.  Protein microarrays can thus be used as a platform to 'target-hop' and identify small mols. that bind and compete with domain-motif interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6diBsmZCcm7Vg90H21EOLACvtfcHk0lj2en-dex5JFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D&md5=5ac8aab47d4a67cbe0f7c1f04d4ae410</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2377%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DDeveloping%2520spindlin1%2520small-molecule%2520inhibitors%2520by%2520using%2520protein%2520microarrays%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D7%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnchembio.2377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragstedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Notterpek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, G. P.</span></span> <span> </span><span class="NLM_article-title">Functional cloning of SPIN-2, a nuclear anti-apoptotic protein with roles in cell cycle progression</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1507</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2402557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fsj.leu.2402557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=12145692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1WqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1507-1518&issue=8&author=B.+S.+Fletcherauthor=C.+Dragstedtauthor=L.+Notterpekauthor=G.+P.+Nolan&title=Functional+cloning+of+SPIN-2%2C+a+nuclear+anti-apoptotic+protein+with+roles+in+cell+cycle+progression&doi=10.1038%2Fsj.leu.2402557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Functional cloning of SPIN-2, a nuclear anti-apoptotic protein with roles in cell cycle progression</span></div><div class="casAuthors">Fletcher, B. S.; Dragstedt, C.; Notterpek, L.; Nolan, G. P.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1507-1518</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The balance between hematopoietic cell viability and apoptosis is regulated by exogenous growth factors, however, the mol. mechanisms by which these trophic factors exert their effects remain obscure.  A functional retroviral cDNA library-based screen was employed to identify genes that prevent growth factor withdrawal-mediated apoptosis in the myeloid progenitor cell 32Dcl3.  This approach identified three classes of genes: those with known roles in apoptosis (bcl-XL and ornithine decarboxylase); genes previously identified but not linked directly to apoptotic signaling (O-linked N-acetylglucosamine transferase); and a previously uncharacterized gene we termed SPIN-2.  In 32Dcl3 cells, expression of exogenous SPIN-2 provides 25% protection from apoptosis following growth factor withdrawal compared to controls which show ∼1-2% survival.  SPIN-2 overexpression slows cell growth rates and increases the percentage of cells in G2/M (32% vs control cells at 12%).  Immunolocalization studies indicate that myc-epitope tagged SPIN-2 proteins, which retain their anti-apoptotic function, reside in the nucleus, whereas a C-terminal deletion mutant that loses its anti-apoptotic activity is located in the cytoplasm.  These studies suggest that SPIN-2 is a novel nuclear protein that functions to regulate cell cycle progression and this activity is related to the inhibition of apoptosis following the removal of essential growth factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCApO1iX6F6LVg90H21EOLACvtfcHk0li57HjmswULiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1WqsLk%253D&md5=37dcb6929dfde2efcade6e898fc462b8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2402557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2402557%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DB.%2BS.%26aulast%3DDragstedt%26aufirst%3DC.%26aulast%3DNotterpek%26aufirst%3DL.%26aulast%3DNolan%26aufirst%3DG.%2BP.%26atitle%3DFunctional%2520cloning%2520of%2520SPIN-2%252C%2520a%2520nuclear%2520anti-apoptotic%2520protein%2520with%2520roles%2520in%2520cell%2520cycle%2520progression%26jtitle%3DLeukemia%26date%3D2002%26volume%3D16%26issue%3D8%26spage%3D1507%26epage%3D1518%26doi%3D10.1038%2Fsj.leu.2402557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozretic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jilg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardelmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span> <span> </span><span class="NLM_article-title">The histone code reader SPIN1 controls RET signaling in liposarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">4773</span>– <span class="NLM_lpage">4789</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.18632%2Foncotarget.3000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25749382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4773-4789&issue=7&author=H.+Franzauthor=H.+Greschikauthor=D.+Willmannauthor=L.+Ozreticauthor=C.+A.+Jilgauthor=E.+Wardelmannauthor=M.+Jungauthor=R.+Buettnerauthor=R.+Schule&title=The+histone+code+reader+SPIN1+controls+RET+signaling+in+liposarcoma&doi=10.18632%2Foncotarget.3000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The histone code reader SPIN1 controls RET signaling in liposarcoma</span></div><div class="casAuthors">Franz Henriette; Greschik Holger; Willmann Dominica; Jilg Cordula Annette; Schule Roland; Ozretic Luka; Buettner Reinhard; Wardelmann Eva; Jung Manfred; Jung Manfred; Schule Roland; Schule Roland</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4773-89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The histone code reader Spindlin1 (SPIN1) has been implicated in tumorigenesis and tumor growth, but the underlying molecular mechanisms remain poorly understood.  Here, we show that reducing SPIN1 levels strongly impairs proliferation and increases apoptosis of liposarcoma cells in vitro and in xenograft mouse models.  Combining signaling pathway, genome-wide chromatin binding, and transcriptome analyses, we found that SPIN1 directly enhances expression of GDNF, an activator of the RET signaling pathway, in cooperation with the transcription factor MAZ.  Accordingly, knockdown of SPIN1 or MAZ results in reduced levels of GDNF and activated RET explaining diminished liposarcoma cell proliferation and survival.  In line with these observations, levels of SPIN1, GDNF, activated RET, and MAZ are increased in human liposarcoma compared to normal adipose tissue or lipoma.  Importantly, a mutation of SPIN1 within the reader domain interfering with chromatin binding reduces liposarcoma cell proliferation and survival.  Together, our data describe a molecular mechanism for SPIN1 function in liposarcoma and suggest that targeting SPIN1 chromatin association with small molecule inhibitors may represent a novel therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS5Ma1donXoAsNIRw0GKUoEfW6udTcc2ebcRJNpn2DS57ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D&md5=4b404511399ebfdf8a553af0822e522f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3000%26sid%3Dliteratum%253Aachs%26aulast%3DFranz%26aufirst%3DH.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DOzretic%26aufirst%3DL.%26aulast%3DJilg%26aufirst%3DC.%2BA.%26aulast%3DWardelmann%26aufirst%3DE.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DThe%2520histone%2520code%2520reader%2520SPIN1%2520controls%2520RET%2520signaling%2520in%2520liposarcoma%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26issue%3D7%26spage%3D4773%26epage%3D4789%26doi%3D10.18632%2Foncotarget.3000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janecki, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smialek, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotecki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginter-Matuszewska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuczynska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolanowski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitazawa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurpisz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaruzelska, J.</span></span> <span> </span><span class="NLM_article-title">SPIN1 is a proto-oncogene and SPIN3 is a tumor suppressor in human seminoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">65</span>),  <span class="NLM_fpage">32466</span>– <span class="NLM_lpage">32477</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.25977</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.18632%2Foncotarget.25977" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30197756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ntVOmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=32466-32477&issue=65&author=D.+M.+Janeckiauthor=M.+Sajekauthor=M.+J.+Smialekauthor=M.+Koteckiauthor=B.+Ginter-Matuszewskaauthor=B.+Kuczynskaauthor=A.+Spikauthor=T.+Kolanowskiauthor=R.+Kitazawaauthor=M.+Kurpiszauthor=J.+Jaruzelska&title=SPIN1+is+a+proto-oncogene+and+SPIN3+is+a+tumor+suppressor+in+human+seminoma&doi=10.18632%2Foncotarget.25977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SPIN1 is a proto-oncogene and SPIN3 is a tumor suppressor in human seminoma</span></div><div class="casAuthors">Janecki Damian Mikolaj; Sajek Marcin; Smialek Maciej Jerzy; Kotecki Maciej; Ginter-Matuszewska Barbara; Kuczynska Bogna; Spik Anna; Kolanowski Tomasz; Kurpisz Maciej; Jaruzelska Jadwiga; Kotecki Maciej; Kolanowski Tomasz; Kitazawa Riko</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">65</span>),
    <span class="NLM_cas:pages">32466-32477</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SPIN1 is necessary for normal meiotic progression in mammals.  It is overexpressed in human ovarian cancers and some cancer cell lines.  Here, we examined the functional significance and regulation of SPIN1 and SPIN3 in the TCam-2 human seminoma cell line.  We found that while SPIN1 overexpression reduced apoptosis in these cells, SPIN3 overexpression induced it.  Similarly, SPIN1 upregulated and SPIN3 downregulated CYCD1, which is a downstream target of the PI3K/AKT pathway and contributes to apoptosis resistance in cancer cell lines.  It appears that SPIN1 is pro-oncogenic and SPIN3 acts as a tumor suppressor in TCam-2 cells.  To our knowledge, this is the first report of SPIN3 tumor suppressor activity.  However, both SPIN1 and SPIN3 stimulated cell cycle progression.  In addition, using luciferase reporters carrying SPIN1 or SPIN3 mRNA 3'UTRs, we found that PUM1 and PUM2 targeted and repressed SPINs.  We also found that PUM1 itself strongly stimulated apoptosis and moderately slowed cell cycle progression in TCam-2 cells, suggesting that PUM1, like SPIN3, is a tumor suppressor.  Our findings suggest that acting, at least in part, through SPIN1 and SPIN3, PUM proteins contribute to a mechanism promoting normal human male germ cell apoptotic status and thus preventing cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhTXmHb6E5uTC850By6mxpfW6udTcc2ebcRJNpn2DS57ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ntVOmtQ%253D%253D&md5=4b0c4b4f7124d1050ee8598f34fb991a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.25977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.25977%26sid%3Dliteratum%253Aachs%26aulast%3DJanecki%26aufirst%3DD.%2BM.%26aulast%3DSajek%26aufirst%3DM.%26aulast%3DSmialek%26aufirst%3DM.%2BJ.%26aulast%3DKotecki%26aufirst%3DM.%26aulast%3DGinter-Matuszewska%26aufirst%3DB.%26aulast%3DKuczynska%26aufirst%3DB.%26aulast%3DSpik%26aufirst%3DA.%26aulast%3DKolanowski%26aufirst%3DT.%26aulast%3DKitazawa%26aufirst%3DR.%26aulast%3DKurpisz%26aufirst%3DM.%26aulast%3DJaruzelska%26aufirst%3DJ.%26atitle%3DSPIN1%2520is%2520a%2520proto-oncogene%2520and%2520SPIN3%2520is%2520a%2520tumor%2520suppressor%2520in%2520human%2520seminoma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26issue%3D65%26spage%3D32466%26epage%3D32477%26doi%3D10.18632%2Foncotarget.25977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mennerich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span> <span> </span><span class="NLM_article-title">The Spin/Ssty repeat: a new motif identified in proteins involved in vertebrate development from gamete to embryo</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">RESEARCH0003</span>, <span class="refDoi"> DOI: 10.1186/gb-2001-3-1-research0003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fgb-2001-3-1-research0003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=11806826" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=RESEARCH0003&issue=1&author=E.+Staubauthor=D.+Mennerichauthor=A.+Rosenthal&title=The+Spin%2FSsty+repeat%3A+a+new+motif+identified+in+proteins+involved+in+vertebrate+development+from+gamete+to+embryo&doi=10.1186%2Fgb-2001-3-1-research0003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fgb-2001-3-1-research0003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2001-3-1-research0003%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DE.%26aulast%3DMennerich%26aufirst%3DD.%26aulast%3DRosenthal%26aufirst%3DA.%26atitle%3DThe%2520Spin%252FSsty%2520repeat%253A%2520a%2520new%2520motif%2520identified%2520in%2520proteins%2520involved%2520in%2520vertebrate%2520development%2520from%2520gamete%2520to%2520embryo%26jtitle%3DGenome%2520Biol.%26date%3D2002%26volume%3D3%26issue%3D1%26spage%3DRESEARCH0003%26doi%3D10.1186%2Fgb-2001-3-1-research0003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Expression, purification, crystallization and preliminary X-ray analysis of human spindlin1, an ovarian cancer-related protein</span>. <i>Protein Pept. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.2174/092986606775101661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.2174%2F092986606775101661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=16472086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=203-205&issue=2&author=F.+Jiangauthor=Q.+Zhaoauthor=L.+Qinauthor=H.+Pangauthor=X.+Peiauthor=Z.+Rao&title=Expression%2C+purification%2C+crystallization+and+preliminary+X-ray+analysis+of+human+spindlin1%2C+an+ovarian+cancer-related+protein&doi=10.2174%2F092986606775101661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, crystallization and preliminary X-ray analysis of human spindlin 1, an ovarian cancer-related protein</span></div><div class="casAuthors">Jiang, Fuguo; Zhao, Qiang; Qin, Lipeng; Pang, Hai; Pei, Xuetao; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Peptide Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">203-205</span>CODEN:
                <span class="NLM_cas:coden">PPELEN</span>;
        ISSN:<span class="NLM_cas:issn">0929-8665</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Human spindlin 1 is a newly screened and identified gene product related to ovarian carcinomas and is highly homologous to mouse spindlin.  It is an abundant maternal transcript expressed in the mouse during the transition from oocyte to embryo.  Here, the recombinant human spindlin1 has been overexpressed in Escherichia coli BL21, purified and crystd. using the hanging-drop vapor-diffusion method.  Crystals diffracting to 2.25 Å resoln. were obtained using ammonium sulfate as precipitant.  The crystals belong to the space group P212121, with unit-cell parameters a = 40.7 Å, b = 84.4 Å, c = 136.4 Å, α=β=γ=90°.  Assuming two mols. per asym. unit, the solvent content is calcd. to be 42.4%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8o5Sz0E4XxrVg90H21EOLACvtfcHk0lg5cvyVWJbMvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D&md5=e0968bde24c824de9633702979233a39</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F092986606775101661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986606775101661%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DPang%26aufirst%3DH.%26aulast%3DPei%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DExpression%252C%2520purification%252C%2520crystallization%2520and%2520preliminary%2520X-ray%2520analysis%2520of%2520human%2520spindlin1%252C%2520an%2520ovarian%2520cancer-related%2520protein%26jtitle%3DProtein%2520Pept.%2520Lett.%26date%3D2006%26volume%3D13%26issue%3D2%26spage%3D203%26epage%3D205%26doi%3D10.2174%2F092986606775101661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span> <span> </span><span class="NLM_article-title">Suppression of SPIN1-mediated PI3K-Akt pathway by miR-489 increases chemosensitivity in breast cancer</span>. <i>J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>239</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1002/path.4743</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fpath.4743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27171498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=239&publication_year=2016&pages=459-472&issue=4&author=X.+Chenauthor=Y.+W.+Wangauthor=A.+Y.+Xingauthor=S.+Xiangauthor=D.+B.+Shiauthor=L.+Liuauthor=Y.+X.+Liauthor=P.+Gao&title=Suppression+of+SPIN1-mediated+PI3K-Akt+pathway+by+miR-489+increases+chemosensitivity+in+breast+cancer&doi=10.1002%2Fpath.4743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of SPIN1-mediated PI3K-Akt pathway by miR-489 increases chemosensitivity in breast cancer</span></div><div class="casAuthors">Chen, Xu; Wang, Ya-Wen; Xing, Ai-Yan; Xiang, Shuai; Shi, Duan-Bo; Liu, Lei; Li, Yan-Xiang; Gao, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">239</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">459-472</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">1096-9896</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Drug resistance is one of the major obstacles for improving the prognosis of breast cancer patients.  Increasing evidence has linked the assocn. of aberrantly expressed microRNAs (miRNAs) with tumor development and progression as well as chemoresistance.  Despite recent advances, there is still little known about the potential role and mechanism of miRNAs in breast cancer chemoresistance.  Here we describe that 16 miRNAs were found to be significantly down-regulated and 11 up-regulated in drug-resistant breast cancer tissues compared with drug-sensitive tissues, using a miRNA microarray.  The results also showed miR-489 to be one of the most down-regulated miRNAs in drug-resistant tissues and cell lines, as confirmed by miRNA microarray screening and real-time quant. PCR.  A decrease in miR-489 expression was assocd. with chemoresistance as well as lymph node metastasis, increased tumor size, advanced pTNM stage and poor prognosis in breast cancer.  Functional anal. revealed that miR-489 increased breast cancer chemosensitivity and inhibited cell proliferation, migration and invasion, both in vitro and in vivo.  Furthermore, SPIN1, VAV3, BCL2 and AKT3 were found to be direct targets of miR-489.  SPIN1 was significantly elevated in drug-resistant and metastatic breast cancer tissues and inversely correlated with miR-489 expression.  High expression of SPIN1 was assocd. with higher histol. grade, lymph node metastasis, advanced pTNM stage and pos. progesterone receptor (PR) status.  Increased SPIN1 expression enhanced cell migration and invasion, inhibited apoptosis and partially antagonized the effects of miR-489 in breast cancer.  PIK3CA, AKT, CREB1 and BCL2 in the PI3K-Akt signalling pathway, demonstrated to be elevated in drug-resistant breast cancer tissues, were identified as downstream effectors of SPIN1.  It was further found that either inhibition of SPIN1 or overexpression of miR-489 suppressed the PI3K-Akt signalling pathway.  These data indicate that miR-489 could reverse the chemoresistance of breast cancer via the PI3K-Akt pathway by targeting SPIN1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlYdZmKYimMbVg90H21EOLACvtfcHk0lg5cvyVWJbMvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O&md5=fe996113e57b3a6c2c0cce1b15c6ad0c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fpath.4743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.4743%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%2BW.%26aulast%3DXing%26aufirst%3DA.%2BY.%26aulast%3DXiang%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DD.%2BB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%2BX.%26aulast%3DGao%26aufirst%3DP.%26atitle%3DSuppression%2520of%2520SPIN1-mediated%2520PI3K-Akt%2520pathway%2520by%2520miR-489%2520increases%2520chemosensitivity%2520in%2520breast%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2016%26volume%3D239%26issue%3D4%26spage%3D459%26epage%3D472%26doi%3D10.1002%2Fpath.4743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. Z.</span></span> <span> </span><span class="NLM_article-title">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">102</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28123636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=90-102&issue=1&author=X.+J.+Chenauthor=H.+Dongauthor=S.+Liuauthor=L.+Yuauthor=D.+X.+Yanauthor=X.+W.+Yaoauthor=W.+J.+Sunauthor=D.+Z.+Hanauthor=G.+Z.+Gao&title=Long+noncoding+RNA+MHENCR+promotes+melanoma+progression+via+regulating+miR-425%2F489-mediated+PI3K-Akt+pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span></div><div class="casAuthors">Chen, Xiangjun; Dong, Hao; Liu, Sha; Yu, Li; Yan, Dexiong; Yao, Xingwei; Sun, Weijing; Han, Dezhi; Gao, Guozhen</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">90-105, S1-S2</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Increasing evidences demonstrated that long noncoding RNAs (lncRNAs) are frequently dysregulated and have crit. roles in many tumors.  However, the roles and functional mechanisms of lncRNAs in melanoma remain largely unknown.  In this study, we identified a novel lncRNA MHENCR which was upregulated in melanoma tissues and further upregulated in metastatic melanoma.  Increased expression of MHENCR indicted poor survival of melanoma patients.  Functional expts. revealed that MHENCR knockdown significantly inhibited melanoma cells proliferation, induced cell cycle arrest and apoptosis, and also attenuated melanoma cells migration in vitro.  Furthermore, we identified MHENCR as a competitively endogenous RNA, which specifically bound to miR-425 and miR-489, upregulated their target genes IGF1 and SPIN1 expression, and further activated PI3K-Akt pathway.  Statistically significant correlations were obsd. between MHENCR expression and IGF1 and SPIN1 in melanoma tissues.  In vivo functional expts. further confirmed the pro-growth and pro-metastasis roles of MHENCR.  Collectively, our findings revealed that MHENCR functions as an oncogene in melanoma via activating miR-425/489-mediated PI3K-Akt pathway, and may be a therapeutic target for melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPAKn59fHqdrVg90H21EOLACvtfcHk0lirC9CRBFEgag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK&md5=829b7d67ae7fe250ad4d6576dd3913c6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%2BJ.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DD.%2BX.%26aulast%3DYao%26aufirst%3DX.%2BW.%26aulast%3DSun%26aufirst%3DW.%2BJ.%26aulast%3DHan%26aufirst%3DD.%2BZ.%26aulast%3DGao%26aufirst%3DG.%2BZ.%26atitle%3DLong%2520noncoding%2520RNA%2520MHENCR%2520promotes%2520melanoma%2520progression%2520via%2520regulating%2520miR-425%252F489-mediated%2520PI3K-Akt%2520pathway%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2017%26volume%3D9%26issue%3D1%26spage%3D90%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drago-Ferrante, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentimalli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlisi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Blasio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saliba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldacchino, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degaetano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debono, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caruana-Dingli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grech, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scerri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesoriere, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vento, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Fiore, R.</span></span> <span> </span><span class="NLM_article-title">Suppressive role exerted by microRNA-29b-1–5p in triple negative breast cancer through SPIN1 regulation</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">28939</span>– <span class="NLM_lpage">28958</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.15960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.18632%2Foncotarget.15960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28423652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BC1cvpt1emsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=28939-28958&issue=17&author=R.+Drago-Ferranteauthor=F.+Pentimalliauthor=D.+Carlisiauthor=A.+De+Blasioauthor=C.+Salibaauthor=S.+Baldacchinoauthor=J.+Degaetanoauthor=J.+Debonoauthor=G.+Caruana-Dingliauthor=G.+Grechauthor=C.+Scerriauthor=G.+Tesoriereauthor=A.+Giordanoauthor=R.+Ventoauthor=R.+Di+Fiore&title=Suppressive+role+exerted+by+microRNA-29b-1%E2%80%935p+in+triple+negative+breast+cancer+through+SPIN1+regulation&doi=10.18632%2Foncotarget.15960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Suppressive role exerted by microRNA-29b-1-5p in triple negative breast cancer through SPIN1 regulation</span></div><div class="casAuthors">Drago-Ferrante Rosa; De Blasio Anna; Vento Renza; Di Fiore Riccardo; Pentimalli Francesca; Carlisi Daniela; Saliba Christian; Baldacchino Shawn; Grech Godfrey; Degaetano James; Scerri Christian; Debono Joseph; Caruana-Dingli Gordon; Scerri Christian; Tesoriere Giovanni; Giordano Antonio; Vento Renza; Tesoriere Giovanni; Vento Renza; Di Fiore Riccardo; Giordano Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">28939-28958</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MiR-29 family dysregulation occurs in various cancers including breast cancers.  We investigated miR-29b-1 functional role in human triple negative breast cancer (TNBC) the most aggressive breast cancer subtype.  We found that miR-29b-1-5p was downregulated in human TNBC tissues and cell lines.  To assess whether miR-29b-1-5p correlated with TNBC regenerative potential, we evaluated cancer stem cell enrichment in our TNBC cell lines, and found that only MDA-MB-231 and BT-20 produced primary, secondary and tertiary mammospheres, which were progressively enriched in OCT4, NANOG and SOX2 stemness genes.  MiR-29b-1-5p expression inversely correlated with mammosphere stemness potential, and miR-29b-1 ectopic overexpression decreased TNBC cell growth, self-renewal, migration, invasiveness and paclitaxel resistance repressing WNT/βcatenin and AKT signaling pathways and stemness regulators.  We identified SPINDLIN1 (SPIN1) among predicted miR-29b-1-5p targets.  Consistently, SPIN1 was overexpressed in most TNBC tissues and cell lines and negatively correlated with miR-29b-1-5p.  Target site inhibition showed that SPIN1 seems to be directly controlled by miR-29b-1-5p.  Silencing SPIN1 mirrored the effects triggered by miR-29b-1 overexpression, whereas SPIN1 rescue by SPIN1miScript protector, determined the reversal of the molecular effects produced by the mimic-miR-29b-1-5p.  Overall, we show that miR-29b-1 deregulation impacts on multiple oncogenic features of TNBC cells and their renewal potential, acting, at least partly, through SPIN1, and suggest that both these factors should be evaluated as new possible therapeutic targets against TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0Sy6WJkt3IgvupmecNw_lfW6udTcc2eb5YBqTA28V0rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvpt1emsg%253D%253D&md5=08705be8348e656d28e67fd8e86013c9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.15960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.15960%26sid%3Dliteratum%253Aachs%26aulast%3DDrago-Ferrante%26aufirst%3DR.%26aulast%3DPentimalli%26aufirst%3DF.%26aulast%3DCarlisi%26aufirst%3DD.%26aulast%3DDe%2BBlasio%26aufirst%3DA.%26aulast%3DSaliba%26aufirst%3DC.%26aulast%3DBaldacchino%26aufirst%3DS.%26aulast%3DDegaetano%26aufirst%3DJ.%26aulast%3DDebono%26aufirst%3DJ.%26aulast%3DCaruana-Dingli%26aufirst%3DG.%26aulast%3DGrech%26aufirst%3DG.%26aulast%3DScerri%26aufirst%3DC.%26aulast%3DTesoriere%26aufirst%3DG.%26aulast%3DGiordano%26aufirst%3DA.%26aulast%3DVento%26aufirst%3DR.%26aulast%3DDi%2BFiore%26aufirst%3DR.%26atitle%3DSuppressive%2520role%2520exerted%2520by%2520microRNA-29b-1%25E2%2580%25935p%2520in%2520triple%2520negative%2520breast%2520cancer%2520through%2520SPIN1%2520regulation%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26issue%3D17%26spage%3D28939%26epage%3D28958%26doi%3D10.18632%2Foncotarget.15960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span> <span> </span><span class="NLM_article-title">SPIN1 promotes tumorigenesis by blocking the uL18 (universal large ribosomal subunit protein 18)-MDM2-p53 pathway in human cancer</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e31275</span> <span class="refDoi"> DOI: 10.7554/eLife.31275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.7554%2FeLife.31275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29547122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVSjur7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=Z.+L.+Fangauthor=B.+Caoauthor=J.+M.+Liaoauthor=J.+Dengauthor=K.+D.+Plummerauthor=P.+Liaoauthor=T.+Liuauthor=W.+S.+Zhangauthor=K.+Zhangauthor=L.+Liauthor=D.+Margolinauthor=S.+X.+Zengauthor=J.+P.+Xiongauthor=H.+Lu&title=SPIN1+promotes+tumorigenesis+by+blocking+the+uL18+%28universal+large+ribosomal+subunit+protein+18%29-MDM2-p53+pathway+in+human+cancer&doi=10.7554%2FeLife.31275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">SPIN1 promotes tumorigenesis by blocking the uL18 (universal large ribosomal subunit protein 18)-MDM2-p53 pathway in human cancer</span></div><div class="casAuthors">Fang, Ziling; Cao, Bo; Liao, Jun-Ming; Deng, Jun; Plummer, Kevin D.; Liao, Peng; Liu, Tao; Zhang, Wensheng; Zhang, Kun; Li, Li; Margolin, David; Zeng, Shelya X.; Xiong, Jianping; Lu, Hua</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e31275/1-e31275/21</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Ribosomal proteins (RPs) play important roles in modulating the MDM2-p53 pathway.  However, less is known about the upstream regulators of the RPs.  Here, we identify SPIN1 (Spindlin 1) as a novel binding partner of human RPL5/uL18 that is important for this pathway.  SPIN1 ablation activates p53, suppresses cell growth, reduces clonogenic ability, and induces apoptosis of human cancer cells.  Mechanistically, SPIN1 sequesters uL18 in the nucleolus, preventing it from interacting with MDM2, and thereby alleviating uL18-mediated inhibition of MDM2 ubiquitin ligase activity toward p53.  SPIN1 deficiency increases ribosome-free uL18 and uL5 (human RPL11), which are required for SPIN1 depletion-induced p53 activation.  Anal. of cancer genomic databases suggests that SPIN1 is highly expressed in several human cancers, and its overexpression is pos. correlated with poor prognosis in cancer patients.  Altogether, our findings reveal that the oncogenic property of SPIN1 may be attributed to its neg. regulation of uL18, leading to p53 inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoTVn20u1NhrVg90H21EOLACvtfcHk0ljvcz7SAzPNdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVSjur7L&md5=67586bfbe1559f0339a7e60aceae6bd0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.7554%2FeLife.31275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.31275%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DZ.%2BL.%26aulast%3DCao%26aufirst%3DB.%26aulast%3DLiao%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DK.%2BD.%26aulast%3DLiao%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%2BS.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMargolin%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DS.%2BX.%26aulast%3DXiong%26aufirst%3DJ.%2BP.%26aulast%3DLu%26aufirst%3DH.%26atitle%3DSPIN1%2520promotes%2520tumorigenesis%2520by%2520blocking%2520the%2520uL18%2520%2528universal%2520large%2520ribosomal%2520subunit%2520protein%252018%2529-MDM2-p53%2520pathway%2520in%2520human%2520cancer%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2FeLife.31275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G. H.</span></span> <span> </span><span class="NLM_article-title">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2017.06.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.biopha.2017.06.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28666210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=435-443&author=Y.+Liauthor=X.+L.+Maauthor=Y.+P.+Wangauthor=G.+H.+Li&title=miR-489+inhibits+proliferation%2C+cell+cycle+progression+and+induces+apoptosis+of+glioma+cells+via+targeting+SPIN1-mediated+PI3K%2FAKT+pathway&doi=10.1016%2Fj.biopha.2017.06.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span></div><div class="casAuthors">Li, Yan; Ma, Xiaolin; Wang, Yanpeng; Li, Guohua</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MicroRNA-489 (miR-489), a newly identified tumor-related miRNA, functions as an oncogene or tumor suppressor via regulating growth and metastasis of human cancers.  But, the clin. significance, biol. function and underlying mechanisms of miR-489 in glioma remain rarely known.  Here, we showed that the levels of miR-489 in glioma tissues were notably underexpressed compared to corresponding non-tumor tissues.  In accordance, the relative levels of miR-489 were decreased in glioma cell lines compared with NHA cells.  Kaplan-Meier plots indicated that miR-489 low expressing glioma patients showed a prominent shorter overall survival.  In addn., miR-489 overexpression prohibited proliferation and cell cycle progression, and promoted apoptosis in U251 cells.  While, miR-489 knockdown showed opposite effects on these cellular processes of U87 cells.  In vivo expts. demonstrated that miR-489 restoration reduced the tumor vol. and wt. of s.c. glioma xenografts in nude mice.  Notably, Spindlin 1 (SPIN1) was inversely and directly regulated by miR-489 in glioma cells.  A neg. correlation between the expression of miR-489 and SPIN1 mRNA was confirmed in glioma tissues.  Interestingly, miR-489 inversely modulated activation of PI3K/AKT pathway and expression of downstream targets including p-mTOR, Cyclin D1 and BCL-XL.  SPIN1 re-expression abolished the effects of miR-489 on U251 cells with enhanced activation of PI3K/AKT pathway and malignant phenotype.  Meanwhile, AKT inhibitor MK-2206 blocked activation of PI3K/AKT pathway and resulted in reduced proliferation, cell cycle arrest and increased apoptosis in miR-489 down-regulating U87 cells.  Altogether, our data support that miR-489 loss facilitates malignant phenotype of glioma cells probably via SPIN1-mediated PI3K/AKT pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq39T3IjNfjurVg90H21EOLACvtfcHk0ljvcz7SAzPNdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE&md5=4cd57f05e0bbfec6fd2206244d73c291</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2017.06.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2017.06.058%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DY.%2BP.%26aulast%3DLi%26aufirst%3DG.%2BH.%26atitle%3DmiR-489%2520inhibits%2520proliferation%252C%2520cell%2520cycle%2520progression%2520and%2520induces%2520apoptosis%2520of%2520glioma%2520cells%2520via%2520targeting%2520SPIN1-mediated%2520PI3K%252FAKT%2520pathway%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2017%26volume%3D93%26spage%3D435%26epage%3D443%26doi%3D10.1016%2Fj.biopha.2017.06.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, S.</span></span> <span> </span><span class="NLM_article-title">miR-409 inhibits human non-small-cell lung cancer progression by directly targeting SPIN1</span>. <i>Mol. Ther.--Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">163</span>, <span class="refDoi"> DOI: 10.1016/j.omtn.2018.08.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.omtn.2018.08.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30290307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVGmsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=154-163&author=Q.+Songauthor=Q.+Jiauthor=J.+Xiaoauthor=F.+Liauthor=L.+Wangauthor=Y.+Chenauthor=Y.+Xuauthor=S.+Jiao&title=miR-409+inhibits+human+non-small-cell+lung+cancer+progression+by+directly+targeting+SPIN1&doi=10.1016%2Fj.omtn.2018.08.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">miR-409 Inhibits Human Non-Small-Cell Lung Cancer Progression by Directly Targeting SPIN1</span></div><div class="casAuthors">Song, Qi; Ji, Quanbo; Xiao, Jingbo; Li, Fang; Wang, Lingxiong; Chen, Yin; Xu, Yameng; Jiao, Shunchang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy--Nucleic Acids</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">154-163</span>CODEN:
                <span class="NLM_cas:coden">MTAOC5</span>;
        ISSN:<span class="NLM_cas:issn">2162-2531</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Lung cancers, the leading cause of cancer mortality worldwide, are characterized by a high metastatic potential.  Growing evidence reveals that Spindlin 1 (SPIN1) is involved in tumor progression and carcinogenesis.  However, the role of SPIN1 in non-small-cell lung cancer (NSCLC) and the mol. mechanisms underlying SPIN1 in human NSCLC remain undetd.  Here we examd. the function of SPIN1 in human NSCLC and found that the expression of SPIN1 was closely correlated with the overall survival and poor prognosis of NSCLC patients.  Aberrant regulation of microRNAs (miRNAs) has an important role in cancer progression.  We revealed that miR-409 inhibits the expression of SPIN1 by binding directly to the 3' UTR of SPIN1 using dual-luciferase reporter assays.  Overexpression of miR-409 significantly suppressed cell migration, growth, and proliferation by inhibiting SPIN1 in vitro and in vivo.  SPIN1 overexpression in miR-409-transfected NSCLC cells effectively rescued the suppression of cell migration, growth, and proliferation regulated by miR-409. miR-409 regulates the PI3K/AKT (protein kinase B) pathway in NSCLC.  Moreover, clin. data showed that NSCLC patients with high levels of miR-409 experienced significantly better survival. miR-409 expression was also neg. assocd. with SPIN1 expression.  Taken together, these findings highlight that the miR-409/SPIN1 axis is a useful pleiotropic regulatory network and could predict the metastatic potential in NSCLC patients early, indicating the possibility that miR-409 and SPIN1 might be attractive prognostic markers for treating NSCLC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIdBArJUj8rVg90H21EOLACvtfcHk0lg6tyZttPnrsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVGmsrfP&md5=2cc1f738488ea98da0961d60897f4a87</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.omtn.2018.08.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.omtn.2018.08.020%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DQ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DJiao%26aufirst%3DS.%26atitle%3DmiR-409%2520inhibits%2520human%2520non-small-cell%2520lung%2520cancer%2520progression%2520by%2520directly%2520targeting%2520SPIN1%26jtitle%3DMol.%2520Ther.--Nucleic%2520Acids%26date%3D2018%26volume%3D13%26spage%3D154%26epage%3D163%26doi%3D10.1016%2Fj.omtn.2018.08.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H. T.</span></span> <span> </span><span class="NLM_article-title">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1101/gad.233239.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1101%2Fgad.233239.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24589551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=622-636&issue=6&author=X.+N.+Suauthor=G.+X.+Zhuauthor=X.+Z.+Dingauthor=S.+Y.+Leeauthor=Y.+L.+Douauthor=B.+Zhuauthor=W.+Wuauthor=H.+T.+Li&title=Molecular+basis+underlying+histone+H3+lysine-arginine+methylation+pattern+readout+by+Spin%2FSsty+repeats+of+Spindlin1&doi=10.1101%2Fgad.233239.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span></div><div class="casAuthors">Su, Xiaonan; Zhu, Guixin; Ding, Xiaozhe; Lee, Shirley Y.; Dou, Yali; Zhu, Bing; Wu, Wei; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">622-636</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone modification patterns and their combinatorial readout have emerged as a fundamental mechanism for epigenetic regulation.  Here the authors characterized Spindlin1 as a histone effector that senses a cis-tail histone H3 methylation pattern involving trimethyllysine 4 (H3K4me3) and asym. dimethylarginine 8 (H3R8me2a) marks.  Spindlin1 consists of triple tudor-like Spin/Ssty repeats.  Cocrystal structure detn. established concurrent recognition of H3K4me3 and H3R8me2a by Spin/Ssty repeats 2 and 1, resp.  Both H3K4me3 and H3R8me2a are recognized using an "insertion cavity" recognition mode, contributing to a methylation state-specific layer of regulation.  In vivo functional studies suggest that Spindlin1 activates Wnt/β-catenin signaling downstream from protein arginine methyltransferase 2 (PRMT2) and the MLL complex, which together are capable of generating a specific H3 "K4me3-R8me2a" pattern.  Mutagenesis of Spindlin1 reader pockets impairs activation of Wnt target genes.  Taken together, this work connects a histone "lysine-arginine" methylation pattern readout by Spindlin1-to-Wnt signaling at the transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2eotrU6XRPrVg90H21EOLACvtfcHk0lg6tyZttPnrsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D&md5=6872607b2651d15d16b80a6d33555bbd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1101%2Fgad.233239.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.233239.113%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DX.%2BN.%26aulast%3DZhu%26aufirst%3DG.%2BX.%26aulast%3DDing%26aufirst%3DX.%2BZ.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DDou%26aufirst%3DY.%2BL.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%2BT.%26atitle%3DMolecular%2520basis%2520underlying%2520histone%2520H3%2520lysine-arginine%2520methylation%2520pattern%2520readout%2520by%2520Spin%252FSsty%2520repeats%2520of%2520Spindlin1%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26issue%3D6%26spage%3D622%26epage%3D636%26doi%3D10.1101%2Fgad.233239.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X. T.</span></span> <span> </span><span class="NLM_article-title">Spindlin1 promotes cancer cell proliferation through activation of WNT/TCF-4 signaling</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-11-0440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1158%2F1541-7786.MCR-11-0440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22258766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=326-335&issue=3&author=J.+X.+Wangauthor=Q.+Zengauthor=L.+Chenauthor=J.+C.+Duauthor=X.+L.+Yanauthor=H.+F.+Yuanauthor=C.+Zhaiauthor=J.+N.+Zhouauthor=Y.+L.+Jiaauthor=W.+Yueauthor=X.+T.+Pei&title=Spindlin1+promotes+cancer+cell+proliferation+through+activation+of+WNT%2FTCF-4+signaling&doi=10.1158%2F1541-7786.MCR-11-0440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">SPINDLIN1 Promotes Cancer Cell Proliferation through Activation of WNT/TCF-4 Signaling</span></div><div class="casAuthors">Wang, Jing-Xue; Zeng, Quan; Chen, Lin; Du, Ji-Chao; Yan, Xin-Long; Yuan, Hong-Feng; Zhai, Chao; Zhou, Jun-Nian; Jia, Ya-Li; Yue, Wen; Pei, Xue-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">326-335</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">SPINDLIN1, a new member of the SPIN/SSTY gene family, was first identified as a gene highly expressed in ovarian cancer cells.  We have previously shown that it is involved in the process of spindle organization and chromosomal stability and plays a role in the development of cancer.  Nevertheless, the mechanisms underlying its oncogenic role are still largely unknown.  Here, we first showed that expression of SPINDLIN1 is upregulated in clin. tumors.  Ectopic expression of SPINDLIN1 promoted cancer cell proliferation and activated WNT/T-cell factor (TCF)-4 signaling.  The Ser84 and Ser99 amino acids within SPINDLIN1 were further identified as the key functional sites in WNT/TCF-4 signaling activation.  Mutation of these two sites of SPINDLIN1 abolished its effects on promoting WNT/TCF-4 signaling and cancer cell proliferation.  We further found that Aurora-A could interact with and phosphorylate SPINDLIN1 at its key functional sites, Ser84 and Ser99, suggesting that phosphorylation of SPINDLIN1 is involved in its oncogenic function.  Collectively, these results suggest that SPINDLIN1, which may be a novel substrate of the Aurora-A kinase, promotes cancer cell growth through WNT/TCF-4 signaling activation.  Mol Cancer Res; 10(3); 326-35.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoie76oMHEUN7Vg90H21EOLACvtfcHk0lg6tyZttPnrsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D&md5=0446631fe80c7b518ce305c394075143</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-11-0440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-11-0440%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BX.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DJ.%2BC.%26aulast%3DYan%26aufirst%3DX.%2BL.%26aulast%3DYuan%26aufirst%3DH.%2BF.%26aulast%3DZhai%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DJ.%2BN.%26aulast%3DJia%26aufirst%3DY.%2BL.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DPei%26aufirst%3DX.%2BT.%26atitle%3DSpindlin1%2520promotes%2520cancer%2520cell%2520proliferation%2520through%2520activation%2520of%2520WNT%252FTCF-4%2520signaling%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2012%26volume%3D10%26issue%3D3%26spage%3D326%26epage%3D335%26doi%3D10.1158%2F1541-7786.MCR-11-0440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Nucleosome-interacting proteins regulated by DNA and histone methylation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>143</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.cell.2010.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21029866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKhs7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2010&pages=470-484&issue=3&author=T.+Bartkeauthor=M.+Vermeulenauthor=B.+Xhemalceauthor=S.+C.+Robsonauthor=M.+Mannauthor=T.+Kouzarides&title=Nucleosome-interacting+proteins+regulated+by+DNA+and+histone+methylation&doi=10.1016%2Fj.cell.2010.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleosome-Interacting Proteins Regulated by DNA and Histone Methylation</span></div><div class="casAuthors">Bartke, Till; Vermeulen, Michiel; Xhemalce, Blerta; Robson, Samuel C.; Mann, Matthias; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-484</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Modifications on histones or DNA recruit proteins that regulate chromatin function.  Here, we use nucleosomes methylated on DNA and on histone H3 in an affinity assay, in conjunction with a SILAC-based proteomic anal., to identify "crosstalk" between these two distinct classes of modification.  Our anal. reveals proteins whose binding to nucleosomes is regulated by methylation of CpGs, H3K4, H3K9, and H3K27 or a combination thereof.  We identify the origin recognition complex (ORC), including LRWD1 as a subunit, to be a methylation-sensitive nucleosome interactor that is recruited cooperatively by DNA and histone methylation.  Other interactors, such as the lysine demethylase Fbxl11/KDM2A, recognize nucleosomes methylated on histones, but their recruitment is disrupted by DNA methylation.  These data establish SILAC nucleosome affinity purifications (SNAP) as a tool for studying the dynamics between different chromatin modifications and provide a modification binding "profile" for proteins regulated by DNA and histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxe7LiuFnuUrVg90H21EOLACvtfcHk0lg6tyZttPnrsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKhs7rO&md5=c5ab50feb4d60e56f69011f7c5773818</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DBartke%26aufirst%3DT.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DXhemalce%26aufirst%3DB.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DNucleosome-interacting%2520proteins%2520regulated%2520by%2520DNA%2520and%2520histone%2520methylation%26jtitle%3DCell%26date%3D2010%26volume%3D143%26issue%3D3%26spage%3D470%26epage%3D484%26doi%3D10.1016%2Fj.cell.2010.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span> <span> </span><span class="NLM_article-title">Nucleolar protein spindlin1 recognizes H3K4 methylation and stimulates the expression of rRNA genes</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1166</span>, <span class="refDoi"> DOI: 10.1038/embor.2011.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fembor.2011.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21960006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GgtrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=1160-1166&issue=11&author=W.+X.+Wangauthor=Z.+Chenauthor=Z.+Maoauthor=H.+H.+Zhangauthor=X.+J.+Dingauthor=S.+Chenauthor=X.+D.+Zhangauthor=R.+M.+Xuauthor=B.+Zhu&title=Nucleolar+protein+spindlin1+recognizes+H3K4+methylation+and+stimulates+the+expression+of+rRNA+genes&doi=10.1038%2Fembor.2011.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleolar protein Spindlin1 recognizes H3K4 methylation and stimulates the expression of rRNA genes</span></div><div class="casAuthors">Wang, Weixiang; Chen, Zhi; Mao, Zhuo; Zhang, Huihui; Ding, Xiaojun; Chen, She; Zhang, Xiaodong; Xu, Ruiming; Zhu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1160-1166</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tandem Tudor-like domain-contg. protein Spindlin 1 (I) has been reported to be a meiotic spindle-assocd. protein.  Here, the authors report that I was not assocd. with the spindle in mouse embryonic fibroblast cells during mitotic divisions.  In interphase cells, I specifically localized to the nucleoli.  Moreover, I was a histone methylation effector protein that specifically recognized H3K4 methylation.  Finally, I localized to active ribosomal DNA (rDNA) repeats, and facilitated the expression of rRNA genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3x3h2CCy9yrVg90H21EOLACvtfcHk0liwx0wsijOgNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GgtrjJ&md5=dc1fa70767d6fa557329310bc688ba67</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fembor.2011.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2011.184%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%2BH.%26aulast%3DDing%26aufirst%3DX.%2BJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%2BD.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DZhu%26aufirst%3DB.%26atitle%3DNucleolar%2520protein%2520spindlin1%2520recognizes%2520H3K4%2520methylation%2520and%2520stimulates%2520the%2520expression%2520of%2520rRNA%2520genes%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26issue%3D11%26spage%3D1160%26epage%3D1166%26doi%3D10.1038%2Fembor.2011.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shanle, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinsky, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridgers, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span> <span> </span><span class="NLM_article-title">Histone peptide microarray screen of chromo and tudor domains defines new histone lysine methylation interactions</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">12</span>, <span class="refDoi"> DOI: 10.1186/s13072-017-0117-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fs13072-017-0117-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28293301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvVSktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=12&author=E.+K.+Shanleauthor=S.+A.+Shinskyauthor=J.+B.+Bridgersauthor=N.+Baeauthor=C.+Sagumauthor=K.+Krajewskiauthor=S.+B.+Rothbartauthor=M.+T.+Bedfordauthor=B.+D.+Strahl&title=Histone+peptide+microarray+screen+of+chromo+and+tudor+domains+defines+new+histone+lysine+methylation+interactions&doi=10.1186%2Fs13072-017-0117-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Histone peptide microarray screen of chromo and tudor domains defines new histone lysine methylation interactions</span></div><div class="casAuthors">Shanle, Erin K.; Shinsky, Stephen A.; Bridgers, Joseph B.; Bae, Narkhyun; Sagum, Cari; Krajewski, Krzysztof; Rothbart, Scott B.; Bedford, Mark T.; Strahl, Brian D.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12/1-12/11</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Histone posttranslational modifications (PTMs) function to regulate chromatin structure and function in part through the recruitment of effector proteins that harbor specialized "reader" domains.  Despite efforts to elucidate reader domain-PTM interactions, the influence of neighboring PTMs and the target specificity of many reader domains is still unclear.  The aim of this study was to use a high-throughput histone peptide microarray platform to interrogate 83 known and putative histone reader domains from the chromo and Tudor domain families to identify their interactions and characterize the influence of neighboring PTMs on these interactions.  Results: Nearly a quarter of the chromo and Tudor domains screened showed interactions with histone PTMs by peptide microarray, revealing known and several novel methyllysine interactions.  Specifically, we found that the CBX/HP1 chromodomains that recognize H3K9me also recognize H3K23me2/3-a poorly understood histone PTM.  We also obsd. that, in addn. to their interaction with H3K4me3, Tudor domains of the Spindlin family also recognized H4K20me3-a previously uncharacterized interaction.  Several Tudor domains also showed novel interactions with H3K4me as well.  Conclusions: These results provide an important resource for the epigenetics and chromatin community on the interactions of many human chromo and Tudor domains.  They also provide the basis for addnl. studies into the functional significance of the novel interactions that were discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8JuJ85vX7hrVg90H21EOLACvtfcHk0liwx0wsijOgNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvVSktrY%253D&md5=15522ac8214eaaff9293f255cf85e2b5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2Fs13072-017-0117-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13072-017-0117-5%26sid%3Dliteratum%253Aachs%26aulast%3DShanle%26aufirst%3DE.%2BK.%26aulast%3DShinsky%26aufirst%3DS.%2BA.%26aulast%3DBridgers%26aufirst%3DJ.%2BB.%26aulast%3DBae%26aufirst%3DN.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DRothbart%26aufirst%3DS.%2BB.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26atitle%3DHistone%2520peptide%2520microarray%2520screen%2520of%2520chromo%2520and%2520tudor%2520domains%2520defines%2520new%2520histone%2520lysine%2520methylation%2520interactions%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2017%26volume%3D10%26spage%3D12%26doi%3D10.1186%2Fs13072-017-0117-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premkumar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">A transcriptional coregulator, SPIN.DOC, attenuates the coactivator activity of spindlin1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">20808</span>– <span class="NLM_lpage">20817</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.814913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1074%2Fjbc.M117.814913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29061846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=20808-20817&issue=51&author=N.+Baeauthor=M.+Gaoauthor=X.+Liauthor=T.+Premkumarauthor=G.+Sbardellaauthor=J.+J.+Chenauthor=M.+T.+Bedford&title=A+transcriptional+coregulator%2C+SPIN.DOC%2C+attenuates+the+coactivator+activity+of+spindlin1&doi=10.1074%2Fjbc.M117.814913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A transcriptional coregulator, SPIN·DOC, attenuates the coactivator activity of Spindlin1</span></div><div class="casAuthors">Bae, Narkhyun; Gao, Min; Li, Xu; Premkumar, Tolkappiyan; Sbardella, Gianluca; Chen, Junjie; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20808-20817</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Spindlin1 (SPIN1) is a transcriptional coactivator with crit. functions in embryonic development and emerging roles in cancer.  SPIN1 harbors three Tudor domains, two of which engage the tail of histone H3 by reading the H3-Lys-4 trimethylation and H3-Arg-8 asym. dimethylation marks.  To gain mechanistic insight into how SPIN1 functions as a transcriptional coactivator, here we purified its interacting proteins.  We identified an uncharacterized protein (C11orf84), which we renamed SPIN1 docking protein (SPIN·DOC), that directly binds SPIN1 and strongly disrupts its histone methylation reading ability, causing it to disassoc. from chromatin.  The Spindlin family of coactivators has five related members (SPIN1, 2A, 2B, 3, and 4), and we found that all of them bind SPIN·DOC.  It has been reported previously that SPIN1 regulates gene expression in the Wnt signaling pathway by directly interacting with transcription factor 4 (TCF4).  We obsd. here that SPIN·DOC assocs. with TCF4 in a SPIN1-dependent manner and dampens SPIN1 coactivator activity in TOPflash reporter assays.  Furthermore, knockdown and overexpression expts. indicated that SPIN·DOC represses the expression of a no. of SPIN1-regulated genes, including those encoding rRNA and the cytokine IL1B.  In conclusion, we have identified SPIN·DOC as a transcriptional repressor that binds SPIN1 and masks its ability to engage the H3-Lys-4 trimethylation activation mark.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_Fa2GbBnnLrVg90H21EOLACvtfcHk0liwx0wsijOgNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE&md5=69c4c937798fe7e27f6234156291217f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.814913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.814913%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPremkumar%26aufirst%3DT.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DA%2520transcriptional%2520coregulator%252C%2520SPIN.DOC%252C%2520attenuates%2520the%2520coactivator%2520activity%2520of%2520spindlin1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26issue%3D51%26spage%3D20808%26epage%3D20817%26doi%3D10.1074%2Fjbc.M117.814913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luise, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span> <span> </span><span class="NLM_article-title">Identification and structure-activity relationship studies of small-molecule inhibitors of the methyllysine reader protein spindlin1</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">2327</span>– <span class="NLM_lpage">2338</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fcmdc.201600362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27634332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWls7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2327-2338&issue=20&author=D.+Robaaauthor=T.+Wagnerauthor=C.+Luiseauthor=L.+Carlinoauthor=J.+McMillanauthor=R.+Flaigauthor=R.+Schuleauthor=M.+Jungauthor=W.+Sippl&title=Identification+and+structure-activity+relationship+studies+of+small-molecule+inhibitors+of+the+methyllysine+reader+protein+spindlin1&doi=10.1002%2Fcmdc.201600362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Structure-Activity Relationship Studies of Small-Molecule Inhibitors of the Methyllysine Reader Protein Spindlin1</span></div><div class="casAuthors">Robaa, Dina; Wagner, Tobias; Luise, Chiara; Carlino, Luca; McMillan, Joel; Flaig, Ralf; Schuele, Roland; Jung, Manfred; Sippl, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2327-2338</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The methyllysine reader protein Spindlin1 has been implicated in the tumorigenesis of several types of cancer and may be an attractive novel therapeutic target.  Small-mol. inhibitors of Spindlin1 should be valuable as chem. probes as well as potential new therapeutics.  We applied an iterative virtual screening campaign, encompassing structure- and ligand-based approaches, to identify potential Spindlin1 inhibitors from databases of com. available compds.  Our in silico studies coupled with in vitro testing were successful in identifying novel Spindlin1 inhibitors.  Several 4-aminoquinazoline and quinazolinethione derivs. were among the active hit compds., which indicated that these scaffolds represent promising lead structures for the development of Spindlin1 inhibitors.  Subsequent lead optimization studies were hence carried out, and numerous derivs. of both lead scaffolds were synthesized.  This resulted in the discovery of novel inhibitors of Spindlin1 and helped explore the structure-activity relationships of these inhibitor series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0z2mZOjLXb7Vg90H21EOLACvtfcHk0lhs8NeN6xV9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWls7%252FN&md5=4537d16c26a111a351c0e795279b8401</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600362%26sid%3Dliteratum%253Aachs%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DLuise%26aufirst%3DC.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DMcMillan%26aufirst%3DJ.%26aulast%3DFlaig%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DIdentification%2520and%2520structure-activity%2520relationship%2520studies%2520of%2520small-molecule%2520inhibitors%2520of%2520the%2520methyllysine%2520reader%2520protein%2520spindlin1%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26issue%3D20%26spage%3D2327%26epage%3D2338%26doi%3D10.1002%2Fcmdc.201600362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of potent and selective inhibitors of histone methyltransferase G9a</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&issue=2&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+L.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+development+of+potent+and+selective+inhibitors+of+histone+methyltransferase+G9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0lhs8NeN6xV9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520development%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D2%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">Open access chemical probes for epigenetic targets</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">1901</span>– <span class="NLM_lpage">1917</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.4155%2Ffmc.15.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26397018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1901-1917&issue=14&author=P.+J.+Brownauthor=S.+Muller&title=Open+access+chemical+probes+for+epigenetic+targets&doi=10.4155%2Ffmc.15.127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Open access chemical probes for epigenetic targets</span></div><div class="casAuthors">Brown, Peter J.; Mueller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1901-1917</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: High attrition rates in drug discovery call for new approaches to improve target validation.  Academia is filling gaps, but often lacks the experience and resources of the pharmaceutical industry resulting in poorly characterized tool compds.  Discussion: The SGC has established an open access chem. probe consortium, currently encompassing ten pharmaceutical companies.  One of its mandates is to create well-characterized inhibitors (chem. probes) for epigenetic targets to enable new biol. and target validation for drug development.  Conclusion: Epigenetic probe compds. have proven to be very valuable and have not only spurred a plethora of novel biol. findings, but also provided starting points for clin. trials.  These probes have proven to be crit. complementation to traditional genetic targeting strategies and provided sometimes surprising results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX4t2GBZPCzLVg90H21EOLACvtfcHk0lhs8NeN6xV9yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK&md5=09973fb3a77bee2a6a65e7fca190f2aa</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.127%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DOpen%2520access%2520chemical%2520probes%2520for%2520epigenetic%2520targets%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26issue%3D14%26spage%3D1901%26epage%3D1917%26doi%3D10.4155%2Ffmc.15.127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Hagan, R. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J</span></span> <span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&issue=8&author=C.+H+Arrowsmithauthor=J.+E+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E+Brennanauthor=P.+J+Brownauthor=M.+E+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M+Campbellauthor=A.+J+Carterauthor=P.+Cohenauthor=R.+A+Copelandauthor=B.+Cravattauthor=J.+L+Dahlinauthor=D.+Dhanakauthor=A.+M+Edwardsauthor=M.+Frederiksenauthor=S.+V+Fryeauthor=N.+Grayauthor=C.+E+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V+M+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D+Kotzauthor=R.+G+Krugerauthor=D.+Loweauthor=M.+M+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+Mullerauthor=R.+C+O%27Haganauthor=J.+P+Overingtonauthor=D.+R+Owenauthor=S.+H+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L+Schreiberauthor=B.+Shoichetauthor=M.+Sundstromauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A+Waltersauthor=T.+M+Willsonauthor=P.+Workmanauthor=R.+N+Youngauthor=W.+J+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lj7QB2mYSoBtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH%26aulast%3DAudia%26aufirst%3DJ.%2BE%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE%26aulast%3DBrown%26aufirst%3DP.%2BJ%26aulast%3DBunnage%26aufirst%3DM.%2BE%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM%26aulast%3DCarter%26aufirst%3DA.%2BJ%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV%2BM%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD%26aulast%3DKruger%26aufirst%3DR.%2BG%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%2527Hagan%26aufirst%3DR.%2BC%26aulast%3DOverington%26aufirst%3DJ.%2BP%26aulast%3DOwen%26aufirst%3DD.%2BR%26aulast%3DRosenberg%26aufirst%3DS.%2BH%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA%26aulast%3DWillson%26aufirst%3DT.%2BM%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN%26aulast%3DZuercher%26aufirst%3DW.%2BJ%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgahn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schott, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranauskiene, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">9</span>), <span class="NLM_elocation-id">e88</span> <span class="refDoi"> DOI: 10.1093/nar/gkw089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1093%2Fnar%2Fgkw089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26893353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&issue=9&author=T.+Wagnerauthor=H.+Greschikauthor=T.+Burgahnauthor=K.+Schmidtkunzauthor=A.+K.+Schottauthor=J.+McMillanauthor=L.+Baranauskieneauthor=Y.+Xiongauthor=O.+Fedorovauthor=J.+Jinauthor=U.+Oppermannauthor=D.+Matulisauthor=R.+Schuleauthor=M.+Jung&title=Identification+of+a+small-molecule+ligand+of+the+epigenetic+reader+protein+Spindlin1+via+a+versatile+screening+platform&doi=10.1093%2Fnar%2Fgkw089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span></div><div class="casAuthors">Wagner, Tobias; Greschik, Holger; Burgahn, Teresa; Schmidtkunz, Karin; Schott, Anne-Kathrin; McMillan, Joel; Baranauskiene, Lina; Xiong, Yan; Fedorov, Oleg; Jin, Jian; Oppermann, Udo; Matulis, Daumantas; Schuele, Roland; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e88/1-e88/22</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Epigenetic modifications of histone tails play an essential role in the regulation of eukaryotic transcription.  Writer and eraser enzymes establish and maintain the epigenetic code by creating or removing posttranslational marks.  Specific binding proteins, called readers, recognize the modifications and mediate epigenetic signalling.  Here, we present a versatile assay platform for the investigation of the interaction between Me lysine readers and their ligands.  This can be utilized for the screening of small mol. inhibitors of such protein-protein interactions and the detailed characterization of the inhibition.  Our platform is constructed in a modular way consisting of orthogonal in vitro binding assays for ligand screening and verification of initial hits and biophys., label-free techniques for further kinetic characterization of confirmed ligands.  A stability assay for the investigation of target engagement in a cellular context complements the platform.  We applied the complete evaluation chain to the Tudor domain contg. protein Spindlin1 and established the in vitro test systems for the double Tudor domain of the histone demethylase JMJD2C.  We finally conducted an exploratory screen for inhibitors of the interaction between Spindlin1 and H3K4me3 and identified A366 as the first nanomolar small-mol. ligand of a Tudor domain contg. Me lysine reader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql-bO41bi49bVg90H21EOLACvtfcHk0lj7QB2mYSoBtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK&md5=3c6536be1ee6f83c5d2849baa02da318</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw089%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DBurgahn%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DSchott%26aufirst%3DA.%2BK.%26aulast%3DMcMillan%26aufirst%3DJ.%26aulast%3DBaranauskiene%26aufirst%3DL.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMatulis%26aufirst%3DD.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520small-molecule%2520ligand%2520of%2520the%2520epigenetic%2520reader%2520protein%2520Spindlin1%2520via%2520a%2520versatile%2520screening%2520platform%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26issue%3D9%26doi%3D10.1093%2Fnar%2Fgkw089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET-a novel BRET platform for the analysis of protein-protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&issue=8&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+Mendezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET-a+novel+BRET+platform+for+the+analysis+of+protein-protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0lgCRw8-99Wakg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DMendez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET-a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein-protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&issue=9&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0lgCRw8-99Wakg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26issue%3D9%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xun, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of small-molecule antagonists of the H3K9me3 binding to UHRF1 tandem tudor domain</span>. <i>Slas Discov</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">930</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1177/2472555218766278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1177%2F2472555218766278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29562800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOgsr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=930-940&issue=9&author=G.+Senisterraauthor=H.+Y.+Zhuauthor=X.+Luoauthor=H.+L.+Zhangauthor=G.+L.+Xunauthor=C.+L.+Luauthor=W.+Xiaoauthor=T.+Hajianauthor=P.+Loppnauauthor=I.+Chauauthor=F.+L.+Liauthor=A.+Allali-Hassaniauthor=P.+Atadjaauthor=C.+Oyangauthor=E.+Liauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=K.+H.+Zhaoauthor=Z.+T.+Yuauthor=M.+Vedadi&title=Discovery+of+small-molecule+antagonists+of+the+H3K9me3+binding+to+UHRF1+tandem+tudor+domain&doi=10.1177%2F2472555218766278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small-Molecule Antagonists of the H3K9me3 Binding to UHRF1 Tandem Tudor Domain</span></div><div class="casAuthors">Senisterra, Guillermo; Zhu, Hugh Y.; Luo, Xiao; Zhang, Hailong; Xun, Guoliang; Lu, Chunliang; Xiao, Wen; Hajian, Taraneh; Loppnau, Peter; Chau, Irene; Li, Fengling; Allali-Hassani, Abdellah; Atadja, Peter; Oyang, Counde; Li, En; Brown, Peter J.; Arrowsmith, Cheryl H.; Zhao, Kehao; Yu, Zhengtian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">930-940</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Ubiquitin-like with PHD and RING finger domains 1 (UHRF1) is a multidomain protein that plays a crit. role in maintaining DNA methylation patterns through concurrent recognition of hemimethylated DNA and histone marks by various domains, and recruitment of DNA methyltransferase 1 (DNMT1).  UHRF1 is overexpressed in various cancers, including breast cancer.  The tandem tudor domain (TTD) of UHRF1 specifically and tightly binds to histone H3 di- or trimethylated at lysine 9 (H3K9me2 or H3K9me3, resp.), and this binding is essential for UHRF1 function.  We developed an H3K9me3 peptide displacement assay, which was used to screen a library of 44,000 compds. for small mols. that disrupt the UHRF1-H3K9me3 interaction.  This screen resulted in the identification of NV01, which bound to UHRF1-TTD with a Kd value of 5 μM.  The structure of UHRF1-TTD in complex with NV01 confirmed binding to the H3K9me3-binding pocket.  Limited structure-based optimization of NV01 led to the discovery of NV03 (Kd of 2.4 μM).  These well-characterized small-mol. antagonists of the UHRF1-H3K9me2/3 interaction could be valuable starting chem. matter for developing more potent and cell-active probes toward further characterizing UHRF1 function, with possible applications as anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtaLz2UKwDbLVg90H21EOLACvtfcHk0lgCRw8-99Wakg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOgsr3E&md5=854c27f0152b7a2caebce853e4f8c5c7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1177%2F2472555218766278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218766278%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DH.%2BY.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%2BL.%26aulast%3DXun%26aufirst%3DG.%2BL.%26aulast%3DLu%26aufirst%3DC.%2BL.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DOyang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DZhao%26aufirst%3DK.%2BH.%26aulast%3DYu%26aufirst%3DZ.%2BT.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520small-molecule%2520antagonists%2520of%2520the%2520H3K9me3%2520binding%2520to%2520UHRF1%2520tandem%2520tudor%2520domain%26jtitle%3DSlas%2520Discov%26date%3D2018%26volume%3D23%26issue%3D9%26spage%3D930%26epage%3D940%26doi%3D10.1177%2F2472555218766278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Chemoproteomics and chemical probes for target discovery</span>. <i>Trends Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1275</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1016/j.tibtech.2018.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.tibtech.2018.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30017093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ygurbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=1275-1286&issue=12&author=G.+Drewesauthor=S.+Knapp&title=Chemoproteomics+and+chemical+probes+for+target+discovery&doi=10.1016%2Fj.tibtech.2018.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoproteomics and Chemical Probes for Target Discovery</span></div><div class="casAuthors">Drewes, Gerard; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biotechnology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1275-1286</span>CODEN:
                <span class="NLM_cas:coden">TRBIDM</span>;
        ISSN:<span class="NLM_cas:issn">0167-7799</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Chem. probes represent versatile tools to validate disease-modifying targets.  However, evaluating the selectivity of chem. probes in complex cellular systems is a major challenge that needs to be addressed to better understand the mode of action of small mols. and the interpretation of their pharmacol. effects.  Chemoproteomics has emerged as a key technol. to characterize the mode of action of pharmacol. modulators such as chem. probes and drugs, and these studies have unraveled the cellular targets of many bioactive compds.  Here we review the role of chem. probes for the validation of new therapeutic targets and their characterization by proteome wide affinity- and activity-based chem. proteomics and recently developed label-free technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBxPBnOO1FYrVg90H21EOLACvtfcHk0lj_xcjq-dBrnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ygurbO&md5=905c056dd30df4438c433e508919f1c6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tibtech.2018.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibtech.2018.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DChemoproteomics%2520and%2520chemical%2520probes%2520for%2520target%2520discovery%26jtitle%3DTrends%2520Biotechnol.%26date%3D2018%26volume%3D36%26issue%3D12%26spage%3D1275%26epage%3D1286%26doi%3D10.1016%2Fj.tibtech.2018.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fik, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strnad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chovanec, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plzak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cada, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hroudova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbanova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paces, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazanek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlcek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabius, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smetana, K.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolar, M.</span></span> <span> </span><span class="NLM_article-title">Smooth muscle actin-expressing stromal fibroblasts in head and neck squamous cell carcinoma: increased expression of galectin-1 and induction of poor prognosis factors</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>131</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2508</span>, <span class="refDoi"> DOI: 10.1002/ijc.27550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fijc.27550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22447203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1ahs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2499-2508&issue=11&author=J.+Valachauthor=Z.+Fikauthor=H.+Strnadauthor=M.+Chovanecauthor=J.+Plzakauthor=Z.+Cadaauthor=P.+Szaboauthor=J.+Sachovaauthor=M.+Hroudovaauthor=M.+Urbanovaauthor=M.+Stefflauthor=J.+Pacesauthor=J.+Mazanekauthor=C.+Vlcekauthor=J.+Betkaauthor=H.+Kaltnerauthor=S.+Andreauthor=H.+J.+Gabiusauthor=R.+Kodetauthor=K.+Smetanaauthor=P.+Galauthor=M.+Kolar&title=Smooth+muscle+actin-expressing+stromal+fibroblasts+in+head+and+neck+squamous+cell+carcinoma%3A+increased+expression+of+galectin-1+and+induction+of+poor+prognosis+factors&doi=10.1002%2Fijc.27550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Smooth muscle actin-expressing stromal fibroblasts in head and neck squamous cell carcinoma: Increased expression of galectin-1 and induction of poor prognosis factors</span></div><div class="casAuthors">Valach, Jaroslav; Fik, Zdenek; Strnad, Hynek; Chovanec, Martin; Plzak, Jan; Cada, Zdenek; Szabo, Pavol; Sachova, Jana; Hroudova, Miluse; Urbanova, Marketa; Steffl, Martin; Paces, Jan; Mazanek, Jiri; Vlcek, Cestmir; Betka, Jan; Kaltner, Herbert; Andre, Sabine; Gabius, Hans-Joachim; Kodet, Roman; Smetana, Karel; Gal, Peter; Kolar, Michal</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2499-2508</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Tumor stroma is an active part influencing the biol. properties of malignancies via mol. cross-talk.  Cancer-assocd. fibroblasts play a significant role in this interaction.  These cells frequently express smooth muscle actin and can be classified as myofibroblasts.  The adhesion/growth-regulatory lectin galectin-1 is an effector for their generation.  In our study, we set the presence of smooth muscle actin-pos. cancer-assocd. fibroblasts in relation to this endogenous lectin and an in vivo competitor (galectin-3).  In squamous cell carcinomas of head and neck, upregulation of galectin-1 presence was highly significantly correlated to presence of smooth muscle actin-pos. cancer-assocd. fibroblasts in the tumor (p = 4 × 10-8).  To pinpoint further correlations on the mol. level, we applied microarray analyses to the transcription profiles of the corresponding tumors.  Significant correlations of several transcripts were detected with the protein level of galectin-1 in the cancer-assocd. fibroblasts.  These activated genes (MAP3K2, TRIM23, PTPLAD1, FUSIP1, SLC25A40 and SPIN1) are related to known squamous-cell-carcinoma poor-prognosis factors, NF-κB upregulation and splicing downregulation.  These results provide new insights into the significance of presence of myofibroblasts in squamous cell carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzu3UUHbdVz7Vg90H21EOLACvtfcHk0lj_xcjq-dBrnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1ahs7o%253D&md5=af28a24b9e7d2849eb55d89fa47e7919</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fijc.27550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27550%26sid%3Dliteratum%253Aachs%26aulast%3DValach%26aufirst%3DJ.%26aulast%3DFik%26aufirst%3DZ.%26aulast%3DStrnad%26aufirst%3DH.%26aulast%3DChovanec%26aufirst%3DM.%26aulast%3DPlzak%26aufirst%3DJ.%26aulast%3DCada%26aufirst%3DZ.%26aulast%3DSzabo%26aufirst%3DP.%26aulast%3DSachova%26aufirst%3DJ.%26aulast%3DHroudova%26aufirst%3DM.%26aulast%3DUrbanova%26aufirst%3DM.%26aulast%3DSteffl%26aufirst%3DM.%26aulast%3DPaces%26aufirst%3DJ.%26aulast%3DMazanek%26aufirst%3DJ.%26aulast%3DVlcek%26aufirst%3DC.%26aulast%3DBetka%26aufirst%3DJ.%26aulast%3DKaltner%26aufirst%3DH.%26aulast%3DAndre%26aufirst%3DS.%26aulast%3DGabius%26aufirst%3DH.%2BJ.%26aulast%3DKodet%26aufirst%3DR.%26aulast%3DSmetana%26aufirst%3DK.%26aulast%3DGal%26aufirst%3DP.%26aulast%3DKolar%26aufirst%3DM.%26atitle%3DSmooth%2520muscle%2520actin-expressing%2520stromal%2520fibroblasts%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%253A%2520increased%2520expression%2520of%2520galectin-1%2520and%2520induction%2520of%2520poor%2520prognosis%2520factors%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26issue%3D11%26spage%3D2499%26epage%3D2508%26doi%3D10.1002%2Fijc.27550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nebert, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span> <span> </span><span class="NLM_article-title">Update of the human and mouse SERPIN gene superfamily</span>. <i>Hum. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">22</span>, <span class="refDoi"> DOI: 10.1186/1479-7364-7-22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2F1479-7364-7-22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24172014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Oruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=22&author=C.+Heitauthor=B.+C.+Jacksonauthor=M.+McAndrewsauthor=M.+W.+Wrightauthor=D.+C.+Thompsonauthor=G.+A.+Silvermanauthor=D.+W.+Nebertauthor=V.+Vasiliou&title=Update+of+the+human+and+mouse+SERPIN+gene+superfamily&doi=10.1186%2F1479-7364-7-22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Update of the human and mouse SERPIN gene superfamily</span></div><div class="casAuthors">Heit, Claire; Jackson, Brian C.; McAndrews, Monica; Wright, Mathew W.; Thompson, David C.; Silverman, Gary A.; Nebert, Daniel W.; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Human Genomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">22/1-22/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">HGUEAT</span>;
        ISSN:<span class="NLM_cas:issn">1479-7364</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The serpin family comprises a structurally similar, yet functionally diverse, set of proteins.  Named originally for their function as serine proteinase inhibitors, many of its members are not inhibitors but rather chaperones, involved in storage, transport, and other roles.  Serpins are found in genomes of all kingdoms, with 36 human protein-coding genes and five pseudogenes.  The mouse has 60 Serpin functional genes, many of which are orthologous to human SERPIN genes and some of which have expanded into multiple paralogous genes.  Serpins are found in tissues throughout the body; whereas most are extracellular, there is a class of intracellular serpins.  Serpins appear to have roles in inflammation, immune function, tumorigenesis, blood clotting, dementia, and cancer metastasis.  Further characterization of these proteins will likely reveal potential biomarkers and therapeutic targets for disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOpQ8_mmIt2rVg90H21EOLACvtfcHk0lj_xcjq-dBrnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Oruro%253D&md5=fe19bf8237e46aaac2a301a1cae209bc</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1479-7364-7-22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-7364-7-22%26sid%3Dliteratum%253Aachs%26aulast%3DHeit%26aufirst%3DC.%26aulast%3DJackson%26aufirst%3DB.%2BC.%26aulast%3DMcAndrews%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DM.%2BW.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DSilverman%26aufirst%3DG.%2BA.%26aulast%3DNebert%26aufirst%3DD.%2BW.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DUpdate%2520of%2520the%2520human%2520and%2520mouse%2520SERPIN%2520gene%2520superfamily%26jtitle%3DHum.%2520Genomics%26date%3D2013%26volume%3D7%26spage%3D22%26doi%3D10.1186%2F1479-7364-7-22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Izuhara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunomura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanri, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aihara, M.</span></span> <span> </span><span class="NLM_article-title">Squamous cell carcinoma antigen 2 (SCCA2, SERPINB4): an emerging biomarker for skin inflammatory diseases</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1102</span>, <span class="refDoi"> DOI: 10.3390/ijms19041102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3390%2Fijms19041102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyjsr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1102&issue=4&author=K.+Izuharaauthor=Y.+Yamaguchiauthor=S.+Ohtaauthor=S.+Nunomuraauthor=Y.+Nanriauthor=Y.+Azumaauthor=N.+Nomuraauthor=Y.+Noguchiauthor=M.+Aihara&title=Squamous+cell+carcinoma+antigen+2+%28SCCA2%2C+SERPINB4%29%3A+an+emerging+biomarker+for+skin+inflammatory+diseases&doi=10.3390%2Fijms19041102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Squamous cell carcinoma antigen 2 (SCCA2, SERPINB4): an emerging biomarker for skin inflammatory diseases</span></div><div class="casAuthors">Izuhara, Kenji; Yamaguchi, Yukie; Ohta, Shoichiro; Nunomura, Satoshi; Nanri, Yasuhiro; Azuma, Yoshinori; Nomura, Noriko; Noguchi, Yasuhiko; Aihara, Michiko</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1102/1-1102/16</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Squamous cell carcinoma antigens 1 and 2 (SCCA1 and 2, SERPIN B3 and B4), members of the ovalbumin serpin (ov-serpin)/clade B serpin family, were originally discovered as tumor-specific antigens and are used as tumor markers for various kinds of squamous cell carcinomas.  Recently, our understanding of the underlying mechanisms of how SCCA1/2 enhance tumor growth has greatly increased.  Moreover, it has been shown that SCCA1/2 are involved in the pathogenesis of several inflammatory diseases: asthma, psoriasis, and atopic dermatitis (AD).  IL-22 and IL-17, signature cytokines of type 17 inflammation, as well as IL-4 and IL-13, signature cytokines of type 2 inflammation, both of which are pos. correlated with the pathogenesis of psoriasis and allergic diseases, resp., can induce expression of SCCA1/2 in airway epithelial cells and/or keratinocytes, leading to high expression of SCCA1/2 in these diseases.  Based on these findings, several trials have been performed to examine the potential of applying SCCA1/2 to biomarkers for these diseases.  The findings show that SCCA2 is useful to aid diagnosis, est. clin. severity and disease type, and assess responses to treatment in psoriasis and AD.  These results suggest that SCCA2 has emerged as a novel biomarker for skin inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZum4NHFRlrVg90H21EOLACvtfcHk0liyDmgIdoLlqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyjsr3J&md5=d4ed23ffd3f3884e70d6290f944f54eb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Fijms19041102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19041102%26sid%3Dliteratum%253Aachs%26aulast%3DIzuhara%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DS.%26aulast%3DNunomura%26aufirst%3DS.%26aulast%3DNanri%26aufirst%3DY.%26aulast%3DAzuma%26aufirst%3DY.%26aulast%3DNomura%26aufirst%3DN.%26aulast%3DNoguchi%26aufirst%3DY.%26aulast%3DAihara%26aufirst%3DM.%26atitle%3DSquamous%2520cell%2520carcinoma%2520antigen%25202%2520%2528SCCA2%252C%2520SERPINB4%2529%253A%2520an%2520emerging%2520biomarker%2520for%2520skin%2520inflammatory%2520diseases%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26issue%3D4%26spage%3D1102%26doi%3D10.3390%2Fijms19041102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugihara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatsuki, K.</span></span> <span> </span><span class="NLM_article-title">Expression of serine protease inhibitors in epidermal keratinocytes is increased by calcium but not 1,25-dihydroxyvitamin D-3 or retinoic acid</span>. <i>Br. J. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>176</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1532</span>, <span class="refDoi"> DOI: 10.1111/bjd.15153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1111%2Fbjd.15153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27797397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsVKksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2017&pages=1525-1532&issue=6&author=M.+Kobashiauthor=S.+Morizaneauthor=S.+Sugimotoauthor=S.+Sugiharaauthor=K.+Iwatsuki&title=Expression+of+serine+protease+inhibitors+in+epidermal+keratinocytes+is+increased+by+calcium+but+not+1%2C25-dihydroxyvitamin+D-3+or+retinoic+acid&doi=10.1111%2Fbjd.15153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of serine protease inhibitors in epidermal keratinocytes is increased by calcium but not 1,25-dihydroxyvitamin D3 or retinoic acid</span></div><div class="casAuthors">Kobashi, M.; Morizane, S.; Sugimoto, S.; Sugihara, S.; Iwatsuki, K.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Dermatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1525-1532</span>CODEN:
                <span class="NLM_cas:coden">BJDEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0007-0963</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Background : In human skin, the serine proteases kallikrein-related peptidase (KLK)5 and KLK7 degrade corneodesmosome proteins, leading to desquamation.  Serine protease activity of the skin is tightly regulated by the interplay between such proteases and serine protease inhibitors, including lymphoepithelial Kazal-type related inhibitor (LEKTI), encoded by SPINK5; secretory leukocyte peptidase inhibitor (SLPI); and elafin.  Expression of KLK5 and KLK7 is controlled and upregulated by stimulants such as calcium, 1,25-dihydroxyvitamin D3 [1,25(OH)2VD3] and retinoic acid (RA).  Objectives : To understand the effect of calcium, 1,25(OH)2VD3 and RA on the expression of serine protease inhibitors in epidermal keratinocytes.  Methods : We stimulated normal human epidermal keratinocytes (NHEKs) with high calcium, 1,25(OH)2VD3 or RA, and then analyzed the expression of serine protease inhibitors using quant. real-time polymerase chain reaction, ELISA and immunocytofluorescence.  We also analyzed trypsin- and chymotrypsin-like serine protease activities in stimulated NHEKs.  Results : High calcium, but not 1,25(OH)2VD3 or RA, significantly induced the expression of LEKTI, SLPI and elafin at both transcript and protein levels in NHEKs.  These inductions were time- and dose-dependent.  The activities of trypsin- and chymotrypsin-like serine proteases were significantly up- and downregulated by high calcium, resp., in NHEKs.  Conclusions : High calcium, but not 1,25(OH)2VD3 or RA, increases the expression of serine protease inhibitors in epidermal keratinocytes.  Our findings contribute to the understanding of the mechanisms by which serine protease activities are regulated by serine proteases and related inhibitors in epidermal keratinocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonLwAAqBLJFrVg90H21EOLACvtfcHk0liyDmgIdoLlqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsVKksLY%253D&md5=681c599c77587b7b97699f38caa251dd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Fbjd.15153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjd.15153%26sid%3Dliteratum%253Aachs%26aulast%3DKobashi%26aufirst%3DM.%26aulast%3DMorizane%26aufirst%3DS.%26aulast%3DSugimoto%26aufirst%3DS.%26aulast%3DSugihara%26aufirst%3DS.%26aulast%3DIwatsuki%26aufirst%3DK.%26atitle%3DExpression%2520of%2520serine%2520protease%2520inhibitors%2520in%2520epidermal%2520keratinocytes%2520is%2520increased%2520by%2520calcium%2520but%2520not%25201%252C25-dihydroxyvitamin%2520D-3%2520or%2520retinoic%2520acid%26jtitle%3DBr.%2520J.%2520Dermatol.%26date%3D2017%26volume%3D176%26issue%3D6%26spage%3D1525%26epage%3D1532%26doi%3D10.1111%2Fbjd.15153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprasad, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinker, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericksen, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hershey, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khurana Hershey, G. K.</span></span> <span> </span><span class="NLM_article-title">SERPINB3/B4 contributes to early inflammation and barrier dysfunction in an experimental murine model of atopic dermatitis</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>135</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1038/jid.2014.353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fjid.2014.353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25111616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2015&pages=160-169&issue=1&author=U.+Sivaprasadauthor=K.+G.+Kinkerauthor=M.+B.+Ericksenauthor=M.+Lindseyauthor=A.+M.+Gibsonauthor=S.+A.+Bassauthor=N.+S.+Hersheyauthor=J.+Y.+Dengauthor=M.+Medvedovicauthor=G.+K.+Khurana+Hershey&title=SERPINB3%2FB4+contributes+to+early+inflammation+and+barrier+dysfunction+in+an+experimental+murine+model+of+atopic+dermatitis&doi=10.1038%2Fjid.2014.353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">SERPINB3/B4 Contributes to Early Inflammation and Barrier Dysfunction in an Experimental Murine Model of Atopic Dermatitis</span></div><div class="casAuthors">Sivaprasad, Umasundari; Kinker, Kayla G.; Ericksen, Mark B.; Lindsey, Mark; Gibson, Aaron M.; Bass, Stacey A.; Hershey, Nicolas S.; Deng, Jingyuan; Medvedovic, Mario; Khurana Hershey, Gurjit K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-169</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Serine proteases are crit. for epidermal barrier homeostasis, and their aberrant expression and/or activity is assocd. with chronic skin diseases.  Elevated levels of the serine protease inhibitors SERPINB3 and SERPINB4 are seen in patients with atopic dermatitis and psoriasis.  However, their mechanistic role in the skin is unknown.  To evaluate the contribution of Serpinb3a (mouse homolog of SERPINB3 and SERPINB4) in atopic dermatitis, we examd. the effect of topical Aspergillus fumigatus ext. exposure in wild-type and Serpinb3a-null mice on transepidermal water loss (TEWL), sensitization, and inflammation.  Allergen exposure induced Serpinb3a expression in the skin, along with increased TEWL, epidermal thickness, and skin inflammation, all of which were attenuated in the absence of Serpinb3a.  Attenuated TEWL correlated with decreased expression of the pro-inflammatory marker S100A8.  Silencing of SERPINB3/B4 in human keratinocytes decreased S100A8 expression, supporting a role for SERPINB3/B4 in the initiation of the acute inflammatory response.  RNA-seq anal. following allergen exposure identified a network of pro-inflammatory genes induced in wild-type mice that was absent in Serpinb3a-null mice.  In conclusion, Serpinb3a deficiency attenuates barrier dysfunction and the early inflammatory response following cutaneous allergen exposure, supporting a role for Serpinb3a (mice) and SERPINB3/B4 (humans) early in atopic dermatitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs1Alg3gb_0LVg90H21EOLACvtfcHk0liyDmgIdoLlqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrzN&md5=9221221bbccc1692ba5c8f6b2b1cda0f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fjid.2014.353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2014.353%26sid%3Dliteratum%253Aachs%26aulast%3DSivaprasad%26aufirst%3DU.%26aulast%3DKinker%26aufirst%3DK.%2BG.%26aulast%3DEricksen%26aufirst%3DM.%2BB.%26aulast%3DLindsey%26aufirst%3DM.%26aulast%3DGibson%26aufirst%3DA.%2BM.%26aulast%3DBass%26aufirst%3DS.%2BA.%26aulast%3DHershey%26aufirst%3DN.%2BS.%26aulast%3DDeng%26aufirst%3DJ.%2BY.%26aulast%3DMedvedovic%26aufirst%3DM.%26aulast%3DKhurana%2BHershey%26aufirst%3DG.%2BK.%26atitle%3DSERPINB3%252FB4%2520contributes%2520to%2520early%2520inflammation%2520and%2520barrier%2520dysfunction%2520in%2520an%2520experimental%2520murine%2520model%2520of%2520atopic%2520dermatitis%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2015%26volume%3D135%26issue%3D1%26spage%3D160%26epage%3D169%26doi%3D10.1038%2Fjid.2014.353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheshadri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, W. X.</span></span> <span> </span><span class="NLM_article-title">SERPINB3 and B4: From biochemistry to biology</span>. <i>Semin. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">170</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1016/j.semcdb.2016.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.semcdb.2016.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27637160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFertrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2017&pages=170-177&author=Y.+Sunauthor=N.+Sheshadriauthor=W.+X.+Zong&title=SERPINB3+and+B4%3A+From+biochemistry+to+biology&doi=10.1016%2Fj.semcdb.2016.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">SERPINB3 and B4: From biochemistry to biology</span></div><div class="casAuthors">Sun, Yu; Sheshadri, Namratha; Zong, Wei-Xing</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cell & Developmental Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">170-177</span>CODEN:
                <span class="NLM_cas:coden">SCDBFX</span>;
        ISSN:<span class="NLM_cas:issn">1084-9521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human SERPINB3 and SERPINB4 are evolutionary duplicated serine/cysteine protease inhibitors.  Genomic anal. indicates that these paralogous genes were encoded from independent loci arising from tandem gene duplication.  Although the two mols. share 92% identity of their amino acid sequences, they are distinct in the Reactive Center Loop (RCL) including a hinge region and catalytic sequences which accounts for altered substrate specificity.  Elevated expression of the two mols. has been reported to contribute to numerous pathol. conditions such as inflammatory diseases and cancer.  In this review, we focus on summarizing the biochem. features of SERPINB3/B4 and discussing the mechanistic basis for their biol. functions and implications in human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbrWqLdYFTOLVg90H21EOLACvtfcHk0livY-4OgWaRrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFertrfK&md5=3da7b9e7a3f737c9212949b17d4ad48f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.semcdb.2016.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcdb.2016.09.005%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DSheshadri%26aufirst%3DN.%26aulast%3DZong%26aufirst%3DW.%2BX.%26atitle%3DSERPINB3%2520and%2520B4%253A%2520From%2520biochemistry%2520to%2520biology%26jtitle%3DSemin.%2520Cell%2520Dev.%2520Biol.%26date%3D2017%26volume%3D62%26spage%3D170%26epage%3D177%26doi%3D10.1016%2Fj.semcdb.2016.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikitakis, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaisuparat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwell-Wild, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gliozzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moutsopoulos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warburton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahl, S. M.</span></span> <span> </span><span class="NLM_article-title">Secretory leukocyte protease inhibitor (SLPI) expression and tumor invasion in oral squamous cell carcinoma</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>178</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2866</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.ajpath.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21641406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslOntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2011&pages=2866-2878&issue=6&author=J.+Wenauthor=N.+G.+Nikitakisauthor=R.+Chaisuparatauthor=T.+Greenwell-Wildauthor=M.+Gliozziauthor=W.+W.+Jinauthor=A.+Adliauthor=N.+Moutsopoulosauthor=T.+X.+Wuauthor=G.+Warburtonauthor=S.+M.+Wahl&title=Secretory+leukocyte+protease+inhibitor+%28SLPI%29+expression+and+tumor+invasion+in+oral+squamous+cell+carcinoma&doi=10.1016%2Fj.ajpath.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Secretory leukocyte protease inhibitor (SLPI) expression and tumor invasion in oral squamous cell carcinoma</span></div><div class="casAuthors">Wen, Jie; Nikitakis, Nikolaos G.; Chaisuparat, Risa; Greenwell-Wild, Teresa; Gliozzi, Maria; Jin, Wenwen; Adli, Azita; Moutsopoulos, Niki; Wu, Tanxia; Warburton, Gary; Wahl, Sharon M.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2866-2878</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Differential expression of secretory leukocyte protease inhibitor (SLPI) impacts on tumor progression.  SLPI directly inhibits elastase and other serine proteases, and regulates matrix metalloproteinases, plasminogen activation, and plasmin downstream targets to influence invasion.  We examd. tissues from human oral squamous cell carcinoma (OSCC) for SLPI expression in parallel with proteases assocd. with tumor progression and evaluated their relationships using tumor cell lines.  Significantly decreased SLPI was detected in OSCC compared to normal oral epithelium.  Furthermore, an inverse correlation between SLPI and histol. parameters assocd. with tumor progression, including stage of invasion, pattern of invasion, invasive cell grade, and composite histol. tumor score was evident.  Conversely, elevated plasmin and elastase were pos. correlated with histol. parameters of tumor invasion.  In addn. to its known inhibition of elastase, we identify SLPI as a novel inhibitor of plasminogen activation through its interaction with annexin A2 with concomitant reduced plasmin generation by macrophages and OSCC cell lines.  In an in vitro assay measuring invasive activity, SLPI blocked protease-dependent tumor cell migration.  Our data suggest that SLPI may possess antitumorigenic activity by virtue of its ability to interfere with multiple requisite proteolytic steps underlying tumor cell invasion and may provide insight into potential stratification of oral cancer according to risk of occult metastasis, guiding treatment strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzXnusWEPiBLVg90H21EOLACvtfcHk0livY-4OgWaRrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslOntbg%253D&md5=e14f347afb856cc4d311dbf5ad037800</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DJ.%26aulast%3DNikitakis%26aufirst%3DN.%2BG.%26aulast%3DChaisuparat%26aufirst%3DR.%26aulast%3DGreenwell-Wild%26aufirst%3DT.%26aulast%3DGliozzi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DW.%2BW.%26aulast%3DAdli%26aufirst%3DA.%26aulast%3DMoutsopoulos%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DT.%2BX.%26aulast%3DWarburton%26aufirst%3DG.%26aulast%3DWahl%26aufirst%3DS.%2BM.%26atitle%3DSecretory%2520leukocyte%2520protease%2520inhibitor%2520%2528SLPI%2529%2520expression%2520and%2520tumor%2520invasion%2520in%2520oral%2520squamous%2520cell%2520carcinoma%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2011%26volume%3D178%26issue%3D6%26spage%3D2866%26epage%3D2878%26doi%3D10.1016%2Fj.ajpath.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kheradmand, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, P. A.</span></span> <span> </span><span class="NLM_article-title">Epidermal detachment, desmosomal dissociation, and destabilization of corneodesmosin in Spink5(−/−) mice</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">2354</span>– <span class="NLM_lpage">2358</span>, <span class="refDoi"> DOI: 10.1101/gad.1232104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1101%2Fgad.1232104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=15466487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD2cXot1yrtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=2354-2358&issue=19&author=T.+Yangauthor=D.+C.+Liangauthor=P.+J.+Kochauthor=D.+Hohlauthor=F.+Kheradmandauthor=P.+A.+Overbeek&title=Epidermal+detachment%2C+desmosomal+dissociation%2C+and+destabilization+of+corneodesmosin+in+Spink5%28%E2%88%92%2F%E2%88%92%29+mice&doi=10.1101%2Fgad.1232104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal detachment, desmosomal dissociation, and destabilization of corneodesmosin in Spink5-/- mice</span></div><div class="casAuthors">Yang, Tao; Liang, Dongcai; Koch, Peter J.; Hohl, Daniel; Kheradmand, Farrah; Overbeek, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2354-2358</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Netherton syndrome (NS) is a human autosomal recessive skin disease caused by mutations in the SPINK5 gene, which encodes the putative proteinase inhibitor LEKTI.  The authors have generated a transgenic mouse line with an insertional mutation that inactivated the mouse SPINK5 ortholog.  Mutant mice exhibit fragile stratum corneum and perinatal death due to dehydration.  The anal. suggests that the phenotype is a consequence of desmosomal fragility assocd. with premature proteolysis of corneodesmosin, an extracellular desmosomal component.  The mouse mutant provides a model system for mol. studies of desmosomal stability and keratinocyte adhesion, and for designing therapeutic strategies to treat NS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVxzkcr8U6k7Vg90H21EOLACvtfcHk0livY-4OgWaRrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXot1yrtb4%253D&md5=626f06c4940884e4314ded50b57ed3fc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1101%2Fgad.1232104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1232104%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DT.%26aulast%3DLiang%26aufirst%3DD.%2BC.%26aulast%3DKoch%26aufirst%3DP.%2BJ.%26aulast%3DHohl%26aufirst%3DD.%26aulast%3DKheradmand%26aufirst%3DF.%26aulast%3DOverbeek%26aufirst%3DP.%2BA.%26atitle%3DEpidermal%2520detachment%252C%2520desmosomal%2520dissociation%252C%2520and%2520destabilization%2520of%2520corneodesmosin%2520in%2520Spink5%2528%25E2%2588%2592%252F%25E2%2588%2592%2529%2520mice%26jtitle%3DGenes%2520Dev.%26date%3D2004%26volume%3D18%26issue%3D19%26spage%3D2354%26epage%3D2358%26doi%3D10.1101%2Fgad.1232104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Catanzaro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheshadri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, W. X.</span></span> <span> </span><span class="NLM_article-title">SerpinB3/B4: mediators of Ras-driven inflammation and oncogenesis</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">3155</span>– <span class="NLM_lpage">3156</span>, <span class="refDoi"> DOI: 10.4161/15384101.2014.969991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.4161%2F15384101.2014.969991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25485489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFalsLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=3155-3156&issue=20&author=J.+M.+Catanzaroauthor=N.+Sheshadriauthor=W.+X.+Zong&title=SerpinB3%2FB4%3A+mediators+of+Ras-driven+inflammation+and+oncogenesis&doi=10.4161%2F15384101.2014.969991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">SerpinB3/B4: mediators of Ras-driven inflammation and oncogenesis</span></div><div class="casAuthors">Catanzaro, Joseph M.; Sheshadri, Namratha; Zong, Wei-Xing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3155-3156</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">In this review serpinB3/B4 mediators of Ras-driven inflammation and oncogenesis were discussed.  This article discuss on squamous cell carcinoma antigens (SCCA) SerpinB3/SerpinB4 as novel pro- tumorigenic mediators downstream of mutant Ras through the activation of NF-κB and pro-inflammatory cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjqsv2I6tPsLVg90H21EOLACvtfcHk0linUpR5ozqSrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFalsLjK&md5=469d0eaecb708451a2de2f7514e80642</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4161%2F15384101.2014.969991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384101.2014.969991%26sid%3Dliteratum%253Aachs%26aulast%3DCatanzaro%26aufirst%3DJ.%2BM.%26aulast%3DSheshadri%26aufirst%3DN.%26aulast%3DZong%26aufirst%3DW.%2BX.%26atitle%3DSerpinB3%252FB4%253A%2520mediators%2520of%2520Ras-driven%2520inflammation%2520and%2520oncogenesis%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26issue%3D20%26spage%3D3155%26epage%3D3156%26doi%3D10.4161%2F15384101.2014.969991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pietzsch, J.</span></span> <span> </span><span class="NLM_article-title">S100 proteins in health and disease</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">760</span>, <span class="refDoi"> DOI: 10.1007/s00726-010-0816-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2Fs00726-010-0816-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21120552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=755-760&issue=4&author=J.+Pietzsch&title=S100+proteins+in+health+and+disease&doi=10.1007%2Fs00726-010-0816-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">S100 proteins in health and disease</span></div><div class="casAuthors">Pietzsch, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">755-760</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">SpringerWienNewYork</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Qc8wWbjUzbVg90H21EOLACvtfcHk0linUpR5ozqSrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKitLfJ&md5=e36635b5c6441a05bbad2595a3aa7868</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs00726-010-0816-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-010-0816-8%26sid%3Dliteratum%253Aachs%26aulast%3DPietzsch%26aufirst%3DJ.%26atitle%3DS100%2520proteins%2520in%2520health%2520and%2520disease%26jtitle%3DAmino%2520Acids%26date%3D2011%26volume%3D41%26issue%3D4%26spage%3D755%26epage%3D760%26doi%3D10.1007%2Fs00726-010-0816-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schonthaler, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinea-Viniegra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wculek, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruppen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ximenez-Embun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guio-Carrion, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E. F.</span></span> <span> </span><span class="NLM_article-title">S100A8-S100A9 protein complex mediates psoriasis by regulating the expression of complement factor C3</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1171</span>– <span class="NLM_lpage">1181</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2013.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.immuni.2013.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24332034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhs7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2013&pages=1171-1181&issue=6&author=H.+B.+Schonthalerauthor=J.+Guinea-Viniegraauthor=S.+K.+Wculekauthor=I.+Ruppenauthor=P.+Ximenez-Embunauthor=A.+Guio-Carrionauthor=R.+Navarroauthor=N.+Hoggauthor=K.+Ashmanauthor=E.+F.+Wagner&title=S100A8-S100A9+protein+complex+mediates+psoriasis+by+regulating+the+expression+of+complement+factor+C3&doi=10.1016%2Fj.immuni.2013.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">S100A8-S100A9 Protein Complex Mediates Psoriasis by Regulating the Expression of Complement Factor C3</span></div><div class="casAuthors">Schonthaler, Helia B.; Guinea-Viniegra, Juan; Wculek, Stefanie K.; Ruppen, Isabel; Ximenez-Embun, Pilar; Guio-Carrion, Ana; Navarro, Raquel; Hogg, Nancy; Ashman, Keith; Wagner, Erwin F.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1171-1181</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Psoriasis is a common heterogeneous inflammatory skin disease with a complex pathophysiol. and limited treatment options.  Here we performed proteomic analyses of human psoriatic epidermis and found S100A8-S100A9, also called calprotectin, as the most upregulated proteins, followed by the complement component C3.  Both S100A8-S100A9 and C3 are specifically expressed in lesional psoriatic skin.  S100A9 is shown here to function as a chromatin component modulating C3 expression in mouse and human cells by binding to a region upstream of the C3 start site.  When S100A9 was genetically deleted in mouse models of skin inflammation, the psoriasis-like skin disease and inflammation were strongly attenuated, with a mild immune infiltrate and decreased amts. of C3.  In addn., inhibition of C3 in the mouse model strongly reduced the inflammatory skin disease.  Thus, S100A8-S100A9 can regulate C3 at the nuclear level and present potential new therapeutic targets for psoriasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNqfBzsz7G2bVg90H21EOLACvtfcHk0linUpR5ozqSrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhs7rM&md5=a3c732f52dd9c97385da241567b0a4f8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2013.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2013.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DSchonthaler%26aufirst%3DH.%2BB.%26aulast%3DGuinea-Viniegra%26aufirst%3DJ.%26aulast%3DWculek%26aufirst%3DS.%2BK.%26aulast%3DRuppen%26aufirst%3DI.%26aulast%3DXimenez-Embun%26aufirst%3DP.%26aulast%3DGuio-Carrion%26aufirst%3DA.%26aulast%3DNavarro%26aufirst%3DR.%26aulast%3DHogg%26aufirst%3DN.%26aulast%3DAshman%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DE.%2BF.%26atitle%3DS100A8-S100A9%2520protein%2520complex%2520mediates%2520psoriasis%2520by%2520regulating%2520the%2520expression%2520of%2520complement%2520factor%2520C3%26jtitle%3DImmunity%26date%3D2013%26volume%3D39%26issue%3D6%26spage%3D1171%26epage%3D1181%26doi%3D10.1016%2Fj.immuni.2013.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braunstein, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toomey, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, X. Q.</span></span> <span> </span><span class="NLM_article-title">S100 proteins as an important regulator of macrophage inflammation</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1908</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2017.01908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3389%2Ffimmu.2017.01908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29379499" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1908&author=C.+Xiaauthor=Z.+Braunsteinauthor=A.+C.+Toomeyauthor=J.+X.+Zhongauthor=X.+Q.+Rao&title=S100+proteins+as+an+important+regulator+of+macrophage+inflammation&doi=10.3389%2Ffimmu.2017.01908"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2017.01908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2017.01908%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DC.%26aulast%3DBraunstein%26aufirst%3DZ.%26aulast%3DToomey%26aufirst%3DA.%2BC.%26aulast%3DZhong%26aufirst%3DJ.%2BX.%26aulast%3DRao%26aufirst%3DX.%2BQ.%26atitle%3DS100%2520proteins%2520as%2520an%2520important%2520regulator%2520of%2520macrophage%2520inflammation%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D8%26spage%3D1908%26doi%3D10.3389%2Ffimmu.2017.01908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diakos, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charles, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. J.</span></span> <span> </span><span class="NLM_article-title">Cancer-related inflammation and treatment effectiveness</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">e493</span>– <span class="NLM_lpage">e503</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(14)70263-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2FS1470-2045%2814%2970263-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25281468" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=e493-e503&issue=11&author=C.+I.+Diakosauthor=K.+A.+Charlesauthor=D.+C.+McMillanauthor=S.+J.+Clarke&title=Cancer-related+inflammation+and+treatment+effectiveness&doi=10.1016%2FS1470-2045%2814%2970263-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2814%2970263-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252814%252970263-3%26sid%3Dliteratum%253Aachs%26aulast%3DDiakos%26aufirst%3DC.%2BI.%26aulast%3DCharles%26aufirst%3DK.%2BA.%26aulast%3DMcMillan%26aufirst%3DD.%2BC.%26aulast%3DClarke%26aufirst%3DS.%2BJ.%26atitle%3DCancer-related%2520inflammation%2520and%2520treatment%2520effectiveness%26jtitle%3DLancet%2520Oncol.%26date%3D2014%26volume%3D15%26issue%3D11%26spage%3De493%26epage%3De503%26doi%3D10.1016%2FS1470-2045%2814%2970263-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shabani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farasat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahdavi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheibi, N.</span></span> <span> </span><span class="NLM_article-title">Calprotectin (S100A8/S100A9): a key protein between inflammation and cancer</span>. <i>Inflammation Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1007/s00011-018-1173-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2Fs00011-018-1173-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30083975" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGju7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=801-812&issue=10&author=F.+Shabaniauthor=A.+Farasatauthor=M.+Mahdaviauthor=N.+Gheibi&title=Calprotectin+%28S100A8%2FS100A9%29%3A+a+key+protein+between+inflammation+and+cancer&doi=10.1007%2Fs00011-018-1173-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Calprotectin (S100A8/S100A9): a key protein between inflammation and cancer</span></div><div class="casAuthors">Shabani, Fatemeh; Farasat, Alireza; Mahdavi, Majid; Gheibi, Nematollah</div><div class="citationInfo"><span class="NLM_cas:title">Inflammation Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">801-812</span>CODEN:
                <span class="NLM_cas:coden">INREFB</span>;
        ISSN:<span class="NLM_cas:issn">1023-3830</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Background: Calprotectin (S100A8/S100A9), a heterodimeric EF-hand Ca2+ binding protein, are abundant in cytosol of neutrophils and are involved in inflammatory processes and several cancerous pathogens.  Objective: The purpose of the present systematic review is to evaluate the pro- and anti-tumorigenic functions of calprotectin and its relation to inflammation.  Materials and methods: We conducted a review of studies published in the Medline (1966-2018), Scopus (2004-2018), ClinicalTrials.gov (2008-2018) and Google Scholar (2004-2018) databases, combined with studies found in the ref. lists of the included studies.  Results: Elevated levels of S100A8/S100A9 were detected in inflammation, neoplastic tumor cells and various human cancers.  Recent data have explained that many cancers arise from sites of infection, chronic irritation, and inflammation.  The inflammatory microenvironment which largely includes calprotectin, has an essential role on high producing of inflammatory factors and then on neoplastic process and metastasis.  Conclusion: Scientists have shown different outcomes in inflammation, malignancy and apoptosis whether the source of the aforementioned protein is extracellular or intracellular.  These findings are offering new insights that anti-inflammatory therapeutic agents and anti-tumorigenic functions of calprotectin can lead to control cancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9oH42g1_GAbVg90H21EOLACvtfcHk0liYcPHAs-kFLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGju7bL&md5=5c5dffea746c874e5f8bafaba772939c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs00011-018-1173-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00011-018-1173-4%26sid%3Dliteratum%253Aachs%26aulast%3DShabani%26aufirst%3DF.%26aulast%3DFarasat%26aufirst%3DA.%26aulast%3DMahdavi%26aufirst%3DM.%26aulast%3DGheibi%26aufirst%3DN.%26atitle%3DCalprotectin%2520%2528S100A8%252FS100A9%2529%253A%2520a%2520key%2520protein%2520between%2520inflammation%2520and%2520cancer%26jtitle%3DInflammation%2520Res.%26date%3D2018%26volume%3D67%26issue%3D10%26spage%3D801%26epage%3D812%26doi%3D10.1007%2Fs00011-018-1173-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Netea, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Veerdonk, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinarello, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joosten, L. A. B.</span></span> <span> </span><span class="NLM_article-title">Inflammasome-independent regulation of IL-1-family cytokines</span>. <i>Annu. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1146/annurev-immunol-032414-112306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1146%2Fannurev-immunol-032414-112306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25493334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVSisLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=49-77&author=M.+G.+Neteaauthor=F.+L.+van+de+Veerdonkauthor=J.+W.+M.+van+der+Meerauthor=C.+A.+Dinarelloauthor=L.+A.+B.+Joosten&title=Inflammasome-independent+regulation+of+IL-1-family+cytokines&doi=10.1146%2Fannurev-immunol-032414-112306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammasome-Independent Regulation of IL-1-Family Cytokines</span></div><div class="casAuthors">Netea, Mihai G.; van de Veerdonk, Frank L.; van der Meer, Jos W. M.; Dinarello, Charles A.; Joosten, Leo A. B.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-77</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Induction, prodn., and release of proinflammatory cytokines are essential steps to establish an effective host defense.  Cytokines of the interleukin-1 (IL-1) family induce inflammation and regulate T lymphocyte responses while also displaying homeostatic and metabolic activities.  With the exception of the IL-1 receptor antagonist, all IL-1 family cytokines lack a signal peptide and require proteolytic processing into an active mol.  One such unique protease is caspase-1, which is activated by protein platforms called the inflammasomes.  However, increasing evidence suggests that inflammasomes and caspase-1 are not the only mechanism for processing IL-1 cytokines.  IL-1 cytokines are often released as precursors and require extracellular processing for activity.  Here we review the inflammasome-independent enzymic processes that are able to activate IL-1 cytokines, paying special attention to neutrophil-derived serine proteases, which subsequently induce inflammation and modulate host defense.  The inflammasome-independent processing of IL-1 cytokines has important consequences for understanding inflammatory diseases, and it impacts the design of IL-1-based modulatory therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzeRfZbo7l0rVg90H21EOLACvtfcHk0liYcPHAs-kFLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVSisLzF&md5=e4f426578799058ea5cf1ca41376c714</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1146%2Fannurev-immunol-032414-112306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-immunol-032414-112306%26sid%3Dliteratum%253Aachs%26aulast%3DNetea%26aufirst%3DM.%2BG.%26aulast%3Dvan%2Bde%2BVeerdonk%26aufirst%3DF.%2BL.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DDinarello%26aufirst%3DC.%2BA.%26aulast%3DJoosten%26aufirst%3DL.%2BA.%2BB.%26atitle%3DInflammasome-independent%2520regulation%2520of%2520IL-1-family%2520cytokines%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2015%26volume%3D33%26spage%3D49%26epage%3D77%26doi%3D10.1146%2Fannurev-immunol-032414-112306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuai, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. Y.</span></span> <span> </span><span class="NLM_article-title">BST2 confers cisplatin resistance via NF-kappa B signaling in nasopharyngeal cancer</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e2874</span> <span class="refDoi"> DOI: 10.1038/cddis.2017.271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fcddis.2017.271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28617432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=C.+M.+Kuangauthor=X.+Fuauthor=Y.+J.+Huaauthor=W.+D.+Shuaiauthor=Z.+H.+Yeauthor=Y.+Liauthor=Q.+H.+Pengauthor=Y.+Z.+Liauthor=S.+Chenauthor=C.+N.+Qianauthor=W.+Huangauthor=R.+Y.+Liu&title=BST2+confers+cisplatin+resistance+via+NF-kappa+B+signaling+in+nasopharyngeal+cancer&doi=10.1038%2Fcddis.2017.271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">BST2 confers cisplatin resistance via NF-κB signaling in nasopharyngeal cancer</span></div><div class="casAuthors">Kuang, Chun-mei; Fu, Xiang; Hua, Yi-jun; Shuai, Wen-di; Ye, Zhi-hua; Li, Yingchang; Peng, Qi-hua; Li, Yi-zhuo; Chen, Shuai; Qian, Chao-nan; Huang, Wenlin; Liu, Ran-yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e2874</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concurrent/adjuvant cisplatin-based chemoradiotherapy is regarded as the std. of treatment for locoregionally advanced nasopharyngeal carcinoma (NPC).  However, patients who do not respond to cisplatin suffer, rather than benefit, from chemotherapy treatment.  The goal of this study was to identify mols. involved in cisplatin resistance and to clarify their mol. mechanisms, which would help in the discovery of potential therapeutic targets and in developing a personalized and precise treatment approach for NPC patients.  We previously generated a cisplatin-sensitive NPC cell line, S16, from CNE2 cells and found that eIF3a, ASNS and MMP19 are upregulated in S16 cells, which contributes to their cisplatin sensitivity.  In this study, we found that BST2 is downregulated in cisplatin-sensitive S16 cells compared with CNE2 cells.  Knockdown of BST2 in NPC cells sensitized their response to cisplatin and promoted cisplatin-induced apoptosis, whereas exogenous overexpression of BST2 increased their cisplatin resistance and inhibited cisplatin-induced apoptosis.  Further investigation demonstrated that BST2-mediated cisplatin resistance depended on the activation of the NF-κB signaling pathway and consequent upregulation of anti-apoptotic genes, such as Bcl-XL and livin.  Moreover, an anal. of clin. data revealed that a high BST2 level might serve as an independent indicator of poor prognosis in patients with locally advanced NPC treated with platinum-based chemoradiotherapy.  These findings suggest that BST2 likely mediates platinum resistance in NPC, offering guidance for personalized and precise treatment strategies for patients with NPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnUDS4Syse97Vg90H21EOLACvtfcHk0liYcPHAs-kFLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurvK&md5=59b258179f24a50ed62e51dcb4da0317</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2017.271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2017.271%26sid%3Dliteratum%253Aachs%26aulast%3DKuang%26aufirst%3DC.%2BM.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DHua%26aufirst%3DY.%2BJ.%26aulast%3DShuai%26aufirst%3DW.%2BD.%26aulast%3DYe%26aufirst%3DZ.%2BH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DQ.%2BH.%26aulast%3DLi%26aufirst%3DY.%2BZ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DQian%26aufirst%3DC.%2BN.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DR.%2BY.%26atitle%3DBST2%2520confers%2520cisplatin%2520resistance%2520via%2520NF-kappa%2520B%2520signaling%2520in%2520nasopharyngeal%2520cancer%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fcddis.2017.271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vannucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allavena, P.</span></span> <span> </span><span class="NLM_article-title">Inflammation as target in cancer therapy</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2017.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.coph.2017.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28618326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXptlShtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=57-65&author=G.+Marelliauthor=A.+Sicaauthor=L.+Vannucciauthor=P.+Allavena&title=Inflammation+as+target+in+cancer+therapy&doi=10.1016%2Fj.coph.2017.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammation as target in cancer therapy</span></div><div class="casAuthors">Marelli, Giulia; Sica, Antonio; Vannucci, Luca; Allavena, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">57-65</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cells of the innate immunity infiltrating tumor tissues promote, rather than halt, cancer cell proliferation and distant spreading.  Tumor-Assocd. Macrophages (TAMs) are abundantly present in the tumor milieu and here trigger and perpetrate a state of chronic inflammation which ultimately supports disease development and contributes to an immune-suppressive environment.  Therapeutic strategies to limit inflammatory cells and their products have been successful in pre-clin. tumor models.  Early clin. trials with specific cytokine and chemokine inhibitors, or with strategies designed to target TAMs, are on their way in different solid malignancies.  Partial clin. responses and stabilization of diseases were obsd. in some patients, in the absence of significant toxicity.  These encouraging results open new perspectives of combination treatments aimed at reducing cancer-promoting inflammation to maximize the anti-tumor efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDCjYnb6LW7Vg90H21EOLACvtfcHk0liUuwo3_B0jWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXptlShtbY%253D&md5=e8d65f2760714c9fb296a1c81fedb5ad</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2017.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2017.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DMarelli%26aufirst%3DG.%26aulast%3DSica%26aufirst%3DA.%26aulast%3DVannucci%26aufirst%3DL.%26aulast%3DAllavena%26aufirst%3DP.%26atitle%3DInflammation%2520as%2520target%2520in%2520cancer%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2017%26volume%3D35%26spage%3D57%26epage%3D65%26doi%3D10.1016%2Fj.coph.2017.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Moraes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olmedo, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faria, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Moura Gallo, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hainaut, P.</span></span> <span> </span><span class="NLM_article-title">Interferon-inducible guanylate binding protein (GBP)-2: a novel p53-regulated tumor marker in esophageal squamous cell carcinomas</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>124</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1002/ijc.23944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fijc.23944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=19003964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FitFSjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2009&pages=272-279&issue=2&author=D.+P.+Guimaraesauthor=I.+M.+Oliveiraauthor=E.+de+Moraesauthor=G.+R.+Paivaauthor=D.+M.+Souzaauthor=C.+Barnasauthor=D.+B.+Olmedoauthor=C.+E.+Pintoauthor=P.+A.+Fariaauthor=C.+V.+De+Moura+Galloauthor=I.+A.+Smallauthor=C.+G.+Ferreiraauthor=P.+Hainaut&title=Interferon-inducible+guanylate+binding+protein+%28GBP%29-2%3A+a+novel+p53-regulated+tumor+marker+in+esophageal+squamous+cell+carcinomas&doi=10.1002%2Fijc.23944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Interferon-inducible guanylate binding protein (GBP)-2: a novel p53-regulated tumor marker in esophageal squamous cell carcinomas</span></div><div class="casAuthors">Guimaraes Denise P; Oliveira Ivanir M; de Moraes Emanuela; Paiva Geisilene R; Souza Daniel M; Barnas Christoph; Olmedo Daniele B; Pinto Carlos E; Faria Paulo A; De Moura Gallo Claudia Vitoria; Small Isabele A; Ferreira Carlos G; Hainaut Pierre</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">272-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">TP53 mutations are common in esophageal squamous cell carcinomas (SCC).  To identify biological markers of possible relevance in esophageal SCC, we (i) searched for genes expressed in a p53-dependent manner in TE-1, an esophageal SCC cell line expressing the temperature-sensitive TP53 mutant V272M, and (ii) investigated the expression of one of those genes, the interferon-inducible Guanylate Binding Protein 2 (GBP-2), in esophageal SCC tissues.  Clontech Human Cancer 1.2 arrays containing 1,176 human cancer gene-related sequences were used to identify differentially expressed genes in TE-1 cells at permissive (32 degrees C) and nonpermissive (37 degrees C) temperatures.  The expression of GBP-2 and IRF-1, its main transcriptional regulator, was analyzed by immunohistochemistry in a retrospective series of 41 esophageal SCC cases with a clear transition zone from noncancer, apparently normal epithelium to invasive cancer.  The expression of the GBP-2 gene is consistently increased in TE-1 at 32 degrees C in a p53-dependent manner, as confirmed by inhibition of p53 expression by RNA interference.  Increase in GBP-2 is accompanied by an increase in protein levels of IRF-1, the main transcriptional regulator of GBP-2, and in the formation of complexes between p53 and IRF-1.  GBP-2 expression is significantly higher in esophageal SCC than in adjacent normal epithelium (p<0.01), in which GBP-2 staining is limited to the basal layer.  Our results suggest that p53 up-regulates GBP-2 by cooperating with IRF-1.  The association of GBP-2 expression with proliferative squamous cells suggests that GBP-2 may represent a marker of interest in esophageal SCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTtHR0juPLiV91QT_1O5WBYfW6udTcc2eZXeOX2xkA5Wrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FitFSjtw%253D%253D&md5=279e6df17dffd110b4f58290b94d1f9a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fijc.23944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.23944%26sid%3Dliteratum%253Aachs%26aulast%3DGuimaraes%26aufirst%3DD.%2BP.%26aulast%3DOliveira%26aufirst%3DI.%2BM.%26aulast%3Dde%2BMoraes%26aufirst%3DE.%26aulast%3DPaiva%26aufirst%3DG.%2BR.%26aulast%3DSouza%26aufirst%3DD.%2BM.%26aulast%3DBarnas%26aufirst%3DC.%26aulast%3DOlmedo%26aufirst%3DD.%2BB.%26aulast%3DPinto%26aufirst%3DC.%2BE.%26aulast%3DFaria%26aufirst%3DP.%2BA.%26aulast%3DDe%2BMoura%2BGallo%26aufirst%3DC.%2BV.%26aulast%3DSmall%26aufirst%3DI.%2BA.%26aulast%3DFerreira%26aufirst%3DC.%2BG.%26aulast%3DHainaut%26aufirst%3DP.%26atitle%3DInterferon-inducible%2520guanylate%2520binding%2520protein%2520%2528GBP%2529-2%253A%2520a%2520novel%2520p53-regulated%2520tumor%2520marker%2520in%2520esophageal%2520squamous%2520cell%2520carcinomas%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2009%26volume%3D124%26issue%3D2%26spage%3D272%26epage%3D279%26doi%3D10.1002%2Fijc.23944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antony, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meitzler, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, K.</span></span> <span> </span><span class="NLM_article-title">Functional activity and tumor-specific expression of dual oxidase 2 in pancreatic cancer cells and human malignancies characterized with a novel monoclonal antibody</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1238</span>, <span class="refDoi"> DOI: 10.3892/ijo.2013.1821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3892%2Fijo.2013.1821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=23404210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1ems70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=1229-1238&issue=4&author=Y.+Z.+Wuauthor=S.+Antonyauthor=S.+M.+Hewittauthor=G.+J.+Jiangauthor=S.+X.+Yangauthor=J.+L.+Meitzlerauthor=A.+Juhaszauthor=J.+M.+Luauthor=H.+Liuauthor=J.+H.+Doroshowauthor=K.+Roy&title=Functional+activity+and+tumor-specific+expression+of+dual+oxidase+2+in+pancreatic+cancer+cells+and+human+malignancies+characterized+with+a+novel+monoclonal+antibody&doi=10.3892%2Fijo.2013.1821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Functional activity and tumor-specific expression of dual oxidase 2 in pancreatic cancer cells and human malignancies characterized with a novel monoclonal antibody</span></div><div class="casAuthors">Wu, Yonghzong; Antony, Smitha; Hewitt, Stephen M.; Jiang, Guojian; Yang, Sherry X.; Meitzler, Jennifer L.; Juhasz, Agnes; Lu, Jiamo; Liu, Han; Doroshow, James H.; Roy, Krishnendu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1229-1238</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">Dual oxidase 2 (Duox2), one of the seven members of the NADPH oxidase gene family, plays a crit. role in generating H2O2 for thyroid hormone biosynthesis and as an integral part of the host defense system of the respiratory epithelium and the gastrointestinal tract.  Recent evidence suggests that the regulation of Duox2 expression is under the control of pro-inflammatory cytokines and that Duox2-induced reactive oxygen species (ROS) contribute to the inflammation-related tissue injury that occurs in two pre-malignant, inflammatory conditions: chronic pancreatitis and inflammatory bowel disease.  Because no reliable Duox antibodies are com. available, we report the development of a murine monoclonal antibody (MAb) to Duox2 (clone Duox S-12) and its use for the characterization of Duox2 expression in human tumors, tumor cell lines and normal tissues.  Duox S-12 specifically detected both endogenously- and ectopically-expressed Duox2 protein by immunoblotting, immunofluorescence microscopy and immunohistochem. (where both membranous and cytoplasmic staining were present).  Duox2 expression detected by Duox S-12 was functionally coupled to the generation of H2O2 in pancreatic cancer cells that expressed Duox2 and its cognate maturation factor DuoxA2.  Although Duox S-12 recognizes ectopically expressed Duox1 protein because of the extensive amino acid homol. between Duox1 and Duox2, the lack of substantial Duox1 mRNA expression in human tumors (except thyroid cancer) allowed us to evaluate Duox2 expression across a wide range of normal and malignant tissues by immunohistochem.  Duox2 was expressed at elevated levels in many human cancers, most notably tumors of the prostate, lung, colon and breast while brain tumors and lymphomas demonstrated the lowest frequency of expression.  The Duox-specific monoclonal antibody described here provides a promising tool for the further examn. of the role of Duox-dependent reactive oxygen prodn. in inflammation-related carcinogenesis, where alterations in oxidant tone play a crit. role in cell growth and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6heKp3f02KrVg90H21EOLACvtfcHk0liUuwo3_B0jWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1ems70%253D&md5=234aea208352a959dcd3f0d61bbcb362</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.3892%2Fijo.2013.1821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2013.1821%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%2BZ.%26aulast%3DAntony%26aufirst%3DS.%26aulast%3DHewitt%26aufirst%3DS.%2BM.%26aulast%3DJiang%26aufirst%3DG.%2BJ.%26aulast%3DYang%26aufirst%3DS.%2BX.%26aulast%3DMeitzler%26aufirst%3DJ.%2BL.%26aulast%3DJuhasz%26aufirst%3DA.%26aulast%3DLu%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DRoy%26aufirst%3DK.%26atitle%3DFunctional%2520activity%2520and%2520tumor-specific%2520expression%2520of%2520dual%2520oxidase%25202%2520in%2520pancreatic%2520cancer%2520cells%2520and%2520human%2520malignancies%2520characterized%2520with%2520a%2520novel%2520monoclonal%2520antibody%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2013%26volume%3D42%26issue%3D4%26spage%3D1229%26epage%3D1238%26doi%3D10.3892%2Fijo.2013.1821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glisson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Y.</span></span> <span> </span><span class="NLM_article-title">Insulin-like growth factor binding protein-3 suppresses vascular endothelial growth factor expression and tumor angiogenesis in head and neck squamous cell carcinoma</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2012.02301.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1111%2Fj.1349-7006.2012.02301.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22494072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsFKmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2012&pages=1259-1266&issue=8&author=S.+H.+Ohauthor=W.+Y.+Kimauthor=O.+H.+Leeauthor=J.+H.+Kangauthor=J.+K.+Wooauthor=J.+H.+Kimauthor=B.+Glissonauthor=H.+Y.+Lee&title=Insulin-like+growth+factor+binding+protein-3+suppresses+vascular+endothelial+growth+factor+expression+and+tumor+angiogenesis+in+head+and+neck+squamous+cell+carcinoma&doi=10.1111%2Fj.1349-7006.2012.02301.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Insulin-like growth factor binding protein-3 suppresses vascular endothelial growth factor expression and tumor angiogenesis in head and neck squamous cell carcinoma</span></div><div class="casAuthors">Oh, Seung-Hyun; Kim, Woo-Young; Lee, Ok-Hee; Kang, Ju-Hee; Woo, Jong-Kyu; Kim, Jai-Hyun; Glisson, Bonnie; Lee, Ho-Young</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1259-1266</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Angiogenesis, the process by which new blood vessels are recruited to existing ones, is essential for tumor development.  Insulin-like growth factor (IGF) binding protein-3 (IGFBP-3), which modulates bioavailability of IGF, has been studied for its potential role in angiogenesis during tissue regeneration and cancer development.  In this study, we assessed the role of IGFBP-3 in tumor angiogenesis in head and neck squamous cell carcinoma (HNSCC) and human umbilical vein endothelial cells (HUVECs) using adenoviral (Ad-BP3) and recombinant (rBP3) IGFBP-3.  Using an in vivo orthotopic tongue tumor model, we confirmed that both Ad-BP3 and rBP3 suppress the growth of UMSCC38 HNSCC cells in vivo.  Ad-BP3 inhibited vascularization in tongue tumors and chorio-allantoic membrane, and suppressed angiogenesis-stimulating activities in UMSCC38 cells.  In HUVECs, Ad-BP3 decreased migration, invasion, and tube formation. rBP3 also suppressed prodn. of vascular endothelial growth factor (VEGF) in HUVECs and UMSCC38 cells.  IGFBP-3-GGG, a mutant IGFBP-3 with loss of IGF binding capacity, suppressed VEGF prodn.  In addn., we found that IGFBP-3 suppressed VEGF expression, even in mouse embryonic fibroblasts from an IGF-1R-null mouse.  Finally, we demonstrated that IGFBP-3-GGG inhibits tumor angiogenesis and growth to the same degree as wild-type IGFBP-3.  Taken together, these results support the hypothesis that IGFBP-3 has anti-angiogenic activity in HNSCC, at least in part due to IGF-independent suppression of VEGF prodn. from vascular endothelial cells and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDKTEwsydrnbVg90H21EOLACvtfcHk0ljKW6PzDqf-ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsFKmtLo%253D&md5=94ea6be25162ebc45ca384db62b0dea8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2012.02301.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2012.02301.x%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DW.%2BY.%26aulast%3DLee%26aufirst%3DO.%2BH.%26aulast%3DKang%26aufirst%3DJ.%2BH.%26aulast%3DWoo%26aufirst%3DJ.%2BK.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DGlisson%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DH.%2BY.%26atitle%3DInsulin-like%2520growth%2520factor%2520binding%2520protein-3%2520suppresses%2520vascular%2520endothelial%2520growth%2520factor%2520expression%2520and%2520tumor%2520angiogenesis%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%26jtitle%3DCancer%2520Sci.%26date%3D2012%26volume%3D103%26issue%3D8%26spage%3D1259%26epage%3D1266%26doi%3D10.1111%2Fj.1349-7006.2012.02301.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, R. C.</span></span> <span> </span><span class="NLM_article-title">IGF binding proteins in cancer: mechanistic and clinical insights</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1038/nrc3720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnrc3720" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24722429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=329-341&issue=5&author=R.+C.+Baxter&title=IGF+binding+proteins+in+cancer%3A+mechanistic+and+clinical+insights&doi=10.1038%2Fnrc3720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">IGF binding proteins in cancer: mechanistic and clinical insights</span></div><div class="casAuthors">Baxter, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">329-341</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The six members of the family of insulin-like growth factor (IGF) binding proteins (IGFBPs) were originally characterized as passive reservoirs of circulating IGFs, but they are now understood to have many actions beyond their endocrine role in IGF transport.  IGFBPs also function in the pericellular and intracellular compartments to regulate cell growth and survival - they interact with many proteins, in addn. to their canonical ligands IGF-I and IGF-II.  Intranuclear roles of IGFBPs in transcriptional regulation, induction of apoptosis and DNA damage repair point to their intimate involvement in tumor development, progression and resistance to treatment.  Tissue or circulating IGFBPs might also be useful as prognostic biomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH1zpqHzHzv7Vg90H21EOLACvtfcHk0ljKW6PzDqf-ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWns7Y%253D&md5=0e88b6b79cffd306e30248bc145e879d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnrc3720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3720%26sid%3Dliteratum%253Aachs%26aulast%3DBaxter%26aufirst%3DR.%2BC.%26atitle%3DIGF%2520binding%2520proteins%2520in%2520cancer%253A%2520mechanistic%2520and%2520clinical%2520insights%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26issue%3D5%26spage%3D329%26epage%3D341%26doi%3D10.1038%2Fnrc3720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polachini, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Paula-Silva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliani, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cury, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gois-Filho, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Carvalho, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leopoldino, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajara, E. H.</span></span> <span> </span><span class="NLM_article-title">Differentially expressed proteins in positive versus negative HNSCC lymph nodes</span>. <i>BMC Med. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">73</span>, <span class="refDoi"> DOI: 10.1186/s12920-018-0382-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fs12920-018-0382-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30157864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Gmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=73&author=A.+Vidottoauthor=G.+M.+Polachiniauthor=M.+de+Paula-Silvaauthor=S.+M.+Olianiauthor=T.+Henriqueauthor=R.+V.+M.+Lopezauthor=P.+M.+Curyauthor=F.+D.+Nunesauthor=J.+F.+Gois-Filhoauthor=M.+B.+de+Carvalhoauthor=A.+M.+Leopoldinoauthor=E.+H.+Tajara&title=Differentially+expressed+proteins+in+positive+versus+negative+HNSCC+lymph+nodes&doi=10.1186%2Fs12920-018-0382-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Differentially expressed proteins in positive versus negative HNSCC lymph nodes</span></div><div class="casAuthors">Vidotto, Alessandra; Polachini, Giovana M.; de Paula-Silva, Marina; Oliani, Sonia M.; Henrique, Tiago; Lopez, Rossana V. M.; Cury, Patricia M.; Nunes, Fabio D.; Gois-Filho, Jose F.; de Carvalho, Marcos B.; Leopoldino, Andreia M.; Tajara, Eloiza H.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Medical Genomics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73/1-73/16</span>CODEN:
                <span class="NLM_cas:coden">BMGMBS</span>;
        ISSN:<span class="NLM_cas:issn">1755-8794</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Lymph node metastasis is one of the most important prognostic factors in head and neck squamous cell carcinomas (HNSCCs) and crit. for delineating their treatment.  However, clin. and histol. criteria for the diagnosis of nodal status remain limited.  In the present study, we aimed to characterize the proteomic profile of lymph node metastasis from HNSCC patients.  Methods: In the present study, we used one- and two-dimensional electrophoresis and mass spectrometry anal. to characterize the proteomic profile of lymph node metastasis from HNSCC.  Results: Comparison of metastatic and non-metastatic lymph nodes showed 52 differentially expressed proteins assocd. with neoplastic development and progression.  The results reinforced the idea that tumors from different anatomical subsites have dissimilar behaviors, which may be influenced by micro-environmental factor including the lymphatic network.  The expression pattern of heat shock proteins and glycolytic enzymes also suggested an effect of the lymph node environment in controlling tumor growth or in metabolic reprogramming of the metastatic cell.  Our study, for the first time, provided direct evidence of annexin A1 overexpression in lymph node metastasis of head and neck cancer, adding information that may be useful for diagnosing aggressive disease.  Conclusions: In brief, this study contributed to our understanding of the metastatic phenotype of HNSCC and provided potential targets for diagnostic in this group of carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVNjfWtoKBwLVg90H21EOLACvtfcHk0ljKW6PzDqf-ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Gmu7s%253D&md5=a460c2e2757ea41acbd57e0681e7a65b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs12920-018-0382-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12920-018-0382-6%26sid%3Dliteratum%253Aachs%26aulast%3DVidotto%26aufirst%3DA.%26aulast%3DPolachini%26aufirst%3DG.%2BM.%26aulast%3Dde%2BPaula-Silva%26aufirst%3DM.%26aulast%3DOliani%26aufirst%3DS.%2BM.%26aulast%3DHenrique%26aufirst%3DT.%26aulast%3DLopez%26aufirst%3DR.%2BV.%2BM.%26aulast%3DCury%26aufirst%3DP.%2BM.%26aulast%3DNunes%26aufirst%3DF.%2BD.%26aulast%3DGois-Filho%26aufirst%3DJ.%2BF.%26aulast%3Dde%2BCarvalho%26aufirst%3DM.%2BB.%26aulast%3DLeopoldino%26aufirst%3DA.%2BM.%26aulast%3DTajara%26aufirst%3DE.%2BH.%26atitle%3DDifferentially%2520expressed%2520proteins%2520in%2520positive%2520versus%2520negative%2520HNSCC%2520lymph%2520nodes%26jtitle%3DBMC%2520Med.%2520Genomics%26date%3D2018%26volume%3D11%26spage%3D73%26doi%3D10.1186%2Fs12920-018-0382-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span> <span> </span><span class="NLM_article-title">Fatty-acid-binding protein 5 promotes cell proliferation and invasion in oral squamous cell carcinoma</span>. <i>J. Oral Pathol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">348</span>, <span class="refDoi"> DOI: 10.1111/j.1600-0714.2009.00836.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1111%2Fj.1600-0714.2009.00836.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20040021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1aqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=342-348&issue=4&author=L.+Y.+Fangauthor=T.+Y.+Wongauthor=W.+F.+Chiangauthor=Y.+L.+Chen&title=Fatty-acid-binding+protein+5+promotes+cell+proliferation+and+invasion+in+oral+squamous+cell+carcinoma&doi=10.1111%2Fj.1600-0714.2009.00836.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Fatty-acid-binding protein 5 promotes cell proliferation and invasion in oral squamous cell carcinoma</span></div><div class="casAuthors">Fang, Lei-Ya; Wong, Tung-Yiu; Chiang, Wei-Fan; Chen, Yuh-Ling</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Oral Pathology & Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">342-348</span>CODEN:
                <span class="NLM_cas:coden">JPMEEA</span>;
        ISSN:<span class="NLM_cas:issn">0904-2512</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND: Oral squamous cell carcinoma (OSCC) is one of the most malignant neoplasms worldwide, and the mol. mechanism of oral tumorigenesis is still unclear.  Fatty-acid-binding protein 5 (FABP5) was found in our previous study to be upregulated in oral squamous cell carcinomas by proteomic anal.  The implications of FABP5 overexpression in oral cancer progression have not yet been elucidated.  MATERIALS AND METHODS: In this study, the recombinant adeno-assocd. virus vectors were used to deliver and increase the expression of FABP5 in human OSCC cell lines.  U6 promoter-driven short-hairpin RNA (shRNA)-triggered RNA interference was used to block FABP5 gene expression.  Transwell Matrigel invasion assay, MTS cell proliferation assay, reverse transcription-polymerase chain reaction, Western blot, and gelatin zymog. anal. were used to investigate the effects of FABP5 on cell invasion, growth, and matrix metalloproteinase (MMP) prodn.  RESULTS: Overexpression of FABP5 in oral cancer cells increased cell proliferation and invasiveness by increasing the expression of MMP-9.  Silencing FABP5 with shRNA significantly suppressed cell proliferation, MMP-9 activities, and invasiveness.  CONCLUSION: Our study provides the first evidence that FABP5 expression modulated MMP-9 prodn. and the invasive behavior of oral cancer cells and suggests that FABP5 may provide novel targets for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMhi0CPQQEw7Vg90H21EOLACvtfcHk0lj39_mKnaXbLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1aqtLc%253D&md5=b2ff5d080eb0202ac2b9aa349f5bbf81</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0714.2009.00836.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0714.2009.00836.x%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DL.%2BY.%26aulast%3DWong%26aufirst%3DT.%2BY.%26aulast%3DChiang%26aufirst%3DW.%2BF.%26aulast%3DChen%26aufirst%3DY.%2BL.%26atitle%3DFatty-acid-binding%2520protein%25205%2520promotes%2520cell%2520proliferation%2520and%2520invasion%2520in%2520oral%2520squamous%2520cell%2520carcinoma%26jtitle%3DJ.%2520Oral%2520Pathol.%2520Med.%26date%3D2010%26volume%3D39%26issue%3D4%26spage%3D342%26epage%3D348%26doi%3D10.1111%2Fj.1600-0714.2009.00836.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span> <span> </span><span class="NLM_article-title">Analysis of gene coexpression network reveals prognostic significance of CNFN in patients with head and neck cancer</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2168</span>– <span class="NLM_lpage">2180</span>, <span class="refDoi"> DOI: 10.3892/or.2019.7019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3892%2For.2019.7019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30816522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWmt7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=2168-2180&author=B.+Liuauthor=G.+Huangauthor=H.+Zhuauthor=Z.+Maauthor=X.+Tianauthor=L.+Yinauthor=X.+Gaoauthor=X.+He&title=Analysis+of+gene+coexpression+network+reveals+prognostic+significance+of+CNFN+in+patients+with+head+and+neck+cancer&doi=10.3892%2For.2019.7019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of gene co-expression network reveals prognostic significance of CNFN in patients with head and neck cancer</span></div><div class="casAuthors">Liu, Baoling; Huang, Guanhong; Zhu, Hongming; Ma, Zhaoming; Tian, Xiaokang; Yin, Li; Gao, Xingya; He, Xia</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2168-2180</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">In patients with head and neck cancer (HNC), lymph node (N) metastases are assocd. with cancer aggressiveness and poor prognosis.  Identifying meaningful gene modules and representative biomarkers relevant to the N stage helps predict prognosis and reveal mechanisms underlying tumor progression.  The present study used a step-wise approach for weighted gene co-expression network anal. (WGCNA).  Dataset GSE65858 was subjected to WGCNA.  RNA sequencing data of HNC downloaded from the Cancer Genome Atlas (TCGA) and dataset GSE39366 were utilized to validate the results.  Following data preprocessing, 4,295 genes were screened, and blue and black modules assocd. with the N stage of HNC were identified.  A total of 16 genes [keratinocyte differentiation assocd. protein, suprabasin, cornifelin (CNFN), small proline rich protein 1B, desmoglein 1 (DSG1), chromosome 10 open reading frame 99, keratin 16 pseudogene 3, gap junction protein β2, dermokine, LY6/PLAUR domain contg. 3, transmembrane protein 79, phospholipase A2 group IVE, transglutaminase 5, potassium two pore domain channel subfamily K member 6, involucrin, kallikrein related peptidase 8] that had a neg. assocn. with the N-stage in the blue module, and two genes (structural maintenance of chromosomes 4 and mutS homolog 6) that had a pos. assocn. in the black module, were identified to be candidate hub genes.  Following further validation in TCGA and dataset GSE65858, it was identified that CNFN and DSG1 were assocd. with the clin. stage of HNC.  Survival anal. of CNFN and DSG1 was subsequently performed.  Patients with increased expression of CNFN displayed better survival probability in dataset GSE65858 and TCGA.  Therefore, CNFN was selected as the hub gene for further verification in the Gene Expression Profiling Interactive Anal. database.  Finally, functional enrichment and gene set enrichment analyses were performed using datasets GSE65858 and GSE39366.  Three gene sets, namely 'P53 pathway', 'estrogen response early' and 'estrogen response late', were enriched in the two datasets.  In conclusion, CNFN, identified via the WGCNA algorithm, may contribute to the prediction of lymph node metastases and prognosis, probably by regulating the pathways assocd. with P53, and the early and late estrogen response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVtfkzm-Shn7Vg90H21EOLACvtfcHk0lj39_mKnaXbLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWmt7jI&md5=2e7fe9a69197f4e883f80f9ecf4b1d4a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.3892%2For.2019.7019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2019.7019%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26atitle%3DAnalysis%2520of%2520gene%2520coexpression%2520network%2520reveals%2520prognostic%2520significance%2520of%2520CNFN%2520in%2520patients%2520with%2520head%2520and%2520neck%2520cancer%26jtitle%3DOncol.%2520Rep.%26date%3D2019%26volume%3D41%26spage%3D2168%26epage%3D2180%26doi%3D10.3892%2For.2019.7019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. K.</span></span> <span> </span><span class="NLM_article-title">Snail interacts with Id2 in the regulation of TNF-alpha-induced cancer cell invasion and migration in OSCC</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1680</span>– <span class="NLM_lpage">1691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26175937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWrsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=1680-1691&issue=5&author=J.+P.+Zhouauthor=Z.+L.+Gaoauthor=M.+L.+Zhouauthor=M.+Y.+Heauthor=X.+H.+Xuauthor=D.+T.+Taoauthor=C.+C.+Yangauthor=L.+K.+Liu&title=Snail+interacts+with+Id2+in+the+regulation+of+TNF-alpha-induced+cancer+cell+invasion+and+migration+in+OSCC"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Snail interacts with Id2 in the regulation of TNF-α-induced cancer cell invasion and migration in OSCC</span></div><div class="casAuthors">Zhou, Jing-Ping; Gao, Zhen-Lin; Zhou, Mei-Ling; He, Meng-Ying; Xu, Xiao-Hui; Tao, De-Tao; Yang, Cong-Chong; Liu, Lai-Kui</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1680-1691</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">The inflammatory tumor microenvironment has been identified to play a pivotal role in tumor development and metastasis.  Tumor necrosis factor-α (TNF-α) is one of the key cytokines that regulate the inflammatory processes in tumor promotion.  In the current study, we treated three oral squamous cell carcinoma (OSCC) cell lines with TNF-α to study its role in inflammation-induced tumor progression.  Here we show that TNF-α induces stabilization of the transcriptional repressor Snail and activates NF-κB pathway in the three OSCC cell lines.  These activities resulted in the increased motility and invasiveness of three OSCC cell lines.  In addn., upon dealing with TNF-α for the indicated time, three OSCC cell lines underwent epithelial-to-mesenchymal transition (EMT), in which they presented a fibroblast-like phenotype and had a decreased expression of epithelial marker (E-cadherin) and an increased expression of mesenchymal marker (vimentin).  We further demonstrated that TNF-α can up-regulate the expression of Id2 while inducing an EMT in oral cancer cells.  Finally, we showed that Id2 interacted with Snail which may constrain Snail-dependent suppression of E-cadherin.  In conclusion, our study indicates that TNF-α induces Snail stabilization is dependent on the activation of NF-κB pathway and results in increasing cell invasion and migration in OSCC cells.  Id2 may contribute to regulate the function of Snail during TNF-α-mediated EMT in OSCC.  These findings have significant implications for inflammation-induced tumor promotion in OSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfetY20QE4yrVg90H21EOLACvtfcHk0lj39_mKnaXbLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWrsrjM&md5=38d2723f9c547852b3cbdc48b7fd8450</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%2BP.%26aulast%3DGao%26aufirst%3DZ.%2BL.%26aulast%3DZhou%26aufirst%3DM.%2BL.%26aulast%3DHe%26aufirst%3DM.%2BY.%26aulast%3DXu%26aufirst%3DX.%2BH.%26aulast%3DTao%26aufirst%3DD.%2BT.%26aulast%3DYang%26aufirst%3DC.%2BC.%26aulast%3DLiu%26aufirst%3DL.%2BK.%26atitle%3DSnail%2520interacts%2520with%2520Id2%2520in%2520the%2520regulation%2520of%2520TNF-alpha-induced%2520cancer%2520cell%2520invasion%2520and%2520migration%2520in%2520OSCC%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2015%26volume%3D5%26issue%3D5%26spage%3D1680%26epage%3D1691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monisha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmavathi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banik, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoloi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khwairakpam, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arfuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnathambi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alahmadi, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alharbi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunnumakkara, A. B.</span></span> <span> </span><span class="NLM_article-title">NGAL is downregulated in oral squamous cell carcinoma and leads to increased survival, proliferation, migration and chemoresistance</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">228</span>, <span class="refDoi"> DOI: 10.3390/cancers10070228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3390%2Fcancers10070228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gru7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=228&issue=7&author=J.+Monishaauthor=N.+K.+Royauthor=G.+Padmavathiauthor=K.+Banikauthor=D.+Bordoloiauthor=A.+D.+Khwairakpamauthor=F.+Arfusoauthor=A.+Chinnathambiauthor=T.+A.+Alahmadiauthor=S.+A.+Alharbiauthor=G.+Sethiauthor=A.+P.+Kumarauthor=A.+B.+Kunnumakkara&title=NGAL+is+downregulated+in+oral+squamous+cell+carcinoma+and+leads+to+increased+survival%2C+proliferation%2C+migration+and+chemoresistance&doi=10.3390%2Fcancers10070228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">NGAL is downregulated in oral squamous cell carcinoma and leads to increased survival, proliferation, migration and chemoresistance</span></div><div class="casAuthors">Monisha, Javadi; Roy, Nand Kishor; Padmavathi, Ganesan; Banik, Kishore; Bordoloi, Devivasha; Khwairakpam, Amrita Devi; Arfuso, Frank; Chinnathambi, Arunachalam; Alahmadi, Tahani Awad; Alharbi, Sulaiman Ali; Sethi, Gautam; Kumar, Alan Prem; Kunnumakkara, Ajaikumar B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">228/1-228/18</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Oral cancer is a major public health burden worldwide.  The lack of biomarkers for early diagnosis has increased the difficulty in managing this disease.  Recent studies have reported that neutrophil gelatinase-assocd. lipocalin (NGAL), a secreted glycoprotein, is upregulated in various tumors.  In our study, we found that NGAL was significantly downregulated in primary malignant and metastatic tissues of oral cancer in comparison to normal tissues.  The downregulation of NGAL was strongly correlated with both degree of differentiation and stage (I-IV); it can also serve as a prognostic biomarker for oral cancer.  Addnl., tobacco carcinogens were found to be involved in the downregulation of NGAL.  Mechanistic studies revealed that knockdown of NGAL increased oral cancer cell proliferation, survival, and migration; it also induced resistance against cisplatin.  Silencing of NGAL activated mammalian target of rapamycin (mTOR)signaling and reduced autophagy by the liver kinase B1 (LKB1)-activated protein kinase (AMPK)-p53-Redd1 signaling axis.  Moreover, cyclin-D1, Bcl-2, and matrix metalloproteinase-9 (MMP-9) were upregulated, and caspase-9 was downregulated, suggesting that silencing of NGAL increases oral cancer cell proliferation, survival, and migration.  Thus, from our study, it is evident that downregulation of NGAL activates the mTOR pathway and helps in the progression of oral cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMDYeK7OvbTrVg90H21EOLACvtfcHk0lhQ1vGxXGeXLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gru7jE&md5=1ad6510ac61634da53628e6cd5de64be</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.3390%2Fcancers10070228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10070228%26sid%3Dliteratum%253Aachs%26aulast%3DMonisha%26aufirst%3DJ.%26aulast%3DRoy%26aufirst%3DN.%2BK.%26aulast%3DPadmavathi%26aufirst%3DG.%26aulast%3DBanik%26aufirst%3DK.%26aulast%3DBordoloi%26aufirst%3DD.%26aulast%3DKhwairakpam%26aufirst%3DA.%2BD.%26aulast%3DArfuso%26aufirst%3DF.%26aulast%3DChinnathambi%26aufirst%3DA.%26aulast%3DAlahmadi%26aufirst%3DT.%2BA.%26aulast%3DAlharbi%26aufirst%3DS.%2BA.%26aulast%3DSethi%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DA.%2BP.%26aulast%3DKunnumakkara%26aufirst%3DA.%2BB.%26atitle%3DNGAL%2520is%2520downregulated%2520in%2520oral%2520squamous%2520cell%2520carcinoma%2520and%2520leads%2520to%2520increased%2520survival%252C%2520proliferation%252C%2520migration%2520and%2520chemoresistance%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26issue%3D7%26spage%3D228%26doi%3D10.3390%2Fcancers10070228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pidugu, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pidugu, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. C.</span></span> <span> </span><span class="NLM_article-title">IFIT1 and IFIT3 promote oral squamous cell carcinoma metastasis and contribute to the anti-tumor effect of gefitinib via enhancing p-EGFR recycling</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">3232</span>– <span class="NLM_lpage">3247</span>, <span class="refDoi"> DOI: 10.1038/s41388-018-0662-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fs41388-018-0662-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30626937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVahtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=3232-3247&author=V.+K.+Piduguauthor=M.+M.+Wuauthor=A.+H.+Yenauthor=H.+B.+Piduguauthor=K.+W.+Changauthor=C.+J.+Liuauthor=T.+C.+Lee&title=IFIT1+and+IFIT3+promote+oral+squamous+cell+carcinoma+metastasis+and+contribute+to+the+anti-tumor+effect+of+gefitinib+via+enhancing+p-EGFR+recycling&doi=10.1038%2Fs41388-018-0662-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">IFIT1 and IFIT3 promote oral squamous cell carcinoma metastasis and contribute to the anti-tumor effect of gefitinib via enhancing p-EGFR recycling</span></div><div class="casAuthors">Pidugu, Vijaya Kumar; Wu, Meei-Maan; Yen, Ai-Hsin; Pidugu, Hima Bindu; Chang, Kuo-Wei; Liu, Chung-Ji; Lee, Te-Chang</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3232-3247</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">IFIT1 and IFIT3 are abundant products of interferon-stimulating genes.  While the importance of IFIT1 and IFIT3 in the prognosis of cancer has been reported, the mol. basis of IFIT1 and IFIT3 in cancer progression remains unexplored.  In the present study, we investigated the modes of action and the clin. significance of IFIT1 and IFIT3 in oral squamous cell carcinoma (OSCC).  Ectopic expression of IFIT1 or IFIT3 induced OSCC cell invasion by promoting the epithelial-mesenchymal transition, whereas IFIT1 or IFIT3 knockdown exhibited opposite effects.  Overexpression of IFIT1 or IFIT3 promoted tumor growth, regional and distant metastasis in xenograft and orthotopic nude mice models.  Most importantly, IFIT1 or IFIT3 overexpression increased the levels of p-EGFRY1068 and p-AKTS473 in OSCC cells and also enhanced tumor inhibitory effect of gefitinib.  By immunopptn. and LC-MS/MS anal., we found that IFIT1 and IFIT3 interacted with ANXA2 that enhanced p-EGFRY1068 endosomal recycling.  Depletion of ANXA2 using siRNA therefore abolished p-EGFRY1068 and p-AKTS473 expression in IFIT1- or IFIT3-overexpressed cells.  Furthermore, a significant pos. assocn. of increased IFIT1 and IFIT3 expression with advanced T-stage, lymph node metastasis, perineural invasion, lymphovascular invasion, extranodal extension, and poor overall survival rate was confirmed in OSCC patients.  We also found a statistically pos. correlation of p-EGFRY1068 expression with IFIT1 and IFIT3 in OSCC tumors and poor clin. outcome in patients.  Collectively, we demonstrated a novel role of IFIT1 and IFIT3 in driving OSCC progression and metastasis by interacting with ANXA2 and hence enhancing p-EGFR recycling and its downstream signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNnDAJ74IIA7Vg90H21EOLACvtfcHk0lhQ1vGxXGeXLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVahtb4%253D&md5=a0113a4f427d9f825723349418c810b6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41388-018-0662-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41388-018-0662-9%26sid%3Dliteratum%253Aachs%26aulast%3DPidugu%26aufirst%3DV.%2BK.%26aulast%3DWu%26aufirst%3DM.%2BM.%26aulast%3DYen%26aufirst%3DA.%2BH.%26aulast%3DPidugu%26aufirst%3DH.%2BB.%26aulast%3DChang%26aufirst%3DK.%2BW.%26aulast%3DLiu%26aufirst%3DC.%2BJ.%26aulast%3DLee%26aufirst%3DT.%2BC.%26atitle%3DIFIT1%2520and%2520IFIT3%2520promote%2520oral%2520squamous%2520cell%2520carcinoma%2520metastasis%2520and%2520contribute%2520to%2520the%2520anti-tumor%2520effect%2520of%2520gefitinib%2520via%2520enhancing%2520p-EGFR%2520recycling%26jtitle%3DOncogene%26date%3D2019%26volume%3D38%26spage%3D3232%26epage%3D3247%26doi%3D10.1038%2Fs41388-018-0662-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldinucci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombatti, A.</span></span> <span> </span><span class="NLM_article-title">The inflammatory chemokine CCL5 and cancer progression</span>. <i>Mediators Inflammation</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">292376</span>, <span class="refDoi"> DOI: 10.1155/2014/292376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1155%2F2014%2F292376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24523569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BC2crmsVKmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=292376&author=D.+Aldinucciauthor=A.+Colombatti&title=The+inflammatory+chemokine+CCL5+and+cancer+progression&doi=10.1155%2F2014%2F292376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">The inflammatory chemokine CCL5 and cancer progression</span></div><div class="casAuthors">Aldinucci Donatella; Colombatti Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Mediators of inflammation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">292376</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Until recently, inflammatory chemokines were viewed mainly as indispensable "gate keepers" of immunity and inflammation.  However, updated research indicates that cancer cells subvert the normal chemokine system and these molecules and their receptors become important constituents of the tumor microenvironment with very different ways to exert tumor-promoting roles.  The CCR5 and the CCL5 ligand have been detected in some hematological malignancies, lymphomas, and a great number of solid tumors, but extensive studies on the role of the CCL5/CCR axis were performed only in a limited number of cancers.  This review summarizes updated information on the role of CCL5 and its receptor CCR5 in cancer cell proliferation, metastasis, and the formation of an immunosuppressive microenvironment and highlights the development of newer therapeutic strategies aimed to inhibit the binding of CCL5 to CCR5, to inhibit CCL5 secretion, or to inhibit the interactions among tumor cells and the microenvironment leading to CCL5 secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWZ0g10qbTe9j_Vg2vObihfW6udTcc2eYUEgklZclyubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crmsVKmtQ%253D%253D&md5=3739b5f31946ab456c3ac3e2bb9f929e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1155%2F2014%2F292376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F292376%26sid%3Dliteratum%253Aachs%26aulast%3DAldinucci%26aufirst%3DD.%26aulast%3DColombatti%26aufirst%3DA.%26atitle%3DThe%2520inflammatory%2520chemokine%2520CCL5%2520and%2520cancer%2520progression%26jtitle%3DMediators%2520Inflammation%26date%3D2014%26volume%3D2014%26spage%3D292376%26doi%3D10.1155%2F2014%2F292376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. H.</span></span> <span> </span><span class="NLM_article-title">Autocrine CCL5 promotes tumor progression in esophageal squamous cell carcinoma in vitro</span>. <i>Cytokine</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1016/j.cyto.2018.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.cyto.2018.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29705397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFWntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2018&pages=94-103&author=Y.+C.+Wuauthor=Y.+C.+Shenauthor=J.+W.+C.+Changauthor=J.+J.+Hsiehauthor=Y.+Chuauthor=C.+H.+Wang&title=Autocrine+CCL5+promotes+tumor+progression+in+esophageal+squamous+cell+carcinoma+in+vitro&doi=10.1016%2Fj.cyto.2018.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Autocrine CCL5 promotes tumor progression in esophageal squamous cell carcinoma in vitro</span></div><div class="casAuthors">Wu, Yi-Cheng; Shen, Yung-Chi; Chang, John Wen-Cheng; Hsieh, Jia-Juan; Chu, Yen; Wang, Cheng-Hsu</div><div class="citationInfo"><span class="NLM_cas:title">Cytokine+</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">94-103</span>CODEN:
                <span class="NLM_cas:coden">CYTIE9</span>;
        ISSN:<span class="NLM_cas:issn">1043-4666</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The pro-tumoral effects of CCL5 have been identified in numerous cancer types.  We successfully cultivated 4 esophageal squamous cell carcinoma (ESCC) cell lines, including TWES-1, TWES-3 and a pair of cell lines derived from primary lesion (TWES-4PT) and metastatic lymph node (TWES-4LN) of the same patient.  Whole genome screening showed that TWES-4LN expressed higher levels of CCL5 compared to that of TWES-4PT; quantification of protein secretion displayed comparable results, suggesting that CCL5 could be assocd. with lymph node metastasis in ESCC.  CCL5 knockdown by siRNA significantly reduced basal growth rate, tumor migration and invasiveness in the paired cell lines; whereas this treatment induced cell apoptosis in TWES-1 and TWES-3.  CCR5 antagonist maraviroc significantly inhibited tumor migration and invasion in the paired cell lines without affecting tumor growth.  Collectively, these results suggest that CCL5 autocrine loop may promote ESCC progression; targeting the CCL5/CCR5 axis could be a potential therapeutic strategy for this deadly disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaeRSzQ3o6y7Vg90H21EOLACvtfcHk0lgSv1P8S57zdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFWntr8%253D&md5=48df28d247242a3b321cb010ea27bbb6</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.cyto.2018.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cyto.2018.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%2BC.%26aulast%3DShen%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DJ.%2BW.%2BC.%26aulast%3DHsieh%26aufirst%3DJ.%2BJ.%26aulast%3DChu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%2BH.%26atitle%3DAutocrine%2520CCL5%2520promotes%2520tumor%2520progression%2520in%2520esophageal%2520squamous%2520cell%2520carcinoma%2520in%2520vitro%26jtitle%3DCytokine%26date%3D2018%26volume%3D110%26spage%3D94%26epage%3D103%26doi%3D10.1016%2Fj.cyto.2018.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barbasiewicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciniak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorynski, M.</span></span> <span> </span><span class="NLM_article-title">Phase transfer alkylation of arylacetonitriles revisited</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">3871</span>– <span class="NLM_lpage">3874</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2006.03.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.tetlet.2006.03.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFertrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=3871-3874&issue=23&author=M.+Barbasiewiczauthor=K.+Marciniakauthor=M.+Fedorynski&title=Phase+transfer+alkylation+of+arylacetonitriles+revisited&doi=10.1016%2Fj.tetlet.2006.03.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Phase transfer alkylation of arylacetonitriles revisited</span></div><div class="casAuthors">Barbasiewicz, Michal; Marciniak, Karolina; Fedorynski, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3871-3874</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phase-transfer alkylations of phenylacetonitrile derivs. carried out in the presence of 60-75% aq. KOH, instead of the typical 50% NaOH, provide substantial improvements in the overall yields and purity of products.  Reactions with simple secondary alkyl halides, as well as cycloalkylations with 1,2- and 1,3-dihalo alkanes proceed with good yields.  Increasing the concn. of base diminishes the formation of byproducts from competitive β-elimination processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHy3iKcyqazbVg90H21EOLACvtfcHk0lgSv1P8S57zdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFertrw%253D&md5=05b7e265e24b2405c0456e51a61a6811</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2006.03.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2006.03.176%26sid%3Dliteratum%253Aachs%26aulast%3DBarbasiewicz%26aufirst%3DM.%26aulast%3DMarciniak%26aufirst%3DK.%26aulast%3DFedorynski%26aufirst%3DM.%26atitle%3DPhase%2520transfer%2520alkylation%2520of%2520arylacetonitriles%2520revisited%26jtitle%3DTetrahedron%2520Lett.%26date%3D2006%26volume%3D47%26issue%3D23%26spage%3D3871%26epage%3D3874%26doi%3D10.1016%2Fj.tetlet.2006.03.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chae, J.</span></span> <span> </span><span class="NLM_article-title">Practical demethylation of aryl methyl ethers using an odorless thiol reagent</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1007/s12272-001-1156-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2Fs12272-001-1156-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=18409042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1ajtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2008&pages=305-309&issue=3&author=J.+Chae&title=Practical+demethylation+of+aryl+methyl+ethers+using+an+odorless+thiol+reagent&doi=10.1007%2Fs12272-001-1156-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Practical demethylation of aryl methyl ethers using an odorless thiol reagent</span></div><div class="casAuthors">Chae, Junghyun</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">305-309</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">A highly practical method for demethylation of aryl Me ethers employing a long-chain thiol was developed.  Under the conditions described herein, clean and fast conversions to the desired phenolic compds. were achieved with a broad range of substrates.  Unlike other thiolate-mediated methods, this newly developed protocol features in-situ generation of sodium alkylthiolate using NaOH, and is almost free from foul smells and potentially harmful gases.  It therefore provided an attractive option for the demethylation of aryl Me ethers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofl51-WtrYQLVg90H21EOLACvtfcHk0lhIZPkWnscG8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1ajtb8%253D&md5=e0affc1b45d71efa2acd83b927baa4bf</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs12272-001-1156-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-001-1156-y%26sid%3Dliteratum%253Aachs%26aulast%3DChae%26aufirst%3DJ.%26atitle%3DPractical%2520demethylation%2520of%2520aryl%2520methyl%2520ethers%2520using%2520an%2520odorless%2520thiol%2520reagent%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2008%26volume%3D31%26issue%3D3%26spage%3D305%26epage%3D309%26doi%3D10.1007%2Fs12272-001-1156-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, T.</span></span> <span> </span><span class="NLM_article-title">2-(Diethylamino) ethanethiol, a new reagent for the odorless deprotection of aromatic methyl ethers</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">7103</span>– <span class="NLM_lpage">7105</span>, <span class="refDoi"> DOI: 10.1021/jo0611059</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo0611059" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvFOnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=7103-7105&issue=18&author=J.+Maganoauthor=M.+H.+Chenauthor=J.+D.+Clarkauthor=T.+Nussbaumer&title=2-%28Diethylamino%29+ethanethiol%2C+a+new+reagent+for+the+odorless+deprotection+of+aromatic+methyl+ethers&doi=10.1021%2Fjo0611059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">2-(Diethylamino)ethanethiol, a new reagent for the odorless deprotection of aromatic methyl ethers</span></div><div class="casAuthors">Magano, Javier; Chen, Michael H.; Clark, Jerry D.; Nussbaumer, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7103-7105</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new reagent for the deprotection of arom. Me ethers, 2-(diethylamino)ethanethiol, was reported.  This compd., com. available as its HCl salt, afforded the corresponding phenols in good to excellent yields on a wide variety of substrates.  A clear advantage of this method over the use of more common thiols, such as ethanethiol, was the easy extn. of both the deprotecting reagent and the byproduct 2-(diethylamino)ethyl Me sulfide into the aq. phase by quenching with dil. acid, which allowed an essentially odorless workup.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsWejx7R5QHrVg90H21EOLACvtfcHk0lhIZPkWnscG8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvFOnsrY%253D&md5=4613aa2752eb17ae109aed146dbf8d4e</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjo0611059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo0611059%26sid%3Dliteratum%253Aachs%26aulast%3DMagano%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DClark%26aufirst%3DJ.%2BD.%26aulast%3DNussbaumer%26aufirst%3DT.%26atitle%3D2-%2528Diethylamino%2529%2520ethanethiol%252C%2520a%2520new%2520reagent%2520for%2520the%2520odorless%2520deprotection%2520of%2520aromatic%2520methyl%2520ethers%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2006%26volume%3D71%26issue%3D18%26spage%3D7103%26epage%3D7105%26doi%3D10.1021%2Fjo0611059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerston, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammarstrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berridge, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graslund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span> <span> </span><span class="NLM_article-title">Expressing the human proteome for affinity proteomics: optimizing expression of soluble protein domains and in vivo biotinylation</span>. <i>New Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">525</span>, <span class="refDoi"> DOI: 10.1016/j.nbt.2011.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.nbt.2011.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22027370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot12lsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=515-525&issue=5&author=T.+Keatesauthor=C.+D.+O.+Cooperauthor=P.+Savitskyauthor=C.+K.+Allerstonauthor=C.+Phillipsauthor=M.+Hammarstromauthor=N.+Dagaauthor=G.+Berridgeauthor=P.+Mahajanauthor=N.+A.+Burgess-Brownauthor=S.+Mullerauthor=S.+Graslundauthor=O.+Gileadi&title=Expressing+the+human+proteome+for+affinity+proteomics%3A+optimizing+expression+of+soluble+protein+domains+and+in+vivo+biotinylation&doi=10.1016%2Fj.nbt.2011.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Expressing the human proteome for affinity proteomics: optimising expression of soluble protein domains and in vivo biotinylation</span></div><div class="casAuthors">Keates, Tracy; Cooper, Christopher D. O.; Savitsky, Pavel; Allerston, Charles K.; Phillips, Claire; Hammarstrom, Martin; Daga, Neha; Berridge, Georgina; Mahajan, Pravin; Burgess-Brown, Nicola A.; Muller, Susanne; Graslund, Susanne; Gileadi, Opher</div><div class="citationInfo"><span class="NLM_cas:title">New Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">515-525</span>CODEN:
                <span class="NLM_cas:coden">NBEIBR</span>;
        ISSN:<span class="NLM_cas:issn">1871-6784</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The generation of affinity reagents to large nos. of human proteins depends on the ability to express the target proteins as high-quality antigens.  The Structural Genomics Consortium (SGC) focuses on the prodn. and structure detn. of human proteins.  In a 7-yr period, the SGC has deposited crystal structures of >800 human protein domains, and has addnl. expressed and purified a similar no. of protein domains that have not yet been crystd.  The targets include a diversity of protein domains, with an attempt to provide high coverage of protein families.  The family approach provides an excellent basis for characterizing the selectivity of affinity reagents.  We present a summary of the approaches used to generate purified human proteins or protein domains, a test case demonstrating the ability to rapidly generate new proteins, and an optimization study on the modification of >70 proteins by biotinylation in vivo.  These results provide a unique synergy between large-scale structural projects and the recent efforts to produce a wide coverage of affinity reagents to the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt-7YKiDQFgLVg90H21EOLACvtfcHk0lhIZPkWnscG8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot12lsrg%253D&md5=6d74e519f2df7e04dafd6e2691818939</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.nbt.2011.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbt.2011.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DCooper%26aufirst%3DC.%2BD.%2BO.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAllerston%26aufirst%3DC.%2BK.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DHammarstrom%26aufirst%3DM.%26aulast%3DDaga%26aufirst%3DN.%26aulast%3DBerridge%26aufirst%3DG.%26aulast%3DMahajan%26aufirst%3DP.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DGraslund%26aufirst%3DS.%26aulast%3DGileadi%26aufirst%3DO.%26atitle%3DExpressing%2520the%2520human%2520proteome%2520for%2520affinity%2520proteomics%253A%2520optimizing%2520expression%2520of%2520soluble%2520protein%2520domains%2520and%2520in%2520vivo%2520biotinylation%26jtitle%3DNew%2520Biotechnol.%26date%3D2012%26volume%3D29%26issue%3D5%26spage%3D515%26epage%3D525%26doi%3D10.1016%2Fj.nbt.2011.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0ljOFG8Zjw2Egw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljOFG8Zjw2Egw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0ljOFG8Zjw2Egw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljOFG8Zjw2Egw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljq7Wm80axN5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljq7Wm80axN5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljq7Wm80axN5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Profiling of small molecules by chemical proteomics</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1394</i></span>,  <span class="NLM_fpage">211</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-3341-9_15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2F978-1-4939-3341-9_15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26700051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1agtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1394&publication_year=2016&pages=211-218&author=K.+V.+M.+Huberauthor=G.+Superti-Furga&title=Profiling+of+small+molecules+by+chemical+proteomics&doi=10.1007%2F978-1-4939-3341-9_15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling of small molecules by chemical proteomics</span></div><div class="casAuthors">Huber, Kilian V. M.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1394</span>
        (<span class="NLM_cas:issue">Proteomics in Systems Biology</span>),
    <span class="NLM_cas:pages">211-218</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Chem. proteomics provides a powerful means to gain systems-level insight into the mode of action of small mols. and/or natural products.  In contrast to high-throughput screening efforts which only interrogate selected subproteomes such as kinases and often only consider individual domains, the methodol. presented herein allows for the detn. of the mol. targets of small mols. or drugs in a more relevant physiol. setting.  As such, the compd. of interest is exposed to the entire variety of cellular proteins considering all naturally occurring posttranslational modifications and activation states.  Samples prepd. according to the procedures described in this protocol are compatible with lysates from cultured cell lines, primary cells, or samples from biopsies.  In combination with state-of-the-art mass spectrometry techniques this approach grants access to a comprehensive view of small mol.-target protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqu_eJ-ips8bVg90H21EOLACvtfcHk0lhOIAskwOTWkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1agtrrE&md5=310ab7cb29d9eb4c39b945a5b19b7bd9</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-3341-9_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-3341-9_15%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProfiling%2520of%2520small%2520molecules%2520by%2520chemical%2520proteomics%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2016%26volume%3D1394%26spage%3D211%26epage%3D218%26doi%3D10.1007%2F978-1-4939-3341-9_15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzberg, S. L.</span></span> <span> </span><span class="NLM_article-title">HISAT: a fast spliced aligner with low memory requirements</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">357</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnmeth.3317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25751142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFOnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=357-360&issue=4&author=D.+Kimauthor=B.+Langmeadauthor=S.+L.+Salzberg&title=HISAT%3A+a+fast+spliced+aligner+with+low+memory+requirements&doi=10.1038%2Fnmeth.3317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">HISAT: a fast spliced aligner with low memory requirements</span></div><div class="casAuthors">Kim, Daehwan; Langmead, Ben; Salzberg, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">357-360</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">HISAT (hierarchical indexing for spliced alignment of transcripts) is a highly efficient system for aligning reads from RNA sequencing expts.  HISAT uses an indexing scheme based on the Burrows-Wheeler transform and the Ferragina-Manzini (FM) index, employing two types of indexes for alignment: a whole-genome FM index to anchor each alignment and numerous local FM indexes for very rapid extensions of these alignments.  HISAT's hierarchical index for the human genome contains 48,000 local FM indexes, each representing a genomic region of ∼64,000 bp.  Tests on real and simulated data sets showed that HISAT is the fastest system currently available, with equal or better accuracy than any other method.  Despite its large no. of indexes, HISAT requires only 4.3 gigabytes of memory.  HISAT supports genomes of any size, including those larger than 4 billion bases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKIAQt7RlLl7Vg90H21EOLACvtfcHk0lhOIAskwOTWkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFOnsL0%253D&md5=6a0895120a3b49200f558b4cf810f89f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3317%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%26aulast%3DLangmead%26aufirst%3DB.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26atitle%3DHISAT%253A%2520a%2520fast%2520spliced%2520aligner%2520with%2520low%2520memory%2520requirements%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26issue%3D4%26spage%3D357%26epage%3D360%26doi%3D10.1038%2Fnmeth.3317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span> <span> </span><span class="NLM_article-title">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">930</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1093%2Fbioinformatics%2Fbtt656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24227677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFGqu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=923-930&issue=7&author=Y.+Liaoauthor=G.+K.+Smythauthor=W.+Shi&title=featureCounts%3A+an+efficient+general+purpose+program+for+assigning+sequence+reads+to+genomic+features&doi=10.1093%2Fbioinformatics%2Fbtt656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span></div><div class="casAuthors">Liao, Yang; Smyth, Gordon K.; Shi, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">923-930</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Next-generation sequencing technologies generate millions of short sequence reads, which are usually aligned to a ref. genome.  In many applications, the key information required for downstream anal. is the no. of reads mapping to each genomic feature, for example to each exon or each gene.  The process of counting reads is called read summarization.  Read summarization is required for a great variety of genomic analyses but has so far received relatively little attention in the literature.  Results: We present featureCounts, a read summarization program suitable for counting reads generated from either RNA or genomic DNA sequencing expts. featureCounts implements highly efficient chromosome hashing and feature blocking techniques.  It is considerably faster than existing methods (by an order of magnitude for gene-level summarization) and requires far less computer memory.  It works with either single or paired-end reads and provides a wide range of options appropriate for different sequencing applications.  Availability and implementation: featureCounts is available under GNU General Public License as part of the Subread (http://subread.sourceforge.net) or Rsubread (http://www.bioconductor.org) software packages.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="a3d0cbcae3d4c6cbca8dc6c7d68dc2d68d">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF_vNcz7InHLVg90H21EOLACvtfcHk0lhOIAskwOTWkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFGqu7c%253D&md5=6549e3ff389f50b088acae7972ddf7af</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt656%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DY.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26aulast%3DShi%26aufirst%3DW.%26atitle%3DfeatureCounts%253A%2520an%2520efficient%2520general%2520purpose%2520program%2520for%2520assigning%2520sequence%2520reads%2520to%2520genomic%2520features%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26issue%3D7%26spage%3D923%26epage%3D930%26doi%3D10.1093%2Fbioinformatics%2Fbtt656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Love, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, S.</span></span> <span> </span><span class="NLM_article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">550</span>, <span class="refDoi"> DOI: 10.1186/s13059-014-0550-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fs13059-014-0550-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25516281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVCrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=550&issue=12&author=M.+I.+Loveauthor=W.+Huberauthor=S.+Anders&title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2&doi=10.1186%2Fs13059-014-0550-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span></div><div class="casAuthors">Love, Michael I.; Huber, Wolfgang; Anders, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Genome Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">550/1-550/21, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">GNBLFW</span>;
        ISSN:<span class="NLM_cas:issn">1474-760X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">In comparative high-throughput sequencing assays, a fundamental task is the anal. of count data, such as read counts per gene in RNA-seq, for evidence of systematic changes across exptl. conditions.  Small replicate nos., discreteness, large dynamic range and the presence of outliers require a suitable statistical approach.  We present DESeq2, a method for differential anal. of count data, using shrinkage estn. for dispersions and fold changes to improve stability and interpretability of ests.  This enables a more quant. anal. focused on the strength rather than the mere presence of differential expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-iepNJ6mO27Vg90H21EOLACvtfcHk0lhOIAskwOTWkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVCrsL8%253D&md5=e1052687faf6b297cdf615a14676c5b0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1186%2Fs13059-014-0550-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13059-014-0550-8%26sid%3Dliteratum%253Aachs%26aulast%3DLove%26aufirst%3DM.%2BI.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DAnders%26aufirst%3DS.%26atitle%3DModerated%2520estimation%2520of%2520fold%2520change%2520and%2520dispersion%2520for%2520RNA-seq%2520data%2520with%2520DESeq2%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26issue%3D12%26spage%3D550%26doi%3D10.1186%2Fs13059-014-0550-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabregat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jupe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidiropoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garapati, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haw, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jassal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korninger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milacic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothfels, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamovsky, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varusai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viteri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Eustachio, P.</span></span> <span> </span><span class="NLM_article-title">The reactome pathway knowledgebase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D649</span>– <span class="NLM_lpage">D655</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1093%2Fnar%2Fgkx1132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29145629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGis7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&author=A.+Fabregatauthor=S.+Jupeauthor=L.+Matthewsauthor=K.+Sidiropoulosauthor=M.+Gillespieauthor=P.+Garapatiauthor=R.+Hawauthor=B.+Jassalauthor=F.+Korningerauthor=B.+Mayauthor=M.+Milacicauthor=C.+D.+Rocaauthor=K.+Rothfelsauthor=C.+Sevillaauthor=V.+Shamovskyauthor=S.+Shorserauthor=T.+Varusaiauthor=G.+Viteriauthor=J.+Weiserauthor=G.+M.+Wuauthor=L.+Steinauthor=H.+Hermjakobauthor=P.+D%E2%80%99Eustachio&title=The+reactome+pathway+knowledgebase&doi=10.1093%2Fnar%2Fgkx1132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The reactome pathway knowledgebase</span></div><div class="casAuthors">Fabregat, Antonio; Jupe, Steven; Matthews, Lisa; Sidiropoulos, Konstantinos; Gillespie, Marc; Garapati, Phani; Haw, Robin; Jassal, Bijay; Korninger, Florian; May, Bruce; Milacic, Marija; Roca, Corina Duenas; Rothfels, Karen; Sevilla, Cristoffer; Shamovsky, Veronica; Shorser, Solomon; Varusai, Thawfeek; Viteri, Guilherme; Weiser, Joel; Wu, Guanming; Stein, Lincoln; Hermjakob, Henning; D'eustachio, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D649-D655</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Reactome Knowledgebase provides mol. details of signal transduction, transport, DNA replication, metab., and other cellular processes as an ordered network of mol. transformations-an extended version of a classic metabolic map, in a single consistent data model.  Reactome functions both as an archive of biol. processes and as a tool for discovering unexpected functional relationships in data such as gene expression profiles or somatic mutation catalogs from tumor cells.  To support the continued brisk growth in the size and complexity of Reactome, we have implemented a graph database, improved performance of data anal. tools, and designed new data structures and strategies to boost diagram viewer performance.  To make our website more accessible to human users, we have improved pathway display and navigation by implementing interactive Enhanced High Level Diagrams (EHLDs) with an assocd. icon library, and subpathway highlighting and zooming, in a simplified and reorganized web site with adaptive design.  To encourage re-use of our content, we have enabled export of pathway diagrams as 'PowerPoint' files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouB231U71u1LVg90H21EOLACvtfcHk0lhUwZ6yuws-yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGis7bM&md5=691a9b4fb2ab28dd06b9d012a3e936b7</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1132%26sid%3Dliteratum%253Aachs%26aulast%3DFabregat%26aufirst%3DA.%26aulast%3DJupe%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DSidiropoulos%26aufirst%3DK.%26aulast%3DGillespie%26aufirst%3DM.%26aulast%3DGarapati%26aufirst%3DP.%26aulast%3DHaw%26aufirst%3DR.%26aulast%3DJassal%26aufirst%3DB.%26aulast%3DKorninger%26aufirst%3DF.%26aulast%3DMay%26aufirst%3DB.%26aulast%3DMilacic%26aufirst%3DM.%26aulast%3DRoca%26aufirst%3DC.%2BD.%26aulast%3DRothfels%26aufirst%3DK.%26aulast%3DSevilla%26aufirst%3DC.%26aulast%3DShamovsky%26aufirst%3DV.%26aulast%3DShorser%26aufirst%3DS.%26aulast%3DVarusai%26aufirst%3DT.%26aulast%3DViteri%26aufirst%3DG.%26aulast%3DWeiser%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DG.%2BM.%26aulast%3DStein%26aufirst%3DL.%26aulast%3DHermjakob%26aufirst%3DH.%26aulast%3DD%25E2%2580%2599Eustachio%26aufirst%3DP.%26atitle%3DThe%2520reactome%2520pathway%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD649%26epage%3DD655%26doi%3D10.1093%2Fnar%2Fgkx1132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Isabelle A. Engelberg, Jiuyang Liu, Jacqueline L. Norris-Drouin, Stephanie H. Cholensky, Samantha A. Ottavi, Stephen V. Frye, Tatiana G. Kutateladze, <span class="NLM_string-name hlFld-ContribAuthor">Lindsey I. James</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of an H3K36me3-Derived Peptidomimetic Ligand with Enhanced Affinity for Plant Homeodomain Finger Protein 1 (PHF1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8510-8522. <a href="https://doi.org/10.1021/acs.jmedchem.1c00430" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00430</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00430%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ban%252BH3K36me3-Derived%252BPeptidomimetic%252BLigand%252Bwith%252BEnhanced%252BAffinity%252Bfor%252BPlant%252BHomeodomain%252BFinger%252BProtein%252B1%252B%252528PHF1%252529%26aulast%3DEngelberg%26aufirst%3DIsabelle%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D09032021%26date%3D17052021%26volume%3D64%26issue%3D12%26spage%3D8510%26epage%3D8522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiaoyu Zhang, Marvin Thielert, Haoxin Li, <span class="NLM_string-name hlFld-ContribAuthor">Benjamin F. Cravatt</span>. </span><span class="cited-content_cbyCitation_article-title">SPIN4 Is a Principal Endogenous Substrate of the E3 Ubiquitin Ligase DCAF16. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2021,</strong> <em>60 </em>
                                    (9)
                                     , 637-642. <a href="https://doi.org/10.1021/acs.biochem.1c00067" title="DOI URL">https://doi.org/10.1021/acs.biochem.1c00067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.1c00067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.1c00067%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DSPIN4%252BIs%252Ba%252BPrincipal%252BEndogenous%252BSubstrate%252Bof%252Bthe%252BE3%252BUbiquitin%252BLigase%252BDCAF16%26aulast%3DZhang%26aufirst%3DXiaoyu%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D24012021%26date%3D22022021%26date%3D26022021%26volume%3D60%26issue%3D9%26spage%3D637%26epage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongming  Du</span>, <span class="hlFld-ContribAuthor ">Yinxia  Yan</span>, <span class="hlFld-ContribAuthor ">Si  Xie</span>, <span class="hlFld-ContribAuthor ">Hao  Huang</span>, <span class="hlFld-ContribAuthor ">Xin  Wang</span>, <span class="hlFld-ContribAuthor ">Ray Kit  Ng</span>, <span class="hlFld-ContribAuthor ">Ming-Ming  Zhou</span>, <span class="hlFld-ContribAuthor ">Chengmin  Qian</span>. </span><span class="cited-content_cbyCitation_article-title">Structural mechanism of bivalent histone H3K4me3K9me3 recognition by the Spindlin1/C11orf84 complex in rRNA transcription activation. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-021-21236-x" title="DOI URL">https://doi.org/10.1038/s41467-021-21236-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-021-21236-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-021-21236-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DStructural%252Bmechanism%252Bof%252Bbivalent%252Bhistone%252BH3K4me3K9me3%252Brecognition%252Bby%252Bthe%252BSpindlin1%25252FC11orf84%252Bcomplex%252Bin%252BrRNA%252Btranscription%252Bactivation%26aulast%3DDu%26aufirst%3DYongming%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isabelle A.  Engelberg</span>, <span class="hlFld-ContribAuthor ">Caroline A.  Foley</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Improved methods for targeting epigenetic reader domains of acetylated and methylated lysine. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>63 </em>, 132-144. <a href="https://doi.org/10.1016/j.cbpa.2021.03.002" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.03.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.03.002%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DImproved%252Bmethods%252Bfor%252Btargeting%252Bepigenetic%252Breader%252Bdomains%252Bof%252Bacetylated%252Band%252Bmethylated%252Blysine%26aulast%3DEngelberg%26aufirst%3DIsabelle%2BA.%26date%3D2021%26volume%3D63%26spage%3D132%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dymytrii  Listunov</span>, <span class="hlFld-ContribAuthor ">Brian M.  Linhares</span>, <span class="hlFld-ContribAuthor ">EunGi  Kim</span>, <span class="hlFld-ContribAuthor ">Alyssa  Winkler</span>, <span class="hlFld-ContribAuthor ">Miranda L.  Simes</span>, <span class="hlFld-ContribAuthor ">Sidney  Weaver</span>, <span class="hlFld-ContribAuthor ">Hyo Je  Cho</span>, <span class="hlFld-ContribAuthor ">Alexandrea  Rizo</span>, <span class="hlFld-ContribAuthor ">Sergey  Zolov</span>, <span class="hlFld-ContribAuthor ">Venkateshwar G.  Keshamouni</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>. </span><span class="cited-content_cbyCitation_article-title">Development of potent dimeric inhibitors of GAS41 YEATS domain. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>66 </em><a href="https://doi.org/10.1016/j.chembiol.2021.06.010" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.06.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.06.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.06.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DDevelopment%252Bof%252Bpotent%252Bdimeric%252Binhibitors%252Bof%252BGAS41%252BYEATS%252Bdomain%26aulast%3DListunov%26aufirst%3DDymytrii%26date%3D2021%26volume%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Pierre  Regenass</span>, <span class="hlFld-ContribAuthor ">David  Maurer</span>, <span class="hlFld-ContribAuthor ">Dmytro  Ostrovskyi</span>, <span class="hlFld-ContribAuthor ">Ludwig  Seifert</span>, <span class="hlFld-ContribAuthor ">Johannes  Bacher</span>, <span class="hlFld-ContribAuthor ">Teresa  Burgahn</span>, <span class="hlFld-ContribAuthor ">Tobias  Wagner</span>, <span class="hlFld-ContribAuthor ">Johannes  Seitz</span>, <span class="hlFld-ContribAuthor ">Holger  Greschik</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Bernhard  Breit</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design, Docking and Binding Free Energy Calculations of A366 Derivatives as Spindlin1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (11)
                                     , 5910. <a href="https://doi.org/10.3390/ijms22115910" title="DOI URL">https://doi.org/10.3390/ijms22115910</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22115910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22115910%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DStructure-Based%252BDesign%25252C%252BDocking%252Band%252BBinding%252BFree%252BEnergy%252BCalculations%252Bof%252BA366%252BDerivatives%252Bas%252BSpindlin1%252BInhibitors%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D11%26spage%3D5910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring aromatic cage flexibility of the histone methyllysine reader protein Spindlin1 and its impact on binding mode prediction: an in silico study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>35 </em>
                                    (6)
                                     , 695-706. <a href="https://doi.org/10.1007/s10822-021-00391-9" title="DOI URL">https://doi.org/10.1007/s10822-021-00391-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-021-00391-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-021-00391-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DExploring%252Baromatic%252Bcage%252Bflexibility%252Bof%252Bthe%252Bhistone%252Bmethyllysine%252Breader%252Bprotein%252BSpindlin1%252Band%252Bits%252Bimpact%252Bon%252Bbinding%252Bmode%252Bprediction%25253A%252Ban%252Bin%252Bsilico%252Bstudy%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2021%26date%3D2021%26volume%3D35%26issue%3D6%26spage%3D695%26epage%3D706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yinping  Guo</span>, <span class="hlFld-ContribAuthor ">Xin  Mao</span>, <span class="hlFld-ContribAuthor ">Liang  Xiong</span>, <span class="hlFld-ContribAuthor ">Anjie  Xia</span>, <span class="hlFld-ContribAuthor ">Jing  You</span>, <span class="hlFld-ContribAuthor ">Guifeng  Lin</span>, <span class="hlFld-ContribAuthor ">Chengyong  Wu</span>, <span class="hlFld-ContribAuthor ">Luyi  Huang</span>, <span class="hlFld-ContribAuthor ">Yiwei  Wang</span>, <span class="hlFld-ContribAuthor ">Shengyong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Guided Discovery of a Potent and Selective Cell‐Active Inhibitor of SETDB1 Tudor Domain. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (16)
                                     , 8760-8765. <a href="https://doi.org/10.1002/anie.202017200" title="DOI URL">https://doi.org/10.1002/anie.202017200</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202017200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202017200%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DStructure%2525E2%252580%252590Guided%252BDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BCell%2525E2%252580%252590Active%252BInhibitor%252Bof%252BSETDB1%252BTudor%252BDomain%26aulast%3DGuo%26aufirst%3DYinping%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D16%26spage%3D8760%26epage%3D8765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yinping  Guo</span>, <span class="hlFld-ContribAuthor ">Xin  Mao</span>, <span class="hlFld-ContribAuthor ">Liang  Xiong</span>, <span class="hlFld-ContribAuthor ">Anjie  Xia</span>, <span class="hlFld-ContribAuthor ">Jing  You</span>, <span class="hlFld-ContribAuthor ">Guifeng  Lin</span>, <span class="hlFld-ContribAuthor ">Chengyong  Wu</span>, <span class="hlFld-ContribAuthor ">Luyi  Huang</span>, <span class="hlFld-ContribAuthor ">Yiwei  Wang</span>, <span class="hlFld-ContribAuthor ">Shengyong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Guided Discovery of a Potent and Selective Cell‐Active Inhibitor of SETDB1 Tudor Domain. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (16)
                                     , 8842-8847. <a href="https://doi.org/10.1002/ange.202017200" title="DOI URL">https://doi.org/10.1002/ange.202017200</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202017200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202017200%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DStructure%2525E2%252580%252590Guided%252BDiscovery%252Bof%252Ba%252BPotent%252Band%252BSelective%252BCell%2525E2%252580%252590Active%252BInhibitor%252Bof%252BSETDB1%252BTudor%252BDomain%26aulast%3DGuo%26aufirst%3DYinping%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D16%26spage%3D8842%26epage%3D8847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanli  Sun</span>, <span class="hlFld-ContribAuthor ">Haibo  Lu</span>, <span class="hlFld-ContribAuthor ">Xueyu  Fang</span>, <span class="hlFld-ContribAuthor ">Senhao  Xiao</span>, <span class="hlFld-ContribAuthor ">Feng  Yang</span>, <span class="hlFld-ContribAuthor ">Yantao  Chen</span>, <span class="hlFld-ContribAuthor ">Hongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaopeng  Li</span>, <span class="hlFld-ContribAuthor ">Jing  Lu</span>, <span class="hlFld-ContribAuthor ">Hua  Lin</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Shijie  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a novel 53BP1 inhibitor through AlphaScreen-based high-throughput screening. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>34 </em>, 116054. <a href="https://doi.org/10.1016/j.bmc.2021.116054" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116054</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116054%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bnovel%252B53BP1%252Binhibitor%252Bthrough%252BAlphaScreen-based%252Bhigh-throughput%252Bscreening%26aulast%3DSun%26aufirst%3DYanli%26date%3D2021%26volume%3D34%26spage%3D116054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jenna N.  Beyer</span>, <span class="hlFld-ContribAuthor ">Nicole R.  Raniszewski</span>, <span class="hlFld-ContribAuthor ">George M.  Burslem</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and Opportunities in Epigenetic Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     , 17-42. <a href="https://doi.org/10.1002/cbic.202000459" title="DOI URL">https://doi.org/10.1002/cbic.202000459</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000459%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DAdvances%252Band%252BOpportunities%252Bin%252BEpigenetic%252BChemical%252BBiology%26aulast%3DBeyer%26aufirst%3DJenna%2BN.%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D1%26spage%3D17%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingyu  Liu</span>, <span class="hlFld-ContribAuthor ">Ying  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhihui  Wen</span>, <span class="hlFld-ContribAuthor ">Yan  Hao</span>, <span class="hlFld-ContribAuthor ">Charles A. S.  Banks</span>, <span class="hlFld-ContribAuthor ">Jeffrey J.  Lange</span>, <span class="hlFld-ContribAuthor ">Brian D.  Slaughter</span>, <span class="hlFld-ContribAuthor ">Jay R.  Unruh</span>, <span class="hlFld-ContribAuthor ">Laurence  Florens</span>, <span class="hlFld-ContribAuthor ">Susan M.  Abmayr</span>, <span class="hlFld-ContribAuthor ">Jerry L.  Workman</span>, <span class="hlFld-ContribAuthor ">Michael P.  Washburn</span>. </span><span class="cited-content_cbyCitation_article-title">Driving integrative structural modeling with serial capture affinity purification. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2020,</strong> <em>117 </em>
                                    (50)
                                     , 31861-31870. <a href="https://doi.org/10.1073/pnas.2007931117" title="DOI URL">https://doi.org/10.1073/pnas.2007931117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2007931117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2007931117%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DDriving%252Bintegrative%252Bstructural%252Bmodeling%252Bwith%252Bserial%252Bcapture%252Baffinity%252Bpurification%26aulast%3DLiu%26aufirst%3DXingyu%26date%3D2020%26date%3D2020%26volume%3D117%26issue%3D50%26spage%3D31861%26epage%3D31870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Reiner</span>, <span class="hlFld-ContribAuthor ">Ludwig  Seifert</span>, <span class="hlFld-ContribAuthor ">Caroline  Deck</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Holger  Stark</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics meets GPCR: inhibition of histone H3 methyltransferase (G9a) and histamine H3 receptor for Prader–Willi Syndrome. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-70523-y" title="DOI URL">https://doi.org/10.1038/s41598-020-70523-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-70523-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-70523-y%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DEpigenetics%252Bmeets%252BGPCR%25253A%252Binhibition%252Bof%252Bhistone%252BH3%252Bmethyltransferase%252B%252528G9a%252529%252Band%252Bhistamine%252BH3%252Breceptor%252Bfor%252BPrader%2525E2%252580%252593Willi%252BSyndrome%26aulast%3DReiner%26aufirst%3DDavid%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandra  Cipriano</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic reader domains by chemical biology. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 82-94. <a href="https://doi.org/10.1016/j.cbpa.2020.05.006" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.05.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.05.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DTargeting%252Bepigenetic%252Breader%252Bdomains%252Bby%252Bchemical%252Bbiology%26aulast%3DCipriano%26aufirst%3DAlessandra%26date%3D2020%26volume%3D57%26spage%3D82%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brian M  Linhares</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic protein–protein interactions with small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (14)
                                     , 1305-1326. <a href="https://doi.org/10.4155/fmc-2020-0082" title="DOI URL">https://doi.org/10.4155/fmc-2020-0082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0082%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bepigenetic%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Bwith%252Bsmall-molecule%252Binhibitors%26aulast%3DLinhares%26aufirst%3DBrian%2BM%26date%3D2020%26volume%3D12%26issue%3D14%26spage%3D1305%26epage%3D1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Crystal structures of A-366 with G9a and SPIN1. (A) Structure of A-366 (cyan) in complex with G9a (pink) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NVQ">4NVQ</a>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (B) Structure of A-366 (cyan) in complex with SPIN1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Y">6I8Y</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of SPIN1 Inhibitors Series 1, 2, and 3<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>General conditions: (a) 1,4-dibromobutane, KOH, TBAB, PhMe, 0 °C-reflux, 3 h, 92%; (b) NaOH, MeI, DMSO, rt, 1 h 98%; (c) HNO<sub>3</sub>, Ac<sub>2</sub>O, AcOH, −5 °C, 1 h, 87–92%; (d) Pd/C, H<sub>2</sub>, 9:1 EtOAc:EtOH, rt, 30–40 min, 76–86%; (e) alkyl bromide, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 18 h, 92–97%; (f) 2° amine, KI, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 48 h; (g) C<sub>12</sub>H<sub>25</sub>SH, KOtBu, DMF, 0–50 °C, 2 h, 79–86%; (h) alkyl halide, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 18 h; (i) Zn, AcOH, 110 °C, 2 h, 5–82%.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. NanoBRET assay. Full-length wild-type SPIN1 and histone 3.3 nanobret assay showing in cell target engagement for compound <b>23</b> and <b>25</b>. Graph shows fold-change in milliBRET inits (mBU) after 24 h treatment with compounds at 1 μM. Mean ± SD, <i>n</i> = 3 independent experiments, <i>n</i> ≥ 3 internal replicates. One-Way ANOVA with Dunnett’s posthoc multiple comparison test, ** <i>p</i> < 0.005.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Bidentate SPIN1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>General conditions: (f) 2° amine, KI, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 48 h; (j) Cu, DMF, 60 °C, 2 h; (i) Zn, AcOH, 110 °C, 2 h, 5–82%.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of SPIN1<sub>49–262</sub> in complex with compound <b>35</b> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8B">6I8B</a>). (left) Electrostatic surface potential of SPIN1 with <b>35</b>. (right) Interactions of <b>35</b> (cyan) with SPIN1 (domains 1, 2, and 3 are shown in magenta, green, and yellow respectively).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Human tudor domain phylogenetic tree with SPIN subfamily highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. NanoBRET assay results for compound <b>35</b> and <b>38</b> using full length wild-type SPIN1 with histone 3.3. Graph shows milliBRET units (mBU) after 24 h treatment with 0.05–15 μM of compound <b>35</b> and <b>38</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (left) Heatmap of selected methyl-Lys/Arg reader domains using DSF (thermal shift assay). Scale shows Δ<i>T</i> °C. (right) Percentage enzyme activity of 33 methyltransferases after compound treatment at 10 and 50 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Affinity Probe <b>41</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>General conditions (f) 2° amine, KI, K<sub>2</sub>CO<sub>3</sub>, CH<sub>3</sub>CN, 80 °C, 48 h; (i) Zn, AcOH, 110 °C, 2 h, 63% over 2 steps; (k) Pd/C, H<sub>2</sub>, MeOH, rt, 18 h; 4-(Boc-amino)butyl bromide, K<sub>2</sub>CO<sub>3</sub>, DMF, rt, 48 h; TFA, DCM, rt, 2 h, 12% overall.</p></p></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Chemoproteomic analyses using compound <b>35</b> (top) or <b>38</b> (bottom) as competitors in human squamous carcinoma cell line SCC040 lysates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/medium/jm9b00562_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Heatmap of overlapping genes modulated by compound <b>35</b>, <b>38</b>, and siRNA knockdown in SCC040 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-20/acs.jmedchem.9b00562/20191018/images/large/jm9b00562_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00562&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i89">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85724" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85724" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 76 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenuwein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span> <span> </span><span class="NLM_article-title">Translating the histone code</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>293</i></span> (<span class="NLM_issue">5532</span>),  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&issue=5532&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0liQJEEK2AaTQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26issue%3D5532%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denu, J. M.</span></span> <span> </span><span class="NLM_article-title">Reading the combinatorial histone language</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">574</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=564-574&issue=3&author=Z.+Suauthor=J.+M.+Denu&title=Reading+the+combinatorial+histone+language&doi=10.1021%2Facschembio.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Reading the Combinatorial Histone Language</span></div><div class="casAuthors">Su, Zhangli; Denu, John M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">564-574</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histones are subject to frequent combinatorial post-translational modifications (PTMs), forming a complex chem. "language" that is interpreted by PTM-specific histone-interacting protein modules (reader domains).  These specific interactions are thought to instruct gene expression and downstream biol. functions.  While the majority of studies have focused on individual modifications, the authors' current understanding of the combinatorial PTM patterns on histones is starting to emerge, benefiting from the convergence of multiple technologies.  Here, the authors review the key tech. advances and progress on discovery and characterization of combinatorial histone PTM patterns.  The authors focus on the interactions between reader domains and combinatorial PTMs, which is essential for understanding the mechanism and biol. meaning of establishing and interpreting information embedded in histone PTM patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHpmgsQnFL7rVg90H21EOLACvtfcHk0liQJEEK2AaTQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVagsL%252FK&md5=819be9f59c1c1f0ae1ee09665b2d51da</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DZ.%26aulast%3DDenu%26aufirst%3DJ.%2BM.%26atitle%3DReading%2520the%2520combinatorial%2520histone%2520language%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26issue%3D3%26spage%3D564%26epage%3D574%26doi%3D10.1021%2Facschembio.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teske, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadden, M. K.</span></span> <span> </span><span class="NLM_article-title">Methyllysine binding domains: structural insight and small molecule probe development</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">14</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.04.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.ejmech.2017.04.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28478342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlygsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2017&pages=14-35&author=K.+A.+Teskeauthor=M.+K.+Hadden&title=Methyllysine+binding+domains%3A+structural+insight+and+small+molecule+probe+development&doi=10.1016%2Fj.ejmech.2017.04.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Methyllysine binding domains: Structural insight and small molecule probe development</span></div><div class="casAuthors">Teske, Kelly A.; Hadden, M. Kyle</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14-35</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  A frequent posttranslational modification that regulates gene expression is the mono-, di-, and/or tri- methylation of lysine residues on the histone tails of chromatin.  The recognition of methylated lysine marks is facilitated by specific reader proteins that contain a methyllysine binding domain.  This class of reader proteins has emerged as a focus of epigenetic research due to its crucial role in gene regulation, oncogenesis and other disease pathways.  The design and synthesis of small mols. that target these domains and disrupt reader/histone protein-protein interactions have demonstrated the druggability of methyllysine binding pockets and provided preliminary evidence that their disruption holds therapeutic potential.  In this review, the authors detail the structures of methyllysine binding domains, highlight the primary roles of these reader proteins in both normal and disease states, and describe the current status of small mol. development against these emerging epigenetic regulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8T6OS1rcnRLVg90H21EOLACvtfcHk0li7617pwoSayQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlygsLk%253D&md5=8017ecb111b7b8b05f87b08a90853bec</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.04.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.04.047%26sid%3Dliteratum%253Aachs%26aulast%3DTeske%26aufirst%3DK.%2BA.%26aulast%3DHadden%26aufirst%3DM.%2BK.%26atitle%3DMethyllysine%2520binding%2520domains%253A%2520structural%2520insight%2520and%2520small%2520molecule%2520probe%2520development%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D136%26spage%3D14%26epage%3D35%26doi%3D10.1016%2Fj.ejmech.2017.04.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solter, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knowles, B. B.</span></span> <span> </span><span class="NLM_article-title">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>124</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">503</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=9053325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=1997&pages=493-503&issue=2&author=B.+Ohauthor=S.+Y.+Hwangauthor=D.+Solterauthor=B.+B.+Knowles&title=Spindlin%2C+a+major+maternal+transcript+expressed+in+the+mouse+during+the+transition+from+oocyte+to+embryo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Spindlin, a major maternal transcript expressed in the mouse during the transition from oocyte to embryo</span></div><div class="casAuthors">Oh, Bermseok; Hwang, Sue-Yun; Solter, Davor; Knowles, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">493-503</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists</span>)
        </div><div class="casAbstract">Timely translation of maternal transcriptions and post-translational modification of their gene products control the initial development of preimplantation-stage embryos.  We have isolated and characterized a gene encoding a stage-specific embryonic protein.  This novel gene, spindlin (Spin), is an abundant maternal transcript present in the unfertilized egg and 2-cell, but not 8-cell, stage embryo.  Spin exhibits high homol. to a multicopy gene, Y-linked spermiogenesis-specific transcript (Ssty), and together they form a new gene family expressed during gametogenesis.  We find that spindlin assocs. with the meiotic spindle and is modified by phosphorylation in a cell-cycle-dependent fashion.  Furthermore, it comigrates with the previously described 30 × 103 Mr metaphase complex which is posttranslationally modified during the first mitotic cell cycle.  Our data suggest that spindlin plays a role in cell-cycle regulation during the transition from gamete to embryo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBqamj85CalbVg90H21EOLACvtfcHk0li7617pwoSayQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvVWmsbk%253D&md5=12c37613f401dedd7d68597ef3e63b0b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DB.%26aulast%3DHwang%26aufirst%3DS.%2BY.%26aulast%3DSolter%26aufirst%3DD.%26aulast%3DKnowles%26aufirst%3DB.%2BB.%26atitle%3DSpindlin%252C%2520a%2520major%2520maternal%2520transcript%2520expressed%2520in%2520the%2520mouse%2520during%2520the%2520transition%2520from%2520oocyte%2520to%2520embryo%26jtitle%3DDevelopment%26date%3D1997%26volume%3D124%26issue%3D2%26spage%3D493%26epage%3D503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">Developing spindlin1 small-molecule inhibitors by using protein microarrays</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">750</span>– <span class="NLM_lpage">756</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnchembio.2377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28504676" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=750-756&issue=7&author=N.+Baeauthor=M.+Vivianoauthor=X.+Suauthor=J.+Lvauthor=D.+Chengauthor=C.+Sagumauthor=S.+Castellanoauthor=X.+Baiauthor=C.+Johnsonauthor=M.+I.+Khalilauthor=J.+Shenauthor=K.+Chenauthor=H.+Liauthor=G.+Sbardellaauthor=M.+T.+Bedford&title=Developing+spindlin1+small-molecule+inhibitors+by+using+protein+microarrays&doi=10.1038%2Fnchembio.2377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Spindlin1 small-molecule inhibitors by using protein microarrays</span></div><div class="casAuthors">Bae, Narkhyun; Viviano, Monica; Su, Xiaonan; Lv, Jie; Cheng, Donghang; Sagum, Cari; Castellano, Sabrina; Bai, Xue; Johnson, Claire; Khalil, Mahmoud Ibrahim; Shen, Jianjun; Chen, Kaifu; Li, Haitao; Sbardella, Gianluca; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">750-756</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of inhibitors of methyl- and acetyl-binding domains has provided evidence for the 'druggability' of epigenetic effector mols.  The small-mol. probe, UNC1215, prevents methyl-dependent protein-protein interactions by engaging the arom. cage of MBT domains and, with lower affinity, Tudor domains.  Here, using a library of tagged UNC1215 analogs, we screened a protein-domain microarray of human methyllysine effector mols. to rapidly detect compds. with new binding profiles with either increased or decreased specificity.  Using this approach, we identified a compd. (EML405) that acquired a novel interaction with Tudor-domain-contg. protein Spindlin1 (SPIN1).  Structural studies facilitated the rational synthesis of SPIN1 inhibitors with increased selectivity (EML631-633), which engaged SPIN1 in cells, blocked its ability to 'read' H3K4me3 marks, and inhibited its transcriptional-coactivator activity.  Protein microarrays can thus be used as a platform to 'target-hop' and identify small mols. that bind and compete with domain-motif interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6diBsmZCcm7Vg90H21EOLACvtfcHk0li7617pwoSayQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslKktrc%253D&md5=5ac8aab47d4a67cbe0f7c1f04d4ae410</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2377%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DViviano%26aufirst%3DM.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DX.%26aulast%3DJohnson%26aufirst%3DC.%26aulast%3DKhalil%26aufirst%3DM.%2BI.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DDeveloping%2520spindlin1%2520small-molecule%2520inhibitors%2520by%2520using%2520protein%2520microarrays%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D7%26spage%3D750%26epage%3D756%26doi%3D10.1038%2Fnchembio.2377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragstedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Notterpek, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, G. P.</span></span> <span> </span><span class="NLM_article-title">Functional cloning of SPIN-2, a nuclear anti-apoptotic protein with roles in cell cycle progression</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1507</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2402557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fsj.leu.2402557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=12145692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1WqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=1507-1518&issue=8&author=B.+S.+Fletcherauthor=C.+Dragstedtauthor=L.+Notterpekauthor=G.+P.+Nolan&title=Functional+cloning+of+SPIN-2%2C+a+nuclear+anti-apoptotic+protein+with+roles+in+cell+cycle+progression&doi=10.1038%2Fsj.leu.2402557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Functional cloning of SPIN-2, a nuclear anti-apoptotic protein with roles in cell cycle progression</span></div><div class="casAuthors">Fletcher, B. S.; Dragstedt, C.; Notterpek, L.; Nolan, G. P.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1507-1518</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The balance between hematopoietic cell viability and apoptosis is regulated by exogenous growth factors, however, the mol. mechanisms by which these trophic factors exert their effects remain obscure.  A functional retroviral cDNA library-based screen was employed to identify genes that prevent growth factor withdrawal-mediated apoptosis in the myeloid progenitor cell 32Dcl3.  This approach identified three classes of genes: those with known roles in apoptosis (bcl-XL and ornithine decarboxylase); genes previously identified but not linked directly to apoptotic signaling (O-linked N-acetylglucosamine transferase); and a previously uncharacterized gene we termed SPIN-2.  In 32Dcl3 cells, expression of exogenous SPIN-2 provides 25% protection from apoptosis following growth factor withdrawal compared to controls which show ∼1-2% survival.  SPIN-2 overexpression slows cell growth rates and increases the percentage of cells in G2/M (32% vs control cells at 12%).  Immunolocalization studies indicate that myc-epitope tagged SPIN-2 proteins, which retain their anti-apoptotic function, reside in the nucleus, whereas a C-terminal deletion mutant that loses its anti-apoptotic activity is located in the cytoplasm.  These studies suggest that SPIN-2 is a novel nuclear protein that functions to regulate cell cycle progression and this activity is related to the inhibition of apoptosis following the removal of essential growth factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCApO1iX6F6LVg90H21EOLACvtfcHk0lguTPHkoBiofA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1WqsLk%253D&md5=37dcb6929dfde2efcade6e898fc462b8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2402557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2402557%26sid%3Dliteratum%253Aachs%26aulast%3DFletcher%26aufirst%3DB.%2BS.%26aulast%3DDragstedt%26aufirst%3DC.%26aulast%3DNotterpek%26aufirst%3DL.%26aulast%3DNolan%26aufirst%3DG.%2BP.%26atitle%3DFunctional%2520cloning%2520of%2520SPIN-2%252C%2520a%2520nuclear%2520anti-apoptotic%2520protein%2520with%2520roles%2520in%2520cell%2520cycle%2520progression%26jtitle%3DLeukemia%26date%3D2002%26volume%3D16%26issue%3D8%26spage%3D1507%26epage%3D1518%26doi%3D10.1038%2Fsj.leu.2402557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willmann, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozretic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jilg, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardelmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buettner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span> <span> </span><span class="NLM_article-title">The histone code reader SPIN1 controls RET signaling in liposarcoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">4773</span>– <span class="NLM_lpage">4789</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.18632%2Foncotarget.3000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25749382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4773-4789&issue=7&author=H.+Franzauthor=H.+Greschikauthor=D.+Willmannauthor=L.+Ozreticauthor=C.+A.+Jilgauthor=E.+Wardelmannauthor=M.+Jungauthor=R.+Buettnerauthor=R.+Schule&title=The+histone+code+reader+SPIN1+controls+RET+signaling+in+liposarcoma&doi=10.18632%2Foncotarget.3000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The histone code reader SPIN1 controls RET signaling in liposarcoma</span></div><div class="casAuthors">Franz Henriette; Greschik Holger; Willmann Dominica; Jilg Cordula Annette; Schule Roland; Ozretic Luka; Buettner Reinhard; Wardelmann Eva; Jung Manfred; Jung Manfred; Schule Roland; Schule Roland</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4773-89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The histone code reader Spindlin1 (SPIN1) has been implicated in tumorigenesis and tumor growth, but the underlying molecular mechanisms remain poorly understood.  Here, we show that reducing SPIN1 levels strongly impairs proliferation and increases apoptosis of liposarcoma cells in vitro and in xenograft mouse models.  Combining signaling pathway, genome-wide chromatin binding, and transcriptome analyses, we found that SPIN1 directly enhances expression of GDNF, an activator of the RET signaling pathway, in cooperation with the transcription factor MAZ.  Accordingly, knockdown of SPIN1 or MAZ results in reduced levels of GDNF and activated RET explaining diminished liposarcoma cell proliferation and survival.  In line with these observations, levels of SPIN1, GDNF, activated RET, and MAZ are increased in human liposarcoma compared to normal adipose tissue or lipoma.  Importantly, a mutation of SPIN1 within the reader domain interfering with chromatin binding reduces liposarcoma cell proliferation and survival.  Together, our data describe a molecular mechanism for SPIN1 function in liposarcoma and suggest that targeting SPIN1 chromatin association with small molecule inhibitors may represent a novel therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS5Ma1donXoAsNIRw0GKUoEfW6udTcc2eZc4bVBtwVGObntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnitFCluw%253D%253D&md5=4b404511399ebfdf8a553af0822e522f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3000%26sid%3Dliteratum%253Aachs%26aulast%3DFranz%26aufirst%3DH.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DOzretic%26aufirst%3DL.%26aulast%3DJilg%26aufirst%3DC.%2BA.%26aulast%3DWardelmann%26aufirst%3DE.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DThe%2520histone%2520code%2520reader%2520SPIN1%2520controls%2520RET%2520signaling%2520in%2520liposarcoma%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26issue%3D7%26spage%3D4773%26epage%3D4789%26doi%3D10.18632%2Foncotarget.3000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janecki, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smialek, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotecki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginter-Matuszewska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuczynska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolanowski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitazawa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurpisz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaruzelska, J.</span></span> <span> </span><span class="NLM_article-title">SPIN1 is a proto-oncogene and SPIN3 is a tumor suppressor in human seminoma</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">65</span>),  <span class="NLM_fpage">32466</span>– <span class="NLM_lpage">32477</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.25977</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.18632%2Foncotarget.25977" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30197756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ntVOmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=32466-32477&issue=65&author=D.+M.+Janeckiauthor=M.+Sajekauthor=M.+J.+Smialekauthor=M.+Koteckiauthor=B.+Ginter-Matuszewskaauthor=B.+Kuczynskaauthor=A.+Spikauthor=T.+Kolanowskiauthor=R.+Kitazawaauthor=M.+Kurpiszauthor=J.+Jaruzelska&title=SPIN1+is+a+proto-oncogene+and+SPIN3+is+a+tumor+suppressor+in+human+seminoma&doi=10.18632%2Foncotarget.25977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">SPIN1 is a proto-oncogene and SPIN3 is a tumor suppressor in human seminoma</span></div><div class="casAuthors">Janecki Damian Mikolaj; Sajek Marcin; Smialek Maciej Jerzy; Kotecki Maciej; Ginter-Matuszewska Barbara; Kuczynska Bogna; Spik Anna; Kolanowski Tomasz; Kurpisz Maciej; Jaruzelska Jadwiga; Kotecki Maciej; Kolanowski Tomasz; Kitazawa Riko</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">65</span>),
    <span class="NLM_cas:pages">32466-32477</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SPIN1 is necessary for normal meiotic progression in mammals.  It is overexpressed in human ovarian cancers and some cancer cell lines.  Here, we examined the functional significance and regulation of SPIN1 and SPIN3 in the TCam-2 human seminoma cell line.  We found that while SPIN1 overexpression reduced apoptosis in these cells, SPIN3 overexpression induced it.  Similarly, SPIN1 upregulated and SPIN3 downregulated CYCD1, which is a downstream target of the PI3K/AKT pathway and contributes to apoptosis resistance in cancer cell lines.  It appears that SPIN1 is pro-oncogenic and SPIN3 acts as a tumor suppressor in TCam-2 cells.  To our knowledge, this is the first report of SPIN3 tumor suppressor activity.  However, both SPIN1 and SPIN3 stimulated cell cycle progression.  In addition, using luciferase reporters carrying SPIN1 or SPIN3 mRNA 3'UTRs, we found that PUM1 and PUM2 targeted and repressed SPINs.  We also found that PUM1 itself strongly stimulated apoptosis and moderately slowed cell cycle progression in TCam-2 cells, suggesting that PUM1, like SPIN3, is a tumor suppressor.  Our findings suggest that acting, at least in part, through SPIN1 and SPIN3, PUM proteins contribute to a mechanism promoting normal human male germ cell apoptotic status and thus preventing cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhTXmHb6E5uTC850By6mxpfW6udTcc2eZc4bVBtwVGObntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ntVOmtQ%253D%253D&md5=4b0c4b4f7124d1050ee8598f34fb991a</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.25977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.25977%26sid%3Dliteratum%253Aachs%26aulast%3DJanecki%26aufirst%3DD.%2BM.%26aulast%3DSajek%26aufirst%3DM.%26aulast%3DSmialek%26aufirst%3DM.%2BJ.%26aulast%3DKotecki%26aufirst%3DM.%26aulast%3DGinter-Matuszewska%26aufirst%3DB.%26aulast%3DKuczynska%26aufirst%3DB.%26aulast%3DSpik%26aufirst%3DA.%26aulast%3DKolanowski%26aufirst%3DT.%26aulast%3DKitazawa%26aufirst%3DR.%26aulast%3DKurpisz%26aufirst%3DM.%26aulast%3DJaruzelska%26aufirst%3DJ.%26atitle%3DSPIN1%2520is%2520a%2520proto-oncogene%2520and%2520SPIN3%2520is%2520a%2520tumor%2520suppressor%2520in%2520human%2520seminoma%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26issue%3D65%26spage%3D32466%26epage%3D32477%26doi%3D10.18632%2Foncotarget.25977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Staub, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mennerich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span> <span> </span><span class="NLM_article-title">The Spin/Ssty repeat: a new motif identified in proteins involved in vertebrate development from gamete to embryo</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">RESEARCH0003</span>, <span class="refDoi"> DOI: 10.1186/gb-2001-3-1-research0003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fgb-2001-3-1-research0003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=11806826" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=RESEARCH0003&issue=1&author=E.+Staubauthor=D.+Mennerichauthor=A.+Rosenthal&title=The+Spin%2FSsty+repeat%3A+a+new+motif+identified+in+proteins+involved+in+vertebrate+development+from+gamete+to+embryo&doi=10.1186%2Fgb-2001-3-1-research0003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fgb-2001-3-1-research0003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2001-3-1-research0003%26sid%3Dliteratum%253Aachs%26aulast%3DStaub%26aufirst%3DE.%26aulast%3DMennerich%26aufirst%3DD.%26aulast%3DRosenthal%26aufirst%3DA.%26atitle%3DThe%2520Spin%252FSsty%2520repeat%253A%2520a%2520new%2520motif%2520identified%2520in%2520proteins%2520involved%2520in%2520vertebrate%2520development%2520from%2520gamete%2520to%2520embryo%26jtitle%3DGenome%2520Biol.%26date%3D2002%26volume%3D3%26issue%3D1%26spage%3DRESEARCH0003%26doi%3D10.1186%2Fgb-2001-3-1-research0003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Expression, purification, crystallization and preliminary X-ray analysis of human spindlin1, an ovarian cancer-related protein</span>. <i>Protein Pept. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">205</span>, <span class="refDoi"> DOI: 10.2174/092986606775101661</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.2174%2F092986606775101661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=16472086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=203-205&issue=2&author=F.+Jiangauthor=Q.+Zhaoauthor=L.+Qinauthor=H.+Pangauthor=X.+Peiauthor=Z.+Rao&title=Expression%2C+purification%2C+crystallization+and+preliminary+X-ray+analysis+of+human+spindlin1%2C+an+ovarian+cancer-related+protein&doi=10.2174%2F092986606775101661"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Expression, purification, crystallization and preliminary X-ray analysis of human spindlin 1, an ovarian cancer-related protein</span></div><div class="casAuthors">Jiang, Fuguo; Zhao, Qiang; Qin, Lipeng; Pang, Hai; Pei, Xuetao; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Protein & Peptide Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">203-205</span>CODEN:
                <span class="NLM_cas:coden">PPELEN</span>;
        ISSN:<span class="NLM_cas:issn">0929-8665</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Human spindlin 1 is a newly screened and identified gene product related to ovarian carcinomas and is highly homologous to mouse spindlin.  It is an abundant maternal transcript expressed in the mouse during the transition from oocyte to embryo.  Here, the recombinant human spindlin1 has been overexpressed in Escherichia coli BL21, purified and crystd. using the hanging-drop vapor-diffusion method.  Crystals diffracting to 2.25 Å resoln. were obtained using ammonium sulfate as precipitant.  The crystals belong to the space group P212121, with unit-cell parameters a = 40.7 Å, b = 84.4 Å, c = 136.4 Å, α=β=γ=90°.  Assuming two mols. per asym. unit, the solvent content is calcd. to be 42.4%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8o5Sz0E4XxrVg90H21EOLACvtfcHk0litObSaQTZA_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitlGntr8%253D&md5=e0968bde24c824de9633702979233a39</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2174%2F092986606775101661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986606775101661%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DPang%26aufirst%3DH.%26aulast%3DPei%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DExpression%252C%2520purification%252C%2520crystallization%2520and%2520preliminary%2520X-ray%2520analysis%2520of%2520human%2520spindlin1%252C%2520an%2520ovarian%2520cancer-related%2520protein%26jtitle%3DProtein%2520Pept.%2520Lett.%26date%3D2006%26volume%3D13%26issue%3D2%26spage%3D203%26epage%3D205%26doi%3D10.2174%2F092986606775101661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, P.</span></span> <span> </span><span class="NLM_article-title">Suppression of SPIN1-mediated PI3K-Akt pathway by miR-489 increases chemosensitivity in breast cancer</span>. <i>J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>239</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">472</span>, <span class="refDoi"> DOI: 10.1002/path.4743</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fpath.4743" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27171498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=239&publication_year=2016&pages=459-472&issue=4&author=X.+Chenauthor=Y.+W.+Wangauthor=A.+Y.+Xingauthor=S.+Xiangauthor=D.+B.+Shiauthor=L.+Liuauthor=Y.+X.+Liauthor=P.+Gao&title=Suppression+of+SPIN1-mediated+PI3K-Akt+pathway+by+miR-489+increases+chemosensitivity+in+breast+cancer&doi=10.1002%2Fpath.4743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of SPIN1-mediated PI3K-Akt pathway by miR-489 increases chemosensitivity in breast cancer</span></div><div class="casAuthors">Chen, Xu; Wang, Ya-Wen; Xing, Ai-Yan; Xiang, Shuai; Shi, Duan-Bo; Liu, Lei; Li, Yan-Xiang; Gao, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">239</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">459-472</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">1096-9896</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Drug resistance is one of the major obstacles for improving the prognosis of breast cancer patients.  Increasing evidence has linked the assocn. of aberrantly expressed microRNAs (miRNAs) with tumor development and progression as well as chemoresistance.  Despite recent advances, there is still little known about the potential role and mechanism of miRNAs in breast cancer chemoresistance.  Here we describe that 16 miRNAs were found to be significantly down-regulated and 11 up-regulated in drug-resistant breast cancer tissues compared with drug-sensitive tissues, using a miRNA microarray.  The results also showed miR-489 to be one of the most down-regulated miRNAs in drug-resistant tissues and cell lines, as confirmed by miRNA microarray screening and real-time quant. PCR.  A decrease in miR-489 expression was assocd. with chemoresistance as well as lymph node metastasis, increased tumor size, advanced pTNM stage and poor prognosis in breast cancer.  Functional anal. revealed that miR-489 increased breast cancer chemosensitivity and inhibited cell proliferation, migration and invasion, both in vitro and in vivo.  Furthermore, SPIN1, VAV3, BCL2 and AKT3 were found to be direct targets of miR-489.  SPIN1 was significantly elevated in drug-resistant and metastatic breast cancer tissues and inversely correlated with miR-489 expression.  High expression of SPIN1 was assocd. with higher histol. grade, lymph node metastasis, advanced pTNM stage and pos. progesterone receptor (PR) status.  Increased SPIN1 expression enhanced cell migration and invasion, inhibited apoptosis and partially antagonized the effects of miR-489 in breast cancer.  PIK3CA, AKT, CREB1 and BCL2 in the PI3K-Akt signalling pathway, demonstrated to be elevated in drug-resistant breast cancer tissues, were identified as downstream effectors of SPIN1.  It was further found that either inhibition of SPIN1 or overexpression of miR-489 suppressed the PI3K-Akt signalling pathway.  These data indicate that miR-489 could reverse the chemoresistance of breast cancer via the PI3K-Akt pathway by targeting SPIN1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlYdZmKYimMbVg90H21EOLACvtfcHk0litObSaQTZA_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SmtL7O&md5=fe996113e57b3a6c2c0cce1b15c6ad0c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fpath.4743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.4743%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%2BW.%26aulast%3DXing%26aufirst%3DA.%2BY.%26aulast%3DXiang%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DD.%2BB.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%2BX.%26aulast%3DGao%26aufirst%3DP.%26atitle%3DSuppression%2520of%2520SPIN1-mediated%2520PI3K-Akt%2520pathway%2520by%2520miR-489%2520increases%2520chemosensitivity%2520in%2520breast%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2016%26volume%3D239%26issue%3D4%26spage%3D459%26epage%3D472%26doi%3D10.1002%2Fpath.4743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, D. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. Z.</span></span> <span> </span><span class="NLM_article-title">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span>. <i>Am. J. Transl. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">102</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28123636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=90-102&issue=1&author=X.+J.+Chenauthor=H.+Dongauthor=S.+Liuauthor=L.+Yuauthor=D.+X.+Yanauthor=X.+W.+Yaoauthor=W.+J.+Sunauthor=D.+Z.+Hanauthor=G.+Z.+Gao&title=Long+noncoding+RNA+MHENCR+promotes+melanoma+progression+via+regulating+miR-425%2F489-mediated+PI3K-Akt+pathway"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Long noncoding RNA MHENCR promotes melanoma progression via regulating miR-425/489-mediated PI3K-Akt pathway</span></div><div class="casAuthors">Chen, Xiangjun; Dong, Hao; Liu, Sha; Yu, Li; Yan, Dexiong; Yao, Xingwei; Sun, Weijing; Han, Dezhi; Gao, Guozhen</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Translational Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">90-105, S1-S2</span>CODEN:
                <span class="NLM_cas:coden">AJTRA7</span>;
        ISSN:<span class="NLM_cas:issn">1943-8141</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">Increasing evidences demonstrated that long noncoding RNAs (lncRNAs) are frequently dysregulated and have crit. roles in many tumors.  However, the roles and functional mechanisms of lncRNAs in melanoma remain largely unknown.  In this study, we identified a novel lncRNA MHENCR which was upregulated in melanoma tissues and further upregulated in metastatic melanoma.  Increased expression of MHENCR indicted poor survival of melanoma patients.  Functional expts. revealed that MHENCR knockdown significantly inhibited melanoma cells proliferation, induced cell cycle arrest and apoptosis, and also attenuated melanoma cells migration in vitro.  Furthermore, we identified MHENCR as a competitively endogenous RNA, which specifically bound to miR-425 and miR-489, upregulated their target genes IGF1 and SPIN1 expression, and further activated PI3K-Akt pathway.  Statistically significant correlations were obsd. between MHENCR expression and IGF1 and SPIN1 in melanoma tissues.  In vivo functional expts. further confirmed the pro-growth and pro-metastasis roles of MHENCR.  Collectively, our findings revealed that MHENCR functions as an oncogene in melanoma via activating miR-425/489-mediated PI3K-Akt pathway, and may be a therapeutic target for melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPAKn59fHqdrVg90H21EOLACvtfcHk0lgqAg7S0yyWPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ChsLzK&md5=829b7d67ae7fe250ad4d6576dd3913c6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%2BJ.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DD.%2BX.%26aulast%3DYao%26aufirst%3DX.%2BW.%26aulast%3DSun%26aufirst%3DW.%2BJ.%26aulast%3DHan%26aufirst%3DD.%2BZ.%26aulast%3DGao%26aufirst%3DG.%2BZ.%26atitle%3DLong%2520noncoding%2520RNA%2520MHENCR%2520promotes%2520melanoma%2520progression%2520via%2520regulating%2520miR-425%252F489-mediated%2520PI3K-Akt%2520pathway%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2017%26volume%3D9%26issue%3D1%26spage%3D90%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drago-Ferrante, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentimalli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlisi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Blasio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saliba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldacchino, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degaetano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debono, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caruana-Dingli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grech, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scerri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesoriere, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vento, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Fiore, R.</span></span> <span> </span><span class="NLM_article-title">Suppressive role exerted by microRNA-29b-1–5p in triple negative breast cancer through SPIN1 regulation</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">17</span>),  <span class="NLM_fpage">28939</span>– <span class="NLM_lpage">28958</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.15960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.18632%2Foncotarget.15960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28423652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BC1cvpt1emsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=28939-28958&issue=17&author=R.+Drago-Ferranteauthor=F.+Pentimalliauthor=D.+Carlisiauthor=A.+De+Blasioauthor=C.+Salibaauthor=S.+Baldacchinoauthor=J.+Degaetanoauthor=J.+Debonoauthor=G.+Caruana-Dingliauthor=G.+Grechauthor=C.+Scerriauthor=G.+Tesoriereauthor=A.+Giordanoauthor=R.+Ventoauthor=R.+Di+Fiore&title=Suppressive+role+exerted+by+microRNA-29b-1%E2%80%935p+in+triple+negative+breast+cancer+through+SPIN1+regulation&doi=10.18632%2Foncotarget.15960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Suppressive role exerted by microRNA-29b-1-5p in triple negative breast cancer through SPIN1 regulation</span></div><div class="casAuthors">Drago-Ferrante Rosa; De Blasio Anna; Vento Renza; Di Fiore Riccardo; Pentimalli Francesca; Carlisi Daniela; Saliba Christian; Baldacchino Shawn; Grech Godfrey; Degaetano James; Scerri Christian; Debono Joseph; Caruana-Dingli Gordon; Scerri Christian; Tesoriere Giovanni; Giordano Antonio; Vento Renza; Tesoriere Giovanni; Vento Renza; Di Fiore Riccardo; Giordano Antonio</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">28939-28958</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">MiR-29 family dysregulation occurs in various cancers including breast cancers.  We investigated miR-29b-1 functional role in human triple negative breast cancer (TNBC) the most aggressive breast cancer subtype.  We found that miR-29b-1-5p was downregulated in human TNBC tissues and cell lines.  To assess whether miR-29b-1-5p correlated with TNBC regenerative potential, we evaluated cancer stem cell enrichment in our TNBC cell lines, and found that only MDA-MB-231 and BT-20 produced primary, secondary and tertiary mammospheres, which were progressively enriched in OCT4, NANOG and SOX2 stemness genes.  MiR-29b-1-5p expression inversely correlated with mammosphere stemness potential, and miR-29b-1 ectopic overexpression decreased TNBC cell growth, self-renewal, migration, invasiveness and paclitaxel resistance repressing WNT/βcatenin and AKT signaling pathways and stemness regulators.  We identified SPINDLIN1 (SPIN1) among predicted miR-29b-1-5p targets.  Consistently, SPIN1 was overexpressed in most TNBC tissues and cell lines and negatively correlated with miR-29b-1-5p.  Target site inhibition showed that SPIN1 seems to be directly controlled by miR-29b-1-5p.  Silencing SPIN1 mirrored the effects triggered by miR-29b-1 overexpression, whereas SPIN1 rescue by SPIN1miScript protector, determined the reversal of the molecular effects produced by the mimic-miR-29b-1-5p.  Overall, we show that miR-29b-1 deregulation impacts on multiple oncogenic features of TNBC cells and their renewal potential, acting, at least partly, through SPIN1, and suggest that both these factors should be evaluated as new possible therapeutic targets against TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ0Sy6WJkt3IgvupmecNw_lfW6udTcc2eZR7fJAphcdCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvpt1emsg%253D%253D&md5=08705be8348e656d28e67fd8e86013c9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.15960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.15960%26sid%3Dliteratum%253Aachs%26aulast%3DDrago-Ferrante%26aufirst%3DR.%26aulast%3DPentimalli%26aufirst%3DF.%26aulast%3DCarlisi%26aufirst%3DD.%26aulast%3DDe%2BBlasio%26aufirst%3DA.%26aulast%3DSaliba%26aufirst%3DC.%26aulast%3DBaldacchino%26aufirst%3DS.%26aulast%3DDegaetano%26aufirst%3DJ.%26aulast%3DDebono%26aufirst%3DJ.%26aulast%3DCaruana-Dingli%26aufirst%3DG.%26aulast%3DGrech%26aufirst%3DG.%26aulast%3DScerri%26aufirst%3DC.%26aulast%3DTesoriere%26aufirst%3DG.%26aulast%3DGiordano%26aufirst%3DA.%26aulast%3DVento%26aufirst%3DR.%26aulast%3DDi%2BFiore%26aufirst%3DR.%26atitle%3DSuppressive%2520role%2520exerted%2520by%2520microRNA-29b-1%25E2%2580%25935p%2520in%2520triple%2520negative%2520breast%2520cancer%2520through%2520SPIN1%2520regulation%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26issue%3D17%26spage%3D28939%26epage%3D28958%26doi%3D10.18632%2Foncotarget.15960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span> <span> </span><span class="NLM_article-title">SPIN1 promotes tumorigenesis by blocking the uL18 (universal large ribosomal subunit protein 18)-MDM2-p53 pathway in human cancer</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e31275</span> <span class="refDoi"> DOI: 10.7554/eLife.31275</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.7554%2FeLife.31275" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29547122" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVSjur7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=Z.+L.+Fangauthor=B.+Caoauthor=J.+M.+Liaoauthor=J.+Dengauthor=K.+D.+Plummerauthor=P.+Liaoauthor=T.+Liuauthor=W.+S.+Zhangauthor=K.+Zhangauthor=L.+Liauthor=D.+Margolinauthor=S.+X.+Zengauthor=J.+P.+Xiongauthor=H.+Lu&title=SPIN1+promotes+tumorigenesis+by+blocking+the+uL18+%28universal+large+ribosomal+subunit+protein+18%29-MDM2-p53+pathway+in+human+cancer&doi=10.7554%2FeLife.31275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">SPIN1 promotes tumorigenesis by blocking the uL18 (universal large ribosomal subunit protein 18)-MDM2-p53 pathway in human cancer</span></div><div class="casAuthors">Fang, Ziling; Cao, Bo; Liao, Jun-Ming; Deng, Jun; Plummer, Kevin D.; Liao, Peng; Liu, Tao; Zhang, Wensheng; Zhang, Kun; Li, Li; Margolin, David; Zeng, Shelya X.; Xiong, Jianping; Lu, Hua</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e31275/1-e31275/21</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Ribosomal proteins (RPs) play important roles in modulating the MDM2-p53 pathway.  However, less is known about the upstream regulators of the RPs.  Here, we identify SPIN1 (Spindlin 1) as a novel binding partner of human RPL5/uL18 that is important for this pathway.  SPIN1 ablation activates p53, suppresses cell growth, reduces clonogenic ability, and induces apoptosis of human cancer cells.  Mechanistically, SPIN1 sequesters uL18 in the nucleolus, preventing it from interacting with MDM2, and thereby alleviating uL18-mediated inhibition of MDM2 ubiquitin ligase activity toward p53.  SPIN1 deficiency increases ribosome-free uL18 and uL5 (human RPL11), which are required for SPIN1 depletion-induced p53 activation.  Anal. of cancer genomic databases suggests that SPIN1 is highly expressed in several human cancers, and its overexpression is pos. correlated with poor prognosis in cancer patients.  Altogether, our findings reveal that the oncogenic property of SPIN1 may be attributed to its neg. regulation of uL18, leading to p53 inactivation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoTVn20u1NhrVg90H21EOLACvtfcHk0lgqAg7S0yyWPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVSjur7L&md5=67586bfbe1559f0339a7e60aceae6bd0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.7554%2FeLife.31275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.31275%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DZ.%2BL.%26aulast%3DCao%26aufirst%3DB.%26aulast%3DLiao%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DPlummer%26aufirst%3DK.%2BD.%26aulast%3DLiao%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DW.%2BS.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMargolin%26aufirst%3DD.%26aulast%3DZeng%26aufirst%3DS.%2BX.%26aulast%3DXiong%26aufirst%3DJ.%2BP.%26aulast%3DLu%26aufirst%3DH.%26atitle%3DSPIN1%2520promotes%2520tumorigenesis%2520by%2520blocking%2520the%2520uL18%2520%2528universal%2520large%2520ribosomal%2520subunit%2520protein%252018%2529-MDM2-p53%2520pathway%2520in%2520human%2520cancer%26jtitle%3DeLife%26date%3D2018%26volume%3D7%26doi%3D10.7554%2FeLife.31275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G. H.</span></span> <span> </span><span class="NLM_article-title">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">443</span>, <span class="refDoi"> DOI: 10.1016/j.biopha.2017.06.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.biopha.2017.06.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28666210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=435-443&author=Y.+Liauthor=X.+L.+Maauthor=Y.+P.+Wangauthor=G.+H.+Li&title=miR-489+inhibits+proliferation%2C+cell+cycle+progression+and+induces+apoptosis+of+glioma+cells+via+targeting+SPIN1-mediated+PI3K%2FAKT+pathway&doi=10.1016%2Fj.biopha.2017.06.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">miR-489 inhibits proliferation, cell cycle progression and induces apoptosis of glioma cells via targeting SPIN1-mediated PI3K/AKT pathway</span></div><div class="casAuthors">Li, Yan; Ma, Xiaolin; Wang, Yanpeng; Li, Guohua</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & Pharmacotherapy</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-443</span>CODEN:
                <span class="NLM_cas:coden">BIPHEX</span>;
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">MicroRNA-489 (miR-489), a newly identified tumor-related miRNA, functions as an oncogene or tumor suppressor via regulating growth and metastasis of human cancers.  But, the clin. significance, biol. function and underlying mechanisms of miR-489 in glioma remain rarely known.  Here, we showed that the levels of miR-489 in glioma tissues were notably underexpressed compared to corresponding non-tumor tissues.  In accordance, the relative levels of miR-489 were decreased in glioma cell lines compared with NHA cells.  Kaplan-Meier plots indicated that miR-489 low expressing glioma patients showed a prominent shorter overall survival.  In addn., miR-489 overexpression prohibited proliferation and cell cycle progression, and promoted apoptosis in U251 cells.  While, miR-489 knockdown showed opposite effects on these cellular processes of U87 cells.  In vivo expts. demonstrated that miR-489 restoration reduced the tumor vol. and wt. of s.c. glioma xenografts in nude mice.  Notably, Spindlin 1 (SPIN1) was inversely and directly regulated by miR-489 in glioma cells.  A neg. correlation between the expression of miR-489 and SPIN1 mRNA was confirmed in glioma tissues.  Interestingly, miR-489 inversely modulated activation of PI3K/AKT pathway and expression of downstream targets including p-mTOR, Cyclin D1 and BCL-XL.  SPIN1 re-expression abolished the effects of miR-489 on U251 cells with enhanced activation of PI3K/AKT pathway and malignant phenotype.  Meanwhile, AKT inhibitor MK-2206 blocked activation of PI3K/AKT pathway and resulted in reduced proliferation, cell cycle arrest and increased apoptosis in miR-489 down-regulating U87 cells.  Altogether, our data support that miR-489 loss facilitates malignant phenotype of glioma cells probably via SPIN1-mediated PI3K/AKT pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq39T3IjNfjurVg90H21EOLACvtfcHk0lg4LcfV4mS7rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyht7%252FE&md5=4cd57f05e0bbfec6fd2206244d73c291</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.biopha.2017.06.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biopha.2017.06.058%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DX.%2BL.%26aulast%3DWang%26aufirst%3DY.%2BP.%26aulast%3DLi%26aufirst%3DG.%2BH.%26atitle%3DmiR-489%2520inhibits%2520proliferation%252C%2520cell%2520cycle%2520progression%2520and%2520induces%2520apoptosis%2520of%2520glioma%2520cells%2520via%2520targeting%2520SPIN1-mediated%2520PI3K%252FAKT%2520pathway%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D2017%26volume%3D93%26spage%3D435%26epage%3D443%26doi%3D10.1016%2Fj.biopha.2017.06.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, S.</span></span> <span> </span><span class="NLM_article-title">miR-409 inhibits human non-small-cell lung cancer progression by directly targeting SPIN1</span>. <i>Mol. Ther.--Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">163</span>, <span class="refDoi"> DOI: 10.1016/j.omtn.2018.08.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.omtn.2018.08.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30290307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVGmsrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=154-163&author=Q.+Songauthor=Q.+Jiauthor=J.+Xiaoauthor=F.+Liauthor=L.+Wangauthor=Y.+Chenauthor=Y.+Xuauthor=S.+Jiao&title=miR-409+inhibits+human+non-small-cell+lung+cancer+progression+by+directly+targeting+SPIN1&doi=10.1016%2Fj.omtn.2018.08.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">miR-409 Inhibits Human Non-Small-Cell Lung Cancer Progression by Directly Targeting SPIN1</span></div><div class="casAuthors">Song, Qi; Ji, Quanbo; Xiao, Jingbo; Li, Fang; Wang, Lingxiong; Chen, Yin; Xu, Yameng; Jiao, Shunchang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy--Nucleic Acids</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">154-163</span>CODEN:
                <span class="NLM_cas:coden">MTAOC5</span>;
        ISSN:<span class="NLM_cas:issn">2162-2531</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Lung cancers, the leading cause of cancer mortality worldwide, are characterized by a high metastatic potential.  Growing evidence reveals that Spindlin 1 (SPIN1) is involved in tumor progression and carcinogenesis.  However, the role of SPIN1 in non-small-cell lung cancer (NSCLC) and the mol. mechanisms underlying SPIN1 in human NSCLC remain undetd.  Here we examd. the function of SPIN1 in human NSCLC and found that the expression of SPIN1 was closely correlated with the overall survival and poor prognosis of NSCLC patients.  Aberrant regulation of microRNAs (miRNAs) has an important role in cancer progression.  We revealed that miR-409 inhibits the expression of SPIN1 by binding directly to the 3' UTR of SPIN1 using dual-luciferase reporter assays.  Overexpression of miR-409 significantly suppressed cell migration, growth, and proliferation by inhibiting SPIN1 in vitro and in vivo.  SPIN1 overexpression in miR-409-transfected NSCLC cells effectively rescued the suppression of cell migration, growth, and proliferation regulated by miR-409. miR-409 regulates the PI3K/AKT (protein kinase B) pathway in NSCLC.  Moreover, clin. data showed that NSCLC patients with high levels of miR-409 experienced significantly better survival. miR-409 expression was also neg. assocd. with SPIN1 expression.  Taken together, these findings highlight that the miR-409/SPIN1 axis is a useful pleiotropic regulatory network and could predict the metastatic potential in NSCLC patients early, indicating the possibility that miR-409 and SPIN1 might be attractive prognostic markers for treating NSCLC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFIdBArJUj8rVg90H21EOLACvtfcHk0lg4LcfV4mS7rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVGmsrfP&md5=2cc1f738488ea98da0961d60897f4a87</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.omtn.2018.08.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.omtn.2018.08.020%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DQ.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DJiao%26aufirst%3DS.%26atitle%3DmiR-409%2520inhibits%2520human%2520non-small-cell%2520lung%2520cancer%2520progression%2520by%2520directly%2520targeting%2520SPIN1%26jtitle%3DMol.%2520Ther.--Nucleic%2520Acids%26date%3D2018%26volume%3D13%26spage%3D154%26epage%3D163%26doi%3D10.1016%2Fj.omtn.2018.08.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H. T.</span></span> <span> </span><span class="NLM_article-title">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1101/gad.233239.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1101%2Fgad.233239.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24589551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=622-636&issue=6&author=X.+N.+Suauthor=G.+X.+Zhuauthor=X.+Z.+Dingauthor=S.+Y.+Leeauthor=Y.+L.+Douauthor=B.+Zhuauthor=W.+Wuauthor=H.+T.+Li&title=Molecular+basis+underlying+histone+H3+lysine-arginine+methylation+pattern+readout+by+Spin%2FSsty+repeats+of+Spindlin1&doi=10.1101%2Fgad.233239.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis underlying histone H3 lysine-arginine methylation pattern readout by Spin/Ssty repeats of Spindlin1</span></div><div class="casAuthors">Su, Xiaonan; Zhu, Guixin; Ding, Xiaozhe; Lee, Shirley Y.; Dou, Yali; Zhu, Bing; Wu, Wei; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">622-636</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Histone modification patterns and their combinatorial readout have emerged as a fundamental mechanism for epigenetic regulation.  Here the authors characterized Spindlin1 as a histone effector that senses a cis-tail histone H3 methylation pattern involving trimethyllysine 4 (H3K4me3) and asym. dimethylarginine 8 (H3R8me2a) marks.  Spindlin1 consists of triple tudor-like Spin/Ssty repeats.  Cocrystal structure detn. established concurrent recognition of H3K4me3 and H3R8me2a by Spin/Ssty repeats 2 and 1, resp.  Both H3K4me3 and H3R8me2a are recognized using an "insertion cavity" recognition mode, contributing to a methylation state-specific layer of regulation.  In vivo functional studies suggest that Spindlin1 activates Wnt/β-catenin signaling downstream from protein arginine methyltransferase 2 (PRMT2) and the MLL complex, which together are capable of generating a specific H3 "K4me3-R8me2a" pattern.  Mutagenesis of Spindlin1 reader pockets impairs activation of Wnt target genes.  Taken together, this work connects a histone "lysine-arginine" methylation pattern readout by Spindlin1-to-Wnt signaling at the transcriptional level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2eotrU6XRPrVg90H21EOLACvtfcHk0lgPwOGQ21ElUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlsF2ntr8%253D&md5=6872607b2651d15d16b80a6d33555bbd</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1101%2Fgad.233239.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.233239.113%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DX.%2BN.%26aulast%3DZhu%26aufirst%3DG.%2BX.%26aulast%3DDing%26aufirst%3DX.%2BZ.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DDou%26aufirst%3DY.%2BL.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DH.%2BT.%26atitle%3DMolecular%2520basis%2520underlying%2520histone%2520H3%2520lysine-arginine%2520methylation%2520pattern%2520readout%2520by%2520Spin%252FSsty%2520repeats%2520of%2520Spindlin1%26jtitle%3DGenes%2520Dev.%26date%3D2014%26volume%3D28%26issue%3D6%26spage%3D622%26epage%3D636%26doi%3D10.1101%2Fgad.233239.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, X. T.</span></span> <span> </span><span class="NLM_article-title">Spindlin1 promotes cancer cell proliferation through activation of WNT/TCF-4 signaling</span>. <i>Mol. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">326</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1158/1541-7786.MCR-11-0440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1158%2F1541-7786.MCR-11-0440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22258766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=326-335&issue=3&author=J.+X.+Wangauthor=Q.+Zengauthor=L.+Chenauthor=J.+C.+Duauthor=X.+L.+Yanauthor=H.+F.+Yuanauthor=C.+Zhaiauthor=J.+N.+Zhouauthor=Y.+L.+Jiaauthor=W.+Yueauthor=X.+T.+Pei&title=Spindlin1+promotes+cancer+cell+proliferation+through+activation+of+WNT%2FTCF-4+signaling&doi=10.1158%2F1541-7786.MCR-11-0440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">SPINDLIN1 Promotes Cancer Cell Proliferation through Activation of WNT/TCF-4 Signaling</span></div><div class="casAuthors">Wang, Jing-Xue; Zeng, Quan; Chen, Lin; Du, Ji-Chao; Yan, Xin-Long; Yuan, Hong-Feng; Zhai, Chao; Zhou, Jun-Nian; Jia, Ya-Li; Yue, Wen; Pei, Xue-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">326-335</span>CODEN:
                <span class="NLM_cas:coden">MCROC5</span>;
        ISSN:<span class="NLM_cas:issn">1541-7786</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">SPINDLIN1, a new member of the SPIN/SSTY gene family, was first identified as a gene highly expressed in ovarian cancer cells.  We have previously shown that it is involved in the process of spindle organization and chromosomal stability and plays a role in the development of cancer.  Nevertheless, the mechanisms underlying its oncogenic role are still largely unknown.  Here, we first showed that expression of SPINDLIN1 is upregulated in clin. tumors.  Ectopic expression of SPINDLIN1 promoted cancer cell proliferation and activated WNT/T-cell factor (TCF)-4 signaling.  The Ser84 and Ser99 amino acids within SPINDLIN1 were further identified as the key functional sites in WNT/TCF-4 signaling activation.  Mutation of these two sites of SPINDLIN1 abolished its effects on promoting WNT/TCF-4 signaling and cancer cell proliferation.  We further found that Aurora-A could interact with and phosphorylate SPINDLIN1 at its key functional sites, Ser84 and Ser99, suggesting that phosphorylation of SPINDLIN1 is involved in its oncogenic function.  Collectively, these results suggest that SPINDLIN1, which may be a novel substrate of the Aurora-A kinase, promotes cancer cell growth through WNT/TCF-4 signaling activation.  Mol Cancer Res; 10(3); 326-35.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoie76oMHEUN7Vg90H21EOLACvtfcHk0lgPwOGQ21ElUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktVamsrc%253D&md5=0446631fe80c7b518ce305c394075143</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1541-7786.MCR-11-0440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1541-7786.MCR-11-0440%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%2BX.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDu%26aufirst%3DJ.%2BC.%26aulast%3DYan%26aufirst%3DX.%2BL.%26aulast%3DYuan%26aufirst%3DH.%2BF.%26aulast%3DZhai%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DJ.%2BN.%26aulast%3DJia%26aufirst%3DY.%2BL.%26aulast%3DYue%26aufirst%3DW.%26aulast%3DPei%26aufirst%3DX.%2BT.%26atitle%3DSpindlin1%2520promotes%2520cancer%2520cell%2520proliferation%2520through%2520activation%2520of%2520WNT%252FTCF-4%2520signaling%26jtitle%3DMol.%2520Cancer%2520Res.%26date%3D2012%26volume%3D10%26issue%3D3%26spage%3D326%26epage%3D335%26doi%3D10.1158%2F1541-7786.MCR-11-0440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bartke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vermeulen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Nucleosome-interacting proteins regulated by DNA and histone methylation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>143</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.cell.2010.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21029866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKhs7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2010&pages=470-484&issue=3&author=T.+Bartkeauthor=M.+Vermeulenauthor=B.+Xhemalceauthor=S.+C.+Robsonauthor=M.+Mannauthor=T.+Kouzarides&title=Nucleosome-interacting+proteins+regulated+by+DNA+and+histone+methylation&doi=10.1016%2Fj.cell.2010.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleosome-Interacting Proteins Regulated by DNA and Histone Methylation</span></div><div class="casAuthors">Bartke, Till; Vermeulen, Michiel; Xhemalce, Blerta; Robson, Samuel C.; Mann, Matthias; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-484</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Modifications on histones or DNA recruit proteins that regulate chromatin function.  Here, we use nucleosomes methylated on DNA and on histone H3 in an affinity assay, in conjunction with a SILAC-based proteomic anal., to identify "crosstalk" between these two distinct classes of modification.  Our anal. reveals proteins whose binding to nucleosomes is regulated by methylation of CpGs, H3K4, H3K9, and H3K27 or a combination thereof.  We identify the origin recognition complex (ORC), including LRWD1 as a subunit, to be a methylation-sensitive nucleosome interactor that is recruited cooperatively by DNA and histone methylation.  Other interactors, such as the lysine demethylase Fbxl11/KDM2A, recognize nucleosomes methylated on histones, but their recruitment is disrupted by DNA methylation.  These data establish SILAC nucleosome affinity purifications (SNAP) as a tool for studying the dynamics between different chromatin modifications and provide a modification binding "profile" for proteins regulated by DNA and histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxe7LiuFnuUrVg90H21EOLACvtfcHk0lgPwOGQ21ElUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKhs7rO&md5=c5ab50feb4d60e56f69011f7c5773818</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DBartke%26aufirst%3DT.%26aulast%3DVermeulen%26aufirst%3DM.%26aulast%3DXhemalce%26aufirst%3DB.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DNucleosome-interacting%2520proteins%2520regulated%2520by%2520DNA%2520and%2520histone%2520methylation%26jtitle%3DCell%26date%3D2010%26volume%3D143%26issue%3D3%26spage%3D470%26epage%3D484%26doi%3D10.1016%2Fj.cell.2010.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span> <span> </span><span class="NLM_article-title">Nucleolar protein spindlin1 recognizes H3K4 methylation and stimulates the expression of rRNA genes</span>. <i>EMBO Rep.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">1160</span>– <span class="NLM_lpage">1166</span>, <span class="refDoi"> DOI: 10.1038/embor.2011.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fembor.2011.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21960006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GgtrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=1160-1166&issue=11&author=W.+X.+Wangauthor=Z.+Chenauthor=Z.+Maoauthor=H.+H.+Zhangauthor=X.+J.+Dingauthor=S.+Chenauthor=X.+D.+Zhangauthor=R.+M.+Xuauthor=B.+Zhu&title=Nucleolar+protein+spindlin1+recognizes+H3K4+methylation+and+stimulates+the+expression+of+rRNA+genes&doi=10.1038%2Fembor.2011.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleolar protein Spindlin1 recognizes H3K4 methylation and stimulates the expression of rRNA genes</span></div><div class="casAuthors">Wang, Weixiang; Chen, Zhi; Mao, Zhuo; Zhang, Huihui; Ding, Xiaojun; Chen, She; Zhang, Xiaodong; Xu, Ruiming; Zhu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1160-1166</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tandem Tudor-like domain-contg. protein Spindlin 1 (I) has been reported to be a meiotic spindle-assocd. protein.  Here, the authors report that I was not assocd. with the spindle in mouse embryonic fibroblast cells during mitotic divisions.  In interphase cells, I specifically localized to the nucleoli.  Moreover, I was a histone methylation effector protein that specifically recognized H3K4 methylation.  Finally, I localized to active ribosomal DNA (rDNA) repeats, and facilitated the expression of rRNA genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3x3h2CCy9yrVg90H21EOLACvtfcHk0lje4xYqYjfeEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GgtrjJ&md5=dc1fa70767d6fa557329310bc688ba67</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fembor.2011.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2011.184%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DH.%2BH.%26aulast%3DDing%26aufirst%3DX.%2BJ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DX.%2BD.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DZhu%26aufirst%3DB.%26atitle%3DNucleolar%2520protein%2520spindlin1%2520recognizes%2520H3K4%2520methylation%2520and%2520stimulates%2520the%2520expression%2520of%2520rRNA%2520genes%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26issue%3D11%26spage%3D1160%26epage%3D1166%26doi%3D10.1038%2Fembor.2011.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shanle, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinsky, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridgers, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajewski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbart, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strahl, B. D.</span></span> <span> </span><span class="NLM_article-title">Histone peptide microarray screen of chromo and tudor domains defines new histone lysine methylation interactions</span>. <i>Epigenet. Chromatin</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">12</span>, <span class="refDoi"> DOI: 10.1186/s13072-017-0117-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fs13072-017-0117-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28293301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvVSktrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=12&author=E.+K.+Shanleauthor=S.+A.+Shinskyauthor=J.+B.+Bridgersauthor=N.+Baeauthor=C.+Sagumauthor=K.+Krajewskiauthor=S.+B.+Rothbartauthor=M.+T.+Bedfordauthor=B.+D.+Strahl&title=Histone+peptide+microarray+screen+of+chromo+and+tudor+domains+defines+new+histone+lysine+methylation+interactions&doi=10.1186%2Fs13072-017-0117-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Histone peptide microarray screen of chromo and tudor domains defines new histone lysine methylation interactions</span></div><div class="casAuthors">Shanle, Erin K.; Shinsky, Stephen A.; Bridgers, Joseph B.; Bae, Narkhyun; Sagum, Cari; Krajewski, Krzysztof; Rothbart, Scott B.; Bedford, Mark T.; Strahl, Brian D.</div><div class="citationInfo"><span class="NLM_cas:title">Epigenetics & Chromatin</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12/1-12/11</span>CODEN:
                <span class="NLM_cas:coden">ECPHAK</span>;
        ISSN:<span class="NLM_cas:issn">1756-8935</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Histone posttranslational modifications (PTMs) function to regulate chromatin structure and function in part through the recruitment of effector proteins that harbor specialized "reader" domains.  Despite efforts to elucidate reader domain-PTM interactions, the influence of neighboring PTMs and the target specificity of many reader domains is still unclear.  The aim of this study was to use a high-throughput histone peptide microarray platform to interrogate 83 known and putative histone reader domains from the chromo and Tudor domain families to identify their interactions and characterize the influence of neighboring PTMs on these interactions.  Results: Nearly a quarter of the chromo and Tudor domains screened showed interactions with histone PTMs by peptide microarray, revealing known and several novel methyllysine interactions.  Specifically, we found that the CBX/HP1 chromodomains that recognize H3K9me also recognize H3K23me2/3-a poorly understood histone PTM.  We also obsd. that, in addn. to their interaction with H3K4me3, Tudor domains of the Spindlin family also recognized H4K20me3-a previously uncharacterized interaction.  Several Tudor domains also showed novel interactions with H3K4me as well.  Conclusions: These results provide an important resource for the epigenetics and chromatin community on the interactions of many human chromo and Tudor domains.  They also provide the basis for addnl. studies into the functional significance of the novel interactions that were discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8JuJ85vX7hrVg90H21EOLACvtfcHk0lje4xYqYjfeEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvVSktrY%253D&md5=15522ac8214eaaff9293f255cf85e2b5</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2Fs13072-017-0117-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13072-017-0117-5%26sid%3Dliteratum%253Aachs%26aulast%3DShanle%26aufirst%3DE.%2BK.%26aulast%3DShinsky%26aufirst%3DS.%2BA.%26aulast%3DBridgers%26aufirst%3DJ.%2BB.%26aulast%3DBae%26aufirst%3DN.%26aulast%3DSagum%26aufirst%3DC.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DRothbart%26aufirst%3DS.%2BB.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26atitle%3DHistone%2520peptide%2520microarray%2520screen%2520of%2520chromo%2520and%2520tudor%2520domains%2520defines%2520new%2520histone%2520lysine%2520methylation%2520interactions%26jtitle%3DEpigenet.%2520Chromatin%26date%3D2017%26volume%3D10%26spage%3D12%26doi%3D10.1186%2Fs13072-017-0117-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bae, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Premkumar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sbardella, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedford, M. T.</span></span> <span> </span><span class="NLM_article-title">A transcriptional coregulator, SPIN.DOC, attenuates the coactivator activity of spindlin1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">20808</span>– <span class="NLM_lpage">20817</span>, <span class="refDoi"> DOI: 10.1074/jbc.M117.814913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1074%2Fjbc.M117.814913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29061846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=20808-20817&issue=51&author=N.+Baeauthor=M.+Gaoauthor=X.+Liauthor=T.+Premkumarauthor=G.+Sbardellaauthor=J.+J.+Chenauthor=M.+T.+Bedford&title=A+transcriptional+coregulator%2C+SPIN.DOC%2C+attenuates+the+coactivator+activity+of+spindlin1&doi=10.1074%2Fjbc.M117.814913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A transcriptional coregulator, SPIN·DOC, attenuates the coactivator activity of Spindlin1</span></div><div class="casAuthors">Bae, Narkhyun; Gao, Min; Li, Xu; Premkumar, Tolkappiyan; Sbardella, Gianluca; Chen, Junjie; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">20808-20817</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Spindlin1 (SPIN1) is a transcriptional coactivator with crit. functions in embryonic development and emerging roles in cancer.  SPIN1 harbors three Tudor domains, two of which engage the tail of histone H3 by reading the H3-Lys-4 trimethylation and H3-Arg-8 asym. dimethylation marks.  To gain mechanistic insight into how SPIN1 functions as a transcriptional coactivator, here we purified its interacting proteins.  We identified an uncharacterized protein (C11orf84), which we renamed SPIN1 docking protein (SPIN·DOC), that directly binds SPIN1 and strongly disrupts its histone methylation reading ability, causing it to disassoc. from chromatin.  The Spindlin family of coactivators has five related members (SPIN1, 2A, 2B, 3, and 4), and we found that all of them bind SPIN·DOC.  It has been reported previously that SPIN1 regulates gene expression in the Wnt signaling pathway by directly interacting with transcription factor 4 (TCF4).  We obsd. here that SPIN·DOC assocs. with TCF4 in a SPIN1-dependent manner and dampens SPIN1 coactivator activity in TOPflash reporter assays.  Furthermore, knockdown and overexpression expts. indicated that SPIN·DOC represses the expression of a no. of SPIN1-regulated genes, including those encoding rRNA and the cytokine IL1B.  In conclusion, we have identified SPIN·DOC as a transcriptional repressor that binds SPIN1 and masks its ability to engage the H3-Lys-4 trimethylation activation mark.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_Fa2GbBnnLrVg90H21EOLACvtfcHk0lje4xYqYjfeEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVejsbrE&md5=69c4c937798fe7e27f6234156291217f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M117.814913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M117.814913%26sid%3Dliteratum%253Aachs%26aulast%3DBae%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DPremkumar%26aufirst%3DT.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DA%2520transcriptional%2520coregulator%252C%2520SPIN.DOC%252C%2520attenuates%2520the%2520coactivator%2520activity%2520of%2520spindlin1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26issue%3D51%26spage%3D20808%26epage%3D20817%26doi%3D10.1074%2Fjbc.M117.814913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luise, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span> <span> </span><span class="NLM_article-title">Identification and structure-activity relationship studies of small-molecule inhibitors of the methyllysine reader protein spindlin1</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">2327</span>– <span class="NLM_lpage">2338</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fcmdc.201600362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27634332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWls7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2327-2338&issue=20&author=D.+Robaaauthor=T.+Wagnerauthor=C.+Luiseauthor=L.+Carlinoauthor=J.+McMillanauthor=R.+Flaigauthor=R.+Schuleauthor=M.+Jungauthor=W.+Sippl&title=Identification+and+structure-activity+relationship+studies+of+small-molecule+inhibitors+of+the+methyllysine+reader+protein+spindlin1&doi=10.1002%2Fcmdc.201600362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Structure-Activity Relationship Studies of Small-Molecule Inhibitors of the Methyllysine Reader Protein Spindlin1</span></div><div class="casAuthors">Robaa, Dina; Wagner, Tobias; Luise, Chiara; Carlino, Luca; McMillan, Joel; Flaig, Ralf; Schuele, Roland; Jung, Manfred; Sippl, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2327-2338</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The methyllysine reader protein Spindlin1 has been implicated in the tumorigenesis of several types of cancer and may be an attractive novel therapeutic target.  Small-mol. inhibitors of Spindlin1 should be valuable as chem. probes as well as potential new therapeutics.  We applied an iterative virtual screening campaign, encompassing structure- and ligand-based approaches, to identify potential Spindlin1 inhibitors from databases of com. available compds.  Our in silico studies coupled with in vitro testing were successful in identifying novel Spindlin1 inhibitors.  Several 4-aminoquinazoline and quinazolinethione derivs. were among the active hit compds., which indicated that these scaffolds represent promising lead structures for the development of Spindlin1 inhibitors.  Subsequent lead optimization studies were hence carried out, and numerous derivs. of both lead scaffolds were synthesized.  This resulted in the discovery of novel inhibitors of Spindlin1 and helped explore the structure-activity relationships of these inhibitor series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0z2mZOjLXb7Vg90H21EOLACvtfcHk0lj0TWHDyv5uCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWls7%252FN&md5=4537d16c26a111a351c0e795279b8401</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600362%26sid%3Dliteratum%253Aachs%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DLuise%26aufirst%3DC.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DMcMillan%26aufirst%3DJ.%26aulast%3DFlaig%26aufirst%3DR.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DIdentification%2520and%2520structure-activity%2520relationship%2520studies%2520of%2520small-molecule%2520inhibitors%2520of%2520the%2520methyllysine%2520reader%2520protein%2520spindlin1%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26issue%3D20%26spage%3D2327%26epage%3D2338%26doi%3D10.1002%2Fcmdc.201600362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweis, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pliushchev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maag, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappano, W. N.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of potent and selective inhibitors of histone methyltransferase G9a</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1021/ml400496h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400496h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=205-209&issue=2&author=R.+F.+Sweisauthor=M.+Pliushchevauthor=P.+J.+Brownauthor=J.+Guoauthor=F.+L.+Liauthor=D.+Maagauthor=A.+M.+Petrosauthor=N.+B.+Soniauthor=C.+Tseauthor=M.+Vedadiauthor=M.+R.+Michaelidesauthor=G.+G.+Chiangauthor=W.+N.+Pappano&title=Discovery+and+development+of+potent+and+selective+inhibitors+of+histone+methyltransferase+G9a&doi=10.1021%2Fml400496h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Development of Potent and Selective Inhibitors of Histone Methyltransferase G9a</span></div><div class="casAuthors">Sweis, Ramzi F.; Pliushchev, Marina; Brown, Peter J.; Guo, Jun; Li, Fengling; Maag, David; Petros, Andrew M.; Soni, Nirupama B.; Tse, Chris; Vedadi, Masoud; Michaelides, Michael R.; Chiang, Gary G.; Pappano, William N.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-209</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G9a is a histone lysine methyltransferase responsible for the methylation of histone H3 lysine 9.  The discovery of A-366 arose from a unique diversity screening hit, which was optimized by incorporation of a propyl-pyrrolidine subunit to occupy the enzyme lysine channel.  A-366 (I) is a potent inhibitor of G9a (IC50: 3.3 nM) with greater than 1000-fold selectivity over 21 other methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMvdTiUgx-oLVg90H21EOLACvtfcHk0lj0TWHDyv5uCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKmsA%253D%253D&md5=26affad72f2950510aa9f253e540d575</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fml400496h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400496h%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26aulast%3DPliushchev%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DMaag%26aufirst%3DD.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DTse%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMichaelides%26aufirst%3DM.%2BR.%26aulast%3DChiang%26aufirst%3DG.%2BG.%26aulast%3DPappano%26aufirst%3DW.%2BN.%26atitle%3DDiscovery%2520and%2520development%2520of%2520potent%2520and%2520selective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520G9a%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26issue%3D2%26spage%3D205%26epage%3D209%26doi%3D10.1021%2Fml400496h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span> <span> </span><span class="NLM_article-title">Open access chemical probes for epigenetic targets</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">1901</span>– <span class="NLM_lpage">1917</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.4155%2Ffmc.15.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26397018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1901-1917&issue=14&author=P.+J.+Brownauthor=S.+Muller&title=Open+access+chemical+probes+for+epigenetic+targets&doi=10.4155%2Ffmc.15.127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Open access chemical probes for epigenetic targets</span></div><div class="casAuthors">Brown, Peter J.; Mueller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1901-1917</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: High attrition rates in drug discovery call for new approaches to improve target validation.  Academia is filling gaps, but often lacks the experience and resources of the pharmaceutical industry resulting in poorly characterized tool compds.  Discussion: The SGC has established an open access chem. probe consortium, currently encompassing ten pharmaceutical companies.  One of its mandates is to create well-characterized inhibitors (chem. probes) for epigenetic targets to enable new biol. and target validation for drug development.  Conclusion: Epigenetic probe compds. have proven to be very valuable and have not only spurred a plethora of novel biol. findings, but also provided starting points for clin. trials.  These probes have proven to be crit. complementation to traditional genetic targeting strategies and provided sometimes surprising results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX4t2GBZPCzLVg90H21EOLACvtfcHk0lj0TWHDyv5uCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK&md5=09973fb3a77bee2a6a65e7fca190f2aa</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.127%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DMuller%26aufirst%3DS.%26atitle%3DOpen%2520access%2520chemical%2520probes%2520for%2520epigenetic%2520targets%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26issue%3D14%26spage%3D1901%26epage%3D1917%26doi%3D10.4155%2Ffmc.15.127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blagg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlin, J. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, S. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimshaw, C. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, J. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mader, M. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O'Hagan, R. C</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundstrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toledo-Sherman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walpole, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J</span></span> <span> </span><span class="NLM_article-title">The promise and peril of chemical probes</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1867</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnchembio.1867" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26196764" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=536-541&issue=8&author=C.+H+Arrowsmithauthor=J.+E+Audiaauthor=C.+Austinauthor=J.+Baellauthor=J.+Bennettauthor=J.+Blaggauthor=C.+Bountraauthor=P.+E+Brennanauthor=P.+J+Brownauthor=M.+E+Bunnageauthor=C.+Buser-Doepnerauthor=R.+M+Campbellauthor=A.+J+Carterauthor=P.+Cohenauthor=R.+A+Copelandauthor=B.+Cravattauthor=J.+L+Dahlinauthor=D.+Dhanakauthor=A.+M+Edwardsauthor=M.+Frederiksenauthor=S.+V+Fryeauthor=N.+Grayauthor=C.+E+Grimshawauthor=D.+Hepworthauthor=T.+Howeauthor=K.+V+M+Huberauthor=J.+Jinauthor=S.+Knappauthor=J.+D+Kotzauthor=R.+G+Krugerauthor=D.+Loweauthor=M.+M+Maderauthor=B.+Marsdenauthor=A.+Mueller-Fahrnowauthor=S.+Mullerauthor=R.+C+O%27Haganauthor=J.+P+Overingtonauthor=D.+R+Owenauthor=S.+H+Rosenbergauthor=R.+Rossauthor=B.+Rothauthor=M.+Schapiraauthor=S.+L+Schreiberauthor=B.+Shoichetauthor=M.+Sundstromauthor=G.+Superti-Furgaauthor=J.+Tauntonauthor=L.+Toledo-Shermanauthor=C.+Walpoleauthor=M.+A+Waltersauthor=T.+M+Willsonauthor=P.+Workmanauthor=R.+N+Youngauthor=W.+J+Zuercher&title=The+promise+and+peril+of+chemical+probes&doi=10.1038%2Fnchembio.1867"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The promise and peril of chemical probes</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Audia, James E.; Austin, Christopher; Baell, Jonathan; Bennett, Jonathan; Blagg, Julian; Bountra, Chas; Brennan, Paul E.; Brown, Peter J.; Bunnage, Mark E.; Buser-Doepner, Carolyn; Campbell, Robert M.; Carter, Adrian J.; Cohen, Philip; Copeland, Robert A.; Cravatt, Ben; Dahlin, Jayme L.; Dhanak, Dashyant; Edwards, Aled M.; Frye, Stephen V.; Gray, Nathanael; Grimshaw, Charles E.; Hepworth, David; Howe, Trevor; Huber, Kilian V. M.; Jin, Jian; Knapp, Stefan; Kotz, Joanne D.; Kruger, Ryan G.; Lowe, Derek; Mader, Mary M.; Marsden, Brian; Mueller-Fahrnow, Anke; Muller, Susanne; O'Hagan, Ronan C.; Overington, John P.; Owen, Dafydd R.; Rosenberg, Saul H.; Roth, Brian; Ross, Ruth; Schapira, Matthieu; Schreiber, Stuart L.; Shoichet, Brian; Sundstrom, Michael; Superti-Furga, Giulio; Taunton, Jack; Toledo-Sherman, Leticia; Walpole, Chris; Walters, Michael A.; Willson, Timothy M.; Workman, Paul; Young, Robert N.; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. probes are powerful reagents with increasing impacts on biomedical research.  However, probes of poor quality or that are used incorrectly generate misleading results.  To help address these shortcomings, we will create a community-driven wiki resource to improve quality and convey current best practice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-li4dxq487rVg90H21EOLACvtfcHk0lhF_P8Ei9Pp0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zO&md5=d8c7a7af94921ab4cdcc871487cceadd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1867&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1867%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH%26aulast%3DAudia%26aufirst%3DJ.%2BE%26aulast%3DAustin%26aufirst%3DC.%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DJ.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBrennan%26aufirst%3DP.%2BE%26aulast%3DBrown%26aufirst%3DP.%2BJ%26aulast%3DBunnage%26aufirst%3DM.%2BE%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DCampbell%26aufirst%3DR.%2BM%26aulast%3DCarter%26aufirst%3DA.%2BJ%26aulast%3DCohen%26aufirst%3DP.%26aulast%3DCopeland%26aufirst%3DR.%2BA%26aulast%3DCravatt%26aufirst%3DB.%26aulast%3DDahlin%26aufirst%3DJ.%2BL%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DEdwards%26aufirst%3DA.%2BM%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DFrye%26aufirst%3DS.%2BV%26aulast%3DGray%26aufirst%3DN.%26aulast%3DGrimshaw%26aufirst%3DC.%2BE%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DHowe%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DK.%2BV%2BM%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKotz%26aufirst%3DJ.%2BD%26aulast%3DKruger%26aufirst%3DR.%2BG%26aulast%3DLowe%26aufirst%3DD.%26aulast%3DMader%26aufirst%3DM.%2BM%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%2527Hagan%26aufirst%3DR.%2BC%26aulast%3DOverington%26aufirst%3DJ.%2BP%26aulast%3DOwen%26aufirst%3DD.%2BR%26aulast%3DRosenberg%26aufirst%3DS.%2BH%26aulast%3DRoss%26aufirst%3DR.%26aulast%3DRoth%26aufirst%3DB.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSchreiber%26aufirst%3DS.%2BL%26aulast%3DShoichet%26aufirst%3DB.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DToledo-Sherman%26aufirst%3DL.%26aulast%3DWalpole%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DM.%2BA%26aulast%3DWillson%26aufirst%3DT.%2BM%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DYoung%26aufirst%3DR.%2BN%26aulast%3DZuercher%26aufirst%3DW.%2BJ%26atitle%3DThe%2520promise%2520and%2520peril%2520of%2520chemical%2520probes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26issue%3D8%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnchembio.1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greschik, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgahn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidtkunz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schott, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranauskiene, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schule, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">9</span>), <span class="NLM_elocation-id">e88</span> <span class="refDoi"> DOI: 10.1093/nar/gkw089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1093%2Fnar%2Fgkw089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26893353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&issue=9&author=T.+Wagnerauthor=H.+Greschikauthor=T.+Burgahnauthor=K.+Schmidtkunzauthor=A.+K.+Schottauthor=J.+McMillanauthor=L.+Baranauskieneauthor=Y.+Xiongauthor=O.+Fedorovauthor=J.+Jinauthor=U.+Oppermannauthor=D.+Matulisauthor=R.+Schuleauthor=M.+Jung&title=Identification+of+a+small-molecule+ligand+of+the+epigenetic+reader+protein+Spindlin1+via+a+versatile+screening+platform&doi=10.1093%2Fnar%2Fgkw089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform</span></div><div class="casAuthors">Wagner, Tobias; Greschik, Holger; Burgahn, Teresa; Schmidtkunz, Karin; Schott, Anne-Kathrin; McMillan, Joel; Baranauskiene, Lina; Xiong, Yan; Fedorov, Oleg; Jin, Jian; Oppermann, Udo; Matulis, Daumantas; Schuele, Roland; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e88/1-e88/22</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Epigenetic modifications of histone tails play an essential role in the regulation of eukaryotic transcription.  Writer and eraser enzymes establish and maintain the epigenetic code by creating or removing posttranslational marks.  Specific binding proteins, called readers, recognize the modifications and mediate epigenetic signalling.  Here, we present a versatile assay platform for the investigation of the interaction between Me lysine readers and their ligands.  This can be utilized for the screening of small mol. inhibitors of such protein-protein interactions and the detailed characterization of the inhibition.  Our platform is constructed in a modular way consisting of orthogonal in vitro binding assays for ligand screening and verification of initial hits and biophys., label-free techniques for further kinetic characterization of confirmed ligands.  A stability assay for the investigation of target engagement in a cellular context complements the platform.  We applied the complete evaluation chain to the Tudor domain contg. protein Spindlin1 and established the in vitro test systems for the double Tudor domain of the histone demethylase JMJD2C.  We finally conducted an exploratory screen for inhibitors of the interaction between Spindlin1 and H3K4me3 and identified A366 as the first nanomolar small-mol. ligand of a Tudor domain contg. Me lysine reader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql-bO41bi49bVg90H21EOLACvtfcHk0lhF_P8Ei9Pp0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslCqurvK&md5=3c6536be1ee6f83c5d2849baa02da318</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw089%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DBurgahn%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DSchott%26aufirst%3DA.%2BK.%26aulast%3DMcMillan%26aufirst%3DJ.%26aulast%3DBaranauskiene%26aufirst%3DL.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMatulis%26aufirst%3DD.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520small-molecule%2520ligand%2520of%2520the%2520epigenetic%2520reader%2520protein%2520Spindlin1%2520via%2520a%2520versatile%2520screening%2520platform%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26issue%3D9%26doi%3D10.1093%2Fnar%2Fgkw089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodroofe, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwinn, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">NanoBRET-a novel BRET platform for the analysis of protein-protein interactions</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1797</span>– <span class="NLM_lpage">1804</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00143</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00143" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1797-1804&issue=8&author=T.+Machleidtauthor=C.+C.+Woodroofeauthor=M.+K.+Schwinnauthor=J.+Mendezauthor=M.+B.+Robersauthor=K.+Zimmermanauthor=P.+Ottoauthor=D.+L.+Danielsauthor=T.+A.+Kirklandauthor=K.+V.+Wood&title=NanoBRET-a+novel+BRET+platform+for+the+analysis+of+protein-protein+interactions&doi=10.1021%2Facschembio.5b00143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">NanoBRET-A Novel BRET Platform for the Analysis of Protein-Protein Interactions</span></div><div class="casAuthors">Machleidt, Thomas; Woodroofe, Carolyn C.; Schwinn, Marie K.; Mendez, Jacqui; Robers, Matthew B.; Zimmerman, Kris; Otto, Paul; Daniels, Danette L.; Kirkland, Thomas A.; Wood, Keith V.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1797-1804</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dynamic interactions between proteins comprise a key mechanism for temporal control of cellular function and thus hold promise for development of novel drug therapies.  It remains tech. challenging, however, to quant. characterize these interactions within the biol. relevant context of living cells.  Although, bioluminescence resonance energy transfer (BRET) has often been used for this purpose, its general applicability has been hindered by limited sensitivity and dynamic range.  We have addressed this by combining an extremely bright luciferase (Nanoluc) with a means for tagging intracellular proteins with a long-wavelength fluorophore (HaloTag).  The small size (19 kDa), high emission intensity, and relatively narrow spectrum (460 nm peak intensity) make Nanoluc luciferase well suited as an energy donor.  By selecting an efficient red-emitting fluorophore (635 nm peak intensity) for attachment onto the HaloTag, an overall spectral sepn. exceeding 175 nm was achieved.  This combination of greater light intensity with improved spectral resoln. results in substantially increased detection sensitivity and dynamic range over current BRET technologies.  Enhanced performance is demonstrated using several established model systems, as well as the ability to image BRET in individual cells.  The capabilities are further exhibited in a novel assay developed for analyzing the interactions of bromodomain proteins with chromatin in living cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6-h9HlReNabVg90H21EOLACvtfcHk0lhF_P8Ei9Pp0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFKkt7c%253D&md5=9f5f773b6cd14c40074a2ffbd6472008</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00143%26sid%3Dliteratum%253Aachs%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DWoodroofe%26aufirst%3DC.%2BC.%26aulast%3DSchwinn%26aufirst%3DM.%2BK.%26aulast%3DMendez%26aufirst%3DJ.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DNanoBRET-a%2520novel%2520BRET%2520platform%2520for%2520the%2520analysis%2520of%2520protein-protein%2520interactions%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26issue%3D8%26spage%3D1797%26epage%3D1804%26doi%3D10.1021%2Facschembio.5b00143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berglund, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2221</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnprot.2007.321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=17853878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2212-2221&issue=9&author=F.+H.+Niesenauthor=H.+Berglundauthor=M.+Vedadi&title=The+use+of+differential+scanning+fluorimetry+to+detect+ligand+interactions+that+promote+protein+stability&doi=10.1038%2Fnprot.2007.321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The use of differential scanning fluorimetry to detect ligand interactions that promote protein stability</span></div><div class="casAuthors">Niesen, Frank H.; Berglund, Helena; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2212-2221</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Differential scanning fluorometry (DSF) is a rapid and inexpensive screening method to identify low-mol.-wt. ligands that bind and stabilize purified proteins.  The temp. at which a protein unfolds is measured by an increase in the fluorescence of a dye with affinity for hydrophobic parts of the protein, which are exposed as the protein unfolds.  A simple fitting procedure allows quick calcn. of the transition midpoint; the difference in the temp. of this midpoint in the presence and absence of ligand is related to the binding affinity of the small mol., which can be a low-mol.-wt. compd., a peptide or a nucleic acid.  DSF is best performed using a conventional real-time PCR instrument.  Ligand solns. from a storage plate are added to a soln. of protein and dye, distributed into the wells of the PCR plate and fluorescence intensity measured as the temp. is raised gradually.  Results can be obtained in a single day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomViqp6Tq7krVg90H21EOLACvtfcHk0liG8ZXhTuOpYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb7N&md5=8a74dbb348ab81179614ab4ae5797d94</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.321%26sid%3Dliteratum%253Aachs%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DBerglund%26aufirst%3DH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DThe%2520use%2520of%2520differential%2520scanning%2520fluorimetry%2520to%2520detect%2520ligand%2520interactions%2520that%2520promote%2520protein%2520stability%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26issue%3D9%26spage%3D2212%26epage%3D2221%26doi%3D10.1038%2Fnprot.2007.321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Senisterra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xun, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oyang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of small-molecule antagonists of the H3K9me3 binding to UHRF1 tandem tudor domain</span>. <i>Slas Discov</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">930</span>– <span class="NLM_lpage">940</span>, <span class="refDoi"> DOI: 10.1177/2472555218766278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1177%2F2472555218766278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29562800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOgsr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=930-940&issue=9&author=G.+Senisterraauthor=H.+Y.+Zhuauthor=X.+Luoauthor=H.+L.+Zhangauthor=G.+L.+Xunauthor=C.+L.+Luauthor=W.+Xiaoauthor=T.+Hajianauthor=P.+Loppnauauthor=I.+Chauauthor=F.+L.+Liauthor=A.+Allali-Hassaniauthor=P.+Atadjaauthor=C.+Oyangauthor=E.+Liauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=K.+H.+Zhaoauthor=Z.+T.+Yuauthor=M.+Vedadi&title=Discovery+of+small-molecule+antagonists+of+the+H3K9me3+binding+to+UHRF1+tandem+tudor+domain&doi=10.1177%2F2472555218766278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small-Molecule Antagonists of the H3K9me3 Binding to UHRF1 Tandem Tudor Domain</span></div><div class="casAuthors">Senisterra, Guillermo; Zhu, Hugh Y.; Luo, Xiao; Zhang, Hailong; Xun, Guoliang; Lu, Chunliang; Xiao, Wen; Hajian, Taraneh; Loppnau, Peter; Chau, Irene; Li, Fengling; Allali-Hassani, Abdellah; Atadja, Peter; Oyang, Counde; Li, En; Brown, Peter J.; Arrowsmith, Cheryl H.; Zhao, Kehao; Yu, Zhengtian; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">930-940</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Ubiquitin-like with PHD and RING finger domains 1 (UHRF1) is a multidomain protein that plays a crit. role in maintaining DNA methylation patterns through concurrent recognition of hemimethylated DNA and histone marks by various domains, and recruitment of DNA methyltransferase 1 (DNMT1).  UHRF1 is overexpressed in various cancers, including breast cancer.  The tandem tudor domain (TTD) of UHRF1 specifically and tightly binds to histone H3 di- or trimethylated at lysine 9 (H3K9me2 or H3K9me3, resp.), and this binding is essential for UHRF1 function.  We developed an H3K9me3 peptide displacement assay, which was used to screen a library of 44,000 compds. for small mols. that disrupt the UHRF1-H3K9me3 interaction.  This screen resulted in the identification of NV01, which bound to UHRF1-TTD with a Kd value of 5 μM.  The structure of UHRF1-TTD in complex with NV01 confirmed binding to the H3K9me3-binding pocket.  Limited structure-based optimization of NV01 led to the discovery of NV03 (Kd of 2.4 μM).  These well-characterized small-mol. antagonists of the UHRF1-H3K9me2/3 interaction could be valuable starting chem. matter for developing more potent and cell-active probes toward further characterizing UHRF1 function, with possible applications as anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtaLz2UKwDbLVg90H21EOLACvtfcHk0liG8ZXhTuOpYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOgsr3E&md5=854c27f0152b7a2caebce853e4f8c5c7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1177%2F2472555218766278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555218766278%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DH.%2BY.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%2BL.%26aulast%3DXun%26aufirst%3DG.%2BL.%26aulast%3DLu%26aufirst%3DC.%2BL.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%2BL.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DOyang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DZhao%26aufirst%3DK.%2BH.%26aulast%3DYu%26aufirst%3DZ.%2BT.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520small-molecule%2520antagonists%2520of%2520the%2520H3K9me3%2520binding%2520to%2520UHRF1%2520tandem%2520tudor%2520domain%26jtitle%3DSlas%2520Discov%26date%3D2018%26volume%3D23%26issue%3D9%26spage%3D930%26epage%3D940%26doi%3D10.1177%2F2472555218766278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Chemoproteomics and chemical probes for target discovery</span>. <i>Trends Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1275</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1016/j.tibtech.2018.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.tibtech.2018.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30017093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ygurbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=1275-1286&issue=12&author=G.+Drewesauthor=S.+Knapp&title=Chemoproteomics+and+chemical+probes+for+target+discovery&doi=10.1016%2Fj.tibtech.2018.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoproteomics and Chemical Probes for Target Discovery</span></div><div class="casAuthors">Drewes, Gerard; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biotechnology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1275-1286</span>CODEN:
                <span class="NLM_cas:coden">TRBIDM</span>;
        ISSN:<span class="NLM_cas:issn">0167-7799</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Chem. probes represent versatile tools to validate disease-modifying targets.  However, evaluating the selectivity of chem. probes in complex cellular systems is a major challenge that needs to be addressed to better understand the mode of action of small mols. and the interpretation of their pharmacol. effects.  Chemoproteomics has emerged as a key technol. to characterize the mode of action of pharmacol. modulators such as chem. probes and drugs, and these studies have unraveled the cellular targets of many bioactive compds.  Here we review the role of chem. probes for the validation of new therapeutic targets and their characterization by proteome wide affinity- and activity-based chem. proteomics and recently developed label-free technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBxPBnOO1FYrVg90H21EOLACvtfcHk0liG8ZXhTuOpYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ygurbO&md5=905c056dd30df4438c433e508919f1c6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tibtech.2018.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibtech.2018.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DChemoproteomics%2520and%2520chemical%2520probes%2520for%2520target%2520discovery%26jtitle%3DTrends%2520Biotechnol.%26date%3D2018%26volume%3D36%26issue%3D12%26spage%3D1275%26epage%3D1286%26doi%3D10.1016%2Fj.tibtech.2018.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fik, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strnad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chovanec, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plzak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cada, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hroudova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbanova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paces, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazanek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlcek, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabius, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodet, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smetana, K.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolar, M.</span></span> <span> </span><span class="NLM_article-title">Smooth muscle actin-expressing stromal fibroblasts in head and neck squamous cell carcinoma: increased expression of galectin-1 and induction of poor prognosis factors</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>131</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2508</span>, <span class="refDoi"> DOI: 10.1002/ijc.27550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fijc.27550" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22447203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1ahs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2499-2508&issue=11&author=J.+Valachauthor=Z.+Fikauthor=H.+Strnadauthor=M.+Chovanecauthor=J.+Plzakauthor=Z.+Cadaauthor=P.+Szaboauthor=J.+Sachovaauthor=M.+Hroudovaauthor=M.+Urbanovaauthor=M.+Stefflauthor=J.+Pacesauthor=J.+Mazanekauthor=C.+Vlcekauthor=J.+Betkaauthor=H.+Kaltnerauthor=S.+Andreauthor=H.+J.+Gabiusauthor=R.+Kodetauthor=K.+Smetanaauthor=P.+Galauthor=M.+Kolar&title=Smooth+muscle+actin-expressing+stromal+fibroblasts+in+head+and+neck+squamous+cell+carcinoma%3A+increased+expression+of+galectin-1+and+induction+of+poor+prognosis+factors&doi=10.1002%2Fijc.27550"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Smooth muscle actin-expressing stromal fibroblasts in head and neck squamous cell carcinoma: Increased expression of galectin-1 and induction of poor prognosis factors</span></div><div class="casAuthors">Valach, Jaroslav; Fik, Zdenek; Strnad, Hynek; Chovanec, Martin; Plzak, Jan; Cada, Zdenek; Szabo, Pavol; Sachova, Jana; Hroudova, Miluse; Urbanova, Marketa; Steffl, Martin; Paces, Jan; Mazanek, Jiri; Vlcek, Cestmir; Betka, Jan; Kaltner, Herbert; Andre, Sabine; Gabius, Hans-Joachim; Kodet, Roman; Smetana, Karel; Gal, Peter; Kolar, Michal</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2499-2508</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Tumor stroma is an active part influencing the biol. properties of malignancies via mol. cross-talk.  Cancer-assocd. fibroblasts play a significant role in this interaction.  These cells frequently express smooth muscle actin and can be classified as myofibroblasts.  The adhesion/growth-regulatory lectin galectin-1 is an effector for their generation.  In our study, we set the presence of smooth muscle actin-pos. cancer-assocd. fibroblasts in relation to this endogenous lectin and an in vivo competitor (galectin-3).  In squamous cell carcinomas of head and neck, upregulation of galectin-1 presence was highly significantly correlated to presence of smooth muscle actin-pos. cancer-assocd. fibroblasts in the tumor (p = 4 × 10-8).  To pinpoint further correlations on the mol. level, we applied microarray analyses to the transcription profiles of the corresponding tumors.  Significant correlations of several transcripts were detected with the protein level of galectin-1 in the cancer-assocd. fibroblasts.  These activated genes (MAP3K2, TRIM23, PTPLAD1, FUSIP1, SLC25A40 and SPIN1) are related to known squamous-cell-carcinoma poor-prognosis factors, NF-κB upregulation and splicing downregulation.  These results provide new insights into the significance of presence of myofibroblasts in squamous cell carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzu3UUHbdVz7Vg90H21EOLACvtfcHk0li5DjP3s04sxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1ahs7o%253D&md5=af28a24b9e7d2849eb55d89fa47e7919</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fijc.27550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27550%26sid%3Dliteratum%253Aachs%26aulast%3DValach%26aufirst%3DJ.%26aulast%3DFik%26aufirst%3DZ.%26aulast%3DStrnad%26aufirst%3DH.%26aulast%3DChovanec%26aufirst%3DM.%26aulast%3DPlzak%26aufirst%3DJ.%26aulast%3DCada%26aufirst%3DZ.%26aulast%3DSzabo%26aufirst%3DP.%26aulast%3DSachova%26aufirst%3DJ.%26aulast%3DHroudova%26aufirst%3DM.%26aulast%3DUrbanova%26aufirst%3DM.%26aulast%3DSteffl%26aufirst%3DM.%26aulast%3DPaces%26aufirst%3DJ.%26aulast%3DMazanek%26aufirst%3DJ.%26aulast%3DVlcek%26aufirst%3DC.%26aulast%3DBetka%26aufirst%3DJ.%26aulast%3DKaltner%26aufirst%3DH.%26aulast%3DAndre%26aufirst%3DS.%26aulast%3DGabius%26aufirst%3DH.%2BJ.%26aulast%3DKodet%26aufirst%3DR.%26aulast%3DSmetana%26aufirst%3DK.%26aulast%3DGal%26aufirst%3DP.%26aulast%3DKolar%26aufirst%3DM.%26atitle%3DSmooth%2520muscle%2520actin-expressing%2520stromal%2520fibroblasts%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%253A%2520increased%2520expression%2520of%2520galectin-1%2520and%2520induction%2520of%2520poor%2520prognosis%2520factors%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26issue%3D11%26spage%3D2499%26epage%3D2508%26doi%3D10.1002%2Fijc.27550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heit, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAndrews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nebert, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasiliou, V.</span></span> <span> </span><span class="NLM_article-title">Update of the human and mouse SERPIN gene superfamily</span>. <i>Hum. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">22</span>, <span class="refDoi"> DOI: 10.1186/1479-7364-7-22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2F1479-7364-7-22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24172014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Oruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=22&author=C.+Heitauthor=B.+C.+Jacksonauthor=M.+McAndrewsauthor=M.+W.+Wrightauthor=D.+C.+Thompsonauthor=G.+A.+Silvermanauthor=D.+W.+Nebertauthor=V.+Vasiliou&title=Update+of+the+human+and+mouse+SERPIN+gene+superfamily&doi=10.1186%2F1479-7364-7-22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Update of the human and mouse SERPIN gene superfamily</span></div><div class="casAuthors">Heit, Claire; Jackson, Brian C.; McAndrews, Monica; Wright, Mathew W.; Thompson, David C.; Silverman, Gary A.; Nebert, Daniel W.; Vasiliou, Vasilis</div><div class="citationInfo"><span class="NLM_cas:title">Human Genomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">22/1-22/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">HGUEAT</span>;
        ISSN:<span class="NLM_cas:issn">1479-7364</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The serpin family comprises a structurally similar, yet functionally diverse, set of proteins.  Named originally for their function as serine proteinase inhibitors, many of its members are not inhibitors but rather chaperones, involved in storage, transport, and other roles.  Serpins are found in genomes of all kingdoms, with 36 human protein-coding genes and five pseudogenes.  The mouse has 60 Serpin functional genes, many of which are orthologous to human SERPIN genes and some of which have expanded into multiple paralogous genes.  Serpins are found in tissues throughout the body; whereas most are extracellular, there is a class of intracellular serpins.  Serpins appear to have roles in inflammation, immune function, tumorigenesis, blood clotting, dementia, and cancer metastasis.  Further characterization of these proteins will likely reveal potential biomarkers and therapeutic targets for disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOpQ8_mmIt2rVg90H21EOLACvtfcHk0li5DjP3s04sxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Oruro%253D&md5=fe19bf8237e46aaac2a301a1cae209bc</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2F1479-7364-7-22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-7364-7-22%26sid%3Dliteratum%253Aachs%26aulast%3DHeit%26aufirst%3DC.%26aulast%3DJackson%26aufirst%3DB.%2BC.%26aulast%3DMcAndrews%26aufirst%3DM.%26aulast%3DWright%26aufirst%3DM.%2BW.%26aulast%3DThompson%26aufirst%3DD.%2BC.%26aulast%3DSilverman%26aufirst%3DG.%2BA.%26aulast%3DNebert%26aufirst%3DD.%2BW.%26aulast%3DVasiliou%26aufirst%3DV.%26atitle%3DUpdate%2520of%2520the%2520human%2520and%2520mouse%2520SERPIN%2520gene%2520superfamily%26jtitle%3DHum.%2520Genomics%26date%3D2013%26volume%3D7%26spage%3D22%26doi%3D10.1186%2F1479-7364-7-22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Izuhara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunomura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanri, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aihara, M.</span></span> <span> </span><span class="NLM_article-title">Squamous cell carcinoma antigen 2 (SCCA2, SERPINB4): an emerging biomarker for skin inflammatory diseases</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1102</span>, <span class="refDoi"> DOI: 10.3390/ijms19041102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3390%2Fijms19041102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyjsr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1102&issue=4&author=K.+Izuharaauthor=Y.+Yamaguchiauthor=S.+Ohtaauthor=S.+Nunomuraauthor=Y.+Nanriauthor=Y.+Azumaauthor=N.+Nomuraauthor=Y.+Noguchiauthor=M.+Aihara&title=Squamous+cell+carcinoma+antigen+2+%28SCCA2%2C+SERPINB4%29%3A+an+emerging+biomarker+for+skin+inflammatory+diseases&doi=10.3390%2Fijms19041102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Squamous cell carcinoma antigen 2 (SCCA2, SERPINB4): an emerging biomarker for skin inflammatory diseases</span></div><div class="casAuthors">Izuhara, Kenji; Yamaguchi, Yukie; Ohta, Shoichiro; Nunomura, Satoshi; Nanri, Yasuhiro; Azuma, Yoshinori; Nomura, Noriko; Noguchi, Yasuhiko; Aihara, Michiko</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1102/1-1102/16</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Squamous cell carcinoma antigens 1 and 2 (SCCA1 and 2, SERPIN B3 and B4), members of the ovalbumin serpin (ov-serpin)/clade B serpin family, were originally discovered as tumor-specific antigens and are used as tumor markers for various kinds of squamous cell carcinomas.  Recently, our understanding of the underlying mechanisms of how SCCA1/2 enhance tumor growth has greatly increased.  Moreover, it has been shown that SCCA1/2 are involved in the pathogenesis of several inflammatory diseases: asthma, psoriasis, and atopic dermatitis (AD).  IL-22 and IL-17, signature cytokines of type 17 inflammation, as well as IL-4 and IL-13, signature cytokines of type 2 inflammation, both of which are pos. correlated with the pathogenesis of psoriasis and allergic diseases, resp., can induce expression of SCCA1/2 in airway epithelial cells and/or keratinocytes, leading to high expression of SCCA1/2 in these diseases.  Based on these findings, several trials have been performed to examine the potential of applying SCCA1/2 to biomarkers for these diseases.  The findings show that SCCA2 is useful to aid diagnosis, est. clin. severity and disease type, and assess responses to treatment in psoriasis and AD.  These results suggest that SCCA2 has emerged as a novel biomarker for skin inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZum4NHFRlrVg90H21EOLACvtfcHk0li5DjP3s04sxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyjsr3J&md5=d4ed23ffd3f3884e70d6290f944f54eb</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Fijms19041102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19041102%26sid%3Dliteratum%253Aachs%26aulast%3DIzuhara%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DS.%26aulast%3DNunomura%26aufirst%3DS.%26aulast%3DNanri%26aufirst%3DY.%26aulast%3DAzuma%26aufirst%3DY.%26aulast%3DNomura%26aufirst%3DN.%26aulast%3DNoguchi%26aufirst%3DY.%26aulast%3DAihara%26aufirst%3DM.%26atitle%3DSquamous%2520cell%2520carcinoma%2520antigen%25202%2520%2528SCCA2%252C%2520SERPINB4%2529%253A%2520an%2520emerging%2520biomarker%2520for%2520skin%2520inflammatory%2520diseases%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26issue%3D4%26spage%3D1102%26doi%3D10.3390%2Fijms19041102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizane, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugihara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwatsuki, K.</span></span> <span> </span><span class="NLM_article-title">Expression of serine protease inhibitors in epidermal keratinocytes is increased by calcium but not 1,25-dihydroxyvitamin D-3 or retinoic acid</span>. <i>Br. J. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>176</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1525</span>– <span class="NLM_lpage">1532</span>, <span class="refDoi"> DOI: 10.1111/bjd.15153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1111%2Fbjd.15153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27797397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsVKksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2017&pages=1525-1532&issue=6&author=M.+Kobashiauthor=S.+Morizaneauthor=S.+Sugimotoauthor=S.+Sugiharaauthor=K.+Iwatsuki&title=Expression+of+serine+protease+inhibitors+in+epidermal+keratinocytes+is+increased+by+calcium+but+not+1%2C25-dihydroxyvitamin+D-3+or+retinoic+acid&doi=10.1111%2Fbjd.15153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of serine protease inhibitors in epidermal keratinocytes is increased by calcium but not 1,25-dihydroxyvitamin D3 or retinoic acid</span></div><div class="casAuthors">Kobashi, M.; Morizane, S.; Sugimoto, S.; Sugihara, S.; Iwatsuki, K.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Dermatology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1525-1532</span>CODEN:
                <span class="NLM_cas:coden">BJDEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0007-0963</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Background : In human skin, the serine proteases kallikrein-related peptidase (KLK)5 and KLK7 degrade corneodesmosome proteins, leading to desquamation.  Serine protease activity of the skin is tightly regulated by the interplay between such proteases and serine protease inhibitors, including lymphoepithelial Kazal-type related inhibitor (LEKTI), encoded by SPINK5; secretory leukocyte peptidase inhibitor (SLPI); and elafin.  Expression of KLK5 and KLK7 is controlled and upregulated by stimulants such as calcium, 1,25-dihydroxyvitamin D3 [1,25(OH)2VD3] and retinoic acid (RA).  Objectives : To understand the effect of calcium, 1,25(OH)2VD3 and RA on the expression of serine protease inhibitors in epidermal keratinocytes.  Methods : We stimulated normal human epidermal keratinocytes (NHEKs) with high calcium, 1,25(OH)2VD3 or RA, and then analyzed the expression of serine protease inhibitors using quant. real-time polymerase chain reaction, ELISA and immunocytofluorescence.  We also analyzed trypsin- and chymotrypsin-like serine protease activities in stimulated NHEKs.  Results : High calcium, but not 1,25(OH)2VD3 or RA, significantly induced the expression of LEKTI, SLPI and elafin at both transcript and protein levels in NHEKs.  These inductions were time- and dose-dependent.  The activities of trypsin- and chymotrypsin-like serine proteases were significantly up- and downregulated by high calcium, resp., in NHEKs.  Conclusions : High calcium, but not 1,25(OH)2VD3 or RA, increases the expression of serine protease inhibitors in epidermal keratinocytes.  Our findings contribute to the understanding of the mechanisms by which serine protease activities are regulated by serine proteases and related inhibitors in epidermal keratinocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonLwAAqBLJFrVg90H21EOLACvtfcHk0ljQht9Bp-21QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsVKksLY%253D&md5=681c599c77587b7b97699f38caa251dd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1111%2Fbjd.15153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjd.15153%26sid%3Dliteratum%253Aachs%26aulast%3DKobashi%26aufirst%3DM.%26aulast%3DMorizane%26aufirst%3DS.%26aulast%3DSugimoto%26aufirst%3DS.%26aulast%3DSugihara%26aufirst%3DS.%26aulast%3DIwatsuki%26aufirst%3DK.%26atitle%3DExpression%2520of%2520serine%2520protease%2520inhibitors%2520in%2520epidermal%2520keratinocytes%2520is%2520increased%2520by%2520calcium%2520but%2520not%25201%252C25-dihydroxyvitamin%2520D-3%2520or%2520retinoic%2520acid%26jtitle%3DBr.%2520J.%2520Dermatol.%26date%3D2017%26volume%3D176%26issue%3D6%26spage%3D1525%26epage%3D1532%26doi%3D10.1111%2Fbjd.15153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprasad, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinker, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericksen, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hershey, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khurana Hershey, G. K.</span></span> <span> </span><span class="NLM_article-title">SERPINB3/B4 contributes to early inflammation and barrier dysfunction in an experimental murine model of atopic dermatitis</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>135</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1038/jid.2014.353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fjid.2014.353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25111616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2015&pages=160-169&issue=1&author=U.+Sivaprasadauthor=K.+G.+Kinkerauthor=M.+B.+Ericksenauthor=M.+Lindseyauthor=A.+M.+Gibsonauthor=S.+A.+Bassauthor=N.+S.+Hersheyauthor=J.+Y.+Dengauthor=M.+Medvedovicauthor=G.+K.+Khurana+Hershey&title=SERPINB3%2FB4+contributes+to+early+inflammation+and+barrier+dysfunction+in+an+experimental+murine+model+of+atopic+dermatitis&doi=10.1038%2Fjid.2014.353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">SERPINB3/B4 Contributes to Early Inflammation and Barrier Dysfunction in an Experimental Murine Model of Atopic Dermatitis</span></div><div class="casAuthors">Sivaprasad, Umasundari; Kinker, Kayla G.; Ericksen, Mark B.; Lindsey, Mark; Gibson, Aaron M.; Bass, Stacey A.; Hershey, Nicolas S.; Deng, Jingyuan; Medvedovic, Mario; Khurana Hershey, Gurjit K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160-169</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Serine proteases are crit. for epidermal barrier homeostasis, and their aberrant expression and/or activity is assocd. with chronic skin diseases.  Elevated levels of the serine protease inhibitors SERPINB3 and SERPINB4 are seen in patients with atopic dermatitis and psoriasis.  However, their mechanistic role in the skin is unknown.  To evaluate the contribution of Serpinb3a (mouse homolog of SERPINB3 and SERPINB4) in atopic dermatitis, we examd. the effect of topical Aspergillus fumigatus ext. exposure in wild-type and Serpinb3a-null mice on transepidermal water loss (TEWL), sensitization, and inflammation.  Allergen exposure induced Serpinb3a expression in the skin, along with increased TEWL, epidermal thickness, and skin inflammation, all of which were attenuated in the absence of Serpinb3a.  Attenuated TEWL correlated with decreased expression of the pro-inflammatory marker S100A8.  Silencing of SERPINB3/B4 in human keratinocytes decreased S100A8 expression, supporting a role for SERPINB3/B4 in the initiation of the acute inflammatory response.  RNA-seq anal. following allergen exposure identified a network of pro-inflammatory genes induced in wild-type mice that was absent in Serpinb3a-null mice.  In conclusion, Serpinb3a deficiency attenuates barrier dysfunction and the early inflammatory response following cutaneous allergen exposure, supporting a role for Serpinb3a (mice) and SERPINB3/B4 (humans) early in atopic dermatitis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs1Alg3gb_0LVg90H21EOLACvtfcHk0ljQht9Bp-21QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrzN&md5=9221221bbccc1692ba5c8f6b2b1cda0f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fjid.2014.353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fjid.2014.353%26sid%3Dliteratum%253Aachs%26aulast%3DSivaprasad%26aufirst%3DU.%26aulast%3DKinker%26aufirst%3DK.%2BG.%26aulast%3DEricksen%26aufirst%3DM.%2BB.%26aulast%3DLindsey%26aufirst%3DM.%26aulast%3DGibson%26aufirst%3DA.%2BM.%26aulast%3DBass%26aufirst%3DS.%2BA.%26aulast%3DHershey%26aufirst%3DN.%2BS.%26aulast%3DDeng%26aufirst%3DJ.%2BY.%26aulast%3DMedvedovic%26aufirst%3DM.%26aulast%3DKhurana%2BHershey%26aufirst%3DG.%2BK.%26atitle%3DSERPINB3%252FB4%2520contributes%2520to%2520early%2520inflammation%2520and%2520barrier%2520dysfunction%2520in%2520an%2520experimental%2520murine%2520model%2520of%2520atopic%2520dermatitis%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D2015%26volume%3D135%26issue%3D1%26spage%3D160%26epage%3D169%26doi%3D10.1038%2Fjid.2014.353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheshadri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, W. X.</span></span> <span> </span><span class="NLM_article-title">SERPINB3 and B4: From biochemistry to biology</span>. <i>Semin. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">170</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1016/j.semcdb.2016.09.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.semcdb.2016.09.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=27637160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFertrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2017&pages=170-177&author=Y.+Sunauthor=N.+Sheshadriauthor=W.+X.+Zong&title=SERPINB3+and+B4%3A+From+biochemistry+to+biology&doi=10.1016%2Fj.semcdb.2016.09.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">SERPINB3 and B4: From biochemistry to biology</span></div><div class="casAuthors">Sun, Yu; Sheshadri, Namratha; Zong, Wei-Xing</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cell & Developmental Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">170-177</span>CODEN:
                <span class="NLM_cas:coden">SCDBFX</span>;
        ISSN:<span class="NLM_cas:issn">1084-9521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Human SERPINB3 and SERPINB4 are evolutionary duplicated serine/cysteine protease inhibitors.  Genomic anal. indicates that these paralogous genes were encoded from independent loci arising from tandem gene duplication.  Although the two mols. share 92% identity of their amino acid sequences, they are distinct in the Reactive Center Loop (RCL) including a hinge region and catalytic sequences which accounts for altered substrate specificity.  Elevated expression of the two mols. has been reported to contribute to numerous pathol. conditions such as inflammatory diseases and cancer.  In this review, we focus on summarizing the biochem. features of SERPINB3/B4 and discussing the mechanistic basis for their biol. functions and implications in human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbrWqLdYFTOLVg90H21EOLACvtfcHk0ljQht9Bp-21QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFertrfK&md5=3da7b9e7a3f737c9212949b17d4ad48f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.semcdb.2016.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcdb.2016.09.005%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DSheshadri%26aufirst%3DN.%26aulast%3DZong%26aufirst%3DW.%2BX.%26atitle%3DSERPINB3%2520and%2520B4%253A%2520From%2520biochemistry%2520to%2520biology%26jtitle%3DSemin.%2520Cell%2520Dev.%2520Biol.%26date%3D2017%26volume%3D62%26spage%3D170%26epage%3D177%26doi%3D10.1016%2Fj.semcdb.2016.09.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikitakis, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaisuparat, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwell-Wild, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gliozzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moutsopoulos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warburton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahl, S. M.</span></span> <span> </span><span class="NLM_article-title">Secretory leukocyte protease inhibitor (SLPI) expression and tumor invasion in oral squamous cell carcinoma</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>178</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">2866</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1016/j.ajpath.2011.02.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.ajpath.2011.02.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21641406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslOntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2011&pages=2866-2878&issue=6&author=J.+Wenauthor=N.+G.+Nikitakisauthor=R.+Chaisuparatauthor=T.+Greenwell-Wildauthor=M.+Gliozziauthor=W.+W.+Jinauthor=A.+Adliauthor=N.+Moutsopoulosauthor=T.+X.+Wuauthor=G.+Warburtonauthor=S.+M.+Wahl&title=Secretory+leukocyte+protease+inhibitor+%28SLPI%29+expression+and+tumor+invasion+in+oral+squamous+cell+carcinoma&doi=10.1016%2Fj.ajpath.2011.02.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Secretory leukocyte protease inhibitor (SLPI) expression and tumor invasion in oral squamous cell carcinoma</span></div><div class="casAuthors">Wen, Jie; Nikitakis, Nikolaos G.; Chaisuparat, Risa; Greenwell-Wild, Teresa; Gliozzi, Maria; Jin, Wenwen; Adli, Azita; Moutsopoulos, Niki; Wu, Tanxia; Warburton, Gary; Wahl, Sharon M.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2866-2878</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Differential expression of secretory leukocyte protease inhibitor (SLPI) impacts on tumor progression.  SLPI directly inhibits elastase and other serine proteases, and regulates matrix metalloproteinases, plasminogen activation, and plasmin downstream targets to influence invasion.  We examd. tissues from human oral squamous cell carcinoma (OSCC) for SLPI expression in parallel with proteases assocd. with tumor progression and evaluated their relationships using tumor cell lines.  Significantly decreased SLPI was detected in OSCC compared to normal oral epithelium.  Furthermore, an inverse correlation between SLPI and histol. parameters assocd. with tumor progression, including stage of invasion, pattern of invasion, invasive cell grade, and composite histol. tumor score was evident.  Conversely, elevated plasmin and elastase were pos. correlated with histol. parameters of tumor invasion.  In addn. to its known inhibition of elastase, we identify SLPI as a novel inhibitor of plasminogen activation through its interaction with annexin A2 with concomitant reduced plasmin generation by macrophages and OSCC cell lines.  In an in vitro assay measuring invasive activity, SLPI blocked protease-dependent tumor cell migration.  Our data suggest that SLPI may possess antitumorigenic activity by virtue of its ability to interfere with multiple requisite proteolytic steps underlying tumor cell invasion and may provide insight into potential stratification of oral cancer according to risk of occult metastasis, guiding treatment strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzXnusWEPiBLVg90H21EOLACvtfcHk0liBuAZMM7Js2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslOntbg%253D&md5=e14f347afb856cc4d311dbf5ad037800</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.ajpath.2011.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajpath.2011.02.017%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DJ.%26aulast%3DNikitakis%26aufirst%3DN.%2BG.%26aulast%3DChaisuparat%26aufirst%3DR.%26aulast%3DGreenwell-Wild%26aufirst%3DT.%26aulast%3DGliozzi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DW.%2BW.%26aulast%3DAdli%26aufirst%3DA.%26aulast%3DMoutsopoulos%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DT.%2BX.%26aulast%3DWarburton%26aufirst%3DG.%26aulast%3DWahl%26aufirst%3DS.%2BM.%26atitle%3DSecretory%2520leukocyte%2520protease%2520inhibitor%2520%2528SLPI%2529%2520expression%2520and%2520tumor%2520invasion%2520in%2520oral%2520squamous%2520cell%2520carcinoma%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2011%26volume%3D178%26issue%3D6%26spage%3D2866%26epage%3D2878%26doi%3D10.1016%2Fj.ajpath.2011.02.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kheradmand, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, P. A.</span></span> <span> </span><span class="NLM_article-title">Epidermal detachment, desmosomal dissociation, and destabilization of corneodesmosin in Spink5(−/−) mice</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>18</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">2354</span>– <span class="NLM_lpage">2358</span>, <span class="refDoi"> DOI: 10.1101/gad.1232104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1101%2Fgad.1232104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=15466487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD2cXot1yrtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=2354-2358&issue=19&author=T.+Yangauthor=D.+C.+Liangauthor=P.+J.+Kochauthor=D.+Hohlauthor=F.+Kheradmandauthor=P.+A.+Overbeek&title=Epidermal+detachment%2C+desmosomal+dissociation%2C+and+destabilization+of+corneodesmosin+in+Spink5%28%E2%88%92%2F%E2%88%92%29+mice&doi=10.1101%2Fgad.1232104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal detachment, desmosomal dissociation, and destabilization of corneodesmosin in Spink5-/- mice</span></div><div class="casAuthors">Yang, Tao; Liang, Dongcai; Koch, Peter J.; Hohl, Daniel; Kheradmand, Farrah; Overbeek, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2354-2358</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Netherton syndrome (NS) is a human autosomal recessive skin disease caused by mutations in the SPINK5 gene, which encodes the putative proteinase inhibitor LEKTI.  The authors have generated a transgenic mouse line with an insertional mutation that inactivated the mouse SPINK5 ortholog.  Mutant mice exhibit fragile stratum corneum and perinatal death due to dehydration.  The anal. suggests that the phenotype is a consequence of desmosomal fragility assocd. with premature proteolysis of corneodesmosin, an extracellular desmosomal component.  The mouse mutant provides a model system for mol. studies of desmosomal stability and keratinocyte adhesion, and for designing therapeutic strategies to treat NS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVxzkcr8U6k7Vg90H21EOLACvtfcHk0liBuAZMM7Js2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXot1yrtb4%253D&md5=626f06c4940884e4314ded50b57ed3fc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1101%2Fgad.1232104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1232104%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DT.%26aulast%3DLiang%26aufirst%3DD.%2BC.%26aulast%3DKoch%26aufirst%3DP.%2BJ.%26aulast%3DHohl%26aufirst%3DD.%26aulast%3DKheradmand%26aufirst%3DF.%26aulast%3DOverbeek%26aufirst%3DP.%2BA.%26atitle%3DEpidermal%2520detachment%252C%2520desmosomal%2520dissociation%252C%2520and%2520destabilization%2520of%2520corneodesmosin%2520in%2520Spink5%2528%25E2%2588%2592%252F%25E2%2588%2592%2529%2520mice%26jtitle%3DGenes%2520Dev.%26date%3D2004%26volume%3D18%26issue%3D19%26spage%3D2354%26epage%3D2358%26doi%3D10.1101%2Fgad.1232104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Catanzaro, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheshadri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zong, W. X.</span></span> <span> </span><span class="NLM_article-title">SerpinB3/B4: mediators of Ras-driven inflammation and oncogenesis</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">3155</span>– <span class="NLM_lpage">3156</span>, <span class="refDoi"> DOI: 10.4161/15384101.2014.969991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.4161%2F15384101.2014.969991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25485489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFalsLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=3155-3156&issue=20&author=J.+M.+Catanzaroauthor=N.+Sheshadriauthor=W.+X.+Zong&title=SerpinB3%2FB4%3A+mediators+of+Ras-driven+inflammation+and+oncogenesis&doi=10.4161%2F15384101.2014.969991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">SerpinB3/B4: mediators of Ras-driven inflammation and oncogenesis</span></div><div class="casAuthors">Catanzaro, Joseph M.; Sheshadri, Namratha; Zong, Wei-Xing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3155-3156</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">In this review serpinB3/B4 mediators of Ras-driven inflammation and oncogenesis were discussed.  This article discuss on squamous cell carcinoma antigens (SCCA) SerpinB3/SerpinB4 as novel pro- tumorigenic mediators downstream of mutant Ras through the activation of NF-κB and pro-inflammatory cytokine prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjqsv2I6tPsLVg90H21EOLACvtfcHk0liBuAZMM7Js2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFalsLjK&md5=469d0eaecb708451a2de2f7514e80642</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.4161%2F15384101.2014.969991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384101.2014.969991%26sid%3Dliteratum%253Aachs%26aulast%3DCatanzaro%26aufirst%3DJ.%2BM.%26aulast%3DSheshadri%26aufirst%3DN.%26aulast%3DZong%26aufirst%3DW.%2BX.%26atitle%3DSerpinB3%252FB4%253A%2520mediators%2520of%2520Ras-driven%2520inflammation%2520and%2520oncogenesis%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26issue%3D20%26spage%3D3155%26epage%3D3156%26doi%3D10.4161%2F15384101.2014.969991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pietzsch, J.</span></span> <span> </span><span class="NLM_article-title">S100 proteins in health and disease</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>41</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">760</span>, <span class="refDoi"> DOI: 10.1007/s00726-010-0816-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2Fs00726-010-0816-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21120552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=755-760&issue=4&author=J.+Pietzsch&title=S100+proteins+in+health+and+disease&doi=10.1007%2Fs00726-010-0816-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">S100 proteins in health and disease</span></div><div class="casAuthors">Pietzsch, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">755-760</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">SpringerWienNewYork</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Qc8wWbjUzbVg90H21EOLACvtfcHk0lhCPrv1Wp2Y4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKitLfJ&md5=e36635b5c6441a05bbad2595a3aa7868</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs00726-010-0816-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-010-0816-8%26sid%3Dliteratum%253Aachs%26aulast%3DPietzsch%26aufirst%3DJ.%26atitle%3DS100%2520proteins%2520in%2520health%2520and%2520disease%26jtitle%3DAmino%2520Acids%26date%3D2011%26volume%3D41%26issue%3D4%26spage%3D755%26epage%3D760%26doi%3D10.1007%2Fs00726-010-0816-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schonthaler, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinea-Viniegra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wculek, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruppen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ximenez-Embun, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guio-Carrion, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Navarro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, E. F.</span></span> <span> </span><span class="NLM_article-title">S100A8-S100A9 protein complex mediates psoriasis by regulating the expression of complement factor C3</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1171</span>– <span class="NLM_lpage">1181</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2013.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.immuni.2013.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24332034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhs7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2013&pages=1171-1181&issue=6&author=H.+B.+Schonthalerauthor=J.+Guinea-Viniegraauthor=S.+K.+Wculekauthor=I.+Ruppenauthor=P.+Ximenez-Embunauthor=A.+Guio-Carrionauthor=R.+Navarroauthor=N.+Hoggauthor=K.+Ashmanauthor=E.+F.+Wagner&title=S100A8-S100A9+protein+complex+mediates+psoriasis+by+regulating+the+expression+of+complement+factor+C3&doi=10.1016%2Fj.immuni.2013.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">S100A8-S100A9 Protein Complex Mediates Psoriasis by Regulating the Expression of Complement Factor C3</span></div><div class="casAuthors">Schonthaler, Helia B.; Guinea-Viniegra, Juan; Wculek, Stefanie K.; Ruppen, Isabel; Ximenez-Embun, Pilar; Guio-Carrion, Ana; Navarro, Raquel; Hogg, Nancy; Ashman, Keith; Wagner, Erwin F.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1171-1181</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Psoriasis is a common heterogeneous inflammatory skin disease with a complex pathophysiol. and limited treatment options.  Here we performed proteomic analyses of human psoriatic epidermis and found S100A8-S100A9, also called calprotectin, as the most upregulated proteins, followed by the complement component C3.  Both S100A8-S100A9 and C3 are specifically expressed in lesional psoriatic skin.  S100A9 is shown here to function as a chromatin component modulating C3 expression in mouse and human cells by binding to a region upstream of the C3 start site.  When S100A9 was genetically deleted in mouse models of skin inflammation, the psoriasis-like skin disease and inflammation were strongly attenuated, with a mild immune infiltrate and decreased amts. of C3.  In addn., inhibition of C3 in the mouse model strongly reduced the inflammatory skin disease.  Thus, S100A8-S100A9 can regulate C3 at the nuclear level and present potential new therapeutic targets for psoriasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNqfBzsz7G2bVg90H21EOLACvtfcHk0lhCPrv1Wp2Y4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhs7rM&md5=a3c732f52dd9c97385da241567b0a4f8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2013.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2013.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DSchonthaler%26aufirst%3DH.%2BB.%26aulast%3DGuinea-Viniegra%26aufirst%3DJ.%26aulast%3DWculek%26aufirst%3DS.%2BK.%26aulast%3DRuppen%26aufirst%3DI.%26aulast%3DXimenez-Embun%26aufirst%3DP.%26aulast%3DGuio-Carrion%26aufirst%3DA.%26aulast%3DNavarro%26aufirst%3DR.%26aulast%3DHogg%26aufirst%3DN.%26aulast%3DAshman%26aufirst%3DK.%26aulast%3DWagner%26aufirst%3DE.%2BF.%26atitle%3DS100A8-S100A9%2520protein%2520complex%2520mediates%2520psoriasis%2520by%2520regulating%2520the%2520expression%2520of%2520complement%2520factor%2520C3%26jtitle%3DImmunity%26date%3D2013%26volume%3D39%26issue%3D6%26spage%3D1171%26epage%3D1181%26doi%3D10.1016%2Fj.immuni.2013.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braunstein, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toomey, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, X. Q.</span></span> <span> </span><span class="NLM_article-title">S100 proteins as an important regulator of macrophage inflammation</span>. <i>Front. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1908</span>, <span class="refDoi"> DOI: 10.3389/fimmu.2017.01908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3389%2Ffimmu.2017.01908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29379499" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=1908&author=C.+Xiaauthor=Z.+Braunsteinauthor=A.+C.+Toomeyauthor=J.+X.+Zhongauthor=X.+Q.+Rao&title=S100+proteins+as+an+important+regulator+of+macrophage+inflammation&doi=10.3389%2Ffimmu.2017.01908"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.3389%2Ffimmu.2017.01908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffimmu.2017.01908%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DC.%26aulast%3DBraunstein%26aufirst%3DZ.%26aulast%3DToomey%26aufirst%3DA.%2BC.%26aulast%3DZhong%26aufirst%3DJ.%2BX.%26aulast%3DRao%26aufirst%3DX.%2BQ.%26atitle%3DS100%2520proteins%2520as%2520an%2520important%2520regulator%2520of%2520macrophage%2520inflammation%26jtitle%3DFront.%2520Immunol.%26date%3D2018%26volume%3D8%26spage%3D1908%26doi%3D10.3389%2Ffimmu.2017.01908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diakos, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charles, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. J.</span></span> <span> </span><span class="NLM_article-title">Cancer-related inflammation and treatment effectiveness</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">e493</span>– <span class="NLM_lpage">e503</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(14)70263-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2FS1470-2045%2814%2970263-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25281468" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=e493-e503&issue=11&author=C.+I.+Diakosauthor=K.+A.+Charlesauthor=D.+C.+McMillanauthor=S.+J.+Clarke&title=Cancer-related+inflammation+and+treatment+effectiveness&doi=10.1016%2FS1470-2045%2814%2970263-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2814%2970263-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252814%252970263-3%26sid%3Dliteratum%253Aachs%26aulast%3DDiakos%26aufirst%3DC.%2BI.%26aulast%3DCharles%26aufirst%3DK.%2BA.%26aulast%3DMcMillan%26aufirst%3DD.%2BC.%26aulast%3DClarke%26aufirst%3DS.%2BJ.%26atitle%3DCancer-related%2520inflammation%2520and%2520treatment%2520effectiveness%26jtitle%3DLancet%2520Oncol.%26date%3D2014%26volume%3D15%26issue%3D11%26spage%3De493%26epage%3De503%26doi%3D10.1016%2FS1470-2045%2814%2970263-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shabani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farasat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahdavi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheibi, N.</span></span> <span> </span><span class="NLM_article-title">Calprotectin (S100A8/S100A9): a key protein between inflammation and cancer</span>. <i>Inflammation Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>67</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1007/s00011-018-1173-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2Fs00011-018-1173-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30083975" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGju7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2018&pages=801-812&issue=10&author=F.+Shabaniauthor=A.+Farasatauthor=M.+Mahdaviauthor=N.+Gheibi&title=Calprotectin+%28S100A8%2FS100A9%29%3A+a+key+protein+between+inflammation+and+cancer&doi=10.1007%2Fs00011-018-1173-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Calprotectin (S100A8/S100A9): a key protein between inflammation and cancer</span></div><div class="casAuthors">Shabani, Fatemeh; Farasat, Alireza; Mahdavi, Majid; Gheibi, Nematollah</div><div class="citationInfo"><span class="NLM_cas:title">Inflammation Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">801-812</span>CODEN:
                <span class="NLM_cas:coden">INREFB</span>;
        ISSN:<span class="NLM_cas:issn">1023-3830</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Background: Calprotectin (S100A8/S100A9), a heterodimeric EF-hand Ca2+ binding protein, are abundant in cytosol of neutrophils and are involved in inflammatory processes and several cancerous pathogens.  Objective: The purpose of the present systematic review is to evaluate the pro- and anti-tumorigenic functions of calprotectin and its relation to inflammation.  Materials and methods: We conducted a review of studies published in the Medline (1966-2018), Scopus (2004-2018), ClinicalTrials.gov (2008-2018) and Google Scholar (2004-2018) databases, combined with studies found in the ref. lists of the included studies.  Results: Elevated levels of S100A8/S100A9 were detected in inflammation, neoplastic tumor cells and various human cancers.  Recent data have explained that many cancers arise from sites of infection, chronic irritation, and inflammation.  The inflammatory microenvironment which largely includes calprotectin, has an essential role on high producing of inflammatory factors and then on neoplastic process and metastasis.  Conclusion: Scientists have shown different outcomes in inflammation, malignancy and apoptosis whether the source of the aforementioned protein is extracellular or intracellular.  These findings are offering new insights that anti-inflammatory therapeutic agents and anti-tumorigenic functions of calprotectin can lead to control cancer development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9oH42g1_GAbVg90H21EOLACvtfcHk0lhCPrv1Wp2Y4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGju7bL&md5=5c5dffea746c874e5f8bafaba772939c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs00011-018-1173-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00011-018-1173-4%26sid%3Dliteratum%253Aachs%26aulast%3DShabani%26aufirst%3DF.%26aulast%3DFarasat%26aufirst%3DA.%26aulast%3DMahdavi%26aufirst%3DM.%26aulast%3DGheibi%26aufirst%3DN.%26atitle%3DCalprotectin%2520%2528S100A8%252FS100A9%2529%253A%2520a%2520key%2520protein%2520between%2520inflammation%2520and%2520cancer%26jtitle%3DInflammation%2520Res.%26date%3D2018%26volume%3D67%26issue%3D10%26spage%3D801%26epage%3D812%26doi%3D10.1007%2Fs00011-018-1173-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Netea, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Veerdonk, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinarello, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joosten, L. A. B.</span></span> <span> </span><span class="NLM_article-title">Inflammasome-independent regulation of IL-1-family cytokines</span>. <i>Annu. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1146/annurev-immunol-032414-112306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1146%2Fannurev-immunol-032414-112306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25493334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVSisLzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=49-77&author=M.+G.+Neteaauthor=F.+L.+van+de+Veerdonkauthor=J.+W.+M.+van+der+Meerauthor=C.+A.+Dinarelloauthor=L.+A.+B.+Joosten&title=Inflammasome-independent+regulation+of+IL-1-family+cytokines&doi=10.1146%2Fannurev-immunol-032414-112306"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammasome-Independent Regulation of IL-1-Family Cytokines</span></div><div class="casAuthors">Netea, Mihai G.; van de Veerdonk, Frank L.; van der Meer, Jos W. M.; Dinarello, Charles A.; Joosten, Leo A. B.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">49-77</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Induction, prodn., and release of proinflammatory cytokines are essential steps to establish an effective host defense.  Cytokines of the interleukin-1 (IL-1) family induce inflammation and regulate T lymphocyte responses while also displaying homeostatic and metabolic activities.  With the exception of the IL-1 receptor antagonist, all IL-1 family cytokines lack a signal peptide and require proteolytic processing into an active mol.  One such unique protease is caspase-1, which is activated by protein platforms called the inflammasomes.  However, increasing evidence suggests that inflammasomes and caspase-1 are not the only mechanism for processing IL-1 cytokines.  IL-1 cytokines are often released as precursors and require extracellular processing for activity.  Here we review the inflammasome-independent enzymic processes that are able to activate IL-1 cytokines, paying special attention to neutrophil-derived serine proteases, which subsequently induce inflammation and modulate host defense.  The inflammasome-independent processing of IL-1 cytokines has important consequences for understanding inflammatory diseases, and it impacts the design of IL-1-based modulatory therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzeRfZbo7l0rVg90H21EOLACvtfcHk0lhtrLG-BTUOnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVSisLzF&md5=e4f426578799058ea5cf1ca41376c714</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1146%2Fannurev-immunol-032414-112306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-immunol-032414-112306%26sid%3Dliteratum%253Aachs%26aulast%3DNetea%26aufirst%3DM.%2BG.%26aulast%3Dvan%2Bde%2BVeerdonk%26aufirst%3DF.%2BL.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DDinarello%26aufirst%3DC.%2BA.%26aulast%3DJoosten%26aufirst%3DL.%2BA.%2BB.%26atitle%3DInflammasome-independent%2520regulation%2520of%2520IL-1-family%2520cytokines%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2015%26volume%3D33%26spage%3D49%26epage%3D77%26doi%3D10.1146%2Fannurev-immunol-032414-112306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shuai, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R. Y.</span></span> <span> </span><span class="NLM_article-title">BST2 confers cisplatin resistance via NF-kappa B signaling in nasopharyngeal cancer</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e2874</span> <span class="refDoi"> DOI: 10.1038/cddis.2017.271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fcddis.2017.271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28617432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=C.+M.+Kuangauthor=X.+Fuauthor=Y.+J.+Huaauthor=W.+D.+Shuaiauthor=Z.+H.+Yeauthor=Y.+Liauthor=Q.+H.+Pengauthor=Y.+Z.+Liauthor=S.+Chenauthor=C.+N.+Qianauthor=W.+Huangauthor=R.+Y.+Liu&title=BST2+confers+cisplatin+resistance+via+NF-kappa+B+signaling+in+nasopharyngeal+cancer&doi=10.1038%2Fcddis.2017.271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">BST2 confers cisplatin resistance via NF-κB signaling in nasopharyngeal cancer</span></div><div class="casAuthors">Kuang, Chun-mei; Fu, Xiang; Hua, Yi-jun; Shuai, Wen-di; Ye, Zhi-hua; Li, Yingchang; Peng, Qi-hua; Li, Yi-zhuo; Chen, Shuai; Qian, Chao-nan; Huang, Wenlin; Liu, Ran-yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e2874</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concurrent/adjuvant cisplatin-based chemoradiotherapy is regarded as the std. of treatment for locoregionally advanced nasopharyngeal carcinoma (NPC).  However, patients who do not respond to cisplatin suffer, rather than benefit, from chemotherapy treatment.  The goal of this study was to identify mols. involved in cisplatin resistance and to clarify their mol. mechanisms, which would help in the discovery of potential therapeutic targets and in developing a personalized and precise treatment approach for NPC patients.  We previously generated a cisplatin-sensitive NPC cell line, S16, from CNE2 cells and found that eIF3a, ASNS and MMP19 are upregulated in S16 cells, which contributes to their cisplatin sensitivity.  In this study, we found that BST2 is downregulated in cisplatin-sensitive S16 cells compared with CNE2 cells.  Knockdown of BST2 in NPC cells sensitized their response to cisplatin and promoted cisplatin-induced apoptosis, whereas exogenous overexpression of BST2 increased their cisplatin resistance and inhibited cisplatin-induced apoptosis.  Further investigation demonstrated that BST2-mediated cisplatin resistance depended on the activation of the NF-κB signaling pathway and consequent upregulation of anti-apoptotic genes, such as Bcl-XL and livin.  Moreover, an anal. of clin. data revealed that a high BST2 level might serve as an independent indicator of poor prognosis in patients with locally advanced NPC treated with platinum-based chemoradiotherapy.  These findings suggest that BST2 likely mediates platinum resistance in NPC, offering guidance for personalized and precise treatment strategies for patients with NPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnUDS4Syse97Vg90H21EOLACvtfcHk0lhtrLG-BTUOnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCiurvK&md5=59b258179f24a50ed62e51dcb4da0317</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2017.271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2017.271%26sid%3Dliteratum%253Aachs%26aulast%3DKuang%26aufirst%3DC.%2BM.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DHua%26aufirst%3DY.%2BJ.%26aulast%3DShuai%26aufirst%3DW.%2BD.%26aulast%3DYe%26aufirst%3DZ.%2BH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DQ.%2BH.%26aulast%3DLi%26aufirst%3DY.%2BZ.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DQian%26aufirst%3DC.%2BN.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DR.%2BY.%26atitle%3DBST2%2520confers%2520cisplatin%2520resistance%2520via%2520NF-kappa%2520B%2520signaling%2520in%2520nasopharyngeal%2520cancer%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fcddis.2017.271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sica, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vannucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allavena, P.</span></span> <span> </span><span class="NLM_article-title">Inflammation as target in cancer therapy</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2017.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.coph.2017.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=28618326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXptlShtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=57-65&author=G.+Marelliauthor=A.+Sicaauthor=L.+Vannucciauthor=P.+Allavena&title=Inflammation+as+target+in+cancer+therapy&doi=10.1016%2Fj.coph.2017.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inflammation as target in cancer therapy</span></div><div class="casAuthors">Marelli, Giulia; Sica, Antonio; Vannucci, Luca; Allavena, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">57-65</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cells of the innate immunity infiltrating tumor tissues promote, rather than halt, cancer cell proliferation and distant spreading.  Tumor-Assocd. Macrophages (TAMs) are abundantly present in the tumor milieu and here trigger and perpetrate a state of chronic inflammation which ultimately supports disease development and contributes to an immune-suppressive environment.  Therapeutic strategies to limit inflammatory cells and their products have been successful in pre-clin. tumor models.  Early clin. trials with specific cytokine and chemokine inhibitors, or with strategies designed to target TAMs, are on their way in different solid malignancies.  Partial clin. responses and stabilization of diseases were obsd. in some patients, in the absence of significant toxicity.  These encouraging results open new perspectives of combination treatments aimed at reducing cancer-promoting inflammation to maximize the anti-tumor efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDCjYnb6LW7Vg90H21EOLACvtfcHk0lhtrLG-BTUOnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXptlShtbY%253D&md5=e8d65f2760714c9fb296a1c81fedb5ad</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2017.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2017.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DMarelli%26aufirst%3DG.%26aulast%3DSica%26aufirst%3DA.%26aulast%3DVannucci%26aufirst%3DL.%26aulast%3DAllavena%26aufirst%3DP.%26atitle%3DInflammation%2520as%2520target%2520in%2520cancer%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2017%26volume%3D35%26spage%3D57%26epage%3D65%26doi%3D10.1016%2Fj.coph.2017.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Moraes, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiva, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olmedo, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faria, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Moura Gallo, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hainaut, P.</span></span> <span> </span><span class="NLM_article-title">Interferon-inducible guanylate binding protein (GBP)-2: a novel p53-regulated tumor marker in esophageal squamous cell carcinomas</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>124</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1002/ijc.23944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1002%2Fijc.23944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=19003964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BD1M%252FitFSjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2009&pages=272-279&issue=2&author=D.+P.+Guimaraesauthor=I.+M.+Oliveiraauthor=E.+de+Moraesauthor=G.+R.+Paivaauthor=D.+M.+Souzaauthor=C.+Barnasauthor=D.+B.+Olmedoauthor=C.+E.+Pintoauthor=P.+A.+Fariaauthor=C.+V.+De+Moura+Galloauthor=I.+A.+Smallauthor=C.+G.+Ferreiraauthor=P.+Hainaut&title=Interferon-inducible+guanylate+binding+protein+%28GBP%29-2%3A+a+novel+p53-regulated+tumor+marker+in+esophageal+squamous+cell+carcinomas&doi=10.1002%2Fijc.23944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Interferon-inducible guanylate binding protein (GBP)-2: a novel p53-regulated tumor marker in esophageal squamous cell carcinomas</span></div><div class="casAuthors">Guimaraes Denise P; Oliveira Ivanir M; de Moraes Emanuela; Paiva Geisilene R; Souza Daniel M; Barnas Christoph; Olmedo Daniele B; Pinto Carlos E; Faria Paulo A; De Moura Gallo Claudia Vitoria; Small Isabele A; Ferreira Carlos G; Hainaut Pierre</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">272-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">TP53 mutations are common in esophageal squamous cell carcinomas (SCC).  To identify biological markers of possible relevance in esophageal SCC, we (i) searched for genes expressed in a p53-dependent manner in TE-1, an esophageal SCC cell line expressing the temperature-sensitive TP53 mutant V272M, and (ii) investigated the expression of one of those genes, the interferon-inducible Guanylate Binding Protein 2 (GBP-2), in esophageal SCC tissues.  Clontech Human Cancer 1.2 arrays containing 1,176 human cancer gene-related sequences were used to identify differentially expressed genes in TE-1 cells at permissive (32 degrees C) and nonpermissive (37 degrees C) temperatures.  The expression of GBP-2 and IRF-1, its main transcriptional regulator, was analyzed by immunohistochemistry in a retrospective series of 41 esophageal SCC cases with a clear transition zone from noncancer, apparently normal epithelium to invasive cancer.  The expression of the GBP-2 gene is consistently increased in TE-1 at 32 degrees C in a p53-dependent manner, as confirmed by inhibition of p53 expression by RNA interference.  Increase in GBP-2 is accompanied by an increase in protein levels of IRF-1, the main transcriptional regulator of GBP-2, and in the formation of complexes between p53 and IRF-1.  GBP-2 expression is significantly higher in esophageal SCC than in adjacent normal epithelium (p<0.01), in which GBP-2 staining is limited to the basal layer.  Our results suggest that p53 up-regulates GBP-2 by cooperating with IRF-1.  The association of GBP-2 expression with proliferative squamous cells suggests that GBP-2 may represent a marker of interest in esophageal SCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTtHR0juPLiV91QT_1O5WBYfW6udTcc2eZB51PXtDxYpLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1M%252FitFSjtw%253D%253D&md5=279e6df17dffd110b4f58290b94d1f9a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fijc.23944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.23944%26sid%3Dliteratum%253Aachs%26aulast%3DGuimaraes%26aufirst%3DD.%2BP.%26aulast%3DOliveira%26aufirst%3DI.%2BM.%26aulast%3Dde%2BMoraes%26aufirst%3DE.%26aulast%3DPaiva%26aufirst%3DG.%2BR.%26aulast%3DSouza%26aufirst%3DD.%2BM.%26aulast%3DBarnas%26aufirst%3DC.%26aulast%3DOlmedo%26aufirst%3DD.%2BB.%26aulast%3DPinto%26aufirst%3DC.%2BE.%26aulast%3DFaria%26aufirst%3DP.%2BA.%26aulast%3DDe%2BMoura%2BGallo%26aufirst%3DC.%2BV.%26aulast%3DSmall%26aufirst%3DI.%2BA.%26aulast%3DFerreira%26aufirst%3DC.%2BG.%26aulast%3DHainaut%26aufirst%3DP.%26atitle%3DInterferon-inducible%2520guanylate%2520binding%2520protein%2520%2528GBP%2529-2%253A%2520a%2520novel%2520p53-regulated%2520tumor%2520marker%2520in%2520esophageal%2520squamous%2520cell%2520carcinomas%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2009%26volume%3D124%26issue%3D2%26spage%3D272%26epage%3D279%26doi%3D10.1002%2Fijc.23944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antony, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meitzler, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, K.</span></span> <span> </span><span class="NLM_article-title">Functional activity and tumor-specific expression of dual oxidase 2 in pancreatic cancer cells and human malignancies characterized with a novel monoclonal antibody</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1229</span>– <span class="NLM_lpage">1238</span>, <span class="refDoi"> DOI: 10.3892/ijo.2013.1821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3892%2Fijo.2013.1821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=23404210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmt1ems70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=1229-1238&issue=4&author=Y.+Z.+Wuauthor=S.+Antonyauthor=S.+M.+Hewittauthor=G.+J.+Jiangauthor=S.+X.+Yangauthor=J.+L.+Meitzlerauthor=A.+Juhaszauthor=J.+M.+Luauthor=H.+Liuauthor=J.+H.+Doroshowauthor=K.+Roy&title=Functional+activity+and+tumor-specific+expression+of+dual+oxidase+2+in+pancreatic+cancer+cells+and+human+malignancies+characterized+with+a+novel+monoclonal+antibody&doi=10.3892%2Fijo.2013.1821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Functional activity and tumor-specific expression of dual oxidase 2 in pancreatic cancer cells and human malignancies characterized with a novel monoclonal antibody</span></div><div class="casAuthors">Wu, Yonghzong; Antony, Smitha; Hewitt, Stephen M.; Jiang, Guojian; Yang, Sherry X.; Meitzler, Jennifer L.; Juhasz, Agnes; Lu, Jiamo; Liu, Han; Doroshow, James H.; Roy, Krishnendu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1229-1238</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">Dual oxidase 2 (Duox2), one of the seven members of the NADPH oxidase gene family, plays a crit. role in generating H2O2 for thyroid hormone biosynthesis and as an integral part of the host defense system of the respiratory epithelium and the gastrointestinal tract.  Recent evidence suggests that the regulation of Duox2 expression is under the control of pro-inflammatory cytokines and that Duox2-induced reactive oxygen species (ROS) contribute to the inflammation-related tissue injury that occurs in two pre-malignant, inflammatory conditions: chronic pancreatitis and inflammatory bowel disease.  Because no reliable Duox antibodies are com. available, we report the development of a murine monoclonal antibody (MAb) to Duox2 (clone Duox S-12) and its use for the characterization of Duox2 expression in human tumors, tumor cell lines and normal tissues.  Duox S-12 specifically detected both endogenously- and ectopically-expressed Duox2 protein by immunoblotting, immunofluorescence microscopy and immunohistochem. (where both membranous and cytoplasmic staining were present).  Duox2 expression detected by Duox S-12 was functionally coupled to the generation of H2O2 in pancreatic cancer cells that expressed Duox2 and its cognate maturation factor DuoxA2.  Although Duox S-12 recognizes ectopically expressed Duox1 protein because of the extensive amino acid homol. between Duox1 and Duox2, the lack of substantial Duox1 mRNA expression in human tumors (except thyroid cancer) allowed us to evaluate Duox2 expression across a wide range of normal and malignant tissues by immunohistochem.  Duox2 was expressed at elevated levels in many human cancers, most notably tumors of the prostate, lung, colon and breast while brain tumors and lymphomas demonstrated the lowest frequency of expression.  The Duox-specific monoclonal antibody described here provides a promising tool for the further examn. of the role of Duox-dependent reactive oxygen prodn. in inflammation-related carcinogenesis, where alterations in oxidant tone play a crit. role in cell growth and proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6heKp3f02KrVg90H21EOLACvtfcHk0lgwAFBi6QwHQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmt1ems70%253D&md5=234aea208352a959dcd3f0d61bbcb362</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.3892%2Fijo.2013.1821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2013.1821%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%2BZ.%26aulast%3DAntony%26aufirst%3DS.%26aulast%3DHewitt%26aufirst%3DS.%2BM.%26aulast%3DJiang%26aufirst%3DG.%2BJ.%26aulast%3DYang%26aufirst%3DS.%2BX.%26aulast%3DMeitzler%26aufirst%3DJ.%2BL.%26aulast%3DJuhasz%26aufirst%3DA.%26aulast%3DLu%26aufirst%3DJ.%2BM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DRoy%26aufirst%3DK.%26atitle%3DFunctional%2520activity%2520and%2520tumor-specific%2520expression%2520of%2520dual%2520oxidase%25202%2520in%2520pancreatic%2520cancer%2520cells%2520and%2520human%2520malignancies%2520characterized%2520with%2520a%2520novel%2520monoclonal%2520antibody%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2013%26volume%3D42%26issue%3D4%26spage%3D1229%26epage%3D1238%26doi%3D10.3892%2Fijo.2013.1821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glisson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Y.</span></span> <span> </span><span class="NLM_article-title">Insulin-like growth factor binding protein-3 suppresses vascular endothelial growth factor expression and tumor angiogenesis in head and neck squamous cell carcinoma</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>103</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1259</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2012.02301.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1111%2Fj.1349-7006.2012.02301.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22494072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38XpsFKmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2012&pages=1259-1266&issue=8&author=S.+H.+Ohauthor=W.+Y.+Kimauthor=O.+H.+Leeauthor=J.+H.+Kangauthor=J.+K.+Wooauthor=J.+H.+Kimauthor=B.+Glissonauthor=H.+Y.+Lee&title=Insulin-like+growth+factor+binding+protein-3+suppresses+vascular+endothelial+growth+factor+expression+and+tumor+angiogenesis+in+head+and+neck+squamous+cell+carcinoma&doi=10.1111%2Fj.1349-7006.2012.02301.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Insulin-like growth factor binding protein-3 suppresses vascular endothelial growth factor expression and tumor angiogenesis in head and neck squamous cell carcinoma</span></div><div class="casAuthors">Oh, Seung-Hyun; Kim, Woo-Young; Lee, Ok-Hee; Kang, Ju-Hee; Woo, Jong-Kyu; Kim, Jai-Hyun; Glisson, Bonnie; Lee, Ho-Young</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1259-1266</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Angiogenesis, the process by which new blood vessels are recruited to existing ones, is essential for tumor development.  Insulin-like growth factor (IGF) binding protein-3 (IGFBP-3), which modulates bioavailability of IGF, has been studied for its potential role in angiogenesis during tissue regeneration and cancer development.  In this study, we assessed the role of IGFBP-3 in tumor angiogenesis in head and neck squamous cell carcinoma (HNSCC) and human umbilical vein endothelial cells (HUVECs) using adenoviral (Ad-BP3) and recombinant (rBP3) IGFBP-3.  Using an in vivo orthotopic tongue tumor model, we confirmed that both Ad-BP3 and rBP3 suppress the growth of UMSCC38 HNSCC cells in vivo.  Ad-BP3 inhibited vascularization in tongue tumors and chorio-allantoic membrane, and suppressed angiogenesis-stimulating activities in UMSCC38 cells.  In HUVECs, Ad-BP3 decreased migration, invasion, and tube formation. rBP3 also suppressed prodn. of vascular endothelial growth factor (VEGF) in HUVECs and UMSCC38 cells.  IGFBP-3-GGG, a mutant IGFBP-3 with loss of IGF binding capacity, suppressed VEGF prodn.  In addn., we found that IGFBP-3 suppressed VEGF expression, even in mouse embryonic fibroblasts from an IGF-1R-null mouse.  Finally, we demonstrated that IGFBP-3-GGG inhibits tumor angiogenesis and growth to the same degree as wild-type IGFBP-3.  Taken together, these results support the hypothesis that IGFBP-3 has anti-angiogenic activity in HNSCC, at least in part due to IGF-independent suppression of VEGF prodn. from vascular endothelial cells and cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDKTEwsydrnbVg90H21EOLACvtfcHk0liMNyt8sEpX0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpsFKmtLo%253D&md5=94ea6be25162ebc45ca384db62b0dea8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2012.02301.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2012.02301.x%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DW.%2BY.%26aulast%3DLee%26aufirst%3DO.%2BH.%26aulast%3DKang%26aufirst%3DJ.%2BH.%26aulast%3DWoo%26aufirst%3DJ.%2BK.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DGlisson%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DH.%2BY.%26atitle%3DInsulin-like%2520growth%2520factor%2520binding%2520protein-3%2520suppresses%2520vascular%2520endothelial%2520growth%2520factor%2520expression%2520and%2520tumor%2520angiogenesis%2520in%2520head%2520and%2520neck%2520squamous%2520cell%2520carcinoma%26jtitle%3DCancer%2520Sci.%26date%3D2012%26volume%3D103%26issue%3D8%26spage%3D1259%26epage%3D1266%26doi%3D10.1111%2Fj.1349-7006.2012.02301.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, R. C.</span></span> <span> </span><span class="NLM_article-title">IGF binding proteins in cancer: mechanistic and clinical insights</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1038/nrc3720</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnrc3720" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24722429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWns7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=329-341&issue=5&author=R.+C.+Baxter&title=IGF+binding+proteins+in+cancer%3A+mechanistic+and+clinical+insights&doi=10.1038%2Fnrc3720"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">IGF binding proteins in cancer: mechanistic and clinical insights</span></div><div class="casAuthors">Baxter, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">329-341</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The six members of the family of insulin-like growth factor (IGF) binding proteins (IGFBPs) were originally characterized as passive reservoirs of circulating IGFs, but they are now understood to have many actions beyond their endocrine role in IGF transport.  IGFBPs also function in the pericellular and intracellular compartments to regulate cell growth and survival - they interact with many proteins, in addn. to their canonical ligands IGF-I and IGF-II.  Intranuclear roles of IGFBPs in transcriptional regulation, induction of apoptosis and DNA damage repair point to their intimate involvement in tumor development, progression and resistance to treatment.  Tissue or circulating IGFBPs might also be useful as prognostic biomarkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH1zpqHzHzv7Vg90H21EOLACvtfcHk0liMNyt8sEpX0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWns7Y%253D&md5=0e88b6b79cffd306e30248bc145e879d</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnrc3720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3720%26sid%3Dliteratum%253Aachs%26aulast%3DBaxter%26aufirst%3DR.%2BC.%26atitle%3DIGF%2520binding%2520proteins%2520in%2520cancer%253A%2520mechanistic%2520and%2520clinical%2520insights%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26issue%3D5%26spage%3D329%26epage%3D341%26doi%3D10.1038%2Fnrc3720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidotto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polachini, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Paula-Silva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliani, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cury, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gois-Filho, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Carvalho, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leopoldino, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajara, E. H.</span></span> <span> </span><span class="NLM_article-title">Differentially expressed proteins in positive versus negative HNSCC lymph nodes</span>. <i>BMC Med. Genomics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">73</span>, <span class="refDoi"> DOI: 10.1186/s12920-018-0382-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fs12920-018-0382-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30157864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1Gmu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2018&pages=73&author=A.+Vidottoauthor=G.+M.+Polachiniauthor=M.+de+Paula-Silvaauthor=S.+M.+Olianiauthor=T.+Henriqueauthor=R.+V.+M.+Lopezauthor=P.+M.+Curyauthor=F.+D.+Nunesauthor=J.+F.+Gois-Filhoauthor=M.+B.+de+Carvalhoauthor=A.+M.+Leopoldinoauthor=E.+H.+Tajara&title=Differentially+expressed+proteins+in+positive+versus+negative+HNSCC+lymph+nodes&doi=10.1186%2Fs12920-018-0382-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Differentially expressed proteins in positive versus negative HNSCC lymph nodes</span></div><div class="casAuthors">Vidotto, Alessandra; Polachini, Giovana M.; de Paula-Silva, Marina; Oliani, Sonia M.; Henrique, Tiago; Lopez, Rossana V. M.; Cury, Patricia M.; Nunes, Fabio D.; Gois-Filho, Jose F.; de Carvalho, Marcos B.; Leopoldino, Andreia M.; Tajara, Eloiza H.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Medical Genomics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">73/1-73/16</span>CODEN:
                <span class="NLM_cas:coden">BMGMBS</span>;
        ISSN:<span class="NLM_cas:issn">1755-8794</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Lymph node metastasis is one of the most important prognostic factors in head and neck squamous cell carcinomas (HNSCCs) and crit. for delineating their treatment.  However, clin. and histol. criteria for the diagnosis of nodal status remain limited.  In the present study, we aimed to characterize the proteomic profile of lymph node metastasis from HNSCC patients.  Methods: In the present study, we used one- and two-dimensional electrophoresis and mass spectrometry anal. to characterize the proteomic profile of lymph node metastasis from HNSCC.  Results: Comparison of metastatic and non-metastatic lymph nodes showed 52 differentially expressed proteins assocd. with neoplastic development and progression.  The results reinforced the idea that tumors from different anatomical subsites have dissimilar behaviors, which may be influenced by micro-environmental factor including the lymphatic network.  The expression pattern of heat shock proteins and glycolytic enzymes also suggested an effect of the lymph node environment in controlling tumor growth or in metabolic reprogramming of the metastatic cell.  Our study, for the first time, provided direct evidence of annexin A1 overexpression in lymph node metastasis of head and neck cancer, adding information that may be useful for diagnosing aggressive disease.  Conclusions: In brief, this study contributed to our understanding of the metastatic phenotype of HNSCC and provided potential targets for diagnostic in this group of carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVNjfWtoKBwLVg90H21EOLACvtfcHk0liMNyt8sEpX0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1Gmu7s%253D&md5=a460c2e2757ea41acbd57e0681e7a65b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1186%2Fs12920-018-0382-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12920-018-0382-6%26sid%3Dliteratum%253Aachs%26aulast%3DVidotto%26aufirst%3DA.%26aulast%3DPolachini%26aufirst%3DG.%2BM.%26aulast%3Dde%2BPaula-Silva%26aufirst%3DM.%26aulast%3DOliani%26aufirst%3DS.%2BM.%26aulast%3DHenrique%26aufirst%3DT.%26aulast%3DLopez%26aufirst%3DR.%2BV.%2BM.%26aulast%3DCury%26aufirst%3DP.%2BM.%26aulast%3DNunes%26aufirst%3DF.%2BD.%26aulast%3DGois-Filho%26aufirst%3DJ.%2BF.%26aulast%3Dde%2BCarvalho%26aufirst%3DM.%2BB.%26aulast%3DLeopoldino%26aufirst%3DA.%2BM.%26aulast%3DTajara%26aufirst%3DE.%2BH.%26atitle%3DDifferentially%2520expressed%2520proteins%2520in%2520positive%2520versus%2520negative%2520HNSCC%2520lymph%2520nodes%26jtitle%3DBMC%2520Med.%2520Genomics%26date%3D2018%26volume%3D11%26spage%3D73%26doi%3D10.1186%2Fs12920-018-0382-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, T. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span> <span> </span><span class="NLM_article-title">Fatty-acid-binding protein 5 promotes cell proliferation and invasion in oral squamous cell carcinoma</span>. <i>J. Oral Pathol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>39</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">348</span>, <span class="refDoi"> DOI: 10.1111/j.1600-0714.2009.00836.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1111%2Fj.1600-0714.2009.00836.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20040021" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1aqtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=342-348&issue=4&author=L.+Y.+Fangauthor=T.+Y.+Wongauthor=W.+F.+Chiangauthor=Y.+L.+Chen&title=Fatty-acid-binding+protein+5+promotes+cell+proliferation+and+invasion+in+oral+squamous+cell+carcinoma&doi=10.1111%2Fj.1600-0714.2009.00836.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Fatty-acid-binding protein 5 promotes cell proliferation and invasion in oral squamous cell carcinoma</span></div><div class="casAuthors">Fang, Lei-Ya; Wong, Tung-Yiu; Chiang, Wei-Fan; Chen, Yuh-Ling</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Oral Pathology & Medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">342-348</span>CODEN:
                <span class="NLM_cas:coden">JPMEEA</span>;
        ISSN:<span class="NLM_cas:issn">0904-2512</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND: Oral squamous cell carcinoma (OSCC) is one of the most malignant neoplasms worldwide, and the mol. mechanism of oral tumorigenesis is still unclear.  Fatty-acid-binding protein 5 (FABP5) was found in our previous study to be upregulated in oral squamous cell carcinomas by proteomic anal.  The implications of FABP5 overexpression in oral cancer progression have not yet been elucidated.  MATERIALS AND METHODS: In this study, the recombinant adeno-assocd. virus vectors were used to deliver and increase the expression of FABP5 in human OSCC cell lines.  U6 promoter-driven short-hairpin RNA (shRNA)-triggered RNA interference was used to block FABP5 gene expression.  Transwell Matrigel invasion assay, MTS cell proliferation assay, reverse transcription-polymerase chain reaction, Western blot, and gelatin zymog. anal. were used to investigate the effects of FABP5 on cell invasion, growth, and matrix metalloproteinase (MMP) prodn.  RESULTS: Overexpression of FABP5 in oral cancer cells increased cell proliferation and invasiveness by increasing the expression of MMP-9.  Silencing FABP5 with shRNA significantly suppressed cell proliferation, MMP-9 activities, and invasiveness.  CONCLUSION: Our study provides the first evidence that FABP5 expression modulated MMP-9 prodn. and the invasive behavior of oral cancer cells and suggests that FABP5 may provide novel targets for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMhi0CPQQEw7Vg90H21EOLACvtfcHk0ljnckacvc--CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1aqtLc%253D&md5=b2ff5d080eb0202ac2b9aa349f5bbf81</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1111%2Fj.1600-0714.2009.00836.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1600-0714.2009.00836.x%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DL.%2BY.%26aulast%3DWong%26aufirst%3DT.%2BY.%26aulast%3DChiang%26aufirst%3DW.%2BF.%26aulast%3DChen%26aufirst%3DY.%2BL.%26atitle%3DFatty-acid-binding%2520protein%25205%2520promotes%2520cell%2520proliferation%2520and%2520invasion%2520in%2520oral%2520squamous%2520cell%2520carcinoma%26jtitle%3DJ.%2520Oral%2520Pathol.%2520Med.%26date%3D2010%26volume%3D39%26issue%3D4%26spage%3D342%26epage%3D348%26doi%3D10.1111%2Fj.1600-0714.2009.00836.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span> <span> </span><span class="NLM_article-title">Analysis of gene coexpression network reveals prognostic significance of CNFN in patients with head and neck cancer</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2168</span>– <span class="NLM_lpage">2180</span>, <span class="refDoi"> DOI: 10.3892/or.2019.7019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3892%2For.2019.7019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30816522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWmt7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2019&pages=2168-2180&author=B.+Liuauthor=G.+Huangauthor=H.+Zhuauthor=Z.+Maauthor=X.+Tianauthor=L.+Yinauthor=X.+Gaoauthor=X.+He&title=Analysis+of+gene+coexpression+network+reveals+prognostic+significance+of+CNFN+in+patients+with+head+and+neck+cancer&doi=10.3892%2For.2019.7019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of gene co-expression network reveals prognostic significance of CNFN in patients with head and neck cancer</span></div><div class="casAuthors">Liu, Baoling; Huang, Guanhong; Zhu, Hongming; Ma, Zhaoming; Tian, Xiaokang; Yin, Li; Gao, Xingya; He, Xia</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2168-2180</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1791-2431</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">In patients with head and neck cancer (HNC), lymph node (N) metastases are assocd. with cancer aggressiveness and poor prognosis.  Identifying meaningful gene modules and representative biomarkers relevant to the N stage helps predict prognosis and reveal mechanisms underlying tumor progression.  The present study used a step-wise approach for weighted gene co-expression network anal. (WGCNA).  Dataset GSE65858 was subjected to WGCNA.  RNA sequencing data of HNC downloaded from the Cancer Genome Atlas (TCGA) and dataset GSE39366 were utilized to validate the results.  Following data preprocessing, 4,295 genes were screened, and blue and black modules assocd. with the N stage of HNC were identified.  A total of 16 genes [keratinocyte differentiation assocd. protein, suprabasin, cornifelin (CNFN), small proline rich protein 1B, desmoglein 1 (DSG1), chromosome 10 open reading frame 99, keratin 16 pseudogene 3, gap junction protein β2, dermokine, LY6/PLAUR domain contg. 3, transmembrane protein 79, phospholipase A2 group IVE, transglutaminase 5, potassium two pore domain channel subfamily K member 6, involucrin, kallikrein related peptidase 8] that had a neg. assocn. with the N-stage in the blue module, and two genes (structural maintenance of chromosomes 4 and mutS homolog 6) that had a pos. assocn. in the black module, were identified to be candidate hub genes.  Following further validation in TCGA and dataset GSE65858, it was identified that CNFN and DSG1 were assocd. with the clin. stage of HNC.  Survival anal. of CNFN and DSG1 was subsequently performed.  Patients with increased expression of CNFN displayed better survival probability in dataset GSE65858 and TCGA.  Therefore, CNFN was selected as the hub gene for further verification in the Gene Expression Profiling Interactive Anal. database.  Finally, functional enrichment and gene set enrichment analyses were performed using datasets GSE65858 and GSE39366.  Three gene sets, namely 'P53 pathway', 'estrogen response early' and 'estrogen response late', were enriched in the two datasets.  In conclusion, CNFN, identified via the WGCNA algorithm, may contribute to the prediction of lymph node metastases and prognosis, probably by regulating the pathways assocd. with P53, and the early and late estrogen response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVtfkzm-Shn7Vg90H21EOLACvtfcHk0ljnckacvc--CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWmt7jI&md5=2e7fe9a69197f4e883f80f9ecf4b1d4a</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.3892%2For.2019.7019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2019.7019%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26atitle%3DAnalysis%2520of%2520gene%2520coexpression%2520network%2520reveals%2520prognostic%2520significance%2520of%2520CNFN%2520in%2520patients%2520with%2520head%2520and%2520neck%2520cancer%26jtitle%3DOncol.%2520Rep.%26date%3D2019%26volume%3D41%26spage%3D2168%26epage%3D2180%26doi%3D10.3892%2For.2019.7019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L. K.</span></span> <span> </span><span class="NLM_article-title">Snail interacts with Id2 in the regulation of TNF-alpha-induced cancer cell invasion and migration in OSCC</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1680</span>– <span class="NLM_lpage">1691</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26175937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWrsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=1680-1691&issue=5&author=J.+P.+Zhouauthor=Z.+L.+Gaoauthor=M.+L.+Zhouauthor=M.+Y.+Heauthor=X.+H.+Xuauthor=D.+T.+Taoauthor=C.+C.+Yangauthor=L.+K.+Liu&title=Snail+interacts+with+Id2+in+the+regulation+of+TNF-alpha-induced+cancer+cell+invasion+and+migration+in+OSCC"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Snail interacts with Id2 in the regulation of TNF-α-induced cancer cell invasion and migration in OSCC</span></div><div class="casAuthors">Zhou, Jing-Ping; Gao, Zhen-Lin; Zhou, Mei-Ling; He, Meng-Ying; Xu, Xiao-Hui; Tao, De-Tao; Yang, Cong-Chong; Liu, Lai-Kui</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1680-1691</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">The inflammatory tumor microenvironment has been identified to play a pivotal role in tumor development and metastasis.  Tumor necrosis factor-α (TNF-α) is one of the key cytokines that regulate the inflammatory processes in tumor promotion.  In the current study, we treated three oral squamous cell carcinoma (OSCC) cell lines with TNF-α to study its role in inflammation-induced tumor progression.  Here we show that TNF-α induces stabilization of the transcriptional repressor Snail and activates NF-κB pathway in the three OSCC cell lines.  These activities resulted in the increased motility and invasiveness of three OSCC cell lines.  In addn., upon dealing with TNF-α for the indicated time, three OSCC cell lines underwent epithelial-to-mesenchymal transition (EMT), in which they presented a fibroblast-like phenotype and had a decreased expression of epithelial marker (E-cadherin) and an increased expression of mesenchymal marker (vimentin).  We further demonstrated that TNF-α can up-regulate the expression of Id2 while inducing an EMT in oral cancer cells.  Finally, we showed that Id2 interacted with Snail which may constrain Snail-dependent suppression of E-cadherin.  In conclusion, our study indicates that TNF-α induces Snail stabilization is dependent on the activation of NF-κB pathway and results in increasing cell invasion and migration in OSCC cells.  Id2 may contribute to regulate the function of Snail during TNF-α-mediated EMT in OSCC.  These findings have significant implications for inflammation-induced tumor promotion in OSCC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfetY20QE4yrVg90H21EOLACvtfcHk0ljnckacvc--CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWrsrjM&md5=38d2723f9c547852b3cbdc48b7fd8450</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%2BP.%26aulast%3DGao%26aufirst%3DZ.%2BL.%26aulast%3DZhou%26aufirst%3DM.%2BL.%26aulast%3DHe%26aufirst%3DM.%2BY.%26aulast%3DXu%26aufirst%3DX.%2BH.%26aulast%3DTao%26aufirst%3DD.%2BT.%26aulast%3DYang%26aufirst%3DC.%2BC.%26aulast%3DLiu%26aufirst%3DL.%2BK.%26atitle%3DSnail%2520interacts%2520with%2520Id2%2520in%2520the%2520regulation%2520of%2520TNF-alpha-induced%2520cancer%2520cell%2520invasion%2520and%2520migration%2520in%2520OSCC%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2015%26volume%3D5%26issue%3D5%26spage%3D1680%26epage%3D1691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monisha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padmavathi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banik, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordoloi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khwairakpam, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arfuso, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnathambi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alahmadi, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alharbi, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunnumakkara, A. B.</span></span> <span> </span><span class="NLM_article-title">NGAL is downregulated in oral squamous cell carcinoma and leads to increased survival, proliferation, migration and chemoresistance</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">228</span>, <span class="refDoi"> DOI: 10.3390/cancers10070228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.3390%2Fcancers10070228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gru7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=228&issue=7&author=J.+Monishaauthor=N.+K.+Royauthor=G.+Padmavathiauthor=K.+Banikauthor=D.+Bordoloiauthor=A.+D.+Khwairakpamauthor=F.+Arfusoauthor=A.+Chinnathambiauthor=T.+A.+Alahmadiauthor=S.+A.+Alharbiauthor=G.+Sethiauthor=A.+P.+Kumarauthor=A.+B.+Kunnumakkara&title=NGAL+is+downregulated+in+oral+squamous+cell+carcinoma+and+leads+to+increased+survival%2C+proliferation%2C+migration+and+chemoresistance&doi=10.3390%2Fcancers10070228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">NGAL is downregulated in oral squamous cell carcinoma and leads to increased survival, proliferation, migration and chemoresistance</span></div><div class="casAuthors">Monisha, Javadi; Roy, Nand Kishor; Padmavathi, Ganesan; Banik, Kishore; Bordoloi, Devivasha; Khwairakpam, Amrita Devi; Arfuso, Frank; Chinnathambi, Arunachalam; Alahmadi, Tahani Awad; Alharbi, Sulaiman Ali; Sethi, Gautam; Kumar, Alan Prem; Kunnumakkara, Ajaikumar B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">228/1-228/18</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Oral cancer is a major public health burden worldwide.  The lack of biomarkers for early diagnosis has increased the difficulty in managing this disease.  Recent studies have reported that neutrophil gelatinase-assocd. lipocalin (NGAL), a secreted glycoprotein, is upregulated in various tumors.  In our study, we found that NGAL was significantly downregulated in primary malignant and metastatic tissues of oral cancer in comparison to normal tissues.  The downregulation of NGAL was strongly correlated with both degree of differentiation and stage (I-IV); it can also serve as a prognostic biomarker for oral cancer.  Addnl., tobacco carcinogens were found to be involved in the downregulation of NGAL.  Mechanistic studies revealed that knockdown of NGAL increased oral cancer cell proliferation, survival, and migration; it also induced resistance against cisplatin.  Silencing of NGAL activated mammalian target of rapamycin (mTOR)signaling and reduced autophagy by the liver kinase B1 (LKB1)-activated protein kinase (AMPK)-p53-Redd1 signaling axis.  Moreover, cyclin-D1, Bcl-2, and matrix metalloproteinase-9 (MMP-9) were upregulated, and caspase-9 was downregulated, suggesting that silencing of NGAL increases oral cancer cell proliferation, survival, and migration.  Thus, from our study, it is evident that downregulation of NGAL activates the mTOR pathway and helps in the progression of oral cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMDYeK7OvbTrVg90H21EOLACvtfcHk0ljEjcM-lEiHGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gru7jE&md5=1ad6510ac61634da53628e6cd5de64be</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.3390%2Fcancers10070228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10070228%26sid%3Dliteratum%253Aachs%26aulast%3DMonisha%26aufirst%3DJ.%26aulast%3DRoy%26aufirst%3DN.%2BK.%26aulast%3DPadmavathi%26aufirst%3DG.%26aulast%3DBanik%26aufirst%3DK.%26aulast%3DBordoloi%26aufirst%3DD.%26aulast%3DKhwairakpam%26aufirst%3DA.%2BD.%26aulast%3DArfuso%26aufirst%3DF.%26aulast%3DChinnathambi%26aufirst%3DA.%26aulast%3DAlahmadi%26aufirst%3DT.%2BA.%26aulast%3DAlharbi%26aufirst%3DS.%2BA.%26aulast%3DSethi%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DA.%2BP.%26aulast%3DKunnumakkara%26aufirst%3DA.%2BB.%26atitle%3DNGAL%2520is%2520downregulated%2520in%2520oral%2520squamous%2520cell%2520carcinoma%2520and%2520leads%2520to%2520increased%2520survival%252C%2520proliferation%252C%2520migration%2520and%2520chemoresistance%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26issue%3D7%26spage%3D228%26doi%3D10.3390%2Fcancers10070228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pidugu, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pidugu, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. C.</span></span> <span> </span><span class="NLM_article-title">IFIT1 and IFIT3 promote oral squamous cell carcinoma metastasis and contribute to the anti-tumor effect of gefitinib via enhancing p-EGFR recycling</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">3232</span>– <span class="NLM_lpage">3247</span>, <span class="refDoi"> DOI: 10.1038/s41388-018-0662-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fs41388-018-0662-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=30626937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVahtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2019&pages=3232-3247&author=V.+K.+Piduguauthor=M.+M.+Wuauthor=A.+H.+Yenauthor=H.+B.+Piduguauthor=K.+W.+Changauthor=C.+J.+Liuauthor=T.+C.+Lee&title=IFIT1+and+IFIT3+promote+oral+squamous+cell+carcinoma+metastasis+and+contribute+to+the+anti-tumor+effect+of+gefitinib+via+enhancing+p-EGFR+recycling&doi=10.1038%2Fs41388-018-0662-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">IFIT1 and IFIT3 promote oral squamous cell carcinoma metastasis and contribute to the anti-tumor effect of gefitinib via enhancing p-EGFR recycling</span></div><div class="casAuthors">Pidugu, Vijaya Kumar; Wu, Meei-Maan; Yen, Ai-Hsin; Pidugu, Hima Bindu; Chang, Kuo-Wei; Liu, Chung-Ji; Lee, Te-Chang</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3232-3247</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">IFIT1 and IFIT3 are abundant products of interferon-stimulating genes.  While the importance of IFIT1 and IFIT3 in the prognosis of cancer has been reported, the mol. basis of IFIT1 and IFIT3 in cancer progression remains unexplored.  In the present study, we investigated the modes of action and the clin. significance of IFIT1 and IFIT3 in oral squamous cell carcinoma (OSCC).  Ectopic expression of IFIT1 or IFIT3 induced OSCC cell invasion by promoting the epithelial-mesenchymal transition, whereas IFIT1 or IFIT3 knockdown exhibited opposite effects.  Overexpression of IFIT1 or IFIT3 promoted tumor growth, regional and distant metastasis in xenograft and orthotopic nude mice models.  Most importantly, IFIT1 or IFIT3 overexpression increased the levels of p-EGFRY1068 and p-AKTS473 in OSCC cells and also enhanced tumor inhibitory effect of gefitinib.  By immunopptn. and LC-MS/MS anal., we found that IFIT1 and IFIT3 interacted with ANXA2 that enhanced p-EGFRY1068 endosomal recycling.  Depletion of ANXA2 using siRNA therefore abolished p-EGFRY1068 and p-AKTS473 expression in IFIT1- or IFIT3-overexpressed cells.  Furthermore, a significant pos. assocn. of increased IFIT1 and IFIT3 expression with advanced T-stage, lymph node metastasis, perineural invasion, lymphovascular invasion, extranodal extension, and poor overall survival rate was confirmed in OSCC patients.  We also found a statistically pos. correlation of p-EGFRY1068 expression with IFIT1 and IFIT3 in OSCC tumors and poor clin. outcome in patients.  Collectively, we demonstrated a novel role of IFIT1 and IFIT3 in driving OSCC progression and metastasis by interacting with ANXA2 and hence enhancing p-EGFR recycling and its downstream signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNnDAJ74IIA7Vg90H21EOLACvtfcHk0ljEjcM-lEiHGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVahtb4%253D&md5=a0113a4f427d9f825723349418c810b6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41388-018-0662-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41388-018-0662-9%26sid%3Dliteratum%253Aachs%26aulast%3DPidugu%26aufirst%3DV.%2BK.%26aulast%3DWu%26aufirst%3DM.%2BM.%26aulast%3DYen%26aufirst%3DA.%2BH.%26aulast%3DPidugu%26aufirst%3DH.%2BB.%26aulast%3DChang%26aufirst%3DK.%2BW.%26aulast%3DLiu%26aufirst%3DC.%2BJ.%26aulast%3DLee%26aufirst%3DT.%2BC.%26atitle%3DIFIT1%2520and%2520IFIT3%2520promote%2520oral%2520squamous%2520cell%2520carcinoma%2520metastasis%2520and%2520contribute%2520to%2520the%2520anti-tumor%2520effect%2520of%2520gefitinib%2520via%2520enhancing%2520p-EGFR%2520recycling%26jtitle%3DOncogene%26date%3D2019%26volume%3D38%26spage%3D3232%26epage%3D3247%26doi%3D10.1038%2Fs41388-018-0662-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aldinucci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombatti, A.</span></span> <span> </span><span class="NLM_article-title">The inflammatory chemokine CCL5 and cancer progression</span>. <i>Mediators Inflammation</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">292376</span>, <span class="refDoi"> DOI: 10.1155/2014/292376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1155%2F2014%2F292376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24523569" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A280%3ADC%252BC2crmsVKmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=292376&author=D.+Aldinucciauthor=A.+Colombatti&title=The+inflammatory+chemokine+CCL5+and+cancer+progression&doi=10.1155%2F2014%2F292376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">The inflammatory chemokine CCL5 and cancer progression</span></div><div class="casAuthors">Aldinucci Donatella; Colombatti Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Mediators of inflammation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">292376</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Until recently, inflammatory chemokines were viewed mainly as indispensable "gate keepers" of immunity and inflammation.  However, updated research indicates that cancer cells subvert the normal chemokine system and these molecules and their receptors become important constituents of the tumor microenvironment with very different ways to exert tumor-promoting roles.  The CCR5 and the CCL5 ligand have been detected in some hematological malignancies, lymphomas, and a great number of solid tumors, but extensive studies on the role of the CCL5/CCR axis were performed only in a limited number of cancers.  This review summarizes updated information on the role of CCL5 and its receptor CCR5 in cancer cell proliferation, metastasis, and the formation of an immunosuppressive microenvironment and highlights the development of newer therapeutic strategies aimed to inhibit the binding of CCL5 to CCR5, to inhibit CCL5 secretion, or to inhibit the interactions among tumor cells and the microenvironment leading to CCL5 secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWZ0g10qbTe9j_Vg2vObihfW6udTcc2eYsigIZD3uydLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2crmsVKmtQ%253D%253D&md5=3739b5f31946ab456c3ac3e2bb9f929e</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1155%2F2014%2F292376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F292376%26sid%3Dliteratum%253Aachs%26aulast%3DAldinucci%26aufirst%3DD.%26aulast%3DColombatti%26aufirst%3DA.%26atitle%3DThe%2520inflammatory%2520chemokine%2520CCL5%2520and%2520cancer%2520progression%26jtitle%3DMediators%2520Inflammation%26date%3D2014%26volume%3D2014%26spage%3D292376%26doi%3D10.1155%2F2014%2F292376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. H.</span></span> <span> </span><span class="NLM_article-title">Autocrine CCL5 promotes tumor progression in esophageal squamous cell carcinoma in vitro</span>. <i>Cytokine</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1016/j.cyto.2018.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.cyto.2018.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29705397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFWntr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2018&pages=94-103&author=Y.+C.+Wuauthor=Y.+C.+Shenauthor=J.+W.+C.+Changauthor=J.+J.+Hsiehauthor=Y.+Chuauthor=C.+H.+Wang&title=Autocrine+CCL5+promotes+tumor+progression+in+esophageal+squamous+cell+carcinoma+in+vitro&doi=10.1016%2Fj.cyto.2018.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Autocrine CCL5 promotes tumor progression in esophageal squamous cell carcinoma in vitro</span></div><div class="casAuthors">Wu, Yi-Cheng; Shen, Yung-Chi; Chang, John Wen-Cheng; Hsieh, Jia-Juan; Chu, Yen; Wang, Cheng-Hsu</div><div class="citationInfo"><span class="NLM_cas:title">Cytokine+</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">94-103</span>CODEN:
                <span class="NLM_cas:coden">CYTIE9</span>;
        ISSN:<span class="NLM_cas:issn">1043-4666</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The pro-tumoral effects of CCL5 have been identified in numerous cancer types.  We successfully cultivated 4 esophageal squamous cell carcinoma (ESCC) cell lines, including TWES-1, TWES-3 and a pair of cell lines derived from primary lesion (TWES-4PT) and metastatic lymph node (TWES-4LN) of the same patient.  Whole genome screening showed that TWES-4LN expressed higher levels of CCL5 compared to that of TWES-4PT; quantification of protein secretion displayed comparable results, suggesting that CCL5 could be assocd. with lymph node metastasis in ESCC.  CCL5 knockdown by siRNA significantly reduced basal growth rate, tumor migration and invasiveness in the paired cell lines; whereas this treatment induced cell apoptosis in TWES-1 and TWES-3.  CCR5 antagonist maraviroc significantly inhibited tumor migration and invasion in the paired cell lines without affecting tumor growth.  Collectively, these results suggest that CCL5 autocrine loop may promote ESCC progression; targeting the CCL5/CCR5 axis could be a potential therapeutic strategy for this deadly disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaeRSzQ3o6y7Vg90H21EOLACvtfcHk0ljEjcM-lEiHGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFWntr8%253D&md5=48df28d247242a3b321cb010ea27bbb6</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2Fj.cyto.2018.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cyto.2018.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%2BC.%26aulast%3DShen%26aufirst%3DY.%2BC.%26aulast%3DChang%26aufirst%3DJ.%2BW.%2BC.%26aulast%3DHsieh%26aufirst%3DJ.%2BJ.%26aulast%3DChu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%2BH.%26atitle%3DAutocrine%2520CCL5%2520promotes%2520tumor%2520progression%2520in%2520esophageal%2520squamous%2520cell%2520carcinoma%2520in%2520vitro%26jtitle%3DCytokine%26date%3D2018%26volume%3D110%26spage%3D94%26epage%3D103%26doi%3D10.1016%2Fj.cyto.2018.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barbasiewicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciniak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorynski, M.</span></span> <span> </span><span class="NLM_article-title">Phase transfer alkylation of arylacetonitriles revisited</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">3871</span>– <span class="NLM_lpage">3874</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2006.03.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.tetlet.2006.03.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFertrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=3871-3874&issue=23&author=M.+Barbasiewiczauthor=K.+Marciniakauthor=M.+Fedorynski&title=Phase+transfer+alkylation+of+arylacetonitriles+revisited&doi=10.1016%2Fj.tetlet.2006.03.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Phase transfer alkylation of arylacetonitriles revisited</span></div><div class="casAuthors">Barbasiewicz, Michal; Marciniak, Karolina; Fedorynski, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3871-3874</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phase-transfer alkylations of phenylacetonitrile derivs. carried out in the presence of 60-75% aq. KOH, instead of the typical 50% NaOH, provide substantial improvements in the overall yields and purity of products.  Reactions with simple secondary alkyl halides, as well as cycloalkylations with 1,2- and 1,3-dihalo alkanes proceed with good yields.  Increasing the concn. of base diminishes the formation of byproducts from competitive β-elimination processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHy3iKcyqazbVg90H21EOLACvtfcHk0lgVX-Z7WYPysA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFertrw%253D&md5=05b7e265e24b2405c0456e51a61a6811</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2006.03.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2006.03.176%26sid%3Dliteratum%253Aachs%26aulast%3DBarbasiewicz%26aufirst%3DM.%26aulast%3DMarciniak%26aufirst%3DK.%26aulast%3DFedorynski%26aufirst%3DM.%26atitle%3DPhase%2520transfer%2520alkylation%2520of%2520arylacetonitriles%2520revisited%26jtitle%3DTetrahedron%2520Lett.%26date%3D2006%26volume%3D47%26issue%3D23%26spage%3D3871%26epage%3D3874%26doi%3D10.1016%2Fj.tetlet.2006.03.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chae, J.</span></span> <span> </span><span class="NLM_article-title">Practical demethylation of aryl methyl ethers using an odorless thiol reagent</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>31</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1007/s12272-001-1156-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2Fs12272-001-1156-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=18409042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1ajtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2008&pages=305-309&issue=3&author=J.+Chae&title=Practical+demethylation+of+aryl+methyl+ethers+using+an+odorless+thiol+reagent&doi=10.1007%2Fs12272-001-1156-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Practical demethylation of aryl methyl ethers using an odorless thiol reagent</span></div><div class="casAuthors">Chae, Junghyun</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">305-309</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">A highly practical method for demethylation of aryl Me ethers employing a long-chain thiol was developed.  Under the conditions described herein, clean and fast conversions to the desired phenolic compds. were achieved with a broad range of substrates.  Unlike other thiolate-mediated methods, this newly developed protocol features in-situ generation of sodium alkylthiolate using NaOH, and is almost free from foul smells and potentially harmful gases.  It therefore provided an attractive option for the demethylation of aryl Me ethers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofl51-WtrYQLVg90H21EOLACvtfcHk0lgVX-Z7WYPysA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1ajtb8%253D&md5=e0affc1b45d71efa2acd83b927baa4bf</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs12272-001-1156-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-001-1156-y%26sid%3Dliteratum%253Aachs%26aulast%3DChae%26aufirst%3DJ.%26atitle%3DPractical%2520demethylation%2520of%2520aryl%2520methyl%2520ethers%2520using%2520an%2520odorless%2520thiol%2520reagent%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2008%26volume%3D31%26issue%3D3%26spage%3D305%26epage%3D309%26doi%3D10.1007%2Fs12272-001-1156-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussbaumer, T.</span></span> <span> </span><span class="NLM_article-title">2-(Diethylamino) ethanethiol, a new reagent for the odorless deprotection of aromatic methyl ethers</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">18</span>),  <span class="NLM_fpage">7103</span>– <span class="NLM_lpage">7105</span>, <span class="refDoi"> DOI: 10.1021/jo0611059</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo0611059" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvFOnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=7103-7105&issue=18&author=J.+Maganoauthor=M.+H.+Chenauthor=J.+D.+Clarkauthor=T.+Nussbaumer&title=2-%28Diethylamino%29+ethanethiol%2C+a+new+reagent+for+the+odorless+deprotection+of+aromatic+methyl+ethers&doi=10.1021%2Fjo0611059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">2-(Diethylamino)ethanethiol, a new reagent for the odorless deprotection of aromatic methyl ethers</span></div><div class="casAuthors">Magano, Javier; Chen, Michael H.; Clark, Jerry D.; Nussbaumer, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7103-7105</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new reagent for the deprotection of arom. Me ethers, 2-(diethylamino)ethanethiol, was reported.  This compd., com. available as its HCl salt, afforded the corresponding phenols in good to excellent yields on a wide variety of substrates.  A clear advantage of this method over the use of more common thiols, such as ethanethiol, was the easy extn. of both the deprotecting reagent and the byproduct 2-(diethylamino)ethyl Me sulfide into the aq. phase by quenching with dil. acid, which allowed an essentially odorless workup.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsWejx7R5QHrVg90H21EOLACvtfcHk0lgVX-Z7WYPysA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvFOnsrY%253D&md5=4613aa2752eb17ae109aed146dbf8d4e</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjo0611059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo0611059%26sid%3Dliteratum%253Aachs%26aulast%3DMagano%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DM.%2BH.%26aulast%3DClark%26aufirst%3DJ.%2BD.%26aulast%3DNussbaumer%26aufirst%3DT.%26atitle%3D2-%2528Diethylamino%2529%2520ethanethiol%252C%2520a%2520new%2520reagent%2520for%2520the%2520odorless%2520deprotection%2520of%2520aromatic%2520methyl%2520ethers%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2006%26volume%3D71%26issue%3D18%26spage%3D7103%26epage%3D7105%26doi%3D10.1021%2Fjo0611059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, C. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allerston, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammarstrom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daga, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berridge, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graslund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span> <span> </span><span class="NLM_article-title">Expressing the human proteome for affinity proteomics: optimizing expression of soluble protein domains and in vivo biotinylation</span>. <i>New Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">525</span>, <span class="refDoi"> DOI: 10.1016/j.nbt.2011.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1016%2Fj.nbt.2011.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=22027370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot12lsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2012&pages=515-525&issue=5&author=T.+Keatesauthor=C.+D.+O.+Cooperauthor=P.+Savitskyauthor=C.+K.+Allerstonauthor=C.+Phillipsauthor=M.+Hammarstromauthor=N.+Dagaauthor=G.+Berridgeauthor=P.+Mahajanauthor=N.+A.+Burgess-Brownauthor=S.+Mullerauthor=S.+Graslundauthor=O.+Gileadi&title=Expressing+the+human+proteome+for+affinity+proteomics%3A+optimizing+expression+of+soluble+protein+domains+and+in+vivo+biotinylation&doi=10.1016%2Fj.nbt.2011.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Expressing the human proteome for affinity proteomics: optimising expression of soluble protein domains and in vivo biotinylation</span></div><div class="casAuthors">Keates, Tracy; Cooper, Christopher D. O.; Savitsky, Pavel; Allerston, Charles K.; Phillips, Claire; Hammarstrom, Martin; Daga, Neha; Berridge, Georgina; Mahajan, Pravin; Burgess-Brown, Nicola A.; Muller, Susanne; Graslund, Susanne; Gileadi, Opher</div><div class="citationInfo"><span class="NLM_cas:title">New Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">515-525</span>CODEN:
                <span class="NLM_cas:coden">NBEIBR</span>;
        ISSN:<span class="NLM_cas:issn">1871-6784</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The generation of affinity reagents to large nos. of human proteins depends on the ability to express the target proteins as high-quality antigens.  The Structural Genomics Consortium (SGC) focuses on the prodn. and structure detn. of human proteins.  In a 7-yr period, the SGC has deposited crystal structures of >800 human protein domains, and has addnl. expressed and purified a similar no. of protein domains that have not yet been crystd.  The targets include a diversity of protein domains, with an attempt to provide high coverage of protein families.  The family approach provides an excellent basis for characterizing the selectivity of affinity reagents.  We present a summary of the approaches used to generate purified human proteins or protein domains, a test case demonstrating the ability to rapidly generate new proteins, and an optimization study on the modification of >70 proteins by biotinylation in vivo.  These results provide a unique synergy between large-scale structural projects and the recent efforts to produce a wide coverage of affinity reagents to the human proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt-7YKiDQFgLVg90H21EOLACvtfcHk0lgRNJBEmlM4bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot12lsrg%253D&md5=6d74e519f2df7e04dafd6e2691818939</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.nbt.2011.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.nbt.2011.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DCooper%26aufirst%3DC.%2BD.%2BO.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DAllerston%26aufirst%3DC.%2BK.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DHammarstrom%26aufirst%3DM.%26aulast%3DDaga%26aufirst%3DN.%26aulast%3DBerridge%26aufirst%3DG.%26aulast%3DMahajan%26aufirst%3DP.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DGraslund%26aufirst%3DS.%26aulast%3DGileadi%26aufirst%3DO.%26atitle%3DExpressing%2520the%2520human%2520proteome%2520for%2520affinity%2520proteomics%253A%2520optimizing%2520expression%2520of%2520soluble%2520protein%2520domains%2520and%2520in%2520vivo%2520biotinylation%26jtitle%3DNew%2520Biotechnol.%26date%3D2012%26volume%3D29%26issue%3D5%26spage%3D515%26epage%3D525%26doi%3D10.1016%2Fj.nbt.2011.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data processing and analysis with the autoPROC toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lgRNJBEmlM4bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgRNJBEmlM4bQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0ljT2UFOmT6whg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljT2UFOmT6whg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljT2UFOmT6whg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgXbYTl26xN6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgXbYTl26xN6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span> <span> </span><span class="NLM_article-title">Profiling of small molecules by chemical proteomics</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1394</i></span>,  <span class="NLM_fpage">211</span>– <span class="NLM_lpage">218</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-3341-9_15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1007%2F978-1-4939-3341-9_15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=26700051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1agtrrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1394&publication_year=2016&pages=211-218&author=K.+V.+M.+Huberauthor=G.+Superti-Furga&title=Profiling+of+small+molecules+by+chemical+proteomics&doi=10.1007%2F978-1-4939-3341-9_15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling of small molecules by chemical proteomics</span></div><div class="casAuthors">Huber, Kilian V. M.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1394</span>
        (<span class="NLM_cas:issue">Proteomics in Systems Biology</span>),
    <span class="NLM_cas:pages">211-218</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Chem. proteomics provides a powerful means to gain systems-level insight into the mode of action of small mols. and/or natural products.  In contrast to high-throughput screening efforts which only interrogate selected subproteomes such as kinases and often only consider individual domains, the methodol. presented herein allows for the detn. of the mol. targets of small mols. or drugs in a more relevant physiol. setting.  As such, the compd. of interest is exposed to the entire variety of cellular proteins considering all naturally occurring posttranslational modifications and activation states.  Samples prepd. according to the procedures described in this protocol are compatible with lysates from cultured cell lines, primary cells, or samples from biopsies.  In combination with state-of-the-art mass spectrometry techniques this approach grants access to a comprehensive view of small mol.-target protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqu_eJ-ips8bVg90H21EOLACvtfcHk0liyN7TG-YU8fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1agtrrE&md5=310ab7cb29d9eb4c39b945a5b19b7bd9</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-3341-9_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-3341-9_15%26sid%3Dliteratum%253Aachs%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DProfiling%2520of%2520small%2520molecules%2520by%2520chemical%2520proteomics%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2016%26volume%3D1394%26spage%3D211%26epage%3D218%26doi%3D10.1007%2F978-1-4939-3341-9_15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langmead, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzberg, S. L.</span></span> <span> </span><span class="NLM_article-title">HISAT: a fast spliced aligner with low memory requirements</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">357</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1038/nmeth.3317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1038%2Fnmeth.3317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25751142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvFOnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=357-360&issue=4&author=D.+Kimauthor=B.+Langmeadauthor=S.+L.+Salzberg&title=HISAT%3A+a+fast+spliced+aligner+with+low+memory+requirements&doi=10.1038%2Fnmeth.3317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">HISAT: a fast spliced aligner with low memory requirements</span></div><div class="casAuthors">Kim, Daehwan; Langmead, Ben; Salzberg, Steven L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">357-360</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">HISAT (hierarchical indexing for spliced alignment of transcripts) is a highly efficient system for aligning reads from RNA sequencing expts.  HISAT uses an indexing scheme based on the Burrows-Wheeler transform and the Ferragina-Manzini (FM) index, employing two types of indexes for alignment: a whole-genome FM index to anchor each alignment and numerous local FM indexes for very rapid extensions of these alignments.  HISAT's hierarchical index for the human genome contains 48,000 local FM indexes, each representing a genomic region of ∼64,000 bp.  Tests on real and simulated data sets showed that HISAT is the fastest system currently available, with equal or better accuracy than any other method.  Despite its large no. of indexes, HISAT requires only 4.3 gigabytes of memory.  HISAT supports genomes of any size, including those larger than 4 billion bases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKIAQt7RlLl7Vg90H21EOLACvtfcHk0ljifLVzYHwyLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvFOnsL0%253D&md5=6a0895120a3b49200f558b4cf810f89f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.3317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.3317%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DD.%26aulast%3DLangmead%26aufirst%3DB.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26atitle%3DHISAT%253A%2520a%2520fast%2520spliced%2520aligner%2520with%2520low%2520memory%2520requirements%26jtitle%3DNat.%2520Methods%26date%3D2015%26volume%3D12%26issue%3D4%26spage%3D357%26epage%3D360%26doi%3D10.1038%2Fnmeth.3317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span> <span> </span><span class="NLM_article-title">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">930</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt656</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1093%2Fbioinformatics%2Fbtt656" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=24227677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFGqu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=923-930&issue=7&author=Y.+Liaoauthor=G.+K.+Smythauthor=W.+Shi&title=featureCounts%3A+an+efficient+general+purpose+program+for+assigning+sequence+reads+to+genomic+features&doi=10.1093%2Fbioinformatics%2Fbtt656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features</span></div><div class="casAuthors">Liao, Yang; Smyth, Gordon K.; Shi, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">923-930</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Next-generation sequencing technologies generate millions of short sequence reads, which are usually aligned to a ref. genome.  In many applications, the key information required for downstream anal. is the no. of reads mapping to each genomic feature, for example to each exon or each gene.  The process of counting reads is called read summarization.  Read summarization is required for a great variety of genomic analyses but has so far received relatively little attention in the literature.  Results: We present featureCounts, a read summarization program suitable for counting reads generated from either RNA or genomic DNA sequencing expts. featureCounts implements highly efficient chromosome hashing and feature blocking techniques.  It is considerably faster than existing methods (by an order of magnitude for gene-level summarization) and requires far less computer memory.  It works with either single or paired-end reads and provides a wide range of options appropriate for different sequencing applications.  Availability and implementation: featureCounts is available under GNU General Public License as part of the Subread (http://subread.sourceforge.net) or Rsubread (http://www.bioconductor.org) software packages.  Contact: <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="6f1c07062f180a0706410a0b1a410e1a41">[email protected]</a></div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF_vNcz7InHLVg90H21EOLACvtfcHk0ljifLVzYHwyLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFGqu7c%253D&md5=6549e3ff389f50b088acae7972ddf7af</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt656%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DY.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26aulast%3DShi%26aufirst%3DW.%26atitle%3DfeatureCounts%253A%2520an%2520efficient%2520general%2520purpose%2520program%2520for%2520assigning%2520sequence%2520reads%2520to%2520genomic%2520features%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26issue%3D7%26spage%3D923%26epage%3D930%26doi%3D10.1093%2Fbioinformatics%2Fbtt656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Love, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, S.</span></span> <span> </span><span class="NLM_article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">550</span>, <span class="refDoi"> DOI: 10.1186/s13059-014-0550-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1186%2Fs13059-014-0550-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=25516281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVCrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=550&issue=12&author=M.+I.+Loveauthor=W.+Huberauthor=S.+Anders&title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2&doi=10.1186%2Fs13059-014-0550-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span></div><div class="casAuthors">Love, Michael I.; Huber, Wolfgang; Anders, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Genome Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">550/1-550/21, 21 pp.</span>CODEN:
                <span class="NLM_cas:coden">GNBLFW</span>;
        ISSN:<span class="NLM_cas:issn">1474-760X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">In comparative high-throughput sequencing assays, a fundamental task is the anal. of count data, such as read counts per gene in RNA-seq, for evidence of systematic changes across exptl. conditions.  Small replicate nos., discreteness, large dynamic range and the presence of outliers require a suitable statistical approach.  We present DESeq2, a method for differential anal. of count data, using shrinkage estn. for dispersions and fold changes to improve stability and interpretability of ests.  This enables a more quant. anal. focused on the strength rather than the mere presence of differential expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-iepNJ6mO27Vg90H21EOLACvtfcHk0ljifLVzYHwyLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVCrsL8%253D&md5=e1052687faf6b297cdf615a14676c5b0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1186%2Fs13059-014-0550-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13059-014-0550-8%26sid%3Dliteratum%253Aachs%26aulast%3DLove%26aufirst%3DM.%2BI.%26aulast%3DHuber%26aufirst%3DW.%26aulast%3DAnders%26aufirst%3DS.%26atitle%3DModerated%2520estimation%2520of%2520fold%2520change%2520and%2520dispersion%2520for%2520RNA-seq%2520data%2520with%2520DESeq2%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26issue%3D12%26spage%3D550%26doi%3D10.1186%2Fs13059-014-0550-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabregat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jupe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidiropoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garapati, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haw, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jassal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korninger, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milacic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothfels, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamovsky, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varusai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viteri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Eustachio, P.</span></span> <span> </span><span class="NLM_article-title">The reactome pathway knowledgebase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">D649</span>– <span class="NLM_lpage">D655</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=10.1093%2Fnar%2Fgkx1132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=29145629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGis7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D649-D655&author=A.+Fabregatauthor=S.+Jupeauthor=L.+Matthewsauthor=K.+Sidiropoulosauthor=M.+Gillespieauthor=P.+Garapatiauthor=R.+Hawauthor=B.+Jassalauthor=F.+Korningerauthor=B.+Mayauthor=M.+Milacicauthor=C.+D.+Rocaauthor=K.+Rothfelsauthor=C.+Sevillaauthor=V.+Shamovskyauthor=S.+Shorserauthor=T.+Varusaiauthor=G.+Viteriauthor=J.+Weiserauthor=G.+M.+Wuauthor=L.+Steinauthor=H.+Hermjakobauthor=P.+D%E2%80%99Eustachio&title=The+reactome+pathway+knowledgebase&doi=10.1093%2Fnar%2Fgkx1132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The reactome pathway knowledgebase</span></div><div class="casAuthors">Fabregat, Antonio; Jupe, Steven; Matthews, Lisa; Sidiropoulos, Konstantinos; Gillespie, Marc; Garapati, Phani; Haw, Robin; Jassal, Bijay; Korninger, Florian; May, Bruce; Milacic, Marija; Roca, Corina Duenas; Rothfels, Karen; Sevilla, Cristoffer; Shamovsky, Veronica; Shorser, Solomon; Varusai, Thawfeek; Viteri, Guilherme; Weiser, Joel; Wu, Guanming; Stein, Lincoln; Hermjakob, Henning; D'eustachio, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D649-D655</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Reactome Knowledgebase provides mol. details of signal transduction, transport, DNA replication, metab., and other cellular processes as an ordered network of mol. transformations-an extended version of a classic metabolic map, in a single consistent data model.  Reactome functions both as an archive of biol. processes and as a tool for discovering unexpected functional relationships in data such as gene expression profiles or somatic mutation catalogs from tumor cells.  To support the continued brisk growth in the size and complexity of Reactome, we have implemented a graph database, improved performance of data anal. tools, and designed new data structures and strategies to boost diagram viewer performance.  To make our website more accessible to human users, we have improved pathway display and navigation by implementing interactive Enhanced High Level Diagrams (EHLDs) with an assocd. icon library, and subpathway highlighting and zooming, in a simplified and reorganized web site with adaptive design.  To encourage re-use of our content, we have enabled export of pathway diagrams as 'PowerPoint' files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouB231U71u1LVg90H21EOLACvtfcHk0ljifLVzYHwyLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGis7bM&md5=691a9b4fb2ab28dd06b9d012a3e936b7</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1132%26sid%3Dliteratum%253Aachs%26aulast%3DFabregat%26aufirst%3DA.%26aulast%3DJupe%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DSidiropoulos%26aufirst%3DK.%26aulast%3DGillespie%26aufirst%3DM.%26aulast%3DGarapati%26aufirst%3DP.%26aulast%3DHaw%26aufirst%3DR.%26aulast%3DJassal%26aufirst%3DB.%26aulast%3DKorninger%26aufirst%3DF.%26aulast%3DMay%26aufirst%3DB.%26aulast%3DMilacic%26aufirst%3DM.%26aulast%3DRoca%26aufirst%3DC.%2BD.%26aulast%3DRothfels%26aufirst%3DK.%26aulast%3DSevilla%26aufirst%3DC.%26aulast%3DShamovsky%26aufirst%3DV.%26aulast%3DShorser%26aufirst%3DS.%26aulast%3DVarusai%26aufirst%3DT.%26aulast%3DViteri%26aufirst%3DG.%26aulast%3DWeiser%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DG.%2BM.%26aulast%3DStein%26aufirst%3DL.%26aulast%3DHermjakob%26aufirst%3DH.%26aulast%3DD%25E2%2580%2599Eustachio%26aufirst%3DP.%26atitle%3DThe%2520reactome%2520pathway%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DD649%26epage%3DD655%26doi%3D10.1093%2Fnar%2Fgkx1132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NVQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NVQ','PDB','4NVQ'); return false;">PDB: 4NVQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Y','PDB','6I8Y'); return false;">PDB: 6I8Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8B','PDB','6I8B'); return false;">PDB: 6I8B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4H75" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4H75','PDB','4H75'); return false;">PDB: 4H75</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MZG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4MZG','PDB','4MZG'); return false;">PDB: 4MZG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L','PDB','6I8L'); return false;">PDB: 6I8L</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LUG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LUG','PDB','5LUG'); return false;">PDB: 5LUG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A1H" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A1H','PDB','5A1H'); return false;">PDB: 5A1H</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i84"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00562">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_15785"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00562">10.1021/acs.jmedchem.9b00562</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">X-ray crystallographic data; analysis of ligand binding and sequence alignment of SPIN family proteins; biophysical and biochemical assays results; cellular assay results; browser track for selected genomic regions; results of in vitro stability, permeability and solubility assays; NMR spectra (with assignments) and HPLC chromatograms of end compounds; chemical structures; chemoproteomic data summary;  (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings, RNAseq results and reactome analysis (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_001.pdf">jm9b00562_si_001.pdf (12.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00562/suppl_file/jm9b00562_si_002.csv">jm9b00562_si_002.csv (1.36 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank: A-366 in complex with SPIN1 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8Y">6I8Y</a>; compound <b>23</b> in complex with SPIN1 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8L">6I8L</a>; compound <b>35</b> in complex with SPIN1 PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6I8B">6I8B</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00562&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00562%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-20%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00562" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b70afb7b23c8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
