
<html lang="en"     class="pb-page"  data-request-id="1f67878e-f61c-45ee-b931-f90f4f08e4cc"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.9b01006;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Kinase Chemodiversity from the Arctic: The Breitfussins" /></meta><meta name="dc.Creator" content="Kine Ø.  Hansen" /></meta><meta name="dc.Creator" content="Jeanette H.  Andersen" /></meta><meta name="dc.Creator" content="Annette  Bayer" /></meta><meta name="dc.Creator" content="Sunil K.  Pandey" /></meta><meta name="dc.Creator" content="Marianne  Lorentzen" /></meta><meta name="dc.Creator" content="Kåre B.  Jørgensen" /></meta><meta name="dc.Creator" content="Magne O.  Sydnes" /></meta><meta name="dc.Creator" content="Yngve  Guttormsen" /></meta><meta name="dc.Creator" content="Matthias  Baumann" /></meta><meta name="dc.Creator" content="Uwe  Koch" /></meta><meta name="dc.Creator" content="Bert  Klebl" /></meta><meta name="dc.Creator" content="Jan  Eickhoff" /></meta><meta name="dc.Creator" content="Bengt Erik  Haug" /></meta><meta name="dc.Creator" content="Johan  Isaksson" /></meta><meta name="dc.Creator" content="Espen H.  Hansen" /></meta><meta name="dc.Description" content="In this work, we demonstrate that the indole-oxazole-pyrrole framework of the breitfussin family of natural products is a promising scaffold for kinase inhibition. Six new halogenated natural produ..." /></meta><meta name="Description" content="In this work, we demonstrate that the indole-oxazole-pyrrole framework of the breitfussin family of natural products is a promising scaffold for kinase inhibition. Six new halogenated natural produ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 24, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01006" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01006" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01006" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01006" /></link>
        
    
    

<title>Kinase Chemodiversity from the Arctic: The Breitfussins | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01006" /></meta><meta property="og:title" content="Kinase Chemodiversity from the Arctic: The Breitfussins" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0011.jpeg" /></meta><meta property="og:description" content="In this work, we demonstrate that the indole-oxazole-pyrrole framework of the breitfussin family of natural products is a promising scaffold for kinase inhibition. Six new halogenated natural products, breitfussin C–H (3 – 8) were isolated and characterized from the Arctic, marine hydrozoan Thuiaria breitfussi. The structures of two of the new natural products were also confirmed by total synthesis. Two of the breitfussins (3 and 4) were found to selectively inhibit the survival of several cancer cell lines, with the lowest IC50 value of 340 nM measured against the drug-resistant triple negative breast cancer cell line MDA-MB-468, while leaving the majority of the tested cell lines not or significantly less affected. When tested against panels of protein kinases, 3 gave IC50 and Kd values as low as 200 and 390 nM against the PIM1 and DRAK1 kinases, respectively. The activity was confirmed to be mediated through ATP competitive binding in the ATP binding pocket of the kinases. Furthermore, evaluation of potential off-target and toxicological effects, as well as relevant in vitro ADME parameters for 3 revealed that the breitfussin scaffold holds promise for the development of selective kinase inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01006"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01006">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01006&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01006&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01006&amp;href=/doi/10.1021/acs.jmedchem.9b01006" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10167-10181</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01000" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01065" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Kinase Chemodiversity from the Arctic: The Breitfussins</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Kine Ø. Hansen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kine Ø. Hansen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Marbio, UiT - The Arctic University of Norway, Breivika, NO-9037 Tromsø, Norway</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#73381a1d165d3c5d3b121d0000161d33061a075d1d1c"><span class="__cf_email__" data-cfemail="7a3113141f543554321b1409091f143a0f130e541415">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kine+%C3%98.++Hansen">Kine Ø. Hansen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9023-1958" title="Orcid link">http://orcid.org/0000-0002-9023-1958</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeanette H. Andersen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeanette H. Andersen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Marbio, UiT - The Arctic University of Norway, Breivika, NO-9037 Tromsø, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeanette+H.++Andersen">Jeanette H. Andersen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Annette Bayer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Annette Bayer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, UiT - The Arctic University of Norway, Breivika, NO-9037 Tromsø, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Annette++Bayer">Annette Bayer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3481-200X" title="Orcid link">http://orcid.org/0000-0003-3481-200X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sunil K. Pandey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sunil K. Pandey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Centre for Pharmacy, University of Bergen, Allégaten 41, NO-5007 Bergen, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sunil+K.++Pandey">Sunil K. Pandey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marianne Lorentzen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marianne Lorentzen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Science and Technology, Department of Chemistry, Bioscience and Environmental Engineering, University of Stavanger, NO-4036 Stavanger, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marianne++Lorentzen">Marianne Lorentzen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kåre B. Jørgensen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kåre B. Jørgensen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Science and Technology, Department of Chemistry, Bioscience and Environmental Engineering, University of Stavanger, NO-4036 Stavanger, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=K%C3%A5re+B.++J%C3%B8rgensen">Kåre B. Jørgensen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0662-1839" title="Orcid link">http://orcid.org/0000-0003-0662-1839</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Magne O. Sydnes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Magne O. Sydnes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Science and Technology, Department of Chemistry, Bioscience and Environmental Engineering, University of Stavanger, NO-4036 Stavanger, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Magne+O.++Sydnes">Magne O. Sydnes</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9413-6969" title="Orcid link">http://orcid.org/0000-0001-9413-6969</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yngve Guttormsen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yngve Guttormsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, UiT - The Arctic University of Norway, Breivika, NO-9037 Tromsø, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yngve++Guttormsen">Yngve Guttormsen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthias Baumann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthias Baumann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthias++Baumann">Matthias Baumann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Uwe Koch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Uwe Koch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Uwe++Koch">Uwe Koch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bert Klebl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bert Klebl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bert++Klebl">Bert Klebl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jan Eickhoff</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jan Eickhoff</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Eickhoff">Jan Eickhoff</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bengt Erik Haug</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bengt Erik Haug</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Centre for Pharmacy, University of Bergen, Allégaten 41, NO-5007 Bergen, Norway</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#b7f5d2d9d0c39af2c5dedc99ffd6c2d0f7c2ded599d9d8"><span class="__cf_email__" data-cfemail="5f1d3a31382b721a2d363471173e2a381f2a363d713130">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bengt+Erik++Haug">Bengt Erik Haug</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3014-9538" title="Orcid link">http://orcid.org/0000-0003-3014-9538</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Johan Isaksson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Johan Isaksson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, UiT - The Arctic University of Norway, Breivika, NO-9037 Tromsø, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Johan++Isaksson">Johan Isaksson</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Espen H. Hansen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Espen H. Hansen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Marbio, UiT - The Arctic University of Norway, Breivika, NO-9037 Tromsø, Norway</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Espen+H.++Hansen">Espen H. Hansen</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01006&amp;href=/doi/10.1021%2Facs.jmedchem.9b01006" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10167–10181</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 24, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 June 2019</li><li><span class="item_label"><b>Published</b> online</span>24 October 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01006" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01006</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10167%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKine%2B%25C3%2598.%2BHansen%252C%2BJeanette%2BH.%2BAndersen%252C%2BAnnette%2BBayer%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D22%26contentID%3Dacs.jmedchem.9b01006%26title%3DKinase%2BChemodiversity%2Bfrom%2Bthe%2BArctic%253A%2BThe%2BBreitfussins%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10181%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01006"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1053</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">9</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01006" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Kinase Chemodiversity from the Arctic: The Breitfussins&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Kine&quot;,&quot;last_name&quot;:&quot;Ø. Hansen&quot;},{&quot;first_name&quot;:&quot;Jeanette&quot;,&quot;last_name&quot;:&quot;H. Andersen&quot;},{&quot;first_name&quot;:&quot;Annette&quot;,&quot;last_name&quot;:&quot;Bayer&quot;},{&quot;first_name&quot;:&quot;Sunil&quot;,&quot;last_name&quot;:&quot;K. Pandey&quot;},{&quot;first_name&quot;:&quot;Marianne&quot;,&quot;last_name&quot;:&quot;Lorentzen&quot;},{&quot;first_name&quot;:&quot;Kåre&quot;,&quot;last_name&quot;:&quot;B. Jørgensen&quot;},{&quot;first_name&quot;:&quot;Magne&quot;,&quot;last_name&quot;:&quot;O. Sydnes&quot;},{&quot;first_name&quot;:&quot;Yngve&quot;,&quot;last_name&quot;:&quot;Guttormsen&quot;},{&quot;first_name&quot;:&quot;Matthias&quot;,&quot;last_name&quot;:&quot;Baumann&quot;},{&quot;first_name&quot;:&quot;Uwe&quot;,&quot;last_name&quot;:&quot;Koch&quot;},{&quot;first_name&quot;:&quot;Bert&quot;,&quot;last_name&quot;:&quot;Klebl&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Eickhoff&quot;},{&quot;first_name&quot;:&quot;Bengt&quot;,&quot;last_name&quot;:&quot;Erik Haug&quot;},{&quot;first_name&quot;:&quot;Johan&quot;,&quot;last_name&quot;:&quot;Isaksson&quot;},{&quot;first_name&quot;:&quot;Espen&quot;,&quot;last_name&quot;:&quot;H. Hansen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10167-10181&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01006&quot;},&quot;abstract&quot;:&quot;In this work, we demonstrate that the indole-oxazole-pyrrole framework of the breitfussin family of natural products is a promising scaffold for kinase inhibition. Six new halogenated natural products, breitfussin C–H (3 – 8) were isolated and characterized from the Arctic, marine hydrozoan Thuiaria breitfussi. The structures of two of the new natural products were also confirmed by total synthesis. Two of the breitfussins (3 and 4) were found to selectively inhibit the survival of several cancer cell lines, with the lowest IC50 value of 340 nM measured against the drug-resistant triple negative breast cancer cell line MDA-MB-468, while leaving the majority of the tested cell lines not or significantly less affected. When tested against panels of protein kinases, 3 gave IC50 and Kd values as low as 200 and 390 nM against the PIM1 and DRAK1 kinases, respectively. The activity was confirmed to be mediated through ATP competitive binding in the ATP binding pocket of the kinases. Furthermore, evaluation of potent&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01006&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01006" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01006&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01006" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01006&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01006" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01006&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01006&amp;href=/doi/10.1021/acs.jmedchem.9b01006" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01006" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01006" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01006%26sid%3Dliteratum%253Aachs%26pmid%3D31647655%26genre%3Darticle%26aulast%3DHansen%26date%3D2019%26atitle%3DKinase%2BChemodiversity%2Bfrom%2Bthe%2BArctic%253A%2BThe%2BBreitfussins%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D22%26spage%3D10167%26epage%3D10181%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291337" title="Carbon">Carbon</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/jmcmar.2019.62.issue-22/20191127/jmcmar.2019.62.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">In this work, we demonstrate that the indole-oxazole-pyrrole framework of the breitfussin family of natural products is a promising scaffold for kinase inhibition. Six new halogenated natural products, breitfussin C–H (<b>3</b> – <b>8</b>) were isolated and characterized from the Arctic, marine hydrozoan <i>Thuiaria breitfussi</i>. The structures of two of the new natural products were also confirmed by total synthesis. Two of the breitfussins (<b>3</b> and <b>4</b>) were found to selectively inhibit the survival of several cancer cell lines, with the lowest IC<sub>50</sub> value of 340 nM measured against the drug-resistant triple negative breast cancer cell line MDA-MB-468, while leaving the majority of the tested cell lines not or significantly less affected. When tested against panels of protein kinases, <b>3</b> gave IC<sub>50</sub> and <i>K</i><sub>d</sub> values as low as 200 and 390 nM against the PIM1 and DRAK1 kinases, respectively. The activity was confirmed to be mediated through ATP competitive binding in the ATP binding pocket of the kinases. Furthermore, evaluation of potential off-target and toxicological effects, as well as relevant in vitro ADME parameters for <b>3</b> revealed that the breitfussin scaffold holds promise for the development of selective kinase inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30678" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30678" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The human genome encodes 538 protein kinases, many of which are known to have fundamental roles in the molecular machinery controlling cell proliferation, survival, and motility.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> As these processes are tightly regulated in normal cells, mutations causing dysregulation of kinase functions have been linked to a wide range of human malignancies.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Several small-molecule kinase inhibitors have been marketed as drugs to treat a variety of different cancers, including leukemia and gastrointestinal tumors.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Inhibition of kinase activity using selective inhibitors provides a targeted treatment option with overall fewer side effects compared to traditional cancer chemotherapeutics.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> As most kinase inhibitors bind to the highly conserved ATP-binding pocket, development of selective inhibitors is challenging.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> Accordingly, new scaffolds for selective kinase inhibition are in high demand.</div><div class="NLM_p">The breitfussins are a family of highly modified halogenated compounds comprising a unique indole-oxazole-pyrrole molecular framework. The first two compounds, breitfussin A and B (<b>1</b> and <b>2</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), were isolated from the cnidarian <i>Thuiaria breitfussi</i> and reported by us in 2012.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of <b>1</b>–<b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Due to the low number of protons compared to heavy atoms (ratio 1:2), the structures of <b>1</b> and <b>2</b> could not be elucidated by traditional methods alone, and atomic force microscopy (AFM) was used for the first time to elucidate the structure of unknown natural products.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The structures of <b>1</b> and <b>2</b> were later confirmed by total synthesis.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Natural products consisting of oxazole-pyrrole<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> or indole-oxazole<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> moieties have previously been reported; however, the breitfussins are the first examples of natural products containing all three heterocycles.</div><div class="NLM_p last">In the present work, we describe further chemical investigation of the organic extracts of <i>T. breitfussi</i> which led to the characterization of six new breitfussin analogues, breitfussin C–H (<b>3</b>–<b>8</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Previously, bioactivity data (against one colon cancer cell line, bacteria- and fungal strains) for synthetically produced <b>2</b> has been reported by others,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> while no bioactivity data has been reported for <b>1</b> and <b>3</b>–<b>8</b>. We herein report on the cytotoxic activity against cancer cell lines and kinase inhibition profiles of <b>1</b>–<b>6</b>. Further, we report the total synthesis of <b>3</b> and <b>4</b>, which followed a similar strategy as reported for <b>1</b> and <b>2</b>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> On the basis of initial bioactivity data, we performed an extended evaluation of the pharmacological potential of <b>3</b>, and the potential of the core structure of the breitfussins as a new ATP-competitive kinase inhibitor scaffold was demonstrated.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51065" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51065" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Isolation and Structural Elucidation</h3><div class="NLM_p">Specimens of <i>T. breitfussi</i> (475.6 g) were collected near Bjørnøya (Barents Sea) during a research cruise in 2011. We have previously reported the isolation and structure elucidation of <b>1</b> and <b>2</b> from a different <i>T. breitfussi</i> sample collected in the same area in 2007.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Repeated purification of the organic <i>T. breitfussi</i> extract, using semipreparative HPLC, led to the isolation of eight related compounds. Two of these were confirmed to be <b>1</b> and <b>2</b>, while <b>3</b>–<b>8</b> were determined to be new breitfussin analogues.</div><div class="NLM_p">Breitfussin C (<b>3</b>) was isolated as a light brown powder. HRMS analysis suggested a molecular formula of C<sub>16</sub>H<sub>12</sub>BrN<sub>3</sub>O<sub>2</sub> (12 degrees of unsaturation, <i>m</i>/<i>z</i> 358.0195 [M + H]<sup>+</sup>, Δmmu = 0.4). The <sup>1</sup>H NMR spectrum of <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) recorded in DMSO-<i>d</i><sub>6</sub> displayed signals for seven aromatic protons with chemical shift values ranging from 7.79 to 6.18, two exchangeable protons at δ<sub>H</sub> = 11.85 and 11.64, as well as signals for one methyl group attached to a heteroatom at δ<sub>H</sub> 3.91. The <sup>13</sup>C NMR spectrum of <b>3</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) confirmed the presence of 16 carbon atoms, including one methyl, seven methine, and eight quaternary carbon atoms. The NMR spectra of <b>3</b> revealed chemical shifts in close agreement with those of <b>1</b> and <b>2</b>. HMBC and HSQC experiments were used to determine the substitution pattern of the breitfussin skeleton of <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). In the HMBC spectrum of <b>3</b>, the methoxy protons (δ<sub>H</sub> = 3.91) correlated with C-6 (δ<sub>C</sub> 150.31). The downfield shift of C-6 in <b>3</b> is in agreement with the shift expected for a methoxylation in this position and was also confirmed by a NOE between H-5 (δ<sub>H</sub> = 7.38) and the methoxy protons (δ<sub>H</sub> = 3.91). C-7 (δ<sub>C</sub> 107.1) was quaternary, and attachment of the bromine atom to C-7 completed the planar structure of <b>3</b>. The structure of <b>3</b> was hence assigned as 5-(6-bromo-5-methoxy-1<i>H</i>-indol-3-yl)-2-(1<i>H</i>-pyrrol-2-yl)oxazole.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selected COSY, HMBC, and NOE correlations obtained for <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Summary of <sup>1</sup>H NMR (600 MHz, (CD<sub>3</sub>)<sub>2</sub>SO) Data for <b>3</b>–<b>8</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>3</b></th><th class="colsep0 rowsep0" align="center"><b>4</b></th><th class="colsep0 rowsep0" align="center"><b>5</b></th><th class="colsep0 rowsep0" align="center"><b>6</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th><th class="colsep0 rowsep0" align="center"><b>8</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">position</th><th class="colsep0 rowsep0" align="center">δ<sub>H</sub> (<i>J</i> in Hz)</th><th class="colsep0 rowsep0" align="center">δ<sub>H</sub> (<i>J</i> in Hz)</th><th class="colsep0 rowsep0" align="center">δ<sub>H</sub> (<i>J</i> in Hz)</th><th class="colsep0 rowsep0" align="center">δ<sub>H</sub> (<i>J</i> in Hz)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">δ<sub>H</sub> (<i>J</i> in Hz)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">δ<sub>H</sub> (<i>J</i> in Hz)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">11.64, d (2.6)</td><td class="colsep0 rowsep0" align="left">11.75, s</td><td class="colsep0 rowsep0" align="left">12.00, s</td><td class="colsep0 rowsep0" align="left">12.25, s</td><td class="colsep0 rowsep0" align="left">11.55, s</td><td class="colsep0 rowsep0" align="left">11.55, s</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">7.79, d (2.7)</td><td class="colsep0 rowsep0" align="left">7.74, s</td><td class="colsep0 rowsep0" align="left">7.76, s</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">7.83, s</td><td class="colsep0 rowsep0" align="left">7.83, s</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">7.38, s</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">7.42, s</td><td class="colsep0 rowsep0" align="left">7.42, s</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">6.73, d (1.6)</td><td class="colsep0 rowsep0" align="left">6.73,d (1.6)</td><td class="colsep0 rowsep0" align="left">6.67, d (1.4)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">7.67, s</td><td class="colsep0 rowsep0" align="left">7.25, d (1.6)</td><td class="colsep0 rowsep0" align="left">7.26,d (1.6)</td><td class="colsep0 rowsep0" align="left">7.13, d (1.4)</td><td class="colsep0 rowsep0" align="left">7.70, s</td><td class="colsep0 rowsep0" align="left">7.77, s</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">7.50, s</td><td class="colsep0 rowsep0" align="left">7.34, s</td><td class="colsep0 rowsep0" align="left">7.36, s</td><td class="colsep0 rowsep0" align="left">7.18, s</td><td class="colsep0 rowsep0" align="left">7.56, s</td><td class="colsep0 rowsep0" align="left">7.58, s</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">6.72, ddd (3.8, 2.4, 1.6)</td><td class="colsep0 rowsep0" align="left">6.72, ddd (3.7, 2.0, 1.4)</td><td class="colsep0 rowsep0" align="left">6.78, d (1.4)</td><td class="colsep0 rowsep0" align="left">6.66, dd (3.7, 1.5)</td><td class="colsep0 rowsep0" align="left">6.74, d (3.8)</td><td class="colsep0 rowsep0" align="left">6.80, d (1.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">6.18, dt (3.5, 2.4)</td><td class="colsep0 rowsep0" align="left">6.20, dt (3.5, 2.3)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">6.16, dd (3.7, 2.6)</td><td class="colsep0 rowsep0" align="left">6.27, d (3.8)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">6.94, td (2.6, 1.5)</td><td class="colsep0 rowsep0" align="left">6.97, td (2.6, 1.5)</td><td class="colsep0 rowsep0" align="left">7.11, d (1.4)</td><td class="colsep0 rowsep0" align="left">6.91, dd (2.6, 1.5)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">7.13, d (1.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">11.85, s</td><td class="colsep0 rowsep0" align="left">11.79, s</td><td class="colsep0 rowsep0" align="left">12.30, s</td><td class="colsep0 rowsep0" align="left">11.77, s</td><td class="colsep0 rowsep0" align="left">12.61, s</td><td class="colsep0 rowsep0" align="left">12.20, s</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">-OMe</td><td class="colsep0 rowsep0" align="left">3.91, s</td><td class="colsep0 rowsep0" align="left">3.95, s</td><td class="colsep0 rowsep0" align="left">3.95, s</td><td class="colsep0 rowsep0" align="left">3.75, s</td><td class="colsep0 rowsep0" align="left">3.95, s</td><td class="colsep0 rowsep0" align="left">3.95, s</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Couplings measured in MeOD for better resolution.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Summary of <sup>13</sup>C NMR (151 MHz, (CD<sub>3</sub>)<sub>2</sub>SO) Data for <b>3</b>–<b>8</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>3</b></th><th class="colsep0 rowsep0" align="center"><b>4</b></th><th class="colsep0 rowsep0" align="center"><b>5</b></th><th class="colsep0 rowsep0" align="center"><b>6</b></th><th class="colsep0 rowsep0" align="center"><b>7</b></th><th class="colsep0 rowsep0" align="center"><b>8</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">position</th><th class="colsep0 rowsep0" align="center">δ<sub>C</sub>, type</th><th class="colsep0 rowsep0" align="center">δ<sub>C</sub>, type</th><th class="colsep0 rowsep0" align="center">δ<sub>C</sub>, type</th><th class="colsep0 rowsep0" align="center">δ<sub>C</sub>, type</th><th class="colsep0 rowsep0" align="center">δ<sub>C</sub>, type</th><th class="colsep0 rowsep0" align="center">δ<sub>C</sub>, type</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">124.7, CH</td><td class="colsep0 rowsep0" align="left">123.6, CH</td><td class="colsep0 rowsep0" align="left">124.0, CH</td><td class="colsep0 rowsep0" align="left">84.4<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>, C</td><td class="colsep0 rowsep0" align="left">125.0, CH</td><td class="colsep0 rowsep0" align="left">125.0, CH</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">104.3, C</td><td class="colsep0 rowsep0" align="left">104.5, C</td><td class="colsep0 rowsep0" align="left">104.2, C</td><td class="colsep0 rowsep0" align="left">109.1<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>, C</td><td class="colsep0 rowsep0" align="left">104.1, C</td><td class="colsep0 rowsep0" align="left">104.1, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">123.6, C</td><td class="colsep0 rowsep0" align="left">113.1, C</td><td class="colsep0 rowsep0" align="left">113.2, C</td><td class="colsep0 rowsep0" align="left">116.2<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>, C</td><td class="colsep0 rowsep0" align="left">123.5, C</td><td class="colsep0 rowsep0" align="left">123.5, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">102.2, CH</td><td class="colsep0 rowsep0" align="left">154.0, C</td><td class="colsep0 rowsep0" align="left">154.1, C</td><td class="colsep0 rowsep0" align="left">152.6, C</td><td class="colsep0 rowsep0" align="left">102.2, CH</td><td class="colsep0 rowsep0" align="left">102.2, CH</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">150.3, C</td><td class="colsep0 rowsep0" align="left">104.3, CH</td><td class="colsep0 rowsep0" align="left">104.4, CH</td><td class="colsep0 rowsep0" align="left">107.0, CH</td><td class="colsep0 rowsep0" align="left">150.4, C</td><td class="colsep0 rowsep0" align="left">150.4, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">107.1, C</td><td class="colsep0 rowsep0" align="left">115.7, C</td><td class="colsep0 rowsep0" align="left">115.8, C</td><td class="colsep0 rowsep0" align="left">115.8, C</td><td class="colsep0 rowsep0" align="left">107.1, C</td><td class="colsep0 rowsep0" align="left">107.1, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">116.6, CH</td><td class="colsep0 rowsep0" align="left">108.4, CH</td><td class="colsep0 rowsep0" align="left">108.5, CH</td><td class="colsep0 rowsep0" align="left">104.1, CH</td><td class="colsep0 rowsep0" align="left">116.6, CH</td><td class="colsep0 rowsep0" align="left">116.6, CH</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">132.0, C</td><td class="colsep0 rowsep0" align="left">138.9, C</td><td class="colsep0 rowsep0" align="left">139.0, C</td><td class="colsep0 rowsep0" align="left">141.2<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>, C</td><td class="colsep0 rowsep0" align="left">132.0, C</td><td class="colsep0 rowsep0" align="left">132.0, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">145.9, C</td><td class="colsep0 rowsep0" align="left">146.1, C</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">144.6, C</td><td class="colsep0 rowsep0" align="left">146.2, C</td><td class="colsep0 rowsep0" align="left">146.2, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">120.5, CH</td><td class="colsep0 rowsep0" align="left">123.8, CH</td><td class="colsep0 rowsep0" align="left">123.1, CH</td><td class="colsep0 rowsep0" align="left">126.2, CH</td><td class="colsep0 rowsep0" align="left">120.5, CH</td><td class="colsep0 rowsep0" align="left">120.5, CH</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">154.4, C</td><td class="colsep0 rowsep0" align="left">154.6, C</td><td class="colsep0 rowsep0" align="left">153.3, C</td><td class="colsep0 rowsep0" align="left">155.8, C</td><td class="colsep0 rowsep0" align="left">153.2, C</td><td class="colsep0 rowsep0" align="left">153.2, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">120.3, C</td><td class="colsep0 rowsep0" align="left">120.5, C</td><td class="colsep0 rowsep0" align="left">121.4, C</td><td class="colsep0 rowsep0" align="left">120.3, C</td><td class="colsep0 rowsep0" align="left">122.2, C</td><td class="colsep0 rowsep0" align="left">121.3, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">109.8, CH</td><td class="colsep0 rowsep0" align="left">109.6, CH</td><td class="colsep0 rowsep0" align="left">111.0, CH</td><td class="colsep0 rowsep0" align="left">109.6, CH</td><td class="colsep0 rowsep0" align="left">111.2, CH</td><td class="colsep0 rowsep0" align="left">111.1, CH</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">109.7, CH</td><td class="colsep0 rowsep0" align="left">109.7, CH</td><td class="colsep0 rowsep0" align="left">96.3, C</td><td class="colsep0 rowsep0" align="left">109.5, CH</td><td class="colsep0 rowsep0" align="left">111.9, CH</td><td class="colsep0 rowsep0" align="left">96.4, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">122.0, CH</td><td class="colsep0 rowsep0" align="left">121.8, CH</td><td class="colsep0 rowsep0" align="left">121.6, CH</td><td class="colsep0 rowsep0" align="left">122.0, CH</td><td class="colsep0 rowsep0" align="left">102.4, C</td><td class="colsep0 rowsep0" align="left">121.6, C</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">-OMe</td><td class="colsep0 rowsep0" align="left">57.0, CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">56.1, CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">55.9, CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">55.7, CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">57.1, CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">57.1, CH<sub>3</sub></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Acquired at pH 3.5.</p></div></div></div><div class="NLM_p">Breitfussin D (<b>4</b>) was isolated as a pale brown oil. Its molecular formula was found to be C<sub>16</sub>H<sub>12</sub>BrN<sub>3</sub>O<sub>2</sub> (12 degrees of unsaturation, <i>m</i>/<i>z</i> 358.0197 [M + H]<sup>+</sup>, Δmmu = 0.6) on the basis of HRMS analysis. The <sup>1</sup>H NMR spectrum of <b>4</b> recorded in DMSO-<i>d</i><sub>6</sub> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) displayed two NH signals: H-18 at δ<sub>H</sub> = 11.79 and H-1 at δ<sub>H</sub> = 11.75, along with a signal for one methoxy group at δ<sub>H</sub> 3.95 and for seven aromatic protons with shift values ranging from δ<sub>H</sub> 7.44–6.20. The <sup>13</sup>C NMR spectrum of <b>4</b> confirmed the presence of 16 carbon atoms, including one methyl, seven methines, and eight quaternary carbon atoms, further supporting the breitfussin carbon framework (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">Comparison of the <sup>1</sup>H and <sup>13</sup>C NMR spectra as well as HMBC coupling patterns of <b>4</b> with those of <b>1</b>–<b>3</b> confirmed the breitfussin skeleton. The downfield shift value of C-5 (δ<sub>C</sub> = 154.0) in <b>4</b> indicated methoxylation in this position of <b>4</b>. Correlations between the methoxy protons (δ<sub>H</sub> = 3.95) and C-5 (δ<sub>C</sub> = 154.0) in the HMBC spectrum of <b>4</b> confirmed the position of the methoxy group. As for <b>3</b>, C-7 (δ<sub>C</sub> = 115.7) was substituted with the bromine atom, thus giving the full structure of <b>4</b> as 5-(6-bromo-4-methoxy-1<i>H</i>-indol-3-yl)-2-(1<i>H</i>-pyrrol-2-yl)oxazole. Breitfussin E (<b>5</b>) was isolated as a pale brown oil. HRMS analysis gave a molecular formula of C<sub>16</sub>H<sub>11</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>2</sub> (12 degrees of unsaturation, <i>m</i>/<i>z</i> 435.9303 [M+H]<sup>+</sup>, Δmmu = 0.7). The <sup>1</sup>H NMR analysis of <b>5</b> revealed two exchangeable NH protons: H-1 and H-18 (δ<sub>H</sub> = 12.00 and δ<sub>H</sub> = 12.30, respectively), six aromatic singlets with δ<sub>H</sub> values ranging from 7.76 – 6.73 in addition to a methoxy resonance at δ<sub>H</sub> 3.95 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Due to low amount of isolated material, it was not feasible to obtain a decent carbon spectrum for <b>5</b>. Through HMBC and HSQC analysis of <b>5</b>, we were able to confirm the presence of 11 out of 12 carbon atoms (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) (C-10 was not detectable) with shift values close to those obtained for <b>2</b>. The remaining of the molecular structure of <b>5</b> was put together in a similar manner as for <b>3</b> and <b>4</b>, leaving only C-10 unassigned. This leaves the breitfussin carbon skeleton as the only feasible structural option. The chemical shift of C-5 (δ<sub>C</sub> 154.1), along with an HMBC correlation between methoxy protons (δ<sub>H</sub> = 3.95) and C-5, indicated methoxylation at this position. The positions of the bromine atoms were assigned to C-7 (δ<sub>C</sub> = 115.8) and C-16 (δ<sub>C</sub> = 96.3).With this, the structure of <b>5</b>was determined to be 2-(4-bromo-1<i>H</i>-pyrrol-2-yl)-5-(6-bromo-4-methoxyl-1<i>H</i>-indol-3-yl)oxazole.</div><div class="NLM_p">Breitfussin F (<b>6</b>) was isolated as a light brown powder. HRMS analysis gave a molecular formula of C<sub>16</sub>H<sub>11</sub>BrIN<sub>3</sub>O<sub>2</sub> (12 degrees of unsaturation, <i>m</i>/<i>z</i> 483.9161 [M + H]<sup>+</sup>, Δmmu = 0.4). The <sup>1</sup>H NMR spectrum of <b>6</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) revealed the presence of a methoxy group (δ<sub>H</sub> = 3.75), four aromatic singlets (δ<sub>H</sub> = 7.18–6.16) and one NH proton H-18 (δ<sub>H</sub> = 11.77). The NH-1 (δ<sub>H</sub> = 12.25) hydrogen could initially not be detected through NMR analysis. Due to the low amount of isolated material, only 12 out of 16 carbon atoms in <b>6</b> gave detectable signals through <sup>13</sup>C NMR analysis. The chemical shift of C-4 (δ<sub>C</sub> = 116.0) was obtained through HMBC analysis, where it was found to correlate to H-6 (δ<sub>H</sub> = 6.67) and H-8 (δ<sub>H</sub> = 7.13). For C-2, C-3, and C-9, no cross peaks could be observed in the HMBC spectrum (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Through the 2D NMR experiments, partial structures of <b>6</b> could be unambiguously assembled: the pyrrole and the oxazole ring constituted one fragment and carbon atoms 4 to 8, including the methoxy group on the 5 position of the indole another fragment. This strongly indicates that <b>6</b> contains the characteristic breitfussin framework and leaves only one possibility for positioning of the iodine, i.e., on C-2 of the indole ring. An iodine at the C-2 position could explain the disappearance of the NH-1 proton as the iodine is expected to make the neighboring NH more acidic. Indeed, lowering the pH of the NMR sample to 3.5 shifted the equilibrium to protonating the nitrogen completely, and the NH-1 did emerge at δ<sub>H</sub> = 12.41 ppm. In this sample, the missing carbon shifts could be found at δ<sub>C</sub> (C-2) = 84.4, δ<sub>C</sub> (C-3) = 109.1, δ<sub>C</sub> (C-4) = 116.2, and δ<sub>C</sub> (C-9) = 141.2. The structure of <b>6</b> was therefore assigned as 5-(6-bromo-2-iodo-4-methoxy-1<i>H</i>-indo-3-yl)-2-(1<i>H</i>-pyrrol-2-yl)oxazole.</div><div class="NLM_p">Breitfussin G (<b>7</b>) and H (<b>8</b>) could not be chromatographically separated using semipreparative HPLC and were isolated together as a pale yellow liquid. UHPLC-HRMS analysis of <b>7</b> and <b>8</b> gave a molecular formula of C<sub>16</sub>H<sub>11</sub>Br<sub>2</sub>N<sub>3</sub>O<sub>2</sub> (12 degrees of unsaturation, <b>7</b>: <i>m</i>/<i>z</i> 435.9302 [M + H]<sup>+</sup>, Δmmu = 0.6; <b>8</b>: <i>m</i>/<i>z</i> 435.9303 [M + H]<sup>+</sup>, Δmmu = 0.7) for both compounds. Comprehensive analysis of 1D (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) and 2D NMR data, similar to the analysis performed for <b>6</b>, confirmed the breitfussin skeleton of <b>7</b> and <b>8</b> to be related to <b>3.</b> The methoxy group was in the 6 position, as in <b>3</b>, but an extra bromine was positioned on the pyrrole. In <b>7</b>, the bromine was placed on the C-17 based on H-15 and H-16 being vicinal as proved by a 3.8 Hz coupling and a strong ROE<sub>H-15/H-16</sub>. In <b>8</b>, the bromine was placed on the C-16, resulting in the absence of any ROE<sub>H-15/H-16</sub> and a weaker <sup>4</sup><i>J</i><sub>H-15/H-17</sub> of 1.7 Hz. The structure of <b>7</b> was therefore assigned as 2-(4-bromo-1<i>H</i>-pyrro-2-yl)-5-(6-bromo-5-methoxyl-1<i>H</i>-indol-3-yl)oxazole, and <b>8</b> was assigned as 2-(5-bromo-1<i>H</i>-pyrrol-2-yl)-5-(6-bromo-5-methoxyl-1<i>H</i>-indol-3-yl)oxazole.</div><div class="NLM_p last">The structures of reisolated <b>1</b> and <b>2</b> were confirmed by comparing the analytical data with those previously reported by us (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_001.pdf" class="ext-link">Tables S1 and S2</a>).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Total Synthesis of <b>3</b> and <b>4</b></h3><div class="NLM_p">In order to carry out an extensive evaluation of the biological activity of <b>3</b>, additional material was required. Previously we have reported the synthesis of <b>1</b> and <b>2</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and the same strategy was applied for the synthesis of <b>3</b>. This time, the total synthesis commenced by TIPS-protection of 5-methoxyindole (<b>9</b>) following a literature procedure forming indole <b>10</b> (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_001.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> Selective bromination of <b>10</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) by directed ortho metalation<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> proved to be difficult, and several approaches including temporary introduction of a TMS group on C-3<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> to induce steric hindrance,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> use of TBS instead of TIPS as a protection group, and the use of different lithiation reagents failed to give 6-bromoindole <b>11</b> selectively.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Total Synthesis of <b>3</b> from Indole <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) <i>sec</i>-BuLi, TMEDA, THF, 15 °C to −5 °C and (ii) Br(CH<sub>2</sub>)<sub>2</sub>Br, −5 °C to rt (67% yield for mixture of <b>11</b> and <b>12</b>); (b) ICl, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt (97% yield); (c) (i) (2-triisopropylsilyl)oxazol-5-yl)boronic acid pinacol ester, Pd(dppf)Cl<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, toluene/H<sub>2</sub>O, 80 °C and (ii) 10% aq HCl, THF, 0 °C (62% yield over 2 steps); (d) (i) LiHMDS, THF, −78 °C and (ii) I<sub>2</sub>, −78 °C (31% yield for <b>15</b>, 47% yield for mixture of <b>16</b> and <b>17</b>); (e) <i>N</i>-Boc-pyrrole-2-boronic acid, Pd(dppf)Cl<sub>2</sub>*CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O (66% yield); (f) (i) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt and (ii) TBAF, THF (68% yield over 2 steps).</p></p></figure><div class="NLM_p">Eventually we found that using <i>sec</i>-BuLi/TMEDA followed by a reaction with 1,2-bromoethane gave a 2:3 mixture of 6-bromoindole <b>11</b> and 4-bromoindole <b>12</b> in 67% combined yield. <b>11</b> could then be isolated in pure form in 14% yield by crystallization from hexane. 6-Bromoindole <b>11</b> could then be iodinated to give <b>13</b> in high yield, which was subsequently submitted to the optimal coupling conditions identified in our previous synthesis of <b>1</b> and <b>2</b>. Treating the cross-coupling product directly with aqueous HCl resulted in selective deprotection of the TIPS group in the 2-position on the oxazole giving oxazole <b>14</b> in good yield. In our earlier synthesis of <b>1</b> and <b>2</b>, we could only successfully couple onto a 2,4-diiodo oxazole derivative and not the 2-iodo oxazole derivative, thus 2,4-diiodo oxazole <b>15</b> was the target for the next step. The iodination of <b>14</b> however proved to be sluggish, and similarly as reported earlier for the iodination en route to <b>1</b> and <b>2</b>,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> we ended up with a mixture of the desired diiodo product <b>15</b> in 31% yield in addition to substantial amounts of monoiodinated oxazoles <b>16</b> and <b>17</b> (47% combined yield of <b>16</b> and <b>17</b>). Monoiodinated oxazoles <b>16</b> and <b>17</b> were not separable by flash chromatography and resubmitting a mixture of <b>16</b> and <b>17</b> to the reaction conditions gave <b>15</b> in 61% yield. This provided sufficient amounts of the desired material, and further Suzuki-Miyaura cross-coupling of <b>15</b> with Boc-protected 2-pyrrolyl boronic acid gave the complete indole-oxazole-pyrrole scaffold <b>18</b> in good yield. Finally, <b>18</b> was submitted to simultaneous acid-mediated Boc-deprotection and deiodination followed by TIPS deprotection to give <b>3</b> in 1.1% yield over 9 steps. Deiodination and deprotection of the earlier reported intermediate <b>19</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> gave natural product <b>4</b>, which has been prepared previously by another group using a different approach.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt and (ii) TBAF, THF (77% yield over 2 steps).</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> In Vitro Cytotoxic Activities of <b>1</b>–<b>6</b></h3><div class="NLM_p">The cytotoxic properties of <b>1</b>–<b>6</b> were evaluated against seven cancer cell lines (MCF-7, MDA-MB-468, and SK-BR-3: breast adenocarcinoma, HT-29: colon adenocarcinoma, MOLT-4 and MV-4–11: leukemia, and A2058: melanoma) and one nonmalignant cell line (MRC-5: lung fibroblasts) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) using the MTS assay.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Compounds <b>7</b> and <b>8</b> were not evaluated, as they were isolated as a mixture.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. IC<sub>50</sub> (μM) of <b>1</b>–<b>6</b> against Seven Malignant and One Non-Malignant Cell Line</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="8" align="center">IC<sub>50</sub> (μM) ± SD</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">comp</th><th class="colsep0 rowsep0" align="center">MCF-7<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">MDA-MB-468<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">SK-BR-3<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">HT-29<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">MOLT-4<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">MV-4–11<a class="ref internalNav" href="#t3fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" align="center">A2058<a class="ref internalNav" href="#t3fn6" aria-label="f">f</a></th><th class="colsep0 rowsep0" align="center">MRC-5<a class="ref internalNav" href="#t3fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">na<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">8.5 ± 0.2</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">7.1 ± 0.2</td><td class="colsep0 rowsep0" align="left">9.9 ± 0.4</td><td class="colsep0 rowsep0" align="left">4.4 ± 0.5</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">7.1 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">3.9 ± 0.3</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">na: not active (IC<sub>50</sub> > 10 μM).</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">MCF-7, MDA-MB-468, and SK-BR-3: breast adenocarcinoma.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">HT-29: colorectal adenocarcinoma.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">MOLT-4: acute lymphoblastic leukemia.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">MV-4–11: myelomonocytic leukemia.</p></div><div class="footnote" id="t3fn6"><sup><sup>f</sup></sup><p class="last">A2058: melanoma.</p></div><div class="footnote" id="t3fn7"><sup><sup>g</sup></sup><p class="last">MRC-5: nonmalignant lung fibroblasts, all of human origin.</p></div></div></div><div class="NLM_p">While <b>1</b>, <b>2</b>, <b>5</b>, and <b>6</b> showed no significant activity toward these cell lines, <b>3</b> and <b>4</b>, displayed cytotoxic activities. Compound <b>3</b> reduced the viability of MCF-7, HT-29, MOLT-4, MV-4–11, and MRC-5, and <b>4</b> affected the viability of MCF-7. Only IC<sub>50</sub> values under 10 μM were measurable, as the compound precipitated at around 20 μM in this assay setup, interfering with the readout of the experiments. Through microscopic evaluation of test wells it was however clear that the sensitivity of the cells toward the compounds varied significantly. For example, compound <b>3</b> had no observable effect toward A2058 viability at concentrations up to 100 μM.</div><div class="NLM_p last">To evaluate whether the activities of <b>3</b> and <b>4</b> were time-dependent, IC<sub>50</sub> values for the compounds were determined after 4, 24, 48, and 72 h of exposure against two cell lines (<b>3</b> against HT-29, <b>4</b> against MCF-7). Exposure of the affected cell lines to <b>3</b> and <b>4</b> gave decreasing IC<sub>50</sub> values with increasing exposure times (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">Figure S3</a>).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Breitfussins as Kinase Inhibitors</h3><div class="NLM_p">To identify potential cellular targets of the breitfussins, <b>1</b>, <b>3</b>, and <b>4</b> were tested against a panel of 140 kinases at the International Centre for Kinase Profiling (ICKP) in Dundee (Scotland, UK) using activity-based radioactive (<sup>33</sup>P-ATP) filter-binding assays.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Compound <b>1</b> was tested at 50 μM, <b>3</b> at 50, 5, and 1 μM, and <b>4</b> at 5 and 1 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">Table S3</a>). At 50 μM, <b>1</b> was the least active of the tested compounds. Three kinases had residual enzyme activities (RA) between 50 and 30% and two below 30% (AMPK: 28%, TSSK1: 21%). Compound <b>1</b> was therefore not tested at lower concentrations. At 50 μM, <b>3</b> gave RAs between 50 and 30% for 23 kinases, between 30 and 10% for 24 kinases, and lower than 10% for 7 kinases (RIPK2: 10%, ERK8: 9%, GCK: 7%, TAK1: 7%, Aurora B: 6%, MLK1: 5%, and PIM1: 2%) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). At 5 μM, the number of kinases inhibited by <b>3</b> decreased, and at 1 μM, <b>3</b> gave RAs between 50 and 30% for 5 kinases (NUAK1: 46%, RIPK2: 41%, TAK1: 35%, Aurora B: 33%, and GCK: 31%), between 30 and 10% for 2 kinases (MLK1: 26%, MLK 3:20%), and lower than 10% for 2 kinases (PIM1:6%, PIM3:7%) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). At 5 μM, <b>4</b> affected 3 kinases to give RAs between 50 and 30%, 5 kinases between 30 and 10%, and 2 kinases to RAs lower than 10% (MLK1: 7%, MLK3: 7%), while at 1 μM, no kinase had RAs below 10%, and one kinase had a RA below 30% (PIM3: 29%).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinome inhibition data of <b>3</b> at (A) 50 and (B) 1 μM against 140 kinases. Disc sizes and colors are encoding the level of residual activity as shown in the figure key.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Selectivity of Breitfussin Kinase Inhibition</h3><div class="NLM_p last">From the data obtained through the kinase inhibition experiments, it was possible to draw Lorenz curves and calculate the Gini coefficient (GC)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> for <b>3</b> at 50, 5, and 1 μM and for <b>4</b> at 5 μM. The GC of <b>3</b> rose from 0.432 at 50 μM to 0.484 at 5 μM, and 0.605 at 1 μM. The GC for <b>4</b> at 5 μM was 0.529. The GC for <b>4</b> at 1 μM could not be calculated, as too few kinases were affected. The GC score of 0.605 for <b>3</b> at 1 μM is high for a natural product whose scaffold has not been optimized through synthesis. It is worth mentioning that lapatinib, gefitinib, and sunitinib, which are all marketed kinase inhibitors used to treat various types of cancers, have GC scores of 0.57, 0.65, and 0.67, respectively, when assayed against 317 kinases at 10 μM.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> GC scores do not take into account selectivity for subsets of related kinases, and the scores cannot be directly compared to other compounds unless tested under the same conditions (same concentration, same panel of kinases, etc.). However, it provides an overall indication of specificity and indicates that <b>3</b> is a narrow-spectrum kinase inhibitor.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> IC<sub>50</sub> of <b>1</b>–<b>6</b> Against Relevant Kinases</h3><div class="NLM_p">On the basis of the kinase inhibition profiling of <b>3</b> and <b>4</b>, 13 kinases were selected for determination of IC<sub>50</sub> values against <b>2</b>–<b>6</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). As <b>1</b> did not significantly inhibit any of the kinases at 50 μM, it was not included in the IC<sub>50</sub> determination. PIM2 and Aurora B were included to highlight the selectivity of inhibition of the closely related kinases PIM1 and PIM3<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> and Aurora A,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> respectively. The difference in the kinase inhibition potency of <b>1</b>–<b>6</b> correlated well with the cancer cell cytotoxic data (MTS assay) recorded for the same compounds. Although we have not observed a functional correlation of the inhibition of a specific or single kinase with antiproliferative activity yet, breitfussin analogs inhibiting multiple kinases in vitro have higher potency in the cytotoxicity assays. This is beyond this initial characterization of the breitfussins as kinase inhibitors, and it is going to be the task of future optimization chemistry to generate highly selective analogs from the breitfussin scaffold, which are expected to show a clear functional correlation in vitro and in functional cellular assays. Compound <b>3</b> was the most potent inhibitor, giving IC<sub>50</sub> values as low as 200 nM ± 50 (PIM1). On the basis of the results from the in vitro cancer cell cytotoxic and kinase inhibition profiles of <b>1</b>–<b>6</b>, <b>3</b> was selected for further studies to evaluate the pharmacological potential of the breitfussin scaffold.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. IC<sub>50</sub> Determination of <b>2</b>–<b>6</b> against 13 Kinases Nominated through Kinase Profiling of <b>3</b> and <b>4</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="6" align="center">compound IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center"><b>1</b></th><th class="colsep0 rowsep0" align="center"><b>2</b></th><th class="colsep0 rowsep0" align="center" char="±"><b>3</b></th><th class="colsep0 rowsep0" align="center"><b>4</b></th><th class="colsep0 rowsep0" align="center"><b>5</b></th><th class="colsep0 rowsep0" align="center"><b>6</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DYRK1A</td><td class="colsep0 rowsep0" align="left">na<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">2.3 ± 0.2</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ERK8</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">1.0 ± 0.1</td><td class="colsep0 rowsep0" align="left">46 ± 7</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIM1</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.05</td><td class="colsep0 rowsep0" align="left">0.9 ± 0.01</td><td class="colsep0 rowsep0" align="left">43 ± 12</td><td class="colsep0 rowsep0" align="left">18 ± 9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIM2</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">11 ± 2</td><td class="colsep0 rowsep0" align="left">22 ± 3</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIM3</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.1</td><td class="colsep0 rowsep0" align="left">3.7 ± 0.8</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SIK3</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.1</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GCK</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.06</td><td class="colsep0 rowsep0" align="left">1.2 ± 0.1</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">40 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TAK1</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.1</td><td class="colsep0 rowsep0" align="left">1.0 ± 0.1</td><td class="colsep0 rowsep0" align="left">36 ± 15</td><td class="colsep0 rowsep0" align="left">18 ± 7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MLK1</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.12</td><td class="colsep0 rowsep0" align="left">0.9 ± 0.04</td><td class="colsep0 rowsep0" align="left">22 ± 2</td><td class="colsep0 rowsep0" align="left">21 ± 7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MLK3</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">0.6 ± 0.05</td><td class="colsep0 rowsep0" align="left">0.5 ± 0.05</td><td class="colsep0 rowsep0" align="left">23 ± 2</td><td class="colsep0 rowsep0" align="left">19 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RIPK2</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">1.1 ± 0.2</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Aurora A</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">22 ± 6</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Aurora B</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="char" char="±">0.8 ± 0.02</td><td class="colsep0 rowsep0" align="left">0.9 ± 0.23</td><td class="colsep0 rowsep0" align="left">7.8 ± 0.2</td><td class="colsep0 rowsep0" align="left">31 ± 3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Not active (IC50 > 50 μM)</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Extended Kinase Inhibition Profile for <b>3</b></h3><div class="NLM_p">The binding affinities of <b>3</b> were tested at 10 μM against 468 kinases and pseudo kinases found in the DiscoverX ScanMax panel (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">Table S4</a>). In these active-site directed competition binding assays, a kinase ligand is immobilized to a solid support. The amount of kinase bound to the immobilized ligand in the presence and absence (control) of <b>3</b> were quantified, and the affinity of <b>3</b> to the kinase active site was reported as percent of control (PoC).<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> At 10 μM, <b>3</b> gave PoC values lower than 35% for 30 kinases. The strongest affinity of <b>3</b> was observed against the three related Ca<sup>2+</sup>/calmodulin-dependent protein kinases MYLK4, DRAK2, and DRAK1, with PoC values of 0.25, 1.2, and 2.9%, respectively. PIM1, the kinase against which the lowest IC<sub>50</sub> value for <b>3</b> was measured in the ICKP activity assay (IC<sub>50</sub> = 0.2 μM), gave a PoC of 9.5%. Compound <b>3</b> had stronger affinity toward PIM1 than PIM2 and PIM3 (PoC: PIM3 = 21%, PIM2 = 54%). The relative affinities of the PIM kinases correlated well with the IC<sub>50</sub> values found in the activity assays (IC<sub>50</sub>: PIM3 = 1.8 μM, PIM2 = 11 μM).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinome binding affinity data of compound <b>3</b> at 10 μM against 468 kinases. The disc sizes encode the level of residual activity as shown in the figure key.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Furthermore, the <i>K</i><sub>d</sub> values of <b>3</b> against the 19 kinases with lowest PoC values were determined (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). When comparing the results obtained for <b>3</b> from the ICKP and the DiscoverX panels, two kinases were found to be among the most sensitive candidates in both assays: PIM1 and PIM3 (PIM1: IC<sub>50</sub> = 200 nM, <i>K</i><sub>d</sub> = 1.2 μM, and PIM3: IC<sub>50</sub> = 1.79 μM, <i>K</i><sub>d</sub> = 4.7 μM). All the 11 remaining kinases, for which IC<sub>50</sub> values were determined at ICKP, were tested at 10 μM at DiscoverX; however, <b>3</b> did not bind strong enough to these for <i>K</i><sub>d</sub> determination to be prioritized [PoC at 10 μM: DYRK1A: 75%; ERK8: 67%; SIK3 (QSK in DiscoverX panel): 94%; GCK (MAP4K2 in DiscoverX panel): 72%; TAK1: 70%; MLK1: 59%; MLK3: 56%; Pim2: 54%; RIPK2: 57%; Aurora A: 80%; and Aurora B: 50%]. The remaining enzymes against which the <i>K</i><sub>d</sub> values of <b>3</b> were determined in the DiscoverX panel were not available in the ICKP panel. Disagreement between binding- and activity-based assays are not uncommon and can be explained by the use of different constructs, expression systems, purification protocols, and assay conditions.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The affinity of <b>3</b> toward kinases and the selective nature of this interaction was however confirmed through the data obtained from DiscoverX.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. <i>K</i><sub>d</sub> Values of <b>3</b> against the DiscoverX Kinases with Strongest Binding Affinities</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>d</sub> (μM)</th><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>d</sub> (μM)</th><th class="colsep0 rowsep0" align="center">kinase</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>d</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DRAK2</td><td class="colsep0 rowsep0" align="char" char="±">0.39 ± 0.04</td><td class="colsep0 rowsep0" align="left">STK35</td><td class="colsep0 rowsep0" align="char" char="±">1.3 ± 0.1</td><td class="colsep0 rowsep0" align="left">DRAK1</td><td class="colsep0 rowsep0" align="char" char="±">2.8 ± 1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MYLK4</td><td class="colsep0 rowsep0" align="char" char="±">0.43 ± 0.07</td><td class="colsep0 rowsep0" align="left">IRAK3</td><td class="colsep0 rowsep0" align="char" char="±">1.7 ± 0.2</td><td class="colsep0 rowsep0" align="left">PIK4CB</td><td class="colsep0 rowsep0" align="char" char="±">2.9 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GAK</td><td class="colsep0 rowsep0" align="char" char="±">0.44 ± 0.02</td><td class="colsep0 rowsep0" align="left">TYK2<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.1</td><td class="colsep0 rowsep0" align="left">RIOK1</td><td class="colsep0 rowsep0" align="char" char="±">2.9 ± 1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JAK1<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">1.0 ± 0.1</td><td class="colsep0 rowsep0" align="left">DCAMKL3</td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.2</td><td class="colsep0 rowsep0" align="left">CSNK1E</td><td class="colsep0 rowsep0" align="char" char="±">3.3 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIKFYVE</td><td class="colsep0 rowsep0" align="char" char="±">1.0 ± 0.1</td><td class="colsep0 rowsep0" align="left">CIT</td><td class="colsep0 rowsep0" align="char" char="±">2.7 ± 0.1</td><td class="colsep0 rowsep0" align="left">PIM3</td><td class="colsep0 rowsep0" align="char" char="±">4.7 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BIKE</td><td class="colsep0 rowsep0" align="char" char="±">1.1 ± 0.1</td><td class="colsep0 rowsep0" align="left">STK36</td><td class="colsep0 rowsep0" align="char" char="±">2.7 ± 0.1</td><td class="colsep0 rowsep0" align="left">FLT3 (N841I mutant)</td><td class="colsep0 rowsep0" align="char" char="±">8.9 ± 4.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PIM1</td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">JH2domain-pseudokinase.</p></div></div></div><div class="NLM_p last">Several of the kinases inhibited by or binding to <b>3</b> are validated as anticancer targets. To name a few, the PIM kinases are known drivers in many cancers, including triple negative breast cancer.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Mutants of FLT3 kinase occur in approximately 30% of all acute myeloid leukemia cases,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and ERK8 and AuroraB are highly expressed in several human cancers, including lung cancer.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> The exact cellular target(s) responsible for the observed cytotoxic activities of <b>3</b> remain to be determined and may be due to pan-kinase inhibition.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Selective Cytotoxic Activity of <b>3</b></h3><div class="NLM_p">To get more information on the selectivity of <b>3</b> against a broader panel of cell lines, IC<sub>50</sub> values of <b>3</b> were determined against 88 cancer cell lines and proliferating peripheral blood mononuclear cells (PBMCs) in the OncoLead cell panel using a sulforhodamine B protein staining assay technique (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">Table S6</a>).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The cell lines included in this panel covers multiple cancer types (overview in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>), and the IC<sub>50</sub> values against the 10 most affected cell lines are listed in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. The results from this screening confirmed the potent and selective cytotoxic activity of <b>3</b> with activities at low μM/sub-μM IC<sub>50</sub> concentrations (lowest IC<sub>50</sub> = 0.34 μM). In comparison, the lowest reported IC<sub>50</sub> value of the kinase inhibitor roscovitine, which is currently under clinical development as an anticancer agent, was 4.9 μM in a comparable assay setup.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In total, 48 out of the 88 cell lines were affected to some extent by exposure to <b>3</b>. The most striking results from these experiments are the ability to specifically inhibit the viability of selected breast cancer cell lines. Most potently the inhibited cell line was the drug-resistant triple negative breast cancer (TNBC) cell line MDA-MB-468 (IC<sub>50</sub> = 340 nM). In our in-house MTS-based cancer cell cytotoxic assay, the MDA-MB-468 cell line was significantly less affected (IC<sub>50</sub> value >10 μM). While the MTS assay is considered a precise and specific in vitro assay for measuring cytotoxic activity of investigated agents, it has been shown that colorimetric assays may underestimate antiproliferative effects.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> For example, damaged mitochondria may still be able to reduce the dye and investigated agents may interfere with mitochondrial dehydrogenase activity.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> This discrepancy is thus most likely due to the difference in assay technique. The following breast cancer cell lines were also affected: MCF-7 (IC<sub>50</sub> = 0.77 μM), MT3 (IC<sub>50</sub> = 0.93 μM), SK-BR-3 (IC<sub>50</sub> = 1.53 μM), JIMT1 (IC<sub>50</sub> = 4.17), and the TNBC cell line HS578T (IC<sub>50</sub> = 5.22 μM).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. IC<sub>50</sub> values for <b>3</b> against the 88 cell lines found in the OncoLead panel (Z-scores in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">Table S8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. 10 Cancer Cell Lines in the OncoLead Panel That Were Most Affected by <b>3</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center">origin</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">Z-score</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB-468</td><td class="colsep0 rowsep0" align="left">breast</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td><td class="colsep0 rowsep0" align="char" char=".">–3.87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCF-7</td><td class="colsep0 rowsep0" align="left">breast</td><td class="colsep0 rowsep0" align="char" char=".">0.77</td><td class="colsep0 rowsep0" align="char" char=".">–2.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MT3</td><td class="colsep0 rowsep0" align="left">breast</td><td class="colsep0 rowsep0" align="char" char=".">0.93</td><td class="colsep0 rowsep0" align="char" char=".">–2.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PLCPRF5</td><td class="colsep0 rowsep0" align="left">liver</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">–1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SK-BR-3</td><td class="colsep0 rowsep0" align="left">breast</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NCIH460</td><td class="colsep0 rowsep0" align="left">lung</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">–1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">A673</td><td class="colsep0 rowsep0" align="left">muscle</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">–1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JAR</td><td class="colsep0 rowsep0" align="left">placenta</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">–0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RDES</td><td class="colsep0 rowsep0" align="left">bone</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">–0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HEPG2</td><td class="colsep0 rowsep0" align="left">liver</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">–0.6</td></tr></tbody></table></div></div><div class="NLM_p">Compound <b>3</b> had no or significantly lower effect against the remaining breast cancer cell lines: BT20, MDA-MB-231, and MDA-MB-436 (IC<sub>50</sub> values > 30 μM). The nonmalignant human PBMCs and 39 cancer cell lines were not affected even at the highest dose of 30 μM of <b>3</b>.</div><div class="NLM_p">Such a differential inhibition profile in a broad cancer cell panel is typically observed when one or a few molecular mechanisms of inhibition are present in the sensitive cell lines. Therefore, it is our assumption that <b>3</b> inhibits one or a few proliferation relevant kinases, which are important drivers of cancer formation.</div><div class="NLM_p last">Natural product often serve as lead templates that are subjected to structural optimization to generate clinically useful structures.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Selective and potent activity against cancer cells (IC<sub>50</sub> values <1 μM) and inhibition of kinases in the nanomolar range show the value of the breitfussin scaffold as an excellent starting point for the development of an anticancer compound. In comparison, rohitukine, a flavonoid alkaloid initially isolated <i>Amoora rohituka</i>, gave IC<sub>50</sub> values of 300 ± 15 nM against the kinase CDK-9 and GI<sub>50</sub> values of 10–28 μM against five cancer cell lines.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Medicinal chemistry efforts based on the rohitukine scaffold rewarded two new promising clinical candidates, flavopiridol and riviciclib.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Both are currently undergoing clinical evaluation as anticancer agents targeting several kinases where the primary mechanism of action is believed to be through CDK-9 inhibition.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Riviciclib has an IC<sub>50</sub> value of 20 nM against CDK-9.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Mode-of-Action of the Breitfussins against the Kinases</h3><div class="NLM_p">Many natural product kinase inhibitors or derivatives thereof, like roscovitine, as well as synthetic small molecule kinase inhibitors inhibit their targets by competition with ATP.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> The ICKP and DiscoverX assay technologies are biased toward ATP competitiveness as mode-of-action. We therefore tested the ability of <b>1</b>–<b>4</b> to bind to the ATP binding pocket employing two different tracer displacement assays. As the assays herein used to measure ATP competitiveness are based on binding affinity, DRAK2 and DRAK1, two kinases with closely related structures,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> were chosen as model kinases as they display strong binding affinity toward <b>3</b> (<i>K</i><sub>d</sub> (μM): DRAK2 = 0.39 ± 0.04, DRAK1 = 2.8 ± 1.3).</div><div class="NLM_p">The cell free ATP-competitive tracer displacement assays<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> are based on the binding and displacement of the Alexa Fluor 647-labeled ATP competitive kinase tracer to glutathione S-transferase (GST)-tagged DRAK1 and DRAK2. Tracer binding was detected by using europium (Eu)-labeled anti-GST antibodies. Simultaneous binding of both fluorescent tracer and antibody to the GST-tagged kinase generated a fluorescence resonance energy transfer (FRET) signal. When adding an inhibitor, which competes with binding of the tracer, a loss of the FRET signal was observed. Using this assay, <b>3</b> and <b>4</b> were shown to bind to the ATP binding pocket (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). To evaluate the binding of <b>1</b>–<b>4</b> to the ATP binding site within living permeabilized cells (cells permeabilized using digitonin), the NanoBRET kinase target engagement assay was used.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> This target engagement assay is based on an energy transfer technique designed to measure molecular proximity in living cells and cell lysates. This intracellular kinase assay measures the apparent affinity of test compounds by competitive displacement of a tracer molecule reversibly bound to the ATP binding site of a kinase fused with the NanoLuc luciferase. In the first step of this assay, a fixed concentration of tracer was added to cells expressing the desired NanoLuc fusion protein to generate a BRET reporter complex. Introduction of competing compounds results in a dose-dependent decrease in NanoBRET energy transfer allowing for quantitation of the intracellular affinity of the target protein for the test compound. In this assay, <b>2</b>–<b>4</b> were shown to bind to the ATP binding pocket of DRAK1 and DRAK2 (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>).</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. ATP Competitiveness of Compounds <b>1</b>–<b>4</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">DRAK1</th><th class="colsep0 rowsep0" align="center" char=".">DRAK2</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" colspan="2" align="center" char=".">cell free tracer displacement assay</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">9.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">9.6</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd</th><th class="colsep0 rowsep0" colspan="2" align="center" char=".">intracellular tracer displacement assay</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td></tr></tbody></table></div></div><div class="NLM_p last">In both assay formats, selected breitfussins were shown to bind to the ATP binding site of the selected kinases by displacing the tracers from the ATP binding sites of DRAK1 and DRAK2.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Docking of <b>3</b> to the ATP-Binding Pocket of PIM1</h3><div class="NLM_p">To complete the scenario of identifying the ATP binding pocket as the site of interaction between <b>3</b> and the kinases, <b>3</b> was docked manually and computationally using Glide into a cocrystal structure of PIM1 (Protein Data Bank access code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B">4K1B</a>) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). Several binding modes were obtained, submitted to a short molecular dynamics run, and minimized. The pyrrole and indole N–H groups are complementary to the two C═O groups available as hydrogen bond acceptors in the hinge region of PIM1. The pyrrole and indole N–H groups form hydrogen bonds to the hinge backbone C═O groups of Pro123 and Glu121. The indole is stacked between lipophilic residues and does not interfere with the gate keeper residue (Leu120). Through this experiment, we were able to determine binding to the ATP-binding pocket as mode-of-action for <b>3</b>.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding mode of <b>3</b> with the ATP-binding pocket of PIM1 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B">4K1B</a>). Compound <b>3</b> is shown with yellow carbon atoms, the hinge backbone carbon atoms of PIM1 are in green (Pro123, Arg122, Glu121, and Leu120). A water molecule present in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B">4K1B</a> is replaced by the pyrrole and is shown as a red sphere. (A) Surface representation of the ATP-binding pocket of PIM1 with compound <b>3</b>. (B) Detailed view of the interaction between the PIM1 hinge residues and <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Potential Off-Target Effects of <b>3</b></h3><div class="NLM_p">To evaluate potential off-targets interactions for <b>3</b>, a diversity panel screen at 10 μM was performed using the Cerep Panlabs diversity panel (previous version now known as the Eurofins discovery safetyscreen panel). The panel was designed to assess potential adverse effects of a lead compound. In total, only 16 out of 98 targets were affected to the point where percentage inhibition of control values (phosphodiesterase type 4D2 and phosphodiesterase type 5, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) and specific binding (rest) were defined as active (above 50%). The remaining 82 targets were below 50% (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">Table S5A,B</a>). On the list of affected targets, the majority were receptors. In addition, a transporter (norepinephrine transporter) and two enzymes (phosphodiesterase type 4D2 and 5) were also affected. As most of the targets will not be affected at concentrations corresponding to the IC<sub>50</sub> or <i>K</i><sub>d</sub> values for the on-target effects, <b>3</b> is considered as having promising selectivity.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Potential off-targets (>50%) of <b>3</b> tested at 10 μM in the Cerep Panlabs diversity profile panel. * = human, # = agonist, $ = antagonist, and § = enzymatic assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> In Vivo Toxicity of <b>3</b></h3><div class="NLM_p">The toxicity of <b>3</b> against zebrafish embryos was evaluated by recording survival and toxic phenotype formations, including cardiac edema, scoliosis, tail degeneration, necrosis, and generalized edema. The assay was conducted at the BioReperia laboratory. Zebrafish embryos are considered a powerful tool to assess toxicity as it allows for whole-animal investigation.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> At 72 h exposure, the concentration resulting in 50% lethality (LD<sub>50</sub>) was 1.4 μM. This value is 4-fold higher than the most potently inhibited cell line in the OncoLead panel. Toxic phenotypes were only evaluated on living embryos, and the results are given as the percent of embryos not exhibiting the toxic phenotype (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>).</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Assessment of toxicity of <b>3</b> against zebrafish embryos.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Embryos Being Healthy and Not Exhibiting the Indicated Issues of Toxicity or Death, Expressed As % of All Living Embryos after Being Exposed to <b>3</b> for 72 h</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="6" align="center">unaffected embryos (%)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">conc (μM)</th><th class="colsep0 rowsep0" align="center">1396</th><th class="colsep0 rowsep0" align="center">140</th><th class="colsep0 rowsep0" align="center">14</th><th class="colsep0 rowsep0" align="center">1.4</th><th class="colsep0 rowsep0" align="center">0.1</th><th class="colsep0 rowsep0" align="center">0</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">toxicological issue</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">survival</td><td class="colsep0 rowsep0" align="left">0,0</td><td class="colsep0 rowsep0" align="left">0,0</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">heart edema</td><td class="colsep0 rowsep0" align="left">NA<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">scoliosis</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">tail degeneration</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">necrosis</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">general edema</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">NA: not available.</p></div></div></div><div class="NLM_p last">In general, heart edema and generalized edema were not observed to any significant degree in response to any nonlethal concentration of <b>3</b>. Scoliosis and tail degeneration were observed in all embryos exposed to the highest nonlethal concentration of <b>3</b> (1.4 μM). Compound <b>3</b> was poorly soluble in the assay media (PTU-E3 water). Precipitated <b>3</b> was evident for the highest test concentrations [1396 (data not shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>), 140 and 14 μM].</div></div><div id="sec2_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Determination of in Vitro ADME Parameters of <b>3</b></h3><div class="NLM_p">Compound <b>3</b> was profiled in an in vitro ADME assay panel to address its aqueous kinetic solubility, permeability through artificial membranes (PAMPA), metabolic stability under oxidative and conjugative conditions, plasma stability, and degree of plasma protein binding (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>).</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. In Vitro ADME Parameters of <b>3</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">assay</th><th class="colsep0 rowsep0" align="center">output</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human MS<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a> phase I Cl<sub>int</sub></td><td class="colsep0 rowsep0" align="left">14 μL/min/mg</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse MS phase I Cl<sub>int</sub></td><td class="colsep0 rowsep0" align="left">47 μL/min/mg</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human MS phase II</td><td class="colsep0 rowsep0" align="left">92 % remaining after 1 h at 5 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse MS phase II</td><td class="colsep0 rowsep0" align="left">100 % remaining after 1 h at 5 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">membrane permeability</td><td class="colsep0 rowsep0" align="left">23 flux (%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human plasma protein binding</td><td class="colsep0 rowsep0" align="left">0.3 % unbound at 5 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse plasma protein binding</td><td class="colsep0 rowsep0" align="left">0.3 % unbound at 5 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human plasma stability</td><td class="colsep0 rowsep0" align="left">83 % remaining after 1 h at 5 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mouse plasma stability</td><td class="colsep0 rowsep0" align="left">84 % remaining after 1 h at 5 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solubility at pH 7.4</td><td class="colsep0 rowsep0" align="left">156 μM</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">MS: Microsomal stability.</p></div></div></div><div class="NLM_p last">In all these assays, <b>3</b> was found to have favorable properties as a lead compound. Particularly, the moderate intrinsic clearance (Cl<sub>int</sub>) value of 47 μL/min/mg obtained in the presence of mouse liver microsomes and the acceptable stability of 85% in mouse plasma predicts good exposure in mice. Mice will thus be a suited species for proof-of-concept studies in the future. In this regard, it will be interesting to determine the pharmacokinetic parameters in mice as a next step to see how well the in vitro profile correlates with the in vivo ADME properties.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81282" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81282" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The results from this investigation into the cytotoxic activities, kinase inhibition, and ADME and toxicological properties of the breitfussins (<b>3</b> in particular) clearly show the potential of the breitfussin scaffold for further development of lead compounds primarily with antiproliferative activity. Even as nonmodified natural products, the breitfussins show preferential binding to kinases and inhibition of cell viability at promising potency and selectivity. We consider the breitfussin to be an interesting scaffold of narrow-spectrum kinase inhibitors. Further exploration of the series might quickly lead to the generation of more selective inhibitors of the indicated kinase subfamilies.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41654" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41654" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Materials and Methods for Natural Product Isolation and Structure Elucidation</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> General Experimental Procedures</h4><div class="NLM_p last">A Waters 600 Pump monitored by a 2996 Photodiode Array UV detector and a 3100 single quadrupole MS were used for mass guided preparative HPLC over a Waters XSelect CSHTM Prep Fluoro-Phenyl column (5 μm 10*250 mm). A Waters LCT Premier time-of-flight mass spectrometer with a Waters Acquity UPLC was used for acquisition of high-resolution mass data. The purity of key compounds (isolated <b>3</b> and <b>4</b> and synthetically produced <b>3</b>) were measured using UHPLC-UV analysis and calculated using the UV purity tool in the Waters UNIFY Scientific Information System software. The purity was determined to be ≥95% for all key compounds.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Reagents and Materials</h4><div class="NLM_p last">HPLC grade acetonitrile (ACN) and methanol (MeOH) (Chromasolve, for preparative HPLC), LC–MS grade ACN (Chromasolve, for LC–MS analysis), and reagent-grade formic acid (≥98%) were purchased from Sigma-Aldrich (Steinheim, Germany). Ultrapure water was prepared using a Millipore Milli-Q purification system (Bedford, MA, USA). Deuterated DMSO and methanol were purchased from Sigma-Aldrich (Steinheim, Germany). Roswell Park Memorial Institute (RPMI, Merck KGaA), Earle’s minimal essential medium (E-MEM, Merck, Germany), Dulbecco’s modified Eagle’s medium (D-MEM, Thermo Fisher Scientific), McCoy’s 5A (Biowest, France), Leibovitz L15 Medium (L15, Biowest, France), and Iscove Basal Medium with stable glutamine (Merck, Germany) cell media were used. D-MEM and RPMI were supplied with 10% fetal bovine serum (FBS, Merck) and 5 mL <span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-glutamine (200 mM, Merck, Germany). E-MEM was supplied with 10% FBS, 5 mL <span class="smallcaps smallerCapital">l</span>-alanyl-<span class="smallcaps smallerCapital">l</span>-glutamine, 5 mL nonessential amino acids (100x, Merck, Germany), 5 mL sodium pyruvate (100 mM, Merck, Germany), and 5 mL sodium bicarbonate (7.5%, Merck, Germany). McCoy’s 5A and iscove basal medium were supplied with 10% FBS. L15 was supplied with 10% FBS and 5 mL sodium bicarbonate. All media were supplied with gentamycin (10 μg/mL, Merck, Germany).</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Biomass Collection</h4><div class="NLM_p last">Specimens of the cnidarian <i>Thuiaria breitfussi</i> (class Hydrozoa, order Leptothecata, family Sertulariidae) were collected with a triangular bottom scrape in September 2011 at Bjørnøya, Svalbard (79.0293 N, 20.8574 E, at 48 m depth). The organism was identified by Robert A. Johansen, the Norwegian national biobank (Marbank), and a voucher specimen (reference number: M13031) was deposited at Marbank, Tromsø, Norway.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Isolation of Breitfussin A–H (<b>1</b>–<b>8</b>)</h4><div class="NLM_p last">The frozen specimens of <i>T. breitfussi</i> (475.6 g) were freeze-dried and minced. The resulting mass (96.6 g) was extracted twice with water (24 h and 30 min, respectively) at 5 °C in the dark. After centrifugation, the remaining sediment was freeze-dried, minced, and extracted twice with a 1:1 (vol:vol) mixture of dichloromethane and MeOH and filtrated. The filtrate was dried, resulting in 6.0 g organic extract. An aliquot of the organic extract (4.3 g) was partitioned between 150 mL hexane and 150 mL 90% MeOH three times to remove very lipophilic compounds. The pooled MeOH fractions were dried under reduced pressure and redissolved in 13 mL MeOH. Aliquots of this solution were injected onto a fluoro-phenyl column (gradient of 45 to 65% of ACN (aq) over 20 min, 6 mL/min). In the end, eight breitfussins were isolated in amounts that enabled structure determination by NMR. Out of these, six were pure compounds: <b>1</b> (4.2 mg), <b>2</b> (3.1 mg), <b>3</b> (7.3 mg), <b>4</b> (2.2 mg), <b>5</b> (2.0 mg), and <b>6</b> (1.9 mg). We were not able to separate <b>7</b> and <b>8</b>, and a mixture of the two (1.33 mg in total) was submitted to NMR analysis.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> NMR Analysis of Isolated <b>3</b>–<b>8</b></h4><div class="NLM_p last">All NMR data were acquired on a Varian/Agilent Inova spectrometer operating at 599.934 and 150.863 MHz for <sup>1</sup>H and <sup>13</sup>C, respectively, using a cryogenically cooled inverse detection HCN probe with enhanced proton channel (second generation). All spectra were acquired at 298 K in DMSO-<i>d</i><sub>6</sub>. Data acquired in DMSO-<i>d</i><sub>6</sub> was referred to the residual solvent peak of DMSO-<i>d</i><sub>6</sub>, <sup>1</sup>H = 2.504 and <sup>13</sup>C = 39.98 ppm. Typically, 16–32k data points were acquired for the 1D 1H experiments using 90° reading pulses, 5 s relaxation delay, and 8–32 transients. For the 2D phase insensitive gradient selected COSY experiments, 1440 × 200 complex data points were acquired using 4–8 transients. Data matrices of 1440 × 200 and 1440 × 256 complex points were acquired for the gradient selected edited HSQC and absolute value HMBC, respectively, using 8 to 32 transients. ROESY spectra were acquired as 1440 × 256 complex data points and 8 transients, with spinlock duration of 300 ms at 36 dB. For carbon 1D spectra, 32k complex data points were acquired, using 45–90° reading pulses, 1–5 s relaxation delay, and 10000–20000 transients applying 1H decoupling during both relaxation and acquisition time. All spectra were acquired at 298 K in both DMSO-<i>d</i><sub>6</sub> (Sigma-Aldrich) and methanol-<i>d</i><sub>4</sub> (Sigma-Aldrich) using sweep widths of 9600 and 30200–37800 Hz for <sup>1</sup>H and <sup>13</sup>C, respectively. Two-dimensional experiments were acquired using the grad-90-grad option with 2 ms homospoil gradient time at 5796 Hz. Data acquired in DMSO-<i>d</i><sub>6</sub> was referenced to the residual solvent peak of DMSO-<i>d</i><sub>6</sub>, <sup>1</sup>H = 2.504 and <sup>13</sup>C = 39.98 ppm.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Synthesis of <b>3</b> and <b>4</b></h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> General</h4><div class="NLM_p last">All reactants, reagents, and solvents were purchased from Sigma-Aldrich and used as delivered unless otherwise stated. Compound <b>19</b> was prepared as previously described.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Moisture sensitive reactions were carried out under an argon atmosphere in oven-dried (130 °C) equipment that had been cooled down under vacuum. Solvents used for moisture sensitive reactions were obtained from an anhydrous solvent delivery system (SPS-800 system from M. Braun GmbH, Garching, Germany). Anhydrous THF was obtained from the SPS-800 or from a sodium/benzophenone still. Flash chromatography was performed using silica gel from Merck (Silica gel 60, 0.040–0.063 mm) or Grace GmbH (35–70 μm). TLC analyses were performed on aluminum sheets coated with Merck TLC silica gel 60 F254, and visualization was achieved by using ultraviolet light (254 nm) or a solution of phosphormolybdic acid in ethanol. The NMR-experiments were recorded on a Bruker Ascend 400 or a Bruker BioSpin AV500 at ambient temperature. <sup>1</sup>H and <sup>13</sup>C chemical shifts (δ) are reported in ppm with reference to the solvent residue peak (CDCl<sub>3</sub>: δ<sub>H</sub> = 7.26 and δ<sub>C</sub> = 77.16; DMSO-<i>d</i><sub>6</sub>: δ<sub>H</sub> = 2.50 and δ<sub>C</sub> = 39.98). All coupling constants are given in Hz. ESI-HRMS was conducted on a Thermo electron LTQ Orbitrap XL spectrometer (Thermos, Bremen, Germany) with methanol as solvent.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 5-(6-Bromo-5-methoxy-1<i>H</i>-indol-3-yl)-2-(1<i>H</i>-pyrrol-2-yl)oxazole (<b>3</b>, Breifussin C)</h4><div class="NLM_p last">A solution of <b>18</b> (70 mg, 0.095 mmol) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (0.7 mL) was cooled to 0 °C and treated with TFA (0.7 mL). The resulting mixture was stirred at rt overnight before it was concentrated under reduced pressure and separated between water (5 mL) and EtOAc (5 mL). The organic layer was washed with saturated NaCl (3 mL), dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The resulting material was dissolved in anhydrous THF (1.0 mL) and cooled to 0 °C before it was treated with TBAF (104 μL, 1 M in THF, 104 μmol). The resulting mixture was stirred for 20 min at 0 °C before saturated NH<sub>4</sub>Cl (5 mL) was added. The resulting mixture was extracted with EtOAc (2 × 5 mL), and the combined organic layers were washed with saturated NaCl (5 mL) and dried over MgSO<sub>4</sub>. After removal of the solvent under reduced pressure, the crude product was purified by flash chromatography (hexane/EtOAc, 7:3 v/v to 2:3 v/v) to give the title compound <b>3</b> (23 mg, 68%) as a colorless solid. <i>R</i><sub>f</sub> (hexanes/EtOAc, 1:1 v/v) = 0.25; <sup>1</sup>H NMR 500 MHz, (DMSO-<i>d</i><sub>6</sub>): δ 11.84 (s, 1H), 11.52 (d, <i>J</i> = 2.7 Hz, 1H), 7.82 (d, <i>J</i> = 2.5 Hz, 1H), 7.70 (s, 1H), 7.54 (s, 1H), 7.43 (s, 1H), 6.98 (q, <i>J</i> = 2.2 Hz, 1H), 6.76 (q, <i>J</i> = 1.9 Hz, 1H), 6.22 (dd, <i>J</i> = 3.8, 1.9 Hz, 1H), 3.96 (s, 3H); <sup>13</sup>C NMR 126 MHz, (DMSO-<i>d</i><sub>6</sub>): δ 154.6, 150.5, 146.0, 132.0, 124.8, 123.7, 122.1, 120.7, 120.5, 116.6, 109.8 (2C), 107.2, 104.4, 102.3, 57.1.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 5-(6-Bromo-4-methoxy-1<i>H</i>-indol-3-yl)-2-(1<i>H</i>-pyrrol-2-yl)oxazole (<b>4</b>, Breitfussin D)</h4><div class="NLM_p last">A solution of <b>19</b> (51 mg, 69 μmol) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (0.5 mL) was cooled to 0 °C and treated with TFA (0.5 mL). The resulting mixture was stirred at rt overnight before it was concentrated under reduced pressure and separated between water (3 mL) and EtOAc (3 mL). The organic layer was washed with saturated NaCl (3 mL), dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The resulting material was dissolved in anhydrous THF (0.7 mL) and cooled to 0 °C before it was treated with TBAF (72 μL, 1 M in THF, 72 μmol). The resulting mixture was stirred for 20 min at 0 °C before saturated NH<sub>4</sub>Cl (4 mL) was added. The resulting mixture was extracted with EtOAc (2 × 4 mL), and the combined organic layers were washed with saturated NaCl (3 mL) and dried over MgSO<sub>4</sub>. After removal of the solvent under reduced pressure, the crude product was purified by flash chromatography (hexane/EtOAc, 7:3 v/v to 2:3 v/v) to give the title compound <b>4</b> (19 mg, 77%) as a colorless solid. <i>R</i><sub>f</sub> (hexanes/EtOAc, 1:1 v/v) = 0.25; <sup>1</sup>H NMR (400 MHz, (DMSO-<i>d</i><sub>6</sub>) δ 11.80 (s, 1H), 11.71 (s, 1H), 7.74 (s, 1H), 7.35 (d, <i>J</i> = 2.8 Hz, 1H), 7.25 (d, <i>J</i> = 2.4 Hz, 1H), 6.97 (s, 1H), 6.73 (br. s, 2H), 6.23–6.18 (m, 1H), 3.95 (s, 3H); <sup>13</sup>C NMR (101 MHz, (DMSO-<i>d</i><sub>6</sub>): δ 154.7, 154.1, 146.2, 139.0, 123.8, 123.2, 122.0, 120.5, 115.8, 113.2, 109.8 (2C), 108.5, 104.6, 104.4, 56.0; HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>13</sub>BrIN<sub>3</sub>O<sub>2</sub><sup>+</sup>, 358.0186 and 360.0165; found, 358.0186 and 360.0164.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 5-Methoxy-1-(triisopropylsilyl)-1<i>H</i>-indole (<b>10</b>)</h4><div class="NLM_p last">A solution of 5-methoxyindole (1.997 g, 13.57 mmol) in dry THF (16 mL) was added to a suspension of sodium hydride (843 mg, 60% w/w in mineral oil, 21.1 mmol) in THF (14 mL) at 0 °C, and the resulting mixture was stirred at 0 °C for 1 h before TIPSCl (1.6 mL, 72.5 mmol) was added. The mixture was warmed to room temperature and stirred overnight (22 h) before saturated NH<sub>4</sub>Cl (60 mL) was added. The resulting mixture was extracted with Et<sub>2</sub>O (3 × 70 mL), and the combined organic layers were dried over MgSO<sub>4</sub>. After filtration and removal of the solvent in vacuo, the crude product was purified using flash chromatography (petroleum ether/toluene, 6:1 v/v) to give the title compound <b>10</b> (3.818 g, 93%) as a transparent oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.40 (dd, <i>J</i> = 9.0, 0.8 Hz, 1H), 7.23 (d, <i>J</i> = 3.2 Hz, 1H), 7.10 (d, <i>J</i> = 2.6 Hz, 1H), 6.81 (dd, <i>J</i> = 9.0, 2.6 Hz, 1H), 6.56 (dd, <i>J</i> = 3.1, 0.8 Hz, 1H), 3.86 (s, 3H), 1.67 (sept, <i>J</i> = 7.5 Hz, 3H), 1.14 (d, <i>J</i> = 7.5 Hz, 18H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 154.1, 135.9, 132.1, 132.0 (2C), 114.6, 111.4, 104.6, 102.3, 55.8, 18.3, 12.9. The analytical data were in accordance with those reported in the literature.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 6-Bromo-5-methoxy-1-(triisopropylsilyl)-1<i>H</i>-indole (<b>11</b>)</h4><div class="NLM_p last">To a solution of <b>10</b> (2.811 g, 9.26 mmol) and TMEDA (1.67 mL, 11.14 mmol) in dry THF (45 mL) was added <i>sec</i>-butyl lithium (9.10, 11.10 mmol) dropwise over 20 min at −15 to −5 °C. The mixture was stirred for an additional 20 min at −5 °C before 1,2-dibromoethane (1.0 mL, 11.60 mmol) was added. The resulting mixture was warmed to room temperature and stirred for 45 min before saturated NH<sub>4</sub>Cl (70 mL) was added, and the resulting mixture was extracted with Et<sub>2</sub>O (3 × 100 mL). The combined organic layers were dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified using flash chromatography (petroleum ether/toluene, 6:1 v/v) to give a 2:3 mixture (2.363 g, 67%) of <b>11</b> and <b>12</b>, from which the title compound <b>12</b> could be isolated as a colorless solid (496 mg, 14%) by crystallization from hexane. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.66 (d, <i>J</i> = 0.9 Hz, 1H), 7.21 (d, <i>J</i> = 3.2 Hz, 1H), 7.12 (s, 1H), 6.54 (dd, <i>J</i> = 3.1, 0.9 Hz, 1H), 3.92 (s, 3H), 1.65 (sept, <i>J</i> = 7.5 Hz, 3H), 1.14 (d, <i>J</i> = 7.5 Hz, 18H); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 150.2, 136.1, 132.5, 131.4, 118.1, 106.6, 104.7, 102.5, 56.7, 18.2, 12.9; HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>29</sub>BrNOSi<sup>+</sup>, 382.1196 and 384.1176; found, 382.1198 and 384.1175.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 6-Bromo-3-iodo-5-methoxy-1-(triisopropylsilyl)-1<i>H</i>-indole (<b>13</b>)</h4><div class="NLM_p last">Protected bromoindole <b>11</b> (315 mg, 0.824 mmol) was dissolved in pyridine (0.5 mL), and the solution was cooled to 0 °C before ICl (1 M in CH<sub>2</sub>Cl<sub>2</sub>,0.90 mL, 0.90 mmol) was added slowly by syringe. After 15 min stirring at 0 °C, the cooling bath was removed and stirring continued for 30 min at rt. The reaction mixture was quenched with cold water (2 mL), diluted with ethyl acetate (20 mL), and HCl (1M, 3 mL) was added. The layers were separated and the aqueous layer extracted with EtOAc (2 × 20 mL), and the combined organic phase was washed with saturated NaCl (20 mL) before it was dried over MgSO<sub>4</sub> and concentrated under vacuum. The residue was purified by flash chromatography (hexane to hexane/EtOAc, 98:2 v/v) to give the title compound <b>13</b> (451 mg, 97%) as a colorless solid. <i>R</i><sub>f</sub> (hexane/EtOAc, 98:2 v/v) = 0.30; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.66 (s, 1H), 7.25 (s, 1H), 6.91 (s, 1H), 3.98 (s, 3H), 1.64 (hept, <i>J</i> = 7.5 Hz, 3H), 1.14 (d, <i>J</i> = 7.5 Hz, 18H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 151.1, 136.1, 135.3, 133.0, 118.3, 108.1, 102.7, 59.7, 56.7, 18.1, 12.8; HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>28</sub>BrINOSi<sup>+</sup>, 508.0163 and 510.0142; found, 508.0161 and 510.0139.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 5-(6-Bromo-5-methoxy-1-(triisopropylsilyl)-1<i>H</i>-indol-3-yl)oxazole (<b>14</b>)</h4><div class="NLM_p last">3-Iodoindole <b>13</b> (0.55 g, 1.08 mmol), K<sub>3</sub>PO<sub>4</sub> (0.689 g, 3.25 mmol), and 2-TIPS-oxazole-5-boronic acid pinacol ester (456 mg, 1.30 mmol) were dissolved in water (4.0 mL) and toluene (8.0 mL), and the mixture was degassed. PdCl<sub>2</sub>(dppf)·CH<sub>2</sub>Cl<sub>2</sub> (44 mg, 0.054 mmol) was added, and the mixture was degassed with argon before it was heated at 50 °C for 2 h. The reaction mixture was cooled to rt, diluted with EtOAc (20 mL), filtered through a short plug of Celite, water added (20 mL), and extracted with EtOAc (3 × 20 mL). The combined organic layers were washed with water (20 mL), saturated NaCl (20 mL), dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The residue was dissolved in THF (10.0 mL), and aliquots of aqueous HCl (3 M, 2.0 mL) were added every 15 min for 1 h until the intermediate product could not be observed by TLC analysis (hexane/EtOAc, 4:1 v/v). The mixture was basified by addition of saturated NaHCO<sub>3</sub> (4.0 mL), partitioned between water and EtOAc (20 mL). The organic layer was washed with water (15 mL) and saturated NaCl (15 mL), dried over MgSO<sub>4</sub>, and evaporated under reduced pressure. The crude product was purified by flash chromatography (hexanes to hexanes/EtOAc, 3:1 v/v) to give the title compound <b>14</b> (303 mg, 62% over two steps) as a colorless thick oil. <i>R</i><sub>f</sub> (hexanes/EtOAc, 3:1 v/v) = 0.35; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.91 (s, 1H), 7.70 (s, 1H), 7.52 (s, 1H), 7.28 (s, 1H), 7.24 (s, 1H), 3.98 (s, 3H), 1.70 (sept, <i>J</i> = 7.5 Hz, 3H), 1.16 (d, <i>J</i> = 7.5 Hz, 18H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 151.2, 149.1, 147.7, 136.4, 130.2, 127.5, 119.7, 118.7, 107.9, 107.4, 101.7, 56.8, 18.1, 12.8; HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>30</sub>BrN<sub>2</sub>O<sub>2</sub>Si<sup>+</sup>, 449.1254 and 451.1234; found, 449.1254 and 449.1232.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 5-(6-Bromo-1-(triisopropylsilyl)-1H-indol-3-yl)-2,4-iodooxazole (<b>15</b>)</h4><div class="NLM_p last">A solution of oxazole <b>14</b> (270 mg, 0.600 mmol) in THF (4.0 mL) was cooled to −78 °C, and LiHMDS (1M, 2.0 mL, 2.0 mmol) was added dropwise and stirring continued at −78 °C for 1 h. A solution of iodine (305 mg, 1.20 mmol) in THF (2.0 mL) was added dropwise and stirring was continued at −78 °C for 30 min, and then the reaction mixture was allowed to warm up to −40 °C and stirred for 1 h. A 10% aqueous solution of Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> (2.0 mL) was added and the resulting mixture diluted with water (20 mL), and the resulting mixture was extracted with EtOAc (3 × 15 mL). The combined organic layers were washed with saturated NaCl (15 mL), dried over MgSO<sub>4</sub>, and concentrated under reduced pressure. The resulting crude product was purified using flash chromatography to give the title compound <b>15</b> (129 mg, 31%) as colorless solid along with a mixture of 2-iodooxazole <b>16</b> and 4-iodooxazole <b>17</b> (162 mg, 47%). <i>R</i><sub>f</sub> (hexanes/EtOAc, 3:1 v/v) = 0.37; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.98 (s, 1H), 7.70 (s, 1H), 7.44 (s, 1H), 3.98 (s, 3H), 1.68 (p, <i>J</i> = 7.5 Hz, 3H), 1.17 (d, <i>J</i> = 7.5 Hz, 18H);<sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 156.0, 151.3, 135.9, 132.2, 128.0, 118.5, 108.2, 105.3, 102.6, 97.9, 77.2, 56.7, 18.1, 12.8.; HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>28</sub>BrI<sub>2</sub>N<sub>2</sub>O<sub>2</sub>Si<sup>+</sup>, 700.9187 and 702.9193; found, 701.0977 and 703.0955.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> <i>tert</i>-Butyl 2-(5-(6-bromo-1-(triisopropylsilyl)-1<i>H</i>-indol-3-yl)-4-iodooxazol-2-yl)-1<i>H</i>-pyrrole-1-carboxylate (<b>18</b>)</h4><div class="NLM_p last">To a suspension of diiodo oxazole <b>15</b> (115 mg, 0.164 mmol), <i>N</i>-Boc-pyrrole-2-boronic acid (52 mg, 0.246 mmol) and Cs<sub>2</sub>CO<sub>3</sub> (160 mg, 0.492 mmol) in 1,4-dioxane (4 mL) and water (1 mL), Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub> (13 mg, 0.016 mmol) was added, and the resulting solution was degassed with argon for 10 min and then stirred at rt for 48 h. The reaction mixture was concentrated in vacuo at 20 °C and diluted with EtOAc (15 mL) and water (10 mL). The organic layer was separated, washed with saturated NaCl, dried over MgSO<sub>4</sub>, and concentrated under vacuum. The crude product was purified by flash chromatography (hexanes/EtOAc, 5:1 v/v to 3:1 v/v) to give the title compound <b>18</b> (80 mg, 66%) as a colorless solid. <i>R</i><sub>f</sub> (hexanes/EtOAc, 3:1 v/v) = 0.25; <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.08 (s, 1H), 7.69 (s, 1H), 7.53 (s, 1H), 7.46 (dd, <i>J</i> = 3.3, 1.8 Hz, 1H), 6.78 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H), 6.31 (t, <i>J</i> = 3.4 Hz, 1H), 3.92 (s, 3H), 1.69 (p, <i>J</i> = 7.5 Hz, 3H), 1.41 (s, 9H), 1.18 (d, <i>J</i> = 7.5 Hz, 18H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 155.2, 151.2, 149.4, 148.5, 136.0, 132.4, 128.1, 125.0, 120.4, 119.1, 118.4, 111.2, 108.2, 106.2, 103.3, 85.0, 76.5, 57.0, 27.9, 18.2, 12.9.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Materials and Methods for Bioactivity Assays</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> MTS Assay</h4><div class="NLM_p last">The cytotoxicities of compounds <b>1</b>–<b>6</b> were determined against the human cancer cell lines MCF-7 (breast adenocarcinoma, American Type Culture Collection (ATCC) HTB-22), HT29 (colon adenocarcinoma, ATCC HTB-38), A2058 (melanoma, ATCC = CRL-11147), MDA-MB-468 (breast adenocarcinoma, ATCC = HTB-132), SK-BR-3 (breast adenocarcinoma, ATCC = HTB-30), MOLT-4 (acute lymphoblastic leukemia, ATCC = CRL-1582) and MV-4–11 (biphenotypic B myelomonocytic leukemia, ATCC = CRL-9591), and MRC5 (nonmalignant human lung fibroblasts, ATCC CCL-171) using the MTS assay.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> HT-29 and MOLT-4 were cultured in RPMI, MCF-7 and MRC-5 in E-MEM, A2058 in D-MEM, MDA-MB-468 in L15, SK-BR-3 in McCoy’s 5A, and MV-4–11 in Iscove basal medium. All cell lines were assayed in RPMI. All cells were incubated in 5% CO2 at 37 °C. When assayed, the adherent cell lines were seeded in 96-well microtiter plates at 2000 cells/well for MCF-7, HT-29, and A2058, 5000 cells/well for MDA-MB-468 and SK-BR-3, and 4000 cells/well for MRC-5. After 24 h, each cancer cell line was treated with various concentrations of compounds <b>1</b>–<b>6</b> and incubated for 72 h. The suspension cell liens MOLT-4 and MV-4–11 were seeded at 10.000 cells/well and were immediately treated with various concentrations of compounds <b>1</b>–<b>6</b> and incubated for 72 h. The negative control was RPMI with 1% DMSO (1% was the highest DMSO concentration used in compound wells). Positive control for MCF-7, HT-29, A2058, and MRC-5 was 0.5% Triton X-100 (Sigma-Aldrich). Positive control for MDA-MB-468, SK-BR-3, MOLT-4, and MV-4–11 was RPMI with 10% DMSO. After 72 h, 10 μL of MTS solution (Cell Titter 96 Aqueous One Solution Reagent, Promega) was added to each well, and the cells were incubated for 1 h at 37 °C. The absorbance was measured at 485 nm using a DTX multimode detector (Beckman Coulter). Cell viability was calculated as follows: cell survival (%) = (absorbance treated wells – absorbance positive control)/(absorbance negative control – absorbance positive control) × 100. The half-maximal inhibitory concertation (IC<sub>50</sub>) was calculated using Graph Pad Prism software. The time dependency of the activity of 3 against HT-29 and 4 against MCF-7 was further evaluated. Cells were added ranging concentrations of <b>3</b> and <b>4</b> 80, 60, and 36 h after seeding to evaluate the effect of the compounds after 4, 24, and 48 h exposure times. Results were measured and evaluated as described above. All viability assays were performed in triplicate in three independent experiments.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <sup>33</sup>P-ATP Filter Binding Kinase Assay</h4><div class="NLM_p last"><b>1</b>, <b>3</b> ,and <b>4</b> were tested against 140 kinases in the Premier Screen kinase panel at the International Centre for Kinase Profiling using radioactive (<sup>33</sup>P-ATP) filter-binding assays.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Compound <b>1</b> was tested at a concentration of 50 μM, <b>3</b> at 50, 5, and 1 μM, and <b>4</b> at 5 and 1 μM. Each experiment was conducted in duplicates. Furthermore, the IC<sub>50</sub> values against 13 relevant kinases were determined for <b>2</b>–<b>6</b>. The experiments are run as a single 10-point concentration curve.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Active-Site Directed Competition Binding Assay</h4><div class="NLM_p last">The binding interactions between <b>3</b> at 10 μM and 468 kinases were determined using the ScanMax kinase panel at DiscoverX.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> In these assays, the ability of a compound to bind to the active site of the kinase is measured. Binding prevents the kinase from binding to an immobilized ligand, thereby reducing the amount of kinase captured by a solid bead. The kinases are labeled with a DNA tag. The amount of kinase captured by the solid bead is quantified by qPCR detecting the DNA tags. The binding affinities are reported as percent of control (% ctrl) and are calculated by comparing captured kinase levels in test compound wells, negative control wells (DMSO), and positive control (control compounds) using the following equation: ((test compound signal – positive control signal)/(negative control signal – positive control signal) × 100 = % ctrl. The binding constant (<i>K</i><sub>d</sub>) of <b>3</b> against 19 selected kinases were determined using K<sub>d</sub>ELECT screening service at DiscoverX using 11-point dose response curves. The assay principle is as described above. <i>K</i><sub>d</sub> values were calculated using the Hill equation.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> In Vitro ATP Competitiveness Assay</h4><div class="NLM_p last">The compounds were diluted in 50 mM HEPES (pH 7.5), 10 mM MgCl<sub>2</sub>, 1 mM EGTA, and 0.01% Brij35. The kinase of interest (5 nM final concentration), fluorescent tracer (Invitrogen, 100 nM final concentration), and LanthaScreen Eu-anti-GST antibody (Invitrogen, 2 nM final concentration) were mixed with the respective compound dilutions (from 5 nM to 10 mM) and incubated for 1 h. The FRET signal was quantified using an EnVision Multilabellreader 2104 (PerkinElmer).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Intracellular ATP Competitiveness Assays</h4><div class="NLM_p last">In brief, HEK293 cells were seeded in 384 well plates and transfected with plasmids for expression of DRAK1 or DRAK2 fused to the NanoLuc luciferase. Twenty-four hour post transfection, cells were washed, permeabilized with digitonin, and treated with an ATP-competitive tracer according to the Manufacturer’s instructions.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Docking</h4><div class="NLM_p last">In brief, the binding mode of <b>3</b> into the ATP-binding pocket of PIM1 was determined by manual and computational docking using the Glide software from Schrödinger. Protein coordinates were extracted from the crystal structure of the structure of PIM1 bound to <i>N</i>-(5-(2-fluorophenyl)-1H-pyrrolo[2,3-<i>b</i>]pyridin-3-yl)-5-((((3<i>R</i>,4<i>R</i>)-3-fluoropiperidin-4-yl)methyl)amino)pyrazolo[1,5-<i>a</i>]pyrimidine-3-carboxamide (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B">4K1B</a>).</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Selective Cytotoxicity</h4><div class="NLM_p last">Compound <b>3</b> was assayed against the cell lines found in the OncoLead cell panel using a sulforhodamin B (SRB) protein staining assay technique<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> (Oncolead, Karlsfeld, Germany). Cancer cells (87 in total) and peripheral blood mononuclear cells (PBMCs) were assayed. Briefly, 24 h post seeding, cells were treated in triplicate with a 10-fold serial dilution over 10 concentrations of <b>3</b>. The highest assayed concentration was 30 μM. Following 72 h incubation, cells growth inhibitions were measured using SRB in a protein staining assay. Results were used to calculate the IC<sub>50</sub> value of <b>3</b> against the 88 cell lines.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Polypharmacological Effects</h4><div class="NLM_p last">The polypharmacological effects of <b>3</b> were evaluated using the CEREP Panlab Diversity Panel. The purpose of this panel of binding and cellular assays performed by CEREP is to identify a subset of potential receptors, transporters, ion channels, etc., for which a compound displays affinity. The panel includes radioligand binding- and cell-based assays, which are run as described by the service provider. The results are expressed as percent-of-control specific binding.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> In Vivo Toxicity against Zebrafish Embryos</h4><div class="NLM_p last">BioReperia (Linköping, Sweden) conducted assays to evaluate the toxicological properties of compound <b>3</b> against zebrafish embryos. Zebrafish embryos were produced by natural breeding from two males and four females, collected, screened for fertilization, and moved to E4 medium. Embryos were incubated and at 28.5 °C. After fertilization (4 h) the embryos were allocated into six groups of 20 individuals. Unfertilized eggs or embryos that did not appear healthy or exhibited any obvious developmental defects were excluded prior to treatment onset at 4 h after fertilization of the egg. The assays were conducted at five concentrations (1396, 140, 14, 1.4, and 0.1 μM). Vehicle (DMSO) was given at 0.5%, corresponding to the highest concentration in the treatment group.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Determination of ADME Parameters</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Chemical Stability Assay</h4><div class="NLM_p last">Chemical stability was determined by incubating test compounds at a final concentration of 2 μM in aqueous buffer at pH 7.4 for 1, 7, and 24 h, respectively. The percentage of remaining compound in relation to the zero time point was calculated following LC–MS-based measurement of sample aliquots of each time point.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Stability in Cell Culture Medium Assay</h4><div class="NLM_p last">Stability in cell culture medium was determined by incubating compound <b>3</b> at a final concentration of 2 μM in D-MEM supplemented with 10% fetal calf serum at pH 7.4 for 1, 7, and 24 h, respectively. The percentage of remaining compound in relation to the zero time point was calculated following LC–MS-based measurement of sample aliquots of each time point.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Kinetic Solubility Assay</h4><div class="NLM_p last">Aqueous solubility of compounds was determined by spectrophotometrical measurement of the kinetic solubility of a 500 μM compound solution in aqueous buffer pH 7.4 compared to a solution in the organic solvent acetonitrile after 90 min of vigorous shaking at room temperature.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Permeability through Artificial Membranes Assay</h4><div class="NLM_p last">Permeability through artificial membranes (PAMPA) was performed at an initial concentration of 500 μM of the compound in the donor compartment. After an incubation period of 20 h, absorption of the receiver wells was measured by spectrophotometry and permeation was calculated by normalization of the compound flux across a blank filter.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Microsomal Stability Phase I Assay</h4><div class="NLM_p last">Metabolic stability under oxidative conditions was measured in liver microsomes from different species by LC–MS-based measuring of depletion of compound at a concentration of 3 μM over time up to 50 min at 37 °C. On the basis of compound half-life <i>t</i><sub>1/2</sub>, in vitro intrinsic clearance CL<sub>int</sub> was calculated. Cl<sub>int</sub> = (V × 0.693)/(<i>t</i><sub>1/2</sub> × mg).</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Microsomal Stability Phase II Assay</h4><div class="NLM_p last">Metabolic stability under conjugative conditions was measured in the glucuronidation assay by LC–MS-based determination of % remaining of selected compounds at a concentration of 5 μM in incubations with liver microsomes supplemented with UDP-glucuronic acid for 1 h at 37 °C.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Plasma Stability Assay</h4><div class="NLM_p last">Plasma stability was measured by LC–MS-based determination of % remaining of selected compounds at a concentration of 5 μM after incubation in 100% plasma obtained from different species for 1 h at 37 °C.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Plasma Protein Binding Assay</h4><div class="NLM_p last">Assessment of plasma protein binding was measured by equilibrium dialysis by incubating the compound of interest at a concentration of 5 μM for 6 h at 37 °C in 50% plasma in buffer (v/v) followed by LC–MS-based determination of final compound concentrations. The resulting fraction unbound at 50% plasma (fu<sub>50%</sub>) is extrapolated to the fraction unbound at 100% plasma (fu<sub>100%</sub>) using the following equation: fu<sub>100%</sub> = fu<sub>50%</sub>/(2 – fu<sub>50%</sub>).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01006" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11524" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11524" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01006" class="ext-link">10.1021/acs.jmedchem.9b01006</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">NMR spectra for compounds <b>3</b>–<b>8</b>, synthetic intermediates toward <b>3</b>, synthetic <b>3</b> and <b>4</b>, isolated <b>1</b> and <b>2</b>, and comparison of <sup>1</sup>H- and <sup>13</sup>C NMR data for isolated and synthetic <b>3</b> and <b>4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">The activities of <b>1</b>, <b>3</b>, and <b>4</b> against 140 kinases using a radioactive (<sup>33</sup>P-ATP) filter-binding assays, binding affinity of <b>3</b> against the 468 kinases in the DiscoverX ScanMax panel, diversity profile of <b>3</b>, IC<sub>50</sub> values of <b>3</b> against the 88 cell lines found in the OncoLead panel, TreeSpot analysis of <b>3</b> at 5 μM based on Discoverx data, curve images for <i>K</i><sub>d</sub> determination of <b>3</b>, dose–response-time curves for <b>3</b> against HT-29 and <b>4</b> against MCF-7 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_001.pdf">jm9b01006_si_001.pdf (2.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf">jm9b01006_si_002.pdf (597.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_003.csv">jm9b01006_si_003.csv (3.2 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01006" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82115" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82115" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kine Ø. Hansen</span> - <span class="hlFld-Affiliation affiliation">Marbio, UiT - The Arctic University of
Norway, Breivika, NO-9037 Tromsø, Norway</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9023-1958" title="Orcid link">http://orcid.org/0000-0002-9023-1958</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e2a98b8c87ccadccaa838c9191878ca2978b96cc8c8d"><span class="__cf_email__" data-cfemail="064d6f68632849284e67687575636846736f72286869">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bengt Erik Haug</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Centre for Pharmacy, University
of Bergen, Allégaten 41, NO-5007 Bergen, Norway</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3014-9538" title="Orcid link">http://orcid.org/0000-0003-3014-9538</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d795b2b9b0a3fa92a5bebcf99fb6a2b097a2beb5f9b9b8"><span class="__cf_email__" data-cfemail="eeac8b80899ac3ab9c8785c0a68f9b89ae9b878cc08081">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeanette H. Andersen</span> - <span class="hlFld-Affiliation affiliation">Marbio, UiT - The Arctic University of
Norway, Breivika, NO-9037 Tromsø, Norway</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annette Bayer</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, UiT - The Arctic University
of Norway, Breivika, NO-9037 Tromsø, Norway</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3481-200X" title="Orcid link">http://orcid.org/0000-0003-3481-200X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sunil K. Pandey</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Centre for Pharmacy, University
of Bergen, Allégaten 41, NO-5007 Bergen, Norway</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marianne Lorentzen</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Science and Technology, Department of Chemistry, Bioscience and
Environmental Engineering, University of
Stavanger, NO-4036 Stavanger, Norway</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kåre B. Jørgensen</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Science and Technology, Department of Chemistry, Bioscience and
Environmental Engineering, University of
Stavanger, NO-4036 Stavanger, Norway</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0662-1839" title="Orcid link">http://orcid.org/0000-0003-0662-1839</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magne O. Sydnes</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Science and Technology, Department of Chemistry, Bioscience and
Environmental Engineering, University of
Stavanger, NO-4036 Stavanger, Norway</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9413-6969" title="Orcid link">http://orcid.org/0000-0001-9413-6969</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yngve Guttormsen</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, UiT - The Arctic University
of Norway, Breivika, NO-9037 Tromsø, Norway</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthias Baumann</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Uwe Koch</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bert Klebl</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Eickhoff</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Johan Isaksson</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, UiT - The Arctic University
of Norway, Breivika, NO-9037 Tromsø, Norway</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Espen H. Hansen</span> - <span class="hlFld-Affiliation affiliation">Marbio, UiT - The Arctic University of
Norway, Breivika, NO-9037 Tromsø, Norway</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>K.Ø.H. and E.H.H. isolated the compounds. J.I. and E.H.H. performed the analytical characterization of the isolated compounds. K.Ø.H. performed the in vitro cytotoxicity assays. S.K.P., M.L., B.E.H., A.B., K.B.J., M.O.S., and Y.G. planned and conducted the synthesis and characterized the synthetic products. B.K. and J.E. performed the mode-of-action studies. M.B. measured the in vitro ADME parameters. U.K. performed the docking study. The manuscript was written through the contribution of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors are grateful to Marbank for collecting the specimens of <i>T. breitfussi</i> and to R. Johansen for taxonomic identification of the species. M.L. would like to thank the University of Stavanger for cofunding the postdoctoral studies. This work was supported by the Research Council of Norway (Grants 174885/130, 224790/O30, and 244264/O30).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">GC</td><td class="NLM_def"><p class="first last">gini coefficient</p></td></tr><tr><td class="NLM_term">RA</td><td class="NLM_def"><p class="first last">residual enzyme activities</p></td></tr><tr><td class="NLM_term">PoC</td><td class="NLM_def"><p class="first last">percent of control</p></td></tr><tr><td class="NLM_term">ICKP</td><td class="NLM_def"><p class="first last">International Centre for Kinase Profiling in Dundee</p></td></tr><tr><td class="NLM_term">TIPS</td><td class="NLM_def"><p class="first last">triisopropyl ether</p></td></tr><tr><td class="NLM_term">TBS</td><td class="NLM_def"><p class="first last"><i>t</i>-butyldimethylsilyl</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51642" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51642" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhullar, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarón, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupasinghe, H. P. V.</span></span> <span> </span><span class="NLM_article-title">Kinase-targeted cancer therapies: progress, challenges and future directions</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">48</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0804-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2Fs12943-018-0804-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=29455673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=48&author=K.+S.+Bhullarauthor=N.+O.+Lagar%C3%B3nauthor=E.+M.+McGowanauthor=I.+Parmarauthor=A.+Jhaauthor=B.+P.+Hubbardauthor=H.+P.+V.+Rupasinghe&title=Kinase-targeted+cancer+therapies%3A+progress%2C+challenges+and+future+directions&doi=10.1186%2Fs12943-018-0804-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-targeted cancer therapies: progress, challenges and future directions</span></div><div class="casAuthors">Bhullar, Khushwant S.; Lagaron, Naiara Orrego; McGowan, Eileen M.; Parmar, Indu; Jha, Amitabh; Hubbard, Basil P.; Rupasinghe, H. P. Vasantha</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48/1-48/20</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">The human genome encodes 538 protein kinases that transfer a γ-phosphate group from ATP to serine, threonine, or tyrosine residues.  Many of these kinases are assocd. with human cancer initiation and progression.  The recent development of small-mol. kinase inhibitors for the treatment of diverse types of cancer has proven successful in clin. therapy.  Significantly, protein kinases are the second most targeted group of drug targets, after the G-proteincoupled receptors.  Since the development of the first protein kinase inhibitor, in the early 1980s, 37 kinase inhibitors have received FDA approval for treatment of malignancies such as breast and lung cancer.  Furthermore, about 150 kinase-targeted drugs are in clin. phase trials, and many kinase-specific inhibitors are in the preclin. stage of drug development.  Nevertheless, many factors confound the clin. efficacy of these mols.  Specific tumor genetics, tumor microenvironment, drug resistance, and pharmacogenomics det. how useful a compd. will be in the treatment of a given cancer.  This review provides an overview of kinase-targeted drug discovery and development in relation to oncol. and highlights the challenges and future potential for kinase-targeted cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb-edOGHyJPbVg90H21EOLACvtfcHk0ljw6U-MO23s4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7fP&md5=30811747f13749327d998c5dadd2161d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0804-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0804-2%26sid%3Dliteratum%253Aachs%26aulast%3DBhullar%26aufirst%3DK.%2BS.%26aulast%3DLagar%25C3%25B3n%26aufirst%3DN.%2BO.%26aulast%3DMcGowan%26aufirst%3DE.%2BM.%26aulast%3DParmar%26aufirst%3DI.%26aulast%3DJha%26aufirst%3DA.%26aulast%3DHubbard%26aufirst%3DB.%2BP.%26aulast%3DRupasinghe%26aufirst%3DH.%2BP.%2BV.%26atitle%3DKinase-targeted%2520cancer%2520therapies%253A%2520progress%252C%2520challenges%2520and%2520future%2520directions%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D48%26doi%3D10.1186%2Fs12943-018-0804-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolch, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halasz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granovskaya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kholodenko, B. N.</span></span> <span> </span><span class="NLM_article-title">The dynamic control of signal transduction networks in cancer cells</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1038/nrc3983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnrc3983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26289315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlKkt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=515-527&author=W.+Kolchauthor=M.+Halaszauthor=M.+Granovskayaauthor=B.+N.+Kholodenko&title=The+dynamic+control+of+signal+transduction+networks+in+cancer+cells&doi=10.1038%2Fnrc3983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic control of signal transduction networks in cancer cells</span></div><div class="casAuthors">Kolch, Walter; Halasz, Melinda; Granovskaya, Marina; Kholodenko, Boris N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">515-527</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer is often considered a genetic disease.  However, much of the enormous plasticity of cancer cells to evolve different phenotypes, to adapt to challenging microenvironments and to withstand therapeutic assaults is encoded by the structure and spatiotemporal dynamics of signal transduction networks.  In this Review, we discuss recent concepts concerning how the rich signalling dynamics afforded by these networks are regulated and how they impinge on cancer cell proliferation, survival, invasiveness and drug resistance.  Understanding this dynamic circuitry by math. modeling could pave the way to new therapeutic approaches and personalized treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4ExQFCUoT6rVg90H21EOLACvtfcHk0ljw6U-MO23s4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlKkt7fJ&md5=b26429e328fecc69aacd7fa7b72c129f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3983%26sid%3Dliteratum%253Aachs%26aulast%3DKolch%26aufirst%3DW.%26aulast%3DHalasz%26aufirst%3DM.%26aulast%3DGranovskaya%26aufirst%3DM.%26aulast%3DKholodenko%26aufirst%3DB.%2BN.%26atitle%3DThe%2520dynamic%2520control%2520of%2520signal%2520transduction%2520networks%2520in%2520cancer%2520cells%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D515%26epage%3D527%26doi%3D10.1038%2Fnrc3983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brognard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span> <span> </span><span class="NLM_article-title">Protein kinase signaling networks in cancer</span>. <i>Curr. Opin. Genet. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.gde.2010.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.gde.2010.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21123047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFKjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4-11&author=J.+Brognardauthor=T.+Hunter&title=Protein+kinase+signaling+networks+in+cancer&doi=10.1016%2Fj.gde.2010.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase signaling networks in cancer</span></div><div class="casAuthors">Brognard, John; Hunter, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Genetics & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-11</span>CODEN:
                <span class="NLM_cas:coden">COGDET</span>;
        ISSN:<span class="NLM_cas:issn">0959-437X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein kinases orchestrate the activation of signaling cascades in response to extracellular and intracellular stimuli to control cell growth, proliferation, and survival.  The complexity of numerous intracellular signaling pathways is highlighted by the no. of kinases encoded by the human genome (539) and the plethora of phosphorylation sites identified in phosphoproteomic studies.  Perturbation of these signaling networks by mutations or abnormal protein expression underlies the cause of many diseases including cancer.  Recent RNAi screens and cancer genomic sequencing studies have revealed that many more kinases than anticipated contribute to tumorigenesis and are potential targets for inhibitor drug development intervention.  This review will highlight recent insights into known pathways essential for tumorigenesis and discuss exciting new pathways for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYS-FgMlgzm7Vg90H21EOLACvtfcHk0ljw6U-MO23s4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFKjtbo%253D&md5=04d06a0c695fb57e72df0bd2b4bc25e3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2010.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2010.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DBrognard%26aufirst%3DJ.%26aulast%3DHunter%26aufirst%3DT.%26atitle%3DProtein%2520kinase%2520signaling%2520networks%2520in%2520cancer%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2011%26volume%3D21%26spage%3D4%26epage%3D11%26doi%3D10.1016%2Fj.gde.2010.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer with kinase inhibitors</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1780</span>– <span class="NLM_lpage">1789</span>, <span class="refDoi"> DOI: 10.1172/JCI76094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1172%2FJCI76094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=25932675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC2MjpvFGitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=1780-1789&author=S.+Grossauthor=R.+Rahalauthor=N.+Stranskyauthor=C.+Lengauerauthor=K.+P.+Hoeflich&title=Targeting+cancer+with+kinase+inhibitors&doi=10.1172%2FJCI76094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with kinase inhibitors</span></div><div class="casAuthors">Gross Stefan; Rahal Rami; Stransky Nicolas; Lengauer Christoph; Hoeflich Klaus P</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1780-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kinase inhibitors have played an increasingly prominent role in the treatment of cancer and other diseases.  Currently, more than 25 oncology drugs that target kinases have been approved, and numerous additional therapeutics are in various stages of clinical evaluation.  In this Review, we provide an in-depth analysis of activation mechanisms for kinases in cancer, highlight recent successes in drug discovery, and demonstrate the clinical impact of selective kinase inhibitors.  We also describe the substantial progress that has been made in designing next-generation inhibitors to circumvent on-target resistance mechanisms, as well as ongoing strategies for combining kinase inhibitors in the clinic.  Last, there are numerous prospects for the discovery of novel kinase targets, and we explore cancer immunotherapy as a new and promising research area for studying kinase biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyaX6QAuw1pbFU0A1HOshhfW6udTcc2eYaAEwk0PnluLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjpvFGitA%253D%253D&md5=b35d25dcf88d08a43ca3a0945c048111</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1172%2FJCI76094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI76094%26sid%3Dliteratum%253Aachs%26aulast%3DGross%26aufirst%3DS.%26aulast%3DRahal%26aufirst%3DR.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DTargeting%2520cancer%2520with%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2015%26volume%3D125%26spage%3D1780%26epage%3D1789%26doi%3D10.1172%2FJCI76094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sawyers, C.</span></span> <span> </span><span class="NLM_article-title">Targeted cancer therapy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">294</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1038/nature03095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnature03095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=15549090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=294-297&author=C.+Sawyers&title=Targeted+cancer+therapy&doi=10.1038%2Fnature03095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted cancer therapy</span></div><div class="casAuthors">Sawyers, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">294-297</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Disruption of the normal regulation of cell-cycle progression and division lies at the heart of the events leading to cancer.  Complex networks of regulatory factors, the tumor microenvironment and stress signals, such as those resulting from damaged DNA, dictate whether cancer cells proliferate or die.  Recent progress in understanding the mol. changes that underlie cancer development offer the prospect of specifically targeting malfunctioning mols. and pathways to achieve more effective and rational cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAGMTW2TYsubVg90H21EOLACvtfcHk0lhcZV9jmU__Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtL8%253D&md5=5bd377d2c9ddcffc136d5cf181facd02</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature03095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03095%26sid%3Dliteratum%253Aachs%26aulast%3DSawyers%26aufirst%3DC.%26atitle%3DTargeted%2520cancer%2520therapy%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D294%26epage%3D297%26doi%3D10.1038%2Fnature03095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroschinsky, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolzel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Bonin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochanek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiehl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellongowski, P.</span></span> <span> </span><span class="NLM_article-title">New drugs, new toxicities: severe side effects of modern targeted and immunotherapy of cancer and their management</span>. <i>Crit. Care.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">89</span>, <span class="refDoi"> DOI: 10.1186/s13054-017-1678-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2Fs13054-017-1678-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=28407743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC1cvns1Ckuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=89&author=F.+Kroschinskyauthor=F.+Stolzelauthor=S.+von+Boninauthor=G.+Beutelauthor=M.+Kochanekauthor=M.+Kiehlauthor=P.+Schellongowski&title=New+drugs%2C+new+toxicities%3A+severe+side+effects+of+modern+targeted+and+immunotherapy+of+cancer+and+their+management&doi=10.1186%2Fs13054-017-1678-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">New drugs, new toxicities: severe side effects of modern targeted and immunotherapy of cancer and their management</span></div><div class="casAuthors">Kroschinsky Frank; Stolzel Friedrich; von Bonin Simone; Beutel Gernot; Kochanek Matthias; Kiehl Michael; Schellongowski Peter</div><div class="citationInfo"><span class="NLM_cas:title">Critical care (London, England)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Pharmacological and cellular treatment of cancer is changing dramatically with benefits for patient outcome and comfort, but also with new toxicity profiles.  The majority of adverse events can be classified as mild or moderate, but severe and life-threatening complications requiring ICU admission also occur.  This review will focus on pathophysiology, symptoms, and management of these events based on the available literature.While standard antineoplastic therapy is associated with immunosuppression and infections, some of the recent approaches induce overwhelming inflammation and autoimmunity.  Cytokine-release syndrome (CRS) describes a complex of symptoms including fever, hypotension, and skin reactions as well as lab abnormalities.  CRS may occur after the infusion of monoclonal or bispecific antibodies (MABs, BABs) targeting immune effectors and tumor cells and is a major concern in recipients of chimeric antigen receptor (CAR) modified T lymphocytes as well.  BAB and CAR T-cell treatment may also be compromised by central nervous system (CNS) toxicities such as encephalopathy, cerebellar alteration, disturbed consciousness, or seizures.  While CRS is known to be induced by exceedingly high levels of inflammatory cytokines, the pathophysiology of CNS events is still unclear.  Treatment with antibodies against inhibiting immune checkpoints can lead to immune-related adverse events (IRAEs); colitis, diarrhea, and endocrine disorders are often the cause for ICU admissions.Respiratory distress is the main reason for ICU treatment in cancer patients and is attributable to infectious agents in most cases.  In addition, some of the new drugs are reported to cause non-infectious lung complications.  While drug-induced interstitial pneumonitis was observed in a substantial number of patients treated with phosphoinositol-3-kinase inhibitors, IRAEs may also affect the lungs.Inhibitors of angiogenetic pathways have increased the antineoplastic portfolio.  However, vessel formation is also essential for regeneration and tissue repair.  Therefore, severe vascular side effects, including thromboembolic events, gastrointestinal bleeding or perforation, hypertension, and congestive heart failure, compromise antitumor efficacy.The limited knowledge of the pathophysiology and management of life-threatening complications relating to new cancer drugs presents a need to provide ICU staff, oncologists, and organ specialists with evidence-based algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDBx0_VHiayEloaUqgkFHkfW6udTcc2eaea47YlHDszrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvns1Ckuw%253D%253D&md5=0d3e40160efb87574938d8886f4aa6f1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2Fs13054-017-1678-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13054-017-1678-1%26sid%3Dliteratum%253Aachs%26aulast%3DKroschinsky%26aufirst%3DF.%26aulast%3DStolzel%26aufirst%3DF.%26aulast%3Dvon%2BBonin%26aufirst%3DS.%26aulast%3DBeutel%26aufirst%3DG.%26aulast%3DKochanek%26aufirst%3DM.%26aulast%3DKiehl%26aufirst%3DM.%26aulast%3DSchellongowski%26aufirst%3DP.%26atitle%3DNew%2520drugs%252C%2520new%2520toxicities%253A%2520severe%2520side%2520effects%2520of%2520modern%2520targeted%2520and%2520immunotherapy%2520of%2520cancer%2520and%2520their%2520management%26jtitle%3DCrit.%2520Care.%26date%3D2017%26volume%3D21%26spage%3D89%26doi%3D10.1186%2Fs13054-017-1678-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lhcZV9jmU__Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0lgdlu_hgh6GTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanssen, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demissie, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svenson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blinov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repisky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruud, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elyashberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspars, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaksson, J.</span></span> <span> </span><span class="NLM_article-title">A combined atomic force microscopy and computational approach for the structural elucidation of breitfussin A and B: Highly modified halogenated dipeptides from Thuiaria breitfussi</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">12238</span>– <span class="NLM_lpage">12241</span>, <span class="refDoi"> DOI: 10.1002/anie.201203960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1002%2Fanie.201203960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFyjsb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=12238-12241&author=K.+O.+Hanssenauthor=B.+Schulerauthor=A.+J.+Williamsauthor=T.+B.+Demissieauthor=E.+Hansenauthor=J.+H.+Andersenauthor=J.+Svensonauthor=K.+Blinovauthor=M.+Repiskyauthor=F.+Mohnauthor=G.+Meyerauthor=J.+S.+Svendsenauthor=K.+Ruudauthor=M.+Elyashbergauthor=L.+Grossauthor=M.+Jasparsauthor=J.+Isaksson&title=A+combined+atomic+force+microscopy+and+computational+approach+for+the+structural+elucidation+of+breitfussin+A+and+B%3A+Highly+modified+halogenated+dipeptides+from+Thuiaria+breitfussi&doi=10.1002%2Fanie.201203960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Combined Atomic Force Microscopy and Computational Approach for the Structural Elucidation of Breitfussin A and B: Highly Modified Halogenated Dipeptides from Thuiaria breitfussi</span></div><div class="casAuthors">Hanssen, Kine O.; Schuler, Bruno; Williams, Antony J.; Demissie, Taye B.; Hansen, Espen; Andersen, Jeanette H.; Svenson, Johan; Blinov, Kirill; Repisky, Michal; Mohn, Fabian; Meyer, Gerhard; Svendsen, John-Sigurd; Ruud, Kenneth; Elyashberg, Mikhail; Gross, Leo; Jaspars, Marcel; Isaksson, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">12238-12241</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The subject of this study was the Arctic hydrozoan Thuiaria breitfussi (Family Sertulariidae).  The sample of Thuiaria breitfussi was collected from Bear island in 2007 and extd. with MeOH/CH2Cl2.  A series of liq.-liq. extns. followed by mass-guided HPLC purifn. yielded 6.2 mg of purified breitfussin A (1) and 4.0 mg breitfussin B (2).  The breitfussins are unusual structures contg. a combination of indole-oxazole-pyrrole units.  Given the limited quantity isolated, x-ray crystallog. was not possible, and structure detn. using spectroscopic data was unable to propose a single unequivocal structure consistent with all the data.  The structure was therefore supported by a combination of AFM, CASE, and DFT calcns., none of which were able to propose a unique soln. individually.  Remarkably, AFM could be used to det. all the connection positions of the cyclic systems as well as those of the substituent groups (MeO, Br, and I); this information is difficult to obtain with other techniques.  Moreover, different AFM contrast above halogen atoms and the MeO group indicate chem. sensitivity within individual mols.  Using such powerful methods on limited quantities of bioactive natural products with complex structures will become more common as scientists begin to access unusual taxa from extreme locations with unique chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeEzw_ZXy0xbVg90H21EOLACvtfcHk0lgdlu_hgh6GTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFyjsb%252FL&md5=9b7e9778ae429f3aa49df0a65ce1b441</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fanie.201203960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201203960%26sid%3Dliteratum%253Aachs%26aulast%3DHanssen%26aufirst%3DK.%2BO.%26aulast%3DSchuler%26aufirst%3DB.%26aulast%3DWilliams%26aufirst%3DA.%2BJ.%26aulast%3DDemissie%26aufirst%3DT.%2BB.%26aulast%3DHansen%26aufirst%3DE.%26aulast%3DAndersen%26aufirst%3DJ.%2BH.%26aulast%3DSvenson%26aufirst%3DJ.%26aulast%3DBlinov%26aufirst%3DK.%26aulast%3DRepisky%26aufirst%3DM.%26aulast%3DMohn%26aufirst%3DF.%26aulast%3DMeyer%26aufirst%3DG.%26aulast%3DSvendsen%26aufirst%3DJ.%2BS.%26aulast%3DRuud%26aufirst%3DK.%26aulast%3DElyashberg%26aufirst%3DM.%26aulast%3DGross%26aufirst%3DL.%26aulast%3DJaspars%26aufirst%3DM.%26aulast%3DIsaksson%26aufirst%3DJ.%26atitle%3DA%2520combined%2520atomic%2520force%2520microscopy%2520and%2520computational%2520approach%2520for%2520the%2520structural%2520elucidation%2520of%2520breitfussin%2520A%2520and%2520B%253A%2520Highly%2520modified%2520halogenated%2520dipeptides%2520from%2520Thuiaria%2520breitfussi%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D12238%26epage%3D12241%26doi%3D10.1002%2Fanie.201203960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guttormsen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, A.</span></span> <span> </span><span class="NLM_article-title">A concise total synthesis of breitfussin A and B</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1021/ol503348n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol503348n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOgsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=122-125&author=S.+K.+Pandeyauthor=Y.+Guttormsenauthor=B.+E.+Haugauthor=C.+Hedbergauthor=A.+Bayer&title=A+concise+total+synthesis+of+breitfussin+A+and+B&doi=10.1021%2Fol503348n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A Concise Total Synthesis of Breitfussin A and B</span></div><div class="casAuthors">Pandey, Sunil Kumar; Guttormsen, Yngve; Haug, Bengt Erik; Hedberg, Christian; Bayer, Annette</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-125</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first total synthesis of breitfussin A and B is described.  The approach features two palladium-catalyzed cross-couplings installing the indole and pyrrole onto the oxazole core and selective lithiation/iodination of a common indole-oxazole fragment providing 2,4-diiodinated or 2-iodinated oxazoles as potential precursors for breitfussin A and B, resp.  An unexpected acid promoted deiodination was utilized in the synthesis of breitfussin B.  Comparison of the synthetic material with previously reported spectral data of isolated breitfussin A and B verified the structure of the breitfussin framework.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmyJQcWaG1rVg90H21EOLACvtfcHk0lgdlu_hgh6GTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOgsrbL&md5=af3d7e450cada9ca23a105f205d365d4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fol503348n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol503348n%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DS.%2BK.%26aulast%3DGuttormsen%26aufirst%3DY.%26aulast%3DHaug%26aufirst%3DB.%2BE.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DBayer%26aufirst%3DA.%26atitle%3DA%2520concise%2520total%2520synthesis%2520of%2520breitfussin%2520A%2520and%2520B%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D122%26epage%3D125%26doi%3D10.1021%2Fol503348n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benayahu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleyer, M.</span></span> <span> </span><span class="NLM_article-title">Phorbazoles A-D, novel chlorinated phenylpyrrolyloxazoles from the marine sponge phorbas aff. clathrata</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2589</span>– <span class="NLM_lpage">2592</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(00)77179-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2FS0040-4039%2800%2977179-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK2cXkslylu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1994&pages=2589-2592&author=A.+Rudiauthor=Z.+Steinauthor=S.+Greenauthor=I.+Goldbergauthor=Y.+Kashmanauthor=Y.+Benayahuauthor=M.+Schleyer&title=Phorbazoles+A-D%2C+novel+chlorinated+phenylpyrrolyloxazoles+from+the+marine+sponge+phorbas+aff.+clathrata&doi=10.1016%2FS0040-4039%2800%2977179-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Phorbazoles A-D, novel chlorinated phenylpyrrolyloxazoles from the marine sponge Phorbas aff. clathrata</span></div><div class="casAuthors">Rudi, A.; Stein, Z.; Green, S.; Goldberg, I.; Kashman, Y.; Benayahu, Y.; Schleyer, M.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2589-92</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">Four new chlorinated phenylpyrrolyloxazoles, phorbazoles A-D (I-IV) were isolated from the sponge P. clathrata.  The structure of phorbazole A was unambiguously detd. by x-ray diffraction studies of its di-Me deriv..  The structures of compds. II-IV were detd. by extensive use of 1- and 2-dimensional NMR techniques.  Phorbazoles A-D represent a new class of marine alkaloids embodying the previously unprecedented, chlorinated pyrrole moiety, from marine sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt4_HTLpSI6LVg90H21EOLACvtfcHk0lis6-FlOGrCeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkslylu7g%253D&md5=875c18bda9ba9a30fb0ec209a34d9602</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2800%2977179-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252800%252977179-6%26sid%3Dliteratum%253Aachs%26aulast%3DRudi%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DZ.%26aulast%3DGreen%26aufirst%3DS.%26aulast%3DGoldberg%26aufirst%3DI.%26aulast%3DKashman%26aufirst%3DY.%26aulast%3DBenayahu%26aufirst%3DY.%26aulast%3DSchleyer%26aufirst%3DM.%26atitle%3DPhorbazoles%2520A-D%252C%2520novel%2520chlorinated%2520phenylpyrrolyloxazoles%2520from%2520the%2520marine%2520sponge%2520phorbas%2520aff.%2520clathrata%26jtitle%3DTetrahedron%2520Lett.%26date%3D1994%26volume%3D35%26spage%3D2589%26epage%3D2592%26doi%3D10.1016%2FS0040-4039%2800%2977179-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenical, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Duyne, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Isolation and structure determination of diazonamides A and B, unusual cytotoxic metabolites from the marine ascidian Diazona chinensis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2303</span>– <span class="NLM_lpage">2304</span>, <span class="refDoi"> DOI: 10.1021/ja00006a060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00006a060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK3MXhtFyrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=1991&pages=2303-2304&author=N.+Lindquistauthor=W.+Fenicalauthor=G.+D.+Van+Duyneauthor=J.+Clardy&title=Isolation+and+structure+determination+of+diazonamides+A+and+B%2C+unusual+cytotoxic+metabolites+from+the+marine+ascidian+Diazona+chinensis&doi=10.1021%2Fja00006a060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and structure determination of diazonamides A and B, unusual cytotoxic metabolites from the marine ascidian Diazona chinensis</span></div><div class="casAuthors">Lindquist, Niels; Fenical, William; Van Duyne, Gregory D.; Clardy, Jon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2303-4</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Two novel cytotoxic metabolites, diazonamides A and B (I and II, resp.), were isolated from the colonial marine ascidian D. chinensis from Siquijor Is., Philippines.  The structures were elucidated by a combination of single crystal x-ray diffraction and spectroscopic techniques.  The diazonamides represent an entirely new class of halogenated, highly unsatd. cyclic peptides contg. derivs. of ≥3 common amino acids: a 3,4,5-trisubstituted L-tyrosine, a tryptophan substituted at the 2 and 4 positions of the indole, and an L-valine.  These fragments and others of less clear biosynthetic origin have been assembled into an extremely rigid framework.  Diazonamide A has potent in vitro activity against HCT-116 human colon carcinoma and B-16 murine melanoma cancer cell lines with IC50 values <15 ng/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0cV4-hmu5k7Vg90H21EOLACvtfcHk0lis6-FlOGrCeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhtFyrsbc%253D&md5=db7c81b87c468c609847c7fee82cc512</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fja00006a060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00006a060%26sid%3Dliteratum%253Aachs%26aulast%3DLindquist%26aufirst%3DN.%26aulast%3DFenical%26aufirst%3DW.%26aulast%3DVan%2BDuyne%26aufirst%3DG.%2BD.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DIsolation%2520and%2520structure%2520determination%2520of%2520diazonamides%2520A%2520and%2520B%252C%2520unusual%2520cytotoxic%2520metabolites%2520from%2520the%2520marine%2520ascidian%2520Diazona%2520chinensis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1991%26volume%3D113%26spage%3D2303%26epage%3D2304%26doi%3D10.1021%2Fja00006a060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiuchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuda, Y.</span></span> <span> </span><span class="NLM_article-title">Martefragin A, a novel indole alkaloid isolated from red alga, inhibits lipid peroxidation</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1527</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1248/cpb.46.1527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1248%2Fcpb.46.1527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=9810689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK1cXntVSltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1998&pages=1527-1529&author=S.+Takahashiauthor=T.+Matsunagaauthor=C.+Hasegawaauthor=H.+Saitoauthor=D.+Fujitaauthor=F.+Kiuchiauthor=Y.+Tsuda&title=Martefragin+A%2C+a+novel+indole+alkaloid+isolated+from+red+alga%2C+inhibits+lipid+peroxidation&doi=10.1248%2Fcpb.46.1527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Martefragin A, a novel indole alkaloid isolated from red alga, inhibits lipid peroxidation</span></div><div class="casAuthors">Takahashi, Satoshi; Matsunaga, Takayuki; Hasegawa, Chika; Saito, Haruo; Fujita, Daisuke; Kiuchi, Fumiyuki; Tsuda, Yoshisuke</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1527-1529</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Martefragin A (I), a novel indole alkaloid, was isolated from a red alga, Martensia fragilis, by repeated column chromatog.  The structure of I was elucidated on the basis of spectral anal. of its Me ester, including 1H- and 13C-NMR, 1H-1H correlation spectroscopy (COSY), and 13C-1H COSY.  A single crystal X-ray anal. of the hydrochloride of I confirmed the assignment.  Martefragin A (I) showed inhibitory activity on NADPH-dependent lipid peroxidn. in rat liver microsomes.  The IC50 values of I, α-tocopherol and ascorbic acid were 2.8, 87 and 200 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq25gztulBuw7Vg90H21EOLACvtfcHk0lis6-FlOGrCeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntVSltrk%253D&md5=af783197b8a39e5774c0e3391709c518</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1248%2Fcpb.46.1527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.46.1527%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DT.%26aulast%3DHasegawa%26aufirst%3DC.%26aulast%3DSaito%26aufirst%3DH.%26aulast%3DFujita%26aufirst%3DD.%26aulast%3DKiuchi%26aufirst%3DF.%26aulast%3DTsuda%26aufirst%3DY.%26atitle%3DMartefragin%2520A%252C%2520a%2520novel%2520indole%2520alkaloid%2520isolated%2520from%2520red%2520alga%252C%2520inhibits%2520lipid%2520peroxidation%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1998%26volume%3D46%26spage%3D1527%26epage%3D1529%26doi%3D10.1248%2Fcpb.46.1527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herald, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. M.</span></span> <span> </span><span class="NLM_article-title">Isolation of Labradorins 1 and 2 from Pseudomonas syringae pv. coronafaciens</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1797</span>, <span class="refDoi"> DOI: 10.1021/np020173x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np020173x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvFCntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2002&pages=1793-1797&author=G.+R.+Pettitauthor=J.+C.+Knightauthor=D.+L.+Heraldauthor=R.+Davenportauthor=R.+K.+Pettitauthor=B.+E.+Tuckerauthor=J.+M.+Schmidt&title=Isolation+of+Labradorins+1+and+2+from+Pseudomonas+syringae+pv.+coronafaciens&doi=10.1021%2Fnp020173x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of labradorins 1 and 2 from Pseudomonas syringae pv. coronafaciens</span></div><div class="casAuthors">Pettit, George R.; Knight, John C.; Herald, Delbert L.; Davenport, Richard; Pettit, Robin K.; Tucker, Bruce E.; Schmidt, Jean M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1793-1797</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Investigation of Pseudomonas syringae pv. coronafaciens cancer cell growth inhibitory constituents led to the isolation of 2-isobutyl-5-(3-indolyl)oxazole (I) and 2-n-pentyl-5-(3-indolyl)oxazole (II), designated labradorins 1 and 2, resp., related to pimprinine (III).  The structures were deduced by spectroscopic techniques and X-ray crystal structure detns.  I afforded GI50 μg/mL values of 9.8 and 6.2 against the human cancer cell lines NCI-H 460 (lung-NSC) and BXPC-3 (pancreas-a).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmmEqRGEdUcbVg90H21EOLACvtfcHk0lipNbjPWHPHHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvFCntbg%253D&md5=c532e55f7a6bb01847729726564770ca</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fnp020173x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp020173x%26sid%3Dliteratum%253Aachs%26aulast%3DPettit%26aufirst%3DG.%2BR.%26aulast%3DKnight%26aufirst%3DJ.%2BC.%26aulast%3DHerald%26aufirst%3DD.%2BL.%26aulast%3DDavenport%26aufirst%3DR.%26aulast%3DPettit%26aufirst%3DR.%2BK.%26aulast%3DTucker%26aufirst%3DB.%2BE.%26aulast%3DSchmidt%26aufirst%3DJ.%2BM.%26atitle%3DIsolation%2520of%2520Labradorins%25201%2520and%25202%2520from%2520Pseudomonas%2520syringae%2520pv.%2520coronafaciens%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2002%26volume%3D65%26spage%3D1793%26epage%3D1797%26doi%3D10.1021%2Fnp020173x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span> <i>Total synthesis of CPI-2081, breitfussin B and synthetic studies towards myriastramide C and goadsporin</i>; <span class="NLM_publisher-name">University of East Anglia</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Liu&title=Total+synthesis+of+CPI-2081%2C+breitfussin+B+and+synthetic+studies+towards+myriastramide+C+and+goadsporin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26btitle%3DTotal%2520synthesis%2520of%2520CPI-2081%252C%2520breitfussin%2520B%2520and%2520synthetic%2520studies%2520towards%2520myriastramide%2520C%2520and%2520goadsporin%26pub%3DUniversity%2520of%2520East%2520Anglia%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of breitfussin B by late-stage bromination</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3718</span>– <span class="NLM_lpage">3721</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01702</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01702" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2ltr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=3718-3721&author=A.+H.+Khanauthor=J.+S.+Chen&title=Synthesis+of+breitfussin+B+by+late-stage+bromination&doi=10.1021%2Facs.orglett.5b01702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Breitfussin B by Late-Stage Bromination</span></div><div class="casAuthors">Khan, Akbar H.; Chen, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3718-3721</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The breitfussins are halogenated natural products (also considered as highly oxidized dipeptides) whose structures were detd. with the assistance of at.-force microscopy.  The site selectivity of N-bromosuccinimide-mediated bromination of a model breitfussin core was found to be strongly dependent on solvent selection; use of acetone led to oxazole bromination, and use of a pyridine-contg. mixt. led to pyrrole bromination.  This tunable site-selective bromination was used in a protecting group-free synthesis of breitfussin B that proceeded in 9.2% yield over 12 reactions and five chromatog. sepns.  A bromooxazole analog of breitfussin A was also prepd. by late-stage bromination but isomerized on silica gel to form breitfussin B.  This isomerization appeared to proceed through a unimol. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0ExPvuykpHLVg90H21EOLACvtfcHk0lipNbjPWHPHHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2ltr7E&md5=80715e9d80090de9cea0e4e72263e397</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01702%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DA.%2BH.%26aulast%3DChen%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520breitfussin%2520B%2520by%2520late-stage%2520bromination%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D3718%26epage%3D3721%26doi%3D10.1021%2Facs.orglett.5b01702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cambeiro, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlsten, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrosa, I.</span></span> <span> </span><span class="NLM_article-title">Au-catalyzed cross-coupling of arenes via double C–H activation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">15636</span>– <span class="NLM_lpage">15639</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b10593</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b10593" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylur7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=15636-15639&author=X.+C.+Cambeiroauthor=N.+Ahlstenauthor=I.+Larrosa&title=Au-catalyzed+cross-coupling+of+arenes+via+double+C%E2%80%93H+activation&doi=10.1021%2Fjacs.5b10593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Au-Catalyzed Cross-Coupling of Arenes via Double C-H Activation</span></div><div class="casAuthors">Cambeiro, Xacobe C.; Ahlsten, Nanna; Larrosa, Igor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15636-15639</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first methodol. for Au(I/III)-catalyzed oxidative cross-coupling of arenes via double C-H activation has been developed.  The reaction is fully selective for the cross-coupling between electron-rich hetero-/ carbocyclic arenes and electron-poor arenes bearing relatively acidic C-H bonds.  The inherently high cross-selectivity of the system obviates the need for directing groups or a large excess of one of the coupling partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFaWknTtjBJbVg90H21EOLACvtfcHk0lipNbjPWHPHHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylur7L&md5=1e18a4db9bcf4491a6d6d09a9fc449eb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b10593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b10593%26sid%3Dliteratum%253Aachs%26aulast%3DCambeiro%26aufirst%3DX.%2BC.%26aulast%3DAhlsten%26aufirst%3DN.%26aulast%3DLarrosa%26aufirst%3DI.%26atitle%3DAu-catalyzed%2520cross-coupling%2520of%2520arenes%2520via%2520double%2520C%25E2%2580%2593H%2520activation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D15636%26epage%3D15639%26doi%3D10.1021%2Fjacs.5b10593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelkey, E. T.</span></span> <span> </span><span class="NLM_article-title">Metalation of Indole</span>. In  <i>Heterocyclic Scaffolds II: Reactions and Applications of Indoles</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribble, G. W.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">191</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1007%2F7081_2010_56" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=141-191&author=E.+T.+Pelkeyauthor=G.+W.+Gribble&title=Heterocyclic+Scaffolds+II%3A+Reactions+and+Applications+of+Indoles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2F7081_2010_56&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F7081_2010_56%26sid%3Dliteratum%253Aachs%26aulast%3DPelkey%26aufirst%3DE.%2BT.%26atitle%3DMetalation%2520of%2520Indole%26btitle%3DHeterocyclic%2520Scaffolds%2520II%253A%2520Reactions%2520and%2520Applications%2520of%2520Indoles%26aulast%3DGribble%26aufirst%3DG.%2BW.%26pub%3DSpringer%26date%3D2010%26spage%3D141%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parton, R. L.</span></span> <span> </span><span class="NLM_article-title">Lithiation of methoxyindoles</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1021/jo00863a044</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00863a044" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaE28XktlOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1976&pages=163-165&author=R.+J.+Sundbergauthor=R.+L.+Parton&title=Lithiation+of+methoxyindoles&doi=10.1021%2Fjo00863a044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lithiation of methoxyindoles</span></div><div class="casAuthors">Sundberg, Richard J.; Parton, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">163-5</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">1-Methyl-5-methoxyindole was lithiated with BuLi and Me3CLi.  The indole ring is lithiated at C-2 by Me3CLi but lithiation with BuLi resulted in lithiation at C-2, C-4, and C-6.  Lithiation of 1-benzenesulfonyl-5-methoxyindole and 1-benzenesulfonyl-6-methoxyindole with Me3CLi occurred selectively at C-2.  The various lithio indoles form the expected adducts with carbonyl compds. such as 2-pyridinecarboxaldehyde, 3-pyridinecarboxaldehyde, and 4-acetylpyridine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp91r3rO5O8irVg90H21EOLACvtfcHk0lgXe8l0EJMz_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XktlOmsg%253D%253D&md5=77227dba0ba1018a2b22c64cab24118e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjo00863a044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00863a044%26sid%3Dliteratum%253Aachs%26aulast%3DSundberg%26aufirst%3DR.%2BJ.%26aulast%3DParton%26aufirst%3DR.%2BL.%26atitle%3DLithiation%2520of%2520methoxyindoles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1976%26volume%3D41%26spage%3D163%26epage%3D165%26doi%3D10.1021%2Fjo00863a044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snieckus, V.</span></span> <span> </span><span class="NLM_article-title">Directed ortho metalation. Tertiary amide and O-carbamate directors in synthetic strategies for polysubstituted aromatics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">879</span>– <span class="NLM_lpage">933</span>, <span class="refDoi"> DOI: 10.1021/cr00104a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00104a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK3cXlsFWqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1990&pages=879-933&author=V.+Snieckus&title=Directed+ortho+metalation.+Tertiary+amide+and+O-carbamate+directors+in+synthetic+strategies+for+polysubstituted+aromatics&doi=10.1021%2Fcr00104a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Directed ortho metalation.  Tertiary amide and O-carbamate directors in synthetic strategies for polysubstituted aromatics</span></div><div class="casAuthors">Snieckus, Victor</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">879-933</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 214 refs.  After a brief overview of general aspects of the directed ortho metalation reaction, the utility and applications of tertiary amide and carbamate directed metalation groups are systematically and comprehensively developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpthWLi2I-h8bVg90H21EOLACvtfcHk0lgXe8l0EJMz_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlsFWqsrc%253D&md5=21de0a0b1b08e9241ec117dde47d9aa1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcr00104a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00104a001%26sid%3Dliteratum%253Aachs%26aulast%3DSnieckus%26aufirst%3DV.%26atitle%3DDirected%2520ortho%2520metalation.%2520Tertiary%2520amide%2520and%2520O-carbamate%2520directors%2520in%2520synthetic%2520strategies%2520for%2520polysubstituted%2520aromatics%26jtitle%3DChem.%2520Rev.%26date%3D1990%26volume%3D90%26spage%3D879%26epage%3D933%26doi%3D10.1021%2Fcr00104a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nirogi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwarampudi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kota, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubey, P.K.</span></span> <span> </span><span class="NLM_article-title">Synthesis and structure activity relationship of rigidized indolyl pyrrolidine derivatives as 5-HT6 receptor ligands</span>. <i>Asian J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">9293</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.14233/ajchem.2013.15443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.14233%2Fajchem.2013.15443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOkurfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2013&pages=9293-9298&author=R.+Nirogiauthor=A.+Dwarampudiauthor=V.+Bhattaauthor=L.+Kotaauthor=P.K.+Dubey&title=Synthesis+and+structure+activity+relationship+of+rigidized+indolyl+pyrrolidine+derivatives+as+5-HT6+receptor+ligands&doi=10.14233%2Fajchem.2013.15443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and structure activity relationship of rigidized indolyl pyrrolidine derivatives as 5-HT6 receptor ligands</span></div><div class="casAuthors">Nirogi, Ramakrishna; Dwarampudi, Adireddy; Bhatta, Venugopalarao; Kota, Laxman; Dubey, P. K.</div><div class="citationInfo"><span class="NLM_cas:title">Asian Journal of Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9293-9298</span>CODEN:
                <span class="NLM_cas:coden">AJCHEW</span>;
        ISSN:<span class="NLM_cas:issn">0970-7077</span>.
    
            (<span class="NLM_cas:orgname">Asian Journal of Chemistry</span>)
        </div><div class="casAbstract">A novel series of rigidized indolyl pyrrolidine derivs. were designed by constraining the tryptamine nitrogen through α and β carbons.  All the synthesized derivs. showed moderate affinities at 5-HT6R when tested in in vitro binding assay.  Synthesis, structure activity relation (SAR), pharmacokinetic profile and in vivo efficacy of a selected compd. is the subject matter of this communication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquuu0k6niSTbVg90H21EOLACvtfcHk0lgXe8l0EJMz_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOkurfL&md5=bfd5c8e436f51cfe195e95a30a8ed684</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.14233%2Fajchem.2013.15443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14233%252Fajchem.2013.15443%26sid%3Dliteratum%253Aachs%26aulast%3DNirogi%26aufirst%3DR.%26aulast%3DDwarampudi%26aufirst%3DA.%26aulast%3DBhatta%26aufirst%3DV.%26aulast%3DKota%26aufirst%3DL.%26aulast%3DDubey%26aufirst%3DP.K.%26atitle%3DSynthesis%2520and%2520structure%2520activity%2520relationship%2520of%2520rigidized%2520indolyl%2520pyrrolidine%2520derivatives%2520as%25205-HT6%2520receptor%2520ligands%26jtitle%3DAsian%2520J.%2520Chem.%26date%3D2013%26volume%3D25%26spage%3D9293%26epage%3D9298%26doi%3D10.14233%2Fajchem.2013.15443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snieckus, V.</span></span> <span> </span><span class="NLM_article-title">Benzenoid ring functionalization of indoles and tryptophols via combined directed ortho metalation-cross coupling methodology</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1484</span>– <span class="NLM_lpage">1485</span>, <span class="refDoi"> DOI: 10.1021/jo00111a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00111a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK2MXktlektro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1995&pages=1484-1485&author=E.+J.+Griffenauthor=D.+G.+Roeauthor=V.+Snieckus&title=Benzenoid+ring+functionalization+of+indoles+and+tryptophols+via+combined+directed+ortho+metalation-cross+coupling+methodology&doi=10.1021%2Fjo00111a003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Benzenoid Ring Functionalization of Indoles and Tryptophols via Combined Directed Ortho Metalation-Cross Coupling Methodology</span></div><div class="casAuthors">Griffen, Edward J.; Roe, David G.; Snieckus, Victor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1484-5</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalation (s-BuLi/TMEDA/THF/-78°) of N,N-diethyl-5-(1-tert-butyldimethylsilylindolyl)carbamate followed by treatment with electrophiles (E = MeI, DMF, cyclohexanone, EtOCOCl, TsN3, camphorsulfonyloxaziridine, C2Cl6, I2, PhSO2SBu-t, and TMSCl) provides a general synthesis of 4-substituted products.  Further metalation of N,N-diethyl-5-(1-tert-butyldimethylsilyl-4-trimethylsilylindolyl)carbamate under the same conditions but -90° followed by electrophile quench leads to 6-substituted products and by anionic carbamoyl rearrangement to 5-acetoxy-1-tert-butyldimethylsilyl-6-N,N-diethylcarboxamido-4-trimethylsilylindole.  Transmetalation of N,N-diethyl-5-(1-tert-butyldimethylsilyl-4-lithioindolyl)carbamate into the corresponding zinc species, followed by cross coupling with 3-bromopyridine under Pd(0) catalysis leads to N,N-diethyl-5-[1-tert-butyldimethylsilyl-4-(3-pyridyl)indolyl]carbamate.  Cross coupling of N,N-diethyl-5-(1-tert-butyldimethylsilylindolyl)carbamate with Grignard reagents (PhMgX and TMSCH2MgX) under Ni(0) catalysis leads to 5-substituted products.  Tryptophol 5-O-carbamates [N,N-diethyl-5-[1-tert-butyldimethylsilyl-3-(2-hydroxyethyl)indolyl]carbamate and N,N-diethyl-5-[1-tert-butyldimethylsilyl-3-(2-tert-butyldimethylsilyloxyethyl)indolyl]carbamate], prepd. via N,N-diethyl-5-(1-tert-butyldimethylsilyl-3-bromoindolyl)carbamate, when metalated under the same conditions followed by treatment with DMF, lead to the corresponding 6-formylated products.  Decarbamoylation of N,N-diethyl-5-(1-tert-butyldimethylsilylindolyl)carbamate to the corresponding phenol and its conversion into 5-methoxy and 5-triflate derivs. was also carried out.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvh46BQ2R-DbVg90H21EOLACvtfcHk0lhRjnnIwY-9Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXktlektro%253D&md5=903834bb86bb6795382d5ed4f112b867</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjo00111a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00111a003%26sid%3Dliteratum%253Aachs%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26aulast%3DRoe%26aufirst%3DD.%2BG.%26aulast%3DSnieckus%26aufirst%3DV.%26atitle%3DBenzenoid%2520ring%2520functionalization%2520of%2520indoles%2520and%2520tryptophols%2520via%2520combined%2520directed%2520ortho%2520metalation-cross%2520coupling%2520methodology%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D1484%26epage%3D1485%26doi%3D10.1021%2Fjo00111a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riss, T. L.</span>; <span class="NLM_string-name">Moravec, R. A.</span>; <span class="NLM_string-name">Niles, A. L.</span>; <span class="NLM_string-name">Duellman, S.</span>; <span class="NLM_string-name">Benink, H. A.</span>; <span class="NLM_string-name">Worzella, T. J.</span>; <span class="NLM_string-name">Minor, L.</span></span> <span> </span><span class="NLM_article-title">Cell viability assays</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=T.+L.+Riss&author=R.+A.+Moravec&author=A.+L.+Niles&author=S.+Duellman&author=H.+A.+Benink&author=T.+J.+Worzella&author=L.+Minor&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DRiss%26aufirst%3DT.%2BL.%26atitle%3DCell%2520viability%2520assays%26btitle%3DAssay%2520Guidance%2520Manual%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hastie, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLauchlan, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span> <span> </span><span class="NLM_article-title">Assay of protein kinases using radiolabeled ATP: a protocol</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">968</span>– <span class="NLM_lpage">971</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnprot.2006.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=17406331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=968-971&author=C.+J.+Hastieauthor=H.+J.+McLauchlanauthor=P.+Cohen&title=Assay+of+protein+kinases+using+radiolabeled+ATP%3A+a+protocol&doi=10.1038%2Fnprot.2006.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Assay of protein kinases using radiolabeled ATP: a protocol</span></div><div class="casAuthors">Hastie, C. James; McLauchlan, Hilary J.; Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">968-971</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein kinase activity results in the incorporation of radiolabeled phosphate from [γ-32P]ATP into a peptide or protein substrate.  The measurement of the amt. of radioactivity incorporated into a substrate as a function of time and enzyme concn. allows enzyme activity to be quantified.  The activity is expressed as a 'unit', where 1 unit corresponds to the amt. of protein kinase that catalyzes the incorporation of 1 nmol of phosphate into the std. substrate in 1 min.  Specific activity is defined as units of activity per mg protein.  The assay format described here is quick, simple, inexpensive, sensitive and accurate, provides a direct measurement of activity and remains the 'gold std.' for the quantification of protein kinase activity.  Up to 40 samples can be assayed manually at one time, and the assay takes one person less than 1 h to complete.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFfHCuf9HbVLVg90H21EOLACvtfcHk0lhRjnnIwY-9Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitb%252FM&md5=d6e443498bece9b9b6dc1c3a8a29580c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.149%26sid%3Dliteratum%253Aachs%26aulast%3DHastie%26aufirst%3DC.%2BJ.%26aulast%3DMcLauchlan%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DAssay%2520of%2520protein%2520kinases%2520using%2520radiolabeled%2520ATP%253A%2520a%2520protocol%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D968%26epage%3D971%26doi%3D10.1038%2Fnprot.2006.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graczyk, P. P.</span></span> <span> </span><span class="NLM_article-title">Gini Coefficient: A new way to express selectivity of kinase inhibitors against a family of kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5773</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/jm070562u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070562u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5773-5779&author=P.+P.+Graczyk&title=Gini+Coefficient%3A+A+new+way+to+express+selectivity+of+kinase+inhibitors+against+a+family+of+kinases&doi=10.1021%2Fjm070562u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Gini Coefficient: A New Way To Express Selectivity of Kinase Inhibitors against a Family of Kinases</span></div><div class="casAuthors">Graczyk, Piotr P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5773-5779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel application of the Gini coeff. for expressing selectivity of kinase inhibitors against a panel of kinases is proposed.  This has been illustrated using single-point inhibition data for 40 com. available kinase inhibitors screened against 85 kinases.  Nonselective inhibitors are characterized by Gini values close to zero (Staurosporine, Gini 0.150).  Highly selective compds. exhibit Gini values close to 1 (PD184352 Gini 0.905).  The relative selectivity of inhibitors does not depend on the ATP concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqUHpmF-C9u7Vg90H21EOLACvtfcHk0lhRjnnIwY-9Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL&md5=51f4902e49196dfef8e135c4a09182d7</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm070562u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070562u%26sid%3Dliteratum%253Aachs%26aulast%3DGraczyk%26aufirst%3DP.%2BP.%26atitle%3DGini%2520Coefficient%253A%2520A%2520new%2520way%2520to%2520express%2520selectivity%2520of%2520kinase%2520inhibitors%2520against%2520a%2520family%2520of%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5773%26epage%3D5779%26doi%3D10.1021%2Fjm070562u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uitdehaag, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, G. J.</span></span> <span> </span><span class="NLM_article-title">A theoretical entropy score as a single value to express inhibitor selectivity</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">94</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-12-94</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2F1471-2105-12-94" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21486481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltV2htb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=94&author=J.+C.+Uitdehaagauthor=G.+J.+Zaman&title=A+theoretical+entropy+score+as+a+single+value+to+express+inhibitor+selectivity&doi=10.1186%2F1471-2105-12-94"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A theoretical entropy score as a single value to express inhibitor selectivity</span></div><div class="casAuthors">Uitdehaag, Joost C. M.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">94</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Designing maximally selective ligands that act on individual targets is the dominant paradigm in drug discovery.  Poor selectivity can underlie toxicity and side effects in the clinic, and for this reason compd. selectivity is increasingly monitored from very early on in the drug discovery process.  To make sense of large amts. of profiling data, and to det. when a compd. is sufficiently selective, there is a need for a proper quant. measure of selectivity.  Here we propose a new theor. entropy score that can be calcd. from a set of IC50 data.  In contrast to previous measures such as the 'selectivity score', Gini score, or partition index, the entropy score is non-arbitary, fully exploits IC50 data, and is not dependent on a ref. kinase.  In addn., the entropy score gives the most robust values with data from different sources, because it is less sensitive to errors.  We apply the new score to kinase and nuclear receptor profiling data, and to high-throughput screening data.  In addn., through analyzing profiles of clin. compds., we show that a more selective inhibitor is not necessarily more drug-like.  For quantifying selectivity from panel profiling, a theor. entropy score is the best method.  It is valuable for studying the mol. mechanisms of selectivity, and to steer compd. progression in drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2tB5EtrxkGbVg90H21EOLACvtfcHk0lgx0u01OA2cBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltV2htb8%253D&md5=a057b5f0222693138d3942cb439aa39c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-94&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-94%26sid%3Dliteratum%253Aachs%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%26atitle%3DA%2520theoretical%2520entropy%2520score%2520as%2520a%2520single%2520value%2520to%2520express%2520inhibitor%2520selectivity%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26spage%3D94%26doi%3D10.1186%2F1471-2105-12-94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>, <span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lgx0u01OA2cBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saurabh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherzer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mims, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockwood, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beverly, L. J.</span></span> <span> </span><span class="NLM_article-title">The PIM family of oncoproteins: small kinases with huge implications in myeloid leukemogenesis and as therapeutic targets</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">8503</span>– <span class="NLM_lpage">8514</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.18632%2Foncotarget.2330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=25238262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC2M7jtVKitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=8503-8514&author=K.+Saurabhauthor=M.+T.+Scherzerauthor=P.+P.+Shahauthor=A.+S.+Mimsauthor=W.+W.+Lockwoodauthor=A.+S.+Kraftauthor=L.+J.+Beverly&title=The+PIM+family+of+oncoproteins%3A+small+kinases+with+huge+implications+in+myeloid+leukemogenesis+and+as+therapeutic+targets&doi=10.18632%2Foncotarget.2330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The PIM family of oncoproteins: small kinases with huge implications in myeloid leukemogenesis and as therapeutic targets</span></div><div class="casAuthors">Saurabh Kumar; Scherzer Michael T; Shah Parag P; Mims Alice S; Kraft Andrew S; Lockwood William W; Beverly Levi J</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8503-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PIM kinases are a family of serine/threonine kinases involved in cell survival and proliferation.  There is significant structural similarity between the three PIM kinases (PIM1, PIM2 and PIM3) and only few amino acid differences.  Although, several studies have specifically monitored the role of PIM1 in tumorigenesis, much less is known about PIM2 and PIM3.  Therefore, in this study we have used in vitro cell culture models and in vivo bone marrow infection/transplantation to assess the comparative signaling and oncogenic potential of each of the three PIM kinases.  All three PIM kinases were able to protect FL5.12 cells from IL3 withdrawal induced death.  Interestingly, the downstream signaling cascades were indistinguishable between the three kinases.  Transplantation of murine bone marrow co-expressing MYC and PIM1, PIM2 or PIM3 caused rapid and uniformly lethal myeloid leukemia.  De-induction of MYC 18 days following transplantation significantly increased the survival of mice, even with continual expression of PIM kinases.  Alternatively, mice treated at the pre-leukemic stage with a PIM kinase inhibitor increased the lifespan of the mice, even with continual expression of the MYC transgene.  These data demonstrate the role of PIM kinases in driving myeloid leukemia, and as candidate molecules for therapy against human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRe0My0fF0UK5p6t8nS6Nc3fW6udTcc2eZeqji9GXDk_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7jtVKitg%253D%253D&md5=a24cfd40890d5ff365cc84d0492e393c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2330%26sid%3Dliteratum%253Aachs%26aulast%3DSaurabh%26aufirst%3DK.%26aulast%3DScherzer%26aufirst%3DM.%2BT.%26aulast%3DShah%26aufirst%3DP.%2BP.%26aulast%3DMims%26aufirst%3DA.%2BS.%26aulast%3DLockwood%26aufirst%3DW.%2BW.%26aulast%3DKraft%26aufirst%3DA.%2BS.%26aulast%3DBeverly%26aufirst%3DL.%2BJ.%26atitle%3DThe%2520PIM%2520family%2520of%2520oncoproteins%253A%2520small%2520kinases%2520with%2520huge%2520implications%2520in%2520myeloid%2520leukemogenesis%2520and%2520as%2520therapeutic%2520targets%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D8503%26epage%3D8514%26doi%3D10.18632%2Foncotarget.2330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willems, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedobbeleer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digregorio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumapat, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogister, B.</span></span> <span> </span><span class="NLM_article-title">The functional diversity of Aurora kinases: a comprehensive review</span>. <i>Cell Div.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">7</span>, <span class="refDoi"> DOI: 10.1186/s13008-018-0040-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2Fs13008-018-0040-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=30250494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvVent7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=7&author=E.+Willemsauthor=M.+Dedobbeleerauthor=M.+Digregorioauthor=A.+Lombardauthor=P.+N.+Lumapatauthor=B.+Rogister&title=The+functional+diversity+of+Aurora+kinases%3A+a+comprehensive+review&doi=10.1186%2Fs13008-018-0040-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The functional diversity of Aurora kinases: a comprehensive review</span></div><div class="casAuthors">Willems, Estelle; Dedobbeleer, Matthias; Digregorio, Marina; Lombard, Arnaud; Lumapat, Paul Noel; Rogister, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Cell Division</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7/1-7/17</span>CODEN:
                <span class="NLM_cas:coden">CDEIAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-1028</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Aurora kinases are serine/threonine kinases essential for the onset and progression of mitosis.  Aurora members share a similar protein structure and kinase activity, but exhibit distinct cellular and subcellular localization.  AurA favors the G2/M transition by promoting centrosome maturation and mitotic spindle assembly.  AurB and AurC are chromosome-passenger complex proteins, crucial for chromosome binding to kinetochores and segregation of chromosomes.  Cellular distribution of AurB is ubiquitous, while AurC expression is mainly restricted to meiotically-active germ cells.  In human tumors, all Aurora kinase members play oncogenic roles related to their mitotic activity and promote cancer cell survival and proliferation.  Furthermore, AurA plays tumor-promoting roles unrelated to mitosis, including tumor stemness, epithelial-to-mesenchymal transition and invasion.  In this review, we aim to understand the functional interplay of Aurora kinases in various types of human cells, including tumor cells.  The understanding of the functional diversity of Aurora kinases could help to evaluate their relevance as potential therapeutic targets in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvSNgg8ZBDULVg90H21EOLACvtfcHk0lje-E-7m3t7YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvVent7g%253D&md5=ac0d2d15b2e29ce4d96ef7f8eebfa4a7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1186%2Fs13008-018-0040-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13008-018-0040-6%26sid%3Dliteratum%253Aachs%26aulast%3DWillems%26aufirst%3DE.%26aulast%3DDedobbeleer%26aufirst%3DM.%26aulast%3DDigregorio%26aufirst%3DM.%26aulast%3DLombard%26aufirst%3DA.%26aulast%3DLumapat%26aufirst%3DP.%2BN.%26aulast%3DRogister%26aufirst%3DB.%26atitle%3DThe%2520functional%2520diversity%2520of%2520Aurora%2520kinases%253A%2520a%2520comprehensive%2520review%26jtitle%3DCell%2520Div.%26date%3D2018%26volume%3D13%26spage%3D7%26doi%3D10.1186%2Fs13008-018-0040-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelias, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lje-E-7m3t7YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miduturu, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brault, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">High-throughput kinase profiling: A more efficient approach towards the discovery of new kinase inhibitors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.chembiol.2011.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21802008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFarurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=868-879&author=C.+V.+Miduturuauthor=X.+Dengauthor=N.+Kwiatkowskiauthor=W.+Yangauthor=L.+Braultauthor=P.+Filippakopoulosauthor=E.+Chungauthor=Q.+Yangauthor=J.+Schwallerauthor=S.+Knappauthor=R.+W.+Kingauthor=J.-D.+Leeauthor=S.+Herrgardauthor=P.+Zarrinkarauthor=N.+S.+Gray&title=High-throughput+kinase+profiling%3A+A+more+efficient+approach+towards+the+discovery+of+new+kinase+inhibitors&doi=10.1016%2Fj.chembiol.2011.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Kinase Profiling: A More Efficient Approach toward the Discovery of New Kinase Inhibitors</span></div><div class="casAuthors">Miduturu, Chandrasekhar V.; Deng, Xianming; Kwiatkowski, Nicholas; Yang, Wannian; Brault, Laurent; Filippakopoulos, Panagis; Chung, Eunah; Yang, Qingkai; Schwaller, Juerg; Knapp, Stefan; King, Randall W.; Lee, Jiing-Dwan; Herrgard, Sanna; Zarrinkar, Patrick; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">868-879</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Selective protein kinase inhibitors have only been developed against a small no. of kinase targets.  Here we demonstrate that "high-throughput kinase profiling" is an efficient method for the discovery of lead compds. for established as well as unexplored kinase targets.  We screened a library of 118 compds. constituting two distinct scaffolds (furan-thiazolidinediones and pyrimido-diazepines) against a panel of 353 kinases.  A distinct kinase selectivity profile was obsd. for each scaffold.  Selective inhibitors were identified with submicromolar cellular activity against PIM1, ERK5, ACK1, MPS1, PLK1-3, and Aurora A,B kinases.  In addn., we identified potent inhibitors for so far unexplored kinases such as DRAK1, HIPK2, and DCAMKL1 that await further evaluation.  This inhibitor-centric approach permits comprehensive assessment of a scaffold of interest and represents an efficient and general strategy for identifying new selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKLvN9eLNvybVg90H21EOLACvtfcHk0lje-E-7m3t7YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFarurw%253D&md5=d62a4771b1215551f3913b93d6eca7ee</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DMiduturu%26aufirst%3DC.%2BV.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DBrault%26aufirst%3DL.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKing%26aufirst%3DR.%2BW.%26aulast%3DLee%26aufirst%3DJ.-D.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DZarrinkar%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DHigh-throughput%2520kinase%2520profiling%253A%2520A%2520more%2520efficient%2520approach%2520towards%2520the%2520discovery%2520of%2520new%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D868%26epage%3D879%26doi%3D10.1016%2Fj.chembiol.2011.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skovgaard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthelsen, J.</span></span> <span> </span><span class="NLM_article-title">A comparison of protein kinases inhibitor screening methods using both enzymatic activity and binding affinity determination</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e98800</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0098800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1371%2Fjournal.pone.0098800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=24915177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1artbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=A.+F.+Rudolfauthor=T.+Skovgaardauthor=S.+Knappauthor=L.+J.+Jensenauthor=J.+Berthelsen&title=A+comparison+of+protein+kinases+inhibitor+screening+methods+using+both+enzymatic+activity+and+binding+affinity+determination&doi=10.1371%2Fjournal.pone.0098800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A comparison of protein kinases inhibitor screening methods using both enzymatic activity and binding affinity determination</span></div><div class="casAuthors">Rudolf, Amalie Frederikke; Skovgaard, Tine; Knapp, Stefan; Juhl Jensen, Lars; Berthelsen, Jens</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e98800/1-e98800/5, 5 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Binding assays are increasingly used as a screening method for protein kinase inhibitors; however, as yet only a weak correlation with enzymic activity-based assays has been demonstrated.  We show that the correlation between the two types of assays can be improved using more precise screening conditions.  Furthermore a marked improvement in the correlation was found by using kinase constructs contg. the catalytic domain in presence of addnl. domains or subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ6qupl5aTgrVg90H21EOLACvtfcHk0lg0WKtZ9oYW7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1artbrO&md5=c79d0fbd01aee6442a6223c2e6eb24f1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098800%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DA.%2BF.%26aulast%3DSkovgaard%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DBerthelsen%26aufirst%3DJ.%26atitle%3DA%2520comparison%2520of%2520protein%2520kinases%2520inhibitor%2520screening%2520methods%2520using%2520both%2520enzymatic%2520activity%2520and%2520binding%2520affinity%2520determination%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0098800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brasó-Maristany, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filosto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catchpole, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francesch-Domenech, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazinska, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liccardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gris-Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheang, M. C. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdrix-Rosell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noël, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaltriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigoriadis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A. N.</span></span> <span> </span><span class="NLM_article-title">PIM1 kinase regulates cell death, tumor growth and chemotherapy response revealing a novel target in triple-negative breast cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1303</span>– <span class="NLM_lpage">1313</span>, <span class="refDoi"> DOI: 10.1038/nm.4198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnm.4198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=27775704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslehur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=1303-1313&author=F.+Bras%C3%B3-Maristanyauthor=S.+Filostoauthor=S.+Catchpoleauthor=R.+Marlowauthor=J.+Quistauthor=E.+Francesch-Domenechauthor=D.+A.+Plumbauthor=L.+Zakkaauthor=P.+Gazinskaauthor=G.+Liccardiauthor=P.+Meierauthor=A.+Gris-Oliverauthor=M.+C.+U.+Cheangauthor=A.+Perdrix-Rosellauthor=M.+Shafatauthor=E.+No%C3%ABlauthor=N.+Patelauthor=K.+McEachernauthor=M.+Scaltritiauthor=P.+Castelauthor=F.+Noorauthor=R.+Buusauthor=S.+Mathewauthor=J.+Watkinsauthor=V.+Serraauthor=P.+Marraauthor=A.+Grigoriadisauthor=A.+N.+Tutt&title=PIM1+kinase+regulates+cell+death%2C+tumor+growth+and+chemotherapy+response+revealing+a+novel+target+in+triple-negative+breast+cancer&doi=10.1038%2Fnm.4198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PIM1 kinase regulates cell death, tumor growth and chemotherapy response in triple-negative breast cancer</span></div><div class="casAuthors">Braso-Maristany, Fara; Filosto, Simone; Catchpole, Steven; Marlow, Rebecca; Quist, Jelmar; Francesch-Domenech, Erika; Plumb, Darren A.; Zakka, Leila; Gazinska, Patrycja; Liccardi, Gianmaria; Meier, Pascal; Gris-Oliver, Albert; Cheang, Maggie Chon U.; Perdrix-Rosell, Anna; Shafat, Manar; Noel, Elodie; Patel, Nirmesh; McEachern, Kristen; Scaltriti, Maurizio; Castel, Pau; Noor, Farzana; Buus, Richard; Mathew, Sumi; Watkins, Johnathan; Serra, Violeta; Marra, Pierfrancesco; Grigoriadis, Anita; Tutt, Andrew N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1303-1313</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBCs) have poor prognosis and lack targeted therapies.  Here we identified increased copy no. and expression of the PIM1 proto-oncogene in genomic data sets of patients with TNBC.  TNBC cells, but not nonmalignant mammary epithelial cells, were dependent on PIM1 for proliferation and protection from apoptosis.  PIM1 knockdown reduced expression of the anti-apoptotic factor BCL2, and dynamic BH3 profiling of apoptotic priming revealed that PIM1 prevents mitochondrial-mediated apoptosis in TNBC cell lines.  In TNBC tumors and their cellular models, PIM1 expression was assocd. with several transcriptional signatures involving the transcription factor MYC, and PIM1 depletion in TNBC cell lines decreased, in a MYC-dependent manner, cell population growth and expression of the MYC target gene MCL1.  Treatment with the pan-PIM kinase inhibitor AZD1208 impaired the growth of both cell line and patient-derived xenografts and sensitized them to std.-of-care chemotherapy.  This work identifies PIM1 as a malignant-cell-selective target in TNBC and the potential use of PIM1 inhibitors for sensitizing TNBC to chemotherapy-induced apoptotic cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3M8bn0hjO67Vg90H21EOLACvtfcHk0lg0WKtZ9oYW7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslehur3E&md5=02c40bff3f1179dab3705af83f79c1f8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnm.4198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4198%26sid%3Dliteratum%253Aachs%26aulast%3DBras%25C3%25B3-Maristany%26aufirst%3DF.%26aulast%3DFilosto%26aufirst%3DS.%26aulast%3DCatchpole%26aufirst%3DS.%26aulast%3DMarlow%26aufirst%3DR.%26aulast%3DQuist%26aufirst%3DJ.%26aulast%3DFrancesch-Domenech%26aufirst%3DE.%26aulast%3DPlumb%26aufirst%3DD.%2BA.%26aulast%3DZakka%26aufirst%3DL.%26aulast%3DGazinska%26aufirst%3DP.%26aulast%3DLiccardi%26aufirst%3DG.%26aulast%3DMeier%26aufirst%3DP.%26aulast%3DGris-Oliver%26aufirst%3DA.%26aulast%3DCheang%26aufirst%3DM.%2BC.%2BU.%26aulast%3DPerdrix-Rosell%26aufirst%3DA.%26aulast%3DShafat%26aufirst%3DM.%26aulast%3DNo%25C3%25ABl%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DMcEachern%26aufirst%3DK.%26aulast%3DScaltriti%26aufirst%3DM.%26aulast%3DCastel%26aufirst%3DP.%26aulast%3DNoor%26aufirst%3DF.%26aulast%3DBuus%26aufirst%3DR.%26aulast%3DMathew%26aufirst%3DS.%26aulast%3DWatkins%26aufirst%3DJ.%26aulast%3DSerra%26aufirst%3DV.%26aulast%3DMarra%26aufirst%3DP.%26aulast%3DGrigoriadis%26aufirst%3DA.%26aulast%3DTutt%26aufirst%3DA.%2BN.%26atitle%3DPIM1%2520kinase%2520regulates%2520cell%2520death%252C%2520tumor%2520growth%2520and%2520chemotherapy%2520response%2520revealing%2520a%2520novel%2520target%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D1303%26epage%3D1313%26doi%3D10.1038%2Fnm.4198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daver, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenk, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levis, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting FLT3 mutations in AML: review of current knowledge and evidence</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/s41375-018-0357-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fs41375-018-0357-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=30651634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyrsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=299-312&author=N.+Daverauthor=R.+F.+Schlenkauthor=N.+H.+Russellauthor=M.+J.+Levis&title=Targeting+FLT3+mutations+in+AML%3A+review+of+current+knowledge+and+evidence&doi=10.1038%2Fs41375-018-0357-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FLT3 mutations in AML: review of current knowledge and evidence</span></div><div class="casAuthors">Daver, Naval; Schlenk, Richard F.; Russell, Nigel H.; Levis, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-312</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Genomic investigations of acute myeloid leukemia (AML) have demonstrated that several genes are recurrently mutated, leading to new genomic classifications, predictive biomarkers, and new therapeutic targets.  Mutations of the FMS-like tyrosine kinase 3 (FLT3) gene occur in approx. 30% of all AML cases, with the internal tandem duplication (ITD) representing the most common type of FLT3 mutation (FLT3-ITD; approx. 25% of all AML cases).  FLT3-ITD is a common driver mutation that presents with a high leukemic burden and confers a poor prognosis in patients with AML.  The prognostic value of a FLT3 mutation in the tyrosine kinase domain (FLT3-TKD), which has a lower incidence in AML (approx. 7-10% of all cases), is uncertain.  Accumulating evidence demonstrates that FLT3 mutational status evolves throughout the disease continuum.  This so-called clonal evolution, together with the identification of FLT3-ITD as a neg. prognostic marker, serves to highlight the importance of FLT3-ITD testing at diagnosis and again at relapse.  Earlier identification of FLT3 mutations will help provide a better understanding of the patient's disease and enable targeted treatment that may help patients achieve longer and more durable remissions.  First-generation FLT3 inhibitors developed for clin. use are broad-spectrum, multikinase inhibitors; however, next-generation FLT3 inhibitors are more specific, more potent, and have fewer toxicities assocd. with off-target effects.  Primary and secondary acquired resistance to FLT3 inhibitors remains a challenge and provides a rationale for combining FLT3 inhibitors with other therapies, both conventional and investigational.  This review focuses on the pathol. and prognostic role of FLT3 mutations in AML, clin. classification of the disease, recent progress with next-generation FLT3 inhibitors, and mechanisms of resistance to FLT3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjK_p2nZ_9j7Vg90H21EOLACvtfcHk0lj3su9PyAy58A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyrsbbI&md5=25ea007d2f7b670aaddeb02866f15c69</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41375-018-0357-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-018-0357-9%26sid%3Dliteratum%253Aachs%26aulast%3DDaver%26aufirst%3DN.%26aulast%3DSchlenk%26aufirst%3DR.%2BF.%26aulast%3DRussell%26aufirst%3DN.%2BH.%26aulast%3DLevis%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520FLT3%2520mutations%2520in%2520AML%253A%2520review%2520of%2520current%2520knowledge%2520and%2520evidence%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26spage%3D299%26epage%3D312%26doi%3D10.1038%2Fs41375-018-0357-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, N.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ok Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.-M.</span></span> <span> </span><span class="NLM_article-title">Extracellular signal-regulated kinase 8-mediated NF-kappaB activation increases sensitivity of human lung cancer cells to arsenic trioxide</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">49144</span>– <span class="NLM_lpage">49155</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.17100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.18632%2Foncotarget.17100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=28467781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC1crjs1Cjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=49144-49155&author=D.-D.+Wuauthor=A.+Lauauthor=F.-Y.+Yuauthor=N.-L.+Caiauthor=L.-J.+Daiauthor=M.+Ok+Kimauthor=D.-Y.+Jinauthor=Y.-M.+Xu&title=Extracellular+signal-regulated+kinase+8-mediated+NF-kappaB+activation+increases+sensitivity+of+human+lung+cancer+cells+to+arsenic+trioxide&doi=10.18632%2Foncotarget.17100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular signal-regulated kinase 8-mediated NF-κB activation increases sensitivity of human lung cancer cells to arsenic trioxide</span></div><div class="casAuthors">Wu Dan-Dan; Lau Andy T Y; Yu Fei-Yuan; Cai Na-Li; Dai Li-Juan; Xu Yan-Ming; Ok Kim Myoung; Jin Dong-Yan</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">49144-49155</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Extracellular signal-regulated kinase 8 (ERK8), also known as mitogen-activated protein kinase 15 (MAPK15), is the most recently identified protein kinase of the ERK family members and yet the least has been studied so far.  Here, we report that ERK8 is highly expressed in several human lung cancer cell lines and is positively correlated with their sensitivities to the anti-cancer drug arsenic trioxide (As2O3).  As2O3 at physiologically relevant concentrations (5-20 μM) potently stimulates the phosphorylation of ERK8 at Thr175 and Tyr177 within the TEY motif in the kinase domain, leading to its activation.  Interestingly, activated ERK8 interacts and directly phosphorylates IkappaBalpha (IκBα) at Ser32 and Ser36, resulting in IκBα degradation.  This in turn promotes nuclear factor-kappaB (NF-κB) p65 nuclear translocation and chromatin-binding, as well as the subsequent induction and activation of proteins involved in apoptosis.  We also show that stable short-hairpin RNA-specific knockdown of endogenous ERK8 or inhibition of NF-κB activity by NF-κB inhibitor in high ERK8 expressing lung cancer H1299 cells blunted the As2O3-induced NF-κB activation and cytotoxicity towards these cells, indicating the critical role of ERK8 and NF-κB in mediating the As2O3 effects.  Taken together, our findings suggest for the first time a regulatory paradigm of NF-κB activation by ERK8 upon As2O3 treatment in human lung cancer cells; and implicate a potential therapeutic advantage of As2O3 that might gain more selective killing of cancer cells with high ERK8 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVn-pfX7PHly86Js-BCLiifW6udTcc2ebxqJxTvKBkBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crjs1Cjsg%253D%253D&md5=11e8e30410c83cdf74894a9bed8dfa50</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.17100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.17100%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DD.-D.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DF.-Y.%26aulast%3DCai%26aufirst%3DN.-L.%26aulast%3DDai%26aufirst%3DL.-J.%26aulast%3DOk%2BKim%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DD.-Y.%26aulast%3DXu%26aufirst%3DY.-M.%26atitle%3DExtracellular%2520signal-regulated%2520kinase%25208-mediated%2520NF-kappaB%2520activation%2520increases%2520sensitivity%2520of%2520human%2520lung%2520cancer%2520cells%2520to%2520arsenic%2520trioxide%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D49144%26epage%3D49155%26doi%3D10.18632%2Foncotarget.17100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chieffi, P.</span></span> <span> </span><span class="NLM_article-title">Aurora B: A new promising therapeutic target in cancer</span>. <i>Intractable Rare Dis. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.5582/irdr.2018.01018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.5582%2Firdr.2018.01018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=29862159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC1MbjtlSnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=141-144&author=P.+Chieffi&title=Aurora+B%3A+A+new+promising+therapeutic+target+in+cancer&doi=10.5582%2Firdr.2018.01018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora B: A new promising therapeutic target in cancer</span></div><div class="casAuthors">Chieffi Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Intractable & rare diseases research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-144</span>
        ISSN:<span class="NLM_cas:issn">2186-3644</span>.
    </div><div class="casAbstract">A critical step for maintenance of genetic stability is chromosome segregation, which requires a high coordination of cellular processes.  Loss of mitotic regulation is a possible cause of aneuploidy in human epithelial malignancy and it is thought to create an abnormal nuclear morphology in cancer cells.  Serine/threonine protein kinase Aurora B gene plays a regulatory role from G2 to cytokinesis, encompassing key cell cycle events such as centrosome duplication, chromosome bi-orientation, and segregation.  The overexpression of Aurora B has been observed in several tumour types, and has been linked with a poor prognosis for cancer patients.  Therapeutic inhibition of Aurora kinase showed great promise as a probable anticancer regime because of its important role during cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxwXYefhgNVEodMGTatQ7LfW6udTcc2ebxqJxTvKBkBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MbjtlSnsg%253D%253D&md5=1f669456745503e18bf416c3f4b50a91</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.5582%2Firdr.2018.01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5582%252Firdr.2018.01018%26sid%3Dliteratum%253Aachs%26aulast%3DChieffi%26aufirst%3DP.%26atitle%3DAurora%2520B%253A%2520A%2520new%2520promising%2520therapeutic%2520target%2520in%2520cancer%26jtitle%3DIntractable%2520Rare%2520Dis.%2520Res.%26date%3D2018%26volume%3D7%26spage%3D141%26epage%3D144%26doi%3D10.5582%2Firdr.2018.01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vichai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirtikara, K.</span></span> <span> </span><span class="NLM_article-title">Sulforhodamine B colorimetric assay for cytotoxicity screening</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnprot.2006.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=17406391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=1112-1116&author=V.+Vichaiauthor=K.+Kirtikara&title=Sulforhodamine+B+colorimetric+assay+for+cytotoxicity+screening&doi=10.1038%2Fnprot.2006.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Sulforhodamine B colorimetric assay for cytotoxicity screening</span></div><div class="casAuthors">Vichai, Vanicha; Kirtikara, Kanyawim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1112-1116</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The sulforhodamine B (SRB) assay is used for cell d. detn., based on the measurement of cellular protein content.  The method described here was optimized for the toxicity screening of compds. to adherent cells in a 96-well format.  After an incubation period, cell monolayers are fixed with 10% (wt/vol) trichloroacetic acid and stained for 30 min, after which the excess dye is removed by washing repeatedly with 1% (vol/vol) acetic acid.  The protein-bound dye is dissolved in 10 mM Tris base soln. for OD detn. at 510 nm using a microplate reader.  The results are linear over a 20-fold range of cell nos. and the sensitivity is comparable to those of fluorometric methods.  The method not only allows a large no. of samples to be tested within a few days, but also requires only simple equipment and inexpensive reagents.  The SRB assay is therefore an efficient and highly cost-effective method for screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGZ8pao808l7Vg90H21EOLACvtfcHk0lj3su9PyAy58A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O&md5=7173035a1f9eb3f02dd7f1a947bca659</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.179%26sid%3Dliteratum%253Aachs%26aulast%3DVichai%26aufirst%3DV.%26aulast%3DKirtikara%26aufirst%3DK.%26atitle%3DSulforhodamine%2520B%2520colorimetric%2520assay%2520for%2520cytotoxicity%2520screening%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D1112%26epage%3D1116%26doi%3D10.1038%2Fnprot.2006.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrie, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelland, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span> <span> </span><span class="NLM_article-title">In vitro and In vivo pharmacokinetic-pharmacodynamic relationships for the trisubstituted aminopurine cyclin-dependent kinase inhibitors olomoucine, bohemine and CYC202</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4875</span>– <span class="NLM_lpage">4887</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-04-2264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1158%2F1078-0432.CCR-04-2264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=16000586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVGgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=4875-4887&author=F.+I.+Raynaudauthor=S.+R.+Whittakerauthor=P.+M.+Fischerauthor=S.+McClueauthor=M.+I.+Waltonauthor=S.+E.+Barrieauthor=M.+D.+Garrettauthor=P.+Rogersauthor=S.+J.+Clarkeauthor=L.+R.+Kellandauthor=M.+Valentiauthor=L.+Bruntonauthor=S.+Ecclesauthor=D.+P.+Laneauthor=P.+Workman&title=In+vitro+and+In+vivo+pharmacokinetic-pharmacodynamic+relationships+for+the+trisubstituted+aminopurine+cyclin-dependent+kinase+inhibitors+olomoucine%2C+bohemine+and+CYC202&doi=10.1158%2F1078-0432.CCR-04-2264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and In vivo Pharmacokinetic-Pharmacodynamic Relationships for the Trisubstituted Aminopurine Cyclin-Dependent Kinase Inhibitors Olomoucine, Bohemine and CYC202</span></div><div class="casAuthors">Raynaud, Florence I.; Whittaker, Steven R.; Fischer, Peter M.; McClue, Steven; Walton, Michael I.; Barrie, S. Elaine; Garrett, Michelle D.; Rogers, Paul; Clarke, Simon J.; Kelland, Lloyd R.; Valenti, Melanie; Brunton, Lisa; Eccles, Suzanne; Lane, David P.; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4875-4887</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: To investigate pharmacokinetic-pharmacodynamic relationships for the trisubstituted aminopurine cyclin-dependent kinase inhibitors olomoucine, bohemine, and CYC202 (R-roscovitine; seliciclib) in the HCT116 human colon carcinoma model.  Exptl. Design: The in vitro activity of the agents was detd. in a human tumor panel using the sulforhodamine B assay.  The concn. and time dependence was established in HCT116 cells.  Mol. biomarkers, including RB phosphorylation and cyclin expression, were assessed by Western blotting.  Pharmacokinetic properties were characterized in mice following anal. by liq. chromatog.-tandem mass spectrometry.  Based on these studies, a dosing regimen was developed for CYC202 that allowed therapeutic exposures in the HCT116 tumor xenograft.  RESULTS: The antitumor potency of the agents in vitro was in the order olomoucine (IC50, 56 μmol/L) < bohemine (IC50, 27 μmol/L) < CYC202 (IC50, 15 μmol/L), corresponding to their activities as cyclin-dependent kinase inhibitors.  Antitumor activity increased with exposure time up to 16 h.  The agents caused inhibition of RB and RNA polymerase II phosphorylation and depletion of cyclins.  They exhibited relatively rapid clearance following administration to mice.  CYC202 displayed the slowest clearance from plasma and the highest tumor uptake, with oral bioavailability of 86%.  Oral dosing of CYC202 gave active concns. in the tumor, modulation of pharmacodynamic markers, and inhibition of tumor growth.  CONCLUSIONS: CYC202 showed therapeutic activity on human cancer cell lines in vitro and on xenografts.  Pharmacodynamic markers are altered in vitro and in vivo, consistent with the inhibition of cyclin-dependent kinases.  Such markers may be potentially useful in the clin. development of CYC202 and other cyclin-dependent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF8GDYVKzb7rVg90H21EOLACvtfcHk0lgGntRl4Mrf7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVGgsLw%253D&md5=ecb8798ef675b44ea4ab5403940ce6ad</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-04-2264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-04-2264%26sid%3Dliteratum%253Aachs%26aulast%3DRaynaud%26aufirst%3DF.%2BI.%26aulast%3DWhittaker%26aufirst%3DS.%2BR.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DMcClue%26aufirst%3DS.%26aulast%3DWalton%26aufirst%3DM.%2BI.%26aulast%3DBarrie%26aufirst%3DS.%2BE.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26aulast%3DRogers%26aufirst%3DP.%26aulast%3DClarke%26aufirst%3DS.%2BJ.%26aulast%3DKelland%26aufirst%3DL.%2BR.%26aulast%3DValenti%26aufirst%3DM.%26aulast%3DBrunton%26aufirst%3DL.%26aulast%3DEccles%26aufirst%3DS.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DIn%2520vitro%2520and%2520In%2520vivo%2520pharmacokinetic-pharmacodynamic%2520relationships%2520for%2520the%2520trisubstituted%2520aminopurine%2520cyclin-dependent%2520kinase%2520inhibitors%2520olomoucine%252C%2520bohemine%2520and%2520CYC202%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2005%26volume%3D11%26spage%3D4875%26epage%3D4887%26doi%3D10.1158%2F1078-0432.CCR-04-2264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aslantürk, Ö. S.</span></span> In  <i>Vitro cytotoxicity and cell viability assays: principles, advantages, and disadvantages</i>; <span class="NLM_publisher-name">InTech</span>, <span class="NLM_year">2018</span>; Vol.  <span class="NLM_volume">2</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=%C3%96.+S.+Aslant%C3%BCrk&title=Vitro+cytotoxicity+and+cell+viability+assays%3A+principles%2C+advantages%2C+and+disadvantages"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAslant%25C3%25BCrk%26aufirst%3D%25C3%2596.%2BS.%26btitle%3DVitro%2520cytotoxicity%2520and%2520cell%2520viability%2520assays%253A%2520principles%252C%2520advantages%252C%2520and%2520disadvantages%26pub%3DInTech%26date%3D2018%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heber, D.</span></span> <span> </span><span class="NLM_article-title">Limitations of MTT and MTS-based assays for measurement of antiproliferative activity of green tea polyphenols</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e10202</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0010202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1371%2Fjournal.pone.0010202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=20419137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC3c3ovVSqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=P.+Wangauthor=S.+M.+Henningauthor=D.+Heber&title=Limitations+of+MTT+and+MTS-based+assays+for+measurement+of+antiproliferative+activity+of+green+tea+polyphenols&doi=10.1371%2Fjournal.pone.0010202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Limitations of MTT and MTS-based assays for measurement of antiproliferative activity of green tea polyphenols</span></div><div class="casAuthors">Wang Piwen; Henning Susanne M; Heber David</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e10202</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The chemopreventive effect of green tea polyphenols, such as (-)-epigallocatechin-3-gallate (EGCG), has been well demonstrated in cell culture studies.  However, a wide range of IC(50) concentrations has been observed in published studies of the anti-proliferative activity of EGCG from different laboratories.  Although the susceptibility to EGCG treatment is largely dependent on cancer cell type, the particular cell viability and proliferation assays utilized may significantly influence quantitative results reported in the literature.  METHODOLOGY/PRINCIPAL FINDINGS:  We compared five widely used methods to measure cell proliferation and viability after EGCG treatment using LNCaP prostate cancer cells and MCF-7 breast cancer cells.  Both methods using dyes to quantify adenosine triphosphate (ATP) and deoxynucleic acid (DNA) showed accuracy in the measurement of viable cells when compared to trypan blue assay and results showed good linear correlation (r = 0.95).  However, the use of MTT (3-(4,5-Dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide) and MTS (3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium) as indicators of metabolically active mitochondria overestimated the number of viable cells by comparison with the ATP, DNA, or trypan blue determinations.  As a result, the observed IC(50) concentration of EGCG was 2-fold higher using MTT and MTS compared to dyes quantifying ATP and DNA.  In contrast, when cells were treated with apigenin MTT and MTS assays showed consistent results with ATP, DNA, or trypan blue assays.  CONCLUSIONS/SIGNIFICANCE:  These results demonstrate that MTT and MTS -based assays will provide an underestimation of the anti-proliferative effect of EGCG, and suggest the importance of careful evaluation of the method for in vitro assessment of cell viability and proliferation depending on the chemical nature of botanical supplements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcToR5Y0J603atenUz0lpNqnfW6udTcc2eby2Fq_gBVnj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3ovVSqsQ%253D%253D&md5=dbd51aea663c0e2772d67a7d3b21d232</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0010202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0010202%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%26aulast%3DHenning%26aufirst%3DS.%2BM.%26aulast%3DHeber%26aufirst%3DD.%26atitle%3DLimitations%2520of%2520MTT%2520and%2520MTS-based%2520assays%2520for%2520measurement%2520of%2520antiproliferative%2520activity%2520of%2520green%2520tea%2520polyphenols%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0010202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sieuwerts, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span> <span> </span><span class="NLM_article-title">The MTT tetrazolium salt assay scrutinized: how to use this assay reliably to measure metabolie activity of cell cultures in vitro for the assessment of growth characteristics, IC50-values and cell survival</span>. <i>Clin. Chem. Lab. Med.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">824</span>, <span class="refDoi"> DOI: 10.1515/cclm.1995.33.11.813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1515%2Fcclm.1995.33.11.813" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1995&pages=813-824&author=A.+M.+Sieuwertsauthor=J.+G.+Klijnauthor=H.+A.+Petersauthor=J.+A.+Foekens&title=The+MTT+tetrazolium+salt+assay+scrutinized%3A+how+to+use+this+assay+reliably+to+measure+metabolie+activity+of+cell+cultures+in+vitro+for+the+assessment+of+growth+characteristics%2C+IC50-values+and+cell+survival&doi=10.1515%2Fcclm.1995.33.11.813"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1515%2Fcclm.1995.33.11.813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fcclm.1995.33.11.813%26sid%3Dliteratum%253Aachs%26aulast%3DSieuwerts%26aufirst%3DA.%2BM.%26aulast%3DKlijn%26aufirst%3DJ.%2BG.%26aulast%3DPeters%26aufirst%3DH.%2BA.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520MTT%2520tetrazolium%2520salt%2520assay%2520scrutinized%253A%2520how%2520to%2520use%2520this%2520assay%2520reliably%2520to%2520measure%2520metabolie%2520activity%2520of%2520cell%2520cultures%2520in%2520vitro%2520for%2520the%2520assessment%2520of%2520growth%2520characteristics%252C%2520IC50-values%2520and%2520cell%2520survival%26jtitle%3DClin.%2520Chem.%2520Lab.%2520Med.%26date%3D1995%26volume%3D33%26spage%3D813%26epage%3D824%26doi%3D10.1515%2Fcclm.1995.33.11.813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris-Natschke, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-H.</span></span> <span> </span><span class="NLM_article-title">Strategies for the optimization of natural leads to anticancer drugs or drug candidates</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1002/med.21377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1002%2Fmed.21377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26359649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVagsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=32-91&author=Z.+Xiaoauthor=S.+L.+Morris-Natschkeauthor=K.-H.+Lee&title=Strategies+for+the+optimization+of+natural+leads+to+anticancer+drugs+or+drug+candidates&doi=10.1002%2Fmed.21377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the Optimization of Natural Leads to Anticancer Drugs or Drug Candidates</span></div><div class="casAuthors">Xiao, Zhiyan; Morris-Natschke, Susan L.; Lee, Kuo-Hsiung</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-91</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Natural products have made significant contribution to cancer chemotherapy over the past decades and remain an indispensable source of mol. and mechanistic diversity for anticancer drug discovery.  More often than not, natural products may serve as leads for further drug development rather than as effective anticancer drugs by themselves.  Generally, optimization of natural leads into anticancer drugs or drug candidates should not only address drug efficacy, but also improve absorption, distribution, metab., excretion, and toxicity (ADMET) profiles and chem. accessibility assocd. with the natural leads.  Optimization strategies involve direct chem. manipulation of functional groups, structure-activity relationship directed optimization and pharmacophore-oriented mol. design based on the natural templates.  Both fundamental medicinal chem. principles (e.g., bioisosterism) and state-of-the-art computer-aided drug design techniques (e.g., structure-based design) can be applied to facilitate optimization efforts.  In this review, the strategies to optimize natural leads to anticancer drugs or drug candidates are illustrated with examples and described according to their purposes.  Furthermore, successful case studies on lead optimization of bioactive compds. performed in the Natural Products Research Labs. at UNC are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6SGKXSp6ip7Vg90H21EOLACvtfcHk0lgEB1r5-nDHlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVagsb%252FJ&md5=dc8c68d23f1f8e9ff6e98847a69558c4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fmed.21377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21377%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DMorris-Natschke%26aufirst%3DS.%2BL.%26aulast%3DLee%26aufirst%3DK.-H.%26atitle%3DStrategies%2520for%2520the%2520optimization%2520of%2520natural%2520leads%2520to%2520anticancer%2520drugs%2520or%2520drug%2520candidates%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D32%26epage%3D91%26doi%3D10.1002%2Fmed.21377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mintoo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guru, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuthakki, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondhe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhushan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Discovery and preclinical development of IIIM-290, an orally active potent cyclin-dependent kinase Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1664</span>– <span class="NLM_lpage">1687</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsl2js7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1664-1687&author=S.+B.+Bharateauthor=V.+Kumarauthor=S.+K.+Jainauthor=M.+J.+Mintooauthor=S.+K.+Guruauthor=V.+K.+Nuthakkiauthor=M.+Sharmaauthor=S.+S.+Bharateauthor=S.+G.+Gandhiauthor=D.+M.+Mondheauthor=S.+Bhushanauthor=R.+A.+Vishwakarma&title=Discovery+and+preclinical+development+of+IIIM-290%2C+an+orally+active+potent+cyclin-dependent+kinase+Inhibitor&doi=10.1021%2Facs.jmedchem.7b01765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Preclinical Development of IIIM-290, an Orally Active Potent Cyclin-Dependent Kinase Inhibitor</span></div><div class="casAuthors">Bharate, Sandip B.; Kumar, Vikas; Jain, Shreyans K.; Mintoo, Mubashir J.; Guru, Santosh K.; Nuthakki, Vijay K.; Sharma, Mohit; Bharate, Sonali S.; Gandhi, Sumit G.; Mondhe, Dilip M.; Bhushan, Shashi; Vishwakarma, Ram A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1664-1687</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Rohitukine (1), a chromone alkaloid isolated from Indian medicinal plant Dysoxylum binectariferum, has inspired the discovery of flavopiridol and riviciclib, both of which are bioavailable only via i.v. route.  With the objective to address the oral bioavailability issue of this scaffold, four series of rohitukine derivs. were prepd. and screened for Cdk inhibition and cellular antiproliferative activity.  The 2,6-dichloro-styryl deriv. IIIM-290 (11d) showed strong inhibition of Cdk-9/T1 (IC50 1.9 nM) kinase and Molt-4/MIAPaCa-2 cell growth (GI50 < 1.0 μM) and was found to be highly selective for cancer cells over normal fibroblast cells.  It inhibited the cell growth of MIAPaCa-2 cells via caspase-dependent apoptosis.  It achieved 71% oral bioavailability with in vivo efficacy in pancreatic, colon, and leukemia xenografts at 50 mg/kg, po.  It did not have CYP/efflux-pump liability, was not mutagenic/genotoxic or cardiotoxic, and was metabolically stable.  The preclin. data presented herein indicates the potential of 11d for advancement in clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeRTRG-f8KqLVg90H21EOLACvtfcHk0ljsKR32cRaFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsl2js7k%253D&md5=13b67cdf1db966e2a414b64cd82a98e8</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01765%26sid%3Dliteratum%253Aachs%26aulast%3DBharate%26aufirst%3DS.%2BB.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DJain%26aufirst%3DS.%2BK.%26aulast%3DMintoo%26aufirst%3DM.%2BJ.%26aulast%3DGuru%26aufirst%3DS.%2BK.%26aulast%3DNuthakki%26aufirst%3DV.%2BK.%26aulast%3DSharma%26aufirst%3DM.%26aulast%3DBharate%26aufirst%3DS.%2BS.%26aulast%3DGandhi%26aufirst%3DS.%2BG.%26aulast%3DMondhe%26aufirst%3DD.%2BM.%26aulast%3DBhushan%26aufirst%3DS.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DDiscovery%2520and%2520preclinical%2520development%2520of%2520IIIM-290%252C%2520an%2520orally%2520active%2520potent%2520cyclin-dependent%2520kinase%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1664%26epage%3D1687%26doi%3D10.1021%2Facs.jmedchem.7b01765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinase inhibition by flavoalkaloids</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">632</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.2174/138955712800626683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.2174%2F138955712800626683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=22512551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC38XoslCkurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=632-649&author=S.+K.+Jainauthor=S.+B.+Bharateauthor=R.+A.+Vishwakarma&title=Cyclin-dependent+kinase+inhibition+by+flavoalkaloids&doi=10.2174%2F138955712800626683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinase inhibition by flavoalkaloids</span></div><div class="casAuthors">Jain, S. K.; Bharate, S. B.; Vishwakarma, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">632-649</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Chromone alkaloids and flavoalkaloids are an important group of natural products possessing promising medicinal properties.  A chromone alkaloid rohitukine is a major bioactive chem. constituent of plant Dysoxylum binectariferum (Meliaceae) which is phylogenetically related to the Ayurvedic plant, D. malabaricum used for treatment of rheumatoid arthritis.  This chromone alkaloid led to discovery of 2 synthetic flavoalkaloids: flavopiridol (Sanofi) and P-276-00 (Piramal) which have reached to advanced stages of clin. development for cancer treatment.  Flavopiridol (Alvocidib; L868275; HMR-1275; NSC 649890 of Sanofi-Aventis + NCI) is approved as an orphan drug for treatment of chronic lymphocytic leukemia and is currently undergoing phase II studies as monotherapy and also as in combination regimes with traditional chemotherapy agents.  P-276-00 (12) is currently in phase II clin. studies for advanced refractory neoplasms and multiple myeloma.  Extensive amt. of medicinal chem. efforts were reported on these flavoalkaloids.  Flavopiridol demonstrated potent and specific in vitro inhibition of variety of cyclin-dependent kinases with clear block in cell cycle progression at the G1/S and G2/M phases.  Preclin. studies demonstrated the capacity of flavopiridol to induce programmed cell death, promote differentiation, inhibit angiogenic processes and modulate transcriptional events.  The co-crystd. structure of deschloro-flavopiridol with CDK-2 is available and key interactions in the ATP binding site were reported.  Flavopiridol was also studied for the treatment of arthritis and atherosclerotic plaque formation.  The present review comprises discovery, medicinal chem., pharmacol. and preclin./clin. development of flavoalkaloids as CDK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDQdAK15QBLLVg90H21EOLACvtfcHk0ljsKR32cRaFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XoslCkurs%253D&md5=98702838814112f01f0f7585d9e6e994</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.2174%2F138955712800626683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138955712800626683%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DS.%2BK.%26aulast%3DBharate%26aufirst%3DS.%2BB.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DCyclin-dependent%2520kinase%2520inhibition%2520by%2520flavoalkaloids%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D632%26epage%3D649%26doi%3D10.2174%2F138955712800626683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanjo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akira, S.</span></span> <span> </span><span class="NLM_article-title">DRAKs, novel serine/threonine kinases related to death-associated protein kinase that trigger apoptosis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">29066</span>– <span class="NLM_lpage">29071</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.44.29066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1074%2Fjbc.273.44.29066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=9786912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK1cXntlaht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=29066-29071&author=H.+Sanjoauthor=T.+Kawaiauthor=S.+Akira&title=DRAKs%2C+novel+serine%2Fthreonine+kinases+related+to+death-associated+protein+kinase+that+trigger+apoptosis&doi=10.1074%2Fjbc.273.44.29066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">DRAKs, novel serine/threonine kinases related to death-associated protein kinase that trigger apoptosis</span></div><div class="casAuthors">Sanjo, Hideki; Kawai, Taro; Akira, Shizuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">29066-29071</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The present study describes the cloning of two novel serine/threonine kinases termed DRAK1 and DRAK2, whose catalytic domains are related to that of death-assocd. protein kinase, a serine/threonine kinase involved in apoptosis.  Both DRAKs are composed of the N-terminal catalytic domain and the C-terminal domain that is responsible for regulation of kinase activity.  DRAK1 and DRAK2 show 59.7% identity and display ubiquitous expression.  An in vitro kinase assay revealed that both DRAKs are autophosphorylated and phosphorylate myosin light chain as an exogenous substrate, although the kinase activity of DRAK2 is significantly lower than that of DRAK1.  Both DRAKs are exclusively localized to the nucleus.  Furthermore, overexpression of both DRAKs induces the morphol. changes of apoptosis in NIH 3T3 cells, suggesting the role of DRAKs in apoptotic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmFXOHrs4s2bVg90H21EOLACvtfcHk0ljsKR32cRaFnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntlaht7s%253D&md5=6b17b0e7dd8e61ec3b435b245cbcb627</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.44.29066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.44.29066%26sid%3Dliteratum%253Aachs%26aulast%3DSanjo%26aufirst%3DH.%26aulast%3DKawai%26aufirst%3DT.%26aulast%3DAkira%26aufirst%3DS.%26atitle%3DDRAKs%252C%2520novel%2520serine%252Fthreonine%2520kinases%2520related%2520to%2520death-associated%2520protein%2520kinase%2520that%2520trigger%2520apoptosis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D29066%26epage%3D29071%26doi%3D10.1074%2Fjbc.273.44.29066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebakken, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddle, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazee, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichling, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span> <span> </span><span class="NLM_article-title">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1177/1087057109339207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1177%2F1087057109339207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=19564447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=924-935&author=C.+S.+Lebakkenauthor=S.+M.+Riddleauthor=U.+Singhauthor=W.+J.+Frazeeauthor=H.+C.+Eliasonauthor=Y.+Gaoauthor=L.+J.+Reichlingauthor=B.+D.+Marksauthor=K.+W.+Vogel&title=Development+and+applications+of+a+broad-coverage%2C+TR-FRET-based+kinase+binding+assay+platform&doi=10.1177%2F1087057109339207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span></div><div class="casAuthors">Lebakken, Connie S.; Riddle, Steven M.; Singh, Upinder; Frazee, W. Jack; Eliason, Hildegard C.; Gao, Yi; Reichling, Laurie J.; Marks, Bryan D.; Vogel, Kurt W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">924-935</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The expansion of kinase assay technologies over the past decade has mirrored the growing interest in kinases as drug targets.  As a result, there is no shortage of convenient, fluorescence-based methods available to assay targets that span the kinome.  The authors recently reported on the development of a non-activity-based assay to characterize kinase inhibitors that depended on displacement of an Alexa Fluor 647 conjugate of staurosporine (a "tracer") from a particular kinase.  Kinase inhibitors were characterized by a change in fluorescence lifetime of the tracer when it was bound to a kinase relative to when it was displaced by an inhibitor.  Here, the authors report on improvements to this strategy by reconfiguring the assay in a time-resolved fluorescence resonance energy transfer (TR-FRET) format that simplifies instrumentation requirements and allows for the use of a substantially lower concn. of kinase than was required in the fluorescence-lifetime-based format.  The authors use this new assay to demonstrate several aspects of the binding assay format that are advantageous relative to traditional activity-based assays.  The TR-FRET binding format facilitates the assay of compds. against low-activity kinases, allows for the characterization of type II kinase inhibitors either using nonactivated kinases or by monitoring compd. potency over time, and ensures that the signal being detected is specific to the kinase of interest and not a contaminating kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqPbcMF0-PD7Vg90H21EOLACvtfcHk0lhtaQJFqtXX9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM&md5=f8df51adf90d821e480cf706213cc096</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1177%2F1087057109339207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109339207%26sid%3Dliteratum%253Aachs%26aulast%3DLebakken%26aufirst%3DC.%2BS.%26aulast%3DRiddle%26aufirst%3DS.%2BM.%26aulast%3DSingh%26aufirst%3DU.%26aulast%3DFrazee%26aufirst%3DW.%2BJ.%26aulast%3DEliason%26aufirst%3DH.%2BC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DReichling%26aufirst%3DL.%2BJ.%26aulast%3DMarks%26aufirst%3DB.%2BD.%26aulast%3DVogel%26aufirst%3DK.%2BW.%26atitle%3DDevelopment%2520and%2520applications%2520of%2520a%2520broad-coverage%252C%2520TR-FRET-based%2520kinase%2520binding%2520assay%2520platform%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D924%26epage%3D935%26doi%3D10.1177%2F1087057109339207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.+T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+Mullerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0lj-KPIdmUgqWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.%2BT.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sipes, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, T. B.</span></span> <span> </span><span class="NLM_article-title">Zebrafish: as an integrative model for twenty-first century toxicity testing</span>. <i>Birth Defects Res., Part C</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1002/bdrc.20214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1002%2Fbdrc.20214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21932434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFyjs7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2011&pages=256-267&author=N.+S.+Sipesauthor=S.+Padillaauthor=T.+B.+Knudsen&title=Zebrafish%3A+as+an+integrative+model+for+twenty-first+century+toxicity+testing&doi=10.1002%2Fbdrc.20214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish-As an integrative model for twenty-first century toxicity testing</span></div><div class="casAuthors">Sipes, Nisha S.; Padilla, Stephanie; Knudsen, Thomas B.</div><div class="citationInfo"><span class="NLM_cas:title">Birth Defects Research, Part C: Embryo Today--Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">256-267</span>CODEN:
                <span class="NLM_cas:coden">BDRPDV</span>;
        ISSN:<span class="NLM_cas:issn">1542-975X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  The zebrafish embryo is a useful small model for investigating vertebrate development because of its transparency, low cost, transgenic and morpholino capabilities, conservation of cell signaling, and concordance with mammalian developmental phenotypes.  From these advantages, the zebrafish embryo has been considered as an alternative model for traditional in vivo developmental toxicity screening.  The use of this organism in conjunction with traditional in vivo developmental toxicity testing has the potential to reduce cost and increase throughput of testing the chem. universe, prioritize chems. for targeted toxicity testing, generate predictive models of developmental toxicants, and elucidate mechanisms and adverse outcome pathways for abnormal development.  This review gives an overview of the zebrafish embryo for predictive toxicol. and 21st century toxicity testing.  Developmental eye defects were selected as an example to evaluate data from the U.S.  Environmental Protection Agency's ToxCast program comparing responses in zebrafish embryos with those from pregnant rats and rabbits for a subset of 24 environmental chems. across >600 in vitro assay targets.  Cross-species comparisons implied a common basis for biol. pathways assocd. with neuronal defects, extracellular matrix remodeling, and mitotic arrest.  Birth Defects Research (Part C) 93:256-267, 2011. © 2011 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3C_KqV8TTKrVg90H21EOLACvtfcHk0lj-KPIdmUgqWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFyjs7rL&md5=e86f008ee9675a5a817d3ea006566b07</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fbdrc.20214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbdrc.20214%26sid%3Dliteratum%253Aachs%26aulast%3DSipes%26aufirst%3DN.%2BS.%26aulast%3DPadilla%26aufirst%3DS.%26aulast%3DKnudsen%26aufirst%3DT.%2BB.%26atitle%3DZebrafish%253A%2520as%2520an%2520integrative%2520model%2520for%2520twenty-first%2520century%2520toxicity%2520testing%26jtitle%3DBirth%2520Defects%2520Res.%252C%2520Part%2520C%26date%3D2011%26volume%3D93%26spage%3D256%26epage%3D267%26doi%3D10.1002%2Fbdrc.20214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 9 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhenxing Zhang, Chunqi Jia, Xiangtao Kong, Muzammal Hussain, Zhenyu Liu, Wanqing Liang, Ling Jiang, Hao Jiang, <span class="NLM_string-name hlFld-ContribAuthor">Junyan Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Sustainable Approach to Azaheterocyclic Acetamides by Decarboxylative Aminoformylation. </span><span class="cited-content_cbyCitation_journal-name">ACS Sustainable Chemistry & Engineering</span><span> <strong>2020,</strong> <em>8 </em>
                                    (44)
                                     , 16463-16468. <a href="https://doi.org/10.1021/acssuschemeng.0c05118" title="DOI URL">https://doi.org/10.1021/acssuschemeng.0c05118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acssuschemeng.0c05118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facssuschemeng.0c05118%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Sustainable%2520Chemistry%2520%2526%2520Engineering%26atitle%3DSustainable%252BApproach%252Bto%252BAzaheterocyclic%252BAcetamides%252Bby%252BDecarboxylative%252BAminoformylation%26aulast%3DZhang%26aufirst%3DZhenxing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12072020%26date%3D08092020%26date%3D06102020%26date%3D22092020%26volume%3D8%26issue%3D44%26spage%3D16463%26epage%3D16468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Da-Yun  Luo</span>, <span class="hlFld-ContribAuthor ">Xing-Mei  Hu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>, <span class="hlFld-ContribAuthor ">Shi-Sheng  Cui</span>, <span class="hlFld-ContribAuthor ">Sheng-Jiao  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Base-promoted relay reaction of heterocyclic ketene aminals with o-difluorobenzene derivatives for the highly site-selective synthesis of functionalized indoles. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2021,</strong> <em>92 </em>, 132275. <a href="https://doi.org/10.1016/j.tet.2021.132275" title="DOI URL">https://doi.org/10.1016/j.tet.2021.132275</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2021.132275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2021.132275%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DBase-promoted%252Brelay%252Breaction%252Bof%252Bheterocyclic%252Bketene%252Baminals%252Bwith%252Bo-difluorobenzene%252Bderivatives%252Bfor%252Bthe%252Bhighly%252Bsite-selective%252Bsynthesis%252Bof%252Bfunctionalized%252Bindoles%26aulast%3DLuo%26aufirst%3DDa-Yun%26date%3D2021%26volume%3D92%26spage%3D132275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ana  Rotter</span>, <span class="hlFld-ContribAuthor ">Michéle  Barbier</span>, <span class="hlFld-ContribAuthor ">Francesco  Bertoni</span>, <span class="hlFld-ContribAuthor ">Atle M.  Bones</span>, <span class="hlFld-ContribAuthor ">M. Leonor  Cancela</span>, <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>, <span class="hlFld-ContribAuthor ">Maria F.  Carvalho</span>, <span class="hlFld-ContribAuthor ">Marta  Cegłowska</span>, <span class="hlFld-ContribAuthor ">Jerónimo  Chirivella-Martorell</span>, <span class="hlFld-ContribAuthor ">Meltem  Conk Dalay</span>, <span class="hlFld-ContribAuthor ">Mercedes  Cueto</span>, <span class="hlFld-ContribAuthor ">Thanos  Dailianis</span>, <span class="hlFld-ContribAuthor ">Irem  Deniz</span>, <span class="hlFld-ContribAuthor ">Ana R.  Díaz-Marrero</span>, <span class="hlFld-ContribAuthor ">Dragana  Drakulovic</span>, <span class="hlFld-ContribAuthor ">Arita  Dubnika</span>, <span class="hlFld-ContribAuthor ">Christine  Edwards</span>, <span class="hlFld-ContribAuthor ">Hjörleifur  Einarsson</span>, <span class="hlFld-ContribAuthor ">Ayşegül  Erdoǧan</span>, <span class="hlFld-ContribAuthor ">Orhan Tufan  Eroldoǧan</span>, <span class="hlFld-ContribAuthor ">David  Ezra</span>, <span class="hlFld-ContribAuthor ">Stefano  Fazi</span>, <span class="hlFld-ContribAuthor ">Richard J.  FitzGerald</span>, <span class="hlFld-ContribAuthor ">Laura M.  Gargan</span>, <span class="hlFld-ContribAuthor ">Susana P.  Gaudêncio</span>, <span class="hlFld-ContribAuthor ">Marija  Gligora Udovič</span>, <span class="hlFld-ContribAuthor ">Nadica  Ivošević DeNardis</span>, <span class="hlFld-ContribAuthor ">Rósa  Jónsdóttir</span>, <span class="hlFld-ContribAuthor ">Marija  Kataržytė</span>, <span class="hlFld-ContribAuthor ">Katja  Klun</span>, <span class="hlFld-ContribAuthor ">Jonne  Kotta</span>, <span class="hlFld-ContribAuthor ">Leila  Ktari</span>, <span class="hlFld-ContribAuthor ">Zrinka  Ljubešić</span>, <span class="hlFld-ContribAuthor ">Lada  Lukić Bilela</span>, <span class="hlFld-ContribAuthor ">Manolis  Mandalakis</span>, <span class="hlFld-ContribAuthor ">Alexia  Massa-Gallucci</span>, <span class="hlFld-ContribAuthor ">Inga  Matijošytė</span>, <span class="hlFld-ContribAuthor ">Hanna  Mazur-Marzec</span>, <span class="hlFld-ContribAuthor ">Mohamed  Mehiri</span>, <span class="hlFld-ContribAuthor ">Søren Laurentius  Nielsen</span>, <span class="hlFld-ContribAuthor ">Lucie  Novoveská</span>, <span class="hlFld-ContribAuthor ">Donata  Overlingė</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Perale</span>, <span class="hlFld-ContribAuthor ">Praveen  Ramasamy</span>, <span class="hlFld-ContribAuthor ">Céline  Rebours</span>, <span class="hlFld-ContribAuthor ">Thorsten  Reinsch</span>, <span class="hlFld-ContribAuthor ">Fernando  Reyes</span>, <span class="hlFld-ContribAuthor ">Baruch  Rinkevich</span>, <span class="hlFld-ContribAuthor ">Johan  Robbens</span>, <span class="hlFld-ContribAuthor ">Eric  Röttinger</span>, <span class="hlFld-ContribAuthor ">Vita  Rudovica</span>, <span class="hlFld-ContribAuthor ">Jerica  Sabotič</span>, <span class="hlFld-ContribAuthor ">Ivo  Safarik</span>, <span class="hlFld-ContribAuthor ">Siret  Talve</span>, <span class="hlFld-ContribAuthor ">Deniz  Tasdemir</span>, <span class="hlFld-ContribAuthor ">Xenia  Theodotou Schneider</span>, <span class="hlFld-ContribAuthor ">Olivier P.  Thomas</span>, <span class="hlFld-ContribAuthor ">Anna  Toruńska-Sitarz</span>, <span class="hlFld-ContribAuthor ">Giovanna Cristina  Varese</span>, <span class="hlFld-ContribAuthor ">Marlen I.  Vasquez</span>. </span><span class="cited-content_cbyCitation_article-title">The Essentials of Marine Biotechnology. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Marine Science</span><span> <strong>2021,</strong> <em>8 </em><a href="https://doi.org/10.3389/fmars.2021.629629" title="DOI URL">https://doi.org/10.3389/fmars.2021.629629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmars.2021.629629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmars.2021.629629%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Marine%2520Science%26atitle%3DThe%252BEssentials%252Bof%252BMarine%252BBiotechnology%26aulast%3DRotter%26aufirst%3DAna%26date%3D2021%26date%3D2021%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony R.  Carroll</span>, <span class="hlFld-ContribAuthor ">Brent R.  Copp</span>, <span class="hlFld-ContribAuthor ">Rohan A.  Davis</span>, <span class="hlFld-ContribAuthor ">Robert A.  Keyzers</span>, <span class="hlFld-ContribAuthor ">Michèle R.  Prinsep</span>. </span><span class="cited-content_cbyCitation_article-title">Marine natural products. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2021,</strong> <em>38 </em>
                                    (2)
                                     , 362-413. <a href="https://doi.org/10.1039/D0NP00089B" title="DOI URL">https://doi.org/10.1039/D0NP00089B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NP00089B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NP00089B%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DMarine%252Bnatural%252Bproducts%26aulast%3DCarroll%26aufirst%3DAnthony%2BR.%26date%3D2021%26date%3D2021%26volume%3D38%26issue%3D2%26spage%3D362%26epage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ikenna E.  Ndukwe</span>, <span class="hlFld-ContribAuthor ">Yu-hong  Lam</span>, <span class="hlFld-ContribAuthor ">Sunil K.  Pandey</span>, <span class="hlFld-ContribAuthor ">Bengt E.  Haug</span>, <span class="hlFld-ContribAuthor ">Annette  Bayer</span>, <span class="hlFld-ContribAuthor ">Edward C.  Sherer</span>, <span class="hlFld-ContribAuthor ">Kirill A.  Blinov</span>, <span class="hlFld-ContribAuthor ">R. Thomas  Williamson</span>, <span class="hlFld-ContribAuthor ">Johan  Isaksson</span>, <span class="hlFld-ContribAuthor ">Mikhail  Reibarkh</span>, <span class="hlFld-ContribAuthor ">Yizhou  Liu</span>, <span class="hlFld-ContribAuthor ">Gary E.  Martin</span>. </span><span class="cited-content_cbyCitation_article-title">Unequivocal structure confirmation of a breitfussin analog by anisotropic NMR measurements. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2020,</strong> <em>11 </em>
                                    (44)
                                     , 12081-12088. <a href="https://doi.org/10.1039/D0SC03664A" title="DOI URL">https://doi.org/10.1039/D0SC03664A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC03664A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC03664A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DUnequivocal%252Bstructure%252Bconfirmation%252Bof%252Ba%252Bbreitfussin%252Banalog%252Bby%252Banisotropic%252BNMR%252Bmeasurements%26aulast%3DNdukwe%26aufirst%3DIkenna%2BE.%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D44%26spage%3D12081%26epage%3D12088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yngve  Guttormsen</span>, <span class="hlFld-ContribAuthor ">Magnus E.  Fairhurst</span>, <span class="hlFld-ContribAuthor ">Sunil K.  Pandey</span>, <span class="hlFld-ContribAuthor ">Johan  Isaksson</span>, <span class="hlFld-ContribAuthor ">Bengt Erik  Haug</span>, <span class="hlFld-ContribAuthor ">Annette  Bayer</span>. </span><span class="cited-content_cbyCitation_article-title">Total Synthesis of Phorbazole B. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (20)
                                     , 4848. <a href="https://doi.org/10.3390/molecules25204848" title="DOI URL">https://doi.org/10.3390/molecules25204848</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25204848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25204848%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DTotal%252BSynthesis%252Bof%252BPhorbazole%252BB%26aulast%3DGuttormsen%26aufirst%3DYngve%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D20%26spage%3D4848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dan  Xu</span>, <span class="hlFld-ContribAuthor ">Zhi  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Indole Alkaloids with Potential Anticancer Activity. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (21)
                                     , 1938-1949. <a href="https://doi.org/10.2174/1568026620666200622150325" title="DOI URL">https://doi.org/10.2174/1568026620666200622150325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026620666200622150325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026620666200622150325%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DIndole%252BAlkaloids%252Bwith%252BPotential%252BAnticancer%252BActivity%26aulast%3DXu%26aufirst%3DDan%26date%3D2020%26volume%3D20%26issue%3D21%26spage%3D1938%26epage%3D1949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanshu  Jia</span>, <span class="hlFld-ContribAuthor ">Xiaoyue  Wen</span>, <span class="hlFld-ContribAuthor ">Yufeng  Gong</span>, <span class="hlFld-ContribAuthor ">Xuefeng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Current scenario of indole derivatives with potential anti-drug-resistant cancer activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112359. <a href="https://doi.org/10.1016/j.ejmech.2020.112359" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112359%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCurrent%252Bscenario%252Bof%252Bindole%252Bderivatives%252Bwith%252Bpotential%252Banti-drug-resistant%252Bcancer%252Bactivity%26aulast%3DJia%26aufirst%3DYanshu%26date%3D2020%26volume%3D200%26spage%3D112359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ida K. Ø.  Hansen</span>, <span class="hlFld-ContribAuthor ">Tomas  Lövdahl</span>, <span class="hlFld-ContribAuthor ">Danijela  Simonovic</span>, <span class="hlFld-ContribAuthor ">Kine Ø.  Hansen</span>, <span class="hlFld-ContribAuthor ">Aaron J. C.  Andersen</span>, <span class="hlFld-ContribAuthor ">Hege  Devold</span>, <span class="hlFld-ContribAuthor ">Céline S. M.  Richard</span>, <span class="hlFld-ContribAuthor ">Jeanette H.  Andersen</span>, <span class="hlFld-ContribAuthor ">Morten B.  Strøm</span>, <span class="hlFld-ContribAuthor ">Tor  Haug</span>. </span><span class="cited-content_cbyCitation_article-title">Antimicrobial Activity of Small Synthetic Peptides Based on the Marine Peptide Turgencin A: Prediction of Antimicrobial Peptide Sequences in a Natural Peptide and Strategy for Optimization of Potency. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (15)
                                     , 5460. <a href="https://doi.org/10.3390/ijms21155460" title="DOI URL">https://doi.org/10.3390/ijms21155460</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21155460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21155460%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DAntimicrobial%252BActivity%252Bof%252BSmall%252BSynthetic%252BPeptides%252BBased%252Bon%252Bthe%252BMarine%252BPeptide%252BTurgencin%252BA%25253A%252BPrediction%252Bof%252BAntimicrobial%252BPeptide%252BSequences%252Bin%252Ba%252BNatural%252BPeptide%252Band%252BStrategy%252Bfor%252BOptimization%252Bof%252BPotency%26aulast%3DHansen%26aufirst%3DIda%2BK.%2B%25C3%2598.%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D15%26spage%3D5460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of <b>1</b>–<b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selected COSY, HMBC, and NOE correlations obtained for <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Total Synthesis of <b>3</b> from Indole <b>10</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) <i>sec</i>-BuLi, TMEDA, THF, 15 °C to −5 °C and (ii) Br(CH<sub>2</sub>)<sub>2</sub>Br, −5 °C to rt (67% yield for mixture of <b>11</b> and <b>12</b>); (b) ICl, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt (97% yield); (c) (i) (2-triisopropylsilyl)oxazol-5-yl)boronic acid pinacol ester, Pd(dppf)Cl<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, toluene/H<sub>2</sub>O, 80 °C and (ii) 10% aq HCl, THF, 0 °C (62% yield over 2 steps); (d) (i) LiHMDS, THF, −78 °C and (ii) I<sub>2</sub>, −78 °C (31% yield for <b>15</b>, 47% yield for mixture of <b>16</b> and <b>17</b>); (e) <i>N</i>-Boc-pyrrole-2-boronic acid, Pd(dppf)Cl<sub>2</sub>*CH<sub>2</sub>Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O (66% yield); (f) (i) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt and (ii) TBAF, THF (68% yield over 2 steps).</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) TFA, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt and (ii) TBAF, THF (77% yield over 2 steps).</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Kinome inhibition data of <b>3</b> at (A) 50 and (B) 1 μM against 140 kinases. Disc sizes and colors are encoding the level of residual activity as shown in the figure key.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinome binding affinity data of compound <b>3</b> at 10 μM against 468 kinases. The disc sizes encode the level of residual activity as shown in the figure key.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. IC<sub>50</sub> values for <b>3</b> against the 88 cell lines found in the OncoLead panel (Z-scores in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf" class="ext-link">Table S8</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding mode of <b>3</b> with the ATP-binding pocket of PIM1 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B">4K1B</a>). Compound <b>3</b> is shown with yellow carbon atoms, the hinge backbone carbon atoms of PIM1 are in green (Pro123, Arg122, Glu121, and Leu120). A water molecule present in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B">4K1B</a> is replaced by the pyrrole and is shown as a red sphere. (A) Surface representation of the ATP-binding pocket of PIM1 with compound <b>3</b>. (B) Detailed view of the interaction between the PIM1 hinge residues and <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Potential off-targets (>50%) of <b>3</b> tested at 10 μM in the Cerep Panlabs diversity profile panel. * = human, # = agonist, $ = antagonist, and § = enzymatic assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/medium/jm9b01006_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Assessment of toxicity of <b>3</b> against zebrafish embryos.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01006/20191122/images/large/jm9b01006_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01006&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13317" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13317" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhullar, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagarón, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parmar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rupasinghe, H. P. V.</span></span> <span> </span><span class="NLM_article-title">Kinase-targeted cancer therapies: progress, challenges and future directions</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">48</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0804-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2Fs12943-018-0804-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=29455673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=48&author=K.+S.+Bhullarauthor=N.+O.+Lagar%C3%B3nauthor=E.+M.+McGowanauthor=I.+Parmarauthor=A.+Jhaauthor=B.+P.+Hubbardauthor=H.+P.+V.+Rupasinghe&title=Kinase-targeted+cancer+therapies%3A+progress%2C+challenges+and+future+directions&doi=10.1186%2Fs12943-018-0804-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase-targeted cancer therapies: progress, challenges and future directions</span></div><div class="casAuthors">Bhullar, Khushwant S.; Lagaron, Naiara Orrego; McGowan, Eileen M.; Parmar, Indu; Jha, Amitabh; Hubbard, Basil P.; Rupasinghe, H. P. Vasantha</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">48/1-48/20</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">The human genome encodes 538 protein kinases that transfer a γ-phosphate group from ATP to serine, threonine, or tyrosine residues.  Many of these kinases are assocd. with human cancer initiation and progression.  The recent development of small-mol. kinase inhibitors for the treatment of diverse types of cancer has proven successful in clin. therapy.  Significantly, protein kinases are the second most targeted group of drug targets, after the G-proteincoupled receptors.  Since the development of the first protein kinase inhibitor, in the early 1980s, 37 kinase inhibitors have received FDA approval for treatment of malignancies such as breast and lung cancer.  Furthermore, about 150 kinase-targeted drugs are in clin. phase trials, and many kinase-specific inhibitors are in the preclin. stage of drug development.  Nevertheless, many factors confound the clin. efficacy of these mols.  Specific tumor genetics, tumor microenvironment, drug resistance, and pharmacogenomics det. how useful a compd. will be in the treatment of a given cancer.  This review provides an overview of kinase-targeted drug discovery and development in relation to oncol. and highlights the challenges and future potential for kinase-targeted cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb-edOGHyJPbVg90H21EOLACvtfcHk0li2sjB2d0tnnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7fP&md5=30811747f13749327d998c5dadd2161d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0804-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0804-2%26sid%3Dliteratum%253Aachs%26aulast%3DBhullar%26aufirst%3DK.%2BS.%26aulast%3DLagar%25C3%25B3n%26aufirst%3DN.%2BO.%26aulast%3DMcGowan%26aufirst%3DE.%2BM.%26aulast%3DParmar%26aufirst%3DI.%26aulast%3DJha%26aufirst%3DA.%26aulast%3DHubbard%26aufirst%3DB.%2BP.%26aulast%3DRupasinghe%26aufirst%3DH.%2BP.%2BV.%26atitle%3DKinase-targeted%2520cancer%2520therapies%253A%2520progress%252C%2520challenges%2520and%2520future%2520directions%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D48%26doi%3D10.1186%2Fs12943-018-0804-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolch, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halasz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granovskaya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kholodenko, B. N.</span></span> <span> </span><span class="NLM_article-title">The dynamic control of signal transduction networks in cancer cells</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1038/nrc3983</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnrc3983" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26289315" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlKkt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=515-527&author=W.+Kolchauthor=M.+Halaszauthor=M.+Granovskayaauthor=B.+N.+Kholodenko&title=The+dynamic+control+of+signal+transduction+networks+in+cancer+cells&doi=10.1038%2Fnrc3983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The dynamic control of signal transduction networks in cancer cells</span></div><div class="casAuthors">Kolch, Walter; Halasz, Melinda; Granovskaya, Marina; Kholodenko, Boris N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">515-527</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer is often considered a genetic disease.  However, much of the enormous plasticity of cancer cells to evolve different phenotypes, to adapt to challenging microenvironments and to withstand therapeutic assaults is encoded by the structure and spatiotemporal dynamics of signal transduction networks.  In this Review, we discuss recent concepts concerning how the rich signalling dynamics afforded by these networks are regulated and how they impinge on cancer cell proliferation, survival, invasiveness and drug resistance.  Understanding this dynamic circuitry by math. modeling could pave the way to new therapeutic approaches and personalized treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4ExQFCUoT6rVg90H21EOLACvtfcHk0li2sjB2d0tnnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlKkt7fJ&md5=b26429e328fecc69aacd7fa7b72c129f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrc3983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3983%26sid%3Dliteratum%253Aachs%26aulast%3DKolch%26aufirst%3DW.%26aulast%3DHalasz%26aufirst%3DM.%26aulast%3DGranovskaya%26aufirst%3DM.%26aulast%3DKholodenko%26aufirst%3DB.%2BN.%26atitle%3DThe%2520dynamic%2520control%2520of%2520signal%2520transduction%2520networks%2520in%2520cancer%2520cells%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D515%26epage%3D527%26doi%3D10.1038%2Fnrc3983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brognard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span> <span> </span><span class="NLM_article-title">Protein kinase signaling networks in cancer</span>. <i>Curr. Opin. Genet. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.gde.2010.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.gde.2010.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21123047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFKjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4-11&author=J.+Brognardauthor=T.+Hunter&title=Protein+kinase+signaling+networks+in+cancer&doi=10.1016%2Fj.gde.2010.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase signaling networks in cancer</span></div><div class="casAuthors">Brognard, John; Hunter, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Genetics & Development</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-11</span>CODEN:
                <span class="NLM_cas:coden">COGDET</span>;
        ISSN:<span class="NLM_cas:issn">0959-437X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein kinases orchestrate the activation of signaling cascades in response to extracellular and intracellular stimuli to control cell growth, proliferation, and survival.  The complexity of numerous intracellular signaling pathways is highlighted by the no. of kinases encoded by the human genome (539) and the plethora of phosphorylation sites identified in phosphoproteomic studies.  Perturbation of these signaling networks by mutations or abnormal protein expression underlies the cause of many diseases including cancer.  Recent RNAi screens and cancer genomic sequencing studies have revealed that many more kinases than anticipated contribute to tumorigenesis and are potential targets for inhibitor drug development intervention.  This review will highlight recent insights into known pathways essential for tumorigenesis and discuss exciting new pathways for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYS-FgMlgzm7Vg90H21EOLACvtfcHk0ljtxIGp1r6HLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFKjtbo%253D&md5=04d06a0c695fb57e72df0bd2b4bc25e3</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2010.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2010.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DBrognard%26aufirst%3DJ.%26aulast%3DHunter%26aufirst%3DT.%26atitle%3DProtein%2520kinase%2520signaling%2520networks%2520in%2520cancer%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2011%26volume%3D21%26spage%3D4%26epage%3D11%26doi%3D10.1016%2Fj.gde.2010.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span> <span> </span><span class="NLM_article-title">Targeting cancer with kinase inhibitors</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1780</span>– <span class="NLM_lpage">1789</span>, <span class="refDoi"> DOI: 10.1172/JCI76094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1172%2FJCI76094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=25932675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC2MjpvFGitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=1780-1789&author=S.+Grossauthor=R.+Rahalauthor=N.+Stranskyauthor=C.+Lengauerauthor=K.+P.+Hoeflich&title=Targeting+cancer+with+kinase+inhibitors&doi=10.1172%2FJCI76094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with kinase inhibitors</span></div><div class="casAuthors">Gross Stefan; Rahal Rami; Stransky Nicolas; Lengauer Christoph; Hoeflich Klaus P</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1780-9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kinase inhibitors have played an increasingly prominent role in the treatment of cancer and other diseases.  Currently, more than 25 oncology drugs that target kinases have been approved, and numerous additional therapeutics are in various stages of clinical evaluation.  In this Review, we provide an in-depth analysis of activation mechanisms for kinases in cancer, highlight recent successes in drug discovery, and demonstrate the clinical impact of selective kinase inhibitors.  We also describe the substantial progress that has been made in designing next-generation inhibitors to circumvent on-target resistance mechanisms, as well as ongoing strategies for combining kinase inhibitors in the clinic.  Last, there are numerous prospects for the discovery of novel kinase targets, and we explore cancer immunotherapy as a new and promising research area for studying kinase biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyaX6QAuw1pbFU0A1HOshhfW6udTcc2eYGwJfhRkGTKbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjpvFGitA%253D%253D&md5=b35d25dcf88d08a43ca3a0945c048111</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1172%2FJCI76094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI76094%26sid%3Dliteratum%253Aachs%26aulast%3DGross%26aufirst%3DS.%26aulast%3DRahal%26aufirst%3DR.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26atitle%3DTargeting%2520cancer%2520with%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2015%26volume%3D125%26spage%3D1780%26epage%3D1789%26doi%3D10.1172%2FJCI76094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sawyers, C.</span></span> <span> </span><span class="NLM_article-title">Targeted cancer therapy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">294</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1038/nature03095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnature03095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=15549090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=294-297&author=C.+Sawyers&title=Targeted+cancer+therapy&doi=10.1038%2Fnature03095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted cancer therapy</span></div><div class="casAuthors">Sawyers, Charles</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">294-297</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Disruption of the normal regulation of cell-cycle progression and division lies at the heart of the events leading to cancer.  Complex networks of regulatory factors, the tumor microenvironment and stress signals, such as those resulting from damaged DNA, dictate whether cancer cells proliferate or die.  Recent progress in understanding the mol. changes that underlie cancer development offer the prospect of specifically targeting malfunctioning mols. and pathways to achieve more effective and rational cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAGMTW2TYsubVg90H21EOLACvtfcHk0ljtxIGp1r6HLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtL8%253D&md5=5bd377d2c9ddcffc136d5cf181facd02</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature03095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03095%26sid%3Dliteratum%253Aachs%26aulast%3DSawyers%26aufirst%3DC.%26atitle%3DTargeted%2520cancer%2520therapy%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D294%26epage%3D297%26doi%3D10.1038%2Fnature03095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kroschinsky, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolzel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Bonin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kochanek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiehl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schellongowski, P.</span></span> <span> </span><span class="NLM_article-title">New drugs, new toxicities: severe side effects of modern targeted and immunotherapy of cancer and their management</span>. <i>Crit. Care.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">89</span>, <span class="refDoi"> DOI: 10.1186/s13054-017-1678-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2Fs13054-017-1678-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=28407743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC1cvns1Ckuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=89&author=F.+Kroschinskyauthor=F.+Stolzelauthor=S.+von+Boninauthor=G.+Beutelauthor=M.+Kochanekauthor=M.+Kiehlauthor=P.+Schellongowski&title=New+drugs%2C+new+toxicities%3A+severe+side+effects+of+modern+targeted+and+immunotherapy+of+cancer+and+their+management&doi=10.1186%2Fs13054-017-1678-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">New drugs, new toxicities: severe side effects of modern targeted and immunotherapy of cancer and their management</span></div><div class="casAuthors">Kroschinsky Frank; Stolzel Friedrich; von Bonin Simone; Beutel Gernot; Kochanek Matthias; Kiehl Michael; Schellongowski Peter</div><div class="citationInfo"><span class="NLM_cas:title">Critical care (London, England)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Pharmacological and cellular treatment of cancer is changing dramatically with benefits for patient outcome and comfort, but also with new toxicity profiles.  The majority of adverse events can be classified as mild or moderate, but severe and life-threatening complications requiring ICU admission also occur.  This review will focus on pathophysiology, symptoms, and management of these events based on the available literature.While standard antineoplastic therapy is associated with immunosuppression and infections, some of the recent approaches induce overwhelming inflammation and autoimmunity.  Cytokine-release syndrome (CRS) describes a complex of symptoms including fever, hypotension, and skin reactions as well as lab abnormalities.  CRS may occur after the infusion of monoclonal or bispecific antibodies (MABs, BABs) targeting immune effectors and tumor cells and is a major concern in recipients of chimeric antigen receptor (CAR) modified T lymphocytes as well.  BAB and CAR T-cell treatment may also be compromised by central nervous system (CNS) toxicities such as encephalopathy, cerebellar alteration, disturbed consciousness, or seizures.  While CRS is known to be induced by exceedingly high levels of inflammatory cytokines, the pathophysiology of CNS events is still unclear.  Treatment with antibodies against inhibiting immune checkpoints can lead to immune-related adverse events (IRAEs); colitis, diarrhea, and endocrine disorders are often the cause for ICU admissions.Respiratory distress is the main reason for ICU treatment in cancer patients and is attributable to infectious agents in most cases.  In addition, some of the new drugs are reported to cause non-infectious lung complications.  While drug-induced interstitial pneumonitis was observed in a substantial number of patients treated with phosphoinositol-3-kinase inhibitors, IRAEs may also affect the lungs.Inhibitors of angiogenetic pathways have increased the antineoplastic portfolio.  However, vessel formation is also essential for regeneration and tissue repair.  Therefore, severe vascular side effects, including thromboembolic events, gastrointestinal bleeding or perforation, hypertension, and congestive heart failure, compromise antitumor efficacy.The limited knowledge of the pathophysiology and management of life-threatening complications relating to new cancer drugs presents a need to provide ICU staff, oncologists, and organ specialists with evidence-based algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRDBx0_VHiayEloaUqgkFHkfW6udTcc2eYGwJfhRkGTKbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvns1Ckuw%253D%253D&md5=0d3e40160efb87574938d8886f4aa6f1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1186%2Fs13054-017-1678-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13054-017-1678-1%26sid%3Dliteratum%253Aachs%26aulast%3DKroschinsky%26aufirst%3DF.%26aulast%3DStolzel%26aufirst%3DF.%26aulast%3Dvon%2BBonin%26aufirst%3DS.%26aulast%3DBeutel%26aufirst%3DG.%26aulast%3DKochanek%26aufirst%3DM.%26aulast%3DKiehl%26aufirst%3DM.%26aulast%3DSchellongowski%26aufirst%3DP.%26atitle%3DNew%2520drugs%252C%2520new%2520toxicities%253A%2520severe%2520side%2520effects%2520of%2520modern%2520targeted%2520and%2520immunotherapy%2520of%2520cancer%2520and%2520their%2520management%26jtitle%3DCrit.%2520Care.%26date%3D2017%26volume%3D21%26spage%3D89%26doi%3D10.1186%2Fs13054-017-1678-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lh4Ckwt7-Nh7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.drudis.2015.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26210956" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=5-10&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=Small-molecule+kinase+inhibitors%3A+an+analysis+of+FDA-approved+drugs&doi=10.1016%2Fj.drudis.2015.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule kinase inhibitors: an analysis of FDA-approved drugs</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors (SMKIs), 28 of which are approved by the US Food and Drug Administration (FDA), have been actively pursued as promising targeted therapeutics.  Here, we assess the key structural and physicochem. properties, target selectivity and mechanism of function, and therapeutic indications of these approved inhibitors.  Our anal. showed that >30% of approved SMKIs have a mol. wt. (MW) exceeding 500 and all have a total ring count of between three and five.  The assumption that type II inhibitors tend to be more selective than type I inhibitors has been proved to be unreliable.  Although previous SMKI research was concd. on tyrosine kinase inhibitors for cancer treatment, recent progress indicates diversification of SMKI research in terms of new targets, mechanistic types, and therapeutic indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZrY68ZOI2xLVg90H21EOLACvtfcHk0lh4Ckwt7-Nh7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gru73P&md5=228524fa464393053ecbfac19e994929</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DSmall-molecule%2520kinase%2520inhibitors%253A%2520an%2520analysis%2520of%2520FDA-approved%2520drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D5%26epage%3D10%26doi%3D10.1016%2Fj.drudis.2015.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanssen, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demissie, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svenson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blinov, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repisky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svendsen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruud, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elyashberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspars, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaksson, J.</span></span> <span> </span><span class="NLM_article-title">A combined atomic force microscopy and computational approach for the structural elucidation of breitfussin A and B: Highly modified halogenated dipeptides from Thuiaria breitfussi</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">12238</span>– <span class="NLM_lpage">12241</span>, <span class="refDoi"> DOI: 10.1002/anie.201203960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1002%2Fanie.201203960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFyjsb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=12238-12241&author=K.+O.+Hanssenauthor=B.+Schulerauthor=A.+J.+Williamsauthor=T.+B.+Demissieauthor=E.+Hansenauthor=J.+H.+Andersenauthor=J.+Svensonauthor=K.+Blinovauthor=M.+Repiskyauthor=F.+Mohnauthor=G.+Meyerauthor=J.+S.+Svendsenauthor=K.+Ruudauthor=M.+Elyashbergauthor=L.+Grossauthor=M.+Jasparsauthor=J.+Isaksson&title=A+combined+atomic+force+microscopy+and+computational+approach+for+the+structural+elucidation+of+breitfussin+A+and+B%3A+Highly+modified+halogenated+dipeptides+from+Thuiaria+breitfussi&doi=10.1002%2Fanie.201203960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A Combined Atomic Force Microscopy and Computational Approach for the Structural Elucidation of Breitfussin A and B: Highly Modified Halogenated Dipeptides from Thuiaria breitfussi</span></div><div class="casAuthors">Hanssen, Kine O.; Schuler, Bruno; Williams, Antony J.; Demissie, Taye B.; Hansen, Espen; Andersen, Jeanette H.; Svenson, Johan; Blinov, Kirill; Repisky, Michal; Mohn, Fabian; Meyer, Gerhard; Svendsen, John-Sigurd; Ruud, Kenneth; Elyashberg, Mikhail; Gross, Leo; Jaspars, Marcel; Isaksson, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">12238-12241</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The subject of this study was the Arctic hydrozoan Thuiaria breitfussi (Family Sertulariidae).  The sample of Thuiaria breitfussi was collected from Bear island in 2007 and extd. with MeOH/CH2Cl2.  A series of liq.-liq. extns. followed by mass-guided HPLC purifn. yielded 6.2 mg of purified breitfussin A (1) and 4.0 mg breitfussin B (2).  The breitfussins are unusual structures contg. a combination of indole-oxazole-pyrrole units.  Given the limited quantity isolated, x-ray crystallog. was not possible, and structure detn. using spectroscopic data was unable to propose a single unequivocal structure consistent with all the data.  The structure was therefore supported by a combination of AFM, CASE, and DFT calcns., none of which were able to propose a unique soln. individually.  Remarkably, AFM could be used to det. all the connection positions of the cyclic systems as well as those of the substituent groups (MeO, Br, and I); this information is difficult to obtain with other techniques.  Moreover, different AFM contrast above halogen atoms and the MeO group indicate chem. sensitivity within individual mols.  Using such powerful methods on limited quantities of bioactive natural products with complex structures will become more common as scientists begin to access unusual taxa from extreme locations with unique chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeEzw_ZXy0xbVg90H21EOLACvtfcHk0lh4Ckwt7-Nh7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFyjsb%252FL&md5=9b7e9778ae429f3aa49df0a65ce1b441</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fanie.201203960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201203960%26sid%3Dliteratum%253Aachs%26aulast%3DHanssen%26aufirst%3DK.%2BO.%26aulast%3DSchuler%26aufirst%3DB.%26aulast%3DWilliams%26aufirst%3DA.%2BJ.%26aulast%3DDemissie%26aufirst%3DT.%2BB.%26aulast%3DHansen%26aufirst%3DE.%26aulast%3DAndersen%26aufirst%3DJ.%2BH.%26aulast%3DSvenson%26aufirst%3DJ.%26aulast%3DBlinov%26aufirst%3DK.%26aulast%3DRepisky%26aufirst%3DM.%26aulast%3DMohn%26aufirst%3DF.%26aulast%3DMeyer%26aufirst%3DG.%26aulast%3DSvendsen%26aufirst%3DJ.%2BS.%26aulast%3DRuud%26aufirst%3DK.%26aulast%3DElyashberg%26aufirst%3DM.%26aulast%3DGross%26aufirst%3DL.%26aulast%3DJaspars%26aufirst%3DM.%26aulast%3DIsaksson%26aufirst%3DJ.%26atitle%3DA%2520combined%2520atomic%2520force%2520microscopy%2520and%2520computational%2520approach%2520for%2520the%2520structural%2520elucidation%2520of%2520breitfussin%2520A%2520and%2520B%253A%2520Highly%2520modified%2520halogenated%2520dipeptides%2520from%2520Thuiaria%2520breitfussi%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D12238%26epage%3D12241%26doi%3D10.1002%2Fanie.201203960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guttormsen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedberg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, A.</span></span> <span> </span><span class="NLM_article-title">A concise total synthesis of breitfussin A and B</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1021/ol503348n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol503348n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOgsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=122-125&author=S.+K.+Pandeyauthor=Y.+Guttormsenauthor=B.+E.+Haugauthor=C.+Hedbergauthor=A.+Bayer&title=A+concise+total+synthesis+of+breitfussin+A+and+B&doi=10.1021%2Fol503348n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A Concise Total Synthesis of Breitfussin A and B</span></div><div class="casAuthors">Pandey, Sunil Kumar; Guttormsen, Yngve; Haug, Bengt Erik; Hedberg, Christian; Bayer, Annette</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-125</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first total synthesis of breitfussin A and B is described.  The approach features two palladium-catalyzed cross-couplings installing the indole and pyrrole onto the oxazole core and selective lithiation/iodination of a common indole-oxazole fragment providing 2,4-diiodinated or 2-iodinated oxazoles as potential precursors for breitfussin A and B, resp.  An unexpected acid promoted deiodination was utilized in the synthesis of breitfussin B.  Comparison of the synthetic material with previously reported spectral data of isolated breitfussin A and B verified the structure of the breitfussin framework.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVmyJQcWaG1rVg90H21EOLACvtfcHk0li-bhrVlv9u0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOgsrbL&md5=af3d7e450cada9ca23a105f205d365d4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fol503348n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol503348n%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DS.%2BK.%26aulast%3DGuttormsen%26aufirst%3DY.%26aulast%3DHaug%26aufirst%3DB.%2BE.%26aulast%3DHedberg%26aufirst%3DC.%26aulast%3DBayer%26aufirst%3DA.%26atitle%3DA%2520concise%2520total%2520synthesis%2520of%2520breitfussin%2520A%2520and%2520B%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D122%26epage%3D125%26doi%3D10.1021%2Fol503348n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benayahu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleyer, M.</span></span> <span> </span><span class="NLM_article-title">Phorbazoles A-D, novel chlorinated phenylpyrrolyloxazoles from the marine sponge phorbas aff. clathrata</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2589</span>– <span class="NLM_lpage">2592</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(00)77179-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2FS0040-4039%2800%2977179-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK2cXkslylu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1994&pages=2589-2592&author=A.+Rudiauthor=Z.+Steinauthor=S.+Greenauthor=I.+Goldbergauthor=Y.+Kashmanauthor=Y.+Benayahuauthor=M.+Schleyer&title=Phorbazoles+A-D%2C+novel+chlorinated+phenylpyrrolyloxazoles+from+the+marine+sponge+phorbas+aff.+clathrata&doi=10.1016%2FS0040-4039%2800%2977179-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Phorbazoles A-D, novel chlorinated phenylpyrrolyloxazoles from the marine sponge Phorbas aff. clathrata</span></div><div class="casAuthors">Rudi, A.; Stein, Z.; Green, S.; Goldberg, I.; Kashman, Y.; Benayahu, Y.; Schleyer, M.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2589-92</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">Four new chlorinated phenylpyrrolyloxazoles, phorbazoles A-D (I-IV) were isolated from the sponge P. clathrata.  The structure of phorbazole A was unambiguously detd. by x-ray diffraction studies of its di-Me deriv..  The structures of compds. II-IV were detd. by extensive use of 1- and 2-dimensional NMR techniques.  Phorbazoles A-D represent a new class of marine alkaloids embodying the previously unprecedented, chlorinated pyrrole moiety, from marine sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrt4_HTLpSI6LVg90H21EOLACvtfcHk0li-bhrVlv9u0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXkslylu7g%253D&md5=875c18bda9ba9a30fb0ec209a34d9602</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2800%2977179-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252800%252977179-6%26sid%3Dliteratum%253Aachs%26aulast%3DRudi%26aufirst%3DA.%26aulast%3DStein%26aufirst%3DZ.%26aulast%3DGreen%26aufirst%3DS.%26aulast%3DGoldberg%26aufirst%3DI.%26aulast%3DKashman%26aufirst%3DY.%26aulast%3DBenayahu%26aufirst%3DY.%26aulast%3DSchleyer%26aufirst%3DM.%26atitle%3DPhorbazoles%2520A-D%252C%2520novel%2520chlorinated%2520phenylpyrrolyloxazoles%2520from%2520the%2520marine%2520sponge%2520phorbas%2520aff.%2520clathrata%26jtitle%3DTetrahedron%2520Lett.%26date%3D1994%26volume%3D35%26spage%3D2589%26epage%3D2592%26doi%3D10.1016%2FS0040-4039%2800%2977179-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lindquist, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenical, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Duyne, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J.</span></span> <span> </span><span class="NLM_article-title">Isolation and structure determination of diazonamides A and B, unusual cytotoxic metabolites from the marine ascidian Diazona chinensis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2303</span>– <span class="NLM_lpage">2304</span>, <span class="refDoi"> DOI: 10.1021/ja00006a060</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00006a060" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK3MXhtFyrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=1991&pages=2303-2304&author=N.+Lindquistauthor=W.+Fenicalauthor=G.+D.+Van+Duyneauthor=J.+Clardy&title=Isolation+and+structure+determination+of+diazonamides+A+and+B%2C+unusual+cytotoxic+metabolites+from+the+marine+ascidian+Diazona+chinensis&doi=10.1021%2Fja00006a060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and structure determination of diazonamides A and B, unusual cytotoxic metabolites from the marine ascidian Diazona chinensis</span></div><div class="casAuthors">Lindquist, Niels; Fenical, William; Van Duyne, Gregory D.; Clardy, Jon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2303-4</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Two novel cytotoxic metabolites, diazonamides A and B (I and II, resp.), were isolated from the colonial marine ascidian D. chinensis from Siquijor Is., Philippines.  The structures were elucidated by a combination of single crystal x-ray diffraction and spectroscopic techniques.  The diazonamides represent an entirely new class of halogenated, highly unsatd. cyclic peptides contg. derivs. of ≥3 common amino acids: a 3,4,5-trisubstituted L-tyrosine, a tryptophan substituted at the 2 and 4 positions of the indole, and an L-valine.  These fragments and others of less clear biosynthetic origin have been assembled into an extremely rigid framework.  Diazonamide A has potent in vitro activity against HCT-116 human colon carcinoma and B-16 murine melanoma cancer cell lines with IC50 values <15 ng/mL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0cV4-hmu5k7Vg90H21EOLACvtfcHk0li-bhrVlv9u0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXhtFyrsbc%253D&md5=db7c81b87c468c609847c7fee82cc512</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fja00006a060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00006a060%26sid%3Dliteratum%253Aachs%26aulast%3DLindquist%26aufirst%3DN.%26aulast%3DFenical%26aufirst%3DW.%26aulast%3DVan%2BDuyne%26aufirst%3DG.%2BD.%26aulast%3DClardy%26aufirst%3DJ.%26atitle%3DIsolation%2520and%2520structure%2520determination%2520of%2520diazonamides%2520A%2520and%2520B%252C%2520unusual%2520cytotoxic%2520metabolites%2520from%2520the%2520marine%2520ascidian%2520Diazona%2520chinensis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1991%26volume%3D113%26spage%3D2303%26epage%3D2304%26doi%3D10.1021%2Fja00006a060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsunaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saito, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiuchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuda, Y.</span></span> <span> </span><span class="NLM_article-title">Martefragin A, a novel indole alkaloid isolated from red alga, inhibits lipid peroxidation</span>. <i>Chem. Pharm. Bull.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1527</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1248/cpb.46.1527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1248%2Fcpb.46.1527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=9810689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK1cXntVSltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1998&pages=1527-1529&author=S.+Takahashiauthor=T.+Matsunagaauthor=C.+Hasegawaauthor=H.+Saitoauthor=D.+Fujitaauthor=F.+Kiuchiauthor=Y.+Tsuda&title=Martefragin+A%2C+a+novel+indole+alkaloid+isolated+from+red+alga%2C+inhibits+lipid+peroxidation&doi=10.1248%2Fcpb.46.1527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Martefragin A, a novel indole alkaloid isolated from red alga, inhibits lipid peroxidation</span></div><div class="casAuthors">Takahashi, Satoshi; Matsunaga, Takayuki; Hasegawa, Chika; Saito, Haruo; Fujita, Daisuke; Kiuchi, Fumiyuki; Tsuda, Yoshisuke</div><div class="citationInfo"><span class="NLM_cas:title">Chemical & Pharmaceutical Bulletin</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1527-1529</span>CODEN:
                <span class="NLM_cas:coden">CPBTAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-2363</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Japan</span>)
        </div><div class="casAbstract">Martefragin A (I), a novel indole alkaloid, was isolated from a red alga, Martensia fragilis, by repeated column chromatog.  The structure of I was elucidated on the basis of spectral anal. of its Me ester, including 1H- and 13C-NMR, 1H-1H correlation spectroscopy (COSY), and 13C-1H COSY.  A single crystal X-ray anal. of the hydrochloride of I confirmed the assignment.  Martefragin A (I) showed inhibitory activity on NADPH-dependent lipid peroxidn. in rat liver microsomes.  The IC50 values of I, α-tocopherol and ascorbic acid were 2.8, 87 and 200 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq25gztulBuw7Vg90H21EOLACvtfcHk0lh2_365jQ0Z2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntVSltrk%253D&md5=af783197b8a39e5774c0e3391709c518</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1248%2Fcpb.46.1527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.46.1527%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DS.%26aulast%3DMatsunaga%26aufirst%3DT.%26aulast%3DHasegawa%26aufirst%3DC.%26aulast%3DSaito%26aufirst%3DH.%26aulast%3DFujita%26aufirst%3DD.%26aulast%3DKiuchi%26aufirst%3DF.%26aulast%3DTsuda%26aufirst%3DY.%26atitle%3DMartefragin%2520A%252C%2520a%2520novel%2520indole%2520alkaloid%2520isolated%2520from%2520red%2520alga%252C%2520inhibits%2520lipid%2520peroxidation%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1998%26volume%3D46%26spage%3D1527%26epage%3D1529%26doi%3D10.1248%2Fcpb.46.1527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herald, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davenport, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettit, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. M.</span></span> <span> </span><span class="NLM_article-title">Isolation of Labradorins 1 and 2 from Pseudomonas syringae pv. coronafaciens</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1793</span>– <span class="NLM_lpage">1797</span>, <span class="refDoi"> DOI: 10.1021/np020173x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np020173x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvFCntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2002&pages=1793-1797&author=G.+R.+Pettitauthor=J.+C.+Knightauthor=D.+L.+Heraldauthor=R.+Davenportauthor=R.+K.+Pettitauthor=B.+E.+Tuckerauthor=J.+M.+Schmidt&title=Isolation+of+Labradorins+1+and+2+from+Pseudomonas+syringae+pv.+coronafaciens&doi=10.1021%2Fnp020173x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation of labradorins 1 and 2 from Pseudomonas syringae pv. coronafaciens</span></div><div class="casAuthors">Pettit, George R.; Knight, John C.; Herald, Delbert L.; Davenport, Richard; Pettit, Robin K.; Tucker, Bruce E.; Schmidt, Jean M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1793-1797</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Investigation of Pseudomonas syringae pv. coronafaciens cancer cell growth inhibitory constituents led to the isolation of 2-isobutyl-5-(3-indolyl)oxazole (I) and 2-n-pentyl-5-(3-indolyl)oxazole (II), designated labradorins 1 and 2, resp., related to pimprinine (III).  The structures were deduced by spectroscopic techniques and X-ray crystal structure detns.  I afforded GI50 μg/mL values of 9.8 and 6.2 against the human cancer cell lines NCI-H 460 (lung-NSC) and BXPC-3 (pancreas-a).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmmEqRGEdUcbVg90H21EOLACvtfcHk0lh2_365jQ0Z2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvFCntbg%253D&md5=c532e55f7a6bb01847729726564770ca</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fnp020173x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp020173x%26sid%3Dliteratum%253Aachs%26aulast%3DPettit%26aufirst%3DG.%2BR.%26aulast%3DKnight%26aufirst%3DJ.%2BC.%26aulast%3DHerald%26aufirst%3DD.%2BL.%26aulast%3DDavenport%26aufirst%3DR.%26aulast%3DPettit%26aufirst%3DR.%2BK.%26aulast%3DTucker%26aufirst%3DB.%2BE.%26aulast%3DSchmidt%26aufirst%3DJ.%2BM.%26atitle%3DIsolation%2520of%2520Labradorins%25201%2520and%25202%2520from%2520Pseudomonas%2520syringae%2520pv.%2520coronafaciens%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2002%26volume%3D65%26spage%3D1793%26epage%3D1797%26doi%3D10.1021%2Fnp020173x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span> <i>Total synthesis of CPI-2081, breitfussin B and synthetic studies towards myriastramide C and goadsporin</i>; <span class="NLM_publisher-name">University of East Anglia</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=K.+Liu&title=Total+synthesis+of+CPI-2081%2C+breitfussin+B+and+synthetic+studies+towards+myriastramide+C+and+goadsporin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DK.%26btitle%3DTotal%2520synthesis%2520of%2520CPI-2081%252C%2520breitfussin%2520B%2520and%2520synthetic%2520studies%2520towards%2520myriastramide%2520C%2520and%2520goadsporin%26pub%3DUniversity%2520of%2520East%2520Anglia%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khan, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. S.</span></span> <span> </span><span class="NLM_article-title">Synthesis of breitfussin B by late-stage bromination</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3718</span>– <span class="NLM_lpage">3721</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01702</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01702" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2ltr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=3718-3721&author=A.+H.+Khanauthor=J.+S.+Chen&title=Synthesis+of+breitfussin+B+by+late-stage+bromination&doi=10.1021%2Facs.orglett.5b01702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Breitfussin B by Late-Stage Bromination</span></div><div class="casAuthors">Khan, Akbar H.; Chen, Jason S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3718-3721</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The breitfussins are halogenated natural products (also considered as highly oxidized dipeptides) whose structures were detd. with the assistance of at.-force microscopy.  The site selectivity of N-bromosuccinimide-mediated bromination of a model breitfussin core was found to be strongly dependent on solvent selection; use of acetone led to oxazole bromination, and use of a pyridine-contg. mixt. led to pyrrole bromination.  This tunable site-selective bromination was used in a protecting group-free synthesis of breitfussin B that proceeded in 9.2% yield over 12 reactions and five chromatog. sepns.  A bromooxazole analog of breitfussin A was also prepd. by late-stage bromination but isomerized on silica gel to form breitfussin B.  This isomerization appeared to proceed through a unimol. pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0ExPvuykpHLVg90H21EOLACvtfcHk0lh2_365jQ0Z2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2ltr7E&md5=80715e9d80090de9cea0e4e72263e397</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01702%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DA.%2BH.%26aulast%3DChen%26aufirst%3DJ.%2BS.%26atitle%3DSynthesis%2520of%2520breitfussin%2520B%2520by%2520late-stage%2520bromination%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D3718%26epage%3D3721%26doi%3D10.1021%2Facs.orglett.5b01702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cambeiro, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlsten, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrosa, I.</span></span> <span> </span><span class="NLM_article-title">Au-catalyzed cross-coupling of arenes via double C–H activation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">15636</span>– <span class="NLM_lpage">15639</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b10593</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b10593" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFylur7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=15636-15639&author=X.+C.+Cambeiroauthor=N.+Ahlstenauthor=I.+Larrosa&title=Au-catalyzed+cross-coupling+of+arenes+via+double+C%E2%80%93H+activation&doi=10.1021%2Fjacs.5b10593"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Au-Catalyzed Cross-Coupling of Arenes via Double C-H Activation</span></div><div class="casAuthors">Cambeiro, Xacobe C.; Ahlsten, Nanna; Larrosa, Igor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15636-15639</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first methodol. for Au(I/III)-catalyzed oxidative cross-coupling of arenes via double C-H activation has been developed.  The reaction is fully selective for the cross-coupling between electron-rich hetero-/ carbocyclic arenes and electron-poor arenes bearing relatively acidic C-H bonds.  The inherently high cross-selectivity of the system obviates the need for directing groups or a large excess of one of the coupling partners.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFaWknTtjBJbVg90H21EOLACvtfcHk0lgCa3ZQWyoL6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFylur7L&md5=1e18a4db9bcf4491a6d6d09a9fc449eb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b10593&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b10593%26sid%3Dliteratum%253Aachs%26aulast%3DCambeiro%26aufirst%3DX.%2BC.%26aulast%3DAhlsten%26aufirst%3DN.%26aulast%3DLarrosa%26aufirst%3DI.%26atitle%3DAu-catalyzed%2520cross-coupling%2520of%2520arenes%2520via%2520double%2520C%25E2%2580%2593H%2520activation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D15636%26epage%3D15639%26doi%3D10.1021%2Fjacs.5b10593" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelkey, E. T.</span></span> <span> </span><span class="NLM_article-title">Metalation of Indole</span>. In  <i>Heterocyclic Scaffolds II: Reactions and Applications of Indoles</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribble, G. W.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">191</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1007%2F7081_2010_56" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=141-191&author=E.+T.+Pelkeyauthor=G.+W.+Gribble&title=Heterocyclic+Scaffolds+II%3A+Reactions+and+Applications+of+Indoles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2F7081_2010_56&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F7081_2010_56%26sid%3Dliteratum%253Aachs%26aulast%3DPelkey%26aufirst%3DE.%2BT.%26atitle%3DMetalation%2520of%2520Indole%26btitle%3DHeterocyclic%2520Scaffolds%2520II%253A%2520Reactions%2520and%2520Applications%2520of%2520Indoles%26aulast%3DGribble%26aufirst%3DG.%2BW.%26pub%3DSpringer%26date%3D2010%26spage%3D141%26epage%3D191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parton, R. L.</span></span> <span> </span><span class="NLM_article-title">Lithiation of methoxyindoles</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1021/jo00863a044</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00863a044" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaE28XktlOmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1976&pages=163-165&author=R.+J.+Sundbergauthor=R.+L.+Parton&title=Lithiation+of+methoxyindoles&doi=10.1021%2Fjo00863a044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Lithiation of methoxyindoles</span></div><div class="casAuthors">Sundberg, Richard J.; Parton, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">163-5</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">1-Methyl-5-methoxyindole was lithiated with BuLi and Me3CLi.  The indole ring is lithiated at C-2 by Me3CLi but lithiation with BuLi resulted in lithiation at C-2, C-4, and C-6.  Lithiation of 1-benzenesulfonyl-5-methoxyindole and 1-benzenesulfonyl-6-methoxyindole with Me3CLi occurred selectively at C-2.  The various lithio indoles form the expected adducts with carbonyl compds. such as 2-pyridinecarboxaldehyde, 3-pyridinecarboxaldehyde, and 4-acetylpyridine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp91r3rO5O8irVg90H21EOLACvtfcHk0lgCa3ZQWyoL6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XktlOmsg%253D%253D&md5=77227dba0ba1018a2b22c64cab24118e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjo00863a044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00863a044%26sid%3Dliteratum%253Aachs%26aulast%3DSundberg%26aufirst%3DR.%2BJ.%26aulast%3DParton%26aufirst%3DR.%2BL.%26atitle%3DLithiation%2520of%2520methoxyindoles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1976%26volume%3D41%26spage%3D163%26epage%3D165%26doi%3D10.1021%2Fjo00863a044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snieckus, V.</span></span> <span> </span><span class="NLM_article-title">Directed ortho metalation. Tertiary amide and O-carbamate directors in synthetic strategies for polysubstituted aromatics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">879</span>– <span class="NLM_lpage">933</span>, <span class="refDoi"> DOI: 10.1021/cr00104a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00104a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK3cXlsFWqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1990&pages=879-933&author=V.+Snieckus&title=Directed+ortho+metalation.+Tertiary+amide+and+O-carbamate+directors+in+synthetic+strategies+for+polysubstituted+aromatics&doi=10.1021%2Fcr00104a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Directed ortho metalation.  Tertiary amide and O-carbamate directors in synthetic strategies for polysubstituted aromatics</span></div><div class="casAuthors">Snieckus, Victor</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">879-933</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 214 refs.  After a brief overview of general aspects of the directed ortho metalation reaction, the utility and applications of tertiary amide and carbamate directed metalation groups are systematically and comprehensively developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpthWLi2I-h8bVg90H21EOLACvtfcHk0lgCa3ZQWyoL6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXlsFWqsrc%253D&md5=21de0a0b1b08e9241ec117dde47d9aa1</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcr00104a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00104a001%26sid%3Dliteratum%253Aachs%26aulast%3DSnieckus%26aufirst%3DV.%26atitle%3DDirected%2520ortho%2520metalation.%2520Tertiary%2520amide%2520and%2520O-carbamate%2520directors%2520in%2520synthetic%2520strategies%2520for%2520polysubstituted%2520aromatics%26jtitle%3DChem.%2520Rev.%26date%3D1990%26volume%3D90%26spage%3D879%26epage%3D933%26doi%3D10.1021%2Fcr00104a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nirogi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwarampudi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kota, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubey, P.K.</span></span> <span> </span><span class="NLM_article-title">Synthesis and structure activity relationship of rigidized indolyl pyrrolidine derivatives as 5-HT6 receptor ligands</span>. <i>Asian J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">9293</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.14233/ajchem.2013.15443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.14233%2Fajchem.2013.15443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOkurfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2013&pages=9293-9298&author=R.+Nirogiauthor=A.+Dwarampudiauthor=V.+Bhattaauthor=L.+Kotaauthor=P.K.+Dubey&title=Synthesis+and+structure+activity+relationship+of+rigidized+indolyl+pyrrolidine+derivatives+as+5-HT6+receptor+ligands&doi=10.14233%2Fajchem.2013.15443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and structure activity relationship of rigidized indolyl pyrrolidine derivatives as 5-HT6 receptor ligands</span></div><div class="casAuthors">Nirogi, Ramakrishna; Dwarampudi, Adireddy; Bhatta, Venugopalarao; Kota, Laxman; Dubey, P. K.</div><div class="citationInfo"><span class="NLM_cas:title">Asian Journal of Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">9293-9298</span>CODEN:
                <span class="NLM_cas:coden">AJCHEW</span>;
        ISSN:<span class="NLM_cas:issn">0970-7077</span>.
    
            (<span class="NLM_cas:orgname">Asian Journal of Chemistry</span>)
        </div><div class="casAbstract">A novel series of rigidized indolyl pyrrolidine derivs. were designed by constraining the tryptamine nitrogen through α and β carbons.  All the synthesized derivs. showed moderate affinities at 5-HT6R when tested in in vitro binding assay.  Synthesis, structure activity relation (SAR), pharmacokinetic profile and in vivo efficacy of a selected compd. is the subject matter of this communication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquuu0k6niSTbVg90H21EOLACvtfcHk0lgxeAW8j-lhvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOkurfL&md5=bfd5c8e436f51cfe195e95a30a8ed684</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.14233%2Fajchem.2013.15443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.14233%252Fajchem.2013.15443%26sid%3Dliteratum%253Aachs%26aulast%3DNirogi%26aufirst%3DR.%26aulast%3DDwarampudi%26aufirst%3DA.%26aulast%3DBhatta%26aufirst%3DV.%26aulast%3DKota%26aufirst%3DL.%26aulast%3DDubey%26aufirst%3DP.K.%26atitle%3DSynthesis%2520and%2520structure%2520activity%2520relationship%2520of%2520rigidized%2520indolyl%2520pyrrolidine%2520derivatives%2520as%25205-HT6%2520receptor%2520ligands%26jtitle%3DAsian%2520J.%2520Chem.%26date%3D2013%26volume%3D25%26spage%3D9293%26epage%3D9298%26doi%3D10.14233%2Fajchem.2013.15443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snieckus, V.</span></span> <span> </span><span class="NLM_article-title">Benzenoid ring functionalization of indoles and tryptophols via combined directed ortho metalation-cross coupling methodology</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1484</span>– <span class="NLM_lpage">1485</span>, <span class="refDoi"> DOI: 10.1021/jo00111a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00111a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK2MXktlektro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1995&pages=1484-1485&author=E.+J.+Griffenauthor=D.+G.+Roeauthor=V.+Snieckus&title=Benzenoid+ring+functionalization+of+indoles+and+tryptophols+via+combined+directed+ortho+metalation-cross+coupling+methodology&doi=10.1021%2Fjo00111a003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Benzenoid Ring Functionalization of Indoles and Tryptophols via Combined Directed Ortho Metalation-Cross Coupling Methodology</span></div><div class="casAuthors">Griffen, Edward J.; Roe, David G.; Snieckus, Victor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1484-5</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Metalation (s-BuLi/TMEDA/THF/-78°) of N,N-diethyl-5-(1-tert-butyldimethylsilylindolyl)carbamate followed by treatment with electrophiles (E = MeI, DMF, cyclohexanone, EtOCOCl, TsN3, camphorsulfonyloxaziridine, C2Cl6, I2, PhSO2SBu-t, and TMSCl) provides a general synthesis of 4-substituted products.  Further metalation of N,N-diethyl-5-(1-tert-butyldimethylsilyl-4-trimethylsilylindolyl)carbamate under the same conditions but -90° followed by electrophile quench leads to 6-substituted products and by anionic carbamoyl rearrangement to 5-acetoxy-1-tert-butyldimethylsilyl-6-N,N-diethylcarboxamido-4-trimethylsilylindole.  Transmetalation of N,N-diethyl-5-(1-tert-butyldimethylsilyl-4-lithioindolyl)carbamate into the corresponding zinc species, followed by cross coupling with 3-bromopyridine under Pd(0) catalysis leads to N,N-diethyl-5-[1-tert-butyldimethylsilyl-4-(3-pyridyl)indolyl]carbamate.  Cross coupling of N,N-diethyl-5-(1-tert-butyldimethylsilylindolyl)carbamate with Grignard reagents (PhMgX and TMSCH2MgX) under Ni(0) catalysis leads to 5-substituted products.  Tryptophol 5-O-carbamates [N,N-diethyl-5-[1-tert-butyldimethylsilyl-3-(2-hydroxyethyl)indolyl]carbamate and N,N-diethyl-5-[1-tert-butyldimethylsilyl-3-(2-tert-butyldimethylsilyloxyethyl)indolyl]carbamate], prepd. via N,N-diethyl-5-(1-tert-butyldimethylsilyl-3-bromoindolyl)carbamate, when metalated under the same conditions followed by treatment with DMF, lead to the corresponding 6-formylated products.  Decarbamoylation of N,N-diethyl-5-(1-tert-butyldimethylsilylindolyl)carbamate to the corresponding phenol and its conversion into 5-methoxy and 5-triflate derivs. was also carried out.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvh46BQ2R-DbVg90H21EOLACvtfcHk0lgxeAW8j-lhvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXktlektro%253D&md5=903834bb86bb6795382d5ed4f112b867</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjo00111a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00111a003%26sid%3Dliteratum%253Aachs%26aulast%3DGriffen%26aufirst%3DE.%2BJ.%26aulast%3DRoe%26aufirst%3DD.%2BG.%26aulast%3DSnieckus%26aufirst%3DV.%26atitle%3DBenzenoid%2520ring%2520functionalization%2520of%2520indoles%2520and%2520tryptophols%2520via%2520combined%2520directed%2520ortho%2520metalation-cross%2520coupling%2520methodology%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D1484%26epage%3D1485%26doi%3D10.1021%2Fjo00111a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riss, T. L.</span>; <span class="NLM_string-name">Moravec, R. A.</span>; <span class="NLM_string-name">Niles, A. L.</span>; <span class="NLM_string-name">Duellman, S.</span>; <span class="NLM_string-name">Benink, H. A.</span>; <span class="NLM_string-name">Worzella, T. J.</span>; <span class="NLM_string-name">Minor, L.</span></span> <span> </span><span class="NLM_article-title">Cell viability assays</span>. In  <i>Assay Guidance Manual</i>; <span class="NLM_publisher-name">Eli Lilly & Company and the National Center for Advancing Translational Sciences</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=T.+L.+Riss&author=R.+A.+Moravec&author=A.+L.+Niles&author=S.+Duellman&author=H.+A.+Benink&author=T.+J.+Worzella&author=L.+Minor&title=Assay+Guidance+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DRiss%26aufirst%3DT.%2BL.%26atitle%3DCell%2520viability%2520assays%26btitle%3DAssay%2520Guidance%2520Manual%26pub%3DEli%2520Lilly%2520%2526%2520Company%2520and%2520the%2520National%2520Center%2520for%2520Advancing%2520Translational%2520Sciences%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hastie, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLauchlan, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span> <span> </span><span class="NLM_article-title">Assay of protein kinases using radiolabeled ATP: a protocol</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">968</span>– <span class="NLM_lpage">971</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnprot.2006.149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=17406331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=968-971&author=C.+J.+Hastieauthor=H.+J.+McLauchlanauthor=P.+Cohen&title=Assay+of+protein+kinases+using+radiolabeled+ATP%3A+a+protocol&doi=10.1038%2Fnprot.2006.149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Assay of protein kinases using radiolabeled ATP: a protocol</span></div><div class="casAuthors">Hastie, C. James; McLauchlan, Hilary J.; Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">968-971</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein kinase activity results in the incorporation of radiolabeled phosphate from [γ-32P]ATP into a peptide or protein substrate.  The measurement of the amt. of radioactivity incorporated into a substrate as a function of time and enzyme concn. allows enzyme activity to be quantified.  The activity is expressed as a 'unit', where 1 unit corresponds to the amt. of protein kinase that catalyzes the incorporation of 1 nmol of phosphate into the std. substrate in 1 min.  Specific activity is defined as units of activity per mg protein.  The assay format described here is quick, simple, inexpensive, sensitive and accurate, provides a direct measurement of activity and remains the 'gold std.' for the quantification of protein kinase activity.  Up to 40 samples can be assayed manually at one time, and the assay takes one person less than 1 h to complete.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFfHCuf9HbVLVg90H21EOLACvtfcHk0lgxeAW8j-lhvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitb%252FM&md5=d6e443498bece9b9b6dc1c3a8a29580c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.149%26sid%3Dliteratum%253Aachs%26aulast%3DHastie%26aufirst%3DC.%2BJ.%26aulast%3DMcLauchlan%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DAssay%2520of%2520protein%2520kinases%2520using%2520radiolabeled%2520ATP%253A%2520a%2520protocol%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D968%26epage%3D971%26doi%3D10.1038%2Fnprot.2006.149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graczyk, P. P.</span></span> <span> </span><span class="NLM_article-title">Gini Coefficient: A new way to express selectivity of kinase inhibitors against a family of kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">5773</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/jm070562u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070562u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=5773-5779&author=P.+P.+Graczyk&title=Gini+Coefficient%3A+A+new+way+to+express+selectivity+of+kinase+inhibitors+against+a+family+of+kinases&doi=10.1021%2Fjm070562u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Gini Coefficient: A New Way To Express Selectivity of Kinase Inhibitors against a Family of Kinases</span></div><div class="casAuthors">Graczyk, Piotr P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5773-5779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel application of the Gini coeff. for expressing selectivity of kinase inhibitors against a panel of kinases is proposed.  This has been illustrated using single-point inhibition data for 40 com. available kinase inhibitors screened against 85 kinases.  Nonselective inhibitors are characterized by Gini values close to zero (Staurosporine, Gini 0.150).  Highly selective compds. exhibit Gini values close to 1 (PD184352 Gini 0.905).  The relative selectivity of inhibitors does not depend on the ATP concn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqUHpmF-C9u7Vg90H21EOLACvtfcHk0ljGsPZD8eEtXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFyhu7rL&md5=51f4902e49196dfef8e135c4a09182d7</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm070562u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070562u%26sid%3Dliteratum%253Aachs%26aulast%3DGraczyk%26aufirst%3DP.%2BP.%26atitle%3DGini%2520Coefficient%253A%2520A%2520new%2520way%2520to%2520express%2520selectivity%2520of%2520kinase%2520inhibitors%2520against%2520a%2520family%2520of%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D5773%26epage%3D5779%26doi%3D10.1021%2Fjm070562u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uitdehaag, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, G. J.</span></span> <span> </span><span class="NLM_article-title">A theoretical entropy score as a single value to express inhibitor selectivity</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">94</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-12-94</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2F1471-2105-12-94" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21486481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltV2htb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=94&author=J.+C.+Uitdehaagauthor=G.+J.+Zaman&title=A+theoretical+entropy+score+as+a+single+value+to+express+inhibitor+selectivity&doi=10.1186%2F1471-2105-12-94"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A theoretical entropy score as a single value to express inhibitor selectivity</span></div><div class="casAuthors">Uitdehaag, Joost C. M.; Zaman, Guido J. R.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">94</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Designing maximally selective ligands that act on individual targets is the dominant paradigm in drug discovery.  Poor selectivity can underlie toxicity and side effects in the clinic, and for this reason compd. selectivity is increasingly monitored from very early on in the drug discovery process.  To make sense of large amts. of profiling data, and to det. when a compd. is sufficiently selective, there is a need for a proper quant. measure of selectivity.  Here we propose a new theor. entropy score that can be calcd. from a set of IC50 data.  In contrast to previous measures such as the 'selectivity score', Gini score, or partition index, the entropy score is non-arbitary, fully exploits IC50 data, and is not dependent on a ref. kinase.  In addn., the entropy score gives the most robust values with data from different sources, because it is less sensitive to errors.  We apply the new score to kinase and nuclear receptor profiling data, and to high-throughput screening data.  In addn., through analyzing profiles of clin. compds., we show that a more selective inhibitor is not necessarily more drug-like.  For quantifying selectivity from panel profiling, a theor. entropy score is the best method.  It is valuable for studying the mol. mechanisms of selectivity, and to steer compd. progression in drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2tB5EtrxkGbVg90H21EOLACvtfcHk0ljGsPZD8eEtXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltV2htb8%253D&md5=a057b5f0222693138d3942cb439aa39c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-12-94&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-12-94%26sid%3Dliteratum%253Aachs%26aulast%3DUitdehaag%26aufirst%3DJ.%2BC.%26aulast%3DZaman%26aufirst%3DG.%2BJ.%26atitle%3DA%2520theoretical%2520entropy%2520score%2520as%2520a%2520single%2520value%2520to%2520express%2520inhibitor%2520selectivity%26jtitle%3DBMC%2520Bioinf.%26date%3D2011%26volume%3D12%26spage%3D94%26doi%3D10.1186%2F1471-2105-12-94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>, <span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0ljGsPZD8eEtXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saurabh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherzer, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mims, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockwood, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraft, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beverly, L. J.</span></span> <span> </span><span class="NLM_article-title">The PIM family of oncoproteins: small kinases with huge implications in myeloid leukemogenesis and as therapeutic targets</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">8503</span>– <span class="NLM_lpage">8514</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.18632%2Foncotarget.2330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=25238262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC2M7jtVKitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=8503-8514&author=K.+Saurabhauthor=M.+T.+Scherzerauthor=P.+P.+Shahauthor=A.+S.+Mimsauthor=W.+W.+Lockwoodauthor=A.+S.+Kraftauthor=L.+J.+Beverly&title=The+PIM+family+of+oncoproteins%3A+small+kinases+with+huge+implications+in+myeloid+leukemogenesis+and+as+therapeutic+targets&doi=10.18632%2Foncotarget.2330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The PIM family of oncoproteins: small kinases with huge implications in myeloid leukemogenesis and as therapeutic targets</span></div><div class="casAuthors">Saurabh Kumar; Scherzer Michael T; Shah Parag P; Mims Alice S; Kraft Andrew S; Lockwood William W; Beverly Levi J</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8503-14</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PIM kinases are a family of serine/threonine kinases involved in cell survival and proliferation.  There is significant structural similarity between the three PIM kinases (PIM1, PIM2 and PIM3) and only few amino acid differences.  Although, several studies have specifically monitored the role of PIM1 in tumorigenesis, much less is known about PIM2 and PIM3.  Therefore, in this study we have used in vitro cell culture models and in vivo bone marrow infection/transplantation to assess the comparative signaling and oncogenic potential of each of the three PIM kinases.  All three PIM kinases were able to protect FL5.12 cells from IL3 withdrawal induced death.  Interestingly, the downstream signaling cascades were indistinguishable between the three kinases.  Transplantation of murine bone marrow co-expressing MYC and PIM1, PIM2 or PIM3 caused rapid and uniformly lethal myeloid leukemia.  De-induction of MYC 18 days following transplantation significantly increased the survival of mice, even with continual expression of PIM kinases.  Alternatively, mice treated at the pre-leukemic stage with a PIM kinase inhibitor increased the lifespan of the mice, even with continual expression of the MYC transgene.  These data demonstrate the role of PIM kinases in driving myeloid leukemia, and as candidate molecules for therapy against human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRe0My0fF0UK5p6t8nS6Nc3fW6udTcc2eafhgdHRFumP7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M7jtVKitg%253D%253D&md5=a24cfd40890d5ff365cc84d0492e393c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2330%26sid%3Dliteratum%253Aachs%26aulast%3DSaurabh%26aufirst%3DK.%26aulast%3DScherzer%26aufirst%3DM.%2BT.%26aulast%3DShah%26aufirst%3DP.%2BP.%26aulast%3DMims%26aufirst%3DA.%2BS.%26aulast%3DLockwood%26aufirst%3DW.%2BW.%26aulast%3DKraft%26aufirst%3DA.%2BS.%26aulast%3DBeverly%26aufirst%3DL.%2BJ.%26atitle%3DThe%2520PIM%2520family%2520of%2520oncoproteins%253A%2520small%2520kinases%2520with%2520huge%2520implications%2520in%2520myeloid%2520leukemogenesis%2520and%2520as%2520therapeutic%2520targets%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D8503%26epage%3D8514%26doi%3D10.18632%2Foncotarget.2330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willems, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dedobbeleer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Digregorio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lumapat, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogister, B.</span></span> <span> </span><span class="NLM_article-title">The functional diversity of Aurora kinases: a comprehensive review</span>. <i>Cell Div.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">7</span>, <span class="refDoi"> DOI: 10.1186/s13008-018-0040-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1186%2Fs13008-018-0040-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=30250494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvVent7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=7&author=E.+Willemsauthor=M.+Dedobbeleerauthor=M.+Digregorioauthor=A.+Lombardauthor=P.+N.+Lumapatauthor=B.+Rogister&title=The+functional+diversity+of+Aurora+kinases%3A+a+comprehensive+review&doi=10.1186%2Fs13008-018-0040-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The functional diversity of Aurora kinases: a comprehensive review</span></div><div class="casAuthors">Willems, Estelle; Dedobbeleer, Matthias; Digregorio, Marina; Lombard, Arnaud; Lumapat, Paul Noel; Rogister, Bernard</div><div class="citationInfo"><span class="NLM_cas:title">Cell Division</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7/1-7/17</span>CODEN:
                <span class="NLM_cas:coden">CDEIAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-1028</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Aurora kinases are serine/threonine kinases essential for the onset and progression of mitosis.  Aurora members share a similar protein structure and kinase activity, but exhibit distinct cellular and subcellular localization.  AurA favors the G2/M transition by promoting centrosome maturation and mitotic spindle assembly.  AurB and AurC are chromosome-passenger complex proteins, crucial for chromosome binding to kinetochores and segregation of chromosomes.  Cellular distribution of AurB is ubiquitous, while AurC expression is mainly restricted to meiotically-active germ cells.  In human tumors, all Aurora kinase members play oncogenic roles related to their mitotic activity and promote cancer cell survival and proliferation.  Furthermore, AurA plays tumor-promoting roles unrelated to mitosis, including tumor stemness, epithelial-to-mesenchymal transition and invasion.  In this review, we aim to understand the functional interplay of Aurora kinases in various types of human cells, including tumor cells.  The understanding of the functional diversity of Aurora kinases could help to evaluate their relevance as potential therapeutic targets in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvSNgg8ZBDULVg90H21EOLACvtfcHk0lhgCdX_2pXlng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvVent7g%253D&md5=ac0d2d15b2e29ce4d96ef7f8eebfa4a7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1186%2Fs13008-018-0040-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13008-018-0040-6%26sid%3Dliteratum%253Aachs%26aulast%3DWillems%26aufirst%3DE.%26aulast%3DDedobbeleer%26aufirst%3DM.%26aulast%3DDigregorio%26aufirst%3DM.%26aulast%3DLombard%26aufirst%3DA.%26aulast%3DLumapat%26aufirst%3DP.%2BN.%26aulast%3DRogister%26aufirst%3DB.%26atitle%3DThe%2520functional%2520diversity%2520of%2520Aurora%2520kinases%253A%2520a%2520comprehensive%2520review%26jtitle%3DCell%2520Div.%26date%3D2018%26volume%3D13%26spage%3D7%26doi%3D10.1186%2Fs13008-018-0040-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabian, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biggs, W. H.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azimioara, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benedetti, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Insko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelias, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velasco, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span> <span> </span><span class="NLM_article-title">A small molecule-kinase interaction map for clinical kinase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1038/nbt1068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnbt1068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=15711537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2005&pages=329-336&author=M.+A.+Fabianauthor=W.+H.+Biggsauthor=D.+K.+Treiberauthor=C.+E.+Atteridgeauthor=M.+D.+Azimioaraauthor=M.+G.+Benedettiauthor=T.+A.+Carterauthor=P.+Ciceriauthor=P.+T.+Edeenauthor=M.+Floydauthor=J.+M.+Fordauthor=M.+Galvinauthor=J.+L.+Gerlachauthor=R.+M.+Grotzfeldauthor=S.+Herrgardauthor=D.+E.+Inskoauthor=M.+A.+Inskoauthor=A.+G.+Laiauthor=J.+M.+Leliasauthor=S.+A.+Mehtaauthor=Z.+V.+Milanovauthor=A.+M.+Velascoauthor=L.+M.+Wodickaauthor=H.+K.+Patelauthor=P.+P.+Zarrinkarauthor=D.+J.+Lockhart&title=A+small+molecule-kinase+interaction+map+for+clinical+kinase+inhibitors&doi=10.1038%2Fnbt1068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule-kinase interaction map for clinical kinase inhibitors</span></div><div class="casAuthors">Fabian, Miles A.; Biggs, William H.; Treiber, Daniel K.; Atteridge, Corey E.; Azimioara, Mihai D.; Benedetti, Michael G.; Carter, Todd A.; Ciceri, Pietro; Edeen, Philip T.; Floyd, Mark; Ford, Julia M.; Galvin, Margaret; Gerlach, Jay L.; Grotzfeld, Robert M.; Herrgard, Sanna; Insko, Darren E.; Insko, Michael A.; Lai, Andiliy G.; Lelias, Jean-Michel; Mehta, Shamal A.; Milanov, Zdravko V.; Velasco, Anne Marie; Wodicka, Lisa M.; Patel, Hitesh K.; Zarrinkar, Patrick P.; Lockhart, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-336</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors show great promise as a new class of therapeutics.  Here the authors describe an efficient way to det. kinase inhibitor specificity by measuring binding of small mols. to the ATP site of kinases.  The authors have profiled 20 kinase inhibitors, including 16 that are approved drugs or in clin. development, against a panel of 119 protein kinases.  The authors find that specificity varies widely and is not strongly correlated with chem. structure or the identity of the intended target.  Many novel interactions were identified, including tight binding of the p38 inhibitor BIRB-796 to an imatinib-resistant variant of the ABL kinase, and binding of imatinib to the SRC-family kinase LCK.  The authors also show that mutations in the epidermal growth factor receptor (EGFR) found in gefitinib-responsive patients do not affect the binding affinity of gefitinib or erlotinib.  Our results represent a systematic small mol.-protein interaction map for clin. compds. across a large no. of related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0vNmMOXrnrVg90H21EOLACvtfcHk0lhgCdX_2pXlng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitF2nt7w%253D&md5=6aedd5ceb8f77cd26ee50425dcec7bdf</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnbt1068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1068%26sid%3Dliteratum%253Aachs%26aulast%3DFabian%26aufirst%3DM.%2BA.%26aulast%3DBiggs%26aufirst%3DW.%2BH.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DAzimioara%26aufirst%3DM.%2BD.%26aulast%3DBenedetti%26aufirst%3DM.%2BG.%26aulast%3DCarter%26aufirst%3DT.%2BA.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DFord%26aufirst%3DJ.%2BM.%26aulast%3DGalvin%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DJ.%2BL.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DInsko%26aufirst%3DD.%2BE.%26aulast%3DInsko%26aufirst%3DM.%2BA.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DLelias%26aufirst%3DJ.%2BM.%26aulast%3DMehta%26aufirst%3DS.%2BA.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DVelasco%26aufirst%3DA.%2BM.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26atitle%3DA%2520small%2520molecule-kinase%2520interaction%2520map%2520for%2520clinical%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2005%26volume%3D23%26spage%3D329%26epage%3D336%26doi%3D10.1038%2Fnbt1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miduturu, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brault, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">High-throughput kinase profiling: A more efficient approach towards the discovery of new kinase inhibitors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">879</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.chembiol.2011.05.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21802008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFarurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=868-879&author=C.+V.+Miduturuauthor=X.+Dengauthor=N.+Kwiatkowskiauthor=W.+Yangauthor=L.+Braultauthor=P.+Filippakopoulosauthor=E.+Chungauthor=Q.+Yangauthor=J.+Schwallerauthor=S.+Knappauthor=R.+W.+Kingauthor=J.-D.+Leeauthor=S.+Herrgardauthor=P.+Zarrinkarauthor=N.+S.+Gray&title=High-throughput+kinase+profiling%3A+A+more+efficient+approach+towards+the+discovery+of+new+kinase+inhibitors&doi=10.1016%2Fj.chembiol.2011.05.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Kinase Profiling: A More Efficient Approach toward the Discovery of New Kinase Inhibitors</span></div><div class="casAuthors">Miduturu, Chandrasekhar V.; Deng, Xianming; Kwiatkowski, Nicholas; Yang, Wannian; Brault, Laurent; Filippakopoulos, Panagis; Chung, Eunah; Yang, Qingkai; Schwaller, Juerg; Knapp, Stefan; King, Randall W.; Lee, Jiing-Dwan; Herrgard, Sanna; Zarrinkar, Patrick; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">868-879</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Selective protein kinase inhibitors have only been developed against a small no. of kinase targets.  Here we demonstrate that "high-throughput kinase profiling" is an efficient method for the discovery of lead compds. for established as well as unexplored kinase targets.  We screened a library of 118 compds. constituting two distinct scaffolds (furan-thiazolidinediones and pyrimido-diazepines) against a panel of 353 kinases.  A distinct kinase selectivity profile was obsd. for each scaffold.  Selective inhibitors were identified with submicromolar cellular activity against PIM1, ERK5, ACK1, MPS1, PLK1-3, and Aurora A,B kinases.  In addn., we identified potent inhibitors for so far unexplored kinases such as DRAK1, HIPK2, and DCAMKL1 that await further evaluation.  This inhibitor-centric approach permits comprehensive assessment of a scaffold of interest and represents an efficient and general strategy for identifying new selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKLvN9eLNvybVg90H21EOLACvtfcHk0liY55JrNEnn_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFarurw%253D&md5=d62a4771b1215551f3913b93d6eca7ee</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DMiduturu%26aufirst%3DC.%2BV.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DBrault%26aufirst%3DL.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKing%26aufirst%3DR.%2BW.%26aulast%3DLee%26aufirst%3DJ.-D.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DZarrinkar%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DHigh-throughput%2520kinase%2520profiling%253A%2520A%2520more%2520efficient%2520approach%2520towards%2520the%2520discovery%2520of%2520new%2520kinase%2520inhibitors%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D868%26epage%3D879%26doi%3D10.1016%2Fj.chembiol.2011.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolf, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skovgaard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthelsen, J.</span></span> <span> </span><span class="NLM_article-title">A comparison of protein kinases inhibitor screening methods using both enzymatic activity and binding affinity determination</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e98800</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0098800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1371%2Fjournal.pone.0098800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=24915177" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1artbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=A.+F.+Rudolfauthor=T.+Skovgaardauthor=S.+Knappauthor=L.+J.+Jensenauthor=J.+Berthelsen&title=A+comparison+of+protein+kinases+inhibitor+screening+methods+using+both+enzymatic+activity+and+binding+affinity+determination&doi=10.1371%2Fjournal.pone.0098800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A comparison of protein kinases inhibitor screening methods using both enzymatic activity and binding affinity determination</span></div><div class="casAuthors">Rudolf, Amalie Frederikke; Skovgaard, Tine; Knapp, Stefan; Juhl Jensen, Lars; Berthelsen, Jens</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e98800/1-e98800/5, 5 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Binding assays are increasingly used as a screening method for protein kinase inhibitors; however, as yet only a weak correlation with enzymic activity-based assays has been demonstrated.  We show that the correlation between the two types of assays can be improved using more precise screening conditions.  Furthermore a marked improvement in the correlation was found by using kinase constructs contg. the catalytic domain in presence of addnl. domains or subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ6qupl5aTgrVg90H21EOLACvtfcHk0liY55JrNEnn_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1artbrO&md5=c79d0fbd01aee6442a6223c2e6eb24f1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098800%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DA.%2BF.%26aulast%3DSkovgaard%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DBerthelsen%26aufirst%3DJ.%26atitle%3DA%2520comparison%2520of%2520protein%2520kinases%2520inhibitor%2520screening%2520methods%2520using%2520both%2520enzymatic%2520activity%2520and%2520binding%2520affinity%2520determination%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0098800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brasó-Maristany, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filosto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catchpole, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francesch-Domenech, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumb, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakka, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazinska, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liccardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gris-Oliver, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheang, M. C. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdrix-Rosell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noël, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaltriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grigoriadis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, A. N.</span></span> <span> </span><span class="NLM_article-title">PIM1 kinase regulates cell death, tumor growth and chemotherapy response revealing a novel target in triple-negative breast cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1303</span>– <span class="NLM_lpage">1313</span>, <span class="refDoi"> DOI: 10.1038/nm.4198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnm.4198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=27775704" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslehur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=1303-1313&author=F.+Bras%C3%B3-Maristanyauthor=S.+Filostoauthor=S.+Catchpoleauthor=R.+Marlowauthor=J.+Quistauthor=E.+Francesch-Domenechauthor=D.+A.+Plumbauthor=L.+Zakkaauthor=P.+Gazinskaauthor=G.+Liccardiauthor=P.+Meierauthor=A.+Gris-Oliverauthor=M.+C.+U.+Cheangauthor=A.+Perdrix-Rosellauthor=M.+Shafatauthor=E.+No%C3%ABlauthor=N.+Patelauthor=K.+McEachernauthor=M.+Scaltritiauthor=P.+Castelauthor=F.+Noorauthor=R.+Buusauthor=S.+Mathewauthor=J.+Watkinsauthor=V.+Serraauthor=P.+Marraauthor=A.+Grigoriadisauthor=A.+N.+Tutt&title=PIM1+kinase+regulates+cell+death%2C+tumor+growth+and+chemotherapy+response+revealing+a+novel+target+in+triple-negative+breast+cancer&doi=10.1038%2Fnm.4198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PIM1 kinase regulates cell death, tumor growth and chemotherapy response in triple-negative breast cancer</span></div><div class="casAuthors">Braso-Maristany, Fara; Filosto, Simone; Catchpole, Steven; Marlow, Rebecca; Quist, Jelmar; Francesch-Domenech, Erika; Plumb, Darren A.; Zakka, Leila; Gazinska, Patrycja; Liccardi, Gianmaria; Meier, Pascal; Gris-Oliver, Albert; Cheang, Maggie Chon U.; Perdrix-Rosell, Anna; Shafat, Manar; Noel, Elodie; Patel, Nirmesh; McEachern, Kristen; Scaltriti, Maurizio; Castel, Pau; Noor, Farzana; Buus, Richard; Mathew, Sumi; Watkins, Johnathan; Serra, Violeta; Marra, Pierfrancesco; Grigoriadis, Anita; Tutt, Andrew N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1303-1313</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBCs) have poor prognosis and lack targeted therapies.  Here we identified increased copy no. and expression of the PIM1 proto-oncogene in genomic data sets of patients with TNBC.  TNBC cells, but not nonmalignant mammary epithelial cells, were dependent on PIM1 for proliferation and protection from apoptosis.  PIM1 knockdown reduced expression of the anti-apoptotic factor BCL2, and dynamic BH3 profiling of apoptotic priming revealed that PIM1 prevents mitochondrial-mediated apoptosis in TNBC cell lines.  In TNBC tumors and their cellular models, PIM1 expression was assocd. with several transcriptional signatures involving the transcription factor MYC, and PIM1 depletion in TNBC cell lines decreased, in a MYC-dependent manner, cell population growth and expression of the MYC target gene MCL1.  Treatment with the pan-PIM kinase inhibitor AZD1208 impaired the growth of both cell line and patient-derived xenografts and sensitized them to std.-of-care chemotherapy.  This work identifies PIM1 as a malignant-cell-selective target in TNBC and the potential use of PIM1 inhibitors for sensitizing TNBC to chemotherapy-induced apoptotic cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3M8bn0hjO67Vg90H21EOLACvtfcHk0liY55JrNEnn_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslehur3E&md5=02c40bff3f1179dab3705af83f79c1f8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnm.4198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4198%26sid%3Dliteratum%253Aachs%26aulast%3DBras%25C3%25B3-Maristany%26aufirst%3DF.%26aulast%3DFilosto%26aufirst%3DS.%26aulast%3DCatchpole%26aufirst%3DS.%26aulast%3DMarlow%26aufirst%3DR.%26aulast%3DQuist%26aufirst%3DJ.%26aulast%3DFrancesch-Domenech%26aufirst%3DE.%26aulast%3DPlumb%26aufirst%3DD.%2BA.%26aulast%3DZakka%26aufirst%3DL.%26aulast%3DGazinska%26aufirst%3DP.%26aulast%3DLiccardi%26aufirst%3DG.%26aulast%3DMeier%26aufirst%3DP.%26aulast%3DGris-Oliver%26aufirst%3DA.%26aulast%3DCheang%26aufirst%3DM.%2BC.%2BU.%26aulast%3DPerdrix-Rosell%26aufirst%3DA.%26aulast%3DShafat%26aufirst%3DM.%26aulast%3DNo%25C3%25ABl%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DMcEachern%26aufirst%3DK.%26aulast%3DScaltriti%26aufirst%3DM.%26aulast%3DCastel%26aufirst%3DP.%26aulast%3DNoor%26aufirst%3DF.%26aulast%3DBuus%26aufirst%3DR.%26aulast%3DMathew%26aufirst%3DS.%26aulast%3DWatkins%26aufirst%3DJ.%26aulast%3DSerra%26aufirst%3DV.%26aulast%3DMarra%26aufirst%3DP.%26aulast%3DGrigoriadis%26aufirst%3DA.%26aulast%3DTutt%26aufirst%3DA.%2BN.%26atitle%3DPIM1%2520kinase%2520regulates%2520cell%2520death%252C%2520tumor%2520growth%2520and%2520chemotherapy%2520response%2520revealing%2520a%2520novel%2520target%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D1303%26epage%3D1313%26doi%3D10.1038%2Fnm.4198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daver, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenk, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levis, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting FLT3 mutations in AML: review of current knowledge and evidence</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/s41375-018-0357-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fs41375-018-0357-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=30651634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyrsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=299-312&author=N.+Daverauthor=R.+F.+Schlenkauthor=N.+H.+Russellauthor=M.+J.+Levis&title=Targeting+FLT3+mutations+in+AML%3A+review+of+current+knowledge+and+evidence&doi=10.1038%2Fs41375-018-0357-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FLT3 mutations in AML: review of current knowledge and evidence</span></div><div class="casAuthors">Daver, Naval; Schlenk, Richard F.; Russell, Nigel H.; Levis, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-312</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Genomic investigations of acute myeloid leukemia (AML) have demonstrated that several genes are recurrently mutated, leading to new genomic classifications, predictive biomarkers, and new therapeutic targets.  Mutations of the FMS-like tyrosine kinase 3 (FLT3) gene occur in approx. 30% of all AML cases, with the internal tandem duplication (ITD) representing the most common type of FLT3 mutation (FLT3-ITD; approx. 25% of all AML cases).  FLT3-ITD is a common driver mutation that presents with a high leukemic burden and confers a poor prognosis in patients with AML.  The prognostic value of a FLT3 mutation in the tyrosine kinase domain (FLT3-TKD), which has a lower incidence in AML (approx. 7-10% of all cases), is uncertain.  Accumulating evidence demonstrates that FLT3 mutational status evolves throughout the disease continuum.  This so-called clonal evolution, together with the identification of FLT3-ITD as a neg. prognostic marker, serves to highlight the importance of FLT3-ITD testing at diagnosis and again at relapse.  Earlier identification of FLT3 mutations will help provide a better understanding of the patient's disease and enable targeted treatment that may help patients achieve longer and more durable remissions.  First-generation FLT3 inhibitors developed for clin. use are broad-spectrum, multikinase inhibitors; however, next-generation FLT3 inhibitors are more specific, more potent, and have fewer toxicities assocd. with off-target effects.  Primary and secondary acquired resistance to FLT3 inhibitors remains a challenge and provides a rationale for combining FLT3 inhibitors with other therapies, both conventional and investigational.  This review focuses on the pathol. and prognostic role of FLT3 mutations in AML, clin. classification of the disease, recent progress with next-generation FLT3 inhibitors, and mechanisms of resistance to FLT3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjK_p2nZ_9j7Vg90H21EOLACvtfcHk0lidWbKQAztDMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyrsbbI&md5=25ea007d2f7b670aaddeb02866f15c69</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41375-018-0357-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41375-018-0357-9%26sid%3Dliteratum%253Aachs%26aulast%3DDaver%26aufirst%3DN.%26aulast%3DSchlenk%26aufirst%3DR.%2BF.%26aulast%3DRussell%26aufirst%3DN.%2BH.%26aulast%3DLevis%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520FLT3%2520mutations%2520in%2520AML%253A%2520review%2520of%2520current%2520knowledge%2520and%2520evidence%26jtitle%3DLeukemia%26date%3D2019%26volume%3D33%26spage%3D299%26epage%3D312%26doi%3D10.1038%2Fs41375-018-0357-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, N.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ok Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.-M.</span></span> <span> </span><span class="NLM_article-title">Extracellular signal-regulated kinase 8-mediated NF-kappaB activation increases sensitivity of human lung cancer cells to arsenic trioxide</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">49144</span>– <span class="NLM_lpage">49155</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.17100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.18632%2Foncotarget.17100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=28467781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC1crjs1Cjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=49144-49155&author=D.-D.+Wuauthor=A.+Lauauthor=F.-Y.+Yuauthor=N.-L.+Caiauthor=L.-J.+Daiauthor=M.+Ok+Kimauthor=D.-Y.+Jinauthor=Y.-M.+Xu&title=Extracellular+signal-regulated+kinase+8-mediated+NF-kappaB+activation+increases+sensitivity+of+human+lung+cancer+cells+to+arsenic+trioxide&doi=10.18632%2Foncotarget.17100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular signal-regulated kinase 8-mediated NF-κB activation increases sensitivity of human lung cancer cells to arsenic trioxide</span></div><div class="casAuthors">Wu Dan-Dan; Lau Andy T Y; Yu Fei-Yuan; Cai Na-Li; Dai Li-Juan; Xu Yan-Ming; Ok Kim Myoung; Jin Dong-Yan</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">49144-49155</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Extracellular signal-regulated kinase 8 (ERK8), also known as mitogen-activated protein kinase 15 (MAPK15), is the most recently identified protein kinase of the ERK family members and yet the least has been studied so far.  Here, we report that ERK8 is highly expressed in several human lung cancer cell lines and is positively correlated with their sensitivities to the anti-cancer drug arsenic trioxide (As2O3).  As2O3 at physiologically relevant concentrations (5-20 μM) potently stimulates the phosphorylation of ERK8 at Thr175 and Tyr177 within the TEY motif in the kinase domain, leading to its activation.  Interestingly, activated ERK8 interacts and directly phosphorylates IkappaBalpha (IκBα) at Ser32 and Ser36, resulting in IκBα degradation.  This in turn promotes nuclear factor-kappaB (NF-κB) p65 nuclear translocation and chromatin-binding, as well as the subsequent induction and activation of proteins involved in apoptosis.  We also show that stable short-hairpin RNA-specific knockdown of endogenous ERK8 or inhibition of NF-κB activity by NF-κB inhibitor in high ERK8 expressing lung cancer H1299 cells blunted the As2O3-induced NF-κB activation and cytotoxicity towards these cells, indicating the critical role of ERK8 and NF-κB in mediating the As2O3 effects.  Taken together, our findings suggest for the first time a regulatory paradigm of NF-κB activation by ERK8 upon As2O3 treatment in human lung cancer cells; and implicate a potential therapeutic advantage of As2O3 that might gain more selective killing of cancer cells with high ERK8 expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSVn-pfX7PHly86Js-BCLiifW6udTcc2eYhEnGvHFHA8bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crjs1Cjsg%253D%253D&md5=11e8e30410c83cdf74894a9bed8dfa50</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.17100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.17100%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DD.-D.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DF.-Y.%26aulast%3DCai%26aufirst%3DN.-L.%26aulast%3DDai%26aufirst%3DL.-J.%26aulast%3DOk%2BKim%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DD.-Y.%26aulast%3DXu%26aufirst%3DY.-M.%26atitle%3DExtracellular%2520signal-regulated%2520kinase%25208-mediated%2520NF-kappaB%2520activation%2520increases%2520sensitivity%2520of%2520human%2520lung%2520cancer%2520cells%2520to%2520arsenic%2520trioxide%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D49144%26epage%3D49155%26doi%3D10.18632%2Foncotarget.17100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chieffi, P.</span></span> <span> </span><span class="NLM_article-title">Aurora B: A new promising therapeutic target in cancer</span>. <i>Intractable Rare Dis. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.5582/irdr.2018.01018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.5582%2Firdr.2018.01018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=29862159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC1MbjtlSnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=141-144&author=P.+Chieffi&title=Aurora+B%3A+A+new+promising+therapeutic+target+in+cancer&doi=10.5582%2Firdr.2018.01018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Aurora B: A new promising therapeutic target in cancer</span></div><div class="casAuthors">Chieffi Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Intractable & rare diseases research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-144</span>
        ISSN:<span class="NLM_cas:issn">2186-3644</span>.
    </div><div class="casAbstract">A critical step for maintenance of genetic stability is chromosome segregation, which requires a high coordination of cellular processes.  Loss of mitotic regulation is a possible cause of aneuploidy in human epithelial malignancy and it is thought to create an abnormal nuclear morphology in cancer cells.  Serine/threonine protein kinase Aurora B gene plays a regulatory role from G2 to cytokinesis, encompassing key cell cycle events such as centrosome duplication, chromosome bi-orientation, and segregation.  The overexpression of Aurora B has been observed in several tumour types, and has been linked with a poor prognosis for cancer patients.  Therapeutic inhibition of Aurora kinase showed great promise as a probable anticancer regime because of its important role during cell division.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxwXYefhgNVEodMGTatQ7LfW6udTcc2eYhEnGvHFHA8bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MbjtlSnsg%253D%253D&md5=1f669456745503e18bf416c3f4b50a91</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.5582%2Firdr.2018.01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5582%252Firdr.2018.01018%26sid%3Dliteratum%253Aachs%26aulast%3DChieffi%26aufirst%3DP.%26atitle%3DAurora%2520B%253A%2520A%2520new%2520promising%2520therapeutic%2520target%2520in%2520cancer%26jtitle%3DIntractable%2520Rare%2520Dis.%2520Res.%26date%3D2018%26volume%3D7%26spage%3D141%26epage%3D144%26doi%3D10.5582%2Firdr.2018.01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vichai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirtikara, K.</span></span> <span> </span><span class="NLM_article-title">Sulforhodamine B colorimetric assay for cytotoxicity screening</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1038%2Fnprot.2006.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=17406391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=1112-1116&author=V.+Vichaiauthor=K.+Kirtikara&title=Sulforhodamine+B+colorimetric+assay+for+cytotoxicity+screening&doi=10.1038%2Fnprot.2006.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Sulforhodamine B colorimetric assay for cytotoxicity screening</span></div><div class="casAuthors">Vichai, Vanicha; Kirtikara, Kanyawim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1112-1116</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The sulforhodamine B (SRB) assay is used for cell d. detn., based on the measurement of cellular protein content.  The method described here was optimized for the toxicity screening of compds. to adherent cells in a 96-well format.  After an incubation period, cell monolayers are fixed with 10% (wt/vol) trichloroacetic acid and stained for 30 min, after which the excess dye is removed by washing repeatedly with 1% (vol/vol) acetic acid.  The protein-bound dye is dissolved in 10 mM Tris base soln. for OD detn. at 510 nm using a microplate reader.  The results are linear over a 20-fold range of cell nos. and the sensitivity is comparable to those of fluorometric methods.  The method not only allows a large no. of samples to be tested within a few days, but also requires only simple equipment and inexpensive reagents.  The SRB assay is therefore an efficient and highly cost-effective method for screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGZ8pao808l7Vg90H21EOLACvtfcHk0lh02P9a3pC9oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O&md5=7173035a1f9eb3f02dd7f1a947bca659</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.179%26sid%3Dliteratum%253Aachs%26aulast%3DVichai%26aufirst%3DV.%26aulast%3DKirtikara%26aufirst%3DK.%26atitle%3DSulforhodamine%2520B%2520colorimetric%2520assay%2520for%2520cytotoxicity%2520screening%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D1112%26epage%3D1116%26doi%3D10.1038%2Fnprot.2006.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrie, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelland, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span> <span> </span><span class="NLM_article-title">In vitro and In vivo pharmacokinetic-pharmacodynamic relationships for the trisubstituted aminopurine cyclin-dependent kinase inhibitors olomoucine, bohemine and CYC202</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4875</span>– <span class="NLM_lpage">4887</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-04-2264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1158%2F1078-0432.CCR-04-2264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=16000586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVGgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=4875-4887&author=F.+I.+Raynaudauthor=S.+R.+Whittakerauthor=P.+M.+Fischerauthor=S.+McClueauthor=M.+I.+Waltonauthor=S.+E.+Barrieauthor=M.+D.+Garrettauthor=P.+Rogersauthor=S.+J.+Clarkeauthor=L.+R.+Kellandauthor=M.+Valentiauthor=L.+Bruntonauthor=S.+Ecclesauthor=D.+P.+Laneauthor=P.+Workman&title=In+vitro+and+In+vivo+pharmacokinetic-pharmacodynamic+relationships+for+the+trisubstituted+aminopurine+cyclin-dependent+kinase+inhibitors+olomoucine%2C+bohemine+and+CYC202&doi=10.1158%2F1078-0432.CCR-04-2264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro and In vivo Pharmacokinetic-Pharmacodynamic Relationships for the Trisubstituted Aminopurine Cyclin-Dependent Kinase Inhibitors Olomoucine, Bohemine and CYC202</span></div><div class="casAuthors">Raynaud, Florence I.; Whittaker, Steven R.; Fischer, Peter M.; McClue, Steven; Walton, Michael I.; Barrie, S. Elaine; Garrett, Michelle D.; Rogers, Paul; Clarke, Simon J.; Kelland, Lloyd R.; Valenti, Melanie; Brunton, Lisa; Eccles, Suzanne; Lane, David P.; Workman, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4875-4887</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: To investigate pharmacokinetic-pharmacodynamic relationships for the trisubstituted aminopurine cyclin-dependent kinase inhibitors olomoucine, bohemine, and CYC202 (R-roscovitine; seliciclib) in the HCT116 human colon carcinoma model.  Exptl. Design: The in vitro activity of the agents was detd. in a human tumor panel using the sulforhodamine B assay.  The concn. and time dependence was established in HCT116 cells.  Mol. biomarkers, including RB phosphorylation and cyclin expression, were assessed by Western blotting.  Pharmacokinetic properties were characterized in mice following anal. by liq. chromatog.-tandem mass spectrometry.  Based on these studies, a dosing regimen was developed for CYC202 that allowed therapeutic exposures in the HCT116 tumor xenograft.  RESULTS: The antitumor potency of the agents in vitro was in the order olomoucine (IC50, 56 μmol/L) < bohemine (IC50, 27 μmol/L) < CYC202 (IC50, 15 μmol/L), corresponding to their activities as cyclin-dependent kinase inhibitors.  Antitumor activity increased with exposure time up to 16 h.  The agents caused inhibition of RB and RNA polymerase II phosphorylation and depletion of cyclins.  They exhibited relatively rapid clearance following administration to mice.  CYC202 displayed the slowest clearance from plasma and the highest tumor uptake, with oral bioavailability of 86%.  Oral dosing of CYC202 gave active concns. in the tumor, modulation of pharmacodynamic markers, and inhibition of tumor growth.  CONCLUSIONS: CYC202 showed therapeutic activity on human cancer cell lines in vitro and on xenografts.  Pharmacodynamic markers are altered in vitro and in vivo, consistent with the inhibition of cyclin-dependent kinases.  Such markers may be potentially useful in the clin. development of CYC202 and other cyclin-dependent kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF8GDYVKzb7rVg90H21EOLACvtfcHk0lh02P9a3pC9oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVGgsLw%253D&md5=ecb8798ef675b44ea4ab5403940ce6ad</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-04-2264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-04-2264%26sid%3Dliteratum%253Aachs%26aulast%3DRaynaud%26aufirst%3DF.%2BI.%26aulast%3DWhittaker%26aufirst%3DS.%2BR.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26aulast%3DMcClue%26aufirst%3DS.%26aulast%3DWalton%26aufirst%3DM.%2BI.%26aulast%3DBarrie%26aufirst%3DS.%2BE.%26aulast%3DGarrett%26aufirst%3DM.%2BD.%26aulast%3DRogers%26aufirst%3DP.%26aulast%3DClarke%26aufirst%3DS.%2BJ.%26aulast%3DKelland%26aufirst%3DL.%2BR.%26aulast%3DValenti%26aufirst%3DM.%26aulast%3DBrunton%26aufirst%3DL.%26aulast%3DEccles%26aufirst%3DS.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DWorkman%26aufirst%3DP.%26atitle%3DIn%2520vitro%2520and%2520In%2520vivo%2520pharmacokinetic-pharmacodynamic%2520relationships%2520for%2520the%2520trisubstituted%2520aminopurine%2520cyclin-dependent%2520kinase%2520inhibitors%2520olomoucine%252C%2520bohemine%2520and%2520CYC202%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2005%26volume%3D11%26spage%3D4875%26epage%3D4887%26doi%3D10.1158%2F1078-0432.CCR-04-2264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aslantürk, Ö. S.</span></span> In  <i>Vitro cytotoxicity and cell viability assays: principles, advantages, and disadvantages</i>; <span class="NLM_publisher-name">InTech</span>, <span class="NLM_year">2018</span>; Vol.  <span class="NLM_volume">2</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=%C3%96.+S.+Aslant%C3%BCrk&title=Vitro+cytotoxicity+and+cell+viability+assays%3A+principles%2C+advantages%2C+and+disadvantages"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAslant%25C3%25BCrk%26aufirst%3D%25C3%2596.%2BS.%26btitle%3DVitro%2520cytotoxicity%2520and%2520cell%2520viability%2520assays%253A%2520principles%252C%2520advantages%252C%2520and%2520disadvantages%26pub%3DInTech%26date%3D2018%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heber, D.</span></span> <span> </span><span class="NLM_article-title">Limitations of MTT and MTS-based assays for measurement of antiproliferative activity of green tea polyphenols</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e10202</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0010202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1371%2Fjournal.pone.0010202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=20419137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A280%3ADC%252BC3c3ovVSqsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&author=P.+Wangauthor=S.+M.+Henningauthor=D.+Heber&title=Limitations+of+MTT+and+MTS-based+assays+for+measurement+of+antiproliferative+activity+of+green+tea+polyphenols&doi=10.1371%2Fjournal.pone.0010202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Limitations of MTT and MTS-based assays for measurement of antiproliferative activity of green tea polyphenols</span></div><div class="casAuthors">Wang Piwen; Henning Susanne M; Heber David</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e10202</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The chemopreventive effect of green tea polyphenols, such as (-)-epigallocatechin-3-gallate (EGCG), has been well demonstrated in cell culture studies.  However, a wide range of IC(50) concentrations has been observed in published studies of the anti-proliferative activity of EGCG from different laboratories.  Although the susceptibility to EGCG treatment is largely dependent on cancer cell type, the particular cell viability and proliferation assays utilized may significantly influence quantitative results reported in the literature.  METHODOLOGY/PRINCIPAL FINDINGS:  We compared five widely used methods to measure cell proliferation and viability after EGCG treatment using LNCaP prostate cancer cells and MCF-7 breast cancer cells.  Both methods using dyes to quantify adenosine triphosphate (ATP) and deoxynucleic acid (DNA) showed accuracy in the measurement of viable cells when compared to trypan blue assay and results showed good linear correlation (r = 0.95).  However, the use of MTT (3-(4,5-Dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide) and MTS (3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2H-tetrazolium) as indicators of metabolically active mitochondria overestimated the number of viable cells by comparison with the ATP, DNA, or trypan blue determinations.  As a result, the observed IC(50) concentration of EGCG was 2-fold higher using MTT and MTS compared to dyes quantifying ATP and DNA.  In contrast, when cells were treated with apigenin MTT and MTS assays showed consistent results with ATP, DNA, or trypan blue assays.  CONCLUSIONS/SIGNIFICANCE:  These results demonstrate that MTT and MTS -based assays will provide an underestimation of the anti-proliferative effect of EGCG, and suggest the importance of careful evaluation of the method for in vitro assessment of cell viability and proliferation depending on the chemical nature of botanical supplements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcToR5Y0J603atenUz0lpNqnfW6udTcc2eaLSpCElJBxqrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3ovVSqsQ%253D%253D&md5=dbd51aea663c0e2772d67a7d3b21d232</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0010202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0010202%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%26aulast%3DHenning%26aufirst%3DS.%2BM.%26aulast%3DHeber%26aufirst%3DD.%26atitle%3DLimitations%2520of%2520MTT%2520and%2520MTS-based%2520assays%2520for%2520measurement%2520of%2520antiproliferative%2520activity%2520of%2520green%2520tea%2520polyphenols%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26doi%3D10.1371%2Fjournal.pone.0010202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sieuwerts, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span> <span> </span><span class="NLM_article-title">The MTT tetrazolium salt assay scrutinized: how to use this assay reliably to measure metabolie activity of cell cultures in vitro for the assessment of growth characteristics, IC50-values and cell survival</span>. <i>Clin. Chem. Lab. Med.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">824</span>, <span class="refDoi"> DOI: 10.1515/cclm.1995.33.11.813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1515%2Fcclm.1995.33.11.813" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1995&pages=813-824&author=A.+M.+Sieuwertsauthor=J.+G.+Klijnauthor=H.+A.+Petersauthor=J.+A.+Foekens&title=The+MTT+tetrazolium+salt+assay+scrutinized%3A+how+to+use+this+assay+reliably+to+measure+metabolie+activity+of+cell+cultures+in+vitro+for+the+assessment+of+growth+characteristics%2C+IC50-values+and+cell+survival&doi=10.1515%2Fcclm.1995.33.11.813"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1515%2Fcclm.1995.33.11.813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fcclm.1995.33.11.813%26sid%3Dliteratum%253Aachs%26aulast%3DSieuwerts%26aufirst%3DA.%2BM.%26aulast%3DKlijn%26aufirst%3DJ.%2BG.%26aulast%3DPeters%26aufirst%3DH.%2BA.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520MTT%2520tetrazolium%2520salt%2520assay%2520scrutinized%253A%2520how%2520to%2520use%2520this%2520assay%2520reliably%2520to%2520measure%2520metabolie%2520activity%2520of%2520cell%2520cultures%2520in%2520vitro%2520for%2520the%2520assessment%2520of%2520growth%2520characteristics%252C%2520IC50-values%2520and%2520cell%2520survival%26jtitle%3DClin.%2520Chem.%2520Lab.%2520Med.%26date%3D1995%26volume%3D33%26spage%3D813%26epage%3D824%26doi%3D10.1515%2Fcclm.1995.33.11.813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris-Natschke, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-H.</span></span> <span> </span><span class="NLM_article-title">Strategies for the optimization of natural leads to anticancer drugs or drug candidates</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1002/med.21377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1002%2Fmed.21377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=26359649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVagsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=32-91&author=Z.+Xiaoauthor=S.+L.+Morris-Natschkeauthor=K.-H.+Lee&title=Strategies+for+the+optimization+of+natural+leads+to+anticancer+drugs+or+drug+candidates&doi=10.1002%2Fmed.21377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for the Optimization of Natural Leads to Anticancer Drugs or Drug Candidates</span></div><div class="casAuthors">Xiao, Zhiyan; Morris-Natschke, Susan L.; Lee, Kuo-Hsiung</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-91</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Natural products have made significant contribution to cancer chemotherapy over the past decades and remain an indispensable source of mol. and mechanistic diversity for anticancer drug discovery.  More often than not, natural products may serve as leads for further drug development rather than as effective anticancer drugs by themselves.  Generally, optimization of natural leads into anticancer drugs or drug candidates should not only address drug efficacy, but also improve absorption, distribution, metab., excretion, and toxicity (ADMET) profiles and chem. accessibility assocd. with the natural leads.  Optimization strategies involve direct chem. manipulation of functional groups, structure-activity relationship directed optimization and pharmacophore-oriented mol. design based on the natural templates.  Both fundamental medicinal chem. principles (e.g., bioisosterism) and state-of-the-art computer-aided drug design techniques (e.g., structure-based design) can be applied to facilitate optimization efforts.  In this review, the strategies to optimize natural leads to anticancer drugs or drug candidates are illustrated with examples and described according to their purposes.  Furthermore, successful case studies on lead optimization of bioactive compds. performed in the Natural Products Research Labs. at UNC are highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6SGKXSp6ip7Vg90H21EOLACvtfcHk0lh0JHE4fZOSyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVagsb%252FJ&md5=dc8c68d23f1f8e9ff6e98847a69558c4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fmed.21377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21377%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DMorris-Natschke%26aufirst%3DS.%2BL.%26aulast%3DLee%26aufirst%3DK.-H.%26atitle%3DStrategies%2520for%2520the%2520optimization%2520of%2520natural%2520leads%2520to%2520anticancer%2520drugs%2520or%2520drug%2520candidates%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2016%26volume%3D36%26spage%3D32%26epage%3D91%26doi%3D10.1002%2Fmed.21377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mintoo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guru, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuthakki, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondhe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhushan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Discovery and preclinical development of IIIM-290, an orally active potent cyclin-dependent kinase Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1664</span>– <span class="NLM_lpage">1687</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsl2js7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1664-1687&author=S.+B.+Bharateauthor=V.+Kumarauthor=S.+K.+Jainauthor=M.+J.+Mintooauthor=S.+K.+Guruauthor=V.+K.+Nuthakkiauthor=M.+Sharmaauthor=S.+S.+Bharateauthor=S.+G.+Gandhiauthor=D.+M.+Mondheauthor=S.+Bhushanauthor=R.+A.+Vishwakarma&title=Discovery+and+preclinical+development+of+IIIM-290%2C+an+orally+active+potent+cyclin-dependent+kinase+Inhibitor&doi=10.1021%2Facs.jmedchem.7b01765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Preclinical Development of IIIM-290, an Orally Active Potent Cyclin-Dependent Kinase Inhibitor</span></div><div class="casAuthors">Bharate, Sandip B.; Kumar, Vikas; Jain, Shreyans K.; Mintoo, Mubashir J.; Guru, Santosh K.; Nuthakki, Vijay K.; Sharma, Mohit; Bharate, Sonali S.; Gandhi, Sumit G.; Mondhe, Dilip M.; Bhushan, Shashi; Vishwakarma, Ram A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1664-1687</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Rohitukine (1), a chromone alkaloid isolated from Indian medicinal plant Dysoxylum binectariferum, has inspired the discovery of flavopiridol and riviciclib, both of which are bioavailable only via i.v. route.  With the objective to address the oral bioavailability issue of this scaffold, four series of rohitukine derivs. were prepd. and screened for Cdk inhibition and cellular antiproliferative activity.  The 2,6-dichloro-styryl deriv. IIIM-290 (11d) showed strong inhibition of Cdk-9/T1 (IC50 1.9 nM) kinase and Molt-4/MIAPaCa-2 cell growth (GI50 < 1.0 μM) and was found to be highly selective for cancer cells over normal fibroblast cells.  It inhibited the cell growth of MIAPaCa-2 cells via caspase-dependent apoptosis.  It achieved 71% oral bioavailability with in vivo efficacy in pancreatic, colon, and leukemia xenografts at 50 mg/kg, po.  It did not have CYP/efflux-pump liability, was not mutagenic/genotoxic or cardiotoxic, and was metabolically stable.  The preclin. data presented herein indicates the potential of 11d for advancement in clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeRTRG-f8KqLVg90H21EOLACvtfcHk0lh0JHE4fZOSyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsl2js7k%253D&md5=13b67cdf1db966e2a414b64cd82a98e8</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01765%26sid%3Dliteratum%253Aachs%26aulast%3DBharate%26aufirst%3DS.%2BB.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DJain%26aufirst%3DS.%2BK.%26aulast%3DMintoo%26aufirst%3DM.%2BJ.%26aulast%3DGuru%26aufirst%3DS.%2BK.%26aulast%3DNuthakki%26aufirst%3DV.%2BK.%26aulast%3DSharma%26aufirst%3DM.%26aulast%3DBharate%26aufirst%3DS.%2BS.%26aulast%3DGandhi%26aufirst%3DS.%2BG.%26aulast%3DMondhe%26aufirst%3DD.%2BM.%26aulast%3DBhushan%26aufirst%3DS.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DDiscovery%2520and%2520preclinical%2520development%2520of%2520IIIM-290%252C%2520an%2520orally%2520active%2520potent%2520cyclin-dependent%2520kinase%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1664%26epage%3D1687%26doi%3D10.1021%2Facs.jmedchem.7b01765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinase inhibition by flavoalkaloids</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">632</span>– <span class="NLM_lpage">649</span>, <span class="refDoi"> DOI: 10.2174/138955712800626683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.2174%2F138955712800626683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=22512551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC38XoslCkurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=632-649&author=S.+K.+Jainauthor=S.+B.+Bharateauthor=R.+A.+Vishwakarma&title=Cyclin-dependent+kinase+inhibition+by+flavoalkaloids&doi=10.2174%2F138955712800626683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinase inhibition by flavoalkaloids</span></div><div class="casAuthors">Jain, S. K.; Bharate, S. B.; Vishwakarma, R. A.</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">632-649</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Chromone alkaloids and flavoalkaloids are an important group of natural products possessing promising medicinal properties.  A chromone alkaloid rohitukine is a major bioactive chem. constituent of plant Dysoxylum binectariferum (Meliaceae) which is phylogenetically related to the Ayurvedic plant, D. malabaricum used for treatment of rheumatoid arthritis.  This chromone alkaloid led to discovery of 2 synthetic flavoalkaloids: flavopiridol (Sanofi) and P-276-00 (Piramal) which have reached to advanced stages of clin. development for cancer treatment.  Flavopiridol (Alvocidib; L868275; HMR-1275; NSC 649890 of Sanofi-Aventis + NCI) is approved as an orphan drug for treatment of chronic lymphocytic leukemia and is currently undergoing phase II studies as monotherapy and also as in combination regimes with traditional chemotherapy agents.  P-276-00 (12) is currently in phase II clin. studies for advanced refractory neoplasms and multiple myeloma.  Extensive amt. of medicinal chem. efforts were reported on these flavoalkaloids.  Flavopiridol demonstrated potent and specific in vitro inhibition of variety of cyclin-dependent kinases with clear block in cell cycle progression at the G1/S and G2/M phases.  Preclin. studies demonstrated the capacity of flavopiridol to induce programmed cell death, promote differentiation, inhibit angiogenic processes and modulate transcriptional events.  The co-crystd. structure of deschloro-flavopiridol with CDK-2 is available and key interactions in the ATP binding site were reported.  Flavopiridol was also studied for the treatment of arthritis and atherosclerotic plaque formation.  The present review comprises discovery, medicinal chem., pharmacol. and preclin./clin. development of flavoalkaloids as CDK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDQdAK15QBLLVg90H21EOLACvtfcHk0lh0JHE4fZOSyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XoslCkurs%253D&md5=98702838814112f01f0f7585d9e6e994</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.2174%2F138955712800626683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138955712800626683%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DS.%2BK.%26aulast%3DBharate%26aufirst%3DS.%2BB.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DCyclin-dependent%2520kinase%2520inhibition%2520by%2520flavoalkaloids%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D632%26epage%3D649%26doi%3D10.2174%2F138955712800626683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanjo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akira, S.</span></span> <span> </span><span class="NLM_article-title">DRAKs, novel serine/threonine kinases related to death-associated protein kinase that trigger apoptosis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">29066</span>– <span class="NLM_lpage">29071</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.44.29066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1074%2Fjbc.273.44.29066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=9786912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADyaK1cXntlaht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=29066-29071&author=H.+Sanjoauthor=T.+Kawaiauthor=S.+Akira&title=DRAKs%2C+novel+serine%2Fthreonine+kinases+related+to+death-associated+protein+kinase+that+trigger+apoptosis&doi=10.1074%2Fjbc.273.44.29066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">DRAKs, novel serine/threonine kinases related to death-associated protein kinase that trigger apoptosis</span></div><div class="casAuthors">Sanjo, Hideki; Kawai, Taro; Akira, Shizuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">29066-29071</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The present study describes the cloning of two novel serine/threonine kinases termed DRAK1 and DRAK2, whose catalytic domains are related to that of death-assocd. protein kinase, a serine/threonine kinase involved in apoptosis.  Both DRAKs are composed of the N-terminal catalytic domain and the C-terminal domain that is responsible for regulation of kinase activity.  DRAK1 and DRAK2 show 59.7% identity and display ubiquitous expression.  An in vitro kinase assay revealed that both DRAKs are autophosphorylated and phosphorylate myosin light chain as an exogenous substrate, although the kinase activity of DRAK2 is significantly lower than that of DRAK1.  Both DRAKs are exclusively localized to the nucleus.  Furthermore, overexpression of both DRAKs induces the morphol. changes of apoptosis in NIH 3T3 cells, suggesting the role of DRAKs in apoptotic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmFXOHrs4s2bVg90H21EOLACvtfcHk0ljcYS7DZrXaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntlaht7s%253D&md5=6b17b0e7dd8e61ec3b435b245cbcb627</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.44.29066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.44.29066%26sid%3Dliteratum%253Aachs%26aulast%3DSanjo%26aufirst%3DH.%26aulast%3DKawai%26aufirst%3DT.%26aulast%3DAkira%26aufirst%3DS.%26atitle%3DDRAKs%252C%2520novel%2520serine%252Fthreonine%2520kinases%2520related%2520to%2520death-associated%2520protein%2520kinase%2520that%2520trigger%2520apoptosis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D29066%26epage%3D29071%26doi%3D10.1074%2Fjbc.273.44.29066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebakken, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riddle, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frazee, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichling, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, K. W.</span></span> <span> </span><span class="NLM_article-title">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1177/1087057109339207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1177%2F1087057109339207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=19564447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=924-935&author=C.+S.+Lebakkenauthor=S.+M.+Riddleauthor=U.+Singhauthor=W.+J.+Frazeeauthor=H.+C.+Eliasonauthor=Y.+Gaoauthor=L.+J.+Reichlingauthor=B.+D.+Marksauthor=K.+W.+Vogel&title=Development+and+applications+of+a+broad-coverage%2C+TR-FRET-based+kinase+binding+assay+platform&doi=10.1177%2F1087057109339207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Development and applications of a broad-coverage, TR-FRET-based kinase binding assay platform</span></div><div class="casAuthors">Lebakken, Connie S.; Riddle, Steven M.; Singh, Upinder; Frazee, W. Jack; Eliason, Hildegard C.; Gao, Yi; Reichling, Laurie J.; Marks, Bryan D.; Vogel, Kurt W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">924-935</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The expansion of kinase assay technologies over the past decade has mirrored the growing interest in kinases as drug targets.  As a result, there is no shortage of convenient, fluorescence-based methods available to assay targets that span the kinome.  The authors recently reported on the development of a non-activity-based assay to characterize kinase inhibitors that depended on displacement of an Alexa Fluor 647 conjugate of staurosporine (a "tracer") from a particular kinase.  Kinase inhibitors were characterized by a change in fluorescence lifetime of the tracer when it was bound to a kinase relative to when it was displaced by an inhibitor.  Here, the authors report on improvements to this strategy by reconfiguring the assay in a time-resolved fluorescence resonance energy transfer (TR-FRET) format that simplifies instrumentation requirements and allows for the use of a substantially lower concn. of kinase than was required in the fluorescence-lifetime-based format.  The authors use this new assay to demonstrate several aspects of the binding assay format that are advantageous relative to traditional activity-based assays.  The TR-FRET binding format facilitates the assay of compds. against low-activity kinases, allows for the characterization of type II kinase inhibitors either using nonactivated kinases or by monitoring compd. potency over time, and ensures that the signal being detected is specific to the kinase of interest and not a contaminating kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqPbcMF0-PD7Vg90H21EOLACvtfcHk0ljcYS7DZrXaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSjtLnM&md5=f8df51adf90d821e480cf706213cc096</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1177%2F1087057109339207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057109339207%26sid%3Dliteratum%253Aachs%26aulast%3DLebakken%26aufirst%3DC.%2BS.%26aulast%3DRiddle%26aufirst%3DS.%2BM.%26aulast%3DSingh%26aufirst%3DU.%26aulast%3DFrazee%26aufirst%3DW.%2BJ.%26aulast%3DEliason%26aufirst%3DH.%2BC.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DReichling%26aufirst%3DL.%2BJ.%26aulast%3DMarks%26aufirst%3DB.%2BD.%26aulast%3DVogel%26aufirst%3DK.%2BW.%26atitle%3DDevelopment%2520and%2520applications%2520of%2520a%2520broad-coverage%252C%2520TR-FRET-based%2520kinase%2520binding%2520assay%2520platform%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2009%26volume%3D14%26spage%3D924%26epage%3D935%26doi%3D10.1177%2F1087057109339207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>– <span class="NLM_lpage">214</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.+T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+Mullerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0ljcYS7DZrXaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.%2BT.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sipes, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, T. B.</span></span> <span> </span><span class="NLM_article-title">Zebrafish: as an integrative model for twenty-first century toxicity testing</span>. <i>Birth Defects Res., Part C</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">256</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1002/bdrc.20214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=10.1002%2Fbdrc.20214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=21932434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFyjs7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2011&pages=256-267&author=N.+S.+Sipesauthor=S.+Padillaauthor=T.+B.+Knudsen&title=Zebrafish%3A+as+an+integrative+model+for+twenty-first+century+toxicity+testing&doi=10.1002%2Fbdrc.20214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Zebrafish-As an integrative model for twenty-first century toxicity testing</span></div><div class="casAuthors">Sipes, Nisha S.; Padilla, Stephanie; Knudsen, Thomas B.</div><div class="citationInfo"><span class="NLM_cas:title">Birth Defects Research, Part C: Embryo Today--Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">256-267</span>CODEN:
                <span class="NLM_cas:coden">BDRPDV</span>;
        ISSN:<span class="NLM_cas:issn">1542-975X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  The zebrafish embryo is a useful small model for investigating vertebrate development because of its transparency, low cost, transgenic and morpholino capabilities, conservation of cell signaling, and concordance with mammalian developmental phenotypes.  From these advantages, the zebrafish embryo has been considered as an alternative model for traditional in vivo developmental toxicity screening.  The use of this organism in conjunction with traditional in vivo developmental toxicity testing has the potential to reduce cost and increase throughput of testing the chem. universe, prioritize chems. for targeted toxicity testing, generate predictive models of developmental toxicants, and elucidate mechanisms and adverse outcome pathways for abnormal development.  This review gives an overview of the zebrafish embryo for predictive toxicol. and 21st century toxicity testing.  Developmental eye defects were selected as an example to evaluate data from the U.S.  Environmental Protection Agency's ToxCast program comparing responses in zebrafish embryos with those from pregnant rats and rabbits for a subset of 24 environmental chems. across >600 in vitro assay targets.  Cross-species comparisons implied a common basis for biol. pathways assocd. with neuronal defects, extracellular matrix remodeling, and mitotic arrest.  Birth Defects Research (Part C) 93:256-267, 2011. © 2011 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3C_KqV8TTKrVg90H21EOLACvtfcHk0lhdVzF4WI_HFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFyjs7rL&md5=e86f008ee9675a5a817d3ea006566b07</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fbdrc.20214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbdrc.20214%26sid%3Dliteratum%253Aachs%26aulast%3DSipes%26aufirst%3DN.%2BS.%26aulast%3DPadilla%26aufirst%3DS.%26aulast%3DKnudsen%26aufirst%3DT.%2BB.%26atitle%3DZebrafish%253A%2520as%2520an%2520integrative%2520model%2520for%2520twenty-first%2520century%2520toxicity%2520testing%26jtitle%3DBirth%2520Defects%2520Res.%252C%2520Part%2520C%26date%3D2011%26volume%3D93%26spage%3D256%26epage%3D267%26doi%3D10.1002%2Fbdrc.20214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4K1B','PDB','4K1B'); return false;">PDB: 4K1B</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i65"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01006">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_68127"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01006">10.1021/acs.jmedchem.9b01006</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">NMR spectra for compounds <b>3</b>–<b>8</b>, synthetic intermediates toward <b>3</b>, synthetic <b>3</b> and <b>4</b>, isolated <b>1</b> and <b>2</b>, and comparison of <sup>1</sup>H- and <sup>13</sup>C NMR data for isolated and synthetic <b>3</b> and <b>4</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_001.pdf">PDF</a>)</p></li><li><p class="inline">The activities of <b>1</b>, <b>3</b>, and <b>4</b> against 140 kinases using a radioactive (<sup>33</sup>P-ATP) filter-binding assays, binding affinity of <b>3</b> against the 468 kinases in the DiscoverX ScanMax panel, diversity profile of <b>3</b>, IC<sub>50</sub> values of <b>3</b> against the 88 cell lines found in the OncoLead panel, TreeSpot analysis of <b>3</b> at 5 μM based on Discoverx data, curve images for <i>K</i><sub>d</sub> determination of <b>3</b>, dose–response-time curves for <b>3</b> against HT-29 and <b>4</b> against MCF-7 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_001.pdf">jm9b01006_si_001.pdf (2.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_002.pdf">jm9b01006_si_002.pdf (597.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01006/suppl_file/jm9b01006_si_003.csv">jm9b01006_si_003.csv (3.2 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01006&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01006%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-22" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01006" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a184cc983c7c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
