
<html lang="en"     class="pb-page"  data-request-id="10b235c4-35a6-4b00-83aa-f6e4b4b3ac93"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b00993;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-3;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53–MDM2/MDMX Protein–Protein Interactions" /></meta><meta name="dc.Creator" content="Peng  Sang" /></meta><meta name="dc.Creator" content="Yan  Shi" /></meta><meta name="dc.Creator" content="Junhao  Lu" /></meta><meta name="dc.Creator" content="Lihong  Chen" /></meta><meta name="dc.Creator" content="Leixiang  Yang" /></meta><meta name="dc.Creator" content="Wade  Borcherds" /></meta><meta name="dc.Creator" content="Sami  Abdulkadir" /></meta><meta name="dc.Creator" content="Qi  Li" /></meta><meta name="dc.Creator" content="Gary  Daughdrill" /></meta><meta name="dc.Creator" content="Jiandong  Chen" /></meta><meta name="dc.Creator" content="Jianfeng  Cai" /></meta><meta name="dc.Description" content="The use of peptidomimetic scaffolds is a promising strategy for the inhibition of protein–protein interactions (PPIs). Herein, we demonstrate that sulfono-γ-AApeptides can be rationally designed to..." /></meta><meta name="Description" content="The use of peptidomimetic scaffolds is a promising strategy for the inhibition of protein–protein interactions (PPIs). Herein, we demonstrate that sulfono-γ-AApeptides can be rationally designed to..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 23, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00993" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00993" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00993" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00993" /></link>
        
    
    

<title>α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53–MDM2/MDMX Protein–Protein Interactions | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00993" /></meta><meta property="og:title" content="α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53–MDM2/MDMX Protein–Protein Interactions" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0009.jpeg" /></meta><meta property="og:description" content="The use of peptidomimetic scaffolds is a promising strategy for the inhibition of protein–protein interactions (PPIs). Herein, we demonstrate that sulfono-γ-AApeptides can be rationally designed to mimic the p53 α-helix and inhibit p53–MDM2 PPIs. The best inhibitor, with Kd and IC50 values of 26 nM and 0.891 μM toward MDM2, respectively, is among the most potent unnatural peptidomimetic inhibitors disrupting the p53–MDM2/MDMX interaction. Using fluorescence polarization assays, circular dichroism, nuclear magnetic resonance spectroscopy, and computational simulations, we demonstrate that sulfono-γ-AApeptides adopt helical structures resembling p53 and competitively inhibit the p53–MDM2 interaction by binding to the hydrophobic cleft of MDM2. Intriguingly, the stapled sulfono-γ-AApeptides showed promising cellular activity by enhancing p53 transcriptional activity and inducing expression of MDM2 and p21. Moreover, sulfono-γ-AApeptides exhibited remarkable resistance to proteolysis, augmenting their biological potential. Our results suggest that sulfono-γ-AApeptides are a new class of unnatural helical foldamers that disrupt PPIs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00993"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00993">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00993&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00993&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00993&amp;href=/doi/10.1021/acs.jmedchem.9b00993" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 975-986</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00739" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01154" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.0c01284"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53–MDM2/MDMX Protein–Protein Interactions</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Peng Sang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peng Sang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peng++Sang">Peng Sang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yan Shi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yan Shi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Shi">Yan Shi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Junhao Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junhao Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junhao++Lu">Junhao Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lihong Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lihong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lihong++Chen">Lihong Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leixiang Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leixiang Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leixiang++Yang">Leixiang Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wade Borcherds</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wade Borcherds</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cell Biology, Microbiology and Molecular Biology, University of South Florida, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wade++Borcherds">Wade Borcherds</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sami Abdulkadir</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sami Abdulkadir</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sami++Abdulkadir">Sami Abdulkadir</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qi Li</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qi Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Shuguang Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#7e120409183e16110a131f1712501d1113"><span class="__cf_email__" data-cfemail="e4889e9382a48c8b9089858d88ca878b89">[email protected]</span></a> (Q.L.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qi++Li">Qi Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gary Daughdrill</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gary Daughdrill</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cell Biology, Microbiology and Molecular Biology, University of South Florida, Tampa, Florida 33620, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#6e090a0f1b09060a1c0702022e1b1d08400b0a1b"><span class="__cf_email__" data-cfemail="aacdcecbdfcdc2ced8c3c6c6eadfd9cc84cfcedf">[email protected]</span></a> (G.D.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gary++Daughdrill">Gary Daughdrill</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiandong Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiandong Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular Oncology, H. Lee Moffitt Cancer Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4f25262e212b202128612c272a210f22202929263b3b61203d28"><span class="__cf_email__" data-cfemail="442e2d252a202b2a236a272c212a04292b22222d30306a2b3623">[email protected]</span></a> (Jiandong Chen)</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiandong++Chen">Jiandong Chen</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jianfeng Cai</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianfeng Cai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#4f25262e21292a21282c2e260f3a3c29612a2b3a"><span class="__cf_email__" data-cfemail="7d17141c131b18131a1e1c143d080e1b53181908">[email protected]</span></a> (Jianfeng Cai).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianfeng++Cai">Jianfeng Cai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3106-3306" title="Orcid link">http://orcid.org/0000-0003-3106-3306</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00993&amp;href=/doi/10.1021%2Facs.jmedchem.9b00993" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 975–986</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 23, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 June 2019</li><li><span class="item_label"><b>Published</b> online</span>23 January 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 February 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00993" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00993</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00993"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2570</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00993" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;α-Helix-Mimicking Sulfono-γ-AApeptide Inhibitors for p53–MDM2/MDMX Protein–Protein Interactions&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Peng&quot;,&quot;last_name&quot;:&quot;Sang&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Junhao&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Lihong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Leixiang&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Wade&quot;,&quot;last_name&quot;:&quot;Borcherds&quot;},{&quot;first_name&quot;:&quot;Sami&quot;,&quot;last_name&quot;:&quot;Abdulkadir&quot;},{&quot;first_name&quot;:&quot;Qi&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Gary&quot;,&quot;last_name&quot;:&quot;Daughdrill&quot;},{&quot;first_name&quot;:&quot;Jiandong&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jianfeng&quot;,&quot;last_name&quot;:&quot;Cai&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;975-986&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00993&quot;},&quot;abstract&quot;:&quot;The use of peptidomimetic scaffolds is a promising strategy for the inhibition of protein–protein interactions (PPIs). Herein, we demonstrate that sulfono-γ-AApeptides can be rationally designed to mimic the p53 α-helix and inhibit p53–MDM2 PPIs. The best inhibitor, with Kd and IC50 values of 26 nM and 0.891 μM toward MDM2, respectively, is among the most potent unnatural peptidomimetic inhibitors disrupting the p53–MDM2/MDMX interaction. Using fluorescence polarization assays, circular dichroism, nuclear magnetic resonance spectroscopy, and computational simulations, we demonstrate that sulfono-γ-AApeptides adopt helical structures resembling p53 and competitively inhibit the p53–MDM2 interaction by binding to the hydrophobic cleft of MDM2. Intriguingly, the stapled sulfono-γ-AApeptides showed promising cellular activity by enhancing p53 transcriptional activity and inducing expression of MDM2 and p21. Moreover, sulfono-γ-AApeptides exhibited remarkable resistance to proteolysis, augmenting their biological &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00993&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00993" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00993&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00993" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00993&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00993" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00993&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00993&amp;href=/doi/10.1021/acs.jmedchem.9b00993" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00993" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00993" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00993%26sid%3Dliteratum%253Aachs%26pmid%3D31971801%26genre%3Darticle%26aulast%3DSang%26date%3D2020%26atitle%3D%25CE%25B1-Helix-Mimicking%2BSulfono-%25CE%25B3-AApeptide%2BInhibitors%2Bfor%2Bp53%25E2%2580%2593MDM2%252FMDMX%2BProtein%25E2%2580%2593Protein%2BInteractions%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D3%26spage%3D975%26epage%3D986%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290631" title="Foldamers">Foldamers</a>,</li><li><a href="/action/doSearch?ConceptID=290914" title="Peptidomimetics">Peptidomimetics</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=291675" title="Stability">Stability</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/jmcmar.2020.63.issue-3/20200213/jmcmar.2020.63.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The use of peptidomimetic scaffolds is a promising strategy for the inhibition of protein–protein interactions (PPIs). Herein, we demonstrate that sulfono-γ-AApeptides can be rationally designed to mimic the p53 α-helix and inhibit p53–MDM2 PPIs. The best inhibitor, with <i>K</i><sub>d</sub> and IC<sub>50</sub> values of 26 nM and 0.891 μM toward MDM2, respectively, is among the most potent unnatural peptidomimetic inhibitors disrupting the p53–MDM2/MDMX interaction. Using fluorescence polarization assays, circular dichroism, nuclear magnetic resonance spectroscopy, and computational simulations, we demonstrate that sulfono-γ-AApeptides adopt helical structures resembling p53 and competitively inhibit the p53–MDM2 interaction by binding to the hydrophobic cleft of MDM2. Intriguingly, the stapled sulfono-γ-AApeptides showed promising cellular activity by enhancing p53 transcriptional activity and inducing expression of MDM2 and p21. Moreover, sulfono-γ-AApeptides exhibited remarkable resistance to proteolysis, augmenting their biological potential. Our results suggest that sulfono-γ-AApeptides are a new class of unnatural helical foldamers that disrupt PPIs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85241" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85241" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Foldamers,<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> the synthetic unnatural oligomers that fold with high stability, have raised new prospects for mimicking the 3D structure and function of bioactive molecules with enhanced resistance to proteolytic degradation and sequence diversity compared to canonical peptides.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15">(6−15)</a> In the past 2 decades, several important foldamer systems have been developed to target proteins<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> and membranes.<a onclick="showRef(event, 'ref6 ref15 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref6 ref15 ref19 ref20 ref21 ref22 ref23">(6,15,19−23)</a> Prominent examples include β-peptides,<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> peptoids,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> β-peptoids,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> oligoureas,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> azapeptides,<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> α-aminoisobutyric acid foldamers,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> oligoproline,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> aromatic amide foldamers,<a onclick="showRef(event, 'ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35">(33−35)</a> and others. In particular, helical foldamers have been explored extensively for inhibition of protein–protein interactions (PPIs).<a onclick="showRef(event, 'ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39">(36−39)</a> For instance, the inhibition of interaction between the tumor suppressor protein 53 and human double minute 2 (p53–MDM2) using designed unnatural helical peptidomimetics has been considerably investigated because p53–MDM2 PPI plays a critical role in cancer development and progression.<a onclick="showRef(event, 'ref17 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref17 ref40 ref41 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50 ref51">(17,40−51)</a> More importantly, the interaction of the p53 helical domain with MDM2 is well characterized (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), and it has been used as a testing ground to prove the ability of peptidomimetics for the mimicry of α-helix.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Interaction of p53 with the crystal structure of MDM2 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). p53 is shown as the cartoon, whereas MDM2 is shown as the surface representation. (B) Chemical structure of sulfono-γ-AApeptides. a and b denote the chiral side chain and the sulfonamido side chain from the building block, respectively. (C) Crystal structure of a sulfono-γ-AApeptide (CCDC: 1841094).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> (D) Top view of (C). (E,F) Schematic representation of distribution of side chains from sulfono-γ-AApeptides. (E) Side view; (F) top view, helical wheel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">MDM2 is an E3 ubiquitin ligase overexpressed or activated in many cancers.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> It specifically binds to p53 to induce its proteasomal degradation. As MDM2 negatively regulates p53 tumor suppressor activity, disrupting p53–MDM2 PPI is a promising strategy for the development of anticancer therapeutics.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The crystal structure of p53–MDM2 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) reveals that the helical domain of p53 binds to the hydrophobic cleft of MDM2. The three critical residues, Phe19, Trp23, and Leu26, which are on the same face of the p53 helix, deeply insert into the MDM2 hydrophobic pocket and contribute bulk of the binding energy.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Therefore, molecules that mimic the α-helix of p53 and project functional groups analogous to those three critical residues in p53 could bind to MDM2 and disrupt the interaction.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Meanwhile, MDMX (also known as MDM4) is a structural analogue of MDM2.<a onclick="showRef(event, 'ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref56 ref57">(55−57)</a> It also binds to p53 and blocks transactivation through N-terminal region, just like MDM2. The difference between them is the overall mechanism of regulation. P53 also induces MDMX mRNA transcription by binding to its promoter in certain cell types.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> MDMX inhibits p53 DNA binding and transcription activation function but is not a significant regulator of its degradation. The expression of MDMX is not regulated by p53 and it is not shown to target p53 to degrade; however, it does inhibit p53 transcriptional activity<a onclick="showRef(event, 'ref55 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref57">(55,57)</a> Because of the different physiological roles of MDMX compared to MDM2, efforts to develop new inhibitors to selectively target p53–MDM2 or p53/MDMX or simultaneously target both p53/MDM2 and p53/MDMX binding pocket are of medicinal significance.</div><div class="NLM_p last">Sulfono-γ-AApeptides are the subclass of γ-AApeptides recently developed in our lab.<a onclick="showRef(event, 'ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref59 ref60">(59,60)</a> Similar to γ-AApeptides, half of the side chains in sulfono-γ-AApeptides are introduced by sulfonyl chlorides, providing enormous chemical diversity (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). In comparison to α-helical peptides, sulfono-γ-AApeptides are highly resistant to proteolytic degradation.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a> Notably, sulfono-γ-AApeptides have excellent folding stability and fold into helical structures with a well-defined hydrogen bonding pattern.<a onclick="showRef(event, 'ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref63 ref64 ref65">(63−65)</a> We recently determined the X-ray crystal structures of a series of homogeneous <span class="smallcaps smallerCapital">l</span>-sulfono-γ-AA foldamers.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> The oligomers fold into an unprecedented and surprisingly left-handed 4<sub>14</sub> helices (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D). Nonetheless, the helical parameters of sulfono-γ-AApeptide foldamers are highly consistent irrespective of side chains, suggesting that the helical propensity is intrinsic and dominated by the molecular scaffold of sulfono-γ-AApeptides. As sulfono-γ-AApeptide foldamers bear a helical pitch of 5.1 Å, which is similar to that of the α-helix (5.4 Å) and display precise arrangement of functional groups in three dimensions, it is envisioned that this class of helical foldamer could be used to mimic α-helices involved in protein–protein interactions (PPIs). Indeed, sulfono-γ-AA foldamers have exactly four side chains per turn, with all side chains aligning perfectly on top of each other in four directions (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C), resulting in a rectangular helical scaffold. The side chain arrangement of sulfono-γ-AApeptides is highly similar to the α-helix, making it straightforward to design α-helical mimetics projecting critical residues on one face of the helix. Based on this concept, we envision that sulfono-γ-AApeptides could be used to develop a new class of α-helix-like inhibitors for PPIs. We recently illustrated this α-helix-mimicking strategy by developing helical sulfono-γ-AApeptides that structurally and functionally mimic the α-helical domain of BCL9 and disrupt BCL9/β-catenin PPI.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of Sulfono-γ-AApeptides and Their Biological Activity</h3><div class="NLM_p">On the basis of the structure of helical sulfono-γ-AApeptides, we set out to design sequences that mimic the α-helix of p53. Although we previously reported homogeneous γ-AApeptides for inhibition of p53–MDM2 interaction,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> because of their inability in helical structure formation, the design was highly speculative and experiential, leading to weak inhibitors (IC<sub>50</sub> ≥ 38 μM). With the availability of helical sulfono-γ-AApeptides, the design became very straightforward.</div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E,F, the first sulfono-γ-AApeptide sequence we designed (<b>PS1</b>) had chiral side chains at positions 2a, 4a, and 6a, which are same as the side chains of Phe19, Trp23, and Leu 26 in p53 that are critical for binding of MDM2 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E,F). The binding affinity of <b>PS1</b> was determined by fluorescence polarization assays (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf" class="ext-link">Figure S3</a>).<a onclick="showRef(event, 'ref47 ref69'); return false;" href="javascript:void(0);" class="ref ref47 ref69">(47,69)</a> Both p53 (16–29) and Nutlin were also included in the study as the comparison.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the <i>K</i><sub>d</sub> of the p53 peptide is 208 nM, which is in good agreement with the literature.<a onclick="showRef(event, 'ref52 ref70'); return false;" href="javascript:void(0);" class="ref ref52 ref70">(52,70)</a> The fluorescence polarization (FP) competition study using a fluorescein isothiocyanate (FITC)-labeled p53 peptide led to an IC<sub>50</sub> of 4.61 μM, again consistent with previous results. Interestingly, <b>PS1</b> bound to MDM2 with <i>K</i><sub>d</sub> value of 98 nM, which is twofold more potent than the p53 peptide. The <b>PS2</b> sequence was created by deleting one sulfono-γ-AA building block (two side chains) at the C-terminus of <b>PS1</b>. <b>PS2</b> exhibited a virtually identical <i>K</i><sub>d</sub> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>PS2</b>) to <b>PS1</b> with a slightly reduced inhibitory activity (IC<sub>50</sub> = 10.3 μM). To further understand the relationship between the binding affinity of sulfono-γ-AApeptides and side chains, we first fixed the three critical side chains (2a, 4a, and 6a) and changed the side chains at different positions relative to the sulfono-γ-AApeptide <b>PS1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>PS3-7</b>). As expected, sequences <b>PS3</b> and <b>PS5-7</b> did not dramatically change the binding affinity, further suggesting that the binding activity was mainly governed by those three critical side chains at position 2a, 4a, and 6a. Interestingly, mutation of the methyl group to the 4-chloro phenyl group in <b>PS4</b> did improve the binding affinity with <i>K</i><sub>d</sub> and IC<sub>50</sub> values of 57 nM (∼fourfold improvement) and 2.8 μM (∼1.5-fold improvement), respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf" class="ext-link">Figure S3</a>), indicating that side chains could affect the binding activity even if they were not involved in direct contact with the MDM2 p53 binding pocket. The vital importance of critical side chains was manifested by <b>PS8</b>. It lacks a single key side chain at position 4a that mimics the residue Trp23 in p53 and completely lost its ability to bind MDM2. It is known that replacement of Leu26 with other bulkier residues could improve the binding affinity of p53 peptide derivatives because of their enhanced hydrophobic interactions with MDM2.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> We therefore designed the sequence <b>PS9</b>, in which the side chain 6a was replaced with the cyclohexylmethyl group. To our surprise, the modification slightly diminished the binding affinity, whereas its ability to inhibit p53–MDM2 interaction was better than p53(16–29). It is likely that the side chain of the cyclohexylmethyl group is too long, which prevented the close contact of <b>PS9</b> with MDM2. Thus, <b>PS10</b> was synthesized with the modification of 6a with the cyclobutylmethyl group, which led to the significant improvement in binding potency to MDM2 with <i>K</i><sub>d</sub> and IC<sub>50</sub> values of 26 nM and 0.891 μM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The inhibitory activity is comparable to that of Nutlin (IC<sub>50</sub> = 0.6 μM), which is a benchmark small molecule MDM2 inhibitor.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> To the best of our knowledge, <b>PS10</b> is among the most potent peptidomimetic foldamers that antagonize p53–MDM2 interaction in the literature.<a onclick="showRef(event, 'ref52 ref56'); return false;" href="javascript:void(0);" class="ref ref52 ref56">(52,56)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>K</i><sub>d</sub> and IC<sub>50</sub> data of pure p53 (A) and sulfono-γ-AApeptide <b>PS10</b> (B) to MDM2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures of Sulfono-γ-AApeptides Investigated for the Disruption p53–MDM2 Interaction<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0007.gif" alt="" id="gr7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The side chains mimicking Phe19, Trp23, and Leu26 in p53 are shown in blue.</p></div></div><div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Circular Dichroism Measurements</h3><div class="NLM_p">We reasoned that the inhibitory activity of sulfono-γ-AApeptides against p53–MDM2 PPI originates from their intrinsic helical propensity. It is well established that the p53 transactivation domain is disordered and has to fold in order to bind MDM2. The loss of configurational entropy associated with p53 folding reduces the binding affinity relative to more helical peptides.<a onclick="showRef(event, 'ref57 ref72'); return false;" href="javascript:void(0);" class="ref ref57 ref72">(57,72)</a> The circular dichroism (CD) spectroscopy for the peptide p53(16–29) and the 10 homogeneous sulfono-γ-AApeptides were next recorded in phosphate-buffered saline (PBS) buffer between 190 and 270 nm. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, each of the sulfono-γ-AApeptides revealed a marked cotton effect with the strong positive maximum at around 208 nm, which is consistent with the homogeneous sulfono-γ-AApeptide single crystal in the literature,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> thus suggesting that sequences <b>PS1-10</b> adopt a similar left-handed helical conformation. On the other hand, a minimum of less than 200 nm and the lack of 222 nm was observed for p53(16–29), signifying random coil and almost no α-helix population. As comparison, the helical stability of sequences <b>PS1-10</b> was also investigated in trifluoroethanol (TFE). The CD spectra of these sequences in TFE revealed a similar left-handed helical conformation as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, which is consistent with a previous report,<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> demonstrating the robust helicity of this class of peptidomimetics.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A). CD spectra of <b>p53</b> and sulfono-γ-AApeptides (100 μM) measured at room temperature in PBS buffer. (B) CD spectra of <b>p53</b> and sulfono-γ-AApeptides (100 μM) in TFE at room temperature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> <sup>15</sup>N–<sup>1</sup>H HSQC NMR of <b>PS10</b> in Complex with MDM2</h3><div class="NLM_p">In order to gain insight into the structural basis of sulfono-γ-AApeptide binding to MDM2, the lead linear peptide <b>PS10</b> was further assessed using nuclear magnetic resonance (NMR) spectroscopy. We have shown that <b>PS10</b> binds to MDM2 with nanomolar affinity and binds 10 times more tightly than p53 TAD. Chemical shift mapping was performed to determine whether the <b>PS10</b> binding site overlaps with the p53 TAD binding site. NMR spectroscopy was used to measure the amide proton and nitrogen chemical shift changes of a <sup>15</sup>N/<sup>13</sup>C-labeled fragment of MDM2 containing residues 17–125 after a stoichiometric excess of <b>PS10</b> was added. <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A shows an overlay of <sup>15</sup>N HSQC spectra of MDM2 before (blue resonances) and after (red resonances) the addition of <b>PS10</b>. Based on the behavior of most of the resonances in the free and bound spectra, we conclude that the binding between MDM2 and <b>PS10</b> is in slow exchange, which is consistent with the binding affinity we observed using fluorescence anisotropy. In <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, resonances that are not observed in the bound state are labeled with a red asterisk. We speculate that these residues are in intermediate-slow exchange and are participating in binding. An orange line marks the average chemical shift change for all the assigned MDM2 residues (0.056 ppm). <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D shows the structure of MDM2 (residues 25–109) bound to p53 TAD. The structure of MDM2 is shown as a gray surface representation and the cyan ribbon structure shows the p53 TAD peptide. MDM2 residues with chemical shift changes greater than 0.056 ppm are colored orange and the residues for resonances in intermediate-slow exchange are colored red. The pattern of chemical shift changes for <b>PS10</b> binding to MDM2 is highly similar to the results in a previous study from the Daughdrill lab of p53 TAD binding to MDM2.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> In particular, some of the MDM2 residues with the largest chemical shift changes upon binding to either <b>PS10</b> or p53 TAD are the same, including M50, I54, F55, L66, T67, and Y100, indicating that the binding interface between <b>PS10</b> and MDM2 overlaps with p53 TAD.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical shift mapping of <b>PS10</b> binding to MDM2. (A) Overlay of <sup>15</sup>N HSQC spectra of MDM2 before (blue resonances) and after (red resonances) the addition of <b>PS10</b>. HSQC spectra were collected with a twofold and fourfold stoichiometric excess of <b>PS10</b>. (B) Average chemical shift changes in part per million (ppm) for the amide proton and nitrogen resonances in MDM2 p53BD residues binding to <b>PS10</b>. (C,D) Surface image of the MDM2 p53BD structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Computational Simulations</h3><div class="NLM_p">The strong binding of <b>PS10</b> to MDM2 was further supported by computer modeling using PyMOL software. <b>PS10</b> was overlaid with the p53 helical structure on the MDM2 binding domain (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>) so that its critical side chains could align with the side chains of p53. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, the side chains of <b>PS10</b> residues 2a, 4a, and 6a insert deeply into the hydrophobic p53 binding pocket of MDM2, with the similar binding mode observed for p53 bound to MDM2. As these side chains of <b>PS10</b> closely mimic those critical residues (F19, W23, and L26) of p53 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), because of its intrinsic folding propensity, <b>PS10</b> could potently inhibit p53–MDM2 PPI.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Binding of <b>PS10</b> to MDM2. The helical structure was built on the crystal structure of <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C (CCDC: 1841094).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> (B) Overlay of side chains 2a, 4a, and 6a of <b>PS10</b> with Phe19, Trp23, and Leu26 of p53 (green) using PyMOL software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Stapled Sulfono-γ-AApeptides Induce Activation of p53 in Cells</h3><div class="NLM_p">To assess the cellular activity of sulfono-γ-AApeptides, we first attempted to investigate the activation of p53 using lead linear compounds <b>PS4</b> and <b>PS10</b>. U2OS cells (p53 wild-type osteosarcoma) with stably integrated BP100-luc (p53-responsive luciferase reporter) and CMV-lacZ (internal control for cell mass and toxicity)<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> were treated with 30 μM compounds for 18 h. Luciferase and lacZ activities were determined and the ratio of luc/lacZ indicates p53 transcriptional activity. Compared with p53 peptide, both <b>PS4</b> and <b>PS10</b> show marginally increased luciferase activity (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A), and none of them is active in the cellular functional assay (data not shown), suggesting that these peptides may not be cell-permeable. To circumvent this issue, the lead linear compound <b>PS4</b> was chosen to prepare stapled sulfono-γ-AApeptides <b>PS11</b> and <b>PS12</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), in the hope to enhance the binding activity to protein target and cell permeability.<a onclick="showRef(event, 'ref73 ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref73 ref74 ref75">(73−75)</a> Interestingly, although <b>PS12</b> was found to have weaker binding activity compared to its linear counterpart <b>PS4</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), which indicates that proper stapling length is needed for stapled sulfono-AApeptide to achieve optimized binding, it however displays enhanced luciferase activity (∼25%), suggesting that macrocyclic stapling indeed enhanced cell permeability. This is further evidenced by <b>PS11</b>, which not only displayed enhanced in vitro binding activity toward MDM2 (IC<sub>50</sub> = 1.9 μM) compared to <b>PS4</b>, but also activated the luciferase reporter by 70% at 30 μM (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Overall, the luciferase assay suggests that stapled γ-AApeptides exhibited enhanced cell activity.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Activation of p53 by stapled sulfono-γ-AApeptides. (A). Luciferase reporter assay. The p53-dependent luciferase transcriptional activation in U2OS cells: the luciferase activities were measured at least three times and the averaged activities along with standard derivations were plotted. (B). Western blotting. Drug treatment lasted for 16 h with 30 μM peptides or 1 μM nutlin. (C) Relative optical density value after 48 h of incubation in the presence of compounds at 200 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structures of Stapled Sulfono-γ-AApeptides Investigated for the Disruption of p53–MDM2 Interaction</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0008.gif" alt="" id="gr8" /></img><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <i>K</i><sub>d</sub> and IC<sub>50</sub> Values of the p53 and Lead Compounds to MDM2/MDMX</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MDM2</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MDMX</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>d</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>d</sub> (nM)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>p53</b></td><td class="colsep0 rowsep0" align="char" char=".">208</td><td class="colsep0 rowsep0" align="char" char=".">4.61</td><td class="colsep0 rowsep0" align="char" char=".">641</td><td class="colsep0 rowsep0" align="left">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PS1</b></td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">3.95</td><td class="colsep0 rowsep0" align="char" char=".">593</td><td class="colsep0 rowsep0" align="left">7.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PS3</b></td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="char" char=".">440</td><td class="colsep0 rowsep0" align="left">16.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PS4</b></td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">370</td><td class="colsep0 rowsep0" align="left">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PS9</b></td><td class="colsep0 rowsep0" align="char" char=".">89.6</td><td class="colsep0 rowsep0" align="char" char=".">3.22</td><td class="colsep0 rowsep0" align="char" char=".">279</td><td class="colsep0 rowsep0" align="left">4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PS10</b></td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">0.891</td><td class="colsep0 rowsep0" align="char" char=".">221</td><td class="colsep0 rowsep0" align="left">3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PS11</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">3.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>PS12</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">6.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nutlin</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">>10</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Regulation of Actin, p53, and p21 Expression by Stapled Sulfono-γ-AApeptides</h3><div class="NLM_p last">To further assess the ability of the stapled sulfono-γ-AApeptides to activate p53 in cells, we next incubated exponentially growing U2OS cells with <b>PS11</b> and <b>PS12</b> at 30 μM for 16 h, and the levels of p21, MDM2, and p53 expression were analyzed by western blotting (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). The MDM2 level was increased moderately after treatment, suggesting these stapled peptides disrupt p53/MDM2 PPI in cells. Moreover, the p21 level increased significantly after treatment with <b>PS11</b> compared to the control (PBS buffer), confirming the significant enhancement of p53 transcriptional activity. It is intriguing that the level of p21 and MDM2 is much more elevated than p53, similar to the effect caused by nutlin. Furthermore, it seems that both <b>PS10</b> and <b>PS11</b> could suppress proliferation of wild-type p53 cells more efficiently than the p53 peptides (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, MTT assay), and they also are more selective toward wild-type p53 cells than p53-deletion cells.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Binding of Sulfono-γ-AApeptides to MDMX</h3><div class="NLM_p last">To confirm that sulfono-γ-AApeptides do bind to MDMX, we next investigated the binding affinity of several lead sulfono-γ-AApeptides to MDMX. Fluorescence polarization assays were used to measure the affinities of lead compounds <b>PS1</b>, <b>PS3</b>, <b>PS4</b>, <b>PS9</b>, <b>PS10</b>, as well as stapled sulfono-γ-AApeptides <b>PS11</b> and <b>PS12</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf" class="ext-link">Figure S4</a>). The ability of the sequences to disrupt the p53–MDMX interaction was also tested, in comparison to those inhibiting p53–MDM2 PPI. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, the <i>K</i><sub>d</sub> and IC<sub>50</sub> values of the p53 peptide to MDMX are 641 nM and 5 μM, respectively, which is in good agreement with the literature.<a onclick="showRef(event, 'ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref56 ref57">(55−57)</a> Interestingly, the helical sulfono-γ-AApeptides designed to bind MDM2 also bind MDMX with good activity (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This is significant as small molecules such as nutlin (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) are generally only active toward MDM2 but not MDMX, which may imply that these peptidomimetics could be an alternative strategy to small molecules for dual targeting of MDM2 and MDMX, which is needed for fully activating p53 in tumors expressing MDMX.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Enzymatic Stability Study</h3><div class="NLM_p last">One of the major reasons for the development of peptidomimetics is their enhanced stability. Although our previous results demonstrated the stability of sulfono-γ-AApeptides,<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> the proteolytic stability of helical sulfono-γ-AApeptides in this study was also investigated and compared to the control peptide p53 (16–29). We incubated 0.1 mg/mL of five lead compounds (<b>PS4</b>, <b>PS9</b>, <b>PS10</b>, <b>PS11</b>, and <b>PS12</b>) and p53 with 0.1 mg/mL proteases in 100 mM ammonium bicarbonate buffer (pH 7.8) at 37 °C for 24 h. Both α-chymotrypsin and pronase were used as representative proteases. The stability of the examined compounds was analyzed by HPLC-MS (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf" class="ext-link">Figures S5–S10</a>). The control peptide p53 was completely degraded by α-chymotrypsin and pronase and produced multiple unidentified peaks with no intact peptide remaining. Strikingly, no detectable degradation occurred for the five peptidomimetic peptides, thus demonstrating the high stability of our sulfono-γ-AApeptides against enzymatic degradation, which augments their potential for disruption of PPIs in the future.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63666" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63666" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we demonstrated that helical sulfono-γ-AApeptides could be rationally designed to mimic p53 α-helix and inhibit p53–MDM2 and p53–MDMX PPI. The lead compound <b>PS10</b> bound to p53–MDM2 interaction with <i>K</i><sub>d</sub> and IC<sub>50</sub> values of 26 nM and 0.891 μM, respectively, and, so far, is one of the most potent unnatural peptidomimetic inhibitors reported to date targeting this interaction. Analysis of the HSQC NMR provides direct evidence that the helical sulfono-γ-AApeptide inhibitor interacts with the p53-binding pocket of MDM2. Furthermore, the stapled sulfono-γ-AApeptides showed promising cellular activity by inducing p53 and MDM2 levels and enhancing p53 transcriptional activity. Enzymatic stability studies have demonstrated high resistance of this class of peptidomimetic foldamer to proteolytic degradation. Thus, this work provides a template that can be applied to explore and generate novel peptidomimetic agents to target p53–MDM2/MDMX protein–protein interactions.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03101" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03101" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Information</h3><div class="NLM_p last">Fmoc-protected amino acids were purchased from Chem-impex (Wood Dale, IL). Rink Amide-MBHA resin (0.646 mmol/g) was purchased from GL Biochem (Shanghai) Ltd. 1-Hydroxybenzotriazole (HOBt) wetted with no less than 20% wt water, 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide, and <i>N</i>,<i>N</i>′-diisopropylcarbodiimide (DIC) was purchased from Oakwood Chemical (Estill, SC). Hoveyda–Grubbs second-generation catalyst was purchased from Sigma-Aldrich, Inc. FITC was purchased from Chemodex (Gallen, Switzerland). Thin-layer chromatography was performed on Sorbtech TLC plates (silica gel w/UV254), visualizing with UV-light 254 nm. Flash column chromatography was performed with ICN silica gel (60 Å, 230–400 mesh, 32–63 μm). Solid-phase synthesis of the peptides was conducted in the peptide synthesis vessels on a Burrell Wrist-Action shaker. All γ-AApeptides were analyzed and purified on a Waters Breeze 2 HPLC system installed with both analytic module (1 mL/min) and preparative module (16 mL/min), by employing a method using 5–100% linear gradient of solvent B [0.1% trifluoroacetic acid (TFA) in acetonitrile] in solvent A (0.1% TFA in water) for over 50 min, followed by 100% solvent B for over 15 min. The pure products were then collected and lyophilized on a Labconco lyophilizer; the purity of the compounds was determined to be >95% by analytical HPLC. Masses of γ-AApeptides were obtained on an Applied Biosystems 4700 Proteomics Analyzer. <sup>1</sup>H NMR spectra were recorded at 400 or 500 MHz using tetramethylsilane (TMS) as the internal standard. <sup>13</sup>C NMR spectra were recorded at 100 or 125 MHz using TMS as the internal standard. The multiplicities are reported as follows: singlet (s), doublet (d), doublet of doublets (dd), triplet (t), quartet (q), multiplet (m). Coupling constants are reported in hertz. High-resolution mass spectra were obtained on an Agilent 6220 using electrospray ionization (ESI) time-of-flight. Other chemicals and all solvents were purchased from Sigma-Aldrich (St. Louis, MO) or Fisher and used without further purification.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis of Sulfono-γ-AApeptide Building Blocks</h3><div class="NLM_p">The sulfono-γ-AApeptide building blocks were synthesized based on a previously report<a onclick="showRef(event, 'ref63 ref64 ref65 ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref63 ref64 ref65 ref66 ref67">(63−67)</a> and Fmoc-protected amino acids were used as the initial starting materials.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)propyl)-<i>N</i>-(methylsulfonyl)glycine (BB1)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.20 Hz, 2H), 7.68 (d, <i>J</i> = 6.00 Hz, 2H), 7.41 (t, <i>J</i> = 6.80 Hz, 2H), 7.34 (d, <i>J</i> = 6.80 Hz, 2H), 7.21 (d, <i>J</i> = 7.60 Hz, 1H), 4.27–4.34 (m, 2H), 4.21 (s, 1H), 3.99 (s, 2H), 3.76 (s, 1H), 3.13–3.20 (m, 2H), 2.94 (s, 3H), 1.04 (d, <i>J</i> = 5.20 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.0, 144.3, 141.2, 128.0, 127.5, 125.6, 120.5, 65.7, 52.4, 49.0, 47.2, 45.8, 18.7. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>21</sub>H<sub>25</sub>N<sub>2</sub>O<sub>6</sub>S, 433.1433; found, 433.1424.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-phenylpropyl)-<i>N</i>-(methylsulfonyl)glycine (BB2)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.87 (d, <i>J</i> = 7.20 Hz, 2H), 7.61 (d, <i>J</i> = 6.80 Hz, 2H), 7.41 (t, <i>J</i> = 6.40 Hz, 2H), 7.31 (d, <i>J</i> = 8.40 Hz, 3H), 7.23 (s, 4H), 7.16 (s, 1H), 4.19 (d, <i>J</i> = 6.00 Hz, 2H), 4.13 (d, <i>J</i> = 6.40 Hz, 1H), 3.94–4.08 (m, 2H), 3.90 (s, 1H), 3.38 (d, <i>J</i> = 10.40 Hz, 1H), 3.22 (t, <i>J</i> = 8.00 Hz, 1H), 2.95 (s, 3H), 2.87–2.92 (m, 1H), 2.60 (t, <i>J</i> = 11.60 Hz, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.2, 144.3, 144.2, 141.1, 139.1, 129.6, 128.5, 128.0, 127.5, 126.5, 125.6, 120.5, 65.8, 51.9, 51.6, 49.0, 47.1, 37.9. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>27</sub>H<sub>29</sub>N<sub>2</sub>O<sub>6</sub>S, 509.1746; found, 509.1733.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-4-methylpentyl)-<i>N</i>-(methylsulfonyl)glycine (BB3)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.20 Hz, 2H), 7.68 (d, <i>J</i> = 6.40 Hz, 2H), 7.41 (t, <i>J</i> = 7.20 Hz, 2H), 7.32 (s, 2H), 7.14 (d, <i>J</i> = 9.20 Hz, 1H), 4.34 (s, 2H), 4.21 (d, <i>J</i> = 5.60 Hz, 1H), 3.97 (s, 2H), 3.71 (s, 1H), 3.24 (d, <i>J</i> = 11.20 Hz, 1H), 3.10 (t, <i>J</i> = 8.80 Hz, 1H), 2.92 (s, 3H), 1.55 (s, 1H), 1.21–1.27 (m, 2H), 0.82–0.86 (m, 6H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.3, 144.4, 144.3, 141.2, 128.0, 127.5, 125.6, 120.5, 65.5, 51.9, 48.9, 48.2, 47.3, 41.3, 24.7, 23.7, 22.0. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>24</sub>H<sub>31</sub>N<sub>2</sub>O<sub>6</sub>S, 475.1903; found, 475.1893.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-4-methylpentyl)-<i>N</i>-(isobutylsulfonyl)glycine (BB4)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.50 Hz, 2H), 7.69 (q, <i>J</i> = 7.00, 4.50 Hz, 2H), 7.41 (t, <i>J</i> = 7.50 Hz, 2H), 7.30–7.34 (m, 2H), 7.17 (d, <i>J</i> = 9.00 Hz, 1H), 4.35 (q, <i>J</i> = 10.50, 7.00 Hz, 1H), 4.28 (q, <i>J</i> = 10.50, 7.00 Hz, 1H), 4.20 (t, <i>J</i> = 7.00 Hz, 1H), 3.97 (s, 2H), 3.68–3.74 (m, 1H), 3.26 (dd, <i>J</i> = 14.50, 5.50 Hz, 1H), 3.12 (q, <i>J</i> = 14.00, 8.50 Hz, 1H), 2.92–3.01 (m, 2H), 2.03–2.13 (m, 1H), 1.51–1.59 (m, 1H), 1.19–1.31 (m, 2H), 0.97 (q, <i>J</i> = 6.50, 5.50 Hz, 6H), 0.84 (dd, <i>J</i> = 10.00, 6.50 Hz, 6H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.3, 144.4, 144.3, 141.2, 128.0, 127.4, 125.6, 120.5, 65.6, 59.4, 51.9, 48.6, 48.1, 47.3, 41.3, 24.7, 23.7, 22.6, 22.6, 22.1. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>27</sub>H<sub>37</sub>N<sub>2</sub>O<sub>6</sub>S, 517.2372; found, 517.2362.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-4-methylpentyl)-<i>N</i>-((4-chlorophenyl)sulfonyl)glycine (BB5)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.87 (d, <i>J</i> = 7.00 Hz, 2H), 7.79 (d, <i>J</i> = 7.50 Hz, 2H), 7.67 (d, <i>J</i> = 6.50 Hz, 2H), 7.60 (d, <i>J</i> = 8.00 Hz, 2H), 7.40 (br s, 2H), 7.31 (q, <i>J</i> = 14.00, 7.00 Hz, 2H), 7.09 (d, <i>J</i> = 9.00 Hz, 1H), 4.31 (t, <i>J</i> = 8.50 Hz, 1H), 4.19–4.25 (m, 2H), 4.00–4.10 (m, 2H), 3.68 (br s, 1H), 3.26 (dd, <i>J</i> = 14.00, 4.50 Hz, 1H), 3.15 (q, <i>J</i> = 13.00, 7.50 Hz, 1H), 1.52 (br s, 1H), 1.19–1.26 (m, 2H), 0.83 (d, <i>J</i> = 6.00 Hz, 3H), 0.78 (d, <i>J</i> = 5.50 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.3, 156.2, 144.4, 144.2, 141.2, 139.0, 138.0, 129.6, 129.4, 128.0, 127.4, 125.6, 120.5, 65.5, 52.5, 48.9, 48.0, 47.3, 41.2, 24.6, 23.7, 21.9. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>29</sub>H<sub>32</sub>ClN<sub>2</sub>O<sub>6</sub>S, 571.1670; found, 571.1654.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-cyclohexylpropyl)-<i>N</i>-(isobutylsulfonyl)glycine (BB6)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.50 Hz, 2H), 7.69 (d, <i>J</i> = 7.00 Hz, 2H), 7.41 (t, <i>J</i> = 7.50 Hz, 2H), 7.32 (d, <i>J</i> = 4.50 Hz, 2H), 7.18 (d, <i>J</i> = 9.00 Hz, 1H), 4.35 (t, <i>J</i> = 9.50 Hz, 1H), 4.21–4.27 (m, 2H), 3.98 (s, 2H), 3.74 (d, <i>J</i> = 4.00 Hz, 1H), 3.24–3.27 (m, 1H), 3.13 (q, <i>J</i> = 13.00, 8.50 Hz, 1H), 2.92–3.01 (m, 2H), 2.08 (t, <i>J</i> = 6.50 Hz, 1H), 1.74 (d, <i>J</i> = 11.50 Hz, 1H), 1.59 (d, <i>J</i> = 14.50 Hz, 4H), 1.27 (s, 3H), 1.16 (t, <i>J</i> = 10.50 Hz, 1H), 1.09 (s, 2H), 0.97 (s, 6H), 0.90 (d, <i>J</i> = 9.50 Hz, 1H), 0.78 (d, <i>J</i> = 7.00 Hz, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.3, 144.3, 141.2, 128.0, 127.5, 125.6, 120.6, 65.7, 59.4, 52.0, 48.6, 47.4, 47.2, 34.1, 34.0, 32.3, 26.6, 26.4, 26.1, 24.7, 22.6. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>30</sub>H<sub>41</sub>N<sub>2</sub>O<sub>6</sub>S, 557.2685; found, 557.2670.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-cyclobutylpropyl)-<i>N</i>-(isobutylsulfonyl)glycine (BB7)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.50 Hz, 2H), 7.69 (t, <i>J</i> = 6.50 Hz, 2H), 7.41 (t, <i>J</i> = 7.50 Hz, 2H), 7.30 (q, <i>J</i> = 12.50, 7.50 Hz, 2H), 7.10 (d, <i>J</i> = 9.00 Hz, 1H), 4.36 (q, <i>J</i> = 10.50, 7.00 Hz, 1H), 4.28 (q, <i>J</i> = 10.50, 7.00 Hz, 1H), 4.20 (t, <i>J</i> = 7.00 Hz, 1H), 3.97 (q, <i>J</i> = 24.00, 18.5 Hz, 2H), 3.53–3.60 (m, 1H), 3.25 (dd, <i>J</i> = 14.50, 5.00 Hz, 1H), 3.12 (q, <i>J</i> = 14.00, 8.50 Hz, 1H), 2.92–3.01 (m, 2H), 2.20–2.29 (m, 1H), 2.04–2.12 (m, 1H), 1.90–1.95 (m, 2H), 1.69–1.82 (m, 2H), 1.49–1.63 (m, 3H), 1.37–1.43 (m, 1H), 0.97 (q, <i>J</i> = 6.50, 4.50 Hz, 6H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.2, 144.4, 144.3, 141.2, 128.0, 127.4, 125.6, 120.5, 65.6, 59.4, 51.6, 48.8, 48.6, 47.3, 33.1, 28.5, 28.3, 24.7, 22.6, 22.6, 18.6. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>28</sub>H<sub>37</sub>N<sub>2</sub>O<sub>6</sub>S, 529.2372; found, 529.2377.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-6-((<i>tert</i>-butoxycarbonyl)amino)hexyl)-<i>N</i>-(methylsulfonyl)glycine (BB8)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.50 Hz, 2H), 7.69 (d, <i>J</i> = 6.50 Hz, 2H), 7.41 (t, <i>J</i> = 7.50 Hz, 2H), 7.31–7.35 (m, 2H), 7.15 (d, <i>J</i> = 9.00 Hz, 1H), 6.74 (t, <i>J</i> = 5.00 Hz, 1H), 4.33 (d, <i>J</i> = 6.50 Hz, 2H), 4.22 (t, <i>J</i> = 7.00 Hz, 1H), 3.99 (s, 2H), 3.63 (t, <i>J</i> = 3.50 Hz, 1H), 3.27 (dd, <i>J</i> = 14.50, 5.50 Hz, 1H), 3.13 (q, <i>J</i> = 14.50, 9.00 Hz, 1H), 2.93 (s, 3H), 2.89 (t, <i>J</i> = 4.50 Hz, 2H), 1.40–1.49 (m, 2H), 1.37 (s, 9H), 1.19–1.33 (m, 4H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.4, 156.0, 144.3, 141.2, 128.0, 127.5, 125.6, 120.5, 77.8, 65.6, 51.6, 50.1, 48.8, 47.3, 31.9, 29.8, 28.7, 23.2. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>29</sub>H<sub>40</sub>N<sub>3</sub>O<sub>8</sub>S, 590.2536; found, 590.2520.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-5-(<i>tert</i>-butoxy)-5-oxopentyl)-<i>N</i>-(methylsulfonyl)glycine (BB9)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.50 Hz, 2H), 7.69 (q, <i>J</i> = 7.00, 4.00 Hz, 2H), 7.41 (t, <i>J</i> = 7.00 Hz, 2H), 7.33 (t, <i>J</i> = 7.50 Hz, 2H), 7.18 (d, <i>J</i> = 9.00 Hz, 1H), 4.31–4.37 (m, 2H), 4.22 (t, <i>J</i> = 7.00 Hz, 1H), 3.98 (s, 2H), 3.65–3.68 (m, 1H), 3.28 (dd, <i>J</i> = 14.50, 5.50 Hz, 1H), 3.15 (q, <i>J</i> = 14.50, 8.50 Hz, 1H), 2.94 (s, 3H), 2.13–2.24 (m, 2H), 1.71–1.77 (m, 1H), 1.45–1.53 (m, 1H), 1.39 (s, 9H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.3, 171.2, 156.4, 144.3, 144.3, 141.2, 128.0, 127.5, 125.6, 120.5, 80.0, 65.6, 51.2, 49.4, 48.6, 47.3, 31.8, 28.2, 27.5. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>27</sub>H<sub>35</sub>N<sub>2</sub>O<sub>8</sub>S, 547.2114; found, 547.2094.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(1-(<i>tert</i>-butoxycarbonyl)-1<i>H</i>-indol-3-yl)propyl)-<i>N</i>-(methylsulfonyl)glycine (BB10)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.02 (d, <i>J</i> = 7.00 Hz, 1H), 7.85 (d, <i>J</i> = 7.50 Hz, 2H), 7.68 (d, <i>J</i> = 7.00 Hz, 1H), 7.55 (d, <i>J</i> = 7.50 Hz, 3H), 7.37 (d, <i>J</i> = 7.50 Hz, 3H), 7.31 (t, <i>J</i> = 7.50 Hz, 1H), 7.21–7.26 (m, 3H), 4.21 (d, <i>J</i> = 6.00 Hz, 2H), 4.15 (d, <i>J</i> = 7.00 Hz, 1H), 4.05 (s, 2H), 3.48 (dd, <i>J</i> = 14.00, 4.00 Hz, 1H), 3.30 (q, <i>J</i> = 14.00, 8.50 Hz, 1H), 2.98 (s, 3H), 2.95 (s, 1H), 2.76 (t, <i>J</i> = 10.00 Hz, 1H), 1.57 (s, 1H), 1.53 (s, 9H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.3, 144.3, 144.2, 141.1, 131.0, 128.0, 127.4, 125.6, 125.5, 124.7,122.9, 120.5, 119.8, 117.8, 115.1, 83.8, 65.9, 51.9, 50.2, 49.2, 47.1, 28.1, 27.7, 27.5. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>34</sub>H<sub>38</sub>N<sub>3</sub>O<sub>8</sub>S, 648.2380; found, 648.2358.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-5-(<i>tert</i>-butoxy)-5-oxopentyl)-<i>N</i>-((4-chlorophenyl)sulfonyl)glycine (BB11)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.00 Hz, 2H), 7.79 (d, <i>J</i> = 8.00 Hz, 2H), 7.67 (br s, 2H), 7.59 (d, <i>J</i> = 7.50 Hz, 2H), 7.40 (t, <i>J</i> = 7.00 Hz, 2H), 7.31 (br s, 2H), 7.12 (d, <i>J</i> = 9.00 Hz, 1H), 4.30 (d, <i>J</i> = 5.50 Hz, 1H), 4.22 (t, <i>J</i> = 6.00 Hz, 2H), 4.04 (q, <i>J</i> = 27.50, 19.00 Hz, 2H), 3.64 (br s, 1H), 3.31 (dd, <i>J</i> = 14.00, 5.00 Hz, 1H), 3.18 (q, <i>J</i> = 13.00, 7.50 Hz, 1H), 2.09–2.23 (m, 2H), 1.73 (d, <i>J</i> = 7.50 Hz, 1H), 1.49 (t, <i>J</i> = 8.00 Hz, 1H), 1.38 (s, 9H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.3, 170.2, 156.2, 144.3, 144.2, 141.2, 138.9, 138.0, 129.6, 129.3, 128.0, 127.5, 125.6, 120.6, 80.0, 65.6, 51.7, 49.3, 48.6, 47.2, 31.8, 28.2, 27.5. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>32</sub>H<sub>36</sub>ClN<sub>2</sub>O<sub>8</sub>S, 643.1881; found, 643.1862.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-6-((<i>tert</i>-butoxycarbonyl)amino)hexyl)-<i>N</i>-(isobutylsulfonyl)glycine (BB12)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.50 Hz, 2H), 7.69 (dd, <i>J</i> = 7.50, 3.00 Hz, 2H), 7.41 (t, <i>J</i> = 7.50 Hz, 2H), 7.31–7.35 (m, 2H), 7.17 (d, <i>J</i> = 9.00 Hz, 1H), 6.74 (t, <i>J</i> = 5.00 Hz, 1H), 4.27–4.37 (m, 2H), 4.21 (t, <i>J</i> = 6.50 Hz, 1H), 3.98 (q, <i>J</i> = 22.50, 19.00 Hz, 2H), 3.60–3.63 (m, 1H), 3.29 (dd, <i>J</i> = 14.50, 5.50 Hz, 1H), 3.15 (q, <i>J</i> = 14.50, 9.00 Hz, 1H), 2.92–3.02 (m, 2H), 2.88 (d, <i>J</i> = 4.50 Hz, 2H), 2.75 (t, <i>J</i> = 6.00 Hz, 1H), 2.04–2.12 (m, 1H), 1.40–1.49 (m, 1H), 1.36 (s, 9H), 1.18–1.33 (m, 4H), 0.97 (q, <i>J</i> = 6.50, 4.50 Hz, 6H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.4, 156.0, 144.3, 144.3, 141.2, 128.0, 127.5, 125.6, 120.5, 77.7, 65.7, 59.4, 51.6, 50.0, 48.6, 47.3, 31.9, 29.8, 28.7, 24.7, 23.2, 22.6, 22.6. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>32</sub>H<sub>46</sub>N<sub>3</sub>O<sub>8</sub>S, 632.3006; found, 632.2988.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>R</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-3-(<i>tert</i>-butoxy)propyl)-<i>N</i>-(methylsulfonyl)glycine (BB13)</h4><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 7.50 Hz, 2H), 7.69 (d, <i>J</i> = 7.50 Hz, 2H), 7.41 (t, <i>J</i> = 7.50 Hz, 2H), 7.30–7.34 (m, 2H), 7.17 (d, <i>J</i> = 8.50 Hz, 1H), 4.28–4.34 (m, 2H), 4.20–4.23 (m, 1H), 3.97–4.07 (m, 2H), 3.74–3.77 (m, 1H), 3.41 (dd, <i>J</i> = 15.00, 5.00 Hz, 1H), 3.19–3.32 (m, 3H), 2.96 (s, 3H), 1.11 (s, 9H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 156.3, 144.3, 141.2, 128.0, 127.5, 125.6, 120.5, 73.0, 65.8, 62.0, 51.0, 49.1, 48.9, 47.2, 27.7. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>25</sub>H<sub>33</sub>N<sub>2</sub>O<sub>7</sub>S, 505.2008; found, 505.2000.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-6-((<i>tert</i>-butoxycarbonyl)amino)hexyl)-<i>N</i>-(but-3-en-1-ylsulfonyl)glycine (BB14)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.83 (br s, 1H), 7.84 (d, <i>J</i> = 7.20 Hz, 2H), 7.65 (d, <i>J</i> = 4.80 Hz, 2H), 7.37 (t, <i>J</i> = 7.20 Hz, 2H), 7.29 (t, <i>J</i> = 6.80 Hz, 2H), 7.13 (d, <i>J</i> = 8.80 Hz, 1H), 6.70 (s, 1H), 5.68–5.78 (m, 1H), 4.99 (q, <i>J</i> = 39.6, 17.60 Hz, 2H), 4.23–4.34 (m, 2H), 4.17 (t, <i>J</i> = 6.80 Hz, 1H), 3.96 (s, 2H), 3.59 (br s, 1H), 3.26–3.30 (m, 2H), 3.10–3.16 (m, 3H), 2.84 (d, <i>J</i> = 3.60 Hz, 2H), 3.36 (d, <i>J</i> = 7.20 Hz, 2H), 1.40 (br s, 1H), 1.33 (s, 9H), 1.11–1.27 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.4, 156.0, 144.3, 144.2, 141.2, 135.5, 128.0, 127.4, 125.6, 120.5, 116.8, 77.7, 65.6, 51.5, 50.0, 48.7, 47.2, 31.9, 29.8, 28.7, 27.6, 23.1. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>32</sub>H<sub>43</sub>N<sub>3</sub>O<sub>8</sub>S, 630.2849; found, 630.2857.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-5-(<i>tert</i>-butoxy)-5-oxopentyl)-<i>N</i>-(hex-5-en-1-ylsulfonyl)glycine (BB15)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.85 (d, <i>J</i> = 7.20 Hz, 2H), 7.65 (d, <i>J</i> = 7.20 Hz, 2H), 7.37 (t, <i>J</i> = 7.20 Hz, 2H), 7.29 (t, <i>J</i> = 7.20 Hz, 2H), 7.16 (d, <i>J</i> = 9.20 Hz, 1H), 5.64–5.74 (m, 1H), 4.90 (q, <i>J</i> = 21.20, 18.80 Hz, 2H), 4.32 (q, <i>J</i> = 10.00, 7.20 Hz, 1H), 4.16–4.24 (m, 2H), 3.13 (s, 2H), 3.76–3.82 (m, 1H), 3.61 (br s, 1H), 3.28 (dd, <i>J</i> = 14.40, 5.20 Hz, 1H), 3.13 (q, <i>J</i> = 14.40, 8.40 Hz, 1H), 3.04 (q, <i>J</i> = 8.00, 4.40 Hz, 2H), 2.46 (s, 1H), 2.07–2.20 (m, 2H), 1.94 (q, <i>J</i> = 13.20, 6.80 Hz, 2H), 1.67–1.69 (m, 1H), 1.55–1.62 (m, 2H), 1.40–1.50 (m, 1H), 1.35 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.3, 171.2, 156.4, 144.3, 141.2, 138.6, 128.0, 127.4, 125.6, 120.5, 115.5, 80.0, 65.7, 52.2, 49.2, 48.5, 47.2, 33.0, 31.7, 28.2, 27.6, 27.2, 22.8. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>32</sub>H<sub>42</sub>N<sub>2</sub>O<sub>8</sub>S, 615.2740; found, 615.2738.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>S</i>)-<i>N</i>-(2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-6-((<i>tert</i>-butoxycarbonyl)amino)hexyl)-<i>N</i>-(hex-5-en-1-ylsulfonyl)glycine (BB16)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.85 (d, <i>J</i> = 7.20 Hz, 2H), 7.65 (d, <i>J</i> = 7.60 Hz, 2H), 7.37 (t, <i>J</i> = 7.60 Hz, 2H), 7.29 (t, <i>J</i> = 7.20 Hz, 2H), 7.13 (d, <i>J</i> = 9.20 Hz, 1H), 6.70 (br s, 1H), 5.63–5.73 (m, 1H), 4.89 (t, <i>J</i> = 20.00 Hz, 2H), 4.30 (q, <i>J</i> = 10.00, 6.80 Hz, 1H), 4.15–4.23 (m, 2H), 3.93 (s, 2H), 3.57 (br s, 5H), 3.26 (dd, <i>J</i> = 14.40, 4.80 Hz, 1H), 3.10 (q, <i>J</i> = 14.40, 8.80 Hz, 1H), 3.02 (q, <i>J</i> = 8.00, 4.40 Hz, 2H), 2.83 (d, <i>J</i> = 4.40 Hz, 2H), 2.46 (s, 1H), 1.91–1.94 (m, 2H), 1.54–1.59 (m, 2H), 1.32 (s, 9H), 1.15–1.26 (m, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.2, 156.4, 156.0, 144.3, 141.2, 138.6, 128.0, 127.4, 125.6, 120.5, 115.5, 77.7, 65.7, 52.2, 49.9, 48.7, 47.2, 33.0, 32.0, 29.8, 28.7, 27.2, 23.1, 22.8. HRMS (ESI): ([M + H]<sup>+</sup>) calcd for C<sub>34</sub>H<sub>47</sub>N<sub>3</sub>O<sub>8</sub>S, 658.3162; found, 658.3180.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Sulfono-γ-AApeptide Preparation</h3><div class="NLM_p last">The sulfono-γ-AApeptide synthesis was carried out on 100 mg of Rink Amide-MBHA resin (0.646 mmol/g) under room temperature at atmosphere pressure. The resin was swelled in dimethylformamide (DMF) for 5 min before use, followed by treatment with 20% piperidine/DMF solution (2 mL) for 15 min (×2) to remove the Fmoc-protecting group, and afterward washed three times with dichloromethane (DCM) and three times with DMF. A premixed solution of sulfono-γ-AApeptide building block (2 equiv), HOBt (4 equiv), and DIC (4 equiv) in 2 mL of DMF was added to the resin and shaken for 4 h to complete the coupling reaction. After washing with DCM and DMF, the resin was treated with 20% piperidine/DMF solution for 15 min (×2). Another sulfono-γ-AApeptide building block (2 equiv) was attached on the resin following the procedure in the first coupling step, and the Fmoc-protecting group was removed after the coupling reaction was done. The reaction cycles were repeated until the desired sulfono-γ-AApeptides were synthesized. The N-terminus of the sequence was capped with acetic anhydride (1 mL) in pyridine (2 mL) (15 min × 2), followed by treatment with TFA/DCM (6 mL, 1:1, v/v) for 3 h. The cleavage solution was collected, and the beads washed with DCM (3 mL × 2). The solution was combined and evaporated under air flow to give the crude product, which was analyzed and purified by a Water HPLC system, at the 1 and 16 mL/min flow rates for analytic and preparative HPLC, respectively. The gradient eluting method of 5–100% of solvent B (0.1% TFA in acetonitrile) in A (0.1% TFA in water) over 50 min was performed. The pure peptides were then collected and lyophilized on a Labconco lyophilizer; the purity of the compounds was determined to be >95% by analytical HPLC. All the sulfono-γ-AApeptides were obtained with moderate yield (39.39–46.72%) after prep-HPLC purification.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Fluorescence Polarization Competition Assays</h3><div class="NLM_p last">The binding affinity (<i>K</i><sub>d</sub>) of the p53 and AApeptides was investigated by FP. GST-MDM2-1-150 containing human MDM2 were expressed in <i>Escherichia coli</i> as previously described by us. An FP experiment was carried out by incubating 50 nM FITC-labeled AApeptide with MDM2 (0.0625–2 μM) in 1× PBS. The binding affinity of the investigated AApeptides to the MDM2 protein (<i>K</i><sub>d</sub>) was obtained by incubating 50 nM FITC-labeled AApeptide in MDM2 ranging from 0.3125 to 55 μM. Dissociation constants (<i>K</i><sub>d</sub>) were determined by plotting the fluorescence anisotropy values as a function of protein concentration, and the plots were fitted to the following equation. The <i>L</i><sub>st</sub> is the concentration of the peptide and the <i>x</i> stands for the concentration of the protein. The experiments were performed in triplicate and repeated three times. The binding affinity of the investigated AApeptides to the MDMX protein (<i>K</i><sub>d</sub>) was obtained by using a similar procedure.<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_m001.gif" alt="" id="_i37" /></img></span></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Circular Dichroism</h3><div class="NLM_p last">CD spectra were measured on an Aviv 215 CD spectrometer using a 1 mm path length quartz cuvette, and compound solutions in PBS buffer (or in TFE) were prepared using dry weight of the lyophilized solid followed by dilution to give the desired concentration (100 μM) and solvent combination. Ten scans were averaged for each sample, independent experiments were conducted three times, and the spectra were averaged. The final spectra were normalized by subtracting the average blank spectra. Molar ellipticity [θ] (deg·cm<sup>2</sup>·dmol<sup>–1</sup>) was calculated using the equation<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_m002.gif" alt="" id="_i39" /></img></span>where θ<sub>obs</sub> is the measured ellipticity in millidegrees, <i>n</i> is the number of side groups, <i>l</i> is the path length in centimeters (0.1 cm), and <i>c</i> is the concentration of the sulfono-γ-AA peptide in molar units.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Luciferase Reporter Assay</h3><div class="NLM_p last">U2OS cells were stably transfected with p53-responsive luciferase reporter plasmid (BP100-luc) and cytomegalovirus (CMV)-lacZ plasmid. Cells were seeded 100,000 per well in 24-well plates 24 h before treatment. After treatment with compounds for 16 h, the cell lysate was analyzed for luciferase and β-gal expression. The ratio of luciferase/β-gal activity was used as an indicator of transcription activity.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Western Blotting</h3><div class="NLM_p last">Cells were lysed in RIPA buffer (with 1 mM phenylmethylsulfonyl fluoride [PMSF], and 1× protease inhibitor mixture) and centrifuged at 4 °C for 10 min at 14,000<i>g</i>. The supernatant was boiled in Laemmli sample buffer for 10 min and subjected to sodium dodecyl sulfate polyacrylamide gel electrophoresis and transferred to Immobilon-P filters (Millipore). The filters were blocked for 1 h with PBS containing 5% nonfat dry milk and 0.1% Tween 20 and incubated with primary and secondary antibodies, and the filters were developed using SuperSignal reagent (Thermo Scientific). MDM2 was detected using monoclonal antibody 3G9. Actin antibodies were purchased from Santa Cruz Biotechnology. DO-1 for human p53 and p21 antibody were from BD Pharmingen.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> MTT Assay</h3><div class="NLM_p last">HCT-116 p53+/+ or p53–/– cells were seeded 4 × 10<sup>4</sup> in 24-well plates. After 2-day treatment of each compounds (p53, <b>PS10</b>, <b>PS11</b>) at a final concentration of 200 μM, cell numbers were analyzed with MTT assay (CellTiter 96 Aqueous One Solution Cell Proliferation Assay, Promega), according to the product’s instructions. Three independent experiments were conducted for statistical analysis.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Enzymatic Stability Study</h3><div class="NLM_p last">Lead compounds and peptide control p53 (0.1 mg/mL) were incubated with 0.1 mg/mL proteases in 100 mM ammonium bicarbonate buffer (pH 7.8) at 37 °C for 24 h. Then, the reaction mixtures were concentrated in a speed vacuum at medium temperature to remove water and ammonium bicarbonate. The resulting residues were re-dissolved in H<sub>2</sub>O/MeCN and analyzed on a Waters analytical HPLC system with 0.8 mL/min flow rate and 5–100% linear gradient of solvent B (0.1% TFA in acetonitrile) in A (0.1% TFA in water) over the duration of 50 min. The UV detector was set to 215 nm.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <sup>15</sup>N–<sup>1</sup>H HSQC NMR of Lead Peptide <b>PS10</b> in Complex with MDM2</h3><div class="NLM_p">Uniformly <sup>15</sup>N- and <sup>13</sup>C-labeled samples of human MDM2 residues 17–125 were expressed and purified as described in previous literature.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> Resonance assignments for free MDM2 and MDM2 bound to <b>PS10</b> were made using sensitivity-enhanced <sup>1</sup>H–<sup>15</sup>N HSQC and three-dimensional HNCA spectra using uniformly <sup>15</sup>N- and <sup>13</sup>C-labeled samples in 50 mM sodium phosphate buffer, 100 NaCl, 1 mM ethylenediaminetetraacetic acid, 0.02% sodium azide, and 2 mM dithiothreitol, at pH 6.8 (10% D<sub>2</sub>O). NMR spectroscopy was carried out on a Varian VNMRS 800 MHz spectrometer with a triple resonance pulse field <i>Z</i>-axis gradient cold probe at 30 °C. For HNCA experiments, data were acquired along <sup>1</sup>H, <sup>13</sup>C and <sup>15</sup>N dimensions using 9689.9228 (t3) × 6433.1377 (t2) × 2430.4290 (t1) Hz sweep widths and 1024 (t3) × 64 (t2) × 32 (t1) complex data points. The sweep widths and complex points of the HSQC were 9689.9228 (t2) × 2430.3853 (t1) Hz and 1024 (t2) × 128 (t1), respectively. Bound spectra were collected using a stoichiometric excess of <b>PS10</b>. Saturation of binding was confirmed by comparing spectra from MDM2 with a twofold and fourfold stoichiometric excess of <b>PS10</b>. The combined average chemical shifts were calculated from the formula Δave = [((Δ1HN)2 + (Δ15N/5)2)/2]1/2. Assignments of free MDM2 were previously reported. Assignment of 81 resonances of MDM2 bound to <b>PS10</b> were confirmed using data from the HNCA experiment. Assignment of 10 additional resonances in the HSQC spectrum of MDM2 bound to <b>PS10</b> were assigned based on the behavior of the peaks in the free and bound spectra and assuming a minimal perturbation in the spectra. Residues 19, 61, 65, 66, 67, 72, 73, 77, 94, and 105 were assigned using these criteria. There were also several resonances that were not detected in the bound state. This includes the resonances for residues 23, 46, 58, 59, 62, 71, 74, 93, 97, and 108. Resonances that disappear in the presence of a ligand are in intermediate exchange and probably involved in binding.</div><div class="NLM_p last">All NMR spectra were processed with NMRFx and analyzed using the NMRViewJ software<sup>.</sup> Apodization was achieved in the <sup>1</sup>H, <sup>13</sup>C, and <sup>15</sup>N dimensions using a squared sine bell function shifted by 70 °C. Apodization was followed by zero filling to double the number of real data points and linear prediction was used in the <sup>15</sup>N dimension for the HNCA spectra.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00993" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00768" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00768" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b00993?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00993</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental procedures of synthesis; characterization and purities of sequences; fluorescence polarization competition assays; and enzymatic stability study (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf">jm9b00993_si_001.pdf (4.16 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_002.csv">jm9b00993_si_002.csv (9.8 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00993" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16414" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16414" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qi Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology, Shuguang Hospital,
Shanghai University of Traditional Chinese Medicine, Shanghai 201203, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#167a6c6170567e79627b777f7a3875797b"><span class="__cf_email__" data-cfemail="731f090415331b1c071e121a1f5d101c1e">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gary Daughdrill</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Microbiology and Molecular Biology, University of South Florida, Tampa, Florida 33620, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#583f3c392d3f303c2a313434182d2b3e763d3c2d"><span class="__cf_email__" data-cfemail="e08784819587888492898c8ca0959386ce858495">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiandong Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2c46454d424843424b024f4449426c41434a4a45585802435e4b"><span class="__cf_email__" data-cfemail="58323139363c37363f763b303d361835373e3e312c2c76372a3f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianfeng Cai</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3106-3306" title="Orcid link">http://orcid.org/0000-0003-3106-3306</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#99f3f0f8f7fffcf7fefaf8f0d9eceaffb7fcfdec"><span class="__cf_email__" data-cfemail="d3b9bab2bdb5b6bdb4b0b2ba93a6a0b5fdb6b7a6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peng Sang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Shi</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junhao Lu</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lihong Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leixiang Yang</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, H. Lee Moffitt Cancer
Center and Research Institute, 12902 Magnolia Drive, Tampa, Florida 33612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wade Borcherds</span> - <span class="hlFld-Affiliation affiliation">Department
of Cell Biology, Microbiology and Molecular Biology, University of South Florida, Tampa, Florida 33620, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sami Abdulkadir</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of South Florida, 4202 E. Fowler Avenue, Tampa, Florida 33620, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>P.S., Y.S., and J.L. contributed equally. Jianfeng Cai led and supervised the project. P.S., Y.S., J.L. and W.B. performed the experiments. P.S., Y.S., Q.L., G.D., Jiandong Chen, and Jianfeng Cai analyzed the data. P.S., G.D., Jiandong Chen, and Jianfeng Cai wrote the paper. All the authors discussed the results, commented on, and proofread the paper.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58302" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58302" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was generously supported by NSF CAREER 1351265 (Jianfeng Cai), NIH 1R01GM112652-01A1 (Jianfeng Cai), NIH2R01CA14124406-A1 (G.D. and Jiandong Chen) and NIH1R01GM115556-01A1(G.D.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">MDM2</td><td class="NLM_def"><p class="first last">murine double minute-2</p></td></tr><tr><td class="NLM_term">MDMX</td><td class="NLM_def"><p class="first last">murine double minute-X</p></td></tr><tr><td class="NLM_term">TFE</td><td class="NLM_def"><p class="first last">trifluoroethanol</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DIC</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>′-diisopropylcarbodiimide</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>d</sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 77 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabhakaran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baruah, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjayan, G. J.</span></span> <span> </span><span class="NLM_article-title">Diversifying the structural architecture of synthetic oligomers: the hetero foldamer approach</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">11593</span>– <span class="NLM_lpage">11611</span>, <span class="refDoi"> DOI: 10.1039/c1cc13313f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc1cc13313f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=21850306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlejtrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=11593-11611&author=A.+Royauthor=P.+Prabhakaranauthor=P.+K.+Baruahauthor=G.+J.+Sanjayan&title=Diversifying+the+structural+architecture+of+synthetic+oligomers%3A+the+hetero+foldamer+approach&doi=10.1039%2Fc1cc13313f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Diversifying the structural architecture of synthetic oligomers: the hetero foldamer approach</span></div><div class="casAuthors">Roy, Arup; Prabhakaran, Panchami; Baruah, Pranjal Kumar; Sanjayan, Gangadhar J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11593-11611</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Conformationally ordered synthetic oligomers, also called "foldamers", have attracted considerable attention in recent years owing to their ability to mimic the structural architecture of biopolymers and also because of their potential applications in biomedical and material science fields.  Until recently, the major focus in this area has been the development of oligomers featuring a single type of monomer building blocks.  However, due to the enormous possibility of augmenting the conformational space available for oligomer design, the hetero foldamer approach has been introduced very recently.  This feature article aims to describe foldamers with unique structural architectures, exclusively featuring heterogeneous backbones (hetero foldamers).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLPCTen5iporVg90H21EOLACvtfcHk0lgsb7QnQBIFmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlejtrjP&md5=6f5fffd8d92c29653afbc5389702eb3b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1039%2Fc1cc13313f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc13313f%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DPrabhakaran%26aufirst%3DP.%26aulast%3DBaruah%26aufirst%3DP.%2BK.%26aulast%3DSanjayan%26aufirst%3DG.%2BJ.%26atitle%3DDiversifying%2520the%2520structural%2520architecture%2520of%2520synthetic%2520oligomers%253A%2520the%2520hetero%2520foldamer%2520approach%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26spage%3D11593%26epage%3D11611%26doi%3D10.1039%2Fc1cc13313f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span> <span> </span><span class="NLM_article-title">Synthetic foldamers</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5933</span>– <span class="NLM_lpage">5941</span>, <span class="refDoi"> DOI: 10.1039/c1cc11137j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc1cc11137j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=21483969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVyju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=5933-5941&author=G.+Guichardauthor=I.+Huc&title=Synthetic+foldamers&doi=10.1039%2Fc1cc11137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic foldamers</span></div><div class="casAuthors">Guichard, Gilles; Huc, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5933-5941</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Foldamers are artificial folded mol. architectures inspired by the structures and functions of biopolymers.  This highlight focuses on important developments concerning foldamers produced by chem. synthesis and on the perspectives that these new self-organized mol. scaffolds offer.  Progress in the field has led to synthetic objects that resemble small proteins in terms of size and complexity yet that may not contain any α-amino acids.  Foldamers have introduced new tools and concepts to develop biol. active substances, synthetic receptors and novel materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow03chLS_njLVg90H21EOLACvtfcHk0lipmCMh69JSwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVyju7c%253D&md5=b304218c67b482fa823a33ddde249ada</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2Fc1cc11137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc11137j%26sid%3Dliteratum%253Aachs%26aulast%3DGuichard%26aufirst%3DG.%26aulast%3DHuc%26aufirst%3DI.%26atitle%3DSynthetic%2520foldamers%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26spage%3D5933%26epage%3D5941%26doi%3D10.1039%2Fc1cc11137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horne, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Foldamers with heterogeneous backbones</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1408</span>, <span class="refDoi"> DOI: 10.1021/ar800009n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800009n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFajtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1399-1408&author=W.+S.+Horneauthor=S.+H.+Gellman&title=Foldamers+with+heterogeneous+backbones&doi=10.1021%2Far800009n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Foldamers with Heterogeneous Backbones</span></div><div class="casAuthors">Horne, W. Seth; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1399-1408</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The functions performed by proteins and nucleic acids provide the foundation for life.  Chemists have recently begun to ask whether it is possible to design synthetic oligomers that approach the structural and functional complexities of these biopolymers.  The study of foldamers, non-natural oligomers displaying discrete folding propensities, has demonstrated that there are several synthetic backbones that exhibit biopolymer-like conformational behavior.  Early work in this area focused on oligomers comprised of a single type of monomer subunit, but recent efforts have highlighted the potential of mixed or "heterogeneous" backbones to expand the structural and functional repertoire of foldamers.  In this Account, we illustrate the promise of heterogeneous backbone foldamers by focusing on examples contg. both α- and β-amino acid residues.  Some β-residues bear protein-like side chains, while others have cyclic structures that confer conformational rigidity.  The study of heterogeneous backbone foldamers has several advantages over that of their homogeneous backbone counterparts, including access to many new mol. shapes based on variations in the stoichiometries and patterns of the subunit combinations and improved prospects for side chain diversification.  Recent efforts to develop α/β-peptide foldamers can be divided into two conceptually distinct classes.  The first includes entities prepd. using a "block" strategy, in which α-peptide segments and β-peptide segments are combined to form a hybrid oligomer.  The second class encompasses designs in which α- and β-amino acid monomers are interspersed in a regular pattern throughout an oligomer sequence.  One α/β-peptide helical secondary structure, contg. C=O(i)···H-N(i+4) H-bonds analogous to those in the α-helix, has been shown via crystallog. to form helix bundle quaternary structures.  Desirable biol. functions have been elicited from α/β-peptide foldamers.  Efforts to mimic naturally occurring host-defense α-peptides have yielded new antimicrobial agents and have led to a reexamn. of the long-held views regarding structure-activity relationships among these α-peptides and their analogs.  Foldamers offer new platforms for mimicry of the mol. surfaces involved in specific protein-protein recognition events; recent achievements in the prepn. of α/β-peptide inhibitors of the protein-protein interactions involved in apoptotic signaling (e.g., between Bcl-xL and pro-apoptotic partners) have revealed the benefits of employing heterogeneous backbones relative to homogeneous backbones for foldamer-based designs.  These initial successes in the development of α/β-peptides exhibiting specific biol. activities highlight the potential of heterogeneous backbone foldamers for use in biomedical applications and provide guidelines for future studies into new target functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT6vMz0rs3sLVg90H21EOLACvtfcHk0lipmCMh69JSwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFajtrY%253D&md5=69f53ccc98f0265b8d7d73362de37b86</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Far800009n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800009n%26sid%3Dliteratum%253Aachs%26aulast%3DHorne%26aufirst%3DW.%2BS.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DFoldamers%2520with%2520heterogeneous%2520backbones%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1399%26epage%3D1408%26doi%3D10.1021%2Far800009n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shandler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">Foldamers as versatile frameworks for the design and evolution of function</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nchembio876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchembio876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17438550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=252-262&author=C.+M.+Goodmanauthor=S.+Choiauthor=S.+Shandlerauthor=W.+F.+DeGrado&title=Foldamers+as+versatile+frameworks+for+the+design+and+evolution+of+function&doi=10.1038%2Fnchembio876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Foldamers as versatile frameworks for the design and evolution of function</span></div><div class="casAuthors">Goodman, Catherine M.; Choi, Sungwook; Shandler, Scott; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">252-262</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Foldamers are sequence-specific oligomers akin to peptides, proteins and oligonucleotides that fold into well-defined three-dimensional structures.  They offer the chem. biologist a broad pallet of building blocks for the construction of mols. that test and extend our understanding of protein folding and function.  Foldamers also provide templates for presenting complex arrays of functional groups in virtually unlimited geometrical patterns, thereby presenting attractive opportunities for the design of mols. that bind in a sequence- and structure-specific manner to oligosaccharides, nucleic acids, membranes and proteins.  We summarize recent advances and highlight the future applications and challenges of this rapidly expanding field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLkvabxBqULVg90H21EOLACvtfcHk0ljS3NUoqA5e2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D&md5=99f0e4b2bb7a8457a586c74f0f98dbb3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio876%26sid%3Dliteratum%253Aachs%26aulast%3DGoodman%26aufirst%3DC.%2BM.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DShandler%26aufirst%3DS.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DFoldamers%2520as%2520versatile%2520frameworks%2520for%2520the%2520design%2520and%2520evolution%2520of%2520function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D252%26epage%3D262%26doi%3D10.1038%2Fnchembio876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S.</span>; <span class="NLM_string-name">Huc, I.</span></span> <i>Foldamers: Structure, Properties, and Applications</i>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2F9783527611478" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=S.+Hecht&author=I.+Huc&title=Foldamers%3A+Structure%2C+Properties%2C+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2F9783527611478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527611478%26sid%3Dliteratum%253Aachs%26aulast%3DHecht%26aufirst%3DS.%26btitle%3DFoldamers%253A%2520Structure%252C%2520Properties%252C%2520and%2520Applications%26pub%3DWiley-VCH%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Poli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawodny, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinonero, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayden, J.</span></span> <span> </span><span class="NLM_article-title">Conformational photoswitching of a synthetic peptide foldamer bound within a phospholipid bilayer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1126/science.aad8352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1126%2Fscience.aad8352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=27033546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1KisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=575-580&author=M.+De%0APoliauthor=W.+Zawodnyauthor=O.+Quinoneroauthor=M.+Lorchauthor=S.+J.+Webbauthor=J.+Clayden&title=Conformational+photoswitching+of+a+synthetic+peptide+foldamer+bound+within+a+phospholipid+bilayer&doi=10.1126%2Fscience.aad8352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational photoswitching of a synthetic peptide foldamer bound within a phospholipid bilayer</span></div><div class="casAuthors">De Poli, Matteo; Zawodny, Wojciech; Quinonero, Ophelie; Lorch, Mark; Webb, Simon J.; Clayden, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">6285</span>),
    <span class="NLM_cas:pages">575-580</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The dynamic properties of foldamers, synthetic mols. that mimic folded biomols., have mainly been explored in free soln.  We report on the design, synthesis, and conformational behavior of photoresponsive foldamers bound in a phospholipid bilayer akin to a biol. membrane phase.  These mols. contain a chromophore, which can be switched between two configurations by different wavelengths of light, attached to a helical synthetic peptide that both promotes membrane insertion and communicates conformational change along its length.  Light-induced structural changes in the chromophore are translated into global conformational changes, which are detected by monitoring the solid-state 19F NMR signals of a remote fluorine-contg. residue located 1 to 2 nm away.  The behavior of the foldamers in the membrane phase is similar to that of analogous compds. in org. solvents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHTa9onjAgULVg90H21EOLACvtfcHk0ljS3NUoqA5e2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1KisL0%253D&md5=9c339a72d54cfc20c2825d61bd6cbafb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad8352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad8352%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BPoli%26aufirst%3DM.%26aulast%3DZawodny%26aufirst%3DW.%26aulast%3DQuinonero%26aufirst%3DO.%26aulast%3DLorch%26aufirst%3DM.%26aulast%3DWebb%26aufirst%3DS.%2BJ.%26aulast%3DClayden%26aufirst%3DJ.%26atitle%3DConformational%2520photoswitching%2520of%2520a%2520synthetic%2520peptide%2520foldamer%2520bound%2520within%2520a%2520phospholipid%2520bilayer%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D575%26epage%3D580%26doi%3D10.1126%2Fscience.aad8352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diemer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raftery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruscoe, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengel, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockroft, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, S. J.</span></span> <span> </span><span class="NLM_article-title">Length-Dependent Formation of Transmembrane Pores by 310-Helical α-Aminoisobutyric Acid Foldamers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">688</span>– <span class="NLM_lpage">695</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b12057</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b12057" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=688-695&author=J.+E.+Jonesauthor=V.+Diemerauthor=C.+Adamauthor=J.+Rafteryauthor=R.+E.+Ruscoeauthor=J.+T.+Sengelauthor=M.+I.+Wallaceauthor=A.+Baderauthor=S.+L.+Cockroftauthor=J.+Claydenauthor=S.+J.+Webb&title=Length-Dependent+Formation+of+Transmembrane+Pores+by+310-Helical+%CE%B1-Aminoisobutyric+Acid+Foldamers&doi=10.1021%2Fjacs.5b12057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Length-Dependent Formation of Transmembrane Pores by 310-Helical α-Aminoisobutyric Acid Foldamers</span></div><div class="casAuthors">Jones, Jennifer E.; Diemer, Vincent; Adam, Catherine; Raftery, James; Ruscoe, Rebecca E.; Sengel, Jason T.; Wallace, Mark I.; Bader, Antoine; Cockroft, Scott L.; Clayden, Jonathan; Webb, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">688-695</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthetic biol. toolbox lacks extendable and conformationally controllable yet easy-to-synthesize building blocks that are long enough to span membranes.  To meet this need, an iterative synthesis of α-aminoisobutyric acid (Aib) oligomers was used to create a library of homologous rigid-rod 310-helical foldamers, which have incrementally increasing lengths and functionalizable N- and C-termini.  This library was used to probe the inter-relationship of foldamer length, self-assocn. strength, and ionophoric ability, which is poorly understood.  Although foldamer self-assocn. in nonpolar chloroform increased with length, with a ∼14-fold increase in dimerization const. from Aib6 to Aib11, ionophoric activity in bilayers showed a stronger length dependence, with the obsd. rate const. for Aib11 ∼70-fold greater than that of Aib6.  The strongest ionophoric activity was obsd. for foldamers with >10 Aib residues, which have end-to-end distances greater than the hydrophobic width of the bilayers used (∼2.8 nm); X-ray crystallog. showed that Aib11 is 2.93 nm long.  These studies suggest that being long enough to span the membrane is more important for good ionophoric activity than strong self-assocn. in the bilayer.  Planar bilayer conductance measurements showed that Aib11 and Aib13, but not Aib7, could form pores.  This pore-forming behavior is strong evidence that Aibm (m ≥ 10) building blocks can span bilayers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjKQwOG7qjGbVg90H21EOLACvtfcHk0lhJRk4F3lIyKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygsb7N&md5=ce8a1028c371a2699f3207fc0f5c06cf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b12057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b12057%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DJ.%2BE.%26aulast%3DDiemer%26aufirst%3DV.%26aulast%3DAdam%26aufirst%3DC.%26aulast%3DRaftery%26aufirst%3DJ.%26aulast%3DRuscoe%26aufirst%3DR.%2BE.%26aulast%3DSengel%26aufirst%3DJ.%2BT.%26aulast%3DWallace%26aufirst%3DM.%2BI.%26aulast%3DBader%26aufirst%3DA.%26aulast%3DCockroft%26aufirst%3DS.%2BL.%26aulast%3DClayden%26aufirst%3DJ.%26aulast%3DWebb%26aufirst%3DS.%2BJ.%26atitle%3DLength-Dependent%2520Formation%2520of%2520Transmembrane%2520Pores%2520by%2520310-Helical%2520%25CE%25B1-Aminoisobutyric%2520Acid%2520Foldamers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D688%26epage%3D695%26doi%3D10.1021%2Fjacs.5b12057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collie, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulka-Ziach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decossas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackereth, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span> <span> </span><span class="NLM_article-title">Shaping quaternary assemblies of water-soluble non-peptide helical foldamers by sequence manipulation</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">878</span>, <span class="refDoi"> DOI: 10.1038/nchem.2353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchem.2353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26492006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFOku7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=871-878&author=G.+W.+Collieauthor=K.+Pulka-Ziachauthor=C.+M.+Lombardoauthor=J.+Fremauxauthor=F.+Rosuauthor=M.+Decossasauthor=L.+Mauranauthor=O.+Lambertauthor=V.+Gabelicaauthor=C.+D.+Mackerethauthor=G.+Guichard&title=Shaping+quaternary+assemblies+of+water-soluble+non-peptide+helical+foldamers+by+sequence+manipulation&doi=10.1038%2Fnchem.2353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Shaping quaternary assemblies of water-soluble non-peptide helical foldamers by sequence manipulation</span></div><div class="casAuthors">Collie, Gavin W.; Pulka-Ziach, Karolina; Lombardo, Caterina M.; Fremaux, Juliette; Rosu, Frederic; Decossas, Marion; Mauran, Laura; Lambert, Olivier; Gabelica, Valerie; Mackereth, Cameron D.; Guichard, Gilles</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">871-878</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design and construction of biomimetic self-assembling systems is a challenging yet potentially highly rewarding endeavour that contributes to the development of new biomaterials, catalysts, drug-delivery systems and tools for the manipulation of biol. processes.  Significant progress has been achieved by engineering self-assembling DNA-, protein- and peptide-based building units.  However, the design of entirely new, completely non-natural folded architectures that resemble biopolymers ('foldamers') and have the ability to self-assemble into atomically precise nanostructures in aq. conditions has proved exceptionally challenging.  Here we report the modular design, formation and structural elucidation at the at. level of a series of diverse quaternary arrangements formed by the self-assembly of short amphiphilic α-helicomimetic foldamers that bear proteinaceous side chains.  We show that the final quaternary assembly can be controlled at the sequence level, which permits the programmed formation of either discrete helical bundles that contain isolated cavities or pH-responsive water-filled channels with controllable pore diams.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO9U5QH5AT8LVg90H21EOLACvtfcHk0lhJRk4F3lIyKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFOku7fI&md5=4ba4da0dc49636de6a566fd9e2f63920</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2353%26sid%3Dliteratum%253Aachs%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DPulka-Ziach%26aufirst%3DK.%26aulast%3DLombardo%26aufirst%3DC.%2BM.%26aulast%3DFremaux%26aufirst%3DJ.%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DDecossas%26aufirst%3DM.%26aulast%3DMauran%26aufirst%3DL.%26aulast%3DLambert%26aufirst%3DO.%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DMackereth%26aufirst%3DC.%2BD.%26aulast%3DGuichard%26aufirst%3DG.%26atitle%3DShaping%2520quaternary%2520assemblies%2520of%2520water-soluble%2520non-peptide%2520helical%2520foldamers%2520by%2520sequence%2520manipulation%26jtitle%3DNat.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D871%26epage%3D878%26doi%3D10.1038%2Fnchem.2353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span> <span> </span><span class="NLM_article-title">Magnetotactic molecular architectures from self-assembly of β-peptide foldamers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8747</span>– <span class="NLM_lpage">8753</span>, <span class="refDoi"> DOI: 10.1038/ncomms9747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fncomms9747" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26510658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCmsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8747-8753&author=S.+Kwonauthor=B.+J.+Kimauthor=H.-K.+Limauthor=K.+Kangauthor=S.+H.+Yooauthor=J.+Gongauthor=E.+Yoonauthor=J.+Leeauthor=I.+S.+Choiauthor=H.+Kimauthor=H.-S.+Lee&title=Magnetotactic+molecular+architectures+from+self-assembly+of+%CE%B2-peptide+foldamers&doi=10.1038%2Fncomms9747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Magnetotactic molecular architectures from self-assembly of β-peptide foldamers</span></div><div class="casAuthors">Kwon, Sunbum; Kim, Beom Jin; Lim, Hyung-Kyu; Kang, Kyungtae; Yoo, Sung Hyun; Gong, Jintaek; Yoon, Eunyoung; Lee, Juno; Choi, Insung S.; Kim, Hyungjun; Lee, Hee-Seung</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8747</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of stimuli-responsive self-assembled mol. systems capable of undergoing mech. work is one of the most important challenges in synthetic chem. and materials science.  Here we report that foldectures, i.e., self-assembled mol. architectures of β-peptide foldamers, uniformly align with respect to an applied static magnetic field, and also show instantaneous orientational motion in a dynamic magnetic field.  This response is explained by the amplified anisotropy of the diamagnetic susceptibilities as a result of the well-ordered mol. packing of the foldectures.  In addn., the motions of foldectures at the microscale can be translated into magnetotactic behavior at the macroscopic scale in a way reminiscent to that of magnetosomes in magnetotactic bacteria.  This study will provide significant inspiration for designing the next generation of biocompatible peptide-based mol. machines with applications in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreIyfWwJdjJLVg90H21EOLACvtfcHk0lgQxGZd_sc1IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCmsrrK&md5=0829ea55bbd5fd090897891693fce789</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fncomms9747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9747%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DB.%2BJ.%26aulast%3DLim%26aufirst%3DH.-K.%26aulast%3DKang%26aufirst%3DK.%26aulast%3DYoo%26aufirst%3DS.%2BH.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DYoon%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DI.%2BS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DH.-S.%26atitle%3DMagnetotactic%2520molecular%2520architectures%2520from%2520self-assembly%2520of%2520%25CE%25B2-peptide%2520foldamers%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8747%26epage%3D8753%26doi%3D10.1038%2Fncomms9747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheloha, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardella, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Backbone modification of a polypeptide drug alters duration of action in vivo</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1038/nbt.2920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnbt.2920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=24929976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=653-655&author=R.+W.+Chelohaauthor=A.+Maedaauthor=T.+Deanauthor=T.+J.+Gardellaauthor=S.+H.+Gellman&title=Backbone+modification+of+a+polypeptide+drug+alters+duration+of+action+in+vivo&doi=10.1038%2Fnbt.2920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone modification of a polypeptide drug alters duration of action in vivo</span></div><div class="casAuthors">Cheloha, Ross W.; Maeda, Akira; Dean, Thomas; Gardella, Thomas J.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">653-655</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Systematic modification of the backbone of bioactive polypeptides through β-amino acid residue incorporation could provide a strategy for generating mols. with improved drug properties, but such alterations can result in lower receptor affinity and potency.  Using an agonist of parathyroid hormone receptor-1 (PTHR1), a G protein-coupled receptor in the B-family, we present an approach for α→β residue replacement that enables both high activity and improved pharmacokinetic properties in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8cWxIA-3shbVg90H21EOLACvtfcHk0lgQxGZd_sc1IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D&md5=100382f0d7e67fe9c11fd39886e7acac</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2920%26sid%3Dliteratum%253Aachs%26aulast%3DCheloha%26aufirst%3DR.%2BW.%26aulast%3DMaeda%26aufirst%3DA.%26aulast%3DDean%26aufirst%3DT.%26aulast%3DGardella%26aufirst%3DT.%2BJ.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DBackbone%2520modification%2520of%2520a%2520polypeptide%2520drug%2520alters%2520duration%2520of%2520action%2520in%2520vivo%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D653%26epage%3D655%26doi%3D10.1038%2Fnbt.2920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buratto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupfel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois d’Estaintot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laguerre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallois, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span> <span> </span><span class="NLM_article-title">Structure of a Complex Formed by a Protein and a Helical Aromatic Oligoamide Foldamer at 2.1 Å Resolution</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">887</span>, <span class="refDoi"> DOI: 10.1002/anie.201309160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201309160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVOhsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=883-887&author=J.+Burattoauthor=C.+Colomboauthor=M.+Stupfelauthor=S.+J.+Dawsonauthor=C.+Dolainauthor=B.+Langlois+d%E2%80%99Estaintotauthor=L.+Fischerauthor=T.+Granierauthor=M.+Laguerreauthor=B.+Galloisauthor=I.+Huc&title=Structure+of+a+Complex+Formed+by+a+Protein+and+a+Helical+Aromatic+Oligoamide+Foldamer+at+2.1+%C3%85+Resolution&doi=10.1002%2Fanie.201309160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a Complex Formed by a Protein and a Helical Aromatic Oligoamide Foldamer at 2.1 Å Resolution</span></div><div class="casAuthors">Buratto, Jeremie; Colombo, Cinzia; Stupfel, Marine; Dawson, Simon J.; Dolain, Christel; Langlois d'Estaintot, Beatrice; Fischer, Lucile; Granier, Thierry; Laguerre, Michel; Gallois, Bernard; Huc, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">883-887</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In the search for mols. that could recognize sizeable areas of protein surfaces, a series of ten helical arom. oligoamide foldamers was synthesized on solid phase.  The foldamers comprise three to five monomers carrying various proteinogenic side chains, and exist as racemic mixts. of interconverting right-handed and left-handed helixes.  Functionalization of the foldamers by a nanomolar ligand of human carbonic anhydrase II (HCA) ensured that they would be held in close proximity to the protein surface.  Foldamer-protein interactions were screened by CD.  One foldamer displayed intense CD bands, indicating that a preferred helix handedness is induced upon interacting with the protein surface.  The crystal structure of the complex between this foldamer and HCA could be resolved at 2.1 Å resoln. and revealed a no. of unanticipated protein-foldamer, foldamer-foldamer, and protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV3mXfBqVLRrVg90H21EOLACvtfcHk0lgQxGZd_sc1IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVOhsbbM&md5=8262f6b947521b3f5c228910df27342b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fanie.201309160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201309160%26sid%3Dliteratum%253Aachs%26aulast%3DBuratto%26aufirst%3DJ.%26aulast%3DColombo%26aufirst%3DC.%26aulast%3DStupfel%26aufirst%3DM.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DDolain%26aufirst%3DC.%26aulast%3DLanglois%2Bd%25E2%2580%2599Estaintot%26aufirst%3DB.%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DGranier%26aufirst%3DT.%26aulast%3DLaguerre%26aufirst%3DM.%26aulast%3DGallois%26aufirst%3DB.%26aulast%3DHuc%26aufirst%3DI.%26atitle%3DStructure%2520of%2520a%2520Complex%2520Formed%2520by%2520a%2520Protein%2520and%2520a%2520Helical%2520Aromatic%2520Oligoamide%2520Foldamer%2520at%25202.1%2520%25C3%2585%2520Resolution%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D883%26epage%3D887%26doi%3D10.1002%2Fanie.201309160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">Building proficient enzymes with foldamer prostheses</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6978</span>– <span class="NLM_lpage">6981</span>, <span class="refDoi"> DOI: 10.1002/anie.201400945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201400945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVyls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6978-6981&author=C.+Mayerauthor=M.+M.+M%C3%BCllerauthor=S.+H.+Gellmanauthor=D.+Hilvert&title=Building+proficient+enzymes+with+foldamer+prostheses&doi=10.1002%2Fanie.201400945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Building Proficient Enzymes with Foldamer Prostheses</span></div><div class="casAuthors">Mayer, Clemens; Mueller, Manuel M.; Gellman, Samuel H.; Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">6978-6981</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Foldamers are non-natural oligomers that adopt stable conformations reminiscent of those found in proteins.  To evaluate the potential of foldameric subunits for catalysis, semisynthetic enzymes contg. foldamer fragments constructed from α- and β-amino acid residues were designed and characterized.  Systematic variation of the α→β substitution pattern and types of β-residue afforded highly proficient hybrid catalysts, thus demonstrating the feasibility of expanding the enzyme-engineering toolkit with non-natural backbones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBv0OoJCNCzbVg90H21EOLACvtfcHk0lgfcI7Qb8TTDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVyls7o%253D&md5=680180c6c0ad89a5c4d4cccc2f09bd0e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201400945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201400945%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%2BM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DBuilding%2520proficient%2520enzymes%2520with%2520foldamer%2520prostheses%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6978%26epage%3D6981%26doi%3D10.1002%2Fanie.201400945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">6810</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/ja5013849</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5013849" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=6810-6813&author=P.+S.+P.+Wangauthor=J.+B.+Nguyenauthor=A.+Schepartz&title=Design+and+High-Resolution+Structure+of+a+%CE%B23-Peptide+Bundle+Catalyst&doi=10.1021%2Fja5013849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span></div><div class="casAuthors">Wang, Pam S. P.; Nguyen, Jennifer B.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6810-6813</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the widespread exploration of α-peptides as catalysts, there are few examples of β-peptides that alter a chem. transformation.  Previous work demonstrated that a special class of β3-peptides spontaneously self-assembles in water into discrete protein-like bundles possessing unique quaternary structures and exceptional thermodn. stability.  Here the authors describe β3-peptide bundles capable of both substrate binding and chem. catalysis-ester hydrolysis.  A combination of kinetic and high-resoln. structural anal. suggests an active site triad composed of residues from at least two strands of the octameric bundle structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJGELSSHUnErVg90H21EOLACvtfcHk0lgfcI7Qb8TTDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D&md5=7489eb9f9626d5e958289732a577455b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja5013849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5013849%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%2BS.%2BP.%26aulast%3DNguyen%26aufirst%3DJ.%2BB.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DDesign%2520and%2520High-Resolution%2520Structure%2520of%2520a%2520%25CE%25B23-Peptide%2520Bundle%2520Catalyst%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D6810%26epage%3D6813%26doi%3D10.1021%2Fja5013849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolffs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delsuc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veldman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anh, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meskers, S. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, R. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenning, A. P. H. J.</span></span> <span> </span><span class="NLM_article-title">Helical aromatic oligoamide foldamers as organizational scaffolds for photoinduced charge transfer</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">4819</span>– <span class="NLM_lpage">4829</span>, <span class="refDoi"> DOI: 10.1021/ja809367u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja809367u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtFyqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=4819-4829&author=M.+Wolffsauthor=N.+Delsucauthor=D.+Veldmanauthor=N.+V.+Anhauthor=R.+M.+Williamsauthor=S.+C.+J.+Meskersauthor=R.+A.+J.+Janssenauthor=I.+Hucauthor=A.+P.+H.+J.+Schenning&title=Helical+aromatic+oligoamide+foldamers+as+organizational+scaffolds+for+photoinduced+charge+transfer&doi=10.1021%2Fja809367u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Aromatic Oligoamide Foldamers as Organizational Scaffolds for Photoinduced Charge Transfer</span></div><div class="casAuthors">Wolffs, Martin; Delsuc, Nicolas; Veldman, Dirk; Nguyexn, Van Anh; Williams, Rene M.; Meskers, Stefan C. J.; Janssen, Rene A. J.; Huc, Ivan; Schenning, Albertus P. H. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4819-4829</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we report the synthesis and characterization of four quinoline-derived foldamers with increasing oligomeric length; dimer O2P, tetramer O4P, pentamer O5P, and nonamer O9P functionalized with on one end an oligo(p-phenylene vinylene) (OPV) and on the other end a perylene bisimide (PB) chromophore. 1H NMR confirms the formation of the expected folded structures in both toluene and chloroform soln.  The structural predictability and rigidity of the oligomeric series enabled us to investigate the effect of a helical bridge and chromophore position on the photoinduced processes in the electron OPV-PB donor-acceptor pair in chloroform and toluene.  The helical properties of the bridge ensured that the chromophore sepn. distance through space is different from the sepn. distance through the bridge.  For all foldamer-solvent combinations studied, excitation of either OPV or PB results in nearly quant. quenching of the fluorescence indicating a fast charge sepn. reaction between the OPV and PB.  Femtosecond photoinduced absorption measurements confirmed the fast formation of a charge-sepd. state.  The recombination reaction involves a combination of direct decay to the ground state and the formation of an intermediate triplet state, with their balance depending on the foldamer-solvent combination.  MO calcns. rationalize the fast photoinduced charge sepn., by revealing that the bridging foldamer mediates the charge transfer from donor to acceptor via the superexchange mechanism.  Remarkably low attenuation factors (βCS ≈ 10-2 Å-1) were obtained using either through space or through bridge sepn. distance.  However, in these calcns. only three of the four foldamers show the expected linear behavior between the logarithm of the charge sepn. rate const. and the distance between the chromophores.  The combined results show when a helical bridge is sepg. the charge transfer couple, hampering the usefulness of a uniform description of the charge-sepn. phenomena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBIZYeMF7LcrVg90H21EOLACvtfcHk0lgfcI7Qb8TTDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtFyqsrs%253D&md5=9b7f38deceff442df983090afc1f27ce</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fja809367u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja809367u%26sid%3Dliteratum%253Aachs%26aulast%3DWolffs%26aufirst%3DM.%26aulast%3DDelsuc%26aufirst%3DN.%26aulast%3DVeldman%26aufirst%3DD.%26aulast%3DAnh%26aufirst%3DN.%2BV.%26aulast%3DWilliams%26aufirst%3DR.%2BM.%26aulast%3DMeskers%26aufirst%3DS.%2BC.%2BJ.%26aulast%3DJanssen%26aufirst%3DR.%2BA.%2BJ.%26aulast%3DHuc%26aufirst%3DI.%26aulast%3DSchenning%26aufirst%3DA.%2BP.%2BH.%2BJ.%26atitle%3DHelical%2520aromatic%2520oligoamide%2520foldamers%2520as%2520organizational%2520scaffolds%2520for%2520photoinduced%2520charge%2520transfer%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D4819%26epage%3D4829%26doi%3D10.1021%2Fja809367u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamuro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">De Novo Design of Antibacterial β-Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">12200</span>, <span class="refDoi"> DOI: 10.1021/ja992728p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja992728p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK1MXnvVClsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=12200&author=Y.+Hamuroauthor=J.+P.+Schneiderauthor=W.+F.+DeGrado&title=De+Novo+Design+of+Antibacterial+%CE%B2-Peptides&doi=10.1021%2Fja992728p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">De Novo design of antibacterial β-peptides</span></div><div class="casAuthors">Hamuro, Yoshi; Schneider, Joel P.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">12200-12201</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of β-peptides that mimic the activities of a class of natural membrane-active peptide toxins and antibiotics, including magainins, bombolitin, cecropins, melittin and mastoparans, is described.  A series of peptides of graded lengths was synthesized and their ability to adopt an L+2 helix in aq. soln. in the presence and absence of micelles and phospholipid bilayers was assessed.  The effect of a hydrophobic Fmoc group on binding to micelles was examd.  Finally, the biol. activities of these peptides were measured using human erythrocytes and Escherichia coli as models for mammalian and bacterial cells, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvqOIr8MCcG7Vg90H21EOLACvtfcHk0lgHu6vZvg-oLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvVClsrg%253D&md5=f6d16b8310aa08bb43fd23c1941b1fb6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fja992728p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja992728p%26sid%3Dliteratum%253Aachs%26aulast%3DHamuro%26aufirst%3DY.%26aulast%3DSchneider%26aufirst%3DJ.%2BP.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DDe%2520Novo%2520Design%2520of%2520Antibacterial%2520%25CE%25B2-Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D12200%26doi%3D10.1021%2Fja992728p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Chimeric (α/β + α)-Peptide Ligands for the BH3-Recognition Cleft of Bcl-xL: Critical Role of the Molecular Scaffold in Protein Surface Recognition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">11966</span>– <span class="NLM_lpage">11968</span>, <span class="refDoi"> DOI: 10.1021/ja053678t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja053678t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntVert7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=11966-11968&author=J.+D.+Sadowskyauthor=M.+A.+Schmittauthor=H.-S.+Leeauthor=N.+Umezawaauthor=S.+Wangauthor=Y.+Tomitaauthor=S.+H.+Gellman&title=Chimeric+%28%CE%B1%2F%CE%B2+%2B+%CE%B1%29-Peptide+Ligands+for+the+BH3-Recognition+Cleft+of+Bcl-xL%3A+Critical+Role+of+the+Molecular+Scaffold+in+Protein+Surface+Recognition&doi=10.1021%2Fja053678t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Chimeric (α/β + α)-Peptide Ligands for the BH3-Recognition Cleft of Bcl-xL: Critical Role of the Molecular Scaffold in Protein Surface Recognition</span></div><div class="casAuthors">Sadowsky, Jack D.; Schmitt, Margaret A.; Lee, Hee-Seung; Umezawa, Naoki; Wang, Shaomeng; Tomita, York; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">11966-11968</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mols. that bind to sp. surface sites on proteins are of great interest from both fundamental and practical perspectives.  We are exploring a ligand development strategy that is based on oligomers with discrete folding propensities ("foldamers"); we target a specific cleft on the cancer-assocd. protein Bcl-xL because this system is well characterized structurally.  In vivo, this cleft binds to α-helical segments (BH3 domains) of other proteins.  We evaluated several types of helical foldamer, built entirely from β-amino acid residues or from mixts. of α- and β-amino acid residues, and ultimately identified foldamers in the latter class that bind very tightly to Bcl-xL.  Our results suggest that combining different types of foldamer backbones will be an effective and general strategy for creating high-affinity and specific ligands for protein surface sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKnEnRm8PJbrVg90H21EOLACvtfcHk0lgHu6vZvg-oLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntVert7g%253D&md5=a1b03551dbe60effd954fa8ff0aa400d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja053678t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja053678t%26sid%3Dliteratum%253Aachs%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DSchmitt%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DChimeric%2520%2528%25CE%25B1%252F%25CE%25B2%2520%252B%2520%25CE%25B1%2529-Peptide%2520Ligands%2520for%2520the%2520BH3-Recognition%2520Cleft%2520of%2520Bcl-xL%253A%2520Critical%2520Role%2520of%2520the%2520Molecular%2520Scaffold%2520in%2520Protein%2520Surface%2520Recognition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D11966%26epage%3D11968%26doi%3D10.1021%2Fja053678t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lear, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodsdon, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">9468</span>– <span class="NLM_lpage">9469</span>, <span class="refDoi"> DOI: 10.1021/ja031625a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja031625a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=9468-9469&author=J.+A.+Kritzerauthor=J.+D.+Learauthor=M.+E.+Hodsdonauthor=A.+Schepartz&title=Helical+%CE%B2-Peptide+Inhibitors+of+the+p53-hDM2+Interaction&doi=10.1021%2Fja031625a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span></div><div class="casAuthors">Kritzer, Joshua A.; Lear, James D.; Hodsdon, Michael E.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9468-9469</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HDM2 is recognized in vivo by a short α-helix within the p53 trans-activation domain (p53AD).  Disruption of the p53·hDM2 interaction is an important goal for cancer therapy.  A functional epitope comprised of three residues on one face of the p53AD helix (F19, W23, and L26) contributes heavily to the binding free energy.  We hypothesized that the p53AD functional epitope would be recapitulated if the side chains of F19, W23, and L26 were presented at successive positions three residues apart on a stabilized β3-peptide 14-helix.  Here, we report a set of β3-peptides that possess significant 14-helix structure in water; one recognizes a cleft on the surface of hDM2 with nanomolar affinity.  The strategy for β3-peptide design that we describe is general and may have advantages over one in which individual or multiple β-amino acid substitutions are introduced into a functional α-peptide, because it is based on homol. at the level of secondary structure, not primary sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVlZAGh6JntbVg90H21EOLACvtfcHk0lgHgUyxRTyGag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D&md5=3fc9e87497a50588ee4591d5f84927ea</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja031625a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja031625a%26sid%3Dliteratum%253Aachs%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26aulast%3DLear%26aufirst%3DJ.%2BD.%26aulast%3DHodsdon%26aufirst%3DM.%2BE.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DHelical%2520%25CE%25B2-Peptide%2520Inhibitors%2520of%2520the%2520p53-hDM2%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D9468%26epage%3D9469%26doi%3D10.1021%2Fja031625a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gademann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebach, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Biological Evaluation of a Cyclo--tetrapeptide as a Somatostatin Analogue</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1226</span>, <span class="refDoi"> DOI: 10.1002/(sici)1521-3773(19990503)38:9<1223::aid-anie1223>3.0.co;2-a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFOrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1223-1226&author=K.+Gademannauthor=M.+Ernstauthor=D.+Hoyerauthor=D.+Seebach&title=Synthesis+and+Biological+Evaluation+of+a+Cyclo%2D%2Dtetrapeptide+as+a+Somatostatin+Analogue&doi=10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of a cyclo-β-tetrapeptide as a somatostatin analog</span></div><div class="casAuthors">Gademann, Karl; Ernst, Martin; Hoyer, Daniel; Seebach, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1223-1226</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Cyclo-β-tetrapeptide I, a β-peptidic analog of octreotide, was prepd. from β-amino acid building blocks synthesized by Arndt-Eistert homologation of appropriately protected α-amino acids.  The affinity of I for five different human somatostatin receptors was measured using radioligand-binding assays.  I had affinity, although the concns. were in the micromolar and not in the nanomolar range as for somatostatin and octreotide.  The authors have demonstrated that a small β-peptide (consisting of only four β-amino acids) can mimic a natural peptide hormone and display biol. activity and micromolar affinity for human receptors.  Thus, β-peptides can, in principle, be considered as peptidomimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovReDe9QljFLVg90H21EOLACvtfcHk0lgHgUyxRTyGag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFOrsLw%253D&md5=4ee1d15ac384ed8730b85b36f8daaf71</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528sici%25291521-3773%252819990503%252938%253A9%253C1223%253A%253Aaid-anie1223%253E3.0.co%253B2-a%26sid%3Dliteratum%253Aachs%26aulast%3DGademann%26aufirst%3DK.%26aulast%3DErnst%26aufirst%3DM.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DSeebach%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520Biological%2520Evaluation%2520of%2520a%2520Cyclo--tetrapeptide%2520as%2520a%2520Somatostatin%2520Analogue%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1223%26epage%3D1226%26doi%3D10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Werder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abele, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebach, D.</span></span> <span> </span><span class="NLM_article-title">β-Peptides as Inhibitors of Small-Intestinal Cholesterol and Fat Absorption</span>. <i>Helv. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1774</span>– <span class="NLM_lpage">1783</span>, <span class="refDoi"> DOI: 10.1002/(sici)1522-2675(19991006)82:10<1774::aid-hlca1774>3.0.co;2-o</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK1MXmvFensbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1999&pages=1774-1783&author=M.+Werderauthor=H.+Hauserauthor=S.+Abeleauthor=D.+Seebach&title=%CE%B2-Peptides+as+Inhibitors+of+Small-Intestinal+Cholesterol+and+Fat+Absorption&doi=10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides as inhibitors of small-intestinal cholesterol and fat absorption</span></div><div class="casAuthors">Werder, Moritz; Hauser, Helmut; Abele, Stefan; Seebach, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Helvetica Chimica Acta</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1774-1783</span>CODEN:
                <span class="NLM_cas:coden">HCACAV</span>;
        ISSN:<span class="NLM_cas:issn">0018-019X</span>.
    
            (<span class="NLM_cas:orgname">Verlag Helvetica Chimica Acta</span>)
        </div><div class="casAbstract">Selective lipid transport through the brush-border membrane in the small intestine of mammals is mediated by membrane-bound proteins, the so-called scavenger receptors of class B, type I or II (SR-BI or -BII).  These, in turn, are inhibited by certain proteins and synthetic α-peptides that have an amphipathic helix as the binding motif.  In whole cells (test with human colonic carcinoma cells, CaCo-2), on the other hand, the inhibitors are subject to proteolysis.  We have now tested six β-peptides (hexa-, hepta-, and nonamers) each carrying one to seven water-solubilizing side chains of either Ser or Lys, with a brush-border-membrane (BBM) vesicle model system and with a tightly packed monolayer of CaCo-2 cells to find if the rate of transport of cholesterol can be reduced to what may be considered the passive diffusion ("background") level.  There is a correlation between the ability of the β-peptides to form an amphipathic-type 314-helical secondary structure in MeOH and their inhibitory effect.  Although the inhibitory activity of the β-peptides is in only the mM range, it is to be compared with no activity at all of previously tested α-peptides and proteins (built of L-amino acids) in CaCo-2 cells.  Furthermore, these active β-peptides contain only seven or nine residues and must be considered simple, first-generation models capable of mimicking the biol. activity of amphipathic α-peptide helixes in living whole cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjsCK3hVeJ_rVg90H21EOLACvtfcHk0lgHgUyxRTyGag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvFensbc%253D&md5=5e553455f52762ca9f32b78a649036fd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528sici%25291522-2675%252819991006%252982%253A10%253C1774%253A%253Aaid-hlca1774%253E3.0.co%253B2-o%26sid%3Dliteratum%253Aachs%26aulast%3DWerder%26aufirst%3DM.%26aulast%3DHauser%26aufirst%3DH.%26aulast%3DAbele%26aufirst%3DS.%26aulast%3DSeebach%26aufirst%3DD.%26atitle%3D%25CE%25B2-Peptides%2520as%2520Inhibitors%2520of%2520Small-Intestinal%2520Cholesterol%2520and%2520Fat%2520Absorption%26jtitle%3DHelv.%2520Chim.%2520Acta%26date%3D1999%26volume%3D82%26spage%3D1774%26epage%3D1783%26doi%3D10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisblum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Non-haemolytic β-amino-acid oligomers</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>404</i></span>,  <span class="NLM_fpage">565</span>, <span class="refDoi"> DOI: 10.1038/35007145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2F35007145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10766230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD3cXis1Gmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2000&pages=565&author=E.+A.+Porterauthor=X.+Wangauthor=H.-S.+Leeauthor=B.+Weisblumauthor=S.+H.+Gellman&title=Non-haemolytic+%CE%B2-amino-acid+oligomers&doi=10.1038%2F35007145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics: Non-hemolytic β-amino-acid oligomers</span></div><div class="casAuthors">Porter, Emile A.; Wang, Xifang; Lee, Hee-Seung; Weisblum, Bernard; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">404</span>
        (<span class="NLM_cas:issue">6778</span>),
    <span class="NLM_cas:pages">565</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The antibiotic activity of two β-amino acid oligomers was shown to be similar against 4 species of bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTsBnhfOd_b7Vg90H21EOLACvtfcHk0li5fINSR3ubjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXis1Gmtrs%253D&md5=30618cc4dd5ebabdf9f9eb94f39fcfb5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2F35007145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35007145%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DE.%2BA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DWeisblum%26aufirst%3DB.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DNon-haemolytic%2520%25CE%25B2-amino-acid%2520oligomers%26jtitle%3DNature%26date%3D2000%26volume%3D404%26spage%3D565%26doi%3D10.1038%2F35007145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seurynck, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barron, A. E.</span></span> <span> </span><span class="NLM_article-title">Simple, Helical Peptoid Analogs of Lung Surfactant Protein B</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1016%2Fj.chembiol.2004.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=15664517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtlKmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=77-88&author=S.+L.+Seurynckauthor=J.+A.+Patchauthor=A.+E.+Barron&title=Simple%2C+Helical+Peptoid+Analogs+of+Lung+Surfactant+Protein+B&doi=10.1016%2Fj.chembiol.2004.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Simple, Helical Peptoid Analogs of Lung Surfactant Protein B</span></div><div class="casAuthors">Seurynck, Shannon L.; Patch, James A.; Barron, Annelise E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-88</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The helical, amphipathic surfactant protein, SP-B, is a crit. element of pulmonary surfactant and hence is an important therapeutic mol.  However, it is difficult to isolate from natural sources in high purity.  We have created and studied three different, nonnatural analogs of a bioactive SP-B fragment (SP-B1-25), using oligo-N-substituted glycines (peptoids) with simple, repetitive sequences designed to favor the formation of amphiphilic helixes.  For comparison, a peptide with a similar repetitive sequence previously shown to be a good SP mimic was also studied, along with SP-B1-25 itself.  Surface pressure-area isotherms, surfactant film phase morphol., and dynamic adsorption behavior all indicate that the peptoids are promising mimics of SP-B1-25.  The extent of biomimicry appears to correlate with peptoid helicity and lipophilicity.  These biostable oligomers could serve in a synthetic surfactant replacement to treat respiratory distress syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqazsFnExRGh7Vg90H21EOLACvtfcHk0li5fINSR3ubjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtlKmtg%253D%253D&md5=8e2e1e3b706187c27231c4a0dd5f9c63</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DSeurynck%26aufirst%3DS.%2BL.%26aulast%3DPatch%26aufirst%3DJ.%2BA.%26aulast%3DBarron%26aufirst%3DA.%2BE.%26atitle%3DSimple%252C%2520Helical%2520Peptoid%2520Analogs%2520of%2520Lung%2520Surfactant%2520Protein%2520B%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D77%26epage%3D88%26doi%3D10.1016%2Fj.chembiol.2004.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barron, A. E.</span></span> <span> </span><span class="NLM_article-title">Helical peptoid mimics of magainin-2 amide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">12092</span>– <span class="NLM_lpage">12093</span>, <span class="refDoi"> DOI: 10.1021/ja037320d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja037320d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Oktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=12092-12093&author=J.+A.+Patchauthor=A.+E.+Barron&title=Helical+peptoid+mimics+of+magainin-2+amide&doi=10.1021%2Fja037320d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Peptoid Mimics of Magainin-2 Amide</span></div><div class="casAuthors">Patch, James A.; Barron, Annelise E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">12092-12093</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of peptoid oligomers were designed as helical, cationic, and facially amphipathic mimics of the magainin-2 amide antibacterial peptide.  We used CD spectroscopy to det. the conformation of these peptoids in aq. buffer and in the presence of bacterial membrane-mimetic lipid vesicles, composed of a 7:3 mol ratio of POPE:POPG.  We found that certain peptoids, which displayed characteristically helical CD in buffer and lipid vesicles, exhibit selective (nonhemolytic) and potent antibacterial activity against both Gram-pos. and Gram-neg. bacteria.  In contrast, peptoids that exhibit weak CD, reminiscent of that of a peptide random coil, were ineffective antibiotics.  In a manner similar to the natural magainin peptides, we find a correlation between peptoid lipophilicity and hemolytic propensity.  We observe that a min. length of ∼12 peptoid residues may be required for antibacterial activity.  We also see evidence that a helix length between 24 and 34 Å may provide optimal antibacterial efficacy.  These results provide the first example of a water-sol., structured, bioactive peptoid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMVi-2AGxjtLVg90H21EOLACvtfcHk0lgKqWbZNcjzSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Oktr0%253D&md5=05bf4917c48b1f2db52e3be689891c6e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja037320d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja037320d%26sid%3Dliteratum%253Aachs%26aulast%3DPatch%26aufirst%3DJ.%2BA.%26aulast%3DBarron%26aufirst%3DA.%2BE.%26atitle%3DHelical%2520peptoid%2520mimics%2520of%2520magainin-2%2520amide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D12092%26epage%3D12093%26doi%3D10.1021%2Fja037320d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pophristic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemparala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">The Design and Evaluation of Heparin-Binding Foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6847</span>, <span class="refDoi"> DOI: 10.1002/anie.200501279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200501279" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2005&pages=6843-6847&author=S.+Choiauthor=D.+J.+Clementsauthor=V.+Pophristicauthor=I.+Ivanovauthor=S.+Vemparalaauthor=J.+S.+Bennettauthor=M.+L.+Kleinauthor=J.+D.+Winklerauthor=W.+F.+DeGrado&title=The+Design+and+Evaluation+of+Heparin-Binding+Foldamers&doi=10.1002%2Fanie.200501279"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fanie.200501279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200501279%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DClements%26aufirst%3DD.%2BJ.%26aulast%3DPophristic%26aufirst%3DV.%26aulast%3DIvanov%26aufirst%3DI.%26aulast%3DVemparala%26aufirst%3DS.%26aulast%3DBennett%26aufirst%3DJ.%2BS.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DThe%2520Design%2520and%2520Evaluation%2520of%2520Heparin-Binding%2520Foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D117%26spage%3D6843%26epage%3D6847%26doi%3D10.1002%2Fanie.200501279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilsl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wéber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mándity, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fülöp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span> <span> </span><span class="NLM_article-title">Unique α,β- and α,α,β,β-Peptide Foldamers Based on cis-β-Aminocyclopentanecarboxylic Acid</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2208</span>– <span class="NLM_lpage">2212</span>, <span class="refDoi"> DOI: 10.1002/anie.201107702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201107702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=2208-2212&author=%C5%81.+Berlickiauthor=L.+Pilslauthor=E.+W%C3%A9berauthor=I.+M.+M%C3%A1ndityauthor=C.+Cabreleauthor=T.+A.+Martinekauthor=F.+F%C3%BCl%C3%B6pauthor=O.+Reiser&title=Unique+%CE%B1%2C%CE%B2-+and+%CE%B1%2C%CE%B1%2C%CE%B2%2C%CE%B2-Peptide+Foldamers+Based+on+cis-%CE%B2-Aminocyclopentanecarboxylic+Acid&doi=10.1002%2Fanie.201107702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unique α,β- and α,α,β,β-peptide foldamers based on cis-β-aminocyclopentanecarboxylic acid</span></div><div class="casAuthors">Berlicki, Lukasz; Pilsl, Ludwig; Weber, Edit; Mandity, Istvan M.; Cabrele, Chiara; Martinek, Tamas A.; Fueloep, Ferenc; Reiser, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2208-2212, S2208/1-S2208/29</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Non-natural oligomers with well-defined three-dimensional structures (foldamers) raise great interest owing to their ability to mimic and expand the world of biopolymer.  In this paper we present helical peptide foldamers with defined spatial distributions of their functional groups in polar solvents including water.  Stable helical structures based on α-amino and cis-β-aminocyclopentanecarboxylic acids have been design following the stereochem. pattering approach.  All the peptides were successfully obtained by solid phase peptide synthesis and catalytic hydrogenation.  CD, NMR and theor. calcns. were employed for detn. of foldamers. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptO-sm-FJwi7Vg90H21EOLACvtfcHk0lgKqWbZNcjzSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D&md5=7a96f663f67145f32e5519393dd6fa4d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107702%26sid%3Dliteratum%253Aachs%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26aulast%3DPilsl%26aufirst%3DL.%26aulast%3DW%25C3%25A9ber%26aufirst%3DE.%26aulast%3DM%25C3%25A1ndity%26aufirst%3DI.%2BM.%26aulast%3DCabrele%26aufirst%3DC.%26aulast%3DMartinek%26aufirst%3DT.%2BA.%26aulast%3DF%25C3%25BCl%25C3%25B6p%26aufirst%3DF.%26aulast%3DReiser%26aufirst%3DO.%26atitle%3DUnique%2520%25CE%25B1%252C%25CE%25B2-%2520and%2520%25CE%25B1%252C%25CE%25B1%252C%25CE%25B2%252C%25CE%25B2-Peptide%2520Foldamers%2520Based%2520on%2520cis-%25CE%25B2-Aminocyclopentanecarboxylic%2520Acid%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D2208%26epage%3D2212%26doi%3D10.1002%2Fanie.201107702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">β-Peptides: From Structure to Function</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3232</span>, <span class="refDoi"> DOI: 10.1021/cr000045i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000045i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=3219-3232&author=R.+P.+Chengauthor=S.+H.+Gellmanauthor=W.+F.+DeGrado&title=%CE%B2-Peptides%3A+From+Structure+to+Function&doi=10.1021%2Fcr000045i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides: From structure to function</span></div><div class="casAuthors">Cheng, Richard P.; Gellman, Samuel H.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3219-3232</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 187 refs., of the conformational properties and biol. functions, esp. as peptidomimetics, of β-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfycZ30AaKLVg90H21EOLACvtfcHk0ljSQoCt4T0btw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D&md5=40cbf3f3d3c00ec8d6250adf407a663a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcr000045i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000045i%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3D%25CE%25B2-Peptides%253A%2520From%2520Structure%2520to%2520Function%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D3219%26epage%3D3232%26doi%3D10.1021%2Fcr000045i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crapster, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzei, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Extraordinarily Robust Polyproline Type I Peptoid Helices Generatedviathe Incorporation of α-Chiral AromaticN-1-Naphthylethyl Side Chains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">15559</span>– <span class="NLM_lpage">15567</span>, <span class="refDoi"> DOI: 10.1021/ja204755p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja204755p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKrs7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=15559-15567&author=J.+R.+Stringerauthor=J.+A.+Crapsterauthor=I.+A.+Guzeiauthor=H.+E.+Blackwell&title=Extraordinarily+Robust+Polyproline+Type+I+Peptoid+Helices+Generatedviathe+Incorporation+of+%CE%B1-Chiral+AromaticN-1-Naphthylethyl+Side+Chains&doi=10.1021%2Fja204755p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Extraordinarily robust polyproline type I peptoid helices generated via the incorporation of α-chiral aromatic N-1-naphthylethyl side chains</span></div><div class="casAuthors">Stringer, Joseph R.; Crapster, J. Aaron; Guzei, Ilia A.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">15559-15567</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptoids, or oligomers of N-substituted glycines, are a class of foldamers that have shown extraordinary functional potential since their inception nearly two decades ago.  However, the generation of well-defined peptoid secondary structures remains a difficult task.  This challenge is due, in part, to the lack of a thorough understanding of peptoid sequence-structure relationships and, consequently, an incomplete understanding of the peptoid folding process.  We seek to delineate sequence-structure relationships through the systematic study of noncovalent interactions in peptoids and the design of novel amide side chains capable of such interactions.  Herein, we report the synthesis and detailed structural anal. of a series of (S)-N-(1-naphthylethyl)glycine (Ns1npe) peptoid homo-oligomers by X-ray crystallog., NMR spectroscopy, and CD spectroscopy.  Four of these peptoids were found to adopt well-defined structures in the solid state, with dihedral angles similar to those obsd. in polyproline type I (PPI) peptide helixes and in peptoids with α-chiral side chains.  The X-ray crystal structure of a representative Ns1npe tetramer revealed an all cis-amide helix, with approx. three residues per turn, and a helical pitch of approx. 6.0 Å.  2D-NMR anal. of the length-dependent Ns1npe series showed that these peptoids have very high overall backbone amide Kcis/trans values in acetonitrile, indicative of conformationally homogeneous structures in soln.  Addnl., CD spectroscopy studies of the Ns1npe homo-oligomers in acetonitrile and methanol revealed a striking length-dependent increase in ellipticity per amide.  These Ns1npe helixes represent the most robust peptoid helixes to be reported, and the incorporation of (S)-N-(1-naphthylethyl)glycines provides a new approach for the generation of stable helical structure in this important class of foldamers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWOAhoe5DJ0bVg90H21EOLACvtfcHk0ljSQoCt4T0btw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKrs7vO&md5=58c83f67e4dc7954a39bb09f8120dbfb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja204755p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja204755p%26sid%3Dliteratum%253Aachs%26aulast%3DStringer%26aufirst%3DJ.%2BR.%26aulast%3DCrapster%26aufirst%3DJ.%2BA.%26aulast%3DGuzei%26aufirst%3DI.%2BA.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DExtraordinarily%2520Robust%2520Polyproline%2520Type%2520I%2520Peptoid%2520Helices%2520Generatedviathe%2520Incorporation%2520of%2520%25CE%25B1-Chiral%2520AromaticN-1-Naphthylethyl%2520Side%2520Chains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D15559%26epage%3D15567%26doi%3D10.1021%2Fja204755p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuckermann, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huebner, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jewell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlowe, C. K.</span></span> <span> </span><span class="NLM_article-title">Peptoids: a modular approach to drug discovery</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">9367</span>– <span class="NLM_lpage">9371</span>, <span class="refDoi"> DOI: 10.1073/pnas.89.20.9367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1073%2Fpnas.89.20.9367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1409642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK3sXhtFKhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=9367-9371&author=R.+J.+Simonauthor=R.+S.+Kaniaauthor=R.+N.+Zuckermannauthor=V.+D.+Huebnerauthor=D.+A.+Jewellauthor=S.+Banvilleauthor=S.+Ngauthor=L.+Wangauthor=S.+Rosenbergauthor=C.+K.+Marlowe&title=Peptoids%3A+a+modular+approach+to+drug+discovery&doi=10.1073%2Fpnas.89.20.9367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Peptoids:  a modular approach to drug discovery</span></div><div class="casAuthors">Simon, Reyna J.; Kania, Robert S.; Zuckermann, Ronald N.; Huebner, Verena D.; Jewell, David A.; Banville, Steven; Ng, Simon; Wang, Liang; Rosenberg, Steven; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9367-71</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Peptoids, oligomers of N-substituted glycines, are described as a motif for the generation of chem. diverse libraries of novel mols.  Ramachandran-type plots were calcd. and indicate a greater diversity of conformational states available for peptoids than for peptides.  The monomers incorporate t-butyl-based side-chain and 9-fluorenylmethoxy-carbonyl α-amine protection.  The controlled oligomerization of the peptoid monomers was performed manually and robotically with in situ activation by either benzotriazol-1-yloxytris(pyrrolidino)phosphonium hexafluorophosphate or bromotris(pyrrolidino)phosphonium hexafluorophosphate.  Other steps were identical to peptide synthesis using α-(9-fluorenylmethoxycarbonyl)amino acids.  A total of 15 monomers and 10 oligomers (peptoids) are described.  Preliminary data are presented on the stability of a representative oligopeptoid to enzymic hydrolysis.  Peptoid versions of peptide ligands of three biol. systems (bovine pancreatic α-amylase, hepatitis A virus 3C proteinase, and human immunodeficiency virus transactivator-responsive element RNA) were found with affinities comparable to those of the corresponding peptides.  The potential use of libraries of these compds. in receptor- or enzyme-based assays is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqjYjAnzlZWbVg90H21EOLACvtfcHk0lhD35q62wB7gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhtFKhsro%253D&md5=072eed656364cd7cf06c71aa9fb001c3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.20.9367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.20.9367%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DR.%2BJ.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DZuckermann%26aufirst%3DR.%2BN.%26aulast%3DHuebner%26aufirst%3DV.%2BD.%26aulast%3DJewell%26aufirst%3DD.%2BA.%26aulast%3DBanville%26aufirst%3DS.%26aulast%3DNg%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DRosenberg%26aufirst%3DS.%26aulast%3DMarlowe%26aufirst%3DC.%2BK.%26atitle%3DPeptoids%253A%2520a%2520modular%2520approach%2520to%2520drug%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1992%26volume%3D89%26spage%3D9367%26epage%3D9371%26doi%3D10.1073%2Fpnas.89.20.9367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fristrup, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">Triangular prism-shaped β-peptoid helices as unique biomimetic scaffolds</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7013</span>– <span class="NLM_lpage">7022</span>, <span class="refDoi"> DOI: 10.1038/ncomms8013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fncomms8013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=25943784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFylt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7013-7022&author=J.+S.+Laursenauthor=P.+Harrisauthor=P.+Fristrupauthor=C.+A.+Olsen&title=Triangular+prism-shaped+%CE%B2-peptoid+helices+as+unique+biomimetic+scaffolds&doi=10.1038%2Fncomms8013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Triangular prism-shaped β-peptoid helices as unique biomimetic scaffolds</span></div><div class="casAuthors">Laursen, Jonas S.; Harris, Pernille; Fristrup, Peter; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7013</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">β-Peptoids are peptidomimetics based on N-alkylated β-aminopropionic acid residues (or N-alkyl-β-alanines).  This type of peptide mimic has previously been incorporated in biol. active ligands and has been hypothesized to be able to exhibit foldamer properties.  Here we show, for the first time, that β-peptoids can be tuned to fold into stable helical structures.  We provide high-resoln. X-ray crystal structures of homomeric β-peptoid hexamers, which reveal right-handed helical conformations with exactly three residues per turn and a helical pitch of 9.6-9.8 Å between turns.  The presence of folded conformations in soln. is supported by CD spectroscopy showing length- and solvent dependency, and mol. dynamics simulations provide further support for a stabilized helical secondary structure in org. solvent.  We thus outline a framework for future design of novel biomimetics that display functional groups with high accuracy in three dimensions, which has potential for development of new functional materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEynUbb3geprVg90H21EOLACvtfcHk0lhD35q62wB7gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFylt7jN&md5=2ae2ab87cd0f7faf8eab3774ed77ca42</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fncomms8013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8013%26sid%3Dliteratum%253Aachs%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DFristrup%26aufirst%3DP.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DTriangular%2520prism-shaped%2520%25CE%25B2-peptoid%2520helices%2520as%2520unique%2520biomimetic%2520scaffolds%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7013%26epage%3D7022%26doi%3D10.1038%2Fncomms8013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claudon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miclet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Didierjean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ennifar, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span> <span> </span><span class="NLM_article-title">The canonical helix of urea oligomers at atomic resolution: insights into folding-induced axial organization</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1070</span>, <span class="refDoi"> DOI: 10.1002/anie.200905592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200905592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1067-1070&author=L.+Fischerauthor=P.+Claudonauthor=N.+Pendemauthor=E.+Micletauthor=C.+Didierjeanauthor=E.+Ennifarauthor=G.+Guichard&title=The+canonical+helix+of+urea+oligomers+at+atomic+resolution%3A+insights+into+folding-induced+axial+organization&doi=10.1002%2Fanie.200905592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fanie.200905592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200905592%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DClaudon%26aufirst%3DP.%26aulast%3DPendem%26aufirst%3DN.%26aulast%3DMiclet%26aufirst%3DE.%26aulast%3DDidierjean%26aufirst%3DC.%26aulast%3DEnnifar%26aufirst%3DE.%26aulast%3DGuichard%26aufirst%3DG.%26atitle%3DThe%2520canonical%2520helix%2520of%2520urea%2520oligomers%2520at%2520atomic%2520resolution%253A%2520insights%2520into%2520folding-induced%2520axial%2520organization%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D49%26spage%3D1067%26epage%3D1070%26doi%3D10.1002%2Fanie.200905592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Proulx, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopewell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García
Ramos, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubell, W. D.</span></span> <span> </span><span class="NLM_article-title">Azapeptides and their therapeutic potential</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1164</span>, <span class="refDoi"> DOI: 10.4155/fmc.11.74</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.4155%2Ffmc.11.74" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=21806378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFeisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1139-1164&author=C.+Proulxauthor=D.+Sabatinoauthor=R.+Hopewellauthor=J.+Spiegelauthor=Y.+Garc%C3%ADa%0ARamosauthor=W.+D.+Lubell&title=Azapeptides+and+their+therapeutic+potential&doi=10.4155%2Ffmc.11.74"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Azapeptides and their therapeutic potential</span></div><div class="casAuthors">Proulx, Caroline; Sabatino, David; Hopewell, Robert; Spiegel, Jochen; Garciia Ramos, Yeesica; Lubell, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1139-1164</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Azapeptides are peptide analogs in which one or more of the amino residues is replaced by a semicarbazide.  This substitution of a nitrogen for the αα-carbon center results in conformational restrictions, which bend the peptide about the aza-amino acid residue away from a linear geometry.  The resulting azapeptide turn conformations have been obsd. by x-ray crystallog. and spectroscopy, as well as predicted based on computational models.  In biol. active peptide analogs, the aza-substitution has led to enhanced activity and selectivity as well as improved properties, such as prolonged duration of action and metabolic stability.  In light of these characteristics, azapeptides have found important uses as receptor ligands, enzyme inhibitors, drugs, pro-drugs, probes and imaging agents.  Recent improvements in synthetic methods for their procurement have ushered in a new era of azapeptide chem.  This review aims to provide a historical look at the development of azapeptide science along with a focus on recent developments and perspectives on the future of this useful tool for medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjgnDgmvF9zbVg90H21EOLACvtfcHk0lj9XNJ7_4Lw9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFeisbo%253D&md5=3f4150c3eedaff90de608d8b1d1167ab</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.74%26sid%3Dliteratum%253Aachs%26aulast%3DProulx%26aufirst%3DC.%26aulast%3DSabatino%26aufirst%3DD.%26aulast%3DHopewell%26aufirst%3DR.%26aulast%3DSpiegel%26aufirst%3DJ.%26aulast%3DGarc%25C3%25ADa%2BRamos%26aufirst%3DY.%26aulast%3DLubell%26aufirst%3DW.%2BD.%26atitle%3DAzapeptides%2520and%2520their%2520therapeutic%2520potential%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D1139%26epage%3D1164%26doi%3D10.4155%2Ffmc.11.74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malachowski, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broadrup, R. L.</span></span> <span> </span><span class="NLM_article-title">The Synthesis of Azapeptidomimetic β-Lactam Molecules as Potential Protease Inhibitors</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">8962</span>– <span class="NLM_lpage">8969</span>, <span class="refDoi"> DOI: 10.1021/jo026280d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo026280d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD38XoslSnsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2002&pages=8962-8969&author=W.+P.+Malachowskiauthor=C.+Tieauthor=K.+Wangauthor=R.+L.+Broadrup&title=The+Synthesis+of+Azapeptidomimetic+%CE%B2-Lactam+Molecules+as+Potential+Protease+Inhibitors&doi=10.1021%2Fjo026280d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Synthesis of Azapeptidomimetic β-Lactam Molecules as Potential Protease Inhibitors</span></div><div class="casAuthors">Malachowski, William P.; Tie, Chenyang; Wang, Katherine; Broadrup, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8962-8969</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic methods for the construction of a novel peptidomimetic structure are reported.  The structure incorporates a β-lactam and an azapeptide in a peptide backbone with the intention of generating rationally designed substrate-based protease inhibitors.  The β-lactam is formed by subjecting serine or threonine-azapeptides to Mitsunobu reaction conditions.  Importantly, the azapeptidomimetic β-lactam structure permits extended binding inhibition and the synthetic methods to create tetrapeptidomimetic structures are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYAXa4rgpnWbVg90H21EOLACvtfcHk0lj9XNJ7_4Lw9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XoslSnsrc%253D&md5=82e7b97f900cbe68f47febf9635f903c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjo026280d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo026280d%26sid%3Dliteratum%253Aachs%26aulast%3DMalachowski%26aufirst%3DW.%2BP.%26aulast%3DTie%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DBroadrup%26aufirst%3DR.%2BL.%26atitle%3DThe%2520Synthesis%2520of%2520Azapeptidomimetic%2520%25CE%25B2-Lactam%2520Molecules%2520as%2520Potential%2520Protease%2520Inhibitors%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2002%26volume%3D67%26spage%3D8962%26epage%3D8969%26doi%3D10.1021%2Fjo026280d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewandowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapp, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennemers, H.</span></span> <span> </span><span class="NLM_article-title">A crystal structure of an oligoproline PPII-helix, at last</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">15829</span>– <span class="NLM_lpage">15832</span>, <span class="refDoi"> DOI: 10.1021/ja507405j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja507405j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=15829-15832&author=P.+Wilhelmauthor=B.+Lewandowskiauthor=N.+Trappauthor=H.+Wennemers&title=A+crystal+structure+of+an+oligoproline+PPII-helix%2C+at+last&doi=10.1021%2Fja507405j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A Crystal Structure of an Oligoproline PPII-Helix, at Last</span></div><div class="casAuthors">Wilhelm, Patrick; Lewandowski, Bartosz; Trapp, Nils; Wennemers, Helma</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">15829-15832</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first crystal structure of an oligoproline adopting an all-trans polyproline II (PPII) helix is presented.  The high-resoln. structure provides detailed insight into the dimensions and conformational properties of oligoprolines that are important for, e.g., their use as "mol. rulers" and "mol. scaffolds".  The structure also showed that the amides interact with each other within a PPII helix and that water is not necessary for PPII helicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7pc_UwBLWl7Vg90H21EOLACvtfcHk0lj9XNJ7_4Lw9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsLnP&md5=ebe46ea8bcd03187632c2595dc5f2959</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja507405j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja507405j%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DP.%26aulast%3DLewandowski%26aufirst%3DB.%26aulast%3DTrapp%26aufirst%3DN.%26aulast%3DWennemers%26aufirst%3DH.%26atitle%3DA%2520crystal%2520structure%2520of%2520an%2520oligoproline%2520PPII-helix%252C%2520at%2520last%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D15829%26epage%3D15832%26doi%3D10.1021%2Fja507405j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.-T.</span></span> <span> </span><span class="NLM_article-title">Aromatic amide foldamers: structures, properties, and functions</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">5271</span>– <span class="NLM_lpage">5316</span>, <span class="refDoi"> DOI: 10.1021/cr300116k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr300116k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFGhsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2012&pages=5271-5316&author=D.-W.+Zhangauthor=X.+Zhaoauthor=J.-L.+Houauthor=Z.-T.+Li&title=Aromatic+amide+foldamers%3A+structures%2C+properties%2C+and+functions&doi=10.1021%2Fcr300116k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatic Amide Foldamers: Structures, Properties, and Functions</span></div><div class="casAuthors">Zhang, Dan-Wei; Zhao, Xin; Hou, Jun-Li; Li, Zhan-Ting</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5271-5316</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This review surveys the study of arom. amide-based foldamers that are stabilized by intramol. hydrogen bonding.  Since the first report of this family of foldamers in 1994, the major effort in this field has been devoted to the synthesis and structural characterization of new foldamers.  Considerable structural complexity has been realized, which may compare with that of natural proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJVTy5WdopfbVg90H21EOLACvtfcHk0lgjL7U1FNQaBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFGhsr7L&md5=bbc3706fb62786384477feca2dde1da9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcr300116k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr300116k%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.-W.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHou%26aufirst%3DJ.-L.%26aulast%3DLi%26aufirst%3DZ.-T.%26atitle%3DAromatic%2520amide%2520foldamers%253A%2520structures%252C%2520properties%252C%2520and%2520functions%26jtitle%3DChem.%2520Rev.%26date%3D2012%26volume%3D112%26spage%3D5271%26epage%3D5316%26doi%3D10.1021%2Fcr300116k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garric, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Léger, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span> <span> </span><span class="NLM_article-title">Molecular apple peels</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1954</span>– <span class="NLM_lpage">1958</span>, <span class="refDoi"> DOI: 10.1002/anie.200462898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200462898" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt12qt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=1954-1958&author=J.+Garricauthor=J.-M.+L%C3%A9gerauthor=I.+Huc&title=Molecular+apple+peels&doi=10.1002%2Fanie.200462898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular apple peels</span></div><div class="casAuthors">Garric, Joachim; Leger, Jean-Michel; Huc, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1954-1958</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peeling the skin of an apple into a single helical ribbon gives a sort of shell that can be wound back around the apple.  Such a shell can be constructed at the mol. scale using a helix with a reduced diam. at both ends which behaves as a capsule and entraps a small guest such as water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-trEn2-7TvbVg90H21EOLACvtfcHk0lgjL7U1FNQaBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt12qt7s%253D&md5=9b32455e77fe77a96ce9118e7dab7d6f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462898%26sid%3Dliteratum%253Aachs%26aulast%3DGarric%26aufirst%3DJ.%26aulast%3DL%25C3%25A9ger%26aufirst%3DJ.-M.%26aulast%3DHuc%26aufirst%3DI.%26atitle%3DMolecular%2520apple%2520peels%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D1954%26epage%3D1958%26doi%3D10.1002%2Fanie.200462898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helsel, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, B.</span></span> <span> </span><span class="NLM_article-title">Highly conducting transmembrane pores formed by aromatic oligoamide macrocycles</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">15784</span>– <span class="NLM_lpage">15785</span>, <span class="refDoi"> DOI: 10.1021/ja807078y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja807078y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlWqtbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=15784-15785&author=A.+J.+Helselauthor=A.+L.+Brownauthor=K.+Yamatoauthor=W.+Fengauthor=L.+Yuanauthor=A.+J.+Clementsauthor=S.+V.+Hardingauthor=G.+Szaboauthor=Z.+Shaoauthor=B.+Gong&title=Highly+conducting+transmembrane+pores+formed+by+aromatic+oligoamide+macrocycles&doi=10.1021%2Fja807078y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Conducting Transmembrane Pores Formed by Aromatic Oligoamide Macrocycles</span></div><div class="casAuthors">Helsel, Amber Jade; Brown, Amy L.; Yamato, Kazuhiro; Feng, Wen; Yuan, Lihua; Clements, Aimee J.; Harding, Stephanie V.; Szabo, Gabor; Shao, Zhifeng; Gong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">15784-15785</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oligoamide macrocycles, which carry membrane-compatible side chains and contain a hydrophilic, noncollapsible cavity, were found to mediate high ion flux across a lipid bilayer, as demonstrated by results from 23Na NMR and planar bilayer conductance measurements.  The measured transmembrane single channel currents are very high, rivaling those typically assocd. with pore-forming protein toxins.  The obtained results have demonstrated the promise of developing large, highly conducting channels based on nanopores formed by oligoamide macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdx0iVYSKWLbVg90H21EOLACvtfcHk0lgkPOL0yMDauw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlWqtbjK&md5=17f69429ef66f1f8f7575121e33b1c1d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja807078y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja807078y%26sid%3Dliteratum%253Aachs%26aulast%3DHelsel%26aufirst%3DA.%2BJ.%26aulast%3DBrown%26aufirst%3DA.%2BL.%26aulast%3DYamato%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DYuan%26aufirst%3DL.%26aulast%3DClements%26aufirst%3DA.%2BJ.%26aulast%3DHarding%26aufirst%3DS.%2BV.%26aulast%3DSzabo%26aufirst%3DG.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DB.%26atitle%3DHighly%2520conducting%2520transmembrane%2520pores%2520formed%2520by%2520aromatic%2520oligoamide%2520macrocycles%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D15784%26epage%3D15785%26doi%3D10.1021%2Fja807078y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Selective and potent proteomimetic inhibitors of intracellular protein-protein interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2960</span>– <span class="NLM_lpage">2965</span>, <span class="refDoi"> DOI: 10.1002/anie.201410810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201410810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFCitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=2960-2965&author=A.+Barnardauthor=K.+Longauthor=H.+L.+Martinauthor=J.+A.+Milesauthor=T.+A.+Edwardsauthor=D.+C.+Tomlinsonauthor=A.+Macdonaldauthor=A.+J.+Wilson&title=Selective+and+potent+proteomimetic+inhibitors+of+intracellular+protein-protein+interactions&doi=10.1002%2Fanie.201410810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Selective and Potent Proteomimetic Inhibitors of Intracellular Protein-Protein Interactions</span></div><div class="casAuthors">Barnard, Anna; Long, Kerya; Martin, Heather L.; Miles, Jennifer A.; Edwards, Thomas A.; Tomlinson, Darren C.; MacDonald, Andrew; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2960-2965</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Inhibition of protein-protein interactions (PPIs) represents a major challenge in chem. biol. and drug discovery. α-Helix mediated PPIs may be amenable to modulation using generic chemotypes, termed "proteomimetics", which can be assembled in a modular manner to reproduce the vectoral presentation of key side chains found on a helical motif from one partner within the PPI.  In this work, it is demonstrated that by using a library of N-alkylated arom. oligoamide helix mimetics, potent helix mimetics which reproduce their biophys. binding selectivity in a cellular context can be identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWm-xJOpoE8bVg90H21EOLACvtfcHk0lgkPOL0yMDauw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFCitr8%253D&md5=784a4d064135ea0adbb2a291f20e7f0e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.201410810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201410810%26sid%3Dliteratum%253Aachs%26aulast%3DBarnard%26aufirst%3DA.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DMartin%26aufirst%3DH.%2BL.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DTomlinson%26aufirst%3DD.%2BC.%26aulast%3DMacdonald%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DSelective%2520and%2520potent%2520proteomimetic%2520inhibitors%2520of%2520intracellular%2520protein-protein%2520interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D2960%26epage%3D2965%26doi%3D10.1002%2Fanie.201410810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azzarito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of α-helix-mediated protein-protein interactions using designed molecules</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">173</span>, <span class="refDoi"> DOI: 10.1038/nchem.1568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchem.1568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=23422557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=161-173&author=V.+Azzaritoauthor=K.+Longauthor=N.+S.+Murphyauthor=A.+J.+Wilson&title=Inhibition+of+%CE%B1-helix-mediated+protein-protein+interactions+using+designed+molecules&doi=10.1038%2Fnchem.1568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of α-helix-mediated protein-protein interactions using designed molecules</span></div><div class="casAuthors">Azzarito, Valeria; Long, Kerya; Murphy, Natasha S.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-173</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inhibition of protein-protein interactions (PPIs) represents a significant challenge because it is unclear how they can be effectively and selectively targeted using small mols.  Achieving this goal is crit. given the defining role of these interactions in biol. processes.  A rational approach to inhibitor design based on the secondary structure at the interface is the focus of much research, and different classes of designed ligands have emerged, some of which effectively and selectively disrupt targeted PPIs.  This Review discusses the relevance of PPIs and, in particular, the importance of α-helix-mediated PPIs to chem. biol. and drug discovery with a focus on designing inhibitors, including constrained peptides, foldamers and proteomimetic-derived ligands.  In doing so, key challenges and major advances in developing generic approaches for the elaboration of PPI inhibitors are highlighted.  The challenges faced in developing such ligands as drug leads - and how criteria applied to these may differ from conventional small-mol. drugs - are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpsQAFaEkKq7Vg90H21EOLACvtfcHk0lgDEcdyhh2s2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D&md5=161f88aa1c745dee9a95b1fab340b9fe</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1568%26sid%3Dliteratum%253Aachs%26aulast%3DAzzarito%26aufirst%3DV.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DN.%2BS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DInhibition%2520of%2520%25CE%25B1-helix-mediated%2520protein-protein%2520interactions%2520using%2520designed%2520molecules%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D161%26epage%3D173%26doi%3D10.1038%2Fnchem.1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of inhibitors of protein-protein interactions: mimicking peptide binding epitopes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8896</span>– <span class="NLM_lpage">8927</span>, <span class="refDoi"> DOI: 10.1002/anie.201412070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201412070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26119925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=8896-8927&author=M.+Pelay-Gimenoauthor=A.+Glasauthor=O.+Kochauthor=T.+N.+Grossmann&title=Structure-based+design+of+inhibitors+of+protein-protein+interactions%3A+mimicking+peptide+binding+epitopes&doi=10.1002%2Fanie.201412070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Inhibitors of Protein-Protein Interactions: Mimicking Peptide Binding Epitopes</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Glas, Adrian; Koch, Oliver; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">8896-8927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are involved at all levels of cellular organization, thus making the development of PPI inhibitors extremely valuable.  The identification of selective inhibitors is challenging because of the shallow and extended nature of PPI interfaces.  Inhibitors can be obtained by mimicking peptide binding epitopes in their bioactive conformation.  For this purpose, several strategies have been evolved to enable a projection of side chain functionalities in analogy to peptide secondary structures, thereby yielding mols. that are generally referred to as peptidomimetics.  Herein, we introduce a new classification of peptidomimetics (classes A-D) that enables a clear assignment of available approaches.  Based on this classification, the Review summarizes strategies that have been applied for the structure-based design of PPI inhibitors through stabilizing or mimicking turns, β-sheets, and helixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPTEmZpatXbrVg90H21EOLACvtfcHk0lgDEcdyhh2s2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM&md5=447c84aab6275b1057fda9a610918f2b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412070%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DO.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DStructure-based%2520design%2520of%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520mimicking%2520peptide%2520binding%2520epitopes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D8896%26epage%3D8927%26doi%3D10.1002%2Fanie.201412070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, T. T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries-van Leeuwen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodarzifard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agten, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Proline primed helix length as a modulator of the nuclear receptor-coactivator interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">4364</span>– <span class="NLM_lpage">4371</span>, <span class="refDoi"> DOI: 10.1021/ja311748r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311748r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVChurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=4364-4371&author=S.+Fuchsauthor=H.+D.+Nguyenauthor=T.+T.+P.+Phanauthor=M.+F.+Burtonauthor=L.+Nietoauthor=I.+J.+de+Vries-van+Leeuwenauthor=A.+Schmidtauthor=M.+Goodarzifardauthor=S.+M.+Agtenauthor=R.+Roseauthor=C.+Ottmannauthor=L.-G.+Milroyauthor=L.+Brunsveld&title=Proline+primed+helix+length+as+a+modulator+of+the+nuclear+receptor-coactivator+interaction&doi=10.1021%2Fja311748r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Proline primed helix length as a modulator of the nuclear receptor-coactivator interaction</span></div><div class="casAuthors">Fuchs, Sascha; Nguyen, Hoang D.; Phan, Trang T. P.; Burton, Matthew F.; Nieto, Lidia; de Vries-van Leeuwen, Ingrid J.; Schmidt, Andrea; Goodarzifard, Monireh; Agten, Stijn M.; Rose, Rolf; Ottmann, Christian; Milroy, Lech-Gustav; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4364-4371</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nuclear receptor binding to coactivator proteins is an obligate first step in the regulation of gene transcription.  Nuclear receptors preferentially bind to an LXXLL peptide motif which is highly conserved throughout the 300 or so natural coactivator proteins.  This knowledge has shaped current understanding of this fundamental protein-protein interaction, and continues to inspire the search for new drug therapies.  However, sequence specificity beyond the LXXLL motif and the mol. functioning of flanking residues still requires urgent addressing.  Here, ribosome display has been used to reassess the estrogen receptor for new and enlarged peptide recognition motifs, leading to the discovery of a potent and highly evolved PXLXXLLXXP binding consensus.  Mol. modeling and X-ray crystallog. studies have provided the mol. insights on the role of the flanking prolines in priming the length of the α-helix and enabling optimal interactions of the α-helix dipole and its surrounding amino acids with the surface charge clamp and the receptor activation function 2.  These findings represent new structural parameters for modulating the nuclear receptor-coactivator interaction based on linear sequences of proteinogenic amino acids and for the design of chem. modified inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Zscu0TYgNrVg90H21EOLACvtfcHk0lgDEcdyhh2s2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVChurw%253D&md5=bb7ccb4ba793d87df61a7f59e1635931</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja311748r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311748r%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DH.%2BD.%26aulast%3DPhan%26aufirst%3DT.%2BT.%2BP.%26aulast%3DBurton%26aufirst%3DM.%2BF.%26aulast%3DNieto%26aufirst%3DL.%26aulast%3Dde%2BVries-van%2BLeeuwen%26aufirst%3DI.%2BJ.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DGoodarzifard%26aufirst%3DM.%26aulast%3DAgten%26aufirst%3DS.%2BM.%26aulast%3DRose%26aufirst%3DR.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DMilroy%26aufirst%3DL.-G.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DProline%2520primed%2520helix%2520length%2520as%2520a%2520modulator%2520of%2520the%2520nuclear%2520receptor-coactivator%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D4364%26epage%3D4371%26doi%3D10.1021%2Fja311748r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grison, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aitken, D. J.</span></span> <span> </span><span class="NLM_article-title">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein-Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11096</span>– <span class="NLM_lpage">11100</span>, <span class="refDoi"> DOI: 10.1002/anie.201604517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201604517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11096-11100&author=C.+M.+Grisonauthor=J.+A.+Milesauthor=S.+Robinauthor=A.+J.+Wilsonauthor=D.+J.+Aitken&title=An+%CE%B1-Helix-Mimicking+12%2C13-Helix%3A+Designed+%CE%B1%2F%CE%B2%2F%CE%B3-Foldamers+as+Selective+Inhibitors+of+Protein-Protein+Interactions&doi=10.1002%2Fanie.201604517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Grison, Claire M.; Miles, Jennifer A.; Robin, Sylvie; Wilson, Andrew J.; Aitken, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">11096-11100</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A major current challenge in bioorg. chem. is the identification of effective mimics of protein secondary structures that act as inhibitors of protein-protein interactions (PPIs).  In this work, trans-2-aminocyclobutanecarboxylic acid (tACBC) was used as the key β-amino acid component in the design of α/β/γ-peptides to structurally mimic a native α-helix.  Suitably functionalized α/β/γ-peptides assume an α-helix-mimicking 12,13-helix conformation in soln., exhibit enhanced proteolytic stability in comparison to the wild-type α-peptide parent sequence from which they are derived, and act as selective inhibitors of the p53/hDM2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx8w9r7WakLrVg90H21EOLACvtfcHk0lgQe2FlyXIteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM&md5=f7f8e78b97daa7452ae4e655056265c7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.201604517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201604517%26sid%3Dliteratum%253Aachs%26aulast%3DGrison%26aufirst%3DC.%2BM.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DRobin%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DAitken%26aufirst%3DD.%2BJ.%26atitle%3DAn%2520%25CE%25B1-Helix-Mimicking%252012%252C13-Helix%253A%2520Designed%2520%25CE%25B1%252F%25CE%25B2%252F%25CE%25B3-Foldamers%2520as%2520Selective%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11096%26epage%3D11100%26doi%3D10.1002%2Fanie.201604517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azzarito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warriner, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Stereocontrolled protein surface recognition using chiral oligoamide proteomimetic foldamers</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2434</span>– <span class="NLM_lpage">2443</span>, <span class="refDoi"> DOI: 10.1039/c4sc03559c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc4sc03559c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=29308155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2434-2443&author=V.+Azzaritoauthor=J.+A.+Milesauthor=J.+Fisherauthor=T.+A.+Edwardsauthor=S.+L.+Warrinerauthor=A.+J.+Wilson&title=Stereocontrolled+protein+surface+recognition+using+chiral+oligoamide+proteomimetic+foldamers&doi=10.1039%2Fc4sc03559c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Stereocontrolled protein surface recognition using chiral oligo-amide proteomimetic foldamers</span></div><div class="casAuthors">Azzarito, Valeria; Miles, Jennifer A.; Fisher, Julie; Edwards, Thomas A.; Warriner, Stuart L.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2434-2443</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The development of foldamers capable of selective mol. recognition of solvent exposed protein surfaces represents an outstanding challenge in supramol. chem. biol.  Here we introduce an oligo-amide foldamer with well-defined conformation that bears all the hallmarks of an information rich oligomer.  Specifically, the foldamer recognizes its target protein hDM2 leading to inhibition of its protein-protein interaction with p53 in a manner that depends upon the compn., spatial projection and stereochem. of functional groups appended to the scaffold.  Most significantly, selective inhibition of p53/hDM2 can be achieved against four other targets and the selectivity for p53/hDM2 inhibition vs. Mcl-1/NOXA-B inhibition is critically dependent upon the stereochem. of the helix mimetic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7zOD1CDOSLbVg90H21EOLACvtfcHk0lgQe2FlyXIteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCitrY%253D&md5=8690223c203659a046f2004d36e643a1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2Fc4sc03559c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4sc03559c%26sid%3Dliteratum%253Aachs%26aulast%3DAzzarito%26aufirst%3DV.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DFisher%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DWarriner%26aufirst%3DS.%2BL.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DStereocontrolled%2520protein%2520surface%2520recognition%2520using%2520chiral%2520oligoamide%2520proteomimetic%2520foldamers%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D2434%26epage%3D2443%26doi%3D10.1039%2Fc4sc03559c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appella, D. H.</span></span> <span> </span><span class="NLM_article-title">Probing the Structural Requirements of Peptoids That Inhibit HDM2–p53 Interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1995</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/ja056344c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja056344c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD28Xksl2ltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=1995-2004&author=T.+Haraauthor=S.+R.+Durellauthor=M.+C.+Myersauthor=D.+H.+Appella&title=Probing+the+Structural+Requirements+of+Peptoids+That+Inhibit+HDM2%E2%80%93p53+Interactions&doi=10.1021%2Fja056344c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Structural Requirements of Peptoids That Inhibit HDM2-p53 Interactions</span></div><div class="casAuthors">Hara, Toshiaki; Durell, Stewart R.; Myers, Michael C.; Appella, Daniel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1995-2004</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many cellular processes are controlled by protein-protein interactions, and selective inhibition of these interactions could lead to the development of new therapies for several diseases.  For example, overexpression of the protein HDM2 (human double minute 2), which binds to and inactivates the protein p53, has been linked to tumor aggressiveness and drug resistance in cancer.  In creating inhibitors of protein-protein interactions, one strategy is to recreate the three-dimensional arrangement of side chains that are involved in the binding of one protein to another, using a nonnatural scaffold as the attachment point for the side chains.  Here, the authors used oligomeric peptoids as the scaffold to develop a general strategy to rationally design synthetic mols. that can be optimized for inhibition of protein-protein interactions.  Structural information on the HDM2-p53 complex was used to design the first class of peptoid inhibitors, and this work provides, in detail, the strategy to modify peptoids with the appropriate side chains that are effective inhibitors of HDM2-p53 binding.  While the authors initially tried to develop rigid, helical peptoids as HDM2 binders, the best inhibitors were surprisingly peptoids that lacked any helix-promoting groups.  These results indicate that starting with rigid peptoid scaffolds may not always be optimal to develop new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFaKncMNdpFbVg90H21EOLACvtfcHk0lgQe2FlyXIteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xksl2ltg%253D%253D&md5=d4fbb08ec90988e85c2a6dec0ea9fc6e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja056344c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja056344c%26sid%3Dliteratum%253Aachs%26aulast%3DHara%26aufirst%3DT.%26aulast%3DDurell%26aufirst%3DS.%2BR.%26aulast%3DMyers%26aufirst%3DM.%2BC.%26aulast%3DAppella%26aufirst%3DD.%2BH.%26atitle%3DProbing%2520the%2520Structural%2520Requirements%2520of%2520Peptoids%2520That%2520Inhibit%2520HDM2%25E2%2580%2593p53%2520Interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D1995%26epage%3D2004%26doi%3D10.1021%2Fja056344c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lao, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarracino, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heindel, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Rational design of topographical helix mimics as potent inhibitors of protein-protein interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">7877</span>– <span class="NLM_lpage">7888</span>, <span class="refDoi"> DOI: 10.1021/ja502310r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502310r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=7877-7888&author=B.+B.+Laoauthor=K.+Drewauthor=D.+A.+Guarracinoauthor=T.+F.+Brewerauthor=D.+W.+Heindelauthor=R.+Bonneauauthor=P.+S.+Arora&title=Rational+design+of+topographical+helix+mimics+as+potent+inhibitors+of+protein-protein+interactions&doi=10.1021%2Fja502310r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Topographical Helix Mimics as Potent Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Lao, Brooke Bullock; Drew, Kevin; Guarracino, Danielle A.; Brewer, Thomas F.; Heindel, Daniel W.; Bonneau, Richard; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7877-7888</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interactions encompass large surface areas, but often a handful of key residues dominate the binding energy landscape.  Rationally designed small mol. scaffolds that reproduce the relative positioning and disposition of important binding residues, termed "hotspot residues", have been shown to successfully inhibit specific protein complexes.  Although this strategy has led to development of novel synthetic inhibitors of protein complexes, often direct mimicry of natural amino acid residues does not lead to potent inhibitors.  Exptl. screening of focused compd. libraries is used to further optimize inhibitors but the no. of possible designs that can be efficiently synthesized and exptl. tested in academic settings is limited.  We have applied the principles of computational protein design to optimization of nonpeptidic helix mimics as ligands for protein complexes.  We describe the development of computational tools to design helix mimetics from canonical and noncanonical residue libraries and their application to two therapeutically important protein-protein interactions: p53-MDM2 and p300-HIF1α.  The overall study provides a streamlined approach for discovering potent peptidomimetic inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNMadvCjMqobVg90H21EOLACvtfcHk0lg5Z_7m6yVTZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D&md5=e19fc2448fcf0e6b71bdc208e1753209</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja502310r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502310r%26sid%3Dliteratum%253Aachs%26aulast%3DLao%26aufirst%3DB.%2BB.%26aulast%3DDrew%26aufirst%3DK.%26aulast%3DGuarracino%26aufirst%3DD.%2BA.%26aulast%3DBrewer%26aufirst%3DT.%2BF.%26aulast%3DHeindel%26aufirst%3DD.%2BW.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DRational%2520design%2520of%2520topographical%2520helix%2520mimics%2520as%2520potent%2520inhibitors%2520of%2520protein-protein%2520interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D7877%26epage%3D7888%26doi%3D10.1021%2Fja502310r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bautista, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appelbaum, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Bridged β<sup>3</sup>-peptide inhibitors of p53-hDM2 complexation: correlation between affinity and cell permeability</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">2904</span>– <span class="NLM_lpage">2906</span>, <span class="refDoi"> DOI: 10.1021/ja910715u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja910715u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvF2rs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=2904-2906&author=A.+D.+Bautistaauthor=J.+S.+Appelbaumauthor=C.+J.+Craigauthor=J.+Michelauthor=A.+Schepartz&title=Bridged+%CE%B23-peptide+inhibitors+of+p53-hDM2+complexation%3A+correlation+between+affinity+and+cell+permeability&doi=10.1021%2Fja910715u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bridged β3-Peptide Inhibitors of p53-hDM2 Complexation: Correlation between Affinity and Cell Permeability</span></div><div class="casAuthors">Bautista, Arjel D.; Appelbaum, Jacob S.; Craig, Cody J.; Michel, Julien; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2904-2906</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">β-Peptides possess several features that are desirable in peptidomimetics; they are easily synthesized, fold into stable secondary structures in physiol. buffers, and resist proteolysis.  They can also bind to a diverse array of proteins to inhibit their interactions with α-helical ligands. β-peptides are usually not cell-permeable, however, and this feature limits their utility as research tools and potential therapeutics.  Appending an Arg8 sequence to a β-peptide improves uptake but adds considerable mass.  We previously reported that embedding a small cationic patch within a PPII, α-, or β-peptide helix improves uptake without the addn. of significant mass.  In another mass-neutral strategy, Verdine, Walensky, and others have reported that insertion of a hydrocarbon bridge between the i and i + 4 positions of an α-helix also increases cell uptake.  Here we describe a series of β-peptides contg. diether and hydrocarbon bridges and compare them on the basis of cell uptake and localization, affinities for hDM2, and 14-helix structure.  Our results highlight the relative merits of the cationic-patch and hydrophobic-bridge strategies for improving β-peptide uptake and identify a surprising correlation between uptake efficiency and hDM2 affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5ae2kqfCt7Vg90H21EOLACvtfcHk0lg5Z_7m6yVTZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvF2rs70%253D&md5=5d5dd9c67f784887c4903479c58d30a1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja910715u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja910715u%26sid%3Dliteratum%253Aachs%26aulast%3DBautista%26aufirst%3DA.%2BD.%26aulast%3DAppelbaum%26aufirst%3DJ.%2BS.%26aulast%3DCraig%26aufirst%3DC.%2BJ.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DBridged%2520%25CE%25B23-peptide%2520inhibitors%2520of%2520p53-hDM2%2520complexation%253A%2520correlation%2520between%2520affinity%2520and%2520cell%2520permeability%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D2904%26epage%3D2906%26doi%3D10.1021%2Fja910715u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span> <span> </span><span class="NLM_article-title">Terphenyl-based helical mimetics that disrupt the p53/HDM2 interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2704</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.1002/anie.200462316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200462316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Chtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=2704-2707&author=H.+Yinauthor=G.-i.+Leeauthor=H.+S.+Parkauthor=G.+A.+Payneauthor=J.+M.+Rodriguezauthor=S.+M.+Sebtiauthor=A.+D.+Hamilton&title=Terphenyl-based+helical+mimetics+that+disrupt+the+p53%2FHDM2+interaction&doi=10.1002%2Fanie.200462316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Terphenyl-based helical mimetics that disrupt the p53/HDM2 interaction</span></div><div class="casAuthors">Yin, Hang; Lee, Gui-in; Park, Hyung Soon; Payne, Gregory A.; Rodriguez, Johanna M.; Sebti, Said M.; Hamilton, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2704-2707</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">HDM2 regulates p53 by binding to its transactivation domain and promoting its ubiquitin-dependent degrdn.  Crystallog. anal. of the HDM2/p53 complex revealed that three hydrophobic residues (F19, W23, L26) along one face of the p53 helical peptide are essential for binding (see picture).  Terphenyl-based antagonists mimic the α-helical region of p53 and disrupt HDM2/p53 complexation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUrbA6KRguLVg90H21EOLACvtfcHk0lg5Z_7m6yVTZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Chtrs%253D&md5=33c8fc8e94ddc3b6d524fcbe0ed1ebfc</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462316%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DG.-i.%26aulast%3DPark%26aufirst%3DH.%2BS.%26aulast%3DPayne%26aufirst%3DG.%2BA.%26aulast%3DRodriguez%26aufirst%3DJ.%2BM.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26atitle%3DTerphenyl-based%2520helical%2520mimetics%2520that%2520disrupt%2520the%2520p53%252FHDM2%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D2704%26epage%3D2707%26doi%3D10.1002%2Fanie.200462316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shaginian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farooqi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searcey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Evaluation of an α-Helix Mimetic Library Targeting Protein–Protein Interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">5564</span>– <span class="NLM_lpage">5572</span>, <span class="refDoi"> DOI: 10.1021/ja810025g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja810025g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFKlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=5564-5572&author=A.+Shaginianauthor=L.+R.+Whitbyauthor=S.+Hongauthor=I.+Hwangauthor=B.+Farooqiauthor=M.+Searceyauthor=J.+Chenauthor=P.+K.+Vogtauthor=D.+L.+Boger&title=Design%2C+Synthesis%2C+and+Evaluation+of+an+%CE%B1-Helix+Mimetic+Library+Targeting+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Fja810025g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of an α-Helix Mimetic Library Targeting Protein-Protein Interactions</span></div><div class="casAuthors">Shaginian, Alex; Whitby, Landon R.; Hong, Sukwon; Hwang, Inkyu; Farooqi, Bilal; Searcey, Mark; Chen, Jiandong; Vogt, Peter K.; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5564-5572</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A library of α-helix mimetics such as I are prepd. and tested for their abilities to inhibit protein-protein binding such as that between the proteins p53 and HDM2/MDM2.  Eighty compds. based on an initial arylcarbonylaminoarylcarbonylaminobenzoic acid lead compd. are prepd. and tested for their ability to inhibit the binding of p53 to MDM2; aminoacylamino(alkoxy)benzoylamino acid I is found to inhibit the binding most effectively, with an IC50 value of 8 μM.  A library of aminoacylamino(alkoxy)benzoylamino acids based on I are prepd.  Base-mediated substitution of 4-nitro-3-fluorobenzoic acid with alcs. on preparative scale yields 3-alkoxy-4-nitrobenzoic acids with amino acid side chains or close analogs at the alkoxy groups; coupling of the alkoxynitrobenzoic acids with amino acid tert-Bu esters mediated by HOAt and EDC yields a library of 400 nitro(alkoxy)benzoyl amino acid tert-Bu esters II [R = 4-ClC6H4CH2O, MeO, PhCH2O, 2-(3-indolyl)ethoxy, Me3CO2CCH2O, H, 4-MeOC6H4CH2CH2, (S)-MeOCH2OCHMeCH2O, Me2CHO, TIPSOCH2CH2O, 4-TIPSOC6H4CH2CH2O, MeSCH2CH2O, 1-(triphenylmethyl)-4-imidazolemethyl, Me2CHCH2O, (R)-EtCHMeO, BocNH(CH2)4O, EtO, H2NCOCH2O, 2-naphthylmethoxy, PhCH2CH2O; R1 = 4-ClC6H4, Me, PhCH2, 2-(3-indolyl)ethyl, Me3CO2CCH2, H, 4-MeOC6H4CH2, (S)-(Me3CO)CHMe, Me2CH, Me3COCH2, 4-Me3COC6H4CH2, MeSCH2CH2, 1-(triphenylmethyl)-4-imidazolemethyl, Me2CHCH2, (R)-EtCHMe, BocNH(CH2)4, Et, H2NCOCH2, 2-naphthylmethyl, PhCH2CH2; R2 = O2N; TIPS = triisopropylsilyl; Boc = tert-butoxycarbonyl] on roughly 100 mg scale in 90-100% yields.  Microwave-mediated redn. of II (R2 = O2N) with nanopowd. zinc metal and ammonium chloride for 30-300 s yields II (R2 = H2N) ; reaction of II (R2 = H2N) with mixts. of twenty protected amino acids mediated by HOAt and EDC followed by addn. of a polystyrene-bound sulfonyl chloride to scavenge excess II and acid deprotection using HCl in dioxane yields a library of 400 mixts. of 20 aminoacylaminobenzoylamino acids.  The variation of inhibition of the binding of p53 to MDM2 by the library is detd.  The library is generated in stored such that it may be preserved and used for further testing as a α-helix peptidomimetic library for lead compd. generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7t2FewsvR7Vg90H21EOLACvtfcHk0liKxS5IXjjG8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFKlsLs%253D&md5=29e00d62d157b709247f1c279662c776</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fja810025g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja810025g%26sid%3Dliteratum%253Aachs%26aulast%3DShaginian%26aufirst%3DA.%26aulast%3DWhitby%26aufirst%3DL.%2BR.%26aulast%3DHong%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DI.%26aulast%3DFarooqi%26aufirst%3DB.%26aulast%3DSearcey%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Evaluation%2520of%2520an%2520%25CE%25B1-Helix%2520Mimetic%2520Library%2520Targeting%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D5564%26epage%3D5572%26doi%3D10.1021%2Fja810025g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henchey, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">High Specificity in Protein Recognition by Hydrogen-Bond-Surrogate α-Helices: Selective Inhibition of the p53/MDM2 Complex</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2104</span>– <span class="NLM_lpage">2107</span>, <span class="refDoi"> DOI: 10.1002/cbic.201000378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fcbic.201000378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=20821791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVKksbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=2104-2107&author=L.+K.+Hencheyauthor=J.+R.+Porterauthor=I.+Ghoshauthor=P.+S.+Arora&title=High+Specificity+in+Protein+Recognition+by+Hydrogen-Bond-Surrogate+%CE%B1-Helices%3A+Selective+Inhibition+of+the+p53%2FMDM2+Complex&doi=10.1002%2Fcbic.201000378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">High specificity in protein recognition by hydrogen-bond-surrogate α-helices: selective inhibition of the p53/MDM2 complex</span></div><div class="casAuthors">Henchey, Laura K.; Porter, Jason R.; Ghosh, Indraneel; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2104-2107</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthetic helixes target chosen protein-protein interactions with high specificity, as assessed by an in vitro split-protein reassembly assay.  By using the p53/MDM2 complex as a model system, both the efficacy of the designed ligands and the suitability of the split-protein assay were detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCog0s7HPIxrVg90H21EOLACvtfcHk0liKxS5IXjjG8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVKksbfK&md5=e574ed4e820a96e900c31b37e80d87ce</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000378%26sid%3Dliteratum%253Aachs%26aulast%3DHenchey%26aufirst%3DL.%2BK.%26aulast%3DPorter%26aufirst%3DJ.%2BR.%26aulast%3DGhosh%26aufirst%3DI.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DHigh%2520Specificity%2520in%2520Protein%2520Recognition%2520by%2520Hydrogen-Bond-Surrogate%2520%25CE%25B1-Helices%253A%2520Selective%2520Inhibition%2520of%2520the%2520p53%252FMDM2%2520Complex%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D2104%26epage%3D2107%26doi%3D10.1002%2Fcbic.201000378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Bcl-2/MDM2 Dual Inhibitors Based on Universal Pyramid-Like α-Helical Mimetics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3152</span>– <span class="NLM_lpage">3162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Cns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3152-3162&author=Z.+Wangauthor=T.+Songauthor=Y.+Fengauthor=Z.+Guoauthor=Y.+Fanauthor=W.+Xuauthor=L.+Liuauthor=A.+Wangauthor=Z.+Zhang&title=Bcl-2%2FMDM2+Dual+Inhibitors+Based+on+Universal+Pyramid-Like+%CE%B1-Helical+Mimetics&doi=10.1021%2Facs.jmedchem.5b01913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Bcl-2/MDM2 Dual Inhibitors Based on Universal Pyramid-Like α-Helical Mimetics</span></div><div class="casAuthors">Wang, Ziqian; Song, Ting; Feng, Yingang; Guo, Zongwei; Fan, Yudan; Xu, Wenjie; Liu, Lu; Wang, Anhui; Zhang, Zhichao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3152-3162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">No α-helical mimetic that exhibits Bcl-2/MDM2 dual inhibition has been rationally designed due to the different helicities of the α-helixes at their binding interfaces.  Herein, the authors extd. a one-turn α-helix-mimicking ortho-triarene unit from o-phenylene foldamers.  Linking benzamide substrates with a rotatable C-N bond, the authors constructed a novel semirigid pyramid-like scaffold that could support its two-turn α-helix mimicry without arom. stacking interactions and could adopt the different dihedral angles of the key residues of p53 and BH3-only peptides.  On the basis of this universal scaffold, a series of substituent groups were installed to capture the key residues of both p53TAD and BimBH3 and balance the differences of the bulks between them.  Identified by FP, ITC, and NMR spectroscopy, compd. I that directly binds to Mcl-1, Bcl-2, and MDM2 with balanced submicromolar affinities was obtained.  Cell-based expts. demonstrated its antitumor ability through Bcl-2/MDM2 dual inhibition simultaneously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGb4PhGSW0_rVg90H21EOLACvtfcHk0liKxS5IXjjG8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Cns78%253D&md5=d489ba0172d4ee96622f79eefb88955d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01913%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DBcl-2%252FMDM2%2520Dual%2520Inhibitors%2520Based%2520on%2520Universal%2520Pyramid-Like%2520%25CE%25B1-Helical%2520Mimetics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3152%26epage%3D3162%26doi%3D10.1021%2Facs.jmedchem.5b01913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patgiri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joy, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Nucleation effects in peptide foldamers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">11495</span>– <span class="NLM_lpage">11502</span>, <span class="refDoi"> DOI: 10.1021/ja301953j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja301953j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFartr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=11495-11502&author=A.+Patgiriauthor=S.+T.+Joyauthor=P.+S.+Arora&title=Nucleation+effects+in+peptide+foldamers&doi=10.1021%2Fja301953j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleation Effects in Peptide Foldamers</span></div><div class="casAuthors">Patgiri, Anupam; Joy, Stephen T.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">11495-11502</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oligomers composed of β3-amino acid residues and a mixt. of α- and β3-residues have emerged as proteolytically stable structural mimics of α-helixes.  An attractive feature of these oligomers is that they adopt defined conformations in short sequences.  In this manuscript, we evaluate the impact of β3-residues as compared to their α-amino acid analogs in prenucleated helixes.  Our hydrogen-deuterium exchange results suggest that heterogeneous sequences composed of "αααβ" repeats are conformationally more rigid than the corresponding homogeneous α-peptide helixes, with the macrocycle templating the helical conformation having a significant influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjAfMlp3dZDrVg90H21EOLACvtfcHk0lgfDiGr2QxDtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFartr4%253D&md5=50fb6403d303dd9da4b30091623e9594</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja301953j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja301953j%26sid%3Dliteratum%253Aachs%26aulast%3DPatgiri%26aufirst%3DA.%26aulast%3DJoy%26aufirst%3DS.%2BT.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DNucleation%2520effects%2520in%2520peptide%2520foldamers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D11495%26epage%3D11502%26doi%3D10.1021%2Fja301953j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prabhakaran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilner, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Aromatic Oligoamide Foldamers with a ″Wet Edge″ as Inhibitors of the α-Helix-Mediated p53-hDM2 Protein-Protein Interaction</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2013</i></span>,  <span class="NLM_fpage">3504</span>– <span class="NLM_lpage">3512</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201300069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fejoc.201300069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnt1ShsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2013&publication_year=2013&pages=3504-3512&author=P.+Prabhakaranauthor=A.+Barnardauthor=N.+S.+Murphyauthor=C.+A.+Kilnerauthor=T.+A.+Edwardsauthor=A.+J.+Wilson&title=Aromatic+Oligoamide+Foldamers+with+a+%E2%80%B3Wet+Edge%E2%80%B3+as+Inhibitors+of+the+%CE%B1-Helix-Mediated+p53-hDM2+Protein-Protein+Interaction&doi=10.1002%2Fejoc.201300069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatic Oligoamide Foldamers with a "Wet Edge" as Inhibitors of the α-Helix-Mediated p53-hDM2 Protein-Protein Interaction</span></div><div class="casAuthors">Prabhakaran, Panchami; Barnard, Anna; Murphy, Natasha S.; Kilner, Colin A.; Edwards, Thomas A.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2013</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3504-3512</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This paper describes the design, synthesis and structural anal. of a 3-O-alkylated arom. oligoamide that incorporates an addnl. hydrophilic 6-O-alkyl substituent in the central monomer.  This oligomer exhibits low μ inhibitory potency against the p53-hDM2 interaction compared with its unfunctionalized counterpart and significantly improved soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtLUYowqcD37Vg90H21EOLACvtfcHk0lgfDiGr2QxDtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnt1ShsLo%253D&md5=58a1e780479683205d34aa93ce413c7b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201300069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201300069%26sid%3Dliteratum%253Aachs%26aulast%3DPrabhakaran%26aufirst%3DP.%26aulast%3DBarnard%26aufirst%3DA.%26aulast%3DMurphy%26aufirst%3DN.%2BS.%26aulast%3DKilner%26aufirst%3DC.%2BA.%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DAromatic%2520Oligoamide%2520Foldamers%2520with%2520a%2520%25E2%2580%25B3Wet%2520Edge%25E2%2580%25B3%2520as%2520Inhibitors%2520of%2520the%2520%25CE%25B1-Helix-Mediated%2520p53-hDM2%2520Protein-Protein%2520Interaction%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2013%26volume%3D2013%26spage%3D3504%26epage%3D3512%26doi%3D10.1002%2Fejoc.201300069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saraogi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span> <span> </span><span class="NLM_article-title">α-Helix mimetics as inhibitors of protein–protein interactions</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1414</span>– <span class="NLM_lpage">1417</span>, <span class="refDoi"> DOI: 10.1042/bst0361414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1042%2Fbst0361414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=19021566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVCgsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=1414-1417&author=I.+Saraogiauthor=A.+D.+Hamilton&title=%CE%B1-Helix+mimetics+as+inhibitors+of+protein%E2%80%93protein+interactions&doi=10.1042%2Fbst0361414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">α-Helix mimetics as inhibitors of protein-protein interactions</span></div><div class="casAuthors">Saraogi, Ishu; Hamilton, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1414-1417</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The inhibition of protein-protein interactions using small mols. is a viable approach for the treatment of a range of pathol. conditions that result from a malfunctioning of these interactions.  Our strategy for the design of such agents involves the mimicry of side-chain residues on one face of the α-helix; these residues frequently play a key role in mediating protein-protein interactions.  The first-generation terphenyl scaffold, with a 3,2',2''-substitution pattern, is able to successfully mimic key helix residues and disrupt therapeutically relevant interactions, including the Bcl-XL-Bak and the p53-hDM2 (human double minute 2) interactions that are implicated in cancer.  The second- and third-generation scaffolds have resulted in greater synthetic accessibility and more drug-like character in these mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCNfqRdM67vrVg90H21EOLACvtfcHk0lgfDiGr2QxDtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVCgsb3P&md5=0cc8778ddb2529e3b1188291f7ab074d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1042%2Fbst0361414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbst0361414%26sid%3Dliteratum%253Aachs%26aulast%3DSaraogi%26aufirst%3DI.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26atitle%3D%25CE%25B1-Helix%2520mimetics%2520as%2520inhibitors%2520of%2520protein%25E2%2580%2593protein%2520interactions%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2008%26volume%3D36%26spage%3D1414%26epage%3D1417%26doi%3D10.1042%2Fbst0361414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interactions: lessons from p53/MDM2</span>. <i>Biopol.—Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">686</span>, <span class="refDoi"> DOI: 10.1002/bip.20741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fbip.20741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17427181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2007&pages=657-686&author=J.+K.+Murrayauthor=S.+H.+Gellman&title=Targeting+protein-protein+interactions%3A+lessons+from+p53%2FMDM2&doi=10.1002%2Fbip.20741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions: lessons from p53/MDM2</span></div><div class="casAuthors">Murray, Justin K.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">657-686</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The tremendous challenge of inhibiting therapeutically important protein-protein interactions has created the opportunity to extend traditional medicinal chem. to a new class of targets and to explore nontraditional strategies.  Here we review a widely studied system, the interaction between tumor suppressor p53 and its natural antagonist MDM2, for which both traditional and nontraditional approaches have been reported.  This system has been a testing ground for novel proteomimetic scaffold-based strategies, i.e., for attempts to mimic the recognition surface displayed by a folded protein with unnatural oligomers.  Retroinverso peptides, peptoids, terphenyls, β-hairpins, p-oligobenzamides, β-peptides, and miniproteins have all been explored as inhibitors of the p53/MDM2 interaction, and we focus on these oligomer-based efforts.  Traditional approaches have been successful as well, and we briefly review small mol. inhibitors along with other strategies for reactivation of the p53 pathway, for comparison with oligomer- based approaches.  We close with comments on an emerging dichotomy among protein-protein interaction targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy7SUsGRc4kbVg90H21EOLACvtfcHk0lgXkDu84vFYiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP&md5=8a70d8e515d8cae1284b9c3d1484eee3</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fbip.20741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20741%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DTargeting%2520protein-protein%2520interactions%253A%2520lessons%2520from%2520p53%252FMDM2%26jtitle%3DBiopol.%25E2%2580%2594Pept.%2520Sci.%26date%3D2007%26volume%3D88%26spage%3D657%26epage%3D686%26doi%3D10.1002%2Fbip.20741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">An Ultrahigh Affinityd-Peptide Antagonist Of MDM2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6237</span>– <span class="NLM_lpage">6241</span>, <span class="refDoi"> DOI: 10.1021/jm3005465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3005465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6237-6241&author=C.+Zhanauthor=L.+Zhaoauthor=X.+Weiauthor=X.+Wuauthor=X.+Chenauthor=W.+Yuanauthor=W.-Y.+Luauthor=M.+Pazgierauthor=W.+Lu&title=An+Ultrahigh+Affinityd-Peptide+Antagonist+Of+MDM2&doi=10.1021%2Fjm3005465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">An Ultrahigh Affinity d-Peptide Antagonist Of MDM2</span></div><div class="casAuthors">Zhan, Changyou; Zhao, Le; Wei, Xiaoli; Wu, Xueji; Chen, Xishan; Yuan, Weirong; Lu, Wei-Yue; Pazgier, Marzena; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6237-6241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncoprotein MDM2 neg. regulates the activity and stability of the p53 tumor suppressor and is an important mol. target for anticancer therapy.  Aided by mirror image phage display and native chem. ligation, we have previously discovered several proteolysis-resistant duodecimal d-peptide antagonists of MDM2, termed DPMI-α, β, γ.  The prototypic d-peptide inhibitor DPMI-α binds (25-109)MDM2 at an affinity of 220 nM and kills tumor cells in vitro and inhibits tumor growth in vivo by reactivating the p53 pathway.  Herein, we report the design of a superactive d-peptide antagonist of MDM2, termed DPMI-δ, of which the binding affinity for (25-109)MDM2 has been improved over DPMI-α by 3 orders of magnitude (Kd = 220 pM).  X-ray crystallog. studies validate DPMI-δ as an exceedingly potent inhibitor of the p53-MDM2 interaction, promising to be a highly attractive lead drug candidate for anticancer therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ_Li8SCEm3rVg90H21EOLACvtfcHk0lgXkDu84vFYiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D&md5=5dd937d5023470c7802f68dc7420af55</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm3005465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3005465%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DW.-Y.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DAn%2520Ultrahigh%2520Affinityd-Peptide%2520Antagonist%2520Of%2520MDM2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6237%26epage%3D6241%26doi%3D10.1021%2Fjm3005465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span> <span> </span><span class="NLM_article-title">Awakening guardian angels: drugging the p53 pathway</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1038/nrc2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnrc2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=19935675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=862-873&author=C.+J.+Brownauthor=S.+Lainauthor=C.+S.+Vermaauthor=A.+R.+Fershtauthor=D.+P.+Lane&title=Awakening+guardian+angels%3A+drugging+the+p53+pathway&doi=10.1038%2Fnrc2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Awakening guardian angels: Drugging the p53 pathway</span></div><div class="casAuthors">Brown, Christopher J.; Lain, Sonia; Verma, Chandra S.; Fersht, Alan R.; Lane, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">862-873</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Currently, around 11 million people are living with a tumor that contains an inactivating mutation of TP53 (the human gene that encodes p53) and another 11 million have tumors in which the p53 pathway is partially abrogated through the inactivation of other signaling or effector components.  The p53 pathway is therefore a prime target for new cancer drug development, and several original approaches to drug discovery that could have wide applications to drug development are being used.  In one approach, mols. that activate p53 by blocking protein-protein interactions with MDM2 are in early clin. development.  Remarkable progress has also been made in the development of p53-binding mols. that can rescue the function of certain p53 mutants.  Finally, cell-based assays are being used to discover compds. that exploit the p53 pathway by either seeking targets and compds. that show synthetic lethality with TP53 mutations or by looking for non-genotoxic activators of the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZW-elIe4w6bVg90H21EOLACvtfcHk0lgXkDu84vFYiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM&md5=072f48d2ed466b2deb9d2b999cc65a53</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnrc2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2763%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DLain%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DLane%26aufirst%3DD.%2BP.%26atitle%3DAwakening%2520guardian%2520angels%253A%2520drugging%2520the%2520p53%2520pathway%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D862%26epage%3D873%26doi%3D10.1038%2Fnrc2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilkes, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Distinct roles of MDMX in the regulation of p53 response to ribosomal stress</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.4161/cc.6.2.3719</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.4161%2Fcc.6.2.3719" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17327702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvFClsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=151-155&author=D.+M.+Gilkesauthor=J.+Chen&title=Distinct+roles+of+MDMX+in+the+regulation+of+p53+response+to+ribosomal+stress&doi=10.4161%2Fcc.6.2.3719"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct roles of MDMX in the regulation of p53 response to ribosomal stress</span></div><div class="casAuthors">Gilkes, Daniele M.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">151-155</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Approx. 50% of protein and RNA synthesis in prolifering cells are devoted to ribosomal biogenesis.  Coordination between ribosome biogenesis, growth, and proliferation is crit. for maintenance of homeostasis and tumor suppression.  Aberrant rRNA expression and processing is sensed by p53.  Ribosomal stress increases the binding between MDM2 and ribosomal proteins L5, L11, and L23, resulting in p53 stabilization.  Our recent study showed that p53 activation by ribosomal stress also involves degrdn. of MDMX in an MDM2-dependent fashion.  Failure to eliminate MDMX due to overexpression results in the sequestration of p53 into inactive complexes, severely impairing p53-dependent cell cycle arrest during ribosomal stress.  Furthermore, MDMX overexpression promotes resistance to the chemotherapeutic agent 5-FU, which at low concns. activates p53 by inhibiting RNA metab.  Therefore, MDMX is an important regulator of p53 response to ribosomal stress.  MDMX overexpression in tumors may significantly influence response to chemotherapy agents that target rRNA biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxs1hKcUKyiLVg90H21EOLACvtfcHk0lgXkDu84vFYiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvFClsLo%253D&md5=2b0b1c5192fd8ca2048b40a7764628f9</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.4161%2Fcc.6.2.3719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.6.2.3719%26sid%3Dliteratum%253Aachs%26aulast%3DGilkes%26aufirst%3DD.%2BM.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DDistinct%2520roles%2520of%2520MDMX%2520in%2520the%2520regulation%2520of%2520p53%2520response%2520to%2520ribosomal%2520stress%26jtitle%3DCell%2520Cycle%26date%3D2007%26volume%3D6%26spage%3D151%26epage%3D155%26doi%3D10.4161%2Fcc.6.2.3719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">In Silico Improvement of β3-Peptide Inhibitors of p53hDM2 and p53hDMX</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">6356</span>– <span class="NLM_lpage">6357</span>, <span class="refDoi"> DOI: 10.1021/ja901478e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja901478e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvVSlt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=6356-6357&author=J.+Michelauthor=E.+A.+Harkerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensenauthor=A.+Schepartz&title=In+Silico+Improvement+of+%CE%B23-Peptide+Inhibitors+of+p53hDM2+and+p53hDMX&doi=10.1021%2Fja901478e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Improvement of β3-Peptide Inhibitors of p53·hDM2 and p53·hDMX</span></div><div class="casAuthors">Michel, Julien; Harker, Elizabeth A.; Tirado-Rives, Julian; Jorgensen, William L.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6356-6357</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is great interest in mols. capable of inhibiting the interactions between p53 and its neg. regulators hDM2 and hDMX, as these mols. have validated potential against cancers in which one or both oncoproteins are overexpressed.  We reported previously that appropriately substituted β3-peptides inhibit these interactions and, more recently, that minimally cationic β3-peptides are sufficiently cell permeable to upregulate p53-dependent genes in live cells.  These observations, coupled with the known stability of β-peptides in a cellular environment, and the recently reported structures of hDM2 and hDMX, motivated us to exploit computational modeling to identify β-peptides with improved potency and/or selectivity.  This exercise successfully identified a new β3-peptide, β53-16, that possesses the highly desirable attribute of high affinity for both hDM2 and hDMX and identifies the 3,4-dichlorophenyl moiety as a novel determinant of hDMX affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcG-5UtJiWA7Vg90H21EOLACvtfcHk0lgOUNfWKvmLqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvVSlt78%253D&md5=6bc5ba84baded67a3edae0bb7c9c2932</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fja901478e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja901478e%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DHarker%26aufirst%3DE.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DIn%2520Silico%2520Improvement%2520of%2520%25CE%25B23-Peptide%2520Inhibitors%2520of%2520p53hDM2%2520and%2520p53hDMX%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D6356%26epage%3D6357%26doi%3D10.1021%2Fja901478e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasprzak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of high affinity peptides inhibiting the interaction of p53 with MDM2 and MDMX</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">2174</span>– <span class="NLM_lpage">2183</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.073056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1074%2Fjbc.m109.073056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=19910468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=2174-2183&author=J.+Phanauthor=Z.+Liauthor=A.+Kasprzakauthor=B.+Liauthor=S.+Sebtiauthor=W.+Guidaauthor=E.+Sch%C3%B6nbrunnauthor=J.+Chen&title=Structure-based+design+of+high+affinity+peptides+inhibiting+the+interaction+of+p53+with+MDM2+and+MDMX&doi=10.1074%2Fjbc.m109.073056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Design of High Affinity Peptides Inhibiting the Interaction of p53 with MDM2 and MDMX</span></div><div class="casAuthors">Phan, Jason; Li, Zhenyu; Kasprzak, Agnieszka; Li, Baozong; Sebti, Said; Guida, Wayne; Schoenbrunn, Ernst; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2174-2183</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">MDM2 and MDMX function as key regulators of p53 by binding to its N terminus, inhibiting its transcriptional activity, and promoting degrdn.  MDM2 and MDMX overexpression or hyperactivation directly contributes to the loss of p53 function during the development of nearly 50% of human cancers.  Recent studies showed that disrupting p53-MDM2 and p53-MDMX interactions can lead to robust activation of p53 but also revealed a need to develop novel dual specific or MDMX-specific inhibitors.  Using phage display we identified a 12-residue peptide (pDI) with inhibitory activity against MDM2 and MDMX.  The co-crystal structures of the pDI and a single mutant deriv. (pDI6W) liganded with the N-terminal domains of human MDMX and MDM2 served as the basis for the design of 11 distinct pDI-deriv. peptides that were tested for inhibitory potential.  The best deriv. (termed pDIQ) contained four amino acid substitutions and exhibited a 5-fold increase in potency over the parent peptide against both MDM2 (IC50 = 8 nM) and MDMX (IC50 = 110 nM).  Further structural studies revealed key mol. features enabling the high affinity binding of the pDIQ to these proteins.  These include large conformational changes of the pDIQ to reach into a hydrophobic site unique to MDMX.  The findings suggest new strategies toward the rational design of small mol. inhibitors efficiently targeting MDMX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajV2RNjc7GbVg90H21EOLACvtfcHk0lgOUNfWKvmLqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsVSquw%253D%253D&md5=b9022411e67fb66bbb04b5de7a1ef426</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m109.073056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m109.073056%26sid%3Dliteratum%253Aachs%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DKasprzak%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DGuida%26aufirst%3DW.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DStructure-based%2520design%2520of%2520high%2520affinity%2520peptides%2520inhibiting%2520the%2520interaction%2520of%2520p53%2520with%2520MDM2%2520and%2520MDMX%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D2174%26epage%3D2183%26doi%3D10.1074%2Fjbc.m109.073056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">p53 inactivation by MDM2 and MDMX negative feedback loops in testicular germ cell tumors</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1411</span>– <span class="NLM_lpage">1420</span>, <span class="refDoi"> DOI: 10.4161/cc.9.7.11255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.4161%2Fcc.9.7.11255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=20372076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ktbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1411-1420&author=B.+Liauthor=Q.+Chengauthor=Z.+Liauthor=J.+Chen&title=p53+inactivation+by+MDM2+and+MDMX+negative+feedback+loops+in+testicular+germ+cell+tumors&doi=10.4161%2Fcc.9.7.11255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">p53 inactivation by MDM2 and MDMX negative feedback loops in testicular germ cell tumors</span></div><div class="casAuthors">Li, Baozong; Cheng, Qian; Li, Zhenyu; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1411-1420</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Testicular germ cell tumors (TGCT) are unique in their excellent response to DNA-damaging chemotherapy.  Mutation of p53 is rare in both untreated and relapsed TGCTs, suggesting that p53 fails to respond effectively against malignant transformation in germ cells.  Previous studies implicated the presence of a poorly defined TGCT-specific mechanism of p53 inactivation.  Here we show that disruption of p53-mdm2 binding using the MDM2-specific inhibitor Nutlin activates p53 in TGCT cells and is sufficient to induce strong apoptosis.  Knockdown of MDMX cooperates with Nutlin to activate p53.  Surprisingly, we found that p53 activation induced a two-fold increase in MDMX mRNA and protein expression in TGCT cells.  A p53-responsive promoter is identified in MDMX intron 1 that contains a functional p53-binding site, suggesting that MDMX also functions as a neg. feedback regulator of p53 in a cell line-dependent fashion.  These findings suggest that MDM2 and MDMX are responsible for the functional inactivation of p53 in TGCT.  Furthermore, TGCT cells are unique in having a strong apoptosis response to p53.  Direct activation of p53 by targeting MDM2 and MDMX may provide a backup approach for the treatment of TGCTs resistant to DNA-damaging drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoApVK333wJULVg90H21EOLACvtfcHk0lgOUNfWKvmLqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ktbnJ&md5=e9c7545437b87a8d45d23a03215b2f6a</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.7.11255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.7.11255%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DCheng%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3Dp53%2520inactivation%2520by%2520MDM2%2520and%2520MDMX%2520negative%2520feedback%2520loops%2520in%2520testicular%2520germ%2520cell%2520tumors%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D1411%26epage%3D1420%26doi%3D10.4161%2Fcc.9.7.11255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">The synthesis of head-to-tail cyclic sulfono-γ-AApeptides</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1039/c4ob02232g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc4ob02232g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=25420701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=672-676&author=H.+Wuauthor=F.+Sheauthor=W.+Gaoauthor=A.+Princeauthor=Y.+Liauthor=L.+Weiauthor=A.+Mercerauthor=L.+Wojtasauthor=S.+Maauthor=J.+Cai&title=The+synthesis+of+head-to-tail+cyclic+sulfono-%CE%B3-AApeptides&doi=10.1039%2Fc4ob02232g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">The synthesis of head-to-tail cyclic sulfono-γ-AApeptides</span></div><div class="casAuthors">Wu, Haifan; She, Fengyu; Gao, Wenyang; Prince, Austin; Li, Yaqiong; Wei, Lulu; Mercer, Allison; Wojtas, Lukasz; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">672-676</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report an efficient method for the prepn. of unprecedented head-to-tail cyclic sulfono-γ-AApeptides.  Following this method, a no. of cyclic sequences bearing two to five subunits were efficiently synthesized.  In addn., the X-ray crystal structure study of a three-membered cyclic sulfono-γ-AApeptide revealed a type II β-turn-like character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNwhl4FWEKMbVg90H21EOLACvtfcHk0ljlYAMsS1tOVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtrnN&md5=d0b2e2b03ad122bd15ffdbf21542b380</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2Fc4ob02232g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4ob02232g%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DPrince%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMercer%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DThe%2520synthesis%2520of%2520head-to-tail%2520cyclic%2520sulfono-%25CE%25B3-AApeptides%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D672%26epage%3D676%26doi%3D10.1039%2Fc4ob02232g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Rapid Access to Multiple Classes of Peptidomimetics from Common γ-AApeptide Building Blocks</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">1760</span>– <span class="NLM_lpage">1765</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201301841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fejoc.201301841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFOhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=1760-1765&author=H.+Wuauthor=P.+Tengauthor=J.+Cai&title=Rapid+Access+to+Multiple+Classes+of+Peptidomimetics+from+Common+%CE%B3-AApeptide+Building+Blocks&doi=10.1002%2Fejoc.201301841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid access to multiple classes of peptidomimetics from common γ-AApeptide building blocks</span></div><div class="casAuthors">Wu, Haifan; Teng, Peng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1760-1765</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein we report a highly efficient new method for prepg. γ-AApeptides capable of theor. contg. any functionality; the method uses a few common N-alloc γ-AApeptide building blocks.  More importantly, using the same approach, new classes of peptidomimetics bearing novel backbone scaffolds can also be readily generated.  Our results not only demonstrate the versatility of this new synthetic method, but also highlight the possibility that the resulting novel peptidomimetics may find discrete biomedical/biomaterial applications in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppIRJmB8Y14LVg90H21EOLACvtfcHk0ljlYAMsS1tOVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFOhsg%253D%253D&md5=5c74d803796da979aad31546baa4053e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201301841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201301841%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRapid%2520Access%2520to%2520Multiple%2520Classes%2520of%2520Peptidomimetics%2520from%2520Common%2520%25CE%25B3-AApeptide%2520Building%2520Blocks%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2014%26volume%3D2014%26spage%3D1760%26epage%3D1765%26doi%3D10.1002%2Fejoc.201301841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: Design, Structure, and Applications</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=428-441&author=Y.+Shiauthor=P.+Tengauthor=P.+Sangauthor=F.+Sheauthor=L.+Weiauthor=J.+Cai&title=%CE%B3-AApeptides%3A+Design%2C+Structure%2C+and+Applications&doi=10.1021%2Facs.accounts.5b00492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: Design, structure, and applications</span></div><div class="casAuthors">Shi, Yan; Teng, Peng; Sang, Peng; She, Fengyu; Wei, Lulu; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">428-441</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of sequence-specific peptidomimetics has led to a variety of fascinating discoveries in chem. biol.  Many peptidomimetics can mimic primary, secondary, and even tertiary structure of peptides and proteins, and because of their unnatural backbones, they also possess significantly enhanced resistance to enzymic hydrolysis, improved bioavailability, and chemodiversity.  It is known that peptide nucleic acids (PNAs) are peptidic sequences developed for the mimicry of nucleic acids; however, their unique backbone as the mol. scaffold of peptidomimetics to mimic structure and function of bioactive peptides has not been investigated systematically.  As such, we recently developed a new class of peptidomimetics, "γ-AApeptides", based on the chiral γ-PNA backbone.  They are termed γ-AApeptides because they are the oligomers of γ-substituted-N-acylated-N-aminoethyl amino acids.  Similar to other classes of peptidomimetics, γ-AApeptides are also resistant to proteolytic degrdn. and possess the potential to enhance chemodiversity.  Moreover, in our scientific journey on the exploration of this class of peptidomimetics, we have discovered some intriguing structures and functions of γ-AApeptides.  In this account, we summarize the current development and application of γ-AApeptides with biol. potential.  Briefly, both linear and cyclic (either through head-to-tail or head-to-side-chain cyclization) γ-AApeptides with diverse functional groups can be synthesized easily on the solid phase using the synthetic protocol we developed. γ-AApeptides could mimic the primary structure of peptides, as they project the same no. of side chains as peptides of the same lengths.  For instance, they could mimic the Tat peptide to permeate cell membranes and bind to HIV RNA with high specificity and affinity.  Certain γ-AApeptides show similar activity to the RGD peptide and target integrin specifically on the cell surface.  γ-AApeptides with function akin to fMLF peptides are also identified.  More importantly, we found that γ-AApeptides can fold into discrete secondary structures, such as helical and β-turn-like structures.  Therefore, they could be rationally designed for a range of biol. applications.  For instance, γ-AApeptides can mimic host-defense peptides and display potent and broad-spectrum activity toward a panel of drug-resistant bacterial pathogens.  Meanwhile, because of their stability against proteolysis and their chemodiversity, γ-AApeptides are also amenable for combinatorial screening.  We demonstrate that, through combinatorial selection, certain γ-AApeptides are identified to inhibit Aβ40 peptide aggregation, suggesting their potential use as a mol. probe to intervene in Alzheimer's disease.  In addn., a few γ-AApeptides identified from the γ-AApeptide library have been shown to bind to the DNA-binding domain of STAT3 and antagonize STAT3/DNA interactions.  Our studies suggest that, with further studies and exploration on both structures and functions, γ-AApeptides may emerge to be a new class of peptidomimetics that play an important role in chem. biol. and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrk8PLbKcOxrVg90H21EOLACvtfcHk0lj-M2ftk5V8FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D&md5=414ab3568ccfdcebb5863d876218804c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00492%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520Design%252C%2520Structure%252C%2520and%2520Applications%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D428%26epage%3D441%26doi%3D10.1021%2Facs.accounts.5b00492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides as a New Class of Peptidomimetics</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5458</span>– <span class="NLM_lpage">5466</span>, <span class="refDoi"> DOI: 10.1002/chem.201504936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fchem.201504936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26945679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5458-5466&author=P.+Tengauthor=Y.+Shiauthor=P.+Sangauthor=J.+Cai&title=%CE%B3-AApeptides+as+a+New+Class+of+Peptidomimetics&doi=10.1002%2Fchem.201504936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides as a New Class of Peptidomimetics</span></div><div class="casAuthors">Teng, Peng; Shi, Yan; Sang, Peng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5458-5466</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Sequence-specific peptidomimetics are mols. that mimic the structure and function of peptides and proteins.  With new backbones and mol. frameworks, peptidomimetics are of considerable interest in addressing challenges encountered in chem. biol. and biomedical sciences.  Based on the γ-PNA backbone, a new class of peptidomimetics - "γ-AApeptides" were recently developed.  Both linear and cyclic γ-AApeptides can be synthesized with high efficiency.  Compared with α-peptides, γ-AApeptides are resistant to enzymic degrdn., and amendable to diversification with a variety of chem. groups.  Moreover, they could mimic primary and secondary structure, as well as the function of peptides, and show promise in biol. applications, such as the development of new agents combating bacteria, cancer, and Alzheimer's disease.  A few research outcomes of γ-AApeptides are highlighted in this Concept article, and a future perspective is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj0LfZU9KRa7Vg90H21EOLACvtfcHk0lj-M2ftk5V8FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D&md5=6f8385337d870a546e1e6b56fd8ef3f1</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fchem.201504936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201504936%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%2520as%2520a%2520New%2520Class%2520of%2520Peptidomimetics%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D5458%26epage%3D5466%26doi%3D10.1002%2Fchem.201504936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerrato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Right-handed helical foldamers consisting of de novo d-AApeptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7363</span>– <span class="NLM_lpage">7369</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b03007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b03007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7363-7369&author=P.+Tengauthor=N.+Maauthor=D.+C.+Cerratoauthor=F.+Sheauthor=T.+Odomauthor=X.+Wangauthor=L.-J.+Mingauthor=A.+van+der%0AVaartauthor=L.+Wojtasauthor=H.+Xuauthor=J.+Cai&title=Right-handed+helical+foldamers+consisting+of+de+novo+d-AApeptides&doi=10.1021%2Fjacs.7b03007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Right-handed helical foldamers consisting of de novo D-AA-peptides</span></div><div class="casAuthors">Teng, Peng; Ma, Ning; Cerrato, Darrell Cole; She, Fengyu; Odom, Timothy; Wang, Xiang; Ming, Li-June; van der Vaart, Arjan; Wojtas, Lukasz; Xu, Hai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7363-7369</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New types of foldamer scaffolds are formidably challenging to design and synthesize, yet highly desirable as structural mimics of peptides/proteins with a wide repertoire of functions.  In particular, the development of peptidomimetic helical foldamers holds promise for new biomaterials, catalysts, and drug mols.  Unnatural L-sulfono-γ-AApeptides were recently developed and shown to have potential applications in both biomedical and material sciences.  However, D-sulfono-γ-AApeptides, the enantiomers of L-sulfono-γ-AApeptides, have never been studied due to the lack of high-resoln. three-dimensional structures to guide structure-based design.  Herein, we report the first synthesis and X-ray crystal structures of a series of 2:1 L-amino acid/D-sulfono-γ-AApeptide hybrid foldamers, and elucidate their folded conformation at the at. level.  Single-crystal X-ray crystallog. indicates that this class of oligomers folds into well-defined right-handed helixes with unique helical parameters.  The helical structures were consistent with data obtained from soln. 2D NMR, CD studies, and mol. dynamics simulations.  Our findings are expected to inspire the structure-based design of this type of unique folding biopolymers for biomaterials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaGg_OJWsqYbVg90H21EOLACvtfcHk0lj-M2ftk5V8FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D&md5=fef3f80500953f027687001f3e67f14b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b03007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b03007%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DCerrato%26aufirst%3DD.%2BC.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMing%26aufirst%3DL.-J.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRight-handed%2520helical%2520foldamers%2520consisting%2520of%2520de%2520novo%2520d-AApeptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7363%26epage%3D7369%26doi%3D10.1021%2Fjacs.7b03007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen-bonding-driven 3D supramolecular assembly of peptidomimetic zipper</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5661</span>– <span class="NLM_lpage">5665</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b11997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5661-5665&author=P.+Tengauthor=Z.+Niuauthor=F.+Sheauthor=M.+Zhouauthor=P.+Sangauthor=G.+M.+Grayauthor=G.+Vermaauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=S.+Maauthor=J.+Cai&title=Hydrogen-bonding-driven+3D+supramolecular+assembly+of+peptidomimetic+zipper&doi=10.1021%2Fjacs.7b11997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span></div><div class="casAuthors">Teng, Peng; Niu, Zheng; She, Fengyu; Zhou, Mi; Sang, Peng; Gray, Geoffrey M.; Verma, Gaurav; Wojtas, Lukasz; van der Vaart, Arjan; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5661-5665</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen-bonding-driven three-dimensional (3D) assembly of a peptidomimetic zipper has been established for the first time by using an α/AApeptide zipper that assembles into a de novo lattice arrangement through two layers of hydrogen-bonded linker-directed interactions.  Via a covalently bridged 1D 413-helix, drastic enhancement in stability has been achieved in the formed 3D cryst. supramol. architecture as evidenced by gas-sorption studies.  As the first example of an unnatural peptidic zipper, the dimensional augmentation of the zipper differs from metal-coordinated strategies, and may have general implications for the prepn. of peptidic functional materials for a variety of future applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6NOPVSQVnN7Vg90H21EOLACvtfcHk0ljndy4wzBPAbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D&md5=75ec49a44f96107f9a2472698475157d</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11997%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DZ.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHydrogen-bonding-driven%25203D%2520supramolecular%2520assembly%2520of%2520peptidomimetic%2520zipper%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5661%26epage%3D5665%26doi%3D10.1021%2Fjacs.7b11997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7778</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1002/anie.201903259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201903259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=7778-7782&author=P.+Tengauthor=G.+M.+Grayauthor=M.+Zhengauthor=S.+Singhauthor=X.+Liauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=Orthogonal+Halogen-Bonding-Driven+3D+Supramolecular+Assembly+of+Right-Handed+Synthetic+Helical+Peptides&doi=10.1002%2Fanie.201903259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span></div><div class="casAuthors">Teng, Peng; Gray, Geoffrey M.; Zheng, Mengmeng; Singh, Sylvia; Li, Xiaopeng; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7778-7782</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide-mediated self-assembly is a prevalent method for creating highly ordered supramol. architectures.  Herein, we report the first example of orthogonal C-X···X-C/C-X···π halogen bonding and hydrogen bonding driven cryst. architectures based on synthetic helical peptides bearing hybrids of L-sulfono-γ-AApeptides and natural amino acids.  The combination of halogen bonding, intra-/intermol. hydrogen bonding, and intermol. hydrophobic interactions enabled novel 3D supramol. assembly.  The orthogonal halogen bonding in the supramol. architecture exerts a novel mechanism for the self-assembly of synthetic peptide foldamers and gives new insights into mol. recognition, supramol. design, and rational design of biomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphs8_rPqx-zrVg90H21EOLACvtfcHk0ljndy4wzBPAbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D&md5=c61d38ec3f3a4e3d79d716a1c88a7ec3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fanie.201903259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201903259%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOrthogonal%2520Halogen-Bonding-Driven%25203D%2520Supramolecular%2520Assembly%2520of%2520Right-Handed%2520Synthetic%2520Helical%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D7778%26epage%3D7782%26doi%3D10.1002%2Fanie.201903259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De novo left-handed synthetic peptidomimetic foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der%0AVaartauthor=J.+Cai&title=De+novo+left-handed+synthetic+peptidomimetic+foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0ljndy4wzBPAbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520novo%2520left-handed%2520synthetic%2520peptidomimetic%2520foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B cell lymphoma 9 protein–protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+cell+lymphoma+9+protein%E2%80%93protein+interaction+using+%CE%B1-helix-mimicking+sulfono-%CE%B3-AApeptide+inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0ljsKP1qNFfeWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520cell%2520lymphoma%25209%2520protein%25E2%2580%2593protein%2520interaction%2520using%2520%25CE%25B1-helix-mimicking%2520sulfono-%25CE%25B3-AApeptide%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: design, synthesis and evaluation</span>. <i>New J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1039/c0nj00943a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc0nj00943a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2011&pages=542-545&author=Y.+Niuauthor=Y.+Huauthor=X.+Liauthor=J.+Chenauthor=J.+Cai&title=%CE%B3-AApeptides%3A+design%2C+synthesis+and+evaluation&doi=10.1039%2Fc0nj00943a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: design, synthesis and evaluation</span></div><div class="casAuthors">Niu, You-Hong; Hu, Yao-Gang; Li, Xiao-Long; Chen, Jian-Dong; Cai, Jian-Feng</div><div class="citationInfo"><span class="NLM_cas:title">New Journal of Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">542-545</span>CODEN:
                <span class="NLM_cas:coden">NJCHE5</span>;
        ISSN:<span class="NLM_cas:issn">1144-0546</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A new class of peptide mimics termed "γ-AApeptides" (derived from γ-chiral peptide nucleic acids, PNAs) has been described.  The design and synthesis of γ-AApeptides, and potential bioactivities towards p53/MDM2 interaction were demonstrated. γ-AApeptides were also found to be highly resistant to proteolysis.  The development of sequence-specific γ-AApeptides may lead to a family of peptidomimetics with a new framework for drug discovery or peptide/protein mimicry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJjedI1LYLnLVg90H21EOLACvtfcHk0ljsKP1qNFfeWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D&md5=bcc3c0c6b390ed5bc3b6019ed02191f9</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1039%2Fc0nj00943a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0nj00943a%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520design%252C%2520synthesis%2520and%2520evaluation%26jtitle%3DNew%2520J.%2520Chem.%26date%3D2011%26volume%3D35%26spage%3D542%26epage%3D545%26doi%3D10.1039%2Fc0nj00943a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, S. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, B. K.</span></span> <span> </span><span class="NLM_article-title">A fluorescence polarization assay for the identification of inhibitors of the p53-DM2 protein-protein interaction</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1006/abio.2001.5468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1006%2Fabio.2001.5468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=11779115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsFekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2002&pages=230-236&author=S.+M.+G.+Knightauthor=N.+Umezawaauthor=H.-S.+Leeauthor=S.+H.+Gellmanauthor=B.+K.+Kay&title=A+fluorescence+polarization+assay+for+the+identification+of+inhibitors+of+the+p53-DM2+protein-protein+interaction&doi=10.1006%2Fabio.2001.5468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence polarization assay for the identification of inhibitors of the p53-DM2 protein-protein interaction</span></div><div class="casAuthors">Knight, Stephen M. G.; Umezawa, Naoki; Lee, Hee-Seung; Gellman, Samuel H.; Kay, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">230-236</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Improper function of the tumor suppressor protein p53 is a contributing factor in many human cancers.  In normal cells, p53 acts to arrest the cell cycle in response to DNA damage or nucleotide depletion.  One mechanism of regulating the amt. of p53 in the cell is through the action of the Double Minute 2 protein, DM2 (also known as MDM2), which ubiquitinates p53 and targets it for proteosomal degrdn.  In a no. of human cancers, the DM2 gene is amplified or overexpressed, leading to inadequate levels of p53 for cell cycle arrest or apoptosis.  With the goal of restoring p53 function in cancers that overexpress DM2, we are developing inhibitors of the p53-DM2 protein-protein interaction that structurally mimic the N-terminal segment of p53 that binds to DM2.  To assist this effort, we have devised a fluorescence polarization assay that quantifies the interaction between the N-terminal regions of both proteins in 384-well microtiter plates.  Using this assay, we have demonstrated that a peptide with a nonhydrolyzable β-amino acid substitution binds DM2 with an affinity comparable to a p53 peptide that is composed of only α-amino acids.  (c) 2002 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpndbiy8bbhfLVg90H21EOLACvtfcHk0ljsKP1qNFfeWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsFekuw%253D%253D&md5=9bfe05d43b9c013a1845e1c9e81d035e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1006%2Fabio.2001.5468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.2001.5468%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DS.%2BM.%2BG.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DKay%26aufirst%3DB.%2BK.%26atitle%3DA%2520fluorescence%2520polarization%2520assay%2520for%2520the%2520identification%2520of%2520inhibitors%2520of%2520the%2520p53-DM2%2520protein-protein%2520interaction%26jtitle%3DAnal.%2520Biochem.%26date%3D2002%26volume%3D300%26spage%3D230%26epage%3D236%26doi%3D10.1006%2Fabio.2001.5468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farooqi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">p53 -Helix mimetics antagonize p53/MDM2 interaction and activate p53</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1019</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-04-0342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1158%2F1535-7163.mct-04-0342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=15956260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1019-1025&author=L.+Chenauthor=H.+Yinauthor=B.+Farooqiauthor=S.+Sebtiauthor=A.+D.+Hamiltonauthor=J.+Chen&title=p53+-Helix+mimetics+antagonize+p53%2FMDM2+interaction+and+activate+p53&doi=10.1158%2F1535-7163.mct-04-0342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">p53 α-Helix mimetics antagonize p53/MDM2 interaction and activate p53</span></div><div class="casAuthors">Chen, Lihong; Yin, Hang; Farooqi, Bilal; Sebti, Said; Hamilton, Andrew D.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1019-1025</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Overexpression or hyperactivation of MDM2 contributes to functional inactivation of wild-type p53 in nearly 50% of tumors.  Inhibition of p53 by MDM2 depends on binding between an NH2-terminal (residues 16-28) p53 α-helical peptide and a hydrophobic pocket on MDM2, presenting an attractive target for development of inhibitors against tumors expressing wild-type p53.  Here we report that novel p53 α-helical peptide mimics based on a terphenyl scaffold can inhibit MDM2-p53 binding in vitro and activate p53 in vivo.  Several active compds. have been identified that inhibit MDM2-p53 binding in an ELISA assay with IC50 of 10 to 20 μmol/L and induce p53 accumulation and activation in cell culture at 15 to 40 μmol/L.  These results suggest that p53 α-helical mimetics based on the terphenyl scaffold may be developed into potent p53 activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNgg0V3He8E7Vg90H21EOLACvtfcHk0lhf_fW3jk5GeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLY%253D&md5=ac7ae39142e67f0a070008c88a71c084</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-04-0342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-04-0342%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DFarooqi%26aufirst%3DB.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3Dp53%2520-Helix%2520mimetics%2520antagonize%2520p53%252FMDM2%2520interaction%2520and%2520activate%2520p53%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2005%26volume%3D4%26spage%3D1019%26epage%3D1025%26doi%3D10.1158%2F1535-7163.mct-04-0342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, D. L.</span></span> <span> </span><span class="NLM_article-title">Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">6504</span>– <span class="NLM_lpage">6510</span>, <span class="refDoi"> DOI: 10.1073/pnas.0701572104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1073%2Fpnas.0701572104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17409186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkvFalsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=6504-6510&author=S.+A.+Sawyerauthor=J.+Parschauthor=Z.+Zhangauthor=D.+L.+Hartl&title=Prevalence+of+positive+selection+among+nearly+neutral+amino+acid+replacements+in+Drosophila&doi=10.1073%2Fpnas.0701572104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila</span></div><div class="casAuthors">Sawyer, Stanley A.; Parsch, John; Zhang, Zhi; Hartl, Daniel L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6504-6510</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We have estd. the selective effects of amino acid replacements in natural populations by comparing levels of polymorphism in 91 genes in African populations of Drosophila melanogaster with their divergence from Drosophila simulans.  The genes include about equal nos. whose level of expression in adults is greater in males, greater in females, or approx. equal in the sexes.  Markov chain Monte Carlo methods were used to sample key parameters in the stationary distribution of polymorphism and divergence in a model in which the selective effect of each nonsynonymous mutation is regarded as a random sample from some underlying normal distribution whose mean may differ from one gene to the next.  Our anal. suggests that ≈95% of all nonsynonymous mutations that could contribute to polymorphism or divergence are deleterious, and that the av. proportion of deleterious amino acid polymorphisms in samples is ≈70%.  On the other hand, ≈95% of fixed differences between species are pos. selected; although, the scaled selection coeff. (Nes) is very small.  We est. that ≈46% of amino acid replacements have Nes < 2, ≈84% have Nes < 4, and ≈99% have Nes < 7.  Although pos. selection among amino acid differences between species seems pervasive, most of the selective effects could be regarded as nearly neutral.  There are significant differences in selection between sex-biased and unbiased genes, which relate primarily to the mean of the distributions of mutational effects and the fraction of slightly deleterious and weakly beneficial mutations that are fixed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyGis9gs3gO7Vg90H21EOLACvtfcHk0lhf_fW3jk5GeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkvFalsbg%253D&md5=1ddc72b1a6694d699214611587b9015e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0701572104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0701572104%26sid%3Dliteratum%253Aachs%26aulast%3DSawyer%26aufirst%3DS.%2BA.%26aulast%3DParsch%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHartl%26aufirst%3DD.%2BL.%26atitle%3DPrevalence%2520of%2520positive%2520selection%2520among%2520nearly%2520neutral%2520amino%2520acid%2520replacements%2520in%2520Drosophila%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D6504%26epage%3D6510%26doi%3D10.1073%2Fpnas.0701572104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borcherds, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theillet, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finzel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishall, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dieterich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selenko, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loewer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G. W.</span></span> <span> </span><span class="NLM_article-title">Disorder and residual helicity alter p53-Mdm2 binding affinity and signaling in cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1002</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchembio.1668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=25362358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGiurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1000-1002&author=W.+Borcherdsauthor=F.-X.+Theilletauthor=A.+Katzerauthor=A.+Finzelauthor=K.+M.+Mishallauthor=A.+T.+Powellauthor=H.+Wuauthor=W.+Manieriauthor=C.+Dieterichauthor=P.+Selenkoauthor=A.+Loewerauthor=G.+W.+Daughdrill&title=Disorder+and+residual+helicity+alter+p53-Mdm2+binding+affinity+and+signaling+in+cells&doi=10.1038%2Fnchembio.1668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Disorder and residual helicity alter p53-Mdm2 binding affinity and signaling in cells</span></div><div class="casAuthors">Borcherds, Wade; Theillet, Francois-Xavier; Katzer, Andrea; Finzel, Ana; Mishall, Katie M.; Powell, Anne T.; Wu, Hongwei; Manieri, Wanda; Dieterich, Christoph; Selenko, Philipp; Loewer, Alexander; Daughdrill, Gary W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1000-1002</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Levels of residual structure in disordered interaction domains det. in vitro binding affinities, but whether they exert similar roles in cells is not known.  Here, we show that increasing residual p53 helicity results in stronger Mdm2 binding, altered p53 dynamics, impaired target gene expression and failure to induce cell cycle arrest upon DNA damage.  These results establish that residual structure is an important determinant of signaling fidelity in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgBzekcYfhRrVg90H21EOLACvtfcHk0lhf_fW3jk5GeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGiurbF&md5=d25c8a6df00483c100327dddfd30184c</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1668%26sid%3Dliteratum%253Aachs%26aulast%3DBorcherds%26aufirst%3DW.%26aulast%3DTheillet%26aufirst%3DF.-X.%26aulast%3DKatzer%26aufirst%3DA.%26aulast%3DFinzel%26aufirst%3DA.%26aulast%3DMishall%26aufirst%3DK.%2BM.%26aulast%3DPowell%26aufirst%3DA.%2BT.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DDieterich%26aufirst%3DC.%26aulast%3DSelenko%26aufirst%3DP.%26aulast%3DLoewer%26aufirst%3DA.%26aulast%3DDaughdrill%26aufirst%3DG.%2BW.%26atitle%3DDisorder%2520and%2520residual%2520helicity%2520alter%2520p53-Mdm2%2520binding%2520affinity%2520and%2520signaling%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1000%26epage%3D1002%26doi%3D10.1038%2Fnchembio.1668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Understanding Cell Penetration of Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10241</span>– <span class="NLM_lpage">10287</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.9b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.9b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10241-10287&author=P.+G.+Doughertyauthor=A.+Sahniauthor=D.+Pei&title=Understanding+Cell+Penetration+of+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.9b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Cell Penetration of Cyclic Peptides</span></div><div class="casAuthors">Dougherty, Patrick G.; Sahni, Ashweta; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10241-10287</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Approx. 75% of all disease-relevant human proteins, including those involved in intracellular protein-protein interactions (PPIs), are undruggable with the current drug modalities (i.e., small mols. and biologics).  Macrocyclic peptides provide a potential soln. to these undruggable targets because their larger sizes (relative to conventional small mols.) endow them the capability of binding to flat PPI interfaces with antibody-like affinity and specificity.  Powerful combinatorial library technologies have been developed to routinely identify cyclic peptides as potent, specific inhibitors against proteins including PPI targets.  However, with the exception of a very small set of sequences, the vast majority of cyclic peptides are impermeable to the cell membrane, preventing their application against intracellular targets.  This Review examines common structural features that render most cyclic peptides membrane impermeable, as well as the unique features that allow the minority of sequences to enter the cell interior by passive diffusion, endocytosis/endosomal escape, or other mechanisms.  We also present the current state of knowledge about the mol. mechanisms of cell penetration, the various strategies for designing cell-permeable, biol. active cyclic peptides against intracellular targets, and the assay methods available to quantify their cell-permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0NDA9U7-1YrVg90H21EOLACvtfcHk0ljxlRaPyHUBPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D&md5=65f68a9b69f3f7f33641b6a91835b1dd</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.9b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.9b00008%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DSahni%26aufirst%3DA.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DUnderstanding%2520Cell%2520Penetration%2520of%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10241%26epage%3D10287%26doi%3D10.1021%2Facs.chemrev.9b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span>; <span class="NLM_string-name">Hilinski, G. J.</span></span> <span> </span><span class="NLM_article-title">Stapled peptides for intracellular drug targets</span>. In  <i>Methods in Enzymology</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittrup, K. D.</span>, <span class="NLM_string-name">Verdine, G. L.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">2012</span>; Vol.  <span class="NLM_volume">503</span>, pp  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">33</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1016%2FB978-0-12-396962-0.00001-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=3-33&author=G.+L.+Verdine&author=G.+J.+Hilinskiauthor=K.+D.+Wittrup&author=G.+L.+Verdine&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-396962-0.00001-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-396962-0.00001-X%26sid%3Dliteratum%253Aachs%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStapled%2520peptides%2520for%2520intracellular%2520drug%2520targets%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DWittrup%26aufirst%3DK.%2BD.%26pub%3DAcademic%2520Press%26date%3D2012%26volume%3D503%26spage%3D3%26epage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J. T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Towards understanding cell penetration by stapled peptides</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1039/c4md00131a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc4md00131a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCgsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=111-119&author=Q.+Chuauthor=R.+E.+Moelleringauthor=G.+J.+Hilinskiauthor=Y.-W.+Kimauthor=T.+N.+Grossmannauthor=J.+T.-H.+Yehauthor=G.+L.+Verdine&title=Towards+understanding+cell+penetration+by+stapled+peptides&doi=10.1039%2Fc4md00131a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Towards understanding cell penetration by stapled peptides</span></div><div class="casAuthors">Chu, Qian; Moellering, Raymond E.; Hilinski, Gerard J.; Kim, Young-Woo; Grossmann, Tom N.; Yeh, Johannes T.-H.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Hydrocarbon-stapled α-helical peptides are a new class of targeting mols. capable of penetrating cells and engaging intracellular targets formerly considered intractable.  This technol. has been applied to the development of cell-permeable ligands targeting key intracellular protein-protein interactions.  However, the properties governing cell penetration of hydrocarbon-stapled peptides have not yet been rigorously investigated.  Herein we report our studies to systematically probe cellular uptake of stapled peptides.  We developed a high-throughput epifluorescence microscopy assay to quant. measure stapled peptide intracellular accumulation and demonstrated that this assay yielded highly reproducible results.  Using this assay, we analyzed more than 200 peptides with various sequences, staple positions and types, and found that cell penetration ability is strongly related to staple type and formal charge, whereas other physicochem. parameters do not appear to have a significant effect.  We next investigated the mechanism(s) involved in stapled peptide internalization and have demonstrated that stapled peptides penetrate cells through a clathrin- and caveolin-independent endocytosis pathway that involves, in part, sulfated cell surface proteoglycans, but that also seems to exploit a novel, uncharacterized pathway.  Taken together, staple type and charge are the key phys. properties in detg. the cell penetration ability of stapled peptides, and anionic cell surface proteoglycans might serve as receptors to mediate stapled peptide internalization.  These findings improve our understanding of stapled peptides as chem. probes and potential targeted therapeutics, and provide useful guidelines for the design of next-generation stapled peptides with enhanced cell permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_6exBOPh1FLVg90H21EOLACvtfcHk0ljxlRaPyHUBPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCgsbnE&md5=11558b56a57ba18f95142579b59e0fcc</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1039%2Fc4md00131a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4md00131a%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DQ.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DYeh%26aufirst%3DJ.%2BT.-H.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DTowards%2520understanding%2520cell%2520penetration%2520by%2520stapled%2520peptides%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D111%26epage%3D119%26doi%3D10.1039%2Fc4md00131a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilkes, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coppola, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeatman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuther, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Regulation of MDMX expression by mitogenic signaling</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1999</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1128/mcb.01633-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1128%2Fmcb.01633-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=18172009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlKjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=1999-2010&author=D.+M.+Gilkesauthor=Y.+Panauthor=D.+Coppolaauthor=T.+Yeatmanauthor=G.+W.+Reutherauthor=J.+Chen&title=Regulation+of+MDMX+expression+by+mitogenic+signaling&doi=10.1128%2Fmcb.01633-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MDMX expression by mitogenic signaling</span></div><div class="casAuthors">Gilkes, Daniele M.; Pan, Yu; Coppola, Domenico; Yeatman, Timothy; Reuther, Gary W.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1999-2010</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">MDMX is an important regulator of p53 transcriptional activity and stress response.  MDMX overexpression and gene amplification are implicated in p53 inactivation and tumor development.  Unlike MDM2, MDMX is not inducible by p53, and little is known about its regulation at the transcriptional level.  We found that MDMX levels in tumor cell lines closely correlate with promoter activity and mRNA level.  Activated K-Ras and insulin-like growth factor 1 induce MDMX expression at the transcriptional level through mechanisms that involve the mitogen-activated protein kinase and c-Ets-1 transcription factors.  Pharmacol. inhibition of MEK results in down-regulation of MDMX in tumor cell lines.  MDMX overexpression was detected in ∼50% of human colon tumors and showed strong correlation with increased extracellular signal-regulated kinase phosphorylation.  Therefore, MDMX expression is regulated by mitogenic signaling pathways.  This mechanism may protect normal proliferating cells from p53 but also hamper p53 response during tumor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdKklelbSbnbVg90H21EOLACvtfcHk0ljxlRaPyHUBPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlKjurc%253D&md5=77fa097717f521a378737bc861718bc1</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1128%2Fmcb.01633-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.01633-07%26sid%3Dliteratum%253Aachs%26aulast%3DGilkes%26aufirst%3DD.%2BM.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DCoppola%26aufirst%3DD.%26aulast%3DYeatman%26aufirst%3DT.%26aulast%3DReuther%26aufirst%3DG.%2BW.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DRegulation%2520of%2520MDMX%2520expression%2520by%2520mitogenic%2520signaling%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D1999%26epage%3D2010%26doi%3D10.1128%2Fmcb.01633-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Helical Antimicrobial Sulfono-γ-AApeptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4802</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4802-4811&author=Y.+Liauthor=H.+Wuauthor=P.+Tengauthor=G.+Baiauthor=X.+Linauthor=X.+Zuoauthor=C.+Caoauthor=J.+Cai&title=Helical+Antimicrobial+Sulfono-%CE%B3-AApeptides&doi=10.1021%2Facs.jmedchem.5b00537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Antimicrobial Sulfono-γ-AApeptides</span></div><div class="casAuthors">Li, Yaqiong; Wu, Haifan; Teng, Peng; Bai, Ge; Lin, Xiaoyang; Zuo, Xiaobing; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4802-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Host-defense peptides (HDPs) such as magainin 2 have emerged as potential therapeutic agents combating antibiotic resistance.  Inspired by their structures and mechanism of action, herein the authors report the first example of antimicrobial helical sulfono-γ-AApeptide foldamers.  The lead mol. displays broad-spectrum and potent antimicrobial activity against multi-drug-resistant Gram-pos. and Gram-neg. bacterial pathogens.  Time-kill studies and fluorescence microscopy suggest that sulfono-γ-AApeptides eradicate bacteria by taking a mode of action analogous to that of HDPs.  Clear structure-function relations exist in the studied sequences.  Longer sequences, presumably adopting more-defined helical structures, are more potent than shorter ones.  The sequence with less helical propensity in soln. could be more selective than the stronger helix-forming sequences.  Moreover, this class of antimicrobial agents are resistant to proteolytic degrdn.  These results may lead to the development of a new class of antimicrobial foldamers combating emerging antibiotic-resistant pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPELcUdcGfXrVg90H21EOLACvtfcHk0lgtugWJE_Fo2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D&md5=5eab0445be6682f250016148fac54969</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00537%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHelical%2520Antimicrobial%2520Sulfono-%25CE%25B3-AApeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4802%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.5b00537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Cangjie Yang, Kevin B. Wu, Yu Deng, Jingsong Yuan, <span class="NLM_string-name hlFld-ContribAuthor">Jia Niu</span>. </span><span class="cited-content_cbyCitation_article-title">Geared Toward Applications: A Perspective on Functional Sequence-Controlled Polymers. </span><span class="cited-content_cbyCitation_journal-name">ACS Macro Letters</span><span> <strong>2021,</strong> <em>10 </em>
                                    (2)
                                     , 243-257. <a href="https://doi.org/10.1021/acsmacrolett.0c00855" title="DOI URL">https://doi.org/10.1021/acsmacrolett.0c00855</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmacrolett.0c00855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmacrolett.0c00855%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Macro%2520Letters%26atitle%3DGeared%252BToward%252BApplications%25253A%252BA%252BPerspective%252Bon%252BFunctional%252BSequence-Controlled%252BPolymers%26aulast%3DYang%26aufirst%3DCangjie%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D12122020%26date%3D20012021%26date%3D22012021%26volume%3D10%26issue%3D2%26spage%3D243%26epage%3D257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yan Shi, Peng Sang, Junhao Lu, Pirada Higbee, Lihong Chen, Leixiang Yang, Timothy Odom, Gary Daughdrill, Jiandong Chen, <span class="NLM_string-name hlFld-ContribAuthor">Jianfeng Cai</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design of Right-Handed Heterogeneous Peptidomimetics as Inhibitors of Protein–Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 13187-13196. <a href="https://doi.org/10.1021/acs.jmedchem.0c01638" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01638</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01638%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRational%252BDesign%252Bof%252BRight-Handed%252BHeterogeneous%252BPeptidomimetics%252Bas%252BInhibitors%252Bof%252BProtein%2525E2%252580%252593Protein%252BInteractions%26aulast%3DShi%26aufirst%3DYan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18092020%26date%3D03112020%26volume%3D63%26issue%3D21%26spage%3D13187%26epage%3D13196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peng Sang, Yan Shi, Bo Huang, Songyi Xue, Timothy Odom, <span class="NLM_string-name hlFld-ContribAuthor">Jianfeng Cai</span>. </span><span class="cited-content_cbyCitation_article-title">Sulfono-γ-AApeptides as Helical Mimetics: Crystal Structures and Applications. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2020,</strong> <em>53 </em>
                                    (10)
                                     , 2425-2442. <a href="https://doi.org/10.1021/acs.accounts.0c00482" title="DOI URL">https://doi.org/10.1021/acs.accounts.0c00482</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.0c00482&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.0c00482%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DSulfono-%2525CE%2525B3-AApeptides%252Bas%252BHelical%252BMimetics%25253A%252BCrystal%252BStructures%252Band%252BApplications%26aulast%3DSang%26aufirst%3DPeng%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24072020%26date%3D17092020%26volume%3D53%26issue%3D10%26spage%3D2425%26epage%3D2442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peng Sang, Yan Shi, Pirada Higbee, Minghui Wang, Sami Abdulkadir, Junhao Lu, Gary Daughdrill, Jiandong Chen, <span class="NLM_string-name hlFld-ContribAuthor">Jianfeng Cai</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design and Synthesis of Right-Handed d-Sulfono-γ-AApeptide Helical Foldamers as Potent Inhibitors of Protein–Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>85 </em>
                                    (16)
                                     , 10552-10560. <a href="https://doi.org/10.1021/acs.joc.0c00996" title="DOI URL">https://doi.org/10.1021/acs.joc.0c00996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.0c00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.0c00996%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DRational%252BDesign%252Band%252BSynthesis%252Bof%252BRight-Handed%252Bd-Sulfono-%2525CE%2525B3-AApeptide%252BHelical%252BFoldamers%252Bas%252BPotent%252BInhibitors%252Bof%252BProtein%2525E2%252580%252593Protein%252BInteractions%26aulast%3DSang%26aufirst%3DPeng%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22042020%26date%3D03082020%26date%3D23072020%26volume%3D85%26issue%3D16%26spage%3D10552%26epage%3D10560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Shi</span>, <span class="hlFld-ContribAuthor ">Peng  Sang</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Cai</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of α-helix-mimicking sulfono-γ-AApeptides as p53−MDM2 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (12)
                                     , 1021-1023. <a href="https://doi.org/10.4155/fmc-2021-0044" title="DOI URL">https://doi.org/10.4155/fmc-2021-0044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0044%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B%2525CE%2525B1-helix-mimicking%252Bsulfono-%2525CE%2525B3-AApeptides%252Bas%252Bp53%2525E2%252588%252592MDM2%252Binhibitors%26aulast%3DShi%26aufirst%3DYan%26date%3D2021%26volume%3D13%26issue%3D12%26spage%3D1021%26epage%3D1023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan J.  Perez</span>, <span class="hlFld-ContribAuthor ">Roman A.  Perez</span>, <span class="hlFld-ContribAuthor ">Alberto  Perez</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Modeling as a Tool to Investigate PPI: From Drug Design to Tissue Engineering. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2021,</strong> <em>8 </em><a href="https://doi.org/10.3389/fmolb.2021.681617" title="DOI URL">https://doi.org/10.3389/fmolb.2021.681617</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2021.681617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2021.681617%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DComputational%252BModeling%252Bas%252Ba%252BTool%252Bto%252BInvestigate%252BPPI%25253A%252BFrom%252BDrug%252BDesign%252Bto%252BTissue%252BEngineering%26aulast%3DPerez%26aufirst%3DJuan%2BJ.%26date%3D2021%26date%3D2021%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dehua  Yu</span>, <span class="hlFld-ContribAuthor ">Zhiyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Xiangdong  Cheng</span>, <span class="hlFld-ContribAuthor ">Jiangjiang  Qin</span>. </span><span class="cited-content_cbyCitation_article-title">The role of miRNAs in MDMX‐p53 interplay. </span><span class="cited-content_cbyCitation_journal-name">Journal of Evidence-Based Medicine</span><span> <strong>2021,</strong> <em>14 </em>
                                    (2)
                                     , 152-160. <a href="https://doi.org/10.1111/jebm.12428" title="DOI URL">https://doi.org/10.1111/jebm.12428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/jebm.12428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fjebm.12428%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Evidence-Based%2520Medicine%26atitle%3DThe%252Brole%252Bof%252BmiRNAs%252Bin%252BMDMX%2525E2%252580%252590p53%252Binterplay%26aulast%3DYu%26aufirst%3DDehua%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D2%26spage%3D152%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. Ro  Han</span>, <span class="hlFld-ContribAuthor ">Trishaladevi  Durgannavar</span>, <span class="hlFld-ContribAuthor ">Dohee  Ahn</span>, <span class="hlFld-ContribAuthor ">Sang J.  Chung</span>. </span><span class="cited-content_cbyCitation_article-title">A FRET‐Based Fluorescent Probe to Screen Anticancer Drugs, Inhibiting p73 Binding to MDM2. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (5)
                                     , 830-833. <a href="https://doi.org/10.1002/cbic.202000660" title="DOI URL">https://doi.org/10.1002/cbic.202000660</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000660%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DA%252BFRET%2525E2%252580%252590Based%252BFluorescent%252BProbe%252Bto%252BScreen%252BAnticancer%252BDrugs%25252C%252BInhibiting%252Bp73%252BBinding%252Bto%252BMDM2%26aulast%3DHan%26aufirst%3DA.%2BRo%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D5%26spage%3D830%26epage%3D833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Computational methods-guided design of modulators targeting protein-protein interactions (PPIs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112764. <a href="https://doi.org/10.1016/j.ejmech.2020.112764" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112764</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112764&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112764%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252Bmethods-guided%252Bdesign%252Bof%252Bmodulators%252Btargeting%252Bprotein-protein%252Binteractions%252B%252528PPIs%252529%26aulast%3DQiu%26aufirst%3DYuran%26date%3D2020%26volume%3D207%26spage%3D112764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peng  Sang</span>, <span class="hlFld-ContribAuthor ">Zhihong  Zhou</span>, <span class="hlFld-ContribAuthor ">Yan  Shi</span>, <span class="hlFld-ContribAuthor ">Candy  Lee</span>, <span class="hlFld-ContribAuthor ">Zaid  Amso</span>, <span class="hlFld-ContribAuthor ">David  Huang</span>, <span class="hlFld-ContribAuthor ">Timothy  Odom</span>, <span class="hlFld-ContribAuthor ">Vân T.B.  Nguyen-Tran</span>, <span class="hlFld-ContribAuthor ">Weijun  Shen</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Cai</span>. </span><span class="cited-content_cbyCitation_article-title">The activity of sulfono-γ-AApeptide helical foldamers that mimic GLP-1. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2020,</strong> <em>6 </em>
                                    (20)
                                     , eaaz4988. <a href="https://doi.org/10.1126/sciadv.aaz4988" title="DOI URL">https://doi.org/10.1126/sciadv.aaz4988</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.aaz4988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.aaz4988%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DThe%252Bactivity%252Bof%252Bsulfono-%2525CE%2525B3-AApeptide%252Bhelical%252Bfoldamers%252Bthat%252Bmimic%252BGLP-1%26aulast%3DSang%26aufirst%3DPeng%26date%3D2020%26date%3D2020%26volume%3D6%26issue%3D20%26spage%3Deaaz4988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Interaction of p53 with the crystal structure of MDM2 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>). p53 is shown as the cartoon, whereas MDM2 is shown as the surface representation. (B) Chemical structure of sulfono-γ-AApeptides. a and b denote the chiral side chain and the sulfonamido side chain from the building block, respectively. (C) Crystal structure of a sulfono-γ-AApeptide (CCDC: 1841094).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> (D) Top view of (C). (E,F) Schematic representation of distribution of side chains from sulfono-γ-AApeptides. (E) Side view; (F) top view, helical wheel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <i>K</i><sub>d</sub> and IC<sub>50</sub> data of pure p53 (A) and sulfono-γ-AApeptide <b>PS10</b> (B) to MDM2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A). CD spectra of <b>p53</b> and sulfono-γ-AApeptides (100 μM) measured at room temperature in PBS buffer. (B) CD spectra of <b>p53</b> and sulfono-γ-AApeptides (100 μM) in TFE at room temperature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical shift mapping of <b>PS10</b> binding to MDM2. (A) Overlay of <sup>15</sup>N HSQC spectra of MDM2 before (blue resonances) and after (red resonances) the addition of <b>PS10</b>. HSQC spectra were collected with a twofold and fourfold stoichiometric excess of <b>PS10</b>. (B) Average chemical shift changes in part per million (ppm) for the amide proton and nitrogen resonances in MDM2 p53BD residues binding to <b>PS10</b>. (C,D) Surface image of the MDM2 p53BD structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) Binding of <b>PS10</b> to MDM2. The helical structure was built on the crystal structure of <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C (CCDC: 1841094).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> (B) Overlay of side chains 2a, 4a, and 6a of <b>PS10</b> with Phe19, Trp23, and Leu26 of p53 (green) using PyMOL software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/medium/jm9b00993_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Activation of p53 by stapled sulfono-γ-AApeptides. (A). Luciferase reporter assay. The p53-dependent luciferase transcriptional activation in U2OS cells: the luciferase activities were measured at least three times and the averaged activities along with standard derivations were plotted. (B). Western blotting. Drug treatment lasted for 16 h with 30 μM peptides or 1 μM nutlin. (C) Relative optical density value after 48 h of incubation in the presence of compounds at 200 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-3/acs.jmedchem.9b00993/20200213/images/large/jm9b00993_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00993&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 77 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prabhakaran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baruah, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanjayan, G. J.</span></span> <span> </span><span class="NLM_article-title">Diversifying the structural architecture of synthetic oligomers: the hetero foldamer approach</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">11593</span>– <span class="NLM_lpage">11611</span>, <span class="refDoi"> DOI: 10.1039/c1cc13313f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc1cc13313f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=21850306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlejtrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=11593-11611&author=A.+Royauthor=P.+Prabhakaranauthor=P.+K.+Baruahauthor=G.+J.+Sanjayan&title=Diversifying+the+structural+architecture+of+synthetic+oligomers%3A+the+hetero+foldamer+approach&doi=10.1039%2Fc1cc13313f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Diversifying the structural architecture of synthetic oligomers: the hetero foldamer approach</span></div><div class="casAuthors">Roy, Arup; Prabhakaran, Panchami; Baruah, Pranjal Kumar; Sanjayan, Gangadhar J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">11593-11611</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Conformationally ordered synthetic oligomers, also called "foldamers", have attracted considerable attention in recent years owing to their ability to mimic the structural architecture of biopolymers and also because of their potential applications in biomedical and material science fields.  Until recently, the major focus in this area has been the development of oligomers featuring a single type of monomer building blocks.  However, due to the enormous possibility of augmenting the conformational space available for oligomer design, the hetero foldamer approach has been introduced very recently.  This feature article aims to describe foldamers with unique structural architectures, exclusively featuring heterogeneous backbones (hetero foldamers).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLPCTen5iporVg90H21EOLACvtfcHk0lip3KkI7LRDTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlejtrjP&md5=6f5fffd8d92c29653afbc5389702eb3b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1039%2Fc1cc13313f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc13313f%26sid%3Dliteratum%253Aachs%26aulast%3DRoy%26aufirst%3DA.%26aulast%3DPrabhakaran%26aufirst%3DP.%26aulast%3DBaruah%26aufirst%3DP.%2BK.%26aulast%3DSanjayan%26aufirst%3DG.%2BJ.%26atitle%3DDiversifying%2520the%2520structural%2520architecture%2520of%2520synthetic%2520oligomers%253A%2520the%2520hetero%2520foldamer%2520approach%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26spage%3D11593%26epage%3D11611%26doi%3D10.1039%2Fc1cc13313f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span> <span> </span><span class="NLM_article-title">Synthetic foldamers</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5933</span>– <span class="NLM_lpage">5941</span>, <span class="refDoi"> DOI: 10.1039/c1cc11137j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc1cc11137j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=21483969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVyju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2011&pages=5933-5941&author=G.+Guichardauthor=I.+Huc&title=Synthetic+foldamers&doi=10.1039%2Fc1cc11137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic foldamers</span></div><div class="casAuthors">Guichard, Gilles; Huc, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5933-5941</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Foldamers are artificial folded mol. architectures inspired by the structures and functions of biopolymers.  This highlight focuses on important developments concerning foldamers produced by chem. synthesis and on the perspectives that these new self-organized mol. scaffolds offer.  Progress in the field has led to synthetic objects that resemble small proteins in terms of size and complexity yet that may not contain any α-amino acids.  Foldamers have introduced new tools and concepts to develop biol. active substances, synthetic receptors and novel materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow03chLS_njLVg90H21EOLACvtfcHk0lip3KkI7LRDTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVyju7c%253D&md5=b304218c67b482fa823a33ddde249ada</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2Fc1cc11137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cc11137j%26sid%3Dliteratum%253Aachs%26aulast%3DGuichard%26aufirst%3DG.%26aulast%3DHuc%26aufirst%3DI.%26atitle%3DSynthetic%2520foldamers%26jtitle%3DChem.%2520Commun.%26date%3D2011%26volume%3D47%26spage%3D5933%26epage%3D5941%26doi%3D10.1039%2Fc1cc11137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horne, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Foldamers with heterogeneous backbones</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1408</span>, <span class="refDoi"> DOI: 10.1021/ar800009n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar800009n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFajtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1399-1408&author=W.+S.+Horneauthor=S.+H.+Gellman&title=Foldamers+with+heterogeneous+backbones&doi=10.1021%2Far800009n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Foldamers with Heterogeneous Backbones</span></div><div class="casAuthors">Horne, W. Seth; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1399-1408</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The functions performed by proteins and nucleic acids provide the foundation for life.  Chemists have recently begun to ask whether it is possible to design synthetic oligomers that approach the structural and functional complexities of these biopolymers.  The study of foldamers, non-natural oligomers displaying discrete folding propensities, has demonstrated that there are several synthetic backbones that exhibit biopolymer-like conformational behavior.  Early work in this area focused on oligomers comprised of a single type of monomer subunit, but recent efforts have highlighted the potential of mixed or "heterogeneous" backbones to expand the structural and functional repertoire of foldamers.  In this Account, we illustrate the promise of heterogeneous backbone foldamers by focusing on examples contg. both α- and β-amino acid residues.  Some β-residues bear protein-like side chains, while others have cyclic structures that confer conformational rigidity.  The study of heterogeneous backbone foldamers has several advantages over that of their homogeneous backbone counterparts, including access to many new mol. shapes based on variations in the stoichiometries and patterns of the subunit combinations and improved prospects for side chain diversification.  Recent efforts to develop α/β-peptide foldamers can be divided into two conceptually distinct classes.  The first includes entities prepd. using a "block" strategy, in which α-peptide segments and β-peptide segments are combined to form a hybrid oligomer.  The second class encompasses designs in which α- and β-amino acid monomers are interspersed in a regular pattern throughout an oligomer sequence.  One α/β-peptide helical secondary structure, contg. C=O(i)···H-N(i+4) H-bonds analogous to those in the α-helix, has been shown via crystallog. to form helix bundle quaternary structures.  Desirable biol. functions have been elicited from α/β-peptide foldamers.  Efforts to mimic naturally occurring host-defense α-peptides have yielded new antimicrobial agents and have led to a reexamn. of the long-held views regarding structure-activity relationships among these α-peptides and their analogs.  Foldamers offer new platforms for mimicry of the mol. surfaces involved in specific protein-protein recognition events; recent achievements in the prepn. of α/β-peptide inhibitors of the protein-protein interactions involved in apoptotic signaling (e.g., between Bcl-xL and pro-apoptotic partners) have revealed the benefits of employing heterogeneous backbones relative to homogeneous backbones for foldamer-based designs.  These initial successes in the development of α/β-peptides exhibiting specific biol. activities highlight the potential of heterogeneous backbone foldamers for use in biomedical applications and provide guidelines for future studies into new target functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqT6vMz0rs3sLVg90H21EOLACvtfcHk0lg0pliey3XCUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFajtrY%253D&md5=69f53ccc98f0265b8d7d73362de37b86</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Far800009n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far800009n%26sid%3Dliteratum%253Aachs%26aulast%3DHorne%26aufirst%3DW.%2BS.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DFoldamers%2520with%2520heterogeneous%2520backbones%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1399%26epage%3D1408%26doi%3D10.1021%2Far800009n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shandler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">Foldamers as versatile frameworks for the design and evolution of function</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nchembio876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchembio876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17438550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=252-262&author=C.+M.+Goodmanauthor=S.+Choiauthor=S.+Shandlerauthor=W.+F.+DeGrado&title=Foldamers+as+versatile+frameworks+for+the+design+and+evolution+of+function&doi=10.1038%2Fnchembio876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Foldamers as versatile frameworks for the design and evolution of function</span></div><div class="casAuthors">Goodman, Catherine M.; Choi, Sungwook; Shandler, Scott; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">252-262</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Foldamers are sequence-specific oligomers akin to peptides, proteins and oligonucleotides that fold into well-defined three-dimensional structures.  They offer the chem. biologist a broad pallet of building blocks for the construction of mols. that test and extend our understanding of protein folding and function.  Foldamers also provide templates for presenting complex arrays of functional groups in virtually unlimited geometrical patterns, thereby presenting attractive opportunities for the design of mols. that bind in a sequence- and structure-specific manner to oligosaccharides, nucleic acids, membranes and proteins.  We summarize recent advances and highlight the future applications and challenges of this rapidly expanding field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkLkvabxBqULVg90H21EOLACvtfcHk0lgdccCaPPWBZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1Kjs7w%253D&md5=99f0e4b2bb7a8457a586c74f0f98dbb3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio876%26sid%3Dliteratum%253Aachs%26aulast%3DGoodman%26aufirst%3DC.%2BM.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DShandler%26aufirst%3DS.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DFoldamers%2520as%2520versatile%2520frameworks%2520for%2520the%2520design%2520and%2520evolution%2520of%2520function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D252%26epage%3D262%26doi%3D10.1038%2Fnchembio876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, S.</span>; <span class="NLM_string-name">Huc, I.</span></span> <i>Foldamers: Structure, Properties, and Applications</i>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2007</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2F9783527611478" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=S.+Hecht&author=I.+Huc&title=Foldamers%3A+Structure%2C+Properties%2C+and+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2F9783527611478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527611478%26sid%3Dliteratum%253Aachs%26aulast%3DHecht%26aufirst%3DS.%26btitle%3DFoldamers%253A%2520Structure%252C%2520Properties%252C%2520and%2520Applications%26pub%3DWiley-VCH%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Poli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawodny, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinonero, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayden, J.</span></span> <span> </span><span class="NLM_article-title">Conformational photoswitching of a synthetic peptide foldamer bound within a phospholipid bilayer</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>352</i></span>,  <span class="NLM_fpage">575</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1126/science.aad8352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1126%2Fscience.aad8352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=27033546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1KisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2016&pages=575-580&author=M.+De%0APoliauthor=W.+Zawodnyauthor=O.+Quinoneroauthor=M.+Lorchauthor=S.+J.+Webbauthor=J.+Clayden&title=Conformational+photoswitching+of+a+synthetic+peptide+foldamer+bound+within+a+phospholipid+bilayer&doi=10.1126%2Fscience.aad8352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational photoswitching of a synthetic peptide foldamer bound within a phospholipid bilayer</span></div><div class="casAuthors">De Poli, Matteo; Zawodny, Wojciech; Quinonero, Ophelie; Lorch, Mark; Webb, Simon J.; Clayden, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">352</span>
        (<span class="NLM_cas:issue">6285</span>),
    <span class="NLM_cas:pages">575-580</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The dynamic properties of foldamers, synthetic mols. that mimic folded biomols., have mainly been explored in free soln.  We report on the design, synthesis, and conformational behavior of photoresponsive foldamers bound in a phospholipid bilayer akin to a biol. membrane phase.  These mols. contain a chromophore, which can be switched between two configurations by different wavelengths of light, attached to a helical synthetic peptide that both promotes membrane insertion and communicates conformational change along its length.  Light-induced structural changes in the chromophore are translated into global conformational changes, which are detected by monitoring the solid-state 19F NMR signals of a remote fluorine-contg. residue located 1 to 2 nm away.  The behavior of the foldamers in the membrane phase is similar to that of analogous compds. in org. solvents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHTa9onjAgULVg90H21EOLACvtfcHk0lgdccCaPPWBZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1KisL0%253D&md5=9c339a72d54cfc20c2825d61bd6cbafb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad8352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad8352%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BPoli%26aufirst%3DM.%26aulast%3DZawodny%26aufirst%3DW.%26aulast%3DQuinonero%26aufirst%3DO.%26aulast%3DLorch%26aufirst%3DM.%26aulast%3DWebb%26aufirst%3DS.%2BJ.%26aulast%3DClayden%26aufirst%3DJ.%26atitle%3DConformational%2520photoswitching%2520of%2520a%2520synthetic%2520peptide%2520foldamer%2520bound%2520within%2520a%2520phospholipid%2520bilayer%26jtitle%3DScience%26date%3D2016%26volume%3D352%26spage%3D575%26epage%3D580%26doi%3D10.1126%2Fscience.aad8352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diemer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raftery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruscoe, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengel, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockroft, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayden, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, S. J.</span></span> <span> </span><span class="NLM_article-title">Length-Dependent Formation of Transmembrane Pores by 310-Helical α-Aminoisobutyric Acid Foldamers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">688</span>– <span class="NLM_lpage">695</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b12057</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b12057" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVygsb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=688-695&author=J.+E.+Jonesauthor=V.+Diemerauthor=C.+Adamauthor=J.+Rafteryauthor=R.+E.+Ruscoeauthor=J.+T.+Sengelauthor=M.+I.+Wallaceauthor=A.+Baderauthor=S.+L.+Cockroftauthor=J.+Claydenauthor=S.+J.+Webb&title=Length-Dependent+Formation+of+Transmembrane+Pores+by+310-Helical+%CE%B1-Aminoisobutyric+Acid+Foldamers&doi=10.1021%2Fjacs.5b12057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Length-Dependent Formation of Transmembrane Pores by 310-Helical α-Aminoisobutyric Acid Foldamers</span></div><div class="casAuthors">Jones, Jennifer E.; Diemer, Vincent; Adam, Catherine; Raftery, James; Ruscoe, Rebecca E.; Sengel, Jason T.; Wallace, Mark I.; Bader, Antoine; Cockroft, Scott L.; Clayden, Jonathan; Webb, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">688-695</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthetic biol. toolbox lacks extendable and conformationally controllable yet easy-to-synthesize building blocks that are long enough to span membranes.  To meet this need, an iterative synthesis of α-aminoisobutyric acid (Aib) oligomers was used to create a library of homologous rigid-rod 310-helical foldamers, which have incrementally increasing lengths and functionalizable N- and C-termini.  This library was used to probe the inter-relationship of foldamer length, self-assocn. strength, and ionophoric ability, which is poorly understood.  Although foldamer self-assocn. in nonpolar chloroform increased with length, with a ∼14-fold increase in dimerization const. from Aib6 to Aib11, ionophoric activity in bilayers showed a stronger length dependence, with the obsd. rate const. for Aib11 ∼70-fold greater than that of Aib6.  The strongest ionophoric activity was obsd. for foldamers with >10 Aib residues, which have end-to-end distances greater than the hydrophobic width of the bilayers used (∼2.8 nm); X-ray crystallog. showed that Aib11 is 2.93 nm long.  These studies suggest that being long enough to span the membrane is more important for good ionophoric activity than strong self-assocn. in the bilayer.  Planar bilayer conductance measurements showed that Aib11 and Aib13, but not Aib7, could form pores.  This pore-forming behavior is strong evidence that Aibm (m ≥ 10) building blocks can span bilayers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjKQwOG7qjGbVg90H21EOLACvtfcHk0lgawdocCvyprA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVygsb7N&md5=ce8a1028c371a2699f3207fc0f5c06cf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b12057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b12057%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DJ.%2BE.%26aulast%3DDiemer%26aufirst%3DV.%26aulast%3DAdam%26aufirst%3DC.%26aulast%3DRaftery%26aufirst%3DJ.%26aulast%3DRuscoe%26aufirst%3DR.%2BE.%26aulast%3DSengel%26aufirst%3DJ.%2BT.%26aulast%3DWallace%26aufirst%3DM.%2BI.%26aulast%3DBader%26aufirst%3DA.%26aulast%3DCockroft%26aufirst%3DS.%2BL.%26aulast%3DClayden%26aufirst%3DJ.%26aulast%3DWebb%26aufirst%3DS.%2BJ.%26atitle%3DLength-Dependent%2520Formation%2520of%2520Transmembrane%2520Pores%2520by%2520310-Helical%2520%25CE%25B1-Aminoisobutyric%2520Acid%2520Foldamers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D688%26epage%3D695%26doi%3D10.1021%2Fjacs.5b12057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collie, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulka-Ziach, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fremaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decossas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauran, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackereth, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span> <span> </span><span class="NLM_article-title">Shaping quaternary assemblies of water-soluble non-peptide helical foldamers by sequence manipulation</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">878</span>, <span class="refDoi"> DOI: 10.1038/nchem.2353</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchem.2353" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26492006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFOku7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=871-878&author=G.+W.+Collieauthor=K.+Pulka-Ziachauthor=C.+M.+Lombardoauthor=J.+Fremauxauthor=F.+Rosuauthor=M.+Decossasauthor=L.+Mauranauthor=O.+Lambertauthor=V.+Gabelicaauthor=C.+D.+Mackerethauthor=G.+Guichard&title=Shaping+quaternary+assemblies+of+water-soluble+non-peptide+helical+foldamers+by+sequence+manipulation&doi=10.1038%2Fnchem.2353"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Shaping quaternary assemblies of water-soluble non-peptide helical foldamers by sequence manipulation</span></div><div class="casAuthors">Collie, Gavin W.; Pulka-Ziach, Karolina; Lombardo, Caterina M.; Fremaux, Juliette; Rosu, Frederic; Decossas, Marion; Mauran, Laura; Lambert, Olivier; Gabelica, Valerie; Mackereth, Cameron D.; Guichard, Gilles</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">871-878</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design and construction of biomimetic self-assembling systems is a challenging yet potentially highly rewarding endeavour that contributes to the development of new biomaterials, catalysts, drug-delivery systems and tools for the manipulation of biol. processes.  Significant progress has been achieved by engineering self-assembling DNA-, protein- and peptide-based building units.  However, the design of entirely new, completely non-natural folded architectures that resemble biopolymers ('foldamers') and have the ability to self-assemble into atomically precise nanostructures in aq. conditions has proved exceptionally challenging.  Here we report the modular design, formation and structural elucidation at the at. level of a series of diverse quaternary arrangements formed by the self-assembly of short amphiphilic α-helicomimetic foldamers that bear proteinaceous side chains.  We show that the final quaternary assembly can be controlled at the sequence level, which permits the programmed formation of either discrete helical bundles that contain isolated cavities or pH-responsive water-filled channels with controllable pore diams.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO9U5QH5AT8LVg90H21EOLACvtfcHk0lgawdocCvyprA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFOku7fI&md5=4ba4da0dc49636de6a566fd9e2f63920</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2353&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2353%26sid%3Dliteratum%253Aachs%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DPulka-Ziach%26aufirst%3DK.%26aulast%3DLombardo%26aufirst%3DC.%2BM.%26aulast%3DFremaux%26aufirst%3DJ.%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DDecossas%26aufirst%3DM.%26aulast%3DMauran%26aufirst%3DL.%26aulast%3DLambert%26aufirst%3DO.%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DMackereth%26aufirst%3DC.%2BD.%26aulast%3DGuichard%26aufirst%3DG.%26atitle%3DShaping%2520quaternary%2520assemblies%2520of%2520water-soluble%2520non-peptide%2520helical%2520foldamers%2520by%2520sequence%2520manipulation%26jtitle%3DNat.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D871%26epage%3D878%26doi%3D10.1038%2Fnchem.2353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span> <span> </span><span class="NLM_article-title">Magnetotactic molecular architectures from self-assembly of β-peptide foldamers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">8747</span>– <span class="NLM_lpage">8753</span>, <span class="refDoi"> DOI: 10.1038/ncomms9747</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fncomms9747" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26510658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCmsrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=8747-8753&author=S.+Kwonauthor=B.+J.+Kimauthor=H.-K.+Limauthor=K.+Kangauthor=S.+H.+Yooauthor=J.+Gongauthor=E.+Yoonauthor=J.+Leeauthor=I.+S.+Choiauthor=H.+Kimauthor=H.-S.+Lee&title=Magnetotactic+molecular+architectures+from+self-assembly+of+%CE%B2-peptide+foldamers&doi=10.1038%2Fncomms9747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Magnetotactic molecular architectures from self-assembly of β-peptide foldamers</span></div><div class="casAuthors">Kwon, Sunbum; Kim, Beom Jin; Lim, Hyung-Kyu; Kang, Kyungtae; Yoo, Sung Hyun; Gong, Jintaek; Yoon, Eunyoung; Lee, Juno; Choi, Insung S.; Kim, Hyungjun; Lee, Hee-Seung</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8747</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of stimuli-responsive self-assembled mol. systems capable of undergoing mech. work is one of the most important challenges in synthetic chem. and materials science.  Here we report that foldectures, i.e., self-assembled mol. architectures of β-peptide foldamers, uniformly align with respect to an applied static magnetic field, and also show instantaneous orientational motion in a dynamic magnetic field.  This response is explained by the amplified anisotropy of the diamagnetic susceptibilities as a result of the well-ordered mol. packing of the foldectures.  In addn., the motions of foldectures at the microscale can be translated into magnetotactic behavior at the macroscopic scale in a way reminiscent to that of magnetosomes in magnetotactic bacteria.  This study will provide significant inspiration for designing the next generation of biocompatible peptide-based mol. machines with applications in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreIyfWwJdjJLVg90H21EOLACvtfcHk0lgJm9xrlpwzQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCmsrrK&md5=0829ea55bbd5fd090897891693fce789</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fncomms9747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9747%26sid%3Dliteratum%253Aachs%26aulast%3DKwon%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DB.%2BJ.%26aulast%3DLim%26aufirst%3DH.-K.%26aulast%3DKang%26aufirst%3DK.%26aulast%3DYoo%26aufirst%3DS.%2BH.%26aulast%3DGong%26aufirst%3DJ.%26aulast%3DYoon%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DI.%2BS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DH.-S.%26atitle%3DMagnetotactic%2520molecular%2520architectures%2520from%2520self-assembly%2520of%2520%25CE%25B2-peptide%2520foldamers%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D8747%26epage%3D8753%26doi%3D10.1038%2Fncomms9747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheloha, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardella, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Backbone modification of a polypeptide drug alters duration of action in vivo</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1038/nbt.2920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnbt.2920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=24929976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=653-655&author=R.+W.+Chelohaauthor=A.+Maedaauthor=T.+Deanauthor=T.+J.+Gardellaauthor=S.+H.+Gellman&title=Backbone+modification+of+a+polypeptide+drug+alters+duration+of+action+in+vivo&doi=10.1038%2Fnbt.2920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone modification of a polypeptide drug alters duration of action in vivo</span></div><div class="casAuthors">Cheloha, Ross W.; Maeda, Akira; Dean, Thomas; Gardella, Thomas J.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">653-655</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Systematic modification of the backbone of bioactive polypeptides through β-amino acid residue incorporation could provide a strategy for generating mols. with improved drug properties, but such alterations can result in lower receptor affinity and potency.  Using an agonist of parathyroid hormone receptor-1 (PTHR1), a G protein-coupled receptor in the B-family, we present an approach for α→β residue replacement that enables both high activity and improved pharmacokinetic properties in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8cWxIA-3shbVg90H21EOLACvtfcHk0lgJm9xrlpwzQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D&md5=100382f0d7e67fe9c11fd39886e7acac</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2920%26sid%3Dliteratum%253Aachs%26aulast%3DCheloha%26aufirst%3DR.%2BW.%26aulast%3DMaeda%26aufirst%3DA.%26aulast%3DDean%26aufirst%3DT.%26aulast%3DGardella%26aufirst%3DT.%2BJ.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DBackbone%2520modification%2520of%2520a%2520polypeptide%2520drug%2520alters%2520duration%2520of%2520action%2520in%2520vivo%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D653%26epage%3D655%26doi%3D10.1038%2Fnbt.2920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buratto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colombo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stupfel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolain, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langlois d’Estaintot, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laguerre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallois, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span> <span> </span><span class="NLM_article-title">Structure of a Complex Formed by a Protein and a Helical Aromatic Oligoamide Foldamer at 2.1 Å Resolution</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">883</span>– <span class="NLM_lpage">887</span>, <span class="refDoi"> DOI: 10.1002/anie.201309160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201309160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVOhsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=883-887&author=J.+Burattoauthor=C.+Colomboauthor=M.+Stupfelauthor=S.+J.+Dawsonauthor=C.+Dolainauthor=B.+Langlois+d%E2%80%99Estaintotauthor=L.+Fischerauthor=T.+Granierauthor=M.+Laguerreauthor=B.+Galloisauthor=I.+Huc&title=Structure+of+a+Complex+Formed+by+a+Protein+and+a+Helical+Aromatic+Oligoamide+Foldamer+at+2.1+%C3%85+Resolution&doi=10.1002%2Fanie.201309160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a Complex Formed by a Protein and a Helical Aromatic Oligoamide Foldamer at 2.1 Å Resolution</span></div><div class="casAuthors">Buratto, Jeremie; Colombo, Cinzia; Stupfel, Marine; Dawson, Simon J.; Dolain, Christel; Langlois d'Estaintot, Beatrice; Fischer, Lucile; Granier, Thierry; Laguerre, Michel; Gallois, Bernard; Huc, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">883-887</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In the search for mols. that could recognize sizeable areas of protein surfaces, a series of ten helical arom. oligoamide foldamers was synthesized on solid phase.  The foldamers comprise three to five monomers carrying various proteinogenic side chains, and exist as racemic mixts. of interconverting right-handed and left-handed helixes.  Functionalization of the foldamers by a nanomolar ligand of human carbonic anhydrase II (HCA) ensured that they would be held in close proximity to the protein surface.  Foldamer-protein interactions were screened by CD.  One foldamer displayed intense CD bands, indicating that a preferred helix handedness is induced upon interacting with the protein surface.  The crystal structure of the complex between this foldamer and HCA could be resolved at 2.1 Å resoln. and revealed a no. of unanticipated protein-foldamer, foldamer-foldamer, and protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV3mXfBqVLRrVg90H21EOLACvtfcHk0lgJm9xrlpwzQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVOhsbbM&md5=8262f6b947521b3f5c228910df27342b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fanie.201309160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201309160%26sid%3Dliteratum%253Aachs%26aulast%3DBuratto%26aufirst%3DJ.%26aulast%3DColombo%26aufirst%3DC.%26aulast%3DStupfel%26aufirst%3DM.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DDolain%26aufirst%3DC.%26aulast%3DLanglois%2Bd%25E2%2580%2599Estaintot%26aufirst%3DB.%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DGranier%26aufirst%3DT.%26aulast%3DLaguerre%26aufirst%3DM.%26aulast%3DGallois%26aufirst%3DB.%26aulast%3DHuc%26aufirst%3DI.%26atitle%3DStructure%2520of%2520a%2520Complex%2520Formed%2520by%2520a%2520Protein%2520and%2520a%2520Helical%2520Aromatic%2520Oligoamide%2520Foldamer%2520at%25202.1%2520%25C3%2585%2520Resolution%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D883%26epage%3D887%26doi%3D10.1002%2Fanie.201309160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilvert, D.</span></span> <span> </span><span class="NLM_article-title">Building proficient enzymes with foldamer prostheses</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6978</span>– <span class="NLM_lpage">6981</span>, <span class="refDoi"> DOI: 10.1002/anie.201400945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201400945" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVyls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6978-6981&author=C.+Mayerauthor=M.+M.+M%C3%BCllerauthor=S.+H.+Gellmanauthor=D.+Hilvert&title=Building+proficient+enzymes+with+foldamer+prostheses&doi=10.1002%2Fanie.201400945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Building Proficient Enzymes with Foldamer Prostheses</span></div><div class="casAuthors">Mayer, Clemens; Mueller, Manuel M.; Gellman, Samuel H.; Hilvert, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">6978-6981</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Foldamers are non-natural oligomers that adopt stable conformations reminiscent of those found in proteins.  To evaluate the potential of foldameric subunits for catalysis, semisynthetic enzymes contg. foldamer fragments constructed from α- and β-amino acid residues were designed and characterized.  Systematic variation of the α→β substitution pattern and types of β-residue afforded highly proficient hybrid catalysts, thus demonstrating the feasibility of expanding the enzyme-engineering toolkit with non-natural backbones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBv0OoJCNCzbVg90H21EOLACvtfcHk0lgRLT-Vmt-fiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVyls7o%253D&md5=680180c6c0ad89a5c4d4cccc2f09bd0e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fanie.201400945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201400945%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DC.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%2BM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DHilvert%26aufirst%3DD.%26atitle%3DBuilding%2520proficient%2520enzymes%2520with%2520foldamer%2520prostheses%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6978%26epage%3D6981%26doi%3D10.1002%2Fanie.201400945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P. S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">6810</span>– <span class="NLM_lpage">6813</span>, <span class="refDoi"> DOI: 10.1021/ja5013849</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja5013849" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=6810-6813&author=P.+S.+P.+Wangauthor=J.+B.+Nguyenauthor=A.+Schepartz&title=Design+and+High-Resolution+Structure+of+a+%CE%B23-Peptide+Bundle+Catalyst&doi=10.1021%2Fja5013849"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Design and High-Resolution Structure of a β3-Peptide Bundle Catalyst</span></div><div class="casAuthors">Wang, Pam S. P.; Nguyen, Jennifer B.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6810-6813</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the widespread exploration of α-peptides as catalysts, there are few examples of β-peptides that alter a chem. transformation.  Previous work demonstrated that a special class of β3-peptides spontaneously self-assembles in water into discrete protein-like bundles possessing unique quaternary structures and exceptional thermodn. stability.  Here the authors describe β3-peptide bundles capable of both substrate binding and chem. catalysis-ester hydrolysis.  A combination of kinetic and high-resoln. structural anal. suggests an active site triad composed of residues from at least two strands of the octameric bundle structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJGELSSHUnErVg90H21EOLACvtfcHk0lgRLT-Vmt-fiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1Ciu74%253D&md5=7489eb9f9626d5e958289732a577455b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja5013849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja5013849%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DP.%2BS.%2BP.%26aulast%3DNguyen%26aufirst%3DJ.%2BB.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DDesign%2520and%2520High-Resolution%2520Structure%2520of%2520a%2520%25CE%25B23-Peptide%2520Bundle%2520Catalyst%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D6810%26epage%3D6813%26doi%3D10.1021%2Fja5013849" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolffs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delsuc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veldman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anh, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meskers, S. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, R. A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenning, A. P. H. J.</span></span> <span> </span><span class="NLM_article-title">Helical aromatic oligoamide foldamers as organizational scaffolds for photoinduced charge transfer</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">4819</span>– <span class="NLM_lpage">4829</span>, <span class="refDoi"> DOI: 10.1021/ja809367u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja809367u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtFyqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=4819-4829&author=M.+Wolffsauthor=N.+Delsucauthor=D.+Veldmanauthor=N.+V.+Anhauthor=R.+M.+Williamsauthor=S.+C.+J.+Meskersauthor=R.+A.+J.+Janssenauthor=I.+Hucauthor=A.+P.+H.+J.+Schenning&title=Helical+aromatic+oligoamide+foldamers+as+organizational+scaffolds+for+photoinduced+charge+transfer&doi=10.1021%2Fja809367u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Aromatic Oligoamide Foldamers as Organizational Scaffolds for Photoinduced Charge Transfer</span></div><div class="casAuthors">Wolffs, Martin; Delsuc, Nicolas; Veldman, Dirk; Nguyexn, Van Anh; Williams, Rene M.; Meskers, Stefan C. J.; Janssen, Rene A. J.; Huc, Ivan; Schenning, Albertus P. H. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4819-4829</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we report the synthesis and characterization of four quinoline-derived foldamers with increasing oligomeric length; dimer O2P, tetramer O4P, pentamer O5P, and nonamer O9P functionalized with on one end an oligo(p-phenylene vinylene) (OPV) and on the other end a perylene bisimide (PB) chromophore. 1H NMR confirms the formation of the expected folded structures in both toluene and chloroform soln.  The structural predictability and rigidity of the oligomeric series enabled us to investigate the effect of a helical bridge and chromophore position on the photoinduced processes in the electron OPV-PB donor-acceptor pair in chloroform and toluene.  The helical properties of the bridge ensured that the chromophore sepn. distance through space is different from the sepn. distance through the bridge.  For all foldamer-solvent combinations studied, excitation of either OPV or PB results in nearly quant. quenching of the fluorescence indicating a fast charge sepn. reaction between the OPV and PB.  Femtosecond photoinduced absorption measurements confirmed the fast formation of a charge-sepd. state.  The recombination reaction involves a combination of direct decay to the ground state and the formation of an intermediate triplet state, with their balance depending on the foldamer-solvent combination.  MO calcns. rationalize the fast photoinduced charge sepn., by revealing that the bridging foldamer mediates the charge transfer from donor to acceptor via the superexchange mechanism.  Remarkably low attenuation factors (βCS ≈ 10-2 Å-1) were obtained using either through space or through bridge sepn. distance.  However, in these calcns. only three of the four foldamers show the expected linear behavior between the logarithm of the charge sepn. rate const. and the distance between the chromophores.  The combined results show when a helical bridge is sepg. the charge transfer couple, hampering the usefulness of a uniform description of the charge-sepn. phenomena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBIZYeMF7LcrVg90H21EOLACvtfcHk0lgRLT-Vmt-fiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtFyqsrs%253D&md5=9b7f38deceff442df983090afc1f27ce</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fja809367u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja809367u%26sid%3Dliteratum%253Aachs%26aulast%3DWolffs%26aufirst%3DM.%26aulast%3DDelsuc%26aufirst%3DN.%26aulast%3DVeldman%26aufirst%3DD.%26aulast%3DAnh%26aufirst%3DN.%2BV.%26aulast%3DWilliams%26aufirst%3DR.%2BM.%26aulast%3DMeskers%26aufirst%3DS.%2BC.%2BJ.%26aulast%3DJanssen%26aufirst%3DR.%2BA.%2BJ.%26aulast%3DHuc%26aufirst%3DI.%26aulast%3DSchenning%26aufirst%3DA.%2BP.%2BH.%2BJ.%26atitle%3DHelical%2520aromatic%2520oligoamide%2520foldamers%2520as%2520organizational%2520scaffolds%2520for%2520photoinduced%2520charge%2520transfer%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D4819%26epage%3D4829%26doi%3D10.1021%2Fja809367u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamuro, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">De Novo Design of Antibacterial β-Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">12200</span>, <span class="refDoi"> DOI: 10.1021/ja992728p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja992728p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK1MXnvVClsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=12200&author=Y.+Hamuroauthor=J.+P.+Schneiderauthor=W.+F.+DeGrado&title=De+Novo+Design+of+Antibacterial+%CE%B2-Peptides&doi=10.1021%2Fja992728p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">De Novo design of antibacterial β-peptides</span></div><div class="casAuthors">Hamuro, Yoshi; Schneider, Joel P.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">12200-12201</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of β-peptides that mimic the activities of a class of natural membrane-active peptide toxins and antibiotics, including magainins, bombolitin, cecropins, melittin and mastoparans, is described.  A series of peptides of graded lengths was synthesized and their ability to adopt an L+2 helix in aq. soln. in the presence and absence of micelles and phospholipid bilayers was assessed.  The effect of a hydrophobic Fmoc group on binding to micelles was examd.  Finally, the biol. activities of these peptides were measured using human erythrocytes and Escherichia coli as models for mammalian and bacterial cells, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvqOIr8MCcG7Vg90H21EOLACvtfcHk0lhMxmNGK4NzvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvVClsrg%253D&md5=f6d16b8310aa08bb43fd23c1941b1fb6</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fja992728p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja992728p%26sid%3Dliteratum%253Aachs%26aulast%3DHamuro%26aufirst%3DY.%26aulast%3DSchneider%26aufirst%3DJ.%2BP.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DDe%2520Novo%2520Design%2520of%2520Antibacterial%2520%25CE%25B2-Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D12200%26doi%3D10.1021%2Fja992728p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Chimeric (α/β + α)-Peptide Ligands for the BH3-Recognition Cleft of Bcl-xL: Critical Role of the Molecular Scaffold in Protein Surface Recognition</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">11966</span>– <span class="NLM_lpage">11968</span>, <span class="refDoi"> DOI: 10.1021/ja053678t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja053678t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntVert7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=11966-11968&author=J.+D.+Sadowskyauthor=M.+A.+Schmittauthor=H.-S.+Leeauthor=N.+Umezawaauthor=S.+Wangauthor=Y.+Tomitaauthor=S.+H.+Gellman&title=Chimeric+%28%CE%B1%2F%CE%B2+%2B+%CE%B1%29-Peptide+Ligands+for+the+BH3-Recognition+Cleft+of+Bcl-xL%3A+Critical+Role+of+the+Molecular+Scaffold+in+Protein+Surface+Recognition&doi=10.1021%2Fja053678t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Chimeric (α/β + α)-Peptide Ligands for the BH3-Recognition Cleft of Bcl-xL: Critical Role of the Molecular Scaffold in Protein Surface Recognition</span></div><div class="casAuthors">Sadowsky, Jack D.; Schmitt, Margaret A.; Lee, Hee-Seung; Umezawa, Naoki; Wang, Shaomeng; Tomita, York; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">11966-11968</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mols. that bind to sp. surface sites on proteins are of great interest from both fundamental and practical perspectives.  We are exploring a ligand development strategy that is based on oligomers with discrete folding propensities ("foldamers"); we target a specific cleft on the cancer-assocd. protein Bcl-xL because this system is well characterized structurally.  In vivo, this cleft binds to α-helical segments (BH3 domains) of other proteins.  We evaluated several types of helical foldamer, built entirely from β-amino acid residues or from mixts. of α- and β-amino acid residues, and ultimately identified foldamers in the latter class that bind very tightly to Bcl-xL.  Our results suggest that combining different types of foldamer backbones will be an effective and general strategy for creating high-affinity and specific ligands for protein surface sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKnEnRm8PJbrVg90H21EOLACvtfcHk0lhMxmNGK4NzvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntVert7g%253D&md5=a1b03551dbe60effd954fa8ff0aa400d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja053678t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja053678t%26sid%3Dliteratum%253Aachs%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DSchmitt%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DChimeric%2520%2528%25CE%25B1%252F%25CE%25B2%2520%252B%2520%25CE%25B1%2529-Peptide%2520Ligands%2520for%2520the%2520BH3-Recognition%2520Cleft%2520of%2520Bcl-xL%253A%2520Critical%2520Role%2520of%2520the%2520Molecular%2520Scaffold%2520in%2520Protein%2520Surface%2520Recognition%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D11966%26epage%3D11968%26doi%3D10.1021%2Fja053678t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lear, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodsdon, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">9468</span>– <span class="NLM_lpage">9469</span>, <span class="refDoi"> DOI: 10.1021/ja031625a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja031625a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=9468-9469&author=J.+A.+Kritzerauthor=J.+D.+Learauthor=M.+E.+Hodsdonauthor=A.+Schepartz&title=Helical+%CE%B2-Peptide+Inhibitors+of+the+p53-hDM2+Interaction&doi=10.1021%2Fja031625a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Helical β-Peptide Inhibitors of the p53-hDM2 Interaction</span></div><div class="casAuthors">Kritzer, Joshua A.; Lear, James D.; Hodsdon, Michael E.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9468-9469</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HDM2 is recognized in vivo by a short α-helix within the p53 trans-activation domain (p53AD).  Disruption of the p53·hDM2 interaction is an important goal for cancer therapy.  A functional epitope comprised of three residues on one face of the p53AD helix (F19, W23, and L26) contributes heavily to the binding free energy.  We hypothesized that the p53AD functional epitope would be recapitulated if the side chains of F19, W23, and L26 were presented at successive positions three residues apart on a stabilized β3-peptide 14-helix.  Here, we report a set of β3-peptides that possess significant 14-helix structure in water; one recognizes a cleft on the surface of hDM2 with nanomolar affinity.  The strategy for β3-peptide design that we describe is general and may have advantages over one in which individual or multiple β-amino acid substitutions are introduced into a functional α-peptide, because it is based on homol. at the level of secondary structure, not primary sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVlZAGh6JntbVg90H21EOLACvtfcHk0lhs5wIGzICdrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlvVSksLo%253D&md5=3fc9e87497a50588ee4591d5f84927ea</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja031625a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja031625a%26sid%3Dliteratum%253Aachs%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26aulast%3DLear%26aufirst%3DJ.%2BD.%26aulast%3DHodsdon%26aufirst%3DM.%2BE.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DHelical%2520%25CE%25B2-Peptide%2520Inhibitors%2520of%2520the%2520p53-hDM2%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D9468%26epage%3D9469%26doi%3D10.1021%2Fja031625a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gademann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebach, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Biological Evaluation of a Cyclo--tetrapeptide as a Somatostatin Analogue</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1223</span>– <span class="NLM_lpage">1226</span>, <span class="refDoi"> DOI: 10.1002/(sici)1521-3773(19990503)38:9<1223::aid-anie1223>3.0.co;2-a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK1MXjtFOrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1223-1226&author=K.+Gademannauthor=M.+Ernstauthor=D.+Hoyerauthor=D.+Seebach&title=Synthesis+and+Biological+Evaluation+of+a+Cyclo%2D%2Dtetrapeptide+as+a+Somatostatin+Analogue&doi=10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of a cyclo-β-tetrapeptide as a somatostatin analog</span></div><div class="casAuthors">Gademann, Karl; Ernst, Martin; Hoyer, Daniel; Seebach, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1223-1226</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Cyclo-β-tetrapeptide I, a β-peptidic analog of octreotide, was prepd. from β-amino acid building blocks synthesized by Arndt-Eistert homologation of appropriately protected α-amino acids.  The affinity of I for five different human somatostatin receptors was measured using radioligand-binding assays.  I had affinity, although the concns. were in the micromolar and not in the nanomolar range as for somatostatin and octreotide.  The authors have demonstrated that a small β-peptide (consisting of only four β-amino acids) can mimic a natural peptide hormone and display biol. activity and micromolar affinity for human receptors.  Thus, β-peptides can, in principle, be considered as peptidomimetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovReDe9QljFLVg90H21EOLACvtfcHk0lhs5wIGzICdrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtFOrsLw%253D&md5=4ee1d15ac384ed8730b85b36f8daaf71</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528sici%25291521-3773%252819990503%252938%253A9%253C1223%253A%253Aaid-anie1223%253E3.0.co%253B2-a%26sid%3Dliteratum%253Aachs%26aulast%3DGademann%26aufirst%3DK.%26aulast%3DErnst%26aufirst%3DM.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DSeebach%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520Biological%2520Evaluation%2520of%2520a%2520Cyclo--tetrapeptide%2520as%2520a%2520Somatostatin%2520Analogue%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1223%26epage%3D1226%26doi%3D10.1002%2F%28sici%291521-3773%2819990503%2938%3A9%3C1223%3A%3Aaid-anie1223%3E3.0.co%3B2-a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Werder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abele, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seebach, D.</span></span> <span> </span><span class="NLM_article-title">β-Peptides as Inhibitors of Small-Intestinal Cholesterol and Fat Absorption</span>. <i>Helv. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1774</span>– <span class="NLM_lpage">1783</span>, <span class="refDoi"> DOI: 10.1002/(sici)1522-2675(19991006)82:10<1774::aid-hlca1774>3.0.co;2-o</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK1MXmvFensbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1999&pages=1774-1783&author=M.+Werderauthor=H.+Hauserauthor=S.+Abeleauthor=D.+Seebach&title=%CE%B2-Peptides+as+Inhibitors+of+Small-Intestinal+Cholesterol+and+Fat+Absorption&doi=10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides as inhibitors of small-intestinal cholesterol and fat absorption</span></div><div class="casAuthors">Werder, Moritz; Hauser, Helmut; Abele, Stefan; Seebach, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">Helvetica Chimica Acta</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1774-1783</span>CODEN:
                <span class="NLM_cas:coden">HCACAV</span>;
        ISSN:<span class="NLM_cas:issn">0018-019X</span>.
    
            (<span class="NLM_cas:orgname">Verlag Helvetica Chimica Acta</span>)
        </div><div class="casAbstract">Selective lipid transport through the brush-border membrane in the small intestine of mammals is mediated by membrane-bound proteins, the so-called scavenger receptors of class B, type I or II (SR-BI or -BII).  These, in turn, are inhibited by certain proteins and synthetic α-peptides that have an amphipathic helix as the binding motif.  In whole cells (test with human colonic carcinoma cells, CaCo-2), on the other hand, the inhibitors are subject to proteolysis.  We have now tested six β-peptides (hexa-, hepta-, and nonamers) each carrying one to seven water-solubilizing side chains of either Ser or Lys, with a brush-border-membrane (BBM) vesicle model system and with a tightly packed monolayer of CaCo-2 cells to find if the rate of transport of cholesterol can be reduced to what may be considered the passive diffusion ("background") level.  There is a correlation between the ability of the β-peptides to form an amphipathic-type 314-helical secondary structure in MeOH and their inhibitory effect.  Although the inhibitory activity of the β-peptides is in only the mM range, it is to be compared with no activity at all of previously tested α-peptides and proteins (built of L-amino acids) in CaCo-2 cells.  Furthermore, these active β-peptides contain only seven or nine residues and must be considered simple, first-generation models capable of mimicking the biol. activity of amphipathic α-peptide helixes in living whole cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjsCK3hVeJ_rVg90H21EOLACvtfcHk0lhs5wIGzICdrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvFensbc%253D&md5=5e553455f52762ca9f32b78a649036fd</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528sici%25291522-2675%252819991006%252982%253A10%253C1774%253A%253Aaid-hlca1774%253E3.0.co%253B2-o%26sid%3Dliteratum%253Aachs%26aulast%3DWerder%26aufirst%3DM.%26aulast%3DHauser%26aufirst%3DH.%26aulast%3DAbele%26aufirst%3DS.%26aulast%3DSeebach%26aufirst%3DD.%26atitle%3D%25CE%25B2-Peptides%2520as%2520Inhibitors%2520of%2520Small-Intestinal%2520Cholesterol%2520and%2520Fat%2520Absorption%26jtitle%3DHelv.%2520Chim.%2520Acta%26date%3D1999%26volume%3D82%26spage%3D1774%26epage%3D1783%26doi%3D10.1002%2F%28sici%291522-2675%2819991006%2982%3A10%3C1774%3A%3Aaid-hlca1774%3E3.0.co%3B2-o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisblum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Non-haemolytic β-amino-acid oligomers</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>404</i></span>,  <span class="NLM_fpage">565</span>, <span class="refDoi"> DOI: 10.1038/35007145</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2F35007145" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10766230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD3cXis1Gmtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2000&pages=565&author=E.+A.+Porterauthor=X.+Wangauthor=H.-S.+Leeauthor=B.+Weisblumauthor=S.+H.+Gellman&title=Non-haemolytic+%CE%B2-amino-acid+oligomers&doi=10.1038%2F35007145"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics: Non-hemolytic β-amino-acid oligomers</span></div><div class="casAuthors">Porter, Emile A.; Wang, Xifang; Lee, Hee-Seung; Weisblum, Bernard; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">404</span>
        (<span class="NLM_cas:issue">6778</span>),
    <span class="NLM_cas:pages">565</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The antibiotic activity of two β-amino acid oligomers was shown to be similar against 4 species of bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTsBnhfOd_b7Vg90H21EOLACvtfcHk0lgUw9oBaOJsGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXis1Gmtrs%253D&md5=30618cc4dd5ebabdf9f9eb94f39fcfb5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2F35007145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F35007145%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DE.%2BA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DWeisblum%26aufirst%3DB.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DNon-haemolytic%2520%25CE%25B2-amino-acid%2520oligomers%26jtitle%3DNature%26date%3D2000%26volume%3D404%26spage%3D565%26doi%3D10.1038%2F35007145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seurynck, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barron, A. E.</span></span> <span> </span><span class="NLM_article-title">Simple, Helical Peptoid Analogs of Lung Surfactant Protein B</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2004.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1016%2Fj.chembiol.2004.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=15664517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmtlKmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=77-88&author=S.+L.+Seurynckauthor=J.+A.+Patchauthor=A.+E.+Barron&title=Simple%2C+Helical+Peptoid+Analogs+of+Lung+Surfactant+Protein+B&doi=10.1016%2Fj.chembiol.2004.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Simple, Helical Peptoid Analogs of Lung Surfactant Protein B</span></div><div class="casAuthors">Seurynck, Shannon L.; Patch, James A.; Barron, Annelise E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-88</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The helical, amphipathic surfactant protein, SP-B, is a crit. element of pulmonary surfactant and hence is an important therapeutic mol.  However, it is difficult to isolate from natural sources in high purity.  We have created and studied three different, nonnatural analogs of a bioactive SP-B fragment (SP-B1-25), using oligo-N-substituted glycines (peptoids) with simple, repetitive sequences designed to favor the formation of amphiphilic helixes.  For comparison, a peptide with a similar repetitive sequence previously shown to be a good SP mimic was also studied, along with SP-B1-25 itself.  Surface pressure-area isotherms, surfactant film phase morphol., and dynamic adsorption behavior all indicate that the peptoids are promising mimics of SP-B1-25.  The extent of biomimicry appears to correlate with peptoid helicity and lipophilicity.  These biostable oligomers could serve in a synthetic surfactant replacement to treat respiratory distress syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqazsFnExRGh7Vg90H21EOLACvtfcHk0lgUw9oBaOJsGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmtlKmtg%253D%253D&md5=8e2e1e3b706187c27231c4a0dd5f9c63</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DSeurynck%26aufirst%3DS.%2BL.%26aulast%3DPatch%26aufirst%3DJ.%2BA.%26aulast%3DBarron%26aufirst%3DA.%2BE.%26atitle%3DSimple%252C%2520Helical%2520Peptoid%2520Analogs%2520of%2520Lung%2520Surfactant%2520Protein%2520B%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D77%26epage%3D88%26doi%3D10.1016%2Fj.chembiol.2004.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patch, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barron, A. E.</span></span> <span> </span><span class="NLM_article-title">Helical peptoid mimics of magainin-2 amide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">12092</span>– <span class="NLM_lpage">12093</span>, <span class="refDoi"> DOI: 10.1021/ja037320d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja037320d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Oktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=12092-12093&author=J.+A.+Patchauthor=A.+E.+Barron&title=Helical+peptoid+mimics+of+magainin-2+amide&doi=10.1021%2Fja037320d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Peptoid Mimics of Magainin-2 Amide</span></div><div class="casAuthors">Patch, James A.; Barron, Annelise E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">12092-12093</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of peptoid oligomers were designed as helical, cationic, and facially amphipathic mimics of the magainin-2 amide antibacterial peptide.  We used CD spectroscopy to det. the conformation of these peptoids in aq. buffer and in the presence of bacterial membrane-mimetic lipid vesicles, composed of a 7:3 mol ratio of POPE:POPG.  We found that certain peptoids, which displayed characteristically helical CD in buffer and lipid vesicles, exhibit selective (nonhemolytic) and potent antibacterial activity against both Gram-pos. and Gram-neg. bacteria.  In contrast, peptoids that exhibit weak CD, reminiscent of that of a peptide random coil, were ineffective antibiotics.  In a manner similar to the natural magainin peptides, we find a correlation between peptoid lipophilicity and hemolytic propensity.  We observe that a min. length of ∼12 peptoid residues may be required for antibacterial activity.  We also see evidence that a helix length between 24 and 34 Å may provide optimal antibacterial efficacy.  These results provide the first example of a water-sol., structured, bioactive peptoid.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMVi-2AGxjtLVg90H21EOLACvtfcHk0lgUw9oBaOJsGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Oktr0%253D&md5=05bf4917c48b1f2db52e3be689891c6e</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja037320d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja037320d%26sid%3Dliteratum%253Aachs%26aulast%3DPatch%26aufirst%3DJ.%2BA.%26aulast%3DBarron%26aufirst%3DA.%2BE.%26atitle%3DHelical%2520peptoid%2520mimics%2520of%2520magainin-2%2520amide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D12092%26epage%3D12093%26doi%3D10.1021%2Fja037320d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pophristic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemparala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">The Design and Evaluation of Heparin-Binding Foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6847</span>, <span class="refDoi"> DOI: 10.1002/anie.200501279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200501279" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2005&pages=6843-6847&author=S.+Choiauthor=D.+J.+Clementsauthor=V.+Pophristicauthor=I.+Ivanovauthor=S.+Vemparalaauthor=J.+S.+Bennettauthor=M.+L.+Kleinauthor=J.+D.+Winklerauthor=W.+F.+DeGrado&title=The+Design+and+Evaluation+of+Heparin-Binding+Foldamers&doi=10.1002%2Fanie.200501279"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fanie.200501279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200501279%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DClements%26aufirst%3DD.%2BJ.%26aulast%3DPophristic%26aufirst%3DV.%26aulast%3DIvanov%26aufirst%3DI.%26aulast%3DVemparala%26aufirst%3DS.%26aulast%3DBennett%26aufirst%3DJ.%2BS.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DThe%2520Design%2520and%2520Evaluation%2520of%2520Heparin-Binding%2520Foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D117%26spage%3D6843%26epage%3D6847%26doi%3D10.1002%2Fanie.200501279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilsl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wéber, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mándity, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fülöp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiser, O.</span></span> <span> </span><span class="NLM_article-title">Unique α,β- and α,α,β,β-Peptide Foldamers Based on cis-β-Aminocyclopentanecarboxylic Acid</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2208</span>– <span class="NLM_lpage">2212</span>, <span class="refDoi"> DOI: 10.1002/anie.201107702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201107702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=2208-2212&author=%C5%81.+Berlickiauthor=L.+Pilslauthor=E.+W%C3%A9berauthor=I.+M.+M%C3%A1ndityauthor=C.+Cabreleauthor=T.+A.+Martinekauthor=F.+F%C3%BCl%C3%B6pauthor=O.+Reiser&title=Unique+%CE%B1%2C%CE%B2-+and+%CE%B1%2C%CE%B1%2C%CE%B2%2C%CE%B2-Peptide+Foldamers+Based+on+cis-%CE%B2-Aminocyclopentanecarboxylic+Acid&doi=10.1002%2Fanie.201107702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Unique α,β- and α,α,β,β-peptide foldamers based on cis-β-aminocyclopentanecarboxylic acid</span></div><div class="casAuthors">Berlicki, Lukasz; Pilsl, Ludwig; Weber, Edit; Mandity, Istvan M.; Cabrele, Chiara; Martinek, Tamas A.; Fueloep, Ferenc; Reiser, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2208-2212, S2208/1-S2208/29</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Non-natural oligomers with well-defined three-dimensional structures (foldamers) raise great interest owing to their ability to mimic and expand the world of biopolymer.  In this paper we present helical peptide foldamers with defined spatial distributions of their functional groups in polar solvents including water.  Stable helical structures based on α-amino and cis-β-aminocyclopentanecarboxylic acids have been design following the stereochem. pattering approach.  All the peptides were successfully obtained by solid phase peptide synthesis and catalytic hydrogenation.  CD, NMR and theor. calcns. were employed for detn. of foldamers. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptO-sm-FJwi7Vg90H21EOLACvtfcHk0lh1HOyWvWRE8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymtL0%253D&md5=7a96f663f67145f32e5519393dd6fa4d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107702%26sid%3Dliteratum%253Aachs%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26aulast%3DPilsl%26aufirst%3DL.%26aulast%3DW%25C3%25A9ber%26aufirst%3DE.%26aulast%3DM%25C3%25A1ndity%26aufirst%3DI.%2BM.%26aulast%3DCabrele%26aufirst%3DC.%26aulast%3DMartinek%26aufirst%3DT.%2BA.%26aulast%3DF%25C3%25BCl%25C3%25B6p%26aufirst%3DF.%26aulast%3DReiser%26aufirst%3DO.%26atitle%3DUnique%2520%25CE%25B1%252C%25CE%25B2-%2520and%2520%25CE%25B1%252C%25CE%25B1%252C%25CE%25B2%252C%25CE%25B2-Peptide%2520Foldamers%2520Based%2520on%2520cis-%25CE%25B2-Aminocyclopentanecarboxylic%2520Acid%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D2208%26epage%3D2212%26doi%3D10.1002%2Fanie.201107702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">β-Peptides: From Structure to Function</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3232</span>, <span class="refDoi"> DOI: 10.1021/cr000045i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000045i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=3219-3232&author=R.+P.+Chengauthor=S.+H.+Gellmanauthor=W.+F.+DeGrado&title=%CE%B2-Peptides%3A+From+Structure+to+Function&doi=10.1021%2Fcr000045i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides: From structure to function</span></div><div class="casAuthors">Cheng, Richard P.; Gellman, Samuel H.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3219-3232</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 187 refs., of the conformational properties and biol. functions, esp. as peptidomimetics, of β-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfycZ30AaKLVg90H21EOLACvtfcHk0lh1HOyWvWRE8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D&md5=40cbf3f3d3c00ec8d6250adf407a663a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fcr000045i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000045i%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3D%25CE%25B2-Peptides%253A%2520From%2520Structure%2520to%2520Function%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D3219%26epage%3D3232%26doi%3D10.1021%2Fcr000045i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stringer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crapster, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzei, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span> <span> </span><span class="NLM_article-title">Extraordinarily Robust Polyproline Type I Peptoid Helices Generatedviathe Incorporation of α-Chiral AromaticN-1-Naphthylethyl Side Chains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">15559</span>– <span class="NLM_lpage">15567</span>, <span class="refDoi"> DOI: 10.1021/ja204755p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja204755p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFKrs7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=15559-15567&author=J.+R.+Stringerauthor=J.+A.+Crapsterauthor=I.+A.+Guzeiauthor=H.+E.+Blackwell&title=Extraordinarily+Robust+Polyproline+Type+I+Peptoid+Helices+Generatedviathe+Incorporation+of+%CE%B1-Chiral+AromaticN-1-Naphthylethyl+Side+Chains&doi=10.1021%2Fja204755p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Extraordinarily robust polyproline type I peptoid helices generated via the incorporation of α-chiral aromatic N-1-naphthylethyl side chains</span></div><div class="casAuthors">Stringer, Joseph R.; Crapster, J. Aaron; Guzei, Ilia A.; Blackwell, Helen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">15559-15567</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptoids, or oligomers of N-substituted glycines, are a class of foldamers that have shown extraordinary functional potential since their inception nearly two decades ago.  However, the generation of well-defined peptoid secondary structures remains a difficult task.  This challenge is due, in part, to the lack of a thorough understanding of peptoid sequence-structure relationships and, consequently, an incomplete understanding of the peptoid folding process.  We seek to delineate sequence-structure relationships through the systematic study of noncovalent interactions in peptoids and the design of novel amide side chains capable of such interactions.  Herein, we report the synthesis and detailed structural anal. of a series of (S)-N-(1-naphthylethyl)glycine (Ns1npe) peptoid homo-oligomers by X-ray crystallog., NMR spectroscopy, and CD spectroscopy.  Four of these peptoids were found to adopt well-defined structures in the solid state, with dihedral angles similar to those obsd. in polyproline type I (PPI) peptide helixes and in peptoids with α-chiral side chains.  The X-ray crystal structure of a representative Ns1npe tetramer revealed an all cis-amide helix, with approx. three residues per turn, and a helical pitch of approx. 6.0 Å.  2D-NMR anal. of the length-dependent Ns1npe series showed that these peptoids have very high overall backbone amide Kcis/trans values in acetonitrile, indicative of conformationally homogeneous structures in soln.  Addnl., CD spectroscopy studies of the Ns1npe homo-oligomers in acetonitrile and methanol revealed a striking length-dependent increase in ellipticity per amide.  These Ns1npe helixes represent the most robust peptoid helixes to be reported, and the incorporation of (S)-N-(1-naphthylethyl)glycines provides a new approach for the generation of stable helical structure in this important class of foldamers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWOAhoe5DJ0bVg90H21EOLACvtfcHk0lgOuHp4-7eB_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKrs7vO&md5=58c83f67e4dc7954a39bb09f8120dbfb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fja204755p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja204755p%26sid%3Dliteratum%253Aachs%26aulast%3DStringer%26aufirst%3DJ.%2BR.%26aulast%3DCrapster%26aufirst%3DJ.%2BA.%26aulast%3DGuzei%26aufirst%3DI.%2BA.%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26atitle%3DExtraordinarily%2520Robust%2520Polyproline%2520Type%2520I%2520Peptoid%2520Helices%2520Generatedviathe%2520Incorporation%2520of%2520%25CE%25B1-Chiral%2520AromaticN-1-Naphthylethyl%2520Side%2520Chains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D15559%26epage%3D15567%26doi%3D10.1021%2Fja204755p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuckermann, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huebner, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jewell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banville, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlowe, C. K.</span></span> <span> </span><span class="NLM_article-title">Peptoids: a modular approach to drug discovery</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">9367</span>– <span class="NLM_lpage">9371</span>, <span class="refDoi"> DOI: 10.1073/pnas.89.20.9367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1073%2Fpnas.89.20.9367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1409642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADyaK3sXhtFKhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=9367-9371&author=R.+J.+Simonauthor=R.+S.+Kaniaauthor=R.+N.+Zuckermannauthor=V.+D.+Huebnerauthor=D.+A.+Jewellauthor=S.+Banvilleauthor=S.+Ngauthor=L.+Wangauthor=S.+Rosenbergauthor=C.+K.+Marlowe&title=Peptoids%3A+a+modular+approach+to+drug+discovery&doi=10.1073%2Fpnas.89.20.9367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Peptoids:  a modular approach to drug discovery</span></div><div class="casAuthors">Simon, Reyna J.; Kania, Robert S.; Zuckermann, Ronald N.; Huebner, Verena D.; Jewell, David A.; Banville, Steven; Ng, Simon; Wang, Liang; Rosenberg, Steven; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9367-71</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Peptoids, oligomers of N-substituted glycines, are described as a motif for the generation of chem. diverse libraries of novel mols.  Ramachandran-type plots were calcd. and indicate a greater diversity of conformational states available for peptoids than for peptides.  The monomers incorporate t-butyl-based side-chain and 9-fluorenylmethoxy-carbonyl α-amine protection.  The controlled oligomerization of the peptoid monomers was performed manually and robotically with in situ activation by either benzotriazol-1-yloxytris(pyrrolidino)phosphonium hexafluorophosphate or bromotris(pyrrolidino)phosphonium hexafluorophosphate.  Other steps were identical to peptide synthesis using α-(9-fluorenylmethoxycarbonyl)amino acids.  A total of 15 monomers and 10 oligomers (peptoids) are described.  Preliminary data are presented on the stability of a representative oligopeptoid to enzymic hydrolysis.  Peptoid versions of peptide ligands of three biol. systems (bovine pancreatic α-amylase, hepatitis A virus 3C proteinase, and human immunodeficiency virus transactivator-responsive element RNA) were found with affinities comparable to those of the corresponding peptides.  The potential use of libraries of these compds. in receptor- or enzyme-based assays is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqjYjAnzlZWbVg90H21EOLACvtfcHk0lgOuHp4-7eB_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhtFKhsro%253D&md5=072eed656364cd7cf06c71aa9fb001c3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.20.9367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.20.9367%26sid%3Dliteratum%253Aachs%26aulast%3DSimon%26aufirst%3DR.%2BJ.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DZuckermann%26aufirst%3DR.%2BN.%26aulast%3DHuebner%26aufirst%3DV.%2BD.%26aulast%3DJewell%26aufirst%3DD.%2BA.%26aulast%3DBanville%26aufirst%3DS.%26aulast%3DNg%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DRosenberg%26aufirst%3DS.%26aulast%3DMarlowe%26aufirst%3DC.%2BK.%26atitle%3DPeptoids%253A%2520a%2520modular%2520approach%2520to%2520drug%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1992%26volume%3D89%26spage%3D9367%26epage%3D9371%26doi%3D10.1073%2Fpnas.89.20.9367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fristrup, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">Triangular prism-shaped β-peptoid helices as unique biomimetic scaffolds</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7013</span>– <span class="NLM_lpage">7022</span>, <span class="refDoi"> DOI: 10.1038/ncomms8013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fncomms8013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=25943784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFylt7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7013-7022&author=J.+S.+Laursenauthor=P.+Harrisauthor=P.+Fristrupauthor=C.+A.+Olsen&title=Triangular+prism-shaped+%CE%B2-peptoid+helices+as+unique+biomimetic+scaffolds&doi=10.1038%2Fncomms8013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Triangular prism-shaped β-peptoid helices as unique biomimetic scaffolds</span></div><div class="casAuthors">Laursen, Jonas S.; Harris, Pernille; Fristrup, Peter; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7013</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">β-Peptoids are peptidomimetics based on N-alkylated β-aminopropionic acid residues (or N-alkyl-β-alanines).  This type of peptide mimic has previously been incorporated in biol. active ligands and has been hypothesized to be able to exhibit foldamer properties.  Here we show, for the first time, that β-peptoids can be tuned to fold into stable helical structures.  We provide high-resoln. X-ray crystal structures of homomeric β-peptoid hexamers, which reveal right-handed helical conformations with exactly three residues per turn and a helical pitch of 9.6-9.8 Å between turns.  The presence of folded conformations in soln. is supported by CD spectroscopy showing length- and solvent dependency, and mol. dynamics simulations provide further support for a stabilized helical secondary structure in org. solvent.  We thus outline a framework for future design of novel biomimetics that display functional groups with high accuracy in three dimensions, which has potential for development of new functional materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEynUbb3geprVg90H21EOLACvtfcHk0lgOuHp4-7eB_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFylt7jN&md5=2ae2ab87cd0f7faf8eab3774ed77ca42</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fncomms8013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8013%26sid%3Dliteratum%253Aachs%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DFristrup%26aufirst%3DP.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DTriangular%2520prism-shaped%2520%25CE%25B2-peptoid%2520helices%2520as%2520unique%2520biomimetic%2520scaffolds%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7013%26epage%3D7022%26doi%3D10.1038%2Fncomms8013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claudon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miclet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Didierjean, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ennifar, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guichard, G.</span></span> <span> </span><span class="NLM_article-title">The canonical helix of urea oligomers at atomic resolution: insights into folding-induced axial organization</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1070</span>, <span class="refDoi"> DOI: 10.1002/anie.200905592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200905592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1067-1070&author=L.+Fischerauthor=P.+Claudonauthor=N.+Pendemauthor=E.+Micletauthor=C.+Didierjeanauthor=E.+Ennifarauthor=G.+Guichard&title=The+canonical+helix+of+urea+oligomers+at+atomic+resolution%3A+insights+into+folding-induced+axial+organization&doi=10.1002%2Fanie.200905592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fanie.200905592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200905592%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DL.%26aulast%3DClaudon%26aufirst%3DP.%26aulast%3DPendem%26aufirst%3DN.%26aulast%3DMiclet%26aufirst%3DE.%26aulast%3DDidierjean%26aufirst%3DC.%26aulast%3DEnnifar%26aufirst%3DE.%26aulast%3DGuichard%26aufirst%3DG.%26atitle%3DThe%2520canonical%2520helix%2520of%2520urea%2520oligomers%2520at%2520atomic%2520resolution%253A%2520insights%2520into%2520folding-induced%2520axial%2520organization%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D49%26spage%3D1067%26epage%3D1070%26doi%3D10.1002%2Fanie.200905592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Proulx, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopewell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiegel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García
Ramos, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubell, W. D.</span></span> <span> </span><span class="NLM_article-title">Azapeptides and their therapeutic potential</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1164</span>, <span class="refDoi"> DOI: 10.4155/fmc.11.74</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.4155%2Ffmc.11.74" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=21806378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFeisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1139-1164&author=C.+Proulxauthor=D.+Sabatinoauthor=R.+Hopewellauthor=J.+Spiegelauthor=Y.+Garc%C3%ADa%0ARamosauthor=W.+D.+Lubell&title=Azapeptides+and+their+therapeutic+potential&doi=10.4155%2Ffmc.11.74"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Azapeptides and their therapeutic potential</span></div><div class="casAuthors">Proulx, Caroline; Sabatino, David; Hopewell, Robert; Spiegel, Jochen; Garciia Ramos, Yeesica; Lubell, William D.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1139-1164</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Azapeptides are peptide analogs in which one or more of the amino residues is replaced by a semicarbazide.  This substitution of a nitrogen for the αα-carbon center results in conformational restrictions, which bend the peptide about the aza-amino acid residue away from a linear geometry.  The resulting azapeptide turn conformations have been obsd. by x-ray crystallog. and spectroscopy, as well as predicted based on computational models.  In biol. active peptide analogs, the aza-substitution has led to enhanced activity and selectivity as well as improved properties, such as prolonged duration of action and metabolic stability.  In light of these characteristics, azapeptides have found important uses as receptor ligands, enzyme inhibitors, drugs, pro-drugs, probes and imaging agents.  Recent improvements in synthetic methods for their procurement have ushered in a new era of azapeptide chem.  This review aims to provide a historical look at the development of azapeptide science along with a focus on recent developments and perspectives on the future of this useful tool for medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjgnDgmvF9zbVg90H21EOLACvtfcHk0lhJFRVXeSI_EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFeisbo%253D&md5=3f4150c3eedaff90de608d8b1d1167ab</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.74%26sid%3Dliteratum%253Aachs%26aulast%3DProulx%26aufirst%3DC.%26aulast%3DSabatino%26aufirst%3DD.%26aulast%3DHopewell%26aufirst%3DR.%26aulast%3DSpiegel%26aufirst%3DJ.%26aulast%3DGarc%25C3%25ADa%2BRamos%26aufirst%3DY.%26aulast%3DLubell%26aufirst%3DW.%2BD.%26atitle%3DAzapeptides%2520and%2520their%2520therapeutic%2520potential%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D1139%26epage%3D1164%26doi%3D10.4155%2Ffmc.11.74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malachowski, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broadrup, R. L.</span></span> <span> </span><span class="NLM_article-title">The Synthesis of Azapeptidomimetic β-Lactam Molecules as Potential Protease Inhibitors</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">8962</span>– <span class="NLM_lpage">8969</span>, <span class="refDoi"> DOI: 10.1021/jo026280d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo026280d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD38XoslSnsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2002&pages=8962-8969&author=W.+P.+Malachowskiauthor=C.+Tieauthor=K.+Wangauthor=R.+L.+Broadrup&title=The+Synthesis+of+Azapeptidomimetic+%CE%B2-Lactam+Molecules+as+Potential+Protease+Inhibitors&doi=10.1021%2Fjo026280d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Synthesis of Azapeptidomimetic β-Lactam Molecules as Potential Protease Inhibitors</span></div><div class="casAuthors">Malachowski, William P.; Tie, Chenyang; Wang, Katherine; Broadrup, Robert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8962-8969</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic methods for the construction of a novel peptidomimetic structure are reported.  The structure incorporates a β-lactam and an azapeptide in a peptide backbone with the intention of generating rationally designed substrate-based protease inhibitors.  The β-lactam is formed by subjecting serine or threonine-azapeptides to Mitsunobu reaction conditions.  Importantly, the azapeptidomimetic β-lactam structure permits extended binding inhibition and the synthetic methods to create tetrapeptidomimetic structures are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYAXa4rgpnWbVg90H21EOLACvtfcHk0lhJFRVXeSI_EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XoslSnsrc%253D&md5=82e7b97f900cbe68f47febf9635f903c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjo026280d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo026280d%26sid%3Dliteratum%253Aachs%26aulast%3DMalachowski%26aufirst%3DW.%2BP.%26aulast%3DTie%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DBroadrup%26aufirst%3DR.%2BL.%26atitle%3DThe%2520Synthesis%2520of%2520Azapeptidomimetic%2520%25CE%25B2-Lactam%2520Molecules%2520as%2520Potential%2520Protease%2520Inhibitors%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2002%26volume%3D67%26spage%3D8962%26epage%3D8969%26doi%3D10.1021%2Fjo026280d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewandowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapp, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wennemers, H.</span></span> <span> </span><span class="NLM_article-title">A crystal structure of an oligoproline PPII-helix, at last</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">15829</span>– <span class="NLM_lpage">15832</span>, <span class="refDoi"> DOI: 10.1021/ja507405j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja507405j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=15829-15832&author=P.+Wilhelmauthor=B.+Lewandowskiauthor=N.+Trappauthor=H.+Wennemers&title=A+crystal+structure+of+an+oligoproline+PPII-helix%2C+at+last&doi=10.1021%2Fja507405j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A Crystal Structure of an Oligoproline PPII-Helix, at Last</span></div><div class="casAuthors">Wilhelm, Patrick; Lewandowski, Bartosz; Trapp, Nils; Wennemers, Helma</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">15829-15832</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The first crystal structure of an oligoproline adopting an all-trans polyproline II (PPII) helix is presented.  The high-resoln. structure provides detailed insight into the dimensions and conformational properties of oligoprolines that are important for, e.g., their use as "mol. rulers" and "mol. scaffolds".  The structure also showed that the amides interact with each other within a PPII helix and that water is not necessary for PPII helicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7pc_UwBLWl7Vg90H21EOLACvtfcHk0li1ZJPnSfT62Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGlsLnP&md5=ebe46ea8bcd03187632c2595dc5f2959</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja507405j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja507405j%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DP.%26aulast%3DLewandowski%26aufirst%3DB.%26aulast%3DTrapp%26aufirst%3DN.%26aulast%3DWennemers%26aufirst%3DH.%26atitle%3DA%2520crystal%2520structure%2520of%2520an%2520oligoproline%2520PPII-helix%252C%2520at%2520last%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D15829%26epage%3D15832%26doi%3D10.1021%2Fja507405j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.-T.</span></span> <span> </span><span class="NLM_article-title">Aromatic amide foldamers: structures, properties, and functions</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">5271</span>– <span class="NLM_lpage">5316</span>, <span class="refDoi"> DOI: 10.1021/cr300116k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr300116k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFGhsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2012&pages=5271-5316&author=D.-W.+Zhangauthor=X.+Zhaoauthor=J.-L.+Houauthor=Z.-T.+Li&title=Aromatic+amide+foldamers%3A+structures%2C+properties%2C+and+functions&doi=10.1021%2Fcr300116k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatic Amide Foldamers: Structures, Properties, and Functions</span></div><div class="casAuthors">Zhang, Dan-Wei; Zhao, Xin; Hou, Jun-Li; Li, Zhan-Ting</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5271-5316</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This review surveys the study of arom. amide-based foldamers that are stabilized by intramol. hydrogen bonding.  Since the first report of this family of foldamers in 1994, the major effort in this field has been devoted to the synthesis and structural characterization of new foldamers.  Considerable structural complexity has been realized, which may compare with that of natural proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJVTy5WdopfbVg90H21EOLACvtfcHk0li1ZJPnSfT62Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFGhsr7L&md5=bbc3706fb62786384477feca2dde1da9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fcr300116k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr300116k%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.-W.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHou%26aufirst%3DJ.-L.%26aulast%3DLi%26aufirst%3DZ.-T.%26atitle%3DAromatic%2520amide%2520foldamers%253A%2520structures%252C%2520properties%252C%2520and%2520functions%26jtitle%3DChem.%2520Rev.%26date%3D2012%26volume%3D112%26spage%3D5271%26epage%3D5316%26doi%3D10.1021%2Fcr300116k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garric, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Léger, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huc, I.</span></span> <span> </span><span class="NLM_article-title">Molecular apple peels</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1954</span>– <span class="NLM_lpage">1958</span>, <span class="refDoi"> DOI: 10.1002/anie.200462898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200462898" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt12qt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=1954-1958&author=J.+Garricauthor=J.-M.+L%C3%A9gerauthor=I.+Huc&title=Molecular+apple+peels&doi=10.1002%2Fanie.200462898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular apple peels</span></div><div class="casAuthors">Garric, Joachim; Leger, Jean-Michel; Huc, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1954-1958</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peeling the skin of an apple into a single helical ribbon gives a sort of shell that can be wound back around the apple.  Such a shell can be constructed at the mol. scale using a helix with a reduced diam. at both ends which behaves as a capsule and entraps a small guest such as water.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-trEn2-7TvbVg90H21EOLACvtfcHk0li1ZJPnSfT62Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjt12qt7s%253D&md5=9b32455e77fe77a96ce9118e7dab7d6f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462898%26sid%3Dliteratum%253Aachs%26aulast%3DGarric%26aufirst%3DJ.%26aulast%3DL%25C3%25A9ger%26aufirst%3DJ.-M.%26aulast%3DHuc%26aufirst%3DI.%26atitle%3DMolecular%2520apple%2520peels%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D1954%26epage%3D1958%26doi%3D10.1002%2Fanie.200462898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helsel, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clements, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szabo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, B.</span></span> <span> </span><span class="NLM_article-title">Highly conducting transmembrane pores formed by aromatic oligoamide macrocycles</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">15784</span>– <span class="NLM_lpage">15785</span>, <span class="refDoi"> DOI: 10.1021/ja807078y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja807078y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlWqtbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=15784-15785&author=A.+J.+Helselauthor=A.+L.+Brownauthor=K.+Yamatoauthor=W.+Fengauthor=L.+Yuanauthor=A.+J.+Clementsauthor=S.+V.+Hardingauthor=G.+Szaboauthor=Z.+Shaoauthor=B.+Gong&title=Highly+conducting+transmembrane+pores+formed+by+aromatic+oligoamide+macrocycles&doi=10.1021%2Fja807078y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Conducting Transmembrane Pores Formed by Aromatic Oligoamide Macrocycles</span></div><div class="casAuthors">Helsel, Amber Jade; Brown, Amy L.; Yamato, Kazuhiro; Feng, Wen; Yuan, Lihua; Clements, Aimee J.; Harding, Stephanie V.; Szabo, Gabor; Shao, Zhifeng; Gong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">15784-15785</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oligoamide macrocycles, which carry membrane-compatible side chains and contain a hydrophilic, noncollapsible cavity, were found to mediate high ion flux across a lipid bilayer, as demonstrated by results from 23Na NMR and planar bilayer conductance measurements.  The measured transmembrane single channel currents are very high, rivaling those typically assocd. with pore-forming protein toxins.  The obtained results have demonstrated the promise of developing large, highly conducting channels based on nanopores formed by oligoamide macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdx0iVYSKWLbVg90H21EOLACvtfcHk0li1ZJPnSfT62Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlWqtbjK&md5=17f69429ef66f1f8f7575121e33b1c1d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja807078y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja807078y%26sid%3Dliteratum%253Aachs%26aulast%3DHelsel%26aufirst%3DA.%2BJ.%26aulast%3DBrown%26aufirst%3DA.%2BL.%26aulast%3DYamato%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DW.%26aulast%3DYuan%26aufirst%3DL.%26aulast%3DClements%26aufirst%3DA.%2BJ.%26aulast%3DHarding%26aufirst%3DS.%2BV.%26aulast%3DSzabo%26aufirst%3DG.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DGong%26aufirst%3DB.%26atitle%3DHighly%2520conducting%2520transmembrane%2520pores%2520formed%2520by%2520aromatic%2520oligoamide%2520macrocycles%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D15784%26epage%3D15785%26doi%3D10.1021%2Fja807078y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Selective and potent proteomimetic inhibitors of intracellular protein-protein interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2960</span>– <span class="NLM_lpage">2965</span>, <span class="refDoi"> DOI: 10.1002/anie.201410810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201410810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFCitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=2960-2965&author=A.+Barnardauthor=K.+Longauthor=H.+L.+Martinauthor=J.+A.+Milesauthor=T.+A.+Edwardsauthor=D.+C.+Tomlinsonauthor=A.+Macdonaldauthor=A.+J.+Wilson&title=Selective+and+potent+proteomimetic+inhibitors+of+intracellular+protein-protein+interactions&doi=10.1002%2Fanie.201410810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Selective and Potent Proteomimetic Inhibitors of Intracellular Protein-Protein Interactions</span></div><div class="casAuthors">Barnard, Anna; Long, Kerya; Martin, Heather L.; Miles, Jennifer A.; Edwards, Thomas A.; Tomlinson, Darren C.; MacDonald, Andrew; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2960-2965</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Inhibition of protein-protein interactions (PPIs) represents a major challenge in chem. biol. and drug discovery. α-Helix mediated PPIs may be amenable to modulation using generic chemotypes, termed "proteomimetics", which can be assembled in a modular manner to reproduce the vectoral presentation of key side chains found on a helical motif from one partner within the PPI.  In this work, it is demonstrated that by using a library of N-alkylated arom. oligoamide helix mimetics, potent helix mimetics which reproduce their biophys. binding selectivity in a cellular context can be identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWm-xJOpoE8bVg90H21EOLACvtfcHk0ljyn7i77899mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFCitr8%253D&md5=784a4d064135ea0adbb2a291f20e7f0e</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.201410810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201410810%26sid%3Dliteratum%253Aachs%26aulast%3DBarnard%26aufirst%3DA.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DMartin%26aufirst%3DH.%2BL.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DTomlinson%26aufirst%3DD.%2BC.%26aulast%3DMacdonald%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DSelective%2520and%2520potent%2520proteomimetic%2520inhibitors%2520of%2520intracellular%2520protein-protein%2520interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D2960%26epage%3D2965%26doi%3D10.1002%2Fanie.201410810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azzarito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of α-helix-mediated protein-protein interactions using designed molecules</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">173</span>, <span class="refDoi"> DOI: 10.1038/nchem.1568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchem.1568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=23422557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=161-173&author=V.+Azzaritoauthor=K.+Longauthor=N.+S.+Murphyauthor=A.+J.+Wilson&title=Inhibition+of+%CE%B1-helix-mediated+protein-protein+interactions+using+designed+molecules&doi=10.1038%2Fnchem.1568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of α-helix-mediated protein-protein interactions using designed molecules</span></div><div class="casAuthors">Azzarito, Valeria; Long, Kerya; Murphy, Natasha S.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">161-173</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inhibition of protein-protein interactions (PPIs) represents a significant challenge because it is unclear how they can be effectively and selectively targeted using small mols.  Achieving this goal is crit. given the defining role of these interactions in biol. processes.  A rational approach to inhibitor design based on the secondary structure at the interface is the focus of much research, and different classes of designed ligands have emerged, some of which effectively and selectively disrupt targeted PPIs.  This Review discusses the relevance of PPIs and, in particular, the importance of α-helix-mediated PPIs to chem. biol. and drug discovery with a focus on designing inhibitors, including constrained peptides, foldamers and proteomimetic-derived ligands.  In doing so, key challenges and major advances in developing generic approaches for the elaboration of PPI inhibitors are highlighted.  The challenges faced in developing such ligands as drug leads - and how criteria applied to these may differ from conventional small-mol. drugs - are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpsQAFaEkKq7Vg90H21EOLACvtfcHk0ljyn7i77899mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXislWms7c%253D&md5=161f88aa1c745dee9a95b1fab340b9fe</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1568%26sid%3Dliteratum%253Aachs%26aulast%3DAzzarito%26aufirst%3DV.%26aulast%3DLong%26aufirst%3DK.%26aulast%3DMurphy%26aufirst%3DN.%2BS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DInhibition%2520of%2520%25CE%25B1-helix-mediated%2520protein-protein%2520interactions%2520using%2520designed%2520molecules%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D161%26epage%3D173%26doi%3D10.1038%2Fnchem.1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of inhibitors of protein-protein interactions: mimicking peptide binding epitopes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8896</span>– <span class="NLM_lpage">8927</span>, <span class="refDoi"> DOI: 10.1002/anie.201412070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201412070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26119925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=8896-8927&author=M.+Pelay-Gimenoauthor=A.+Glasauthor=O.+Kochauthor=T.+N.+Grossmann&title=Structure-based+design+of+inhibitors+of+protein-protein+interactions%3A+mimicking+peptide+binding+epitopes&doi=10.1002%2Fanie.201412070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Inhibitors of Protein-Protein Interactions: Mimicking Peptide Binding Epitopes</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Glas, Adrian; Koch, Oliver; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">8896-8927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are involved at all levels of cellular organization, thus making the development of PPI inhibitors extremely valuable.  The identification of selective inhibitors is challenging because of the shallow and extended nature of PPI interfaces.  Inhibitors can be obtained by mimicking peptide binding epitopes in their bioactive conformation.  For this purpose, several strategies have been evolved to enable a projection of side chain functionalities in analogy to peptide secondary structures, thereby yielding mols. that are generally referred to as peptidomimetics.  Herein, we introduce a new classification of peptidomimetics (classes A-D) that enables a clear assignment of available approaches.  Based on this classification, the Review summarizes strategies that have been applied for the structure-based design of PPI inhibitors through stabilizing or mimicking turns, β-sheets, and helixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPTEmZpatXbrVg90H21EOLACvtfcHk0ljyn7i77899mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM&md5=447c84aab6275b1057fda9a610918f2b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412070%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DO.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DStructure-based%2520design%2520of%2520inhibitors%2520of%2520protein-protein%2520interactions%253A%2520mimicking%2520peptide%2520binding%2520epitopes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D8896%26epage%3D8927%26doi%3D10.1002%2Fanie.201412070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, T. T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieto, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vries-van Leeuwen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodarzifard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agten, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milroy, L.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunsveld, L.</span></span> <span> </span><span class="NLM_article-title">Proline primed helix length as a modulator of the nuclear receptor-coactivator interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">4364</span>– <span class="NLM_lpage">4371</span>, <span class="refDoi"> DOI: 10.1021/ja311748r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja311748r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVChurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=4364-4371&author=S.+Fuchsauthor=H.+D.+Nguyenauthor=T.+T.+P.+Phanauthor=M.+F.+Burtonauthor=L.+Nietoauthor=I.+J.+de+Vries-van+Leeuwenauthor=A.+Schmidtauthor=M.+Goodarzifardauthor=S.+M.+Agtenauthor=R.+Roseauthor=C.+Ottmannauthor=L.-G.+Milroyauthor=L.+Brunsveld&title=Proline+primed+helix+length+as+a+modulator+of+the+nuclear+receptor-coactivator+interaction&doi=10.1021%2Fja311748r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Proline primed helix length as a modulator of the nuclear receptor-coactivator interaction</span></div><div class="casAuthors">Fuchs, Sascha; Nguyen, Hoang D.; Phan, Trang T. P.; Burton, Matthew F.; Nieto, Lidia; de Vries-van Leeuwen, Ingrid J.; Schmidt, Andrea; Goodarzifard, Monireh; Agten, Stijn M.; Rose, Rolf; Ottmann, Christian; Milroy, Lech-Gustav; Brunsveld, Luc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4364-4371</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nuclear receptor binding to coactivator proteins is an obligate first step in the regulation of gene transcription.  Nuclear receptors preferentially bind to an LXXLL peptide motif which is highly conserved throughout the 300 or so natural coactivator proteins.  This knowledge has shaped current understanding of this fundamental protein-protein interaction, and continues to inspire the search for new drug therapies.  However, sequence specificity beyond the LXXLL motif and the mol. functioning of flanking residues still requires urgent addressing.  Here, ribosome display has been used to reassess the estrogen receptor for new and enlarged peptide recognition motifs, leading to the discovery of a potent and highly evolved PXLXXLLXXP binding consensus.  Mol. modeling and X-ray crystallog. studies have provided the mol. insights on the role of the flanking prolines in priming the length of the α-helix and enabling optimal interactions of the α-helix dipole and its surrounding amino acids with the surface charge clamp and the receptor activation function 2.  These findings represent new structural parameters for modulating the nuclear receptor-coactivator interaction based on linear sequences of proteinogenic amino acids and for the design of chem. modified inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Zscu0TYgNrVg90H21EOLACvtfcHk0ljH5-nLPiLXMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVChurw%253D&md5=bb7ccb4ba793d87df61a7f59e1635931</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja311748r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja311748r%26sid%3Dliteratum%253Aachs%26aulast%3DFuchs%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DH.%2BD.%26aulast%3DPhan%26aufirst%3DT.%2BT.%2BP.%26aulast%3DBurton%26aufirst%3DM.%2BF.%26aulast%3DNieto%26aufirst%3DL.%26aulast%3Dde%2BVries-van%2BLeeuwen%26aufirst%3DI.%2BJ.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DGoodarzifard%26aufirst%3DM.%26aulast%3DAgten%26aufirst%3DS.%2BM.%26aulast%3DRose%26aufirst%3DR.%26aulast%3DOttmann%26aufirst%3DC.%26aulast%3DMilroy%26aufirst%3DL.-G.%26aulast%3DBrunsveld%26aufirst%3DL.%26atitle%3DProline%2520primed%2520helix%2520length%2520as%2520a%2520modulator%2520of%2520the%2520nuclear%2520receptor-coactivator%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D4364%26epage%3D4371%26doi%3D10.1021%2Fja311748r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grison, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aitken, D. J.</span></span> <span> </span><span class="NLM_article-title">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein-Protein Interactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">11096</span>– <span class="NLM_lpage">11100</span>, <span class="refDoi"> DOI: 10.1002/anie.201604517</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201604517" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=11096-11100&author=C.+M.+Grisonauthor=J.+A.+Milesauthor=S.+Robinauthor=A.+J.+Wilsonauthor=D.+J.+Aitken&title=An+%CE%B1-Helix-Mimicking+12%2C13-Helix%3A+Designed+%CE%B1%2F%CE%B2%2F%CE%B3-Foldamers+as+Selective+Inhibitors+of+Protein-Protein+Interactions&doi=10.1002%2Fanie.201604517"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">An α-Helix-Mimicking 12,13-Helix: Designed α/β/γ-Foldamers as Selective Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Grison, Claire M.; Miles, Jennifer A.; Robin, Sylvie; Wilson, Andrew J.; Aitken, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">11096-11100</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A major current challenge in bioorg. chem. is the identification of effective mimics of protein secondary structures that act as inhibitors of protein-protein interactions (PPIs).  In this work, trans-2-aminocyclobutanecarboxylic acid (tACBC) was used as the key β-amino acid component in the design of α/β/γ-peptides to structurally mimic a native α-helix.  Suitably functionalized α/β/γ-peptides assume an α-helix-mimicking 12,13-helix conformation in soln., exhibit enhanced proteolytic stability in comparison to the wild-type α-peptide parent sequence from which they are derived, and act as selective inhibitors of the p53/hDM2 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx8w9r7WakLrVg90H21EOLACvtfcHk0ljH5-nLPiLXMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1GmtbvM&md5=f7f8e78b97daa7452ae4e655056265c7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fanie.201604517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201604517%26sid%3Dliteratum%253Aachs%26aulast%3DGrison%26aufirst%3DC.%2BM.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DRobin%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26aulast%3DAitken%26aufirst%3DD.%2BJ.%26atitle%3DAn%2520%25CE%25B1-Helix-Mimicking%252012%252C13-Helix%253A%2520Designed%2520%25CE%25B1%252F%25CE%25B2%252F%25CE%25B3-Foldamers%2520as%2520Selective%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D11096%26epage%3D11100%26doi%3D10.1002%2Fanie.201604517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Azzarito, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warriner, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Stereocontrolled protein surface recognition using chiral oligoamide proteomimetic foldamers</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2434</span>– <span class="NLM_lpage">2443</span>, <span class="refDoi"> DOI: 10.1039/c4sc03559c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc4sc03559c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=29308155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=2434-2443&author=V.+Azzaritoauthor=J.+A.+Milesauthor=J.+Fisherauthor=T.+A.+Edwardsauthor=S.+L.+Warrinerauthor=A.+J.+Wilson&title=Stereocontrolled+protein+surface+recognition+using+chiral+oligoamide+proteomimetic+foldamers&doi=10.1039%2Fc4sc03559c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Stereocontrolled protein surface recognition using chiral oligo-amide proteomimetic foldamers</span></div><div class="casAuthors">Azzarito, Valeria; Miles, Jennifer A.; Fisher, Julie; Edwards, Thomas A.; Warriner, Stuart L.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2434-2443</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The development of foldamers capable of selective mol. recognition of solvent exposed protein surfaces represents an outstanding challenge in supramol. chem. biol.  Here we introduce an oligo-amide foldamer with well-defined conformation that bears all the hallmarks of an information rich oligomer.  Specifically, the foldamer recognizes its target protein hDM2 leading to inhibition of its protein-protein interaction with p53 in a manner that depends upon the compn., spatial projection and stereochem. of functional groups appended to the scaffold.  Most significantly, selective inhibition of p53/hDM2 can be achieved against four other targets and the selectivity for p53/hDM2 inhibition vs. Mcl-1/NOXA-B inhibition is critically dependent upon the stereochem. of the helix mimetic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7zOD1CDOSLbVg90H21EOLACvtfcHk0ljH5-nLPiLXMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCitrY%253D&md5=8690223c203659a046f2004d36e643a1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1039%2Fc4sc03559c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4sc03559c%26sid%3Dliteratum%253Aachs%26aulast%3DAzzarito%26aufirst%3DV.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DFisher%26aufirst%3DJ.%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DWarriner%26aufirst%3DS.%2BL.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DStereocontrolled%2520protein%2520surface%2520recognition%2520using%2520chiral%2520oligoamide%2520proteomimetic%2520foldamers%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D2434%26epage%3D2443%26doi%3D10.1039%2Fc4sc03559c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durell, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appella, D. H.</span></span> <span> </span><span class="NLM_article-title">Probing the Structural Requirements of Peptoids That Inhibit HDM2–p53 Interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1995</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/ja056344c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja056344c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD28Xksl2ltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=1995-2004&author=T.+Haraauthor=S.+R.+Durellauthor=M.+C.+Myersauthor=D.+H.+Appella&title=Probing+the+Structural+Requirements+of+Peptoids+That+Inhibit+HDM2%E2%80%93p53+Interactions&doi=10.1021%2Fja056344c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Structural Requirements of Peptoids That Inhibit HDM2-p53 Interactions</span></div><div class="casAuthors">Hara, Toshiaki; Durell, Stewart R.; Myers, Michael C.; Appella, Daniel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1995-2004</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many cellular processes are controlled by protein-protein interactions, and selective inhibition of these interactions could lead to the development of new therapies for several diseases.  For example, overexpression of the protein HDM2 (human double minute 2), which binds to and inactivates the protein p53, has been linked to tumor aggressiveness and drug resistance in cancer.  In creating inhibitors of protein-protein interactions, one strategy is to recreate the three-dimensional arrangement of side chains that are involved in the binding of one protein to another, using a nonnatural scaffold as the attachment point for the side chains.  Here, the authors used oligomeric peptoids as the scaffold to develop a general strategy to rationally design synthetic mols. that can be optimized for inhibition of protein-protein interactions.  Structural information on the HDM2-p53 complex was used to design the first class of peptoid inhibitors, and this work provides, in detail, the strategy to modify peptoids with the appropriate side chains that are effective inhibitors of HDM2-p53 binding.  While the authors initially tried to develop rigid, helical peptoids as HDM2 binders, the best inhibitors were surprisingly peptoids that lacked any helix-promoting groups.  These results indicate that starting with rigid peptoid scaffolds may not always be optimal to develop new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFaKncMNdpFbVg90H21EOLACvtfcHk0lhcFSr9PYhIZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xksl2ltg%253D%253D&md5=d4fbb08ec90988e85c2a6dec0ea9fc6e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja056344c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja056344c%26sid%3Dliteratum%253Aachs%26aulast%3DHara%26aufirst%3DT.%26aulast%3DDurell%26aufirst%3DS.%2BR.%26aulast%3DMyers%26aufirst%3DM.%2BC.%26aulast%3DAppella%26aufirst%3DD.%2BH.%26atitle%3DProbing%2520the%2520Structural%2520Requirements%2520of%2520Peptoids%2520That%2520Inhibit%2520HDM2%25E2%2580%2593p53%2520Interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D1995%26epage%3D2004%26doi%3D10.1021%2Fja056344c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lao, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarracino, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewer, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heindel, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Rational design of topographical helix mimics as potent inhibitors of protein-protein interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">7877</span>– <span class="NLM_lpage">7888</span>, <span class="refDoi"> DOI: 10.1021/ja502310r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja502310r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=7877-7888&author=B.+B.+Laoauthor=K.+Drewauthor=D.+A.+Guarracinoauthor=T.+F.+Brewerauthor=D.+W.+Heindelauthor=R.+Bonneauauthor=P.+S.+Arora&title=Rational+design+of+topographical+helix+mimics+as+potent+inhibitors+of+protein-protein+interactions&doi=10.1021%2Fja502310r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Topographical Helix Mimics as Potent Inhibitors of Protein-Protein Interactions</span></div><div class="casAuthors">Lao, Brooke Bullock; Drew, Kevin; Guarracino, Danielle A.; Brewer, Thomas F.; Heindel, Daniel W.; Bonneau, Richard; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7877-7888</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interactions encompass large surface areas, but often a handful of key residues dominate the binding energy landscape.  Rationally designed small mol. scaffolds that reproduce the relative positioning and disposition of important binding residues, termed "hotspot residues", have been shown to successfully inhibit specific protein complexes.  Although this strategy has led to development of novel synthetic inhibitors of protein complexes, often direct mimicry of natural amino acid residues does not lead to potent inhibitors.  Exptl. screening of focused compd. libraries is used to further optimize inhibitors but the no. of possible designs that can be efficiently synthesized and exptl. tested in academic settings is limited.  We have applied the principles of computational protein design to optimization of nonpeptidic helix mimics as ligands for protein complexes.  We describe the development of computational tools to design helix mimetics from canonical and noncanonical residue libraries and their application to two therapeutically important protein-protein interactions: p53-MDM2 and p300-HIF1α.  The overall study provides a streamlined approach for discovering potent peptidomimetic inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNMadvCjMqobVg90H21EOLACvtfcHk0lhcFSr9PYhIZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1WltLo%253D&md5=e19fc2448fcf0e6b71bdc208e1753209</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja502310r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja502310r%26sid%3Dliteratum%253Aachs%26aulast%3DLao%26aufirst%3DB.%2BB.%26aulast%3DDrew%26aufirst%3DK.%26aulast%3DGuarracino%26aufirst%3DD.%2BA.%26aulast%3DBrewer%26aufirst%3DT.%2BF.%26aulast%3DHeindel%26aufirst%3DD.%2BW.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DRational%2520design%2520of%2520topographical%2520helix%2520mimics%2520as%2520potent%2520inhibitors%2520of%2520protein-protein%2520interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D7877%26epage%3D7888%26doi%3D10.1021%2Fja502310r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bautista, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appelbaum, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">Bridged β<sup>3</sup>-peptide inhibitors of p53-hDM2 complexation: correlation between affinity and cell permeability</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>,  <span class="NLM_fpage">2904</span>– <span class="NLM_lpage">2906</span>, <span class="refDoi"> DOI: 10.1021/ja910715u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja910715u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvF2rs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=2904-2906&author=A.+D.+Bautistaauthor=J.+S.+Appelbaumauthor=C.+J.+Craigauthor=J.+Michelauthor=A.+Schepartz&title=Bridged+%CE%B23-peptide+inhibitors+of+p53-hDM2+complexation%3A+correlation+between+affinity+and+cell+permeability&doi=10.1021%2Fja910715u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bridged β3-Peptide Inhibitors of p53-hDM2 Complexation: Correlation between Affinity and Cell Permeability</span></div><div class="casAuthors">Bautista, Arjel D.; Appelbaum, Jacob S.; Craig, Cody J.; Michel, Julien; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2904-2906</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">β-Peptides possess several features that are desirable in peptidomimetics; they are easily synthesized, fold into stable secondary structures in physiol. buffers, and resist proteolysis.  They can also bind to a diverse array of proteins to inhibit their interactions with α-helical ligands. β-peptides are usually not cell-permeable, however, and this feature limits their utility as research tools and potential therapeutics.  Appending an Arg8 sequence to a β-peptide improves uptake but adds considerable mass.  We previously reported that embedding a small cationic patch within a PPII, α-, or β-peptide helix improves uptake without the addn. of significant mass.  In another mass-neutral strategy, Verdine, Walensky, and others have reported that insertion of a hydrocarbon bridge between the i and i + 4 positions of an α-helix also increases cell uptake.  Here we describe a series of β-peptides contg. diether and hydrocarbon bridges and compare them on the basis of cell uptake and localization, affinities for hDM2, and 14-helix structure.  Our results highlight the relative merits of the cationic-patch and hydrophobic-bridge strategies for improving β-peptide uptake and identify a surprising correlation between uptake efficiency and hDM2 affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5ae2kqfCt7Vg90H21EOLACvtfcHk0liWwL-s8gK7rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvF2rs70%253D&md5=5d5dd9c67f784887c4903479c58d30a1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja910715u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja910715u%26sid%3Dliteratum%253Aachs%26aulast%3DBautista%26aufirst%3DA.%2BD.%26aulast%3DAppelbaum%26aufirst%3DJ.%2BS.%26aulast%3DCraig%26aufirst%3DC.%2BJ.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DBridged%2520%25CE%25B23-peptide%2520inhibitors%2520of%2520p53-hDM2%2520complexation%253A%2520correlation%2520between%2520affinity%2520and%2520cell%2520permeability%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D2904%26epage%3D2906%26doi%3D10.1021%2Fja910715u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Payne, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span> <span> </span><span class="NLM_article-title">Terphenyl-based helical mimetics that disrupt the p53/HDM2 interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2704</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.1002/anie.200462316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.200462316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Chtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=2704-2707&author=H.+Yinauthor=G.-i.+Leeauthor=H.+S.+Parkauthor=G.+A.+Payneauthor=J.+M.+Rodriguezauthor=S.+M.+Sebtiauthor=A.+D.+Hamilton&title=Terphenyl-based+helical+mimetics+that+disrupt+the+p53%2FHDM2+interaction&doi=10.1002%2Fanie.200462316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Terphenyl-based helical mimetics that disrupt the p53/HDM2 interaction</span></div><div class="casAuthors">Yin, Hang; Lee, Gui-in; Park, Hyung Soon; Payne, Gregory A.; Rodriguez, Johanna M.; Sebti, Said M.; Hamilton, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2704-2707</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">HDM2 regulates p53 by binding to its transactivation domain and promoting its ubiquitin-dependent degrdn.  Crystallog. anal. of the HDM2/p53 complex revealed that three hydrophobic residues (F19, W23, L26) along one face of the p53 helical peptide are essential for binding (see picture).  Terphenyl-based antagonists mimic the α-helical region of p53 and disrupt HDM2/p53 complexation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUrbA6KRguLVg90H21EOLACvtfcHk0liWwL-s8gK7rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Chtrs%253D&md5=33c8fc8e94ddc3b6d524fcbe0ed1ebfc</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462316%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DG.-i.%26aulast%3DPark%26aufirst%3DH.%2BS.%26aulast%3DPayne%26aufirst%3DG.%2BA.%26aulast%3DRodriguez%26aufirst%3DJ.%2BM.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26atitle%3DTerphenyl-based%2520helical%2520mimetics%2520that%2520disrupt%2520the%2520p53%252FHDM2%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D2704%26epage%3D2707%26doi%3D10.1002%2Fanie.200462316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shaginian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitby, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farooqi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searcey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Evaluation of an α-Helix Mimetic Library Targeting Protein–Protein Interactions</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">5564</span>– <span class="NLM_lpage">5572</span>, <span class="refDoi"> DOI: 10.1021/ja810025g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja810025g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvFKlsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=5564-5572&author=A.+Shaginianauthor=L.+R.+Whitbyauthor=S.+Hongauthor=I.+Hwangauthor=B.+Farooqiauthor=M.+Searceyauthor=J.+Chenauthor=P.+K.+Vogtauthor=D.+L.+Boger&title=Design%2C+Synthesis%2C+and+Evaluation+of+an+%CE%B1-Helix+Mimetic+Library+Targeting+Protein%E2%80%93Protein+Interactions&doi=10.1021%2Fja810025g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of an α-Helix Mimetic Library Targeting Protein-Protein Interactions</span></div><div class="casAuthors">Shaginian, Alex; Whitby, Landon R.; Hong, Sukwon; Hwang, Inkyu; Farooqi, Bilal; Searcey, Mark; Chen, Jiandong; Vogt, Peter K.; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5564-5572</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A library of α-helix mimetics such as I are prepd. and tested for their abilities to inhibit protein-protein binding such as that between the proteins p53 and HDM2/MDM2.  Eighty compds. based on an initial arylcarbonylaminoarylcarbonylaminobenzoic acid lead compd. are prepd. and tested for their ability to inhibit the binding of p53 to MDM2; aminoacylamino(alkoxy)benzoylamino acid I is found to inhibit the binding most effectively, with an IC50 value of 8 μM.  A library of aminoacylamino(alkoxy)benzoylamino acids based on I are prepd.  Base-mediated substitution of 4-nitro-3-fluorobenzoic acid with alcs. on preparative scale yields 3-alkoxy-4-nitrobenzoic acids with amino acid side chains or close analogs at the alkoxy groups; coupling of the alkoxynitrobenzoic acids with amino acid tert-Bu esters mediated by HOAt and EDC yields a library of 400 nitro(alkoxy)benzoyl amino acid tert-Bu esters II [R = 4-ClC6H4CH2O, MeO, PhCH2O, 2-(3-indolyl)ethoxy, Me3CO2CCH2O, H, 4-MeOC6H4CH2CH2, (S)-MeOCH2OCHMeCH2O, Me2CHO, TIPSOCH2CH2O, 4-TIPSOC6H4CH2CH2O, MeSCH2CH2O, 1-(triphenylmethyl)-4-imidazolemethyl, Me2CHCH2O, (R)-EtCHMeO, BocNH(CH2)4O, EtO, H2NCOCH2O, 2-naphthylmethoxy, PhCH2CH2O; R1 = 4-ClC6H4, Me, PhCH2, 2-(3-indolyl)ethyl, Me3CO2CCH2, H, 4-MeOC6H4CH2, (S)-(Me3CO)CHMe, Me2CH, Me3COCH2, 4-Me3COC6H4CH2, MeSCH2CH2, 1-(triphenylmethyl)-4-imidazolemethyl, Me2CHCH2, (R)-EtCHMe, BocNH(CH2)4, Et, H2NCOCH2, 2-naphthylmethyl, PhCH2CH2; R2 = O2N; TIPS = triisopropylsilyl; Boc = tert-butoxycarbonyl] on roughly 100 mg scale in 90-100% yields.  Microwave-mediated redn. of II (R2 = O2N) with nanopowd. zinc metal and ammonium chloride for 30-300 s yields II (R2 = H2N) ; reaction of II (R2 = H2N) with mixts. of twenty protected amino acids mediated by HOAt and EDC followed by addn. of a polystyrene-bound sulfonyl chloride to scavenge excess II and acid deprotection using HCl in dioxane yields a library of 400 mixts. of 20 aminoacylaminobenzoylamino acids.  The variation of inhibition of the binding of p53 to MDM2 by the library is detd.  The library is generated in stored such that it may be preserved and used for further testing as a α-helix peptidomimetic library for lead compd. generation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt7t2FewsvR7Vg90H21EOLACvtfcHk0liWwL-s8gK7rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjvFKlsLs%253D&md5=29e00d62d157b709247f1c279662c776</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fja810025g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja810025g%26sid%3Dliteratum%253Aachs%26aulast%3DShaginian%26aufirst%3DA.%26aulast%3DWhitby%26aufirst%3DL.%2BR.%26aulast%3DHong%26aufirst%3DS.%26aulast%3DHwang%26aufirst%3DI.%26aulast%3DFarooqi%26aufirst%3DB.%26aulast%3DSearcey%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DVogt%26aufirst%3DP.%2BK.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Evaluation%2520of%2520an%2520%25CE%25B1-Helix%2520Mimetic%2520Library%2520Targeting%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D5564%26epage%3D5572%26doi%3D10.1021%2Fja810025g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henchey, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">High Specificity in Protein Recognition by Hydrogen-Bond-Surrogate α-Helices: Selective Inhibition of the p53/MDM2 Complex</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2104</span>– <span class="NLM_lpage">2107</span>, <span class="refDoi"> DOI: 10.1002/cbic.201000378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fcbic.201000378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=20821791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVKksbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=2104-2107&author=L.+K.+Hencheyauthor=J.+R.+Porterauthor=I.+Ghoshauthor=P.+S.+Arora&title=High+Specificity+in+Protein+Recognition+by+Hydrogen-Bond-Surrogate+%CE%B1-Helices%3A+Selective+Inhibition+of+the+p53%2FMDM2+Complex&doi=10.1002%2Fcbic.201000378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">High specificity in protein recognition by hydrogen-bond-surrogate α-helices: selective inhibition of the p53/MDM2 complex</span></div><div class="casAuthors">Henchey, Laura K.; Porter, Jason R.; Ghosh, Indraneel; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2104-2107</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthetic helixes target chosen protein-protein interactions with high specificity, as assessed by an in vitro split-protein reassembly assay.  By using the p53/MDM2 complex as a model system, both the efficacy of the designed ligands and the suitability of the split-protein assay were detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCog0s7HPIxrVg90H21EOLACvtfcHk0ljgCvK9rAaIPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVKksbfK&md5=e574ed4e820a96e900c31b37e80d87ce</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201000378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201000378%26sid%3Dliteratum%253Aachs%26aulast%3DHenchey%26aufirst%3DL.%2BK.%26aulast%3DPorter%26aufirst%3DJ.%2BR.%26aulast%3DGhosh%26aufirst%3DI.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DHigh%2520Specificity%2520in%2520Protein%2520Recognition%2520by%2520Hydrogen-Bond-Surrogate%2520%25CE%25B1-Helices%253A%2520Selective%2520Inhibition%2520of%2520the%2520p53%252FMDM2%2520Complex%26jtitle%3DChemBioChem%26date%3D2010%26volume%3D11%26spage%3D2104%26epage%3D2107%26doi%3D10.1002%2Fcbic.201000378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Bcl-2/MDM2 Dual Inhibitors Based on Universal Pyramid-Like α-Helical Mimetics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3152</span>– <span class="NLM_lpage">3162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01913</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01913" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Cns78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3152-3162&author=Z.+Wangauthor=T.+Songauthor=Y.+Fengauthor=Z.+Guoauthor=Y.+Fanauthor=W.+Xuauthor=L.+Liuauthor=A.+Wangauthor=Z.+Zhang&title=Bcl-2%2FMDM2+Dual+Inhibitors+Based+on+Universal+Pyramid-Like+%CE%B1-Helical+Mimetics&doi=10.1021%2Facs.jmedchem.5b01913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Bcl-2/MDM2 Dual Inhibitors Based on Universal Pyramid-Like α-Helical Mimetics</span></div><div class="casAuthors">Wang, Ziqian; Song, Ting; Feng, Yingang; Guo, Zongwei; Fan, Yudan; Xu, Wenjie; Liu, Lu; Wang, Anhui; Zhang, Zhichao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3152-3162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">No α-helical mimetic that exhibits Bcl-2/MDM2 dual inhibition has been rationally designed due to the different helicities of the α-helixes at their binding interfaces.  Herein, the authors extd. a one-turn α-helix-mimicking ortho-triarene unit from o-phenylene foldamers.  Linking benzamide substrates with a rotatable C-N bond, the authors constructed a novel semirigid pyramid-like scaffold that could support its two-turn α-helix mimicry without arom. stacking interactions and could adopt the different dihedral angles of the key residues of p53 and BH3-only peptides.  On the basis of this universal scaffold, a series of substituent groups were installed to capture the key residues of both p53TAD and BimBH3 and balance the differences of the bulks between them.  Identified by FP, ITC, and NMR spectroscopy, compd. I that directly binds to Mcl-1, Bcl-2, and MDM2 with balanced submicromolar affinities was obtained.  Cell-based expts. demonstrated its antitumor ability through Bcl-2/MDM2 dual inhibition simultaneously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGb4PhGSW0_rVg90H21EOLACvtfcHk0ljgCvK9rAaIPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Cns78%253D&md5=d489ba0172d4ee96622f79eefb88955d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01913%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DT.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DBcl-2%252FMDM2%2520Dual%2520Inhibitors%2520Based%2520on%2520Universal%2520Pyramid-Like%2520%25CE%25B1-Helical%2520Mimetics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3152%26epage%3D3162%26doi%3D10.1021%2Facs.jmedchem.5b01913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patgiri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joy, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Nucleation effects in peptide foldamers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">11495</span>– <span class="NLM_lpage">11502</span>, <span class="refDoi"> DOI: 10.1021/ja301953j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja301953j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFartr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=11495-11502&author=A.+Patgiriauthor=S.+T.+Joyauthor=P.+S.+Arora&title=Nucleation+effects+in+peptide+foldamers&doi=10.1021%2Fja301953j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleation Effects in Peptide Foldamers</span></div><div class="casAuthors">Patgiri, Anupam; Joy, Stephen T.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">11495-11502</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oligomers composed of β3-amino acid residues and a mixt. of α- and β3-residues have emerged as proteolytically stable structural mimics of α-helixes.  An attractive feature of these oligomers is that they adopt defined conformations in short sequences.  In this manuscript, we evaluate the impact of β3-residues as compared to their α-amino acid analogs in prenucleated helixes.  Our hydrogen-deuterium exchange results suggest that heterogeneous sequences composed of "αααβ" repeats are conformationally more rigid than the corresponding homogeneous α-peptide helixes, with the macrocycle templating the helical conformation having a significant influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjAfMlp3dZDrVg90H21EOLACvtfcHk0ljgCvK9rAaIPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFartr4%253D&md5=50fb6403d303dd9da4b30091623e9594</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja301953j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja301953j%26sid%3Dliteratum%253Aachs%26aulast%3DPatgiri%26aufirst%3DA.%26aulast%3DJoy%26aufirst%3DS.%2BT.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DNucleation%2520effects%2520in%2520peptide%2520foldamers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D11495%26epage%3D11502%26doi%3D10.1021%2Fja301953j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prabhakaran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilner, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. J.</span></span> <span> </span><span class="NLM_article-title">Aromatic Oligoamide Foldamers with a ″Wet Edge″ as Inhibitors of the α-Helix-Mediated p53-hDM2 Protein-Protein Interaction</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2013</i></span>,  <span class="NLM_fpage">3504</span>– <span class="NLM_lpage">3512</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201300069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fejoc.201300069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnt1ShsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2013&publication_year=2013&pages=3504-3512&author=P.+Prabhakaranauthor=A.+Barnardauthor=N.+S.+Murphyauthor=C.+A.+Kilnerauthor=T.+A.+Edwardsauthor=A.+J.+Wilson&title=Aromatic+Oligoamide+Foldamers+with+a+%E2%80%B3Wet+Edge%E2%80%B3+as+Inhibitors+of+the+%CE%B1-Helix-Mediated+p53-hDM2+Protein-Protein+Interaction&doi=10.1002%2Fejoc.201300069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Aromatic Oligoamide Foldamers with a "Wet Edge" as Inhibitors of the α-Helix-Mediated p53-hDM2 Protein-Protein Interaction</span></div><div class="casAuthors">Prabhakaran, Panchami; Barnard, Anna; Murphy, Natasha S.; Kilner, Colin A.; Edwards, Thomas A.; Wilson, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2013</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3504-3512</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This paper describes the design, synthesis and structural anal. of a 3-O-alkylated arom. oligoamide that incorporates an addnl. hydrophilic 6-O-alkyl substituent in the central monomer.  This oligomer exhibits low μ inhibitory potency against the p53-hDM2 interaction compared with its unfunctionalized counterpart and significantly improved soly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtLUYowqcD37Vg90H21EOLACvtfcHk0lhsylTk3gUDQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnt1ShsLo%253D&md5=58a1e780479683205d34aa93ce413c7b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201300069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201300069%26sid%3Dliteratum%253Aachs%26aulast%3DPrabhakaran%26aufirst%3DP.%26aulast%3DBarnard%26aufirst%3DA.%26aulast%3DMurphy%26aufirst%3DN.%2BS.%26aulast%3DKilner%26aufirst%3DC.%2BA.%26aulast%3DEdwards%26aufirst%3DT.%2BA.%26aulast%3DWilson%26aufirst%3DA.%2BJ.%26atitle%3DAromatic%2520Oligoamide%2520Foldamers%2520with%2520a%2520%25E2%2580%25B3Wet%2520Edge%25E2%2580%25B3%2520as%2520Inhibitors%2520of%2520the%2520%25CE%25B1-Helix-Mediated%2520p53-hDM2%2520Protein-Protein%2520Interaction%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2013%26volume%3D2013%26spage%3D3504%26epage%3D3512%26doi%3D10.1002%2Fejoc.201300069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saraogi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span> <span> </span><span class="NLM_article-title">α-Helix mimetics as inhibitors of protein–protein interactions</span>. <i>Biochem. Soc. Trans.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1414</span>– <span class="NLM_lpage">1417</span>, <span class="refDoi"> DOI: 10.1042/bst0361414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1042%2Fbst0361414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=19021566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVCgsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=1414-1417&author=I.+Saraogiauthor=A.+D.+Hamilton&title=%CE%B1-Helix+mimetics+as+inhibitors+of+protein%E2%80%93protein+interactions&doi=10.1042%2Fbst0361414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">α-Helix mimetics as inhibitors of protein-protein interactions</span></div><div class="casAuthors">Saraogi, Ishu; Hamilton, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Society Transactions</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1414-1417</span>CODEN:
                <span class="NLM_cas:coden">BCSTB5</span>;
        ISSN:<span class="NLM_cas:issn">0300-5127</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The inhibition of protein-protein interactions using small mols. is a viable approach for the treatment of a range of pathol. conditions that result from a malfunctioning of these interactions.  Our strategy for the design of such agents involves the mimicry of side-chain residues on one face of the α-helix; these residues frequently play a key role in mediating protein-protein interactions.  The first-generation terphenyl scaffold, with a 3,2',2''-substitution pattern, is able to successfully mimic key helix residues and disrupt therapeutically relevant interactions, including the Bcl-XL-Bak and the p53-hDM2 (human double minute 2) interactions that are implicated in cancer.  The second- and third-generation scaffolds have resulted in greater synthetic accessibility and more drug-like character in these mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCNfqRdM67vrVg90H21EOLACvtfcHk0lhsylTk3gUDQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVCgsb3P&md5=0cc8778ddb2529e3b1188291f7ab074d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1042%2Fbst0361414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbst0361414%26sid%3Dliteratum%253Aachs%26aulast%3DSaraogi%26aufirst%3DI.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26atitle%3D%25CE%25B1-Helix%2520mimetics%2520as%2520inhibitors%2520of%2520protein%25E2%2580%2593protein%2520interactions%26jtitle%3DBiochem.%2520Soc.%2520Trans.%26date%3D2008%26volume%3D36%26spage%3D1414%26epage%3D1417%26doi%3D10.1042%2Fbst0361414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interactions: lessons from p53/MDM2</span>. <i>Biopol.—Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">657</span>– <span class="NLM_lpage">686</span>, <span class="refDoi"> DOI: 10.1002/bip.20741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fbip.20741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17427181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2007&pages=657-686&author=J.+K.+Murrayauthor=S.+H.+Gellman&title=Targeting+protein-protein+interactions%3A+lessons+from+p53%2FMDM2&doi=10.1002%2Fbip.20741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions: lessons from p53/MDM2</span></div><div class="casAuthors">Murray, Justin K.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">657-686</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The tremendous challenge of inhibiting therapeutically important protein-protein interactions has created the opportunity to extend traditional medicinal chem. to a new class of targets and to explore nontraditional strategies.  Here we review a widely studied system, the interaction between tumor suppressor p53 and its natural antagonist MDM2, for which both traditional and nontraditional approaches have been reported.  This system has been a testing ground for novel proteomimetic scaffold-based strategies, i.e., for attempts to mimic the recognition surface displayed by a folded protein with unnatural oligomers.  Retroinverso peptides, peptoids, terphenyls, β-hairpins, p-oligobenzamides, β-peptides, and miniproteins have all been explored as inhibitors of the p53/MDM2 interaction, and we focus on these oligomer-based efforts.  Traditional approaches have been successful as well, and we briefly review small mol. inhibitors along with other strategies for reactivation of the p53 pathway, for comparison with oligomer- based approaches.  We close with comments on an emerging dichotomy among protein-protein interaction targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy7SUsGRc4kbVg90H21EOLACvtfcHk0lhsylTk3gUDQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFKms7nP&md5=8a70d8e515d8cae1284b9c3d1484eee3</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1002%2Fbip.20741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.20741%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DJ.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DTargeting%2520protein-protein%2520interactions%253A%2520lessons%2520from%2520p53%252FMDM2%26jtitle%3DBiopol.%25E2%2580%2594Pept.%2520Sci.%26date%3D2007%26volume%3D88%26spage%3D657%26epage%3D686%26doi%3D10.1002%2Fbip.20741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazgier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span> <span> </span><span class="NLM_article-title">An Ultrahigh Affinityd-Peptide Antagonist Of MDM2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6237</span>– <span class="NLM_lpage">6241</span>, <span class="refDoi"> DOI: 10.1021/jm3005465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3005465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6237-6241&author=C.+Zhanauthor=L.+Zhaoauthor=X.+Weiauthor=X.+Wuauthor=X.+Chenauthor=W.+Yuanauthor=W.-Y.+Luauthor=M.+Pazgierauthor=W.+Lu&title=An+Ultrahigh+Affinityd-Peptide+Antagonist+Of+MDM2&doi=10.1021%2Fjm3005465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">An Ultrahigh Affinity d-Peptide Antagonist Of MDM2</span></div><div class="casAuthors">Zhan, Changyou; Zhao, Le; Wei, Xiaoli; Wu, Xueji; Chen, Xishan; Yuan, Weirong; Lu, Wei-Yue; Pazgier, Marzena; Lu, Wuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6237-6241</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncoprotein MDM2 neg. regulates the activity and stability of the p53 tumor suppressor and is an important mol. target for anticancer therapy.  Aided by mirror image phage display and native chem. ligation, we have previously discovered several proteolysis-resistant duodecimal d-peptide antagonists of MDM2, termed DPMI-α, β, γ.  The prototypic d-peptide inhibitor DPMI-α binds (25-109)MDM2 at an affinity of 220 nM and kills tumor cells in vitro and inhibits tumor growth in vivo by reactivating the p53 pathway.  Herein, we report the design of a superactive d-peptide antagonist of MDM2, termed DPMI-δ, of which the binding affinity for (25-109)MDM2 has been improved over DPMI-α by 3 orders of magnitude (Kd = 220 pM).  X-ray crystallog. studies validate DPMI-δ as an exceedingly potent inhibitor of the p53-MDM2 interaction, promising to be a highly attractive lead drug candidate for anticancer therapeutic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ_Li8SCEm3rVg90H21EOLACvtfcHk0lh1bK2JUPe6Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsL4%253D&md5=5dd937d5023470c7802f68dc7420af55</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm3005465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3005465%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DW.-Y.%26aulast%3DPazgier%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26atitle%3DAn%2520Ultrahigh%2520Affinityd-Peptide%2520Antagonist%2520Of%2520MDM2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6237%26epage%3D6241%26doi%3D10.1021%2Fjm3005465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span> <span> </span><span class="NLM_article-title">Awakening guardian angels: drugging the p53 pathway</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">873</span>, <span class="refDoi"> DOI: 10.1038/nrc2763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnrc2763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=19935675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=862-873&author=C.+J.+Brownauthor=S.+Lainauthor=C.+S.+Vermaauthor=A.+R.+Fershtauthor=D.+P.+Lane&title=Awakening+guardian+angels%3A+drugging+the+p53+pathway&doi=10.1038%2Fnrc2763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Awakening guardian angels: Drugging the p53 pathway</span></div><div class="casAuthors">Brown, Christopher J.; Lain, Sonia; Verma, Chandra S.; Fersht, Alan R.; Lane, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">862-873</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Currently, around 11 million people are living with a tumor that contains an inactivating mutation of TP53 (the human gene that encodes p53) and another 11 million have tumors in which the p53 pathway is partially abrogated through the inactivation of other signaling or effector components.  The p53 pathway is therefore a prime target for new cancer drug development, and several original approaches to drug discovery that could have wide applications to drug development are being used.  In one approach, mols. that activate p53 by blocking protein-protein interactions with MDM2 are in early clin. development.  Remarkable progress has also been made in the development of p53-binding mols. that can rescue the function of certain p53 mutants.  Finally, cell-based assays are being used to discover compds. that exploit the p53 pathway by either seeking targets and compds. that show synthetic lethality with TP53 mutations or by looking for non-genotoxic activators of the p53 response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZW-elIe4w6bVg90H21EOLACvtfcHk0lh1bK2JUPe6Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVGlur%252FM&md5=072f48d2ed466b2deb9d2b999cc65a53</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnrc2763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2763%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DC.%2BJ.%26aulast%3DLain%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DLane%26aufirst%3DD.%2BP.%26atitle%3DAwakening%2520guardian%2520angels%253A%2520drugging%2520the%2520p53%2520pathway%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D862%26epage%3D873%26doi%3D10.1038%2Fnrc2763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilkes, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Distinct roles of MDMX in the regulation of p53 response to ribosomal stress</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">155</span>, <span class="refDoi"> DOI: 10.4161/cc.6.2.3719</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.4161%2Fcc.6.2.3719" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17327702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvFClsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=151-155&author=D.+M.+Gilkesauthor=J.+Chen&title=Distinct+roles+of+MDMX+in+the+regulation+of+p53+response+to+ribosomal+stress&doi=10.4161%2Fcc.6.2.3719"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct roles of MDMX in the regulation of p53 response to ribosomal stress</span></div><div class="casAuthors">Gilkes, Daniele M.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">151-155</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">A review.  Approx. 50% of protein and RNA synthesis in prolifering cells are devoted to ribosomal biogenesis.  Coordination between ribosome biogenesis, growth, and proliferation is crit. for maintenance of homeostasis and tumor suppression.  Aberrant rRNA expression and processing is sensed by p53.  Ribosomal stress increases the binding between MDM2 and ribosomal proteins L5, L11, and L23, resulting in p53 stabilization.  Our recent study showed that p53 activation by ribosomal stress also involves degrdn. of MDMX in an MDM2-dependent fashion.  Failure to eliminate MDMX due to overexpression results in the sequestration of p53 into inactive complexes, severely impairing p53-dependent cell cycle arrest during ribosomal stress.  Furthermore, MDMX overexpression promotes resistance to the chemotherapeutic agent 5-FU, which at low concns. activates p53 by inhibiting RNA metab.  Therefore, MDMX is an important regulator of p53 response to ribosomal stress.  MDMX overexpression in tumors may significantly influence response to chemotherapy agents that target rRNA biogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxs1hKcUKyiLVg90H21EOLACvtfcHk0lh1bK2JUPe6Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvFClsLo%253D&md5=2b0b1c5192fd8ca2048b40a7764628f9</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.4161%2Fcc.6.2.3719&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.6.2.3719%26sid%3Dliteratum%253Aachs%26aulast%3DGilkes%26aufirst%3DD.%2BM.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DDistinct%2520roles%2520of%2520MDMX%2520in%2520the%2520regulation%2520of%2520p53%2520response%2520to%2520ribosomal%2520stress%26jtitle%3DCell%2520Cycle%26date%3D2007%26volume%3D6%26spage%3D151%26epage%3D155%26doi%3D10.4161%2Fcc.6.2.3719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schepartz, A.</span></span> <span> </span><span class="NLM_article-title">In Silico Improvement of β3-Peptide Inhibitors of p53hDM2 and p53hDMX</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">6356</span>– <span class="NLM_lpage">6357</span>, <span class="refDoi"> DOI: 10.1021/ja901478e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja901478e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvVSlt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=6356-6357&author=J.+Michelauthor=E.+A.+Harkerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensenauthor=A.+Schepartz&title=In+Silico+Improvement+of+%CE%B23-Peptide+Inhibitors+of+p53hDM2+and+p53hDMX&doi=10.1021%2Fja901478e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">In Silico Improvement of β3-Peptide Inhibitors of p53·hDM2 and p53·hDMX</span></div><div class="casAuthors">Michel, Julien; Harker, Elizabeth A.; Tirado-Rives, Julian; Jorgensen, William L.; Schepartz, Alanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6356-6357</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is great interest in mols. capable of inhibiting the interactions between p53 and its neg. regulators hDM2 and hDMX, as these mols. have validated potential against cancers in which one or both oncoproteins are overexpressed.  We reported previously that appropriately substituted β3-peptides inhibit these interactions and, more recently, that minimally cationic β3-peptides are sufficiently cell permeable to upregulate p53-dependent genes in live cells.  These observations, coupled with the known stability of β-peptides in a cellular environment, and the recently reported structures of hDM2 and hDMX, motivated us to exploit computational modeling to identify β-peptides with improved potency and/or selectivity.  This exercise successfully identified a new β3-peptide, β53-16, that possesses the highly desirable attribute of high affinity for both hDM2 and hDMX and identifies the 3,4-dichlorophenyl moiety as a novel determinant of hDMX affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcG-5UtJiWA7Vg90H21EOLACvtfcHk0lgwalwgaweRRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvVSlt78%253D&md5=6bc5ba84baded67a3edae0bb7c9c2932</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fja901478e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja901478e%26sid%3Dliteratum%253Aachs%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DHarker%26aufirst%3DE.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DSchepartz%26aufirst%3DA.%26atitle%3DIn%2520Silico%2520Improvement%2520of%2520%25CE%25B23-Peptide%2520Inhibitors%2520of%2520p53hDM2%2520and%2520p53hDMX%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D6356%26epage%3D6357%26doi%3D10.1021%2Fja901478e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasprzak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of high affinity peptides inhibiting the interaction of p53 with MDM2 and MDMX</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">2174</span>– <span class="NLM_lpage">2183</span>, <span class="refDoi"> DOI: 10.1074/jbc.m109.073056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1074%2Fjbc.m109.073056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=19910468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjsVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=2174-2183&author=J.+Phanauthor=Z.+Liauthor=A.+Kasprzakauthor=B.+Liauthor=S.+Sebtiauthor=W.+Guidaauthor=E.+Sch%C3%B6nbrunnauthor=J.+Chen&title=Structure-based+design+of+high+affinity+peptides+inhibiting+the+interaction+of+p53+with+MDM2+and+MDMX&doi=10.1074%2Fjbc.m109.073056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based Design of High Affinity Peptides Inhibiting the Interaction of p53 with MDM2 and MDMX</span></div><div class="casAuthors">Phan, Jason; Li, Zhenyu; Kasprzak, Agnieszka; Li, Baozong; Sebti, Said; Guida, Wayne; Schoenbrunn, Ernst; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">2174-2183</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">MDM2 and MDMX function as key regulators of p53 by binding to its N terminus, inhibiting its transcriptional activity, and promoting degrdn.  MDM2 and MDMX overexpression or hyperactivation directly contributes to the loss of p53 function during the development of nearly 50% of human cancers.  Recent studies showed that disrupting p53-MDM2 and p53-MDMX interactions can lead to robust activation of p53 but also revealed a need to develop novel dual specific or MDMX-specific inhibitors.  Using phage display we identified a 12-residue peptide (pDI) with inhibitory activity against MDM2 and MDMX.  The co-crystal structures of the pDI and a single mutant deriv. (pDI6W) liganded with the N-terminal domains of human MDMX and MDM2 served as the basis for the design of 11 distinct pDI-deriv. peptides that were tested for inhibitory potential.  The best deriv. (termed pDIQ) contained four amino acid substitutions and exhibited a 5-fold increase in potency over the parent peptide against both MDM2 (IC50 = 8 nM) and MDMX (IC50 = 110 nM).  Further structural studies revealed key mol. features enabling the high affinity binding of the pDIQ to these proteins.  These include large conformational changes of the pDIQ to reach into a hydrophobic site unique to MDMX.  The findings suggest new strategies toward the rational design of small mol. inhibitors efficiently targeting MDMX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajV2RNjc7GbVg90H21EOLACvtfcHk0lgwalwgaweRRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjsVSquw%253D%253D&md5=b9022411e67fb66bbb04b5de7a1ef426</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m109.073056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m109.073056%26sid%3Dliteratum%253Aachs%26aulast%3DPhan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DKasprzak%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DGuida%26aufirst%3DW.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DStructure-based%2520design%2520of%2520high%2520affinity%2520peptides%2520inhibiting%2520the%2520interaction%2520of%2520p53%2520with%2520MDM2%2520and%2520MDMX%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D2174%26epage%3D2183%26doi%3D10.1074%2Fjbc.m109.073056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">p53 inactivation by MDM2 and MDMX negative feedback loops in testicular germ cell tumors</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1411</span>– <span class="NLM_lpage">1420</span>, <span class="refDoi"> DOI: 10.4161/cc.9.7.11255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.4161%2Fcc.9.7.11255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=20372076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ktbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=1411-1420&author=B.+Liauthor=Q.+Chengauthor=Z.+Liauthor=J.+Chen&title=p53+inactivation+by+MDM2+and+MDMX+negative+feedback+loops+in+testicular+germ+cell+tumors&doi=10.4161%2Fcc.9.7.11255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">p53 inactivation by MDM2 and MDMX negative feedback loops in testicular germ cell tumors</span></div><div class="casAuthors">Li, Baozong; Cheng, Qian; Li, Zhenyu; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1411-1420</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Testicular germ cell tumors (TGCT) are unique in their excellent response to DNA-damaging chemotherapy.  Mutation of p53 is rare in both untreated and relapsed TGCTs, suggesting that p53 fails to respond effectively against malignant transformation in germ cells.  Previous studies implicated the presence of a poorly defined TGCT-specific mechanism of p53 inactivation.  Here we show that disruption of p53-mdm2 binding using the MDM2-specific inhibitor Nutlin activates p53 in TGCT cells and is sufficient to induce strong apoptosis.  Knockdown of MDMX cooperates with Nutlin to activate p53.  Surprisingly, we found that p53 activation induced a two-fold increase in MDMX mRNA and protein expression in TGCT cells.  A p53-responsive promoter is identified in MDMX intron 1 that contains a functional p53-binding site, suggesting that MDMX also functions as a neg. feedback regulator of p53 in a cell line-dependent fashion.  These findings suggest that MDM2 and MDMX are responsible for the functional inactivation of p53 in TGCT.  Furthermore, TGCT cells are unique in having a strong apoptosis response to p53.  Direct activation of p53 by targeting MDM2 and MDMX may provide a backup approach for the treatment of TGCTs resistant to DNA-damaging drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoApVK333wJULVg90H21EOLACvtfcHk0lgwalwgaweRRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ktbnJ&md5=e9c7545437b87a8d45d23a03215b2f6a</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.4161%2Fcc.9.7.11255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.9.7.11255%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DCheng%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3Dp53%2520inactivation%2520by%2520MDM2%2520and%2520MDMX%2520negative%2520feedback%2520loops%2520in%2520testicular%2520germ%2520cell%2520tumors%26jtitle%3DCell%2520Cycle%26date%3D2010%26volume%3D9%26spage%3D1411%26epage%3D1420%26doi%3D10.4161%2Fcc.9.7.11255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">The synthesis of head-to-tail cyclic sulfono-γ-AApeptides</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1039/c4ob02232g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc4ob02232g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=25420701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=672-676&author=H.+Wuauthor=F.+Sheauthor=W.+Gaoauthor=A.+Princeauthor=Y.+Liauthor=L.+Weiauthor=A.+Mercerauthor=L.+Wojtasauthor=S.+Maauthor=J.+Cai&title=The+synthesis+of+head-to-tail+cyclic+sulfono-%CE%B3-AApeptides&doi=10.1039%2Fc4ob02232g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">The synthesis of head-to-tail cyclic sulfono-γ-AApeptides</span></div><div class="casAuthors">Wu, Haifan; She, Fengyu; Gao, Wenyang; Prince, Austin; Li, Yaqiong; Wei, Lulu; Mercer, Allison; Wojtas, Lukasz; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">672-676</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report an efficient method for the prepn. of unprecedented head-to-tail cyclic sulfono-γ-AApeptides.  Following this method, a no. of cyclic sequences bearing two to five subunits were efficiently synthesized.  In addn., the X-ray crystal structure study of a three-membered cyclic sulfono-γ-AApeptide revealed a type II β-turn-like character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNwhl4FWEKMbVg90H21EOLACvtfcHk0lib_lY2NlqYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2jtrnN&md5=d0b2e2b03ad122bd15ffdbf21542b380</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1039%2Fc4ob02232g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4ob02232g%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DPrince%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMercer%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DThe%2520synthesis%2520of%2520head-to-tail%2520cyclic%2520sulfono-%25CE%25B3-AApeptides%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D672%26epage%3D676%26doi%3D10.1039%2Fc4ob02232g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Rapid Access to Multiple Classes of Peptidomimetics from Common γ-AApeptide Building Blocks</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>,  <span class="NLM_fpage">1760</span>– <span class="NLM_lpage">1765</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201301841</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fejoc.201301841" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFOhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&pages=1760-1765&author=H.+Wuauthor=P.+Tengauthor=J.+Cai&title=Rapid+Access+to+Multiple+Classes+of+Peptidomimetics+from+Common+%CE%B3-AApeptide+Building+Blocks&doi=10.1002%2Fejoc.201301841"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid access to multiple classes of peptidomimetics from common γ-AApeptide building blocks</span></div><div class="casAuthors">Wu, Haifan; Teng, Peng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1760-1765</span>CODEN:
                <span class="NLM_cas:coden">EJOCFK</span>;
        ISSN:<span class="NLM_cas:issn">1099-0690</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Herein we report a highly efficient new method for prepg. γ-AApeptides capable of theor. contg. any functionality; the method uses a few common N-alloc γ-AApeptide building blocks.  More importantly, using the same approach, new classes of peptidomimetics bearing novel backbone scaffolds can also be readily generated.  Our results not only demonstrate the versatility of this new synthetic method, but also highlight the possibility that the resulting novel peptidomimetics may find discrete biomedical/biomaterial applications in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppIRJmB8Y14LVg90H21EOLACvtfcHk0lib_lY2NlqYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFOhsg%253D%253D&md5=5c74d803796da979aad31546baa4053e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201301841&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201301841%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRapid%2520Access%2520to%2520Multiple%2520Classes%2520of%2520Peptidomimetics%2520from%2520Common%2520%25CE%25B3-AApeptide%2520Building%2520Blocks%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2014%26volume%3D2014%26spage%3D1760%26epage%3D1765%26doi%3D10.1002%2Fejoc.201301841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: Design, Structure, and Applications</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">428</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=428-441&author=Y.+Shiauthor=P.+Tengauthor=P.+Sangauthor=F.+Sheauthor=L.+Weiauthor=J.+Cai&title=%CE%B3-AApeptides%3A+Design%2C+Structure%2C+and+Applications&doi=10.1021%2Facs.accounts.5b00492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: Design, structure, and applications</span></div><div class="casAuthors">Shi, Yan; Teng, Peng; Sang, Peng; She, Fengyu; Wei, Lulu; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">428-441</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The development of sequence-specific peptidomimetics has led to a variety of fascinating discoveries in chem. biol.  Many peptidomimetics can mimic primary, secondary, and even tertiary structure of peptides and proteins, and because of their unnatural backbones, they also possess significantly enhanced resistance to enzymic hydrolysis, improved bioavailability, and chemodiversity.  It is known that peptide nucleic acids (PNAs) are peptidic sequences developed for the mimicry of nucleic acids; however, their unique backbone as the mol. scaffold of peptidomimetics to mimic structure and function of bioactive peptides has not been investigated systematically.  As such, we recently developed a new class of peptidomimetics, "γ-AApeptides", based on the chiral γ-PNA backbone.  They are termed γ-AApeptides because they are the oligomers of γ-substituted-N-acylated-N-aminoethyl amino acids.  Similar to other classes of peptidomimetics, γ-AApeptides are also resistant to proteolytic degrdn. and possess the potential to enhance chemodiversity.  Moreover, in our scientific journey on the exploration of this class of peptidomimetics, we have discovered some intriguing structures and functions of γ-AApeptides.  In this account, we summarize the current development and application of γ-AApeptides with biol. potential.  Briefly, both linear and cyclic (either through head-to-tail or head-to-side-chain cyclization) γ-AApeptides with diverse functional groups can be synthesized easily on the solid phase using the synthetic protocol we developed. γ-AApeptides could mimic the primary structure of peptides, as they project the same no. of side chains as peptides of the same lengths.  For instance, they could mimic the Tat peptide to permeate cell membranes and bind to HIV RNA with high specificity and affinity.  Certain γ-AApeptides show similar activity to the RGD peptide and target integrin specifically on the cell surface.  γ-AApeptides with function akin to fMLF peptides are also identified.  More importantly, we found that γ-AApeptides can fold into discrete secondary structures, such as helical and β-turn-like structures.  Therefore, they could be rationally designed for a range of biol. applications.  For instance, γ-AApeptides can mimic host-defense peptides and display potent and broad-spectrum activity toward a panel of drug-resistant bacterial pathogens.  Meanwhile, because of their stability against proteolysis and their chemodiversity, γ-AApeptides are also amenable for combinatorial screening.  We demonstrate that, through combinatorial selection, certain γ-AApeptides are identified to inhibit Aβ40 peptide aggregation, suggesting their potential use as a mol. probe to intervene in Alzheimer's disease.  In addn., a few γ-AApeptides identified from the γ-AApeptide library have been shown to bind to the DNA-binding domain of STAT3 and antagonize STAT3/DNA interactions.  Our studies suggest that, with further studies and exploration on both structures and functions, γ-AApeptides may emerge to be a new class of peptidomimetics that play an important role in chem. biol. and biomedical sciences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrk8PLbKcOxrVg90H21EOLACvtfcHk0lib_lY2NlqYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVWkt7w%253D&md5=414ab3568ccfdcebb5863d876218804c</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00492%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520Design%252C%2520Structure%252C%2520and%2520Applications%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D428%26epage%3D441%26doi%3D10.1021%2Facs.accounts.5b00492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides as a New Class of Peptidomimetics</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">5458</span>– <span class="NLM_lpage">5466</span>, <span class="refDoi"> DOI: 10.1002/chem.201504936</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fchem.201504936" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=26945679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=5458-5466&author=P.+Tengauthor=Y.+Shiauthor=P.+Sangauthor=J.+Cai&title=%CE%B3-AApeptides+as+a+New+Class+of+Peptidomimetics&doi=10.1002%2Fchem.201504936"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides as a New Class of Peptidomimetics</span></div><div class="casAuthors">Teng, Peng; Shi, Yan; Sang, Peng; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5458-5466</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Sequence-specific peptidomimetics are mols. that mimic the structure and function of peptides and proteins.  With new backbones and mol. frameworks, peptidomimetics are of considerable interest in addressing challenges encountered in chem. biol. and biomedical sciences.  Based on the γ-PNA backbone, a new class of peptidomimetics - "γ-AApeptides" were recently developed.  Both linear and cyclic γ-AApeptides can be synthesized with high efficiency.  Compared with α-peptides, γ-AApeptides are resistant to enzymic degrdn., and amendable to diversification with a variety of chem. groups.  Moreover, they could mimic primary and secondary structure, as well as the function of peptides, and show promise in biol. applications, such as the development of new agents combating bacteria, cancer, and Alzheimer's disease.  A few research outcomes of γ-AApeptides are highlighted in this Concept article, and a future perspective is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj0LfZU9KRa7Vg90H21EOLACvtfcHk0li_vV7v92AgcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjslGitbk%253D&md5=6f8385337d870a546e1e6b56fd8ef3f1</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fchem.201504936&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201504936%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%2520as%2520a%2520New%2520Class%2520of%2520Peptidomimetics%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D5458%26epage%3D5466%26doi%3D10.1002%2Fchem.201504936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerrato, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, L.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Right-handed helical foldamers consisting of de novo d-AApeptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">7363</span>– <span class="NLM_lpage">7369</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b03007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b03007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=7363-7369&author=P.+Tengauthor=N.+Maauthor=D.+C.+Cerratoauthor=F.+Sheauthor=T.+Odomauthor=X.+Wangauthor=L.-J.+Mingauthor=A.+van+der%0AVaartauthor=L.+Wojtasauthor=H.+Xuauthor=J.+Cai&title=Right-handed+helical+foldamers+consisting+of+de+novo+d-AApeptides&doi=10.1021%2Fjacs.7b03007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Right-handed helical foldamers consisting of de novo D-AA-peptides</span></div><div class="casAuthors">Teng, Peng; Ma, Ning; Cerrato, Darrell Cole; She, Fengyu; Odom, Timothy; Wang, Xiang; Ming, Li-June; van der Vaart, Arjan; Wojtas, Lukasz; Xu, Hai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7363-7369</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">New types of foldamer scaffolds are formidably challenging to design and synthesize, yet highly desirable as structural mimics of peptides/proteins with a wide repertoire of functions.  In particular, the development of peptidomimetic helical foldamers holds promise for new biomaterials, catalysts, and drug mols.  Unnatural L-sulfono-γ-AApeptides were recently developed and shown to have potential applications in both biomedical and material sciences.  However, D-sulfono-γ-AApeptides, the enantiomers of L-sulfono-γ-AApeptides, have never been studied due to the lack of high-resoln. three-dimensional structures to guide structure-based design.  Herein, we report the first synthesis and X-ray crystal structures of a series of 2:1 L-amino acid/D-sulfono-γ-AApeptide hybrid foldamers, and elucidate their folded conformation at the at. level.  Single-crystal X-ray crystallog. indicates that this class of oligomers folds into well-defined right-handed helixes with unique helical parameters.  The helical structures were consistent with data obtained from soln. 2D NMR, CD studies, and mol. dynamics simulations.  Our findings are expected to inspire the structure-based design of this type of unique folding biopolymers for biomaterials and biomedical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaGg_OJWsqYbVg90H21EOLACvtfcHk0li_vV7v92AgcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Chu7w%253D&md5=fef3f80500953f027687001f3e67f14b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b03007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b03007%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DCerrato%26aufirst%3DD.%2BC.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DMing%26aufirst%3DL.-J.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DRight-handed%2520helical%2520foldamers%2520consisting%2520of%2520de%2520novo%2520d-AApeptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D7363%26epage%3D7369%26doi%3D10.1021%2Fjacs.7b03007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Hydrogen-bonding-driven 3D supramolecular assembly of peptidomimetic zipper</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5661</span>– <span class="NLM_lpage">5665</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b11997</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b11997" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5661-5665&author=P.+Tengauthor=Z.+Niuauthor=F.+Sheauthor=M.+Zhouauthor=P.+Sangauthor=G.+M.+Grayauthor=G.+Vermaauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=S.+Maauthor=J.+Cai&title=Hydrogen-bonding-driven+3D+supramolecular+assembly+of+peptidomimetic+zipper&doi=10.1021%2Fjacs.7b11997"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen-Bonding-Driven 3D Supramolecular Assembly of Peptidomimetic Zipper</span></div><div class="casAuthors">Teng, Peng; Niu, Zheng; She, Fengyu; Zhou, Mi; Sang, Peng; Gray, Geoffrey M.; Verma, Gaurav; Wojtas, Lukasz; van der Vaart, Arjan; Ma, Shengqian; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5661-5665</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hydrogen-bonding-driven three-dimensional (3D) assembly of a peptidomimetic zipper has been established for the first time by using an α/AApeptide zipper that assembles into a de novo lattice arrangement through two layers of hydrogen-bonded linker-directed interactions.  Via a covalently bridged 1D 413-helix, drastic enhancement in stability has been achieved in the formed 3D cryst. supramol. architecture as evidenced by gas-sorption studies.  As the first example of an unnatural peptidic zipper, the dimensional augmentation of the zipper differs from metal-coordinated strategies, and may have general implications for the prepn. of peptidic functional materials for a variety of future applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6NOPVSQVnN7Vg90H21EOLACvtfcHk0li_vV7v92AgcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsLY%253D&md5=75ec49a44f96107f9a2472698475157d</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b11997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b11997%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DNiu%26aufirst%3DZ.%26aulast%3DShe%26aufirst%3DF.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DSang%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHydrogen-bonding-driven%25203D%2520supramolecular%2520assembly%2520of%2520peptidomimetic%2520zipper%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5661%26epage%3D5665%26doi%3D10.1021%2Fjacs.7b11997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7778</span>– <span class="NLM_lpage">7782</span>, <span class="refDoi"> DOI: 10.1002/anie.201903259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201903259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=7778-7782&author=P.+Tengauthor=G.+M.+Grayauthor=M.+Zhengauthor=S.+Singhauthor=X.+Liauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=Orthogonal+Halogen-Bonding-Driven+3D+Supramolecular+Assembly+of+Right-Handed+Synthetic+Helical+Peptides&doi=10.1002%2Fanie.201903259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Orthogonal Halogen-Bonding-Driven 3D Supramolecular Assembly of Right-Handed Synthetic Helical Peptides</span></div><div class="casAuthors">Teng, Peng; Gray, Geoffrey M.; Zheng, Mengmeng; Singh, Sylvia; Li, Xiaopeng; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7778-7782</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide-mediated self-assembly is a prevalent method for creating highly ordered supramol. architectures.  Herein, we report the first example of orthogonal C-X···X-C/C-X···π halogen bonding and hydrogen bonding driven cryst. architectures based on synthetic helical peptides bearing hybrids of L-sulfono-γ-AApeptides and natural amino acids.  The combination of halogen bonding, intra-/intermol. hydrogen bonding, and intermol. hydrophobic interactions enabled novel 3D supramol. assembly.  The orthogonal halogen bonding in the supramol. architecture exerts a novel mechanism for the self-assembly of synthetic peptide foldamers and gives new insights into mol. recognition, supramol. design, and rational design of biomimetic structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphs8_rPqx-zrVg90H21EOLACvtfcHk0lg8pEH6zY8GZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1Sjsbc%253D&md5=c61d38ec3f3a4e3d79d716a1c88a7ec3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fanie.201903259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201903259%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DG.%2BM.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DOrthogonal%2520Halogen-Bonding-Driven%25203D%2520Supramolecular%2520Assembly%2520of%2520Right-Handed%2520Synthetic%2520Helical%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D7778%26epage%3D7782%26doi%3D10.1002%2Fanie.201903259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De novo left-handed synthetic peptidomimetic foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der%0AVaartauthor=J.+Cai&title=De+novo+left-handed+synthetic+peptidomimetic+foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0lg8pEH6zY8GZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520novo%2520left-handed%2520synthetic%2520peptidomimetic%2520foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B cell lymphoma 9 protein–protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+cell+lymphoma+9+protein%E2%80%93protein+interaction+using+%CE%B1-helix-mimicking+sulfono-%CE%B3-AApeptide+inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0lg8pEH6zY8GZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520cell%2520lymphoma%25209%2520protein%25E2%2580%2593protein%2520interaction%2520using%2520%25CE%25B1-helix-mimicking%2520sulfono-%25CE%25B3-AApeptide%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">γ-AApeptides: design, synthesis and evaluation</span>. <i>New J. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1039/c0nj00943a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc0nj00943a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2011&pages=542-545&author=Y.+Niuauthor=Y.+Huauthor=X.+Liauthor=J.+Chenauthor=J.+Cai&title=%CE%B3-AApeptides%3A+design%2C+synthesis+and+evaluation&doi=10.1039%2Fc0nj00943a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">γ-AApeptides: design, synthesis and evaluation</span></div><div class="casAuthors">Niu, You-Hong; Hu, Yao-Gang; Li, Xiao-Long; Chen, Jian-Dong; Cai, Jian-Feng</div><div class="citationInfo"><span class="NLM_cas:title">New Journal of Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">542-545</span>CODEN:
                <span class="NLM_cas:coden">NJCHE5</span>;
        ISSN:<span class="NLM_cas:issn">1144-0546</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A new class of peptide mimics termed "γ-AApeptides" (derived from γ-chiral peptide nucleic acids, PNAs) has been described.  The design and synthesis of γ-AApeptides, and potential bioactivities towards p53/MDM2 interaction were demonstrated. γ-AApeptides were also found to be highly resistant to proteolysis.  The development of sequence-specific γ-AApeptides may lead to a family of peptidomimetics with a new framework for drug discovery or peptide/protein mimicry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJjedI1LYLnLVg90H21EOLACvtfcHk0lhHl5GeP59G7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFKisLw%253D&md5=bcc3c0c6b390ed5bc3b6019ed02191f9</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1039%2Fc0nj00943a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0nj00943a%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3D%25CE%25B3-AApeptides%253A%2520design%252C%2520synthesis%2520and%2520evaluation%26jtitle%3DNew%2520J.%2520Chem.%26date%3D2011%26volume%3D35%26spage%3D542%26epage%3D545%26doi%3D10.1039%2Fc0nj00943a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, S. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, B. K.</span></span> <span> </span><span class="NLM_article-title">A fluorescence polarization assay for the identification of inhibitors of the p53-DM2 protein-protein interaction</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">230</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1006/abio.2001.5468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1006%2Fabio.2001.5468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=11779115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsFekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2002&pages=230-236&author=S.+M.+G.+Knightauthor=N.+Umezawaauthor=H.-S.+Leeauthor=S.+H.+Gellmanauthor=B.+K.+Kay&title=A+fluorescence+polarization+assay+for+the+identification+of+inhibitors+of+the+p53-DM2+protein-protein+interaction&doi=10.1006%2Fabio.2001.5468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">A fluorescence polarization assay for the identification of inhibitors of the p53-DM2 protein-protein interaction</span></div><div class="casAuthors">Knight, Stephen M. G.; Umezawa, Naoki; Lee, Hee-Seung; Gellman, Samuel H.; Kay, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">230-236</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Improper function of the tumor suppressor protein p53 is a contributing factor in many human cancers.  In normal cells, p53 acts to arrest the cell cycle in response to DNA damage or nucleotide depletion.  One mechanism of regulating the amt. of p53 in the cell is through the action of the Double Minute 2 protein, DM2 (also known as MDM2), which ubiquitinates p53 and targets it for proteosomal degrdn.  In a no. of human cancers, the DM2 gene is amplified or overexpressed, leading to inadequate levels of p53 for cell cycle arrest or apoptosis.  With the goal of restoring p53 function in cancers that overexpress DM2, we are developing inhibitors of the p53-DM2 protein-protein interaction that structurally mimic the N-terminal segment of p53 that binds to DM2.  To assist this effort, we have devised a fluorescence polarization assay that quantifies the interaction between the N-terminal regions of both proteins in 384-well microtiter plates.  Using this assay, we have demonstrated that a peptide with a nonhydrolyzable β-amino acid substitution binds DM2 with an affinity comparable to a p53 peptide that is composed of only α-amino acids.  (c) 2002 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpndbiy8bbhfLVg90H21EOLACvtfcHk0lhHl5GeP59G7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsFekuw%253D%253D&md5=9bfe05d43b9c013a1845e1c9e81d035e</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1006%2Fabio.2001.5468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.2001.5468%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DS.%2BM.%2BG.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DLee%26aufirst%3DH.-S.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DKay%26aufirst%3DB.%2BK.%26atitle%3DA%2520fluorescence%2520polarization%2520assay%2520for%2520the%2520identification%2520of%2520inhibitors%2520of%2520the%2520p53-DM2%2520protein-protein%2520interaction%26jtitle%3DAnal.%2520Biochem.%26date%3D2002%26volume%3D300%26spage%3D230%26epage%3D236%26doi%3D10.1006%2Fabio.2001.5468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farooqi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">p53 -Helix mimetics antagonize p53/MDM2 interaction and activate p53</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1019</span>– <span class="NLM_lpage">1025</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-04-0342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1158%2F1535-7163.mct-04-0342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=15956260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1019-1025&author=L.+Chenauthor=H.+Yinauthor=B.+Farooqiauthor=S.+Sebtiauthor=A.+D.+Hamiltonauthor=J.+Chen&title=p53+-Helix+mimetics+antagonize+p53%2FMDM2+interaction+and+activate+p53&doi=10.1158%2F1535-7163.mct-04-0342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">p53 α-Helix mimetics antagonize p53/MDM2 interaction and activate p53</span></div><div class="casAuthors">Chen, Lihong; Yin, Hang; Farooqi, Bilal; Sebti, Said; Hamilton, Andrew D.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1019-1025</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Overexpression or hyperactivation of MDM2 contributes to functional inactivation of wild-type p53 in nearly 50% of tumors.  Inhibition of p53 by MDM2 depends on binding between an NH2-terminal (residues 16-28) p53 α-helical peptide and a hydrophobic pocket on MDM2, presenting an attractive target for development of inhibitors against tumors expressing wild-type p53.  Here we report that novel p53 α-helical peptide mimics based on a terphenyl scaffold can inhibit MDM2-p53 binding in vitro and activate p53 in vivo.  Several active compds. have been identified that inhibit MDM2-p53 binding in an ELISA assay with IC50 of 10 to 20 μmol/L and induce p53 accumulation and activation in cell culture at 15 to 40 μmol/L.  These results suggest that p53 α-helical mimetics based on the terphenyl scaffold may be developed into potent p53 activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNgg0V3He8E7Vg90H21EOLACvtfcHk0lhgkxL_mmLZew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLY%253D&md5=ac7ae39142e67f0a070008c88a71c084</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-04-0342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-04-0342%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DFarooqi%26aufirst%3DB.%26aulast%3DSebti%26aufirst%3DS.%26aulast%3DHamilton%26aufirst%3DA.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3Dp53%2520-Helix%2520mimetics%2520antagonize%2520p53%252FMDM2%2520interaction%2520and%2520activate%2520p53%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2005%26volume%3D4%26spage%3D1019%26epage%3D1025%26doi%3D10.1158%2F1535-7163.mct-04-0342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartl, D. L.</span></span> <span> </span><span class="NLM_article-title">Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">6504</span>– <span class="NLM_lpage">6510</span>, <span class="refDoi"> DOI: 10.1073/pnas.0701572104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1073%2Fpnas.0701572104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=17409186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkvFalsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=6504-6510&author=S.+A.+Sawyerauthor=J.+Parschauthor=Z.+Zhangauthor=D.+L.+Hartl&title=Prevalence+of+positive+selection+among+nearly+neutral+amino+acid+replacements+in+Drosophila&doi=10.1073%2Fpnas.0701572104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of positive selection among nearly neutral amino acid replacements in Drosophila</span></div><div class="casAuthors">Sawyer, Stanley A.; Parsch, John; Zhang, Zhi; Hartl, Daniel L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6504-6510</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We have estd. the selective effects of amino acid replacements in natural populations by comparing levels of polymorphism in 91 genes in African populations of Drosophila melanogaster with their divergence from Drosophila simulans.  The genes include about equal nos. whose level of expression in adults is greater in males, greater in females, or approx. equal in the sexes.  Markov chain Monte Carlo methods were used to sample key parameters in the stationary distribution of polymorphism and divergence in a model in which the selective effect of each nonsynonymous mutation is regarded as a random sample from some underlying normal distribution whose mean may differ from one gene to the next.  Our anal. suggests that ≈95% of all nonsynonymous mutations that could contribute to polymorphism or divergence are deleterious, and that the av. proportion of deleterious amino acid polymorphisms in samples is ≈70%.  On the other hand, ≈95% of fixed differences between species are pos. selected; although, the scaled selection coeff. (Nes) is very small.  We est. that ≈46% of amino acid replacements have Nes < 2, ≈84% have Nes < 4, and ≈99% have Nes < 7.  Although pos. selection among amino acid differences between species seems pervasive, most of the selective effects could be regarded as nearly neutral.  There are significant differences in selection between sex-biased and unbiased genes, which relate primarily to the mean of the distributions of mutational effects and the fraction of slightly deleterious and weakly beneficial mutations that are fixed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyGis9gs3gO7Vg90H21EOLACvtfcHk0lhgkxL_mmLZew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkvFalsbg%253D&md5=1ddc72b1a6694d699214611587b9015e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0701572104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0701572104%26sid%3Dliteratum%253Aachs%26aulast%3DSawyer%26aufirst%3DS.%2BA.%26aulast%3DParsch%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHartl%26aufirst%3DD.%2BL.%26atitle%3DPrevalence%2520of%2520positive%2520selection%2520among%2520nearly%2520neutral%2520amino%2520acid%2520replacements%2520in%2520Drosophila%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D6504%26epage%3D6510%26doi%3D10.1073%2Fpnas.0701572104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borcherds, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theillet, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finzel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishall, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dieterich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selenko, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loewer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daughdrill, G. W.</span></span> <span> </span><span class="NLM_article-title">Disorder and residual helicity alter p53-Mdm2 binding affinity and signaling in cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1000</span>– <span class="NLM_lpage">1002</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1668</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1038%2Fnchembio.1668" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=25362358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGiurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1000-1002&author=W.+Borcherdsauthor=F.-X.+Theilletauthor=A.+Katzerauthor=A.+Finzelauthor=K.+M.+Mishallauthor=A.+T.+Powellauthor=H.+Wuauthor=W.+Manieriauthor=C.+Dieterichauthor=P.+Selenkoauthor=A.+Loewerauthor=G.+W.+Daughdrill&title=Disorder+and+residual+helicity+alter+p53-Mdm2+binding+affinity+and+signaling+in+cells&doi=10.1038%2Fnchembio.1668"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Disorder and residual helicity alter p53-Mdm2 binding affinity and signaling in cells</span></div><div class="casAuthors">Borcherds, Wade; Theillet, Francois-Xavier; Katzer, Andrea; Finzel, Ana; Mishall, Katie M.; Powell, Anne T.; Wu, Hongwei; Manieri, Wanda; Dieterich, Christoph; Selenko, Philipp; Loewer, Alexander; Daughdrill, Gary W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1000-1002</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Levels of residual structure in disordered interaction domains det. in vitro binding affinities, but whether they exert similar roles in cells is not known.  Here, we show that increasing residual p53 helicity results in stronger Mdm2 binding, altered p53 dynamics, impaired target gene expression and failure to induce cell cycle arrest upon DNA damage.  These results establish that residual structure is an important determinant of signaling fidelity in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgBzekcYfhRrVg90H21EOLACvtfcHk0lhgkxL_mmLZew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGiurbF&md5=d25c8a6df00483c100327dddfd30184c</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1668&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1668%26sid%3Dliteratum%253Aachs%26aulast%3DBorcherds%26aufirst%3DW.%26aulast%3DTheillet%26aufirst%3DF.-X.%26aulast%3DKatzer%26aufirst%3DA.%26aulast%3DFinzel%26aufirst%3DA.%26aulast%3DMishall%26aufirst%3DK.%2BM.%26aulast%3DPowell%26aufirst%3DA.%2BT.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DDieterich%26aufirst%3DC.%26aulast%3DSelenko%26aufirst%3DP.%26aulast%3DLoewer%26aufirst%3DA.%26aulast%3DDaughdrill%26aufirst%3DG.%2BW.%26atitle%3DDisorder%2520and%2520residual%2520helicity%2520alter%2520p53-Mdm2%2520binding%2520affinity%2520and%2520signaling%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1000%26epage%3D1002%26doi%3D10.1038%2Fnchembio.1668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Understanding Cell Penetration of Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10241</span>– <span class="NLM_lpage">10287</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.9b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.9b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10241-10287&author=P.+G.+Doughertyauthor=A.+Sahniauthor=D.+Pei&title=Understanding+Cell+Penetration+of+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.9b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Cell Penetration of Cyclic Peptides</span></div><div class="casAuthors">Dougherty, Patrick G.; Sahni, Ashweta; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10241-10287</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Approx. 75% of all disease-relevant human proteins, including those involved in intracellular protein-protein interactions (PPIs), are undruggable with the current drug modalities (i.e., small mols. and biologics).  Macrocyclic peptides provide a potential soln. to these undruggable targets because their larger sizes (relative to conventional small mols.) endow them the capability of binding to flat PPI interfaces with antibody-like affinity and specificity.  Powerful combinatorial library technologies have been developed to routinely identify cyclic peptides as potent, specific inhibitors against proteins including PPI targets.  However, with the exception of a very small set of sequences, the vast majority of cyclic peptides are impermeable to the cell membrane, preventing their application against intracellular targets.  This Review examines common structural features that render most cyclic peptides membrane impermeable, as well as the unique features that allow the minority of sequences to enter the cell interior by passive diffusion, endocytosis/endosomal escape, or other mechanisms.  We also present the current state of knowledge about the mol. mechanisms of cell penetration, the various strategies for designing cell-permeable, biol. active cyclic peptides against intracellular targets, and the assay methods available to quantify their cell-permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0NDA9U7-1YrVg90H21EOLACvtfcHk0linZhlUqlpA6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D&md5=65f68a9b69f3f7f33641b6a91835b1dd</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.9b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.9b00008%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DSahni%26aufirst%3DA.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DUnderstanding%2520Cell%2520Penetration%2520of%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10241%26epage%3D10287%26doi%3D10.1021%2Facs.chemrev.9b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span>; <span class="NLM_string-name">Hilinski, G. J.</span></span> <span> </span><span class="NLM_article-title">Stapled peptides for intracellular drug targets</span>. In  <i>Methods in Enzymology</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittrup, K. D.</span>, <span class="NLM_string-name">Verdine, G. L.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">2012</span>; Vol.  <span class="NLM_volume">503</span>, pp  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">33</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1016%2FB978-0-12-396962-0.00001-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=3-33&author=G.+L.+Verdine&author=G.+J.+Hilinskiauthor=K.+D.+Wittrup&author=G.+L.+Verdine&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-396962-0.00001-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-396962-0.00001-X%26sid%3Dliteratum%253Aachs%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStapled%2520peptides%2520for%2520intracellular%2520drug%2520targets%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DWittrup%26aufirst%3DK.%2BD.%26pub%3DAcademic%2520Press%26date%3D2012%26volume%3D503%26spage%3D3%26epage%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J. T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Towards understanding cell penetration by stapled peptides</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1039/c4md00131a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1039%2Fc4md00131a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCgsbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=111-119&author=Q.+Chuauthor=R.+E.+Moelleringauthor=G.+J.+Hilinskiauthor=Y.-W.+Kimauthor=T.+N.+Grossmannauthor=J.+T.-H.+Yehauthor=G.+L.+Verdine&title=Towards+understanding+cell+penetration+by+stapled+peptides&doi=10.1039%2Fc4md00131a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Towards understanding cell penetration by stapled peptides</span></div><div class="casAuthors">Chu, Qian; Moellering, Raymond E.; Hilinski, Gerard J.; Kim, Young-Woo; Grossmann, Tom N.; Yeh, Johannes T.-H.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Hydrocarbon-stapled α-helical peptides are a new class of targeting mols. capable of penetrating cells and engaging intracellular targets formerly considered intractable.  This technol. has been applied to the development of cell-permeable ligands targeting key intracellular protein-protein interactions.  However, the properties governing cell penetration of hydrocarbon-stapled peptides have not yet been rigorously investigated.  Herein we report our studies to systematically probe cellular uptake of stapled peptides.  We developed a high-throughput epifluorescence microscopy assay to quant. measure stapled peptide intracellular accumulation and demonstrated that this assay yielded highly reproducible results.  Using this assay, we analyzed more than 200 peptides with various sequences, staple positions and types, and found that cell penetration ability is strongly related to staple type and formal charge, whereas other physicochem. parameters do not appear to have a significant effect.  We next investigated the mechanism(s) involved in stapled peptide internalization and have demonstrated that stapled peptides penetrate cells through a clathrin- and caveolin-independent endocytosis pathway that involves, in part, sulfated cell surface proteoglycans, but that also seems to exploit a novel, uncharacterized pathway.  Taken together, staple type and charge are the key phys. properties in detg. the cell penetration ability of stapled peptides, and anionic cell surface proteoglycans might serve as receptors to mediate stapled peptide internalization.  These findings improve our understanding of stapled peptides as chem. probes and potential targeted therapeutics, and provide useful guidelines for the design of next-generation stapled peptides with enhanced cell permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_6exBOPh1FLVg90H21EOLACvtfcHk0linZhlUqlpA6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCgsbnE&md5=11558b56a57ba18f95142579b59e0fcc</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1039%2Fc4md00131a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4md00131a%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DQ.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DYeh%26aufirst%3DJ.%2BT.-H.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DTowards%2520understanding%2520cell%2520penetration%2520by%2520stapled%2520peptides%26jtitle%3DMedChemComm%26date%3D2015%26volume%3D6%26spage%3D111%26epage%3D119%26doi%3D10.1039%2Fc4md00131a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilkes, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coppola, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeatman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reuther, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Regulation of MDMX expression by mitogenic signaling</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1999</span>– <span class="NLM_lpage">2010</span>, <span class="refDoi"> DOI: 10.1128/mcb.01633-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=10.1128%2Fmcb.01633-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=18172009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlKjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=1999-2010&author=D.+M.+Gilkesauthor=Y.+Panauthor=D.+Coppolaauthor=T.+Yeatmanauthor=G.+W.+Reutherauthor=J.+Chen&title=Regulation+of+MDMX+expression+by+mitogenic+signaling&doi=10.1128%2Fmcb.01633-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MDMX expression by mitogenic signaling</span></div><div class="casAuthors">Gilkes, Daniele M.; Pan, Yu; Coppola, Domenico; Yeatman, Timothy; Reuther, Gary W.; Chen, Jiandong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1999-2010</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">MDMX is an important regulator of p53 transcriptional activity and stress response.  MDMX overexpression and gene amplification are implicated in p53 inactivation and tumor development.  Unlike MDM2, MDMX is not inducible by p53, and little is known about its regulation at the transcriptional level.  We found that MDMX levels in tumor cell lines closely correlate with promoter activity and mRNA level.  Activated K-Ras and insulin-like growth factor 1 induce MDMX expression at the transcriptional level through mechanisms that involve the mitogen-activated protein kinase and c-Ets-1 transcription factors.  Pharmacol. inhibition of MEK results in down-regulation of MDMX in tumor cell lines.  MDMX overexpression was detected in ∼50% of human colon tumors and showed strong correlation with increased extracellular signal-regulated kinase phosphorylation.  Therefore, MDMX expression is regulated by mitogenic signaling pathways.  This mechanism may protect normal proliferating cells from p53 but also hamper p53 response during tumor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdKklelbSbnbVg90H21EOLACvtfcHk0lhg1QBK6wiWDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlKjurc%253D&md5=77fa097717f521a378737bc861718bc1</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1128%2Fmcb.01633-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.01633-07%26sid%3Dliteratum%253Aachs%26aulast%3DGilkes%26aufirst%3DD.%2BM.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DCoppola%26aufirst%3DD.%26aulast%3DYeatman%26aufirst%3DT.%26aulast%3DReuther%26aufirst%3DG.%2BW.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DRegulation%2520of%2520MDMX%2520expression%2520by%2520mitogenic%2520signaling%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D1999%26epage%3D2010%26doi%3D10.1128%2Fmcb.01633-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Helical Antimicrobial Sulfono-γ-AApeptides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4802</span>– <span class="NLM_lpage">4811</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4802-4811&author=Y.+Liauthor=H.+Wuauthor=P.+Tengauthor=G.+Baiauthor=X.+Linauthor=X.+Zuoauthor=C.+Caoauthor=J.+Cai&title=Helical+Antimicrobial+Sulfono-%CE%B3-AApeptides&doi=10.1021%2Facs.jmedchem.5b00537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Helical Antimicrobial Sulfono-γ-AApeptides</span></div><div class="casAuthors">Li, Yaqiong; Wu, Haifan; Teng, Peng; Bai, Ge; Lin, Xiaoyang; Zuo, Xiaobing; Cao, Chuanhai; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4802-4811</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Host-defense peptides (HDPs) such as magainin 2 have emerged as potential therapeutic agents combating antibiotic resistance.  Inspired by their structures and mechanism of action, herein the authors report the first example of antimicrobial helical sulfono-γ-AApeptide foldamers.  The lead mol. displays broad-spectrum and potent antimicrobial activity against multi-drug-resistant Gram-pos. and Gram-neg. bacterial pathogens.  Time-kill studies and fluorescence microscopy suggest that sulfono-γ-AApeptides eradicate bacteria by taking a mode of action analogous to that of HDPs.  Clear structure-function relations exist in the studied sequences.  Longer sequences, presumably adopting more-defined helical structures, are more potent than shorter ones.  The sequence with less helical propensity in soln. could be more selective than the stronger helix-forming sequences.  Moreover, this class of antimicrobial agents are resistant to proteolytic degrdn.  These results may lead to the development of a new class of antimicrobial foldamers combating emerging antibiotic-resistant pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPELcUdcGfXrVg90H21EOLACvtfcHk0lhg1QBK6wiWDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptFWqsb4%253D&md5=5eab0445be6682f250016148fac54969</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00537%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DBai%26aufirst%3DG.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DHelical%2520Antimicrobial%2520Sulfono-%25CE%25B3-AApeptides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4802%26epage%3D4811%26doi%3D10.1021%2Facs.jmedchem.5b00537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR','PDB','1YCR'); return false;">PDB: 1YCR</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00993">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_27763"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b00993?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00993</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental procedures of synthesis; characterization and purities of sequences; fluorescence polarization competition assays; and enzymatic stability study (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_001.pdf">jm9b00993_si_001.pdf (4.16 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00993/suppl_file/jm9b00993_si_002.csv">jm9b00993_si_002.csv (9.8 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank under accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YCR">1YCR</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00993&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00993%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-3%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00993" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997ce4fd5922a8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
