
<html lang="en"     class="pb-page"  data-request-id="29b344c6-4efa-4923-b0d9-254ccd1659ab"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.9b00716;issue:issue:10.1021/jmcmar.2019.62.issue-21;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Free Ligand 1D NMR Conformational Signatures To Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles" /></meta><meta name="dc.Creator" content="Amber Y. S.  Balazs" /></meta><meta name="dc.Creator" content="Rodrigo J.  Carbajo" /></meta><meta name="dc.Creator" content="Nichola L.  Davies" /></meta><meta name="dc.Creator" content="Yu  Dong" /></meta><meta name="dc.Creator" content="Alexander W.  Hird" /></meta><meta name="dc.Creator" content="Jeffrey W.  Johannes" /></meta><meta name="dc.Creator" content="Michelle L.  Lamb" /></meta><meta name="dc.Creator" content="William  McCoull" /></meta><meta name="dc.Creator" content="Piotr  Raubo" /></meta><meta name="dc.Creator" content="Graeme R.  Robb" /></meta><meta name="dc.Creator" content="Martin J.  Packer" /></meta><meta name="dc.Creator" content="Elisabetta  Chiarparin" /></meta><meta name="dc.Description" content="The three-dimensional conformations adopted by a free ligand in solution impact bioactivity and physicochemical properties. Solution 1D NMR spectra inherently contain information on ligand conforma..." /></meta><meta name="Description" content="The three-dimensional conformations adopted by a free ligand in solution impact bioactivity and physicochemical properties. Solution 1D NMR spectra inherently contain information on ligand conforma..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 30, 2019" /></meta><meta name="dc.Type" content="news" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00716" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00716" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00716" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00716" /></link>
        
    
    

<title>Free Ligand 1D NMR Conformational Signatures To Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00716" /></meta><meta property="og:title" content="Free Ligand 1D NMR Conformational Signatures To Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0015.jpeg" /></meta><meta property="og:description" content="The three-dimensional conformations adopted by a free ligand in solution impact bioactivity and physicochemical properties. Solution 1D NMR spectra inherently contain information on ligand conformational flexibility and three-dimensional shape, as well as the propensity of the free ligand to fully preorganize into the bioactive conformation. Herein we discuss some key learnings, distilled from our experience developing potent and selective synthetic macrocyclic inhibitors, including Mcl-1 clinical candidate AZD5991. Case studies have been selected from recent oncology research projects, demonstrating how 1D NMR conformational signatures can complement X-ray protein–ligand structural information to guide medicinal chemistry optimization. Learning to extract free ligand conformational information from routinely available 1D NMR signatures has proven to be fast enough to guide medicinal chemistry decisions within design cycles for compound optimization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00716"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00716">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00716&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00716&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00716&amp;href=/doi/10.1021/acs.jmedchem.9b00716" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9418-9437</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01688" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.9b00747" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.1c00273"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Free Ligand 1D NMR Conformational Signatures To Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Amber Y. S. Balazs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amber Y. S. Balazs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amber+Y.+S.++Balazs">Amber Y. S. Balazs</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rodrigo J. Carbajo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rodrigo J. Carbajo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rodrigo+J.++Carbajo">Rodrigo J. Carbajo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nichola L. Davies</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nichola L. Davies</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nichola+L.++Davies">Nichola L. Davies</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmaron Beijing Co., Ltd., Beijing 100176, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Dong">Yu Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alexander W. Hird</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alexander W. Hird</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alexander+W.++Hird">Alexander W. Hird</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeffrey W. Johannes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeffrey W. Johannes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeffrey+W.++Johannes">Jeffrey W. Johannes</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7242-6682" title="Orcid link">http://orcid.org/0000-0001-7242-6682</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michelle L. Lamb</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michelle L. Lamb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michelle+L.++Lamb">Michelle L. Lamb</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3005-8065" title="Orcid link">http://orcid.org/0000-0002-3005-8065</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">William McCoull</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William McCoull</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William++McCoull">William McCoull</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2977-1744" title="Orcid link">http://orcid.org/0000-0002-2977-1744</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Piotr Raubo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Piotr Raubo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Piotr++Raubo">Piotr Raubo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Graeme R. Robb</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Graeme R. Robb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Graeme+R.++Robb">Graeme R. Robb</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4531-4375" title="Orcid link">http://orcid.org/0000-0002-4531-4375</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin J. Packer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin J. Packer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin+J.++Packer">Martin J. Packer</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Elisabetta Chiarparin</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabetta Chiarparin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemistry, R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +44-7818524246. E-mail: <a href="/cdn-cgi/l/email-protection#42072e2b312320273636236c012a2b23303223302b2c02233136302338272c2721236c212d2f"><span class="__cf_email__" data-cfemail="01446d68726063647575602f4269686073716073686f4160727573607b646f6462602f626e6c">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabetta++Chiarparin">Elisabetta Chiarparin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2998-1346" title="Orcid link">http://orcid.org/0000-0002-2998-1346</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00716&amp;href=/doi/10.1021%2Facs.jmedchem.9b00716" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9418–9437</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 30, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 May 2019</li><li><span class="item_label"><b>Published</b> online</span>30 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00716" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00716</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ec-1612971054143.svg" alt="ACS Editors’ Choice" /></img><span>ACS Editors’ Choice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00716"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7983</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00716" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Free Ligand 1D NMR Conformational Signatures To Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Amber&quot;,&quot;last_name&quot;:&quot;Y. S. Balazs&quot;},{&quot;first_name&quot;:&quot;Rodrigo&quot;,&quot;last_name&quot;:&quot;J. Carbajo&quot;},{&quot;first_name&quot;:&quot;Nichola&quot;,&quot;last_name&quot;:&quot;L. Davies&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;W. Hird&quot;},{&quot;first_name&quot;:&quot;Jeffrey&quot;,&quot;last_name&quot;:&quot;W. Johannes&quot;},{&quot;first_name&quot;:&quot;Michelle&quot;,&quot;last_name&quot;:&quot;L. Lamb&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;McCoull&quot;},{&quot;first_name&quot;:&quot;Piotr&quot;,&quot;last_name&quot;:&quot;Raubo&quot;},{&quot;first_name&quot;:&quot;Graeme&quot;,&quot;last_name&quot;:&quot;R. Robb&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;J. Packer&quot;},{&quot;first_name&quot;:&quot;Elisabetta&quot;,&quot;last_name&quot;:&quot;Chiarparin&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;9418-9437&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00716&quot;},&quot;abstract&quot;:&quot;The three-dimensional conformations adopted by a free ligand in solution impact bioactivity and physicochemical properties. Solution 1D NMR spectra inherently contain information on ligand conformational flexibility and three-dimensional shape, as well as the propensity of the free ligand to fully preorganize into the bioactive conformation. Herein we discuss some key learnings, distilled from our experience developing potent and selective synthetic macrocyclic inhibitors, including Mcl-1 clinical candidate AZD5991. Case studies have been selected from recent oncology research projects, demonstrating how 1D NMR conformational signatures can complement X-ray protein–ligand structural information to guide medicinal chemistry optimization. Learning to extract free ligand conformational information from routinely available 1D NMR signatures has proven to be fast enough to guide medicinal chemistry decisions within design cycles for compound optimization.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00716&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00716" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00716&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00716" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00716&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00716" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00716&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00716&amp;href=/doi/10.1021/acs.jmedchem.9b00716" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00716" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00716" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00716%26sid%3Dliteratum%253Aachs%26pmid%3D31361481%26genre%3Darticle%26aulast%3DBalazs%26date%3D2019%26atitle%3DFree%2BLigand%2B1D%2BNMR%2BConformational%2BSignatures%2BTo%2BEnhance%2BStructure%2BBased%2BDrug%2BDesign%2Bof%2Ba%2BMcl-1%2BInhibitor%2B%2528AZD5991%2529%2Band%2BOther%2BSynthetic%2BMacrocycles%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D21%26spage%3D9418%26epage%3D9437%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291911" title="Hydrogen">Hydrogen</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/jmcmar.2019.62.issue-21/20191114/jmcmar.2019.62.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The three-dimensional conformations adopted by a free ligand in solution impact bioactivity and physicochemical properties. Solution 1D NMR spectra inherently contain information on ligand conformational flexibility and three-dimensional shape, as well as the propensity of the free ligand to fully preorganize into the bioactive conformation. Herein we discuss some key learnings, distilled from our experience developing potent and selective synthetic macrocyclic inhibitors, including Mcl-1 clinical candidate AZD5991. Case studies have been selected from recent oncology research projects, demonstrating how 1D NMR conformational signatures can complement X-ray protein–ligand structural information to guide medicinal chemistry optimization. Learning to extract free ligand conformational information from routinely available 1D NMR signatures has proven to be fast enough to guide medicinal chemistry decisions within design cycles for compound optimization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51140" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51140" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Synthetic macrocycles are an increasingly attractive modality for pharmaceutical drug discovery projects. Macrocyclic scaffolds can enhance potency at their target and selectivity over antitargets and improve metabolic stability and physicochemical properties compared to their acyclic counterparts while maintaining oral bioavailability.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6">(1−6)</a> In addition, perhaps the greatest advantage of macrocycles in contrast to most acyclic small molecules is their ability to efficiently bind large, flat, featureless, difficult-to-drug binding sites, such as protein–protein interaction (PPI) regions. On the other hand, a disadvantage may be the increased synthetic challenge of macrocycles over their acyclic counterparts. Consequently, careful design strategies to focus synthetic efforts and identify optimal linkers to cyclize into a bioactive conformation are desired.</div><div class="NLM_p">Protein–ligand X-ray crystallography data and associated visualization tools are a vital part of rational drug design, providing insights that help focus molecular design hypotheses toward a goal of high target affinity. However, optimization of ligand potency can still be an empirical process, even when structure-based drug design is enabled through the availability of crystal structures of small molecules bound to target proteins. Protein–ligand X-ray structures generally provide a static snapshot of the final protein–ligand complex, without any information on the journey taken by the ligand to arrive at the bound, bioactive conformation.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> While crystallographic data provide fine detail on interatomic interactions, it is not possible to quantify the amount of ligand strain incurred to adopt the observed binding mode without additional information from the conformational preference of the free ligand.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">In our view, information available on a ligand’s conformation in solution is often underutilized or even overlooked, even though it can provide a critical “missing link” in 3D structure-based drug design.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> The extent to which the free ligand preorganizes in solution and whether the bioactive conformation is preferred can influence design hypotheses and subsequent design steps. Accurate and robust predictions of the free and bound conformations of a ligand can be pivotal to successful structure-enabled drug design campaigns.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> In addition to optimizing opportunities for ligand–target interactions, binding potency can be maximized by designing free ligands with an innate preference for their bioactive conformation.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Preorganization of the free ligand reduces free energy penalties due to torsional strain and intramolecular rearrangement as part of the protein–ligand binding event.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p last">While the conformational preferences of a small molecule in solution encode an abundance of useful information for medicinal chemistry design, strategies to extract the maximum amount of practical information from the smallest resource investment have been an evolving area of interest. Synthetic chemists typically rely on NMR for structure verification and to check the purity of newly synthesized compounds. These routine 1D NMR spectra implicitly encode 3D information about free ligand conformations, as NMR chemical shifts and <i>J</i>-coupling patterns are exquisitely sensitive to both atomic electronic environment and molecular dynamics.<a onclick="showRef(event, 'ref9 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref9 ref14 ref15">(9,14,15)</a> Including routine conformational analysis in design through NMR spectral signatures for flexibility and shape can be advantageous. Herein we provide a set of case studies where molecular design has benefited from timely conformational insights obtained from analysis of routine 1D NMR data. The case studies encompass both prospective insight, in which NMR signals were used as markers of success in design iterations, and retrospective analysis in which SAR trends were rationalized based on NMR data. While direct impact cannot be guaranteed in every application of NMR to ligand conformational analysis, we have found that analysis of 1D NMR data enhances understanding of chemical series and is especially valuable for understanding the impact of macrocyclization on conformational dynamics.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83788" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83788" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> 1D NMR Conformational Signatures: Free Ligand Flexibility and Shape</h3><div class="NLM_p">Solution 1D NMR spectra inherently contain information on ligand conformational flexibility and three-dimensional shape. To illustrate this principle, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> compares 1D NMR spectra between piperidine and its 2-methylated counterpart. Methylation is a common rigidification strategy in a medicinal chemist’s toolbox.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a, a separation of signals is observed for the axial and equatorial hydrogen atoms attached to the same carbon atom (whether w, x, y, or z), producing distinct axial and equatorial chemical shift and <i>J</i>-coupling patterns, suggesting conformational preorganization. The 1D NMR spectral signature in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a is a hallmark of a chair conformation, with the methyl adopting a preferred equatorial orientation. This observation is familiar to most chemists. By comparison, the nonmethylated analog displays fewer resolved signals, suggesting conformational averaging of the local electronic environment, hence a readout of conformational flexibility (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b) or in other words low populations of preferred conformations.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Free ligand 1D NMR conformational signatures can often be deduced from routine NMR spectra (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). (a) 2-Methylpiperidine demonstrates conformational preorganization of the chair conformation stabilized on the millisecond time scale. This manifests as distinct chemical shifts and <i>J</i>-couplings for each separate methylene hydrogen atom. (b) Piperidine demonstrates conformational flexibility. The methylene hydrogen atom chemical shifts are the same due to dynamic averaging of chemical environments, such as rapid exchange between axial and equatorial orientations. These conformational signatures are conserved across solvents and protonation states (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This approach of deducing 3D conformational preferences from routine 1D NMR spectra extends to more complex molecules preorganized in solution. <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> gives the case of a potent and selective PPI inhibitor for the oncology target Mcl-1, AZD5991 (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> currently in clinical trials (NCT03218683). The 1D NMR spectrum of free ligand <b>1</b> in solution (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b) reveals molecular conformational preorganization: distinct chemical shifts for each hydrogen atom correspond to unique electronic environments. This is particularly evident in the aliphatic region with separated methylene signals each with an integral of 1. In order to measure what is the preferred conformation and its population, we performed conformational analysis by NMR<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> encompassing an exhaustive analysis of multiple NMR constraints, including chemical shifts, through-space distances between pairs of atoms (derived from NOEs and/or dipolar couplings), and torsion angles (derived from <i>J</i>-couplings).<a onclick="showRef(event, 'ref9 ref12 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref9 ref12 ref18 ref19 ref20 ref21">(9,12,18−21)</a> However, recognizing that this process still requires expert interpretations, especially for complex NMR spectra from macrocycles, we also developed a diagnostic NMR conformational approach for rapid conformational verification, which can be used directly by chemists to confirm the presence of expected conformational signatures in new compounds.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> describes how we applied this approach to <b>1</b>. The 2D molecular structure is used to predict chemical shifts using empirical algorithms embedded in NMR processing software (Advanced Chemistry Development, Inc. (ACD/Labs)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> or MestreLab Research S.L. (MNova)).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> For example, in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a, based on the 2D structure of <b>1</b>, a chemical shift of 5.8 ppm was predicted for the pyrazole CH (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S1</a> for the full table of 2D predicted chemical shifts), while the experimental chemical shift value is unexpectedly low, at 4.8 ppm (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). This discrepancy between the 2D predicted and experimental chemical shifts is a strong indication of a three-dimensional, shape-dependent, conformational effect resulting in a shielding of the pyrazole proton through an intramolecular interaction. In fact, it is widely acknowledged that proton predictions using empirical databases are not reliable in predicting experimental shifts from 2D structures, precisely because of the relative magnitude of 3D conformational effects on experimental chemical shift values. This fact can be used to great advantage in using chemical shifts to identify ligand 3D conformational effects directly from routine 1D NMR spectra, since any significant discrepancy between predicted shift values from a 2D structure and observed values immediately suggests an opportunity to identify a conformationally dependent chemical shift reporter atom. The conformational reporter signal can be flagged by comparing 2D predicted shifts with experimentally measured chemical shift values (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S1</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Conformational analysis platform applied to AZD5991 illustrates the basic tools used to extract conformational and bioactivity reporter signals from experimental free ligand data. (a) A chemical shift of 5.8 ppm is predicted for the pyrazole CH proton (highlighted in green), based on the 2D structure. (b) The pyrazole CH proton appears experimentally at 4.8 ppm, highlighted in green on the 1D NMR spectrum (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). Assignment of three of the methylene groups (x, y, z) is explicitly given to illustrate the signal complexity consistent with conformational preorganization (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S1</a> gives a complete assignment). (c) Conformation-dependent QM calculations give 4.1 ppm for the global energy minimum of the nonbioactive conformation (labeled and colored red) and 4.8 ppm for both the free ligand solution NMR conformation (labeled “bioactive” and colored green) and the bound ligand conformation from the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS0">6FS0</a>). The dotted line shown on the bioactive conformations emphasizes the conformational origin of the pyrazole CH nuclear shielding arising from ring current effects.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Free ligand 1D NMR conformational signatures of <b>2</b> and <b>3</b>, with selected aliphatic proton signals as markers of conformational preorganization labeled, for two macrocycles (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). The 3D bioactive free ligand conformations, derived from X-ray crystal structures of ligand–protein complexes, are shown in green. The macrocyclic portion is emphasized with dark green. (a) <b>2</b> also exhibits a bioactive conformational reporter signal, with the NMR signal highlighted in green at 6.4 ppm, derived by comparing the 2D predicted shift (7.6 ppm) with QM calculations from the bioactive X-ray conformation (6.5 ppm; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). (b) The NMR signature is calculated for the bioactive conformation of <b>3</b> and is interpreted across the proton spectrum without a specific reporter atom signal (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S3</a>). Additional spectral support of preorganization is seen in the sharp spectral line widths of the CH signals, x* is highlighted as an example.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To determine the 3D preferred conformation, commercial software provides accessibility to nonexperts for quantum mechanical (QM) chemical shift calculations from one or more proposed 3D conformation(s).<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> Advances in high performance computing have enabled us to routinely use QM models to generate conformational ensembles, as well as calculate NMR parameters, such as <sup>1</sup>H chemical shifts and <sup>1</sup>H, <sup>1</sup>H <i>J</i>-couplings, providing reliable free ligand 3D structural predictions within a time frame suitable for synthesis and testing cycles encountered in drug design projects. These conformation dependent shift calculations are compared to an experimental 1D NMR spectrum to validate the proposal. This process of validation of 3D conformations of preorganized ligands is analogous to routine validation of 2D chemical structures of newly synthesized molecules by predicting carbon chemical shift values for various 2D structural proposals. <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c shows two different proposed 3D conformations: (1) the calculated global minimum determined with a QM potential (B3LYP/6-31G*), including a polarizable continuum model (PCM) as the continuum solvation method for water, shown in red with the naphthyl ring “down”; (2) the bioactive conformation, shown in green, derived from the crystal structure of <b>1</b> in complex with Mcl-1 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS0">6FS0</a>), shown on the far right. The 3D conformation dependent QM calculated shifts are tabulated and compared to experimental values (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). This comparison is a quick method to determine if the proposed conformation is consistent with the experimental conformation. In this manner, the QM-derived global minimum conformation, with the naphthyl ring “down”, cannot be regarded as the dominant conformer in solution (labeled and colored red) as the discrepancy with experimental NMR data is too large to support such a conformation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). Notably, the pyrazole CH was calculated to be 4.1 ppm, confirming it is not the bioactive conformation. The dominant free ligand conformation in solution (labeled “bioactive” and colored green) was determined by fitting experimental NMR parameters to the full QM refined conformational ensemble using MSpin.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The experimental chemical shift of the pyrazole CH, at 4.8 ppm, is identical to the QM calculated shift value for the bioactive conformation.</div><div class="NLM_p">Although this proton identifies clearly as a sensitive conformational reporter signal, even small changes in ligand conformation correspond to measurable shift differences across the whole 1D NMR spectrum, which effectively is a unique fingerprint for a preorganized conformation. It is worth noting that the free ligand conformational preference of <b>1</b> determined in DMSO-<i>d</i><sub>6</sub> is preserved in aqueous buffer at pH = 7.4, as established by measuring 1D NMR conformational signatures in buffer (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Figure S3</a>). A definitive advantage of using 1D NMR conformational signatures to validate 3D conformations is to enable characterization of molecules with submicromolar solubility in any solvent; using high field spectrometers equipped with cryoprobes, such a 1D proton spectrum can be acquired in 1–6 h.</div><div class="NLM_p">To summarize, free AZD5991 (<b>1</b>) in solution is fully preorganized into its bioactive conformation for Mcl-1 binding and the pyrazole signal was chosen as a bioactivity reporter signal. The recognition that the calculated global QM minimum is not experimentally confirmed by NMR succeeded in devalidating any design hypotheses aimed at changing a putative in silico free ligand conformation to match the bioactive X-ray conformation, and confirming the full potential for activity, in terms of protein complementarity and free ligand preorganization, was likely already achieved. Such timely information to design teams is crucial in saving synthetic effort, by avoiding less promising design ideas, especially when lengthy macrocycle syntheses are involved.</div><div class="NLM_p">Single bioactive conformational reporter signals of preorganized ligands in solution can often but not always be identified in a 1D NMR spectrum. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> reports the free ligand conformations of a second Mcl-1 advanced lead<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and of a pyrazolo[1,5-<i>a</i>]pyrimidine BCL6 PPI inhibitor<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and their unique 1D conformational signatures. The molecules clearly preorganize in solution, as demonstrated by the complex, yet defined signals corresponding to each hydrogen atom, especially in the aliphatic region. The molecules are also preorganized into the bioactive conformation in solution, as demonstrated by overlap of their experimental chemical shifts with the values calculated by QM based on the bioactive conformation as observed in the ligand-protein crystal structures (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information tables</a>). A conformational reporter signal can be identified for the tetrahydrobenzazepinone aromatic proton of <b>2</b>, highlighted in green, which appears at 6.4 ppm (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a), very different from the predicted shift using empirical 2D based calculations, which is 7.6 ppm (ACD/Labs or 7.4 ppm using MNova). This is not the case for <b>3</b>, where no single hydrogen atom is flagged by the 2D structural predictions as being anomalous (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S3</a>). In such cases, the entire unique disposition of the 1D NMR chemical shifts and <i>J</i>-couplings will need to be used over a single reporter signal as an indication of the bioactive conformation. Interestingly, the chemical shift value of the hydroxyl proton for <b>3</b> could be compatible with an internal hydrogen bond (IHB). This was confirmed with a spectrum in chloroform (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Figure S3</a>). The presence of this dynamic IHB effectively decreases the exposed polarity of the molecule in a nonpolar environment and can facilitate membrane permeability, demonstrating how the free ligand conformation can affect the molecule’s physicochemical properties.</div><div class="NLM_p last">In conclusion, the examples above show that recognition of 1D NMR conformational signatures can help assess the degree of flexibility/rigidity and bioactivity of a molecule, effectively enhancing 3D structure-based analysis. The inherent simplicity of using routinely available data for every compound synthesized confers a practical advantage of accuracy and speed to enable the data to influence design decisions within a design cycle of a lead optimization project.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> 1D NMR Guided Macrocyclization</h3><div class="NLM_p">Designing compounds that preorganize into a bioactive conformation is a well-known and effective strategy for optimizing potency.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31">(28−31)</a> Some strategies include addition of chiral centers,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> rings, hydrophobic collapse,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> intramolecular interactions, etc., with macrocyclization as one of those strategies that is particularly successful with PPI targets. The clinical candidate AZD5991 (<b>1</b>) and the other two advanced leads, <b>2</b> and <b>3</b> described in the previous section, are fully preorganized into the bioactive conformation and exhibit very high binding affinities. This correlation between ligand preorganization into the bioactive state in solution and potency is observed in several SBDD cases, supported by X-ray protein–ligand structures and NMR free ligand conformations.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> This is consistent with achieving the maximum potential for ligand preorganization and protein complementarity.<a onclick="showRef(event, 'ref9 ref10 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref31 ref32">(9,10,31,32)</a> In the following we will show how combining 1D NMR free ligand conformational signatures and X-ray crystallography protein ligand data can effectively guide macrocyclization strategies. Specifically, we will show how to prioritize design hypotheses that exploit protein complementarity while minimizing free energy penalties and torsional strain of free ligands on binding.</div><div class="NLM_p">Johannes et al.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> effectively used NMR-derived information on conformations of a ligand free in solution and bound to its protein of interest to guide macrocyclization design of potent inhibitors of the Mcl-1 protein. Compound <b>4</b> was identified as a low affinity hit from a DNA-encoded library screen.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a shows the methylene hydrogen signals of the NMR spectrum, which indicate some degree of conformational preorganization (distinct hydrogen atom chemical shifts within a methylene group), while some areas retain higher flexibility (overlapping signals) suggesting absence of a preferred conformation in solution. A mostly flexible conformation, with local rigidity around hydrogen atom x, is preferred by the acyclic free ligand in solution. On the other hand, the X-ray structure of this hit bound to Mcl-1 shows a distinct bioactive shape. Taken together, this structural information on both the free and bound ligands suggests the opportunity for a macrocyclization design, aimed at stabilizing the free ligand conformation into the bioactive state. The X-ray structure provided a clear rationale for linking at the periphery of the binding site to create a macrocycle that retained the observed pharmacophore for binding to the Mcl-1 pocket, while the free ligand NMR conformational flexibility highlighted the potential gain in potency through pursuing this strategy (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. 2D structures, free ligand experimental 3D solution conformations, bound ligand conformations, and the free ligand 1D NMR signatures (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz) of <b>4</b> and <b>2</b>. (a) The 2D structure shows the conformational reporter hydrogen atom highlighted in red. The free ligand in solution is flexible, shown as a superposition of low energy conformers (gray) against the U-shaped bioactive conformation (green) derived from the ligand docked into a crystal structure of the monobromide analog (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KU9">5KU9</a>). (b) Macrocyclization improved potency. Highlighted in green, the conformational reporter hydrogen atom experimental chemical shift matches the QM calculated value based on the bioactive conformation at 6.4 and 6.5 ppm, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). The bound ligand conformation is shown docked into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MEV">5MEV</a> with the macrocycle emphasized in dark green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Macrocyclization did lead to an increase in potency (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). The spectrum of the free ligand <b>2</b> confirms the success of the design as the molecule is fully preorganized into the bioactive state in solution (see previous section). Notably, the large increase in shielding of x″ is consistent with ring current effects from the expected bioactive geometry of the dichloro-substituted aromatic ring, as seen in the crystal structure of the protein complex. Three-dimensional shape constraints due to macrocyclization are particularly well suited to NMR signatures. The unique chemical shift of each proton in the methylene pair becomes increasingly distinct as the macrocycle rigidifies their local environments. This example demonstrates that the experimental knowledge of ligand flexibility can be sufficient to trigger ligand rigidification designs.</div><div class="NLM_p">Macrocyclization that led to AZD5991 (<b>1</b>) also showed progressive preorganization of the bioactive conformation in solution associated with increases in potency. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> shows acyclic precursors of <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). A previously reported<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> acyclic indole-2-carboxylic acid <b>5</b> that binds to the BH3-binding domain of Mcl-1 shows conformational flexibility, as determined by the NMR aliphatic signature (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a). The chemical shift values of the methylene protons in <b>5</b> are similar, indicating motional averaging and flexibility. Addition of a thioether, <b>6</b>, leads to an increase in potency and broad, separated signals on the 1D NMR spectrum for one pair of methylene hydrogen atoms proximal to the sulfur atom indicating partial rigidification of this region (w′ and w″, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). An X-ray structure shows the bioactive conformation of <b>6</b> adopts a U-shape and highlights the proximity of regions which would be widely separated in an open conformation (pyrazole to naphthyl). The bioactive molecular disposition could explain the discrepancy between the observed pyrazole chemical shift (5.6 ppm) and the value of 6.0 ppm from empirical 2D based shift predictions (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Figure S4</a>). However, the remaining methylene groups still show clear flexibility. This example emphasizes the molecular details provided by simple 1D NMR spectra: such atom-specific information is not available by other methods. Macrocyclization led to a 10-fold increase in potency, and a further boost in potency was achieved by rigidification through atropisomerization, i.e., methylation of the indazole nitrogen and addition of a chlorine atom.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The clinical candidate AZD5991 (<b>1)</b>, a subnanomolar inhibitor of Mcl-1, is the active atropisomer, and it is fully preorganized into the bioactive conformation as confirmed by its 1D NMR conformational signature (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. 1D NMR free ligand conformational signatures, 3D solution conformations, and protein bound ligand conformations of two key acyclic molecules leading to AZD5991 (<b>1</b>). (a) Conformational flexibility of the free ligand <b>5</b> in solution (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 300 MHz), shown as a superposition of low energy conformers (gray) superposed with the bioactive conformation (green) modeled by docking <b>5</b> into 6FS1 (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (b) <b>6</b> has the same rigid indole core with varying degrees of aliphatic conformational flexibility (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 500 MHz). The ensemble of conformers (gray) is shown superposed with the closest minimized conformation of the bound ligand (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS1">6FS1</a>, 3.6 Å interatomic distance between the naphthyl 3-carbon atom and the pyrazole 5-methyl group). QM chemical shift calculations indicate the bioactive conformation is not highly populated by the free ligand in solution. The conformational reporter signal, highlighted in red, appears by NMR at 5.6 ppm highly deshielded compared to the QM calculated value of 4.3 ppm for the bioactive conformation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Figure S4</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In a third example, McCoull et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> developed a potent and specific probe molecule to evaluate BCL6 as an oncogenic drug target. The proton spectral signature of <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) shows reduced flexibility. Of note is the broad line width of a strongly shielded aromatic CH proton (5.6 ppm, empirical 2D based shift predictions (ACD/Labs and MNova) >6.1 ppm), indicative of conformational dynamics on the millisecond time scale consistent with slowed rotation around the bridging NH rotatable bonds. The 3D bioactive conformation (based on the crystal structure of a structural analog, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21">5N21</a>, with a carboxylic acid substituted pyrrolidine, in lieu of the hydroxyl substitution, see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>) has a proximal 7 Å interatomic distance, highlighted in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a, which suggested an opportunity for macrocyclization to increase potency. <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b shows the potent macrocycle <b>3</b> adopts a preferred molecular conformation in solution. Preorganization is demonstrated by the loss of line broadening of the aromatic CH hydrogen atom (x*) and the clear separation of most of the methylene hydrogen atoms, apart from the pair labeled z, only beginning to show differentiation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>b, inset). The fine features in the inset reflect, relative to the other well differentiated methylene pairs, changes in localized dynamics and/or extent of difference in electronic shielding between the two distinct chemical environments of the hydrogen atoms of interest.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. 1D NMR free ligand conformational signatures and 3D ligand conformations (free and bound) for two BCL6 probe molecules (<b>7</b> and <b>3</b>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (a) The acyclic molecule shows conformational flexibility (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 30 °C, 400 MHz), shown as a superposition of low energy conformers (gray) superposed with the bioactive conformation (green) modeled by docking <b>7</b> into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21">5N21</a>, a crystal structure of a protein bound structural analog, with a carboxylic acid substituted pyrrolidine in lieu of the hydroxyl substitution (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (b) 1D NMR (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz) of the macrocycle reveals free ligand preorganization and correlates with improved potency (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> 1D NMR Signatures To Enhance SAR for Linker Optimization</h3><div class="NLM_p">The correlation between ligand affinity and the degree of ligand bioactivity as captured by the 1D NMR conformational signatures is well suited for a project series of analogs, where a bioactive reporter signal can be identified and used repeatedly across multiple compounds to support SAR and design hypothesis generation. By investment in discernment of a bioactive 1D NMR conformational marker for a representative chemical series in solution, those learnings can be applied to other similar scaffolds in a series directly from routine 1D NMR spectra.</div><div class="NLM_p">Johannes et al.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> effectively utilized the knowledge of free ligand conformation in solution to explain unexpected SAR findings upon amide N-methylation. In solution, the non-natural peptidic macrocyclic Mcl-1 inhibitor <b>2</b>, with an N-methylated amide, adopts an <i>E</i> configuration at the amide bond (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a), measured using solution NMR NOEs and conformational analysis (see previously published NMR Supporting Information<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>). The <i>E</i> configuration adopted in solution is in accordance with the bioactive conformation seen in the crystal structure of the protein-bound complex (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). The bioactive conformation was similarly quickly deduced by the presence of the bioactive reporter signal, highlighted in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>. The chemical shift of this hydrogen atom, being sensitive to the relative geometry of the dichloro-substituted aromatic ring, provides a quick method to assess that the bioactive conformation is highly populated in solution. In the bioactive conformation, the ring currents shift the observed proton peak from an expected 7.6 ppm, predicted by 2D structure based empirical methods (ACD/Labs and MNova), to an observed 6.4 ppm, in agreement with 3D shape dependent QM calculations performed on the bioactive conformation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S2</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. 1D NMR conformational signatures of two structurally similar but conformationally different macrocycles, <b>2</b> and <b>8</b> (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). Both compounds have the same 2D predicted value of 7.6 ppm for the hydrogen reporter atom (highlights in 2D, asterisks by 3D), identified by QM calculations as a bioactivity reporter signal (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). (a) The bioactive ligand conformation of the methylated amide shown here is derived from docking to PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MEV">5MEV</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>), a protein crystal structure where the ligand has a methyl sulfone in place of the chlorine atom of the monochlorophenyl moiety. The <i>E</i> configuration adopted around the amide bond is also observed in the free ligand in solution, as determined by proton NOEs (see Supporting Information of Johannes et al.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>). (b) In contrast, the matched pair NH amide, also conformationally rigid, adopts a nonbioactive <i>trans</i> or <i>Z</i> configuration, corresponding with a global change in conformation of the solution macrocycle (no crystal structure available; conformational analysis determined by experimental NOEs, spectra published previously in Supporting Information NMR<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast, the matched pair des-methyl macrocycle <b>8</b> in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b adopts a nonbioactive conformation in solution. This is quickly ascertained from inspection of the 1D NMR spectrum. The bioactive reporter signal is no longer shifted to lower ppm values by the ring current effect. A conformational analysis, using NMR NOE distance measurements<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (no crystal structure available), reveals that <b>8</b> is preorganized preferentially into the <i>trans</i> or <i>Z</i> configuration (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). The difference in torsion angles affects the position of the functional group substitutions off the main macrocycle. The result is an alteration in the location of the reporter hydrogen atom relative to the dichloro substituted aromatic ring, changing the electronic environment such that the signal appears at a higher ppm value and providing a distinctive conformational signature of the nonbioactive state (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b). The observed chemical shift for the nonbioactive conformation is at 7.1 ppm, approaching the higher empirical predictions of 7.6 ppm (ACD/Labs) or 7.4 ppm (MNova). Due to molecular preorganization within this series of inhibitors, the signal of this reporter hydrogen atom serves as a quick and useful marker of the bioactive free ligand solution conformation: ∼6.4 ppm, the bioactive conformation is highly populated, compared with ∼7.1 ppm, when less than 1% of the macrocycle adopts the desired bioactive conformation. This is consistent with the ∼30-fold poorer potency of the nonbioactive conformation <b>8</b> relative to <b>2</b>, which preferentially adopts the correct bioactive conformation. The +0.6-fold increase in log <i>D</i> is insufficient to account for the ∼30-fold change in potency, further suggesting that the differences in potency are driven by locking <b>8</b> into a nonbioactive state. Consequently, the greater understanding of conformation can significantly affect design hypotheses.</div><div class="NLM_p">Within this macrocycle series, the bioactive reporter signal provided a handle to enhance SAR for linker optimization, by using 1D NMR guided SAR to understand the effect of a methyl scan on the linker. Potency will be affected by entropic penalties to binding (conformational flexibility), energy penalties for free ligand conformational rearrangements required for binding (wrong conformation), ligand–protein complementarity (sterics and/or loss or gain of polar interactions), and solvation/desolvation contributions. The individual contributions from each of these different factors are not apparent from the measured enzyme potency values. Separating free ligand preferences for the bioactive conformation from bound ligand complementarity to protein enhances understanding of linker SAR. <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a> demonstrates how to deconvolute potency contributions to help focus design strategies aimed to maximize bioactivity. The 1D NMR conformational signature of each molecule shown in this example has a typical difference in chemical shift between bioactive and nonbioactive conformations of ∼1 ppm due to differences arising from through-space ring current effects, as described above for this series (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). In this example, four different linkers that vary in the methyl group position all result in rigid molecules by 1D NMR (complex spectra in the aliphatic region, <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). Three of the compounds are highly populated with the bioactive conformation as demonstrated by the presence of the bioactive reporter signal. An enhanced SAR analysis using free ligand 3D structural information can be achieved with routine 1D NMR spectra acquired for the initial purpose of compound registration. Comparisons across ligands are best made with the same solvent under similar conditions such as temperature, hydration, and protonation states, as these can influence the extent of preorganization and the preferred conformation. This type of standardization works well with automated NMR spectrometers.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Macrocycle matched pair Mcl-1 inhibitors, with NMR bioactivity reporter signals marked by asterisks color coded to correspond to bioactive (green) or nonbioactive (red) conformations (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). Deconvoluting potency contributions arising from free ligand conformations versus ligand–protein complementarity enhance structure–activity analyses. (a) 2D structure with a bioactivity reporter hydrogen atom marked explicitly and the 3D bioactive conformation with the same hydrogen atom shown in white for compound <b>2</b> (R1 = R2 = H). (b) R1 = (<i>R</i>)-CH<sub>3</sub> and R2 = H, bioactive conformation. (c) R1 = (<i>S</i>)-CH<sub>3</sub> and R2 = H, bioactive conformation. (d) Epimer 1, R1 = H and R2 = CH<sub>3</sub>, nonbioactive conformation. (e) Epimer 2, R1 = H and R2 = CH<sub>3</sub>, bioactive conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The ability to experimentally observe conformational flexibility in solution with atomic-level resolution provides a more complete diagnosis of inadequate bioactivity and is not available by any other methodology. Two linker designs with similarly poor potency may need different strategies for improvement of potency. These macrocycles demonstrate the ability to quickly access atomic resolution experimental data on free ligand flexibility and how to use it to enhance SAR understanding during linker optimization. The combination of NMR signatures and binding affinities, through comparison of matched pairs or more broadly with compounds in a series, may inspire alternative design hypotheses ultimately focusing synthetic efforts on designs with higher success rates.</div><div class="NLM_p">The BCL6 series provides a further example of using methylation to introduce rigidity into a linker. In <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>, the SAR analyses of matched pair BCL6 macrocycles are reported. The molecule <b>13</b> has 7.6-fold lower affinity compared to the methylated analog <b>14</b>. The increase in potency is not solely attributable to the +0.4 unit increase in log <i>D</i>. Additionally, the racemate of <b>14</b> is a weak binder (BCL6 TR-FRET IC<sub>50</sub> > 10 μM). This supports our thesis that affinity is driven by an increased population of the bioactive state of the free ligand. This can be assessed directly by inspecting the aliphatic region as a marker of preorganization, without the need of a full NMR conformational analysis.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. A matched pair of BCL6 inhibitors (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 500 MHz). (a) The free ligand <b>13</b> shows conformationally flexibility. (b) Linker methylation rigidifies the free ligand <b>14</b> which adopts the bioactive conformation in solution.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Free Ligand Conformations and Structure Kinetics Relationships</h3><div class="NLM_p">Biophysical and biochemical methods can provide high quality data on the kinetics of protein–ligand interactions.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> One challenge, though, is how to incorporate these data into the design of new and improved compounds. As conformation, desolvation and binding site complementarity influence on- and off-rate constants;<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> combining NMR conformational signatures and X-ray structures of bound ligands with experimental on- and off-rate constants can provide insights suitable to guide design. Combined structural and biophysical data can focus design toward those specific molecular determinants of drug–target binding with the most to gain from optimization efforts.</div><div class="NLM_p">The Mcl-1 binders encountered in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> provide an interesting case study. This matched pair exhibits dramatic conformational differences upon small structural changes (N-methylation). Biophysical SPR data are available for comparison to tease out the contributions of ligand preorganization vs protein complementarity on binding potency. The two compounds are both rigidly preorganized in solution. Only the N-methylated amide <b>2</b> adopts the bioactive conformation; the NH amide <b>8</b> is highly populated in a nonbioactive conformation. Only the bioactive conformations of the free ligand will bind the target; hence once bound, protein complementarity for the two compounds is nearly the same, as evaluated by the similar experimental off-rate constants (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The similar off-rates mean that the ∼10-fold difference in binding affinity is driven entirely by a ∼10-fold change in on-rate kinetics.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Potency Is Driven by Free-Ligand Preorganization into the Bioactive Conformation</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">structural comments</th><th class="colsep0 rowsep0" align="center">free ligand conformation</th><th class="colsep0 rowsep0" align="center">log <i>D</i> (pH 7.4)</th><th class="colsep0 rowsep0" align="center">SPR <i>K</i><sub>d</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>on</sub> (μM<sup>–1</sup> s<sup>–1</sup>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>off</sub> (ms<sup>–1</sup>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">bioactive</td><td class="colsep0 rowsep0" align="left">3.1</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">N-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">bioactive</td><td class="colsep0 rowsep0" align="left">4.3<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">370</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">212</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">bioactive</td><td class="colsep0 rowsep0" align="left">2.2</td><td class="colsep0 rowsep0" align="left">6.5</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">8.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">acyclic (<b>1</b>)</td><td class="colsep0 rowsep0" align="left">flexible</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left">480</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">280</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">acyclic (<b>1</b>)</td><td class="colsep0 rowsep0" align="left">flexible</td><td class="colsep0 rowsep0" align="left">4.2</td><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">NH</td><td class="colsep0 rowsep0" align="left">nonbioactive</td><td class="colsep0 rowsep0" align="left">3.7<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">4480</td><td class="colsep0 rowsep0" align="left">0.05</td><td class="colsep0 rowsep0" align="left">158</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">N-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">bioactive</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="left">66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">acyclic</td><td class="colsep0 rowsep0" align="left">flexible</td><td class="colsep0 rowsep0" align="left">–0.4</td><td class="colsep0 rowsep0" align="left">1100</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">770</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">bioactive</td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">140</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Mean of at least two independent measurements.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">log <i>D</i> = 4.30, clogD = 4.31.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">clogD.</p></div></div></div><div class="NLM_p">The larger <i>k</i><sub>on</sub> of <b>2</b> correlates with the higher population of the bioactive conformation in solution. Preorganization into the bioactive conformation is expected to translate to a faster on-rate arising from an increased effective concentration of the free ligand bioactive conformation. The free energy penalty on the <i>trans</i> NH isomer originates from the need to rearrange into the bioactive conformation prior to binding. Consequently, the <i>trans</i> NH free ligand <b>8</b> disfavors effective interactions with the protein (unlike the <i>cis</i> N-CH<sub>3</sub> isomer), and this results in poorer on-rate kinetics. This shows the power of combining experimental free ligand conformational information with SPR data to validate hypotheses.<a onclick="showRef(event, 'ref36 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38 ref39">(36−39)</a></div><div class="NLM_p">The correlation of <i>k</i><sub>on</sub> with free ligand preorganization implies 1D NMR signatures can augment structure–kinetic relationships; when <i>k</i><sub>on</sub> is optimized by highly populated bioactive states, potency will be driven by <i>k</i><sub>off</sub>. Disentangling ligand preorganization from protein–ligand complementarity provides a method to evaluate the effectiveness of putative engagement of exposed hydrophobic and polar functional groups to increase residence times. In <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>, two structurally different Mcl-1 inhibitors are compared. Both molecules are preorganized in solution into the preferred binding mode, and consequently, on-rate kinetic constants are similar (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The 100-fold difference in affinity correlates with the difference in the off-rate, which must reflect a greater protein complementarity/an ability to form further specific hydrophilic and hydrophobic interactions with <b>1</b>, specifically with the Arg-263 residue.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Structure–kinetic relationships for Mcl-1 inhibitors that preorganize into the bioactive conformation as free ligands in solution. (a) <b>15</b> (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 400 MHz). The bioactive conformation was determined from docking into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MES">5MES</a> (which has a chlorine atom in place of the methyl sulfone; see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (b) <b>1</b> (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz), adopts the bioactive conformation seen in PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS0">6FS0</a>. The two molecules (<b>1</b> and <b>15</b>) have similar on-rate kinetic constants, as expected from ligand preorganization into the bioactive state, so the 100-fold difference in affinity is reflected in the 100-fold difference in off-rate kinetics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">McCoull et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> used this strategy to increase affinity of BCL6 macrocyclic inhibitors. Three BCL6 inhibitors are compared in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>. The flexible acyclic molecule <b>16</b> has the lowest on-rate kinetics, in line with a low population of the bioactive state. In contrast, the macrocyclic analogs <b>17</b> and <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>b and <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>c) are fully preorganized into the bioactive conformation, as can be inferred from the 1D NMR signature for a rigid conformation and the strong binding affinity as measured by the dissociation constant (preorganization into the wrong conformation results in poor binding affinity). The on-rate constants for <b>17</b> and <b>3</b> are of the same order of magnitude (5.5 × 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup> vs 1.6 × 10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup>). The difference in binding affinity between <b>17</b> and <b>3</b> is dictated by a 17-fold decrease in off-rate (0.140 s<sup>–1</sup> vs. 0.0083 s<sup>–1</sup>). A docked pose of the macrocycle in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>b suggested addition of a polar group to the pyrrolidine on the ligand could pick up an extra protein interaction with the proximal Arg-28 residue. Off-rate kinetics were reduced by incorporating an (<i>S</i>)-methanol group to interact favorably with a stable conserved water close to the Arg residue. This decrease in off-rate and improvement in potency are thus derived from increased protein complementarity.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structure–kinetics relationships for BCL6 inhibitors. (a) A flexible acyclic inhibitor (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 30 °C, 400 MHz) with suboptimal on-rate kinetics. Protein–ligand complex crystal structure is PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21">5N21</a>. (b) A cyclized analog (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 30 °C, 400 MHz) with 50-fold higher affinity significantly assisted by preorganization into the bioactive conformation, <b>17</b> is shown docked into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21">5N21</a> (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (c) A further 3.3-fold improvement in affinity is dominated by off-rate kinetics by increasing ligand–protein interactions. Protein–ligand complex crystal structure is PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N1Z">5N1Z</a>. An increase in protein complementarity arises from the engagement of ligand to protein Arg-28 polar interactions, leading to increased residence time.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">These three examples show that structure–kinetics relationships can be rationalized if structural information from both the free ligand and the protein-bound ligand is available. Improvement of ligand design using rigidification strategies that increase the population of free ligand preferentially preorganized into the bioactive conformation increases ligand affinity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Three flexible acyclic ligands, with interconversion rates of milliseconds (broad 1D NMR line widths) and nanoseconds (narrow 1D NMR line widths), are an order of magnitude smaller in <i>k</i><sub>on</sub> compared to the potent bioactive preorganized macrocycles. Notably, optimization of the free ligand conformation into the bioactive conformation drives on-rate binding kinetics, while optimization of protein ligand complementarity most strongly influences off-rate kinetics. Interestingly, the three structurally unrelated potent macrocyclic PPI inhibitors of Mcl-1 and BCL6 are bioactive molecules and display on-rate kinetics of the same order of magnitude of ∼10<sup>6</sup> M<sup>–1</sup> s<sup>–1</sup>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Prediction Methods for Linker Design Based on Free Ligand Conformation</h3><div class="NLM_p">The examples herein demonstrate that preferred free ligand 3D conformations determined from NMR signatures are a valuable means of relating the preorganization effect to observed affinity changes. Having established this relationship, it would be beneficial to be able to predict the linkers that stabilize the free ligand conformation prior to synthesis, enabling chemists to prioritize only macrocyclic compounds with the correct bioactive conformation. Though a single major conformation may dominate, manifesting as distinct NMR signatures, the major conformation of a ligand is still dynamic and exists as an ensemble of similar conformations in rapid equilibrium. Accurate representation of this ensemble and the relative conformation populations therein is the goal of computational conformation prediction methods for molecules in design.</div><div class="NLM_p">In general, there are two classes of techniques used to predict free ligand conformation ensembles: rule-based techniques,<a onclick="showRef(event, 'ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42">(40−42)</a> whereby the dihedral angle of each bond is systematically rotated through predefined energy minima, and simulation-based techniques<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> where the macrocycle is allowed to move and explore conformational space defined by a molecular mechanics force field. The former technique is much faster (typically seconds per molecule) but is hampered by poorly defined internal strain energy of each conformation. Without an accurate energy, it is impossible to determine relative populations of each conformation. Furthermore, these rule-based methods have been developed primarily for acyclic molecules and perform poorly for macrocyclic ligands.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> The latter technique has the potential to be much more accurate, directly quantifying conformation populations as the summation of the frames of a molecular dynamics simulation and being applicable to both acyclic and macrocyclic ligands. However, to ensure complete sampling of the conformational space, particularly in the case of conformationally restricted macrocycles, long simulations (typically hours per molecule) are required.</div><div class="NLM_p">McCoull et al.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> used an accelerated simulation method to determine populations accurately in less time.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The mixed Monte Carlo/molecular dynamics (MC/MD) method uses MC for the initial randomized atom positions, followed by a short MD run and minimization to define each conformation. This was repeated 10 000 times for each molecule to ensure adequate sampling and that an equilibrium conformational population had been obtained. Rather than cluster conformations as representative averages, calculations were performed on the entire ensemble. The RMSD was calculated for all heavy atoms in each conformation (except those in the variable linker) compared to the known bioactive conformation from X-ray crystal structures. It was observed that the proportion of the ensemble with low RMSD was large when affinity of the ligands for BCL6 was improved and was consistent with NMR signals showing preorganization of the ligand for binding (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>). Though an imperfect correlation, this model was subsequently used to prioritize linker designs for synthesis and to optimize ligands for potency. It is interesting to note that compound <b>3</b> is an outlier to the trend. Most likely this is a consequence of an additional protein–ligand interaction with the proximal Arg-28 residue, as stated above.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. BCL6 FRET affinity as a function of conformation ensemble RMSD. (a) Relationship between IC<sub>50</sub> and calculated conformation population for a set of compounds with an identical binding pharmacophore, where the line is a linear fit, <i>R</i><sup>2</sup> = 0.49. (b) Depiction of RMSD distribution to bioactive conformation for compound <b>16</b>, showing a small population at low RMSD. (c) Depiction of RMSD distribution to bioactive conformation for compound <b>17</b>, showing a higher population at low RMSD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03565" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03565" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, we have shown that free ligand conformational preferences can be derived from routinely available 1D NMR spectra by using a validation approach relying on QM calculation of conformational proposals and NMR parameters. We have presented three macrocyclic structure-based drug design cases enhanced by the ready knowledge of free ligand conformations. The opportunity for macrocyclization as a ligand rigidification strategy was highlighted by the experimental knowledge that the acyclic inhibitors were highly flexible in solution, while the linker design was informed by the availability of the protein–ligand crystal structure. Effective real time lead optimization with 3D-based structure–activity and structure–kinetics relationships was obtained for all synthesized compounds by using routinely available 1D NMR spectra as markers of conformational preference. The resulting optimized macrocyclic inhibitors, including the clinical candidate AZD5991 (<b>1</b>), are highly preorganized in solution and display exquisite potency and selectivity as PPI inhibitors, suggesting that ligand rigidity achieved by macrocyclizaton is an important component of their activity. Interestingly, inspection of ligand kinetics reveals that structurally different preorganized bioactive macrocyclic PPI inhibitors have on-rate kinetics within the same order of magnitude, suggesting that ligand affinity of preorganized macrocycles is dictated by off-rate kinetics. Notably for the clinical candidate <b>1</b> and the DEL-derived series <b>2</b> the 100-fold difference in affinity and off-rate kinetics reflects a greater protein complementarity and an ability to form further specific hydrophilic and hydrophobic interactions with <b>1</b>, specifically with the Arg-263 residue. Conversely, locking a macrocycle into a nonbioactive conformation is detrimental to affinity and more specifically to on-rate kinetics, as demonstrated by the methylated amide matched pair <b>2</b> and <b>8</b>.</div><div class="NLM_p last">In conclusion, we have shown that the ability to read 1D NMR signatures of the free macrocyclic ligands in solution provides medicinal chemists with a quick analytical marker of conformational flexibility and molecular shape and can guide medicinal chemistry design decisions during lead optimization while focusing synthetic efforts on productive design hypotheses. 1D NMR conformational reporter signals and/or broader signal fingerprints across a range of compounds within a series can be used to assess whether conformational bioactivity has been retained or successfully imparted to newly designed molecules or conversely if the ligand has adopted a nonbioactive conformation. The inherent simplicity of using routinely available data for every synthesized compound confers a practical advantage of accuracy and speed to enable NMR data to influence design decisions between design cycles of a lead optimization project. In our experience, the ability to infer free ligand conformation from routine 1D NMR spectra is applicable to a broad range of drug modalities, whether the ligands are small molecules or peptides or more recently established modalities, such as PROTACs.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> This approach is compatible with the analysis of compound libraries and opens the door to automation with machine learning approaches, with notable NMR-derived conformational signatures identified without human intervention.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General</h3><div class="NLM_p">Proton magnetic resonance spectra were acquired on Bruker spectrometers, ranging from 300–600 MHz proton resonance frequency, with standard Bruker pulse sequences. Unless stated otherwise, spectra were typically recorded at 1−5 mM in DMSO-<i>d</i><sub>6</sub> at 298−300 K, with shifts referenced to the residual proton solvent signal at 2.5 ppm. ACD/Labs<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and MestreLabs<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Research software packages were used for analysis, plotting, and 2D empirical predictions (HOSE code algorithms accessed from ACD/Labs). Piperidine (Aldrich, 99.5%) and 2-methylpiperidine (Acros, 99%) solutions used 0.05 mL of compound as received into 0.75 mL of DMSO-<i>d</i><sub>6</sub>, and spectra were acquired with a Bruker Avance III 600. AstraZeneca macrocycles were obtained internally, and details of the syntheses can be found in the cited literature. <b>1</b> atropisomers were separated by chiral SFC as follows: column, Chiralpak ID (4.6 mm × 100 mm); mobile phase, CO<sub>2</sub>–EtOH (60:40) at 5.0 mL/min. Kinetics data were obtained using a Biacore direct binding assay and immobilization of target via nickel capture and NHS/EDC amine coupling to a NTA Biacore sensor chip; analyses were performed in the Biacore T200 BIA evalutation software. Images of ligand–target complexes were generated using MOE.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The log <i>D</i> values were determined at pH 7.4 using shake flask methodologies. Alternatively, clogD values (<b>2</b> and <b>8</b>) were calculated using a conformal SVM signature model based on measured data of compounds within and beyond the same series.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><div class="NLM_p last">Conformational searches to visualize flexibility by NMR signatures used a lowModeMD search on acyclic compounds using MOE with default values (7 kcal/mol energy window). Conformational searches on macrocycles were performed in Schrodinger/Maestro<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> or MOE.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Full conformational analysis of free ligands in solution used ROESY distance constraints acquired with Bruker spectrometers and analyzed using either ACD/Labs<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> or MNova.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Conversions from NOEs to distances was done in an Excel spreadsheet. MSpin<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> was used to select the best match between conformer sets and experimental NMR distances. Quantum mechanical calculations were carried out using Gaussian 16.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Geometry optimization and NMR chemical shift tensors employed the GIAO DFT method at the B3LYP/6-31G(d,p) level with PCM solvent modeling in DMSO (same as experimental NMR). Chemical shifts were referenced to TMS. Preoptimized structures were extracted from the X-ray structures of the protein-bound ligands. Bound ligands shown in figures as structural analogs were docked into the given crystal structure using MOE<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> template docking.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry</h3><div class="NLM_p last">Unless otherwise stated, commercially available reagents were used as supplied. All reactions requiring anhydrous conditions were conducted in dried apparatus under an atmosphere of nitrogen. <sup>1</sup>H NMR spectra were recorded using a Bruker AVIII 300 or AVIIIHD 400 NMR. Chemical shifts δ are reported in ppm, and multiplicity of signals are denoted s = singlet, d = doublet, t = triplet, and m = multiplet. Mass spectra were recorded using a Shimadzu LCMS-2020 instrument (ESI+).</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> (<i>Z</i>)-4,11-Dimethyl-12-oxo-11,12,13,14-tetrahydro-10-oxa-2,4-diaza-1(6,8)-quinolina-3(7,5)-pyrazolo[1,5-<i>a</i>]pyrimidinacyclodecaphane-33-carbonitrile (<b>13</b>), <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a></h3><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> <i>tert</i>-Butyl (5-((6-Nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)carbamate (<b>18</b>)</h4><div class="NLM_p">K<sub>2</sub>CO<sub>3</sub> (617 mg, 4.47 mmol) was added to 8-hydroxy-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (310 mg, 1.49 mmol) and <i>tert</i>-butyl (5-bromopentyl)carbamate (595 mg, 2.23 mmol) in acetone (20 mL) at 18 °C under nitrogen. The resulting mixture was stirred at 50 °C for 2 h. The resultant mixture was concentrated under vacuum. The reaction mixture was diluted with ethyl acetate (100 mL) and washed sequentially with water (100 mL) and saturated brine (50 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography,with elution gradient 0–5% MeOH in DCM to afford the title compound (580 mg, 99%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.44 (s, 9H), 1.46–1.67 (m, 4H), 1.84–1.92 (m, 2H), 2.72 (t, <i>J</i> = 5.4 Hz, 2H), 3.07 (t, <i>J</i> = 5.4 Hz, 2H), 3.10–3.19 (br, 2H), 4.13 (t, <i>J</i> = 5.1 Hz, 2H), 7.66 (s, 1H), 7.79 (s, 1H), 7.94 (br, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 394.<named-content content-type="anchor" rid="sch1" type="simple"></named-content></div><figure id="sch1" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, acetone, 50 °C, 99%; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 99%; (c) MeI, NaH, DMF, rt, 38%; (d) H<sub>2</sub>, Pd/C, MeOH, rt, 81%; (e) <b>21</b>, DIPEA, THF, rt; (f) HCl, EtOH, rt, 92%; (g) DIPEA, DMA, 130 °C, 27%.</p></p></figure></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>tert</i>-Butyl Methyl (5-((1-methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)carbamate (<b>19</b>)</h4><div id="sec4_2_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> Step 1: <i>tert</i>-Butyl (5-((1-Methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)carbamate</h5><div class="NLM_p last">Iodomethane (725 mg, 5.11 mmol) was added to <i>tert</i>-butyl (5-((6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)carbamate (670 mg, 1.70 mmol) and K<sub>2</sub>CO<sub>3</sub> (706 mg, 5.11 mmol) in DMF (20 mL) at 18 °C under air. The resulting mixture was stirred at 80 °C for 2 h. The reaction mixture was diluted with ethyl acetate (200 mL) and washed sequentially with water (200 mL) and saturated brine (100 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography with elution gradient 0–5% MeOH in DCM to afford the title compound (687 mg, 99%) as a yellow solid. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 408.</div></div><div id="sec4_2_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> Step 2: <i>tert</i>-Butyl Methyl-(5-((1-methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)carbamate</h5><div class="NLM_p last">Iodomethane (207 mg, 1.46 mmol) was added to <i>tert</i>-butyl (5-((1-methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)carbamate (595 mg, 1.46 mmol) and NaH (105 mg, 4.38 mmol) in DMF (30 mL) at 18 °C under air. The resulting mixture was stirred at 18 °C for 14 h. The reaction mixture was quenched with saturated NaHCO<sub>3</sub> (100 mL), extracted with ethyl acetate (2 × 100 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a yellow oil. The crude product was purified by flash C18-flash chromatography with elution gradient 0–30% MeCN in water (0.1% formic acid). Pure fractions were evaporated to dryness to afford the title compound (235 mg, 38%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.46 (s, 9H), 1.86–1.96 (m, 2H), 2.60–2.64 (m, 1H), 2.85 (d, <i>J</i> = 4.2 Hz, 2H), 2.89 (s, 3H), 2.90–2.96 (m, 2H), 2.97 (s, 3H), 3.24 (q, <i>J</i> = 6 Hz, 2H), 3.44 (s, 2H), 4.09 (t, <i>J</i> = 6.4 Hz, 2H), 7.76–7.67 (m, 1H), 8.03 (s, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 422.</div></div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (<i>Z</i>)-4,11-Dimethyl-12-oxo-11,12,13,14-tetrahydro-10-oxa-2,4-diaza-1(6,8)-quinolina-3(7,5)-pyrazolo[1,5-<i>a</i>]pyrimidinacyclodecaphane-33-carbonitrile (<b>13</b>)</h4><div id="sec4_2_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> Step 1: <i>tert</i>-Butyl (5-((6-Amino-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)(methyl)carbamate (<b>20</b>)</h5><div class="NLM_p last">A mixture of Pd/C (58 mg, 0.55 mmol) and <i>tert</i>-butyl methyl-(5-((1-methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)carbamate (230 mg, 0.55 mmol) in MeOH (10 mL) was stirred under an atmosphere of hydrogen at 18 °C for 1 h. The reaction mixture was filtered and concentrated under vacuum. The crude product was purified by flash silica chromatography with elution gradient 0–5% MeOH in DCM to afford the title compound (172 mg, 81%) as a yellow oil. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 392.</div></div><div id="sec4_2_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Step 2: <i>tert</i>-Butyl (5-((6-((5-Chloro-3-cyanopyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)amino)-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)(methyl)carbamate</h5><div class="NLM_p last">DIPEA (0.107 mL, 0.61 mmol) was added to <i>tert</i>-butyl (5-((6-amino-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)(methyl)carbamate (80 mg, 0.20 mmol) and 5,7-dichloropyrazolo[1,5-<i>a</i>]pyrimidine-3-carbonitrile (65.3 mg, 0.31 mmol) in THF (16 mL) at 25 °C under air. The resulting mixture was stirred at 25 °C for 3 h and then concentrated to afford the crude title product (105 mg, 90%) that was used in the next step without further purification. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 568.</div></div><div id="sec4_2_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Step 3: 5-Chloro-7-((1-methyl-8-((5-(methylamino)pentyl)oxy)-2-oxo-1,2,3,4-tetrahydroquinolin-6-yl)amino)pyrazolo[1,5-<i>a</i>]pyrimidine-3-carbonitrile Hydrochloride</h5><div class="NLM_p last">A solution of hydrogen chloride in EtOH (1.9 M, 1 mL) was added to a stirred mixture of <i>tert</i>-butyl (5-((6-((5-chloro-3-cyanopyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)amino)-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)pentyl)(methyl)carbamate (105 mg, 0.18 mmol) in EtOH (1 mL) at 18 °C under air. The resulting mixture was stirred at 18 °C for 5 h. The mixture was concentrated and azeotroped with toluene, then evaporated to dryness to afford the title compound (80 mg, 92%) as a yellow solid. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 468.</div></div><div id="sec4_2_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Step 4: (<i>Z</i>)-4,11-Dimethyl-12-oxo-11,12,13,14-tetrahydro-10-oxa-2,4-diaza-1(6,8)-quinolina-3(7,5)-pyrazolo[1,5-<i>a</i>]pyrimidinacyclodecaphane-33-carbonitrile (<b>13</b>)</h5><div class="NLM_p last">5-Chloro-7-((1-methyl-8-((5-(methylamino)pentyl)oxy)-2-oxo-1,2,3,4-tetrahydroquinolin-6-yl)amino)pyrazolo[1,5-<i>a</i>]pyrimidine-3-carbonitrile (80 mg, 0.17 mmol) was added to DIPEA (0.09 mL, 0.51 mmol) in DMA (12 mL) under nitrogen. The resulting mixture was stirred at 130 °C for 4 h, and then the mixture was concentrated. The crude product was purified by preparative HPLC (Waters XBridge Prep C18 OBD column, 5 μm silica, 19 mm diameter, 100 mm length), using decreasingly polar mixtures of water (containing 0.05% formic acid) and MeCN as eluents to afford the title compound (20 mg, 27%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.24–1.31 (br, 2H), 1.69–1.76 (m, 4H), 2.33–2.46 (m, 2H), 2.76–2.79 (m, 2H), 3.11 (s, 3H), 3.17 (s, 3H), 3.30–3.43 (m, 2H), 4.32 (br, 2H), 6.21 (s, 1H), 6.83 (s, 1H), 7.42 (s, 1H), 8.39 (s, 1H), 9.81 (br, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 432.</div></div></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> (<i>R</i>,<i>Z</i>)-4,9,11-Trimethyl-12-oxo-11,12,13,14-tetrahydro-10-oxa-2,4-diaza-1(6,8)-quinolina-3(7,5)-pyrazolo[1,5-<i>a</i>]pyrimidinacyclodecaphane-33-carbonitrile (<b>14</b>), <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a></h3><div id="sec4_2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 6-((<i>tert</i>-Butyldimethylsilyl)oxy)hexan-2-ol (<b>22</b>)</h4><div class="NLM_p"><i>tert</i>-Butylchlorodimethylsilane (2.55 g, 16.92 mmol) was added to 1<i>H</i>-imidazole (0.576 g, 8.46 mmol) and hexane-1,5-diol (1 g, 8.46 mmol) in DMF (10 mL) under nitrogen. The resulting mixture was stirred at 25 °C for 20 h. The reaction mixture was quenched with water (50 mL) and extracted with EtOAc (3 × 50 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a yellow liquid. The crude product was purified by flash silica chromatography with elution gradient 1–20% petroleum ether in ethyl acetate to afford the title compound (1.65 g, 84%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 0.01 (s, 6H), 0.85 (s, 9H), 1.15 (d, <i>J</i> = 6 Hz, 3H), 1.37–1.55 (m, 6H), 3.64 (t, <i>J</i> = 6 Hz, 2H), 3.78 (br, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 233.<named-content content-type="anchor" rid="sch2" type="simple"></named-content></div><figure id="sch2" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TBSCl, imidazole<sub>cat</sub>, rt, 84%; (b) diethyl azodicarboxylate, Ph<sub>3</sub>P, THF, rt, 68%; (c) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 97%; (d) TBAF, THF, rt, quant; (e) PCC, CH<sub>2</sub>Cl<sub>2</sub>, rt, 60%; (f) MeNH<sub>2</sub>, NaBH<sub>3</sub>CN AcONa, MeOH, 96%; (g) K<sub>2</sub>CO<sub>3</sub>, Boc<sub>2</sub>O, THF, H<sub>2</sub>O, rt, quant; (h) H<sub>2</sub>, Pd/C, MeOH, rt, 81%; (i) <b>21</b>, DIPEA, THF, rt, 66%; (j) HCl, EtOH, rt, 95%; (k) DIPEA, DMA, 125 °C, 23%.</p></p></figure></div><div id="sec4_2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 8-((6-((<i>tert</i>-Butyldimethylsilyl)oxy)hexan-2-yl)oxy)-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (<b>23</b>)</h4><div class="NLM_p last">Diethyl azodicarboxylate (0.627 mL, 3.96 mmol) was added to triphenylphosphine (1.04 g, 3.96 mmol), 6-((<i>tert</i>-butyldimethylsilyl)oxy)hexan-2-ol (737 mg, 3.17 mmol) and 8-hydroxy-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (550 mg, 2.64 mmol) in THF (31 mL) under nitrogen. The resulting mixture was stirred at 25 °C for 20 h. The reaction mixture was quenched with water (20 mL), extracted with ethyl acetate (3 × 50 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford yellow oil. The crude product was purified by flash silica chromatography with elution gradient 1–20% EtOAc in petroleum ether to afford the title compound (762 mg, 68%) as a yellow oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 0.03 (s, 6H), 0.88 (s, 9H), 1.25 (d, <i>J</i> = 6 Hz 3H), 1.26–1.31 (m, 2H), 1.43–1.50 (m, 2H), 1.51–1.52 (m, 2H), 2.70 (t, <i>J</i> = 6 Hz, 2H), 3.06 (t, <i>J</i> = 6 Hz, 2H), 4.25–4.31 (m, 1H), 4.49–4.52 (m, 2H), 7.64 (s, 1H), 7.75 (s, 1H), 7.96 (br, 1H); MS (ESI) [M + H]+ <i>m</i>/<i>z</i> 423.</div></div><div id="sec4_2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 8-((6-Hydroxyhexan-2-yl)oxy)-1-methyl-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one</h4><div id="sec4_2_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Step 1: 8-((6-((<i>tert</i>-Butyldimethylsilyl)oxy)hexan-2-yl)oxy)-1-methyl-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one</h5><div class="NLM_p last">Iodomethane (758 mg, 5.34 mmol) was added to K<sub>2</sub>CO<sub>3</sub> (738 mg, 5.34 mmol) and 8-((6-((<i>tert</i>-butyldimethylsilyl)oxy)hexan-2-yl)oxy)-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (752 mg, 1.78 mmol) in DMF (10 mL) at 18 °C under air. The resulting mixture was stirred at 80 °C for 2 h. The reaction mixture was diluted with ethyl acetate (100 mL) and washed sequentially with water (2 × 100 mL) and saturated brine (100 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography with elution gradient 0–5% MeOH in DCM to afford the title compound (750 mg, 97%) as a yellow solid. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 437.</div></div><div id="sec4_2_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Step 2: 8-((6-Hydroxyhexan-2-yl)oxy)-1-methyl-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one</h5><div class="NLM_p last">Tetra-<i>n</i>-butylammonium fluoride trihydrate (535 mg, 1.69 mmol) was added to 8-((6-((<i>tert</i>-butyldimethylsilyl)oxy)hexan-2-yl)oxy)-1-methyl-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (740 mg, 1.69 mmol) in THF (20 mL) at 18 °C under air. The resulting mixture was stirred at 18 °C for 14 h. The reaction mixture was concentrated under vacuum. The reaction mixture was diluted with ethyl acetate (100 mL) and washed sequentially with water (3 × 100 mL) and saturated brine (100 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography with elution gradient 0–10% MeOH in dichloromethane to afford the title compound (560 mg, quant) as a yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.21 (d, <i>J</i> = 6 Hz, 3H), 1.22–1.26 (m, 2H), 1.48–1.52 (m, 4H), 2.64 (t, <i>J</i> = 6 Hz, 2H), 2.94 (t, <i>J</i> = 6 Hz, 2H), 3.45 (s, 3H), 4.20–4.24 (m, 2H), 4.51–4.54 (m, 1H), 7.29 (s, 1H), 7.72 (s, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 323.</div></div></div><div id="sec4_2_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>tert</i>-Butyl (5-((6-Amino-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexyl)(methyl)carbamate</h4><div id="sec4_2_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Step 1: 5-((1-Methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexanal</h5><div class="NLM_p last">PCC (552 mg, 2.56 mmol) was added to 8-((6-hydroxyhexan-2-yl)oxy)-1-methyl-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (550 mg, 1.71 mmol) in dichloromethane (100 mL) at 0 °C over a period of 5 min under air. The resulting mixture was stirred at 18 °C for 14 h. The reaction mixture was filtered and concentrated in vacuo. The crude product was purified by flash silica chromatography with elution gradient 0–5% MeOH in DCM to afford the title compound (438 mg, 80%) as a yellow solid. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 321.</div></div><div id="sec4_2_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Step 2: 1-Methyl-8-((6-(methylamino)hexan-2-yl)oxy)-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (<b>24</b>)</h5><div class="NLM_p last">Sodium cyanoborohydride (235 mg, 3.75 mmol) was added to 5-((1-methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexanal (400 mg, 1.25 mmol), sodium acetate (307 mg, 3.75 mmol), and methanamine hydrochloride (253 mg, 3.75 mmol) in MeOH (50 mL) at 18 °C under air. The resulting mixture was stirred at 18 °C for 15 h. The resulting mixture was concentrated under vacuum. The reaction mixture was diluted with ethyl acetate (100 mL) and washed sequentially with water (100 mL) and saturated brine (100 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography with elution gradient 0–5% MeOH in dichloromethane to afford the title compound (400 mg, 96%) as a yellow solid. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 336.</div></div><div id="sec4_2_2_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Step 3: <i>tert</i>-Butyl Methyl (5-((1-methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexyl)carbamate</h5><div class="NLM_p last">K<sub>2</sub>CO<sub>3</sub> (482 mg, 3.49 mmol) was added to BOC-anhydride (0.540 mL, 2.33 mmol) and 1-methyl-8-((6-(methylamino)hexan-2-yl)oxy)-6-nitro-3,4-dihydroquinolin-2(1<i>H</i>)-one (390 mg, 1.16 mmol) in THF (20 mL) and water (10 mL) at 20 °C under air. The resulting mixture was stirred at 20 °C for 14 h. The reaction mixture was diluted with ethyl acetate (100 mL) and washed sequentially with water (100 mL) and saturated brine (75 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford crude product. The crude product was purified by flash silica chromatography with elution gradient 0–5% MeOH in DCM to afford the title compound (505 mg, 100%) as a yellow solid. MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 436.</div></div><div id="sec4_2_2_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Step 4: <i>tert</i>-Butyl (5-((6-Amino-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexyl)(methyl)carbamate (<b>25</b>)</h5><div class="NLM_p last">Pd/C (122 mg, 1.15 mmol) and <i>tert</i>-butyl methyl (5-((1-methyl-6-nitro-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexyl)carbamate (500 mg, 1.15 mmol) in MeOH (100 mL) was stirred under an atmosphere of hydrogen at 20 °C for 1 h. The reaction mixture was filtered and concentrated in vacuo. The reaction mixture was concentrated under vacuum to afford the title compound (410 mg, 88%) as yellow solid. <sup>1</sup>H NMR (300 MHz, MeOH-<i>d</i><sub>4</sub>) δ 1.19–1.37 (m, 4H), 1.46 (s, 9H), 1.51–1.86 (m, 4H), 2.43–2.51 (m, 2H), 2.66–2.76 (m, 2H), 2.84 (s, 3H), 3.25 (t, <i>J</i> = 6 Hz, 2H), 3.32 (s, 3H), 6.21(s, 1H), 6.37 (s, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 406.</div></div></div><div id="sec4_2_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>R</i>,<i>Z</i>)-4,9,11-Trimethyl-12-oxo-11,12,13,14-tetrahydro-10-oxa-2,4-diaza-1(6,8)-quinolina-3(7,5)-pyrazolo[1,5-<i>a</i>]pyrimidinacyclodecaphane-33-carbonitrile (<b>14</b>)</h4><div id="sec4_2_2_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Step 1: <i>tert</i>-Butyl (5-((6-((5-Chloro-3-cyanopyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)amino)-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexyl)(methyl)carbamate</h5><div class="NLM_p last">DIPEA (0.26 mL, 1.48 mmol) was added to <i>tert</i>-butyl (5-((6-amino-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexyl)(methyl)carbamate (200 mg, 0.49 mmol) and 5,7-dichloropyrazolo[1,5-<i>a</i>]pyrimidine-3-carbonitrile (126 mg, 0.59 mmol) in THF (5 mL) at 22 °C under air. The resulting mixture was stirred at 22 °C for 3 h. The crude product was purified by flash silica chromatography with elution gradient 0–5% MeOH in DCM to afford the title compound (190 mg, 66%) as a yellow solid.</div></div><div id="sec4_2_2_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Step 2: 5-Chloro-7-((1-methyl-8-((6-(methylamino)hexan-2-yl)oxy)-2-oxo-1,2,3,4-tetrahydroquinolin-6-yl)amino)pyrazolo[1,5-<i>a</i>]pyrimidine-3-carbonitrile</h5><div class="NLM_p last">A solution of hydrogen chloride in ethanol (3.2 M, 1 mL) was added to a stirred solution of <i>tert</i>-butyl (5-((6-((5-chloro-3-cyanopyrazolo[1,5-<i>a</i>]pyrimidin-7-yl)amino)-1-methyl-2-oxo-1,2,3,4-tetrahydroquinolin-8-yl)oxy)hexyl)(methyl)carbamate (185 mg, 0.32 mmol) in EtOH (1 mL) at 22 °C under air. The resulting solution was stirred at 22 °C for 30 min. The reaction mixture was concentrated under vacuum to afford the title compound (145 mg, 95%) as yellow solid.</div></div><div id="sec4_2_2_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Step 3: (<i>R</i>,<i>Z</i>)-4,9,11-Trimethyl-12-oxo-11,12,13,14-tetrahydro-10-oxa-2,4-diaza-1(6,8)-quinolina-3(7,5)-pyrazolo[1,5-<i>a</i>]pyrimidinacyclodecaphane-33-carbonitrile (<b>14</b>)</h5><div class="NLM_p last">DIPEA (0.152 mL, 0.87 mmol) was added to 5-chloro-7-((1-methyl-8-((6-(methylamino)hexan-2-yl)oxy)-2-oxo-1,2,3,4-tetrahydroquinolin-6-yl)amino)pyrazolo[1,5-<i>a</i>]pyrimidine-3-carbonitrile (140 mg, 0.29 mmol) in DMA (5 mL) at 22 °C under air. The resulting mixture was stirred at 125 °C for 2 h. The crude product was purified by preparative HPLC (XBridge Prep C18 OBD column, 5 μm silica, 19 mm diameter, 150 mm length), using decreasingly polar mixtures of water (containing 0.1% NH<sub>3</sub>) and MeCN as eluents to afford the racemic title compound (30 mg, 23%) as a white solid. Both enantiomers were separated using preparative chiral HPLC (conditions: column type, AD/H column; mobile phase, <i>n</i>-heptane/isopropanol (50/50), neutral; peak 1 at 8.9 min; peak 2 at 12.2 min) to afford isomer 1 (13 mg, 43%) as a white solid and isomer 2 (12 mg, 39%) as white solid.</div></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Isomer 1 (<b>13</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.21–1.45 (m, 6H), 1.52–1.61 (m, 1H), 1.70–1.80 (m, 1H), 2.01–2.11 (m, 1H), 2.41–2.45 (m, 2H), 1.70–1.83 (m, 2H), 3.00–3.10 (m, 1H), 3.15 (s, 3H), 3.26 (s, 3H), 3.60–3.70 (m, 1H), 4.55–4.64 (m, 1H), 6.21(s, 1H), 6.80 (s, 1H), 7.28 (s, 1H), 8.37 (s, 1H), 9.80 (s, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 446.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Isomer 2</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.25–1.42 (m, 6H), 1.50–1.60 (m, 1H), 1.70–1.80 (m, 1H), 2.01–2.11 (m, 1H), 2.41–2.45 (m, 2H), 1.68–1.83 (m, 2H), 3.00–3.10 (m, 1H), 3.15 (s, 3H), 3.26 (s, 3H), 3.60–3.70 (m, 1H), 4.55–4.64 (m, 1H), 6.21 (s, 1H), 6.80 (s, 1H), 7.29 (s, 1H), 8.37 (s, 1H), 9.80 (s, 1H); MS (ESI) [M + H]<sup>+</sup><i>m</i>/<i>z</i> 446.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00716" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16610" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16610" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00716" class="ext-link">10.1021/acs.jmedchem.9b00716</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary figures and tables as referred to in the article (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biophysical data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf">jm9b00716_si_001.pdf (1.1 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_002.csv">jm9b00716_si_002.csv (1.68 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00716" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabetta Chiarparin</span> - <span class="hlFld-Affiliation affiliation">Chemistry,
R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2998-1346" title="Orcid link">http://orcid.org/0000-0002-2998-1346</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#db9eb7b2a8bab9beafafbaf598b3b2baa9abbaa9b2b59bbaa8afa9baa1beb5beb8baf5b8b4b6"><span class="__cf_email__" data-cfemail="c481a8adb7a5a6a1b0b0a5ea87acada5b6b4a5b6adaa84a5b7b0b6a5bea1aaa1a7a5eaa7aba9">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amber Y. S. Balazs</span> - <span class="hlFld-Affiliation affiliation">Chemistry, R&D
Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rodrigo J. Carbajo</span> - <span class="hlFld-Affiliation affiliation">Chemistry,
R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nichola L. Davies</span> - <span class="hlFld-Affiliation affiliation">Chemistry,
R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Dong</span> - <span class="hlFld-Affiliation affiliation">Pharmaron Beijing Co., Ltd., Beijing 100176, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander W. Hird</span> - <span class="hlFld-Affiliation affiliation">Chemistry, R&D
Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeffrey W. Johannes</span> - <span class="hlFld-Affiliation affiliation">Chemistry, R&D
Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7242-6682" title="Orcid link">http://orcid.org/0000-0001-7242-6682</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michelle L. Lamb</span> - <span class="hlFld-Affiliation affiliation">Chemistry, R&D
Oncology, AstraZeneca, Waltham, Massachusetts 02451, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3005-8065" title="Orcid link">http://orcid.org/0000-0002-3005-8065</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William McCoull</span> - <span class="hlFld-Affiliation affiliation">Chemistry,
R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2977-1744" title="Orcid link">http://orcid.org/0000-0002-2977-1744</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Piotr Raubo</span> - <span class="hlFld-Affiliation affiliation">Chemistry,
R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Graeme R. Robb</span> - <span class="hlFld-Affiliation affiliation">Chemistry,
R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4531-4375" title="Orcid link">http://orcid.org/0000-0002-4531-4375</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin J. Packer</span> - <span class="hlFld-Affiliation affiliation">Chemistry,
R&D Oncology, AstraZeneca, Cambridge CB4 0QA, United Kingdom</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05142" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05142" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Katherine Gibson, Camil Joubran, and Richard Lewis for NMR discussions and contributions. We thank the many current and former colleagues involved in the design, generation, purification, and management of compounds for the Mcl-1<a onclick="showRef(event, 'ref17 ref22'); return false;" href="javascript:void(0);" class="ref ref17 ref22">(17,22)</a> and BCL6<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> AstraZeneca Oncology projects.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">QM</td><td class="NLM_def"><p class="first last">quantum mechanical</p></td></tr><tr><td class="NLM_term">PCM</td><td class="NLM_def"><p class="first last">polarizable continuum model</p></td></tr><tr><td class="NLM_term">Mcl-1</td><td class="NLM_def"><p class="first last">induced myeloid leukemia cell differentiation protein</p></td></tr><tr><td class="NLM_term">BCL6</td><td class="NLM_def"><p class="first last">B-cell lymphoma 6 protein</p></td></tr><tr><td class="NLM_term">IHB</td><td class="NLM_def"><p class="first last">intramolecular hydrogen bond</p></td></tr><tr><td class="NLM_term">DEL</td><td class="NLM_def"><p class="first last">DNA-encoded library</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64224" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64224" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rask, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Accurate and reliable prediction of the binding affinities of macrocycles to their protein targets</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">6290</span>– <span class="NLM_lpage">6300</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00885</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00885" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=6290-6300&author=H.+S.+Yuauthor=Y.+Dengauthor=Y.+Wuauthor=D.+Sindhikaraauthor=A.+R.+Raskauthor=T.+Kimuraauthor=R.+Abelauthor=L.+Wang&title=Accurate+and+reliable+prediction+of+the+binding+affinities+of+macrocycles+to+their+protein+targets&doi=10.1021%2Facs.jctc.7b00885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of the Binding Affinities of Macrocycles to Their Protein Targets</span></div><div class="casAuthors">Yu, Haoyu S.; Deng, Yuqing; Wu, Yujie; Sindhikara, Dan; Rask, Amy R.; Kimura, Takayuki; Abel, Robert; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6290-6300</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocycles have been emerging as a very important drug class in the past few decades largely due to their expanded chem. diversity benefiting from advances in synthetic methods.  Macrocyclization has been recognized as an effective way to restrict the conformational space of acyclic small mol. inhibitors with the hope of improving potency, selectivity, and metabolic stability.  Because of their relatively larger size as compared to typical small mol. drugs and the complexity of the structures, efficient sampling of the accessible macrocycle conformational space and accurate prediction of their binding affinities to their target protein receptors poses a great challenge of central importance in computational macrocycle drug design.  In this article, the authors present a novel method for relative binding free energy calcns. between macrocycles with different ring sizes and between the macrocycles and their corresponding acyclic counterparts.  The authors have applied the method to seven pharmaceutically interesting data sets taken from recent drug discovery projects including 33 macrocyclic ligands covering a diverse chem. space.  The predicted binding free energies are in good agreement with exptl. data with an overall root-mean-square error (RMSE) of 0.94 kcal/mol.  This is to the authors' knowledge the first time where the free energy of the macrocyclization of linear mols. has been directly calcd. with rigorous physics-based free energy calcn. methods, and the authors anticipate the outstanding accuracy demonstrated here across a broad range of target classes may have significant implications for macrocycle drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibClnk_AFJrVg90H21EOLACvtfcHk0liJTfvHLmr4ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP&md5=62d194fa339aaadd9c46611730dee5ef</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00885%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%2BS.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DRask%26aufirst%3DA.%2BR.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520the%2520binding%2520affinities%2520of%2520macrocycles%2520to%2520their%2520protein%2520targets%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D6290%26epage%3D6300%26doi%3D10.1021%2Facs.jctc.7b00885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Cell permeability beyond the rule of 5</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.addr.2016.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=27067608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslSgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=42-61&author=P.+Matssonauthor=B.+C.+Doakauthor=B.+Overauthor=J.+Kihlberg&title=Cell+permeability+beyond+the+rule+of+5&doi=10.1016%2Fj.addr.2016.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cell permeability beyond the rule of 5</span></div><div class="casAuthors">Matsson, Paer; Doak, Bradley C.; Over, Bjoern; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-61</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug discovery for difficult targets that have large and flat binding sites is often better suited to compds. beyond the "rule of 5" (bRo5).  However, such compds. carry higher pharmacokinetic risks, such as low soly. and permeability, and increased efflux and metab.  Interestingly, recent drug approvals and studies suggest that cell permeable and orally bioavailable drugs can be discovered far into bRo5 space.  Tactics such as redn. or shielding of polarity by N-methylation, bulky side chains and intramol. hydrogen bonds may be used to increase cell permeability in this space, but often results in decreased soly.  Conformationally flexible compds. can, however, combine high permeability and soly., properties that are keys for cell permeability and intestinal absorption.  Recent developments in computational conformational anal. will aid design of such compds. and hence prediction of cell permeability.  Transporter mediated efflux occurs for most investigated drugs in bRo5 space, however it is commonly overcome by high local intestinal concns. on oral administration.  In contrast, there is little data to support significant impact of transporter-mediated intestinal absorption in bRo5 space.  Current knowledge of compd. properties that govern transporter effects of bRo5 drugs is limited and requires further fundamental and comprehensive studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3dS_FpMvvCbVg90H21EOLACvtfcHk0lgfB17_G448AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslSgsb0%253D&md5=1eacb5ab6e50159f3679e3d8a1f719c4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DCell%2520permeability%2520beyond%2520the%2520rule%2520of%25205%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D42%26epage%3D61%26doi%3D10.1016%2Fj.addr.2016.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span>; <span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based macrocycle design in small-molecule drug discovery and simple metrics to identify opportunities for macrocyclization of small-molecule ligands</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> . Published Online: March 22, 2019.</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=M.+D.+Cummings&author=S.+Sekharan&title=Structure-based+macrocycle+design+in+small-molecule+drug+discovery+and+simple+metrics+to+identify+opportunities+for+macrocyclization+of+small-molecule+ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26atitle%3DStructure-based%2520macrocycle%2520design%2520in%2520small-molecule%2520drug%2520discovery%2520and%2520simple%2520metrics%2520to%2520identify%2520opportunities%2520for%2520macrocyclization%2520of%2520small-molecule%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi Sebastiano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backlund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Impact of dynamically exposed polarity on permeability and solubility of chameleonic drugs beyond the rule of 5</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4189</span>– <span class="NLM_lpage">4202</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslGqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4189-4202&author=M.+Rossi+Sebastianoauthor=B.+C.+Doakauthor=M.+Backlundauthor=V.+Poongavanamauthor=B.+Overauthor=G.+Ermondiauthor=G.+Caronauthor=P.+Matssonauthor=J.+Kihlberg&title=Impact+of+dynamically+exposed+polarity+on+permeability+and+solubility+of+chameleonic+drugs+beyond+the+rule+of+5&doi=10.1021%2Facs.jmedchem.8b00347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Dynamically Exposed Polarity on Permeability and Solubility of Chameleonic Drugs Beyond the Rule of 5</span></div><div class="casAuthors">Rossi Sebastiano, Matteo; Doak, Bradley C.; Backlund, Maria; Poongavanam, Vasanthanathan; Over, Bjoern; Ermondi, Giuseppe; Caron, Giulia; Matsson, Paer; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4189-4202</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility has been proposed to significantly affect drug properties outside rule-of-5 (Ro5) chem. space.  Here, we investigated the influence of dynamically exposed polarity on cell permeability and aq. soly. for a structurally diverse set of drugs and clin. candidates far beyond the Ro5, all of which populated multiple distinct conformations as revealed by X-ray crystallog.  Efflux-inhibited (passive) Caco-2 cell permeability correlated strongly with the compds.' min. solvent-accessible 3D polar surface areas (PSA), whereas aq. soly. depended less on the specific 3D conformation.  Inspection of the crystal structures highlighted flexibly linked arom. side chains and dynamically forming intramol. hydrogen bonds as particularly effective in providing "chameleonic" properties that allow compds. to display both high cell permeability and aq. soly.  These structural features, in combination with permeability predictions based on the correlation to solvent-accessible 3D PSA, should inspire drug design in the challenging chem. space far beyond the Ro5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHNeutc0SurrVg90H21EOLACvtfcHk0lgfB17_G448AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslGqtL4%253D&md5=81a3400d434cdc22d1639513e08abeac</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00347%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%2BSebastiano%26aufirst%3DM.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DBacklund%26aufirst%3DM.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DImpact%2520of%2520dynamically%2520exposed%2520polarity%2520on%2520permeability%2520and%2520solubility%2520of%2520chameleonic%2520drugs%2520beyond%2520the%2520rule%2520of%25205%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4189%26epage%3D4202%26doi%3D10.1021%2Facs.jmedchem.8b00347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Macrocycles: Lessons from the distant past, recent developments, and future directions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1039/C4SC03089C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1039%2FC4SC03089C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28553456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=30-49&author=A.+K.+Yudin&title=Macrocycles%3A+Lessons+from+the+distant+past%2C+recent+developments%2C+and+future+directions&doi=10.1039%2FC4SC03089C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles: lessons from the distant past, recent developments, and future directions</span></div><div class="casAuthors">Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-49</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A noticeable increase in mol. complexity of drug targets has created an unmet need in the therapeutic agents that are larger than traditional small mols.  Macrocycles, which are cyclic compds. comprising 12 atoms or more, are now recognized as mols. that "are up to the task" to interrogate extended protein interfaces.  However, because macrocycles (particularly the ones based on peptides) are equipped with large polar surface areas, achieving cellular permeability and bioavailability is anything but straightforward.  While one might consider this to be the Achilles' heel of this class of compds., the synthetic community continues to develop creative approaches toward the synthesis of macrocycles and their site-selective modification.  This perspective provides an overview of both mechanistic and structural issues that bear on macrocycles as a unique class of mols.  The reader is offered a historical foray into some of the classic studies that have resulted in the current renaissance of macrocycles.  In addn., an attempt is made to overview the more recent developments that give hope that macrocycles might indeed turn into a useful therapeutic modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCRWplVAQ417Vg90H21EOLACvtfcHk0lhlY-pJ73ccyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE&md5=48262b4650417f04063c0180b652a886</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2FC4SC03089C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC03089C%26sid%3Dliteratum%253Aachs%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DMacrocycles%253A%2520Lessons%2520from%2520the%2520distant%2520past%252C%2520recent%2520developments%252C%2520and%2520future%2520directions%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D30%26epage%3D49%26doi%3D10.1039%2FC4SC03089C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alihodžić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukvić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenkov, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutinec, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pešić, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašković, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žiher, D.</span></span> <span> </span><span class="NLM_article-title">Current trends in macrocyclic drug discovery and beyond-Ro5</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1016/bs.pmch.2018.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fbs.pmch.2018.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=29680148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=113-233&author=S.+Alihod%C5%BEi%C4%87author=M.+Bukvi%C4%87author=I.+J.+Elenkovauthor=A.+Hutinecauthor=S.+Ko%C5%A1trunauthor=D.+Pe%C5%A1i%C4%87author=G.+Saxtyauthor=L.+Toma%C5%A1kovi%C4%87author=D.+%C5%BDiher&title=Current+trends+in+macrocyclic+drug+discovery+and+beyond-Ro5&doi=10.1016%2Fbs.pmch.2018.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Macrocyclic Drug Discovery and beyond-Ro5</span></div><div class="casAuthors">Alihodzic Sulejman; Bukvic Mirjana; Elenkov Ivaylo J; Hutinec Antun; Kostrun Sanja; Pesic Dijana; Saxty Gordon; Tomaskovic Linda; Ziher Dinko</div><div class="citationInfo"><span class="NLM_cas:title">Progress in medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-233</span>
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    </div><div class="casAbstract">This chapter will discuss the recent literature of macrocycles and drug-like property space moving beyond the rule of five (bRo5).  Trends in chemical classes that fall within this definition are discussed and the impact of the latest technologies in the field assessed.  The physicochemical properties, which have provided both successes and challenges, especially in scale-up, are discussed.  A recent patent literature is reviewed and the chapter concludes with a perspective on the future of macrocyclic drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZ9pfcWWVZ5hWfpMXQLRstfW6udTcc2eYWJjJ72-dk-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D&md5=ac41d1a69cb0056d106d554b0b323e61</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fbs.pmch.2018.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.pmch.2018.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DS.%26aulast%3DBukvi%25C4%2587%26aufirst%3DM.%26aulast%3DElenkov%26aufirst%3DI.%2BJ.%26aulast%3DHutinec%26aufirst%3DA.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DPe%25C5%25A1i%25C4%2587%26aufirst%3DD.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DToma%25C5%25A1kovi%25C4%2587%26aufirst%3DL.%26aulast%3D%25C5%25BDiher%26aufirst%3DD.%26atitle%3DCurrent%2520trends%2520in%2520macrocyclic%2520drug%2520discovery%2520and%2520beyond-Ro5%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D57%26spage%3D113%26epage%3D233%26doi%3D10.1016%2Fbs.pmch.2018.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finch, H.</span></span> <span> </span><span class="NLM_article-title">The conformational musings of a medicinal chemist</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.drudis.2013.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=24184532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVShu73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=320-325&author=H.+Finch&title=The+conformational+musings+of+a+medicinal+chemist&doi=10.1016%2Fj.drudis.2013.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational musings of a medicinal chemist</span></div><div class="casAuthors">Finch, Harry</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">320-325</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design strategies based on X-ray crystallog. data of ligands bound to biol. targets or computationally derived pharmacophore models have been introduced over the past 25 years or so.  These have now matured and are deeply embedded in the drug discovery process in most pharmaceutical and biotechnol. companies where they continue to play a major part in the discovery of new medicines and drug candidates.  Newly developed NMR methods can now provide a full description of the conformations in which ligands exist in free soln., crucially allowing those that are dominant to be identified.  Integrating exptl. detd. conformational information on active and inactive mols. in drug discovery programs, alongside the existing techniques, should have a major impact on the success of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDMOte9AlxMLVg90H21EOLACvtfcHk0liAUU1gPvVT7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVShu73J&md5=808beba5779675737de9ddaff35e8731</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DFinch%26aufirst%3DH.%26atitle%3DThe%2520conformational%2520musings%2520of%2520a%2520medicinal%2520chemist%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D320%26epage%3D325%26doi%3D10.1016%2Fj.drudis.2013.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of drug-like molecules bound to proteins: An extensive study of ligand reorganization upon binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2510</span>, <span class="refDoi"> DOI: 10.1021/jm030563w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030563w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2499-2510&author=E.+Perolaauthor=P.+S.+Charifson&title=Conformational+analysis+of+drug-like+molecules+bound+to+proteins%3A+An+extensive+study+of+ligand+reorganization+upon+binding&doi=10.1021%2Fjm030563w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Drug-Like Molecules Bound to Proteins: An Extensive Study of Ligand Reorganization upon Binding</span></div><div class="casAuthors">Perola, Emanuele; Charifson, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2499-2510</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes a large-scale study on the nature and the energetics of the conformational changes drug-like mols. experience upon binding.  Ligand strain energies and conformational reorganization were analyzed with different computational methods on 150 crystal structures of pharmaceutically relevant protein-ligand complexes.  The common knowledge that ligands rarely bind in their lowest calcd. energy conformation was confirmed.  Addnl., the authors found that over 60% of the ligands do not bind in a local min. conformation.  While approx. 60% of the ligands were calcd. to bind with strain energies lower than 5 kcal/mol, strain energies over 9 kcal/mol were calcd. in at least 10% of the cases regardless of the method used.  A clear correlation was found between acceptable strain energy and ligand flexibility, while there was no correlation between strain energy and binding affinity, thus indicating that expensive conformational rearrangements can be tolerated in some cases without overly penalizing the tightness of binding.  On the basis of the trends obsd., thresholds for the acceptable strain energies of bioactive conformations were defined with consideration of the impact of ligand flexibility.  An anal. of the degree of folding of the bound ligands confirmed the general tendency of small mols. to bind in an extended conformation.  The results suggest that the unfolding of hydrophobic ligands during binding, which exposes hydrophobic surfaces to contact with protein residues, could be one of the factors accounting for high reorganization energies.  Finally, different methods for conformational anal. were evaluated, and guidelines were defined to maximize the prevalence of bioactive conformations in computationally generated ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMrN0dFKglcbVg90H21EOLACvtfcHk0liAUU1gPvVT7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D&md5=8f54a8285ef1ffc5e1a19f748f4674bf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm030563w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030563w%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3DConformational%2520analysis%2520of%2520drug-like%2520molecules%2520bound%2520to%2520proteins%253A%2520An%2520extensive%2520study%2520of%2520ligand%2520reorganization%2520upon%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2499%26epage%3D2510%26doi%3D10.1021%2Fjm030563w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M. J.</span></span> <span> </span><span class="NLM_article-title">Measurement, interpretation and use of free ligand solution conformations in drug discovery</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/bs.pmch.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fbs.pmch.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=26852934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC28nos1KmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=45-147&author=C.+D.+Blundellauthor=T.+Nowakauthor=M.+J.+Watson&title=Measurement%2C+interpretation+and+use+of+free+ligand+solution+conformations+in+drug+discovery&doi=10.1016%2Fbs.pmch.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement, Interpretation and Use of Free Ligand Solution Conformations in Drug Discovery</span></div><div class="casAuthors">Blundell Charles D; Nowak Thorsten; Watson Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Progress in medicinal chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-147</span>
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOLsGcWRMxs0BuFdjgVNKSfW6udTcc2ebr8f925i3NgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nos1KmsA%253D%253D&md5=35cea492313ecb36fbc0c64ad0245b65</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fbs.pmch.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.pmch.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DC.%2BD.%26aulast%3DNowak%26aufirst%3DT.%26aulast%3DWatson%26aufirst%3DM.%2BJ.%26atitle%3DMeasurement%252C%2520interpretation%2520and%2520use%2520of%2520free%2520ligand%2520solution%2520conformations%2520in%2520drug%2520discovery%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D55%26spage%3D45%26epage%3D147%26doi%3D10.1016%2Fbs.pmch.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Experimental free ligand conformations: A missing link in structure-based drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.4155%2Ffmc-2018-0339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=30648432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=79-82&author=E.+Chiarparinauthor=M.+J.+Packerauthor=D.+M.+Wilson&title=Experimental+free+ligand+conformations%3A+A+missing+link+in+structure-based+drug+discovery&doi=10.4155%2Ffmc-2018-0339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental free ligand conformations: a missing link in structure-based drug discovery</span></div><div class="casAuthors">Chiarparin, Elisabetta; Packer, Martin J.; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">79-82</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2BcuFdu3d5LVg90H21EOLACvtfcHk0lgFQGaAt-mDKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFejurw%253D&md5=c6dab61511fc3a5e9b55e47a8aead791</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0339%26sid%3Dliteratum%253Aachs%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DExperimental%2520free%2520ligand%2520conformations%253A%2520A%2520missing%2520link%2520in%2520structure-based%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D79%26epage%3D82%26doi%3D10.4155%2Ffmc-2018-0339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span> <span> </span><span class="NLM_article-title">Towards understanding the unbound state of drug compounds: implications for the intramolecular reorganization energy upon binding</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2159</span>– <span class="NLM_lpage">2189</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmc.2016.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=27061672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsF2lsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=2159-2189&author=N.+Foloppeauthor=I.+J.+Chen&title=Towards+understanding+the+unbound+state+of+drug+compounds%3A+implications+for+the+intramolecular+reorganization+energy+upon+binding&doi=10.1016%2Fj.bmc.2016.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Towards understanding the unbound state of drug compounds: Implications for the intramolecular reorganization energy upon binding</span></div><div class="casAuthors">Foloppe, Nicolas; Chen, I.-Jen</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2159-2189</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">There has been an explosion of structural information for pharmaceutical compds. bound to biol. targets, but the conformations and dynamics of compds. free in soln. are poorly characterized, if at all.  Yet, knowledge of the unbound state is essential to understand the fundamentals of mol. recognition, including the much debated conformational intramol. reorganization energy of a compd. upon binding (ΔEReorg).  Also, dependable observation of the unbound compds. is important for ligand-based drug discovery, e.g. with pharmacophore modeling.  Here, these questions are addressed with long (≥0.5 μs) state-of-the-art mol. dynamics (MD) simulations of 26 compds. (including 7 approved drugs) unbound in explicit solvent.  These compds. were selected to be chem. diverse, with a range of flexibility, and good quality bioactive x-ray structures.  The MD-simulated free compds. are compared to their bioactive structure and conformers generated with ad hoc sampling in vacuo or with implicit generalized Born (GB) aq. solvation models.  The GB conformational models clearly depart from those obtained in explicit solvent, and suffer from conformational collapse almost as severe as in vacuo.  Thus, the global energy min. in vacuo or with GB are not suitable representations of the unbound state, which can instead be extensively sampled by MD simulations.  Many, but not all, MD-simulated compds. displayed some structural similarity to their bioactive structure, supporting the notion of conformational preorganization for binding.  The ligand-protein complexes were also simulated in explicit solvent, to est. ΔEReorg as an enthalpic difference ΔHReorg between the intramol. energies in the bound and unbound states.  This fresh approach yielded ΔHReorg values ≤ 6 kcal/mol for 18 out of 26 compds.  For three particularly polar compds. 15 ≤ ΔHReorg ≤ 20 kcal/mol, supporting the notion that ΔHReorg can be substantial.  Those large ΔHReorg values correspond to a redistribution of electrostatic interactions upon binding.  Overall, the study illustrates how MD simulations offer a promising avenue to characterize the unbound state of medicinal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlcBqm8bRf_bVg90H21EOLACvtfcHk0lgFQGaAt-mDKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsF2lsLc%253D&md5=a4083169d7c616b125ec04aa50da58c1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DI.%2BJ.%26atitle%3DTowards%2520understanding%2520the%2520unbound%2520state%2520of%2520drug%2520compounds%253A%2520implications%2520for%2520the%2520intramolecular%2520reorganization%2520energy%2520upon%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D2159%26epage%3D2189%26doi%3D10.1016%2Fj.bmc.2016.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdelyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Conformational sampling of macrocyclic drugs in different environments: can we find the relevant conformations?</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11742</span>– <span class="NLM_lpage">11757</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01379</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=11742-11757&author=V.+Poongavanamauthor=E.+Daneliusauthor=S.+Peintnerauthor=L.+Alcarazauthor=G.+Caronauthor=M.+D.+Cummingsauthor=S.+Wlodekauthor=M.+Erdelyiauthor=P.+C.+D.+Hawkinsauthor=G.+Ermondiauthor=J.+Kihlberg&title=Conformational+sampling+of+macrocyclic+drugs+in+different+environments%3A+can+we+find+the+relevant+conformations%3F&doi=10.1021%2Facsomega.8b01379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Danelius, Emma; Peintner, Stefan; Alcaraz, Lilian; Caron, Giulia; Cummings, Maxwell D.; Wlodek, Stanislaw; Erdelyi, Mate; Hawkins, Paul C. D.; Ermondi, Giuseppe; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11742-11757</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility is a major determinant of the properties of macrocycles and other drugs in beyond rule of 5 (bRo5) space.  Prediction of conformations is essential for design of drugs in this space and the authors have evaluated three tools for conformational sampling of a set of ten bRo5 drugs and clin. candidates in polar and apolar environments.  The distance-geometry based OMEGA was found to yield ensembles spanning larger structure and property spaces than the two MD based methods, MOE-LowMode MD (MOE) and MacroModel-Macrocycle Conformational Sampling (MC).  Both MC and OMEGA, but not MOE, generated different ensembles for polar and apolar environments.  All three conformational search methods generated conformers similar to the crystal structure conformers for nine of the ten compds., with OMEGA performing somewhat better than MOE and MC.  MOE and OMEGA found all six conformers of roxithromycin that were identified by NMR in aq. solns., while only OMEGA sampled the three conformers obsd. in chloroform.  The authors suggest that characterization of conformers using mol. descriptors, e.g. the radius of gyration (Rgyr) and polar surface area (PSA), is preferred to energy- or RMSD-based methods for selection of biol. relevant conformers in drug discovery in bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoetOG-yDeciLVg90H21EOLACvtfcHk0lg8uV_lfnrbeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI&md5=a8722551b161e907daf55146f22b6d57</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01379%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DErdelyi%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DConformational%2520sampling%2520of%2520macrocyclic%2520drugs%2520in%2520different%2520environments%253A%2520can%2520we%2520find%2520the%2520relevant%2520conformations%253F%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D11742%26epage%3D11757%26doi%3D10.1021%2Facsomega.8b01379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N.</span></span> <span> </span><span class="NLM_article-title">Overcoming the unfavourable entropic contribution of ligand binding with a macrocyclic inhibitor bound to penicillopepsin</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1007/978-1-4615-5373-1_50</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1007%2F978-1-4615-5373-1_50" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=9561242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADyaK1cXlsFKnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=1998&pages=355-359&author=M.+E.+Fraserauthor=J.+H.+Meyerauthor=P.+A.+Bartlettauthor=M.+N.+James&title=Overcoming+the+unfavourable+entropic+contribution+of+ligand+binding+with+a+macrocyclic+inhibitor+bound+to+penicillopepsin&doi=10.1007%2F978-1-4615-5373-1_50"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming the unfavorable entropic contribution of ligand binding with a macrocyclic inhibitor bound to penicillopepsin</span></div><div class="casAuthors">Fraser, Marie F.; Meyer, J. Hoyt; Bartlett, Paul A.; James, Michael N. G.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">Aspartic Proteinases</span>),
    <span class="NLM_cas:pages">355-359</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Plenum Publishing Corp.</span>)
        </div><div class="casAbstract">One way to improve the binding of a ligand to its protein receptor is by forcing the unbound ligand to adopt the same conformation as it has when bound to the receptor.  The conformational constraint then reduces the entropic disadvantage of ordered binding to the protein.  The constraint should change the structure of the ligand as little as possible, so as to retain the favorable enthalpic interactions of the ligand with the receptor.  Including a ring structure in the design of a ligand can reduce its conformational flexibility and it may be possible to introduce the ring without changing how the ligand binds to the receptor.  A macrocyclic inhibitor of penicillopepsin, as well as two acyclic analogs, were synthesized.  Availability of the macrocyclic inhibitor and the acyclic analogs make it possible to evaluate the entropic effect of the cyclization, provided that they bind to penicillopepsin in the same manner.  Enzyme inhibition by the various inhibitors was evaluated, and the crystal structure of the macrocyclic inhibitor co-crystd. with penicillopepsin was detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqywUc6QyslELVg90H21EOLACvtfcHk0lg8uV_lfnrbeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlsFKnsrY%253D&md5=7d64b9744103afdb02bb131109050bfe</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2F978-1-4615-5373-1_50&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4615-5373-1_50%26sid%3Dliteratum%253Aachs%26aulast%3DFraser%26aufirst%3DM.%2BE.%26aulast%3DMeyer%26aufirst%3DJ.%2BH.%26aulast%3DBartlett%26aufirst%3DP.%2BA.%26aulast%3DJames%26aufirst%3DM.%2BN.%26atitle%3DOvercoming%2520the%2520unfavourable%2520entropic%2520contribution%2520of%2520ligand%2520binding%2520with%2520a%2520macrocyclic%2520inhibitor%2520bound%2520to%2520penicillopepsin%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1998%26volume%3D436%26spage%3D355%26epage%3D359%26doi%3D10.1007%2F978-1-4615-5373-1_50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, R. J.</span>; <span class="NLM_string-name">Mobli, M.</span></span> <i>Modelling 1H NMR Spectra of Organic Compounds: Theory, Applications and NMR Prediction Software</i>; <span class="NLM_publisher-name">John Wiley & Sons Ltd.</span> (<span class="NLM_publisher-loc">The Atrium, Southern Gate, Chichester, West Sussex, PO19 8SQ, U.K.</span>), <span class="NLM_year">2008</span>; <a href="https://onlinelibrary.wiley.com/doi/book/10.1002/9780470721803" class="extLink">https://onlinelibrary.wiley.com/doi/book/10.1002/9780470721803</a>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1002%2F9780470721803" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=R.+J.+Abraham&author=M.+Mobli&title=Modelling+1H+NMR+Spectra+of+Organic+Compounds%3A+Theory%2C+Applications+and+NMR+Prediction+Software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2F9780470721803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470721803%26sid%3Dliteratum%253Aachs%26aulast%3DAbraham%26aufirst%3DR.%2BJ.%26btitle%3DModelling%25201H%2520NMR%2520Spectra%2520of%2520Organic%2520Compounds%253A%2520Theory%252C%2520Applications%2520and%2520NMR%2520Prediction%2520Software%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%2520Ltd%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweckstetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M.</span></span> <span> </span><span class="NLM_article-title">Exploring free-energy landscapes of intrinsically disordered proteins at atomic resolution using NMR spectroscopy</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">6632</span>– <span class="NLM_lpage">6660</span>, <span class="refDoi"> DOI: 10.1021/cr400688u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400688u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVegtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=6632-6660&author=M.+R.+Jensenauthor=M.+Zweckstetterauthor=J.+R.+Huangauthor=M.+Blackledge&title=Exploring+free-energy+landscapes+of+intrinsically+disordered+proteins+at+atomic+resolution+using+NMR+spectroscopy&doi=10.1021%2Fcr400688u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Free-Energy Landscapes of Intrinsically Disordered Proteins at Atomic Resolution Using NMR Spectroscopy</span></div><div class="casAuthors">Jensen, Malene Ringkjobing; Zweckstetter, Markus; Huang, Jie-rong; Blackledge, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6632-6660</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the study of free-energy landscapes of intrinsically disordered proteins at at. resoln. using NMR spectroscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1wesUYoP29LVg90H21EOLACvtfcHk0lihjHSz4S7skA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVegtLo%253D&md5=7d4fabed6618ede86586aeb6214eb33a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcr400688u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400688u%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DZweckstetter%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DJ.%2BR.%26aulast%3DBlackledge%26aufirst%3DM.%26atitle%3DExploring%2520free-energy%2520landscapes%2520of%2520intrinsically%2520disordered%2520proteins%2520at%2520atomic%2520resolution%2520using%2520NMR%2520spectroscopy%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D6632%26epage%3D6660%26doi%3D10.1021%2Fcr400688u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cernak, T.</span></span> <span> </span><span class="NLM_article-title">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">12256</span>– <span class="NLM_lpage">12267</span>, <span class="refDoi"> DOI: 10.1002/anie.201303207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1002%2Fanie.201303207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=12256-12267&author=H.+Sch%C3%B6nherrauthor=T.+Cernak&title=Profound+methyl+effects+in+drug+discovery+and+a+call+for+new+C-H+methylation+reactions&doi=10.1002%2Fanie.201303207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span></div><div class="casAuthors">Schonherr Heike; Cernak Tim</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12256-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The methyl group is one of the most commonly occurring carbon fragments in small-molecule drugs.  This simplest alkyl fragment appears in more than 67 % of the top-selling drugs of 2011 and can modulate both the biological and physical properties of a molecule.  This Review focuses on so-called magic methyl effects on binding potency, where the seemingly mundane change of C H to C Me improves the IC50  value of a drug candidate more than 100-fold.  This discussion is followed by a survey of recent advances in synthetic chemistry that allow the direct methylation of C(sp(2) ) H and C(sp(3) ) H bonds.  It is our hope that the relevance of the meager methyl group to drug discovery as presented herein will inspire reports on new C H methylation reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyoLZDDgvsMh0LsK7uFuY7fW6udTcc2eZhhj4EQDEJ1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D&md5=ef08f80e55b2c530b49e4306da024902</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303207%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DH.%26aulast%3DCernak%26aufirst%3DT.%26atitle%3DProfound%2520methyl%2520effects%2520in%2520drug%2520discovery%2520and%2520a%2520call%2520for%2520new%2520C-H%2520methylation%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D12256%26epage%3D12267%26doi%3D10.1002%2Fanie.201303207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tron, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belmonte, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aquila, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boise, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Embrey, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmirski, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nooka, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuller, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Secrist, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Mcl-1-specific inhibitor AZD5991 and preclinical activity in multiple myeloma and acute myeloid leukemia</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5341</span>– <span class="NLM_lpage">5355</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07551-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fs41467-018-07551-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=30559424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFersLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=5341-5355&author=A.+E.+Tronauthor=M.+A.+Belmonteauthor=A.+Adamauthor=B.+M.+Aquilaauthor=L.+H.+Boiseauthor=E.+Chiarparinauthor=J.+Cidadoauthor=K.+J.+Embreyauthor=E.+Ganglauthor=F.+D.+Gibbonsauthor=G.+P.+Gregoryauthor=D.+Hargreavesauthor=J.+A.+Hendricksauthor=J.+W.+Johannesauthor=R.+W.+Johnstoneauthor=S.+L.+Kazmirskiauthor=J.+G.+Kettleauthor=M.+L.+Lambauthor=S.+M.+Matulisauthor=A.+K.+Nookaauthor=M.+J.+Packerauthor=B.+Pengauthor=P.+B.+Rawlinsauthor=D.+W.+Robbinsauthor=A.+G.+Schullerauthor=N.+Suauthor=W.+Yangauthor=Q.+Yeauthor=X.+Zhengauthor=J.+P.+Secristauthor=E.+A.+Clarkauthor=D.+M.+Wilsonauthor=S.+E.+Fawellauthor=A.+W.+Hird&title=Discovery+of+Mcl-1-specific+inhibitor+AZD5991+and+preclinical+activity+in+multiple+myeloma+and+acute+myeloid+leukemia&doi=10.1038%2Fs41467-018-07551-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Mcl-1-specific inhibitor AZD5991 and preclinical activity in multiple myeloma and acute myeloid leukemia</span></div><div class="casAuthors">Tron, Adriana E.; Belmonte, Matthew A.; Adam, Ammar; Aquila, Brian M.; Boise, Lawrence H.; Chiarparin, Elisabetta; Cidado, Justin; Embrey, Kevin J.; Gangl, Eric; Gibbons, Francis D.; Gregory, Gareth P.; Hargreaves, David; Hendricks, J. Adam; Johannes, Jeffrey W.; Johnstone, Ricky W.; Kazmirski, Steven L.; Kettle, Jason G.; Lamb, Michelle L.; Matulis, Shannon M.; Nooka, Ajay K.; Packer, Martin J.; Peng, Bo; Rawlins, Philip B.; Robbins, Daniel W.; Schuller, Alwin G.; Su, Nancy; Yang, Wenzhan; Ye, Qing; Zheng, Xiaolan; Secrist, J. Paul; Clark, Edwin A.; Wilson, David M.; Fawell, Stephen E.; Hird, Alexander W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5341</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mcl-1 is a member of the Bcl-2 family of proteins that promotes cell survival by preventing induction of apoptosis in many cancers.  High expression of Mcl-1 causes tumorigenesis and resistance to anticancer therapies highlighting the potential of Mcl-1 inhibitors as anticancer drugs.  Here, we describe AZD5991, a rationally designed macrocyclic mol. with high selectivity and affinity for Mcl-1 currently in clin. development.  Our studies demonstrate that AZD5991 binds directly to Mcl-1 and induces rapid apoptosis in cancer cells, most notably myeloma and acute myeloid leukemia, by activating the Bak-dependent mitochondrial apoptotic pathway.  AZD5991 shows potent antitumor activity in vivo with complete tumor regression in several models of multiple myeloma and acute myeloid leukemia after a single tolerated dose as monotherapy or in combination with bortezomib or venetoclax.  Based on these promising data, a Phase I clin. trial has been launched for evaluation of AZD5991 in patients with hematol. malignancies (NCT03218683).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLv1vdwF2TY7Vg90H21EOLACvtfcHk0lihjHSz4S7skA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFersLjK&md5=10ccf1fda947aca212bede3927eaabfa</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07551-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07551-w%26sid%3Dliteratum%253Aachs%26aulast%3DTron%26aufirst%3DA.%2BE.%26aulast%3DBelmonte%26aufirst%3DM.%2BA.%26aulast%3DAdam%26aufirst%3DA.%26aulast%3DAquila%26aufirst%3DB.%2BM.%26aulast%3DBoise%26aufirst%3DL.%2BH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DEmbrey%26aufirst%3DK.%2BJ.%26aulast%3DGangl%26aufirst%3DE.%26aulast%3DGibbons%26aufirst%3DF.%2BD.%26aulast%3DGregory%26aufirst%3DG.%2BP.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHendricks%26aufirst%3DJ.%2BA.%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DKazmirski%26aufirst%3DS.%2BL.%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DMatulis%26aufirst%3DS.%2BM.%26aulast%3DNooka%26aufirst%3DA.%2BK.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobbins%26aufirst%3DD.%2BW.%26aulast%3DSchuller%26aufirst%3DA.%2BG.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DSecrist%26aufirst%3DJ.%2BP.%26aulast%3DClark%26aufirst%3DE.%2BA.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DHird%26aufirst%3DA.%2BW.%26atitle%3DDiscovery%2520of%2520Mcl-1-specific%2520inhibitor%2520AZD5991%2520and%2520preclinical%2520activity%2520in%2520multiple%2520myeloma%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D5341%26epage%3D5355%26doi%3D10.1038%2Fs41467-018-07551-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willoughby, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansma, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoye, T. R.</span></span> <span> </span><span class="NLM_article-title">A guide to small-molecule structure assignment through computation of (<sup>1</sup>H and <sup>13</sup>C) NMR chemical shifts</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">643</span>– <span class="NLM_lpage">660</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fnprot.2014.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=24556787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVehu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=643-660&author=P.+H.+Willoughbyauthor=M.+J.+Jansmaauthor=T.+R.+Hoye&title=A+guide+to+small-molecule+structure+assignment+through+computation+of+%281H+and+13C%29+NMR+chemical+shifts&doi=10.1038%2Fnprot.2014.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to small-molecule structure assignment through computation of (1H and 13C) NMR chemical shifts</span></div><div class="casAuthors">Willoughby, Patrick H.; Jansma, Matthew J.; Hoye, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">643-660</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This protocol is intended to provide chemists who discover or make new org. compds. with a valuable tool for validating the structural assignments of those new chem. entities.  Exptl. 1H and/or 13C NMR spectral data and its proper interpretation for the compd. of interest is required as a starting point.  The approach involves the following steps: (i) using mol. mechanics calcns. (with, e.g., MacroModel) to generate a library of conformers; (ii) using d. functional theory (DFT) calcns. (with, e.g., Gaussian 09) to det. optimal geometry, free energies and chem. shifts for each conformer; (iii) detg. Boltzmann-weighted proton and carbon chem. shifts; and (iv) comparing the computed chem. shifts for two or more candidate structures with exptl. data to det. the best fit.  For a typical structure assignment of a small org. mol. (e.g., fewer than ∼10 non-H atoms or up to ∼180 a.m.u. and ∼20 conformers), this protocol can be completed in ∼2 h of active effort over a 2-d period; for more complex mols. (e.g., fewer than ∼30 non-H atoms or up to ∼500 a.m.u. and ∼50 conformers), the protocol requires ∼36 h of active effort over a 2-wk period.  To demonstrate the method, we have chosen the anal. of the cis- vs. the trans-diastereoisomers of 3-methylcyclohexanol (1-cis vs. 1-trans).  The protocol is written in a manner that makes the computation of chem. shifts tractable for chemists who may otherwise have only rudimentary computational experience.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnhwPmqKqlJ7Vg90H21EOLACvtfcHk0lhiZ5SHpEmkIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVehu7w%253D&md5=0aa9872e60076e3e88c50f14981733b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.042%26sid%3Dliteratum%253Aachs%26aulast%3DWilloughby%26aufirst%3DP.%2BH.%26aulast%3DJansma%26aufirst%3DM.%2BJ.%26aulast%3DHoye%26aufirst%3DT.%2BR.%26atitle%3DA%2520guide%2520to%2520small-molecule%2520structure%2520assignment%2520through%2520computation%2520of%2520%25281H%2520and%252013C%2529%2520NMR%2520chemical%2520shifts%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D643%26epage%3D660%26doi%3D10.1038%2Fnprot.2014.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimblat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarotti, A. M.</span></span> <span> </span><span class="NLM_article-title">Computational chemistry to the rescue: Modern toolboxes for the assignment of complex molecules by GIAO NMR calculations</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">12246</span>– <span class="NLM_lpage">12261</span>, <span class="refDoi"> DOI: 10.1002/chem.201601150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1002%2Fchem.201601150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=27405775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFGjurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=12246-12261&author=N.+Grimblatauthor=A.+M.+Sarotti&title=Computational+chemistry+to+the+rescue%3A+Modern+toolboxes+for+the+assignment+of+complex+molecules+by+GIAO+NMR+calculations&doi=10.1002%2Fchem.201601150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Chemistry to the Rescue: Modern Toolboxes for the Assignment of Complex Molecules by GIAO NMR Calculations</span></div><div class="casAuthors">Grimblat, Nicolas; Sarotti, Ariel M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12246-12261</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The calcns. of NMR properties of mols. using quantum chem. methods have deeply impacted several branches of org. chem.  They are particularly important in structural or stereochem. assignments of org. compds., with implications in total synthesis, stereoselective reactions, and natural products chem.  In studying the evolution of the strategies developed to support (or reject) a structural proposal, it becomes clear that the most effective and accurate ones involve sophisticated procedures to correlate exptl. and computational data.  Owing to their relatively high math. complexity, such calcns. (CP3, DP4, ANN-PRA) are often carried out using addnl. computational resources provided by the authors (such as applets or Excel files).  This Minireview will cover the state-of-the-art of these toolboxes in the assignment of org. mols., including math. definitions, updates, and discussion of relevant examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1esvt38PZ5LVg90H21EOLACvtfcHk0lhiZ5SHpEmkIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFGjurnK&md5=1436b69b48a6a9344f981202cdb8954f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fchem.201601150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201601150%26sid%3Dliteratum%253Aachs%26aulast%3DGrimblat%26aufirst%3DN.%26aulast%3DSarotti%26aufirst%3DA.%2BM.%26atitle%3DComputational%2520chemistry%2520to%2520the%2520rescue%253A%2520Modern%2520toolboxes%2520for%2520the%2520assignment%2520of%2520complex%2520molecules%2520by%2520GIAO%2520NMR%2520calculations%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D12246%26epage%3D12261%26doi%3D10.1002%2Fchem.201601150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butts, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, V. K.</span></span> <span> </span><span class="NLM_article-title">Synergy of synthesis, computation and NMR reveals correct baulamycin structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>547</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nature23265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fnature23265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28748934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1ent7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=547&publication_year=2017&pages=436-440&author=J.+Wuauthor=P.+Lorenzoauthor=S.+Zhongauthor=M.+Aliauthor=C.+P.+Buttsauthor=E.+L.+Myersauthor=V.+K.+Aggarwal&title=Synergy+of+synthesis%2C+computation+and+NMR+reveals+correct+baulamycin+structures&doi=10.1038%2Fnature23265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy of synthesis, computation and NMR reveals correct baulamycin structures</span></div><div class="casAuthors">Wu, Jingjing; Lorenzo, Paula; Zhong, Siying; Ali, Muhammad; Butts, Craig P.; Myers, Eddie L.; Aggarwal, Varinder K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">547</span>
        (<span class="NLM_cas:issue">7664</span>),
    <span class="NLM_cas:pages">436-440</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol., biol. active natural products continue to be our most rewarding source of, and inspiration for, new medicines.  Sometimes we happen upon such mols. in minute quantities in unique, difficult-to-reach, and often fleeting environments, perhaps never to be discovered again.  In these cases, detg. the structure of a mol.-including assigning its relative and abs. configurations-is paramount, enabling one to understand its biol. activity.  Mols. that comprise stereochem. complex acyclic and conformationally flexible carbon chains make such a task extremely challenging.  The baulamycins (A and B) serve as a contemporary example.  Isolated in small quantities and shown to have promising antimicrobial activity, the structure of the conformationally flexible mols. was detd. largely through J-based configurational anal., but has been found to be incorrect.  Our subsequent campaign to identify the true structures of the baulamycins has revealed a powerful method for the rapid structural elucidation of such mols.  Specifically, the prediction of NMR (NMR) parameters through d. functional theory-combined with an efficient sequence of boron-based synthetic transformations, which allowed an encoded (labeled) mixt. of natural-product diastereomers to be prepd.-enabled us rapidly to pinpoint and synthesize the correct structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNtULzfiHMbVg90H21EOLACvtfcHk0lhiZ5SHpEmkIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1ent7fJ&md5=c48c205289a43e43d4b2ab670619a64a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature23265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23265%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLorenzo%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DAli%26aufirst%3DM.%26aulast%3DButts%26aufirst%3DC.%2BP.%26aulast%3DMyers%26aufirst%3DE.%2BL.%26aulast%3DAggarwal%26aufirst%3DV.%2BK.%26atitle%3DSynergy%2520of%2520synthesis%252C%2520computation%2520and%2520NMR%2520reveals%2520correct%2520baulamycin%2520structures%26jtitle%3DNature%26date%3D2017%26volume%3D547%26spage%3D436%26epage%3D440%26doi%3D10.1038%2Fnature23265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almond, A.</span></span> <span> </span><span class="NLM_article-title">Quantification of free ligand conformational preferences by NMR and their relationship to the bioactive conformation</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4976</span>– <span class="NLM_lpage">4987</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.06.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmc.2013.06.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=23886813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCqt77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4976-4987&author=C.+D.+Blundellauthor=M.+J.+Packerauthor=A.+Almond&title=Quantification+of+free+ligand+conformational+preferences+by+NMR+and+their+relationship+to+the+bioactive+conformation&doi=10.1016%2Fj.bmc.2013.06.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of free ligand conformational preferences by NMR and their relationship to the bioactive conformation</span></div><div class="casAuthors">Blundell, Charles D.; Packer, Martin J.; Almond, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4976-4987</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Accurate unbound soln. 3D-structures of ligands provide unique opportunities for medicinal chem. and, in particular, a context to understand binding thermodn. and kinetics.  Previous methods of deriving these 3D-structures have had neither the accuracy nor resoln. needed for drug design and have not yet realized their potential.  Here, we describe and apply a NMR methodol. to the aminoglycoside streptomycin that can accurately quantify accessible 3D-space and rank the occupancy of obsd. conformers to a resoln. that enables medicinal chem. understanding and design.  Importantly, it is based upon conventional small mol. NMR techniques and can be performed in physiol.-relevant solvents.  The methodol. uses multiple datasets, an order of magnitude more exptl. data than previous NMR approaches and a dynamic model during refinement, is independent of computational chem. and avoids the problem of virtual conformations.  The refined set of soln. 3D-shapes for streptomycin can be grouped into two major families, of which the most populated is almost identical to the 30S ribosomal subunit bioactive shape.  We therefore propose that accurate unbound ligand soln. conformations may, in some cases, provide a subsidiary route to bioactive shape without crystallog.  This exptl. technique opens up new opportunities for drug design and more so when complemented with protein co-crystal structures, SAR data and pharmacophore modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUZZsawI_fmLVg90H21EOLACvtfcHk0lhiZ5SHpEmkIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCqt77K&md5=3ac32a01828e59160794384644aacbf4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.06.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.06.056%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DC.%2BD.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DAlmond%26aufirst%3DA.%26atitle%3DQuantification%2520of%2520free%2520ligand%2520conformational%2520preferences%2520by%2520NMR%2520and%2520their%2520relationship%2520to%2520the%2520bioactive%2520conformation%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4976%26epage%3D4987%26doi%3D10.1016%2Fj.bmc.2013.06.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span> <i>ACD/NMR Workbook</i>, version <span class="NLM_edition">2018.2</span>; <span class="NLM_publisher-name">Advanced Chemistry Development, Inc.</span>: <span class="NLM_publisher-loc">Toronto, ON, Canada</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ACD%2FNMR+Workbook%2C+version+2018.2%3B+Advanced+Chemistry+Development%2C+Inc.%3A+Toronto%2C+ON%2C+Canada%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DACD%252FNMR%2520Workbook%26pub%3DAdvanced%2520Chemistry%2520Development%252C%2520Inc%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span> <i>MestReNova, MSpin</i>, version <span class="NLM_edition">14.0.0-23239</span>; <span class="NLM_publisher-name">Mestrelab
Research S.L.</span>: <span class="NLM_publisher-loc">Santiago de Compostela,
Spain</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+MestReNova%2C+MSpin%2C+version+14.0.0-23239%3B+Mestrelab%0AResearch+S.L.%3A+Santiago+de+Compostela%2C%0ASpain%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMestReNova%252C%2520MSpin%26pub%3DMestrelab%250AResearch%2520S.L%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span> <i>Molecular Operating Environment (MOE)</i>, version <span class="NLM_edition">2018.01</span>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span> (<span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite No. 910, Montreal,
QC H3A 2R7, Canada</span>), <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+version+2018.01%3B+Chemical+Computing+Group+ULC+%281010+Sherbooke+St.+West%2C+Suite+No.+910%2C+Montreal%2C%0AQC+H3A+2R7%2C+Canada%29%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Marenich, A. V.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Janesko, B. G.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Williams-Young, D.</span>; <span class="NLM_string-name">Ding, F.</span>; <span class="NLM_string-name">Lipparini, F.</span>; <span class="NLM_string-name">Egidi, F.</span>; <span class="NLM_string-name">Goings, J.</span>; <span class="NLM_string-name">Peng, B.</span>; <span class="NLM_string-name">Petrone, A.</span>; <span class="NLM_string-name">Henderson, T.</span>; <span class="NLM_string-name">Ranasinghe, D.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Gao, J.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Liang, W.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Throssell, K.</span>; <span class="NLM_string-name">Montgomery, J. A.,  Jr.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M. J.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E. N.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Keith, T. A.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A. P.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Farkas, O.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Fox, D. J.</span></span>,  <i>Gaussian 16</i>, revision <span class="NLM_edition">B.01</span>; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=G.+A.+Petersson&author=H.+Nakatsuji&author=X.+Li&author=M.+Caricato&author=A.+V.+Marenich&author=J.+Bloino&author=B.+G.+Janesko&author=R.+Gomperts&author=B.+Mennucci&author=H.+P.+Hratchian&author=J.+V.+Ortiz&author=A.+F.+Izmaylov&author=J.+L.+Sonnenberg&author=D.+Williams-Young&author=F.+Ding&author=F.+Lipparini&author=F.+Egidi&author=J.+Goings&author=B.+Peng&author=A.+Petrone&author=T.+Henderson&author=D.+Ranasinghe&author=V.+G.+Zakrzewski&author=J.+Gao&author=N.+Rega&author=G.+Zheng&author=W.+Liang&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=K.+Throssell&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+J.+Bearpark&author=J.+J.+Heyd&author=E.+N.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=T.+A.+Keith&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+P.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=J.+M.+Millam&author=M.+Klene&author=C.+Adamo&author=R.+Cammi&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=O.+Farkas&author=J.+B.+Foresman&author=D.+J.+Fox&title=Gaussian+16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252016%26pub%3DGaussian%252C%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beigie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belmonte, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joubran, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmirski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mlynarski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickson, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span> <span> </span><span class="NLM_article-title">Structure based design of non-natural peptidic macrocyclic Mcl-1 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCqtLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=239-244&author=J.+W.+Johannesauthor=S.+Batesauthor=C.+Beigieauthor=M.+A.+Belmonteauthor=J.+Breenauthor=S.+Caoauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=J.+W.+Cuozzoauthor=C.+E.+Dumelinauthor=A.+D.+Fergusonauthor=S.+Habeshianauthor=D.+Hargreavesauthor=C.+Joubranauthor=S.+Kazmirskiauthor=A.+D.+Keefeauthor=M.+L.+Lambauthor=H.+Lanauthor=Y.+Liauthor=H.+Maauthor=S.+Mlynarskiauthor=M.+J.+Packerauthor=P.+B.+Rawlinsauthor=D.+W.+Robbinsauthor=H.+Shenauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=N.+Suauthor=D.+M.+Troastauthor=H.+Wangauthor=K.+F.+Wicksonauthor=C.+Wuauthor=Y.+Zhangauthor=Q.+Zhaoauthor=X.+Zhengauthor=A.+W.+Hird&title=Structure+based+design+of+non-natural+peptidic+macrocyclic+Mcl-1+inhibitors&doi=10.1021%2Facsmedchemlett.6b00464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure based design of non-natural peptidic macrocyclic Mcl-1 inhibitors</span></div><div class="casAuthors">Johannes, Jeffrey W.; Bates, Stephanie; Beigie, Carl; Belmonte, Matt; Breen, John; Cao, Shenggen; Centrella, Paolo A.; Clark, Matthew A.; Cuozzo, John W.; Dumelin, Christoph E.; Ferguson, Andrew D.; Habeshian, Sevan; Hargreaves, David; Joubran, Camil; Kazmirski, Steven; Keefe, Anthony D.; Lamb, Michelle L.; Lan, Haiye; Li, Yunxia; Ma, Hao; Mlynarski, Scott; Packer, Martin J.; Rawlins, Philip B.; Robbins, Daniel W.; Shen, Haidong; Sigel, Eric A.; Soutter, Holly H.; Su, Nancy; Troast, Dawn M.; Wang, Haiyun; Wickson, Kate F.; Wu, Chengyan; Zhang, Ying; Zhao, Qiuying; Zheng, Xiaolan; Hird, Alexander W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-244</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mcl-1 is a pro-apoptotic BH3 protein family member similar to Bcl-2 and Bcl-xL.  Overexpression of Mcl-1 is often seen in various tumors and allows cancer cells to evade apoptosis.  Here we report the discovery and optimization of a series of non-natural peptide Mcl-1 inhibitors.  Screening of DNA-encoded libraries resulted in hit compd. (I), a 1.5 μM Mcl-1 inhibitor.  A subsequent crystal structure demonstrated that compd. I bound to Mcl-1 in a β-turn conformation, such that the two ends of the peptide were close together.  This proximity allowed for the linking of the two ends of the peptide to form a macrocycle.  Macrocyclization resulted in an approx. 10-fold improvement in binding potency.  Further exploration of a key hydrophobic interaction with Mcl-1 protein and also with the moiety that engages Arg256 led to addnl. potency improvements.  The use of protein-ligand crystal structures and binding kinetics contributed to the design and understanding of the potency gains.  Optimized compd. (II) is a <3 nM Mcl-1 inhibitor, while inhibiting Bcl-2 at only 5 μM and Bcl-xL at >99 μM, and induces cleaved caspase-3 in MV4-11 cells with an IC50 of 3 μM after 6 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGRmmxgTy5drVg90H21EOLACvtfcHk0ljjx1wmfJ15Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCqtLzP&md5=42f9a8d863aedfe95231e29ffa4673b8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00464%26sid%3Dliteratum%253Aachs%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DBates%26aufirst%3DS.%26aulast%3DBeigie%26aufirst%3DC.%26aulast%3DBelmonte%26aufirst%3DM.%2BA.%26aulast%3DBreen%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DS.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DJoubran%26aufirst%3DC.%26aulast%3DKazmirski%26aufirst%3DS.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DLan%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DMlynarski%26aufirst%3DS.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobbins%26aufirst%3DD.%2BW.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWickson%26aufirst%3DK.%2BF.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DHird%26aufirst%3DA.%2BW.%26atitle%3DStructure%2520based%2520design%2520of%2520non-natural%2520peptidic%2520macrocyclic%2520Mcl-1%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D239%26epage%3D244%26doi%3D10.1021%2Facsmedchemlett.6b00464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of pyrazolo[1,5-a]pyrimidine B-cell lymphoma 6 (BCL6) binders and optimization to high affinity macrocyclic inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4386</span>– <span class="NLM_lpage">4402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00359</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00359" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4386-4402&author=W.+McCoullauthor=R.+D.+Abramsauthor=E.+Andersonauthor=K.+Bladesauthor=P.+Bartonauthor=M.+Boxauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=C.+Chuaquiauthor=R.+J.+Carbajoauthor=T.+Cheungauthor=E.+Codeauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=E.+Ganglauthor=N.+Gaoauthor=M.+Gristauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=P.+D.+Kemmittauthor=J.+E.+Nelsonauthor=N.+O%E2%80%99Connellauthor=D.+B.+Princeauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=G.+R.+Robbauthor=J.+Shiauthor=M.+J.+Waringauthor=D.+Whittakerauthor=M.+Wylotauthor=X.+Zhu&title=Discovery+of+pyrazolo%5B1%2C5-a%5Dpyrimidine+B-cell+lymphoma+6+%28BCL6%29+binders+and+optimization+to+high+affinity+macrocyclic+inhibitors&doi=10.1021%2Facs.jmedchem.7b00359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrazolo[1,5-a]pyrimidine B-Cell Lymphoma 6 (BCL6) Binders and Optimization to High Affinity Macrocyclic Inhibitors</span></div><div class="casAuthors">McCoull, William; Abrams, Roman D.; Anderson, Erica; Blades, Kevin; Barton, Peter; Box, Matthew; Burgess, Jonathan; Byth, Kate; Cao, Qing; Chuaqui, Claudio; Carbajo, Rodrigo J.; Cheung, Tony; Code, Erin; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gangl, Eric; Gao, Ning; Grist, Matthew; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kemmitt, Paul D.; Nelson, Jennifer E.; O'Connell, Nichole; Prince, D. Bryan; Raubo, Piotr; Rawlins, Philip B.; Robb, Graeme R.; Shi, Junjie; Waring, Michael J.; Whittaker, David; Wylot, Marta; Zhu, Xiahui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4386-4402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the protein-protein interaction between B-cell lymphoma 6 (BCL6) and corepressors has been implicated as a therapeutic target in diffuse large B-cell lymphoma (DLBCL) cancers and profiling of potent and selective BCL6 inhibitors are crit. to test this hypothesis.  We identified a pyrazolo[1,5-a]pyrimidine series of BCL6 binders from a fragment screen in parallel with a virtual screen.  Using structure-based drug design, binding affinity was increased 100000-fold.  This involved displacing crystallog. water, forming new ligand-protein interactions and a macrocyclization to favor the bioactive conformation of the ligands.  Optimization for slow off-rate const. kinetics was conducted as well as improving selectivity against an off-target kinase, CK2.  Potency in a cellular BCL6 assay was further optimized to afford highly selective probe mols.  Only weak antiproliferative effects were obsd. across a no. of DLBCL lines and a multiple myeloma cell line without a clear relationship to BCL6 potency.  As a result, we conclude that the BCL6 hypothesis in DLBCL cancer remains unproven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWp2OlXAZDFLVg90H21EOLACvtfcHk0ljjx1wmfJ15Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D&md5=fc4142453a0580edbf04802e15d4d9aa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00359%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DAbrams%26aufirst%3DR.%2BD.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGangl%26aufirst%3DE.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DNelson%26aufirst%3DJ.%2BE.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPrince%26aufirst%3DD.%2BB.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWylot%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520pyrazolo%255B1%252C5-a%255Dpyrimidine%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520binders%2520and%2520optimization%2520to%2520high%2520affinity%2520macrocyclic%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4386%26epage%3D4402%26doi%3D10.1021%2Facs.jmedchem.7b00359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaPlante, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubry, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, S. H.</span></span> <span> </span><span class="NLM_article-title">Ligand bioactive conformation plays a critical role in the design of drugs that target the hepatitis C virus NS3 protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1777</span>– <span class="NLM_lpage">1789</span>, <span class="refDoi"> DOI: 10.1021/jm401338c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401338c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Cktb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1777-1789&author=S.+R.+LaPlanteauthor=H.+Narauthor=C.+T.+Lemkeauthor=A.+Jakalianauthor=N.+Aubryauthor=S.+H.+Kawai&title=Ligand+bioactive+conformation+plays+a+critical+role+in+the+design+of+drugs+that+target+the+hepatitis+C+virus+NS3+protease&doi=10.1021%2Fjm401338c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Bioactive Conformation Plays a Critical Role in the Design of Drugs That Target the Hepatitis C Virus NS3 Protease</span></div><div class="casAuthors">LaPlante, Steven R.; Nar, Herbert; Lemke, Christopher T.; Jakalian, Araz; Aubry, Norman; Kawai, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1777-1789</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A ligand-focused strategy employed NMR, X-ray, modeling, and medicinal chem. to expose the crit. role that bioactive conformation played in the design of a variety of drugs that target the HCV protease.  The bioactive conformation (bound states) were detd. for key inhibitors identified along our drug discovery pathway from the hit to clin. compds.  All adopt similar bioactive conformations for the common core derived from the hit peptide DDIVPC.  A carefully designed SAR anal., based on the advanced inhibitor 1 in which the P1 to P3 side chains and the N-terminal Boc were sequentially truncated, revealed a correlation between affinity and the relative predominance of the bioactive conformation in the free state.  Interestingly, synergistic conformation effects on potency were also noted.  Comparisons with clin. and recently marketed drugs from the pharmaceutical industry showed that all have the same core and similar bioactive conformations.  This suggested that the variety of appendages discovered for these compds. also properly satisfy the bioactive conformation requirements and allowed for a large variety of HCV protease drug candidates to be designed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFi-4R4F9wwbVg90H21EOLACvtfcHk0lg8jRLlv5M7JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Cktb7P&md5=2ccb36504903255ba012223529318a69</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401338c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401338c%26sid%3Dliteratum%253Aachs%26aulast%3DLaPlante%26aufirst%3DS.%2BR.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DLemke%26aufirst%3DC.%2BT.%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DAubry%26aufirst%3DN.%26aulast%3DKawai%26aufirst%3DS.%2BH.%26atitle%3DLigand%2520bioactive%2520conformation%2520plays%2520a%2520critical%2520role%2520in%2520the%2520design%2520of%2520drugs%2520that%2520target%2520the%2520hepatitis%2520C%2520virus%2520NS3%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1777%26epage%3D1789%26doi%3D10.1021%2Fjm401338c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di Fabio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellacani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerrard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thewlis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemp, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massagrande, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchioro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsi, M.</span></span> <span> </span><span class="NLM_article-title">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5562</span>– <span class="NLM_lpage">5567</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.06.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmcl.2011.06.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=21831639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=5562-5567&author=R.+Di+Fabioauthor=A.+Pellacaniauthor=S.+Faedoauthor=A.+Rothauthor=L.+Piccoliauthor=P.+Gerrardauthor=R.+A.+Porterauthor=C.+N.+Johnsonauthor=K.+Thewlisauthor=D.+Donatiauthor=L.+Stasiauthor=S.+Spadaauthor=G.+Stempauthor=D.+Nashauthor=C.+Branchauthor=L.+Kindonauthor=M.+Massagrandeauthor=A.+Poffeauthor=S.+Braggioauthor=E.+Chiarparinauthor=C.+Marchioroauthor=E.+Rattiauthor=M.+Corsi&title=Discovery+process+and+pharmacological+characterization+of+a+novel+dual+orexin+1+and+orexin+2+receptor+antagonist+useful+for+treatment+of+sleep+disorders&doi=10.1016%2Fj.bmcl.2011.06.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span></div><div class="casAuthors">Di Fabio, Romano; Pellacani, Annalisa; Faedo, Stefania; Roth, Adelheid; Piccoli, Laura; Gerrard, Philip; Porter, Rod A.; Johnson, Christopher N.; Thewlis, Kevin; Donati, Daniele; Stasi, Luigi; Spada, Simone; Stemp, Geoffrey; Nash, David; Branch, Clive; Kindon, Leanda; Massagrande, Mario; Poffe, Alessandro; Braggio, Simone; Chiarparin, Elisabetta; Marchioro, Carla; Ratti, Emiliangelo; Corsi, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5562-5567</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The hypothalamic peptides orexin-A and orexin-B are potent agonists of two G-protein coupled receptors, namely the OX1 and the OX2 receptor.  These receptors are widely distributed, though differentially, in the rat brain.  In particular, the OX1 receptor is highly expressed throughout the hypothalamus, while the OX2 receptor is mainly located in the ventral posterior nucleus.  A large body of compelling evidence, both pre-clin. and clin., suggests that the orexin system is profoundly implicated in sleep disorders.  In particular, modulation of the orexin receptors activation by appropriate antagonists was proven to be an efficacious strategy for the treatment of insomnia in man.  A novel, drug-like bis-amido piperidine deriv. (I) was identified as potent dual OX1 and OX2 receptor antagonists, highly effective in a pre-clin. model of sleep.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3g2IG2vbbbVg90H21EOLACvtfcHk0lg8jRLlv5M7JA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL&md5=02d4a2f2682a9e35689a9e1553858e69</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.06.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.06.086%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BFabio%26aufirst%3DR.%26aulast%3DPellacani%26aufirst%3DA.%26aulast%3DFaedo%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DA.%26aulast%3DPiccoli%26aufirst%3DL.%26aulast%3DGerrard%26aufirst%3DP.%26aulast%3DPorter%26aufirst%3DR.%2BA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DThewlis%26aufirst%3DK.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DStasi%26aufirst%3DL.%26aulast%3DSpada%26aufirst%3DS.%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DNash%26aufirst%3DD.%26aulast%3DBranch%26aufirst%3DC.%26aulast%3DKindon%26aufirst%3DL.%26aulast%3DMassagrande%26aufirst%3DM.%26aulast%3DPoffe%26aufirst%3DA.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DMarchioro%26aufirst%3DC.%26aulast%3DRatti%26aufirst%3DE.%26aulast%3DCorsi%26aufirst%3DM.%26atitle%3DDiscovery%2520process%2520and%2520pharmacological%2520characterization%2520of%2520a%2520novel%2520dual%2520orexin%25201%2520and%2520orexin%25202%2520receptor%2520antagonist%2520useful%2520for%2520treatment%2520of%2520sleep%2520disorders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D5562%26epage%3D5567%26doi%3D10.1016%2Fj.bmcl.2011.06.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinsmore, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogusky, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culberson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homnick, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zartman, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosser, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaber, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koblan, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huff, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, T. M.</span></span> <span> </span><span class="NLM_article-title">Conformational restriction of flexible ligands guided by the transferred NOE experiment: Potent macrocyclic inhibitors of farnesyltransferase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">2107</span>– <span class="NLM_lpage">2108</span>, <span class="refDoi"> DOI: 10.1021/ja003673q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja003673q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD3MXpslOjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=2107-2108&author=C.+J.+Dinsmoreauthor=M.+J.+Boguskyauthor=J.+C.+Culbersonauthor=J.+M.+Bergmanauthor=C.+F.+Homnickauthor=C.+B.+Zartmanauthor=S.+D.+Mosserauthor=M.+D.+Schaberauthor=R.+G.+Robinsonauthor=K.+S.+Koblanauthor=H.+E.+Huberauthor=S.+L.+Grahamauthor=G.+D.+Hartmanauthor=J.+R.+Huffauthor=T.+M.+Williams&title=Conformational+restriction+of+flexible+ligands+guided+by+the+transferred+NOE+experiment%3A+Potent+macrocyclic+inhibitors+of+farnesyltransferase&doi=10.1021%2Fja003673q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Restriction of Flexible Ligands Guided by the Transferred NOE Experiment: Potent Macrocyclic Inhibitors of Farnesyltransferase</span></div><div class="casAuthors">Dinsmore, Christopher J.; Bogusky, Michael J.; Culberson, J. Christopher; Bergman, Jeffrey M.; Homnick, Carl F.; Zartman, C. Blair; Mosser, Scott D.; Schaber, Michael D.; Robinson, Ronald G.; Koblan, Kenneth S.; Huber, Hans E.; Graham, Samuel L.; Hartman, George D.; Huff, Joel R.; Williams, Theresa M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2107-2108</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Farnesyltransferase is an important post translational processing enzyme involved in the signal transduction during cell proliferation.  Inhibitors of this enzyme are promising antitumor agents and several are currently being evaluated in the human clin. trials.  This paper describes the design and synthesis of a highly potent macrocyclic farnesyltransferase inhibitor based on the transferred NOE structure of a conformationally flexible analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMVw4lvnkqI7Vg90H21EOLACvtfcHk0ljCtyTszk8fPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXpslOjsQ%253D%253D&md5=3ff79e2cbb07ebab6ba7f03c4f1a4417</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja003673q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja003673q%26sid%3Dliteratum%253Aachs%26aulast%3DDinsmore%26aufirst%3DC.%2BJ.%26aulast%3DBogusky%26aufirst%3DM.%2BJ.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DBergman%26aufirst%3DJ.%2BM.%26aulast%3DHomnick%26aufirst%3DC.%2BF.%26aulast%3DZartman%26aufirst%3DC.%2BB.%26aulast%3DMosser%26aufirst%3DS.%2BD.%26aulast%3DSchaber%26aufirst%3DM.%2BD.%26aulast%3DRobinson%26aufirst%3DR.%2BG.%26aulast%3DKoblan%26aufirst%3DK.%2BS.%26aulast%3DHuber%26aufirst%3DH.%2BE.%26aulast%3DGraham%26aufirst%3DS.%2BL.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26aulast%3DHuff%26aufirst%3DJ.%2BR.%26aulast%3DWilliams%26aufirst%3DT.%2BM.%26atitle%3DConformational%2520restriction%2520of%2520flexible%2520ligands%2520guided%2520by%2520the%2520transferred%2520NOE%2520experiment%253A%2520Potent%2520macrocyclic%2520inhibitors%2520of%2520farnesyltransferase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D2107%26epage%3D2108%26doi%3D10.1021%2Fja003673q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span> <span> </span><span class="NLM_article-title">Conformational control in structure-based drug design</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2825</span>– <span class="NLM_lpage">2837</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.04.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmcl.2017.04.079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28479196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFamtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2825-2837&author=Y.+Zhengauthor=C.+M.+Ticeauthor=S.+B.+Singh&title=Conformational+control+in+structure-based+drug+design&doi=10.1016%2Fj.bmcl.2017.04.079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational control in structure-based drug design</span></div><div class="casAuthors">Zheng, Yajun; Tice, Colin M.; Singh, Suresh B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2825-2837</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In structure-based drug design, the basic goal is to design mols. that fit complementarily to a given binding pocket.  Since such computationally modeled mols. may not adopt the intended bound conformation outside the binding pocket, one challenge is to ensure that the designed ligands adopt similar low energy conformations both inside and outside of the binding pocket.  Computational chem. methods and conformational preferences of small mols. from PDB and Cambridge Structural Database (CSD) can be used to predict the bound structures of the designed mols.  Herein, we review applications of conformational control in structure-based drug design using selected examples from the recent medicinal chem. literature.  The main purpose is to highlight some intriguing conformational features that can be applied to other drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbfYQ3iP4BT7Vg90H21EOLACvtfcHk0ljCtyTszk8fPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFamtr8%253D&md5=aef5a7aa51f9b6a8454345a4e38d95db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.04.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.04.079%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DTice%26aufirst%3DC.%2BM.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26atitle%3DConformational%2520control%2520in%2520structure-based%2520drug%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2825%26epage%3D2837%26doi%3D10.1016%2Fj.bmcl.2017.04.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, A. D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacCoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, J. P.</span></span> <span> </span><span class="NLM_article-title">Importance of rigidity in designing small molecule drugs to tackle protein-protein interactions (PPIs) through stabilization of desired conformers</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4283</span>– <span class="NLM_lpage">4289</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4283-4289&author=A.+D.+G.+Lawsonauthor=M.+MacCossauthor=J.+P.+Heer&title=Importance+of+rigidity+in+designing+small+molecule+drugs+to+tackle+protein-protein+interactions+%28PPIs%29+through+stabilization+of+desired+conformers&doi=10.1021%2Facs.jmedchem.7b01120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Importance of Rigidity in Designing Small Molecule Drugs To Tackle Protein-Protein Interactions (PPIs) through Stabilization of Desired Conformers</span></div><div class="casAuthors">Lawson, Alastair D. G.; MacCoss, Malcolm; Heer, Jag P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4283-4289</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tackling PPIs, particularly by stabilizing clin. favored conformations of target proteins, with orally available, bona fide small mols. remains a significant but immensely worthwhile challenge for the pharmaceutical industry.  Success may be more likely through the application of nature's learnings to build intrinsic rigidity into the design of clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN_JJHmVzv_rVg90H21EOLACvtfcHk0ljDsP3kpQWKVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtb7J&md5=665f95b08aeea2757f0a3b4dc4f813ed</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01120%26sid%3Dliteratum%253Aachs%26aulast%3DLawson%26aufirst%3DA.%2BD.%2BG.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DHeer%26aufirst%3DJ.%2BP.%26atitle%3DImportance%2520of%2520rigidity%2520in%2520designing%2520small%2520molecule%2520drugs%2520to%2520tackle%2520protein-protein%2520interactions%2520%2528PPIs%2529%2520through%2520stabilization%2520of%2520desired%2520conformers%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4283%26epage%3D4289%26doi%3D10.1021%2Facs.jmedchem.7b01120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span>; <span class="NLM_string-name">Souers, A. J.</span>; <span class="NLM_string-name">Bruncko, M.</span>; <span class="NLM_string-name">Song, X.</span>; <span class="NLM_string-name">Wang, X.</span>; <span class="NLM_string-name">Hasvold, L. A.</span>; <span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Kunzer, A. R.</span>; <span class="NLM_string-name">Park, C.-M.</span>; <span class="NLM_string-name">Wendt, M. D.</span>; <span class="NLM_string-name">Tao, Z.-F.</span></span> <span> </span><span class="NLM_article-title">7-Substituted indoles as mcl-1 protein inhibitors and their preparation</span>. PCT Int. Appl. <span class="NLM_patent">WO2008131000A2</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=S.+W.+Elmore&author=A.+J.+Souers&author=M.+Bruncko&author=X.+Song&author=X.+Wang&author=L.+A.+Hasvold&author=L.+Wang&author=A.+R.+Kunzer&author=C.-M.+Park&author=M.+D.+Wendt&author=Z.-F.+Tao&title=7-Substituted+indoles+as+mcl-1+protein+inhibitors+and+their+preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3D7-Substituted%2520indoles%2520as%2520mcl-1%2520protein%2520inhibitors%2520and%2520their%2520preparation%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Almes, F. J.</span></span> <span> </span><span class="NLM_article-title">Kinetic binding assays for the analysis of protein-ligand interactions</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.ddtec.2015.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=26724330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1egtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=1-8&author=F.+J.+Meyer-Almes&title=Kinetic+binding+assays+for+the+analysis+of+protein-ligand+interactions&doi=10.1016%2Fj.ddtec.2015.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic binding assays for the analysis of protein-ligand interactions</span></div><div class="casAuthors">Meyer-Almes Franz-Josef</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The importance of binding kinetics in terms of residence time and on-rate in drug discovery has been broadly accepted in the past few years.  Furthermore, evidence has accumulated that the optimal binding mechanism of a drug to its target molecule is related to physiological efficacy as well as selectivity and thus drug safety.  Homogeneous fluorescence-based binding assays have been shown to enable high throughput kinetics requiring only small amounts of protein and can be developed to elucidate even complex mechanisms of molecular recognition.  A generalized approach is proposed that combines high quality kinetic and equilibrium data in an Integrated Global Fit analysis yielding the most probable binding mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwRMy18rtTEzjk2McQZ6XmfW6udTcc2eZ8n7naF7o3zbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1egtg%253D%253D&md5=be48951ac65ad4bfb9bc2ec4a398ee1f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer-Almes%26aufirst%3DF.%2BJ.%26atitle%3DKinetic%2520binding%2520assays%2520for%2520the%2520analysis%2520of%2520protein-ligand%2520interactions%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ddtec.2015.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Folmer, R. H. A.</span></span> <span> </span><span class="NLM_article-title">Drug target residence time: A misleading concept</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.07.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.drudis.2017.07.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28782685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSitbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=12-16&author=R.+H.+A.+Folmer&title=Drug+target+residence+time%3A+A+misleading+concept&doi=10.1016%2Fj.drudis.2017.07.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Drug target residence time: a misleading concept</span></div><div class="casAuthors">Folmer, Rutger H. A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-16</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Since the importance of drug target residence time was first highlighted more 10 years ago, slow binding kinetics has received much attention in the drug discovery literature, and indeed within pharmaceutical research.  However, the residence concept as presented in most papers is supported by rather misleading simulations and arguments, and by examples where compds. are taken out of their pharmacokinetic context.  Moreover, fast assocn. is typically more desirable than slow, and advantages of long residence time, notably a potential disconnect between pharmacodynamics (PD) and pharmacokinetics (PK), would be partially or completely offset by slow on-rate.  Therefore, plain potency is likely a better predictor of drug development success than is residence time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5NQgx9uuufLVg90H21EOLACvtfcHk0ljyUGdYItFZng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSitbjI&md5=61f68447e7f8eaf6b93c4b09fc5afeda</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.07.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.07.016%26sid%3Dliteratum%253Aachs%26aulast%3DFolmer%26aufirst%3DR.%2BH.%2BA.%26atitle%3DDrug%2520target%2520residence%2520time%253A%2520A%2520misleading%2520concept%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D12%26epage%3D16%26doi%3D10.1016%2Fj.drudis.2017.07.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span> <span> </span><span class="NLM_article-title">On-rate based optimization of structure-kinetic relationship--surfing the kinetic map</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.ddtec.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=26724331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=9-15&author=A.+Schoopauthor=F.+Dey&title=On-rate+based+optimization+of+structure-kinetic+relationship%2D%2Dsurfing+the+kinetic+map&doi=10.1016%2Fj.ddtec.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">On-rate based optimization of structure-kinetic relationship--surfing the kinetic map</span></div><div class="casAuthors">Schoop Andreas; Dey Fabian</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In the lead discovery process residence time has become an important parameter for the identification and characterization of the most efficacious compounds in vivo.  To enable the success of compound optimization by medicinal chemistry toward a desired residence time the understanding of structure-kinetic relationship (SKR) is essential.  This article reviews various approaches to monitor SKR and suggests using the on-rate as the key monitoring parameter.  The literature is reviewed and examples of compound series with low variability as well as with significant changes in on-rates are highlighted.  Furthermore, findings of kinetic on-rate changes are presented and potential underlying rationales are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwRMy18rtTE3T4ugvCDgnZfW6udTcc2ebyCJyx7t6MP7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D&md5=a7f0dee9cd02f38a945123e7faa9bc1c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoop%26aufirst%3DA.%26aulast%3DDey%26aufirst%3DF.%26atitle%3DOn-rate%2520based%2520optimization%2520of%2520structure-kinetic%2520relationship--surfing%2520the%2520kinetic%2520map%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.ddtec.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span> <span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">730</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0lhzmHLdbCqhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunn, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, P.</span></span> <span> </span><span class="NLM_article-title">Investigation of the effect of molecular properties on the binding kinetics of a ligand to its biological target</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1039/c2md00270a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1039%2Fc2md00270a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC38Xks1Orsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=449-452&author=D.+C.+Millerauthor=G.+Lunnauthor=P.+Jonesauthor=Y.+Sabnisauthor=N.+Daviesauthor=P.+Driscoll&title=Investigation+of+the+effect+of+molecular+properties+on+the+binding+kinetics+of+a+ligand+to+its+biological+target&doi=10.1039%2Fc2md00270a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of the effect of molecular properties on the binding kinetics of a ligand to its biological target</span></div><div class="casAuthors">Miller, Duncan C.; Lunn, Graham; Jones, Peter; Sabnis, Yogesh; Davies, Nichola L.; Driscoll, Paul</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-452</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A database of binding kinetic data across multiple targets has been created.  Anal. of this database provides insights into trends for the effects of physicochem. properties (mol. wt., clogP, rotatable bond count) on the dissocn. kinetics of ligands from their biol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrvjiZtiu9VLVg90H21EOLACvtfcHk0lhzmHLdbCqhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xks1Orsro%253D&md5=84c6b9d95ec6df61145fe678c78d5491</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2Fc2md00270a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2md00270a%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DLunn%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DDavies%26aufirst%3DN.%26aulast%3DDriscoll%26aufirst%3DP.%26atitle%3DInvestigation%2520of%2520the%2520effect%2520of%2520molecular%2520properties%2520on%2520the%2520binding%2520kinetics%2520of%2520a%2520ligand%2520to%2520its%2520biological%2520target%26jtitle%3DMedChemComm%26date%3D2012%26volume%3D3%26spage%3D449%26epage%3D452%26doi%3D10.1039%2Fc2md00270a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, P.</span></span> <span> </span><span class="NLM_article-title">How fast is protein-ligand association?</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2017.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.tibs.2017.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28917971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgtbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=847-849&author=S.+Gianniauthor=P.+Jemth&title=How+fast+is+protein-ligand+association%3F&doi=10.1016%2Fj.tibs.2017.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">How Fast Is Protein-Ligand Association?</span></div><div class="casAuthors">Gianni, Stefano; Jemth, Per</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-849</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">There is increasing interest in studying protein interactions and their role in cell biol. using kinetics.  However, there is confusion about the proper terminol. in terms of the distinction between rates and rate consts.  We recommend a more stringent use of the words speed, fast, slow, rate, and rate const.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruENTH_GopPLVg90H21EOLACvtfcHk0lhzmHLdbCqhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgtbrF&md5=6707642cbeb3bbd645c49d3ca667b735</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2017.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2017.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DGianni%26aufirst%3DS.%26aulast%3DJemth%26aufirst%3DP.%26atitle%3DHow%2520fast%2520is%2520protein-ligand%2520association%253F%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2017%26volume%3D42%26spage%3D847%26epage%3D849%26doi%3D10.1016%2Fj.tibs.2017.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer generation with OMEGA: Algorithm and validation using high quality structures from the protein databank and cambridge structural database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+Algorithm+and+validation+using+high+quality+structures+from+the+protein+databank+and+cambridge+structural+database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lhb29Lua_pHsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520protein%2520databank%2520and%2520cambridge%2520structural%2520database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span> <span> </span><span class="NLM_article-title">Reproducing the conformations of protein-bound ligands: A critical evaluation of several popular conformational searching tools</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1023/A:1015930826903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1023%2FA%3A1015930826903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=12160095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BD38vgsF2rtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=1137-1152&author=J.+Bostr%C3%B6m&title=Reproducing+the+conformations+of+protein-bound+ligands%3A+A+critical+evaluation+of+several+popular+conformational+searching+tools&doi=10.1023%2FA%3A1015930826903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Reproducing the conformations of protein-bound ligands: a critical evaluation of several popular conformational searching tools</span></div><div class="casAuthors">Bostrom J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1137-52</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">Several programs (Catalyst, Confort, Flo99, MacroModel, and Omega) that are commonly used to generate conformational ensembles have been tested for their ability to reproduce bioactive conformations.  The ligands from thirty-two different ligand-protein complexes determined by high-resolution (< 2.0 A) X-ray crystallography have been analyzed.  The Low-Mode Conformational Search method (with AMBER* and the GB/SA hydration model), as implemented in MacroModel, was found to perform better than the other algorithms.  The rule-based method Omega, which is orders of magnitude faster than the other methods, also gave reasonable results but were found to be dependent on the input structure.  The methods supporting diverse sampling (Catalyst, Confort) performed least well.  For the seven ligands in the set having eight or more rotatable bonds, none of the bioactive conformations were ever found, save for one exception (Flo99).  These ligands do not bind in a local minimum conformation according to AMBER*\GB/SA.  Taking these last two observations together, it is clear that geometrically similar structures should be collected in order to increase the probability of finding the bioactive conformation among the generated ensembles.  Factors influencing bioactive conformational retrieval have been identified and are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSG5txM82Py66cY7HFp97qMfW6udTcc2ebSpwtkjQNytLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38vgsF2rtw%253D%253D&md5=5b08faf2b096c32cf8cda40f3b8218d0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015930826903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015930826903%26sid%3Dliteratum%253Aachs%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26atitle%3DReproducing%2520the%2520conformations%2520of%2520protein-bound%2520ligands%253A%2520A%2520critical%2520evaluation%2520of%2520several%2520popular%2520conformational%2520searching%2520tools%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D1137%26epage%3D1152%26doi%3D10.1023%2FA%3A1015930826903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izrailev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational sampling of bioactive molecules: A comparative study</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1086</span>, <span class="refDoi"> DOI: 10.1021/ci6005454</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci6005454" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvFKgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1067-1086&author=D.+K.+Agrafiotisauthor=A.+C.+Gibbsauthor=F.+Zhuauthor=S.+Izrailevauthor=E.+Martin&title=Conformational+sampling+of+bioactive+molecules%3A+A+comparative+study&doi=10.1021%2Fci6005454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational sampling of bioactive molecules: a comparative study</span></div><div class="casAuthors">Agrafiotis, Dimitris K.; Gibbs, Alan C.; Zhu, Fangqiang; Izrailev, Sergei; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1067-1086</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The necessity to generate conformations that sample the entire conformational space accessible to a given mol. is ubiquitous in the field of computer-aided drug design.  Protein-ligand docking, 3D database searching, and 3D QSAR are three commonly used techniques that depend critically upon the quality and diversity of the generated conformers.  Although there are a wide range of conformational search algorithms available, the extent to which they sample conformational space is often unclear.  To address this question, we conducted a robust comparison of the search algorithms implemented in several widely used mol. modeling packages, including Catalyst, Macromodel, Omega, MOE, and Rubicon as well as our own method, stochastic proximity embedding (SPE).  We found that SPE used in conjunction with conformational boosting, a heuristic for biasing conformational search toward more extended or compact geometries, along with Catalyst, are significantly more effective in sampling the full range of conformational space compared to the other methods, which show distinct preferences for either more extended or more compact geometries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYXaKrPdkIBrVg90H21EOLACvtfcHk0lhb29Lua_pHsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvFKgt7s%253D&md5=ef380e9f957b8e7188a6c92ee61ab098</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci6005454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6005454%26sid%3Dliteratum%253Aachs%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DGibbs%26aufirst%3DA.%2BC.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DIzrailev%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520sampling%2520of%2520bioactive%2520molecules%253A%2520A%2520comparative%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1067%26epage%3D1086%26doi%3D10.1021%2Fci6005454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">LowModeMD--implicit low-mode velocity filtering applied to conformational search of macrocycles and protein loops</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1021/ci900508k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900508k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=792-800&author=P.+Labute&title=LowModeMD%2D%2Dimplicit+low-mode+velocity+filtering+applied+to+conformational+search+of+macrocycles+and+protein+loops&doi=10.1021%2Fci900508k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">LowModeMD - Implicit Low-Mode Velocity Filtering Applied to Conformational Search of Macrocycles and Protein Loops</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">792-800</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for conformational search of complex mol. systems such as macrocycles and protein loops.  The method is based on perturbing an existing conformation along a mol. dynamics trajectory using initial at. velocities with kinetic energy concd. on the low-frequency vibrational modes, followed by energy minimization.  A novel Chebyshev polynomial filter is used to heavily dampen the high-frequency components of a randomly generated Maxwell-Boltzmann velocity vector.  The method is very efficient, even for large systems; it is straightforward to implement and requires only std. force-field energy and gradient evaluations.  The results of several computational expts. suggest that the method is capable of efficiently sampling low-strain energy conformations of complex systems with nontrivial nonbonded interaction networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfWZjZhFqqUbVg90H21EOLACvtfcHk0lgIb8tp_RgknQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D&md5=bb800eb2f9635a661578b978dae9da0a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fci900508k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900508k%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DLowModeMD--implicit%2520low-mode%2520velocity%2520filtering%2520applied%2520to%2520conformational%2520search%2520of%2520macrocycles%2520and%2520protein%2520loops%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D792%26epage%3D800%26doi%3D10.1021%2Fci900508k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minehardt, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Car, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J. D.</span></span> <span> </span><span class="NLM_article-title">Protonation-induced stereoisomerism in nicotine: conformational studies using classical (AMBER) and ab initio (Car-Parrinello) molecular dynamics</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1007/s10822-005-0096-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1007%2Fs10822-005-0096-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=16059663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Ons7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=1-15&author=P.+S.+Hammondauthor=Y.+Wuauthor=R.+Harrisauthor=T.+J.+Minehardtauthor=R.+Carauthor=J.+D.+Schmitt&title=Protonation-induced+stereoisomerism+in+nicotine%3A+conformational+studies+using+classical+%28AMBER%29+and+ab+initio+%28Car-Parrinello%29+molecular+dynamics&doi=10.1007%2Fs10822-005-0096-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Protonation-induced stereoisomerism in nicotine: Conformational studies using classical (AMBER) and ab initio (Car-Parrinello) molecular dynamics</span></div><div class="casAuthors">Hammond, Philip S.; Wu, Yudong; Harris, Rebecca; Minehardt, Todd J.; Car, Roberto; Schmitt, Jeffrey D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A variety of biol. active small mols. contain prochiral tertiary amines, which become chiral centers upon protonation.  S-nicotine, the prototypical nicotinic acetylcholine receptor agonist, produces two diastereomers on protonation.  Results, using both classical (AMBER) and ab initio (Car-Parrinello) mol. dynamical studies, illustrate the significant differences in conformational space explored by each diastereomer.  As is expected, this phenomenon has an appreciable effect on nicotine's energy hypersurface and leads to differentiation in mol. shape and divergent sampling.  Thus, protonation induced isomerism can produce dynamic effects that may influence the behavior of a mol. in its interaction with a target protein.  We also examine differences in the conformational dynamics for each diastereomer as quantified by both mol. dynamics methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_oE3cUQDNcrVg90H21EOLACvtfcHk0lgIb8tp_RgknQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Ons7k%253D&md5=f38bcd45b40a07f015ce82bf8af973c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs10822-005-0096-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-005-0096-7%26sid%3Dliteratum%253Aachs%26aulast%3DHammond%26aufirst%3DP.%2BS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DMinehardt%26aufirst%3DT.%2BJ.%26aulast%3DCar%26aufirst%3DR.%26aulast%3DSchmitt%26aufirst%3DJ.%2BD.%26atitle%3DProtonation-induced%2520stereoisomerism%2520in%2520nicotine%253A%2520conformational%2520studies%2520using%2520classical%2520%2528AMBER%2529%2520and%2520ab%2520initio%2520%2528Car-Parrinello%2529%2520molecular%2520dynamics%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2005%26volume%3D19%26spage%3D1%26epage%3D15%26doi%3D10.1007%2Fs10822-005-0096-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of macrocycles: finding what common search methods miss</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2242</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.1021/ci900238a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900238a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2242-2259&author=P.+Bonnetauthor=D.+K.+Agrafiotisauthor=F.+Zhuauthor=E.+Martin&title=Conformational+analysis+of+macrocycles%3A+finding+what+common+search+methods+miss&doi=10.1021%2Fci900238a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span></div><div class="casAuthors">Bonnet, Pascal; Agrafiotis, Dimitris K.; Zhu, Fangqiang; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2242-2259</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As computational drug design becomes increasingly reliant on virtual screening and on high-throughput 3D modeling, the need for fast, robust, and reliable methods for sampling mol. conformations has become greater than ever.  Furthermore, chem. novelty is at a premium, forcing medicinal chemists to explore more complex structural motifs and unusual topologies.  This necessitates the use of conformational sampling techniques that work well in all cases.  Here, we compare the performance of several popular conformational search algorithms on three broad classes of macrocyclic mols.  These methods include Catalyst, CAESAR, MacroModel, MOE, Omega, Rubicon and two newer self-organizing algorithms known as stochastic proximity embedding (SPE) and self-organizing superimposition (SOS) that have been developed at Johnson & Johnson.  Our results show a compelling advantage for the three distance geometry methods (SOS, SPE, and Rubicon) followed to a lesser extent by MacroModel.  The remaining techniques, particularly those based on systematic search, often failed to identify any of the lowest energy conformations and are unsuitable for this class of structures.  Taken together with our previous study on drug-like mols., these results suggest that SPE and SOS are two of the most robust and universally applicable conformational search methods, with the latter being preferred because of its superior speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIMu1b7U477Vg90H21EOLACvtfcHk0lgIb8tp_RgknQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF&md5=9b4a5919415ef0660c4f9ff18d9b76f4</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci900238a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900238a%26sid%3Dliteratum%253Aachs%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520analysis%2520of%2520macrocycles%253A%2520finding%2520what%2520common%2520search%2520methods%2520miss%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2242%26epage%3D2259%26doi%3D10.1021%2Fci900238a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span> <span> </span><span class="NLM_article-title">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7898</span>– <span class="NLM_lpage">7920</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmc.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=24184215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7898-7920&author=I.+J.+Chenauthor=N.+Foloppe&title=Tackling+the+conformational+sampling+of+larger+flexible+compounds+and+macrocycles+in+pharmacology+and+drug+discovery&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span></div><div class="casAuthors">Chen, I-Jen; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7920</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational conformational sampling underpins much of mol. modeling and design in pharmaceutical work.  The sampling of smaller drug-like compds. has been an active area of research.  However, few studies have tested in details the sampling of larger more flexible compds., which are also relevant to drug discovery, including therapeutic peptides, macrocycles, and inhibitors of protein-protein interactions.  Here, we investigate extensively mainstream conformational sampling methods on three carefully curated compd. sets, namely the 'Drug-like', larger 'Flexible', and 'Macrocycle' compds.  These test mols. are chem. diverse with reliable X-ray protein-bound bioactive structures.  The compared sampling methods include Stochastic Search and the recent LowModeMD from MOE, all the low-mode based approaches from MacroModel, and MD/LLMOD recently developed for macrocycles.  In addn. to default settings, key parameters of the sampling protocols were explored.  The performance of the computational protocols was assessed via (i) the reprodn. of the X-ray bioactive structures, (ii) the size, coverage and diversity of the output conformational ensembles, (iii) the compactness/extendedness of the conformers, and (iv) the ability to locate the global energy min.  The influence of the stochastic nature of the searches on the results was also examd.  Much better results were obtained by adopting search parameters enhanced over the default settings, while maintaining computational tractability.  In MOE, the recent LowModeMD emerged as the method of choice.  Mixed torsional/low-mode from MacroModel performed as well as LowModeMD, and MD/LLMOD performed well for macrocycles.  The low-mode based approaches yielded very encouraging results with the flexible and macrocycle sets.  Thus, one can productively tackle the computational conformational search of larger flexible compds. for drug discovery, including macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwhGIpE1mMbVg90H21EOLACvtfcHk0lgIb8tp_RgknQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK&md5=2699227e69410a5673dec749169fd578</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DTackling%2520the%2520conformational%2520sampling%2520of%2520larger%2520flexible%2520compounds%2520and%2520macrocycles%2520in%2520pharmacology%2520and%2520drug%2520discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7898%26epage%3D7920%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spronk, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span> <span> </span><span class="NLM_article-title">Improving accuracy, diversity, and speed with prime macrocycle conformational sampling</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1881</span>– <span class="NLM_lpage">1894</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1881-1894&author=D.+Sindhikaraauthor=S.+A.+Spronkauthor=T.+Dayauthor=K.+Borrelliauthor=D.+L.+Cheneyauthor=S.+L.+Posy&title=Improving+accuracy%2C+diversity%2C+and+speed+with+prime+macrocycle+conformational+sampling&doi=10.1021%2Facs.jcim.7b00052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Improving Accuracy, Diversity, and Speed with Prime Macrocycle Conformational Sampling</span></div><div class="casAuthors">Sindhikara, Dan; Spronk, Steven A.; Day, Tyler; Borrelli, Ken; Cheney, Daniel L.; Posy, Shana L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1881-1894</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for exploring macrocycle conformational space, Prime Macrocycle Conformational Sampling (Prime-MCS), is introduced and evaluated in the context of other available algorithms (Mol. Dynamics, LowModeMD in MOE, and MacroModel Baseline Search).  The algorithms were benchmarked on a dataset of 208 macrocycles which was curated for diversity from the Cambridge Structural Database, the Protein Databank, and the Biol. Interesting Mol. Ref. Dictionary.  The algorithms were evaluated in terms of accuracy (ability to reproduce the crystal structure), diversity (coverage of conformational space), and computational speed.  Prime-MCS most reliably reproduced crystallog. structures for RMSD thresholds >1.0 Å, most often produced the most diverse conformational ensemble, and was most often the fastest algorithm.  Detailed anal. and examn. of both typical and outlier cases were performed to reveal characteristics, shortcomings, expected performance, and complementarity of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfeDx3IcpZD7Vg90H21EOLACvtfcHk0liNKjwb2p0Fkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ&md5=a2b20d25c2ad2cede1c3a5fd0b5840ce</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00052%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DSpronk%26aufirst%3DS.%2BA.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DBorrelli%26aufirst%3DK.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26atitle%3DImproving%2520accuracy%252C%2520diversity%252C%2520and%2520speed%2520with%2520prime%2520macrocycle%2520conformational%2520sampling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D1881%26epage%3D1894%26doi%3D10.1021%2Facs.jcim.7b00052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0liNKjwb2p0Fkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span> <span> </span><span class="NLM_article-title">Defining optimum lipophilicity and molecular weight ranges for drug candidates-Molecular weight dependent lower logD limits based on permeability</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2844</span>– <span class="NLM_lpage">2851</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmcl.2009.03.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=19361989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltl2rtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2844-2851&author=M.+J.+Waring&title=Defining+optimum+lipophilicity+and+molecular+weight+ranges+for+drug+candidates-Molecular+weight+dependent+lower+logD+limits+based+on+permeability&doi=10.1016%2Fj.bmcl.2009.03.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Defining optimum lipophilicity and molecular weight ranges for drug candidates-Molecular weight dependent lower log D limits based on permeability</span></div><div class="casAuthors">Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2844-2851</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Anal. of a large, structurally diverse Caco-2 permeability dataset using a variety of statistical techniques suggests that log D and mol. wt. are the most important factors in detg. the permeability of drug candidates.  The limit for log D is shown to be dependent on mol. wt.  These limits are shown to be potentially superior to current guidelines in increasing the chances of finding highly permeable compds.  When combined with suggested upper limits for lipophilicity suggested in the literature based on the avoidance of toxicol. and other adverse effects, this helps define a lipophilicity range that is optimum for drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXwn3KyrzERrVg90H21EOLACvtfcHk0liNKjwb2p0Fkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltl2rtr0%253D&md5=1f04795ddf6e6b84d132a81e554c2516</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.109%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DDefining%2520optimum%2520lipophilicity%2520and%2520molecular%2520weight%2520ranges%2520for%2520drug%2520candidates-Molecular%2520weight%2520dependent%2520lower%2520logD%2520limits%2520based%2520on%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2844%26epage%3D2851%26doi%3D10.1016%2Fj.bmcl.2009.03.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span> <i>Small-Molecule
Drug Discovery Suite 2018-4</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Small-Molecule%0ADrug+Discovery+Suite+2018-4%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSmall-Molecule%250ADrug%2520Discovery%2520Suite%25202018-4%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Paul D. Leeson, A. Patricia Bento, Anna Gaulton, Anne Hersey, Emma J. Manners, Chris J. Radoux, <span class="NLM_string-name hlFld-ContribAuthor">Andrew R. Leach</span>. </span><span class="cited-content_cbyCitation_article-title">Target-Based Evaluation of “Drug-Like” Properties and Ligand Efficiencies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7210-7230. <a href="https://doi.org/10.1021/acs.jmedchem.1c00416" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00416</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00416%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTarget-Based%252BEvaluation%252Bof%252B%2525E2%252580%25259CDrug-Like%2525E2%252580%25259D%252BProperties%252Band%252BLigand%252BEfficiencies%26aulast%3DLeeson%26aufirst%3DPaul%2BD.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08032021%26date%3D13052021%26volume%3D64%26issue%3D11%26spage%3D7210%26epage%3D7230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christian Comeau, Benjamin Ries, Thomas Stadelmann, Jacob Tremblay, Sylvain Poulet, Ulrike Fröhlich, Jérôme Côté, Pierre-Luc Boudreault, Rabeb Mouna Derbali, Philippe Sarret, Michel Grandbois, Grégoire Leclair, Sereina Riniker, <span class="NLM_string-name hlFld-ContribAuthor">Éric Marsault</span>. </span><span class="cited-content_cbyCitation_article-title">Modulation of the Passive Permeability of Semipeptidic Macrocycles: N- and C-Methylations Fine-Tune Conformation and Properties. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5365-5383. <a href="https://doi.org/10.1021/acs.jmedchem.0c02036" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02036%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DModulation%252Bof%252Bthe%252BPassive%252BPermeability%252Bof%252BSemipeptidic%252BMacrocycles%25253A%252BN-%252Band%252BC-Methylations%252BFine-Tune%252BConformation%252Band%252BProperties%26aulast%3DComeau%26aufirst%3DChristian%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D23112020%26date%3D22032021%26volume%3D64%26issue%3D9%26spage%3D5365%26epage%3D5383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shinya Kobori, Sungjoon Huh, Solomon D. Appavoo, <span class="NLM_string-name hlFld-ContribAuthor">Andrei K. Yudin</span>. </span><span class="cited-content_cbyCitation_article-title">Two-Dimensional Barriers for Probing Conformational Shifts in Macrocycles. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2021,</strong> <em>143 </em>
                                    (13)
                                     , 5166-5171. <a href="https://doi.org/10.1021/jacs.1c01248" title="DOI URL">https://doi.org/10.1021/jacs.1c01248</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.1c01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.1c01248%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DTwo-Dimensional%252BBarriers%252Bfor%252BProbing%252BConformational%252BShifts%252Bin%252BMacrocycles%26aulast%3DKobori%26aufirst%3DShinya%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01022021%26date%3D23032021%26volume%3D143%26issue%3D13%26spage%3D5166%26epage%3D5171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Neal J. Zondlo</span>. </span><span class="cited-content_cbyCitation_article-title">SAR by 1D NMR. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9415-9417. <a href="https://doi.org/10.1021/acs.jmedchem.9b01688" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01688</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01688%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSAR%252Bby%252B1D%252BNMR%26aulast%3DZondlo%26aufirst%3DNeal%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D11102019%26date%3D30102019%26volume%3D62%26issue%3D21%26spage%3D9415%26epage%3D9417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Piotr  Raubo</span>, <span class="hlFld-ContribAuthor ">Rodrigo J.  Carbajo</span>, <span class="hlFld-ContribAuthor ">William  McCoull</span>, <span class="hlFld-ContribAuthor ">Joanna  Raubo</span>, <span class="hlFld-ContribAuthor ">Morgan  Thomas</span>. </span><span class="cited-content_cbyCitation_article-title">Diversity-orientated synthesis of macrocyclic heterocycles using a double S
              N
              Ar approach. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>19 </em>
                                    (28)
                                     , 6274-6290. <a href="https://doi.org/10.1039/D1OB00612F" title="DOI URL">https://doi.org/10.1039/D1OB00612F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1OB00612F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1OB00612F%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DDiversity-orientated%252Bsynthesis%252Bof%252Bmacrocyclic%252Bheterocycles%252Busing%252Ba%252Bdouble%252BS%252BN%252BAr%252Bapproach%26aulast%3DRaubo%26aufirst%3DPiotr%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D28%26spage%3D6274%26epage%3D6290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anikó  Udvarhelyi</span>, <span class="hlFld-ContribAuthor ">Stephane  Rodde</span>, <span class="hlFld-ContribAuthor ">Rainer  Wilcken</span>. </span><span class="cited-content_cbyCitation_article-title">ReSCoSS: a flexible quantum chemistry workflow identifying relevant solution conformers of drug-like molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2021,</strong> <em>35 </em>
                                    (4)
                                     , 399-415. <a href="https://doi.org/10.1007/s10822-020-00337-7" title="DOI URL">https://doi.org/10.1007/s10822-020-00337-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00337-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00337-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DReSCoSS%25253A%252Ba%252Bflexible%252Bquantum%252Bchemistry%252Bworkflow%252Bidentifying%252Brelevant%252Bsolution%252Bconformers%252Bof%252Bdrug-like%252Bmolecules%26aulast%3DUdvarhelyi%26aufirst%3DAnik%25C3%25B3%26date%3D2021%26date%3D2020%26volume%3D35%26issue%3D4%26spage%3D399%26epage%3D415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nipanshu  Agarwal</span>, <span class="hlFld-ContribAuthor ">Vivek  Chetry</span>, <span class="hlFld-ContribAuthor ">Krishna Mohan  Poluri</span>. </span><span class="cited-content_cbyCitation_article-title">Elucidating Protein-Ligand Interactions Using High Throughput Biophysical Techniques. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 205-244. <a href="https://doi.org/10.1007/978-981-15-8936-2_9" title="DOI URL">https://doi.org/10.1007/978-981-15-8936-2_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-15-8936-2_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-15-8936-2_9%26sid%3Dliteratum%253Aachs%26atitle%3DElucidating%252BProtein-Ligand%252BInteractions%252BUsing%252BHigh%252BThroughput%252BBiophysical%252BTechniques%26aulast%3DAgarwal%26aufirst%3DNipanshu%26date%3D2021%26date%3D2021%26spage%3D205%26epage%3D244%26pub%3DSpringer%2520Singapore%26atitle%3DInnovations%252Band%252BImplementations%252Bof%252BComputer%252BAided%252BDrug%252BDiscovery%252BStrategies%252Bin%252BRational%252BDrug%252BDesign%26aulast%3DSingh%26aufirst%3DSanjeev%2BKumar%26date%3D2021%26volume%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amber Y. S.  Balazs</span>, <span class="hlFld-ContribAuthor ">Nichola L.  Davies</span>, <span class="hlFld-ContribAuthor ">David  Longmire</span>, <span class="hlFld-ContribAuthor ">Martin J.  Packer</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Chiarparin</span>. </span><span class="cited-content_cbyCitation_article-title">Nuclear magnetic resonance free ligand conformations and atomic resolution dynamics. </span><span class="cited-content_cbyCitation_journal-name">Magnetic Resonance</span><span> <strong>2021,</strong> <em>2 </em>
                                    (1)
                                     , 489-498. <a href="https://doi.org/10.5194/mr-2-489-2021" title="DOI URL">https://doi.org/10.5194/mr-2-489-2021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5194/mr-2-489-2021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5194%2Fmr-2-489-2021%26sid%3Dliteratum%253Aachs%26jtitle%3DMagnetic%2520Resonance%26atitle%3DNuclear%252Bmagnetic%252Bresonance%252Bfree%252Bligand%252Bconformations%252Band%252Batomic%252Bresolution%252Bdynamics%26aulast%3DBalazs%26aufirst%3DAmber%2BY.%2BS.%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D1%26spage%3D489%26epage%3D498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Reto  Horst</span>, <span class="hlFld-ContribAuthor ">Kathleen A.  Farley</span>, <span class="hlFld-ContribAuthor ">Bethany L.  Kormos</span>, <span class="hlFld-ContribAuthor ">Jane M.  Withka</span>. </span><span class="cited-content_cbyCitation_article-title">NMR spectroscopy: the swiss army knife of drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular NMR</span><span> <strong>2020,</strong> <em>74 </em>
                                    (10-11)
                                     , 509-519. <a href="https://doi.org/10.1007/s10858-020-00330-0" title="DOI URL">https://doi.org/10.1007/s10858-020-00330-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10858-020-00330-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10858-020-00330-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520NMR%26atitle%3DNMR%252Bspectroscopy%25253A%252Bthe%252Bswiss%252Barmy%252Bknife%252Bof%252Bdrug%252Bdiscovery%26aulast%3DHorst%26aufirst%3DReto%26date%3D2020%26date%3D2020%26volume%3D74%26issue%3D10-11%26spage%3D509%26epage%3D519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abhinav  Dubey</span>, <span class="hlFld-ContribAuthor ">Koh  Takeuchi</span>, <span class="hlFld-ContribAuthor ">Mikhail  Reibarkh</span>, <span class="hlFld-ContribAuthor ">Haribabu  Arthanari</span>. </span><span class="cited-content_cbyCitation_article-title">The role of NMR in leveraging dynamics and entropy in drug design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular NMR</span><span> <strong>2020,</strong> <em>74 </em>
                                    (10-11)
                                     , 479-498. <a href="https://doi.org/10.1007/s10858-020-00335-9" title="DOI URL">https://doi.org/10.1007/s10858-020-00335-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10858-020-00335-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10858-020-00335-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520NMR%26atitle%3DThe%252Brole%252Bof%252BNMR%252Bin%252Bleveraging%252Bdynamics%252Band%252Bentropy%252Bin%252Bdrug%252Bdesign%26aulast%3DDubey%26aufirst%3DAbhinav%26date%3D2020%26date%3D2020%26volume%3D74%26issue%3D10-11%26spage%3D479%26epage%3D498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Neil P.  Grimster</span>, <span class="hlFld-ContribAuthor ">Lakshmaiah  Gingipalli</span>, <span class="hlFld-ContribAuthor ">Bernard  Barlaam</span>, <span class="hlFld-ContribAuthor ">Qibin  Su</span>, <span class="hlFld-ContribAuthor ">XiaoLan  Zheng</span>, <span class="hlFld-ContribAuthor ">David  Watson</span>, <span class="hlFld-ContribAuthor ">Haixia  Wang</span>, <span class="hlFld-ContribAuthor ">Iain  Simpson</span>, <span class="hlFld-ContribAuthor ">Andy  Pike</span>, <span class="hlFld-ContribAuthor ">Amber  Balazs</span>, <span class="hlFld-ContribAuthor ">Scott  Boiko</span>, <span class="hlFld-ContribAuthor ">Timothy P.  Ikeda</span>, <span class="hlFld-ContribAuthor ">Anna C.  Impastato</span>, <span class="hlFld-ContribAuthor ">Natalie H.  Jones</span>, <span class="hlFld-ContribAuthor ">Sameer  Kawatkar</span>, <span class="hlFld-ContribAuthor ">Paul  Kemmitt</span>, <span class="hlFld-ContribAuthor ">Scott  Lamont</span>, <span class="hlFld-ContribAuthor ">Joe  Patel</span>, <span class="hlFld-ContribAuthor ">Jon  Read</span>, <span class="hlFld-ContribAuthor ">Ujjal  Sarkar</span>, <span class="hlFld-ContribAuthor ">Li  Sha</span>, <span class="hlFld-ContribAuthor ">Ronald C.  Tomlinson</span>, <span class="hlFld-ContribAuthor ">Haiyun  Wang</span>, <span class="hlFld-ContribAuthor ">David M.  Wilson</span>, <span class="hlFld-ContribAuthor ">Troy E.  Zehnder</span>, <span class="hlFld-ContribAuthor ">Lianghe  Wang</span>, <span class="hlFld-ContribAuthor ">Peng  Wang</span>, <span class="hlFld-ContribAuthor ">Frederick W.  Goldberg</span>, <span class="hlFld-ContribAuthor ">Wenlin  Shao</span>, <span class="hlFld-ContribAuthor ">Stephen  Fawell</span>, <span class="hlFld-ContribAuthor ">Hannah  Dry</span>, <span class="hlFld-ContribAuthor ">James E.  Dowling</span>, <span class="hlFld-ContribAuthor ">Scott D.  Edmondson</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of a series of potent, selective and orally bioavailable SYK inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (19)
                                     , 127433. <a href="https://doi.org/10.1016/j.bmcl.2020.127433" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127433</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127433%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DOptimization%252Bof%252Ba%252Bseries%252Bof%252Bpotent%25252C%252Bselective%252Band%252Borally%252Bbioavailable%252BSYK%252Binhibitors%26aulast%3DGrimster%26aufirst%3DNeil%2BP.%26date%3D2020%26volume%3D30%26issue%3D19%26spage%3D127433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Chun-Nan  Kuo</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Anaplastic lymphoma kinase inhibitors: an updated patent review (2014–2018). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2020,</strong> <em>30 </em>
                                    (5)
                                     , 351-373. <a href="https://doi.org/10.1080/13543776.2020.1738389" title="DOI URL">https://doi.org/10.1080/13543776.2020.1738389</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2020.1738389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2020.1738389%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DAnaplastic%252Blymphoma%252Bkinase%252Binhibitors%25253A%252Ban%252Bupdated%252Bpatent%252Breview%252B%2525282014%2525E2%252580%2525932018%252529%26aulast%3DLiu%26aufirst%3DYi-Min%26date%3D2020%26date%3D2020%26volume%3D30%26issue%3D5%26spage%3D351%26epage%3D373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah J.  Martin</span>, <span class="hlFld-ContribAuthor ">I-Jen  Chen</span>, <span class="hlFld-ContribAuthor ">A.W. Edith  Chan</span>, <span class="hlFld-ContribAuthor ">Nicolas  Foloppe</span>. </span><span class="cited-content_cbyCitation_article-title">Modelling the binding mode of macrocycles: Docking and conformational sampling. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (1)
                                     , 115143. <a href="https://doi.org/10.1016/j.bmc.2019.115143" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.115143</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.115143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.115143%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DModelling%252Bthe%252Bbinding%252Bmode%252Bof%252Bmacrocycles%25253A%252BDocking%252Band%252Bconformational%252Bsampling%26aulast%3DMartin%26aufirst%3DSarah%2BJ.%26date%3D2020%26volume%3D28%26issue%3D1%26spage%3D115143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Free ligand 1D NMR conformational signatures can often be deduced from routine NMR spectra (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). (a) 2-Methylpiperidine demonstrates conformational preorganization of the chair conformation stabilized on the millisecond time scale. This manifests as distinct chemical shifts and <i>J</i>-couplings for each separate methylene hydrogen atom. (b) Piperidine demonstrates conformational flexibility. The methylene hydrogen atom chemical shifts are the same due to dynamic averaging of chemical environments, such as rapid exchange between axial and equatorial orientations. These conformational signatures are conserved across solvents and protonation states (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Conformational analysis platform applied to AZD5991 illustrates the basic tools used to extract conformational and bioactivity reporter signals from experimental free ligand data. (a) A chemical shift of 5.8 ppm is predicted for the pyrazole CH proton (highlighted in green), based on the 2D structure. (b) The pyrazole CH proton appears experimentally at 4.8 ppm, highlighted in green on the 1D NMR spectrum (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). Assignment of three of the methylene groups (x, y, z) is explicitly given to illustrate the signal complexity consistent with conformational preorganization (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S1</a> gives a complete assignment). (c) Conformation-dependent QM calculations give 4.1 ppm for the global energy minimum of the nonbioactive conformation (labeled and colored red) and 4.8 ppm for both the free ligand solution NMR conformation (labeled “bioactive” and colored green) and the bound ligand conformation from the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS0">6FS0</a>). The dotted line shown on the bioactive conformations emphasizes the conformational origin of the pyrazole CH nuclear shielding arising from ring current effects.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Free ligand 1D NMR conformational signatures of <b>2</b> and <b>3</b>, with selected aliphatic proton signals as markers of conformational preorganization labeled, for two macrocycles (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). The 3D bioactive free ligand conformations, derived from X-ray crystal structures of ligand–protein complexes, are shown in green. The macrocyclic portion is emphasized with dark green. (a) <b>2</b> also exhibits a bioactive conformational reporter signal, with the NMR signal highlighted in green at 6.4 ppm, derived by comparing the 2D predicted shift (7.6 ppm) with QM calculations from the bioactive X-ray conformation (6.5 ppm; <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). (b) The NMR signature is calculated for the bioactive conformation of <b>3</b> and is interpreted across the proton spectrum without a specific reporter atom signal (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S3</a>). Additional spectral support of preorganization is seen in the sharp spectral line widths of the CH signals, x* is highlighted as an example.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. 2D structures, free ligand experimental 3D solution conformations, bound ligand conformations, and the free ligand 1D NMR signatures (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz) of <b>4</b> and <b>2</b>. (a) The 2D structure shows the conformational reporter hydrogen atom highlighted in red. The free ligand in solution is flexible, shown as a superposition of low energy conformers (gray) against the U-shaped bioactive conformation (green) derived from the ligand docked into a crystal structure of the monobromide analog (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KU9">5KU9</a>). (b) Macrocyclization improved potency. Highlighted in green, the conformational reporter hydrogen atom experimental chemical shift matches the QM calculated value based on the bioactive conformation at 6.4 and 6.5 ppm, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). The bound ligand conformation is shown docked into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MEV">5MEV</a> with the macrocycle emphasized in dark green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. 1D NMR free ligand conformational signatures, 3D solution conformations, and protein bound ligand conformations of two key acyclic molecules leading to AZD5991 (<b>1</b>). (a) Conformational flexibility of the free ligand <b>5</b> in solution (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 300 MHz), shown as a superposition of low energy conformers (gray) superposed with the bioactive conformation (green) modeled by docking <b>5</b> into 6FS1 (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (b) <b>6</b> has the same rigid indole core with varying degrees of aliphatic conformational flexibility (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 500 MHz). The ensemble of conformers (gray) is shown superposed with the closest minimized conformation of the bound ligand (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS1">6FS1</a>, 3.6 Å interatomic distance between the naphthyl 3-carbon atom and the pyrazole 5-methyl group). QM chemical shift calculations indicate the bioactive conformation is not highly populated by the free ligand in solution. The conformational reporter signal, highlighted in red, appears by NMR at 5.6 ppm highly deshielded compared to the QM calculated value of 4.3 ppm for the bioactive conformation (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Figure S4</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. 1D NMR free ligand conformational signatures and 3D ligand conformations (free and bound) for two BCL6 probe molecules (<b>7</b> and <b>3</b>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> (a) The acyclic molecule shows conformational flexibility (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 30 °C, 400 MHz), shown as a superposition of low energy conformers (gray) superposed with the bioactive conformation (green) modeled by docking <b>7</b> into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21">5N21</a>, a crystal structure of a protein bound structural analog, with a carboxylic acid substituted pyrrolidine in lieu of the hydroxyl substitution (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (b) 1D NMR (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz) of the macrocycle reveals free ligand preorganization and correlates with improved potency (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. 1D NMR conformational signatures of two structurally similar but conformationally different macrocycles, <b>2</b> and <b>8</b> (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). Both compounds have the same 2D predicted value of 7.6 ppm for the hydrogen reporter atom (highlights in 2D, asterisks by 3D), identified by QM calculations as a bioactivity reporter signal (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S2</a>). (a) The bioactive ligand conformation of the methylated amide shown here is derived from docking to PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MEV">5MEV</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>), a protein crystal structure where the ligand has a methyl sulfone in place of the chlorine atom of the monochlorophenyl moiety. The <i>E</i> configuration adopted around the amide bond is also observed in the free ligand in solution, as determined by proton NOEs (see Supporting Information of Johannes et al.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>). (b) In contrast, the matched pair NH amide, also conformationally rigid, adopts a nonbioactive <i>trans</i> or <i>Z</i> configuration, corresponding with a global change in conformation of the solution macrocycle (no crystal structure available; conformational analysis determined by experimental NOEs, spectra published previously in Supporting Information NMR<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Macrocycle matched pair Mcl-1 inhibitors, with NMR bioactivity reporter signals marked by asterisks color coded to correspond to bioactive (green) or nonbioactive (red) conformations (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz). Deconvoluting potency contributions arising from free ligand conformations versus ligand–protein complementarity enhance structure–activity analyses. (a) 2D structure with a bioactivity reporter hydrogen atom marked explicitly and the 3D bioactive conformation with the same hydrogen atom shown in white for compound <b>2</b> (R1 = R2 = H). (b) R1 = (<i>R</i>)-CH<sub>3</sub> and R2 = H, bioactive conformation. (c) R1 = (<i>S</i>)-CH<sub>3</sub> and R2 = H, bioactive conformation. (d) Epimer 1, R1 = H and R2 = CH<sub>3</sub>, nonbioactive conformation. (e) Epimer 2, R1 = H and R2 = CH<sub>3</sub>, bioactive conformation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. A matched pair of BCL6 inhibitors (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 500 MHz). (a) The free ligand <b>13</b> shows conformationally flexibility. (b) Linker methylation rigidifies the free ligand <b>14</b> which adopts the bioactive conformation in solution.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Structure–kinetic relationships for Mcl-1 inhibitors that preorganize into the bioactive conformation as free ligands in solution. (a) <b>15</b> (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 400 MHz). The bioactive conformation was determined from docking into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MES">5MES</a> (which has a chlorine atom in place of the methyl sulfone; see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (b) <b>1</b> (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 27 °C, 600 MHz), adopts the bioactive conformation seen in PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS0">6FS0</a>. The two molecules (<b>1</b> and <b>15</b>) have similar on-rate kinetic constants, as expected from ligand preorganization into the bioactive state, so the 100-fold difference in affinity is reflected in the 100-fold difference in off-rate kinetics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Structure–kinetics relationships for BCL6 inhibitors. (a) A flexible acyclic inhibitor (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 30 °C, 400 MHz) with suboptimal on-rate kinetics. Protein–ligand complex crystal structure is PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21">5N21</a>. (b) A cyclized analog (<sup>1</sup>H, DMSO-<i>d</i><sub>6</sub>, 30 °C, 400 MHz) with 50-fold higher affinity significantly assisted by preorganization into the bioactive conformation, <b>17</b> is shown docked into PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21">5N21</a> (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf" class="ext-link">Supporting Information Table S4</a>). (c) A further 3.3-fold improvement in affinity is dominated by off-rate kinetics by increasing ligand–protein interactions. Protein–ligand complex crystal structure is PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N1Z">5N1Z</a>. An increase in protein complementarity arises from the engagement of ligand to protein Arg-28 polar interactions, leading to increased residence time.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. BCL6 FRET affinity as a function of conformation ensemble RMSD. (a) Relationship between IC<sub>50</sub> and calculated conformation population for a set of compounds with an identical binding pharmacophore, where the line is a linear fit, <i>R</i><sup>2</sup> = 0.49. (b) Depiction of RMSD distribution to bioactive conformation for compound <b>16</b>, showing a small population at low RMSD. (c) Depiction of RMSD distribution to bioactive conformation for compound <b>17</b>, showing a higher population at low RMSD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) K<sub>2</sub>CO<sub>3</sub>, acetone, 50 °C, 99%; (b) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 99%; (c) MeI, NaH, DMF, rt, 38%; (d) H<sub>2</sub>, Pd/C, MeOH, rt, 81%; (e) <b>21</b>, DIPEA, THF, rt; (f) HCl, EtOH, rt, 92%; (g) DIPEA, DMA, 130 °C, 27%.</p></p></figure><figure data-id="sch2" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/medium/jm9b00716_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b00716/20191108/images/large/jm9b00716_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00716&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TBSCl, imidazole<sub>cat</sub>, rt, 84%; (b) diethyl azodicarboxylate, Ph<sub>3</sub>P, THF, rt, 68%; (c) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 97%; (d) TBAF, THF, rt, quant; (e) PCC, CH<sub>2</sub>Cl<sub>2</sub>, rt, 60%; (f) MeNH<sub>2</sub>, NaBH<sub>3</sub>CN AcONa, MeOH, 96%; (g) K<sub>2</sub>CO<sub>3</sub>, Boc<sub>2</sub>O, THF, H<sub>2</sub>O, rt, quant; (h) H<sub>2</sub>, Pd/C, MeOH, rt, 81%; (i) <b>21</b>, DIPEA, THF, rt, 66%; (j) HCl, EtOH, rt, 95%; (k) DIPEA, DMA, 125 °C, 23%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63163" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63163" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rask, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Accurate and reliable prediction of the binding affinities of macrocycles to their protein targets</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">6290</span>– <span class="NLM_lpage">6300</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00885</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00885" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=6290-6300&author=H.+S.+Yuauthor=Y.+Dengauthor=Y.+Wuauthor=D.+Sindhikaraauthor=A.+R.+Raskauthor=T.+Kimuraauthor=R.+Abelauthor=L.+Wang&title=Accurate+and+reliable+prediction+of+the+binding+affinities+of+macrocycles+to+their+protein+targets&doi=10.1021%2Facs.jctc.7b00885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of the Binding Affinities of Macrocycles to Their Protein Targets</span></div><div class="casAuthors">Yu, Haoyu S.; Deng, Yuqing; Wu, Yujie; Sindhikara, Dan; Rask, Amy R.; Kimura, Takayuki; Abel, Robert; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6290-6300</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocycles have been emerging as a very important drug class in the past few decades largely due to their expanded chem. diversity benefiting from advances in synthetic methods.  Macrocyclization has been recognized as an effective way to restrict the conformational space of acyclic small mol. inhibitors with the hope of improving potency, selectivity, and metabolic stability.  Because of their relatively larger size as compared to typical small mol. drugs and the complexity of the structures, efficient sampling of the accessible macrocycle conformational space and accurate prediction of their binding affinities to their target protein receptors poses a great challenge of central importance in computational macrocycle drug design.  In this article, the authors present a novel method for relative binding free energy calcns. between macrocycles with different ring sizes and between the macrocycles and their corresponding acyclic counterparts.  The authors have applied the method to seven pharmaceutically interesting data sets taken from recent drug discovery projects including 33 macrocyclic ligands covering a diverse chem. space.  The predicted binding free energies are in good agreement with exptl. data with an overall root-mean-square error (RMSE) of 0.94 kcal/mol.  This is to the authors' knowledge the first time where the free energy of the macrocyclization of linear mols. has been directly calcd. with rigorous physics-based free energy calcn. methods, and the authors anticipate the outstanding accuracy demonstrated here across a broad range of target classes may have significant implications for macrocycle drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibClnk_AFJrVg90H21EOLACvtfcHk0ljBHDDlhw1J5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP&md5=62d194fa339aaadd9c46611730dee5ef</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00885%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%2BS.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DRask%26aufirst%3DA.%2BR.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DAccurate%2520and%2520reliable%2520prediction%2520of%2520the%2520binding%2520affinities%2520of%2520macrocycles%2520to%2520their%2520protein%2520targets%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D6290%26epage%3D6300%26doi%3D10.1021%2Facs.jctc.7b00885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Cell permeability beyond the rule of 5</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.03.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.addr.2016.03.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=27067608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslSgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=42-61&author=P.+Matssonauthor=B.+C.+Doakauthor=B.+Overauthor=J.+Kihlberg&title=Cell+permeability+beyond+the+rule+of+5&doi=10.1016%2Fj.addr.2016.03.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cell permeability beyond the rule of 5</span></div><div class="casAuthors">Matsson, Paer; Doak, Bradley C.; Over, Bjoern; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-61</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug discovery for difficult targets that have large and flat binding sites is often better suited to compds. beyond the "rule of 5" (bRo5).  However, such compds. carry higher pharmacokinetic risks, such as low soly. and permeability, and increased efflux and metab.  Interestingly, recent drug approvals and studies suggest that cell permeable and orally bioavailable drugs can be discovered far into bRo5 space.  Tactics such as redn. or shielding of polarity by N-methylation, bulky side chains and intramol. hydrogen bonds may be used to increase cell permeability in this space, but often results in decreased soly.  Conformationally flexible compds. can, however, combine high permeability and soly., properties that are keys for cell permeability and intestinal absorption.  Recent developments in computational conformational anal. will aid design of such compds. and hence prediction of cell permeability.  Transporter mediated efflux occurs for most investigated drugs in bRo5 space, however it is commonly overcome by high local intestinal concns. on oral administration.  In contrast, there is little data to support significant impact of transporter-mediated intestinal absorption in bRo5 space.  Current knowledge of compd. properties that govern transporter effects of bRo5 drugs is limited and requires further fundamental and comprehensive studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3dS_FpMvvCbVg90H21EOLACvtfcHk0ljBHDDlhw1J5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslSgsb0%253D&md5=1eacb5ab6e50159f3679e3d8a1f719c4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.03.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.03.013%26sid%3Dliteratum%253Aachs%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DCell%2520permeability%2520beyond%2520the%2520rule%2520of%25205%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2016%26volume%3D101%26spage%3D42%26epage%3D61%26doi%3D10.1016%2Fj.addr.2016.03.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span>; <span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-based macrocycle design in small-molecule drug discovery and simple metrics to identify opportunities for macrocyclization of small-molecule ligands</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> . Published Online: March 22, 2019.</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=M.+D.+Cummings&author=S.+Sekharan&title=Structure-based+macrocycle+design+in+small-molecule+drug+discovery+and+simple+metrics+to+identify+opportunities+for+macrocyclization+of+small-molecule+ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26atitle%3DStructure-based%2520macrocycle%2520design%2520in%2520small-molecule%2520drug%2520discovery%2520and%2520simple%2520metrics%2520to%2520identify%2520opportunities%2520for%2520macrocyclization%2520of%2520small-molecule%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rossi Sebastiano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backlund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Impact of dynamically exposed polarity on permeability and solubility of chameleonic drugs beyond the rule of 5</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4189</span>– <span class="NLM_lpage">4202</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslGqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4189-4202&author=M.+Rossi+Sebastianoauthor=B.+C.+Doakauthor=M.+Backlundauthor=V.+Poongavanamauthor=B.+Overauthor=G.+Ermondiauthor=G.+Caronauthor=P.+Matssonauthor=J.+Kihlberg&title=Impact+of+dynamically+exposed+polarity+on+permeability+and+solubility+of+chameleonic+drugs+beyond+the+rule+of+5&doi=10.1021%2Facs.jmedchem.8b00347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Dynamically Exposed Polarity on Permeability and Solubility of Chameleonic Drugs Beyond the Rule of 5</span></div><div class="casAuthors">Rossi Sebastiano, Matteo; Doak, Bradley C.; Backlund, Maria; Poongavanam, Vasanthanathan; Over, Bjoern; Ermondi, Giuseppe; Caron, Giulia; Matsson, Paer; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4189-4202</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility has been proposed to significantly affect drug properties outside rule-of-5 (Ro5) chem. space.  Here, we investigated the influence of dynamically exposed polarity on cell permeability and aq. soly. for a structurally diverse set of drugs and clin. candidates far beyond the Ro5, all of which populated multiple distinct conformations as revealed by X-ray crystallog.  Efflux-inhibited (passive) Caco-2 cell permeability correlated strongly with the compds.' min. solvent-accessible 3D polar surface areas (PSA), whereas aq. soly. depended less on the specific 3D conformation.  Inspection of the crystal structures highlighted flexibly linked arom. side chains and dynamically forming intramol. hydrogen bonds as particularly effective in providing "chameleonic" properties that allow compds. to display both high cell permeability and aq. soly.  These structural features, in combination with permeability predictions based on the correlation to solvent-accessible 3D PSA, should inspire drug design in the challenging chem. space far beyond the Ro5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHNeutc0SurrVg90H21EOLACvtfcHk0lhD5eVHugxVgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslGqtL4%253D&md5=81a3400d434cdc22d1639513e08abeac</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00347%26sid%3Dliteratum%253Aachs%26aulast%3DRossi%2BSebastiano%26aufirst%3DM.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DBacklund%26aufirst%3DM.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DImpact%2520of%2520dynamically%2520exposed%2520polarity%2520on%2520permeability%2520and%2520solubility%2520of%2520chameleonic%2520drugs%2520beyond%2520the%2520rule%2520of%25205%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4189%26epage%3D4202%26doi%3D10.1021%2Facs.jmedchem.8b00347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Macrocycles: Lessons from the distant past, recent developments, and future directions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1039/C4SC03089C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1039%2FC4SC03089C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28553456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=30-49&author=A.+K.+Yudin&title=Macrocycles%3A+Lessons+from+the+distant+past%2C+recent+developments%2C+and+future+directions&doi=10.1039%2FC4SC03089C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles: lessons from the distant past, recent developments, and future directions</span></div><div class="casAuthors">Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-49</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A noticeable increase in mol. complexity of drug targets has created an unmet need in the therapeutic agents that are larger than traditional small mols.  Macrocycles, which are cyclic compds. comprising 12 atoms or more, are now recognized as mols. that "are up to the task" to interrogate extended protein interfaces.  However, because macrocycles (particularly the ones based on peptides) are equipped with large polar surface areas, achieving cellular permeability and bioavailability is anything but straightforward.  While one might consider this to be the Achilles' heel of this class of compds., the synthetic community continues to develop creative approaches toward the synthesis of macrocycles and their site-selective modification.  This perspective provides an overview of both mechanistic and structural issues that bear on macrocycles as a unique class of mols.  The reader is offered a historical foray into some of the classic studies that have resulted in the current renaissance of macrocycles.  In addn., an attempt is made to overview the more recent developments that give hope that macrocycles might indeed turn into a useful therapeutic modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCRWplVAQ417Vg90H21EOLACvtfcHk0lhD5eVHugxVgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE&md5=48262b4650417f04063c0180b652a886</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1039%2FC4SC03089C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC03089C%26sid%3Dliteratum%253Aachs%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DMacrocycles%253A%2520Lessons%2520from%2520the%2520distant%2520past%252C%2520recent%2520developments%252C%2520and%2520future%2520directions%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D30%26epage%3D49%26doi%3D10.1039%2FC4SC03089C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alihodžić, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukvić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elenkov, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutinec, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koštrun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pešić, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saxty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašković, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žiher, D.</span></span> <span> </span><span class="NLM_article-title">Current trends in macrocyclic drug discovery and beyond-Ro5</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1016/bs.pmch.2018.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fbs.pmch.2018.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=29680148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=113-233&author=S.+Alihod%C5%BEi%C4%87author=M.+Bukvi%C4%87author=I.+J.+Elenkovauthor=A.+Hutinecauthor=S.+Ko%C5%A1trunauthor=D.+Pe%C5%A1i%C4%87author=G.+Saxtyauthor=L.+Toma%C5%A1kovi%C4%87author=D.+%C5%BDiher&title=Current+trends+in+macrocyclic+drug+discovery+and+beyond-Ro5&doi=10.1016%2Fbs.pmch.2018.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Current Trends in Macrocyclic Drug Discovery and beyond-Ro5</span></div><div class="casAuthors">Alihodzic Sulejman; Bukvic Mirjana; Elenkov Ivaylo J; Hutinec Antun; Kostrun Sanja; Pesic Dijana; Saxty Gordon; Tomaskovic Linda; Ziher Dinko</div><div class="citationInfo"><span class="NLM_cas:title">Progress in medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-233</span>
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    </div><div class="casAbstract">This chapter will discuss the recent literature of macrocycles and drug-like property space moving beyond the rule of five (bRo5).  Trends in chemical classes that fall within this definition are discussed and the impact of the latest technologies in the field assessed.  The physicochemical properties, which have provided both successes and challenges, especially in scale-up, are discussed.  A recent patent literature is reviewed and the chapter concludes with a perspective on the future of macrocyclic drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSZ9pfcWWVZ5hWfpMXQLRstfW6udTcc2eYcrJYboGq1Vbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjltFOgsA%253D%253D&md5=ac41d1a69cb0056d106d554b0b323e61</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fbs.pmch.2018.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.pmch.2018.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DAlihod%25C5%25BEi%25C4%2587%26aufirst%3DS.%26aulast%3DBukvi%25C4%2587%26aufirst%3DM.%26aulast%3DElenkov%26aufirst%3DI.%2BJ.%26aulast%3DHutinec%26aufirst%3DA.%26aulast%3DKo%25C5%25A1trun%26aufirst%3DS.%26aulast%3DPe%25C5%25A1i%25C4%2587%26aufirst%3DD.%26aulast%3DSaxty%26aufirst%3DG.%26aulast%3DToma%25C5%25A1kovi%25C4%2587%26aufirst%3DL.%26aulast%3D%25C5%25BDiher%26aufirst%3DD.%26atitle%3DCurrent%2520trends%2520in%2520macrocyclic%2520drug%2520discovery%2520and%2520beyond-Ro5%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D57%26spage%3D113%26epage%3D233%26doi%3D10.1016%2Fbs.pmch.2018.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finch, H.</span></span> <span> </span><span class="NLM_article-title">The conformational musings of a medicinal chemist</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.drudis.2013.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=24184532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVShu73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=320-325&author=H.+Finch&title=The+conformational+musings+of+a+medicinal+chemist&doi=10.1016%2Fj.drudis.2013.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The conformational musings of a medicinal chemist</span></div><div class="casAuthors">Finch, Harry</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">320-325</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design strategies based on X-ray crystallog. data of ligands bound to biol. targets or computationally derived pharmacophore models have been introduced over the past 25 years or so.  These have now matured and are deeply embedded in the drug discovery process in most pharmaceutical and biotechnol. companies where they continue to play a major part in the discovery of new medicines and drug candidates.  Newly developed NMR methods can now provide a full description of the conformations in which ligands exist in free soln., crucially allowing those that are dominant to be identified.  Integrating exptl. detd. conformational information on active and inactive mols. in drug discovery programs, alongside the existing techniques, should have a major impact on the success of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDMOte9AlxMLVg90H21EOLACvtfcHk0lgpU6-7Y33xgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVShu73J&md5=808beba5779675737de9ddaff35e8731</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DFinch%26aufirst%3DH.%26atitle%3DThe%2520conformational%2520musings%2520of%2520a%2520medicinal%2520chemist%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D320%26epage%3D325%26doi%3D10.1016%2Fj.drudis.2013.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charifson, P. S.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of drug-like molecules bound to proteins: An extensive study of ligand reorganization upon binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2499</span>– <span class="NLM_lpage">2510</span>, <span class="refDoi"> DOI: 10.1021/jm030563w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030563w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2499-2510&author=E.+Perolaauthor=P.+S.+Charifson&title=Conformational+analysis+of+drug-like+molecules+bound+to+proteins%3A+An+extensive+study+of+ligand+reorganization+upon+binding&doi=10.1021%2Fjm030563w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Drug-Like Molecules Bound to Proteins: An Extensive Study of Ligand Reorganization upon Binding</span></div><div class="casAuthors">Perola, Emanuele; Charifson, Paul S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2499-2510</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes a large-scale study on the nature and the energetics of the conformational changes drug-like mols. experience upon binding.  Ligand strain energies and conformational reorganization were analyzed with different computational methods on 150 crystal structures of pharmaceutically relevant protein-ligand complexes.  The common knowledge that ligands rarely bind in their lowest calcd. energy conformation was confirmed.  Addnl., the authors found that over 60% of the ligands do not bind in a local min. conformation.  While approx. 60% of the ligands were calcd. to bind with strain energies lower than 5 kcal/mol, strain energies over 9 kcal/mol were calcd. in at least 10% of the cases regardless of the method used.  A clear correlation was found between acceptable strain energy and ligand flexibility, while there was no correlation between strain energy and binding affinity, thus indicating that expensive conformational rearrangements can be tolerated in some cases without overly penalizing the tightness of binding.  On the basis of the trends obsd., thresholds for the acceptable strain energies of bioactive conformations were defined with consideration of the impact of ligand flexibility.  An anal. of the degree of folding of the bound ligands confirmed the general tendency of small mols. to bind in an extended conformation.  The results suggest that the unfolding of hydrophobic ligands during binding, which exposes hydrophobic surfaces to contact with protein residues, could be one of the factors accounting for high reorganization energies.  Finally, different methods for conformational anal. were evaluated, and guidelines were defined to maximize the prevalence of bioactive conformations in computationally generated ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMrN0dFKglcbVg90H21EOLACvtfcHk0li034OmQtqahg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXivVans78%253D&md5=8f54a8285ef1ffc5e1a19f748f4674bf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm030563w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030563w%26sid%3Dliteratum%253Aachs%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3DConformational%2520analysis%2520of%2520drug-like%2520molecules%2520bound%2520to%2520proteins%253A%2520An%2520extensive%2520study%2520of%2520ligand%2520reorganization%2520upon%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2499%26epage%3D2510%26doi%3D10.1021%2Fjm030563w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M. J.</span></span> <span> </span><span class="NLM_article-title">Measurement, interpretation and use of free ligand solution conformations in drug discovery</span>. <i>Prog. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/bs.pmch.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fbs.pmch.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=26852934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC28nos1KmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=45-147&author=C.+D.+Blundellauthor=T.+Nowakauthor=M.+J.+Watson&title=Measurement%2C+interpretation+and+use+of+free+ligand+solution+conformations+in+drug+discovery&doi=10.1016%2Fbs.pmch.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement, Interpretation and Use of Free Ligand Solution Conformations in Drug Discovery</span></div><div class="casAuthors">Blundell Charles D; Nowak Thorsten; Watson Martin J</div><div class="citationInfo"><span class="NLM_cas:title">Progress in medicinal chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-147</span>
        ISSN:<span class="NLM_cas:issn">0079-6468</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOLsGcWRMxs0BuFdjgVNKSfW6udTcc2eYWXp3sUXzZO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nos1KmsA%253D%253D&md5=35cea492313ecb36fbc0c64ad0245b65</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fbs.pmch.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.pmch.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DC.%2BD.%26aulast%3DNowak%26aufirst%3DT.%26aulast%3DWatson%26aufirst%3DM.%2BJ.%26atitle%3DMeasurement%252C%2520interpretation%2520and%2520use%2520of%2520free%2520ligand%2520solution%2520conformations%2520in%2520drug%2520discovery%26jtitle%3DProg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D55%26spage%3D45%26epage%3D147%26doi%3D10.1016%2Fbs.pmch.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">Experimental free ligand conformations: A missing link in structure-based drug discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.4155%2Ffmc-2018-0339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=30648432" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=79-82&author=E.+Chiarparinauthor=M.+J.+Packerauthor=D.+M.+Wilson&title=Experimental+free+ligand+conformations%3A+A+missing+link+in+structure-based+drug+discovery&doi=10.4155%2Ffmc-2018-0339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental free ligand conformations: a missing link in structure-based drug discovery</span></div><div class="casAuthors">Chiarparin, Elisabetta; Packer, Martin J.; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">79-82</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2BcuFdu3d5LVg90H21EOLACvtfcHk0ljZ7EqDpM4Q4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFejurw%253D&md5=c6dab61511fc3a5e9b55e47a8aead791</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0339%26sid%3Dliteratum%253Aachs%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DExperimental%2520free%2520ligand%2520conformations%253A%2520A%2520missing%2520link%2520in%2520structure-based%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D79%26epage%3D82%26doi%3D10.4155%2Ffmc-2018-0339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span> <span> </span><span class="NLM_article-title">Towards understanding the unbound state of drug compounds: implications for the intramolecular reorganization energy upon binding</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2159</span>– <span class="NLM_lpage">2189</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmc.2016.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=27061672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsF2lsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=2159-2189&author=N.+Foloppeauthor=I.+J.+Chen&title=Towards+understanding+the+unbound+state+of+drug+compounds%3A+implications+for+the+intramolecular+reorganization+energy+upon+binding&doi=10.1016%2Fj.bmc.2016.03.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Towards understanding the unbound state of drug compounds: Implications for the intramolecular reorganization energy upon binding</span></div><div class="casAuthors">Foloppe, Nicolas; Chen, I.-Jen</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2159-2189</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">There has been an explosion of structural information for pharmaceutical compds. bound to biol. targets, but the conformations and dynamics of compds. free in soln. are poorly characterized, if at all.  Yet, knowledge of the unbound state is essential to understand the fundamentals of mol. recognition, including the much debated conformational intramol. reorganization energy of a compd. upon binding (ΔEReorg).  Also, dependable observation of the unbound compds. is important for ligand-based drug discovery, e.g. with pharmacophore modeling.  Here, these questions are addressed with long (≥0.5 μs) state-of-the-art mol. dynamics (MD) simulations of 26 compds. (including 7 approved drugs) unbound in explicit solvent.  These compds. were selected to be chem. diverse, with a range of flexibility, and good quality bioactive x-ray structures.  The MD-simulated free compds. are compared to their bioactive structure and conformers generated with ad hoc sampling in vacuo or with implicit generalized Born (GB) aq. solvation models.  The GB conformational models clearly depart from those obtained in explicit solvent, and suffer from conformational collapse almost as severe as in vacuo.  Thus, the global energy min. in vacuo or with GB are not suitable representations of the unbound state, which can instead be extensively sampled by MD simulations.  Many, but not all, MD-simulated compds. displayed some structural similarity to their bioactive structure, supporting the notion of conformational preorganization for binding.  The ligand-protein complexes were also simulated in explicit solvent, to est. ΔEReorg as an enthalpic difference ΔHReorg between the intramol. energies in the bound and unbound states.  This fresh approach yielded ΔHReorg values ≤ 6 kcal/mol for 18 out of 26 compds.  For three particularly polar compds. 15 ≤ ΔHReorg ≤ 20 kcal/mol, supporting the notion that ΔHReorg can be substantial.  Those large ΔHReorg values correspond to a redistribution of electrostatic interactions upon binding.  Overall, the study illustrates how MD simulations offer a promising avenue to characterize the unbound state of medicinal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlcBqm8bRf_bVg90H21EOLACvtfcHk0ljZ7EqDpM4Q4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsF2lsLc%253D&md5=a4083169d7c616b125ec04aa50da58c1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DI.%2BJ.%26atitle%3DTowards%2520understanding%2520the%2520unbound%2520state%2520of%2520drug%2520compounds%253A%2520implications%2520for%2520the%2520intramolecular%2520reorganization%2520energy%2520upon%2520binding%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D2159%26epage%3D2189%26doi%3D10.1016%2Fj.bmc.2016.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdelyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Conformational sampling of macrocyclic drugs in different environments: can we find the relevant conformations?</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11742</span>– <span class="NLM_lpage">11757</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01379</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=11742-11757&author=V.+Poongavanamauthor=E.+Daneliusauthor=S.+Peintnerauthor=L.+Alcarazauthor=G.+Caronauthor=M.+D.+Cummingsauthor=S.+Wlodekauthor=M.+Erdelyiauthor=P.+C.+D.+Hawkinsauthor=G.+Ermondiauthor=J.+Kihlberg&title=Conformational+sampling+of+macrocyclic+drugs+in+different+environments%3A+can+we+find+the+relevant+conformations%3F&doi=10.1021%2Facsomega.8b01379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Danelius, Emma; Peintner, Stefan; Alcaraz, Lilian; Caron, Giulia; Cummings, Maxwell D.; Wlodek, Stanislaw; Erdelyi, Mate; Hawkins, Paul C. D.; Ermondi, Giuseppe; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11742-11757</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility is a major determinant of the properties of macrocycles and other drugs in beyond rule of 5 (bRo5) space.  Prediction of conformations is essential for design of drugs in this space and the authors have evaluated three tools for conformational sampling of a set of ten bRo5 drugs and clin. candidates in polar and apolar environments.  The distance-geometry based OMEGA was found to yield ensembles spanning larger structure and property spaces than the two MD based methods, MOE-LowMode MD (MOE) and MacroModel-Macrocycle Conformational Sampling (MC).  Both MC and OMEGA, but not MOE, generated different ensembles for polar and apolar environments.  All three conformational search methods generated conformers similar to the crystal structure conformers for nine of the ten compds., with OMEGA performing somewhat better than MOE and MC.  MOE and OMEGA found all six conformers of roxithromycin that were identified by NMR in aq. solns., while only OMEGA sampled the three conformers obsd. in chloroform.  The authors suggest that characterization of conformers using mol. descriptors, e.g. the radius of gyration (Rgyr) and polar surface area (PSA), is preferred to energy- or RMSD-based methods for selection of biol. relevant conformers in drug discovery in bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoetOG-yDeciLVg90H21EOLACvtfcHk0ljZ7EqDpM4Q4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI&md5=a8722551b161e907daf55146f22b6d57</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01379%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DErdelyi%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DConformational%2520sampling%2520of%2520macrocyclic%2520drugs%2520in%2520different%2520environments%253A%2520can%2520we%2520find%2520the%2520relevant%2520conformations%253F%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D11742%26epage%3D11757%26doi%3D10.1021%2Facsomega.8b01379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, M. N.</span></span> <span> </span><span class="NLM_article-title">Overcoming the unfavourable entropic contribution of ligand binding with a macrocyclic inhibitor bound to penicillopepsin</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1007/978-1-4615-5373-1_50</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1007%2F978-1-4615-5373-1_50" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=9561242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADyaK1cXlsFKnsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=1998&pages=355-359&author=M.+E.+Fraserauthor=J.+H.+Meyerauthor=P.+A.+Bartlettauthor=M.+N.+James&title=Overcoming+the+unfavourable+entropic+contribution+of+ligand+binding+with+a+macrocyclic+inhibitor+bound+to+penicillopepsin&doi=10.1007%2F978-1-4615-5373-1_50"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming the unfavorable entropic contribution of ligand binding with a macrocyclic inhibitor bound to penicillopepsin</span></div><div class="casAuthors">Fraser, Marie F.; Meyer, J. Hoyt; Bartlett, Paul A.; James, Michael N. G.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">Aspartic Proteinases</span>),
    <span class="NLM_cas:pages">355-359</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Plenum Publishing Corp.</span>)
        </div><div class="casAbstract">One way to improve the binding of a ligand to its protein receptor is by forcing the unbound ligand to adopt the same conformation as it has when bound to the receptor.  The conformational constraint then reduces the entropic disadvantage of ordered binding to the protein.  The constraint should change the structure of the ligand as little as possible, so as to retain the favorable enthalpic interactions of the ligand with the receptor.  Including a ring structure in the design of a ligand can reduce its conformational flexibility and it may be possible to introduce the ring without changing how the ligand binds to the receptor.  A macrocyclic inhibitor of penicillopepsin, as well as two acyclic analogs, were synthesized.  Availability of the macrocyclic inhibitor and the acyclic analogs make it possible to evaluate the entropic effect of the cyclization, provided that they bind to penicillopepsin in the same manner.  Enzyme inhibition by the various inhibitors was evaluated, and the crystal structure of the macrocyclic inhibitor co-crystd. with penicillopepsin was detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqywUc6QyslELVg90H21EOLACvtfcHk0ljaKCRqt-WNGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlsFKnsrY%253D&md5=7d64b9744103afdb02bb131109050bfe</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2F978-1-4615-5373-1_50&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4615-5373-1_50%26sid%3Dliteratum%253Aachs%26aulast%3DFraser%26aufirst%3DM.%2BE.%26aulast%3DMeyer%26aufirst%3DJ.%2BH.%26aulast%3DBartlett%26aufirst%3DP.%2BA.%26aulast%3DJames%26aufirst%3DM.%2BN.%26atitle%3DOvercoming%2520the%2520unfavourable%2520entropic%2520contribution%2520of%2520ligand%2520binding%2520with%2520a%2520macrocyclic%2520inhibitor%2520bound%2520to%2520penicillopepsin%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1998%26volume%3D436%26spage%3D355%26epage%3D359%26doi%3D10.1007%2F978-1-4615-5373-1_50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, R. J.</span>; <span class="NLM_string-name">Mobli, M.</span></span> <i>Modelling 1H NMR Spectra of Organic Compounds: Theory, Applications and NMR Prediction Software</i>; <span class="NLM_publisher-name">John Wiley & Sons Ltd.</span> (<span class="NLM_publisher-loc">The Atrium, Southern Gate, Chichester, West Sussex, PO19 8SQ, U.K.</span>), <span class="NLM_year">2008</span>; <a href="https://onlinelibrary.wiley.com/doi/book/10.1002/9780470721803" class="extLink">https://onlinelibrary.wiley.com/doi/book/10.1002/9780470721803</a>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1002%2F9780470721803" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=R.+J.+Abraham&author=M.+Mobli&title=Modelling+1H+NMR+Spectra+of+Organic+Compounds%3A+Theory%2C+Applications+and+NMR+Prediction+Software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2F9780470721803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470721803%26sid%3Dliteratum%253Aachs%26aulast%3DAbraham%26aufirst%3DR.%2BJ.%26btitle%3DModelling%25201H%2520NMR%2520Spectra%2520of%2520Organic%2520Compounds%253A%2520Theory%252C%2520Applications%2520and%2520NMR%2520Prediction%2520Software%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%2520Ltd%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweckstetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M.</span></span> <span> </span><span class="NLM_article-title">Exploring free-energy landscapes of intrinsically disordered proteins at atomic resolution using NMR spectroscopy</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">6632</span>– <span class="NLM_lpage">6660</span>, <span class="refDoi"> DOI: 10.1021/cr400688u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400688u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVegtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=6632-6660&author=M.+R.+Jensenauthor=M.+Zweckstetterauthor=J.+R.+Huangauthor=M.+Blackledge&title=Exploring+free-energy+landscapes+of+intrinsically+disordered+proteins+at+atomic+resolution+using+NMR+spectroscopy&doi=10.1021%2Fcr400688u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring Free-Energy Landscapes of Intrinsically Disordered Proteins at Atomic Resolution Using NMR Spectroscopy</span></div><div class="casAuthors">Jensen, Malene Ringkjobing; Zweckstetter, Markus; Huang, Jie-rong; Blackledge, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6632-6660</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the study of free-energy landscapes of intrinsically disordered proteins at at. resoln. using NMR spectroscopy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1wesUYoP29LVg90H21EOLACvtfcHk0ljaKCRqt-WNGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVegtLo%253D&md5=7d4fabed6618ede86586aeb6214eb33a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fcr400688u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400688u%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DZweckstetter%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DJ.%2BR.%26aulast%3DBlackledge%26aufirst%3DM.%26atitle%3DExploring%2520free-energy%2520landscapes%2520of%2520intrinsically%2520disordered%2520proteins%2520at%2520atomic%2520resolution%2520using%2520NMR%2520spectroscopy%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D6632%26epage%3D6660%26doi%3D10.1021%2Fcr400688u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cernak, T.</span></span> <span> </span><span class="NLM_article-title">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">12256</span>– <span class="NLM_lpage">12267</span>, <span class="refDoi"> DOI: 10.1002/anie.201303207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1002%2Fanie.201303207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=12256-12267&author=H.+Sch%C3%B6nherrauthor=T.+Cernak&title=Profound+methyl+effects+in+drug+discovery+and+a+call+for+new+C-H+methylation+reactions&doi=10.1002%2Fanie.201303207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Profound methyl effects in drug discovery and a call for new C-H methylation reactions</span></div><div class="casAuthors">Schonherr Heike; Cernak Tim</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">12256-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The methyl group is one of the most commonly occurring carbon fragments in small-molecule drugs.  This simplest alkyl fragment appears in more than 67 % of the top-selling drugs of 2011 and can modulate both the biological and physical properties of a molecule.  This Review focuses on so-called magic methyl effects on binding potency, where the seemingly mundane change of C H to C Me improves the IC50  value of a drug candidate more than 100-fold.  This discussion is followed by a survey of recent advances in synthetic chemistry that allow the direct methylation of C(sp(2) ) H and C(sp(3) ) H bonds.  It is our hope that the relevance of the meager methyl group to drug discovery as presented herein will inspire reports on new C H methylation reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyoLZDDgvsMh0LsK7uFuY7fW6udTcc2eYlLGCPHzRnVbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c%252FptVahuw%253D%253D&md5=ef08f80e55b2c530b49e4306da024902</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fanie.201303207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201303207%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DH.%26aulast%3DCernak%26aufirst%3DT.%26atitle%3DProfound%2520methyl%2520effects%2520in%2520drug%2520discovery%2520and%2520a%2520call%2520for%2520new%2520C-H%2520methylation%2520reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2013%26volume%3D52%26spage%3D12256%26epage%3D12267%26doi%3D10.1002%2Fanie.201303207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tron, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belmonte, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aquila, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boise, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Embrey, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendricks, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmirski, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulis, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nooka, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuller, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Secrist, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of Mcl-1-specific inhibitor AZD5991 and preclinical activity in multiple myeloma and acute myeloid leukemia</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5341</span>– <span class="NLM_lpage">5355</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07551-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fs41467-018-07551-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=30559424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFersLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=5341-5355&author=A.+E.+Tronauthor=M.+A.+Belmonteauthor=A.+Adamauthor=B.+M.+Aquilaauthor=L.+H.+Boiseauthor=E.+Chiarparinauthor=J.+Cidadoauthor=K.+J.+Embreyauthor=E.+Ganglauthor=F.+D.+Gibbonsauthor=G.+P.+Gregoryauthor=D.+Hargreavesauthor=J.+A.+Hendricksauthor=J.+W.+Johannesauthor=R.+W.+Johnstoneauthor=S.+L.+Kazmirskiauthor=J.+G.+Kettleauthor=M.+L.+Lambauthor=S.+M.+Matulisauthor=A.+K.+Nookaauthor=M.+J.+Packerauthor=B.+Pengauthor=P.+B.+Rawlinsauthor=D.+W.+Robbinsauthor=A.+G.+Schullerauthor=N.+Suauthor=W.+Yangauthor=Q.+Yeauthor=X.+Zhengauthor=J.+P.+Secristauthor=E.+A.+Clarkauthor=D.+M.+Wilsonauthor=S.+E.+Fawellauthor=A.+W.+Hird&title=Discovery+of+Mcl-1-specific+inhibitor+AZD5991+and+preclinical+activity+in+multiple+myeloma+and+acute+myeloid+leukemia&doi=10.1038%2Fs41467-018-07551-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Mcl-1-specific inhibitor AZD5991 and preclinical activity in multiple myeloma and acute myeloid leukemia</span></div><div class="casAuthors">Tron, Adriana E.; Belmonte, Matthew A.; Adam, Ammar; Aquila, Brian M.; Boise, Lawrence H.; Chiarparin, Elisabetta; Cidado, Justin; Embrey, Kevin J.; Gangl, Eric; Gibbons, Francis D.; Gregory, Gareth P.; Hargreaves, David; Hendricks, J. Adam; Johannes, Jeffrey W.; Johnstone, Ricky W.; Kazmirski, Steven L.; Kettle, Jason G.; Lamb, Michelle L.; Matulis, Shannon M.; Nooka, Ajay K.; Packer, Martin J.; Peng, Bo; Rawlins, Philip B.; Robbins, Daniel W.; Schuller, Alwin G.; Su, Nancy; Yang, Wenzhan; Ye, Qing; Zheng, Xiaolan; Secrist, J. Paul; Clark, Edwin A.; Wilson, David M.; Fawell, Stephen E.; Hird, Alexander W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5341</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Mcl-1 is a member of the Bcl-2 family of proteins that promotes cell survival by preventing induction of apoptosis in many cancers.  High expression of Mcl-1 causes tumorigenesis and resistance to anticancer therapies highlighting the potential of Mcl-1 inhibitors as anticancer drugs.  Here, we describe AZD5991, a rationally designed macrocyclic mol. with high selectivity and affinity for Mcl-1 currently in clin. development.  Our studies demonstrate that AZD5991 binds directly to Mcl-1 and induces rapid apoptosis in cancer cells, most notably myeloma and acute myeloid leukemia, by activating the Bak-dependent mitochondrial apoptotic pathway.  AZD5991 shows potent antitumor activity in vivo with complete tumor regression in several models of multiple myeloma and acute myeloid leukemia after a single tolerated dose as monotherapy or in combination with bortezomib or venetoclax.  Based on these promising data, a Phase I clin. trial has been launched for evaluation of AZD5991 in patients with hematol. malignancies (NCT03218683).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLv1vdwF2TY7Vg90H21EOLACvtfcHk0ljBekMYKdZ9Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFersLjK&md5=10ccf1fda947aca212bede3927eaabfa</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07551-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07551-w%26sid%3Dliteratum%253Aachs%26aulast%3DTron%26aufirst%3DA.%2BE.%26aulast%3DBelmonte%26aufirst%3DM.%2BA.%26aulast%3DAdam%26aufirst%3DA.%26aulast%3DAquila%26aufirst%3DB.%2BM.%26aulast%3DBoise%26aufirst%3DL.%2BH.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DEmbrey%26aufirst%3DK.%2BJ.%26aulast%3DGangl%26aufirst%3DE.%26aulast%3DGibbons%26aufirst%3DF.%2BD.%26aulast%3DGregory%26aufirst%3DG.%2BP.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHendricks%26aufirst%3DJ.%2BA.%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DKazmirski%26aufirst%3DS.%2BL.%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DMatulis%26aufirst%3DS.%2BM.%26aulast%3DNooka%26aufirst%3DA.%2BK.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DPeng%26aufirst%3DB.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobbins%26aufirst%3DD.%2BW.%26aulast%3DSchuller%26aufirst%3DA.%2BG.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DYe%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DSecrist%26aufirst%3DJ.%2BP.%26aulast%3DClark%26aufirst%3DE.%2BA.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DFawell%26aufirst%3DS.%2BE.%26aulast%3DHird%26aufirst%3DA.%2BW.%26atitle%3DDiscovery%2520of%2520Mcl-1-specific%2520inhibitor%2520AZD5991%2520and%2520preclinical%2520activity%2520in%2520multiple%2520myeloma%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D5341%26epage%3D5355%26doi%3D10.1038%2Fs41467-018-07551-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willoughby, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansma, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoye, T. R.</span></span> <span> </span><span class="NLM_article-title">A guide to small-molecule structure assignment through computation of (<sup>1</sup>H and <sup>13</sup>C) NMR chemical shifts</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">643</span>– <span class="NLM_lpage">660</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fnprot.2014.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=24556787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVehu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=643-660&author=P.+H.+Willoughbyauthor=M.+J.+Jansmaauthor=T.+R.+Hoye&title=A+guide+to+small-molecule+structure+assignment+through+computation+of+%281H+and+13C%29+NMR+chemical+shifts&doi=10.1038%2Fnprot.2014.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A guide to small-molecule structure assignment through computation of (1H and 13C) NMR chemical shifts</span></div><div class="casAuthors">Willoughby, Patrick H.; Jansma, Matthew J.; Hoye, Thomas R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">643-660</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This protocol is intended to provide chemists who discover or make new org. compds. with a valuable tool for validating the structural assignments of those new chem. entities.  Exptl. 1H and/or 13C NMR spectral data and its proper interpretation for the compd. of interest is required as a starting point.  The approach involves the following steps: (i) using mol. mechanics calcns. (with, e.g., MacroModel) to generate a library of conformers; (ii) using d. functional theory (DFT) calcns. (with, e.g., Gaussian 09) to det. optimal geometry, free energies and chem. shifts for each conformer; (iii) detg. Boltzmann-weighted proton and carbon chem. shifts; and (iv) comparing the computed chem. shifts for two or more candidate structures with exptl. data to det. the best fit.  For a typical structure assignment of a small org. mol. (e.g., fewer than ∼10 non-H atoms or up to ∼180 a.m.u. and ∼20 conformers), this protocol can be completed in ∼2 h of active effort over a 2-d period; for more complex mols. (e.g., fewer than ∼30 non-H atoms or up to ∼500 a.m.u. and ∼50 conformers), the protocol requires ∼36 h of active effort over a 2-wk period.  To demonstrate the method, we have chosen the anal. of the cis- vs. the trans-diastereoisomers of 3-methylcyclohexanol (1-cis vs. 1-trans).  The protocol is written in a manner that makes the computation of chem. shifts tractable for chemists who may otherwise have only rudimentary computational experience.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnhwPmqKqlJ7Vg90H21EOLACvtfcHk0lje_8RfLSXuOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVehu7w%253D&md5=0aa9872e60076e3e88c50f14981733b5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.042%26sid%3Dliteratum%253Aachs%26aulast%3DWilloughby%26aufirst%3DP.%2BH.%26aulast%3DJansma%26aufirst%3DM.%2BJ.%26aulast%3DHoye%26aufirst%3DT.%2BR.%26atitle%3DA%2520guide%2520to%2520small-molecule%2520structure%2520assignment%2520through%2520computation%2520of%2520%25281H%2520and%252013C%2529%2520NMR%2520chemical%2520shifts%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D643%26epage%3D660%26doi%3D10.1038%2Fnprot.2014.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimblat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarotti, A. M.</span></span> <span> </span><span class="NLM_article-title">Computational chemistry to the rescue: Modern toolboxes for the assignment of complex molecules by GIAO NMR calculations</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">12246</span>– <span class="NLM_lpage">12261</span>, <span class="refDoi"> DOI: 10.1002/chem.201601150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1002%2Fchem.201601150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=27405775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFGjurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=12246-12261&author=N.+Grimblatauthor=A.+M.+Sarotti&title=Computational+chemistry+to+the+rescue%3A+Modern+toolboxes+for+the+assignment+of+complex+molecules+by+GIAO+NMR+calculations&doi=10.1002%2Fchem.201601150"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Chemistry to the Rescue: Modern Toolboxes for the Assignment of Complex Molecules by GIAO NMR Calculations</span></div><div class="casAuthors">Grimblat, Nicolas; Sarotti, Ariel M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12246-12261</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The calcns. of NMR properties of mols. using quantum chem. methods have deeply impacted several branches of org. chem.  They are particularly important in structural or stereochem. assignments of org. compds., with implications in total synthesis, stereoselective reactions, and natural products chem.  In studying the evolution of the strategies developed to support (or reject) a structural proposal, it becomes clear that the most effective and accurate ones involve sophisticated procedures to correlate exptl. and computational data.  Owing to their relatively high math. complexity, such calcns. (CP3, DP4, ANN-PRA) are often carried out using addnl. computational resources provided by the authors (such as applets or Excel files).  This Minireview will cover the state-of-the-art of these toolboxes in the assignment of org. mols., including math. definitions, updates, and discussion of relevant examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1esvt38PZ5LVg90H21EOLACvtfcHk0lje_8RfLSXuOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFGjurnK&md5=1436b69b48a6a9344f981202cdb8954f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fchem.201601150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201601150%26sid%3Dliteratum%253Aachs%26aulast%3DGrimblat%26aufirst%3DN.%26aulast%3DSarotti%26aufirst%3DA.%2BM.%26atitle%3DComputational%2520chemistry%2520to%2520the%2520rescue%253A%2520Modern%2520toolboxes%2520for%2520the%2520assignment%2520of%2520complex%2520molecules%2520by%2520GIAO%2520NMR%2520calculations%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D12246%26epage%3D12261%26doi%3D10.1002%2Fchem.201601150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butts, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, V. K.</span></span> <span> </span><span class="NLM_article-title">Synergy of synthesis, computation and NMR reveals correct baulamycin structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>547</i></span>,  <span class="NLM_fpage">436</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1038/nature23265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fnature23265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28748934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1ent7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=547&publication_year=2017&pages=436-440&author=J.+Wuauthor=P.+Lorenzoauthor=S.+Zhongauthor=M.+Aliauthor=C.+P.+Buttsauthor=E.+L.+Myersauthor=V.+K.+Aggarwal&title=Synergy+of+synthesis%2C+computation+and+NMR+reveals+correct+baulamycin+structures&doi=10.1038%2Fnature23265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy of synthesis, computation and NMR reveals correct baulamycin structures</span></div><div class="casAuthors">Wu, Jingjing; Lorenzo, Paula; Zhong, Siying; Ali, Muhammad; Butts, Craig P.; Myers, Eddie L.; Aggarwal, Varinder K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">547</span>
        (<span class="NLM_cas:issue">7664</span>),
    <span class="NLM_cas:pages">436-440</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol., biol. active natural products continue to be our most rewarding source of, and inspiration for, new medicines.  Sometimes we happen upon such mols. in minute quantities in unique, difficult-to-reach, and often fleeting environments, perhaps never to be discovered again.  In these cases, detg. the structure of a mol.-including assigning its relative and abs. configurations-is paramount, enabling one to understand its biol. activity.  Mols. that comprise stereochem. complex acyclic and conformationally flexible carbon chains make such a task extremely challenging.  The baulamycins (A and B) serve as a contemporary example.  Isolated in small quantities and shown to have promising antimicrobial activity, the structure of the conformationally flexible mols. was detd. largely through J-based configurational anal., but has been found to be incorrect.  Our subsequent campaign to identify the true structures of the baulamycins has revealed a powerful method for the rapid structural elucidation of such mols.  Specifically, the prediction of NMR (NMR) parameters through d. functional theory-combined with an efficient sequence of boron-based synthetic transformations, which allowed an encoded (labeled) mixt. of natural-product diastereomers to be prepd.-enabled us rapidly to pinpoint and synthesize the correct structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNtULzfiHMbVg90H21EOLACvtfcHk0lje_8RfLSXuOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1ent7fJ&md5=c48c205289a43e43d4b2ab670619a64a</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature23265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23265%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLorenzo%26aufirst%3DP.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DAli%26aufirst%3DM.%26aulast%3DButts%26aufirst%3DC.%2BP.%26aulast%3DMyers%26aufirst%3DE.%2BL.%26aulast%3DAggarwal%26aufirst%3DV.%2BK.%26atitle%3DSynergy%2520of%2520synthesis%252C%2520computation%2520and%2520NMR%2520reveals%2520correct%2520baulamycin%2520structures%26jtitle%3DNature%26date%3D2017%26volume%3D547%26spage%3D436%26epage%3D440%26doi%3D10.1038%2Fnature23265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almond, A.</span></span> <span> </span><span class="NLM_article-title">Quantification of free ligand conformational preferences by NMR and their relationship to the bioactive conformation</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4976</span>– <span class="NLM_lpage">4987</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.06.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmc.2013.06.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=23886813" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFCqt77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4976-4987&author=C.+D.+Blundellauthor=M.+J.+Packerauthor=A.+Almond&title=Quantification+of+free+ligand+conformational+preferences+by+NMR+and+their+relationship+to+the+bioactive+conformation&doi=10.1016%2Fj.bmc.2013.06.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Quantification of free ligand conformational preferences by NMR and their relationship to the bioactive conformation</span></div><div class="casAuthors">Blundell, Charles D.; Packer, Martin J.; Almond, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4976-4987</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Accurate unbound soln. 3D-structures of ligands provide unique opportunities for medicinal chem. and, in particular, a context to understand binding thermodn. and kinetics.  Previous methods of deriving these 3D-structures have had neither the accuracy nor resoln. needed for drug design and have not yet realized their potential.  Here, we describe and apply a NMR methodol. to the aminoglycoside streptomycin that can accurately quantify accessible 3D-space and rank the occupancy of obsd. conformers to a resoln. that enables medicinal chem. understanding and design.  Importantly, it is based upon conventional small mol. NMR techniques and can be performed in physiol.-relevant solvents.  The methodol. uses multiple datasets, an order of magnitude more exptl. data than previous NMR approaches and a dynamic model during refinement, is independent of computational chem. and avoids the problem of virtual conformations.  The refined set of soln. 3D-shapes for streptomycin can be grouped into two major families, of which the most populated is almost identical to the 30S ribosomal subunit bioactive shape.  We therefore propose that accurate unbound ligand soln. conformations may, in some cases, provide a subsidiary route to bioactive shape without crystallog.  This exptl. technique opens up new opportunities for drug design and more so when complemented with protein co-crystal structures, SAR data and pharmacophore modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUZZsawI_fmLVg90H21EOLACvtfcHk0ljtzzkzai6EOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCqt77K&md5=3ac32a01828e59160794384644aacbf4</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.06.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.06.056%26sid%3Dliteratum%253Aachs%26aulast%3DBlundell%26aufirst%3DC.%2BD.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DAlmond%26aufirst%3DA.%26atitle%3DQuantification%2520of%2520free%2520ligand%2520conformational%2520preferences%2520by%2520NMR%2520and%2520their%2520relationship%2520to%2520the%2520bioactive%2520conformation%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4976%26epage%3D4987%26doi%3D10.1016%2Fj.bmc.2013.06.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span> <i>ACD/NMR Workbook</i>, version <span class="NLM_edition">2018.2</span>; <span class="NLM_publisher-name">Advanced Chemistry Development, Inc.</span>: <span class="NLM_publisher-loc">Toronto, ON, Canada</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ACD%2FNMR+Workbook%2C+version+2018.2%3B+Advanced+Chemistry+Development%2C+Inc.%3A+Toronto%2C+ON%2C+Canada%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DACD%252FNMR%2520Workbook%26pub%3DAdvanced%2520Chemistry%2520Development%252C%2520Inc%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span> <i>MestReNova, MSpin</i>, version <span class="NLM_edition">14.0.0-23239</span>; <span class="NLM_publisher-name">Mestrelab
Research S.L.</span>: <span class="NLM_publisher-loc">Santiago de Compostela,
Spain</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+MestReNova%2C+MSpin%2C+version+14.0.0-23239%3B+Mestrelab%0AResearch+S.L.%3A+Santiago+de+Compostela%2C%0ASpain%2C+2019."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMestReNova%252C%2520MSpin%26pub%3DMestrelab%250AResearch%2520S.L%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span> <i>Molecular Operating Environment (MOE)</i>, version <span class="NLM_edition">2018.01</span>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span> (<span class="NLM_publisher-loc">1010 Sherbooke St. West, Suite No. 910, Montreal,
QC H3A 2R7, Canada</span>), <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+version+2018.01%3B+Chemical+Computing+Group+ULC+%281010+Sherbooke+St.+West%2C+Suite+No.+910%2C+Montreal%2C%0AQC+H3A+2R7%2C+Canada%29%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frisch, M. J.</span>; <span class="NLM_string-name">Trucks, G. W.</span>; <span class="NLM_string-name">Schlegel, H. B.</span>; <span class="NLM_string-name">Scuseria, G. E.</span>; <span class="NLM_string-name">Robb, M. A.</span>; <span class="NLM_string-name">Cheeseman, J. R.</span>; <span class="NLM_string-name">Scalmani, G.</span>; <span class="NLM_string-name">Barone, V.</span>; <span class="NLM_string-name">Petersson, G. A.</span>; <span class="NLM_string-name">Nakatsuji, H.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Caricato, M.</span>; <span class="NLM_string-name">Marenich, A. V.</span>; <span class="NLM_string-name">Bloino, J.</span>; <span class="NLM_string-name">Janesko, B. G.</span>; <span class="NLM_string-name">Gomperts, R.</span>; <span class="NLM_string-name">Mennucci, B.</span>; <span class="NLM_string-name">Hratchian, H. P.</span>; <span class="NLM_string-name">Ortiz, J. V.</span>; <span class="NLM_string-name">Izmaylov, A. F.</span>; <span class="NLM_string-name">Sonnenberg, J. L.</span>; <span class="NLM_string-name">Williams-Young, D.</span>; <span class="NLM_string-name">Ding, F.</span>; <span class="NLM_string-name">Lipparini, F.</span>; <span class="NLM_string-name">Egidi, F.</span>; <span class="NLM_string-name">Goings, J.</span>; <span class="NLM_string-name">Peng, B.</span>; <span class="NLM_string-name">Petrone, A.</span>; <span class="NLM_string-name">Henderson, T.</span>; <span class="NLM_string-name">Ranasinghe, D.</span>; <span class="NLM_string-name">Zakrzewski, V. G.</span>; <span class="NLM_string-name">Gao, J.</span>; <span class="NLM_string-name">Rega, N.</span>; <span class="NLM_string-name">Zheng, G.</span>; <span class="NLM_string-name">Liang, W.</span>; <span class="NLM_string-name">Hada, M.</span>; <span class="NLM_string-name">Ehara, M.</span>; <span class="NLM_string-name">Toyota, K.</span>; <span class="NLM_string-name">Fukuda, R.</span>; <span class="NLM_string-name">Hasegawa, J.</span>; <span class="NLM_string-name">Ishida, M.</span>; <span class="NLM_string-name">Nakajima, T.</span>; <span class="NLM_string-name">Honda, Y.</span>; <span class="NLM_string-name">Kitao, O.</span>; <span class="NLM_string-name">Nakai, H.</span>; <span class="NLM_string-name">Vreven, T.</span>; <span class="NLM_string-name">Throssell, K.</span>; <span class="NLM_string-name">Montgomery, J. A.,  Jr.</span>; <span class="NLM_string-name">Peralta, J. E.</span>; <span class="NLM_string-name">Ogliaro, F.</span>; <span class="NLM_string-name">Bearpark, M. J.</span>; <span class="NLM_string-name">Heyd, J. J.</span>; <span class="NLM_string-name">Brothers, E. N.</span>; <span class="NLM_string-name">Kudin, K. N.</span>; <span class="NLM_string-name">Staroverov, V. N.</span>; <span class="NLM_string-name">Keith, T. A.</span>; <span class="NLM_string-name">Kobayashi, R.</span>; <span class="NLM_string-name">Normand, J.</span>; <span class="NLM_string-name">Raghavachari, K.</span>; <span class="NLM_string-name">Rendell, A. P.</span>; <span class="NLM_string-name">Burant, J. C.</span>; <span class="NLM_string-name">Iyengar, S. S.</span>; <span class="NLM_string-name">Tomasi, J.</span>; <span class="NLM_string-name">Cossi, M.</span>; <span class="NLM_string-name">Millam, J. M.</span>; <span class="NLM_string-name">Klene, M.</span>; <span class="NLM_string-name">Adamo, C.</span>; <span class="NLM_string-name">Cammi, R.</span>; <span class="NLM_string-name">Ochterski, J. W.</span>; <span class="NLM_string-name">Martin, R. L.</span>; <span class="NLM_string-name">Morokuma, K.</span>; <span class="NLM_string-name">Farkas, O.</span>; <span class="NLM_string-name">Foresman, J. B.</span>; <span class="NLM_string-name">Fox, D. J.</span></span>,  <i>Gaussian 16</i>, revision <span class="NLM_edition">B.01</span>; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=G.+A.+Petersson&author=H.+Nakatsuji&author=X.+Li&author=M.+Caricato&author=A.+V.+Marenich&author=J.+Bloino&author=B.+G.+Janesko&author=R.+Gomperts&author=B.+Mennucci&author=H.+P.+Hratchian&author=J.+V.+Ortiz&author=A.+F.+Izmaylov&author=J.+L.+Sonnenberg&author=D.+Williams-Young&author=F.+Ding&author=F.+Lipparini&author=F.+Egidi&author=J.+Goings&author=B.+Peng&author=A.+Petrone&author=T.+Henderson&author=D.+Ranasinghe&author=V.+G.+Zakrzewski&author=J.+Gao&author=N.+Rega&author=G.+Zheng&author=W.+Liang&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=K.+Throssell&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+J.+Bearpark&author=J.+J.+Heyd&author=E.+N.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=T.+A.+Keith&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+P.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=J.+M.+Millam&author=M.+Klene&author=C.+Adamo&author=R.+Cammi&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=O.+Farkas&author=J.+B.+Foresman&author=D.+J.+Fox&title=Gaussian+16"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252016%26pub%3DGaussian%252C%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beigie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belmonte, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joubran, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazmirski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mlynarski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robbins, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickson, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span> <span> </span><span class="NLM_article-title">Structure based design of non-natural peptidic macrocyclic Mcl-1 inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">239</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00464</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00464" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCqtLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=239-244&author=J.+W.+Johannesauthor=S.+Batesauthor=C.+Beigieauthor=M.+A.+Belmonteauthor=J.+Breenauthor=S.+Caoauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=J.+W.+Cuozzoauthor=C.+E.+Dumelinauthor=A.+D.+Fergusonauthor=S.+Habeshianauthor=D.+Hargreavesauthor=C.+Joubranauthor=S.+Kazmirskiauthor=A.+D.+Keefeauthor=M.+L.+Lambauthor=H.+Lanauthor=Y.+Liauthor=H.+Maauthor=S.+Mlynarskiauthor=M.+J.+Packerauthor=P.+B.+Rawlinsauthor=D.+W.+Robbinsauthor=H.+Shenauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=N.+Suauthor=D.+M.+Troastauthor=H.+Wangauthor=K.+F.+Wicksonauthor=C.+Wuauthor=Y.+Zhangauthor=Q.+Zhaoauthor=X.+Zhengauthor=A.+W.+Hird&title=Structure+based+design+of+non-natural+peptidic+macrocyclic+Mcl-1+inhibitors&doi=10.1021%2Facsmedchemlett.6b00464"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure based design of non-natural peptidic macrocyclic Mcl-1 inhibitors</span></div><div class="casAuthors">Johannes, Jeffrey W.; Bates, Stephanie; Beigie, Carl; Belmonte, Matt; Breen, John; Cao, Shenggen; Centrella, Paolo A.; Clark, Matthew A.; Cuozzo, John W.; Dumelin, Christoph E.; Ferguson, Andrew D.; Habeshian, Sevan; Hargreaves, David; Joubran, Camil; Kazmirski, Steven; Keefe, Anthony D.; Lamb, Michelle L.; Lan, Haiye; Li, Yunxia; Ma, Hao; Mlynarski, Scott; Packer, Martin J.; Rawlins, Philip B.; Robbins, Daniel W.; Shen, Haidong; Sigel, Eric A.; Soutter, Holly H.; Su, Nancy; Troast, Dawn M.; Wang, Haiyun; Wickson, Kate F.; Wu, Chengyan; Zhang, Ying; Zhao, Qiuying; Zheng, Xiaolan; Hird, Alexander W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-244</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mcl-1 is a pro-apoptotic BH3 protein family member similar to Bcl-2 and Bcl-xL.  Overexpression of Mcl-1 is often seen in various tumors and allows cancer cells to evade apoptosis.  Here we report the discovery and optimization of a series of non-natural peptide Mcl-1 inhibitors.  Screening of DNA-encoded libraries resulted in hit compd. (I), a 1.5 μM Mcl-1 inhibitor.  A subsequent crystal structure demonstrated that compd. I bound to Mcl-1 in a β-turn conformation, such that the two ends of the peptide were close together.  This proximity allowed for the linking of the two ends of the peptide to form a macrocycle.  Macrocyclization resulted in an approx. 10-fold improvement in binding potency.  Further exploration of a key hydrophobic interaction with Mcl-1 protein and also with the moiety that engages Arg256 led to addnl. potency improvements.  The use of protein-ligand crystal structures and binding kinetics contributed to the design and understanding of the potency gains.  Optimized compd. (II) is a <3 nM Mcl-1 inhibitor, while inhibiting Bcl-2 at only 5 μM and Bcl-xL at >99 μM, and induces cleaved caspase-3 in MV4-11 cells with an IC50 of 3 μM after 6 h.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGRmmxgTy5drVg90H21EOLACvtfcHk0lh18nJDMb91yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCqtLzP&md5=42f9a8d863aedfe95231e29ffa4673b8</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00464%26sid%3Dliteratum%253Aachs%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DBates%26aufirst%3DS.%26aulast%3DBeigie%26aufirst%3DC.%26aulast%3DBelmonte%26aufirst%3DM.%2BA.%26aulast%3DBreen%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DS.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DJoubran%26aufirst%3DC.%26aulast%3DKazmirski%26aufirst%3DS.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DLan%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DMlynarski%26aufirst%3DS.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobbins%26aufirst%3DD.%2BW.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DSu%26aufirst%3DN.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWickson%26aufirst%3DK.%2BF.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DHird%26aufirst%3DA.%2BW.%26atitle%3DStructure%2520based%2520design%2520of%2520non-natural%2520peptidic%2520macrocyclic%2520Mcl-1%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D239%26epage%3D244%26doi%3D10.1021%2Facsmedchemlett.6b00464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blades, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chuaqui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fillery, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangl, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grist, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemmitt, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prince, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of pyrazolo[1,5-a]pyrimidine B-cell lymphoma 6 (BCL6) binders and optimization to high affinity macrocyclic inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4386</span>– <span class="NLM_lpage">4402</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00359</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00359" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4386-4402&author=W.+McCoullauthor=R.+D.+Abramsauthor=E.+Andersonauthor=K.+Bladesauthor=P.+Bartonauthor=M.+Boxauthor=J.+Burgessauthor=K.+Bythauthor=Q.+Caoauthor=C.+Chuaquiauthor=R.+J.+Carbajoauthor=T.+Cheungauthor=E.+Codeauthor=A.+D.+Fergusonauthor=S.+Filleryauthor=N.+O.+Fullerauthor=E.+Ganglauthor=N.+Gaoauthor=M.+Gristauthor=D.+Hargreavesauthor=M.+R.+Howardauthor=J.+Huauthor=P.+D.+Kemmittauthor=J.+E.+Nelsonauthor=N.+O%E2%80%99Connellauthor=D.+B.+Princeauthor=P.+Rauboauthor=P.+B.+Rawlinsauthor=G.+R.+Robbauthor=J.+Shiauthor=M.+J.+Waringauthor=D.+Whittakerauthor=M.+Wylotauthor=X.+Zhu&title=Discovery+of+pyrazolo%5B1%2C5-a%5Dpyrimidine+B-cell+lymphoma+6+%28BCL6%29+binders+and+optimization+to+high+affinity+macrocyclic+inhibitors&doi=10.1021%2Facs.jmedchem.7b00359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrazolo[1,5-a]pyrimidine B-Cell Lymphoma 6 (BCL6) Binders and Optimization to High Affinity Macrocyclic Inhibitors</span></div><div class="casAuthors">McCoull, William; Abrams, Roman D.; Anderson, Erica; Blades, Kevin; Barton, Peter; Box, Matthew; Burgess, Jonathan; Byth, Kate; Cao, Qing; Chuaqui, Claudio; Carbajo, Rodrigo J.; Cheung, Tony; Code, Erin; Ferguson, Andrew D.; Fillery, Shaun; Fuller, Nathan O.; Gangl, Eric; Gao, Ning; Grist, Matthew; Hargreaves, David; Howard, Martin R.; Hu, Jun; Kemmitt, Paul D.; Nelson, Jennifer E.; O'Connell, Nichole; Prince, D. Bryan; Raubo, Piotr; Rawlins, Philip B.; Robb, Graeme R.; Shi, Junjie; Waring, Michael J.; Whittaker, David; Wylot, Marta; Zhu, Xiahui</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4386-4402</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the protein-protein interaction between B-cell lymphoma 6 (BCL6) and corepressors has been implicated as a therapeutic target in diffuse large B-cell lymphoma (DLBCL) cancers and profiling of potent and selective BCL6 inhibitors are crit. to test this hypothesis.  We identified a pyrazolo[1,5-a]pyrimidine series of BCL6 binders from a fragment screen in parallel with a virtual screen.  Using structure-based drug design, binding affinity was increased 100000-fold.  This involved displacing crystallog. water, forming new ligand-protein interactions and a macrocyclization to favor the bioactive conformation of the ligands.  Optimization for slow off-rate const. kinetics was conducted as well as improving selectivity against an off-target kinase, CK2.  Potency in a cellular BCL6 assay was further optimized to afford highly selective probe mols.  Only weak antiproliferative effects were obsd. across a no. of DLBCL lines and a multiple myeloma cell line without a clear relationship to BCL6 potency.  As a result, we conclude that the BCL6 hypothesis in DLBCL cancer remains unproven.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWp2OlXAZDFLVg90H21EOLACvtfcHk0li4BuqesHS-ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFOmsbo%253D&md5=fc4142453a0580edbf04802e15d4d9aa</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00359%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DAbrams%26aufirst%3DR.%2BD.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DBlades%26aufirst%3DK.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DBox%26aufirst%3DM.%26aulast%3DBurgess%26aufirst%3DJ.%26aulast%3DByth%26aufirst%3DK.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DChuaqui%26aufirst%3DC.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DCheung%26aufirst%3DT.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DFillery%26aufirst%3DS.%26aulast%3DFuller%26aufirst%3DN.%2BO.%26aulast%3DGangl%26aufirst%3DE.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DGrist%26aufirst%3DM.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DHoward%26aufirst%3DM.%2BR.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DKemmitt%26aufirst%3DP.%2BD.%26aulast%3DNelson%26aufirst%3DJ.%2BE.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DN.%26aulast%3DPrince%26aufirst%3DD.%2BB.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRawlins%26aufirst%3DP.%2BB.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWylot%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520pyrazolo%255B1%252C5-a%255Dpyrimidine%2520B-cell%2520lymphoma%25206%2520%2528BCL6%2529%2520binders%2520and%2520optimization%2520to%2520high%2520affinity%2520macrocyclic%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4386%26epage%3D4402%26doi%3D10.1021%2Facs.jmedchem.7b00359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaPlante, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nar, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aubry, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawai, S. H.</span></span> <span> </span><span class="NLM_article-title">Ligand bioactive conformation plays a critical role in the design of drugs that target the hepatitis C virus NS3 protease</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1777</span>– <span class="NLM_lpage">1789</span>, <span class="refDoi"> DOI: 10.1021/jm401338c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401338c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Cktb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1777-1789&author=S.+R.+LaPlanteauthor=H.+Narauthor=C.+T.+Lemkeauthor=A.+Jakalianauthor=N.+Aubryauthor=S.+H.+Kawai&title=Ligand+bioactive+conformation+plays+a+critical+role+in+the+design+of+drugs+that+target+the+hepatitis+C+virus+NS3+protease&doi=10.1021%2Fjm401338c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Bioactive Conformation Plays a Critical Role in the Design of Drugs That Target the Hepatitis C Virus NS3 Protease</span></div><div class="casAuthors">LaPlante, Steven R.; Nar, Herbert; Lemke, Christopher T.; Jakalian, Araz; Aubry, Norman; Kawai, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1777-1789</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A ligand-focused strategy employed NMR, X-ray, modeling, and medicinal chem. to expose the crit. role that bioactive conformation played in the design of a variety of drugs that target the HCV protease.  The bioactive conformation (bound states) were detd. for key inhibitors identified along our drug discovery pathway from the hit to clin. compds.  All adopt similar bioactive conformations for the common core derived from the hit peptide DDIVPC.  A carefully designed SAR anal., based on the advanced inhibitor 1 in which the P1 to P3 side chains and the N-terminal Boc were sequentially truncated, revealed a correlation between affinity and the relative predominance of the bioactive conformation in the free state.  Interestingly, synergistic conformation effects on potency were also noted.  Comparisons with clin. and recently marketed drugs from the pharmaceutical industry showed that all have the same core and similar bioactive conformations.  This suggested that the variety of appendages discovered for these compds. also properly satisfy the bioactive conformation requirements and allowed for a large variety of HCV protease drug candidates to be designed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFi-4R4F9wwbVg90H21EOLACvtfcHk0li4BuqesHS-ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Cktb7P&md5=2ccb36504903255ba012223529318a69</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm401338c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401338c%26sid%3Dliteratum%253Aachs%26aulast%3DLaPlante%26aufirst%3DS.%2BR.%26aulast%3DNar%26aufirst%3DH.%26aulast%3DLemke%26aufirst%3DC.%2BT.%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DAubry%26aufirst%3DN.%26aulast%3DKawai%26aufirst%3DS.%2BH.%26atitle%3DLigand%2520bioactive%2520conformation%2520plays%2520a%2520critical%2520role%2520in%2520the%2520design%2520of%2520drugs%2520that%2520target%2520the%2520hepatitis%2520C%2520virus%2520NS3%2520protease%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1777%26epage%3D1789%26doi%3D10.1021%2Fjm401338c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di Fabio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellacani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerrard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thewlis, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spada, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemp, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massagrande, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braggio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marchioro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsi, M.</span></span> <span> </span><span class="NLM_article-title">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5562</span>– <span class="NLM_lpage">5567</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.06.086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmcl.2011.06.086" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=21831639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=5562-5567&author=R.+Di+Fabioauthor=A.+Pellacaniauthor=S.+Faedoauthor=A.+Rothauthor=L.+Piccoliauthor=P.+Gerrardauthor=R.+A.+Porterauthor=C.+N.+Johnsonauthor=K.+Thewlisauthor=D.+Donatiauthor=L.+Stasiauthor=S.+Spadaauthor=G.+Stempauthor=D.+Nashauthor=C.+Branchauthor=L.+Kindonauthor=M.+Massagrandeauthor=A.+Poffeauthor=S.+Braggioauthor=E.+Chiarparinauthor=C.+Marchioroauthor=E.+Rattiauthor=M.+Corsi&title=Discovery+process+and+pharmacological+characterization+of+a+novel+dual+orexin+1+and+orexin+2+receptor+antagonist+useful+for+treatment+of+sleep+disorders&doi=10.1016%2Fj.bmcl.2011.06.086"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery process and pharmacological characterization of a novel dual orexin 1 and orexin 2 receptor antagonist useful for treatment of sleep disorders</span></div><div class="casAuthors">Di Fabio, Romano; Pellacani, Annalisa; Faedo, Stefania; Roth, Adelheid; Piccoli, Laura; Gerrard, Philip; Porter, Rod A.; Johnson, Christopher N.; Thewlis, Kevin; Donati, Daniele; Stasi, Luigi; Spada, Simone; Stemp, Geoffrey; Nash, David; Branch, Clive; Kindon, Leanda; Massagrande, Mario; Poffe, Alessandro; Braggio, Simone; Chiarparin, Elisabetta; Marchioro, Carla; Ratti, Emiliangelo; Corsi, Mauro</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5562-5567</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The hypothalamic peptides orexin-A and orexin-B are potent agonists of two G-protein coupled receptors, namely the OX1 and the OX2 receptor.  These receptors are widely distributed, though differentially, in the rat brain.  In particular, the OX1 receptor is highly expressed throughout the hypothalamus, while the OX2 receptor is mainly located in the ventral posterior nucleus.  A large body of compelling evidence, both pre-clin. and clin., suggests that the orexin system is profoundly implicated in sleep disorders.  In particular, modulation of the orexin receptors activation by appropriate antagonists was proven to be an efficacious strategy for the treatment of insomnia in man.  A novel, drug-like bis-amido piperidine deriv. (I) was identified as potent dual OX1 and OX2 receptor antagonists, highly effective in a pre-clin. model of sleep.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3g2IG2vbbbVg90H21EOLACvtfcHk0lh35GAN39utmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKisLzL&md5=02d4a2f2682a9e35689a9e1553858e69</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.06.086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.06.086%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BFabio%26aufirst%3DR.%26aulast%3DPellacani%26aufirst%3DA.%26aulast%3DFaedo%26aufirst%3DS.%26aulast%3DRoth%26aufirst%3DA.%26aulast%3DPiccoli%26aufirst%3DL.%26aulast%3DGerrard%26aufirst%3DP.%26aulast%3DPorter%26aufirst%3DR.%2BA.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DThewlis%26aufirst%3DK.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DStasi%26aufirst%3DL.%26aulast%3DSpada%26aufirst%3DS.%26aulast%3DStemp%26aufirst%3DG.%26aulast%3DNash%26aufirst%3DD.%26aulast%3DBranch%26aufirst%3DC.%26aulast%3DKindon%26aufirst%3DL.%26aulast%3DMassagrande%26aufirst%3DM.%26aulast%3DPoffe%26aufirst%3DA.%26aulast%3DBraggio%26aufirst%3DS.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DMarchioro%26aufirst%3DC.%26aulast%3DRatti%26aufirst%3DE.%26aulast%3DCorsi%26aufirst%3DM.%26atitle%3DDiscovery%2520process%2520and%2520pharmacological%2520characterization%2520of%2520a%2520novel%2520dual%2520orexin%25201%2520and%2520orexin%25202%2520receptor%2520antagonist%2520useful%2520for%2520treatment%2520of%2520sleep%2520disorders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D5562%26epage%3D5567%26doi%3D10.1016%2Fj.bmcl.2011.06.086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dinsmore, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogusky, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culberson, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homnick, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zartman, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosser, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaber, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koblan, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huff, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, T. M.</span></span> <span> </span><span class="NLM_article-title">Conformational restriction of flexible ligands guided by the transferred NOE experiment: Potent macrocyclic inhibitors of farnesyltransferase</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">2107</span>– <span class="NLM_lpage">2108</span>, <span class="refDoi"> DOI: 10.1021/ja003673q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja003673q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD3MXpslOjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2001&pages=2107-2108&author=C.+J.+Dinsmoreauthor=M.+J.+Boguskyauthor=J.+C.+Culbersonauthor=J.+M.+Bergmanauthor=C.+F.+Homnickauthor=C.+B.+Zartmanauthor=S.+D.+Mosserauthor=M.+D.+Schaberauthor=R.+G.+Robinsonauthor=K.+S.+Koblanauthor=H.+E.+Huberauthor=S.+L.+Grahamauthor=G.+D.+Hartmanauthor=J.+R.+Huffauthor=T.+M.+Williams&title=Conformational+restriction+of+flexible+ligands+guided+by+the+transferred+NOE+experiment%3A+Potent+macrocyclic+inhibitors+of+farnesyltransferase&doi=10.1021%2Fja003673q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Restriction of Flexible Ligands Guided by the Transferred NOE Experiment: Potent Macrocyclic Inhibitors of Farnesyltransferase</span></div><div class="casAuthors">Dinsmore, Christopher J.; Bogusky, Michael J.; Culberson, J. Christopher; Bergman, Jeffrey M.; Homnick, Carl F.; Zartman, C. Blair; Mosser, Scott D.; Schaber, Michael D.; Robinson, Ronald G.; Koblan, Kenneth S.; Huber, Hans E.; Graham, Samuel L.; Hartman, George D.; Huff, Joel R.; Williams, Theresa M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2107-2108</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Farnesyltransferase is an important post translational processing enzyme involved in the signal transduction during cell proliferation.  Inhibitors of this enzyme are promising antitumor agents and several are currently being evaluated in the human clin. trials.  This paper describes the design and synthesis of a highly potent macrocyclic farnesyltransferase inhibitor based on the transferred NOE structure of a conformationally flexible analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMVw4lvnkqI7Vg90H21EOLACvtfcHk0lhbQAozItegOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXpslOjsQ%253D%253D&md5=3ff79e2cbb07ebab6ba7f03c4f1a4417</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja003673q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja003673q%26sid%3Dliteratum%253Aachs%26aulast%3DDinsmore%26aufirst%3DC.%2BJ.%26aulast%3DBogusky%26aufirst%3DM.%2BJ.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DBergman%26aufirst%3DJ.%2BM.%26aulast%3DHomnick%26aufirst%3DC.%2BF.%26aulast%3DZartman%26aufirst%3DC.%2BB.%26aulast%3DMosser%26aufirst%3DS.%2BD.%26aulast%3DSchaber%26aufirst%3DM.%2BD.%26aulast%3DRobinson%26aufirst%3DR.%2BG.%26aulast%3DKoblan%26aufirst%3DK.%2BS.%26aulast%3DHuber%26aufirst%3DH.%2BE.%26aulast%3DGraham%26aufirst%3DS.%2BL.%26aulast%3DHartman%26aufirst%3DG.%2BD.%26aulast%3DHuff%26aufirst%3DJ.%2BR.%26aulast%3DWilliams%26aufirst%3DT.%2BM.%26atitle%3DConformational%2520restriction%2520of%2520flexible%2520ligands%2520guided%2520by%2520the%2520transferred%2520NOE%2520experiment%253A%2520Potent%2520macrocyclic%2520inhibitors%2520of%2520farnesyltransferase%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2001%26volume%3D123%26spage%3D2107%26epage%3D2108%26doi%3D10.1021%2Fja003673q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tice, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span> <span> </span><span class="NLM_article-title">Conformational control in structure-based drug design</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2825</span>– <span class="NLM_lpage">2837</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.04.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmcl.2017.04.079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28479196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntFamtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2825-2837&author=Y.+Zhengauthor=C.+M.+Ticeauthor=S.+B.+Singh&title=Conformational+control+in+structure-based+drug+design&doi=10.1016%2Fj.bmcl.2017.04.079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational control in structure-based drug design</span></div><div class="casAuthors">Zheng, Yajun; Tice, Colin M.; Singh, Suresh B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2825-2837</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In structure-based drug design, the basic goal is to design mols. that fit complementarily to a given binding pocket.  Since such computationally modeled mols. may not adopt the intended bound conformation outside the binding pocket, one challenge is to ensure that the designed ligands adopt similar low energy conformations both inside and outside of the binding pocket.  Computational chem. methods and conformational preferences of small mols. from PDB and Cambridge Structural Database (CSD) can be used to predict the bound structures of the designed mols.  Herein, we review applications of conformational control in structure-based drug design using selected examples from the recent medicinal chem. literature.  The main purpose is to highlight some intriguing conformational features that can be applied to other drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbfYQ3iP4BT7Vg90H21EOLACvtfcHk0lhbQAozItegOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntFamtr8%253D&md5=aef5a7aa51f9b6a8454345a4e38d95db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.04.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.04.079%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DTice%26aufirst%3DC.%2BM.%26aulast%3DSingh%26aufirst%3DS.%2BB.%26atitle%3DConformational%2520control%2520in%2520structure-based%2520drug%2520design%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2825%26epage%3D2837%26doi%3D10.1016%2Fj.bmcl.2017.04.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, A. D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacCoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heer, J. P.</span></span> <span> </span><span class="NLM_article-title">Importance of rigidity in designing small molecule drugs to tackle protein-protein interactions (PPIs) through stabilization of desired conformers</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4283</span>– <span class="NLM_lpage">4289</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01120</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01120" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4283-4289&author=A.+D.+G.+Lawsonauthor=M.+MacCossauthor=J.+P.+Heer&title=Importance+of+rigidity+in+designing+small+molecule+drugs+to+tackle+protein-protein+interactions+%28PPIs%29+through+stabilization+of+desired+conformers&doi=10.1021%2Facs.jmedchem.7b01120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Importance of Rigidity in Designing Small Molecule Drugs To Tackle Protein-Protein Interactions (PPIs) through Stabilization of Desired Conformers</span></div><div class="casAuthors">Lawson, Alastair D. G.; MacCoss, Malcolm; Heer, Jag P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4283-4289</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tackling PPIs, particularly by stabilizing clin. favored conformations of target proteins, with orally available, bona fide small mols. remains a significant but immensely worthwhile challenge for the pharmaceutical industry.  Success may be more likely through the application of nature's learnings to build intrinsic rigidity into the design of clin. candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN_JJHmVzv_rVg90H21EOLACvtfcHk0lhbQAozItegOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtb7J&md5=665f95b08aeea2757f0a3b4dc4f813ed</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01120%26sid%3Dliteratum%253Aachs%26aulast%3DLawson%26aufirst%3DA.%2BD.%2BG.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DHeer%26aufirst%3DJ.%2BP.%26atitle%3DImportance%2520of%2520rigidity%2520in%2520designing%2520small%2520molecule%2520drugs%2520to%2520tackle%2520protein-protein%2520interactions%2520%2528PPIs%2529%2520through%2520stabilization%2520of%2520desired%2520conformers%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4283%26epage%3D4289%26doi%3D10.1021%2Facs.jmedchem.7b01120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span>; <span class="NLM_string-name">Souers, A. J.</span>; <span class="NLM_string-name">Bruncko, M.</span>; <span class="NLM_string-name">Song, X.</span>; <span class="NLM_string-name">Wang, X.</span>; <span class="NLM_string-name">Hasvold, L. A.</span>; <span class="NLM_string-name">Wang, L.</span>; <span class="NLM_string-name">Kunzer, A. R.</span>; <span class="NLM_string-name">Park, C.-M.</span>; <span class="NLM_string-name">Wendt, M. D.</span>; <span class="NLM_string-name">Tao, Z.-F.</span></span> <span> </span><span class="NLM_article-title">7-Substituted indoles as mcl-1 protein inhibitors and their preparation</span>. PCT Int. Appl. <span class="NLM_patent">WO2008131000A2</span>, <span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=S.+W.+Elmore&author=A.+J.+Souers&author=M.+Bruncko&author=X.+Song&author=X.+Wang&author=L.+A.+Hasvold&author=L.+Wang&author=A.+R.+Kunzer&author=C.-M.+Park&author=M.+D.+Wendt&author=Z.-F.+Tao&title=7-Substituted+indoles+as+mcl-1+protein+inhibitors+and+their+preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DElmore%26aufirst%3DS.%2BW.%26atitle%3D7-Substituted%2520indoles%2520as%2520mcl-1%2520protein%2520inhibitors%2520and%2520their%2520preparation%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyer-Almes, F. J.</span></span> <span> </span><span class="NLM_article-title">Kinetic binding assays for the analysis of protein-ligand interactions</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.ddtec.2015.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=26724330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1egtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=1-8&author=F.+J.+Meyer-Almes&title=Kinetic+binding+assays+for+the+analysis+of+protein-ligand+interactions&doi=10.1016%2Fj.ddtec.2015.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic binding assays for the analysis of protein-ligand interactions</span></div><div class="casAuthors">Meyer-Almes Franz-Josef</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The importance of binding kinetics in terms of residence time and on-rate in drug discovery has been broadly accepted in the past few years.  Furthermore, evidence has accumulated that the optimal binding mechanism of a drug to its target molecule is related to physiological efficacy as well as selectivity and thus drug safety.  Homogeneous fluorescence-based binding assays have been shown to enable high throughput kinetics requiring only small amounts of protein and can be developed to elucidate even complex mechanisms of molecular recognition.  A generalized approach is proposed that combines high quality kinetic and equilibrium data in an Integrated Global Fit analysis yielding the most probable binding mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwRMy18rtTEzjk2McQZ6XmfW6udTcc2ebN3er902DcK7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1egtg%253D%253D&md5=be48951ac65ad4bfb9bc2ec4a398ee1f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer-Almes%26aufirst%3DF.%2BJ.%26atitle%3DKinetic%2520binding%2520assays%2520for%2520the%2520analysis%2520of%2520protein-ligand%2520interactions%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.ddtec.2015.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Folmer, R. H. A.</span></span> <span> </span><span class="NLM_article-title">Drug target residence time: A misleading concept</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.07.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.drudis.2017.07.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28782685" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSitbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=12-16&author=R.+H.+A.+Folmer&title=Drug+target+residence+time%3A+A+misleading+concept&doi=10.1016%2Fj.drudis.2017.07.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Drug target residence time: a misleading concept</span></div><div class="casAuthors">Folmer, Rutger H. A.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-16</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Since the importance of drug target residence time was first highlighted more 10 years ago, slow binding kinetics has received much attention in the drug discovery literature, and indeed within pharmaceutical research.  However, the residence concept as presented in most papers is supported by rather misleading simulations and arguments, and by examples where compds. are taken out of their pharmacokinetic context.  Moreover, fast assocn. is typically more desirable than slow, and advantages of long residence time, notably a potential disconnect between pharmacodynamics (PD) and pharmacokinetics (PK), would be partially or completely offset by slow on-rate.  Therefore, plain potency is likely a better predictor of drug development success than is residence time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5NQgx9uuufLVg90H21EOLACvtfcHk0lgSR9EBRzeKug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSitbjI&md5=61f68447e7f8eaf6b93c4b09fc5afeda</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.07.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.07.016%26sid%3Dliteratum%253Aachs%26aulast%3DFolmer%26aufirst%3DR.%2BH.%2BA.%26atitle%3DDrug%2520target%2520residence%2520time%253A%2520A%2520misleading%2520concept%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D12%26epage%3D16%26doi%3D10.1016%2Fj.drudis.2017.07.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span> <span> </span><span class="NLM_article-title">On-rate based optimization of structure-kinetic relationship--surfing the kinetic map</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2015.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.ddtec.2015.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=26724331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=9-15&author=A.+Schoopauthor=F.+Dey&title=On-rate+based+optimization+of+structure-kinetic+relationship%2D%2Dsurfing+the+kinetic+map&doi=10.1016%2Fj.ddtec.2015.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">On-rate based optimization of structure-kinetic relationship--surfing the kinetic map</span></div><div class="casAuthors">Schoop Andreas; Dey Fabian</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In the lead discovery process residence time has become an important parameter for the identification and characterization of the most efficacious compounds in vivo.  To enable the success of compound optimization by medicinal chemistry toward a desired residence time the understanding of structure-kinetic relationship (SKR) is essential.  This article reviews various approaches to monitor SKR and suggests using the on-rate as the key monitoring parameter.  The literature is reviewed and examples of compound series with low variability as well as with significant changes in on-rates are highlighted.  Furthermore, findings of kinetic on-rate changes are presented and potential underlying rationales are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwRMy18rtTE3T4ugvCDgnZfW6udTcc2eYfkqtkMrEL6Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28rms1egtw%253D%253D&md5=a7f0dee9cd02f38a945123e7faa9bc1c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2015.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2015.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DSchoop%26aufirst%3DA.%26aulast%3DDey%26aufirst%3DF.%26atitle%3DOn-rate%2520based%2520optimization%2520of%2520structure-kinetic%2520relationship--surfing%2520the%2520kinetic%2520map%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2015%26volume%3D17%26spage%3D9%26epage%3D15%26doi%3D10.1016%2Fj.ddtec.2015.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pompliano, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meek, T. D.</span></span> <span> </span><span class="NLM_article-title">Drug-target residence time and its implications for lead optimization</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">730</span>– <span class="NLM_lpage">739</span>, <span class="refDoi"> DOI: 10.1038/nrd2082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1038%2Fnrd2082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=16888652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=730-739&author=R.+A.+Copelandauthor=D.+L.+Pomplianoauthor=T.+D.+Meek&title=Drug-target+residence+time+and+its+implications+for+lead+optimization&doi=10.1038%2Fnrd2082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target residence time and its implications for lead optimization</span></div><div class="casAuthors">Copeland, Robert A.; Pompliano, David L.; Meek, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">730-739</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Much of drug discovery today is predicated on the concept of selective targeting of particular bioactive macromols. by low-mol.-mass drugs.  The binding of drugs to their macromol. targets is therefore seen as paramount for pharmacol. activity.  In vitro assessment of drug-target interactions is classically quantified in terms of binding parameters such as IC50 or Kd.  This article presents an alternative perspective on drug optimization in terms of drug-target binary complex residence time, as quantified by the dissociative half-life of the drug-target binary complex.  We describe the potential advantages of long residence time in terms of duration of pharmacol. effect and target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry6SJrMlrrTbVg90H21EOLACvtfcHk0lgA1Pd3mCXPeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XptVCltro%253D&md5=60ede2301584b10ac4e8fa18e1e6d107</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnrd2082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2082%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DPompliano%26aufirst%3DD.%2BL.%26aulast%3DMeek%26aufirst%3DT.%2BD.%26atitle%3DDrug-target%2520residence%2520time%2520and%2520its%2520implications%2520for%2520lead%2520optimization%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D730%26epage%3D739%26doi%3D10.1038%2Fnrd2082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunn, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, P.</span></span> <span> </span><span class="NLM_article-title">Investigation of the effect of molecular properties on the binding kinetics of a ligand to its biological target</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1039/c2md00270a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1039%2Fc2md00270a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC38Xks1Orsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=449-452&author=D.+C.+Millerauthor=G.+Lunnauthor=P.+Jonesauthor=Y.+Sabnisauthor=N.+Daviesauthor=P.+Driscoll&title=Investigation+of+the+effect+of+molecular+properties+on+the+binding+kinetics+of+a+ligand+to+its+biological+target&doi=10.1039%2Fc2md00270a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of the effect of molecular properties on the binding kinetics of a ligand to its biological target</span></div><div class="casAuthors">Miller, Duncan C.; Lunn, Graham; Jones, Peter; Sabnis, Yogesh; Davies, Nichola L.; Driscoll, Paul</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-452</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A database of binding kinetic data across multiple targets has been created.  Anal. of this database provides insights into trends for the effects of physicochem. properties (mol. wt., clogP, rotatable bond count) on the dissocn. kinetics of ligands from their biol. targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrvjiZtiu9VLVg90H21EOLACvtfcHk0lgA1Pd3mCXPeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xks1Orsro%253D&md5=84c6b9d95ec6df61145fe678c78d5491</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1039%2Fc2md00270a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2md00270a%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DLunn%26aufirst%3DG.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DDavies%26aufirst%3DN.%26aulast%3DDriscoll%26aufirst%3DP.%26atitle%3DInvestigation%2520of%2520the%2520effect%2520of%2520molecular%2520properties%2520on%2520the%2520binding%2520kinetics%2520of%2520a%2520ligand%2520to%2520its%2520biological%2520target%26jtitle%3DMedChemComm%26date%3D2012%26volume%3D3%26spage%3D449%26epage%3D452%26doi%3D10.1039%2Fc2md00270a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, P.</span></span> <span> </span><span class="NLM_article-title">How fast is protein-ligand association?</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2017.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.tibs.2017.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=28917971" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgtbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=847-849&author=S.+Gianniauthor=P.+Jemth&title=How+fast+is+protein-ligand+association%3F&doi=10.1016%2Fj.tibs.2017.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">How Fast Is Protein-Ligand Association?</span></div><div class="casAuthors">Gianni, Stefano; Jemth, Per</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-849</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">There is increasing interest in studying protein interactions and their role in cell biol. using kinetics.  However, there is confusion about the proper terminol. in terms of the distinction between rates and rate consts.  We recommend a more stringent use of the words speed, fast, slow, rate, and rate const.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruENTH_GopPLVg90H21EOLACvtfcHk0lijoOGOrf4B3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgtbrF&md5=6707642cbeb3bbd645c49d3ca667b735</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2017.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2017.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DGianni%26aufirst%3DS.%26aulast%3DJemth%26aufirst%3DP.%26atitle%3DHow%2520fast%2520is%2520protein-ligand%2520association%253F%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2017%26volume%3D42%26spage%3D847%26epage%3D849%26doi%3D10.1016%2Fj.tibs.2017.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer generation with OMEGA: Algorithm and validation using high quality structures from the protein databank and cambridge structural database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+Algorithm+and+validation+using+high+quality+structures+from+the+protein+databank+and+cambridge+structural+database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lijoOGOrf4B3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520protein%2520databank%2520and%2520cambridge%2520structural%2520database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boström, J.</span></span> <span> </span><span class="NLM_article-title">Reproducing the conformations of protein-bound ligands: A critical evaluation of several popular conformational searching tools</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1152</span>, <span class="refDoi"> DOI: 10.1023/A:1015930826903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1023%2FA%3A1015930826903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=12160095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A280%3ADC%252BD38vgsF2rtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=1137-1152&author=J.+Bostr%C3%B6m&title=Reproducing+the+conformations+of+protein-bound+ligands%3A+A+critical+evaluation+of+several+popular+conformational+searching+tools&doi=10.1023%2FA%3A1015930826903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Reproducing the conformations of protein-bound ligands: a critical evaluation of several popular conformational searching tools</span></div><div class="casAuthors">Bostrom J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of computer-aided molecular design</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1137-52</span>
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">Several programs (Catalyst, Confort, Flo99, MacroModel, and Omega) that are commonly used to generate conformational ensembles have been tested for their ability to reproduce bioactive conformations.  The ligands from thirty-two different ligand-protein complexes determined by high-resolution (< 2.0 A) X-ray crystallography have been analyzed.  The Low-Mode Conformational Search method (with AMBER* and the GB/SA hydration model), as implemented in MacroModel, was found to perform better than the other algorithms.  The rule-based method Omega, which is orders of magnitude faster than the other methods, also gave reasonable results but were found to be dependent on the input structure.  The methods supporting diverse sampling (Catalyst, Confort) performed least well.  For the seven ligands in the set having eight or more rotatable bonds, none of the bioactive conformations were ever found, save for one exception (Flo99).  These ligands do not bind in a local minimum conformation according to AMBER*\GB/SA.  Taking these last two observations together, it is clear that geometrically similar structures should be collected in order to increase the probability of finding the bioactive conformation among the generated ensembles.  Factors influencing bioactive conformational retrieval have been identified and are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSG5txM82Py66cY7HFp97qMfW6udTcc2eYZv60pvcCJ07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38vgsF2rtw%253D%253D&md5=5b08faf2b096c32cf8cda40f3b8218d0</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1023%2FA%3A1015930826903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1015930826903%26sid%3Dliteratum%253Aachs%26aulast%3DBostr%25C3%25B6m%26aufirst%3DJ.%26atitle%3DReproducing%2520the%2520conformations%2520of%2520protein-bound%2520ligands%253A%2520A%2520critical%2520evaluation%2520of%2520several%2520popular%2520conformational%2520searching%2520tools%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2001%26volume%3D15%26spage%3D1137%26epage%3D1152%26doi%3D10.1023%2FA%3A1015930826903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbs, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Izrailev, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational sampling of bioactive molecules: A comparative study</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1086</span>, <span class="refDoi"> DOI: 10.1021/ci6005454</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci6005454" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjvFKgt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1067-1086&author=D.+K.+Agrafiotisauthor=A.+C.+Gibbsauthor=F.+Zhuauthor=S.+Izrailevauthor=E.+Martin&title=Conformational+sampling+of+bioactive+molecules%3A+A+comparative+study&doi=10.1021%2Fci6005454"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational sampling of bioactive molecules: a comparative study</span></div><div class="casAuthors">Agrafiotis, Dimitris K.; Gibbs, Alan C.; Zhu, Fangqiang; Izrailev, Sergei; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1067-1086</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The necessity to generate conformations that sample the entire conformational space accessible to a given mol. is ubiquitous in the field of computer-aided drug design.  Protein-ligand docking, 3D database searching, and 3D QSAR are three commonly used techniques that depend critically upon the quality and diversity of the generated conformers.  Although there are a wide range of conformational search algorithms available, the extent to which they sample conformational space is often unclear.  To address this question, we conducted a robust comparison of the search algorithms implemented in several widely used mol. modeling packages, including Catalyst, Macromodel, Omega, MOE, and Rubicon as well as our own method, stochastic proximity embedding (SPE).  We found that SPE used in conjunction with conformational boosting, a heuristic for biasing conformational search toward more extended or compact geometries, along with Catalyst, are significantly more effective in sampling the full range of conformational space compared to the other methods, which show distinct preferences for either more extended or more compact geometries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYXaKrPdkIBrVg90H21EOLACvtfcHk0lgMvkECfplkZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjvFKgt7s%253D&md5=ef380e9f957b8e7188a6c92ee61ab098</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fci6005454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6005454%26sid%3Dliteratum%253Aachs%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DGibbs%26aufirst%3DA.%2BC.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DIzrailev%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520sampling%2520of%2520bioactive%2520molecules%253A%2520A%2520comparative%2520study%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1067%26epage%3D1086%26doi%3D10.1021%2Fci6005454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">LowModeMD--implicit low-mode velocity filtering applied to conformational search of macrocycles and protein loops</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1021/ci900508k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900508k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=792-800&author=P.+Labute&title=LowModeMD%2D%2Dimplicit+low-mode+velocity+filtering+applied+to+conformational+search+of+macrocycles+and+protein+loops&doi=10.1021%2Fci900508k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">LowModeMD - Implicit Low-Mode Velocity Filtering Applied to Conformational Search of Macrocycles and Protein Loops</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">792-800</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for conformational search of complex mol. systems such as macrocycles and protein loops.  The method is based on perturbing an existing conformation along a mol. dynamics trajectory using initial at. velocities with kinetic energy concd. on the low-frequency vibrational modes, followed by energy minimization.  A novel Chebyshev polynomial filter is used to heavily dampen the high-frequency components of a randomly generated Maxwell-Boltzmann velocity vector.  The method is very efficient, even for large systems; it is straightforward to implement and requires only std. force-field energy and gradient evaluations.  The results of several computational expts. suggest that the method is capable of efficiently sampling low-strain energy conformations of complex systems with nontrivial nonbonded interaction networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfWZjZhFqqUbVg90H21EOLACvtfcHk0lh5X9wcGb4D-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D&md5=bb800eb2f9635a661578b978dae9da0a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fci900508k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900508k%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DLowModeMD--implicit%2520low-mode%2520velocity%2520filtering%2520applied%2520to%2520conformational%2520search%2520of%2520macrocycles%2520and%2520protein%2520loops%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D792%26epage%3D800%26doi%3D10.1021%2Fci900508k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minehardt, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Car, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J. D.</span></span> <span> </span><span class="NLM_article-title">Protonation-induced stereoisomerism in nicotine: conformational studies using classical (AMBER) and ab initio (Car-Parrinello) molecular dynamics</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1007/s10822-005-0096-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1007%2Fs10822-005-0096-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=16059663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkt1Ons7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=1-15&author=P.+S.+Hammondauthor=Y.+Wuauthor=R.+Harrisauthor=T.+J.+Minehardtauthor=R.+Carauthor=J.+D.+Schmitt&title=Protonation-induced+stereoisomerism+in+nicotine%3A+conformational+studies+using+classical+%28AMBER%29+and+ab+initio+%28Car-Parrinello%29+molecular+dynamics&doi=10.1007%2Fs10822-005-0096-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Protonation-induced stereoisomerism in nicotine: Conformational studies using classical (AMBER) and ab initio (Car-Parrinello) molecular dynamics</span></div><div class="casAuthors">Hammond, Philip S.; Wu, Yudong; Harris, Rebecca; Minehardt, Todd J.; Car, Roberto; Schmitt, Jeffrey D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A variety of biol. active small mols. contain prochiral tertiary amines, which become chiral centers upon protonation.  S-nicotine, the prototypical nicotinic acetylcholine receptor agonist, produces two diastereomers on protonation.  Results, using both classical (AMBER) and ab initio (Car-Parrinello) mol. dynamical studies, illustrate the significant differences in conformational space explored by each diastereomer.  As is expected, this phenomenon has an appreciable effect on nicotine's energy hypersurface and leads to differentiation in mol. shape and divergent sampling.  Thus, protonation induced isomerism can produce dynamic effects that may influence the behavior of a mol. in its interaction with a target protein.  We also examine differences in the conformational dynamics for each diastereomer as quantified by both mol. dynamics methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_oE3cUQDNcrVg90H21EOLACvtfcHk0lh5X9wcGb4D-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXkt1Ons7k%253D&md5=f38bcd45b40a07f015ce82bf8af973c8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs10822-005-0096-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-005-0096-7%26sid%3Dliteratum%253Aachs%26aulast%3DHammond%26aufirst%3DP.%2BS.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DMinehardt%26aufirst%3DT.%2BJ.%26aulast%3DCar%26aufirst%3DR.%26aulast%3DSchmitt%26aufirst%3DJ.%2BD.%26atitle%3DProtonation-induced%2520stereoisomerism%2520in%2520nicotine%253A%2520conformational%2520studies%2520using%2520classical%2520%2528AMBER%2529%2520and%2520ab%2520initio%2520%2528Car-Parrinello%2529%2520molecular%2520dynamics%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2005%26volume%3D19%26spage%3D1%26epage%3D15%26doi%3D10.1007%2Fs10822-005-0096-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational analysis of macrocycles: finding what common search methods miss</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2242</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.1021/ci900238a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900238a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2242-2259&author=P.+Bonnetauthor=D.+K.+Agrafiotisauthor=F.+Zhuauthor=E.+Martin&title=Conformational+analysis+of+macrocycles%3A+finding+what+common+search+methods+miss&doi=10.1021%2Fci900238a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span></div><div class="casAuthors">Bonnet, Pascal; Agrafiotis, Dimitris K.; Zhu, Fangqiang; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2242-2259</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As computational drug design becomes increasingly reliant on virtual screening and on high-throughput 3D modeling, the need for fast, robust, and reliable methods for sampling mol. conformations has become greater than ever.  Furthermore, chem. novelty is at a premium, forcing medicinal chemists to explore more complex structural motifs and unusual topologies.  This necessitates the use of conformational sampling techniques that work well in all cases.  Here, we compare the performance of several popular conformational search algorithms on three broad classes of macrocyclic mols.  These methods include Catalyst, CAESAR, MacroModel, MOE, Omega, Rubicon and two newer self-organizing algorithms known as stochastic proximity embedding (SPE) and self-organizing superimposition (SOS) that have been developed at Johnson & Johnson.  Our results show a compelling advantage for the three distance geometry methods (SOS, SPE, and Rubicon) followed to a lesser extent by MacroModel.  The remaining techniques, particularly those based on systematic search, often failed to identify any of the lowest energy conformations and are unsuitable for this class of structures.  Taken together with our previous study on drug-like mols., these results suggest that SPE and SOS are two of the most robust and universally applicable conformational search methods, with the latter being preferred because of its superior speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIMu1b7U477Vg90H21EOLACvtfcHk0lh5X9wcGb4D-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF&md5=9b4a5919415ef0660c4f9ff18d9b76f4</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fci900238a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900238a%26sid%3Dliteratum%253Aachs%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520analysis%2520of%2520macrocycles%253A%2520finding%2520what%2520common%2520search%2520methods%2520miss%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2242%26epage%3D2259%26doi%3D10.1021%2Fci900238a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span> <span> </span><span class="NLM_article-title">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7898</span>– <span class="NLM_lpage">7920</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmc.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=24184215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7898-7920&author=I.+J.+Chenauthor=N.+Foloppe&title=Tackling+the+conformational+sampling+of+larger+flexible+compounds+and+macrocycles+in+pharmacology+and+drug+discovery&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span></div><div class="casAuthors">Chen, I-Jen; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7920</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational conformational sampling underpins much of mol. modeling and design in pharmaceutical work.  The sampling of smaller drug-like compds. has been an active area of research.  However, few studies have tested in details the sampling of larger more flexible compds., which are also relevant to drug discovery, including therapeutic peptides, macrocycles, and inhibitors of protein-protein interactions.  Here, we investigate extensively mainstream conformational sampling methods on three carefully curated compd. sets, namely the 'Drug-like', larger 'Flexible', and 'Macrocycle' compds.  These test mols. are chem. diverse with reliable X-ray protein-bound bioactive structures.  The compared sampling methods include Stochastic Search and the recent LowModeMD from MOE, all the low-mode based approaches from MacroModel, and MD/LLMOD recently developed for macrocycles.  In addn. to default settings, key parameters of the sampling protocols were explored.  The performance of the computational protocols was assessed via (i) the reprodn. of the X-ray bioactive structures, (ii) the size, coverage and diversity of the output conformational ensembles, (iii) the compactness/extendedness of the conformers, and (iv) the ability to locate the global energy min.  The influence of the stochastic nature of the searches on the results was also examd.  Much better results were obtained by adopting search parameters enhanced over the default settings, while maintaining computational tractability.  In MOE, the recent LowModeMD emerged as the method of choice.  Mixed torsional/low-mode from MacroModel performed as well as LowModeMD, and MD/LLMOD performed well for macrocycles.  The low-mode based approaches yielded very encouraging results with the flexible and macrocycle sets.  Thus, one can productively tackle the computational conformational search of larger flexible compds. for drug discovery, including macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwhGIpE1mMbVg90H21EOLACvtfcHk0ljegWKs0nZQ7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK&md5=2699227e69410a5673dec749169fd578</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.%2BJ.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DTackling%2520the%2520conformational%2520sampling%2520of%2520larger%2520flexible%2520compounds%2520and%2520macrocycles%2520in%2520pharmacology%2520and%2520drug%2520discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7898%26epage%3D7920%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spronk, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span> <span> </span><span class="NLM_article-title">Improving accuracy, diversity, and speed with prime macrocycle conformational sampling</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1881</span>– <span class="NLM_lpage">1894</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1881-1894&author=D.+Sindhikaraauthor=S.+A.+Spronkauthor=T.+Dayauthor=K.+Borrelliauthor=D.+L.+Cheneyauthor=S.+L.+Posy&title=Improving+accuracy%2C+diversity%2C+and+speed+with+prime+macrocycle+conformational+sampling&doi=10.1021%2Facs.jcim.7b00052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Improving Accuracy, Diversity, and Speed with Prime Macrocycle Conformational Sampling</span></div><div class="casAuthors">Sindhikara, Dan; Spronk, Steven A.; Day, Tyler; Borrelli, Ken; Cheney, Daniel L.; Posy, Shana L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1881-1894</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for exploring macrocycle conformational space, Prime Macrocycle Conformational Sampling (Prime-MCS), is introduced and evaluated in the context of other available algorithms (Mol. Dynamics, LowModeMD in MOE, and MacroModel Baseline Search).  The algorithms were benchmarked on a dataset of 208 macrocycles which was curated for diversity from the Cambridge Structural Database, the Protein Databank, and the Biol. Interesting Mol. Ref. Dictionary.  The algorithms were evaluated in terms of accuracy (ability to reproduce the crystal structure), diversity (coverage of conformational space), and computational speed.  Prime-MCS most reliably reproduced crystallog. structures for RMSD thresholds >1.0 Å, most often produced the most diverse conformational ensemble, and was most often the fastest algorithm.  Detailed anal. and examn. of both typical and outlier cases were performed to reveal characteristics, shortcomings, expected performance, and complementarity of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfeDx3IcpZD7Vg90H21EOLACvtfcHk0ljegWKs0nZQ7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ&md5=a2b20d25c2ad2cede1c3a5fd0b5840ce</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00052%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DSpronk%26aufirst%3DS.%2BA.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DBorrelli%26aufirst%3DK.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26atitle%3DImproving%2520accuracy%252C%2520diversity%252C%2520and%2520speed%2520with%2520prime%2520macrocycle%2520conformational%2520sampling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D1881%26epage%3D1894%26doi%3D10.1021%2Facs.jcim.7b00052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0ljegWKs0nZQ7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span> <span> </span><span class="NLM_article-title">Defining optimum lipophilicity and molecular weight ranges for drug candidates-Molecular weight dependent lower logD limits based on permeability</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2844</span>– <span class="NLM_lpage">2851</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=10.1016%2Fj.bmcl.2009.03.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=19361989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltl2rtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2844-2851&author=M.+J.+Waring&title=Defining+optimum+lipophilicity+and+molecular+weight+ranges+for+drug+candidates-Molecular+weight+dependent+lower+logD+limits+based+on+permeability&doi=10.1016%2Fj.bmcl.2009.03.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Defining optimum lipophilicity and molecular weight ranges for drug candidates-Molecular weight dependent lower log D limits based on permeability</span></div><div class="casAuthors">Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2844-2851</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Anal. of a large, structurally diverse Caco-2 permeability dataset using a variety of statistical techniques suggests that log D and mol. wt. are the most important factors in detg. the permeability of drug candidates.  The limit for log D is shown to be dependent on mol. wt.  These limits are shown to be potentially superior to current guidelines in increasing the chances of finding highly permeable compds.  When combined with suggested upper limits for lipophilicity suggested in the literature based on the avoidance of toxicol. and other adverse effects, this helps define a lipophilicity range that is optimum for drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXwn3KyrzERrVg90H21EOLACvtfcHk0ljZFSH7yKErJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltl2rtr0%253D&md5=1f04795ddf6e6b84d132a81e554c2516</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.109%26sid%3Dliteratum%253Aachs%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DDefining%2520optimum%2520lipophilicity%2520and%2520molecular%2520weight%2520ranges%2520for%2520drug%2520candidates-Molecular%2520weight%2520dependent%2520lower%2520logD%2520limits%2520based%2520on%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2844%26epage%3D2851%26doi%3D10.1016%2Fj.bmcl.2009.03.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span> <i>Small-Molecule
Drug Discovery Suite 2018-4</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Small-Molecule%0ADrug+Discovery+Suite+2018-4%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C+NY%2C+2018."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSmall-Molecule%250ADrug%2520Discovery%2520Suite%25202018-4%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS0','PDB','6FS0'); return false;">PDB: 6FS0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KU9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5KU9','PDB','5KU9'); return false;">PDB: 5KU9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MEV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MEV','PDB','5MEV'); return false;">PDB: 5MEV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6FS1','PDB','6FS1'); return false;">PDB: 6FS1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N21','PDB','5N21'); return false;">PDB: 5N21</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MES" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MES','PDB','5MES'); return false;">PDB: 5MES</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N1Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5N1Z','PDB','5N1Z'); return false;">PDB: 5N1Z</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i41"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00716">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_48503"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00716">10.1021/acs.jmedchem.9b00716</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary figures and tables as referred to in the article (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biophysical data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_001.pdf">jm9b00716_si_001.pdf (1.1 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00716/suppl_file/jm9b00716_si_002.csv">jm9b00716_si_002.csv (1.68 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00716%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-21%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00716" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994a530fe73d00","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
