
<html lang="en"     class="pb-page"  data-request-id="17cbefcd-809c-482f-9be5-9c132167d584"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02152;issue:issue:10.1021/jmcmar.2021.64.issue-8;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Design of Selective LONP1 Inhibitors for Probing In Vitro Biology" /></meta><meta name="dc.Creator" content="Laura J.  Kingsley" /></meta><meta name="dc.Creator" content="Xiaohui  He" /></meta><meta name="dc.Creator" content="Matthew  McNeill" /></meta><meta name="dc.Creator" content="John  Nelson" /></meta><meta name="dc.Creator" content="Victor  Nikulin" /></meta><meta name="dc.Creator" content="Zhiwei  Ma" /></meta><meta name="dc.Creator" content="Wenshuo  Lu" /></meta><meta name="dc.Creator" content="Vicki W.  Zhou" /></meta><meta name="dc.Creator" content="Mari  Manuia" /></meta><meta name="dc.Creator" content="Andreas  Kreusch" /></meta><meta name="dc.Creator" content="Mu-Yun  Gao" /></meta><meta name="dc.Creator" content="Darbi  Witmer" /></meta><meta name="dc.Creator" content="Mei-Ting  Vaillancourt" /></meta><meta name="dc.Creator" content="Min  Lu" /></meta><meta name="dc.Creator" content="Sarah  Greenblatt" /></meta><meta name="dc.Creator" content="Christian  Lee" /></meta><meta name="dc.Creator" content="Ajay  Vashisht" /></meta><meta name="dc.Creator" content="Steven  Bender" /></meta><meta name="dc.Creator" content="Glen  Spraggon" /></meta><meta name="dc.Creator" content="Pierre-Yves  Michellys" /></meta><meta name="dc.Creator" content="Yong  Jia" /></meta><meta name="dc.Creator" content="Jacob R.  Haling" /></meta><meta name="dc.Creator" content="Gérald  Lelais" /></meta><meta name="dc.Description" content="LONP1 is an AAA+ protease that maintains mitochondrial homeostasis by removing damaged or misfolded proteins. Elevated activity and expression of LONP1 promotes cancer cell proliferation and resist..." /></meta><meta name="Description" content="LONP1 is an AAA+ protease that maintains mitochondrial homeostasis by removing damaged or misfolded proteins. Elevated activity and expression of LONP1 promotes cancer cell proliferation and resist..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 6, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02152" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02152" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02152" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02152" /></link>
        
    
    

<title>Structure-Based Design of Selective LONP1 Inhibitors for Probing In Vitro Biology | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02152" /></meta><meta property="og:title" content="Structure-Based Design of Selective LONP1 Inhibitors for Probing In Vitro Biology" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0007.jpeg" /></meta><meta property="og:description" content="LONP1 is an AAA+ protease that maintains mitochondrial homeostasis by removing damaged or misfolded proteins. Elevated activity and expression of LONP1 promotes cancer cell proliferation and resistance to apoptosis-inducing reagents. Despite the importance of LONP1 in human biology and disease, very few LONP1 inhibitors have been described in the literature. Herein, we report the development of selective boronic acid-based LONP1 inhibitors using structure-based drug design as well as the first structures of human LONP1 bound to various inhibitors. Our efforts led to several nanomolar LONP1 inhibitors with little to no activity against the 20S proteasome that serve as tool compounds to investigate LONP1 biology." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02152"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02152">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02152&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02152&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02152&amp;href=/doi/10.1021/acs.jmedchem.0c02152" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 4857-4869</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02213" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02247" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Design of Selective LONP1 Inhibitors for Probing <i>In Vitro</i> Biology</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Laura J. Kingsley</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura J. Kingsley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#e985889c9b88c78280878e9a858c90a9848c9b8a82c78a8684"><span class="__cf_email__" data-cfemail="ec808d999e8dc28785828b9f808995ac81899e8f87c28f8381">[email protected]</span></a>. Phone: +1-628-444-7236.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura+J.++Kingsley">Laura J. Kingsley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaohui He</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaohui He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaohui++He">Xiaohui He</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1003-2631" title="Orcid link">http://orcid.org/0000-0003-1003-2631</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew McNeill</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew McNeill</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew++McNeill">Matthew McNeill</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John Nelson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John Nelson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John++Nelson">John Nelson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Victor Nikulin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Victor Nikulin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Victor++Nikulin">Victor Nikulin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhiwei Ma</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhiwei Ma</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhiwei++Ma">Zhiwei Ma</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wenshuo Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenshuo Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenshuo++Lu">Wenshuo Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vicki W. Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vicki W. Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vicki+W.++Zhou">Vicki W. Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mari Manuia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mari Manuia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mari++Manuia">Mari Manuia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Kreusch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Kreusch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Kreusch">Andreas Kreusch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mu-Yun Gao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mu-Yun Gao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mu-Yun++Gao">Mu-Yun Gao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darbi Witmer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darbi Witmer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darbi++Witmer">Darbi Witmer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mei-Ting Vaillancourt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mei-Ting Vaillancourt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mei-Ting++Vaillancourt">Mei-Ting Vaillancourt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Min Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Min Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Min++Lu">Min Lu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sarah Greenblatt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sarah Greenblatt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sarah++Greenblatt">Sarah Greenblatt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Lee">Christian Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ajay Vashisht</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ajay Vashisht</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ajay++Vashisht">Ajay Vashisht</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steven Bender</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven Bender</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven++Bender">Steven Bender</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Glen Spraggon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Glen Spraggon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Glen++Spraggon">Glen Spraggon</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pierre-Yves Michellys</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pierre-Yves Michellys</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pierre-Yves++Michellys">Pierre-Yves Michellys</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yong Jia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yong Jia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yong++Jia">Yong Jia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jacob R. Haling</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jacob R. Haling</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jacob+R.++Haling">Jacob R. Haling</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Gérald Lelais</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gérald Lelais</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Genomics Institute of the Novartis Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a5c2c9c0c9c4ccd6e5c2cbc38bcad7c2"><span class="__cf_email__" data-cfemail="c2a5aea7aea3abb182a5aca4ecadb0a5">[email protected]</span></a>. Phone: +1-858-332-4452.</span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=G%C3%A9rald++Lelais">Gérald Lelais</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6213-9863" title="Orcid link">http://orcid.org/0000-0002-6213-9863</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02152&amp;href=/doi/10.1021%2Facs.jmedchem.0c02152" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 4857–4869</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 6, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 December 2020</li><li><span class="item_label"><b>Published</b> online</span>6 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 April 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02152" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02152</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4857%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLaura%2BJ.%2BKingsley%252C%2BXiaohui%2BHe%252C%2BMatthew%2BMcNeill%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D8%26contentID%3Dacs.jmedchem.0c02152%26title%3DStructure-Based%2BDesign%2Bof%2BSelective%2BLONP1%2BInhibitors%2Bfor%2BProbing%2BIn%2BVitro%2BBiology%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4869%26publicationDate%3DApril%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02152"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">800</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02152" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Design of Selective LONP1 Inhibitors for Probing In Vitro Biology&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;J. Kingsley&quot;},{&quot;first_name&quot;:&quot;Xiaohui&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;McNeill&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Nelson&quot;},{&quot;first_name&quot;:&quot;Victor&quot;,&quot;last_name&quot;:&quot;Nikulin&quot;},{&quot;first_name&quot;:&quot;Zhiwei&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Wenshuo&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Vicki&quot;,&quot;last_name&quot;:&quot;W. Zhou&quot;},{&quot;first_name&quot;:&quot;Mari&quot;,&quot;last_name&quot;:&quot;Manuia&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Kreusch&quot;},{&quot;first_name&quot;:&quot;Mu-Yun&quot;,&quot;last_name&quot;:&quot;Gao&quot;},{&quot;first_name&quot;:&quot;Darbi&quot;,&quot;last_name&quot;:&quot;Witmer&quot;},{&quot;first_name&quot;:&quot;Mei-Ting&quot;,&quot;last_name&quot;:&quot;Vaillancourt&quot;},{&quot;first_name&quot;:&quot;Min&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;Greenblatt&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Ajay&quot;,&quot;last_name&quot;:&quot;Vashisht&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;Bender&quot;},{&quot;first_name&quot;:&quot;Glen&quot;,&quot;last_name&quot;:&quot;Spraggon&quot;},{&quot;first_name&quot;:&quot;Pierre-Yves&quot;,&quot;last_name&quot;:&quot;Michellys&quot;},{&quot;first_name&quot;:&quot;Yong&quot;,&quot;last_name&quot;:&quot;Jia&quot;},{&quot;first_name&quot;:&quot;Jacob&quot;,&quot;last_name&quot;:&quot;R. Haling&quot;},{&quot;first_name&quot;:&quot;Gérald&quot;,&quot;last_name&quot;:&quot;Lelais&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;4857-4869&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02152&quot;},&quot;abstract&quot;:&quot;LONP1 is an AAA+ protease that maintains mitochondrial homeostasis by removing damaged or misfolded proteins. Elevated activity and expression of LONP1 promotes cancer cell proliferation and resistance to apoptosis-inducing reagents. Despite the importance of LONP1 in human biology and disease, very few LONP1 inhibitors have been described in the literature. Herein, we report the development of selective boronic acid-based LONP1 inhibitors using structure-based drug design as well as the first structures of human LONP1 bound to various inhibitors. Our efforts led to several nanomolar LONP1 inhibitors with little to no activity against the 20S proteasome that serve as tool compounds to investigate LONP1 biology.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02152&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02152" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02152&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02152" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02152&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02152" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02152&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02152&amp;href=/doi/10.1021/acs.jmedchem.0c02152" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02152" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02152" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02152%26sid%3Dliteratum%253Aachs%26pmid%3D33821636%26genre%3Darticle%26aulast%3DKingsley%26date%3D2021%26atitle%3DStructure-Based%2BDesign%2Bof%2BSelective%2BLONP1%2BInhibitors%2Bfor%2BProbing%2BIn%2BVitro%2BBiology%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D8%26spage%3D4857%26epage%3D4869%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (6)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/jmcmar.2021.64.issue-8/20210422/jmcmar.2021.64.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">LONP1 is an AAA+ protease that maintains mitochondrial homeostasis by removing damaged or misfolded proteins. Elevated activity and expression of LONP1 promotes cancer cell proliferation and resistance to apoptosis-inducing reagents. Despite the importance of LONP1 in human biology and disease, very few LONP1 inhibitors have been described in the literature. Herein, we report the development of selective boronic acid-based LONP1 inhibitors using structure-based drug design as well as the first structures of human LONP1 bound to various inhibitors. Our efforts led to several nanomolar LONP1 inhibitors with little to no activity against the 20S proteasome that serve as tool compounds to investigate LONP1 biology.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33401" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33401" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">LONP1 is a mitochondrial serine protease that is widely conserved across eukaryotic species. LON proteases along with ClpXP, ClpAP, ClpCP, HslUV, FtsH, and PAN/20S make up the AAA+ (ATPase associated with diverse cellular activities) family of proteases.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> This family of proteases is responsible for maintaining cellular homeostasis by removing unwanted or damaged proteins within the cell.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> As the name suggests, each member of this family contains at least one AAA+ domain, which utilizes ATP as an energy source to unfold substrate proteins. AAA+ proteases function as multimers, forming barrel-like structures through which target proteins are forcibly threaded prior to degradation.<a onclick="showRef(event, 'ref1 ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref1 ref5 ref6 ref7">(1,5−7)</a> While physical seclusion of the active site in AAA+ proteases provides some level of control, substrate selection and regulation of AAA+ proteases are still poorly understood.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Recent structural studies on the LON family member, LonA, have provided some insights into the structure, function, and regulation of the LON proteases. The LON proteases function as homo-hexamers in which each individual protein unit contains both the ATPase and protease functions.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> This contrasts with other family members where the ATPase and protease functions are found in different protein units. In LON proteases, the protease domain linker connects the ATPase to the protease domain and is thought to be critical for substrate unfolding as it allows for a large, ATP-driven conformational change that pulls substrate proteins into the protease active site.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Studies have indicated that in the bacterial MtaLonA, Mg<sup>2+</sup> is a key cofactor for LONP1 activation.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Additionally, a disulfide-controlled redox switch is thought to modulate LON activity by controlling the exit pore size; however, these key cysteines are not conserved in human LON proteases suggesting alternative regulation mechanism(s).<a onclick="showRef(event, 'ref6 ref10'); return false;" href="javascript:void(0);" class="ref ref6 ref10">(6,10)</a></div><div class="NLM_p">While the regulatory mechanisms of LON proteases, including LONP1, are not fully understood, it is clear that dysregulation of LONP1 leads to disease. Genetic studies have found that, although rare, mutations that impair the function of LONP1 lead to severe developmental complications in the cerebral, ocular, dental, auricular, and skeletal systems known as CODAS syndrome and other mitochondrial diseases.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Conversely, recent studies suggest that elevated expression of LONP1 plays a pro-tumorigenic role in several cancers including colorectal, cervical, and melanoma cancers.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a></div><div class="NLM_p last">Despite a growing understanding of the complex regulatory mechanisms of LONP1 and its importance in various disease states, LONP1-specific tool compounds are lacking. Most studies to date have relied on a limited set of non-selective LONP1 tool compounds including oleanane 2-cyano-3,12-dioxooleana-1,9-dien-28-oic acid and its derivatives, (−)-sesamin, obtusilactone A, and bortezomib.<a onclick="showRef(event, 'ref4 ref16'); return false;" href="javascript:void(0);" class="ref ref4 ref16">(4,16)</a> Furthermore, homozygous LONP1 knockouts are embryonically lethal, making it difficult to study the biological effects of a complete lack of LONP1.<a onclick="showRef(event, 'ref14 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref17">(14,17)</a> While silencing approaches have been used to help elucidate the roles of LONP1,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> they are often challenging to implement.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Coupling genetic strategies to pharmacological intervention can both offer deeper insights and pave the way for therapeutic strategy development. Recently, the importance of “probe” compounds has been highlighted in the literature as necessary to understand biology.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Herein, we report our efforts toward better understanding the structure–function relationships in LONP1 inhibition and the identification of specific LONP1 inhibitors that can be used to dissect the LONP1 biological function in cancer and other diseases.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43001" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43001" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Characterization of LONP1 Domains</h3><div class="NLM_p">To understand the cross-regulation between the ATPase and the protease domains of LONP1, we first studied the two domains independently. The ATPase domain (a.a. 408–752) alone retained ∼60% of ATPase activity compared to the full-length LONP1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A); in contrast, the protease domain alone (a.a. 754–959) was devoid of proteolytic activity, even when tested at 50-fold higher concentration than full-length LONP1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). We then attempted to reconstitute “full-length” LONP1 protease activity in trans by mixing the ATPase and protease domains at a 1:1 ratio, but this reconstituted protein demonstrated only minimal protease activity (0.14% of full-length LONP1 activity; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Our findings indicate that the protease activity is dependent on the AAA+ domain ATPase activity, but not vice versa, and the protease domain linker that connects the ATPase to the protease domain is critical for this intramolecular domain regulation.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Biochemical characterization of human LONP1. (A) ATPase activity determination of LONP1 full-length protein and ATPase domain-only construct (a.a. 408–752) using the ADP-Transcreener assay. (B) Protease activity determination of LONP1 full-length protein (triangles), protease domain-only construct (a.a 754–959) (circles), and mixture of ATPase domain and protease domain at 1:1 ratio (squares) using the LONP1 protease assay. (C) Full-length LONP1 characterization under various conditions: (a) in the presence of all components (defined as 100% control activity); (b) in the absence of enzyme (defined as background); (c) in the absence of ATP; (d) in the presence of excess bortezomib (10 μM; 55-fold above its IC<sub>50</sub>); and (e) in the absence of metal cofactor (CaCl<sub>2</sub>). (D) Kinetics of LONP1 activation in the presence of either 1 μM ATP or 1 μM AMP-PNP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To further understand the effect of ATPase domain on protease activity, we tested full-length LONP1 under various conditions. The protease activity was completely abolished in the absence of ATP (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C(c)) similar to the effect of full protease inhibition by bortezomib (10 μM; 55-fold above its IC<sub>50</sub>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C(d)). However, when the metal cofactor was removed while keeping ATP in the assay mixture, residual protease activity was observed (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C(e)), indicating that there exists a non-ATPase-dependent protease activity requiring binding of ATP to LONP1. To test if nucleotide binding, rather than hydrolysis, is necessary for protease activity, AMP-PNP was titrated into the LONP1 protease assay in the presence of 1 μM ATP. Interestingly, we observed a dose-dependent increase in LONP1 activation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Figure S1</a>), suggesting that binding alone to the ATP site is capable of promoting some protease activity. To confirm this, we replaced ATP with AMP-PNP in the assay. In the absence of both ATP and AMP-PNP, no protease activity was detected. On the other hand, AMP-PNP (1 μM, equivalent to ATP concentration used in the assay) was able to fully rescue the protease activity of LONP1, even though the kinetics of the two reactions were very different. Instead of showing a linear kinetic similar to ATP, a lag phase was observed for AMP-PNP (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D), indicating that binding of AMP-PNP likely induces a large conformational change of the protein, priming the protease domain into an active conformation for substrate degradation. These results suggest that ATP (or its analogues) occupying the ATP site, not the ATPase activity itself, is necessary for the protease activity of LONP1. Thus, from a drug discovery perspective, our results indicate that a protease inhibitor, but not an ATPase inhibitor, will be most effective for LONP1 inhibition. Our findings are consistent with what was described for prokaryotic evolutionarily conserved LonA from <i>Meiothermus taiwanensis</i>.<a onclick="showRef(event, 'ref9 ref21'); return false;" href="javascript:void(0);" class="ref ref9 ref21">(9,21)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Assay Design, Primary Hit Identification, and Chemistry</h3><div class="NLM_p">Our strategy for identifying selective LONP1 protease site inhibitors started with a comprehensive high-throughput screening (HTS) of the Novartis compound library using recombinant hexameric LONP1. The LONP1 biochemical assay was modified based on the assay reported by Fishovitz <i>et al</i>.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> In brief, the LONP1 HTS assay measures activity via cleavage of a substrate peptide. Initially, the substrate peptide is intramolecularly quenched, but cleavage by LONP1 results in the release of fluorescent fragments. Compound inhibition of LONP1 prevents peptide cleavage and therefore leads to a reduction in a fluorescent signal. Compounds were screened at a single-point concentration in a 1536-well format. The screen quality was high with a <i>Z</i>′-factor of 0.6. Compounds were designated as hits if the percent inhibition exceeded 3 standard deviations from the mean activity. The hit rate was 1.16%. Hits were confirmed using a 12-point dose response in the same assay.</div><div class="NLM_p">Boronic acid peptides were identified as one of the top series in the HTS. The members of this series, including bortezomib and ixazomib, have been previously reported to have nanomolar activity against two of the three subunits of the 20S proteasome and also LONP1 from various source organisms.<a onclick="showRef(event, 'ref9 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref9 ref23 ref24 ref25 ref26">(9,23−26)</a> A medicinal chemistry campaign was initiated to develop a bortezomib analogue devoid of 20S proteasome activity and identify tool compounds useful for investigating LONP1 biology. The analogues were prepared following known synthetic procedures (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> Two routes were used: on the one hand, amino acid tertbutyl esters (<b>1a–c</b>) were first acylated with pyrazine-2-carboxylic acid (<b>2</b>), then coupled with amino boronate esters, previously prepared through known procedures,<a onclick="showRef(event, 'ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32">(29−32)</a> and the resulting boronic esters were deprotected to afford the desired compounds (<b>5a–b</b>, <b>11a–f</b>, <b>12a</b>, and <b>12d–g</b>). Alternatively, Boc-protected amino acids were first coupled with amino boronate esters, Boc-deprotected, and coupled with the appropriate heteroaryl carboxylic acids before final boronate ester deprotection to afford the corresponding LONP1 inhibitors (<b>9a–i</b> and <b>14</b>).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Analogues Following Known Synthetic Procedures<a onclick="showRef(event, 'ref26 ref27 ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29 ref30 ref31 ref32">(26−32)</a> <sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIPEA, HATU, DMF or DCM, and −15 °C to room temperature. (b) HCl/dioxane. (c) For <b>5a–b</b>: <b>4</b>, TBTU, DIEA, DCM, and −15 °C to room temperature. For <b>9a–i</b>: <b>2</b>, HATU, DIPEA, DMF or DCM, and −15 °C to room temperature. For <b>12a</b> and <b>12d–g</b>: <b>10a</b> or <b>10d–g</b>, HATU, DIPEA, DMF or DCM, and −15 °C to room temperature. For <b>14</b>: 2,4-dimethyloxazole-5-carboxylic acid, HATU, DIPEA, DCM, and −15 °C to room temperature. (d) Isobutyl boronic acid, HCl, and heptane/MeOH.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Conserved Backbone Interactions Contribute to Lack of Specificity for Bortezomib</h3><div class="NLM_p">To better understand the structure–activity relationship (SAR) of bortezomib in the context of LONP1 specificity, we co-crystallized bortezomib with the LONP1 protease domain (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) and compared it to the previously reported<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> bortezomib-bound structure of the human 20S proteasome (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF3">5LF3</a>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). For clarity, structural comparisons in the main text between LONP1 and the 20S proteasome will refer specifically to the chymotrypsin-like (β5) site as it has the highest affinity for bortezomib;<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> however, comparisons to the caspase (β1) and trypsin-like (β2) subunits can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Supporting Information</a> (Figure S2A,B). The overall sequence identity between LONP1 and the chymotrypsin-like (β5) subunit of the 20S proteasome is low, 15.1% (27.1% homology), as is the overall architecture of the bortezomib binding pocket. Of the residues lining the binding site, the conservation is even lower; only 4 of the 18 residues within 5 Å of the LONP1 protease binding site are roughly structurally conserved (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Figure S2E</a>).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding site comparison of LONP1 and the 20S proteasome β5-subunit. Hydrogen bonds to the protein backbones are shown in green and the reactive serine or threonine residues for LONP1 and 20S proteasome, respectively, are indicated by an asterisk (*). (A) Bortezomib bound to human LONP1 protease domain (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>). (B) Bortezomib bound to the β5 site of the human 20S proteasome (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF3">5LF3</a>). (C) Comparison of the binding mode of bortezomib in LONP1 (yellow) vs the 20S proteasome β5 site (cyan). (D) Crystal structure of <b>9a</b> bound to LONP1 (dark purple; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a>) and docked pose (light purple). The root-mean-square deviation (RMSD) between the docked and crystal pose is 0.63. (E) Docking model of <b>9a</b> bound to the β5 subunit of the human 20S proteasome. (F) Overlay of the crystal pose of <b>9a</b> bound to LONP1 (purple; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a>) with the predicted binding mode in the β5 subunit of the human 20S proteasome (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite the structural and sequence differences within the LONP1 and chymotrypsin pockets, the binding affinity of bortezomib is similar between the two proteases (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). One explanation for this may be that the binding interactions are congruous between the two proteases (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). The reactive oxygen that forms a reversible covalent bond to the boronic acid group of bortezomib (S855 in LONP1 and T1 in the 20S proteasome) is spatially conserved in both species (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B, reactive residues indicated with *). Additionally, the primary contacts between bortezomib and the protein backbone are maintained (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A–C). These conserved interactions result in a similar binding pose of bortezomib in LONP1 and the 20S proteasome and likely explain the similar affinity in the two proteases (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). An alternate conformation for the P2 benzyl moiety of bortezomib was observed in the LONP1–bortezomib co-structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>) and modeled for all chains but B, with refined occupancies of 0.36–0.45.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical Activity Profiles of Bortezomib and Analogues in LONP1 and 20S Proteasome<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">LONP1</th><th class="colsep0 rowsep0" align="center">20S proteasome</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bortezomib</td><td class="colsep0 rowsep0" align="char" char="±">0.183 ± 0.150</td><td class="colsep0 rowsep0" align="left">0.097 ± 0.072</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5a</b></td><td class="colsep0 rowsep0" align="char" char="±">2.111 ± 0.129</td><td class="colsep0 rowsep0" align="left">1.524 ± 0.268</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="char" char="±">2.980 ± 0.134</td><td class="colsep0 rowsep0" align="left">1.701 ± 0.235</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.253 ± 0.161</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9b</b></td><td class="colsep0 rowsep0" align="char" char="±">0.433 ± 0.161</td><td class="colsep0 rowsep0" align="left">2.676 ± 0.905</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9c</b></td><td class="colsep0 rowsep0" align="char" char="±">0.408 ± 0.212</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.187 ± 0.092</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9e</b></td><td class="colsep0 rowsep0" align="char" char="±">0.093 ± 0.036</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9f</b></td><td class="colsep0 rowsep0" align="char" char="±">0.546 ± 0.258</td><td class="colsep0 rowsep0" align="left">3.971 ± 4.608</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9g</b></td><td class="colsep0 rowsep0" align="char" char="±">2.850 ± 0.632</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9h</b></td><td class="colsep0 rowsep0" align="char" char="±">0.137 ± 0.077</td><td class="colsep0 rowsep0" align="left">0.843 ± 0.182</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9i</b></td><td class="colsep0 rowsep0" align="char" char="±">6.199 ± 0.032</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.092 ± 0.015</td><td class="colsep0 rowsep0" align="left">0.541 ± 0.582</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11b</b></td><td class="colsep0 rowsep0" align="char" char="±">0.065 ± 0.013</td><td class="colsep0 rowsep0" align="left">1.036 ± 0.490</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11c</b></td><td class="colsep0 rowsep0" align="char" char="±">0.077 ± 0.020</td><td class="colsep0 rowsep0" align="left">1.943 ± 0.938</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.018 ± 0.004</td><td class="colsep0 rowsep0" align="left">0.259 ± 0.194</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11e</b></td><td class="colsep0 rowsep0" align="char" char="±">0.034 ± 0.015</td><td class="colsep0 rowsep0" align="left">0.262 ± 0.051</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11f</b></td><td class="colsep0 rowsep0" align="char" char="±">0.109 ± 0.032</td><td class="colsep0 rowsep0" align="left">0.679 ± 0.227</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.136 ± 0.134</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.017 ± 0.012</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12e</b></td><td class="colsep0 rowsep0" align="char" char="±">0.092 ± 0.054</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12f</b></td><td class="colsep0 rowsep0" align="char" char="±">0.556 ± 0.354</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12g</b></td><td class="colsep0 rowsep0" align="char" char="±">0.038 ± 0.010</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char="±">0.059 ± 0.046</td><td class="colsep0 rowsep0" align="left">>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">HTRF IC<sub>50</sub> values (μM) were determined after a 1 h compound treatment. All data are an average of at least duplicate measurements.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Stereochemical Inversion at the P2 Position Improves Selectivity by Reducing the Potency on 20S Proteasome</h3><div class="NLM_p">Among the bortezomib analogues included in the Novartis library, there were indications that stereochemistry could impact activity on both LONP1 and 20S proteasome (data not shown). For this reason, we prepared the bortezomib diastereomers <b>5a–b</b> and <b>9a</b> and assessed their activity in both targets (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Inverting the boron stereocenter (referred as P1 in this paper; <b>5a</b>) or converting both bortezomib stereocenters (<b>5b</b>) had detrimental effect on LONP1 and 20S proteasome. In contrast, conversion of the amino acid stereocenter (referred as P2 position herein) to <i>R</i> (<b>9a</b>) resulted in more than 100-fold reduction in potency toward the 20S proteasome, with only a slight drop in potency toward LONP1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p last">To rationalize the observed improvement in selectivity and relative changes in potency, we developed a series of docking models for both LONP1 and the 20S proteasome. For both proteases, the models predict an alternative binding conformation wherein the orientation of the P2 and P3 substituents is switched due to the stereoinversion at the P2 site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D–F). In the 20S proteasome, the alternative conformation of <b>9a</b> results in the weakening and loss of H-bonds from the P3 amide (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E) and also shifts the P2 benzyl group out of the pocket toward the solvent. Docking of <b>9a</b> to LONP1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D, light purple) suggested the same weakening and loss of the H-bonds from the P3 amide; however, in LONP1, the pyrazine ring of P3 forms a π–π stack with W770 of LONP1, as was later confirmed by crystallography (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D, dark purple, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a>, ligand RMSD = 1.1). Additional density for the <b>9a</b> co-structure with LONP1 was observed around the boronic acid but its identity could not be unambiguously confirmed. The possibility exists that this density is a function of a five- and/or six-membered glycerol boronate formed from glycerol used in protein storage buffer and cryoprotection of crystals during harvesting, although we were unable to confirm it by mass spectrometry studies and is a subject of future studies.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Changes in P2 Drive Selectivity by Reducing the 20S Proteasome Potency while Expansion into the Flexible P1 Pocket Drives Selectivity by Improving the LONP1 Potency</h3><div class="NLM_p">Despite the improved selectivity of <b>9a</b>, the potency toward the 20S proteasome was not entirely mitigated and a second round of SAR was initiated. Both the docking models and crystal structures suggested that further modification of the P3 pyrazine would be unlikely to lead to improved potency or selectivity due to the conserved backbone interaction points. We instead focused our efforts on both P1 and P2 substituents (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). First, we explored the SAR at the P2 region by replacing the benzyl group with methyl (<b>9b</b>) or linear alkyls (<b>9c–e</b>). The activity on LONP1 was mostly unchanged (within ±2 fold). In contrast, there was a trend of loss of 20S potency with longer, unsubstituted alkyl chains (<b>9c–e</b>), probably due to the extension of these hydrophobic groups into a solvent-exposed region of the 20S proteasome. Interestingly, α-substituted groups [<i>e.g.</i>, <i>i</i>-Pr (<b>9f</b>) or <i>t</i>-Bu (<b>9g</b>)] trended toward reduced LONP1 potency, while replacing the benzyl group for a cyclohexylmethyl (<b>9h</b>) was tolerated on both proteases. Finally, removal of the chiral center with <i>gem</i>-dimethyl (<b>9i</b>) led to 14-fold reduction in potency compared to <b>9b</b>, suggesting the importance of the stereochemistry at P2.</div><div class="NLM_p">The P1 binding region in both LONP1 and the 20S proteasome is fairly devoid of functional groups and both are relatively hydrophobic; however, in LONP1, the P1 binding site is more of a pocket, whereas in the 20S proteasome, it is more groove-like. Another notable difference is the relative size of the pockets. In the LONP1 crystal structures, the apo (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS">6WYS</a>) and bortezomib-bound (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>) pockets are fairly similar in size, 212.7 and 223.7 Å<sup>3</sup>, respectively. In the 20S proteasome, the apo pocket (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LE5">5LE5</a>) is 149.2 Å<sup>3</sup>, slightly smaller than the pocket of LONP1; however, the bortezomib-bound pocket (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF3">5LF3</a>) is more than double the size at 333.8 Å<sup>3</sup>. Given the large difference between the apo and the bortezomib-bound pockets, we turned to molecular dynamics (MD) simulations to explore potential differences in binding site residue dynamics that could be exploited to improve compound selectivity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MD simulation and docking poses of LONP1 inhibitors with LONP1 and 20S proteasome. (A) Binding pocket dynamics over 200 ns of MD simulation of the apo- and bortezomib-bound LONP1 and (B) 20S proteasome β5 subunit. (C) Conformational states of L778 observed in LONP1 crystal structures (top panel; PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS">6WYS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M">6X1M</a>). The interior pocket surface and volume are shown for the bortezomib-bound (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>) and <b>12d</b>-bound (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M">6X1M</a>) crystal structures (lower panel). (D) Binding mode of <b>12d</b> in LONP1 displaying hydrogen bonds to the backbone and the π–π stacking with W770. (E) Predicted binding modes of <b>11d</b> (green) and <b>12d</b> (yellow) using the 20S proteasome-expanded pocket model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">MD simulations of the protease domains of LONP1 and the 20S proteasome were performed both with and without bortezomib binding (PDBs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS">6WYS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LE3">5LE3</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF5">5LF5</a>). Interestingly, despite having a smaller pocket in the bortezomib-bound crystal form, the MD simulations suggest that, on average, the ligand binding pocket in LONP1 is not only larger but also more dynamic than that of the 20S proteasome (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B; <a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Figure S2C,D</a>). A detailed investigation of each binding site residue (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Figure S3–S6</a>) revealed that, in LONP1, residue L778 adopts several distinct, stable conformations that change the P1 pocket size and shape (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">S3H</a>). This is in contrast to the 20S proteasome where the back of the pocket is formed by a relatively invariable valine (V31, <a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Figures S4–S6</a>). This finding suggests that an increase in the size of the P1 substituent may be tolerated by LONP1 but less by the 20S proteasome.</div><div class="NLM_p">To test this hypothesis, we synthesized a series of <b>9a</b> analogues with varying P1 substituents (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <b>11a–f</b>). Interestingly, replacing the isobutyl group of <b>9a</b> with larger substituents resulted in up to 10-fold improvements in LONP1 activity, yet no selectivity increases versus the 20S proteasome. Docking models extracted from the MD simulations of LONP1 suggested that P1 substituents up to 130 Å<sup>3</sup> (roughly the size of (CH<sub>2</sub>)<sub>3</sub>Ph) could be accommodated when L778 is in an upward facing conformation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,C). This finding was later confirmed by the co-structure of LONP1 with <b>12d</b> (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M">6X1M</a>), where the pocket volume expands from 212.7 to 292.7 Å<sup>3</sup>, and L778 is found in an upward facing conformation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Alternatively, in the 20S proteasome where the P1 pocket is less dynamic, large P1 substituents are predicted to fold up into the P1 groove, away from the center of the protein, while the P2 and P3 positions occupy roughly the same regions (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). Based on these findings, we attribute the increase in potency on LONP1 to the flexibility of residue L778 and the sampling of a deep hydrophobic pocket that can be appropriately filled with large P1 groups (<i>e.g.</i>, <b>11d–e</b>).</div><div class="NLM_p last">In compounds with large P1 substituents, modification of the P2 position from aromatic to aliphatic was found to have a strong effect on the potency against the 20S proteasome. All compounds with a benzylic P2 substituent bind with at least moderate affinity to the 20S proteasome; however, when the P2 position is aliphatic, the binding drops off precipitously and is completely abolished when the P1 position is optimized (compounds <b>12a</b> and <b>d–g</b>). Despite the lack of observed binding affinity, the docking model does not provide a clear rationalization for this observation. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E, the model predicts that compounds with an aliphatic P2 substituent (<b>12a</b> and <b>d–g</b>) can be accommodated in a similar fashion as those with an aromatic P2 substituent (<b>9a</b>, <b>11a</b>, and <b>d–f</b>). Under the assumption that the docking model is accurate, this finding suggests that entropic factors rather than enthalpic contributions may be responsible for the loss in potency against the 20S proteasome.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Biochemically Potent LONP1 Inhibitors Demonstrate Potent Cellular Activity in LONP1-Dependent Cell Lines</h3><div class="NLM_p">Given the improved selectivity and potency profile of <b>12d</b>, we attempted to improve its overall physicochemical properties by modifying P3, a position that appeared less critical to overall binding based on docking. We prepared multiple analogues that included (hetero)aromatic or (hetero)alkyl amides, carbamates, ureas, and sulfonamides as P3. All of them maintained sub-micromolar activity on LONP1, with only a fraction demonstrating 20S proteasome IC<sub>50</sub> < 10 μM (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A), and all but one analogue having at least 80-fold selectivity versus the 20S proteasome. In an effort to differentiate these biochemically active and selective compounds, we turned our attention to cells to identify a pharmacodynamic (PD) biomarker that is altered upon inhibition of LONP1. Among all the analogues, <b>14</b> stood out as a potent and selective tool compound for <i>in vitro</i> LONP1 biology exploration. In fact, profiling of <b>14</b> in a protease panel at Nanosyn demonstrated its exquisite selectivity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Table S4</a>). HEK293 cells stably expressing the LONP1-3X-FLAG construct were treated with <b>14</b> and subjected to an affinity purification and mass spectrometry analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Figure S7A</a>). MMADHC was identified as an abundant protein PD biomarker that was significantly increased from treatment with <b>14</b> compared to dimethyl sulfoxide (DMSO) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Although several other proteins were also enriched by <b>14</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">Figure S7B–D</a>), further confirmation of MMADHC induction by orthogonal methods was observed upon treatment of LONP1-dependent cell lines previously identified by genetic screens<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">S7E</a>). Furthermore, <b>14</b> exhibited a ∼30–150-fold EC<sub>50</sub> shift of MMADHC induction compared to bortezomib in Calu6, H1568, and H1944 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">S7E</a>). Viability studies in both LONP1-dependent (Calu6) and independent (H358) lines further demonstrate the selectivity of <b>14</b> and analogues compared to bortezomib (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). Thus, the biochemically potent LONP1 inhibitors result in a cellular increase of MMADHC and cell death in LONP1-dependent cell lines tested in this study.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical and cellular activity of LONP1 inhibitors. Full circles represent P3 SAR of <b>12d</b>, while open circles represent a subset of compounds from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. All scatter plot data at 10 μM IC<sub>50</sub> correspond to compounds with IC<sub>50</sub>s > 10 μM. (A) Scatter plot of LONP1 (<i>y</i>-axis) and 20S proteasome (<i>x</i>-axis) biochemical IC<sub>50</sub>s. (B) Relative abundance of MMADHC following affinity purification and mass spectrometry analysis of cells treated with DMSO or <b>14</b>. Abundance ratios were calculated based on the label-free quantitation from MS1 intensity of precursor ions using Thermo Scientific Proteome Discoverer software version 2.4. (C) WES analysis of MMADHC in H1944 cells following treatment with a 1:3 serial dilution of <b>14</b> starting at 10 μM. (D) MMADHC measured by HTRF assay in Calu6 cells following treatment with indicated compounds. (E) Biochemical LONP1 IC<sub>50</sub> (<i>y</i>-axis) and cell viability IC<sub>50</sub> (<i>x</i>-axis) from LONP1-sensitive Calu6 and LONP1-insensitive H358 cell lines, respectively. Compounds are colored by their activity in the biochemical 20S proteasome assay. With the exception of bortezomib (red circle), only compounds with 20S proteasome IC<sub>50</sub> > 10 μM (green) were tested in viability.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56031" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56031" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, we have described our efforts toward the identification of potent and selective boronic acid-based LONP1 inhibitors that can serve as tool compounds for the exploration of LONP1 biology. We have used X-ray crystallography and computational models to shed light on the LONP1 function and provide hypotheses on the structural differences of LONP1 and the 20S proteasome that led to the identification of selective LONP1 inhibitors. Studies aimed at understanding the impact of LONP1 upregulation in cancer are currently ongoing and data will be reported at due course.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Experimental Procedures</h3><div class="NLM_p">Unless otherwise noted, materials were obtained from commercial suppliers and were used without further purification. Removal of solvent under reduced pressure or concentration refers to distillation using Büchi or Heidolph rotary evaporators attached to a vacuum pump (3 mmHg). The products obtained as solids or high boiling oils were dried under vacuum (1 mmHg).</div><div class="NLM_p last">Purification of compounds by preparative RP-HPLC was achieved using a Waters autopurification system consisting of a 2767 autosampler/fraction collector, a 2545 binary gradient module, a 2489 UV detector, and a QDa mass spectrometer. The compounds were purified using a flow rate of 100 mL/min with a 50 mm × 19 mm i.d. Waters Atlantis T3 prep OBD 10 μm column (Waters, Milford, MA). A 3 min linear gradient from 10% solvent A [acetonitrile (MeCN) with 0.035% trifluoroacetic acid (TFA)] in solvent B (water [H<sub>2</sub>O] with 0.05% TFA) to 30–90% A was used. Silica gel chromatography was performed by a CombiFlash (separation system <i>R</i><sub>f</sub>, TELEDYNE ISCO). Reverse-phase chromatography was performed by CombiFlash (separation system NEXTGEN 300+, TELEDYNE ISCO). <sup>1</sup>H NMR spectra were recorded on Avance-NEO with cryo-QNP (proton, carbon, phosphorous, and fluorine) (400 MHz) and Avance-III with smart probe (500 MHz) spectrometers. Proton resonances are reported in parts per million (ppm) downfield from tetramethylsilane (TMS). <sup>1</sup>H NMR data are reported as multiplicity (s, singlet; d, doublet; t, triplet; q, quartet; m, multiplets; dd, doublet of doublets; dt, doublet of triplets; dq, doublet of quartets; td, triplet of doublets; tt, triplet of triplets; ddd, doublet of doublet of doublets; and ddt, doublet of doublet of triplets). For spectra obtained in dimethylsulfoxide-<i>d</i><sub>6</sub> (DMSO-<i>d</i><sub>6</sub>) and methanol-<i>d</i><sub>4</sub> (CD<sub>3</sub>OD), the residual protons (2.50 and 3.31 ppm, respectively) were used as the internal reference.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Purity of Compounds</h3><div class="NLM_p last">The purity of all final compounds is ≥95%, as analyzed by an LC/MS system composed of an Agilent G1312B Binary Pump SL, an Agilent G1379B Degasser, an Agilent G1367C High-Performance Autosampler SL, an Agilent G1315C Diode Array Detector, a SOFT-A 1300 Evaporative Light Scattering Detector, and an Agilent G6140A MSD operated with ChemStation software. Samples were injected to the 0.9 mL per minute mobile phase flow starting at either 90% H<sub>2</sub>O + 0.05% TFA (A) and 10% MeCN + 0.035% TFA (B) or 70% H<sub>2</sub>O + 0.05% TFA (A) and 30% MeCN + 0.035% TFA (B), with a linear gradient to 90% B at 1.35 min, followed by 100% B from 1.36 to 1.95 min, and subsequent return to initial conditions until the end of the run at 2 min. The system employed a Waters Acquity HSS T3, 2.1 × 50.0 mm, 1.8 μ C18 column, which was kept at 60 °C. The mass spectrometer was operated in the positive mode with an electrospray voltage of 3 kV, nitrogen temperature at 350 °C, and skimmer set to 35 V. Chiral compounds were also analyzed by chiral HPLC or chiral SFC and demonstrated at least 95% ee.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> General Procedure A: Amide Coupling</h3><div class="NLM_p last">A mixture of carboxylic acid (1.1–1.3 equiv) and 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU) or 2-(1<i>H</i>-benzo[<i>d</i>][1,2,3]triazol-1-yl)-1,1,3,3-tetramethyluronium (TBTU) (1.2–1.4 equiv) in dichloromethane (DCM) or dimethylformamide (DMF) (0.1–0.25 M) was cooled to −15 °C and then treated with diisopropylethylamine (DIEA) (3–4 equiv) and stirred for 5 min. The mixture was then treated with the corresponding amine (free base, hydrochloride, or trifluoroacetate) (1.0 equiv), stirred for 15 min at −15 °C, and then allowed to warm to room temperature. After 1–4 h, the reaction mixture was concentrated under reduced pressure and purified by reverse-phase chromatography (C<sub>18</sub>, 30–100% MeCN/H<sub>2</sub>O) to afford the desired amide.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Procedure B: <i>tert</i>-Butyl Ester and <i>tert</i>-Butyl Carbamate Hydrolysis</h3><div class="NLM_p last">The ester or carbamate (1.0 equiv) was treated with 4 M hydrochloric acid (HCl) in dioxane (20–50 equiv) and stirred for 16 h. The volatiles were removed under reduced pressure to afford the desired carboxylic acid or amine hydrochloride.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedure C: Boronic Ester Hydrolysis</h3><div class="NLM_p">The amidoboronate was partitioned between heptane and 3 M HCl in MeOH (1:1 v/v, 15–30 equiv HCl) and then treated with isobutylboronic acid (10 equiv). The reaction mixture was stirred for 16 h, and then the layers were separated. The heptane layer was extracted once with MeOH and the combined MeOH extracts were washed twice with heptane. The MeOH solution was then concentrated under reduced pressure, partitioned between ethyl acetate (EtOAc) and saturated sodium bicarbonate (NaHCO<sub>3</sub>), and the aqueous layer extracted twice with EtOAc. The combined organic layers were dried over sodium sulfate, concentrated under reduced pressure, and purified by reverse-phase chromatography (10–80% MeCN/H<sub>2</sub>O) to afford the desired boronic acid.</div><div id="sec3_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Synthesis of Compounds <b>3a–c</b></h4><div class="NLM_p">Compounds <b>3a–c</b> were prepared using general procedure A, followed by general procedure B. (<i>S</i>)-3-Phenyl-2-(pyrazine-2-carboxamido)propanoic acid (<b>3a</b>, 2.06 g, 99%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.04 (s, 1H), 9.14 (d, <i>J</i> = 1.5 Hz, 1H), 8.89 (d, <i>J</i> = 2.5 Hz, 1H), 8.86 (d, <i>J</i> = 8.2 Hz, 1H), 8.75 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 7.28–7.21 (m, 4H), 7.18 (ddt, <i>J</i> = 6.6, 4.9, 2.7 Hz, 1H), 4.75 (td, <i>J</i> = 8.3, 5.3 Hz, 1H), 3.28–3.16 (m, 2H); ESMS <i>m</i>/<i>z</i>: 272.1 (M + H)<sup>+</sup>.</div><div class="NLM_p">(<i>R</i>)-3-Phenyl-2-(pyrazine-2-carboxamido)propanoic acid (<b>3b</b>, 2.06 g, 99% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.13 (d, <i>J</i> = 1.5 Hz, 1H), 8.90–8.83 (m, 2H), 8.74 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 7.22 (s, 4H), 7.21–7.13 (m, 1H), 4.74 (td, <i>J</i> = 8.2, 5.5 Hz, 1H), 3.26–3.15 (m, 2H); ESMS <i>m</i>/<i>z</i>: 272.1 (M + H)<sup>+</sup>.</div><div class="NLM_p last">(<i>R</i>)-2-(Pyrazine-2-carboxamido)pentanoic acid (<b>3c</b>, 1.08 g, 100% yield). ESMS <i>m</i>/<i>z</i>: 224.2 (M + H)<sup>+</sup>.</div></div><div id="sec3_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of Compounds <b>5a–b</b>, <b>11a–f</b>, <b>12a</b>, and <b>12d–g</b></h4><div class="NLM_p">Compounds <b>5a–b</b>, <b>11a–f</b>, <b>12a</b>, and <b>12d–g</b> were prepared using general procedure A, followed by general procedure C. ((<i>S</i>)-3-Methyl-1-((<i>S</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)butyl)boronic acid (<b>5a</b>, 4 mg, 10%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.18 (dd, <i>J</i> = 8.4, 1.5 Hz, 1H), 8.80 (dd, <i>J</i> = 5.2, 2.5 Hz, 1H), 8.70 (ddd, <i>J</i> = 4.2, 2.5, 1.6 Hz, 1H), 7.34–7.26 (m, 4H), 7.26–7.19 (m, 1H), 5.15–5.00 (m, 1H), 3.28–3.19 (m, 2H), 2.69 (t, <i>J</i> = 7.7 Hz, 1H), 1.63–1.31 (m, 2H), 1.22–1.19 (m, 1H), 0.87 (ddd, <i>J</i> = 14.3, 6.6, 5.0 Hz, 6H); ESMS <i>m</i>/<i>z</i>: 407.2 (M + Na)<sup>+</sup>.</div><div class="NLM_p">((<i>S</i>)-3-Methyl-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)butyl)boronic acid (<b>5b</b>, 3 mg, 73% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.17 (dd, <i>J</i> = 1.5, 8.4 Hz, 1H), 8.80 (dd, <i>J</i> = 2.5, 5.1 Hz, 1H), 8.70 (ddd, <i>J</i> = 1.5, 2.5, 4.3 Hz, 1H), 7.33–7.27 (m, 4H), 7.27–7.21 (m, 1H), 5.17–5.00 (m, 1H), 3.25–3.20 (m, 2H), 2.69 (t, <i>J</i> = 7.7 Hz, 1H), 1.65–1.30 (m, 2H), 1.29–0.98 (m, 1H), 0.85 (dd, <i>J</i> = 6.6, 4.8 Hz, 6H); ESMS <i>m</i>/<i>z</i>: 407.2 (M + Na)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-2-Cyclohexyl-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)ethyl)boronic acid (<b>11a</b>, 11 mg, 84% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.18 (dd, <i>J</i> = 1.5, 8.8 Hz, 1H), 8.80 (dd, <i>J</i> = 2.5, 5.1 Hz, 1H), 8.71 (ddd, <i>J</i> = 1.5, 2.5, 4.2 Hz, 1H), 7.34–7.27 (m, 4H), 7.27–7.18 (m, 1H), 5.19–4.99 (m, 1H), 3.28–3.24 (m, 1H), 2.72 (dd, <i>J</i> = 5.9, 9.2 Hz, 1H), 1.80 (d, <i>J</i> = 13.1 Hz, 1H), 1.73–1.55 (m, 4H), 1.39–1.27 (m, 2H), 1.27–1.01 (m, 5H), 0.93–0.75 (m, 2H); ESMS <i>m</i>/<i>z</i>: 407.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-5-Methyl-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)hexyl)boronic acid (<b>11b</b>, 8 mg, 59% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.17 (dd, <i>J</i> = 1.5, 7.0 Hz, 1H), 8.80 (dd, <i>J</i> = 2.5, 4.1 Hz, 1H), 8.70 (ddd, <i>J</i> = 1.5, 2.5, 3.3 Hz, 1H), 7.34–7.25 (m, 4H), 7.25–7.18 (m, 1H), 5.15–4.99 (m, 1H), 3.26–3.22 (m, 1H), 2.56 (t, <i>J</i> = 6.9 Hz, 1H), 1.59–1.48 (m, 1H), 1.48–1.27 (m, 3H), 1.27–1.09 (m, 4H), 0.89–0.85 (m, 6H); ESMS <i>m</i>/<i>z</i>: 395.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-6-Bromo-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)hexyl)boronic acid (<b>11c</b>, 21 mg, 69% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.19 (d, <i>J</i> = 1.5 Hz, 1H), 8.79 (d, <i>J</i> = 2.5 Hz, 1H), 8.68 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 7.28–7.24 (m, 4H), 7.23–7.17 (m, 1H), 4.84–4.77 (m, 1H), 3.42 (t, <i>J</i> = 6.8 Hz, 2H), 3.17 (dd, <i>J</i> = 6.6, 1.8 Hz, 1H), 3.15–3.07 (m, 2H), 1.87–1.77 (m, 2H), 1.47–1.38 (m, 4H), 1.29–1.19 (m, 2H); ESMS <i>m</i>/<i>z</i>: 459.1 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-4-Phenyl-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)butyl)boronic acid (<b>11d</b>, 8 mg, 53% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.17 (dd, <i>J</i> = 1.5, 9.0 Hz, 1H), 8.80 (dd, <i>J</i> = 2.5, 4.7 Hz, 1H), 8.70 (ddd, <i>J</i> = 1.5, 2.5, 4.2 Hz, 1H), 7.32–7.27 (m, 3H), 7.27–7.20 (m, 4H), 7.17–7.09 (m, 3H), 5.12–5.01 (m, 1H), 3.37–3.34 (m, 1H), 3.26–3.17 (m, 1H), 2.65–2.49 (m, 3H), 1.69–1.57 (m, 1H), 1.49 (ddt, <i>J</i> = 7.3, 14.6, 29.7 Hz, 2H), 1.35–1.23 (m, 1H); ESMS <i>m</i>/<i>z</i>: 429.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-3-Phenyl-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)propyl)boronic acid (<b>11e</b>, 9 mg, 93% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.18 (s, 1H), 8.80 (s, 1H), 8.70 (s, 1H), 7.34–7.27 (m, 4H), 7.25–7.19 (m, 3H), 7.18–7.08 (m, 3H), 5.14–5.03 (m, 1H), 3.28–3.21 (m, 2H), 2.61 (q, <i>J</i> = 7.1, 8.4 Hz, 2H), 2.50 (t, <i>J</i> = 7.1 Hz, 1H), 1.88–1.71 (m, 1H), 1.70–1.46 (m, 1H); ESMS <i>m</i>/<i>z</i>: 415.2 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-2-Phenyl-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)ethyl)boronic acid (<b>11f</b>, 9 mg, 86% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.17 (d, <i>J</i> = 6.5 Hz, 1H), 8.87–8.72 (m, 1H), 8.69 (d, <i>J</i> = 2.2 Hz, 1H), 7.38–7.23 (m, 5H), 7.23–7.15 (m, 3H), 7.15–7.07 (m, 1H), 7.03–6.97 (m, 1H), 5.12–4.98 (m, 1H), 3.27–3.16 (m, 2H), 2.93–2.73 (m, 2H), 2.68–2.30 (m, 1H); ESMS <i>m</i>/<i>z</i>: 401.20 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-2-Cyclohexyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)pentanamido)ethyl)boronic acid (<b>12a</b>, 9 mg, 45% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.24 (d, <i>J</i> = 1.4 Hz, 1H), 8.82 (d, <i>J</i> = 2.5 Hz, 1H), 8.73 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 4.87–4.80 (m, 1H), 2.76 (t, <i>J</i> = 6.9 Hz, 1H), 1.98–1.85 (m, 2H), 1.76–1.66 (m, 4H), 1.59–1.40 (m, 3H), 1.40–1.31 (m, 3H), 1.30–1.17 (m, 3H), 1.00 (t, <i>J</i> = 7.4 Hz, 3H), 0.97–0.77 (m, 2H); ESMS <i>m</i>/<i>z</i>: 359.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-4-Phenyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)pentanamido)butyl)boronic acid (<b>12d</b>, 15 mg, 84% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.23 (d, <i>J</i> = 1.5 Hz, 1H), 8.82 (d, <i>J</i> = 2.5 Hz, 1H), 8.72 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 7.25–7.19 (m, 2H), 7.19–7.15 (m, 2H), 7.14–7.09 (m, 1H), 4.83 (dd, <i>J</i> = 8.9, 5.7 Hz, 1H), 2.69–2.53 (m, 3H), 2.04–1.80 (m, 2H), 1.80–1.61 (m, 2H), 1.61–1.34 (m, 4H), 0.99 (t, <i>J</i> = 7.4 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 381.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-3-Phenyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)pentanamido)propyl)boronic acid (<b>12e</b>, 29 mg, 62% yield). <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.14 (d, <i>J</i> = 1.5 Hz, 1H), 8.70 (d, <i>J</i> = 2.5 Hz, 1H), 8.64–8.58 (m, 1H), 7.17–7.05 (m, 4H), 7.02 (td, <i>J</i> = 7.0, 3.3 Hz, 1H), 4.77–4.74 (m, 1H), 2.62–2.48 (m, 3H), 1.93–1.79 (m, 2H), 1.74 (ddt, <i>J</i> = 13.3, 9.7, 6.6 Hz, 1H), 1.63 (ddt, <i>J</i> = 13.7, 9.4, 6.6 Hz, 1H), 1.38 (m, 2H), 0.89 (t, <i>J</i> = 7.4 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 367.2 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-2-Phenyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)pentanamido)ethyl)boronic acid (<b>12f</b>, 24 mg, 45% yield). <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.13 (dd, <i>J</i> = 1.5, 6.9 Hz, 1H), 8.71 (dd, <i>J</i> = 2.5, 4.9 Hz, 1H), 8.60 (ddd, <i>J</i> = 1.5, 2.5, 6.6 Hz, 1H), 7.14 (d, <i>J</i> = 4.8 Hz, 1H), 7.11–6.98 (m, 4H), 4.71–4.64 (m, 1H), 2.88–2.74 (m, 2H), 2.52 (td, <i>J</i> = 9.3, 14.2 Hz, 1H), 1.87–1.72 (m, 2H), 1.46–1.26 (m, 2H), 0.93–0.83 (m, 3H); ESMS <i>m</i>/<i>z</i>: 353.2 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p last">((<i>R</i>)-1-((<i>R</i>)-2-(Pyrazine-2-carboxamido)pentanamido)pentyl)boronic acid (<b>12g</b>, 19 mg, 36% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.24 (d, <i>J</i> = 1.5 Hz, 1H), 8.82 (d, <i>J</i> = 2.5 Hz, 1H), 8.73 (dd, <i>J</i> = 1.5, 2.5 Hz, 1H), 4.88–4.81 (m, 1H), 2.59 (t, <i>J</i> = 7.3 Hz, 1H), 2.04–1.86 (m, 2H), 1.59–1.28 (m, 8H), 1.00 (t, <i>J</i> = 7.4 Hz, 3H), 0.96–0.87 (m, 3H); ESMS <i>m</i>/<i>z</i>: 319.2 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div></div><div id="sec3_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Synthesis of Compounds <b>8a–i</b>, <b>13</b></h4><div class="NLM_p">Compounds <b>8a–i</b>, <b>13</b> were prepared using general procedure A. <i>tert</i>-Butyl((<i>R</i>)-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxo-3-phenylpropan-2-yl)carbamate (<b>8a</b>, 64 mg, 95%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.26 (m, 5H), 4.51 (dd, <i>J</i> = 8.5, 6.3 Hz, 1H), 4.21 (dd, <i>J</i> = 8.6, 2.2 Hz, 1H), 3.13 (dd, <i>J</i> = 13.7, 6.0 Hz, 1H), 2.87 (dd, <i>J</i> = 13.7, 8.8 Hz, 1H), 2.65 (t, <i>J</i> = 7.7 Hz, 1H), 2.41–2.29 (m, 1H), 2.14 (ddd, <i>J</i> = 10.0, 6.0, 1.8 Hz, 1H), 1.98 (t, <i>J</i> = 5.5 Hz, 1H), 1.88 (tt, <i>J</i> = 5.6, 2.9 Hz, 1H), 1.81 (dt, <i>J</i> = 14.1, 2.5 Hz, 1H), 1.71 (dt, <i>J</i> = 13.4, 6.7 Hz, 1H), 1.48 (d, <i>J</i> = 10.3 Hz, 1H), 1.41 (s, 3H), 1.33 (m, 12H), 0.96–0.83 (s, 9H); ESMS <i>m</i>/<i>z</i>: 513.4 (M + H)<sup>+</sup>.</div><div class="NLM_p"><i>tert</i>-Butyl((<i>R</i>)-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxopropan-2-yl)carbamate (<b>8b</b>, 118 mg, 69% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 4.40–4.24 (m, 1H), 4.18 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 2.70 (t, <i>J</i> = 7.7 Hz, 1H), 2.43–2.27 (m, 1H), 2.22–2.07 (m, 1H), 1.95 (t, <i>J</i> = 5.6 Hz, 1H), 1.86 (td, <i>J</i> = 5.7, 2.9 Hz, 1H), 1.80 (d, <i>J</i> = 6.0 Hz, 1H), 1.76 (d, <i>J</i> = 5.8 Hz, 1H), 1.49–1.46 (m, 1H), 1.45 (s, 9H), 1.41 (s, 1H), 1.39 (s, 1H), 1.37 (s, 3H), 1.35 (d, <i>J</i> = 7.2 Hz, 3H), 1.29 (s, 3H), 0.93 (dd, <i>J</i> = 6.6, 1.9 Hz, 6H), 0.88 (s, 3H); ESMS <i>m</i>/<i>z</i>: 437.4 (M + H)<sup>+</sup>.</div><div class="NLM_p"><i>tert</i>-Butyl((<i>R</i>)-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxobutan-2-yl)carbamate (<b>8c</b>, 131 mg, 75% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 4.28–4.09 (m, 2H), 2.69 (t, <i>J</i> = 7.8 Hz, 1H), 2.33 (ddt, <i>J</i> = 11.1, 8.9, 2.5 Hz, 1H), 2.19–2.06 (m, 1H), 1.95 (t, <i>J</i> = 5.6 Hz, 1H), 1.86 (dq, <i>J</i> = 5.6, 2.8 Hz, 1H), 1.82 (d, <i>J</i> = 12.4 Hz, 1H), 1.79 (d, <i>J</i> = 3.5 Hz, 1H), 1.77 (d, <i>J</i> = 3.2 Hz, 1H), 1.75 (d, <i>J</i> = 6.9 Hz, 1H), 1.66 (dt, <i>J</i> = 14.4, 7.6 Hz, 1H), 1.45 (s, 9H), 1.40–1.38 (m, 1H), 1.37 (s, 3H), 1.36–1.34 (m, 1H), 1.29 (s, 3H), 0.97 (t, <i>J</i> = 7.4 Hz, 3H), 0.93 (d, <i>J</i> = 6.6 Hz, 6H), 0.88 (s, 3H); ESMS <i>m</i>/<i>z</i>: 451.4 (M + H)<sup>+</sup>.</div><div class="NLM_p"><i>tert</i>-Butyl((<i>R</i>)-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxopentan-2-yl)carbamate (<b>8d</b>, 125 mg, 70% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 4.25 (dd, <i>J</i> = 9.0, 5.7 Hz, 1H), 4.17 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 2.72–2.66 (m, 1H), 2.40–2.29 (m, 1H), 2.17–2.08 (m, 1H), 1.95 (t, <i>J</i> = 5.6 Hz, 1H), 1.90–1.82 (m, 1H), 1.81–1.78 (m, 1H), 1.78–1.76 (m, 1H), 1.76–1.71 (m, 1H), 1.71–1.64 (m, 1H), 1.64–1.57 (m, 1H), 1.48 (s, 1H), 1.45 (s, 9H), 1.42–1.38 (m, 2H), 1.37 (s, 3H), 1.36–1.33 (m, 1H), 1.29 (s, 3H), 0.96 (t, <i>J</i> = 7.4 Hz, 3H), 0.93 (d, <i>J</i> = 6.2 Hz, 6H), 0.88 (s, 3H); ESMS <i>m</i>/<i>z</i>: 465.4 (M + H)<sup>+</sup>.</div><div class="NLM_p"><i>tert</i>-Butyl((<i>R</i>)-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxohexan-2-yl)carbamate (<b>8e</b>, 86 mg, 46% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 4.23 (t, <i>J</i> = 7.4 Hz, 1H), 4.17 (dd, <i>J</i> = 8.7, 2.3 Hz, 1H), 2.68 (t, <i>J</i> = 7.7 Hz, 1H), 2.42–2.28 (m, 1H), 2.18–2.05 (m, 1H), 1.95 (t, <i>J</i> = 5.6 Hz, 1H), 1.89–1.83 (m, 1H), 1.82–1.78 (m, 1H), 1.78–1.70 (m, 2H), 1.69–1.56 (m, 1H), 1.48 (s, 1H), 1.45 (s, 9H), 1.38 (s, 2H), 1.37 (s, 3H), 1.36 (s, 4H), 1.29 (s, 3H), 0.96–0.89 (m, 9H), 0.88 (s, 3H); ESMS <i>m</i>/<i>z</i>: 479.4 (M + H)<sup>+</sup>.</div><div class="NLM_p"><i>tert</i>-Butyl((<i>R</i>)-3-methyl-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxobutan-2-yl)carbamate (<b>8f</b>, 118 mg, 66% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 4.17 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 4.07 (d, <i>J</i> = 7.2 Hz, 1H), 2.68 (t, <i>J</i> = 7.7 Hz, 1H), 2.34 (ddd, <i>J</i> = 14.2, 6.9, 4.5 Hz, 1H), 2.18–2.09 (m, 1H), 2.09–2.00 (m, 1H), 1.96 (t, <i>J</i> = 5.6 Hz, 1H), 1.86 (tt, <i>J</i> = 5.6, 3.0 Hz, 1H), 1.82–1.78 (m, 1H), 1.78–1.70 (m, 1H), 1.47 (s, 1H), 1.45 (s, 9H), 1.38 (s, 1H), 1.37 (s, 3H), 1.35 (s, 1H), 1.29 (s, 3H), 0.96 (s, 3H), 0.95–0.93 (m, 6H), 0.92 (s, 3H), 0.88 (s, 3H); ESMS <i>m</i>/<i>z</i>: 465.4 (M + H)<sup>+</sup>.</div><div class="NLM_p"><i>tert</i>-Butyl((<i>R</i>)-3,3-dimethyl-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxobutan-2-yl)carbamate (<b>8g</b>, 112 mg, 61% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 4.22–4.06 (m, 2H), 2.65 (t, <i>J</i> = 7.7 Hz, 1H), 2.40–2.27 (m, 1H), 2.20–2.07 (m, 1H), 1.96 (t, <i>J</i> = 5.6 Hz, 1H), 1.86 (tt, <i>J</i> = 5.6, 3.0 Hz, 1H), 1.82–1.70 (m, 2H), 1.50–1.46 (m, 1H), 1.45 (s, 9H), 1.38 (s, 3H), 1.37–1.33 (m, 2H), 1.29 (s, 3H), 1.00 (s, 9H), 0.93 (d, <i>J</i> = 6.8 Hz, 6H), 0.88 (s, 3H); ESMS <i>m</i>/<i>z</i>: 479.4 (M + H)<sup>+</sup>.</div><div class="NLM_p"><i>tert</i>-Butyl((<i>R</i>)-3-cyclohexyl-1-(((<i>R</i>)-3-methyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)-1-oxopropan-2-yl)carbamate (<b>8h</b>, 97 mg, 48% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 4.36 (dd, <i>J</i> = 9.9, 5.5 Hz, 1H), 4.17 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 2.67 (t, <i>J</i> = 7.7 Hz, 1H), 2.42–2.26 (m, 1H), 2.21–2.06 (m, 1H), 1.95 (t, <i>J</i> = 5.6 Hz, 1H), 1.90–1.83 (m, 1H), 1.83–1.75 (m, 3H), 1.72 (d, <i>J</i> = 13.5 Hz, 3H), 1.65 (s, 1H), 1.62–1.57 (m, 1H), 1.54 (dd, <i>J</i> = 9.6, 5.2 Hz, 1H), 1.48 (s, 1H), 1.45 (s, 9H), 1.41 (s, 2H), 1.37 (s, 3H), 1.35–1.30 (m, 2H), 1.29 (s, 3H), 1.27–1.23 (m, 1H), 1.23–1.12 (m, 2H), 1.05–0.96 (m, 1H), 0.93 (d, <i>J</i> = 6.5 Hz, 6H), 0.88 (m, 3H); ESMS <i>m</i>/<i>z</i>: 519.5 (M + H)<sup>+</sup>.</div><div class="NLM_p last">2,4-Dimethyl-<i>N</i>-((<i>R</i>)-1-oxo-1-(((<i>R</i>)-4-phenyl-1-((3a<i>S</i>,4<i>S</i>,6<i>S</i>,7a<i>R</i>)-3a,5,5-trimethylhexahydro-4,6-methanobenzo[<i>d</i>][1,3,2]dioxaborol-2-yl)butyl)amino)pentan-2-yl)oxazole-5-carboxamide (<b>13</b>, 27.7 g, 77% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.25–7.19 (m, 2H), 7.19–7.14 (m, 2H), 7.15–7.09 (m, 1H), 4.72 (dd, <i>J</i> = 9.4, 5.7 Hz, 1H), 4.17 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 2.71–2.55 (m, 3H), 2.50 (s, 3H), 2.39 (s, 3H), 2.37–2.28 (m, 1H), 2.16–2.06 (m, 1H), 1.95 (t, <i>J</i> = 5.6 Hz, 1H), 1.91–1.81 (m, 2H), 1.81–1.69 (m, 4H), 1.65–1.53 (m, 1H), 1.53–1.48 (m, 1H), 1.48–1.38 (m, 3H), 1.37 (s, 3H), 1.29 (s, 3H), 0.97 (t, <i>J</i> = 7.4 Hz, 3H), 0.88 (s, 3H); ESMS <i>m</i>/<i>z</i>: 550.4 (M + H)<sup>+</sup>.</div></div><div id="sec3_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Synthesis of Compounds <b>9a–i</b>, <b>14</b></h4><div class="NLM_p">Compounds <b>9a–i</b>, <b>14</b> were prepared using general procedure B, followed by general procedure A, followed by general procedure C. ((<i>R</i>)-3-Methyl-1-((<i>R</i>)-3-phenyl-2-(pyrazine-2-carboxamido)propanamido)butyl)boronic acid (<b>9a</b>, 17 mg, 44% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.16 (d, <i>J</i> = 1.5 Hz, 1H), 8.79 (d, <i>J</i> = 2.5 Hz, 1H), 8.69 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 7.35–7.17 (m, 5H), 5.15–5.04 (m, 1H), 3.35 (s, 1H), 3.22 (dd, <i>J</i> = 13.8, 8.4 Hz, 1H), 2.68 (t, <i>J</i> = 7.6 Hz, 1H), 1.59 (dq, <i>J</i> = 13.2, 6.7 Hz, 1H), 1.33 (t, <i>J</i> = 7.4 Hz, 2H), 0.88 (dd, <i>J</i> = 6.6, 4.9 Hz, 6H); ESMS <i>m</i>/<i>z</i>: 367.2 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-3-Methyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)propanamido)butyl)boronic acid (<b>9b</b>, 17 mg, 66% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.23 (d, <i>J</i> = 1.5 Hz, 1H), 8.80 (d, <i>J</i> = 2.5 Hz, 1H), 8.73–8.69 (m, 1H), 4.89 (q, <i>J</i> = 7.2 Hz, 1H), 2.73 (t, <i>J</i> = 7.6 Hz, 1H), 1.70–1.65 (m, 1H), 1.58 (d, <i>J</i> = 7.2 Hz, 3H), 1.36 (t, <i>J</i> = 7.3 Hz, 2H), 0.92 (d, <i>J</i> = 2.4 Hz, 3H), 0.90 (d, <i>J</i> = 2.4 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 291.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-3-Methyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)butanamido)butyl)boronic acid (<b>9c</b>, 17 mg, 79% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.24 (d, <i>J</i> = 1.5 Hz, 1H), 8.81 (d, <i>J</i> = 2.5 Hz, 1H), 8.72 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 4.74 (dd, <i>J</i> = 8.6, 5.9 Hz, 1H), 2.74 (t, <i>J</i> = 7.6 Hz, 1H), 2.14–1.99 (m, 1H), 1.99–1.86 (m, 1H), 1.78–1.59 (m, 1H), 1.36 (t, <i>J</i> = 7.3 Hz, 2H), 1.05 (t, <i>J</i> = 7.4 Hz, 3H), 0.92 (d, <i>J</i> = 2.2 Hz, 3H), 0.91 (d, <i>J</i> = 2.1 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 305.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-3-Methyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)pentanamido)butyl)boronic acid (<b>9d</b>, 19 mg, 67% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.24 (d, <i>J</i> = 1.5 Hz, 1H), 8.81 (d, <i>J</i> = 2.5 Hz, 1H), 8.72 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 4.88–4.84 (m, 1H), 2.73 (t, <i>J</i> = 7.6 Hz, 1H), 2.03–1.83 (m, 2H), 1.74–1.61 (m, 1H), 1.59–1.38 (m, 2H), 1.35 (t, <i>J</i> = 7.3 Hz, 2H), 0.99 (t, <i>J</i> = 7.4 Hz, 3H), 0.92 (d, <i>J</i> = 2.2 Hz, 3H), 0.90 (d, <i>J</i> = 2.3 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 319.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-3-Methyl-1-((<i>R</i>)-2-(pyrazine-2-carboxamido)hexanamido)butyl)boronic acid (<b>9e</b>, 20 mg, 71% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.24 (d, <i>J</i> = 1.5 Hz, 1H), 8.81 (d, <i>J</i> = 2.5 Hz, 1H), 8.72 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 4.81–4.79 (m, 1H), 2.73 (t, <i>J</i> = 7.6 Hz, 1H), 2.10–1.95 (m, 1H), 1.95–1.82 (m, 1H), 1.74–1.58 (m, 1H), 1.54–1.37 (m, 4H), 1.35 (t, <i>J</i> = 7.3 Hz, 2H), 0.99–0.92 (m, 3H), 0.92 (d, <i>J</i> = 2.1 Hz, 3H), 0.91 (d, <i>J</i> = 2.1 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 333.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-3-Methyl-1-((<i>R</i>)-3-methyl-2-(pyrazine-2-carboxamido)butanamido)butyl)boronic acid (<b>9f</b>, 24 mg, 78% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.24 (d, <i>J</i> = 1.5 Hz, 1H), 8.82 (d, <i>J</i> = 2.5 Hz, 1H), 8.72 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 4.64 (d, <i>J</i> = 7.6 Hz, 1H), 2.75 (t, <i>J</i> = 7.6 Hz, 1H), 2.43–2.22 (m, 1H), 1.79–1.58 (m, 1H), 1.42–1.29 (m, 2H), 1.06 (t, <i>J</i> = 6.4 Hz, 6H), 0.93 (d, <i>J</i> = 1.7 Hz, 3H), 0.91 (d, <i>J</i> = 1.7 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 319.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-1-((<i>R</i>)-3,3-Dimethyl-2-(pyrazine-2-carboxamido)butanamido)-3-methylbutyl)boronic acid (<b>9g</b>, 24 mg, 77% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.26 (d, <i>J</i> = 1.5 Hz, 1H), 8.84 (d, <i>J</i> = 2.5 Hz, 1H), 8.72 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 4.73 (s, 1H), 2.74 (t, <i>J</i> = 7.6 Hz, 1H), 1.77–1.60 (m, 1H), 1.38–1.32 (m, 2H), 1.12 (s, 9H), 0.93 (d, <i>J</i> = 2.1 Hz, 3H), 0.92 (d, <i>J</i> = 2.2 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 333.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">((<i>R</i>)-1-((<i>R</i>)-3-Cyclohexyl-2-(pyrazine-2-carboxamido)propanamido)-3-methylbutyl)boronic acid (<b>9h</b>, 28 mg, 72% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.24 (d, <i>J</i> = 1.5 Hz, 1H), 8.81 (d, <i>J</i> = 2.5 Hz, 1H), 8.72 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 4.94 (dd, <i>J</i> = 9.8, 5.5 Hz, 1H), 2.71 (t, <i>J</i> = 7.6 Hz, 1H), 1.87–1.78 (m, 3H), 1.74 (ddd, <i>J</i> = 15.3, 5.9, 3.7 Hz, 3H), 1.69–1.59 (m, 2H), 1.49–1.38 (m, 1H), 1.34 (t, <i>J</i> = 7.3 Hz, 2H), 1.30–1.24 (m, 1H), 1.20 (dd, <i>J</i> = 11.3, 8.3 Hz, 2H), 1.10–0.94 (m, 2H), 0.92 (d, <i>J</i> = 2.0 Hz, 3H), 0.90 (d, <i>J</i> = 2.0 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 373.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p">(<i>R</i>)-(3-Methyl-1-(2-methyl-2-(pyrazine-2-carboxamido)propanamido)butyl)boronic acid (<b>9i</b>, 10 mg, 17% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 9.20 (d, <i>J</i> = 1.5 Hz, 1H), 8.81 (d, <i>J</i> = 2.6 Hz, 1H), 8.71 (dd, <i>J</i> = 2.5, 1.5 Hz, 1H), 2.71 (t, <i>J</i> = 7.6 Hz, 1H), 1.69 (s, 7H), 1.42–1.30 (m, 2H), 0.90 (dd, <i>J</i> = 6.6, 0.9 Hz, 6H); ESMS <i>m</i>/<i>z</i>: 305.2 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div><div class="NLM_p last">((<i>R</i>)-1-((<i>R</i>)-2-(2,4-Dimethyloxazole-5-carboxamido)pentanamido)-4-phenylbutyl)boronic acid (<b>14</b>, 30 mg, 72% yield). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): δ 7.27–7.20 (m, 2H), 7.20–7.16 (m, 2H), 7.15–7.10 (m, 1H), 4.73 (dd, <i>J</i> = 5.7, 9.3 Hz, 1H), 2.71–2.51 (m, 3H), 2.50 (s, 3H), 2.38 (s, 3H), 1.96–1.78 (m, 2H), 1.70 (m, 2H), 1.59–1.37 (m, 4H), 0.98 (t, <i>J</i> = 7.4 Hz, 3H); ESMS <i>m</i>/<i>z</i>: 398.3 (M – H<sub>2</sub>O + H)<sup>+</sup>.</div></div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Biochemical LONP1 Protease Assay</h3><div class="NLM_p last">The biochemical LONP1 protease assay was carried out in a 384-well format utilizing the 5-FAM (fluorophore) and QXL520 (quencher) pair. The final assay mixture contains 10 mM Tris pH 7.5, 50 mM NaCl, 10 mM CaCl<sub>2</sub>, 2 mM DTT, 0.01% BSA, 40 nM LONP1, 1 μM ATP, and 10 μM of fluorogenic peptide substrate QXL520-YRGITCSGRQK(5-FAM)-NH<sub>2</sub> in a total volume of 10 μL. The assay was carried out at room temperature for 50 min, and fluorescence was read on a PheraStar reader. As the quencher and fluorophore are built into the peptide, no change in fluorescence will be detected unless the peptide is cleaved by the enzyme.</div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> LONP1 HTS</h3><div class="NLM_p last">The biochemical LONP1 protease assay was further miniaturized into a 1536-well format for HTS. The assay condition remained similar with a reduction in reaction volume (6 μL) and enzyme concentration (25 nM). The final compound concentration in the screen was either 10 μM or 50 μM (depending on the stock concentration). The HTS was performed on a GNF automation system.</div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Biochemical 20S Protease Assay</h3><div class="NLM_p last">The biochemical 20S protease assay was carried out in the same format as the LONP1 protease assay. The final assay mixture contains 10 mM Tris pH 7.5, 50 mM NaCl, 2 mM DTT, 0.01% BSA, 0.025% SDS, 0.571 nM 20S proteasome (ENZO, Cat # BML-PW8720-0050), and 10 μM of fluorogenic peptide substrate QXL520-YRGITCSGRQK(5-FAM)-NH<sub>2</sub> in a total volume of 10 μL. The enzyme solution (in reaction buffer) was first incubated at room temperature for 10 min to allow the 20S activation by SDS, followed by addition of substrate peptide to start the reaction. The assay was carried out at room temperature for 60 min, and fluorescence was read on a PheraStar reader.</div></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> IC<sub>50</sub> Determination</h3><div class="NLM_p last">Compounds were serial-diluted and Echo-transferred (50 nL) into the reaction plate. The compound and enzyme mixture was first incubated at room temperature for 15 min, and then the substrate mixture was added to start the reaction. The final compound concentrations ranged from 0.00028 to 50 μM, and IC<sub>50</sub> values were determined from the resulting 12-point inhibition curves based on the Michaelis–Menten equation.</div></div><div id="sec3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Cellular Assays</h3><div class="NLM_p last">All cell lines were purchased from ATCC and maintained as described previously.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> One thousand cells were seeded in 1536-well plates pre-spotted with a 1:3 serial dilution of compound starting from 10 μM and incubated for 5 days. The cell viability was measured using the Cell Titer-Glo Luminescent Cell Viability Assay Kit (Promega) and read on an Envision (PerkinElmer) plate reader. The IC<sub>50</sub> values were calculated from dose−response curves generated in GraphPad PRISM using a nonlinear regression four-parameter curve fitting model.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02152" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75302" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75302" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02152?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02152</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">LONP1 biochemical characterization, MD simulations, binding site dynamics of both LONP1 and 20S proteasome subunits, cell PD experiments, docking models, crystallographic conditions, and HPLC traces of <b>9d</b>, <b>12d</b>, and <b>14</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline"><b>9a</b> Docking model in LONP1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><b>9a</b> Docking model in the β5-subunit of the 20S proteasome (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><b>11d</b> Docking model in the expanded pocket of the β5-subunit of 20S proteasome (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><b>12d</b> Docking model in the expanded pocket of the β5-subunit of the 20S proteasome (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_006.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf">jm0c02152_si_001.pdf (2.93 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_002.pdb">jm0c02152_si_002.pdb (230.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_003.pdb">jm0c02152_si_003.pdb (248.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_004.pdb">jm0c02152_si_004.pdb (126.57 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_005.pdb">jm0c02152_si_005.pdb (126.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_006.csv">jm0c02152_si_006.csv (2.15 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The authors will release the atomic coordinates and experimental data upon article publication. The crystallographic coordinates and structure factors have been deposited in the Protein Data Bank, <a href="http://www.pdb.org" class="extLink">www.pdb.org</a> [PDB ID codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS">6WYS</a> (apo LONP1), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a> (bortezomib co-structure with LONP1), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a> (<b>9a</b> co-structure with LONP1), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M">6X1M</a> (<b>12d</b> co-structure with LONP1)].</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02152" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34482" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34482" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura J. Kingsley</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span>; 
    <span>Present Address:
                        Merck Research Laboratories, 213 E Grand Ave, South San Francisco, CA 94080</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2448455156450a4f4d4a435748415d64494156474f0a474b49"><span class="__cf_email__" data-cfemail="325e534740531c595b5c55415e574b725f574051591c515d5f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gérald Lelais</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6213-9863" title="Orcid link">http://orcid.org/0000-0002-6213-9863</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7b1c171e171a12083b1c151d5514091c"><span class="__cf_email__" data-cfemail="2b4c474e474a42586b4c454d0544594c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaohui He</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1003-2631" title="Orcid link">http://orcid.org/0000-0003-1003-2631</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew McNeill</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Nelson</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Victor Nikulin</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhiwei Ma</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenshuo Lu</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vicki W. Zhou</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mari Manuia</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Kreusch</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mu-Yun Gao</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darbi Witmer</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mei-Ting Vaillancourt</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Min Lu</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah Greenblatt</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Lee</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span>; 
    <span>Present Address:
                        Zentalis Pharmaceuticals, 10835 Road to the Cure, Suite 205, San Diego, CA 92121</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajay Vashisht</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven Bender</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span>; 
    <span>Present Address:
                        NexTx Insights, LLC, San Diego</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Glen Spraggon</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pierre-Yves Michellys</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yong Jia</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacob R. Haling</span> - <span class="hlFld-Affiliation affiliation">Genomics Institute of the Novartis
Research Foundation, 10675 John J. Hopkins Dr., San Diego, California 92121, United States</span>; 
    <span>Present Address:
                        Mirati Therapeutics, 9393 Towne Centre Dr #200, San Diego, CA 92121</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33644" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33644" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank all the supporting functions including Kimberly Sasher and her team (compound management), Thomas Hollenbeck and his team (analytical group) and Xiaoqi Gao (assay development).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AAA+</td><td class="NLM_def"><p class="first last">ATPase associated with diverse cellular activities</p></td></tr><tr><td class="NLM_term">AMP-PNP</td><td class="NLM_def"><p class="first last">adenylyl-imidodiphosphate</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">LONP1</td><td class="NLM_def"><p class="first last">Lon peptidase 1, mitochondrial</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamic</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66870" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66870" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 37 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T. A.</span></span> <span> </span><span class="NLM_article-title">AAA+ proteases: ATP-fueled machines of protein destruction</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">587</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060408-172623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1146%2Fannurev-biochem-060408-172623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=21469952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=587-612&author=R.+T.+Sauerauthor=T.+A.+Baker&title=AAA%2B+proteases%3A+ATP-fueled+machines+of+protein+destruction&doi=10.1146%2Fannurev-biochem-060408-172623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">AAA+ proteases: ATP-fueled machines of protein destruction</span></div><div class="casAuthors">Sauer, Robert T.; Baker, Tania A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">587-612</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  AAA+ family proteolytic machines (ClpXP, ClpAP, ClpCP, HslUV, Lon, FtsH, PAN/20S, and the 26S proteasome) perform protein quality control and are used in regulatory circuits in all cells.  These machines contain a compartmental protease, with active sites sequestered in an interior chamber, and a hexameric ring of AAA+ ATPases.  Substrate proteins are tethered to the ring, either directly or via adaptor proteins.  An unstructured region of the substrate is engaged in the axial pore of the AAA+ ring, and cycles of ATP binding/hydrolysis drive conformational changes that create pulses of pulling that denature the substrate and translocate the unfolded polypeptide through the pore and into the degrdn. chamber.  Here, the authors review the current understanding of the mol. mechanisms of substrate recognition, adaptor function, and ATP-fueled unfolding and translocation.  The unfolding activities of these and related AAA+ machines can also be used to disassemble or remodel macromol. complexes and to resolubilize aggregates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8taxvJbAb9rVg90H21EOLACvtfcHk0lgDVE4J22ogJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbw%253D&md5=76e004df8ead113c31b4df0c44baadcf</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060408-172623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060408-172623%26sid%3Dliteratum%253Aachs%26aulast%3DSauer%26aufirst%3DR.%2BT.%26aulast%3DBaker%26aufirst%3DT.%2BA.%26atitle%3DAAA%252B%2520proteases%253A%2520ATP-fueled%2520machines%2520of%2520protein%2520destruction%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2011%26volume%3D80%26spage%3D587%26epage%3D612%26doi%3D10.1146%2Fannurev-biochem-060408-172623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olivares, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, R. T.</span></span> <span> </span><span class="NLM_article-title">Mechanistic insights into bacterial AAA+ proteases and protein-remodelling machines</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1038/nrmicro.2015.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnrmicro.2015.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=26639779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=33-44&author=A.+O.+Olivaresauthor=T.+A.+Bakerauthor=R.+T.+Sauer&title=Mechanistic+insights+into+bacterial+AAA%2B+proteases+and+protein-remodelling+machines&doi=10.1038%2Fnrmicro.2015.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insights into bacterial AAA+ proteases and protein-remodelling machines</span></div><div class="casAuthors">Olivares, Adrian O.; Baker, Tania A.; Sauer, Robert T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-44</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">To maintain protein homeostasis, AAA+ proteolytic machines degrade damaged and unneeded proteins in bacteria, archaea and eukaryotes.  This process involves the ATP-dependent unfolding of a target protein and its subsequent translocation into a self-compartmentalized proteolytic chamber.  Related AAA+ enzymes also disaggregate and remodel proteins.  Recent structural and biochem. studies, in combination with direct visualization of unfolding and translocation in single-mol. expts., have illuminated the mol. mechanisms behind these processes and suggest how remodelling of macromol. complexes by AAA+ enzymes could occur without global denaturation.  In this Review, we discuss the structural and mechanistic features of AAA+ proteases and remodelling machines, focusing on the bacterial ClpXP and ClpX as paradigms.  We also consider the potential of these enzymes as antibacterial targets and outline future challenges for the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrrZLtmwRVKrVg90H21EOLACvtfcHk0lgDVE4J22ogJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLnM&md5=4154878ee24f6543ae4642cc1096d0e5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro.2015.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro.2015.4%26sid%3Dliteratum%253Aachs%26aulast%3DOlivares%26aufirst%3DA.%2BO.%26aulast%3DBaker%26aufirst%3DT.%2BA.%26aulast%3DSauer%26aufirst%3DR.%2BT.%26atitle%3DMechanistic%2520insights%2520into%2520bacterial%2520AAA%252B%2520proteases%2520and%2520protein-remodelling%2520machines%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2016%26volume%3D14%26spage%3D33%26epage%3D44%26doi%3D10.1038%2Fnrmicro.2015.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bota, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, K. J. A.</span></span> <span> </span><span class="NLM_article-title">Lon protease preferentially degrades oxidized mitochondrial aconitase by an ATP-stimulated mechanism</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/ncb836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fncb836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=12198491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1aht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=674-680&author=D.+A.+Botaauthor=K.+J.+A.+Davies&title=Lon+protease+preferentially+degrades+oxidized+mitochondrial+aconitase+by+an+ATP-stimulated+mechanism&doi=10.1038%2Fncb836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lon protease preferentially degrades oxidized mitochondrial aconitase by an ATP-stimulated mechanism</span></div><div class="casAuthors">Bota, Daniela A.; Davies, Kelvin J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">674-680</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mitochondrial aconitase is sensitive to oxidative inactivation and can aggregate and accumulate in many age-related disorders.  Here we report that Lon protease, an ATP-stimulated mitochondrial matrix protein, selectively recognizes and degrades the oxidized, hydrophobic form of aconitase after mild oxidative modification, but that severe oxidn. results in aconitase aggregation, which makes it a poor substrate for Lon.  Similarly, a morpholino oligodeoxynucleotide directed against the lon gene markedly decreases the amt. of Lon protein, Lon activity and aconitase degrdn. in WI-38 VA-13 human lung fibroblasts and causes accumulation of oxidatively modified aconitase.  The ATP-stimulated Lon protease may be an essential defense against the stress of life in an oxygen environment.  By recognizing minor oxidative changes to protein structure and rapidly degrading the mildly modified protein, Lon protease may prevent extensive oxidn., aggregation and accumulation of aconitase, which could otherwise compromise mitochondrial function and cellular viability.  Aconitase is probably only one of many mitochondrial matrix proteins that are preferentially degraded by Lon protease after oxidative modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3_bTQEgj9tbVg90H21EOLACvtfcHk0lg1MIhWokFlKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1aht70%253D&md5=e24bc60e8ff5684f45e2a312a6a82c8f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncb836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb836%26sid%3Dliteratum%253Aachs%26aulast%3DBota%26aufirst%3DD.%2BA.%26aulast%3DDavies%26aufirst%3DK.%2BJ.%2BA.%26atitle%3DLon%2520protease%2520preferentially%2520degrades%2520oxidized%2520mitochondrial%2520aconitase%2520by%2520an%2520ATP-stimulated%2520mechanism%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2002%26volume%3D4%26spage%3D674%26epage%3D680%26doi%3D10.1038%2Fncb836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. S.</span>; <span class="NLM_string-name">Houry, W. A.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Targeting Human Mitochondrial AAA+ Proteases to Develop Novel Cancer Therapeutics</span>. In  <i>Mitochondria in Health and in Sickness</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbani, A.</span>, <span class="NLM_string-name">Babu, M.</span></span>, Eds.; <span class="NLM_publisher-name">Springer Singapore</span>: <span class="NLM_publisher-loc">Singapore</span>, <span class="NLM_year">2019</span>; pp  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">142</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1007%2F978-981-13-8367-0_8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=119-142&author=K.+S.+Wong&author=W.+A.+Houryauthor=A.+Urbani&author=M.+Babu&title=Mitochondria+in+Health+and+in+Sickness"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2F978-981-13-8367-0_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-981-13-8367-0_8%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DK.%2BS.%26atitle%3DRecent%2520Advances%2520in%2520Targeting%2520Human%2520Mitochondrial%2520AAA%252B%2520Proteases%2520to%2520Develop%2520Novel%2520Cancer%2520Therapeutics%26btitle%3DMitochondria%2520in%2520Health%2520and%2520in%2520Sickness%26aulast%3DUrbani%26aufirst%3DA.%26pub%3DSpringer%2520Singapore%26date%3D2019%26spage%3D119%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glynn, S. E.</span></span> <span> </span><span class="NLM_article-title">Multifunctional mitochondrial AAA proteases</span>. <i>Front. Mol. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">34</span>, <span class="refDoi"> DOI: 10.3389/fmolb.2017.00034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.3389%2Ffmolb.2017.00034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=28589125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaqs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=34&author=S.+E.+Glynn&title=Multifunctional+mitochondrial+AAA+proteases&doi=10.3389%2Ffmolb.2017.00034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Multifunctional mitochondrial AAA proteases</span></div><div class="casAuthors">Glynn, Steven E.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Molecular Biosciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/11</span>CODEN:
                <span class="NLM_cas:coden">FMBRBS</span>;
        ISSN:<span class="NLM_cas:issn">2296-889X</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Mitochondria perform numerous functions necessary for the survival of eukaryotic cells.  These activities are coordinated by a diverse complement of proteins encoded in both the nuclear andmitochondrial genomes thatmust be properly organized andmaintained.  Misregulation of mitochondrial proteostasis impairs organellar function and can result in the development of severe human diseases.  ATP-driven AAA+ proteins play crucial roles in preserving mitochondrial activity by removing and remodeling protein mols. in accordance with the needs of the cell.  Two mitochondrial AAA proteases, i-AAA and m-AAA, are anchored to either face of the mitochondrial inner membrane, where they engage and process an array of substrates to impact protein biogenesis, quality control, and the regulation of key metabolic pathways.  The functionality of these proteases is extended through multiple substrate-dependent modes of action, including complete degrdn., partial processing, or dislocation from the membrane without proteolysis.  This review discusses recent advances made toward elucidating the mechanisms of substrate recognition, handling, and degrdn. that allow these versatile proteases to control diverse activities in this multifunctional organelle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcLJ1O1VbyTLVg90H21EOLACvtfcHk0lg1MIhWokFlKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaqs73E&md5=cd4cdba91674d1e0c83fdbff99e7f683</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3389%2Ffmolb.2017.00034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmolb.2017.00034%26sid%3Dliteratum%253Aachs%26aulast%3DGlynn%26aufirst%3DS.%2BE.%26atitle%3DMultifunctional%2520mitochondrial%2520AAA%2520proteases%26jtitle%3DFront.%2520Mol.%2520Biosci.%26date%3D2017%26volume%3D4%26spage%3D34%26doi%3D10.3389%2Ffmolb.2017.00034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kereïche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovácik, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednár, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunová, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrovicová, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellová, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutejová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raska, I.</span></span> <span> </span><span class="NLM_article-title">The N-terminal domain plays a crucial role in the structure of a full-length human mitochondrial Lon protease</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">33631</span>, <span class="refDoi"> DOI: 10.1038/srep33631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fsrep33631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27632940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFentL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=33631&author=S.+Kere%C3%AFcheauthor=L.+Kov%C3%A1cikauthor=J.+Bedn%C3%A1rauthor=V.+Pevalaauthor=N.+Kunov%C3%A1author=G.+Ondrovicov%C3%A1author=J.+Bauerauthor=L.+Ambroauthor=J.+Bellov%C3%A1author=E.+Kutejov%C3%A1author=I.+Raska&title=The+N-terminal+domain+plays+a+crucial+role+in+the+structure+of+a+full-length+human+mitochondrial+Lon+protease&doi=10.1038%2Fsrep33631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The N-terminal domain plays a crucial role in the structure of a full-length human mitochondrial Lon protease</span></div><div class="casAuthors">Kereiche, Sami; Kovacik, Lubomir; Bednar, Jan; Pevala, Vladimir; Kunova, Nina; Ondrovicova, Gabriela; Bauer, Jacob; Ambro, Lubos; Bellova, Jana; Kutejova, Eva; Raska, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33631</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lon is an essential, multitasking AAA+ protease regulating many cellular processes in species across all kingdoms of life.  Altered expression levels of the human mitochondrial Lon protease (hLon) are linked to serious diseases including myopathies, paraplegia, and cancer.  Here, we present the first 3D structure of full-length hLon using cryo-electron microscopy.  HLon has a unique three-dimensional structure, in which the proteolytic and ATP-binding domains (AP-domain) form a hexameric chamber, while the N-terminal domain is arranged as a trimer of dimers.  These two domains are linked by a narrow trimeric channel composed likely of coiled-coil helixes.  In the presence of AMP-PNP, the AP-domain has a closed-ring conformation and its N-terminal entry gate appears closed, but in ADP binding, it switches to a lock-washer conformation and its N-terminal gate opens, which is accompanied by a rearrangement of the N-terminal domain.  We have also found that both the enzymic activities and the 3D structure of a hLon mutant lacking the first 156 amino acids are severely disturbed, showing that hLon's N-terminal domains are crucial for the overall structure of the hLon, maintaining a conformation allowing its proper functioning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhnhLJCeIMNrVg90H21EOLACvtfcHk0lg1MIhWokFlKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFentL3M&md5=2191d273cefe1a21d180a7e9fde42365</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsrep33631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep33631%26sid%3Dliteratum%253Aachs%26aulast%3DKere%25C3%25AFche%26aufirst%3DS.%26aulast%3DKov%25C3%25A1cik%26aufirst%3DL.%26aulast%3DBedn%25C3%25A1r%26aufirst%3DJ.%26aulast%3DPevala%26aufirst%3DV.%26aulast%3DKunov%25C3%25A1%26aufirst%3DN.%26aulast%3DOndrovicov%25C3%25A1%26aufirst%3DG.%26aulast%3DBauer%26aufirst%3DJ.%26aulast%3DAmbro%26aufirst%3DL.%26aulast%3DBellov%25C3%25A1%26aufirst%3DJ.%26aulast%3DKutejov%25C3%25A1%26aufirst%3DE.%26aulast%3DRaska%26aufirst%3DI.%26atitle%3DThe%2520N-terminal%2520domain%2520plays%2520a%2520crucial%2520role%2520in%2520the%2520structure%2520of%2520a%2520full-length%2520human%2520mitochondrial%2520Lon%2520protease%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D33631%26doi%3D10.1038%2Fsrep33631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snider, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houry, W. A.</span></span> <span> </span><span class="NLM_article-title">The AAA+ superfamily of functionally diverse proteins</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">216</span>, <span class="refDoi"> DOI: 10.1186/gb-2008-9-4-216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1186%2Fgb-2008-9-4-216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=18466635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FivVentQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=216&author=J.+Sniderauthor=G.+Thibaultauthor=W.+A.+Houry&title=The+AAA%2B+superfamily+of+functionally+diverse+proteins&doi=10.1186%2Fgb-2008-9-4-216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The AAA+ superfamily of functionally diverse proteins</span></div><div class="casAuthors">Snider Jamie; Thibault Guillaume; Houry Walid A</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">216</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SUMMARY:  The AAA+ superfamily is a large and functionally diverse superfamily of NTPases that are characterized by a conserved nucleotide-binding and catalytic module, the AAA+ module.  Members are involved in an astonishing range of different cellular processes, attaining this functional diversity through additions of structural motifs and modifications to the core AAA+ module.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRh_wt6tM9EwpotVjB3IRIyfW6udTcc2ebfAPtSIwgxbrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FivVentQ%253D%253D&md5=f50bf3d3c1d1b86e8843f1ac981f5da2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2Fgb-2008-9-4-216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2008-9-4-216%26sid%3Dliteratum%253Aachs%26aulast%3DSnider%26aufirst%3DJ.%26aulast%3DThibault%26aufirst%3DG.%26aulast%3DHoury%26aufirst%3DW.%2BA.%26atitle%3DThe%2520AAA%252B%2520superfamily%2520of%2520functionally%2520diverse%2520proteins%26jtitle%3DGenome%2520Biol.%26date%3D2008%26volume%3D9%26spage%3D216%26doi%3D10.1186%2Fgb-2008-9-4-216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-I.</span></span> <span> </span><span class="NLM_article-title">Structural insights into the allosteric operation of the Lon AAA+ protease</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.str.2016.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27041592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsVWisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=667-675&author=C.-C.+Linauthor=S.-C.+Suauthor=M.-Y.+Suauthor=P.-H.+Liangauthor=C.-C.+Fengauthor=S.-H.+Wuauthor=C.-I.+Chang&title=Structural+insights+into+the+allosteric+operation+of+the+Lon+AAA%2B+protease&doi=10.1016%2Fj.str.2016.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into the Allosteric Operation of the Lon AAA+ Protease</span></div><div class="casAuthors">Lin, Chien-Chu; Su, Shih-Chieh; Su, Ming-Yuan; Liang, Pi-Hui; Feng, Chia-Cheng; Wu, Shih-Hsiung; Chang, Chung-I.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-675</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Lon AAA+ protease (LonA) is an evolutionarily conserved protease that couples the ATPase cycle into motion to drive substrate translocation and degrdn.  A hallmark feature shared by AAA+ proteases is the stimulation of ATPase activity by substrates.  Here the authors report the structure of LonA bound to three ADPs, revealing the first AAA+ protease assembly where the six protomers are arranged alternately in nucleotide-free and bound states.  Nucleotide binding induces large coordinated movements of conserved pore loops from two pairs of three non-adjacent protomers and shuttling of the proteolytic groove between the ATPase site and a previously unknown Arg paddle.  Structural and biochem. evidence supports the roles of the substrate-bound proteolytic groove in allosteric stimulation of ATPase activity and the conserved Arg paddle in driving substrate degrdn.  Altogether, this work provides a mol. framework for understanding how ATP-dependent chemomech. movements drive allosteric processes for substrate degrdn. in a major protein-destruction machine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxui42oJVTprVg90H21EOLACvtfcHk0lg0Xvdg9NZRRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsVWisbk%253D&md5=60e828c93927d00b1754692183f90857</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.-C.%26aulast%3DSu%26aufirst%3DS.-C.%26aulast%3DSu%26aufirst%3DM.-Y.%26aulast%3DLiang%26aufirst%3DP.-H.%26aulast%3DFeng%26aufirst%3DC.-C.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DChang%26aufirst%3DC.-I.%26atitle%3DStructural%2520insights%2520into%2520the%2520allosteric%2520operation%2520of%2520the%2520Lon%2520AAA%252B%2520protease%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D667%26epage%3D675%26doi%3D10.1016%2Fj.str.2016.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-I.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the magnesium-dependent activation and hexamerization of the Lon AAA+ protease</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">676</span>– <span class="NLM_lpage">686</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.str.2016.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27041593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsVWisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=676-686&author=S.-C.+Suauthor=C.-C.+Linauthor=H.-C.+Taiauthor=M.-Y.+Changauthor=M.-R.+Hoauthor=C.+S.+Babuauthor=J.-H.+Liaoauthor=S.-H.+Wuauthor=Y.-C.+Changauthor=C.+Limauthor=C.-I.+Chang&title=Structural+basis+for+the+magnesium-dependent+activation+and+hexamerization+of+the+Lon+AAA%2B+protease&doi=10.1016%2Fj.str.2016.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Magnesium-Dependent Activation and Hexamerization of the Lon AAA+ Protease</span></div><div class="casAuthors">Su, Shih-Chieh; Lin, Chien-Chu; Tai, Hui-Chung; Chang, Mu-Yueh; Ho, Meng-Ru; Babu, C. Satheesan; Liao, Jiahn-Haur; Wu, Shih-Hsiung; Chang, Yuan-Chih; Lim, Carmay; Chang, Chung-I.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">676-686</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Lon AAA+ protease (LonA) plays important roles in protein homeostasis and regulation of diverse biol. processes.  LonA behaves as a homomeric hexamer in the presence of magnesium (Mg2+) and performs ATP-dependent proteolysis.  However, also LonA can carry out Mg2+-dependent degrdn. of unfolded protein substrate in an ATP-independent manner.  In the presence of Mg2+ LonA forms a non-secluded hexameric barrel with prominent openings, which explains why Mg2+-activated LonA can operate as a diffusion-based chambered protease to degrade unstructured protein and peptide substrates efficiently in the absence of ATP.  A 1.85 Å crystal structure of Mg2+-activated protease domain reveals Mg2+-dependent remodeling of a substrate-binding loop and a potential metal-binding site near the Ser-Lys catalytic dyad, supported by biophys. binding assays and mol. dynamics simulations.  Together, these findings reveal the specific roles of Mg2+ in the mol. assembly and activation of LonA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEImndg5i6bVg90H21EOLACvtfcHk0lg0Xvdg9NZRRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsVWisb0%253D&md5=716cec22a38a22fe1c83220d398c98b3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.-C.%26aulast%3DLin%26aufirst%3DC.-C.%26aulast%3DTai%26aufirst%3DH.-C.%26aulast%3DChang%26aufirst%3DM.-Y.%26aulast%3DHo%26aufirst%3DM.-R.%26aulast%3DBabu%26aufirst%3DC.%2BS.%26aulast%3DLiao%26aufirst%3DJ.-H.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DChang%26aufirst%3DY.-C.%26aulast%3DLim%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DC.-I.%26atitle%3DStructural%2520basis%2520for%2520the%2520magnesium-dependent%2520activation%2520and%2520hexamerization%2520of%2520the%2520Lon%2520AAA%252B%2520protease%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D676%26epage%3D686%26doi%3D10.1016%2Fj.str.2016.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishii, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukimoto-Niino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">A redox switch shapes the Lon protease exit pore to facultatively regulate proteolysis</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnchembio.1688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=25383757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKlsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=46-51&author=W.+Nishiiauthor=M.+Kukimoto-Niinoauthor=T.+Teradaauthor=M.+Shirouzuauthor=T.+Muramatsuauthor=M.+Kojimaauthor=H.+Kiharaauthor=S.+Yokoyama&title=A+redox+switch+shapes+the+Lon+protease+exit+pore+to+facultatively+regulate+proteolysis&doi=10.1038%2Fnchembio.1688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A redox switch shapes the Lon protease exit pore to facultatively regulate proteolysis</span></div><div class="casAuthors">Nishii, Wataru; Kukimoto-Niino, Mutsuko; Terada, Takaho; Shirouzu, Mikako; Muramatsu, Tomonari; Kojima, Masaki; Kihara, Hiroshi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-51</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Lon AAA+ protease degrades damaged or misfolded proteins in its intramol. chamber.  Its activity must be precisely controlled, but the mechanism by which Lon is regulated in response to different environments is not known.  Facultative anaerobes in the Enterobacteriaceae family, mostly symbionts and pathogens, encounter both anaerobic and aerobic environments inside and outside the host's body, resp.  The bacteria characteristically have two cysteine residues on the Lon protease (P) domain surface that unusually form a disulfide bond.  Here we show that the cysteine residues act as a redox switch of Lon.  Upon disulfide bond redn., the exit pore of the P-domain ring narrows by ∼30%, thus interrupting product passage and decreasing activity by 80%; disulfide bonding by oxidn. restores the pore size and activity.  The redox switch (E°' = -227 mV) is appropriately tuned to respond to variation between anaerobic and aerobic conditions, thus optimizing the cellular proteolysis level for each environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKuaoFS5ZXJrVg90H21EOLACvtfcHk0lj4Ovg_l_0ldg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKlsrrN&md5=758d7025e61e175e4419496b61f8fef4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1688%26sid%3Dliteratum%253Aachs%26aulast%3DNishii%26aufirst%3DW.%26aulast%3DKukimoto-Niino%26aufirst%3DM.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DKojima%26aufirst%3DM.%26aulast%3DKihara%26aufirst%3DH.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DA%2520redox%2520switch%2520shapes%2520the%2520Lon%2520protease%2520exit%2520pore%2520to%2520facultatively%2520regulate%2520proteolysis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D46%26epage%3D51%26doi%3D10.1038%2Fnchembio.1688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dikoglu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfaiz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertola, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, T.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derbent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alanay, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, O.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llerenar Jr, J. C.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos-Xavier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonafé, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unger, S.</span></span> <span> </span><span class="NLM_article-title">Mutations in LONP1, a mitochondrial matrix protease, cause CODAS syndrome</span>. <i>Am. J. Med. Genet., Part A</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1501</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1002/ajmg.a.37029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1002%2Fajmg.a.37029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=25808063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVegsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2015&pages=1501-1509&author=E.+Dikogluauthor=A.+Alfaizauthor=M.+Gornaauthor=D.+Bertolaauthor=J.+H.+Chaeauthor=T.-J.+Choauthor=M.+Derbentauthor=Y.+Alanayauthor=T.+Guranauthor=O.-H.+Kimauthor=J.+C.+Llerenar+Jrauthor=G.+Yamamotoauthor=G.+Superti-Furgaauthor=A.+Reymondauthor=I.+Xenariosauthor=B.+Stevensonauthor=B.+Campos-Xavierauthor=L.+Bonaf%C3%A9author=A.+Superti-Furgaauthor=S.+Unger&title=Mutations+in+LONP1%2C+a+mitochondrial+matrix+protease%2C+cause+CODAS+syndrome&doi=10.1002%2Fajmg.a.37029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in LONP1, a mitochondrial matrix protease, cause CODAS syndrome</span></div><div class="casAuthors">Dikoglu, Esra; Alfaiz, Ali; Gorna, Maria; Bertola, Deborah; Chae, Jong Hee; Cho, Tae-Joon; Derbent, Murat; Alanay, Yasemin; Guran, Tulay; Kim, Ok-Hwa; Llerenar, Juan C., jr.; Yamamoto, Guillerme; Superti-Furga, Giulio; Reymond, Alexandre; Xenarios, Ioannis; Stevenson, Brian; Campos-Xavier, Belinda; Bonafe, Luisa; Superti-Furga, Andrea; Unger, Sheila</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Medical Genetics, Part A</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1501-1509</span>CODEN:
                <span class="NLM_cas:coden">AJMGB8</span>;
        ISSN:<span class="NLM_cas:issn">1552-4825</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cerebral, ocular, dental, auricular, skeletal anomalies (CODAS) syndrome (MIM 600373) was first described and named by Shehib et al, in 1991 in a single patient.  The anomalies referred to in the acronym are as follows: cerebral-developmental delay, ocular-cataracts, dental-aberrant cusp morphol. and delayed eruption, auricular-malformations of the external ear, and skeletal-spondyloepiphyseal dysplasia.  This distinctive constellation of anatomical findings should allow easy recognition but despite this only four apparently sporadic patients have been reported in the last 20 years indicating that the full phenotype is indeed very rare with perhaps milder or a typical presentations that are allelic but without sufficient phenotypic resemblance to permit clin. diagnosis.  We performed exome sequencing in three patients (an isolated case and a brother and sister sib pair) with classical features of CODAS.  Sanger sequencing was used to confirm results as well as for mutation discovery in a further four unrelated patients ascertained via their skeletal features.  Compd. heterozygous or homozygous mutations in LONP1 were found in all (8 sep. mutations; 6 missense, 1 nonsense, 1 small in-frame deletion) thus establishing the genetic basis of CODAS and the pattern of inheritance (autosomal recessive).  LONP1 encodes an enzyme of bacterial ancestry that participates in protein turnover within the mitochondrial matrix.  The mutations cluster at the ATP-binding and proteolytic domains of the enzyme.  Biallelic inheritance and clustering of mutations confirm dysfunction of LONP1 activity as the mol. basis of CODAS but the pathogenesis remains to be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnLIR81-tzQrVg90H21EOLACvtfcHk0lj4Ovg_l_0ldg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVegsrzE&md5=bc26ae2d47d630eb0a161bb1519b2ec0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fajmg.a.37029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fajmg.a.37029%26sid%3Dliteratum%253Aachs%26aulast%3DDikoglu%26aufirst%3DE.%26aulast%3DAlfaiz%26aufirst%3DA.%26aulast%3DGorna%26aufirst%3DM.%26aulast%3DBertola%26aufirst%3DD.%26aulast%3DChae%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DT.-J.%26aulast%3DDerbent%26aufirst%3DM.%26aulast%3DAlanay%26aufirst%3DY.%26aulast%3DGuran%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DO.-H.%26aulast%3DLlerenar%2BJr%26aufirst%3DJ.%2BC.%26aulast%3DYamamoto%26aufirst%3DG.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DReymond%26aufirst%3DA.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DStevenson%26aufirst%3DB.%26aulast%3DCampos-Xavier%26aufirst%3DB.%26aulast%3DBonaf%25C3%25A9%26aufirst%3DL.%26aulast%3DSuperti-Furga%26aufirst%3DA.%26aulast%3DUnger%26aufirst%3DS.%26atitle%3DMutations%2520in%2520LONP1%252C%2520a%2520mitochondrial%2520matrix%2520protease%252C%2520cause%2520CODAS%2520syndrome%26jtitle%3DAm.%2520J.%2520Med.%2520Genet.%252C%2520Part%2520A%26date%3D2015%26volume%3D167%26spage%3D1501%26epage%3D1509%26doi%3D10.1002%2Fajmg.a.37029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddington, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oláhová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommerville, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siibak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrzanowska-Lightowlers, Z. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightowlers, R. N.</span></span> <span> </span><span class="NLM_article-title">Defective mitochondrial protease LonP1 can cause classical mitochondrial disease</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1743</span>– <span class="NLM_lpage">1753</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddy080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1093%2Fhmg%2Fddy080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=29518248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGmtLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=1743-1753&author=B.+Peterauthor=C.+L.+Waddingtonauthor=M.+Ol%C3%A1hov%C3%A1author=E.+W.+Sommervilleauthor=S.+Hoptonauthor=A.+Pyleauthor=M.+Championauthor=M.+Ohlsonauthor=T.+Siibakauthor=Z.+M.+A.+Chrzanowska-Lightowlersauthor=R.+W.+Taylorauthor=M.+Falkenbergauthor=R.+N.+Lightowlers&title=Defective+mitochondrial+protease+LonP1+can+cause+classical+mitochondrial+disease&doi=10.1093%2Fhmg%2Fddy080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Defective mitochondrial protease LonP1 can cause classical mitochondrial disease</span></div><div class="casAuthors">Peter, Bradley; Waddington, Christie L.; Olahova, Monika; Sommerville, Ewen W.; Hopton, Sila; Pyle, Angela; Champion, Michael; Ohlson, Monica; Siibak, Triinu; Chrzanowska-Lightowlers, Zofia M. A.; Taylor, Robert W.; Falkenberg, Maria; Lightowlers, Robert N.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1743-1753</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">1460-2083</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">LonP1 is a mitochondrial matrix protease whose selective substrate specificity is essential for maintaining mitochondrial homeostasis.  Recessively inherited, pathogenic defects in LonP1 have been previously reported to underlie cerebral, ocular, dental, auricular and skeletal anomalies (CODAS) syndrome, a complex multisystemic and developmental disorder.  Intriguingly, although classical mitochondrial disease presentations are well-known to exhibit marked clin. heterogeneity, the skeletal and dental features assocd. with CODAS syndrome are pathognomonic.  We have applied whole exome sequencing to a patient with congenital lactic acidosis, muscle weakness, profound deficiencies in mitochondrial oxidative phosphorylation assocd. with loss of mtDNA copy no. and MRI abnormalities consistent with Leigh syndrome, identifying biallelic variants in the LONP1 (NM_004793.3) gene; c.1693T > C predicting p.(Tyr565His) and c.2197G > A predicting p.(Glu733Lys); no evidence of the classical skeletal or dental defects obsd. in CODAS syndrome patients were noted in our patient.  In vitro expts. confirmed the p.(Tyr565His) LonP1 mutant alone could not bind or degrade a substrate, consistent with the predicted function of Tyr565, while a second missense [p.(Glu733Lys)] variant had minimal effect.  Mixts. of p.(Tyr565His) mutant and wild-type LonP1 retained partial protease activity but this was severely depleted when the p.(Tyr565His) mutant was mixed with the p.(Glu733Lys) mutant, data consistent with the compd. heterozygosity detected in our patient.  In summary, we conclude that pathogenic LONP1 variants can lead to a classical mitochondrial disease presentations assocd. with severe biochem. defects in oxidative phosphorylation in clin. relevant tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHN_4cc2OkKLVg90H21EOLACvtfcHk0lj4Ovg_l_0ldg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGmtLjJ&md5=61601e3cdbc5ed404f792908486e02d4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddy080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddy080%26sid%3Dliteratum%253Aachs%26aulast%3DPeter%26aufirst%3DB.%26aulast%3DWaddington%26aufirst%3DC.%2BL.%26aulast%3DOl%25C3%25A1hov%25C3%25A1%26aufirst%3DM.%26aulast%3DSommerville%26aufirst%3DE.%2BW.%26aulast%3DHopton%26aufirst%3DS.%26aulast%3DPyle%26aufirst%3DA.%26aulast%3DChampion%26aufirst%3DM.%26aulast%3DOhlson%26aufirst%3DM.%26aulast%3DSiibak%26aufirst%3DT.%26aulast%3DChrzanowska-Lightowlers%26aufirst%3DZ.%2BM.%2BA.%26aulast%3DTaylor%26aufirst%3DR.%2BW.%26aulast%3DFalkenberg%26aufirst%3DM.%26aulast%3DLightowlers%26aufirst%3DR.%2BN.%26atitle%3DDefective%2520mitochondrial%2520protease%2520LonP1%2520can%2520cause%2520classical%2520mitochondrial%2520disease%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2018%26volume%3D27%26spage%3D1743%26epage%3D1753%26doi%3D10.1093%2Fhmg%2Fddy080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span> <span> </span><span class="NLM_article-title">Down-regulating overexpressed human Lon in cervical cancer suppresses cell proliferation and bioenergetics</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e81084</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0081084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1371%2Fjournal.pone.0081084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=24260536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=X.+Nieauthor=M.+Liauthor=B.+Luauthor=Y.+Zhangauthor=L.+Lanauthor=L.+Chenauthor=J.+Lu&title=Down-regulating+overexpressed+human+Lon+in+cervical+cancer+suppresses+cell+proliferation+and+bioenergetics&doi=10.1371%2Fjournal.pone.0081084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Down-regulating overexpressed human Lon in cervical cancer suppresses cell proliferation and bioenergetics</span></div><div class="casAuthors">Nie, Xiaobo; Li, Min; Lu, Bin; Zhang, Yuxin; Lan, Linhua; Chen, Lin; Lu, Jianxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e81084/1-e81084/9</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human mitochondrial ATP-dependent Lon protease functions in regulating the metab. and quality control of proteins and mitochondrial DNA (mtDNA).  However, the role of Lon in cancer is not well understood.  Therefore, this study was undertaken to investigate the importance of Lon in cervical cancer cells from patients and in established cell lines.  Microarray anal. from 30 cancer and 10 normal cervical tissues were analyzed by immunohistochem. for Lon protein levels.  The expression of Lon was also examd. by immunoblotting 16 fresh cervical cancer tissues and their resp. non-tumor cervical tissues.  In all cases, Lon expression was significantly elevated in cervical carcinomas as compared to normal tissues.  Augmented Lon expression in tissue microarrays did not vary between age, tumor-node-metastasis grades or lymph node metastasis.  Knocking down Lon in HeLa cervical cancer cells by lentivrial transduction resulted in a substantial decrease in both mRNA and protein levels.  Such down-regulation of Lon expression significantly blocked HeLa cell proliferation.  In addn., knocking down Lon resulted in decreased cellular bioenergetics as detd. by measuring aerobic respiration and glycolysis using the Seahorse XF24 extracellular flux analyzer.  Together, these data demonstrate that Lon plays a potential role in the oncogenesis of cervical cancer and may be a useful biomarker and target in the treatment of cervical cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0Hgd0bBBkk7Vg90H21EOLACvtfcHk0lh5hnYO3fyvhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2hsw%253D%253D&md5=3d82ccdf7d96509514ac59c4e2f8fad0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0081084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0081084%26sid%3Dliteratum%253Aachs%26aulast%3DNie%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DDown-regulating%2520overexpressed%2520human%2520Lon%2520in%2520cervical%2520cancer%2520suppresses%2520cell%2520proliferation%2520and%2520bioenergetics%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0081084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quirós, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Español, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acín-Pérez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bárcena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loureiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-García, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fueyo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enríquez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Otín, C.</span></span> <span> </span><span class="NLM_article-title">ATP-dependent Lon protease controls tumor bioenergetics by reprogramming mitochondrial activity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.celrep.2014.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=25017063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=542-556&author=P.+M.+Quir%C3%B3sauthor=Y.+Espa%C3%B1olauthor=R.+Ac%C3%ADn-P%C3%A9rezauthor=F.+Rodr%C3%ADguezauthor=C.+B%C3%A1rcenaauthor=K.+Watanabeauthor=E.+Calvoauthor=M.+Loureiroauthor=M.+S.+Fern%C3%A1ndez-Garc%C3%ADaauthor=A.+Fueyoauthor=J.+V%C3%A1zquezauthor=J.+A.+Enr%C3%ADquezauthor=C.+L%C3%B3pez-Ot%C3%ADn&title=ATP-dependent+Lon+protease+controls+tumor+bioenergetics+by+reprogramming+mitochondrial+activity&doi=10.1016%2Fj.celrep.2014.06.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-Dependent Lon Protease Controls Tumor Bioenergetics by Reprogramming Mitochondrial Activity</span></div><div class="casAuthors">Quiros, Pedro M.; Espanol, Yaiza; Acin-Perez, Rebeca; Rodriguez, Francisco; Barcena, Clea; Watanabe, Kenta; Calvo, Enrique; Loureiro, Marta; Fernandez-Garcia, M. Soledad; Fueyo, Antonio; Vazquez, Jesus; Enriquez, Jose Antonio; Lopez-Otin, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">542-556</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We generated mice deficient in Lon protease (LONP1), a major enzyme of the mitochondrial quality control machinery.  Homozygous deletion of Lonp1 causes early embryonic lethality, whereas its haploinsufficiency protects against colorectal and skin tumors.  Furthermore, LONP1 knockdown inhibits cellular proliferation and tumor and metastasis formation, whereas its overexpression increases tumorigenesis.  Clin. studies indicate that high levels of LONP1 are a poor prognosis marker in human colorectal cancer and melanoma.  Addnl., functional analyses show that LONP1 plays a key role in metabolic reprogramming by remodeling OXPHOS complexes and protecting against senescence.  Our findings demonstrate the relevance of LONP1 for cellular and organismal viability and identify this protease as a central regulator of mitochondrial activity in oncogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRD-inqp2_r7Vg90H21EOLACvtfcHk0lh5hnYO3fyvhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FI&md5=f85dcdc898cf61a442c4093f4b4661a0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DQuir%25C3%25B3s%26aufirst%3DP.%2BM.%26aulast%3DEspa%25C3%25B1ol%26aufirst%3DY.%26aulast%3DAc%25C3%25ADn-P%25C3%25A9rez%26aufirst%3DR.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DF.%26aulast%3DB%25C3%25A1rcena%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DK.%26aulast%3DCalvo%26aufirst%3DE.%26aulast%3DLoureiro%26aufirst%3DM.%26aulast%3DFern%25C3%25A1ndez-Garc%25C3%25ADa%26aufirst%3DM.%2BS.%26aulast%3DFueyo%26aufirst%3DA.%26aulast%3DV%25C3%25A1zquez%26aufirst%3DJ.%26aulast%3DEnr%25C3%25ADquez%26aufirst%3DJ.%2BA.%26aulast%3DL%25C3%25B3pez-Ot%25C3%25ADn%26aufirst%3DC.%26atitle%3DATP-dependent%2520Lon%2520protease%2520controls%2520tumor%2520bioenergetics%2520by%2520reprogramming%2520mitochondrial%2520activity%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D542%26epage%3D556%26doi%3D10.1016%2Fj.celrep.2014.06.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Biasi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span> <span> </span><span class="NLM_article-title">Emerging role of Lon protease as a master regulator of mitochondrial functions</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1857</i></span>,  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1306</span>, <span class="refDoi"> DOI: 10.1016/j.bbabio.2016.03.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.bbabio.2016.03.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27033304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1OjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1857&publication_year=2016&pages=1300-1306&author=M.+Pintiauthor=L.+Gibelliniauthor=M.+Nasiauthor=S.+De+Biasiauthor=C.+A.+Bortolottiauthor=A.+Iannoneauthor=A.+Cossarizza&title=Emerging+role+of+Lon+protease+as+a+master+regulator+of+mitochondrial+functions&doi=10.1016%2Fj.bbabio.2016.03.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging role of Lon protease as a master regulator of mitochondrial functions</span></div><div class="casAuthors">Pinti, Marcello; Gibellini, Lara; Nasi, Milena; De Biasi, Sara; Bortolotti, Carlo Augusto; Iannone, Anna; Cossarizza, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Bioenergetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1857</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1300-1306</span>CODEN:
                <span class="NLM_cas:coden">BBBEB4</span>;
        ISSN:<span class="NLM_cas:issn">0005-2728</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">Lon protease is a nuclear-encoded, mitochondrial ATP-dependent protease highly conserved throughout the evolution, crucial for the maintenance of mitochondrial homeostasis.  Lon acts as a chaperone of misfolded proteins, and is necessary for maintaining mitochondrial DNA.  The impairment of these functions has a deep impact on mitochondrial functionality and morphol.  An altered expression of Lon leads to a profound reprogramming of cell metab., with a switch from respiration to glycolysis, which is often obsd. in cancer cells.  Mutations of Lon, which likely impair its chaperone properties, are at the basis of a genetic inherited disease named of the cerebral, ocular, dental, auricular, skeletal (CODAS) syndrome.  This article is part of a Special Issue entitled 'EBEC 2016: 19th European Bioenergetics Conference, Riva del Garda, Italy, July 2-6, 2016', edited by Prof.  Paolo Bernardi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroqYZs21ZfILVg90H21EOLACvtfcHk0lh5hnYO3fyvhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1OjsLw%253D&md5=d4a141d356d944d01103d6845b60e19c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbabio.2016.03.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbabio.2016.03.025%26sid%3Dliteratum%253Aachs%26aulast%3DPinti%26aufirst%3DM.%26aulast%3DGibellini%26aufirst%3DL.%26aulast%3DNasi%26aufirst%3DM.%26aulast%3DDe%2BBiasi%26aufirst%3DS.%26aulast%3DBortolotti%26aufirst%3DC.%2BA.%26aulast%3DIannone%26aufirst%3DA.%26aulast%3DCossarizza%26aufirst%3DA.%26atitle%3DEmerging%2520role%2520of%2520Lon%2520protease%2520as%2520a%2520master%2520regulator%2520of%2520mitochondrial%2520functions%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2016%26volume%3D1857%26spage%3D1300%26epage%3D1306%26doi%3D10.1016%2Fj.bbabio.2016.03.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morozov, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogenhagen, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temiakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, C. K.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of human TFAM in mitochondria impairs DNA binding and promotes degradation by the AAA+ Lon protease</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.molcel.2012.10.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=23201127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqsrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=121-132&author=B.+Luauthor=J.+Leeauthor=X.+Nieauthor=M.+Liauthor=Y.+I.+Morozovauthor=S.+Venkateshauthor=D.+F.+Bogenhagenauthor=D.+Temiakovauthor=C.+K.+Suzuki&title=Phosphorylation+of+human+TFAM+in+mitochondria+impairs+DNA+binding+and+promotes+degradation+by+the+AAA%2B+Lon+protease&doi=10.1016%2Fj.molcel.2012.10.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of Human TFAM in Mitochondria Impairs DNA Binding and Promotes Degradation by the AAA+ Lon Protease</span></div><div class="casAuthors">Lu, Bin; Lee, Jae; Nie, Xiaobo; Li, Min; Morozov, Yaroslav I.; Venkatesh, Sundararajan; Bogenhagen, Daniel F.; Temiakov, Dmitry; Suzuki, Carolyn K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-132</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Human mitochondrial transcription factor A (TFAM) is a high-mobility group (HMG) protein at the nexus of mitochondrial DNA (mtDNA) replication, transcription, and inheritance.  Little is known about the mechanisms underlying its posttranslational regulation.  Here, we demonstrate that TFAM is phosphorylated within its HMG box 1 (HMG1) by cAMP-dependent protein kinase in mitochondria.  HMG1 phosphorylation impairs the ability of TFAM to bind DNA and to activate transcription.  We show that only DNA-free TFAM is degraded by the Lon protease, which is inhibited by the anticancer drug bortezomib.  In cells with normal mtDNA levels, HMG1-phosphorylated TFAM is degraded by Lon.  However, in cells with severe mtDNA deficits, nonphosphorylated TFAM is also degraded, as it is DNA free.  Depleting Lon in these cells increases levels of TFAM and upregulates mtDNA content, albeit transiently.  Phosphorylation and proteolysis thus provide mechanisms for rapid fine-tuning of TFAM function and abundance in mitochondria, which are crucial for maintaining and expressing mtDNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr-71KRef3PLVg90H21EOLACvtfcHk0liGXi8ZLuIBkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqsrbI&md5=74f6241a069401d65779a6e6894df180</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DNie%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DMorozov%26aufirst%3DY.%2BI.%26aulast%3DVenkatesh%26aufirst%3DS.%26aulast%3DBogenhagen%26aufirst%3DD.%2BF.%26aulast%3DTemiakov%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DC.%2BK.%26atitle%3DPhosphorylation%2520of%2520human%2520TFAM%2520in%2520mitochondria%2520impairs%2520DNA%2520binding%2520and%2520promotes%2520degradation%2520by%2520the%2520AAA%252B%2520Lon%2520protease%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D49%26spage%3D121%26epage%3D132%26doi%3D10.1016%2Fj.molcel.2012.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bárcena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Otín, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittig, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, G.</span></span> <span> </span><span class="NLM_article-title">Global proteome of LonP1+/– mouse embryonal fibroblasts reveals impact on respiratory chain, but no interdependence between eral1 and mitoribosomes</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4523</span>, <span class="refDoi"> DOI: 10.3390/ijms20184523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.3390%2Fijms20184523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFSnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=4523&author=J.+Keyauthor=A.+Kohliauthor=C.+B%C3%A1rcenaauthor=C.+L%C3%B3pez-Ot%C3%ADnauthor=J.+Heidlerauthor=I.+Wittigauthor=G.+Auburger&title=Global+proteome+of+LonP1%2B%2F%E2%80%93+mouse+embryonal+fibroblasts+reveals+impact+on+respiratory+chain%2C+but+no+interdependence+between+eral1+and+mitoribosomes&doi=10.3390%2Fijms20184523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Global proteome of LonP1+/-mouse embryonal fibroblasts reveals impact on respiratory chain, but no interdependence between Eral1 and mitoribosomes</span></div><div class="casAuthors">Key, Jana; Kohli, Aneesha; Barcena, Clea; Lopez-Otin, Carlos; Heidler, Juliana; Wittig, Ilka; Auburger, Georg</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4523</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Research on healthy aging shows that lifespan redns. are often caused by mitochondrial dysfunction.  Thus, it is very interesting that the deletion of mitochondrial matrix peptidase LonP1 was obsd. to abolish embryogenesis, while deletion of the mitochondrial matrix peptidase Caseinolytic Mitochondrial Matrix Peptidase Proteolytic Subunit (ClpP) prolonged survival.  To unveil the targets of each enzyme, we documented the global proteome of LonP1+/- mouse embryonal fibroblasts (MEF), for comparison with ClpP-/- depletion.  Proteomic profiles of LonP1+/- MEF generated by label-free mass spectrometry were further processed with the STRING (Search tool for the retrieval of interacting genes) webserver Heidelberg for protein interactions.  ClpP was previously reported to degrade Eral1 as a chaperone involved in mitoribosome assembly, so ClpP deficiency triggers the accumulation of mitoribosomal subunits and inefficient translation.  LonP1+/- MEF also showed Eral1 accumulation, but no systematic effect on mitoribosomal subunits.  In contrast to ClpP-/- profiles, several components of the respiratory complex-I membrane arm, of the glutathione pathway and of lysosomes were accumulated, whereas the upregulation of numerous innate immune defense components was similar.  Overall, LonP1, as opposed to ClpP, appears to have no effect on translational machinery, instead it shows enhanced respiratory dysfunction; this agrees with reports on the human CODAS syndrome (syndrome with cerebral, ocular, dental, auricular, and skeletal anomalies) caused by LonP1 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Gnv13iaGA7Vg90H21EOLACvtfcHk0liGXi8ZLuIBkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFSnsLs%253D&md5=410002d95971ead3a99ecb19bb5634da</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3390%2Fijms20184523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20184523%26sid%3Dliteratum%253Aachs%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DKohli%26aufirst%3DA.%26aulast%3DB%25C3%25A1rcena%26aufirst%3DC.%26aulast%3DL%25C3%25B3pez-Ot%25C3%25ADn%26aufirst%3DC.%26aulast%3DHeidler%26aufirst%3DJ.%26aulast%3DWittig%26aufirst%3DI.%26aulast%3DAuburger%26aufirst%3DG.%26atitle%3DGlobal%2520proteome%2520of%2520LonP1%252B%252F%25E2%2580%2593%2520mouse%2520embryonal%2520fibroblasts%2520reveals%2520impact%2520on%2520respiratory%2520chain%252C%2520but%2520no%2520interdependence%2520between%2520eral1%2520and%2520mitoribosomes%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D4523%26doi%3D10.3390%2Fijms20184523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Biasi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo Tartaro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecorini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patergnani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Gaetano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carnevale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisciotta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span> <span> </span><span class="NLM_article-title">LonP1 differently modulates mitochondrial function and bioenergetics of primary versus metastatic colon cancer cells</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">254</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.3389%2Ffonc.2018.00254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=30038898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ht1Crtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=254&author=L.+Gibelliniauthor=L.+Losiauthor=S.+De+Biasiauthor=M.+Nasiauthor=D.+Lo+Tartaroauthor=S.+Pecoriniauthor=S.+Patergnaniauthor=P.+Pintonauthor=A.+De+Gaetanoauthor=G.+Carnevaleauthor=A.+Pisciottaauthor=F.+Marianiauthor=L.+Roncucciauthor=A.+Iannoneauthor=A.+Cossarizzaauthor=M.+Pinti&title=LonP1+differently+modulates+mitochondrial+function+and+bioenergetics+of+primary+versus+metastatic+colon+cancer+cells&doi=10.3389%2Ffonc.2018.00254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LonP1 Differently Modulates Mitochondrial Function and Bioenergetics of Primary Versus Metastatic Colon Cancer Cells</span></div><div class="casAuthors">Gibellini Lara; Lo Tartaro Domenico; Cossarizza Andrea; Losi Lorena; De Biasi Sara; Pinti Marcello; Nasi Milena; Pecorini Simone; De Gaetano Anna; Carnevale Gianluca; Pisciotta Alessandra; Patergnani Simone; Pinton Paolo; Mariani Francesco; Roncucci Luca; Iannone Anna</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">254</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Mitochondrial Lon protease (LonP1) is a multi-function enzyme that regulates mitochondrial functions in several human malignancies, including colorectal cancer (CRC).  The mechanism(s) by which LonP1 contributes to colorectal carcinogenesis is not fully understood.  We found that silencing LonP1 leads to severe mitochondrial impairment and apoptosis in colon cancer cells.  Here, we investigate the role of LonP1 in mitochondrial functions, metabolism, and epithelial-mesenchymal transition (EMT) in colon tumor cells and in metastasis.  LonP1 was almost absent in normal mucosa, gradually increased from aberrant crypt foci to adenoma, and was most abundant in CRC.  Moreover, LonP1 was preferentially upregulated in colorectal samples with mutated p53 or nuclear β-catenin, and its overexpression led to increased levels of β-catenin and decreased levels of E-cadherin, key proteins in EMT, in vitro.  LonP1 upregulation also induced opposite changes in oxidative phosphorylation, glycolysis, and pentose pathway in SW480 primary colon tumor cells when compared to SW620 metastatic colon cancer cells.  In conclusion, basal LonP1 expression is essential for normal mitochondrial function, and increased LonP1 levels in SW480 and SW620 cells induce a metabolic shift toward glycolysis, leading to EMT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRR3-Hvit6XqDslzEEsg76ufW6udTcc2ebN9nQA8knXyLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ht1Crtg%253D%253D&md5=369dea5c17a0ffe9c3e2854c37b0d050</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00254%26sid%3Dliteratum%253Aachs%26aulast%3DGibellini%26aufirst%3DL.%26aulast%3DLosi%26aufirst%3DL.%26aulast%3DDe%2BBiasi%26aufirst%3DS.%26aulast%3DNasi%26aufirst%3DM.%26aulast%3DLo%2BTartaro%26aufirst%3DD.%26aulast%3DPecorini%26aufirst%3DS.%26aulast%3DPatergnani%26aufirst%3DS.%26aulast%3DPinton%26aufirst%3DP.%26aulast%3DDe%2BGaetano%26aufirst%3DA.%26aulast%3DCarnevale%26aufirst%3DG.%26aulast%3DPisciotta%26aufirst%3DA.%26aulast%3DMariani%26aufirst%3DF.%26aulast%3DRoncucci%26aufirst%3DL.%26aulast%3DIannone%26aufirst%3DA.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DPinti%26aufirst%3DM.%26atitle%3DLonP1%2520differently%2520modulates%2520mitochondrial%2520function%2520and%2520bioenergetics%2520of%2520primary%2520versus%2520metastatic%2520colon%2520cancer%2520cells%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D254%26doi%3D10.3389%2Ffonc.2018.00254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nchembio1207-739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnchembio1207-739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=18007642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=739-744&author=W.+A.+Weissauthor=S.+S.+Taylorauthor=K.+M.+Shokat&title=Recognizing+and+exploiting+differences+between+RNAi+and+small-molecule+inhibitors&doi=10.1038%2Fnchembio1207-739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span></div><div class="casAuthors">Weiss, William A.; Taylor, Stephen S.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The biol. of RNA interference has greatly facilitated anal. of loss-of-function phenotypes, but correlating these phenotypes with small-mol. inhibition profiles is not always straightforward.  The authors examine the rationale of comparing RNA interference to pharmacol. intervention in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JPzqVyshS7Vg90H21EOLACvtfcHk0ljS5vGOaAgHEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO&md5=9a26c990b358a6b2c717d1e361cac2da</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1207-739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1207-739%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DRecognizing%2520and%2520exploiting%2520differences%2520between%2520RNAi%2520and%2520small-molecule%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnchembio1207-739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span> <span> </span><span class="NLM_article-title">Target 2035: probing the human proteome</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2111</span>– <span class="NLM_lpage">2115</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2019.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.drudis.2019.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=31278990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2111-2115&author=A.+J.+Carterauthor=O.+Kraemerauthor=M.+Zwickauthor=A.+Mueller-Fahrnowauthor=C.+H.+Arrowsmithauthor=A.+M.+Edwards&title=Target+2035%3A+probing+the+human+proteome&doi=10.1016%2Fj.drudis.2019.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Target 2035: probing the human proteome</span></div><div class="casAuthors">Carter, Adrian J.; Kraemer, Oliver; Zwick, Matthias; Mueller-Fahrnow, Anke; Arrowsmith, Cheryl H.; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2111-2115</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Biomedical scientists tend to focus on only a small fraction of the proteins encoded by the human genome despite overwhelming genetic evidence that many understudied proteins are important for human disease.  One of the best ways to interrogate the function of a protein and to det. its relevance as a drug target is by using a pharmacol. modulator, such as a chem. probe or an antibody.  If these tools were available for most human proteins, it should be possible to translate the tremendous advances in genomics into a greater understanding of human health and disease, and catalyze the creation of innovative new medicines.  Target 2035 is a global federation for developing and applying new technologies with the goal of creating chemogenomic libraries, chem. probes, and/or functional antibodies for the entire proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY5b9ZXh0ALVg90H21EOLACvtfcHk0ljS5vGOaAgHEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN&md5=212e832238c8860439b343cbeef17b74</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DZwick%26aufirst%3DM.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DTarget%25202035%253A%2520probing%2520the%2520human%2520proteome%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D2111%26epage%3D2115%26doi%3D10.1016%2Fj.drudis.2019.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Melderen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertsen, A.</span></span> <span> </span><span class="NLM_article-title">Regulation and quality control by Lon-dependent proteolysis</span>. <i>Res. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">645</span>– <span class="NLM_lpage">651</span>, <span class="refDoi"> DOI: 10.1016/j.resmic.2009.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.resmic.2009.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=19772918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVens7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2009&pages=645-651&author=L.+Van%0AMelderenauthor=A.+Aertsen&title=Regulation+and+quality+control+by+Lon-dependent+proteolysis&doi=10.1016%2Fj.resmic.2009.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and quality control by Lon-dependent proteolysis</span></div><div class="casAuthors">Van Melderen, Laurence; Aertsen, Abram</div><div class="citationInfo"><span class="NLM_cas:title">Research in Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">645-651</span>CODEN:
                <span class="NLM_cas:coden">RMCREW</span>;
        ISSN:<span class="NLM_cas:issn">0923-2508</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  After their first discovery in Escherichia coli, Lon homologues were found to be widely distributed among prokaryotes to eukaryotes.  The ATP-dependent Lon protease belongs to the AAA+ (ATPases assocd. with a variety of cellular activities) superfamily, and is involved in both general quality control by degrading abnormal proteins and in the specific control of several regulatory proteins.  As such, this enzyme has a pivotal role in quality control and cellular physiol.  This review focuses on mechanisms of degrdn. both from the protease and substrate points of view, and discusses the role of Lon in global regulation, stress response and virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqrcB5DNPwErVg90H21EOLACvtfcHk0ljS5vGOaAgHEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVens7bN&md5=7f16df349f0a135b0899703b578f339a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.resmic.2009.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.resmic.2009.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BMelderen%26aufirst%3DL.%26aulast%3DAertsen%26aufirst%3DA.%26atitle%3DRegulation%2520and%2520quality%2520control%2520by%2520Lon-dependent%2520proteolysis%26jtitle%3DRes.%2520Microbiol.%26date%3D2009%26volume%3D160%26spage%3D645%26epage%3D651%26doi%3D10.1016%2Fj.resmic.2009.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fishovitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.</span></span> <span> </span><span class="NLM_article-title">Active-site-directed chemical tools for profiling mitochondrial Lon protease</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">788</span>, <span class="refDoi"> DOI: 10.1021/cb100408w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100408w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlslCisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=781-788&author=J.+Fishovitzauthor=M.+Liauthor=H.+Fraseauthor=J.+Hudakauthor=S.+Craigauthor=K.+Koauthor=A.+J.+Berdisauthor=C.+K.+Suzukiauthor=I.+Lee&title=Active-site-directed+chemical+tools+for+profiling+mitochondrial+Lon+protease&doi=10.1021%2Fcb100408w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Active-Site-Directed Chemical Tools for Profiling Mitochondrial Lon Protease</span></div><div class="casAuthors">Fishovitz, Jennifer; Li, Min; Frase, Hilary; Hudak, Jason; Craig, Sandra; Ko, Kristin; Berdis, Anthony J.; Suzuki, Carolyn K.; Lee, Irene</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lon and ClpXP are the only sol. ATP-dependent proteases within the mammalian mitochondria matrix, which function in protein quality control by selectively degrading misfolded, misassembled, or damaged proteins.  Chem. tools to study these proteases in biol. samples have not been identified, thereby hindering a clear understanding of their resp. functions in normal and disease states.  In this study, we applied a proteolytic site-directed approach to identify a peptide reporter substrate and a peptide inhibitor that are selective for Lon but not ClpXP.  These chem. tools permit quant. measurements that distinguish Lon-mediated proteolysis from that of ClpXP in biochem. assays with purified proteases, as well as in intact mitochondria and mitochondrial lysates.  This chem. biol. approach provides needed tools to further our understanding of mitochondrial ATP-dependent proteolysis and contributes to the future development of diagnostic and pharmacol. agents for treating diseases assocd. with defects in mitochondrial protein quality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8weIfQwHGArVg90H21EOLACvtfcHk0likkQEpYTfmNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlslCisrk%253D&md5=fe959dbb4cd00e3203ad4b4d4f56f9bf</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fcb100408w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100408w%26sid%3Dliteratum%253Aachs%26aulast%3DFishovitz%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFrase%26aufirst%3DH.%26aulast%3DHudak%26aufirst%3DJ.%26aulast%3DCraig%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DK.%26aulast%3DBerdis%26aufirst%3DA.%2BJ.%26aulast%3DSuzuki%26aufirst%3DC.%2BK.%26aulast%3DLee%26aufirst%3DI.%26atitle%3DActive-site-directed%2520chemical%2520tools%2520for%2520profiling%2520mitochondrial%2520Lon%2520protease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D781%26epage%3D788%26doi%3D10.1021%2Fcb100408w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csizmadia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hales, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senn, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadambi, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolenski, F. S.</span></span> <span> </span><span class="NLM_article-title">Proteasome inhibitors bortezomib and carfilzomib used for the treatment of multiple myeloma do not inhibit the serine protease HtrA2/Omi</span>. <i>Toxicol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1619</span>– <span class="NLM_lpage">1628</span>, <span class="refDoi"> DOI: 10.1039/c6tx00220j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1039%2FC6TX00220J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSlu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1619-1628&author=V.+Csizmadiaauthor=P.+Halesauthor=C.+Tsuauthor=J.+Maauthor=J.+Chenauthor=P.+Shahauthor=P.+Flemingauthor=J.+J.+Sennauthor=V.+J.+Kadambiauthor=L.+Dickauthor=F.+S.+Wolenski&title=Proteasome+inhibitors+bortezomib+and+carfilzomib+used+for+the+treatment+of+multiple+myeloma+do+not+inhibit+the+serine+protease+HtrA2%2FOmi&doi=10.1039%2Fc6tx00220j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome inhibitors bortezomib and carfilzomib used for the treatment of multiple myeloma do not inhibit the serine protease HtrA2/Omi</span></div><div class="casAuthors">Csizmadia, Vilmos; Hales, Paul; Tsu, Christopher; Ma, Jingya; Chen, Jiejin; Shah, Pooja; Fleming, Paul; Senn, Joseph J.; Kadambi, Vivek J.; Dick, Larry; Wolenski, Francis S.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Research (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1619-1628</span>CODEN:
                <span class="NLM_cas:coden">TROEE8</span>;
        ISSN:<span class="NLM_cas:issn">2045-4538</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The proteasome inhibitor bortezomib is assocd. with the development of peripheral neuropathy in patients, but the mechanism by which bortezomib can induce peripheral neuropathy is not fully understood.  One study suggested that off-target inhibition of proteases other than the proteasome, particularly HtraA2/Omi, may be the underlying mechanism of the neuropathy.  The same study also concluded that carfilzomib, a second proteasome inhibitor that is assocd. with less peripheral neuropathy in patients than bortezomib, showed no inhibition of HtrA2/Omi.  The goal of the work described here was to det. whether either proteasome inhibitors truly affected HtrA2/Omi activity.  A variety of methods were used to test the effects of both bortezomib and carfilzomib on HtrA2/Omi activity that included in vitro recombinant enzyme assays, and studies with the human neuroblastoma SH-SY5Y cell line and HtrA2/Omi-knockout mouse embryonic fibroblasts.  The compd. ucf-101 was used to assess the effects of specific HtrA2/Omi inhibition.  In contrast to previously published data, our results clearly demonstrated that neither bortezomib nor carfilzomib inhibited HtrA2/Omi activity in recombinant enzyme assays at concns. up to 100 μM, while the specific inhibitor ucf-101 did inhibit the enzyme.  The proteasome inhibitors did not inhibit HtrA2/Omi activity in either SH-SY5Y cells or mouse embryonic fibroblasts, as detd. by expression of the HtrA2/Omi substrates eIF4G1 and UCH-L1.  Based on our biochem. and cell-based assays, we conclude that neither bortezomib nor carfilzomib inhibited HtrA2/Omi activity.  Therefore, it is unlikely that bortezomib assocd. peripheral neuropathy is a direct result of off-target inhibition of HtrA2/Omi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_aWn0wUKFt7Vg90H21EOLACvtfcHk0likkQEpYTfmNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSlu7zM&md5=56c1421e9a9c24b3c11b9bd6fb7443ec</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC6TX00220J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6TX00220J%26sid%3Dliteratum%253Aachs%26aulast%3DCsizmadia%26aufirst%3DV.%26aulast%3DHales%26aufirst%3DP.%26aulast%3DTsu%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DSenn%26aufirst%3DJ.%2BJ.%26aulast%3DKadambi%26aufirst%3DV.%2BJ.%26aulast%3DDick%26aufirst%3DL.%26aulast%3DWolenski%26aufirst%3DF.%2BS.%26atitle%3DProteasome%2520inhibitors%2520bortezomib%2520and%2520carfilzomib%2520used%2520for%2520the%2520treatment%2520of%2520multiple%2520myeloma%2520do%2520not%2520inhibit%2520the%2520serine%2520protease%2520HtrA2%252FOmi%26jtitle%3DToxicol.%2520Res.%26date%3D2016%26volume%3D5%26spage%3D1619%26epage%3D1628%26doi%3D10.1039%2Fc6tx00220j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakatsuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-I.</span></span> <span> </span><span class="NLM_article-title">Structures of an ATP-independent Lon-like protease and its complexes with covalent inhibitors</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1395</span>– <span class="NLM_lpage">1402</span>, <span class="refDoi"> DOI: 10.1107/s0907444913008214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1107%2FS0907444913008214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=23897463" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1395-1402&author=J.-H.+Liaoauthor=K.+Iharaauthor=C.-I.+Kuoauthor=K.-F.+Huangauthor=S.+Wakatsukiauthor=S.-H.+Wuauthor=C.-I.+Chang&title=Structures+of+an+ATP-independent+Lon-like+protease+and+its+complexes+with+covalent+inhibitors&doi=10.1107%2Fs0907444913008214"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444913008214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913008214%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DJ.-H.%26aulast%3DIhara%26aufirst%3DK.%26aulast%3DKuo%26aufirst%3DC.-I.%26aulast%3DHuang%26aufirst%3DK.-F.%26aulast%3DWakatsuki%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DChang%26aufirst%3DC.-I.%26atitle%3DStructures%2520of%2520an%2520ATP-independent%2520Lon-like%2520protease%2520and%2520its%2520complexes%2520with%2520covalent%2520inhibitors%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2013%26volume%3D69%26spage%3D1395%26epage%3D1402%26doi%3D10.1107%2Fs0907444913008214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kupperman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannerman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hales, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzzese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfredi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of the proteasome inhibitor MLN9708 in preclinical models of human cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1970</span>– <span class="NLM_lpage">1980</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-09-2766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1158%2F0008-5472.CAN-09-2766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=20160034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFWisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=1970-1980&author=E.+Kuppermanauthor=E.+C.+Leeauthor=Y.+Caoauthor=B.+Bannermanauthor=M.+Fitzgeraldauthor=A.+Bergerauthor=J.+Yuauthor=Y.+Yangauthor=P.+Halesauthor=F.+Bruzzeseauthor=J.+Liuauthor=J.+Blankauthor=K.+Garciaauthor=C.+Tsuauthor=L.+Dickauthor=P.+Flemingauthor=L.+Yuauthor=M.+Manfrediauthor=M.+Rolfeauthor=J.+Bolen&title=Evaluation+of+the+proteasome+inhibitor+MLN9708+in+preclinical+models+of+human+cancer&doi=10.1158%2F0008-5472.can-09-2766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the Proteasome Inhibitor MLN9708 in Preclinical Models of Human Cancer</span></div><div class="casAuthors">Kupperman, Erik; Lee, Edmund C.; Cao, Yueying; Bannerman, Bret; Fitzgerald, Michael; Berger, Allison; Yu, Jie; Yang, Yu; Bruzzese, Frank; Liu, Jane; Blank, Jonathan; Garcia, Khristofer; Tsu, Christopher; Dick, Larry; Fleming, Paul; Yu, Li; Manfredi, Mark; Rolfe, Mark; Bolen, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1970-1980</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The proteasome was validated as an oncol. target following the clin. success of VELCADE (bortezomib) for injection for the treatment of multiple myeloma and recurring mantle cell lymphoma.  Consequently, several groups are pursuing the development of addnl. small-mol. proteasome inhibitors for both hematol. and solid tumor indications.  Here, we describe MLN9708, a selective, orally bioavailable, second-generation proteasome inhibitor that is in phase I clin. development.  MLN9708 has a shorter proteasome dissocn. half-life and improved pharmacokinetics, pharmacodynamics, and antitumor activity compared with bortezomib.  MLN9708 has a larger blood vol. distribution at steady state, and anal. of 20S proteasome inhibition and markers of the unfolded protein response confirmed that MLN9708 has greater pharmacodynamic effects in tissues than bortezomib.  MLN9708 showed activity in both solid tumor and hematol. preclin. xenograft models, and we found a correlation between greater pharmacodynamic responses and improved antitumor activity.  Moreover, antitumor activity was shown via multiple dosing routes, including oral gavage.  Taken together, these data support the clin. development of MLN9708 for both hematol. and solid tumor indications.  Cancer Res; 70(5); 1970-80.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjauqAFEgDA7Vg90H21EOLACvtfcHk0lj8qAYZGHV5EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFWisro%253D&md5=45636404582fc0eb66581c6c19420458</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-2766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-2766%26sid%3Dliteratum%253Aachs%26aulast%3DKupperman%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DBannerman%26aufirst%3DB.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHales%26aufirst%3DP.%26aulast%3DBruzzese%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DGarcia%26aufirst%3DK.%26aulast%3DTsu%26aufirst%3DC.%26aulast%3DDick%26aufirst%3DL.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DManfredi%26aufirst%3DM.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520the%2520proteasome%2520inhibitor%2520MLN9708%2520in%2520preclinical%2520models%2520of%2520human%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D1970%26epage%3D1980%26doi%3D10.1158%2F0008-5472.can-09-2766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruickshank, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klunder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plamondon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, R. L.</span></span> <span> </span><span class="NLM_article-title">Potent and selective inhibitors of the proteasome: dipeptidyl boronic acids</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(98)00029-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2FS0960-894X%2898%2900029-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=9871680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaK1cXhsFKjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=333-338&author=J.+Adamsauthor=M.+Behnkeauthor=S.+Chenauthor=A.+A.+Cruickshankauthor=L.+R.+Dickauthor=L.+Grenierauthor=J.+M.+Klunderauthor=Y.-T.+Maauthor=L.+Plamondonauthor=R.+L.+Stein&title=Potent+and+selective+inhibitors+of+the+proteasome%3A+dipeptidyl+boronic+acids&doi=10.1016%2Fs0960-894x%2898%2900029-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective inhibitors of the proteasome: dipeptidyl boronic acids</span></div><div class="casAuthors">Adams, Julian; Behnke, Mark; Chen, Shaowu; Cruickshank, Amy A.; Dick, Lawrence R.; Grenier, Louis; Klunder, Jancie M.; Ma, Yu-Ting; Plamondon, Louis; Stein, Ross L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">333-338</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Potent and selective dipeptidyl boronic acid proteasome inhibitors are described.  As compared to peptidyl aldehyde compds., boronic acids in this series display dramatically enhanced potency.  Compds. (I; R = Ph, 1-naphthyl), in particular I (R = Ph), are promising new therapeutics for treatment of cancer and inflammatory diseases.  I (R = Ph) and I (R = 1-naphthyl) in vitro inhibited 20S proteasome with Ki of 0.62 and 0.18, resp., whereas I (R = Ph) showed Ki of 2,300, 630, 320, and 13,000 against common serine proteases such as human leukocyte elastase, human cathepsin G, human chymotrypsin, and thrombin, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeLVVXMKjfPbVg90H21EOLACvtfcHk0lj8qAYZGHV5EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhsFKjsr8%253D&md5=e9287f4977f5ab655fedd1fd702aa456</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2898%2900029-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252898%252900029-8%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%26aulast%3DBehnke%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCruickshank%26aufirst%3DA.%2BA.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DGrenier%26aufirst%3DL.%26aulast%3DKlunder%26aufirst%3DJ.%2BM.%26aulast%3DMa%26aufirst%3DY.-T.%26aulast%3DPlamondon%26aufirst%3DL.%26aulast%3DStein%26aufirst%3DR.%2BL.%26atitle%3DPotent%2520and%2520selective%2520inhibitors%2520of%2520the%2520proteasome%253A%2520dipeptidyl%2520boronic%2520acids%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1998%26volume%3D8%26spage%3D333%26epage%3D338%26doi%3D10.1016%2Fs0960-894x%2898%2900029-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J.</span>; <span class="NLM_string-name">Ma, Y.-T.</span>; <span class="NLM_string-name">Stein, R.</span>; <span class="NLM_string-name">Baevsky, M.</span>; <span class="NLM_string-name">Grenier, L.</span>; <span class="NLM_string-name">Plamondon, L.</span></span> <span> </span><span class="NLM_article-title">Boronic Ester and Acid Compounds</span>. U.S. Patent <span class="NLM_patent">5,780,454 A</span>, <span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=J.+Adams&author=Y.-T.+Ma&author=R.+Stein&author=M.+Baevsky&author=L.+Grenier&author=L.+Plamondon&title=Boronic+Ester+and+Acid+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%26atitle%3DBoronic%2520Ester%2520and%2520Acid%2520Compounds%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhalnina, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shishkov, S. V.</span></span> <span> </span><span class="NLM_article-title">A convergent approach to synthesis of bortezomib: the use of TBTU suppresses racemization in the fragment condensation</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">7105</span>– <span class="NLM_lpage">7108</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2009.06.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.tet.2009.06.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptVGgtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=7105-7108&author=A.+S.+Ivanovauthor=A.+A.+Zhalninaauthor=S.+V.+Shishkov&title=A+convergent+approach+to+synthesis+of+bortezomib%3A+the+use+of+TBTU+suppresses+racemization+in+the+fragment+condensation&doi=10.1016%2Fj.tet.2009.06.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A convergent approach to synthesis of bortezomib: the use of TBTU suppresses racemization in the fragment condensation</span></div><div class="casAuthors">Ivanov, Andrey S.; Zhalnina, Anna A.; Shishkov, Sergey V.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">7105-7108</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bortezomib (I) is a first-in-class therapeutic antineoplastic agent used for treating patients with multiple myeloma and mantle cell lymphoma.  In this paper we report an improved method for synthesis of the title compd. using a convergent approach.  TBTU was found to efficiently suppress racemization in the fragment condensation.  In comparison with the original synthesis, the presented one is shorter by two steps, higher in yield, and provides better atom economy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoju9EMTW_imrVg90H21EOLACvtfcHk0liHJ6HoByA2TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptVGgtL8%253D&md5=e6bc84fc1b41d91164910fc186697986</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2009.06.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2009.06.042%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DA.%2BS.%26aulast%3DZhalnina%26aufirst%3DA.%2BA.%26aulast%3DShishkov%26aufirst%3DS.%2BV.%26atitle%3DA%2520convergent%2520approach%2520to%2520synthesis%2520of%2520bortezomib%253A%2520the%2520use%2520of%2520TBTU%2520suppresses%2520racemization%2520in%2520the%2520fragment%2520condensation%26jtitle%3DTetrahedron%26date%3D2009%26volume%3D65%26spage%3D7105%26epage%3D7108%26doi%3D10.1016%2Fj.tet.2009.06.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matteson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadhu, K. M.</span></span> <span> </span><span class="NLM_article-title">Boronic ester homologation with 99% chiral selectivity and its use in syntheses of the insect pheromones (3S,4S)-4-methyl-3-heptanol and exo-brevicomin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">2077</span>– <span class="NLM_lpage">2078</span>, <span class="refDoi"> DOI: 10.1021/ja00345a074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00345a074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaL3sXhs1Snurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=1983&pages=2077-2078&author=D.+S.+Mattesonauthor=K.+M.+Sadhu&title=Boronic+ester+homologation+with+99%25+chiral+selectivity+and+its+use+in+syntheses+of+the+insect+pheromones+%283S%2C4S%29-4-methyl-3-heptanol+and+exo-brevicomin&doi=10.1021%2Fja00345a074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Boronic ester homologation with 99% chiral selectivity and its use in syntheses of the insect pheromones (3S,4S)-4-methyl-3-heptanol and exo-brevicomin</span></div><div class="casAuthors">Matteson, Donald S.; Sadhu, Kizhakethil M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2077-8</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The directed chiral synthesis of α-chloro boronic esters I [R = Me, Pr, Bu, CH2CHMe2, CH2Ph, 3-(2-methyl-2-dioxolanyl)propyl, 1-benzyloxy-3-(2-methyl-2-dioxolanyl)propyl] from II has been dramatically improved by ZnCl2 catalysis of the rearrangement of the intermediate borate complexes.  Diastereoselectivities are 95-99.5%, yields 82-99%.  Also homologated was II (R = 2-pentyl) with high but not fully proved diastereoselectivity.  Two insect pheromone components, (3S,4S)-4-methyl-3-heptanol (European elm bark beetle, Scolytus multistriatus) and exo-brevicomin (western pine beetle, Dendroctonus brevicomis) were prepd. by this route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonw-PWPn86YrVg90H21EOLACvtfcHk0liHJ6HoByA2TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhs1Snurw%253D&md5=6a3c22886313ed92a9150e555e3aefcc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fja00345a074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00345a074%26sid%3Dliteratum%253Aachs%26aulast%3DMatteson%26aufirst%3DD.%2BS.%26aulast%3DSadhu%26aufirst%3DK.%2BM.%26atitle%3DBoronic%2520ester%2520homologation%2520with%252099%2525%2520chiral%2520selectivity%2520and%2520its%2520use%2520in%2520syntheses%2520of%2520the%2520insect%2520pheromones%2520%25283S%252C4S%2529-4-methyl-3-heptanol%2520and%2520exo-brevicomin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1983%26volume%3D105%26spage%3D2077%26epage%3D2078%26doi%3D10.1021%2Fja00345a074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matteson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jesthi, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadhu, K. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and properties of pinanediol alpha-amido boronic esters</span>. <i>Organometallics</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1284</span>– <span class="NLM_lpage">1288</span>, <span class="refDoi"> DOI: 10.1021/om00086a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/om00086a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaL2cXltFWgtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1984&pages=1284-1288&author=D.+S.+Mattesonauthor=P.+K.+Jesthiauthor=K.+M.+Sadhu&title=Synthesis+and+properties+of+pinanediol+alpha-amido+boronic+esters&doi=10.1021%2Fom00086a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and properties of pinanediol α-amido boronic esters</span></div><div class="casAuthors">Matteson, Donald S.; Jesthi, Pradipta K.; Sadhu, Kizhakethil M.</div><div class="citationInfo"><span class="NLM_cas:title">Organometallics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1284-8</span>CODEN:
                <span class="NLM_cas:coden">ORGND7</span>;
        ISSN:<span class="NLM_cas:issn">0276-7333</span>.
    </div><div class="casAbstract">(+)-Pinanediol was used as a chiral-directing group for the synthesis of boronic acid analogs of N-acyl L-amino acids.  Thus, (+)-pinanediol boronate ester I (R = Me2CH) underwent homologation by treatment with LiCHCl2 to give boronic ester II (R1 = Cl, R2 = H), which was treated with lithiohexamethyldisilazane to give II [R1 = H, R2 = N(SiMe3)2], which was treated with AcOH/Ac2O to give boronate III (R3 = Me2CH, R4 = Me).  The later was cleaved by BCl3 to give boronic amino acid IV.  III (R3 = Me2CHCH2, Me, R4 = Me, R3 = PhCH2, R4 = OCH2Ph) were prepd. similarly.  The hydroboration of R5CH2CH:CH2 (R5 = Cl, Br, PhCH2O) with (1,2-phenyldioxy)borane gave boronic esters V (R5 = same), which underwent transesterification with (+)-pinanediol to give I [R = R5(CH2)3].  I [R = Cl(CH2)3, Br(CH2)3] were converted into III (R3 = Cl(CH2)3, Br(CH2)3, R4 = Me) by the above reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppp1-45n2y8rVg90H21EOLACvtfcHk0liHJ6HoByA2TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXltFWgtL8%253D&md5=19d9707588c4cabc895bc403cbf61711</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fom00086a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fom00086a024%26sid%3Dliteratum%253Aachs%26aulast%3DMatteson%26aufirst%3DD.%2BS.%26aulast%3DJesthi%26aufirst%3DP.%2BK.%26aulast%3DSadhu%26aufirst%3DK.%2BM.%26atitle%3DSynthesis%2520and%2520properties%2520of%2520pinanediol%2520alpha-amido%2520boronic%2520esters%26jtitle%3DOrganometallics%26date%3D1984%26volume%3D3%26spage%3D1284%26epage%3D1288%26doi%3D10.1021%2Fom00086a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matteson, D. S.</span></span> <span> </span><span class="NLM_article-title">Alpha-halo boronic esters - intermediates for stereodirected synthesis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1535</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/cr00097a009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00097a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaL1MXmtFCms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1989&pages=1535-1551&author=D.+S.+Matteson&title=Alpha-halo+boronic+esters+-+intermediates+for+stereodirected+synthesis&doi=10.1021%2Fcr00097a009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">α-Halo boronic esters:  intermediates for stereodirected synthesis</span></div><div class="casAuthors">Matteson, Donald S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1535-51</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with >156 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAvBFy-5DUy7Vg90H21EOLACvtfcHk0liHJ6HoByA2TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtFCms7Y%253D&md5=242a507dec19b2e3a807fd57e909393e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcr00097a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00097a009%26sid%3Dliteratum%253Aachs%26aulast%3DMatteson%26aufirst%3DD.%2BS.%26atitle%3DAlpha-halo%2520boronic%2520esters%2520-%2520intermediates%2520for%2520stereodirected%2520synthesis%26jtitle%3DChem.%2520Rev.%26date%3D1989%26volume%3D89%26spage%3D1535%26epage%3D1551%26doi%3D10.1021%2Fcr00097a009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buesking, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacauanu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis of protected alpha-amino boronic acid derivatives with an air- and moisture-stable Cu(II) catalyst</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">3671</span>– <span class="NLM_lpage">3677</span>, <span class="refDoi"> DOI: 10.1021/jo500300t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo500300t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFGmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2014&pages=3671-3677&author=A.+W.+Bueskingauthor=V.+Bacauanuauthor=I.+Caiauthor=J.+A.+Ellman&title=Asymmetric+synthesis+of+protected+alpha-amino+boronic+acid+derivatives+with+an+air-+and+moisture-stable+Cu%28II%29+catalyst&doi=10.1021%2Fjo500300t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric synthesis of protected α-amino boronic acid derivatives with an air- and moisture-stable Cu(II) catalyst</span></div><div class="casAuthors">Buesking, Andrew W.; Bacauanu, Vlad; Cai, Irene; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3671-3677</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Both diastereomers of α-(sulfinylamino)alkylboronic acids and trifluoroborates were accessible in stereoselective addn. of bis-pinacolatodiboron to (RS)-N-(tert-butylsulfinyl)aldimines and -ketimines, catalyzed by CuSO4/L compns.  Addn. of B2pin2 to (RS)-RR1C:NSOtBu (1), catalyzed by CuSO4/PCy3·HBF4 gave α-aminoboronates RR1C(Bpin)NHSOtBu (2; R = iBu, 4-MeC6H4, 4-MeOC6H4, 4-ClC6H4, 4-CF3C6H4; R1 = H, Me) and the corresponding trifluoroborates NH4[RR1C(BF3)NHSOtBu] (3) upon fluorination.  Whereas ligand-free reaction and phosphite-modified catalyst favors (RS,S)-configuration of the diastereomers 2, CuSO4/PCy3·HBF4-catalyzed reaction gives (RS,R)-diastereomers 2 with up to 94% dr.  The asym. borylation of N-tert-butanesulfinyl imines with bis(pinacolato)diboron is achieved using a Cu(II) catalyst and provides access to synthetically useful and pharmaceutically relevant α-amino boronic acid derivs.  The Cu(II)-catalyzed reaction is performed on the benchtop in air at room temp. using com. available, inexpensive reagents at low catalyst loadings.  A variety of N-tert-butanesulfinyl imines, including ketimines, react readily to provide α-sulfinamido boronate esters in good yields and with high stereoselectivity.  In addn., this transformation is applied to the straightforward, telescoped synthesis of α-sulfinamido trifluoroborates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSIyE2LDmv87Vg90H21EOLACvtfcHk0lj5a9HE5N_LIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFGmsb4%253D&md5=f5e3ba7d40a9b907825726564cde3362</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjo500300t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo500300t%26sid%3Dliteratum%253Aachs%26aulast%3DBuesking%26aufirst%3DA.%2BW.%26aulast%3DBacauanu%26aufirst%3DV.%26aulast%3DCai%26aufirst%3DI.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DAsymmetric%2520synthesis%2520of%2520protected%2520alpha-amino%2520boronic%2520acid%2520derivatives%2520with%2520an%2520air-%2520and%2520moisture-stable%2520Cu%2528II%2529%2520catalyst%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2014%26volume%3D79%26spage%3D3671%26epage%3D3677%26doi%3D10.1021%2Fjo500300t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henneberg, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mata, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tittmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, A.</span></span> <span> </span><span class="NLM_article-title">The inhibition mechanism of human 20S proteasomes enables next-generation inhibitor design</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">594</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1126/science.aaf8993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1126%2Fscience.aaf8993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27493187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=594-598&author=J.+Schraderauthor=F.+Hennebergauthor=R.+A.+Mataauthor=K.+Tittmannauthor=T.+R.+Schneiderauthor=H.+Starkauthor=G.+Bourenkovauthor=A.+Chari&title=The+inhibition+mechanism+of+human+20S+proteasomes+enables+next-generation+inhibitor+design&doi=10.1126%2Fscience.aaf8993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The inhibition mechanism of human 20S proteasomes enables next-generation inhibitor design</span></div><div class="casAuthors">Schrader, Jil; Henneberg, Fabian; Mata, Ricardo A.; Tittmann, Kai; Schneider, Thomas R.; Stark, Holger; Bourenkov, Gleb; Chari, Ashwin</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6299</span>),
    <span class="NLM_cas:pages">594-598</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The proteasome is a validated target for anticancer therapy, and proteasome inhibition is employed in the clinic for the treatment of tumors and hematol. malignancies.  Here, the authors describe crystal structures of the native human 20S proteasome and its complexes with inhibitors, which either are drugs approved for cancer treatment or are in clin. trials.  The structure of the native human 20S proteasome was detd. at an unprecedented resoln. of 1.8 angstroms.  Addnl., six inhibitor-proteasome complex structures were elucidated at resolns. between 1.9 and 2.1 angstroms.  Collectively, the high-resoln. structures provide new insights into the catalytic mechanisms of inhibition and necessitate a revised description of the proteasome active site.  Knowledge about inhibition mechanisms provides insights into peptide hydrolysis and can guide strategies for the development of next-generation proteasome-based cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZCuIDBbOXPLVg90H21EOLACvtfcHk0lj5a9HE5N_LIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7vL&md5=8216d0007cbf06c352e06fcb05d644a9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf8993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf8993%26sid%3Dliteratum%253Aachs%26aulast%3DSchrader%26aufirst%3DJ.%26aulast%3DHenneberg%26aufirst%3DF.%26aulast%3DMata%26aufirst%3DR.%2BA.%26aulast%3DTittmann%26aufirst%3DK.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DBourenkov%26aufirst%3DG.%26aulast%3DChari%26aufirst%3DA.%26atitle%3DThe%2520inhibition%2520mechanism%2520of%2520human%252020S%2520proteasomes%2520enables%2520next-generation%2520inhibitor%2520design%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D594%26epage%3D598%26doi%3D10.1126%2Fscience.aaf8993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkers, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploegh, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the boronic acid-based proteasome inhibitor bortezomib in complex with the yeast 20S proteasome</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1016/j.str.2005.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.str.2005.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=16531229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD28Xit1ylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=451-456&author=M.+Grollauthor=C.+R.+Berkersauthor=H.+L.+Ploeghauthor=H.+Ovaa&title=Crystal+structure+of+the+boronic+acid-based+proteasome+inhibitor+bortezomib+in+complex+with+the+yeast+20S+proteasome&doi=10.1016%2Fj.str.2005.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Boronic Acid-Based Proteasome Inhibitor Bortezomib in Complex with the Yeast 20S Proteasome</span></div><div class="casAuthors">Groll, Michael; Berkers, Celia R.; Ploegh, Hidde L.; Ovaa, Huib</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">451-456</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The dipeptide boronic acid bortezomib, also termed VELCADE is a proteasome inhibitor now in use for the treatment of multiple myeloma, and its use for the treatment of other malignancies is being explored.  We detd. the crystal structure of the yeast 20S proteasome in complex with bortezomib to establish the specificity and binding mode of bortezomib to the proteasome's different catalytically active sites.  This structure should enable the rational design of new boronic acid derivs. with improved affinities and specificities for individual active subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpusGzFSuveALVg90H21EOLACvtfcHk0lj5a9HE5N_LIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xit1ylsL8%253D&md5=d5b4e46e5a9f5259cce758770302b4fe</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DBerkers%26aufirst%3DC.%2BR.%26aulast%3DPloegh%26aufirst%3DH.%2BL.%26aulast%3DOvaa%26aufirst%3DH.%26atitle%3DCrystal%2520structure%2520of%2520the%2520boronic%2520acid-based%2520proteasome%2520inhibitor%2520bortezomib%2520in%2520complex%2520with%2520the%2520yeast%252020S%2520proteasome%26jtitle%3DStructure%26date%3D2006%26volume%3D14%26spage%3D451%26epage%3D456%26doi%3D10.1016%2Fj.str.2005.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">The versatility of boron in biological target engagement</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">742</span>, <span class="refDoi"> DOI: 10.1038/nchem.2814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnchem.2814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=28754930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Gku7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=731-742&author=D.+B.+Diazauthor=A.+K.+Yudin&title=The+versatility+of+boron+in+biological+target+engagement&doi=10.1038%2Fnchem.2814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The versatility of boron in biological target engagement</span></div><div class="casAuthors">Diaz, Diego B.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">731-742</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Boron-contg. mols. have been extensively used for the purposes of chem. sensing, biol. probe development, and drug discovery.  Due to boron's empty p orbital, it can coordinate to heteroatoms such as oxygen and nitrogen.  This reversible covalent mode of interaction has led to the use of boron as bait for nucleophilic residues in disease-assocd. proteins, culminating in the approval of new therapeutics that work by covalent mechanisms.  Our anal. of a wide range of covalent inhibitors with electrophilic groups suggests that boron is a unique electrophile in its chameleonic ability to engage protein targets.  Here, we review boron's interactions with a range of protein side-chain residues and reveal that boron's properties are nuanced and arise from its uncommon coordination preferences.  These mechanistic and structural insights should serve as a guide for the development of selective boron-based bioactive mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYy4b-paw5prVg90H21EOLACvtfcHk0lg0eXLVvTsh9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Gku7bF&md5=cf5669d184a5badc7101aa2e3d1f2500</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2814%26sid%3Dliteratum%253Aachs%26aulast%3DDiaz%26aufirst%3DD.%2BB.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DThe%2520versatility%2520of%2520boron%2520in%2520biological%2520target%2520engagement%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D731%26epage%3D742%26doi%3D10.1038%2Fnchem.2814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollé, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: a compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&author=E.+R.+McDonaldauthor=A.+de+Weckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cort%C3%A9s-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Moll%C3%A9author=R.+A.+Pagliariniauthor=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+a+compendium+of+cancer+dependencies+and+synthetic+lethal+relationships+uncovered+by+large-scale%2C+deep+RNAi+screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0lg0eXLVvTsh9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCort%25C3%25A9s-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMoll%25C3%25A9%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520a%2520compendium%2520of%2520cancer%2520dependencies%2520and%2520synthetic%2520lethal%2520relationships%2520uncovered%2520by%2520large-scale%252C%2520deep%2520RNAi%2520screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caponigro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehár, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monahan, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morais, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meltzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korejwa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jané-Valbuena, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapa, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bric-Furlong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipway, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspesi, P.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palescandolo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sougnez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onofrio, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefeld, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacConaill, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winckler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">The cancer cell line encyclopedia enables predictive modelling of anticancer drug sensitivity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">607</span>, <span class="refDoi"> DOI: 10.1038/nature11003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnature11003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=22460905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=603-607&author=J.+Barretinaauthor=G.+Caponigroauthor=N.+Stranskyauthor=K.+Venkatesanauthor=A.+A.+Margolinauthor=S.+Kimauthor=C.+J.+Wilsonauthor=J.+Leh%C3%A1rauthor=G.+V.+Kryukovauthor=D.+Sonkinauthor=A.+Reddyauthor=M.+Liuauthor=L.+Murrayauthor=M.+F.+Bergerauthor=J.+E.+Monahanauthor=P.+Moraisauthor=J.+Meltzerauthor=A.+Korejwaauthor=J.+Jan%C3%A9-Valbuenaauthor=F.+A.+Mapaauthor=J.+Thibaultauthor=E.+Bric-Furlongauthor=P.+Ramanauthor=A.+Shipwayauthor=I.+H.+Engelsauthor=J.+Chengauthor=G.+K.+Yuauthor=J.+Yuauthor=P.+Aspesiauthor=M.+de+Silvaauthor=K.+Jagtapauthor=M.+D.+Jonesauthor=L.+Wangauthor=C.+Hattonauthor=E.+Palescandoloauthor=S.+Guptaauthor=S.+Mahanauthor=C.+Sougnezauthor=R.+C.+Onofrioauthor=T.+Liefeldauthor=L.+MacConaillauthor=W.+Wincklerauthor=M.+Reichauthor=N.+Liauthor=J.+P.+Mesirovauthor=S.+B.+Gabrielauthor=G.+Getzauthor=K.+Ardlieauthor=V.+Chanauthor=V.+E.+Myerauthor=B.+L.+Weberauthor=J.+Porterauthor=M.+Warmuthauthor=P.+Finanauthor=J.+L.+Harrisauthor=M.+Meyersonauthor=T.+R.+Golubauthor=M.+P.+Morrisseyauthor=W.+R.+Sellersauthor=R.+Schlegelauthor=L.+A.+Garraway&title=The+cancer+cell+line+encyclopedia+enables+predictive+modelling+of+anticancer+drug+sensitivity&doi=10.1038%2Fnature11003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity</span></div><div class="casAuthors">Barretina, Jordi; Caponigro, Giordano; Stransky, Nicolas; Venkatesan, Kavitha; Margolin, Adam A.; Kim, Sungjoon; Wilson, Christopher J.; Lehar, Joseph; Kryukov, Gregory V.; Sonkin, Dmitriy; Reddy, Anupama; Liu, Manway; Murray, Lauren; Berger, Michael F.; Monahan, John E.; Morais, Paula; Meltzer, Jodi; Korejwa, Adam; Jane-Valbuena, Judit; Mapa, Felipa A.; Thibault, Joseph; Bric-Furlong, Eva; Raman, Pichai; Shipway, Aaron; Engels, Ingo H.; Cheng, Jill; Yu, Guoying K.; Yu, Jianjun; Aspesi, Peter; de Silva, Melanie; Jagtap, Kalpana; Jones, Michael D.; Wang, Li; Hatton, Charles; Palescandolo, Emanuele; Gupta, Supriya; Mahan, Scott; Sougnez, Carrie; Onofrio, Robert C.; Liefeld, Ted; MacConaill, Laura; Winckler, Wendy; Reich, Michael; Li, Nanxin; Mesirov, Jill P.; Gabriel, Stacey B.; Getz, Gad; Ardlie, Kristin; Chan, Vivien; Myer, Vic E.; Weber, Barbara L.; Porter, Jeff; Warmuth, Markus; Finan, Peter; Harris, Jennifer L.; Meyerson, Matthew; Golub, Todd R.; Morrissey, Michael P.; Sellers, William R.; Schlegel, Robert; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">603-607</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The systematic translation of cancer genomic data into knowledge of tumor biol. and therapeutic possibilities remains challenging.  Such efforts should be greatly aided by robust preclin. model systems that reflect the genomic diversity of human cancers and for which detailed genetic and pharmacol. annotation is available.  Here we describe the Cancer Cell Line Encyclopedia (CCLE): a compilation of gene expression, chromosomal copy no. and massively parallel sequencing data from 947 human cancer cell lines.  When coupled with pharmacol. profiles for 24 anticancer drugs across 479 of the cell lines, this collection allowed identification of genetic, lineage, and gene-expression-based predictors of drug sensitivity.  In addn. to known predictors, we found that plasma cell lineage correlated with sensitivity to IGF1 receptor inhibitors; AHR expression was assocd. with MEK inhibitor efficacy in NRAS-mutant lines; and SLFN11 expression predicted sensitivity to topoisomerase inhibitors.  Together, our results indicate that large, annotated cell-line collections may help to enable preclin. stratification schemata for anticancer agents.  The generation of genetic predictions of drug response in the preclin. setting and their incorporation into cancer clin. trial design could speed the emergence of personalized' therapeutic regimens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHUeZ7mtYpwLVg90H21EOLACvtfcHk0lj5WOxpSA2HPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D&md5=4aefaa8c7abd1e30c35c4eeca811687b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature11003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11003%26sid%3Dliteratum%253Aachs%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DCaponigro%26aufirst%3DG.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DMargolin%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DLeh%25C3%25A1r%26aufirst%3DJ.%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DSonkin%26aufirst%3DD.%26aulast%3DReddy%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DMonahan%26aufirst%3DJ.%2BE.%26aulast%3DMorais%26aufirst%3DP.%26aulast%3DMeltzer%26aufirst%3DJ.%26aulast%3DKorejwa%26aufirst%3DA.%26aulast%3DJan%25C3%25A9-Valbuena%26aufirst%3DJ.%26aulast%3DMapa%26aufirst%3DF.%2BA.%26aulast%3DThibault%26aufirst%3DJ.%26aulast%3DBric-Furlong%26aufirst%3DE.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DShipway%26aufirst%3DA.%26aulast%3DEngels%26aufirst%3DI.%2BH.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DG.%2BK.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DAspesi%26aufirst%3DP.%26aulast%3Dde%2BSilva%26aufirst%3DM.%26aulast%3DJagtap%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPalescandolo%26aufirst%3DE.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DMahan%26aufirst%3DS.%26aulast%3DSougnez%26aufirst%3DC.%26aulast%3DOnofrio%26aufirst%3DR.%2BC.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DMacConaill%26aufirst%3DL.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DGabriel%26aufirst%3DS.%2BB.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DV.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DWeber%26aufirst%3DB.%2BL.%26aulast%3DPorter%26aufirst%3DJ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DFinan%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DMorrissey%26aufirst%3DM.%2BP.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DThe%2520cancer%2520cell%2520line%2520encyclopedia%2520enables%2520predictive%2520modelling%2520of%2520anticancer%2520drug%2520sensitivity%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D603%26epage%3D607%26doi%3D10.1038%2Fnature11003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Biochemical characterization of human LONP1. (A) ATPase activity determination of LONP1 full-length protein and ATPase domain-only construct (a.a. 408–752) using the ADP-Transcreener assay. (B) Protease activity determination of LONP1 full-length protein (triangles), protease domain-only construct (a.a 754–959) (circles), and mixture of ATPase domain and protease domain at 1:1 ratio (squares) using the LONP1 protease assay. (C) Full-length LONP1 characterization under various conditions: (a) in the presence of all components (defined as 100% control activity); (b) in the absence of enzyme (defined as background); (c) in the absence of ATP; (d) in the presence of excess bortezomib (10 μM; 55-fold above its IC<sub>50</sub>); and (e) in the absence of metal cofactor (CaCl<sub>2</sub>). (D) Kinetics of LONP1 activation in the presence of either 1 μM ATP or 1 μM AMP-PNP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Analogues Following Known Synthetic Procedures<a onclick="showRef(event, 'ref26 ref27 ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29 ref30 ref31 ref32">(26−32)</a> <sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DIPEA, HATU, DMF or DCM, and −15 °C to room temperature. (b) HCl/dioxane. (c) For <b>5a–b</b>: <b>4</b>, TBTU, DIEA, DCM, and −15 °C to room temperature. For <b>9a–i</b>: <b>2</b>, HATU, DIPEA, DMF or DCM, and −15 °C to room temperature. For <b>12a</b> and <b>12d–g</b>: <b>10a</b> or <b>10d–g</b>, HATU, DIPEA, DMF or DCM, and −15 °C to room temperature. For <b>14</b>: 2,4-dimethyloxazole-5-carboxylic acid, HATU, DIPEA, DCM, and −15 °C to room temperature. (d) Isobutyl boronic acid, HCl, and heptane/MeOH.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding site comparison of LONP1 and the 20S proteasome β5-subunit. Hydrogen bonds to the protein backbones are shown in green and the reactive serine or threonine residues for LONP1 and 20S proteasome, respectively, are indicated by an asterisk (*). (A) Bortezomib bound to human LONP1 protease domain (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>). (B) Bortezomib bound to the β5 site of the human 20S proteasome (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF3">5LF3</a>). (C) Comparison of the binding mode of bortezomib in LONP1 (yellow) vs the 20S proteasome β5 site (cyan). (D) Crystal structure of <b>9a</b> bound to LONP1 (dark purple; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a>) and docked pose (light purple). The root-mean-square deviation (RMSD) between the docked and crystal pose is 0.63. (E) Docking model of <b>9a</b> bound to the β5 subunit of the human 20S proteasome. (F) Overlay of the crystal pose of <b>9a</b> bound to LONP1 (purple; PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a>) with the predicted binding mode in the β5 subunit of the human 20S proteasome (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. MD simulation and docking poses of LONP1 inhibitors with LONP1 and 20S proteasome. (A) Binding pocket dynamics over 200 ns of MD simulation of the apo- and bortezomib-bound LONP1 and (B) 20S proteasome β5 subunit. (C) Conformational states of L778 observed in LONP1 crystal structures (top panel; PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS">6WYS</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M">6X1M</a>). The interior pocket surface and volume are shown for the bortezomib-bound (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a>) and <b>12d</b>-bound (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M">6X1M</a>) crystal structures (lower panel). (D) Binding mode of <b>12d</b> in LONP1 displaying hydrogen bonds to the backbone and the π–π stacking with W770. (E) Predicted binding modes of <b>11d</b> (green) and <b>12d</b> (yellow) using the 20S proteasome-expanded pocket model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/medium/jm0c02152_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Biochemical and cellular activity of LONP1 inhibitors. Full circles represent P3 SAR of <b>12d</b>, while open circles represent a subset of compounds from <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. All scatter plot data at 10 μM IC<sub>50</sub> correspond to compounds with IC<sub>50</sub>s > 10 μM. (A) Scatter plot of LONP1 (<i>y</i>-axis) and 20S proteasome (<i>x</i>-axis) biochemical IC<sub>50</sub>s. (B) Relative abundance of MMADHC following affinity purification and mass spectrometry analysis of cells treated with DMSO or <b>14</b>. Abundance ratios were calculated based on the label-free quantitation from MS1 intensity of precursor ions using Thermo Scientific Proteome Discoverer software version 2.4. (C) WES analysis of MMADHC in H1944 cells following treatment with a 1:3 serial dilution of <b>14</b> starting at 10 μM. (D) MMADHC measured by HTRF assay in Calu6 cells following treatment with indicated compounds. (E) Biochemical LONP1 IC<sub>50</sub> (<i>y</i>-axis) and cell viability IC<sub>50</sub> (<i>x</i>-axis) from LONP1-sensitive Calu6 and LONP1-insensitive H358 cell lines, respectively. Compounds are colored by their activity in the biochemical 20S proteasome assay. With the exception of bortezomib (red circle), only compounds with 20S proteasome IC<sub>50</sub> > 10 μM (green) were tested in viability.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-8/acs.jmedchem.0c02152/20210415/images/large/jm0c02152_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02152&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i36">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 37 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T. A.</span></span> <span> </span><span class="NLM_article-title">AAA+ proteases: ATP-fueled machines of protein destruction</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">587</span>– <span class="NLM_lpage">612</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060408-172623</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1146%2Fannurev-biochem-060408-172623" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=21469952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=587-612&author=R.+T.+Sauerauthor=T.+A.+Baker&title=AAA%2B+proteases%3A+ATP-fueled+machines+of+protein+destruction&doi=10.1146%2Fannurev-biochem-060408-172623"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">AAA+ proteases: ATP-fueled machines of protein destruction</span></div><div class="casAuthors">Sauer, Robert T.; Baker, Tania A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">587-612</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  AAA+ family proteolytic machines (ClpXP, ClpAP, ClpCP, HslUV, Lon, FtsH, PAN/20S, and the 26S proteasome) perform protein quality control and are used in regulatory circuits in all cells.  These machines contain a compartmental protease, with active sites sequestered in an interior chamber, and a hexameric ring of AAA+ ATPases.  Substrate proteins are tethered to the ring, either directly or via adaptor proteins.  An unstructured region of the substrate is engaged in the axial pore of the AAA+ ring, and cycles of ATP binding/hydrolysis drive conformational changes that create pulses of pulling that denature the substrate and translocate the unfolded polypeptide through the pore and into the degrdn. chamber.  Here, the authors review the current understanding of the mol. mechanisms of substrate recognition, adaptor function, and ATP-fueled unfolding and translocation.  The unfolding activities of these and related AAA+ machines can also be used to disassemble or remodel macromol. complexes and to resolubilize aggregates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8taxvJbAb9rVg90H21EOLACvtfcHk0liAB4SL7TbTOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVCnsbw%253D&md5=76e004df8ead113c31b4df0c44baadcf</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060408-172623&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060408-172623%26sid%3Dliteratum%253Aachs%26aulast%3DSauer%26aufirst%3DR.%2BT.%26aulast%3DBaker%26aufirst%3DT.%2BA.%26atitle%3DAAA%252B%2520proteases%253A%2520ATP-fueled%2520machines%2520of%2520protein%2520destruction%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2011%26volume%3D80%26spage%3D587%26epage%3D612%26doi%3D10.1146%2Fannurev-biochem-060408-172623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olivares, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauer, R. T.</span></span> <span> </span><span class="NLM_article-title">Mechanistic insights into bacterial AAA+ proteases and protein-remodelling machines</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">44</span>, <span class="refDoi"> DOI: 10.1038/nrmicro.2015.4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnrmicro.2015.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=26639779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=33-44&author=A.+O.+Olivaresauthor=T.+A.+Bakerauthor=R.+T.+Sauer&title=Mechanistic+insights+into+bacterial+AAA%2B+proteases+and+protein-remodelling+machines&doi=10.1038%2Fnrmicro.2015.4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insights into bacterial AAA+ proteases and protein-remodelling machines</span></div><div class="casAuthors">Olivares, Adrian O.; Baker, Tania A.; Sauer, Robert T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-44</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">To maintain protein homeostasis, AAA+ proteolytic machines degrade damaged and unneeded proteins in bacteria, archaea and eukaryotes.  This process involves the ATP-dependent unfolding of a target protein and its subsequent translocation into a self-compartmentalized proteolytic chamber.  Related AAA+ enzymes also disaggregate and remodel proteins.  Recent structural and biochem. studies, in combination with direct visualization of unfolding and translocation in single-mol. expts., have illuminated the mol. mechanisms behind these processes and suggest how remodelling of macromol. complexes by AAA+ enzymes could occur without global denaturation.  In this Review, we discuss the structural and mechanistic features of AAA+ proteases and remodelling machines, focusing on the bacterial ClpXP and ClpX as paradigms.  We also consider the potential of these enzymes as antibacterial targets and outline future challenges for the field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrrZLtmwRVKrVg90H21EOLACvtfcHk0liSRg35AqrG8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLnM&md5=4154878ee24f6543ae4642cc1096d0e5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrmicro.2015.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrmicro.2015.4%26sid%3Dliteratum%253Aachs%26aulast%3DOlivares%26aufirst%3DA.%2BO.%26aulast%3DBaker%26aufirst%3DT.%2BA.%26aulast%3DSauer%26aufirst%3DR.%2BT.%26atitle%3DMechanistic%2520insights%2520into%2520bacterial%2520AAA%252B%2520proteases%2520and%2520protein-remodelling%2520machines%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2016%26volume%3D14%26spage%3D33%26epage%3D44%26doi%3D10.1038%2Fnrmicro.2015.4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bota, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, K. J. A.</span></span> <span> </span><span class="NLM_article-title">Lon protease preferentially degrades oxidized mitochondrial aconitase by an ATP-stimulated mechanism</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/ncb836</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fncb836" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=12198491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1aht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=674-680&author=D.+A.+Botaauthor=K.+J.+A.+Davies&title=Lon+protease+preferentially+degrades+oxidized+mitochondrial+aconitase+by+an+ATP-stimulated+mechanism&doi=10.1038%2Fncb836"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lon protease preferentially degrades oxidized mitochondrial aconitase by an ATP-stimulated mechanism</span></div><div class="casAuthors">Bota, Daniela A.; Davies, Kelvin J. A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">674-680</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mitochondrial aconitase is sensitive to oxidative inactivation and can aggregate and accumulate in many age-related disorders.  Here we report that Lon protease, an ATP-stimulated mitochondrial matrix protein, selectively recognizes and degrades the oxidized, hydrophobic form of aconitase after mild oxidative modification, but that severe oxidn. results in aconitase aggregation, which makes it a poor substrate for Lon.  Similarly, a morpholino oligodeoxynucleotide directed against the lon gene markedly decreases the amt. of Lon protein, Lon activity and aconitase degrdn. in WI-38 VA-13 human lung fibroblasts and causes accumulation of oxidatively modified aconitase.  The ATP-stimulated Lon protease may be an essential defense against the stress of life in an oxygen environment.  By recognizing minor oxidative changes to protein structure and rapidly degrading the mildly modified protein, Lon protease may prevent extensive oxidn., aggregation and accumulation of aconitase, which could otherwise compromise mitochondrial function and cellular viability.  Aconitase is probably only one of many mitochondrial matrix proteins that are preferentially degraded by Lon protease after oxidative modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3_bTQEgj9tbVg90H21EOLACvtfcHk0liSRg35AqrG8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1aht70%253D&md5=e24bc60e8ff5684f45e2a312a6a82c8f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncb836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb836%26sid%3Dliteratum%253Aachs%26aulast%3DBota%26aufirst%3DD.%2BA.%26aulast%3DDavies%26aufirst%3DK.%2BJ.%2BA.%26atitle%3DLon%2520protease%2520preferentially%2520degrades%2520oxidized%2520mitochondrial%2520aconitase%2520by%2520an%2520ATP-stimulated%2520mechanism%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2002%26volume%3D4%26spage%3D674%26epage%3D680%26doi%3D10.1038%2Fncb836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. S.</span>; <span class="NLM_string-name">Houry, W. A.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Targeting Human Mitochondrial AAA+ Proteases to Develop Novel Cancer Therapeutics</span>. In  <i>Mitochondria in Health and in Sickness</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbani, A.</span>, <span class="NLM_string-name">Babu, M.</span></span>, Eds.; <span class="NLM_publisher-name">Springer Singapore</span>: <span class="NLM_publisher-loc">Singapore</span>, <span class="NLM_year">2019</span>; pp  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">142</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1007%2F978-981-13-8367-0_8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=119-142&author=K.+S.+Wong&author=W.+A.+Houryauthor=A.+Urbani&author=M.+Babu&title=Mitochondria+in+Health+and+in+Sickness"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2F978-981-13-8367-0_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-981-13-8367-0_8%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DK.%2BS.%26atitle%3DRecent%2520Advances%2520in%2520Targeting%2520Human%2520Mitochondrial%2520AAA%252B%2520Proteases%2520to%2520Develop%2520Novel%2520Cancer%2520Therapeutics%26btitle%3DMitochondria%2520in%2520Health%2520and%2520in%2520Sickness%26aulast%3DUrbani%26aufirst%3DA.%26pub%3DSpringer%2520Singapore%26date%3D2019%26spage%3D119%26epage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glynn, S. E.</span></span> <span> </span><span class="NLM_article-title">Multifunctional mitochondrial AAA proteases</span>. <i>Front. Mol. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">34</span>, <span class="refDoi"> DOI: 10.3389/fmolb.2017.00034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.3389%2Ffmolb.2017.00034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=28589125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFaqs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2017&pages=34&author=S.+E.+Glynn&title=Multifunctional+mitochondrial+AAA+proteases&doi=10.3389%2Ffmolb.2017.00034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Multifunctional mitochondrial AAA proteases</span></div><div class="casAuthors">Glynn, Steven E.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Molecular Biosciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/11</span>CODEN:
                <span class="NLM_cas:coden">FMBRBS</span>;
        ISSN:<span class="NLM_cas:issn">2296-889X</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Mitochondria perform numerous functions necessary for the survival of eukaryotic cells.  These activities are coordinated by a diverse complement of proteins encoded in both the nuclear andmitochondrial genomes thatmust be properly organized andmaintained.  Misregulation of mitochondrial proteostasis impairs organellar function and can result in the development of severe human diseases.  ATP-driven AAA+ proteins play crucial roles in preserving mitochondrial activity by removing and remodeling protein mols. in accordance with the needs of the cell.  Two mitochondrial AAA proteases, i-AAA and m-AAA, are anchored to either face of the mitochondrial inner membrane, where they engage and process an array of substrates to impact protein biogenesis, quality control, and the regulation of key metabolic pathways.  The functionality of these proteases is extended through multiple substrate-dependent modes of action, including complete degrdn., partial processing, or dislocation from the membrane without proteolysis.  This review discusses recent advances made toward elucidating the mechanisms of substrate recognition, handling, and degrdn. that allow these versatile proteases to control diverse activities in this multifunctional organelle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcLJ1O1VbyTLVg90H21EOLACvtfcHk0liSRg35AqrG8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFaqs73E&md5=cd4cdba91674d1e0c83fdbff99e7f683</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3389%2Ffmolb.2017.00034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffmolb.2017.00034%26sid%3Dliteratum%253Aachs%26aulast%3DGlynn%26aufirst%3DS.%2BE.%26atitle%3DMultifunctional%2520mitochondrial%2520AAA%2520proteases%26jtitle%3DFront.%2520Mol.%2520Biosci.%26date%3D2017%26volume%3D4%26spage%3D34%26doi%3D10.3389%2Ffmolb.2017.00034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kereïche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovácik, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednár, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pevala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunová, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrovicová, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellová, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutejová, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raska, I.</span></span> <span> </span><span class="NLM_article-title">The N-terminal domain plays a crucial role in the structure of a full-length human mitochondrial Lon protease</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">33631</span>, <span class="refDoi"> DOI: 10.1038/srep33631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fsrep33631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27632940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFentL3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=33631&author=S.+Kere%C3%AFcheauthor=L.+Kov%C3%A1cikauthor=J.+Bedn%C3%A1rauthor=V.+Pevalaauthor=N.+Kunov%C3%A1author=G.+Ondrovicov%C3%A1author=J.+Bauerauthor=L.+Ambroauthor=J.+Bellov%C3%A1author=E.+Kutejov%C3%A1author=I.+Raska&title=The+N-terminal+domain+plays+a+crucial+role+in+the+structure+of+a+full-length+human+mitochondrial+Lon+protease&doi=10.1038%2Fsrep33631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The N-terminal domain plays a crucial role in the structure of a full-length human mitochondrial Lon protease</span></div><div class="casAuthors">Kereiche, Sami; Kovacik, Lubomir; Bednar, Jan; Pevala, Vladimir; Kunova, Nina; Ondrovicova, Gabriela; Bauer, Jacob; Ambro, Lubos; Bellova, Jana; Kutejova, Eva; Raska, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">33631</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lon is an essential, multitasking AAA+ protease regulating many cellular processes in species across all kingdoms of life.  Altered expression levels of the human mitochondrial Lon protease (hLon) are linked to serious diseases including myopathies, paraplegia, and cancer.  Here, we present the first 3D structure of full-length hLon using cryo-electron microscopy.  HLon has a unique three-dimensional structure, in which the proteolytic and ATP-binding domains (AP-domain) form a hexameric chamber, while the N-terminal domain is arranged as a trimer of dimers.  These two domains are linked by a narrow trimeric channel composed likely of coiled-coil helixes.  In the presence of AMP-PNP, the AP-domain has a closed-ring conformation and its N-terminal entry gate appears closed, but in ADP binding, it switches to a lock-washer conformation and its N-terminal gate opens, which is accompanied by a rearrangement of the N-terminal domain.  We have also found that both the enzymic activities and the 3D structure of a hLon mutant lacking the first 156 amino acids are severely disturbed, showing that hLon's N-terminal domains are crucial for the overall structure of the hLon, maintaining a conformation allowing its proper functioning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhnhLJCeIMNrVg90H21EOLACvtfcHk0lg4pkQ13zSiMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFentL3M&md5=2191d273cefe1a21d180a7e9fde42365</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fsrep33631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep33631%26sid%3Dliteratum%253Aachs%26aulast%3DKere%25C3%25AFche%26aufirst%3DS.%26aulast%3DKov%25C3%25A1cik%26aufirst%3DL.%26aulast%3DBedn%25C3%25A1r%26aufirst%3DJ.%26aulast%3DPevala%26aufirst%3DV.%26aulast%3DKunov%25C3%25A1%26aufirst%3DN.%26aulast%3DOndrovicov%25C3%25A1%26aufirst%3DG.%26aulast%3DBauer%26aufirst%3DJ.%26aulast%3DAmbro%26aufirst%3DL.%26aulast%3DBellov%25C3%25A1%26aufirst%3DJ.%26aulast%3DKutejov%25C3%25A1%26aufirst%3DE.%26aulast%3DRaska%26aufirst%3DI.%26atitle%3DThe%2520N-terminal%2520domain%2520plays%2520a%2520crucial%2520role%2520in%2520the%2520structure%2520of%2520a%2520full-length%2520human%2520mitochondrial%2520Lon%2520protease%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D33631%26doi%3D10.1038%2Fsrep33631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snider, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houry, W. A.</span></span> <span> </span><span class="NLM_article-title">The AAA+ superfamily of functionally diverse proteins</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">216</span>, <span class="refDoi"> DOI: 10.1186/gb-2008-9-4-216</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1186%2Fgb-2008-9-4-216" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=18466635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A280%3ADC%252BC3c%252FivVentQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=216&author=J.+Sniderauthor=G.+Thibaultauthor=W.+A.+Houry&title=The+AAA%2B+superfamily+of+functionally+diverse+proteins&doi=10.1186%2Fgb-2008-9-4-216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The AAA+ superfamily of functionally diverse proteins</span></div><div class="casAuthors">Snider Jamie; Thibault Guillaume; Houry Walid A</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">216</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">SUMMARY:  The AAA+ superfamily is a large and functionally diverse superfamily of NTPases that are characterized by a conserved nucleotide-binding and catalytic module, the AAA+ module.  Members are involved in an astonishing range of different cellular processes, attaining this functional diversity through additions of structural motifs and modifications to the core AAA+ module.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRh_wt6tM9EwpotVjB3IRIyfW6udTcc2ebNfvgJ8E_ROLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c%252FivVentQ%253D%253D&md5=f50bf3d3c1d1b86e8843f1ac981f5da2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2Fgb-2008-9-4-216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb-2008-9-4-216%26sid%3Dliteratum%253Aachs%26aulast%3DSnider%26aufirst%3DJ.%26aulast%3DThibault%26aufirst%3DG.%26aulast%3DHoury%26aufirst%3DW.%2BA.%26atitle%3DThe%2520AAA%252B%2520superfamily%2520of%2520functionally%2520diverse%2520proteins%26jtitle%3DGenome%2520Biol.%26date%3D2008%26volume%3D9%26spage%3D216%26doi%3D10.1186%2Fgb-2008-9-4-216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-I.</span></span> <span> </span><span class="NLM_article-title">Structural insights into the allosteric operation of the Lon AAA+ protease</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.03.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.str.2016.03.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27041592" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsVWisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=667-675&author=C.-C.+Linauthor=S.-C.+Suauthor=M.-Y.+Suauthor=P.-H.+Liangauthor=C.-C.+Fengauthor=S.-H.+Wuauthor=C.-I.+Chang&title=Structural+insights+into+the+allosteric+operation+of+the+Lon+AAA%2B+protease&doi=10.1016%2Fj.str.2016.03.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into the Allosteric Operation of the Lon AAA+ Protease</span></div><div class="casAuthors">Lin, Chien-Chu; Su, Shih-Chieh; Su, Ming-Yuan; Liang, Pi-Hui; Feng, Chia-Cheng; Wu, Shih-Hsiung; Chang, Chung-I.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-675</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Lon AAA+ protease (LonA) is an evolutionarily conserved protease that couples the ATPase cycle into motion to drive substrate translocation and degrdn.  A hallmark feature shared by AAA+ proteases is the stimulation of ATPase activity by substrates.  Here the authors report the structure of LonA bound to three ADPs, revealing the first AAA+ protease assembly where the six protomers are arranged alternately in nucleotide-free and bound states.  Nucleotide binding induces large coordinated movements of conserved pore loops from two pairs of three non-adjacent protomers and shuttling of the proteolytic groove between the ATPase site and a previously unknown Arg paddle.  Structural and biochem. evidence supports the roles of the substrate-bound proteolytic groove in allosteric stimulation of ATPase activity and the conserved Arg paddle in driving substrate degrdn.  Altogether, this work provides a mol. framework for understanding how ATP-dependent chemomech. movements drive allosteric processes for substrate degrdn. in a major protein-destruction machine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxui42oJVTprVg90H21EOLACvtfcHk0lg4pkQ13zSiMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsVWisbk%253D&md5=60e828c93927d00b1754692183f90857</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.03.001%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DC.-C.%26aulast%3DSu%26aufirst%3DS.-C.%26aulast%3DSu%26aufirst%3DM.-Y.%26aulast%3DLiang%26aufirst%3DP.-H.%26aulast%3DFeng%26aufirst%3DC.-C.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DChang%26aufirst%3DC.-I.%26atitle%3DStructural%2520insights%2520into%2520the%2520allosteric%2520operation%2520of%2520the%2520Lon%2520AAA%252B%2520protease%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D667%26epage%3D675%26doi%3D10.1016%2Fj.str.2016.03.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tai, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-I.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the magnesium-dependent activation and hexamerization of the Lon AAA+ protease</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">676</span>– <span class="NLM_lpage">686</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.str.2016.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27041593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsVWisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=676-686&author=S.-C.+Suauthor=C.-C.+Linauthor=H.-C.+Taiauthor=M.-Y.+Changauthor=M.-R.+Hoauthor=C.+S.+Babuauthor=J.-H.+Liaoauthor=S.-H.+Wuauthor=Y.-C.+Changauthor=C.+Limauthor=C.-I.+Chang&title=Structural+basis+for+the+magnesium-dependent+activation+and+hexamerization+of+the+Lon+AAA%2B+protease&doi=10.1016%2Fj.str.2016.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Magnesium-Dependent Activation and Hexamerization of the Lon AAA+ Protease</span></div><div class="casAuthors">Su, Shih-Chieh; Lin, Chien-Chu; Tai, Hui-Chung; Chang, Mu-Yueh; Ho, Meng-Ru; Babu, C. Satheesan; Liao, Jiahn-Haur; Wu, Shih-Hsiung; Chang, Yuan-Chih; Lim, Carmay; Chang, Chung-I.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">676-686</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Lon AAA+ protease (LonA) plays important roles in protein homeostasis and regulation of diverse biol. processes.  LonA behaves as a homomeric hexamer in the presence of magnesium (Mg2+) and performs ATP-dependent proteolysis.  However, also LonA can carry out Mg2+-dependent degrdn. of unfolded protein substrate in an ATP-independent manner.  In the presence of Mg2+ LonA forms a non-secluded hexameric barrel with prominent openings, which explains why Mg2+-activated LonA can operate as a diffusion-based chambered protease to degrade unstructured protein and peptide substrates efficiently in the absence of ATP.  A 1.85 Å crystal structure of Mg2+-activated protease domain reveals Mg2+-dependent remodeling of a substrate-binding loop and a potential metal-binding site near the Ser-Lys catalytic dyad, supported by biophys. binding assays and mol. dynamics simulations.  Together, these findings reveal the specific roles of Mg2+ in the mol. assembly and activation of LonA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEImndg5i6bVg90H21EOLACvtfcHk0lgc2NXtK5p5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsVWisb0%253D&md5=716cec22a38a22fe1c83220d398c98b3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.-C.%26aulast%3DLin%26aufirst%3DC.-C.%26aulast%3DTai%26aufirst%3DH.-C.%26aulast%3DChang%26aufirst%3DM.-Y.%26aulast%3DHo%26aufirst%3DM.-R.%26aulast%3DBabu%26aufirst%3DC.%2BS.%26aulast%3DLiao%26aufirst%3DJ.-H.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DChang%26aufirst%3DY.-C.%26aulast%3DLim%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DC.-I.%26atitle%3DStructural%2520basis%2520for%2520the%2520magnesium-dependent%2520activation%2520and%2520hexamerization%2520of%2520the%2520Lon%2520AAA%252B%2520protease%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D676%26epage%3D686%26doi%3D10.1016%2Fj.str.2016.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishii, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukimoto-Niino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">A redox switch shapes the Lon protease exit pore to facultatively regulate proteolysis</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnchembio.1688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=25383757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKlsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=46-51&author=W.+Nishiiauthor=M.+Kukimoto-Niinoauthor=T.+Teradaauthor=M.+Shirouzuauthor=T.+Muramatsuauthor=M.+Kojimaauthor=H.+Kiharaauthor=S.+Yokoyama&title=A+redox+switch+shapes+the+Lon+protease+exit+pore+to+facultatively+regulate+proteolysis&doi=10.1038%2Fnchembio.1688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A redox switch shapes the Lon protease exit pore to facultatively regulate proteolysis</span></div><div class="casAuthors">Nishii, Wataru; Kukimoto-Niino, Mutsuko; Terada, Takaho; Shirouzu, Mikako; Muramatsu, Tomonari; Kojima, Masaki; Kihara, Hiroshi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-51</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Lon AAA+ protease degrades damaged or misfolded proteins in its intramol. chamber.  Its activity must be precisely controlled, but the mechanism by which Lon is regulated in response to different environments is not known.  Facultative anaerobes in the Enterobacteriaceae family, mostly symbionts and pathogens, encounter both anaerobic and aerobic environments inside and outside the host's body, resp.  The bacteria characteristically have two cysteine residues on the Lon protease (P) domain surface that unusually form a disulfide bond.  Here we show that the cysteine residues act as a redox switch of Lon.  Upon disulfide bond redn., the exit pore of the P-domain ring narrows by ∼30%, thus interrupting product passage and decreasing activity by 80%; disulfide bonding by oxidn. restores the pore size and activity.  The redox switch (E°' = -227 mV) is appropriately tuned to respond to variation between anaerobic and aerobic conditions, thus optimizing the cellular proteolysis level for each environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKuaoFS5ZXJrVg90H21EOLACvtfcHk0lgc2NXtK5p5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKlsrrN&md5=758d7025e61e175e4419496b61f8fef4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1688%26sid%3Dliteratum%253Aachs%26aulast%3DNishii%26aufirst%3DW.%26aulast%3DKukimoto-Niino%26aufirst%3DM.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DKojima%26aufirst%3DM.%26aulast%3DKihara%26aufirst%3DH.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DA%2520redox%2520switch%2520shapes%2520the%2520Lon%2520protease%2520exit%2520pore%2520to%2520facultatively%2520regulate%2520proteolysis%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D46%26epage%3D51%26doi%3D10.1038%2Fnchembio.1688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dikoglu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfaiz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertola, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, T.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derbent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alanay, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, O.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llerenar Jr, J. C.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reymond, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xenarios, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos-Xavier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonafé, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unger, S.</span></span> <span> </span><span class="NLM_article-title">Mutations in LONP1, a mitochondrial matrix protease, cause CODAS syndrome</span>. <i>Am. J. Med. Genet., Part A</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1501</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1002/ajmg.a.37029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1002%2Fajmg.a.37029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=25808063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVegsrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2015&pages=1501-1509&author=E.+Dikogluauthor=A.+Alfaizauthor=M.+Gornaauthor=D.+Bertolaauthor=J.+H.+Chaeauthor=T.-J.+Choauthor=M.+Derbentauthor=Y.+Alanayauthor=T.+Guranauthor=O.-H.+Kimauthor=J.+C.+Llerenar+Jrauthor=G.+Yamamotoauthor=G.+Superti-Furgaauthor=A.+Reymondauthor=I.+Xenariosauthor=B.+Stevensonauthor=B.+Campos-Xavierauthor=L.+Bonaf%C3%A9author=A.+Superti-Furgaauthor=S.+Unger&title=Mutations+in+LONP1%2C+a+mitochondrial+matrix+protease%2C+cause+CODAS+syndrome&doi=10.1002%2Fajmg.a.37029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in LONP1, a mitochondrial matrix protease, cause CODAS syndrome</span></div><div class="casAuthors">Dikoglu, Esra; Alfaiz, Ali; Gorna, Maria; Bertola, Deborah; Chae, Jong Hee; Cho, Tae-Joon; Derbent, Murat; Alanay, Yasemin; Guran, Tulay; Kim, Ok-Hwa; Llerenar, Juan C., jr.; Yamamoto, Guillerme; Superti-Furga, Giulio; Reymond, Alexandre; Xenarios, Ioannis; Stevenson, Brian; Campos-Xavier, Belinda; Bonafe, Luisa; Superti-Furga, Andrea; Unger, Sheila</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Medical Genetics, Part A</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1501-1509</span>CODEN:
                <span class="NLM_cas:coden">AJMGB8</span>;
        ISSN:<span class="NLM_cas:issn">1552-4825</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Cerebral, ocular, dental, auricular, skeletal anomalies (CODAS) syndrome (MIM 600373) was first described and named by Shehib et al, in 1991 in a single patient.  The anomalies referred to in the acronym are as follows: cerebral-developmental delay, ocular-cataracts, dental-aberrant cusp morphol. and delayed eruption, auricular-malformations of the external ear, and skeletal-spondyloepiphyseal dysplasia.  This distinctive constellation of anatomical findings should allow easy recognition but despite this only four apparently sporadic patients have been reported in the last 20 years indicating that the full phenotype is indeed very rare with perhaps milder or a typical presentations that are allelic but without sufficient phenotypic resemblance to permit clin. diagnosis.  We performed exome sequencing in three patients (an isolated case and a brother and sister sib pair) with classical features of CODAS.  Sanger sequencing was used to confirm results as well as for mutation discovery in a further four unrelated patients ascertained via their skeletal features.  Compd. heterozygous or homozygous mutations in LONP1 were found in all (8 sep. mutations; 6 missense, 1 nonsense, 1 small in-frame deletion) thus establishing the genetic basis of CODAS and the pattern of inheritance (autosomal recessive).  LONP1 encodes an enzyme of bacterial ancestry that participates in protein turnover within the mitochondrial matrix.  The mutations cluster at the ATP-binding and proteolytic domains of the enzyme.  Biallelic inheritance and clustering of mutations confirm dysfunction of LONP1 activity as the mol. basis of CODAS but the pathogenesis remains to be explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnLIR81-tzQrVg90H21EOLACvtfcHk0lgc2NXtK5p5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVegsrzE&md5=bc26ae2d47d630eb0a161bb1519b2ec0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fajmg.a.37029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fajmg.a.37029%26sid%3Dliteratum%253Aachs%26aulast%3DDikoglu%26aufirst%3DE.%26aulast%3DAlfaiz%26aufirst%3DA.%26aulast%3DGorna%26aufirst%3DM.%26aulast%3DBertola%26aufirst%3DD.%26aulast%3DChae%26aufirst%3DJ.%2BH.%26aulast%3DCho%26aufirst%3DT.-J.%26aulast%3DDerbent%26aufirst%3DM.%26aulast%3DAlanay%26aufirst%3DY.%26aulast%3DGuran%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DO.-H.%26aulast%3DLlerenar%2BJr%26aufirst%3DJ.%2BC.%26aulast%3DYamamoto%26aufirst%3DG.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DReymond%26aufirst%3DA.%26aulast%3DXenarios%26aufirst%3DI.%26aulast%3DStevenson%26aufirst%3DB.%26aulast%3DCampos-Xavier%26aufirst%3DB.%26aulast%3DBonaf%25C3%25A9%26aufirst%3DL.%26aulast%3DSuperti-Furga%26aufirst%3DA.%26aulast%3DUnger%26aufirst%3DS.%26atitle%3DMutations%2520in%2520LONP1%252C%2520a%2520mitochondrial%2520matrix%2520protease%252C%2520cause%2520CODAS%2520syndrome%26jtitle%3DAm.%2520J.%2520Med.%2520Genet.%252C%2520Part%2520A%26date%3D2015%26volume%3D167%26spage%3D1501%26epage%3D1509%26doi%3D10.1002%2Fajmg.a.37029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddington, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oláhová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommerville, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Champion, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siibak, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chrzanowska-Lightowlers, Z. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falkenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightowlers, R. N.</span></span> <span> </span><span class="NLM_article-title">Defective mitochondrial protease LonP1 can cause classical mitochondrial disease</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1743</span>– <span class="NLM_lpage">1753</span>, <span class="refDoi"> DOI: 10.1093/hmg/ddy080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1093%2Fhmg%2Fddy080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=29518248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGmtLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=1743-1753&author=B.+Peterauthor=C.+L.+Waddingtonauthor=M.+Ol%C3%A1hov%C3%A1author=E.+W.+Sommervilleauthor=S.+Hoptonauthor=A.+Pyleauthor=M.+Championauthor=M.+Ohlsonauthor=T.+Siibakauthor=Z.+M.+A.+Chrzanowska-Lightowlersauthor=R.+W.+Taylorauthor=M.+Falkenbergauthor=R.+N.+Lightowlers&title=Defective+mitochondrial+protease+LonP1+can+cause+classical+mitochondrial+disease&doi=10.1093%2Fhmg%2Fddy080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Defective mitochondrial protease LonP1 can cause classical mitochondrial disease</span></div><div class="casAuthors">Peter, Bradley; Waddington, Christie L.; Olahova, Monika; Sommerville, Ewen W.; Hopton, Sila; Pyle, Angela; Champion, Michael; Ohlson, Monica; Siibak, Triinu; Chrzanowska-Lightowlers, Zofia M. A.; Taylor, Robert W.; Falkenberg, Maria; Lightowlers, Robert N.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1743-1753</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">1460-2083</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">LonP1 is a mitochondrial matrix protease whose selective substrate specificity is essential for maintaining mitochondrial homeostasis.  Recessively inherited, pathogenic defects in LonP1 have been previously reported to underlie cerebral, ocular, dental, auricular and skeletal anomalies (CODAS) syndrome, a complex multisystemic and developmental disorder.  Intriguingly, although classical mitochondrial disease presentations are well-known to exhibit marked clin. heterogeneity, the skeletal and dental features assocd. with CODAS syndrome are pathognomonic.  We have applied whole exome sequencing to a patient with congenital lactic acidosis, muscle weakness, profound deficiencies in mitochondrial oxidative phosphorylation assocd. with loss of mtDNA copy no. and MRI abnormalities consistent with Leigh syndrome, identifying biallelic variants in the LONP1 (NM_004793.3) gene; c.1693T > C predicting p.(Tyr565His) and c.2197G > A predicting p.(Glu733Lys); no evidence of the classical skeletal or dental defects obsd. in CODAS syndrome patients were noted in our patient.  In vitro expts. confirmed the p.(Tyr565His) LonP1 mutant alone could not bind or degrade a substrate, consistent with the predicted function of Tyr565, while a second missense [p.(Glu733Lys)] variant had minimal effect.  Mixts. of p.(Tyr565His) mutant and wild-type LonP1 retained partial protease activity but this was severely depleted when the p.(Tyr565His) mutant was mixed with the p.(Glu733Lys) mutant, data consistent with the compd. heterozygosity detected in our patient.  In summary, we conclude that pathogenic LONP1 variants can lead to a classical mitochondrial disease presentations assocd. with severe biochem. defects in oxidative phosphorylation in clin. relevant tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHN_4cc2OkKLVg90H21EOLACvtfcHk0lhfbS8-CpWDRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGmtLjJ&md5=61601e3cdbc5ed404f792908486e02d4</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddy080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddy080%26sid%3Dliteratum%253Aachs%26aulast%3DPeter%26aufirst%3DB.%26aulast%3DWaddington%26aufirst%3DC.%2BL.%26aulast%3DOl%25C3%25A1hov%25C3%25A1%26aufirst%3DM.%26aulast%3DSommerville%26aufirst%3DE.%2BW.%26aulast%3DHopton%26aufirst%3DS.%26aulast%3DPyle%26aufirst%3DA.%26aulast%3DChampion%26aufirst%3DM.%26aulast%3DOhlson%26aufirst%3DM.%26aulast%3DSiibak%26aufirst%3DT.%26aulast%3DChrzanowska-Lightowlers%26aufirst%3DZ.%2BM.%2BA.%26aulast%3DTaylor%26aufirst%3DR.%2BW.%26aulast%3DFalkenberg%26aufirst%3DM.%26aulast%3DLightowlers%26aufirst%3DR.%2BN.%26atitle%3DDefective%2520mitochondrial%2520protease%2520LonP1%2520can%2520cause%2520classical%2520mitochondrial%2520disease%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2018%26volume%3D27%26spage%3D1743%26epage%3D1753%26doi%3D10.1093%2Fhmg%2Fddy080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span> <span> </span><span class="NLM_article-title">Down-regulating overexpressed human Lon in cervical cancer suppresses cell proliferation and bioenergetics</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e81084</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0081084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1371%2Fjournal.pone.0081084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=24260536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=X.+Nieauthor=M.+Liauthor=B.+Luauthor=Y.+Zhangauthor=L.+Lanauthor=L.+Chenauthor=J.+Lu&title=Down-regulating+overexpressed+human+Lon+in+cervical+cancer+suppresses+cell+proliferation+and+bioenergetics&doi=10.1371%2Fjournal.pone.0081084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Down-regulating overexpressed human Lon in cervical cancer suppresses cell proliferation and bioenergetics</span></div><div class="casAuthors">Nie, Xiaobo; Li, Min; Lu, Bin; Zhang, Yuxin; Lan, Linhua; Chen, Lin; Lu, Jianxin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e81084/1-e81084/9</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The human mitochondrial ATP-dependent Lon protease functions in regulating the metab. and quality control of proteins and mitochondrial DNA (mtDNA).  However, the role of Lon in cancer is not well understood.  Therefore, this study was undertaken to investigate the importance of Lon in cervical cancer cells from patients and in established cell lines.  Microarray anal. from 30 cancer and 10 normal cervical tissues were analyzed by immunohistochem. for Lon protein levels.  The expression of Lon was also examd. by immunoblotting 16 fresh cervical cancer tissues and their resp. non-tumor cervical tissues.  In all cases, Lon expression was significantly elevated in cervical carcinomas as compared to normal tissues.  Augmented Lon expression in tissue microarrays did not vary between age, tumor-node-metastasis grades or lymph node metastasis.  Knocking down Lon in HeLa cervical cancer cells by lentivrial transduction resulted in a substantial decrease in both mRNA and protein levels.  Such down-regulation of Lon expression significantly blocked HeLa cell proliferation.  In addn., knocking down Lon resulted in decreased cellular bioenergetics as detd. by measuring aerobic respiration and glycolysis using the Seahorse XF24 extracellular flux analyzer.  Together, these data demonstrate that Lon plays a potential role in the oncogenesis of cervical cancer and may be a useful biomarker and target in the treatment of cervical cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0Hgd0bBBkk7Vg90H21EOLACvtfcHk0lhfbS8-CpWDRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2hsw%253D%253D&md5=3d82ccdf7d96509514ac59c4e2f8fad0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0081084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0081084%26sid%3Dliteratum%253Aachs%26aulast%3DNie%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26atitle%3DDown-regulating%2520overexpressed%2520human%2520Lon%2520in%2520cervical%2520cancer%2520suppresses%2520cell%2520proliferation%2520and%2520bioenergetics%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0081084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quirós, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Español, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acín-Pérez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bárcena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loureiro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-García, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fueyo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enríquez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Otín, C.</span></span> <span> </span><span class="NLM_article-title">ATP-dependent Lon protease controls tumor bioenergetics by reprogramming mitochondrial activity</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">542</span>– <span class="NLM_lpage">556</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.celrep.2014.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=25017063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=542-556&author=P.+M.+Quir%C3%B3sauthor=Y.+Espa%C3%B1olauthor=R.+Ac%C3%ADn-P%C3%A9rezauthor=F.+Rodr%C3%ADguezauthor=C.+B%C3%A1rcenaauthor=K.+Watanabeauthor=E.+Calvoauthor=M.+Loureiroauthor=M.+S.+Fern%C3%A1ndez-Garc%C3%ADaauthor=A.+Fueyoauthor=J.+V%C3%A1zquezauthor=J.+A.+Enr%C3%ADquezauthor=C.+L%C3%B3pez-Ot%C3%ADn&title=ATP-dependent+Lon+protease+controls+tumor+bioenergetics+by+reprogramming+mitochondrial+activity&doi=10.1016%2Fj.celrep.2014.06.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-Dependent Lon Protease Controls Tumor Bioenergetics by Reprogramming Mitochondrial Activity</span></div><div class="casAuthors">Quiros, Pedro M.; Espanol, Yaiza; Acin-Perez, Rebeca; Rodriguez, Francisco; Barcena, Clea; Watanabe, Kenta; Calvo, Enrique; Loureiro, Marta; Fernandez-Garcia, M. Soledad; Fueyo, Antonio; Vazquez, Jesus; Enriquez, Jose Antonio; Lopez-Otin, Carlos</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">542-556</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We generated mice deficient in Lon protease (LONP1), a major enzyme of the mitochondrial quality control machinery.  Homozygous deletion of Lonp1 causes early embryonic lethality, whereas its haploinsufficiency protects against colorectal and skin tumors.  Furthermore, LONP1 knockdown inhibits cellular proliferation and tumor and metastasis formation, whereas its overexpression increases tumorigenesis.  Clin. studies indicate that high levels of LONP1 are a poor prognosis marker in human colorectal cancer and melanoma.  Addnl., functional analyses show that LONP1 plays a key role in metabolic reprogramming by remodeling OXPHOS complexes and protecting against senescence.  Our findings demonstrate the relevance of LONP1 for cellular and organismal viability and identify this protease as a central regulator of mitochondrial activity in oncogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRD-inqp2_r7Vg90H21EOLACvtfcHk0lg6__ZEOhNzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FI&md5=f85dcdc898cf61a442c4093f4b4661a0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DQuir%25C3%25B3s%26aufirst%3DP.%2BM.%26aulast%3DEspa%25C3%25B1ol%26aufirst%3DY.%26aulast%3DAc%25C3%25ADn-P%25C3%25A9rez%26aufirst%3DR.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DF.%26aulast%3DB%25C3%25A1rcena%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DK.%26aulast%3DCalvo%26aufirst%3DE.%26aulast%3DLoureiro%26aufirst%3DM.%26aulast%3DFern%25C3%25A1ndez-Garc%25C3%25ADa%26aufirst%3DM.%2BS.%26aulast%3DFueyo%26aufirst%3DA.%26aulast%3DV%25C3%25A1zquez%26aufirst%3DJ.%26aulast%3DEnr%25C3%25ADquez%26aufirst%3DJ.%2BA.%26aulast%3DL%25C3%25B3pez-Ot%25C3%25ADn%26aufirst%3DC.%26atitle%3DATP-dependent%2520Lon%2520protease%2520controls%2520tumor%2520bioenergetics%2520by%2520reprogramming%2520mitochondrial%2520activity%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D542%26epage%3D556%26doi%3D10.1016%2Fj.celrep.2014.06.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Biasi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span> <span> </span><span class="NLM_article-title">Emerging role of Lon protease as a master regulator of mitochondrial functions</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1857</i></span>,  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1306</span>, <span class="refDoi"> DOI: 10.1016/j.bbabio.2016.03.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.bbabio.2016.03.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27033304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls1OjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1857&publication_year=2016&pages=1300-1306&author=M.+Pintiauthor=L.+Gibelliniauthor=M.+Nasiauthor=S.+De+Biasiauthor=C.+A.+Bortolottiauthor=A.+Iannoneauthor=A.+Cossarizza&title=Emerging+role+of+Lon+protease+as+a+master+regulator+of+mitochondrial+functions&doi=10.1016%2Fj.bbabio.2016.03.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging role of Lon protease as a master regulator of mitochondrial functions</span></div><div class="casAuthors">Pinti, Marcello; Gibellini, Lara; Nasi, Milena; De Biasi, Sara; Bortolotti, Carlo Augusto; Iannone, Anna; Cossarizza, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Bioenergetics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1857</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1300-1306</span>CODEN:
                <span class="NLM_cas:coden">BBBEB4</span>;
        ISSN:<span class="NLM_cas:issn">0005-2728</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">Lon protease is a nuclear-encoded, mitochondrial ATP-dependent protease highly conserved throughout the evolution, crucial for the maintenance of mitochondrial homeostasis.  Lon acts as a chaperone of misfolded proteins, and is necessary for maintaining mitochondrial DNA.  The impairment of these functions has a deep impact on mitochondrial functionality and morphol.  An altered expression of Lon leads to a profound reprogramming of cell metab., with a switch from respiration to glycolysis, which is often obsd. in cancer cells.  Mutations of Lon, which likely impair its chaperone properties, are at the basis of a genetic inherited disease named of the cerebral, ocular, dental, auricular, skeletal (CODAS) syndrome.  This article is part of a Special Issue entitled 'EBEC 2016: 19th European Bioenergetics Conference, Riva del Garda, Italy, July 2-6, 2016', edited by Prof.  Paolo Bernardi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroqYZs21ZfILVg90H21EOLACvtfcHk0lg6__ZEOhNzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls1OjsLw%253D&md5=d4a141d356d944d01103d6845b60e19c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbabio.2016.03.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbabio.2016.03.025%26sid%3Dliteratum%253Aachs%26aulast%3DPinti%26aufirst%3DM.%26aulast%3DGibellini%26aufirst%3DL.%26aulast%3DNasi%26aufirst%3DM.%26aulast%3DDe%2BBiasi%26aufirst%3DS.%26aulast%3DBortolotti%26aufirst%3DC.%2BA.%26aulast%3DIannone%26aufirst%3DA.%26aulast%3DCossarizza%26aufirst%3DA.%26atitle%3DEmerging%2520role%2520of%2520Lon%2520protease%2520as%2520a%2520master%2520regulator%2520of%2520mitochondrial%2520functions%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2016%26volume%3D1857%26spage%3D1300%26epage%3D1306%26doi%3D10.1016%2Fj.bbabio.2016.03.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morozov, Y. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogenhagen, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Temiakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, C. K.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of human TFAM in mitochondria impairs DNA binding and promotes degradation by the AAA+ Lon protease</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.10.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.molcel.2012.10.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=23201127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqsrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=121-132&author=B.+Luauthor=J.+Leeauthor=X.+Nieauthor=M.+Liauthor=Y.+I.+Morozovauthor=S.+Venkateshauthor=D.+F.+Bogenhagenauthor=D.+Temiakovauthor=C.+K.+Suzuki&title=Phosphorylation+of+human+TFAM+in+mitochondria+impairs+DNA+binding+and+promotes+degradation+by+the+AAA%2B+Lon+protease&doi=10.1016%2Fj.molcel.2012.10.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of Human TFAM in Mitochondria Impairs DNA Binding and Promotes Degradation by the AAA+ Lon Protease</span></div><div class="casAuthors">Lu, Bin; Lee, Jae; Nie, Xiaobo; Li, Min; Morozov, Yaroslav I.; Venkatesh, Sundararajan; Bogenhagen, Daniel F.; Temiakov, Dmitry; Suzuki, Carolyn K.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">121-132</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Human mitochondrial transcription factor A (TFAM) is a high-mobility group (HMG) protein at the nexus of mitochondrial DNA (mtDNA) replication, transcription, and inheritance.  Little is known about the mechanisms underlying its posttranslational regulation.  Here, we demonstrate that TFAM is phosphorylated within its HMG box 1 (HMG1) by cAMP-dependent protein kinase in mitochondria.  HMG1 phosphorylation impairs the ability of TFAM to bind DNA and to activate transcription.  We show that only DNA-free TFAM is degraded by the Lon protease, which is inhibited by the anticancer drug bortezomib.  In cells with normal mtDNA levels, HMG1-phosphorylated TFAM is degraded by Lon.  However, in cells with severe mtDNA deficits, nonphosphorylated TFAM is also degraded, as it is DNA free.  Depleting Lon in these cells increases levels of TFAM and upregulates mtDNA content, albeit transiently.  Phosphorylation and proteolysis thus provide mechanisms for rapid fine-tuning of TFAM function and abundance in mitochondria, which are crucial for maintaining and expressing mtDNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr-71KRef3PLVg90H21EOLACvtfcHk0lg6__ZEOhNzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqsrbI&md5=74f6241a069401d65779a6e6894df180</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.10.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.10.023%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DNie%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DMorozov%26aufirst%3DY.%2BI.%26aulast%3DVenkatesh%26aufirst%3DS.%26aulast%3DBogenhagen%26aufirst%3DD.%2BF.%26aulast%3DTemiakov%26aufirst%3DD.%26aulast%3DSuzuki%26aufirst%3DC.%2BK.%26atitle%3DPhosphorylation%2520of%2520human%2520TFAM%2520in%2520mitochondria%2520impairs%2520DNA%2520binding%2520and%2520promotes%2520degradation%2520by%2520the%2520AAA%252B%2520Lon%2520protease%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D49%26spage%3D121%26epage%3D132%26doi%3D10.1016%2Fj.molcel.2012.10.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bárcena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Otín, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittig, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auburger, G.</span></span> <span> </span><span class="NLM_article-title">Global proteome of LonP1+/– mouse embryonal fibroblasts reveals impact on respiratory chain, but no interdependence between eral1 and mitoribosomes</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4523</span>, <span class="refDoi"> DOI: 10.3390/ijms20184523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.3390%2Fijms20184523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFSnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=4523&author=J.+Keyauthor=A.+Kohliauthor=C.+B%C3%A1rcenaauthor=C.+L%C3%B3pez-Ot%C3%ADnauthor=J.+Heidlerauthor=I.+Wittigauthor=G.+Auburger&title=Global+proteome+of+LonP1%2B%2F%E2%80%93+mouse+embryonal+fibroblasts+reveals+impact+on+respiratory+chain%2C+but+no+interdependence+between+eral1+and+mitoribosomes&doi=10.3390%2Fijms20184523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Global proteome of LonP1+/-mouse embryonal fibroblasts reveals impact on respiratory chain, but no interdependence between Eral1 and mitoribosomes</span></div><div class="casAuthors">Key, Jana; Kohli, Aneesha; Barcena, Clea; Lopez-Otin, Carlos; Heidler, Juliana; Wittig, Ilka; Auburger, Georg</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4523</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Research on healthy aging shows that lifespan redns. are often caused by mitochondrial dysfunction.  Thus, it is very interesting that the deletion of mitochondrial matrix peptidase LonP1 was obsd. to abolish embryogenesis, while deletion of the mitochondrial matrix peptidase Caseinolytic Mitochondrial Matrix Peptidase Proteolytic Subunit (ClpP) prolonged survival.  To unveil the targets of each enzyme, we documented the global proteome of LonP1+/- mouse embryonal fibroblasts (MEF), for comparison with ClpP-/- depletion.  Proteomic profiles of LonP1+/- MEF generated by label-free mass spectrometry were further processed with the STRING (Search tool for the retrieval of interacting genes) webserver Heidelberg for protein interactions.  ClpP was previously reported to degrade Eral1 as a chaperone involved in mitoribosome assembly, so ClpP deficiency triggers the accumulation of mitoribosomal subunits and inefficient translation.  LonP1+/- MEF also showed Eral1 accumulation, but no systematic effect on mitoribosomal subunits.  In contrast to ClpP-/- profiles, several components of the respiratory complex-I membrane arm, of the glutathione pathway and of lysosomes were accumulated, whereas the upregulation of numerous innate immune defense components was similar.  Overall, LonP1, as opposed to ClpP, appears to have no effect on translational machinery, instead it shows enhanced respiratory dysfunction; this agrees with reports on the human CODAS syndrome (syndrome with cerebral, ocular, dental, auricular, and skeletal anomalies) caused by LonP1 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Gnv13iaGA7Vg90H21EOLACvtfcHk0lg6__ZEOhNzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFSnsLs%253D&md5=410002d95971ead3a99ecb19bb5634da</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3390%2Fijms20184523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20184523%26sid%3Dliteratum%253Aachs%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DKohli%26aufirst%3DA.%26aulast%3DB%25C3%25A1rcena%26aufirst%3DC.%26aulast%3DL%25C3%25B3pez-Ot%25C3%25ADn%26aufirst%3DC.%26aulast%3DHeidler%26aufirst%3DJ.%26aulast%3DWittig%26aufirst%3DI.%26aulast%3DAuburger%26aufirst%3DG.%26atitle%3DGlobal%2520proteome%2520of%2520LonP1%252B%252F%25E2%2580%2593%2520mouse%2520embryonal%2520fibroblasts%2520reveals%2520impact%2520on%2520respiratory%2520chain%252C%2520but%2520no%2520interdependence%2520between%2520eral1%2520and%2520mitoribosomes%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D4523%26doi%3D10.3390%2Fijms20184523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibellini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Biasi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo Tartaro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecorini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patergnani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Gaetano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carnevale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisciotta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mariani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossarizza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinti, M.</span></span> <span> </span><span class="NLM_article-title">LonP1 differently modulates mitochondrial function and bioenergetics of primary versus metastatic colon cancer cells</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">254</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.3389%2Ffonc.2018.00254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=30038898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A280%3ADC%252BB3c7ht1Crtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=254&author=L.+Gibelliniauthor=L.+Losiauthor=S.+De+Biasiauthor=M.+Nasiauthor=D.+Lo+Tartaroauthor=S.+Pecoriniauthor=S.+Patergnaniauthor=P.+Pintonauthor=A.+De+Gaetanoauthor=G.+Carnevaleauthor=A.+Pisciottaauthor=F.+Marianiauthor=L.+Roncucciauthor=A.+Iannoneauthor=A.+Cossarizzaauthor=M.+Pinti&title=LonP1+differently+modulates+mitochondrial+function+and+bioenergetics+of+primary+versus+metastatic+colon+cancer+cells&doi=10.3389%2Ffonc.2018.00254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">LonP1 Differently Modulates Mitochondrial Function and Bioenergetics of Primary Versus Metastatic Colon Cancer Cells</span></div><div class="casAuthors">Gibellini Lara; Lo Tartaro Domenico; Cossarizza Andrea; Losi Lorena; De Biasi Sara; Pinti Marcello; Nasi Milena; Pecorini Simone; De Gaetano Anna; Carnevale Gianluca; Pisciotta Alessandra; Patergnani Simone; Pinton Paolo; Mariani Francesco; Roncucci Luca; Iannone Anna</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">254</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Mitochondrial Lon protease (LonP1) is a multi-function enzyme that regulates mitochondrial functions in several human malignancies, including colorectal cancer (CRC).  The mechanism(s) by which LonP1 contributes to colorectal carcinogenesis is not fully understood.  We found that silencing LonP1 leads to severe mitochondrial impairment and apoptosis in colon cancer cells.  Here, we investigate the role of LonP1 in mitochondrial functions, metabolism, and epithelial-mesenchymal transition (EMT) in colon tumor cells and in metastasis.  LonP1 was almost absent in normal mucosa, gradually increased from aberrant crypt foci to adenoma, and was most abundant in CRC.  Moreover, LonP1 was preferentially upregulated in colorectal samples with mutated p53 or nuclear β-catenin, and its overexpression led to increased levels of β-catenin and decreased levels of E-cadherin, key proteins in EMT, in vitro.  LonP1 upregulation also induced opposite changes in oxidative phosphorylation, glycolysis, and pentose pathway in SW480 primary colon tumor cells when compared to SW620 metastatic colon cancer cells.  In conclusion, basal LonP1 expression is essential for normal mitochondrial function, and increased LonP1 levels in SW480 and SW620 cells induce a metabolic shift toward glycolysis, leading to EMT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRR3-Hvit6XqDslzEEsg76ufW6udTcc2eYwX45MnUdLEbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7ht1Crtg%253D%253D&md5=369dea5c17a0ffe9c3e2854c37b0d050</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00254%26sid%3Dliteratum%253Aachs%26aulast%3DGibellini%26aufirst%3DL.%26aulast%3DLosi%26aufirst%3DL.%26aulast%3DDe%2BBiasi%26aufirst%3DS.%26aulast%3DNasi%26aufirst%3DM.%26aulast%3DLo%2BTartaro%26aufirst%3DD.%26aulast%3DPecorini%26aufirst%3DS.%26aulast%3DPatergnani%26aufirst%3DS.%26aulast%3DPinton%26aufirst%3DP.%26aulast%3DDe%2BGaetano%26aufirst%3DA.%26aulast%3DCarnevale%26aufirst%3DG.%26aulast%3DPisciotta%26aufirst%3DA.%26aulast%3DMariani%26aufirst%3DF.%26aulast%3DRoncucci%26aufirst%3DL.%26aulast%3DIannone%26aufirst%3DA.%26aulast%3DCossarizza%26aufirst%3DA.%26aulast%3DPinti%26aufirst%3DM.%26atitle%3DLonP1%2520differently%2520modulates%2520mitochondrial%2520function%2520and%2520bioenergetics%2520of%2520primary%2520versus%2520metastatic%2520colon%2520cancer%2520cells%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D254%26doi%3D10.3389%2Ffonc.2018.00254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">744</span>, <span class="refDoi"> DOI: 10.1038/nchembio1207-739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnchembio1207-739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=18007642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=739-744&author=W.+A.+Weissauthor=S.+S.+Taylorauthor=K.+M.+Shokat&title=Recognizing+and+exploiting+differences+between+RNAi+and+small-molecule+inhibitors&doi=10.1038%2Fnchembio1207-739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span></div><div class="casAuthors">Weiss, William A.; Taylor, Stephen S.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The biol. of RNA interference has greatly facilitated anal. of loss-of-function phenotypes, but correlating these phenotypes with small-mol. inhibition profiles is not always straightforward.  The authors examine the rationale of comparing RNA interference to pharmacol. intervention in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JPzqVyshS7Vg90H21EOLACvtfcHk0ljccysxv1jgOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO&md5=9a26c990b358a6b2c717d1e361cac2da</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1207-739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1207-739%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DRecognizing%2520and%2520exploiting%2520differences%2520between%2520RNAi%2520and%2520small-molecule%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnchembio1207-739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carter, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller-Fahrnow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A. M.</span></span> <span> </span><span class="NLM_article-title">Target 2035: probing the human proteome</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2111</span>– <span class="NLM_lpage">2115</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2019.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.drudis.2019.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=31278990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=2111-2115&author=A.+J.+Carterauthor=O.+Kraemerauthor=M.+Zwickauthor=A.+Mueller-Fahrnowauthor=C.+H.+Arrowsmithauthor=A.+M.+Edwards&title=Target+2035%3A+probing+the+human+proteome&doi=10.1016%2Fj.drudis.2019.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Target 2035: probing the human proteome</span></div><div class="casAuthors">Carter, Adrian J.; Kraemer, Oliver; Zwick, Matthias; Mueller-Fahrnow, Anke; Arrowsmith, Cheryl H.; Edwards, Aled M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2111-2115</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Biomedical scientists tend to focus on only a small fraction of the proteins encoded by the human genome despite overwhelming genetic evidence that many understudied proteins are important for human disease.  One of the best ways to interrogate the function of a protein and to det. its relevance as a drug target is by using a pharmacol. modulator, such as a chem. probe or an antibody.  If these tools were available for most human proteins, it should be possible to translate the tremendous advances in genomics into a greater understanding of human health and disease, and catalyze the creation of innovative new medicines.  Target 2035 is a global federation for developing and applying new technologies with the goal of creating chemogenomic libraries, chem. probes, and/or functional antibodies for the entire proteome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY5b9ZXh0ALVg90H21EOLACvtfcHk0lhf38zsuEp1yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlahtrnN&md5=212e832238c8860439b343cbeef17b74</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2019.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2019.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DA.%2BJ.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DZwick%26aufirst%3DM.%26aulast%3DMueller-Fahrnow%26aufirst%3DA.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DEdwards%26aufirst%3DA.%2BM.%26atitle%3DTarget%25202035%253A%2520probing%2520the%2520human%2520proteome%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2019%26volume%3D24%26spage%3D2111%26epage%3D2115%26doi%3D10.1016%2Fj.drudis.2019.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Melderen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertsen, A.</span></span> <span> </span><span class="NLM_article-title">Regulation and quality control by Lon-dependent proteolysis</span>. <i>Res. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>160</i></span>,  <span class="NLM_fpage">645</span>– <span class="NLM_lpage">651</span>, <span class="refDoi"> DOI: 10.1016/j.resmic.2009.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.resmic.2009.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=19772918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVens7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=160&publication_year=2009&pages=645-651&author=L.+Van%0AMelderenauthor=A.+Aertsen&title=Regulation+and+quality+control+by+Lon-dependent+proteolysis&doi=10.1016%2Fj.resmic.2009.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and quality control by Lon-dependent proteolysis</span></div><div class="casAuthors">Van Melderen, Laurence; Aertsen, Abram</div><div class="citationInfo"><span class="NLM_cas:title">Research in Microbiology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">160</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">645-651</span>CODEN:
                <span class="NLM_cas:coden">RMCREW</span>;
        ISSN:<span class="NLM_cas:issn">0923-2508</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  After their first discovery in Escherichia coli, Lon homologues were found to be widely distributed among prokaryotes to eukaryotes.  The ATP-dependent Lon protease belongs to the AAA+ (ATPases assocd. with a variety of cellular activities) superfamily, and is involved in both general quality control by degrading abnormal proteins and in the specific control of several regulatory proteins.  As such, this enzyme has a pivotal role in quality control and cellular physiol.  This review focuses on mechanisms of degrdn. both from the protease and substrate points of view, and discusses the role of Lon in global regulation, stress response and virulence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqrcB5DNPwErVg90H21EOLACvtfcHk0lhf38zsuEp1yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVens7bN&md5=7f16df349f0a135b0899703b578f339a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.resmic.2009.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.resmic.2009.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BMelderen%26aufirst%3DL.%26aulast%3DAertsen%26aufirst%3DA.%26atitle%3DRegulation%2520and%2520quality%2520control%2520by%2520Lon-dependent%2520proteolysis%26jtitle%3DRes.%2520Microbiol.%26date%3D2009%26volume%3D160%26spage%3D645%26epage%3D651%26doi%3D10.1016%2Fj.resmic.2009.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fishovitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frase, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.</span></span> <span> </span><span class="NLM_article-title">Active-site-directed chemical tools for profiling mitochondrial Lon protease</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">781</span>– <span class="NLM_lpage">788</span>, <span class="refDoi"> DOI: 10.1021/cb100408w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100408w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlslCisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=781-788&author=J.+Fishovitzauthor=M.+Liauthor=H.+Fraseauthor=J.+Hudakauthor=S.+Craigauthor=K.+Koauthor=A.+J.+Berdisauthor=C.+K.+Suzukiauthor=I.+Lee&title=Active-site-directed+chemical+tools+for+profiling+mitochondrial+Lon+protease&doi=10.1021%2Fcb100408w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Active-Site-Directed Chemical Tools for Profiling Mitochondrial Lon Protease</span></div><div class="casAuthors">Fishovitz, Jennifer; Li, Min; Frase, Hilary; Hudak, Jason; Craig, Sandra; Ko, Kristin; Berdis, Anthony J.; Suzuki, Carolyn K.; Lee, Irene</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">781-788</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lon and ClpXP are the only sol. ATP-dependent proteases within the mammalian mitochondria matrix, which function in protein quality control by selectively degrading misfolded, misassembled, or damaged proteins.  Chem. tools to study these proteases in biol. samples have not been identified, thereby hindering a clear understanding of their resp. functions in normal and disease states.  In this study, we applied a proteolytic site-directed approach to identify a peptide reporter substrate and a peptide inhibitor that are selective for Lon but not ClpXP.  These chem. tools permit quant. measurements that distinguish Lon-mediated proteolysis from that of ClpXP in biochem. assays with purified proteases, as well as in intact mitochondria and mitochondrial lysates.  This chem. biol. approach provides needed tools to further our understanding of mitochondrial ATP-dependent proteolysis and contributes to the future development of diagnostic and pharmacol. agents for treating diseases assocd. with defects in mitochondrial protein quality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8weIfQwHGArVg90H21EOLACvtfcHk0lhf38zsuEp1yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlslCisrk%253D&md5=fe959dbb4cd00e3203ad4b4d4f56f9bf</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fcb100408w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100408w%26sid%3Dliteratum%253Aachs%26aulast%3DFishovitz%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFrase%26aufirst%3DH.%26aulast%3DHudak%26aufirst%3DJ.%26aulast%3DCraig%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DK.%26aulast%3DBerdis%26aufirst%3DA.%2BJ.%26aulast%3DSuzuki%26aufirst%3DC.%2BK.%26aulast%3DLee%26aufirst%3DI.%26atitle%3DActive-site-directed%2520chemical%2520tools%2520for%2520profiling%2520mitochondrial%2520Lon%2520protease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D781%26epage%3D788%26doi%3D10.1021%2Fcb100408w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csizmadia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hales, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senn, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadambi, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolenski, F. S.</span></span> <span> </span><span class="NLM_article-title">Proteasome inhibitors bortezomib and carfilzomib used for the treatment of multiple myeloma do not inhibit the serine protease HtrA2/Omi</span>. <i>Toxicol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1619</span>– <span class="NLM_lpage">1628</span>, <span class="refDoi"> DOI: 10.1039/c6tx00220j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1039%2FC6TX00220J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVSlu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1619-1628&author=V.+Csizmadiaauthor=P.+Halesauthor=C.+Tsuauthor=J.+Maauthor=J.+Chenauthor=P.+Shahauthor=P.+Flemingauthor=J.+J.+Sennauthor=V.+J.+Kadambiauthor=L.+Dickauthor=F.+S.+Wolenski&title=Proteasome+inhibitors+bortezomib+and+carfilzomib+used+for+the+treatment+of+multiple+myeloma+do+not+inhibit+the+serine+protease+HtrA2%2FOmi&doi=10.1039%2Fc6tx00220j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Proteasome inhibitors bortezomib and carfilzomib used for the treatment of multiple myeloma do not inhibit the serine protease HtrA2/Omi</span></div><div class="casAuthors">Csizmadia, Vilmos; Hales, Paul; Tsu, Christopher; Ma, Jingya; Chen, Jiejin; Shah, Pooja; Fleming, Paul; Senn, Joseph J.; Kadambi, Vivek J.; Dick, Larry; Wolenski, Francis S.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Research (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1619-1628</span>CODEN:
                <span class="NLM_cas:coden">TROEE8</span>;
        ISSN:<span class="NLM_cas:issn">2045-4538</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The proteasome inhibitor bortezomib is assocd. with the development of peripheral neuropathy in patients, but the mechanism by which bortezomib can induce peripheral neuropathy is not fully understood.  One study suggested that off-target inhibition of proteases other than the proteasome, particularly HtraA2/Omi, may be the underlying mechanism of the neuropathy.  The same study also concluded that carfilzomib, a second proteasome inhibitor that is assocd. with less peripheral neuropathy in patients than bortezomib, showed no inhibition of HtrA2/Omi.  The goal of the work described here was to det. whether either proteasome inhibitors truly affected HtrA2/Omi activity.  A variety of methods were used to test the effects of both bortezomib and carfilzomib on HtrA2/Omi activity that included in vitro recombinant enzyme assays, and studies with the human neuroblastoma SH-SY5Y cell line and HtrA2/Omi-knockout mouse embryonic fibroblasts.  The compd. ucf-101 was used to assess the effects of specific HtrA2/Omi inhibition.  In contrast to previously published data, our results clearly demonstrated that neither bortezomib nor carfilzomib inhibited HtrA2/Omi activity in recombinant enzyme assays at concns. up to 100 μM, while the specific inhibitor ucf-101 did inhibit the enzyme.  The proteasome inhibitors did not inhibit HtrA2/Omi activity in either SH-SY5Y cells or mouse embryonic fibroblasts, as detd. by expression of the HtrA2/Omi substrates eIF4G1 and UCH-L1.  Based on our biochem. and cell-based assays, we conclude that neither bortezomib nor carfilzomib inhibited HtrA2/Omi activity.  Therefore, it is unlikely that bortezomib assocd. peripheral neuropathy is a direct result of off-target inhibition of HtrA2/Omi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_aWn0wUKFt7Vg90H21EOLACvtfcHk0lg9F6EoEszxIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVSlu7zM&md5=56c1421e9a9c24b3c11b9bd6fb7443ec</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC6TX00220J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6TX00220J%26sid%3Dliteratum%253Aachs%26aulast%3DCsizmadia%26aufirst%3DV.%26aulast%3DHales%26aufirst%3DP.%26aulast%3DTsu%26aufirst%3DC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DSenn%26aufirst%3DJ.%2BJ.%26aulast%3DKadambi%26aufirst%3DV.%2BJ.%26aulast%3DDick%26aufirst%3DL.%26aulast%3DWolenski%26aufirst%3DF.%2BS.%26atitle%3DProteasome%2520inhibitors%2520bortezomib%2520and%2520carfilzomib%2520used%2520for%2520the%2520treatment%2520of%2520multiple%2520myeloma%2520do%2520not%2520inhibit%2520the%2520serine%2520protease%2520HtrA2%252FOmi%26jtitle%3DToxicol.%2520Res.%26date%3D2016%26volume%3D5%26spage%3D1619%26epage%3D1628%26doi%3D10.1039%2Fc6tx00220j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ihara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, C.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakatsuki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, C.-I.</span></span> <span> </span><span class="NLM_article-title">Structures of an ATP-independent Lon-like protease and its complexes with covalent inhibitors</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1395</span>– <span class="NLM_lpage">1402</span>, <span class="refDoi"> DOI: 10.1107/s0907444913008214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1107%2FS0907444913008214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=23897463" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1395-1402&author=J.-H.+Liaoauthor=K.+Iharaauthor=C.-I.+Kuoauthor=K.-F.+Huangauthor=S.+Wakatsukiauthor=S.-H.+Wuauthor=C.-I.+Chang&title=Structures+of+an+ATP-independent+Lon-like+protease+and+its+complexes+with+covalent+inhibitors&doi=10.1107%2Fs0907444913008214"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444913008214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913008214%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DJ.-H.%26aulast%3DIhara%26aufirst%3DK.%26aulast%3DKuo%26aufirst%3DC.-I.%26aulast%3DHuang%26aufirst%3DK.-F.%26aulast%3DWakatsuki%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DS.-H.%26aulast%3DChang%26aufirst%3DC.-I.%26atitle%3DStructures%2520of%2520an%2520ATP-independent%2520Lon-like%2520protease%2520and%2520its%2520complexes%2520with%2520covalent%2520inhibitors%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2013%26volume%3D69%26spage%3D1395%26epage%3D1402%26doi%3D10.1107%2Fs0907444913008214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kupperman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannerman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hales, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruzzese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfredi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation of the proteasome inhibitor MLN9708 in preclinical models of human cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1970</span>– <span class="NLM_lpage">1980</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-09-2766</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1158%2F0008-5472.CAN-09-2766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=20160034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisFWisro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=1970-1980&author=E.+Kuppermanauthor=E.+C.+Leeauthor=Y.+Caoauthor=B.+Bannermanauthor=M.+Fitzgeraldauthor=A.+Bergerauthor=J.+Yuauthor=Y.+Yangauthor=P.+Halesauthor=F.+Bruzzeseauthor=J.+Liuauthor=J.+Blankauthor=K.+Garciaauthor=C.+Tsuauthor=L.+Dickauthor=P.+Flemingauthor=L.+Yuauthor=M.+Manfrediauthor=M.+Rolfeauthor=J.+Bolen&title=Evaluation+of+the+proteasome+inhibitor+MLN9708+in+preclinical+models+of+human+cancer&doi=10.1158%2F0008-5472.can-09-2766"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of the Proteasome Inhibitor MLN9708 in Preclinical Models of Human Cancer</span></div><div class="casAuthors">Kupperman, Erik; Lee, Edmund C.; Cao, Yueying; Bannerman, Bret; Fitzgerald, Michael; Berger, Allison; Yu, Jie; Yang, Yu; Bruzzese, Frank; Liu, Jane; Blank, Jonathan; Garcia, Khristofer; Tsu, Christopher; Dick, Larry; Fleming, Paul; Yu, Li; Manfredi, Mark; Rolfe, Mark; Bolen, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1970-1980</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The proteasome was validated as an oncol. target following the clin. success of VELCADE (bortezomib) for injection for the treatment of multiple myeloma and recurring mantle cell lymphoma.  Consequently, several groups are pursuing the development of addnl. small-mol. proteasome inhibitors for both hematol. and solid tumor indications.  Here, we describe MLN9708, a selective, orally bioavailable, second-generation proteasome inhibitor that is in phase I clin. development.  MLN9708 has a shorter proteasome dissocn. half-life and improved pharmacokinetics, pharmacodynamics, and antitumor activity compared with bortezomib.  MLN9708 has a larger blood vol. distribution at steady state, and anal. of 20S proteasome inhibition and markers of the unfolded protein response confirmed that MLN9708 has greater pharmacodynamic effects in tissues than bortezomib.  MLN9708 showed activity in both solid tumor and hematol. preclin. xenograft models, and we found a correlation between greater pharmacodynamic responses and improved antitumor activity.  Moreover, antitumor activity was shown via multiple dosing routes, including oral gavage.  Taken together, these data support the clin. development of MLN9708 for both hematol. and solid tumor indications.  Cancer Res; 70(5); 1970-80.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjauqAFEgDA7Vg90H21EOLACvtfcHk0lg9F6EoEszxIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisFWisro%253D&md5=45636404582fc0eb66581c6c19420458</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-2766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-2766%26sid%3Dliteratum%253Aachs%26aulast%3DKupperman%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DBannerman%26aufirst%3DB.%26aulast%3DFitzgerald%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHales%26aufirst%3DP.%26aulast%3DBruzzese%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBlank%26aufirst%3DJ.%26aulast%3DGarcia%26aufirst%3DK.%26aulast%3DTsu%26aufirst%3DC.%26aulast%3DDick%26aufirst%3DL.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DManfredi%26aufirst%3DM.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%26atitle%3DEvaluation%2520of%2520the%2520proteasome%2520inhibitor%2520MLN9708%2520in%2520preclinical%2520models%2520of%2520human%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D1970%26epage%3D1980%26doi%3D10.1158%2F0008-5472.can-09-2766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behnke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruickshank, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klunder, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plamondon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, R. L.</span></span> <span> </span><span class="NLM_article-title">Potent and selective inhibitors of the proteasome: dipeptidyl boronic acids</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">338</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(98)00029-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2FS0960-894X%2898%2900029-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=9871680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaK1cXhsFKjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=333-338&author=J.+Adamsauthor=M.+Behnkeauthor=S.+Chenauthor=A.+A.+Cruickshankauthor=L.+R.+Dickauthor=L.+Grenierauthor=J.+M.+Klunderauthor=Y.-T.+Maauthor=L.+Plamondonauthor=R.+L.+Stein&title=Potent+and+selective+inhibitors+of+the+proteasome%3A+dipeptidyl+boronic+acids&doi=10.1016%2Fs0960-894x%2898%2900029-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective inhibitors of the proteasome: dipeptidyl boronic acids</span></div><div class="casAuthors">Adams, Julian; Behnke, Mark; Chen, Shaowu; Cruickshank, Amy A.; Dick, Lawrence R.; Grenier, Louis; Klunder, Jancie M.; Ma, Yu-Ting; Plamondon, Louis; Stein, Ross L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">333-338</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Potent and selective dipeptidyl boronic acid proteasome inhibitors are described.  As compared to peptidyl aldehyde compds., boronic acids in this series display dramatically enhanced potency.  Compds. (I; R = Ph, 1-naphthyl), in particular I (R = Ph), are promising new therapeutics for treatment of cancer and inflammatory diseases.  I (R = Ph) and I (R = 1-naphthyl) in vitro inhibited 20S proteasome with Ki of 0.62 and 0.18, resp., whereas I (R = Ph) showed Ki of 2,300, 630, 320, and 13,000 against common serine proteases such as human leukocyte elastase, human cathepsin G, human chymotrypsin, and thrombin, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeLVVXMKjfPbVg90H21EOLACvtfcHk0lg9F6EoEszxIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhsFKjsr8%253D&md5=e9287f4977f5ab655fedd1fd702aa456</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2898%2900029-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252898%252900029-8%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%26aulast%3DBehnke%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DCruickshank%26aufirst%3DA.%2BA.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DGrenier%26aufirst%3DL.%26aulast%3DKlunder%26aufirst%3DJ.%2BM.%26aulast%3DMa%26aufirst%3DY.-T.%26aulast%3DPlamondon%26aufirst%3DL.%26aulast%3DStein%26aufirst%3DR.%2BL.%26atitle%3DPotent%2520and%2520selective%2520inhibitors%2520of%2520the%2520proteasome%253A%2520dipeptidyl%2520boronic%2520acids%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1998%26volume%3D8%26spage%3D333%26epage%3D338%26doi%3D10.1016%2Fs0960-894x%2898%2900029-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J.</span>; <span class="NLM_string-name">Ma, Y.-T.</span>; <span class="NLM_string-name">Stein, R.</span>; <span class="NLM_string-name">Baevsky, M.</span>; <span class="NLM_string-name">Grenier, L.</span>; <span class="NLM_string-name">Plamondon, L.</span></span> <span> </span><span class="NLM_article-title">Boronic Ester and Acid Compounds</span>. U.S. Patent <span class="NLM_patent">5,780,454 A</span>, <span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=J.+Adams&author=Y.-T.+Ma&author=R.+Stein&author=M.+Baevsky&author=L.+Grenier&author=L.+Plamondon&title=Boronic+Ester+and+Acid+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%26atitle%3DBoronic%2520Ester%2520and%2520Acid%2520Compounds%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhalnina, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shishkov, S. V.</span></span> <span> </span><span class="NLM_article-title">A convergent approach to synthesis of bortezomib: the use of TBTU suppresses racemization in the fragment condensation</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">7105</span>– <span class="NLM_lpage">7108</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2009.06.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.tet.2009.06.042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptVGgtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=7105-7108&author=A.+S.+Ivanovauthor=A.+A.+Zhalninaauthor=S.+V.+Shishkov&title=A+convergent+approach+to+synthesis+of+bortezomib%3A+the+use+of+TBTU+suppresses+racemization+in+the+fragment+condensation&doi=10.1016%2Fj.tet.2009.06.042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A convergent approach to synthesis of bortezomib: the use of TBTU suppresses racemization in the fragment condensation</span></div><div class="casAuthors">Ivanov, Andrey S.; Zhalnina, Anna A.; Shishkov, Sergey V.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">7105-7108</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bortezomib (I) is a first-in-class therapeutic antineoplastic agent used for treating patients with multiple myeloma and mantle cell lymphoma.  In this paper we report an improved method for synthesis of the title compd. using a convergent approach.  TBTU was found to efficiently suppress racemization in the fragment condensation.  In comparison with the original synthesis, the presented one is shorter by two steps, higher in yield, and provides better atom economy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoju9EMTW_imrVg90H21EOLACvtfcHk0ljXduSiWN3F5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptVGgtL8%253D&md5=e6bc84fc1b41d91164910fc186697986</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2009.06.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2009.06.042%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DA.%2BS.%26aulast%3DZhalnina%26aufirst%3DA.%2BA.%26aulast%3DShishkov%26aufirst%3DS.%2BV.%26atitle%3DA%2520convergent%2520approach%2520to%2520synthesis%2520of%2520bortezomib%253A%2520the%2520use%2520of%2520TBTU%2520suppresses%2520racemization%2520in%2520the%2520fragment%2520condensation%26jtitle%3DTetrahedron%26date%3D2009%26volume%3D65%26spage%3D7105%26epage%3D7108%26doi%3D10.1016%2Fj.tet.2009.06.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matteson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadhu, K. M.</span></span> <span> </span><span class="NLM_article-title">Boronic ester homologation with 99% chiral selectivity and its use in syntheses of the insect pheromones (3S,4S)-4-methyl-3-heptanol and exo-brevicomin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">2077</span>– <span class="NLM_lpage">2078</span>, <span class="refDoi"> DOI: 10.1021/ja00345a074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00345a074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaL3sXhs1Snurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=1983&pages=2077-2078&author=D.+S.+Mattesonauthor=K.+M.+Sadhu&title=Boronic+ester+homologation+with+99%25+chiral+selectivity+and+its+use+in+syntheses+of+the+insect+pheromones+%283S%2C4S%29-4-methyl-3-heptanol+and+exo-brevicomin&doi=10.1021%2Fja00345a074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Boronic ester homologation with 99% chiral selectivity and its use in syntheses of the insect pheromones (3S,4S)-4-methyl-3-heptanol and exo-brevicomin</span></div><div class="casAuthors">Matteson, Donald S.; Sadhu, Kizhakethil M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2077-8</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The directed chiral synthesis of α-chloro boronic esters I [R = Me, Pr, Bu, CH2CHMe2, CH2Ph, 3-(2-methyl-2-dioxolanyl)propyl, 1-benzyloxy-3-(2-methyl-2-dioxolanyl)propyl] from II has been dramatically improved by ZnCl2 catalysis of the rearrangement of the intermediate borate complexes.  Diastereoselectivities are 95-99.5%, yields 82-99%.  Also homologated was II (R = 2-pentyl) with high but not fully proved diastereoselectivity.  Two insect pheromone components, (3S,4S)-4-methyl-3-heptanol (European elm bark beetle, Scolytus multistriatus) and exo-brevicomin (western pine beetle, Dendroctonus brevicomis) were prepd. by this route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonw-PWPn86YrVg90H21EOLACvtfcHk0ljXduSiWN3F5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhs1Snurw%253D&md5=6a3c22886313ed92a9150e555e3aefcc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fja00345a074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00345a074%26sid%3Dliteratum%253Aachs%26aulast%3DMatteson%26aufirst%3DD.%2BS.%26aulast%3DSadhu%26aufirst%3DK.%2BM.%26atitle%3DBoronic%2520ester%2520homologation%2520with%252099%2525%2520chiral%2520selectivity%2520and%2520its%2520use%2520in%2520syntheses%2520of%2520the%2520insect%2520pheromones%2520%25283S%252C4S%2529-4-methyl-3-heptanol%2520and%2520exo-brevicomin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1983%26volume%3D105%26spage%3D2077%26epage%3D2078%26doi%3D10.1021%2Fja00345a074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matteson, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jesthi, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadhu, K. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and properties of pinanediol alpha-amido boronic esters</span>. <i>Organometallics</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1284</span>– <span class="NLM_lpage">1288</span>, <span class="refDoi"> DOI: 10.1021/om00086a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/om00086a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaL2cXltFWgtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1984&pages=1284-1288&author=D.+S.+Mattesonauthor=P.+K.+Jesthiauthor=K.+M.+Sadhu&title=Synthesis+and+properties+of+pinanediol+alpha-amido+boronic+esters&doi=10.1021%2Fom00086a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and properties of pinanediol α-amido boronic esters</span></div><div class="casAuthors">Matteson, Donald S.; Jesthi, Pradipta K.; Sadhu, Kizhakethil M.</div><div class="citationInfo"><span class="NLM_cas:title">Organometallics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1284-8</span>CODEN:
                <span class="NLM_cas:coden">ORGND7</span>;
        ISSN:<span class="NLM_cas:issn">0276-7333</span>.
    </div><div class="casAbstract">(+)-Pinanediol was used as a chiral-directing group for the synthesis of boronic acid analogs of N-acyl L-amino acids.  Thus, (+)-pinanediol boronate ester I (R = Me2CH) underwent homologation by treatment with LiCHCl2 to give boronic ester II (R1 = Cl, R2 = H), which was treated with lithiohexamethyldisilazane to give II [R1 = H, R2 = N(SiMe3)2], which was treated with AcOH/Ac2O to give boronate III (R3 = Me2CH, R4 = Me).  The later was cleaved by BCl3 to give boronic amino acid IV.  III (R3 = Me2CHCH2, Me, R4 = Me, R3 = PhCH2, R4 = OCH2Ph) were prepd. similarly.  The hydroboration of R5CH2CH:CH2 (R5 = Cl, Br, PhCH2O) with (1,2-phenyldioxy)borane gave boronic esters V (R5 = same), which underwent transesterification with (+)-pinanediol to give I [R = R5(CH2)3].  I [R = Cl(CH2)3, Br(CH2)3] were converted into III (R3 = Cl(CH2)3, Br(CH2)3, R4 = Me) by the above reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppp1-45n2y8rVg90H21EOLACvtfcHk0ljXduSiWN3F5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXltFWgtL8%253D&md5=19d9707588c4cabc895bc403cbf61711</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fom00086a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fom00086a024%26sid%3Dliteratum%253Aachs%26aulast%3DMatteson%26aufirst%3DD.%2BS.%26aulast%3DJesthi%26aufirst%3DP.%2BK.%26aulast%3DSadhu%26aufirst%3DK.%2BM.%26atitle%3DSynthesis%2520and%2520properties%2520of%2520pinanediol%2520alpha-amido%2520boronic%2520esters%26jtitle%3DOrganometallics%26date%3D1984%26volume%3D3%26spage%3D1284%26epage%3D1288%26doi%3D10.1021%2Fom00086a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matteson, D. S.</span></span> <span> </span><span class="NLM_article-title">Alpha-halo boronic esters - intermediates for stereodirected synthesis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1535</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/cr00097a009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00097a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADyaL1MXmtFCms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1989&pages=1535-1551&author=D.+S.+Matteson&title=Alpha-halo+boronic+esters+-+intermediates+for+stereodirected+synthesis&doi=10.1021%2Fcr00097a009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">α-Halo boronic esters:  intermediates for stereodirected synthesis</span></div><div class="casAuthors">Matteson, Donald S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1535-51</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with >156 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAvBFy-5DUy7Vg90H21EOLACvtfcHk0ljSFwyGGmSoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtFCms7Y%253D&md5=242a507dec19b2e3a807fd57e909393e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcr00097a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00097a009%26sid%3Dliteratum%253Aachs%26aulast%3DMatteson%26aufirst%3DD.%2BS.%26atitle%3DAlpha-halo%2520boronic%2520esters%2520-%2520intermediates%2520for%2520stereodirected%2520synthesis%26jtitle%3DChem.%2520Rev.%26date%3D1989%26volume%3D89%26spage%3D1535%26epage%3D1551%26doi%3D10.1021%2Fcr00097a009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buesking, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacauanu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis of protected alpha-amino boronic acid derivatives with an air- and moisture-stable Cu(II) catalyst</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">3671</span>– <span class="NLM_lpage">3677</span>, <span class="refDoi"> DOI: 10.1021/jo500300t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo500300t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltFGmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2014&pages=3671-3677&author=A.+W.+Bueskingauthor=V.+Bacauanuauthor=I.+Caiauthor=J.+A.+Ellman&title=Asymmetric+synthesis+of+protected+alpha-amino+boronic+acid+derivatives+with+an+air-+and+moisture-stable+Cu%28II%29+catalyst&doi=10.1021%2Fjo500300t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric synthesis of protected α-amino boronic acid derivatives with an air- and moisture-stable Cu(II) catalyst</span></div><div class="casAuthors">Buesking, Andrew W.; Bacauanu, Vlad; Cai, Irene; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3671-3677</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Both diastereomers of α-(sulfinylamino)alkylboronic acids and trifluoroborates were accessible in stereoselective addn. of bis-pinacolatodiboron to (RS)-N-(tert-butylsulfinyl)aldimines and -ketimines, catalyzed by CuSO4/L compns.  Addn. of B2pin2 to (RS)-RR1C:NSOtBu (1), catalyzed by CuSO4/PCy3·HBF4 gave α-aminoboronates RR1C(Bpin)NHSOtBu (2; R = iBu, 4-MeC6H4, 4-MeOC6H4, 4-ClC6H4, 4-CF3C6H4; R1 = H, Me) and the corresponding trifluoroborates NH4[RR1C(BF3)NHSOtBu] (3) upon fluorination.  Whereas ligand-free reaction and phosphite-modified catalyst favors (RS,S)-configuration of the diastereomers 2, CuSO4/PCy3·HBF4-catalyzed reaction gives (RS,R)-diastereomers 2 with up to 94% dr.  The asym. borylation of N-tert-butanesulfinyl imines with bis(pinacolato)diboron is achieved using a Cu(II) catalyst and provides access to synthetically useful and pharmaceutically relevant α-amino boronic acid derivs.  The Cu(II)-catalyzed reaction is performed on the benchtop in air at room temp. using com. available, inexpensive reagents at low catalyst loadings.  A variety of N-tert-butanesulfinyl imines, including ketimines, react readily to provide α-sulfinamido boronate esters in good yields and with high stereoselectivity.  In addn., this transformation is applied to the straightforward, telescoped synthesis of α-sulfinamido trifluoroborates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSIyE2LDmv87Vg90H21EOLACvtfcHk0ljSFwyGGmSoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltFGmsb4%253D&md5=f5e3ba7d40a9b907825726564cde3362</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjo500300t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo500300t%26sid%3Dliteratum%253Aachs%26aulast%3DBuesking%26aufirst%3DA.%2BW.%26aulast%3DBacauanu%26aufirst%3DV.%26aulast%3DCai%26aufirst%3DI.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DAsymmetric%2520synthesis%2520of%2520protected%2520alpha-amino%2520boronic%2520acid%2520derivatives%2520with%2520an%2520air-%2520and%2520moisture-stable%2520Cu%2528II%2529%2520catalyst%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2014%26volume%3D79%26spage%3D3671%26epage%3D3677%26doi%3D10.1021%2Fjo500300t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henneberg, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mata, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tittmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourenkov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, A.</span></span> <span> </span><span class="NLM_article-title">The inhibition mechanism of human 20S proteasomes enables next-generation inhibitor design</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>353</i></span>,  <span class="NLM_fpage">594</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1126/science.aaf8993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1126%2Fscience.aaf8993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=27493187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=353&publication_year=2016&pages=594-598&author=J.+Schraderauthor=F.+Hennebergauthor=R.+A.+Mataauthor=K.+Tittmannauthor=T.+R.+Schneiderauthor=H.+Starkauthor=G.+Bourenkovauthor=A.+Chari&title=The+inhibition+mechanism+of+human+20S+proteasomes+enables+next-generation+inhibitor+design&doi=10.1126%2Fscience.aaf8993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The inhibition mechanism of human 20S proteasomes enables next-generation inhibitor design</span></div><div class="casAuthors">Schrader, Jil; Henneberg, Fabian; Mata, Ricardo A.; Tittmann, Kai; Schneider, Thomas R.; Stark, Holger; Bourenkov, Gleb; Chari, Ashwin</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">353</span>
        (<span class="NLM_cas:issue">6299</span>),
    <span class="NLM_cas:pages">594-598</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The proteasome is a validated target for anticancer therapy, and proteasome inhibition is employed in the clinic for the treatment of tumors and hematol. malignancies.  Here, the authors describe crystal structures of the native human 20S proteasome and its complexes with inhibitors, which either are drugs approved for cancer treatment or are in clin. trials.  The structure of the native human 20S proteasome was detd. at an unprecedented resoln. of 1.8 angstroms.  Addnl., six inhibitor-proteasome complex structures were elucidated at resolns. between 1.9 and 2.1 angstroms.  Collectively, the high-resoln. structures provide new insights into the catalytic mechanisms of inhibition and necessitate a revised description of the proteasome active site.  Knowledge about inhibition mechanisms provides insights into peptide hydrolysis and can guide strategies for the development of next-generation proteasome-based cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZCuIDBbOXPLVg90H21EOLACvtfcHk0ljSFwyGGmSoKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Oks7vL&md5=8216d0007cbf06c352e06fcb05d644a9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaf8993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaf8993%26sid%3Dliteratum%253Aachs%26aulast%3DSchrader%26aufirst%3DJ.%26aulast%3DHenneberg%26aufirst%3DF.%26aulast%3DMata%26aufirst%3DR.%2BA.%26aulast%3DTittmann%26aufirst%3DK.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DBourenkov%26aufirst%3DG.%26aulast%3DChari%26aufirst%3DA.%26atitle%3DThe%2520inhibition%2520mechanism%2520of%2520human%252020S%2520proteasomes%2520enables%2520next-generation%2520inhibitor%2520design%26jtitle%3DScience%26date%3D2016%26volume%3D353%26spage%3D594%26epage%3D598%26doi%3D10.1126%2Fscience.aaf8993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkers, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploegh, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the boronic acid-based proteasome inhibitor bortezomib in complex with the yeast 20S proteasome</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">451</span>– <span class="NLM_lpage">456</span>, <span class="refDoi"> DOI: 10.1016/j.str.2005.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.str.2005.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=16531229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BD28Xit1ylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2006&pages=451-456&author=M.+Grollauthor=C.+R.+Berkersauthor=H.+L.+Ploeghauthor=H.+Ovaa&title=Crystal+structure+of+the+boronic+acid-based+proteasome+inhibitor+bortezomib+in+complex+with+the+yeast+20S+proteasome&doi=10.1016%2Fj.str.2005.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Boronic Acid-Based Proteasome Inhibitor Bortezomib in Complex with the Yeast 20S Proteasome</span></div><div class="casAuthors">Groll, Michael; Berkers, Celia R.; Ploegh, Hidde L.; Ovaa, Huib</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">451-456</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: The dipeptide boronic acid bortezomib, also termed VELCADE is a proteasome inhibitor now in use for the treatment of multiple myeloma, and its use for the treatment of other malignancies is being explored.  We detd. the crystal structure of the yeast 20S proteasome in complex with bortezomib to establish the specificity and binding mode of bortezomib to the proteasome's different catalytically active sites.  This structure should enable the rational design of new boronic acid derivs. with improved affinities and specificities for individual active subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpusGzFSuveALVg90H21EOLACvtfcHk0ljpF4JRaLjapg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xit1ylsL8%253D&md5=d5b4e46e5a9f5259cce758770302b4fe</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2005.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2005.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DBerkers%26aufirst%3DC.%2BR.%26aulast%3DPloegh%26aufirst%3DH.%2BL.%26aulast%3DOvaa%26aufirst%3DH.%26atitle%3DCrystal%2520structure%2520of%2520the%2520boronic%2520acid-based%2520proteasome%2520inhibitor%2520bortezomib%2520in%2520complex%2520with%2520the%2520yeast%252020S%2520proteasome%26jtitle%3DStructure%26date%3D2006%26volume%3D14%26spage%3D451%26epage%3D456%26doi%3D10.1016%2Fj.str.2005.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">The versatility of boron in biological target engagement</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">742</span>, <span class="refDoi"> DOI: 10.1038/nchem.2814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnchem.2814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=28754930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Gku7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=731-742&author=D.+B.+Diazauthor=A.+K.+Yudin&title=The+versatility+of+boron+in+biological+target+engagement&doi=10.1038%2Fnchem.2814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The versatility of boron in biological target engagement</span></div><div class="casAuthors">Diaz, Diego B.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">731-742</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Boron-contg. mols. have been extensively used for the purposes of chem. sensing, biol. probe development, and drug discovery.  Due to boron's empty p orbital, it can coordinate to heteroatoms such as oxygen and nitrogen.  This reversible covalent mode of interaction has led to the use of boron as bait for nucleophilic residues in disease-assocd. proteins, culminating in the approval of new therapeutics that work by covalent mechanisms.  Our anal. of a wide range of covalent inhibitors with electrophilic groups suggests that boron is a unique electrophile in its chameleonic ability to engage protein targets.  Here, we review boron's interactions with a range of protein side-chain residues and reveal that boron's properties are nuanced and arise from its uncommon coordination preferences.  These mechanistic and structural insights should serve as a guide for the development of selective boron-based bioactive mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYy4b-paw5prVg90H21EOLACvtfcHk0ljpF4JRaLjapg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Gku7bF&md5=cf5669d184a5badc7101aa2e3d1f2500</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2814%26sid%3Dliteratum%253Aachs%26aulast%3DDiaz%26aufirst%3DD.%2BB.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DThe%2520versatility%2520of%2520boron%2520in%2520biological%2520target%2520engagement%26jtitle%3DNat.%2520Chem.%26date%3D2017%26volume%3D9%26spage%3D731%26epage%3D742%26doi%3D10.1038%2Fnchem.2814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollé, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: a compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&author=E.+R.+McDonaldauthor=A.+de+Weckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cort%C3%A9s-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Moll%C3%A9author=R.+A.+Pagliariniauthor=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+a+compendium+of+cancer+dependencies+and+synthetic+lethal+relationships+uncovered+by+large-scale%2C+deep+RNAi+screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0lhjnzm2tqkoIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCort%25C3%25A9s-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMoll%25C3%25A9%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520a%2520compendium%2520of%2520cancer%2520dependencies%2520and%2520synthetic%2520lethal%2520relationships%2520uncovered%2520by%2520large-scale%252C%2520deep%2520RNAi%2520screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caponigro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehár, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonkin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monahan, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morais, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meltzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korejwa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jané-Valbuena, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapa, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bric-Furlong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipway, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspesi, P.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Silva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagtap, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palescandolo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sougnez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onofrio, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liefeld, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacConaill, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winckler, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesirov, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">The cancer cell line encyclopedia enables predictive modelling of anticancer drug sensitivity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">607</span>, <span class="refDoi"> DOI: 10.1038/nature11003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=10.1038%2Fnature11003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=22460905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=603-607&author=J.+Barretinaauthor=G.+Caponigroauthor=N.+Stranskyauthor=K.+Venkatesanauthor=A.+A.+Margolinauthor=S.+Kimauthor=C.+J.+Wilsonauthor=J.+Leh%C3%A1rauthor=G.+V.+Kryukovauthor=D.+Sonkinauthor=A.+Reddyauthor=M.+Liuauthor=L.+Murrayauthor=M.+F.+Bergerauthor=J.+E.+Monahanauthor=P.+Moraisauthor=J.+Meltzerauthor=A.+Korejwaauthor=J.+Jan%C3%A9-Valbuenaauthor=F.+A.+Mapaauthor=J.+Thibaultauthor=E.+Bric-Furlongauthor=P.+Ramanauthor=A.+Shipwayauthor=I.+H.+Engelsauthor=J.+Chengauthor=G.+K.+Yuauthor=J.+Yuauthor=P.+Aspesiauthor=M.+de+Silvaauthor=K.+Jagtapauthor=M.+D.+Jonesauthor=L.+Wangauthor=C.+Hattonauthor=E.+Palescandoloauthor=S.+Guptaauthor=S.+Mahanauthor=C.+Sougnezauthor=R.+C.+Onofrioauthor=T.+Liefeldauthor=L.+MacConaillauthor=W.+Wincklerauthor=M.+Reichauthor=N.+Liauthor=J.+P.+Mesirovauthor=S.+B.+Gabrielauthor=G.+Getzauthor=K.+Ardlieauthor=V.+Chanauthor=V.+E.+Myerauthor=B.+L.+Weberauthor=J.+Porterauthor=M.+Warmuthauthor=P.+Finanauthor=J.+L.+Harrisauthor=M.+Meyersonauthor=T.+R.+Golubauthor=M.+P.+Morrisseyauthor=W.+R.+Sellersauthor=R.+Schlegelauthor=L.+A.+Garraway&title=The+cancer+cell+line+encyclopedia+enables+predictive+modelling+of+anticancer+drug+sensitivity&doi=10.1038%2Fnature11003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity</span></div><div class="casAuthors">Barretina, Jordi; Caponigro, Giordano; Stransky, Nicolas; Venkatesan, Kavitha; Margolin, Adam A.; Kim, Sungjoon; Wilson, Christopher J.; Lehar, Joseph; Kryukov, Gregory V.; Sonkin, Dmitriy; Reddy, Anupama; Liu, Manway; Murray, Lauren; Berger, Michael F.; Monahan, John E.; Morais, Paula; Meltzer, Jodi; Korejwa, Adam; Jane-Valbuena, Judit; Mapa, Felipa A.; Thibault, Joseph; Bric-Furlong, Eva; Raman, Pichai; Shipway, Aaron; Engels, Ingo H.; Cheng, Jill; Yu, Guoying K.; Yu, Jianjun; Aspesi, Peter; de Silva, Melanie; Jagtap, Kalpana; Jones, Michael D.; Wang, Li; Hatton, Charles; Palescandolo, Emanuele; Gupta, Supriya; Mahan, Scott; Sougnez, Carrie; Onofrio, Robert C.; Liefeld, Ted; MacConaill, Laura; Winckler, Wendy; Reich, Michael; Li, Nanxin; Mesirov, Jill P.; Gabriel, Stacey B.; Getz, Gad; Ardlie, Kristin; Chan, Vivien; Myer, Vic E.; Weber, Barbara L.; Porter, Jeff; Warmuth, Markus; Finan, Peter; Harris, Jennifer L.; Meyerson, Matthew; Golub, Todd R.; Morrissey, Michael P.; Sellers, William R.; Schlegel, Robert; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">603-607</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The systematic translation of cancer genomic data into knowledge of tumor biol. and therapeutic possibilities remains challenging.  Such efforts should be greatly aided by robust preclin. model systems that reflect the genomic diversity of human cancers and for which detailed genetic and pharmacol. annotation is available.  Here we describe the Cancer Cell Line Encyclopedia (CCLE): a compilation of gene expression, chromosomal copy no. and massively parallel sequencing data from 947 human cancer cell lines.  When coupled with pharmacol. profiles for 24 anticancer drugs across 479 of the cell lines, this collection allowed identification of genetic, lineage, and gene-expression-based predictors of drug sensitivity.  In addn. to known predictors, we found that plasma cell lineage correlated with sensitivity to IGF1 receptor inhibitors; AHR expression was assocd. with MEK inhibitor efficacy in NRAS-mutant lines; and SLFN11 expression predicted sensitivity to topoisomerase inhibitors.  Together, our results indicate that large, annotated cell-line collections may help to enable preclin. stratification schemata for anticancer agents.  The generation of genetic predictions of drug response in the preclin. setting and their incorporation into cancer clin. trial design could speed the emergence of personalized' therapeutic regimens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHUeZ7mtYpwLVg90H21EOLACvtfcHk0liQTg_VQZZFuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D&md5=4aefaa8c7abd1e30c35c4eeca811687b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature11003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11003%26sid%3Dliteratum%253Aachs%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DCaponigro%26aufirst%3DG.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DMargolin%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DLeh%25C3%25A1r%26aufirst%3DJ.%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DSonkin%26aufirst%3DD.%26aulast%3DReddy%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DMonahan%26aufirst%3DJ.%2BE.%26aulast%3DMorais%26aufirst%3DP.%26aulast%3DMeltzer%26aufirst%3DJ.%26aulast%3DKorejwa%26aufirst%3DA.%26aulast%3DJan%25C3%25A9-Valbuena%26aufirst%3DJ.%26aulast%3DMapa%26aufirst%3DF.%2BA.%26aulast%3DThibault%26aufirst%3DJ.%26aulast%3DBric-Furlong%26aufirst%3DE.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DShipway%26aufirst%3DA.%26aulast%3DEngels%26aufirst%3DI.%2BH.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DG.%2BK.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DAspesi%26aufirst%3DP.%26aulast%3Dde%2BSilva%26aufirst%3DM.%26aulast%3DJagtap%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPalescandolo%26aufirst%3DE.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DMahan%26aufirst%3DS.%26aulast%3DSougnez%26aufirst%3DC.%26aulast%3DOnofrio%26aufirst%3DR.%2BC.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DMacConaill%26aufirst%3DL.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DGabriel%26aufirst%3DS.%2BB.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DV.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DWeber%26aufirst%3DB.%2BL.%26aulast%3DPorter%26aufirst%3DJ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DFinan%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DMorrissey%26aufirst%3DM.%2BP.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DThe%2520cancer%2520cell%2520line%2520encyclopedia%2520enables%2520predictive%2520modelling%2520of%2520anticancer%2520drug%2520sensitivity%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D603%26epage%3D607%26doi%3D10.1038%2Fnature11003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27','PDB','6X27'); return false;">PDB: 6X27</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF3','PDB','5LF3'); return false;">PDB: 5LF3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV','PDB','6WZV'); return false;">PDB: 6WZV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS','PDB','6WYS'); return false;">PDB: 6WYS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LE5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LE5','PDB','5LE5'); return false;">PDB: 5LE5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M','PDB','6X1M'); return false;">PDB: 6X1M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LE3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LE3','PDB','5LE3'); return false;">PDB: 5LE3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LF5','PDB','5LF5'); return false;">PDB: 5LF5</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02152">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78358"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02152?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02152</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">LONP1 biochemical characterization, MD simulations, binding site dynamics of both LONP1 and 20S proteasome subunits, cell PD experiments, docking models, crystallographic conditions, and HPLC traces of <b>9d</b>, <b>12d</b>, and <b>14</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf">PDF</a>)</p></li><li><p class="inline"><b>9a</b> Docking model in LONP1 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_002.pdb">PDB</a>)</p></li><li><p class="inline"><b>9a</b> Docking model in the β5-subunit of the 20S proteasome (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_003.pdb">PDB</a>)</p></li><li><p class="inline"><b>11d</b> Docking model in the expanded pocket of the β5-subunit of 20S proteasome (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_004.pdb">PDB</a>)</p></li><li><p class="inline"><b>12d</b> Docking model in the expanded pocket of the β5-subunit of the 20S proteasome (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_006.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_001.pdf">jm0c02152_si_001.pdf (2.93 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_002.pdb">jm0c02152_si_002.pdb (230.89 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_003.pdb">jm0c02152_si_003.pdb (248.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_004.pdb">jm0c02152_si_004.pdb (126.57 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_005.pdb">jm0c02152_si_005.pdb (126.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02152/suppl_file/jm0c02152_si_006.csv">jm0c02152_si_006.csv (2.15 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The authors will release the atomic coordinates and experimental data upon article publication. The crystallographic coordinates and structure factors have been deposited in the Protein Data Bank, <a href="http://www.pdb.org" class="extLink">www.pdb.org</a> [PDB ID codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WYS">6WYS</a> (apo LONP1), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X27">6X27</a> (bortezomib co-structure with LONP1), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WZV">6WZV</a> (<b>9a</b> co-structure with LONP1), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6X1M">6X1M</a> (<b>12d</b> co-structure with LONP1)].</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02152&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02152%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-8%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02152" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799815368763c70","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
