
<html lang="en"     class="pb-page"  data-request-id="69d088f4-57af-43e1-91e7-f48f2bd3210a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.1c00103;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Highly Potent and Selective IRAK1 Degraders to Probe Scaffolding Functions of IRAK1 in ABC DLBCL" /></meta><meta name="dc.Creator" content="Liqiang  Fu" /></meta><meta name="dc.Creator" content="Jing  Zhang" /></meta><meta name="dc.Creator" content="Bin  Shen" /></meta><meta name="dc.Creator" content="Linglong  Kong" /></meta><meta name="dc.Creator" content="Yingtao  Liu" /></meta><meta name="dc.Creator" content="Wangyang  Tu" /></meta><meta name="dc.Creator" content="Wenqian  Wang" /></meta><meta name="dc.Creator" content="Xin  Cai" /></meta><meta name="dc.Creator" content="Xiaotao  Wang" /></meta><meta name="dc.Creator" content="Na  Cheng" /></meta><meta name="dc.Creator" content="Mingxuan  Xia" /></meta><meta name="dc.Creator" content="Tianyuan  Zhou" /></meta><meta name="dc.Creator" content="Qian  Liu" /></meta><meta name="dc.Creator" content="Yanping  Xu" /></meta><meta name="dc.Creator" content="Jennifer  Yang" /></meta><meta name="dc.Creator" content="Paul  Gavine" /></meta><meta name="dc.Creator" content="Ulrike  Philippar" /></meta><meta name="dc.Creator" content="Ricardo  Attar" /></meta><meta name="dc.Creator" content="James P.  Edwards" /></meta><meta name="dc.Creator" content="Jennifer D.  Venable" /></meta><meta name="dc.Creator" content="Xuedong  Dai" /></meta><meta name="dc.Description" content="MyD88 gene mutation has been identified as one of the most prevalent driver mutations in the activated B-cell-like diffuse large B-cell lymphoma (ABC DLBCL). The published literature suggests that ..." /></meta><meta name="Description" content="MyD88 gene mutation has been identified as one of the most prevalent driver mutations in the activated B-cell-like diffuse large B-cell lymphoma (ABC DLBCL). The published literature suggests that ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 19, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00103" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00103" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00103" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00103" /></link>
        
    
    

<title>Discovery of Highly Potent and Selective IRAK1 Degraders to Probe Scaffolding Functions of IRAK1 in ABC DLBCL | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00103" /></meta><meta property="og:title" content="Discovery of Highly Potent and Selective IRAK1 Degraders to Probe Scaffolding Functions of IRAK1 in ABC DLBCL" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0011.jpeg" /></meta><meta property="og:description" content="MyD88 gene mutation has been identified as one of the most prevalent driver mutations in the activated B-cell-like diffuse large B-cell lymphoma (ABC DLBCL). The published literature suggests that interleukin-1 receptor-associated kinase 1 (IRAK1) is an essential gene for ABC DLBCL harboring MyD88 mutation. Importantly, the scaffolding function of IRAK1, rather than its kinase activity, is required for tumor cell survival. Herein, we present our design, synthesis, and biological evaluation of a novel series of potent and selective IRAK1 degraders. One of the most potent compounds, Degrader-3 (JNJ-1013), effectively degraded cellular IRAK1 protein with a DC50 of 3 nM in HBL-1 cells. Furthermore, JNJ-1013 potently inhibited IRAK1 downstream signaling pathways and demonstrated strong anti-proliferative effects in ABC DLBCL cells with MyD88 mutation. This work suggests that IRAK1 degraders have the potential for treating cancers that are dependent on the IRAK1 scaffolding function." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00103"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00103">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00103&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00103&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00103&amp;href=/doi/10.1021/acs.jmedchem.1c00103" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, XXXX</span><span class="cit-fg-issue">, XXX</span><span class="cit-fg-pageRange">, XXX-XXX</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/0/0" title="Return to Articles ASAP" id="returnToASAP" class="content-navigation__btn--return"><span>RETURN TO ARTICLES ASAP</span></a><a href="/doi/10.1021/acs.jmedchem.1c00805" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00367" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Highly Potent and Selective IRAK1 Degraders to Probe Scaffolding Functions of IRAK1 in ABC DLBCL</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Liqiang Fu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liqiang Fu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Chemistry, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liqiang++Fu">Liqiang Fu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jing Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Zhang">Jing Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bin Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bin Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bin++Shen">Bin Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Linglong Kong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Linglong Kong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Chemistry, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Linglong++Kong">Linglong Kong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yingtao Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yingtao Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Chemistry, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yingtao++Liu">Yingtao Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wangyang Tu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wangyang Tu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Chemistry, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wangyang++Tu">Wangyang Tu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wenqian Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenqian Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenqian++Wang">Wenqian Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Cai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Cai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomarker, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Cai">Xin Cai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaotao Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaotao Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomarker, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaotao++Wang">Xiaotao Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Na Cheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Na Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Na++Cheng">Na Cheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mingxuan Xia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingxuan Xia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingxuan++Xia">Mingxuan Xia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tianyuan Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tianyuan Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tianyuan++Zhou">Tianyuan Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qian Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qian Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Chemistry, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qian++Liu">Qian Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanping Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanping Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Chemistry, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanping++Xu">Yanping Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer++Yang">Jennifer Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Gavine</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Gavine</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Janssen Research & Development, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Gavine">Paul Gavine</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulrike Philippar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulrike Philippar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Janssen Pharmaceutical Research & Development, Beerse 2340, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulrike++Philippar">Ulrike Philippar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ricardo Attar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ricardo Attar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Janssen Pharmaceutical Research & Development, Spring House, Pennsylvania 19477, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ricardo++Attar">Ricardo Attar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James P. Edwards</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James P. Edwards</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, Janssen Research & Development, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+P.++Edwards">James P. Edwards</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jennifer D. Venable</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jennifer D. Venable</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, Janssen Research & Development, San Diego, California 92121, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#5a302c3f343b38363f1a332e297430343074393537"><span class="__cf_email__" data-cfemail="19736f7c77787b757c59706d6a37737773377a7674">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jennifer+D.++Venable">Jennifer D. Venable</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Xuedong Dai</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xuedong Dai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, Janssen (China) Research & Development, Shanghai 201210, P.R. China</div><div class="loa-info-affiliations-info">Discovery Chemistry, Janssen Research & Development, Shanghai 201210, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#49312d2820787109203d3a67232723672a2624"><span class="__cf_email__" data-cfemail="89f1ede8e0b8b1c9e0fdfaa7e3e7e3a7eae6e4">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xuedong++Dai">Xuedong Dai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4358-0319" title="Orcid link">https://orcid.org/0000-0003-4358-0319</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00103&amp;href=/doi/10.1021%2Facs.jmedchem.1c00103" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, XXXX</span><span class="cit-issue">, XXX</span><span class="cit-pageRange">, XXX-XXX</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 19, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>20 January 2021</li><li><span class="item_label"><b>Published</b> online</span>19 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00103" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00103</a></div><div class="article_header-article-copyright"><strong>© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLiqiang%2BFu%252C%2BJing%2BZhang%252C%2BBin%2BShen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D0%26issueNum%3D0%26contentID%3Dacs.jmedchem.1c00103%26title%3DDiscovery%2Bof%2BHighly%2BPotent%2Band%2BSelective%2BIRAK1%2BDegraders%2Bto%2BProbe%2BScaffolding%2BFunctions%2Bof%2BIRAK1%2Bin%2BABC%2BDLBCL%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D12%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00103"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1159</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00103" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Highly Potent and Selective IRAK1 Degraders to Probe Scaffolding Functions of IRAK1 in ABC DLBCL&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Liqiang&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Bin&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Linglong&quot;,&quot;last_name&quot;:&quot;Kong&quot;},{&quot;first_name&quot;:&quot;Yingtao&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Wangyang&quot;,&quot;last_name&quot;:&quot;Tu&quot;},{&quot;first_name&quot;:&quot;Wenqian&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Cai&quot;},{&quot;first_name&quot;:&quot;Xiaotao&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Na&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Mingxuan&quot;,&quot;last_name&quot;:&quot;Xia&quot;},{&quot;first_name&quot;:&quot;Tianyuan&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Qian&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yanping&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Gavine&quot;},{&quot;first_name&quot;:&quot;Ulrike&quot;,&quot;last_name&quot;:&quot;Philippar&quot;},{&quot;first_name&quot;:&quot;Ricardo&quot;,&quot;last_name&quot;:&quot;Attar&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;P. Edwards&quot;},{&quot;first_name&quot;:&quot;Jennifer&quot;,&quot;last_name&quot;:&quot;D. Venable&quot;},{&quot;first_name&quot;:&quot;Xuedong&quot;,&quot;last_name&quot;:&quot;Dai&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;0&quot;,&quot;volume&quot;:&quot;0&quot;,&quot;pages&quot;:&quot;&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00103&quot;},&quot;abstract&quot;:&quot;MyD88 gene mutation has been identified as one of the most prevalent driver mutations in the activated B-cell-like diffuse large B-cell lymphoma (ABC DLBCL). The published literature suggests that interleukin-1 receptor-associated kinase 1 (IRAK1) is an essential gene for ABC DLBCL harboring MyD88 mutation. Importantly, the scaffolding function of IRAK1, rather than its kinase activity, is required for tumor cell survival. Herein, we present our design, synthesis, and biological evaluation of a novel series of potent and selective IRAK1 degraders. One of the most potent compounds, Degrader-3 (JNJ-1013), effectively degraded cellular IRAK1 protein with a DC50 of 3 nM in HBL-1 cells. Furthermore, JNJ-1013 potently inhibited IRAK1 downstream signaling pathways and demonstrated strong anti-proliferative effects in ABC DLBCL cells with MyD88 mutation. This work suggests that IRAK1 degraders have the potential for treating cancers that are dependent on the IRAK1 scaffolding function.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00103&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00103" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00103&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00103" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00103&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00103" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00103&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00103&amp;href=/doi/10.1021/acs.jmedchem.1c00103" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00103" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00103" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00103%26sid%3Dliteratum%253Aachs%26pmid%3D34279092%26genre%3Darticle%26aulast%3DFu%26date%3D2021%26atitle%3DDiscovery%2Bof%2BHighly%2BPotent%2Band%2BSelective%2BIRAK1%2BDegraders%2Bto%2BProbe%2BScaffolding%2BFunctions%2Bof%2BIRAK1%2Bin%2BABC%2BDLBCL%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D0%26issue%3D0%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=291297" title="Cell signaling">Cell signaling</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/0/0" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">MyD88 gene mutation has been identified as one of the most prevalent driver mutations in the activated B-cell-like diffuse large B-cell lymphoma (ABC DLBCL). The published literature suggests that interleukin-1 receptor-associated kinase 1 (IRAK1) is an essential gene for ABC DLBCL harboring MyD88 mutation. Importantly, the scaffolding function of IRAK1, rather than its kinase activity, is required for tumor cell survival. Herein, we present our design, synthesis, and biological evaluation of a novel series of potent and selective IRAK1 degraders. One of the most potent compounds, <b>Degrader-3</b> (<b>JNJ-1013</b>), effectively degraded cellular IRAK1 protein with a DC<sub>50</sub> of 3 nM in HBL-1 cells. Furthermore, <b>JNJ-1013</b> potently inhibited IRAK1 downstream signaling pathways and demonstrated strong anti-proliferative effects in ABC DLBCL cells with MyD88 mutation. This work suggests that IRAK1 degraders have the potential for treating cancers that are dependent on the IRAK1 scaffolding function.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26335" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26335" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Interleukin-1 receptor-activated kinases (IRAK1, IRAK2, IRAK3, and IRAK4) are serine/threonine kinases that mediate innate immune and inflammatory responses. IRAKs play important roles in transducing inflammatory signals downstream of the toll-like receptors (TLRs) and interleukin-1 receptors (IL-1Rs).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> When activated, TLRs or IL-1Rs dimerize and recruit the myeloid differentiation primary response 88 (MyD88) adaptor protein through their intracellular domains. MyD88 further recruits IRAK4 and IRA1/2 to form a multiprotein complex, namely, the Myddosome. IRAK4 phosphorylates IRAK1 to first prime its activity, leading to further activation, hyperphosphorylation, and dissociation from the Myddosome and subsequent interaction with the E3 ubiquitin ligase, TNF receptor-associated factor 6 (TRAF6). The activated TRAF6 complex in turn drives downstream events, including the IκB kinase (IKK)-nuclear factor-κB (NF-κB) and mitogen-activated protein kinase (MAPK) signaling pathways that ultimately result in the upregulation of proinflammatory cytokines.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Diffuse large B-cell lymphoma (DLBCL) is the most common aggressive non-Hodgkin lymphoma (NHL), accounting for about 30–35% of NHL cases around the world. DLBCL can be divided into germinal center B-cell-like (GCB) and activated B-cell-like (ABC) subtypes.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> ABC DLBCL is associated with poor prognosis compared to the GCB subtype. MyD88 gain-of-function mutation frequently appears in ABC DLBCL, resulting in constitutive activation of the IRAK4- and IRAK1-mediated signaling.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, in our previous efforts to inhibit the MyD88 pathway, neither a selective IRAK4 kinase inhibitor nor a new series of potent and selective IRAK4 degraders that dismantle the IRAK4 scaffolding function demonstrated noticeable anti-proliferation effects. These results suggest that IRAK4 is dispensable for ABC DLBCL cell survival.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> On the other hand, genome-wide CRISPR screens pointed to IRAK1 as an essential gene for the survival of ABC DLBCL cells harboring MyD88 hotspot mutation.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> In a recent publication, Gray’s group demonstrated the anti-proliferative activity of an IRAK1 covalent inhibitor in ABC DLBCL cell lines.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Interestingly, this IRAK1 kinase inhibitor <b>JH-X-119-01</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which is very potent and selective in enzymatic assays (IRAK1 IC<sub>50</sub> of 8 nM), only modestly inhibited the proliferation of ABC DLBCL cell lines with IC<sub>50</sub> values in the micromolar range.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> These results suggested that kinase inhibition alone was not sufficient to fully disrupt myddosome signaling and effectively suppress lymphoma cell growth. We therefore hypothesize that the kinase-independent activities of IRAK1 are engaged and that the removal of the protein scaffold functions may be required to achieve maximal antagonism of the signaling pathway in MyD88 L265P mutant DLBCL cells.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of the IRAK1 kinase inhibitors and the co-crystal structure of <b>JH-I-25</b> (green) with IRAK1 kinase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BFN">6BFN</a>). The linker attachment point is highlighted by the white arrow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Here, we sought to eliminate the IRAK1 scaffolding function using bivalent chemical degraders. Degraders of this type, often represented as proteolysis targeting chimeras (PROTACs), are heterobifunctional small molecules that can induce degradation of a target protein by bringing it into proximity with an E3 ligase. When a ternary complex is formed, the E3 ligase ubiquitinates the target protein, leading to its proteasomal degradation.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a> In this study, we report the design, synthesis, and biological evaluation of a series of IRAK1-targeting degraders. Several of these heterobifunctional molecules could efficiently and selectively degrade IRAK1 protein at low concentrations, providing new pharmacological tools to probe for the IRAK1 scaffolding function. It was demonstrated that IRAK1 PROTACs effectively inhibited IRAK1 downstream signaling pathways and led to strong growth inhibition of MyD88-dependent ABC DLBCL cells.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85882" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85882" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A set of published IRAK1 inhibitors were chosen as a starting point to develop IRAK1 PROTACs. Compound <b>JH-I-25</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) was identified as a potent IRAK1/4 dual inhibitor with the first IRAK1 co-crystal structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BFN">6BFN</a>) reported recently.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> This compound was also investigated to understand and verify its IRAK1 kinase inhibition activity. The crystal structure shows that the morpholine ring is in a solvent-exposed region, indicating a suitable site for linker attachment. Therefore, morpholine was replaced with piperazine to facilitate linker conjugation <i>via</i> amine alkylation (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><div class="NLM_p">For E3 ligase ligand selection, cereblon (CRBN) ligand thalidomide derivatives and the von Hippel–Lindau (VHL) ligand VH032 are the most frequently used ligands in PROTAC design.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Previous studies indicated that thalidomide derivatives could potentially induce degradation of multiple neo-substrates, such as Ikaros (IKZF1), Aiolos (IKZF3), GSPT1, SALL4, and p63.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18">(15−18)</a> Thalidomide derivatives that induce degradation of Ikaros and Aiolos could potentially lead to strong proliferation inhibition and apoptosis in ABC DLBCL cell lines, confounding the interpretation of IRAK1-dependent effects.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> To avoid potential off-target degradation from the CRBN E3 ligase and enable conclusive data for IRAK1 degradation validation, the VHL E3 ligase ligand VH032 was selected for IRAK1 PROTAC design and connected to the anchor point of the piperazine ring moiety from the <b>JH-1-25</b> IRAK1 inhibitor <i>via</i> different lengths and composition linkers (<i>e.g</i>., alkyl linkers and polyethylene glycol linkers, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of IRAK1-targeting PROTACs and the chemical structure of <b>Degrader-1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To generate the degrader compounds, a convergent synthesis method (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) was developed to connect the piperazine group of the IRAK1 warhead with different linkers and a VHL amine intermediate (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> After library synthesis, the degradation potency (<i>D</i><sub>max</sub> and DC<sub>50</sub>) of these PROTAC degraders was assessed by immunoblotting assays in HEK293T cells after 24 h of treatment. Interestingly, <b>Degrader-1</b> with the shortest linker (two methylene units) showed the most promising profile, demonstrating a DC<sub>50</sub> of 134 nM and a <i>D</i><sub>max</sub> of 88% at 10 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). We also profiled the <i>in vitro</i> biochemical properties of both <b>Degrader-1</b> and <b>JH-I-25</b>. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> summarizes the biochemical activity, determined by a Caliper kinase activity assay, of the designed PROTACs and the parent ligand to IRAK1 and IRAK4. The degrader molecule maintained a high-double-digit nM IRAK1 biochemical potency that was comparable to that of its parent compound <b>JH-I-25</b>, confirming the success of the rationally designed attachment vector. In an IRAK4 biochemical assay, however, the potency of compound <b>Degrader-1</b> dropped 15-fold, presumably because the front pocket of IRAK4 is largely occluded, while the front pocket of IRAK1 is open and accessible for binding.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Interestingly, no IRAK4 protein degradation was observed with <b>Degrader-1</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf" class="ext-link">Figure S4</a>) even though it exhibited reasonable IRAK4 biochemical potency with an IC<sub>50</sub> of 368 nM. Despite the high sequence similarity (>90%) of IRAK1 and IRAK4 in their ATP binding pockets, significant degradation selectivity for IRAK1 was achieved through heterobifunctional degraders over functional inhibition selectivity by <b>JH-I-25</b> and its degrader analogues. This suggests that the basis of degradation selectivity may reside in protein–protein interactions between the VHL E3 ubiquitin ligase and the targeted protein, that is, the ternary complex.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In other words, the engaged proteins may not lead to a productive ternary complex in the case of IRAK4. Independent experiments are in progress to confirm this hypothesis.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Degrader-1<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1-(tetrahydro-2<i>H</i>-pyran-2-yl)-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-pyrazole, Pd(dppf)Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O. (b) (i) NaOH, MeOH/H<sub>2</sub>O. (ii) HCl, dioxane. (c) <i>tert</i>-Butyl piperazine-1-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMF. (d) H<sub>2</sub>, Pd/C, MeOH. (e) HATU, TEA, DMF. (f) HCl, EtOAc. (g) Methyl 3-bromopropanoate, TEA, KI, NMP, microwave. (h) NaOH, THF/H<sub>2</sub>O. (i) EDCI, HOBt, DIPEA, DMF.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical and Protein Degradation Data of <b>Degrader-1</b> and Warhead Compound <b>JH-I-25</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compounds</th><th class="colsep0 rowsep0" align="center" char=".">IRAK1 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">IRAK4 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IRAK1 <i>D</i><sub>max</sub>(% of control)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">IRAK1 DC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>Degrader-1</b></td><td class="colsep0 rowsep0" align="char" char=".">80</td><td class="colsep0 rowsep0" align="char" char=".">368</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">134</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>JH-I-25</b></td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IRAK1 kinase inhibition potency (IC<sub>50</sub>) determined by a Caliper biochemical assay.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Maximal IRAK1 and IRAK4 protein degradation (<i>D</i><sub>max</sub>) determined by immunoblotting 24 h after treatment. IRAK1 degradation was calculated from quantified Western blots.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Degradation potency (DC<sub>50</sub>) of IRAK1 PROTACs determined by immunoblotting 24 h after treatment with degraders in the 293T cell line.</p></div></div></div><div class="NLM_p">To evaluate the kinetics of IRAK1 degradation, we performed time course experiments in HEK293T cells by Western blot analysis using different concentrations of <b>Degrader-1</b>. Degradation occurred rapidly, with over 50% IRAK1 degradation achieved in 2 h across all ≥ 1 μM doses, and over 90% of protein degradation was achieved after 8 h (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). At 10 μM, reduced degradation of the IRAK1 protein was observed, which is consistent with the “hook effect” commonly observed at high concentrations of PROTACs where individual binary complexes become saturated and compete for the formation of the E3 ligase–substrate–ligand ternary complex, causing the loss of degradation efficiency.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IRAK1 protein degradation profile in HEK293T cells for <b>Degrader-1</b>, <b>Degrader-2</b>, and <b>Degrader-3</b>. (A) Degradation effects of <b>Degrader-1</b> on IRAK1 proteins in HEK293T cells by Western blotting after 24 h treatment by <b>Degrader-1</b> at illustrated concentrations. (B) Characterization of <b>Degrader-1</b> for its concentration- and time-dependent IRAK1 protein degradation by Western blot analysis. (C) <b>Degrader-2</b> and <b>Degrader-3</b> dose-dependently degrades IRAK1 protein after 24 h incubation. (D) Quantitation of DC<sub>50</sub> values of <b>Degrader-1</b>, <b>Degrader-2</b>, and <b>Degrader-3</b> after 24 h incubation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To our delight, through the replacement of the <i>N</i>-linked piperazine ring in the linker region of <b>Degrader-1</b> with an O-linked piperidine version (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), the DC<sub>50</sub> of <b>Degrader-2</b> was improved to a low-double-digit nM range with a <i>D</i><sub>max</sub> of 95% (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Meanwhile, the addition of a methyl group in the <i>S</i>-configuration in the VHL binder moiety of <b>Degrader-2</b> yielded a 5-fold more potent compound, <b>Degrader-3</b> (<b>JNJ-1013</b>, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Both PROTACs effectively degraded IRAK1 protein in a dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D). Interestingly, in a direct comparison, <b>Degrader-1</b>, <b>Degrader-2</b>, and <b>Degrader-3</b> did not exhibit distinct potency differences in IRAK1 biochemical activity or in the VHL fluorescence polarization (FP) binding assay (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This again indicates that there is no significant correlation between the binary binding potency and degradation potency. In addition, <b>Degrader-4</b>, the diastereoisomer of <b>Degrader-3</b> with an inverted hydroxyl stereocenter, was designed as a reference. As expected, <b>Degrader-4</b> lost the VHL binding affinity (IC<sub>50</sub> > 10 μM in a VHL FP assay) and did not show significant IRAK1 degradation activity (DC<sub>50</sub> > 10 μM).</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Degrader-2, Degrader-3, and Degrader-4<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl 2-bromoacetate, NaH, THF; (b) Pd/C, H<sub>2</sub> (30 psi), MeOH; (c) HATU, DIPEA, DMF; (d) HCl (4 M in dioxane); and (e) EDCI, HOBt, DIPEA, DMF.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Profiles of IRAK1 PROTACs with Different Linker Lengths and Compositions</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">IRAK1 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">IRAK4 IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">VHL FP IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">IRAK1 <i>D</i><sub>max</sub> (% of control)<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">IRAK1 DC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>Degrader-1</b></td><td class="colsep0 rowsep0" align="char" char=".">80</td><td class="colsep0 rowsep0" align="char" char=".">368</td><td class="colsep0 rowsep0" align="left">1646</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">134</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>Degrader-2</b></td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">260</td><td class="colsep0 rowsep0" align="left">1400</td><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>Degrader-3 (JNJ-1013)</b></td><td class="colsep0 rowsep0" align="char" char=".">72</td><td class="colsep0 rowsep0" align="char" char=".">443</td><td class="colsep0 rowsep0" align="left">1071</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">3.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>Degrader-4</b></td><td class="colsep0 rowsep0" align="char" char=".">79</td><td class="colsep0 rowsep0" align="char" char=".">320</td><td class="colsep0 rowsep0" align="left">>10,000</td><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">>10,000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>JH-I-25</b></td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">NA</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IRAK1 kinase inhibition potency (IC<sub>50</sub>) determined by a Caliper biochemistry assay.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Binding affinity of degraders to VHL protein (IC<sub>50</sub>) determined by an FP assay.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Maximal IRAK1 and IRAK4 protein degradation (<i>D</i><sub>max</sub>) determined by immunoblotting 24 h after treatment with degraders.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Degradation potency (DC<sub>50</sub>) of IRAK1 degraders determined by immunoblotting 24 h after treatment.</p></div></div></div><div class="NLM_p">Next, <b>Degrader-3</b> was selected as a representative IRAK1 PROTAC for further evaluation. To verify that the degradation of IRAK1 protein was mediated through the ubiquitination and proteasome system, we utilized a proteasome inhibitor, MG-132, to disrupt proteasome function and a VHL ligand, VH298, to compete for VHL binding.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B, IRAK1 degradation was completely prevented by either the proteasome inhibitor MG-132 or competitive VHL ligand VH298, confirming that the degradation of IRAK1 protein is through the proteasome mediated by the VHL E3 ubiquitin ligase.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mechanistic study of <b>Degrader-3</b> in HEK293T cells by Western blotting analysis. (A) <b>Degrader-3</b>-mediated IRAK1 degradation is abrogated by pretreatment with the proteasome inhibitor MG-132. (B) <b>Degrader-3</b>-mediated IRAK1 degradation is abrogated by pretreatment with the VHL ligand VH298.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To compare biochemical selectivity profiles, we submitted <b>JH-I-25</b>, <b>Degrader-3</b>, and <b>Degrader-4</b> for the KinomeScan competition binding assay panel (DiscoverX), which comprises 469 kinases. <b>JH-I-25</b> is a relatively selective IRAK1 inhibitor as a limited number of off-targets were inhibited at 1 μM, as shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. Both <b>Degrader-3</b> and <b>Degrader-4</b> degraders demonstrated higher selectivity toward IRAK1 compared to <b>JH-I-25</b> as fewer off-targets were inhibited in the KinomeScan assay. Conversion of <b>JH-I-25</b> into a PROTAC retained the IRAK1 binding but significantly reduced the binding affinity for the other members of the kinase superfamily that had been inhibited by <b>JH-I-25</b>. Compared to the 17 off-targets with more than 90% inhibition at 1 μM from <b>JH-I-25</b>, only 3 off-targets were identified for <b>Degrader-3</b> (BIKE, DRAK2, and YSK4). It is worth mentioning that the VHL epimer control compound <b>Degrader-4</b> showed an almost identical kinome profile to that of <b>Degrader-3</b>.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. TREEspot interaction map of kinases for <b>JH-I-25</b>, <b>Degrader-3</b>, and <b>Degrader-4</b> from the KinomeScan competition binding assay profiled at 1 μM concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To determine the global proteome-wide degradation profiles of the IRAK1 degraders, an unbiased quantitative mass spectrometry-based approach was employed. A GCB DLBCL cell line (Toledo) was selected due to sufficient expression of endogenous IRAK1 within this line, allowing simultaneous detection of IRAK1 loss with other key components of the NF-κB machinery. In the experiment, Toledo cells were treated with either DMSO (control) or <b>Degrader-3</b> at 1 μM for 6 h before the cell lysates were collected (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf" class="ext-link">Figure S1</a>). Proteome-wide quantitative mass spectrometric analysis was then performed as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf" class="ext-link">Supporting Information</a>. Significance cutoff was set at a 1.5-fold change with a minimum of two unique peptide measurements and <i>p</i>-values of less than 0.05. In total, 7550 proteins were successfully quantified in this experiment, among which IRAK1, cyclin G-associated kinase (GAK), and NmrA-like family domain-containing protein 1 (NMRAL1) were the only three proteins which exhibited a significant decrease in abundance after <b>Degrader-3</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The proteome analysis showed that the degradation profile was far more selective than the kinase binding profile, where only the kinases IRAK1 and GAK were decreased, while the changes for other kinase off-targets were negligible. It is noteworthy to point out that similar proteomics assays were carried out in the ABC-DLBCL cell line HBL1, where low endogenous levels of IRAK1 were detectable only by Western blotting but were beyond the sensitivity of mass spectrometry. GAK and NMRAL1 proteins were also the only two off-targets which were observed after treatment of <b>Degrader-3</b> at 1 μM for 6 h (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf" class="ext-link">Figure S2</a>). Surprisingly, though <b>Degrader-3</b> did not exhibit a strong GAK binding affinity in the KinomeScan competition binding assay (63% inhibition at 1 μM), it degraded GAK to 50% at 1 μM after 6 h treatment in the quantitative mass spectroscopy experiment. Our efforts to confirm the finding is limited by the availability of a specific antibody. On the other hand, degradation of NMRAL1 by <b>Degrader-3</b> was reproduced in both HEK293T and HBL-1 cells by Western blotting (data not shown). The degradation mechanism for NMRAL1 is inconclusive since the protein is not a kinase, and degradation may be mediated through other mechanisms which are under further investigation. Consistent with the findings from the proteomics study, Degrader-3 degraded only IRAK1, not IRAK4, as shown by Western blotting analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf" class="ext-link">Figure S4</a>). Collectively, the KinomeScan and proteomics data support that <b>Degrader-3</b> acts as a potent and selective IRAK1 degrader.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Volcano plots of abundance proteomics data of <b>Degrader-3</b> by proteomics analysis. Log fold change in abundance of protein with two unique peptide measurement using quantitative mass spectrometry after 6 h treatment of Toledo cells with 1 μM <b>Degrader-3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, utilizing the highly selective IRAK1 PROTAC <b>Degrader-3</b> and its epimer control <b>Degrader-4,</b> we aimed to investigate the biological consequence of IRAK1 depletion in ABC DLBCL cell lines harboring MyD88 L265P mutation and dependent on the MyD88 signaling pathway for survival, such as HBL-1 and OCI-LY10. We initially evaluated the IRAK1 degradation effect by immunoblotting analysis after 24 h treatment with degrader compounds in both ABC DLBCL cell lines. Although <b>Degrader-3</b> demonstrated comparable IRAK1 degradation potency in HBL-1 cell lines compared to HEK293T cells with DC<sub>50</sub> values of 3.0 and 3.3 nM, respectively, a decreased degradation potency was observed in the OCI-LY10 cell line with a DC<sub>50</sub> of 20 nM (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A).</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Characterization of <b>Degrader-3</b> and <b>Degrader-4</b> in ABC DLBCL cell lines. (A) DLBCL cell lines were treated with <b>Degrader-3</b> at the indicated concentrations for 24 h. IRAK1 protein levels were determined by Western blotting. (B) HBL-1 cells were treated with <b>Degrader-3</b> or its epimer <b>Degrader-4</b> at the indicated doses for 24 h. Phosphorylated IkBα and STAT3 protein levels were determined by Western blotting. (C) Cleaved PARP was determined by Western blotting. (D) Cell proliferative assay in HBL-1 and OCI-LY10 cells with MyD88 mutation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To evaluate the effects of IRAK1 depletion on the IRAK1 downstream signaling pathway in ABC DLBCL cell lines, HBL-1 cells were treated with <b>Degrader-3</b> and its epimer <b>Degrader-4</b> for 24 h before lysis and immunoblotting to detect phosphorylation events downstream of IRAK1 signaling. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B, <b>Degrader-3</b> resulted in a significant decrease in p-IκBα and p-STAT3 and minimal changes in total STAT3 levels, consistent with the notion that oncogenic MyD88/IRAK signaling converges on NF-κB activation and STAT3 phosphorylation and contributes to the survival of DLBCL cells.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26,27)</a> Not surprisingly, although the epimer control compound <b>Degrader-4</b> binds to the IRAK1 kinase domain with the same affinity (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and thus may act as an IRAK1 kinase inhibitor, it was unable to degrade IRAK1 at all doses tested. Minimal changes in cell signaling were observed by <b>Degrader-4</b>, supporting the hypothesis that the IRAK1 protein scaffolding function, as opposed to its kinase activity, is important in maintaining constitutive NF-κB activation in ABC DLBCL cells.<a onclick="showRef(event, 'ref4 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref6">(4,6)</a></div><div class="NLM_p">We then evaluated the effects of targeted IRAK1 degradation on cellular apoptosis. Accordingly, a 24 h treatment of <b>Degrader-3</b> in HBL-1 cell lines increased apoptosis, as monitored by PARP cleavage in a Western blot assay. As expected, treatment with equivalent doses of <b>Degrader-4</b> did not induce IRAK1 protein degradation and produced no change in PARP cleavage (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). These data suggest that the IRAK1 protein scaffolding function plays an important role in maintaining constitutive NF-κB and STAT3 activation in ABC DLBCL cells.</div><div class="NLM_p last">Next, we sought to determine the effect of IRAK1 depletion on ABC DLBCL cell proliferation. HBL-1 and OCI-LY10 cells with MyD88 L265P mutation were treated with <b>Degrader-3</b> for 12 days, and cell viability was determined by CTG assays. In line with the requirement for VHL engagement, we also profiled the control epimer <b>Degrader-4</b>. In the HBL-1 cell line, <b>Degrader-3</b> exhibited significantly enhanced cytotoxicity as compared with <b>Degrader-4</b> (60 nM <i>vs</i> > 1.5 μM); similar trends were observed in OCI-LY10 cell lines (170 nM <i>vs</i> > 3 μM) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D). In contrast, <b>JH-I-25,</b> a dual kinase inhibitor of IRAK1 and IRAK4, showed marginal cell growth inhibition in HBL1 with an IC<sub>50</sub> of 2.8 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf" class="ext-link">Figure S3</a>). The results demonstrate the distinct advantage of IRAK1 degradation over kinase inhibition as the degrader showed significantly improved anti-proliferative effects (over 40-fold) compared to the kinase inhibitor in HBL-1. Taken together, these data suggest that the IRAK1 scaffolding function, rather than its kinase activity, is required for ABC DLBCL cell survival and proliferation.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15361" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15361" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have discovered a novel series of VHL E3 ligase-based IRAK1 PROTAC degraders. One of the most potent compounds, <b>Degrader-3</b> (<b>JNJ-1013</b>), effectively degrades IRAK1 with a DC<sub>50</sub> of 3 nM in HBL-1 cells. In ABC DLBCL cell lines with MyD88 L265P mutation, <b>JNJ-1013</b> potently inhibits IRAK1 downstream signaling pathways, induces apoptosis, and demonstrates strong anti-proliferative effects, suggesting a critical role for the IRAK1 scaffolding function in ABC DLBCL cell survival. This class of highly selective IRAK1 degraders offers a pharmacological approach to evaluate the biological roles of IRAK1 beyond its enzymatic activity in various disease indications.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Chemistry</h3><div class="NLM_p last">All commercially obtained solvents and reagents were used as received. All solvents used for chemical reactions were of anhydrous grade, unless specifically indicated. Structures of the target compounds in this work were assigned by the use of NMR spectroscopy and mass spectroscopy (MS). The purities of all non-salt compounds were >95%, as determined on an Agilent 1200 HPLC system, an XTerra 3.5 μm 4.6 × 150 mm MS C18 column, using 0.04 (v/v) TFA in water and 0.02 (v/v) TFA in acetonitrile as the mobile phase. <sup>1</sup>H NMR spectra were recorded on a Bruker ADVANCE III (400 MHz) or a Varian 400MR (400 MHz) NMR spectrometer. Chemical shifts are reported in parts per million (ppm, δ) using the residual solvent line as an internal reference. Splitting patterns are reported as s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), br m (broad multiplet), or br s (broad singlet). Coupling constants (<i>J</i>) are reported in Hertz (Hz). Mass spectrometric analyses for nucleosides were performed on an Agilent 1200 HPLC system with an Agilent 6110/6140/1956C MSD mass spectrometer using ESI as ionization, a Phenomenex Luna C18 5 μm 5.0 × 20 mm column, mobile phase: 0.04%(v/v) TFA in water and 0.02%(v/v) TFA in acetonitrile, 40 °C, and a flow rate of 0.4 mL/min. Unless specifically indicated, chromatography refers to flash chromatography on a silica gel column.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> 6-(1<i>H</i>-Pyrazol-5-yl)picolinic Acid Hydrochloride (<b>2</b>)</h3><div class="NLM_p">Step 1: Ethyl 6-bromopicolinate (<b>1)</b> (15.0 g, 65.2 mmol), 1-(tetrahydro-2<i>H</i>-pyran-2-yl)-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-pyrazole (19.9 g, 71.5 mmol), and cesium carbonate (63.7 g, 196 mmol) in 1,4-dioxane (200 mL) and H<sub>2</sub>O (50 mL) were added to a 500 mL round-bottomed flask. The mixture was sparged with nitrogen three times and then treated with Pd(dppf)Cl<sub>2</sub> (3.62 g, 6.53 mmol). Then, the mixture was sparged with nitrogen three times and heated to 90 °C for 16 h. The reaction mixture was filtered and diluted with ethyl acetate (100 mL). The aqueous solution was acidified to pH = 5–6 by citric acid, extracted with dichloromethane (100 mL × 5), washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic layer was filtered and concentrated under reduced pressure to give the title compound as a gray solid (15.0 g, 76%). LCMS (ESI<sup>+</sup>): calcd for C<sub>16</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub> = 301.1, found; [M + H]<sup>+</sup> = 302.2, <i>t</i><sub>R</sub> = 1.10 min.</div><div class="NLM_p last">Step 2: Ethyl 6-(1-(tetrahydro-2<i>H</i>-pyran-2-yl)-1<i>H</i>-pyrazol-5-yl)picolinate (15.0 g, 49.8 mmol) and NaOH (2.39 g, 59.8 mmol) in MeOH (45 mL) and H<sub>2</sub>O (45 mL) were added to a 250 mL round-bottomed flask. The mixture was stirred at 20 °C for 2 h. The reaction mixture was concentrated, and the residue was diluted with H<sub>2</sub>O (50 mL). The mixture was acidified to pH = 5–6 by citric acid, extracted with dichloromethane (100 mL × 5), washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic layer was filtered and concentrated under reduced pressure to give 6-(1-(tetrahydro-2<i>H</i>-pyran-2-yl)-1<i>H</i>-pyrazol-5-yl)picolinic acid (12.0 g, 75.0%) as a gray solid. The crude product was then dissolved in dichloromethane (20 mL) and HCl/dioxane (50 mL, 4 M) and added to a 250 mL round-bottomed flask. The reaction mixture was stirred at 25 °C for 1 h. Then, the mixture was filtered, and the filter cake was dried to give the title product as a white solid (9.8 g, 98.3%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.67 (br s, 2H), 8.17 (dd, <i>J</i> = 0.8, 7.6 Hz, 1H), 8.05 (t, <i>J</i> = 7.6 Hz, 1H), 7.99–7.94 (m, 1H), 7.81 (d, <i>J</i> = 2.0 Hz, 1H), 7.01 (d, <i>J</i> = 2.0 Hz,1H). LCMS (ESI<sup>+</sup>) calcd for C<sub>9</sub>H<sub>7</sub>N<sub>3</sub>O<sub>2</sub> = 189.0, found; [M + H]<sup>+</sup> = 189.9, <i>t</i><sub>R</sub> = 0.84 min.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <i>tert</i>-Butyl 4-(4-amino-3-methoxyphenyl)piperazine-1-carboxylate (<b>4</b>)</h3><div class="NLM_p">Step 1: 4-Fluoro-2-methoxy-1-nitrobenzene (<b>3</b>) (10.0 g, 58.4 mmol), <i>tert</i>-butyl piperazine-1-carboxylate (11.4 g, 61.2 mmol), potassium carbonate (20.2 g, 87.7 mmol), and DMF (80 mL) were added to a 250 mL round-bottomed flask. The reaction mixture was stirred at 80 °C for 16 h. The reaction mixture was poured into 150 mL water and stirred at 25 °C for 0.5 h. The mixture was filtered, the cake was washed with petroleum ether (60 mL × 2), and the resultant mixture was concentrated to dryness under reduced pressure to afford the desired product as a yellow solid (19.0 g, 91.2%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.89–7.82 (m, 1H), 6.58–6.51 (m, 1H), 6.50–6.45 (m, 1H), 3.87 (s, 3H), 3.45–3.40 (m, 8H), 1.38 (s, 9H). LCMS (ESI<sup>+</sup>): calcd for C<sub>16</sub>H<sub>23</sub>N<sub>3</sub>O<sub>5</sub> = 337.2, found; [M + H]<sup>+</sup> = 338.1, <i>t</i><sub>R</sub> = 0.88 min.</div><div class="NLM_p last">Step 2: <i>tert</i>-Butyl 4-(3-methoxy-4-nitrophenyl)piperazine-1-carboxylate (56.0 g, 166 mmol), wet Pd/C (5.00 g, 10% on activated carbon), and MeOH (1 L) were added to a 2 L reaction flask. The suspension was degassed under vacuum and purged with H<sub>2</sub> three times. The mixture was stirred under H<sub>2</sub> (30 psi) at 25 °C for 12 h. The mixture was filtered through a pad of celite, and the filter cake was washed with MeOH (50 mL). The filtrate was evaporated to dryness to give the title compound as a purple solid (55.0 g, 97.0%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 6.57–6.47 (m, 2H), 6.35–6.26 (m, 1H), 4.51–4.10 (m, 2H), 3.74 (s, 3H), 3.50–3.39 (m, 4H), 2.93–2.81 (m, 4H), 1.41 (s, 9H). LCMS (ESI<sup>+</sup>): calcd for C<sub>16</sub>H<sub>25</sub>N<sub>3</sub>O<sub>3</sub> = 307.2, found; [M + H]<sup>+</sup> = 308.3, <i>t</i><sub>R</sub> = 0.74 min.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> <i>tert</i>-Butyl 4-(4-(6-(1<i>H</i>-pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperazine-1-carboxylate (<b>5</b>)</h3><div class="NLM_p last">A solution of <i>tert</i>-butyl 4-(4-amino-3-methoxyphenyl)piperazine-1-carboxylate (<b>4</b>) (10.0 g, 32.5 mmol), 6-(1<i>H</i>-pyrazol-5-yl)picolinic acid hydrochloride (<b>2</b>) (6.15 g, 32.5 mmol), HATU (14.8 g, 38.9 mmol), and TEA (13.6 mL, 97.6 mmol) in DMF (100 mL) was stirred at 25 °C for 8 h. The reaction mixture was diluted with water (200 mL), extracted with ethyl acetate (300 mL × 2), washed with brine (200 mL × 3), and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic layer was filtered and concentrated under reduced pressure to give a crude product, which was purified by chromatography on silica gel (eluent—petroleum ether:ethyl acetate = 1:0 to 0:1) to give the title compound as a gray solid (11.0 g, 70.7%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.94–13.17 (m, 1H), 10.65–10.36 (m, 1H), 8.29–8.18 (m, 1H), 8.14–8.00 (m, 2H), 7.94 (s, 1H), 7.67–7.57 (m, 1H), 7.04–6.96 (m, 1H), 6.79–6.71 (m, 1H), 6.61–6.55 (m, 1H), 3.99 (s, 2H), 3.86 (s, 1H), 3.48 (br s, 4H), 3.13 (d, <i>J</i> = 4.5 Hz, 4H), 1.43 (s, 9H). LCMS (ESI<sup>+</sup>): calcd for C<sub>25</sub>H<sub>30</sub>N<sub>6</sub>O<sub>4</sub> = 478.2, found; [M + H]<sup>+</sup> = 479.1, <i>t</i><sub>R</sub> = 0.925 min.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Methyl 3-(4-(4-(6-(1<i>H</i>-Pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperazin-1-yl)propanoate (<b>6</b>)</h3><div class="NLM_p">Step 1: <i>tert</i>-Butyl 4-(4-(6-(1<i>H</i>-pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperazine-1-carboxylate (8.50 g, 17.8 mmol) and HCl in EtOAc (80 mL, 4 M) were added to a 250 mL round-bottomed flask. The reaction mixture was stirred at room temperature for 2 h. The mixture was concentrated under reduced pressure to give the title compound as a white solid (7.00 g, 93.9%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.58 (s, 1H), 9.68–9.31 (m, 2H), 8.20–8.00 (m, 4H), 7.91–7.85 (m, 1H), 7.00 (s, 1H), 6.87–6.80 (m, 1H), 6.69–6.63 (m, 1H), 3.96 (s, 3H), 3.53–3.39 (m, 4H), 3.32–3.12 (m, 4H). LCMS (ESI<sup>+</sup>): calcd for C<sub>20</sub>H<sub>22</sub>N<sub>6</sub>O<sub>2</sub> = 378.2, found; [M + H]<sup>+</sup> = 379.2, <i>t</i><sub>R</sub> = 0.69 min.</div><div class="NLM_p last">Step 2: Methyl 3-bromopropanoate (100 mg, 0.599 mmol), <i>N</i>-(2-methoxy-4-(piperidin-4-yl)phenyl)-6-(1H-pyrazol-5-yl)picolinamide hydrochloride (249 mg, 0.600 mmol), triethylamine (0.8 mL, 5.74 mmol), potassium iodide (99.0 mg, 0.596 mmol), and 1-methyl-2-pyrrolidinone (2 mL) were added to a 10 mL sealed tube. The mixture was heated and stirred at 100 °C under a microwave initiator for 2 h. The mixture was then poured into water (10 mL) and extracted with ethyl acetate (20 mL × 2). The organic extracts were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford the crude product, which was purified by chromatography on silica gel (eluent—dichloromethane:methanol = 1:0 to 10:1) to afford the title product as a brown oil (150 mg, 48.5%). LCMS (ESI<sup>+</sup>): calcd for C<sub>24</sub>H<sub>28</sub>N<sub>6</sub>O<sub>4</sub> = 464.2, found; [M + H]<sup>+</sup> = 465.2, <i>t</i><sub>R</sub> = 0.67 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 13.88–13.23 (br s, 1H), 10.58–10.40 (br s, 1H), 8.30–7.45 (m, 5H), 7.06–6.91 (m, 1H), 6.78–6.63 (m, 1H), 6.54 (d, <i>J</i> = 8.8 Hz, 1H), 4.02–3.82 (m, 3H), 3.61 (s, 3H), 3.18–3.12 (m, 4H), 2.66–2.59 (m, 2H), 2.54 (d, <i>J</i> = 6.1 Hz, 4H), 2.20–2.16 (m, 2H).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> 3-(4-(4-(6-(1<i>H</i>-Pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperazin-1-yl)propanoic Acid (<b>7</b>)</h3><div class="NLM_p last">Methyl 3-(4-(4-(6-(1<i>H</i>-pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperidin-1-yl)propanoate (<b>6</b>) (150 mg, 0.323 mmol), aqueous NaOH (6 mL, 1 M), and THF (3 mL) were added to a 50 mL round-bottomed flask. The reaction mixture was stirred at 20 °C for 1 h. The reaction mixture was concentrated with pH adjusted to 6–7 with aqueous HCl (1 M), and a pale-yellow precipitate was formed. The precipitate was filtrated and dried to give the title product as a yellow solid (120 mg, 82.5%). LCMS (ESI<sup>+</sup>): calcd for C<sub>23</sub>H<sub>26</sub>N<sub>6</sub>O<sub>4</sub> = 450.2, found; [M + H]<sup>+</sup> = 451.0, <i>t</i><sub>R</sub> = 0.77 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ = 13.83–12.25 (m, 1H), 10.70–10.42 (m, 1H), 8.23–7.98 (m, 4H), 7.92–7.79 (m, 1H), 7.02–6.96 (m, 1H), 6.84–6.77 (m, 1H), 6.65–6.57 (m, 1H), 4.06–3.92 (m, 3H), 3.35–3.03 (m, 11H), 2.93–2.78 (m, 2H).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-Amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide Hydrochloride (<b>8</b>)</h3><div class="NLM_p last">The free amine compound was prepared according to the procedure published.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a><i>tert</i>-Butyl ((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (1.00 g, 1.88 mmol) and 4 M HCl in dioxane (8 mL) were added into a 50 mL round-bottomed flask. The mixture was stirred at 25 °C for 30 min. The reaction mixture was concentrated to dryness under reduced pressure to afford the title product as a pale white solid (880 mg, crude). Data match the previous reports.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> <i>N</i>-(4-(4-(3-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-3-oxopropyl)piperazin-1-yl)-2-methoxyphenyl)-6-(1<i>H</i>-pyrazol-3-yl)picolinamide (Degrader-1)</h3><div class="NLM_p last">3-(4-(4-(6-(1<i>H</i>-Pyrazol-3-yl)picolinamido)-3-methoxyphenyl)piperazin-1-yl)propanoic acid (<b>7</b>) (100 mg, 0.212 mmol), 1-hydroxybenzotriazole (43.0 mg, 0.318 mmol), <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine (138 mg, 1.07 mmol), 1-(3-dimethylaminopropyl)-3-ethylcarbodiimide hydrochloride (61.0 mg, 0.318 mmol), and DMF (3 mL) were added to a 50 mL round-bottomed flask. (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-Amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide·HCl (<b>8</b>) (99.0 mg, 0.212 mmol) was added to the mixture. The reaction mixture was stirred at room temperature for 8 h. The mixture was then poured into water (8 mL) and extracted with ethyl acetate (15 mL × 2). The organic extracts were washed with brine (8 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford the crude product, which was purified by prep-TLC (dichloromethane:methanol = 10:1, <i>R</i><sub>f</sub> = 0.4) to afford the title product as a yellow powder (50.2 mg, 27.1% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.90 (br s, 0.3H), 13.24 (br s, 0.7H), 10.66–10.21 (m, 1H), 9.02–8.87 (m, 1H), 8.73–8.43 (m, 2H), 8.32–7.51 (m, 5H), 7.48–7.27 (m, 4H), 7.07–6.90 (m, 1H), 6.81–6.62 (m, 1H), 6.58–6.46 (m, 1H), 5.17 (d, <i>J</i> = 3.4 Hz, 1H), 4.58 (d, <i>J</i> = 9.5 Hz, 1H), 4.50–4.32 (m, 3H), 4.28–4.16 (m, 1H), 4.03–3.77 (m, 3H), 3.72–3.59 (m, 2H), 3.31–3.14 (m, 4H), 2.71–2.53 (m, 6H), 2.49–2.44 (m, 1H), 2.40 (s, 3H), 2.37–2.29 (m, 1H), 2.07–2.00 (m, 1H), 1.96–1.87 (m, 1H), 0.96 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.5, 171.4, 170.1, 160.9, 151.7, 151.6, 150.4, 149.9, 149.5, 148.7, 148.1, 139.9, 139.5, 131.6, 130.9, 130.1, 129.1, 128.5, 127.8, 122.4, 120.5, 120.0, 119.6, 107.2, 103.5, 100.1, 69.4, 59.2, 56.9, 56.7, 56.6, 54.5, 52.8, 48.9, 42.1, 40.6, 38.4, 35.9, 32.8, 26.9, 16.4. LCMS (ESI<sup>+</sup>): calcd for C<sub>45</sub>H<sub>54</sub>N<sub>10</sub>O<sub>6</sub>S = 862.4, found, [M + H]<sup>+</sup> = 863.4, <i>t</i><sub>R</sub> = 0.4 min. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> Calcd for C<sub>45</sub>H<sub>54</sub>N<sub>10</sub>O<sub>6</sub>S, 863.4027; found, 863.4036.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> <i>tert</i>-Butyl 2-((1-(3-methoxy-4-nitrophenyl)piperidin-4-yl)oxy)acetate (<b>10</b>)</h3><div class="NLM_p last">1-(3-Methoxy-4-nitrophenyl)piperidin-4-ol (500 mg, 1.98 mmol) (<b>9</b>) and THF (20 mL) were added to a 50 mL round-bottomed flask at 0 °C before being treated with sodium hydride (119 mg, 2.97 mmol), and the mixture was stirred at 0 °C for 5 min. The reaction mixture was treated with <i>tert</i>-butyl 2-bromoacetate (770 mg, 3.96 mmol) at 0 °C, warmed, and stirred at 25 °C for 16 h. Brine (10 mL) was added to the mixture, and the mixture was concentrated under reduced pressure to remove THF before being diluted with H<sub>2</sub>O (10 mL); then, the mixture was extracted with EtOAc (30 mL × 3). The combined organic extracts were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to dryness under reduced pressure to give the crude product, which was purified by chromatography on silica gel (eluent—petroleum ether:ethyl acetate = 100:0 to 1:1) to afford the desired compound as a brown oil (650 mg, 76%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.87 (d, <i>J</i> = 9.6 Hz, 1H), 6.59 (dd, <i>J</i> = 2.4, 9.2 Hz, 1H), 6.52–6.49 (m, 1H), 4.05 (s, 2H), 3.90 (s, 3H), 3.81–3.73 (m, 2H), 3.66–3.59 (m, 1H), 3.28–3.19 (m, 2H), 1.95–1.86 (m, 2H), 1.56–1.46 (m, 2H), 1.43 (s, 9H).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> <i>tert</i>-Butyl 2-((1-(4-Amino-3-methoxyphenyl)piperidin-4-yl)oxy)acetate (<b>11</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl 2-((1-(3-methoxy-4-nitrophenyl)piperidin-4-yl)oxy)acetate (<b>10</b>) (650 mg, 1.77 mmol), wet Pd/C (200 mg), and MeOH (20 mL) were added to a hydrogenation bottle under a N<sub>2</sub> atmosphere. The suspension was degassed under vacuum, purged with N<sub>2</sub> three times, and then purged with hydrogen another three times. The resulting mixture was stirred under hydrogen (30 psi) at 25 °C for 16 h. The mixture was filtered through a pad of celite, and the filter cake was washed with methanol (30 mL × 3). The combined filtrates were concentrated under reduced pressure to give the product as a brown oil (450 mg, 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 6.52–6.47 (m, 2H), 6.31–6.26 (m, 1H), 4.26–4.13 (m, 2H), 4.01 (s, 2H), 3.73 (s, 3H), 3.45–3.40 (m, 1H), 3.27–3.21 (m, 2H), 2.68–2.61 (m, 2H), 1.92 (m, 2H), 1.60–1.51 (m, 2H), 1.43 (s, 9H). LCMS (ESI<sup>+</sup>): calcd for C<sub>18</sub>H<sub>28</sub>N<sub>2</sub>O<sub>4</sub> = 336.2, found; [M + H]<sup>+</sup> = 337.2, <i>t</i><sub>R</sub> = 0.70 min.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>tert</i>-Butyl 2-((1-(4-(6-(1<i>H</i>-pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperidin-4-yl)oxy)acetate (<b>12</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl 2-((1-(4-amino-3-methoxyphenyl)piperidin-4-yl)oxy)acetate (<b>11</b>) (250 mg, 0.743 mmol), 6-(1<i>H</i>-pyrazol-5-yl)picolinic acid hydrochloride (<b>2</b>) (140 mg, 0.740 mmol), HATU (171 mg, 0.892 mmol), DIPEA (301 mg, 2.23 mmol), and DMF (3 mL) were added to a 10 mL round-bottomed flask. The reaction mixture was stirred at 25 °C for 2 h. The reaction mixture was concentrated under reduced pressure to obtain a residue, which was diluted with water (30 mL) and extracted with ethyl acetate (50 mL × 3). The combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to dryness under reduced pressure to afford the crude product, which was purified by chromatography on silica gel (eluent—petroleum ether:ethyl acetate = 1:4 to 1:2) to give the desired product as a brown–green oil (216 mg, 57%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.12 (s, 1H), 10.44 (s, 1H), 8.09–7.71 (m, 5H), 6.89 (d, <i>J</i> = 2.0 Hz, 1H), 6.62 (d, <i>J</i> = 2.0 Hz, 1H), 6.46 (dd, <i>J</i> = 2.4, 8.8 Hz, 1H), 4.02–3.90 (m, 2H), 3.85 (s, 3H), 3.43 (td, <i>J</i> = 4.2, 8.3 Hz, 2H), 2.87–2.73 (m, 3H), 1.93–1.80 (m, 2H), 1.53–1.42 (m, 2H), 1.34 (s, 9H). LCMS (ESI<sup>+</sup>): calcd for C<sub>27</sub>H<sub>33</sub>N<sub>5</sub>O<sub>5</sub> = 507.2, found, [M + H]<sup>+</sup> = 508.2, <i>t</i><sub>R</sub> = 0.85 min.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 2-((1-(4-(6-(1<i>H</i>-Pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperidin-4-yl)oxy)acetic Acid Hydrochloride (<b>13</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl 2-((1-(4-(6-(1<i>H</i>-pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperidin-4-yl)oxy)acetate (<b>12</b>) (210 mg, 0.414 mmol) and HCl (4 mL, 4 M in dioxane) were added to a 50 mL round-bottomed flask. The resultant mixture was stirred at 25 °C for 2 h and then concentrated to dryness under reduced pressure to afford the title product as a white solid. (250 mg, crude). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.19–12.79 (br m, 2H), 10.53 (s, 1H), 8.18–8.12 (m, 1H), 8.12–7.98 (m, 3H), 7.90–7.81 (m, 1H), 6.99 (d, <i>J</i> = 2.0 Hz, 1H), 6.72 (d, <i>J</i> = 2.0 Hz, 1H), 6.55 (dd, <i>J</i> = 2.0, 8.8 Hz, 1H), 4.08 (s, 2H), 3.94 (s, 3H), 3.60–3.48 (m, 3H), 2.94–2.82 (m, 2H), 2.02–1.91 (m, 2H), 1.63–1.52 (m, 2H). LCMS (ESI<sup>+</sup>): calcd for C<sub>23</sub>H<sub>25</sub>N<sub>5</sub>O<sub>5</sub> = 451.2, found, [M + H]<sup>+</sup> = 452.2, <i>t</i><sub>R</sub> = 2.11 min.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-Amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (14) and Epimer (2<i>S</i>,4<i>S</i>)-1-((<i>S</i>)-2-Amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>15</b>)</h3><div class="NLM_p last">The compound was prepared according to the procedure of <i>J. Med. Chem.</i><b>2019</b>, <i>62</i>, 1420–1442. Data matched previous reports.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>N</i>-(4-(4-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)piperidin-1-yl)-2-methoxyphenyl)-6-(1<i>H</i>-pyrazol-5-yl)picolinamide (Degrader-2)</h3><div class="NLM_p last">2-((1-(4-(6-(1<i>H</i>-pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperidin-4-yl)oxy)acetic acid hydrochloride (<b>13</b>) (100 mg, 0.205 mmol), (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide hydrochloride (<b>8</b>) (95.7 mg, 0.205 mmol), 1-hydroxybenzotriazole (41.5 mg, 0.307 mmol), DIPEA (79.5 mg, 0.615 mmol), and DMF (2 mL) were added to a 50 mL round-bottomed flask. Then, 1-(3-dimethylaminopropyl)-3-ethylcarbodiimide hydrochloride (58.9 mg, 0.307 mmol) was added to the mixture. The reaction mixture was stirred at 40 °C for 2 h. The mixture was purified by prep-HPLC (Column: Agela Durashell C18 150*25 5 μm, mobile phase A: water (10 mM NH<sub>4</sub>HCO<sub>3</sub>)–ACN, mobile phase B: acetonitrile, flow rate: 25 mL/min, gradient condition from 30% B to 60%). The pure fractions were collected, and the solvent was evaporated under vacuum to give a residue, which was lyophilized to give the title product as a yellow powder (20.7 mg, 11.7%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.87 (br s, 0.3H), 13.22 (br s, 0.8H), 10.66–10.30 (m, 1H), 9.03–8.92 (m, 1H), 8.77–8.55 (m, 1H), 8.23–7.92 (m, 4H), 7.68–7.17 (m, 6H), 7.04–6.93 (m, 1H), 6.78–6.65 (m, 1H), 6.60–6.49 (m, 1H), 5.17 (d, <i>J</i> = 3.2 Hz, 1H), 4.56 (d, <i>J</i> = 9.5 Hz, 1H), 4.49–4.21 (m, 4H), 4.09–3.91 (m, 4H), 3.84 (br s, 1H), 3.73–3.46 (m, 5H), 3.04–2.85 (m, 2H), 2.47–2.40 (m, 3H), 2.13–1.83 (m, 4H), 1.73–1.54 (m, 2H), 0.99–0.91 (m, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.2, 171.8, 169.6, 169.2, 161.3, 152.0, 151.9, 149.5, 148.2, 139.9, 139.6, 131.6, 130.2, 129.4, 129.2, 128.6, 128.0, 122.6, 120.7, 108.4, 108.2, 108.2, 103.8, 101.0, 75.3, 75.3, 75.2, 69.4, 67.5, 67.3, 59.2, 57.1, 56.5, 56.2, 47.5, 47.4, 42.2, 38.4, 36.7, 36.4, 30.9, 30.8, 26.8, 26.7,16.4, 16.4. LCMS (ESI<sup>+</sup>): calcd for C<sub>45</sub>H<sub>53</sub>N<sub>9</sub>O<sub>7</sub>S = 863.4, found, [M + H]<sup>+</sup> = 864.4, <i>t</i><sub>R</sub> = 2.09 min. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> Calcd for C<sub>45</sub>H<sub>53</sub>N<sub>19</sub>O<sub>7</sub>S, 864.3859; found, 864.3855.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>N</i>-(4-(4-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)piperidin-1-yl)-2-methoxyphenyl)-6-(1<i>H</i>-pyrazol-5-yl)picolinamide (Degrader-3)</h3><div class="NLM_p last">2-((1-(4-(6-(1<i>H</i>-Pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperidin-4-yl)oxy)acetic acid hydrochloride (<b>13</b>) (650 mg, 1.33 mmol), (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide hydrochloride (<b>14</b>) (641 mg, 1.33 mmol), HOBt (269 mg, 1.99 mmol), DIPEA (516 mg, 3.99 mmol), and DMF (5 mL) were added to a 50 mL round-bottomed flask. Then, EDCI<sup>.</sup> HCl (381 mg, 1.99 mmol) was added to the mixture. The reaction mixture was stirred at 40 °C for 2 h. The reaction mixture was purified by prep-HPLC (column: Phenomenex Synergi Max-RP 250*50 mm*10 μm, mobile phase A: water (0.225%FA), mobile phase B: acetonitrile, flow rate: 60 mL/min, and gradient condition: from 35% B to 65% B). The pure fractions were collected, and the solvent was evaporated under vacuum to give a residue, which was lyophilized to give the product as a yellow powder (218.5 mg, 18.7%).<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.97–13.03 (m, 1H), 10.58 (br s, 1H), 8.98 (s, 1H), 8.48 (d, <i>J</i> = 7.6 Hz, 1H), 8.27–7.87 (m, 5H), 7.47–7.33 (m, 5H), 6.99 (br s, 1H), 6.74 (br s, 1H), 6.61–6.54 (m, 1H), 5.21–5.00 (m, 1H), 4.94–4.84 (m, 1H), 4.59–4.41 (m, 2H), 4.28 (br s, 1H), 4.10–3.90 (m, 5H), 3.63–3.49 (m, 5H), 3.00–2.88 (m, 2H), 2.45 (s, 3H), 2.11–1.95 (m, 3H), 1.80–1.57 (m, 3H), 1.50–1.34 (m, 3H), 0.94 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.9, 170.0, 169.5, 169.2, 168.5, 163.5, 161.2, 161.2, 151.9, 149.6, 148.2, 145.3, 145.0, 139.6, 131.6, 130.2, 130.1, 129.4,129.3, 126.8, 122.5, 120.7, 107.8, 103.8, 103.8, 100.7, 75.5, 69.3, 67.4, 67.3, 59.2, 59.0, 57.1, 56.5, 56.1, 48.3, 47.0, 38.2, 36.7, 36.4, 31.0, 30.9, 26.8, 26.7,23.0, 22.6, 16.5. LCMS (ESI<sup>+</sup>): calcd for C<sub>46</sub>H<sub>55</sub>N<sub>9</sub>O<sub>7</sub>S = 877.4, found, [M + H]<sup>+</sup> = 878.4, <i>t</i><sub>R</sub> = 4.5 min. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> Calcd for C<sub>46</sub>H<sub>55</sub>N<sub>9</sub>O<sub>7</sub>S, 878.4023; found, 878.4036.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>N</i>-(4-(4-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>S</i>)-4-Hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)piperidin-1-yl)-2-methoxyphenyl)-6-(1<i>H</i>-pyrazol-5-yl)picolinamide (Degrader-4)</h3><div class="NLM_p last">2-((1-(4-(6-(1<i>H</i>-Pyrazol-5-yl)picolinamido)-3-methoxyphenyl)piperidin-4-yl)oxy)acetic acid (<b>12</b>) (46 mg, 0.102 mmol), (2<i>S</i>,4<i>S</i>)-1-((<i>S</i>)-2-amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>15</b>) (49.5 mg, 0.103 mmol), HOBt (20.7 mg, 0.153 mmol), DIPEA (39.6 mg, 0.306 mmol), and DMF (1 mL) were added to a 50 mL round-bottomed flask. Then, EDCI<sup>.</sup>HCl (29.4 mg, 0.153 mmol) was added to the mixture. The reaction mixture was stirred at 40 °C for 12 h. The mixture was poured into water (10 mL) and extracted by ethyl acetate (30 mL × 3). The organic layer was collected and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give the residue, which was purified by prep-HPLC (column: DuraShell 150*25 mm*5 μm, mobile phase A: water (0.2% FA), mobile phase B: acetonitrile, flow rate: 25 mL/min, and gradient condition: from 28% B to 58%). The pure fractions were collected, and the solvent was evaporated under vacuum to give a residue, which was lyophilized to give the product as a yellow powder (20.9 mg, 23.4%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.88 (br s, 0.2H), 13.22 (br s, 0.6H), 10.63–10.35 (m, 1H), 8.97 (s, 1H), 8.47–7.50 (m, 6H), 7.47–7.33 (m, 5H), 6.98 (br s, 1H), 6.73 (br s, 1H), 6.61–6.54 (m, 1H), 5.38–5.31 (m, 1H), 4.96–4.87 (m, 1H), 4.52 (d, <i>J</i> = 9.0 Hz, 1H), 4.44–4.31 (m, 1H), 4.26–4.17 (m, 1H), 4.09–3.74 (m, 6H), 3.60–3.43 (m, 4H), 3.00–2.87 (m, 2H), 2.46–2.43 (m, 3H), 2.39–2.31 (m, 1H), 2.05–1.91 (m, 2H), 1.72–1.54 (m, 3H), 1.38 (m, 3H), 1.03–0.88 (m, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.3, 170.1, 169.7, 169.4, 161.2, 151.9, 149.6, 148.2, 144.8, 139.6, 131.6, 130.3, 129.3, 126.8, 122.5, 120.7, 107.8, 100.7,75.5, 69.4, 67.8, 67.2, 58.9, 56.5, 56.2, 56.0, 48.3, 47.0, 37.3, 36.6, 35.9, 31.0, 30.9, 26.9, 26.7, 22.8, 22.4, 16.4. LCMS (ESI<sup>+</sup>): calcd for C<sub>46</sub>H<sub>55</sub>N<sub>9</sub>O<sub>7</sub>S = 877.4, found, [M + H]<sup>+</sup> = 878.4, <i>t</i><sub>R</sub> = 4.22 min. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup>calcd for C<sub>46</sub>H<sub>55</sub>N<sub>9</sub>O<sub>7</sub>S, 878.4023; found, 878.4009.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00103" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55808" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55808" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00103?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00103</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Western blot validation of proteomics studies of <b>Degrader-3</b>; global proteomic data of <b>Degrader-3</b> in HBL-1 cells; anti-proliferation potency of <b>JH-I-25</b> in HBL-1 cells; Western blot of IRAK1 and IRAK4 protein degradation in HEK293T cells; kinase activity assay; VHL FP assay; immunoblotting protocol; cell culture and cell viability assay; global proteomics assay protocol; <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra for conjugates <b>Degrader-1</b>, <b>Degrader-2</b>, <b>Degrader-3</b>, and <b>Degrader-4</b>; and KinomeScan raw data of <b>JH-I-25</b>, <b>Degrader-3</b>, and <b>Degrader-4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Proteomics raw data of <b>Degrader-3</b> in Toledo (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Proteomics raw data of <b>Degrader-3</b> in HBL-1 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf">jm1c00103_si_001.pdf (854.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_002.csv">jm1c00103_si_002.csv (478.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_003.csv">jm1c00103_si_003.csv (969.31 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00103" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71861" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71861" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer D. Venable</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, Janssen Research & Development, San Diego, California 92121, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f19b87949f90939d94b1988582df9b9f9bdf929e9c"><span class="__cf_email__" data-cfemail="59332f3c37383b353c19302d2a77333733773a3634">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xuedong Dai</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, Janssen (China) Research &
Development, Shanghai 201210, P.R. China</span>; 
    <span class="hlFld-Affiliation affiliation">Discovery
Chemistry, Janssen Research & Development, Shanghai 201210, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4358-0319" title="Orcid link">https://orcid.org/0000-0003-4358-0319</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#40382421297178002934336e2a2e2a6e232f2d"><span class="__cf_email__" data-cfemail="79011d1810484139100d0a57131713571a1614">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liqiang Fu</span> - <span class="hlFld-Affiliation affiliation">Discovery
Chemistry, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bin Shen</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linglong Kong</span> - <span class="hlFld-Affiliation affiliation">Discovery
Chemistry, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingtao Liu</span> - <span class="hlFld-Affiliation affiliation">Discovery
Chemistry, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wangyang Tu</span> - <span class="hlFld-Affiliation affiliation">Discovery
Chemistry, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenqian Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Cai</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomarker, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaotao Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomarker, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Na Cheng</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingxuan Xia</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tianyuan Zhou</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qian Liu</span> - <span class="hlFld-Affiliation affiliation">Discovery
Chemistry, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanping Xu</span> - <span class="hlFld-Affiliation affiliation">Discovery
Chemistry, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jennifer Yang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Gavine</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology, Janssen Research & Development, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulrike Philippar</span> - <span class="hlFld-Affiliation affiliation">Janssen
Pharmaceutical Research & Development, Beerse 2340, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ricardo Attar</span> - <span class="hlFld-Affiliation affiliation">Janssen
Pharmaceutical Research & Development, Spring House, Pennsylvania 19477, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James P. Edwards</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, Janssen Research & Development, San Diego, California 92121, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.F. and J.Z. contributed equally. L.F wrote the original draft, J.Z., L.K., X.W., T.Z., P.G., J.P.E., J.D.V, and X.D revised the draft, and all authors reviewed and edited the manuscript. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77163" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77163" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank the members of the Discovery Science and Oncology Translational Research team at Janssen including Charu Chaudhry and Haopeng Rui for insightful discussion and Yan Wang for help with graphical abstract design.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ABC</td><td class="NLM_def"><p class="first last">activated B-cell-like</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>, <i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">GAK</td><td class="NLM_def"><p class="first last">cyclin G-associated kinase</p></td></tr><tr><td class="NLM_term">GCB</td><td class="NLM_def"><p class="first last">center B-cell-like</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[<i>bis</i>(dimethylamino)methylene]-1<i>H</i>-1,2,3- triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">NHL</td><td class="NLM_def"><p class="first last">non-Hodgkin lymphoma</p></td></tr><tr><td class="NLM_term">IL-1R</td><td class="NLM_def"><p class="first last">interleukin-1 receptor</p></td></tr><tr><td class="NLM_term">IRAK1</td><td class="NLM_def"><p class="first last">interleukin-1 receptor-associated kinase 1</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">MyD88</td><td class="NLM_def"><p class="first last">myeloid differentiation primary response 88</p></td></tr><tr><td class="NLM_term">NF-κB</td><td class="NLM_def"><p class="first last">IκB kinase (IKK)-nuclear factor-κB</p></td></tr><tr><td class="NLM_term">NMRAL1</td><td class="NLM_def"><p class="first last">NmrA-like family domain-containing protein 1</p></td></tr><tr><td class="NLM_term">PROTACs</td><td class="NLM_def"><p class="first last">proteolysis-targeting chimeras</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethanolamine</p></td></tr><tr><td class="NLM_term">TLRs</td><td class="NLM_def"><p class="first last">toll-like receptors</p></td></tr><tr><td class="NLM_term">TRAF6</td><td class="NLM_def"><p class="first last">TNF receptor-associated factor 6</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71361" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71361" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 28 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Nardo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balka, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardona Gloria, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, S. L.</span></span> <span> </span><span class="NLM_article-title">Interleukin-1 receptor-associated kinase 4 (IRAK4) plays a dual role in myddosome formation and Toll-like receptor signaling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">15195</span>– <span class="NLM_lpage">15207</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.003314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1074%2Fjbc.RA118.003314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=30076215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWhu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=15195-15207&author=D.+De%0ANardoauthor=K.+R.+Balkaauthor=Y.+Cardona+Gloriaauthor=V.+R.+Raoauthor=E.+Latzauthor=S.+L.+Masters&title=Interleukin-1+receptor-associated+kinase+4+%28IRAK4%29+plays+a+dual+role+in+myddosome+formation+and+Toll-like+receptor+signaling&doi=10.1074%2Fjbc.ra118.003314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Interleukin-1 receptor-associated kinase 4 (IRAK4) plays a dual role in myddosome formation and Toll-like receptor signaling</span></div><div class="casAuthors">De Nardo, Dominic; Balka, Katherine R.; Gloria, Yamel Cardona; Rao, Vikram R.; Latz, Eicke; Masters, Seth L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">15195-15207</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Toll-like receptors (TLRs) form part of the host innate immune system, in which they act as sensors of microbial and endogenous danger signals.  Upon TLR activation, the intracellular Toll/interleukin-1 receptor domains of TLR dimers initiate oligomerization of a multiprotein signaling platform comprising myeloid differentiation primary response 88 (MyD88) and members of the interleukin-1 receptor-assocd. kinase (IRAK) family.  Formation of this myddosome complex initiates signal transduction pathways, leading to the activation of transcription factors and the prodn. of inflammatory cytokines.  To date, little is known about the assembly and disassembly of the myddosome and about the mechanisms by which these complexes mediate multiple downstream signaling pathways.  Here, we isolated myddosome complexes from whole-cell lysates of TLR-activated primary mouse macrophages and from IRAK reporter macrophages to examine the kinetics of myddosome assembly and disassembly.  Using a selective inhibitor of IRAK4's kinase activity, we found that whereas TLR cytokine responses were ablated, myddosome formation was stabilized in the absence of IRAK4's kinase activity.  Of note, IRAK4 inhibition had only a minimal effect on NF-κB and mitogen-activated protein kinase (MAPK) signaling.  In summary, our results indicate that IRAK4 has a crit. scaffold function in myddosome formation and that its kinase activity is dispensable for myddosome assembly and activation of the NF-κB and MAPK pathways but is essential for MyD88-dependent prodn. of inflammatory cytokines.  Our findings suggest that the scaffold function of IRAK4 may be an attractive target for treating inflammatory and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNgAjazN469rVg90H21EOLACvtfcHk0ligLC_JbjMOPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWhu7zK&md5=9005c249071ba957ee8232d861d82610</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.003314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.003314%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BNardo%26aufirst%3DD.%26aulast%3DBalka%26aufirst%3DK.%2BR.%26aulast%3DCardona%2BGloria%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DV.%2BR.%26aulast%3DLatz%26aufirst%3DE.%26aulast%3DMasters%26aufirst%3DS.%2BL.%26atitle%3DInterleukin-1%2520receptor-associated%2520kinase%25204%2520%2528IRAK4%2529%2520plays%2520a%2520dual%2520role%2520in%2520myddosome%2520formation%2520and%2520Toll-like%2520receptor%2520signaling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D15195%26epage%3D15207%26doi%3D10.1074%2Fjbc.ra118.003314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span> <span> </span><span class="NLM_article-title">Helical assembly in the MyD88-IRAK4-IRAK2 complex in TLR/IL-1R signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>465</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1038/nature09121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnature09121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=20485341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1eku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2010&pages=885-890&author=S.-C.+Linauthor=Y.-C.+Loauthor=H.+Wu&title=Helical+assembly+in+the+MyD88-IRAK4-IRAK2+complex+in+TLR%2FIL-1R+signalling&doi=10.1038%2Fnature09121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Helical assembly in the MyD88-IRAK4-IRAK2 complex in TLR/IL-1R signalling</span></div><div class="casAuthors">Lin, Su-Chang; Lo, Yu-Chih; Wu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">7300</span>),
    <span class="NLM_cas:pages">885-890</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MyD88, IRAK4 and IRAK2 are crit. signaling mediators of the TLR/IL1-R superfamily.  Here the authors report the crystal structure of the MyD88-IRAK4-IRAK2 death domain (DD) complex, which surprisingly reveals a left-handed helical oligomer that consists of 6 MyD88, 4 IRAK4 and 4 IRAK2 DDs.  Assembly of this helical signaling tower is hierarchical, in which MyD88 recruits IRAK4 and the MyD88-IRAK4 complex recruits the IRAK4 substrates IRAK2 or the related IRAK1.  Formation of these Myddosome complexes brings the kinase domains of IRAKs into proximity for phosphorylation and activation.  Composite binding sites are required for recruitment of the individual DDs in the complex, which are confirmed by mutagenesis and previously identified signaling mutations.  Specificities in Myddosome formation are dictated by both mol. complementarity and correspondence of surface electrostatics.  The MyD88-IRAK4-IRAK2 complex provides a template for Toll signaling in Drosophila and an elegant mechanism for versatile assembly and regulation of DD complexes in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8zWmiKDKlVbVg90H21EOLACvtfcHk0ligLC_JbjMOPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1eku70%253D&md5=af8b6ea34425853777839fd0e6af9602</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature09121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09121%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DS.-C.%26aulast%3DLo%26aufirst%3DY.-C.%26aulast%3DWu%26aufirst%3DH.%26atitle%3DHelical%2520assembly%2520in%2520the%2520MyD88-IRAK4-IRAK2%2520complex%2520in%2520TLR%252FIL-1R%2520signalling%26jtitle%3DNature%26date%3D2010%26volume%3D465%26spage%3D885%26epage%3D890%26doi%3D10.1038%2Fnature09121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roschewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. H.</span></span> <span> </span><span class="NLM_article-title">Diffuse large B-cell lymphoma-treatment approaches in the molecular era</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1038/nrclinonc.2013.197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnrclinonc.2013.197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=24217204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyrsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=12-23&author=M.+Roschewskiauthor=L.+M.+Staudtauthor=W.+H.+Wilson&title=Diffuse+large+B-cell+lymphoma-treatment+approaches+in+the+molecular+era&doi=10.1038%2Fnrclinonc.2013.197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Diffuse large B-cell lymphoma-treatment approaches in the molecular era</span></div><div class="casAuthors">Roschewski, Mark; Staudt, Louis M.; Wilson, Wyndham H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-23</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Diffuse large B-cell lymphoma (DLBCL) is an aggressive B-cell non-Hodgkin lymphoma that affects patients of all ages with a wide range of clin. presentations.  Although DLBCL is curable even in advanced stages, up to one-third of patients will not achieve cure with initial therapy.  In the modern era of rituximab-based therapy as the first-line treatment, the prognoses of patients who require salvage therapy are poor and most will eventually succumb to their disease.  Insight into the complex mol. circuitry of DLBCL reveals a diverse range of somatic mutations and aberrant intracellular signalling pathways that characterize distinct mol. subsets of the disease.  The next major breakthrough in DLBCL therapy during this 'mol. era' of disease definition will be the identification of combinations of novel agents that target the oncogenic drivers of these subsets.  Well-conducted clin. trials, with translational mol. investigations, will be essential to achieve the goal of precision medicine and expand the no. of patients with DLBCL who achieve a cure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBpteNHDsYabVg90H21EOLACvtfcHk0ligLC_JbjMOPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyrsbzL&md5=af716d22a25172e77cb3c636cc5aedec</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2013.197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2013.197%26sid%3Dliteratum%253Aachs%26aulast%3DRoschewski%26aufirst%3DM.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26aulast%3DWilson%26aufirst%3DW.%2BH.%26atitle%3DDiffuse%2520large%2520B-cell%2520lymphoma-treatment%2520approaches%2520in%2520the%2520molecular%2520era%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2014%26volume%3D11%26spage%3D12%26epage%3D23%26doi%3D10.1038%2Fnrclinonc.2013.197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhavar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhammer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romesser, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenwald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Hermelink, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connors, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimsza, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smeland, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braziel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubbs, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisenburger, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">Oncogenically active MYD88 mutations in human lymphoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>470</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1038/nature09671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnature09671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=21179087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVCksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=470&publication_year=2011&pages=115-119&author=V.+N.+Ngoauthor=R.+M.+Youngauthor=R.+Schmitzauthor=S.+Jhavarauthor=W.+Xiaoauthor=K.-H.+Limauthor=H.+Kohlhammerauthor=W.+Xuauthor=Y.+Yangauthor=H.+Zhaoauthor=A.+L.+Shafferauthor=P.+Romesserauthor=G.+Wrightauthor=J.+Powellauthor=A.+Rosenwaldauthor=H.+K.+Muller-Hermelinkauthor=G.+Ottauthor=R.+D.+Gascoyneauthor=J.+M.+Connorsauthor=L.+M.+Rimszaauthor=E.+Campoauthor=E.+S.+Jaffeauthor=J.+Delabieauthor=E.+B.+Smelandauthor=R.+I.+Fisherauthor=R.+M.+Brazielauthor=R.+R.+Tubbsauthor=J.+R.+Cookauthor=D.+D.+Weisenburgerauthor=W.+C.+Chanauthor=L.+M.+Staudt&title=Oncogenically+active+MYD88+mutations+in+human+lymphoma&doi=10.1038%2Fnature09671"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenically active MYD88 mutations in human lymphoma</span></div><div class="casAuthors">Ngo, Vu N.; Young, Ryan M.; Schmitz, Roland; Jhavar, Sameer; Xiao, Wenming; Lim, Kian-Huat; Kohlhammer, Holger; Xu, Weihong; Yang, Yandan; Zhao, Hong; Shaffer, Arthur L.; Romesser, Paul; Wright, George; Powell, John; Rosenwald, Andreas; Muller-Hermelink, Hans Konrad; Ott, German; Gascoyne, Randy D.; Connors, Joseph M.; Rimsza, Lisa M.; Campo, Elias; Jaffe, Elaine S.; Delabie, Jan; Smeland, Erlend B.; Fisher, Richard I.; Braziel, Rita M.; Tubbs, Raymond R.; Cook, J. R.; Weisenburger, Denny D.; Chan, Wing C.; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">470</span>
        (<span class="NLM_cas:issue">7332</span>),
    <span class="NLM_cas:pages">115-119</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The activated B-cell-like (ABC) subtype of diffuse large B-cell lymphoma (DLBCL) remains the least curable form of this malignancy despite recent advances in therapy.  Constitutive nuclear factor (NF)-κB and JAK kinase signaling promotes malignant cell survival in these lymphomas, but the genetic basis for this signaling is incompletely understood.  Here we describe the dependence of ABC DLBCLs on MYD88, an adaptor protein that mediates toll and interleukin (IL)-1 receptor signaling, and the discovery of highly recurrent oncogenic mutations affecting MYD88 in ABC DLBCL tumors.  RNA interference screening revealed that MYD88 and the assocd. kinases IRAK1 and IRAK4 are essential for ABC DLBCL survival.  High-throughput RNA resequencing uncovered MYD88 mutations in ABC DLBCL lines.  Notably, 29% of ABC DLBCL tumors harbored the same amino acid substitution, L265P, in the MYD88 Toll/IL-1 receptor (TIR) domain at an evolutionarily invariant residue in its hydrophobic core.  This mutation was rare or absent in other DLBCL subtypes and Burkitt's lymphoma, but was obsd. in 9% of mucosa-assocd. lymphoid tissue lymphomas.  At a lower frequency, addnl. mutations were obsd. in the MYD88 TIR domain, occurring in both the ABC and germinal center B-cell-like (GCB) DLBCL subtypes.  Survival of ABC DLBCL cells bearing the L265P mutation was sustained by the mutant but not the wild-type MYD88 isoform, demonstrating that L265P is a gain-of-function driver mutation.  The L265P mutant promoted cell survival by spontaneously assembling a protein complex contg. IRAK1 and IRAK4, leading to IRAK4 kinase activity, IRAK1 phosphorylation, NF-κB signaling, JAK kinase activation of STAT3, and secretion of IL-6, IL-10 and interferon-β.  Hence, the MYD88 signaling pathway is integral to the pathogenesis of ABC DLBCL, supporting the development of inhibitors of IRAK4 kinase and other components of this pathway for the treatment of tumors bearing oncogenic MYD88 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5pEPxh1G6J7Vg90H21EOLACvtfcHk0ljFSpBwHd9n2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVCksQ%253D%253D&md5=3dff4d54af8f64bb9e57f04cf274fee8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature09671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09671%26sid%3Dliteratum%253Aachs%26aulast%3DNgo%26aufirst%3DV.%2BN.%26aulast%3DYoung%26aufirst%3DR.%2BM.%26aulast%3DSchmitz%26aufirst%3DR.%26aulast%3DJhavar%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DLim%26aufirst%3DK.-H.%26aulast%3DKohlhammer%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DRomesser%26aufirst%3DP.%26aulast%3DWright%26aufirst%3DG.%26aulast%3DPowell%26aufirst%3DJ.%26aulast%3DRosenwald%26aufirst%3DA.%26aulast%3DMuller-Hermelink%26aufirst%3DH.%2BK.%26aulast%3DOtt%26aufirst%3DG.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DConnors%26aufirst%3DJ.%2BM.%26aulast%3DRimsza%26aufirst%3DL.%2BM.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DDelabie%26aufirst%3DJ.%26aulast%3DSmeland%26aufirst%3DE.%2BB.%26aulast%3DFisher%26aufirst%3DR.%2BI.%26aulast%3DBraziel%26aufirst%3DR.%2BM.%26aulast%3DTubbs%26aufirst%3DR.%2BR.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DWeisenburger%26aufirst%3DD.%2BD.%26aulast%3DChan%26aufirst%3DW.%2BC.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DOncogenically%2520active%2520MYD88%2520mutations%2520in%2520human%2520lymphoma%26jtitle%3DNature%26date%3D2011%26volume%3D470%26spage%3D115%26epage%3D119%26doi%3D10.1038%2Fnature09671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Z.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span> <span> </span><span class="NLM_article-title">Assessing IRAK4 functions in ABC DLBCL by IRAK4 kinase inhibition and protein degradation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1500</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.chembiol.2020.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=32888499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsl2it7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=1500-1509&author=J.+Zhangauthor=L.+Fuauthor=B.+Shenauthor=Y.+Liuauthor=W.+Wangauthor=X.+Caiauthor=L.+Kongauthor=Y.+Yanauthor=R.+Mengauthor=Z.+Zhangauthor=Y.-N.+P.+Chenauthor=Q.+Liuauthor=Z.-K.+Wanauthor=T.+Zhouauthor=X.+Wangauthor=P.+Gavineauthor=A.+Del+Rosarioauthor=K.+Ahnauthor=U.+Philipparauthor=R.+Attarauthor=J.+Yangauthor=Y.+Xuauthor=J.+P.+Edwardsauthor=X.+Dai&title=Assessing+IRAK4+functions+in+ABC+DLBCL+by+IRAK4+kinase+inhibition+and+protein+degradation&doi=10.1016%2Fj.chembiol.2020.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing IRAK4 Functions in ABC DLBCL by IRAK4 Kinase Inhibition and Protein Degradation</span></div><div class="casAuthors">Zhang, Jing; Fu, Liqiang; Shen, Bin; Liu, Yingtao; Wang, Wenqian; Cai, Xin; Kong, Linglong; Yan, Yilin; Meng, Ryan; Zhang, Zhuming; Chen, Ying-Nan P.; Liu, Qian; Wan, Zhao-Kui; Zhou, Tianyuan; Wang, Xiaotao; Gavine, Paul; Del Rosario, Amanda; Ahn, Kay; Philippar, Ulrike; Attar, Ricardo; Yang, Jennifer; Xu, Yanping; Edwards, James P.; Dai, Xuedong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1500-1509.e13</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The interleukin-1 receptor-activated kinase 4 (IRAK4) belongs to the IRAK family of serine/threonine kinases and plays a central role in the innate immune response.  However, the function of IRAK4 in tumor growth and progression remains elusive.  Here we sought to det. the enzymic and scaffolding functions of IRAK4 in activated B-cell-like diffuse large B cell lymphoma (ABC DLBCL).  We chose a highly selective IRAK4 kinase inhibitor to probe the biol. effects of kinase inhibition and developed a series of IRAK4 degraders to evaluate the effects of protein degrdn. in ABC DLBCL cells.  Interestingly, the results demonstrated that neither IRAK4 kinase inhibition nor protein degrdn. led to cell death or growth inhibition, suggesting a redundant role for IRAK4 in ABC DLBCL cell survival.  IRAK4 degraders characterized in this study provide useful tools for understanding IRAK4 protein scaffolding function, which was previously unachievable using pharmacol. perturbation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-fxMUd7pQEbVg90H21EOLACvtfcHk0ljFSpBwHd9n2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsl2it7fJ&md5=0d1f9d102b101c3da4aa2be668ed7851</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DKong%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.-N.%2BP.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWan%26aufirst%3DZ.-K.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DGavine%26aufirst%3DP.%26aulast%3DDel%2BRosario%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DK.%26aulast%3DPhilippar%26aufirst%3DU.%26aulast%3DAttar%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DDai%26aufirst%3DX.%26atitle%3DAssessing%2520IRAK4%2520functions%2520in%2520ABC%2520DLBCL%2520by%2520IRAK4%2520kinase%2520inhibition%2520and%2520protein%2520degradation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D1500%26epage%3D1509%26doi%3D10.1016%2Fj.chembiol.2020.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roulland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasbekar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceribelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palisoc, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valadez, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies-Hill, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenwald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimsza, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estephan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdhoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruhlak, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittaluga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oellerich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>560</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0290-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fs41586-018-0290-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=29925955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=560&publication_year=2018&pages=387-391&author=J.+D.+Phelanauthor=R.+M.+Youngauthor=D.+E.+Websterauthor=S.+Roullandauthor=G.+W.+Wrightauthor=M.+Kasbekarauthor=A.+L.+Shafferauthor=M.+Ceribelliauthor=J.+Q.+Wangauthor=R.+Schmitzauthor=M.+Nakagawaauthor=E.+Bachyauthor=D.+W.+Huangauthor=Y.+Jiauthor=L.+Chenauthor=Y.+Yangauthor=H.+Zhaoauthor=X.+Yuauthor=W.+Xuauthor=M.+M.+Palisocauthor=R.+R.+Valadezauthor=T.+Davies-Hillauthor=W.+H.+Wilsonauthor=W.+C.+Chanauthor=E.+S.+Jaffeauthor=R.+D.+Gascoyneauthor=E.+Campoauthor=A.+Rosenwaldauthor=G.+Ottauthor=J.+Delabieauthor=L.+M.+Rimszaauthor=F.+J.+Rodriguezauthor=F.+Estephanauthor=M.+Holdhoffauthor=M.+J.+Kruhlakauthor=S.+M.+Hewittauthor=C.+J.+Thomasauthor=S.+Pittalugaauthor=T.+Oellerichauthor=L.+M.+Staudt&title=A+multiprotein+supercomplex+controlling+oncogenic+signalling+in+lymphoma&doi=10.1038%2Fs41586-018-0290-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</span></div><div class="casAuthors">Phelan, James D.; Young, Ryan M.; Webster, Daniel E.; Roulland, Sandrine; Wright, George W.; Kasbekar, Monica; Shaffer, IArthur L., III; Ceribelli, Michele; Wang, James Q.; Schmitz, Roland; Nakagawa, Masao; Bachy, Emmanuel; Huang, Da Wei; Ji, Yanlong; Chen, Lu; Yang, Yandan; Zhao, Hong; Yu, Xin; Xu, Weihong; Palisoc, Maryknoll M.; Valadez, Racquel R.; Davies-Hill, Theresa; Wilson, Wyndham H.; Chan, Wing C.; Jaffe, Elaine S.; Gascoyne, Randy D.; Campo, Elias; Rosenwald, Andreas; Ott, German; Delabie, Jan; Rimsza, Lisa M.; Rodriguez, Fausto J.; Estephan, Fayez; Holdhoff, Matthias; Kruhlak, Michael J.; Hewitt, Stephen M.; Thomas, Craig J.; Pittaluga, Stefania; Oellerich, Thomas; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">560</span>
        (<span class="NLM_cas:issue">7718</span>),
    <span class="NLM_cas:pages">387-391</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B cell receptor (BCR) signaling has emerged as a therapeutic target in B cell lymphomas, but inhibiting this pathway in diffuse large B cell lymphoma (DLBCL) has benefited only a subset of patients1.  Gene expression profiling identified two major DLBCL subtypes, known as germinal center (GC) B cell-like (GCB) and activated B cell-like (ABC)2,3, with inferior outcomes following immunochemotherapy in ABC.  Autoantigens drive BCR-dependent activation of NF-κB in ABC DLBCL through a kinase cascade of SYK, BTK and PKCβ to promote the assembly of the CARD11-BCL10-MALT1 (CBM) adapter complex that recruits and activates IκB kinase (IKK)4-6.  Genome sequencing revealed gain-of-function mutations targeting the CD79A and CD79B BCR subunits and the Toll-like receptor (TLR) signaling adapter MYD885,7, with MYD88L265P being the most prevalent isoform.  In a clin. trial, the BTK inhibitor, ibrutinib, produced responses in 37% of ABC cases1.  The most striking response rate (80%) was obsd. in tumors with both CD79B and MYD88L265P mutations, but how these mutations cooperate to promote dependence on BCR signaling remains unclear.  Herein, we used genome-wide CRISPR-Cas9 screening and functional proteomics to understand the mol. basis of exceptional clin. responses to ibrutinib.  We discovered a new mode of oncogenic BCR signaling in ibrutinib-responsive cell lines and biopsies, coordinated by a multiprotein supercomplex formed by MYD88, TLR9, and the BCR (My-T-BCR).  The My-T-BCR co-localizes with mTOR on endolysosomes, where it drives pro-survival NF-κB and mTOR signaling.  Inhibitors of BCR and mTOR signaling cooperatively decreased My-T-BCR supercomplex formation and function, providing mechanistic insight into their synergistic toxicity for My-T-BCR+ DLBCL cells.  My-T-BCR complexes characterized ibrutinib-responsive malignancies and distinguished ibrutinib responders from non-responders.  Our data provide a roadmap for the rational deployment of oncogenic signaling inhibitors in molecularly-defined subsets of DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmeclQGnynR7Vg90H21EOLACvtfcHk0li6B8NSAdMKHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL7F&md5=86543a2209046fea5eac50fa9a0aade3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0290-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0290-0%26sid%3Dliteratum%253Aachs%26aulast%3DPhelan%26aufirst%3DJ.%2BD.%26aulast%3DYoung%26aufirst%3DR.%2BM.%26aulast%3DWebster%26aufirst%3DD.%2BE.%26aulast%3DRoulland%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DG.%2BW.%26aulast%3DKasbekar%26aufirst%3DM.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DCeribelli%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BQ.%26aulast%3DSchmitz%26aufirst%3DR.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DBachy%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DD.%2BW.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DPalisoc%26aufirst%3DM.%2BM.%26aulast%3DValadez%26aufirst%3DR.%2BR.%26aulast%3DDavies-Hill%26aufirst%3DT.%26aulast%3DWilson%26aufirst%3DW.%2BH.%26aulast%3DChan%26aufirst%3DW.%2BC.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DRosenwald%26aufirst%3DA.%26aulast%3DOtt%26aufirst%3DG.%26aulast%3DDelabie%26aufirst%3DJ.%26aulast%3DRimsza%26aufirst%3DL.%2BM.%26aulast%3DRodriguez%26aufirst%3DF.%2BJ.%26aulast%3DEstephan%26aufirst%3DF.%26aulast%3DHoldhoff%26aufirst%3DM.%26aulast%3DKruhlak%26aufirst%3DM.%2BJ.%26aulast%3DHewitt%26aufirst%3DS.%2BM.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DPittaluga%26aufirst%3DS.%26aulast%3DOellerich%26aufirst%3DT.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DA%2520multiprotein%2520supercomplex%2520controlling%2520oncogenic%2520signalling%2520in%2520lymphoma%26jtitle%3DNature%26date%3D2018%26volume%3D560%26spage%3D387%26epage%3D391%26doi%3D10.1038%2Fs41586-018-0290-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffitt, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Love, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leppa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasanen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meriranta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karjalainen-Lindsberg, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nørgaard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gang, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Høgdall, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heavican, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lone, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Healy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedoriw, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernal-Mizrachi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koff, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flowers, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paltiel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldschmidt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calaminici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clear, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribben, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czader, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrejka, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsi, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au-Yeung, R. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, W. W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evens, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilichowska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadburn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rempel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Happ, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dave, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dave, S. S.</span></span> <span> </span><span class="NLM_article-title">Genetic and functional drivers of diffuse large B cell lymphoma</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">494</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.cell.2017.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=28985567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1WqsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=481-494&author=A.+Reddyauthor=J.+Zhangauthor=N.+S.+Davisauthor=A.+B.+Moffittauthor=C.+L.+Loveauthor=A.+Waldropauthor=S.+Leppaauthor=A.+Pasanenauthor=L.+Merirantaauthor=M.-L.+Karjalainen-Lindsbergauthor=P.+N%C3%B8rgaardauthor=M.+Pedersenauthor=A.+O.+Gangauthor=E.+H%C3%B8gdallauthor=T.+B.+Heavicanauthor=W.+Loneauthor=J.+Iqbalauthor=Q.+Qinauthor=G.+Liauthor=S.+Y.+Kimauthor=J.+Healyauthor=K.+L.+Richardsauthor=Y.+Fedoriwauthor=L.+Bernal-Mizrachiauthor=J.+L.+Koffauthor=A.+D.+Statonauthor=C.+R.+Flowersauthor=O.+Paltielauthor=N.+Goldschmidtauthor=M.+Calaminiciauthor=A.+Clearauthor=J.+Gribbenauthor=E.+Nguyenauthor=M.+B.+Czaderauthor=S.+L.+Ondrejkaauthor=A.+Collieauthor=E.+D.+Hsiauthor=E.+Tseauthor=R.+K.+H.+Au-Yeungauthor=Y.-L.+Kwongauthor=G.+Srivastavaauthor=W.+W.+L.+Choiauthor=A.+M.+Evensauthor=M.+Pilichowskaauthor=M.+Sengarauthor=N.+Reddyauthor=S.+Liauthor=A.+Chadburnauthor=L.+I.+Gordonauthor=E.+S.+Jaffeauthor=S.+Levyauthor=R.+Rempelauthor=T.+Tzengauthor=L.+E.+Happauthor=T.+Daveauthor=D.+Rajagopalanauthor=J.+Dattaauthor=D.+B.+Dunsonauthor=S.+S.+Dave&title=Genetic+and+functional+drivers+of+diffuse+large+B+cell+lymphoma&doi=10.1016%2Fj.cell.2017.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic and functional drivers of diffuse large-cell B-cell lymphoma</span></div><div class="casAuthors">Reddy, Anupama; Zhang, Jenny; Davis, Nicholas S.; Moffitt, Andrea B.; Love, Cassandra L.; Waldrop, Alexander; Leppa, Sirpa; Pasanen, Annika; Meriranta, Leo; Karjalainen-Lindsberg, Marja-Liisa; Noergaard, Peter; Pedersen, Mette; Gang, Anne O.; Hoegdall, Estrid; Heavican, Tayla B.; Lone, Waseem; Iqbal, Javeed; Qin, Qiu; Li, Guojie; Kim, So Young; Healy, Jane; Richards, Kristy L.; Fedoriw, Yuri; Bernal-Mizrachi, Leon; Koff, Jean L.; Staton, Ashley D.; Flowers, Christopher R.; Paltiel, Ora; Goldschmidt, Neta; Calaminici, Maria; Clear, Andrew; Gribben, John; Nguyen, Evelyn; Czader, Magdalena B.; Ondrejka, Sarah L.; Collie, Angela; Hsi, Eric D.; Tse, Eric; Au-Yeung, Rex K. H.; Kwong, Yok-Lam; Srivastava, Gopesh; Choi, William W. L.; Evens, Andrew M.; Pilichowska, Monika; Sengar, Manju; Reddy, Nishitha; Li, Shaoying; Chadburn, Amy; Gordon, Leo I.; Jaffe, Elaine S.; Levy, Shawn; Rempel, Rachel; Tzeng, Tiffany; Happ, Lanie E.; Dave, Tushar; Rajagopalan, Deepthi; Datta, Jyotishka; Dunson, David B.; Dave, Sandeep S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">481-494.e15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Diffuse large B cell lymphoma (DLBCL) is the most common form of blood cancer and is characterized by a striking degree of genetic and clin. heterogeneity.  This heterogeneity poses a major barrier to understanding the genetic basis of the disease and its response to therapy.  Here, we performed an integrative anal. of whole-exome sequencing and transcriptome sequencing in a cohort of 1001 DLBCL patients to comprehensively define the landscape of 150 genetic drivers of the disease.  We characterized the functional impact of these genes using an unbiased CRISPR screen of DLBCL cell lines to define oncogenes that promote cell growth.  A prognostic model comprising these genetic alterations outperformed current established methods: cell of origin, the International Prognostic Index comprising clin. variables, and dual MYC and BCL2 expression.  These results comprehensively define the genetic drivers and their functional roles in DLBCL to identify new therapeutic opportunities in the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs_9SXwsqp-rVg90H21EOLACvtfcHk0li6B8NSAdMKHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1WqsrbE&md5=8dc716cdd54891b35ca98361c3d3c64d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DN.%2BS.%26aulast%3DMoffitt%26aufirst%3DA.%2BB.%26aulast%3DLove%26aufirst%3DC.%2BL.%26aulast%3DWaldrop%26aufirst%3DA.%26aulast%3DLeppa%26aufirst%3DS.%26aulast%3DPasanen%26aufirst%3DA.%26aulast%3DMeriranta%26aufirst%3DL.%26aulast%3DKarjalainen-Lindsberg%26aufirst%3DM.-L.%26aulast%3DN%25C3%25B8rgaard%26aufirst%3DP.%26aulast%3DPedersen%26aufirst%3DM.%26aulast%3DGang%26aufirst%3DA.%2BO.%26aulast%3DH%25C3%25B8gdall%26aufirst%3DE.%26aulast%3DHeavican%26aufirst%3DT.%2BB.%26aulast%3DLone%26aufirst%3DW.%26aulast%3DIqbal%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DHealy%26aufirst%3DJ.%26aulast%3DRichards%26aufirst%3DK.%2BL.%26aulast%3DFedoriw%26aufirst%3DY.%26aulast%3DBernal-Mizrachi%26aufirst%3DL.%26aulast%3DKoff%26aufirst%3DJ.%2BL.%26aulast%3DStaton%26aufirst%3DA.%2BD.%26aulast%3DFlowers%26aufirst%3DC.%2BR.%26aulast%3DPaltiel%26aufirst%3DO.%26aulast%3DGoldschmidt%26aufirst%3DN.%26aulast%3DCalaminici%26aufirst%3DM.%26aulast%3DClear%26aufirst%3DA.%26aulast%3DGribben%26aufirst%3DJ.%26aulast%3DNguyen%26aufirst%3DE.%26aulast%3DCzader%26aufirst%3DM.%2BB.%26aulast%3DOndrejka%26aufirst%3DS.%2BL.%26aulast%3DCollie%26aufirst%3DA.%26aulast%3DHsi%26aufirst%3DE.%2BD.%26aulast%3DTse%26aufirst%3DE.%26aulast%3DAu-Yeung%26aufirst%3DR.%2BK.%2BH.%26aulast%3DKwong%26aufirst%3DY.-L.%26aulast%3DSrivastava%26aufirst%3DG.%26aulast%3DChoi%26aufirst%3DW.%2BW.%2BL.%26aulast%3DEvens%26aufirst%3DA.%2BM.%26aulast%3DPilichowska%26aufirst%3DM.%26aulast%3DSengar%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DGordon%26aufirst%3DL.%2BI.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DLevy%26aufirst%3DS.%26aulast%3DRempel%26aufirst%3DR.%26aulast%3DTzeng%26aufirst%3DT.%26aulast%3DHapp%26aufirst%3DL.%2BE.%26aulast%3DDave%26aufirst%3DT.%26aulast%3DRajagopalan%26aufirst%3DD.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DDunson%26aufirst%3DD.%2BB.%26aulast%3DDave%26aufirst%3DS.%2BS.%26atitle%3DGenetic%2520and%2520functional%2520drivers%2520of%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D481%26epage%3D494%26doi%3D10.1016%2Fj.cell.2017.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective, covalent IRAK1 inhibitor with antiproliferative activity in MYD88 mutated B-cell lymphoma</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2238</span>– <span class="NLM_lpage">2243</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVSrsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2238-2243&author=J.+M.+Hatcherauthor=G.+Yangauthor=L.+Wangauthor=S.+B.+Ficarroauthor=S.+Buhrlageauthor=H.+Wuauthor=J.+A.+Martoauthor=S.+P.+Treonauthor=N.+S.+Gray&title=Discovery+of+a+selective%2C+covalent+IRAK1+inhibitor+with+antiproliferative+activity+in+MYD88+mutated+B-cell+lymphoma&doi=10.1021%2Facsmedchemlett.0c00378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective, Covalent IRAK1 Inhibitor with Antiproliferative Activity in MYD88 Mutated B-Cell Lymphoma</span></div><div class="casAuthors">Hatcher, John M.; Yang, Guang; Wang, Li; Ficarro, Scott B.; Buhrlage, Sara; Wu, Hao; Marto, Jarrod A.; Treon, Steven P.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2238-2243</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interleukin 1 (IL-1) receptor-assocd. kinases (IRAKs) are serine/threonine kinases that play crit. roles in initiating the innate immune response against foreign pathogens.  Addnl., dysregulation of IRAK1 signaling plays a role in neoplastic disorders.  For example, IRAK1 was shown to be important for survival and proliferation in many B-cell lymphomas, including Waldenstr.ovrddot.om's macroglobulinemia (WM) and ABC subtype Diffused Large B-cell Lymphoma (DLBCL) cells.  Here, we report the discovery of a highly potent and selective covalent inhibitor of IRAK1, JH-X-119-01.  Intact protein MS labeling studies confirmed that JH-X-119-01 irreversibly labels IRAK1 at C302.  This compd. exhibited cytotoxic activity at single digit micromolar concns. in a panel of WM, DLBCL, and lymphoma cell lines expressing MYD88.  Cotreatment of JH-X-119-01 with the BTK inhibitor ibrutinib resulted in synergistic killing effects in these systems.  Taken together, JH-X-119-01 represents a highly selective probe of IRAK1 for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNZtBwczMlbbVg90H21EOLACvtfcHk0lh3nnAK_MFstA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVSrsb3N&md5=e5957d9438ed3a31af10a323d77b6777</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00378%26sid%3Dliteratum%253Aachs%26aulast%3DHatcher%26aufirst%3DJ.%2BM.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DBuhrlage%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520a%2520selective%252C%2520covalent%2520IRAK1%2520inhibitor%2520with%2520antiproliferative%2520activity%2520in%2520MYD88%2520mutated%2520B-cell%2520lymphoma%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D2238%26epage%3D2243%26doi%3D10.1021%2Facsmedchemlett.0c00378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0lh3nnAK_MFstA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lheYZgn0zn9uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs)—past, present and future</span>. <i>Drug Discov. Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29%E2%80%94past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eYdCSFgidgGfLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%25E2%2580%2594past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discov.%2520Today%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Harnessing the power of proteolysis for targeted protein inactivation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.molcel.2020.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=32004468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2020&pages=446-460&author=R.+Vermaauthor=D.+Mohlauthor=R.+J.+Deshaies&title=Harnessing+the+power+of+proteolysis+for+targeted+protein+inactivation&doi=10.1016%2Fj.molcel.2020.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing the Power of Proteolysis for Targeted Protein Inactivation</span></div><div class="casAuthors">Verma, Rati; Mohl, Dane; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-460</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Two decades into the twenty-first century, a confluence of breakthrough technologies wielded at the mol. level is presenting biologists with unique opportunities to unravel the complexities of the cellular world.  CRISPR/Cas9 allows gene knock-outs, knock-ins, and single-base editing at chromosomal loci.  RNA-based tools such as siRNA, antisense oligos, and morpholinos can be used to silence expression of specific genes.  Meanwhile, protein knockdown tools that draw inspiration from natural regulatory mechanisms and facilitate elimination of native or degron-tagged proteins from cells are rapidly emerging.  The acute and reversible redn. in protein levels enabled by these methods allows for precise detn. of loss-of-function phenotypes free from secondary effects or compensatory adaptation that can confound nucleic-acid-based methods that involve slow depletion or permanent loss of a protein.  In this Review, we summarize the ingenious ways biologists have exploited natural mechanisms for protein degrdn. to direct the elimination of specific proteins at will.  This has led to advancements not only in basic research but also in the therapeutic space with the introduction of PROTACs into clin. trials for cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-2OyHAw-uMLVg90H21EOLACvtfcHk0lheYZgn0zn9uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D&md5=8d74ed00d55184be33786d0ee44ad787</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DR.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DHarnessing%2520the%2520power%2520of%2520proteolysis%2520for%2520targeted%2520protein%2520inactivation%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D77%26spage%3D446%26epage%3D460%26doi%3D10.1016%2Fj.molcel.2020.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higueruelo, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rausch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. L.</span></span> <span> </span><span class="NLM_article-title">IRAK-4 inhibitors. Part 1: a series of amides</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3211</span>– <span class="NLM_lpage">3214</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.04.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.bmcl.2008.04.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=18474425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BD1cXms1yiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3211-3214&author=G.+M.+Buckleyauthor=L.+Gowersauthor=A.+P.+Higuerueloauthor=K.+Jenkinsauthor=S.+R.+Mackauthor=T.+Morganauthor=D.+M.+Parryauthor=W.+R.+Pittauthor=O.+Rauschauthor=M.+D.+Richardauthor=V.+Sabinauthor=J.+L.+Fraser&title=IRAK-4+inhibitors.+Part+1%3A+a+series+of+amides&doi=10.1016%2Fj.bmcl.2008.04.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK-4 inhibitors. Part 1: A series of amides</span></div><div class="casAuthors">Buckley, George M.; Gowers, Lewis; Higueruelo, Alicia Perez; Jenkins, Kerry; Mack, Stephen R.; Morgan, Trevor; Parry, David M.; Pitt, William R.; Rausch, Oliver; Richard, Marianna D.; Sabin, Verity; Fraser, Joanne L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3211-3214</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The synthesis and profile of a series of amides, e.g., I, are described.  Some of these compds. were potent IRAK-4 inhibitors and two examples were evaluated in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2WnghW2pEwbVg90H21EOLACvtfcHk0ljE7OVBoVx8mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXms1yiu78%253D&md5=75fb794d58ae52827a58b8ff59394111</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.04.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.04.058%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DG.%2BM.%26aulast%3DGowers%26aufirst%3DL.%26aulast%3DHigueruelo%26aufirst%3DA.%2BP.%26aulast%3DJenkins%26aufirst%3DK.%26aulast%3DMack%26aufirst%3DS.%2BR.%26aulast%3DMorgan%26aufirst%3DT.%26aulast%3DParry%26aufirst%3DD.%2BM.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DRausch%26aufirst%3DO.%26aulast%3DRichard%26aufirst%3DM.%2BD.%26aulast%3DSabin%26aufirst%3DV.%26aulast%3DFraser%26aufirst%3DJ.%2BL.%26atitle%3DIRAK-4%2520inhibitors.%2520Part%25201%253A%2520a%2520series%2520of%2520amides%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3211%26epage%3D3214%26doi%3D10.1016%2Fj.bmcl.2008.04.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human IRAK1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">13507</span>– <span class="NLM_lpage">13512</span>, <span class="refDoi"> DOI: 10.1073/pnas.1714386114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1073%2Fpnas.1714386114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=29208712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmtLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=13507-13512&author=L.+Wangauthor=Q.+Qiaoauthor=R.+Ferraoauthor=C.+Shenauthor=J.+M.+Hatcherauthor=S.+J.+Buhrlageauthor=N.+S.+Grayauthor=H.+Wu&title=Crystal+structure+of+human+IRAK1&doi=10.1073%2Fpnas.1714386114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human IRAK1</span></div><div class="casAuthors">Wang, Li; Qiao, Qi; Ferrao, Ryan; Shen, Chen; Hatcher, John M.; Buhrlage, Sara J.; Gray, Nathanael S.; Wu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">13507-13512</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Interleukin 1 (IL-1) receptor-assocd. kinases (IRAKs) are serine/threonine kinases that play crit. roles in initiating innate immune responses against foreign pathogens and other types of dangers through their role in Toll-like receptor (TLR) and interleukin 1 receptor (IL-1R) mediated signaling pathways.  Upon ligand binding, TLRs and IL-1Rs recruit adaptor proteins, such as myeloid differentiation primary response gene 88 (MyD88), to the membrane, which in turn recruit IRAKs via the death domains in these proteins to form the Myddosome complex, leading to IRAK kinase activation.  Despite their biol. and clin. significance, only the IRAK4 kinase domain structure has been detd. among the four IRAK family members.  Here, we report the crystal structure of the human IRAK1 kinase domain in complex with a small mol. inhibitor.  The structure reveals both similarities and differences between IRAK1 and IRAK4 and is suggestive of approaches to develop IRAK1- or IRAK4-specific inhibitors for potential therapeutic applications.  While the IRAK4 kinase domain is capable of homodimerization in the unphosphorylated state, we found that the IRAK1 kinase domain is constitutively monomeric regardless of its phosphorylation state.  Addnl., the IRAK1 kinase domain forms heterodimers with the phosphorylated, but not unphosphorylated, IRAK4 kinase domain.  Collectively, these data indicate a two-step kinase activation process in which the IRAK4 kinase domain first homodimerizes in the Myddosome, leading to its trans-autophosphorylation and activation.  The phosphorylated IRAK4 kinase domain then forms heterodimers with the IRAK1 kinase domain within the Myddosome, leading to its subsequent phosphorylation and activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8eF96qYaD-7Vg90H21EOLACvtfcHk0ljE7OVBoVx8mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmtLrF&md5=6e8711f19b9608e522e29e423915e20c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1714386114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1714386114%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DFerrao%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DHatcher%26aufirst%3DJ.%2BM.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWu%26aufirst%3DH.%26atitle%3DCrystal%2520structure%2520of%2520human%2520IRAK1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D13507%26epage%3D13512%26doi%3D10.1073%2Fpnas.1714386114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plantevin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitefield, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kercher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebinger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccotti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havens, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leftheris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baculi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBrun, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khambatta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of CRBN E3 ligase modulator CC-92480 for the treatment of relapsed and refractory multiple myeloma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6648</span>– <span class="NLM_lpage">6676</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktlOku78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6648-6676&author=J.+D.+Hansenauthor=M.+Correaauthor=M.+A.+Nagyauthor=M.+Alexanderauthor=V.+Plantevinauthor=V.+Grantauthor=B.+Whitefieldauthor=D.+Huangauthor=T.+Kercherauthor=R.+Harrisauthor=R.+K.+Narlaauthor=J.+Leistenauthor=Y.+Tangauthor=M.+Moghaddamauthor=K.+Ebingerauthor=J.+Piccottiauthor=C.+G.+Havensauthor=B.+Cathersauthor=J.+Carmichaelauthor=T.+Danielauthor=R.+Vesseyauthor=L.+G.+Hamannauthor=K.+Leftherisauthor=D.+Mendyauthor=F.+Baculiauthor=L.+A.+LeBrunauthor=G.+Khambattaauthor=A.+Lopez-Girona&title=Discovery+of+CRBN+E3+ligase+modulator+CC-92480+for+the+treatment+of+relapsed+and+refractory+multiple+myeloma&doi=10.1021%2Facs.jmedchem.9b01928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CRBN E3 Ligase Modulator CC-92480 for the Treatment of Relapsed and Refractory Multiple Myeloma</span></div><div class="casAuthors">Hansen, Joshua D.; Correa, Matthew; Nagy, Mark A.; Alexander, Matt; Plantevin, Veronique; Grant, Virginia; Whitefield, Brandon; Huang, Dehua; Kercher, Timothy; Harris, Roy; Narla, Rama Krishna; Leisten, Jim; Tang, Yang; Moghaddam, Mehran; Ebinger, Katalin; Piccotti, Joseph; Havens, Courtney G.; Cathers, Brian; Carmichael, James; Daniel, Thomas; Vessey, Rupert; Hamann, Lawrence G.; Leftheris, Katerina; Mendy, Derek; Baculi, Frans; LeBrun, Laurie A.; Khambatta, Gody; Lopez-Girona, Antonia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6648-6676</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many patients with multiple myeloma (MM) initially respond to treatment with modern combination regimens including immunomodulatory agents (lenalidomide and pomalidomide) and proteasome inhibitors.  However, some patients lack an initial response to therapy (i.e., are refractory), and although the mean survival of MM patients has more than doubled in recent years, most patients will eventually relapse.  To address this need, we explored the potential of novel cereblon E3 ligase modulators (CELMoDs) for the treatment of patients with relapsed or refractory multiple myeloma (RRMM).  We found that optimization beyond potency of degrdn., including degrdn. efficiency and kinetics, could provide efficacy in a lenalidomide-resistant setting.  Guided by both phenotypic and protein degrdn. data, we describe a series of CELMoDs for the treatment of RRMM, culminating in the discovery of CC-92480, a novel protein degrader and the first CELMoD to enter clin. development that was specifically designed for efficient and rapid protein degrdn. kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBUWnyqWibcbVg90H21EOLACvtfcHk0ljE7OVBoVx8mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktlOku78%253D&md5=bce7d7a9033474a2800944baf21fed81</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01928%26sid%3Dliteratum%253Aachs%26aulast%3DHansen%26aufirst%3DJ.%2BD.%26aulast%3DCorrea%26aufirst%3DM.%26aulast%3DNagy%26aufirst%3DM.%2BA.%26aulast%3DAlexander%26aufirst%3DM.%26aulast%3DPlantevin%26aufirst%3DV.%26aulast%3DGrant%26aufirst%3DV.%26aulast%3DWhitefield%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DKercher%26aufirst%3DT.%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DNarla%26aufirst%3DR.%2BK.%26aulast%3DLeisten%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DMoghaddam%26aufirst%3DM.%26aulast%3DEbinger%26aufirst%3DK.%26aulast%3DPiccotti%26aufirst%3DJ.%26aulast%3DHavens%26aufirst%3DC.%2BG.%26aulast%3DCathers%26aufirst%3DB.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%26aulast%3DVessey%26aufirst%3DR.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DLeftheris%26aufirst%3DK.%26aulast%3DMendy%26aufirst%3DD.%26aulast%3DBaculi%26aufirst%3DF.%26aulast%3DLeBrun%26aufirst%3DL.%2BA.%26aulast%3DKhambatta%26aufirst%3DG.%26aulast%3DLopez-Girona%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520CRBN%2520E3%2520ligase%2520modulator%2520CC-92480%2520for%2520the%2520treatment%2520of%2520relapsed%2520and%2520refractory%2520multiple%2520myeloma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6648%26epage%3D6676%26doi%3D10.1021%2Facs.jmedchem.9b01928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4(CRBN) ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/nature18611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.+A.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4%28CRBN%29+ubiquitin+ligase&doi=10.1038%2Fnature18611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0lhCgIB2AbvnNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%2BA.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4%2528CRBN%2529%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257%26doi%3D10.1038%2Fnature18611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdubek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blease, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">SALL4 mediates teratogenicity as a thalidomide-dependent cereblon substrate</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">981</span>– <span class="NLM_lpage">987</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0129-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fs41589-018-0129-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=30190590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Kgu7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=981-987&author=M.+E.+Matyskielaauthor=S.+Coutoauthor=X.+Zhengauthor=G.+Luauthor=J.+Huiauthor=K.+Stampauthor=C.+Drewauthor=Y.+Renauthor=M.+Wangauthor=A.+Carpenterauthor=C.-W.+Leeauthor=T.+Claytonauthor=W.+Fangauthor=C.-C.+Luauthor=M.+Rileyauthor=P.+Abdubekauthor=K.+Bleaseauthor=J.+Hartkeauthor=G.+Kumarauthor=R.+Vesseyauthor=M.+Rolfeauthor=L.+G.+Hamannauthor=P.+P.+Chamberlain&title=SALL4+mediates+teratogenicity+as+a+thalidomide-dependent+cereblon+substrate&doi=10.1038%2Fs41589-018-0129-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SALL4 mediates teratogenicity as a thalidomide-dependent cereblon substrate</span></div><div class="casAuthors">Matyskiela, Mary E.; Couto, Suzana; Zheng, Xinde; Lu, Gang; Hui, Julia; Stamp, Katie; Drew, Clifton; Ren, Yan; Wang, Maria; Carpenter, Aaron; Lee, Chung-Wein; Clayton, Thomas; Fang, Wei; Lu, Chin-Chun; Riley, Mariko; Abdubek, Polat; Blease, Kate; Hartke, James; Kumar, Gondi; Vessey, Rupert; Rolfe, Mark; Hamann, Lawrence G.; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">981-987</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. via small-mol. modulation of cereblon offers vast potential for the development of new therapeutics.  Cereblon-binding therapeutics carry the safety risks of thalidomide, which caused an epidemic of severe birth defects characterized by forelimb shortening or phocomelia.  Here we show that thalidomide is not teratogenic in transgenic mice expressing human cereblon, indicating that binding to cereblon is not sufficient to cause birth defects.  Instead, we identify SALL4 as a thalidomide-dependent cereblon neosubstrate.  Human mutations in SALL4 cause Duane-radial ray, IVIC, and acro-renal-ocular syndromes with overlapping clin. presentations to thalidomide embryopathy, including phocomelia.  SALL4 is degraded in rabbits but not in resistant organisms such as mice because of SALL4 sequence variations.  This work expands the scope of cereblon neosubstrate activity within the formerly 'undruggable' C2H2 zinc finger family and offers a path toward safer therapeutics through an improved understanding of the mol. basis of thalidomide-induced teratogenicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpePTE78HX85rVg90H21EOLACvtfcHk0lhCgIB2AbvnNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Kgu7nM&md5=7530c3850ed7a763b30cba44dab640ac</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0129-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0129-x%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DCouto%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DHui%26aufirst%3DJ.%26aulast%3DStamp%26aufirst%3DK.%26aulast%3DDrew%26aufirst%3DC.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCarpenter%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.-W.%26aulast%3DClayton%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DAbdubek%26aufirst%3DP.%26aulast%3DBlease%26aufirst%3DK.%26aulast%3DHartke%26aufirst%3DJ.%26aulast%3DKumar%26aufirst%3DG.%26aulast%3DVessey%26aufirst%3DR.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DSALL4%2520mediates%2520teratogenicity%2520as%2520a%2520thalidomide-dependent%2520cereblon%2520substrate%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D981%26epage%3D987%26doi%3D10.1038%2Fs41589-018-0129-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asatsuma-Okumura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Simone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asakawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span> <span> </span><span class="NLM_article-title">p63 is a cereblon substrate involved in thalidomide teratogenicity</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1084</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0366-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fs41589-019-0366-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=31591562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFaksbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1077-1084&author=T.+Asatsuma-Okumuraauthor=H.+Andoauthor=M.+De+Simoneauthor=J.+Yamamotoauthor=T.+Satoauthor=N.+Shimizuauthor=K.+Asakawaauthor=Y.+Yamaguchiauthor=T.+Itoauthor=L.+Guerriniauthor=H.+Handa&title=p63+is+a+cereblon+substrate+involved+in+thalidomide+teratogenicity&doi=10.1038%2Fs41589-019-0366-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">p63 is a cereblon substrate involved in thalidomide teratogenicity</span></div><div class="casAuthors">Asatsuma-Okumura, Tomoko; Ando, Hideki; De Simone, Marco; Yamamoto, Junichi; Sato, Tomomi; Shimizu, Nobuyuki; Asakawa, Kazuhide; Yamaguchi, Yuki; Ito, Takumi; Guerrini, Luisa; Handa, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1077-1084</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cereblon (CRBN) is a primary target of thalidomide and mediates its multiple pharmacol. activities, including teratogenic and antimyeloma activities.  CRBN functions as a substrate receptor of the E3 ubiquitin ligase CRL4, whose substrate specificity is modulated by thalidomide and its analogs.  Although a no. of CRL4CRBN substrates have recently been identified, the substrate involved in thalidomide teratogenicity is unclear.  Here we show that p63 isoforms are thalidomide-dependent CRL4CRBN neosubstrates that are responsible, at least in part, for its teratogenic effects.  The p53 family member p63 is assocd. with multiple developmental processes.  ΔNp63α is essential for limb development, while TAp63α is important for cochlea development and hearing.  Using a zebrafish model, we demonstrate that thalidomide exerts its teratogenic effects on pectoral fins and otic vesicles by inducing the degrdn. of ΔNp63α and TAp63α, resp.  These results may contribute to the invention of new thalidomide analogs lacking teratogenic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ4WsPMCzNSrVg90H21EOLACvtfcHk0lhvziWpjDa3QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFaksbfL&md5=c35c85968ace67e49978e40579e2706c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0366-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0366-7%26sid%3Dliteratum%253Aachs%26aulast%3DAsatsuma-Okumura%26aufirst%3DT.%26aulast%3DAndo%26aufirst%3DH.%26aulast%3DDe%2BSimone%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DJ.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DShimizu%26aufirst%3DN.%26aulast%3DAsakawa%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DGuerrini%26aufirst%3DL.%26aulast%3DHanda%26aufirst%3DH.%26atitle%3Dp63%2520is%2520a%2520cereblon%2520substrate%2520involved%2520in%2520thalidomide%2520teratogenicity%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D1077%26epage%3D1084%26doi%3D10.1038%2Fs41589-019-0366-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagner, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanillo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjorklund, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havens, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rychak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khambatta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amantangelo, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pourdehnad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakurta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span> <span> </span><span class="NLM_article-title">CC-122, a pleiotropic pathway modifier, mimics an interferon response and has antitumor activity in DLBCL</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1182/blood-2015-02-628669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1182%2Fblood-2015-02-628669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=26002965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFahtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=779-789&author=P.+R.+Hagnerauthor=H.-W.+Manauthor=C.+Fontanilloauthor=M.+Wangauthor=S.+Coutoauthor=M.+Breiderauthor=C.+Bjorklundauthor=C.+G.+Havensauthor=G.+Luauthor=E.+Rychakauthor=H.+Raymonauthor=R.+K.+Narlaauthor=L.+Barnesauthor=G.+Khambattaauthor=H.+Chiuauthor=J.+Kosekauthor=J.+Kangauthor=M.+D.+Amantangeloauthor=M.+Waldmanauthor=A.+Lopez-Gironaauthor=T.+Caiauthor=M.+Pourdehnadauthor=M.+Trotterauthor=T.+O.+Danielauthor=P.+H.+Schaferauthor=A.+Klippelauthor=A.+Thakurtaauthor=R.+Chopraauthor=A.+K.+Gandhi&title=CC-122%2C+a+pleiotropic+pathway+modifier%2C+mimics+an+interferon+response+and+has+antitumor+activity+in+DLBCL&doi=10.1182%2Fblood-2015-02-628669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CC-122, a pleiotropic pathway modifier, mimics an interferon response and has antitumor activity in DLBCL</span></div><div class="casAuthors">Hagner, Patrick R.; Man, Hon-Wah; Fontanillo, Celia; Wang, Maria; Couto, Suzana; Breider, Mike; Bjorklund, Chad; Havens, Courtney G.; Lu, Gang; Rychak, Emily; Raymon, Heather; Narla, Rama Krishna; Barnes, Leo; Khambatta, Gody; Chiu, Hsiling; Kosek, Jolanta; Kang, Jian; Amantangelo, Michael D.; Waldman, Michelle; Lopez-Girona, Antonia; Cai, Ti; Pourdehnad, Michael; Trotter, Matthew; Daniel, Thomas O.; Schafer, Peter H.; Klippel, Anke; Thakurta, Anjan; Chopra, Rajesh; Gandhi, Anita K.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">779-789</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Cereblon (CRBN), a substrate receptor of the Cullin 4 RING E3 ubiquitin ligase complex, is the target of the immunomodulatory drugs lenalidomide and pomalidomide.  Recently, it was demonstrated that binding of these drugs to CRBN promotes the ubiquitination and subsequent degrdn. of 2 common substrates, transcription factors Aiolos and Ikaros.  Here we report that CC-122, a new chem. entity termed pleiotropic pathway modifier, binds CRBN and promotes degrdn. of Aiolos and Ikaros in diffuse large B-cell lymphoma (DLBCL) and T cells in vitro, in vivo, and in patients, resulting in both cell autonomous as well as immunostimulatory effects.  In DLBCL cell lines, CC-122-induced degrdn. or short hairpin RNA-mediated knockdown of Aiolos and Ikaros correlates with increased transcription of interferon (IFN)-stimulated genes independent of IFN-α, -β, and -γ prodn. and/or secretion and results in apoptosis in both activated B-cell (ABC) and germinal center B-cell DLBCL cell lines.  Our results provide mechanistic insight into the cell-of-origin independent antilymphoma activity of CC-122, in contrast to the ABC subtype selective activity of lenalidomide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Ayj7KrKYWrVg90H21EOLACvtfcHk0lhvziWpjDa3QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFahtL7K&md5=786ea247e42ebc90dec67fc6f7a4e48b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2015-02-628669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2015-02-628669%26sid%3Dliteratum%253Aachs%26aulast%3DHagner%26aufirst%3DP.%2BR.%26aulast%3DMan%26aufirst%3DH.-W.%26aulast%3DFontanillo%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCouto%26aufirst%3DS.%26aulast%3DBreider%26aufirst%3DM.%26aulast%3DBjorklund%26aufirst%3DC.%26aulast%3DHavens%26aufirst%3DC.%2BG.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DRychak%26aufirst%3DE.%26aulast%3DRaymon%26aufirst%3DH.%26aulast%3DNarla%26aufirst%3DR.%2BK.%26aulast%3DBarnes%26aufirst%3DL.%26aulast%3DKhambatta%26aufirst%3DG.%26aulast%3DChiu%26aufirst%3DH.%26aulast%3DKosek%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DAmantangelo%26aufirst%3DM.%2BD.%26aulast%3DWaldman%26aufirst%3DM.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DT.%26aulast%3DPourdehnad%26aufirst%3DM.%26aulast%3DTrotter%26aufirst%3DM.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DKlippel%26aufirst%3DA.%26aulast%3DThakurta%26aufirst%3DA.%26aulast%3DChopra%26aufirst%3DR.%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26atitle%3DCC-122%252C%2520a%2520pleiotropic%2520pathway%2520modifier%252C%2520mimics%2520an%2520interferon%2520response%2520and%2520has%2520antitumor%2520activity%2520in%2520DLBCL%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D779%26epage%3D789%26doi%3D10.1182%2Fblood-2015-02-628669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhvziWpjDa3QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-based optimization of potent and cell-active inhibitors of the von Hippel-Lindau (VHL) E3 ubiquitin ligase: structure-activity relationships leading to the chemical probe (2S,4R)-1-((S)-2-(1-cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=599-618&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-based+optimization+of+potent+and+cell-active+inhibitors+of+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase%3A+structure-activity+relationships+leading+to+the+chemical+probe+%282S%2C4R%29-1-%28%28S%29-2-%281-cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyl%29-4-hydroxy-N-%284-%284-methylthiazol-5-yl%29benzyl%29pyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Group-Based Optimization of Potent and Cell-Active Inhibitors of the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase: Structure-Activity Relationships Leading to the Chemical Probe (2S,4R)-1-((S)-2-(1-Cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span></div><div class="casAuthors">Soares, Pedro; Gadd, Morgan S.; Frost, Julianty; Galdeano, Carles; Ellis, Lucy; Epemolu, Ola; Rocha, Sonia; Read, Kevin D.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">599-618</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau tumor suppressor protein is the substrate binding subunit of the VHL E3 ubiquitin ligase, which targets hydroxylated α subunit of hypoxia inducible factors (HIFs) for ubiquitination and subsequent proteasomal degrdn.  VHL is a potential target for treating anemia and ischemic diseases, motivating the development of inhibitors of the VHL:HIF-α protein-protein interaction.  Addnl., bifunctional proteolysis targeting chimeras (PROTACs) contg. a VHL ligand can hijack the E3 ligase activity to induce degrdn. of target proteins.  We report the structure-guided design and group-based optimization of a series of VHL inhibitors with low nanomolar potencies and improved cellular permeability.  Structure-activity relationships led to the discovery of potent inhibitors 10 and chem. probe VH298, with dissocn. consts. <100 nM, which induced marked HIF-1α intracellular stabilization.  Our study provides new chem. tools to probe the VHL-HIF pathways and new VHL ligands for next-generation PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQfJe2l2WgX7Vg90H21EOLACvtfcHk0ljwpKbq7SL4Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P&md5=1c1e7c976c2de9a9681773ff216fdfd7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-based%2520optimization%2520of%2520potent%2520and%2520cell-active%2520inhibitors%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%253A%2520structure-activity%2520relationships%2520leading%2520to%2520the%2520chemical%2520probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyl%2529-4-hydroxy-N-%25284-%25284-methylthiazol-5-yl%2529benzyl%2529pyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0ljwpKbq7SL4Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gütschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinebach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voell, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricelj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosič, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnakenburg, G.</span></span> <span> </span><span class="NLM_article-title">A facile synthesis of ligands for the von Hippel–Lindau E3 ligase</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2521</span>– <span class="NLM_lpage">2527</span>, <span class="refDoi"> DOI: 10.1055/s-0040-1707400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1055%2Fs-0040-1707400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVanu7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2020&pages=2521-2527&author=M.+G%C3%BCtschowauthor=C.+Steinebachauthor=S.+A.+Voellauthor=L.+P.+Vuauthor=A.+Briceljauthor=I.+Sosi%C4%8Dauthor=G.+Schnakenburg&title=A+facile+synthesis+of+ligands+for+the+von+Hippel%E2%80%93Lindau+E3+ligase&doi=10.1055%2Fs-0040-1707400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A facile synthesis of ligands for the von Hippel-Lindau E3 ligase</span></div><div class="casAuthors">Steinebach, Christian; Voell, Sabine Anna; Vu, Lan Phuong; Bricelj, Alesa; Sosic, Izidor; Schnakenburg, Gregor; Guetschow, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2521-2527</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">The proteolysis-targeting chimeras (PROTACs) have become an integral part of different stages of drug discovery.  This growing field, therefore, benefits from advancements in all segments of the design of these compds.  Herein, an efficient and optimized synthetic protocol to various von Hippel-Lindau (VHL) ligands is presented, which enables easy access to multigram quantities of these essential PROTAC building blocks.  Moreover, the elaborated synthesis represents a straightforward approach to further explore the chem. space of VHL ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyw4H4Rq7HfrVg90H21EOLACvtfcHk0ljwpKbq7SL4Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVanu7bN&md5=893543d726fa49e0ddb2a251c7efb792</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1055%2Fs-0040-1707400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0040-1707400%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DSteinebach%26aufirst%3DC.%26aulast%3DVoell%26aufirst%3DS.%2BA.%26aulast%3DVu%26aufirst%3DL.%2BP.%26aulast%3DBricelj%26aufirst%3DA.%26aulast%3DSosi%25C4%258D%26aufirst%3DI.%26aulast%3DSchnakenburg%26aufirst%3DG.%26atitle%3DA%2520facile%2520synthesis%2520of%2520ligands%2520for%2520the%2520von%2520Hippel%25E2%2580%2593Lindau%2520E3%2520ligase%26jtitle%3DSynthesis%26date%3D2020%26volume%3D52%26spage%3D2521%26epage%3D2527%26doi%3D10.1055%2Fs-0040-1707400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0li-d9ITAOjCDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kostic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Critical assessment of targeted protein degradation as a research tool and pharmacological modality</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2020.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.tips.2020.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=32222318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1Wmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=305-317&author=M.+Kosticauthor=L.+H.+Jones&title=Critical+assessment+of+targeted+protein+degradation+as+a+research+tool+and+pharmacological+modality&doi=10.1016%2Fj.tips.2020.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Critical Assessment of Targeted Protein Degradation as a Research Tool and Pharmacological Modality</span></div><div class="casAuthors">Kostic, Milka; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">305-317</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Small mols. continue to dominate drug discovery because of their ease of use, lower cost of manufg., and access to intracellular targets.  However, despite these advantages, small mols. are more likely to fail in clin. trials compared with biologicals and their development remains limited to a small subset of disease-relevant 'druggable' targets.  Targeted protein degrdn. has recently emerged as a novel pharmacol. modality that promises to overcome small mol. limitations while retaining their key advantages.  Here, we use a Strengths-Weaknesses-Opportunities-Threats (SWOT) framework to critically assess the current status of this rapidly evolving field.  We expect that degrader mols. are only the beginning of a range of novel targeting modalities that hijack existing endogenous cellular machineries to chem. redirect biol. targets and pathways.  Therefore, this piece may offer a roadmap for enhancing development of both degraders and related modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBLMTr8vJkrrVg90H21EOLACvtfcHk0li-d9ITAOjCDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1Wmu7c%253D&md5=46a57cafb2427eff7a0b8ab0067492c2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2020.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2020.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DKostic%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DCritical%2520assessment%2520of%2520targeted%2520protein%2520degradation%2520as%2520a%2520research%2520tool%2520and%2520pharmacological%2520modality%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2020%26volume%3D41%26spage%3D305%26epage%3D317%26doi%3D10.1016%2Fj.tips.2020.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span> <span> </span><span class="NLM_article-title">Insights into the molecular pathogenesis of activated B-cell-like diffuse large B-cell lymphoma and its therapeutic implications</span>. <i>Cancers (Basel)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.3390/cancers7020812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.3390%2Fcancers7020812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=26010601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfns1ahtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=811-822&author=G.+Lenz&title=Insights+into+the+molecular+pathogenesis+of+activated+B-cell-like+diffuse+large+B-cell+lymphoma+and+its+therapeutic+implications&doi=10.3390%2Fcancers7020812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Molecular Pathogenesis of Activated B-Cell-like Diffuse Large B-Cell Lymphoma and Its Therapeutic Implications</span></div><div class="casAuthors">Lenz Georg; Lenz Georg</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">811-22</span>
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    </div><div class="casAbstract">Within the last couple of years, the understanding of the molecular mechanisms that drive the pathogenesis of diffuse large B-cell lymphoma (DLBCL) has significantly improved.  Large-scale gene expression profiling studies have led to the discovery of several molecularly defined subtypes that are characterized by specific oncogene addictions and significant differences in their outcome.  Next generation sequencing efforts combined with RNA interference screens frequently identify crucial oncogenes that lead to constitutive activation of various signaling pathways that drive lymphomagenesis.  This review summarizes our current understanding of the molecular pathogenesis of the activated B-cell-like (ABC) DLBCL subtype that is characterized by poor prognosis.  A special emphasis is put on findings that might impact therapeutic strategies of affected patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuSkyEGfdySdts4TFFB11gfW6udTcc2eahYOsKdGLn27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfns1ahtA%253D%253D&md5=388cf80a8d7cda9b1828e47578bcf681</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fcancers7020812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers7020812%26sid%3Dliteratum%253Aachs%26aulast%3DLenz%26aufirst%3DG.%26atitle%3DInsights%2520into%2520the%2520molecular%2520pathogenesis%2520of%2520activated%2520B-cell-like%2520diffuse%2520large%2520B-cell%2520lymphoma%2520and%2520its%2520therapeutic%2520implications%26jtitle%3DCancers%2520%2528Basel%2529%26date%3D2015%26volume%3D7%26spage%3D811%26epage%3D822%26doi%3D10.3390%2Fcancers7020812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreiro, D. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komives, E. A.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of system control of NF-kappaB signaling by IkappaBalpha</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1567</span>, <span class="refDoi"> DOI: 10.1021/bi901948j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901948j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFantrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=1560-1567&author=D.+U.+Ferreiroauthor=E.+A.+Komives&title=Molecular+mechanisms+of+system+control+of+NF-kappaB+signaling+by+IkappaBalpha&doi=10.1021%2Fbi901948j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanisms of System Control of NF-κB Signaling by IκBα</span></div><div class="casAuthors">Ferreiro, Diego U.; Komives, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1560-1567</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The NF-κB family of transcription factors responds to inflammatory cytokines with rapid transcriptional activation and subsequent signal repression.  Much of the system control depends on the unique characteristics of its major inhibitor, IκBα, which appears to have folding dynamics that underlie the biophys. properties of its activity.  Theor. folding studies followed by expts. have shown that a portion of the ankyrin repeat domain of IκBα folds on binding.  In resting cells, IκBα is constantly being synthesized, but most of it is rapidly degraded, leaving only a very small pool of free IκBα.  Nearly all of the NF-κB is bound to IκBα, resulting in near-complete inhibition of nuclear localization and transcriptional activation.  Combined soln. biophys. measurements and quant. protein half-life measurements inside cells have allowed the authors to understand how the inhibition occurs, why IκBα can be degraded quickly in the free state but remain extremely stable in the bound state, and how signal activation and repression can be tuned by IκB folding dynamics.  This review summarizes results of in vitro and in vivo expts. that converge demonstrating the effective interplay between biophysics and cell biol. in understanding transcriptional control by the NF-κB signaling module.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdGq5LKH_dj7Vg90H21EOLACvtfcHk0lglcHbnvT_1aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFantrs%253D&md5=25dc6d77be6dcea8d8761dbf7c5cc8e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fbi901948j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901948j%26sid%3Dliteratum%253Aachs%26aulast%3DFerreiro%26aufirst%3DD.%2BU.%26aulast%3DKomives%26aufirst%3DE.%2BA.%26atitle%3DMolecular%2520mechanisms%2520of%2520system%2520control%2520of%2520NF-kappaB%2520signaling%2520by%2520IkappaBalpha%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D1560%26epage%3D1567%26doi%3D10.1021%2Fbi901948j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darricarrere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">HaloPROTACS: use of small molecule PROTACs to induce degradation of HaloTag fusion proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1831-1837&author=D.+L.+Buckleyauthor=K.+Rainaauthor=N.+Darricarrereauthor=J.+Hinesauthor=J.+L.+Gustafsonauthor=I.+E.+Smithauthor=A.+H.+Miahauthor=J.+D.+Harlingauthor=C.+M.+Crews&title=HaloPROTACS%3A+use+of+small+molecule+PROTACs+to+induce+degradation+of+HaloTag+fusion+proteins&doi=10.1021%2Facschembio.5b00442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">HaloPROTACS: Use of Small Molecule PROTACs to Induce Degradation of HaloTag Fusion Proteins</span></div><div class="casAuthors">Buckley, Dennis L.; Raina, Kanak; Darricarrere, Nicole; Hines, John; Gustafson, Jeffrey L.; Smith, Ian E.; Miah, Afjal H.; Harling, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1831-1837</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol.-induced protein degrdn. is an attractive strategy for the development of chem. probes.  One method for inducing targeted protein degrdn. involves the use of PROTACs, heterobifunctional mols. that can recruit specific E3 ligases to a desired protein of interest.  PROTACs have been successfully used to degrade numerous proteins in cells, but the peptidic E3 ligase ligands used in previous PROTACs have hindered their development into more mature chem. probes or therapeutics.  The authors report the design of a novel class of PROTACs that incorporate small mol. VHL ligands to successfully degrade HaloTag7 fusion proteins.  These HaloPROTACs will inspire the development of future PROTACs with more drug-like properties.  Addnl., these HaloPROTACs are useful chem. genetic tools, due to their ability to chem. knock down widely used HaloTag7 fusion proteins in a general fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotj-Sgj4TyKrVg90H21EOLACvtfcHk0lglcHbnvT_1aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN&md5=b074c58e983f15c384818777b25e248d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00442%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDarricarrere%26aufirst%3DN.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHaloPROTACS%253A%2520use%2520of%2520small%2520molecule%2520PROTACs%2520to%2520induce%2520degradation%2520of%2520HaloTag%2520fusion%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1831%26epage%3D1837%26doi%3D10.1021%2Facschembio.5b00442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of the IRAK1 kinase inhibitors and the co-crystal structure of <b>JH-I-25</b> (green) with IRAK1 kinase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BFN">6BFN</a>). The linker attachment point is highlighted by the white arrow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of IRAK1-targeting PROTACs and the chemical structure of <b>Degrader-1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Degrader-1<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1-(tetrahydro-2<i>H</i>-pyran-2-yl)-5-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-pyrazole, Pd(dppf)Cl<sub>2</sub>, Cs<sub>2</sub>CO<sub>3</sub>, dioxane/H<sub>2</sub>O. (b) (i) NaOH, MeOH/H<sub>2</sub>O. (ii) HCl, dioxane. (c) <i>tert</i>-Butyl piperazine-1-carboxylate, K<sub>2</sub>CO<sub>3</sub>, DMF. (d) H<sub>2</sub>, Pd/C, MeOH. (e) HATU, TEA, DMF. (f) HCl, EtOAc. (g) Methyl 3-bromopropanoate, TEA, KI, NMP, microwave. (h) NaOH, THF/H<sub>2</sub>O. (i) EDCI, HOBt, DIPEA, DMF.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IRAK1 protein degradation profile in HEK293T cells for <b>Degrader-1</b>, <b>Degrader-2</b>, and <b>Degrader-3</b>. (A) Degradation effects of <b>Degrader-1</b> on IRAK1 proteins in HEK293T cells by Western blotting after 24 h treatment by <b>Degrader-1</b> at illustrated concentrations. (B) Characterization of <b>Degrader-1</b> for its concentration- and time-dependent IRAK1 protein degradation by Western blot analysis. (C) <b>Degrader-2</b> and <b>Degrader-3</b> dose-dependently degrades IRAK1 protein after 24 h incubation. (D) Quantitation of DC<sub>50</sub> values of <b>Degrader-1</b>, <b>Degrader-2</b>, and <b>Degrader-3</b> after 24 h incubation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Degrader-2, Degrader-3, and Degrader-4<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>tert</i>-butyl 2-bromoacetate, NaH, THF; (b) Pd/C, H<sub>2</sub> (30 psi), MeOH; (c) HATU, DIPEA, DMF; (d) HCl (4 M in dioxane); and (e) EDCI, HOBt, DIPEA, DMF.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Mechanistic study of <b>Degrader-3</b> in HEK293T cells by Western blotting analysis. (A) <b>Degrader-3</b>-mediated IRAK1 degradation is abrogated by pretreatment with the proteasome inhibitor MG-132. (B) <b>Degrader-3</b>-mediated IRAK1 degradation is abrogated by pretreatment with the VHL ligand VH298.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. TREEspot interaction map of kinases for <b>JH-I-25</b>, <b>Degrader-3</b>, and <b>Degrader-4</b> from the KinomeScan competition binding assay profiled at 1 μM concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Volcano plots of abundance proteomics data of <b>Degrader-3</b> by proteomics analysis. Log fold change in abundance of protein with two unique peptide measurement using quantitative mass spectrometry after 6 h treatment of Toledo cells with 1 μM <b>Degrader-3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/medium/jm1c00103_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Characterization of <b>Degrader-3</b> and <b>Degrader-4</b> in ABC DLBCL cell lines. (A) DLBCL cell lines were treated with <b>Degrader-3</b> at the indicated concentrations for 24 h. IRAK1 protein levels were determined by Western blotting. (B) HBL-1 cells were treated with <b>Degrader-3</b> or its epimer <b>Degrader-4</b> at the indicated doses for 24 h. Phosphorylated IkBα and STAT3 protein levels were determined by Western blotting. (C) Cleaved PARP was determined by Western blotting. (D) Cell proliferative assay in HBL-1 and OCI-LY10 cells with MyD88 mutation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/0/jmcmar.ahead-of-print/acs.jmedchem.1c00103/20210719/images/large/jm1c00103_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00103&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32653" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32653" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 28 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Nardo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balka, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardona Gloria, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masters, S. L.</span></span> <span> </span><span class="NLM_article-title">Interleukin-1 receptor-associated kinase 4 (IRAK4) plays a dual role in myddosome formation and Toll-like receptor signaling</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">15195</span>– <span class="NLM_lpage">15207</span>, <span class="refDoi"> DOI: 10.1074/jbc.ra118.003314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1074%2Fjbc.RA118.003314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=30076215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVWhu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=15195-15207&author=D.+De%0ANardoauthor=K.+R.+Balkaauthor=Y.+Cardona+Gloriaauthor=V.+R.+Raoauthor=E.+Latzauthor=S.+L.+Masters&title=Interleukin-1+receptor-associated+kinase+4+%28IRAK4%29+plays+a+dual+role+in+myddosome+formation+and+Toll-like+receptor+signaling&doi=10.1074%2Fjbc.ra118.003314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Interleukin-1 receptor-associated kinase 4 (IRAK4) plays a dual role in myddosome formation and Toll-like receptor signaling</span></div><div class="casAuthors">De Nardo, Dominic; Balka, Katherine R.; Gloria, Yamel Cardona; Rao, Vikram R.; Latz, Eicke; Masters, Seth L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">15195-15207</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Toll-like receptors (TLRs) form part of the host innate immune system, in which they act as sensors of microbial and endogenous danger signals.  Upon TLR activation, the intracellular Toll/interleukin-1 receptor domains of TLR dimers initiate oligomerization of a multiprotein signaling platform comprising myeloid differentiation primary response 88 (MyD88) and members of the interleukin-1 receptor-assocd. kinase (IRAK) family.  Formation of this myddosome complex initiates signal transduction pathways, leading to the activation of transcription factors and the prodn. of inflammatory cytokines.  To date, little is known about the assembly and disassembly of the myddosome and about the mechanisms by which these complexes mediate multiple downstream signaling pathways.  Here, we isolated myddosome complexes from whole-cell lysates of TLR-activated primary mouse macrophages and from IRAK reporter macrophages to examine the kinetics of myddosome assembly and disassembly.  Using a selective inhibitor of IRAK4's kinase activity, we found that whereas TLR cytokine responses were ablated, myddosome formation was stabilized in the absence of IRAK4's kinase activity.  Of note, IRAK4 inhibition had only a minimal effect on NF-κB and mitogen-activated protein kinase (MAPK) signaling.  In summary, our results indicate that IRAK4 has a crit. scaffold function in myddosome formation and that its kinase activity is dispensable for myddosome assembly and activation of the NF-κB and MAPK pathways but is essential for MyD88-dependent prodn. of inflammatory cytokines.  Our findings suggest that the scaffold function of IRAK4 may be an attractive target for treating inflammatory and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNgAjazN469rVg90H21EOLACvtfcHk0ljLLk8CWQr3LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVWhu7zK&md5=9005c249071ba957ee8232d861d82610</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA118.003314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA118.003314%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BNardo%26aufirst%3DD.%26aulast%3DBalka%26aufirst%3DK.%2BR.%26aulast%3DCardona%2BGloria%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DV.%2BR.%26aulast%3DLatz%26aufirst%3DE.%26aulast%3DMasters%26aufirst%3DS.%2BL.%26atitle%3DInterleukin-1%2520receptor-associated%2520kinase%25204%2520%2528IRAK4%2529%2520plays%2520a%2520dual%2520role%2520in%2520myddosome%2520formation%2520and%2520Toll-like%2520receptor%2520signaling%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D15195%26epage%3D15207%26doi%3D10.1074%2Fjbc.ra118.003314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span> <span> </span><span class="NLM_article-title">Helical assembly in the MyD88-IRAK4-IRAK2 complex in TLR/IL-1R signalling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>465</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">890</span>, <span class="refDoi"> DOI: 10.1038/nature09121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnature09121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=20485341" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt1eku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2010&pages=885-890&author=S.-C.+Linauthor=Y.-C.+Loauthor=H.+Wu&title=Helical+assembly+in+the+MyD88-IRAK4-IRAK2+complex+in+TLR%2FIL-1R+signalling&doi=10.1038%2Fnature09121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Helical assembly in the MyD88-IRAK4-IRAK2 complex in TLR/IL-1R signalling</span></div><div class="casAuthors">Lin, Su-Chang; Lo, Yu-Chih; Wu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">7300</span>),
    <span class="NLM_cas:pages">885-890</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">MyD88, IRAK4 and IRAK2 are crit. signaling mediators of the TLR/IL1-R superfamily.  Here the authors report the crystal structure of the MyD88-IRAK4-IRAK2 death domain (DD) complex, which surprisingly reveals a left-handed helical oligomer that consists of 6 MyD88, 4 IRAK4 and 4 IRAK2 DDs.  Assembly of this helical signaling tower is hierarchical, in which MyD88 recruits IRAK4 and the MyD88-IRAK4 complex recruits the IRAK4 substrates IRAK2 or the related IRAK1.  Formation of these Myddosome complexes brings the kinase domains of IRAKs into proximity for phosphorylation and activation.  Composite binding sites are required for recruitment of the individual DDs in the complex, which are confirmed by mutagenesis and previously identified signaling mutations.  Specificities in Myddosome formation are dictated by both mol. complementarity and correspondence of surface electrostatics.  The MyD88-IRAK4-IRAK2 complex provides a template for Toll signaling in Drosophila and an elegant mechanism for versatile assembly and regulation of DD complexes in signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8zWmiKDKlVbVg90H21EOLACvtfcHk0ljLLk8CWQr3LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt1eku70%253D&md5=af8b6ea34425853777839fd0e6af9602</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature09121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09121%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DS.-C.%26aulast%3DLo%26aufirst%3DY.-C.%26aulast%3DWu%26aufirst%3DH.%26atitle%3DHelical%2520assembly%2520in%2520the%2520MyD88-IRAK4-IRAK2%2520complex%2520in%2520TLR%252FIL-1R%2520signalling%26jtitle%3DNature%26date%3D2010%26volume%3D465%26spage%3D885%26epage%3D890%26doi%3D10.1038%2Fnature09121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roschewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. H.</span></span> <span> </span><span class="NLM_article-title">Diffuse large B-cell lymphoma-treatment approaches in the molecular era</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1038/nrclinonc.2013.197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnrclinonc.2013.197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=24217204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyrsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2014&pages=12-23&author=M.+Roschewskiauthor=L.+M.+Staudtauthor=W.+H.+Wilson&title=Diffuse+large+B-cell+lymphoma-treatment+approaches+in+the+molecular+era&doi=10.1038%2Fnrclinonc.2013.197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Diffuse large B-cell lymphoma-treatment approaches in the molecular era</span></div><div class="casAuthors">Roschewski, Mark; Staudt, Louis M.; Wilson, Wyndham H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-23</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Diffuse large B-cell lymphoma (DLBCL) is an aggressive B-cell non-Hodgkin lymphoma that affects patients of all ages with a wide range of clin. presentations.  Although DLBCL is curable even in advanced stages, up to one-third of patients will not achieve cure with initial therapy.  In the modern era of rituximab-based therapy as the first-line treatment, the prognoses of patients who require salvage therapy are poor and most will eventually succumb to their disease.  Insight into the complex mol. circuitry of DLBCL reveals a diverse range of somatic mutations and aberrant intracellular signalling pathways that characterize distinct mol. subsets of the disease.  The next major breakthrough in DLBCL therapy during this 'mol. era' of disease definition will be the identification of combinations of novel agents that target the oncogenic drivers of these subsets.  Well-conducted clin. trials, with translational mol. investigations, will be essential to achieve the goal of precision medicine and expand the no. of patients with DLBCL who achieve a cure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBpteNHDsYabVg90H21EOLACvtfcHk0ljLLk8CWQr3LA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyrsbzL&md5=af716d22a25172e77cb3c636cc5aedec</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2013.197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2013.197%26sid%3Dliteratum%253Aachs%26aulast%3DRoschewski%26aufirst%3DM.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26aulast%3DWilson%26aufirst%3DW.%2BH.%26atitle%3DDiffuse%2520large%2520B-cell%2520lymphoma-treatment%2520approaches%2520in%2520the%2520molecular%2520era%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2014%26volume%3D11%26spage%3D12%26epage%3D23%26doi%3D10.1038%2Fnrclinonc.2013.197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhavar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlhammer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romesser, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenwald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Hermelink, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connors, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimsza, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smeland, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braziel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubbs, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisenburger, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">Oncogenically active MYD88 mutations in human lymphoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>470</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1038/nature09671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnature09671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=21179087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVCksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=470&publication_year=2011&pages=115-119&author=V.+N.+Ngoauthor=R.+M.+Youngauthor=R.+Schmitzauthor=S.+Jhavarauthor=W.+Xiaoauthor=K.-H.+Limauthor=H.+Kohlhammerauthor=W.+Xuauthor=Y.+Yangauthor=H.+Zhaoauthor=A.+L.+Shafferauthor=P.+Romesserauthor=G.+Wrightauthor=J.+Powellauthor=A.+Rosenwaldauthor=H.+K.+Muller-Hermelinkauthor=G.+Ottauthor=R.+D.+Gascoyneauthor=J.+M.+Connorsauthor=L.+M.+Rimszaauthor=E.+Campoauthor=E.+S.+Jaffeauthor=J.+Delabieauthor=E.+B.+Smelandauthor=R.+I.+Fisherauthor=R.+M.+Brazielauthor=R.+R.+Tubbsauthor=J.+R.+Cookauthor=D.+D.+Weisenburgerauthor=W.+C.+Chanauthor=L.+M.+Staudt&title=Oncogenically+active+MYD88+mutations+in+human+lymphoma&doi=10.1038%2Fnature09671"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenically active MYD88 mutations in human lymphoma</span></div><div class="casAuthors">Ngo, Vu N.; Young, Ryan M.; Schmitz, Roland; Jhavar, Sameer; Xiao, Wenming; Lim, Kian-Huat; Kohlhammer, Holger; Xu, Weihong; Yang, Yandan; Zhao, Hong; Shaffer, Arthur L.; Romesser, Paul; Wright, George; Powell, John; Rosenwald, Andreas; Muller-Hermelink, Hans Konrad; Ott, German; Gascoyne, Randy D.; Connors, Joseph M.; Rimsza, Lisa M.; Campo, Elias; Jaffe, Elaine S.; Delabie, Jan; Smeland, Erlend B.; Fisher, Richard I.; Braziel, Rita M.; Tubbs, Raymond R.; Cook, J. R.; Weisenburger, Denny D.; Chan, Wing C.; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">470</span>
        (<span class="NLM_cas:issue">7332</span>),
    <span class="NLM_cas:pages">115-119</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The activated B-cell-like (ABC) subtype of diffuse large B-cell lymphoma (DLBCL) remains the least curable form of this malignancy despite recent advances in therapy.  Constitutive nuclear factor (NF)-κB and JAK kinase signaling promotes malignant cell survival in these lymphomas, but the genetic basis for this signaling is incompletely understood.  Here we describe the dependence of ABC DLBCLs on MYD88, an adaptor protein that mediates toll and interleukin (IL)-1 receptor signaling, and the discovery of highly recurrent oncogenic mutations affecting MYD88 in ABC DLBCL tumors.  RNA interference screening revealed that MYD88 and the assocd. kinases IRAK1 and IRAK4 are essential for ABC DLBCL survival.  High-throughput RNA resequencing uncovered MYD88 mutations in ABC DLBCL lines.  Notably, 29% of ABC DLBCL tumors harbored the same amino acid substitution, L265P, in the MYD88 Toll/IL-1 receptor (TIR) domain at an evolutionarily invariant residue in its hydrophobic core.  This mutation was rare or absent in other DLBCL subtypes and Burkitt's lymphoma, but was obsd. in 9% of mucosa-assocd. lymphoid tissue lymphomas.  At a lower frequency, addnl. mutations were obsd. in the MYD88 TIR domain, occurring in both the ABC and germinal center B-cell-like (GCB) DLBCL subtypes.  Survival of ABC DLBCL cells bearing the L265P mutation was sustained by the mutant but not the wild-type MYD88 isoform, demonstrating that L265P is a gain-of-function driver mutation.  The L265P mutant promoted cell survival by spontaneously assembling a protein complex contg. IRAK1 and IRAK4, leading to IRAK4 kinase activity, IRAK1 phosphorylation, NF-κB signaling, JAK kinase activation of STAT3, and secretion of IL-6, IL-10 and interferon-β.  Hence, the MYD88 signaling pathway is integral to the pathogenesis of ABC DLBCL, supporting the development of inhibitors of IRAK4 kinase and other components of this pathway for the treatment of tumors bearing oncogenic MYD88 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5pEPxh1G6J7Vg90H21EOLACvtfcHk0ljBz7CLpre_Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVCksQ%253D%253D&md5=3dff4d54af8f64bb9e57f04cf274fee8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature09671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09671%26sid%3Dliteratum%253Aachs%26aulast%3DNgo%26aufirst%3DV.%2BN.%26aulast%3DYoung%26aufirst%3DR.%2BM.%26aulast%3DSchmitz%26aufirst%3DR.%26aulast%3DJhavar%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DLim%26aufirst%3DK.-H.%26aulast%3DKohlhammer%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DRomesser%26aufirst%3DP.%26aulast%3DWright%26aufirst%3DG.%26aulast%3DPowell%26aufirst%3DJ.%26aulast%3DRosenwald%26aufirst%3DA.%26aulast%3DMuller-Hermelink%26aufirst%3DH.%2BK.%26aulast%3DOtt%26aufirst%3DG.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DConnors%26aufirst%3DJ.%2BM.%26aulast%3DRimsza%26aufirst%3DL.%2BM.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DDelabie%26aufirst%3DJ.%26aulast%3DSmeland%26aufirst%3DE.%2BB.%26aulast%3DFisher%26aufirst%3DR.%2BI.%26aulast%3DBraziel%26aufirst%3DR.%2BM.%26aulast%3DTubbs%26aufirst%3DR.%2BR.%26aulast%3DCook%26aufirst%3DJ.%2BR.%26aulast%3DWeisenburger%26aufirst%3DD.%2BD.%26aulast%3DChan%26aufirst%3DW.%2BC.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DOncogenically%2520active%2520MYD88%2520mutations%2520in%2520human%2520lymphoma%26jtitle%3DNature%26date%3D2011%26volume%3D470%26spage%3D115%26epage%3D119%26doi%3D10.1038%2Fnature09671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Z.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavine, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Rosario, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philippar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span> <span> </span><span class="NLM_article-title">Assessing IRAK4 functions in ABC DLBCL by IRAK4 kinase inhibition and protein degradation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1500</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2020.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.chembiol.2020.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=32888499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsl2it7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=1500-1509&author=J.+Zhangauthor=L.+Fuauthor=B.+Shenauthor=Y.+Liuauthor=W.+Wangauthor=X.+Caiauthor=L.+Kongauthor=Y.+Yanauthor=R.+Mengauthor=Z.+Zhangauthor=Y.-N.+P.+Chenauthor=Q.+Liuauthor=Z.-K.+Wanauthor=T.+Zhouauthor=X.+Wangauthor=P.+Gavineauthor=A.+Del+Rosarioauthor=K.+Ahnauthor=U.+Philipparauthor=R.+Attarauthor=J.+Yangauthor=Y.+Xuauthor=J.+P.+Edwardsauthor=X.+Dai&title=Assessing+IRAK4+functions+in+ABC+DLBCL+by+IRAK4+kinase+inhibition+and+protein+degradation&doi=10.1016%2Fj.chembiol.2020.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing IRAK4 Functions in ABC DLBCL by IRAK4 Kinase Inhibition and Protein Degradation</span></div><div class="casAuthors">Zhang, Jing; Fu, Liqiang; Shen, Bin; Liu, Yingtao; Wang, Wenqian; Cai, Xin; Kong, Linglong; Yan, Yilin; Meng, Ryan; Zhang, Zhuming; Chen, Ying-Nan P.; Liu, Qian; Wan, Zhao-Kui; Zhou, Tianyuan; Wang, Xiaotao; Gavine, Paul; Del Rosario, Amanda; Ahn, Kay; Philippar, Ulrike; Attar, Ricardo; Yang, Jennifer; Xu, Yanping; Edwards, James P.; Dai, Xuedong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1500-1509.e13</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The interleukin-1 receptor-activated kinase 4 (IRAK4) belongs to the IRAK family of serine/threonine kinases and plays a central role in the innate immune response.  However, the function of IRAK4 in tumor growth and progression remains elusive.  Here we sought to det. the enzymic and scaffolding functions of IRAK4 in activated B-cell-like diffuse large B cell lymphoma (ABC DLBCL).  We chose a highly selective IRAK4 kinase inhibitor to probe the biol. effects of kinase inhibition and developed a series of IRAK4 degraders to evaluate the effects of protein degrdn. in ABC DLBCL cells.  Interestingly, the results demonstrated that neither IRAK4 kinase inhibition nor protein degrdn. led to cell death or growth inhibition, suggesting a redundant role for IRAK4 in ABC DLBCL cell survival.  IRAK4 degraders characterized in this study provide useful tools for understanding IRAK4 protein scaffolding function, which was previously unachievable using pharmacol. perturbation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-fxMUd7pQEbVg90H21EOLACvtfcHk0ljBz7CLpre_Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsl2it7fJ&md5=0d1f9d102b101c3da4aa2be668ed7851</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2020.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2020.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DCai%26aufirst%3DX.%26aulast%3DKong%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.-N.%2BP.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWan%26aufirst%3DZ.-K.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DGavine%26aufirst%3DP.%26aulast%3DDel%2BRosario%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DK.%26aulast%3DPhilippar%26aufirst%3DU.%26aulast%3DAttar%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DEdwards%26aufirst%3DJ.%2BP.%26aulast%3DDai%26aufirst%3DX.%26atitle%3DAssessing%2520IRAK4%2520functions%2520in%2520ABC%2520DLBCL%2520by%2520IRAK4%2520kinase%2520inhibition%2520and%2520protein%2520degradation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D1500%26epage%3D1509%26doi%3D10.1016%2Fj.chembiol.2020.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roulland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasbekar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. L.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceribelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palisoc, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valadez, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies-Hill, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenwald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delabie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimsza, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estephan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdhoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruhlak, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittaluga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oellerich, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>560</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">391</span>, <span class="refDoi"> DOI: 10.1038/s41586-018-0290-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fs41586-018-0290-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=29925955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=560&publication_year=2018&pages=387-391&author=J.+D.+Phelanauthor=R.+M.+Youngauthor=D.+E.+Websterauthor=S.+Roullandauthor=G.+W.+Wrightauthor=M.+Kasbekarauthor=A.+L.+Shafferauthor=M.+Ceribelliauthor=J.+Q.+Wangauthor=R.+Schmitzauthor=M.+Nakagawaauthor=E.+Bachyauthor=D.+W.+Huangauthor=Y.+Jiauthor=L.+Chenauthor=Y.+Yangauthor=H.+Zhaoauthor=X.+Yuauthor=W.+Xuauthor=M.+M.+Palisocauthor=R.+R.+Valadezauthor=T.+Davies-Hillauthor=W.+H.+Wilsonauthor=W.+C.+Chanauthor=E.+S.+Jaffeauthor=R.+D.+Gascoyneauthor=E.+Campoauthor=A.+Rosenwaldauthor=G.+Ottauthor=J.+Delabieauthor=L.+M.+Rimszaauthor=F.+J.+Rodriguezauthor=F.+Estephanauthor=M.+Holdhoffauthor=M.+J.+Kruhlakauthor=S.+M.+Hewittauthor=C.+J.+Thomasauthor=S.+Pittalugaauthor=T.+Oellerichauthor=L.+M.+Staudt&title=A+multiprotein+supercomplex+controlling+oncogenic+signalling+in+lymphoma&doi=10.1038%2Fs41586-018-0290-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A multiprotein supercomplex controlling oncogenic signalling in lymphoma</span></div><div class="casAuthors">Phelan, James D.; Young, Ryan M.; Webster, Daniel E.; Roulland, Sandrine; Wright, George W.; Kasbekar, Monica; Shaffer, IArthur L., III; Ceribelli, Michele; Wang, James Q.; Schmitz, Roland; Nakagawa, Masao; Bachy, Emmanuel; Huang, Da Wei; Ji, Yanlong; Chen, Lu; Yang, Yandan; Zhao, Hong; Yu, Xin; Xu, Weihong; Palisoc, Maryknoll M.; Valadez, Racquel R.; Davies-Hill, Theresa; Wilson, Wyndham H.; Chan, Wing C.; Jaffe, Elaine S.; Gascoyne, Randy D.; Campo, Elias; Rosenwald, Andreas; Ott, German; Delabie, Jan; Rimsza, Lisa M.; Rodriguez, Fausto J.; Estephan, Fayez; Holdhoff, Matthias; Kruhlak, Michael J.; Hewitt, Stephen M.; Thomas, Craig J.; Pittaluga, Stefania; Oellerich, Thomas; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">560</span>
        (<span class="NLM_cas:issue">7718</span>),
    <span class="NLM_cas:pages">387-391</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">B cell receptor (BCR) signaling has emerged as a therapeutic target in B cell lymphomas, but inhibiting this pathway in diffuse large B cell lymphoma (DLBCL) has benefited only a subset of patients1.  Gene expression profiling identified two major DLBCL subtypes, known as germinal center (GC) B cell-like (GCB) and activated B cell-like (ABC)2,3, with inferior outcomes following immunochemotherapy in ABC.  Autoantigens drive BCR-dependent activation of NF-κB in ABC DLBCL through a kinase cascade of SYK, BTK and PKCβ to promote the assembly of the CARD11-BCL10-MALT1 (CBM) adapter complex that recruits and activates IκB kinase (IKK)4-6.  Genome sequencing revealed gain-of-function mutations targeting the CD79A and CD79B BCR subunits and the Toll-like receptor (TLR) signaling adapter MYD885,7, with MYD88L265P being the most prevalent isoform.  In a clin. trial, the BTK inhibitor, ibrutinib, produced responses in 37% of ABC cases1.  The most striking response rate (80%) was obsd. in tumors with both CD79B and MYD88L265P mutations, but how these mutations cooperate to promote dependence on BCR signaling remains unclear.  Herein, we used genome-wide CRISPR-Cas9 screening and functional proteomics to understand the mol. basis of exceptional clin. responses to ibrutinib.  We discovered a new mode of oncogenic BCR signaling in ibrutinib-responsive cell lines and biopsies, coordinated by a multiprotein supercomplex formed by MYD88, TLR9, and the BCR (My-T-BCR).  The My-T-BCR co-localizes with mTOR on endolysosomes, where it drives pro-survival NF-κB and mTOR signaling.  Inhibitors of BCR and mTOR signaling cooperatively decreased My-T-BCR supercomplex formation and function, providing mechanistic insight into their synergistic toxicity for My-T-BCR+ DLBCL cells.  My-T-BCR complexes characterized ibrutinib-responsive malignancies and distinguished ibrutinib responders from non-responders.  Our data provide a roadmap for the rational deployment of oncogenic signaling inhibitors in molecularly-defined subsets of DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmeclQGnynR7Vg90H21EOLACvtfcHk0ljhpN0OgND70Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsL7F&md5=86543a2209046fea5eac50fa9a0aade3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs41586-018-0290-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-018-0290-0%26sid%3Dliteratum%253Aachs%26aulast%3DPhelan%26aufirst%3DJ.%2BD.%26aulast%3DYoung%26aufirst%3DR.%2BM.%26aulast%3DWebster%26aufirst%3DD.%2BE.%26aulast%3DRoulland%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DG.%2BW.%26aulast%3DKasbekar%26aufirst%3DM.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DCeribelli%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BQ.%26aulast%3DSchmitz%26aufirst%3DR.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DBachy%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DD.%2BW.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DPalisoc%26aufirst%3DM.%2BM.%26aulast%3DValadez%26aufirst%3DR.%2BR.%26aulast%3DDavies-Hill%26aufirst%3DT.%26aulast%3DWilson%26aufirst%3DW.%2BH.%26aulast%3DChan%26aufirst%3DW.%2BC.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DCampo%26aufirst%3DE.%26aulast%3DRosenwald%26aufirst%3DA.%26aulast%3DOtt%26aufirst%3DG.%26aulast%3DDelabie%26aufirst%3DJ.%26aulast%3DRimsza%26aufirst%3DL.%2BM.%26aulast%3DRodriguez%26aufirst%3DF.%2BJ.%26aulast%3DEstephan%26aufirst%3DF.%26aulast%3DHoldhoff%26aufirst%3DM.%26aulast%3DKruhlak%26aufirst%3DM.%2BJ.%26aulast%3DHewitt%26aufirst%3DS.%2BM.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DPittaluga%26aufirst%3DS.%26aulast%3DOellerich%26aufirst%3DT.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DA%2520multiprotein%2520supercomplex%2520controlling%2520oncogenic%2520signalling%2520in%2520lymphoma%26jtitle%3DNature%26date%3D2018%26volume%3D560%26spage%3D387%26epage%3D391%26doi%3D10.1038%2Fs41586-018-0290-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffitt, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Love, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leppa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasanen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meriranta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karjalainen-Lindsberg, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nørgaard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gang, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Høgdall, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heavican, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lone, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Healy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedoriw, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernal-Mizrachi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koff, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staton, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flowers, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paltiel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldschmidt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calaminici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clear, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gribben, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czader, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ondrejka, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsi, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Au-Yeung, R. K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, W. W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evens, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilichowska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadburn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rempel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Happ, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dave, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopalan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dave, S. S.</span></span> <span> </span><span class="NLM_article-title">Genetic and functional drivers of diffuse large B cell lymphoma</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">494</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.09.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.cell.2017.09.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=28985567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1WqsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=481-494&author=A.+Reddyauthor=J.+Zhangauthor=N.+S.+Davisauthor=A.+B.+Moffittauthor=C.+L.+Loveauthor=A.+Waldropauthor=S.+Leppaauthor=A.+Pasanenauthor=L.+Merirantaauthor=M.-L.+Karjalainen-Lindsbergauthor=P.+N%C3%B8rgaardauthor=M.+Pedersenauthor=A.+O.+Gangauthor=E.+H%C3%B8gdallauthor=T.+B.+Heavicanauthor=W.+Loneauthor=J.+Iqbalauthor=Q.+Qinauthor=G.+Liauthor=S.+Y.+Kimauthor=J.+Healyauthor=K.+L.+Richardsauthor=Y.+Fedoriwauthor=L.+Bernal-Mizrachiauthor=J.+L.+Koffauthor=A.+D.+Statonauthor=C.+R.+Flowersauthor=O.+Paltielauthor=N.+Goldschmidtauthor=M.+Calaminiciauthor=A.+Clearauthor=J.+Gribbenauthor=E.+Nguyenauthor=M.+B.+Czaderauthor=S.+L.+Ondrejkaauthor=A.+Collieauthor=E.+D.+Hsiauthor=E.+Tseauthor=R.+K.+H.+Au-Yeungauthor=Y.-L.+Kwongauthor=G.+Srivastavaauthor=W.+W.+L.+Choiauthor=A.+M.+Evensauthor=M.+Pilichowskaauthor=M.+Sengarauthor=N.+Reddyauthor=S.+Liauthor=A.+Chadburnauthor=L.+I.+Gordonauthor=E.+S.+Jaffeauthor=S.+Levyauthor=R.+Rempelauthor=T.+Tzengauthor=L.+E.+Happauthor=T.+Daveauthor=D.+Rajagopalanauthor=J.+Dattaauthor=D.+B.+Dunsonauthor=S.+S.+Dave&title=Genetic+and+functional+drivers+of+diffuse+large+B+cell+lymphoma&doi=10.1016%2Fj.cell.2017.09.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic and functional drivers of diffuse large-cell B-cell lymphoma</span></div><div class="casAuthors">Reddy, Anupama; Zhang, Jenny; Davis, Nicholas S.; Moffitt, Andrea B.; Love, Cassandra L.; Waldrop, Alexander; Leppa, Sirpa; Pasanen, Annika; Meriranta, Leo; Karjalainen-Lindsberg, Marja-Liisa; Noergaard, Peter; Pedersen, Mette; Gang, Anne O.; Hoegdall, Estrid; Heavican, Tayla B.; Lone, Waseem; Iqbal, Javeed; Qin, Qiu; Li, Guojie; Kim, So Young; Healy, Jane; Richards, Kristy L.; Fedoriw, Yuri; Bernal-Mizrachi, Leon; Koff, Jean L.; Staton, Ashley D.; Flowers, Christopher R.; Paltiel, Ora; Goldschmidt, Neta; Calaminici, Maria; Clear, Andrew; Gribben, John; Nguyen, Evelyn; Czader, Magdalena B.; Ondrejka, Sarah L.; Collie, Angela; Hsi, Eric D.; Tse, Eric; Au-Yeung, Rex K. H.; Kwong, Yok-Lam; Srivastava, Gopesh; Choi, William W. L.; Evens, Andrew M.; Pilichowska, Monika; Sengar, Manju; Reddy, Nishitha; Li, Shaoying; Chadburn, Amy; Gordon, Leo I.; Jaffe, Elaine S.; Levy, Shawn; Rempel, Rachel; Tzeng, Tiffany; Happ, Lanie E.; Dave, Tushar; Rajagopalan, Deepthi; Datta, Jyotishka; Dunson, David B.; Dave, Sandeep S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">481-494.e15</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Diffuse large B cell lymphoma (DLBCL) is the most common form of blood cancer and is characterized by a striking degree of genetic and clin. heterogeneity.  This heterogeneity poses a major barrier to understanding the genetic basis of the disease and its response to therapy.  Here, we performed an integrative anal. of whole-exome sequencing and transcriptome sequencing in a cohort of 1001 DLBCL patients to comprehensively define the landscape of 150 genetic drivers of the disease.  We characterized the functional impact of these genes using an unbiased CRISPR screen of DLBCL cell lines to define oncogenes that promote cell growth.  A prognostic model comprising these genetic alterations outperformed current established methods: cell of origin, the International Prognostic Index comprising clin. variables, and dual MYC and BCL2 expression.  These results comprehensively define the genetic drivers and their functional roles in DLBCL to identify new therapeutic opportunities in the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs_9SXwsqp-rVg90H21EOLACvtfcHk0ljhpN0OgND70Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1WqsrbE&md5=8dc716cdd54891b35ca98361c3d3c64d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.09.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.09.027%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DN.%2BS.%26aulast%3DMoffitt%26aufirst%3DA.%2BB.%26aulast%3DLove%26aufirst%3DC.%2BL.%26aulast%3DWaldrop%26aufirst%3DA.%26aulast%3DLeppa%26aufirst%3DS.%26aulast%3DPasanen%26aufirst%3DA.%26aulast%3DMeriranta%26aufirst%3DL.%26aulast%3DKarjalainen-Lindsberg%26aufirst%3DM.-L.%26aulast%3DN%25C3%25B8rgaard%26aufirst%3DP.%26aulast%3DPedersen%26aufirst%3DM.%26aulast%3DGang%26aufirst%3DA.%2BO.%26aulast%3DH%25C3%25B8gdall%26aufirst%3DE.%26aulast%3DHeavican%26aufirst%3DT.%2BB.%26aulast%3DLone%26aufirst%3DW.%26aulast%3DIqbal%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DHealy%26aufirst%3DJ.%26aulast%3DRichards%26aufirst%3DK.%2BL.%26aulast%3DFedoriw%26aufirst%3DY.%26aulast%3DBernal-Mizrachi%26aufirst%3DL.%26aulast%3DKoff%26aufirst%3DJ.%2BL.%26aulast%3DStaton%26aufirst%3DA.%2BD.%26aulast%3DFlowers%26aufirst%3DC.%2BR.%26aulast%3DPaltiel%26aufirst%3DO.%26aulast%3DGoldschmidt%26aufirst%3DN.%26aulast%3DCalaminici%26aufirst%3DM.%26aulast%3DClear%26aufirst%3DA.%26aulast%3DGribben%26aufirst%3DJ.%26aulast%3DNguyen%26aufirst%3DE.%26aulast%3DCzader%26aufirst%3DM.%2BB.%26aulast%3DOndrejka%26aufirst%3DS.%2BL.%26aulast%3DCollie%26aufirst%3DA.%26aulast%3DHsi%26aufirst%3DE.%2BD.%26aulast%3DTse%26aufirst%3DE.%26aulast%3DAu-Yeung%26aufirst%3DR.%2BK.%2BH.%26aulast%3DKwong%26aufirst%3DY.-L.%26aulast%3DSrivastava%26aufirst%3DG.%26aulast%3DChoi%26aufirst%3DW.%2BW.%2BL.%26aulast%3DEvens%26aufirst%3DA.%2BM.%26aulast%3DPilichowska%26aufirst%3DM.%26aulast%3DSengar%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChadburn%26aufirst%3DA.%26aulast%3DGordon%26aufirst%3DL.%2BI.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DLevy%26aufirst%3DS.%26aulast%3DRempel%26aufirst%3DR.%26aulast%3DTzeng%26aufirst%3DT.%26aulast%3DHapp%26aufirst%3DL.%2BE.%26aulast%3DDave%26aufirst%3DT.%26aulast%3DRajagopalan%26aufirst%3DD.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DDunson%26aufirst%3DD.%2BB.%26aulast%3DDave%26aufirst%3DS.%2BS.%26atitle%3DGenetic%2520and%2520functional%2520drivers%2520of%2520diffuse%2520large%2520B%2520cell%2520lymphoma%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D481%26epage%3D494%26doi%3D10.1016%2Fj.cell.2017.09.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ficarro, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marto, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a selective, covalent IRAK1 inhibitor with antiproliferative activity in MYD88 mutated B-cell lymphoma</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2238</span>– <span class="NLM_lpage">2243</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00378</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00378" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVSrsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2238-2243&author=J.+M.+Hatcherauthor=G.+Yangauthor=L.+Wangauthor=S.+B.+Ficarroauthor=S.+Buhrlageauthor=H.+Wuauthor=J.+A.+Martoauthor=S.+P.+Treonauthor=N.+S.+Gray&title=Discovery+of+a+selective%2C+covalent+IRAK1+inhibitor+with+antiproliferative+activity+in+MYD88+mutated+B-cell+lymphoma&doi=10.1021%2Facsmedchemlett.0c00378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective, Covalent IRAK1 Inhibitor with Antiproliferative Activity in MYD88 Mutated B-Cell Lymphoma</span></div><div class="casAuthors">Hatcher, John M.; Yang, Guang; Wang, Li; Ficarro, Scott B.; Buhrlage, Sara; Wu, Hao; Marto, Jarrod A.; Treon, Steven P.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2238-2243</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Interleukin 1 (IL-1) receptor-assocd. kinases (IRAKs) are serine/threonine kinases that play crit. roles in initiating the innate immune response against foreign pathogens.  Addnl., dysregulation of IRAK1 signaling plays a role in neoplastic disorders.  For example, IRAK1 was shown to be important for survival and proliferation in many B-cell lymphomas, including Waldenstr.ovrddot.om's macroglobulinemia (WM) and ABC subtype Diffused Large B-cell Lymphoma (DLBCL) cells.  Here, we report the discovery of a highly potent and selective covalent inhibitor of IRAK1, JH-X-119-01.  Intact protein MS labeling studies confirmed that JH-X-119-01 irreversibly labels IRAK1 at C302.  This compd. exhibited cytotoxic activity at single digit micromolar concns. in a panel of WM, DLBCL, and lymphoma cell lines expressing MYD88.  Cotreatment of JH-X-119-01 with the BTK inhibitor ibrutinib resulted in synergistic killing effects in these systems.  Taken together, JH-X-119-01 represents a highly selective probe of IRAK1 for further development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNZtBwczMlbbVg90H21EOLACvtfcHk0ljEcVt336hEpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVSrsb3N&md5=e5957d9438ed3a31af10a323d77b6777</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00378%26sid%3Dliteratum%253Aachs%26aulast%3DHatcher%26aufirst%3DJ.%2BM.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DBuhrlage%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520a%2520selective%252C%2520covalent%2520IRAK1%2520inhibitor%2520with%2520antiproliferative%2520activity%2520in%2520MYD88%2520mutated%2520B-cell%2520lymphoma%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D2238%26epage%3D2243%26doi%3D10.1021%2Facsmedchemlett.0c00378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0ljEcVt336hEpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0ljEcVt336hEpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">PROteolysis TArgeting Chimeras (PROTACs)—past, present and future</span>. <i>Drug Discov. Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.ddtec.2019.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=31200855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=15-27&author=M.+Petterssonauthor=C.+M.+Crews&title=PROteolysis+TArgeting+Chimeras+%28PROTACs%29%E2%80%94past%2C+present+and+future&doi=10.1016%2Fj.ddtec.2019.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">PROteolysis TArgeting Chimeras (PROTACs) - Past, present and future</span></div><div class="casAuthors">Pettersson Mariell; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The majority of currently used therapeutics are small molecule-based and utilize occupancy-driven pharmacology as the mode of action (MOA), in which the protein function is modulated via temporary inhibition.  New modalities that operate using alternative MOAs are essential for tapping into the "undruggable" proteome.  The PROteolysis Targeting Chimera (PROTAC) technology provides an attractive new approach that utilizes an event-driven MOA.  Small molecule-based heterobifunctional PROTACs modulate protein target levels by hijacking the ubiquitin-proteasome system to induce degradation of the target.  Here, we address important milestones in the development of the PROTAC technology, as well as emphasize key findings from this previous year and highlight future directions of this promising drug discovery modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoRfKb6ELK8t0fW6udTcc2eaQD81KljGb57ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWkuw%253D%253D&md5=c629f3128321e136f1f247e05ee6f182</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DPettersson%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DPROteolysis%2520TArgeting%2520Chimeras%2520%2528PROTACs%2529%25E2%2580%2594past%252C%2520present%2520and%2520future%26jtitle%3DDrug%2520Discov.%2520Today%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.ddtec.2019.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohl, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Harnessing the power of proteolysis for targeted protein inactivation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.molcel.2020.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=32004468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2020&pages=446-460&author=R.+Vermaauthor=D.+Mohlauthor=R.+J.+Deshaies&title=Harnessing+the+power+of+proteolysis+for+targeted+protein+inactivation&doi=10.1016%2Fj.molcel.2020.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing the Power of Proteolysis for Targeted Protein Inactivation</span></div><div class="casAuthors">Verma, Rati; Mohl, Dane; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-460</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Two decades into the twenty-first century, a confluence of breakthrough technologies wielded at the mol. level is presenting biologists with unique opportunities to unravel the complexities of the cellular world.  CRISPR/Cas9 allows gene knock-outs, knock-ins, and single-base editing at chromosomal loci.  RNA-based tools such as siRNA, antisense oligos, and morpholinos can be used to silence expression of specific genes.  Meanwhile, protein knockdown tools that draw inspiration from natural regulatory mechanisms and facilitate elimination of native or degron-tagged proteins from cells are rapidly emerging.  The acute and reversible redn. in protein levels enabled by these methods allows for precise detn. of loss-of-function phenotypes free from secondary effects or compensatory adaptation that can confound nucleic-acid-based methods that involve slow depletion or permanent loss of a protein.  In this Review, we summarize the ingenious ways biologists have exploited natural mechanisms for protein degrdn. to direct the elimination of specific proteins at will.  This has led to advancements not only in basic research but also in the therapeutic space with the introduction of PROTACs into clin. trials for cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-2OyHAw-uMLVg90H21EOLACvtfcHk0lhzO2DgYz3zHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFCjsr4%253D&md5=8d74ed00d55184be33786d0ee44ad787</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DR.%26aulast%3DMohl%26aufirst%3DD.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DHarnessing%2520the%2520power%2520of%2520proteolysis%2520for%2520targeted%2520protein%2520inactivation%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D77%26spage%3D446%26epage%3D460%26doi%3D10.1016%2Fj.molcel.2020.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higueruelo, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rausch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. L.</span></span> <span> </span><span class="NLM_article-title">IRAK-4 inhibitors. Part 1: a series of amides</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3211</span>– <span class="NLM_lpage">3214</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2008.04.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.bmcl.2008.04.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=18474425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BD1cXms1yiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3211-3214&author=G.+M.+Buckleyauthor=L.+Gowersauthor=A.+P.+Higuerueloauthor=K.+Jenkinsauthor=S.+R.+Mackauthor=T.+Morganauthor=D.+M.+Parryauthor=W.+R.+Pittauthor=O.+Rauschauthor=M.+D.+Richardauthor=V.+Sabinauthor=J.+L.+Fraser&title=IRAK-4+inhibitors.+Part+1%3A+a+series+of+amides&doi=10.1016%2Fj.bmcl.2008.04.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">IRAK-4 inhibitors. Part 1: A series of amides</span></div><div class="casAuthors">Buckley, George M.; Gowers, Lewis; Higueruelo, Alicia Perez; Jenkins, Kerry; Mack, Stephen R.; Morgan, Trevor; Parry, David M.; Pitt, William R.; Rausch, Oliver; Richard, Marianna D.; Sabin, Verity; Fraser, Joanne L.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3211-3214</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The synthesis and profile of a series of amides, e.g., I, are described.  Some of these compds. were potent IRAK-4 inhibitors and two examples were evaluated in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2WnghW2pEwbVg90H21EOLACvtfcHk0lhzO2DgYz3zHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXms1yiu78%253D&md5=75fb794d58ae52827a58b8ff59394111</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.04.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.04.058%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DG.%2BM.%26aulast%3DGowers%26aufirst%3DL.%26aulast%3DHigueruelo%26aufirst%3DA.%2BP.%26aulast%3DJenkins%26aufirst%3DK.%26aulast%3DMack%26aufirst%3DS.%2BR.%26aulast%3DMorgan%26aufirst%3DT.%26aulast%3DParry%26aufirst%3DD.%2BM.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DRausch%26aufirst%3DO.%26aulast%3DRichard%26aufirst%3DM.%2BD.%26aulast%3DSabin%26aufirst%3DV.%26aulast%3DFraser%26aufirst%3DJ.%2BL.%26atitle%3DIRAK-4%2520inhibitors.%2520Part%25201%253A%2520a%2520series%2520of%2520amides%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3211%26epage%3D3214%26doi%3D10.1016%2Fj.bmcl.2008.04.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhrlage, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of human IRAK1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">13507</span>– <span class="NLM_lpage">13512</span>, <span class="refDoi"> DOI: 10.1073/pnas.1714386114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1073%2Fpnas.1714386114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=29208712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmtLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=13507-13512&author=L.+Wangauthor=Q.+Qiaoauthor=R.+Ferraoauthor=C.+Shenauthor=J.+M.+Hatcherauthor=S.+J.+Buhrlageauthor=N.+S.+Grayauthor=H.+Wu&title=Crystal+structure+of+human+IRAK1&doi=10.1073%2Fpnas.1714386114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of human IRAK1</span></div><div class="casAuthors">Wang, Li; Qiao, Qi; Ferrao, Ryan; Shen, Chen; Hatcher, John M.; Buhrlage, Sara J.; Gray, Nathanael S.; Wu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">13507-13512</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Interleukin 1 (IL-1) receptor-assocd. kinases (IRAKs) are serine/threonine kinases that play crit. roles in initiating innate immune responses against foreign pathogens and other types of dangers through their role in Toll-like receptor (TLR) and interleukin 1 receptor (IL-1R) mediated signaling pathways.  Upon ligand binding, TLRs and IL-1Rs recruit adaptor proteins, such as myeloid differentiation primary response gene 88 (MyD88), to the membrane, which in turn recruit IRAKs via the death domains in these proteins to form the Myddosome complex, leading to IRAK kinase activation.  Despite their biol. and clin. significance, only the IRAK4 kinase domain structure has been detd. among the four IRAK family members.  Here, we report the crystal structure of the human IRAK1 kinase domain in complex with a small mol. inhibitor.  The structure reveals both similarities and differences between IRAK1 and IRAK4 and is suggestive of approaches to develop IRAK1- or IRAK4-specific inhibitors for potential therapeutic applications.  While the IRAK4 kinase domain is capable of homodimerization in the unphosphorylated state, we found that the IRAK1 kinase domain is constitutively monomeric regardless of its phosphorylation state.  Addnl., the IRAK1 kinase domain forms heterodimers with the phosphorylated, but not unphosphorylated, IRAK4 kinase domain.  Collectively, these data indicate a two-step kinase activation process in which the IRAK4 kinase domain first homodimerizes in the Myddosome, leading to its trans-autophosphorylation and activation.  The phosphorylated IRAK4 kinase domain then forms heterodimers with the IRAK1 kinase domain within the Myddosome, leading to its subsequent phosphorylation and activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8eF96qYaD-7Vg90H21EOLACvtfcHk0lhzO2DgYz3zHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmtLrF&md5=6e8711f19b9608e522e29e423915e20c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1714386114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1714386114%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DQiao%26aufirst%3DQ.%26aulast%3DFerrao%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DHatcher%26aufirst%3DJ.%2BM.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWu%26aufirst%3DH.%26atitle%3DCrystal%2520structure%2520of%2520human%2520IRAK1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D13507%26epage%3D13512%26doi%3D10.1073%2Fpnas.1714386114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plantevin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitefield, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kercher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moghaddam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebinger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccotti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havens, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leftheris, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baculi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBrun, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khambatta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span> <span> </span><span class="NLM_article-title">Discovery of CRBN E3 ligase modulator CC-92480 for the treatment of relapsed and refractory multiple myeloma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6648</span>– <span class="NLM_lpage">6676</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01928</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktlOku78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6648-6676&author=J.+D.+Hansenauthor=M.+Correaauthor=M.+A.+Nagyauthor=M.+Alexanderauthor=V.+Plantevinauthor=V.+Grantauthor=B.+Whitefieldauthor=D.+Huangauthor=T.+Kercherauthor=R.+Harrisauthor=R.+K.+Narlaauthor=J.+Leistenauthor=Y.+Tangauthor=M.+Moghaddamauthor=K.+Ebingerauthor=J.+Piccottiauthor=C.+G.+Havensauthor=B.+Cathersauthor=J.+Carmichaelauthor=T.+Danielauthor=R.+Vesseyauthor=L.+G.+Hamannauthor=K.+Leftherisauthor=D.+Mendyauthor=F.+Baculiauthor=L.+A.+LeBrunauthor=G.+Khambattaauthor=A.+Lopez-Girona&title=Discovery+of+CRBN+E3+ligase+modulator+CC-92480+for+the+treatment+of+relapsed+and+refractory+multiple+myeloma&doi=10.1021%2Facs.jmedchem.9b01928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of CRBN E3 Ligase Modulator CC-92480 for the Treatment of Relapsed and Refractory Multiple Myeloma</span></div><div class="casAuthors">Hansen, Joshua D.; Correa, Matthew; Nagy, Mark A.; Alexander, Matt; Plantevin, Veronique; Grant, Virginia; Whitefield, Brandon; Huang, Dehua; Kercher, Timothy; Harris, Roy; Narla, Rama Krishna; Leisten, Jim; Tang, Yang; Moghaddam, Mehran; Ebinger, Katalin; Piccotti, Joseph; Havens, Courtney G.; Cathers, Brian; Carmichael, James; Daniel, Thomas; Vessey, Rupert; Hamann, Lawrence G.; Leftheris, Katerina; Mendy, Derek; Baculi, Frans; LeBrun, Laurie A.; Khambatta, Gody; Lopez-Girona, Antonia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6648-6676</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many patients with multiple myeloma (MM) initially respond to treatment with modern combination regimens including immunomodulatory agents (lenalidomide and pomalidomide) and proteasome inhibitors.  However, some patients lack an initial response to therapy (i.e., are refractory), and although the mean survival of MM patients has more than doubled in recent years, most patients will eventually relapse.  To address this need, we explored the potential of novel cereblon E3 ligase modulators (CELMoDs) for the treatment of patients with relapsed or refractory multiple myeloma (RRMM).  We found that optimization beyond potency of degrdn., including degrdn. efficiency and kinetics, could provide efficacy in a lenalidomide-resistant setting.  Guided by both phenotypic and protein degrdn. data, we describe a series of CELMoDs for the treatment of RRMM, culminating in the discovery of CC-92480, a novel protein degrader and the first CELMoD to enter clin. development that was specifically designed for efficient and rapid protein degrdn. kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBUWnyqWibcbVg90H21EOLACvtfcHk0liQI5k0oZSg1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktlOku78%253D&md5=bce7d7a9033474a2800944baf21fed81</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01928%26sid%3Dliteratum%253Aachs%26aulast%3DHansen%26aufirst%3DJ.%2BD.%26aulast%3DCorrea%26aufirst%3DM.%26aulast%3DNagy%26aufirst%3DM.%2BA.%26aulast%3DAlexander%26aufirst%3DM.%26aulast%3DPlantevin%26aufirst%3DV.%26aulast%3DGrant%26aufirst%3DV.%26aulast%3DWhitefield%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DD.%26aulast%3DKercher%26aufirst%3DT.%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DNarla%26aufirst%3DR.%2BK.%26aulast%3DLeisten%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DMoghaddam%26aufirst%3DM.%26aulast%3DEbinger%26aufirst%3DK.%26aulast%3DPiccotti%26aufirst%3DJ.%26aulast%3DHavens%26aufirst%3DC.%2BG.%26aulast%3DCathers%26aufirst%3DB.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%26aulast%3DVessey%26aufirst%3DR.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DLeftheris%26aufirst%3DK.%26aulast%3DMendy%26aufirst%3DD.%26aulast%3DBaculi%26aufirst%3DF.%26aulast%3DLeBrun%26aufirst%3DL.%2BA.%26aulast%3DKhambatta%26aufirst%3DG.%26aulast%3DLopez-Girona%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520CRBN%2520E3%2520ligase%2520modulator%2520CC-92480%2520for%2520the%2520treatment%2520of%2520relapsed%2520and%2520refractory%2520multiple%2520myeloma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6648%26epage%3D6676%26doi%3D10.1021%2Facs.jmedchem.9b01928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4(CRBN) ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/nature18611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.+A.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4%28CRBN%29+ubiquitin+ligase&doi=10.1038%2Fnature18611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0liQI5k0oZSg1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%2BA.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4%2528CRBN%2529%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257%26doi%3D10.1038%2Fnature18611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clayton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdubek, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blease, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">SALL4 mediates teratogenicity as a thalidomide-dependent cereblon substrate</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">981</span>– <span class="NLM_lpage">987</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0129-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fs41589-018-0129-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=30190590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Kgu7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=981-987&author=M.+E.+Matyskielaauthor=S.+Coutoauthor=X.+Zhengauthor=G.+Luauthor=J.+Huiauthor=K.+Stampauthor=C.+Drewauthor=Y.+Renauthor=M.+Wangauthor=A.+Carpenterauthor=C.-W.+Leeauthor=T.+Claytonauthor=W.+Fangauthor=C.-C.+Luauthor=M.+Rileyauthor=P.+Abdubekauthor=K.+Bleaseauthor=J.+Hartkeauthor=G.+Kumarauthor=R.+Vesseyauthor=M.+Rolfeauthor=L.+G.+Hamannauthor=P.+P.+Chamberlain&title=SALL4+mediates+teratogenicity+as+a+thalidomide-dependent+cereblon+substrate&doi=10.1038%2Fs41589-018-0129-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">SALL4 mediates teratogenicity as a thalidomide-dependent cereblon substrate</span></div><div class="casAuthors">Matyskiela, Mary E.; Couto, Suzana; Zheng, Xinde; Lu, Gang; Hui, Julia; Stamp, Katie; Drew, Clifton; Ren, Yan; Wang, Maria; Carpenter, Aaron; Lee, Chung-Wein; Clayton, Thomas; Fang, Wei; Lu, Chin-Chun; Riley, Mariko; Abdubek, Polat; Blease, Kate; Hartke, James; Kumar, Gondi; Vessey, Rupert; Rolfe, Mark; Hamann, Lawrence G.; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">981-987</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeted protein degrdn. via small-mol. modulation of cereblon offers vast potential for the development of new therapeutics.  Cereblon-binding therapeutics carry the safety risks of thalidomide, which caused an epidemic of severe birth defects characterized by forelimb shortening or phocomelia.  Here we show that thalidomide is not teratogenic in transgenic mice expressing human cereblon, indicating that binding to cereblon is not sufficient to cause birth defects.  Instead, we identify SALL4 as a thalidomide-dependent cereblon neosubstrate.  Human mutations in SALL4 cause Duane-radial ray, IVIC, and acro-renal-ocular syndromes with overlapping clin. presentations to thalidomide embryopathy, including phocomelia.  SALL4 is degraded in rabbits but not in resistant organisms such as mice because of SALL4 sequence variations.  This work expands the scope of cereblon neosubstrate activity within the formerly 'undruggable' C2H2 zinc finger family and offers a path toward safer therapeutics through an improved understanding of the mol. basis of thalidomide-induced teratogenicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpePTE78HX85rVg90H21EOLACvtfcHk0lh7OyAVSS4GNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Kgu7nM&md5=7530c3850ed7a763b30cba44dab640ac</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0129-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0129-x%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DCouto%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DHui%26aufirst%3DJ.%26aulast%3DStamp%26aufirst%3DK.%26aulast%3DDrew%26aufirst%3DC.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCarpenter%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.-W.%26aulast%3DClayton%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DAbdubek%26aufirst%3DP.%26aulast%3DBlease%26aufirst%3DK.%26aulast%3DHartke%26aufirst%3DJ.%26aulast%3DKumar%26aufirst%3DG.%26aulast%3DVessey%26aufirst%3DR.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DSALL4%2520mediates%2520teratogenicity%2520as%2520a%2520thalidomide-dependent%2520cereblon%2520substrate%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D981%26epage%3D987%26doi%3D10.1038%2Fs41589-018-0129-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asatsuma-Okumura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ando, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Simone, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asakawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span> <span> </span><span class="NLM_article-title">p63 is a cereblon substrate involved in thalidomide teratogenicity</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1084</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0366-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1038%2Fs41589-019-0366-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=31591562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFaksbfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1077-1084&author=T.+Asatsuma-Okumuraauthor=H.+Andoauthor=M.+De+Simoneauthor=J.+Yamamotoauthor=T.+Satoauthor=N.+Shimizuauthor=K.+Asakawaauthor=Y.+Yamaguchiauthor=T.+Itoauthor=L.+Guerriniauthor=H.+Handa&title=p63+is+a+cereblon+substrate+involved+in+thalidomide+teratogenicity&doi=10.1038%2Fs41589-019-0366-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">p63 is a cereblon substrate involved in thalidomide teratogenicity</span></div><div class="casAuthors">Asatsuma-Okumura, Tomoko; Ando, Hideki; De Simone, Marco; Yamamoto, Junichi; Sato, Tomomi; Shimizu, Nobuyuki; Asakawa, Kazuhide; Yamaguchi, Yuki; Ito, Takumi; Guerrini, Luisa; Handa, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1077-1084</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cereblon (CRBN) is a primary target of thalidomide and mediates its multiple pharmacol. activities, including teratogenic and antimyeloma activities.  CRBN functions as a substrate receptor of the E3 ubiquitin ligase CRL4, whose substrate specificity is modulated by thalidomide and its analogs.  Although a no. of CRL4CRBN substrates have recently been identified, the substrate involved in thalidomide teratogenicity is unclear.  Here we show that p63 isoforms are thalidomide-dependent CRL4CRBN neosubstrates that are responsible, at least in part, for its teratogenic effects.  The p53 family member p63 is assocd. with multiple developmental processes.  ΔNp63α is essential for limb development, while TAp63α is important for cochlea development and hearing.  Using a zebrafish model, we demonstrate that thalidomide exerts its teratogenic effects on pectoral fins and otic vesicles by inducing the degrdn. of ΔNp63α and TAp63α, resp.  These results may contribute to the invention of new thalidomide analogs lacking teratogenic activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ4WsPMCzNSrVg90H21EOLACvtfcHk0lh7OyAVSS4GNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFaksbfL&md5=c35c85968ace67e49978e40579e2706c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0366-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0366-7%26sid%3Dliteratum%253Aachs%26aulast%3DAsatsuma-Okumura%26aufirst%3DT.%26aulast%3DAndo%26aufirst%3DH.%26aulast%3DDe%2BSimone%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DJ.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DShimizu%26aufirst%3DN.%26aulast%3DAsakawa%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DY.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DGuerrini%26aufirst%3DL.%26aulast%3DHanda%26aufirst%3DH.%26atitle%3Dp63%2520is%2520a%2520cereblon%2520substrate%2520involved%2520in%2520thalidomide%2520teratogenicity%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D1077%26epage%3D1084%26doi%3D10.1038%2Fs41589-019-0366-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagner, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, H.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanillo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjorklund, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havens, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rychak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narla, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khambatta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amantangelo, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pourdehnad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trotter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schafer, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakurta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, A. K.</span></span> <span> </span><span class="NLM_article-title">CC-122, a pleiotropic pathway modifier, mimics an interferon response and has antitumor activity in DLBCL</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1182/blood-2015-02-628669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1182%2Fblood-2015-02-628669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=26002965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFahtL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=779-789&author=P.+R.+Hagnerauthor=H.-W.+Manauthor=C.+Fontanilloauthor=M.+Wangauthor=S.+Coutoauthor=M.+Breiderauthor=C.+Bjorklundauthor=C.+G.+Havensauthor=G.+Luauthor=E.+Rychakauthor=H.+Raymonauthor=R.+K.+Narlaauthor=L.+Barnesauthor=G.+Khambattaauthor=H.+Chiuauthor=J.+Kosekauthor=J.+Kangauthor=M.+D.+Amantangeloauthor=M.+Waldmanauthor=A.+Lopez-Gironaauthor=T.+Caiauthor=M.+Pourdehnadauthor=M.+Trotterauthor=T.+O.+Danielauthor=P.+H.+Schaferauthor=A.+Klippelauthor=A.+Thakurtaauthor=R.+Chopraauthor=A.+K.+Gandhi&title=CC-122%2C+a+pleiotropic+pathway+modifier%2C+mimics+an+interferon+response+and+has+antitumor+activity+in+DLBCL&doi=10.1182%2Fblood-2015-02-628669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">CC-122, a pleiotropic pathway modifier, mimics an interferon response and has antitumor activity in DLBCL</span></div><div class="casAuthors">Hagner, Patrick R.; Man, Hon-Wah; Fontanillo, Celia; Wang, Maria; Couto, Suzana; Breider, Mike; Bjorklund, Chad; Havens, Courtney G.; Lu, Gang; Rychak, Emily; Raymon, Heather; Narla, Rama Krishna; Barnes, Leo; Khambatta, Gody; Chiu, Hsiling; Kosek, Jolanta; Kang, Jian; Amantangelo, Michael D.; Waldman, Michelle; Lopez-Girona, Antonia; Cai, Ti; Pourdehnad, Michael; Trotter, Matthew; Daniel, Thomas O.; Schafer, Peter H.; Klippel, Anke; Thakurta, Anjan; Chopra, Rajesh; Gandhi, Anita K.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">779-789</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Cereblon (CRBN), a substrate receptor of the Cullin 4 RING E3 ubiquitin ligase complex, is the target of the immunomodulatory drugs lenalidomide and pomalidomide.  Recently, it was demonstrated that binding of these drugs to CRBN promotes the ubiquitination and subsequent degrdn. of 2 common substrates, transcription factors Aiolos and Ikaros.  Here we report that CC-122, a new chem. entity termed pleiotropic pathway modifier, binds CRBN and promotes degrdn. of Aiolos and Ikaros in diffuse large B-cell lymphoma (DLBCL) and T cells in vitro, in vivo, and in patients, resulting in both cell autonomous as well as immunostimulatory effects.  In DLBCL cell lines, CC-122-induced degrdn. or short hairpin RNA-mediated knockdown of Aiolos and Ikaros correlates with increased transcription of interferon (IFN)-stimulated genes independent of IFN-α, -β, and -γ prodn. and/or secretion and results in apoptosis in both activated B-cell (ABC) and germinal center B-cell DLBCL cell lines.  Our results provide mechanistic insight into the cell-of-origin independent antilymphoma activity of CC-122, in contrast to the ABC subtype selective activity of lenalidomide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Ayj7KrKYWrVg90H21EOLACvtfcHk0lhi7BrK9eBXyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFahtL7K&md5=786ea247e42ebc90dec67fc6f7a4e48b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2015-02-628669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2015-02-628669%26sid%3Dliteratum%253Aachs%26aulast%3DHagner%26aufirst%3DP.%2BR.%26aulast%3DMan%26aufirst%3DH.-W.%26aulast%3DFontanillo%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DCouto%26aufirst%3DS.%26aulast%3DBreider%26aufirst%3DM.%26aulast%3DBjorklund%26aufirst%3DC.%26aulast%3DHavens%26aufirst%3DC.%2BG.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DRychak%26aufirst%3DE.%26aulast%3DRaymon%26aufirst%3DH.%26aulast%3DNarla%26aufirst%3DR.%2BK.%26aulast%3DBarnes%26aufirst%3DL.%26aulast%3DKhambatta%26aufirst%3DG.%26aulast%3DChiu%26aufirst%3DH.%26aulast%3DKosek%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DAmantangelo%26aufirst%3DM.%2BD.%26aulast%3DWaldman%26aufirst%3DM.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DT.%26aulast%3DPourdehnad%26aufirst%3DM.%26aulast%3DTrotter%26aufirst%3DM.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DSchafer%26aufirst%3DP.%2BH.%26aulast%3DKlippel%26aufirst%3DA.%26aulast%3DThakurta%26aufirst%3DA.%26aulast%3DChopra%26aufirst%3DR.%26aulast%3DGandhi%26aufirst%3DA.%2BK.%26atitle%3DCC-122%252C%2520a%2520pleiotropic%2520pathway%2520modifier%252C%2520mimics%2520an%2520interferon%2520response%2520and%2520has%2520antitumor%2520activity%2520in%2520DLBCL%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D779%26epage%3D789%26doi%3D10.1182%2Fblood-2015-02-628669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhi7BrK9eBXyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-based optimization of potent and cell-active inhibitors of the von Hippel-Lindau (VHL) E3 ubiquitin ligase: structure-activity relationships leading to the chemical probe (2S,4R)-1-((S)-2-(1-cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=599-618&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-based+optimization+of+potent+and+cell-active+inhibitors+of+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase%3A+structure-activity+relationships+leading+to+the+chemical+probe+%282S%2C4R%29-1-%28%28S%29-2-%281-cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyl%29-4-hydroxy-N-%284-%284-methylthiazol-5-yl%29benzyl%29pyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Group-Based Optimization of Potent and Cell-Active Inhibitors of the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase: Structure-Activity Relationships Leading to the Chemical Probe (2S,4R)-1-((S)-2-(1-Cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy-N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span></div><div class="casAuthors">Soares, Pedro; Gadd, Morgan S.; Frost, Julianty; Galdeano, Carles; Ellis, Lucy; Epemolu, Ola; Rocha, Sonia; Read, Kevin D.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">599-618</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The von Hippel-Lindau tumor suppressor protein is the substrate binding subunit of the VHL E3 ubiquitin ligase, which targets hydroxylated α subunit of hypoxia inducible factors (HIFs) for ubiquitination and subsequent proteasomal degrdn.  VHL is a potential target for treating anemia and ischemic diseases, motivating the development of inhibitors of the VHL:HIF-α protein-protein interaction.  Addnl., bifunctional proteolysis targeting chimeras (PROTACs) contg. a VHL ligand can hijack the E3 ligase activity to induce degrdn. of target proteins.  We report the structure-guided design and group-based optimization of a series of VHL inhibitors with low nanomolar potencies and improved cellular permeability.  Structure-activity relationships led to the discovery of potent inhibitors 10 and chem. probe VH298, with dissocn. consts. <100 nM, which induced marked HIF-1α intracellular stabilization.  Our study provides new chem. tools to probe the VHL-HIF pathways and new VHL ligands for next-generation PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQfJe2l2WgX7Vg90H21EOLACvtfcHk0lhi7BrK9eBXyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWms77P&md5=1c1e7c976c2de9a9681773ff216fdfd7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-based%2520optimization%2520of%2520potent%2520and%2520cell-active%2520inhibitors%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%253A%2520structure-activity%2520relationships%2520leading%2520to%2520the%2520chemical%2520probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyl%2529-4-hydroxy-N-%25284-%25284-methylthiazol-5-yl%2529benzyl%2529pyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1420</span>– <span class="NLM_lpage">1442</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01572</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01572" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1420-1442&author=J.+Huauthor=B.+Huauthor=M.+Wangauthor=F.+Xuauthor=B.+Miaoauthor=C.-Y.+Yangauthor=M.+Wangauthor=Z.+Liuauthor=D.+F.+Hayesauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ERD-308+as+a+highly+potent+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+estrogen+receptor+%28ER%29&doi=10.1021%2Facs.jmedchem.8b01572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ERD-308 as a highly potent proteolysis targeting chimera (PROTAC) degrader of estrogen receptor (ER)</span></div><div class="casAuthors">Hu, Jiantao; Hu, Biao; Wang, Mingliang; Xu, Fuming; Miao, Bukeyan; Yang, Chao-Yie; Wang, Mi; Liu, Zhaomin; Hayes, Daniel F.; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1420-1442</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-pos. (ER+) breast cancer.  Here, we describe the design, synthesis, and extensive structure-activity relationship (SAR) studies of small-mol. ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept.  Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 I.  I achieves DC50 (concn. causing 50% of protein degrdn.) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, resp., and induces >95% of ER degrdn. at concns. as low as 5 nM in both cell lines.  Significantly, I induces more complete ER degrdn. than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells.  Further optimization of I may lead to a new therapy for advanced ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHRlcISYyFXLVg90H21EOLACvtfcHk0lj2GJRic3Y4Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Cqs7s%253D&md5=678e82dd460c065e2054ba18ea8af18a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DMiao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ERD-308%2520as%2520a%2520highly%2520potent%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520estrogen%2520receptor%2520%2528ER%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1420%26epage%3D1442%26doi%3D10.1021%2Facs.jmedchem.8b01572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gütschow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinebach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voell, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricelj, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sosič, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnakenburg, G.</span></span> <span> </span><span class="NLM_article-title">A facile synthesis of ligands for the von Hippel–Lindau E3 ligase</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2521</span>– <span class="NLM_lpage">2527</span>, <span class="refDoi"> DOI: 10.1055/s-0040-1707400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1055%2Fs-0040-1707400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVanu7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2020&pages=2521-2527&author=M.+G%C3%BCtschowauthor=C.+Steinebachauthor=S.+A.+Voellauthor=L.+P.+Vuauthor=A.+Briceljauthor=I.+Sosi%C4%8Dauthor=G.+Schnakenburg&title=A+facile+synthesis+of+ligands+for+the+von+Hippel%E2%80%93Lindau+E3+ligase&doi=10.1055%2Fs-0040-1707400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A facile synthesis of ligands for the von Hippel-Lindau E3 ligase</span></div><div class="casAuthors">Steinebach, Christian; Voell, Sabine Anna; Vu, Lan Phuong; Bricelj, Alesa; Sosic, Izidor; Schnakenburg, Gregor; Guetschow, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2521-2527</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">The proteolysis-targeting chimeras (PROTACs) have become an integral part of different stages of drug discovery.  This growing field, therefore, benefits from advancements in all segments of the design of these compds.  Herein, an efficient and optimized synthetic protocol to various von Hippel-Lindau (VHL) ligands is presented, which enables easy access to multigram quantities of these essential PROTAC building blocks.  Moreover, the elaborated synthesis represents a straightforward approach to further explore the chem. space of VHL ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyw4H4Rq7HfrVg90H21EOLACvtfcHk0lj2GJRic3Y4Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVanu7bN&md5=893543d726fa49e0ddb2a251c7efb792</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1055%2Fs-0040-1707400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0040-1707400%26sid%3Dliteratum%253Aachs%26aulast%3DG%25C3%25BCtschow%26aufirst%3DM.%26aulast%3DSteinebach%26aufirst%3DC.%26aulast%3DVoell%26aufirst%3DS.%2BA.%26aulast%3DVu%26aufirst%3DL.%2BP.%26aulast%3DBricelj%26aufirst%3DA.%26aulast%3DSosi%25C4%258D%26aufirst%3DI.%26aulast%3DSchnakenburg%26aufirst%3DG.%26atitle%3DA%2520facile%2520synthesis%2520of%2520ligands%2520for%2520the%2520von%2520Hippel%25E2%2580%2593Lindau%2520E3%2520ligase%26jtitle%3DSynthesis%26date%3D2020%26volume%3D52%26spage%3D2521%26epage%3D2527%26doi%3D10.1055%2Fs-0040-1707400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lj2GJRic3Y4Tw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kostic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Critical assessment of targeted protein degradation as a research tool and pharmacological modality</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2020.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.1016%2Fj.tips.2020.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=32222318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlt1Wmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2020&pages=305-317&author=M.+Kosticauthor=L.+H.+Jones&title=Critical+assessment+of+targeted+protein+degradation+as+a+research+tool+and+pharmacological+modality&doi=10.1016%2Fj.tips.2020.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Critical Assessment of Targeted Protein Degradation as a Research Tool and Pharmacological Modality</span></div><div class="casAuthors">Kostic, Milka; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">305-317</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Small mols. continue to dominate drug discovery because of their ease of use, lower cost of manufg., and access to intracellular targets.  However, despite these advantages, small mols. are more likely to fail in clin. trials compared with biologicals and their development remains limited to a small subset of disease-relevant 'druggable' targets.  Targeted protein degrdn. has recently emerged as a novel pharmacol. modality that promises to overcome small mol. limitations while retaining their key advantages.  Here, we use a Strengths-Weaknesses-Opportunities-Threats (SWOT) framework to critically assess the current status of this rapidly evolving field.  We expect that degrader mols. are only the beginning of a range of novel targeting modalities that hijack existing endogenous cellular machineries to chem. redirect biol. targets and pathways.  Therefore, this piece may offer a roadmap for enhancing development of both degraders and related modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBLMTr8vJkrrVg90H21EOLACvtfcHk0lhHK_cKHGNbqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlt1Wmu7c%253D&md5=46a57cafb2427eff7a0b8ab0067492c2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2020.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2020.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DKostic%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DCritical%2520assessment%2520of%2520targeted%2520protein%2520degradation%2520as%2520a%2520research%2520tool%2520and%2520pharmacological%2520modality%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2020%26volume%3D41%26spage%3D305%26epage%3D317%26doi%3D10.1016%2Fj.tips.2020.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span> <span> </span><span class="NLM_article-title">Insights into the molecular pathogenesis of activated B-cell-like diffuse large B-cell lymphoma and its therapeutic implications</span>. <i>Cancers (Basel)</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.3390/cancers7020812</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=10.3390%2Fcancers7020812" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=26010601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfns1ahtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=811-822&author=G.+Lenz&title=Insights+into+the+molecular+pathogenesis+of+activated+B-cell-like+diffuse+large+B-cell+lymphoma+and+its+therapeutic+implications&doi=10.3390%2Fcancers7020812"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Molecular Pathogenesis of Activated B-Cell-like Diffuse Large B-Cell Lymphoma and Its Therapeutic Implications</span></div><div class="casAuthors">Lenz Georg; Lenz Georg</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">811-22</span>
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    </div><div class="casAbstract">Within the last couple of years, the understanding of the molecular mechanisms that drive the pathogenesis of diffuse large B-cell lymphoma (DLBCL) has significantly improved.  Large-scale gene expression profiling studies have led to the discovery of several molecularly defined subtypes that are characterized by specific oncogene addictions and significant differences in their outcome.  Next generation sequencing efforts combined with RNA interference screens frequently identify crucial oncogenes that lead to constitutive activation of various signaling pathways that drive lymphomagenesis.  This review summarizes our current understanding of the molecular pathogenesis of the activated B-cell-like (ABC) DLBCL subtype that is characterized by poor prognosis.  A special emphasis is put on findings that might impact therapeutic strategies of affected patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRuSkyEGfdySdts4TFFB11gfW6udTcc2ea8UAc7125lxLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfns1ahtA%253D%253D&md5=388cf80a8d7cda9b1828e47578bcf681</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fcancers7020812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers7020812%26sid%3Dliteratum%253Aachs%26aulast%3DLenz%26aufirst%3DG.%26atitle%3DInsights%2520into%2520the%2520molecular%2520pathogenesis%2520of%2520activated%2520B-cell-like%2520diffuse%2520large%2520B-cell%2520lymphoma%2520and%2520its%2520therapeutic%2520implications%26jtitle%3DCancers%2520%2528Basel%2529%26date%3D2015%26volume%3D7%26spage%3D811%26epage%3D822%26doi%3D10.3390%2Fcancers7020812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreiro, D. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komives, E. A.</span></span> <span> </span><span class="NLM_article-title">Molecular mechanisms of system control of NF-kappaB signaling by IkappaBalpha</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1567</span>, <span class="refDoi"> DOI: 10.1021/bi901948j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi901948j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFantrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=1560-1567&author=D.+U.+Ferreiroauthor=E.+A.+Komives&title=Molecular+mechanisms+of+system+control+of+NF-kappaB+signaling+by+IkappaBalpha&doi=10.1021%2Fbi901948j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Mechanisms of System Control of NF-κB Signaling by IκBα</span></div><div class="casAuthors">Ferreiro, Diego U.; Komives, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1560-1567</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The NF-κB family of transcription factors responds to inflammatory cytokines with rapid transcriptional activation and subsequent signal repression.  Much of the system control depends on the unique characteristics of its major inhibitor, IκBα, which appears to have folding dynamics that underlie the biophys. properties of its activity.  Theor. folding studies followed by expts. have shown that a portion of the ankyrin repeat domain of IκBα folds on binding.  In resting cells, IκBα is constantly being synthesized, but most of it is rapidly degraded, leaving only a very small pool of free IκBα.  Nearly all of the NF-κB is bound to IκBα, resulting in near-complete inhibition of nuclear localization and transcriptional activation.  Combined soln. biophys. measurements and quant. protein half-life measurements inside cells have allowed the authors to understand how the inhibition occurs, why IκBα can be degraded quickly in the free state but remain extremely stable in the bound state, and how signal activation and repression can be tuned by IκB folding dynamics.  This review summarizes results of in vitro and in vivo expts. that converge demonstrating the effective interplay between biophysics and cell biol. in understanding transcriptional control by the NF-κB signaling module.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdGq5LKH_dj7Vg90H21EOLACvtfcHk0lhHK_cKHGNbqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFantrs%253D&md5=25dc6d77be6dcea8d8761dbf7c5cc8e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fbi901948j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi901948j%26sid%3Dliteratum%253Aachs%26aulast%3DFerreiro%26aufirst%3DD.%2BU.%26aulast%3DKomives%26aufirst%3DE.%2BA.%26atitle%3DMolecular%2520mechanisms%2520of%2520system%2520control%2520of%2520NF-kappaB%2520signaling%2520by%2520IkappaBalpha%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D1560%26epage%3D1567%26doi%3D10.1021%2Fbi901948j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darricarrere, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">HaloPROTACS: use of small molecule PROTACs to induce degradation of HaloTag fusion proteins</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1831</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00442</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00442" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1831-1837&author=D.+L.+Buckleyauthor=K.+Rainaauthor=N.+Darricarrereauthor=J.+Hinesauthor=J.+L.+Gustafsonauthor=I.+E.+Smithauthor=A.+H.+Miahauthor=J.+D.+Harlingauthor=C.+M.+Crews&title=HaloPROTACS%3A+use+of+small+molecule+PROTACs+to+induce+degradation+of+HaloTag+fusion+proteins&doi=10.1021%2Facschembio.5b00442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">HaloPROTACS: Use of Small Molecule PROTACs to Induce Degradation of HaloTag Fusion Proteins</span></div><div class="casAuthors">Buckley, Dennis L.; Raina, Kanak; Darricarrere, Nicole; Hines, John; Gustafson, Jeffrey L.; Smith, Ian E.; Miah, Afjal H.; Harling, John D.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1831-1837</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol.-induced protein degrdn. is an attractive strategy for the development of chem. probes.  One method for inducing targeted protein degrdn. involves the use of PROTACs, heterobifunctional mols. that can recruit specific E3 ligases to a desired protein of interest.  PROTACs have been successfully used to degrade numerous proteins in cells, but the peptidic E3 ligase ligands used in previous PROTACs have hindered their development into more mature chem. probes or therapeutics.  The authors report the design of a novel class of PROTACs that incorporate small mol. VHL ligands to successfully degrade HaloTag7 fusion proteins.  These HaloPROTACs will inspire the development of future PROTACs with more drug-like properties.  Addnl., these HaloPROTACs are useful chem. genetic tools, due to their ability to chem. knock down widely used HaloTag7 fusion proteins in a general fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotj-Sgj4TyKrVg90H21EOLACvtfcHk0lhHK_cKHGNbqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSht7fN&md5=b074c58e983f15c384818777b25e248d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00442%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDarricarrere%26aufirst%3DN.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHaloPROTACS%253A%2520use%2520of%2520small%2520molecule%2520PROTACs%2520to%2520induce%2520degradation%2520of%2520HaloTag%2520fusion%2520proteins%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1831%26epage%3D1837%26doi%3D10.1021%2Facschembio.5b00442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BFN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BFN','PDB','6BFN'); return false;">PDB: 6BFN</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i26"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00103">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78112"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00103?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00103</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Western blot validation of proteomics studies of <b>Degrader-3</b>; global proteomic data of <b>Degrader-3</b> in HBL-1 cells; anti-proliferation potency of <b>JH-I-25</b> in HBL-1 cells; Western blot of IRAK1 and IRAK4 protein degradation in HEK293T cells; kinase activity assay; VHL FP assay; immunoblotting protocol; cell culture and cell viability assay; global proteomics assay protocol; <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra for conjugates <b>Degrader-1</b>, <b>Degrader-2</b>, <b>Degrader-3</b>, and <b>Degrader-4</b>; and KinomeScan raw data of <b>JH-I-25</b>, <b>Degrader-3</b>, and <b>Degrader-4</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Proteomics raw data of <b>Degrader-3</b> in Toledo (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_002.csv">CSV</a>)</p></li><li><p class="inline">Proteomics raw data of <b>Degrader-3</b> in HBL-1 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_001.pdf">jm1c00103_si_001.pdf (854.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_002.csv">jm1c00103_si_002.csv (478.01 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00103/suppl_file/jm1c00103_si_003.csv">jm1c00103_si_003.csv (969.31 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00103&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00103%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00103" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799213ee96f3c58","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
