
<html lang="en"     class="pb-page"  data-request-id="3dc615ce-9d5a-4c3a-aaad-586c320a00d7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/jm040084c;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2004.47.issue-27;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="A Screening Library for Peptide Activated G-Protein Coupled Receptors. 1. The Test Set" /></meta><meta name="dc.Creator" content="Karine  Lavrador,†" /></meta><meta name="dc.Creator" content="Brian  Murphy,‡" /></meta><meta name="dc.Creator" content="John  Saunders,*,†" /></meta><meta name="dc.Creator" content="Scott  Struthers,‡" /></meta><meta name="dc.Creator" content="Xiaochuan  Wang,† and" /></meta><meta name="dc.Creator" content="John  Williams†" /></meta><meta name="dc.Description" content="One subset of the G-protein coupled receptor (GPCR) superfamily is that which is activated by a peptide carrying an obligatory positively charged residue (GPCR−PA+). This subclass is exemplified by..." /></meta><meta name="Description" content="One subset of the G-protein coupled receptor (GPCR) superfamily is that which is activated by a peptide carrying an obligatory positively charged residue (GPCR−PA+). This subclass is exemplified by..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="December 1, 2004" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm040084c" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-47-27-6864" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(04)00084-6" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2004 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm040084c" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm040084c" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm040084c" /></link>
        
    
    

<title>A Screening Library for Peptide Activated G-Protein Coupled Receptors. 1. The Test Set | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm040084c" /></meta><meta property="og:title" content="A Screening Library for Peptide Activated G-Protein Coupled Receptors. 1. The Test Set" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cn00001.jpeg" /></meta><meta property="og:description" content=" One subset of the G-protein coupled receptor (GPCR) superfamily is that which is activated by a peptide carrying an obligatory positively charged residue (GPCR−PA+). This subclass is exemplified by receptors for melanocortins, GnRH, galanin, MCH, orexin, and some chemokine receptors variously involved in eating disorders, reproductive disorders, pain, narcolepsy, obesity, and inflammation. Using the methods described in this study, a region of chemical property space enriched in GPCR ligands was identified. This information was used to design and synthesize a “test” library of 2025 single, pure compounds to sample portions of this property space associated with GPCR−PA+ ligands. The library was evaluated by high-throughput screening against three different receptors, rMCH, hMC4, and hGnRH, and found to be highly enriched in active ligands (4.5−61-fold) compared to a control set of 2024 randomly selected compounds. In addition, the analysis suggested that about 7000 compounds will be necessary to complete the sampling of this GPCR−PA+ ligand-rich region and to better define its borders.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm040084c"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm040084c">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm040084c&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm040084c&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm040084c&amp;href=/doi/10.1021/jm040084c" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2004</span><span class="cit-fg-volume">, 47</span><span class="cit-fg-issue">, 27</span><span class="cit-fg-pageRange">, 6864-6874</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/47/27" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm040129%2B" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm049556d" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Screening Library for Peptide Activated G-Protein Coupled Receptors. 1. The Test Set</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Karine++Lavrador">Karine Lavrador</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brian++Murphy">Brian Murphy</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John++Saunders">John Saunders</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Scott++Struthers">Scott Struthers</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiaochuan++Wang">Xiaochuan Wang</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John++Williams">John Williams</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Chemistry and Biology Departments, Neurocrine Biosciences, 12790 El Camino Real, San Diego, California 92130 </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm040084c&amp;href=/doi/10.1021%2Fjm040084c" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2004</span></span><span class="cit-volume">, 47</span><span class="cit-issue">, 27</span><span class="cit-pageRange">, 6864–6874</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 1, 2004</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 April 2004</li><li><span class="item_label"><b>Published</b> online</span>1 December 2004</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 December 2004</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm040084c" title="DOI URL">https://doi.org/10.1021/jm040084c</a></div><div class="article_header-article-copyright"><strong>Copyright © 2004 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6864%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKarine%2BLavrador%252C%2BBrian%2BMurphy%252C%2BJohn%2BSaunders%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D47%26issueNum%3D27%26contentID%3Djm040084c%26title%3DA%2BScreening%2BLibrary%2Bfor%2BPeptide%2BActivated%2BG-Protein%2BCoupled%2BReceptors.%2B1.%2BThe%2BTest%2BSet%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6874%26publicationDate%3DDecember%2B2004">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm040084c"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">471</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">18</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm040084c" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Screening Library for Peptide Activated G-Protein Coupled Receptors. 1. The Test Set&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Karine&quot;,&quot;last_name&quot;:&quot;Lavrador&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;Murphy&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Saunders&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;Struthers&quot;},{&quot;first_name&quot;:&quot;Xiaochuan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Williams&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2004&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;27&quot;,&quot;volume&quot;:&quot;47&quot;,&quot;pages&quot;:&quot;6864-6874&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm040084c&quot;},&quot;abstract&quot;:&quot;One subset of the G-protein coupled receptor (GPCR) superfamily is that which is activated by a peptide carrying an obligatory positively charged residue (GPCR−PA+). This subclass is exemplified by receptors for melanocortins, GnRH, galanin, MCH, orexin, and some chemokine receptors variously involved in eating disorders, reproductive disorders, pain, narcolepsy, obesity, and inflammation. Using the methods described in this study, a region of chemical property space enriched in GPCR ligands was identified. This information was used to design and synthesize a “test” library of 2025 single, pure compounds to sample portions of this property space associated with GPCR−PA+ ligands. The library was evaluated by high-throughput screening against three different receptors, rMCH, hMC4, and hGnRH, and found to be highly enriched in active ligands (4.5−61-fold) compared to a control set of 2024 randomly selected compounds. In addition, the analysis suggested that about 7000 compounds will be necessary to complete the &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm040084c&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm040084c" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm040084c&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm040084c" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm040084c&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm040084c" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm040084c&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm040084c&amp;href=/doi/10.1021/jm040084c" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm040084c" /></input><a href="/doi/pdf/10.1021/jm040084c" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (391 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm040084c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm040084c%26sid%3Dliteratum%253Aachs%26pmid%3D15615535%26genre%3Darticle%26aulast%3DLavrador%26date%3D2004%26atitle%3DA%2BScreening%2BLibrary%2Bfor%2BPeptide%2BActivated%2BG-Protein%2BCoupled%2BReceptors.%2B1.%2BThe%2BTest%2BSet%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D47%26issue%3D27%26spage%3D6864%26epage%3D6874%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/47/27" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">One subset of the G-protein coupled receptor (GPCR) superfamily is that which is activated by a peptide carrying an obligatory positively charged residue (GPCR−PA<sup>+</sup>). This subclass is exemplified by receptors for melanocortins, GnRH, galanin, MCH, orexin, and some chemokine receptors variously involved in eating disorders, reproductive disorders, pain, narcolepsy, obesity, and inflammation. Using the methods described in this study, a region of chemical property space enriched in GPCR ligands was identified. This information was used to design and synthesize a “test” library of 2025 single, pure compounds to sample portions of this property space associated with GPCR−PA<sup>+</sup> ligands. The library was evaluated by high-throughput screening against three different receptors, rMCH, hMC4, and hGnRH, and found to be highly enriched in active ligands (4.5−61-fold) compared to a control set of 2024 randomly selected compounds. In addition, the analysis suggested that about 7000 compounds will be necessary to complete the sampling of this GPCR−PA<sup>+</sup> ligand-rich region and to better define its borders. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm040084cAF2"><span><h4><span class="NLM_fn-symbol">†</span></h4></span><span><p class="last">  Chemistry Department.</p></span></div><div class="fnGroupItem" id="jm040084cAF3"><span><h4><span class="NLM_fn-symbol">‡</span></h4></span><span><p class="last">  Biology Department.</p></span></div><div class="fnGroupItem" id="jm040084cAF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  Corresponding author. Tel:  858 617 7637. E-mail:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="771d041602191312050437">[email protected]</a> neurocrine.com. </p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25416" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25416" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Stimulation of receptors linked to G-protein activation represents a primary mechanism by which cells sense changes in their external environment and convey that information to the cytosol through various effector mechanisms.<sup><a onclick="showRef(event, 'jm040084cb00001 jm040084cb00002'); return false;" href="javascript:void(0);" class="ref jm040084cb00001 jm040084cb00002">1,2</a></sup> Historically, GPCR's have represented a “gold mine” for drug discovery, and over 30% of currently approved prescription medicines act as either agonists or antagonists at such sites<sup><a onclick="showRef(event, 'jm040084cb00003'); return false;" href="javascript:void(0);" class="ref jm040084cb00003">3</a></sup>. With some notable exceptions, most successes have been achieved by drugs interacting at receptors for the simpler ligands, particularly the monoamines; the purpose of this study was to facilitate the design and synthesis of new drugs that compete with the more complex peptide ligands.<sup><a onclick="showRef(event, 'jm040084cb00004'); return false;" href="javascript:void(0);" class="ref jm040084cb00004">4</a></sup></div><div class="NLM_p">The GPCRs have been classified<sup>1</sup> into five categories (A, rhodopsin-like; B, secretin-like; C, metabotropic glutamate-like; D, fungal pheromone; E, c-AMP) on the basis of extensive phylogenetic studies, and these categories have been further subdivided into subclasses on the basis of the level of protein sequence homology (e.g., A−I, melanocortin; B−II, parathyroid). Those receptors that are activated by peptide ligands do not neatly fall into one of these subcategories but are distributed within subclasses A and B. The current study was focused on receptors within these subclasses that are activated by positively charged peptide ligands exemplified by receptors for melanocortins (MC, A−I), gonadotropin releasing hormone (GNRH, A−III), bradykinin (A−IV), melanin concentrating hormone (MCH, A−V), calcitonin gene related peptide (CGRP, B−I), and vasoactive intestinal peptide (VIP, B−III). The increased interest in GPCRs activated by endogenous peptides is a reflection of the growing number of gene sequences that have now been assigned to such receptors and also the belief that the design of non-peptide ligands may now be a tractable problem, thereby overcoming the well-documented limitations of peptides themselves as drugs.<sup><a onclick="showRef(event, 'jm040084cb00005'); return false;" href="javascript:void(0);" class="ref jm040084cb00005">5</a></sup> In addition, many of the new orphan GPCRs may turn out to be opportunities for therapeutic intervention and those that are activated by peptides would naturally fall into this subcategory. </div><div class="NLM_p last">To date, strategies for the discovery of non-peptide ligands<sup><a onclick="showRef(event, 'jm040084cb00006'); return false;" href="javascript:void(0);" class="ref jm040084cb00006">4,6</a></sup> have included high-throughput screening of random, large compound collections and/or combinatorial libraries, progressive replacement of amide bonds in fragments of peptides thought to be critical for binding, and the synthesis of novel templates putatively mimicking a presumed secondary structural feature, such as a β-turn mimetic. While all of these approaches are valid and have had some notable successes, they do not offer a systematic approach for the family of receptors under review. Central to the current study is the concept that the “property space” associated for ligands for peptide-activated GPCRs is definable and forms only a small fragment of what is referred to as a “druglike” space. Here we describe in detail how this space may be characterized and hence associated with a designed combinatorial “test” library consisting of around 2000 molecule that is highly enriched in ligands to these receptors. Furthermore, we show that the designed library performed significantly better than a random set of equal size selected from our corporate compound collection when tested against three relevant receptors. It is essential to appreciate that the objective of this test set, and eventually of the completed library, is to define an area of drug space for GPCR−PA<sup>+</sup>. Once this has been defined, specific areas can subsequently be highly populated with novel molecules targeting a specific receptor in an iterative approach. Therefore, it is not anticipated that potent molecules for a given receptor will be found directly in this library. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Computational Chemistry Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00654" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00654" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><a name="jm004702706865" id="jm004702706865"></a>We have defined “drug space” computationally by an analysis of drugs and druglike molecules known in the literature and representative, biologically active “in-house” compounds selected from over 20 historical discovery programs at Neurocrine; collectively this set totals 81 560 and the space it defines is termed “NBI-drug space” (NBI-DS). To delineate GPCR−PA<sup>+</sup> property space as a portion of the property space occupied by all drugs, a database composed of 630 molecules active against over 40 distinct GPCRs was constructed from data available in the literature and from Neurocrine's proprietary compound database. So as not to populate the set with the vast number of ligands described for the “classical” monoaminergic receptors, which were the subject of intense study during the 1970−1990 era, or for specific, well-studied GPCR−PA<sup>+</sup> (e.g. somatostatin<sup><a onclick="showRef(event, 'jm040084cb00007'); return false;" href="javascript:void(0);" class="ref jm040084cb00007">7</a></sup>), only representative structures for each receptor were included. </div><div class="NLM_p">Using the DiverseSolutions software package, BCUT metrics<sup><a onclick="showRef(event, 'jm040084cb00008 jm040084cb00009'); return false;" href="javascript:void(0);" class="ref jm040084cb00008 jm040084cb00009">8,9</a></sup> were calculated for NBI-DS. Examination of different combinations of metrics allowed us to select a five-dimensional space (H-bond donor and acceptor, two metrics of polarizability, and charge) in which the compounds are widely distributed. To accelerate the analysis of large virtual libraries, where spatial locations of millions of compounds must be calculated and compared, we have typically employed cell-based methods. With this approach, each of the five axes is divided into 10 bins resulting in the partitioning of the entire space into 100 000 individual cells. Comparing the location or index of the occupied cells can be used to measure the diversity/similarity between compounds or collections of compounds. Numbers of compounds within a given cell or range of cells is also a convenient measure of sampling density. </div><div class="NLM_p">Importantly, we have introduced the concept of analyzing neighboring cells which, in the context of five-dimensional space, requires that each occupied cell has 3<sup>5</sup> − 1 = 242 neighbors. Empirically, this is important for the following reasons:  (1) The objective of the screening library is to (almost) guarantee multiple hits against any GPCR−PA<sup>+</sup> target; with drug space for a specific receptor so defined, subsequent, more dense sampling focused on this space will lead to the nanomolar ligands required of drug candidates. Thus, while highly active molecules reside within a few cells or even a single cell, we have shown that progressively less active compounds (but still active!) are layered in neighboring cells. (2) Different levels of receptor promiscuity may determine the volume of drug space for a given receptor; it is well-established, for example, that the dopamine D<sub>4</sub> receptor is highly promiscuous based on the ease with which antagonists can be made. (3) Compounds residing in one cell may lie at, or close to, the boundaries of that cell so that compounds in the nearest neighboring cell may actually be closer than if they were located at the extremities of the occupied cell. </div><div class="NLM_p">The locations of compounds known to be active against GPCRs activated by positively charged peptide ligands projected into three of the five dimensions of this diversity space is shown graphically in Figure 1<a data-tab="pane-pcw-Figures" href="#jm040084cf00001" id="" class=" internalNav"></a>A. Analysis of their cell occupancies together with other categories of ligands in the full five-dimensional space is presented in Table 1<a class="ref showTableEvent internalNav" data-ID="jm040084ct00001" href="#jm040084ct00001"></a>. The various GPCR ligand classes occupy only small regions of this chemical space as compared to the original 81 560 compounds used to define its extent. Thus, GPCR ligands occupy 397 cells, whereas the broader collection, the NBI drug space set, occupies 8506 cells. If neighboring cells are included, all GPCR ligands occupy only 19% of drug space; in <a name="jm004702706866" id="jm004702706866"></a> addition, GPCR−PA<sup>+</sup> ligands form a distinct subspace (6%). An important goal of this preliminary investigation was then to evaluate whether these smaller regions of GPCR and GPCR−PA<sup>+</sup> space reflect a true mapping of properties common to these receptor classes and are predictive for designing new ligands or are simply due to the smaller numbers of compounds used to define the regions, compared to the overall space. <div id="_i5"><figure id="jm040084cf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00001.jpeg" id="_i6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 (A) Set of known non-peptide ligands for GPCR−PA<sup>+</sup> (filled circles) mapped into three dimensions of BCUT space vs the WDI−NBI (gray dots; representative sampling only). (B) Projection of the test set (crosses) into drug space using the BCUT methodology each compared to GPCR−PA<sup>+</sup> space (filled circles). (<i>P</i> = polarizability axis).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm040084ct00001"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 1.  Composition of the GPCR Training, Test, and Random Sets</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">ligand set</td><td class="colsep2 rowsep2">no. of ligands</td><td class="colsep2 rowsep2">no. of cells occupied</td><td class="colsep2 rowsep2">occupied + neighbor cells</td><td class="colsep2 rowsep2">% drug space (NBI-DS) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">all drugs + NBI (NBI-DS) </td><td class="colsep2 rowsep2">81560 </td><td class="colsep2 rowsep2">8506 </td><td class="colsep2 rowsep2">91080 </td><td class="colsep2 rowsep2">91 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">all GPCR </td><td class="colsep2 rowsep2">630 </td><td class="colsep2 rowsep2">397 </td><td class="colsep2 rowsep2">19225 </td><td class="colsep2 rowsep2">19 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">monoamines </td><td class="colsep2 rowsep2">201 </td><td class="colsep2 rowsep2">165 </td><td class="colsep2 rowsep2">8727 </td><td class="colsep2 rowsep2">9 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">monoacids </td><td class="colsep2 rowsep2">35 </td><td class="colsep2 rowsep2">34 </td><td class="colsep2 rowsep2">2853 </td><td class="colsep2 rowsep2">3 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">GPCR-PA<sup>+</sup></td><td class="colsep2 rowsep2">111 </td><td class="colsep2 rowsep2">97 </td><td class="colsep2 rowsep2">6437 </td><td class="colsep2 rowsep2">6 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">GPCR-PA<sup>-</sup></td><td class="colsep2 rowsep2">106 </td><td class="colsep2 rowsep2">79 </td><td class="colsep2 rowsep2">4050 </td><td class="colsep2 rowsep2">4 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">random set </td><td class="colsep2 rowsep2">2024 </td><td class="colsep2 rowsep2">1169 </td><td class="colsep2 rowsep2">31098 </td><td class="colsep2 rowsep2">31 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">random basic set </td><td class="colsep2 rowsep2">1401 </td><td class="colsep2 rowsep2">352 </td><td class="colsep2 rowsep2">15174 </td><td class="colsep2 rowsep2">15 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">test set </td><td class="colsep2 rowsep2">2025 </td><td class="colsep2 rowsep2">506 </td><td class="colsep2 rowsep2">10692 </td><td class="colsep2 rowsep2">11</td></tr></tbody></table></div></div></div><div class="NLM_p"><b>Virtual Library Enumeration.</b> There were 19 templates in the Neurocrine template database with at least one basic center in the central scaffold (Figure 2<a data-tab="pane-pcw-Figures" href="#jm040084cf00002" id="" class=" internalNav"></a>) that were selected for computational analysis so that this key feature would be present in all of the final molecules. Electronic lists of commercially available reagents (alkyl halides, aldehydes, carboxylic acids, and sulfonyl halides) that serve as side chains in each of these templates were prepared using the Available Chemicals Directory (ACD). Reagents with reactive functionality that could not be blocked with a protecting group, potentially toxic moieties (polyaromatics, Michael acceptors, etc.), or reagents that would give rise to compounds in the virtual library with molecular weights greater than 600 were discarded from the set. Clusters within these lists were established so that a subset of each reagent list could be selected, since the virtual molecules were stored as structural data (sd) files and electronic storage space was limited. Combination of templates and reagents lists yielded virtual molecules that spanned the NBI-drug space. The reagent lists would be fully expanded in the event of a positive hit from the screening process (i.e. as part of the optimization process). The virtual libraries were assembled with the aid of a commercially available computer program (Afferent Structure 3.0) to yield a virtual collection summarized in Table 2<a class="ref showTableEvent internalNav" data-ID="jm040084ct00002" href="#jm040084ct00002"></a>. <div id="_i7"><figure id="jm040084cf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00002.jpeg" id="_i8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Templates selected for virtual library enumeration (points of monomer attachment are via NH or COOH groups). Numbers located near to the reacting functional group indicate an association with a specific monomer listed in Table 2. In some instances, because of symmetry in the template, this nomenclature is redundant. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm040084ct00002"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 2.  Summary of the Virtual Library</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">template</td><td class="colsep2 rowsep2">available monomer 1</td><td class="colsep2 rowsep2">NBI-VL monomer 1</td><td class="colsep2 rowsep2">available monomer 2</td><td class="colsep2 rowsep2">NBI-VL monomer 2</td><td class="colsep2 rowsep2">available monomer 3</td><td class="colsep2 rowsep2">NBI-VL monomer 3</td><td class="colsep2 rowsep2">virtual library </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">305694 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">305694 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3a </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">634 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">291006 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3b </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">634 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">291006 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3c </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">305694 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">RCOCl </td><td class="colsep2 rowsep2">282 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">129438 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">305694 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">RNH2 </td><td class="colsep2 rowsep2">719 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">330021 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">RCOCl </td><td class="colsep2 rowsep2">282 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">187812 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">8 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">RCOCl </td><td class="colsep2 rowsep2">282 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">187812 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">RNH2 </td><td class="colsep2 rowsep2">719 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">478854 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">10<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">RNH2 </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">RNH2 </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">100 </td><td class="colsep2 rowsep2">640000 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">11<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">100 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">100 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">800000 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">12a<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">100 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">100 </td><td class="colsep2 rowsep2">RNH2 </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">800000 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">12b<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">100 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">RNH2 </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">800000 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">12c<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">100 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">RNH2 </td><td class="colsep2 rowsep2">80 </td><td class="colsep2 rowsep2">640000 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">13 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">634 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">422244 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">634 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">291006 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">15 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">634 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">291006 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">16 </td><td class="colsep2 rowsep2">RSO2Cl </td><td class="colsep2 rowsep2">158 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">72522 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">17 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">634 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">422244 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">18 </td><td class="colsep2 rowsep2">RCO2H </td><td class="colsep2 rowsep2">634 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">422244 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">19 </td><td class="colsep2 rowsep2">RCHO </td><td class="colsep2 rowsep2">666 </td><td class="colsep2 rowsep2">RCl </td><td class="colsep2 rowsep2">459 </td><td class="colsep2 rowsep2">NA </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">305694 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">total </td><td class="colsep2 rowsep2">9025685</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Template has three positions of diversity.</p></div></div></div><div class="NLM_p">The five BCUT metrics discussed above were calculated for all of the virtual compounds (Table 2) in order to evaluate their location relative to the GPCR−PA<sup>+</sup> space defined above. Virtual libraries selected for synthesis by the design algorithm were first explored synthetically by generation of a small (typically 20 compound) trial library, and synthetic failure resulted in rejection of the template. A substitute template that closely mirrored the drug space defined by the rejected templates was then selected. Of the 19 templates (Figure 2) that were subjected to computational analysis, seven were utilized in library production (boxed in Figure 2) to form the basis of the 2025 member test set. </div><div class="NLM_p"><b>Synthetic Chemistry.</b> The design principle for the test set required that a basic nitrogen atom, which will be predominantly protonated at physiological pH, be present in each molecule. As a result, the synthetic chemistry was restricted to reactions that preserved at least one basic nitrogen atom in the original core. Combinatorial libraries were prepared around templates <b>3</b>, <b>4</b>, <b>6</b>, <b>9</b>, <b>13</b>, <b>14</b>, and <b>18 </b>utilizing synthetic procedures analogous to those described in Schemes<a data-tab="pane-pcw-Figures" href="#jm040084ch00001" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm040084ch00002" id="" class=" internalNav"></a> 1 and 2. The template was orthogonally protected and synthesized on a 20-g scale. Side chain reagents were then reacted with the unprotected functional group and the intermediates were purified via automated flash chromatography.<sup><a onclick="showRef(event, 'jm040084cb00010'); return false;" href="javascript:void(0);" class="ref jm040084cb00010">10</a></sup> The final compounds were then purified and characterized by automated preparative high performance liquid chromatography coupled to a mass spectrometer.<sup><a onclick="showRef(event, 'jm040084cb00011'); return false;" href="javascript:void(0);" class="ref jm040084cb00011">11</a></sup> Each was prepared on a 10-mg scale and only compounds that were greater than 85% pure were screened to avoid misinterpretation of biological data. Validated hits were purified to >95% purity prior to <i>K</i><sub>i</sub> determination. <div id="_i9"><figure id="jm040084ch00001" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084ch00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00001.jpeg" id="_i10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 1.  Compounds Derived from Template <b>3</b> and Synthetic Routes<i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> Reagents:  (a) R<sup>1</sup>CHO, Na(OAc)<sub>3</sub>BH, DCE, rt, 4 h; or (2-bromoethyl)benzene or 1-bromo-3,5,5-trimethylhexane, DMF, Hunig's base, 40°C, 12 h; (b) TFA, DCM, 1 h, rt; (c) R<sup>2</sup>CO<sub>2</sub>H, EDCI, HOBT, DMF/NMP, rt, 12 h; (d) 5-(4-chlorophenyl)-2-furaldehyde, Na(OAc)<sub>3</sub>BH, DCE, rt, 12 h.</p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084ch00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm040084ch00002" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084ch00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00002.jpeg" id="_i11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 2.  Synthetic Chemistry Associated with Template <b>4</b><i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> Reagents:  (a) R<sup>1</sup>CO<sub>2</sub>H, EDCI, HOBT, DCM, rt, 4 h, 40−85%; (b) TFA, DCM, rt, 1 h, 100%; (c) R<sup>2</sup>Br, Hunig's base, 40 °C, 16 h, 50−85%.</p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084ch00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><a name="jm004702706867" id="jm004702706867"></a>Synthetic procedures for two templates (<b>3</b>, <b>4</b>), which afforded several of the hits for the MCH1−R, are discussed in more detail. Derivatives of template <b>3</b> were prepared (Scheme 1) from the monoprotected diamine 3-Boc amino pyrrolidine. Orthogonal protection of the primary amine was required in order to conduct a selective <i>N</i>-alkylation of the secondary amine, and the Boc protecting group was chosen due to the fact that it is removed rapidly under acidic conditions at room temperature. The intermediates <b>20a </b>and <b>20b </b>were prepared by reductive amination of 3-Boc amino pyrrolidine with the corresponding aldehydes in the presence of diisopropylethylamine (3 equiv) and sodium triacetoxyborohydride.<sup><a onclick="showRef(event, 'jm040084cb00012'); return false;" href="javascript:void(0);" class="ref jm040084cb00012">12</a></sup> The intermediates <b>20c </b>and <b>20d</b> were respectively prepared by N-alkylation of 3-Boc amino pyrrolidine with (2-bromoethyl)benzene and 1-bromo-3,5,5-trimethylhexane in the presence of diisopropylethylamine in DMF at 40 °C. Deprotection of <b>20a</b><b>−</b><b>d</b> was accomplished by treatment with trifluoroacetic acid in dichloromethane, and then the intermediate primary amines <b>21a</b><b>−</b><b>c</b> were reacted with the corresponding carboxylic acids in the presence of EDCI and HOBT to yield the final compounds exemplified by <b>3a</b>−<b>3g</b>. Dichloroethane solutions of amines <b>21b</b> and<b> 21d </b>were added to the corresponding aldehyde in the presence of diisopropylethylamine (3 equiv) and sodium <a name="jm004702706868" id="jm004702706868"></a> triacetoxyborohydride. The resultant mixtures were agitated at room temperature for 12 h to yield the diamines <b>3h </b>and<b> 3i</b>. </div><div class="NLM_p last">The synthesis of the active compounds derived from 2-aminomethylpyrrolidine template <b>4</b> is outlined in Scheme 2. Acylation of the monoprotected diamines (<i>R</i>)- and (<i>S</i>)-2-aminomethyl-1-Boc-pyrrolidine with the corresponding carboxylic acids in DMF in the presence of EDCI and HOBT at room temperature for 12 h yielded the intermediate amides (<i>R</i>)- and (<i>S</i>)-<b>22a,b</b>. Deprotection of (<i>R</i>)- and (<i>S</i>)-<b>22a,b</b> was accomplished with TFA in dichloromethane, and the product secondary amines (<i>R</i>)- and (<i>S</i>)-<b>23a</b><b>,</b><b>b</b> were then reacted with either 3,4-dichlorobenzyl bromide or phenethyl bromide in 3 equiv of diisopropylethylamine and <i>N</i>-methylpyrrolidinone as solvent to provide (<i>R</i>)- and (<i>S</i>)-<b>4a,b</b>. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Screening Results. </b>The 2025 compounds in the test set were arrayed in 96-well plates and dissolved in DMSO to a standard concentration of 15 μM. A “random set” of 2024 compounds was selected from Neurocrine's corporate screening collection and used as a control library. For practical convenience, it was possible to retrieve and assay only plates of compounds rather than discrete samples for the “random set”. Each of 23 random plates selected contained 88 compounds per plate. We were concerned that the designed set may generate more hits than the random set, simply because the ubiquitous presence of a positive charge in the former may bias the outcome. For that reason, we also wished to compare the designed set's performance against a second set that was still random other than having an obligatory basic center. Fortuitously, we were in a position to make this comparison because all compounds made at Neurocrine are simultaneously assayed in all current screens, since we expect some “cross-talk” between the GPCRs within the area of interest. Thus, as before, compounds (not plates on this occasion) were selected at random from a list of over 30 000 compounds bearing a basic center. The caveat here is that compounds originally designed as ligands for either of the three receptors employed in this study were deselected from the pick for that receptor, which resulted in a 1401 member “random basic” set. Both the test and random sets of compounds were simultaneously tested for activity in three high-throughput screens (HTS) against receptors of high therapeutic potential. The melanocortin-4 receptor (MC4−R, obesity<sup><a onclick="showRef(event, 'jm040084cb00013 jm040084cb00014'); return false;" href="javascript:void(0);" class="ref jm040084cb00013 jm040084cb00014">13,14</a></sup>), melanin concentrating hormone receptor (MCH1−R, obesity and depression<sup><a onclick="showRef(event, 'jm040084cb00015 jm040084cb00016'); return false;" href="javascript:void(0);" class="ref jm040084cb00015 jm040084cb00016">15,16</a></sup>), and gonadotropin-releasing hormone receptor (GnRH−R, endometriosis, uterine fibroids, prostate cancer<sup><a onclick="showRef(event, 'jm040084cb00017 jm040084cb00018'); return false;" href="javascript:void(0);" class="ref jm040084cb00017 jm040084cb00018">17,18</a></sup>) are members of the class A subclass of GPCRs activated by peptides bearing a positive charge. Compounds were screened in singlet at 15 μM in radioligand binding assays in standard HTS mode in 96-well format. Each plate contained six “total” wells containing no compounds and six “background” wells containing an excess of a cognate competitor to define specific binding for each plate (total cpm − backgound cpm). The <i>Z</i>‘ factor of each assay plate was approximately 0.5, verifying the suitability of the assays for HTS.<sup><a onclick="showRef(event, 'jm040084cb00019'); return false;" href="javascript:void(0);" class="ref jm040084cb00019">19</a></sup> Compounds that displaced 50% or more of the specifically bound radioligand were designated as a “hit” and their activity was confirmed by repeating them in duplicate on a separate day. Compounds that displaced 50% or more of the specifically bound radioligand in the follow-up assay were defined as “confirmed hits” (Table 3<a class="ref showTableEvent internalNav" data-ID="jm040084ct00003" href="#jm040084ct00003"></a>). Active compounds were then subjected to 12-point dose−response curve analysis to determine binding affinities (typical curves are exemplified for MCH1−R in Figure 3<a data-tab="pane-pcw-Figures" href="#jm040084cf00003" id="" class=" internalNav"></a>) and the resulting <i>K</i><sub>i</sub>s from templates <b>3</b> and <b>4</b> are listed in Table 4<a class="ref showTableEvent internalNav" data-ID="jm040084ct00004" href="#jm040084ct00004"></a>. A second random set of 1401 compounds, which all contain a basic nitrogen, was selected from Neurocrine's corporate collection and used as a basic/random control library. This random basic set was tested under identical conditions to the test and random sets. <div id="_i13"><figure id="jm040084cf00003" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00003.jpeg" id="_i14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Representative dose−response curves of MCH−R hits from the test set. The <i>K</i><sub>i</sub> values are derived from the IC<sub>50</sub> values of inhibition using the Cheng−Prusoff equation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm040084ct00003"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 3.  Distribution of Confirmed Hits for HTS for the Test, Random, and Random Basic Sets<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">confirmed hits</td><td class="colsep2 rowsep2" colspan="3">hit rate (%)</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">receptor</td><td class="colsep2 rowsep2">test set</td><td class="colsep2 rowsep2">random set</td><td class="colsep2 rowsep2">random basic set</td><td class="colsep2 rowsep2">test set</td><td class="colsep2 rowsep2">random set</td><td class="colsep2 rowsep2">random basic set </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">hMC4 </td><td class="colsep2 rowsep2">61 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">3.00 </td><td class="colsep2 rowsep2">0.05 </td><td class="colsep2 rowsep2">0.35 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">rMCH1 </td><td class="colsep2 rowsep2">123 </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">6.10 </td><td class="colsep2 rowsep2">0.30 </td><td class="colsep2 rowsep2">0.35 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">GnRH </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">0.44 </td><td class="colsep2 rowsep2">0.10 </td><td class="colsep2 rowsep2">0.07</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Confirmed hits represent compounds that displaced 50% or more binding in both the original HTS and the confirmation assay. The hit rate was determined as (no. confirmed hits/total no. of compounds tested) × 100. The test set, random set, and random basic set contained 2025, 2024, and 1401 compounds, respectively.</p></div></div><div class="NLM_table-wrap" id="jm040084ct00004"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 4.  Summary of Binding Data for MCH Hits from Templates 3 and 4<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cu00004a.gif" alt="" id="_i15" /></img></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">compd</td><td class="colsep2 rowsep2">Y</td><td class="colsep2 rowsep2">R<sup>1</sup></td><td class="colsep2 rowsep2">R<sup>2</sup></td><td class="colsep2 rowsep2">MCH1-R binding, <i>K</i><sub>i</sub> (μM)<sup><i>a</i></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">SNAP-7941 </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">0.002 (±0.001)<i><sup>b</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3a </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">PhCH<sub>2</sub></td><td class="colsep2 rowsep2">PhCH<sub>2</sub></td><td class="colsep2 rowsep2">>10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3b </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">PhCH<sub>2</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">PhCH<sub>2</sub></td><td class="colsep2 rowsep2">>10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3c </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">3,4-di-Cl-C<sub>6</sub>H<sub>3</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">PhCH<sub>2</sub></td><td class="colsep2 rowsep2">0.98 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3d </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">3,4-di-Cl-C<sub>6</sub>H<sub>3</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">PhCH<sub>2</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">>10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3e </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">3,4-di-Cl-C<sub>6</sub>H<sub>3</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">4-PhO-C<sub>6</sub>H<sub>4</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">0.51 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3f </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">3,4-di-Cl-C<sub>6</sub>H<sub>3</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">A </td><td class="colsep2 rowsep2">0.63 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3g </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">3,4-di-Cl-C<sub>6</sub>H<sub>3</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">B </td><td class="colsep2 rowsep2">0.36 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3h </td><td class="colsep2 rowsep2">CH<sub>2</sub></td><td class="colsep2 rowsep2">Ph<sub>2</sub>CHCH<sub>2</sub></td><td class="colsep2 rowsep2">C </td><td class="colsep2 rowsep2">1.7 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3i </td><td class="colsep2 rowsep2">CH<sub>2</sub></td><td class="colsep2 rowsep2">D </td><td class="colsep2 rowsep2">C </td><td class="colsep2 rowsep2">5.0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">(<i>R</i>)-4a </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">3,4-di-Cl-C<sub>6</sub>H<sub>3</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">B </td><td class="colsep2 rowsep2">9.8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">(<i>S</i>)-4a </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">3,4-di-Cl-C<sub>6</sub>H<sub>3</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">B </td><td class="colsep2 rowsep2">6.0 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">(<i>R</i>)-4b </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">PhCH<sub>2</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">PhCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">3.1 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">(<i>S</i>)-4b </td><td class="colsep2 rowsep2">CO </td><td class="colsep2 rowsep2">PhCH<sub>2</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">PhCH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub></td><td class="colsep2 rowsep2">>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Value is an average of three determinations unless reported otherwise.<i><sup>b</sup></i><sup></sup> <i>n</i> = 7.</p></div></div></div><div class="NLM_p">As an initial proof of concept for the design process, the number of hits identified in the designed set was significantly greater than those obtained from the random sets for each of the three receptors (Table 3). Hit enrichment rates ranged from 4.5-fold (GnRH−R) to 61-fold (MC4−R) when compared with the random set. Although the random basic set picked up more hits than the random set for MC4, the designed set was still more impressive, even allowing for the smaller size of the former. Thus, for MC4−R, the 2025 compound test set gave a number of hits that would have required screening more than 120000 compounds of a typical corporate collection. The absolute number of hits varied from 9 for the GnRH receptor to 123 for the MCH1 receptor. </div><div class="NLM_p">On the basis of cell occupancy (Table 1), it is clear that the designed set (506 and 10 692 cells occupied and occupied plus neighboring cells, respectively) is less diverse than the random set (1169 and 31098). This was verified by a standard diversity/similarity analysis, whereby MACCS keys were used as the fingerprint and Tanimoto coefficients used as the similarity measure. The method was used to calculate the similarity of every possible pair of compounds in each set with the result that the designed set was least diverse (mean similarity 0.56), the random set most diverse (0.33), and the random basic set (0.48) somewhere between the two. </div><div class="NLM_p"><b>Structure</b><b>−</b><b>Activity Observations:  MCH1</b><b>−</b><b>R.</b> The current library was not designed to establish structure−activity predictions for each receptor; the intent was more that the set should provide a starting point from which such predictions can be made. Since many of the highest affinity ligands that bound the MCH receptor identified from this library originated from templates <b>3</b> and <b>4</b> (Table 4), some observations may be made. SNAP-7941,<sup>16</sup> a known selective and highly potent <a name="jm004702706869" id="jm004702706869"></a> MCH1−R ligand, was used as a positive reference in the biological assays for <i>K</i><sub>i</sub> determination. Benzyl (<b>3a</b>) and phenethyl (<b>3b</b>) substituents at the secondary nitrogen of the amino amide derivative of template <b>3</b> did not result in a compound that bound MCH1−R, but the addition of chlorine atoms at the 3 and 4 positions of the benzyl group did result in a compound with moderate affinity for the MCH1 receptor (<b>3c</b>, <i>K</i><sub>i</sub> = 0.98 μM). Increasing the distance between the core and the phenyl group at R<sup>2</sup> did not improve the biological activity, as seen in compound <b>3d</b>, but increased binding affinity was observed when an additional phenoxy substituent was appended to the R<sup>2</sup> phenyl group (<b>3e</b>, <i>K</i><sub>i</sub> = 0.51 μM). Other related high-affinity MCH1−R compounds from the library incorporate a bicyclic moiety, as illustrated in compounds <b>3f</b> and <b>3g</b>, which possess <i>K</i><sub>i</sub> values of 0.63 and 0.36 μM, respectively. </div><div class="NLM_p">Reduction of the amide bond in template <b>3</b> also resulted in diamines that are moderate affinity ligands for MCH1−R. Two diamine derivatives, <b>3h</b> (<i>K</i><sub>i</sub> = 1.7 μM) and <b>3i</b> (<i>K</i><sub>i</sub> = 5 μM), share an arylfuranyl group that is one atom closer to the core in the R<sup>2</sup> region of the template, and the R<sup>1</sup> region is significantly more lipophilic than the corresponding region in the amino amide derivatives. This suggests that the optimal relationship between side chain structure and MCH1−R affinity of the template <b>3</b> diamines will be distinct to that of aforementioned amino amides <b>3a</b><b>−</b><b>g</b>. </div><div class="NLM_p">In general there were fewer and less active compounds derived from template <b>4</b> versus template <b>3</b>, perhaps suggesting that the extra conformational freedom is unfavorable entropically. Where a direct comparison can be made, such as between <b>3g</b> (<i>K</i><sub>i</sub> = 0.36 μM) and <b>(</b><b><i>S</i></b><b>)-4a</b> (<i>K</i><sub>i</sub> = 6.0 μM), almost a 20-fold difference in activity is observed. A slight improvement in MCH1−R binding is realized when the R<sup>1</sup> and R<sup>2</sup> substituents are phenethyl and phenpropyl, respectively [<b>(</b><b><i>R</i></b><b>)</b>-<b>4b</b>, <i>K</i><sub>i</sub> = 3.1 μM]. The difference in the electronic nature and shape of the side chains as well as a difference in the absolute stereochemistry of the template [<b>(</b><b><i>R</i></b><b>)-4a </b>and <b>(</b><b><i>S</i></b><b>)-4b</b> are less active than the corresponding enantiomers] again suggests distinct SAR within this subseries of MCH1−R ligands. </div><div class="NLM_p last">Using these “hints” of SAR, the synthesis of more focused libraries is underway; it is clear that both templates <b>3</b> and <b>4</b> could be used as starting points for further lead optimization. The designed library adds yet another dimension; however, actives are spread across more than one series of compounds (i.e. derived from several templates). This facet strengthens the resulting computational model, the next step being to derive 3-D pharmacophores and further increases the chance of being able to optimize activity below 10 nM following further iterations of design, synthesis, and assay. In the context of lead optimization, it is important to note that the set contains 1036 and 864 mono- and diamines, respectively (the remainder are triamines), and since all the hits are within these two classes, these properties should not present any difficulties. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04303" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04303" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">While the random set was shown to occupy 1169 cells totaling 31098 when neighboring cells were included, the test designed library (Figure 1B) physically occupied 506 and, with neighbors, 10692 cells, respectively. The screening hits from each of the three receptors have been projected into GPCR−PA<sup>+</sup> space (Figure 4<a data-tab="pane-pcw-Figures" href="#jm040084cf00004" id="" class=" internalNav"></a>) noting, of course, that we are unable to visualize simultaneously the remaining two dimensions. Of the 193 total hits, it was observed that MCH1−R, MC4−R, and GnRH−R respectively occupied 73, 35, and 7 cells, and with the exception of one GnRH−R hit, all are located in cells occupied by GPCR−PA<sup>+</sup> or their neighbors. This is very suggestive that GPCR−PA<sup>+</sup> space does indeed exist and <a name="jm004702706870" id="jm004702706870"></a> can assist ligand design for this subclass of receptors. Of the 73 cells occupied by the hits for MCH1−R, 15 and two overlapped with MC4−R and GnRH−R, respectively. For the MC4 receptor, 35 occupied cells had two cells in common with GnRH−R. Two cells had hits for all three receptors, and this information may be relevant for receptor selectivity. The GnRH hit referred to above as an “exception” is a consequence of the convenience of matrix synthesis, which results in several compounds being prepared that fall outside of the desired space. In contrast, the small number of hits from the random set arises from a significant, albeit low, occupation of GPCR−PA<sup>+</sup> space. <div id="_i17"><figure id="jm040084cf00004" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00004.jpeg" id="_i18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Screening hits (squares) from the test and random sets projected into drug space against a background of GPCR−PA<sup>+</sup> ligands (open circles). (A) MCH−R hits from the test set; (B) GnRH−R hits from the test set; (C) MC4−R hits from the test set; (D) hits from the random set [(1) region of GnRH random hits; (2) MC4, and (3) MCH)].</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The data presented above show that a test library of 2025 compounds synthesized on the basis of their location within a GPCR−PA<sup>+</sup> ligand-enriched region of drug space is highly enriched (4.5−61-fold) in ligands for three receptors of this subfamily; the absolute hit rate ranged from 0.45% to 6%. If one postulates the existence of a discrete region of chemistry space that is enriched in ligands for this class of receptor, as is suggested by our preliminary data, then this variability in hit rate and enrichment ratio could be due to uneven sampling of either compounds or receptors in these experiments. As was discussed above, we observed that individual receptors appear to prefer compounds that reside in discrete subregions. It may be that the population of compounds in the initial set was not evenly distributed into regions of chemical space preferred by each of the three test receptors. Alternatively, the three receptors selected may have intrinsic characteristics (such as the number, size, shape, or complexity of potential ligand binding pockets) that result in greater or lesser degrees of ligand selectivity. Further, by querying the library with only three receptors, each of which can detect ligands only from their cognate subregion, one would not expect to delineate the entire range of the GPCR−PA<sup>+</sup> ligand-enriched region. </div><div class="NLM_p last">The design and synthesis of the completed library are now underway to attempt to fill, wherever possible, the remaining unoccupied cells together with their neighbors. What is already certain is that the task cannot be completed by another round of matrix-based parallel synthesis; rather a cherry-picking approach will be adopted so as to avoid the synthesis of large numbers of compounds falling outside the desired space and to avoid overpopulating cells already covered. It is expected that around 7000 compounds will be the minimum required to cover the relevant space (GPCR−PA<sup>+</sup> occupied plus neighboring cells = 6437) although, with some redundancy, this may be significantly larger. On the other hand, initial analysis suggests that it may not be possible to occupy some neighboring cells with real molecules, and this will reduce the number of compounds to be made. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30108" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30108" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Synthetic Chemistry. General Methods.</b> Proton NMR spectra were obtained with a Bruker 500 MHz spectrometer at Numega Labs. Inc., San Diego, CA, and chemical shifts are <a name="jm004702706871" id="jm004702706871"></a> reported in parts per million (δ) downfield from tetramethylsilane as internal standard. Matrix-assisted laser desorption/ionization (MALDI) FTMS experiments were performed on an IonSpec FTMS mass spectrometer at The Scripps Research Institute, San Diego, CA. Samples were irradiated with a nitrogen laser operated at 337 nm, and the laser beam was controlled by a variable attenuator and focused on the sample target. A Sciex API-150EX mass spectrometer operating in electrospray positive ionization mode was used to obtain [MH<sup>+</sup>] ion of the target molecules. Optical activity was measured at 589 nm, 25 °C, using a Perkin-Elmer 341 polarimeter. Key intermediates were synthesized in normal laboratory glassware and purified by automated flash chromatography (Jones Chromatography, Lakewood, CO). Parallel synthesis experiments were performed under a nitrogen atmosphere using a liquid handler (Gilson 215, Middleton, WI) to deliver each solution into a deep-well microtiter plate with 96 750-μL glass inserts. The plates were agitated on a vortexing heating block (J-Kem, St. Louis, MO). Evaporation of the volatiles in vacuo was performed using a Savant Discovery SpeedVac. Purification of final compounds was conducted on a prep-LCMS Dionex system, using an Alpha C18 30 × 75 mm reverse phase column at a flow rate of 45 mL/min. The mobile phase was a gradient of 95/5 to 5/95 A/B, where A = H<sub>2</sub>O−0.1% TFA, and B = CH<sub>3</sub>CN−0.1% TFA. All compounds after purification were reanalyzed on a reverse phase HPLC−MS system (DIONEX with ESI+ ionization mode) and shown to be at least 85% pure on the basis of both UV wavelengths (220, 254 nm) and total ion current (TIC) from the mass spectrum. </div><div class="NLM_p"><b>[1-(3,4-Dichlorobenzyl)pyrrolidin-3-yl]carbamic Acid </b><b><i>tert-</i></b><b>Butyl Ester (20a). </b>To a solution of 3-(<i>tert</i>-butoxycarbonylamino)pyrrolidine (1.5 g, 8.06 mmol, 1 equiv) and <i>N,N</i>-diisopropylethylamine (2.80 mL, 2.0 equiv) in dichloroethane (20 mL) was added 3,4-dichlorobenzaldehyde (1.455 g, 8.06 mmol). The solution was stirred at room temperature for 10 min and then NaBH(OAc)<sub>3</sub> (2.570 g, 12.09 mmol) was added. The resultant mixture was allowed to stir for an additional 10 h and then the reaction mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and washed with water (10 mL). The organic phase was dried over sodium sulfate and concentrated under reduced pressure. The crude product was purified by automated flash chromatography (gradient elution:  30% EtOAc in hexane to 60% EtOAc in hexane) to afford [1-(3,4-dichlorobenzyl)pyrrolidin-3-yl]carbamic acid <i>tert-</i>butyl ester as an oil (1.80 g, 66.0%). MALDI−FTMS calcd for C<sub>16</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>:  345.1131 [MH<sup>+</sup>]. Found:  345.1136. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 1.43 (s, 9H), 1.58 (m, 1H), 2.27 (m, 2H), 2.59 (m, 1H), 2.75 (m, 1H), 3.53 (m, 2H), 4.16 (s, 1H), 4.82 (s, 1H), 7.14 (d, <i>J</i> = 8.0 Hz, 1H), 7.36 (d, <i>J</i> = 8.0 Hz, 1H), 7.41 (s, 1H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 28.6, 32.8, 50.0, 52.5, 59.0, 61.0, 79.6, 128.1, 130.5, 130.7, 131.2, 132.6, 139.2, 155.5. </div><div class="NLM_p"><b>[1-(2,2-Diphenylethyl)pyrrolidin-3-yl]carbamic Acid </b><b><i>tert</i></b><b>-Butyl Ester (20b).</b> This compound was prepared according to the procedure described above using diphenylacetaldehyde in place of 3,4-dichlorobenzaldehyde (2.5 g, 84.5%). MALDI−FTMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>2</sub>O<sub>2</sub>:  367.2380 [MH<sup>+</sup>]. Found:  367.2362. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 1.40 (s, 9H), 1.50 (m, 1H), 2.10 (m, 1H), 2.40 (m, 1H), 2.48 (m, 1H), 2.68 (m, 1H), 2.77 (m, 1H), 3.12 (d, <i>J</i> = 7.6 Hz, 2H), 3.31 (m, 1H), 3.98 (m, 1H), 4.16 (t, <i>J</i> = 7.6 Hz, 1H), 7.14−7.30 (aromatic, 10H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 28.8, 32.6, 50.8, 52.0, 54.4, 62.1, 62.6, 80.2, 127.5, 129.2, 129.6, 145.3, 158.0. </div><div class="NLM_p"><b>[1-(3,5,5-Trimethylhexyl)pyrrolidin-3-yl]carbamic Acid </b><b><i>tert</i></b><b>-Butyl Ester (20c).</b> To a solution of 3-(<i>tert</i>-butoxycarbonylamino)pyrrolidine (1.5 g, 8.06 mmol, 1 equiv) and <i>N,N</i>-diisopropylethylamine (2.80 mL, 2.0 equiv) in anhydrous DMF (20 mL) was added 1-bromo-3,5,5-trimethylhexane (1.75 g, 8.46 mmol, 1.05 equiv). The reaction mixture was stirred at 40 °C for 12 h, and after cooling to room temperature, the solvent was removed under reduced pressure. The crude product was purified by automated flash chromatography (gradient elution:  30% EtOAc in hexane to 60% EtOAc in hexane with 5% methanol) to afford [1-(3,5,5-trimethylhexyl)pyrrolidin-3-yl]carbamic acid <i>tert</i>-butyl ester (1.3 g, 52.0%). MALDI−FTMS calcd for C<sub>18</sub>H<sub>36</sub>N<sub>2</sub>O<sub>2</sub>:  312.5 [MH<sup>+</sup>]. Found:  313.2841. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 0.92 (s, 9H), 0.94 (m, 1H), 0.96 (d, <i>J</i> = 6.5 Hz, 3H), 1.12 (AB, <i>J</i> = 5.8 and 14.0 Hz, 1H), 1.28 (AB, <i>J</i> = 2.4 and 14.0 Hz, 1H), 1.48 (s, 9H), 1.50 (m, 4H), 2.20 (m, 1H), 2.50 (m, 4H), 2.69 (m, 1H), 2.86 (m, 1H), 4.08 (m, 1H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 23.2, 28.7, 29.0, 30.4, 31.9, 32.3, 38.8, 50.6, 52.5, 54.0, 55.5, 55.7, 61.4, 80.2, 158.0. </div><div class="NLM_p"><b>1-(3,4-Dichlorobenzyl)pyrrolidin-3-ylamine, TFA Salt (21a).</b> To a stirred solution of [1-(3,4-dichlorobenzyl)pyrrolidin-3-yl]carbamic acid <i>tert-</i>butyl ester <b>20a</b> (350 mg, 1.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was added trifluoroacetic acid (2 mL). The resultant solution was stirred at room temperature for 2 h and the CH<sub>2</sub>Cl<sub>2</sub>/TFA solvent was removed under reduced pressure. The residue was coevaporated with CH<sub>2</sub>Cl<sub>2</sub>, dissolved in MeOH (10 mL), and purified by prep-LCMS to yield the desired TFA salt as a colorless liquid (211.6 mg, 86.4%). MALDI−FTMS calcd for C<sub>11</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>2</sub>:  245.0607 [MH<sup>+</sup>]. Found:  245.0596. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 2.08 (m, 1H), 2.52 (m, 1H), 3.13 (m, 1H), 3.29 (m, 1H), 3.42 (m, 2H), 4.04 (m, 1H), 4.20 (m, 2H), 4.88 (s, 2H), 7.42 (dd, <i>J</i> = 8.2 Hz, 1.9 Hz, 1H), 7.58 (d, <i>J</i> = 8.2 Hz, 1H), 7.71 (d, <i>J</i> = 1.9 Hz, 1H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 30.0, 50.0, 53.7, 57.4, 58.3, 131.0, 132.3, 133.2, 134.1, 144.5. </div><div class="NLM_p"><b>1-(2,2-Diphenylethyl)pyrrolidin-3-ylamine, TFA Salt (21b).</b> The compound was prepared according to the procedure given immediately above using <b>20b</b> in place of <b>20a</b> (176.0 mg, 66.0.%). MALDI−FTMS calcd for C<sub>18</sub>H<sub>22</sub>N<sub>2</sub>:  267.1856 [MH<sup>+</sup>]. Found:  267.1851. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 2.07 (m, 1H), 2.49 (m, 1H), 3.38 (m, 1H), 3.57 (m, 1H), 3.64 (m, 1H), 4.00 (m, 3H), 4.48 (t, <i>J</i> = 7.8 Hz, 1H), 4.86 (s, 1H), 7.24−7.31 (aromatic, 10H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 29.6, 49.6, 49.8, 55.2, 58.2, 60.9, 128.8, 129.0, 130.3, 130.4, 141.9, 142, 157. </div><div class="NLM_p"><b>1-(3,5,5-Trimethylhexyl)pyrrolidin-3-ylamine, TFA Salt (21c).</b> The compound was prepared according to the procedure given immediately above using <b>20c</b> in place of <b>20a </b>(255.0 mg, 81.5%). MALDI−FTMS calcd for C<sub>13</sub>H<sub>28</sub>N<sub>2</sub>:  213.2325 [MH<sup>+</sup>]. Found:  213.2320. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 0.93 (s, 9H), 0.97 (m, 1H), 1.05 (d, 3H), 1.16 (AB, <i>J</i> = 6.0 and 14.0 Hz, 1H), 1.27 (AB, <i>J</i> = 3.3, 3.4, 1.3 Hz, 1H), 1.62 (m, 2H), 1.75 (m, 1H), 2.2 (m, 1H), 2.65 (m, 1H), 3.30 (m, 3H), 3.6 (m, 1H), 4.15 (m, 1H), 4.89 (m, 3H). </div><div class="NLM_p"><b><i>N</i></b><b>-(1-Benzylpyrrolidin-3-yl)-2-phenylacetamide (3a).</b> To a solution of phenylacetic acid (40 mg, 0.3 mmol) in NMP (1.5 mL) was added a solution of commercially available 1-benzyl-pyrrolidin-3-ylamine (44 mg, 0.25 mmol) and <i>N,N</i>-diisopropylethylamine (130.0 μL, 0.75 mmol) in DMF (1.0 mL). This followed by a solution of EDCI (86 mg, 0.45 mmol) and HOBT (59 mg, 0.45 mmol) in NMP (1.0 mL). The reaction mixture was stirred at room temperature for 16 h and then purified by prep-LCMS to give the desired product as the TFA salt (62.0 mg, 85.0%). MALDI−FTMS calcd for C<sub>19</sub>H<sub>22</sub>N<sub>2</sub>O:  295.1805 [MH<sup>+</sup>]. Found:  295.1808. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 1.77 (m, 1H), 2.12 (m, 1H), 2.41 (m, 1H), 2.89 (m, 1H), 3.02 (m, 1H), 3.36 m, 1H), 3.46 (s, 2H), 3.70 (m, 1H), 4.08 (m, 1H), 4.26 (m, 1H), 4.79 (m, 1H), 7.25 (m, 5H), 7.43 (m, 5H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 30.6, 43.5, 47.2, 52.6, 58.8, 59.0, 115.7, 118.0, 127.1, 128.8, 128.9, 129.2, 129.4, 129.8, 130.4, 130.6, 135.2, 171.7. </div><div class="NLM_p"><b><i>N</i></b><b>-(1-Phenethylpyrrolidin-3-yl)-2-phenylacetamide (3b).</b> The compound was prepared according to the procedure described for compound <b>3a</b> using 1-phenethylpyrrolidin-3-ylamine<sup><a onclick="showRef(event, 'jm040084cb00020'); return false;" href="javascript:void(0);" class="ref jm040084cb00020">20</a></sup> in place of 1-benzylpyrrolidin-3-ylamine (54.9 mg, 70%). MALDI−FTMS calcd for C<sub>20</sub>H<sub>24</sub>N<sub>2</sub>O:  309.1961 [MH<sup>+</sup>]. Found:  309.1968. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 2.13 (m, 1H), 2.43 (m, 1H), 2.81 (m, 1H), 2.96 (m, 1H), 3.07 (m, 2H), 3.24 (m, 2H), 3.49 (s, 2H), 3.63 (m, 1H), 3.83 (m, 1H), 4.22 (m, 2H), 7.15−7.52 (aromatics, 10H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 24.0, 30.6, 43.5, 47.5, 53.5, 57.2, 60.2, 127.2−135.8, 171.8 </div><div class="NLM_p"><b><i>N</i></b><b>-[1-(3,4-Dichlorobenzyl)pyrrolidin-3-yl]-2-phenylacetamide (3c).</b> To a solution of phenylacetic acid (40 mg, 0.3 mmol) in NMP (1.5 mL) was added a solution of 1-(3,4-dichlorobenzyl)pyrrolidin-3-ylamine <b>21a</b> (73 mg, 0.25 mmol) <a name="jm004702706872" id="jm004702706872"></a> and <i>N,N</i>-diisopropylethylamine (130.0 μL, 0.75 mmol) in DMF (1.0 mL). This was followed by a solution of EDCI (86 mg, 0.45 mmol) and HOBT (59 mg, 0.45 mmol) in NMP (1.0 mL). The reaction mixture was stirred at room temperature for 16 h and then purified by prep-LCMS to give the desired product as the TFA salt (63.5 mg, 70.0%). MALDI−FTMS calcd for C<sub>19</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>2</sub>O:  363.1025 [MH<sup>+</sup>]. Found:  363.1026. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 2.12 (m, 1H), 2.44 (m, 1H), 2.87 (m, 1H), 3.03 (m, 1H), 3.34 d, <i>J</i> = 12.5 Hz, 1H), 4.17 (d, <i>J</i> = 12.5 Hz, 1H), 4.88 (m, 1H), 7.26 (m, 5H), 7.34 (dd, <i>J</i> = 8.2 Hz, 1.7 Hz, 1H), 7.50 (d, <i>J</i> = 8.2 Hz, 1H), 7.54 (s, 1H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 30.4, 43.4, 47.2, 52.9, 57.5, 59.3, 127.2, 128.8, 129.4, 129.7, 131.8, 132.4, 134.0, 135.1, 135.2, 171.8. </div><div class="NLM_p"><b><i>N</i></b><b>-[1-(3,4-Dichlorobenzyl)pyrrolidin-3-yl]-2-(4-phenoxyphenyl)acetamide (3e).</b> The compound was prepared according to the procedure described for compound <b>3c</b> using 4-phenoxyphenylacetic acid in place of phenylacetic acid (70.3 mg, 62%). MALDI−FTMS calcd for C<sub>25</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>2</sub>:  455.1288 [MH<sup>+</sup>]. Found:  455.1283. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 2.08 (m, 1H), 2.45 (m, 1H), 2.87 (m, 1H), 3.03 (m, 1H), 3.29 (m, 1H), 3.44 (s, 2H), 3.60 (m, 1H), 3.99 (d, <i>J</i> = 12.9 Hz, 1H), 4.12 (d, <i>J</i> = 12.9 Hz, 1H), 4.74 (m, 1H), 6.91−7.55 (aromatics, 12H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 30.6, 42.6, 47.6, 52.9, 57.7, 59.2, 118.0, 119.1, 123.5, 129.5, 129.9, 130.7, 131.7, 132.3, 133.9, 156.5, 157.3, 171.7. </div><div class="NLM_p"><b>2-Benzo[1,3]dioxol-5-yl-</b><b><i>N</i></b><b>-[1-(3,4-dichlorobenzyl)pyrrolidin-3-yl]acetamide (3f). </b>The compound was prepared according to the procedure described for compound <b>3c</b>, using 3,4-methylenedioxyphenylacetic acid in place of phenylacetic acid (86.0 mg, 85.0%). MALDI−FTMS calcd for C<sub>20</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub>:  407.0924 [MH<sup>+</sup>]. Found:  407.0915. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 2.12 (m, 1H), 2.45 (m, 1H), 2.89 (m, 1H), 3.06 (m, 1H), 3.35 (m, 1H), 3.37 (s, 2H), 3.68 (m, 1H), 4.03 (m, 1H), 4.19 (d, <i>J</i> = 13.0 Hz, 1H), 4.80 (m, 1H), 5.90 (s, 2H), 7.12 (dd, <i>J</i> = 8.2 Hz, 1.8 Hz, 1H), 7.32−7.54 (aromatics, 5H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 30.4, 43.0, 47.3, 53.0, 57.5, 59.2, 101.2, 108.6, 109.7, 122.5, 129.6, 131.8, 132.4, 133.9, 135.2, 146.9, 148.0, 172.0. </div><div class="NLM_p"><b>2-Benzo[</b><b><i>b</i></b><b>]thiophen-3-yl-</b><b><i>N</i></b><b>-[1-(3,4-dichlorobenzyl)pyrrolidin-3-yl]acetamide (3</b><b>g</b><b>).</b> This compound was prepared according to the procedure described for compound <b>3c</b>, using benzo[<i>b</i>]thiophene-3-acetic acid in place of phenylacetic acid (52.3 mg, 50.0%). MALDI−FTMS calcd for C<sub>21</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>2</sub>OS:  419.0746 [MH<sup>+</sup>]. Found:  419.0756. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub><i>-d</i>):  δ 2.04 (m, 1H), 2.38 (m, 1H), 2.77 (m, 1H), 2.97 (m, 1H), 3.24 (m, 1H), 3.47 (s, 1H), 3.55 (m, 1H), 3.72 (m, 1H), 3.88 (d, <i>J</i> = 12.8 Hz, 1H), 4.7 (d, <i>J</i> = 12.8 Hz, 1H), 4.74 (m, 1H), 7.26−7.82 (11H aromatic). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 30.4, 36.5, 48.0, 52.9, 57.5, 59.1, 122.1, 123.0, 124.4, 124.6, 125.0, 129.4, 129.6, 129.9, 131.7, 132.3, 133.9, 135.0, 138.8, 140.5, 170.6. </div><div class="NLM_p"><b>[5-(4-Chlorophenyl)furan-2-ylmethyl][1-(2,2-diphenylethyl)pyrrolidin-3-yl]amine (3h).</b> To a stirred solution of 1-(2,2-diphenylethyl)pyrrolidin-3-ylamine, TFA salt <b>21b</b> (114 mg, 0.3 mmol) in dichloroethane (750 μL) in the presence of <i>N,N</i>-diisopropylethylamine (110 μL, 0.6 mmol) was added 5-(4-chlorophenyl)-2-furaldehyde (57.0 mg, 0.28 mmol). The reaction mixture was stirred at room temperature for 10 min and then NaBH(OAc)<sub>3</sub> (63 mg, 0.3 mmol) was added and the mixture was further stirred for 2 h. The reaction mixture was then diluted with water (100 μL) and the crude mixture was directly purified by prep-LCMS without any workup to give the desired diamine as a bis-TFA salt (25.6 mg, 20.0%). MALDI−FTMS calcd for C<sub>29</sub>H<sub>29</sub>ClN<sub>2</sub>O:  457.2041 [MH<sup>+</sup>]. Found:  457.2039. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub><i>-d</i>):  δ 1.97 (m, 1H), 2.22 (m, 1H), 2.91 (m, 1H), 3.15 (m, 2H), 3.46 (s, 2H), 3.60 (m, 2H), 3.84 (m, 1H), 3.89 (s, 1H), 4.87 (t, <i>J</i> = 7.10 Hz, 1H), 6.37 (d, <i>J</i> = 2.8 Hz, 1H), 6.50 (d, <i>J</i> = 2.8 Hz, 1H), 7.20−7.53 (aromatics, 14H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub><i>-d</i>):  δ 28.4, 42.6, 48.8, 53.7, 54.9, 57.0, 60.2, 106.5, 113.7, 125.3, 127.7, 127.8, 129.2, 134.0, 140.6, 140.7, 154.3, 162.7. </div><div class="NLM_p"><b>[5-(4-Chlorophenyl)-furan-2-ylmethyl][1-(3,5,5-trimethylhexyl)pyrrolidin-3-yl]amine (3i).</b> This compound was prepared according to the procedure described for compound <b>3h</b>, using 1-(3,5,5-trimethylhexyl)pyrrolidin-3-ylamine, TFA salt <b>21c</b> in place of 1-(2,2-diphenylethyl)pyrrolidin-3-ylamine, TFA salt<b> 21b </b>(100.0 mg, 83.7%). MALDI−FTMS calcd for C<sub>24</sub>H<sub>35</sub>ClN<sub>2</sub>O:  403.2511 [MH<sup>+</sup>]. Found:  403.2508. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  0.92 (s, 9H), 0.97 (d, <i>J</i> = 6.5 Hz, 3H), 1.14 (AB, <i>J</i> = 6.0 Hz and 14.0 Hz, 1H), 1.23 (AB, 1H), 1.6 (m, 4H), 2.25 (m, 1H), 2.6 (m, 1H), 3.25 (m, 2H), 3.64 (m, 3H), 4.12 (m, 1H), 4.38 (m, 2H), 6.72 (d, <i>J</i> = 3.4 Hz, 1H), 6.87 (d, <i>J</i> = 3.4 Hz, 1H), 7.40 (m, 2H), 7.74 (m, 2H<b>). </b><sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 22.7, 28.7, 29.2, 30.4, 31.9, 35.9, 44.0, 52.0, 54.0, 55.2, 55.9, 56.5, 108.1, 115.6, 126.6, 130.2, 135.0, 148.0, 156.0. </div><div class="NLM_p"><b>(</b><b><i>R</i></b><b>)-2-[(2-Benzo[</b><b><i>b</i></b><b>]thiophen-3-yl-acetylamino)methyl]pyrrolidine-1-carboxylic Acid </b><b><i>tert</i></b><b>-Butyl Ester ((</b><b><i>R</i></b><b>)-22a).</b> To a solution of benzo[<i>b</i>]thiophene-3-acetic acid (1.15 g, 6 mmol) and <i>N,N</i>-diisopropylethylamine (3.1 mL, 15 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) was added (<i>R</i>)-1-<i>tert</i>-butoxycarbonyl-2-aminomethylpyrrolidine (1.0 g, 5 mmol), followed by EDCI (1.432 g, 7.5 mmol) and HOBT (1.01 g, 7.5 mmol). The reaction mixture was stirred at room temperature for 16 h, diluted with CH<sub>2</sub>Cl<sub>2</sub> (25 mL), and washed with aqueous 2.0 M NaOH solution (20 mL) and then brine (20 mL). The organic extract was dried over sodium sulfate and the solvent was removed in vacuo. The residue was purified by automated flash chromatography (gradient elution:  10% EtOAc in hexane to 35% EtOAc in hexane) to yield the desired compound as a white oil (1.3 g, 71.0%). MALDI−FTMS calcd for C<sub>20</sub>H<sub>26</sub>N<sub>2</sub>O<sub>3</sub>S:  397.1556 [MNa<sup>+</sup>]. Found:  397.1564. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  1.43 (d, 9H), 1.72 (m, 3H), 1.84 (m, 1H), 3.17 (m, 1H), 3.26 (m, 2H), 3.43 (m, 1H), 3.78 (s, 2H), 3.85 (m, 1H), 7.36 (m, 3H), 7.85 (m, 2H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 23.8, 28.9, 29.7, 37.2, 42.7, 47.6, 81.2, 122.9, 123.9, 125.3, 125.6, 125.7, 125.9, 140.1, 141.9, 173.3. </div><div class="NLM_p"><b>(</b><b><i>S</i></b><b>)-2-[(2-Benzo[</b><b><i>b</i></b><b>]thiophen-3-yl-acetylamino)methyl]pyrrolidine-1-carboxylic Acid </b><b><i>tert</i></b><b>-Butyl Ester ((</b><b><i>S</i></b><b>)-22a).</b> This compound was prepared according to the procedure described for compound <b>(</b><b><i>R</i></b><b>)-22a </b>using (<i>S</i>)-1-<i>tert</i>-butoxycarbonyl-2-aminomethylpyrrolidine in place of (<i>R</i>)-1-<i>tert</i>-butoxycarbonyl-2-aminomethylpyrrolidine (1.0 g, 53.4%). MALDI−FTMS calcd for C<sub>20</sub>H<sub>26</sub>N<sub>2</sub>O<sub>3</sub>S:  397.1556 [MNa<sup>+</sup>]. Found: 397.1564. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  1.44 (d, 9H), 1.72 (m, 3H), 1.84 (m, 1H), 3.18 (m, 1H), 3.27 (m, 2H), 3.45 (m, 1H), 3.78 (s, 2H), 3.85 (m, 1H), 7.36 (m, 3H), 7.86 (m, 2H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 23.8, 28.4, 29.5, 37.2, 42.7, 48.1, 81.2, 122.9, 124.0, 125.3, 125.6, 125.7, 125.9, 140.1, 141.9, 173.3. </div><div class="NLM_p"><b>(</b><b><i>R</i></b><b>)-2-[(4-Phenylbutyrylamino)methyl]pyrrolidine-1-carboxylic Acid </b><b><i>tert</i></b><b>-Butyl Ester </b><b>(</b><b>(</b><b><i>R</i></b><b>)-22b).</b> This compound was prepared according to the procedure described for compound <b>(</b><b><i>S</i></b><b>)-22a </b>using 4-phenylbutyric acid in place of benzo[<i>b</i>]thiophene-3-acetic acid (1.6 g, 94.6%). MALDI−FTMS calcd for C<sub>20</sub>H<sub>30</sub>N<sub>2</sub>O<sub>3</sub>:  369.2149 [MNa<sup>+</sup>]. Found:  369.2138. <sup>1</sup>H NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 1.44 (s, 9H), 1.66 (m, 1H), 1.87 (m, 3H), 1.94 (m, 2H), 2.19 (m, 2H), 2.63 (m, 2H), 3.19 (m, 1H), 3.32 (m, 2H), 4.02 (m, 1H), 7.14−7.27 (m, 5H). <sup>13</sup>C NMR (500 MHz, CHCl<sub>3</sub>-<i>d</i>):  δ 24.1, 27.4, 28.6, 29.7, 35.6, 36.4, 46.4, 47.3, 56.4, 80.3, 126.0, 128.5, 128.7, 141.9, 157.0, 173.4. </div><div class="NLM_p"><b>(</b><b><i>S</i></b><b>)-2-[(4-Phenyl-butyrylamino)methyl]pyrrolidine-1-carboxylic Acid </b><b><i>tert</i></b><b>-Butyl Ester </b><b>((</b><b><i>S</i></b><b>)-22b).</b> This compound was prepared according to the procedure described for compound <b>(</b><b><i>S</i></b><b>)-22a </b>using 4-phenylbutyric acid in place of benzo[<i>b</i>]thiophene-3-acetic acid (1.3 g, 80%). MALDI−FTMS calcd for C<sub>20</sub>H<sub>30</sub>N<sub>2</sub>O<sub>3</sub>:  369.2149 [MNa<sup>+</sup>]. Found:  369.2138. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 1.46 (d, 9H), 1.87 (m, 4H), 1.91 (m, 2H), 2.20 (t, <i>J</i> = 7.5 Hz, 2H), 2.62 (t, <i>J</i> = 7.6 Hz, 2H), 3.38 (m, 4H), 3.9 (m, 1H), 7.17−7.25 (m, 5H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 23.8, 28.9, 36.7, 42.5, 47.6, 58.4, 81.0, 127.1, 129.5, 129.6, 129.7, 143.0, 176.2. </div><div class="NLM_p"><b>(</b><b><i>R</i></b><b>)-2-Benzo[</b><b><i>b</i></b><b>]thiophen-3-yl-</b><b><i>N</i></b><b>-1-pyrrolidin-2-ylmethylacetamide ((</b><b><i>R</i></b><b>)-23a).</b> To a stirred solution of (<i>R</i>)-2-[(2-benzo[<i>b</i>]thiophen-3-yl-acetylamino)methyl]pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester <b>(</b><b><i>R</i></b><b>)-22a </b>(800 mg, 2.1 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) was added trifluoroacetic acid (5 mL). The resultant solution was stirred at room temperature for 2 h and <a name="jm004702706873" id="jm004702706873"></a> the CH<sub>2</sub>Cl<sub>2</sub>/TFA solvent was removed under reduced pressure. The residue was coevaporated with CH<sub>2</sub>Cl<sub>2</sub>, dissolved in MeOH (10 mL), and purified by prep-LCMS to yield the desired compound as TFA salt (500.0 mg, 84.8%). MALDI−FTMS calcd for C<sub>15</sub>H<sub>18</sub>N<sub>2</sub>OS:  275.1213 [MH<sup>+</sup>]. Found:  275.1205. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 1.74 (m, 1H), 1.98 (m, 2H), 2.08 (m, 1H), 3.22 (t, <i>J</i> = 7.6 Hz, 2H), 3.49 (d, <i>J</i> = 5.8 Hz, 2H), 3.66 (m, 2H), 3.86 (s, 2H), 7.6 (m, 2H), 7.47 (s, 1H), 7.83 (dd, 1H, <i>J</i> = 1.0 and 10 Hz), 7.89 (dd, <i>J</i> = 1.0 and 10 Hz). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 24.5, 28.5, 36.6, 41.9, 46.7, 62.2, 122.9, 123.9, 125.4, 125.7, 126.2, 130.5, 140.1, 141.9, 175.1. </div><div class="NLM_p"><b>(</b><b><i>S</i></b><b>)-2-Benzo[</b><b><i>b</i></b><b>]thiophen-3-yl-</b><b><i>N</i></b><b>-1-pyrrolidin-2-ylmethylacetamide ((</b><b><i>S</i></b><b>)</b><b>-</b><b>23a).</b> This compound was prepared according to the procedure described for compound <b>(</b><b><i>R</i></b><b>)-23a </b>using (<i>S</i>)-2-[(2-benzo[<i>b</i>]thiophen-3-ylacetylamino)methyl]pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester <b>(</b><b><i>S</i></b><b>)-22a </b>in place of (<i>R</i>)-2-[(2-benzo[<i>b</i>]thiophen-3-ylacetylamino)methyl]pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester <b>(</b><b><i>R</i></b><b>)-22a </b>(527.0 mg, 90.0%). MALDI−FTMS calcd for C<sub>15</sub>H<sub>18</sub>N<sub>2</sub>OS:  275.1213 [MH<sup>+</sup>]. Found:  275.1209. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 1.93 (m, 1H), 1.98 (m, 2H), 2.08 (m, 1H), 3.22 (t, <i>J</i> = 7.3 Hz, 2H), 3.49 (d, <i>J</i> = 5.8 Hz, 2H), 3.66 (m, 2H), 3.86 (s, 2H), 7.6 (m, 2H), 7.47 (s, 1H), 7.83 (d, 1H, <i>J</i> = 7.4 Hz), 7.89 (d, <i>J</i> = 7.4 Hz). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 24.5, 28.5, 36.6, 41.9, 46.7, 62.2, 122.9, 123.9, 125.4, 125.7, 126.2, 130.5, 140.1, 141.9, 175.1 </div><div class="NLM_p"><b>(</b><b><i>R</i></b><b>)-4-Phenyl-</b><b><i>N</i></b><b>-1-pyrrolidin-2-ylmethylbutyramide ((</b><b><i>R</i></b><b>)-23b).</b> This compound was prepared according to the procedure described for compound (<b><i>R</i></b><b>)-23a </b>using (<i>R</i>)-2-[(4-phenylbutyrylamino)methyl]pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester <b>(</b><b><i>R</i></b><b>)-22b</b> in place of (<i>R</i>)-2-[(2-benzo[<i>b</i>]thiophen-3-ylacetylamino)methyl]pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester <b>(</b><b><i>R</i></b><b>)-22a </b>(484.0 mg, 92.0%). MALDI−FTMS calcd for C<sub>15</sub>H<sub>22</sub>N<sub>2</sub>O:  247.1805 [MH<sup>+</sup>]. Found:  247.1802. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 1.74 (m, 1H), 1.94 (m, 2H), 2.02 (m, 1H), 2.03 (m, 1H), 2.07 (m, 1H), 2.26 (t, <i>J</i> = 7.5 Hz, 2H), 2.64 (t, <i>J</i> = 7.5 Hz, 2H), 3.30 (m, 2H), 3.45 (m, 2H), 3.67 (m, 1H), 7.17−7.25 (m, 5H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 24.5, 28.6, 36.2, 36.4, 41.7, 46.7, 62.2, 127.1, 129.5, 129.6, 143.0, 177.8. </div><div class="NLM_p"><b>(</b><b><i>S</i></b><b>)-4-Phenyl-</b><b><i>N</i></b><b>-1-pyrrolidin-2-ylmethylbutyramide ((</b><b><i>S</i></b><b>)-23b).</b> This compound was prepared according to the procedure described for compound <b>(</b><b><i>R</i></b><b>)-23a </b>using (<i>S</i>)-2-[(4-phenylbutyrylamino)methyl]pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester <b>(</b><b><i>S</i></b><b>)-22b</b> in place of (<i>R</i>)-2-[(2-benzo[<i>b</i>]thiophen-3-ylacetylamino)methyl]pyrrolidine-1-carboxylic acid <i>tert</i>-butyl ester <b>(</b><b><i>R</i></b><b>)-22a </b>(447.0 mg, 85.0%). MALDI−FTMS calcd for C<sub>15</sub>H<sub>22</sub>N<sub>2</sub>O:  247.1805 [MH<sup>+</sup>]. Found:  247.1800. <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 1.74 (m, 1H), 2.01 (m, 5H), 2.27 (t, <i>J</i> = 7.3 Hz, 2H), 2.64 (t, <i>J</i> = 7.5 Hz, 2H), 3.27 (m, 1H), 3.44 (d, <i>J</i> = 5.8 Hz, 2H), 3.66 (m, 1H), 7.2 (m, 5H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 24.5, 28.5, 36.2, 36.4, 41.7, 46.7, 62.3, 127.1, 129.5, 129.6, 129.7, 142.9, 177.9. </div><div class="NLM_p"><b>(</b><b><i>R</i></b><b>)-2-Benzo[</b><b><i>b</i></b><b>]thiophen-3-yl-</b><b><i>N</i></b><b>-[1-(3,4-dichlorobenzyl)pyrrolidin-2-ylmethyl]acetamide </b><b>((</b><b><i>R</i></b><b>)-4a).</b> To a solution of (<i>R</i>)-2-benzo[<i>b</i>]thiophen-3-yl-<i>N</i>-1-pyrrolidin-2-ylmethylacetamide <b>(</b><b><i>R</i></b><b>)-23a </b>(200.0 mg, 0.72 mmol) in the presence of <i>N,N</i>-diisopropylethylamine (375 μL, 2.16 mmol) in anhydrous DMF (3.6 mL) was added 3,4-dichlorobenzyl bromide (209.0 mg, 0.87 mmol). The reaction mixture was stirred at 40 °C for 12 h, cooled to room temperature, and purified by prep-LCMS to give the desired compound as the TFA salt (243.0 mg, 78.0%). MALDI−FTMS calcd for C<sub>22</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>2</sub>OS:  433.0903 [MH<sup>+</sup>]. Found:  433.0899. [α]<sub>D</sub> −10.457 (<i>c</i> 1, MeOH). NMR (methanol-<i>d</i><sub>4</sub>):  δ 1.84 (m, 2H), 2.04 (m, 2H), 2.23 (m, 1H), 3.13 (m, 1H), 3.26 (m, 1H), 3.46 (m, 1H), 3.68 (m, 2H), 3.89 (s, 2H), 4.05 (d, <i>J</i> = 12.5 Hz, 1H), 4.48 (d, <i>J</i> = 12.5 Hz, 1H), 7.34 (m, 3H), 7.5 (s, 1H), 7.55 (d, 1H), 7.66 (d, 1H), 7.85 (m, 2H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 23.1, 28.0, 36.6, 41.3, 55.3, 57.9, 70.3, 122.9, 124.0, 125.4, 125.7, 126.4, 130.4, 131.5, 132.4, 132.5, 133.8, 134.3, 135.3, 139.9, 141.9, 176.2. </div><div class="NLM_p"><b>(</b><b><i>S</i></b><b>)-2-Benzo[</b><b><i>b</i></b><b>]thiophen-3-yl-</b><b><i>N</i></b><b>-[1-(3,4-dichloro-benzyl)pyrrolidin-2-ylmethyl] -acetamide </b><b>((</b><b><i>S</i></b><b>)-4a).</b> This compound was prepared according to the procedure described for compound (<b><i>R</i></b><b>)-4a </b>using (<i>S</i>)-2-Benzo[<i>b</i>]thiophen-3-yl-<i>N</i>-1-pyrrolidin-2-ylmethyl-acetamide (<b><i>S</i></b><b>)-23a </b>in place of (<i>R</i>)-2-Benzo[<i>b</i>]thiophen-3-yl-<i>N</i>-1-pyrrolidin-2-ylmethyl-acetamide (<b><i>R</i></b><b>)-23a </b>(126.0 mg, 50.0%). MALDI−FTMS calcd for C<sub>22</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>2</sub>OS:  433.0903 [MH<sup>+</sup>]. Found:  433.0912. [α]<sub>D</sub> +1.769 (<i>c</i> 1, MeOH). <sup>1</sup>H NMR (methanol-<i>d</i><sub>4</sub>):  δ 1.83 (m, 2H), 2.03 (m, 2H), 2.21 (m, 1H), 3.13 (m, 1H), 3.24 (m, 1H), 3.47 (m, 1H), 3.64 (m, 2H), 3.89 (s, 2H), 4.04 (d, <i>J</i> = 12.5 Hz, 1H), 4.51 (d, <i>J</i> = 12.5 Hz, 1H), 7.33 (m, 3H), 7.51 (s, 1H), 7.55 (d, 1H), 7.66 (d, 1H), 7.86 (m, 2H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 23.2, 28.0, 36.6, 41.4, 55.3, 58.0, 70.4, 122.9, 124.0, 125.4, 125.7, 126.4, 130.4, 131.5, 132.5, 133.8, 134.3, 139.9, 141.9, 176.3. </div><div class="NLM_p"><b>(</b><b><i>R</i></b><b>)-</b><b><i>N</i></b><b>-(1-Phenethylpyrrolidin-2-ylmethyl)-4-phenylbutyramide ((</b><b><i>R</i></b><b>)-4b). </b>This compound was prepared according to the procedure described for compound (<b><i>R</i></b><b>)-4a </b>using (2-chloroethyl)benzene in place of 3,4-dichlorobenzyl bromide (126 mg, 50.0%). MALDI−FTMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>2</sub>O:  351.2431 [MH<sup>+</sup>]. Found:  351.2423. [α]<sub>D</sub> +11.382 (<i>c</i> 1, MeOH). <sup>1</sup>H NMR (methanol-<i>d</i><sub>4</sub>):  δ 1.93 (m, 5H), 2.2 (m, 4H), 2.64 (t, <i>J</i> = 7.5 Hz, 2H), 3.0.6 (m, 2H), 3.30 (m, 2H), 3.45 (m, 1H), 3.66 (m, 3H), 7.26 (m, 10H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>): δ 23.5, 27.9, 28.5, 33.2, 36.1, 36.4, 40.7, 55.6, 57.3, 70.1, 127.2, 128.5, 129.5, 129.6, 129.9, 130.1, 137.6, 142.8, 178.7. </div><div class="NLM_p"><b>(</b><b><i>S</i></b><b>)-</b><b><i>N</i></b><b>-(1-Phenethylpyrrolidin-2-ylmethyl)-4-phenylbutyramide ((</b><b><i>S</i></b><b>)-4b).</b> This compound was prepared according to the procedure described for compound (<b><i>S</i></b><b>)-4a </b>using (2-chloroethyl)benzene in place of 3,4-dichlorobenzyl bromide (189.0 mg, 75.0%). MALDI−FTMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>2</sub>O: 351.2431 [MH<sup>+</sup>]. Found:  351.2426. [α]<sub>D</sub> −15.314 (<i>c</i> 1, MeOH). <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 1.36 (m, 2H), 1.93 (m, 3H), 2.13 (m, 1H), 2.24 (m, 3H), 2.64 (t, <i>J</i> = 7.5 Hz, 2H), 3.09 (m, 2H), 3.30 (m, 1H), 3.44 (m, 1H), 3.67 (m, 5H), 7.22 (m, 10H). <sup>13</sup>C NMR (500 MHz, methanol-<i>d</i><sub>4</sub>):  δ 23.54, 28.0, 28.5, 33.2, 36.1, 36.4, 40.7, 55.6, 57.3, 70.1, 127.2, 128.5, 129.6, 137.6, 142.8, 178.7. </div><div class="NLM_p"><b>Computational Chemistry.</b> Using an input SD file of the 81 560 compounds making up NBI-DS, standard 3D BCUT metrics were calculated for each member of the collection. A χ<sup>2</sup> approach provided by Diverse Solutions automatically chose and optimized which metric combinations would comprise the optimal chemistry space to represent maximum diversity of this set. The result was a chemical space defined by five BCUT metrics:  charge, H-bond donor, H-bond acceptor, polarizability I (low eigenvalue), and polarizability II (high eigenvalue). Arbitrarily dividing each axis into 10 bins afforded 100 000 cells, so cell occupancy can be used as a measure of the diversity/similarity between the molecules based on the coordinates of each cell. </div><div class="NLM_p">A similar process was used to describe the location of the known GPCR ligands having reported affinity for specific a GPCR (630 molecules), the training set (111 ligands for peptide-activated GPCRs), and the virtual libraries from which the test set (2025 compounds) was selected. In the case of the latter, virtual libraries for each of the 19 templates were enumerated separately using the Afferent and Project Library (MDL), and the resulting SD files contained over 9 million compounds. For reasons already described, the location of the training set (97 occupied cells) was expanded to include all neighboring cells (6437 cells). For every molecule in each virtual library, the same five BCUT metrics were calculated to give 19 cell-index files, which were then compared with the locations of molecules within the expanded training set. The cell-index files also provided information on how many cells were occupied by molecules from each virtual library, and the degree of overlap with the training set was assessed using an in house algorithm, “Cell Compare”. Those virtual libraries having the highest percentage of occupied cells overlapping with the GPCR−PA<sup>+</sup> subspace were selected for synthetic trials; in all, 10 templates were thereby selected. As additional selection criteria, so as to achieve a balance between relevant space coverage and density of cell population, no more than 25 compounds per cell were selected for the test set, selection was biased to maximize the number of templates used, and finally, matrixes for parallel synthesis were chosen so as to maximize cell occupancy in GPCR−PA<sup>+</sup> subspace. </div><div class="NLM_p"><a name="jm004702706874" id="jm004702706874"></a><b>Biological Assays.</b> MC4−R and MCH1−R binding assays were carried out with membrane preparations derived from mammalian cell lines stably transfected with human MC4 receptor (2) or rat MCH1 receptor (3). Crude membrane preparations were generated by nitrogen cavitation of cells followed by differential centrifugation. Specifically, cells were suspended in “binding buffer” and subjected to nitrogen cavitation at 900 psi on ice for 30 min. Lysed cells were centrifuged at 800<i>g</i> for 10 min, and the resultant supernatant was centrifuged at 38 000<i>g</i> for 20 min. Pellets were resuspended in the appropriate binding buffer and stored at −80 °C until use. MC4−R binding assays were carried out in 25 mM HEPES (pH 7.0), 1.5 mM CaCl<sub>2</sub>, 1.0 mM MgSO<sub>4</sub>, and 100 mM NaCl. MCH1−R binding assays were carried out in 50 mM HEPES (pH 7.0), 10 mM MgCl<sub>2</sub>, 2 mM EGTA. Binding reactions were carried out at equilibrium in 96-well format and were terminated by rapid vacuum filtration using a Packard Unifilter-96 plate harvester, over glass-fiber (GF−C) filter plates pretreated for 15 min with 0.5% polyethylenimine. Filters were washed four times with 0.2 mL of binding buffer. </div><div class="NLM_p">GnRH receptor binding assays were carried out in whole-cell format using mammalian cells stably transfected with human GnRH receptor (4). Cells were harvested on the day of assay and assayed at 0.5 × 10<sup>6</sup> cells per well in 10 mM Hepes (pH 7.5), 150 mM NaCl, and 0.1% BSA. Binding reactions were carried out at equilibrium in 96-well format and terminated by rapid vacuum filtration over Millipore Multiscreen (GF−C) filter plates pretreated for 120 min with 0.1% polyethylenimine. Filters were washed two times with 0.2 mL phosphate-buffered saline (pH 7.4). In all binding assays, filters were dried, 0.05 mL Microscint-20 was added, and radioactivity was monitored by liquid scintillation counting. </div><div class="NLM_p last">In all binding experiments, specific binding is defined as that displaced by a known, pharmacologically relevant unlabeled competitor. The concentration of competitor was defined from competition curves for each assay where the competitor inhibited greater than 95% of tracer binding. Percent inhibition of binding for test compounds is defined as [1 − (specific binding in the presence of test compound)/(control specific binding)] × 100. Binding data for <i>K</i><sub>i</sub> determinations were analyzed by the nonlinear least squares curve-fitting program GraphPad Prism. </div></div><div class="NLM_back"><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm040084c/suppl_file/jm040084c_s.pdf">jm040084c_s.pdf (59.58 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm040084c" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60173" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60173" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karine Lavrador</span> - <span class="hlFld-Affiliation affiliation">Chemistry and Biology Departments, Neurocrine Biosciences, 12790 El Camino Real, San Diego, California 92130 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian Murphy</span> - <span class="hlFld-Affiliation affiliation">Chemistry and Biology Departments, Neurocrine Biosciences, 12790 El Camino Real, San Diego, California 92130 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Saunders</span> - <span class="hlFld-Affiliation affiliation">Chemistry and Biology Departments, Neurocrine Biosciences, 12790 El Camino Real, San Diego, California 92130 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott Struthers</span> - <span class="hlFld-Affiliation affiliation">Chemistry and Biology Departments, Neurocrine Biosciences, 12790 El Camino Real, San Diego, California 92130 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaochuan Wang</span> - <span class="hlFld-Affiliation affiliation">Chemistry and Biology Departments, Neurocrine Biosciences, 12790 El Camino Real, San Diego, California 92130 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Williams</span> - <span class="hlFld-Affiliation affiliation">Chemistry and Biology Departments, Neurocrine Biosciences, 12790 El Camino Real, San Diego, California 92130 </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27012" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27012" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part by a grant from the National Institutes of Health (Grant Number 5R44 GM61491-03). In addition, we thank Dr. Christian Baber for help in analyzing the final data set. </p></div><div class="refs-header-label"></div><p class="references-count">
            This article references 20 other publications.
        </p><ol id="references" class="useLabel"><li id="jm040084cb00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm040084cb00001_1"><span><span class="NLM_string-ref">The most up to date listing can be viewed and searched on the GPCRDB home page at <a class="ext-link" href="http://www.darmstadt.gmd.de/~gpcrdb" target="_blank">http://www.darmstadt.gmd.de/∼gpcrdb</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+most+up+to+date+listing+can+be+viewed+and+searched+on+the+GPCRDB+home+page+at+http%3A%2F%2Fwww.darmstadt.gmd.de%2F%E2%88%BCgpcrdb.+The+most+up+to+date+listing+can+be+viewed+and+searched+on+the+GPCRDB+home+page+at+http%3A%2F%2Fwww.darmstadt.gmd.de%2F%E2%88%BCgpcrdb."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.darmstadt.gmd.de%2F%E2%88%BCgpcrdb" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm040084cb00002_1"><span><span class="NLM_string-ref">Watson, S.; Arkinstall, S. <i>  The G-Protein Linked Receptor FactsBook</i>; Academic Press:  San Diego, CA, 1996.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=S.+Watson&author=S.+Arkinstall&title=The+G-Protein+Linked+Receptor+FactsBook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DS.%26btitle%3DThe%2520G-Protein%2520Linked%2520Receptor%2520FactsBook%26pub%3DAcademic%2520Press%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm040084cb00003_1"><span><span class="NLM_string-ref">For a list of the Top 200 Prescription Medicines see:  <a class="ext-link" href="http://www.rxlist.com/top200.htm" target="_blank">http://www.rxlist.com/top200.htm</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+For+a+list+of+the+Top+200+Prescription+Medicines+see%3A%E2%80%89+http%3A%2F%2Fwww.rxlist.com%2Ftop200.htm.+For+a+list+of+the+Top+200+Prescription+Medicines+see%3A%E2%80%89+http%3A%2F%2Fwww.rxlist.com%2Ftop200.htm."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.rxlist.com%2Ftop200.htm" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm040084cb00004_1"><span><span class="NLM_string-ref">Marshall, G. R. Peptide Interactions with G-Protein Coupled Receptors. <i>  Biopolymers (Pept. Sci.)</i><b>2001</b>, <i>60</i>, 246−277.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2001&pages=277&author=G.+R+Marshall&title=Peptide+Interactions+with+G-Protein+Coupled+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarshall%26aufirst%3DG.%2BR%26atitle%3DPeptide%2520Interactions%2520with%2520G-Protein%2520Coupled%2520Receptors%26jtitle%3DBiopolymers%2520%2528Pept.%2520Sci.%2529%26date%3D2001%26volume%3D60%26spage%3D277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm040084cb00005_1"><span><span class="NLM_string-ref">Rees, D. C. Non-Peptide Ligands for Neuropeptide Receptors. <i>  Annu. Rep. </i><i>Med. Chem</i>. <b>1993</b>, <i>28</i>, 59−68.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1993&pages=68&author=D.+C+Rees&title=Non-Peptide+Ligands+for+Neuropeptide+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC%26atitle%3DNon-Peptide%2520Ligands%2520for%2520Neuropeptide%2520Receptors%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D1993%26volume%3D28%26spage%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm040084cb00006_1"><span><span class="NLM_string-ref">Sugg, E. E. Non-peptide Agonists for Peptide Receptors:  Lessons from Ligands. <i>  Annu. Rep. Med Chem</i>. <b>1997</b>, <i>32</i>, 277.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1997&pages=277&author=E.+E+Sugg&title=Non-peptide+Agonists+for+Peptide+Receptors%3A%E2%80%89+Lessons+from+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSugg%26aufirst%3DE.%2BE%26atitle%3DNon-peptide%2520Agonists%2520for%2520Peptide%2520Receptors%253A%25E2%2580%2589%2520Lessons%2520from%2520Ligands%26jtitle%3DAnnu.%2520Rep.%2520Med%2520Chem.%26date%3D1997%26volume%3D32%26spage%3D277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm040084cb00007_1"><span><span class="NLM_string-ref">Yang, L. Non-Peptide Somatostatin Receptor Ligands. <i>  Annu. Rep. Med. </i><i>Chem.</i><b>1999</b>, <i>34</i>, 209−218.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=218&author=L+Yang&title=Non-Peptide+Somatostatin+Receptor+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL%26atitle%3DNon-Peptide%2520Somatostatin%2520Receptor%2520Ligands%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D1999%26volume%3D34%26spage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm040084cb00008_1"><span><span class="NLM_string-ref">Pearlman, R. S.; Smith, K. M. Novel Software Tools for Chemical Diversity. <i>  Perspect. Drug Discovery</i><b>1998</b>, <i>9</i>, 339−353.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1998&pages=353&author=R.+S.+Pearlman&author=K.+M+Smith&title=Novel+Software+Tools+for+Chemical+Diversity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DR.%2BS.%26atitle%3DNovel%2520Software%2520Tools%2520for%2520Chemical%2520Diversity%26jtitle%3DPerspect.%2520Drug%2520Discovery%26date%3D1998%26volume%3D9%26spage%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm040084cb00009_1"><span><span class="NLM_string-ref">Pearlman, R. S.; Smith, K. M. Metric Validation and the Receptor-Relevant Subspace Concept. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>1999</b>, <i>39</i>, 28−35.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=35&author=R.+S.+Pearlman&author=K.+M+Smith&title=Metric+Validation+and+the+Receptor-Relevant+Subspace+Concept"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DR.%2BS.%26atitle%3DMetric%2520Validation%2520and%2520the%2520Receptor-Relevant%2520Subspace%2520Concept%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00010"><div class="reference"><div class="NLM_citation" id="jm040084cb00010_1"><span><span class="NLM_string-ref">Williams, J. P.; Lavrador, K. A Solution-Phase Combinatorial Synthesis of Selective Dopamine D<sub>4</sub> Ligands. <i>  Comb. Chem. HTS</i><b>2000</b>, <i>3</i>, 43−50.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2000&pages=50&author=J.+P.+Williams&author=K+Lavrador&title=A+Solution-Phase+Combinatorial+Synthesis+of+Selective+Dopamine+D4+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BP.%26atitle%3DA%2520Solution-Phase%2520Combinatorial%2520Synthesis%2520of%2520Selective%2520Dopamine%2520D4%2520Ligands%26jtitle%3DComb.%2520Chem.%2520HTS%26date%3D2000%26volume%3D3%26spage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm040084cb00011_1"><span><span class="NLM_string-ref">Zeng, L.; Burton, L.; Yung, K.; Shushan, B.; Kassel, D. B. Automated Analytical/Preparative High-Performance Liquid Chromatography−Mass Spectrometry System for the Rapid Characterization and Purification of Compound Libraries. <i>  J. Chromatogr. A</i><b>1998</b>, <i>794</i>, 3−13.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=794&publication_year=1998&pages=13&author=L.+Zeng&author=L.+Burton&author=K.+Yung&author=B.+Shushan&author=D.+B.+Kassel&author=+Automated+Analytical&title=System+for+the+Rapid+Characterization+and+Purification+of+Compound+Libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26atitle%3DSystem%2520for%2520the%2520Rapid%2520Characterization%2520and%2520Purification%2520of%2520Compound%2520Libraries%26jtitle%3DJ.%2520Chromatogr.%2520A%26date%3D1998%26volume%3D794%26spage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm040084cb00012_1"><span><span class="NLM_string-ref">Abdel-Magid, A. F.; Carson, K. G.; Harris, B. D.; Maryanoff, C. A.; Shah, R. D. Reductive Amination of Aldehydes and Ketones with Sodium Triacetoxyborohydride. Studies on Direct and Indirect Reductive Amination Procedures. <i>  J. Org. Chem</i>. <b>1996</b>, <i>61</i>, 3849−3862.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1996&pages=3862&author=A.+F.+Abdel-Magid&author=K.+G.+Carson&author=B.+D.+Harris&author=C.+A.+Maryanoff&author=R.+D+Shah&title=Reductive+Amination+of+Aldehydes+and+Ketones+with+Sodium+Triacetoxyborohydride.+Studies+on+Direct+and+Indirect+Reductive+Amination+Procedures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26atitle%3DReductive%2520Amination%2520of%2520Aldehydes%2520and%2520Ketones%2520with%2520Sodium%2520Triacetoxyborohydride.%2520Studies%2520on%2520Direct%2520and%2520Indirect%2520Reductive%2520Amination%2520Procedures%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1996%26volume%3D61%26spage%3D3862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm040084cb00013_1"><span><span class="NLM_string-ref">Cone, R. D.; Lu, D.; Koppula, S.; Vage, D. I.; Klungland, H.; Boston, B.; Chen, W.; Orth, D. N.; Pouton, C.; Kesterson, R. A. The Melanocortin Receptors:  Agonists, Antagonists, and the Hormonal Control of Pigmentation. <i>  Rec. Prog. Hor. Res.</i><b>1996</b>, <i>51</i>, 287−318.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1996&pages=318&author=R.+D.+Cone&author=D.+Lu&author=S.+Koppula&author=D.+I.+Vage&author=H.+Klungland&author=B.+Boston&author=W.+Chen&author=D.+N.+Orth&author=C.+Pouton&author=R.+A+Kesterson&title=The+Melanocortin+Receptors%3A%E2%80%89+Agonists%2C+Antagonists%2C+and+the+Hormonal+Control+of+Pigmentation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCone%26aufirst%3DR.%2BD.%26atitle%3DThe%2520Melanocortin%2520Receptors%253A%25E2%2580%2589%2520Agonists%252C%2520Antagonists%252C%2520and%2520the%2520Hormonal%2520Control%2520of%2520Pigmentation%26jtitle%3DRec.%2520Prog.%2520Hor.%2520Res.%26date%3D1996%26volume%3D51%26spage%3D318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm040084cb00014_1"><span><span class="NLM_string-ref">Goodfellow, V. S.; Saunders, J. The Melanocortin System and Its Role in Obesity and Cachexia. <i>  Curr. Top. Med. Chem.</i><b>2003</b>, <i>3</i>, 855−883.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=883&author=V.+S.+Goodfellow&author=J+Saunders&title=The+Melanocortin+System+and+Its+Role+in+Obesity+and+Cachexia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoodfellow%26aufirst%3DV.%2BS.%26atitle%3DThe%2520Melanocortin%2520System%2520and%2520Its%2520Role%2520in%2520Obesity%2520and%2520Cachexia%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2003%26volume%3D3%26spage%3D883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm040084cb00015_1"><span><span class="NLM_string-ref">Qu, D.; Ludwig, D. S.; Gammeltoft, S.; Piper, M.; Pelleymounter, M. A.; Cullen, M. J.; Mathes, W. F.; Przypek, J.; Kanarek, R.; Maratos-Flier, E. A Role For Melanin-Concentrating Hormone in the Central Regulation of Feeding Behavior. <i>  Nature</i><b>1996</b>, <i>380</i>, 243−7.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=1996&pages=7&author=D.+Qu&author=D.+S.+Ludwig&author=S.+Gammeltoft&author=M.+Piper&author=M.+A.+Pelleymounter&author=M.+J.+Cullen&author=W.+F.+Mathes&author=J.+Przypek&author=R.+Kanarek&author=E+Maratos-Flier&title=A+Role+For+Melanin-Concentrating+Hormone+in+the+Central+Regulation+of+Feeding+Behavior"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQu%26aufirst%3DD.%26atitle%3DA%2520Role%2520For%2520Melanin-Concentrating%2520Hormone%2520in%2520the%2520Central%2520Regulation%2520of%2520Feeding%2520Behavior%26jtitle%3DNature%26date%3D1996%26volume%3D380%26spage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm040084cb00016_1"><span><span class="NLM_string-ref">Borowsky, B.; Durkin, M. M.; Ogozalek, K.; Marzabadi, M. R.; DeLeon, J.; Lagu, B.; Heurich, R.; Lichtblau, H.; Shaposhnik, Z.; Daniewska, I.; Blackburn, T. P.; Branchek, T. A.; Gerald, C.; Vaysse, P. J.; Forray, C. Antidepressant, Anxiolytic and Anorectic Effects of a Melanin-Concentrating Hormone-1 Receptor Antagonist. <i>  Nat. Med</i>. <b>2002</b>, <i>8</i>, 825−30.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2002&pages=30&author=B.+Borowsky&author=M.+M.+Durkin&author=K.+Ogozalek&author=M.+R.+Marzabadi&author=J.+DeLeon&author=B.+Lagu&author=R.+Heurich&author=H.+Lichtblau&author=Z.+Shaposhnik&author=I.+Daniewska&author=T.+P.+Blackburn&author=T.+A.+Branchek&author=C.+Gerald&author=P.+J.+Vaysse&author=C.+Forray&author=+Antidepressant&title=Effects+of+a+Melanin-Concentrating+Hormone-1+Receptor+Antagonist"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBorowsky%26aufirst%3DB.%26atitle%3DEffects%2520of%2520a%2520Melanin-Concentrating%2520Hormone-1%2520Receptor%2520Antagonist%26jtitle%3DNat.%2520Med.%26date%3D2002%26volume%3D8%26spage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm040084cb00017_1"><span><span class="NLM_string-ref">Sealfon, S. C.; Weinstein, H.; Millar, R. P. Molecular Mechanisms of Ligand Interaction with the Gonadotropin-Releasing Hormone Receptor. <i>  Endocr. Rev</i>. <b>1997</b>, <i>18</i>, 180−205.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=205&author=S.+C.+Sealfon&author=H.+Weinstein&author=R.+P+Millar&title=Molecular+Mechanisms+of+Ligand+Interaction+with+the+Gonadotropin-Releasing+Hormone+Receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSealfon%26aufirst%3DS.%2BC.%26atitle%3DMolecular%2520Mechanisms%2520of%2520Ligand%2520Interaction%2520with%2520the%2520Gonadotropin-Releasing%2520Hormone%2520Receptor%26jtitle%3DEndocr.%2520Rev.%26date%3D1997%26volume%3D18%26spage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm040084cb00018_1"><span><span class="NLM_string-ref">Emons, G.; Grundker, C.; Gunthert, A. R.; Westphalen, S.; Kavanagh, J.; Versschraege, C. GnRH Antagonists in the Treatment of Gynaecological and Breast Cancers. <i>  Endocr. Relat. Cancer</i><b>2003</b>, <i>10</i>, 291−299.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=299&author=G.+Emons&author=C.+Grundker&author=A.+R.+Gunthert&author=S.+Westphalen&author=J.+Kavanagh&author=C+Versschraege&title=GnRH+Antagonists+in+the+Treatment+of+Gynaecological+and+Breast+Cancers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEmons%26aufirst%3DG.%26atitle%3DGnRH%2520Antagonists%2520in%2520the%2520Treatment%2520of%2520Gynaecological%2520and%2520Breast%2520Cancers%26jtitle%3DEndocr.%2520Relat.%2520Cancer%26date%3D2003%26volume%3D10%26spage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm040084cb00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm040084cb00019_1"><span><span class="NLM_string-ref">Zhang, J.-H.; Chung, T. D. Y.; Oldenburg, K. R. A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays. <i>  J. Biomol. Screening</i><b>1999</b>, <i>4</i>, 67−73.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=73&author=J.-H.+Zhang&author=T.+D.+Y.+Chung&author=K.+R+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.-H.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D1999%26volume%3D4%26spage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm040084cb00020"><div class="reference"><div class="NLM_citation" id="jm040084cb00020_1"><span><span class="NLM_string-ref">Ohmori, J.; Maeno, K.; Hidaka, K.; Nakato, K.; Matsumoto, M.; Tada, S.; Hattori, H.; Sakamoto, S.; Tsukamoto, S. Dopamine D3 and D4 receptors Antagonists:  Synthesis and Structure−Activity Relationships of (S)-(+)-N-(1-Benzyl-3-pyrrolidinyl)-5-chloro-4-[(cyclopropylcarbonyl)amino]-2-methoxybenzamide (YM-43611) and related compounds. <i>  J. Med. </i><i>Chem.</i><b>1996</b>, <i>39</i>, 2764−2772.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2772&author=J.+Ohmori&author=K.+Maeno&author=K.+Hidaka&author=K.+Nakato&author=M.+Matsumoto&author=S.+Tada&author=H.+Hattori&author=S.+Sakamoto&author=S+Tsukamoto&title=Dopamine+D3+and+D4+receptors+Antagonists%3A%E2%80%89+Synthesis+and+Structure%E2%88%92Activity+Relationships+of+%28S%29-%28%2B%29-N-%281-Benzyl-3-pyrrolidinyl%29-5-chloro-4-%5B%28cyclopropylcarbonyl%29amino%5D-2-methoxybenzamide+%28YM-43611%29+and+related+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOhmori%26aufirst%3DJ.%26atitle%3DDopamine%2520D3%2520and%2520D4%2520receptors%2520Antagonists%253A%25E2%2580%2589%2520Synthesis%2520and%2520Structure%25E2%2588%2592Activity%2520Relationships%2520of%2520%2528S%2529-%2528%252B%2529-N-%25281-Benzyl-3-pyrrolidinyl%2529-5-chloro-4-%255B%2528cyclopropylcarbonyl%2529amino%255D-2-methoxybenzamide%2520%2528YM-43611%2529%2520and%2520related%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm040084cb00001':['jm040084cb00001_1'],'jm040084cb00002':['jm040084cb00002_1'],'jm040084cb00003':['jm040084cb00003_1'],'jm040084cb00004':['jm040084cb00004_1'],'jm040084cb00005':['jm040084cb00005_1'],'jm040084cb00006':['jm040084cb00006_1'],'jm040084cb00007':['jm040084cb00007_1'],'jm040084cb00008':['jm040084cb00008_1'],'jm040084cb00009':['jm040084cb00009_1'],'jm040084cb00010':['jm040084cb00010_1'],'jm040084cb00011':['jm040084cb00011_1'],'jm040084cb00012':['jm040084cb00012_1'],'jm040084cb00013':['jm040084cb00013_1'],'jm040084cb00014':['jm040084cb00014_1'],'jm040084cb00015':['jm040084cb00015_1'],'jm040084cb00016':['jm040084cb00016_1'],'jm040084cb00017':['jm040084cb00017_1'],'jm040084cb00018':['jm040084cb00018_1'],'jm040084cb00019':['jm040084cb00019_1'],'jm040084cb00020':['jm040084cb00020_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 18 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hamid R.  Hoveyda</span>, <span class="hlFld-ContribAuthor ">Eric  Marsault</span>, <span class="hlFld-ContribAuthor ">René  Gagnon</span>, <span class="hlFld-ContribAuthor ">Axel P.  Mathieu</span>, <span class="hlFld-ContribAuthor ">Martin  Vézina</span>, <span class="hlFld-ContribAuthor ">Annick  Landry</span>, <span class="hlFld-ContribAuthor ">Zhigang  Wang</span>, <span class="hlFld-ContribAuthor ">Kamel  Benakli</span>, <span class="hlFld-ContribAuthor ">Sylvie  Beaubien</span>, <span class="hlFld-ContribAuthor ">Carl  Saint-Louis</span>, <span class="hlFld-ContribAuthor ">Martin  Brassard</span>, <span class="hlFld-ContribAuthor ">Jean-François  Pinault</span>, <span class="hlFld-ContribAuthor ">Luc  Ouellet</span>, <span class="hlFld-ContribAuthor ">Shridhar  Bhat</span>, <span class="hlFld-ContribAuthor ">Mahesh  Ramaseshan</span>, <span class="hlFld-ContribAuthor ">Xiaowen  Peng</span>, <span class="hlFld-ContribAuthor ">Laurence  Foucher</span>, <span class="hlFld-ContribAuthor ">Sophie  Beauchemin</span>, <span class="hlFld-ContribAuthor ">Patrick  Bhérer</span>, <span class="hlFld-ContribAuthor ">Daniel F.  Veber</span>, <span class="hlFld-ContribAuthor ">Mark L.  Peterson</span>, and <span class="hlFld-ContribAuthor ">Graeme L.  Fraser</span>  . </span><span class="cited-content_cbyCitation_article-title">Optimization of the Potency and Pharmacokinetic Properties of a Macrocyclic Ghrelin Receptor Agonist (Part I): Development of Ulimorelin (TZP-101) from Hit to Clinic. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (24)
                                     , 8305-8320. <a href="https://doi.org/10.1021/jm2007062" title="DOI URL">https://doi.org/10.1021/jm2007062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm2007062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm2007062%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252Bthe%252BPotency%252Band%252BPharmacokinetic%252BProperties%252Bof%252Ba%252BMacrocyclic%252BGhrelin%252BReceptor%252BAgonist%252B%252528Part%252BI%252529%25253A%252BDevelopment%252Bof%252BUlimorelin%252B%252528TZP-101%252529%252Bfrom%252BHit%252Bto%252BClinic%26aulast%3DHoveyda%26aufirst%3DHamid%2BR.%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D02062011%26date%3D22112011%26date%3D22122011%26date%3D07112011%26volume%3D54%26issue%3D24%26spage%3D8305%26epage%3D8320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Massari</span>, <span class="hlFld-ContribAuthor ">Laura  Panelli</span>, <span class="hlFld-ContribAuthor ">Mark  Hughes</span>, <span class="hlFld-ContribAuthor ">Federica  Stazi</span>, <span class="hlFld-ContribAuthor ">William  Maton</span>, <span class="hlFld-ContribAuthor ">Pieter  Westerduin</span>, <span class="hlFld-ContribAuthor ">Federico  Scaravelli</span>, and <span class="hlFld-ContribAuthor ">Sergio  Bacchi</span>. </span><span class="cited-content_cbyCitation_article-title">A Mechanistic Insight into a Simple C−N Bond Formation via SN2 Displacement: A Synergistic Kinetics and Design of Experiment Approach. </span><span class="cited-content_cbyCitation_journal-name">Organic Process Research & Development</span><span> <strong>2010,</strong> <em>14 </em>
                                    (6)
                                     , 1364-1372. <a href="https://doi.org/10.1021/op100176u" title="DOI URL">https://doi.org/10.1021/op100176u</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/op100176u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fop100176u%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Process%2520Research%2520%2526%2520Development%26atitle%3DA%252BMechanistic%252BInsight%252Binto%252Ba%252BSimple%252BC%2525E2%252588%252592N%252BBond%252BFormation%252Bvia%252BSN2%252BDisplacement%25253A%252BA%252BSynergistic%252BKinetics%252Band%252BDesign%252Bof%252BExperiment%252BApproach%26aulast%3DMassari%26aufirst%3DLuca%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D24%252006%25202010%26date%3D04102010%26date%3D19112010%26volume%3D14%26issue%3D6%26spage%3D1364%26epage%3D1372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rama K.  Mishra</span> and <span class="hlFld-ContribAuthor ">Jasbir  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Generation, Validation, and Utilization of a Three-Dimensional Pharmacophore Model for EP3 Antagonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (8)
                                     , 1502-1509. <a href="https://doi.org/10.1021/ci100003q" title="DOI URL">https://doi.org/10.1021/ci100003q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100003q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100003q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DGeneration%25252C%252BValidation%25252C%252Band%252BUtilization%252Bof%252Ba%252BThree-Dimensional%252BPharmacophore%252BModel%252Bfor%252BEP3%252BAntagonists%26aulast%3DMishra%26aufirst%3DRama%2BK.%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D2%252001%25202010%26date%3D20072010%26date%3D23082010%26volume%3D50%26issue%3D8%26spage%3D1502%26epage%3D1509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Smriti  Sharma</span>, <span class="hlFld-ContribAuthor ">Vinayak  Bhatia</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Modeling of Anticancer Drugs: Recent Advances. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 275-296. <a href="https://doi.org/10.1007/978-3-030-71975-3_10" title="DOI URL">https://doi.org/10.1007/978-3-030-71975-3_10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-71975-3_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-71975-3_10%26sid%3Dliteratum%253Aachs%26atitle%3DIn%252BSilico%252BModeling%252Bof%252BAnticancer%252BDrugs%25253A%252BRecent%252BAdvances%26aulast%3DSharma%26aufirst%3DSmriti%26date%3D2021%26date%3D2021%26spage%3D275%26epage%3D296%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DAdvanced%252BMachine%252BLearning%252BApproaches%252Bin%252BCancer%252BPrognosis%26aulast%3DNayak%26aufirst%3DJanmenjoy%26date%3D2021%26volume%3D204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akira  Mizuno</span>, <span class="hlFld-ContribAuthor ">Tomoshi  Kameda</span>, <span class="hlFld-ContribAuthor ">Tomoki  Kuwahara</span>, <span class="hlFld-ContribAuthor ">Hideyuki  Endoh</span>, <span class="hlFld-ContribAuthor ">Yoshihiko  Ito</span>, <span class="hlFld-ContribAuthor ">Shizuo  Yamada</span>, <span class="hlFld-ContribAuthor ">Kimiko  Hasegawa</span>, <span class="hlFld-ContribAuthor ">Akihito  Yamano</span>, <span class="hlFld-ContribAuthor ">Mizuki  Watanabe</span>, <span class="hlFld-ContribAuthor ">Mitsuhiro  Arisawa</span>, <span class="hlFld-ContribAuthor ">Satoshi  Shuto</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclopropane-Based Peptidomimetics Mimicking Wide-Ranging Secondary Structures of Peptides: Conformational Analysis and Their Use in Rational Ligand Optimization. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (13)
                                     , 3159-3168. <a href="https://doi.org/10.1002/chem.201605312" title="DOI URL">https://doi.org/10.1002/chem.201605312</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201605312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201605312%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DCyclopropane-Based%252BPeptidomimetics%252BMimicking%252BWide-Ranging%252BSecondary%252BStructures%252Bof%252BPeptides%25253A%252BConformational%252BAnalysis%252Band%252BTheir%252BUse%252Bin%252BRational%252BLigand%252BOptimization%26aulast%3DMizuno%26aufirst%3DAkira%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D13%26spage%3D3159%26epage%3D3168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.D.  Pickett</span>. </span><span class="cited-content_cbyCitation_article-title">Library Design: Reactant and Product-Based Approaches. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,<a href="https://doi.org/10.1016/B978-0-12-409547-2.02568-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.02568-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.02568-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.02568-3%26sid%3Dliteratum%253Aachs%26atitle%3DLibrary%252BDesign%25253A%252BReactant%252Band%252BProduct-Based%252BApproaches%26aulast%3DPickett%26aufirst%3DS.D.%26date%3D2013%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Markus  Boehm</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening of Chemical Space: From Generic Compound Collections to Tailored Screening Libraries. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 1-33. <a href="https://doi.org/10.1002/9783527633326.ch1" title="DOI URL">https://doi.org/10.1002/9783527633326.ch1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527633326.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527633326.ch1%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%252Bof%252BChemical%252BSpace%25253A%252BFrom%252BGeneric%252BCompound%252BCollections%252Bto%252BTailored%252BScreening%252BLibraries%26aulast%3DBoehm%26aufirst%3DMarkus%26date%3D2011%26date%3D2011%26spage%3D1%26epage%3D33%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DVirtual%252BScreening%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2011%26volume%3D432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roland E.  Dolle</span>, <span class="hlFld-ContribAuthor ">Karin  Worm</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Chemistry in Lead Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2010,</strong>,, 259-290. <a href="https://doi.org/10.1002/9780470584170.ch9" title="DOI URL">https://doi.org/10.1002/9780470584170.ch9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9780470584170.ch9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9780470584170.ch9%26sid%3Dliteratum%253Aachs%26atitle%3DRole%252Bof%252BChemistry%252Bin%252BLead%252BDiscovery%26aulast%3DDolle%26aufirst%3DRoland%2BE.%26date%3D2010%26date%3D2010%26spage%3D259%26epage%3D290%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26atitle%3DLead%252BGeneration%252BApproaches%252Bin%252BDrug%252BDiscovery%26aulast%3DRankovic%26aufirst%3DZoran%26date%3D2010%26date%3D2010%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongbo  Li</span>, <span class="hlFld-ContribAuthor ">Meng-an  Hao</span>, <span class="hlFld-ContribAuthor ">Liping  Wang</span>, <span class="hlFld-ContribAuthor ">Wu  Liang</span>, <span class="hlFld-ContribAuthor ">Kai  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Preparation of Mono
              Boc
              -Protected Unsymmetrical Diamines. </span><span class="cited-content_cbyCitation_journal-name">Organic Preparations and Procedures International</span><span> <strong>2009,</strong> <em>41 </em>
                                    (4)
                                     , 301-307. <a href="https://doi.org/10.1080/00304940903077998" title="DOI URL">https://doi.org/10.1080/00304940903077998</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00304940903077998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00304940903077998%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Preparations%2520and%2520Procedures%2520International%26atitle%3DPreparation%252Bof%252BMono%252BBoc%252B-Protected%252BUnsymmetrical%252BDiamines%26aulast%3DLi%26aufirst%3DHongbo%26date%3D2009%26volume%3D41%26issue%3D4%26spage%3D301%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dae Woo  Lee</span>, <span class="hlFld-ContribAuthor ">Hyun‐Joon  Ha</span>, <span class="hlFld-ContribAuthor ">Won Koo  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Mono‐BOC Protection of Diamines. </span><span class="cited-content_cbyCitation_journal-name">Synthetic Communications</span><span> <strong>2007,</strong> <em>37 </em>
                                    (5)
                                     , 737-742. <a href="https://doi.org/10.1080/00397910601131403" title="DOI URL">https://doi.org/10.1080/00397910601131403</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/00397910601131403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F00397910601131403%26sid%3Dliteratum%253Aachs%26jtitle%3DSynthetic%2520Communications%26atitle%3DSelective%252BMono%2525E2%252580%252590BOC%252BProtection%252Bof%252BDiamines%26aulast%3DLee%26aufirst%3DDae%2BWoo%26date%3D2007%26volume%3D37%26issue%3D5%26spage%3D737%26epage%3D742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.D.  Pickett</span>. </span><span class="cited-content_cbyCitation_article-title">Library Design: Reactant and Product-Based Approaches. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2007,</strong>,, 337-378. <a href="https://doi.org/10.1016/B0-08-045044-X/00257-1" title="DOI URL">https://doi.org/10.1016/B0-08-045044-X/00257-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B0-08-045044-X/00257-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB0-08-045044-X%2F00257-1%26sid%3Dliteratum%253Aachs%26atitle%3DLibrary%252BDesign%25253A%252BReactant%252Band%252BProduct-Based%252BApproaches%26aulast%3DPickett%26aufirst%3DS.D.%26date%3D2007%26spage%3D337%26epage%3D378%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BII%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wanlong  Jiang</span>, <span class="hlFld-ContribAuthor ">Fabio C.  Tucci</span>, <span class="hlFld-ContribAuthor ">Caroline W.  Chen</span>, <span class="hlFld-ContribAuthor ">Melissa  Arellano</span>, <span class="hlFld-ContribAuthor ">Joe A.  Tran</span>, <span class="hlFld-ContribAuthor ">Nicole S.  White</span>, <span class="hlFld-ContribAuthor ">Dragan  Marinkovic</span>, <span class="hlFld-ContribAuthor ">Joseph  Pontillo</span>, <span class="hlFld-ContribAuthor ">Beth A.  Fleck</span>, <span class="hlFld-ContribAuthor ">Jenny  Wen</span>, <span class="hlFld-ContribAuthor ">John  Saunders</span>, <span class="hlFld-ContribAuthor ">Ajay  Madan</span>, <span class="hlFld-ContribAuthor ">Alan C.  Foster</span>, <span class="hlFld-ContribAuthor ">Chen  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Arylpropionylpiperazines as antagonists of the human melanocortin-4 receptor. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2006,</strong> <em>16 </em>
                                    (17)
                                     , 4674-4678. <a href="https://doi.org/10.1016/j.bmcl.2006.05.088" title="DOI URL">https://doi.org/10.1016/j.bmcl.2006.05.088</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2006.05.088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2006.05.088%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DArylpropionylpiperazines%252Bas%252Bantagonists%252Bof%252Bthe%252Bhuman%252Bmelanocortin-4%252Breceptor%26aulast%3DJiang%26aufirst%3DWanlong%26date%3D2006%26volume%3D16%26issue%3D17%26spage%3D4674%26epage%3D4678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guillaume J  Hervieu</span>. </span><span class="cited-content_cbyCitation_article-title">Further insights into the neurobiology of melanin-concentrating hormone in energy and mood balances. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2006,</strong> <em>10 </em>
                                    (2)
                                     , 211-229. <a href="https://doi.org/10.1517/14728222.10.2.211" title="DOI URL">https://doi.org/10.1517/14728222.10.2.211</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/14728222.10.2.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F14728222.10.2.211%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DFurther%252Binsights%252Binto%252Bthe%252Bneurobiology%252Bof%252Bmelanin-concentrating%252Bhormone%252Bin%252Benergy%252Band%252Bmood%252Bbalances%26aulast%3DHervieu%26aufirst%3DGuillaume%2BJ%26date%3D2006%26date%3D2006%26volume%3D10%26issue%3D2%26spage%3D211%26epage%3D229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Trond  Ulven</span>, <span class="hlFld-ContribAuthor ">Paul Brian  Little</span>, <span class="hlFld-ContribAuthor ">Jean-Marie  Receveur</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Frimurer</span>, <span class="hlFld-ContribAuthor ">Øystein  Rist</span>, <span class="hlFld-ContribAuthor ">Pia K.  Nørregaard</span>, <span class="hlFld-ContribAuthor ">Thomas  Högberg</span>. </span><span class="cited-content_cbyCitation_article-title">6-Acylamino-2-amino-4-methylquinolines as potent melanin-concentrating hormone 1 receptor antagonists: Structure–activity exploration of eastern and western parts. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2006,</strong> <em>16 </em>
                                    (4)
                                     , 1070-1075. <a href="https://doi.org/10.1016/j.bmcl.2005.10.066" title="DOI URL">https://doi.org/10.1016/j.bmcl.2005.10.066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2005.10.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2005.10.066%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3D6-Acylamino-2-amino-4-methylquinolines%252Bas%252Bpotent%252Bmelanin-concentrating%252Bhormone%252B1%252Breceptor%252Bantagonists%25253A%252BStructure%2525E2%252580%252593activity%252Bexploration%252Bof%252Beastern%252Band%252Bwestern%252Bparts%26aulast%3DUlven%26aufirst%3DTrond%26date%3D2006%26volume%3D16%26issue%3D4%26spage%3D1070%26epage%3D1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lutz  Weber</span>. </span><span class="cited-content_cbyCitation_article-title">Current Status of Virtual Combinatorial Library Design. </span><span class="cited-content_cbyCitation_journal-name">QSAR & Combinatorial Science</span><span> <strong>2005,</strong> <em>24 </em>
                                    (7)
                                     , 809-823. <a href="https://doi.org/10.1002/qsar.200510120" title="DOI URL">https://doi.org/10.1002/qsar.200510120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/qsar.200510120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fqsar.200510120%26sid%3Dliteratum%253Aachs%26jtitle%3DQSAR%2520%2526%2520Combinatorial%2520Science%26atitle%3DCurrent%252BStatus%252Bof%252BVirtual%252BCombinatorial%252BLibrary%252BDesign%26aulast%3DWeber%26aufirst%3DLutz%26date%3D2005%26volume%3D24%26issue%3D7%26spage%3D809%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tao  Guo</span>, <span class="hlFld-ContribAuthor ">Rachael C.  Hunter</span>, <span class="hlFld-ContribAuthor ">Huizhong  Gu</span>, <span class="hlFld-ContribAuthor ">Laura L.  Rokosz</span>, <span class="hlFld-ContribAuthor ">Tara M.  Stauffer</span>, <span class="hlFld-ContribAuthor ">Doug W.  Hobbs</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and SAR of 4-amino-2-biarylbutylurea MCH 1 receptor antagonists through solid-phase parallel synthesis. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2005,</strong> <em>15 </em>
                                    (16)
                                     , 3691-3695. <a href="https://doi.org/10.1016/j.bmcl.2005.05.039" title="DOI URL">https://doi.org/10.1016/j.bmcl.2005.05.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2005.05.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2005.05.039%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252BSAR%252Bof%252B4-amino-2-biarylbutylurea%252BMCH%252B1%252Breceptor%252Bantagonists%252Bthrough%252Bsolid-phase%252Bparallel%252Bsynthesis%26aulast%3DGuo%26aufirst%3DTao%26date%3D2005%26volume%3D15%26issue%3D16%26spage%3D3691%26epage%3D3695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tao  Guo</span>, <span class="hlFld-ContribAuthor ">Yuefei  Shao</span>, <span class="hlFld-ContribAuthor ">Gang  Qian</span>, <span class="hlFld-ContribAuthor ">Laura L.  Rokosz</span>, <span class="hlFld-ContribAuthor ">Tara M.  Stauffer</span>, <span class="hlFld-ContribAuthor ">Rachael C.  Hunter</span>, <span class="hlFld-ContribAuthor ">Suresh D.  Babu</span>, <span class="hlFld-ContribAuthor ">Huizhong  Gu</span>, <span class="hlFld-ContribAuthor ">Doug W.  Hobbs</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and SAR of biaryl piperidine MCH1 receptor antagonists through solid-phase encoded combinatorial synthesis. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2005,</strong> <em>15 </em>
                                    (16)
                                     , 3696-3700. <a href="https://doi.org/10.1016/j.bmcl.2005.05.085" title="DOI URL">https://doi.org/10.1016/j.bmcl.2005.05.085</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2005.05.085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2005.05.085%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252BSAR%252Bof%252Bbiaryl%252Bpiperidine%252BMCH1%252Breceptor%252Bantagonists%252Bthrough%252Bsolid-phase%252Bencoded%252Bcombinatorial%252Bsynthesis%26aulast%3DGuo%26aufirst%3DTao%26date%3D2005%26volume%3D15%26issue%3D16%26spage%3D3696%26epage%3D3700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N.K.  Terrett</span>. </span><span class="cited-content_cbyCitation_article-title">Combinatorial Chemistry Online. </span><span class="cited-content_cbyCitation_journal-name">Combinatorial Chemistry - an Online Journal</span><span> <strong>2005,</strong> <em>7 </em>
                                    (3)
                                     , 9-13. <a href="https://doi.org/10.1016/j.comche.2005.02.001" title="DOI URL">https://doi.org/10.1016/j.comche.2005.02.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.comche.2005.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.comche.2005.02.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCombinatorial%2520Chemistry%2520-%2520an%2520Online%2520Journal%26atitle%3DCombinatorial%252BChemistry%252BOnline%26aulast%3DTerrett%26aufirst%3DN.K.%26date%3D2005%26volume%3D7%26issue%3D3%26spage%3D9%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm040084cf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00001.jpeg" id="rightTab-_i6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 (A) Set of known non-peptide ligands for GPCR−PA<sup>+</sup> (filled circles) mapped into three dimensions of BCUT space vs the WDI−NBI (gray dots; representative sampling only). (B) Projection of the test set (crosses) into drug space using the BCUT methodology each compared to GPCR−PA<sup>+</sup> space (filled circles). (<i>P</i> = polarizability axis).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm040084cf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00002.jpeg" id="rightTab-_i8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Templates selected for virtual library enumeration (points of monomer attachment are via NH or COOH groups). Numbers located near to the reacting functional group indicate an association with a specific monomer listed in Table 2. In some instances, because of symmetry in the template, this nomenclature is redundant. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm040084ch00001" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084ch00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00001.jpeg" id="rightTab-_i10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 1.  Compounds Derived from Template <b>3</b> and Synthetic Routes<i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> Reagents:  (a) R<sup>1</sup>CHO, Na(OAc)<sub>3</sub>BH, DCE, rt, 4 h; or (2-bromoethyl)benzene or 1-bromo-3,5,5-trimethylhexane, DMF, Hunig's base, 40°C, 12 h; (b) TFA, DCM, 1 h, rt; (c) R<sup>2</sup>CO<sub>2</sub>H, EDCI, HOBT, DMF/NMP, rt, 12 h; (d) 5-(4-chlorophenyl)-2-furaldehyde, Na(OAc)<sub>3</sub>BH, DCE, rt, 12 h.</p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084ch00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm040084ch00002" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084ch00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00002.jpeg" id="rightTab-_i11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 2.  Synthetic Chemistry Associated with Template <b>4</b><i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> Reagents:  (a) R<sup>1</sup>CO<sub>2</sub>H, EDCI, HOBT, DCM, rt, 4 h, 40−85%; (b) TFA, DCM, rt, 1 h, 100%; (c) R<sup>2</sup>Br, Hunig's base, 40 °C, 16 h, 50−85%.</p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084ch00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084ch00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm040084cf00003" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00003.jpeg" id="rightTab-_i14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Representative dose−response curves of MCH−R hits from the test set. The <i>K</i><sub>i</sub> values are derived from the IC<sub>50</sub> values of inhibition using the Cheng−Prusoff equation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm040084cf00004" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/medium/jm040084cf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00004.jpeg" id="rightTab-_i18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Screening hits (squares) from the test and random sets projected into drug space against a background of GPCR−PA<sup>+</sup> ligands (open circles). (A) MCH−R hits from the test set; (B) GnRH−R hits from the test set; (C) MC4−R hits from the test set; (D) hits from the random set [(1) region of GnRH random hits; (2) MC4, and (3) MCH)].</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-27/jm040084c/production/images/large/jm040084cf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm040084c&amp;id=jm040084cf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"></div><p class="rightTab-references-count">
            This article references 20 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm040084cb00001"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00001_1"><span><span class="NLM_string-ref">The most up to date listing can be viewed and searched on the GPCRDB home page at <a class="ext-link" href="http://www.darmstadt.gmd.de/~gpcrdb" target="_blank">http://www.darmstadt.gmd.de/∼gpcrdb</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+most+up+to+date+listing+can+be+viewed+and+searched+on+the+GPCRDB+home+page+at+http%3A%2F%2Fwww.darmstadt.gmd.de%2F%E2%88%BCgpcrdb.+The+most+up+to+date+listing+can+be+viewed+and+searched+on+the+GPCRDB+home+page+at+http%3A%2F%2Fwww.darmstadt.gmd.de%2F%E2%88%BCgpcrdb."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.darmstadt.gmd.de%2F%E2%88%BCgpcrdb" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00002"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00002_1"><span><span class="NLM_string-ref">Watson, S.; Arkinstall, S. <i>  The G-Protein Linked Receptor FactsBook</i>; Academic Press:  San Diego, CA, 1996.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=S.+Watson&author=S.+Arkinstall&title=The+G-Protein+Linked+Receptor+FactsBook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DS.%26btitle%3DThe%2520G-Protein%2520Linked%2520Receptor%2520FactsBook%26pub%3DAcademic%2520Press%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00003"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00003_1"><span><span class="NLM_string-ref">For a list of the Top 200 Prescription Medicines see:  <a class="ext-link" href="http://www.rxlist.com/top200.htm" target="_blank">http://www.rxlist.com/top200.htm</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+For+a+list+of+the+Top+200+Prescription+Medicines+see%3A%E2%80%89+http%3A%2F%2Fwww.rxlist.com%2Ftop200.htm.+For+a+list+of+the+Top+200+Prescription+Medicines+see%3A%E2%80%89+http%3A%2F%2Fwww.rxlist.com%2Ftop200.htm."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.rxlist.com%2Ftop200.htm" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00004"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00004_1"><span><span class="NLM_string-ref">Marshall, G. R. Peptide Interactions with G-Protein Coupled Receptors. <i>  Biopolymers (Pept. Sci.)</i><b>2001</b>, <i>60</i>, 246−277.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2001&pages=277&author=G.+R+Marshall&title=Peptide+Interactions+with+G-Protein+Coupled+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarshall%26aufirst%3DG.%2BR%26atitle%3DPeptide%2520Interactions%2520with%2520G-Protein%2520Coupled%2520Receptors%26jtitle%3DBiopolymers%2520%2528Pept.%2520Sci.%2529%26date%3D2001%26volume%3D60%26spage%3D277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00005"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00005_1"><span><span class="NLM_string-ref">Rees, D. C. Non-Peptide Ligands for Neuropeptide Receptors. <i>  Annu. Rep. </i><i>Med. Chem</i>. <b>1993</b>, <i>28</i>, 59−68.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1993&pages=68&author=D.+C+Rees&title=Non-Peptide+Ligands+for+Neuropeptide+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC%26atitle%3DNon-Peptide%2520Ligands%2520for%2520Neuropeptide%2520Receptors%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D1993%26volume%3D28%26spage%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00006"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00006_1"><span><span class="NLM_string-ref">Sugg, E. E. Non-peptide Agonists for Peptide Receptors:  Lessons from Ligands. <i>  Annu. Rep. Med Chem</i>. <b>1997</b>, <i>32</i>, 277.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1997&pages=277&author=E.+E+Sugg&title=Non-peptide+Agonists+for+Peptide+Receptors%3A%E2%80%89+Lessons+from+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSugg%26aufirst%3DE.%2BE%26atitle%3DNon-peptide%2520Agonists%2520for%2520Peptide%2520Receptors%253A%25E2%2580%2589%2520Lessons%2520from%2520Ligands%26jtitle%3DAnnu.%2520Rep.%2520Med%2520Chem.%26date%3D1997%26volume%3D32%26spage%3D277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00007"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00007_1"><span><span class="NLM_string-ref">Yang, L. Non-Peptide Somatostatin Receptor Ligands. <i>  Annu. Rep. Med. </i><i>Chem.</i><b>1999</b>, <i>34</i>, 209−218.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=218&author=L+Yang&title=Non-Peptide+Somatostatin+Receptor+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL%26atitle%3DNon-Peptide%2520Somatostatin%2520Receptor%2520Ligands%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D1999%26volume%3D34%26spage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00008"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00008_1"><span><span class="NLM_string-ref">Pearlman, R. S.; Smith, K. M. Novel Software Tools for Chemical Diversity. <i>  Perspect. Drug Discovery</i><b>1998</b>, <i>9</i>, 339−353.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1998&pages=353&author=R.+S.+Pearlman&author=K.+M+Smith&title=Novel+Software+Tools+for+Chemical+Diversity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DR.%2BS.%26atitle%3DNovel%2520Software%2520Tools%2520for%2520Chemical%2520Diversity%26jtitle%3DPerspect.%2520Drug%2520Discovery%26date%3D1998%26volume%3D9%26spage%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00009"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00009_1"><span><span class="NLM_string-ref">Pearlman, R. S.; Smith, K. M. Metric Validation and the Receptor-Relevant Subspace Concept. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>1999</b>, <i>39</i>, 28−35.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=35&author=R.+S.+Pearlman&author=K.+M+Smith&title=Metric+Validation+and+the+Receptor-Relevant+Subspace+Concept"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPearlman%26aufirst%3DR.%2BS.%26atitle%3DMetric%2520Validation%2520and%2520the%2520Receptor-Relevant%2520Subspace%2520Concept%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm040084cb00010_1"><span><span class="NLM_string-ref">Williams, J. P.; Lavrador, K. A Solution-Phase Combinatorial Synthesis of Selective Dopamine D<sub>4</sub> Ligands. <i>  Comb. Chem. HTS</i><b>2000</b>, <i>3</i>, 43−50.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2000&pages=50&author=J.+P.+Williams&author=K+Lavrador&title=A+Solution-Phase+Combinatorial+Synthesis+of+Selective+Dopamine+D4+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BP.%26atitle%3DA%2520Solution-Phase%2520Combinatorial%2520Synthesis%2520of%2520Selective%2520Dopamine%2520D4%2520Ligands%26jtitle%3DComb.%2520Chem.%2520HTS%26date%3D2000%26volume%3D3%26spage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00011"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00011_1"><span><span class="NLM_string-ref">Zeng, L.; Burton, L.; Yung, K.; Shushan, B.; Kassel, D. B. Automated Analytical/Preparative High-Performance Liquid Chromatography−Mass Spectrometry System for the Rapid Characterization and Purification of Compound Libraries. <i>  J. Chromatogr. A</i><b>1998</b>, <i>794</i>, 3−13.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=794&publication_year=1998&pages=13&author=L.+Zeng&author=L.+Burton&author=K.+Yung&author=B.+Shushan&author=D.+B.+Kassel&author=+Automated+Analytical&title=System+for+the+Rapid+Characterization+and+Purification+of+Compound+Libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26atitle%3DSystem%2520for%2520the%2520Rapid%2520Characterization%2520and%2520Purification%2520of%2520Compound%2520Libraries%26jtitle%3DJ.%2520Chromatogr.%2520A%26date%3D1998%26volume%3D794%26spage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00012"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00012_1"><span><span class="NLM_string-ref">Abdel-Magid, A. F.; Carson, K. G.; Harris, B. D.; Maryanoff, C. A.; Shah, R. D. Reductive Amination of Aldehydes and Ketones with Sodium Triacetoxyborohydride. Studies on Direct and Indirect Reductive Amination Procedures. <i>  J. Org. Chem</i>. <b>1996</b>, <i>61</i>, 3849−3862.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1996&pages=3862&author=A.+F.+Abdel-Magid&author=K.+G.+Carson&author=B.+D.+Harris&author=C.+A.+Maryanoff&author=R.+D+Shah&title=Reductive+Amination+of+Aldehydes+and+Ketones+with+Sodium+Triacetoxyborohydride.+Studies+on+Direct+and+Indirect+Reductive+Amination+Procedures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26atitle%3DReductive%2520Amination%2520of%2520Aldehydes%2520and%2520Ketones%2520with%2520Sodium%2520Triacetoxyborohydride.%2520Studies%2520on%2520Direct%2520and%2520Indirect%2520Reductive%2520Amination%2520Procedures%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1996%26volume%3D61%26spage%3D3862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00013"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00013_1"><span><span class="NLM_string-ref">Cone, R. D.; Lu, D.; Koppula, S.; Vage, D. I.; Klungland, H.; Boston, B.; Chen, W.; Orth, D. N.; Pouton, C.; Kesterson, R. A. The Melanocortin Receptors:  Agonists, Antagonists, and the Hormonal Control of Pigmentation. <i>  Rec. Prog. Hor. Res.</i><b>1996</b>, <i>51</i>, 287−318.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1996&pages=318&author=R.+D.+Cone&author=D.+Lu&author=S.+Koppula&author=D.+I.+Vage&author=H.+Klungland&author=B.+Boston&author=W.+Chen&author=D.+N.+Orth&author=C.+Pouton&author=R.+A+Kesterson&title=The+Melanocortin+Receptors%3A%E2%80%89+Agonists%2C+Antagonists%2C+and+the+Hormonal+Control+of+Pigmentation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCone%26aufirst%3DR.%2BD.%26atitle%3DThe%2520Melanocortin%2520Receptors%253A%25E2%2580%2589%2520Agonists%252C%2520Antagonists%252C%2520and%2520the%2520Hormonal%2520Control%2520of%2520Pigmentation%26jtitle%3DRec.%2520Prog.%2520Hor.%2520Res.%26date%3D1996%26volume%3D51%26spage%3D318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00014"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00014_1"><span><span class="NLM_string-ref">Goodfellow, V. S.; Saunders, J. The Melanocortin System and Its Role in Obesity and Cachexia. <i>  Curr. Top. Med. Chem.</i><b>2003</b>, <i>3</i>, 855−883.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2003&pages=883&author=V.+S.+Goodfellow&author=J+Saunders&title=The+Melanocortin+System+and+Its+Role+in+Obesity+and+Cachexia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoodfellow%26aufirst%3DV.%2BS.%26atitle%3DThe%2520Melanocortin%2520System%2520and%2520Its%2520Role%2520in%2520Obesity%2520and%2520Cachexia%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2003%26volume%3D3%26spage%3D883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00015"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00015_1"><span><span class="NLM_string-ref">Qu, D.; Ludwig, D. S.; Gammeltoft, S.; Piper, M.; Pelleymounter, M. A.; Cullen, M. J.; Mathes, W. F.; Przypek, J.; Kanarek, R.; Maratos-Flier, E. A Role For Melanin-Concentrating Hormone in the Central Regulation of Feeding Behavior. <i>  Nature</i><b>1996</b>, <i>380</i>, 243−7.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=380&publication_year=1996&pages=7&author=D.+Qu&author=D.+S.+Ludwig&author=S.+Gammeltoft&author=M.+Piper&author=M.+A.+Pelleymounter&author=M.+J.+Cullen&author=W.+F.+Mathes&author=J.+Przypek&author=R.+Kanarek&author=E+Maratos-Flier&title=A+Role+For+Melanin-Concentrating+Hormone+in+the+Central+Regulation+of+Feeding+Behavior"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DQu%26aufirst%3DD.%26atitle%3DA%2520Role%2520For%2520Melanin-Concentrating%2520Hormone%2520in%2520the%2520Central%2520Regulation%2520of%2520Feeding%2520Behavior%26jtitle%3DNature%26date%3D1996%26volume%3D380%26spage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00016"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00016_1"><span><span class="NLM_string-ref">Borowsky, B.; Durkin, M. M.; Ogozalek, K.; Marzabadi, M. R.; DeLeon, J.; Lagu, B.; Heurich, R.; Lichtblau, H.; Shaposhnik, Z.; Daniewska, I.; Blackburn, T. P.; Branchek, T. A.; Gerald, C.; Vaysse, P. J.; Forray, C. Antidepressant, Anxiolytic and Anorectic Effects of a Melanin-Concentrating Hormone-1 Receptor Antagonist. <i>  Nat. Med</i>. <b>2002</b>, <i>8</i>, 825−30.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2002&pages=30&author=B.+Borowsky&author=M.+M.+Durkin&author=K.+Ogozalek&author=M.+R.+Marzabadi&author=J.+DeLeon&author=B.+Lagu&author=R.+Heurich&author=H.+Lichtblau&author=Z.+Shaposhnik&author=I.+Daniewska&author=T.+P.+Blackburn&author=T.+A.+Branchek&author=C.+Gerald&author=P.+J.+Vaysse&author=C.+Forray&author=+Antidepressant&title=Effects+of+a+Melanin-Concentrating+Hormone-1+Receptor+Antagonist"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBorowsky%26aufirst%3DB.%26atitle%3DEffects%2520of%2520a%2520Melanin-Concentrating%2520Hormone-1%2520Receptor%2520Antagonist%26jtitle%3DNat.%2520Med.%26date%3D2002%26volume%3D8%26spage%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00017"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00017_1"><span><span class="NLM_string-ref">Sealfon, S. C.; Weinstein, H.; Millar, R. P. Molecular Mechanisms of Ligand Interaction with the Gonadotropin-Releasing Hormone Receptor. <i>  Endocr. Rev</i>. <b>1997</b>, <i>18</i>, 180−205.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=205&author=S.+C.+Sealfon&author=H.+Weinstein&author=R.+P+Millar&title=Molecular+Mechanisms+of+Ligand+Interaction+with+the+Gonadotropin-Releasing+Hormone+Receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSealfon%26aufirst%3DS.%2BC.%26atitle%3DMolecular%2520Mechanisms%2520of%2520Ligand%2520Interaction%2520with%2520the%2520Gonadotropin-Releasing%2520Hormone%2520Receptor%26jtitle%3DEndocr.%2520Rev.%26date%3D1997%26volume%3D18%26spage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00018"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00018_1"><span><span class="NLM_string-ref">Emons, G.; Grundker, C.; Gunthert, A. R.; Westphalen, S.; Kavanagh, J.; Versschraege, C. GnRH Antagonists in the Treatment of Gynaecological and Breast Cancers. <i>  Endocr. Relat. Cancer</i><b>2003</b>, <i>10</i>, 291−299.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=299&author=G.+Emons&author=C.+Grundker&author=A.+R.+Gunthert&author=S.+Westphalen&author=J.+Kavanagh&author=C+Versschraege&title=GnRH+Antagonists+in+the+Treatment+of+Gynaecological+and+Breast+Cancers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEmons%26aufirst%3DG.%26atitle%3DGnRH%2520Antagonists%2520in%2520the%2520Treatment%2520of%2520Gynaecological%2520and%2520Breast%2520Cancers%26jtitle%3DEndocr.%2520Relat.%2520Cancer%26date%3D2003%26volume%3D10%26spage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00019"><div class="reference"><strong class="refLabel"><a href="#jm040084cb00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm040084cb00019_1"><span><span class="NLM_string-ref">Zhang, J.-H.; Chung, T. D. Y.; Oldenburg, K. R. A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays. <i>  J. Biomol. Screening</i><b>1999</b>, <i>4</i>, 67−73.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=73&author=J.-H.+Zhang&author=T.+D.+Y.+Chung&author=K.+R+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.-H.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D1999%26volume%3D4%26spage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm040084cb00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm040084cb00020_1"><span><span class="NLM_string-ref">Ohmori, J.; Maeno, K.; Hidaka, K.; Nakato, K.; Matsumoto, M.; Tada, S.; Hattori, H.; Sakamoto, S.; Tsukamoto, S. Dopamine D3 and D4 receptors Antagonists:  Synthesis and Structure−Activity Relationships of (S)-(+)-N-(1-Benzyl-3-pyrrolidinyl)-5-chloro-4-[(cyclopropylcarbonyl)amino]-2-methoxybenzamide (YM-43611) and related compounds. <i>  J. Med. </i><i>Chem.</i><b>1996</b>, <i>39</i>, 2764−2772.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2772&author=J.+Ohmori&author=K.+Maeno&author=K.+Hidaka&author=K.+Nakato&author=M.+Matsumoto&author=S.+Tada&author=H.+Hattori&author=S.+Sakamoto&author=S+Tsukamoto&title=Dopamine+D3+and+D4+receptors+Antagonists%3A%E2%80%89+Synthesis+and+Structure%E2%88%92Activity+Relationships+of+%28S%29-%28%2B%29-N-%281-Benzyl-3-pyrrolidinyl%29-5-chloro-4-%5B%28cyclopropylcarbonyl%29amino%5D-2-methoxybenzamide+%28YM-43611%29+and+related+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm040084cb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOhmori%26aufirst%3DJ.%26atitle%3DDopamine%2520D3%2520and%2520D4%2520receptors%2520Antagonists%253A%25E2%2580%2589%2520Synthesis%2520and%2520Structure%25E2%2588%2592Activity%2520Relationships%2520of%2520%2528S%2529-%2528%252B%2529-N-%25281-Benzyl-3-pyrrolidinyl%2529-5-chloro-4-%255B%2528cyclopropylcarbonyl%2529amino%255D-2-methoxybenzamide%2520%2528YM-43611%2529%2520and%2520related%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="NLM_notes" id="rightTab-"><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21"><a href="/doi/suppl/10.1021%2Fjm040084c">Supporting Information Available</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00034" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00034" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">HPLC purity data and MALDI−FTMS data. Color versions of Figures 1 and 4 are available form the authors on request. This material is available free of charge via the Internet at htpp://pubs.acs.org. </div></div></div><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm040084c/suppl_file/jm040084c_s.pdf">jm040084c_s.pdf (59.58 kb)</a></li></ul></div></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm040084c&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Fjm040084c%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2004.47.issue-27%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm040084c" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                391KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679918f688e33d4e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
