
<html lang="en"     class="pb-page"  data-request-id="1eee5f05-7a58-4af4-9a63-9e535d951d13"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2018.61.issue-16;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b00989;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Sulfonamido-2-arylbenzoxazole GroEL/ES Inhibitors as Potent Antibacterials against Methicillin-Resistant Staphylococcus aureus (MRSA)" /></meta><meta name="dc.Creator" content="Sanofar  Abdeen" /></meta><meta name="dc.Creator" content="Trent  Kunkle" /></meta><meta name="dc.Creator" content="Nilshad  Salim" /></meta><meta name="dc.Creator" content="Anne-Marie  Ray" /></meta><meta name="dc.Creator" content="Najiba  Mammadova" /></meta><meta name="dc.Creator" content="Corey  Summers" /></meta><meta name="dc.Creator" content="Mckayla  Stevens" /></meta><meta name="dc.Creator" content="Andrew J.  Ambrose" /></meta><meta name="dc.Creator" content="Yangshin  Park" /></meta><meta name="dc.Creator" content="Peter G.  Schultz" /></meta><meta name="dc.Creator" content="Arthur L.  Horwich" /></meta><meta name="dc.Creator" content="Quyen Q.  Hoang" /></meta><meta name="dc.Creator" content="Eli  Chapman" /></meta><meta name="dc.Creator" content="Steven M.  Johnson" /></meta><meta name="dc.Description" content="Extending from a study we recently published examining the antitrypanosomal effects of a series of GroEL/ES inhibitors based on a pseudosymmetrical bis-sulfonamido-2-phenylbenzoxazole scaffold, her..." /></meta><meta name="Description" content="Extending from a study we recently published examining the antitrypanosomal effects of a series of GroEL/ES inhibitors based on a pseudosymmetrical bis-sulfonamido-2-phenylbenzoxazole scaffold, her..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 30, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00989" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00989" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00989" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00989" /></link>
        
    
    

<title>Sulfonamido-2-arylbenzoxazole GroEL/ES Inhibitors as Potent Antibacterials against Methicillin-Resistant Staphylococcus aureus (MRSA) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00989" /></meta><meta property="og:title" content="Sulfonamido-2-arylbenzoxazole GroEL/ES Inhibitors as Potent Antibacterials against Methicillin-Resistant Staphylococcus aureus (MRSA)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0001.jpeg" /></meta><meta property="og:description" content="Extending from a study we recently published examining the antitrypanosomal effects of a series of GroEL/ES inhibitors based on a pseudosymmetrical bis-sulfonamido-2-phenylbenzoxazole scaffold, here, we report the antibiotic effects of asymmetric analogs of this scaffold against a panel of bacteria known as the ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species). While GroEL/ES inhibitors were largely ineffective against K. pneumoniae, A. baumannii, P. aeruginosa, and E. cloacae (Gram-negative bacteria), many analogs were potent inhibitors of E. faecium and S. aureus proliferation (Gram-positive bacteria, EC50 values of the most potent analogs were in the 1–2 μM range). Furthermore, even though some compounds inhibit human HSP60/10 biochemical functions in vitro (IC50 values in the 1–10 μM range), many of these exhibited moderate to low cytotoxicity to human liver and kidney cells (CC50 values &gt; 20 μM)." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00989"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00989">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00989&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00989&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00989&amp;href=/doi/10.1021/acs.jmedchem.8b00989" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 7345-7357</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00906" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01007" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Sulfonamido-2-arylbenzoxazole GroEL/ES Inhibitors as Potent Antibacterials against Methicillin-Resistant <i>Staphylococcus aureus</i> (MRSA)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sanofar Abdeen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sanofar Abdeen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sanofar++Abdeen">Sanofar Abdeen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Trent Kunkle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Trent Kunkle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Trent++Kunkle">Trent Kunkle</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nilshad Salim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nilshad Salim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nilshad++Salim">Nilshad Salim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anne-Marie Ray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anne-Marie Ray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anne-Marie++Ray">Anne-Marie Ray</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Najiba Mammadova</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Najiba Mammadova</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Najiba++Mammadova">Najiba Mammadova</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Corey Summers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Corey Summers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Corey++Summers">Corey Summers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mckayla Stevens</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mckayla Stevens</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mckayla++Stevens">Mckayla Stevens</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrew J. Ambrose</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrew J. Ambrose</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Department of Pharmacology and Toxicology, The University of Arizona, 1703 East Mabel Street, P.O. Box 210207, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+J.++Ambrose">Andrew J. Ambrose</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2932-4514" title="Orcid link">http://orcid.org/0000-0002-2932-4514</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yangshin Park</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yangshin Park</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div><div class="loa-info-affiliations-info">Stark Neurosciences Research Institute, Indiana University School of Medicine, 320 West 15th Street, Suite 414, Indianapolis, Indiana 46202, United States</div><div class="loa-info-affiliations-info">Department of Neurology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yangshin++Park">Yangshin Park</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter G. Schultz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter G. Schultz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+G.++Schultz">Peter G. Schultz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3188-1202" title="Orcid link">http://orcid.org/0000-0003-3188-1202</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Arthur L. Horwich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Arthur L. Horwich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">HHMI, Department of Genetics, Yale School of Medicine, Boyer Center for Molecular Medicine, 295 Congress Avenue, New Haven, Connecticut 06510, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Arthur+L.++Horwich">Arthur L. Horwich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Quyen Q. Hoang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Quyen Q. Hoang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div><div class="loa-info-affiliations-info">Stark Neurosciences Research Institute, Indiana University School of Medicine, 320 West 15th Street, Suite 414, Indianapolis, Indiana 46202, United States</div><div class="loa-info-affiliations-info">Department of Neurology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Quyen+Q.++Hoang">Quyen Q. Hoang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Eli Chapman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eli Chapman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Pharmacy, Department of Pharmacology and Toxicology, The University of Arizona, 1703 East Mabel Street, P.O. Box 210207, Tucson, Arizona 85721, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eli++Chapman">Eli Chapman</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Steven M. Johnson</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steven M. Johnson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Indiana University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#7f151017110c0b123f160a511a1b0a"><span class="__cf_email__" data-cfemail="03696c6b6d70776e436a762d666776">[email protected]</span></a>. Phone: 317-274-2458. Fax: 317-274-4686.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steven+M.++Johnson">Steven M. Johnson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7254-4128" title="Orcid link">http://orcid.org/0000-0001-7254-4128</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00989&amp;href=/doi/10.1021%2Facs.jmedchem.8b00989" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 7345–7357</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 30, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 June 2018</li><li><span class="item_label"><b>Published</b> online</span>30 July 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 August 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00989" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00989</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7345%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSanofar%2BAbdeen%252C%2BTrent%2BKunkle%252C%2BNilshad%2BSalim%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D16%26contentID%3Dacs.jmedchem.8b00989%26title%3DSulfonamido-2-arylbenzoxazole%2BGroEL%252FES%2BInhibitors%2Bas%2BPotent%2BAntibacterials%2Bagainst%2BMethicillin-Resistant%2BStaphylococcus%2Baureus%2B%2528MRSA%2529%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7357%26publicationDate%3DAugust%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00989"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1345</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00989" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Sulfonamido-2-arylbenzoxazole GroEL/ES Inhibitors as Potent Antibacterials against Methicillin-Resistant Staphylococcus aureus (MRSA)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sanofar&quot;,&quot;last_name&quot;:&quot;Abdeen&quot;},{&quot;first_name&quot;:&quot;Trent&quot;,&quot;last_name&quot;:&quot;Kunkle&quot;},{&quot;first_name&quot;:&quot;Nilshad&quot;,&quot;last_name&quot;:&quot;Salim&quot;},{&quot;first_name&quot;:&quot;Anne-Marie&quot;,&quot;last_name&quot;:&quot;Ray&quot;},{&quot;first_name&quot;:&quot;Najiba&quot;,&quot;last_name&quot;:&quot;Mammadova&quot;},{&quot;first_name&quot;:&quot;Corey&quot;,&quot;last_name&quot;:&quot;Summers&quot;},{&quot;first_name&quot;:&quot;Mckayla&quot;,&quot;last_name&quot;:&quot;Stevens&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;J. Ambrose&quot;},{&quot;first_name&quot;:&quot;Yangshin&quot;,&quot;last_name&quot;:&quot;Park&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;G. Schultz&quot;},{&quot;first_name&quot;:&quot;Arthur&quot;,&quot;last_name&quot;:&quot;L. Horwich&quot;},{&quot;first_name&quot;:&quot;Quyen&quot;,&quot;last_name&quot;:&quot;Q. Hoang&quot;},{&quot;first_name&quot;:&quot;Eli&quot;,&quot;last_name&quot;:&quot;Chapman&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;M. Johnson&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;7345-7357&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00989&quot;},&quot;abstract&quot;:&quot;Extending from a study we recently published examining the antitrypanosomal effects of a series of GroEL/ES inhibitors based on a pseudosymmetrical bis-sulfonamido-2-phenylbenzoxazole scaffold, here, we report the antibiotic effects of asymmetric analogs of this scaffold against a panel of bacteria known as the ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species). While GroEL/ES inhibitors were largely ineffective against K. pneumoniae, A. baumannii, P. aeruginosa, and E. cloacae (Gram-negative bacteria), many analogs were potent inhibitors of E. faecium and S. aureus proliferation (Gram-positive bacteria, EC50 values of the most potent analogs were in the 1–2 μM range). Furthermore, even though some compounds inhibit human HSP60/10 biochemical functions in vitro (IC50 values in the 1–10 μM range), many of these exhibited moderate to low cytotoxicity to human liver and kidney cells (CC50 values &gt; 20 μM)&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00989&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00989" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00989&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00989" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00989&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00989" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00989&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00989&amp;href=/doi/10.1021/acs.jmedchem.8b00989" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00989" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00989" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00989%26sid%3Dliteratum%253Aachs%26pmid%3D30060666%26genre%3Darticle%26aulast%3DAbdeen%26date%3D2018%26atitle%3DSulfonamido-2-arylbenzoxazole%2BGroEL%252FES%2BInhibitors%2Bas%2BPotent%2BAntibacterials%2Bagainst%2BMethicillin-Resistant%2BStaphylococcus%2Baureus%2B%2528MRSA%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D16%26spage%3D7345%26epage%3D7357%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=291214" title="Bacteria">Bacteria</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290808" title="Toxicity">Toxicity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/jmcmar.2018.61.issue-16/20180823/jmcmar.2018.61.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Extending from a study we recently published examining the antitrypanosomal effects of a series of GroEL/ES inhibitors based on a pseudosymmetrical bis-sulfonamido-2-phenylbenzoxazole scaffold, here, we report the antibiotic effects of asymmetric analogs of this scaffold against a panel of bacteria known as the ESKAPE pathogens (<i>Enterococcus faecium</i>, <i>Staphylococcus aureus</i>, <i>Klebsiella pneumoniae</i>, <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa</i>, and <i>Enterobacter</i> species). While GroEL/ES inhibitors were largely ineffective against <i>K. pneumoniae</i>, <i>A. baumannii</i>, <i>P. aeruginosa</i>, and <i>E. cloacae</i> (Gram-negative bacteria), many analogs were potent inhibitors of <i>E. faecium</i> and <i>S. aureus</i> proliferation (Gram-positive bacteria, EC<sub>50</sub> values of the most potent analogs were in the 1–2 μM range). Furthermore, even though some compounds inhibit human HSP60/10 biochemical functions in vitro (IC<sub>50</sub> values in the 1–10 μM range), many of these exhibited moderate to low cytotoxicity to human liver and kidney cells (CC<sub>50</sub> values > 20 μM).</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27858" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27858" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The persistence of antibiotic resistant pathogens is a significant health and economic burden worldwide. Six of the most problematic drug resistant Gram-positive (Gr+) and Gram-negative (Gr-) bacteria are commonly referred to as the ESKAPE pathogens. This panel of drug resistant bacteria includes <i>Enterococcus faecium</i> (Gr+), <i>Staphylococcus aureus</i> (Gr+), <i>Klebsiella pneumoniae</i> (Gr-), <i>Acinetobacter baumannii</i> (Gr-), <i>Pseudomonas aeruginosa</i> (Gr-), and <i>Enterobacter</i> species (Gr-).<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> Of particular significance, the CDC recently estimated that ∼80 000 individuals are infected with MRSA in the U.S. annually, with ∼11 000 deaths, the highest mortality rate among all of the antibiotic resistant bacteria. <i>S. aureus</i> associated infections range from mild skin infections to life-threatening endocarditis, osteomyelitis, or bacteremia, which are usually treated with penicillinase-resistant beta-lactams or vancomycin. For more resistant strains, newer classes of antibiotics can be prescribed, such as linezolid, daptomycin, dalfopristin, telavancin, tigecycline, ceftaroline, oritavancin, dalbavancin, or tedizolid.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> Unfortunately, despite the availability of several antimicrobials, treating MRSA infections remains a significant challenge owing to off-target toxicities of some antibiotics, lack of efficacy in life-threatening clinical infections such as bacteraemia, endocarditis, etc., and the emergence of pan-drug resistant strains.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Thus, there is an urgent need for new antibacterial drugs that function against previously unexploited targets and pathways to circumvent predisposed resistance mechanisms.</div><div class="NLM_p">Toward developing mechanistically unique antibacterial candidates, we have focused on exploiting bacterial protein homeostasis pathways. A network of molecular chaperones and proteases collectively functions to maintain protein homeostasis by assisting proteins to fold to their native, functional states or ensuring their proper degradation and recycling.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> Since such quality control mechanisms are vital to cell survival, targeting them with small molecule inhibitors should be an effective antibacterial strategy. While recent studies have investigated targeting DnaK (a molecular chaperone belonging to the HSP70 family) and Clp proteases, targeting of the bacterial GroEL/GroES chaperonin system has gone largely unexplored.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14">(11−14)</a><i>E. coli</i> GroEL, which is the prototypical member of the HSP60 chaperonin family of molecular chaperones, is a homotetradecameric protein that forms two, seven-subunit rings that stack back-to-back with one another.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a> Through a series of events driven by ATP binding and hydrolysis, unfolded substrate polypeptides are bound within the central cavity of a GroEL ring and encapsulated by the GroES cochaperonin “lid”, allowing protein folding within the sequestered chamber.<a onclick="showRef(event, 'ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20 ref21">(17−21)</a> Since the GroEL/ES chaperonin system is essential for <i>E. coli</i> survival under all growth conditions, as it likely is for other bacteria, it represents an excellent target for antibacterial development.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> Furthermore, GroEL is highly conserved in prokaryotes (generally >50% sequence identity between bacterial species, whether Gram-positive or Gram-negative); thus, targeting this molecular machinery has the potential for broad spectrum applicability. However, since human HSP60 is also highly conserved (48% identity with <i>E. coli</i> GroEL), we also need to consider whether GroEL inhibitors have off-target effects against HSP60 in human cells.</div><div class="NLM_p">Toward our goal of exploiting the GroEL/ES chaperonin system as an antibacterial strategy, we previously reported a high-throughput screen that identified 235 small molecule inhibitors of the <i>E. coli</i> GroEL/ES chaperonin system.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In a subsequent study, we evaluated 22 of these GroEL/ES inhibitor hits for their antibacterial properties against the ESKAPE pathogens.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In another study, we evaluated a series of compound <b>1</b>-based GroEL/ES inhibitors for their antibiotic effects against <i>Trypanosoma brucei</i> parasites, the causative agents of African sleeping sickness (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> As an extension of those two studies, herein, we have explored additional compound <b>1</b> analogs for their ability to selectively inhibit the prototypical <i>E. coli</i> GroEL/ES chaperonin system and growth of the ESKAPE pathogens over human HSP60/10 and liver and kidney cells. Overall, this study enabled us to better understand the bioactivity profiles of this molecular scaffold, which provided invaluable SAR to guide future studies to more rationally optimize the pharmacological properties of these antibacterial GroEL/ES chaperonin system inhibitors.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Progression of hit-to-lead compound <b>1</b> analogs. A. Compound <b>1</b> was an initial hit that emerged from our high-throughput screening for GroEL/ES inhibitors.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> We recently reported on a series of pseudosymmetrical compound <b>1</b> analogs (<b>2</b>–<b>14</b>, R<sup>1</sup> represents a variety of alkyl, aryl, and substituted phenyl substructures) that have antibiotic properties against <i>Trypanosoma brucei</i> parasites.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> B. In the present study, we initially screened the <b>1</b>–<b>14</b> analogs for their ability to inhibit a panel of Gram-positive and Gram-negative bacteria and found that the thiophene (<b>6</b>) and <i>para</i>-chloro-substituted (<b>2c-</b><i><b>p</b></i>) analogs generated the most potent and selective inhibitors of MRSA proliferation. Data plotted in the gray zones represent EC<sub>50</sub> and CC<sub>50</sub> results beyond the assay detection limits (i.e., > 100 μM). C. Based on initial SAR, compound <b>15</b> was developed (bearing the 2-chlorothiophene group on both sides) along with two parallel series of asymmetrical 2-chlorothiophene-based analogs (<b>16</b>–<b>34</b>) containing variable alkyl and aryl substructures at the R<sup>2</sup> positions were developed in an attempt to improve their selectivity indices between efficacy against bacterial proliferation and cytotoxicity in human cell viability assays (structures of the different R<sup>2</sup> groups are presented in Tables S3A and S4A in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88668" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88668" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identifying Preliminary SAR of Previously Developed Pseudosymmetrical Compound <b>1</b> Analogs for Antibacterial Effects against the ESKAPE Pathogens</h3><div class="NLM_p">From our previous study exploring the antibacterial effects of our different GroEL/ES inhibitor scaffolds, we found that the parent hit, compound <b>1</b> (based on the benzimidazole core, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A), exhibited no antibacterial effects against any of the ESKAPE pathogens.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Therefore, before embarking on the synthesis of new compound <b>1</b> analogs, we tested our previously developed benzoxazole-based GroEL/ES inhibitors (compounds <b>2</b>–<b>14</b>), which have antitrypanosomal activities, for their antibacterial properties.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We expected that this initial evaluation would tell us whether or not this scaffold would be worthwhile to pursue for antibacterial development and, if so, allow us to identify which substituents, substitution patterns, and aryl substructures would provide the most potent antibacterial effects. We employed a standard antibacterial efficacy assay in liquid culture as previously reported, with one modification: the media was cation-adjusted by addition of 25 mg/L Ca<sup>2+</sup> and 12.5 mg/L Mg<sup>2+</sup> to better mimic the concentrations of these divalent cations in vivo, which can alter the antibacterial effects of some compounds (e.g., daptomycin is a more potent inhibitor when media is supplemented with additional Ca<sup>2+</sup> and Mg<sup>2+</sup> ions).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Briefly, bacteria were grown at 37 °C without shaking (stagnant assay) in media stamped with test compounds. Compounds were first tested at single concentrations of 100 μM, then in dose–response for those that exhibited >50% inhibition of bacterial proliferation (refer to Tables S1A and S2A in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a> for a tabulation of all EC<sub>50</sub> results). After 6–8 h (<i>S. aureus</i>, MRSA, <i>K. pneumonia</i>, <i>E. cloacae</i>, and <i>P. aeruginosa</i>) or 24 h (<i>E. faecium</i> and <i>A. baumannii</i>), the absorbance of each well was read at 600 nm to monitor turbidity from bacterial growth, from which EC<sub>50</sub> results were obtained by plotting the dose–response measurements and fitting data with nonlinear regression. A detailed protocol for this assay is listed in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p last">Preliminary results from these proliferation assays indicated that all but one compound (<b>2h-</b><i><b>p</b></i>) were inactive against the four Gram-negative KAPE bacterial species and only three were able to inhibit <i>E. faecium</i> (the <i>ortho-</i>, <i>meta-</i>, and <i>para-</i>substituted analogs of hydroxylated compound <b>2h</b>); however, several compounds were able to inhibit <i>S. aureus</i> and MRSA proliferation (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and Tables S1A and S2A in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>). Furthermore, these compounds were reasonably selective at killing MRSA in relation to their cytotoxicity CC<sub>50</sub> values obtained from liver (THLE 3) and kidney (HEK 293) cell viability assays, which were determined in the previous study targeting <i>T. brucei</i> parasites.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Preliminary SAR revealed that the sulfonamide end-capping thiophene and <i>para</i>-chlorophenyl groups gave the most potent and selective aryl and substituent/substitution patterns, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Based on these initial results, we decided to combine these two moieties, reasoning that inhibition properties of each could be additive, and developed two parallel series of 2-chlorothiophene-based asymmetrical analogs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C, analogs <b>15</b>–<b>34</b>) containing variable substructures at the R<sup>2</sup> positions.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis of the Two Parallel Series of 2-Chlorothiophene-Based Analogs</h3><div class="NLM_p">We rationalized that the previously developed pseudosymmetrical compound <b>1</b> analogs (i.e., with the same sulfonamide end-capping substructures on either side of the molecules) may not be truly optimized for binding to and inhibiting the chaperonin system. Thus, to better complement the envisioned asymmetric binding sites, we designed two parallel series of analogs with a variety of alkyl and aryl substructures at the R<sup>2</sup> positions on the right (R-series) and left (L-series) sides of the 2-phenylbenzoxazole scaffold, while keeping the 2-chlorothiophenesulfonamide moiety on the opposite sides (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). The same R<sup>2</sup>-groups were used to create the panel of 19 matched-pairs for the different R- and L-series analogs, with an additional pseudosymmetrical analog (<b>15</b>) that contained the 2-chlorothiophenesulfonamide on both sides (thus, a total of 39 new compounds were synthesized for evaluation). We expected that these matched pairs of analogs would help us to identify whether or not the directionality of the 2-phenylbenzoxazole core would significantly impact inhibitor effects in the various biochemical and cell-based assays they would be tested in. The general syntheses of these analogs are shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, with detailed procedures and compound characterizations presented in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>. Each of the R- and L-series of analogs were synthesized through 5-step linear protocols employing facile reactions. All final test compounds were characterized by <sup>1</sup>H NMR and LC-MS analyses for structural confirmation, and by two independent sets of HPLC conditions for purity identification (all were >95% pure under both conditions).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Syntheses of R- and L-Series Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: A. (a) CH<sub>2</sub>Cl<sub>2</sub>, rt (94%); (b) tin powder, 10% HCl in AcOH, rt (82%); (c) 4-nitrobenzaldehyde, NaHCO<sub>3</sub>, Na<sub>2</sub>SO<sub>4</sub>, THF, reflux to rt, then DDQ added (95%); (d) tin powder, 10% HCl in AcOH, rt (41%); (e) R<sup>2</sup>-SO<sub>2</sub>Cl, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, rt. B. (f) CH<sub>2</sub>Cl<sub>2</sub>, rt (98%); (g) LiOH·H<sub>2</sub>O, 1:1:1 THF:MeOH:H<sub>2</sub>O, rt (96%); (h) SOCl<sub>2</sub>, 60 °C for 1 h, concentrated, then 2-amino-4-nitrophenol and pyridine in CH<sub>2</sub>Cl<sub>2</sub>, reflux to rt, then TsOH·H<sub>2</sub>O, xylenes, reflux (45%); (i) tin powder, 10% HCl in AcOH, rt (96%); (j) R<sup>2</sup>-SO<sub>2</sub>Cl, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, rt.</p></p></figure><div class="NLM_p">For synthesizing the <b>R</b>-series analogs, 2-chlorothiophenesulfonyl chloride was first coupled to 4-amino-2-nitrophenol in dichloromethane using pyridine as base, giving <b>35</b>.<a onclick="showRef(event, 'ref26 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref28 ref29">(26,28,29)</a> Reduction of the nitro group was accomplished by reacting with tin powder in a 1:10 v/v mixture of hydrochloric acid in glacial acetic acid.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a> Next, an intermediate Schiff base was formed by refluxing <b>36</b> and <i>para</i>-nitrobenzaldehyde with sodium bicarbonate in anhydrous THF, which was then cooled and cyclized to benzoxazole <b>37</b> by addition of DDQ.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The nitro group was again reduced using tin powder in HCl/AcOH, affording intermediate <b>38</b>, from which the final bis-sulfonamido analogs (<b>15</b> and <b>16R</b>–<b>34R</b>) could be rapidly generated by coupling with a variety of differentially substituted sulfonyl chlorides (R<sup>2</sup>-SO<sub>2</sub>Cl).</div><div class="NLM_p last">For synthesizing the L-series analogs, 2-chlorothiophenesulfonyl chloride was first coupled to methyl-4-aminobenzoate in dichloromethane using pyridine as base, giving <b>39</b>. The ester was then hydrolyzed with LiOH in a mixture of H<sub>2</sub>O/MeOH/THF, affording acid <b>40</b>.<a onclick="showRef(event, 'ref28 ref30 ref32'); return false;" href="javascript:void(0);" class="ref ref28 ref30 ref32">(28,30,32)</a> The acid was then converted to the acyl chloride, concentrated, and coupled to 2-amino-4-nitrophenol in dichloromethane with pyridine as base, affording the intermediate amide, which was then cyclodehydrated to <b>41</b> by refluxing with toluenesulfonic acid in xylenes.<a onclick="showRef(event, 'ref29 ref33'); return false;" href="javascript:void(0);" class="ref ref29 ref33">(29,33)</a> The nitro group was subsequently reduced with tin powder in HCl/AcOH, affording intermediate <b>42</b>, from which the final bis-sulfonamido analogs (<b>16L</b>–<b>34L</b>) could be rapidly generated by coupling with a variety of differentially substituted sulfonyl chlorides (R<sup>2</sup>-SO<sub>2</sub>Cl).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Evaluating Compounds for Antibiotic Effects against the ESKAPE Bacteria and Inhibition of GroEL/ES-Mediated Folding of Substrate Proteins</h3><div class="NLM_p">Before proceeding with extensive biochemical and biophysical characterization of the new 2-chlorothiophene-based analogs, we first tested for their antibiotic effects on the ESKAPE bacteria using the proliferation assay described above (refer to Tables S3A and S4A in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a> for tabulations of all EC<sub>50</sub> results). Much like the initial testing with the pseudosymmetrical analogs, all but one of the 2-chlorothiophene analogs were inactive against the four Gram-negative KAPE bacterial species (<b>27R</b> had an EC<sub>50</sub> of 37 μM against <i>A. baumannii</i>). However, 14 of the 39 new analogs (36%) exhibited antibacterial effects against <i>E. faecium</i>, compared to 3 out of 50 (6%) of the pseudosymmetrical compounds. What was particularly impressive was that 27 out of the 39 new analogs (69%) inhibited <i>S. aureus</i> and MRSA, with greater overall potency, compared to the 20 out of 50 (40%) of the pseudosymmetrical compounds (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A presents a correlation plot of EC<sub>50</sub> values for each compound determined in these two assays). To compare assay results between the R- and L-series analogs, we analyzed the log(EC<sub>50</sub>) values of each data set using two-tailed, paired <i>t</i> tests (95% confidence level) and looked at differences between paired values (results are plotted in Figure S1 in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>). For the <i>E. faecium</i> EC<sub>50</sub> results, we did not see a statistical difference between results from the R- and L-series analogs (<a class="ref internalNav" href="#notes-3" aria-label="Figure S1A">Figure S1A</a>). However, there was a statistically significant difference between results of the two series for inhibiting <i>S. aureus</i> and MRSA proliferation, with the L-series generally providing more potent inhibitors of bacterial proliferation for each strain (<a class="ref internalNav" href="#notes-3" aria-label="Figures S1B and S1C">Figures S1B and S1C</a>). Although exceptions are evident for individual compounds, there was no statistical difference between inhibiting the drug susceptible <i>S. aureus</i> and MRSA strains when the data sets were analyzed for all 89 compounds as a whole (<a class="ref internalNav" href="#notes-3" aria-label="Figure S5A">Figure S5A</a>).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Correlation plots of IC<sub>50</sub> and EC<sub>50</sub> values for compounds tested in the respective GroEL/ES-dMDH or dRho refolding assays and bacterial proliferation assays. Each data point represents results for individual compounds tested in the respective assays (plotted from results presented in Tables S1–S4 in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>). Data plotted in the gray zones represent results beyond the assay detection limits (i.e., > 100 μM). A. Compounds inhibit the proliferation of <i>S. aureus</i> (drug susceptible) and MRSA (drug resistant) bacteria nearly equipotently. The L- and R-series asymmetric inhibitors are generally more potent than the previously developed pseudosymmetrical inhibitors (<b>1</b>–<b>14</b>). B. Compounds are on-target for inhibiting chaperonin-mediated substrate refolding as a correlation is evident between IC<sub>50</sub> values obtained from the GroEL/ES-dMDH and -dRho refolding assays, and compounds do not inhibit in both the native MDH and Rho enzymatic activity counter-screens. C. A general trend is observed between inhibiting the GroEL/ES chaperonin system and MRSA proliferation, supporting on-target effects in bacteria.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Having ascertained that the majority of the R- and L-series analogs were potent inhibitors of <i>S. aureus</i> and MRSA proliferation, we next evaluated their abilities to inhibit the biochemical function of the GroEL/ES chaperonin system. For this, we employed our previously reported assays that monitor GroEL/ES-mediated refolding of two denatured substrate enzymes, malate dehydrogenase (dMDH) and rhodanese (dRho).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Since these were coupled assays that monitor chaperonin-mediated refolding of substrates by virtue of the enzymatic activities of the refolded substrates, we further counter-screened against the native MDH and Rho enzymes to ensure that compounds were not simply false-positives of the reporter reactions. Detailed protocols for these assays and tabulation of all IC<sub>50</sub> results are presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p">Because the previously developed pseudosymmetrical analogs (<b>1</b>–<b>14</b>) were not tested in the GroEL/ES-dRho refolding assay nor the native rhodanese enzymatic reporter reaction counter screen, we also evaluated those compounds in these two assays. When looking at all 89 compounds as a whole, although a correlation is evident, we noticed a statistically significant difference between the GroEL/ES-dMDH and GroEL/ES-dRho refolding IC<sub>50</sub> results (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B and <a class="ref internalNav" href="#notes-3" aria-label="S5B">S5B</a>), where compounds were slightly more potent at inhibiting GroEL/ES-mediated refolding of denatured MDH. This difference does not appear to be a result of compounds preferentially inhibiting the native MDH reporter enzyme over rhodanese; while 11 analogs inhibited the native MDH enzymatic reporter reaction, five inhibited native rhodanese, and only to a minor extent. Since only one compound inhibits both native malate dehydrogenase and native rhodanese (<b>28R</b>), and only to a minor extent, this supports that compounds are on-target for inhibiting the GroEL/ES-mediated folding cycle.</div><div class="NLM_p">For the purposes of categorizing inhibitor potencies in the GroEL/ES-mediated refolding assays, we consider compounds with IC<sub>50</sub> values >100 μM to be inactive, 30–100 μM to be weak inhibitors, 10–30 μM moderate inhibitors, 1–10 μM potent inhibitors, and <1 μM very potent and acting near stoichiometrically since the concentration of GroEL tetradecamer is 50 nM during the refolding cycle (i.e., 700 nM of GroEL subunits). Upon further dissection of the GroEL/ES-dMDH refolding assay IC<sub>50</sub> results, we observed that the asymmetric R- and L-series analogs are generally more potent than the previously developed pseudosymmetric analogs: 28 out of 38 (74%) of the asymmetric R- and L-series analogs (<b>16</b>–<b>34</b>) have IC<sub>50</sub> values less than 10 μM, whereas only 16 out of 51 (31%) of the pseudosymmetric analogs (<b>1</b>–<b>15</b>) have IC<sub>50</sub> values less than 10 μM. These results are not entirely surprising since the R- and L-series analogs were hypothesized to better complement the envisioned asymmetric binding sites than the pseudosymmetric compounds. However, we caution on overinterpreting these results since the analog groupings did contain different alkyl and aryl substructures on the sulfonamide end-caps. When comparing the R- and L-series analogs with each other, though, we do not see any statistically significant differences in either the GroEL/ES-dMDH or GroEL/ES-dRho refolding assay results (<a class="ref internalNav" href="#notes-3" aria-label="Figure S2A and S2B">Figure S2A and S2B</a>), suggesting that the orientation of the 2-phenylbenzoxazole core scaffold does not play a significant factor in these compounds binding to and inhibiting the GroEL/ES chaperonin system.</div><div class="NLM_p last">When comparing IC<sub>50</sub> results for compounds tested in the biochemical GroEL/ES-dMDH refolding assay with EC<sub>50</sub> results for testing in the <i>S. aureus</i> proliferation assay (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C and <a class="ref internalNav" href="#notes-3" aria-label="Tables S3A-B and S4A-B">Tables S3A-B and S4A-B</a>), we note that no compounds are active against bacteria unless they are able to inhibit GroEL. In general, the more potent compounds are at inhibiting the GroEL/ES chaperonin system, the more potent their antibacterial effects, in particular for the L-series inhibitors. However, several exceptions are evident where potent GroEL/ES inhibitors are poorly effective, or ineffective, against bacteria, which may owe to poor cell wall permeability and/or efflux, as we had previously observed for other GroEL/ES inhibitor scaffolds.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Also, while correlations between biochemical IC<sub>50</sub> and cell-based EC<sub>50</sub> are only suggestive of mechanisms of action in cells, they are not confirmatory. Thus, whether compounds are “on-target” for GroEL within bacteria is still unknown and warrants further investigation. These studies are ongoing and will be reported in the future.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Characterizing GroEL-Inhibitor Binding Interactions Using Isothermal Titration Calorimetry</h3><div class="NLM_p">To further interrogate inhibitor mechanisms of action against GroEL, we used isothermal titration calorimetry (ITC) to identify the thermodynamic parameters, binding affinities, and binding stoichiometries of compounds <b>20R</b> and <b>20L</b>. While these two compounds were only moderately active at inhibiting <i>S. aureus</i> and MRSA proliferation, they were strong inhibitors of GroEL/ES-mediated folding functions and thus likely had high binding affinities. Furthermore, because they bore primary amines that would be charged under physiological conditions, they were much more soluble than other inhibitors and thus more amenable to ITC analysis, which requires high concentrations of both protein and ligands in matched aqueous buffers. We performed ITC analyses by titrating 400 μM GroEL (monomer concentration) into solutions containing either 50 μM <b>20R</b> or <b>20L</b> (detailed protocols are presented in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Two representative isotherms for the binding of <b>20R</b> and <b>20L</b> to GroEL are presented in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. After subtraction of background heats of mixing and dilution, plots of the integrated heats from compound binding fit well to a single-site binding model.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Averaged results for the various binding parameters (<i>K</i><sub>d</sub>, <i>n</i>, Δ<i>H</i>, Δ<i>S</i>, and Δ<i>G</i>) obtained from replicate analyses (six replicates for <b>20R</b> and five replicates for <b>20L</b>) are presented in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representative analyses of the binding of <b>20R</b> (panel A) and <b>20L</b> (panel B) to GroEL measured by isothermal titration calorimetry (ITC). Top panels show binding isotherms obtained by titrating GroEL (400 μM monomer concentration) into solutions of either <b>20R</b> or <b>20L</b> (50 μM) in the ITC cell. Lower panels show the integrated data (solid squares) fit to a single-site binding model (solid lines). The molar ratio refers to the binding stoichiometry of monomeric GroEL to molecules <b>20R</b> or <b>20L</b>. Average results for the various binding parameters (<i>K</i><sub>d</sub>, <i>n</i>, Δ<i>H</i>, Δ<i>S</i>, and Δ<i>G</i>) obtained from replicate analyses are presented in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Isothermal Titration Calorimetry (ITC) Results for Compounds <b>20R</b> and <b>20L</b><a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>20R</b></th><th class="colsep0 rowsep0" align="center"><b>20L</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left"><b>Assay IC<sub>50</sub> (μM)</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GroEL/ES-dMDH refolding</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GroEL/ES-dRho refolding</td><td class="colsep0 rowsep0" align="left">3.2</td><td class="colsep0 rowsep0" align="left">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left"><b>Stoichiometry <i>n</i></b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GroEL<sub>monomer</sub>:molecule</td><td class="colsep0 rowsep0" align="left">0.77 ± 0.16</td><td class="colsep0 rowsep0" align="left">0.6 ± 0.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">molecule:GroEL<sub>monomer</sub></td><td class="colsep0 rowsep0" align="left">1.30 ± 0.27</td><td class="colsep0 rowsep0" align="left">1.67 ± 0.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">molecule:GroEL<sub>oligomer</sub></td><td class="colsep0 rowsep0" align="left">18.2 ± 3.8</td><td class="colsep0 rowsep0" align="left">23.3 ± 5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left"><b>Thermodynamic parameters</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">log(<i>K</i><sub>d</sub>) (nM)</td><td class="colsep0 rowsep0" align="left">1.56 ± 0.20</td><td class="colsep0 rowsep0" align="left">2.18 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>d</sub> (nM)</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">150</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ<i>H</i> (kcal/mol)</td><td class="colsep0 rowsep0" align="left">–9.78 ± 0.80</td><td class="colsep0 rowsep0" align="left">–8.30 ± 0.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ<i>S</i> (kcal/mol·K)</td><td class="colsep0 rowsep0" align="left">0.001 ± 0.003</td><td class="colsep0 rowsep0" align="left">0.003 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i>Δ<i>S</i>(kcal/mol)</td><td class="colsep0 rowsep0" align="left">0.20 ± 0.82</td><td class="colsep0 rowsep0" align="left">0.84 ± 0.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ<i>G</i> (kcal/mol)</td><td class="colsep0 rowsep0" align="left">–9.98 ± 0.26</td><td class="colsep0 rowsep0" align="left">–9.14 ± 0.06</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last">Results are averaged from six replicate analyses for <b>20R</b> and five replicate analyses for <b>20L</b>.</p></div></div></div><div class="NLM_p last">From the averaged results, we found that <b>20R</b> had a <i>K</i><sub>d</sub> of 36 nM for binding to <i>E. coli</i> GroEL, while the <i>K</i><sub>d</sub> of <b>20L</b> was about 4-fold higher (150 nM), which corresponds reasonably well with the relative IC<sub>50</sub> values for inhibiting GroEL/ES-mediated refolding of dMDH and dRho; the IC<sub>50</sub> values for <b>20L</b> were typically 2- to 4-fold higher than those for <b>20R</b>. For both compounds, binding was predominantly enthalpically driven, with minor entropic contributions to affinity. What was particularly interesting from these analyses was the stoichiometry of binding of each compound to GroEL. Since GroEL consists of 14 identical subunits, we anticipated that compounds could bind with a stoichiometry of 14, or potentially 7 if there is negative cooperativity between the two GroEL rings (also assuming that for inhibitor binding there is no negative cooperativity between subunits within a ring). Since IC<sub>50</sub> values were previously found to correlate between GroEL/ES-mediated refolding and ATPase assays for this inhibitor scaffold, the most likely binding sites may be the ATP pockets, of which there are 1 per GroEL subunit (14 per oligomer).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, <b>20R</b> bound with a stoichiometry of roughly 18 molecules per GroEL tetradecamer and <b>20L</b> with 23 molecules, which could indicate more than one potential binding site per GroEL subunit and potentially an unknown site outside of the ATP pockets. We are currently pursuing X-ray crystallographic studies to identify specific inhibitor binding sites, which we will report on in future studies.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Counter-Screening Compounds for Inhibition of HSP60/10-Mediated Refolding of Denatured MDH and for Cytotoxicity in Human Cell Viability Assays</h3><div class="NLM_p">Having established that the R- and L-series analogs were potent GroEL/ES chaperonin system inhibitors with antistaphylococcal properties, we next investigated their selectivity profiles compared to the human HSP60/10 chaperonin system. For this, we evaluated compounds in our previously reported HSP60/10-dMDH refolding assay, which is analogous to that used for determining inhibition of bacterial GroEL/ES but alternatively employs the human chaperonin system. A detailed protocol and tabulation of all IC<sub>50</sub> results for this assay are presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>. We previously reported that the pseudosymmetrical analogs (<b>1</b>–<b>14</b>) were highly selective for inhibiting <i>E. coli</i> GroEL/ES over human HSP60/10 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a class="ref internalNav" href="#notes-3" aria-label="S3A">S3A</a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> However, for the asymmetric compounds developed in this study, selectivity for inhibiting GroEL/ES over HSP60/10 was statistically significant for the R-series analogs (<a class="ref internalNav" href="#notes-3" aria-label="Figure S3C">Figure S3C</a>) but was lost for the L-series analogs (<a class="ref internalNav" href="#notes-3" aria-label="Figure S3B">Figure S3B</a>). Correspondingly, differences in IC<sub>50</sub> values between the R- and L-series inhibitors in the HSP60/10-dMDH refolding assay were statistically significant (<a class="ref internalNav" href="#notes-3" aria-label="Figure S4A">Figure S4A</a>), which suggests that directionality of the 2-phenylbenzoxazole core does have an effect on compounds binding to and inhibiting the human chaperonin system, unlike the situation for <i>E. coli</i> GroEL/ES.</div><div class="NLM_p">We next evaluated the cytotoxicity of our compounds to human liver (THLE 3) and kidney (HEK 293) cell lines in a well-established, 48 h, Alamar Blue-based cell viability assay. Detailed protocols and tabulation of all cytotoxicity CC<sub>50</sub> results for these assays are presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>. Intriguingly, even though the L-series analogs are more potent HSP60/10 inhibitors than the R-series, we observed no statistically significant differences between the L- and R-series CC<sub>50</sub> values in the HEK 293 kidney cell viability assay (<a class="ref internalNav" href="#notes-3" aria-label="Figure S4B">Figure S4B</a>). Furthermore, we observed an opposing trend where the L-series was less cytotoxic to the THLE 3 liver cells compared to the R-series analogs, although the differences were small (<a class="ref internalNav" href="#notes-3" aria-label="Figure S4B,C">Figure S4B,C</a>). These results suggest that the compounds may be unable to penetrate the mitochondrial matrix to engage with the HSP60/10 chaperonin system and instead may be interacting with other targets in the cytosol or other subcellular compartments; however, we have yet to confirm this hypothesis. What is particularly promising with these results is that the cytotoxicity of lead analogs has not concomitantly increased as with antibacterial potency. Compared to the previously developed pseudosymmetrical analogs, several lead asymmetrical analogs have appreciably higher selectivity indices (SI) for inhibiting MRSA proliferation over cytotoxicity in the liver cell viability assay (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). The biochemical and cell-based IC<sub>50</sub>, EC<sub>50</sub>, and CC<sub>50</sub> results for the top four lead inhibitors of the pseudosymmetrical, R-series, and L-series analogs are presented in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, ranked based on their selectivity indices. Notably, compounds <b>24L</b> and <b>25L</b> have selectivity indices of 31 and 46, respectively.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation plots of IC<sub>50</sub>, EC<sub>50</sub>, and CC<sub>50</sub> values for compounds tested in the respective GroEL/ES-dMDH and HSP60/10-dMDH refolding assays, liver cell cytotoxicity assay, and MRSA proliferation assay. Each data point represents results for individual compounds tested in the respective assays (plotted from results presented in Tables S1–S4 in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>). Data plotted in the gray zones represent results beyond the assay detection limits (i.e., >100 μM). A. The R-series and previously developed analogs selectively inhibit the <i>E. coli</i> GroEL/ES over the human HSP60/10 chaperonin system but the L-series analogs are not selective. B. No correlation is evident between cytotoxicity of compounds in the liver cell viability assay and the HSP60/10-dMDH refolding assay, suggesting cytotoxicity may be from off-target effects in cells. C. Many compounds are able to selectively inhibit MRSA proliferation with moderate to low cytotoxicity to human liver cells. Results are similar for compounds tested in the kidney cell viability assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biochemical and Cell-Based IC<sub>50</sub>, EC<sub>50</sub>, and CC<sub>50</sub> Results for the Top Four Lead Inhibitors Based on Selectivity Index (SI) for Inhibiting MRSA Proliferation over Cytotoxicity in the Liver Cell Viability Assay</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0008.gif" alt="" id="fx1" /></img><div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Investigating the Potential for MRSA to Gain Resistance to Lead Inhibitors</h3><div class="NLM_p">While selectivity indices (and therapeutic windows with regards to in vivo evaluation) are important criteria to consider for antibacterial development, it is also important to determine whether or not bacteria will be able to rapidly develop resistance to lead candidates, especially if they function through new mechanisms of action. To gauge the efficacy of our GroEL/ES inhibitors for eluding acute antibacterial resistance mechanisms, we performed a stepwise, liquid culture resistance assay by consecutively passaging the MRSA strain (ATCC BAA-44) for 12 cycles over 12 consecutive days in the presence of serially diluted inhibitors <b>20R</b>, <b>28R</b>, and vancomycin for comparison (a detailed protocol is presented in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>).<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> At the end of each cycle (i.e., each 24 h passage), EC<sub>50</sub> values were determined for the test compounds, with the assumption that EC<sub>50</sub> values would increase over time if MRSA was able to generate acute resistance. A plot of test compound EC<sub>50</sub> values over time is presented in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A. Despite the fact that <b>28R</b> was a potent MRSA growth inhibitor (initial EC<sub>50</sub> value of 1.7 μM), the bacteria rapidly developed resistance to compound concentrations in excess of 100 μM. However, after culturing in the absence of inhibitors for 2 days (24 h on agar and another 24 h in liquid media), bacteria regained sensitivity to <b>28R</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B), suggesting resistance is reversible, potentially from up-regulation of efflux pumps. Most importantly, we found that the parent MRSA strain was unable to generate resistance against <b>20R</b> over the 12 day passage, even though this analog was only a moderate inhibitor of MRSA proliferation (EC<sub>50</sub> = 19 μM). These results suggest that pan-resistance to these GroEL inhibitor analogs may be difficult to develop, which supports the continued optimization of lead antibacterial candidates based on this molecular scaffold.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Exploring adaptive tolerance by MRSA bacteria to analogs <b>20R</b>, <b>28R</b>, and vancomycin. A. MRSA was serially passaged for 12 days in fresh liquid media supplemented with the selected compounds. Average EC<sub>50</sub> values of compounds tested after each 24 h passage are plotted from triplicate analyses. MRSA rapidly evolved resistance to <b>28R</b> but retained sensitivity to <b>20R</b> and vancomycin throughout the 12 day experiment. Data plotted in the gray zones represent EC<sub>50</sub> results beyond the assay detection limits (i.e., >100 μM). B. After reculturing the <b>28R</b>-resistant MRSA strain for 2 days in the absence of inhibitors (24 h on agar, then another 24 h in liquid medium), the “resistant” MRSA strain regained sensitivity to <b>28R</b> (average % growth inhibition results (±SD) are plotted from 4 replicate analyses).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32073" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32073" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">From exploratory screening of the antibacterial effects of our previously reported pseudosymmetrical compound <b>1</b> analogs,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> we were able to develop two parallel R- and L-series of 2-chlorothiophene-based asymmetrical analogs with significantly improved antibacterial efficacy profiles against MRSA. While there were no statistical differences between either the R- or L-series analogs for inhibiting in the GroEL/ES-mediated refolding assays, the L-series analogs were found to be more potent at inhibiting <i>S. aureus</i> and MRSA proliferation, i.e., preferred from an antibiotic development perspective (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). While a general trend is observed between inhibitor potencies in the biochemical GroEL/ES-dMDH refolding assay and the MRSA proliferation assay (in particular for the L-series analogs), further studies are needed to conclusively determine whether or not inhibitors are on-target in bacteria. Using ITC to characterize the binding interactions between GroEL and inhibitors <b>20R</b> and <b>20L</b>, we found that inhibitor binding was predominantly enthalpically driven. Furthermore, we observed that >14 molecules were bound per GroEL tetradecamer (i.e., more than one molecule bound per GroEL subunit), suggesting that inhibitors could be binding to unknown sites outside of the ATP pockets. Current studies are underway to identify and characterize these putative binding sites so that more rigorous structure-based optimization of lead candidates can be conducted. Even though the R-series analogs were overall less potent at inhibiting HSP60/10, the L-series was less cytotoxic against the human liver cells tested and is likely preferential for further antibacterial optimization going forward. However, before pursuing in vivo development of this series, further medicinal chemistry optimization is warranted to increase the selectivity indices for killing MRSA over cytotoxicity to human cells.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Comparison of the L- and R-Series Analogs for Which Overall Scaffolds Exhibited Preferred Inhibition Profiles in the Various Biochemical and Cell-Based Assays<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">assay inhibition</th><th class="colsep0 rowsep0" align="center">L-series</th><th class="colsep0 rowsep0" align="center">R-series</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S. aureus</i> growth</td><td class="colsep0 rowsep0" align="left">preferred</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MRSA growth</td><td class="colsep0 rowsep0" align="left">preferred</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GroEL/ES-dMDH refolding</td><td class="colsep0 rowsep0" colspan="2" align="center">no statistical difference</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GroEL/ES-dRho refolding</td><td class="colsep0 rowsep0" colspan="2" align="center">no statistical difference</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HSP60/10-dMDH refolding</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">preferred</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">liver cell viability</td><td class="colsep0 rowsep0" align="left">preferred</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kidney cell viability</td><td class="colsep0 rowsep0" colspan="2" align="center">no statistical difference</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup><sup>a</sup></sup><p class="last">Lead antibacterial candidates are preferred to potently inhibit GroEL/ES and bacteria and not inhibit human HSP60/10 and liver/kidney cell viability. The log(IC<sub>50</sub>), log(EC<sub>50</sub>), or log(CC<sub>50</sub>) values for the 20 matched L- and R-series analogs were analyzed by two-tailed, paired t-tests with 95% confidence intervals. Comparative plots are presented in Figures S1–S5 in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</p></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26173" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26173" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Synthetic Methods</h3><div class="NLM_p last">Unless otherwise stated, all chemicals were purchased from commercial suppliers and used without further purification. Reaction progress was monitored by thin-layer chromatography on silica gel 60 F254 coated glass plates (EM Sciences). Flash chromatography was performed using a Biotage Isolera One flash chromatography system and eluting through Biotage KP-Sil Zip or Snap silica gel columns for normal-phase separations (hexanes:EtOAc gradients), or Snap KP-C18-HS columns for reverse-phase separations (H<sub>2</sub>O:MeOH gradients). Reverse-phase high-performance liquid chromatography (RP-HPLC) was performed using a Waters 1525 binary pump, 2489 tunable UV/vis detector (254 and 280 nm detection), and 2707 autosampler. For preparatory HPLC purification, samples were chromatographically separated using a Waters XSelect CSH C18 OBD prep column (part number 186005422, 130 Å pore size, 5 μm particle size, 19 × 150 mm), eluting with a H<sub>2</sub>O:CH<sub>3</sub>CN gradient solvent system. Linear gradients were run from either 100:0, 80:20, or 60:40 A:B to 0:100 A:B (A = 95:5 H<sub>2</sub>O:CH<sub>3</sub>CN, 0.05% TFA; B = 5:95 H<sub>2</sub>O:CH<sub>3</sub>CN, 0.05% TFA). Products from normal-phase separations were concentrated directly, and reverse-phase separations were concentrated, diluted with H<sub>2</sub>O, frozen, and lyophilized. For primary compound purity analyses (HPLC-1), samples were chromatographically separated using a Waters XSelect CSH C18 column (part number 186005282, 130 Å pore size, 5 μm particle size, 3.0 × 150 mm), eluting with the above H<sub>2</sub>O:CH<sub>3</sub>CN gradient solvent systems. For secondary purity analyses (HPLC-2) of final test compounds, samples were chromatographically separated using a Waters XBridge C18 column (either part number 186003027, 130 Å pore size, 3.5 μm particle size, 3.0 × 100 mm, or part number 186003132, 130 Å pore size, 5.0 μm particle size, 3.0 × 100 mm), eluting with a H<sub>2</sub>O:MeOH gradient solvent system. Linear gradients were run from either 100:0, 80:20, 60:40, or 20:80 A:B to 0:100 A:B (A = 95:5 H<sub>2</sub>O:MeOH, 0.05% TFA; B = 5:95 H<sub>2</sub>O:MeOH, 0.05% TFA). Test compounds were found to be >95% in purity from both RP-HPLC analyses. Mass spectrometry data were collected using either an Agilent analytical LC-MS at the IU Chemical Genomics Core Facility (CGCF) or a Thermo-Finnigan LTQ LC-MS in-lab. <sup>1</sup>H NMR spectra were recorded on either Bruker 300 or 500 MHz spectrometers. Chemical shifts are reported in parts per million and calibrated to the <i>d</i><sub>6</sub>-DMSO solvent peaks at 2.50 ppm. We previously synthesized compounds <b>1</b>–<b>14</b> (including <b>2a-m-</b><i><b>o</b></i><b>/</b><i><b>m</b></i><b>/</b><i><b>p</b></i>)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and resynthesized where necessary due to stock depletion. Synthesis and characterization of intermediates <b>35</b>–<b>42</b> are presented below. General sulfonamide coupling steps are presented for analogs <b>15</b> and <b>16R–34R</b> and <b>16L–34L</b> below, with compound characterizations for each analog presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> 5-Chloro-<i>N</i>-(4-hydroxy-3-nitrophenyl) thiophene-2-sulfonamide (<b>35</b>)</h3><div class="NLM_p last">To a stirring mixture of 4-amino-2-nitrophenol (5.34 g, 24.6 mmol) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (50 mL) was added 5-chlorothiophene-2-sulfonyl chloride (4.21 g, 27.3 mmol) and pyridine (2.40 mL, 29.4 mmol). The reaction was allowed to stir at room temperature for 18 h and was then diluted with hexanes, and the precipitate was filtered, rinsed with 1 M HCl and water, collected, and dried to afford <b>35</b> as a reddish-brown solid (7.70 g, 94% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 10.98 (br s, 1H), 10.62 (br s, 1H), 7.60 (d, <i>J</i> = 2.7 Hz, 1H), 7.39 (d, <i>J</i> = 4.1 Hz, 1H), 7.30 (dd, <i>J</i> = 8.9, 2.7 Hz, 1H), 7.22 (d, <i>J</i> = 4.1 Hz, 1H), 7.10 (d, <i>J</i> = 8.9 Hz, 1H); MS (ESI) C<sub>10</sub>H<sub>6</sub>ClN<sub>2</sub>O<sub>5</sub>S<sub>2</sub> [M-H]<sup>−</sup><i>m</i>/<i>z</i> expected = 332.9, observed = 332.8; HPLC-1 = >99% (RT = 8.1 min).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> <i>N</i>-(3-Amino-4-hydroxyphenyl)-5-chlorothiophene-2-sulfonamide (<b>36</b>)</h3><div class="NLM_p last">Tin powder (8.73 g, 73.6 mmol) was added slowly to a stirring mixture of <b>35</b> (8.11 g, 24.2 mmol) in a 1:10 mixture of HCl:AcOH (24 mL). The reaction was allowed to stir at RT for 18 h, then diluted with EtOAc and H<sub>2</sub>O, neutralized with NaHCO<sub>3</sub>, and filtered. The filtrate was extracted with EtOAc, and the organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was diluted in a 50% mixture of DCM in hexanes and the precipitate was filtered, rinsed with hexanes, collected, and dried to afford <b>36</b> as a brown powder (6.08 g, 82% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 9.81 (br s, 1H), 8.99 (br s, 1H) 7.28 (d, <i>J</i> = 4.1 Hz, 1H), 7.18 (d, <i>J</i> = 4.1 Hz, 1H), 6.49 (d, <i>J</i> = 8.3 Hz, 1H), 6.41 (d, <i>J</i> = 2.5 Hz, 1H), 6.13 (dd, <i>J</i> = 8.3, 2.6 Hz, 1H), 4.64 (br s, 2H); MS (ESI) C<sub>10</sub>H<sub>10</sub>ClN<sub>2</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> expected = 305.0, observed = 304.9; HPLC-1 = 97% (RT = 6.0 min).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 5-Chloro-<i>N</i>-(2-(4-nitrophenyl)benzo[d]oxazol-5-yl)thiophene-2-sulfonamide (<b>37</b>)</h3><div class="NLM_p last">Compound <b>36</b> (2.94 g, 9.65 mmol), 4-nitrobenzaldehyde (2.03 g, 13.4 mmol), NaHCO<sub>3</sub> (2.08 g, 24.8 mmol), and Na<sub>2</sub>SO<sub>4</sub> (3.35 g) were stirred in THF (40 mL) for 4 h at reflux (under Ar) and then cooled to RT. DDQ (2.85 g, 12.6 mmol) was then add portion-wise, and the reaction was left to stir for 2 h and then filtered. The filtrate was extracted into EtOAc and rinsed with saturated NaHCO<sub>3</sub>, 1 M HCl, and brine. The organics layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was diluted in a 25% mixture of DCM in hexanes, and the precipitate was filtered, rinsed with hexanes, collected, and dried to afford <b>37</b> as a brown powder (4.01 g, 95% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 10.74 (s, 1H), 8.37–8.48 (m, 4H), 7.82 (d, <i>J</i> = 8.8 Hz, 1H), 7.60 (d, <i>J</i> = 1.9 Hz, 1H), 7.43 (d, <i>J</i> = 4.0 Hz, 1H), 7.25 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.19 (d, <i>J</i> = 4.1 Hz, 1H); MS (ESI) C<sub>17</sub>H<sub>9</sub>ClN<sub>3</sub>O<sub>5</sub>S<sub>2</sub> [M-H]<sup>−</sup><i>m</i>/<i>z</i> expected = 434.0, observed = 433.8; HPLC-1 = 91% (RT = 7.7 min).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>N</i>-(2-(4-Aminophenyl)benzo[d]oxazol-5-yl)-5-chlorothiophene-2-sulfonamide (<b>38</b>)</h3><div class="NLM_p last">Tin powder (2.74 g, mmol) was added slowly to a stirring mixture of <b>37</b> (2.43 g, 5.58 mmol) in a 1:10 mixture of HCl:AcOH (15 mL). The reaction was allowed to stir at RT for 18 h, then diluted with EtOAc and H<sub>2</sub>O, neutralized with NaHCO<sub>3</sub>, and filtered. The filtrate was extracted with EtOAc, and the organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was chromatographed over silica (hexanes:EtOAc gradient) and concentrated to afford <b>38</b> as an orange solid (936 mg, 41% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 10.54 (s, 1H), 7.78–7.86 (m, 2H), 7.60 (d, J = 8.7 Hz, 1H), 7.35–7.40 (m, 2H), 7.18 (d, J = 4.1 Hz, 1H), 7.04 (dd, <i>J</i> = 8.6, 2.2 Hz, 1H), 6.64–6.71 (m, 2H), 6.04 (br s, 2H); MS (ESI) C<sub>17</sub>H<sub>13</sub>ClN<sub>3</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> expected = 406.0, observed = 405.9; HPLC-1 = 97% (RT = 7.6 min).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Sulfonamidation Procedure for the Synthesis of <b>16R–34R</b></h3><div class="NLM_p last">To a stirring mixture of compound <b>38</b> (1 equiv) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (5 mL) was added the respective R<sup>2</sup> sulfonyl chloride (1.2 equiv) followed by anhydrous pyridine (1.2 equiv). The reaction was allowed to stir at room temperature for 18 h and was then chromatographed over silica (hexanes:EtOAc gradient) and concentrated. If necessary, the product was further purified by preparatory RP-HPLC (H<sub>2</sub>O:CH<sub>3</sub>CN gradient), concentrated, and lyophilized. Refer to the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a> for individual compound characterization data.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Methyl 4-((5-chlorothiophene)-2-sulfonamido)benzoate (<b>39</b>)</h3><div class="NLM_p last">To a stirring mixture of methyl-4-aminobenzoate (2.76 g, 18.3 mmol) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (50 mL) was added 5-chlorothiophene-2-sulfonyl chloride (4.78 g, 22.0 mmol) and pyridine (1.80 mL, 22.1 mmol). The reaction was allowed to stir at room temperature for 18 h and was then diluted with hexanes and acidified with 1 M HCl. The precipitate was then filtered, rinsed with 1 M HCl and water, collected, and dried to afford <b>39</b> as a pinkish-orange solid (5.94 g, 98% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.16 (br s, 1H), 7.86–7.93 (m, 2H), 7.56 (d, <i>J</i> = 4.1 Hz, 1H), 7.25–7.31 (m, 2H), 7.22 (d, <i>J</i> = 4.1 Hz, 1H), 3.80 (s, 3H); HPLC-1 = 99% (RT = 6.2 min).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4-((5-Chlorothiophene)-2-sulfonamido)benzoic acid (<b>40</b>)</h3><div class="NLM_p last">LiOH·H<sub>2</sub>O (7.02 g, 167 mmol) was added to a stirring mixture of <b>39</b> (5.67 g, 17.1 mmol) in THF (20 mL), MeOH (20 mL), and H<sub>2</sub>O (20 mL). The reaction was allowed to stir at room temperature for 2 days and was then diluted with 1 M HCl. The precipitate was filtered, washed with H<sub>2</sub>O, collected, and dried to afford <b>40</b> as an off-white solid (5.20 g, 96% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 12.85 (br s, 1H), 11.09 (br s, 1H), 7.83–7.90 (m, 2H), 7.54 (d, <i>J</i> = 4.1 Hz, 1H), 7.19–7.28 (m, 3H); HPLC-1 = >99% (RT = 6.7 min).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 5-Chloro-<i>N</i>-(4-(5-nitrobenzo[d]oxazol-2-yl)phenyl)thiophene-2-sulfonamide (<b>41</b>)</h3><div class="NLM_p last">Compound <b>40</b> (2.96 g, 9.31 mmol) was stirred in SOCl<sub>2</sub> (10 mL) for 6 h at 60 °C and then was concentrated to a solid. This was refluxed with 2-amino-4-nitrophenol (2.96 g, 19.2 mmol) and pyridine (1.50 mL, 18.4 mmol) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (50 mL) for 6 h and then stirred at RT for 3 days. The reaction was then extracted into EtOAc and rinsed with 1 M HCl and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. This amide intermediate was then refluxed with TsOH ·H<sub>2</sub>O (3.61 g, 19.0 mmol) in xylenes (50 mL) using a Dean–Stark apparatus to remove the residual H<sub>2</sub>O. After 18 h, the reaction was cooled, the xylenes were decanted off, the sludge was extracted with EtOAc, and the combined organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. Flash chromatographic purification over silica (hexanes:EtOAc gradient) afforded <b>41</b> as a peach solid (1.81 g, 45% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.25 (br s, 1H), 8.64 (br s, 1H), 8.32 (d, <i>J</i> = 8.8 Hz, 1H), 8.18 (d, <i>J</i> = 8.1 Hz, 2H), 8.01 (d, <i>J</i> = 8.7 Hz, 1H), 7.60 (br s, 1H), 7.41 (d, <i>J</i> = 8.2 Hz, 2H), 7.24 (br s, 1H); MS (ESI) C<sub>17</sub>H<sub>11</sub>ClN<sub>3</sub>O<sub>5</sub>S<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> expected = 436.0, observed = 436.1; HPLC-1 = 88% (RT = 5.5 min).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>N</i>-(4-(5-Aminobenzo[d]oxazol-2-yl)phenyl)-5-chlorothiophene-2-sulfonamide (<b>42</b>)</h3><div class="NLM_p last">Tin powder (1.53 g, 12.9 mmol) was added slowly to a stirring mixture of <b>41</b> (1.78 g, 4.08 mmol) in a 1:7 mixture of HCl:AcOH (8 mL). The reaction was stirred and heated at 60 °C for 3 h, then cooled to RT, diluted with EtOAc and H<sub>2</sub>O, neutralized with NaHCO<sub>3</sub>, and filtered. The filtrate was extracted with EtOAc, and the organics were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The residue was diluted in a 10% mixture of DCM in hexanes, and the precipitate was filtered, rinsed with hexanes, collected, and dried to afford <b>42</b> as a pale-yellow powder (1.59 g, 96% yield). <sup>1</sup>H NMR (300 MHz, <i>d</i><sub>6</sub>-DMSO) δ 8.01–8.09 (m, 2H), 7.55 (d, <i>J</i> = 4.1 Hz, 1H), 7.30–7.41 (m, 3H), 7.22 (d, <i>J</i> = 4.1 Hz, 1H), 6.83 (d, <i>J</i> = 2.0 Hz, 1H), 6.64 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H); MS (ESI) C<sub>17</sub>H<sub>13</sub>ClN<sub>3</sub>O<sub>3</sub>S<sub>2</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> expected = 406.0, observed = 406.1; HPLC-1 = 86% (RT = 6.8 min).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Sulfonamidation Procedure for the Synthesis of <b>16L–34L</b></h3><div class="NLM_p last">To a stirring mixture of compound <b>42</b> (1 equiv) in anhydrous CH<sub>2</sub>Cl<sub>2</sub> (5 mL) was added the respective R<sup>2</sup> sulfonyl chloride (1.2 equiv) followed by anhydrous pyridine (1.2 equiv). The reaction was allowed to stir at room temperature for 18 h and was then chromatographed over silica (hexanes:EtOAc gradient) and concentrated. If necessary, the product was further purified by preparatory RP-HPLC (H<sub>2</sub>O:CH<sub>3</sub>CN gradient), concentrated, and lyophilized. Refer to the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a> for individual compound synthesis and characterization data.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Cell Information for Compound Evaluation</h3><div class="NLM_p last">The ESKAPE bacteria were purchased from the American Type Culture Collection (ATCC): <i>E. faecium</i> (Orla-Jensen) Schleifer and Kilpper-Balz strain NCTC 7171 (ATCC 19434); <i>S. aureus</i> subsp. <i>aureus</i> Rosenbach strain Seattle 1945 (ATCC 25923); Multidrug resistant <i>S. aureus</i> (MRSA) subsp. <i>aureus</i> Rosenbach strain HPV107 (ATCC BAA-44); <i>K. pneumonia</i>, subsp. <i>pneumoniae</i> (Schroeter) Trevisan strain NCTC 9633 (ATCC 13883); <i>A. baumannii</i> Bouvet and Grimont strain 2208 (ATCC 19606); <i>P. aeruginosa</i> (Schroeter) Migula strain NCTC 10332 (ATCC 10145); <i>E. cloacae</i>, subsp. <i>cloacae</i> (Jordan) Hormaeche and Edwards strain CDC 442-68 (ATCC 13047). HEK 293 kidney cells (ATCC CRL-1573) and THLE-3 liver cells (ATCC CRL-11233) were used for compound toxicity assays.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Evaluation of Compounds for Inhibition of Bacterial Cell Proliferation</h3><div class="NLM_p last">All compounds were evaluated for inhibiting the proliferation of each of the ESKAPE bacteria as per previously reported procedures with one minor modification: Liquid growth media was cation-adjusted by addition of 25 mg/L Ca<sup>2+</sup> and 12.5 mg/L Mg<sup>2+</sup> to reflect the free concentrations of these divalent cations in vivo.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Detailed protocols for bacterial growth assays are presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Protein Expression and Purification</h3><div class="NLM_p last"><i>E. coli</i> GroEL and GroES, and human HSP60 and HSP10, were expressed and purified as previously reported.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Detailed protocols for these protein purifications are presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Evaluation of Compounds in GroEL/ES and HSP60/10-Mediated dMDH and dRho Refolding Assays</h3><div class="NLM_p last">All compounds were evaluated for inhibiting <i>E. coli</i> GroEL/ES and human HSP60/10-mediated refolding of the denatured MDH and denatured Rho reporter enzymes as per previously reported procedures.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Detailed protocols for these assays are presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Counter-Screening Compounds for Inhibition of Native MDH and Rho Enzymatic Activity</h3><div class="NLM_p last">All compounds were counter-screened for inhibiting the enzymatic activity of the native MDH and native Rho reporter enzymes as per previously reported procedures.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Detailed protocols for the assays are presented in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Evaluation of Inhibitors <b>20R</b> and <b>20L</b> Binding to GroEL using Isothermal Titration Calorimetry (ITC)</h3><div class="NLM_p last">All ITC experiments were performed on a MicroCal VP-ITC system. To minimize background heats of dilution, matched solutions of GroEL and compounds <b>20R</b> or <b>20L</b> were prepared in buffer containing 20 mM HEPES, pH 7.4, 50 mM KCl, 10 mM MgCl<sub>2</sub>, and 1% DMSO. All buffers were freshly prepared from stock solutions on the day of use and degassed under vacuum. For each analytical run, a solution of 400 μM GroEL (monomer concentration) in the syringe was titrated into 50 μM solutions of either compound <b>20R</b> or <b>20L</b> in the sample cell (equilibrated at 20 °C for 300 s): the first injection was 0.4 μL (discarded during analysis) and subsequent injections (2–20) were each 2.0 μL. Each injection was made over 4 s durations with 3 min intervals between subsequent injections. The reference power of the experiment was set to 6 μcal/s and a 5 s filter period. Sample stirring was set at 600 rpm for all measurements. Six independent replicates were conducted to analyze compound <b>20R</b> binding and five replicates for <b>20L</b>. Raw thermograms from each replicate were analyzed independently using the ORIGIN instrument software (MicroCal Inc., version 7.0). In each experiment, the upper baseline was collected after the binding reaction was saturated, and the terminal 7–8 points in the linear region were fit to a straight line and subtracted from the entire data set to remove contributions from background heats of mixing and dilution. The Δ<i>H</i> was obtained by nonlinear least-squares fitting of the plot of Δ<i>H</i> (mol of injectant)<sup>−1</sup> versus molar ratio using a single site binding model.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The Δ<i>H</i>, Δ<i>S</i>, <i>K</i><sub>d</sub> (<i>K</i><sub>A</sub> = 1/<i>K</i><sub>d</sub>), and binding stoichiometries (<i>n</i>) in each titration were obtained, and Δ<i>G</i> was calculated using the Gibbs equation at a temperature of 20 °C (293.15 K). Standard deviations were determined based on results from each of the replicate analyses.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Evaluation of Compound Effects on HEK 293 and THLE-3 Cell Viability</h3><div class="NLM_p last">Evaluation of compound cytotoxicities to HEK 293 kidney and THLE-3 liver cells were performed using Alamar Blue-based viability assays. HEK 293 cells were maintained in MEM medium (Corning Cellgro, 10-009 CV) supplemented with 10% FBS (Sigma, F2242). THLE-3 cells were maintained in Clonetics BEBM medium (Lonza, CC-3171) supplemented with the BEGM bullet kit (Lonza, CC-3170) and 10% FBS. All assays were carried out in 384-well plates (BRAND cell culture grade plates, 781980). Cells at 80% confluence were harvested and diluted in growth medium, then 45 μL of the HEK 293 cells (1500 cells/well) or THLE-3 cells (1500 cells/well) were dispensed per well, and plates were sealed with “Breathe Easy” oxygen permeable membranes (Diversified Biotech) and incubated at 37 °C, 5% CO<sub>2</sub>, for 24 h. The following day, 1 μL of the compound stocks (10 mM to 4.6 μM, 3-fold dilutions in DMSO) was prediluted by pin-transfer into 25 μL of the relevant growth mediums. Then, 15 μL aliquots of the diluted compounds were added to the cell assay plates to give inhibitor concentration ranges of 100 μM to 46 nM during the assay (final DMSO concentration of 0.1% was maintained during the assay). Plates were sealed with “Breathe Easy” oxygen permeable membranes and incubated for an additional 48 h at 37 °C and 5% CO<sub>2</sub>. The Alamar Blue reporter reagents were then added to a final concentration of 10%, the plates incubated at 37 °C and 5% CO<sub>2</sub>, and sample fluorescence (535 nm excitation, 590 nm emission) was read using a Molecular Devices FlexStation II 384-well plate reader (readings taken between 4 and 24 h of incubation so as to achieve signals in the 30–60% range for conversion of resazurin to resorufin). Cell viability was calculated as per vendor instructions (Thermo Fisher - Alamar Blue cell viability assay manual). Cytotoxicity CC<sub>50</sub> values for the test compounds were obtained by plotting the % resazurin reduction results in GraphPad Prism 6 and analyzing by nonlinear regression using the log(inhibitor) vs response (variable slope) equation. Results presented represent the averages of CC<sub>50</sub> values obtained from at least triplicate experiments.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Evaluation of MRSA Resistance Generation against Lead Inhibitors</h3><div class="NLM_p last">To identify potential resistance toward inhibitors, a liquid culture, 12-day serial passage assay was employed as per procedures reported by Kim et al., using the MRSA strain (ATCC BAA-44).<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> Briefly, MRSA bacteria were streaked onto a Tryptic Soy agar plate and grown overnight at 37 °C. A fresh aliquot of Tryptic Soy Broth (TSB) was inoculated with a single bacterial colony, and the cultures were grown overnight at 37 °C with shaking (250 rpm). The overnight culture was then subcultured (1:5 dilution) into a fresh aliquot of media and grown at 37 °C for 1 h with shaking and then diluted into fresh media to achieve a final OD<sub>600</sub> reading of 0.01. Aliquots of the diluted culture (200 μL) were dispensed to 96 well plates along with addition of 2 μL of test compounds in DMSO (<b>20R</b>, <b>28R</b>, and vancomycin as a control). The inhibitor concentration range during the resistance assay was 100 μM to 48.8 nM (2-fold dilution series). Plates were sealed with “Breathe Easy” oxygen permeable membranes (Diversified Biotech) and left to incubate at 37 °C without shaking (stagnant assay). OD<sub>600</sub> readings were taken at the 24 h time point to monitor for bacterial growth. A second set of baseline control plates were prepared analogously, without any bacteria added, to correct for possible compound absorbance and/or precipitation, as well as plate and media baseline effects. For inoculations on subsequent days, bacteria from the wells with the highest drug concentration where the OD<sub>600</sub> was >0.2 were diluted with fresh media to OD<sub>600</sub> of 0.01 and dispensed into a new 96-well plate. Test compounds were added, and the bacteria were propagated again as described above. This procedure was repeated each day for a total of 12 days to observe changes in EC<sub>50</sub> values over each passage. EC<sub>50</sub> values for the test compounds were obtained by plotting the OD<sub>600</sub> results from each passage in GraphPad Prism 6 and analyzing by nonlinear regression using the log(inhibitor) vs response (variable slope) equation. Results presented represent the averages of EC<sub>50</sub> values obtained from at triplicate experiments.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Calculation of IC<sub>50</sub>/EC<sub>50</sub>/CC<sub>50</sub> Values and Statistical Considerations</h3><div class="NLM_p last">All IC<sub>50</sub>/EC<sub>50</sub>/CC<sub>50</sub> results reported are averages of values determined from individual dose–response curves in replicate assays as follows: (1) individual I/E/CC<sub>50</sub> values from replicate assays were first log-transformed and the average log(I/E/CC<sub>50</sub>) values and standard deviations (SD) calculated; (2) replicate log(I/E/CC<sub>50</sub>) values were evaluated for outliers using the ROUT method in GraphPad Prism 6 (Q of 10%); and (3) average I/E/CC<sub>50</sub> values were then back-calculated from the average log(IC<sub>50</sub>) values. To compare statistical differences between log(I/E/CC<sub>50</sub>) values between the matched R- and L-series of analogs, two-tailed, paired <i>t</i> tests were performed using GraphPad Prism 6 (95% confidence level) and looking at differences between paired values (results are plotted in Figures S1–S5 in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>). For compounds where log(I/E/CC<sub>50</sub>) values were greater than the maximum compound concentrations tested (i.e., >1.8 and >2.0 or >63 and >100 μM, respectively), results were represented as 0.1 log units higher than the maximum concentrations tested (i.e., 1.9 and 2.1 or 79 and 126 μM, respectively) so as not to overly bias comparisons because of the unavailability of definitive values for these inactive compounds.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00989" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74232" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74232" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00989" class="ext-link">10.1021/acs.jmedchem.8b00989</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">SMILES strings of compound structures (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Tabulation of all biochemical IC<sub>50</sub>, bacterial proliferation EC<sub>50</sub>, and human cell viability CC<sub>50</sub> results; log(IC<sub>50</sub>), log(EC<sub>50</sub>), and log(CC<sub>50</sub>) results with standard deviations; plots of statistical differences between the R- and L-series analogs in the various assays; synthetic protocols and characterization data for test compounds (HPLC purity, MS, and <sup>1</sup>H NMR); experimental protocols for protein synthesis and purification, and biochemical, bacterial proliferation, and human cell viability assays (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_001.csv">jm8b00989_si_001.csv (7.76 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_002.pdf">jm8b00989_si_002.pdf (1.33 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00989" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64774" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64774" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven M. Johnson</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7254-4128" title="Orcid link">http://orcid.org/0000-0001-7254-4128</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#90fafff8fee3e4fdd0f9e5bef5f4e5"><span class="__cf_email__" data-cfemail="b4dedbdcdac7c0d9f4ddc19ad1d0c1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sanofar Abdeen</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Trent Kunkle</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nilshad Salim</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anne-Marie Ray</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Najiba Mammadova</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span>; 
    <span>Present Address:
                        Department of Genetics, Development and Cell Biology, Iowa State University, 1210 Molecular Biology Building, Pannel Dr, Ames, IA 50011</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Corey Summers</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span>; 
    <span>Present Address:
                        Department of Kinesiology, Iowa State University, 235 Barbara E. Forker Building, Beach Rd, Ames, IA 50011</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mckayla Stevens</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew J. Ambrose</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Department of Pharmacology and Toxicology, The University of Arizona, 1703 East Mabel Street, P.O. Box
210207, Tucson, Arizona 85721, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2932-4514" title="Orcid link">http://orcid.org/0000-0002-2932-4514</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yangshin Park</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Stark
Neurosciences Research Institute, Indiana
University School of Medicine, 320 West 15th Street, Suite 414, Indianapolis, Indiana 46202, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Neurology, Indiana University School
of Medicine, 635 Barnhill
Drive, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter G. Schultz</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, California 92037, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3188-1202" title="Orcid link">http://orcid.org/0000-0003-3188-1202</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Arthur L. Horwich</span> - <span class="hlFld-Affiliation affiliation">HHMI,
Department of Genetics, Yale School of Medicine, Boyer Center for Molecular Medicine, 295 Congress Avenue, New Haven, Connecticut 06510, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Quyen Q. Hoang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Indiana
University School of Medicine, 635 Barnhill Drive, Indianapolis, Indiana 46202, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Stark
Neurosciences Research Institute, Indiana
University School of Medicine, 320 West 15th Street, Suite 414, Indianapolis, Indiana 46202, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Neurology, Indiana University School
of Medicine, 635 Barnhill
Drive, Indianapolis, Indiana 46202, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eli Chapman</span> - <span class="hlFld-Affiliation affiliation">College
of Pharmacy, Department of Pharmacology and Toxicology, The University of Arizona, 1703 East Mabel Street, P.O. Box
210207, Tucson, Arizona 85721, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2879-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00841" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00841" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Research reported in this publication was supported by the National Institute of General Medical Sciences (NIGMS) of the National Institutes of Health (NIH) under Award Number R01GM120350. Q.Q.H. and Y.P. additionally acknowledge support from NIH Grants 5R01GM111639 and 5R01GM115844. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH. This work was also supported, in part, by the Howard Hughes Medical Institute (AH), an IU Biomedical Research Grant (SMJ), and startup funds from the IU School of Medicine (SMJ) and the University of Arizona (EC). The human HSP60 expression plasmid (lacking the 26 amino acid <i>N</i>-terminal mitochondrial signal peptide) was generously donated by Dr. Abdussalam Azem from Tel Aviv University, Faculty of Life Sciences, Department of Biochemistry, Israel.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">MDH</td><td class="NLM_def"><p class="first last">malate dehydrogenase</p></td></tr><tr><td class="NLM_term">Rho</td><td class="NLM_def"><p class="first last">rhodanese</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term"><sup>1</sup>H NMR</td><td class="NLM_def"><p class="first last">Proton nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">Mass spectrometry</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">AcOH</td><td class="NLM_def"><p class="first last">acetic acid</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">Inhibitory concentration for half-maximal signal in biochemical assay</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">effective concentration for half-maximal signal in bacterial proliferation assays</p></td></tr><tr><td class="NLM_term">CC<sub>50</sub></td><td class="NLM_def"><p class="first last">cytotoxicity concentration for half-maximal signal in human cell viability assays</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00138" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00138" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 36 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span>; <span class="NLM_string-name">Magrini, N.</span></span> <i>Global priority list of antibiotic-resistant bacteria to guide research, discovery, and development of new antibiotics</i>; <span class="NLM_publisher-name">World Health Organization</span>: <span class="NLM_publisher-loc">Geneva, Switzerland</span>, <span class="NLM_year">2017</span>; p  <span class="NLM_fpage">7</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=7&author=E.+Tacconelli&author=N.+Magrini&title=Global+priority+list+of+antibiotic-resistant+bacteria+to+guide+research%2C+discovery%2C+and+development+of+new+antibiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DTacconelli%26aufirst%3DE.%26btitle%3DGlobal%2520priority%2520list%2520of%2520antibiotic-resistant%2520bacteria%2520to%2520guide%2520research%252C%2520discovery%252C%2520and%2520development%2520of%2520new%2520antibiotics%26pub%3DWorld%2520Health%2520Organization%26date%3D2017%26spage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Centers
for Disease Control and Prevention</span> <i>Antibiotic Resistance
Threats in the United States, 2013</i>; <span class="NLM_publisher-name">Centers
for Disease Control and Prevention</span>: <span class="NLM_publisher-loc">Atlanta, Georgia</span>, <span class="NLM_year">2013</span>; p  <span class="NLM_fpage">114</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=114&author=Centers%0Afor+Disease+Control+and+Prevention&title=Antibiotic+Resistance%0AThreats+in+the+United+States%2C+2013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DAntibiotic%2520Resistance%250AThreats%2520in%2520the%2520United%2520States%252C%25202013%26pub%3DCenters%250Afor%2520Disease%2520Control%2520and%2520Prevention%26date%3D2013%26spage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karlowsky, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackel, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lob, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahm, D. F.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial susceptibility of Gram-negative ESKAPE pathogens isolated from hospitalized patients with intra-abdominal and urinary tract infections in Asia-Pacific countries: SMART 2013–2015</span>. <i>J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1099/jmm.0.000421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1099%2Fjmm.0.000421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=28051952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BC1c7itF2rug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=61-69&author=J.+A.+Karlowskyauthor=D.+J.+Hobanauthor=M.+A.+Hackelauthor=S.+H.+Lobauthor=D.+F.+Sahm&title=Antimicrobial+susceptibility+of+Gram-negative+ESKAPE+pathogens+isolated+from+hospitalized+patients+with+intra-abdominal+and+urinary+tract+infections+in+Asia-Pacific+countries%3A+SMART+2013%E2%80%932015&doi=10.1099%2Fjmm.0.000421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial susceptibility of Gram-negative ESKAPE pathogens isolated from hospitalized patients with intra-abdominal and urinary tract infections in Asia-Pacific countries: SMART 2013-2015</span></div><div class="casAuthors">Karlowsky James A; Hoban Daryl J; Hackel Meredith A; Lob Sibylle H; Sahm Daniel F</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medical microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-69</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Gram-negative ESKAPE pathogens (Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) are responsible for increases in antimicrobial-resistant infections worldwide.  We determined in vitro susceptibilities to eight parenteral antimicrobial agents using Clinical and Laboratory Standards Institute broth microdilution methodology for Gram-negative ESKAPE pathogens isolated from hospitalized patients with intra-abdominal infections (IAIs) (n=3052) and urinary tract infections (UTIs) (n=1088) in 11 Asia-Pacific countries/regions from 2013 to 2015.  Amikacin (98.3, 96.4 %), imipenem (97.1, 95.5 %) and ertapenem (95.3, 93.2 %) demonstrated the highest rates of susceptibility for isolates of K. pneumoniae from IAI and UTI, respectively, whereas susceptibility to advanced-generation cephalosporins was <84 and <71 %, respectively.  K. pneumoniae with an extended-spectrum β-lactamase-positive phenotype were more common in UTI (27.1 %) than IAI (16.2 %).  Imipenem and amikacin were the most active agents against extended-spectrum β-lactamase-positive K. pneumoniae from IAI (95.1, 91.8 %) and UTI (94.9, 92.3 %), respectively, whereas <54 % were susceptible to piperacillin-tazobactam.  Against Enterobacter spp. and P. aeruginosa, amikacin demonstrated the highest rates of susceptibility for isolates from IAI (99.7, 95.5 %) and UTI (90.9, 91.5 %), respectively.  K. pneumoniae, Enterobacter spp. and P. aeruginosa from urine demonstrated lower susceptibility to levofloxacin (74.1, 81.8 and 73.8 %) than from IAI (87.6, 91.8 and 85.4 %).  For A. baumannii, rates of susceptibility to all agents tested were <43 %.  We conclude that the studied Gram-negative ESKAPE pathogens demonstrated reduced susceptibility to commonly prescribed advanced-generation cephalosporins, piperacillin-tazobactam and levofloxacin, while amikacin and carbapenems were the most active.  Ongoing surveillance to monitor evolving resistance trends and the development of novel antimicrobial agents with potent activity against Gram-negative ESKAPE pathogens are mandatory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPKOnN8YplpDUE-swuRqsHfW6udTcc2eY8VQJ5rN7w0rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7itF2rug%253D%253D&md5=ace8f3676d7540bd820cdc708c9b4d76</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1099%2Fjmm.0.000421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fjmm.0.000421%26sid%3Dliteratum%253Aachs%26aulast%3DKarlowsky%26aufirst%3DJ.%2BA.%26aulast%3DHoban%26aufirst%3DD.%2BJ.%26aulast%3DHackel%26aufirst%3DM.%2BA.%26aulast%3DLob%26aufirst%3DS.%2BH.%26aulast%3DSahm%26aufirst%3DD.%2BF.%26atitle%3DAntimicrobial%2520susceptibility%2520of%2520Gram-negative%2520ESKAPE%2520pathogens%2520isolated%2520from%2520hospitalized%2520patients%2520with%2520intra-abdominal%2520and%2520urinary%2520tract%2520infections%2520in%2520Asia-Pacific%2520countries%253A%2520SMART%25202013%25E2%2580%25932015%26jtitle%3DJ.%2520Med.%2520Microbiol.%26date%3D2017%26volume%3D66%26spage%3D61%26epage%3D69%26doi%3D10.1099%2Fjmm.0.000421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santajit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indrawattana, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of antimicrobial resistance in ESKAPE pathogens</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">2475067</span>, <span class="refDoi"> DOI: 10.1155/2016/2475067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1155%2F2016%2F2475067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=27274985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=2475067&author=S.+Santajitauthor=N.+Indrawattana&title=Mechanisms+of+antimicrobial+resistance+in+ESKAPE+pathogens&doi=10.1155%2F2016%2F2475067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Antimicrobial Resistance in ESKAPE Pathogens</span></div><div class="casAuthors">Santajit Sirijan; Indrawattana Nitaya</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2016</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2475067</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) are the leading cause of nosocomial infections throughout the world.  Most of them are multidrug resistant isolates, which is one of the greatest challenges in clinical practice.  Multidrug resistance is amongst the top three threats to global public health and is usually caused by excessive drug usage or prescription, inappropriate use of antimicrobials, and substandard pharmaceuticals.  Understanding the resistance mechanisms of these bacteria is crucial for the development of novel antimicrobial agents or other alternative tools to combat these public health challenges.  Greater mechanistic understanding would also aid in the prediction of underlying or even unknown mechanisms of resistance, which could be applied to other emerging multidrug resistant pathogens.  In this review, we summarize the known antimicrobial resistance mechanisms of ESKAPE pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSM_xN5GU5d0ntlzo9xastyfW6udTcc2eYQjfYqiWW897ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D&md5=e42353b6d72db948734b5354e31b501e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1155%2F2016%2F2475067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2016%252F2475067%26sid%3Dliteratum%253Aachs%26aulast%3DSantajit%26aufirst%3DS.%26aulast%3DIndrawattana%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520antimicrobial%2520resistance%2520in%2520ESKAPE%2520pathogens%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2016%26volume%3D2016%26spage%3D2475067%26doi%3D10.1155%2F2016%2F2475067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. B.</span></span> <span> </span><span class="NLM_article-title">Federal funding for the study of antimicrobial resistance in nosocomial pathogens: No ESKAPE</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>197</i></span>,  <span class="NLM_fpage">1079</span>– <span class="NLM_lpage">1081</span>, <span class="refDoi"> DOI: 10.1086/533452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1086%2F533452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=18419525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BD1c3lvFChug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2008&pages=1079-1081&author=L.+B.+Rice&title=Federal+funding+for+the+study+of+antimicrobial+resistance+in+nosocomial+pathogens%3A+No+ESKAPE&doi=10.1086%2F533452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Federal funding for the study of antimicrobial resistance in nosocomial pathogens: no ESKAPE</span></div><div class="casAuthors">Rice Louis B</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of infectious diseases</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">197</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1079-81</span>
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcReTA_wnRzWZTslzEEsg76ufW6udTcc2eYQjfYqiWW897ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3lvFChug%253D%253D&md5=c2c5ec8ad941f702f588f89741d9410e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1086%2F533452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F533452%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DL.%2BB.%26atitle%3DFederal%2520funding%2520for%2520the%2520study%2520of%2520antimicrobial%2520resistance%2520in%2520nosocomial%2520pathogens%253A%2520No%2520ESKAPE%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2008%26volume%3D197%26spage%3D1079%26epage%3D1081%26doi%3D10.1086%2F533452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, H. F.</span></span> <span> </span><span class="NLM_article-title">Treatment of methicillin-resistant Staphylococcus aureus bacteremia</span>. <i>Infect. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.3947/ic.2016.48.4.267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.3947%2Fic.2016.48.4.267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=28032484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=267-273&author=E.+J.+Chooauthor=H.+F.+Chambers&title=Treatment+of+methicillin-resistant+Staphylococcus+aureus+bacteremia&doi=10.3947%2Fic.2016.48.4.267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of methicillin-resistant Staphylococcus aureus bacteremia</span></div><div class="casAuthors">Choo, Eun Ju; Chambers, Henry F.</div><div class="citationInfo"><span class="NLM_cas:title">Infection & Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">267-273</span>CODEN:
                <span class="NLM_cas:coden">ICNHAK</span>;
        ISSN:<span class="NLM_cas:issn">2093-2340</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Infectious Diseases</span>)
        </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) is a significant cause of health care-assocd. infections.  Vancomycin remains an acceptable treatment option.  There has been a welcome increase in the no. of agents available for the treatment of MRSA infection.  These drugs have certain differentiating attributes and may offer some advantages over vancomycin, but they also have significant limitations.  These agents provide some alternative when no other options arc available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIc_NWfevGv7Vg90H21EOLACvtfcHk0ljDyrkm6BAVdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKiur4%253D&md5=4a25789660d694935631b23cf2de0363</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3947%2Fic.2016.48.4.267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3947%252Fic.2016.48.4.267%26sid%3Dliteratum%253Aachs%26aulast%3DChoo%26aufirst%3DE.%2BJ.%26aulast%3DChambers%26aufirst%3DH.%2BF.%26atitle%3DTreatment%2520of%2520methicillin-resistant%2520Staphylococcus%2520aureus%2520bacteremia%26jtitle%3DInfect.%2520Chemother.%26date%3D2016%26volume%3D48%26spage%3D267%26epage%3D273%26doi%3D10.3947%2Fic.2016.48.4.267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassoun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linden, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, B.</span></span> <span> </span><span class="NLM_article-title">Incidence, prevalence, and management of MRSA bacteremia across patient populations-a review of recent developments in MRSA management and treatment</span>. <i>Crit. Care</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">211</span>, <span class="refDoi"> DOI: 10.1186/s13054-017-1801-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1186%2Fs13054-017-1801-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=28807042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BC1cfnsVKrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=211&author=A.+Hassounauthor=P.+K.+Lindenauthor=B.+Friedman&title=Incidence%2C+prevalence%2C+and+management+of+MRSA+bacteremia+across+patient+populations-a+review+of+recent+developments+in+MRSA+management+and+treatment&doi=10.1186%2Fs13054-017-1801-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Incidence, prevalence, and management of MRSA bacteremia across patient populations-a review of recent developments in MRSA management and treatment</span></div><div class="casAuthors">Hassoun Ali; Linden Peter K; Friedman Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Critical care (London, England)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">211</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) infection is still a major global healthcare problem.  Of concern is S. aureus bacteremia, which exhibits high rates of morbidity and mortality and can cause metastatic or complicated infections such as infective endocarditis or sepsis.  MRSA is responsible for most global S. aureus bacteremia cases, and compared with methicillin-sensitive S. aureus, MRSA infection is associated with poorer clinical outcomes.  S. aureus virulence is affected by the unique combination of toxin and immune-modulatory gene products, which may differ by geographic location and healthcare- or community-associated acquisition.  Management of S. aureus bacteremia involves timely identification of the infecting strain and source of infection, proper choice of antibiotic treatment, and robust prevention strategies.  Resistance and nonsusceptibility to first-line antimicrobials combined with a lack of equally effective alternatives complicates MRSA bacteremia treatment.  This review describes trends in epidemiology and factors that influence the incidence of MRSA bacteremia.  Current and developing diagnostic tools, treatments, and prevention strategies are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRY-2a_VQlONUBuFdjgVNKSfW6udTcc2ea7YWR7vA2AUrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfnsVKrtg%253D%253D&md5=a004de0c857f92a6dddfd7ea6a0d3493</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2Fs13054-017-1801-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13054-017-1801-3%26sid%3Dliteratum%253Aachs%26aulast%3DHassoun%26aufirst%3DA.%26aulast%3DLinden%26aufirst%3DP.%2BK.%26aulast%3DFriedman%26aufirst%3DB.%26atitle%3DIncidence%252C%2520prevalence%252C%2520and%2520management%2520of%2520MRSA%2520bacteremia%2520across%2520patient%2520populations-a%2520review%2520of%2520recent%2520developments%2520in%2520MRSA%2520management%2520and%2520treatment%26jtitle%3DCrit.%2520Care%26date%3D2017%26volume%3D21%26spage%3D211%26doi%3D10.1186%2Fs13054-017-1801-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razonable, R. R.</span></span> <span> </span><span class="NLM_article-title">Serious infections caused by methicillin-resistant Staphylococcus aureus</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">S183</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1086/653519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1086%2F653519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=20731576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yhsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=S183-197&author=H.+Boucherauthor=L.+G.+Millerauthor=R.+R.+Razonable&title=Serious+infections+caused+by+methicillin-resistant+Staphylococcus+aureus&doi=10.1086%2F653519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Serious infections caused by methicillin-resistant Staphylococcus aureus</span></div><div class="casAuthors">Boucher, Helen; Miller, Loren G.; Razonable, Raymund R.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Infectious Diseases</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S183-S197</span>CODEN:
                <span class="NLM_cas:coden">CIDIEL</span>;
        ISSN:<span class="NLM_cas:issn">1058-4838</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">A review.  Although first identified just >4 decades ago, methicillin-resistant Staphylococcus aureus (MRSA) has undergone rapid evolutionary changes and epidemiol. expansion to become a major cause of nosocomial and community-acquired infections worldwide.  Increasing resistance to vancomycin among MRSA strains in conjunction with availability of new antibiotics, including daptomycin and linezolid, have increased treatment choices but made clin. treatment decisions more challenging.  This article describes the clin. features and management issues of 2 challenging-to-treat manifestations of MRSA infection, bacteremia and/or endocarditis and osteomyelitis.  It also presents a brief review of community-assocd. MRSA infections and preventive strategies directed against MRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVLYfFk5mCbVg90H21EOLACvtfcHk0liByPzEpWBtTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yhsbfO&md5=e4fb086393d4fa614e148306002d50af</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1086%2F653519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F653519%26sid%3Dliteratum%253Aachs%26aulast%3DBoucher%26aufirst%3DH.%26aulast%3DMiller%26aufirst%3DL.%2BG.%26aulast%3DRazonable%26aufirst%3DR.%2BR.%26atitle%3DSerious%2520infections%2520caused%2520by%2520methicillin-resistant%2520Staphylococcus%2520aureus%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2010%26volume%3D51%26spage%3DS183%26epage%3D197%26doi%3D10.1086%2F653519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houry, W. A.</span></span> <span> </span><span class="NLM_article-title">Chaperone networks in bacteria: analysis of protein homeostasis in minimal cells</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2003.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.jsb.2003.11.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2004&pages=79-89&author=P.+Wongauthor=W.+A.+Houry&title=Chaperone+networks+in+bacteria%3A+analysis+of+protein+homeostasis+in+minimal+cells&doi=10.1016%2Fj.jsb.2003.11.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2003.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2003.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DP.%26aulast%3DHoury%26aufirst%3DW.%2BA.%26atitle%3DChaperone%2520networks%2520in%2520bacteria%253A%2520analysis%2520of%2520protein%2520homeostasis%2520in%2520minimal%2520cells%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2004%26volume%3D146%26spage%3D79%26epage%3D89%26doi%3D10.1016%2Fj.jsb.2003.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mogk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span> <span> </span><span class="NLM_article-title">Integrating protein homeostasis strategies in prokaryotes</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">a004366</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a004366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1101%2Fcshperspect.a004366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlejsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=a004366&author=A.+Mogkauthor=D.+Huberauthor=B.+Bukau&title=Integrating+protein+homeostasis+strategies+in+prokaryotes&doi=10.1101%2Fcshperspect.a004366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating protein homeostasis strategies in prokaryotes</span></div><div class="casAuthors">Mogk, Axel; Huber, Damon; Bukau, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">a004366/1-a004366/19</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Bacterial cells are frequently exposed to dramatic fluctuations in their environment, which cause perturbation in protein homeostasis and lead to protein misfolding.  Bacteria have therefore evolved powerful quality control networks consisting of chaperones and proteases that cooperate to monitor the folding states of proteins and to remove misfolded conformers through either refolding or degrdn.  The levels of the quality control components are adjusted to the folding state of the cellular proteome through the induction of compartment specific stress responses.  In addn., the activities of several quality control components are directly controlled by these stresses, allowing for fast activation.  Severe stress can, however, overcome the protective function of the proteostasis network leading to the formation of protein aggregates, which are sequestered at the cell poles.  Protein aggregates are either solubilized by AAA+ chaperones or eliminated through cell division, allowing for the generation of damage-free daughter cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgphXy0ybMALVg90H21EOLACvtfcHk0liByPzEpWBtTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlejsL0%253D&md5=f0d5cf1d8055ce3664e40fd02d0245bc</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a004366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a004366%26sid%3Dliteratum%253Aachs%26aulast%3DMogk%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DD.%26aulast%3DBukau%26aufirst%3DB.%26atitle%3DIntegrating%2520protein%2520homeostasis%2520strategies%2520in%2520prokaryotes%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3Da004366%26doi%3D10.1101%2Fcshperspect.a004366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiappori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanesi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merelli, I.</span></span> <span> </span><span class="NLM_article-title">DnaK as antibiotic target: Hot spot residues analysis for differential inhibition of the bacterial protein in comparison with the human HSP70</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0124563</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0124563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1371%2Fjournal.pone.0124563" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0124563&author=F.+Chiapporiauthor=M.+Fumianauthor=L.+Milanesiauthor=I.+Merelli&title=DnaK+as+antibiotic+target%3A+Hot+spot+residues+analysis+for+differential+inhibition+of+the+bacterial+protein+in+comparison+with+the+human+HSP70&doi=10.1371%2Fjournal.pone.0124563"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0124563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0124563%26sid%3Dliteratum%253Aachs%26aulast%3DChiappori%26aufirst%3DF.%26aulast%3DFumian%26aufirst%3DM.%26aulast%3DMilanesi%26aufirst%3DL.%26aulast%3DMerelli%26aufirst%3DI.%26atitle%3DDnaK%2520as%2520antibiotic%2520target%253A%2520Hot%2520spot%2520residues%2520analysis%2520for%2520differential%2520inhibition%2520of%2520the%2520bacterial%2520protein%2520in%2520comparison%2520with%2520the%2520human%2520HSP70%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0124563%26doi%3D10.1371%2Fjournal.pone.0124563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arita-Morioka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizunoe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, S.</span></span> <span> </span><span class="NLM_article-title">Novel strategy for biofilm inhibition by using small molecules targeting molecular chaperone DnaK</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1128/AAC.04465-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1128%2FAAC.04465-14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=633-641&author=K.+Arita-Moriokaauthor=K.+Yamanakaauthor=Y.+Mizunoeauthor=T.+Oguraauthor=S.+Sugimoto&title=Novel+strategy+for+biofilm+inhibition+by+using+small+molecules+targeting+molecular+chaperone+DnaK&doi=10.1128%2FAAC.04465-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FAAC.04465-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.04465-14%26sid%3Dliteratum%253Aachs%26aulast%3DArita-Morioka%26aufirst%3DK.%26aulast%3DYamanaka%26aufirst%3DK.%26aulast%3DMizunoe%26aufirst%3DY.%26aulast%3DOgura%26aufirst%3DT.%26aulast%3DSugimoto%26aufirst%3DS.%26atitle%3DNovel%2520strategy%2520for%2520biofilm%2520inhibition%2520by%2520using%2520small%2520molecules%2520targeting%2520molecular%2520chaperone%2520DnaK%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D633%26epage%3D641%26doi%3D10.1128%2FAAC.04465-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sass, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famulla, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahl, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamoen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotz-Oesterhelt, H.</span></span> <span> </span><span class="NLM_article-title">Antibiotic acyldepsipeptides activate ClpP peptidase to degrade the cell division protein FtsZ</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">17474</span>– <span class="NLM_lpage">17479</span>, <span class="refDoi"> DOI: 10.1073/pnas.1110385108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1073%2Fpnas.1110385108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=21969594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtl2ltb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=17474-17479&author=P.+Sassauthor=M.+Jostenauthor=K.+Famullaauthor=G.+Schifferauthor=H.+G.+Sahlauthor=L.+Hamoenauthor=H.+Brotz-Oesterhelt&title=Antibiotic+acyldepsipeptides+activate+ClpP+peptidase+to+degrade+the+cell+division+protein+FtsZ&doi=10.1073%2Fpnas.1110385108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic acyldepsipeptides activate ClpP peptidase to degrade the cell division protein FtsZ</span></div><div class="casAuthors">Sass, Peter; Josten, Michaele; Famulla, Kirsten; Schiffer, Guido; Sahl, Hans-Georg; Hamoen, Leendert; Broetz-Oesterhelt, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">17474-17479, S17474/1-S17474/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The worldwide spread of antibiotic-resistant bacteria has lent urgency to the search for antibiotics with new modes of action that are devoid of preexisting cross-resistances.  We previously described a unique class of acyldepsipeptides (ADEPs) that exerts prominent antibacterial activity against Gram-pos. pathogens including streptococci, enterococci, as well as multi-drug-resistant Staphylococcus aureus.  Here, we report that ADEP prevents cell division in Gram-pos. bacteria and induces strong filamentation of rod-shaped Bacillus subtilis and swelling of coccoid S. aureus and Streptococcus pneumoniae.  It emerged that ADEP treatment inhibits septum formation at the stage of Z-ring assembly, and that central cell division proteins delocalize from midcell positions.  Using in vivo and in vitro studies, we show that the inhibition of Z-ring formation is a consequence of the proteolytic degrdn. of the essential cell division protein FtsZ.  ADEP switches the bacterial ClpP peptidase from a regulated to an uncontrolled protease, and it turned out that FtsZ is particularly prone to degrdn. by the ADEP-CIpP complex.  By preventing cell division, ADEP inhibits a vital cellular process of bacteria that is not targeted by any therapeutically applied antibiotic so far.  Their unique multi-faceted mechanism of action and antibacterial potency makes them promising lead structures for future antibiotic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc8ifJ9CVpkLVg90H21EOLACvtfcHk0ljfCTCfZZE_nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtl2ltb3F&md5=7f6f9401c63b9705cc81b917fb69fe81</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1110385108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1110385108%26sid%3Dliteratum%253Aachs%26aulast%3DSass%26aufirst%3DP.%26aulast%3DJosten%26aufirst%3DM.%26aulast%3DFamulla%26aufirst%3DK.%26aulast%3DSchiffer%26aufirst%3DG.%26aulast%3DSahl%26aufirst%3DH.%2BG.%26aulast%3DHamoen%26aufirst%3DL.%26aulast%3DBrotz-Oesterhelt%26aufirst%3DH.%26atitle%3DAntibiotic%2520acyldepsipeptides%2520activate%2520ClpP%2520peptidase%2520to%2520degrade%2520the%2520cell%2520division%2520protein%2520FtsZ%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D17474%26epage%3D17479%26doi%3D10.1073%2Fpnas.1110385108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Heat shock protein 70 (hsp70) as an emerging drug target</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">4585</span>– <span class="NLM_lpage">4602</span>, <span class="refDoi"> DOI: 10.1021/jm100054f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100054f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4585-4602&author=C.+G.+Evansauthor=L.+Changauthor=J.+E.+Gestwicki&title=Heat+shock+protein+70+%28hsp70%29+as+an+emerging+drug+target&doi=10.1021%2Fjm100054f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Heat Shock Protein 70 (Hsp70) as an Emerging Drug Target</span></div><div class="casAuthors">Evans, Christopher G.; Chang, Lyra; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4585-4602</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5hrQMA77XsrVg90H21EOLACvtfcHk0lhQU9vUO-Q_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejsLc%253D&md5=b87bdc9b76e66d62c130a162037387b5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm100054f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100054f%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DC.%2BG.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DHeat%2520shock%2520protein%252070%2520%2528hsp70%2529%2520as%2520an%2520emerging%2520drug%2520target%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4585%26epage%3D4602%26doi%3D10.1021%2Fjm100054f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braig, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boisvert, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigler, P. B.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of the bacterial chaperonin GroEL at 2.8 A</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1038/371578a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1038%2F371578a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=7935790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK2cXmslCrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=1994&pages=578-586&author=K.+Braigauthor=Z.+Otwinowskiauthor=R.+Hegdeauthor=D.+C.+Boisvertauthor=A.+Joachimiakauthor=A.+L.+Horwichauthor=P.+B.+Sigler&title=The+crystal+structure+of+the+bacterial+chaperonin+GroEL+at+2.8+A&doi=10.1038%2F371578a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the bacterial chaperonin GroEL at 2.9 Å</span></div><div class="casAuthors">Braig, Kerstin; Otwinowski, Zbyszek; Hegde, Rashmi; Boisvert, David C.; Joachimiak, Andrzej; Horwich, Arthur L.; Sigler, Paul B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">578-86</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The crystal structure of Escherichia coli chaperonin GroEL (I) showed a porous cylinder of 14 subunits made of 2 nearly 7-fold rotationally sym. rings stacked back-to-back with dyad symmetry.  The I subunits consisted of 3 domains: a large equatorial domain that forms the foundation of the assembly at its waist and holds the rings together; a large loosely structured apical domain that forms the ends of the cylinder; and a small slender intermediate domain that connects the 2, creating side-windows.  The 3-dimensional structure placed most of the mutationally defined functional sites on the channel walls and its outward invaginations, and at the ends of the cylinder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOMsEsmFMy_7Vg90H21EOLACvtfcHk0lhQU9vUO-Q_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmslCrsbk%253D&md5=6b1a0fa2aeb3affd03c5bda6256c8764</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2F371578a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F371578a0%26sid%3Dliteratum%253Aachs%26aulast%3DBraig%26aufirst%3DK.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DHegde%26aufirst%3DR.%26aulast%3DBoisvert%26aufirst%3DD.%2BC.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DSigler%26aufirst%3DP.%2BB.%26atitle%3DThe%2520crystal%2520structure%2520of%2520the%2520bacterial%2520chaperonin%2520GroEL%2520at%25202.8%2520A%26jtitle%3DNature%26date%3D1994%26volume%3D371%26spage%3D578%26epage%3D586%26doi%3D10.1038%2F371578a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sigler, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rye, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burston, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Structure and function in GroEL-mediated protein folding</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.67.1.581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1146%2Fannurev.biochem.67.1.581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=9759498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK1cXlsFOmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1998&pages=581-608&author=P.+B.+Siglerauthor=Z.+Xuauthor=H.+S.+Ryeauthor=S.+G.+Burstonauthor=W.+A.+Fentonauthor=A.+L.+Horwich&title=Structure+and+function+in+GroEL-mediated+protein+folding&doi=10.1146%2Fannurev.biochem.67.1.581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function in GroEL-mediated protein folding</span></div><div class="casAuthors">Sigler, Paul B.; Xu, Zhaohui; Rye, Hays S.; Burston, Steven G.; Fenton, Wayne A.; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">581-608</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review, with ∼83 refs.  Recent structural and biochem. investigations have come together to allow a better understanding of the mechanism of chaperonin (GroEL, Hsp60)-mediated protein folding, the final step in the accurate expression of genetic information.  Major, asym. conformational changes in the GroEL double toroid accompany binding of ATP and the cochaperonin GroES.  When a nonnative polypeptide, bound to one of the GroEL rings, is encapsulated by GroES to form a cis ternary complex, these changes drive the polypeptide into the sequestered cavity and initiate its folding.  ATP hydrolysis in the cis ring primes release of the products, and ATP binding in the trans ring then disrupts the cis complex.  This process allows the polypeptide to achieve its final native state, if folding was completed, or to recycle to another chaperonin mol., if the folding process did not result in a form committed to the native state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrss2vZQv_Th7Vg90H21EOLACvtfcHk0lhQU9vUO-Q_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlsFOmsb0%253D&md5=3f0f7aaa6c4bba7761daabd0ea4ffd62</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.67.1.581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.67.1.581%26sid%3Dliteratum%253Aachs%26aulast%3DSigler%26aufirst%3DP.%2BB.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DRye%26aufirst%3DH.%2BS.%26aulast%3DBurston%26aufirst%3DS.%2BG.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520and%2520function%2520in%2520GroEL-mediated%2520protein%2520folding%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1998%26volume%3D67%26spage%3D581%26epage%3D608%26doi%3D10.1146%2Fannurev.biochem.67.1.581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farr, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span> <span> </span><span class="NLM_article-title">GroEL-GroES-mediated protein folding</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">1917</span>– <span class="NLM_lpage">1930</span>, <span class="refDoi"> DOI: 10.1021/cr040435v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040435v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFKiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=1917-1930&author=A.+L.+Horwichauthor=G.+W.+Farrauthor=W.+A.+Fenton&title=GroEL-GroES-mediated+protein+folding&doi=10.1021%2Fcr040435v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">GroEL-GroES-mediated protein folding</span></div><div class="casAuthors">Horwich, Arthur L.; Farr, George W.; Fenton, Wayne A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1917-1930</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Chaperonin-mediated folding is an essential ATP-dependent reaction that provides kinetic assistance to the process of protein folding o the native state in a variety of cellular compartments.  The Escherichia coli reaction is carried out by a megadalton-sized double ring "machine" consisting of chaperonin GroEL in complex with its co-chaperonin GroES.  The binding of GroES to GroEL is ATP-dependent with the nucleotide rapidly and cooperatively binding to 7 sites on the GroEL ring.  Here, the catalytic mechanism of GroEL-GroES-mediated protein folding is discussed along with a more detailed consideration of the transition between polypeptide binding in an open ring and productive protein folding in a co-chaperone-encapsulated one.  The free energy of binding of the γ-phosphate of ATP functions as a crit. element of the folding trigger.  The bulk of existing evidence appears to support a model of GroEL action in which ATP and GroES binding drive major conformational changes in GroEL that simultaneously and immediately release a substrate protein from the apical binding sites and initiate refolding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoTMQWAk1wD7Vg90H21EOLACvtfcHk0lgS3v6m9IcdOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFKiur8%253D&md5=3e71992bed612e9bc567d6bc9318b414</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fcr040435v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040435v%26sid%3Dliteratum%253Aachs%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DFarr%26aufirst%3DG.%2BW.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26atitle%3DGroEL-GroES-mediated%2520protein%2520folding%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D1917%26epage%3D1930%26doi%3D10.1021%2Fcr040435v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Residues in chaperonin GroEL required for polypeptide binding and release</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/371614a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1038%2F371614a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=7935796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK2cXmslCrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=1994&pages=614-619&author=W.+A.+Fentonauthor=Y.+Kashiauthor=K.+Furtakauthor=A.+L.+Norwich&title=Residues+in+chaperonin+GroEL+required+for+polypeptide+binding+and+release&doi=10.1038%2F371614a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Residues in chaperonin GroEL required for polypeptide binding and release</span></div><div class="casAuthors">Fenton, Wayne A.; Kashi, Yechezkel; Furtak, Krystyna; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">614-19</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Chaperonins are ring-shaped protein complexes that are essential in the cell, mediating ATP-dependent polypeptide folding in a variety of compartments.  Recent studies suggest that they function through multiple rounds of binding and release of non-native proteins: with each round of ATP-driven release into the bulk soln., a substrate protein kinetically partitions between folding to the native state or rebinding to another chaperonin mol.  To gain further insight into the mechanism of polypeptide bind and release by the chaperonin GroEL from Escherichia coli, a mutational anal. was undertaken that relates the functional properties of GroEL to its crystal structure.  The functional tests identify a putative polypeptide-binding site on the inside surface of the apical domain, facing the central channel, consisting of hydrophobic residues.  These same residues are essential for binding of the co-chaperonin GroES, which is required for productive polypeptide release.  A highly conserved residue, Asp-87, positioned within a putative nucleotide-binding pocket in the top of the equatorial domain, is essential for ATP hydrolysis and polypeptide release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGwwF9c_IU0LVg90H21EOLACvtfcHk0lgS3v6m9IcdOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmslCrsbY%253D&md5=912a0f11eb449acba4a1b2b13371db0f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2F371614a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F371614a0%26sid%3Dliteratum%253Aachs%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DKashi%26aufirst%3DY.%26aulast%3DFurtak%26aufirst%3DK.%26aulast%3DNorwich%26aufirst%3DA.%2BL.%26atitle%3DResidues%2520in%2520chaperonin%2520GroEL%2520required%2520for%2520polypeptide%2520binding%2520and%2520release%26jtitle%3DNature%26date%3D1994%26volume%3D371%26spage%3D614%26epage%3D619%26doi%3D10.1038%2F371614a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">GroEL-mediated protein folding</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">760</span>, <span class="refDoi"> DOI: 10.1002/pro.5560060401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1002%2Fpro.5560060401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=9098884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK2sXisFWhu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=743-760&author=W.+A.+Fentonauthor=A.+L.+Horwich&title=GroEL-mediated+protein+folding&doi=10.1002%2Fpro.5560060401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">GroEL-mediated protein folding</span></div><div class="casAuthors">Fenton, Wayne A.; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">743-760</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review, with ∼120 refs., on the structural and functional studies that have led to the current view of GroEL-mediated protein folding and aspects of chaperonin mechanics that are not, as yet, understood.  Topics include (1) architecture of GroEL and GroES and the reaction pathway, (2) polypeptide binding, (3) ATP binding and hydrolysis in driving the reaction cycle, and (4) GroEL-GroES-polypeptide ternary complexes and the folding-active cis complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqlO7K1EQL1LVg90H21EOLACvtfcHk0lgS3v6m9IcdOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisFWhu7o%253D&md5=c36df13f76cbccb702a52120a5df5a5f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560060401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560060401%26sid%3Dliteratum%253Aachs%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DGroEL-mediated%2520protein%2520folding%26jtitle%3DProtein%2520Sci.%26date%3D1997%26volume%3D6%26spage%3D743%26epage%3D760%26doi%3D10.1002%2Fpro.5560060401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farr, G. W.</span></span> <span> </span><span class="NLM_article-title">Two families of chaperonin: physiology and mechanism</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1146/annurev.cellbio.23.090506.123555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1146%2Fannurev.cellbio.23.090506.123555" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=115-145&author=A.+L.+Horwichauthor=W.+A.+Fentonauthor=E.+Chapmanauthor=G.+W.+Farr&title=Two+families+of+chaperonin%3A+physiology+and+mechanism&doi=10.1146%2Fannurev.cellbio.23.090506.123555"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1146%2Fannurev.cellbio.23.090506.123555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.cellbio.23.090506.123555%26sid%3Dliteratum%253Aachs%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DFarr%26aufirst%3DG.%2BW.%26atitle%3DTwo%2520families%2520of%2520chaperonin%253A%2520physiology%2520and%2520mechanism%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D2007%26volume%3D23%26spage%3D115%26epage%3D145%26doi%3D10.1146%2Fannurev.cellbio.23.090506.123555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saibil, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clare, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Structure and allostery of the chaperonin GroEL</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">1476</span>– <span class="NLM_lpage">1487</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.jmb.2012.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=23183375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVylsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2013&pages=1476-1487&author=H.+R.+Saibilauthor=W.+A.+Fentonauthor=D.+K.+Clareauthor=A.+L.+Horwich&title=Structure+and+allostery+of+the+chaperonin+GroEL&doi=10.1016%2Fj.jmb.2012.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Allostery of the Chaperonin GroEL</span></div><div class="casAuthors">Saibil, Helen R.; Fenton, Wayne A.; Clare, Daniel K.; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1476-1487</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Chaperonins are intricate allosteric machines formed of two back-to-back, stacked rings of subunits presenting end cavities lined with hydrophobic binding sites for nonnative polypeptides.  Once bound, substrates are subjected to forceful, concerted movements that result in their ejection from the binding surface and simultaneous encapsulation inside a hydrophilic chamber that favors their folding.  Here, we review the allosteric machine movements that are choreographed by ATP binding, which triggers concerted tilting and twisting of subunit domains.  These movements distort the ring of hydrophobic binding sites and split it apart, potentially unfolding the multiply bound substrate.  Then, GroES binding is accompanied by a 100° twist of the binding domains that removes the hydrophobic sites from the cavity lining and forms the folding chamber.  ATP hydrolysis is not needed for a single round of binding and encapsulation but is necessary to allow the next round of ATP binding in the opposite ring.  It is this remote ATP binding that triggers dismantling of the folding chamber and release of the encapsulated substrate, whether folded or not.  The basis for these ordered actions is an elegant system of nested cooperativity of the ATPase machinery.  ATP binds to a ring with pos. cooperativity, and movements of the interlinked subunit domains are concerted.  In contrast, there is neg. cooperativity between the rings, so that they act in alternation.  It is remarkable that a process as specific as protein folding can be guided by the chaperonin machine in a way largely independent of substrate protein structure or sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTPmznXddOjrVg90H21EOLACvtfcHk0lgS3v6m9IcdOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVylsrjF&md5=b070d6535d8e5b7df1bfea2c0c009b5b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DSaibil%26aufirst%3DH.%2BR.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DClare%26aufirst%3DD.%2BK.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520and%2520allostery%2520of%2520the%2520chaperonin%2520GroEL%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D425%26spage%3D1476%26epage%3D1487%26doi%3D10.1016%2Fj.jmb.2012.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farr, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usaite, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hondorp, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pypaert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Global aggregation of newly translated proteins in an Escherichia coli strain deficient of the chaperonin GroEL</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">15800</span>– <span class="NLM_lpage">15805</span>, <span class="refDoi"> DOI: 10.1073/pnas.0607534103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1073%2Fpnas.0607534103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=17043235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFymtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=15800-15805&author=E.+Chapmanauthor=G.+W.+Farrauthor=R.+Usaiteauthor=K.+Furtakauthor=W.+A.+Fentonauthor=T.+K.+Chaudhuriauthor=E.+R.+Hondorpauthor=R.+G.+Matthewsauthor=S.+G.+Wolfauthor=J.+R.+Yatesauthor=M.+Pypaertauthor=A.+L.+Horwich&title=Global+aggregation+of+newly+translated+proteins+in+an+Escherichia+coli+strain+deficient+of+the+chaperonin+GroEL&doi=10.1073%2Fpnas.0607534103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Global aggregation of newly translated proteins in an Escherichia coli strain deficient of the chaperonin GroEL</span></div><div class="casAuthors">Chapman, Eli; Farr, George W.; Usaite, Renata; Furtak, Krystyna; Fenton, Wayne A.; Chaudhuri, Tapan K.; Hondorp, Elise R.; Matthews, Rowena G.; Wolf, Sharon G.; Yates, John R.; Pypaert, Marc; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15800-15805</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In a newly isolated temp.-sensitive lethal Escherichia coli mutant affecting the chaperonin GroEL, we obsd. wholesale aggregation of newly translated proteins.  After temp. shift, transcription, translation, and growth slowed over two to three generations, accompanied by filamentation and accretion (in ≈2% of cells) of paracryst. arrays contg. mutant chaperonin complex.  A biochem. isolated inclusion body fraction contained the collective of abundant proteins of the bacterial cytoplasm as detd. by SDS/PAGE and proteolysis/MS analyses.  Pulse-chase expts. revealed that newly made proteins, but not preexistent ones, were recruited to this insol. fraction.  Although aggregation of "stringent" GroEL/GroES-dependent substrates may secondarily produce an "avalanche" of aggregation, the observations raise the possibility, supported by in vitro refolding expts., that the widespread aggregation reflects that GroEL function supports the proper folding of a majority of newly translated polypeptides, not just the limited no. indicated by interaction studies and in vitro expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCPMpZGkZW77Vg90H21EOLACvtfcHk0lhRtJb0RbuqFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFymtbbL&md5=d153b49e89b7c2d3251b0d7d5b21c419</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0607534103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0607534103%26sid%3Dliteratum%253Aachs%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DFarr%26aufirst%3DG.%2BW.%26aulast%3DUsaite%26aufirst%3DR.%26aulast%3DFurtak%26aufirst%3DK.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DChaudhuri%26aufirst%3DT.%2BK.%26aulast%3DHondorp%26aufirst%3DE.%2BR.%26aulast%3DMatthews%26aufirst%3DR.%2BG.%26aulast%3DWolf%26aufirst%3DS.%2BG.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DPypaert%26aufirst%3DM.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520aggregation%2520of%2520newly%2520translated%2520proteins%2520in%2520an%2520Escherichia%2520coli%2520strain%2520deficient%2520of%2520the%2520chaperonin%2520GroEL%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D15800%26epage%3D15805%26doi%3D10.1073%2Fpnas.0607534103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fayet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelhoffer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgopoulos, C.</span></span> <span> </span><span class="NLM_article-title">The groES and groEL heat shock gene products of Escherichia coli are essential for bacterial growth at all temperatures</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1385</span>, <span class="refDoi"> DOI: 10.1128/jb.171.3.1379-1385.1989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1128%2Fjb.171.3.1379-1385.1989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=2563997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaL1MXhs1OjtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=1989&pages=1379-1385&author=O.+Fayetauthor=T.+Ziegelhofferauthor=C.+Georgopoulos&title=The+groES+and+groEL+heat+shock+gene+products+of+Escherichia+coli+are+essential+for+bacterial+growth+at+all+temperatures&doi=10.1128%2Fjb.171.3.1379-1385.1989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The groES and groEL heat shock gene products of Escherichia coli are essential for bacterial growth at all temperatures</span></div><div class="casAuthors">Fayet, Olivier; Ziegelhoffer, Thomas; Georgopoulos, Costa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1379-85</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    </div><div class="casAbstract">The products of the groES and groEL genes of E. coli, constituting the groE operon, are known to be required for growth at high temp. (42°) and are members of the heat shock regulon.  Using a genetic approach, the requirement for these gene products for bacterial growth at low temp. (17 to 30°) was investigated.  To do this, various groES groEL heterodiploid deriv. strains were constructed.  By inactivating 1 of the groE operons by a polar insertion, it was shown by bacteriophage P1 transduction that at least 1 of the groE genes was essential for growth at low temp.  Further P1 transduction expts. with strains that were heterodiploid for only 1 of the groE genes demonstrated that both groE gene products were required for growth at low temp., which suggested a fundamental role for the groE proteins in E. coli growth and physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9Cn9oLtKQLVg90H21EOLACvtfcHk0lhRtJb0RbuqFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhs1OjtL8%253D&md5=5ffafff6732ccfd78cfa42d82432ce75</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2Fjb.171.3.1379-1385.1989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.171.3.1379-1385.1989%26sid%3Dliteratum%253Aachs%26aulast%3DFayet%26aufirst%3DO.%26aulast%3DZiegelhoffer%26aufirst%3DT.%26aulast%3DGeorgopoulos%26aufirst%3DC.%26atitle%3DThe%2520groES%2520and%2520groEL%2520heat%2520shock%2520gene%2520products%2520of%2520Escherichia%2520coli%2520are%2520essential%2520for%2520bacterial%2520growth%2520at%2520all%2520temperatures%26jtitle%3DJ.%2520Bacteriol.%26date%3D1989%26volume%3D171%26spage%3D1379%26epage%3D1385%26doi%3D10.1128%2Fjb.171.3.1379-1385.1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharif, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span> <span> </span><span class="NLM_article-title">A biochemical screen for GroEL/GroES inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.12.100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2013.12.100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=24418775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Oguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=786-789&author=S.+M.+Johnsonauthor=O.+Sharifauthor=P.+A.+Makauthor=H.+T.+Wangauthor=I.+H.+Engelsauthor=A.+Brinkerauthor=P.+G.+Schultzauthor=A.+L.+Horwichauthor=E.+Chapman&title=A+biochemical+screen+for+GroEL%2FGroES+inhibitors&doi=10.1016%2Fj.bmcl.2013.12.100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A biochemical screen for GroEL/GroES inhibitors</span></div><div class="casAuthors">Johnson, Steven M.; Sharif, Orzala; Mak, Puiying Annie; Wang, Hsiao-Ting; Engels, Ingo H.; Brinker, Achim; Schultz, Peter G.; Horwich, Arthur L.; Chapman, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">786-789</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">High-throughput screening of 700,000 small mols. has identified 235 inhibitors of the GroEL/GroES-mediated refolding cycle.  Dose-response anal. of a subset of these hits revealed that 21 compds. are potent inhibitors of GroEL/GroES-mediated refolding (IC50 <10 μM).  The screening results presented herein represent the first steps in a broader aim of developing mol. probes to study chaperonin biochem. and physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8ZxlFMFjvarVg90H21EOLACvtfcHk0lhRtJb0RbuqFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Oguw%253D%253D&md5=babc99bf5fc9a4f1b9398bcc64014fa5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.12.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.12.100%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DSharif%26aufirst%3DO.%26aulast%3DMak%26aufirst%3DP.%2BA.%26aulast%3DWang%26aufirst%3DH.%2BT.%26aulast%3DEngels%26aufirst%3DI.%2BH.%26aulast%3DBrinker%26aufirst%3DA.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DChapman%26aufirst%3DE.%26atitle%3DA%2520biochemical%2520screen%2520for%2520GroEL%252FGroES%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D786%26epage%3D789%26doi%3D10.1016%2Fj.bmcl.2013.12.100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdeen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mammadova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrose, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span> <span> </span><span class="NLM_article-title">GroEL/ES inhibitors as potential antibiotics</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3127</span>– <span class="NLM_lpage">3134</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2016.04.089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=27184767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC28XotV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=3127-3134&author=S.+Abdeenauthor=N.+Salimauthor=N.+Mammadovaauthor=C.+M.+Summersauthor=R.+Franksonauthor=A.+J.+Ambroseauthor=G.+G.+Andersonauthor=P.+G.+Schultzauthor=A.+L.+Horwichauthor=E.+Chapmanauthor=S.+M.+Johnson&title=GroEL%2FES+inhibitors+as+potential+antibiotics&doi=10.1016%2Fj.bmcl.2016.04.089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GroEL/ES inhibitors as potential antibiotics</span></div><div class="casAuthors">Abdeen, Sanofar; Salim, Nilshad; Mammadova, Najiba; Summers, Corey M.; Frankson, Rochelle; Ambrose, Andrew J.; Anderson, Gregory G.; Schultz, Peter G.; Horwich, Arthur L.; Chapman, Eli; Johnson, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3127-3134</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We recently reported results from a high-throughput screening effort that identified 235 inhibitors of the Escherichia coli GroEL/ES chaperonin system [Bioorg. Med. Chem. Lett.2014, 24, 786].  As the GroEL/ES chaperonin system is essential for growth under all conditions, we reasoned that targeting GroEL/ES with small mol. inhibitors could be a viable antibacterial strategy.  Extending from our initial screen, we report here the antibacterial activities of 22 GroEL/ES inhibitors against a panel of Gram-pos. and Gram-neg. bacteria, including E. coli, Bacillus subtilis, Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter cloacae.  GroEL/ES inhibitors were more effective at blocking the proliferation of Gram-pos. bacteria, in particular S. aureus, where lead compds. exhibited antibiotic effects from the low-μM to mid-nM range.  While several compds. inhibited the human HSP60/10 refolding cycle, some were able to selectively target the bacterial GroEL/ES system.  Despite inhibiting HSP60/10, many compds. exhibited low to no cytotoxicity against human liver and kidney cell lines.  Two lead candidates emerged from the panel, that exhibit >50-fold selectivity for inhibiting S. aureus growth compared to liver or kidney cell cytotoxicity.  These compds. inhibited drug-sensitive and methicillin-resistant S. aureus strains with potencies comparable to vancomycin, daptomycin, and streptomycin, and are promising candidates to explore for validating the GroEL/ES chaperonin system as a viable antibiotic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrazQ1vHFwVlrVg90H21EOLACvtfcHk0ljSH0wQLCPQjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotV2gsrk%253D&md5=d2bfc6eb7a2a02d3f91961f13b22bff3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.089%26sid%3Dliteratum%253Aachs%26aulast%3DAbdeen%26aufirst%3DS.%26aulast%3DSalim%26aufirst%3DN.%26aulast%3DMammadova%26aufirst%3DN.%26aulast%3DSummers%26aufirst%3DC.%2BM.%26aulast%3DFrankson%26aufirst%3DR.%26aulast%3DAmbrose%26aufirst%3DA.%2BJ.%26aulast%3DAnderson%26aufirst%3DG.%2BG.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26atitle%3DGroEL%252FES%2520inhibitors%2520as%2520potential%2520antibiotics%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D3127%26epage%3D3134%26doi%3D10.1016%2Fj.bmcl.2016.04.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdeen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mammadova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith-Pestana, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon-Pratt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the HSP60/10 chaperonin systems of Trypanosoma brucei as a strategy for treating African sleeping sickness</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5247</span>– <span class="NLM_lpage">5253</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2016.09.051" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5247-5253&author=S.+Abdeenauthor=N.+Salimauthor=N.+Mammadovaauthor=C.+M.+Summersauthor=K.+Goldsmith-Pestanaauthor=D.+McMahon-Prattauthor=P.+G.+Schultzauthor=A.+L.+Horwichauthor=E.+Chapmanauthor=S.+M.+Johnson&title=Targeting+the+HSP60%2F10+chaperonin+systems+of+Trypanosoma+brucei+as+a+strategy+for+treating+African+sleeping+sickness&doi=10.1016%2Fj.bmcl.2016.09.051"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.051%26sid%3Dliteratum%253Aachs%26aulast%3DAbdeen%26aufirst%3DS.%26aulast%3DSalim%26aufirst%3DN.%26aulast%3DMammadova%26aufirst%3DN.%26aulast%3DSummers%26aufirst%3DC.%2BM.%26aulast%3DGoldsmith-Pestana%26aufirst%3DK.%26aulast%3DMcMahon-Pratt%26aufirst%3DD.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26atitle%3DTargeting%2520the%2520HSP60%252F10%2520chaperonin%2520systems%2520of%2520Trypanosoma%2520brucei%2520as%2520a%2520strategy%2520for%2520treating%2520African%2520sleeping%2520sickness%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5247%26epage%3D5253%26doi%3D10.1016%2Fj.bmcl.2016.09.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilpert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E.</span></span> <span> </span><span class="NLM_article-title">Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1038%2Fnprot.2007.521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=18274517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvVyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=163-175&author=I.+Wiegandauthor=K.+Hilpertauthor=R.+E.+Hancock&title=Agar+and+broth+dilution+methods+to+determine+the+minimal+inhibitory+concentration+%28MIC%29+of+antimicrobial+substances&doi=10.1038%2Fnprot.2007.521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances</span></div><div class="casAuthors">Wiegand, Irith; Hilpert, Kai; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-175</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The aim of broth and agar diln. methods is to det. the lowest concn. of the assayed antimicrobial agent (minimal inhibitory concn., MIC) that, under defined test conditions, inhibits the visible growth of the bacterium being investigated.  MIC values are used to det. susceptibilities of bacteria to drugs and also to evaluate the activity of new antimicrobial agents.  Agar diln. involves the incorporation of different concns. of the antimicrobial substance into a nutrient agar medium followed by the application of a standardized no. of cells to the surface of the agar plate.  For broth diln., often detd. in 96-well microtiter plate format, bacteria are inoculated into a liq. growth medium in the presence of different concns. of an antimicrobial agent.  Growth is assessed after incubation for a defined period of time (16-20 h) and the MIC value is read.  This protocol applies only to aerobic bacteria and can be completed in 3 d.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd1uwvRDLPQ7Vg90H21EOLACvtfcHk0ljSH0wQLCPQjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvVyls78%253D&md5=a235d207868342de52085d4100062471</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.521%26sid%3Dliteratum%253Aachs%26aulast%3DWiegand%26aufirst%3DI.%26aulast%3DHilpert%26aufirst%3DK.%26aulast%3DHancock%26aufirst%3DR.%2BE.%26atitle%3DAgar%2520and%2520broth%2520dilution%2520methods%2520to%2520determine%2520the%2520minimal%2520inhibitory%2520concentration%2520%2528MIC%2529%2520of%2520antimicrobial%2520substances%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D163%26epage%3D175%26doi%3D10.1038%2Fnprot.2007.521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Toward optimization of the linker substructure common to transthyretin amyloidogenesis inhibitors using biochemical and structural studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6348</span>– <span class="NLM_lpage">6358</span>, <span class="refDoi"> DOI: 10.1021/jm800435s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800435s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFGntrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6348-6358&author=S.+M.+Johnsonauthor=S.+Connellyauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Toward+optimization+of+the+linker+substructure+common+to+transthyretin+amyloidogenesis+inhibitors+using+biochemical+and+structural+studies&doi=10.1021%2Fjm800435s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Optimization of the Linker Substructure Common to Transthyretin Amyloidogenesis Inhibitors Using Biochemical and Structural Studies</span></div><div class="casAuthors">Johnson, Steven M.; Connelly, Stephen; Wilson, Ian A.; Kelly, Jeffery W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6348-6358</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To develop potent and highly selective transthyretin (TTR) amyloidogenesis inhibitors, it is useful to systematically optimize the three substructural elements that compose a typical TTR kinetic stabilizer: the two aryl rings and the linker joining them.  Herein, the authors evaluated 40 bisaryl mols. based on 10 unique linker substructures to det. how these linkages influence inhibitor potency and selectivity.  These linkers connect one unsubstituted arom. ring to either a 3,5-X2 or a 3,5-X2-4-OH Ph substructure (X = Br or CH3).  Consideration of amyloid inhibition and ex vivo plasma TTR binding selectivity data reveal that direct connection of the two aryls or linkage through nonpolar E-olefin or -CH2CH2- substructures generates the most potent and selective TTR amyloidogenesis inhibitors exhibiting minimal undesirable binding to the thyroid hormone nuclear receptor or the COX-1 enzyme.  Five high-resoln. TTR·inhibitor crystal structures (1.4-1.8 Å) provide insight into why such linkers afford inhibitors with greater potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI-adFkm-anbVg90H21EOLACvtfcHk0ljujwidz_GkEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFGntrvL&md5=43789d84a079d73268685760f36262c3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm800435s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800435s%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DToward%2520optimization%2520of%2520the%2520linker%2520substructure%2520common%2520to%2520transthyretin%2520amyloidogenesis%2520inhibitors%2520using%2520biochemical%2520and%2520structural%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6348%26epage%3D6358%26doi%3D10.1021%2Fjm800435s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span> <span> </span><span class="NLM_article-title">Semi-quantitative models for identifying potent and selective transthyretin amyloidogenesis inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3441</span>– <span class="NLM_lpage">3449</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2017.05.080" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3441-3449&author=S.+Connellyauthor=D.+E.+Mortensonauthor=S.+Choiauthor=I.+A.+Wilsonauthor=E.+T.+Powersauthor=J.+W.+Kellyauthor=S.+M.+Johnson&title=Semi-quantitative+models+for+identifying+potent+and+selective+transthyretin+amyloidogenesis+inhibitors&doi=10.1016%2Fj.bmcl.2017.05.080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.080%26sid%3Dliteratum%253Aachs%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DMortenson%26aufirst%3DD.%2BE.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DPowers%26aufirst%3DE.%2BT.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26atitle%3DSemi-quantitative%2520models%2520for%2520identifying%2520potent%2520and%2520selective%2520transthyretin%2520amyloidogenesis%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3441%26epage%3D3449%26doi%3D10.1016%2Fj.bmcl.2017.05.080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Toward optimization of the second aryl substructure common to transthyretin amyloidogenesis inhibitors using biochemical and structural studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1125</span>, <span class="refDoi"> DOI: 10.1021/jm801347s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801347s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1115-1125&author=S.+M.+Johnsonauthor=S.+Connellyauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Toward+optimization+of+the+second+aryl+substructure+common+to+transthyretin+amyloidogenesis+inhibitors+using+biochemical+and+structural+studies&doi=10.1021%2Fjm801347s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Optimization of the Second Aryl Substructure Common to Transthyretin Amyloidogenesis Inhibitors Using Biochemical and Structural Studies</span></div><div class="casAuthors">Johnson, Steven M.; Connelly, Stephen; Wilson, Ian A.; Kelly, Jeffery W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1115-1125</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Transthyretin (TTR) amyloidogenesis inhibitors are typically composed of two arom. rings and a linker.  We have previously established optimal structures for one arom. ring and the linker.  Herein, we employ a suboptimal linker and an optimal aryl-X substructure to rank order the desirability of aryl-Z substructures-using a library of 56 N-(3,5-dibromo-4-hydroxyphenyl)benzamides.  Coconsideration of amyloid inhibition potency and ex vivo plasma TTR binding selectivity data reveal that 2,6, 2,5, 2, 3,4,5, and 3,5 substituted aryls bearing small substituents generate the most potent and selective inhibitors, in descending order.  These benzamides generally lack undesirable thyroid hormone receptor binding and COX-1 inhibition activity.  Three high-resoln. TTR·inhibitor crystal structures (1.31-1.35 Å) provide insight into why these inhibitors are potent and selective, enabling future structure-based design of TTR kinetic stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCf9iuLPAegLVg90H21EOLACvtfcHk0ljujwidz_GkEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSltLg%253D&md5=77591581a43967e91119239f94302306</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm801347s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801347s%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DToward%2520optimization%2520of%2520the%2520second%2520aryl%2520substructure%2520common%2520to%2520transthyretin%2520amyloidogenesis%2520inhibitors%2520using%2520biochemical%2520and%2520structural%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1115%26epage%3D1125%26doi%3D10.1021%2Fjm801347s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tipparaju, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyasawal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">In pursuit of natural product leads: Synthesis and biological evaluation of 2-[3-hydroxy-2-[(3-hydroxypyridine-2-carbonyl)amino]phenyl]benzoxazole-4-carboxylic acid (A-33853) and its analogues: Discovery of N-(2-benzoxazol-2-ylphenyl)benzamides as novel antileishmanial chemotypes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7344</span>– <span class="NLM_lpage">7347</span>, <span class="refDoi"> DOI: 10.1021/jm801241n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801241n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7344-7347&author=S.+K.+Tipparajuauthor=S.+Joyasawalauthor=M.+Pieroniauthor=M.+Kaiserauthor=R.+Brunauthor=A.+P.+Kozikowski&title=In+pursuit+of+natural+product+leads%3A+Synthesis+and+biological+evaluation+of+2-%5B3-hydroxy-2-%5B%283-hydroxypyridine-2-carbonyl%29amino%5Dphenyl%5Dbenzoxazole-4-carboxylic+acid+%28A-33853%29+and+its+analogues%3A+Discovery+of+N-%282-benzoxazol-2-ylphenyl%29benzamides+as+novel+antileishmanial+chemotypes&doi=10.1021%2Fjm801241n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm801241n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801241n%26sid%3Dliteratum%253Aachs%26aulast%3DTipparaju%26aufirst%3DS.%2BK.%26aulast%3DJoyasawal%26aufirst%3DS.%26aulast%3DPieroni%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DIn%2520pursuit%2520of%2520natural%2520product%2520leads%253A%2520Synthesis%2520and%2520biological%2520evaluation%2520of%25202-%255B3-hydroxy-2-%255B%25283-hydroxypyridine-2-carbonyl%2529amino%255Dphenyl%255Dbenzoxazole-4-carboxylic%2520acid%2520%2528A-33853%2529%2520and%2520its%2520analogues%253A%2520Discovery%2520of%2520N-%25282-benzoxazol-2-ylphenyl%2529benzamides%2520as%2520novel%2520antileishmanial%2520chemotypes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7344%26epage%3D7347%26doi%3D10.1021%2Fjm801241n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, R.
L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelker, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Kinetic stabilization of an oligomeric protein by a single ligand binding event</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">5540</span>– <span class="NLM_lpage">5551</span>, <span class="refDoi"> DOI: 10.1021/ja042929f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja042929f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD2MXis1Ohsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=5540-5551&author=R.%0AL.+Wisemanauthor=S.+M.+Johnsonauthor=M.+S.+Kelkerauthor=T.+Fossauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Kinetic+stabilization+of+an+oligomeric+protein+by+a+single+ligand+binding+event&doi=10.1021%2Fja042929f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Stabilization of an Oligomeric Protein by a Single Ligand Binding Event</span></div><div class="casAuthors">Wiseman, R. Luke; Johnson, Steven M.; Kelker, Matthew S.; Foss, Ted; Wilson, Ian A.; Kelly, Jeffery W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5540-5551</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein native state stabilization imposed by small mol. binding is an attractive strategy to prevent the misfolding and misassembly processes assocd. with amyloid diseases.  Transthyretin (TTR) amyloidogenesis requires rate-limiting tetramer dissocn. before misassembly of a partially denatured monomer ensues.  Selective stabilization of the native TTR tetramer over the dissociative transition state by small mol. binding to both thyroxine binding sites raises the kinetic barrier of tetramer dissocn., preventing amyloidogenesis.  Assessing the amyloidogenicity of a TTR tetramer having only one amyloidogenesis inhibitor (I) bound is challenging because the two small mol. binding consts. are generally not distinct enough to allow for the exclusive formation of TTR·I in soln. to the exclusion of TTR·I2 and unliganded TTR.  Herein, we report a method to tether one fibril formation inhibitor to TTR by disulfide bond formation.  Occupancy of only one of the two thyroxine binding sites is sufficient to inhibit tetramer dissocn. in 6.0 M urea and amyloidogenesis under acidic conditions by imposing kinetic stabilization on the entire tetramer.  The sufficiency of single occupancy for stabilizing the native state of TTR provides the incentive to search for compds. displaying striking neg. binding cooperativity (e.g., Kd1 in nanomolar range and Kd2 in the micromolar to millimolar range), enabling lower doses of inhibitor to be employed in the clinic, mitigating potential side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq6dHdnympB7Vg90H21EOLACvtfcHk0lgNrekESlkMOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXis1Ohsb4%253D&md5=a027c65727cfb52014d9dca606417915</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja042929f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042929f%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DR.%2BL.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DKelker%26aufirst%3DM.%2BS.%26aulast%3DFoss%26aufirst%3DT.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DKinetic%2520stabilization%2520of%2520an%2520oligomeric%2520protein%2520by%2520a%2520single%2520ligand%2520binding%2520event%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D5540%26epage%3D5551%26doi%3D10.1021%2Fja042929f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Biochemical and structural evaluation of highly selective 2-arylbenzoxazole-based transthyretin amyloidogenesis inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">270</span>, <span class="refDoi"> DOI: 10.1021/jm0708735</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0708735" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=260-270&author=S.+M.+Johnsonauthor=S.+Connellyauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Biochemical+and+structural+evaluation+of+highly+selective+2-arylbenzoxazole-based+transthyretin+amyloidogenesis+inhibitors&doi=10.1021%2Fjm0708735"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm0708735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0708735%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DBiochemical%2520and%2520structural%2520evaluation%2520of%2520highly%2520selective%25202-arylbenzoxazole-based%2520transthyretin%2520amyloidogenesis%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D260%26epage%3D270%26doi%3D10.1021%2Fjm0708735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. N.</span></span> <span> </span><span class="NLM_article-title">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.+N.+Lin&title=Rapid+measurement+of+binding+constants+and+heats+of+binding+using+a+new+titration+calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0lgNrekESlkMOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.%2BN.%26atitle%3DRapid%2520measurement%2520of%2520binding%2520constants%2520and%2520heats%2520of%2520binding%2520using%2520a%2520new%2520titration%2520calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Alternating antibiotic treatments constrain evolutionary paths to multidrug resistance</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">14494</span>– <span class="NLM_lpage">14499</span>, <span class="refDoi"> DOI: 10.1073/pnas.1409800111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1073%2Fpnas.1409800111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=25246554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=14494-14499&author=S.+Kimauthor=T.+D.+Liebermanauthor=R.+Kishony&title=Alternating+antibiotic+treatments+constrain+evolutionary+paths+to+multidrug+resistance&doi=10.1073%2Fpnas.1409800111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Alternating antibiotic treatments constrain evolutionary paths to multidrug resistance</span></div><div class="casAuthors">Kim, Seungsoo; Lieberman, Tami D.; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">14494-14499</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Alternating antibiotic therapy, in which pairs of drugs are cycled during treatment, has been suggested as a means to inhibit the evolution of de novo resistance while avoiding the toxicity assocd. with more traditional combination therapy.  However, it remains unclear under which conditions and by what means such alternating treatments impede the evolution of resistance.  Here, the authors tracked multistep evolution of resistance in replicate populations of Staphylococcus aureus during 22 d of continuously increasing single, mixed, and alternating drug treatment.  In all three tested drug pairs, the alternating treatment reduced the overall rate of resistance by slowing the acquisition of resistance to one of the two component drugs, sometimes as effectively as mixed treatment.  This slower rate of evolution is reflected in the genome-wide mutational profiles; under alternating treatments, bacteria acquire mutations in different genes than under corresponding single-drug treatments.  To test whether this obsd. constraint on adaptive paths reflects trade-offs in which resistance to one drug is accompanied by sensitivity to a second drug, we profiled many single-step mutants for cross-resistance.  Indeed, the av. cross-resistance of single-step mutants can help predict whether or not evolution was slower in alternating drugs.  Together, these results show that despite the complex evolutionary landscape of multidrug resistance, alternating-drug therapy can slow evolution by constraining the mutational paths toward resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1oDn_1SS_IbVg90H21EOLACvtfcHk0lhJHlqPFKnUXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtbbL&md5=5b019b5e5f74aa9014f12a891f93bbaa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1409800111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1409800111%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLieberman%26aufirst%3DT.%2BD.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DAlternating%2520antibiotic%2520treatments%2520constrain%2520evolutionary%2520paths%2520to%2520multidrug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D14494%26epage%3D14499%26doi%3D10.1073%2Fpnas.1409800111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fleeman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoi, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nefzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welmaker, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giulianotti, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghten, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, L. N.</span></span> <span> </span><span class="NLM_article-title">Combinatorial libraries as a tool for the discovery of novel, broad-spectrum antibacterial agents targeting the ESKAPE pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3340</span>– <span class="NLM_lpage">3355</span>, <span class="refDoi"> DOI: 10.1021/jm501628s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501628s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVOrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3340-3355&author=R.+Fleemanauthor=T.+M.+LaVoiauthor=R.+G.+Santosauthor=A.+Moralesauthor=A.+Nefziauthor=G.+S.+Welmakerauthor=J.+L.+Medina-Francoauthor=M.+A.+Giulianottiauthor=R.+A.+Houghtenauthor=L.+N.+Shaw&title=Combinatorial+libraries+as+a+tool+for+the+discovery+of+novel%2C+broad-spectrum+antibacterial+agents+targeting+the+ESKAPE+pathogens&doi=10.1021%2Fjm501628s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial Libraries As a Tool for the Discovery of Novel, Broad-Spectrum Antibacterial Agents Targeting the ESKAPE Pathogens</span></div><div class="casAuthors">Fleeman, Renee; LaVoi, Travis M.; Santos, Radleigh G.; Morales, Angela; Nefzi, Adel; Welmaker, Gregory S.; Medina-Franco, Jose L.; Giulianotti, Marc A.; Houghten, Richard A.; Shaw, Lindsey N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3340-3355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mixt. based synthetic combinatorial libraries offer a tremendous enhancement for the rate of drug discovery, allowing the activity of millions of compds. to be assessed through the testing of exponentially fewer samples.  In this study, the authors used a scaffold-ranking library to screen 37 different libraries for antibacterial activity against the ESKAPE pathogens.  Each library contained between 10000 and 750000 structural analogs for a total of >6 million compds.  From this, the authors identified a bis-cyclic guanidine library that displayed strong antibacterial activity.  A positional scanning library for these compds. was developed and used to identify the most effective functional groups at each variant position.  Individual compds. were synthesized that were broadly active against all ESKAPE organisms at concns. <2 μM.  In addn., these compds. were bactericidal, had antibiofilm effects, showed limited potential for the development of resistance, and displayed almost no toxicity when tested against human lung cells and erythrocytes.  Using a murine model of peritonitis, the authors also demonstrate that these agents are highly efficacious in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQZewS666QerVg90H21EOLACvtfcHk0lhJHlqPFKnUXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVOrtro%253D&md5=a8c23a0c8c31166c5caedf632b71348f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm501628s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501628s%26sid%3Dliteratum%253Aachs%26aulast%3DFleeman%26aufirst%3DR.%26aulast%3DLaVoi%26aufirst%3DT.%2BM.%26aulast%3DSantos%26aufirst%3DR.%2BG.%26aulast%3DMorales%26aufirst%3DA.%26aulast%3DNefzi%26aufirst%3DA.%26aulast%3DWelmaker%26aufirst%3DG.%2BS.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DGiulianotti%26aufirst%3DM.%2BA.%26aulast%3DHoughten%26aufirst%3DR.%2BA.%26aulast%3DShaw%26aufirst%3DL.%2BN.%26atitle%3DCombinatorial%2520libraries%2520as%2520a%2520tool%2520for%2520the%2520discovery%2520of%2520novel%252C%2520broad-spectrum%2520antibacterial%2520agents%2520targeting%2520the%2520ESKAPE%2520pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3340%26epage%3D3355%26doi%3D10.1021%2Fjm501628s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shuimu Lin, Jiayong Liu, Hongxia Li, Ying Liu, Yongzhi Chen, Jiachun Luo, <span class="NLM_string-name hlFld-ContribAuthor">Shouping Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Highly Potent Carbazole Amphiphiles as Membrane-Targeting Antimicrobials for Treating Gram-Positive Bacterial Infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9284-9299. <a href="https://doi.org/10.1021/acs.jmedchem.0c00433" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00433</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00433%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252BHighly%252BPotent%252BCarbazole%252BAmphiphiles%252Bas%252BMembrane-Targeting%252BAntimicrobials%252Bfor%252BTreating%252BGram-Positive%252BBacterial%252BInfections%26aulast%3DLin%26aufirst%3DShuimu%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14032020%26date%3D01092020%26date%3D12082020%26volume%3D63%26issue%3D17%26spage%3D9284%26epage%3D9299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Taejung Kim, Sin-Ae Lee, Taesub Noh, Pilju Choi, Seon-Jun Choi, Bong Geun Song, Youngseok Kim, Young-Tae Park, Gyuwon Huh, Young-Joo Kim, <span class="NLM_string-name hlFld-ContribAuthor">Jungyeob Ham</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, Structure Revision, and Cytotoxicity of Nocarbenzoxazole G. </span><span class="cited-content_cbyCitation_journal-name">Journal of Natural Products</span><span> <strong>2019,</strong> <em>82 </em>
                                    (5)
                                     , 1325-1330. <a href="https://doi.org/10.1021/acs.jnatprod.9b00072" title="DOI URL">https://doi.org/10.1021/acs.jnatprod.9b00072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jnatprod.9b00072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jnatprod.9b00072%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Natural%2520Products%26atitle%3DSynthesis%25252C%252BStructure%252BRevision%25252C%252Band%252BCytotoxicity%252Bof%252BNocarbenzoxazole%252BG%26aulast%3DKim%26aufirst%3DTaejung%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D25012019%26date%3D08042019%26volume%3D82%26issue%3D5%26spage%3D1325%26epage%3D1330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Trent Kunkle, Sanofar Abdeen, Nilshad Salim, Anne-Marie Ray, Mckayla Stevens, Andrew J. Ambrose, José Victorino, Yangshin Park, Quyen Q. Hoang, Eli Chapman, <span class="NLM_string-name hlFld-ContribAuthor">Steven M. Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Hydroxybiphenylamide GroEL/ES Inhibitors Are Potent Antibacterials against Planktonic and Biofilm Forms of Staphylococcus aureus. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (23)
                                     , 10651-10664. <a href="https://doi.org/10.1021/acs.jmedchem.8b01293" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01293</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01293%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHydroxybiphenylamide%252BGroEL%25252FES%252BInhibitors%252BAre%252BPotent%252BAntibacterials%252Bagainst%252BPlanktonic%252Band%252BBiofilm%252BForms%252Bof%252BStaphylococcus%252Baureus%26aulast%3DKunkle%26aufirst%3DTrent%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D15082018%26date%3D15112018%26date%3D04112018%26volume%3D61%26issue%3D23%26spage%3D10651%26epage%3D10664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anne-Marie  Ray</span>, <span class="hlFld-ContribAuthor ">Nilshad  Salim</span>, <span class="hlFld-ContribAuthor ">Mckayla  Stevens</span>, <span class="hlFld-ContribAuthor ">Siddhi  Chitre</span>, <span class="hlFld-ContribAuthor ">Sanofar  Abdeen</span>, <span class="hlFld-ContribAuthor ">Alex  Washburn</span>, <span class="hlFld-ContribAuthor ">Jared  Sivinski</span>, <span class="hlFld-ContribAuthor ">Heather M.  O'Hagan</span>, <span class="hlFld-ContribAuthor ">Eli  Chapman</span>, <span class="hlFld-ContribAuthor ">Steven M.  Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting the HSP60/10 chaperonin system as a chemotherapeutic target for colorectal cancer. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>40 </em>, 116129. <a href="https://doi.org/10.1016/j.bmc.2021.116129" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116129%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DExploiting%252Bthe%252BHSP60%25252F10%252Bchaperonin%252Bsystem%252Bas%252Ba%252Bchemotherapeutic%252Btarget%252Bfor%252Bcolorectal%252Bcancer%26aulast%3DRay%26aufirst%3DAnne-Marie%26date%3D2021%26volume%3D40%26spage%3D116129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hakimeh  Teymourinia</span>, <span class="hlFld-ContribAuthor ">Omid  Amiri</span>, <span class="hlFld-ContribAuthor ">Masoud  Salavati-Niasari</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and characterization of cotton-silver-graphene quantum dots (cotton/Ag/GQDs) nanocomposite as a new antibacterial nanopad. </span><span class="cited-content_cbyCitation_journal-name">Chemosphere</span><span> <strong>2021,</strong> <em>267 </em>, 129293. <a href="https://doi.org/10.1016/j.chemosphere.2020.129293" title="DOI URL">https://doi.org/10.1016/j.chemosphere.2020.129293</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chemosphere.2020.129293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chemosphere.2020.129293%26sid%3Dliteratum%253Aachs%26jtitle%3DChemosphere%26atitle%3DSynthesis%252Band%252Bcharacterization%252Bof%252Bcotton-silver-graphene%252Bquantum%252Bdots%252B%252528cotton%25252FAg%25252FGQDs%252529%252Bnanocomposite%252Bas%252Ba%252Bnew%252Bantibacterial%252Bnanopad%26aulast%3DTeymourinia%26aufirst%3DHakimeh%26date%3D2021%26volume%3D267%26spage%3D129293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jared  Sivinski</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Ambrose</span>, <span class="hlFld-ContribAuthor ">Iliya  Panfilenko</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Zerio</span>, <span class="hlFld-ContribAuthor ">Jason M.  Machulis</span>, <span class="hlFld-ContribAuthor ">Niloufar  Mollasalehi</span>, <span class="hlFld-ContribAuthor ">Lynn K.  Kaneko</span>, <span class="hlFld-ContribAuthor ">Mckayla  Stevens</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Ray</span>, <span class="hlFld-ContribAuthor ">Yangshin  Park</span>, <span class="hlFld-ContribAuthor ">Chunxiang  Wu</span>, <span class="hlFld-ContribAuthor ">Quyen Q.  Hoang</span>, <span class="hlFld-ContribAuthor ">Steven M.  Johnson</span>, <span class="hlFld-ContribAuthor ">Eli  Chapman</span>, . </span><span class="cited-content_cbyCitation_article-title">Functional Differences between E. coli and ESKAPE Pathogen GroES/GroEL. </span><span class="cited-content_cbyCitation_journal-name">mBio</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1128/mBio.02167-20" title="DOI URL">https://doi.org/10.1128/mBio.02167-20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/mBio.02167-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FmBio.02167-20%26sid%3Dliteratum%253Aachs%26jtitle%3DmBio%26atitle%3DFunctional%252BDifferences%252Bbetween%252BE.%252Bcoli%252Band%252BESKAPE%252BPathogen%252BGroES%25252FGroEL%26aulast%3DSivinski%26aufirst%3DJared%26date%3D2021%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John  Bremner</span>. </span><span class="cited-content_cbyCitation_article-title">Future Possibilities. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 159-187. <a href="https://doi.org/10.1007/978-981-16-0999-2_5" title="DOI URL">https://doi.org/10.1007/978-981-16-0999-2_5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-16-0999-2_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-16-0999-2_5%26sid%3Dliteratum%253Aachs%26atitle%3DFuture%252BPossibilities%26aulast%3DBremner%26aufirst%3DJohn%26date%3D2021%26date%3D2021%26spage%3D159%26epage%3D187%26pub%3DSpringer%2520Singapore%26atitle%3DMultiple%252BAction-Based%252BDesign%252BApproaches%252Bto%252BAntibacterials%26aulast%3DBremner%26aufirst%3DJohn%26date%3D2021%26volume%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Santosh  Kumar Verma</span>, <span class="hlFld-ContribAuthor ">Rameshwari  Verma</span>, <span class="hlFld-ContribAuthor ">Fan  Xue</span>, <span class="hlFld-ContribAuthor ">Piyush  Kumar Thakur</span>, <span class="hlFld-ContribAuthor ">Y.R.  Girish</span>, <span class="hlFld-ContribAuthor ">K.P.  Rakesh</span>. </span><span class="cited-content_cbyCitation_article-title">Antibacterial activities of sulfonyl or sulfonamide containing heterocyclic derivatives and its structure-activity relationships (SAR) studies: A critical review. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>105 </em>, 104400. <a href="https://doi.org/10.1016/j.bioorg.2020.104400" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104400</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104400%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DAntibacterial%252Bactivities%252Bof%252Bsulfonyl%252Bor%252Bsulfonamide%252Bcontaining%252Bheterocyclic%252Bderivatives%252Band%252Bits%252Bstructure-activity%252Brelationships%252B%252528SAR%252529%252Bstudies%25253A%252BA%252Bcritical%252Breview%26aulast%3DKumar%2BVerma%26aufirst%3DSantosh%26date%3D2020%26volume%3D105%26spage%3D104400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhenyu  Yao</span>, <span class="hlFld-ContribAuthor ">Xing  Lin</span>, <span class="hlFld-ContribAuthor ">Remi  Chauvin</span>, <span class="hlFld-ContribAuthor ">Lianhui  Wang</span>, <span class="hlFld-ContribAuthor ">Emmanuel  Gras</span>, <span class="hlFld-ContribAuthor ">Xiuling  Cui</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphine-phosphonium ylides as ligands in palladium-catalysed C2-H arylation of benzoxazoles. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2020,</strong> <em>31 </em>
                                    (12)
                                     , 3250-3254. <a href="https://doi.org/10.1016/j.cclet.2020.04.008" title="DOI URL">https://doi.org/10.1016/j.cclet.2020.04.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2020.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2020.04.008%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DPhosphine-phosphonium%252Bylides%252Bas%252Bligands%252Bin%252Bpalladium-catalysed%252BC2-H%252Barylation%252Bof%252Bbenzoxazoles%26aulast%3DYao%26aufirst%3DZhenyu%26date%3D2020%26volume%3D31%26issue%3D12%26spage%3D3250%26epage%3D3254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mckayla  Stevens</span>, <span class="hlFld-ContribAuthor ">Chris  Howe</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Ray</span>, <span class="hlFld-ContribAuthor ">Alex  Washburn</span>, <span class="hlFld-ContribAuthor ">Siddhi  Chitre</span>, <span class="hlFld-ContribAuthor ">Jared  Sivinski</span>, <span class="hlFld-ContribAuthor ">Yangshin  Park</span>, <span class="hlFld-ContribAuthor ">Quyen Q.  Hoang</span>, <span class="hlFld-ContribAuthor ">Eli  Chapman</span>, <span class="hlFld-ContribAuthor ">Steven M.  Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Analogs of nitrofuran antibiotics are potent GroEL/ES inhibitor pro-drugs. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (22)
                                     , 115710. <a href="https://doi.org/10.1016/j.bmc.2020.115710" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115710</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115710%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DAnalogs%252Bof%252Bnitrofuran%252Bantibiotics%252Bare%252Bpotent%252BGroEL%25252FES%252Binhibitor%252Bpro-drugs%26aulast%3DStevens%26aufirst%3DMckayla%26date%3D2020%26volume%3D28%26issue%3D22%26spage%3D115710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Volodymyr  Horishny</span>, <span class="hlFld-ContribAuthor ">Victor  Kartsev</span>, <span class="hlFld-ContribAuthor ">Vasyl  Matiychuk</span>, <span class="hlFld-ContribAuthor ">Athina  Geronikaki</span>, <span class="hlFld-ContribAuthor ">Petrou  Anthi</span>, <span class="hlFld-ContribAuthor ">Pavel  Pogodin</span>, <span class="hlFld-ContribAuthor ">Vladimir  Poroikov</span>, <span class="hlFld-ContribAuthor ">Marija  Ivanov</span>, <span class="hlFld-ContribAuthor ">Marina  Kostic</span>, <span class="hlFld-ContribAuthor ">Marina D.  Soković</span>, <span class="hlFld-ContribAuthor ">Phaedra  Eleftheriou</span>. </span><span class="cited-content_cbyCitation_article-title">3-Amino-5-(indol-3-yl)methylene-4-oxo-2-thioxothiazolidine Derivatives as Antimicrobial Agents: Synthesis, Computational and Biological Evaluation. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2020,</strong> <em>13 </em>
                                    (9)
                                     , 229. <a href="https://doi.org/10.3390/ph13090229" title="DOI URL">https://doi.org/10.3390/ph13090229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph13090229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph13090229%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3D3-Amino-5-%252528indol-3-yl%252529methylene-4-oxo-2-thioxothiazolidine%252BDerivatives%252Bas%252BAntimicrobial%252BAgents%25253A%252BSynthesis%25252C%252BComputational%252Band%252BBiological%252BEvaluation%26aulast%3DHorishny%26aufirst%3DVolodymyr%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D9%26spage%3D229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giovanni  Stelitano</span>, <span class="hlFld-ContribAuthor ">José Camilla  Sammartino</span>, <span class="hlFld-ContribAuthor ">Laurent Roberto  Chiarelli</span>. </span><span class="cited-content_cbyCitation_article-title">Multitargeting Compounds: A Promising Strategy to Overcome Multi-Drug Resistant Tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (5)
                                     , 1239. <a href="https://doi.org/10.3390/molecules25051239" title="DOI URL">https://doi.org/10.3390/molecules25051239</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25051239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25051239%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMultitargeting%252BCompounds%25253A%252BA%252BPromising%252BStrategy%252Bto%252BOvercome%252BMulti-Drug%252BResistant%252BTuberculosis%26aulast%3DStelitano%26aufirst%3DGiovanni%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D5%26spage%3D1239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Han  Cao</span>, <span class="hlFld-ContribAuthor ">Xue-Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Fu-sheng  Bie</span>, <span class="hlFld-ContribAuthor ">Peng  Yan</span>, <span class="hlFld-ContribAuthor ">Jie  Ma</span>, <span class="hlFld-ContribAuthor ">Yi-jun  Shi</span>, <span class="hlFld-ContribAuthor ">Ying  Han</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient Synthesis of 2-Functionalized Benzoxazoles Catalyzed by Copper Iodide. </span><span class="cited-content_cbyCitation_journal-name">HETEROCYCLES</span><span> <strong>2020,</strong> <em>100 </em>
                                    (8)
                                     , 1218. <a href="https://doi.org/10.3987/COM-20-14276" title="DOI URL">https://doi.org/10.3987/COM-20-14276</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3987/COM-20-14276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3987%2FCOM-20-14276%26sid%3Dliteratum%253Aachs%26jtitle%3DHETEROCYCLES%26atitle%3DEfficient%252BSynthesis%252Bof%252B2-Functionalized%252BBenzoxazoles%252BCatalyzed%252Bby%252BCopper%252BIodide%26aulast%3DCao%26aufirst%3DHan%26date%3D2020%26volume%3D100%26issue%3D8%26spage%3D1218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Srikanth  Gatadi</span>, <span class="hlFld-ContribAuthor ">Y.V.  Madhavi</span>, <span class="hlFld-ContribAuthor ">Sidharth  Chopra</span>, <span class="hlFld-ContribAuthor ">Srinivas  Nanduri</span>. </span><span class="cited-content_cbyCitation_article-title">Promising antibacterial agents against multidrug resistant Staphylococcus aureus. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2019,</strong> <em>92 </em>, 103252. <a href="https://doi.org/10.1016/j.bioorg.2019.103252" title="DOI URL">https://doi.org/10.1016/j.bioorg.2019.103252</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2019.103252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2019.103252%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DPromising%252Bantibacterial%252Bagents%252Bagainst%252Bmultidrug%252Bresistant%252BStaphylococcus%252Baureus%26aulast%3DGatadi%26aufirst%3DSrikanth%26date%3D2019%26volume%3D92%26spage%3D103252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hakimeh  Teymourinia</span>, <span class="hlFld-ContribAuthor ">Masoud  Salavati-Niasari</span>, <span class="hlFld-ContribAuthor ">Omid  Amiri</span>. </span><span class="cited-content_cbyCitation_article-title">Simple synthesis of Cu2O/GQDs nanocomposite with different morphologies fabricated by tuning the synthesis parameters as novel antibacterial material. </span><span class="cited-content_cbyCitation_journal-name">Composites Part B: Engineering</span><span> <strong>2019,</strong> <em>172 </em>, 785-794. <a href="https://doi.org/10.1016/j.compositesb.2019.05.047" title="DOI URL">https://doi.org/10.1016/j.compositesb.2019.05.047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compositesb.2019.05.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compositesb.2019.05.047%26sid%3Dliteratum%253Aachs%26jtitle%3DComposites%2520Part%2520B%253A%2520Engineering%26atitle%3DSimple%252Bsynthesis%252Bof%252BCu2O%25252FGQDs%252Bnanocomposite%252Bwith%252Bdifferent%252Bmorphologies%252Bfabricated%252Bby%252Btuning%252Bthe%252Bsynthesis%252Bparameters%252Bas%252Bnovel%252Bantibacterial%252Bmaterial%26aulast%3DTeymourinia%26aufirst%3DHakimeh%26date%3D2019%26volume%3D172%26spage%3D785%26epage%3D794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alex  Washburn</span>, <span class="hlFld-ContribAuthor ">Sanofar  Abdeen</span>, <span class="hlFld-ContribAuthor ">Yulia  Ovechkina</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Ray</span>, <span class="hlFld-ContribAuthor ">Mckayla  Stevens</span>, <span class="hlFld-ContribAuthor ">Siddhi  Chitre</span>, <span class="hlFld-ContribAuthor ">Jared  Sivinski</span>, <span class="hlFld-ContribAuthor ">Yangshin  Park</span>, <span class="hlFld-ContribAuthor ">James  Johnson</span>, <span class="hlFld-ContribAuthor ">Quyen Q.  Hoang</span>, <span class="hlFld-ContribAuthor ">Eli  Chapman</span>, <span class="hlFld-ContribAuthor ">Tanya  Parish</span>, <span class="hlFld-ContribAuthor ">Steven M.  Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Dual-targeting GroEL/ES chaperonin and protein tyrosine phosphatase B (PtpB) inhibitors: A polypharmacology strategy for treating Mycobacterium tuberculosis infections. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (13)
                                     , 1665-1672. <a href="https://doi.org/10.1016/j.bmcl.2019.04.034" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.04.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.04.034%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDual-targeting%252BGroEL%25252FES%252Bchaperonin%252Band%252Bprotein%252Btyrosine%252Bphosphatase%252BB%252B%252528PtpB%252529%252Binhibitors%25253A%252BA%252Bpolypharmacology%252Bstrategy%252Bfor%252Btreating%252BMycobacterium%252Btuberculosis%252Binfections%26aulast%3DWashburn%26aufirst%3DAlex%26date%3D2019%26volume%3D29%26issue%3D13%26spage%3D1665%26epage%3D1672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mckayla  Stevens</span>, <span class="hlFld-ContribAuthor ">Sanofar  Abdeen</span>, <span class="hlFld-ContribAuthor ">Nilshad  Salim</span>, <span class="hlFld-ContribAuthor ">Anne-Marie  Ray</span>, <span class="hlFld-ContribAuthor ">Alex  Washburn</span>, <span class="hlFld-ContribAuthor ">Siddhi  Chitre</span>, <span class="hlFld-ContribAuthor ">Jared  Sivinski</span>, <span class="hlFld-ContribAuthor ">Yangshin  Park</span>, <span class="hlFld-ContribAuthor ">Quyen Q.  Hoang</span>, <span class="hlFld-ContribAuthor ">Eli  Chapman</span>, <span class="hlFld-ContribAuthor ">Steven M.  Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">HSP60/10 chaperonin systems are inhibited by a variety of approved drugs, natural products, and known bioactive molecules. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (9)
                                     , 1106-1112. <a href="https://doi.org/10.1016/j.bmcl.2019.02.028" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.02.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.02.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.02.028%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DHSP60%25252F10%252Bchaperonin%252Bsystems%252Bare%252Binhibited%252Bby%252Ba%252Bvariety%252Bof%252Bapproved%252Bdrugs%25252C%252Bnatural%252Bproducts%25252C%252Band%252Bknown%252Bbioactive%252Bmolecules%26aulast%3DStevens%26aufirst%3DMckayla%26date%3D2019%26volume%3D29%26issue%3D9%26spage%3D1106%26epage%3D1112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hamza  Olleik</span>, <span class="hlFld-ContribAuthor ">Samir  Yahiaoui</span>, <span class="hlFld-ContribAuthor ">Brayan  Roulier</span>, <span class="hlFld-ContribAuthor ">Elise  Courvoisier-Dezord</span>, <span class="hlFld-ContribAuthor ">Josette  Perrier</span>, <span class="hlFld-ContribAuthor ">Basile  Pérès</span>, <span class="hlFld-ContribAuthor ">Akram  Hijazi</span>, <span class="hlFld-ContribAuthor ">Elias  Baydoun</span>, <span class="hlFld-ContribAuthor ">Josette  Raymond</span>, <span class="hlFld-ContribAuthor ">Ahcène  Boumendjel</span>, <span class="hlFld-ContribAuthor ">Marc  Maresca</span>, <span class="hlFld-ContribAuthor ">Romain  Haudecoeur</span>. </span><span class="cited-content_cbyCitation_article-title">Aurone derivatives as promising antibacterial agents against resistant Gram-positive pathogens. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>165 </em>, 133-141. <a href="https://doi.org/10.1016/j.ejmech.2019.01.022" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.022%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAurone%252Bderivatives%252Bas%252Bpromising%252Bantibacterial%252Bagents%252Bagainst%252Bresistant%252BGram-positive%252Bpathogens%26aulast%3DOlleik%26aufirst%3DHamza%26date%3D2019%26volume%3D165%26spage%3D133%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonio  Palumbo Piccionello</span>, <span class="hlFld-ContribAuthor ">Paola  Marzullo</span>, <span class="hlFld-ContribAuthor ">Silvestre  Buscemi</span>, <span class="hlFld-ContribAuthor ">Andrea  Pace</span>. </span><span class="cited-content_cbyCitation_article-title">Hsp60 Inhibitors and Modulators. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 27-39. <a href="https://doi.org/10.1007/978-3-030-23154-5_3" title="DOI URL">https://doi.org/10.1007/978-3-030-23154-5_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-23154-5_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-23154-5_3%26sid%3Dliteratum%253Aachs%26atitle%3DHsp60%252BInhibitors%252Band%252BModulators%26aulast%3DPalumbo%2BPiccionello%26aufirst%3DAntonio%26date%3D2019%26date%3D2019%26spage%3D27%26epage%3D39%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHeat%252BShock%252BProtein%252B60%252Bin%252BHuman%252BDiseases%252Band%252BDisorders%26aulast%3DAsea%26aufirst%3DAlexzander%2BA.%2BA.%26date%3D2019%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomohiro  Mizobata</span>, <span class="hlFld-ContribAuthor ">Yasushi  Kawata</span>. </span><span class="cited-content_cbyCitation_article-title">Utilizing the Unique Architecture and Abilities of HSP60 in Drug Development. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 57-68. <a href="https://doi.org/10.1007/978-3-030-23154-5_5" title="DOI URL">https://doi.org/10.1007/978-3-030-23154-5_5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-23154-5_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-23154-5_5%26sid%3Dliteratum%253Aachs%26atitle%3DUtilizing%252Bthe%252BUnique%252BArchitecture%252Band%252BAbilities%252Bof%252BHSP60%252Bin%252BDrug%252BDevelopment%26aulast%3DMizobata%26aufirst%3DTomohiro%26date%3D2019%26date%3D2019%26spage%3D57%26epage%3D68%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHeat%252BShock%252BProtein%252B60%252Bin%252BHuman%252BDiseases%252Band%252BDisorders%26aulast%3DAsea%26aufirst%3DAlexzander%2BA.%2BA.%26date%3D2019%26volume%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Progression of hit-to-lead compound <b>1</b> analogs. A. Compound <b>1</b> was an initial hit that emerged from our high-throughput screening for GroEL/ES inhibitors.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> We recently reported on a series of pseudosymmetrical compound <b>1</b> analogs (<b>2</b>–<b>14</b>, R<sup>1</sup> represents a variety of alkyl, aryl, and substituted phenyl substructures) that have antibiotic properties against <i>Trypanosoma brucei</i> parasites.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> B. In the present study, we initially screened the <b>1</b>–<b>14</b> analogs for their ability to inhibit a panel of Gram-positive and Gram-negative bacteria and found that the thiophene (<b>6</b>) and <i>para</i>-chloro-substituted (<b>2c-</b><i><b>p</b></i>) analogs generated the most potent and selective inhibitors of MRSA proliferation. Data plotted in the gray zones represent EC<sub>50</sub> and CC<sub>50</sub> results beyond the assay detection limits (i.e., > 100 μM). C. Based on initial SAR, compound <b>15</b> was developed (bearing the 2-chlorothiophene group on both sides) along with two parallel series of asymmetrical 2-chlorothiophene-based analogs (<b>16</b>–<b>34</b>) containing variable alkyl and aryl substructures at the R<sup>2</sup> positions were developed in an attempt to improve their selectivity indices between efficacy against bacterial proliferation and cytotoxicity in human cell viability assays (structures of the different R<sup>2</sup> groups are presented in Tables S3A and S4A in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Syntheses of R- and L-Series Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: A. (a) CH<sub>2</sub>Cl<sub>2</sub>, rt (94%); (b) tin powder, 10% HCl in AcOH, rt (82%); (c) 4-nitrobenzaldehyde, NaHCO<sub>3</sub>, Na<sub>2</sub>SO<sub>4</sub>, THF, reflux to rt, then DDQ added (95%); (d) tin powder, 10% HCl in AcOH, rt (41%); (e) R<sup>2</sup>-SO<sub>2</sub>Cl, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, rt. B. (f) CH<sub>2</sub>Cl<sub>2</sub>, rt (98%); (g) LiOH·H<sub>2</sub>O, 1:1:1 THF:MeOH:H<sub>2</sub>O, rt (96%); (h) SOCl<sub>2</sub>, 60 °C for 1 h, concentrated, then 2-amino-4-nitrophenol and pyridine in CH<sub>2</sub>Cl<sub>2</sub>, reflux to rt, then TsOH·H<sub>2</sub>O, xylenes, reflux (45%); (i) tin powder, 10% HCl in AcOH, rt (96%); (j) R<sup>2</sup>-SO<sub>2</sub>Cl, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, rt.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Correlation plots of IC<sub>50</sub> and EC<sub>50</sub> values for compounds tested in the respective GroEL/ES-dMDH or dRho refolding assays and bacterial proliferation assays. Each data point represents results for individual compounds tested in the respective assays (plotted from results presented in Tables S1–S4 in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>). Data plotted in the gray zones represent results beyond the assay detection limits (i.e., > 100 μM). A. Compounds inhibit the proliferation of <i>S. aureus</i> (drug susceptible) and MRSA (drug resistant) bacteria nearly equipotently. The L- and R-series asymmetric inhibitors are generally more potent than the previously developed pseudosymmetrical inhibitors (<b>1</b>–<b>14</b>). B. Compounds are on-target for inhibiting chaperonin-mediated substrate refolding as a correlation is evident between IC<sub>50</sub> values obtained from the GroEL/ES-dMDH and -dRho refolding assays, and compounds do not inhibit in both the native MDH and Rho enzymatic activity counter-screens. C. A general trend is observed between inhibiting the GroEL/ES chaperonin system and MRSA proliferation, supporting on-target effects in bacteria.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representative analyses of the binding of <b>20R</b> (panel A) and <b>20L</b> (panel B) to GroEL measured by isothermal titration calorimetry (ITC). Top panels show binding isotherms obtained by titrating GroEL (400 μM monomer concentration) into solutions of either <b>20R</b> or <b>20L</b> (50 μM) in the ITC cell. Lower panels show the integrated data (solid squares) fit to a single-site binding model (solid lines). The molar ratio refers to the binding stoichiometry of monomeric GroEL to molecules <b>20R</b> or <b>20L</b>. Average results for the various binding parameters (<i>K</i><sub>d</sub>, <i>n</i>, Δ<i>H</i>, Δ<i>S</i>, and Δ<i>G</i>) obtained from replicate analyses are presented in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Correlation plots of IC<sub>50</sub>, EC<sub>50</sub>, and CC<sub>50</sub> values for compounds tested in the respective GroEL/ES-dMDH and HSP60/10-dMDH refolding assays, liver cell cytotoxicity assay, and MRSA proliferation assay. Each data point represents results for individual compounds tested in the respective assays (plotted from results presented in Tables S1–S4 in the <a class="ref internalNav" href="#notes-3" aria-label="Supporting Information">Supporting Information</a>). Data plotted in the gray zones represent results beyond the assay detection limits (i.e., >100 μM). A. The R-series and previously developed analogs selectively inhibit the <i>E. coli</i> GroEL/ES over the human HSP60/10 chaperonin system but the L-series analogs are not selective. B. No correlation is evident between cytotoxicity of compounds in the liver cell viability assay and the HSP60/10-dMDH refolding assay, suggesting cytotoxicity may be from off-target effects in cells. C. Many compounds are able to selectively inhibit MRSA proliferation with moderate to low cytotoxicity to human liver cells. Results are similar for compounds tested in the kidney cell viability assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/medium/jm-2018-00989f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Exploring adaptive tolerance by MRSA bacteria to analogs <b>20R</b>, <b>28R</b>, and vancomycin. A. MRSA was serially passaged for 12 days in fresh liquid media supplemented with the selected compounds. Average EC<sub>50</sub> values of compounds tested after each 24 h passage are plotted from triplicate analyses. MRSA rapidly evolved resistance to <b>28R</b> but retained sensitivity to <b>20R</b> and vancomycin throughout the 12 day experiment. Data plotted in the gray zones represent EC<sub>50</sub> results beyond the assay detection limits (i.e., >100 μM). B. After reculturing the <b>28R</b>-resistant MRSA strain for 2 days in the absence of inhibitors (24 h on agar, then another 24 h in liquid medium), the “resistant” MRSA strain regained sensitivity to <b>28R</b> (average % growth inhibition results (±SD) are plotted from 4 replicate analyses).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-16/acs.jmedchem.8b00989/20180816/images/large/jm-2018-00989f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00989&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33826" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33826" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 36 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span>; <span class="NLM_string-name">Magrini, N.</span></span> <i>Global priority list of antibiotic-resistant bacteria to guide research, discovery, and development of new antibiotics</i>; <span class="NLM_publisher-name">World Health Organization</span>: <span class="NLM_publisher-loc">Geneva, Switzerland</span>, <span class="NLM_year">2017</span>; p  <span class="NLM_fpage">7</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=7&author=E.+Tacconelli&author=N.+Magrini&title=Global+priority+list+of+antibiotic-resistant+bacteria+to+guide+research%2C+discovery%2C+and+development+of+new+antibiotics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DTacconelli%26aufirst%3DE.%26btitle%3DGlobal%2520priority%2520list%2520of%2520antibiotic-resistant%2520bacteria%2520to%2520guide%2520research%252C%2520discovery%252C%2520and%2520development%2520of%2520new%2520antibiotics%26pub%3DWorld%2520Health%2520Organization%26date%3D2017%26spage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Centers
for Disease Control and Prevention</span> <i>Antibiotic Resistance
Threats in the United States, 2013</i>; <span class="NLM_publisher-name">Centers
for Disease Control and Prevention</span>: <span class="NLM_publisher-loc">Atlanta, Georgia</span>, <span class="NLM_year">2013</span>; p  <span class="NLM_fpage">114</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=114&author=Centers%0Afor+Disease+Control+and+Prevention&title=Antibiotic+Resistance%0AThreats+in+the+United+States%2C+2013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DAntibiotic%2520Resistance%250AThreats%2520in%2520the%2520United%2520States%252C%25202013%26pub%3DCenters%250Afor%2520Disease%2520Control%2520and%2520Prevention%26date%3D2013%26spage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karlowsky, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackel, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lob, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahm, D. F.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial susceptibility of Gram-negative ESKAPE pathogens isolated from hospitalized patients with intra-abdominal and urinary tract infections in Asia-Pacific countries: SMART 2013–2015</span>. <i>J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1099/jmm.0.000421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1099%2Fjmm.0.000421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=28051952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BC1c7itF2rug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=61-69&author=J.+A.+Karlowskyauthor=D.+J.+Hobanauthor=M.+A.+Hackelauthor=S.+H.+Lobauthor=D.+F.+Sahm&title=Antimicrobial+susceptibility+of+Gram-negative+ESKAPE+pathogens+isolated+from+hospitalized+patients+with+intra-abdominal+and+urinary+tract+infections+in+Asia-Pacific+countries%3A+SMART+2013%E2%80%932015&doi=10.1099%2Fjmm.0.000421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antimicrobial susceptibility of Gram-negative ESKAPE pathogens isolated from hospitalized patients with intra-abdominal and urinary tract infections in Asia-Pacific countries: SMART 2013-2015</span></div><div class="casAuthors">Karlowsky James A; Hoban Daryl J; Hackel Meredith A; Lob Sibylle H; Sahm Daniel F</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medical microbiology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-69</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Gram-negative ESKAPE pathogens (Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) are responsible for increases in antimicrobial-resistant infections worldwide.  We determined in vitro susceptibilities to eight parenteral antimicrobial agents using Clinical and Laboratory Standards Institute broth microdilution methodology for Gram-negative ESKAPE pathogens isolated from hospitalized patients with intra-abdominal infections (IAIs) (n=3052) and urinary tract infections (UTIs) (n=1088) in 11 Asia-Pacific countries/regions from 2013 to 2015.  Amikacin (98.3, 96.4 %), imipenem (97.1, 95.5 %) and ertapenem (95.3, 93.2 %) demonstrated the highest rates of susceptibility for isolates of K. pneumoniae from IAI and UTI, respectively, whereas susceptibility to advanced-generation cephalosporins was <84 and <71 %, respectively.  K. pneumoniae with an extended-spectrum β-lactamase-positive phenotype were more common in UTI (27.1 %) than IAI (16.2 %).  Imipenem and amikacin were the most active agents against extended-spectrum β-lactamase-positive K. pneumoniae from IAI (95.1, 91.8 %) and UTI (94.9, 92.3 %), respectively, whereas <54 % were susceptible to piperacillin-tazobactam.  Against Enterobacter spp. and P. aeruginosa, amikacin demonstrated the highest rates of susceptibility for isolates from IAI (99.7, 95.5 %) and UTI (90.9, 91.5 %), respectively.  K. pneumoniae, Enterobacter spp. and P. aeruginosa from urine demonstrated lower susceptibility to levofloxacin (74.1, 81.8 and 73.8 %) than from IAI (87.6, 91.8 and 85.4 %).  For A. baumannii, rates of susceptibility to all agents tested were <43 %.  We conclude that the studied Gram-negative ESKAPE pathogens demonstrated reduced susceptibility to commonly prescribed advanced-generation cephalosporins, piperacillin-tazobactam and levofloxacin, while amikacin and carbapenems were the most active.  Ongoing surveillance to monitor evolving resistance trends and the development of novel antimicrobial agents with potent activity against Gram-negative ESKAPE pathogens are mandatory.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPKOnN8YplpDUE-swuRqsHfW6udTcc2eajUtcl1hzwPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7itF2rug%253D%253D&md5=ace8f3676d7540bd820cdc708c9b4d76</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1099%2Fjmm.0.000421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fjmm.0.000421%26sid%3Dliteratum%253Aachs%26aulast%3DKarlowsky%26aufirst%3DJ.%2BA.%26aulast%3DHoban%26aufirst%3DD.%2BJ.%26aulast%3DHackel%26aufirst%3DM.%2BA.%26aulast%3DLob%26aufirst%3DS.%2BH.%26aulast%3DSahm%26aufirst%3DD.%2BF.%26atitle%3DAntimicrobial%2520susceptibility%2520of%2520Gram-negative%2520ESKAPE%2520pathogens%2520isolated%2520from%2520hospitalized%2520patients%2520with%2520intra-abdominal%2520and%2520urinary%2520tract%2520infections%2520in%2520Asia-Pacific%2520countries%253A%2520SMART%25202013%25E2%2580%25932015%26jtitle%3DJ.%2520Med.%2520Microbiol.%26date%3D2017%26volume%3D66%26spage%3D61%26epage%3D69%26doi%3D10.1099%2Fjmm.0.000421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santajit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indrawattana, N.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of antimicrobial resistance in ESKAPE pathogens</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2016</i></span>,  <span class="NLM_fpage">2475067</span>, <span class="refDoi"> DOI: 10.1155/2016/2475067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1155%2F2016%2F2475067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=27274985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=2475067&author=S.+Santajitauthor=N.+Indrawattana&title=Mechanisms+of+antimicrobial+resistance+in+ESKAPE+pathogens&doi=10.1155%2F2016%2F2475067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Antimicrobial Resistance in ESKAPE Pathogens</span></div><div class="casAuthors">Santajit Sirijan; Indrawattana Nitaya</div><div class="citationInfo"><span class="NLM_cas:title">BioMed research international</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2016</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2475067</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) are the leading cause of nosocomial infections throughout the world.  Most of them are multidrug resistant isolates, which is one of the greatest challenges in clinical practice.  Multidrug resistance is amongst the top three threats to global public health and is usually caused by excessive drug usage or prescription, inappropriate use of antimicrobials, and substandard pharmaceuticals.  Understanding the resistance mechanisms of these bacteria is crucial for the development of novel antimicrobial agents or other alternative tools to combat these public health challenges.  Greater mechanistic understanding would also aid in the prediction of underlying or even unknown mechanisms of resistance, which could be applied to other emerging multidrug resistant pathogens.  In this review, we summarize the known antimicrobial resistance mechanisms of ESKAPE pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSM_xN5GU5d0ntlzo9xastyfW6udTcc2eajUtcl1hzwPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FotFantA%253D%253D&md5=e42353b6d72db948734b5354e31b501e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1155%2F2016%2F2475067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2016%252F2475067%26sid%3Dliteratum%253Aachs%26aulast%3DSantajit%26aufirst%3DS.%26aulast%3DIndrawattana%26aufirst%3DN.%26atitle%3DMechanisms%2520of%2520antimicrobial%2520resistance%2520in%2520ESKAPE%2520pathogens%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2016%26volume%3D2016%26spage%3D2475067%26doi%3D10.1155%2F2016%2F2475067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. B.</span></span> <span> </span><span class="NLM_article-title">Federal funding for the study of antimicrobial resistance in nosocomial pathogens: No ESKAPE</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>197</i></span>,  <span class="NLM_fpage">1079</span>– <span class="NLM_lpage">1081</span>, <span class="refDoi"> DOI: 10.1086/533452</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1086%2F533452" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=18419525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BD1c3lvFChug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=197&publication_year=2008&pages=1079-1081&author=L.+B.+Rice&title=Federal+funding+for+the+study+of+antimicrobial+resistance+in+nosocomial+pathogens%3A+No+ESKAPE&doi=10.1086%2F533452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Federal funding for the study of antimicrobial resistance in nosocomial pathogens: no ESKAPE</span></div><div class="casAuthors">Rice Louis B</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of infectious diseases</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">197</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1079-81</span>
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcReTA_wnRzWZTslzEEsg76ufW6udTcc2eajUtcl1hzwPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3lvFChug%253D%253D&md5=c2c5ec8ad941f702f588f89741d9410e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1086%2F533452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F533452%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DL.%2BB.%26atitle%3DFederal%2520funding%2520for%2520the%2520study%2520of%2520antimicrobial%2520resistance%2520in%2520nosocomial%2520pathogens%253A%2520No%2520ESKAPE%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2008%26volume%3D197%26spage%3D1079%26epage%3D1081%26doi%3D10.1086%2F533452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, H. F.</span></span> <span> </span><span class="NLM_article-title">Treatment of methicillin-resistant Staphylococcus aureus bacteremia</span>. <i>Infect. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">267</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.3947/ic.2016.48.4.267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.3947%2Fic.2016.48.4.267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=28032484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKiur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2016&pages=267-273&author=E.+J.+Chooauthor=H.+F.+Chambers&title=Treatment+of+methicillin-resistant+Staphylococcus+aureus+bacteremia&doi=10.3947%2Fic.2016.48.4.267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment of methicillin-resistant Staphylococcus aureus bacteremia</span></div><div class="casAuthors">Choo, Eun Ju; Chambers, Henry F.</div><div class="citationInfo"><span class="NLM_cas:title">Infection & Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">267-273</span>CODEN:
                <span class="NLM_cas:coden">ICNHAK</span>;
        ISSN:<span class="NLM_cas:issn">2093-2340</span>.
    
            (<span class="NLM_cas:orgname">Korean Society of Infectious Diseases</span>)
        </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) is a significant cause of health care-assocd. infections.  Vancomycin remains an acceptable treatment option.  There has been a welcome increase in the no. of agents available for the treatment of MRSA infection.  These drugs have certain differentiating attributes and may offer some advantages over vancomycin, but they also have significant limitations.  These agents provide some alternative when no other options arc available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIc_NWfevGv7Vg90H21EOLACvtfcHk0lg08nliWCDV-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKiur4%253D&md5=4a25789660d694935631b23cf2de0363</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3947%2Fic.2016.48.4.267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3947%252Fic.2016.48.4.267%26sid%3Dliteratum%253Aachs%26aulast%3DChoo%26aufirst%3DE.%2BJ.%26aulast%3DChambers%26aufirst%3DH.%2BF.%26atitle%3DTreatment%2520of%2520methicillin-resistant%2520Staphylococcus%2520aureus%2520bacteremia%26jtitle%3DInfect.%2520Chemother.%26date%3D2016%26volume%3D48%26spage%3D267%26epage%3D273%26doi%3D10.3947%2Fic.2016.48.4.267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassoun, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linden, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, B.</span></span> <span> </span><span class="NLM_article-title">Incidence, prevalence, and management of MRSA bacteremia across patient populations-a review of recent developments in MRSA management and treatment</span>. <i>Crit. Care</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">211</span>, <span class="refDoi"> DOI: 10.1186/s13054-017-1801-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1186%2Fs13054-017-1801-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=28807042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A280%3ADC%252BC1cfnsVKrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=211&author=A.+Hassounauthor=P.+K.+Lindenauthor=B.+Friedman&title=Incidence%2C+prevalence%2C+and+management+of+MRSA+bacteremia+across+patient+populations-a+review+of+recent+developments+in+MRSA+management+and+treatment&doi=10.1186%2Fs13054-017-1801-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Incidence, prevalence, and management of MRSA bacteremia across patient populations-a review of recent developments in MRSA management and treatment</span></div><div class="casAuthors">Hassoun Ali; Linden Peter K; Friedman Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Critical care (London, England)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">211</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) infection is still a major global healthcare problem.  Of concern is S. aureus bacteremia, which exhibits high rates of morbidity and mortality and can cause metastatic or complicated infections such as infective endocarditis or sepsis.  MRSA is responsible for most global S. aureus bacteremia cases, and compared with methicillin-sensitive S. aureus, MRSA infection is associated with poorer clinical outcomes.  S. aureus virulence is affected by the unique combination of toxin and immune-modulatory gene products, which may differ by geographic location and healthcare- or community-associated acquisition.  Management of S. aureus bacteremia involves timely identification of the infecting strain and source of infection, proper choice of antibiotic treatment, and robust prevention strategies.  Resistance and nonsusceptibility to first-line antimicrobials combined with a lack of equally effective alternatives complicates MRSA bacteremia treatment.  This review describes trends in epidemiology and factors that influence the incidence of MRSA bacteremia.  Current and developing diagnostic tools, treatments, and prevention strategies are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRY-2a_VQlONUBuFdjgVNKSfW6udTcc2eaLsj9Lf7wkAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfnsVKrtg%253D%253D&md5=a004de0c857f92a6dddfd7ea6a0d3493</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2Fs13054-017-1801-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13054-017-1801-3%26sid%3Dliteratum%253Aachs%26aulast%3DHassoun%26aufirst%3DA.%26aulast%3DLinden%26aufirst%3DP.%2BK.%26aulast%3DFriedman%26aufirst%3DB.%26atitle%3DIncidence%252C%2520prevalence%252C%2520and%2520management%2520of%2520MRSA%2520bacteremia%2520across%2520patient%2520populations-a%2520review%2520of%2520recent%2520developments%2520in%2520MRSA%2520management%2520and%2520treatment%26jtitle%3DCrit.%2520Care%26date%3D2017%26volume%3D21%26spage%3D211%26doi%3D10.1186%2Fs13054-017-1801-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Razonable, R. R.</span></span> <span> </span><span class="NLM_article-title">Serious infections caused by methicillin-resistant Staphylococcus aureus</span>. <i>Clin. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">S183</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1086/653519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1086%2F653519" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=20731576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yhsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=S183-197&author=H.+Boucherauthor=L.+G.+Millerauthor=R.+R.+Razonable&title=Serious+infections+caused+by+methicillin-resistant+Staphylococcus+aureus&doi=10.1086%2F653519"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Serious infections caused by methicillin-resistant Staphylococcus aureus</span></div><div class="casAuthors">Boucher, Helen; Miller, Loren G.; Razonable, Raymund R.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Infectious Diseases</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">S183-S197</span>CODEN:
                <span class="NLM_cas:coden">CIDIEL</span>;
        ISSN:<span class="NLM_cas:issn">1058-4838</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">A review.  Although first identified just >4 decades ago, methicillin-resistant Staphylococcus aureus (MRSA) has undergone rapid evolutionary changes and epidemiol. expansion to become a major cause of nosocomial and community-acquired infections worldwide.  Increasing resistance to vancomycin among MRSA strains in conjunction with availability of new antibiotics, including daptomycin and linezolid, have increased treatment choices but made clin. treatment decisions more challenging.  This article describes the clin. features and management issues of 2 challenging-to-treat manifestations of MRSA infection, bacteremia and/or endocarditis and osteomyelitis.  It also presents a brief review of community-assocd. MRSA infections and preventive strategies directed against MRSA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVLYfFk5mCbVg90H21EOLACvtfcHk0lg08nliWCDV-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yhsbfO&md5=e4fb086393d4fa614e148306002d50af</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1086%2F653519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F653519%26sid%3Dliteratum%253Aachs%26aulast%3DBoucher%26aufirst%3DH.%26aulast%3DMiller%26aufirst%3DL.%2BG.%26aulast%3DRazonable%26aufirst%3DR.%2BR.%26atitle%3DSerious%2520infections%2520caused%2520by%2520methicillin-resistant%2520Staphylococcus%2520aureus%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2010%26volume%3D51%26spage%3DS183%26epage%3D197%26doi%3D10.1086%2F653519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houry, W. A.</span></span> <span> </span><span class="NLM_article-title">Chaperone networks in bacteria: analysis of protein homeostasis in minimal cells</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2003.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.jsb.2003.11.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2004&pages=79-89&author=P.+Wongauthor=W.+A.+Houry&title=Chaperone+networks+in+bacteria%3A+analysis+of+protein+homeostasis+in+minimal+cells&doi=10.1016%2Fj.jsb.2003.11.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2003.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2003.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DP.%26aulast%3DHoury%26aufirst%3DW.%2BA.%26atitle%3DChaperone%2520networks%2520in%2520bacteria%253A%2520analysis%2520of%2520protein%2520homeostasis%2520in%2520minimal%2520cells%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2004%26volume%3D146%26spage%3D79%26epage%3D89%26doi%3D10.1016%2Fj.jsb.2003.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mogk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bukau, B.</span></span> <span> </span><span class="NLM_article-title">Integrating protein homeostasis strategies in prokaryotes</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">a004366</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a004366</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1101%2Fcshperspect.a004366" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlejsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=a004366&author=A.+Mogkauthor=D.+Huberauthor=B.+Bukau&title=Integrating+protein+homeostasis+strategies+in+prokaryotes&doi=10.1101%2Fcshperspect.a004366"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating protein homeostasis strategies in prokaryotes</span></div><div class="casAuthors">Mogk, Axel; Huber, Damon; Bukau, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">a004366/1-a004366/19</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Bacterial cells are frequently exposed to dramatic fluctuations in their environment, which cause perturbation in protein homeostasis and lead to protein misfolding.  Bacteria have therefore evolved powerful quality control networks consisting of chaperones and proteases that cooperate to monitor the folding states of proteins and to remove misfolded conformers through either refolding or degrdn.  The levels of the quality control components are adjusted to the folding state of the cellular proteome through the induction of compartment specific stress responses.  In addn., the activities of several quality control components are directly controlled by these stresses, allowing for fast activation.  Severe stress can, however, overcome the protective function of the proteostasis network leading to the formation of protein aggregates, which are sequestered at the cell poles.  Protein aggregates are either solubilized by AAA+ chaperones or eliminated through cell division, allowing for the generation of damage-free daughter cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgphXy0ybMALVg90H21EOLACvtfcHk0liF_he89fsFzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlejsL0%253D&md5=f0d5cf1d8055ce3664e40fd02d0245bc</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a004366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a004366%26sid%3Dliteratum%253Aachs%26aulast%3DMogk%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DD.%26aulast%3DBukau%26aufirst%3DB.%26atitle%3DIntegrating%2520protein%2520homeostasis%2520strategies%2520in%2520prokaryotes%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2011%26volume%3D3%26spage%3Da004366%26doi%3D10.1101%2Fcshperspect.a004366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiappori, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanesi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merelli, I.</span></span> <span> </span><span class="NLM_article-title">DnaK as antibiotic target: Hot spot residues analysis for differential inhibition of the bacterial protein in comparison with the human HSP70</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0124563</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0124563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1371%2Fjournal.pone.0124563" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0124563&author=F.+Chiapporiauthor=M.+Fumianauthor=L.+Milanesiauthor=I.+Merelli&title=DnaK+as+antibiotic+target%3A+Hot+spot+residues+analysis+for+differential+inhibition+of+the+bacterial+protein+in+comparison+with+the+human+HSP70&doi=10.1371%2Fjournal.pone.0124563"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0124563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0124563%26sid%3Dliteratum%253Aachs%26aulast%3DChiappori%26aufirst%3DF.%26aulast%3DFumian%26aufirst%3DM.%26aulast%3DMilanesi%26aufirst%3DL.%26aulast%3DMerelli%26aufirst%3DI.%26atitle%3DDnaK%2520as%2520antibiotic%2520target%253A%2520Hot%2520spot%2520residues%2520analysis%2520for%2520differential%2520inhibition%2520of%2520the%2520bacterial%2520protein%2520in%2520comparison%2520with%2520the%2520human%2520HSP70%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0124563%26doi%3D10.1371%2Fjournal.pone.0124563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arita-Morioka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamanaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizunoe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, S.</span></span> <span> </span><span class="NLM_article-title">Novel strategy for biofilm inhibition by using small molecules targeting molecular chaperone DnaK</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">633</span>– <span class="NLM_lpage">641</span>, <span class="refDoi"> DOI: 10.1128/AAC.04465-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1128%2FAAC.04465-14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=633-641&author=K.+Arita-Moriokaauthor=K.+Yamanakaauthor=Y.+Mizunoeauthor=T.+Oguraauthor=S.+Sugimoto&title=Novel+strategy+for+biofilm+inhibition+by+using+small+molecules+targeting+molecular+chaperone+DnaK&doi=10.1128%2FAAC.04465-14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FAAC.04465-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.04465-14%26sid%3Dliteratum%253Aachs%26aulast%3DArita-Morioka%26aufirst%3DK.%26aulast%3DYamanaka%26aufirst%3DK.%26aulast%3DMizunoe%26aufirst%3DY.%26aulast%3DOgura%26aufirst%3DT.%26aulast%3DSugimoto%26aufirst%3DS.%26atitle%3DNovel%2520strategy%2520for%2520biofilm%2520inhibition%2520by%2520using%2520small%2520molecules%2520targeting%2520molecular%2520chaperone%2520DnaK%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D633%26epage%3D641%26doi%3D10.1128%2FAAC.04465-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sass, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Josten, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famulla, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahl, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamoen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotz-Oesterhelt, H.</span></span> <span> </span><span class="NLM_article-title">Antibiotic acyldepsipeptides activate ClpP peptidase to degrade the cell division protein FtsZ</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">17474</span>– <span class="NLM_lpage">17479</span>, <span class="refDoi"> DOI: 10.1073/pnas.1110385108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1073%2Fpnas.1110385108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=21969594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtl2ltb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=17474-17479&author=P.+Sassauthor=M.+Jostenauthor=K.+Famullaauthor=G.+Schifferauthor=H.+G.+Sahlauthor=L.+Hamoenauthor=H.+Brotz-Oesterhelt&title=Antibiotic+acyldepsipeptides+activate+ClpP+peptidase+to+degrade+the+cell+division+protein+FtsZ&doi=10.1073%2Fpnas.1110385108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic acyldepsipeptides activate ClpP peptidase to degrade the cell division protein FtsZ</span></div><div class="casAuthors">Sass, Peter; Josten, Michaele; Famulla, Kirsten; Schiffer, Guido; Sahl, Hans-Georg; Hamoen, Leendert; Broetz-Oesterhelt, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">17474-17479, S17474/1-S17474/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The worldwide spread of antibiotic-resistant bacteria has lent urgency to the search for antibiotics with new modes of action that are devoid of preexisting cross-resistances.  We previously described a unique class of acyldepsipeptides (ADEPs) that exerts prominent antibacterial activity against Gram-pos. pathogens including streptococci, enterococci, as well as multi-drug-resistant Staphylococcus aureus.  Here, we report that ADEP prevents cell division in Gram-pos. bacteria and induces strong filamentation of rod-shaped Bacillus subtilis and swelling of coccoid S. aureus and Streptococcus pneumoniae.  It emerged that ADEP treatment inhibits septum formation at the stage of Z-ring assembly, and that central cell division proteins delocalize from midcell positions.  Using in vivo and in vitro studies, we show that the inhibition of Z-ring formation is a consequence of the proteolytic degrdn. of the essential cell division protein FtsZ.  ADEP switches the bacterial ClpP peptidase from a regulated to an uncontrolled protease, and it turned out that FtsZ is particularly prone to degrdn. by the ADEP-CIpP complex.  By preventing cell division, ADEP inhibits a vital cellular process of bacteria that is not targeted by any therapeutically applied antibiotic so far.  Their unique multi-faceted mechanism of action and antibacterial potency makes them promising lead structures for future antibiotic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc8ifJ9CVpkLVg90H21EOLACvtfcHk0liF_he89fsFzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtl2ltb3F&md5=7f6f9401c63b9705cc81b917fb69fe81</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1110385108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1110385108%26sid%3Dliteratum%253Aachs%26aulast%3DSass%26aufirst%3DP.%26aulast%3DJosten%26aufirst%3DM.%26aulast%3DFamulla%26aufirst%3DK.%26aulast%3DSchiffer%26aufirst%3DG.%26aulast%3DSahl%26aufirst%3DH.%2BG.%26aulast%3DHamoen%26aufirst%3DL.%26aulast%3DBrotz-Oesterhelt%26aufirst%3DH.%26atitle%3DAntibiotic%2520acyldepsipeptides%2520activate%2520ClpP%2520peptidase%2520to%2520degrade%2520the%2520cell%2520division%2520protein%2520FtsZ%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D17474%26epage%3D17479%26doi%3D10.1073%2Fpnas.1110385108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Heat shock protein 70 (hsp70) as an emerging drug target</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">4585</span>– <span class="NLM_lpage">4602</span>, <span class="refDoi"> DOI: 10.1021/jm100054f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100054f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4585-4602&author=C.+G.+Evansauthor=L.+Changauthor=J.+E.+Gestwicki&title=Heat+shock+protein+70+%28hsp70%29+as+an+emerging+drug+target&doi=10.1021%2Fjm100054f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Heat Shock Protein 70 (Hsp70) as an Emerging Drug Target</span></div><div class="casAuthors">Evans, Christopher G.; Chang, Lyra; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4585-4602</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5hrQMA77XsrVg90H21EOLACvtfcHk0lj0LKobRT-Qyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejsLc%253D&md5=b87bdc9b76e66d62c130a162037387b5</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm100054f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100054f%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DC.%2BG.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DHeat%2520shock%2520protein%252070%2520%2528hsp70%2529%2520as%2520an%2520emerging%2520drug%2520target%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4585%26epage%3D4602%26doi%3D10.1021%2Fjm100054f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braig, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boisvert, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigler, P. B.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of the bacterial chaperonin GroEL at 2.8 A</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1038/371578a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1038%2F371578a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=7935790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK2cXmslCrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=1994&pages=578-586&author=K.+Braigauthor=Z.+Otwinowskiauthor=R.+Hegdeauthor=D.+C.+Boisvertauthor=A.+Joachimiakauthor=A.+L.+Horwichauthor=P.+B.+Sigler&title=The+crystal+structure+of+the+bacterial+chaperonin+GroEL+at+2.8+A&doi=10.1038%2F371578a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of the bacterial chaperonin GroEL at 2.9 Å</span></div><div class="casAuthors">Braig, Kerstin; Otwinowski, Zbyszek; Hegde, Rashmi; Boisvert, David C.; Joachimiak, Andrzej; Horwich, Arthur L.; Sigler, Paul B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">578-86</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">The crystal structure of Escherichia coli chaperonin GroEL (I) showed a porous cylinder of 14 subunits made of 2 nearly 7-fold rotationally sym. rings stacked back-to-back with dyad symmetry.  The I subunits consisted of 3 domains: a large equatorial domain that forms the foundation of the assembly at its waist and holds the rings together; a large loosely structured apical domain that forms the ends of the cylinder; and a small slender intermediate domain that connects the 2, creating side-windows.  The 3-dimensional structure placed most of the mutationally defined functional sites on the channel walls and its outward invaginations, and at the ends of the cylinder.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOMsEsmFMy_7Vg90H21EOLACvtfcHk0lj0LKobRT-Qyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmslCrsbk%253D&md5=6b1a0fa2aeb3affd03c5bda6256c8764</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2F371578a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F371578a0%26sid%3Dliteratum%253Aachs%26aulast%3DBraig%26aufirst%3DK.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DHegde%26aufirst%3DR.%26aulast%3DBoisvert%26aufirst%3DD.%2BC.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DSigler%26aufirst%3DP.%2BB.%26atitle%3DThe%2520crystal%2520structure%2520of%2520the%2520bacterial%2520chaperonin%2520GroEL%2520at%25202.8%2520A%26jtitle%3DNature%26date%3D1994%26volume%3D371%26spage%3D578%26epage%3D586%26doi%3D10.1038%2F371578a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sigler, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rye, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burston, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Structure and function in GroEL-mediated protein folding</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.67.1.581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1146%2Fannurev.biochem.67.1.581" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=9759498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK1cXlsFOmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1998&pages=581-608&author=P.+B.+Siglerauthor=Z.+Xuauthor=H.+S.+Ryeauthor=S.+G.+Burstonauthor=W.+A.+Fentonauthor=A.+L.+Horwich&title=Structure+and+function+in+GroEL-mediated+protein+folding&doi=10.1146%2Fannurev.biochem.67.1.581"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function in GroEL-mediated protein folding</span></div><div class="casAuthors">Sigler, Paul B.; Xu, Zhaohui; Rye, Hays S.; Burston, Steven G.; Fenton, Wayne A.; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">581-608</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review, with ∼83 refs.  Recent structural and biochem. investigations have come together to allow a better understanding of the mechanism of chaperonin (GroEL, Hsp60)-mediated protein folding, the final step in the accurate expression of genetic information.  Major, asym. conformational changes in the GroEL double toroid accompany binding of ATP and the cochaperonin GroES.  When a nonnative polypeptide, bound to one of the GroEL rings, is encapsulated by GroES to form a cis ternary complex, these changes drive the polypeptide into the sequestered cavity and initiate its folding.  ATP hydrolysis in the cis ring primes release of the products, and ATP binding in the trans ring then disrupts the cis complex.  This process allows the polypeptide to achieve its final native state, if folding was completed, or to recycle to another chaperonin mol., if the folding process did not result in a form committed to the native state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrss2vZQv_Th7Vg90H21EOLACvtfcHk0lh858x2PzeBsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXlsFOmsb0%253D&md5=3f0f7aaa6c4bba7761daabd0ea4ffd62</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.67.1.581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.67.1.581%26sid%3Dliteratum%253Aachs%26aulast%3DSigler%26aufirst%3DP.%2BB.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DRye%26aufirst%3DH.%2BS.%26aulast%3DBurston%26aufirst%3DS.%2BG.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520and%2520function%2520in%2520GroEL-mediated%2520protein%2520folding%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D1998%26volume%3D67%26spage%3D581%26epage%3D608%26doi%3D10.1146%2Fannurev.biochem.67.1.581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farr, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span> <span> </span><span class="NLM_article-title">GroEL-GroES-mediated protein folding</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">1917</span>– <span class="NLM_lpage">1930</span>, <span class="refDoi"> DOI: 10.1021/cr040435v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040435v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFKiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=1917-1930&author=A.+L.+Horwichauthor=G.+W.+Farrauthor=W.+A.+Fenton&title=GroEL-GroES-mediated+protein+folding&doi=10.1021%2Fcr040435v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">GroEL-GroES-mediated protein folding</span></div><div class="casAuthors">Horwich, Arthur L.; Farr, George W.; Fenton, Wayne A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1917-1930</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Chaperonin-mediated folding is an essential ATP-dependent reaction that provides kinetic assistance to the process of protein folding o the native state in a variety of cellular compartments.  The Escherichia coli reaction is carried out by a megadalton-sized double ring "machine" consisting of chaperonin GroEL in complex with its co-chaperonin GroES.  The binding of GroES to GroEL is ATP-dependent with the nucleotide rapidly and cooperatively binding to 7 sites on the GroEL ring.  Here, the catalytic mechanism of GroEL-GroES-mediated protein folding is discussed along with a more detailed consideration of the transition between polypeptide binding in an open ring and productive protein folding in a co-chaperone-encapsulated one.  The free energy of binding of the γ-phosphate of ATP functions as a crit. element of the folding trigger.  The bulk of existing evidence appears to support a model of GroEL action in which ATP and GroES binding drive major conformational changes in GroEL that simultaneously and immediately release a substrate protein from the apical binding sites and initiate refolding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoTMQWAk1wD7Vg90H21EOLACvtfcHk0lh858x2PzeBsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFKiur8%253D&md5=3e71992bed612e9bc567d6bc9318b414</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fcr040435v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040435v%26sid%3Dliteratum%253Aachs%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DFarr%26aufirst%3DG.%2BW.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26atitle%3DGroEL-GroES-mediated%2520protein%2520folding%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D1917%26epage%3D1930%26doi%3D10.1021%2Fcr040435v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Residues in chaperonin GroEL required for polypeptide binding and release</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/371614a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1038%2F371614a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=7935796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK2cXmslCrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=1994&pages=614-619&author=W.+A.+Fentonauthor=Y.+Kashiauthor=K.+Furtakauthor=A.+L.+Norwich&title=Residues+in+chaperonin+GroEL+required+for+polypeptide+binding+and+release&doi=10.1038%2F371614a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Residues in chaperonin GroEL required for polypeptide binding and release</span></div><div class="casAuthors">Fenton, Wayne A.; Kashi, Yechezkel; Furtak, Krystyna; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">614-19</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Chaperonins are ring-shaped protein complexes that are essential in the cell, mediating ATP-dependent polypeptide folding in a variety of compartments.  Recent studies suggest that they function through multiple rounds of binding and release of non-native proteins: with each round of ATP-driven release into the bulk soln., a substrate protein kinetically partitions between folding to the native state or rebinding to another chaperonin mol.  To gain further insight into the mechanism of polypeptide bind and release by the chaperonin GroEL from Escherichia coli, a mutational anal. was undertaken that relates the functional properties of GroEL to its crystal structure.  The functional tests identify a putative polypeptide-binding site on the inside surface of the apical domain, facing the central channel, consisting of hydrophobic residues.  These same residues are essential for binding of the co-chaperonin GroES, which is required for productive polypeptide release.  A highly conserved residue, Asp-87, positioned within a putative nucleotide-binding pocket in the top of the equatorial domain, is essential for ATP hydrolysis and polypeptide release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGwwF9c_IU0LVg90H21EOLACvtfcHk0lh858x2PzeBsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmslCrsbY%253D&md5=912a0f11eb449acba4a1b2b13371db0f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2F371614a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F371614a0%26sid%3Dliteratum%253Aachs%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DKashi%26aufirst%3DY.%26aulast%3DFurtak%26aufirst%3DK.%26aulast%3DNorwich%26aufirst%3DA.%2BL.%26atitle%3DResidues%2520in%2520chaperonin%2520GroEL%2520required%2520for%2520polypeptide%2520binding%2520and%2520release%26jtitle%3DNature%26date%3D1994%26volume%3D371%26spage%3D614%26epage%3D619%26doi%3D10.1038%2F371614a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">GroEL-mediated protein folding</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">760</span>, <span class="refDoi"> DOI: 10.1002/pro.5560060401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1002%2Fpro.5560060401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=9098884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaK2sXisFWhu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=743-760&author=W.+A.+Fentonauthor=A.+L.+Horwich&title=GroEL-mediated+protein+folding&doi=10.1002%2Fpro.5560060401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">GroEL-mediated protein folding</span></div><div class="casAuthors">Fenton, Wayne A.; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">743-760</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review, with ∼120 refs., on the structural and functional studies that have led to the current view of GroEL-mediated protein folding and aspects of chaperonin mechanics that are not, as yet, understood.  Topics include (1) architecture of GroEL and GroES and the reaction pathway, (2) polypeptide binding, (3) ATP binding and hydrolysis in driving the reaction cycle, and (4) GroEL-GroES-polypeptide ternary complexes and the folding-active cis complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqlO7K1EQL1LVg90H21EOLACvtfcHk0lh858x2PzeBsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisFWhu7o%253D&md5=c36df13f76cbccb702a52120a5df5a5f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560060401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560060401%26sid%3Dliteratum%253Aachs%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DGroEL-mediated%2520protein%2520folding%26jtitle%3DProtein%2520Sci.%26date%3D1997%26volume%3D6%26spage%3D743%26epage%3D760%26doi%3D10.1002%2Fpro.5560060401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farr, G. W.</span></span> <span> </span><span class="NLM_article-title">Two families of chaperonin: physiology and mechanism</span>. <i>Annu. Rev. Cell Dev. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1146/annurev.cellbio.23.090506.123555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1146%2Fannurev.cellbio.23.090506.123555" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=115-145&author=A.+L.+Horwichauthor=W.+A.+Fentonauthor=E.+Chapmanauthor=G.+W.+Farr&title=Two+families+of+chaperonin%3A+physiology+and+mechanism&doi=10.1146%2Fannurev.cellbio.23.090506.123555"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1146%2Fannurev.cellbio.23.090506.123555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.cellbio.23.090506.123555%26sid%3Dliteratum%253Aachs%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DFarr%26aufirst%3DG.%2BW.%26atitle%3DTwo%2520families%2520of%2520chaperonin%253A%2520physiology%2520and%2520mechanism%26jtitle%3DAnnu.%2520Rev.%2520Cell%2520Dev.%2520Biol.%26date%3D2007%26volume%3D23%26spage%3D115%26epage%3D145%26doi%3D10.1146%2Fannurev.cellbio.23.090506.123555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saibil, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clare, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Structure and allostery of the chaperonin GroEL</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">1476</span>– <span class="NLM_lpage">1487</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.11.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.jmb.2012.11.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=23183375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVylsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2013&pages=1476-1487&author=H.+R.+Saibilauthor=W.+A.+Fentonauthor=D.+K.+Clareauthor=A.+L.+Horwich&title=Structure+and+allostery+of+the+chaperonin+GroEL&doi=10.1016%2Fj.jmb.2012.11.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Allostery of the Chaperonin GroEL</span></div><div class="casAuthors">Saibil, Helen R.; Fenton, Wayne A.; Clare, Daniel K.; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">425</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1476-1487</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Chaperonins are intricate allosteric machines formed of two back-to-back, stacked rings of subunits presenting end cavities lined with hydrophobic binding sites for nonnative polypeptides.  Once bound, substrates are subjected to forceful, concerted movements that result in their ejection from the binding surface and simultaneous encapsulation inside a hydrophilic chamber that favors their folding.  Here, we review the allosteric machine movements that are choreographed by ATP binding, which triggers concerted tilting and twisting of subunit domains.  These movements distort the ring of hydrophobic binding sites and split it apart, potentially unfolding the multiply bound substrate.  Then, GroES binding is accompanied by a 100° twist of the binding domains that removes the hydrophobic sites from the cavity lining and forms the folding chamber.  ATP hydrolysis is not needed for a single round of binding and encapsulation but is necessary to allow the next round of ATP binding in the opposite ring.  It is this remote ATP binding that triggers dismantling of the folding chamber and release of the encapsulated substrate, whether folded or not.  The basis for these ordered actions is an elegant system of nested cooperativity of the ATPase machinery.  ATP binds to a ring with pos. cooperativity, and movements of the interlinked subunit domains are concerted.  In contrast, there is neg. cooperativity between the rings, so that they act in alternation.  It is remarkable that a process as specific as protein folding can be guided by the chaperonin machine in a way largely independent of substrate protein structure or sequence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTPmznXddOjrVg90H21EOLACvtfcHk0ljYUmKHhrShJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVylsrjF&md5=b070d6535d8e5b7df1bfea2c0c009b5b</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.11.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.11.028%26sid%3Dliteratum%253Aachs%26aulast%3DSaibil%26aufirst%3DH.%2BR.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DClare%26aufirst%3DD.%2BK.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520and%2520allostery%2520of%2520the%2520chaperonin%2520GroEL%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D425%26spage%3D1476%26epage%3D1487%26doi%3D10.1016%2Fj.jmb.2012.11.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farr, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usaite, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furtak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hondorp, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pypaert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span> <span> </span><span class="NLM_article-title">Global aggregation of newly translated proteins in an Escherichia coli strain deficient of the chaperonin GroEL</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">15800</span>– <span class="NLM_lpage">15805</span>, <span class="refDoi"> DOI: 10.1073/pnas.0607534103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1073%2Fpnas.0607534103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=17043235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFymtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=15800-15805&author=E.+Chapmanauthor=G.+W.+Farrauthor=R.+Usaiteauthor=K.+Furtakauthor=W.+A.+Fentonauthor=T.+K.+Chaudhuriauthor=E.+R.+Hondorpauthor=R.+G.+Matthewsauthor=S.+G.+Wolfauthor=J.+R.+Yatesauthor=M.+Pypaertauthor=A.+L.+Horwich&title=Global+aggregation+of+newly+translated+proteins+in+an+Escherichia+coli+strain+deficient+of+the+chaperonin+GroEL&doi=10.1073%2Fpnas.0607534103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Global aggregation of newly translated proteins in an Escherichia coli strain deficient of the chaperonin GroEL</span></div><div class="casAuthors">Chapman, Eli; Farr, George W.; Usaite, Renata; Furtak, Krystyna; Fenton, Wayne A.; Chaudhuri, Tapan K.; Hondorp, Elise R.; Matthews, Rowena G.; Wolf, Sharon G.; Yates, John R.; Pypaert, Marc; Horwich, Arthur L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15800-15805</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In a newly isolated temp.-sensitive lethal Escherichia coli mutant affecting the chaperonin GroEL, we obsd. wholesale aggregation of newly translated proteins.  After temp. shift, transcription, translation, and growth slowed over two to three generations, accompanied by filamentation and accretion (in ≈2% of cells) of paracryst. arrays contg. mutant chaperonin complex.  A biochem. isolated inclusion body fraction contained the collective of abundant proteins of the bacterial cytoplasm as detd. by SDS/PAGE and proteolysis/MS analyses.  Pulse-chase expts. revealed that newly made proteins, but not preexistent ones, were recruited to this insol. fraction.  Although aggregation of "stringent" GroEL/GroES-dependent substrates may secondarily produce an "avalanche" of aggregation, the observations raise the possibility, supported by in vitro refolding expts., that the widespread aggregation reflects that GroEL function supports the proper folding of a majority of newly translated polypeptides, not just the limited no. indicated by interaction studies and in vitro expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCPMpZGkZW77Vg90H21EOLACvtfcHk0ljYUmKHhrShJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFymtbbL&md5=d153b49e89b7c2d3251b0d7d5b21c419</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0607534103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0607534103%26sid%3Dliteratum%253Aachs%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DFarr%26aufirst%3DG.%2BW.%26aulast%3DUsaite%26aufirst%3DR.%26aulast%3DFurtak%26aufirst%3DK.%26aulast%3DFenton%26aufirst%3DW.%2BA.%26aulast%3DChaudhuri%26aufirst%3DT.%2BK.%26aulast%3DHondorp%26aufirst%3DE.%2BR.%26aulast%3DMatthews%26aufirst%3DR.%2BG.%26aulast%3DWolf%26aufirst%3DS.%2BG.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DPypaert%26aufirst%3DM.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520aggregation%2520of%2520newly%2520translated%2520proteins%2520in%2520an%2520Escherichia%2520coli%2520strain%2520deficient%2520of%2520the%2520chaperonin%2520GroEL%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D15800%26epage%3D15805%26doi%3D10.1073%2Fpnas.0607534103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fayet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegelhoffer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georgopoulos, C.</span></span> <span> </span><span class="NLM_article-title">The groES and groEL heat shock gene products of Escherichia coli are essential for bacterial growth at all temperatures</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1385</span>, <span class="refDoi"> DOI: 10.1128/jb.171.3.1379-1385.1989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1128%2Fjb.171.3.1379-1385.1989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=2563997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaL1MXhs1OjtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=1989&pages=1379-1385&author=O.+Fayetauthor=T.+Ziegelhofferauthor=C.+Georgopoulos&title=The+groES+and+groEL+heat+shock+gene+products+of+Escherichia+coli+are+essential+for+bacterial+growth+at+all+temperatures&doi=10.1128%2Fjb.171.3.1379-1385.1989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The groES and groEL heat shock gene products of Escherichia coli are essential for bacterial growth at all temperatures</span></div><div class="casAuthors">Fayet, Olivier; Ziegelhoffer, Thomas; Georgopoulos, Costa</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1379-85</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    </div><div class="casAbstract">The products of the groES and groEL genes of E. coli, constituting the groE operon, are known to be required for growth at high temp. (42°) and are members of the heat shock regulon.  Using a genetic approach, the requirement for these gene products for bacterial growth at low temp. (17 to 30°) was investigated.  To do this, various groES groEL heterodiploid deriv. strains were constructed.  By inactivating 1 of the groE operons by a polar insertion, it was shown by bacteriophage P1 transduction that at least 1 of the groE genes was essential for growth at low temp.  Further P1 transduction expts. with strains that were heterodiploid for only 1 of the groE genes demonstrated that both groE gene products were required for growth at low temp., which suggested a fundamental role for the groE proteins in E. coli growth and physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpp9Cn9oLtKQLVg90H21EOLACvtfcHk0ljYUmKHhrShJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhs1OjtL8%253D&md5=5ffafff6732ccfd78cfa42d82432ce75</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2Fjb.171.3.1379-1385.1989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.171.3.1379-1385.1989%26sid%3Dliteratum%253Aachs%26aulast%3DFayet%26aufirst%3DO.%26aulast%3DZiegelhoffer%26aufirst%3DT.%26aulast%3DGeorgopoulos%26aufirst%3DC.%26atitle%3DThe%2520groES%2520and%2520groEL%2520heat%2520shock%2520gene%2520products%2520of%2520Escherichia%2520coli%2520are%2520essential%2520for%2520bacterial%2520growth%2520at%2520all%2520temperatures%26jtitle%3DJ.%2520Bacteriol.%26date%3D1989%26volume%3D171%26spage%3D1379%26epage%3D1385%26doi%3D10.1128%2Fjb.171.3.1379-1385.1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharif, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span> <span> </span><span class="NLM_article-title">A biochemical screen for GroEL/GroES inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.12.100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2013.12.100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=24418775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Oguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=786-789&author=S.+M.+Johnsonauthor=O.+Sharifauthor=P.+A.+Makauthor=H.+T.+Wangauthor=I.+H.+Engelsauthor=A.+Brinkerauthor=P.+G.+Schultzauthor=A.+L.+Horwichauthor=E.+Chapman&title=A+biochemical+screen+for+GroEL%2FGroES+inhibitors&doi=10.1016%2Fj.bmcl.2013.12.100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A biochemical screen for GroEL/GroES inhibitors</span></div><div class="casAuthors">Johnson, Steven M.; Sharif, Orzala; Mak, Puiying Annie; Wang, Hsiao-Ting; Engels, Ingo H.; Brinker, Achim; Schultz, Peter G.; Horwich, Arthur L.; Chapman, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">786-789</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">High-throughput screening of 700,000 small mols. has identified 235 inhibitors of the GroEL/GroES-mediated refolding cycle.  Dose-response anal. of a subset of these hits revealed that 21 compds. are potent inhibitors of GroEL/GroES-mediated refolding (IC50 <10 μM).  The screening results presented herein represent the first steps in a broader aim of developing mol. probes to study chaperonin biochem. and physiol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8ZxlFMFjvarVg90H21EOLACvtfcHk0liYwIyFUX6Ogg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Oguw%253D%253D&md5=babc99bf5fc9a4f1b9398bcc64014fa5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.12.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.12.100%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DSharif%26aufirst%3DO.%26aulast%3DMak%26aufirst%3DP.%2BA.%26aulast%3DWang%26aufirst%3DH.%2BT.%26aulast%3DEngels%26aufirst%3DI.%2BH.%26aulast%3DBrinker%26aufirst%3DA.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DChapman%26aufirst%3DE.%26atitle%3DA%2520biochemical%2520screen%2520for%2520GroEL%252FGroES%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D786%26epage%3D789%26doi%3D10.1016%2Fj.bmcl.2013.12.100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdeen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mammadova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrose, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span> <span> </span><span class="NLM_article-title">GroEL/ES inhibitors as potential antibiotics</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3127</span>– <span class="NLM_lpage">3134</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2016.04.089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=27184767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC28XotV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=3127-3134&author=S.+Abdeenauthor=N.+Salimauthor=N.+Mammadovaauthor=C.+M.+Summersauthor=R.+Franksonauthor=A.+J.+Ambroseauthor=G.+G.+Andersonauthor=P.+G.+Schultzauthor=A.+L.+Horwichauthor=E.+Chapmanauthor=S.+M.+Johnson&title=GroEL%2FES+inhibitors+as+potential+antibiotics&doi=10.1016%2Fj.bmcl.2016.04.089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GroEL/ES inhibitors as potential antibiotics</span></div><div class="casAuthors">Abdeen, Sanofar; Salim, Nilshad; Mammadova, Najiba; Summers, Corey M.; Frankson, Rochelle; Ambrose, Andrew J.; Anderson, Gregory G.; Schultz, Peter G.; Horwich, Arthur L.; Chapman, Eli; Johnson, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3127-3134</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We recently reported results from a high-throughput screening effort that identified 235 inhibitors of the Escherichia coli GroEL/ES chaperonin system [Bioorg. Med. Chem. Lett.2014, 24, 786].  As the GroEL/ES chaperonin system is essential for growth under all conditions, we reasoned that targeting GroEL/ES with small mol. inhibitors could be a viable antibacterial strategy.  Extending from our initial screen, we report here the antibacterial activities of 22 GroEL/ES inhibitors against a panel of Gram-pos. and Gram-neg. bacteria, including E. coli, Bacillus subtilis, Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter cloacae.  GroEL/ES inhibitors were more effective at blocking the proliferation of Gram-pos. bacteria, in particular S. aureus, where lead compds. exhibited antibiotic effects from the low-μM to mid-nM range.  While several compds. inhibited the human HSP60/10 refolding cycle, some were able to selectively target the bacterial GroEL/ES system.  Despite inhibiting HSP60/10, many compds. exhibited low to no cytotoxicity against human liver and kidney cell lines.  Two lead candidates emerged from the panel, that exhibit >50-fold selectivity for inhibiting S. aureus growth compared to liver or kidney cell cytotoxicity.  These compds. inhibited drug-sensitive and methicillin-resistant S. aureus strains with potencies comparable to vancomycin, daptomycin, and streptomycin, and are promising candidates to explore for validating the GroEL/ES chaperonin system as a viable antibiotic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrazQ1vHFwVlrVg90H21EOLACvtfcHk0liYwIyFUX6Ogg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotV2gsrk%253D&md5=d2bfc6eb7a2a02d3f91961f13b22bff3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.089%26sid%3Dliteratum%253Aachs%26aulast%3DAbdeen%26aufirst%3DS.%26aulast%3DSalim%26aufirst%3DN.%26aulast%3DMammadova%26aufirst%3DN.%26aulast%3DSummers%26aufirst%3DC.%2BM.%26aulast%3DFrankson%26aufirst%3DR.%26aulast%3DAmbrose%26aufirst%3DA.%2BJ.%26aulast%3DAnderson%26aufirst%3DG.%2BG.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26atitle%3DGroEL%252FES%2520inhibitors%2520as%2520potential%2520antibiotics%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D3127%26epage%3D3134%26doi%3D10.1016%2Fj.bmcl.2016.04.089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdeen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salim, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mammadova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldsmith-Pestana, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon-Pratt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horwich, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the HSP60/10 chaperonin systems of Trypanosoma brucei as a strategy for treating African sleeping sickness</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5247</span>– <span class="NLM_lpage">5253</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2016.09.051" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=5247-5253&author=S.+Abdeenauthor=N.+Salimauthor=N.+Mammadovaauthor=C.+M.+Summersauthor=K.+Goldsmith-Pestanaauthor=D.+McMahon-Prattauthor=P.+G.+Schultzauthor=A.+L.+Horwichauthor=E.+Chapmanauthor=S.+M.+Johnson&title=Targeting+the+HSP60%2F10+chaperonin+systems+of+Trypanosoma+brucei+as+a+strategy+for+treating+African+sleeping+sickness&doi=10.1016%2Fj.bmcl.2016.09.051"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.051%26sid%3Dliteratum%253Aachs%26aulast%3DAbdeen%26aufirst%3DS.%26aulast%3DSalim%26aufirst%3DN.%26aulast%3DMammadova%26aufirst%3DN.%26aulast%3DSummers%26aufirst%3DC.%2BM.%26aulast%3DGoldsmith-Pestana%26aufirst%3DK.%26aulast%3DMcMahon-Pratt%26aufirst%3DD.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26aulast%3DHorwich%26aufirst%3DA.%2BL.%26aulast%3DChapman%26aufirst%3DE.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26atitle%3DTargeting%2520the%2520HSP60%252F10%2520chaperonin%2520systems%2520of%2520Trypanosoma%2520brucei%2520as%2520a%2520strategy%2520for%2520treating%2520African%2520sleeping%2520sickness%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D5247%26epage%3D5253%26doi%3D10.1016%2Fj.bmcl.2016.09.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiegand, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilpert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E.</span></span> <span> </span><span class="NLM_article-title">Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">175</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.521</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1038%2Fnprot.2007.521" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=18274517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvVyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=163-175&author=I.+Wiegandauthor=K.+Hilpertauthor=R.+E.+Hancock&title=Agar+and+broth+dilution+methods+to+determine+the+minimal+inhibitory+concentration+%28MIC%29+of+antimicrobial+substances&doi=10.1038%2Fnprot.2007.521"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Agar and broth dilution methods to determine the minimal inhibitory concentration (MIC) of antimicrobial substances</span></div><div class="casAuthors">Wiegand, Irith; Hilpert, Kai; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-175</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The aim of broth and agar diln. methods is to det. the lowest concn. of the assayed antimicrobial agent (minimal inhibitory concn., MIC) that, under defined test conditions, inhibits the visible growth of the bacterium being investigated.  MIC values are used to det. susceptibilities of bacteria to drugs and also to evaluate the activity of new antimicrobial agents.  Agar diln. involves the incorporation of different concns. of the antimicrobial substance into a nutrient agar medium followed by the application of a standardized no. of cells to the surface of the agar plate.  For broth diln., often detd. in 96-well microtiter plate format, bacteria are inoculated into a liq. growth medium in the presence of different concns. of an antimicrobial agent.  Growth is assessed after incubation for a defined period of time (16-20 h) and the MIC value is read.  This protocol applies only to aerobic bacteria and can be completed in 3 d.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd1uwvRDLPQ7Vg90H21EOLACvtfcHk0lj2Q-vZdq-skQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvVyls78%253D&md5=a235d207868342de52085d4100062471</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.521%26sid%3Dliteratum%253Aachs%26aulast%3DWiegand%26aufirst%3DI.%26aulast%3DHilpert%26aufirst%3DK.%26aulast%3DHancock%26aufirst%3DR.%2BE.%26atitle%3DAgar%2520and%2520broth%2520dilution%2520methods%2520to%2520determine%2520the%2520minimal%2520inhibitory%2520concentration%2520%2528MIC%2529%2520of%2520antimicrobial%2520substances%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D163%26epage%3D175%26doi%3D10.1038%2Fnprot.2007.521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Toward optimization of the linker substructure common to transthyretin amyloidogenesis inhibitors using biochemical and structural studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">6348</span>– <span class="NLM_lpage">6358</span>, <span class="refDoi"> DOI: 10.1021/jm800435s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800435s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFGntrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=6348-6358&author=S.+M.+Johnsonauthor=S.+Connellyauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Toward+optimization+of+the+linker+substructure+common+to+transthyretin+amyloidogenesis+inhibitors+using+biochemical+and+structural+studies&doi=10.1021%2Fjm800435s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Optimization of the Linker Substructure Common to Transthyretin Amyloidogenesis Inhibitors Using Biochemical and Structural Studies</span></div><div class="casAuthors">Johnson, Steven M.; Connelly, Stephen; Wilson, Ian A.; Kelly, Jeffery W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6348-6358</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To develop potent and highly selective transthyretin (TTR) amyloidogenesis inhibitors, it is useful to systematically optimize the three substructural elements that compose a typical TTR kinetic stabilizer: the two aryl rings and the linker joining them.  Herein, the authors evaluated 40 bisaryl mols. based on 10 unique linker substructures to det. how these linkages influence inhibitor potency and selectivity.  These linkers connect one unsubstituted arom. ring to either a 3,5-X2 or a 3,5-X2-4-OH Ph substructure (X = Br or CH3).  Consideration of amyloid inhibition and ex vivo plasma TTR binding selectivity data reveal that direct connection of the two aryls or linkage through nonpolar E-olefin or -CH2CH2- substructures generates the most potent and selective TTR amyloidogenesis inhibitors exhibiting minimal undesirable binding to the thyroid hormone nuclear receptor or the COX-1 enzyme.  Five high-resoln. TTR·inhibitor crystal structures (1.4-1.8 Å) provide insight into why such linkers afford inhibitors with greater potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI-adFkm-anbVg90H21EOLACvtfcHk0lj2Q-vZdq-skQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFGntrvL&md5=43789d84a079d73268685760f36262c3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm800435s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800435s%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DToward%2520optimization%2520of%2520the%2520linker%2520substructure%2520common%2520to%2520transthyretin%2520amyloidogenesis%2520inhibitors%2520using%2520biochemical%2520and%2520structural%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D6348%26epage%3D6358%26doi%3D10.1021%2Fjm800435s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, E. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span> <span> </span><span class="NLM_article-title">Semi-quantitative models for identifying potent and selective transthyretin amyloidogenesis inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">3441</span>– <span class="NLM_lpage">3449</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2Fj.bmcl.2017.05.080" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=3441-3449&author=S.+Connellyauthor=D.+E.+Mortensonauthor=S.+Choiauthor=I.+A.+Wilsonauthor=E.+T.+Powersauthor=J.+W.+Kellyauthor=S.+M.+Johnson&title=Semi-quantitative+models+for+identifying+potent+and+selective+transthyretin+amyloidogenesis+inhibitors&doi=10.1016%2Fj.bmcl.2017.05.080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.080%26sid%3Dliteratum%253Aachs%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DMortenson%26aufirst%3DD.%2BE.%26aulast%3DChoi%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DPowers%26aufirst%3DE.%2BT.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26atitle%3DSemi-quantitative%2520models%2520for%2520identifying%2520potent%2520and%2520selective%2520transthyretin%2520amyloidogenesis%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D3441%26epage%3D3449%26doi%3D10.1016%2Fj.bmcl.2017.05.080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Toward optimization of the second aryl substructure common to transthyretin amyloidogenesis inhibitors using biochemical and structural studies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1115</span>– <span class="NLM_lpage">1125</span>, <span class="refDoi"> DOI: 10.1021/jm801347s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801347s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1115-1125&author=S.+M.+Johnsonauthor=S.+Connellyauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Toward+optimization+of+the+second+aryl+substructure+common+to+transthyretin+amyloidogenesis+inhibitors+using+biochemical+and+structural+studies&doi=10.1021%2Fjm801347s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Toward Optimization of the Second Aryl Substructure Common to Transthyretin Amyloidogenesis Inhibitors Using Biochemical and Structural Studies</span></div><div class="casAuthors">Johnson, Steven M.; Connelly, Stephen; Wilson, Ian A.; Kelly, Jeffery W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1115-1125</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Transthyretin (TTR) amyloidogenesis inhibitors are typically composed of two arom. rings and a linker.  We have previously established optimal structures for one arom. ring and the linker.  Herein, we employ a suboptimal linker and an optimal aryl-X substructure to rank order the desirability of aryl-Z substructures-using a library of 56 N-(3,5-dibromo-4-hydroxyphenyl)benzamides.  Coconsideration of amyloid inhibition potency and ex vivo plasma TTR binding selectivity data reveal that 2,6, 2,5, 2, 3,4,5, and 3,5 substituted aryls bearing small substituents generate the most potent and selective inhibitors, in descending order.  These benzamides generally lack undesirable thyroid hormone receptor binding and COX-1 inhibition activity.  Three high-resoln. TTR·inhibitor crystal structures (1.31-1.35 Å) provide insight into why these inhibitors are potent and selective, enabling future structure-based design of TTR kinetic stabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCf9iuLPAegLVg90H21EOLACvtfcHk0lj2Q-vZdq-skQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSltLg%253D&md5=77591581a43967e91119239f94302306</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm801347s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801347s%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DToward%2520optimization%2520of%2520the%2520second%2520aryl%2520substructure%2520common%2520to%2520transthyretin%2520amyloidogenesis%2520inhibitors%2520using%2520biochemical%2520and%2520structural%2520studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1115%26epage%3D1125%26doi%3D10.1021%2Fjm801347s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tipparaju, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyasawal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pieroni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozikowski, A. P.</span></span> <span> </span><span class="NLM_article-title">In pursuit of natural product leads: Synthesis and biological evaluation of 2-[3-hydroxy-2-[(3-hydroxypyridine-2-carbonyl)amino]phenyl]benzoxazole-4-carboxylic acid (A-33853) and its analogues: Discovery of N-(2-benzoxazol-2-ylphenyl)benzamides as novel antileishmanial chemotypes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">7344</span>– <span class="NLM_lpage">7347</span>, <span class="refDoi"> DOI: 10.1021/jm801241n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801241n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7344-7347&author=S.+K.+Tipparajuauthor=S.+Joyasawalauthor=M.+Pieroniauthor=M.+Kaiserauthor=R.+Brunauthor=A.+P.+Kozikowski&title=In+pursuit+of+natural+product+leads%3A+Synthesis+and+biological+evaluation+of+2-%5B3-hydroxy-2-%5B%283-hydroxypyridine-2-carbonyl%29amino%5Dphenyl%5Dbenzoxazole-4-carboxylic+acid+%28A-33853%29+and+its+analogues%3A+Discovery+of+N-%282-benzoxazol-2-ylphenyl%29benzamides+as+novel+antileishmanial+chemotypes&doi=10.1021%2Fjm801241n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm801241n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801241n%26sid%3Dliteratum%253Aachs%26aulast%3DTipparaju%26aufirst%3DS.%2BK.%26aulast%3DJoyasawal%26aufirst%3DS.%26aulast%3DPieroni%26aufirst%3DM.%26aulast%3DKaiser%26aufirst%3DM.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26atitle%3DIn%2520pursuit%2520of%2520natural%2520product%2520leads%253A%2520Synthesis%2520and%2520biological%2520evaluation%2520of%25202-%255B3-hydroxy-2-%255B%25283-hydroxypyridine-2-carbonyl%2529amino%255Dphenyl%255Dbenzoxazole-4-carboxylic%2520acid%2520%2528A-33853%2529%2520and%2520its%2520analogues%253A%2520Discovery%2520of%2520N-%25282-benzoxazol-2-ylphenyl%2529benzamides%2520as%2520novel%2520antileishmanial%2520chemotypes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7344%26epage%3D7347%26doi%3D10.1021%2Fjm801241n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, R.
L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelker, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Kinetic stabilization of an oligomeric protein by a single ligand binding event</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">5540</span>– <span class="NLM_lpage">5551</span>, <span class="refDoi"> DOI: 10.1021/ja042929f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja042929f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BD2MXis1Ohsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=5540-5551&author=R.%0AL.+Wisemanauthor=S.+M.+Johnsonauthor=M.+S.+Kelkerauthor=T.+Fossauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Kinetic+stabilization+of+an+oligomeric+protein+by+a+single+ligand+binding+event&doi=10.1021%2Fja042929f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Stabilization of an Oligomeric Protein by a Single Ligand Binding Event</span></div><div class="casAuthors">Wiseman, R. Luke; Johnson, Steven M.; Kelker, Matthew S.; Foss, Ted; Wilson, Ian A.; Kelly, Jeffery W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5540-5551</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein native state stabilization imposed by small mol. binding is an attractive strategy to prevent the misfolding and misassembly processes assocd. with amyloid diseases.  Transthyretin (TTR) amyloidogenesis requires rate-limiting tetramer dissocn. before misassembly of a partially denatured monomer ensues.  Selective stabilization of the native TTR tetramer over the dissociative transition state by small mol. binding to both thyroxine binding sites raises the kinetic barrier of tetramer dissocn., preventing amyloidogenesis.  Assessing the amyloidogenicity of a TTR tetramer having only one amyloidogenesis inhibitor (I) bound is challenging because the two small mol. binding consts. are generally not distinct enough to allow for the exclusive formation of TTR·I in soln. to the exclusion of TTR·I2 and unliganded TTR.  Herein, we report a method to tether one fibril formation inhibitor to TTR by disulfide bond formation.  Occupancy of only one of the two thyroxine binding sites is sufficient to inhibit tetramer dissocn. in 6.0 M urea and amyloidogenesis under acidic conditions by imposing kinetic stabilization on the entire tetramer.  The sufficiency of single occupancy for stabilizing the native state of TTR provides the incentive to search for compds. displaying striking neg. binding cooperativity (e.g., Kd1 in nanomolar range and Kd2 in the micromolar to millimolar range), enabling lower doses of inhibitor to be employed in the clinic, mitigating potential side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq6dHdnympB7Vg90H21EOLACvtfcHk0lgxymVohWFhZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXis1Ohsb4%253D&md5=a027c65727cfb52014d9dca606417915</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja042929f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042929f%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DR.%2BL.%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DKelker%26aufirst%3DM.%2BS.%26aulast%3DFoss%26aufirst%3DT.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DKinetic%2520stabilization%2520of%2520an%2520oligomeric%2520protein%2520by%2520a%2520single%2520ligand%2520binding%2520event%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D5540%26epage%3D5551%26doi%3D10.1021%2Fja042929f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, J. W.</span></span> <span> </span><span class="NLM_article-title">Biochemical and structural evaluation of highly selective 2-arylbenzoxazole-based transthyretin amyloidogenesis inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">270</span>, <span class="refDoi"> DOI: 10.1021/jm0708735</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0708735" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=260-270&author=S.+M.+Johnsonauthor=S.+Connellyauthor=I.+A.+Wilsonauthor=J.+W.+Kelly&title=Biochemical+and+structural+evaluation+of+highly+selective+2-arylbenzoxazole-based+transthyretin+amyloidogenesis+inhibitors&doi=10.1021%2Fjm0708735"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm0708735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0708735%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DS.%2BM.%26aulast%3DConnelly%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DKelly%26aufirst%3DJ.%2BW.%26atitle%3DBiochemical%2520and%2520structural%2520evaluation%2520of%2520highly%2520selective%25202-arylbenzoxazole-based%2520transthyretin%2520amyloidogenesis%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D260%26epage%3D270%26doi%3D10.1021%2Fjm0708735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wiseman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L. N.</span></span> <span> </span><span class="NLM_article-title">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0003-2697(89)90213-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1016%2F0003-2697%2889%2990213-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=2757186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1989&pages=131-137&author=T.+Wisemanauthor=S.+Willistonauthor=J.+F.+Brandtsauthor=L.+N.+Lin&title=Rapid+measurement+of+binding+constants+and+heats+of+binding+using+a+new+titration+calorimeter&doi=10.1016%2F0003-2697%2889%2990213-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid measurement of binding constants and heats of binding using a new titration calorimeter</span></div><div class="casAuthors">Wiseman, Thomas; Williston, Samuel; Brandts, John F.; Lin, Lung Nan</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-7</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A new titrn. calorimeter is described and results are presented for the binding of 2'-CMP to the active site of RNase A.  The instrument characteristics include very high sensitivity, rapid calorimetric response, and fast thermal equilibration.  Convenient software is available for instrument operation, data collection, data redn., and deconvolution to obtain least-squares ests. of binding parameters n, ΔH°, ΔS°, and the binding const. K.  Sample through-put for the instrument is high, and under favorable conditions binding consts. as large as 108 M-1 can be measured.  The bovine RNase A (RNase)/2'-CMP system was studied over a 50-fold range of RNase concn. and at 2 different temps.  The binding consts. were in the 105 to 106 M-1 range, depending on conditions, and heats of binding ∼-15,000 cal/mol.  Repeat detns. suggested errors of only a few percent in n, ΔH°, and K values over the most favorable concn. range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJRWydqrfWArVg90H21EOLACvtfcHk0lgxymVohWFhZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXktlKjsrw%253D&md5=53fcdb7144fd85e57963add0bcdb6041</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2889%2990213-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252889%252990213-3%26sid%3Dliteratum%253Aachs%26aulast%3DWiseman%26aufirst%3DT.%26aulast%3DWilliston%26aufirst%3DS.%26aulast%3DBrandts%26aufirst%3DJ.%2BF.%26aulast%3DLin%26aufirst%3DL.%2BN.%26atitle%3DRapid%2520measurement%2520of%2520binding%2520constants%2520and%2520heats%2520of%2520binding%2520using%2520a%2520new%2520titration%2520calorimeter%26jtitle%3DAnal.%2520Biochem.%26date%3D1989%26volume%3D179%26spage%3D131%26epage%3D137%26doi%3D10.1016%2F0003-2697%2889%2990213-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishony, R.</span></span> <span> </span><span class="NLM_article-title">Alternating antibiotic treatments constrain evolutionary paths to multidrug resistance</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">14494</span>– <span class="NLM_lpage">14499</span>, <span class="refDoi"> DOI: 10.1073/pnas.1409800111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=10.1073%2Fpnas.1409800111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=25246554" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=14494-14499&author=S.+Kimauthor=T.+D.+Liebermanauthor=R.+Kishony&title=Alternating+antibiotic+treatments+constrain+evolutionary+paths+to+multidrug+resistance&doi=10.1073%2Fpnas.1409800111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Alternating antibiotic treatments constrain evolutionary paths to multidrug resistance</span></div><div class="casAuthors">Kim, Seungsoo; Lieberman, Tami D.; Kishony, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">14494-14499</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Alternating antibiotic therapy, in which pairs of drugs are cycled during treatment, has been suggested as a means to inhibit the evolution of de novo resistance while avoiding the toxicity assocd. with more traditional combination therapy.  However, it remains unclear under which conditions and by what means such alternating treatments impede the evolution of resistance.  Here, the authors tracked multistep evolution of resistance in replicate populations of Staphylococcus aureus during 22 d of continuously increasing single, mixed, and alternating drug treatment.  In all three tested drug pairs, the alternating treatment reduced the overall rate of resistance by slowing the acquisition of resistance to one of the two component drugs, sometimes as effectively as mixed treatment.  This slower rate of evolution is reflected in the genome-wide mutational profiles; under alternating treatments, bacteria acquire mutations in different genes than under corresponding single-drug treatments.  To test whether this obsd. constraint on adaptive paths reflects trade-offs in which resistance to one drug is accompanied by sensitivity to a second drug, we profiled many single-step mutants for cross-resistance.  Indeed, the av. cross-resistance of single-step mutants can help predict whether or not evolution was slower in alternating drugs.  Together, these results show that despite the complex evolutionary landscape of multidrug resistance, alternating-drug therapy can slow evolution by constraining the mutational paths toward resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1oDn_1SS_IbVg90H21EOLACvtfcHk0lgxymVohWFhZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyhtbbL&md5=5b019b5e5f74aa9014f12a891f93bbaa</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1409800111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1409800111%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLieberman%26aufirst%3DT.%2BD.%26aulast%3DKishony%26aufirst%3DR.%26atitle%3DAlternating%2520antibiotic%2520treatments%2520constrain%2520evolutionary%2520paths%2520to%2520multidrug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D14494%26epage%3D14499%26doi%3D10.1073%2Fpnas.1409800111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fleeman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaVoi, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nefzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welmaker, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giulianotti, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houghten, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, L. N.</span></span> <span> </span><span class="NLM_article-title">Combinatorial libraries as a tool for the discovery of novel, broad-spectrum antibacterial agents targeting the ESKAPE pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3340</span>– <span class="NLM_lpage">3355</span>, <span class="refDoi"> DOI: 10.1021/jm501628s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501628s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVOrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3340-3355&author=R.+Fleemanauthor=T.+M.+LaVoiauthor=R.+G.+Santosauthor=A.+Moralesauthor=A.+Nefziauthor=G.+S.+Welmakerauthor=J.+L.+Medina-Francoauthor=M.+A.+Giulianottiauthor=R.+A.+Houghtenauthor=L.+N.+Shaw&title=Combinatorial+libraries+as+a+tool+for+the+discovery+of+novel%2C+broad-spectrum+antibacterial+agents+targeting+the+ESKAPE+pathogens&doi=10.1021%2Fjm501628s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial Libraries As a Tool for the Discovery of Novel, Broad-Spectrum Antibacterial Agents Targeting the ESKAPE Pathogens</span></div><div class="casAuthors">Fleeman, Renee; LaVoi, Travis M.; Santos, Radleigh G.; Morales, Angela; Nefzi, Adel; Welmaker, Gregory S.; Medina-Franco, Jose L.; Giulianotti, Marc A.; Houghten, Richard A.; Shaw, Lindsey N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3340-3355</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mixt. based synthetic combinatorial libraries offer a tremendous enhancement for the rate of drug discovery, allowing the activity of millions of compds. to be assessed through the testing of exponentially fewer samples.  In this study, the authors used a scaffold-ranking library to screen 37 different libraries for antibacterial activity against the ESKAPE pathogens.  Each library contained between 10000 and 750000 structural analogs for a total of >6 million compds.  From this, the authors identified a bis-cyclic guanidine library that displayed strong antibacterial activity.  A positional scanning library for these compds. was developed and used to identify the most effective functional groups at each variant position.  Individual compds. were synthesized that were broadly active against all ESKAPE organisms at concns. <2 μM.  In addn., these compds. were bactericidal, had antibiofilm effects, showed limited potential for the development of resistance, and displayed almost no toxicity when tested against human lung cells and erythrocytes.  Using a murine model of peritonitis, the authors also demonstrate that these agents are highly efficacious in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQZewS666QerVg90H21EOLACvtfcHk0lha8GQJPif0cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVOrtro%253D&md5=a8c23a0c8c31166c5caedf632b71348f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm501628s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501628s%26sid%3Dliteratum%253Aachs%26aulast%3DFleeman%26aufirst%3DR.%26aulast%3DLaVoi%26aufirst%3DT.%2BM.%26aulast%3DSantos%26aufirst%3DR.%2BG.%26aulast%3DMorales%26aufirst%3DA.%26aulast%3DNefzi%26aufirst%3DA.%26aulast%3DWelmaker%26aufirst%3DG.%2BS.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DGiulianotti%26aufirst%3DM.%2BA.%26aulast%3DHoughten%26aufirst%3DR.%2BA.%26aulast%3DShaw%26aufirst%3DL.%2BN.%26atitle%3DCombinatorial%2520libraries%2520as%2520a%2520tool%2520for%2520the%2520discovery%2520of%2520novel%252C%2520broad-spectrum%2520antibacterial%2520agents%2520targeting%2520the%2520ESKAPE%2520pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3340%26epage%3D3355%26doi%3D10.1021%2Fjm501628s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i36"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00989">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80483"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00989">10.1021/acs.jmedchem.8b00989</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">SMILES strings of compound structures (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_001.csv">CSV</a>)</p></li><li><p class="inline">Tabulation of all biochemical IC<sub>50</sub>, bacterial proliferation EC<sub>50</sub>, and human cell viability CC<sub>50</sub> results; log(IC<sub>50</sub>), log(EC<sub>50</sub>), and log(CC<sub>50</sub>) results with standard deviations; plots of statistical differences between the R- and L-series analogs in the various assays; synthetic protocols and characterization data for test compounds (HPLC purity, MS, and <sup>1</sup>H NMR); experimental protocols for protein synthesis and purification, and biochemical, bacterial proliferation, and human cell viability assays (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_001.csv">jm8b00989_si_001.csv (7.76 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00989/suppl_file/jm8b00989_si_002.pdf">jm8b00989_si_002.pdf (1.33 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00989&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-16%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00989%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00989" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679985e0ff9c3de3","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
