
<html lang="en"     class="pb-page"  data-request-id="16384adb-43d0-4161-8a53-e936ca093c15"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01084;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Series of Novel and Highly Potent Cyclic Peptide PCSK9 Inhibitors Derived from an mRNA Display Screen and Optimized via Structure-Based Design" /></meta><meta name="dc.Creator" content="Candice  Alleyne" /></meta><meta name="dc.Creator" content="Rupesh P.  Amin" /></meta><meta name="dc.Creator" content="Bhavana  Bhatt" /></meta><meta name="dc.Creator" content="Elisabetta  Bianchi" /></meta><meta name="dc.Creator" content="J. Craig  Blain" /></meta><meta name="dc.Creator" content="Nicolas  Boyer" /></meta><meta name="dc.Creator" content="Danila  Branca" /></meta><meta name="dc.Creator" content="Mark W.  Embrey" /></meta><meta name="dc.Creator" content="Sookhee N.  Ha" /></meta><meta name="dc.Creator" content="Kelli  Jette" /></meta><meta name="dc.Creator" content="Douglas G.  Johns" /></meta><meta name="dc.Creator" content="Angela D.  Kerekes" /></meta><meta name="dc.Creator" content="Kenneth A.  Koeplinger" /></meta><meta name="dc.Creator" content="Derek  LaPlaca" /></meta><meta name="dc.Creator" content="Nianyu  Li" /></meta><meta name="dc.Creator" content="Beth  Murphy" /></meta><meta name="dc.Creator" content="Peter  Orth" /></meta><meta name="dc.Creator" content="Alonso  Ricardo" /></meta><meta name="dc.Creator" content="Scott  Salowe" /></meta><meta name="dc.Creator" content="Kathleen  Seyb" /></meta><meta name="dc.Creator" content="Aurash  Shahripour" /></meta><meta name="dc.Creator" content="Joseph R.  Stringer" /></meta><meta name="dc.Creator" content="Yili  Sun" /></meta><meta name="dc.Creator" content="Rodger  Tracy" /></meta><meta name="dc.Creator" content="Chengwei  Wu" /></meta><meta name="dc.Creator" content="Yusheng  Xiong" /></meta><meta name="dc.Creator" content="Hyewon  Youm" /></meta><meta name="dc.Creator" content="Hratch J.  Zokian" /></meta><meta name="dc.Creator" content="Thomas J.  Tucker" /></meta><meta name="dc.Description" content="Proprotein convertase subtilisin-like/kexin type 9 (PCSK9) is a key regulator of plasma LDL-cholesterol (LDL-C) and a clinically validated target for the treatment of hypercholesterolemia and coron..." /></meta><meta name="Description" content="Proprotein convertase subtilisin-like/kexin type 9 (PCSK9) is a key regulator of plasma LDL-cholesterol (LDL-C) and a clinically validated target for the treatment of hypercholesterolemia and coron..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 10, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01084" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01084" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01084" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01084" /></link>
        
    
    

<title>Series of Novel and Highly Potent Cyclic Peptide PCSK9 Inhibitors Derived from an mRNA Display Screen and Optimized via Structure-Based Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01084" /></meta><meta property="og:title" content="Series of Novel and Highly Potent Cyclic Peptide PCSK9 Inhibitors Derived from an mRNA Display Screen and Optimized via Structure-Based Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0037.jpeg" /></meta><meta property="og:description" content="Proprotein convertase subtilisin-like/kexin type 9 (PCSK9) is a key regulator of plasma LDL-cholesterol (LDL-C) and a clinically validated target for the treatment of hypercholesterolemia and coronary artery disease. In this paper, we describe a series of novel cyclic peptides derived from an mRNA display screen which inhibit the protein–protein interaction between PCSK9 and LDLR. Using a structure-based drug design approach, we were able to modify our original screening lead 2 to optimize the potency and metabolic stability and minimize the molecular weight to provide novel bicyclic next-generation PCSK9 inhibitor peptides such as 78. These next-generation peptides serve as a critical foundation for continued exploration of potential oral, once-a-day PCSK9 therapeutics for the treatment of cardiovascular disease." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01084"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01084">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01084&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01084&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01084&amp;href=/doi/10.1021/acs.jmedchem.0c01084" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 13796-13824</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01234" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01329" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Series of Novel and Highly Potent Cyclic Peptide PCSK9 Inhibitors Derived from an mRNA Display Screen and Optimized via Structure-Based Design</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Candice Alleyne</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Candice Alleyne</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Pharmaceutical Sciences, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Candice++Alleyne">Candice Alleyne</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rupesh P. Amin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rupesh P. Amin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Safety Assessment, Merck & Comapny, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rupesh+P.++Amin">Rupesh P. Amin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bhavana Bhatt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bhavana Bhatt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Safety Assessment, Merck & Comapny, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bhavana++Bhatt">Bhavana Bhatt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elisabetta Bianchi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabetta Bianchi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">IRBM S.p.A., Via Pontina km 30600, Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabetta++Bianchi">Elisabetta Bianchi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">J. Craig Blain</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">J. Craig Blain</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J.+Craig++Blain">J. Craig Blain</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicolas Boyer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicolas Boyer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Boyer">Nicolas Boyer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Danila Branca</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Danila Branca</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">IRBM S.p.A., Via Pontina km 30600, Pomezia, Rome 00071, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Danila++Branca">Danila Branca</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark W. Embrey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark W. Embrey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, Merck & Company, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+W.++Embrey">Mark W. Embrey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sookhee N. Ha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sookhee N. Ha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Modeling and Informatics, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sookhee+N.++Ha">Sookhee N. Ha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kelli Jette</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kelli Jette</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kelli++Jette">Kelli Jette</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Douglas G. Johns</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Douglas G. Johns</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Douglas+G.++Johns">Douglas G. Johns</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Angela D. Kerekes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Angela D. Kerekes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Angela+D.++Kerekes">Angela D. Kerekes</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kenneth A. Koeplinger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenneth A. Koeplinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmacokinetics Pharmacodynamics and Drug Metabolism, Merck & Company, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenneth+A.++Koeplinger">Kenneth A. Koeplinger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Derek LaPlaca</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Derek LaPlaca</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Derek++LaPlaca">Derek LaPlaca</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8513-2445" title="Orcid link">http://orcid.org/0000-0002-8513-2445</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nianyu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nianyu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Safety Assessment, Merck & Comapny, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nianyu++Li">Nianyu Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Beth Murphy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Beth Murphy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Beth++Murphy">Beth Murphy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Orth</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Orth</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Sciences, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Orth">Peter Orth</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alonso Ricardo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alonso Ricardo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alonso++Ricardo">Alonso Ricardo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott Salowe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott Salowe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott++Salowe">Scott Salowe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kathleen Seyb</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kathleen Seyb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kathleen++Seyb">Kathleen Seyb</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aurash Shahripour</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aurash Shahripour</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aurash++Shahripour">Aurash Shahripour</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph R. Stringer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph R. Stringer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+R.++Stringer">Joseph R. Stringer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yili Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yili Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCB Ra Pharma, 87 Cambridge Park Drive, Cambridge, Massachusetts 02140, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yili++Sun">Yili Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rodger Tracy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rodger Tracy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharmacokinetics Pharmacodynamics and Drug Metabolism, Merck & Company, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rodger++Tracy">Rodger Tracy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chengwei Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chengwei Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, Merck & Company, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chengwei++Wu">Chengwei Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yusheng Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yusheng Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yusheng++Xiong">Yusheng Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hyewon Youm</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hyewon Youm</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hyewon++Youm">Hyewon Youm</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hratch J. Zokian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hratch J. Zokian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hratch+J.++Zokian">Hratch J. Zokian</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Thomas J. Tucker</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas J. Tucker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Medicinal Chemistry, Merck & Company, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c3b7acae9cb7b6a0a8a6b183aea6b1a0a8eda0acae"><span class="__cf_email__" data-cfemail="f0849f9daf8485939b9582b09d9582939bde939f9d">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+J.++Tucker">Thomas J. Tucker</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5616-0802" title="Orcid link">http://orcid.org/0000-0001-5616-0802</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01084&amp;href=/doi/10.1021%2Facs.jmedchem.0c01084" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 13796–13824</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 10, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 August 2020</li><li><span class="item_label"><b>Published</b> online</span>10 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 November 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01084" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01084</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D13796%26pageCount%3D29%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCandice%2BAlleyne%252C%2BRupesh%2BP.%2BAmin%252C%2BBhavana%2BBhatt%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D22%26contentID%3Dacs.jmedchem.0c01084%26title%3DSeries%2Bof%2BNovel%2Band%2BHighly%2BPotent%2BCyclic%2BPeptide%2BPCSK9%2BInhibitors%2BDerived%2Bfrom%2Ban%2BmRNA%2BDisplay%2BScreen%2Band%2BOptimized%2Bvia%2BStructure-Based%2BDesign%26numPages%3D29%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D13824%26publicationDate%3DNovember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01084"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2483</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01084" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Series of Novel and Highly Potent Cyclic Peptide PCSK9 Inhibitors Derived from an mRNA Display Screen and Optimized via Structure-Based Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Candice&quot;,&quot;last_name&quot;:&quot;Alleyne&quot;},{&quot;first_name&quot;:&quot;Rupesh&quot;,&quot;last_name&quot;:&quot;P. Amin&quot;},{&quot;first_name&quot;:&quot;Bhavana&quot;,&quot;last_name&quot;:&quot;Bhatt&quot;},{&quot;first_name&quot;:&quot;Elisabetta&quot;,&quot;last_name&quot;:&quot;Bianchi&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Craig Blain&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Boyer&quot;},{&quot;first_name&quot;:&quot;Danila&quot;,&quot;last_name&quot;:&quot;Branca&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;W. Embrey&quot;},{&quot;first_name&quot;:&quot;Sookhee&quot;,&quot;last_name&quot;:&quot;N. Ha&quot;},{&quot;first_name&quot;:&quot;Kelli&quot;,&quot;last_name&quot;:&quot;Jette&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;G. Johns&quot;},{&quot;first_name&quot;:&quot;Angela&quot;,&quot;last_name&quot;:&quot;D. Kerekes&quot;},{&quot;first_name&quot;:&quot;Kenneth&quot;,&quot;last_name&quot;:&quot;A. Koeplinger&quot;},{&quot;first_name&quot;:&quot;Derek&quot;,&quot;last_name&quot;:&quot;LaPlaca&quot;},{&quot;first_name&quot;:&quot;Nianyu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Beth&quot;,&quot;last_name&quot;:&quot;Murphy&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Orth&quot;},{&quot;first_name&quot;:&quot;Alonso&quot;,&quot;last_name&quot;:&quot;Ricardo&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;Salowe&quot;},{&quot;first_name&quot;:&quot;Kathleen&quot;,&quot;last_name&quot;:&quot;Seyb&quot;},{&quot;first_name&quot;:&quot;Aurash&quot;,&quot;last_name&quot;:&quot;Shahripour&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;R. Stringer&quot;},{&quot;first_name&quot;:&quot;Yili&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Rodger&quot;,&quot;last_name&quot;:&quot;Tracy&quot;},{&quot;first_name&quot;:&quot;Chengwei&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Yusheng&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Hyewon&quot;,&quot;last_name&quot;:&quot;Youm&quot;},{&quot;first_name&quot;:&quot;Hratch&quot;,&quot;last_name&quot;:&quot;J. Zokian&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;J. Tucker&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;13796-13824&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01084&quot;},&quot;abstract&quot;:&quot;Proprotein convertase subtilisin-like/kexin type 9 (PCSK9) is a key regulator of plasma LDL-cholesterol (LDL-C) and a clinically validated target for the treatment of hypercholesterolemia and coronary artery disease. In this paper, we describe a series of novel cyclic peptides derived from an mRNA display screen which inhibit the protein–protein interaction between PCSK9 and LDLR. Using a structure-based drug design approach, we were able to modify our original screening lead 2 to optimize the potency and metabolic stability and minimize the molecular weight to provide novel bicyclic next-generation PCSK9 inhibitor peptides such as 78. These next-generation peptides serve as a critical foundation for continued exploration of potential oral, once-a-day PCSK9 therapeutics for the treatment of cardiovascular disease.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01084&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01084" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01084&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01084" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01084&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01084" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01084&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01084&amp;href=/doi/10.1021/acs.jmedchem.0c01084" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01084" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01084" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01084%26sid%3Dliteratum%253Aachs%26pmid%3D33170686%26genre%3Darticle%26aulast%3DAlleyne%26date%3D2020%26atitle%3DSeries%2Bof%2BNovel%2Band%2BHighly%2BPotent%2BCyclic%2BPeptide%2BPCSK9%2BInhibitors%2BDerived%2Bfrom%2Ban%2BmRNA%2BDisplay%2BScreen%2Band%2BOptimized%2Bvia%2BStructure-Based%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D22%26spage%3D13796%26epage%3D13824%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (6)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291675" title="Stability">Stability</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/jmcmar.2020.63.issue-22/20201125/jmcmar.2020.63.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0037.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Proprotein convertase subtilisin-like/kexin type 9 (PCSK9) is a key regulator of plasma LDL-cholesterol (LDL-C) and a clinically validated target for the treatment of hypercholesterolemia and coronary artery disease. In this paper, we describe a series of novel cyclic peptides derived from an mRNA display screen which inhibit the protein–protein interaction between PCSK9 and LDLR. Using a structure-based drug design approach, we were able to modify our original screening lead <b>2</b> to optimize the potency and metabolic stability and minimize the molecular weight to provide novel bicyclic next-generation PCSK9 inhibitor peptides such as <b>78</b>. These next-generation peptides serve as a critical foundation for continued exploration of potential oral, once-a-day PCSK9 therapeutics for the treatment of cardiovascular disease.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72642" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72642" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Proprotein convertase subtilisin-like/kexin type 9 (PCSK9) is a key regulator of plasma LDL-cholesterol (LDL-C) and an important target for treating coronary artery disease.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In genetic studies, it was demonstrated that humans heterozygous for specific gain-of-function truncation or missense PCSK9 mutations (∼2–3% of the population) have a 15–40% reduction in plasma LDL-C and a 50–90% reduction in the risk of CHD.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It has also been demonstrated via similar genetic data that humans who express specific loss-of-function mutations in PCSK9 have severe hypercholesterolemia and a high risk of coronary artery disease.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">The mechanism for the effect of PCSK9 on LDL-cholesterol (LDL-C) is well defined. PCSK9 in circulation decreases LDL-C levels by binding to the LDL-receptor (LDLR) on the cell surface, and the resulting endocytosis and lysosomal degradation serves as a key regulatory mechanism for LDL-C levels.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Direct blockade of the LDL-receptor–PCSK9 interaction therefore serves to increase the steady-state levels of LDLR, making more LDLR available to assist in the clearance of circulating LDL-C.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> There are currently two commercial therapeutic antibodies (alirocumab<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and evolocumab<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a>) that have been approved for the treatment of familial hypercholesterolemia and for lowering LDL-C levels in patients who are not at goal via statin therapy. Phase III outcome studies with both have shown beneficial effects on cardiovascular events but demonstrated effects on lowering overall mortality await further data.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> These antibodies are administered subcutaneously either biweekly or monthly, and their initial uptake in the marketplace has not been as large as anticipated largely because of pricing pressures.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">It is estimated that of the approximately 70 million patients treated with statins, as many as 50% do not reach their target LDL-C levels.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> Of those that do achieve treatment goals, another several million suffer major cardiovascular events annually.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> The need exists for additional therapeutic options for those that do not achieve their LDL-C goals, patients with familial hypercholesterolemia, patients that do achieve goal but have cardiovascular events, patients with other comorbidities such as type 2-diabetes mellitus, and also in statin-intolerant patient populations. An oral, once daily PCSK9 inhibitor therapy could potentially offer a more convenient alternative for administration and better patient adherence. The orally delivered therapy of this type would easily align with other oral therapies that are standard of care in the at-risk population and would provide the benefit of easy dosing and level control compared to parenteral administration. Another benefit would be the ease of therapy withdrawal by simply discontinuing the dosing regimen.</div><div class="NLM_p">Based on these factors, we chose to develop novel inhibitors of PCSK9 with the goal of eventually developing oral therapeutics for the treatment of hypercholesterolemia and coronary artery disease. Strategically, our intent was to develop PCSK9 inhibitors that generate equivalent LDL-C lowering to the established mAb standards evolocumab<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and alirocumab.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The target of such inhibitors would be to orthosterically block the protein–protein interaction (PPI) between PCSK9 and LDLR by binding to the large, flat LDLR binding surface on the PCSK9 protein, thus preventing the direct interaction between the proteins.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The proteins and their interaction are well characterized in the literature,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and this provided a well-validated target for therapeutic intervention.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> There are a large number of PCSK9 inhibitors reported in the literature, with several recent descriptions of novel small-molecule<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8−11)</a> and peptide<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> inhibitors, as well as siRNA-based therapeutics such as inclisiran.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The PCSK9 inhibitor area has recently been the subject of several comprehensive reviews.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> However, none of the known inhibitors have the overall profile of potency and favorable druglike properties necessary to be viable as oral therapeutic agents.</div><div class="NLM_p">We chose to initially pursue macrocyclic peptides as our modality of choice because of their potentially superior stability characteristics<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> as well as their proven ability to bind to the large, flat surfaces that are characteristic of PPIs such as the interaction between PCSK9 and LDLR. In collaboration with our partner Ra Pharmaceuticals,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> we used the mRNA display technology to find potent, cyclic peptide inhibitors of PCSK9 that we could then engineer using internal medicinal chemistry expertise to be deliverable via oral dosing. The use of mRNA display as a proven and powerful platform for early hit finding is well documented in the literature.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> Herein, we detail our initial screening hit peptides and the optimization of these macrocyclic peptides to provide highly potent, novel, and specific inhibitors of PCSK9 that serve as key starting points for the development of oral PCSK9 inhibitor therapeutics.</div><div class="NLM_p">The initial mRNA display libraries of linear peptides of various lengths were prepared containing two reactive thiol moieties, allowing for direct cyclization of the libraries using the well-known 1,3-di(bromomethyl)benzene (DBX) as the key cross-linking element (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The DBX group is a commonly used and highly efficient cross-linking element for cyclization of peptide libraries derived from mRNA display.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> From an initial series of mRNA screens targeting cyclic peptides of various sizes, we obtained two novel hit series (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) was identified as a potent 13 amino acid hit with good potency against PCSK9 using the LDLR-FRET assay. A second, less potent but structurally unique 12 amino acid series was also found, with compound <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) emerging as the key lead molecule from this series. The initial comparisons of the two series showed less overall homology, and analysis strongly suggested differing binding modes for each of the molecules. An early critical goal for the team was to obtain crystal structures of each of these representative structures bound to the LDLR binding site on PCSK9. We quickly obtained a crystal structure of the chemical series exemplified by <b>1</b> but were not able to initially obtain a structure for <b>2</b> or any early analogues, likely because of the large differences in potency between the two series, and had to rely primarily on empirical data and molecular modeling based on that data. Details of the efforts around <b>1</b> will be described in another paper currently in preparation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The current manuscript will focus on the optimization of the lead molecule <b>2</b>, initially using standard empirical medicinal chemistry strategies, which were later coupled with advanced structure-based drug design approaches.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0003.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. mRNA display approach using Ra Pharma Platform.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Two lead series derived from mRNA display screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76123" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76123" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Screening and Early Structure–Activity Relationship</h3><div class="NLM_p">Test peptides were initially evaluated for PCSK9 inhibition activity using an LDLR-FRET assay (described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>), and the results are initially reported as IC<sub>50</sub> values. In general, the peptides were synthesized using standard solid-phase peptide synthesis (SPPS) approaches to prepare the crude linear peptides after cleavage and deprotection (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The crude peptide could then be easily cyclized by addition of 1,3-di(bromomethyl)benzene (DBX), followed by purification to give the desired cyclic peptide products (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Specific synthetic details are provided in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Starting from compound <b>2</b>, we envisioned an approach by which we would attempt to reduce the size and molecular weight of the target peptides while also simultaneously addressing the potency and peptide stability. Starting with the N-terminal lasso “tail” region, we were quickly able to remove the amino acids in the tail to provide <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), which demonstrated a 20-fold improvement in potency in our LDLR-FRET assay. Further truncation of the N-terminus to completely remove it provided <b>4</b>, which lost only about 2-fold potency versus <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Compound <b>4</b> became a key early analogue for our program, and we decided to characterize overall properties in more depth. We wanted to address the stability of our cyclic peptide platform early on and chose to evaluate our peptides against a series of important gastrointestinal proteases (trypsin, chymotrypsin, pepsin, and elastase) for ultimate oral dosing and in whole blood for intravenous (IV) dosing. Early IV PK studies were primarily performed in C57BL/6 mice to keep the amount of compound needed from synthesis at a minimum. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> summarizes the IV PK for a number of molecules discussed throughout this paper, and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> details the observed protease and whole blood stability. Compound <b>4</b> was shown to have a short half-life and moderately high clearance in the mouse, and the whole blood stability indicated a moderate half-life in mouse whole blood of about 1 h. The protease panel clearly showed poor stability in the presence of chymotrypsin, elastase, and trypsin. For <b>4</b>, and in general across all the peptides tested, full stability to pepsin was observed. Metabolite identification (Met ID) studies after incubation with each of the key proteases also showed major metabolic vulnerabilities at four amide bonds in the N-terminal region of the peptide (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), and we were able to correlate the most vulnerable individual amide bonds with the corresponding responsible protease. Met ID studies with the peptide after incubation in mouse whole blood showed a major metabolic site at the amide bond between the proline at position 8 and the cysteine at position 9 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Early progression of SAR from original mRNA display lead <b>2</b>. Key structural changes are highlighted in red for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Primary metabolic sites on a representative molecule based on met ID studies with key proteases and mouse whole blood.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0020.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Approach to the Synthesis of Cyclic Peptides</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Mouse IV PK Parameters for Various Compounds<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a><sup>,</sup><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound number</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–inf</sub> (nM h)</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–<i>x</i></sub> (nM h)</th><th class="colsep0 rowsep0" align="center" char="±">clearance (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="±"><i>V</i><sub>dss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="±"><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">protein binding<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> (% bound, mouse plasma)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">241.93 ± 66.81</td><td class="colsep0 rowsep0" align="char" char="±">240.9 ± 42.20</td><td class="colsep0 rowsep0" align="char" char="±">51.73 ± 15.23</td><td class="colsep0 rowsep0" align="char" char="±">0.70 ± 0.48</td><td class="colsep0 rowsep0" align="char" char="±">1.04 ± 0.11</td><td class="colsep0 rowsep0" align="char" char=".">95.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">181.43 ± 45.51</td><td class="colsep0 rowsep0" align="char" char="±">184.73 ± 45.34</td><td class="colsep0 rowsep0" align="char" char="±">65.40 ± 14.10</td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.59</td><td class="colsep0 rowsep0" align="char" char="±">0.33 ± 0.33</td><td class="colsep0 rowsep0" align="char" char=".">94.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">491.78 ± 44.53</td><td class="colsep0 rowsep0" align="char" char="±">490.65 ± 43.34</td><td class="colsep0 rowsep0" align="char" char="±">24.25 ± 1.55</td><td class="colsep0 rowsep0" align="char" char="±">0.29 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">0.78 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">99.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">2505.11 ± 164.56</td><td class="colsep0 rowsep0" align="char" char="±">2494.04 ± 157.65</td><td class="colsep0 rowsep0" align="char" char="±">4.62 ± 0.29</td><td class="colsep0 rowsep0" align="char" char="±">0.10 ± 0</td><td class="colsep0 rowsep0" align="char" char="±">0.25 ± 0.03</td><td class="colsep0 rowsep0" align="char" char=".">99.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">171.52 ± 3.03</td><td class="colsep0 rowsep0" align="char" char="±">170.56 ± 3.35</td><td class="colsep0 rowsep0" align="char" char="±">17.25 ± 0.25</td><td class="colsep0 rowsep0" align="char" char="±">0.37 ± 0</td><td class="colsep0 rowsep0" align="char" char="±">1.60 ± 0.29</td><td class="colsep0 rowsep0" align="char" char=".">98.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78</b></td><td class="colsep0 rowsep0" align="char" char="±">840.94 ± 174.30</td><td class="colsep0 rowsep0" align="char" char="±">839.90 ± 174.25</td><td class="colsep0 rowsep0" align="char" char="±">16.00 ± 3.30</td><td class="colsep0 rowsep0" align="char" char="±">0.23 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">0.66 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">98.50</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Compounds were tested in C57BL/6 mice at 1 mpk using a 30% propylene glycol/70% PBS vehicle at a concentration of 0.5 mg/mL or at 1 mpk using a 20% DMSO/60% propylene glycol/20% water vehicle at a concentration of 1 mg/mL.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">For compounds <b>4</b>, <b>30</b>, <b>40</b>, and <b>44</b>, the values are the average of three mice from a single experiment with the standard deviations noted.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">For compounds <b>53</b> and <b>78</b>, the values are the average of two mice from a single experiment, and the indicated errors are the difference between the mean and the individual values.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">All protein binding values are derived from a single experiment.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Raw concentration vs time profiles for all compounds are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_001.pdf" class="ext-link">Supporting Information</a>, along with <i>T</i><sub>max</sub> and <i>C</i><sub>max</sub> data.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Stability of Key Compounds against Gut Proteases and Whole Blood<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound #</th><th class="colsep0 rowsep0" align="center" char=".">chymotrypsin, 1 h (% remaining)</th><th class="colsep0 rowsep0" align="center" char=".">elastase, 1 h (% remaining)</th><th class="colsep0 rowsep0" align="center" char=".">pepsin, 1 h (% remaining)</th><th class="colsep0 rowsep0" align="center" char=".">trypsin, 1 h (% remaining)</th><th class="colsep0 rowsep0" align="center">mouse whole blood <i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">Cyno whole blood <i>T</i><sub>1/2</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">60.7</td><td class="colsep0 rowsep0" align="char" char=".">95</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="left">63.2</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">15.5</td><td class="colsep0 rowsep0" align="char" char=".">54.9</td><td class="colsep0 rowsep0" align="char" char=".">92.5</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="left">89.6</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="char" char=".">10.9</td><td class="colsep0 rowsep0" align="char" char=".">52.7</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="left">311.3</td><td class="colsep0 rowsep0" align="left">>360</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="char" char=".">7.4</td><td class="colsep0 rowsep0" align="char" char=".">46.1</td><td class="colsep0 rowsep0" align="char" char=".">99.1</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="left">333.5</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="char" char=".">42.7</td><td class="colsep0 rowsep0" align="char" char=".">99.1</td><td class="colsep0 rowsep0" align="char" char=".">93.7</td><td class="colsep0 rowsep0" align="char" char=".">79.5</td><td class="colsep0 rowsep0" align="left">186.1</td><td class="colsep0 rowsep0" align="left">>360</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="char" char=".">20.1</td><td class="colsep0 rowsep0" align="char" char=".">96.5</td><td class="colsep0 rowsep0" align="char" char=".">96.6</td><td class="colsep0 rowsep0" align="char" char=".">91.0</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">>360</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="char" char=".">76.4</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">94.3</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="char" char=".">91</td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">95</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="char" char=".">91</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">99.5</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">186</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">98.1</td><td class="colsep0 rowsep0" align="char" char=".">95.9</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="char" char=".">24.2</td><td class="colsep0 rowsep0" align="char" char=".">68.2</td><td class="colsep0 rowsep0" align="char" char=".">87.5</td><td class="colsep0 rowsep0" align="char" char=".">98.4</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="char" char=".">39.5</td><td class="colsep0 rowsep0" align="char" char=".">96.3</td><td class="colsep0 rowsep0" align="char" char=".">98.6</td><td class="colsep0 rowsep0" align="char" char=".">96.0</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="char" char=".">91.8</td><td class="colsep0 rowsep0" align="char" char=".">96.6</td><td class="colsep0 rowsep0" align="char" char=".">98.9</td><td class="colsep0 rowsep0" align="char" char=".">99.2</td><td class="colsep0 rowsep0" align="left">ND</td><td class="colsep0 rowsep0" align="left">>360</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78</b></td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">98.2</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">113</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79</b></td><td class="colsep0 rowsep0" align="char" char=".">95.7</td><td class="colsep0 rowsep0" align="char" char=".">98.8</td><td class="colsep0 rowsep0" align="char" char=".">94.4</td><td class="colsep0 rowsep0" align="char" char=".">97.6</td><td class="colsep0 rowsep0" align="left">>360</td><td class="colsep0 rowsep0" align="left">>360</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Compounds were incubated in duplicate in a single experiment for 1 h at 37 °C; ND indicates that the value was not determined for the specific molecule.</p></div></div></div><div class="NLM_p">Alanine, <span class="smallcaps smallerCapital">d</span>-alanine, and <i>N</i>-methyl amino acid scans were also performed using <b>4</b> as the scaffold. The results of these scans are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. The alanine scan (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <b>5–12</b>) showed some tolerance for side chain modifications at positions 1 (Lys; <b>5</b>), 3 (5F-Trp at position 3; <b>7</b>), 5(Asp; <b>9</b>), and 6 (His; <b>10</b>). Interestingly, 5-fluoro-tryptophan (5F-Trp) at position 3 appeared to be much more amenable to change than the one at position 4. The <span class="smallcaps smallerCapital">d</span>-amino acid scan (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <b>13–20</b>) showed less tolerance for <span class="smallcaps smallerCapital">d</span>-amino acids, although position 2 (Gly) was not surveyed, and this would later become of importance. Similarly, the <i>N</i>-methyl amino acid scan (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, <b>21–28</b>) showed less tolerance for backbone N-methylations, with the exceptions being positions 1 (Lys; <b>21</b>) and 7 (Tyr; <b>27</b>) where N-methylation appeared to be reasonably well tolerated.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Alanine, <span class="smallcaps smallerCapital">d</span>-Amino Acid, and <i>N</i>-Me Amino Acid Scans on <b>4</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0027.gif" alt="" id="GRAPHIC-d7e1502-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0028.gif" alt="" id="gr26" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">For all determinations, data is the average of <i>n</i> = 2, with the variability of the two determinations reported for molecules <1 μM. Data for the parent molecule <b>4</b> is highlighted in green. Purity of all peptides >95%; all peptides exhibited correct MS; MPA = 3-mercaptopropionic acid, all peptides cyclized from MPA to CYS using 1,3-di(bromomethyl)benzene (DBX). For peptides <b>14</b> and <b>15</b>, <span class="smallcaps smallerCapital">d</span>-Trp was substituted for the unavailable <span class="smallcaps smallerCapital">d</span>-5F-Trp; for peptides <b>23</b> and <b>24</b>, racemic <i>N</i>-Me Trp was used in the scans, and two diastereomers were recovered and tested for each; all were inactive.</p></div></div><div></div></div><div class="NLM_p">Removal of the C-terminal amide of <b>4</b> provided <b>29</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), which demonstrated a small 2-fold decrease in inhibitory potency but removed two H-bond donors. As indicated above, a metabolic liability was identified for the amide bond between proline and cysteine that we theorized was likely due to a prolyl-endopeptidase or similar enzyme, and we wanted to address this issue immediately. For this reason, the proline was replaced with an α-methyl proline to give <b>30</b>. Peptide <b>30</b> turned out to be a key molecule for the evolution of our entire program.</div><div class="NLM_p">We observed an 8-fold improvement in inhibitory potency versus parent peptide <b>4</b> and much improved whole blood stability (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), suggesting alleviation of the metabolic issue at this center. Despite having much improved stability in mouse whole blood, compound <b>30</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) was shown in the mouse to have a moderate clearance and extremely short half-life. The molecule demonstrated poor stability against our protease panel (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">In an attempt to explore some general structure–activity relationship (SAR) around the peptides, we initiated a broader survey of the SAR at several of the key positions along the peptide, using the peptide backbones from peptides <b>3</b> and <b>4</b>. <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Tables <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a> summarize the key SAR around positions 3 and 4 (5F-Trp moieties) and 8 (proline), respectively. Of great interest were the unusual consecutive 5F-Trp moieties at positions 3 and 4. From the data in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, it became quite clear that the 5F-Trp moiety at position 4 was playing a crucial role in the binding of the peptide. Replacement of 5F-Trp at position 3 with a simple tryptophan (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, <b>31</b>) resulted in a small loss of potency. However, replacement of 5F-Trp at position 4 with a tryptophan (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, <b>32</b>) resulted in a more pronounced 23-fold loss of potency. In addition, 5F-Trp at position 3 could be replaced with an alpha-naphthylalanine (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, 1-Nal; <b>33</b>) with potency maintained. Replacement of 5F-Trp at position 3 with either a larger 4-phenylphenylalanine (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>; <b>34</b>) or a homophenylalanine (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>; <b>35</b>) demonstrated that while there was some tolerability at this position, larger or longer side chains were not well tolerated. <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> summarizes a few changes at the Pro moiety at position 8.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR for Peptide Amino Acid Positions 3 and 4<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0029.gif" alt="" id="GRAPHIC-d7e1646-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound #</th><th class="colsep0 rowsep0" align="center">N-terminus</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">LDLR-FRET <i>K</i><sub><i>i</i></sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">AcNHCys</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">47.2 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">MPA</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">111 ± 1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">AcNHCys</td><td class="colsep0 rowsep0" align="left">Trp</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">103 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">AcNHCys</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">Trp</td><td class="colsep0 rowsep0" align="left">1070 ± 220<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">MPA</td><td class="colsep0 rowsep0" align="left">1-Nal</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">63 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">MPA</td><td class="colsep0 rowsep0" align="left">4Ph-Phe</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">29,345 (<i>n</i> = 2)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">MPA</td><td class="colsep0 rowsep0" align="left">homePhe</td><td class="colsep0 rowsep0" align="left">5F-Trp</td><td class="colsep0 rowsep0" align="left">2211 (<i>n</i> = 2)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">FRET <i>K</i><sub><i>i</i></sub> values are the average of at least three determinations with the standard deviation indicated unless otherwise noted; MPA = 3-mercaptopropionic acid; alpha Nal = alpha-naphthylalanine; and 4-PhPhe = 4-phenyl-phenylalanine.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">For <i>n</i> = 2, the error is reported for molecules <1 μM; error is defined as the difference between the mean and the individual determinations.</p></div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR for Peptide Amino Acid Position 8 (Proline)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0030.gif" alt="" id="GRAPHIC-d7e1817-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound #</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">LDLR-FRET <i>K</i><sub><i>i</i></sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">Pro</td><td class="colsep0 rowsep0" align="left">115 ± 1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">alpha-Me Pro</td><td class="colsep0 rowsep0" align="left">14.0 ± 1.7 (<i>n</i> = >200)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">Sar</td><td class="colsep0 rowsep0" align="left">10,740 ± 20.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">Aze</td><td class="colsep0 rowsep0" align="left">67.5 ± 33.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">Pip</td><td class="colsep0 rowsep0" align="left">969 ± 110</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">LDLR-FRET <i>K</i><sub><i>i</i></sub> values are the average of three determinations with the standard deviation noted unless otherwise indicated, except for <b>30</b>, which was used as an assay standard; Sar = sarcosine; Aze = 2(<i>S</i>)-azetidinecarboxylic acid; and Pip = 2(<i>S</i>)-piperidinecarboxylic acid.</p></div></div></div><div class="NLM_p last">As stated earlier, the addition of an alpha-Me proline to the peptide provided an 8-fold potency enhancement versus <b>4</b> and was a key SAR element at this position. Opening the proline ring by replacement with a sarcosine provided <b>36</b>, which demonstrated a large potency loss. Replacement of the proline with a four-membered azetidine amino acid <b>37</b> provided only a slight loss of inhibitory potency versus <b>30</b>, while enlargement of the ring to a six-membered ring in <b>38</b> showed a substantial loss of inhibitory potency versus both <b>4</b> and <b>30</b>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Crystal Structure of Compound <b>30</b></h3><div class="NLM_p">The most important aspect of the synthesis of compound <b>30</b> was that we were able to use this compound to generate the first cocrystal structure in this series with PCSK9. We were unable to generate suitable crystals with earlier inhibitors in this series, most likely because of the limited potency observed with these molecules. The crystal structure of <b>30</b> was incredibly informative in multiple dimensions and is shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. The peptide assumes an overall “doughnut-like”, circular binding conformation, with the side chain of 5F-Trp at position 4 in the center of the doughnut, sticking into a small pocket on the relatively flat surface of the binding site. The fluorine sits nicely between the two H-bonded β-sheet regions of the protein, with the shallow pocket defined by the backbone NH and carbonyls of Gly 370, Val 380, Ser 381, and Ile 369 and the side chains of Ile 369, Ser 381, and Phe 379 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). The structure suggests possible favorable direct interactions between the aryl fluorine atom and the NH of Gly 370, the carbonyl oxygens of Val 380 and Ile 369, as well as various interactions with the side chain carbons of the aforementioned amino acid side chains. Such noncovalent interactions between fluorine atoms and various amino acids are well documented in the literature.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The specific nature of the fit of the fluorine into this small surface pocket along with the previously described analogue work at this position confirms the critical nature of the interaction of the aryl fluoride with the small pocket on the PCSK9 surface. The selection of the 5F-Trp amino acid as one of the amino acids for the original mRNA display campaign was clearly critical to the entire screening and hit finding process with these peptides.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0007.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray crystal structure of 30 (salmon) bound to the LDLR-binding site of PCSK9 at 1.55 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIB">6XIB</a>). The molecule binds in an overall circular conformation, with 5F-Trp at position 2 outside of the doughnut and 5F-Trp at position 3 inside the macrocycle, with the fluorine atom in a small pocket on the surface. The side chains of position 1 (Lys), 5 (Asp), and 6 (His) are all solvent-exposed and do not directly interact with the protein. The side chain of position 7 (Tyr) lies in a small groove on the surface, and the aryl ring of the cross-linker lies directly on top of Ile 369.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0008.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Close-up of the shallow binding pocket occupied by 5F-Trp in position 4 of <b>30</b>. The pocket is defined by two hydrogen-bonded beta sheets, with the aryl ring and 5-fluroro substituent of the ligand fitting downward into the space between.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The lysine side chain at the N-terminal position 1 is fully solvent-exposed and does not appear to be making any specific interactions with the protein (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The first 5F-Trp in the sequence at position 3 lies outside of the macrocyclic ring, opposite the inner 5F-Trp at position 4, partially solvent-exposed and contacting a lipophilic region consisting of the disulfide bond between Cys 375 and 378 and the side chains of Val 380 and Asp 374. The Asp–His region provides a turn-like geometry, with the Asp and His side chains of the inhibitor forming an ion pair that helps to lock this conformation. A key aspect of this entire N-terminal region is the presence of five key hydrogen bonding interactions with a β-sheet on the protein that lies directly under the N-terminal region of the inhibitor. <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> schematically details each of these H-bonding interactions, which can be characterized as follows:<table class="listgroup" border="0" width="95%" id="list1" list-type="label"><tr class="li1"><td valign="top">1.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The NH of the glycine moiety in position 2 of the inhibitor makes an interaction with the side chain OH of Ser 381.</p></td></tr><tr class="li1"><td valign="top">2.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The carbonyl of the same glycine on the inhibitor makes an interaction with the backbone NH of Ser381.</p></td></tr><tr class="li1"><td valign="top">3.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The NH of 5F-Trp at position 3 of the inhibitor makes an interaction with the backbone carbonyl of Phe379.</p></td></tr><tr class="li1"><td valign="top">4.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The carbonyl of 5F-Trp at position 4 of the inhibitor makes an interaction with the backbone NH of Phe379.</p></td></tr><tr class="li1"><td valign="top">5.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The backbone carbonyl of the aspartic acid residue at position 5 of the inhibitor makes an interaction with the side chain OH of Thr377.</p></td></tr></table></div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0009.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Schematic representation of the five key hydrogen bonding interactions between inhibitor <b>30</b> (salmon) and the PCSK9 active site β-sheet (white). The key interactions are described in detail in the text.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">These key H-bonding interactions are critical for the good potency observed with these peptides. The tyrosine moiety of the inhibitor lies in one of the few true pockets in the binding surface (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). This shallow pocket is defined by the side chains of Phe379, Glu195, Asp238, and Ala239 and has a small, solvent-exposed opening near where the para-position of the tyrosine of the inhibitor resides. Interestingly, the proline ring sits quite close to the 5F-Trp side chain of the inhibitor that lies inside the macrocycle, suggesting a conformationally reinforcing intramolecular interaction between the two groups, likely reinforced further by the addition of the potency enhancing alpha-methyl group to the proline. These intramolecular interactions appear to be crucial for locking the molecule into a desirable conformation for binding. The C-terminal amide is largely solvent-exposed and does not interact with the protein, suggesting that it is likely not critical for binding, as suggested by data from compound <b>29</b>. Finally, to our surprise, we observed that the DBX cross-linking group was contributing to potency and directly interacting with the PCSK9 surface. The aryl ring lies directly above the side chain of Ile 369, almost sitting directly on top of the side chain in a manner analogous to a tabletop.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Expanded SAR around Compound <b>30</b></h3><div class="NLM_p">As we moved forward with analogues of improved potency, we began to observe a bottoming of the LDLR-FRET values and differentiation between compounds below the 10 nM range became difficult. We therefore established a second, more accurate Alexa-FRET assay that was able to better distinguish between compounds with more potent inhibitory potency (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). The values for both assays are reported for a number of compounds to allow for comparison, but we eventually moved completely to the Alexa-FRET assay for more optimized molecules.</div><div class="NLM_p">In addition to the key binding insights described above, the crystal structure of <b>30</b> also provided several critical design insights that would prove to be the key to driving subsequent efforts (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). We removed the C-terminus of <b>30</b> to provide <b>39</b>, which demonstrated minimal loss of potency and a similar behavior in our stability assays (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The β-sheet binding region also contains the four protease-labile amide bonds previously identified in these molecules (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Although the alanine scan with <b>4</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) clearly indicated that an alanine was not tolerated at position 2 (Gly) because of a likely unfavorable interaction with 5F-Trp that lies inside the macrocycle, the crystal structure of <b>30</b> strongly suggested that a <span class="smallcaps smallerCapital">d</span>-alanine would be well tolerated at this position. The region that would be occupied by a substituent of a <span class="smallcaps smallerCapital">d</span>-amino acid points away from the protein and into an open, solvent-exposed region in parallel with the side chain of Lys at position 1. We theorized that using a <span class="smallcaps smallerCapital">d</span>-amino acid at this position might also help to block some of the metabolism observed in this region of the molecule. We therefore prepared the analogue <b>40</b> containing a <span class="smallcaps smallerCapital">d</span>-alanine at position 2 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). This molecule was shown to be essentially equipotent with <b>30</b> and <b>39</b>. <b>40</b> was shown to have improved stability versus both elastase and trypsin, the enzymes responsible for proteolysis at the two amide bonds flanking the <span class="smallcaps smallerCapital">d</span>-alanine at position 2 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Therefore, our strategy to insert a <span class="smallcaps smallerCapital">d</span>-amino acid at this position based on the crystallographic data from <b>30</b> appeared to be successful, allowing for the stabilization of two of the four vulnerable amide bonds. Interestingly, the mouse whole blood stability of <b>40</b> decreased slightly but was still quite good. However, <b>40</b> still showed a short half-life after IV dosing to mice (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Nonetheless, the noted improvement in protease stability became a key feature for future compound design. <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a> shows the crystal structure of <b>40</b>; it is superimposable with the previously shown structure of <b>30</b>, with the addition of the key <span class="smallcaps smallerCapital">d</span>-alanine moiety at position 2. The methyl group of the <span class="smallcaps smallerCapital">d</span>-amino acid can be clearly seen pointing away from the protein and pointing toward solvent. It is also easy to see why an <span class="smallcaps smallerCapital">l</span>-amino acid side chain is not tolerated at this position as indicated in the alanine scan, as it would likely bump the indole ring system of the key 5F-Trp moiety at position 4, causing a displacement of this critical interaction.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0010.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Continued SAR evolution of key lead molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0011.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray crystal structure of 40 (lt. yellow) bound to the LDLR binding site on the surface of PCSK9 at 1.38 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIC">6XIC</a>). The molecule binds in an identical manner as <b>30</b>, with the methyl group of the <span class="smallcaps smallerCapital">d</span>-Ala at position two (highlighted in the black box) pointing away from the protein toward the solvent. The <span class="smallcaps smallerCapital">d</span>-amino acid at this position also blocks proteolytic degradation at the two flanking amide bonds caused by trypsin and elastase activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on the crystal structure of <b>30</b>, it was observed that there was some space available on the aryl ring of the DBX cross-linker to put a substituent in the position meta to the two sulfurs. Adding a fluorine to this position provided <b>41</b> (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>), which added a small amount of potency versus <b>40</b> but did not add any additional stability to the peptides (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), with the peptide remaining vulnerable to chymotrypsin proteolysis at the remaining two previously identified amide bonds. In addition, we also capped the OH of the tyrosine aryl ring at position 7 with a methyl group to give <b>42</b>, which retained similar potency to <b>40</b> but eliminated an H-bond donating group, an SAR item that could be of interest for later designs targeted at optimizing permeability/oral bioavailability.</div><div class="NLM_p">To more broadly understand the scope and limitations of the <span class="smallcaps smallerCapital">d</span>-amino acid side chain at position 2, we also prepared several direct analogues of <b>40</b> with various <span class="smallcaps smallerCapital">d</span>-amino acids replacing the <span class="smallcaps smallerCapital">d</span>-alanine. The results of this small SAR scan are detailed in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. Lipophilic side chains such as those on <b>43</b>, <b>44</b>, <b>47</b>, and <b>48</b> were all well tolerated, as were polar side chains such as <b>45</b> and <b>46</b>. In addition, <b>46</b> demonstrates that while the stereochemistry of the two centers is different, the substitution on positions 1 and 2 is likely interchangeable because both of these side chains point away from the protein and into solvent. The SAR in general suggested that position 2 was extremely tolerant and could likely be exploited via a wide variety of substituents and might later be useful as a handle for modifying solubility and physical properties or for adding various labels. In addition, <b>44</b> was screened for IV PK in the mouse; however, it displayed a similar short half-life to earlier analogues (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compound <b>44</b> also showed a protease stability profile similar to <b>40</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), although the bulkier side chain of <b>44</b> did appear to be able to impact the vulnerability to chymotrypsin as well.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SAR for <span class="smallcaps smallerCapital">d</span>-Amino Acids at Position 2<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0031.gif" alt="" id="gr27" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0039.gif" alt="" id="GRAPHIC-d7e2280-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">All values are the average of at least three determinations unless otherwise indicated; standard deviation is indicated for all <i>n</i> > 2; for <i>n</i> = 2, the reported error is the difference between the mean and the individual; values. <span class="smallcaps smallerCapital">d</span>-Nvl = <span class="smallcaps smallerCapital">d</span>-norvaline; <span class="smallcaps smallerCapital">d</span>-Val = <span class="smallcaps smallerCapital">d</span>-valine; <span class="smallcaps smallerCapital">d</span>-Thr = <span class="smallcaps smallerCapital">d</span>-threonine; <span class="smallcaps smallerCapital">d</span>-Lys = <span class="smallcaps smallerCapital">d</span>-lysine; <span class="smallcaps smallerCapital">d</span>-Cpg = <span class="smallcaps smallerCapital">d</span>-cyclopentyl gly; and <span class="smallcaps smallerCapital">d</span>-Tbg = <span class="smallcaps smallerCapital">d</span>-<i>tert</i>-butyl glycine. ND = not determined.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Discovery of Pro–Thr Backbones to Replace Asp–His</h3><div class="NLM_p">At this point, we wanted to take advantage of the finding that the <span class="smallcaps smallerCapital">d</span>-amino acid could add stability and wanted to couple this with other changes in the backbone to address the other metabolically labile amide bonds. In examining the crystal structures of compounds <b>30</b> and <b>40</b>, it was apparent that the Asp–His amino acids at positions 5 and 6 could be replaced with amino acids that could similarly mimic the turn-like geometry of this region. One simple replacement that immediately came to mind was Pro–Ala. The Pro–Ala dipeptide could serve as a simplified mimetic of Asp–His, while also using the five-membered ring of the proline to mask an additional NH, which we hoped would in turn stabilize one of the remaining two metabolically labile amide bonds. <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a> details the progression of several analogues based on Asp–His replacements. The simple Pro–Ala analogue <b>49</b> showed the viability of this modification, although some inhibitory potency was lost. The Pro–Thr analogue <b>50</b> regained some of the lost potency. Combining this Pro–Thr backbone change with the <span class="smallcaps smallerCapital">d</span>-alanine at position 2 and the meta fluoro-DBX linker (previously shown in <b>41</b>) provided analogue <b>51</b>. We were able to obtain a crystal structure of compound <b>51</b> (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>), which showed that Pro–Thr was a superimposable replacement for Asp–His with similar levels of potency and the ability to mask an additional amide bond and hopefully stabilize this region further against chymotrypsin. The proline ring perfectly mimics the turn-like geometry of Asp–His, and the side chain of Thr does not interact directly with the protein and is solvent-exposed. The fluorine atom on the aryl ring is shown to occupy open space in the center of the macrocycle and may participate in and enhance the intramolecular interactions between the α-Me Pro and the indole ring of 5F-Trp at position 4. This intramolecular interaction may in turn help to further stabilize the overall conformation of the molecule.</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0012.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Pro-Ala and Pro-Thr as replacements for Asp-His at positions 5 and 6 of the macrocyclic peptide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0013.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Crystal structure of 51 (gray) bound to the LDLR-binding site on PCSK9 at 1.48 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XID">6XID</a>). The macrocyclic peptide binds similarly to the previously shown crystal structures, with the Pro-Thr providing an excellent replacement for the original Asp-His at positions 5 and 6. The Thr side chain hydroxyl is solvent-exposed and does not make any direct contact with the protein. The fluorine atom on the aryl ring is shown to occupy open space in the center of the macrocycle and may participate in and enhance the intramolecular interactions between the alpha-Me Pro and the inner 5F-Trp aromatic.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Methylation of threonine OH to remove another H-bond donor provided <b>52</b>, which was equipotent with the unmethylated analogue. This change was performed to eliminate one H-bond donor from the molecule, and the addition or subtraction of a methyl group here did not appear to affect the potency or stability of the molecule greatly and as such were used interchangeably. Guided by the crystal structure, we were able to replace the proline in position 5 with the commercially available 4(<i>S</i>)-<i>cis</i> fluoroproline in an attempt to further increase the chymotrypsin stability. Combining this with the <span class="smallcaps smallerCapital">d</span>-alanine at position 2 provided <b>53</b>, which showed a 4-fold increase in inhibitory potency. Compound <b>53</b> demonstrated very good stability against the entire panel of proteases including chymotrypsin and good stability in mouse plasma (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, after IV dosing to mice, the compound was shown to have a moderate clearance and a relatively short half-life (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The introduction of a proline to this region appeared to help stability against some proteases; however, it did not solve the short <i>in vivo</i> half-life seen with these analogues despite their apparent improved <i>in vitro</i> stability. We therefore chose to pause work on Pro–Thr analogues.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Discovery and Remediation of Mast Cell Degranulation Activity</h3><div class="NLM_p">At this point, based on the observations from previous internal drug discovery programs, we became concerned about the potential for mast cell degranulation related to these peptides. We noted that all the molecules we were synthesizing contained a highly basic lysine side chain in an overall somewhat lipophilic framework. We had in previous peptide and peptidomimetic drug discovery programs correlated this generic structural phenotype in some cases with histamine release <i>in vivo</i>, likely because of the extensive mast cell degranulation activity.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> In an effort to confirm this hypothesis, we tested several molecules in our rat mast cell degranulation assay (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). For the three molecules initially screened, the results were striking. Moderately potent mast cell degranulation activity was observed for <b>4</b>, <b>30</b>, and <b>53</b>, all of which contained a lysine side chain in position 1 of the peptides.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Rat Mast Cell Degranulation Activity for Select Molecules<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound #</th><th class="colsep0 rowsep0" align="center">position 1 amino acid side chain</th><th class="colsep0 rowsep0" align="center">rat mast cell degranulation (EC<sub>50</sub>, μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">Lys</td><td class="colsep0 rowsep0" align="left"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="65150a160c110c130025534b5352">[email protected]</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">Lys</td><td class="colsep0 rowsep0" align="left"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="d7a7b8a4bea3bea1b297e1f9e1e0">[email protected]</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left">Lys</td><td class="colsep0 rowsep0" align="left"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="0979667a607d607f6c493a27383b">[email protected]</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Histamine release activity was determined in rat peritoneal mast cells, data for all compounds is the average of at least two determinations. Rat peritoneal lavage contains about 5% mast cells, which is sufficient to initiate a robust response (confirmed by the positive control 48/80); assay end point is histamine release in supernatant. Cells were incubated with test articles for 30 min in 1–3% DMSO. For all tested molecules and DMSO controls, there was no loss in cell viability up to 60 μM.</p></div></div></div><div class="NLM_p">However, we needed to gather additional data to support or disprove the mast cell degranulation hypothesis. We prepared a series of analogues at the position 2 lysine to try to directly confirm our initial hypothesis and also looked back at a few earlier analogues that were made at this position as well. <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a> details the potency and rat mast cell degranulation activity for these Lys analogues (<b>54–66</b>). All these analogues exhibited moderate to good inhibitory potency, again confirming the solvent-exposed nature of this substituent and the good tolerability to various changes at this position. A strong correlation was found between the basicity of the position 1 side chain and the observed mast cell degranulation activity. More basic, free amine side chain-containing amino acids such as ornithine (<b>55</b>) and epsilon <i>N</i>,<i>N</i>-dimethyl Lys (<b>61</b>) demonstrated mast cell degranulation activity, while the acidic side chain amino acids (<b>54</b>, <b>58</b>, <b>59</b>) and neutral side chain amino acids (<b>56</b>, <b>57</b>) were all negative in the assay. Interestingly, the permanently positively charged epsilon trimethyl Lys analogue <b>60</b> was also clean in our mast cell degranulation assay. Despite the potential stability issues and metabolic vulnerabilities based on the structure, <b>60</b> appeared to have superior solubility characteristics and we decided to evaluate the IV PK properties of the molecule. Peptide <b>60</b> was dosed IV to mice and demonstrated improved pharmacokinetics (PKs) with a relatively moderate clearance and 1.2 h half-life (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a>).</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Lysine Analogues at Position 2: Inhibitory Potency and Rat Mast Cell Degranulation Activity for <b>70–63</b>, <b>66</b><a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0032.gif" alt="" id="GRAPHIC-d7e2579-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0033.gif" alt="" id="gr28" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">LDLR-FRET and Alexa-FRET data is the average of at least three determinations unless noted otherwise in table with standard error indicated; for <i>n</i> = 2, the difference between the mean and the individual data points is noted as the error. The histamine release activity was determined in rat peritoneal mast cells, data for all compounds is derived from at least two agreeing determinations. Rat peritoneal lavage contains about 5% mast cells, which is sufficient to initiate a robust response; assay end point is histamine release in supernatant. Cells are incubated with test articles for 30 min in 1–3% DMSO. For all tested molecules and DMSO controls, there was no loss in cell viability up to 60 μM.</p></div><div class="footnote" id="t8fn2"><sup><sup>b</sup></sup><p class="last">48/80 (refs <a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and <a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>) is used as a positive control for the assay.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Mouse and Rat IV Pharmacokinetics for <b>60</b><a class="ref internalNav" href="#t9fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0034.gif" alt="" id="GRAPHIC-d7e2611-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–inf</sub> (nM h)</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–<i>x</i></sub> (nM h)</th><th class="colsep0 rowsep0" align="center" char="±">clearance (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="±"><i>V</i><sub>dss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="±"><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">protein binding<a class="ref internalNav" href="#t9fn3" aria-label="c">c</a> (% bound)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">C57BL/6 mice<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">932.72 ± 102.63</td><td class="colsep0 rowsep0" align="char" char="±">895.96 ± 70.06</td><td class="colsep0 rowsep0" align="char" char="±">6.20 ± 0.68</td><td class="colsep0 rowsep0" align="char" char="±">0.45 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">1.16 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">99.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wistar Han rat<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">3648.33 ± 422.20</td><td class="colsep0 rowsep0" align="char" char="±">3646.9 ± 421.60</td><td class="colsep0 rowsep0" align="char" char="±">3.16 ± 0.40</td><td class="colsep0 rowsep0" align="char" char="±">0.17 ± 0.00</td><td class="colsep0 rowsep0" align="char" char="±">2.46 ± 1.34</td><td class="colsep0 rowsep0" align="char" char=".">99.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">Compound was tested in C57BL/6 mice at 1 mpk using a 30% propylene glycol/70% PBS vehicle at a concentration of 0.5 mg/mL or at 1 mpk using a 20% DMSO/60% propylene glycol/20% water vehicle at a concentration of 1 mg/mL. All determinations are the average of two mice from a single experiment. The indicated errors are the difference between the mean and the individual values.</p></div><div class="footnote" id="t9fn2"><sup><sup>b</sup></sup><p class="last">Compound was tested in Wistar Han rats at 1 mpk using a 20% DMSO/60% PEG/20% water vehicle at a concentration of 1 mg/mL. All determinations are the average of two rats from a single experiment. The indicated errors are the difference between the mean and the individual values.</p></div><div class="footnote" id="t9fn3"><sup><sup>c</sup></sup><p class="last">Protein binding values are a single determination.</p></div></div></div><div class="NLM_p">Based on this result, <b>60</b> was also dosed IV in Wistar Han rats, again demonstrating much improved IV PK with a low clearance, 2.5 h half-life, and improved <i>in vivo</i> tolerability. This molecule demonstrated the best overall IV PK in rodents of any of the early molecules tested in this series. Acylating the lysine epsilon amino group with a peg as in <b>62</b> (synthesized as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) also eliminated mast cell degranulation activity, as did the addition of moderately basic side chain amines such as those in the PEGylated analogues <b>63</b> (synthesized as detailed in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) and <b>66</b> (synthesized as detailed in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The SAR at this position clearly confirmed that free basic amino acid side chains such as Lys and Orn were major contributors to the observed mast cell degranulation activity and removing the basic amine or substantially moderating the basicity remediated the undesired activity. In general, the observed mast cell degranulation/associated histamine release appears to be a compound/series issue, directly related to the overall properties of the rest of the molecule that contains the offending basic group.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> In general, most amine-containing drug-like molecules do not exhibit this activity; however, we have observed this phenomenon internally with a number of diverse molecules across various programs, suggesting that this behavior should be suspected anytime a molecule containing a basic amine side chain in a somewhat overall lipophilic framework demonstrates an unusual or idiosyncratic toxicity <i>in vivo</i>. Having obtained a reasonable understanding of this issue and how to remediate it, we returned to our efforts to optimize the overall properties of our molecules.</div><figure id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0021.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>62–63</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0022.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>66</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Novel Cyclizations</h3><div class="NLM_p">Upon further studying the crystal structures of <b>30</b> and <b>51</b>, we were struck by several potential opportunities for novel cyclizations. From the crystal structure of <b>51</b>, it was apparent that there was more room in the space occupied by the fluorine atom on the DBX linker. In fact, the fluorine atom points directly at the NH of the indole of the inner 5F-Trp, suggesting that the two groups could possibly be linked together (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>). We also chose to remove the C-terminal amide from this target molecule. We considered a number of possible cross-linking strategies, but a copper-catalyzed alkyne–azide cyclization (Click)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> reaction between an <i>N</i>-propargyl-substituted 5F-Trp and an azido-substituted linker aryl ring presented an opportunity to bridge the distance with a novel spacer of the appropriate length.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> This approach (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> also provided a very synthetically accessible route and the appropriate orthogonality of the associated functional groups to the target molecule. It was necessary to synthesize the requisite <i>N</i>-propargyl 5F-Trp amino acid and the azido-modified DBX cross-linker. <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> details the synthesis of the Fmoc-protected <i>N</i>-propargyl 5F-Trp amino acid <b>70</b>, starting from the commercially available 5F-Trp amino acid. The four-step synthesis was performed in excellent overall yield to give the desired product, which was used directly in solid-phase synthesis to assemble the linear peptide. The aryl azide intermediate <b>73</b> was prepared in two steps from a commercial starting material as shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The final peptide was assembled as shown in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. The linear peptide was assembled using standard SPPS, and the crude linear peptide cleaved and deprotected. The crude linear peptide intermediate was cyclized under standard conditions using the DBX-azide described above (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>, <b>73</b>) to give <b>74</b>, which was then subjected to the second cyclization<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> conditions to provide the novel bicyclic peptide <b>75</b>. Compound <b>75</b> showed similar potency (Alexa-FRET IC<sub>50</sub> = 1.36 nM) to earlier analogues in the Asp–His series (such as <b>30</b>) and some improvement versus analogues in the Pro–Thr series (such as <b>51</b>). The molecule showed excellent stability in our protease panel (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and was dosed IV to both rats and cynomolgus monkeys (<a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">Table <a class="ref showTableEvent internalNav" data-ID="tbl10" href="#tbl10">10</a></a>) to evaluate <i>in vivo</i> PKs. Interestingly, the compound showed extremely poor IV PK performance in rats; however, it was somewhat better in monkeys, suggesting that the issues seen <i>in vivo</i> with molecules containing a Pro at position 5 might be a rodent-specific issue. Despite its poor PK performance in rats, compound <b>75</b> represents a key, novel structure in the series, demonstrating good potency, improved stability, and reasonable IV PK performance in cynomolgus monkeys. Unfortunately, we were unable to obtain suitable crystals to allow for determination of a crystal structure of <b>75</b>. However, based on the structure shown in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a> as well as the data for <b>75</b>, one can deduce that the triazolomethyl linker does not drastically change the overall conformation of the molecule and likely serves as a simple direct link from the NH of the Trp to the aryl ring of the DBX group.</div><figure id="fig12" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0014.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Proposed novel cross-link (green arrow) from bis-meta position of the DBX linker to the indole nitrogen of the 5F Trp in position 4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch4" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0023.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <i>N</i>-Propargyl Fmoc-Protected 5F-Trp <b>70</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch5" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0024.jpeg" id="gr22" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Arylazide Intermediate <b>73</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch6" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0025.jpeg" id="gr23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Bicyclic Peptide <b>75</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="cht1" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0002.jpeg" id="gr25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Retrosynthetic Analysis of the Proposed Bicyclic Target Peptide</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. Rat and Cyno Monkey IV PKs for <b>75</b><a class="ref internalNav" href="#t10fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t10fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t10fn3" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0035.gif" alt="" id="GRAPHIC-d7e2958-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–inf</sub> (nM h)</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–<i>x</i></sub> (nM h)</th><th class="colsep0 rowsep0" align="center" char="±">clearance (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="±"><i>V</i><sub>dss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="±"><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char=".">protein binding<a class="ref internalNav" href="#t10fn3" aria-label="c">c</a> (% bound)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cynomolgus monkey<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">306.10 ± 48.85</td><td class="colsep0 rowsep0" align="char" char="±">303.71 ± 49.57</td><td class="colsep0 rowsep0" align="char" char="±">20.15 ± 3.15</td><td class="colsep0 rowsep0" align="char" char="±">1.06 ± 0.13</td><td class="colsep0 rowsep0" align="char" char="±">2.84 ± 1.46</td><td class="colsep0 rowsep0" align="char" char=".">97.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wistar Han rat<a class="ref internalNav" href="#t10fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">23.3 ± 4.1</td><td class="colsep0 rowsep0" align="char" char="±">19.4 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">107 ± 18</td><td class="colsep0 rowsep0" align="char" char="±">0.48 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="±">0.06 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">99.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup><sup>a</sup></sup><p class="last">Compound was tested in cynomolgus monkeys at 0.5 mpk using a 30% propylene glycol/70% PBS vehicle at a concentration of 0.5 mg/mL. All determinations are the average of two monkeys from a single experiment. The indicated errors are the difference between the mean and the individual values.</p></div><div class="footnote" id="t10fn2"><sup><sup>b</sup></sup><p class="last">Compound was tested in Wistar Han rats at 1 mpk using a 20% DMSO/60% PEG/20% water vehicle at a concentration of 1 mg/mL. All determinations are the average of two rats from a single experiment. The indicated errors are the difference between the mean and the individual values.</p></div><div class="footnote" id="t10fn3"><sup><sup>c</sup></sup><p class="last">Protein binding values are a single determination.</p></div></div></div><div class="NLM_p">There was also a second possible cyclization opportunity that became apparent to us from the crystal structures of <b>30</b> and <b>51</b>. We observed that the indole ring of 5F-Trp at position 3 and the side chain of Asp at position 5 were pointing outward into the solvent and were relatively close to each other. This suggested the possibility of closing a ring between the side chain of the Asp and the aryl ring of the side chain of 5F-Trp, as illustrated in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>. Such a ring could potentially serve to block metabolism at most or all of the remaining vulnerable amide bonds while also helping to lock the conformation of the molecule in its preferred binding conformation, thereby providing potential entropic gains. Of course, this would remove the existing intramolecular interaction between the Asp and His side chains but would constrain the overall conformation in a different manner.</div><figure id="fig13" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0015.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Proposed cyclization between the side chain of position 3 and position 5. The side chains point in the same direction and are in close proximity to each other, and the new bicyclic linker would lie in a mostly solvent-exposed region.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We decided to strip off the side chain of Lys at position 1 to a methyl group for ease of synthesis and to temporarily avoid the mast cell degranulation issue. After scanning the literature and available starting materials, we discovered that the meta-(aminomethyl)phenylalanine was commercially available and decided to use this as a replacement for 5F-Trp in position 3. We could then use amino acids containing carboxylic acid side chains at the Asp position 5 and attempt macrolactamization of these two amino acids to provide an amide cross-linked analogue. <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a> details the synthesis of these lactam cross-linked analogues, and <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a> details the SAR progression of the bicyclic series of inhibitors. The linear peptides were prepared on resin, followed by cleavage and deprotection to provide <b>76</b>. <b>76</b> was then cyclized using meta-dibromomethyl benzene (DBX), and the crude product purified via reversed phase prep HPLC to give the purified macrocycle <b>77</b>. The peptide was then treated with the coupling agent AOP<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> in the presence of DIEA to form the cyclic lactam, and the crude material again purified via reverse-phase prep HPLC to give the desired bicyclic lactam analogue <b>78</b>. This general synthetic approach was used to prepare the various amide analogues in this series. Compound <b>78</b> showed excellent potency in the LDLR-FRET assay, thus confirming our hypothesis about cyclization in this region. We were able to obtain a crystal structure of <b>78</b>, which is detailed in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a> and <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a>. Other than a slight twist of the DBX cross-linker group (there is some enlargement in the density here, suggesting some mobility for the aryl ring of the DBX cross-linking group), the crystal structure of compound <b>78</b> is very similar to the previous crystal structures in this series. <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a> shows that the molecule retains all the interactions of the previous inhibitors, with the addition of the lactam bicycle. <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>, which shows an overlay of <b>78</b> versus <b>30</b>, confirms the superimposability of the two structures. The crystal structure also suggests that all the vulnerable amide bonds are now in some way shielded, suggesting the potential for better stability. However, when we looked at the stability of compound <b>78</b> in our panel of proteases and in whole blood, we noted that a potential vulnerability to chymotrypsin remained (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Despite marginal solubility, compound <b>78</b> was evaluated for IV PK in the mouse (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), and the compound was shown to have a short half-life <i>in vivo</i>. In <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a> (highlighted in the black box), one of the amide nitrogen atoms previously identified as being susceptible to proteolysis by chymotrypsin remains unsubstituted in the structure and despite the presence of the bicycle this remains a vulnerability.</div><figure id="fig14" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0016.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. SAR and evolution of bicyclic amide PCSK9 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig15" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0017.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Crystal structure of <b>78</b> bound to the LDLR-binding site of PCSK9 at 1.43 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIE">6XIE</a>). The new bicyclic ring between positions 3 and 5 is clearly visible. The phenylalanine ring at position 3 occupies the same space as the fused ring system of 5F-Trp it replaces. Other than a slight twist of the DBX cross-linker aromatic ring, the rest of the molecule fits in a virtually identical manner to previous crystal structures in this series. The nitrogen of the amide bond between the inner 5F-Trp at position 4 and the glutamic acid at position 5 (indicated by the black box) remains exposed and vulnerable to chymotrypsin (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The crystal structure suggests that this nitrogen should be amenable to methylation because it points upward and away from the protein and does not participate in any interactions with the protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig16" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0018.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Superposition of the crystal structures of <b>30</b> (salmon) and <b>78</b> (green). The two molecules bind in a similar fashion, with all of the key hydrogen bonding and lipophilic interactions maintained. There is a slight twist in the DBX cross-linker (extra density here indicates likely some flexibility here). The new ring formed between the side chains of position 3 and position 5 is clearly visible, with the aryl ring sitting in a similar position as the indole of 5F-Trp and the newly formed cyclic amide solvent-exposed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch7" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0026.jpeg" id="gr24" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Representative Synthesis of Lactam Analogues</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because this amide bond does not participate in any hydrogen bonding interactions with the protein and points upward and away from the protein, based on the crystal structure, we rationalized that it should be amenable to N-methylation. Our earlier <i>N</i>-methyl scan had suggested that N-methylation at this position would not be well tolerated; however, given the somewhat different nature of this bicyclic structure, we decided to attempt N-methylation here. N-Methylation of the amide nitrogen between positions 4 and 5 (synthesized via a procedure identical to the one detailed earlier for <b>78</b> but using <i>N</i>-Me Glu at position 5 in place of Glu) provided <b>79</b>, which demonstrated good potency, although somewhat reduced from that seen with the parent (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>). Note that in going from peptide <b>78</b> to <b>79</b>, we also introduced an OMe Thr moiety in place of Thr. As stated earlier for compounds <b>50</b> versus <b>52</b>, we had previously established that Thr and OMe Thr provided identical inhibitory potency and did not affect stability (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), and these amino acids were used interchangeably in several cases. While the comparison between <b>78</b> and <b>79</b> is not perfectly direct, the change from <i>N</i>-H to <i>N</i>-Me at this key backbone position clearly provided a stability advantage (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) against chymotrypsin similar to that observed when going from a Asp–His backbone like <b>30</b> to a Pro–Thr backbone such as <b>50</b>. Compound <b>79</b> showed full stability against our panel of proteases as well as in the presence of mouse and cyno monkey whole blood, suggesting that the structure-guided changes we had made in the backbone were able to effectively block potential metabolism by key proteases. This was an important finding for our program and shows that structural information is key in drug discovery programs and can be used to not only enhance potency and drive SAR but also to guide efforts to stabilize metabolically vulnerable functionalities on key structures. Given the extremely poor solubility of compound <b>79</b>, we chose not to dose it <i>in vivo</i>.</div><div class="NLM_p">Additional studies were done around compounds <b>78–79</b> to look at various ring sizes for the lactam bicycle (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>). The peptides were synthesized in an identical manner to <b>78</b>, substituting the appropriate amino acids in the SPPS. Contracting the ring by one carbon (Asp at position 5) to give <b>80</b> provided a substantial potency loss. Expanding the ring by one carbon versus <b>78</b> (homoGlu at position 5) gave <b>81</b>, which lost 20-fold in inhibitory potency. Further enlarging the ring by one additional carbon versus <b>78</b> (bishomoGlu at position 5) gave <b>82</b>, which almost fully restored inhibitory potency. Finally, moving the substitution pattern of the aminomethyl group on the phenylalanine ring at position 2 from meta to para and cyclizing with the lengthened bishomoGlu used in analogue <b>82</b> provided <b>83</b>, which lost about 100-fold in potency but still retained somewhat interesting activity. We also continued to look for replacements for the 5F-Trp moiety in position 4 that bound inside the macrocycle with the fluorine embedded in a small pocket on the surface of the binding site. We were able to purchase the racemic 5F-4-quinoline amino acid, and synthesis of the peptide using this amino acid gave <b>84</b> (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>), which was isolated as an approximately 1:2 mixture of isomers at the quinoline amino acid center after chromatography and was tested as a mixture. The mixture of diastereomers <b>84</b> showed a slightly improved potency to the analogous 5F-Trp analogue and represented the only amino acid side chain that we found that was able to duplicate the activity seen with 5F-Trp at this position. We were able to obtain a crystal structure of <b>84</b>, shown in <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>, overlaid with the crystal structure of <b>30</b>. Only the correct diastereomer is able to bind and form suitable crystals and as such is the isomer represented in the crystal structure. The overlay shows that the fluoroquinoline ring of <b>84</b> appears to be a fully superimposable replacement for the 5F-Trp moiety of most of the inhibitors. This data once again reaffirmed the narrow SAR observed at this position and the critical nature of the aryl fluorine substitution here. Based on the combination of the marginal PK demonstrated with <b>78</b> as well as the poor solubility in general of these analogues, compounds <b>80–84</b> were not evaluated <i>in vivo</i>.</div><figure id="fig17" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0019.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Crystal structure of <b>84</b> at 1.77 Angstrom resolution (green; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIF">6XIF</a>) overlaid with the crystal structure of <b>30</b> (salmon) bound to the LDLR binding site of PCSK9. The overall fit of the molecule is identical to all of the previous inhibitor structures described in this publication. The fluoroquinoline at position 4 is shown to occupy the same pocket in an identical manner as 5F-Trp in previous crystal structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58104" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58104" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, by leveraging a structure-based design approach, we have taken an initial lead from an mRNA display screen and optimized the molecule, increasing the potency by almost 1000-fold, while finding two unique structural approaches to stabilize the molecules to key gut proteases and also reducing the molecular weight by 30%. We have also investigated an observed off-target issue of mast cell degranulation and have developed an SAR to mitigate this issue, which is a potential issue across all peptide platforms and should be suspected when the structural phenotype is present in the molecule or idiosyncratic toxicity is observed. Having optimized potency and stability to key gut enzymes and engineered out an important off-target issue, we are continuing our efforts in this series using these bicyclic platforms as a foundation, with the goal of finding novel, orally bioavailable cyclic peptide PCSK9 inhibitors. Although these novel bicyclic peptide inhibitors are not fully optimized, they provide a novel, potent, and metabolically stable platform for continued lead optimization in this space.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03437" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03437" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> General Methods</h3><div class="NLM_p">Amino acid building blocks and reagents were obtained from various commercial sources such as Sigma-Aldrich, Fisher, Novabiochem, Bachem, Chem-Impex, Iris, Genscript, Frontier, or Nagase & Co. Ltd., WuXi Apptec. The noncanonical amino acid Fmoc-5F-Trp was supplied by WuXi Apptec. SPPS resins were obtained from EMD and Anaspec. The purities of intermediates and final compounds were determined by LC/MS, conditions as follows unless otherwise noted using these methods:</div><div class="NLM_p">Method A—Waters Acquity BEH C18 column (2.1 × 100 mm, 1.7 μm) eluting with 0.1% trifluoroacetic acid (TFA) in water (solvent A) and 0.1% TFA in acetonitrile (solvent B) at a flow rate of 0.4 mL/min at 45 °C using a 0–4.0 min:25–45% B, 4.0–4.5 min 80% B gradient, detection via UV @ wavelengths of 215 nm. The mass spectra (MS) were recorded on a Waters ZQ mass spectrometer using the electrospray-positive ionization mode [ES<sup>+</sup> to give (MH)<sup>+</sup> molecular ions]. The cone voltage was 20 V.</div><div class="NLM_p">Method B—Waters Acquity BEH C18 column (2.1 × 100 mm, 1.7 μm) eluting with 0.1% TFA in water (solvent A) and 0.1% TFA in acetonitrile (solvent B) at a flow rate of 0.4 mL/min at 45 °C using a 0–4.0 min:20–40% B, 4.0–4.5 min 80% B. The mass spectra (MS) were recorded on a Waters ZQ mass spectrometer using the electrospray-positive ionization mode [ES<sup>+</sup> to give (MH)<sup>+</sup> molecular ions]. The cone voltage was 20 V.</div><div class="NLM_p">Method C—Waters BEH 130 C18 Acquity 2.1 × 100 mm, 1.7 u column, A/B mobile phases—A = 0.1% TFA in water; B = 0.1% TFA in acetonitrile; flow rate = 0.4 mL/min at 45 °C using a 5–95% B gradient over 10 min; and detection via UV @ wavelengths of 215 nm/254 nm (for some examples, 280 nm also), MS: API-ES. Cone voltage = 20 V.</div><div class="NLM_p">Method D—Waters Acquity BEH C18 50 mm × 1 mm × 1.7 μm, eluting with 0.05% TFA in water (solvent A) and 0.05% TFA in acetonitrile (solvent B) at a flow rate of 0.3 mL/min, a column temperature of 50 °C; 5–95% B (0.05% TFA) in 2 min. Detection at 215 nm. MS: API-ES. Cone voltage = 20 V.</div><div class="NLM_p">Method E—Waters BEH 130 C18 Acquity 2.1 × 100 mm, 1.7 u column, A/B mobile phases—A = 0.1% TFA in water; B = 0.1% TFA in acetonitrile; flow rate = 0.4 mL/min at 45 °C using a 5–95% B gradient over 15 min; detection via UV @ wavelengths of 215 nm/254 nm; the mass spectra were recorded on a Waters ZQ mass spectrometer using the electrospray-positive ionization mode [ES<sup>+</sup> to give (MH)<sup>+</sup> molecular ions]. The cone voltage was 20 V.</div><div class="NLM_p">Method F—Phenomenex EV C18 2.1 × 100 mm, 1.7 u; A/B mobile phases—A = 0.1% TFA in water; B = 0.1% TFA in acetonitrile; flow rate = 0.4 mL/min at 45 °C using a 20–65% B gradient over 6 min. The mass spectra were recorded on a Waters ZQ mass spectrometer using the electrospray-positive ionization mode [ES<sup>+</sup> to give (MH)<sup>+</sup> molecular ions]. The cone voltage was 20 V.</div><div class="NLM_p">Preparative HPLC was performed as indicated below. LC/MS data for all final compounds is summarized in <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">Table <a class="ref showTableEvent internalNav" data-ID="tbl11" href="#tbl11">11</a></a>. The purities of all final compounds were 95% or greater unless otherwise noted. <sup>1</sup>H NMR spectra were recorded in various solvents at ambient temperature using a Varian 600 MHz NMR.</div><div class="NLM_table-wrap" id="tbl11"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 11. Tabulated LC/MS Data for All Final Molecules<a class="ref internalNav" href="#t11fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t11fn2" aria-label="b">b</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound #</th><th class="colsep0 rowsep0" align="center" char=".">MW</th><th class="colsep0 rowsep0" align="center">MS [M + H]<sup>+</sup></th><th class="colsep0 rowsep0" align="center" char=".">MS [M + 2H]<sup>+</sup></th><th class="colsep0 rowsep0" align="center">LC method</th><th class="colsep0 rowsep0" align="center"><i>t</i><sub>R</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">1890.14</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">945.3</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">1475.67</td><td class="colsep0 rowsep0" align="left">1476.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">1418.62</td><td class="colsep0 rowsep0" align="left">1419.4</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">1359.48</td><td class="colsep0 rowsep0" align="left">1360.7</td><td class="colsep0 rowsep0" align="char" char=".">681.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">3.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">1429.6</td><td class="colsep0 rowsep0" align="left">1431.2</td><td class="colsep0 rowsep0" align="char" char=".">716.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">1282.5</td><td class="colsep0 rowsep0" align="left">1283.9</td><td class="colsep0 rowsep0" align="char" char=".">642.6</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">1283.45</td><td class="colsep0 rowsep0" align="left">1284</td><td class="colsep0 rowsep0" align="char" char=".">642.8</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.47</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">1371.6</td><td class="colsep0 rowsep0" align="left">1372.9</td><td class="colsep0 rowsep0" align="char" char=".">687.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">1350.51</td><td class="colsep0 rowsep0" align="left">1351.6</td><td class="colsep0 rowsep0" align="char" char=".">676.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">3.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">1323.5</td><td class="colsep0 rowsep0" align="left">1324.9</td><td class="colsep0 rowsep0" align="char" char=".">663.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">1390.54</td><td class="colsep0 rowsep0" align="left">1391.5</td><td class="colsep0 rowsep0" align="char" char=".">696.2</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.57</td><td class="colsep0 rowsep0" align="left">1417.8</td><td class="colsep0 rowsep0" align="char" char=".">709.2</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">1398.57</td><td class="colsep0 rowsep0" align="left">1399</td><td class="colsep0 rowsep0" align="char" char=".">700.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">1398.57</td><td class="colsep0 rowsep0" align="left">1398.8</td><td class="colsep0 rowsep0" align="char" char=".">700.3</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">3.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.57</td><td class="colsep0 rowsep0" align="left">1417.3</td><td class="colsep0 rowsep0" align="char" char=".">709.6</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">4.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.57</td><td class="colsep0 rowsep0" align="left">1417</td><td class="colsep0 rowsep0" align="char" char=".">709.6</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.57</td><td class="colsep0 rowsep0" align="left">1417.7</td><td class="colsep0 rowsep0" align="char" char=".">709.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.57</td><td class="colsep0 rowsep0" align="left">1416.9</td><td class="colsep0 rowsep0" align="char" char=".">709.4</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">3.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.57</td><td class="colsep0 rowsep0" align="left">1417</td><td class="colsep0 rowsep0" align="char" char=".">709.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">1430.6</td><td class="colsep0 rowsep0" align="left">1431.0</td><td class="colsep0 rowsep0" align="char" char=".">716.4</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="char" char=".">1430.6</td><td class="colsep0 rowsep0" align="left">1431.1</td><td class="colsep0 rowsep0" align="char" char=".">716.2</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23a</b></td><td class="colsep0 rowsep0" align="char" char=".">1412.63</td><td class="colsep0 rowsep0" align="left">1413.2</td><td class="colsep0 rowsep0" align="char" char=".">707.3</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">3.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23b</b></td><td class="colsep0 rowsep0" align="char" char=".">1412.63</td><td class="colsep0 rowsep0" align="left">1413.4</td><td class="colsep0 rowsep0" align="char" char=".">707.3</td><td class="colsep0 rowsep0" align="left">B</td><td class="colsep0 rowsep0" align="left">3.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24a</b></td><td class="colsep0 rowsep0" align="char" char=".">1412.63</td><td class="colsep0 rowsep0" align="left">1413.1</td><td class="colsep0 rowsep0" align="char" char=".">707.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24b</b></td><td class="colsep0 rowsep0" align="char" char=".">1412.63</td><td class="colsep0 rowsep0" align="left">1413.1</td><td class="colsep0 rowsep0" align="char" char=".">707.4</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="char" char=".">1430.62</td><td class="colsep0 rowsep0" align="left">1432</td><td class="colsep0 rowsep0" align="char" char=".">716.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="char" char=".">1430.6</td><td class="colsep0 rowsep0" align="left">1431.1</td><td class="colsep0 rowsep0" align="char" char=".">716.3</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="char" char=".">1430.62</td><td class="colsep0 rowsep0" align="left">1431.1</td><td class="colsep0 rowsep0" align="char" char=".">716.3</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="char" char=".">1431.64</td><td class="colsep0 rowsep0" align="left">1432.7</td><td class="colsep0 rowsep0" align="char" char=".">716.7</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">2.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="char" char=".">1372.53</td><td class="colsep0 rowsep0" align="left">1372.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="char" char=".">1429.55</td><td class="colsep0 rowsep0" align="left">1429.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char=".">1454.57</td><td class="colsep0 rowsep0" align="left">1454.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="char" char=".">1454.57</td><td class="colsep0 rowsep0" align="left">1454.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="char" char=".">1408.55</td><td class="colsep0 rowsep0" align="left">1408.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="char" char=".">1434.57</td><td class="colsep0 rowsep0" align="left">1434.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="char" char=".">1374.61</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="char" char=".">1390.5</td><td class="colsep0 rowsep0" align="left">1390.9</td><td class="colsep0 rowsep0" align="char" char=".">969.1</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="char" char=".">1401.52</td><td class="colsep0 rowsep0" align="left">1401.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="char" char=".">1429.55</td><td class="colsep0 rowsep0" align="left">1429.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="char" char=".">1388.62</td><td class="colsep0 rowsep0" align="left">1389.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="char" char=".">1402.64</td><td class="colsep0 rowsep0" align="left">1403.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="char" char=".">1461.6</td><td class="colsep0 rowsep0" align="left">1462</td><td class="colsep0 rowsep0" align="char" char=".">732</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">0.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.6</td><td class="colsep0 rowsep0" align="left">1417.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">3.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="char" char=".">1444.72</td><td class="colsep0 rowsep0" align="left">1445.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="char" char=".">1444.72</td><td class="colsep0 rowsep0" align="left">1445.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="char" char=".">1445.59</td><td class="colsep0 rowsep0" align="left">1446.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="char" char=".">1414.6</td><td class="colsep0 rowsep0" align="left">1415.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="char" char=".">1468.7</td><td class="colsep0 rowsep0" align="left">1469.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="char" char=".">1456.7</td><td class="colsep0 rowsep0" align="left">1457.7</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="char" char=".">1332.57</td><td class="colsep0 rowsep0" align="left">1333.5</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="char" char=".">1376.62</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">688.31</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">0.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="char" char=".">1407.6</td><td class="colsep0 rowsep0" align="left">1408.8</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="char" char=".">1390.65</td><td class="colsep0 rowsep0" align="left">1391.9</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="char" char=".">1408.64</td><td class="colsep0 rowsep0" align="left">410</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">0.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>54</b></td><td class="colsep0 rowsep0" align="char" char=".">1402.58</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">701.9</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="char" char=".">1416.57</td><td class="colsep0 rowsep0" align="left">1417.8</td><td class="colsep0 rowsep0" align="char" char=".">708</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">0.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="char" char=".">1401.59</td><td class="colsep0 rowsep0" align="left">1402.8</td><td class="colsep0 rowsep0" align="char" char=".">701.4</td><td class="colsep0 rowsep0" align="left">D</td><td class="colsep0 rowsep0" align="left">0.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>57</b></td><td class="colsep0 rowsep0" align="char" char=".">1415.6</td><td class="colsep0 rowsep0" align="left">1416.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="char" char=".">1430.63</td><td class="colsep0 rowsep0" align="left">1431.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="char" char=".">1443.6</td><td class="colsep0 rowsep0" align="left">1444.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="char" char=".">1457.7</td><td class="colsep0 rowsep0" align="left">1457.64 (M<sup>+</sup>)</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">E</td><td class="colsep0 rowsep0" align="left">4.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="char" char=".">1441.7</td><td class="colsep0 rowsep0" align="left">1442.61</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">E</td><td class="colsep0 rowsep0" align="left">4.75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="char" char=".">1647.7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">825.4</td><td class="colsep0 rowsep0" align="left">E</td><td class="colsep0 rowsep0" align="left">4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63</b></td><td class="colsep0 rowsep0" align="char" char=".">1676.7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">839.6</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">4.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66</b></td><td class="colsep0 rowsep0" align="char" char=".">1517.61</td><td class="colsep0 rowsep0" align="left">1518.3</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">E</td><td class="colsep0 rowsep0" align="left">4.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="char" char=".">1383.62</td><td class="colsep0 rowsep0" align="left">1383.8</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">6.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78</b></td><td class="colsep0 rowsep0" align="char" char=".">1290.56</td><td class="colsep0 rowsep0" align="left">1291.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79</b></td><td class="colsep0 rowsep0" align="char" char=".">1317.6</td><td class="colsep0 rowsep0" align="left">1318.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">6.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>80</b></td><td class="colsep0 rowsep0" align="char" char=".">1276.6</td><td class="colsep0 rowsep0" align="left">1277.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81</b></td><td class="colsep0 rowsep0" align="char" char=".">1304.58</td><td class="colsep0 rowsep0" align="left">1305.1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>82</b></td><td class="colsep0 rowsep0" align="char" char=".">1318.61</td><td class="colsep0 rowsep0" align="left">1319.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83</b></td><td class="colsep0 rowsep0" align="char" char=".">1317.6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">659.5</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="left">5.28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84</b><a class="ref internalNav" href="#t11fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">1328.6</td><td class="colsep0 rowsep0" align="left">1329.57</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">3.90, 4.11</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t11fn1"><div class="footnote" id="t11fn1"><sup><sup>a</sup></sup><p class="last">LC methods are described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>; all molecules demonstrated >95% purity by LC; representative LC traces for key molecules are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div><div class="footnote" id="t11fn2"><sup><sup>b</sup></sup><p class="last">Compound <b>84</b> was recovered as a mixture of diastereomers, combined purity > 99%; MS for each peak was identical.</p></div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> LDLR-FRET Assay</h3><div class="NLM_p last">A time-resolved fluorescence resonance energy (TR-FRET) assay was used to measure inhibition of PCSK9-LDLR PPI. Briefly, 20 nM avitag-biotinylated human PCSK9 was incubated with 20 nM His-tagged human LDLR EGFa domain in the presence of 5 nM LANCE Ulight Streptavidin (PerkinElmer) and 5 nM europium-Anti-6xHis (Perkin Elmer) for 2 h covered at room temperature in buffer containing 50 mM HEPES, 150 mM NaCl, 5 mM CaCl<sub>2</sub>, 0.01% BSA, and 0.01% surfactant P20. Compounds were tested in dose–response and concentrations giving half-maximal inhibition calculated using a four-parameter fit equation. Dose–response curves consisted of 10-point, half-log dilutions over a concentration range from 0.0015532 to 48.8 μM. The goodness of fit for IC<sub>50</sub> is a four-parameter logistic fit based on the Levenberg–Marquardt algorithm. <i>K</i><sub><i>i</i></sub> was then calculated from the IC<sub>50</sub> and <i>K</i><sub>D</sub> of PCSK9 to LDLR (480 nM, determined by SPR). The number of replicates varies for the compounds, and full details are provided in the table footnotes. For determinations of <i>n</i> < 3, the error is characterized as the difference between the individual values and the mean. For determinations of <i>n</i> = 3 or greater, the error is characterized as the standard deviation. All n values are the result of independent determinations.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Alexa-FRET Assay</h3><div class="NLM_p">The PCSK9 TR-FRET Alexa-FRET Standard assay measures the interaction between PCSK9 (biotinylated/C-Avitagged) and an AlexaFluor647 (AF)-tagged cyclic peptide, reagent A (<i>K</i><sub>D</sub> = 83 nM). A solution containing 1 nM biotinylated PCSK9 + 2.5 nM Lance Streptavidin Europium (Strep-Eu) was made in 50 mM HEPES pH 7.4, 0.15 M NaCl, 5 mM CaCl<sub>2</sub>, 0.01% BSA, and 0.01% surfactant P20. A separate solution containing 40 nM of the AlexaFluor-tagged cyclic peptide was made in the same buffer system. An Echo robot was used to transfer 0.750 μL of test compound to an assay plate, followed by the addition of 15 μL of PCSK9+Stept-Eu and 15 μL of AF peptide. The final assay volume was 30.750 μL containing 0.5 nM PCSK9, 1.25 nM Strep-Eu, and 20 nM AF cyclic peptide. The reaction was incubated at room temperature for at least 2 h prior to fluorescence measurements using an Envision Multilabel Reader. IC<sub>50</sub> values were determined by fitting data to a sigmoidal dose–response curve using nonlinear regression. Compounds were tested in dose–response and concentrations giving half-maximal inhibition were calculated using a four-parameter fit equation. The dose–response curves consisted of 10-point, half-log dilutions over a concentration range from 0.0015532 to 48.8 μM. The goodness of fit for IC<sub>50</sub> is a four-parameter logistic fit based on the Levenberg–Marquardt algorithm. <i>K</i><sub><i>i</i></sub> was then calculated from the IC<sub>50</sub> and <i>K</i><sub>D</sub> of the AF cyclic peptide. Counts (B-counts) of the europium-labeled PCSK9 were followed to observe if compounds were adversely affecting PCSK9. A fall-off of the B-counts likely indicated a false positive of inhibition. The number of replicates varies for the compounds, and full details are provided in the table footnotes. For determinations of <i>n</i> < 3, the error is characterized as the difference between the individual values and the mean. For determinations of <i>n</i> = 3 or greater, the error is characterized as the standard deviation. All <i>n</i> values are the result of independent determinations.</div><div class="NLM_p">Reagent A was prepared as shown below:<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0036.gif" alt="" id="dgr1" /></img></div><div class="NLM_p last">Synthesis of reagent A: compound <b>30</b> (15 mg) was dissolved in 0.2 mL of dry DMSO. Then, 15 mg of AlexaFluor 647NHS Ester (A37566, Life Technology) dissolved in 1.5 mL of dry DMSO was added. Dry DIEA (20 μL) was added. The reaction was left under stirring at room temperature for 12 h under the nitrogen atmosphere in the dark, quenched with TFA (pH to 3–4), and purified by RP-HPLC (Dr Maish, Reprosil Gold C18, 250 × 20 mm, 120 c5, 10 μm; 20 to 35% of 0.1% TFA in ACN/0.1% TFA in H<sub>2</sub>O, over 20 min, then 35 to 40% over 5 min at 20 mL/min flow rate). The collected fractions were lyophilized to afford 16.1 mg of purified reagent A. LCMS analysis was calcd for C<sub>105</sub>H<sub>122</sub>F<sub>2</sub>N<sub>17</sub>O<sub>26</sub>S<sub>6</sub><sup>3–</sup>, calc. 2268.58;  found 1135.8 [M + 2H]<sup>+</sup>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> General Procedure for the Preparation of Peptides <b>2–61</b></h3><div class="NLM_p">Methods A and B below were used interchangeably with similar results obtained from each. In general, the yields of final peptides <b>2-61</b> varied from 5 to 50% after SPPS, cleavage, cyclization, and purification, and in all cases, the desired products were recovered after purification in >95% purity. In the cases of compounds <b>41</b> and <b>51</b>, the commercially available 1,3-di(bromomethyl)-5-fluorobenzene was substituted for 1,3-di(bromomethyl)benzene (DBX) in step 3 (cyclization) using identical conditions.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Method A</h4><div id="sec4_4_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Step 1: Peptide Synthesis</h5><div class="NLM_p last">The peptides were synthesized using commercial Fmoc-protected amino acids on a solid-phase Rink Amide MBHA (NovaBiochem, ∼0.5 mequiv/g, 100–200 mesh) or AM (NovaBiochem, ∼0.73 mequiv/g, 100–200 mesh) or 2-aminoethanethiol-2-chlorotrityl resin (Anaspec, ∼0.9 mequiv/g, 200–400 mesh) with a CEM Liberty Blue automated microwave peptide synthesizer. The peptides were typically synthesized on a 0.1 mmol scale. Typical reaction conditions were as follows: deprotection conditions: 20% piperidine (v/v) in dimethylformamide (DMF) (2 × 2 min at 75 °C); residue coupling conditions: 5 equiv (relative to resin) of activated amino acid (2.5 mL of a 0.2 M amino acid stock solution in DMF) was delivered to the resin, followed by 5 equiv of DIC activator (1 mL of a 0.5 M solution in DMF) and 5 equiv of Oxyma Pure (0.5 mL of a 1 M solution in DMF) and allowed to react for 5 min at 75 °C. The resultant product was used directly in the next step.</div></div><div id="sec4_4_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Step 2: Cleavage from Solid Support</h5><div class="NLM_p last">The resin from step 1 was transferred to either a 50 mL polypropylene centrifuge tube or a 10 mL fritted syringe. The peptides were cleaved from their solid support using TFA/triisopropylsilane/<span class="smallcaps smallerCapital">dl</span>-dithiothreitol/water (92.5/2.5/2.5 (w/v)/2.5) mixture. For 0.1 mmol scale of resin, ∼10 mL cleavage solution was used. The suspended resin was rocked in cleavage solution for 3 h at room temperature. The filtrate was collected in 50 mL polypropylene centrifuge tubes and precipitated with chilled diethyl ether (∼50 mL per ∼5 mL cleavage filtrate). The precipitated crude peptide was collected by centrifugation. The white pellet was then subsequently suspended in chilled diethyl ether and collected by centrifugation two additional times. The resulting solid was air-dried to afford the crude peptide.</div></div><div id="sec4_4_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Step 3: Cyclization</h5><div class="NLM_p last">For a 0.1 mmol scale, the linear peptide was synthesized, cleaved from the solid support, and isolated according to step 2. The solid crude peptide was dissolved with stirring in 30 mL of a degassed water/acetonitrile (1:1) solution in a 50 mL polypropylene centrifuge tube. Additional acetonitrile was added as needed to ensure complete dissolution of the peptide. To this stirred solution, aqueous ammonium bicarbonate (200 mM, degassed) was added dropwise to bring the pH to ∼8. Alkylating reagent 1,3-di(bromomethyl)benzene (0.1 mmol/1 equiv) in acetonitrile (1 mL) was added dropwise to the reaction over ∼2 min. After complete addition, the reaction was tested to ensure that the pH was maintained at ∼8. If the pH was below ∼8, additional aqueous ammonium bicarbonate (200 mM, degassed) was added dropwise to the reaction until the pH was ∼8. The centrifuge tube was capped and the resulting reaction mixture was stirred at room temperature for 1 h. The reaction was then acidified to pH ∼1 with TFA and lyophilized to afford the crude cyclic peptide as a powder. The residue was redissolved in 19:1 DMSO–water (v/v) and purified using gradient elution on reverse phase (50 × 250 mm Sunfire Prep C18; 25–75% CH<sub>3</sub>CN/water w/0.1% TFA modifier over 30 min). The fractions were then concentrated <i>in vacuo</i> to provide the cyclized peptide.</div></div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Method B: Alternative HATU Method on CEM Liberty</h4><div id="sec4_4_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Step 1: Peptide Synthesis</h5><div class="NLM_p last">The peptide was synthesized using Fmoc/<i>t</i>-Bu chemistry on cysteamine 4-methoxytrityl resin (NovaBioChem, 0.82 mmol/g) with a CEM Liberty automated microwave peptide synthesizer. The peptide sequence was typically synthesized on a 0.25 mmol scale. Typical reaction conditions were as follows: Fmoc deprotections were performed using 20% (v/v) piperidine in DMF (10 mL, 90 s at 90 °C). Residue coupling conditions: single couplings of 4 equiv (relative to resin) of Fmoc-protected amino acids (as a 0.2 M DMF solution) along with 3.6 equiv of HATU (as a 0.45 M DMF solution) and 8 equiv of 2 M DIEA in NMP. Coupling cycle: 300 s at 75 °C or for trityl side chain-protected residues: 120 s at 20 °C, followed by 240 s at 50 °C.</div></div><div id="sec4_4_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> Step 2: Cleavage from Solid Support</h5><div class="NLM_p last">After isolation of the resin via filtration, the peptide was cleaved from solid support using 15 mL of TFA solution (v/v) (90.5% TFA:2.5% DODT:2.5% phenol:2% triisopropylsilane:2.5% water) for approximately 4 h at room temperature. The resin was filtered and washed with 5 mL of TFA solution. Combined filtrate was concentrated and precipitated in approximately 60 mL of cold ethyl ether (−78 °C). Crude peptide pellet collected by centrifugation was washed in cold ethyl ether and collected by centrifugation twice more. The resulting solid was air-dried to afford the crude peptide.</div></div><div id="sec4_4_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Step 3: Cyclization</h5><div class="NLM_p last">Crude linear peptide (0.25 mmol) was redissolved in a solution of degassed acetonitrile (150 mL) and degassed 20 mM aqueous solution of ammonium carbonate (63 mL, 1.26 mmol), a degassed solution of 1,3-di(bromomethyl)benzene (78 mg, 0.29 mmol) in ACN (5 mL total with rinse) was added, and the reaction under an atmosphere of nitrogen for 18 h was stirred. The reaction was filtered and then concentrated <i>in vacuo</i>. The residue was redissolved in 19:1 DMSO–water and purified using gradient elution on reverse phase (50 × 250 mm Sunfire Prep C18; 25–75% CH<sub>3</sub>CN/water w/0.1% TFA modifier over 30 min). The fractions were then concentrated <i>in vacuo</i> to provide the cyclized peptide.</div></div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Synthesis of Compound <b>62</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h3><div class="NLM_p last">Compound <b>4</b> (4.4 mg, 3.08 μmol) was dissolved in 1.0 mL of DMF. 2,5-Dioxopyrrolidin-1-yl 2,5,8,11-tetraoxatetradecan-14-oate (2.2 mg, 6.60 μmol) was dissolved in 100 μL of DMF and then was added directly to the peptide solution. The reaction was stirred at room temperature for 2 h, at which time LCMS showed the reaction was completed. The reaction mixture was diluted with DMF and water to 3 mL, then purified on RP-HPLC using a Vydac protein & peptide C18, 250 × 10 mm, wide pore; gradient: 20–60% over 50 min.; mobile phases: A = 0.1% TFA in water; B = 0.1% TFA in acetonitrile. Compound <b>62</b> (1.90 mg) was obtained with purity >95% (HPLC), calcd C<sub>79</sub>H<sub>99</sub>F<sub>2</sub>N<sub>15</sub>O<sub>18</sub>S<sub>2</sub>, 1647.67; observed mass [M + 2H]<sup>+</sup>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Synthesis of Compound <b>63</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h3><div class="NLM_p">Compound <b>4</b> (12 mg, 8.39 μmol) was dissolved in 0.5 mL of DMF. 2,2-Dimethyl-4-oxo-3,8,11,14,17-pentaoxa-5-azaicosan-20-oic acid (6.13 mg, 0.017 mmol), (<i>E</i>)-ethyl-2-cyano-2-(hydroxyimino)acetate (Oxyma Pure; 0.017 mL, 0.017 mmol), and <i>N</i>,<i>N</i>′-methanediylidenebis (propan-2-amine) (DIC; 0.034 mL, 0.017 mmol) were mixed together, allowed to stand at room temperature for 30 min. Then, this mixture was added to the peptide solution. LCMS showed the reaction was ongoing. Preactivated Boc-PEG-COOH (0.017 mmol) was added twice to the solution. After 48 h, the reaction was completed by LCMS analysis. The reaction was concentrated, and 2 mL of 1:1 TFA in DCM was added directly to the residue. After 30 min, the TFA solution was concentrated and suspended in cold diethyl ether. The mixture was centrifuged, the precipitate was dissolved in acetonitrile/water, and the solution was freeze-dried to a fluffy white material. Purification was done with RP_HPLC using a Vydac protein & peptide C18, 250 × 10 mm, wide pore, with a 20–60% A gradient over 50 min; mobile phases: A = 0.1% TFA in water; B = 0.1% TFA in acetonitrile.</div><div class="NLM_p last">Compound <b>63</b> (5.72 mg) was obtained with purity > 95% (HPLC), calc. C<sub>80</sub>H<sub>102</sub>F<sub>2</sub>N<sub>16</sub>O<sub>18</sub>S<sub>2</sub> 1676.70; observed [M + 2H]<sup>+</sup> = 839.64.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Synthesis of Compound <b>66</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>)</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Step 1—Synthesis of Compound <b>64</b></h4><div class="NLM_p">The peptidyl resin intermediate was synthesized using Fmoc/<i>t</i>-Bu chemistry on cysteamine 4-methoxytrityl resin (Novabiochem, 0.82 mmol/g) with a CEM Liberty automated microwave peptide synthesizer. The peptide sequence was synthesized on a 1.0 mmol scale using single couplings of 3.3 equiv of Fmoc-protected amino acids as a 0.2 M DMF solution along with 3.0 equiv of HATU as a 0.45 M DMF solution and 6.6 equiv of 2 M DIEA in NMP. Fmoc deprotections were performed using 20% (v/v) piperidine in DMF.</div><div class="NLM_p last">Compound <b>64</b> was sampled for cleavage in 2 mL of 88% TFA, 5% DODT (3,6-dioxa-1,8-octanedithiol), 2% phenol, 2.5% triisopropylsilane, and 2.5% water to check the peptide quality. The correct linear peptide <b>64</b> was confirmed. <b>64</b> was used as the starting resin in the next step. LCMS anal calcd for C<sub>53</sub>H<sub>62</sub>F<sub>2</sub>N<sub>12</sub>O<sub>10</sub>S, 1096.44; found, 1097.0 [M+H]<sup>+</sup>.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Step 2—Synthesis of Compound <b>65</b></h4><div class="NLM_p">Synthesis of linear peptide was continued on <b>64</b> (0.1 mmol) on a Biotage Alstra peptide synthesizer (BiotageCorp.) with two cycles of elongation using 4 equiv of double coupling with DIC/Oxyma Pure as activators, and the amino acids (<i>S</i>)-2-((((9<i>H</i>-fluoren-9-yl)methoxy)carbonyl)amino)-3-((1-(4,4-dimethyl-2,6-dioxocyclohexylidene)-3-methylbutyl)amino)propanoic acid and 3-(tritylthio)propanoic acid, respectively. Fmoc deprotections were performed using 20% (v/v) piperidine in DMF.</div><div class="NLM_p">The peptidyl resin was treated with 5% hydrazine in DMF for 10 min (repeated twice) to remove the side chain protection group ivDde. The peptidyl resin was manually coupled with 4 equiv of 2,2-dimethyl-4-oxo-3,8,11-trioxa-5-azatridecan-13-oic acid (2.000 mL, 0.4 mmol) using DIC/Oxyma Pure for 1 h.</div><div class="NLM_p">The fully assembled peptide on-resin was cleaved using 10 mL of 88% TFA, 5% DODT (3,6-dioxa-1,8-octanedithiol), 2% phenol, 2.5% triisopropylsilane, and 2.5% water for 2 h at room temperature. Another 5 mL of TFA solution was used to wash the peptidyl resin. After filtering, combined TFA solutions were concentrated <i>in vacuo</i>. Approximately 40 mL of chilled diethyl ether was added to the mixture to precipitate the peptide. The crude peptide was collected by centrifugation on a Thermo ST40R at 2900 rpm for 15 min. The pellet was then washed with 40 mL of chilled ether again. The washed peptide pellet was dissolved in 0.1% TFA acetonitrile/water solution and then was lyophilized to dryness. The correct linear peptide <b>65</b> was confirmed. <b>65</b> was used directly as a crude in the next step reaction.</div><div class="NLM_p last">LCMS anal calcd for C<sub>65</sub>H<sub>83</sub>F<sub>2</sub>N<sub>15</sub>O<sub>15</sub>S<sub>2</sub>, 1415.56; found <i>m</i>/<i>z</i><sup>+2</sup>, 709.08.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Step 3—Synthesis of Compound <b>66</b></h4><div class="NLM_p">Crude <b>65</b> (0.1 mmol) was redissolved in a solution of degassed acetonitrile (60 mL) and 20 mM NH<sub>4</sub>HCO<sub>3</sub> in water (90 mL). To this was slowly added a degassed solution of 1,3-di(bromomethyl)benzene (52.8 mg, 0.2 mmol) in acetonitrile (60 mL total with rinse). The reaction was stirred at room temperature until completed (18 h). The reaction was lyophilized to dryness and then purified using gradient elution on the reverse phase (30 × 150 mm OBD, Waters Xselect CSH130 C18, 5 μm; 31–36% ACN/water w/0.16% TFA modifier over 25 min). The fractions were lyophilized to provide <b>66</b> in 96% purity (4.6 mg), along with 1.5 mg of a slightly earlier eluting second product identified as a racemate at an undetermined stereocenter (verified by Alexa-FRET assay = 385 nM).</div><div class="NLM_p last">LCMS anal calcd for C<sub>73</sub>H<sub>89</sub>F<sub>2</sub>N<sub>15</sub>O<sub>15</sub>S<sub>2</sub>, 1517.61; found, 1517.61.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Synthesis of Compound <b>75</b> (<a class="ref internalNav" href="#sch4" aria-label="Schemes 4">Schemes 4</a>–<a class="ref internalNav" href="#sch6" aria-label="6">6</a>)</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Step 1—Synthesis of Intermediate Compound <b>67</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>)</h4><div class="NLM_p">In a flame-dried flask under an atmosphere of nitrogen, (<i>S</i>)-2-amino-3-(5-fluoro-1<i>H</i>-indol-3-yl)propanoic acid (10.8 g, 48.6 mmol) was dissolved in anhydrous MeOH (200 mL) and cooled in an ice bath. Trimethylchlorosilane (TMS-Cl) was added (35.6 mL, 279 mmol) dropwise, and the resulting solution was stirred at room temperature overnight. An additional 4 mL of TMS-Cl was added, and the reaction was allowed to stir for an additional 18 h to drive the formation of the desired methyl ester.</div><div class="NLM_p">The reaction was cooled in an ice bath, and triethylamine (38 mL, 273 mmol) was added to raise the reaction pH to ∼9. A solution of BOC-anhydride (12.98 mL, 55.9 mmol) dissolved in methanol (20 mL) was added, and the reaction was stirred at room temperature. After 1 h, an additional 6 mL of triethylamine was added to make the reaction basic, followed by stirring at room temperature for another hour. The reaction was concentrated <i>in vacuo</i> and was partitioned residue between water and ether. The aqueous layer was extracted 3× with ether, and the combined extracts were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered through a plug of silica gel and concentrated <i>in vacuo</i>. The residue was concentrated <i>in vacuo</i> to dryness. The resultant crystalline solid was triturated with hexanes 3× to provide <b>67</b> (15.5 g, 95%).</div><div class="NLM_p">LCMS anal calcd for C<sub>17</sub>H<sub>21</sub>FN<sub>2</sub>O<sub>4</sub>, 336.4; found, 673.5 (2M + 1)<sup>+</sup>.</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, chloroform-<i>d</i>): δ 8.08 (s, 1H), 7.27–7.24 (m, 1H) (mixed with CHCl<sub>3</sub> peak), 7.17 (d, <i>J</i> = 9.4 Hz, 1H), 7.04 (s, 1H), 6.96–6.90 (m, 1H), 5.07 (d, <i>J</i> = 7.5 Hz, 1H), 4.65–4.61 (m, 1H), 3.69 (s, 3H), 3.30–3.18 (m, 2H), 1.43 (s, 9H).</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Step 2—Synthesis of Intermediate Compound <b>68</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>)</h4><div class="NLM_p">In a flame-dried flask under an atmosphere of nitrogen, <b>67</b> (1330 mg, 3.95 mmol) was dissolved in anhydrous DMF (30 mL) and the resulting solution was cooled in an ice bath. Sodium hydride (60% disp.) was added (125 mg, 4.94 mmol) portionwise. The mixture was stirred for 20 min at 0 °C until it became a pale-colored solution. 3-Bromoprop-1-yne (80% in xylene) (0.529 mL, 4.74 mmol) was added dropwise, and the resulting solution was stirred at 0 °C for an hour. Aqueous lithium hydroxide (1 M, 6 mL, 6.00 mmol) was added dropwise to the solution and the mixture was stirred at room temperature for 90 min. The solution was quenched with 6 mL of 1 N HCl to adjust the pH ∼3. The mixture was extracted 3× with EtOAc, and the combined organic extracts were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to provide <b>68</b>, which was used as a crude in the next step (1.425 mg, 100%).</div><div class="NLM_p last">LCMS anal. calcd for C<sub>19</sub>H<sub>21</sub>FN<sub>2</sub>O<sub>4</sub>, 360.4; found, 721.5 (2M + 1)<sup>+</sup>.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Step 3—Synthesis of Intermediate Compound <b>69</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>)</h4><div class="NLM_p">Crude <b>68</b> (1425 mg, 3.95 mmol) was dissolved in 4 M HCl in dioxane (20 mL, 80 mmol) at room temperature. After 45 min, the reaction was concentrated <i>in vacuo</i>. The residue was concentrated twice from ether to form a yellow solid, which was dried <i>in vacuo</i> to provide <b>69</b>, which was used as a crude in the next step (1173 mg, 100%).</div><div class="NLM_p last">LCMS anal. calcd for C<sub>14</sub>H<sub>13</sub>FN<sub>2</sub>O<sub>2</sub>, 260.3; found, 261.2 (M + 1)<sup>+</sup>.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Step 4—Synthesis of Intermediate Compound <b>70</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>)</h4><div class="NLM_p">Crude <b>69</b> (1173 mg, 3.95 mmol) was dissolved in acetone (30 mL) along with a solution of sodium carbonate (838 mg, 7.91 mmol) in water (60 mL). A solution of Fmoc-OSu (1334 mg, 3.95 mmol) dissolved in 30 mL of acetone was added, and the solution was stirred at room temperature for 1 h. The reaction was acidified by addition of 1 N aq HCl (∼11 mL) dropwise to lower pH to ∼3. The solution was extracted 3× EtOAc, and the combined organic extracts were washed with brine and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solution was filtered, and the filtrate concentrated <i>in vacuo</i>. The residue was dissolved in DCM and purified by flash chromatography (120 g SiO<sub>2</sub>, 10–80% EtOAc/hexanes). The appropriate fractions were concentrated <i>in vacuo</i> to provide <b>70</b> (1600 mg, 84%). <b>70</b> was used in the assembly of the crude linear peptide <b>74</b>.</div><div class="NLM_p">LCMS anal calcd for C<sub>29</sub>H<sub>23</sub>FN<sub>2</sub>O<sub>4</sub>, 482.5; found, 483.3 (M + 1)<sup>+</sup>.</div><div class="NLM_p last"><sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.71 (s, 1H), 7.88 (d, <i>J</i> = 7.5 Hz, 2H), 7.74 (d, <i>J</i> = 8.3 Hz, 1H), 7.66 (dd, <i>J</i> = 13.2, 7.5 Hz, 2H), 7.47 (dd, <i>J</i> = 8.9, 4.4 Hz, 1H), 7.44–7.36 (m, 3H), 7.34–7.23 (m, 3H), 7.08–7.00 (m, 1H), 5.08–5.00 (m, 2H), 4.27–4.15 (m, 4H), 3.38 (s, 1H), 3.15 (dd, <i>J</i> = 14.6, 4.6 Hz, 1H), 3.00 (dd, <i>J</i> = 14.6, 9.7 Hz, 1H).</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Step 5—Synthesis of Intermediate Compound <b>73</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>)</h4><div class="NLM_p">Compound <b>71</b> (5-amino-1,3-phenylene) dimethanol (3 g, 19.59 mmol) was dissolved in acetonitrile (200 mL) under an atmosphere of nitrogen. The solution was cooled in an ice bath and <i>tert</i>-butyl nitrite (3.85 mL, 32.4 mmol) was added dropwise to form an orange solution. After 15 min of stirring at 0 °C, trimethylsilyl azide (2.99 mL, 22.52 mmol) was added dropwise over 5 min. The reaction was warmed to room temperature and stirred for 2 h and then concentrated <i>in vacuo</i>. The residue was stirred in 2:1 toluene–ether, filtered with 2:1 toluene–ether and 3:1 toluene–hexane rinses, and dried to provide <b>72</b> (2.62 g, 74.7%)</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 7.12 (s, 1H); 6.96 (s, 2H); 4.59 (s, 4H).</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Step 6—Synthesis of Intermediate Compound <b>73</b>  (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>)</h4><div class="NLM_p">Compound <b>72</b> (2.62 g, 14.62 mmol) was partially dissolved in DCM (400 mL) under an atmosphere of nitrogen and cooled in an ice bath. Triphenylphosphine dibromide (14.0 g, 33.2 mmol) was added portionwise over 5 min. The resulting mixture was stirred at room temperature for 90 min and the reaction eventually became a solution. TLC (60% EtOAc/hexanes) analysis at this point indicated that the reaction was not complete. An additional 600 mg of triphenylphosphine dibromide was added and the reaction was stirred for an additional 20 min. The reaction was filtered through a 120 g silica gel column, rinsing with 350 mL DCM. The filtrate was concentrated <i>in vacuo</i> and dried to provide <b>73</b> as a crystalline tan solid, which was stored in a freezer (3.78 g, 85%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.18 (s, 1 H), 6.98 (s, 2 H), 4.43 (s, 4 H).</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Step 7—Synthesis of Compound <b>75</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>)</h4><div class="NLM_p">The linear peptide intermediate <b>74</b> was prepared as described above in the General Procedure using Method B, steps 1 and 2 incorporating the above synthesized amino acid <b>70</b> at the appropriate point in the peptide sequence. After SPPS and deprotection/cleavage as described in Method B, steps 1 and 2, the crude linear peptide <b>74</b> (prepared on 0.25 mmol scale) was cyclized as described in Method B, step 3, using 103 mg (0.34 mmol) of <b>73</b> (prepared above). The crude product after lyophilization was used as isolated in the next reaction.</div><div class="NLM_p last">The crude product from the above was dissolved in a solution of 100 mL of <i>t</i>-BuOH/50 mL water and the soln. bubbled with N<sub>2</sub>. To the solution was added 26.5 mg (0.05 mmol) of tris[(1-benzyl-1<i>H</i>-1,2,3-triazol-4-yl)methyl]amine and 49.5 mg (0.25 mmol) of sodium ascorbate, followed by 18.6 mg (0.05 mmol) of tetrakis(acetonitrile)copper(I) hexafluorophosphate. The reaction was stirred at room temperature, but there was no reaction observed after 90 min. Additional portions of the three reagents were added again, and the reaction was heated at 50 °C. The reaction was complete after 90 min. The precipitate was filtered off and washed with <i>t</i>-BuOH and water. The resulting filtrate was concentrated <i>in vacuo</i> and resuspended in a mixture of acetonitrile and methanol. The crude material was purified by RP prep LC using a Vydac protein & peptide C18, 250 × 10 mm, wide pore, with a 20–60% A gradient over 50 min; mobile phases: A = 0.1% TFA in water; B = 0.1% TFA in acetonitrile. The product fractions were combined and lyophilized to give the desired product (17.5 mg) <b>75</b> as a white amorphous powder. LC/MS purity = >95.5%; LC/MS calcd 1537.81; found, 1538.0 [M+H]<sup>+</sup>.</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> General Synthesis of Bicyclic Lactam Analogues (Example for Compound <b>79</b>)</h3><div class="NLM_p">Crude linear peptide (0.25 mmol) was redissolved in a solution of degassed acetonitrile (150 mL) and degassed 20 mM aqueous solution of ammonium carbonate (63 mL, 1.26 mmol). To this solution was added a degassed solution of 1,3-di(bromomethyl)benzene (78 mg, 0.29 mmol) in ACN (5 mL total with rinse). The solution was stirred under an atmosphere of nitrogen for 18 h. The reaction was filtered and then concentrated <i>in vacuo</i> to give the crude cyclized peptide. The residue was redissolved in 19:1 DMSO–water and purified using gradient elution on the reverse phase (50 × 250 mm Sunfire Prep C18; 25–75% CH<sub>3</sub>CN/water w/0.1% TFA modifier over 30 min). The fractions were then concentrated <i>in vacuo</i> to provide the mono-cyclized peptide. To a solution of AOP (19.9 mg, 45 μmol) and DIEA (30 μL, 172 μmol) in DCM (100 mL) was added a solution of the above mono-cyclized peptide (50 mg, 34 μmol) in DMF (6 mL) over 60 min via a syringe pump. The syringe was rinsed with DMF (1.5 mL). The reaction was stirred overnight at room temperature. The reaction was then concentrated <i>in vacuo</i> to remove the DCM and the residue was purified using gradient elution on the reverse phase (30 × 150 mm Sunfire Prep OBD C18; 30–75% CH<sub>3</sub>CN/water w/0.1% TFA modifier over 20 min). The fractions were lyophilized to provide 21 mg of compound <b>79</b> as a white powder. Dimeric lactam product was also recovered as a byproduct.</div><div class="NLM_p last">LCMS: 95.6% purity. RT: 6.23 min (@215 nM on 5–95% ACN–water (0.1% TFA) over 10 min on Sunfire C18) anal. calcd for C<sub>67</sub>H<sub>84</sub>FN<sub>11</sub>O<sub>12</sub>S<sub>2</sub>, 1317.6; found, 1318.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Stability and Metabolite ID Studies</h3><div class="NLM_p">The metabolism of various compounds was studied upon incubation with fully activated porcine gastric pepsin, bovine pancreatic trypsin, porcine pancreatic type IV elastase, and bovine pancreatic α-chymotrypsin. All reported data is derived from a single determination.</div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Pepsin</h4><div class="NLM_p last">A 100 μg/mL solution of porcine pepsin was prepared by dissolving lyophilized enzyme in 10 mM aqueous hydrochloric acid (pH 2.0). Incubation mixtures were prepared in duplicate in a 96-well propropylene plate by adding 5.0 μL aliquots of 100 μM substrate in DMSO or DMSO only (no-substrate control) to 45 μL of the 100 μg/mL solution of porcine pepsin in 10 mM aqueous hydrochloric acid (pH 2.0), which had been prewarmed at 37 °C. The reactions were terminated after an incubation time at 37 °C of 60 min by the addition of 2 volumes (100 μL) of acetonitrile containing 4.5 μM of an internal standard peptide. True time zero controls were prepared by adding the stopping solution prior to the addition of the substrate. The plates were then vortex-mixed and centrifuged at 2250<i>g</i> for 10 min. A 50 μL volume of the supernatant from each well was added to 300 μL of 50/50 acetonitrile/20% aqueous formic acid. The resulting supernatants were subjected to LC–MS/MS analysis.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Trypsin</h4><div class="NLM_p last">A 100 μg/mL solution of bovine pancreatic trypsin was prepared by dissolving lyophilized bovine pancreatic trypsin enzyme (in 100 mM sodium phosphate buffer, pH 7.40). Incubation mixtures were prepared in duplicate in a 96-well polypropylene plate by adding 5.0 μL aliquots of 100 μM substrate in DMSO or DMSO only (no-substrate control) to 45 μL of the 100 μg/mL solution of bovine pancreatic trypsin in 100 mM sodium phosphate buffer (pH 7.40), which had been prewarmed at 37 °C. The reactions were terminated after an incubation time of 1 and 6 h at 37 °C by the addition of 2 volumes (100 μL) of acetonitrile (with formic acid) containing an internal standard peptide. True time zero controls were prepared by adding the stopping solution prior to the addition of the substrate. The plates were then vortex-mixed and centrifuged at 2250<i>g</i> for 10 min. A 50 μL volume of the supernatant from each well was added to 300 μL of 50/50 acetonitrile/20% aqueous formic acid. The resulting supernatants were subjected to LC–MS/MS analysis.</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> α-Chymotrypsin</h4><div class="NLM_p last">A 100 μg/mL solution of bovine pancreatic α-chymotrypsin was prepared by dissolving lyophilized enzyme in 100 mM sodium phosphate buffer, pH 7.40. Incubation mixtures were prepared in duplicate in a 96-well polypropylene plate by adding 5.0 μL aliquots of 100 μM substrate in DMSO or DMSO only (no-substrate control) to 45 μL of the 100 μg/mL solution of bovine pancreatic α-chymotrypsin in 100 mM sodium phosphate buffer (pH 7.40), which had been prewarmed at 37 °C. The reactions were terminated after an incubation time at 37 °C of 1 and 6 h by the addition of 2 volumes(100 μL) of acetonitrile (with formic acid) containing 4.5 μM of an internal standard peptide. True time zero controls were prepared by adding the stopping solution prior to the addition of the substrate. The plates were then vortex-mixed and centrifuged at 2250<i>g</i> for 10 min. A 50 μL volume of the supernatant from each well was added to 300 μL of 50/50 acetonitrile/20% aqueous formic acid. The resulting supernatants were subjected to LC–MS/MS analysis.</div></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Type IV Pancreatic Elastase</h4><div class="NLM_p last">A 100 μg/mL solution of porcine type IV pancreatic elastase was prepared by dissolving lyophilized enzyme in 100 mM sodium phosphate buffer, pH 7.40. Incubation mixtures were prepared in duplicate in a 96-well polypropylene plate by adding 5.0 μL aliquots of 100 μM substrate in DMSO or DMSO only (no-substrate control) to 45 μL of the 100 μg/mL solution of porcine type IV pancreatic elastase in 100 mM sodium phosphate buffer (pH 7.40), which had been prewarmed at 37 °C. Reactions were terminated after an incubation time at 37 °C of 1 and 6 h by the addition of 2 volumes (100 μL) of acetonitrile (with formic acid) containing 4.5 μM of an internal standard peptide. True time zero controls were prepared by adding the stopping solution prior to the addition of the substrate. The plates were then vortex-mixed and centrifuged at 2250<i>g</i> for 10 min. A 50 μL volume of the supernatant from each well was added to 300 μL of 50/50 acetonitrile/20% aqueous formic acid. The resulting supernatants were subjected to LC–MS/MS analysis.</div></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Metabolic Stability with Mouse and Cynomolgus Monkey Whole Blood</h3><div class="NLM_p last">The metabolism of peptide candidates was studied upon incubation with C57BL/6 mouse and cynomolgus monkey whole blood. Incubation mixtures were prepared in duplicate in a 96-well propropylene plate by adding 5.0 μL aliquots of either 100 μM substrate peptide in DMSO or DMSO only (no-substrate control) to 45 μL of whole blood, which had been prewarmed at 37 °C. Incubations were terminated after an incubation time at 37 °C of 1, 2, 4, and 6 h by the addition of 2 volumes (100 μL) of acetonitrile containing 0.1% formic acid and 4.5 μM of an internal standard peptide. True time zero controls were prepared by adding the stopping solution prior to the addition of substrate. Plates were then vortex-mixed and centrifuged at 2250<i>g</i> for 10 min. A 30 μL volume of the supernatant from each well was added to 180 μL of 50/50 acetonitrile/20% aqueous formic acid. The resulting supernatants were subjected to LC–MS/MS analysis.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Mouse Protein Binding</h3><div class="NLM_p">The binding of test molecules to plasma proteins was determined by equilibrium dialysis using the HT Dialysis apparatus (HT Dialysis LLC, Gales Ferry, CT). The peptide was added to mouse plasma at a final concentration of 2.5 μM. Phosphate-buffered saline (100 mM, pH 7.4) was added to one side of the dialysis plate and an equal volume of plasma containing peptide was added to the other side of the plate in replicates of six for each drug concentration tested. The dialysis plates were incubated at 37 °C under 5% CO<sub>2</sub> for 4 h. Following the dialysis, 50 μL of plasma and buffer were removed from each dialysis well, a matrix match was added to bring the pre-extracted samples to 50% plasma and 50% buffer, and the analyte was extracted by the addition of 250 μL acetonitrile containing 200 nM of the internal standard mixture (labetalol, imipramine, and diclofenac). The samples were vortex-mixed and centrifuged for 5 min at 3200<i>g</i>, and the resulting supernatant fractions were analyzed by LC–MS/MS. The below calculation was used to determine percent bound<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_m001.gif" alt="" id="_i65" /></img></span>where <i>C</i><sub>b</sub> is the postincubation concentration of peptide in the buffer side and <i>C</i><sub>p</sub> is the postincubation concentration in the plasma side.</div><div class="NLM_p last">LC–MS/MS analysis was performed on a LC–MS/MS system equipped with an LEAP autosampler (Leap Technologies, Morrisville, NC) and a Thermo Scientific Dionex Ultimate 3000 RS Pump (Thermo Fisher Scientific, Waltham, MA) interfaced to a Sciex 4500 mass spectrometer (Sciex, Framingham, MA) utilizing an electrospray source operating in the positive ion mode using multiple reaction monitoring. Chromatographic separation was achieved on an Acquity HSS T3 UPLC column 2.1 mm × 50 mm, 1.8 μm (Waters Cat# 186003538, Milford, MA), using a mobile phase consisting of 0.1% formic acid in water (solvent A) and 0.1% formic acid in acetonitrile (solvent B) at a flow rate of 0.75 mL/min. The chromatography was run using a gradient elution as follows: the column was equilibrated at 5% solvent B; after sample injection, the gradient was held at 5% solvent B for 0.5 min. Solvent B was then increased to 95% over a 1.5 min period. Solvent B was then maintained at 95% for 0.42 min before it was returned to the initial conditions for 0.83 min. The total run time was 3.25 min.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Crystallography</h3><div class="NLM_p last">PCSK9 lacking the C-terminal domain (PCSK9-ΔCRD) was produced and purified as described.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The final S60 gel filtration was performed in 20 mM Tris pH 8.0, 300 mM NaCl, 100 mM CaCl<sub>2</sub>, and 5% glycerol. Fractions containing PCSK9-ΔCRD were combined, concentrated to 12 mg/mL, and frozen in 50 mL aliquots till further use. Peptides were dissolved in DMSO at 100 mM and added with a 2-fold molar excess to PCSK9-ΔCRD. Complexes were incubated for 1 h on ice and screened against commercially available crystallization screens. Suitable crystals appeared with 20% PEG3350, 200 mM CaCl<sub>2</sub>, and 100 mM MES at pH 6 as the precipitant. Crystals were harvested and cryoprotected with an additional 10% glycerol. X-ray diffraction data were collected at IMCA beamline ID17 and processed using the Globalphasing<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and CCP4<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> packages. Structures were solved by molecular replacement using <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NMX">4NMX</a><a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and refined using BUSTER<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and Coot.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Peptides were modeled using standard amino acid residue definitions or restrains generated using Flynn.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Coordinates and experimental data are available from the Protein Data Bank with the accession codes: compound <b>30</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIB">6XIB</a>; compound <b>40</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIC">6XIC</a>; compound <b>51</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XID">6XID</a>; compound <b>78</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIE">6XIE</a>; and compound <b>84</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIF">6XIF</a>. Authors will release the atomic coordinates and experimental data upon article publication.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> Rat Mast Cell Degranulation Assay</h3><div class="NLM_p last">Isolation of rat peritoneal lavage was conducted as described in the literature. In brief, Wistar Han rats (males or females) were anesthetized with xylene. About 10 mL of Tyrode buffer (12 mM NaHCO<sub>3</sub>, 127 mM NaCl, 5 mM KCl, 0.5 mM NaH<sub>2</sub>PO<sub>4</sub>, 1 mM MgCl<sub>2</sub>, 5 mM glucose, and 10 mM HEPES, pH 7.4) was injected into the peritoneal cavity, followed by gentle abdominal massage for 30 s. Peritoneal lavage was then collected and centrifuged at 350<i>g</i> for 10 min. After discarding the supernatant, the cells are resuspended in Tyrode buffer to achieve a final concentration of 1 × 10<sup>6</sup> cells/mL. For peptide treatment, 100 μL of freshly isolated rat peritoneal cells was added to a tissue-culture treated 96-well plate. Cells were incubated with either positive control (48/80; used to confirm assay responsiveness)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> or a titrating concentration of various peptides for 30 min at 37 °C. After incubation, the plate was centrifuged at 350<i>g</i> for 2 min at room temperature. The supernatants were collected for histamine measurement and cell pellets are resuspended in Tyrode buffer for cell viability analysis. Histamine and cell viability were measured using the commercially available kits, following vendor’s instructions. The amount of histamine in the supernatant was presented as ng/mL, and cell viability was presented as percent viable cells compared to vehicle-treated cells. Data were reported in for two determinations, both of which were in close agreement in all cases.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> PK Studies</h3><div class="NLM_p">All PK studies in preclinical species were conducted according to the highest ethical standards in accordance with and using procedures approved by the Institutional Animal Care and Use Committee of Merck and Co., Inc., West Point, PA, USA.</div><div class="NLM_p">Male C57BL/6 mice and male Wistar Hannover rats (<i>n</i> = 2–3 in each arm) were used for IV PK studies. Mice and rats were allowed to fast overnight prior to dosing and fed 2 h postdose. Dual jugular vein and carotid artery surgically cannulated mice or rats were used for dose administration and blood sampling, respectively. Blood samples were collected using an automated blood sampling system (Instech, Plymouth Meeting, PA) after IV administration. All blood samples were collected in EDTA-coated tubes at predose, 0.08, 0.25, 0.5, 1, 2, 4, 7, 12, 18, and 24 h following IV dose administration. Rat blood samples were centrifuged, and the supernatant plasma samples were stored at −70 °C until bioanalysis. In mouse studies, whole blood was diluted with citrate anticoagulant and whole blood PK rather than plasma PK was measured.</div><div class="NLM_p">Male cynomolgus monkeys (<i>n</i> = 2 per group) weighing 5–10 kg were dosed IV. Monkeys were allowed to fast prior to dosing and fed 2–4 h postdose. IV dose was administered as a bolus via the cephalic or saphenous vein. Blood samples were collected serially from the saphenous, femoral, or cephalic veins in EDTA-coated tubes at predose, 0.08, 0.25, 0.5, 1, 2, 4, 6, 24, 48, and 72 h. All blood samples were centrifuged and the supernatant plasma samples were stored at 70 °C until bioanalysis.</div><div class="NLM_p">Concentrations in mouse whole blood, rat plasma, and monkey plasma were determined by LC–MS/MS assays following a protein precipitation step. Aliquots (50 μL) of plasma were precipitated by addition of 200 μL of methanol/acetonitrile (80:20) containing a related macrocyclic peptide as an internal standard, followed by centrifugation at 4000 rpm for 5 min. A 200 μL aliquot of the supernatant was transferred into a 96-well plate injecting 2 μL of each sample for analysis. Tandem LC–MS analysis was performed on a Waters Acquity UPLC system interfaced to an AB Sciex QTRAP-5500 or API6500 mass spectrometer utilizing the turbo ion spray interface (AB Sciex, Framingham, MA). Separation was achieved on a Waters Acquity HSS T3 C18 UPLC column (50 × 2.1 mm, 1.8 μm) using a mobile phase consisting of water containing 0.1% formic acid (solvent A) and acetonitrile containing 0.1% formic acid (solvent B) at a flow rate of 0.75 mL/min. Quantification was accomplished by monitoring the MS/MS transition of each peptide. The concentration was determined across a linear concentration range of compound matching standards (standard curve) in the appropriate matrix.</div><div class="NLM_p last">PK parameters were calculated using noncompartmental methods in Watson v.7.4. The area under the plasma concentration–time curve (AUC<sub>0–<i>t</i></sub>) was calculated from the first time point up to the last time point using the linear trapezoidal method for ascending concentrations and the log trapezoidal method for descending concentrations. The concentration at 0 h after IV administration was back-extrapolated using the first three time points (0.03, 0.13, and 0.25 h). The remaining area under the plasma concentration–time curve (AUC <i>t</i>–∞) was estimated by dividing the observed concentration at the last time point by the elimination rate constant. This value was added to AUC<sub>0–<i>t</i></sub> to estimate the AUC<sub>0–∞</sub>. The IV plasma clearance was calculated by dividing the dose by AUC<sub>0–∞</sub>. The terminal half-life of elimination was determined by unweighted linear regression analysis of the log-transformed data. The volume of distribution at steady state (<i>V</i><sub>dss</sub>) was obtained from the product of plasma clearance and mean residence time (determined by dividing the extrapolated area under the first moment curve by extrapolated the area under the curve). The maximum plasma concentration (<i>C</i><sub>max</sub>) and the time at which maximum concentration occurred (<i>T</i><sub>max</sub>) were obtained by graphical inspection of the plasma concentration–time data. For all <i>n</i> = 3, the error is characterized as standard deviation. For <i>n</i> = 2, the error is characterized as the difference between the mean and the individual determinations.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01084" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37514" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37514" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01084?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01084</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Representative LC traces for key molecules in the paper, sample IC<sub>50</sub> curves for LDLR-FRET and Alexa-FRET assay control peptide, and raw concentration versus time profiles for molecules tested in Mouse IV PK 3D PDB files for each of the new crystal structures have also been uploaded; PDB codes for the crystal structures shown in the paper are as follows: compound <b>30</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIB">6XIB</a>; compound <b>40</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIC">6XIC</a>; compound <b>51</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XID">6XID</a>; compound <b>78</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIE">6XIE</a>; and compound <b>84</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIF">6XIF</a>; and authors will release the atomic coordinates and experimental data upon article publication (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">6xib (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">6xic (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">6xid (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">6xie (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">6xif (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_006.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_001.pdf">jm0c01084_si_001.pdf (616.41 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_002.pdb">jm0c01084_si_002.pdb (274.18 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_003.pdb">jm0c01084_si_003.pdb (279.28 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_004.pdb">jm0c01084_si_004.pdb (276.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_005.pdb">jm0c01084_si_005.pdb (275.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_006.pdb">jm0c01084_si_006.pdb (267.71 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01084" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31708" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31708" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas J. Tucker</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, Merck & Company,
Inc., 770 Sumneytown
Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5616-0802" title="Orcid link">http://orcid.org/0000-0001-5616-0802</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a2d6cdcffdd6d7c1c9c7d0e2cfc7d0c1c98cc1cdcf"><span class="__cf_email__" data-cfemail="d7a3b8ba88a3a2b4bcb2a597bab2a5b4bcf9b4b8ba">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Candice Alleyne</span> - <span class="hlFld-Affiliation affiliation">Discovery
Pharmaceutical Sciences, Merck & Company,
Inc., 2000 Galloping
Hill Road, Kenilworth, New
Jersey 07033, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rupesh P. Amin</span> - <span class="hlFld-Affiliation affiliation">Safety
Assessment, Merck & Comapny, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bhavana Bhatt</span> - <span class="hlFld-Affiliation affiliation">Safety
Assessment, Merck & Comapny, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabetta Bianchi</span> - <span class="hlFld-Affiliation affiliation">IRBM
S.p.A., Via Pontina km
30600, Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Craig Blain</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Boyer</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span>; 
    <span>Present Address:
                        Department of Medicinal Chemistry, Merck and Co, Inc., 33 Avenue E Louis Pasteur, Boston, MA 02115 USA</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danila Branca</span> - <span class="hlFld-Affiliation affiliation">IRBM
S.p.A., Via Pontina km
30600, Pomezia, Rome 00071, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark W. Embrey</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, Merck & Company,
Inc., 770 Sumneytown
Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sookhee N. Ha</span> - <span class="hlFld-Affiliation affiliation">Modeling
and Informatics, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kelli Jette</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas G. Johns</span> - <span class="hlFld-Affiliation affiliation">Discovery
Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angela D. Kerekes</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, Merck & Company,
Inc., 2000 Galloping
Hill Road, Kenilworth, New
Jersey 07033, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenneth A. Koeplinger</span> - <span class="hlFld-Affiliation affiliation">Pharmacokinetics
Pharmacodynamics and Drug Metabolism, Merck
& Company, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Derek LaPlaca</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span>; 
    <span>Present Address:
                        Medicinal Chemistry Department, Syros Pharmaceutical Inc., 35 Cambridge Park Drive, Cambridge, MA, USA 02139-4815</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8513-2445" title="Orcid link">http://orcid.org/0000-0002-8513-2445</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nianyu Li</span> - <span class="hlFld-Affiliation affiliation">Safety
Assessment, Merck & Comapny, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Beth Murphy</span> - <span class="hlFld-Affiliation affiliation">Discovery
Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Orth</span> - <span class="hlFld-Affiliation affiliation">Structural
Sciences, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alonso Ricardo</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott Salowe</span> - <span class="hlFld-Affiliation affiliation">Discovery
Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span>; 
    <span>Present Address:
                        Venenum Biodesign LLC, 8 Black Forest Road, Hamilton, NJ, USA 08691</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kathleen Seyb</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aurash Shahripour</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, Merck & Company,
Inc., 2000 Galloping
Hill Road, Kenilworth, New
Jersey 07033, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph R. Stringer</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yili Sun</span> - <span class="hlFld-Affiliation affiliation">UCB Ra
Pharma, 87 Cambridge
Park Drive, Cambridge, Massachusetts 02140, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rodger Tracy</span> - <span class="hlFld-Affiliation affiliation">Pharmacokinetics
Pharmacodynamics and Drug Metabolism, Merck
& Company, Inc., 770 Sumneytown Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chengwei Wu</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, Merck & Company,
Inc., 770 Sumneytown
Pike, P.O. Box 4, West Point, Pennsylvania 19486, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yusheng Xiong</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, Merck & Company,
Inc., 2000 Galloping
Hill Road, Kenilworth, New
Jersey 07033, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hyewon Youm</span> - <span class="hlFld-Affiliation affiliation">Departments
of Medicinal Chemistry, Merck & Company,
Inc., 2000 Galloping
Hill Road, Kenilworth, New
Jersey 07033, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hratch J. Zokian</span> - <span class="hlFld-Affiliation affiliation">Discovery
Biology, Merck & Company, Inc., 2000 Galloping Hill Road, Kenilworth, New Jersey 07033, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i73">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78710" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78710" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors acknowledge Abbas Walji and H. Blair Wood for helpful discussions and editing related to the manuscript and scientific content. The authors are also grateful to Jennifer Johnston for her excellent graphics work in support of the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ACN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">AOP</td><td class="NLM_def"><p class="first last">tris(dimethylamino)(3<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridin-3-yloxy)phosphorus hexafluorophosphate</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>t</i>-butoxycarbonyl</p></td></tr><tr><td class="NLM_term">DBX</td><td class="NLM_def"><p class="first last">di-(1,3-bromomethyl)benzene</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIC</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylcarbodiimide</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last">diisopropylethyl amine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DODT</td><td class="NLM_def"><p class="first last">3,6-dioxa-1,8-octanedithiol</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">5F-Trp</td><td class="NLM_def"><p class="first last">5-fluoro-tryptophan</p></td></tr><tr><td class="NLM_term">Fmoc-OSu</td><td class="NLM_def"><p class="first last">Fmoc <i>N</i>-hydroxysuccinimide ester</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">IV</td><td class="NLM_def"><p class="first last">intravenous</p></td></tr><tr><td class="NLM_term">ivDde</td><td class="NLM_def"><p class="first last">[l-(4,4-dimethyl-2,6-dioxocyclohex-1-ylidene)-ethyl]protecting group</p></td></tr><tr><td class="NLM_term">LC–MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">LDL-C</td><td class="NLM_def"><p class="first last">LDL-cholesterol</p></td></tr><tr><td class="NLM_term">LDLR</td><td class="NLM_def"><p class="first last">LDL receptor</p></td></tr><tr><td class="NLM_term">mAb</td><td class="NLM_def"><p class="first last">monoclonal antibody</p></td></tr><tr><td class="NLM_term">Met ID</td><td class="NLM_def"><p class="first last">metabolite identification</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl pyrrolidinone</p></td></tr><tr><td class="NLM_term">PCSK9</td><td class="NLM_def"><p class="first last">proprotein convertase subtilisin-like/kexin type 9</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">RP-HPLC</td><td class="NLM_def"><p class="first last">reverse phase high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SPPS</td><td class="NLM_def"><p class="first last">solid phase peptide synthesis</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>tert</i>-butanol</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">TMS-Cl</td><td class="NLM_def"><p class="first last">trimethylchlorosilane</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i75">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sediah, N.G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chretien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mbikay, M.</span></span> <span> </span><span class="NLM_article-title">PCSK9: A Key Modulator of Cardiovascular Health</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1036</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.114.301621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1161%2FCIRCRESAHA.114.301621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=24625727" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=1022-1036&author=N.G.+Sediahauthor=Z.+Awanauthor=M.+Chretienauthor=M.+Mbikay&title=PCSK9%3A+A+Key+Modulator+of+Cardiovascular+Health&doi=10.1161%2FCIRCRESAHA.114.301621"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.114.301621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.114.301621%26sid%3Dliteratum%253Aachs%26aulast%3DSediah%26aufirst%3DN.G.%26aulast%3DAwan%26aufirst%3DZ.%26aulast%3DChretien%26aufirst%3DM.%26aulast%3DMbikay%26aufirst%3DM.%26atitle%3DPCSK9%253A%2520A%2520Key%2520Modulator%2520of%2520Cardiovascular%2520Health%26jtitle%3DCirc.%2520Res.%26date%3D2014%26volume%3D114%26spage%3D1022%26epage%3D1036%26doi%3D10.1161%2FCIRCRESAHA.114.301621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alings, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittner, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelberg, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanotin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jukema, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabor Kiss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecorps, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahaffey, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moryusef, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pordy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tricoci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeiher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steg, G.</span></span> <span> </span><span class="NLM_article-title">Alcirocumab Reduces Total Nonfatal Cardiovascular and Fatal Events</span>. <i>J. Am. Coll. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1016/j.jacc.2018.10.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.jacc.2018.10.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=389-396&author=M.+Szarekauthor=H.+D.+Whiteauthor=G.+G.+Schwartzauthor=M.+Alingsauthor=D.+L.+Bhattauthor=V.+A.+Bittnerauthor=C.+Chiangauthor=R.+Diazauthor=J.+M.+Edelbergauthor=S.+G.+Goodmanauthor=C.+Hanotinauthor=R.+A.+Harringtonauthor=J.+W.+Jukemaauthor=T.+Kimuraauthor=R.+Gabor+Kissauthor=G.+Lecorpsauthor=K.+W.+Mahaffeyauthor=A.+Moryusefauthor=R.+Pordyauthor=M.+T.+Roeauthor=P.+Tricociauthor=D.+Xavierauthor=A.+M.+Zeiherauthor=G.+Steg&title=Alcirocumab+Reduces+Total+Nonfatal+Cardiovascular+and+Fatal+Events&doi=10.1016%2Fj.jacc.2018.10.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.jacc.2018.10.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacc.2018.10.039%26sid%3Dliteratum%253Aachs%26aulast%3DSzarek%26aufirst%3DM.%26aulast%3DWhite%26aufirst%3DH.%2BD.%26aulast%3DSchwartz%26aufirst%3DG.%2BG.%26aulast%3DAlings%26aufirst%3DM.%26aulast%3DBhatt%26aufirst%3DD.%2BL.%26aulast%3DBittner%26aufirst%3DV.%2BA.%26aulast%3DChiang%26aufirst%3DC.%26aulast%3DDiaz%26aufirst%3DR.%26aulast%3DEdelberg%26aufirst%3DJ.%2BM.%26aulast%3DGoodman%26aufirst%3DS.%2BG.%26aulast%3DHanotin%26aufirst%3DC.%26aulast%3DHarrington%26aufirst%3DR.%2BA.%26aulast%3DJukema%26aufirst%3DJ.%2BW.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DGabor%2BKiss%26aufirst%3DR.%26aulast%3DLecorps%26aufirst%3DG.%26aulast%3DMahaffey%26aufirst%3DK.%2BW.%26aulast%3DMoryusef%26aufirst%3DA.%26aulast%3DPordy%26aufirst%3DR.%26aulast%3DRoe%26aufirst%3DM.%2BT.%26aulast%3DTricoci%26aufirst%3DP.%26aulast%3DXavier%26aufirst%3DD.%26aulast%3DZeiher%26aufirst%3DA.%2BM.%26aulast%3DSteg%26aufirst%3DG.%26atitle%3DAlcirocumab%2520Reduces%2520Total%2520Nonfatal%2520Cardiovascular%2520and%2520Fatal%2520Events%26jtitle%3DJ.%2520Am.%2520Coll.%2520Cardiol.%26date%3D2019%26volume%3D73%26spage%3D389%26epage%3D396%26doi%3D10.1016%2Fj.jacc.2018.10.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabatine, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giugliano, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keech, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honarpour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiviott, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuder, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sever, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, T. R.</span></span> <span> </span><span class="NLM_article-title">Evolocumab and Clinical Outcomes in Patients with Cardiovascular Disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>376</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1722</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1615664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1056%2FNEJMoa1615664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=28304224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOns7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2017&pages=1713-1722&author=M.+S.+Sabatineauthor=R.+P.+Giuglianoauthor=A.+C.+Keechauthor=N.+Honarpourauthor=S.+D.+Wiviottauthor=S.+A.+Murphyauthor=J.+F.+Kuderauthor=H.+Wangauthor=T.+Liuauthor=S.+M.+Wassermanauthor=P.+S.+Severauthor=T.+R.+Pedersen&title=Evolocumab+and+Clinical+Outcomes+in+Patients+with+Cardiovascular+Disease&doi=10.1056%2Fnejmoa1615664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Evolocumab and clinical outcomes in patients with cardiovascular disease</span></div><div class="casAuthors">Sabatine, Marc S.; Giugliano, Robert P.; Keech, Anthony C.; Honarpour, Narimon; Wiviott, Stephen D.; Murphy, Sabina A.; Kuder, Julia F.; Wang, Huei; Liu, Thomas; Wasserman, Scott M.; Sever, Peter S.; Pedersen, Terje R.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">376</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1713-1722</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Evolocumab is a monoclonal antibody that inhibits proprotein convertase subtilisin-kexin type 9 (PCSK9) and lowers low-d. lipoprotein (LDL) cholesterol levels by approx. 60%.  Whether it prevents cardiovascular events is uncertain.  We conducted a randomized, double-blind, placebo-controlled trial involving 27,564 patients with atherosclerotic cardiovascular disease and LDL cholesterol levels of 70 mg per dL (1.8 mmol per L) or higher who were receiving statin therapy.  Patients were randomly assigned to receive evolocumab (either 140 mg every 2 wk or 420 mg monthly) or matching placebo as s.c. injections.  The primary efficacy end point was the composite of cardiovascular death, myocardial infarction, stroke, hospitalization for unstable angina, or coronary revascularization.  The key secondary efficacy end point was the composite of cardiovascular death, myocardial infarction, or stroke.  The median duration of follow-up was 2.2 years.  At 48 wk, the least-squares mean percentage redn. in LDL cholesterol levels with evolocumab, as compared with placebo, was 59%, from a median baseline value of 92 mg per dL (2.4 mmol per L) to 30 mg per dL (0.78 mmol per L) (P<0.001).  Relative to placebo, evolocumab treatment significantly reduced the risk of the primary end point (1344 patients [9.8%] vs. 1563 patients [11.3%]; hazard ratio, 0.85; 95% confidence interval [CI], 0.79 to 0.92; P<0.001) and the key secondary end point (816 [5.9%] vs. 1013 [7.4%]; hazard ratio, 0.80; 95% CI, 0.73 to 0.88; P<0.001).  The results were consistent across key subgroups, including the subgroup of patients in the lowest quartile for baseline LDL cholesterol levels (median, 74 mg per dL [1.9 mmol per L]).  There was no significant difference between the study groups with regard to adverse events (including new-onset diabetes and neurocognitive events), with the exception of injection-site reactions, which were more common with evolocumab (2.1% vs. 1.6%).  In our trial, inhibition of PCSK9 with evolocumab on a background of statin therapy lowered LDL cholesterol levels to a median of 30 mg per dL (0.78 mmol per L) and reduced the risk of cardiovascular events.  These findings show that patients with atherosclerotic cardiovascular disease benefit from lowering of LDL cholesterol levels below current targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBM3VQXU81RLVg90H21EOLACvtfcHk0ljULRaoTQBvTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOns7nM&md5=8995d48576066c8b9d1cfb3446de270e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1615664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1615664%26sid%3Dliteratum%253Aachs%26aulast%3DSabatine%26aufirst%3DM.%2BS.%26aulast%3DGiugliano%26aufirst%3DR.%2BP.%26aulast%3DKeech%26aufirst%3DA.%2BC.%26aulast%3DHonarpour%26aufirst%3DN.%26aulast%3DWiviott%26aufirst%3DS.%2BD.%26aulast%3DMurphy%26aufirst%3DS.%2BA.%26aulast%3DKuder%26aufirst%3DJ.%2BF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DWasserman%26aufirst%3DS.%2BM.%26aulast%3DSever%26aufirst%3DP.%2BS.%26aulast%3DPedersen%26aufirst%3DT.%2BR.%26atitle%3DEvolocumab%2520and%2520Clinical%2520Outcomes%2520in%2520Patients%2520with%2520Cardiovascular%2520Disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2017%26volume%3D376%26spage%3D1713%26epage%3D1722%26doi%3D10.1056%2Fnejmoa1615664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nasir, K.</span></span> <span> </span><span class="NLM_article-title">Just Price for PCSK9 Inhibitors: No Less, No More</span>. <i>J. Am. Heart Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e010884</span> <span class="refDoi"> DOI: 10.1161/jaha.118.010884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1161%2FJAHA.118.010884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30571392" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=K.+Nasir&title=Just+Price+for+PCSK9+Inhibitors%3A+No+Less%2C+No+More&doi=10.1161%2Fjaha.118.010884"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1161%2FJAHA.118.010884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FJAHA.118.010884%26sid%3Dliteratum%253Aachs%26aulast%3DNasir%26aufirst%3DK.%26atitle%3DJust%2520Price%2520for%2520PCSK9%2520Inhibitors%253A%2520No%2520Less%252C%2520No%2520More%26jtitle%3DJ.%2520Am.%2520Heart%2520Assoc.%26date%3D2018%26volume%3D7%26doi%3D10.1161%2Fjaha.118.010884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krähenbühl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavik-Mezzour, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Eckardstein, A.</span></span> <span> </span><span class="NLM_article-title">Unmet Needs in LDL-C Lowering: When Statins Won’t Do!</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">1175</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1007/s40265-016-0613-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1007%2Fs40265-016-0613-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=27456066" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1175-1190&author=S.+Kr%C3%A4henb%C3%BChlauthor=I.+Pavik-Mezzourauthor=A.+von+Eckardstein&title=Unmet+Needs+in+LDL-C+Lowering%3A+When+Statins+Won%E2%80%99t+Do%21&doi=10.1007%2Fs40265-016-0613-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs40265-016-0613-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-016-0613-0%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4henb%25C3%25BChl%26aufirst%3DS.%26aulast%3DPavik-Mezzour%26aufirst%3DI.%26aulast%3Dvon%2BEckardstein%26aufirst%3DA.%26atitle%3DUnmet%2520Needs%2520in%2520LDL-C%2520Lowering%253A%2520When%2520Statins%2520Won%25E2%2580%2599t%2520Do%2521%26jtitle%3DDrugs%26date%3D2016%26volume%3D76%26spage%3D1175%26epage%3D1190%26doi%3D10.1007%2Fs40265-016-0613-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akyea, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, S. F.</span></span> <span> </span><span class="NLM_article-title">Sub-optimal Cholesterol Response to Initiation of Statins and Future Risk of Cardiovascular Disease</span>. <i>Heart</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">975</span>– <span class="NLM_lpage">981</span>, <span class="refDoi"> DOI: 10.1136/heartjnl-2018-314253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1136%2Fheartjnl-2018-314253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30988003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyqurfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2019&pages=975-981&author=R.+K.+Akyeaauthor=J.+Kaiauthor=N.+Qureshiauthor=B.+Iyenauthor=S.+F.+Weng&title=Sub-optimal+Cholesterol+Response+to+Initiation+of+Statins+and+Future+Risk+of+Cardiovascular+Disease&doi=10.1136%2Fheartjnl-2018-314253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Sub-optimal cholesterol response to initiation of statins and future risk of cardiovascular disease</span></div><div class="casAuthors">Akyea, Ralph Kwame; Kai, Joe; Qureshi, Nadeem; Iyen, Barbara; Weng, Stephen F.</div><div class="citationInfo"><span class="NLM_cas:title">Heart (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">975-981</span>CODEN:
                <span class="NLM_cas:coden">HEARFR</span>;
        ISSN:<span class="NLM_cas:issn">1355-6037</span>.
    
            (<span class="NLM_cas:orgname">BMJ</span>)
        </div><div class="casAbstract">Objective To assess low-d. lipoprotein cholesterol (LDL-C) response in patients after initiation of statins, and future risk of cardiovascular disease (CVD).  Methods Prospective cohort study of 165 411 primary care patients, from the UK Clin. Practice Research Datalink, who were free of CVD before statin initiation, and had at least one pre-treatment LDL-C within 12 mo before, and one post-treatment LDL-C within 24 mo after, statin initiation.  Based on current national guidelines, <40% redn. in baseline LDL-C within 24 mo was classified as a sub-optimal statin response.  Cox proportional regression and competing-risks survival regression models were used to det. adjusted hazard ratios (HRs) and sub-HRs for incident CVD outcomes for LDL-C response to statins.  Results 84 609 (51.2%) patients had a sub-optimal LDL-C response to initiated statin therapy within 24 mo.  During 1 077 299 person-years of follow-up (median follow-up 6.2 years), there were 22 798 CVD events (12 142 in sub-optimal responders and 10 656 in optimal responders).  In sub-optimal responders, compared with optimal responders, the HR for incident CVD was 1.17 (95% CI 1.13 to 1.20) and 1.22 (95% CI 1.19 to 1.25) after adjusting for age and baseline untreated LDL-C.  Considering competing risks resulted in lower but similar sub-HRs for both unadjusted (1.13, 95% CI 1.10 to 1.16) and adjusted (1.19, 95% CI 1.16 to 1.23) cumulative incidence function of CVD.  Conclusions Optimal lowering of LDL-C is not achieved within 2 years in over half of patients in the general population initiated on statin therapy, and these patients will experience significantly increased risk of future CVD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi4gmHe-gcU7Vg90H21EOLACvtfcHk0ljYTozMJZHUxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyqurfL&md5=bc9f498ce08e854acbb8fab25e63f722</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1136%2Fheartjnl-2018-314253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fheartjnl-2018-314253%26sid%3Dliteratum%253Aachs%26aulast%3DAkyea%26aufirst%3DR.%2BK.%26aulast%3DKai%26aufirst%3DJ.%26aulast%3DQureshi%26aufirst%3DN.%26aulast%3DIyen%26aufirst%3DB.%26aulast%3DWeng%26aufirst%3DS.%2BF.%26atitle%3DSub-optimal%2520Cholesterol%2520Response%2520to%2520Initiation%2520of%2520Statins%2520and%2520Future%2520Risk%2520of%2520Cardiovascular%2520Disease%26jtitle%3DHeart%26date%3D2019%26volume%3D105%26spage%3D975%26epage%3D981%26doi%3D10.1136%2Fheartjnl-2018-314253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, R. O.</span></span> <span> </span><span class="NLM_article-title">A Two-step Binding Model of PCSK9 Interaction with the Low-Density Lipoprotein Receptor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">5464</span>– <span class="NLM_lpage">5470</span>, <span class="refDoi"> DOI: 10.1074/jbc.m110.199042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1074%2Fjbc.M110.199042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=21149300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVCjsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=5464-5470&author=T.+Yamamotoauthor=C.+Luauthor=R.+O.+Ryan&title=A+Two-step+Binding+Model+of+PCSK9+Interaction+with+the+Low-Density+Lipoprotein+Receptor&doi=10.1074%2Fjbc.m110.199042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Two-step Binding Model of PCSK9 Interaction with the Low Density Lipoprotein Receptor</span></div><div class="casAuthors">Yamamoto, Taichi; Lu, Christine; Ryan, Robert O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5464-5470</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PCSK9 (proprotein convertase subtilisin-like/kexin type 9) is an emerging target for pharmaceutical intervention.  This multidomain protein interacts with the LDL receptor (LDLR), promoting receptor degrdn.  Insofar as PCSK9 inhibition induces a decrease in plasma cholesterol levels, understanding the nature of the binding interaction between PCSK9 and the LDLR is of crit. importance.  In this study, the ability of PCSK9 to compete with apoE3 N-terminal domain-contg. reconstituted HDL for receptor binding was examd.  Whereas full-length PCSK9 was an effective competitor, the N-terminal Pro-Cat domain (composed of the prodomain and catalytic domain) was not.  Surprisingly, the C-terminal domain (CT domain) of PCSK9 was able to compete.  Using a direct binding interaction assay, we show that the PCSK9 CT domain bound to the LDLR in a calcium-dependent manner and that co-incubation with the prodomain and catalytic domain had no effect on this binding.  To further characterize this interaction, two LDLR fragments, the classical ligand-binding domain (LBD) and the EGF precursor homol. domain, were expressed in stably transfected HEK 293 cells and isolated.  Binding assays showed that the PCSK9 CT domain bound to the LBD at pH 5.4.  Thus, CT domain interaction with the LBD of the LDLR at endosomal pH constitutes a second step in the PCSK9-mediated LDLR binding that leads to receptor degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo69xyoaQQ1A7Vg90H21EOLACvtfcHk0ljYTozMJZHUxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVCjsrY%253D&md5=62aa5b98989d592f72de4b2e7b77074b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.199042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.199042%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DRyan%26aufirst%3DR.%2BO.%26atitle%3DA%2520Two-step%2520Binding%2520Model%2520of%2520PCSK9%2520Interaction%2520with%2520the%2520Low-Density%2520Lipoprotein%2520Receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D5464%26epage%3D5470%26doi%3D10.1074%2Fjbc.m110.199042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taechalertpaisarn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of the PCSK9-LDLR Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3242</span>– <span class="NLM_lpage">3249</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=3242-3249&author=J.+Taechalertpaisarnauthor=B.+Zhaoauthor=X.+Liangauthor=K.+Burgess&title=Small+Molecule+Inhibitors+of+the+PCSK9-LDLR+Interaction&doi=10.1021%2Fjacs.7b09360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of the PCSK9·LDLR Interaction</span></div><div class="casAuthors">Taechalertpaisarn, Jaru; Zhao, Bosheng; Liang, Xiaowen; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3242-3249</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction between proprotein convertase subtilisin/kexin type 9 (PCSK9) and low-d. lipoprotein receptor (LDLR) is a relatively new, and extremely important, validated therapeutic target for treatment and prevention of heart disease.  Experts in the area agree that the first small mols. to disrupt PCSK9•LDLR would represent a milestone in the field, yet no credible leads have been reported.  This paper describes how side-chain orientations in preferred conformations of carefully designed chemotypes were compared with LDLR side-chains at the PCSK9•LDLR interface to find mols. that would mimic interface regions of LDLR.  This approach is an example of the procedure called EKO (Exploring Key Orientations).  The guiding hypothesis on which EKO is based is that good matches indicate the chemotypes bearing the same side-chains as the protein at the sites of overlay have the potential to disrupt the parent protein-protein interaction (PPI).  In the event, the EKO procedure and one round of combinatorial fragment-based virtual docking, led to the discovery of seven compds. that bound PCSK9 (SPR and ELISA) and had a favorable outcome in a cellular assay (hepatocyte uptake of fluorescently labeled LDL particles) and increased the expression LDLR on hepatocytes in culture.  Three promising hit compds. in this series had dissocn. consts. for PCSK9 binding in the 20-40 μM range, and one of these was modified with a photoaffinity label and shown to form a covalent conjugate with PCSK9 on photolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4QTYDzFsr9bVg90H21EOLACvtfcHk0li8Q6aZUOAM3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGlt7Y%253D&md5=488cbeef333e4bcd13c78481c5745cc5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09360%26sid%3Dliteratum%253Aachs%26aulast%3DTaechalertpaisarn%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520the%2520PCSK9-LDLR%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D3242%26epage%3D3249%26doi%3D10.1021%2Fjacs.7b09360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stucchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazioso, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvani, A.</span></span> <span> </span><span class="NLM_article-title">Disrupting he PCSK9/LDLR Protein-Protein Interaction by an Imidazole-Based Minimalist Peptidomimetic</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">9736</span>– <span class="NLM_lpage">9740</span>, <span class="refDoi"> DOI: 10.1039/c6ob01642a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1039%2FC6OB01642A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=27722650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=9736-9740&author=M.+Stucchiauthor=G.+Graziosoauthor=C.+Lammiauthor=S.+Manaraauthor=C.+Zanoniauthor=A.+Arnoldiauthor=G.+Lesmaauthor=A.+Silvani&title=Disrupting+he+PCSK9%2FLDLR+Protein-Protein+Interaction+by+an+Imidazole-Based+Minimalist+Peptidomimetic&doi=10.1039%2Fc6ob01642a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the PCSK9/LDLR protein-protein interaction by an imidazole-based minimalist peptidomimetic</span></div><div class="casAuthors">Stucchi, Mattia; Grazioso, Giovanni; Lammi, Carmen; Manara, Silvia; Zanoni, Chiara; Arnoldi, Anna; Lesma, Giordano; Silvani, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9736-9740</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein we report on the multicomponent synthesis of a novel imidazole-based compd., able to act efficiently as a minimalist β-strand mimic.  Biol. evaluation proved its ability to impair the LDLR-PCSK9 protein-protein interaction, disclosing it as the first small mol. exerting a PCSK9-mediated hypocholesterolemic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVtgyGhfHK_bVg90H21EOLACvtfcHk0li8Q6aZUOAM3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLrP&md5=015606884b7a48349de93c76a160f96b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1039%2FC6OB01642A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01642A%26sid%3Dliteratum%253Aachs%26aulast%3DStucchi%26aufirst%3DM.%26aulast%3DGrazioso%26aufirst%3DG.%26aulast%3DLammi%26aufirst%3DC.%26aulast%3DManara%26aufirst%3DS.%26aulast%3DZanoni%26aufirst%3DC.%26aulast%3DArnoldi%26aufirst%3DA.%26aulast%3DLesma%26aufirst%3DG.%26aulast%3DSilvani%26aufirst%3DA.%26atitle%3DDisrupting%2520he%2520PCSK9%252FLDLR%2520Protein-Protein%2520Interaction%2520by%2520an%2520Imidazole-Based%2520Minimalist%2520Peptidomimetic%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D9736%26epage%3D9740%26doi%3D10.1039%2Fc6ob01642a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Londregan, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lintner, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dullea, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genovino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuma, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesp, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspnes, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reidich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salatto, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabot, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cate, J. H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, D. W.</span></span> <span> </span><span class="NLM_article-title">Small Molecule PCSK9 Inhibitors: Hit to Lead Optimization of Systemic Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5704</span>– <span class="NLM_lpage">5718</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00650</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00650" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5704-5718&author=A.+T.+Londreganauthor=L.+Weiauthor=J.+Xiaoauthor=N.+G.+Lintnerauthor=D.+Petersenauthor=R.+G.+Dulleaauthor=K.+F.+McClureauthor=M.+W.+Boltauthor=J.+S.+Warmusauthor=S.+B.+Coffeyauthor=C.+Limberakisauthor=J.+Genovinoauthor=B.+A.+Thumaauthor=K.+D.+Hespauthor=G.+E.+Aspnesauthor=B.+Reidichauthor=C.+T.+Salattoauthor=J.+R.+Chabotauthor=J.+H.+D.+Cateauthor=S.+Lirasauthor=D.+W.+Piotrowski&title=Small+Molecule+PCSK9+Inhibitors%3A+Hit+to+Lead+Optimization+of+Systemic+Agents&doi=10.1021%2Facs.jmedchem.8b00650"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Proprotein Convertase Subtilisin/Kexin Type 9 (PCSK9) Inhibitors: Hit to Lead Optimization of Systemic Agents</span></div><div class="casAuthors">Londregan, Allyn T.; Wei, Liuqing; Xiao, Jun; Lintner, Nathanael G.; Petersen, Donna; Dullea, Robert G.; McClure, Kim F.; Bolt, Michael W.; Warmus, Joseph S.; Coffey, Steven B.; Limberakis, Chris; Genovino, Julien; Thuma, Benjamin A.; Hesp, Kevin D.; Aspnes, Gary E.; Reidich, Benjamin; Salatto, Christopher T.; Chabot, Jeffrey R.; Cate, Jamie H. D.; Liras, Spiros; Piotrowski, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5704-5718</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The optimization of a new class of small mol. PCSK9 mRNA translation inhibitors is described.  The potency, physicochem. properties, and off-target pharmacol. assocd. with the hit compd. (1) were improved by changes to two regions of the mol.  The last step in the synthesis of the congested amide center was enabled by three different routes.  Subtle structural changes yielded significant changes in pharmacol. and off-target margins.  These efforts led to the identification of 7l and 7n with overall profiles suitable for in vivo evaluation.  In a 14-day toxicol. study, 7l demonstrated an improved safety profile vs lead 7f.  We hypothesize that the improved safety profile is related to diminished binding of 7l to nontranslating ribosomes and an apparent improvement in transcript selectivity due to the lower strength of 7l stalling of off-target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj72SuB2HcJbVg90H21EOLACvtfcHk0li8Q6aZUOAM3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyltbnI&md5=14f6b537fdf7c0684790c6d663c5e3f3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00650&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00650%26sid%3Dliteratum%253Aachs%26aulast%3DLondregan%26aufirst%3DA.%2BT.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DLintner%26aufirst%3DN.%2BG.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DDullea%26aufirst%3DR.%2BG.%26aulast%3DMcClure%26aufirst%3DK.%2BF.%26aulast%3DBolt%26aufirst%3DM.%2BW.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26aulast%3DCoffey%26aufirst%3DS.%2BB.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DGenovino%26aufirst%3DJ.%26aulast%3DThuma%26aufirst%3DB.%2BA.%26aulast%3DHesp%26aufirst%3DK.%2BD.%26aulast%3DAspnes%26aufirst%3DG.%2BE.%26aulast%3DReidich%26aufirst%3DB.%26aulast%3DSalatto%26aufirst%3DC.%2BT.%26aulast%3DChabot%26aufirst%3DJ.%2BR.%26aulast%3DCate%26aufirst%3DJ.%2BH.%2BD.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPiotrowski%26aufirst%3DD.%2BW.%26atitle%3DSmall%2520Molecule%2520PCSK9%2520Inhibitors%253A%2520Hit%2520to%2520Lead%2520Optimization%2520of%2520Systemic%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5704%26epage%3D5718%26doi%3D10.1021%2Facs.jmedchem.8b00650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrilli, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abeywickrema, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agnani, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baysarowich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNunzio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lisnock, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palyha, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salowe, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strack, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youm, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zink, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zokian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addona, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akinsanya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tata, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbriglio, J. E.</span></span> <span> </span><span class="NLM_article-title">From Screening to Targeted Degredation: Strategies for the Discovery and Optimization of Small Molecule Ligands for PCSK9</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">40.e3</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.chembiol.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=31653597" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=32-40.e3&author=W.+L.+Petrilliauthor=G.+C.+Adamauthor=R.+S.+Erdmannauthor=P.+Abeywickremaauthor=V.+Agnaniauthor=X.+Aiauthor=J.+Baysarowichauthor=N.+Byrneauthor=J.+P.+Caldwellauthor=W.+Changauthor=E.+DiNunzioauthor=Z.+Fengauthor=R.+Fordauthor=S.+Haauthor=Y.+Huangauthor=B.+Hubbardauthor=J.+J.+Johnstonauthor=M.+Kavanaauthor=J.-M.+Lisnockauthor=R.+Liangauthor=J.+Luauthor=J.+Mengauthor=P.+Orthauthor=O.+Palyhaauthor=G.+Parthasarathyauthor=S.+P.+Saloweauthor=S.+Sharmaauthor=J.+Shipmanauthor=S.+M.+Soissonauthor=A.+M.+Strackauthor=H.+Youmauthor=K.+Zhaoauthor=D.+L.+Zinkauthor=H.+Zokianauthor=G.+H.+Addonaauthor=K.+Akinsanyaauthor=J.+R.+Tataauthor=Y.+Xiongauthor=J.+E.+Imbriglio&title=From+Screening+to+Targeted+Degredation%3A+Strategies+for+the+Discovery+and+Optimization+of+Small+Molecule+Ligands+for+PCSK9&doi=10.1016%2Fj.chembiol.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DPetrilli%26aufirst%3DW.%2BL.%26aulast%3DAdam%26aufirst%3DG.%2BC.%26aulast%3DErdmann%26aufirst%3DR.%2BS.%26aulast%3DAbeywickrema%26aufirst%3DP.%26aulast%3DAgnani%26aufirst%3DV.%26aulast%3DAi%26aufirst%3DX.%26aulast%3DBaysarowich%26aufirst%3DJ.%26aulast%3DByrne%26aufirst%3DN.%26aulast%3DCaldwell%26aufirst%3DJ.%2BP.%26aulast%3DChang%26aufirst%3DW.%26aulast%3DDiNunzio%26aufirst%3DE.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DFord%26aufirst%3DR.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DHubbard%26aufirst%3DB.%26aulast%3DJohnston%26aufirst%3DJ.%2BJ.%26aulast%3DKavana%26aufirst%3DM.%26aulast%3DLisnock%26aufirst%3DJ.-M.%26aulast%3DLiang%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DPalyha%26aufirst%3DO.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DSalowe%26aufirst%3DS.%2BP.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DShipman%26aufirst%3DJ.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DStrack%26aufirst%3DA.%2BM.%26aulast%3DYoum%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DZink%26aufirst%3DD.%2BL.%26aulast%3DZokian%26aufirst%3DH.%26aulast%3DAddona%26aufirst%3DG.%2BH.%26aulast%3DAkinsanya%26aufirst%3DK.%26aulast%3DTata%26aufirst%3DJ.%2BR.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DImbriglio%26aufirst%3DJ.%2BE.%26atitle%3DFrom%2520Screening%2520to%2520Targeted%2520Degredation%253A%2520Strategies%2520for%2520the%2520Discovery%2520and%2520Optimization%2520of%2520Small%2520Molecule%2520Ligands%2520for%2520PCSK9%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D27%26spage%3D32%26epage%3D40.e3%26doi%3D10.1016%2Fj.chembiol.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong-Beltran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchhofer, D.</span></span> <span> </span><span class="NLM_article-title">Identification of a Small Peptide that Inhibits PCSK9 Protein Binding to the Low-Density Lipoprotein Receptor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">942</span>– <span class="NLM_lpage">955</span>, <span class="refDoi"> DOI: 10.1074/jbc.m113.514067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1074%2Fjbc.M113.514067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=24225950" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2013&pages=942-955&author=Y.+Zhangauthor=C.+Eigenbrotauthor=L.+Zhouauthor=S.+Shiaauthor=W.+Liauthor=C.+Quanauthor=J.+Tomauthor=P.+Moranauthor=P.+Di+Lelloauthor=N.+J.+Skeltonauthor=M.+Kong-Beltranauthor=A.+Petersonauthor=D.+Kirchhofer&title=Identification+of+a+Small+Peptide+that+Inhibits+PCSK9+Protein+Binding+to+the+Low-Density+Lipoprotein+Receptor&doi=10.1074%2Fjbc.m113.514067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.514067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.514067%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DShia%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DQuan%26aufirst%3DC.%26aulast%3DTom%26aufirst%3DJ.%26aulast%3DMoran%26aufirst%3DP.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DKong-Beltran%26aufirst%3DM.%26aulast%3DPeterson%26aufirst%3DA.%26aulast%3DKirchhofer%26aufirst%3DD.%26atitle%3DIdentification%2520of%2520a%2520Small%2520Peptide%2520that%2520Inhibits%2520PCSK9%2520Protein%2520Binding%2520to%2520the%2520Low-Density%2520Lipoprotein%2520Receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D289%26spage%3D942%26epage%3D955%26doi%3D10.1074%2Fjbc.m113.514067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong-Beltran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchhofer, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Cryptic Peptide-Binding site on PCSK9 and Design of Antagonists</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">848</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1038%2Fnsmb.3453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=28825733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=848-856&author=Y.+Zhangauthor=M.+Ultschauthor=N.+J.+Skeltonauthor=D.+J.+Burdickauthor=M.+H.+Beresiniauthor=W.+Liauthor=M.+Kong-Beltranauthor=A.+Petersonauthor=J.+Quinnauthor=C.+Chiuauthor=Y.+Wuauthor=S.+Shiaauthor=P.+Moranauthor=P.+Di+Lelloauthor=C.+Eigenbrotauthor=D.+Kirchhofer&title=Discovery+of+a+Cryptic+Peptide-Binding+site+on+PCSK9+and+Design+of+Antagonists&doi=10.1038%2Fnsmb.3453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a cryptic peptide-binding site on PCSK9 and design of antagonists</span></div><div class="casAuthors">Zhang, Yingnan; Ultsch, Mark; Skelton, Nicholas J.; Burdick, Daniel J.; Beresini, Maureen H.; Li, Wei; Kong-Beltran, Monica; Peterson, Andrew; Quinn, John; Chiu, Cecilia; Wu, Yan; Shia, Steven; Moran, Paul; Di Lello, Paola; Eigenbrot, Charles; Kirchhofer, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">848-856</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Proprotein convertase subtilisin/kexin type 9 (PCSK9) regulates plasma LDL cholesterol (LDL-c) levels by promoting the degrdn. of liver LDL receptors (LDLRs).  Antibodies that inhibit PCSK9 binding to the EGF(A) domain of the LDLR are effective in lowering LDL-c.  However, the discovery of small-mol. therapeutics is hampered by difficulty in targeting the relatively flat EGF(A)-binding site on PCSK9.  Here we demonstrate that it is possible to target this site, based on the finding that the PCSK9 P' helix displays conformational flexibility.  As a consequence, the vacated N-terminal groove of PCSK9, which is adjacent to the EGF(A)-binding site, is in fact accessible to small peptides.  In phage-display expts., the EGF(A)-mimicking peptide Pep2-8 was used as an anchor peptide for the attachment of an extension peptide library directed toward the groove site.  Guided by structural information, we further engineered the identified groove-binding peptides into antagonists, which encroach on the EGF(A)-binding site and inhibit LDLR binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC8qO6Y3Nck7Vg90H21EOLACvtfcHk0ljL0dSpwXPWyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbjO&md5=1f449068f5745918cd8b7e0b7d7dd6ae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3453%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DKong-Beltran%26aufirst%3DM.%26aulast%3DPeterson%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DJ.%26aulast%3DChiu%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DShia%26aufirst%3DS.%26aulast%3DMoran%26aufirst%3DP.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DKirchhofer%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520a%2520Cryptic%2520Peptide-Binding%2520site%2520on%2520PCSK9%2520and%2520Design%2520of%2520Antagonists%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D848%26epage%3D856%26doi%3D10.1038%2Fnsmb.3453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Inhibitors of Protein – Protein Interactions: Mimicking Peptide Binding Epitopes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8896</span>– <span class="NLM_lpage">8927</span>, <span class="refDoi"> DOI: 10.1002/anie.201412070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1002%2Fanie.201412070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=26119925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=8896-8927&author=M.+Pelay-Gimenoauthor=A.+Glasauthor=O.+Kochauthor=T.+N.+Grossmann&title=Structure-Based+Design+of+Inhibitors+of+Protein+%E2%80%93+Protein+Interactions%3A+Mimicking+Peptide+Binding+Epitopes&doi=10.1002%2Fanie.201412070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Inhibitors of Protein-Protein Interactions: Mimicking Peptide Binding Epitopes</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Glas, Adrian; Koch, Oliver; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">8896-8927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are involved at all levels of cellular organization, thus making the development of PPI inhibitors extremely valuable.  The identification of selective inhibitors is challenging because of the shallow and extended nature of PPI interfaces.  Inhibitors can be obtained by mimicking peptide binding epitopes in their bioactive conformation.  For this purpose, several strategies have been evolved to enable a projection of side chain functionalities in analogy to peptide secondary structures, thereby yielding mols. that are generally referred to as peptidomimetics.  Herein, we introduce a new classification of peptidomimetics (classes A-D) that enables a clear assignment of available approaches.  Based on this classification, the Review summarizes strategies that have been applied for the structure-based design of PPI inhibitors through stabilizing or mimicking turns, β-sheets, and helixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPTEmZpatXbrVg90H21EOLACvtfcHk0li5VGoZFlK5YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM&md5=447c84aab6275b1057fda9a610918f2b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412070%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DO.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DStructure-Based%2520Design%2520of%2520Inhibitors%2520of%2520Protein%2520%25E2%2580%2593%2520Protein%2520Interactions%253A%2520Mimicking%2520Peptide%2520Binding%2520Epitopes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D8896%26epage%3D8927%26doi%3D10.1002%2Fanie.201412070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dyrbuś, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gąsior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, M.</span></span> <span> </span><span class="NLM_article-title">Inclisiran – New Hope in the Management of Lipid Disorders</span>. <i>J. Clin. Lipidol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.jacl.2019.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.jacl.2019.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=31879073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A280%3ADC%252BB3Mbkt1Kksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2020&pages=16-27&author=K.+Dyrbu%C5%9Bauthor=M.+G%C4%85siorauthor=P.+Pensonauthor=K.+K.+Rayauthor=M.+Banach&title=Inclisiran+%E2%80%93+New+Hope+in+the+Management+of+Lipid+Disorders&doi=10.1016%2Fj.jacl.2019.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inclisiran-New hope in the management of lipid disorders?</span></div><div class="casAuthors">Dyrbus Krzysztof; GAsior Mariusz; Penson Peter; Ray Kausik K; Banach Maciej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical lipidology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-27</span>
        ISSN:<span class="NLM_cas:issn">1933-2874</span>.
    </div><div class="casAbstract">Drugs reducing plasma concentrations of apolipoprotein B-containing lipoproteins have been demonstrated to reduce the risk of cardiovascular disease (CVD) in both primary and secondary prevention.  Despite the demonstrated efficacy of statins and ezetimibe on low-density lipoprotein (LDL) concentration and long-term CVD risk, a large number of patients do not achieve their therapeutic goals.  The introduction of monoclonal antibodies against proprotein convertase subtilisin/kexin type 9 (PCSK9) protein was a milestone in the treatment of lipid disorders, as their administration leads to unprecedentedly low LDL cholesterol concentrations.  Inclisiran represents an entirely new mechanism of PSCK9 protein inhibition in hepatocytes, targeting the messenger RNA for PCSK9.  Its administration is necessary only every 3 to 6 months, which is an essential advantage over statin and monoclonal antibody therapy.  The infrequent administration regimen can increase the number of patients who maintain their therapeutic goals, especially in patients struggling to comply with daily or biweekly pharmacotherapy.  Preclinical studies and Phase I and Phase II clinical trials of inclisiran have demonstrated its tolerability and efficacy in promoting long-term reduction of both PCSK9 protein and LDL cholesterol.  The efficacy and safety of inclisiran will continue to be assessed in ongoing and forthcoming trials on larger patient groups.  If the results of these trials reflect previously published data, they will add further evidence that inclisiran might be a revolutionary new tool in the pharmacologic management of plasma lipids.  This review summarizes the currently available literature data on inclisiran with respect to its mechanism of action, effectiveness, and safety as a lipid-lowering drug for CVD prevention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiaUOLUK_X8O98rul4nhSdfW6udTcc2eYUsIfH0dU1yLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mbkt1Kksg%253D%253D&md5=19712f7a6457cc73022ea7396944be58</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jacl.2019.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacl.2019.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DDyrbu%25C5%259B%26aufirst%3DK.%26aulast%3DG%25C4%2585sior%26aufirst%3DM.%26aulast%3DPenson%26aufirst%3DP.%26aulast%3DRay%26aufirst%3DK.%2BK.%26aulast%3DBanach%26aufirst%3DM.%26atitle%3DInclisiran%2520%25E2%2580%2593%2520New%2520Hope%2520in%2520the%2520Management%2520of%2520Lipid%2520Disorders%26jtitle%3DJ.%2520Clin.%2520Lipidol.%26date%3D2020%26volume%3D14%26spage%3D16%26epage%3D27%26doi%3D10.1016%2Fj.jacl.2019.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecules as Inhibitors of PCSK9: Current Status and Future Challenges</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">212</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.ejmech.2018.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30448414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Cht7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2019&pages=212-233&author=S.+Xuauthor=S.+Luoauthor=Z.+Zhuauthor=J.+Xu&title=Small+Molecules+as+Inhibitors+of+PCSK9%3A+Current+Status+and+Future+Challenges&doi=10.1016%2Fj.ejmech.2018.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules as inhibitors of PCSK9: Current status and future challenges</span></div><div class="casAuthors">Xu, Shengtao; Luo, Shanshan; Zhu, Zheying; Xu, Jinyi</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">212-233</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Proprotein convertase subtilisin/kexin type 9 (PCSK9) plays an important role in regulating lipoprotein metab. by binding to low-d. lipoprotein receptors (LDLRs), leading to their degrdn.  LDL cholesterol (LDL-C) lowering drugs that operate through the inhibition of PCSK9 are being pursued for the management of hypercholesterolemia and reducing its assocd. atherosclerotic cardiovascular disease (CVD) risk.  Two PCSK9-blocking monoclonal antibodies (mAbs), alirocumab and evolocumab, were approved in 2015.  However, the high costs of PCSK9 antibody drugs impede their prior authorization practices and reduce their long-term adherence.  Given the potential of small-mol. drugs, the development of small-mol. PCSK9 inhibitors has attracted considerable attention.  This article provides an overview of the recent development of small-mol. PCSK9 inhibitors disclosed in the literature and patent applications, and different approaches that have been pursued to modulate the functional activity of PCSK9 using small mols. are described.  Challenges and potential strategies in developing small-mol. PCSK9 inhibitors are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz6U3osadAYbVg90H21EOLACvtfcHk0li5VGoZFlK5YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Cht7fM&md5=75a981cd8d564043a3685e9d2d8ca6ce</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DSmall%2520Molecules%2520as%2520Inhibitors%2520of%2520PCSK9%253A%2520Current%2520Status%2520and%2520Future%2520Challenges%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D162%26spage%3D212%26epage%3D233%26doi%3D10.1016%2Fj.ejmech.2018.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavecchia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchia, C.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Developing PCSK9 Inhibitors for Lipid-Lowering Therapy</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.4155%2Ffmc-2018-0294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30892945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmslGnu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=423-441&author=A.+Lavecchiaauthor=C.+Cerchia&title=Recent+Advances+in+Developing+PCSK9+Inhibitors+for+Lipid-Lowering+Therapy&doi=10.4155%2Ffmc-2018-0294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in developing PCSK9 inhibitors for lipid-lowering therapy</span></div><div class="casAuthors">Lavecchia, Antonio; Cerchia, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">423-441</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Cardiovascular disease is the major cause of death globally, with hypercholesterolemia being an important risk factor.  The PCSK9 represents an attractive therapeutic target for hypercholesterolemia treatment and is currently in the spotlight of the scientific community.  After autocatalytic activation in the hepatocyte endoplasmic reticulum, this convertase binds to the LDLR and channels it to the degrdn. pathway.  This review gives an overview on the latest developments in the inhibition of PCSK9, including disruption of the protein-protein interaction (PPI) between PCSK9 and LDLR by peptidomimetics, adnectins and monoclonal antibodies and the suppression of PCSK9 expression by small mols., siRNA and genome editing techniques.  In addn., we discuss alternative approaches, such as anti-PCSK9 active vaccination and heparin mimetics.  Graphical abstr. :.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk-DYJfvdpxbVg90H21EOLACvtfcHk0lhaTm0Uju4CaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmslGnu7k%253D&md5=7bc29631a590d091bd3c1129fa052e6b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0294%26sid%3Dliteratum%253Aachs%26aulast%3DLavecchia%26aufirst%3DA.%26aulast%3DCerchia%26aufirst%3DC.%26atitle%3DRecent%2520Advances%2520in%2520Developing%2520PCSK9%2520Inhibitors%2520for%2520Lipid-Lowering%2520Therapy%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D423%26epage%3D441%26doi%3D10.4155%2Ffmc-2018-0294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-P.</span></span> <span> </span><span class="NLM_article-title">PCSK9 Inhibitors: Novel Therapeutic Strategies for Lowering LDL Cholesterol</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">176</span>, <span class="refDoi"> DOI: 10.2174/1389557518666180423111442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.2174%2F1389557518666180423111442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=29692249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVertw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=165-176&author=Y.+Wangauthor=Z.-P.+Liu&title=PCSK9+Inhibitors%3A+Novel+Therapeutic+Strategies+for+Lowering+LDL+Cholesterol&doi=10.2174%2F1389557518666180423111442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">PCSK9 Inhibitors: Novel Therapeutic Strategies for Lowering LDLCholesterol</span></div><div class="casAuthors">Wang, Yan; Liu, Zhao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-176</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Statins are currently the major therapeutic strategies to lower low-d. lipoprotein cholesterol (LDL-C) levels.  However, a no. of hypercholesterolemia patients still have a residual cardiovascular disease (CVD) risk despite taking the max.-tolerated dose of statins.  Proprotein convertase subtilisin/kexin type 9 (PCSK9) binds to low-d. lipoprotein receptor (LDLR), inducing its degrdn. in the lysosome and inhibiting LDLR recirculating to the cell membranes.  The gain-offunction mutations in PCSK9 elevate the LDL-C levels in plasma.  Therefore, PCSK9 inhibitors become novel therapeutic approaches in the treatment of hypercholesterolemia.  Several PCSK9 inhibitors have been under investigation, and much progress has been made in clin. trials, esp. for monoclonal antibodies (MoAbs).  Two MoAbs, evolocumab and alirocumab, are now in clin. use.  In this review, we summarize the development of PCSK9 inhibitors, including antisense oligonucleotides (ASOs), small interfering RNA (siRNA), small mol. inhibitor, MoAbs, mimetic peptides and adnectins, and the related safety issues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLkfyLZV7Fy7Vg90H21EOLACvtfcHk0lhaTm0Uju4CaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVertw%253D%253D&md5=d8ed5787f60a90b0c439818cd2e3ec3d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F1389557518666180423111442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557518666180423111442%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.-P.%26atitle%3DPCSK9%2520Inhibitors%253A%2520Novel%2520Therapeutic%2520Strategies%2520for%2520Lowering%2520LDL%2520Cholesterol%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2019%26volume%3D19%26spage%3D165%26epage%3D176%26doi%3D10.2174%2F1389557518666180423111442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4181</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=4167-4181&author=A.+A.+Vinogradovauthor=Y.+Yinauthor=H.+Suga&title=Macrocyclic+Peptides+as+Drug+Candidates%3A+Recent+Progress+and+Remaining+Challenges&doi=10.1021%2Fjacs.8b13178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span></div><div class="casAuthors">Vinogradov, Alexander A.; Yin, Yizhen; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4167-4181</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptides as a therapeutic modality attract much attention due to their synthetic accessibility, high degree of specific binding, and the ability to target protein surfaces traditionally considered "undruggable".  Unfortunately, at the same time, other pharmacol. properties of a generic peptide, such as metabolic stability and cell permeability, are quite poor, which limits the success of de novo discovered biol. active peptides as drug candidates.  Here, we review how macrocyclization as well as the incorporation of nonproteogenic amino acids and various conjugation strategies may be utilized to improve on these characteristics to create better drug candidates.  We analyze recent progress and remaining challenges in improving individual pharmacol. properties of bioactive peptides, and offer our opinion on interfacing these, often conflicting, considerations, to create balanced drug candidates as a potential way to make further progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRPkCAGk8-rVg90H21EOLACvtfcHk0lhaTm0Uju4CaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D&md5=59b6197874d4709a74089367a09119cc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13178%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradov%26aufirst%3DA.%2BA.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptides%2520as%2520Drug%2520Candidates%253A%2520Recent%2520Progress%2520and%2520Remaining%2520Challenges%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D4167%26epage%3D4181%26doi%3D10.1021%2Fjacs.8b13178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Josephson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricardo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szostak, J. W.</span></span> <span> </span><span class="NLM_article-title">mRNA Display: From Basic Principles to Macrocycle Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">399</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.drudis.2013.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=24157402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsleltrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=388-399&author=K.+Josephsonauthor=A.+Ricardoauthor=J.+W.+Szostak&title=mRNA+Display%3A+From+Basic+Principles+to+Macrocycle+Drug+Discovery&doi=10.1016%2Fj.drudis.2013.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">mRNA display: from basic principles to macrocycle drug discovery</span></div><div class="casAuthors">Josephson, Kristopher; Ricardo, Alonso; Szostak, Jack W.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">388-399</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  We describe a new discovery technol. that uses mRNA-display to rapidly synthesize and screen macrocyclic peptide libraries to explore a valuable region of chem. space typified by natural products.  This technol. allows high-affinity peptidic macrocycles contg. modified backbones and unnatural side chains to be readily selected based on target binding.  Success stories covering the first examples of these libraries suggest that they could be used for the discovery of intracellular protein-protein interaction inhibitors, highly selective enzyme inhibitors or synthetic replacements for monoclonal antibodies.  The review concludes with a look to the future regarding how this technol. might be improved with respect to library design for cell permeability and bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobuL9lWFPZIbVg90H21EOLACvtfcHk0ljCL7lODTGbgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsleltrnI&md5=6676fef2ca8f0a81cc0e993ee4ec7424</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DJosephson%26aufirst%3DK.%26aulast%3DRicardo%26aufirst%3DA.%26aulast%3DSzostak%26aufirst%3DJ.%2BW.%26atitle%3DmRNA%2520Display%253A%2520From%2520Basic%2520Principles%2520to%2520Macrocycle%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D388%26epage%3D399%26doi%3D10.1016%2Fj.drudis.2013.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiedmann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">RNA Display Methods for the Discovery of Bioactive Macrocycles</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10360</span>– <span class="NLM_lpage">10391</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOht7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10360-10391&author=Y.+Huangauthor=M.+M.+Wiedmannauthor=H.+Suga&title=RNA+Display+Methods+for+the+Discovery+of+Bioactive+Macrocycles&doi=10.1021%2Facs.chemrev.8b00430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">RNA Display Methods for the Discovery of Bioactive Macrocycles</span></div><div class="casAuthors">Huang, Yichao; Wiedmann, Mareike Margarete; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10360-10391</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The past two decades have witnessed the emergence of macrocycles, including macrocyclic peptides, as a promising yet underexploited class of de novo drug candidates.  Both rational/computational design and in vitro display systems have contributed tremendously to the development of cyclic peptide binders of either traditional targets such as cell-surface receptors and enzymes or challenging targets such as protein-protein interaction surfaces. mRNA display, a key platform technol. for the discovery of cyclic peptide ligands, has become one of the leading strategies that can generate natural-product-like macrocyclic peptide binders with antibody-like affinities.  On the basis of the original cell-free transcription/translation system, mRNA display is highly evolvable to realize its full potential by applying genetic reprogramming and chem./enzymic modifications.  In addn., mRNA display also allows the follow-up hit-to-lead development using high-throughput focused affinity maturation.  Finally, mRNA-displayed peptides can be readily engineered to create chem. conjugates based on known small mols. or biologics.  This review covers the birth and growth of mRNA display and discusses the above features of mRNA display with success stories and future perspectives and is up to date as of August 2018.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorCyvw1CP_xLVg90H21EOLACvtfcHk0ljCL7lODTGbgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOht7bJ&md5=dbf0aec0744b0767f8e634fe60ed2596</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00430%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWiedmann%26aufirst%3DM.%2BM.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DRNA%2520Display%2520Methods%2520for%2520the%2520Discovery%2520of%2520Bioactive%2520Macrocycles%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10360%26epage%3D10391%26doi%3D10.1021%2Facs.chemrev.8b00430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerekes, A. D.</span></span> Unpublished results from our laboratories. A manuscript describing the alternative series of PCSK9 molecules is currently in preparation.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kerekes%2C+A.+D.+Unpublished+results+from+our+laboratories.+A+manuscript+describing+the+alternative+series+of+PCSK9+molecules+is+currently+in+preparation."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DKerekes%26aufirst%3DA.%2BD." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendels, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obst-Sander, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Fluorine in Medicinal Chemistry</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">643</span>, <span class="refDoi"> DOI: 10.1002/cbic.200301023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1002%2Fcbic.200301023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=15122635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A280%3ADC%252BD2c3it1egug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=637-643&author=H.-J.+B%C3%B6hmauthor=D.+Bannerauthor=S.+Bendelsauthor=M.+Kansyauthor=B.+Kuhnauthor=K.+M%C3%BCllerauthor=U.+Obst-Sanderauthor=M.+Stahl&title=Fluorine+in+Medicinal+Chemistry&doi=10.1002%2Fcbic.200301023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorine in medicinal chemistry</span></div><div class="casAuthors">Bohm Hans-Joachim; Banner David; Bendels Stefanie; Kansy Manfred; Kuhn Bernd; Muller Klaus; Obst-Sander Ulrike; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Chembiochem : a European journal of chemical biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">637-43</span>
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    </div><div class="casAbstract">Fluorinated compounds are synthesized in pharmaceutical research on a routine basis and many marketed compounds contain fluorine.  The present review summarizes some of the most frequently employed strategies for using fluorine substituents in medicinal chemistry.  Quite often, fluorine is introduced to improve the metabolic stability by blocking metabolically labile sites.  However, fluorine can also be used to modulate the physicochemical properties, such as lipophilicity or basicity.  It may exert a substantial effect on the conformation of a molecule.  Increasingly, fluorine is used to enhance the binding affinity to the target protein.  Recent 3D-structure determinations of protein complexes with bound fluorinated ligands have led to an improved understanding of the nonbonding protein-ligand interactions that involve fluorine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4zDG_wo1YGtts4TFFB11gfW6udTcc2eZgpFDmGRzhl7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3it1egug%253D%253D&md5=fa5ca2cfc278b83c80d5c96b18dfe312</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200301023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200301023%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26aulast%3DBanner%26aufirst%3DD.%26aulast%3DBendels%26aufirst%3DS.%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DObst-Sander%26aufirst%3DU.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DFluorine%2520in%2520Medicinal%2520Chemistry%26jtitle%3DChemBioChem%26date%3D2004%26volume%3D5%26spage%3D637%26epage%3D643%26doi%3D10.1002%2Fcbic.200301023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McNeil, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pundir, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Undem, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span> <span> </span><span class="NLM_article-title">Indentification of a Mast-Cell-Specific Receptor Crucial for Pseudo-allergic Drug Reactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>519</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1038/nature14022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1038%2Fnature14022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=25517090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKksr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=237-241&author=B.+D.+McNeilauthor=P.+Pundirauthor=S.+Meekerauthor=L.+Hanauthor=B.+J.+Undemauthor=M.+Kulkaauthor=X.+Dong&title=Indentification+of+a+Mast-Cell-Specific+Receptor+Crucial+for+Pseudo-allergic+Drug+Reactions&doi=10.1038%2Fnature14022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a mast-cell-specific receptor crucial for pseudo-allergic drug reactions</span></div><div class="casAuthors">McNeil, Benjamin D.; Pundir, Priyanka; Meeker, Sonya; Han, Liang; Undem, Bradley J.; Kulka, Marianna; Dong, Xinzhong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7542</span>),
    <span class="NLM_cas:pages">237-241</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mast cells are primary effectors in allergic reactions, and may have important roles in disease by secreting histamine and various inflammatory and immunomodulatory substances.  Although they are classically activated by Ig (Ig)E antibodies, a unique property of mast cells is their antibody-independent responsiveness to a range of cationic substances, collectively called basic secretagogues, including inflammatory peptides and drugs assocd. with allergic-type reactions.  The pathogenic roles of these substances have prompted a decades-long search for their receptor(s).  Here the authors report that basic secretagogues activate mouse mast cells in vitro and in vivo through a single receptor, Mrgprb2, the orthologue of the human G-protein-coupled receptor MRGPRX2.  Secretagogue-induced histamine release, inflammation and airway contraction are abolished in Mrgprb2-null mutant mice.  Furthermore, the authors show that most classes of US Food and Drug Administration (FDA)-approved peptidergic drugs assocd. with allergic-type injection-site reactions also activate Mrgprb2 and MRGPRX2, and that injection-site inflammation is absent in mutant mice.  Finally, the authors det. that Mrgprb2 and MRGPRX2 are targets of many small-mol. drugs assocd. with systemic pseudo-allergic, or anaphylactoid, reactions; the authors show that drug-induced symptoms of anaphylactoid responses are significantly reduced in knockout mice; and the authors identify a common chem. motif in several of these mols. that may help predict side effects of other compds.  These discoveries introduce a mouse model to study mast cell activation by basic secretagogues and identify MRGPRX2 as a potential therapeutic target to reduce a subset of drug-induced adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrztm32h9jAMbVg90H21EOLACvtfcHk0lj43iPJVeO8Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKksr%252FL&md5=a77aa1c03938f35de07482715986f1bc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature14022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14022%26sid%3Dliteratum%253Aachs%26aulast%3DMcNeil%26aufirst%3DB.%2BD.%26aulast%3DPundir%26aufirst%3DP.%26aulast%3DMeeker%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DUndem%26aufirst%3DB.%2BJ.%26aulast%3DKulka%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DX.%26atitle%3DIndentification%2520of%2520a%2520Mast-Cell-Specific%2520Receptor%2520Crucial%2520for%2520Pseudo-allergic%2520Drug%2520Reactions%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D237%26epage%3D241%26doi%3D10.1038%2Fnature14022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, H.</span></span> <span> </span><span class="NLM_article-title">Roles of Mas-related G Protein-Coupled Receptor X2 on Mast Cell-Mediated Host Defense, Pseudoallergic Drug Reactions, and Inflammatory Diseases</span>. <i>J. Allergy Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1016/j.jaci.2016.04.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.jaci.2016.04.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=27448446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SlurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=700-710&author=H.+Subramanianauthor=K.+Guptaauthor=H.+Ali&title=Roles+of+Mas-related+G+Protein-Coupled+Receptor+X2+on+Mast+Cell-Mediated+Host+Defense%2C+Pseudoallergic+Drug+Reactions%2C+and+Inflammatory+Diseases&doi=10.1016%2Fj.jaci.2016.04.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of Mas-related G protein-coupled receptor X2 on mast cell-mediated host defense, pseudoallergic drug reactions, and chronic inflammatory diseases</span></div><div class="casAuthors">Subramanian, Hariharan; Gupta, Kshitij; Ali, Hydar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Allergy and Clinical Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">700-710</span>CODEN:
                <span class="NLM_cas:coden">JACIBY</span>;
        ISSN:<span class="NLM_cas:issn">0091-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Mast cells (MCs), which are granulated tissue-resident cells of hematopoietic lineage, contribute to vascular homeostasis, innate/adaptive immunity, and wound healing.  However, MCs are best known for their roles in allergic and inflammatory diseases, such as anaphylaxis, food allergy, rhinitis, itch, urticaria, atopic dermatitis, and asthma.  In addn. to the high-affinity IgE receptor (FcεRI), MCs express numerous G protein-coupled receptors (GPCRs), which are the largest group of membrane receptor proteins and the most common targets of drug therapy.  Antimicrobial host defense peptides, neuropeptides, major basic protein, eosinophil peroxidase, and many US Food and Drug Administration-approved peptidergic drugs activate human MCs through a novel GPCR known as Mas-related G protein-coupled receptor X2 (MRGPRX2; formerly known as MrgX2).  Unique features of MRGPRX2 that distinguish it from other GPCRs include their presence both on the plasma membrane and intracellular sites and their selective expression in MCs.  In this article we review the possible roles of MRGPRX2 on host defense, drug-induced anaphylactoid reactions, neurogenic inflammation, pain, itch, and chronic inflammatory diseases, such as urticaria and asthma.  We propose that host defense peptides that kill microbes directly and activate MCs through MRGPRX2 could serve as novel GPCR targets to modulate host defense against microbial infection.  Furthermore, mAbs or small-mol. inhibitors of MRGPRX2 could be developed for the treatment of MC-dependent allergic and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCGk6hW_9_2bVg90H21EOLACvtfcHk0lj43iPJVeO8Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SlurjJ&md5=84191e86547dd9ad97cd4cdb88a31a62</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.jaci.2016.04.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jaci.2016.04.051%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DH.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DAli%26aufirst%3DH.%26atitle%3DRoles%2520of%2520Mas-related%2520G%2520Protein-Coupled%2520Receptor%2520X2%2520on%2520Mast%2520Cell-Mediated%2520Host%2520Defense%252C%2520Pseudoallergic%2520Drug%2520Reactions%252C%2520and%2520Inflammatory%2520Diseases%26jtitle%3DJ.%2520Allergy%2520Clin.%2520Immunol.%26date%3D2016%26volume%3D138%26spage%3D700%26epage%3D710%26doi%3D10.1016%2Fj.jaci.2016.04.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hein, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span> <span> </span><span class="NLM_article-title">Click Chemistry, A Powerful Tool for Pharmaceutical Sciences</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2216</span>– <span class="NLM_lpage">2230</span>, <span class="refDoi"> DOI: 10.1007/s11095-008-9616-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1007%2Fs11095-008-9616-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=18509602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFOru7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2008&pages=2216-2230&author=C.+D.+Heinauthor=X.-M.+Liuauthor=D.+Wang&title=Click+Chemistry%2C+A+Powerful+Tool+for+Pharmaceutical+Sciences&doi=10.1007%2Fs11095-008-9616-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry, a powerful tool for pharmaceutical sciences</span></div><div class="casAuthors">Hein, Christopher D.; Liu, Xin-Ming; Wang, Dong</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2216-2230</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Click chem. refers to a group of reactions that are fast, simple to use, easy to purify, versatile, regiospecific, and give high product yields.  While there are a no. of reactions that fulfill the criteria, the Huisgen 1,3-dipolar cycloaddn. of azides and terminal alkynes has emerged as the frontrunner.  It has found applications in a wide variety of research areas, including materials sciences, polymer chem., and pharmaceutical sciences.  In this manuscript, important aspects of the Huisgen cycloaddn. will be reviewed, along with some of its many pharmaceutical applications.  Bioconjugation, nanoparticle surface modification, and pharmaceutical-related polymer chem. will all be covered.  Limitations of the reaction will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfV89MGuuCrVg90H21EOLACvtfcHk0lj43iPJVeO8Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFOru7%252FI&md5=7e0c89178299b2eb8a5b0099ac661a34</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs11095-008-9616-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-008-9616-1%26sid%3Dliteratum%253Aachs%26aulast%3DHein%26aufirst%3DC.%2BD.%26aulast%3DLiu%26aufirst%3DX.-M.%26aulast%3DWang%26aufirst%3DD.%26atitle%3DClick%2520Chemistry%252C%2520A%2520Powerful%2520Tool%2520for%2520Pharmaceutical%2520Sciences%26jtitle%3DPharm.%2520Res.%26date%3D2008%26volume%3D25%26spage%3D2216%26epage%3D2230%26doi%3D10.1007%2Fs11095-008-9616-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barral, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moorhouse, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moses, J. E.</span></span> <span> </span><span class="NLM_article-title">Efficient Conversion of Aromatic Amines into Azides: A One-Pot Synthesis of Triazole Linkages</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1809</span>– <span class="NLM_lpage">1811</span>, <span class="refDoi"> DOI: 10.1021/ol070527h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol070527h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsV2lsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=1809-1811&author=K.+Barralauthor=A.+D.+Moorhouseauthor=J.+E.+Moses&title=Efficient+Conversion+of+Aromatic+Amines+into+Azides%3A+A+One-Pot+Synthesis+of+Triazole+Linkages&doi=10.1021%2Fol070527h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Conversion of Aromatic Amines into Azides: A One-Pot Synthesis of Triazole Linkages</span></div><div class="casAuthors">Barral, Karine; Moorhouse, Adam D.; Moses, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1809-1811</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient and improved procedure for the prepn. of arom. azides and their application in the Cu(I)-catalyzed azide-alkyne 1,3-dipolar cycloaddn. ("click reaction") is described.  The synthesis of arom. azides from the corresponding amines is accomplished under mild conditions with tert-Bu nitrite and azidotrimethylsilane.  1,4-Disubstituted 1,2,3-triazoles, e.g., I, were obtained in excellent yields from a variety of arom. amines without the need for isolation of the azide intermediates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUlYdS6yIKhbVg90H21EOLACvtfcHk0lj43iPJVeO8Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsV2lsbk%253D&md5=9699b68ebaf593234ca9a5f45122ba49</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fol070527h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol070527h%26sid%3Dliteratum%253Aachs%26aulast%3DBarral%26aufirst%3DK.%26aulast%3DMoorhouse%26aufirst%3DA.%2BD.%26aulast%3DMoses%26aufirst%3DJ.%2BE.%26atitle%3DEfficient%2520Conversion%2520of%2520Aromatic%2520Amines%2520into%2520Azides%253A%2520A%2520One-Pot%2520Synthesis%2520of%2520Triazole%2520Linkages%26jtitle%3DOrg.%2520Lett.%26date%3D2007%26volume%3D9%26spage%3D1809%26epage%3D1811%26doi%3D10.1021%2Fol070527h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-A.</span></span> <span> </span><span class="NLM_article-title">Recent Development of Peptide Coupling Reagents in Organic Synthesis</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2447</span>– <span class="NLM_lpage">2467</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2004.01.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.tet.2004.01.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFeqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2447-2467&author=S.-Y.+Hanauthor=Y.-A.+Kim&title=Recent+Development+of+Peptide+Coupling+Reagents+in+Organic+Synthesis&doi=10.1016%2Fj.tet.2004.01.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Recent development of peptide coupling reagents in organic synthesis</span></div><div class="casAuthors">Han, So-Yeop; Kim, Young-Ah</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2447-2467</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  Various types of peptide coupling reagents (i.e., phosphonium, uronium, immonium, carbodiimide, imidazolium, organophosphorus, acid halogenating compds., chloroformate, pyridinium, etc.) are discussed for the synthesis of bioactive mols. contg. peptide linkages.  Methods used to suppress racemization during the peptide coupling step are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5MxkBBhDXubVg90H21EOLACvtfcHk0lhd7PmD3n73uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFeqsro%253D&md5=8c9ca02776710dd0e204262a85da630b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2004.01.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2004.01.020%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DS.-Y.%26aulast%3DKim%26aufirst%3DY.-A.%26atitle%3DRecent%2520Development%2520of%2520Peptide%2520Coupling%2520Reagents%2520in%2520Organic%2520Synthesis%26jtitle%3DTetrahedron%26date%3D2004%26volume%3D60%26spage%3D2447%26epage%3D2467%26doi%3D10.1016%2Fj.tet.2004.01.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T.</span></span> (<span class="NLM_year">2017</span>)  <i>BUSTER</i>, version 2.11.6. <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>: <span class="NLM_publisher-name">Global Phasing Ltd</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+Smart&author=C.+Vonrhein&author=T.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26date%3D2017%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhd7PmD3n73uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=487-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D487%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Automated Ligand Placement and Refinement with a Combined Force Field and Shape Potential</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1107/s0907444906016076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1107%2FS0907444906016076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=16790930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtVSnur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=741-749&author=S.+Wlodekauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Automated+Ligand+Placement+and+Refinement+with+a+Combined+Force+Field+and+Shape+Potential&doi=10.1107%2Fs0907444906016076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Automated ligand placement and refinement with a combined force field and shape potential</span></div><div class="casAuthors">Wlodek, S.; Skillman, A. G.; Nicholls, A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">741-749</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">An automated computational procedure for fitting a ligand into its electron d. with the use of the MMFF94 force field and a Gaussian shape description has been developed.  It employs a series of adiabatic optimizations of gradually increasing shape potential.  Starting from a set of energy-relaxed ligand conformations, the final results are structures realistically strained to fit the crystallog. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbAunEGTbprrVg90H21EOLACvtfcHk0ljSSBg67IF_2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtVSnur0%253D&md5=1571b77a211aec76fb4e3a376fc95d8a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444906016076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906016076%26sid%3Dliteratum%253Aachs%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DAutomated%2520Ligand%2520Placement%2520and%2520Refinement%2520with%2520a%2520Combined%2520Force%2520Field%2520and%2520Shape%2520Potential%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D741%26epage%3D749%26doi%3D10.1107%2Fs0907444906016076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rothschild, A. M.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Histamine Release by Compound 48/80</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.1970.tb10354.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1111%2Fj.1476-5381.1970.tb10354.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=4189829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADyaE3cXpvVyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1970&pages=253-262&author=A.+M.+Rothschild&title=Mechanisms+of+Histamine+Release+by+Compound+48%2F80&doi=10.1111%2Fj.1476-5381.1970.tb10354.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of histamine release by compound 48/80</span></div><div class="casAuthors">Rothschild, Adolfo M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-62</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    </div><div class="casAbstract">Rat and guinea-pig lung tissues were incubated for 20 min at 37° in Krebs-Ringer phosphate buffer at pH 7.4, or in Tyrode-Tris buffer at pH 8.2, and the release of histamine produced by adding different concns. of compound 48/80 (I) to the incubation medium was detd.  At pH 7.4, increasing concns. of I increased the release of histamine from the rat lung, with a tendency towards a max.  No release of histamine from guinea-pig lung was observed at this pH.  At pH 8.2, histamine release occurred both from rat and guinea-pig lung, and was proportional to the logarithm of the concn. of I.  Histamine release from rat lung by 20 μg/ml of I decreased when the pH was raised from 7.4 to 8.2; but the release caused by 1 mg/ml I increased both in rat and guinea-pig lung as the pH was raised.  2,4-Dinitrophenol (DNP) inhibited the release of histamine from rat lung by a concn. of 20 μg/ml of I; the inhibition was prevented by glucose.  DNP did not affect histamine release from rat or guinea-pig lung by a concn. of 1 mg/ml of I and enhanced the release when the pH was raised from 7.4 to 8.2.  One mg/ml of I did not inhibit the enhanced O consumption produced by DNP in the isolated rat diaphragm.  Iodoacetic acid (IAA) or a Ca/Mg-free medium inhibited the release of histamine by 20 μg/ml of I from rat lung but not the release produced by 1 mg/ml in either rat or guinea-pig lung.  The degranulation of rat mesentery mast cells caused by 20 μg/ml of I was inhibited by DNP.  The degranulation evoked by 1 mg/ml of I was also sensitive to this inhibitor; in this instance, however, the metachromatic staining reaction of the mesentery mast cells was greatly diminished.  Two processes of histamine release by I occur in rat lung.  One, dependent on cell metabolism, involves mast cell granule secretion.  The other, independent of cell metabolism, seems to consist of a simple exchange reaction between histamine and I, and this is the only one occurring in guinea-pig lung.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpphvfOc3RrrVg90H21EOLACvtfcHk0ljSSBg67IF_2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXpvVyrsw%253D%253D&md5=72d120593ddbe928757bf93bd223cb33</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.1970.tb10354.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.1970.tb10354.x%26sid%3Dliteratum%253Aachs%26aulast%3DRothschild%26aufirst%3DA.%2BM.%26atitle%3DMechanisms%2520of%2520Histamine%2520Release%2520by%2520Compound%252048%252F80%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1970%26volume%3D38%26spage%3D253%26epage%3D262%26doi%3D10.1111%2Fj.1476-5381.1970.tb10354.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Kévin Bourbiaux, Baptiste Legrand, Pascal Verdié, Sergio Mallart, Géraldine Manette, Claire Minoletti, J. David Stepp, Philippe Prigent, Jean-Christophe Le Bail, Laurence Gauzy-Lazo, Olivier Duclos, Jean Martinez, <span class="NLM_string-name hlFld-ContribAuthor">Muriel Amblard</span>. </span><span class="cited-content_cbyCitation_article-title">Potent Lys Patch-Containing Stapled Peptides Targeting PCSK9. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02051%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPotent%252BLys%252BPatch-Containing%252BStapled%252BPeptides%252BTargeting%252BPCSK9%26aulast%3DBourbiaux%26aufirst%3DK%25C3%25A9vin%26date%3D2021%26date%3D2020%26date%3D25112020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin J.  Tombling</span>, <span class="hlFld-ContribAuthor ">Carmen  Lammi</span>, <span class="hlFld-ContribAuthor ">Carlotta  Bollati</span>, <span class="hlFld-ContribAuthor ">Anna  Anoldi</span>, <span class="hlFld-ContribAuthor ">David J.  Craik</span>, <span class="hlFld-ContribAuthor ">Conan K.  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Increased Valency Improves Inhibitory Activity of Peptides Targeting Proprotein Convertase Subtilisin/Kexin Type 9 (PCSK9). </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (12)
                                     , 2154-2160. <a href="https://doi.org/10.1002/cbic.202100103" title="DOI URL">https://doi.org/10.1002/cbic.202100103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202100103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202100103%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DIncreased%252BValency%252BImproves%252BInhibitory%252BActivity%252Bof%252BPeptides%252BTargeting%252BProprotein%252BConvertase%252BSubtilisin%25252FKexin%252BType%252B9%252B%252528PCSK9%252529%26aulast%3DTombling%26aufirst%3DBenjamin%2BJ.%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D12%26spage%3D2154%26epage%3D2160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Benjamin J.  Tombling</span>, <span class="hlFld-ContribAuthor ">Yuhui  Zhang</span>, <span class="hlFld-ContribAuthor ">Yen-Hua  Huang</span>, <span class="hlFld-ContribAuthor ">David J.  Craik</span>, <span class="hlFld-ContribAuthor ">Conan K.  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">The emerging landscape of peptide-based inhibitors of PCSK9. </span><span class="cited-content_cbyCitation_journal-name">Atherosclerosis</span><span> <strong>2021,</strong> <em>37 </em><a href="https://doi.org/10.1016/j.atherosclerosis.2021.06.903" title="DOI URL">https://doi.org/10.1016/j.atherosclerosis.2021.06.903</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.atherosclerosis.2021.06.903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.atherosclerosis.2021.06.903%26sid%3Dliteratum%253Aachs%26jtitle%3DAtherosclerosis%26atitle%3DThe%252Bemerging%252Blandscape%252Bof%252Bpeptide-based%252Binhibitors%252Bof%252BPCSK9%26aulast%3DTombling%26aufirst%3DBenjamin%2BJ.%26date%3D2021%26volume%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Golnaz  Kamalinia</span>, <span class="hlFld-ContribAuthor ">Brian J.  Grindel</span>, <span class="hlFld-ContribAuthor ">Terry T.  Takahashi</span>, <span class="hlFld-ContribAuthor ">Steven W.  Millward</span>, <span class="hlFld-ContribAuthor ">Richard W.  Roberts</span>. </span><span class="cited-content_cbyCitation_article-title">Directing evolution of novel ligands by mRNA display. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2021,</strong> <em>537 </em><a href="https://doi.org/10.1039/D1CS00160D" title="DOI URL">https://doi.org/10.1039/D1CS00160D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CS00160D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CS00160D%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DDirecting%252Bevolution%252Bof%252Bnovel%252Bligands%252Bby%252BmRNA%252Bdisplay%26aulast%3DKamalinia%26aufirst%3DGolnaz%26date%3D2021%26date%3D2021%26volume%3D537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0037.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0037.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0003.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. mRNA display approach using Ra Pharma Platform.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Two lead series derived from mRNA display screening.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Early progression of SAR from original mRNA display lead <b>2</b>. Key structural changes are highlighted in red for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Primary metabolic sites on a representative molecule based on met ID studies with key proteases and mouse whole blood.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0020.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Approach to the Synthesis of Cyclic Peptides</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0007.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. X-ray crystal structure of 30 (salmon) bound to the LDLR-binding site of PCSK9 at 1.55 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIB">6XIB</a>). The molecule binds in an overall circular conformation, with 5F-Trp at position 2 outside of the doughnut and 5F-Trp at position 3 inside the macrocycle, with the fluorine atom in a small pocket on the surface. The side chains of position 1 (Lys), 5 (Asp), and 6 (His) are all solvent-exposed and do not directly interact with the protein. The side chain of position 7 (Tyr) lies in a small groove on the surface, and the aryl ring of the cross-linker lies directly on top of Ile 369.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0008.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Close-up of the shallow binding pocket occupied by 5F-Trp in position 4 of <b>30</b>. The pocket is defined by two hydrogen-bonded beta sheets, with the aryl ring and 5-fluroro substituent of the ligand fitting downward into the space between.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0009.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Schematic representation of the five key hydrogen bonding interactions between inhibitor <b>30</b> (salmon) and the PCSK9 active site β-sheet (white). The key interactions are described in detail in the text.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0010.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Continued SAR evolution of key lead molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0011.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. X-ray crystal structure of 40 (lt. yellow) bound to the LDLR binding site on the surface of PCSK9 at 1.38 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIC">6XIC</a>). The molecule binds in an identical manner as <b>30</b>, with the methyl group of the <span class="smallcaps smallerCapital">d</span>-Ala at position two (highlighted in the black box) pointing away from the protein toward the solvent. The <span class="smallcaps smallerCapital">d</span>-amino acid at this position also blocks proteolytic degradation at the two flanking amide bonds caused by trypsin and elastase activity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0012.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Pro-Ala and Pro-Thr as replacements for Asp-His at positions 5 and 6 of the macrocyclic peptide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0013.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Crystal structure of 51 (gray) bound to the LDLR-binding site on PCSK9 at 1.48 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XID">6XID</a>). The macrocyclic peptide binds similarly to the previously shown crystal structures, with the Pro-Thr providing an excellent replacement for the original Asp-His at positions 5 and 6. The Thr side chain hydroxyl is solvent-exposed and does not make any direct contact with the protein. The fluorine atom on the aryl ring is shown to occupy open space in the center of the macrocycle and may participate in and enhance the intramolecular interactions between the alpha-Me Pro and the inner 5F-Trp aromatic.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0021.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>62–63</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0022.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>66</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0014.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Proposed novel cross-link (green arrow) from bis-meta position of the DBX linker to the indole nitrogen of the 5F Trp in position 4.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0023.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0023.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <i>N</i>-Propargyl Fmoc-Protected 5F-Trp <b>70</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0024.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0024.jpeg" id="rightTab-gr22" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Arylazide Intermediate <b>73</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0024.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0025.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0025.jpeg" id="rightTab-gr23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Bicyclic Peptide <b>75</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0002.jpeg" id="rightTab-gr25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Retrosynthetic Analysis of the Proposed Bicyclic Target Peptide</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0015.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Proposed cyclization between the side chain of position 3 and position 5. The side chains point in the same direction and are in close proximity to each other, and the new bicyclic linker would lie in a mostly solvent-exposed region.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0016.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. SAR and evolution of bicyclic amide PCSK9 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0017.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Crystal structure of <b>78</b> bound to the LDLR-binding site of PCSK9 at 1.43 Angstrom resolution (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIE">6XIE</a>). The new bicyclic ring between positions 3 and 5 is clearly visible. The phenylalanine ring at position 3 occupies the same space as the fused ring system of 5F-Trp it replaces. Other than a slight twist of the DBX cross-linker aromatic ring, the rest of the molecule fits in a virtually identical manner to previous crystal structures in this series. The nitrogen of the amide bond between the inner 5F-Trp at position 4 and the glutamic acid at position 5 (indicated by the black box) remains exposed and vulnerable to chymotrypsin (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The crystal structure suggests that this nitrogen should be amenable to methylation because it points upward and away from the protein and does not participate in any interactions with the protein.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0018.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Superposition of the crystal structures of <b>30</b> (salmon) and <b>78</b> (green). The two molecules bind in a similar fashion, with all of the key hydrogen bonding and lipophilic interactions maintained. There is a slight twist in the DBX cross-linker (extra density here indicates likely some flexibility here). The new ring formed between the side chains of position 3 and position 5 is clearly visible, with the aryl ring sitting in a similar position as the indole of 5F-Trp and the newly formed cyclic amide solvent-exposed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="24" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0026.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0026.jpeg" id="rightTab-gr24" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Representative Synthesis of Lactam Analogues</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig17" data-index="25" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/medium/jm0c01084_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0019.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Crystal structure of <b>84</b> at 1.77 Angstrom resolution (green; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIF">6XIF</a>) overlaid with the crystal structure of <b>30</b> (salmon) bound to the LDLR binding site of PCSK9. The overall fit of the molecule is identical to all of the previous inhibitor structures described in this publication. The fluoroquinoline at position 4 is shown to occupy the same pocket in an identical manner as 5F-Trp in previous crystal structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-22/acs.jmedchem.0c01084/20201120/images/large/jm0c01084_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01084&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i75">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73087" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73087" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sediah, N.G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chretien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mbikay, M.</span></span> <span> </span><span class="NLM_article-title">PCSK9: A Key Modulator of Cardiovascular Health</span>. <i>Circ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">1022</span>– <span class="NLM_lpage">1036</span>, <span class="refDoi"> DOI: 10.1161/CIRCRESAHA.114.301621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1161%2FCIRCRESAHA.114.301621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=24625727" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=1022-1036&author=N.G.+Sediahauthor=Z.+Awanauthor=M.+Chretienauthor=M.+Mbikay&title=PCSK9%3A+A+Key+Modulator+of+Cardiovascular+Health&doi=10.1161%2FCIRCRESAHA.114.301621"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1161%2FCIRCRESAHA.114.301621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCRESAHA.114.301621%26sid%3Dliteratum%253Aachs%26aulast%3DSediah%26aufirst%3DN.G.%26aulast%3DAwan%26aufirst%3DZ.%26aulast%3DChretien%26aufirst%3DM.%26aulast%3DMbikay%26aufirst%3DM.%26atitle%3DPCSK9%253A%2520A%2520Key%2520Modulator%2520of%2520Cardiovascular%2520Health%26jtitle%3DCirc.%2520Res.%26date%3D2014%26volume%3D114%26spage%3D1022%26epage%3D1036%26doi%3D10.1161%2FCIRCRESAHA.114.301621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alings, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatt, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittner, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelberg, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanotin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jukema, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabor Kiss, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecorps, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahaffey, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moryusef, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pordy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tricoci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeiher, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steg, G.</span></span> <span> </span><span class="NLM_article-title">Alcirocumab Reduces Total Nonfatal Cardiovascular and Fatal Events</span>. <i>J. Am. Coll. Cardiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1016/j.jacc.2018.10.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.jacc.2018.10.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=389-396&author=M.+Szarekauthor=H.+D.+Whiteauthor=G.+G.+Schwartzauthor=M.+Alingsauthor=D.+L.+Bhattauthor=V.+A.+Bittnerauthor=C.+Chiangauthor=R.+Diazauthor=J.+M.+Edelbergauthor=S.+G.+Goodmanauthor=C.+Hanotinauthor=R.+A.+Harringtonauthor=J.+W.+Jukemaauthor=T.+Kimuraauthor=R.+Gabor+Kissauthor=G.+Lecorpsauthor=K.+W.+Mahaffeyauthor=A.+Moryusefauthor=R.+Pordyauthor=M.+T.+Roeauthor=P.+Tricociauthor=D.+Xavierauthor=A.+M.+Zeiherauthor=G.+Steg&title=Alcirocumab+Reduces+Total+Nonfatal+Cardiovascular+and+Fatal+Events&doi=10.1016%2Fj.jacc.2018.10.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.jacc.2018.10.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacc.2018.10.039%26sid%3Dliteratum%253Aachs%26aulast%3DSzarek%26aufirst%3DM.%26aulast%3DWhite%26aufirst%3DH.%2BD.%26aulast%3DSchwartz%26aufirst%3DG.%2BG.%26aulast%3DAlings%26aufirst%3DM.%26aulast%3DBhatt%26aufirst%3DD.%2BL.%26aulast%3DBittner%26aufirst%3DV.%2BA.%26aulast%3DChiang%26aufirst%3DC.%26aulast%3DDiaz%26aufirst%3DR.%26aulast%3DEdelberg%26aufirst%3DJ.%2BM.%26aulast%3DGoodman%26aufirst%3DS.%2BG.%26aulast%3DHanotin%26aufirst%3DC.%26aulast%3DHarrington%26aufirst%3DR.%2BA.%26aulast%3DJukema%26aufirst%3DJ.%2BW.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DGabor%2BKiss%26aufirst%3DR.%26aulast%3DLecorps%26aufirst%3DG.%26aulast%3DMahaffey%26aufirst%3DK.%2BW.%26aulast%3DMoryusef%26aufirst%3DA.%26aulast%3DPordy%26aufirst%3DR.%26aulast%3DRoe%26aufirst%3DM.%2BT.%26aulast%3DTricoci%26aufirst%3DP.%26aulast%3DXavier%26aufirst%3DD.%26aulast%3DZeiher%26aufirst%3DA.%2BM.%26aulast%3DSteg%26aufirst%3DG.%26atitle%3DAlcirocumab%2520Reduces%2520Total%2520Nonfatal%2520Cardiovascular%2520and%2520Fatal%2520Events%26jtitle%3DJ.%2520Am.%2520Coll.%2520Cardiol.%26date%3D2019%26volume%3D73%26spage%3D389%26epage%3D396%26doi%3D10.1016%2Fj.jacc.2018.10.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabatine, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giugliano, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keech, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honarpour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiviott, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuder, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sever, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, T. R.</span></span> <span> </span><span class="NLM_article-title">Evolocumab and Clinical Outcomes in Patients with Cardiovascular Disease</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>376</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1722</span>, <span class="refDoi"> DOI: 10.1056/nejmoa1615664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1056%2FNEJMoa1615664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=28304224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOns7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=376&publication_year=2017&pages=1713-1722&author=M.+S.+Sabatineauthor=R.+P.+Giuglianoauthor=A.+C.+Keechauthor=N.+Honarpourauthor=S.+D.+Wiviottauthor=S.+A.+Murphyauthor=J.+F.+Kuderauthor=H.+Wangauthor=T.+Liuauthor=S.+M.+Wassermanauthor=P.+S.+Severauthor=T.+R.+Pedersen&title=Evolocumab+and+Clinical+Outcomes+in+Patients+with+Cardiovascular+Disease&doi=10.1056%2Fnejmoa1615664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Evolocumab and clinical outcomes in patients with cardiovascular disease</span></div><div class="casAuthors">Sabatine, Marc S.; Giugliano, Robert P.; Keech, Anthony C.; Honarpour, Narimon; Wiviott, Stephen D.; Murphy, Sabina A.; Kuder, Julia F.; Wang, Huei; Liu, Thomas; Wasserman, Scott M.; Sever, Peter S.; Pedersen, Terje R.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">376</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1713-1722</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Evolocumab is a monoclonal antibody that inhibits proprotein convertase subtilisin-kexin type 9 (PCSK9) and lowers low-d. lipoprotein (LDL) cholesterol levels by approx. 60%.  Whether it prevents cardiovascular events is uncertain.  We conducted a randomized, double-blind, placebo-controlled trial involving 27,564 patients with atherosclerotic cardiovascular disease and LDL cholesterol levels of 70 mg per dL (1.8 mmol per L) or higher who were receiving statin therapy.  Patients were randomly assigned to receive evolocumab (either 140 mg every 2 wk or 420 mg monthly) or matching placebo as s.c. injections.  The primary efficacy end point was the composite of cardiovascular death, myocardial infarction, stroke, hospitalization for unstable angina, or coronary revascularization.  The key secondary efficacy end point was the composite of cardiovascular death, myocardial infarction, or stroke.  The median duration of follow-up was 2.2 years.  At 48 wk, the least-squares mean percentage redn. in LDL cholesterol levels with evolocumab, as compared with placebo, was 59%, from a median baseline value of 92 mg per dL (2.4 mmol per L) to 30 mg per dL (0.78 mmol per L) (P<0.001).  Relative to placebo, evolocumab treatment significantly reduced the risk of the primary end point (1344 patients [9.8%] vs. 1563 patients [11.3%]; hazard ratio, 0.85; 95% confidence interval [CI], 0.79 to 0.92; P<0.001) and the key secondary end point (816 [5.9%] vs. 1013 [7.4%]; hazard ratio, 0.80; 95% CI, 0.73 to 0.88; P<0.001).  The results were consistent across key subgroups, including the subgroup of patients in the lowest quartile for baseline LDL cholesterol levels (median, 74 mg per dL [1.9 mmol per L]).  There was no significant difference between the study groups with regard to adverse events (including new-onset diabetes and neurocognitive events), with the exception of injection-site reactions, which were more common with evolocumab (2.1% vs. 1.6%).  In our trial, inhibition of PCSK9 with evolocumab on a background of statin therapy lowered LDL cholesterol levels to a median of 30 mg per dL (0.78 mmol per L) and reduced the risk of cardiovascular events.  These findings show that patients with atherosclerotic cardiovascular disease benefit from lowering of LDL cholesterol levels below current targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBM3VQXU81RLVg90H21EOLACvtfcHk0lgy1div7iQ4Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOns7nM&md5=8995d48576066c8b9d1cfb3446de270e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1615664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1615664%26sid%3Dliteratum%253Aachs%26aulast%3DSabatine%26aufirst%3DM.%2BS.%26aulast%3DGiugliano%26aufirst%3DR.%2BP.%26aulast%3DKeech%26aufirst%3DA.%2BC.%26aulast%3DHonarpour%26aufirst%3DN.%26aulast%3DWiviott%26aufirst%3DS.%2BD.%26aulast%3DMurphy%26aufirst%3DS.%2BA.%26aulast%3DKuder%26aufirst%3DJ.%2BF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DWasserman%26aufirst%3DS.%2BM.%26aulast%3DSever%26aufirst%3DP.%2BS.%26aulast%3DPedersen%26aufirst%3DT.%2BR.%26atitle%3DEvolocumab%2520and%2520Clinical%2520Outcomes%2520in%2520Patients%2520with%2520Cardiovascular%2520Disease%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2017%26volume%3D376%26spage%3D1713%26epage%3D1722%26doi%3D10.1056%2Fnejmoa1615664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nasir, K.</span></span> <span> </span><span class="NLM_article-title">Just Price for PCSK9 Inhibitors: No Less, No More</span>. <i>J. Am. Heart Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e010884</span> <span class="refDoi"> DOI: 10.1161/jaha.118.010884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1161%2FJAHA.118.010884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30571392" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&author=K.+Nasir&title=Just+Price+for+PCSK9+Inhibitors%3A+No+Less%2C+No+More&doi=10.1161%2Fjaha.118.010884"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1161%2FJAHA.118.010884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FJAHA.118.010884%26sid%3Dliteratum%253Aachs%26aulast%3DNasir%26aufirst%3DK.%26atitle%3DJust%2520Price%2520for%2520PCSK9%2520Inhibitors%253A%2520No%2520Less%252C%2520No%2520More%26jtitle%3DJ.%2520Am.%2520Heart%2520Assoc.%26date%3D2018%26volume%3D7%26doi%3D10.1161%2Fjaha.118.010884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krähenbühl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavik-Mezzour, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Eckardstein, A.</span></span> <span> </span><span class="NLM_article-title">Unmet Needs in LDL-C Lowering: When Statins Won’t Do!</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">1175</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1007/s40265-016-0613-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1007%2Fs40265-016-0613-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=27456066" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=1175-1190&author=S.+Kr%C3%A4henb%C3%BChlauthor=I.+Pavik-Mezzourauthor=A.+von+Eckardstein&title=Unmet+Needs+in+LDL-C+Lowering%3A+When+Statins+Won%E2%80%99t+Do%21&doi=10.1007%2Fs40265-016-0613-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs40265-016-0613-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-016-0613-0%26sid%3Dliteratum%253Aachs%26aulast%3DKr%25C3%25A4henb%25C3%25BChl%26aufirst%3DS.%26aulast%3DPavik-Mezzour%26aufirst%3DI.%26aulast%3Dvon%2BEckardstein%26aufirst%3DA.%26atitle%3DUnmet%2520Needs%2520in%2520LDL-C%2520Lowering%253A%2520When%2520Statins%2520Won%25E2%2580%2599t%2520Do%2521%26jtitle%3DDrugs%26date%3D2016%26volume%3D76%26spage%3D1175%26epage%3D1190%26doi%3D10.1007%2Fs40265-016-0613-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akyea, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qureshi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, S. F.</span></span> <span> </span><span class="NLM_article-title">Sub-optimal Cholesterol Response to Initiation of Statins and Future Risk of Cardiovascular Disease</span>. <i>Heart</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">975</span>– <span class="NLM_lpage">981</span>, <span class="refDoi"> DOI: 10.1136/heartjnl-2018-314253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1136%2Fheartjnl-2018-314253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30988003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFyqurfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2019&pages=975-981&author=R.+K.+Akyeaauthor=J.+Kaiauthor=N.+Qureshiauthor=B.+Iyenauthor=S.+F.+Weng&title=Sub-optimal+Cholesterol+Response+to+Initiation+of+Statins+and+Future+Risk+of+Cardiovascular+Disease&doi=10.1136%2Fheartjnl-2018-314253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Sub-optimal cholesterol response to initiation of statins and future risk of cardiovascular disease</span></div><div class="casAuthors">Akyea, Ralph Kwame; Kai, Joe; Qureshi, Nadeem; Iyen, Barbara; Weng, Stephen F.</div><div class="citationInfo"><span class="NLM_cas:title">Heart (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">975-981</span>CODEN:
                <span class="NLM_cas:coden">HEARFR</span>;
        ISSN:<span class="NLM_cas:issn">1355-6037</span>.
    
            (<span class="NLM_cas:orgname">BMJ</span>)
        </div><div class="casAbstract">Objective To assess low-d. lipoprotein cholesterol (LDL-C) response in patients after initiation of statins, and future risk of cardiovascular disease (CVD).  Methods Prospective cohort study of 165 411 primary care patients, from the UK Clin. Practice Research Datalink, who were free of CVD before statin initiation, and had at least one pre-treatment LDL-C within 12 mo before, and one post-treatment LDL-C within 24 mo after, statin initiation.  Based on current national guidelines, <40% redn. in baseline LDL-C within 24 mo was classified as a sub-optimal statin response.  Cox proportional regression and competing-risks survival regression models were used to det. adjusted hazard ratios (HRs) and sub-HRs for incident CVD outcomes for LDL-C response to statins.  Results 84 609 (51.2%) patients had a sub-optimal LDL-C response to initiated statin therapy within 24 mo.  During 1 077 299 person-years of follow-up (median follow-up 6.2 years), there were 22 798 CVD events (12 142 in sub-optimal responders and 10 656 in optimal responders).  In sub-optimal responders, compared with optimal responders, the HR for incident CVD was 1.17 (95% CI 1.13 to 1.20) and 1.22 (95% CI 1.19 to 1.25) after adjusting for age and baseline untreated LDL-C.  Considering competing risks resulted in lower but similar sub-HRs for both unadjusted (1.13, 95% CI 1.10 to 1.16) and adjusted (1.19, 95% CI 1.16 to 1.23) cumulative incidence function of CVD.  Conclusions Optimal lowering of LDL-C is not achieved within 2 years in over half of patients in the general population initiated on statin therapy, and these patients will experience significantly increased risk of future CVD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi4gmHe-gcU7Vg90H21EOLACvtfcHk0lgCN2cqxP_MdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyqurfL&md5=bc9f498ce08e854acbb8fab25e63f722</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1136%2Fheartjnl-2018-314253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fheartjnl-2018-314253%26sid%3Dliteratum%253Aachs%26aulast%3DAkyea%26aufirst%3DR.%2BK.%26aulast%3DKai%26aufirst%3DJ.%26aulast%3DQureshi%26aufirst%3DN.%26aulast%3DIyen%26aufirst%3DB.%26aulast%3DWeng%26aufirst%3DS.%2BF.%26atitle%3DSub-optimal%2520Cholesterol%2520Response%2520to%2520Initiation%2520of%2520Statins%2520and%2520Future%2520Risk%2520of%2520Cardiovascular%2520Disease%26jtitle%3DHeart%26date%3D2019%26volume%3D105%26spage%3D975%26epage%3D981%26doi%3D10.1136%2Fheartjnl-2018-314253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, R. O.</span></span> <span> </span><span class="NLM_article-title">A Two-step Binding Model of PCSK9 Interaction with the Low-Density Lipoprotein Receptor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">5464</span>– <span class="NLM_lpage">5470</span>, <span class="refDoi"> DOI: 10.1074/jbc.m110.199042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1074%2Fjbc.M110.199042" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=21149300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvVCjsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=5464-5470&author=T.+Yamamotoauthor=C.+Luauthor=R.+O.+Ryan&title=A+Two-step+Binding+Model+of+PCSK9+Interaction+with+the+Low-Density+Lipoprotein+Receptor&doi=10.1074%2Fjbc.m110.199042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Two-step Binding Model of PCSK9 Interaction with the Low Density Lipoprotein Receptor</span></div><div class="casAuthors">Yamamoto, Taichi; Lu, Christine; Ryan, Robert O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5464-5470</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">PCSK9 (proprotein convertase subtilisin-like/kexin type 9) is an emerging target for pharmaceutical intervention.  This multidomain protein interacts with the LDL receptor (LDLR), promoting receptor degrdn.  Insofar as PCSK9 inhibition induces a decrease in plasma cholesterol levels, understanding the nature of the binding interaction between PCSK9 and the LDLR is of crit. importance.  In this study, the ability of PCSK9 to compete with apoE3 N-terminal domain-contg. reconstituted HDL for receptor binding was examd.  Whereas full-length PCSK9 was an effective competitor, the N-terminal Pro-Cat domain (composed of the prodomain and catalytic domain) was not.  Surprisingly, the C-terminal domain (CT domain) of PCSK9 was able to compete.  Using a direct binding interaction assay, we show that the PCSK9 CT domain bound to the LDLR in a calcium-dependent manner and that co-incubation with the prodomain and catalytic domain had no effect on this binding.  To further characterize this interaction, two LDLR fragments, the classical ligand-binding domain (LBD) and the EGF precursor homol. domain, were expressed in stably transfected HEK 293 cells and isolated.  Binding assays showed that the PCSK9 CT domain bound to the LBD at pH 5.4.  Thus, CT domain interaction with the LBD of the LDLR at endosomal pH constitutes a second step in the PCSK9-mediated LDLR binding that leads to receptor degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo69xyoaQQ1A7Vg90H21EOLACvtfcHk0lgCN2cqxP_MdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvVCjsrY%253D&md5=62aa5b98989d592f72de4b2e7b77074b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.199042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.199042%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DRyan%26aufirst%3DR.%2BO.%26atitle%3DA%2520Two-step%2520Binding%2520Model%2520of%2520PCSK9%2520Interaction%2520with%2520the%2520Low-Density%2520Lipoprotein%2520Receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D5464%26epage%3D5470%26doi%3D10.1074%2Fjbc.m110.199042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taechalertpaisarn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of the PCSK9-LDLR Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3242</span>– <span class="NLM_lpage">3249</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVGlt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=3242-3249&author=J.+Taechalertpaisarnauthor=B.+Zhaoauthor=X.+Liangauthor=K.+Burgess&title=Small+Molecule+Inhibitors+of+the+PCSK9-LDLR+Interaction&doi=10.1021%2Fjacs.7b09360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of the PCSK9·LDLR Interaction</span></div><div class="casAuthors">Taechalertpaisarn, Jaru; Zhao, Bosheng; Liang, Xiaowen; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3242-3249</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The protein-protein interaction between proprotein convertase subtilisin/kexin type 9 (PCSK9) and low-d. lipoprotein receptor (LDLR) is a relatively new, and extremely important, validated therapeutic target for treatment and prevention of heart disease.  Experts in the area agree that the first small mols. to disrupt PCSK9•LDLR would represent a milestone in the field, yet no credible leads have been reported.  This paper describes how side-chain orientations in preferred conformations of carefully designed chemotypes were compared with LDLR side-chains at the PCSK9•LDLR interface to find mols. that would mimic interface regions of LDLR.  This approach is an example of the procedure called EKO (Exploring Key Orientations).  The guiding hypothesis on which EKO is based is that good matches indicate the chemotypes bearing the same side-chains as the protein at the sites of overlay have the potential to disrupt the parent protein-protein interaction (PPI).  In the event, the EKO procedure and one round of combinatorial fragment-based virtual docking, led to the discovery of seven compds. that bound PCSK9 (SPR and ELISA) and had a favorable outcome in a cellular assay (hepatocyte uptake of fluorescently labeled LDL particles) and increased the expression LDLR on hepatocytes in culture.  Three promising hit compds. in this series had dissocn. consts. for PCSK9 binding in the 20-40 μM range, and one of these was modified with a photoaffinity label and shown to form a covalent conjugate with PCSK9 on photolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4QTYDzFsr9bVg90H21EOLACvtfcHk0liyXho7-ArygA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVGlt7Y%253D&md5=488cbeef333e4bcd13c78481c5745cc5</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09360%26sid%3Dliteratum%253Aachs%26aulast%3DTaechalertpaisarn%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520the%2520PCSK9-LDLR%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D3242%26epage%3D3249%26doi%3D10.1021%2Fjacs.7b09360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stucchi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grazioso, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanoni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvani, A.</span></span> <span> </span><span class="NLM_article-title">Disrupting he PCSK9/LDLR Protein-Protein Interaction by an Imidazole-Based Minimalist Peptidomimetic</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">9736</span>– <span class="NLM_lpage">9740</span>, <span class="refDoi"> DOI: 10.1039/c6ob01642a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1039%2FC6OB01642A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=27722650" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=9736-9740&author=M.+Stucchiauthor=G.+Graziosoauthor=C.+Lammiauthor=S.+Manaraauthor=C.+Zanoniauthor=A.+Arnoldiauthor=G.+Lesmaauthor=A.+Silvani&title=Disrupting+he+PCSK9%2FLDLR+Protein-Protein+Interaction+by+an+Imidazole-Based+Minimalist+Peptidomimetic&doi=10.1039%2Fc6ob01642a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the PCSK9/LDLR protein-protein interaction by an imidazole-based minimalist peptidomimetic</span></div><div class="casAuthors">Stucchi, Mattia; Grazioso, Giovanni; Lammi, Carmen; Manara, Silvia; Zanoni, Chiara; Arnoldi, Anna; Lesma, Giordano; Silvani, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">9736-9740</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein we report on the multicomponent synthesis of a novel imidazole-based compd., able to act efficiently as a minimalist β-strand mimic.  Biol. evaluation proved its ability to impair the LDLR-PCSK9 protein-protein interaction, disclosing it as the first small mol. exerting a PCSK9-mediated hypocholesterolemic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVtgyGhfHK_bVg90H21EOLACvtfcHk0liyXho7-ArygA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOrsLrP&md5=015606884b7a48349de93c76a160f96b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1039%2FC6OB01642A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01642A%26sid%3Dliteratum%253Aachs%26aulast%3DStucchi%26aufirst%3DM.%26aulast%3DGrazioso%26aufirst%3DG.%26aulast%3DLammi%26aufirst%3DC.%26aulast%3DManara%26aufirst%3DS.%26aulast%3DZanoni%26aufirst%3DC.%26aulast%3DArnoldi%26aufirst%3DA.%26aulast%3DLesma%26aufirst%3DG.%26aulast%3DSilvani%26aufirst%3DA.%26atitle%3DDisrupting%2520he%2520PCSK9%252FLDLR%2520Protein-Protein%2520Interaction%2520by%2520an%2520Imidazole-Based%2520Minimalist%2520Peptidomimetic%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D9736%26epage%3D9740%26doi%3D10.1039%2Fc6ob01642a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Londregan, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lintner, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dullea, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolt, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genovino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thuma, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesp, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aspnes, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reidich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salatto, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chabot, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cate, J. H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, D. W.</span></span> <span> </span><span class="NLM_article-title">Small Molecule PCSK9 Inhibitors: Hit to Lead Optimization of Systemic Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5704</span>– <span class="NLM_lpage">5718</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00650</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00650" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5704-5718&author=A.+T.+Londreganauthor=L.+Weiauthor=J.+Xiaoauthor=N.+G.+Lintnerauthor=D.+Petersenauthor=R.+G.+Dulleaauthor=K.+F.+McClureauthor=M.+W.+Boltauthor=J.+S.+Warmusauthor=S.+B.+Coffeyauthor=C.+Limberakisauthor=J.+Genovinoauthor=B.+A.+Thumaauthor=K.+D.+Hespauthor=G.+E.+Aspnesauthor=B.+Reidichauthor=C.+T.+Salattoauthor=J.+R.+Chabotauthor=J.+H.+D.+Cateauthor=S.+Lirasauthor=D.+W.+Piotrowski&title=Small+Molecule+PCSK9+Inhibitors%3A+Hit+to+Lead+Optimization+of+Systemic+Agents&doi=10.1021%2Facs.jmedchem.8b00650"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Proprotein Convertase Subtilisin/Kexin Type 9 (PCSK9) Inhibitors: Hit to Lead Optimization of Systemic Agents</span></div><div class="casAuthors">Londregan, Allyn T.; Wei, Liuqing; Xiao, Jun; Lintner, Nathanael G.; Petersen, Donna; Dullea, Robert G.; McClure, Kim F.; Bolt, Michael W.; Warmus, Joseph S.; Coffey, Steven B.; Limberakis, Chris; Genovino, Julien; Thuma, Benjamin A.; Hesp, Kevin D.; Aspnes, Gary E.; Reidich, Benjamin; Salatto, Christopher T.; Chabot, Jeffrey R.; Cate, Jamie H. D.; Liras, Spiros; Piotrowski, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5704-5718</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The optimization of a new class of small mol. PCSK9 mRNA translation inhibitors is described.  The potency, physicochem. properties, and off-target pharmacol. assocd. with the hit compd. (1) were improved by changes to two regions of the mol.  The last step in the synthesis of the congested amide center was enabled by three different routes.  Subtle structural changes yielded significant changes in pharmacol. and off-target margins.  These efforts led to the identification of 7l and 7n with overall profiles suitable for in vivo evaluation.  In a 14-day toxicol. study, 7l demonstrated an improved safety profile vs lead 7f.  We hypothesize that the improved safety profile is related to diminished binding of 7l to nontranslating ribosomes and an apparent improvement in transcript selectivity due to the lower strength of 7l stalling of off-target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj72SuB2HcJbVg90H21EOLACvtfcHk0ljda4aNH3XaOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyltbnI&md5=14f6b537fdf7c0684790c6d663c5e3f3</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00650&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00650%26sid%3Dliteratum%253Aachs%26aulast%3DLondregan%26aufirst%3DA.%2BT.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DLintner%26aufirst%3DN.%2BG.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DDullea%26aufirst%3DR.%2BG.%26aulast%3DMcClure%26aufirst%3DK.%2BF.%26aulast%3DBolt%26aufirst%3DM.%2BW.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26aulast%3DCoffey%26aufirst%3DS.%2BB.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DGenovino%26aufirst%3DJ.%26aulast%3DThuma%26aufirst%3DB.%2BA.%26aulast%3DHesp%26aufirst%3DK.%2BD.%26aulast%3DAspnes%26aufirst%3DG.%2BE.%26aulast%3DReidich%26aufirst%3DB.%26aulast%3DSalatto%26aufirst%3DC.%2BT.%26aulast%3DChabot%26aufirst%3DJ.%2BR.%26aulast%3DCate%26aufirst%3DJ.%2BH.%2BD.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPiotrowski%26aufirst%3DD.%2BW.%26atitle%3DSmall%2520Molecule%2520PCSK9%2520Inhibitors%253A%2520Hit%2520to%2520Lead%2520Optimization%2520of%2520Systemic%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5704%26epage%3D5718%26doi%3D10.1021%2Facs.jmedchem.8b00650" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrilli, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdmann, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abeywickrema, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agnani, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baysarowich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNunzio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lisnock, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palyha, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salowe, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strack, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youm, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zink, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zokian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Addona, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akinsanya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tata, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbriglio, J. E.</span></span> <span> </span><span class="NLM_article-title">From Screening to Targeted Degredation: Strategies for the Discovery and Optimization of Small Molecule Ligands for PCSK9</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">32</span>– <span class="NLM_lpage">40.e3</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.chembiol.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=31653597" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=32-40.e3&author=W.+L.+Petrilliauthor=G.+C.+Adamauthor=R.+S.+Erdmannauthor=P.+Abeywickremaauthor=V.+Agnaniauthor=X.+Aiauthor=J.+Baysarowichauthor=N.+Byrneauthor=J.+P.+Caldwellauthor=W.+Changauthor=E.+DiNunzioauthor=Z.+Fengauthor=R.+Fordauthor=S.+Haauthor=Y.+Huangauthor=B.+Hubbardauthor=J.+J.+Johnstonauthor=M.+Kavanaauthor=J.-M.+Lisnockauthor=R.+Liangauthor=J.+Luauthor=J.+Mengauthor=P.+Orthauthor=O.+Palyhaauthor=G.+Parthasarathyauthor=S.+P.+Saloweauthor=S.+Sharmaauthor=J.+Shipmanauthor=S.+M.+Soissonauthor=A.+M.+Strackauthor=H.+Youmauthor=K.+Zhaoauthor=D.+L.+Zinkauthor=H.+Zokianauthor=G.+H.+Addonaauthor=K.+Akinsanyaauthor=J.+R.+Tataauthor=Y.+Xiongauthor=J.+E.+Imbriglio&title=From+Screening+to+Targeted+Degredation%3A+Strategies+for+the+Discovery+and+Optimization+of+Small+Molecule+Ligands+for+PCSK9&doi=10.1016%2Fj.chembiol.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DPetrilli%26aufirst%3DW.%2BL.%26aulast%3DAdam%26aufirst%3DG.%2BC.%26aulast%3DErdmann%26aufirst%3DR.%2BS.%26aulast%3DAbeywickrema%26aufirst%3DP.%26aulast%3DAgnani%26aufirst%3DV.%26aulast%3DAi%26aufirst%3DX.%26aulast%3DBaysarowich%26aufirst%3DJ.%26aulast%3DByrne%26aufirst%3DN.%26aulast%3DCaldwell%26aufirst%3DJ.%2BP.%26aulast%3DChang%26aufirst%3DW.%26aulast%3DDiNunzio%26aufirst%3DE.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DFord%26aufirst%3DR.%26aulast%3DHa%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DHubbard%26aufirst%3DB.%26aulast%3DJohnston%26aufirst%3DJ.%2BJ.%26aulast%3DKavana%26aufirst%3DM.%26aulast%3DLisnock%26aufirst%3DJ.-M.%26aulast%3DLiang%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DPalyha%26aufirst%3DO.%26aulast%3DParthasarathy%26aufirst%3DG.%26aulast%3DSalowe%26aufirst%3DS.%2BP.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DShipman%26aufirst%3DJ.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DStrack%26aufirst%3DA.%2BM.%26aulast%3DYoum%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DZink%26aufirst%3DD.%2BL.%26aulast%3DZokian%26aufirst%3DH.%26aulast%3DAddona%26aufirst%3DG.%2BH.%26aulast%3DAkinsanya%26aufirst%3DK.%26aulast%3DTata%26aufirst%3DJ.%2BR.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DImbriglio%26aufirst%3DJ.%2BE.%26atitle%3DFrom%2520Screening%2520to%2520Targeted%2520Degredation%253A%2520Strategies%2520for%2520the%2520Discovery%2520and%2520Optimization%2520of%2520Small%2520Molecule%2520Ligands%2520for%2520PCSK9%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D27%26spage%3D32%26epage%3D40.e3%26doi%3D10.1016%2Fj.chembiol.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong-Beltran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchhofer, D.</span></span> <span> </span><span class="NLM_article-title">Identification of a Small Peptide that Inhibits PCSK9 Protein Binding to the Low-Density Lipoprotein Receptor</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">942</span>– <span class="NLM_lpage">955</span>, <span class="refDoi"> DOI: 10.1074/jbc.m113.514067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1074%2Fjbc.M113.514067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=24225950" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2013&pages=942-955&author=Y.+Zhangauthor=C.+Eigenbrotauthor=L.+Zhouauthor=S.+Shiaauthor=W.+Liauthor=C.+Quanauthor=J.+Tomauthor=P.+Moranauthor=P.+Di+Lelloauthor=N.+J.+Skeltonauthor=M.+Kong-Beltranauthor=A.+Petersonauthor=D.+Kirchhofer&title=Identification+of+a+Small+Peptide+that+Inhibits+PCSK9+Protein+Binding+to+the+Low-Density+Lipoprotein+Receptor&doi=10.1074%2Fjbc.m113.514067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.514067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.514067%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DShia%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DQuan%26aufirst%3DC.%26aulast%3DTom%26aufirst%3DJ.%26aulast%3DMoran%26aufirst%3DP.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DKong-Beltran%26aufirst%3DM.%26aulast%3DPeterson%26aufirst%3DA.%26aulast%3DKirchhofer%26aufirst%3DD.%26atitle%3DIdentification%2520of%2520a%2520Small%2520Peptide%2520that%2520Inhibits%2520PCSK9%2520Protein%2520Binding%2520to%2520the%2520Low-Density%2520Lipoprotein%2520Receptor%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D289%26spage%3D942%26epage%3D955%26doi%3D10.1074%2Fjbc.m113.514067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ultsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burdick, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong-Beltran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchhofer, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Cryptic Peptide-Binding site on PCSK9 and Design of Antagonists</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">848</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3453</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1038%2Fnsmb.3453" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=28825733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=848-856&author=Y.+Zhangauthor=M.+Ultschauthor=N.+J.+Skeltonauthor=D.+J.+Burdickauthor=M.+H.+Beresiniauthor=W.+Liauthor=M.+Kong-Beltranauthor=A.+Petersonauthor=J.+Quinnauthor=C.+Chiuauthor=Y.+Wuauthor=S.+Shiaauthor=P.+Moranauthor=P.+Di+Lelloauthor=C.+Eigenbrotauthor=D.+Kirchhofer&title=Discovery+of+a+Cryptic+Peptide-Binding+site+on+PCSK9+and+Design+of+Antagonists&doi=10.1038%2Fnsmb.3453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a cryptic peptide-binding site on PCSK9 and design of antagonists</span></div><div class="casAuthors">Zhang, Yingnan; Ultsch, Mark; Skelton, Nicholas J.; Burdick, Daniel J.; Beresini, Maureen H.; Li, Wei; Kong-Beltran, Monica; Peterson, Andrew; Quinn, John; Chiu, Cecilia; Wu, Yan; Shia, Steven; Moran, Paul; Di Lello, Paola; Eigenbrot, Charles; Kirchhofer, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">848-856</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Proprotein convertase subtilisin/kexin type 9 (PCSK9) regulates plasma LDL cholesterol (LDL-c) levels by promoting the degrdn. of liver LDL receptors (LDLRs).  Antibodies that inhibit PCSK9 binding to the EGF(A) domain of the LDLR are effective in lowering LDL-c.  However, the discovery of small-mol. therapeutics is hampered by difficulty in targeting the relatively flat EGF(A)-binding site on PCSK9.  Here we demonstrate that it is possible to target this site, based on the finding that the PCSK9 P' helix displays conformational flexibility.  As a consequence, the vacated N-terminal groove of PCSK9, which is adjacent to the EGF(A)-binding site, is in fact accessible to small peptides.  In phage-display expts., the EGF(A)-mimicking peptide Pep2-8 was used as an anchor peptide for the attachment of an extension peptide library directed toward the groove site.  Guided by structural information, we further engineered the identified groove-binding peptides into antagonists, which encroach on the EGF(A)-binding site and inhibit LDLR binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC8qO6Y3Nck7Vg90H21EOLACvtfcHk0lhEgBWBPFz5Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbjO&md5=1f449068f5745918cd8b7e0b7d7dd6ae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3453%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DUltsch%26aufirst%3DM.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DBurdick%26aufirst%3DD.%2BJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DKong-Beltran%26aufirst%3DM.%26aulast%3DPeterson%26aufirst%3DA.%26aulast%3DQuinn%26aufirst%3DJ.%26aulast%3DChiu%26aufirst%3DC.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DShia%26aufirst%3DS.%26aulast%3DMoran%26aufirst%3DP.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DKirchhofer%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520a%2520Cryptic%2520Peptide-Binding%2520site%2520on%2520PCSK9%2520and%2520Design%2520of%2520Antagonists%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D848%26epage%3D856%26doi%3D10.1038%2Fnsmb.3453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Inhibitors of Protein – Protein Interactions: Mimicking Peptide Binding Epitopes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8896</span>– <span class="NLM_lpage">8927</span>, <span class="refDoi"> DOI: 10.1002/anie.201412070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1002%2Fanie.201412070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=26119925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=8896-8927&author=M.+Pelay-Gimenoauthor=A.+Glasauthor=O.+Kochauthor=T.+N.+Grossmann&title=Structure-Based+Design+of+Inhibitors+of+Protein+%E2%80%93+Protein+Interactions%3A+Mimicking+Peptide+Binding+Epitopes&doi=10.1002%2Fanie.201412070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Inhibitors of Protein-Protein Interactions: Mimicking Peptide Binding Epitopes</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Glas, Adrian; Koch, Oliver; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">8896-8927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are involved at all levels of cellular organization, thus making the development of PPI inhibitors extremely valuable.  The identification of selective inhibitors is challenging because of the shallow and extended nature of PPI interfaces.  Inhibitors can be obtained by mimicking peptide binding epitopes in their bioactive conformation.  For this purpose, several strategies have been evolved to enable a projection of side chain functionalities in analogy to peptide secondary structures, thereby yielding mols. that are generally referred to as peptidomimetics.  Herein, we introduce a new classification of peptidomimetics (classes A-D) that enables a clear assignment of available approaches.  Based on this classification, the Review summarizes strategies that have been applied for the structure-based design of PPI inhibitors through stabilizing or mimicking turns, β-sheets, and helixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPTEmZpatXbrVg90H21EOLACvtfcHk0lhEgBWBPFz5Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM&md5=447c84aab6275b1057fda9a610918f2b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412070%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DO.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DStructure-Based%2520Design%2520of%2520Inhibitors%2520of%2520Protein%2520%25E2%2580%2593%2520Protein%2520Interactions%253A%2520Mimicking%2520Peptide%2520Binding%2520Epitopes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D8896%26epage%3D8927%26doi%3D10.1002%2Fanie.201412070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dyrbuś, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gąsior, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, M.</span></span> <span> </span><span class="NLM_article-title">Inclisiran – New Hope in the Management of Lipid Disorders</span>. <i>J. Clin. Lipidol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.jacl.2019.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.jacl.2019.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=31879073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A280%3ADC%252BB3Mbkt1Kksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2020&pages=16-27&author=K.+Dyrbu%C5%9Bauthor=M.+G%C4%85siorauthor=P.+Pensonauthor=K.+K.+Rayauthor=M.+Banach&title=Inclisiran+%E2%80%93+New+Hope+in+the+Management+of+Lipid+Disorders&doi=10.1016%2Fj.jacl.2019.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inclisiran-New hope in the management of lipid disorders?</span></div><div class="casAuthors">Dyrbus Krzysztof; GAsior Mariusz; Penson Peter; Ray Kausik K; Banach Maciej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical lipidology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-27</span>
        ISSN:<span class="NLM_cas:issn">1933-2874</span>.
    </div><div class="casAbstract">Drugs reducing plasma concentrations of apolipoprotein B-containing lipoproteins have been demonstrated to reduce the risk of cardiovascular disease (CVD) in both primary and secondary prevention.  Despite the demonstrated efficacy of statins and ezetimibe on low-density lipoprotein (LDL) concentration and long-term CVD risk, a large number of patients do not achieve their therapeutic goals.  The introduction of monoclonal antibodies against proprotein convertase subtilisin/kexin type 9 (PCSK9) protein was a milestone in the treatment of lipid disorders, as their administration leads to unprecedentedly low LDL cholesterol concentrations.  Inclisiran represents an entirely new mechanism of PSCK9 protein inhibition in hepatocytes, targeting the messenger RNA for PCSK9.  Its administration is necessary only every 3 to 6 months, which is an essential advantage over statin and monoclonal antibody therapy.  The infrequent administration regimen can increase the number of patients who maintain their therapeutic goals, especially in patients struggling to comply with daily or biweekly pharmacotherapy.  Preclinical studies and Phase I and Phase II clinical trials of inclisiran have demonstrated its tolerability and efficacy in promoting long-term reduction of both PCSK9 protein and LDL cholesterol.  The efficacy and safety of inclisiran will continue to be assessed in ongoing and forthcoming trials on larger patient groups.  If the results of these trials reflect previously published data, they will add further evidence that inclisiran might be a revolutionary new tool in the pharmacologic management of plasma lipids.  This review summarizes the currently available literature data on inclisiran with respect to its mechanism of action, effectiveness, and safety as a lipid-lowering drug for CVD prevention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRiaUOLUK_X8O98rul4nhSdfW6udTcc2eb-YJx1UXZpC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mbkt1Kksg%253D%253D&md5=19712f7a6457cc73022ea7396944be58</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.jacl.2019.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jacl.2019.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DDyrbu%25C5%259B%26aufirst%3DK.%26aulast%3DG%25C4%2585sior%26aufirst%3DM.%26aulast%3DPenson%26aufirst%3DP.%26aulast%3DRay%26aufirst%3DK.%2BK.%26aulast%3DBanach%26aufirst%3DM.%26atitle%3DInclisiran%2520%25E2%2580%2593%2520New%2520Hope%2520in%2520the%2520Management%2520of%2520Lipid%2520Disorders%26jtitle%3DJ.%2520Clin.%2520Lipidol.%26date%3D2020%26volume%3D14%26spage%3D16%26epage%3D27%26doi%3D10.1016%2Fj.jacl.2019.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecules as Inhibitors of PCSK9: Current Status and Future Challenges</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>162</i></span>,  <span class="NLM_fpage">212</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.ejmech.2018.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30448414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1Cht7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=162&publication_year=2019&pages=212-233&author=S.+Xuauthor=S.+Luoauthor=Z.+Zhuauthor=J.+Xu&title=Small+Molecules+as+Inhibitors+of+PCSK9%3A+Current+Status+and+Future+Challenges&doi=10.1016%2Fj.ejmech.2018.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules as inhibitors of PCSK9: Current status and future challenges</span></div><div class="casAuthors">Xu, Shengtao; Luo, Shanshan; Zhu, Zheying; Xu, Jinyi</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">162</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">212-233</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Proprotein convertase subtilisin/kexin type 9 (PCSK9) plays an important role in regulating lipoprotein metab. by binding to low-d. lipoprotein receptors (LDLRs), leading to their degrdn.  LDL cholesterol (LDL-C) lowering drugs that operate through the inhibition of PCSK9 are being pursued for the management of hypercholesterolemia and reducing its assocd. atherosclerotic cardiovascular disease (CVD) risk.  Two PCSK9-blocking monoclonal antibodies (mAbs), alirocumab and evolocumab, were approved in 2015.  However, the high costs of PCSK9 antibody drugs impede their prior authorization practices and reduce their long-term adherence.  Given the potential of small-mol. drugs, the development of small-mol. PCSK9 inhibitors has attracted considerable attention.  This article provides an overview of the recent development of small-mol. PCSK9 inhibitors disclosed in the literature and patent applications, and different approaches that have been pursued to modulate the functional activity of PCSK9 using small mols. are described.  Challenges and potential strategies in developing small-mol. PCSK9 inhibitors are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz6U3osadAYbVg90H21EOLACvtfcHk0lj9UC4u7D3p2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1Cht7fM&md5=75a981cd8d564043a3685e9d2d8ca6ce</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DJ.%26atitle%3DSmall%2520Molecules%2520as%2520Inhibitors%2520of%2520PCSK9%253A%2520Current%2520Status%2520and%2520Future%2520Challenges%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D162%26spage%3D212%26epage%3D233%26doi%3D10.1016%2Fj.ejmech.2018.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavecchia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerchia, C.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Developing PCSK9 Inhibitors for Lipid-Lowering Therapy</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.4155%2Ffmc-2018-0294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=30892945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmslGnu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=423-441&author=A.+Lavecchiaauthor=C.+Cerchia&title=Recent+Advances+in+Developing+PCSK9+Inhibitors+for+Lipid-Lowering+Therapy&doi=10.4155%2Ffmc-2018-0294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in developing PCSK9 inhibitors for lipid-lowering therapy</span></div><div class="casAuthors">Lavecchia, Antonio; Cerchia, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">423-441</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Cardiovascular disease is the major cause of death globally, with hypercholesterolemia being an important risk factor.  The PCSK9 represents an attractive therapeutic target for hypercholesterolemia treatment and is currently in the spotlight of the scientific community.  After autocatalytic activation in the hepatocyte endoplasmic reticulum, this convertase binds to the LDLR and channels it to the degrdn. pathway.  This review gives an overview on the latest developments in the inhibition of PCSK9, including disruption of the protein-protein interaction (PPI) between PCSK9 and LDLR by peptidomimetics, adnectins and monoclonal antibodies and the suppression of PCSK9 expression by small mols., siRNA and genome editing techniques.  In addn., we discuss alternative approaches, such as anti-PCSK9 active vaccination and heparin mimetics.  Graphical abstr. :.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk-DYJfvdpxbVg90H21EOLACvtfcHk0ljm8K7Nd111HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmslGnu7k%253D&md5=7bc29631a590d091bd3c1129fa052e6b</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0294%26sid%3Dliteratum%253Aachs%26aulast%3DLavecchia%26aufirst%3DA.%26aulast%3DCerchia%26aufirst%3DC.%26atitle%3DRecent%2520Advances%2520in%2520Developing%2520PCSK9%2520Inhibitors%2520for%2520Lipid-Lowering%2520Therapy%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D423%26epage%3D441%26doi%3D10.4155%2Ffmc-2018-0294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-P.</span></span> <span> </span><span class="NLM_article-title">PCSK9 Inhibitors: Novel Therapeutic Strategies for Lowering LDL Cholesterol</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">176</span>, <span class="refDoi"> DOI: 10.2174/1389557518666180423111442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.2174%2F1389557518666180423111442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=29692249" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVertw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=165-176&author=Y.+Wangauthor=Z.-P.+Liu&title=PCSK9+Inhibitors%3A+Novel+Therapeutic+Strategies+for+Lowering+LDL+Cholesterol&doi=10.2174%2F1389557518666180423111442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">PCSK9 Inhibitors: Novel Therapeutic Strategies for Lowering LDLCholesterol</span></div><div class="casAuthors">Wang, Yan; Liu, Zhao-Peng</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">165-176</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Statins are currently the major therapeutic strategies to lower low-d. lipoprotein cholesterol (LDL-C) levels.  However, a no. of hypercholesterolemia patients still have a residual cardiovascular disease (CVD) risk despite taking the max.-tolerated dose of statins.  Proprotein convertase subtilisin/kexin type 9 (PCSK9) binds to low-d. lipoprotein receptor (LDLR), inducing its degrdn. in the lysosome and inhibiting LDLR recirculating to the cell membranes.  The gain-offunction mutations in PCSK9 elevate the LDL-C levels in plasma.  Therefore, PCSK9 inhibitors become novel therapeutic approaches in the treatment of hypercholesterolemia.  Several PCSK9 inhibitors have been under investigation, and much progress has been made in clin. trials, esp. for monoclonal antibodies (MoAbs).  Two MoAbs, evolocumab and alirocumab, are now in clin. use.  In this review, we summarize the development of PCSK9 inhibitors, including antisense oligonucleotides (ASOs), small interfering RNA (siRNA), small mol. inhibitor, MoAbs, mimetic peptides and adnectins, and the related safety issues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLkfyLZV7Fy7Vg90H21EOLACvtfcHk0ljm8K7Nd111HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVertw%253D%253D&md5=d8ed5787f60a90b0c439818cd2e3ec3d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.2174%2F1389557518666180423111442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557518666180423111442%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.-P.%26atitle%3DPCSK9%2520Inhibitors%253A%2520Novel%2520Therapeutic%2520Strategies%2520for%2520Lowering%2520LDL%2520Cholesterol%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2019%26volume%3D19%26spage%3D165%26epage%3D176%26doi%3D10.2174%2F1389557518666180423111442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4181</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=4167-4181&author=A.+A.+Vinogradovauthor=Y.+Yinauthor=H.+Suga&title=Macrocyclic+Peptides+as+Drug+Candidates%3A+Recent+Progress+and+Remaining+Challenges&doi=10.1021%2Fjacs.8b13178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span></div><div class="casAuthors">Vinogradov, Alexander A.; Yin, Yizhen; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4167-4181</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptides as a therapeutic modality attract much attention due to their synthetic accessibility, high degree of specific binding, and the ability to target protein surfaces traditionally considered "undruggable".  Unfortunately, at the same time, other pharmacol. properties of a generic peptide, such as metabolic stability and cell permeability, are quite poor, which limits the success of de novo discovered biol. active peptides as drug candidates.  Here, we review how macrocyclization as well as the incorporation of nonproteogenic amino acids and various conjugation strategies may be utilized to improve on these characteristics to create better drug candidates.  We analyze recent progress and remaining challenges in improving individual pharmacol. properties of bioactive peptides, and offer our opinion on interfacing these, often conflicting, considerations, to create balanced drug candidates as a potential way to make further progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRPkCAGk8-rVg90H21EOLACvtfcHk0ljm8K7Nd111HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D&md5=59b6197874d4709a74089367a09119cc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13178%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradov%26aufirst%3DA.%2BA.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptides%2520as%2520Drug%2520Candidates%253A%2520Recent%2520Progress%2520and%2520Remaining%2520Challenges%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D4167%26epage%3D4181%26doi%3D10.1021%2Fjacs.8b13178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Josephson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ricardo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szostak, J. W.</span></span> <span> </span><span class="NLM_article-title">mRNA Display: From Basic Principles to Macrocycle Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">399</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2013.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.drudis.2013.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=24157402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsleltrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=388-399&author=K.+Josephsonauthor=A.+Ricardoauthor=J.+W.+Szostak&title=mRNA+Display%3A+From+Basic+Principles+to+Macrocycle+Drug+Discovery&doi=10.1016%2Fj.drudis.2013.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">mRNA display: from basic principles to macrocycle drug discovery</span></div><div class="casAuthors">Josephson, Kristopher; Ricardo, Alonso; Szostak, Jack W.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">388-399</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  We describe a new discovery technol. that uses mRNA-display to rapidly synthesize and screen macrocyclic peptide libraries to explore a valuable region of chem. space typified by natural products.  This technol. allows high-affinity peptidic macrocycles contg. modified backbones and unnatural side chains to be readily selected based on target binding.  Success stories covering the first examples of these libraries suggest that they could be used for the discovery of intracellular protein-protein interaction inhibitors, highly selective enzyme inhibitors or synthetic replacements for monoclonal antibodies.  The review concludes with a look to the future regarding how this technol. might be improved with respect to library design for cell permeability and bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobuL9lWFPZIbVg90H21EOLACvtfcHk0lgAHKxpXGSCUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsleltrnI&md5=6676fef2ca8f0a81cc0e993ee4ec7424</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2013.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2013.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DJosephson%26aufirst%3DK.%26aulast%3DRicardo%26aufirst%3DA.%26aulast%3DSzostak%26aufirst%3DJ.%2BW.%26atitle%3DmRNA%2520Display%253A%2520From%2520Basic%2520Principles%2520to%2520Macrocycle%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D388%26epage%3D399%26doi%3D10.1016%2Fj.drudis.2013.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiedmann, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">RNA Display Methods for the Discovery of Bioactive Macrocycles</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10360</span>– <span class="NLM_lpage">10391</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOht7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10360-10391&author=Y.+Huangauthor=M.+M.+Wiedmannauthor=H.+Suga&title=RNA+Display+Methods+for+the+Discovery+of+Bioactive+Macrocycles&doi=10.1021%2Facs.chemrev.8b00430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">RNA Display Methods for the Discovery of Bioactive Macrocycles</span></div><div class="casAuthors">Huang, Yichao; Wiedmann, Mareike Margarete; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10360-10391</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The past two decades have witnessed the emergence of macrocycles, including macrocyclic peptides, as a promising yet underexploited class of de novo drug candidates.  Both rational/computational design and in vitro display systems have contributed tremendously to the development of cyclic peptide binders of either traditional targets such as cell-surface receptors and enzymes or challenging targets such as protein-protein interaction surfaces. mRNA display, a key platform technol. for the discovery of cyclic peptide ligands, has become one of the leading strategies that can generate natural-product-like macrocyclic peptide binders with antibody-like affinities.  On the basis of the original cell-free transcription/translation system, mRNA display is highly evolvable to realize its full potential by applying genetic reprogramming and chem./enzymic modifications.  In addn., mRNA display also allows the follow-up hit-to-lead development using high-throughput focused affinity maturation.  Finally, mRNA-displayed peptides can be readily engineered to create chem. conjugates based on known small mols. or biologics.  This review covers the birth and growth of mRNA display and discusses the above features of mRNA display with success stories and future perspectives and is up to date as of August 2018.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorCyvw1CP_xLVg90H21EOLACvtfcHk0lgAHKxpXGSCUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOht7bJ&md5=dbf0aec0744b0767f8e634fe60ed2596</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00430%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWiedmann%26aufirst%3DM.%2BM.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DRNA%2520Display%2520Methods%2520for%2520the%2520Discovery%2520of%2520Bioactive%2520Macrocycles%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10360%26epage%3D10391%26doi%3D10.1021%2Facs.chemrev.8b00430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerekes, A. D.</span></span> Unpublished results from our laboratories. A manuscript describing the alternative series of PCSK9 molecules is currently in preparation.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kerekes%2C+A.+D.+Unpublished+results+from+our+laboratories.+A+manuscript+describing+the+alternative+series+of+PCSK9+molecules+is+currently+in+preparation."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dunpublished%26sid%3Dliteratum%253Aachs%26aulast%3DKerekes%26aufirst%3DA.%2BD." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bendels, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obst-Sander, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M.</span></span> <span> </span><span class="NLM_article-title">Fluorine in Medicinal Chemistry</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">643</span>, <span class="refDoi"> DOI: 10.1002/cbic.200301023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1002%2Fcbic.200301023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=15122635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A280%3ADC%252BD2c3it1egug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=637-643&author=H.-J.+B%C3%B6hmauthor=D.+Bannerauthor=S.+Bendelsauthor=M.+Kansyauthor=B.+Kuhnauthor=K.+M%C3%BCllerauthor=U.+Obst-Sanderauthor=M.+Stahl&title=Fluorine+in+Medicinal+Chemistry&doi=10.1002%2Fcbic.200301023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorine in medicinal chemistry</span></div><div class="casAuthors">Bohm Hans-Joachim; Banner David; Bendels Stefanie; Kansy Manfred; Kuhn Bernd; Muller Klaus; Obst-Sander Ulrike; Stahl Martin</div><div class="citationInfo"><span class="NLM_cas:title">Chembiochem : a European journal of chemical biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">637-43</span>
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    </div><div class="casAbstract">Fluorinated compounds are synthesized in pharmaceutical research on a routine basis and many marketed compounds contain fluorine.  The present review summarizes some of the most frequently employed strategies for using fluorine substituents in medicinal chemistry.  Quite often, fluorine is introduced to improve the metabolic stability by blocking metabolically labile sites.  However, fluorine can also be used to modulate the physicochemical properties, such as lipophilicity or basicity.  It may exert a substantial effect on the conformation of a molecule.  Increasingly, fluorine is used to enhance the binding affinity to the target protein.  Recent 3D-structure determinations of protein complexes with bound fluorinated ligands have led to an improved understanding of the nonbonding protein-ligand interactions that involve fluorine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4zDG_wo1YGtts4TFFB11gfW6udTcc2eaPnWer63hqerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3it1egug%253D%253D&md5=fa5ca2cfc278b83c80d5c96b18dfe312</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200301023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200301023%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26aulast%3DBanner%26aufirst%3DD.%26aulast%3DBendels%26aufirst%3DS.%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DObst-Sander%26aufirst%3DU.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DFluorine%2520in%2520Medicinal%2520Chemistry%26jtitle%3DChemBioChem%26date%3D2004%26volume%3D5%26spage%3D637%26epage%3D643%26doi%3D10.1002%2Fcbic.200301023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McNeil, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pundir, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Undem, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span> <span> </span><span class="NLM_article-title">Indentification of a Mast-Cell-Specific Receptor Crucial for Pseudo-allergic Drug Reactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>519</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1038/nature14022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1038%2Fnature14022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=25517090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFKksr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=237-241&author=B.+D.+McNeilauthor=P.+Pundirauthor=S.+Meekerauthor=L.+Hanauthor=B.+J.+Undemauthor=M.+Kulkaauthor=X.+Dong&title=Indentification+of+a+Mast-Cell-Specific+Receptor+Crucial+for+Pseudo-allergic+Drug+Reactions&doi=10.1038%2Fnature14022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a mast-cell-specific receptor crucial for pseudo-allergic drug reactions</span></div><div class="casAuthors">McNeil, Benjamin D.; Pundir, Priyanka; Meeker, Sonya; Han, Liang; Undem, Bradley J.; Kulka, Marianna; Dong, Xinzhong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7542</span>),
    <span class="NLM_cas:pages">237-241</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mast cells are primary effectors in allergic reactions, and may have important roles in disease by secreting histamine and various inflammatory and immunomodulatory substances.  Although they are classically activated by Ig (Ig)E antibodies, a unique property of mast cells is their antibody-independent responsiveness to a range of cationic substances, collectively called basic secretagogues, including inflammatory peptides and drugs assocd. with allergic-type reactions.  The pathogenic roles of these substances have prompted a decades-long search for their receptor(s).  Here the authors report that basic secretagogues activate mouse mast cells in vitro and in vivo through a single receptor, Mrgprb2, the orthologue of the human G-protein-coupled receptor MRGPRX2.  Secretagogue-induced histamine release, inflammation and airway contraction are abolished in Mrgprb2-null mutant mice.  Furthermore, the authors show that most classes of US Food and Drug Administration (FDA)-approved peptidergic drugs assocd. with allergic-type injection-site reactions also activate Mrgprb2 and MRGPRX2, and that injection-site inflammation is absent in mutant mice.  Finally, the authors det. that Mrgprb2 and MRGPRX2 are targets of many small-mol. drugs assocd. with systemic pseudo-allergic, or anaphylactoid, reactions; the authors show that drug-induced symptoms of anaphylactoid responses are significantly reduced in knockout mice; and the authors identify a common chem. motif in several of these mols. that may help predict side effects of other compds.  These discoveries introduce a mouse model to study mast cell activation by basic secretagogues and identify MRGPRX2 as a potential therapeutic target to reduce a subset of drug-induced adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrztm32h9jAMbVg90H21EOLACvtfcHk0lhQ2LzYkLt6oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFKksr%252FL&md5=a77aa1c03938f35de07482715986f1bc</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature14022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14022%26sid%3Dliteratum%253Aachs%26aulast%3DMcNeil%26aufirst%3DB.%2BD.%26aulast%3DPundir%26aufirst%3DP.%26aulast%3DMeeker%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DUndem%26aufirst%3DB.%2BJ.%26aulast%3DKulka%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DX.%26atitle%3DIndentification%2520of%2520a%2520Mast-Cell-Specific%2520Receptor%2520Crucial%2520for%2520Pseudo-allergic%2520Drug%2520Reactions%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D237%26epage%3D241%26doi%3D10.1038%2Fnature14022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, H.</span></span> <span> </span><span class="NLM_article-title">Roles of Mas-related G Protein-Coupled Receptor X2 on Mast Cell-Mediated Host Defense, Pseudoallergic Drug Reactions, and Inflammatory Diseases</span>. <i>J. Allergy Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">700</span>– <span class="NLM_lpage">710</span>, <span class="refDoi"> DOI: 10.1016/j.jaci.2016.04.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.jaci.2016.04.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=27448446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1SlurjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=700-710&author=H.+Subramanianauthor=K.+Guptaauthor=H.+Ali&title=Roles+of+Mas-related+G+Protein-Coupled+Receptor+X2+on+Mast+Cell-Mediated+Host+Defense%2C+Pseudoallergic+Drug+Reactions%2C+and+Inflammatory+Diseases&doi=10.1016%2Fj.jaci.2016.04.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of Mas-related G protein-coupled receptor X2 on mast cell-mediated host defense, pseudoallergic drug reactions, and chronic inflammatory diseases</span></div><div class="casAuthors">Subramanian, Hariharan; Gupta, Kshitij; Ali, Hydar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Allergy and Clinical Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">700-710</span>CODEN:
                <span class="NLM_cas:coden">JACIBY</span>;
        ISSN:<span class="NLM_cas:issn">0091-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Mast cells (MCs), which are granulated tissue-resident cells of hematopoietic lineage, contribute to vascular homeostasis, innate/adaptive immunity, and wound healing.  However, MCs are best known for their roles in allergic and inflammatory diseases, such as anaphylaxis, food allergy, rhinitis, itch, urticaria, atopic dermatitis, and asthma.  In addn. to the high-affinity IgE receptor (FcεRI), MCs express numerous G protein-coupled receptors (GPCRs), which are the largest group of membrane receptor proteins and the most common targets of drug therapy.  Antimicrobial host defense peptides, neuropeptides, major basic protein, eosinophil peroxidase, and many US Food and Drug Administration-approved peptidergic drugs activate human MCs through a novel GPCR known as Mas-related G protein-coupled receptor X2 (MRGPRX2; formerly known as MrgX2).  Unique features of MRGPRX2 that distinguish it from other GPCRs include their presence both on the plasma membrane and intracellular sites and their selective expression in MCs.  In this article we review the possible roles of MRGPRX2 on host defense, drug-induced anaphylactoid reactions, neurogenic inflammation, pain, itch, and chronic inflammatory diseases, such as urticaria and asthma.  We propose that host defense peptides that kill microbes directly and activate MCs through MRGPRX2 could serve as novel GPCR targets to modulate host defense against microbial infection.  Furthermore, mAbs or small-mol. inhibitors of MRGPRX2 could be developed for the treatment of MC-dependent allergic and inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCGk6hW_9_2bVg90H21EOLACvtfcHk0lhQ2LzYkLt6oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1SlurjJ&md5=84191e86547dd9ad97cd4cdb88a31a62</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.jaci.2016.04.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jaci.2016.04.051%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DH.%26aulast%3DGupta%26aufirst%3DK.%26aulast%3DAli%26aufirst%3DH.%26atitle%3DRoles%2520of%2520Mas-related%2520G%2520Protein-Coupled%2520Receptor%2520X2%2520on%2520Mast%2520Cell-Mediated%2520Host%2520Defense%252C%2520Pseudoallergic%2520Drug%2520Reactions%252C%2520and%2520Inflammatory%2520Diseases%26jtitle%3DJ.%2520Allergy%2520Clin.%2520Immunol.%26date%3D2016%26volume%3D138%26spage%3D700%26epage%3D710%26doi%3D10.1016%2Fj.jaci.2016.04.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hein, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span> <span> </span><span class="NLM_article-title">Click Chemistry, A Powerful Tool for Pharmaceutical Sciences</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2216</span>– <span class="NLM_lpage">2230</span>, <span class="refDoi"> DOI: 10.1007/s11095-008-9616-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1007%2Fs11095-008-9616-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=18509602" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFOru7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2008&pages=2216-2230&author=C.+D.+Heinauthor=X.-M.+Liuauthor=D.+Wang&title=Click+Chemistry%2C+A+Powerful+Tool+for+Pharmaceutical+Sciences&doi=10.1007%2Fs11095-008-9616-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry, a powerful tool for pharmaceutical sciences</span></div><div class="casAuthors">Hein, Christopher D.; Liu, Xin-Ming; Wang, Dong</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2216-2230</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Click chem. refers to a group of reactions that are fast, simple to use, easy to purify, versatile, regiospecific, and give high product yields.  While there are a no. of reactions that fulfill the criteria, the Huisgen 1,3-dipolar cycloaddn. of azides and terminal alkynes has emerged as the frontrunner.  It has found applications in a wide variety of research areas, including materials sciences, polymer chem., and pharmaceutical sciences.  In this manuscript, important aspects of the Huisgen cycloaddn. will be reviewed, along with some of its many pharmaceutical applications.  Bioconjugation, nanoparticle surface modification, and pharmaceutical-related polymer chem. will all be covered.  Limitations of the reaction will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtfV89MGuuCrVg90H21EOLACvtfcHk0lhQ2LzYkLt6oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFOru7%252FI&md5=7e0c89178299b2eb8a5b0099ac661a34</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs11095-008-9616-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11095-008-9616-1%26sid%3Dliteratum%253Aachs%26aulast%3DHein%26aufirst%3DC.%2BD.%26aulast%3DLiu%26aufirst%3DX.-M.%26aulast%3DWang%26aufirst%3DD.%26atitle%3DClick%2520Chemistry%252C%2520A%2520Powerful%2520Tool%2520for%2520Pharmaceutical%2520Sciences%26jtitle%3DPharm.%2520Res.%26date%3D2008%26volume%3D25%26spage%3D2216%26epage%3D2230%26doi%3D10.1007%2Fs11095-008-9616-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barral, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moorhouse, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moses, J. E.</span></span> <span> </span><span class="NLM_article-title">Efficient Conversion of Aromatic Amines into Azides: A One-Pot Synthesis of Triazole Linkages</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1809</span>– <span class="NLM_lpage">1811</span>, <span class="refDoi"> DOI: 10.1021/ol070527h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol070527h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsV2lsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=1809-1811&author=K.+Barralauthor=A.+D.+Moorhouseauthor=J.+E.+Moses&title=Efficient+Conversion+of+Aromatic+Amines+into+Azides%3A+A+One-Pot+Synthesis+of+Triazole+Linkages&doi=10.1021%2Fol070527h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Conversion of Aromatic Amines into Azides: A One-Pot Synthesis of Triazole Linkages</span></div><div class="casAuthors">Barral, Karine; Moorhouse, Adam D.; Moses, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1809-1811</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient and improved procedure for the prepn. of arom. azides and their application in the Cu(I)-catalyzed azide-alkyne 1,3-dipolar cycloaddn. ("click reaction") is described.  The synthesis of arom. azides from the corresponding amines is accomplished under mild conditions with tert-Bu nitrite and azidotrimethylsilane.  1,4-Disubstituted 1,2,3-triazoles, e.g., I, were obtained in excellent yields from a variety of arom. amines without the need for isolation of the azide intermediates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUlYdS6yIKhbVg90H21EOLACvtfcHk0lhQ2LzYkLt6oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsV2lsbk%253D&md5=9699b68ebaf593234ca9a5f45122ba49</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fol070527h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol070527h%26sid%3Dliteratum%253Aachs%26aulast%3DBarral%26aufirst%3DK.%26aulast%3DMoorhouse%26aufirst%3DA.%2BD.%26aulast%3DMoses%26aufirst%3DJ.%2BE.%26atitle%3DEfficient%2520Conversion%2520of%2520Aromatic%2520Amines%2520into%2520Azides%253A%2520A%2520One-Pot%2520Synthesis%2520of%2520Triazole%2520Linkages%26jtitle%3DOrg.%2520Lett.%26date%3D2007%26volume%3D9%26spage%3D1809%26epage%3D1811%26doi%3D10.1021%2Fol070527h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-A.</span></span> <span> </span><span class="NLM_article-title">Recent Development of Peptide Coupling Reagents in Organic Synthesis</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2447</span>– <span class="NLM_lpage">2467</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2004.01.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1016%2Fj.tet.2004.01.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFeqsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2447-2467&author=S.-Y.+Hanauthor=Y.-A.+Kim&title=Recent+Development+of+Peptide+Coupling+Reagents+in+Organic+Synthesis&doi=10.1016%2Fj.tet.2004.01.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Recent development of peptide coupling reagents in organic synthesis</span></div><div class="casAuthors">Han, So-Yeop; Kim, Young-Ah</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2447-2467</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review.  Various types of peptide coupling reagents (i.e., phosphonium, uronium, immonium, carbodiimide, imidazolium, organophosphorus, acid halogenating compds., chloroformate, pyridinium, etc.) are discussed for the synthesis of bioactive mols. contg. peptide linkages.  Methods used to suppress racemization during the peptide coupling step are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5MxkBBhDXubVg90H21EOLACvtfcHk0lgx521rsjKRUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFeqsro%253D&md5=8c9ca02776710dd0e204262a85da630b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2004.01.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2004.01.020%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DS.-Y.%26aulast%3DKim%26aufirst%3DY.-A.%26atitle%3DRecent%2520Development%2520of%2520Peptide%2520Coupling%2520Reagents%2520in%2520Organic%2520Synthesis%26jtitle%3DTetrahedron%26date%3D2004%26volume%3D60%26spage%3D2447%26epage%3D2467%26doi%3D10.1016%2Fj.tet.2004.01.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T.</span></span> (<span class="NLM_year">2017</span>)  <i>BUSTER</i>, version 2.11.6. <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>: <span class="NLM_publisher-name">Global Phasing Ltd</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+Smart&author=C.+Vonrhein&author=T.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26date%3D2017%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lgx521rsjKRUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=487-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D487%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Automated Ligand Placement and Refinement with a Combined Force Field and Shape Potential</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1107/s0907444906016076</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1107%2FS0907444906016076" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=16790930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtVSnur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=741-749&author=S.+Wlodekauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Automated+Ligand+Placement+and+Refinement+with+a+Combined+Force+Field+and+Shape+Potential&doi=10.1107%2Fs0907444906016076"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Automated ligand placement and refinement with a combined force field and shape potential</span></div><div class="casAuthors">Wlodek, S.; Skillman, A. G.; Nicholls, A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">741-749</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">An automated computational procedure for fitting a ligand into its electron d. with the use of the MMFF94 force field and a Gaussian shape description has been developed.  It employs a series of adiabatic optimizations of gradually increasing shape potential.  Starting from a set of energy-relaxed ligand conformations, the final results are structures realistically strained to fit the crystallog. data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbAunEGTbprrVg90H21EOLACvtfcHk0liZDKgh8-iFbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtVSnur0%253D&md5=1571b77a211aec76fb4e3a376fc95d8a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444906016076&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906016076%26sid%3Dliteratum%253Aachs%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DAutomated%2520Ligand%2520Placement%2520and%2520Refinement%2520with%2520a%2520Combined%2520Force%2520Field%2520and%2520Shape%2520Potential%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D741%26epage%3D749%26doi%3D10.1107%2Fs0907444906016076" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rothschild, A. M.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Histamine Release by Compound 48/80</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.1970.tb10354.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=10.1111%2Fj.1476-5381.1970.tb10354.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=4189829" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;key=1%3ACAS%3A528%3ADyaE3cXpvVyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1970&pages=253-262&author=A.+M.+Rothschild&title=Mechanisms+of+Histamine+Release+by+Compound+48%2F80&doi=10.1111%2Fj.1476-5381.1970.tb10354.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of histamine release by compound 48/80</span></div><div class="casAuthors">Rothschild, Adolfo M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">253-62</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    </div><div class="casAbstract">Rat and guinea-pig lung tissues were incubated for 20 min at 37° in Krebs-Ringer phosphate buffer at pH 7.4, or in Tyrode-Tris buffer at pH 8.2, and the release of histamine produced by adding different concns. of compound 48/80 (I) to the incubation medium was detd.  At pH 7.4, increasing concns. of I increased the release of histamine from the rat lung, with a tendency towards a max.  No release of histamine from guinea-pig lung was observed at this pH.  At pH 8.2, histamine release occurred both from rat and guinea-pig lung, and was proportional to the logarithm of the concn. of I.  Histamine release from rat lung by 20 μg/ml of I decreased when the pH was raised from 7.4 to 8.2; but the release caused by 1 mg/ml I increased both in rat and guinea-pig lung as the pH was raised.  2,4-Dinitrophenol (DNP) inhibited the release of histamine from rat lung by a concn. of 20 μg/ml of I; the inhibition was prevented by glucose.  DNP did not affect histamine release from rat or guinea-pig lung by a concn. of 1 mg/ml of I and enhanced the release when the pH was raised from 7.4 to 8.2.  One mg/ml of I did not inhibit the enhanced O consumption produced by DNP in the isolated rat diaphragm.  Iodoacetic acid (IAA) or a Ca/Mg-free medium inhibited the release of histamine by 20 μg/ml of I from rat lung but not the release produced by 1 mg/ml in either rat or guinea-pig lung.  The degranulation of rat mesentery mast cells caused by 20 μg/ml of I was inhibited by DNP.  The degranulation evoked by 1 mg/ml of I was also sensitive to this inhibitor; in this instance, however, the metachromatic staining reaction of the mesentery mast cells was greatly diminished.  Two processes of histamine release by I occur in rat lung.  One, dependent on cell metabolism, involves mast cell granule secretion.  The other, independent of cell metabolism, seems to consist of a simple exchange reaction between histamine and I, and this is the only one occurring in guinea-pig lung.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpphvfOc3RrrVg90H21EOLACvtfcHk0liZDKgh8-iFbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXpvVyrsw%253D%253D&md5=72d120593ddbe928757bf93bd223cb33</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.1970.tb10354.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.1970.tb10354.x%26sid%3Dliteratum%253Aachs%26aulast%3DRothschild%26aufirst%3DA.%2BM.%26atitle%3DMechanisms%2520of%2520Histamine%2520Release%2520by%2520Compound%252048%252F80%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D1970%26volume%3D38%26spage%3D253%26epage%3D262%26doi%3D10.1111%2Fj.1476-5381.1970.tb10354.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIB','PDB','6XIB'); return false;">PDB: 6XIB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIC','PDB','6XIC'); return false;">PDB: 6XIC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XID" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XID','PDB','6XID'); return false;">PDB: 6XID</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIE','PDB','6XIE'); return false;">PDB: 6XIE</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIF','PDB','6XIF'); return false;">PDB: 6XIF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NMX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NMX','PDB','4NMX'); return false;">PDB: 4NMX</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i69"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01084">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_68785"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01084?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01084</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Representative LC traces for key molecules in the paper, sample IC<sub>50</sub> curves for LDLR-FRET and Alexa-FRET assay control peptide, and raw concentration versus time profiles for molecules tested in Mouse IV PK 3D PDB files for each of the new crystal structures have also been uploaded; PDB codes for the crystal structures shown in the paper are as follows: compound <b>30</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIB">6XIB</a>; compound <b>40</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIC">6XIC</a>; compound <b>51</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XID">6XID</a>; compound <b>78</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIE">6XIE</a>; and compound <b>84</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XIF">6XIF</a>; and authors will release the atomic coordinates and experimental data upon article publication (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_001.pdf">PDF</a>)</p></li><li><p class="inline">6xib (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_002.pdb">PDB</a>)</p></li><li><p class="inline">6xic (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_003.pdb">PDB</a>)</p></li><li><p class="inline">6xid (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_004.pdb">PDB</a>)</p></li><li><p class="inline">6xie (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_005.pdb">PDB</a>)</p></li><li><p class="inline">6xif (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_006.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_001.pdf">jm0c01084_si_001.pdf (616.41 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_002.pdb">jm0c01084_si_002.pdb (274.18 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_003.pdb">jm0c01084_si_003.pdb (279.28 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_004.pdb">jm0c01084_si_004.pdb (276.56 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_005.pdb">jm0c01084_si_005.pdb (275.96 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01084/suppl_file/jm0c01084_si_006.pdb">jm0c01084_si_006.pdb (267.71 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01084&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01084%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-22" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01084" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679920dfaee22290","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
