
<html lang="en"     class="pb-page"  data-request-id="3acf0537-0922-4836-8f38-de2e0d29d885"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c02232;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Development of Dimethylisoxazole-Attached Imidazo[1,2-a]pyridines as Potent and Selective CBP/P300 Inhibitors" /></meta><meta name="dc.Creator" content="Alex  Muthengi" /></meta><meta name="dc.Creator" content="Virangika K.  Wimalasena" /></meta><meta name="dc.Creator" content="Hailemichael O.  Yosief" /></meta><meta name="dc.Creator" content="Melissa J.  Bikowitz" /></meta><meta name="dc.Creator" content="Logan H.  Sigua" /></meta><meta name="dc.Creator" content="Tingjian  Wang" /></meta><meta name="dc.Creator" content="Deyao  Li" /></meta><meta name="dc.Creator" content="Zied  Gaieb" /></meta><meta name="dc.Creator" content="Gagan  Dhawan" /></meta><meta name="dc.Creator" content="Shuai  Liu" /></meta><meta name="dc.Creator" content="Jon  Erickson" /></meta><meta name="dc.Creator" content="Rommie E.  Amaro" /></meta><meta name="dc.Creator" content="Ernst  Schönbrunn" /></meta><meta name="dc.Creator" content="Jun  Qi" /></meta><meta name="dc.Creator" content="Wei  Zhang" /></meta><meta name="dc.Description" content="The use of epigenetic bromodomain inhibitors as anticancer therapeutics has transitioned from targeting bromodomain extraterminal domain (BET) proteins into targeting non-BET bromodomains. The two ..." /></meta><meta name="Description" content="The use of epigenetic bromodomain inhibitors as anticancer therapeutics has transitioned from targeting bromodomain extraterminal domain (BET) proteins into targeting non-BET bromodomains. The two ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 19, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02232" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02232" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02232" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02232" /></link>
        
    
    

<title>Development of Dimethylisoxazole-Attached Imidazo[1,2-a]pyridines as Potent and Selective CBP/P300 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02232" /></meta><meta property="og:title" content="Development of Dimethylisoxazole-Attached Imidazo[1,2-a]pyridines as Potent and Selective CBP/P300 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0015.jpeg" /></meta><meta property="og:description" content="The use of epigenetic bromodomain inhibitors as anticancer therapeutics has transitioned from targeting bromodomain extraterminal domain (BET) proteins into targeting non-BET bromodomains. The two most relevant non-BET bromodomain oncology targets are cyclic AMP response element-binding protein (CBP) and E1A binding protein P300 (EP300). To explore the growing CBP/EP300 interest, we developed a highly efficient two-step synthetic route for dimethylisoxazole-attached imidazo[1,2-a]pyridine scaffold-containing inhibitors. Our efficient two-step reactions enabled high-throughput synthesis of compounds designed by molecular modeling, which together with structure–activity relationship (SAR) studies facilitated an overarching understanding of selective targeting of CBP/EP300 over non-BET bromodomains. This led to the identification of a new potent and selective CBP/EP300 bromodomain inhibitor, UMB298 (compound 23, CBP IC50 72 nM and bromodomain 4, BRD4 IC50 5193 nM). The SAR we established is in good agreement with literature-reported CBP inhibitors, such as CBP30, and demonstrates the advantage of utilizing our two-step approach for inhibitor development of other bromodomains." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02232"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02232">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02232&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02232&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02232&amp;href=/doi/10.1021/acs.jmedchem.0c02232" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 5787-5801</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02230" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02237" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Development of Dimethylisoxazole-Attached Imidazo[1,2-<i>a</i>]pyridines as Potent and Selective CBP/P300 Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Alex Muthengi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alex Muthengi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alex++Muthengi">Alex Muthengi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Virangika K. Wimalasena</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Virangika K. Wimalasena</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Virangika+K.++Wimalasena">Virangika K. Wimalasena</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hailemichael O. Yosief</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hailemichael O. Yosief</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hailemichael+O.++Yosief">Hailemichael O. Yosief</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Melissa J. Bikowitz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Melissa J. Bikowitz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Department, Moffit Cancer Center, Tampa, Florida 33612, United States</div><div class="loa-info-affiliations-info">Department of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Melissa+J.++Bikowitz">Melissa J. Bikowitz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4985-3461" title="Orcid link">http://orcid.org/0000-0003-4985-3461</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Logan H. Sigua</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Logan H. Sigua</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Logan+H.++Sigua">Logan H. Sigua</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tingjian Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tingjian Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tingjian++Wang">Tingjian Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Deyao Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Deyao Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Deyao++Li">Deyao Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zied Gaieb</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zied Gaieb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry & Biochemistry, University of California, San Diego, 9500 Gilman Dr, LA Jolla, California 92093, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zied++Gaieb">Zied Gaieb</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gagan Dhawan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gagan Dhawan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biomedical Science, Acharya Narendra Dev College, University of Delhi, Delhi, New Delhi 110019, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gagan++Dhawan">Gagan Dhawan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7492-225X" title="Orcid link">http://orcid.org/0000-0001-7492-225X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shuai Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuai Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuai++Liu">Shuai Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jon Erickson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jon Erickson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jon++Erickson">Jon Erickson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rommie E. Amaro</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rommie E. Amaro</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry & Biochemistry, University of California, San Diego, 9500 Gilman Dr, LA Jolla, California 92093, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rommie+E.++Amaro">Rommie E. Amaro</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9275-9553" title="Orcid link">http://orcid.org/0000-0002-9275-9553</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ernst Schönbrunn</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ernst Schönbrunn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery Department, Moffit Cancer Center, Tampa, Florida 33612, United States</div><div class="loa-info-affiliations-info">Department of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#efaa9d819c9bc1bc8c8780818d9d9a8181af82808989869b9bc1809d88"><span class="__cf_email__" data-cfemail="3b7e4955484f15685853545559494e55557b56545d5d524f4f1554495c">[email protected]</span></a>. Tel: +1 (813) 745-4703. Fax: +1 (813) 745-6748.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ernst++Sch%C3%B6nbrunn">Ernst Schönbrunn</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3589-3510" title="Orcid link">http://orcid.org/0000-0002-3589-3510</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jun Qi</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jun Qi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div><div class="loa-info-affiliations-info">Department of Medicine, Harvard Medical School, 220 Longwood Avenue, Boston, Massachusetts 02115, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c5afb0ab9ab4ac85a1a3a6acebada4b7b3a4b7a1eba0a1b0"><span class="__cf_email__" data-cfemail="0c667962537d654c686a6f6522646d7e7a6d7e6822696879">[email protected]</span></a>. Tel: +1 (617) 632-6629. Fax: +1(617) 582-7370.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jun++Qi">Jun Qi</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Wei Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Center for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#9becfef2a9b5e1f3faf5fcdbeef6f9b5feffee"><span class="__cf_email__" data-cfemail="1f687a762d3165777e71785f6a727d317a7b6a">[email protected]</span></a>. Tel: +1 (617) 287-6147. Fax: +1 (617) 287-6030.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Zhang">Wei Zhang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6097-2763" title="Orcid link">http://orcid.org/0000-0002-6097-2763</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02232&amp;href=/doi/10.1021%2Facs.jmedchem.0c02232" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 5787–5801</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 19, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 December 2020</li><li><span class="item_label"><b>Published</b> online</span>19 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02232" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02232</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5787%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAlex%2BMuthengi%252C%2BVirangika%2BK.%2BWimalasena%252C%2BHailemichael%2BO.%2BYosief%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c02232%26title%3DDevelopment%2Bof%2BDimethylisoxazole-Attached%2BImidazo%255B1%252C2-a%255Dpyridines%2Bas%2BPotent%2Band%2BSelective%2BCBP%252FP300%2BInhibitors%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5801%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02232"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1190</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02232" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Development of Dimethylisoxazole-Attached Imidazo[1,2-a]pyridines as Potent and Selective CBP/P300 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Alex&quot;,&quot;last_name&quot;:&quot;Muthengi&quot;},{&quot;first_name&quot;:&quot;Virangika&quot;,&quot;last_name&quot;:&quot;K. Wimalasena&quot;},{&quot;first_name&quot;:&quot;Hailemichael&quot;,&quot;last_name&quot;:&quot;O. Yosief&quot;},{&quot;first_name&quot;:&quot;Melissa&quot;,&quot;last_name&quot;:&quot;J. Bikowitz&quot;},{&quot;first_name&quot;:&quot;Logan&quot;,&quot;last_name&quot;:&quot;H. Sigua&quot;},{&quot;first_name&quot;:&quot;Tingjian&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Deyao&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Zied&quot;,&quot;last_name&quot;:&quot;Gaieb&quot;},{&quot;first_name&quot;:&quot;Gagan&quot;,&quot;last_name&quot;:&quot;Dhawan&quot;},{&quot;first_name&quot;:&quot;Shuai&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;Erickson&quot;},{&quot;first_name&quot;:&quot;Rommie&quot;,&quot;last_name&quot;:&quot;E. Amaro&quot;},{&quot;first_name&quot;:&quot;Ernst&quot;,&quot;last_name&quot;:&quot;Schönbrunn&quot;},{&quot;first_name&quot;:&quot;Jun&quot;,&quot;last_name&quot;:&quot;Qi&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;5787-5801&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02232&quot;},&quot;abstract&quot;:&quot;The use of epigenetic bromodomain inhibitors as anticancer therapeutics has transitioned from targeting bromodomain extraterminal domain (BET) proteins into targeting non-BET bromodomains. The two most relevant non-BET bromodomain oncology targets are cyclic AMP response element-binding protein (CBP) and E1A binding protein P300 (EP300). To explore the growing CBP/EP300 interest, we developed a highly efficient two-step synthetic route for dimethylisoxazole-attached imidazo[1,2-a]pyridine scaffold-containing inhibitors. Our efficient two-step reactions enabled high-throughput synthesis of compounds designed by molecular modeling, which together with structure–activity relationship (SAR) studies facilitated an overarching understanding of selective targeting of CBP/EP300 over non-BET bromodomains. This led to the identification of a new potent and selective CBP/EP300 bromodomain inhibitor, UMB298 (compound 23, CBP IC50 72 nM and bromodomain 4, BRD4 IC50 5193 nM). The SAR we established is in good agreement wit&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02232&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02232" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02232&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02232" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02232&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02232" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02232&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02232&amp;href=/doi/10.1021/acs.jmedchem.0c02232" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02232" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02232" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02232%26sid%3Dliteratum%253Aachs%26pmid%3D33872011%26genre%3Darticle%26aulast%3DMuthengi%26date%3D2021%26atitle%3DDevelopment%2Bof%2BDimethylisoxazole-Attached%2BImidazo%255B1%252C2-a%255Dpyridines%2Bas%2BPotent%2Band%2BSelective%2BCBP%252FP300%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D5787%26epage%3D5801%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (11)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The use of epigenetic bromodomain inhibitors as anticancer therapeutics has transitioned from targeting bromodomain extraterminal domain (BET) proteins into targeting non-BET bromodomains. The two most relevant non-BET bromodomain oncology targets are cyclic AMP response element-binding protein (CBP) and E1A binding protein P300 (EP300). To explore the growing CBP/EP300 interest, we developed a highly efficient two-step synthetic route for dimethylisoxazole-attached imidazo[1,2-<i>a</i>]pyridine scaffold-containing inhibitors. Our efficient two-step reactions enabled high-throughput synthesis of compounds designed by molecular modeling, which together with structure–activity relationship (SAR) studies facilitated an overarching understanding of selective targeting of CBP/EP300 over non-BET bromodomains. This led to the identification of a new potent and selective CBP/EP300 bromodomain inhibitor, UMB298 (compound <b>23</b>, CBP IC<sub>50</sub> 72 nM and bromodomain 4, BRD4 IC<sub>50</sub> 5193 nM). The SAR we established is in good agreement with literature-reported CBP inhibitors, such as CBP30, and demonstrates the advantage of utilizing our two-step approach for inhibitor development of other bromodomains.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25010" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25010" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Bromodomains (BRDs) are epigenetic reader domains that selectively recognize acetyl-lysine residues found in histone and nonhistone proteins and function in transcriptional regulation.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Failure to regulate these protein reader modules can result in a broad range of diseases such as cancer and inflammatory and cardiovascular diseases. As a result, they are pursued as attractive targets for developing therapeutic agents for the treatment of different disease conditions.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Although much progress has been made in the development bromodomain extraterminal domain (BET) bromodomain small-molecule inhibitors, some of which are undergoing human clinical trials for various diseases,<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> selective and potent non-BET bromodomain inhibitors and their use in biological studies are still limited. Out of the 57 non-BET bromodomains, the cyclic AMP response element binding protein (CBP) bromodomain, which was once considered to have “medium druggability”, has generated recent interest in oncology applications, and three new double-digit nanomolar inhibitors have been developed.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> Additionally, CBP has a paralogue that conducts similar activity, called the E1A binding protein (EP300 or P300), both of which are multidomain proteins containing BRD and other domains, such as lysine acetyl transferase (KAT), which acetylates and recruits histone and nonhistone proteins.</div><div id="sec1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Overview of the Role of CBP/EP300 in Diseases</h3><div class="NLM_p last">The BRD of CBP/EP300 selectively recognizes acetylated lysine residues, in contrast to the KAT domain, which acetylates lysine residues on both histone tails and signature tumor transcription factors, such as p53 and c-<i>MYC</i>.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9−12)</a> There are eight acetylation sites on p53, of which K382 is necessary for coactivator (CBP and TRRAP) recruitment, which importantly results in a p53 stabilizing conformation change and therefore active p53 and activation of proapoptotic genes.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> The CBP/P300 bromodomains have been linked to disorders like acute myeloid leukemia (AML) and inflammatory and neurodegenerative diseases.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Recently, Olzscha and co-workers also demonstrated that CBP/P300 bromodomains are required for protein aggregation, which disturbs proteostasis by impairing the ubiquitin proteasome system (UPS) and protein translation, leading to decreased cell viability.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Overall, CBP/EP300 are considered the most promising non-BET bromodomain targets for cancer and other pathological conditions.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Therefore, this study was aimed at developing a novel strategy to discover potent and selective CBP inhibitors for use as chemical probes to provide further insights into CBP/EP300-disease pathogenesis.</div></div><div id="sec1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Inhibitors Targeting the Bromodomain of CBP/EP300</h3><div class="NLM_p">According to the structural genomics consortium (SGC) requirement, a chemical probe should possess inhibitory activity at less than 100 nM concentration (IC<sub>50</sub> < 100 nM), with selectivity greater than 30-fold toward other proteins (>30-fold) and cellular activity at ∼1 μM concentration.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> To better understand and investigate the biological function of CBP bromodomains, a number of CBP inhibitors have been reported with submicromolar to nanomolar inhibitory activity against the bromodomain of CBP or P300, including CBP30,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> R-2,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> PF-CBP1,<a onclick="showRef(event, 'ref9 ref21'); return false;" href="javascript:void(0);" class="ref ref9 ref21">(9,21)</a> TPO146,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> CPI-637,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> GNE-272,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and GNE-781<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Most of these compounds are prepared through multistep synthesis with low overall yields, which hampers further optimization of the compounds to improve their biological activity.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> We have recently reported a two-step synthesis of dimethylisoxazole-attached imidazo[1,2-<i>a</i>]pyridine BET inhibitors.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Overall, we envisioned that we could employ the optimized two-step synthetic route toward developing new CBP/EP300 inhibitors in conjunction with structure–activity relationship (SAR) studies to generate lead compounds. Additionally, the methods can be further extended to the development of inhibitors targeting other bromodomains.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Reported potent CBP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26508" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26508" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Analysis of the chemical structures of reported CBP inhibitors revealed that potent inhibitors contain two important components. The first is an acetyl-lysine mimic motif (blue unit in the structures shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) such as 3,5-dimethylisoxazole, and the second is an appendix portion outside the binding pocket, such as 3-chloro-4-methoxyphenethyl and ethylmorpholinyl motifs, which can engage in additional interactions with the protein outside the binding pocket. This analysis inspired us to use our modified multicomponent reaction (MCR) to assemble a substituted imidazo[1,2-<i>a</i>]pyridine scaffold followed by the introduction of a binding motif through the Suzuki coupling (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Two-Step Synthesis of Imidazo[1,2-<i>a</i>]pyridine-Based CBP/P300 Inhibitors; Reaction Conditions: (a) Sc(OTf)<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH (3:1), μw, 100 °C, 40 min 90–95%; (b) 3,5-Dimethylisoxazole-4-boronic Acid Pinacol Ester, PdCl<sub>2</sub> (dppf)·CH<sub>2</sub>Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub> DME/H<sub>2</sub>O (2:1), μw, 120 °C, 40 min, 89–93%</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">By utilizing our highly efficient synthesis involving three-component Groebke–Blackburn–Bienayme (GBB)<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> reactions followed by the Suzuki coupling reaction,<a onclick="showRef(event, 'ref26 ref30'); return false;" href="javascript:void(0);" class="ref ref26 ref30">(26,30)</a> we generated a set of molecules with different binding motifs linked to the imidazo[1,2-<i>a</i>]pyridine scaffold in two steps. These compounds were assessed for their binding activity against the CBP bromodomain utilizing an AlphaScreen assay using biotinylated peptide and recombinant CBP bromodomain as assay partners (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and against bromodomain 4 (BRD4) in our previously developed BRD4 AlphaScreen assay.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Overall, AlphaScreen showed that the compounds exhibited selective micromolar inhibitory activity against the CBP bromodomain, over BRD4. Significantly improved potency and selectivity were seen when the known strong acetylated-lysine mimetic (3,5-dimethylisoxazole) substituent group<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> was introduced at the R<sup>2</sup> position (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Previous studies showed that 3,5-dimethylisoxazole acts as an acetyl lysine mimic, where the isoxazole oxygen forms a hydrogen-bonding interaction with the Asn1168 amide moiety, while the isoxazole nitrogen points toward one of the water molecules that bridges with Tyr1125.<a onclick="showRef(event, 'ref10 ref26'); return false;" href="javascript:void(0);" class="ref ref10 ref26">(10,26)</a> Interestingly, compound <b>4</b> exhibited modest inhibitory activity against CBP and BRD4 with IC<sub>50</sub> values of 1.17 and 2.96 μM, respectively. These promising data indicated that our strategy could facilitate SAR studies to fine-tune the potency and selectivity in a high-throughput manner. Encouraged by these results, we decided to fully explore the CBP inhibitor design and development using the imidazo[1,2-<i>a</i>]pyridine scaffold as the core (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure–Activity Relationship (SAR) on the Imidazo[1,2-<i>a</i>]pyridine Scaffold<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0012.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">CBP</th><th class="colsep0 rowsep0" align="center">BRD4</th><th class="colsep0 rowsep0" align="center" char=".">selectivity<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">adamantyl</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">16.09 ± 0.76</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">adamantyl</td><td class="colsep0 rowsep0" align="left">3,5-dimethylisoxazolyl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">2.21 ± 0.43</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="left">3,5-dimethylisoxazolyl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">14.19 ± 0.81</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left"><b>t-butyl acetyl</b></td><td class="colsep0 rowsep0" align="left"><b>3,5-dimethylisoxazolyl</b></td><td class="colsep0 rowsep0" align="left"><b>H</b></td><td class="colsep0 rowsep0" align="left"><b>1.17</b> ± 0.05</td><td class="colsep0 rowsep0" align="left"><b>2.95</b> ± 0.04</td><td class="colsep0 rowsep0" align="char" char="."><b>3</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">2,2,4-trimethylpentyl</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">16.33 ± 0.53</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CBP30</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.028 ± 0.04</td><td class="colsep0 rowsep0" align="left">7.94 ± 0.22</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">All IC<sub>50</sub> values are reported as means of values from at least four determinations. AlphaScreen assay with the isolated CBP (using biotinylated H3K56ac peptide)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> or BRD4 bromodomain (with biotinylated JQ1).<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Selectivity is defined by [BRD4(1) IC<sub>50</sub> (μM)/CBP IC<sub>50</sub> (μM)].</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Bold indicates the most active compound.</p></div></div></div><div class="NLM_p">While the peptide-based AlphaScreen assay can assess compound activity, the performance of this assay is not robust due to the limited stability of the peptide used. We therefore developed a novel small-molecule-based AlphaScreen assay for CBP and EP300 bromodomain using biotinylated probe <b>C</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) with recombinant proteins (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Since the bromodomains of EP300 and CBP have greater than 95% homology, we utilized the CBP bromodomain to represent both CBP and EP300 bromodomains. This probe compound was derivatized from compound <b>2</b> shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> guided by the cocrystal structure of CBP30 with CBP (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a>) reported in the literature. In this assay, biotinylated compound <b>C</b> was utilized as a novel probe to evaluate the competitive displacement by our series of CBP bromodomain inhibitors. Purified His-tagged CBP or EP300 bromodomains bound to nickel beads and biotinylated compound <b>C</b> bound to Streptavidin beads were brought into close proximity to one another upon probe binding to the protein. Upon excitation, the beads created an emission due to their proximity, which was disrupted by competitive binding by our bromodomain inhibitors, displacing the probe, allowing for a dose–response binding curve to be extrapolated. To optimize the assay condition, we systematically titrated both ligand (<b>C</b>) and protein (His-CBP) against each other at varying concentrations in the presence of 20 ng/mL Streptavidin donor beads and 20 ng/mL nickel acceptor beads. As these are standard α bead concentrations, this two-dimensional (2D) titration is performed to yield an optimal AlphaScreen signal. Because we are forming a ternary complex in this AlphaScreen reaction, there is an optimal “hook point” concentration that correlates with the optimal signal.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> We titrated both the compounds ranging from 10 μM to 5 nM and found that Compound C was able to generate this characteristic hook point at 10 nM ligand concentration and 50 nM His-tagged CBP protein concentration. We further confirmed that the signal intensity of the small-molecule-probe-based assay is similar to that of the peptide-based assay. The small-molecule-probed-based assay performs more consistently compared to the peptide-based assay and is used for IC50 assessment for establishing SAR.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Development of small-molecule-based CBP/P300 AlphaScreening assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0011.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Biotinylated-CBP/P300 Bromodomain Binder for AlphaScreening Assay Development</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To have a better understanding of the observed potency and selectivity, we utilized molecular modeling to explore the binding modes of our imidazo[1,2-<i>a</i>]pyridine inhibitor compound <b>4</b> (UMB278) with CBP using the cocrystal structure of CBP30 with the CBP bromodomain (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The isoxazole of compound <b>4</b> showed critical hydrogen bonding with N1168, mimicking the acetylated lysine interaction observed between CBP and its substrate. We observed extra hydrogen bonds around imidazo[1,2-<i>a</i>]pyridine with surrounding crystallographic water molecules that might also contribute favorably to binding activity. Thus, we decided to maintain 3,5-dimethylisoxazole as a warhead and imidazo[1,2-<i>a</i>]pyridine as a scaffold because they are responsible for the improved inhibitory activity of our inhibitors. We noticed that the solvent-accessible methoxyphenyl group might contribute to the low affinity of the molecules since the planarity of the bond between the benzimidazole and benzene groups might prohibit the molecules from forming cation−π interactions with R1173. Thus, we explored the effect of modifying the two appendix groups R<sup>1</sup> and R<sup>3</sup> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docked poses of compounds <b>3</b> (cyan) and <b>4</b> (orange) in CBP (PDBs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structure–Activity Relationship on Compound <b>7</b> Analogues<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0013.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">CBP</th><th class="colsep0 rowsep0" align="center">BRD4</th><th class="colsep0 rowsep0" align="center" char=".">selectivity<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left"><i>t</i>-butyl acetyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">7.828</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b> (UMB276)</td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.718 ± 0.042</td><td class="colsep0 rowsep0" align="left">7.43 ± 1.18</td><td class="colsep0 rowsep0" align="char" char=".">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">adamantyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.507 ± 0.045</td><td class="colsep0 rowsep0" align="left">3.38 ± 0.63</td><td class="colsep0 rowsep0" align="char" char=".">7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="left"><b>c</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">8.12 ± 0.96</td><td class="colsep0 rowsep0" align="left">7.55 ± 0.56</td><td class="colsep0 rowsep0" align="char" char=".">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="left"><b>b</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">8.02 ± 0.06</td><td class="colsep0 rowsep0" align="left">0.51±0.01</td><td class="colsep0 rowsep0" align="char" char=".">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="left"><b>d</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">3.02 ± 0.51</td><td class="colsep0 rowsep0" align="char" char=".">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">2,2,4-trimethylpentyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">1.88 ± 0.03</td><td class="colsep0 rowsep0" align="left">2.77 ± 0.08</td><td class="colsep0 rowsep0" align="char" char=".">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left"><i>t</i>-butyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.95 ± 0.02</td><td class="colsep0 rowsep0" align="left">5.63 ± 0.31</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">adamantyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">0.923 ± 0.012</td><td class="colsep0 rowsep0" align="left">7.11 ± 0.31</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">4-ethylmorpholinyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">0.46 ± 0.09</td><td class="colsep0 rowsep0" align="left">3.42 ± 0.01</td><td class="colsep0 rowsep0" align="char" char=".">7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left"><b>cyclohexyl</b></td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left"><b>OMe</b></td><td class="colsep0 rowsep0" align="left"><b>0.159 ± 0.021</b></td><td class="colsep0 rowsep0" align="left"><b>6.59 ± 0.05</b></td><td class="colsep0 rowsep0" align="char" char="."><b>41</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">isopropyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">0.29 ± 0.01</td><td class="colsep0 rowsep0" align="left">10.09 ± 0.21</td><td class="colsep0 rowsep0" align="char" char=".">34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left"><i>t</i>-butyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">2.33 ± 0.60</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OEt</td><td class="colsep0 rowsep0" align="left">0.214 ± 0.009</td><td class="colsep0 rowsep0" align="left">6.48 ± 0.41</td><td class="colsep0 rowsep0" align="char" char=".">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OPr</td><td class="colsep0 rowsep0" align="left">0.633 ± 0.006</td><td class="colsep0 rowsep0" align="left">17.92 ± 1.21</td><td class="colsep0 rowsep0" align="char" char=".">28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">benzyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">1.92 ± 0.12</td><td class="colsep0 rowsep0" align="left">8.46 ± 0.55</td><td class="colsep0 rowsep0" align="char" char=".">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">4-chlorobenzyl</td><td class="colsep0 rowsep0" align="left"><b>a</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">1.60 ± 0.21</td><td class="colsep0 rowsep0" align="left">7.94 ± 0.45</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>CBP30</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.077 ± 0.011</td><td class="colsep0 rowsep0" align="left">10.1 ± 0.55</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All IC50 values are reported as means of values from at least two determinations. AlphaScreen assay with the isolated CBP (with compound C) or BRD4 (with biotinylated JQ1) bromodomain.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Selectivity is defined by [BRD4(1) IC<sub>50</sub> (μM)/CBP IC50 (μM)] for the consistency mentioned.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">The IC50 of compound 4 with CBP is measured using the biotinylated-peptide-based AlphaScreen assay as in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Bold indicates the most active compound.</p></div></div></div><div class="NLM_p">Based on our hypothesis, we introduced a rotatable bond between methoxyphenyl and imidazo[1,2-<i>a</i>]pyridine to extend the appending group and increase the binding interactions with R1173 on the CBP bromodomain outside the binding pocket (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). We synthesized compounds by introducing a two-carbon linker between the core structure and the 4-methoxyphenyl group. The extension of the phenyl group in compounds <b>7</b> and <b>8</b> led to improved potency and selectivity compared to compound <b>4</b>, suggesting that the flexibility we introduced allows the R<sup>3</sup> position to contribute additional interactions with the target site. Usually, the two-carbon linker’s flexibility allows the R3 position to contribute additional interactions with the active site. Compound <b>6</b> is an exception that may be due to a different binding mode/preferred orientation of compound <b>6</b> in the drug–receptor complex. The lifetime of the drug–receptor complex is a dynamic process. This can also be explained by the duration of time that the drug–receptor complex persists, i.e., residence time—maybe it readily dissociates from their target receptor. We continued to explore whether this improved potency and selectivity could apply to other acetyl lysine mimics. By taking advantage of our two-step synthesis, other warheads like 3-methyl-3,4-dihydroquinazolin-2(1<i>H</i>)-one (compound <b>9</b>), 3-methyl-2<i>H</i>-indazole (compound <b>10</b>), and 3-methyl-3a,7a-dihydrobenzo[<i>d</i>]isoxazole (compound <b>11</b>) were introduced to the core via Suzuki coupling. These warheads, however, showed a greater than 10-fold loss in inhibitory activity against CBP (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This is due to the difference in the moieties’ size, and the induced fit is different. The distance between the key/critical hydrogen-bond acceptors of the acetylated lysine mimic and the imidazopyridine core increases, leading to a decrease in the hydrogen-bonding affinity. Compounds <b>10</b> and <b>11</b> displayed greater than 15-fold increase in the inhibitory activity against BRD4 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This result reinforced our interest in using 3,5-dimethylisoxazole as the warhead for CBP bromodomain inhibitor development. We next explored the R<sup>1</sup> position by introducing different polar and hydrophobic groups. Use of the cyclohexyl group at R<sup>1</sup> while maintaining a methoxy group at R<sup>3</sup> led to a potent and selective inhibitor, compound <b>16</b>, with CBP and BRD4 inhibitory potencies of 0.159 and 6.59 μM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). When <i>t</i>-butyl (compounds <b>13</b> and <b>18</b>) and adamantyl (compound <b>14</b>) groups were introduced at the R<sup>1</sup> position, the CBP inhibitory activity decreased by >5-fold, while the use of 4-ethylmorpholinyl (compound <b>15</b>) and isopropyl (compound <b>17</b>) led to roughly a 2- and 3-fold decrease in potency, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Unfortunately, use of benzyl groups for R<sup>1</sup>, compounds <b>21</b> and <b>22</b>, led to a 10-fold decrease in the CBP inhibitory potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). At the same time, we observed that compound <b>16</b> exhibited a 5-fold increase in the CBP inhibitory potency compared to compound <b>7</b>, which lacks an electron-donating group, <i>p</i>-methoxy, on the aryl ring.</div><div class="NLM_p">Subsequently, we assessed the impact of different alkyl groups at the R<sup>3</sup> position (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The ethoxy group (compound <b>19</b>) showed a subtle decrease in potency against the CBP bromodomain compared to compound <b>16</b>, while the introduction of the propyloxyl group (compound <b>20</b>) caused a ∼4-fold decrease in potency (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Therefore, the <i>p</i>-methoxy group on the aryl ring was retained as it proved to be important for the increased CBP inhibitory activity without compromising selectivity (41-fold) against BRD4.</div><div class="NLM_p">Molecular docking confirmed that the presence of an electron-donating group, <i>p</i>-methoxy, on the aryl ring further reinforces the cation−π interactions of the CBP inhibitor with the Arg1173 of the CBP bromodomain (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Additionally, the hydrophobic methyl in the methoxy group of compound <b>16</b> stabilizes solvent-accessible hydrophobic residues on the protein side (PRO1106, Val1174, and Phe1177). Compound <b>16</b> exhibited improved binding affinity to CBP compared to compounds <b>3</b>, <b>4</b>, and <b>7</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docked poses of compounds <b>7</b> (UMB276, green) and <b>16</b> (UMB295, yellow) in CBP. PDBs (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a> for CBP).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our highly efficient two-step synthesis allowed us to quickly explore the SAR around the core of compound <b>16</b> by introducing substituent group(s) on the phenyl ring of the <i>p</i>-methoxyphenyl group as well as by changing the distance between the phenyl ring and imidazo[1,2-<i>a</i>]pyridine (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). First, we hypothesized that the introduction of a halogen group could further improve the potency against CBP by promoting electrostatic interactions with the target and also increase lipophilicity of the compound, consistent with previous reports.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Indeed, introduction of an <i>m</i>-chloro group at the aryl ring resulted in a more potent and selective CBP inhibitor, compound <b>23</b>, CBP IC<sub>50</sub> 72 nM and BRD4 IC<sub>50</sub> 5193 nM, which was 2-fold more potent than compound <b>16</b>. Additionally, compound <b>23</b> displayed 72-fold selectivity for CBP over BRD4 as a selective CBP inhibitor (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Notably, we discovered that the distance between the phenyl ring and the imidazo[1,2-<i>a</i>]pyridine is also important, as evidenced by the large impact on potency seen with different linker lengths. Reducing the length of the linker to methylene (compound <b>24</b> compared to compound <b>16</b>) or increasing the length (compounds <b>25</b> and <b>30</b> compared to compound <b>23</b>) led to a loss of CBP inhibitory activity while maintaining BRD4 activity (less than 2-fold). Therefore, we identified the optimal linker for this scaffold and the importance of introducing the chloro-methoxyphenyl group. In this compound series, cyclohexyl was the optimal group for R<sup>1</sup>, while other groups led to a loss of activity (compounds <b>26</b>–<b>28</b> and <b>34</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>)). Since an electron-donating moiety was well-tolerated on the aryl group connecting the linker (compound <b>23</b>), we decided to introduce two hydrogen-bond acceptors, <i>m</i>- and <i>p</i>-methoxy groups. Surprisingly, the CBP inhibitory potency was lost by ∼10-fold, with insignificant change in the BRD4 activity (compound <b>32</b>) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), indicating that the <i>m</i>-chloro group was important for improved activity of compound <b>23</b>. Similarly, use of isopropyl for R<sup>1</sup>, compound <b>33</b>, led to 49- and 4-fold loss in CBP and BRD4 inhibitory activities, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structure–Activity Relationship on Compound <b>19</b> Analogues<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0014.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>n</i></th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">CBP</th><th class="colsep0 rowsep0" align="center">BRD4</th><th class="colsep0 rowsep0" align="center" char=".">selectivity<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b> (UMB295)</td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.16 ± 0.02</td><td class="colsep0 rowsep0" align="left">6.59 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b> (UMB298)</td><td class="colsep0 rowsep0" align="left"><b>cyclohexyl</b></td><td class="colsep0 rowsep0" align="char" char="."><b>2</b></td><td class="colsep0 rowsep0" align="left"><b>OMe</b></td><td class="colsep0 rowsep0" align="left"><b>Cl</b></td><td class="colsep0 rowsep0" align="left"><b>0.072 ± 0.002</b></td><td class="colsep0 rowsep0" align="left"><b>5.19 ± 0.02</b></td><td class="colsep0 rowsep0" align="char" char="."><b>72</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">6.07 ± 0.04</td><td class="colsep0 rowsep0" align="left">11.22 ± 1.01</td><td class="colsep0 rowsep0" align="char" char=".">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4.95 ± 0.41</td><td class="colsep0 rowsep0" align="left">12.33 ± 1.11</td><td class="colsep0 rowsep0" align="char" char=".">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">2,2,4-trimethylpentyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">20.89 ± 2.08</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">2,2,4-trimethylpentyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">OEt</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">38.49 ± 7.12</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">2,2,4-trimethylpentyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">O<i>i</i>Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">OEt</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">2.53 ± 0.09</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">8.96 ± 0.91</td><td class="colsep0 rowsep0" align="left">24.53 ± 2.33</td><td class="colsep0 rowsep0" align="char" char=".">3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">2,2,4-trimethylpentyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">OEt</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">20.46 ± 2.24</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">>2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">cyclohexyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">0.75 ± 0.06</td><td class="colsep0 rowsep0" align="left">6.21 ± 0.46</td><td class="colsep0 rowsep0" align="char" char=".">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">isopropyl</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">3.51 ± 0.08</td><td class="colsep0 rowsep0" align="left">22.64 ± 0.28</td><td class="colsep0 rowsep0" align="char" char=".">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">2,2,4-trimethylpentyl</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="char" char=".">1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">All IC50 values are reported as means of values from at least two determinations. AlphaScreen assay with the isolated CBP (with compound C) or BRD4 (with biotinylated JQ1) bromodomain.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Selectivity is defined by [BRD4(1) IC50 (μM)/CBP IC50 (μM)] for the consistency mentioned.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Bold indicates the most active compound.</p></div></div></div><div class="NLM_p">The docking model provided the explanation to this improvement. Introduction of the halogen group (<i>m</i>-Cl) in compound <b>23</b> further stabilizes the solvent-accessible environment around the aryl group and R1173 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A), thus improving the binding affinity. Analysis of the binding poses of compound <b>23</b> or CBP30 with CBP showed no clear difference between the benzimidazole of CBP30 and the imidazopyridine of compound <b>23</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). The low affinity for BRD4 of the three compounds is due to the methyl (in the OMe group) and cyclohexyl groups being completely solvent-exposed in BRD4 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). R1173 in CBP anchors the ethylene linker in compounds <b>7</b>, <b>16</b>, and <b>23</b> by forming cation−π interactions with the aryl group. However, R1173 is replaced by D145 in BRD4. This amino acid difference abolishes the receptor’s ability to form cation−π interactions with molecules in the binding pocket of BRD4 and rationalizes the selectivity that emerges for compounds <b>7</b>, <b>16</b>, and <b>23</b>. Overall, the docking analyses of compounds <b>4</b>, <b>7</b>, <b>16</b>, and <b>23</b> with CBP and BRD4 bromodomains were consistent with the binding affinities obtained in our biochemical assays.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Docking pose of compound <b>23</b>. (A) Docked poses of compound <b>23</b> (UMB298) (magenta) in CBP. (B) Docked poses of compounds <b>7</b> (UMB276) (green), <b>16</b> (UMB295) (yellow), and <b>23</b> (UMB298) (magenta) in BRD4. Water molecules are denoted W, and residues are denoted by their three-letter amino acid code and their residue number in their respective PDBs (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a> for CBP and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4">5BT4</a> for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=BRD4">BRD4</a>). Hydrogen bonds between the molecule and surrounding residues and water are shown with red lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison of the cocrystal structure of compound <b>23</b> with CBP30. (A) Crystal structures of CBP30 bound to CBP (left, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a>) and (B) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=BRD4">BRD4</a> (right, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4">5BT4</a>). Water molecules are denoted W, and residues are denoted by their three-letter amino acid code. Hydrogen bonds among the molecule, surrounding residues, and water are shown with red lines. (C) Compound <b>23</b> (UMB298) (yellow) bound to the KAc site of CBP (gray) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KPY">7KPY</a>). H-bonds with Asn1168 (magenta) and water molecules (cyan) are indicated as black dotted lines. The inset shows the 2Fo-Fc electron density of the inhibitor. (D) Schematic presentation of H-bonding (black dotted lines) and hydrophobic van der Waals interactions (green wiggle lines) of UMB298 with KAc site residues. Arg1173 appears to establish cation−π interactions with the chloromethoxybenzene ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Excitingly, we obtained the cocrystal structure of compound <b>23</b> (UMB298) with the bromodomain of CBP (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C,D). As expected from our docking model, the crystal structure confirmed that the 1,3-dimethylisoxazole ring acted as an acetyl lysine mimetic to interact with the conserved Asn1168. The fused imidazo[1,2-<i>a</i>]pyridine core showed the same orientation as CBP30 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A) and extended the chloro, methoxyl phenyl group into the pocket formed by Arg1173 and Phe1177, as predicted by docking. This extension further explains the CBP selectivity we observed with our lead molecule <b>23</b> and confirms the accuracy of our docking model.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selectivity Assessment and Cellular Activity</h3><div class="NLM_p">To further confirm the selectivity observed in our biochemical assay, we utilized differential scanning fluorometry (DSF) to measure the binding of compounds with BRD4 BD1 (the first bromodomain of BRD4), the CBP bromodomain, and the EP300 (P300) bromodomain (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf" class="ext-link">Figures S1</a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A). Compound <b>7</b> showed very modest activity toward BRD4 but was more active against CBP and EP300. Compounds <b>16</b> and <b>23</b> showed greater selectivity than the known inhibitor, CBP30, with a slightly reduced potency. Further profiling using BROMOSCAN against 34 bromodomains at 1 μM compound concentration showed promising binding and selectivity values for the three compounds, with compound <b>23</b> having a better binding and selectivity compared to compounds <b>7</b> and <b>16</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf" class="ext-link">Table S1</a>). The binding affinity (<i>K</i><sub>d</sub>) values of compounds <b>16</b> and <b>23</b> were further confirmed by BromoELECT<sup>SM</sup> with CBP, EP300, and BRD4 (BD1). We observed similar binding trends and comparable selectivity to our AlphaScreen data.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Potency and selectivity profiling. (A). Tabulated temperature shift values of compounds <b>7</b>, <b>16, 23</b>, and CBP30 with CBP (bromodomain), EP300 (bromodomain), and BRD4(BD1). (B). BROMOSCAN data on compounds <b>7</b>, <b>16</b>, and <b>23</b> against 34 bromodomains. (C). <i>K</i><sub>d</sub> value of compounds <b>16</b>, <b>23</b>, and biotinylated probe compound C toward CBP, EP300, and BRD4 (BD1) determined by BromoELECT. All <i>K</i><sub>d</sub> values are measured in duplicates following the standard protocol developed by Eurofins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next evaluated the cellular activity of our lead molecule <b>23</b> in the AML cell line MOLM13 and several multiple myeloma lines in comparison with CBP30. Compound <b>23</b> inhibited MOLM13 and MM cell growth with IC50 values similar to CBP30 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf" class="ext-link">S2</a>). These compounds can inhibit leukemia cell growth at high concentrations as described in the literature, where the CBP/EP300 bromodomain inhibitor alone has limited effect on blocking CBP/EP300 functions.<a onclick="showRef(event, 'ref18 ref35'); return false;" href="javascript:void(0);" class="ref ref18 ref35">(18,35)</a> We further investigated on-target inhibition of our compounds by checking the impact of bromodomain inhibition on H3K27ac and Myc expression levels. Treatment with compound <b>23</b> reduced the H3K27ac level similar to CBP30 and caused MYC depletion as a signature of CBP inhibition in AML (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). The MYC-level reduction was further confirmed by reverse transcription-polymerase chain reaction (RT-PCR) experiments (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C). To confirm whether compound <b>23</b> exerted MYC deletion through on-target CBP/EP300 inhibition, we assessed the effect of the selective CBP inhibitor (CBP30 and compound <b>23</b>) for their effect on IRF4/MYC function in the multiple myeloma cell line (MM.1S)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> using compounds <b>10</b> and <b>11</b>, selective toward BRD4 as the negative control. We observed that the IRF4/MYC axis was perturbed by CBP30 and compound <b>23</b> at similar levels but not with the BRD4 inhibitor as we expected (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Compound <b>23</b> showed similar but slightly weaker cellular activity compared to CBP30, which matches the binding activities we observed in biochemical assays. Thus, the core we built with our GBB reaction can produce the EP300/CBP inhibitors with good cellular activity.</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cellular activity of CBP/EP300 inhibitors in multiple myeloma cell lines. (A) Dose-proportional effect of CBP30 and compound <b>23</b> (UMB298) (5 days) on MOLM13 cellular viability as approximated by ATP-dependent luminescence (means ± standard error of the mean, SEM, <i>n</i> = 4). (B) Immunoblot of MYC, β-actin, H2K27ac, and H3 following 2 h of incubation with dimethyl sulfoxide (DMSO) or the indicated concentrations of CBP30 or compound <b>23</b> (UMB298) in MOLM13 cells. (C) Quantitative reverse transcription-polymerase chain reaction (RT-PCR) analysis of transcript levels of MYC after a 2 h treatment of MOLM13 cells with DMSO, 3.0 μM CBP30, or 3.0 μM UMB298. Values represent three biologic replicate means ± SEM. *<i>P</i> = 0.01–0.05; **<i>P</i> = 0.001– 0.01; ***<i>P</i> = 0.0001–0.001; ****<i>P</i> < 0.0001. (D) Immunoblot of MYC, IRF4 following 24 h incubation with DMSO or 2.5 μM CBP30, compound <b>23</b> (UMB298), compound <b>10</b>, and compound <b>11</b> in MM.1S cells. β-actin is used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72185" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72185" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Developing potent inhibitors that can disrupt protein–protein interactions remains a challenging but attractive task for drug development.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In this work, we developed an effective synthetic method to assemble the core of a panel of potent bromodomain inhibitors. Our two-step synthesis involved the Groebke–Blackburn–Bienayme (GBB) reaction<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> to build the core and decorate the core with various functional groups and different appendix groups in a relatively high-throughput manner. The Suzuki coupling installed the acetyl lysine mimetic with a high overall yield (>90%). The approach generated a broad spectrum of compounds with different potencies and selectivities against CBP over BRD4. In parallel, we established a computational model that explains the observed potency and selectivity. The model provided valuable information, which aided the rational design of novel potent and selective inhibitors. More importantly, this model and the related explanation have been further confirmed by the cocrystal structure of our lead molecule compound <b>23</b> with the CBP bromodomain. The lead compound <b>23</b> built upon the imidazo[1,2-<i>a</i>]pyridine scaffold with optimization on other parts (R<sup>1–3</sup>) of the molecule showed comparable potency and selectivity with other reported CBP inhibitors in biochemical and cellular assays. Compared to the synthetic approaches previously employed for the synthesis of reported CBP inhibitors, our method is simple and resource-efficient, allowing quick and comprehensive SAR studies. Compound <b>23</b> qualifies to be used to assist in the interrogation of the biological roles of this protein in diseases. More importantly, we envision that our novel and effective approach can be utilized for bromodomain inhibitor development targeting other bromodomains outside the BET and CBP/EP300 families.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div class="NLM_p last">The synthesis of the imidazo[1,2-<i>a</i>]pyridine scaffold-based CBP inhibitors was done in two steps: three-component reaction<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> followed by the Suzuki coupling.<a onclick="showRef(event, 'ref30 ref38'); return false;" href="javascript:void(0);" class="ref ref30 ref38">(30,38)</a> Synthesis of compound <b>23</b> involved a multicomponent reaction by combining 4-bromopyridin-2-amine, 3-(3-chloro-4-methoxyphenyl)propanal, and cyclohexylisocyanide under microwave irradiation for 40 min. This was followed by the Suzuki coupling of the intermediate with 3,5-dimethylisoxazole-4-boronic acid pinacol ester. Both steps had yields and produced the final product with an overall good yield. It is a green synthesis with high overall yield as well as atom- and step-economy. Previously, the Brennan group reported CBP30, a potent and selective CBP inhibitor (CBP, IC<sub>50</sub> = 79 nM), with a selectivity window of 40-fold against BRD4.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> It contains a benzimidazole scaffold.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The synthesis of CBP30 involved five steps through a linear or multistep synthesis, leading to a low overall yield of about 4%.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> In this study, an imidazo[1,2-<i>a</i>]pyridine scaffold was used, with substituent groups around the imidazo[1,2-<i>a</i>]pyridine scaffold varied by use of different aldehydes, isocyanides, and also warheads as acetyl lysine mimic, 3,5-dimethylisoxazole, 3-methyl-2<i>H</i>-indazole, 3-methyl-3a,7a-dihydrobenzo[<i>d</i>]isoxazole, and 3-methyl-3,4-dihydroquinazolin-2(1<i>H</i>)-one to obtain a panel of compounds (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The synthesized compounds were screened for their inhibitory activity against CBP and BRD4 using the AlphaScreen assay to obtain IC<sub>50</sub> values.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> General Synthetic Information</h3><div class="NLM_p last">Chemicals and solvents were purchased from commercial suppliers and used as received. All of the compounds used for the biological assay were >95% pure as determined by NMR and liquid chromatography–mass spectrometry (LC–MS). <sup>1</sup>H NMR (400 MHz) and <sup>13</sup>C NMR spectra (101 MHz) were recorded on Agilent NMR spectrometers. The chemical shifts were reported in parts per million (ppm), and the residual solvent peak was used as an internal reference: proton (chloroform δ 7.26) and carbon (chloroform δ 77.0). Multiplicities were indicated as follows: s (singlet), d (doublet), t (triplet), q (quartet), m (multiplet), dd (doublet of doublet), and br s (broad singlet). Coupling constants were reported in hertz (Hz). LC–MS was performed on an Agilent 2100 LC with a 6130 quadruple mass spectrometer (MS), and a C18 column (5.0 μm, 6.0 mm × 50 mm) was used for separation. The mobile phases were MeOH and H<sub>2</sub>O, both containing 0.01% trifluoroacetic acid. A linear gradient of 50:50 (v/v) MeOH/H<sub>2</sub>O to 100% MeOH was used over 7.0 min at a flow rate of 0.7 mL/min. The chromatograms were detected at UV wavelengths 210, 254, and 365 nm. Low-resolution mass spectra were recorded in APCI (atmospheric pressure chemical ionization). The microwave reactions were performed on a Biotage Initiator 8 system equipped with an infrared (IR) sensor (external surface sensor) to monitor the reaction temperature. Flash chromatography separation was performed on the YAMAZEN AI-580 system with Agela silica gel (12 or 20 g, 230–400 μm mesh) cartridges. Some of the final products were purified using an Agela HP-100 pre-LC system with a Venusil PrepG C18 column (10 μm, 120 Å, 21.2 mm × 250 mm). HRMS was analyzed by RP-LC–MS: 1 μL of each sample was combined and 500 mL of optima-grade MeCN (0.1% formic acid) and 500 mL of optima-grade H<sub>2</sub>O (0.1% formic acid) were added to the mixture. This mixture was then diluted by another factor of 10 with a 75/25 mixture of the optima-grade MeCN and H<sub>2</sub>O. Then, 1 mL (10 fmol of each on the column) of this mixture was analyzed by RP-LC–MS. The mass analyzer was Orbitrap.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthesis of Biotinylated-CBP/P300 Bromodomain Binder <b>C</b></h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 5-(4-(4-(3-(Cyclohexylamino)-6-(3,5-dimethylisoxazol-4-yl)imidazo[1,2-<i>a</i>]pyridin-2-yl)phenyl)piperazin-1-yl)-5-oxopentanoic Acid (Compound <b>B</b>)</h4><div class="NLM_p last">To the solution of <b>A</b> (20.0 mg, 0.0394 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) (13.2 μL, 0.0788 mmol) in dichloromethane (DCM) (2 mL) was added glutaric anhydride (6.74 mg, 0.0591 mmol) at room temperature. The mixture was stirred at room temperature for 4 h and then purified via ISCO (MeOH/DCM = 0–15%) to afford <b>B</b> as a colorless oil (18.0 mg, 78% yield). MS: <i>m</i>/<i>z</i> (M + 1)<sup>+</sup>: 585.27.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 5-(4-(4-(3-(Cyclohexylamino)-6-(3,5-dimethylisoxazol-4-yl)imidazo[1,2-<i>a</i>]pyridin-2-yl)phenyl)piperazin-1-yl)-5-oxo-<i>N</i>-(15-oxo-19-((4<i>S</i>)-2-oxohexahydro-1<i>H</i>-thieno[3,4-<i>d</i>]imidazol-4-yl)-4,7,10-trioxa-14-azanonadecyl)pentanamide (Compound <b>C</b>)</h4><div class="NLM_p last">To a solution of <b>B</b> (18.0 mg, 0.031 mmol), <i>N</i>-(13-amino-4,7,10-trioxatridecanyl)-<span class="smallcaps smallerCapital">d</span>-biotinamide (CAS #183896-00-6) (13.8 mg, 0.031 mmol), and HATU (23.6 mg, 0.062 mmol) in dimethylformamide (DMF) (1 mL) was added DIPEA (10.4 mL, 0.062 mmol) at room temperature for 1 h. After workup of the reaction, the residue was purified by high performance liquid chromatography (HPLC) (0.1% TFA/MeCN) to afford <b>C</b> (3.5 mg, 11% yield). MS: <i>m</i>/<i>z</i> (M + 1)<sup>+</sup>: 1013.61.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Representative Procedure for the Three-Component Reaction</h3><div class="NLM_p last">A 10 mL microwave reaction tube was charged with 4-bromopyridin-2-amine (1 mmol, 1 equiv), aldehyde, e.g., 3-(3-chloro-4-methoxyphenyl)propanal (1.2 mmol, 1.2 equiv), isocyanide, e.g., cyclohexylisocyanide (1 mmol, 1 equiv), and Sc(OTf)<sub>3</sub> (0.08 mmol, 0.08 equiv) in 2 mL of 3:1 CH<sub>2</sub>Cl<sub>2</sub>/MeOH (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The reaction mixture was heated under microwave irradiation at 100 °C for 40 min. The mixture was concentrated and purified by flash chromatography eluted with 20% EtOAc in hexane to provide the product as a yellow solid (95–97%).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Representative Procedure for the Suzuki Reaction</h3><div class="NLM_p">To a solution of Groebke–Blackburn–Bienayme reaction adduct (1 mmol, 1 equiv), 3,5-dimethylisoxazole-4-boronic acid pinacol ester (1.2 mmol, 1.2 equiv), K<sub>2</sub>CO<sub>3</sub> (2.0 mmol, 2.0 equiv), and Pd(dppf)Cl<sub>2</sub>·CH<sub>2</sub>Cl<sub>2</sub> (0.08 mmol, 0.08 equiv 8% mol) in 2 mL of 2:1 DME/H<sub>2</sub>O were heated under microwave irradiation at 120 °C for 40 min (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The mixture was concentrated and purified by flash chromatography eluted with 30% EtOAc in hexane to provide the desired product in 90–97% yield.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>N</i>-((3<i>s</i>,5<i>s</i>,7<i>s</i>)-Adamantan-1-yl)-2-(4-methoxyphenyl)-5,7-dimethylimidazo[1,2-<i>a</i>]pyridin-3-amine (<b>1</b>, 91% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.76–7.69 (m, 2H), 7.15 (s, 1H), 6.97–6.93 (m, 2H), 6.27 (s, 1H), 3.86 (s, 3H), 3.49 (s, 1H), 2.93 (s, 3H), 2.32 (s, 3H), 1.52–1.40 (m, 6H), 1.35–1.25 (m, 2H), 0.97–0.87 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 158.76, 143.62, 136.03, 134.63, 129.72, 128.69, 116.43, 113.96, 113.60, 56.62, 55.57, 55.23, 42.97, 36.25, 29.64, 20.96, 20.89. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>26</sub>H<sub>31</sub>N<sub>3</sub>O 402.25454, found 402.2545.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>N</i>-((3<i>s</i>,5<i>s</i>,7<i>s</i>)-Adamantan-1-yl)-7-(3,5-dimethylisoxazol-4-yl)-2-(4-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>2</b>, 90% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.31 (d, <i>J</i> = 8.0 Hz, 1H), 7.91–7.88 (m, 2H), 7.71 (dd, <i>J</i> = 5.8, 3.3 Hz, 1H), 7.53 (dd, <i>J</i> = 5.7, 3.3 Hz, 1H), 7.00–6.96 (m, 2H), 3.88 (s, 3H), 3.49 (s, 1H) 2.49 (s, 3H), 2.35 (s, 3H), 1.45–1.39 (m, 3H), 1.37–1.28 (m, 12H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.76, 159.02, 158.31, 147.09, 144.52, 141.71, 139.80, 132.41, 130.88, 129.26, 128.79, 123.56, 116.50, 113.71, 112.14, 56.66, 55.25, 43.91, 38.70, 36.13, 30.34, 29.66, 28.91, 23.72, 22.99, 14.07, 11.87, 10.96. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>32</sub>N<sub>4</sub>O<sub>2</sub> 469.26036, found 469.2604.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>N</i>-Cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)-2-(4-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>3</b>, 89% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.15 (d, <i>J</i> = 8.0 Hz, 1H), 8.00 (s, 1H), 7.72 (d, <i>J</i> = 3.3 Hz, 1H), 7.54–7.52 (m, 2H), 7.41 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.03–6.99 (m, 2H), 3.87 (s, 3H), 3.00 (t, <i>J</i> = 10.2 Hz, 1H), 2.48 (s, 3H), 2.35 (s, 3H), 1.68 (dd, <i>J</i> = 12.2, 6.1 Hz, 4H), 1.46–1.39 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.75, 165.76, 158.55, 144.63, 141.26, 132.40, 130.87, 128.77, 128.26, 124.30, 122.69, 116.62, 113.99, 112.47, 56.95, 55.27, 38.69, 30.32, 28.90, 23.71, 22.97, 14.05, 10.95. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>25</sub>H<sub>28</sub>N<sub>4</sub>O<sub>2</sub> 417.22907, found 417.2291.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 2-(4-Methoxyphenyl)-5,7-dimethyl-<i>N</i>-(2,4,4-trimethylpentan-2-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>5</b>, 92% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.70–7.68 (m, 1H), 7.67 (d, <i>J</i> = 2.2 Hz, 1H), 7.15 (s, 1H), 6.98–6.95 (m, 2H), 6.28 (s, 1H), 3.85 (s, 3H), 3.49 (s, 1H), 2.91 (s, 3H), 2.31 (s, 3H), 1.45 (s, 2H), 0.95 (s, 9H), 0.77 (s, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 158.88, 143.68, 135.93, 134.64, 130.86, 129.87, 128.77, 124.29, 116.44, 113.68, 60.55, 56.22, 55.26, 31.83, 30.33, 28.33, 22.98, 20.77, 14.05. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>24</sub>H<sub>33</sub>N<sub>3</sub>O 380.27019, found 380.2702.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-Cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)-2-phenethylimidazo[1,2-<i>a</i>]pyridin-3-amine (<b>7</b>, 88% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.22 (dd, <i>J</i> = 7.1, 0.9 Hz, 1H), 7.37 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.29 (dd, <i>J</i> = 7.9, 6.7 Hz, 2H), 7.23–7.19 (m, 3H), 6.64 (dd, <i>J</i> = 7.1, 1.7 Hz, 1H), 3.14–3.04 (m, 4H), 3.50 (1H), 2.48 (s, 3H), 2.35 (s, 3H), 1.66 (d, <i>J</i> = 2.8 Hz, 6H), 1.59 (d, <i>J</i> = 22.3 Hz, 4H), 0.91 (dt, <i>J</i> = 9.7, 7.3 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.70, 158.51, 142.13, 141.76, 141.19, 128.57, 128.41, 125.98, 125.67, 123.53, 122.93, 116.10, 115.25, 111.80, 68.12, 55.40, 43.90, 36.17, 30.39, 29.64, 24.86, 11.84, 11.04. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>26</sub>H<sub>30</sub>N<sub>4</sub>O 415.2480, found 415.2499.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-((3<i>s</i>,5<i>s</i>,7<i>s</i>)-Adamantan-1-yl)-7-(3,5-dimethylisoxazol-4-yl)-2-phenethylimidazo[1,2-<i>a</i>]pyridin-3-amine (<b>8</b>, 90% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (d, <i>J</i> = 7.9 Hz, 1H), 7.71 (dd, <i>J</i> = 5.7, 3.3 Hz, 1H), 7.39 (s, 1H), 7.29 (d, <i>J</i> = 6.8 Hz, 1H), 7.24–7.18 (m, 3H), 6.66 (dd, <i>J</i> = 7.0, 1.7 Hz, 1H), 3.09 (d, <i>J</i> = 7.0 Hz, 2H), 3.07–3.02 (m, 2H), 2.71 (s, 1H), 2.47 (s, 3H), 2.34 (s, 3H), 1.74 (dd, <i>J</i> = 22.8, 7.9 Hz, 4H), 1.45–1.30 (m, 6H), 1.15 (t, <i>J</i> = 10.1 Hz, 5H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.71, 158.51, 142.03, 141.17, 139.08, 132.40, 130.86, 128.77, 128.59, 128.41, 126.03, 125.54, 122.62, 116.32, 112.25, 57.15, 38.69, 35.84, 34.19, 29.93, 28.90, 25.70, 24.81, 22.97, 14.05, 11.00. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>34</sub>N<sub>4</sub>O 467.28109, found 467.2811.</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 7-(3,5-Dimethylisoxazol-4-yl)-2-phenethyl-<i>N</i>-(2,4,4-trimethylpentan-2-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>12</b>, 89%)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.17 (dd, <i>J</i> = 7.1, 1.0 Hz, 1H), 7.36 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.32–7.27 (m, 2H), 7.24–7.17 (m, 3H), 6.65 (dd, <i>J</i> = 7.1, 1.7 Hz, 1H), 3.49 (s, 1H), 3.13–3.05 (m, 4H), 2.48 (s, 3H), 2.34 (s, 3H), 1.59 (s, 2H), 1.14 (s, 6H), 1.07 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.72, 158.52, 142.09, 141.86, 141.32, 128.53, 128.44, 126.02, 125.72, 123.51, 116.21, 115.23, 111.86, 59.50, 56.77, 35.84, 31.94, 31.78, 30.54, 29.22, 11.84, 11.04. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>36</sub>N<sub>4</sub>O 445.29674, found 445.2967.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i>-(<i>tert</i>-Butyl)-7-(3,5-dimethylisoxazol-4-yl)-2-phenethylimidazo[1,2-<i>a</i>]pyridin-3-amine (<b>13</b>, 89% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.17 (dd, <i>J</i> = 7.1, 1.0 Hz, 1H), 7.37 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.31–7.27 (m, 2H), 7.22–7.18 (m, 3H), 6.64 (dd, <i>J</i> = 7.1, 1.7 Hz, 1H), 3.49 (s, 1H), 3.10–3.07 (m, 4H), 2.48 (s, 3H), 2.34 (s, 3H), 1.15 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.73, 158.51, 142.09, 141.81, 141.11, 128.55, 128.44, 126.01, 125.80, 123.44, 116.18, 111.88, 110.78, 55.37, 35.82, 30.94, 30.29, 24.85, 11.83, 11.03. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>24</sub>H<sub>28</sub>N<sub>4</sub>O 389.24415, found 389.2441.</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-((3<i>s</i>,5<i>s</i>,7<i>s</i>)-Adamantan-1-yl)-7-(3,5-dimethylisoxazol-4-yl)-2-(4-methoxyphenethyl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>14</b>, 87% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.23 (dd, <i>J</i> = 7.1, 0.9 Hz, 1H), 7.36 (dd, <i>J</i> = 1.8, 1.0 Hz, 1H), 7.14–7.10 (m, 2H), 6.85–6.81 (m, 2H), 6.64 (dd, <i>J</i> = 7.1, 1.7 Hz, 1H), 3.79 (s, 3H), 3.50 (s, 1H), 3.04 (td, <i>J</i> = 3.2, 1.8 Hz, 4H), 2.48 (s, 3H), 2.35 (s, 3H), 1.67–1.58 (m, 15H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 158.54, 157.88, 134.22, 129.48, 123.54, 116.11, 113.83, 111.82, 55.44, 55.29, 43.92, 43.87, 36.18, 34.88, 30.62, 29.66, 11.87, 11.07. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>36</sub>N<sub>4</sub>O<sub>2</sub> 497.29166, found 497.2917.</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>N</i>-Cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)-2-(4-methoxyphenethyl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>16</b>, 90% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (dd, <i>J</i> = 7.0, 0.9 Hz, 1H), 7.36 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.13–7.09 (m, 2H), 6.85–6.81 (m, 2H), 6.65 (dd, <i>J</i> = 7.0, 1.7 Hz, 1H), 3.79 (s, 3H), 3.49 (s, 1H), 3.02 (dtd, <i>J</i> = 11.8, 5.9, 1.7 Hz, 4H), 2.47 (s, 3H), 2.34 (s, 3H), 1.81–1.66 (m, 5H), 1.27–1.09 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.68, 158.53, 157.89, 141.21, 139.34, 134.13, 129.47, 125.36, 122.62, 116.37, 115.31, 113.81, 112.15, 57.20, 55.26, 34.91, 34.22, 30.21, 25.72, 24.83, 11.81, 11.01. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>32</sub>N<sub>4</sub>O<sub>2</sub> 445.26036, found 445.2605.</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 7-(3,5-Dimethylisoxazol-4-yl)-<i>N</i>-isopropyl-2-(4-methoxyphenethyl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>17</b>, 88% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.25 (d, <i>J</i> = 7.1 Hz, 1H), 8.16 (s, 1H), 7.11 (dd, <i>J</i> = 7.0, 1.3 Hz, 1H), 7.08–7.04 (m, 2H), 6.83–6.79 (m, 2H), 4.27–4.15 (m, 1H), 3.78 (s, 3H), 3.49 (s, 1H), 3.15 (q, <i>J</i> = 8.2, 7.6 Hz, 2H), 3.09 (dd, <i>J</i> = 9.6, 6.4 Hz, 2H), 2.55 (s, 3H), 2.38 (s, 3H), 1.05 (s, 3H), 1.03 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.47, 158.35, 157.86, 137.33, 133.98, 132.14, 129.59, 126.49, 123.42, 116.40, 114.13, 113.52, 112.59, 55.32, 49.71, 38.68, 33.93, 27.25, 23.22, 12.12, 11.09. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>24</sub>H<sub>28</sub>N<sub>4</sub>O<sub>2</sub> 405.22907, found 405.2291.</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i>-Cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)-2-(4-ethoxyphenethyl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>19</b>, 90% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (dd, <i>J</i> = 7.0, 1.0 Hz, 1H), 7.37 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.11–7.07 (m, 2H), 6.83–6.78 (m, 2H), 6.66 (dd, <i>J</i> = 7.0, 1.7 Hz, 1H), 4.01 (q, <i>J</i> = 7.0 Hz, 2H), 3.49 (s, 1H), 3.07–2.97 (m, 4H), 2.74–2.62 (m, 1H), 2.47 (s, 3H), 2.34 (s, 3H), 1.80–1.68 (m, 4H), 1.40 (t, <i>J</i> = 7.0 Hz, 3H), 1.26–1.12 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.68, 158.51, 157.24, 141.16, 139.26, 133.95, 129.44, 125.41, 125.39, 122.62, 116.31, 115.29, 114.41, 112.18, 63.39, 57.19, 34.92, 34.20, 30.16, 25.71, 24.81, 14.88, 11.80, 11.00. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>34</sub>N<sub>4</sub>O<sub>2</sub> 459.27601, found 459.2760.</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2-(3-Chloro-4-methoxyphenethyl)-<i>N</i>-cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>23</b>, 90% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (dd, <i>J</i> = 7.0, 1.0 Hz, 1H), 7.36 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.23 (d, <i>J</i> = 2.2 Hz, 1H), 7.04 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 6.83 (d, <i>J</i> = 8.4 Hz, 1H), 6.66 (dd, <i>J</i> = 7.0, 1.7 Hz, 1H), 3.87 (s, 3H), 3.49 (s, 1H), 3.08–2.97 (m, 4H), 2.66–2.52 (m, 1H), 2.48 (s, 3H), 2.34 (s, 3H), 1.80–1.69 (m, 4H), 1.28–1.11 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.70, 158.50, 153.24, 141.27, 138.97, 135.23, 130.19, 127.75, 125.53, 122.64, 122.07, 116.37, 115.27, 112.23, 112.00, 57.24, 56.15, 34.47, 34.23, 29.74, 25.70, 24.81, 11.80, 11.00. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>27</sub>H<sub>31</sub>N<sub>4</sub>O<sub>2</sub>Cl 479.22139, found 479.2216.</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-Cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)-2-(3-(4-methoxyphenyl)propyl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>25</b>, 89% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.06 (dd, <i>J</i> = 7.0, 1.0 Hz, 1H), 7.35 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.15–7.11 (m, 2H), 6.86–6.81 (m, 2H), 6.66 (dd, <i>J</i> = 7.0, 1.7 Hz, 1H), 3.79 (s, 3H), 3.49 (s, 1H), 2.79 (q, <i>J</i> = 4.0, 3.5 Hz, 1H), 2.76–2.70 (m, 2H), 2.67 (t, <i>J</i> = 7.5 Hz, 2H), 2.46 (s, 3H), 2.32 (s, 3H), 2.11 (tt, <i>J</i> = 8.8, 6.8 Hz, 2H), 1.84–1.71 (m, 4H), 1.24 (s, 2H), 1.20–1.14 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.69, 158.54, 157.71, 141.17, 139.92, 134.13, 129.37, 125.31, 124.95, 122.53, 116.39, 115.33, 113.70, 112.20, 57.21, 55.26, 34.57, 34.22, 30.94, 26.46, 25.74, 24.85, 11.78, 10.99. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>34</sub>N<sub>4</sub>O<sub>2</sub> 459.27601, found 459.2761.</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 7-(3,5-Dimethylisoxazol-4-yl)-2-(4-methoxyphenethyl)-<i>N</i>-(2,4,4-trimethylpentan-2-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>26</b>, 91% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.18 (d, <i>J</i> = 7.0 Hz, 1H), 7.38 (s, 1H), 7.11–7.07 (m, 2H), 6.85–6.80 (m, 2H), 6.65 (dd, <i>J</i> = 7.1, 1.6 Hz, 1H), 3.79 (s, 3H), 3.04 (s, 3H), 2.48 (s, 3H), 2.46–2.35 (m, 1H), 2.35 (s, 3H), 2.32–2.20 (m, 1H), 1.25 (s, 2H), 1.14 (s, 6H), 1.08 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ <sup>13</sup>C NMR (101 MHz, cdcl<sub>3</sub>) δ 165.74, 158.50, 157.87, 134.08, 129.43, 123.83, 123.51, 116.11, 115.20, 113.82, 111.92, 59.50, 56.77, 55.21, 34.92, 31.93, 31.86, 31.77, 30.73, 29.23, 24.85, 11.85, 11.04. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>29</sub>H<sub>38</sub>N<sub>4</sub>O<sub>2</sub> 475.30732, found 475.3073.</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 7-(3,5-Dimethylisoxazol-4-yl)-2-(4-ethoxyphenethyl)-<i>N</i>-(2,4,4-trimethylpentan-2-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>27</b>, 90% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.17 (dd, J = 7.1, 0.9 Hz, 1H), 7.37 (dd, J = 1.7, 0.9 Hz, 1H), 7.12–7.03 (m, 2H), 6.86–6.77 (m, 2H), 6.65 (dd, J = 7.1, 1.7 Hz, 1H), 4.01 (q, J = 6.9 Hz, 2H), 3.49 (s, 1H), 3.03 (s, 4H), 2.48 (s, 3H), 2.34 (s, 3H), 1.58 (s, 2H), 1.41 (t, J = 7.0 Hz, 3H), 1.13 (s, 6H), 1.07 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.75, 158.53, 157.26, 141.77, 133.95, 129.43, 123.87, 123.53, 116.13, 115.22, 114.42, 111.91, 63.36, 59.51, 56.78, 34.96, 31.94, 31.79, 30.77, 29.24, 14.90, 11.86, 11.05. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>2</sub> 489.32297, found 489.3228.</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 7-(3,5-Dimethylisoxazol-4-yl)-2-(4-isopropoxyphenethyl)-<i>N</i>-(2,4,4-trimethylpentan-2-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>28</b>, 89% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.17 (dd, <i>J</i> = 7.0, 1.0 Hz, 1H), 7.38 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.08–7.04 (m, 2H), 6.82–6.78 (m, 2H), 6.65 (dd, <i>J</i> = 7.1, 1.7 Hz, 1H), 4.50 (p, <i>J</i> = 6.1 Hz, 1H), 3.49 (s, 1H), 3.03 (d, <i>J</i> = 2.0 Hz, 4H), 2.48 (s, 3H), 2.34 (s, 3H), 1.59 (s, 2H), 1.33 (d, <i>J</i> = 6.1 Hz, 6H), 1.13 (s, 6H), 1.07 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.74, 158.52, 156.16, 141.79, 141.27, 133.91, 129.44, 125.77, 123.85, 123.52, 116.12, 115.82, 115.23, 111.89, 75.00, 69.80, 59.49, 56.79, 34.96, 31.94, 31.77, 30.70, 29.21, 24.86, 22.10, 11.85, 11.04. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>31</sub>H<sub>42</sub>N<sub>4</sub>O<sub>2</sub> 503.33862, found 503.3388.</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 2-(3-Chloro-4-ethoxyphenethyl)-<i>N</i>-cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>29</b>, 91% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05 (dd, <i>J</i> = 7.1, 1.0 Hz, 1H), 7.36 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.22 (d, <i>J</i> = 2.2 Hz, 1H), 7.01 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 6.82 (d, <i>J</i> = 8.4 Hz, 1H), 6.66 (dd, <i>J</i> = 7.0, 1.7 Hz, 1H), 4.08 (q, <i>J</i> = 7.0 Hz, 2H), 3.49 (s, 1H), 3.07–2.96 (m, 4H), 2.53 (d, <i>J</i> = 14.2 Hz, 1H), 2.48 (s, 3H), 2.34 (s, 3H), 1.82–1.69 (m, 4H), 1.45 (t, <i>J</i> = 7.0 Hz, 3H), 1.32–1.04 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.71, 158.52, 152.70, 141.28, 139.04, 135.15, 130.19, 127.68, 125.50, 125.31, 122.65, 122.57, 116.40, 115.30, 113.44, 112.23, 64.80, 57.25, 34.54, 34.25, 29.78, 25.73, 24.83, 14.77, 11.82, 11.02. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>33</sub>N<sub>4</sub>O<sub>2</sub>Cl 493.23704, found 493.2371.</div></div><div id="sec4_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 2-(3-(3-Chloro-4-methoxyphenyl)propyl)-<i>N</i>-cyclohexyl-7-(3,5-dimethylisoxazol-4-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>30</b>, 90% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.07 (dd, <i>J</i> = 7.0, 1.0 Hz, 1H), 7.34 (s, 1H), 7.22 (d, <i>J</i> = 2.1 Hz, 1H), 7.07 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 6.85 (d, <i>J</i> = 8.4 Hz, 1H), 6.66 (dd, <i>J</i> = 7.0, 1.6 Hz, 1H), 3.88 (s, 3H), 3.49 (s, 1H), 2.74 (d, <i>J</i> = 7.4 Hz, 2H), 2.65 (t, <i>J</i> = 7.5 Hz, 2H), 2.46 (s, 3H), 2.45–2.40 (m, 1H), 2.32 (s, 3H), 2.11 (q, <i>J</i> = 7.8 Hz, 2H), 1.46–1.25 (m, 4H), 1.23–1.14 (m, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.68, 158.53, 153.08, 141.23, 139.82, 135.33, 130.19, 127.61, 125.34, 124.91, 122.54, 122.03, 116.44, 115.33, 112.22, 111.98, 106.09, 57.19, 56.19, 34.37, 34.27, 30.75, 26.46, 25.74, 24.86, 11.79, 10.99. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>28</sub>H<sub>33</sub>N<sub>4</sub>O<sub>2</sub>Cl 493.23704, found 493.2370.</div></div><div id="sec4_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-(3-Chloro-4-ethoxyphenethyl)-7-(3,5-dimethylisoxazol-4-yl)-<i>N</i>-(2,4,4-trimethylpentan-2-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>31</b>, 91% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.18 (dd, <i>J</i> = 7.1, 1.0 Hz, 1H), 7.35 (dd, <i>J</i> = 1.6, 0.9 Hz, 1H), 7.20 (d, <i>J</i> = 2.1 Hz, 1H), 6.98 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 6.81 (d, <i>J</i> = 8.4 Hz, 1H), 6.65 (dd, <i>J</i> = 7.1, 1.7 Hz, 1H), 4.08 (q, <i>J</i> = 7.0 Hz, 2H), 3.35 (s, 1H), 3.03 (s, 4H), 2.48 (s, 3H), 2.46–2.36 (m, 1H), 2.34 (s, 3H), 1.61 (s, 2H), 1.46 (t, <i>J</i> = 7.0 Hz, 3H), 1.14 (s, 6H), 1.09 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 165.74, 158.51, 152.69, 141.88, 141.04, 135.13, 130.16, 127.60, 125.83, 123.79, 123.52, 122.58, 116.20, 115.22, 113.41, 111.93, 64.76, 59.56, 56.82, 34.58, 31.94, 31.80, 30.41, 29.26, 14.77, 11.85, 11.04. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>2</sub>Cl 524.29182, found 524.2918.</div></div><div id="sec4_5_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-(3,4-Dimethoxyphenethyl)-7-(3,5-dimethylisoxazol-4-yl)-<i>N</i>-isopropylimidazo[1,2-<i>a</i>]pyridin-3-amine (<b>33</b>, 92% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.06 (d, <i>J</i> = 7.0 Hz, 1H), 7.38 (s, 1H), 6.78 (q, <i>J</i> = 8.2 Hz, 2H), 6.67 (d, <i>J</i> = 11.2 Hz, 2H), 4.22 (t, <i>J</i> = 6.0 Hz, 1H), 3.86 (s, 3H), 3.77 (s, 3H), 3.49 (s, 1H) 3.18 (h, <i>J</i> = 6.1 Hz, 2H), 2.47 (s, 3H), 2.33 (s, 3H), 1.48–1.37 (m, 2H), 1.06 (d, <i>J</i> = 6.3 Hz, 6H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.75, 165.70, 158.49, 148.73, 147.29, 141.27, 139.51, 134.63, 130.88, 128.78, 125.56, 122.61, 120.25, 116.32, 111.96, 111.20, 55.95, 55.71, 49.62, 38.70, 30.34, 28.91, 23.45, 14.06, 10.96. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>25</sub>H<sub>30</sub>N<sub>4</sub>O<sub>3</sub> 435.23964, found 435.2397.</div></div><div id="sec4_5_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 7-(3,5-Dimethylisoxazol-4-yl)-2-(3-(4-methoxyphenyl)propyl)-<i>N</i>-(2,4,4-trimethylpentan-2-yl)imidazo[1,2-<i>a</i>]pyridin-3-amine (<b>34</b>, 91% Yield)</h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.23 (d, <i>J</i> = 7.1 Hz, 1H), 8.10 (s, 2H), 7.11 (d, <i>J</i> = 8.5 Hz, 2H), 6.82 (d, <i>J</i> = 8.6 Hz, 2H), 3.78 (s, 3H), 3.49 (s, 1H), 2.76 (dd, <i>J</i> = 8.8, 6.6 Hz, 2H), 2.65 (t, <i>J</i> = 7.2 Hz, 2H), 2.48 (s, 3H), 2.34 (s, 3H), 2.15 (p, <i>J</i> = 7.5 Hz, 2H), 1.61 (s, 2H), 1.11 (s, 6H), 1.08 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.76, 165.96, 157.73, 134.21, 133.85, 132.42, 130.89, 129.48, 129.37, 128.79, 123.51, 113.72, 67.75, 59.68, 56.77, 38.88, 38.70, 31.92, 31.78, 30.56, 29.14, 23.97, 22.97, 14.07, 11.10. HRMS (ESI-Orbitrap) <i>m</i>/<i>z</i>: (M + H)<sup>+</sup> calcd for C<sub>30</sub>H<sub>40</sub>N<sub>4</sub>O<sub>2</sub> 489.32297, found 489.3229.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> BRD4 AlphaScreen Assay</h3><div class="NLM_p last">BRD4 (BD1) assays were performed with minimal modifications from the manufacturer’s protocol (PerkinElmer). All reagents were diluted in 50 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), 150 mM NaCl, 0.1% w/v bovine serum albumin (BSA), 0.01% w/v Tween-20, pH 7.5 and allowed to equilibrate to room temperature prior to addition to plates. After the addition of α beads to master solutions, all subsequent steps were performed under low light conditions. A 2× solution of components with final concentrations of His-BRD4 at 40 nM, Ni-coated acceptor beads at 25 μg/mL, and 20 nM biotinylated JQ1(S) was added in 10 μL to 384-well plates (AlphaPlate-384, PerkinElmer). Plates were spun down at 150g, after which 100 nL of the compound in DMSO from stock plates was added by pin-transfer using a Janus workstation (PerkinElmer). The streptavidin-coated donor beads (25 μg/mL final) were added in the same manner as the previous solution, in a 2× solution of 10 μL volume. Following this addition, plates were sealed with foil to prevent light exposure and evaporation. The plates were spun down again at 150g. Plates were incubated at room temperature for 1 h and then read on an Envision 2104 (PerkinElmer) using the manufacturer’s protocol.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> BRD4(1) and CBP AlphaScreen Assays</h3><div class="NLM_p">Assays were performed with minor modifications from the manufacturer’s protocol (PerkinElmer). All reagents were diluted in an AlphaScreen buffer (50 mM HEPES, 150 mM NaCl, 0.01% v/v Tween-20, 0.1% w/v BSA, pH 7.4). After addition of α beads to master solutions, all subsequent steps were performed under low light conditions. A 2× solution of components with final concentrations of His-BRD4(1) at 20 nM or His-CBP at 50 nM, Ni-coated acceptor bead at 10 μg/mL, biotinylated-JQ at 10 nM or biotinylated peptide at 100 nM or Compound C at 10 nM was added in 10 μL to 384-well plates (AlphaPlate-384, PerkinElmer) using an EL406 liquid handler (Biotek). Plates were spun down at 1000 rpm. A 10-point 1:<span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_m001.gif" alt="" id="_i45" /></img></span> serial dilution of compounds in DMSO was prepared at 200x the final concentration. Then, 100 nL of the compound from these stock plates was added by pin-transfer using a Janus workstation (PerkinElmer). A 2× solution of streptavidin-coated donor beads with a final concentration of 10 μg/mL was added in a 10 μL volume. The plates were spun down again at 1000 rpm and sealed with foil to prevent light exposure and evaporation. The plates were then incubated at room temperature for 1 h and read on an Envision 2104 (PerkinElmer) using the manufacturer’s protocol. IC<sub>50</sub> values were calculated using a four-parameter logistic curve in Prism 6 (GraphPad Software) after normalization to DMSO-treated negative control wells.</div><div class="NLM_p last">The profiling data and the <i>K</i><sub>d</sub> data are generated by Eurofins using BromoMAX<sup>SM</sup> and BromoKdELECT<sup>SM</sup> services.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Molecular Docking Experiments</h3><div class="NLM_p last">All steps for the docking experiments in this study were performed using the Schrödinger 2018-3 suite (Small-Molecule Drug Discovery Suite 2018-3: Maestro, Schrödinger, LLC, New York, NY, 2018). CBP (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a>) and BRD4 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4">5BT4</a>) were first processed to extract individual protein chains and remove the cocrystallized ligand using the <i>split_structures.py</i> command lines while keeping all crystal waters. Both structures were processed using the protein prep wizard command line.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> This step adds hydrogen atoms, assigns bond orders using CCD, caps termini, assigns protonation states using PROPKA with pH of 7, optimizes hydrogen bonds, and runs a restrained minimization of the full protein. The receptor grids were then generated for a 25 Å × 25 Å × 25 Å outer box and a 15 Å × 15 Å × 15 Å inner box centered at the centroid of pocket residues 6 Å away from the protein’s respective cocrystallized ligand. CBP’s pocket includes residues 1106, 1109–1111, 1113, 1115, 1120, 1122, 1125, 1164, 1167, 1168, 1173, 1174, and 1177, and BRD4’s pocket includes residues 79–89, 91, 92, 94, 97, 101, 105, 106, 132, 133, 135–140, 144, 145–147, 149, and 150. Smiles strings of compounds <b>3</b>, <b>4</b>, <b>7</b>, <b>16</b>, <b>23</b>, and CBP30 were generated using the BindingDB<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> molecular drawing tool. All molecules were converted to mae files using the <i>structconvert</i> command and processed using the <i>ligprep</i> command to add explicit hydrogen atoms using the Schrödinger 2018-3 suite. All generated ligands were docked into the pockets of both receptors using Glide with extra-precision docking and default parameters to generate 10 poses for each ligand–receptor complex. The top poses from each ligand were selected for analysis. The PDBs were chosen because their cocrystallized ligands were from the same chemical series, and the top poses resulted in very similar binding modes of the shared chemical moiety. As mentioned in the Methods section, PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4">5BT4</a> were used for CBP and BRD4, respectively. We also added the PDB codes to the figure legends for further clarity. <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> shows molecular graphics of the cocrystallized PDBs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4">5BT4</a>.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Protein Expression and Purification</h3><div class="NLM_p last">The expression plasmid for CBP BRD was purchased from Addgene (#38977) and transformed into <i>Escherichia coli</i> BL21 (DE3)-RIL cells, grown at 37 °C in an LB medium (Fisher Scientific) containing carbenicillin and chloramphenicol. At OD600 of 0.6, the culture was cooled down to 18 °C and induced with 0.1 mM IPTG. After 18 h growth, the culture was harvested by centrifugation at 6000<i>g</i> for 25 min and stored at −80 °C. Harvested cell pellets were resuspended in 50 mM Na/K phosphate buffer (pH 7.4) containing 100 mM NaCl, 40 mM imidazole, 0.01% w/v lysozyme, and 0.01% v/v Triton X-100 at 4 °C for 1 h, subjected to sonication, and the lysate was clarified by centrifugation (30 000<i>g</i> for 45 min at 4 °C). CBP BRD was purified by FPLC at 4 °C using columns and chromatography materials from GE Healthcare. The lysate was subjected to an immobilized Ni<sup>2+</sup> affinity chromatography column equilibrated with 50 mM Na/K phosphate buffer (pH 7.4) containing 100 mM NaCl and 40 mM imidazole using a gradient from 40 to 500 mM imidazole. Fractions containing the target protein were combined and incubated overnight with TEV protease at 4 °C, and the cleaved His6-tag was removed by a second Ni<sup>2+</sup> affinity column. CBP was purified to homogeneity by size-exclusion chromatography using Superdex 75 in elution buffer 50 mM HEPES/100 mM NaCl/1 mM DTT (pH 7.5). CBP BRD eluted as a monomer and was of crystallization-grade quality (>95% purity as determined by sodium dodecyl sulfate polyacrylamide gel electrophoresis, SDS-PAGE). Protein was concentrated to 11 mg/mL; aliquots were flash-frozen in liquid N<sub>2</sub> and stored at −80 °C.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Crystallization and X-ray Crystallography</h3><div class="NLM_p last">Crystallization experiments were performed at 19 °C. Purified cleaved CBP BRD was subjected to screening campaigns across different precipitants using a mosquito crystallization robot (TP Labtech) in 96-well sitting drop iQ plates. CBP (10 mg/mL) was incubated with 2 mM UMB289 and 5% DMSO on ice for 30 min prior to addition of the precipitant in ratios 1:2, 1:1, and 2:1 per condition. X-ray-grade crystals grew from 0.1 M Bis-Tris pH 5.5, 25% w/v poly(ethylene glycol) 3,350. For data collection, crystals were cryoprotected using a precipitant supplemented with 25% ethylene glycol and flash-frozen in liquid nitrogen. X-ray diffraction data were collected at −180 °C using beamline GMCA 23ID-D of the Advanced Photon Source at Argonne National Laboratories. Data were reduced and scaled with XDS.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> PHASER<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> was employed for molecular replacement using PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a> as the search model. Refinement was carried out with PHENIX,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and model building was performed with Coot.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The initial inhibitor model was generated with ligand restraints from eLBOW of the PHENIX suite. Figures were prepared using PyMOL (Schrödinger). Data collection and refinement statistics are shown in Supporting Information <a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf" class="ext-link">Table S2</a>. The coordinates and structure factors of the CBP–UMB298 (compound <b>23</b>) complex have been deposited in the PDB (accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KPY">7KPY</a>).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Western Blots</h3><div class="NLM_p last">MOLM13 cells were seeded in 6-well plates at 1 × 10<sup>6</sup> cells/well and treated with DMSO, 3.0 μM CBP30, or 3.0 μM compound <b>23</b> for 2 h. Total protein was extracted from cells using the RIPA buffer (Abcam) supplemented with the Halt Protease Inhibitor Cocktail (Millipore-Sigma). Samples were loaded to Invitrogen Bolt Bis-Tris Plus gels (ThermoFisher Scientific) and transferred to Invitrolon PVDF membranes (ThermoFisher Scientific) using the Invitrogen Bolt wet-gel Transfer Device. Immunodetection was performed with standard techniques. The following primary and secondary antibodies were used: anti-MYC, β-actin, H3 (Cell Signaling Technology), anti-H3K27ac (abcam), and IRDye secondary antibodies at the concentration recommended by the manufacturer.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Real-Time qPCR</h3><div class="NLM_p last">Cells were treated with DMSO, 3.0 μM CBP30, or 3.0 μM compound <b>23</b> for 2 h, and total RNA was isolated using the RNeasy kit (Qiagen). Three biological replicates were performed for each condition. cDNA was synthesized using the Tetro cDNA synthesis kit (Thomas Scientific), and real-time qPCR was performed using the PowerUp SYBR Green Master Mix (Life Technologies) on Applied Biosystems ViiA 7 Real-Time PCR System (Applied Biosystems). The following primers were utilized: MYC.F 5′GGCTCCTGGCAAAAGGTCA, MYC.R 5′CTGCGTAGTTGTGCTGATGT, GAPDH.F 5′CTGGGCTACACTGAGCACC, and GAPDH.R 5′AAGTGGTCGTTGAGGGCAATG. Reactions were carried out in triplicate, and expression levels were normalized to GAPDH.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Cell Viability</h3><div class="NLM_p last">Cells were seeded in 384-well plates (ThermoFisher Scientific) at 500 cells/well, and transfer of serial prediluted compounds (100 nL) was performed using a Janus workstation (PerkinElmer). The CellTiter-Glo Luminescent cell viability assay was used to determine the cell viability. Luminescent signals were read on 2104 EnVision Multilabel Plate Readers (PerkinElmer) after a 5 day incubation. Data were normalized to the DMSO control, and the IC50 values were determined via the nonlinear regression curve fit using GraphPad Prism 9.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Statistical Analysis</h3><div class="NLM_p last">Prism 9 software was used to process the data. Student’s <i>t</i>-test was applied to compare compound-induced changes to respective controls. A <i>P</i> value of less than 0.05 was chosen as a threshold for statistical significance.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i53"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02232" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02232?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02232</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">BromoSCAN, differential scanning fluorometry (DSF), and cellular activity data of compounds <b>7</b>, <b>16</b>, and <b>23</b>; crystallographic data collection and refinement statistics of compound <b>23</b>; <sup>1</sup>H NMR and <sup>13</sup>C NMR of representative compounds; and HPLC traces of compounds <b>7</b>, <b>16</b>, and <b>23</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Homology (docking) models of BRD4_5BT4_Cpd7 (UMB276) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of BRD4_5BT4_Cpd-16 (UMB295) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of BRD4_5BT4_Cpd-23 (UMB298) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-3 (UMB274) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_006.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-4 (UMB278) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_007.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-7 (UMB276) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_008.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-16 (UMB295) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_009.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-23 (UMB298) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_010.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Cocrystallographic data of compound <b>23</b> (UMB298) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_011.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_001.csv">jm0c02232_si_001.csv (2.77 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf">jm0c02232_si_002.pdf (2.16 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_003.pdb">jm0c02232_si_003.pdb (181.1 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_004.pdb">jm0c02232_si_004.pdb (178.17 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_005.pdb">jm0c02232_si_005.pdb (178.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_006.pdb">jm0c02232_si_006.pdb (178.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_007.pdb">jm0c02232_si_007.pdb (194.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_008.pdb">jm0c02232_si_008.pdb (194.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_009.pdb">jm0c02232_si_009.pdb (194.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_010.pdb">jm0c02232_si_010.pdb (194.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_011.pdb">jm0c02232_si_011.pdb (194.68 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB accession code for the cocrystal structure of the CBP bromodomain liganded with compound <b>23</b> (UMB298) is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KPY">7KPY</a>. Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c02232" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ernst Schönbrunn</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, Moffit Cancer Center, Tampa, Florida 33612, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3589-3510" title="Orcid link">http://orcid.org/0000-0002-3589-3510</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c782b5a9b4b3e994a4afa8a9a5b5b2a9a987aaa8a1a1aeb3b3e9a8b5a0"><span class="__cf_email__" data-cfemail="b4f1c6dac7c09ae7d7dcdbdad6c6c1dadaf4d9dbd2d2ddc0c09adbc6d3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jun Qi</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medicine, Harvard Medical School, 220 Longwood Avenue, Boston, Massachusetts 02115, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#315b445f6e405871555752581f595043475043551f545544"><span class="__cf_email__" data-cfemail="28425d46775941684c4e4b410640495a5e495a4c064d4c5d">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Zhang</span> - <span class="hlFld-Affiliation affiliation">Center
for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6097-2763" title="Orcid link">http://orcid.org/0000-0002-6097-2763</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#81f6e4e8b3affbe9e0efe6c1f4ece3afe4e5f4"><span class="__cf_email__" data-cfemail="3f485a560d1145575e51587f4a525d115a5b4a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alex Muthengi</span> - <span class="hlFld-Affiliation affiliation">Center
for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Virangika K. Wimalasena</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hailemichael O. Yosief</span> - <span class="hlFld-Affiliation affiliation">Center
for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Melissa J. Bikowitz</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Department, Moffit Cancer Center, Tampa, Florida 33612, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Molecular Medicine, USF Morsani College of Medicine, University of South Florida, Tampa, Florida 33612, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4985-3461" title="Orcid link">http://orcid.org/0000-0003-4985-3461</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Logan H. Sigua</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tingjian Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Deyao Li</span> - <span class="hlFld-Affiliation affiliation">Department
of Cancer Biology, Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zied Gaieb</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry & Biochemistry, University
of California, San Diego, 9500 Gilman Dr, LA Jolla, California 92093, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gagan Dhawan</span> - <span class="hlFld-Affiliation affiliation">Department
of Biomedical Science, Acharya Narendra Dev College, University of Delhi, Delhi, New Delhi 110019, India</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7492-225X" title="Orcid link">http://orcid.org/0000-0001-7492-225X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuai Liu</span> - <span class="hlFld-Affiliation affiliation">Center
for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon Erickson</span> - <span class="hlFld-Affiliation affiliation">Center
for Green Chemistry and Department of Chemistry, University of Massachusetts Boston, 100 Morrissey Boulevard, Boston, Massachusetts 02125, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rommie E. Amaro</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry & Biochemistry, University
of California, San Diego, 9500 Gilman Dr, LA Jolla, California 92093, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9275-9553" title="Orcid link">http://orcid.org/0000-0002-9275-9553</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.M., V.K.W., H.O.Y., and M.J.B. contributed equally to this work.</p><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4301-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i57">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23261" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23261" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We acknowledge undergraduate student Francesca Corsini for her assistance with compound synthesis and Dr. Jason Evans for HRMS analysis of some samples. We also thank Eurofins for their expedite service on the <i>K</i><sub>d</sub> and selectivity evaluation. This work was supported by National Institutes of Health U54 grant CA156732 (W.Z.), National Cancer Institute (NCI) grants P01-CA066996-19, U54HD093540-03, P50-CA100707-15 (J.Q.), and P30-CA076292 (E.S.), and Alex’s Lemonade Stand Foundation Innovation Award (J.Q.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extraterminal domain</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain 4</p></td></tr><tr><td class="NLM_term">CREBBP or CBP</td><td class="NLM_def"><p class="first last">cyclic AMP response element binding protein</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorometry</p></td></tr><tr><td class="NLM_term">EP300 or P300</td><td class="NLM_def"><p class="first last">E1A binding protein P300</p></td></tr><tr><td class="NLM_term">MCR</td><td class="NLM_def"><p class="first last">multicomponent reaction</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SGC</td><td class="NLM_def"><p class="first last">structural genomics consortium</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15487" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15487" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0lgJTu059mBDaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem.%26date%3D2019%26volume%3D14%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.05.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.cbpa.2016.05.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27295577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=58-66&author=N.+H.+Theodoulouauthor=N.+C.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Clinical+progress+and+pharmacology+of+small+molecule+bromodomain+inhibitors&doi=10.1016%2Fj.cbpa.2016.05.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains have emerged as an exciting target class for drug discovery over the past decade.  Research has primarily focused on the bromodomain and extra terminal (BET) family of bromodomains, which has led to the development of multiple small mol. inhibitors and an increasing no. of clin. assets.  The excitement centered on the clin. potential of BET inhibition has stimulated intense interest in the broader family and the growing no. of non-BET bromodomain chem. probes has facilitated phenotypic investigations, implicating these targets in a variety of disease pathways including cancer, inflammation, embryonic development and neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEE0jCnUuyk7Vg90H21EOLACvtfcHk0liQAGLVEoPemg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D&md5=4bcd42755b2dd3cebd7fae417e69e83f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.05.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.05.028%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DClinical%2520progress%2520and%2520pharmacology%2520of%2520small%2520molecule%2520bromodomain%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.cbpa.2016.05.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Progress in the development and application of small molecule inhibitors of bromodomain-acetyl-lysine interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9393</span>– <span class="NLM_lpage">9413</span>, <span class="refDoi"> DOI: 10.1021/jm300915b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300915b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1OmtL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9393-9413&author=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=L.+E.+Jenningsauthor=D.+A.+Hayauthor=C.+J.+Schofieldauthor=P.+E.+Brennanauthor=S.+Knappauthor=S.+J.+Conway&title=Progress+in+the+development+and+application+of+small+molecule+inhibitors+of+bromodomain-acetyl-lysine+interactions&doi=10.1021%2Fjm300915b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development and Application of Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Hewings, David S.; Rooney, Timothy P. C.; Jennings, Laura E.; Hay, Duncan A.; Schofield, Christopher J.; Brennan, Paul E.; Knapp, Stefan; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9393-9413</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains, protein modules that recognize and bind to acetylated lysine, are emerging as important components of cellular machinery.  These acetyl-lysine (KAc) reader domains are part of the write-read-erase concept that has been linked with the transfer of epigenetic information.  By reading KAc marks on histones, bromodomains mediate protein-protein interactions between a diverse array of partners.  There has been intense activity in developing potent and selective small mol. probes that disrupt the interaction between a given bromodomain and KAc.  Rapid success has been achieved with the BET family of bromodomains, and a no. of potent and selective probes have been reported.  These compds. have enabled linking of the BET bromodomains with diseases, including cancer and inflammation, suggesting that bromodomains are druggable targets.  Herein, we review the biol. of the bromodomains and discuss the SAR for the existing small mol. probes.  The biol. that has been enabled by these compds. is summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8pl0pnLXruLVg90H21EOLACvtfcHk0liQAGLVEoPemg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1OmtL%252FJ&md5=486a87abb406c4c833404ccce61c1389</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm300915b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300915b%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DProgress%2520in%2520the%2520development%2520and%2520application%2520of%2520small%2520molecule%2520inhibitors%2520of%2520bromodomain-acetyl-lysine%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9393%26epage%3D9413%26doi%3D10.1021%2Fjm300915b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liQAGLVEoPemg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain: a new target in emerging epigenetic medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&author=S.+G.+Smithauthor=M.+M.+Zhou&title=The+Bromodomain%3A+a+new+target+in+emerging+epigenetic+medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0lilAPp9cnaBNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520Bromodomain%253A%2520a%2520new%2520target%2520in%2520emerging%2520epigenetic%2520medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0lilAPp9cnaBNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karakikes, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moshkina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajjar, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">A small molecule binding to the coactivator CREB-binding protein blocks apoptosis in cardiomyocytes</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.chembiol.2010.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=21513889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=531-541&author=J.+C.+Borahauthor=S.+Mujtabaauthor=I.+Karakikesauthor=L.+Zengauthor=M.+Mullerauthor=J.+Patelauthor=N.+Moshkinaauthor=K.+Morohashiauthor=W.+Zhangauthor=G.+Gerona-Navarroauthor=R.+J.+Hajjarauthor=M.+M.+Zhou&title=A+small+molecule+binding+to+the+coactivator+CREB-binding+protein+blocks+apoptosis+in+cardiomyocytes&doi=10.1016%2Fj.chembiol.2010.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Binding to the Coactivator CREB-Binding Protein Blocks Apoptosis in Cardiomyocytes</span></div><div class="casAuthors">Borah, Jagat C.; Mujtaba, Shiraz; Karakikes, Ioannis; Zeng, Lei; Muller, Michaela; Patel, Jigneshkumar; Moshkina, Natasha; Morohashi, Keita; Zhang, Weijia; Gerona-Navarro, Guillermo; Hajjar, Roger J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-541</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: As a master transcription factor in cellular responses to external stress, tumor suppressor p53 is tightly regulated.  Excessive p53 activity during myocardial ischemia causes irreversible cellular injury and cardiomyocyte death.  P53 activation is dependent on lysine acetylation by the lysine acetyltransferase and transcriptional coactivator CREB-binding protein (CBP) and on acetylation-directed CBP recruitment for p53 target gene expression.  Here, we report a small mol. ischemin, developed with a structure-guided approach to inhibit the acetyl-lysine binding activity of the bromodomain of CBP.  We show that ischemin alters post-translational modifications on p53 and histones, inhibits p53 interaction with CBP and transcriptional activity in cells, and prevents apoptosis in ischemic cardiomyocytes.  Our study suggests small mol. modulation of acetylation-mediated interactions in gene transcription as a new approach to therapeutic interventions of human disorders such as myocardial ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHKOaQAGSYTbVg90H21EOLACvtfcHk0lilAPp9cnaBNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D&md5=0c1d7500e1fff40bc335d1b10ac3c27a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DBorah%26aufirst%3DJ.%2BC.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DKarakikes%26aufirst%3DI.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DHajjar%26aufirst%3DR.%2BJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DA%2520small%2520molecule%2520binding%2520to%2520the%2520coactivator%2520CREB-binding%2520protein%2520blocks%2520apoptosis%2520in%2520cardiomyocytes%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D531%26epage%3D541%26doi%3D10.1016%2Fj.chembiol.2010.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd.</span></span> <span> </span><span class="NLM_article-title">Bromodomains: a new target class for drug development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0030-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1038%2Fs41573-019-0030-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=31273347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=609-628&author=A.+G.+Cochranauthor=A.+R.+Coneryauthor=R.+J.+Sims&title=Bromodomains%3A+a+new+target+class+for+drug+development&doi=10.1038%2Fs41573-019-0030-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: a new target class for drug development</span></div><div class="casAuthors">Cochran, Andrea G.; Conery, Andrew R.; Sims, III, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">609-628</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Less than a decade ago, it was shown that bromodomains, acetyl lysine 'reader' modules found in proteins with varied functions, were highly tractable small-mol. targets.  This is an unusual property for protein-protein or protein-peptide interaction domains, and it prompted a wave of chem. probe discovery to understand the biol. potential of new agents that targeted bromodomains.  The original examples, inhibitors of the bromodomain and extra-terminal (BET) class of bromodomains, showed enticing anti-inflammatory and anticancer activities, and several compds. have since advanced to human clin. trials.  Here, we review the current state of BET inhibitor biol. in relation to clin. development, and we discuss the next wave of bromodomain inhibitors with clin. potential in oncol. and non-oncol. indications.  The lessons learned from BET inhibitor programs should affect efforts to develop drugs that target non-BET bromodomains and other epigenetic readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6QwxqowW9NbVg90H21EOLACvtfcHk0li3hKUeRcUu8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL&md5=88e56eec8edeb7c325d28156ec131ed8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomains%253A%2520a%2520new%2520target%2520class%2520for%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D609%26epage%3D628%26doi%3D10.1038%2Fs41573-019-0030-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of highly selective inhibitors of the CREB binding protein Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5349</span>– <span class="NLM_lpage">5363</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5349-5363&author=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+Stockauthor=P.+Sahasrabudheauthor=P.+Boninauthor=D.+A.+Hayauthor=P.+E.+Brennanauthor=M.+Pletcherauthor=L.+H.+Jonesauthor=E.+L.+P.+Chekler&title=Structure-based+design+of+highly+selective+inhibitors+of+the+CREB+binding+protein+Bromodomain&doi=10.1021%2Facs.jmedchem.6b01839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span></div><div class="casAuthors">Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid; Sahasrabudhe, Parag; Bonin, Paul; Hay, Duncan A.; Brennan, Paul E.; Pletcher, Mathew; Jones, Lyn H.; Chekler, Eugene L. Piatnitski</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5349-5363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes are required for preclin. target validation to interrogate novel biol. targets and pathways.  Selective inhibitors of the CREB binding protein (CREBBP)/EP300 bromodomains are required to facilitate the elucidation of biol. assocd. with these important epigenetic targets.  Medicinal chem. optimization that paid particular attention to physiochem. properties delivered chem. probes with desirable potency, selectivity, and permeability attributes.  An important feature of the optimization process was the successful application of rational structure-based drug design to address bromodomain selectivity issues (particularly against the structurally related BRD4 protein).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLWkqF-qAmdbVg90H21EOLACvtfcHk0li3hKUeRcUu8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D&md5=23676a973abba59088dedd5502877105</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01839%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DPletcher%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26atitle%3DStructure-based%2520design%2520of%2520highly%2520selective%2520inhibitors%2520of%2520the%2520CREB%2520binding%2520protein%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5349%26epage%3D5363%26doi%3D10.1021%2Facs.jmedchem.6b01839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of small-molecule ligands for the CBP/p300 bromodomains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">9308</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1021/ja412434f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=C.+M.+Rogersauthor=S.+J.+Conwayauthor=T.+P.+Rooneyauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+optimization+of+small-molecule+ligands+for+the+CBP%2Fp300+bromodomains&doi=10.1021%2Fja412434f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0li3hKUeRcUu8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520small-molecule%2520ligands%2520for%2520the%2520CBP%252Fp300%2520bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319%26doi%3D10.1021%2Fja412434f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dancy, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Protein lysine acetylation by p300/CBP</span>. <i>Chem Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">2419</span>– <span class="NLM_lpage">2452</span>, <span class="refDoi"> DOI: 10.1021/cr500452k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500452k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=2419-2452&author=B.+M.+Dancyauthor=P.+A.+Cole&title=Protein+lysine+acetylation+by+p300%2FCBP&doi=10.1021%2Fcr500452k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Acetylation by p300/CBP</span></div><div class="casAuthors">Dancy, Beverley M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2419-2452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human protein lysine acetyltransferase encoded by the paralogous p300 and CBP genes is involved in regulating the expression of genes.  In this capacity, p300/CBP controls several basic cellular processes such as proliferation and homeostasis, and plays a role in a variety of human diseases, particularly solid tumors.  This review summarizes information related to the current understanding of p300/CBP, including the novel technologies that have co-evolved with p300/CBP research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYgTImMiuo7rVg90H21EOLACvtfcHk0liR87Sp0rowvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D&md5=40a46cf68990f4a7d11a338c54081628</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcr500452k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500452k%26sid%3Dliteratum%253Aachs%26aulast%3DDancy%26aufirst%3DB.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DProtein%2520lysine%2520acetylation%2520by%2520p300%252FCBP%26jtitle%3DChem%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D2419%26epage%3D2452%26doi%3D10.1021%2Fcr500452k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teufel, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bycroft, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">Four domains of p300 each bind tightly to a sequence spanning both transactivation subdomains of p53</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">7009</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1073/pnas.0702010104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1073%2Fpnas.0702010104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=17438265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1Oktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=7009-7014&author=D.+P.+Teufelauthor=S.+M.+Freundauthor=M.+Bycroftauthor=A.+R.+Fersht&title=Four+domains+of+p300+each+bind+tightly+to+a+sequence+spanning+both+transactivation+subdomains+of+p53&doi=10.1073%2Fpnas.0702010104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Four domains of p300 each bind tightly to a sequence spanning both transactivation subdomains of p53</span></div><div class="casAuthors">Teufel, Daniel P.; Freund, Stefan M.; Bycroft, Mark; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7009-7014</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The transcriptional coactivator p300 binds to and mediates the transcriptional functions of the tetrameric tumor suppressor p53.  Both proteins consist of independently folded domains linked by intrinsically disordered sequences.  A well studied short sequence of the p53 transactivation domain, p53(15-29), binds weakly to four folded domains of p300 [Taz1/cysteine-histidine-rich region 1 (CH1), Kix, Taz2/CH3, IBiD], with dissocn. consts. (KD) in the 100 μM region.  However, we found that a longer N-terminal transactivation domain construct p53(1-57) bound tightly to each p300 domain.  Taz2/CH3 had the greatest affinity (KD = 27 nM) and competes with the N-terminal domain of Mdm2 for the p53 N terminus.  P300 thus can protect the N terminus of p53 against the binding of other proteins.  Mutations of p53 that abrogate transactivation (L22Q/W23S, W53Q/F54S) greatly weakened binding to each p300 domain, linking phenotypic defects to weakened coactivator binding.  We propose a complex between tetrameric p53 and p300 in which four domains of p300 wrap around the four transactivation domains of p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvdOsvMUQZu7Vg90H21EOLACvtfcHk0liR87Sp0rowvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXls1Oktb4%253D&md5=95a601ed428c7fc61b8a9b01b58d9778</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0702010104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0702010104%26sid%3Dliteratum%253Aachs%26aulast%3DTeufel%26aufirst%3DD.%2BP.%26aulast%3DFreund%26aufirst%3DS.%2BM.%26aulast%3DBycroft%26aufirst%3DM.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DFour%2520domains%2520of%2520p300%2520each%2520bind%2520tightly%2520to%2520a%2520sequence%2520spanning%2520both%2520transactivation%2520subdomains%2520of%2520p53%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D7009%26epage%3D7014%26doi%3D10.1073%2Fpnas.0702010104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structural insights into acetylated-histone H4 recognition by the bromodomain-PHD finger module of human transcriptional coactivator CBP</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.str.2013.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=24361270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=353-360&author=A.+N.+Plotnikovauthor=S.+Yangauthor=T.+J.+Zhouauthor=E.+Rusinovaauthor=A.+Frascaauthor=M.+M.+Zhou&title=Structural+insights+into+acetylated-histone+H4+recognition+by+the+bromodomain-PHD+finger+module+of+human+transcriptional+coactivator+CBP&doi=10.1016%2Fj.str.2013.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into Acetylated-Histone H4 Recognition by the Bromodomain-PHD Finger Module of Human Transcriptional Coactivator CBP</span></div><div class="casAuthors">Plotnikov, Alexander N.; Yang, Shuai; Zhou, Thomas Jiachi; Rusinova, Elena; Frasca, Antonio; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomain functions as the acetyl-lysine binding domains to regulate gene transcription in chromatin.  Bromodomains are rapidly emerging as new epigenetic drug targets for human diseases.  However, owing to their transient nature and modest affinity, histone-binding selectivity of bromodomains has remained mostly elusive.  Here, we report high-resoln. crystal structures of the bromodomain-PHD tandem module of human transcriptional coactivator CBP bound to lysine-acetylated histone H4 peptides.  The structures reveal that the PHD finger serves a structural role in the tandem module and that the bromodomain prefers lysine-acetylated motifs comprising a hydrophobic or arom. residue at -2 and a lysine or arginine at -3 or -4 position from the acetylated lysine.  Our study further provides structural insights into distinct modes of singly and diacetylated histone H4 recognition by the bromodomains of CBP and BRD4 that function differently as a transcriptional coactivator and chromatin organizer, resp., explaining their distinct roles in control of gene expression in chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy6eG4JtL6sbVg90H21EOLACvtfcHk0liR87Sp0rowvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhtrvN&md5=15389e25c153d9378cb9b9f81818dc25</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DT.%2BJ.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DFrasca%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructural%2520insights%2520into%2520acetylated-histone%2520H4%2520recognition%2520by%2520the%2520bromodomain-PHD%2520finger%2520module%2520of%2520human%2520transcriptional%2520coactivator%2520CBP%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D353%26epage%3D360%26doi%3D10.1016%2Fj.str.2013.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachchidanand</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structural mechanism of the bromodomain of the coactivator CBP in p53 transcriptional activation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(03)00528-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2FS1097-2765%2803%2900528-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=14759370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFOhsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2004&pages=251-263&author=S.+Mujtabaauthor=Y.+Heauthor=L.+Zengauthor=S.+Yanauthor=O.+Plotnikovaauthor=+Sachchidanandauthor=R.+Sanchezauthor=N.+J.+Zeleznik-Leauthor=Z.+Ronaiauthor=M.+M.+Zhou&title=Structural+mechanism+of+the+bromodomain+of+the+coactivator+CBP+in+p53+transcriptional+activation&doi=10.1016%2FS1097-2765%2803%2900528-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism of the bromodomain of the coactivator CBP in p53 transcriptional activation</span></div><div class="casAuthors">Mujtaba, Shiraz; He, Yan; Zeng, Lei; Yan, Sherry; Plotnikova, Olga; Sachchidanand; Sanchez, Roberto; Zeleznik-Le, Nancy J.; Ronai, Ze'ev; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">251-263</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine acetylation of the tumor suppressor protein p53 in response to a wide variety of cellular stress signals is required for its activation as a transcription factor that regulates cell cycle arrest, senescence, or apoptosis.  Here, the authors report that the conserved bromodomain of the transcriptional coactivator CBP (CREB binding protein) binds specifically to p53 at the C-terminal acetylated lysine 382.  This bromodomain/acetyllysine binding is responsible for p53 acetylation-dependent coactivator recruitment after DNA damage, a step essential for p53-induced transcriptional activation of the cyclin-dependent kinase inhibitor p21 in G1 cell cycle arrest.  The authors further present the three-dimensional NMR structure of the CBP bromodomain in complex with a lysine 382-acetylated p53 peptide.  Using structural and biochem. analyses, the authors define the mol. determinants for the specificity of this mol. recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3g6bC53YUKbVg90H21EOLACvtfcHk0lg6oCDMpFtv7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFOhsr0%253D&md5=10127e2d2ea5165fd8439bf26a6f9241</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900528-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900528-8%26sid%3Dliteratum%253Aachs%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DPlotnikova%26aufirst%3DO.%26aulast%3DSachchidanand%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26aulast%3DRonai%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructural%2520mechanism%2520of%2520the%2520bromodomain%2520of%2520the%2520coactivator%2520CBP%2520in%2520p53%2520transcriptional%2520activation%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D13%26spage%3D251%26epage%3D263%26doi%3D10.1016%2FS1097-2765%2803%2900528-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wimalasena, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigua, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span> <span> </span><span class="NLM_article-title">Using chemical epigenetics to target cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1086</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.molcel.2020.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=32407673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptlersLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=1086-1095&author=V.+K.+Wimalasenaauthor=T.+Wangauthor=L.+H.+Siguaauthor=A.+D.+Durbinauthor=J.+Qi&title=Using+chemical+epigenetics+to+target+cancer&doi=10.1016%2Fj.molcel.2020.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Using Chemical Epigenetics to Target Cancer</span></div><div class="casAuthors">Wimalasena, Virangika K.; Wang, Tingjian; Sigua, Logan H.; Durbin, Adam D.; Qi, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1086-1095</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Transcription is epigenetically regulated by the orchestrated function of chromatin-binding proteins that tightly control the expression of master transcription factors, effectors, and supportive housekeeping genes required for establishing and propagating the normal and malignant cell state.  Rapid advances in chem. biol. and functional genomics have facilitated exploration of targeting epigenetic proteins, yielding effective strategies to target transcription while reducing toxicities to untransformed cells.  Here, we review recent developments in conventional active site and allosteric inhibitors, peptidomimetics, and novel proteolysis-targeted chimera (PROTAC) technol. that have deepened our understanding of transcriptional processes and led to promising preclin. compds. for therapeutic translation, particularly in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2E4fU5dl1obVg90H21EOLACvtfcHk0lg6oCDMpFtv7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptlersLg%253D&md5=573beb6b682321f14bbd7dc8a535ad1e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DWimalasena%26aufirst%3DV.%2BK.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DSigua%26aufirst%3DL.%2BH.%26aulast%3DDurbin%26aufirst%3DA.%2BD.%26aulast%3DQi%26aufirst%3DJ.%26atitle%3DUsing%2520chemical%2520epigenetics%2520to%2520target%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D1086%26epage%3D1095%26doi%3D10.1016%2Fj.molcel.2020.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakle, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setser, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAlmeida-Nagata, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">Regulatory T cell modulation by CBP/EP300 bromodomain inhibition</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">13014</span>– <span class="NLM_lpage">13027</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.708560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1074%2Fjbc.M115.708560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27056325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=13014-13027&author=S.+Ghoshauthor=A.+Taylorauthor=M.+Chinauthor=H.+R.+Huangauthor=A.+R.+Coneryauthor=J.+A.+Mertzauthor=A.+Salmeronauthor=P.+J.+Dakleauthor=D.+Meleauthor=A.+Coteauthor=H.+Jayaramauthor=J.+W.+Setserauthor=F.+Poyauthor=G.+Hatzivassiliouauthor=D.+DeAlmeida-Nagataauthor=P.+Sandyauthor=C.+Hattonauthor=F.+A.+Romeroauthor=E.+Chiangauthor=T.+Reimerauthor=T.+Crawfordauthor=E.+Pardoauthor=V.+G.+Watsonauthor=V.+Tsuiauthor=A.+G.+Cochranauthor=L.+Zawadzkeauthor=J.+C.+Harmangeauthor=J.+E.+Audiaauthor=B.+M.+Bryantauthor=R.+T.+Cummingsauthor=S.+R.+Magnusonauthor=J.+L.+Groganauthor=S.+F.+Bellonauthor=B.+K.+Albrechtauthor=R.+J.+Simsauthor=J.+M.+Lora&title=Regulatory+T+cell+modulation+by+CBP%2FEP300+bromodomain+inhibition&doi=10.1074%2Fjbc.M115.708560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory T Cell Modulation by CBP/EP300 Bromodomain Inhibition</span></div><div class="casAuthors">Ghosh, Srimoyee; Taylor, Alexander; Chin, Melissa; Huang, Hon-Ren; Conery, Andrew R.; Mertz, Jennifer A.; Salmeron, Andres; Dakle, Pranal J.; Mele, Deanna; Cote, Alexandre; Jayaram, Hari; Setser, Jeremy W.; Poy, Florence; Hatzivassiliou, Georgia; DeAlmeida-Nagata, Denise; Sandy, Peter; Hatton, Charlie; Romero, F. Anthony; Chiang, Eugene; Reimer, Thornik; Crawford, Terry; Pardo, Eneida; Watson, Venita G.; Tsui, Vickie; Cochran, Andrea G.; Zawadzke, Laura; Harmange, Jean-Christophe; Audia, James E.; Bryant, Barbara M.; Cummings, Richard T.; Magnuson, Steven R.; Grogan, Jane L.; Bellon, Steve F.; Albrecht, Brian K.; Sims, Robert J. III; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">13014-13027</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Covalent modification of histones is a fundamental mechanism of regulated gene expression in eukaryotes, and interpretation of histone modifications is an essential feature of epigenetic control.  Bromodomains are specialized binding modules that interact with acetylated histones, linking chromatin recognition to gene transcription.  Because of their ability to function in a domain-specific fashion, selective disruption of bromodomain:acetylated histone interactions with chem. probes serves as a powerful means for understanding biol. processes regulated by these chromatin adaptors.  Here the authors describe the discovery and characterization of potent and selective small mol. inhibitors for the bromodomains of CREBBP/EP300 that engage their target in cellular assays.  The authors use these tools to demonstrate a crit. role for CREBBP/EP300 bromodomains in regulatory T cell biol.  Because regulatory T cell recruitment to tumors is a major mechanism of immune evasion by cancer cells, the authors' data highlight the importance of CREBBP/EP300 bromodomain inhibition as a novel, small mol.-based approach for cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9maWuQVImyLVg90H21EOLACvtfcHk0lg6oCDMpFtv7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F&md5=bc8b4edba81bdbef30b1d8fcfaa19f1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.708560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.708560%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.%2BR.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DDakle%26aufirst%3DP.%2BJ.%26aulast%3DMele%26aufirst%3DD.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%2BW.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DDeAlmeida-Nagata%26aufirst%3DD.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DChiang%26aufirst%3DE.%26aulast%3DReimer%26aufirst%3DT.%26aulast%3DCrawford%26aufirst%3DT.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DWatson%26aufirst%3DV.%2BG.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DGrogan%26aufirst%3DJ.%2BL.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DRegulatory%2520T%2520cell%2520modulation%2520by%2520CBP%252FEP300%2520bromodomain%2520inhibition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D13014%26epage%3D13027%26doi%3D10.1074%2Fjbc.M115.708560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olzscha, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span> <span> </span><span class="NLM_article-title">CBP/p300 bromodomains regulate amyloid-like protein aggregation upon aberrant lysine acetylation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.chembiol.2016.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27989401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWnsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=9-23&author=H.+Olzschaauthor=O.+Fedorovauthor=B.+M.+Kesslerauthor=S.+Knappauthor=N.+B.+La+Thangue&title=CBP%2Fp300+bromodomains+regulate+amyloid-like+protein+aggregation+upon+aberrant+lysine+acetylation&doi=10.1016%2Fj.chembiol.2016.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">CBP/p300 Bromodomains Regulate Amyloid-like Protein Aggregation upon Aberrant Lysine Acetylation</span></div><div class="casAuthors">Olzscha, Heidi; Fedorov, Oleg; Kessler, Benedikt M.; Knapp, Stefan; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-23</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine acetylation is becoming increasingly recognized as a general biol. principle in cellular homeostasis, and is subject to abnormal control in different human pathologies.  Here, we describe a global effect on amyloid-like protein aggregation in human cells that results from aberrant lysine acetylation.  Bromodomain reader proteins are involved in the aggregation process and, using chem. biol. and gene silencing, we establish that p300/CBP bromodomains are necessary for aggregation to occur.  Moreover, protein aggregation disturbs proteostasis by impairing the ubiquitin proteasome system (UPS) and protein translation, resulting in decreased cell viability. p300/CBP bromodomain inhibitors impede aggregation, which coincides with enhanced UPS function and increased cell viability.  Aggregation of a pathol. relevant form of huntingtin protein is similarly affected by p300/CBP inhibition.  Our results have implications for understanding the mol. basis of protein aggregation, and highlight the possibility of treating amyloid-like pathologies and related protein folding diseases with bromodomain inhibitor-based strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqql6b79dRZDbVg90H21EOLACvtfcHk0lhKDZfixJompg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWnsb%252FO&md5=01aa97352d581eccff8c9745c6d545f5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DOlzscha%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DCBP%252Fp300%2520bromodomains%2520regulate%2520amyloid-like%2520protein%2520aggregation%2520upon%2520aberrant%2520lysine%2520acetylation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D9%26epage%3D23%26doi%3D10.1016%2Fj.chembiol.2016.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallipoli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papaemmanuil, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göttgens, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Deursen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J.</span></span> <span> </span><span class="NLM_article-title">The epigenetic regulators CBP and p300 facilitate leukemogenesis and represent therapeutic targets in acute myeloid leukemia</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.92</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1038%2Fonc.2015.92" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=25893291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFWhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=279-89&author=G.+Giotopoulosauthor=W.+I.+Chanauthor=S.+J.+Hortonauthor=D.+Ruauauthor=P.+Gallipoliauthor=A.+Fowlerauthor=C.+Crawleyauthor=E.+Papaemmanuilauthor=P.+J.+Campbellauthor=B.+G%C3%B6ttgensauthor=J.+M.+Van+Deursenauthor=P.+A.+Coleauthor=B.+J.+Huntly&title=The+epigenetic+regulators+CBP+and+p300+facilitate+leukemogenesis+and+represent+therapeutic+targets+in+acute+myeloid+leukemia&doi=10.1038%2Fonc.2015.92"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenetic regulators CBP and p300 facilitate leukemogenesis and represent therapeutic targets in acute myeloid leukemia</span></div><div class="casAuthors">Giotopoulos, G.; Chan, W.-I.; Horton, S. J.; Ruau, D.; Gallipoli, P.; Fowler, A.; Crawley, C.; Papaemmanuil, E.; Campbell, P. J.; Gottgens, B.; Van Deursen, J. M.; Cole, P. A.; Huntly, B. J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">279-289</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Growing evidence links abnormal epigenetic control to the development of hematol. malignancies.  Accordingly, inhibition of epigenetic regulators is emerging as a promising therapeutic strategy.  The acetylation status of lysine residues in histone tails is one of a no. of epigenetic post-translational modifications that alter DNA-templated processes, such as transcription, to facilitate malignant transformation.  Although histone deacetylases are already being clin. targeted, the role of histone lysine acetyltransferases (KAT) in malignancy is less well characterized.  We chose to study this question in the context of acute myeloid leukemia (AML), where, using in vitro and in vivo genetic ablation and knockdown expts. in murine models, we demonstrate a role for the epigenetic regulators CBP and p300 in the induction and maintenance of AML.  Furthermore, using selective small mol. inhibitors of their lysine acetyltransferase activity, we validate CBP/p300 as therapeutic targets in vitro across a wide range of human AML subtypes.  We proceed to show that growth retardation occurs through the induction of transcriptional changes that induce apoptosis and cell-cycle arrest in leukemia cells and finally demonstrate the efficacy of the KAT inhibitors in decreasing clonogenic growth of primary AML patient samples.  Taken together, these data suggest that CBP/p300 are promising therapeutic targets across multiple subtypes in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb1PaE0j_PhLVg90H21EOLACvtfcHk0lhKDZfixJompg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFWhsLc%253D&md5=3d91ea01fb6110f1c56aac137b068c85</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.92&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.92%26sid%3Dliteratum%253Aachs%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DHorton%26aufirst%3DS.%2BJ.%26aulast%3DRuau%26aufirst%3DD.%26aulast%3DGallipoli%26aufirst%3DP.%26aulast%3DFowler%26aufirst%3DA.%26aulast%3DCrawley%26aufirst%3DC.%26aulast%3DPapaemmanuil%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DG%25C3%25B6ttgens%26aufirst%3DB.%26aulast%3DVan%2BDeursen%26aufirst%3DJ.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26atitle%3DThe%2520epigenetic%2520regulators%2520CBP%2520and%2520p300%2520facilitate%2520leukemogenesis%2520and%2520represent%2520therapeutic%2520targets%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D279%26epage%3D89%26doi%3D10.1038%2Fonc.2015.92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Open access chemical probes for epigenetic targets</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1901</span>– <span class="NLM_lpage">1917</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.4155%2Ffmc.15.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26397018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1901-1917&author=P.+J.+Brownauthor=S.+M%C3%BCller&title=Open+access+chemical+probes+for+epigenetic+targets&doi=10.4155%2Ffmc.15.127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Open access chemical probes for epigenetic targets</span></div><div class="casAuthors">Brown, Peter J.; Mueller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1901-1917</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: High attrition rates in drug discovery call for new approaches to improve target validation.  Academia is filling gaps, but often lacks the experience and resources of the pharmaceutical industry resulting in poorly characterized tool compds.  Discussion: The SGC has established an open access chem. probe consortium, currently encompassing ten pharmaceutical companies.  One of its mandates is to create well-characterized inhibitors (chem. probes) for epigenetic targets to enable new biol. and target validation for drug development.  Conclusion: Epigenetic probe compds. have proven to be very valuable and have not only spurred a plethora of novel biol. findings, but also provided starting points for clin. trials.  These probes have proven to be crit. complementation to traditional genetic targeting strategies and provided sometimes surprising results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX4t2GBZPCzLVg90H21EOLACvtfcHk0lj_Pjms9EaOlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK&md5=09973fb3a77bee2a6a65e7fca190f2aa</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.127%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DOpen%2520access%2520chemical%2520probes%2520for%2520epigenetic%2520targets%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D1901%26epage%3D1917%26doi%3D10.4155%2Ffmc.15.127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">A series of potent CREBBP bromodomain ligands reveals an induced-fit pocket stabilized by a cation-π interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6126</span>– <span class="NLM_lpage">6130</span>, <span class="refDoi"> DOI: 10.1002/anie.201402750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fanie.201402750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+series+of+potent+CREBBP+bromodomain+ligands+reveals+an+induced-fit+pocket+stabilized+by+a+cation-%CF%80+interaction&doi=10.1002%2Fanie.201402750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-π Interaction</span></div><div class="casAuthors">Rooney, Timothy P. C.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Cortopassi, Wilian A.; Hay, Duncan A.; Martin, Sarah; Tumber, Anthony; Rogers, Catherine M.; Philpott, Martin; Wang, Minghua; Thompson, Amber L.; Heightman, Tom D.; Pryde, David C.; Cook, Andrew; Paton, Robert S.; Mueller, Susanne; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6126-6130</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Benzoxazinone and dihydroquinoxalinone fragments were employed as novel acetyl lysine mimics in the development of CREBBP bromodomain ligands.  While the benzoxazinone series showed low affinity for the CREBBP bromodomain, expansion of the dihydroquinoxalinone series resulted in the first potent inhibitors, e.g., I, of a bromodomain outside the BET family.  Structural and computational studies revealed that an internal hydrogen bond stabilized the protein-bound conformation of the dihydroquinoxalinone series.  The side chain of this series binds in an induced-fit pocket forming a cation-π interaction with R1173 of CREBBP.  Compd. I, the most potent compd., inhibited binding of CREBBP to chromatin in U2OS cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHFgTVUKe7lrVg90H21EOLACvtfcHk0lj_Pjms9EaOlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D&md5=42b12e1fe855d2af02c49f8e1031d39e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.201402750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201402750%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520series%2520of%2520potent%2520CREBBP%2520bromodomain%2520ligands%2520reveals%2520an%2520induced-fit%2520pocket%2520stabilized%2520by%2520a%2520cation-%25CF%2580%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130%26doi%3D10.1002%2Fanie.201402750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanz, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletcher, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Transcriptional profiling of a selective cREB Binding protein bromodomain inhibitor highlights therapeutic opportunities</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1588</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.chembiol.2015.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26670081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKis7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1588-1596&author=E.+L.+Cheklerauthor=J.+A.+Pellegrinoauthor=T.+A.+Lanzauthor=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+A.+Stockauthor=P.+Sahasrabudheauthor=P.+D.+Boninauthor=K.+Leeauthor=M.+T.+Pletcherauthor=L.+H.+Jones&title=Transcriptional+profiling+of+a+selective+cREB+Binding+protein+bromodomain+inhibitor+highlights+therapeutic+opportunities&doi=10.1016%2Fj.chembiol.2015.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Profiling of a Selective CREB Binding Protein Bromodomain Inhibitor Highlights Therapeutic Opportunities</span></div><div class="casAuthors">Chekler, Eugene L. Piatnitski; Pellegrino, Jessica A.; Lanz, Thomas A.; Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid A.; Sahasrabudhe, Parag; Bonin, Paul D.; Lee, Kevin; Pletcher, Mathew T.; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1588-1596</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomains are involved in transcriptional regulation through the recognition of acetyl lysine modifications on diverse proteins.  Selective pharmacol. modulators of bromodomains are lacking, although the largely hydrophobic nature of the pocket makes these modules attractive targets for small-mol. inhibitors.  This work describes the structure-based design of a highly selective inhibitor of the CREB binding protein (CBP) bromodomain and its use in cell-based transcriptional profiling expts.  The inhibitor downregulated a no. of inflammatory genes in macrophages that were not affected by a selective BET bromodomain inhibitor.  In addn., the CBP bromodomain inhibitor modulated the mRNA level of the regulator of G-protein signaling 4 (RGS4) gene in neurons, suggesting a potential therapeutic opportunity for CBP inhibitors in the treatment of neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLDeCJH-gX5LVg90H21EOLACvtfcHk0lj_Pjms9EaOlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKis7rE&md5=dc223d58c99b1994f0cdb15b734a5b21</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DChekler%26aufirst%3DE.%2BL.%26aulast%3DPellegrino%26aufirst%3DJ.%2BA.%26aulast%3DLanz%26aufirst%3DT.%2BA.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%2BA.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%2BD.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPletcher%26aufirst%3DM.%2BT.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTranscriptional%2520profiling%2520of%2520a%2520selective%2520cREB%2520Binding%2520protein%2520bromodomain%2520inhibitor%2520highlights%2520therapeutic%2520opportunities%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D1588%26epage%3D1596%26doi%3D10.1016%2Fj.chembiol.2015.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popp, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, F.</span></span> <span> </span><span class="NLM_article-title">Development of selective CBP/P300 benzoxazepine bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8889</span>– <span class="NLM_lpage">8912</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00774</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00774" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8889-8912&author=T.+A.+Poppauthor=C.+Tallantauthor=C.+Rogersauthor=O.+Fedorovauthor=P.+E.+Brennanauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=F.+Bracher&title=Development+of+selective+CBP%2FP300+benzoxazepine+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.6b00774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Selective CBP/P300 Benzoxazepine Bromodomain Inhibitors</span></div><div class="casAuthors">Popp, Tobias A.; Tallant, Cynthia; Rogers, Catherine; Fedorov, Oleg; Brennan, Paul E.; Mueller, Susanne; Knapp, Stefan; Bracher, Franz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8889-8912</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CBP (CREB (cAMP responsive element binding protein) binding protein (CREBBP)) and P300 (adenovirus E1A-assocd. 300 kDa protein) are two closely related histone acetyltransferases (HATs) that play a key role in the regulation of gene transcription.  Both proteins contain a bromodomain flanking the HAT catalytic domain that is important for the targeting of CBP/P300 to chromatin and which offers an opportunity for the development of protein-protein interaction inhibitors.  Here the authors present the development of CBP/P300 bromodomain inhibitors with 2,3,4,5-tetrahydro-1,4-benzoxazepine backbone, an N-acetyl-lysine mimetic scaffold that led to the recent development of the chem. probe I-CBP112.  The authors present comprehensive SAR of this inhibitor class as well as demonstration of cellular on target activity of the most potent and selective inhibitor I, which showed 134 nM affinity for CBP with excellent selectivity over other bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwAZFuNP0NkLVg90H21EOLACvtfcHk0lh4Dpy7TXV6Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E&md5=6bd914d24abf00a8f074bf1918fb4ee1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00774%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DT.%2BA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBracher%26aufirst%3DF.%26atitle%3DDevelopment%2520of%2520selective%2520CBP%252FP300%2520benzoxazepine%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8889%26epage%3D8912%26doi%3D10.1021%2Facs.jmedchem.6b00774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon
Boenig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of a selective and cell-active benzodiazepinone CBP/EP300 bromodomain Inhibitor (CPI-637)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">536</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00075</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00075" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=531-536&author=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=M.+C.+Hewittauthor=R.+Pastorauthor=Y.+Leblancauthor=C.+G.+Nasveschukauthor=F.+A.+Romeroauthor=T.+D.+Crawfordauthor=N.+Cantoneauthor=H.+Jayaramauthor=J.+Setserauthor=J.+Murrayauthor=M.+H.+Beresiniauthor=G.+de+Leon%0ABoenigauthor=Z.+Chenauthor=A.+R.+Coneryauthor=R.+T.+Cummingsauthor=L.+A.+Dakinauthor=E.+M.+Flynnauthor=O.+W.+Huangauthor=S.+Kaufmanauthor=P.+J.+Kellerauthor=J.+R.+Kieferauthor=T.+Laiauthor=Y.+Liauthor=J.+Liaoauthor=W.+Liuauthor=H.+Luauthor=E.+Pardoauthor=V.+Tsuiauthor=J.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=F.+Yanauthor=D.+Yuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=X.+Zhuauthor=R.+J.+Simsauthor=A.+G.+Cochranauthor=S.+Bellonauthor=J.+E.+Audiaauthor=S.+Magnusonauthor=B.+K.+Albrecht&title=Fragment-based+discovery+of+a+selective+and+cell-active+benzodiazepinone+CBP%2FEP300+bromodomain+Inhibitor+%28CPI-637%29&doi=10.1021%2Facsmedchemlett.6b00075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of a Selective and Cell-Active Benzodiazepinone CBP/EP300 Bromodomain Inhibitor (CPI-637)</span></div><div class="casAuthors">Taylor, Alexander M.; Cote, Alexandre; Hewitt, Michael C.; Pastor, Richard; Leblanc, Yves; Nasveschuk, Christopher G.; Romero, F. Anthony; Crawford, Terry D.; Cantone, Nico; Jayaram, Hariharan; Setser, Jeremy; Murray, Jeremy; Beresini, Maureen H.; de Leon Boenig, Gladys; Chen, Zhongguo; Conery, Andrew R.; Cummings, Richard T.; Dakin, Leslie A.; Flynn, E. Megan; Huang, Oscar W.; Kaufman, Susan; Keller, Patricia J.; Kiefer, James R.; Lai, Tommy; Li, Yingjie; Liao, Jiangpeng; Liu, Wenfeng; Lu, Henry; Pardo, Eneida; Tsui, Vickie; Wang, Jian; Wang, Yongyun; Xu, Zhaowu; Yan, Fen; Yu, Dong; Zawadzke, Laura; Zhu, Xiaoqin; Zhu, Xiaoyu; Sims, Robert J.; Cochran, Andrea G.; Bellon, Steve; Audia, James E.; Magnuson, Steven; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">531-536</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Benzo-fused heterocycles, particularly benzodiazepinones such as I, were prepd. as inhibitors of the bromodomain-contg. proteins CBP (CREB, cAMP response element binding protein) and EP300 (adenoviral E1A-binding protein of 300 kDa) for potential use as antitumor agents; the core benzodiazepinone moiety was identified using fragment-based screening.  The structures of four benzodiazepinones, including I, bound to the bromodomain fragment of human CREB were detd. by X-ray crystallog.  The free energy, enthalpy, and entropy of the binding of I to CREB and its selectivity for CREB and EP300 over other bromodomain-contg. proteins was detd.; I inhibited expression of the transcription factor MYC in human plasmacytoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNm7Blycl0-7Vg90H21EOLACvtfcHk0lh4Dpy7TXV6Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D&md5=8440a9bb3460ff66d43f15041dbbc19d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00075%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DFragment-based%2520discovery%2520of%2520a%2520selective%2520and%2520cell-active%2520benzodiazepinone%2520CBP%252FEP300%2520bromodomain%2520Inhibitor%2520%2528CPI-637%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D531%26epage%3D536%26doi%3D10.1021%2Facsmedchemlett.6b00075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon Boenig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharbanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillane, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and selective in vivo probe (GNE-272) for the bromodomains of CBP/EP300</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10549</span>– <span class="NLM_lpage">10563</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10549-10563&author=T.+D.+Crawfordauthor=F.+A.+Romeroauthor=K.+W.+Laiauthor=V.+Tsuiauthor=A.+M.+Taylorauthor=G.+de+Leon+Boenigauthor=C.+L.+Nolandauthor=J.+Murrayauthor=J.+Lyauthor=E.+F.+Chooauthor=T.+L.+Hunsakerauthor=E.+W.+Chanauthor=M.+Merchantauthor=S.+Kharbandaauthor=K.+E.+Gascoigneauthor=S.+Kaufmanauthor=M.+H.+Beresiniauthor=J.+Liaoauthor=W.+Liuauthor=K.+X.+Chenauthor=Z.+Chenauthor=A.+R.+Coneryauthor=A.+C%C3%B4t%C3%A9author=H.+Jayaramauthor=Y.+Jiangauthor=J.+R.+Kieferauthor=T.+Kleinheinzauthor=Y.+Liauthor=J.+Maherauthor=E.+Pardoauthor=F.+Poyauthor=K.+L.+Spillaneauthor=F.+Wangauthor=J.+Wangauthor=X.+Weiauthor=Z.+Xuauthor=Z.+Xuauthor=I.+Yenauthor=L.+Zawadzkeauthor=X.+Zhuauthor=S.+Bellonauthor=R.+Cummingsauthor=A.+G.+Cochranauthor=B.+K.+Albrechtauthor=S.+Magnuson&title=Discovery+of+a+potent+and+selective+in+vivo+probe+%28GNE-272%29+for+the+bromodomains+of+CBP%2FEP300&doi=10.1021%2Facs.jmedchem.6b01022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective in Vivo Probe (GNE-272) for the Bromodomains of CBP/EP300</span></div><div class="casAuthors">Crawford, Terry D.; Romero, F. Anthony; Lai, Kwong Wah; Tsui, Vickie; Taylor, Alexander M.; de Leon Boenig, Gladys; Noland, Cameron L.; Murray, Jeremy; Ly, Justin; Choo, Edna F.; Hunsaker, Thomas L.; Chan, Emily W.; Merchant, Mark; Kharbanda, Samir; Gascoigne, Karen E.; Kaufman, Susan; Beresini, Maureen H.; Liao, Jiangpeng; Liu, Wenfeng; Chen, Kevin X.; Chen, Zhongguo; Conery, Andrew R.; Cote, Alexandre; Jayaram, Hariharan; Jiang, Ying; Kiefer, James R.; Kleinheinz, Tracy; Li, Yingjie; Maher, Jonathan; Pardo, Eneida; Poy, Florence; Spillane, Kerry L.; Wang, Fei; Wang, Jian; Wei, Xiaocang; Xu, Zhaowu; Xu, Zhongya; Yen, Ivana; Zawadzke, Laura; Zhu, Xiaoyu; Bellon, Steven; Cummings, Richard; Cochran, Andrea G.; Albrecht, Brian K.; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10549-10563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The single bromodomain of the closely related transcriptional regulators CBP/EP300 is a target of much recent interest in cancer and immune system regulation.  A co-crystal structure of a ligand-efficient screening hit and the CBP bromodomain guided initial design targeting the LPF shelf, ZA loop, and acetylated lysine binding regions.  Structure-activity relationship studies allowed the authors to identify a more potent analog.  Optimization of permeability and microsomal stability and subsequent improvement of mouse hepatocyte stability afforded 59 (GNE-272, TR-FRET IC50 = 0.02 μM, BRET IC50 = 0.41 μM, BRD4(1) IC50 = 13 μM) that retained the best balance of cell potency, selectivity, and in vivo PK.  GNE-272 showed a marked antiproliferative effect in hematol. cancer cell lines and modulates MYC expression in vivo that corresponds with antitumor activity in an AML tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6T5ODn2Q-2LVg90H21EOLACvtfcHk0lg9acpJdwHyow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO&md5=22ed46cb031022aa6aff23c20d07e7dd</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01022%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520in%2520vivo%2520probe%2520%2528GNE-272%2529%2520for%2520the%2520bromodomains%2520of%2520CBP%252FEP300%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10549%26epage%3D10563%26doi%3D10.1021%2Facs.jmedchem.6b01022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon Boenig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronner, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clagg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida
Nagata, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">GNE-781, a highly advanced potent and selective bromodomain inhibitor of cyclic adenosine monophosphate response element binding protein, binding protein (CBP)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9162</span>– <span class="NLM_lpage">9183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgs7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9162-9183&author=F.+A.+Romeroauthor=J.+Murrayauthor=K.+W.+Laiauthor=V.+Tsuiauthor=B.+K.+Albrechtauthor=L.+Anauthor=M.+H.+Beresiniauthor=G.+de+Leon+Boenigauthor=S.+M.+Bronnerauthor=E.+W.+Chanauthor=K.+X.+Chenauthor=Z.+Chenauthor=E.+F.+Chooauthor=K.+Claggauthor=K.+Clarkauthor=T.+D.+Crawfordauthor=P.+Cyrauthor=D.+de+Almeida%0ANagataauthor=K.+E.+Gascoigneauthor=J.+L.+Groganauthor=G.+Hatzivassiliouauthor=W.+Huangauthor=T.+L.+Hunsakerauthor=S.+Kaufmanauthor=S.+G.+Koenigauthor=R.+Liauthor=Y.+Liauthor=X.+Liangauthor=J.+Liaoauthor=W.+Liuauthor=J.+Lyauthor=J.+Maherauthor=C.+Masuiauthor=M.+Merchantauthor=Y.+Ranauthor=A.+M.+Taylorauthor=J.+Waiauthor=F.+Wangauthor=X.+Weiauthor=D.+Yuauthor=B.+Y.+Zhuauthor=X.+Zhuauthor=S.+Magnuson&title=GNE-781%2C+a+highly+advanced+potent+and+selective+bromodomain+inhibitor+of+cyclic+adenosine+monophosphate+response+element+binding+protein%2C+binding+protein+%28CBP%29&doi=10.1021%2Facs.jmedchem.7b00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-781, A Highly Advanced Potent and Selective Bromodomain Inhibitor of Cyclic Adenosine Monophosphate Response Element Binding Protein, Binding Protein (CBP)</span></div><div class="casAuthors">Romero, F. Anthony; Murray, Jeremy; Lai, Kwong Wah; Tsui, Vickie; Albrecht, Brian K.; An, Le; Beresini, Maureen H.; de Leon Boenig, Gladys; Bronner, Sarah M.; Chan, Emily W.; Chen, Kevin X.; Chen, Zhongguo; Choo, Edna F.; Clagg, Kyle; Clark, Kevin; Crawford, Terry D.; Cyr, Patrick; de Almeida Nagata, Denise; Gascoigne, Karen E.; Grogan, Jane L.; Hatzivassiliou, Georgia; Huang, Wei; Hunsaker, Thomas L.; Kaufman, Susan; Koenig, Stefan G.; Li, Ruina; Li, Yingjie; Liang, Xiaorong; Liao, Jiangpeng; Liu, Wenfeng; Ly, Justin; Maher, Jonathan; Masui, Colin; Merchant, Mark; Ran, Yingqing; Taylor, Alexander M.; Wai, John; Wang, Fei; Wei, Xiaocang; Yu, Dong; Zhu, Bing-Yan; Zhu, Xiaoyu; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9162-9183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the bromodomain of the transcriptional regulator CBP/P300 is an esp. interesting new therapeutic approach in oncol.  We recently disclosed in vivo chem. tool 1 (GNE-272) for the bromodomain of CBP that was moderately potent and selective over BRD4(1).  In pursuit of a more potent and selective CBP inhibitor, we used structure-based design.  Constraining the aniline of 1 into a tetrahydroquinoline motif maintained potency and increased selectivity 2-fold.  Structure-activity relationship studies coupled with further structure-based design targeting the LPF shelf, BC loop, and KAc regions allowed us to significantly increase potency and selectivity, resulting in the identification of non-CNS penetrant 19 (GNE-781, TR-FRET IC50 = 0.94 nM, BRET IC50 = 6.2 nM; BRD4(1) IC50 = 5100 nΜ) that maintained good in vivo PK properties in multiple species.  Compd. 19 displays antitumor activity in an AML tumor model and was also shown to decrease Foxp3 transcript levels in a dose dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYowYgusjfzrVg90H21EOLACvtfcHk0lgteePL3UyqlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgs7zI&md5=16403939224df3949cd3e3ffd8cf1fd1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00796%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DBronner%26aufirst%3DS.%2BM.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DClagg%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DK.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCyr%26aufirst%3DP.%26aulast%3Dde%2BAlmeida%2BNagata%26aufirst%3DD.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DGrogan%26aufirst%3DJ.%2BL.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKoenig%26aufirst%3DS.%2BG.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DMasui%26aufirst%3DC.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DB.%2BY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DGNE-781%252C%2520a%2520highly%2520advanced%2520potent%2520and%2520selective%2520bromodomain%2520inhibitor%2520of%2520cyclic%2520adenosine%2520monophosphate%2520response%2520element%2520binding%2520protein%252C%2520binding%2520protein%2520%2528CBP%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9162%26epage%3D9183%26doi%3D10.1021%2Facs.jmedchem.7b00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marineau, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Biased multicomponent reactions to develop novel bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9019</span>– <span class="NLM_lpage">9127</span>, <span class="refDoi"> DOI: 10.1021/jm501120z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9127&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+multicomponent+reactions+to+develop+novel+bromodomain+inhibitors&doi=10.1021%2Fjm501120z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span></div><div class="casAuthors">McKeown, Michael R.; Shaw, Daniel L.; Fu, Harry; Liu, Shuai; Xu, Xiang; Marineau, Jason J.; Huang, Yibo; Zhang, Xiaofeng; Buckley, Dennis L.; Kadam, Asha; Zhang, Zijuan; Blacklow, Stephen C.; Qi, Jun; Zhang, Wei; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9019-9027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET bromodomain inhibition has contributed new insights into gene regulation and emerged as a promising therapeutic strategy in cancer.  Structural analogy of early methyl-triazolo BET inhibitors has prompted a need for structurally dissimilar ligands as probes of bromodomain function.  Using fluorous-tagged multicomponent reactions, we developed a focused chem. library of bromodomain inhibitors around a 3,5-dimethylisoxazole biasing element with micromolar biochem. IC50.  Iterative synthesis and biochem. assessment allowed optimization of novel BET bromodomain inhibitors based on an imidazo[1,2-a]pyrazine scaffold.  Lead compd. I (UMB-32) binds BRD4 with a Kd of 550 nM and 724 nM cellular potency in BRD4-dependent lines.  Addnl., compd. I shows potency against TAF1, a bromodomain-contg. transcription factor previously unapproached by discovery chem.  Compd. I was cocrystd. with BRD4, yielding a 1.56 Å resoln. crystal structure.  This research showcases new applications of fluorous and multicomponent chem. synthesis for the development of novel epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWEf4LoDJTrVg90H21EOLACvtfcHk0lg4LRxJe8h-LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN&md5=a4e20101f5a0b547451814af73c0418f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520multicomponent%2520reactions%2520to%2520develop%2520novel%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9127%26doi%3D10.1021%2Fjm501120z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groebke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehlin, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis of imidazo[1,2-a] annulated pyridines, pyrazines and pyrimidines by a novel three-component condensation</span>. <i>Synlett</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>1998</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1055/s-1998-1721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1055%2Fs-1998-1721" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1998&publication_year=1998&pages=661-663&author=K.+Groebkeauthor=L.+Weberauthor=F.+Mehlin&title=Synthesis+of+imidazo%5B1%2C2-a%5D+annulated+pyridines%2C+pyrazines+and+pyrimidines+by+a+novel+three-component+condensation&doi=10.1055%2Fs-1998-1721"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1055%2Fs-1998-1721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1998-1721%26sid%3Dliteratum%253Aachs%26aulast%3DGroebke%26aufirst%3DK.%26aulast%3DWeber%26aufirst%3DL.%26aulast%3DMehlin%26aufirst%3DF.%26atitle%3DSynthesis%2520of%2520imidazo%255B1%252C2-a%255D%2520annulated%2520pyridines%252C%2520pyrazines%2520and%2520pyrimidines%2520by%2520a%2520novel%2520three-component%2520condensation%26jtitle%3DSynlett%26date%3D1998%26volume%3D1998%26spage%3D661%26epage%3D663%26doi%3D10.1055%2Fs-1998-1721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiosaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S.</span></span> <span> </span><span class="NLM_article-title">Parallel synthesis of 3-aminoimidazo[1,2-a]pyridines and pyrazines by a new three-component condensation</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">3635</span>– <span class="NLM_lpage">3638</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(98)00653-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2FS0040-4039%2898%2900653-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADyaK1cXjtlWrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1998&pages=3635-3638&author=C.+Blackburnauthor=B.+Guanauthor=P.+Flemingauthor=K.+Shiosakiauthor=S.+Tsai&title=Parallel+synthesis+of+3-aminoimidazo%5B1%2C2-a%5Dpyridines+and+pyrazines+by+a+new+three-component+condensation&doi=10.1016%2FS0040-4039%2898%2900653-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel synthesis of 3-aminoimidazo[1,2-a]pyridines and pyrazines by a new three-component condensation</span></div><div class="casAuthors">Blackburn, Christopher; Guan, Bing; Fleming, Paul; Shiosaki, Kazumi; Tsai, Shirling</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3635-3638</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A three-component condensation reaction between 2-aminopyridine, an aldehyde and an isonitrile catalyzed by scandium triflate affords derivs. of 3-aminoimidazo[1,2-a]pyridine; aminopyrazine reacts similarly.  A library of heterocycles, prepd. in high yield by parallel synthesis and purifn. on an ion-exchange resin, was subjected to further reaction at the amino group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrndWK-azaOEbVg90H21EOLACvtfcHk0lhy-iErohDAcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtlWrtLs%253D&md5=dfdddf23bcb1051d2ff6375f9f2616f5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2898%2900653-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252898%252900653-4%26sid%3Dliteratum%253Aachs%26aulast%3DBlackburn%26aufirst%3DC.%26aulast%3DGuan%26aufirst%3DB.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DShiosaki%26aufirst%3DK.%26aulast%3DTsai%26aufirst%3DS.%26atitle%3DParallel%2520synthesis%2520of%25203-aminoimidazo%255B1%252C2-a%255Dpyridines%2520and%2520pyrazines%2520by%2520a%2520new%2520three-component%2520condensation%26jtitle%3DTetrahedron%2520Lett.%26date%3D1998%26volume%3D39%26spage%3D3635%26epage%3D3638%26doi%3D10.1016%2FS0040-4039%2898%2900653-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bienaymé, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouzid, K.</span></span> <span> </span><span class="NLM_article-title">A new heterocyclic multicomponent reaction for the combinatorial synthesis of fused 3-aminoimidazoles</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2234</span>– <span class="NLM_lpage">2237</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19980904)37:16<2234::AID-ANIE2234>3.0.CO;2-R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADyaK1cXmt1Crt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=2234-2237&author=H.+Bienaym%C3%A9author=K.+Bouzid&title=A+new+heterocyclic+multicomponent+reaction+for+the+combinatorial+synthesis+of+fused+3-aminoimidazoles&doi=10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A new heterocyclic multicomponent reaction for the combinatorial synthesis of fused 3-aminoimidazoles</span></div><div class="casAuthors">Bienayme, Hugues; Bouzid, Kamel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2234-2237</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Reaction of heteroarom. amidines, aldehydes, and isonitriles in the presence of a catalytic amt. of protic acids gave fused 3-aminoimidazoles.  E.g., HClO4-catalyzed reaction of 2-aminopyrimidine, PhCHO, and Me3CNC gave 82% imidazopyrimidine I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1LwDCSCFs_rVg90H21EOLACvtfcHk0lhy-iErohDAcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmt1Crt7Y%253D&md5=6c11f8f883d1699e9a3def969b5f3e04</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819980904%252937%253A16%253C2234%253A%253AAID-ANIE2234%253E3.0.CO%253B2-R%26sid%3Dliteratum%253Aachs%26aulast%3DBienaym%25C3%25A9%26aufirst%3DH.%26aulast%3DBouzid%26aufirst%3DK.%26atitle%3DA%2520new%2520heterocyclic%2520multicomponent%2520reaction%2520for%2520the%2520combinatorial%2520synthesis%2520of%2520fused%25203-aminoimidazoles%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1998%26volume%3D37%26spage%3D2234%26epage%3D2237%26doi%3D10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span> <span> </span><span class="NLM_article-title">Cross-coupling reactions of organoboranes: an easy way to construct C-C bonds (Nobel Lecture)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">6722</span>– <span class="NLM_lpage">6737</span>, <span class="refDoi"> DOI: 10.1002/anie.201101379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fanie.201101379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3MXms1Skurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6722-6737&author=A.+Suzuki&title=Cross-coupling+reactions+of+organoboranes%3A+an+easy+way+to+construct+C-C+bonds+%28Nobel+Lecture%29&doi=10.1002%2Fanie.201101379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-Coupling Reactions Of Organoboranes: An Easy Way To Construct C-C Bonds (Nobel Lecture)</span></div><div class="casAuthors">Suzuki, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6722-6737</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The Nobel Prize in Chemie 2010 was awarded for research on palladium-catalyzed cross-coupling in org. synthesis.  Reported is the historical development and current status of the Suzuki coupling reaction.  A variety of organoboron compds. have been shown to be suitable coupling partners, including boronic acids, boronic esters, and sterically hindered boranes, allowing carbon-carbon bond formation of aryl, vinyl, alkyl, and alkynyl moieties.  The products obtained with this methodol. range from simple dienes, biaryls, and alkanes to complex natural products and pharmaceuticals.  The optimization of catalytic conditions for sterically hindered and unactivated substrates is demonstrated, as well as application to the Suzuki polycondensation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsO3da5SzNe7Vg90H21EOLACvtfcHk0lhyf69LFQ63Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXms1Skurw%253D&md5=f5a1adae37cc486e1484fede5db572c3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201101379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201101379%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DCross-coupling%2520reactions%2520of%2520organoboranes%253A%2520an%2520easy%2520way%2520to%2520construct%2520C-C%2520bonds%2520%2528Nobel%2520Lecture%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26spage%3D6722%26epage%3D6737%26doi%3D10.1002%2Fanie.201101379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">A bead-based proximity assay for BRD4 ligand discovery</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch150024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2F9780470559277.ch150024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26629616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVOlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=263-278&author=J.+M.+Robertsauthor=J.+E.+Bradner&title=A+bead-based+proximity+assay+for+BRD4+ligand+discovery&doi=10.1002%2F9780470559277.ch150024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Bead-Based Proximity Assay for BRD4 Ligand Discovery</span></div><div class="casAuthors">Roberts, Justin M.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protocols in Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-278</span>CODEN:
                <span class="NLM_cas:coden">CPCBD9</span>;
        ISSN:<span class="NLM_cas:issn">2160-4762</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Bromodomain-contg. proteins have emerged as desirable targets for anti-neoplastic and anti-inflammatory drug discovery.  Toward the development of selective inhibitors of the BET family of bromodomains, we optimized bead-based assays to detect interactions between bromodomains and poly-acetylated histone peptides.  Donor and acceptor beads bound to target and ligand are brought into proximity by this protein-protein interaction.  After laser illumination, singlet oxygen evolved from donor beads travels to the spatially close acceptor beads, resulting in chemiluminesence.  This AlphaScreen assay has proven amendable to high-throughput screening, secondary validation, and specificity profiling during lead discovery and optimization.  Here we report our protocol for assay development to measure inhibition of ligand binding to bromodomain-contg. protein 4 (BRD4).  We discuss the discovery of an appropriate probe, optimization of bead, probe, and protein concns., and the derivation of protein-probe inhibition curves.  Finally, we explore the implementation of this technol. for high-throughput screening of potential BRD4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohs1bOu1z4tbVg90H21EOLACvtfcHk0lhyf69LFQ63Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVOlsbg%253D&md5=4d6961bcb98443349fcd54b7a23ac998</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch150024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch150024%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DA%2520bead-based%2520proximity%2520assay%2520for%2520BRD4%2520ligand%2520discovery%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D7%26spage%3D263%26epage%3D278%26doi%3D10.1002%2F9780470559277.ch150024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel series of BET family bromodomain inhibitors: binding mode and profile of I-BET151 (GSK1210151A)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2968</span>– <span class="NLM_lpage">2972</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.02.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.-W.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+novel+series+of+BET+family+bromodomain+inhibitors%3A+binding+mode+and+profile+of+I-BET151+%28GSK1210151A%29&doi=10.1016%2Fj.bmcl.2012.02.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0lhyf69LFQ63Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520series%2520of%2520BET%2520family%2520bromodomain%2520inhibitors%253A%2520binding%2520mode%2520and%2520profile%2520of%2520I-BET151%2520%2528GSK1210151A%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972%26doi%3D10.1016%2Fj.bmcl.2012.02.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">697</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697-705&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0lhIGtB2i2_Xaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D697%26epage%3D705%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheryl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=+Arrowsmithauthor=H.+Cherylauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhIGtB2i2_Xaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aulast%3DCheryl%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span> <span> </span><span class="NLM_article-title">Current development of CBP/p300 inhibitors in the last decade</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>209</i></span>, <span class="NLM_elocation-id">112861</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.ejmech.2020.112861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=33045661" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=209&publication_year=2020&author=Z.+X.+Heauthor=B.+F.+Weiauthor=X.+Zhangauthor=Y.+P.+Gongauthor=L.+Y.+Maauthor=W.+Zhao&title=Current+development+of+CBP%2Fp300+inhibitors+in+the+last+decade&doi=10.1016%2Fj.ejmech.2020.112861"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112861%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DZ.%2BX.%26aulast%3DWei%26aufirst%3DB.%2BF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGong%26aufirst%3DY.%2BP.%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DZhao%26aufirst%3DW.%26atitle%3DCurrent%2520development%2520of%2520CBP%252Fp300%2520inhibitors%2520in%2520the%2520last%2520decade%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D209%26doi%3D10.1016%2Fj.ejmech.2020.112861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centore, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillane, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommi-Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibition of the transcriptional coactivators CBP/EP300 as a therapeutic strategy to target the IRF4 network in multiple myeloma</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e10483</span> <span class="refDoi"> DOI: 10.7554/eLife.10483.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.7554%2FeLife.10483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26731516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslahtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&author=A.+R.+Coneryauthor=R.+C.+Centoreauthor=A.+Neissauthor=P.+J.+Kellerauthor=S.+Joshiauthor=K.+L.+Spillaneauthor=P.+Sandyauthor=C.+Hattonauthor=E.+Pardoauthor=L.+Zawadzkeauthor=A.+Bommi-Reddyauthor=K.+E.+Gascoigneauthor=B.+M.+Bryantauthor=J.+A.+Mertzauthor=R.+J.+Sims&title=Bromodomain+inhibition+of+the+transcriptional+coactivators+CBP%2FEP300+as+a+therapeutic+strategy+to+target+the+IRF4+network+in+multiple+myeloma&doi=10.7554%2FeLife.10483.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibition of the transcriptional coactivators CBP/EP300 as atherapeutic strategy to target the IRF4 network in multiple myeloma</span></div><div class="casAuthors">Conery, Andrew R.; Centore, Richard C.; Neiss, Adrianne; Keller, Patricia J.; Joshi, Shivangi; Spillane, Kerry L.; Sandy, Peter; Hatton, Charlie; Pardo, Eneida; Zawadzke, Laura; Bommi-Reddy, Archana; Gascoigne, Karen E.; Bryant, Barbara M.; Mertz, Jennifer A.; Sims, Robert J., III</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e10483/1-e10483/17</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Pharmacol. inhibition of chromatin co-regulatory factors represents a clin. validated strategy to modulate oncogenic signaling through selective attenuation of gene expression.  Here, we demonstrate that CBP/EP300 bromodomain inhibition preferentially abrogates the viability of multiple myeloma cell lines.  Selective targeting of multiple myeloma cell lines through CBP/EP300 bromodomain inhibition is the result of direct transcriptional suppression of the lymphocyte-specific transcription factor IRF4, which is essential for the viability of myeloma cells, and the concomitant repression of the IRF4 target gene c-MYC.  Ectopic expression of either IRF4 or MYC antagonizes the phenotypic and transcriptional effects of CBP/EP300 bromodomain inhibition, highlighting the IRF4/MYC axis as a key component of its mechanism of action.  These findings suggest that CBP/EP300 bromodomain inhibition represents a viable therapeutic strategy for targeting multiple myeloma and other lymphoid malignancies dependent on the IRF4 network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfAd8Yl2ENq7Vg90H21EOLACvtfcHk0lhIGtB2i2_Xaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslahtro%253D&md5=d0037d5abb6fbd85ce1ce2763c535370</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7554%2FeLife.10483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.10483%26sid%3Dliteratum%253Aachs%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCentore%26aufirst%3DR.%2BC.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomain%2520inhibition%2520of%2520the%2520transcriptional%2520coactivators%2520CBP%252FEP300%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520the%2520IRF4%2520network%2520in%2520multiple%2520myeloma%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26doi%3D10.7554%2FeLife.10483.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1655</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.4155/fmc-2016-0059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.4155%2Ffmc-2016-0059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27193077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1655-1680&author=C.+Galdeanoauthor=A.+Ciulli&title=Selectivity+on-target+of+bromodomain+chemical+probes+by+structure-guided+medicinal+chemistry+and+chemical+biology&doi=10.4155%2Ffmc-2016-0059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span></div><div class="casAuthors">Galdeano, Carles; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1655-1680</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Targeting epigenetic proteins is a rapidly growing area for medicinal chem. and drug discovery.  Recent years have seen an explosion of interest in developing small mols. binding to bromodomains, the readers of acetyl-lysine modifications.  A plethora of co-crystal structures has motivated focused fragment-based design and optimization programs within both industry and academia.  These efforts have yielded several compds. entering the clinic, and many more are increasingly being used as chem. probes to interrogate bromodomain biol.  High selectivity of chem. probes is necessary to ensure biol. activity is due to an on-target effect.  Here, we review the state-of-the-art of bromodomain-targeting compds., focusing on the structural basis for their on-target selectivity or lack thereof.  We also highlight chem. biol. approaches to enhance on-target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1bg8K3kh-RLVg90H21EOLACvtfcHk0lg5uFy7TW79Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K&md5=a4c02dd517e8582755a918eabd7bfeea</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0059%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelectivity%2520on-target%2520of%2520bromodomain%2520chemical%2520probes%2520by%2520structure-guided%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D1655%26epage%3D1680%26doi%3D10.4155%2Ffmc-2016-0059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Microwave-assisted synthesis of a 3-aminoimidazo[1,2-a]-pyridine/pyrazine library by fluorous multicomponent reactions and subsequent cross-coupling reactions</span>. <i>QSAR Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">835</span>, <span class="refDoi"> DOI: 10.1002/qsar.200420045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fqsar.200420045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=18542716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktVamsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=827-835&author=Y.+Luauthor=W.+Zhang&title=Microwave-assisted+synthesis+of+a+3-aminoimidazo%5B1%2C2-a%5D-pyridine%2Fpyrazine+library+by+fluorous+multicomponent+reactions+and+subsequent+cross-coupling+reactions&doi=10.1002%2Fqsar.200420045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Microwave-assisted synthesis of a 3-aminoimidazo[1,2-a]-pyridine/pyrazine library by fluorous multicomponent reactions and subsequent cross-coupling reactions</span></div><div class="casAuthors">Lu, Yimin; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">QSAR & Combinatorial Science</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">827-835</span>CODEN:
                <span class="NLM_cas:coden">QCSSAU</span>;
        ISSN:<span class="NLM_cas:issn">1611-020X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Microwave and fluorous technologies are employed to increase reaction rate and reduce sepn. time in soln.-phase parallel synthesis of a substituted 3-aminoimidazo[1,2-a]pyridine and 3-aminoimidazo[1,2-a]pyrazine library.  Multicomponent reactions of perfluorooctanesulfonyl-tagged benzaldehydes with 2-aminopyridines (or 2-aminopyrazines) and isocyanides followed by Pd-catalyzed cross-coupling reactions with arylboronic acids or thiols gave polysubstituted 3-aminoimidazo[1,2-a]pyridines and 3-aminoimidazo[1,2-a]pyrazines I (X = N, CH; R1 = H, MeO; R2 = H, Cl, Me; R3 = n-Bu, cyclohexyl, PhCH2; R4 = 3-ClC6H4, 4-MeCOC6H4, 4-MeOC6H4, cyclohexylthio, 4-MeOC6H4S).  Products were purified by fluorous solid-phase extn. (F-SPE) or crystn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmw3GZnlqnSbVg90H21EOLACvtfcHk0lg5uFy7TW79Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktVamsQ%253D%253D&md5=c5d67e63b511ebc4c2c2af7c68cf5ebf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fqsar.200420045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqsar.200420045%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DMicrowave-assisted%2520synthesis%2520of%2520a%25203-aminoimidazo%255B1%252C2-a%255D-pyridine%252Fpyrazine%2520library%2520by%2520fluorous%2520multicomponent%2520reactions%2520and%2520subsequent%2520cross-coupling%2520reactions%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2004%26volume%3D23%26spage%3D827%26epage%3D835%26doi%3D10.1002%2Fqsar.200420045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baitaluk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, J.</span></span> <span> </span><span class="NLM_article-title">BindingDB in 2015: A public database for medicinal chemistry, computational chemistry and systems pharmacology</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D1045</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1093%2Fnar%2Fgkv1072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26481362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D1045-1053&author=M.+K.+Gilsonauthor=T.+Liuauthor=M.+Baitalukauthor=G.+Nicolaauthor=L.+Hwangauthor=J.+Chong&title=BindingDB+in+2015%3A+A+public+database+for+medicinal+chemistry%2C+computational+chemistry+and+systems+pharmacology&doi=10.1093%2Fnar%2Fgkv1072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB in 2015: A public database for medicinal chemistry, computational chemistry and systems pharmacology</span></div><div class="casAuthors">Gilson, Michael K.; Liu, Tiqing; Baitaluk, Michael; Nicola, George; Hwang, Linda; Chong, Jenny</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1045-D1053</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB, www.bindingdb.org, is a publicly accessible database of exptl. protein-small mol. interaction data.  Its collection of over a million data entries derives primarily from scientific articles and, increasingly, US patents.  BindingDB provides many ways to browse and search for data of interest, including an advanced search tool, which can cross searches ofmultiple query types, including text, chem. structure, protein sequence and numerical affinities.  The PDB and PubMed provide links to data in BindingDB, and vice versa; and BindingDB provides links to pathway information, the ZINC catalog of available compds., and other resources.  The BindingDB website offers specialized tools that take advantage of its large data collection, including ones to generate hypotheses for the protein targets bound by a bioactive compd., and for the compds. bound by a new protein of known sequence; and virtual compd. screening by maximal chem. similarity, binary kernel discrimination, and support vector machine methods.  Specialized data sets are also available, such as binding data for hundreds of congeneric series of ligands, drawn from BindingDB and organized for use in validating drug design methods.  BindingDB offers several forms of programmatic access, and comes with extensive background material and documentation.  Here, we provide the first update of BindingDB since 2007, focusing on new and unique features and highlighting directions of importance to the field as a whole.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOJGtudYTYv7Vg90H21EOLACvtfcHk0lg5uFy7TW79Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7fM&md5=3d243b2579640559825285bdcbbacbfd</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1072%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DBaitaluk%26aufirst%3DM.%26aulast%3DNicola%26aufirst%3DG.%26aulast%3DHwang%26aufirst%3DL.%26aulast%3DChong%26aufirst%3DJ.%26atitle%3DBindingDB%2520in%25202015%253A%2520A%2520public%2520database%2520for%2520medicinal%2520chemistry%252C%2520computational%2520chemistry%2520and%2520systems%2520pharmacology%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD1045%26epage%3D1053%26doi%3D10.1093%2Fnar%2Fgkv1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lhhdumoJJWFQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">phenix.model_vs_data: a high-level tool for the calculation of crystallographic model and data statistics</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1107/S0021889810015608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1107%2FS0021889810015608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=20648263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=669-676&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=V.+B.+Chenauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=phenix.model_vs_data%3A+a+high-level+tool+for+the+calculation+of+crystallographic+model+and+data+statistics&doi=10.1107%2FS0021889810015608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">phenix.model vs. data: a high-level tool for the calculation of crystallographic model and data statistics</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Chen, Vincent B.; Headd, Jeffrey J.; Moriarty, Nigel W.; Richardson, Jane S.; Richardson, David C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.modelvsdata is a high-level command-line tool for the computation of crystallog. model and data statistics, and the evaluation of the fit of the model to data.  Anal. of all Protein Data Bank structures that have exptl. data available shows that in most cases the reported statistics, in particular R factors, can be reproduced within a few percentage points.  However, there are a no. of outliers where the recomputed R values are significantly different from those originally reported.  The reasons for these discrepancies are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBhMlnjsRfhLVg90H21EOLACvtfcHk0lhhdumoJJWFQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D&md5=5cd851a911ecd8b518e3ec4cdd5d9dd7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0021889810015608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889810015608%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Dphenix.model_vs_data%253A%2520a%2520high-level%2520tool%2520for%2520the%2520calculation%2520of%2520crystallographic%2520model%2520and%2520data%2520statistics%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D669%26epage%3D676%26doi%3D10.1107%2FS0021889810015608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhhdumoJJWFQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0015.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Reported potent CBP inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Two-Step Synthesis of Imidazo[1,2-<i>a</i>]pyridine-Based CBP/P300 Inhibitors; Reaction Conditions: (a) Sc(OTf)<sub>3</sub>, CH<sub>2</sub>Cl<sub>2</sub>/MeOH (3:1), μw, 100 °C, 40 min 90–95%; (b) 3,5-Dimethylisoxazole-4-boronic Acid Pinacol Ester, PdCl<sub>2</sub> (dppf)·CH<sub>2</sub>Cl<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub> DME/H<sub>2</sub>O (2:1), μw, 120 °C, 40 min, 89–93%</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Development of small-molecule-based CBP/P300 AlphaScreening assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0011.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Biotinylated-CBP/P300 Bromodomain Binder for AlphaScreening Assay Development</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docked poses of compounds <b>3</b> (cyan) and <b>4</b> (orange) in CBP (PDBs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docked poses of compounds <b>7</b> (UMB276, green) and <b>16</b> (UMB295, yellow) in CBP. PDBs (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a> for CBP).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Docking pose of compound <b>23</b>. (A) Docked poses of compound <b>23</b> (UMB298) (magenta) in CBP. (B) Docked poses of compounds <b>7</b> (UMB276) (green), <b>16</b> (UMB295) (yellow), and <b>23</b> (UMB298) (magenta) in BRD4. Water molecules are denoted W, and residues are denoted by their three-letter amino acid code and their residue number in their respective PDBs (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a> for CBP and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4">5BT4</a> for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=BRD4">BRD4</a>). Hydrogen bonds between the molecule and surrounding residues and water are shown with red lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Comparison of the cocrystal structure of compound <b>23</b> with CBP30. (A) Crystal structures of CBP30 bound to CBP (left, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7">4NR7</a>) and (B) <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=BRD4">BRD4</a> (right, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4">5BT4</a>). Water molecules are denoted W, and residues are denoted by their three-letter amino acid code. Hydrogen bonds among the molecule, surrounding residues, and water are shown with red lines. (C) Compound <b>23</b> (UMB298) (yellow) bound to the KAc site of CBP (gray) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KPY">7KPY</a>). H-bonds with Asn1168 (magenta) and water molecules (cyan) are indicated as black dotted lines. The inset shows the 2Fo-Fc electron density of the inhibitor. (D) Schematic presentation of H-bonding (black dotted lines) and hydrophobic van der Waals interactions (green wiggle lines) of UMB298 with KAc site residues. Arg1173 appears to establish cation−π interactions with the chloromethoxybenzene ring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Potency and selectivity profiling. (A). Tabulated temperature shift values of compounds <b>7</b>, <b>16, 23</b>, and CBP30 with CBP (bromodomain), EP300 (bromodomain), and BRD4(BD1). (B). BROMOSCAN data on compounds <b>7</b>, <b>16</b>, and <b>23</b> against 34 bromodomains. (C). <i>K</i><sub>d</sub> value of compounds <b>16</b>, <b>23</b>, and biotinylated probe compound C toward CBP, EP300, and BRD4 (BD1) determined by BromoELECT. All <i>K</i><sub>d</sub> values are measured in duplicates following the standard protocol developed by Eurofins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/medium/jm0c02232_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cellular activity of CBP/EP300 inhibitors in multiple myeloma cell lines. (A) Dose-proportional effect of CBP30 and compound <b>23</b> (UMB298) (5 days) on MOLM13 cellular viability as approximated by ATP-dependent luminescence (means ± standard error of the mean, SEM, <i>n</i> = 4). (B) Immunoblot of MYC, β-actin, H2K27ac, and H3 following 2 h of incubation with dimethyl sulfoxide (DMSO) or the indicated concentrations of CBP30 or compound <b>23</b> (UMB298) in MOLM13 cells. (C) Quantitative reverse transcription-polymerase chain reaction (RT-PCR) analysis of transcript levels of MYC after a 2 h treatment of MOLM13 cells with DMSO, 3.0 μM CBP30, or 3.0 μM UMB298. Values represent three biologic replicate means ± SEM. *<i>P</i> = 0.01–0.05; **<i>P</i> = 0.001– 0.01; ***<i>P</i> = 0.0001–0.001; ****<i>P</i> < 0.0001. (D) Immunoblot of MYC, IRF4 following 24 h incubation with DMSO or 2.5 μM CBP30, compound <b>23</b> (UMB298), compound <b>10</b>, and compound <b>11</b> in MM.1S cells. β-actin is used as the loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02232/20210506/images/large/jm0c02232_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02232&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i59">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88110" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88110" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clegg, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Advancements in the development of non-BET bromodomain chemical probes</span>. <i>ChemMedChem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fcmdc.201800738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=30624862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=362-385&author=M.+A.+Cleggauthor=N.+C.+O.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Advancements+in+the+development+of+non-BET+bromodomain+chemical+probes&doi=10.1002%2Fcmdc.201800738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Advancements in the Development of non-BET Bromodomain Chemical Probes</span></div><div class="casAuthors">Clegg, Michael A.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">362-385</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extra terminal (BET) family of bromodomain-contg. proteins (BCPs) have been the subject of extensive research over the past decade, resulting in a plethora of high-quality chem. probes for their tandem bromodomains.  In turn, these chem. probes have helped reveal the profound biol. role of the BET bromodomains and their role in disease, ultimately leading to a no. of mols. in active clin. development.  However, the BET subfamily represents just 8/61 of the known human bromodomains, and attention has now expanded to the biol. role of the remaining 53 non-BET bromodomains.  Rapid growth of this research area has been accompanied by a greater understanding of the requirements for an effective bromodomain chem. probe and has led to a no. of new non-BET bromodomain chem. probes being developed.  Advances since Dec. 2015 are discussed, highlighting the strengths/caveats of each mol., and the value they add toward validating the non-BET bromodomains as tractable therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_U9uvpaCEd7Vg90H21EOLACvtfcHk0lhfaaWLGhGzrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Ohs70%253D&md5=a93505759fe45ffd0352de2f0e17885a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800738%26sid%3Dliteratum%253Aachs%26aulast%3DClegg%26aufirst%3DM.%2BA.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%2BO.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DAdvancements%2520in%2520the%2520development%2520of%2520non-BET%2520bromodomain%2520chemical%2520probes%26jtitle%3DChemMedChem.%26date%3D2019%26volume%3D14%26spage%3D362%26epage%3D385%26doi%3D10.1002%2Fcmdc.201800738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Theodoulou, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomkinson, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span> <span> </span><span class="NLM_article-title">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2016.05.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.cbpa.2016.05.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27295577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2016&pages=58-66&author=N.+H.+Theodoulouauthor=N.+C.+Tomkinsonauthor=R.+K.+Prinjhaauthor=P.+G.+Humphreys&title=Clinical+progress+and+pharmacology+of+small+molecule+bromodomain+inhibitors&doi=10.1016%2Fj.cbpa.2016.05.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical progress and pharmacology of small molecule bromodomain inhibitors</span></div><div class="casAuthors">Theodoulou, Natalie H.; Tomkinson, Nicholas C. O.; Prinjha, Rab K.; Humphreys, Philip G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Bromodomains have emerged as an exciting target class for drug discovery over the past decade.  Research has primarily focused on the bromodomain and extra terminal (BET) family of bromodomains, which has led to the development of multiple small mol. inhibitors and an increasing no. of clin. assets.  The excitement centered on the clin. potential of BET inhibition has stimulated intense interest in the broader family and the growing no. of non-BET bromodomain chem. probes has facilitated phenotypic investigations, implicating these targets in a variety of disease pathways including cancer, inflammation, embryonic development and neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEE0jCnUuyk7Vg90H21EOLACvtfcHk0ljbgRU2vBG8ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xps1artb8%253D&md5=4bcd42755b2dd3cebd7fae417e69e83f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2016.05.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2016.05.028%26sid%3Dliteratum%253Aachs%26aulast%3DTheodoulou%26aufirst%3DN.%2BH.%26aulast%3DTomkinson%26aufirst%3DN.%2BC.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26atitle%3DClinical%2520progress%2520and%2520pharmacology%2520of%2520small%2520molecule%2520bromodomain%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D33%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.cbpa.2016.05.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewings, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">Progress in the development and application of small molecule inhibitors of bromodomain-acetyl-lysine interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">9393</span>– <span class="NLM_lpage">9413</span>, <span class="refDoi"> DOI: 10.1021/jm300915b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300915b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1OmtL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9393-9413&author=D.+S.+Hewingsauthor=T.+P.+Rooneyauthor=L.+E.+Jenningsauthor=D.+A.+Hayauthor=C.+J.+Schofieldauthor=P.+E.+Brennanauthor=S.+Knappauthor=S.+J.+Conway&title=Progress+in+the+development+and+application+of+small+molecule+inhibitors+of+bromodomain-acetyl-lysine+interactions&doi=10.1021%2Fjm300915b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Progress in the Development and Application of Small Molecule Inhibitors of Bromodomain-Acetyl-lysine Interactions</span></div><div class="casAuthors">Hewings, David S.; Rooney, Timothy P. C.; Jennings, Laura E.; Hay, Duncan A.; Schofield, Christopher J.; Brennan, Paul E.; Knapp, Stefan; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9393-9413</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Bromodomains, protein modules that recognize and bind to acetylated lysine, are emerging as important components of cellular machinery.  These acetyl-lysine (KAc) reader domains are part of the write-read-erase concept that has been linked with the transfer of epigenetic information.  By reading KAc marks on histones, bromodomains mediate protein-protein interactions between a diverse array of partners.  There has been intense activity in developing potent and selective small mol. probes that disrupt the interaction between a given bromodomain and KAc.  Rapid success has been achieved with the BET family of bromodomains, and a no. of potent and selective probes have been reported.  These compds. have enabled linking of the BET bromodomains with diseases, including cancer and inflammation, suggesting that bromodomains are druggable targets.  Herein, we review the biol. of the bromodomains and discuss the SAR for the existing small mol. probes.  The biol. that has been enabled by these compds. is summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8pl0pnLXruLVg90H21EOLACvtfcHk0ljbgRU2vBG8ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1OmtL%252FJ&md5=486a87abb406c4c833404ccce61c1389</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm300915b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300915b%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DProgress%2520in%2520the%2520development%2520and%2520application%2520of%2520small%2520molecule%2520inhibitors%2520of%2520bromodomain-acetyl-lysine%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9393%26epage%3D9413%26doi%3D10.1021%2Fjm300915b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0ljbgRU2vBG8ZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">The Bromodomain: a new target in emerging epigenetic medicine</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">598</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00831</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00831" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=598-608&author=S.+G.+Smithauthor=M.+M.+Zhou&title=The+Bromodomain%3A+a+new+target+in+emerging+epigenetic+medicine&doi=10.1021%2Facschembio.5b00831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The Bromodomain: A New Target in Emerging Epigenetic Medicine</span></div><div class="casAuthors">Smith, Steven G.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">598-608</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain (BrD) is a conserved protein modular domain found in many chromatin- and transcription-assocd. proteins that has the ability to recognize acetylated lysine residues.  This activity allows bromodomains to play a vital role in many acetylation-mediated protein-protein interactions in the cell, ranging from substrate recruitment for histone acetyltransferases (HATs) to aiding in multiple-protein complex assembly for gene transcriptional activation or suppression in chromatin.  In recent years, considerable efforts have been made to develop chem. inhibitors of these bromodomains in an effort to probe their cellular functions.  Potent and selective inhibitors have been extensively developed for one group of the bromodomain family termed BET proteins that consist of tandem bromodomains followed by an extra terminal domain.  Drug developers are actively designing inhibitors of other bromodomains and working to move the most successful inhibitors into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTQF2Qjn_iuLVg90H21EOLACvtfcHk0li364wXgmJg6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyit7fJ&md5=dd34fc591550ff1528e6dccd1d04f1c1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00831%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DS.%2BG.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DThe%2520Bromodomain%253A%2520a%2520new%2520target%2520in%2520emerging%2520epigenetic%2520medicine%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D598%26epage%3D608%26doi%3D10.1021%2Facschembio.5b00831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Druggability analysis and structural classification of bromodomain acetyl-lysine binding sites</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7346</span>– <span class="NLM_lpage">7359</span>, <span class="refDoi"> DOI: 10.1021/jm300346w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300346w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7346-7359&author=L.+R.+Vidlerauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Druggability+analysis+and+structural+classification+of+bromodomain+acetyl-lysine+binding+sites&doi=10.1021%2Fjm300346w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability Analysis and Structural Classification of Bromodomain Acetyl-lysine Binding Sites</span></div><div class="casAuthors">Vidler, Lewis R.; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7346-7359</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains are readers of the epigenetic code that specifically bind acetyl-lysine contg. recognition sites on proteins.  Recently the BET family of bromodomains has been demonstrated to be druggable through the discovery of potent inhibitors, sparking an interest in protein-protein interaction inhibitors that directly target gene transcription.  Here, we assess the druggability of diverse members of the bromodomain family using SiteMap and show that there are significant differences in predicted druggability.  Furthermore, we trace these differences in druggability back to unique amino acid signatures in the bromodomain acetyl-lysine binding sites.  These signatures were then used to generate a new classification of the bromodomain family, visualized as a classification tree.  This represents the first anal. of this type for the bromodomain family and can prove useful in the discovery of inhibitors, particularly for anticipating screening hit rates, identifying inhibitors that can be explored for lead hopping approaches, and selecting proteins for selectivity screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxYqQyqqj7JrVg90H21EOLACvtfcHk0li364wXgmJg6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWht7fN&md5=bfb08154c1964aab5df51cb3513f2c1d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm300346w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300346w%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDruggability%2520analysis%2520and%2520structural%2520classification%2520of%2520bromodomain%2520acetyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7346%26epage%3D7359%26doi%3D10.1021%2Fjm300346w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karakikes, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moshkina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajjar, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">A small molecule binding to the coactivator CREB-binding protein blocks apoptosis in cardiomyocytes</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2010.12.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.chembiol.2010.12.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=21513889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=531-541&author=J.+C.+Borahauthor=S.+Mujtabaauthor=I.+Karakikesauthor=L.+Zengauthor=M.+Mullerauthor=J.+Patelauthor=N.+Moshkinaauthor=K.+Morohashiauthor=W.+Zhangauthor=G.+Gerona-Navarroauthor=R.+J.+Hajjarauthor=M.+M.+Zhou&title=A+small+molecule+binding+to+the+coactivator+CREB-binding+protein+blocks+apoptosis+in+cardiomyocytes&doi=10.1016%2Fj.chembiol.2010.12.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Small Molecule Binding to the Coactivator CREB-Binding Protein Blocks Apoptosis in Cardiomyocytes</span></div><div class="casAuthors">Borah, Jagat C.; Mujtaba, Shiraz; Karakikes, Ioannis; Zeng, Lei; Muller, Michaela; Patel, Jigneshkumar; Moshkina, Natasha; Morohashi, Keita; Zhang, Weijia; Gerona-Navarro, Guillermo; Hajjar, Roger J.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">531-541</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: As a master transcription factor in cellular responses to external stress, tumor suppressor p53 is tightly regulated.  Excessive p53 activity during myocardial ischemia causes irreversible cellular injury and cardiomyocyte death.  P53 activation is dependent on lysine acetylation by the lysine acetyltransferase and transcriptional coactivator CREB-binding protein (CBP) and on acetylation-directed CBP recruitment for p53 target gene expression.  Here, we report a small mol. ischemin, developed with a structure-guided approach to inhibit the acetyl-lysine binding activity of the bromodomain of CBP.  We show that ischemin alters post-translational modifications on p53 and histones, inhibits p53 interaction with CBP and transcriptional activity in cells, and prevents apoptosis in ischemic cardiomyocytes.  Our study suggests small mol. modulation of acetylation-mediated interactions in gene transcription as a new approach to therapeutic interventions of human disorders such as myocardial ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHKOaQAGSYTbVg90H21EOLACvtfcHk0li364wXgmJg6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFartb4%253D&md5=0c1d7500e1fff40bc335d1b10ac3c27a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2010.12.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2010.12.021%26sid%3Dliteratum%253Aachs%26aulast%3DBorah%26aufirst%3DJ.%2BC.%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DKarakikes%26aufirst%3DI.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DMuller%26aufirst%3DM.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DMoshkina%26aufirst%3DN.%26aulast%3DMorohashi%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DHajjar%26aufirst%3DR.%2BJ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DA%2520small%2520molecule%2520binding%2520to%2520the%2520coactivator%2520CREB-binding%2520protein%2520blocks%2520apoptosis%2520in%2520cardiomyocytes%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D531%26epage%3D541%26doi%3D10.1016%2Fj.chembiol.2010.12.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd.</span></span> <span> </span><span class="NLM_article-title">Bromodomains: a new target class for drug development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0030-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1038%2Fs41573-019-0030-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=31273347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=609-628&author=A.+G.+Cochranauthor=A.+R.+Coneryauthor=R.+J.+Sims&title=Bromodomains%3A+a+new+target+class+for+drug+development&doi=10.1038%2Fs41573-019-0030-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: a new target class for drug development</span></div><div class="casAuthors">Cochran, Andrea G.; Conery, Andrew R.; Sims, III, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">609-628</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Less than a decade ago, it was shown that bromodomains, acetyl lysine 'reader' modules found in proteins with varied functions, were highly tractable small-mol. targets.  This is an unusual property for protein-protein or protein-peptide interaction domains, and it prompted a wave of chem. probe discovery to understand the biol. potential of new agents that targeted bromodomains.  The original examples, inhibitors of the bromodomain and extra-terminal (BET) class of bromodomains, showed enticing anti-inflammatory and anticancer activities, and several compds. have since advanced to human clin. trials.  Here, we review the current state of BET inhibitor biol. in relation to clin. development, and we discuss the next wave of bromodomain inhibitors with clin. potential in oncol. and non-oncol. indications.  The lessons learned from BET inhibitor programs should affect efforts to develop drugs that target non-BET bromodomains and other epigenetic readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6QwxqowW9NbVg90H21EOLACvtfcHk0lgKOHS3jc80UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL&md5=88e56eec8edeb7c325d28156ec131ed8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomains%253A%2520a%2520new%2520target%2520class%2520for%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D609%26epage%3D628%26doi%3D10.1038%2Fs41573-019-0030-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletcher, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L. P.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of highly selective inhibitors of the CREB binding protein Bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5349</span>– <span class="NLM_lpage">5363</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01839</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01839" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5349-5363&author=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+Stockauthor=P.+Sahasrabudheauthor=P.+Boninauthor=D.+A.+Hayauthor=P.+E.+Brennanauthor=M.+Pletcherauthor=L.+H.+Jonesauthor=E.+L.+P.+Chekler&title=Structure-based+design+of+highly+selective+inhibitors+of+the+CREB+binding+protein+Bromodomain&doi=10.1021%2Facs.jmedchem.6b01839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Highly Selective Inhibitors of the CREB Binding Protein Bromodomain</span></div><div class="casAuthors">Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid; Sahasrabudhe, Parag; Bonin, Paul; Hay, Duncan A.; Brennan, Paul E.; Pletcher, Mathew; Jones, Lyn H.; Chekler, Eugene L. Piatnitski</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5349-5363</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. probes are required for preclin. target validation to interrogate novel biol. targets and pathways.  Selective inhibitors of the CREB binding protein (CREBBP)/EP300 bromodomains are required to facilitate the elucidation of biol. assocd. with these important epigenetic targets.  Medicinal chem. optimization that paid particular attention to physiochem. properties delivered chem. probes with desirable potency, selectivity, and permeability attributes.  An important feature of the optimization process was the successful application of rational structure-based drug design to address bromodomain selectivity issues (particularly against the structurally related BRD4 protein).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLWkqF-qAmdbVg90H21EOLACvtfcHk0lgKOHS3jc80UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1Glu70%253D&md5=23676a973abba59088dedd5502877105</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01839%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DPletcher%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DChekler%26aufirst%3DE.%2BL.%2BP.%26atitle%3DStructure-based%2520design%2520of%2520highly%2520selective%2520inhibitors%2520of%2520the%2520CREB%2520binding%2520protein%2520Bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D5349%26epage%3D5363%26doi%3D10.1021%2Facs.jmedchem.6b01839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singleton, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Discovery and optimization of small-molecule ligands for the CBP/p300 bromodomains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">9308</span>– <span class="NLM_lpage">9319</span>, <span class="refDoi"> DOI: 10.1021/ja412434f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja412434f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=9308-9319&author=D.+A.+Hayauthor=O.+Fedorovauthor=S.+Martinauthor=D.+C.+Singletonauthor=C.+Tallantauthor=C.+Wellsauthor=S.+Picaudauthor=M.+Philpottauthor=O.+P.+Monteiroauthor=C.+M.+Rogersauthor=S.+J.+Conwayauthor=T.+P.+Rooneyauthor=A.+Tumberauthor=C.+Yappauthor=P.+Filippakopoulosauthor=M.+E.+Bunnageauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=C.+J.+Schofieldauthor=P.+E.+Brennan&title=Discovery+and+optimization+of+small-molecule+ligands+for+the+CBP%2Fp300+bromodomains&doi=10.1021%2Fja412434f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Small-Molecule Ligands for the CBP/p300 Bromodomains</span></div><div class="casAuthors">Hay, Duncan A.; Fedorov, Oleg; Martin, Sarah; Singleton, Dean C.; Tallant, Cynthia; Wells, Christopher; Picaud, Sarah; Philpott, Martin; Monteiro, Octovia P.; Rogers, Catherine M.; Conway, Stuart J.; Rooney, Timothy P. C.; Tumber, Anthony; Yapp, Clarence; Filippakopoulos, Panagis; Bunnage, Mark E.; Muller, Susanne; Knapp, Stefan; Schofield, Christopher J.; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">9308-9319</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors that target bromodomains outside of the bromodomain and extra-terminal (BET) sub-family are lacking.  Here, the authors describe highly potent and selective ligands for the bromodomain module of the human lysine acetyl transferase CBP/p300, developed from a series of 5-isoxazolyl-benzimidazoles.  The starting point was a fragment hit, which was optimized into a more potent and selective lead using parallel synthesis employing Suzuki couplings, benzimidazole-forming reactions, and reductive aminations.  The selectivity of the lead compd. against other bromodomain family members was investigated using a thermal stability assay, which revealed some inhibition of the structurally related BET family members.  To address the BET selectivity issue, x-ray crystal structures of the lead compd. bound to the CREB binding protein (CBP) and the first bromodomain of BRD4 (BRD4(1)) were used to guide the design of more selective compds.  The crystal structures obtained revealed two distinct binding modes.  By varying the aryl substitution pattern and developing conformationally constrained analogs, selectivity for CBP over BRD4(1) was increased.  The optimized compd. is highly potent (Kd = 21 nM) and selective, displaying 40-fold selectivity over BRD4(1).  Cellular activity was demonstrated using fluorescence recovery after photo-bleaching (FRAP) and a p53 reporter assay.  The optimized compds. are cell-active and have nanomolar affinity for CBP/p300; therefore, they should be useful in studies investigating the biol. roles of CBP and p300 and to validate the CBP and p300 bromodomains as therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFzgs1DD4QZLVg90H21EOLACvtfcHk0lgKOHS3jc80UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsLvP&md5=e5da3e96a4d234453a481c7ee5e788f2</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fja412434f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja412434f%26sid%3Dliteratum%253Aachs%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSingleton%26aufirst%3DD.%2BC.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMonteiro%26aufirst%3DO.%2BP.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520small-molecule%2520ligands%2520for%2520the%2520CBP%252Fp300%2520bromodomains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D9308%26epage%3D9319%26doi%3D10.1021%2Fja412434f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dancy, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">Protein lysine acetylation by p300/CBP</span>. <i>Chem Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">2419</span>– <span class="NLM_lpage">2452</span>, <span class="refDoi"> DOI: 10.1021/cr500452k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr500452k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2015&pages=2419-2452&author=B.+M.+Dancyauthor=P.+A.+Cole&title=Protein+lysine+acetylation+by+p300%2FCBP&doi=10.1021%2Fcr500452k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Lysine Acetylation by p300/CBP</span></div><div class="casAuthors">Dancy, Beverley M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2419-2452</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human protein lysine acetyltransferase encoded by the paralogous p300 and CBP genes is involved in regulating the expression of genes.  In this capacity, p300/CBP controls several basic cellular processes such as proliferation and homeostasis, and plays a role in a variety of human diseases, particularly solid tumors.  This review summarizes information related to the current understanding of p300/CBP, including the novel technologies that have co-evolved with p300/CBP research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYgTImMiuo7rVg90H21EOLACvtfcHk0lhhPzfhd47nZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVCiu7Y%253D&md5=40a46cf68990f4a7d11a338c54081628</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcr500452k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr500452k%26sid%3Dliteratum%253Aachs%26aulast%3DDancy%26aufirst%3DB.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DProtein%2520lysine%2520acetylation%2520by%2520p300%252FCBP%26jtitle%3DChem%2520Rev.%26date%3D2015%26volume%3D115%26spage%3D2419%26epage%3D2452%26doi%3D10.1021%2Fcr500452k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teufel, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bycroft, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fersht, A. R.</span></span> <span> </span><span class="NLM_article-title">Four domains of p300 each bind tightly to a sequence spanning both transactivation subdomains of p53</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">7009</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1073/pnas.0702010104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1073%2Fpnas.0702010104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=17438265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BD2sXls1Oktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=7009-7014&author=D.+P.+Teufelauthor=S.+M.+Freundauthor=M.+Bycroftauthor=A.+R.+Fersht&title=Four+domains+of+p300+each+bind+tightly+to+a+sequence+spanning+both+transactivation+subdomains+of+p53&doi=10.1073%2Fpnas.0702010104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Four domains of p300 each bind tightly to a sequence spanning both transactivation subdomains of p53</span></div><div class="casAuthors">Teufel, Daniel P.; Freund, Stefan M.; Bycroft, Mark; Fersht, Alan R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7009-7014</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The transcriptional coactivator p300 binds to and mediates the transcriptional functions of the tetrameric tumor suppressor p53.  Both proteins consist of independently folded domains linked by intrinsically disordered sequences.  A well studied short sequence of the p53 transactivation domain, p53(15-29), binds weakly to four folded domains of p300 [Taz1/cysteine-histidine-rich region 1 (CH1), Kix, Taz2/CH3, IBiD], with dissocn. consts. (KD) in the 100 μM region.  However, we found that a longer N-terminal transactivation domain construct p53(1-57) bound tightly to each p300 domain.  Taz2/CH3 had the greatest affinity (KD = 27 nM) and competes with the N-terminal domain of Mdm2 for the p53 N terminus.  P300 thus can protect the N terminus of p53 against the binding of other proteins.  Mutations of p53 that abrogate transactivation (L22Q/W23S, W53Q/F54S) greatly weakened binding to each p300 domain, linking phenotypic defects to weakened coactivator binding.  We propose a complex between tetrameric p53 and p300 in which four domains of p300 wrap around the four transactivation domains of p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvdOsvMUQZu7Vg90H21EOLACvtfcHk0lhhPzfhd47nZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXls1Oktb4%253D&md5=95a601ed428c7fc61b8a9b01b58d9778</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0702010104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0702010104%26sid%3Dliteratum%253Aachs%26aulast%3DTeufel%26aufirst%3DD.%2BP.%26aulast%3DFreund%26aufirst%3DS.%2BM.%26aulast%3DBycroft%26aufirst%3DM.%26aulast%3DFersht%26aufirst%3DA.%2BR.%26atitle%3DFour%2520domains%2520of%2520p300%2520each%2520bind%2520tightly%2520to%2520a%2520sequence%2520spanning%2520both%2520transactivation%2520subdomains%2520of%2520p53%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D7009%26epage%3D7014%26doi%3D10.1073%2Fpnas.0702010104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikov, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frasca, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structural insights into acetylated-histone H4 recognition by the bromodomain-PHD finger module of human transcriptional coactivator CBP</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">360</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.str.2013.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=24361270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOhtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=353-360&author=A.+N.+Plotnikovauthor=S.+Yangauthor=T.+J.+Zhouauthor=E.+Rusinovaauthor=A.+Frascaauthor=M.+M.+Zhou&title=Structural+insights+into+acetylated-histone+H4+recognition+by+the+bromodomain-PHD+finger+module+of+human+transcriptional+coactivator+CBP&doi=10.1016%2Fj.str.2013.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Insights into Acetylated-Histone H4 Recognition by the Bromodomain-PHD Finger Module of Human Transcriptional Coactivator CBP</span></div><div class="casAuthors">Plotnikov, Alexander N.; Yang, Shuai; Zhou, Thomas Jiachi; Rusinova, Elena; Frasca, Antonio; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomain functions as the acetyl-lysine binding domains to regulate gene transcription in chromatin.  Bromodomains are rapidly emerging as new epigenetic drug targets for human diseases.  However, owing to their transient nature and modest affinity, histone-binding selectivity of bromodomains has remained mostly elusive.  Here, we report high-resoln. crystal structures of the bromodomain-PHD tandem module of human transcriptional coactivator CBP bound to lysine-acetylated histone H4 peptides.  The structures reveal that the PHD finger serves a structural role in the tandem module and that the bromodomain prefers lysine-acetylated motifs comprising a hydrophobic or arom. residue at -2 and a lysine or arginine at -3 or -4 position from the acetylated lysine.  Our study further provides structural insights into distinct modes of singly and diacetylated histone H4 recognition by the bromodomains of CBP and BRD4 that function differently as a transcriptional coactivator and chromatin organizer, resp., explaining their distinct roles in control of gene expression in chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy6eG4JtL6sbVg90H21EOLACvtfcHk0lhhPzfhd47nZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOhtrvN&md5=15389e25c153d9378cb9b9f81818dc25</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DT.%2BJ.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DFrasca%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructural%2520insights%2520into%2520acetylated-histone%2520H4%2520recognition%2520by%2520the%2520bromodomain-PHD%2520finger%2520module%2520of%2520human%2520transcriptional%2520coactivator%2520CBP%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D353%26epage%3D360%26doi%3D10.1016%2Fj.str.2013.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mujtaba, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnikova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sachchidanand</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structural mechanism of the bromodomain of the coactivator CBP in p53 transcriptional activation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">263</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(03)00528-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2FS1097-2765%2803%2900528-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=14759370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFOhsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2004&pages=251-263&author=S.+Mujtabaauthor=Y.+Heauthor=L.+Zengauthor=S.+Yanauthor=O.+Plotnikovaauthor=+Sachchidanandauthor=R.+Sanchezauthor=N.+J.+Zeleznik-Leauthor=Z.+Ronaiauthor=M.+M.+Zhou&title=Structural+mechanism+of+the+bromodomain+of+the+coactivator+CBP+in+p53+transcriptional+activation&doi=10.1016%2FS1097-2765%2803%2900528-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural mechanism of the bromodomain of the coactivator CBP in p53 transcriptional activation</span></div><div class="casAuthors">Mujtaba, Shiraz; He, Yan; Zeng, Lei; Yan, Sherry; Plotnikova, Olga; Sachchidanand; Sanchez, Roberto; Zeleznik-Le, Nancy J.; Ronai, Ze'ev; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">251-263</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine acetylation of the tumor suppressor protein p53 in response to a wide variety of cellular stress signals is required for its activation as a transcription factor that regulates cell cycle arrest, senescence, or apoptosis.  Here, the authors report that the conserved bromodomain of the transcriptional coactivator CBP (CREB binding protein) binds specifically to p53 at the C-terminal acetylated lysine 382.  This bromodomain/acetyllysine binding is responsible for p53 acetylation-dependent coactivator recruitment after DNA damage, a step essential for p53-induced transcriptional activation of the cyclin-dependent kinase inhibitor p21 in G1 cell cycle arrest.  The authors further present the three-dimensional NMR structure of the CBP bromodomain in complex with a lysine 382-acetylated p53 peptide.  Using structural and biochem. analyses, the authors define the mol. determinants for the specificity of this mol. recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3g6bC53YUKbVg90H21EOLACvtfcHk0ljFrnfNgDzz9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFOhsr0%253D&md5=10127e2d2ea5165fd8439bf26a6f9241</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900528-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900528-8%26sid%3Dliteratum%253Aachs%26aulast%3DMujtaba%26aufirst%3DS.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DS.%26aulast%3DPlotnikova%26aufirst%3DO.%26aulast%3DSachchidanand%26aulast%3DSanchez%26aufirst%3DR.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26aulast%3DRonai%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructural%2520mechanism%2520of%2520the%2520bromodomain%2520of%2520the%2520coactivator%2520CBP%2520in%2520p53%2520transcriptional%2520activation%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D13%26spage%3D251%26epage%3D263%26doi%3D10.1016%2FS1097-2765%2803%2900528-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wimalasena, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigua, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durbin, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span> <span> </span><span class="NLM_article-title">Using chemical epigenetics to target cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1086</span>– <span class="NLM_lpage">1095</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.molcel.2020.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=32407673" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptlersLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=1086-1095&author=V.+K.+Wimalasenaauthor=T.+Wangauthor=L.+H.+Siguaauthor=A.+D.+Durbinauthor=J.+Qi&title=Using+chemical+epigenetics+to+target+cancer&doi=10.1016%2Fj.molcel.2020.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Using Chemical Epigenetics to Target Cancer</span></div><div class="casAuthors">Wimalasena, Virangika K.; Wang, Tingjian; Sigua, Logan H.; Durbin, Adam D.; Qi, Jun</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1086-1095</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Transcription is epigenetically regulated by the orchestrated function of chromatin-binding proteins that tightly control the expression of master transcription factors, effectors, and supportive housekeeping genes required for establishing and propagating the normal and malignant cell state.  Rapid advances in chem. biol. and functional genomics have facilitated exploration of targeting epigenetic proteins, yielding effective strategies to target transcription while reducing toxicities to untransformed cells.  Here, we review recent developments in conventional active site and allosteric inhibitors, peptidomimetics, and novel proteolysis-targeted chimera (PROTAC) technol. that have deepened our understanding of transcriptional processes and led to promising preclin. compds. for therapeutic translation, particularly in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2E4fU5dl1obVg90H21EOLACvtfcHk0ljFrnfNgDzz9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptlersLg%253D&md5=573beb6b682321f14bbd7dc8a535ad1e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DWimalasena%26aufirst%3DV.%2BK.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DSigua%26aufirst%3DL.%2BH.%26aulast%3DDurbin%26aufirst%3DA.%2BD.%26aulast%3DQi%26aufirst%3DJ.%26atitle%3DUsing%2520chemical%2520epigenetics%2520to%2520target%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D1086%26epage%3D1095%26doi%3D10.1016%2Fj.molcel.2020.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salmeron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakle, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mele, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cote, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setser, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeAlmeida-Nagata, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, V. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lora, J. M.</span></span> <span> </span><span class="NLM_article-title">Regulatory T cell modulation by CBP/EP300 bromodomain inhibition</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">13014</span>– <span class="NLM_lpage">13027</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.708560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1074%2Fjbc.M115.708560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27056325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=13014-13027&author=S.+Ghoshauthor=A.+Taylorauthor=M.+Chinauthor=H.+R.+Huangauthor=A.+R.+Coneryauthor=J.+A.+Mertzauthor=A.+Salmeronauthor=P.+J.+Dakleauthor=D.+Meleauthor=A.+Coteauthor=H.+Jayaramauthor=J.+W.+Setserauthor=F.+Poyauthor=G.+Hatzivassiliouauthor=D.+DeAlmeida-Nagataauthor=P.+Sandyauthor=C.+Hattonauthor=F.+A.+Romeroauthor=E.+Chiangauthor=T.+Reimerauthor=T.+Crawfordauthor=E.+Pardoauthor=V.+G.+Watsonauthor=V.+Tsuiauthor=A.+G.+Cochranauthor=L.+Zawadzkeauthor=J.+C.+Harmangeauthor=J.+E.+Audiaauthor=B.+M.+Bryantauthor=R.+T.+Cummingsauthor=S.+R.+Magnusonauthor=J.+L.+Groganauthor=S.+F.+Bellonauthor=B.+K.+Albrechtauthor=R.+J.+Simsauthor=J.+M.+Lora&title=Regulatory+T+cell+modulation+by+CBP%2FEP300+bromodomain+inhibition&doi=10.1074%2Fjbc.M115.708560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Regulatory T Cell Modulation by CBP/EP300 Bromodomain Inhibition</span></div><div class="casAuthors">Ghosh, Srimoyee; Taylor, Alexander; Chin, Melissa; Huang, Hon-Ren; Conery, Andrew R.; Mertz, Jennifer A.; Salmeron, Andres; Dakle, Pranal J.; Mele, Deanna; Cote, Alexandre; Jayaram, Hari; Setser, Jeremy W.; Poy, Florence; Hatzivassiliou, Georgia; DeAlmeida-Nagata, Denise; Sandy, Peter; Hatton, Charlie; Romero, F. Anthony; Chiang, Eugene; Reimer, Thornik; Crawford, Terry; Pardo, Eneida; Watson, Venita G.; Tsui, Vickie; Cochran, Andrea G.; Zawadzke, Laura; Harmange, Jean-Christophe; Audia, James E.; Bryant, Barbara M.; Cummings, Richard T.; Magnuson, Steven R.; Grogan, Jane L.; Bellon, Steve F.; Albrecht, Brian K.; Sims, Robert J. III; Lora, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">13014-13027</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Covalent modification of histones is a fundamental mechanism of regulated gene expression in eukaryotes, and interpretation of histone modifications is an essential feature of epigenetic control.  Bromodomains are specialized binding modules that interact with acetylated histones, linking chromatin recognition to gene transcription.  Because of their ability to function in a domain-specific fashion, selective disruption of bromodomain:acetylated histone interactions with chem. probes serves as a powerful means for understanding biol. processes regulated by these chromatin adaptors.  Here the authors describe the discovery and characterization of potent and selective small mol. inhibitors for the bromodomains of CREBBP/EP300 that engage their target in cellular assays.  The authors use these tools to demonstrate a crit. role for CREBBP/EP300 bromodomains in regulatory T cell biol.  Because regulatory T cell recruitment to tumors is a major mechanism of immune evasion by cancer cells, the authors' data highlight the importance of CREBBP/EP300 bromodomain inhibition as a novel, small mol.-based approach for cancer immunotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9maWuQVImyLVg90H21EOLACvtfcHk0liSN65mf-xreA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksL3F&md5=bc8b4edba81bdbef30b1d8fcfaa19f1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.708560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.708560%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DH.%2BR.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSalmeron%26aufirst%3DA.%26aulast%3DDakle%26aufirst%3DP.%2BJ.%26aulast%3DMele%26aufirst%3DD.%26aulast%3DCote%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%2BW.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DDeAlmeida-Nagata%26aufirst%3DD.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DChiang%26aufirst%3DE.%26aulast%3DReimer%26aufirst%3DT.%26aulast%3DCrawford%26aufirst%3DT.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DWatson%26aufirst%3DV.%2BG.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DMagnuson%26aufirst%3DS.%2BR.%26aulast%3DGrogan%26aufirst%3DJ.%2BL.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DLora%26aufirst%3DJ.%2BM.%26atitle%3DRegulatory%2520T%2520cell%2520modulation%2520by%2520CBP%252FEP300%2520bromodomain%2520inhibition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D13014%26epage%3D13027%26doi%3D10.1074%2Fjbc.M115.708560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olzscha, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span> <span> </span><span class="NLM_article-title">CBP/p300 bromodomains regulate amyloid-like protein aggregation upon aberrant lysine acetylation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.11.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.chembiol.2016.11.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27989401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWnsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=9-23&author=H.+Olzschaauthor=O.+Fedorovauthor=B.+M.+Kesslerauthor=S.+Knappauthor=N.+B.+La+Thangue&title=CBP%2Fp300+bromodomains+regulate+amyloid-like+protein+aggregation+upon+aberrant+lysine+acetylation&doi=10.1016%2Fj.chembiol.2016.11.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">CBP/p300 Bromodomains Regulate Amyloid-like Protein Aggregation upon Aberrant Lysine Acetylation</span></div><div class="casAuthors">Olzscha, Heidi; Fedorov, Oleg; Kessler, Benedikt M.; Knapp, Stefan; La Thangue, Nicholas B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9-23</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Lysine acetylation is becoming increasingly recognized as a general biol. principle in cellular homeostasis, and is subject to abnormal control in different human pathologies.  Here, we describe a global effect on amyloid-like protein aggregation in human cells that results from aberrant lysine acetylation.  Bromodomain reader proteins are involved in the aggregation process and, using chem. biol. and gene silencing, we establish that p300/CBP bromodomains are necessary for aggregation to occur.  Moreover, protein aggregation disturbs proteostasis by impairing the ubiquitin proteasome system (UPS) and protein translation, resulting in decreased cell viability. p300/CBP bromodomain inhibitors impede aggregation, which coincides with enhanced UPS function and increased cell viability.  Aggregation of a pathol. relevant form of huntingtin protein is similarly affected by p300/CBP inhibition.  Our results have implications for understanding the mol. basis of protein aggregation, and highlight the possibility of treating amyloid-like pathologies and related protein folding diseases with bromodomain inhibitor-based strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqql6b79dRZDbVg90H21EOLACvtfcHk0liSN65mf-xreA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWnsb%252FO&md5=01aa97352d581eccff8c9745c6d545f5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.11.009%26sid%3Dliteratum%253Aachs%26aulast%3DOlzscha%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DCBP%252Fp300%2520bromodomains%2520regulate%2520amyloid-like%2520protein%2520aggregation%2520upon%2520aberrant%2520lysine%2520acetylation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D9%26epage%3D23%26doi%3D10.1016%2Fj.chembiol.2016.11.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallipoli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papaemmanuil, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göttgens, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Deursen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J.</span></span> <span> </span><span class="NLM_article-title">The epigenetic regulators CBP and p300 facilitate leukemogenesis and represent therapeutic targets in acute myeloid leukemia</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">279</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.92</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1038%2Fonc.2015.92" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=25893291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFWhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=279-89&author=G.+Giotopoulosauthor=W.+I.+Chanauthor=S.+J.+Hortonauthor=D.+Ruauauthor=P.+Gallipoliauthor=A.+Fowlerauthor=C.+Crawleyauthor=E.+Papaemmanuilauthor=P.+J.+Campbellauthor=B.+G%C3%B6ttgensauthor=J.+M.+Van+Deursenauthor=P.+A.+Coleauthor=B.+J.+Huntly&title=The+epigenetic+regulators+CBP+and+p300+facilitate+leukemogenesis+and+represent+therapeutic+targets+in+acute+myeloid+leukemia&doi=10.1038%2Fonc.2015.92"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenetic regulators CBP and p300 facilitate leukemogenesis and represent therapeutic targets in acute myeloid leukemia</span></div><div class="casAuthors">Giotopoulos, G.; Chan, W.-I.; Horton, S. J.; Ruau, D.; Gallipoli, P.; Fowler, A.; Crawley, C.; Papaemmanuil, E.; Campbell, P. J.; Gottgens, B.; Van Deursen, J. M.; Cole, P. A.; Huntly, B. J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">279-289</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Growing evidence links abnormal epigenetic control to the development of hematol. malignancies.  Accordingly, inhibition of epigenetic regulators is emerging as a promising therapeutic strategy.  The acetylation status of lysine residues in histone tails is one of a no. of epigenetic post-translational modifications that alter DNA-templated processes, such as transcription, to facilitate malignant transformation.  Although histone deacetylases are already being clin. targeted, the role of histone lysine acetyltransferases (KAT) in malignancy is less well characterized.  We chose to study this question in the context of acute myeloid leukemia (AML), where, using in vitro and in vivo genetic ablation and knockdown expts. in murine models, we demonstrate a role for the epigenetic regulators CBP and p300 in the induction and maintenance of AML.  Furthermore, using selective small mol. inhibitors of their lysine acetyltransferase activity, we validate CBP/p300 as therapeutic targets in vitro across a wide range of human AML subtypes.  We proceed to show that growth retardation occurs through the induction of transcriptional changes that induce apoptosis and cell-cycle arrest in leukemia cells and finally demonstrate the efficacy of the KAT inhibitors in decreasing clonogenic growth of primary AML patient samples.  Taken together, these data suggest that CBP/p300 are promising therapeutic targets across multiple subtypes in AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb1PaE0j_PhLVg90H21EOLACvtfcHk0liSN65mf-xreA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFWhsLc%253D&md5=3d91ea01fb6110f1c56aac137b068c85</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.92&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.92%26sid%3Dliteratum%253Aachs%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DHorton%26aufirst%3DS.%2BJ.%26aulast%3DRuau%26aufirst%3DD.%26aulast%3DGallipoli%26aufirst%3DP.%26aulast%3DFowler%26aufirst%3DA.%26aulast%3DCrawley%26aufirst%3DC.%26aulast%3DPapaemmanuil%26aufirst%3DE.%26aulast%3DCampbell%26aufirst%3DP.%2BJ.%26aulast%3DG%25C3%25B6ttgens%26aufirst%3DB.%26aulast%3DVan%2BDeursen%26aufirst%3DJ.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26atitle%3DThe%2520epigenetic%2520regulators%2520CBP%2520and%2520p300%2520facilitate%2520leukemogenesis%2520and%2520represent%2520therapeutic%2520targets%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D279%26epage%3D89%26doi%3D10.1038%2Fonc.2015.92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span> <span> </span><span class="NLM_article-title">Open access chemical probes for epigenetic targets</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1901</span>– <span class="NLM_lpage">1917</span>, <span class="refDoi"> DOI: 10.4155/fmc.15.127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.4155%2Ffmc.15.127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26397018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=1901-1917&author=P.+J.+Brownauthor=S.+M%C3%BCller&title=Open+access+chemical+probes+for+epigenetic+targets&doi=10.4155%2Ffmc.15.127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Open access chemical probes for epigenetic targets</span></div><div class="casAuthors">Brown, Peter J.; Mueller, Susanne</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1901-1917</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Background: High attrition rates in drug discovery call for new approaches to improve target validation.  Academia is filling gaps, but often lacks the experience and resources of the pharmaceutical industry resulting in poorly characterized tool compds.  Discussion: The SGC has established an open access chem. probe consortium, currently encompassing ten pharmaceutical companies.  One of its mandates is to create well-characterized inhibitors (chem. probes) for epigenetic targets to enable new biol. and target validation for drug development.  Conclusion: Epigenetic probe compds. have proven to be very valuable and have not only spurred a plethora of novel biol. findings, but also provided starting points for clin. trials.  These probes have proven to be crit. complementation to traditional genetic targeting strategies and provided sometimes surprising results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX4t2GBZPCzLVg90H21EOLACvtfcHk0lj_vTVzGCG3jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KrsLrK&md5=09973fb3a77bee2a6a65e7fca190f2aa</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4155%2Ffmc.15.127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.15.127%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DOpen%2520access%2520chemical%2520probes%2520for%2520epigenetic%2520targets%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D1901%26epage%3D1917%26doi%3D10.4155%2Ffmc.15.127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortopassi, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pryde, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conway, S. J.</span></span> <span> </span><span class="NLM_article-title">A series of potent CREBBP bromodomain ligands reveals an induced-fit pocket stabilized by a cation-π interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6126</span>– <span class="NLM_lpage">6130</span>, <span class="refDoi"> DOI: 10.1002/anie.201402750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fanie.201402750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6126-6130&author=T.+P.+Rooneyauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=W.+A.+Cortopassiauthor=D.+A.+Hayauthor=S.+Martinauthor=A.+Tumberauthor=C.+M.+Rogersauthor=M.+Philpottauthor=M.+Wangauthor=A.+L.+Thompsonauthor=T.+D.+Heightmanauthor=D.+C.+Prydeauthor=A.+Cookauthor=R.+S.+Patonauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=P.+E.+Brennanauthor=S.+J.+Conway&title=A+series+of+potent+CREBBP+bromodomain+ligands+reveals+an+induced-fit+pocket+stabilized+by+a+cation-%CF%80+interaction&doi=10.1002%2Fanie.201402750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A Series of Potent CREBBP Bromodomain Ligands Reveals an Induced-Fit Pocket Stabilized by a Cation-π Interaction</span></div><div class="casAuthors">Rooney, Timothy P. C.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Cortopassi, Wilian A.; Hay, Duncan A.; Martin, Sarah; Tumber, Anthony; Rogers, Catherine M.; Philpott, Martin; Wang, Minghua; Thompson, Amber L.; Heightman, Tom D.; Pryde, David C.; Cook, Andrew; Paton, Robert S.; Mueller, Susanne; Knapp, Stefan; Brennan, Paul E.; Conway, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6126-6130</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Benzoxazinone and dihydroquinoxalinone fragments were employed as novel acetyl lysine mimics in the development of CREBBP bromodomain ligands.  While the benzoxazinone series showed low affinity for the CREBBP bromodomain, expansion of the dihydroquinoxalinone series resulted in the first potent inhibitors, e.g., I, of a bromodomain outside the BET family.  Structural and computational studies revealed that an internal hydrogen bond stabilized the protein-bound conformation of the dihydroquinoxalinone series.  The side chain of this series binds in an induced-fit pocket forming a cation-π interaction with R1173 of CREBBP.  Compd. I, the most potent compd., inhibited binding of CREBBP to chromatin in U2OS cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHFgTVUKe7lrVg90H21EOLACvtfcHk0lj_vTVzGCG3jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvFWgsb0%253D&md5=42b12e1fe855d2af02c49f8e1031d39e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.201402750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201402750%26sid%3Dliteratum%253Aachs%26aulast%3DRooney%26aufirst%3DT.%2BP.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DCortopassi%26aufirst%3DW.%2BA.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRogers%26aufirst%3DC.%2BM.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DThompson%26aufirst%3DA.%2BL.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DPryde%26aufirst%3DD.%2BC.%26aulast%3DCook%26aufirst%3DA.%26aulast%3DPaton%26aufirst%3DR.%2BS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DA%2520series%2520of%2520potent%2520CREBBP%2520bromodomain%2520ligands%2520reveals%2520an%2520induced-fit%2520pocket%2520stabilized%2520by%2520a%2520cation-%25CF%2580%2520interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D6126%26epage%3D6130%26doi%3D10.1002%2Fanie.201402750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chekler, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanz, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langille, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stock, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahasrabudhe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pletcher, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span> <span> </span><span class="NLM_article-title">Transcriptional profiling of a selective cREB Binding protein bromodomain inhibitor highlights therapeutic opportunities</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1588</span>– <span class="NLM_lpage">1596</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.10.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.chembiol.2015.10.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26670081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFKis7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=1588-1596&author=E.+L.+Cheklerauthor=J.+A.+Pellegrinoauthor=T.+A.+Lanzauthor=R.+A.+Dennyauthor=A.+C.+Flickauthor=J.+Coeauthor=J.+Langilleauthor=A.+Basakauthor=S.+Liuauthor=I.+A.+Stockauthor=P.+Sahasrabudheauthor=P.+D.+Boninauthor=K.+Leeauthor=M.+T.+Pletcherauthor=L.+H.+Jones&title=Transcriptional+profiling+of+a+selective+cREB+Binding+protein+bromodomain+inhibitor+highlights+therapeutic+opportunities&doi=10.1016%2Fj.chembiol.2015.10.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional Profiling of a Selective CREB Binding Protein Bromodomain Inhibitor Highlights Therapeutic Opportunities</span></div><div class="casAuthors">Chekler, Eugene L. Piatnitski; Pellegrino, Jessica A.; Lanz, Thomas A.; Denny, R. Aldrin; Flick, Andrew C.; Coe, Jotham; Langille, Jonathan; Basak, Arindrajit; Liu, Shenping; Stock, Ingrid A.; Sahasrabudhe, Parag; Bonin, Paul D.; Lee, Kevin; Pletcher, Mathew T.; Jones, Lyn H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1588-1596</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Bromodomains are involved in transcriptional regulation through the recognition of acetyl lysine modifications on diverse proteins.  Selective pharmacol. modulators of bromodomains are lacking, although the largely hydrophobic nature of the pocket makes these modules attractive targets for small-mol. inhibitors.  This work describes the structure-based design of a highly selective inhibitor of the CREB binding protein (CBP) bromodomain and its use in cell-based transcriptional profiling expts.  The inhibitor downregulated a no. of inflammatory genes in macrophages that were not affected by a selective BET bromodomain inhibitor.  In addn., the CBP bromodomain inhibitor modulated the mRNA level of the regulator of G-protein signaling 4 (RGS4) gene in neurons, suggesting a potential therapeutic opportunity for CBP inhibitors in the treatment of neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLDeCJH-gX5LVg90H21EOLACvtfcHk0lhyQJulyNE-cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKis7rE&md5=dc223d58c99b1994f0cdb15b734a5b21</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.10.013%26sid%3Dliteratum%253Aachs%26aulast%3DChekler%26aufirst%3DE.%2BL.%26aulast%3DPellegrino%26aufirst%3DJ.%2BA.%26aulast%3DLanz%26aufirst%3DT.%2BA.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DFlick%26aufirst%3DA.%2BC.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DLangille%26aufirst%3DJ.%26aulast%3DBasak%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DStock%26aufirst%3DI.%2BA.%26aulast%3DSahasrabudhe%26aufirst%3DP.%26aulast%3DBonin%26aufirst%3DP.%2BD.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPletcher%26aufirst%3DM.%2BT.%26aulast%3DJones%26aufirst%3DL.%2BH.%26atitle%3DTranscriptional%2520profiling%2520of%2520a%2520selective%2520cREB%2520Binding%2520protein%2520bromodomain%2520inhibitor%2520highlights%2520therapeutic%2520opportunities%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D1588%26epage%3D1596%26doi%3D10.1016%2Fj.chembiol.2015.10.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Popp, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bracher, F.</span></span> <span> </span><span class="NLM_article-title">Development of selective CBP/P300 benzoxazepine bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8889</span>– <span class="NLM_lpage">8912</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00774</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00774" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8889-8912&author=T.+A.+Poppauthor=C.+Tallantauthor=C.+Rogersauthor=O.+Fedorovauthor=P.+E.+Brennanauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=F.+Bracher&title=Development+of+selective+CBP%2FP300+benzoxazepine+bromodomain+inhibitors&doi=10.1021%2Facs.jmedchem.6b00774"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Selective CBP/P300 Benzoxazepine Bromodomain Inhibitors</span></div><div class="casAuthors">Popp, Tobias A.; Tallant, Cynthia; Rogers, Catherine; Fedorov, Oleg; Brennan, Paul E.; Mueller, Susanne; Knapp, Stefan; Bracher, Franz</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8889-8912</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">CBP (CREB (cAMP responsive element binding protein) binding protein (CREBBP)) and P300 (adenovirus E1A-assocd. 300 kDa protein) are two closely related histone acetyltransferases (HATs) that play a key role in the regulation of gene transcription.  Both proteins contain a bromodomain flanking the HAT catalytic domain that is important for the targeting of CBP/P300 to chromatin and which offers an opportunity for the development of protein-protein interaction inhibitors.  Here the authors present the development of CBP/P300 bromodomain inhibitors with 2,3,4,5-tetrahydro-1,4-benzoxazepine backbone, an N-acetyl-lysine mimetic scaffold that led to the recent development of the chem. probe I-CBP112.  The authors present comprehensive SAR of this inhibitor class as well as demonstration of cellular on target activity of the most potent and selective inhibitor I, which showed 134 nM affinity for CBP with excellent selectivity over other bromodomains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwAZFuNP0NkLVg90H21EOLACvtfcHk0lhMgk78w2dcrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKis77E&md5=6bd914d24abf00a8f074bf1918fb4ee1</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00774%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DT.%2BA.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DRogers%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBracher%26aufirst%3DF.%26atitle%3DDevelopment%2520of%2520selective%2520CBP%252FP300%2520benzoxazepine%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8889%26epage%3D8912%26doi%3D10.1021%2Facs.jmedchem.6b00774" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Setser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon
Boenig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, O. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span> <span> </span><span class="NLM_article-title">Fragment-based discovery of a selective and cell-active benzodiazepinone CBP/EP300 bromodomain Inhibitor (CPI-637)</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">536</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00075</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00075" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=531-536&author=A.+M.+Taylorauthor=A.+C%C3%B4t%C3%A9author=M.+C.+Hewittauthor=R.+Pastorauthor=Y.+Leblancauthor=C.+G.+Nasveschukauthor=F.+A.+Romeroauthor=T.+D.+Crawfordauthor=N.+Cantoneauthor=H.+Jayaramauthor=J.+Setserauthor=J.+Murrayauthor=M.+H.+Beresiniauthor=G.+de+Leon%0ABoenigauthor=Z.+Chenauthor=A.+R.+Coneryauthor=R.+T.+Cummingsauthor=L.+A.+Dakinauthor=E.+M.+Flynnauthor=O.+W.+Huangauthor=S.+Kaufmanauthor=P.+J.+Kellerauthor=J.+R.+Kieferauthor=T.+Laiauthor=Y.+Liauthor=J.+Liaoauthor=W.+Liuauthor=H.+Luauthor=E.+Pardoauthor=V.+Tsuiauthor=J.+Wangauthor=Y.+Wangauthor=Z.+Xuauthor=F.+Yanauthor=D.+Yuauthor=L.+Zawadzkeauthor=X.+Zhuauthor=X.+Zhuauthor=R.+J.+Simsauthor=A.+G.+Cochranauthor=S.+Bellonauthor=J.+E.+Audiaauthor=S.+Magnusonauthor=B.+K.+Albrecht&title=Fragment-based+discovery+of+a+selective+and+cell-active+benzodiazepinone+CBP%2FEP300+bromodomain+Inhibitor+%28CPI-637%29&doi=10.1021%2Facsmedchemlett.6b00075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Discovery of a Selective and Cell-Active Benzodiazepinone CBP/EP300 Bromodomain Inhibitor (CPI-637)</span></div><div class="casAuthors">Taylor, Alexander M.; Cote, Alexandre; Hewitt, Michael C.; Pastor, Richard; Leblanc, Yves; Nasveschuk, Christopher G.; Romero, F. Anthony; Crawford, Terry D.; Cantone, Nico; Jayaram, Hariharan; Setser, Jeremy; Murray, Jeremy; Beresini, Maureen H.; de Leon Boenig, Gladys; Chen, Zhongguo; Conery, Andrew R.; Cummings, Richard T.; Dakin, Leslie A.; Flynn, E. Megan; Huang, Oscar W.; Kaufman, Susan; Keller, Patricia J.; Kiefer, James R.; Lai, Tommy; Li, Yingjie; Liao, Jiangpeng; Liu, Wenfeng; Lu, Henry; Pardo, Eneida; Tsui, Vickie; Wang, Jian; Wang, Yongyun; Xu, Zhaowu; Yan, Fen; Yu, Dong; Zawadzke, Laura; Zhu, Xiaoqin; Zhu, Xiaoyu; Sims, Robert J.; Cochran, Andrea G.; Bellon, Steve; Audia, James E.; Magnuson, Steven; Albrecht, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">531-536</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Benzo-fused heterocycles, particularly benzodiazepinones such as I, were prepd. as inhibitors of the bromodomain-contg. proteins CBP (CREB, cAMP response element binding protein) and EP300 (adenoviral E1A-binding protein of 300 kDa) for potential use as antitumor agents; the core benzodiazepinone moiety was identified using fragment-based screening.  The structures of four benzodiazepinones, including I, bound to the bromodomain fragment of human CREB were detd. by X-ray crystallog.  The free energy, enthalpy, and entropy of the binding of I to CREB and its selectivity for CREB and EP300 over other bromodomain-contg. proteins was detd.; I inhibited expression of the transcription factor MYC in human plasmacytoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNm7Blycl0-7Vg90H21EOLACvtfcHk0lgzZGgfSBoanQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1Cms7c%253D&md5=8440a9bb3460ff66d43f15041dbbc19d</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00075%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DSetser%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DHuang%26aufirst%3DO.%2BW.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DMagnuson%26aufirst%3DS.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26atitle%3DFragment-based%2520discovery%2520of%2520a%2520selective%2520and%2520cell-active%2520benzodiazepinone%2520CBP%252FEP300%2520bromodomain%2520Inhibitor%2520%2528CPI-637%2529%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D531%26epage%3D536%26doi%3D10.1021%2Facsmedchemlett.6b00075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon Boenig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noland, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharbanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillane, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and selective in vivo probe (GNE-272) for the bromodomains of CBP/EP300</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">10549</span>– <span class="NLM_lpage">10563</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=10549-10563&author=T.+D.+Crawfordauthor=F.+A.+Romeroauthor=K.+W.+Laiauthor=V.+Tsuiauthor=A.+M.+Taylorauthor=G.+de+Leon+Boenigauthor=C.+L.+Nolandauthor=J.+Murrayauthor=J.+Lyauthor=E.+F.+Chooauthor=T.+L.+Hunsakerauthor=E.+W.+Chanauthor=M.+Merchantauthor=S.+Kharbandaauthor=K.+E.+Gascoigneauthor=S.+Kaufmanauthor=M.+H.+Beresiniauthor=J.+Liaoauthor=W.+Liuauthor=K.+X.+Chenauthor=Z.+Chenauthor=A.+R.+Coneryauthor=A.+C%C3%B4t%C3%A9author=H.+Jayaramauthor=Y.+Jiangauthor=J.+R.+Kieferauthor=T.+Kleinheinzauthor=Y.+Liauthor=J.+Maherauthor=E.+Pardoauthor=F.+Poyauthor=K.+L.+Spillaneauthor=F.+Wangauthor=J.+Wangauthor=X.+Weiauthor=Z.+Xuauthor=Z.+Xuauthor=I.+Yenauthor=L.+Zawadzkeauthor=X.+Zhuauthor=S.+Bellonauthor=R.+Cummingsauthor=A.+G.+Cochranauthor=B.+K.+Albrechtauthor=S.+Magnuson&title=Discovery+of+a+potent+and+selective+in+vivo+probe+%28GNE-272%29+for+the+bromodomains+of+CBP%2FEP300&doi=10.1021%2Facs.jmedchem.6b01022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective in Vivo Probe (GNE-272) for the Bromodomains of CBP/EP300</span></div><div class="casAuthors">Crawford, Terry D.; Romero, F. Anthony; Lai, Kwong Wah; Tsui, Vickie; Taylor, Alexander M.; de Leon Boenig, Gladys; Noland, Cameron L.; Murray, Jeremy; Ly, Justin; Choo, Edna F.; Hunsaker, Thomas L.; Chan, Emily W.; Merchant, Mark; Kharbanda, Samir; Gascoigne, Karen E.; Kaufman, Susan; Beresini, Maureen H.; Liao, Jiangpeng; Liu, Wenfeng; Chen, Kevin X.; Chen, Zhongguo; Conery, Andrew R.; Cote, Alexandre; Jayaram, Hariharan; Jiang, Ying; Kiefer, James R.; Kleinheinz, Tracy; Li, Yingjie; Maher, Jonathan; Pardo, Eneida; Poy, Florence; Spillane, Kerry L.; Wang, Fei; Wang, Jian; Wei, Xiaocang; Xu, Zhaowu; Xu, Zhongya; Yen, Ivana; Zawadzke, Laura; Zhu, Xiaoyu; Bellon, Steven; Cummings, Richard; Cochran, Andrea G.; Albrecht, Brian K.; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10549-10563</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The single bromodomain of the closely related transcriptional regulators CBP/EP300 is a target of much recent interest in cancer and immune system regulation.  A co-crystal structure of a ligand-efficient screening hit and the CBP bromodomain guided initial design targeting the LPF shelf, ZA loop, and acetylated lysine binding regions.  Structure-activity relationship studies allowed the authors to identify a more potent analog.  Optimization of permeability and microsomal stability and subsequent improvement of mouse hepatocyte stability afforded 59 (GNE-272, TR-FRET IC50 = 0.02 μM, BRET IC50 = 0.41 μM, BRD4(1) IC50 = 13 μM) that retained the best balance of cell potency, selectivity, and in vivo PK.  GNE-272 showed a marked antiproliferative effect in hematol. cancer cell lines and modulates MYC expression in vivo that corresponds with antitumor activity in an AML tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6T5ODn2Q-2LVg90H21EOLACvtfcHk0lgzZGgfSBoanQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGhsb%252FO&md5=22ed46cb031022aa6aff23c20d07e7dd</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01022%26sid%3Dliteratum%253Aachs%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DNoland%26aufirst%3DC.%2BL.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DYen%26aufirst%3DI.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520in%2520vivo%2520probe%2520%2528GNE-272%2529%2520for%2520the%2520bromodomains%2520of%2520CBP%252FEP300%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D10549%26epage%3D10563%26doi%3D10.1021%2Facs.jmedchem.6b01022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romero, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leon Boenig, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronner, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clagg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida
Nagata, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grogan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatzivassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunsaker, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnuson, S.</span></span> <span> </span><span class="NLM_article-title">GNE-781, a highly advanced potent and selective bromodomain inhibitor of cyclic adenosine monophosphate response element binding protein, binding protein (CBP)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9162</span>– <span class="NLM_lpage">9183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgs7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9162-9183&author=F.+A.+Romeroauthor=J.+Murrayauthor=K.+W.+Laiauthor=V.+Tsuiauthor=B.+K.+Albrechtauthor=L.+Anauthor=M.+H.+Beresiniauthor=G.+de+Leon+Boenigauthor=S.+M.+Bronnerauthor=E.+W.+Chanauthor=K.+X.+Chenauthor=Z.+Chenauthor=E.+F.+Chooauthor=K.+Claggauthor=K.+Clarkauthor=T.+D.+Crawfordauthor=P.+Cyrauthor=D.+de+Almeida%0ANagataauthor=K.+E.+Gascoigneauthor=J.+L.+Groganauthor=G.+Hatzivassiliouauthor=W.+Huangauthor=T.+L.+Hunsakerauthor=S.+Kaufmanauthor=S.+G.+Koenigauthor=R.+Liauthor=Y.+Liauthor=X.+Liangauthor=J.+Liaoauthor=W.+Liuauthor=J.+Lyauthor=J.+Maherauthor=C.+Masuiauthor=M.+Merchantauthor=Y.+Ranauthor=A.+M.+Taylorauthor=J.+Waiauthor=F.+Wangauthor=X.+Weiauthor=D.+Yuauthor=B.+Y.+Zhuauthor=X.+Zhuauthor=S.+Magnuson&title=GNE-781%2C+a+highly+advanced+potent+and+selective+bromodomain+inhibitor+of+cyclic+adenosine+monophosphate+response+element+binding+protein%2C+binding+protein+%28CBP%29&doi=10.1021%2Facs.jmedchem.7b00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GNE-781, A Highly Advanced Potent and Selective Bromodomain Inhibitor of Cyclic Adenosine Monophosphate Response Element Binding Protein, Binding Protein (CBP)</span></div><div class="casAuthors">Romero, F. Anthony; Murray, Jeremy; Lai, Kwong Wah; Tsui, Vickie; Albrecht, Brian K.; An, Le; Beresini, Maureen H.; de Leon Boenig, Gladys; Bronner, Sarah M.; Chan, Emily W.; Chen, Kevin X.; Chen, Zhongguo; Choo, Edna F.; Clagg, Kyle; Clark, Kevin; Crawford, Terry D.; Cyr, Patrick; de Almeida Nagata, Denise; Gascoigne, Karen E.; Grogan, Jane L.; Hatzivassiliou, Georgia; Huang, Wei; Hunsaker, Thomas L.; Kaufman, Susan; Koenig, Stefan G.; Li, Ruina; Li, Yingjie; Liang, Xiaorong; Liao, Jiangpeng; Liu, Wenfeng; Ly, Justin; Maher, Jonathan; Masui, Colin; Merchant, Mark; Ran, Yingqing; Taylor, Alexander M.; Wai, John; Wang, Fei; Wei, Xiaocang; Yu, Dong; Zhu, Bing-Yan; Zhu, Xiaoyu; Magnuson, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9162-9183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of the bromodomain of the transcriptional regulator CBP/P300 is an esp. interesting new therapeutic approach in oncol.  We recently disclosed in vivo chem. tool 1 (GNE-272) for the bromodomain of CBP that was moderately potent and selective over BRD4(1).  In pursuit of a more potent and selective CBP inhibitor, we used structure-based design.  Constraining the aniline of 1 into a tetrahydroquinoline motif maintained potency and increased selectivity 2-fold.  Structure-activity relationship studies coupled with further structure-based design targeting the LPF shelf, BC loop, and KAc regions allowed us to significantly increase potency and selectivity, resulting in the identification of non-CNS penetrant 19 (GNE-781, TR-FRET IC50 = 0.94 nM, BRET IC50 = 6.2 nM; BRD4(1) IC50 = 5100 nΜ) that maintained good in vivo PK properties in multiple species.  Compd. 19 displays antitumor activity in an AML tumor model and was also shown to decrease Foxp3 transcript levels in a dose dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYowYgusjfzrVg90H21EOLACvtfcHk0li9NxKtqo1DdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgs7zI&md5=16403939224df3949cd3e3ffd8cf1fd1</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00796%26sid%3Dliteratum%253Aachs%26aulast%3DRomero%26aufirst%3DF.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DK.%2BW.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3Dde%2BLeon%2BBoenig%26aufirst%3DG.%26aulast%3DBronner%26aufirst%3DS.%2BM.%26aulast%3DChan%26aufirst%3DE.%2BW.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DClagg%26aufirst%3DK.%26aulast%3DClark%26aufirst%3DK.%26aulast%3DCrawford%26aufirst%3DT.%2BD.%26aulast%3DCyr%26aufirst%3DP.%26aulast%3Dde%2BAlmeida%2BNagata%26aufirst%3DD.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DGrogan%26aufirst%3DJ.%2BL.%26aulast%3DHatzivassiliou%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DHunsaker%26aufirst%3DT.%2BL.%26aulast%3DKaufman%26aufirst%3DS.%26aulast%3DKoenig%26aufirst%3DS.%2BG.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DLy%26aufirst%3DJ.%26aulast%3DMaher%26aufirst%3DJ.%26aulast%3DMasui%26aufirst%3DC.%26aulast%3DMerchant%26aufirst%3DM.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DB.%2BY.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMagnuson%26aufirst%3DS.%26atitle%3DGNE-781%252C%2520a%2520highly%2520advanced%2520potent%2520and%2520selective%2520bromodomain%2520inhibitor%2520of%2520cyclic%2520adenosine%2520monophosphate%2520response%2520element%2520binding%2520protein%252C%2520binding%2520protein%2520%2528CBP%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9162%26epage%3D9183%26doi%3D10.1021%2Facs.jmedchem.7b00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marineau, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Biased multicomponent reactions to develop novel bromodomain inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9019</span>– <span class="NLM_lpage">9127</span>, <span class="refDoi"> DOI: 10.1021/jm501120z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501120z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9019-9127&author=M.+R.+McKeownauthor=D.+L.+Shawauthor=H.+Fuauthor=S.+Liuauthor=X.+Xuauthor=J.+J.+Marineauauthor=Y.+Huangauthor=X.+Zhangauthor=D.+L.+Buckleyauthor=A.+Kadamauthor=Z.+Zhangauthor=S.+C.+Blacklowauthor=J.+Qiauthor=W.+Zhangauthor=J.+E.+Bradner&title=Biased+multicomponent+reactions+to+develop+novel+bromodomain+inhibitors&doi=10.1021%2Fjm501120z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Biased Multicomponent Reactions to Develop Novel Bromodomain Inhibitors</span></div><div class="casAuthors">McKeown, Michael R.; Shaw, Daniel L.; Fu, Harry; Liu, Shuai; Xu, Xiang; Marineau, Jason J.; Huang, Yibo; Zhang, Xiaofeng; Buckley, Dennis L.; Kadam, Asha; Zhang, Zijuan; Blacklow, Stephen C.; Qi, Jun; Zhang, Wei; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9019-9027</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BET bromodomain inhibition has contributed new insights into gene regulation and emerged as a promising therapeutic strategy in cancer.  Structural analogy of early methyl-triazolo BET inhibitors has prompted a need for structurally dissimilar ligands as probes of bromodomain function.  Using fluorous-tagged multicomponent reactions, we developed a focused chem. library of bromodomain inhibitors around a 3,5-dimethylisoxazole biasing element with micromolar biochem. IC50.  Iterative synthesis and biochem. assessment allowed optimization of novel BET bromodomain inhibitors based on an imidazo[1,2-a]pyrazine scaffold.  Lead compd. I (UMB-32) binds BRD4 with a Kd of 550 nM and 724 nM cellular potency in BRD4-dependent lines.  Addnl., compd. I shows potency against TAF1, a bromodomain-contg. transcription factor previously unapproached by discovery chem.  Compd. I was cocrystd. with BRD4, yielding a 1.56 Å resoln. crystal structure.  This research showcases new applications of fluorous and multicomponent chem. synthesis for the development of novel epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWEf4LoDJTrVg90H21EOLACvtfcHk0li9NxKtqo1DdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslChsbbN&md5=a4e20101f5a0b547451814af73c0418f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm501120z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501120z%26sid%3Dliteratum%253Aachs%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DShaw%26aufirst%3DD.%2BL.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DKadam%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DBiased%2520multicomponent%2520reactions%2520to%2520develop%2520novel%2520bromodomain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9019%26epage%3D9127%26doi%3D10.1021%2Fjm501120z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groebke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehlin, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis of imidazo[1,2-a] annulated pyridines, pyrazines and pyrimidines by a novel three-component condensation</span>. <i>Synlett</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>1998</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1055/s-1998-1721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1055%2Fs-1998-1721" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1998&publication_year=1998&pages=661-663&author=K.+Groebkeauthor=L.+Weberauthor=F.+Mehlin&title=Synthesis+of+imidazo%5B1%2C2-a%5D+annulated+pyridines%2C+pyrazines+and+pyrimidines+by+a+novel+three-component+condensation&doi=10.1055%2Fs-1998-1721"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1055%2Fs-1998-1721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1998-1721%26sid%3Dliteratum%253Aachs%26aulast%3DGroebke%26aufirst%3DK.%26aulast%3DWeber%26aufirst%3DL.%26aulast%3DMehlin%26aufirst%3DF.%26atitle%3DSynthesis%2520of%2520imidazo%255B1%252C2-a%255D%2520annulated%2520pyridines%252C%2520pyrazines%2520and%2520pyrimidines%2520by%2520a%2520novel%2520three-component%2520condensation%26jtitle%3DSynlett%26date%3D1998%26volume%3D1998%26spage%3D661%26epage%3D663%26doi%3D10.1055%2Fs-1998-1721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleming, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiosaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S.</span></span> <span> </span><span class="NLM_article-title">Parallel synthesis of 3-aminoimidazo[1,2-a]pyridines and pyrazines by a new three-component condensation</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">3635</span>– <span class="NLM_lpage">3638</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(98)00653-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2FS0040-4039%2898%2900653-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADyaK1cXjtlWrtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1998&pages=3635-3638&author=C.+Blackburnauthor=B.+Guanauthor=P.+Flemingauthor=K.+Shiosakiauthor=S.+Tsai&title=Parallel+synthesis+of+3-aminoimidazo%5B1%2C2-a%5Dpyridines+and+pyrazines+by+a+new+three-component+condensation&doi=10.1016%2FS0040-4039%2898%2900653-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Parallel synthesis of 3-aminoimidazo[1,2-a]pyridines and pyrazines by a new three-component condensation</span></div><div class="casAuthors">Blackburn, Christopher; Guan, Bing; Fleming, Paul; Shiosaki, Kazumi; Tsai, Shirling</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3635-3638</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A three-component condensation reaction between 2-aminopyridine, an aldehyde and an isonitrile catalyzed by scandium triflate affords derivs. of 3-aminoimidazo[1,2-a]pyridine; aminopyrazine reacts similarly.  A library of heterocycles, prepd. in high yield by parallel synthesis and purifn. on an ion-exchange resin, was subjected to further reaction at the amino group.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrndWK-azaOEbVg90H21EOLACvtfcHk0lip52xek3dhPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtlWrtLs%253D&md5=dfdddf23bcb1051d2ff6375f9f2616f5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2898%2900653-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252898%252900653-4%26sid%3Dliteratum%253Aachs%26aulast%3DBlackburn%26aufirst%3DC.%26aulast%3DGuan%26aufirst%3DB.%26aulast%3DFleming%26aufirst%3DP.%26aulast%3DShiosaki%26aufirst%3DK.%26aulast%3DTsai%26aufirst%3DS.%26atitle%3DParallel%2520synthesis%2520of%25203-aminoimidazo%255B1%252C2-a%255Dpyridines%2520and%2520pyrazines%2520by%2520a%2520new%2520three-component%2520condensation%26jtitle%3DTetrahedron%2520Lett.%26date%3D1998%26volume%3D39%26spage%3D3635%26epage%3D3638%26doi%3D10.1016%2FS0040-4039%2898%2900653-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bienaymé, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouzid, K.</span></span> <span> </span><span class="NLM_article-title">A new heterocyclic multicomponent reaction for the combinatorial synthesis of fused 3-aminoimidazoles</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">2234</span>– <span class="NLM_lpage">2237</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19980904)37:16<2234::AID-ANIE2234>3.0.CO;2-R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADyaK1cXmt1Crt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=2234-2237&author=H.+Bienaym%C3%A9author=K.+Bouzid&title=A+new+heterocyclic+multicomponent+reaction+for+the+combinatorial+synthesis+of+fused+3-aminoimidazoles&doi=10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A new heterocyclic multicomponent reaction for the combinatorial synthesis of fused 3-aminoimidazoles</span></div><div class="casAuthors">Bienayme, Hugues; Bouzid, Kamel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2234-2237</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Reaction of heteroarom. amidines, aldehydes, and isonitriles in the presence of a catalytic amt. of protic acids gave fused 3-aminoimidazoles.  E.g., HClO4-catalyzed reaction of 2-aminopyrimidine, PhCHO, and Me3CNC gave 82% imidazopyrimidine I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1LwDCSCFs_rVg90H21EOLACvtfcHk0lip52xek3dhPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmt1Crt7Y%253D&md5=6c11f8f883d1699e9a3def969b5f3e04</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819980904%252937%253A16%253C2234%253A%253AAID-ANIE2234%253E3.0.CO%253B2-R%26sid%3Dliteratum%253Aachs%26aulast%3DBienaym%25C3%25A9%26aufirst%3DH.%26aulast%3DBouzid%26aufirst%3DK.%26atitle%3DA%2520new%2520heterocyclic%2520multicomponent%2520reaction%2520for%2520the%2520combinatorial%2520synthesis%2520of%2520fused%25203-aminoimidazoles%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1998%26volume%3D37%26spage%3D2234%26epage%3D2237%26doi%3D10.1002%2F%28SICI%291521-3773%2819980904%2937%3A16%3C2234%3A%3AAID-ANIE2234%3E3.0.CO%3B2-R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span> <span> </span><span class="NLM_article-title">Cross-coupling reactions of organoboranes: an easy way to construct C-C bonds (Nobel Lecture)</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">6722</span>– <span class="NLM_lpage">6737</span>, <span class="refDoi"> DOI: 10.1002/anie.201101379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fanie.201101379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3MXms1Skurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=6722-6737&author=A.+Suzuki&title=Cross-coupling+reactions+of+organoboranes%3A+an+easy+way+to+construct+C-C+bonds+%28Nobel+Lecture%29&doi=10.1002%2Fanie.201101379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Cross-Coupling Reactions Of Organoboranes: An Easy Way To Construct C-C Bonds (Nobel Lecture)</span></div><div class="casAuthors">Suzuki, Akira</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6722-6737</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The Nobel Prize in Chemie 2010 was awarded for research on palladium-catalyzed cross-coupling in org. synthesis.  Reported is the historical development and current status of the Suzuki coupling reaction.  A variety of organoboron compds. have been shown to be suitable coupling partners, including boronic acids, boronic esters, and sterically hindered boranes, allowing carbon-carbon bond formation of aryl, vinyl, alkyl, and alkynyl moieties.  The products obtained with this methodol. range from simple dienes, biaryls, and alkanes to complex natural products and pharmaceuticals.  The optimization of catalytic conditions for sterically hindered and unactivated substrates is demonstrated, as well as application to the Suzuki polycondensation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsO3da5SzNe7Vg90H21EOLACvtfcHk0liKEwZkwxcz3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXms1Skurw%253D&md5=f5a1adae37cc486e1484fede5db572c3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201101379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201101379%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DCross-coupling%2520reactions%2520of%2520organoboranes%253A%2520an%2520easy%2520way%2520to%2520construct%2520C-C%2520bonds%2520%2528Nobel%2520Lecture%2529%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26spage%3D6722%26epage%3D6737%26doi%3D10.1002%2Fanie.201101379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">A bead-based proximity assay for BRD4 ligand discovery</span>. <i>Curr. Protoc. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1002/9780470559277.ch150024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2F9780470559277.ch150024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26629616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVOlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=263-278&author=J.+M.+Robertsauthor=J.+E.+Bradner&title=A+bead-based+proximity+assay+for+BRD4+ligand+discovery&doi=10.1002%2F9780470559277.ch150024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Bead-Based Proximity Assay for BRD4 Ligand Discovery</span></div><div class="casAuthors">Roberts, Justin M.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protocols in Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">263-278</span>CODEN:
                <span class="NLM_cas:coden">CPCBD9</span>;
        ISSN:<span class="NLM_cas:issn">2160-4762</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Bromodomain-contg. proteins have emerged as desirable targets for anti-neoplastic and anti-inflammatory drug discovery.  Toward the development of selective inhibitors of the BET family of bromodomains, we optimized bead-based assays to detect interactions between bromodomains and poly-acetylated histone peptides.  Donor and acceptor beads bound to target and ligand are brought into proximity by this protein-protein interaction.  After laser illumination, singlet oxygen evolved from donor beads travels to the spatially close acceptor beads, resulting in chemiluminesence.  This AlphaScreen assay has proven amendable to high-throughput screening, secondary validation, and specificity profiling during lead discovery and optimization.  Here we report our protocol for assay development to measure inhibition of ligand binding to bromodomain-contg. protein 4 (BRD4).  We discuss the discovery of an appropriate probe, optimization of bead, probe, and protein concns., and the derivation of protein-probe inhibition curves.  Finally, we explore the implementation of this technol. for high-throughput screening of potential BRD4 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohs1bOu1z4tbVg90H21EOLACvtfcHk0liKEwZkwxcz3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVOlsbg%253D&md5=4d6961bcb98443349fcd54b7a23ac998</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2F9780470559277.ch150024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F9780470559277.ch150024%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DA%2520bead-based%2520proximity%2520assay%2520for%2520BRD4%2520ligand%2520discovery%26jtitle%3DCurr.%2520Protoc.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D7%26spage%3D263%26epage%3D278%26doi%3D10.1002%2F9780470559277.ch150024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamotte, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donche, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krysa, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleary, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Identification of a novel series of BET family bromodomain inhibitors: binding mode and profile of I-BET151 (GSK1210151A)</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2968</span>– <span class="NLM_lpage">2972</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.02.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.bmcl.2012.02.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=22437115" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2968-2972&author=J.+Sealauthor=Y.+Lamotteauthor=F.+Doncheauthor=A.+Bouillotauthor=O.+Mirguetauthor=F.+Gellibertauthor=E.+Nicodemeauthor=G.+Krysaauthor=J.+Kirilovskyauthor=S.+Beinkeauthor=S.+McClearyauthor=I.+Riojaauthor=P.+Bamboroughauthor=C.-W.+Chungauthor=L.+Gordonauthor=T.+Lewisauthor=A.+L.+Walkerauthor=L.+Cutlerauthor=D.+Lugoauthor=D.+M.+Wilsonauthor=J.+Witheringtonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Identification+of+a+novel+series+of+BET+family+bromodomain+inhibitors%3A+binding+mode+and+profile+of+I-BET151+%28GSK1210151A%29&doi=10.1016%2Fj.bmcl.2012.02.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel series of BET family bromodomain inhibitors: Binding mode and profile of I-BET151 (GSK1210151A)</span></div><div class="casAuthors">Seal, Jonathan; Lamotte, Yann; Donche, Frederic; Bouillot, Anne; Mirguet, Olivier; Gellibert, Francoise; Nicodeme, Edwige; Krysa, Gael; Kirilovsky, Jorge; Beinke, Soren; McCleary, Scott; Rioja, Inma; Bamborough, Paul; Chung, Chun-Wa; Gordon, Laurie; Lewis, Toni; Walker, Ann L.; Cutler, Leanne; Lugo, David; Wilson, David M.; Witherington, Jason; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2968-2972</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of quinoline isoxazole BET family bromodomain inhibitors are discussed.  Crystallog. is used to illustrate binding modes and rationalize their SAR.  One member, I-BET151 (GSK1210151A), shows good oral bioavailability in both the rat and minipig as well as demonstrating efficient suppression of bacterial induced inflammation and sepsis in a murine in vivo endotoxemia model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJKtxBCr-pRbVg90H21EOLACvtfcHk0liKEwZkwxcz3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktFOqsr8%253D&md5=a8037c89bcb93948d663208bd1ebfd55</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.02.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.02.041%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DLamotte%26aufirst%3DY.%26aulast%3DDonche%26aufirst%3DF.%26aulast%3DBouillot%26aufirst%3DA.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGellibert%26aufirst%3DF.%26aulast%3DNicodeme%26aufirst%3DE.%26aulast%3DKrysa%26aufirst%3DG.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DMcCleary%26aufirst%3DS.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.-W.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DLewis%26aufirst%3DT.%26aulast%3DWalker%26aufirst%3DA.%2BL.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DIdentification%2520of%2520a%2520novel%2520series%2520of%2520BET%2520family%2520bromodomain%2520inhibitors%253A%2520binding%2520mode%2520and%2520profile%2520of%2520I-BET151%2520%2528GSK1210151A%2529%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2968%26epage%3D2972%26doi%3D10.1016%2Fj.bmcl.2012.02.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaidos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gouvedenou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotolo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Mahdi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahemtulla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleijnen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karadimitris, A.</span></span> <span> </span><span class="NLM_article-title">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">697</span>– <span class="NLM_lpage">705</span>, <span class="refDoi"> DOI: 10.1182/blood-2013-01-478420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1182%2Fblood-2013-01-478420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=24335499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2014&pages=697-705&author=A.+Chaidosauthor=V.+Caputoauthor=K.+Gouvedenouauthor=B.+Liuauthor=I.+Marigoauthor=M.+S.+Chaudhryauthor=A.+Rotoloauthor=D.+F.+Toughauthor=N.+N.+Smithersauthor=A.+K.+Bassilauthor=T.+D.+Chapmanauthor=N.+R.+Harkerauthor=O.+Barbashauthor=P.+Tumminoauthor=N.+Al-Mahdiauthor=A.+C.+Haynesauthor=L.+Cutlerauthor=B.+Leauthor=A.+Rahemtullaauthor=I.+Robertsauthor=M.+Kleijnenauthor=J.+J.+Witheringtonauthor=N.+J.+Parrauthor=R.+K.+Prinjhaauthor=A.+Karadimitris&title=Potent+antimyeloma+activity+of+the+novel+bromodomain+inhibitors+I-BET151+and+I-BET762&doi=10.1182%2Fblood-2013-01-478420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Potent antimyeloma activity of the novel bromodomain inhibitors I-BET151 and I-BET762</span></div><div class="casAuthors">Chaidos, Aristeidis; Caputo, Valentina; Gouvedenou, Katerina; Liu, Binbin; Marigo, Ilaria; Chaudhry, Mohammed Suhail; Rotolo, Antonia; Tough, David F.; Smithers, Nicholas N.; Bassil, Anna K.; Chapman, Trevor D.; Harker, Nicola R.; Barbash, Olena; Tummino, Peter; Al-Mahdi, Niam; Haynes, Andrea C.; Cutler, Leanne; Le, Bao Chau; Rahemtulla, Amin; Roberts, Irene; Kleijnen, Maurits; Witherington, Jason J.; Parr, Nigel J.; Prinjha, Rab K.; Karadimitris, Anastasios</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">697-705</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) protein BRD2-4 inhibitors hold therapeutic promise in preclin. models of hematol. malignancies.  However, translation of these data to mols. suitable for clin. development has yet to be accomplished.  Herein we expand the mechanistic understanding of BET inhibitors in multiple myeloma by using the chem. probe mol. I-BET151.  I-BET151 induces apoptosis and exerts strong antiproliferative effect in vitro and in vivo.  This is assocd. with contrasting effects on oncogenic MYC and HEXIM1, an inhibitor of the transcriptional activator P-TEFb.  I-BET151 causes transcriptional repression of MYC and MYC-dependent programs by abrogating recruitment to the chromatin of the P-TEFb component CDK9 in a BRD2-4-dependent manner.  In contrast, transcriptional upregulation of HEXIM1 is BRD2-4 independent.  Finally, preclin. studies show that I-BET762 has a favorable pharmacol. profile as an oral agent and that it inhibits myeloma cell proliferation, resulting in survival advantage in a systemic myeloma xenograft model.  These data provide a strong rationale for extending the clin. testing of the novel antimyeloma agent I-BET762 and reveal insights into biol. pathways required for myeloma cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0bNtpJQBXsrVg90H21EOLACvtfcHk0ljSABpyljin4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVGqtLY%253D&md5=f8f734f84a07d3b86ebdc264f6741b22</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-01-478420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-01-478420%26sid%3Dliteratum%253Aachs%26aulast%3DChaidos%26aufirst%3DA.%26aulast%3DCaputo%26aufirst%3DV.%26aulast%3DGouvedenou%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DMarigo%26aufirst%3DI.%26aulast%3DChaudhry%26aufirst%3DM.%2BS.%26aulast%3DRotolo%26aufirst%3DA.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DAl-Mahdi%26aufirst%3DN.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DLe%26aufirst%3DB.%26aulast%3DRahemtulla%26aufirst%3DA.%26aulast%3DRoberts%26aufirst%3DI.%26aulast%3DKleijnen%26aufirst%3DM.%26aulast%3DWitherington%26aufirst%3DJ.%2BJ.%26aulast%3DParr%26aufirst%3DN.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DKaradimitris%26aufirst%3DA.%26atitle%3DPotent%2520antimyeloma%2520activity%2520of%2520the%2520novel%2520bromodomain%2520inhibitors%2520I-BET151%2520and%2520I-BET762%26jtitle%3DBlood%26date%3D2014%26volume%3D123%26spage%3D697%26epage%3D705%26doi%3D10.1182%2Fblood-2013-01-478420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheryl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.-P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+M%C3%BCllerauthor=T.+Pawsonauthor=A.-C.+Gingrasauthor=+Arrowsmithauthor=H.+Cherylauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0ljSABpyljin4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.-P.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.-C.%26aulast%3DArrowsmith%26aulast%3DCheryl%26aufirst%3DH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, Z. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span> <span> </span><span class="NLM_article-title">Current development of CBP/p300 inhibitors in the last decade</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>209</i></span>, <span class="NLM_elocation-id">112861</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112861</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1016%2Fj.ejmech.2020.112861" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=33045661" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=209&publication_year=2020&author=Z.+X.+Heauthor=B.+F.+Weiauthor=X.+Zhangauthor=Y.+P.+Gongauthor=L.+Y.+Maauthor=W.+Zhao&title=Current+development+of+CBP%2Fp300+inhibitors+in+the+last+decade&doi=10.1016%2Fj.ejmech.2020.112861"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112861&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112861%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DZ.%2BX.%26aulast%3DWei%26aufirst%3DB.%2BF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGong%26aufirst%3DY.%2BP.%26aulast%3DMa%26aufirst%3DL.%2BY.%26aulast%3DZhao%26aufirst%3DW.%26atitle%3DCurrent%2520development%2520of%2520CBP%252Fp300%2520inhibitors%2520in%2520the%2520last%2520decade%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D209%26doi%3D10.1016%2Fj.ejmech.2020.112861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centore, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillane, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zawadzke, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bommi-Reddy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoigne, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span> <span> </span><span class="NLM_article-title">Bromodomain inhibition of the transcriptional coactivators CBP/EP300 as a therapeutic strategy to target the IRF4 network in multiple myeloma</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e10483</span> <span class="refDoi"> DOI: 10.7554/eLife.10483.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.7554%2FeLife.10483" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26731516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslahtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&author=A.+R.+Coneryauthor=R.+C.+Centoreauthor=A.+Neissauthor=P.+J.+Kellerauthor=S.+Joshiauthor=K.+L.+Spillaneauthor=P.+Sandyauthor=C.+Hattonauthor=E.+Pardoauthor=L.+Zawadzkeauthor=A.+Bommi-Reddyauthor=K.+E.+Gascoigneauthor=B.+M.+Bryantauthor=J.+A.+Mertzauthor=R.+J.+Sims&title=Bromodomain+inhibition+of+the+transcriptional+coactivators+CBP%2FEP300+as+a+therapeutic+strategy+to+target+the+IRF4+network+in+multiple+myeloma&doi=10.7554%2FeLife.10483.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain inhibition of the transcriptional coactivators CBP/EP300 as atherapeutic strategy to target the IRF4 network in multiple myeloma</span></div><div class="casAuthors">Conery, Andrew R.; Centore, Richard C.; Neiss, Adrianne; Keller, Patricia J.; Joshi, Shivangi; Spillane, Kerry L.; Sandy, Peter; Hatton, Charlie; Pardo, Eneida; Zawadzke, Laura; Bommi-Reddy, Archana; Gascoigne, Karen E.; Bryant, Barbara M.; Mertz, Jennifer A.; Sims, Robert J., III</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e10483/1-e10483/17</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Pharmacol. inhibition of chromatin co-regulatory factors represents a clin. validated strategy to modulate oncogenic signaling through selective attenuation of gene expression.  Here, we demonstrate that CBP/EP300 bromodomain inhibition preferentially abrogates the viability of multiple myeloma cell lines.  Selective targeting of multiple myeloma cell lines through CBP/EP300 bromodomain inhibition is the result of direct transcriptional suppression of the lymphocyte-specific transcription factor IRF4, which is essential for the viability of myeloma cells, and the concomitant repression of the IRF4 target gene c-MYC.  Ectopic expression of either IRF4 or MYC antagonizes the phenotypic and transcriptional effects of CBP/EP300 bromodomain inhibition, highlighting the IRF4/MYC axis as a key component of its mechanism of action.  These findings suggest that CBP/EP300 bromodomain inhibition represents a viable therapeutic strategy for targeting multiple myeloma and other lymphoid malignancies dependent on the IRF4 network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfAd8Yl2ENq7Vg90H21EOLACvtfcHk0lh0R8MthxGhwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslahtro%253D&md5=d0037d5abb6fbd85ce1ce2763c535370</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.7554%2FeLife.10483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.10483%26sid%3Dliteratum%253Aachs%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DCentore%26aufirst%3DR.%2BC.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DKeller%26aufirst%3DP.%2BJ.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DSpillane%26aufirst%3DK.%2BL.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DZawadzke%26aufirst%3DL.%26aulast%3DBommi-Reddy%26aufirst%3DA.%26aulast%3DGascoigne%26aufirst%3DK.%2BE.%26aulast%3DBryant%26aufirst%3DB.%2BM.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26atitle%3DBromodomain%2520inhibition%2520of%2520the%2520transcriptional%2520coactivators%2520CBP%252FEP300%2520as%2520a%2520therapeutic%2520strategy%2520to%2520target%2520the%2520IRF4%2520network%2520in%2520multiple%2520myeloma%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26doi%3D10.7554%2FeLife.10483.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1655</span>– <span class="NLM_lpage">1680</span>, <span class="refDoi"> DOI: 10.4155/fmc-2016-0059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.4155%2Ffmc-2016-0059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=27193077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=1655-1680&author=C.+Galdeanoauthor=A.+Ciulli&title=Selectivity+on-target+of+bromodomain+chemical+probes+by+structure-guided+medicinal+chemistry+and+chemical+biology&doi=10.4155%2Ffmc-2016-0059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity on-target of bromodomain chemical probes by structure-guided medicinal chemistry and chemical biology</span></div><div class="casAuthors">Galdeano, Carles; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1655-1680</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Targeting epigenetic proteins is a rapidly growing area for medicinal chem. and drug discovery.  Recent years have seen an explosion of interest in developing small mols. binding to bromodomains, the readers of acetyl-lysine modifications.  A plethora of co-crystal structures has motivated focused fragment-based design and optimization programs within both industry and academia.  These efforts have yielded several compds. entering the clinic, and many more are increasingly being used as chem. probes to interrogate bromodomain biol.  High selectivity of chem. probes is necessary to ensure biol. activity is due to an on-target effect.  Here, we review the state-of-the-art of bromodomain-targeting compds., focusing on the structural basis for their on-target selectivity or lack thereof.  We also highlight chem. biol. approaches to enhance on-target selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1bg8K3kh-RLVg90H21EOLACvtfcHk0lh0R8MthxGhwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVKhsr3K&md5=a4c02dd517e8582755a918eabd7bfeea</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0059%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelectivity%2520on-target%2520of%2520bromodomain%2520chemical%2520probes%2520by%2520structure-guided%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D1655%26epage%3D1680%26doi%3D10.4155%2Ffmc-2016-0059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Microwave-assisted synthesis of a 3-aminoimidazo[1,2-a]-pyridine/pyrazine library by fluorous multicomponent reactions and subsequent cross-coupling reactions</span>. <i>QSAR Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">835</span>, <span class="refDoi"> DOI: 10.1002/qsar.200420045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1002%2Fqsar.200420045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=18542716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktVamsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=827-835&author=Y.+Luauthor=W.+Zhang&title=Microwave-assisted+synthesis+of+a+3-aminoimidazo%5B1%2C2-a%5D-pyridine%2Fpyrazine+library+by+fluorous+multicomponent+reactions+and+subsequent+cross-coupling+reactions&doi=10.1002%2Fqsar.200420045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Microwave-assisted synthesis of a 3-aminoimidazo[1,2-a]-pyridine/pyrazine library by fluorous multicomponent reactions and subsequent cross-coupling reactions</span></div><div class="casAuthors">Lu, Yimin; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">QSAR & Combinatorial Science</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">827-835</span>CODEN:
                <span class="NLM_cas:coden">QCSSAU</span>;
        ISSN:<span class="NLM_cas:issn">1611-020X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Microwave and fluorous technologies are employed to increase reaction rate and reduce sepn. time in soln.-phase parallel synthesis of a substituted 3-aminoimidazo[1,2-a]pyridine and 3-aminoimidazo[1,2-a]pyrazine library.  Multicomponent reactions of perfluorooctanesulfonyl-tagged benzaldehydes with 2-aminopyridines (or 2-aminopyrazines) and isocyanides followed by Pd-catalyzed cross-coupling reactions with arylboronic acids or thiols gave polysubstituted 3-aminoimidazo[1,2-a]pyridines and 3-aminoimidazo[1,2-a]pyrazines I (X = N, CH; R1 = H, MeO; R2 = H, Cl, Me; R3 = n-Bu, cyclohexyl, PhCH2; R4 = 3-ClC6H4, 4-MeCOC6H4, 4-MeOC6H4, cyclohexylthio, 4-MeOC6H4S).  Products were purified by fluorous solid-phase extn. (F-SPE) or crystn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmw3GZnlqnSbVg90H21EOLACvtfcHk0lh0R8MthxGhwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXktVamsQ%253D%253D&md5=c5d67e63b511ebc4c2c2af7c68cf5ebf</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fqsar.200420045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fqsar.200420045%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DMicrowave-assisted%2520synthesis%2520of%2520a%25203-aminoimidazo%255B1%252C2-a%255D-pyridine%252Fpyrazine%2520library%2520by%2520fluorous%2520multicomponent%2520reactions%2520and%2520subsequent%2520cross-coupling%2520reactions%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2004%26volume%3D23%26spage%3D827%26epage%3D835%26doi%3D10.1002%2Fqsar.200420045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilson, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baitaluk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicola, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, J.</span></span> <span> </span><span class="NLM_article-title">BindingDB in 2015: A public database for medicinal chemistry, computational chemistry and systems pharmacology</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D1045</span>– <span class="NLM_lpage">1053</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv1072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1093%2Fnar%2Fgkv1072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=26481362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D1045-1053&author=M.+K.+Gilsonauthor=T.+Liuauthor=M.+Baitalukauthor=G.+Nicolaauthor=L.+Hwangauthor=J.+Chong&title=BindingDB+in+2015%3A+A+public+database+for+medicinal+chemistry%2C+computational+chemistry+and+systems+pharmacology&doi=10.1093%2Fnar%2Fgkv1072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">BindingDB in 2015: A public database for medicinal chemistry, computational chemistry and systems pharmacology</span></div><div class="casAuthors">Gilson, Michael K.; Liu, Tiqing; Baitaluk, Michael; Nicola, George; Hwang, Linda; Chong, Jenny</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1045-D1053</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">BindingDB, www.bindingdb.org, is a publicly accessible database of exptl. protein-small mol. interaction data.  Its collection of over a million data entries derives primarily from scientific articles and, increasingly, US patents.  BindingDB provides many ways to browse and search for data of interest, including an advanced search tool, which can cross searches ofmultiple query types, including text, chem. structure, protein sequence and numerical affinities.  The PDB and PubMed provide links to data in BindingDB, and vice versa; and BindingDB provides links to pathway information, the ZINC catalog of available compds., and other resources.  The BindingDB website offers specialized tools that take advantage of its large data collection, including ones to generate hypotheses for the protein targets bound by a bioactive compd., and for the compds. bound by a new protein of known sequence; and virtual compd. screening by maximal chem. similarity, binary kernel discrimination, and support vector machine methods.  Specialized data sets are also available, such as binding data for hundreds of congeneric series of ligands, drawn from BindingDB and organized for use in validating drug design methods.  BindingDB offers several forms of programmatic access, and comes with extensive background material and documentation.  Here, we provide the first update of BindingDB since 2007, focusing on new and unique features and highlighting directions of importance to the field as a whole.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOJGtudYTYv7Vg90H21EOLACvtfcHk0lixUulnTUjTxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7fM&md5=3d243b2579640559825285bdcbbacbfd</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1072%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DBaitaluk%26aufirst%3DM.%26aulast%3DNicola%26aufirst%3DG.%26aulast%3DHwang%26aufirst%3DL.%26aulast%3DChong%26aufirst%3DJ.%26atitle%3DBindingDB%2520in%25202015%253A%2520A%2520public%2520database%2520for%2520medicinal%2520chemistry%252C%2520computational%2520chemistry%2520and%2520systems%2520pharmacology%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD1045%26epage%3D1053%26doi%3D10.1093%2Fnar%2Fgkv1072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lixUulnTUjTxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">phenix.model_vs_data: a high-level tool for the calculation of crystallographic model and data statistics</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1107/S0021889810015608</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1107%2FS0021889810015608" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=20648263" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=669-676&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=V.+B.+Chenauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=phenix.model_vs_data%3A+a+high-level+tool+for+the+calculation+of+crystallographic+model+and+data+statistics&doi=10.1107%2FS0021889810015608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">phenix.model vs. data: a high-level tool for the calculation of crystallographic model and data statistics</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Chen, Vincent B.; Headd, Jeffrey J.; Moriarty, Nigel W.; Richardson, Jane S.; Richardson, David C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">669-676</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.modelvsdata is a high-level command-line tool for the computation of crystallog. model and data statistics, and the evaluation of the fit of the model to data.  Anal. of all Protein Data Bank structures that have exptl. data available shows that in most cases the reported statistics, in particular R factors, can be reproduced within a few percentage points.  However, there are a no. of outliers where the recomputed R values are significantly different from those originally reported.  The reasons for these discrepancies are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBhMlnjsRfhLVg90H21EOLACvtfcHk0lixUulnTUjTxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVygtr4%253D&md5=5cd851a911ecd8b518e3ec4cdd5d9dd7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0021889810015608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889810015608%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Dphenix.model_vs_data%253A%2520a%2520high-level%2520tool%2520for%2520the%2520calculation%2520of%2520crystallographic%2520model%2520and%2520data%2520statistics%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D669%26epage%3D676%26doi%3D10.1107%2FS0021889810015608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liVT1X3IOHZyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4NR7','PDB','4NR7'); return false;">PDB: 4NR7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BT4','PDB','5BT4'); return false;">PDB: 5BT4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=BRD4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=BRD4','PDB','BRD4'); return false;">PDB: BRD4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KPY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KPY','PDB','7KPY'); return false;">PDB: 7KPY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i53"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_31007"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02232?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02232</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_001.csv">CSV</a>)</p></li><li><p class="inline">BromoSCAN, differential scanning fluorometry (DSF), and cellular activity data of compounds <b>7</b>, <b>16</b>, and <b>23</b>; crystallographic data collection and refinement statistics of compound <b>23</b>; <sup>1</sup>H NMR and <sup>13</sup>C NMR of representative compounds; and HPLC traces of compounds <b>7</b>, <b>16</b>, and <b>23</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Homology (docking) models of BRD4_5BT4_Cpd7 (UMB276) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of BRD4_5BT4_Cpd-16 (UMB295) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of BRD4_5BT4_Cpd-23 (UMB298) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-3 (UMB274) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_006.pdb">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-4 (UMB278) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_007.pdb">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-7 (UMB276) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_008.pdb">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-16 (UMB295) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_009.pdb">PDB</a>)</p></li><li><p class="inline">Homology (docking) models of CBP_4NR7_Cpd-23 (UMB298) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_010.pdb">PDB</a>)</p></li><li><p class="inline">Cocrystallographic data of compound <b>23</b> (UMB298) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_011.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_001.csv">jm0c02232_si_001.csv (2.77 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_002.pdf">jm0c02232_si_002.pdf (2.16 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_003.pdb">jm0c02232_si_003.pdb (181.1 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_004.pdb">jm0c02232_si_004.pdb (178.17 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_005.pdb">jm0c02232_si_005.pdb (178.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_006.pdb">jm0c02232_si_006.pdb (178.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_007.pdb">jm0c02232_si_007.pdb (194.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_008.pdb">jm0c02232_si_008.pdb (194.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_009.pdb">jm0c02232_si_009.pdb (194.37 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_010.pdb">jm0c02232_si_010.pdb (194.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02232/suppl_file/jm0c02232_si_011.pdb">jm0c02232_si_011.pdb (194.68 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The PDB accession code for the cocrystal structure of the CBP bromodomain liganded with compound <b>23</b> (UMB298) is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KPY">7KPY</a>. Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02232&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02232%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02232" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67998f5e9aa13cd6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
