
<html lang="en"     class="pb-page"  data-request-id="39d03735-de21-4587-97a8-d9e97fc0b41d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm100139b;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2010.53.issue-14;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Analysis of the Binding of Mixed Lineage Leukemia 1 (MLL1) and Histone 3 Peptides to WD Repeat Domain 5 (WDR5) for the Design of Inhibitors of the MLL1−WDR5 Interaction" /></meta><meta name="dc.Creator" content="Hacer  Karatas" /></meta><meta name="dc.Creator" content="Elizabeth C.  Townsend" /></meta><meta name="dc.Creator" content="Denzil  Bernard" /></meta><meta name="dc.Creator" content="Yali  Dou" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="MLL1 is a histone 3 lysine 4 (H3K4) methyltransferase and a promising new cancer therapeutic target. The catalytic activity of MLL1 is regulated by the formation of a core complex consisting of MLL..." /></meta><meta name="Description" content="MLL1 is a histone 3 lysine 4 (H3K4) methyltransferase and a promising new cancer therapeutic target. The catalytic activity of MLL1 is regulated by the formation of a core complex consisting of MLL..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 24, 2010" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm100139b" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2010 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm100139b" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm100139b" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm100139b" /></link>
        
    
    

<title>Analysis of the Binding of Mixed Lineage Leukemia 1 (MLL1) and Histone 3 Peptides to WD Repeat Domain 5 (WDR5) for the Design of Inhibitors of the MLL1−WDR5 Interaction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm100139b" /></meta><meta property="og:title" content="Analysis of the Binding of Mixed Lineage Leukemia 1 (MLL1) and Histone 3 Peptides to WD Repeat Domain 5 (WDR5) for the Design of Inhibitors of the MLL1−WDR5 Interaction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0005.jpeg" /></meta><meta property="og:description" content="MLL1 is a histone 3 lysine 4 (H3K4) methyltransferase and a promising new cancer therapeutic target. The catalytic activity of MLL1 is regulated by the formation of a core complex consisting of MLL1, WDR5, RbBP5, and Ash2L. The interaction between WDR5 and MLL1 plays an essential role in regulation of the H3K4 methyltransferase activity of MLL1 and targeting this interaction using small molecules may represent an attractive therapeutic strategy. In this study, we have defined the essential elements in MLL1 required for its high-affinity binding to WDR5. Our data showed that the minimal elements crucial for high-affinity binding of MLL1 to WDR5 are −CO-ARA-NH− motif and two intramolecular hydrogen bonds that stabilize the conformation of this motif. Two 3-mer peptides, Ac-ARA-NH2 and Ac-ART-NH2, were designed based upon MLL1 and H3 sequences and achieved Ki values of 120 and 20 nM to WDR5, respectively. Our study provides a concrete basis for the design of potent peptidomimetics and nonpeptidic compounds to inhibit MLL1 activity by targeting the MLL1 and WDR5 interaction." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm100139b"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm100139b">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm100139b&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm100139b&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm100139b&amp;href=/doi/10.1021/jm100139b" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2010</span><span class="cit-fg-volume">, 53</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 5179-5185</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/53/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm1001088" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm100350r" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Analysis of the Binding of Mixed Lineage Leukemia 1 (MLL1) and Histone 3 Peptides to WD Repeat Domain 5 (WDR5) for the Design of Inhibitors of the MLL1−WDR5 Interaction</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hacer++Karatas">Hacer Karatas</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth+C.++Townsend">Elizabeth C. Townsend</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Denzil++Bernard">Denzil Bernard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yali++Dou">Yali Dou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Medicinal Chemistry</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Internal Medicine</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Pathology</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Pharmacology</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Comprehensive Cancer Center</span></div><div class="aff-info" id="aff6"><span class="aff-text">University of Michigan, Ann Arbor, Michigan 48109</span></div><div class="corresp-info"><strong>*</strong>To whom correspondence should be addressed. Phone: 1-734-615-0362. Fax: 1-734-647-9647. E-mail: <a href="/cdn-cgi/l/email-protection#23504b424c4e464d4463564e4a404b0d464756"><span class="__cf_email__" data-cfemail="24574c454b49414a436451494d474c0a414051">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm100139b&amp;href=/doi/10.1021%2Fjm100139b" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2010</span></span><span class="cit-volume">, 53</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 5179–5185</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 24, 2010</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 February 2010</li><li><span class="item_label"><b>Published</b> online</span>24 June 2010</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 July 2010</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm100139b" title="DOI URL">https://doi.org/10.1021/jm100139b</a></div><div class="article_header-article-copyright"><strong>Copyright © 2010 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5179%26pageCount%3D7%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DHacer%2BKaratas%252C%2BElizabeth%2BC.%2BTownsend%252C%2BDenzil%2BBernard%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D53%26issueNum%3D14%26contentID%3Djm100139b%26title%3DAnalysis%2Bof%2Bthe%2BBinding%2Bof%2BMixed%2BLineage%2BLeukemia%2B1%2B%2528MLL1%2529%2Band%2BHistone%2B3%2BPeptides%2Bto%2BWD%2BRepeat%2BDomain%2B5%2B%2528WDR5%2529%2Bfor%2Bthe%2BDesign%2Bof%2BInhibitors%2Bof%2Bthe%2BMLL1%25E2%2588%2592WDR5%2BInteraction%26numPages%3D7%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5185%26publicationDate%3DJuly%2B2010">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm100139b"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2220</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">44</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm100139b" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Analysis of the Binding of Mixed Lineage Leukemia 1 (MLL1) and Histone 3 Peptides to WD Repeat Domain 5 (WDR5) for the Design of Inhibitors of the MLL1−WDR5 Interaction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Hacer&quot;,&quot;last_name&quot;:&quot;Karatas&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;C. Townsend&quot;},{&quot;first_name&quot;:&quot;Denzil&quot;,&quot;last_name&quot;:&quot;Bernard&quot;},{&quot;first_name&quot;:&quot;Yali&quot;,&quot;last_name&quot;:&quot;Dou&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2010&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;53&quot;,&quot;pages&quot;:&quot;5179-5185&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm100139b&quot;},&quot;abstract&quot;:&quot;MLL1 is a histone 3 lysine 4 (H3K4) methyltransferase and a promising new cancer therapeutic target. The catalytic activity of MLL1 is regulated by the formation of a core complex consisting of MLL1, WDR5, RbBP5, and Ash2L. The interaction between WDR5 and MLL1 plays an essential role in regulation of the H3K4 methyltransferase activity of MLL1 and targeting this interaction using small molecules may represent an attractive therapeutic strategy. In this study, we have defined the essential elements in MLL1 required for its high-affinity binding to WDR5. Our data showed that the minimal elements crucial for high-affinity binding of MLL1 to WDR5 are −CO-ARA-NH− motif and two intramolecular hydrogen bonds that stabilize the conformation of this motif. Two 3-mer peptides, Ac-ARA-NH2 and Ac-ART-NH2, were designed based upon MLL1 and H3 sequences and achieved Ki values of 120 and 20 nM to WDR5, respectively. Our study provides a concrete basis for the design of potent peptidomimetics and nonpeptidic compounds to in&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm100139b&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100139b" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm100139b&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100139b" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm100139b&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100139b" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm100139b&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm100139b&amp;href=/doi/10.1021/jm100139b" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm100139b" /></input><a href="/doi/pdf/10.1021/jm100139b" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm100139b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm100139b%26sid%3Dliteratum%253Aachs%26pmid%3D20575550%26genre%3Darticle%26aulast%3DKaratas%26date%3D2010%26atitle%3DAnalysis%2Bof%2Bthe%2BBinding%2Bof%2BMixed%2BLineage%2BLeukemia%2B1%2B%2528MLL1%2529%2Band%2BHistone%2B3%2BPeptides%2Bto%2BWD%2BRepeat%2BDomain%2B5%2B%2528WDR5%2529%2Bfor%2Bthe%2BDesign%2Bof%2BInhibitors%2Bof%2Bthe%2BMLL1%25E2%2588%2592WDR5%2BInteraction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D53%26issue%3D14%26spage%3D5179%26epage%3D5185%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=291912" title="Fluorescence">Fluorescence</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/53/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jmcmar.2010.53.issue-14/production/jmcmar.2010.53.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">MLL1 is a histone 3 lysine 4 (H3K4) methyltransferase and a promising new cancer therapeutic target. The catalytic activity of MLL1 is regulated by the formation of a core complex consisting of MLL1, WDR5, RbBP5, and Ash2L. The interaction between WDR5 and MLL1 plays an essential role in regulation of the H3K4 methyltransferase activity of MLL1 and targeting this interaction using small molecules may represent an attractive therapeutic strategy. In this study, we have defined the essential elements in MLL1 required for its high-affinity binding to WDR5. Our data showed that the minimal elements crucial for high-affinity binding of MLL1 to WDR5 are −CO-ARA-NH− motif and two intramolecular hydrogen bonds that stabilize the conformation of this motif. Two 3-mer peptides, Ac-ARA-NH<sub>2</sub> and Ac-ART-NH<sub>2</sub>, were designed based upon MLL1 and H3 sequences and achieved <i>K</i><sub>i</sub> values of 120 and 20 nM to WDR5, respectively. Our study provides a concrete basis for the design of potent peptidomimetics and nonpeptidic compounds to inhibit MLL1 activity by targeting the MLL1 and WDR5 interaction.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histones are important in the organization of DNA into a chromatin structure and in retrieval of genetic information. Specific modifications on histones regulate gene activity, leading to either expression or silencing.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Of all the modifications in the euchromatins of eukaryotes that have been examined, histone 3 lysine 4 (H3K4<a class="ref internalNav" href="#fn1" aria-label="a">a</a><div class="NLM_fn" id="fn1">a<p class="last">Abbreviations: MLL1, mixed lineage leukemia 1; WDR5, WD repeat domain 5; H3K4, histone 3 lysine 4; <i>Hox</i>, homeobox; Ash2L, absent small or homeotic-2-like; RbBP5, retinoblastoma binding protein 5; WIN, WDR5 interacting motif; FP, fluorescence polarization; MD, molecular dynamics; 5-FAM, 5-carboxy fluorescein; mP, millipolarization.</p></div>) trimethylation is recognized as a hallmark of transcriptionally active genes.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> It is believed that trimethylated H3K4 is a recognition site for the recruitment of additional factors required for transcription.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a></div><div class="NLM_p">Abnormalities in H3K4 methylating enzymes have been observed in various cancers,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> the most prominent example of which is mixed lineage leukemia 1 (MLL1).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> MLL1 is enzymatically active in a multiprotein complex and acts as both a global and a specific gene regulator.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> The most well-known targets for MLL1 are the <i>homeobox</i> (<i>Hox</i>) genes such as <i>Hox-a9</i> and <i>Hox-c8</i>. These genes encode for a class of homeodomain transcriptional factors that regulate organ formation during embryo development as well as proper hematopoiesis in adults.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11-13)</a> Increased expression levels of certain <i>Hox</i> genes, accompanied by MLL1 aberrations such as gene fusion and amplification, are frequently observed in acute leukemias such as acute lymphoblastic leukemia (ALL) and acute myeloid leukemia (AML).<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14-16)</a> Injection of cells overexpressing <i>Hox-a7</i> and <i>Hox-c8</i> into nude mice results in well vascularized tumors in 4−5 weeks.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Abnormal <i>Hox</i> gene expression is also observed in solid tumors such as prostate carcinoma and primary colorectal tumors.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> These observations suggest that MLL1 might be a promising new therapeutic target for several forms of leukemias and solid tumors.</div><div class="NLM_p">Immediately after translation, MLL1 is proteolytically cleaved to yield 180-kDa C-terminus (MLL1<sup>C</sup>) and 320-kDa N-terminus fragments (MLL1<sup>N</sup>).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> These are assembled together in a multisubunit complex along with several other proteins, including WD repeat domain 5 (WDR5), absent small or homeotic-2-like (Ash2L), and retinoblastoma binding protein 5 (RbBP5), each of which is a common component of all known human H3K4 methylating complexes. MLL1<sup>C</sup> (hereafter called MLL1 in this paper) forms a catalytically active core complex with WDR5, RbBP5, and Ash2L that can dimethylate H3K4 <i>in vitro</i>.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Although MLL1 alone can partially monomethylate H3K4, all the other members of the core complex are required for dimethylation, including WDR5, which forms a bridge between MLL1 and the remainder of the core complex. In the absence of WDR5, MLL1 is unable to associate with RbBP5 and Ash2L and fails to dimethylate H3K4 in vitro.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> Knock-down of WDR5 is known to result in a significant decrease in the levels of H3K4 trimethylation and expression of <i>Hox-a9</i> and <i>Hox-c8</i> genes in 293 cells.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> These results indicate that blocking the WDR5-MLL1 interaction may be an effective strategy with which to inhibit MLL1 activity.</div><div class="NLM_p">It has recently been shown that MLL1 binds to WDR5 via an arginine (R3765) containing sequence,<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24, 25)</a> which is similar to that used by the N-terminal of H3 in its interaction with WDR5.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26-29)</a> Indeed, WDR5 has a canonical conformation that contains a central cavity and both H3 and MLL1 peptides use an arginine residue to interact with this cavity. Interestingly, although the crystal structures show that H3 and MLL1 peptides have very similar binding modes to WDR5, MLL1 peptides exhibit higher affinity.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> MLL1-derived 12-residue WDR5 interacting motif (WIN) peptide (residues 3762−3773) (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) has been shown to dissociate MLL1 from the remainder of the complex <i>in vitro</i>.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Hence, the WIN peptide represents an excellent starting point for the design of small molecule inhibitors that block the interaction of MLL1 with WDR5.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Sequence of WIN Peptide and N-Terminus of H3 Peptide<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top"><td><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0006.gif" alt="" id="_i3" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Residues 1−10 in H3 and 3762-3773 in MLL1 are shown. Numbering assigned here compares the residues in these two peptides.</p></div></div></div><div class="NLM_p last">Despite the availability of the crystal structures of H3 and MLL1 peptides in complex with WDR5, the essential key binding elements in MLL1 required for its high-affinity binding to WDR5 have not been clearly defined nor have the key structural features responsible for the large binding affinity difference between MLL1 and H3 peptides to WDR5. In the present study, we have performed a systematic analysis of the interaction of MLL1 and H3 peptides with WDR5 and found that in the MLL1 derived peptides, −CO-Ala-Arg-Ala-NH− is the minimum motif for their high-affinity binding to WDR5. Our analysis further showed that two intramolecular hydrogen bonds formed within this motif play a key role for the high-affinity binding to WDR5. The absence of one of these intramolecular hydrogen bonds in H3 peptides is responsible for their weak binding affinity to WDR5. This analysis also led to the design of two 3-mer peptides based upon MLL1 and H3 sequences, Ac-ARA-NH<sub>2</sub> and Ac-ART-NH<sub>2</sub>, which bind to WDR5 with <i>K</i><sub>i</sub> values of 120 and 20 nM, respectively. Our study provides a concrete basis for the design of potent peptidomimetics and nonpeptidic compounds that target the interaction of MLL1 and WDR5.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26401" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26401" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Determination of the Minimal Motif in MLL1 for High-Affinity Binding to WDR5</h3><div class="NLM_p">The MLL1 peptide containing the WIN sequence was shown to bind to WDR5 with a <i>K</i><sub>d</sub> of 0.12 μM.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> To identify the residues required for the high-affinity binding, we systematically deleted residues, whose numbering is given in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, from either the N-terminus or the C-terminus of the WIN peptide (Ac-GSARAEVHLRKS-NH<sub>2</sub>). Unless stated otherwise, all the peptides produced were acetylated at the N-terminus and capped with amide at the C-terminus. The binding affinity of each of the peptides to WDR5 was measured with our optimized fluorescence polarization (FP) based competitive binding assay, and the data are summarized in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinities of Truncated MLL1 Peptides to WDR5</div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col /></col></colgroup><tbody><tr valign="top"><td><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0001.gif" alt="" id="_i7" /></img></td></tr></tbody></table></div></div><div class="NLM_p">As can be seen in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, the 12-residue WIN peptide has a <i>K</i><sub>i</sub> value of 0.16 μM, which is similar to the reported values.<a onclick="showRef(event, 'ref21 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref24">(21, 24)</a> Removal of the Gly from the −2 position does not affect the binding significantly. Interestingly, further deletion of Ser from the −1 position in the 11-residue peptide (Ac-11mer) results in a highly potent 10-residue peptide (Ac-10mer), which has <i>K</i><sub>i</sub> = 3 nM, 50 times more potent than that of the WIN peptide. We also found that the N-terminal acetyl group in Ac-11mer is detrimental to its binding to WDR5. The H<sub>2</sub>N-11mer peptide with a free N-terminus is 10-times more potent than Ac-11mer. Further deletion of Ala1 from Ac-10mer, giving Ac-9mer, decreases the binding affinity by 1500 times.</div><div class="NLM_p last">We next truncated the sequence from the C-terminus starting from Ac-10mer. Simultaneous deletion of Arg8, Lys9, and Ser10, residues unresolved in the cocrystal structure of WIN-WDR5 complex,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> led to Ac-7mer, which has a 10-fold lower binding affinity than Ac-10mer. Further stepwise deletions from the C-terminus generated Ac-6mer, Ac-5mer, Ac-4mer, and Ac-3mer, all of which have binding affinities similar to the 12-residue WIN peptide. However, removal of Ala3 from the Ac-3mer peptide results in >200-fold loss of binding affinity to WDR5. We conclude, therefore, that the Ac-3mer (Ac-ARA-NH<sub>2</sub>) is the shortest MLL1 peptide to achieve high-affinity binding to WDR5.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Role of Two Intramolecular Hydrogen Bonds in MLL1 Peptides for High-Affinity Binding to WDR5</h3><div class="NLM_p">In the crystal structure of WDR5−WIN complex, two intramolecular hydrogen bonds in the main chain of the WIN peptide are present.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The first is between the Ser carbonyl at the −1 position and the amide proton of Ala3 and the second is between the Ala1 carbonyl and the Glu4 amide proton. These intramolecular hydrogen bonds allow the peptide to adopt a 3<sub>10</sub> helical secondary structure and may contribute to the binding affinity of the WIN peptide to WDR5. Ac-ARA-NH<sub>2</sub> shows a binding affinity similar to that of the 12mer WIN peptide, and we have employed this 3mer, which maintains these two intramolecular hydrogen bonds (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>a and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>b) for our investigation.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding models for Ac-ARA-NH<sub>2</sub> and H<sub>2</sub>N-ARA-NH<sub>2</sub> in complex with WDR5 and key intramolecular hydrogen bonds within these peptides. (a) Hydrogen bonds 1 and 2 and the atoms involved in Ac-ARA-NH<sub>2</sub> peptide. (b) Predicted binding model of Ac-ARA-NH<sub>2</sub> in complex with WDR5. (c) Predicted binding model of H<sub>2</sub>N-ARA-NH<sub>2</sub> in complex with WDR5. (b,c) The intramolecular hydrogen bonds are shown as yellow dots. Carbon atoms are shown in cyan in the peptide and gray in WDR5. The nitrogen and oxygen atoms are colored in blue and red, respectively. (d) Probability of the hydrogen bond distance between the nitrogen and oxygen pairs in hydrogen bonds 1 and 2 in Ac-ARA-NH<sub>2</sub> based upon MD simulation. (e) Probability of the hydrogen bond distance between the nitrogen and oxygen pairs in hydrogen bond 2 in NH2-ARA-NH<sub>2</sub> based upon MD simulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We first disrupted hydrogen bond 1 by removing the acetyl group from Ala1 or methylating the Ala3 nitrogen. Removal of the acetyl group from the N-terminal Ala1 of the Ac-3mer peptide yields H<sub>2</sub>N-3mer, which does not bind to WDR5 up to the concentrations tested (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). To further confirm the critical importance of this hydrogen bond, we also removed the acetyl group from the most potent Ac-10mer peptide and observed a >1500-fold decrease in its binding affinity. A similar fold loss of binding affinity was observed when the N-terminal acetyl group was removed from the Ac-7mer, Ac-6mer, and Ac-5mer peptides (results not shown). Methylation of the Ala3 nitrogen (peptide Δ1), which also disrupts hydrogen bond 1, led to a complete loss of its binding to WDR5. Next we removed the methyl group from the N-terminal acetyl, yielding CHO-ARA-NH<sub>2</sub> peptide, in order to explore the contribution of carbonyl in the absence of methyl. Replacement of the acetyl by a formyl group decreases the binding affinity by a factor of 25, suggesting that the methyl group in the N-terminal acetyl contributes to the increase in binding affinity, albeit to a smaller extent. Together, these data show that hydrogen bond 1 plays a vital role in maintaining the high binding affinities of these potent MLL1 peptides to WDR5, presumably through stabilization of the bound conformation.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding Affinities of Peptide Analogues of Ac-ARA-NH<sub>2</sub> Designed to Investigate the Role of Intramolecular Hydrogen Bonds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center">formula</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> ± SD (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ac-10mer</td><td class="colsep0 rowsep0" align="left">Ac-ARAEVHLRKS-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">0.02 ± 0.004</td><td class="colsep0 rowsep0" align="left">0.003 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-10mer</td><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-ARAEVHLRKS-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">34 ± 3</td><td class="colsep0 rowsep0" align="left">7.30 ± 0.70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ac-3mer</td><td class="colsep0 rowsep0" align="left">Ac-ARA-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">0.54 ± 0.03</td><td class="colsep0 rowsep0" align="left">0.12 ± 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-3mer</td><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-ARA-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">>300</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CHO-3mer</td><td class="colsep0 rowsep0" align="left">CHO-ARA-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">14.9 ± 1.4</td><td class="colsep0 rowsep0" align="left">3.20 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ1</td><td class="colsep0 rowsep0" align="left">Ac-AR-(N-Me)A-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">>300</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ2a</td><td class="colsep0 rowsep0" align="left">Ac-ARA-CONHMe</td><td class="colsep0 rowsep0" align="left">0.70 ± 0.14</td><td class="colsep0 rowsep0" align="left">0.15 ± 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ2b</td><td class="colsep0 rowsep0" align="left">Ac-ARA-CONMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">30 ± 5</td><td class="colsep0 rowsep0" align="left">6.50 ± 1.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Δ2c</td><td class="colsep0 rowsep0" align="left">Ac-ARA-COOCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">7.30 ± 0.80</td><td class="colsep0 rowsep0" align="left">1.60 ± 0.20</td></tr></tbody></table></div></div><div class="NLM_p">To perturb hydrogen bond 2, we first mono- or dimethylated the C-terminal amide using Ac-3mer (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). The monomethylated derivative (Δ2a) was shown by modeling to maintain hydrogen bond 2 in its bound conformation and has <i>K</i><sub>i</sub> = 0.15 μM, similar to that of Ac-3mer, but the dimethylated derivative (Δ2b) is 50 times less potent than Ac-3mer. Replacement of the C-terminal amide group with a methyl ester gave Δ2c, which is 10 times less potent than Ac-3mer. These data indicate that while hydrogen bond 2 makes an important contribution to the binding affinity of Ac-3mer, increasing it by perhaps an order of magnitude, it is less critical than hydrogen bond 1.</div><div class="NLM_p">Molecular dynamics (MD) simulations of Ac-ARA-NH<sub>2</sub> and H<sub>2</sub>N-ARA-NH<sub>2</sub> peptides were performed to further investigate hydrogen bonds 1 and 2. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>d, the probability for the distance between the nitrogen and oxygen atoms in hydrogen bond 1 in Ac-ARA-NH<sub>2</sub> has a narrow distribution and a maximum around 3 Å, indicating a very stable and strong hydrogen bond. In comparison, the probability for the distance between the nitrogen and oxygen atoms in hydrogen bond 2 in Ac-ARA-NH<sub>2</sub> has a broad distribution and a maximum around 3 Å, indicating that the hydrogen bond 2 is weaker than hydrogen bond 1. Furthermore, the probability for the distance between the nitrogen and oxygen atoms in hydrogen bond 2 in NH<sub>2</sub>-ARA-NH<sub>2</sub> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>e) has a narrow distribution but a maximum around 7 Å, indicating absence of a hydrogen bond. Our simulations thus suggest that hydrogen bond 1 is stronger than hydrogen bond 2 and is also more critical in maintaining the proper bound conformation of the peptide for interaction with WDR5.</div><div class="NLM_p">On the basis of these data, we conclude that −CO-ARA-NH− is the minimal motif within the WIN peptide for high-affinity binding to WDR5.</div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Binding of H3 Peptides to WDR5</h4><div class="NLM_p">Next we analyzed the binding of H3 peptides to WDR5. While H3 and MLL1 peptides have similar binding modes to WDR5, H3 peptides show much weaker affinities than MLL1 peptides.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div><div class="NLM_p">The H3 peptides have an ART binding motif similar to the ARA binding motif in the MLL1 peptides but possess a free amino group at the Ala1 residue. To investigate if this free amino group is responsible for their weaker binding affinities, we acetylated H3-3mer and H3-10mer peptides (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). In both cases, acetylation of this free amino group increases binding affinity to WDR5 significantly (1500 times for Ac-H3-3mer and >10000 times for Ac-H3-10mer). The Ac-H3-3mer and Ac-H3-10mer peptides have <i>K</i><sub>i</sub> values of 20 and <1 nM, respectively, to WDR5, and are both also more potent than the corresponding MLL1 peptides Ac-3mer and Ac-10mer (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> and Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). These data show that the primary feature underlying the weaker binding affinities of H3 peptides to WDR5 is the free amino group in Ala1 and the absence of intramolecular hydrogen bond 1.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Binding Affinities of Histone 3 Peptides to WDR5</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center">formula</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> ± SD (μM)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub> ± SD (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H3-10mer</td><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-ARTKQTARKS-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">70 ± 6</td><td class="colsep0 rowsep0" align="left">15.10 ± 1.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ac-H3-10mer</td><td class="colsep0 rowsep0" align="left">Ac-ARTKQTARKS-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">0.006 ± 0.002</td><td class="colsep0 rowsep0" align="left"><0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H3-3mer</td><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-ART-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">127 ± 12</td><td class="colsep0 rowsep0" align="left">27.30 ± 2.50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ac-H3-3mer</td><td class="colsep0 rowsep0" align="left">Ac-ART-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">0.08 ± 0.003</td><td class="colsep0 rowsep0" align="left">0.02 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H3-10mer-K4Me</td><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-ARTK(Me)QTARKS-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">69 ± 7</td><td class="colsep0 rowsep0" align="left">15.00 ± 1.60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H3-10mer-K4Me<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-ARTK(Me<sub>2</sub>)QTARKS-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">42 ± 6</td><td class="colsep0 rowsep0" align="left">9.00 ± 1.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H3-10mer-K4Me<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H<sub>2</sub>N-ARTK(Me<sub>3</sub>)QTARKS-NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char="±">73 ± 6</td><td class="colsep0 rowsep0" align="left">15.60 ± 1.20</td></tr></tbody></table></div></div><div class="NLM_p last">We next examined if methylation of Lys4 (K4) significantly changes the binding affinity of H3-10mer to WDR5. Mono-, di-, or trimethylation of K4 led to no significant difference in the binding affinity to WDR5 compared to that of H3-10mer (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). This is consistent with previous experiments using an isothermal titration curve measurement.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Design and Optimization of Fluorescently Tagged Peptides for Assay Development</h4><div class="NLM_p">Development and optimization of our quantitative FP-based assays was important for accurate evaluation of the binding affinities of the compounds under investigation. We initially designed and synthesized two 5-carboxy fluorescein (5-FAM) tagged tracers (WIN-FAM-1 and WIN-FAM-2) using the WIN sequence with two different linkers, shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>. These two tracers were found to have similar <i>K</i><sub>d</sub> values and dynamic ranges in the WDR5 saturation experiments (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Saturation binding experiments of tracers with WDR5. (a) Saturation curves of WIN-FAM-1, WIN-FAM-2, 10mer-Thr-FAM, and 10mer-Ala-FAM with WDR5; 0.6 nM of the tracer was used for these experiments. (b) Stability of the saturation binding experiment for 10mer-Thr-FAM tracer over 24 h. mP values were measured at the times indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Binding Affinities of Fluorescently Tagged Tracers to WDR5 Protein</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="3" align="center">Ac-amino acid sequence-linker-K(5-FAM)-NH<sub>2</sub></th><th class="colsep0 rowsep0" align="char" char="±"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">tracer name</th><th class="colsep0 rowsep0" align="center">amino acid sequence</th><th class="colsep0 rowsep0" align="center">linker</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>d</sub> ± SD (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WIN−FAM-1</td><td class="colsep0 rowsep0" align="left">GSARAEVHLRKS</td><td class="colsep0 rowsep0" align="left">βA-βA-βA<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">0.86 ± 0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WIN−FAM-2</td><td class="colsep0 rowsep0" align="left">GSARAEVHLRKS</td><td class="colsep0 rowsep0" align="left">Ahx-Ahx<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">0.94 ± 0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10mer-Ala-FAM</td><td class="colsep0 rowsep0" align="left">ARAEVHLRKS</td><td class="colsep0 rowsep0" align="left">Ahx-Ahx</td><td class="colsep0 rowsep0" align="char" char="±">0.014 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10mer-Thr-FAM</td><td class="colsep0 rowsep0" align="left">ARTEVHLRKS</td><td class="colsep0 rowsep0" align="left">Ahx-Ahx</td><td class="colsep0 rowsep0" align="char" char="±">0.001 ± 0.0003</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">βA = betaalanine.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">Ahx = 6-amino hexanoic acid.</p></div></div></div><div class="NLM_p">An FP-based competitive binding assay was developed with WIN-FAM-1, and screening of our designed MLL1 peptides with this initial assay revealed that Ac-10mer is much more potent than the WIN peptide and led to the design of 10mer-Ala-FAM (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>). Screening of designed H3 peptides also suggested that substitution of Ala3 with Thr in MLL1 peptides can further improve the binding affinity to WDR5, and this led to the design of 10mer-Thr-FAM (Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>).</div><div class="NLM_p last">Saturation experiments showed that while 10mer-Ala-FAM and 10mer-Thr-FAM have similar dynamic ranges (ΔmP), both peptides have higher affinities for WDR5 than WIN-FAM-1 and WIN-FAM-2, and the affinity of 10mer-Thr-FAM is 10-times higher than that of 10mer-Ala-FAM (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a).</div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Development and Optimization of Competitive Binding Assay Conditions</h4><div class="NLM_p">10mer-Thr-FAM, with the highest binding affinity for WDR5 in the saturation experiments (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>a), was chosen for development of an optimized FP-based competitive binding assay.</div><div class="NLM_p">First, we evaluated the minimal concentration of 10mer-Thr-FAM needed to produce a total fluorescence intensity of 100000 in the competitive binding assay. It was found that this was achieved at the concentrations of 0.6 nM or higher, and this was selected as the tracer concentration for further evaluations of the assay conditions. Next the millipolarization (mP) values were measured at different time points in order to determine equilibrium duration and stability. The equilibrium between the tracer and protein was reached at 2 h and was stable for over 24 h (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b).</div><div class="NLM_p">Higher protein concentrations can increase the dynamic range (ΔmP) of the assay, but for the sake of assay sensitivity, the protein concentration should not exceed the linear range of the saturation curve.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Accordingly, we investigated the optimal protein concentrations for the competitive assay. We evaluated 2, 3, and 4 nM of WDR5 with 0.6 nM of 10mer-Thr-FAM in competitive binding assays and determined the binding affinities of Ac-10mer under these assay conditions. Although very similar IC<sub>50</sub> values were obtained with these three protein concentrations, the dynamic range increased from 31 to 40 mP when the protein concentration increased from 2 to 4 nM. Hence we chose 4 nM as the optimal WDR5 protein concentration and 0.6 nM of 10mer-Thr-FAM tracer in our competitive binding assay for determination of the IC<sub>50</sub> values of all the peptides shown in Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>−<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. <i>K</i><sub>i</sub> values were calculated using the equation and its associated software developed previously for FP-based assays.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> shows representative examples of competitive binding curves for WIN, Ac-10mer, Ac-3mer, and H<sub>2</sub>N-10mer peptides using the optimized competitive binding assay conditions.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Competitive binding curves for Ac-10mer, Ac-3mer, WIN, and H<sub>2</sub>N-10mer as determined using a fluorescence polarization based assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21035" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21035" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">MLL1, one of the H3K4 trimethylating enzymes, is frequently found to be unregulated in cancers, resulting in increased expression levels of <i>Hox</i> target genes which link MLL1 with its tumorigenic properties.<a onclick="showRef(event, 'ref8 ref32'); return false;" href="javascript:void(0);" class="ref ref8 ref32">(8, 32)</a> Consequently, inhibition of MLL1 activity may prove to be a new, attractive strategy for cancer therapy.</div><div class="NLM_p">While MLL1 protein alone has minimal enzymatic activity for the monomethylation of H3K4 <i>in vitro</i>, it is incapable of di- and trimethylation and its overall catalytic activity is dramatically enhanced when it forms a core complex with WDR5, Ash2L, and RbBP5 proteins.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Previous studies have clearly established that interaction between WDR5 and MLL1 is required for the H3K4 catalytic activity of the MLL1 core complex.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> Therefore inhibition of WDR5−MLL1 interaction with small-molecule inhibitors can effectively inhibit the enzymatic activity of MLL1. Previous studies have shown that short MLL1 peptides bind to WDR5 with high affinity, and although MLL1 and H3 peptides interact with WDR5 in similar binding modes, MLL1 peptides have much higher affinity for WDR5 than H3 peptides.<a onclick="showRef(event, 'ref24 ref25 ref30'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref30">(24, 25, 30)</a></div><div class="NLM_p">To facilitate the design of small-molecule inhibitors of the MLL1−WDR5 interaction, we have sought to define the critical elements required for the high-affinity binding of MLL1 to WDR5 and to determine the structural features responsible for the large difference in binding affinities of the MLL1 and H3 peptides to WDR5. Starting from the 12-mer MLL1 WIN peptide and through systematic analysis, we determined that −CO-ARA-NH− is the minimal binding motif in the MLL1 peptides required for the high binding affinity to WDR5. The 3-mer peptide Ac-ARA-NH<sub>2</sub> has <i>K</i><sub>i</sub> = 0.12 μM with WDR5 in our optimized, FP-based competitive binding assay, essentially the same as that of the 12-residue WIN peptide (<i>K</i><sub>i</sub> = 0.16 μM) under the same assay conditions. Interestingly, the residues RKS at the C-terminus of the WIN peptide, which were not resolved in the crystal structure of the WIN peptide complexed with WDR5,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> appear to enhance the binding affinity to WDR5 by a factor of 10. The most potent peptide derived from the MLL1 peptide sequence is Ac-10mer (Ac-ARAEVHLRKS), with <i>K</i><sub>i</sub> = 3 nM, 50 times more potent than the original 12-residue WIN peptide.</div><div class="NLM_p">We observed a dramatic increase in binding affinities of the MLL1 peptides upon N-terminal acetylation of Ala1, which results in formation of an intramolecular hydrogen bond. Our investigation using the Ac-ARA-NH<sub>2</sub> as the template molecule showed that the hydrogen bond 1 is absolutely necessary for this high-affinity binding to WDR5. Furthermore, MD simulations suggest that hydrogen bond 1 is required for the stability of hydrogen bond 2. On the other hand, disruption of the hydrogen bond 2 while maintaining hydrogen bond 1 decreases the binding affinity by only 10-fold. We also determined that the methyl group of the N-terminal acetyl contributes to the binding affinity. Therefore we conclude that both of the intramolecular hydrogen bonds and the methyl of the acetyl group contribute to the improvement in binding affinity upon N-terminal acetylation.</div><div class="NLM_p">Although H3 peptides contain an ART motif, similar to the ARA motif in the MLL1-based peptides, they have a free NH<sub>2</sub> group at the N-terminus of the ART motif and are thus incapable of forming hydrogen bond 1 that is observed in the MLL1 peptides. We hypothesized that the lack of a similar hydrogen bond 1 in H3 peptides underlies their much reduced binding affinities to WDR5 compared to the MLL1 peptides. The wild-type H3 peptide (ARTKQTARKS), with a free N-terminus, has a <i>K</i><sub>i</sub> value of 15 μM to WDR5 in our optimized FP-based competitive binding assay, but introduction of an acetyl group at its N-terminus to establish the missing intramolecular hydrogen bond 1 improves the binding affinity by a factor of >10000. Furthermore, Ac-ART-NH<sub>2</sub>, which is based upon the H3 peptide sequence, binds to WDR5 with <i>K</i><sub>i</sub> = 20 nM and is 6 times more potent than Ac-ARA-NH<sub>2</sub> whose design is based upon the MLL1 peptide sequence.</div><div class="NLM_p">Acetylation of the N-terminus of MLL1 peptides and the 3mer H3 peptide (ART), improves their binding affinities to WDR5 by a factor of ∼1500, but we observe >10000 times improvement with the acetylation of H3-10mer. Ac-ARTKQTA (Ac-H3-7mer) also shows a >10000 increase in binding affinity to WDR5 upon N-terminal acetylation. Therefore it is likely that one or more of the residues (KQTA) in Ac-H3-10mer or Ac-H3-7mer, which are not present in the MLL1 peptides and Ac-ART, contribute to the further increase in binding affinity.</div><div class="NLM_p">We have designed and optimized an FP-based competitive binding assay for accurate determination of the binding affinities of our designed peptides. We initially designed and synthesized two fluorescently (5-FAM) tagged tracers (WIN-FAM-1 and WIN-FAM-2) based upon the WIN peptide with two different linkers. Our data showed these two tracers to have <i>K</i><sub>d</sub> = 0.86 μM and 0.94 μM, respectively, and similar dynamic ranges in the saturation experiments with WDR5. Although these two tracer compounds show only modest affinities to WDR5, they were useful in the development of an initial, FP-based competitive binding assays for the screening of our designed peptides.</div><div class="NLM_p">The screening data suggested that two peptides, Ac-ARAEVHLRKS and Ac-ARTEVHLRKS, may have much higher affinities than the initial 12-residue WIN peptide. Accordingly, we designed two new tracers (10mer-Ala-FAM and 10mer-Thr-FAM) based upon these two peptides and determined that they have <i>K</i><sub>d</sub> values of 14 and 1 nM to WDR5, respectively. Saturation experiments showed that while 10mer-Ala-FAM and 10mer-Thr-FAM have similar dynamic ranges (ΔmP), both peptides have much higher affinities than WIN-FAM-1 and WIN-FAM-2 and 10mer-Thr-FAM is 10 times more potent than 10mer-Ala-FAM. Accordingly, we employed 10mer-Thr-FAM as the tracer with which to establish an optimized FP-based competitive binding assay for accurate determination of the binding affinities for all of our designed peptides.</div><div class="NLM_p last">In summary, our study provides a reliable basis for the design of potent peptidomimetics and nonpeptidic compounds targeting the interaction of MLL1 and WDR5 to inhibit MLL1 activity, which will be reported in due course.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02601" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02601" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Solid Phase Peptide Synthesis</h4><div class="NLM_p last">Peptides were synthesized manually or with an ABI 433 peptide synthesizer using Fmoc chemistry. Rink amide resin was used as the solid support. To avoid side reactions, amino acid residues were protected as follows: Glu (OtBu), His (Trt), Lys (Boc or Mtt), Gln (Trt), Arg (Pbf), Ser (OtBu), Thr (OtBu). HOBt/HBTU or DIC/HOAt was used as the coupling reagent. HCOOH/DIC/HOAt in DMF was used for on-bead formylation where the reaction was carried out in a flask rotated overnight at room temperature using rotavapor without applying vacuum. All the peptides were cleaved from the resin using a TFA:DTT:TIS:H<sub>2</sub>O (17.5 mL:0.5 g:0.5 mL:1 mL) cleavage cocktail which also led to removal of the protecting groups. The cleavage solution was evaporated and the crude product was precipitated with diethyl ether followed by HPLC purification using a C18 reversed phase column (Waters, Sunfire Prep C18, 19 mm × 150 mm, 5 μm). All the purified final peptides were analyzed by analytical RP-HPLC (Waters, Sunfire C18, 4.6 mm × 150 mm, 5 μm) for purity, and the characterization of the peptides was determined by electrospray ionization mass spectroscopy (ESI-MS) as described in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of Peptides Δ2a and Δ2b</h4><div class="NLM_p last">The Ac-AR(Pbf)A-COOH intermediate was synthesized using Fmoc-solid phase chemistry and 2-chlorotrityl chloride resin as the solid support. The first residue (3 equiv)  in DCM was loaded in the presence of 3 equiv of DIEA for 3 h. The carboxylic acid intermediate was cleaved from the resin by treatment with 4 mL of 1% TFA in DCM (3 × 10 min). The filtrate was evaporated and purified over the HPLC using the C18 reverse phase column. The −COOH intermediate (0.2 mmol) dissolved in 10 mL of THF was mixed with 2 equiv of HATU, 2 equiv of HOAt, 1 equiv of DIEA, and 1 equiv of the corresponding amine solution in THF. The reaction mixture was stirred at room temperature for 3 h, the solvent was evaporated, and the crude product was purified by HPLC. The Pbf protected group from the arginine was removed by treatment with the cleavage cocktail, followed by HPLC purification.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Synthesis of Δ2c</h4><div class="NLM_p last">The Ac-AR(Pbf)A-COOH intermediate (0.2 mol) was dissolved in absolute MeOH, and 0.5 mL of TFA was added. The reaction mixture was stirred at room temperature for three days, the solvent was evaporated, and the Pbf protecting group was removed by treatment with the cleavage cocktail to give a final product which was purified by HPLC.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Synthesis of Tracers</h4><div class="NLM_p last">The peptide was synthesized as described above with an Mtt protecting group on the C-terminal lysine residue. The fluorescein label, 5-FAM, was introduced to the side chain of C-terminal lysine group by removal of the Mtt protecting group with 5 mL of 1% TFA in DCM (4 × 10 min) followed by overnight treatment with 1.5 equiv of 5-carboxy fluorescein succinimide ester (5-FAM, SE) and 4 equiv of DIEA. The peptide was cleaved from the resin and purified over HPLC as described above.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Binding Assay</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Protein Expression and Purification for the Binding Assay</h4><div class="NLM_p last">N-terminal His-tagged WDR5Δ23 (residues 24−334) was expressed from the pET28-MHL vector in Rosetta2(DE3) pLysS cells (Novagen). Cells were grown to OD<sub>600</sub> = 0.4−0.6 in 4 L of 2XTY at 30 °C, induced with 0.1 mM IPTG at 16 °C for 16 h and harvested in 20 mM HEPES pH 7.5, 500 mM KCl, 10% glycerol, 0.1 mg/mL PMSF, and 0.05% NP40. Cells were lysed by addition of 0.2 mg/mL hen egg white lysozyme followed by sonication and clarification by centrifuging for 30 min at 15000 rpm. The resin was washed 3 times for 10 min with 40 mL of lysis buffer. His-WDR5Δ23 was eluted from the resin by 3 × 15 min elutions with 20 mM HEPES pH 7.5, 100 mM KCl, 10% glycerol, 250 mM imidazole, pH 7.5. Eluates were clarified by centrifugation at 2000 rpm for 1 min, syringe-filtered through a 0.45 μM membrane (Millipore), and then loaded onto two 5 mL SP-Sepharose Hi-Trap columns using the AKTApurifier (GE Healthcare). Fractions were eluted in 20 mM HEPES pH 7.5, 10% glycerol with a KCl gradient from 0 to 1000 mM, and peak fractions were pooled and concentrated to 64 μM using an Amicon Ultra centrifugal filter, 10000 MWCO (Amicon). Concentrated protein was aliquoted and samples were frozen on dry ice and stored at −80 °C.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> FP-Based Experiments</h4><div class="NLM_p last">All the FP-based experiments were performed in Microfluor 2 Black, “U” Bottom, 96-well microtiter plates (Thermo Scientific) and FP was measured as mP units in a microplate reader (Tecan Ultra) with excitation at 485 nm and emission at 530 nm. The <i>K</i><sub>d</sub> of the tracers and the IC<sub>50</sub> value of the inhibitors were calculated using GraphPad Prism 4 software.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Saturation Binding Experiment to Determine Dissociation Constant (<i>K</i><sub>d</sub>) of the Tracers</h4><div class="NLM_p last">To dilutions of WDR5Δ23 (2.2−0 μM) in 100 μL of assay buffer (0.1 M phosphate, 25 mM KCl, 0.01% Triton, pH 6.5), 20 μL of a fixed concentration of the tracer in the assay buffer was added, followed by an addition of 5 μL of DMSO to give 125 μL of total volume. Each assay had two controls: blank (without protein and tracer) and tracer only (without protein). The plates were incubated on a shaker at room temperature to reach equilibrium, and mP values were measured at the 3 h time point.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Competitive Binding Experiments</h4><div class="NLM_p last">The binding affinities of the synthetic peptides shown in the study were measured using this competitive binding assay. A preincubated complex solution of WDR5Δ23 and the tracer in 120 μL of assay buffer were added to dilutions of the test compound in 5 μL of DMSO, giving final concentrations of WDR5Δ23 and the tracer of 4 and 0.6 nM, respectively. Three control wells were included in each plate: blank (without protein and tracer), 100% inhibition (tracer only), and 0% inhibition (complex solution only). The plates were incubated with shaking at room temperature. The mP values were measured after five hours of incubation, and <i>K</i><sub>i</sub> values were calculated using the equation described previously.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Molecular Dynamics (MD) Simulation</h4><div class="NLM_p last">The crystal structure of WDR5 in complex with the WIN peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6">3EG6</a>) was used to construct the models of the peptides using Sybyl (Tripos, Inc.) with minimization of any mutation introduced. MD simulations were performed with Amber<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> for 3 ns using explicit TIP3P solvent. After initial minimization of the solvent, the system was further relaxed with constraints on the backbone before final minimization. MD simulations involved a gradual increase in temperature to 300 K over 30 ps, while holding the solute constrained, followed by another 30 ps of simulation with the constraint only on the backbone. Further equilibration was performed for 40 ps before the production run. SHAKE was applied to all bonds involving hydrogen atoms to permit a time step of 0.002 ps. Structures were saved every 1 ps for analysis.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29"><a href="/doi/suppl/10.1021/jm100139b">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48065" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48065" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Table containing the purity and the mass spectroscopy data of the peptides described here. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm100139b/suppl_file/jm100139b_si_001.pdf">jm100139b_si_001.pdf (90.28 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm100139b" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15457" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15457" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Internal Medicine</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Pharmacology</span>; 
    <span class="hlFld-Affiliation affiliation">Comprehensive Cancer Center</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#611209000e0c040f0621140c0802094f040514"><span class="__cf_email__" data-cfemail="097a616866646c676e497c64606a61276c6d7c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hacer Karatas</span> - <span class="hlFld-Affiliation affiliation">Department of Medicinal Chemistry</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Internal Medicine</span>; 
    <span class="hlFld-Affiliation affiliation">Comprehensive Cancer Center</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth C. Townsend</span> - <span class="hlFld-Affiliation affiliation">Department of Pathology</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Denzil Bernard</span> - <span class="hlFld-Affiliation affiliation">Department of Internal Medicine</span>; 
    <span class="hlFld-Affiliation affiliation">Comprehensive Cancer Center</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yali Dou</span> - <span class="hlFld-Affiliation affiliation">Department of Pathology</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11385" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11385" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Bill Milne for his critical reading of the manuscript.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65320" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65320" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 34 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lh5RDKgpa8JCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0ljzhV4FN4DMiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Shilatifard, A.</span><span> </span><span class="NLM_article-title">Molecular implementation and physiological roles for histone H3 lysine 4 (H3K4) methylation</span> <span class="citation_source-journal">Curr. Opin. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.ceb.2008.03.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=18508253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2008&pages=341-348&author=A.+Shilatifard&title=Molecular+implementation+and+physiological+roles+for+histone+H3+lysine+4+%28H3K4%29+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular implementation and physiological roles for histone H3 lysine 4 (H3K4) methylation</span></div><div class="casAuthors">Shilatifard, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">341-348</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Chromosomal surfaces are ornamented with a variety of post-translational modifications of histones, which are required for the regulation of many of the DNA-templated processes.  Such histone modifications include acetylation, sumoylation, phosphorylation, ubiquitination, and methylation.  Histone modifications can either function by disrupting chromosomal contacts or by regulating non-histone protein interactions with chromatin.  In this review, recent findings will be discussed regarding the regulation of the implementation and physiol. significance for one such histone modification, histone H3 lysine 4 (H3K4) methylation by the yeast COMPASS and mammalian COMPASS-like complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuhSAfNqAX67Vg90H21EOLACvtfcHk0ljzhV4FN4DMiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL0%253D&md5=c797a57ac3102c7bc78dc3773b5283b6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2008.03.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2008.03.019%26sid%3Dliteratum%253Aachs%26aulast%3DShilatifard%26aufirst%3DA.%26atitle%3DMolecular%2520implementation%2520and%2520physiological%2520roles%2520for%2520histone%2520H3%2520lysine%25204%2520%2528H3K4%2529%2520methylation%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2008%26volume%3D20%26spage%3D341%26epage%3D348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Sims, R. J.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Histone H3 Lys 4 methylation: caught in a bind?</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2779</span><span class="NLM_x">–</span> <span class="NLM_lpage">2786</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=2779-2786&author=R.+J.+Simsauthor=D.+Reinberg&title=Histone+H3+Lys+4+methylation%3A+caught+in+a+bind%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DHistone%2520H3%2520Lys%25204%2520methylation%253A%2520caught%2520in%2520a%2520bind%253F%26jtitle%3DGenes%2520Dev.%26date%3D2006%26volume%3D20%26spage%3D2779%26epage%3D2786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Wysocka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swigut, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tackett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badenhorst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodelling</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">442</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnature04815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16728976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFWnt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=442&publication_year=2006&pages=86-90&author=J.+Wysockaauthor=T.+Swigutauthor=H.+Xiaoauthor=T.+A.+Milneauthor=S.+Y.+Kwonauthor=J.+Landryauthor=M.+Kauerauthor=A.+J.+Tackettauthor=B.+T.+Chaitauthor=P.+Badenhorstauthor=C.+Wuauthor=C.+D.+Allis&title=A+PHD+finger+of+NURF+couples+histone+H3+lysine+4+trimethylation+with+chromatin+remodelling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodeling</span></div><div class="casAuthors">Wysocka, Joanna; Swigut, Tomek; Xiao, Hua; Milne, Thomas A.; Kwon, So Yeon; Landry, Joe; Kauer, Monika; Tackett, Alan J.; Chait, Brian T.; Badenhorst, Paul; Wu, Carl; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">442</span>
        (<span class="NLM_cas:issue">7098</span>),
    <span class="NLM_cas:pages">86-90</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lysine methylation of histones is recognized as an important component of an epigenetic indexing system demarcating transcriptionally active and inactive chromatin domains.  Trimethylation of histone H3 lysine 4 (H3K4me3) marks transcription start sites of virtually all active genes.  Recently, we reported that the WD40-repeat protein WDR5 is important for global levels of H3K4me3 and control of HOX gene expression.  Here we show that a plant homeodomain (PHD) finger of nucleosome remodelling factor (NURF), an ISWI-contg. ATP-dependent chromatin-remodelling complex, mediates a direct preferential assocn. with H3K4me3 tails.  Depletion of H3K4me3 causes partial release of the NURF subunit, BPTF (bromodomain and PHD finger transcription factor), from chromatin and defective recruitment of the assocd. ATPase, SNF2L (also known as ISWI and SMARCA1), to the HOXC8 promoter.  Loss of BPTF in Xenopus embryos mimics WDR5 loss-of-function phenotypes, and compromises spatial control of Hox gene expression.  These results strongly suggest that WDR5 and NURF function in a common biol. pathway in vivo, and that NURF-mediated ATP-dependent chromatin remodelling is directly coupled to H3K4 trimethylation to maintain Hox gene expression patterns during development.  We also identify a previously unknown function for the PHD finger as a highly specialized methyl-lysine-binding domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrypvJ5nCemXbVg90H21EOLACvtfcHk0ljzhV4FN4DMiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFWnt7g%253D&md5=87bc9ef9cab5a400a9b92669917e4439</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04815%26sid%3Dliteratum%253Aachs%26aulast%3DWysocka%26aufirst%3DJ.%26aulast%3DSwigut%26aufirst%3DT.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DKwon%26aufirst%3DS.%2BY.%26aulast%3DLandry%26aufirst%3DJ.%26aulast%3DKauer%26aufirst%3DM.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DBadenhorst%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DA%2520PHD%2520finger%2520of%2520NURF%2520couples%2520histone%2520H3%2520lysine%25204%2520trimethylation%2520with%2520chromatin%2520remodelling%26jtitle%3DNature%26date%3D2006%26volume%3D442%26spage%3D86%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Huntsman, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muleris, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batley, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, V. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedemann, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldas, C.</span><span> </span><span class="NLM_article-title">MLL2, the second human homolog of the <i>Drosophila trithorax</i> gene, maps to 19q13.1 and is amplified in solid tumor cell lines</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">7975</span><span class="NLM_x">–</span> <span class="NLM_lpage">7984</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=7975-7984&author=D.+G.+Huntsmanauthor=S.+F.+Chinauthor=M.+Mulerisauthor=S.+J.+Batleyauthor=V.+P.+Collinsauthor=L.+M.+Wiedemannauthor=S.+Aparicioauthor=C.+Caldas&title=MLL2%2C+the+second+human+homolog+of+the+Drosophila+trithorax+gene%2C+maps+to+19q13.1+and+is+amplified+in+solid+tumor+cell+lines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuntsman%26aufirst%3DD.%2BG.%26aulast%3DChin%26aufirst%3DS.%2BF.%26aulast%3DMuleris%26aufirst%3DM.%26aulast%3DBatley%26aufirst%3DS.%2BJ.%26aulast%3DCollins%26aufirst%3DV.%2BP.%26aulast%3DWiedemann%26aufirst%3DL.%2BM.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DCaldas%26aufirst%3DC.%26atitle%3DMLL2%252C%2520the%2520second%2520human%2520homolog%2520of%2520the%2520Drosophila%2520trithorax%2520gene%252C%2520maps%2520to%252019q13.1%2520and%2520is%2520amplified%2520in%2520solid%2520tumor%2520cell%2520lines%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D7975%26epage%3D7984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Ruault, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brun, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ventura, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roizes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Sario, A.</span><span> </span><span class="NLM_article-title">MLL3, a new human member of the TRX/MLL gene family, maps to 7q36, a chromosome region frequently deleted in myeloid leukaemia</span> <span class="citation_source-journal">Gene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2002&pages=73-81&author=M.+Ruaultauthor=M.+E.+Brunauthor=M.+Venturaauthor=G.+Roizesauthor=A.+De+Sario&title=MLL3%2C+a+new+human+member+of+the+TRX%2FMLL+gene+family%2C+maps+to+7q36%2C+a+chromosome+region+frequently+deleted+in+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRuault%26aufirst%3DM.%26aulast%3DBrun%26aufirst%3DM.%2BE.%26aulast%3DVentura%26aufirst%3DM.%26aulast%3DRoizes%26aufirst%3DG.%26aulast%3DDe%2BSario%26aufirst%3DA.%26atitle%3DMLL3%252C%2520a%2520new%2520human%2520member%2520of%2520the%2520TRX%252FMLL%2520gene%2520family%252C%2520maps%2520to%25207q36%252C%2520a%2520chromosome%2520region%2520frequently%2520deleted%2520in%2520myeloid%2520leukaemia%26jtitle%3DGene%26date%3D2002%26volume%3D284%26spage%3D73%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL: a histone methyltransferase disrupted in leukemia</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">500</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=500-507&author=J.+L.+Hess&title=MLL%3A+a+histone+methyltransferase+disrupted+in+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%253A%2520a%2520histone%2520methyltransferase%2520disrupted%2520in%2520leukemia%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D500%26epage%3D507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Guenther, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenner, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chevalier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croce, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canaani, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span> </span><span class="NLM_article-title">Global and Hox-specific roles for the MLL1 methyltransferase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">8603</span><span class="NLM_x">–</span> <span class="NLM_lpage">8608</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8603-8608&author=M.+G.+Guentherauthor=R.+G.+Jennerauthor=B.+Chevalierauthor=T.+Nakamuraauthor=C.+M.+Croceauthor=E.+Canaaniauthor=R.+A.+Young&title=Global+and+Hox-specific+roles+for+the+MLL1+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGuenther%26aufirst%3DM.%2BG.%26aulast%3DJenner%26aufirst%3DR.%2BG.%26aulast%3DChevalier%26aufirst%3DB.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCanaani%26aufirst%3DE.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DGlobal%2520and%2520Hox-specific%2520roles%2520for%2520the%2520MLL1%2520methyltransferase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A%26date%3D2005%26volume%3D102%26spage%3D8603%26epage%3D8608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Mishra, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ansari, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandal, S. S.</span><span> </span><span class="NLM_article-title">Dynamic association of MLL1, H3K4 trimethylation with chromatin and Hox gene expression during the cell cycle</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">1629</span><span class="NLM_x">–</span> <span class="NLM_lpage">1640</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2009&pages=1629-1640&author=B.+P.+Mishraauthor=K.+I.+Ansariauthor=S.+S.+Mandal&title=Dynamic+association+of+MLL1%2C+H3K4+trimethylation+with+chromatin+and+Hox+gene+expression+during+the+cell+cycle"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DB.%2BP.%26aulast%3DAnsari%26aufirst%3DK.%2BI.%26aulast%3DMandal%26aufirst%3DS.%2BS.%26atitle%3DDynamic%2520association%2520of%2520MLL1%252C%2520H3K4%2520trimethylation%2520with%2520chromatin%2520and%2520Hox%2520gene%2520expression%2520during%2520the%2520cell%2520cycle%26jtitle%3DFEBS%2520J.%26date%3D2009%26volume%3D276%26spage%3D1629%26epage%3D1640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Hombria, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovegrove, B.</span><span> </span><span class="NLM_article-title">Beyond homeosis—HOX function in morphogenesis and organogenesis</span> <span class="citation_source-journal">Differentiation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2003&pages=461-476&author=J.+C.+Hombriaauthor=B.+Lovegrove&title=Beyond+homeosis%E2%80%94HOX+function+in+morphogenesis+and+organogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHombria%26aufirst%3DJ.%2BC.%26aulast%3DLovegrove%26aufirst%3DB.%26atitle%3DBeyond%2520homeosis%25E2%2580%2594HOX%2520function%2520in%2520morphogenesis%2520and%2520organogenesis%26jtitle%3DDifferentiation%26date%3D2003%26volume%3D71%26spage%3D461%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Monier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tevy, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capovilla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semeriva, M.</span><span> </span><span class="NLM_article-title">Downstream of homeotic genes: in the heart of Hox function</span> <span class="citation_source-journal">Fly (Austin)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A280%3ADC%252BD1cnjtVOhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=59-67&author=B.+Monierauthor=M.+F.+Tevyauthor=L.+Perrinauthor=M.+Capovillaauthor=M.+Semeriva&title=Downstream+of+homeotic+genes%3A+in+the+heart+of+Hox+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Downstream of homeotic genes: in the heart of Hox function</span></div><div class="casAuthors">Monier Bruno; Tevy Maria Florencia; Perrin Laurent; Capovilla Maria; Semeriva Michel</div><div class="citationInfo"><span class="NLM_cas:title">Fly</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">59-67</span>
        ISSN:<span class="NLM_cas:issn">1933-6934</span>.
    </div><div class="casAbstract">A functional organ is constituted of diverse cell types.  Each one occupies a distinct position and is associated to specific morphological and physiological functions.  The identification of the genetic programs controlling these elaborated and highly precise features of organogenesis is crucial to understand how a mature organ works under normal conditions, and how pathologies can develop.  Recently, a number of studies have reported a critical role for Hox genes in one example of organogenesis: cardiogenesis in Drosophila.  Beyond the interest in understanding the molecular basis of functional cardiogenesis, this system might provide a model for proposing new paradigms of how Hox genes achieve their action throughout development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_VmBzmR4aGKx48Rj5GnGofW6udTcc2ebYHgOnPoaLaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cnjtVOhtQ%253D%253D&md5=abcb2cb1383b1532dd001cd45b7dc493</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMonier%26aufirst%3DB.%26aulast%3DTevy%26aufirst%3DM.%2BF.%26aulast%3DPerrin%26aufirst%3DL.%26aulast%3DCapovilla%26aufirst%3DM.%26aulast%3DSemeriva%26aufirst%3DM.%26atitle%3DDownstream%2520of%2520homeotic%2520genes%253A%2520in%2520the%2520heart%2520of%2520Hox%2520function%26jtitle%3DFly%2520%2528Austin%2529%26date%3D2007%26volume%3D1%26spage%3D59%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Jude, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Climer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artinger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span> </span><span class="NLM_article-title">Unique and independent roles for MLL in adult hematopoietic stem cells and progenitors</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=324-337&author=C.+D.+Judeauthor=L.+Climerauthor=D.+Xuauthor=E.+Artingerauthor=J.+K.+Fisherauthor=P.+Ernst&title=Unique+and+independent+roles+for+MLL+in+adult+hematopoietic+stem+cells+and+progenitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJude%26aufirst%3DC.%2BD.%26aulast%3DClimer%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DArtinger%26aufirst%3DE.%26aulast%3DFisher%26aufirst%3DJ.%2BK.%26aulast%3DErnst%26aufirst%3DP.%26atitle%3DUnique%2520and%2520independent%2520roles%2520for%2520MLL%2520in%2520adult%2520hematopoietic%2520stem%2520cells%2520and%2520progenitors%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2007%26volume%3D1%26spage%3D324%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Ferrando, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neuberg, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sallan, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, A. T.</span><span> </span><span class="NLM_article-title">Gene expression signatures in MLL-rearranged T-lineage and B-precursor acute leukemias: dominance of HOX dysregulation</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2003&pages=262-268&author=A.+A.+Ferrandoauthor=S.+A.+Armstrongauthor=D.+S.+Neubergauthor=S.+E.+Sallanauthor=L.+B.+Silvermanauthor=S.+J.+Korsmeyerauthor=A.+T.+Look&title=Gene+expression+signatures+in+MLL-rearranged+T-lineage+and+B-precursor+acute+leukemias%3A+dominance+of+HOX+dysregulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DNeuberg%26aufirst%3DD.%2BS.%26aulast%3DSallan%26aufirst%3DS.%2BE.%26aulast%3DSilverman%26aufirst%3DL.%2BB.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DLook%26aufirst%3DA.%2BT.%26atitle%3DGene%2520expression%2520signatures%2520in%2520MLL-rearranged%2520T-lineage%2520and%2520B-precursor%2520acute%2520leukemias%253A%2520dominance%2520of%2520HOX%2520dysregulation%26jtitle%3DBlood%26date%3D2003%26volume%3D102%26spage%3D262%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Harper, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aplan, P. D.</span><span> </span><span class="NLM_article-title">Chromosomal rearrangements leading to MLL gene fusions: clinical and biological aspects</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">10024</span><span class="NLM_x">–</span> <span class="NLM_lpage">10027</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=10024-10027&author=D.+P.+Harperauthor=P.+D.+Aplan&title=Chromosomal+rearrangements+leading+to+MLL+gene+fusions%3A+clinical+and+biological+aspects"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DD.%2BP.%26aulast%3DAplan%26aufirst%3DP.%2BD.%26atitle%3DChromosomal%2520rearrangements%2520leading%2520to%2520MLL%2520gene%2520fusions%253A%2520clinical%2520and%2520biological%2520aspects%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D10024%26epage%3D10027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Argiropoulos, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span> </span><span class="NLM_article-title">Hox genes in hematopoiesis and leukemogenesis</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">6766</span><span class="NLM_x">–</span> <span class="NLM_lpage">6776</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=6766-6776&author=B.+Argiropoulosauthor=R.+K.+Humphries&title=Hox+genes+in+hematopoiesis+and+leukemogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DArgiropoulos%26aufirst%3DB.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26atitle%3DHox%2520genes%2520in%2520hematopoiesis%2520and%2520leukemogenesis%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D6766%26epage%3D6776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Maulbecker, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruss, P.</span><span> </span><span class="NLM_article-title">The oncogenic potential of deregulated homeobox genes</span> <span class="citation_source-journal">Cell Growth Differ.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">441</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=8100147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADyaK3sXmsVKqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1993&pages=431-441&author=C.+C.+Maulbeckerauthor=P.+Gruss&title=The+oncogenic+potential+of+deregulated+homeobox+genes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The oncogenic potential of deregulated homeobox genes</span></div><div class="casAuthors">Maulbecker, Catharina C.; Gruss, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cell Growth & Differentiation</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-41</span>CODEN:
                <span class="NLM_cas:coden">CGDIE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-9523</span>.
    </div><div class="casAbstract">Homeobox genes encode DNA-binding proteins that regulate gene expression during embryonic development.  Deregulation of homeobox gene expression was invoked as the mol. basis of a no. of leukemias.  The authors now demonstrate, using in vitro and in vivo transformation assays, that the overexpression of homeobox genes is the basis for transformation and tumorigenesis.  Thus, clustered homeobox genes related to Drosophila Hom genes (Hoxa-7, Hoxa-5, Hoxa-1, Hoxb-7, and Hoxc-8) as well as the homeobox genes Evx-1 and Cdx-1 constitute a new family of nuclear protooncogenes.  Apparently, many homeobox genes regulating embryogenesis can - if deregulated - also contribute to tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCBoX-SR6PC7Vg90H21EOLACvtfcHk0lh0-3TWjB3VPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmsVKqs7Y%253D&md5=cae393897ab9a897b24f10743ed7d5c9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaulbecker%26aufirst%3DC.%2BC.%26aulast%3DGruss%26aufirst%3DP.%26atitle%3DThe%2520oncogenic%2520potential%2520of%2520deregulated%2520homeobox%2520genes%26jtitle%3DCell%2520Growth%2520Differ.%26date%3D1993%26volume%3D4%26spage%3D431%26epage%3D441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Waltregny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alami, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausse, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leval, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castronovo, V.</span><span> </span><span class="NLM_article-title">Overexpression of the homeobox gene HOXC8 in human prostate cancer correlates with loss of tumor differentiation</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">162</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2002&pages=162-169&author=D.+Waltregnyauthor=Y.+Alamiauthor=N.+Clausseauthor=J.+de+Levalauthor=V.+Castronovo&title=Overexpression+of+the+homeobox+gene+HOXC8+in+human+prostate+cancer+correlates+with+loss+of+tumor+differentiation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWaltregny%26aufirst%3DD.%26aulast%3DAlami%26aufirst%3DY.%26aulast%3DClausse%26aufirst%3DN.%26aulast%3Dde%2BLeval%26aufirst%3DJ.%26aulast%3DCastronovo%26aufirst%3DV.%26atitle%3DOverexpression%2520of%2520the%2520homeobox%2520gene%2520HOXC8%2520in%2520human%2520prostate%2520cancer%2520correlates%2520with%2520loss%2520of%2520tumor%2520differentiation%26jtitle%3DProstate%26date%3D2002%26volume%3D50%26spage%3D162%26epage%3D169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">De Vita, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barba, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Odartchenko, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givel, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freschi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucciarelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magli, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boncinelli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cillo, C.</span><span> </span><span class="NLM_article-title">Expression of homeobox-containing genes in primary and metastatic colorectal cancer</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">29A</span><span class="NLM_x">, </span> <span class="NLM_fpage">887</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29A&publication_year=1993&pages=887-893&author=G.+De+Vitaauthor=P.+Barbaauthor=N.+Odartchenkoauthor=J.+C.+Givelauthor=G.+Freschiauthor=G.+Bucciarelliauthor=M.+C.+Magliauthor=E.+Boncinelliauthor=C.+Cillo&title=Expression+of+homeobox-containing+genes+in+primary+and+metastatic+colorectal+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVita%26aufirst%3DG.%26aulast%3DBarba%26aufirst%3DP.%26aulast%3DOdartchenko%26aufirst%3DN.%26aulast%3DGivel%26aufirst%3DJ.%2BC.%26aulast%3DFreschi%26aufirst%3DG.%26aulast%3DBucciarelli%26aufirst%3DG.%26aulast%3DMagli%26aufirst%3DM.%2BC.%26aulast%3DBoncinelli%26aufirst%3DE.%26aulast%3DCillo%26aufirst%3DC.%26atitle%3DExpression%2520of%2520homeobox-containing%2520genes%2520in%2520primary%2520and%2520metastatic%2520colorectal%2520cancer%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D1993%26volume%3D29A%26spage%3D887%26epage%3D893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Hsieh, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span> </span><span class="NLM_article-title">Proteolytic cleavage of MLL generates a complex of N- and C-terminal fragments that confers protein stability and subnuclear localization</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1128%2FMCB.23.1.186-194.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=12482972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVOksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=186-194&author=J.+J.+Hsiehauthor=P.+Ernstauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+J.+Korsmeyer&title=Proteolytic+cleavage+of+MLL+generates+a+complex+of+N-+and+C-terminal+fragments+that+confers+protein+stability+and+subnuclear+localization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolytic cleavage of MLL generates a complex of N- and C-terminal fragments that confers protein stability and subnuclear localization</span></div><div class="casAuthors">Hsieh, James J.-D.; Ernst, Patricia; Erdjument-Bromage, Hediye; Tempst, Paul; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-194</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mixed-lineage leukemia gene (MLL, ALL1, HRX) encodes a 3,969-amino-acid nuclear protein homologous to Drosophila trithorax and is required to maintain proper Hox gene expression.  Chromosome translocations in human leukemia disrupt MLL (11q23), generating chimeric proteins between the N terminus of MLL and multiple translocation partners.  Here we report that MLL is normally cleaved at two conserved sites (D/GADD and D/GVDD) and that mutation of these sites abolishes the proteolysis.  MLL cleavage generates N-terminal p320 (N320) and C-terminal p180 (C180) fragments, which form a stable complex that localizes to a subnuclear compartment.  The FYRN domain of N320 directly interacts with the FYRC and SET domains of C180.  Disrupting the interaction between N320 and C180 leads to a marked decrease in the level of N320 and a redistribution of C180 to a diffuse nuclear pattern.  These data suggest a model in which a dynamic post-cleavage assocn. confers stability to N320 and correct nuclear sublocalization of the complex, to control the availability of N320 for target genes.  This predicts that MLL fusion proteins of leukemia which would lose the ability to complex with C180 have their stability conferred instead by the fusion partners, thus providing one mechanism for altered target gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfRH2aOlJKXLVg90H21EOLACvtfcHk0lh0-3TWjB3VPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVOksQ%253D%253D&md5=91eefe6c60909debacfc930c16906e7d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.1.186-194.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.1.186-194.2003%26sid%3Dliteratum%253Aachs%26aulast%3DHsieh%26aufirst%3DJ.%2BJ.%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DProteolytic%2520cleavage%2520of%2520MLL%2520generates%2520a%2520complex%2520of%2520N-%2520and%2520C-terminal%2520fragments%2520that%2520confers%2520protein%2520stability%2520and%2520subnuclear%2520localization%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D186%26epage%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">A conserved arginine-containing motif crucial for the assembly and enzymatic activity of the mixed lineage leukemia protein-1 core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32162</span><span class="NLM_x">–</span> <span class="NLM_lpage">32175</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+arginine-containing+motif+crucial+for+the+assembly+and+enzymatic+activity+of+the+mixed+lineage+leukemia+protein-1+core+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0liijCzKmT1ZRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520arginine-containing%2520motif%2520crucial%2520for%2520the%2520assembly%2520and%2520enzymatic%2520activity%2520of%2520the%2520mixed%2520lineage%2520leukemia%2520protein-1%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">713</span><span class="NLM_x">–</span> <span class="NLM_lpage">719</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+methyltransferase+activity+by+its+core+components"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0liijCzKmT1ZRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520by%2520its%2520core%2520components%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Wysocka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swigut, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burlingame, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brivanlou, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">WDR5 associates with histone H3 methylated at K4 and is essential for H3 K4 methylation and vertebrate development</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">872</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.cell.2005.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=15960974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1Gks7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2005&pages=859-872&author=J.+Wysockaauthor=T.+Swigutauthor=T.+A.+Milneauthor=Y.+Douauthor=X.+Zhangauthor=A.+L.+Burlingameauthor=R.+G.+Roederauthor=A.+H.+Brivanlouauthor=C.+D.+Allis&title=WDR5+associates+with+histone+H3+methylated+at+K4+and+is+essential+for+H3+K4+methylation+and+vertebrate+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 associates with histone H3 methylated at K4 and is essential for H3 K4 methylation and vertebrate development</span></div><div class="casAuthors">Wysocka, Joanna; Swigut, Tomek; Milne, Thomas A.; Dou, Yali; Zhang, Xin; Burlingame, Alma L.; Roeder, Robert G.; Brivanlou, Ali H.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-872</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone H3 lysine 4 (K4) methylation has been linked to the transcriptional activation in a variety of eukaryotic species.  Here authors show that a common component of MLL1, MLL2, and hSet1 H3 K4 methyltransferase complexes, the WD40-repeat protein WDR5, directly assocs. with histone H3 di- and trimethylated at K4 and with H3-K4-dimethylated nucleosomes.  WDR5 is required for binding of the methyltransferase complex to the K4-dimethylated H3 tail as well as for global H3 K4 trimethylation and HOX gene activation in human cells.  WDR5 is essential for vertebrate development, in that WDR5-depleted X. laevis tadpoles exhibit a variety of developmental defects and abnormal spatial Hox gene expression.  These results are the first demonstration that a WD40-repeat protein acts as a module for recognition of a specific histone modification and suggest a mechanism for reading and writing an epigenetic mark for gene activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA3qOoBU6MFbVg90H21EOLACvtfcHk0liijCzKmT1ZRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1Gks7g%253D&md5=c0dcb0420bf36dc8f35071c855a5f0dc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DWysocka%26aufirst%3DJ.%26aulast%3DSwigut%26aufirst%3DT.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBurlingame%26aufirst%3DA.%2BL.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DBrivanlou%26aufirst%3DA.%2BH.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DWDR5%2520associates%2520with%2520histone%2520H3%2520methylated%2520at%2520K4%2520and%2520is%2520essential%2520for%2520H3%2520K4%2520methylation%2520and%2520vertebrate%2520development%26jtitle%3DCell%26date%3D2005%26volume%3D121%26spage%3D859%26epage%3D872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Song, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingston, R. E.</span><span> </span><span class="NLM_article-title">WDR5 interacts with mixed lineage leukemia (MLL) protein via the histone H3-binding pocket</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">35258</span><span class="NLM_x">–</span> <span class="NLM_lpage">35264</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=35258-35264&author=J.+J.+Songauthor=R.+E.+Kingston&title=WDR5+interacts+with+mixed+lineage+leukemia+%28MLL%29+protein+via+the+histone+H3-binding+pocket"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%2BJ.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DWDR5%2520interacts%2520with%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520protein%2520via%2520the%2520histone%2520H3-binding%2520pocket%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D35258%26epage%3D35264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">Structure of WDR5 bound to mixed lineage leukemia protein-1 peptide</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32158</span><span class="NLM_x">–</span> <span class="NLM_lpage">32161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+bound+to+mixed+lineage+leukemia+protein-1+peptide"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0lhTU_Gcdi3oTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520bound%2520to%2520mixed%2520lineage%2520leukemia%2520protein-1%2520peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Schuetz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bochkarev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span> </span><span class="NLM_article-title">Structural basis for molecular recognition and presentation of histone H3 by WDR5</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">4245</span><span class="NLM_x">–</span> <span class="NLM_lpage">4252</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fsj.emboj.7601316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16946699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpslamur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=4245-4252&author=A.+Schuetzauthor=A.+Allali-Hassaniauthor=F.+Martinauthor=P.+Loppnauauthor=M.+Vedadiauthor=A.+Bochkarevauthor=A.+N.+Plotnikovauthor=C.+H.+Arrowsmithauthor=J.+Min&title=Structural+basis+for+molecular+recognition+and+presentation+of+histone+H3+by+WDR5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for molecular recognition and presentation of histone H3 By WDR5</span></div><div class="casAuthors">Schuetz, Anja; Allali-Hassani, Abdellah; Martin, Fernando; Loppnau, Peter; Vedadi, Masoud; Bochkarev, Alexey; Plotnikov, Alexander N.; Arrowsmith, Cheryl H.; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4245-4252</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation at specific lysine residues brings about various downstream events that are mediated by different effector proteins.  The WD40 domain of WDR5 represents a new class of histone methyl-lysine recognition domains that is important for recruiting H3K4 methyltransferases to K4-dimethylated histone H3 tail as well as for global and gene-specific K4 trimethylation.  Here we report the crystal structures of full-length WDR5, WDR5Δ23 and its complexes with unmodified, mono-, di- and trimethylated histone H3K4 peptides.  The structures reveal that WDR5 is able to bind all of these histone H3 peptides, but only H3K4me2 peptide forms extra interactions with WDR5 by use of both water-mediated hydrogen bonding and the altered hydrophilicity of the modified lysine 4.  We propose a mechanism for the involvement of WDR5 in binding and presenting histone H3K4 for further methylation as a component of MLL complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVttgBFfxEWLVg90H21EOLACvtfcHk0lhTU_Gcdi3oTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpslamur4%253D&md5=26d05eb16b1691485a7555fda13cf556</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601316%26sid%3Dliteratum%253Aachs%26aulast%3DSchuetz%26aufirst%3DA.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DF.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBochkarev%26aufirst%3DA.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520molecular%2520recognition%2520and%2520presentation%2520of%2520histone%2520H3%2520by%2520WDR5%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D4245%26epage%3D4252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Han, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, J.</span><span> </span><span class="NLM_article-title">Structural basis for the specific recognition of methylated histone H3 lysine 4 by the WD-40 protein WDR5</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">137</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=137-144&author=Z.+Hanauthor=L.+Guoauthor=H.+Wangauthor=Y.+Shenauthor=X.+W.+Dengauthor=J.+Chai&title=Structural+basis+for+the+specific+recognition+of+methylated+histone+H3+lysine+4+by+the+WD-40+protein+WDR5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%2BW.%26aulast%3DChai%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520the%2520specific%2520recognition%2520of%2520methylated%2520histone%2520H3%2520lysine%25204%2520by%2520the%2520WD-40%2520protein%2520WDR5%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D22%26spage%3D137%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Molecular recognition of histone H3 by the WD40 protein WDR5</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb1116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16829960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=698-703&author=J.+F.+Coutureauthor=E.+Collazoauthor=R.+C.+Trievel&title=Molecular+recognition+of+histone+H3+by+the+WD40+protein+WDR5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of histone H3 by the WD40 protein WDR5</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">698-703</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The WD40-repeat protein WDR5 is a conserved subunit of Trithorax (TRX) histone methyltransferase complexes.  WDR5 has been reported to selectively bind dimethylated Lys-4 (K4me2) in histone H3 to promote K4 trimethylation by TRX.  To elucidate the basis of this binding specificity, we have detd. the crystal structure of WDR5 bound to a histone H3 peptide bearing K4me2.  The structure reveals that the N terminus of histone H3 binds as a 310-helix in the central depression formed by the WD40 repeats.  R2 in histone H3 is bound in the acidic channel in the protein's core, whereas K4me2 is solvent exposed and does not engage in direct interactions with WDR5.  Functional studies confirm that WDR5 recognizes A1, R2 and T3 in histone H3 but has virtually identical affinities for the unmodified and mono-, di- and trimethylated forms of K4, demonstrating that it does not discriminate among different degrees of methylation of this residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNU0kVOM0Y3rVg90H21EOLACvtfcHk0lhTU_Gcdi3oTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D&md5=8c77c52b072810965f8c68fb6358a69c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1116%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520recognition%2520of%2520histone%2520H3%2520by%2520the%2520WD40%2520protein%2520WDR5%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D698%26epage%3D703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graybosch, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span> </span><span class="NLM_article-title">Histone H3 recognition and presentation by the WDR5 module of the MLL1 complex</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">704</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb1119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16829959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=704-712&author=A.+J.+Ruthenburgauthor=W.+Wangauthor=D.+M.+Grayboschauthor=H.+Liauthor=C.+D.+Allisauthor=D.+J.+Patelauthor=G.+L.+Verdine&title=Histone+H3+recognition+and+presentation+by+the+WDR5+module+of+the+MLL1+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 recognition and presentation by the WDR5 module of the MLL1 complex</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Wang, Wooikoon; Graybosch, Daina M.; Li, Haitao; Allis, C. David; Patel, Dinshaw J.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">704-712</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">WDR5 is a core component of SET1-family complexes that achieve transcriptional activation via methylation of histone H3 on Nζ of Lys-4 (H3K4).  The role of WDR5 in the MLL1 complex has recently been described as specific recognition of dimethyl-K4 in the context of a histone H3 N-terminus; WDR5 is essential for vertebrate development, Hox gene activation and global H3K4 trimethylation.  We report the high-resoln. x-ray structures of WDR5 in the unliganded form and complexed with histone H3 peptides having unmodified and mono-, di- and trimethylated K4, which together provide the first comprehensive anal. of methylated histone recognition by the ubiquitous WD40-repeat fold.  Contrary to predictions, the structures reveal that WDR5 does not read out the methylation state of K4 directly, but instead serves to present the K4 side chain for further methylation by SET1-family complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWXFgXXkjzvrVg90H21EOLACvtfcHk0ljyT4XzSjcBVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrc%253D&md5=0bc8aab1a1543fd9f7ff8b1fef4ff80c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1119%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGraybosch%26aufirst%3DD.%2BM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DHistone%2520H3%2520recognition%2520and%2520presentation%2520by%2520the%2520WDR5%2520module%2520of%2520the%2520MLL1%2520complex%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D704%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Trievel, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shilatifard, A.</span><span> </span><span class="NLM_article-title">WDR5, a complexed protein</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">680</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb0709-678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=19578375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVCltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=678-680&author=R.+C.+Trievelauthor=A.+Shilatifard&title=WDR5%2C+a+complexed+protein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5, a complexed protein</span></div><div class="casAuthors">Trievel, Raymond C.; Shilatifard, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">678-680</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The WD40 protein WDR5 is a core subunit of the human MLL and SET1 (hCOMPASS) histone H3 Lys4 (H3K4) methyltransferase complexes.  Although initial studies suggested that WDR5 interacts with methylated H3K4 to catalyze Lys4 trimethylation, recent work has revealed that it binds an arginine-bearing motif in MLL1, promoting complex assembly and activity.  These findings suggest that WDR5 functions as a peptidyl arginine-recognition factor that facilitates the assembly of hCOMPASS and other chromatin-modifying complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrStMNXz6SAy7Vg90H21EOLACvtfcHk0ljyT4XzSjcBVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVCltr4%253D&md5=f2f47d834467f501653b8ab27828809f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb0709-678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb0709-678%26sid%3Dliteratum%253Aachs%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26aulast%3DShilatifard%26aufirst%3DA.%26atitle%3DWDR5%252C%2520a%2520complexed%2520protein%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D678%26epage%3D680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Nikolovska-Coleska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0ljyT4XzSjcBVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Faber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van den Heuvel-Eibrink, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwaan, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">HOXA9 is required for survival in human MLL-rearranged acute leukemias</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">2375</span><span class="NLM_x">–</span> <span class="NLM_lpage">2385</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=2375-2385&author=J.+Faberauthor=A.+V.+Krivtsovauthor=M.+C.+Stubbsauthor=R.+Wrightauthor=T.+N.+Davisauthor=M.+van+den+Heuvel-Eibrinkauthor=C.+M.+Zwaanauthor=A.+L.+Kungauthor=S.+A.+Armstrong&title=HOXA9+is+required+for+survival+in+human+MLL-rearranged+acute+leukemias"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFaber%26aufirst%3DJ.%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DStubbs%26aufirst%3DM.%2BC.%26aulast%3DWright%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3Dvan%2Bden%2BHeuvel-Eibrink%26aufirst%3DM.%26aulast%3DZwaan%26aufirst%3DC.%2BM.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DHOXA9%2520is%2520required%2520for%2520survival%2520in%2520human%2520MLL-rearranged%2520acute%2520leukemias%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D2375%26epage%3D2385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">On the mechanism of multiple lysine methylation by the human mixed lineage leukemia protein-1 (MLL1) core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">24242</span><span class="NLM_x">–</span> <span class="NLM_lpage">24256</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1074%2Fjbc.M109.014498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=19556245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24242-24256&author=A.+Patelauthor=V.+Dharmarajanauthor=V.+E.+Voughtauthor=M.+S.+Cosgrove&title=On+the+mechanism+of+multiple+lysine+methylation+by+the+human+mixed+lineage+leukemia+protein-1+%28MLL1%29+core+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">On the Mechanism of Multiple Lysine Methylation by the Human Mixed Lineage Leukemia Protein-1 (MLL1) Core Complex</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Vought, Valarie E.; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">24242-24256</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcription in eukaryotic genomes depends on enzymes that regulate the degree of histone H3 lysine 4 (H3K4) methylation.  The mixed lineage leukemia protein-1 (MLL1) is a member of the SET1 family of H3K4 methyltransferases and is frequently rearranged in acute leukemias.  Despite sequence comparisons that predict that SET1 family enzymes should only monomethylate their substrates, mono-, di-, and trimethylation of H3K4 has been attributed to SET1 family complexes in vivo and in vitro.  To better understand this paradox, we have biochem. reconstituted and characterized a five-component 200-kDa MLL1 core complex contg. human MLL1, WDR5, RbBP5, Ash2L, and DPY-30.  We demonstrate that the isolated MLL1 SET domain is a slow monomethyltransferase and that tyrosine 3942 of MLL1 prevents di- and trimethylation of H3K4.  In contrast, a complex contg. the MLL1 SET domain, WDR5, RbBP5, Ash2L, and DPY-30, displays a marked ∼600-fold increase in enzymic activity but only to the di-Me form of H3K4.  Single turnover kinetic expts. reveal that the reaction leading to H3K4 dimethylation involves the transient accumulation of a monomethylated species, suggesting that the MLL1 core complex uses a non-processive mechanism to catalyze multiple lysine methylation.  We have also discovered that the non-SET domain components of the MLL1 core complex possess a previously unrecognized methyltransferase activity that catalyzes H3K4 dimethylation within the MLL1 core complex.  Our results suggest that the mechanism of multiple lysine methylation by the MLL1 core complex involves the sequential addn. of two Me groups at two distinct active sites within the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZXmcwVcpRrVg90H21EOLACvtfcHk0lgwOzQ0EGrZcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF&md5=4289813be10179ac818b8d93fe6ebc6a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.014498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.014498%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DOn%2520the%2520mechanism%2520of%2520multiple%2520lysine%2520methylation%2520by%2520the%2520human%2520mixed%2520lineage%2520leukemia%2520protein-1%2520%2528MLL1%2529%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24242%26epage%3D24256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">D.A. Case, T. A. D.; T.E. Cheatham, C. L.,  III; Simmerling, J.; Wang, R. E. D.; R. Luo, K. M.; Merz, D. A.; Pearlman, M.; Crowley, R. C.; Walker, W. Z.; B. Wang, S.; Hayik, A.; Roitberg, G.; Seabra, K. F.; Wong, F. P.; X. Wu, S.; Brozell, V.; Tsui, H.; Gohlke, L.; Yang, C.; Tan, J. M.; V. Hornak, G.; Cui, P.; Beroza, D. H.; Mathews, C.; Schafmeister, W. S. R.; Kollman, P.A.</span> <span class="citation_source-book">AMBER 9</span>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=T.+A.+D.+D.A.+Case&author=C.+L.+T.E.+Cheatham&author=J.+Simmerling&author=R.+E.+D.+Wang&author=K.+M.+R.+Luo&author=D.+A.+Merz&author=M.+Pearlman&author=R.+C.+Crowley&author=W.+Z.+Walker&author=S.+B.+Wang&author=A.+Hayik&author=G.+Roitberg&author=K.+F.+Seabra&author=F.+P.+Wong&author=S.+X.+Wu&author=V.+Brozell&author=H.+Tsui&author=L.+Gohlke&author=C.+Yang&author=J.+M.+Tan&author=G.+V.+Hornak&author=P.+Cui&author=D.+H.+Beroza&author=C.+Mathews&author=W.+S.+R.+Schafmeister&author=P.A.+Kollman&title=AMBER+9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DD.A.%2BCase%26aufirst%3DT.%2BA.%2BD.%26btitle%3DAMBER%25209%26pub%3DUniversity%2520of%2520California%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 44 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Weilin Chen, Xin Chen, Dongdong Li, Jianrui Zhou, Zhengyu Jiang, Qidong You, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoke Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of DDO-2213 as a Potent and Orally Bioavailable Inhibitor of the WDR5–Mixed Lineage Leukemia 1 Protein–Protein Interaction for the Treatment of MLL Fusion Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8221-8245. <a href="https://doi.org/10.1021/acs.jmedchem.1c00091" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00091%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BDDO-2213%252Bas%252Ba%252BPotent%252Band%252BOrally%252BBioavailable%252BInhibitor%252Bof%252Bthe%252BWDR5%2525E2%252580%252593Mixed%252BLineage%252BLeukemia%252B1%252BProtein%2525E2%252580%252593Protein%252BInteraction%252Bfor%252Bthe%252BTreatment%252Bof%252BMLL%252BFusion%252BLeukemia%26aulast%3DChen%26aufirst%3DWeilin%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D19012021%26date%3D09062021%26volume%3D64%26issue%3D12%26spage%3D8221%26epage%3D8245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianhua Tian, Kevin B. Teuscher, Erin R. Aho, Joseph R. Alvarado, Jonathan J. Mills, Kenneth M. Meyers, Rocco D. Gogliotti, Changho Han, Jonathan D. Macdonald, Jiqing Sai, J. Grace Shaw, John L. Sensintaffar, Bin Zhao, Tyson A. Rietz, Lance R. Thomas, William G. Payne, William J. Moore, Gordon M. Stott, Jumpei Kondo, Masahiro Inoue, Robert J. Coffey, William P. Tansey, Shaun R. Stauffer, Taekyu Lee, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Structure-Based Optimization of Potent and Selective WD Repeat Domain 5 (WDR5) Inhibitors Containing a Dihydroisoquinolinone Bicyclic Core. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 656-675. <a href="https://doi.org/10.1021/acs.jmedchem.9b01608" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01608</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01608%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BStructure-Based%252BOptimization%252Bof%252BPotent%252Band%252BSelective%252BWD%252BRepeat%252BDomain%252B5%252B%252528WDR5%252529%252BInhibitors%252BContaining%252Ba%252BDihydroisoquinolinone%252BBicyclic%252BCore%26aulast%3DTian%26aufirst%3DJianhua%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D27092019%26date%3D07012020%26date%3D20122019%26volume%3D63%26issue%3D2%26spage%3D656%26epage%3D675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Feng Wang, Kyu Ok Jeon, James M. Salovich, Jonathan D. Macdonald, Joseph Alvarado, Rocco D. Gogliotti, Jason Phan, Edward T. Olejniczak, Qi Sun, Shidong Wang, DeMarco Camper, Joannes P. Yuh, J. Grace Shaw, Jiqing Sai, Olivia W. Rossanese, William P. Tansey, Shaun R. Stauffer, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent 2-Aryl-6,7-dihydro-5H-pyrrolo[1,2-a]imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (13)
                                     , 5623-5642. <a href="https://doi.org/10.1021/acs.jmedchem.8b00375" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00375%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252B2-Aryl-6%25252C7-dihydro-5H-pyrrolo%25255B1%25252C2-a%25255Dimidazoles%252Bas%252BWDR5-WIN-Site%252BInhibitors%252BUsing%252BFragment-Based%252BMethods%252Band%252BStructure-Based%252BDesign%26aulast%3DWang%26aufirst%3DFeng%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D08032018%26date%3D29062018%26date%3D11062018%26volume%3D61%26issue%3D13%26spage%3D5623%26epage%3D5642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hacer  Karatas</span>, <span class="hlFld-ContribAuthor ">Yangbing  Li</span>, <span class="hlFld-ContribAuthor ">Liu  Liu</span>, <span class="hlFld-ContribAuthor ">Jiao  Ji</span>, <span class="hlFld-ContribAuthor ">Shirley  Lee</span>, <span class="hlFld-ContribAuthor ">Yong  Chen</span>, <span class="hlFld-ContribAuthor ">Jiuling  Yang</span>, <span class="hlFld-ContribAuthor ">Liyue  Huang</span>, <span class="hlFld-ContribAuthor ">Denzil  Bernard</span>, <span class="hlFld-ContribAuthor ">Jing  Xu</span>, <span class="hlFld-ContribAuthor ">Elizabeth C.  Townsend</span>, <span class="hlFld-ContribAuthor ">Fang  Cao</span>, <span class="hlFld-ContribAuthor ">Xu  Ran</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Li</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Duxin  Sun</span>, <span class="hlFld-ContribAuthor ">Jeanne A  Stuckey</span>, <span class="hlFld-ContribAuthor ">Ming  Lei</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, and <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (12)
                                     , 4818-4839. <a href="https://doi.org/10.1021/acs.jmedchem.6b01796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01796</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01796%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BHighly%252BPotent%25252C%252BCell-Permeable%252BMacrocyclic%252BPeptidomimetic%252B%252528MM-589%252529%252BTargeting%252Bthe%252BWD%252BRepeat%252BDomain%252B5%252BProtein%252B%252528WDR5%252529%2525E2%252580%252593Mixed%252BLineage%252BLeukemia%252B%252528MLL%252529%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DKaratas%26aufirst%3DHacer%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D09122016%26date%3D12062017%26date%3D22062017%26volume%3D60%26issue%3D12%26spage%3D4818%26epage%3D4839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthäus  Getlik</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Carlos  Zepeda-Velázquez</span>, <span class="hlFld-ContribAuthor ">Yuri  Bolshan</span>, <span class="hlFld-ContribAuthor ">Gennady  Poda</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Kuznetsova</span>, <span class="hlFld-ContribAuthor ">Richard  Marcellus</span>, <span class="hlFld-ContribAuthor ">Guillermo  Senisterra</span>, <span class="hlFld-ContribAuthor ">Ludmila  Dombrovski</span>, <span class="hlFld-ContribAuthor ">Taraneh  Hajian</span>, <span class="hlFld-ContribAuthor ">Taira  Kiyota</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, and <span class="hlFld-ContribAuthor ">Rima  Al-awar</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (6)
                                     , 2478-2496. <a href="https://doi.org/10.1021/acs.jmedchem.5b01630" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01630</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BOptimization%252Bof%252Ba%252BSmall%252BMolecule%252BAntagonist%252Bof%252Bthe%252BInteraction%252BBetween%252BWD%252BRepeat-Containing%252BProtein%252B5%252B%252528WDR5%252529%252Band%252BMixed-Lineage%252BLeukemia%252B1%252B%252528MLL1%252529%26aulast%3DGetlik%26aufirst%3DMatth%25C3%25A4us%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D16102015%26date%3D09032016%26date%3D24032016%26volume%3D59%26issue%3D6%26spage%3D2478%26epage%3D2496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hacer  Karatas</span>, <span class="hlFld-ContribAuthor ">Elizabeth C.  Townsend</span>, <span class="hlFld-ContribAuthor ">Fang  Cao</span>, <span class="hlFld-ContribAuthor ">Yong  Chen</span>, <span class="hlFld-ContribAuthor ">Denzil  Bernard</span>, <span class="hlFld-ContribAuthor ">Liu  Liu</span>, <span class="hlFld-ContribAuthor ">Ming  Lei</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, and <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">High-Affinity, Small-Molecule Peptidomimetic Inhibitors of MLL1/WDR5 Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2013,</strong> <em>135 </em>
                                    (2)
                                     , 669-682. <a href="https://doi.org/10.1021/ja306028q" title="DOI URL">https://doi.org/10.1021/ja306028q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ja306028q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fja306028q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DHigh-Affinity%25252C%252BSmall-Molecule%252BPeptidomimetic%252BInhibitors%252Bof%252BMLL1%25252FWDR5%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DKaratas%26aufirst%3DHacer%26date%3D2013%26date%3D2012%26date%3D2012%26date%3D20062012%26date%3D27122012%26date%3D16012013%26date%3D04122012%26volume%3D135%26issue%3D2%26spage%3D669%26epage%3D682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Structure, function and inhibition of critical protein–protein interactions involving mixed lineage leukemia 1 and its fusion oncoproteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01057-7" title="DOI URL">https://doi.org/10.1186/s13045-021-01057-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01057-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01057-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DStructure%25252C%252Bfunction%252Band%252Binhibition%252Bof%252Bcritical%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Binvolving%252Bmixed%252Blineage%252Bleukemia%252B1%252Band%252Bits%252Bfusion%252Boncoproteins%26aulast%3DLi%26aufirst%3DXin%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weilin  Chen</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Dongdong  Li</span>, <span class="hlFld-ContribAuthor ">Xianghan  Wang</span>, <span class="hlFld-ContribAuthor ">Guanlu  Long</span>, <span class="hlFld-ContribAuthor ">Zhengyu  Jiang</span>, <span class="hlFld-ContribAuthor ">Qidong  You</span>, <span class="hlFld-ContribAuthor ">Xiaoke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent MLL1 and WDR5 protein-protein interaction inhibitor with in vivo antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>223 </em>, 113677. <a href="https://doi.org/10.1016/j.ejmech.2021.113677" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113677%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252BMLL1%252Band%252BWDR5%252Bprotein-protein%252Binteraction%252Binhibitor%252Bwith%252Bin%2525C2%2525A0vivo%252Bantitumor%252Bactivity%26aulast%3DChen%26aufirst%3DWeilin%26date%3D2021%26volume%3D223%26spage%3D113677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ali  Imran</span>, <span class="hlFld-ContribAuthor ">Brandon S.  Moyer</span>, <span class="hlFld-ContribAuthor ">Ashley J.  Canning</span>, <span class="hlFld-ContribAuthor ">Dan  Kalina</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Duncan</span>, <span class="hlFld-ContribAuthor ">Kelsey J.  Moody</span>, <span class="hlFld-ContribAuthor ">Aaron J.  Wolfe</span>, <span class="hlFld-ContribAuthor ">Michael S.  Cosgrove</span>, <span class="hlFld-ContribAuthor ">Liviu  Movileanu</span>. </span><span class="cited-content_cbyCitation_article-title">Kinetics of the multitasking high-affinity Win binding site of WDR5 in restricted and unrestricted conditions. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2021,</strong> <em>478 </em>
                                    (11)
                                     , 2145-2161. <a href="https://doi.org/10.1042/BCJ20210253" title="DOI URL">https://doi.org/10.1042/BCJ20210253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20210253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20210253%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DKinetics%252Bof%252Bthe%252Bmultitasking%252Bhigh-affinity%252BWin%252Bbinding%252Bsite%252Bof%252BWDR5%252Bin%252Brestricted%252Band%252Bunrestricted%252Bconditions%26aulast%3DImran%26aufirst%3DAli%26date%3D2021%26date%3D2021%26volume%3D478%26issue%3D11%26spage%3D2145%26epage%3D2161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeyapal  Sugeedha</span>, <span class="hlFld-ContribAuthor ">Jyoti  Gautam</span>, <span class="hlFld-ContribAuthor ">Shweta  Tyagi</span>. </span><span class="cited-content_cbyCitation_article-title">SET1/MLL family of proteins: functions beyond histone methylation. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2021,</strong> <em>16 </em>
                                    (5)
                                     , 469-487. <a href="https://doi.org/10.1080/15592294.2020.1809873" title="DOI URL">https://doi.org/10.1080/15592294.2020.1809873</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2020.1809873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2020.1809873%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DSET1%25252FMLL%252Bfamily%252Bof%252Bproteins%25253A%252Bfunctions%252Bbeyond%252Bhistone%252Bmethylation%26aulast%3DSugeedha%26aufirst%3DJeyapal%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D469%26epage%3D487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alissa D.  Guarnaccia</span>, <span class="hlFld-ContribAuthor ">Kristie L.  Rose</span>, <span class="hlFld-ContribAuthor ">Jing  Wang</span>, <span class="hlFld-ContribAuthor ">Bin  Zhao</span>, <span class="hlFld-ContribAuthor ">Tessa M.  Popay</span>, <span class="hlFld-ContribAuthor ">Christina E.  Wang</span>, <span class="hlFld-ContribAuthor ">Kiana  Guerrazzi</span>, <span class="hlFld-ContribAuthor ">Salisha  Hill</span>, <span class="hlFld-ContribAuthor ">Chase M.  Woodley</span>, <span class="hlFld-ContribAuthor ">Tyler J.  Hansen</span>, <span class="hlFld-ContribAuthor ">Shelly L.  Lorey</span>, <span class="hlFld-ContribAuthor ">J. Grace  Shaw</span>, <span class="hlFld-ContribAuthor ">William G.  Payne</span>, <span class="hlFld-ContribAuthor ">April M.  Weissmiller</span>, <span class="hlFld-ContribAuthor ">Edward T.  Olejniczak</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Fesik</span>, <span class="hlFld-ContribAuthor ">Qi  Liu</span>, <span class="hlFld-ContribAuthor ">William P.  Tansey</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of WIN site inhibitor on the WDR5 interactome. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2021,</strong> <em>34 </em>
                                    (3)
                                     , 108636. <a href="https://doi.org/10.1016/j.celrep.2020.108636" title="DOI URL">https://doi.org/10.1016/j.celrep.2020.108636</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2020.108636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2020.108636%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DImpact%252Bof%252BWIN%252Bsite%252Binhibitor%252Bon%252Bthe%252BWDR5%252Binteractome%26aulast%3DGuarnaccia%26aufirst%3DAlissa%2BD.%26date%3D2021%26volume%3D34%26issue%3D3%26spage%3D108636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqing  Ye</span>, <span class="hlFld-ContribAuthor ">Gang  Chen</span>, <span class="hlFld-ContribAuthor ">Jia  Jin</span>, <span class="hlFld-ContribAuthor ">Binzhong  Zhang</span>, <span class="hlFld-ContribAuthor ">Yinda  Wang</span>, <span class="hlFld-ContribAuthor ">Zhenhai  Cai</span>, <span class="hlFld-ContribAuthor ">Fei  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">The Development of Inhibitors Targeting the Mixed Lineage Leukemia 1 (MLL1)-WD Repeat Domain 5 Protein (WDR5) Protein- Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (33)
                                     , 5530-5542. <a href="https://doi.org/10.2174/0929867326666190528080514" title="DOI URL">https://doi.org/10.2174/0929867326666190528080514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867326666190528080514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867326666190528080514%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DThe%252BDevelopment%252Bof%252BInhibitors%252BTargeting%252Bthe%252BMixed%252BLineage%252BLeukemia%252B1%252B%252528MLL1%252529-WD%252BRepeat%252BDomain%252B5%252BProtein%252B%252528WDR5%252529%252BProtein-%252BProtein%252BInteraction%26aulast%3DYe%26aufirst%3DXiaoqing%26date%3D2020%26volume%3D27%26issue%3D33%26spage%3D5530%26epage%3D5542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brian M  Linhares</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic protein–protein interactions with small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (14)
                                     , 1305-1326. <a href="https://doi.org/10.4155/fmc-2020-0082" title="DOI URL">https://doi.org/10.4155/fmc-2020-0082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0082%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bepigenetic%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Bwith%252Bsmall-molecule%252Binhibitors%26aulast%3DLinhares%26aufirst%3DBrian%2BM%26date%3D2020%26volume%3D12%26issue%3D14%26spage%3D1305%26epage%3D1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Ying-Zhe  Wang</span>, <span class="hlFld-ContribAuthor ">Jun-Jie  Xu</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Proton pump inhibitors selectively suppress MLL rearranged leukemia cells via disrupting MLL1-WDR5 protein-protein interaction. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>188 </em>, 112027. <a href="https://doi.org/10.1016/j.ejmech.2019.112027" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.112027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.112027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.112027%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProton%252Bpump%252Binhibitors%252Bselectively%252Bsuppress%252BMLL%252Brearranged%252Bleukemia%252Bcells%252Bvia%252Bdisrupting%252BMLL1-WDR5%252Bprotein-protein%252Binteraction%26aulast%3DChen%26aufirst%3DWei-Lin%26date%3D2020%26volume%3D188%26spage%3D112027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chun‐wa  Chung</span>, <span class="hlFld-ContribAuthor ">Michael M.  Hann</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Protein–Protein Interactions Perspective. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 479-502. <a href="https://doi.org/10.1002/9781118681121.ch20" title="DOI URL">https://doi.org/10.1002/9781118681121.ch20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch20%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252BProtein%2525E2%252580%252593Protein%252BInteractions%252BPerspective%26aulast%3DChung%26aufirst%3DChun%25E2%2580%2590wa%26date%3D2020%26date%3D2020%26spage%3D479%26epage%3D502%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Hongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic machinery: Emerging novel allosteric inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2019,</strong> <em>204 </em>, 107406. <a href="https://doi.org/10.1016/j.pharmthera.2019.107406" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2019.107406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2019.107406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2019.107406%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bepigenetic%252Bmachinery%25253A%252BEmerging%252Bnovel%252Ballosteric%252Binhibitors%26aulast%3DYe%26aufirst%3DFei%26date%3D2019%26volume%3D204%26spage%3D107406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lea  Albert</span>, <span class="hlFld-ContribAuthor ">Olalla  Vázquez</span>. </span><span class="cited-content_cbyCitation_article-title">Photoswitchable peptides for spatiotemporal control of biological functions. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (69)
                                     , 10192-10213. <a href="https://doi.org/10.1039/C9CC03346G" title="DOI URL">https://doi.org/10.1039/C9CC03346G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CC03346G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CC03346G%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DPhotoswitchable%252Bpeptides%252Bfor%252Bspatiotemporal%252Bcontrol%252Bof%252Bbiological%252Bfunctions%26aulast%3DAlbert%26aufirst%3DLea%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D69%26spage%3D10192%26epage%3D10213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lambert</span>, <span class="hlFld-ContribAuthor ">Alioui</span>, <span class="hlFld-ContribAuthor ">Jambon</span>, <span class="hlFld-ContribAuthor ">Depauw</span>, <span class="hlFld-ContribAuthor ">Van Seuningen</span>, <span class="hlFld-ContribAuthor ">David-Cordonnier</span>. </span><span class="cited-content_cbyCitation_article-title">Direct and Indirect Targeting of HOXA9 Transcription Factor in Acute Myeloid Leukemia.. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (6)
                                     , 837. <a href="https://doi.org/10.3390/cancers11060837" title="DOI URL">https://doi.org/10.3390/cancers11060837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11060837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11060837%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DDirect%252Band%252BIndirect%252BTargeting%252Bof%252BHOXA9%252BTranscription%252BFactor%252Bin%252BAcute%252BMyeloid%252BLeukemia.%26aulast%3DLambert%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D6%26spage%3D837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erin R.  Aho</span>, <span class="hlFld-ContribAuthor ">Jing  Wang</span>, <span class="hlFld-ContribAuthor ">Rocco D.  Gogliotti</span>, <span class="hlFld-ContribAuthor ">Gregory C.  Howard</span>, <span class="hlFld-ContribAuthor ">Jason  Phan</span>, <span class="hlFld-ContribAuthor ">Pankaj  Acharya</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Macdonald</span>, <span class="hlFld-ContribAuthor ">Ken  Cheng</span>, <span class="hlFld-ContribAuthor ">Shelly L.  Lorey</span>, <span class="hlFld-ContribAuthor ">Bin  Lu</span>, <span class="hlFld-ContribAuthor ">Sabine  Wenzel</span>, <span class="hlFld-ContribAuthor ">Audra M.  Foshage</span>, <span class="hlFld-ContribAuthor ">Joseph  Alvarado</span>, <span class="hlFld-ContribAuthor ">Feng  Wang</span>, <span class="hlFld-ContribAuthor ">J. Grace  Shaw</span>, <span class="hlFld-ContribAuthor ">Bin  Zhao</span>, <span class="hlFld-ContribAuthor ">April M.  Weissmiller</span>, <span class="hlFld-ContribAuthor ">Lance R.  Thomas</span>, <span class="hlFld-ContribAuthor ">Christopher R.  Vakoc</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Hall</span>, <span class="hlFld-ContribAuthor ">Scott W.  Hiebert</span>, <span class="hlFld-ContribAuthor ">Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Shaun R.  Stauffer</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Fesik</span>, <span class="hlFld-ContribAuthor ">William P.  Tansey</span>. </span><span class="cited-content_cbyCitation_article-title">Displacement of WDR5 from Chromatin by a WIN Site Inhibitor with Picomolar Affinity. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2019,</strong> <em>26 </em>
                                    (11)
                                     , 2916-2928.e13. <a href="https://doi.org/10.1016/j.celrep.2019.02.047" title="DOI URL">https://doi.org/10.1016/j.celrep.2019.02.047</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2019.02.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2019.02.047%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DDisplacement%252Bof%252BWDR5%252Bfrom%252BChromatin%252Bby%252Ba%252BWIN%252BSite%252BInhibitor%252Bwith%252BPicomolar%252BAffinity%26aulast%3DAho%26aufirst%3DErin%2BR.%26date%3D2019%26volume%3D26%26issue%3D11%26spage%3D2916%26epage%3D2928.e13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Methyl-Readers and Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 339-399. <a href="https://doi.org/10.1007/7355_2019_78" title="DOI URL">https://doi.org/10.1007/7355_2019_78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_78%26sid%3Dliteratum%253Aachs%26atitle%3DMethyl-Readers%252Band%252BInhibitors%26aulast%3DSbardella%26aufirst%3DGianluca%26date%3D2019%26date%3D2019%26spage%3D339%26epage%3D399%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xu  Ran</span>, <span class="hlFld-ContribAuthor ">Eileen T.  Burchfiel</span>, <span class="hlFld-ContribAuthor ">Bushu  Dong</span>, <span class="hlFld-ContribAuthor ">Nicholas J.  Rettko</span>, <span class="hlFld-ContribAuthor ">Bryan M.  Dunyak</span>, <span class="hlFld-ContribAuthor ">Hao  Shao</span>, <span class="hlFld-ContribAuthor ">Dennis J.  Thiele</span>, <span class="hlFld-ContribAuthor ">Jason E.  Gestwicki</span>. </span><span class="cited-content_cbyCitation_article-title">Rational design and screening of peptide-based inhibitors of heat shock factor 1 (HSF1). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (19)
                                     , 5299-5306. <a href="https://doi.org/10.1016/j.bmc.2018.04.018" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.04.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.04.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.04.018%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DRational%252Bdesign%252Band%252Bscreening%252Bof%252Bpeptide-based%252Binhibitors%252Bof%252Bheat%252Bshock%252Bfactor%252B1%252B%252528HSF1%252529%26aulast%3DRan%26aufirst%3DXu%26date%3D2018%26volume%3D26%26issue%3D19%26spage%3D5299%26epage%3D5306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alissa  Guarnaccia</span>, <span class="hlFld-ContribAuthor ">William  Tansey</span>. </span><span class="cited-content_cbyCitation_article-title">Moonlighting with WDR5: A Cellular Multitasker. </span><span class="cited-content_cbyCitation_journal-name">Journal of Clinical Medicine</span><span> <strong>2018,</strong> <em>7 </em>
                                    (2)
                                     , 21. <a href="https://doi.org/10.3390/jcm7020021" title="DOI URL">https://doi.org/10.3390/jcm7020021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/jcm7020021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fjcm7020021%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Clinical%2520Medicine%26atitle%3DMoonlighting%252Bwith%252BWDR5%25253A%252BA%252BCellular%252BMultitasker%26aulast%3DGuarnaccia%26aufirst%3DAlissa%26date%3D2018%26date%3D2018%26volume%3D7%26issue%3D2%26spage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein-protein interaction between MLL1 and reciprocal proteins for leukemia therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (2)
                                     , 356-365. <a href="https://doi.org/10.1016/j.bmc.2017.11.045" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.11.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.11.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.11.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bprotein-protein%252Binteraction%252Bbetween%252BMLL1%252Band%252Breciprocal%252Bproteins%252Bfor%252Bleukemia%252Btherapy%26aulast%3DWang%26aufirst%3DZhi-Hui%26date%3D2018%26volume%3D26%26issue%3D2%26spage%3D356%26epage%3D365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrei V.  Krivtsov</span>, <span class="hlFld-ContribAuthor ">Takayuki  Hoshii</span>, <span class="hlFld-ContribAuthor ">Scott A.  Armstrong</span>. </span><span class="cited-content_cbyCitation_article-title">Mixed-Lineage Leukemia Fusions and Chromatin in Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Cold Spring Harbor Perspectives in Medicine</span><span> <strong>2017,</strong> <em>7 </em>
                                    (11)
                                     , a026658. <a href="https://doi.org/10.1101/cshperspect.a026658" title="DOI URL">https://doi.org/10.1101/cshperspect.a026658</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/cshperspect.a026658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fcshperspect.a026658%26sid%3Dliteratum%253Aachs%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspectives%2520in%2520Medicine%26atitle%3DMixed-Lineage%252BLeukemia%252BFusions%252Band%252BChromatin%252Bin%252BLeukemia%26aulast%3DKrivtsov%26aufirst%3DAndrei%2BV.%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D11%26spage%3Da026658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Levi  Blazer</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Taraneh  Hajian</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting human SET1/MLL family of proteins. </span><span class="cited-content_cbyCitation_journal-name">Protein Science</span><span> <strong>2017,</strong> <em>26 </em>
                                    (4)
                                     , 662-676. <a href="https://doi.org/10.1002/pro.3129" title="DOI URL">https://doi.org/10.1002/pro.3129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pro.3129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpro.3129%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Science%26atitle%3DTargeting%252Bhuman%252BSET1%25252FMLL%252Bfamily%252Bof%252Bproteins%26aulast%3DVedadi%26aufirst%3DMasoud%26date%3D2017%26date%3D2017%26volume%3D26%26issue%3D4%26spage%3D662%26epage%3D676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Qi</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Justin  Gu</span>, <span class="hlFld-ContribAuthor ">Ying  Huang</span>, <span class="hlFld-ContribAuthor ">Youzhen  Wang</span>, <span class="hlFld-ContribAuthor ">Hailong  Zhang</span>, <span class="hlFld-ContribAuthor ">Man  Zhang</span>, <span class="hlFld-ContribAuthor ">Jeff  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhengtian  Yu</span>, <span class="hlFld-ContribAuthor ">Ling  Li</span>, <span class="hlFld-ContribAuthor ">Lin  Teng</span>, <span class="hlFld-ContribAuthor ">Shannon  Chuai</span>, <span class="hlFld-ContribAuthor ">Chao  Zhang</span>, <span class="hlFld-ContribAuthor ">Mengxi  Zhao</span>, <span class="hlFld-ContribAuthor ">HoMan  Chan</span>, <span class="hlFld-ContribAuthor ">Zijun  Chen</span>, <span class="hlFld-ContribAuthor ">Douglas  Fang</span>, <span class="hlFld-ContribAuthor ">Qi  Fei</span>, <span class="hlFld-ContribAuthor ">Leying  Feng</span>, <span class="hlFld-ContribAuthor ">Lijian  Feng</span>, <span class="hlFld-ContribAuthor ">Yuan  Gao</span>, <span class="hlFld-ContribAuthor ">Hui  Ge</span>, <span class="hlFld-ContribAuthor ">Xinjian  Ge</span>, <span class="hlFld-ContribAuthor ">Guobin  Li</span>, <span class="hlFld-ContribAuthor ">Andreas  Lingel</span>, <span class="hlFld-ContribAuthor ">Ying  Lin</span>, <span class="hlFld-ContribAuthor ">Yueqin  Liu</span>, <span class="hlFld-ContribAuthor ">Fangjun  Luo</span>, <span class="hlFld-ContribAuthor ">Minlong  Shi</span>, <span class="hlFld-ContribAuthor ">Long  Wang</span>, <span class="hlFld-ContribAuthor ">Zhaofu  Wang</span>, <span class="hlFld-ContribAuthor ">Yanyan  Yu</span>, <span class="hlFld-ContribAuthor ">Jue  Zeng</span>, <span class="hlFld-ContribAuthor ">Chenhui  Zeng</span>, <span class="hlFld-ContribAuthor ">Lijun  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiong  Zhang</span>, <span class="hlFld-ContribAuthor ">Shaolian  Zhou</span>, <span class="hlFld-ContribAuthor ">Counde  Oyang</span>, <span class="hlFld-ContribAuthor ">Peter  Atadja</span>, <span class="hlFld-ContribAuthor ">En  Li</span>. </span><span class="cited-content_cbyCitation_article-title">An allosteric PRC2 inhibitor targeting the H3K27me3 binding pocket of EED. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2017,</strong> <em>13 </em>
                                    (4)
                                     , 381-388. <a href="https://doi.org/10.1038/nchembio.2304" title="DOI URL">https://doi.org/10.1038/nchembio.2304</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nchembio.2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnchembio.2304%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DAn%252Ballosteric%252BPRC2%252Binhibitor%252Btargeting%252Bthe%252BH3K27me3%252Bbinding%252Bpocket%252Bof%252BEED%26aulast%3DQi%26aufirst%3DWei%26date%3D2017%26date%3D2017%26volume%3D13%26issue%3D4%26spage%3D381%26epage%3D388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lea  Albert</span>, <span class="hlFld-ContribAuthor ">Jing  Xu</span>, <span class="hlFld-ContribAuthor ">Ruiwei  Wan</span>, <span class="hlFld-ContribAuthor ">Vasundara  Srinivasan</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, <span class="hlFld-ContribAuthor ">Olalla  Vázquez</span>. </span><span class="cited-content_cbyCitation_article-title">Controlled inhibition of methyltransferases using photoswitchable peptidomimetics: towards an epigenetic regulation of leukemia. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2017,</strong> <em>8 </em>
                                    (6)
                                     , 4612-4618. <a href="https://doi.org/10.1039/C7SC00137A" title="DOI URL">https://doi.org/10.1039/C7SC00137A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7SC00137A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7SC00137A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DControlled%252Binhibition%252Bof%252Bmethyltransferases%252Busing%252Bphotoswitchable%252Bpeptidomimetics%25253A%252Btowards%252Ban%252Bepigenetic%252Bregulation%252Bof%252Bleukemia%26aulast%3DAlbert%26aufirst%3DLea%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D6%26spage%3D4612%26epage%3D4618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Yi-Yue  Xie</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based design of ester compounds to inhibit MLL complex catalytic activity by targeting mixed lineage leukemia 1 (MLL1)–WDR5 interaction. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (22)
                                     , 6109-6118. <a href="https://doi.org/10.1016/j.bmc.2016.09.073" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.09.073</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.09.073%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bdesign%252Bof%252Bester%252Bcompounds%252Bto%252Binhibit%252BMLL%252Bcomplex%252Bcatalytic%252Bactivity%252Bby%252Btargeting%252Bmixed%252Blineage%252Bleukemia%252B1%252B%252528MLL1%252529%2525E2%252580%252593WDR5%252Binteraction%26aulast%3DLi%26aufirst%3DDong-Dong%26date%3D2016%26volume%3D24%26issue%3D22%26spage%3D6109%26epage%3D6118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Yi-Yue  Xie</span>, <span class="hlFld-ContribAuthor ">Xiao-Li  Xu</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>, <span class="hlFld-ContribAuthor ">Xiao-Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>124 </em>, 480-489. <a href="https://doi.org/10.1016/j.ejmech.2016.08.036" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.08.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-affinity%252Bsmall%252Bmolecular%252Bblockers%252Bof%252Bmixed%252Blineage%252Bleukemia%252B1%252B%252528MLL1%252529-WDR5%252Binteraction%252Binhibit%252BMLL1%252Bcomplex%252BH3K4%252Bmethyltransferase%252Bactivity%26aulast%3DLi%26aufirst%3DDong-Dong%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nilda L.  Alicea-Velázquez</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Shinsky</span>, <span class="hlFld-ContribAuthor ">Daniel M.  Loh</span>, <span class="hlFld-ContribAuthor ">Jeong-Heon  Lee</span>, <span class="hlFld-ContribAuthor ">David G.  Skalnik</span>, <span class="hlFld-ContribAuthor ">Michael S.  Cosgrove</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted Disruption of the Interaction between WD-40 Repeat Protein 5 (WDR5) and Mixed Lineage Leukemia (MLL)/SET1 Family Proteins Specifically Inhibits MLL1 and SETd1A Methyltransferase Complexes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2016,</strong> <em>291 </em>
                                    (43)
                                     , 22357-22372. <a href="https://doi.org/10.1074/jbc.M116.752626" title="DOI URL">https://doi.org/10.1074/jbc.M116.752626</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M116.752626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M116.752626%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DTargeted%252BDisruption%252Bof%252Bthe%252BInteraction%252Bbetween%252BWD-40%252BRepeat%252BProtein%252B5%252B%252528WDR5%252529%252Band%252BMixed%252BLineage%252BLeukemia%252B%252528MLL%252529%25252FSET1%252BFamily%252BProteins%252BSpecifically%252BInhibits%252BMLL1%252Band%252BSETd1A%252BMethyltransferase%252BComplexes%26aulast%3DAlicea-Vel%25C3%25A1zquez%26aufirst%3DNilda%2BL.%26date%3D2016%26volume%3D291%26issue%3D43%26spage%3D22357%26epage%3D22372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Xiao-Li  Xu</span>, <span class="hlFld-ContribAuthor ">Fen  Jiang</span>, <span class="hlFld-ContribAuthor ">Lei  Wang</span>, <span class="hlFld-ContribAuthor ">Yi-Yue  Xie</span>, <span class="hlFld-ContribAuthor ">Xiao-Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Hao-Peng  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based design and synthesis of small molecular inhibitors disturbing the interaction of MLL1-WDR5. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>118 </em>, 1-8. <a href="https://doi.org/10.1016/j.ejmech.2016.04.032" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.04.032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.04.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.04.032%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bdesign%252Band%252Bsynthesis%252Bof%252Bsmall%252Bmolecular%252Binhibitors%252Bdisturbing%252Bthe%252Binteraction%252Bof%252BMLL1-WDR5%26aulast%3DLi%26aufirst%3DDong-Dong%26date%3D2016%26volume%3D118%26spage%3D1%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tobias  Wagner</span>, <span class="hlFld-ContribAuthor ">Holger  Greschik</span>, <span class="hlFld-ContribAuthor ">Teresa  Burgahn</span>, <span class="hlFld-ContribAuthor ">Karin  Schmidtkunz</span>, <span class="hlFld-ContribAuthor ">Anne-Kathrin  Schott</span>, <span class="hlFld-ContribAuthor ">Joel  McMillan</span>, <span class="hlFld-ContribAuthor ">Lina  Baranauskienė</span>, <span class="hlFld-ContribAuthor ">Yan  Xiong</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>, <span class="hlFld-ContribAuthor ">Udo  Oppermann</span>, <span class="hlFld-ContribAuthor ">Daumantas  Matulis</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a small-molecule ligand of the epigenetic reader protein Spindlin1 via a versatile screening platform. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2016,</strong> <em>44 </em>
                                    (9)
                                     , e88-e88. <a href="https://doi.org/10.1093/nar/gkw089" title="DOI URL">https://doi.org/10.1093/nar/gkw089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkw089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkw089%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DIdentification%252Bof%252Ba%252Bsmall-molecule%252Bligand%252Bof%252Bthe%252Bepigenetic%252Breader%252Bprotein%252BSpindlin1%252Bvia%252Ba%252Bversatile%252Bscreening%252Bplatform%26aulast%3DWagner%26aufirst%3DTobias%26date%3D2016%26date%3D2016%26volume%3D44%26issue%3D9%26spage%3De88%26epage%3De88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiajun  Zhu</span>, <span class="hlFld-ContribAuthor ">Morgan A.  Sammons</span>, <span class="hlFld-ContribAuthor ">Greg  Donahue</span>, <span class="hlFld-ContribAuthor ">Zhixun  Dou</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>, <span class="hlFld-ContribAuthor ">Matthäus  Getlik</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Rima  Al-awar</span>, <span class="hlFld-ContribAuthor ">Bryson W.  Katona</span>, <span class="hlFld-ContribAuthor ">Ali  Shilatifard</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Xianxin  Hua</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Shelley L.  Berger</span>. </span><span class="cited-content_cbyCitation_article-title">Gain-of-function p53 mutants co-opt chromatin pathways to drive cancer growth. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2015,</strong> <em>525 </em>
                                    (7568)
                                     , 206-211. <a href="https://doi.org/10.1038/nature15251" title="DOI URL">https://doi.org/10.1038/nature15251</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nature15251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnature15251%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DGain-of-function%252Bp53%252Bmutants%252Bco-opt%252Bchromatin%252Bpathways%252Bto%252Bdrive%252Bcancer%252Bgrowth%26aulast%3DZhu%26aufirst%3DJiajun%26date%3D2015%26date%3D2015%26volume%3D525%26issue%3D7568%26spage%3D206%26epage%3D211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajesh C.  Rao</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>. </span><span class="cited-content_cbyCitation_article-title">Hijacked in cancer: the KMT2 (MLL) family of methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2015,</strong> <em>15 </em>
                                    (6)
                                     , 334-346. <a href="https://doi.org/10.1038/nrc3929" title="DOI URL">https://doi.org/10.1038/nrc3929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrc3929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrc3929%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DHijacked%252Bin%252Bcancer%25253A%252Bthe%252BKMT2%252B%252528MLL%252529%252Bfamily%252Bof%252Bmethyltransferases%26aulast%3DRao%26aufirst%3DRajesh%2BC.%26date%3D2015%26date%3D2015%26volume%3D15%26issue%3D6%26spage%3D334%26epage%3D346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Chopra</span>, <span class="hlFld-ContribAuthor ">Stefan K.  Bohlander</span>. </span><span class="cited-content_cbyCitation_article-title">Disturbing the histone code in leukemia: translocations and mutations affecting histone methyl transferases. </span><span class="cited-content_cbyCitation_journal-name">Cancer Genetics</span><span> <strong>2015,</strong> <em>208 </em>
                                    (5)
                                     , 192-205. <a href="https://doi.org/10.1016/j.cancergen.2014.10.005" title="DOI URL">https://doi.org/10.1016/j.cancergen.2014.10.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cancergen.2014.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cancergen.2014.10.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Genetics%26atitle%3DDisturbing%252Bthe%252Bhistone%252Bcode%252Bin%252Bleukemia%25253A%252Btranslocations%252Band%252Bmutations%252Baffecting%252Bhistone%252Bmethyl%252Btransferases%26aulast%3DChopra%26aufirst%3DMartin%26date%3D2015%26volume%3D208%26issue%3D5%26spage%3D192%26epage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joseph J.  Ferry</span>, <span class="hlFld-ContribAuthor ">Robert F.  Smith</span>, <span class="hlFld-ContribAuthor ">Natalie  Denney</span>, <span class="hlFld-ContribAuthor ">Colin P.  Walsh</span>, <span class="hlFld-ContribAuthor ">Lauren  McCauley</span>, <span class="hlFld-ContribAuthor ">Jie  Qian</span>, <span class="hlFld-ContribAuthor ">Haiching  Ma</span>, <span class="hlFld-ContribAuthor ">Kurumi Y.  Horiuchi</span>, <span class="hlFld-ContribAuthor ">Konrad T.  Howitz</span>. </span><span class="cited-content_cbyCitation_article-title">Development and Use of Assay Conditions Suited to Screening for and Profiling of SET-Domain-Targeted Inhibitors of the MLL/SET1 Family of Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">ASSAY and Drug Development Technologies</span><span> <strong>2015,</strong> <em>13 </em>
                                    (4)
                                     , 221-234. <a href="https://doi.org/10.1089/adt.2015.646" title="DOI URL">https://doi.org/10.1089/adt.2015.646</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/adt.2015.646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fadt.2015.646%26sid%3Dliteratum%253Aachs%26jtitle%3DASSAY%2520and%2520Drug%2520Development%2520Technologies%26atitle%3DDevelopment%252Band%252BUse%252Bof%252BAssay%252BConditions%252BSuited%252Bto%252BScreening%252Bfor%252Band%252BProfiling%252Bof%252BSET-Domain-Targeted%252BInhibitors%252Bof%252Bthe%252BMLL%25252FSET1%252BFamily%252Bof%252BLysine%252BMethyltransferases%26aulast%3DFerry%26aufirst%3DJoseph%2BJ.%26date%3D2015%26volume%3D13%26issue%3D4%26spage%3D221%26epage%3D234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hacer  Karatas</span>, <span class="hlFld-ContribAuthor ">Shirley Y.  Lee</span>, <span class="hlFld-ContribAuthor ">Elizabeth C.  Townsend</span>, <span class="hlFld-ContribAuthor ">Fang  Cao</span>, <span class="hlFld-ContribAuthor ">Jing  Xu</span>, <span class="hlFld-ContribAuthor ">Denzil  Bernard</span>, <span class="hlFld-ContribAuthor ">Liu  Liu</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based design of conformationally constrained cyclic peptidomimetics to target the MLL1-WDR5 protein–protein interaction as inhibitors of the MLL1 methyltransferase activity. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2015,</strong> <em>26 </em>
                                    (4)
                                     , 455-458. <a href="https://doi.org/10.1016/j.cclet.2015.03.030" title="DOI URL">https://doi.org/10.1016/j.cclet.2015.03.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2015.03.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2015.03.030%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DStructure-based%252Bdesign%252Bof%252Bconformationally%252Bconstrained%252Bcyclic%252Bpeptidomimetics%252Bto%252Btarget%252Bthe%252BMLL1-WDR5%252Bprotein%2525E2%252580%252593protein%252Binteraction%252Bas%252Binhibitors%252Bof%252Bthe%252BMLL1%252Bmethyltransferase%252Bactivity%26aulast%3DKaratas%26aufirst%3DHacer%26date%3D2015%26volume%3D26%26issue%3D4%26spage%3D455%26epage%3D458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein-protein interactions in hematologic malignancies: still a challenge or a great opportunity for future therapies?. </span><span class="cited-content_cbyCitation_journal-name">Immunological Reviews</span><span> <strong>2015,</strong> <em>263 </em>
                                    (1)
                                     , 279-301. <a href="https://doi.org/10.1111/imr.12244" title="DOI URL">https://doi.org/10.1111/imr.12244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/imr.12244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fimr.12244%26sid%3Dliteratum%253Aachs%26jtitle%3DImmunological%2520Reviews%26atitle%3DTargeting%252Bprotein-protein%252Binteractions%252Bin%252Bhematologic%252Bmalignancies%25253A%252Bstill%252Ba%252Bchallenge%252Bor%252Ba%252Bgreat%252Bopportunity%252Bfor%252Bfuture%252Btherapies%25253F%26aulast%3DCierpicki%26aufirst%3DTomasz%26date%3D2015%26date%3D2014%26volume%3D263%26issue%3D1%26spage%3D279%26epage%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guillermo  Senisterra</span>, <span class="hlFld-ContribAuthor ">Hong  Wu</span>, <span class="hlFld-ContribAuthor ">Abdellah  Allali-Hassani</span>, <span class="hlFld-ContribAuthor ">Gregory A.  Wasney</span>, <span class="hlFld-ContribAuthor ">Dalia  Barsyte-Lovejoy</span>, <span class="hlFld-ContribAuthor ">Ludmila  Dombrovski</span>, <span class="hlFld-ContribAuthor ">Aiping  Dong</span>, <span class="hlFld-ContribAuthor ">Kong T.  Nguyen</span>, <span class="hlFld-ContribAuthor ">David  Smil</span>, <span class="hlFld-ContribAuthor ">Yuri  Bolshan</span>, <span class="hlFld-ContribAuthor ">Taraneh  Hajian</span>, <span class="hlFld-ContribAuthor ">Hao  He</span>, <span class="hlFld-ContribAuthor ">Alma  Seitova</span>, <span class="hlFld-ContribAuthor ">Irene  Chau</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Gennadiy  Poda</span>, <span class="hlFld-ContribAuthor ">Jean-François  Couture</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Rima  Al-Awar</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule inhibition of MLL activity by disruption of its interaction with WDR5. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2013,</strong> <em>449 </em>
                                    (1)
                                     , 151-159. <a href="https://doi.org/10.1042/BJ20121280" title="DOI URL">https://doi.org/10.1042/BJ20121280</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BJ20121280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBJ20121280%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DSmall-molecule%252Binhibition%252Bof%252BMLL%252Bactivity%252Bby%252Bdisruption%252Bof%252Bits%252Binteraction%252Bwith%252BWDR5%26aulast%3DSenisterra%26aufirst%3DGuillermo%26date%3D2013%26date%3D2012%26volume%3D449%26issue%3D1%26spage%3D151%26epage%3D159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yong  Ding</span>, <span class="hlFld-ContribAuthor ">Ivan  Ndamukong</span>, <span class="hlFld-ContribAuthor ">Zaoshi  Xu</span>, <span class="hlFld-ContribAuthor ">Hanna  Lapko</span>, <span class="hlFld-ContribAuthor ">Michael  Fromm</span>, <span class="hlFld-ContribAuthor ">Zoya  Avramova</span>, . </span><span class="cited-content_cbyCitation_article-title">ATX1-Generated H3K4me3 Is Required for Efficient Elongation of Transcription, Not Initiation, at ATX1-Regulated Genes. </span><span class="cited-content_cbyCitation_journal-name">PLoS Genetics</span><span> <strong>2012,</strong> <em>8 </em>
                                    (12)
                                     , e1003111. <a href="https://doi.org/10.1371/journal.pgen.1003111" title="DOI URL">https://doi.org/10.1371/journal.pgen.1003111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pgen.1003111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pgen.1003111%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520Genetics%26atitle%3DATX1-Generated%252BH3K4me3%252BIs%252BRequired%252Bfor%252BEfficient%252BElongation%252Bof%252BTranscription%25252C%252BNot%252BInitiation%25252C%252Bat%252BATX1-Regulated%252BGenes%26aulast%3DDing%26aufirst%3DYong%26date%3D2012%26date%3D2012%26volume%3D8%26issue%3D12%26spage%3De1003111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erica  Ballabio</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Milne</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular and Epigenetic Mechanisms of MLL in Human Leukemogenesis. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2012,</strong> <em>4 </em>
                                    (3)
                                     , 904-944. <a href="https://doi.org/10.3390/cancers4030904" title="DOI URL">https://doi.org/10.3390/cancers4030904</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers4030904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers4030904%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DMolecular%252Band%252BEpigenetic%252BMechanisms%252Bof%252BMLL%252Bin%252BHuman%252BLeukemogenesis%26aulast%3DBallabio%26aufirst%3DErica%26date%3D2012%26date%3D2012%26volume%3D4%26issue%3D3%26spage%3D904%26epage%3D944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Venkatasubramanian  Dharmarajan</span>, <span class="hlFld-ContribAuthor ">Jeong-Heon  Lee</span>, <span class="hlFld-ContribAuthor ">Anamika  Patel</span>, <span class="hlFld-ContribAuthor ">David G.  Skalnik</span>, <span class="hlFld-ContribAuthor ">Michael S.  Cosgrove</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Basis for WDR5 Interaction (Win) Motif Recognition in Human SET1 Family Histone Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2012,</strong> <em>287 </em>
                                    (33)
                                     , 27275-27289. <a href="https://doi.org/10.1074/jbc.M112.364125" title="DOI URL">https://doi.org/10.1074/jbc.M112.364125</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M112.364125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M112.364125%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DStructural%252BBasis%252Bfor%252BWDR5%252BInteraction%252B%252528Win%252529%252BMotif%252BRecognition%252Bin%252BHuman%252BSET1%252BFamily%252BHistone%252BMethyltransferases%26aulast%3DDharmarajan%26aufirst%3DVenkatasubramanian%26date%3D2012%26volume%3D287%26issue%3D33%26spage%3D27275%26epage%3D27289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Irfan A.  Qureshi</span>, <span class="hlFld-ContribAuthor ">Mark F.  Mehler</span>. </span><span class="cited-content_cbyCitation_article-title">Chromatin-Modifying Agents for Epigenetic Reprogramming and Endogenous Neural Stem Cell-Mediated Repair in Stroke. </span><span class="cited-content_cbyCitation_journal-name">Translational Stroke Research</span><span> <strong>2011,</strong> <em>2 </em>
                                    (1)
                                     , 7-16. <a href="https://doi.org/10.1007/s12975-010-0051-3" title="DOI URL">https://doi.org/10.1007/s12975-010-0051-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12975-010-0051-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12975-010-0051-3%26sid%3Dliteratum%253Aachs%26jtitle%3DTranslational%2520Stroke%2520Research%26atitle%3DChromatin-Modifying%252BAgents%252Bfor%252BEpigenetic%252BReprogramming%252Band%252BEndogenous%252BNeural%252BStem%252BCell-Mediated%252BRepair%252Bin%252BStroke%26aulast%3DQureshi%26aufirst%3DIrfan%2BA.%26date%3D2011%26date%3D2010%26volume%3D2%26issue%3D1%26spage%3D7%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vanja  Avdic</span>, <span class="hlFld-ContribAuthor ">Pamela  Zhang</span>, <span class="hlFld-ContribAuthor ">Sylvain  Lanouette</span>, <span class="hlFld-ContribAuthor ">Anastassia  Voronova</span>, <span class="hlFld-ContribAuthor ">Ilona  Skerjanc</span>, <span class="hlFld-ContribAuthor ">Jean‐Francois  Couture</span>. </span><span class="cited-content_cbyCitation_article-title">Fine‐tuning the stimulation of MLL1 methyltransferase activity by a histone H3‐based peptide mimetic. </span><span class="cited-content_cbyCitation_journal-name">The FASEB Journal</span><span> <strong>2011,</strong> <em>25 </em>
                                    (3)
                                     , 960-967. <a href="https://doi.org/10.1096/fj.10-171959" title="DOI URL">https://doi.org/10.1096/fj.10-171959</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1096/fj.10-171959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1096%2Ffj.10-171959%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FASEB%2520Journal%26atitle%3DFine%2525E2%252580%252590tuning%252Bthe%252Bstimulation%252Bof%252BMLL1%252Bmethyltransferase%252Bactivity%252Bby%252Ba%252Bhistone%252BH3%2525E2%252580%252590based%252Bpeptide%252Bmimetic%26aulast%3DAvdic%26aufirst%3DVanja%26date%3D2011%26date%3D2010%26volume%3D25%26issue%3D3%26spage%3D960%26epage%3D967" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding models for Ac-ARA-NH<sub>2</sub> and H<sub>2</sub>N-ARA-NH<sub>2</sub> in complex with WDR5 and key intramolecular hydrogen bonds within these peptides. (a) Hydrogen bonds 1 and 2 and the atoms involved in Ac-ARA-NH<sub>2</sub> peptide. (b) Predicted binding model of Ac-ARA-NH<sub>2</sub> in complex with WDR5. (c) Predicted binding model of H<sub>2</sub>N-ARA-NH<sub>2</sub> in complex with WDR5. (b,c) The intramolecular hydrogen bonds are shown as yellow dots. Carbon atoms are shown in cyan in the peptide and gray in WDR5. The nitrogen and oxygen atoms are colored in blue and red, respectively. (d) Probability of the hydrogen bond distance between the nitrogen and oxygen pairs in hydrogen bonds 1 and 2 in Ac-ARA-NH<sub>2</sub> based upon MD simulation. (e) Probability of the hydrogen bond distance between the nitrogen and oxygen pairs in hydrogen bond 2 in NH2-ARA-NH<sub>2</sub> based upon MD simulation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Saturation binding experiments of tracers with WDR5. (a) Saturation curves of WIN-FAM-1, WIN-FAM-2, 10mer-Thr-FAM, and 10mer-Ala-FAM with WDR5; 0.6 nM of the tracer was used for these experiments. (b) Stability of the saturation binding experiment for 10mer-Thr-FAM tracer over 24 h. mP values were measured at the times indicated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/medium/jm-2010-00139b_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Competitive binding curves for Ac-10mer, Ac-3mer, WIN, and H<sub>2</sub>N-10mer as determined using a fluorescence polarization based assay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2010/jmcmar.2010.53.issue-14/jm100139b/production/images/large/jm-2010-00139b_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm100139b&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i30">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44147" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44147" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 34 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0ljQTWoki4HxUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0ljAnyTxNsz-fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Shilatifard, A.</span><span> </span><span class="NLM_article-title">Molecular implementation and physiological roles for histone H3 lysine 4 (H3K4) methylation</span> <span class="citation_source-journal">Curr. Opin. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">–</span> <span class="NLM_lpage">348</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.ceb.2008.03.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=18508253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2008&pages=341-348&author=A.+Shilatifard&title=Molecular+implementation+and+physiological+roles+for+histone+H3+lysine+4+%28H3K4%29+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular implementation and physiological roles for histone H3 lysine 4 (H3K4) methylation</span></div><div class="casAuthors">Shilatifard, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">341-348</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Chromosomal surfaces are ornamented with a variety of post-translational modifications of histones, which are required for the regulation of many of the DNA-templated processes.  Such histone modifications include acetylation, sumoylation, phosphorylation, ubiquitination, and methylation.  Histone modifications can either function by disrupting chromosomal contacts or by regulating non-histone protein interactions with chromatin.  In this review, recent findings will be discussed regarding the regulation of the implementation and physiol. significance for one such histone modification, histone H3 lysine 4 (H3K4) methylation by the yeast COMPASS and mammalian COMPASS-like complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuhSAfNqAX67Vg90H21EOLACvtfcHk0ljAnyTxNsz-fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmvV2lsL0%253D&md5=c797a57ac3102c7bc78dc3773b5283b6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2008.03.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2008.03.019%26sid%3Dliteratum%253Aachs%26aulast%3DShilatifard%26aufirst%3DA.%26atitle%3DMolecular%2520implementation%2520and%2520physiological%2520roles%2520for%2520histone%2520H3%2520lysine%25204%2520%2528H3K4%2529%2520methylation%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2008%26volume%3D20%26spage%3D341%26epage%3D348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Sims, R. J.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Histone H3 Lys 4 methylation: caught in a bind?</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2779</span><span class="NLM_x">–</span> <span class="NLM_lpage">2786</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=2779-2786&author=R.+J.+Simsauthor=D.+Reinberg&title=Histone+H3+Lys+4+methylation%3A+caught+in+a+bind%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DHistone%2520H3%2520Lys%25204%2520methylation%253A%2520caught%2520in%2520a%2520bind%253F%26jtitle%3DGenes%2520Dev.%26date%3D2006%26volume%3D20%26spage%3D2779%26epage%3D2786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Wysocka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swigut, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landry, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tackett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chait, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badenhorst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodelling</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">442</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnature04815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16728976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsFWnt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=442&publication_year=2006&pages=86-90&author=J.+Wysockaauthor=T.+Swigutauthor=H.+Xiaoauthor=T.+A.+Milneauthor=S.+Y.+Kwonauthor=J.+Landryauthor=M.+Kauerauthor=A.+J.+Tackettauthor=B.+T.+Chaitauthor=P.+Badenhorstauthor=C.+Wuauthor=C.+D.+Allis&title=A+PHD+finger+of+NURF+couples+histone+H3+lysine+4+trimethylation+with+chromatin+remodelling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A PHD finger of NURF couples histone H3 lysine 4 trimethylation with chromatin remodeling</span></div><div class="casAuthors">Wysocka, Joanna; Swigut, Tomek; Xiao, Hua; Milne, Thomas A.; Kwon, So Yeon; Landry, Joe; Kauer, Monika; Tackett, Alan J.; Chait, Brian T.; Badenhorst, Paul; Wu, Carl; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">442</span>
        (<span class="NLM_cas:issue">7098</span>),
    <span class="NLM_cas:pages">86-90</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Lysine methylation of histones is recognized as an important component of an epigenetic indexing system demarcating transcriptionally active and inactive chromatin domains.  Trimethylation of histone H3 lysine 4 (H3K4me3) marks transcription start sites of virtually all active genes.  Recently, we reported that the WD40-repeat protein WDR5 is important for global levels of H3K4me3 and control of HOX gene expression.  Here we show that a plant homeodomain (PHD) finger of nucleosome remodelling factor (NURF), an ISWI-contg. ATP-dependent chromatin-remodelling complex, mediates a direct preferential assocn. with H3K4me3 tails.  Depletion of H3K4me3 causes partial release of the NURF subunit, BPTF (bromodomain and PHD finger transcription factor), from chromatin and defective recruitment of the assocd. ATPase, SNF2L (also known as ISWI and SMARCA1), to the HOXC8 promoter.  Loss of BPTF in Xenopus embryos mimics WDR5 loss-of-function phenotypes, and compromises spatial control of Hox gene expression.  These results strongly suggest that WDR5 and NURF function in a common biol. pathway in vivo, and that NURF-mediated ATP-dependent chromatin remodelling is directly coupled to H3K4 trimethylation to maintain Hox gene expression patterns during development.  We also identify a previously unknown function for the PHD finger as a highly specialized methyl-lysine-binding domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrypvJ5nCemXbVg90H21EOLACvtfcHk0ljAnyTxNsz-fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsFWnt7g%253D&md5=87bc9ef9cab5a400a9b92669917e4439</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature04815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04815%26sid%3Dliteratum%253Aachs%26aulast%3DWysocka%26aufirst%3DJ.%26aulast%3DSwigut%26aufirst%3DT.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DKwon%26aufirst%3DS.%2BY.%26aulast%3DLandry%26aufirst%3DJ.%26aulast%3DKauer%26aufirst%3DM.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DChait%26aufirst%3DB.%2BT.%26aulast%3DBadenhorst%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DA%2520PHD%2520finger%2520of%2520NURF%2520couples%2520histone%2520H3%2520lysine%25204%2520trimethylation%2520with%2520chromatin%2520remodelling%26jtitle%3DNature%26date%3D2006%26volume%3D442%26spage%3D86%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Huntsman, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muleris, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batley, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, V. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedemann, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldas, C.</span><span> </span><span class="NLM_article-title">MLL2, the second human homolog of the <i>Drosophila trithorax</i> gene, maps to 19q13.1 and is amplified in solid tumor cell lines</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">7975</span><span class="NLM_x">–</span> <span class="NLM_lpage">7984</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=7975-7984&author=D.+G.+Huntsmanauthor=S.+F.+Chinauthor=M.+Mulerisauthor=S.+J.+Batleyauthor=V.+P.+Collinsauthor=L.+M.+Wiedemannauthor=S.+Aparicioauthor=C.+Caldas&title=MLL2%2C+the+second+human+homolog+of+the+Drosophila+trithorax+gene%2C+maps+to+19q13.1+and+is+amplified+in+solid+tumor+cell+lines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuntsman%26aufirst%3DD.%2BG.%26aulast%3DChin%26aufirst%3DS.%2BF.%26aulast%3DMuleris%26aufirst%3DM.%26aulast%3DBatley%26aufirst%3DS.%2BJ.%26aulast%3DCollins%26aufirst%3DV.%2BP.%26aulast%3DWiedemann%26aufirst%3DL.%2BM.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DCaldas%26aufirst%3DC.%26atitle%3DMLL2%252C%2520the%2520second%2520human%2520homolog%2520of%2520the%2520Drosophila%2520trithorax%2520gene%252C%2520maps%2520to%252019q13.1%2520and%2520is%2520amplified%2520in%2520solid%2520tumor%2520cell%2520lines%26jtitle%3DOncogene%26date%3D1999%26volume%3D18%26spage%3D7975%26epage%3D7984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Ruault, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brun, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ventura, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roizes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Sario, A.</span><span> </span><span class="NLM_article-title">MLL3, a new human member of the TRX/MLL gene family, maps to 7q36, a chromosome region frequently deleted in myeloid leukaemia</span> <span class="citation_source-journal">Gene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2002&pages=73-81&author=M.+Ruaultauthor=M.+E.+Brunauthor=M.+Venturaauthor=G.+Roizesauthor=A.+De+Sario&title=MLL3%2C+a+new+human+member+of+the+TRX%2FMLL+gene+family%2C+maps+to+7q36%2C+a+chromosome+region+frequently+deleted+in+myeloid+leukaemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRuault%26aufirst%3DM.%26aulast%3DBrun%26aufirst%3DM.%2BE.%26aulast%3DVentura%26aufirst%3DM.%26aulast%3DRoizes%26aufirst%3DG.%26aulast%3DDe%2BSario%26aufirst%3DA.%26atitle%3DMLL3%252C%2520a%2520new%2520human%2520member%2520of%2520the%2520TRX%252FMLL%2520gene%2520family%252C%2520maps%2520to%25207q36%252C%2520a%2520chromosome%2520region%2520frequently%2520deleted%2520in%2520myeloid%2520leukaemia%26jtitle%3DGene%26date%3D2002%26volume%3D284%26spage%3D73%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL: a histone methyltransferase disrupted in leukemia</span> <span class="citation_source-journal">Trends Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">500</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=500-507&author=J.+L.+Hess&title=MLL%3A+a+histone+methyltransferase+disrupted+in+leukemia"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%253A%2520a%2520histone%2520methyltransferase%2520disrupted%2520in%2520leukemia%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2004%26volume%3D10%26spage%3D500%26epage%3D507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Guenther, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenner, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chevalier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croce, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canaani, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span> </span><span class="NLM_article-title">Global and Hox-specific roles for the MLL1 methyltransferase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">8603</span><span class="NLM_x">–</span> <span class="NLM_lpage">8608</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=8603-8608&author=M.+G.+Guentherauthor=R.+G.+Jennerauthor=B.+Chevalierauthor=T.+Nakamuraauthor=C.+M.+Croceauthor=E.+Canaaniauthor=R.+A.+Young&title=Global+and+Hox-specific+roles+for+the+MLL1+methyltransferase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGuenther%26aufirst%3DM.%2BG.%26aulast%3DJenner%26aufirst%3DR.%2BG.%26aulast%3DChevalier%26aufirst%3DB.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCanaani%26aufirst%3DE.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DGlobal%2520and%2520Hox-specific%2520roles%2520for%2520the%2520MLL1%2520methyltransferase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A%26date%3D2005%26volume%3D102%26spage%3D8603%26epage%3D8608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Mishra, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ansari, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mandal, S. S.</span><span> </span><span class="NLM_article-title">Dynamic association of MLL1, H3K4 trimethylation with chromatin and Hox gene expression during the cell cycle</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">1629</span><span class="NLM_x">–</span> <span class="NLM_lpage">1640</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2009&pages=1629-1640&author=B.+P.+Mishraauthor=K.+I.+Ansariauthor=S.+S.+Mandal&title=Dynamic+association+of+MLL1%2C+H3K4+trimethylation+with+chromatin+and+Hox+gene+expression+during+the+cell+cycle"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMishra%26aufirst%3DB.%2BP.%26aulast%3DAnsari%26aufirst%3DK.%2BI.%26aulast%3DMandal%26aufirst%3DS.%2BS.%26atitle%3DDynamic%2520association%2520of%2520MLL1%252C%2520H3K4%2520trimethylation%2520with%2520chromatin%2520and%2520Hox%2520gene%2520expression%2520during%2520the%2520cell%2520cycle%26jtitle%3DFEBS%2520J.%26date%3D2009%26volume%3D276%26spage%3D1629%26epage%3D1640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Hombria, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovegrove, B.</span><span> </span><span class="NLM_article-title">Beyond homeosis—HOX function in morphogenesis and organogenesis</span> <span class="citation_source-journal">Differentiation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2003&pages=461-476&author=J.+C.+Hombriaauthor=B.+Lovegrove&title=Beyond+homeosis%E2%80%94HOX+function+in+morphogenesis+and+organogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHombria%26aufirst%3DJ.%2BC.%26aulast%3DLovegrove%26aufirst%3DB.%26atitle%3DBeyond%2520homeosis%25E2%2580%2594HOX%2520function%2520in%2520morphogenesis%2520and%2520organogenesis%26jtitle%3DDifferentiation%26date%3D2003%26volume%3D71%26spage%3D461%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Monier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tevy, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capovilla, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semeriva, M.</span><span> </span><span class="NLM_article-title">Downstream of homeotic genes: in the heart of Hox function</span> <span class="citation_source-journal">Fly (Austin)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A280%3ADC%252BD1cnjtVOhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=59-67&author=B.+Monierauthor=M.+F.+Tevyauthor=L.+Perrinauthor=M.+Capovillaauthor=M.+Semeriva&title=Downstream+of+homeotic+genes%3A+in+the+heart+of+Hox+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Downstream of homeotic genes: in the heart of Hox function</span></div><div class="casAuthors">Monier Bruno; Tevy Maria Florencia; Perrin Laurent; Capovilla Maria; Semeriva Michel</div><div class="citationInfo"><span class="NLM_cas:title">Fly</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">59-67</span>
        ISSN:<span class="NLM_cas:issn">1933-6934</span>.
    </div><div class="casAbstract">A functional organ is constituted of diverse cell types.  Each one occupies a distinct position and is associated to specific morphological and physiological functions.  The identification of the genetic programs controlling these elaborated and highly precise features of organogenesis is crucial to understand how a mature organ works under normal conditions, and how pathologies can develop.  Recently, a number of studies have reported a critical role for Hox genes in one example of organogenesis: cardiogenesis in Drosophila.  Beyond the interest in understanding the molecular basis of functional cardiogenesis, this system might provide a model for proposing new paradigms of how Hox genes achieve their action throughout development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_VmBzmR4aGKx48Rj5GnGofW6udTcc2eZhE7E7KxViXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cnjtVOhtQ%253D%253D&md5=abcb2cb1383b1532dd001cd45b7dc493</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMonier%26aufirst%3DB.%26aulast%3DTevy%26aufirst%3DM.%2BF.%26aulast%3DPerrin%26aufirst%3DL.%26aulast%3DCapovilla%26aufirst%3DM.%26aulast%3DSemeriva%26aufirst%3DM.%26atitle%3DDownstream%2520of%2520homeotic%2520genes%253A%2520in%2520the%2520heart%2520of%2520Hox%2520function%26jtitle%3DFly%2520%2528Austin%2529%26date%3D2007%26volume%3D1%26spage%3D59%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Jude, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Climer, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artinger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span> </span><span class="NLM_article-title">Unique and independent roles for MLL in adult hematopoietic stem cells and progenitors</span> <span class="citation_source-journal">Cell Stem Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">324</span><span class="NLM_x">–</span> <span class="NLM_lpage">337</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=324-337&author=C.+D.+Judeauthor=L.+Climerauthor=D.+Xuauthor=E.+Artingerauthor=J.+K.+Fisherauthor=P.+Ernst&title=Unique+and+independent+roles+for+MLL+in+adult+hematopoietic+stem+cells+and+progenitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJude%26aufirst%3DC.%2BD.%26aulast%3DClimer%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DD.%26aulast%3DArtinger%26aufirst%3DE.%26aulast%3DFisher%26aufirst%3DJ.%2BK.%26aulast%3DErnst%26aufirst%3DP.%26atitle%3DUnique%2520and%2520independent%2520roles%2520for%2520MLL%2520in%2520adult%2520hematopoietic%2520stem%2520cells%2520and%2520progenitors%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2007%26volume%3D1%26spage%3D324%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Ferrando, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neuberg, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sallan, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, A. T.</span><span> </span><span class="NLM_article-title">Gene expression signatures in MLL-rearranged T-lineage and B-precursor acute leukemias: dominance of HOX dysregulation</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">262</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2003&pages=262-268&author=A.+A.+Ferrandoauthor=S.+A.+Armstrongauthor=D.+S.+Neubergauthor=S.+E.+Sallanauthor=L.+B.+Silvermanauthor=S.+J.+Korsmeyerauthor=A.+T.+Look&title=Gene+expression+signatures+in+MLL-rearranged+T-lineage+and+B-precursor+acute+leukemias%3A+dominance+of+HOX+dysregulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DNeuberg%26aufirst%3DD.%2BS.%26aulast%3DSallan%26aufirst%3DS.%2BE.%26aulast%3DSilverman%26aufirst%3DL.%2BB.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DLook%26aufirst%3DA.%2BT.%26atitle%3DGene%2520expression%2520signatures%2520in%2520MLL-rearranged%2520T-lineage%2520and%2520B-precursor%2520acute%2520leukemias%253A%2520dominance%2520of%2520HOX%2520dysregulation%26jtitle%3DBlood%26date%3D2003%26volume%3D102%26spage%3D262%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Harper, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aplan, P. D.</span><span> </span><span class="NLM_article-title">Chromosomal rearrangements leading to MLL gene fusions: clinical and biological aspects</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">10024</span><span class="NLM_x">–</span> <span class="NLM_lpage">10027</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=10024-10027&author=D.+P.+Harperauthor=P.+D.+Aplan&title=Chromosomal+rearrangements+leading+to+MLL+gene+fusions%3A+clinical+and+biological+aspects"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DD.%2BP.%26aulast%3DAplan%26aufirst%3DP.%2BD.%26atitle%3DChromosomal%2520rearrangements%2520leading%2520to%2520MLL%2520gene%2520fusions%253A%2520clinical%2520and%2520biological%2520aspects%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D10024%26epage%3D10027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Argiropoulos, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span> </span><span class="NLM_article-title">Hox genes in hematopoiesis and leukemogenesis</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">6766</span><span class="NLM_x">–</span> <span class="NLM_lpage">6776</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=6766-6776&author=B.+Argiropoulosauthor=R.+K.+Humphries&title=Hox+genes+in+hematopoiesis+and+leukemogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DArgiropoulos%26aufirst%3DB.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26atitle%3DHox%2520genes%2520in%2520hematopoiesis%2520and%2520leukemogenesis%26jtitle%3DOncogene%26date%3D2007%26volume%3D26%26spage%3D6766%26epage%3D6776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Maulbecker, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruss, P.</span><span> </span><span class="NLM_article-title">The oncogenic potential of deregulated homeobox genes</span> <span class="citation_source-journal">Cell Growth Differ.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">431</span><span class="NLM_x">–</span> <span class="NLM_lpage">441</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=8100147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADyaK3sXmsVKqs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1993&pages=431-441&author=C.+C.+Maulbeckerauthor=P.+Gruss&title=The+oncogenic+potential+of+deregulated+homeobox+genes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The oncogenic potential of deregulated homeobox genes</span></div><div class="casAuthors">Maulbecker, Catharina C.; Gruss, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Cell Growth & Differentiation</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">431-41</span>CODEN:
                <span class="NLM_cas:coden">CGDIE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-9523</span>.
    </div><div class="casAbstract">Homeobox genes encode DNA-binding proteins that regulate gene expression during embryonic development.  Deregulation of homeobox gene expression was invoked as the mol. basis of a no. of leukemias.  The authors now demonstrate, using in vitro and in vivo transformation assays, that the overexpression of homeobox genes is the basis for transformation and tumorigenesis.  Thus, clustered homeobox genes related to Drosophila Hom genes (Hoxa-7, Hoxa-5, Hoxa-1, Hoxb-7, and Hoxc-8) as well as the homeobox genes Evx-1 and Cdx-1 constitute a new family of nuclear protooncogenes.  Apparently, many homeobox genes regulating embryogenesis can - if deregulated - also contribute to tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCBoX-SR6PC7Vg90H21EOLACvtfcHk0lhkOcUTiPb_QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmsVKqs7Y%253D&md5=cae393897ab9a897b24f10743ed7d5c9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaulbecker%26aufirst%3DC.%2BC.%26aulast%3DGruss%26aufirst%3DP.%26atitle%3DThe%2520oncogenic%2520potential%2520of%2520deregulated%2520homeobox%2520genes%26jtitle%3DCell%2520Growth%2520Differ.%26date%3D1993%26volume%3D4%26spage%3D431%26epage%3D441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Waltregny, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alami, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clausse, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Leval, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castronovo, V.</span><span> </span><span class="NLM_article-title">Overexpression of the homeobox gene HOXC8 in human prostate cancer correlates with loss of tumor differentiation</span> <span class="citation_source-journal">Prostate</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">162</span><span class="NLM_x">–</span> <span class="NLM_lpage">169</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2002&pages=162-169&author=D.+Waltregnyauthor=Y.+Alamiauthor=N.+Clausseauthor=J.+de+Levalauthor=V.+Castronovo&title=Overexpression+of+the+homeobox+gene+HOXC8+in+human+prostate+cancer+correlates+with+loss+of+tumor+differentiation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWaltregny%26aufirst%3DD.%26aulast%3DAlami%26aufirst%3DY.%26aulast%3DClausse%26aufirst%3DN.%26aulast%3Dde%2BLeval%26aufirst%3DJ.%26aulast%3DCastronovo%26aufirst%3DV.%26atitle%3DOverexpression%2520of%2520the%2520homeobox%2520gene%2520HOXC8%2520in%2520human%2520prostate%2520cancer%2520correlates%2520with%2520loss%2520of%2520tumor%2520differentiation%26jtitle%3DProstate%26date%3D2002%26volume%3D50%26spage%3D162%26epage%3D169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">De Vita, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barba, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Odartchenko, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givel, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freschi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucciarelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magli, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boncinelli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cillo, C.</span><span> </span><span class="NLM_article-title">Expression of homeobox-containing genes in primary and metastatic colorectal cancer</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">29A</span><span class="NLM_x">, </span> <span class="NLM_fpage">887</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29A&publication_year=1993&pages=887-893&author=G.+De+Vitaauthor=P.+Barbaauthor=N.+Odartchenkoauthor=J.+C.+Givelauthor=G.+Freschiauthor=G.+Bucciarelliauthor=M.+C.+Magliauthor=E.+Boncinelliauthor=C.+Cillo&title=Expression+of+homeobox-containing+genes+in+primary+and+metastatic+colorectal+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVita%26aufirst%3DG.%26aulast%3DBarba%26aufirst%3DP.%26aulast%3DOdartchenko%26aufirst%3DN.%26aulast%3DGivel%26aufirst%3DJ.%2BC.%26aulast%3DFreschi%26aufirst%3DG.%26aulast%3DBucciarelli%26aufirst%3DG.%26aulast%3DMagli%26aufirst%3DM.%2BC.%26aulast%3DBoncinelli%26aufirst%3DE.%26aulast%3DCillo%26aufirst%3DC.%26atitle%3DExpression%2520of%2520homeobox-containing%2520genes%2520in%2520primary%2520and%2520metastatic%2520colorectal%2520cancer%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D1993%26volume%3D29A%26spage%3D887%26epage%3D893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Hsieh, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ernst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span> </span><span class="NLM_article-title">Proteolytic cleavage of MLL generates a complex of N- and C-terminal fragments that confers protein stability and subnuclear localization</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1128%2FMCB.23.1.186-194.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=12482972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVOksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=186-194&author=J.+J.+Hsiehauthor=P.+Ernstauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=S.+J.+Korsmeyer&title=Proteolytic+cleavage+of+MLL+generates+a+complex+of+N-+and+C-terminal+fragments+that+confers+protein+stability+and+subnuclear+localization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolytic cleavage of MLL generates a complex of N- and C-terminal fragments that confers protein stability and subnuclear localization</span></div><div class="casAuthors">Hsieh, James J.-D.; Ernst, Patricia; Erdjument-Bromage, Hediye; Tempst, Paul; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-194</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The mixed-lineage leukemia gene (MLL, ALL1, HRX) encodes a 3,969-amino-acid nuclear protein homologous to Drosophila trithorax and is required to maintain proper Hox gene expression.  Chromosome translocations in human leukemia disrupt MLL (11q23), generating chimeric proteins between the N terminus of MLL and multiple translocation partners.  Here we report that MLL is normally cleaved at two conserved sites (D/GADD and D/GVDD) and that mutation of these sites abolishes the proteolysis.  MLL cleavage generates N-terminal p320 (N320) and C-terminal p180 (C180) fragments, which form a stable complex that localizes to a subnuclear compartment.  The FYRN domain of N320 directly interacts with the FYRC and SET domains of C180.  Disrupting the interaction between N320 and C180 leads to a marked decrease in the level of N320 and a redistribution of C180 to a diffuse nuclear pattern.  These data suggest a model in which a dynamic post-cleavage assocn. confers stability to N320 and correct nuclear sublocalization of the complex, to control the availability of N320 for target genes.  This predicts that MLL fusion proteins of leukemia which would lose the ability to complex with C180 have their stability conferred instead by the fusion partners, thus providing one mechanism for altered target gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfRH2aOlJKXLVg90H21EOLACvtfcHk0lhkOcUTiPb_QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVOksQ%253D%253D&md5=91eefe6c60909debacfc930c16906e7d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1128%2FMCB.23.1.186-194.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.23.1.186-194.2003%26sid%3Dliteratum%253Aachs%26aulast%3DHsieh%26aufirst%3DJ.%2BJ.%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DProteolytic%2520cleavage%2520of%2520MLL%2520generates%2520a%2520complex%2520of%2520N-%2520and%2520C-terminal%2520fragments%2520that%2520confers%2520protein%2520stability%2520and%2520subnuclear%2520localization%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2003%26volume%3D23%26spage%3D186%26epage%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">A conserved arginine-containing motif crucial for the assembly and enzymatic activity of the mixed lineage leukemia protein-1 core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32162</span><span class="NLM_x">–</span> <span class="NLM_lpage">32175</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+arginine-containing+motif+crucial+for+the+assembly+and+enzymatic+activity+of+the+mixed+lineage+leukemia+protein-1+core+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0ljdVe-11Glq-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520arginine-containing%2520motif%2520crucial%2520for%2520the%2520assembly%2520and%2520enzymatic%2520activity%2520of%2520the%2520mixed%2520lineage%2520leukemia%2520protein-1%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">713</span><span class="NLM_x">–</span> <span class="NLM_lpage">719</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb1128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16878130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=713-719&author=Y.+Douauthor=T.+A.+Milneauthor=A.+J.+Ruthenburgauthor=S.+Leeauthor=J.+W.+Leeauthor=G.+L.+Verdineauthor=C.+D.+Allisauthor=R.+G.+Roeder&title=Regulation+of+MLL1+H3K4+methyltransferase+activity+by+its+core+components"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of MLL1 H3K4 methyltransferase activity by its core components</span></div><div class="casAuthors">Dou, Yali; Milne, Thomas A.; Ruthenburg, Alexander J.; Lee, Seunghee; Lee, Jae Woon; Verdine, Gregory L.; Allis, C. David; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">713-719</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone H3 Lys4 (H3K4) methylation is a prevalent mark assocd. with transcription activation.  A common feature of several H3K4 methyltransferase complexes is the presence of three structural components (RbBP5, Ash2L and WDR5) and a catalytic subunit contg. a SET domain.  Here we report the first biochem. reconstitution of a functional four-component mixed-lineage leukemia protein-1 (MLL1) core complex.  This reconstitution, combined with in vivo assays, allows direct anal. of the contribution of each component to MLL1 enzymic activity and their roles in transcriptional regulation.  Moreover, taking clues from a crystal structure anal., we demonstrate that WDR5 mediates interactions of the MLL1 catalytic unit both with the common structural platform and with the histone substrate.  Mechanistic insights gained from this study can be generalized to the whole family of SET1-like histone methyltransferases in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp65Iy--Is42bVg90H21EOLACvtfcHk0ljdVe-11Glq-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jt7Y%253D&md5=d6fa704fa0a2fa69afddb540fbff28ae</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1128%26sid%3Dliteratum%253Aachs%26aulast%3DDou%26aufirst%3DY.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%2BW.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DRegulation%2520of%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520by%2520its%2520core%2520components%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D713%26epage%3D719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Wysocka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swigut, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burlingame, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brivanlou, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">WDR5 associates with histone H3 methylated at K4 and is essential for H3 K4 methylation and vertebrate development</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">872</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.cell.2005.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=15960974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD2MXls1Gks7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2005&pages=859-872&author=J.+Wysockaauthor=T.+Swigutauthor=T.+A.+Milneauthor=Y.+Douauthor=X.+Zhangauthor=A.+L.+Burlingameauthor=R.+G.+Roederauthor=A.+H.+Brivanlouauthor=C.+D.+Allis&title=WDR5+associates+with+histone+H3+methylated+at+K4+and+is+essential+for+H3+K4+methylation+and+vertebrate+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 associates with histone H3 methylated at K4 and is essential for H3 K4 methylation and vertebrate development</span></div><div class="casAuthors">Wysocka, Joanna; Swigut, Tomek; Milne, Thomas A.; Dou, Yali; Zhang, Xin; Burlingame, Alma L.; Roeder, Robert G.; Brivanlou, Ali H.; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">859-872</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone H3 lysine 4 (K4) methylation has been linked to the transcriptional activation in a variety of eukaryotic species.  Here authors show that a common component of MLL1, MLL2, and hSet1 H3 K4 methyltransferase complexes, the WD40-repeat protein WDR5, directly assocs. with histone H3 di- and trimethylated at K4 and with H3-K4-dimethylated nucleosomes.  WDR5 is required for binding of the methyltransferase complex to the K4-dimethylated H3 tail as well as for global H3 K4 trimethylation and HOX gene activation in human cells.  WDR5 is essential for vertebrate development, in that WDR5-depleted X. laevis tadpoles exhibit a variety of developmental defects and abnormal spatial Hox gene expression.  These results are the first demonstration that a WD40-repeat protein acts as a module for recognition of a specific histone modification and suggest a mechanism for reading and writing an epigenetic mark for gene activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA3qOoBU6MFbVg90H21EOLACvtfcHk0ljdVe-11Glq-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXls1Gks7g%253D&md5=c0dcb0420bf36dc8f35071c855a5f0dc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DWysocka%26aufirst%3DJ.%26aulast%3DSwigut%26aufirst%3DT.%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBurlingame%26aufirst%3DA.%2BL.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DBrivanlou%26aufirst%3DA.%2BH.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DWDR5%2520associates%2520with%2520histone%2520H3%2520methylated%2520at%2520K4%2520and%2520is%2520essential%2520for%2520H3%2520K4%2520methylation%2520and%2520vertebrate%2520development%26jtitle%3DCell%26date%3D2005%26volume%3D121%26spage%3D859%26epage%3D872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Song, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingston, R. E.</span><span> </span><span class="NLM_article-title">WDR5 interacts with mixed lineage leukemia (MLL) protein via the histone H3-binding pocket</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">35258</span><span class="NLM_x">–</span> <span class="NLM_lpage">35264</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=35258-35264&author=J.+J.+Songauthor=R.+E.+Kingston&title=WDR5+interacts+with+mixed+lineage+leukemia+%28MLL%29+protein+via+the+histone+H3-binding+pocket"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DJ.%2BJ.%26aulast%3DKingston%26aufirst%3DR.%2BE.%26atitle%3DWDR5%2520interacts%2520with%2520mixed%2520lineage%2520leukemia%2520%2528MLL%2529%2520protein%2520via%2520the%2520histone%2520H3-binding%2520pocket%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D35258%26epage%3D35264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">Structure of WDR5 bound to mixed lineage leukemia protein-1 peptide</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32158</span><span class="NLM_x">–</span> <span class="NLM_lpage">32161</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+bound+to+mixed+lineage+leukemia+protein-1+peptide"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0liYTi0co6qGmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520bound%2520to%2520mixed%2520lineage%2520leukemia%2520protein-1%2520peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Schuetz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loppnau, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bochkarev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plotnikov, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span> </span><span class="NLM_article-title">Structural basis for molecular recognition and presentation of histone H3 by WDR5</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">4245</span><span class="NLM_x">–</span> <span class="NLM_lpage">4252</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fsj.emboj.7601316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16946699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpslamur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=4245-4252&author=A.+Schuetzauthor=A.+Allali-Hassaniauthor=F.+Martinauthor=P.+Loppnauauthor=M.+Vedadiauthor=A.+Bochkarevauthor=A.+N.+Plotnikovauthor=C.+H.+Arrowsmithauthor=J.+Min&title=Structural+basis+for+molecular+recognition+and+presentation+of+histone+H3+by+WDR5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for molecular recognition and presentation of histone H3 By WDR5</span></div><div class="casAuthors">Schuetz, Anja; Allali-Hassani, Abdellah; Martin, Fernando; Loppnau, Peter; Vedadi, Masoud; Bochkarev, Alexey; Plotnikov, Alexander N.; Arrowsmith, Cheryl H.; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4245-4252</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation at specific lysine residues brings about various downstream events that are mediated by different effector proteins.  The WD40 domain of WDR5 represents a new class of histone methyl-lysine recognition domains that is important for recruiting H3K4 methyltransferases to K4-dimethylated histone H3 tail as well as for global and gene-specific K4 trimethylation.  Here we report the crystal structures of full-length WDR5, WDR5Δ23 and its complexes with unmodified, mono-, di- and trimethylated histone H3K4 peptides.  The structures reveal that WDR5 is able to bind all of these histone H3 peptides, but only H3K4me2 peptide forms extra interactions with WDR5 by use of both water-mediated hydrogen bonding and the altered hydrophilicity of the modified lysine 4.  We propose a mechanism for the involvement of WDR5 in binding and presenting histone H3K4 for further methylation as a component of MLL complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVttgBFfxEWLVg90H21EOLACvtfcHk0liYTi0co6qGmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpslamur4%253D&md5=26d05eb16b1691485a7555fda13cf556</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601316%26sid%3Dliteratum%253Aachs%26aulast%3DSchuetz%26aufirst%3DA.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DMartin%26aufirst%3DF.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBochkarev%26aufirst%3DA.%26aulast%3DPlotnikov%26aufirst%3DA.%2BN.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520molecular%2520recognition%2520and%2520presentation%2520of%2520histone%2520H3%2520by%2520WDR5%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D4245%26epage%3D4252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Han, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, J.</span><span> </span><span class="NLM_article-title">Structural basis for the specific recognition of methylated histone H3 lysine 4 by the WD-40 protein WDR5</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">137</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=137-144&author=Z.+Hanauthor=L.+Guoauthor=H.+Wangauthor=Y.+Shenauthor=X.+W.+Dengauthor=J.+Chai&title=Structural+basis+for+the+specific+recognition+of+methylated+histone+H3+lysine+4+by+the+WD-40+protein+WDR5"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%2BW.%26aulast%3DChai%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520the%2520specific%2520recognition%2520of%2520methylated%2520histone%2520H3%2520lysine%25204%2520by%2520the%2520WD-40%2520protein%2520WDR5%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D22%26spage%3D137%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Molecular recognition of histone H3 by the WD40 protein WDR5</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb1116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16829960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=698-703&author=J.+F.+Coutureauthor=E.+Collazoauthor=R.+C.+Trievel&title=Molecular+recognition+of+histone+H3+by+the+WD40+protein+WDR5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of histone H3 by the WD40 protein WDR5</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">698-703</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The WD40-repeat protein WDR5 is a conserved subunit of Trithorax (TRX) histone methyltransferase complexes.  WDR5 has been reported to selectively bind dimethylated Lys-4 (K4me2) in histone H3 to promote K4 trimethylation by TRX.  To elucidate the basis of this binding specificity, we have detd. the crystal structure of WDR5 bound to a histone H3 peptide bearing K4me2.  The structure reveals that the N terminus of histone H3 binds as a 310-helix in the central depression formed by the WD40 repeats.  R2 in histone H3 is bound in the acidic channel in the protein's core, whereas K4me2 is solvent exposed and does not engage in direct interactions with WDR5.  Functional studies confirm that WDR5 recognizes A1, R2 and T3 in histone H3 but has virtually identical affinities for the unmodified and mono-, di- and trimethylated forms of K4, demonstrating that it does not discriminate among different degrees of methylation of this residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNU0kVOM0Y3rVg90H21EOLACvtfcHk0liYTi0co6qGmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrY%253D&md5=8c77c52b072810965f8c68fb6358a69c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1116%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DMolecular%2520recognition%2520of%2520histone%2520H3%2520by%2520the%2520WD40%2520protein%2520WDR5%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D698%26epage%3D703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Ruthenburg, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graybosch, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdine, G. L.</span><span> </span><span class="NLM_article-title">Histone H3 recognition and presentation by the WDR5 module of the MLL1 complex</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">704</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb1119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=16829959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=704-712&author=A.+J.+Ruthenburgauthor=W.+Wangauthor=D.+M.+Grayboschauthor=H.+Liauthor=C.+D.+Allisauthor=D.+J.+Patelauthor=G.+L.+Verdine&title=Histone+H3+recognition+and+presentation+by+the+WDR5+module+of+the+MLL1+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 recognition and presentation by the WDR5 module of the MLL1 complex</span></div><div class="casAuthors">Ruthenburg, Alexander J.; Wang, Wooikoon; Graybosch, Daina M.; Li, Haitao; Allis, C. David; Patel, Dinshaw J.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">704-712</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">WDR5 is a core component of SET1-family complexes that achieve transcriptional activation via methylation of histone H3 on Nζ of Lys-4 (H3K4).  The role of WDR5 in the MLL1 complex has recently been described as specific recognition of dimethyl-K4 in the context of a histone H3 N-terminus; WDR5 is essential for vertebrate development, Hox gene activation and global H3K4 trimethylation.  We report the high-resoln. x-ray structures of WDR5 in the unliganded form and complexed with histone H3 peptides having unmodified and mono-, di- and trimethylated K4, which together provide the first comprehensive anal. of methylated histone recognition by the ubiquitous WD40-repeat fold.  Contrary to predictions, the structures reveal that WDR5 does not read out the methylation state of K4 directly, but instead serves to present the K4 side chain for further methylation by SET1-family complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWXFgXXkjzvrVg90H21EOLACvtfcHk0ljStoQE3d5Dww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xnsl2jtrc%253D&md5=0bc8aab1a1543fd9f7ff8b1fef4ff80c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1119%26sid%3Dliteratum%253Aachs%26aulast%3DRuthenburg%26aufirst%3DA.%2BJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGraybosch%26aufirst%3DD.%2BM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DHistone%2520H3%2520recognition%2520and%2520presentation%2520by%2520the%2520WDR5%2520module%2520of%2520the%2520MLL1%2520complex%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D704%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Trievel, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shilatifard, A.</span><span> </span><span class="NLM_article-title">WDR5, a complexed protein</span> <span class="citation_source-journal">Nature Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">678</span><span class="NLM_x">–</span> <span class="NLM_lpage">680</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1038%2Fnsmb0709-678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=19578375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVCltr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=678-680&author=R.+C.+Trievelauthor=A.+Shilatifard&title=WDR5%2C+a+complexed+protein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5, a complexed protein</span></div><div class="casAuthors">Trievel, Raymond C.; Shilatifard, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">678-680</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The WD40 protein WDR5 is a core subunit of the human MLL and SET1 (hCOMPASS) histone H3 Lys4 (H3K4) methyltransferase complexes.  Although initial studies suggested that WDR5 interacts with methylated H3K4 to catalyze Lys4 trimethylation, recent work has revealed that it binds an arginine-bearing motif in MLL1, promoting complex assembly and activity.  These findings suggest that WDR5 functions as a peptidyl arginine-recognition factor that facilitates the assembly of hCOMPASS and other chromatin-modifying complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrStMNXz6SAy7Vg90H21EOLACvtfcHk0ljStoQE3d5Dww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVCltr4%253D&md5=f2f47d834467f501653b8ab27828809f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnsmb0709-678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb0709-678%26sid%3Dliteratum%253Aachs%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26aulast%3DShilatifard%26aufirst%3DA.%26atitle%3DWDR5%252C%2520a%2520complexed%2520protein%26jtitle%3DNature%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D678%26epage%3D680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Nikolovska-Coleska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saito, N. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stuckey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">332</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1016%2Fj.ab.2004.05.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=15325294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=332&publication_year=2004&pages=261-273&author=Z.+Nikolovska-Coleskaauthor=R.+Wangauthor=X.+Fangauthor=H.+Panauthor=Y.+Tomitaauthor=P.+Liauthor=P.+P.+Rollerauthor=K.+Krajewskiauthor=N.+G.+Saitoauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Development+and+optimization+of+a+binding+assay+for+the+XIAP+BIR3+domain+using+fluorescence+polarization"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development and optimization of a binding assay for the XIAP BIR3 domain using fluorescence polarization</span></div><div class="casAuthors">Nikolovska-Coleska, Zaneta; Wang, Renxiao; Fang, Xueliang; Pan, Hongguang; Tomita, York; Li, Peng; Roller, Peter P.; Krajewski, Krzysztof; Saito, Naoyuki G.; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">332</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The X-linked inhibitor of apoptosis protein (XIAP) is a potent cellular inhibitor of apoptosis.  Designing small-mol. inhibitors that target the BIR3 domain of XIAP, where Smac/DIABLO (second mitochondria-derived activator of caspase/direct IAP-binding protein with low pI) and caspase-9 bind, is a promising strategy for inhibiting the antiapoptotic activity of XIAP and for overcoming apoptosis resistance of cancer cells mediated by XIAP.  Herein, we report the development of a homogeneous high-throughput assay based on fluorescence polarization for measuring the binding affinities of small-mol. inhibitors to the BIR3 domain of XIAP.  Among four fluorescent probes tested, a mutated N-terminal Smac peptide (AbuRPFK-(5-Fam)-NH2) showed the highest affinity (Kd = 17.92 nM) and a large dynamic range (ΔmP=231±0.9), and was selected as the most suitable probe for the binding assay.  The binding conditions (DMSO tolerance and stability) have been investigated.  Under optimized conditions, a Z' factor of 0.88 was achieved in a 96-well format for high-throughput screening.  It was found that the popular Cheng-Prusoff equation is invalid for the calcn. of the competitive inhibition consts. (Ki values) for inhibitors in the FP-based competitive binding assay conditions, and accordingly, a new math. equation was developed, validated, and used to compute the Ki values.  An assocd. Web-based computer program was also developed for this task.  Several known Smac peptides with high and low affinities have been evaluated under the assay conditions and the results obtained indicated that the FP-based competitive binding assay performs correctly as designed: it can quant. and accurately det. the binding affinities of Smac-based peptide inhibitors with a wide range of affinities, and is suitable for high-throughput screening of inhibitors binding to the XIAP BIR3 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWjEsqtWsZRbVg90H21EOLACvtfcHk0ljStoQE3d5Dww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvFeltL0%253D&md5=23cade47a4996c7ca4c01278d3e0d3a6</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2004.05.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2004.05.055%26sid%3Dliteratum%253Aachs%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DTomita%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DKrajewski%26aufirst%3DK.%26aulast%3DSaito%26aufirst%3DN.%2BG.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDevelopment%2520and%2520optimization%2520of%2520a%2520binding%2520assay%2520for%2520the%2520XIAP%2520BIR3%2520domain%2520using%2520fluorescence%2520polarization%26jtitle%3DAnal.%2520Biochem.%26date%3D2004%26volume%3D332%26spage%3D261%26epage%3D273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Faber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van den Heuvel-Eibrink, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwaan, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">HOXA9 is required for survival in human MLL-rearranged acute leukemias</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">113</span><span class="NLM_x">, </span> <span class="NLM_fpage">2375</span><span class="NLM_x">–</span> <span class="NLM_lpage">2385</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2009&pages=2375-2385&author=J.+Faberauthor=A.+V.+Krivtsovauthor=M.+C.+Stubbsauthor=R.+Wrightauthor=T.+N.+Davisauthor=M.+van+den+Heuvel-Eibrinkauthor=C.+M.+Zwaanauthor=A.+L.+Kungauthor=S.+A.+Armstrong&title=HOXA9+is+required+for+survival+in+human+MLL-rearranged+acute+leukemias"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFaber%26aufirst%3DJ.%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DStubbs%26aufirst%3DM.%2BC.%26aulast%3DWright%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3Dvan%2Bden%2BHeuvel-Eibrink%26aufirst%3DM.%26aulast%3DZwaan%26aufirst%3DC.%2BM.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DHOXA9%2520is%2520required%2520for%2520survival%2520in%2520human%2520MLL-rearranged%2520acute%2520leukemias%26jtitle%3DBlood%26date%3D2009%26volume%3D113%26spage%3D2375%26epage%3D2385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">On the mechanism of multiple lysine methylation by the human mixed lineage leukemia protein-1 (MLL1) core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">24242</span><span class="NLM_x">–</span> <span class="NLM_lpage">24256</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm100139b&amp;key=10.1074%2Fjbc.M109.014498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm100139b&amp;key=19556245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm100139b&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24242-24256&author=A.+Patelauthor=V.+Dharmarajanauthor=V.+E.+Voughtauthor=M.+S.+Cosgrove&title=On+the+mechanism+of+multiple+lysine+methylation+by+the+human+mixed+lineage+leukemia+protein-1+%28MLL1%29+core+complex"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">On the Mechanism of Multiple Lysine Methylation by the Human Mixed Lineage Leukemia Protein-1 (MLL1) Core Complex</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Vought, Valarie E.; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">24242-24256</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcription in eukaryotic genomes depends on enzymes that regulate the degree of histone H3 lysine 4 (H3K4) methylation.  The mixed lineage leukemia protein-1 (MLL1) is a member of the SET1 family of H3K4 methyltransferases and is frequently rearranged in acute leukemias.  Despite sequence comparisons that predict that SET1 family enzymes should only monomethylate their substrates, mono-, di-, and trimethylation of H3K4 has been attributed to SET1 family complexes in vivo and in vitro.  To better understand this paradox, we have biochem. reconstituted and characterized a five-component 200-kDa MLL1 core complex contg. human MLL1, WDR5, RbBP5, Ash2L, and DPY-30.  We demonstrate that the isolated MLL1 SET domain is a slow monomethyltransferase and that tyrosine 3942 of MLL1 prevents di- and trimethylation of H3K4.  In contrast, a complex contg. the MLL1 SET domain, WDR5, RbBP5, Ash2L, and DPY-30, displays a marked ∼600-fold increase in enzymic activity but only to the di-Me form of H3K4.  Single turnover kinetic expts. reveal that the reaction leading to H3K4 dimethylation involves the transient accumulation of a monomethylated species, suggesting that the MLL1 core complex uses a non-processive mechanism to catalyze multiple lysine methylation.  We have also discovered that the non-SET domain components of the MLL1 core complex possess a previously unrecognized methyltransferase activity that catalyzes H3K4 dimethylation within the MLL1 core complex.  Our results suggest that the mechanism of multiple lysine methylation by the MLL1 core complex involves the sequential addn. of two Me groups at two distinct active sites within the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ZXmcwVcpRrVg90H21EOLACvtfcHk0lgHRy1fE2NSXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVGgsr%252FF&md5=4289813be10179ac818b8d93fe6ebc6a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.014498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.014498%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DOn%2520the%2520mechanism%2520of%2520multiple%2520lysine%2520methylation%2520by%2520the%2520human%2520mixed%2520lineage%2520leukemia%2520protein-1%2520%2528MLL1%2529%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24242%26epage%3D24256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">D.A. Case, T. A. D.; T.E. Cheatham, C. L.,  III; Simmerling, J.; Wang, R. E. D.; R. Luo, K. M.; Merz, D. A.; Pearlman, M.; Crowley, R. C.; Walker, W. Z.; B. Wang, S.; Hayik, A.; Roitberg, G.; Seabra, K. F.; Wong, F. P.; X. Wu, S.; Brozell, V.; Tsui, H.; Gohlke, L.; Yang, C.; Tan, J. M.; V. Hornak, G.; Cui, P.; Beroza, D. H.; Mathews, C.; Schafmeister, W. S. R.; Kollman, P.A.</span> <span class="citation_source-book">AMBER 9</span>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2006</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=T.+A.+D.+D.A.+Case&author=C.+L.+T.E.+Cheatham&author=J.+Simmerling&author=R.+E.+D.+Wang&author=K.+M.+R.+Luo&author=D.+A.+Merz&author=M.+Pearlman&author=R.+C.+Crowley&author=W.+Z.+Walker&author=S.+B.+Wang&author=A.+Hayik&author=G.+Roitberg&author=K.+F.+Seabra&author=F.+P.+Wong&author=S.+X.+Wu&author=V.+Brozell&author=H.+Tsui&author=L.+Gohlke&author=C.+Yang&author=J.+M.+Tan&author=G.+V.+Hornak&author=P.+Cui&author=D.+H.+Beroza&author=C.+Mathews&author=W.+S.+R.+Schafmeister&author=P.A.+Kollman&title=AMBER+9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DD.A.%2BCase%26aufirst%3DT.%2BA.%2BD.%26btitle%3DAMBER%25209%26pub%3DUniversity%2520of%2520California%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EG6','PDB','3EG6'); return false;">PDB: 3EG6</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i29"><a href="/doi/suppl/10.1021/jm100139b">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_66141"></div></div></div></div></div><hr /></hr><p class="last">Table containing the purity and the mass spectroscopy data of the peptides described here. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm100139b/suppl_file/jm100139b_si_001.pdf">jm100139b_si_001.pdf (90.28 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm100139b&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm100139b%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2010.53.issue-14%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm100139b" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679963784a793ce2","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
