
<html lang="en"     class="pb-page"  data-request-id="b64792f1-721a-419c-9902-a85d21d66675"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-10;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.1c00067;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Fragment-Based Design of a Potent MAT2a Inhibitor and in Vivo Evaluation in an MTAP Null Xenograft Model" /></meta><meta name="dc.Creator" content="Claudia  De Fusco" /></meta><meta name="dc.Creator" content="Marianne  Schimpl" /></meta><meta name="dc.Creator" content="Ulf  Börjesson" /></meta><meta name="dc.Creator" content="Tony  Cheung" /></meta><meta name="dc.Creator" content="Iain  Collie" /></meta><meta name="dc.Creator" content="Laura  Evans" /></meta><meta name="dc.Creator" content="Priyanka  Narasimhan" /></meta><meta name="dc.Creator" content="Christopher  Stubbs" /></meta><meta name="dc.Creator" content="Mercedes  Vazquez-Chantada" /></meta><meta name="dc.Creator" content="David J.  Wagner" /></meta><meta name="dc.Creator" content="Michael  Grondine" /></meta><meta name="dc.Creator" content="Matthew G.  Sanders" /></meta><meta name="dc.Creator" content="Sharon  Tentarelli" /></meta><meta name="dc.Creator" content="Elizabeth  Underwood" /></meta><meta name="dc.Creator" content="Argyrides  Argyrou" /></meta><meta name="dc.Creator" content="James M.  Smith" /></meta><meta name="dc.Creator" content="James T.  Lynch" /></meta><meta name="dc.Creator" content="Elisabetta  Chiarparin" /></meta><meta name="dc.Creator" content="Graeme  Robb" /></meta><meta name="dc.Creator" content="Sharan K.  Bagal" /></meta><meta name="dc.Creator" content="James S.  Scott" /></meta><meta name="dc.Description" content="MAT2a is a methionine adenosyltransferase that synthesizes the essential metabolite S-adenosylmethionine (SAM) from methionine and ATP. Tumors bearing the co-deletion of p16 and MTAP genes have bee..." /></meta><meta name="Description" content="MAT2a is a methionine adenosyltransferase that synthesizes the essential metabolite S-adenosylmethionine (SAM) from methionine and ATP. Tumors bearing the co-deletion of p16 and MTAP genes have bee..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 26, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00067" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00067" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00067" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00067" /></link>
        
    
    

<title>Fragment-Based Design of a Potent MAT2a Inhibitor and in Vivo Evaluation in an MTAP Null Xenograft Model | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00067" /></meta><meta property="og:title" content="Fragment-Based Design of a Potent MAT2a Inhibitor and in Vivo Evaluation in an MTAP Null Xenograft Model" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0016.jpeg" /></meta><meta property="og:description" content="MAT2a is a methionine adenosyltransferase that synthesizes the essential metabolite S-adenosylmethionine (SAM) from methionine and ATP. Tumors bearing the co-deletion of p16 and MTAP genes have been shown to be sensitive to MAT2a inhibition, making it an attractive target for treatment of MTAP-deleted cancers. A fragment-based lead generation campaign identified weak but efficient hits binding in a known allosteric site. By use of structure-guided design and systematic SAR exploration, the hits were elaborated through a merging and growing strategy into an arylquinazolinone series of potent MAT2a inhibitors. The selected in vivo tool compound 28 reduced SAM-dependent methylation events in cells and inhibited proliferation of MTAP-null cells in vitro. In vivo studies showed that 28 was able to induce antitumor response in an MTAP knockout HCT116 xenograft model." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00067"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00067">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00067&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00067&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00067&amp;href=/doi/10.1021/acs.jmedchem.1c00067" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 6814-6826</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00056" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00086" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-Based Design of a Potent MAT2a Inhibitor and <i>in Vivo</i> Evaluation in an MTAP Null Xenograft Model</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Claudia De Fusco</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Claudia De Fusco</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>C.D.F.: email, <a href="/cdn-cgi/l/email-protection#88ebe4e9fdece1e9a6ecedeefdfbebe7c8efe5e9e1e4a6ebe7e5"><span class="__cf_email__" data-cfemail="3c5f505d4958555d1258595a494f5f537c5b515d5550125f5351">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Claudia++De+Fusco">Claudia De Fusco</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8807-4976" title="Orcid link">http://orcid.org/0000-0001-8807-4976</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marianne Schimpl</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marianne Schimpl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>M.S.: email, <a href="/cdn-cgi/l/email-protection#81ece0f3e8e0efefe4aff2e2e9e8ecf1edc1e0f2f5f3e0fbe4efe4e2e0afe2eeec"><span class="__cf_email__" data-cfemail="b1dcd0c3d8d0dfdfd49fc2d2d9d8dcc1ddf1d0c2c5c3d0cbd4dfd4d2d09fd2dedc">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marianne++Schimpl">Marianne Schimpl</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2284-5250" title="Orcid link">http://orcid.org/0000-0003-2284-5250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulf Börjesson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulf Börjesson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Gothenburg SE-431 83, Sweden</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulf++B%C3%B6rjesson">Ulf Börjesson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tony Cheung</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tony Cheung</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Boston, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tony++Cheung">Tony Cheung</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Iain Collie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Iain Collie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Iain++Collie">Iain Collie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura Evans</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura Evans</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Evans">Laura Evans</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Priyanka Narasimhan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Priyanka Narasimhan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Priyanka++Narasimhan">Priyanka Narasimhan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christopher Stubbs</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher Stubbs</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher++Stubbs">Christopher Stubbs</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mercedes Vazquez-Chantada</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mercedes Vazquez-Chantada</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mercedes++Vazquez-Chantada">Mercedes Vazquez-Chantada</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David J. Wagner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David J. Wagner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Boston, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Wagner">David J. Wagner</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5805-930X" title="Orcid link">http://orcid.org/0000-0002-5805-930X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Grondine</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Grondine</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Boston, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Grondine">Michael Grondine</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew G. Sanders</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew G. Sanders</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+G.++Sanders">Matthew G. Sanders</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sharon Tentarelli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sharon Tentarelli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Boston, Massachusetts 02451, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sharon++Tentarelli">Sharon Tentarelli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elizabeth Underwood</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elizabeth Underwood</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth++Underwood">Elizabeth Underwood</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Argyrides Argyrou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Argyrides Argyrou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Discovery Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Argyrides++Argyrou">Argyrides Argyrou</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3141-9122" title="Orcid link">http://orcid.org/0000-0003-3141-9122</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James M. Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James M. Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+M.++Smith">James M. Smith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6750-265X" title="Orcid link">http://orcid.org/0000-0002-6750-265X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James T. Lynch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James T. Lynch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+T.++Lynch">James T. Lynch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elisabetta Chiarparin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elisabetta Chiarparin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elisabetta++Chiarparin">Elisabetta Chiarparin</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2998-1346" title="Orcid link">http://orcid.org/0000-0002-2998-1346</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Graeme Robb</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Graeme Robb</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Graeme++Robb">Graeme Robb</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4531-4375" title="Orcid link">http://orcid.org/0000-0002-4531-4375</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sharan K. Bagal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sharan K. Bagal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sharan+K.++Bagal">Sharan K. Bagal</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">James S. Scott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James S. Scott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Oncology R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+S.++Scott">James S. Scott</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2263-7024" title="Orcid link">http://orcid.org/0000-0002-2263-7024</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00067&amp;href=/doi/10.1021%2Facs.jmedchem.1c00067" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 6814–6826</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 26, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>14 January 2021</li><li><span class="item_label"><b>Published</b> online</span>26 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00067" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00067</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by-nc"><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nc-1612971093307.svg" alt="NC: Only noncommercial uses of the work are permitted" /></img><span><strong>NC: </strong>Only noncommercial uses of the work are permitted</span></a><a href="https://creativecommons.org/licenses/by-nc-nd/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_nd-1612971093637.svg" alt="ND: No derivatives or adaptations of the work are permitted" /></img><span><strong>ND: </strong>No derivatives or adaptations of the work are permitted</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00067"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4737</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00067" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-Based Design of a Potent MAT2a Inhibitor and in Vivo Evaluation in an MTAP Null Xenograft Model&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Claudia&quot;,&quot;last_name&quot;:&quot;De Fusco&quot;},{&quot;first_name&quot;:&quot;Marianne&quot;,&quot;last_name&quot;:&quot;Schimpl&quot;},{&quot;first_name&quot;:&quot;Ulf&quot;,&quot;last_name&quot;:&quot;Börjesson&quot;},{&quot;first_name&quot;:&quot;Tony&quot;,&quot;last_name&quot;:&quot;Cheung&quot;},{&quot;first_name&quot;:&quot;Iain&quot;,&quot;last_name&quot;:&quot;Collie&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Evans&quot;},{&quot;first_name&quot;:&quot;Priyanka&quot;,&quot;last_name&quot;:&quot;Narasimhan&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;Stubbs&quot;},{&quot;first_name&quot;:&quot;Mercedes&quot;,&quot;last_name&quot;:&quot;Vazquez-Chantada&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Wagner&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Grondine&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;G. Sanders&quot;},{&quot;first_name&quot;:&quot;Sharon&quot;,&quot;last_name&quot;:&quot;Tentarelli&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;Underwood&quot;},{&quot;first_name&quot;:&quot;Argyrides&quot;,&quot;last_name&quot;:&quot;Argyrou&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;M. Smith&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;T. Lynch&quot;},{&quot;first_name&quot;:&quot;Elisabetta&quot;,&quot;last_name&quot;:&quot;Chiarparin&quot;},{&quot;first_name&quot;:&quot;Graeme&quot;,&quot;last_name&quot;:&quot;Robb&quot;},{&quot;first_name&quot;:&quot;Sharan&quot;,&quot;last_name&quot;:&quot;K. Bagal&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;S. Scott&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;6814-6826&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00067&quot;},&quot;abstract&quot;:&quot;MAT2a is a methionine adenosyltransferase that synthesizes the essential metabolite S-adenosylmethionine (SAM) from methionine and ATP. Tumors bearing the co-deletion of p16 and MTAP genes have been shown to be sensitive to MAT2a inhibition, making it an attractive target for treatment of MTAP-deleted cancers. A fragment-based lead generation campaign identified weak but efficient hits binding in a known allosteric site. By use of structure-guided design and systematic SAR exploration, the hits were elaborated through a merging and growing strategy into an arylquinazolinone series of potent MAT2a inhibitors. The selected in vivo tool compound 28 reduced SAM-dependent methylation events in cells and inhibited proliferation of MTAP-null cells in vitro. In vivo studies showed that 28 was able to induce antitumor response in an MTAP knockout HCT116 xenograft model.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00067&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00067" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00067&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00067" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00067&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00067" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00067&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00067&amp;href=/doi/10.1021/acs.jmedchem.1c00067" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00067" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00067" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00067%26sid%3Dliteratum%253Aachs%26pmid%3D33900758%26genre%3Darticle%26aulast%3DDe%2BFusco%26date%3D2021%26atitle%3DFragment-Based%2BDesign%2Bof%2Ba%2BPotent%2BMAT2a%2BInhibitor%2Band%2Bin%2BVivo%2BEvaluation%2Bin%2Ban%2BMTAP%2BNull%2BXenograft%2BModel%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D10%26spage%3D6814%26epage%3D6826%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290888" title="Liquid liquid equilibrium">Liquid liquid equilibrium</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/jmcmar.2021.64.issue-10/20210527/jmcmar.2021.64.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">MAT2a is a methionine adenosyltransferase that synthesizes the essential metabolite <i>S</i>-adenosylmethionine (SAM) from methionine and ATP. Tumors bearing the co-deletion of p16 and MTAP genes have been shown to be sensitive to MAT2a inhibition, making it an attractive target for treatment of MTAP-deleted cancers. A fragment-based lead generation campaign identified weak but efficient hits binding in a known allosteric site. By use of structure-guided design and systematic SAR exploration, the hits were elaborated through a merging and growing strategy into an arylquinazolinone series of potent MAT2a inhibitors. The selected <i>in vivo</i> tool compound <b>28</b> reduced SAM-dependent methylation events in cells and inhibited proliferation of MTAP-null cells <i>in vitro</i>. <i>In vivo</i> studies showed that <b>28</b> was able to induce antitumor response in an MTAP knockout HCT116 xenograft model.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08480" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08480" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The methionine adenosyltransferases MAT1a and MAT2a synthesize the key cofactor <i>S</i>-adenosylmethionine (SAM) from methionine and ATP and play an important role in cellular growth and survival.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In adults, MAT1a is primarily expressed in the liver and plays an important role in metabolizing the bulk of dietary methionine. In contrast, MAT2a is mainly expressed in extrahepatic tissues and its expression in the liver is primarily limited to cases of cellular dedifferentiation, such as in fetal liver, or upon liver injury or hepatocellular carcinoma (HCC).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> A link between MAT2a and other cancers was demonstrated when a shRNA screen in 390 cancer cell lines identified MAT2a knockdown, along with knockdown of PRMT5 and PRMT5 cofactors, as synthetic lethal in MTAP-deficient cancer cells.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Marjon and colleagues demonstrated <i>in vivo</i> that an inducible knockdown of MAT2a reduced tumor growth in an MTAP null MCF7 tumor model.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">The MTAP gene is colocated on chromosome 9p21 with the p16 gene (MTS1/CDK4I/CDKN2A), and homozygous co-deletion of MTAP and p16 genes occurs in 80–90% of tumors with p16 deletion.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> p16/CDKN2 deletion is among the most frequent cancer-driving mutations, but targeting a loss of function mutant is not trivial due to the lack of a target protein. The co-deletion of MTAP gene and resulting sensitization to MAT2a inhibition represent a tremendous opportunity to circumvent this issue and provide an option for therapeutic intervention. MTAP-deficient cancer cells are unable to metabolize methylthioadenosine (MTA) as part of the methionine salvage pathway<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and thereby accumulate intracellular MTA, a potent and selective inhibitor of the methyl transferase PRMT5.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> This inhibition renders these cancer cells vulnerable to further PRMT5 inhibition either directly or indirectly through inhibition of the upstream metabolic enzyme MAT2a.</div><div class="NLM_p">Few MAT2a inhibitors were known until very recently. Methionine analogues have been reported as inhibitors of SAM synthesis,<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> but being amino acid mimetics and having low potency have limited their applicability. The first drug-like MAT2a inhibitor described in the literature was PF-9366 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Its binding mode was characterized by X-ray crystallography, which revealed that PF-9366 acted as an allosteric inhibitor binding at the interface of two MAT2a subunits, at a site of regulatory control otherwise occupied by the regulatory subunit MAT2b (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(11)</a> The authors showed target engagement for PF-9366 and that the compound was able to modulate SAM synthesis in cancer cells with an IC<sub>50</sub> of 1.2 μM. However, this did not translate to inhibition of cell proliferation. The upregulation of MAT2a gene expression as a feedback mechanism to MAT2a inhibition was suggested to be the cause of lack of translation. However, given the weak cell potency of PF-9366, the question remained if a more potent compound would be able to overcome the effect of the feedback loop and MAT2a upregulation. In 2017 Agios disclosed their <i>in vivo</i> tool AGI-25696 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C) and its related chemical series in a number of patents,<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(12−14)</a> and in the following year, the first clinical trial with a MAT2a inhibitor, AG-270, was opened, enrolling patients with advanced solid tumors or lymphomas with homozygous MTAP deletion.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. MAT2a is a functional homodimer with two catalytic sites and a site of allosteric regulation located at the dimer interface. (A) The two MAT2a monomers are depicted as cartoon in blue and gray. At the interface, the reaction products <i>S</i>-adenosylmethionine (SAM) and (β-γ-imido)triphosphate are captured in one of the two active sites (sticks with yellow carbons). Binding of the carboxy-terminal peptide of MAT2b (purple cartoon) to the dimer interface leads to allosteric regulation of SAM biosynthesis (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ndn">4ndn</a>, chains A, B, E). (B) Surface view of MAT2a. Gray shading depicts cavities at the dimer interface, and the largest of these harbor the active and allosteric sites. (C) Recently reported allosteric inhibitors of MAT2a.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This paper describes a fragment-based approach to discovering new MAT2a inhibitors. Several marketed drugs and a growing number of clinical candidates have been generated by fragment-based drug discovery, which is based on the identification of small but efficient hits through highly sensitive and robust biophysical methods.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(15,16)</a> One of the underlying principles is the efficient sampling of chemical space by screening molecules with restricted molecular weight. Consequently, the most productive ligand–protein interactions and binders with high ligand efficiency can be identified.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(17)</a> With high quality binders in hand and relying heavily on structure-based design, it is possible via strategies of fragment merging or linking to combine desirable features and to generate more complex and higher affinity compounds. Hits can also be expanded by adding atoms or functional groups to target new interactions and/or fill available subpockets (growing).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(18)</a> Evolution of the hits through merging, linking, and growing strategies can effectively provide advanced leads while closely controlling the physicochemical properties.</div><div class="NLM_p last">A biophysical fragment screen against MAT2a identified two highly efficient hits binding in the allosteric site. These fragments were merged to generate a new core characterized by higher ligand efficiency and able to inhibit the catalytic activity of MAT2a. A growing strategy was then adopted, resulting in compound <b>28</b>, which is a potent and orally bioavailable compound that was used to investigate the effect of MAT2a inhibition <i>in vivo</i>.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37872" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37872" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Fragment Screening and Hits</h3><div class="NLM_p">The hit finding strategy comprised three parallel screening approaches: high throughput thermal shift/differential scanning fluorimetry assay of the AstraZeneca fragment library, surface plasmon resonance (SPR) of a high solubility subset of the library, and a cocktail-based crystallographic screen of a different subset. Each of these methods yielded a number of hits that were validated in the screening cascade (data not shown). The most potent and efficient hits originated from the SPR screen and were triazinone <b>1</b> and quinazoline <b>2</b> with respective affinities of 250 μM and 6.2 μM in SPR and high ligand efficiency (LE) of 0.38 for both. Given the low log <i>D</i><sub>7.4</sub> (0.23) of compound <b>1</b>, its lipophilic ligand efficiency (LLE) was a promising 3.4, and overall <b>1</b> was deemed a good starting point for a hit-to-lead campaign (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). Quinazoline <b>2</b> was a highly lipophilic compound, with a log <i>D</i><sub>7.4</sub> of 4.1 and a LLE of 1.1. Crystal structures were obtained of MAT2a in complex with the fragment hits, showing that they bound in the same site as the regulatory subunit MAT2b. This site has two identical halves, being located at the dimer interface and on the rotational 2-fold axis of cyclic C2 symmetry (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">S1</a>). While MAT2b binds once per dimer in a 1:2 molar ratio, small-molecule ligands bound with 1:1 stoichiometry, each ligand occupying one-half of the binding site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Ligand binding caused the side chain of Phe333 to rotate by 180°, reshaping the site from a short and wide pocket to a long and narrow cleft. In its new orientation, Phe333 partially occluded solvent access (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Figure S1</a>). Moreover, the position vacated by this side chain formed a cryptic pocket occupied by the ethoxy group of <b>1</b> and the chlorine of <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B–D).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Fragment binding induces changes in the allosteric site. (A) Fragment hits <b>1</b> and <b>2</b> and their MAT2a binding affinities as determined by SPR. (B) Crystal structure of <b>2</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs">7bhs</a>, 1.1 Å) shows two molecules of the ligand (green sticks and yellow surface) occupying the symmetric site at the dimer interface. Overlay of the MAT2b-bound state (purple sticks and turquoise surface) illustrates the cryptic pocket accessed by small-molecule ligands. (C, D) Close-up view of the binding site and interactions formed by triazinone hit <b>1</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhr">7bhr</a>, 1.1 Å) and quinazoline hit <b>2</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs">7bhs</a>, 1.1 Å) with Arg313.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Both fragments formed a bidentate interaction with Arg313 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D), which assumed slightly different conformations in the two crystal structures. The smaller compound, <b>1</b>, protruded into the cryptic pocket with its ethoxy substituent, while the dimethylamine pointed toward solvent. There was an additional contact to a water molecule in the subsite that harbors a His or Phe side chain in the MAT2b complex with the dimeric MAT2a (“aromatic side chain pocket”, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Compound <b>2</b> displaced this water and occupied the subsite with a phenyl moiety while positioning a chlorine in the cryptic pocket. Given that this site ordinarily harbors a phenylalanine side chain, it offers predominantly lipophilic contacts.</div><div class="NLM_p">It was important to determine whether binding of compounds at the allosteric site translated to inhibition of enzyme activity. The hits were, therefore, further characterized in an enzymatic functional assay; the assay followed the formation of free phosphate (<a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>), which is a product of the ATP dependent MAT2a-catalyzed reaction converting methionine to SAM (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">eq S1</a>).</div><div class="NLM_p last">Despite its small size and modest affinity, <b>1</b> showed inhibition with an IC<sub>50</sub> of 206 μM, which correlated well with its binding affinity, while <b>2</b> was surprisingly inactive in this assay.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Initial SAR and Fragment Merging</h3><div class="NLM_p">Initial exploration of triazinone <b>1</b> showed a very tight SAR, with close analogues having no affinity for MAT2a. Key compounds are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Methylation of the triazinone NH in <b>3</b> led to loss of binding, likely due to the perturbation of the water network and the inefficient filling of the aromatic side chain pocket. Conversely, a much bigger phenyl substituent in compound <b>4</b> was tolerated and led to a ∼6-fold increase in binding affinity (<i>K</i><sub>d</sub> = 40 μM vs 250 μM of the initial hit), but unexpectedly, this did not translate into an increase in functional activity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Initial SAR Exploration around Hit <b>1</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0009.gif" alt="" id="GRAPHIC-d7e655-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub> <a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>d</sub> (μM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">LE<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">LLE<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td><td class="colsep0 rowsep0" align="char" char=".">250</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td><td class="colsep0 rowsep0" align="char" char=".">206</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">0.66</td><td class="colsep0 rowsep0" align="char" char=".">>200</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">0.32</td><td class="colsep0 rowsep0" align="char" char=".">307</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">log <i>D</i><sub>7.4</sub> determined by shake flask method.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">All <i>K</i><sub>d</sub> and IC<sub>50</sub> data are the mean values of at least <i>n</i> ≥ 3 independent measurements. Each has a SD ± 0.3 log units unless otherwise noted. See <a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Supporting Information</a> for details.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Ligand efficiency (LE) = 1.4·p<i>K</i><sub>d</sub>/(heavy atom count).</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Lipophilic ligand efficiency (LLE) = pIC<sub>50</sub> – log <i>D</i><sub>7.4</sub>.</p></div></div></div><div class="NLM_p">Following these initial findings, a different core was proposed based on the overlay of the two fragment hits <b>1</b> and <b>2</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). This new core, 2-quinazolinone <b>5</b>, not only maintained the binding affinity of the most potent of the two hits but also had a much better ligand efficiency due to its small size (LE = 0.49 vs 0.38 of each of the two hits). Moreover, <b>5</b> had a significant improvement in inhibitory potency of 8.3 μM and given its log <i>D</i><sub>7.4</sub> of 1.3 had a promising LLE of 3.8. The quinazolinone was found to bind as expected, maintaining the bidentate interaction of <b>1</b> with Arg313 and occupying more fully the lipophilic cryptic pocket where the ethoxy substituent of <b>1</b> was located. The Phe333 side chain displaced from this site participated in a hydrophobic stacking interaction with the ligand, concomitantly shielding the compound from solvent. The ligand participated in two additional π-stacking interactions with the side chains of Phe20 and Trp274. The crystal structure also clearly showed a distinct divergence from planarity in this compound, with the dimethylamine group on the one hand, and the carbonyl O atom on the other, being on opposite sides of the extended plane of the phenyl ring (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). This substituent effect on aromatic ring conformation appeared to originate from steric interaction between the aromatic CH in position 5 and the alkylamine substituent in position 4.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Merging strategy from hits <b>1</b> and <b>2</b> to quinazolinone <b>5.</b> Binding mode confirmed by crystallography (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bht">7bht</a>, 1.1 Å) shows that key interactions are maintained, notably the bidentate hydrogen bonding to Arg313 and hydrophobic stacking with Phe333. Compound <b>5</b> is visibly nonplanar as seen in the electron density (blue mesh shows 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1σ); see also <a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Figure S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The binding site of <b>5</b> was analyzed computationally by characterizing putative water sites using the software WaterMap<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(19,20)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Figure S2</a>). Three water sites, corresponding to the crystallographic waters seen near N1, were determined computationally to have excess free energies Δ<i>G</i> significantly greater than 0 compared to bulk water (i.e., “unstable”). The water site nearest N1, at about 3.4 Å distance, had the greatest calculated excess free energy (Δ<i>G</i> = +6.8 kcal/mol) and was potentially displaceable by substitution off the N1 position.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> SAR Understanding of the Quinazolinone Core and Fragment Growing</h3><div class="NLM_p">With this new core in hand, the first step was to understand the key pharmacophoric elements. The chlorine and the dimethylamine substituents were both found to be essential for activity, with compounds <b>6</b> and <b>7</b> being inactive (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Methylamine <b>8</b> lost activity (IC<sub>50</sub> = 55 μM), and primary amine <b>9</b> was completely inactive. Acetylating the amine (compound <b>10</b>) was not tolerated. The O-linked compounds (<b>11</b> and <b>12</b>) and the phenyl analogue <b>13</b> were also inactive. This could be due to a combination of reasons, ranging from disrupting the water network around the Arg313, steric clashes, and/or the inability of the compounds to adopt a nonplanar conformation similar to that of compound <b>5</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Key SAR around the Quinazolinone Core and Alternative 4-Substituents<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0010.gif" alt="" id="GRAPHIC-d7e942-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">LLE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">N(Me)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">8.3</td><td class="colsep0 rowsep0" align="left">3.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left">2.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">NHAc</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">OPh</td><td class="colsep0 rowsep0" align="left">2.7</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Legend as in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></div></div><div class="NLM_p">Further investigation focused on exploring the tertiary amine SAR, and the results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Compound <b>14</b> bearing a methylethylamine substituent showed an improvement in both potency and LLE compared to <b>5</b> (IC<sub>50</sub> = 0.74 μM, LLE = 4.4 vs IC<sub>50</sub> = 8.3 μM, LLE = 3.8). In an attempt to make an interaction with Glu342, a base was appended to the ethyl substituent (<b>15</b>), but this change abolished all activity. An <i>N</i>-linked phenyl substituent was not tolerated (<b>16</b>), while <i>N</i>-benzyl <b>17</b> had some residual activity but was ∼10-fold weaker than the starting point and with a much lower LLE. An ethyl linker to the Ph resulted in a complete loss of potency (<b>18</b>). Cyclic amines were then investigated, and while the azetidine <b>19</b> was inactive, the larger pyrrolidine substituent in <b>20</b> had a similar activity to <b>5</b>. A contributing factor to this piece of SAR may be the peculiar substituent effect on the conformation of the quinazolinone ring system. Quantum-mechanical calculations suggested that the active compounds <b>5</b> and <b>20</b> had energy minima that involve a clear divergence from planarity of the quinazolinone ring system and substituents being partially out-of-plane (as observed by X-ray for <b>5</b> bound to MAT2a and described above; <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), while the inactive <b>19</b> had an essentially flat predicted minimum-energy conformation with coplanar substituents (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Figure S3</a>). The observed out-of-plane conformations of <b>5</b> and <b>20</b> may therefore contribute favorably to their binding to MAT2a. Despite comparable activity to compound <b>5</b>, pyrrolidine <b>20</b> had a higher log <i>D</i><sub>7.4</sub> and a slightly lower LLE of 3.5. Both increasing the size of the ring to six atoms (<b>21</b>) or substitution of the 5-membered ring (<b>22</b>, <b>23</b>, and <b>24</b>) did not improve potency.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Fragment Growth from the Tertiary Amine in Position 4<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0011.gif" alt="" id="GRAPHIC-d7e1190-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0012.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Legend as <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></div><div></div></div><div class="NLM_p">Given the encouraging results from expanding triazinone <b>4</b> into the aromatic side chain pocket (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), substitution of the endocyclic N became the following area of exploration with data summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(21)</a> Me-substituted <b>25</b> was ∼20-fold weaker than the starting point, consistent with the result obtained by methylating the triazinone (<b>3</b> vs <b>1</b>). As stated before, this was believed to be due to perturbation of the water network and the formation of a high-energy void. In order to compensate for the loss of water-mediated interactions with the pocket, it was envisaged that a hydroxyl group at the right distance would be able to form a hydrogen bond with the backbone carbonyl of Gly273. Gratifyingly, compound <b>26</b> was found to be equipotent with <b>5</b>, but given its lower log <i>D</i><sub>7.4</sub>, its LLE increased to 4.4 compared to 3.8 for <b>5</b>, suggesting that the compound was able to efficiently displace the water and form a further productive interaction within the pocket. The crystal structure confirmed this predicted interaction, with the compound adopting two distinct conformations that positioned the hydroxyl within hydrogen bonding distance of Gly273 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). The <sup><i>c</i></sup>Pr substituent in <b>27</b> was also tolerated but did not bring any advantage in terms of potency or efficiency. In the original triazinone core the H to phenyl modification had led to a ∼6-fold improvement in affinity but not to an increase in biochemical potency. Conversely, in this elaborated scaffold, the addition of a Ph substituent led to <b>28</b> with a biochemical potency of 22 nM and a LLE of 5.6, bringing an almost 400-fold increase in potency compared to the parent <b>5</b>. The crystal structure showed the additional ring effectively displacing two water molecules and interacting with Phe333 in a CH−π interaction. The greatly enhanced potency and efficiency of <b>28</b> can be rationalized in terms of the good complementarity between the ligand and the pocket as well as the low rotational freedom of the ligand in the unbound state. Also, the displacement of two “unstable” water molecules (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Figures S2 and 4B</a>) likely contributed to its potency gain. From further analysis of the crystal structure, a small lipophilic pocket was identified and a methyl substituent in the meta position was envisaged to fill it. The resulting compound <b>29</b> had a biochemical potency of 16 nM, similar to <b>28</b>, but because it was more lipophilic, the LLE was not improved. Given the water network present in the aromatic side chain pocket in the absence of the Ph, it was hypothesized that a nitrogen in the ring could stabilize this network by interacting with one of the waters. 3-Pyridyl <b>31</b> was found to be optimal with an IC<sub>50</sub> of 21 nM and an impressive LLE of 6.9. The crystal structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D) showed a perfect overlay with the phenyl analogue <b>28</b> and a water-mediated interaction between the pyridine N of the ligand and the protein. The electron-poor nature of the pyridine likely increased the ability of the proton in position 5 to form a weak CH-hydrogen bond with the Gly273 carbonyl group, all contributing to the potency and the efficiency of <b>31</b>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Growth from Position 1<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0013.gif" alt="" id="GRAPHIC-d7e1296-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0014.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Legend as <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Standard deviation is 0.35.</p></div></div><div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Optimization of the substituent occupying the aromatic side chain subsite. (A) The hydroxyethyl substituent in <b>26</b> forms a hydrogen bond with the backbone carbonyl of Gly273. Two alternative conformations were observed in the crystal structure, shown as light and dark gray sticks (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhu">7bhu</a>, 1.2 Å). (B) The phenyl substitution in <b>28</b> (pink) brings an almost 400-fold potency boost compared to <b>5</b> (orange). The superposition of both crystal structures shows that the phenyl group displaces two water molecules. The side chain of Phe333 undergoes a slight rotation, maximizing a CH−π interaction (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhv">7bhv</a>, 1.2 Å). (C) <b>29</b> (yellow, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhw">7bhw</a>, 1.2 Å) fills a small hydrophobic side pocket with a methyl substituent. (D) <b>31</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhx">7bhx</a>, 1.1 Å) features a pyridine to capture a water interaction. A weak CH-hydrogen bond to Gly273 is indicated by gray dashes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Activity in Cell and Combination of SAR</h3><div class="NLM_p">A selection of compounds were profiled in a cellular assay which measures symmetric dimethyl arginine (SDMA) levels. Symmetric arginine methylation is a downstream measure of MAT2a activity through PRMT5, which is inhibited by being starved of its substrate SAM. The phenyl quinazolinone <b>28</b> showed a potent IC<sub>50</sub> of 25 nM in this assay. Given the promising effect in cells, the following step was to relate MAT2a inhibition to the ability of the compounds to inhibit cellular proliferation. Gratifyingly, compound <b>28</b> was able to inhibit proliferation of HCT116 MTAP knockout cells with an IC<sub>50</sub> of 250 nM. Pyridine <b>31</b> showed a similar cellular potency and proliferation inhibition for a lower log <i>D</i>.</div><div class="NLM_p last">A final round of optimization combined the <i>N-</i>4-ethyl, identified from compound <b>14</b>, with the optimal aryl groups from <b>28</b> and <b>31</b> generating <b>32</b> and <b>33</b>, respectively. Both these compounds showed single digit nanomolar potency in cells and proliferation inhibition potencies of 140 nM.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Pharmacokinetics</h3><div class="NLM_p"><i>In vitro</i> DMPK properties of compounds <b>28</b> and <b>31—33</b> were profiled. These four compounds displayed moderate to high solubility, high permeability, and low efflux, and they also showed low to moderate human hepatocyte intrinsic clearance (CL<sub>int</sub>) (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) which generally tracked with log <i>D</i> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). The compounds were screened for inhibition against a panel of five cytochrome P450 isoforms (CYP1A2, CYP2C9, CYP2C19, CYP2D6, and CYP3A4) in human liver microsomes. Only weak inhibition of CYP1A2 and CYP3A4 was observed for compounds <b>32</b> and <b>33</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) with all others showing no activity (IC<sub>50</sub> > 30 μM). No compound showed activity against the hERG ion channel up to 40 μM.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Combination of Substituents in Positions 1 and 4<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0015.gif" alt="" id="GRAPHIC-d7e1441-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i><sub>7.4</sub></th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">LLE</th><th class="colsep0 rowsep0" align="center" char=".">cell IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">prolif IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">0.022</td><td class="colsep0 rowsep0" align="char" char=".">5.6</td><td class="colsep0 rowsep0" align="char" char=".">0.025</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char=".">0.82</td><td class="colsep0 rowsep0" align="char" char=".">0.021</td><td class="colsep0 rowsep0" align="char" char=".">6.9</td><td class="colsep0 rowsep0" align="char" char=".">0.017</td><td class="colsep0 rowsep0" align="char" char=".">0.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">C</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">0.013</td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="char" char=".">0.0063</td><td class="colsep0 rowsep0" align="char" char=".">0.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">0.018</td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">0.0052</td><td class="colsep0 rowsep0" align="char" char=".">0.14<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Legend as <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Average of two repeats.</p></div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. <i>In Vitro</i> Human DMPK Properties</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" rowspan="2" align="center" valign="middle">compd</th><th class="colsep0 rowsep0" align="center" char=".">human hep CL<sub>int</sub></th><th class="colsep0 rowsep0" align="center" char=".">human</th><th class="colsep0 rowsep0" align="center">Caco2</th><th class="colsep0 rowsep0" align="center" char=".">solubility</th><th class="colsep0 rowsep0" rowspan="2" align="center" valign="middle" char=".">CYP3A4<a class="ref internalNav" href="#t6fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" rowspan="2" align="center" valign="middle" char=".">CYP1A2<a class="ref internalNav" href="#t6fn5" aria-label="e">e</a></th><th class="colsep0 rowsep0" rowspan="2" align="center" valign="middle" char=".">hERG<a class="ref internalNav" href="#t6fn6" aria-label="f">f</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">(μL/min)/10<sup>6</sup> cells <a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">% free<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><i>P</i><sub>app</sub>/ER<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char=".">μM<a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="left">29/0.97</td><td class="colsep0 rowsep0" align="char" char=".">103</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="left">64/1.6</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">8.1</td><td class="colsep0 rowsep0" align="left">73/0.86</td><td class="colsep0 rowsep0" align="char" char=".">220</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="left">44/2.2</td><td class="colsep0 rowsep0" align="char" char=".">930</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">>30</td><td class="colsep0 rowsep0" align="char" char=".">>40</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Rate of metabolism ((μL/min)/10<sup>6</sup> cells) determined from DMSO stock solution in isolated hepatocytes diluted to 10<sup>6</sup> cells/mL.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Determined from DMSO stock solution by equilibrium dialysis in 10% human plasma supplied by BioreclamationIVT.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Compounds were incubated at 10 μM in cultured Caco2 cells. Intrinsic permeability was measured in units of ×10<sup>–6</sup> cm/s in the apical direction in the presence of transport inhibitors quinidine (50 μM), sulfasalazine (20 μM), and benzbromarone (30 μM) with an apical/basolateral pH of 6.5:7.4, respectively. Efflux ratio is reported as the ratio of basolateral (B to A) to apical (A to B) transport in the absence of any inhibitors and an apical/basolateral pH of 7.4:7.4.</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last">Solubility of compounds in aqueous phosphate buffer at pH 7.4 after 24 h at 25 °C from DMSO stock solution.</p></div><div class="footnote" id="t6fn5"><sup><sup>e</sup></sup><p class="last">Inhibition of cytochrome P450 enzymes IC<sub>50</sub>.</p></div><div class="footnote" id="t6fn6"><sup><sup>f</sup></sup><p class="last">Inhibition of the hERG tail current was measured using a plate-based planar patch clamp system (Syncropatch).</p></div></div></div><div class="NLM_p">Low to moderate CL<sub>int</sub> values were observed in rat hepatocytes which corresponded to low to moderate <i>in vivo</i> rat clearance (noncompartmental) (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Bioavailability in rat (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>) was moderate to high (57–100%) consistent with high absorption expected given the <i>in vitro</i> DMPK properties of these compounds. The volume of distribution was generally moderate and consistent with the neutral ion class (0.56–2.1 L/kg). <b>28</b> and <b>31</b> were selected to be evaluated in mouse pharmacokinetic studies to understand suitability for pharmacodynamic studies.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. <i>In Vitro</i> and <i>in Vivo</i> Rodent DMPK Properties<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char=".">CL<sub>int</sub> ((μL/min)/10<sup>6</sup> cells)</th><th class="colsep0 rowsep0" align="center" char=".">% free</th><th class="colsep0 rowsep0" align="center" char=".">Cl ((mL/min)/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="char" char=".">4.6</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">0.85</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">45</td><td class="colsep0 rowsep0" align="char" char=".">19</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">0.43</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">rat</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="char" char=".">0.56</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="char" char=".">67</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">109</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">mouse</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">56</td><td class="colsep0 rowsep0" align="char" char=".">62</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">3.7</td><td class="colsep0 rowsep0" align="char" char=".">79</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Compound was dosed in rat intravenously at 0.5 and orally at 1 mg/kg. The formulation was 5% DMSO/95% hydroxylpropyl β-cyclodextrin (30% w/v) at a volume of 1 mL/kg and 4 mL/kg, iv and po, respectively.</p></div></div></div><div class="NLM_p">Unfortunately, <i>in vitro</i> mouse metabolism was much higher than rat or human for these compounds, which correlated with much higher mouse <i>in vivo</i> clearance when compared with rat (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). Due to the higher mouse clearance, both subcutaneous (sc) dosing and oral (po) dosing of <b>28</b> and <b>31</b> were evaluated at high doses to enable <i>in vivo</i> efficacy studies (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The lower solubility of <b>28</b> relative to <b>31</b> (103 vs >1000 μM) allowed for a suspension SC formulation (0.5% w/v hydroxypropylmethylcellulose/0.1% w/v Tween 80) which displayed dissolution rate limited kinetics that prolonged plasma half-life and lowered <i>C</i><sub>max</sub>. Conversely, <b>31</b> displayed rapid absorption from both sc and po administration displaying similar profiles due to higher solubility as solution formulations (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Due to the favorable PK profile of sc dosing <b>28</b>, this compound and formulation were progressed to target engagement and efficacy studies. 50 mg/kg sc dosing of <b>28</b> was utilized in target engagement and efficacy studies after observing similar exposure and prolonged half-life at both 50 mg/kg and 100 mg/kg sc after repeat dosing (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Figure S4</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mouse plasma concentrations of <b>28</b> po (blue circle), <b>28</b> sc (red square), <b>31</b> po (green triangle), and <b>31</b> sc (purple triangle) at 100 mg/kg.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> <i>In Vivo</i> Target Engagement and Efficacy</h3><div class="NLM_p"><i>In vivo</i> profiling of <b>28</b> was carried out through an efficacy study in an HCT116 MTAP knockout xenograft tumor model. 50 mg/kg of <b>28</b> was administered once daily sc to female Ncr nude mice and resulted in an average reduction of tumor SAM concentration by 74% compared to vehicle control (0.5% w/v HPMC/0.1% w/v Tween 80) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Consequently, a significant antitumor response was observed leading to tumor stasis (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, left). However, while compound <b>28</b> was tolerated over the course of the 28 day treatment, mice did exhibit a maximum body weight loss of 8% (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B, right).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Subcutaneous dosing of <b>28</b> at 50 mg/kg QD: (A) reduction in tumor SAM; (B) antitumor response (left) and body weight change (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis of Selected Compounds</h3><div class="NLM_p">Quinazolinone <b>5</b> was prepared by S<sub>N</sub>Ar reaction on 2,4,7-trichloroquinazoline <b>5a</b> with dimethylamine, followed by hydrolysis with acetic acid (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dimethylamine, THF, rt, 2 h; (b) acetic acid, 100 °C, 12 h, 29% over two steps.</p></p></figure><div class="NLM_p">Quinazolinones <b>28</b> and <b>31</b>–<b>33</b> were synthesized as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Commercial 4-chloro-2-fluorobenzamide was treated with oxalyl chloride to form the isocyanate <i>in situ</i>. The reaction was then cooled to room temperature and the corresponding aniline added to form acyl ureas <b>34a</b> and <b>34b</b>. Intermediate <b>34</b> was carried through to the next step, together with an oxalamide byproduct impurity (approximately 20–25%) identified after purification. A KHMDS-mediated cyclization of <b>34</b> was used to generate the quinazolinone ring (intermediates <b>35a</b> and <b>35b</b>). PyBOP coupling with the corresponding amine completed the synthesis of the desired compounds <b>28</b>, <b>31</b>, <b>32</b>, and <b>33</b> in respectable yields (50–92%).</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>28</b>, <b>31</b>, <b>32</b>, and <b>33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) oxalyl chloride, DCE, 80 °C, 30 min; (b) aniline, rt, 1 h, 62–75% over two steps; (c) KHMDS, THF, −78 °C, 1 h, then rt, 3.5 h, 67–80%; (d) PyBOP, DBU, amine, MeCN, rt, 2.5 h, 50–92%.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">A series of MAT2a inhibitors was developed originating from weak fragment hits through a structure-based merging strategy. This led to quinazolinone <b>5</b> which was a low molecular weight, potent, and efficient compound and formed the basis for further optimization. By use of structure-based design, fragment growing led to compounds with over 1000-fold improvement in potency over the original fragment hits together with excellent physicochemical properties. Arylquinazolinones <b>28</b> and <b>31</b>–<b>33</b> showed inhibition of PRMT5-mediated symmetric dimethylation of arginine (SDMA) and antiproliferative activity in MTAP null cells. Compound <b>28</b> was identified as a promising <i>in vivo</i> tool that exhibited reduction in tumor SAM levels and antitumor response in an MTAP knockout HCT116 xenograft model.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63250" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63250" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Unless otherwise stated, commercially available reagents were used as supplied. Preparative reverse phase HPLC was performed on an Agilent 1290 Infinity II Preparative system equipped with a SQ MS detector (multimode ESI/APCI source), with a Waters CSH C18 OBD column (5 μm silica, 30 mm diameter, 100 mm length, flow rate of 50 mL/min) using decreasingly polar mixtures of water (containing 0.1—0.3% aqueous ammonium) or water (containing 0.1% formic acid) and acetonitrile as eluents. Preparative SFC purification was performed on either a Sepiatec P100 SFC system or Waters Prep 100 SFC system equipped with QDa MS detector, using the chromatographic conditions as detailed in corresponding experimental data. Proton magnetic resonance spectra were determined using a Bruker Avance 500 (500 MHz) and Bruker Avance 400 (400 MHz) instruments and were determined in CDCl<sub>3</sub>, DMSO-<i>d</i><sub>6</sub>, or MeOH-<i>d</i><sub>4</sub>. Measurements were taken at ambient temperature unless otherwise specified. The following abbreviations have been used: s, singlet; d, doublet; t, triplet; q, quartet; m, multiplet; dd, doublet of doublets; ddd, doublet of doublets of doublets; dt, doublet of triplets; tt, triplet of triplets; bs, broad signal. End products were also characterized by mass spectrometry following liquid chromatography (LCMS or UPLC). Reverse-phase C18 silica was used with a flow rate of 1 mL/min, and detection was by electrospray mass spectrometry and by UV absorbance recording a wavelength range of 220–320 nm. Analytical UPLC was performed on CSH C18 reverse-phase silica, using a Waters Acquity UPLC CSH C18 column with dimensions 2.1 mm × 50 mm and particle size 1.7 μm). Gradient analysis was employed using decreasingly polar mixtures as eluent, for example, decreasingly polar mixtures of water (containing 0.1% formic acid or 0.1% ammonia) as solvent A and acetonitrile as solvent B. A typical 2 min analytical UPLC method would employ a solvent gradient over 1.3 min, at approximately 1 mL/min, from a 97:3 mixture of solvents A and B respectively to a 3:97 mixture of solvents A and B. The reported molecular ion corresponds to the [M + H]<sup>+</sup> unless otherwise specified. For molecules with multiple isotopic patterns (Br, Cl, etc.) the reported value is the one obtained for the lowest isotope mass unless otherwise specified. Purities were assessed using LCMS by UV absorbance and <sup>1</sup>H NMR and are ≥95% for screening compounds.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4-Chloro-2-fluoro-<i>N</i>-(phenylcarbamoyl)benzamide (<b>34a</b>)</h3><div class="NLM_p last">To a stirred suspension of 4-chloro-2-fluorobenzamide (10.0 g, 57.6 mmol) in DCE (200 mL) at rt was added oxalyl dichloride (5.36 mL, 63.4 mmol). The resultant suspension was heated to 80 °C for 30 min. The reaction mixture was allowed to cool to rt, and aniline (10.5 mL, 115 mmol) was added to the reaction mixture (on addition a suspension formed). The resultant reaction mixture was stirred at rt for 1 h. The reaction mixture was cooled using an ice bath and was then quenched by carefully pouring into a saturated aqueous solution NaHCO<sub>3</sub> (1 L). This was stirred for approximately 10 min. The precipitate was collected by filtration, washed with water (1 L), and dried under vacuum to afford 4-chloro-2-fluoro-<i>N</i>-(phenylcarbamoyl)benzamide <b>34a</b> (75% pure; 16.8 g, 99%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): 7.12 (tt, <i>J</i> = 7.8, 1.1 Hz, 1H), 7.31–7.4 (m, 2H), 7.41–7.48 (m, 1H), 7.52–7.6 (m, 2H), 7.63 (dd, <i>J</i> = 10.3, 1.9 Hz, 1H), 7.68–7.77 (m, 1H), 10.37 (s, 1H), 11.06 (s, 1H). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 293.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 7-Chloro-1-phenylquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>35a</b>)</h3><div class="NLM_p last">To a stirred white suspension of 4-chloro-2-fluoro-<i>N</i>-(phenylcarbamoyl)benzamide <b>34a</b> (75% pure; 16.8 g, 43.0 mmol) in anhydrous THF (100 mL) under a nitrogen atmosphere at −78 °C was added KHMDS (126 mL, 1 M in THF solution, 126 mmol) dropwise. On addition, the reaction mixture turned an orange color. The reaction mixture was stirred at −78 °C for 1 h and then warmed to rt for 3.5 h. The reaction mixture was slowly quenched with saturated aqueous NH<sub>4</sub>Cl solution. A precipitate was formed to give a thick suspension, which was filtered and washed with water to afford the crude product. The crude product was stirred in approximately 200 mL of water for 30 min. The white suspension was filtered, washing the solids with water, dried under vacuum at 50 °C to afford 7-chloro-1-phenylquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione <b>35a</b> (85% pure; 14.7 g, 80%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): 6.32 (d, <i>J</i> = 1.8 Hz, 1H), 7.31 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 7.42–7.5 (m, 2H), 7.53–7.68 (m, 3H), 8.01–8.08 (m, 1H), 11.79 (s, 1H). <i>m</i>/<i>z</i>: ES– [M – H]<sup>−</sup> 271.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 7-Chloro-4-(dimethylamino)-1-phenylquinazolin-2(1<i>H</i>)-one (<b>28</b>)</h3><div class="NLM_p last">DBU (11.0 mL, 73.9 mmol) was added dropwise to a stirred suspension of 7-chloro-1-phenylquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione <b>35a</b> (85% pure, 6.77 g, 21.1 mmol), PyBOP (16.5 g, 31.7 mmol), and dimethylamine (26.4 mL, 2 M in THF, 52.8 mmol) in acetonitrile (110 mL) at rt. The resultant suspension was stirred at rt for 2.5 h. Additional PyBOP (4.00 g, 10.6 mmol), dimethylamine (7.0 mL, 2 M in THF, 14 mmol), and DBU (3.0 mL, 20 mmol) were added, and the reaction mixture was left stirring at rt for 30 min. The reaction mixture was concentrated to dryness. The crude mixture was diluted with DCM and washed sequentially with water and saturated brine. The organic layer was dried with a phase separating cartridge, filtered, and concentrated to afford the crude product as a brown oil (approximately 31 g). The crude product was dissolved in acetonitrile (80 mL). A fine white precipitate started to crash out. The suspension was slowly passed through a phase separator cartridge into a stirred flask with water (800 mL). Upon addition a solid precipitated. This was carefully filtered and the solid washed with water. The solid was dried under vacuum at 50 °C to afford the crude product (6.78 g) as a light brown solid. The crude sample (6.78 g) was dissolved to a concentration of 113 mg/mL in 60 mL of DCM/DMF/DMSO (1:1:1), heated, sonicated, and filtered. The resulting solution was purified by preparative SFC (column, YMC Diol, 30 mm × 250 mm, 5 μm, mobile phase of 15–25% MeOH + 0.1% NH<sub>3</sub>/85–75% scCO<sub>2</sub> over 7 min, 1.5 min flush at 50% MeOH; flow rate of 100 mL/min; BPR, 120 bar; column temperature of 40 °C). Fractions containing the desired product were concentrated to dryness to afford 7-chloro-4-(dimethylamino)-1-phenylquinazolin-2(1<i>H</i>)-one <b>28</b> (5.80 g, 92%) as an off white solid. <sup>1</sup>H NMR (400 MHz, DMSO): 3.31 (s, 6H), 6.33 (d, <i>J</i> = 2.1 Hz, 1H), 7.19 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.26–7.35 (m, 2H), 7.45–7.56 (m, 1H), 7.60 (ddd, <i>J</i> = 7.7, 6.5, 1.3 Hz, 2H), 8.04 (d, <i>J</i> = 8.8 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO, 37 °C) δ: 41.3, 108.6, 114.0, 120.3, 128.5, 129.0, 129.4, 129.9, 137.4, 137.6, 146.2, 153.4, 163.5. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 300; HRMS (ESI) for C<sub>16</sub>H<sub>14</sub>ClN<sub>3</sub>O (MH<sup>+</sup>) calculated 300.08982; found 300.08911.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 4-Chloro-2-fluoro-<i>N</i>-(pyridin-3-ylcarbamoyl)benzamide (<b>34b</b>)</h3><div class="NLM_p last">To a stirred suspension of 4-chloro-2-fluorobenzamide (7.50 g, 43.2 mmol) in DCE (150 mL) at rt was added oxalyl dichloride (4.02 mL, 47.5 mmol). The resultant suspension was heated to 80 °C for 45 min. The reaction mixture was allowed to cool to rt, and pyridin-3-amine (8.13 g, 86.4 mmol) was added. The reaction mixture was then stirred at rt for 4 h. The reaction mixture was cooled using an ice bath, and the resultant suspension was filtered, washing the solids with DCM. The solid was suspended in a mixture of MeOH/DCM (1:1). The suspension was filtered and the solid dried under vacuum to afford 4-chloro-2-fluoro-<i>N</i>-(pyridin-3-ylcarbamoyl)benzamide <b>34b</b> (85% pure; 9.26 g, 73%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): 7.36–7.5 (m, 2H), 7.64 (dd, <i>J</i> = 10.3, 1.9 Hz, 1H), 7.74 (t, <i>J</i> = 8.0 Hz, 1H), 8.10 (ddd, <i>J</i> = 8.4, 2.6, 1.5 Hz, 1H), 8.35 (dd, <i>J</i> = 4.8, 1.4 Hz, 1H), 8.79 (d, <i>J</i> = 2.1 Hz, 1H), 10.46 (s, 1H), 11.21 (s, 1H). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 294.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 7-Chloro-1-(pyridin-3-yl)quinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>35b</b>)</h3><div class="NLM_p last">To a stirred solution of 4-chloro-2-fluoro-<i>N</i>-(pyridin-3-ylcarbamoyl)benzamide <b>34b</b> (85% pure; 9.26 g, 26.8 mmol) in anhydrous THF (80 mL) under a nitrogen atmosphere at −78 °C was added potassium bis(trimethylsilyl)amide (69.4 mL, 1 M in THF, 69.4 mmol) dropwise. On addition the reaction mixture turned a dark brown color. The reaction mixture was stirred at −78 °C for 1 h and then warmed to rt for 18 h. The reaction mixture was slowly quenched with saturated aqueous NH<sub>4</sub>Cl solution during which a solid precipitated to give a thick suspension, which was filtered and washed with acetonitrile to afford the crude product (13.4 g) as an off white solid. The crude product was stirred in water (100 mL) for 30 min. The white suspension was filtered, washing the solids with water and diethyl ether, dried under vacuum at 50 °C to afford 7-chloro-1-(pyridin-3-yl)quinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione <b>35b</b> (80% pure; 6.16 g, 84%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): 6.40 (d, <i>J</i> = 1.7 Hz, 1H), 7.34 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 7.68 (ddd, <i>J</i> = 8.1, 4.8, 0.8 Hz, 1H), 7.98 (ddd, <i>J</i> = 8.1, 2.5, 1.6 Hz, 1H), 8.06 (d, <i>J</i> = 8.4 Hz, 1H), 8.68 (dd, <i>J</i> = 2.5, 0.7 Hz, 1H), 8.76 (dd, <i>J</i> = 4.8, 1.5 Hz, 1H), 11.76 (s, 1H). <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 274.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 7-Chloro-4-(dimethylamino)-1-(pyridin-3-yl)quinazolin-2(1<i>H</i>)-one (<b>31</b>)</h3><div class="NLM_p last">DBU (1.91 mL, 12.8 mmol) was added dropwise to a stirred suspension of 7-chloro-1-(pyridin-3-yl)quinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione <b>35b</b> (80% pure; 1.25 g, 3.65 mmol), PyBOP (2.85 g, 5.48 mmol), and dimethylamine (4.57 mL, 2 M in THF, 9.13 mmol) in acetonitrile (30 mL) at rt. The resulting yellow solution was stirred at rt for 18 h. The reaction mixture was concentrated to dryness. The crude mixture was diluted with DCM and washed sequentially with water and saturated brine. The organic layer was dried with a phase separating cartridge, filtered, and concentrated to dryness to afford the crude product as an orange oil (approximately 4 g). The sample was dissolved in DMSO and purified by preparative SFC (column, YMC Cyano 30 × 250 mm, 5 μm, mobile phase of 10–12% MeOH + 0.1% NH<sub>3</sub>/90—88% scCO<sub>2</sub> over 8 min, 1.5 min flush at 50% MeOH + 0.1% NH<sub>3</sub>; flow rate of 100 mL/min; BPR of 120 bar; column temperature of 40 °C). Fractions containing the desired product were concentrated to dryness to afford 7-chloro-4-(dimethylamino)-1-(pyridin-3-yl)quinazolin-2(1<i>H</i>)-one <b>31</b> (0.945 g, 86%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): 3.32 (s, 6H), 6.38 (d, <i>J</i> = 2.1 Hz, 1H), 7.23 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.64 (ddd, <i>J</i> = 8.1, 4.8, 0.7 Hz, 1H), 7.84 (ddd, <i>J</i> = 8.1, 2.5, 1.6 Hz, 1H), 8.06 (d, <i>J</i> = 8.8 Hz, 1H), 8.54 (dd, <i>J</i> = 2.5, 0.7 Hz, 1H), 8.71 (dd, <i>J</i> = 4.8, 1.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO, 37 °C) δ: 41.4, 108.7, 113.9, 120.7, 124.7, 129.6, 134.4, 137.2, 137. 7, 146.0, 149.4, 150.0, 153.4, 163.6. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 301. HRMS (ESI) for C<sub>15</sub>H<sub>13</sub>ON<sub>4</sub>Cl (MH<sup>+</sup>) calculated 301.08507; found 301.08453.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 7-Chloro-4-(ethyl(methyl)amino)-1-phenylquinazolin-2(1<i>H</i>)-one (<b>32</b>)</h3><div class="NLM_p last">DBU (0.148 mL, 0.99 mmol) was added dropwise to a stirred suspension of 7-chloro-1-phenylquinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione <b>35a</b> (85% pure; 95 mg, 0.28 mmol), PyBOP (220 mg, 0.42 mmol), and <i>N</i>-methylethanamine (0.061 mL, 0.71 mmol) in acetonitrile (1.0 mL) at rt. The resultant suspension was stirred at rt for 2.5 h. Additional <i>N</i>-methylethanamine (0.061 mL, 0.71 mmol), PyBOP (220 mg, 0.42 mmol), and DBU (0.15 mL, 0.99 mmol) were added, and the resultant reaction mixture was stirred at rt for 2 h. The reaction mixture was concentrated to dryness, and the crude residue was diluted with DCM. The organic layer was washed sequentially with water and saturated brine, dried with a phase separating cartridge, filtered, and evaporated to afford the crude product as a brown oil (753 mg). The crude sample (753 mg) was dissolved in DMSO (6 mL) and was purified by preparative HPLC (Waters CSH C18 OBD column, 30 mm × 100 mm id, 5 μm particle size), using decreasingly polar mixtures of water (containing 0.3% NH<sub>3</sub> aq) and MeCN as eluents, followed by a further step using decreasingly polar mixtures of water (containing 0.1% formic acid (aq)) and MeCN as eluents. Fractions containing the desired product were concentrated to dryness to afford 7-chloro-4-(ethyl(methyl)amino)-1-phenylquinazolin-2(1<i>H</i>)-one <b>32</b> (44 mg, 50%) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO, 37 °C) 1.32 (t, <i>J</i> = 7.0 Hz, 3H), 3.30 (s, 3H), 3.71 (q, <i>J</i> = 7.0 Hz, 2H), 6.34 (d, <i>J</i> = 2.1 Hz, 1H), 7.20 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.27–7.35 (m, 2H), 7.47–7.57 (m, 1H), 7.54–7.65 (m, 2H), 7.97 (d, <i>J</i> = 8.8 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO, 37 °C) δ: 12.1, 38.6, 47.0, 108.6, 114.0, 120.4, 128.5, 129.0, 129.1, 129.9, 137.4, 137.6, 146.3, 153.4, 163.0. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 314. HRMS (ESI) for C<sub>17</sub>H<sub>16</sub>ON<sub>3</sub>Cl (MH<sup>+</sup>) calculated 314.10547; found 314.10486.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 7-Chloro-4-(ethyl(methyl)amino)-1-(pyridin-3-yl)quinazolin-2(1<i>H</i>)-one (<b>33</b>)</h3><div class="NLM_p last">DBU (0.12 mL, 0.80 mmol) was added dropwise to a stirred suspension of 7-chloro-1-(pyridin-3-yl)quinazoline-2,4(1<i>H</i>,3<i>H</i>)-dione (80% pure; 125 mg, 0.370 mmol), PyBOP (247 mg, 0.480 mmol), and <i>N</i>-methylethanamine (0.047 mL, 0.55 mmol) in acetonitrile (3.0 mL) at rt. The resulting yellow solution was stirred at room temperature for 2 h. Additional DBU (0.12 mL, 0.80 mmol) was added, and the reaction mixture was warmed to 50 °C for 10 min. The reaction mixture was concentrated to dryness, and the crude residue was diluted with DCM. The organic was washed sequentially with water and saturated brine, dried with a phase separating cartridge, filtered, and concentrated to dryness to afford the crude product (600 mg). The crude product was dissolved in DMF (2.0 mL) and was purified by preparative HPLC (Waters CSH C18 OBD column, 30 mm × 100 mm id, 5 μm particle size), using decreasingly polar mixtures of water (containing 0.3% NH<sub>3</sub> aq) and MeCN as eluents. Fractions containing the desired compound were concentrated to dryness to afford 7-chloro-4-(ethyl(methyl)amino)-1-(pyridin-3-yl)quinazolin-2(1<i>H</i>)-one <b>33</b> (69.0 mg, 60%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO): 1.31 (t, <i>J</i> = 7.1 Hz, 3H), 3.32 (s, 3H), 3.72 (q, <i>J</i> = 7.0 Hz, 2H), 6.38 (d, <i>J</i> = 2.1 Hz, 1H), 7.24 (dd, <i>J</i> = 8.8, 2.1 Hz, 1H), 7.64 (ddd, <i>J</i> = 8.1, 4.8, 0.8 Hz, 1H), 7.85 (ddd, <i>J</i> = 8.1, 2.5, 1.6 Hz, 1H), 7.99 (d, <i>J</i> = 8.8 Hz, 1H), 8.55 (dd, <i>J</i> = 2.5, 0.7 Hz, 1H), 8.71 (dd, <i>J</i> = 4.8, 1.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO, 37 °C) δ: 12.0, 38.7, 47.0, 108.8, 114.0, 120.8, 124.7, 129.3, 134.4, 137.2, 137.6, 146.0, 149.4, 150.1, 153.4, 163.1. <i>m</i>/<i>z</i>: ES+ [M + H]<sup>+</sup> 315. HRMS (ESI) for C<sub>16</sub>H<sub>15</sub>ON<sub>4</sub>Cl (MH<sup>+</sup>) calculated 315.10072; found 315.09995.</div></div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Surface Plasmon Resonance</h3><div class="NLM_p last">SPR experiments were performed using a Biacore 4000, Biacore T200, or Biacore 8K instrument (Cytiva) at 25 °C. A series S sensor chip SA (Cytiva) was docked into the system in 20 mM Bicine-Na, pH 7.5, 100 mM KCl, 5 mM MgCl<sub>2</sub>, 0.1 mM TCEP, 0.05% (w/w) Tween-20, and 2% (v/v) DMSO. The sensor chip surface was conditioned with 3 × 60 s injections of 50 mM NaOH/1 M NaCl at 30 μL/min before capturing biotinylated Avi-MAT2A protein diluted to 100 nM in running buffer on the active flow cells (3000–5000 RU captured). All remaining biotin binding sites were blocked on the active and reference surfaces with a 60 s injection of 100 μM amine-PEG2-biotin (ThermoScientific) in running buffer. Compounds were tested over a suitable dose–response range by either multicycle kinetics or single-cycle kinetics (slowly dissociating compounds).</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Biochemical Assay</h3><div class="NLM_p last">Inhibition of MAT2a activity was determined using BIOMOL GREEN Reagent (Enzo Life Sciences BML-KI102) which measures the free phosphate released during the assay. For protein production, see the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Supporting Information</a>. Assays were performed in 384-well plates (Greiner 781101) in 20 μL volume, 1% (v/v) final DMSO concentration and with the following buffer conditions: 50 mM Tris-HCl (pH 7.4), 5 mM MgCl<sub>2</sub>, 100 mM KCl, 1 mM DTT, and 0.01% Brij23. Twelve point half-log compound concentration–response curves, with a top concentration of 100 μM (1 mM for fragments), were generated from 10 mM (100 mM for fragments) stocks of compounds solubilized in DMSO using an Echo 555 (Labcyte Inc., Sunnyvale, CA). 10 μL of enzyme (MAT2a and inorganic pyrophosphatase) and 10 μL of substrate (methionine and ATP) were added to the compound plate, and reaction was incubated at room temperature for 90 min. Final concentrations in the reaction mix were 40 nM MAT2a, 1 μg/mL inorganic pyrophosphatase, 20 μM methionine, and 70 μM ATP. Enzyme reaction was stopped by addition of 40 μL of BIOMOL GREEN reagent to each well and incubated for 20 min. Absorbance at 620 nm was read using an Envision plate reader. IC<sub>50</sub> values were calculated using a Genedata Screener (Genedata AG, Basel, Switzerland).</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> SDMA Imaging</h3><div class="NLM_p">HCT116 KO-MTAP cells were cultured in cell media composed of McCoys media (Sigma no. M8403), 10% (v/v) fetal calf serum, and 1% (v/v) <span class="smallcaps smallerCapital">l</span>-glutamine. After harvesting, cells were dispensed into black, 384-well Costar plates (no. 3712, Corning) to give 2000 cells per well in a total volume of 40 μL of cell media. Test compounds and reference controls were dosed directly into the cell plates using a Labcyte Echo 555 acoustic dispenser following a 12 point half-log compound concentration–response with a top concentration of 30 μM.</div><div class="NLM_p">The cell plates were then incubated for 48 h at 37 °C before being fixed by the addition of 40 μL of 8% paraformaldehyde in PBS/A (4% final concentration), followed by a 10 min rt incubation. Plates were then washed twice with 150 μL of PBS using a BioTek ELx406 platewasher, permeabilized for 10 min with 20 μL/well 0.1% saponin in PBS, washed again, and blocked with 20 μL/well 2% BSA in PBS-T (Sigma no. A8022) for 1 h at rt.</div><div class="NLM_p">Primary anti-SDMA histone 4 antibody (Milipore no. 07-947) was diluted 1:1000 in PBS-T + 0.05% BSA, 20 μL added per well, and plates were incubated at 4 °C overnight. Cell plates were washed 3× with 200 μL of PBS/T, and then 20 μL of 1:500 dilution in assay buffer of Alexa Fluor 488 goat anti-rabbit IgG secondary antibody (Thermo no. A11008,), with a 1:1000 dilution of Hoechst 33342, was added per well. Following a 1 h incubation at rt, plates were washed 3× with 200 μL of PBS/T, and 40 μL of PBS without Ca, Mg, and Na Bicarb (Gibco no. 14190-094) was added per well.</div><div class="NLM_p last">Stained cell plates were covered with black seals and then read on the Cell Insight imaging platform (Thermo Scientific), with a 10× objective. The primary channel (Hoechst blue fluorescence 405 nm, BGRFR_386_23) is used to autofocus and to count the number of events (this will provide information about cytotoxicity of the compounds tested). The secondary channel (Green 488 nM, BGRFR_485_20) measures SDMA staining. Data were analyzed, and IC<sub>50</sub> values were calculated using Genedata Screener software. After normalization to maximum signal/top (vehicle control) and minimum signal/bottom (30 μM of PRMT5 inhibitor GSK-3326595,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(22)</a> CAS no. 1616392-22-3), data were fitted to a four-parameter (Hill) IC<sub>50</sub> model.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Proliferation Assay</h3><div class="NLM_p">HCT116 KO MTAP cells were cultured in cell media composed of McCoys media (Sigma no. M8403), 10% (v/v) fetal calf serum, and 1% (v/v) <span class="smallcaps smallerCapital">l</span>-glutamine. After harvesting, cells were dispensed into black, 384-well Costar plates (no. 3712, Corning) to give 400 cells per well in a total volume of 40 μL of cell media. Test compounds and reference controls were dosed directly into the cell plates using a Labcyte Echo 555 acoustic dispenser following a 12 point duplicate half-log compound concentration–response with a top concentration of 30 μM.</div><div class="NLM_p">A day 0 plate is created in parallel but not dosed, after plating 4 μL of Alamar Blue (Thermo no. DAL1100) and incubating for 3 h at 37 °C, 5% CO<sub>2</sub>. Following incubation, plates are read using an EnVision plate reader with fluorescence excitation wavelength of 540–570 nm (peak excitation is 570 nm), fluorescence emission at 580–610 nm (peak emission is 585 nm).</div><div class="NLM_p last">The dosed cell plates were then incubated for 5 d at 37 °C before adding Alamar blue and being read in the Envision following the same protocol as for first plate. Data were analyzed, and IG<sub>50</sub> values were calculated using Genedata Screener software by normalizing to plate 0.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00067" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28715" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28715" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00067?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00067</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary figures and table; extended methods (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf">jm1c00067_si_001.pdf (1.02 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_002.csv">jm1c00067_si_002.csv (1.22 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Crystal structures of MAT2a small molecule complexes have been deposited in the PDB under the following accession codes: <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhr">7bhr</a>), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs">7bhs</a>), <b>5</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bht">7bht</a>), <b>26</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhu">7bhu</a>), <b>28</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhv">7bhv</a>), <b>29</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhw">7bhw</a>), <b>31</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhx">7bhx</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.1c00067" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72023" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72023" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claudia De Fusco</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8807-4976" title="Orcid link">http://orcid.org/0000-0001-8807-4976</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#76151a1703121f17581213100305151936111b171f1a5815191b"><span class="__cf_email__" data-cfemail="17747b7662737e76397372716264747857707a767e7b3974787a">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marianne Schimpl</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2284-5250" title="Orcid link">http://orcid.org/0000-0003-2284-5250</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#86ebe7f4efe7e8e8e3a8f5e5eeefebf6eac6e7f5f2f4e7fce3e8e3e5e7a8e5e9eb"><span class="__cf_email__" data-cfemail="e48985968d858a8a81ca97878c8d899488a485979096859e818a818785ca878b89">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulf Börjesson</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Gothenburg SE-431 83, Sweden</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tony Cheung</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Boston, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iain Collie</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Evans</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Priyanka Narasimhan</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher Stubbs</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mercedes Vazquez-Chantada</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Wagner</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Boston, Massachusetts 02451, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5805-930X" title="Orcid link">http://orcid.org/0000-0002-5805-930X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Grondine</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Boston, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew G. Sanders</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sharon Tentarelli</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Boston, Massachusetts 02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth Underwood</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Argyrides Argyrou</span> - <span class="hlFld-Affiliation affiliation">Discovery
Sciences, R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3141-9122" title="Orcid link">http://orcid.org/0000-0003-3141-9122</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James M. Smith</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6750-265X" title="Orcid link">http://orcid.org/0000-0002-6750-265X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James T. Lynch</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabetta Chiarparin</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2998-1346" title="Orcid link">http://orcid.org/0000-0002-2998-1346</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Graeme Robb</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4531-4375" title="Orcid link">http://orcid.org/0000-0002-4531-4375</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sharan K. Bagal</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James S. Scott</span> - <span class="hlFld-Affiliation affiliation">Oncology
R&D, AstraZeneca, Cambridge CB4 0WG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2263-7024" title="Orcid link">http://orcid.org/0000-0002-2263-7024</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): At the time of writing, all authors were employees of, and owned shares in, AstraZeneca PLC.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors gratefully acknowledge Nin Guan’s valued contribution to preparing samples for mass spectrometry analysis; Crystal Brown’s significant effort in carrying out compound tolerability and efficacy studies; Jason Breed’s help with crystallographic data collection; Eva Lenz and Paul Davey for compound purity checks; and Ian Storer’s contributions regarding the manuscript.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05746" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05746" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">A new reference 11 was added May 5, 2021.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">clearance</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DMPK</td><td class="NLM_def"><p class="first last">drug metabolism and pharmacokinetics</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">efflux ratio</p></td></tr><tr><td class="NLM_term">hep</td><td class="NLM_def"><p class="first last">hepatocyte</p></td></tr><tr><td class="NLM_term">KHMDS</td><td class="NLM_def"><p class="first last">potassium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">ligand lipophilic efficiency</p></td></tr><tr><td class="NLM_term">MAT</td><td class="NLM_def"><p class="first last">methionine adenosyltransferase</p></td></tr><tr><td class="NLM_term">MDCK</td><td class="NLM_def"><p class="first last">Madin–Darby canine kidney</p></td></tr><tr><td class="NLM_term">MTA</td><td class="NLM_def"><p class="first last">methylthioadenosine</p></td></tr><tr><td class="NLM_term"><i>P</i><sub>app</sub></td><td class="NLM_def"><p class="first last">apparent permeability</p></td></tr><tr><td class="NLM_term">po</td><td class="NLM_def"><p class="first last"><i>per os</i> (oral dosing)</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">benzotriazol-1-yl-oxytripyrrolidinophosphonium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">sc</td><td class="NLM_def"><p class="first last">subcutaneous</p></td></tr><tr><td class="NLM_term">SDMA</td><td class="NLM_def"><p class="first last">symmetric dimethyl arginine</p></td></tr><tr><td class="NLM_term">SFC</td><td class="NLM_def"><p class="first last">supercritical fluid chromatography</p></td></tr><tr><td class="NLM_term">S<sub>N</sub>Ar</td><td class="NLM_def"><p class="first last">aromatic nucleophilic substitution</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>1/2</sub></td><td class="NLM_def"><p class="first last">half-life</p></td></tr><tr><td class="NLM_term"><i>V</i><sub>ss</sub></td><td class="NLM_def"><p class="first last">volume of distribution at steady state</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24575" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24575" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 22 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mato, J. M.</span></span> <span> </span><span class="NLM_article-title">S-adenosylmethionine in liver health, injury, and cancer</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1515</span>– <span class="NLM_lpage">1542</span>, <span class="refDoi"> DOI: 10.1152/physrev.00047.2011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1152%2Fphysrev.00047.2011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=23073625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1515-1542&issue=4&author=S.+C.+Luauthor=J.+M.+Mato&title=S-adenosylmethionine+in+liver+health%2C+injury%2C+and+cancer&doi=10.1152%2Fphysrev.00047.2011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">S-adenosylmethionine in liver health, injury, and cancer</span></div><div class="casAuthors">Lu, Shelly C.; Mato, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1515-1542</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  S-adenosylmethionine (AdoMet, also known as SAM and SAMe) is the principal biol. Me donor synthesized in all mammalian cells but most abundantly in the liver.  Biosynthesis of AdoMet requires the enzyme methionine adenosyltransferase (MAT).  In mammals, two genes, MAT1A that is largely expressed by normal liver and MAT2A that is expressed by all extrahepatic tissues, encode MAT.  Patients with chronic liver disease have reduced MAT activity and AdoMet levels.  Mice lacking Mat1a have reduced hepatic AdoMet levels and develop oxidative stress, steatohepatitis, and hepatocellular carcinoma (HCC).  In these mice, several signaling pathways are abnormal that can contribute to HCC formation.  However, injury and HCC also occur if hepatic AdoMet level is excessive chronically.  This can result from inactive mutation of the enzyme glycine N-methyltransferase (GNMT).  Children with GNMT mutation have elevated liver transaminases, and Gnmt knockout mice develop liver injury, fibrosis, and HCC.  Thus a normal hepatic AdoMet level is necessary to maintain liver health and prevent injury and HCC.  AdoMet is effective in cholestasis of pregnancy, and its role in other human liver diseases remains to be better defined.  In exptl. models, it is effective as a chemopreventive agent in HCC and perhaps other forms of cancer as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriGQKr7t0oIrVg90H21EOLACvtfcHk0liGjGYE2OBtew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnP&md5=6479e1e0b3d84bf946747ad7e580ea22</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00047.2011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00047.2011%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%2BC.%26aulast%3DMato%26aufirst%3DJ.%2BM.%26atitle%3DS-adenosylmethionine%2520in%2520liver%2520health%252C%2520injury%252C%2520and%2520cancer%26jtitle%3DPhysiol.%2520Rev.%26date%3D2012%26volume%3D92%26issue%3D4%26spage%3D1515%26epage%3D1542%26doi%3D10.1152%2Fphysrev.00047.2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mato, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span> <span> </span><span class="NLM_article-title">Role of methionine adenosyltransferase genes in hepatocarcinogenesis</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">1480</span>– <span class="NLM_lpage">1497</span>, <span class="refDoi"> DOI: 10.3390/cancers3021480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.3390%2Fcancers3021480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=24212770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvV2nsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1480-1497&issue=2&author=K.+Ramaniauthor=J.+M.+Matoauthor=S.+C.+Lu&title=Role+of+methionine+adenosyltransferase+genes+in+hepatocarcinogenesis&doi=10.3390%2Fcancers3021480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Role of methionine adenosyltransferase genes in hepatocarcinogenesis</span></div><div class="casAuthors">Ramani, Komal; Mato, Jose M.; Lu, Shelly C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1480-1497</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Hepatocellular carcinoma (HCC) is the most common primary malignant tumor of the liver.  Detection of HCC can be difficult, as most of the patients who develop this tumor have no symptoms other than those related to their longstanding liver disease.  There is an urgent need to understand the mol. mechanisms that are responsible for the development of this disease so that appropriate therapies can be designed.  Methionine adenosyltransferase (MAT) is an essential enzyme required for the biosynthesis of S-adenosylmethionine (AdoMet), an important Me donor in the cell.  Alterations in the expression of MAT genes and a decline in AdoMet biosynthesis are known to be assocd. with liver injury, cirrhosis and HCC.  This review focuses on the role of MAT genes in HCC development and the scope for therapeutic strategies using these genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSoGTREhz3e7Vg90H21EOLACvtfcHk0lg99yUg9wwtmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvV2nsrs%253D&md5=d37cac20dfc5e1b5d560819306a9919a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3390%2Fcancers3021480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3021480%26sid%3Dliteratum%253Aachs%26aulast%3DRamani%26aufirst%3DK.%26aulast%3DMato%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DS.%2BC.%26atitle%3DRole%2520of%2520methionine%2520adenosyltransferase%2520genes%2520in%2520hepatocarcinogenesis%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26issue%3D2%26spage%3D1480%26epage%3D1497%26doi%3D10.3390%2Fcancers3021480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollé, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: A Compendium of Cancer Dependencies and Synthetic Lethal Relationships Uncovered by Large-Scale, Deep RNAi Screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&issue=3&author=E.+R.+McDonaldauthor=A.+de+Weckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cort%C3%A9s-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Moll%C3%A9author=R.+A.+Pagliariniauthor=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+A+Compendium+of+Cancer+Dependencies+and+Synthetic+Lethal+Relationships+Uncovered+by+Large-Scale%2C+Deep+RNAi+Screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0lg99yUg9wwtmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCort%25C3%25A9s-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMoll%25C3%25A9%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520A%2520Compendium%2520of%2520Cancer%2520Dependencies%2520and%2520Synthetic%2520Lethal%2520Relationships%2520Uncovered%2520by%2520Large-Scale%252C%2520Deep%2520RNAi%2520Screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26issue%3D3%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">6278</span>),  <span class="NLM_fpage">1214</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1126/science.aad5214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1126%2Fscience.aad5214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=26912360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1214-8&issue=6278&author=G.+V.+Kryukovauthor=F.+H.+Wilsonauthor=J.+R.+Ruthauthor=J.+Paulkauthor=A.+Tsherniakauthor=S.+E.+Marlowauthor=F.+Vazquezauthor=B.+A.+Weirauthor=M.+E.+Fitzgeraldauthor=M.+Tanakaauthor=C.+M.+Bielskiauthor=J.+M.+Scottauthor=C.+Dennisauthor=G.+S.+Cowleyauthor=J.+S.+Boehmauthor=D.+E.+Rootauthor=T.+R.+Golubauthor=C.+B.+Clishauthor=J.+E.+Bradnerauthor=W.+C.+Hahnauthor=L.+A.+Garraway&title=MTAP+deletion+confers+enhanced+dependency+on+the+PRMT5+arginine+methyltransferase+in+cancer+cells&doi=10.1126%2Fscience.aad5214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span></div><div class="casAuthors">Kryukov, Gregory V.; Wilson, Frederick H.; Ruth, Jason R.; Paulk, Joshiawa; Tsherniak, Aviad; Marlow, Sara E.; Vazquez, Francisca; Weir, Barbara A.; Fitzgerald, Mark E.; Tanaka, Minoru; Bielski, Craig M.; Scott, Justin M.; Dennis, Courtney; Cowley, Glenn S.; Boehm, Jesse S.; Root, David E.; Golub, Todd R.; Clish, Clary B.; Bradner, James E.; Hahn, William C.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1214-1218</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The discovery of cancer dependencies has the potential to inform therapeutic strategies and to identify putative drug targets.  Integrating data from comprehensive genomic profiling of cancer cell lines and from functional characterization of cancer cell dependencies, we discovered that loss of the enzyme methylthioadenosine phosphorylase (MTAP) confers a selective dependence on protein arginine methyltransferase 5 (PRMT5) and its binding partner WDR77.  MTAP is frequently lost due to its proximity to the commonly deleted tumor suppressor gene, CDKN2A.  We obsd. increased intracellular concns. of methylthioadenosine (MTA, the metabolite cleaved by MTAP) in cells harboring MTAP deletions.  Furthermore, MTA specifically inhibited PRMT5 enzymic activity.  Administration of either MTA or a small-mol. PRMT5 inhibitor showed a modest preferential impairment of cell viability for MTAP-null cancer cell lines compared with isogenic MTAP-expressing counterparts.  Together, our findings reveal PRMT5 as a potential vulnerability across multiple cancer lineages augmented by a common "passenger" genomic alteration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94dQPnMgrqbVg90H21EOLACvtfcHk0lgNzV_3RUqvEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D&md5=2b017a8c54972828c9024a9d85ff2e2d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5214%26sid%3Dliteratum%253Aachs%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DWilson%26aufirst%3DF.%2BH.%26aulast%3DRuth%26aufirst%3DJ.%2BR.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DMarlow%26aufirst%3DS.%2BE.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DBielski%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DJ.%2BM.%26aulast%3DDennis%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DMTAP%2520deletion%2520confers%2520enhanced%2520dependency%2520on%2520the%2520PRMT5%2520arginine%2520methyltransferase%2520in%2520cancer%2520cells%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2016%26volume%3D351%26issue%3D6278%26spage%3D1214%26epage%3D8%26doi%3D10.1126%2Fscience.aad5214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marjon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McVay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kernytsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span> <span> </span><span class="NLM_article-title">MTAP Deletions in Cancer Create Vulnerability to Targeting of the MAT2A/PRMT5/RIOK1 Axis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.03.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2Fj.celrep.2016.03.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=27068473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=574-587&issue=3&author=K.+Marjonauthor=M.+J.+Cameronauthor=P.+Quangauthor=M.+F.+Clasquinauthor=E.+Mandleyauthor=K.+Kuniiauthor=M.+McVayauthor=S.+Choeauthor=A.+Kernytskyauthor=S.+Grossauthor=Z.+Konteatisauthor=J.+Murtieauthor=M.+L.+Blakeauthor=J.+Travinsauthor=M.+Dorschauthor=S.+A.+Billerauthor=K.+M.+Marks&title=MTAP+Deletions+in+Cancer+Create+Vulnerability+to+Targeting+of+the+MAT2A%2FPRMT5%2FRIOK1+Axis&doi=10.1016%2Fj.celrep.2016.03.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP Deletions in Cancer Create Vulnerability to Targeting of the MAT2A/PRMT5/RIOK1 Axis</span></div><div class="casAuthors">Marjon, Katya; Cameron, Michael J.; Quang, Phong; Clasquin, Michelle F.; Mandley, Everton; Kunii, Kaiko; McVay, Michael; Choe, Sung; Kernytsky, Andrew; Gross, Stefan; Konteatis, Zenon; Murtie, Joshua; Blake, Michelle L.; Travins, Jeremy; Dorsch, Marion; Biller, Scott A.; Marks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-587</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Homozygous deletions of p16/CDKN2A are prevalent in cancer, and these mutations commonly involve co-deletion of adjacent genes, including methylthioadenosine phosphorylase (MTAP).  Here, we used shRNA screening and identified the metabolic enzyme, methionine adenosyltransferase II alpha (MAT2A), and the arginine methyltransferase, PRMT5, as vulnerable enzymes in cells with MTAP deletion.  Metabolomic and biochem. studies revealed a mechanistic basis for this synthetic lethality.  The MTAP substrate methylthioadenosine (MTA) accumulates upon MTAP loss.  Biochem. profiling of a methyltransferase enzyme panel revealed that MTA is a potent and selective inhibitor of PRMT5.  MTAP-deleted cells have reduced PRMT5 methylation activity and increased sensitivity to PRMT5 depletion.  MAT2A produces the PRMT5 substrate S-adenosylmethionine (SAM), and MAT2A depletion reduces growth and PRMT5 methylation activity selectively in MTAP-deleted cells.  Furthermore, this vulnerability extends to PRMT5 co-complex proteins such as RIOK1.  Thus, the unique biochem. features of PRMT5 create an axis of targets vulnerable in CDKN2A/MTAP-deleted cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokW739dwptcbVg90H21EOLACvtfcHk0lgNzV_3RUqvEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D&md5=73a6839e63b85a71f3661dc9e0c7b4ea</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.03.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.03.043%26sid%3Dliteratum%253Aachs%26aulast%3DMarjon%26aufirst%3DK.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DQuang%26aufirst%3DP.%26aulast%3DClasquin%26aufirst%3DM.%2BF.%26aulast%3DMandley%26aufirst%3DE.%26aulast%3DKunii%26aufirst%3DK.%26aulast%3DMcVay%26aufirst%3DM.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DKernytsky%26aufirst%3DA.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DKonteatis%26aufirst%3DZ.%26aulast%3DMurtie%26aufirst%3DJ.%26aulast%3DBlake%26aufirst%3DM.%2BL.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DDorsch%26aufirst%3DM.%26aulast%3DBiller%26aufirst%3DS.%2BA.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26atitle%3DMTAP%2520Deletions%2520in%2520Cancer%2520Create%2520Vulnerability%2520to%2520Targeting%2520of%2520the%2520MAT2A%252FPRMT5%252FRIOK1%2520Axis%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D15%26issue%3D3%26spage%3D574%26epage%3D587%26doi%3D10.1016%2Fj.celrep.2016.03.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, T. M.</span></span> <span> </span><span class="NLM_article-title">Codeletion of the genes for p16INK4, methylthioadenosine phosphorylase, interferon-α1, interferon-β1, and other 9p21 markers in human malignant cell lines</span>. <i>Cancer Genet. Cytogenet.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>86</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/0165-4608(95)00157-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2F0165-4608%2895%2900157-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=8616780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADyaK28XitVylsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1996&pages=22-28&issue=1&author=H.+Zhangauthor=Z.-H.+Chenauthor=T.+M.+Savarese&title=Codeletion+of+the+genes+for+p16INK4%2C+methylthioadenosine+phosphorylase%2C+interferon-%CE%B11%2C+interferon-%CE%B21%2C+and+other+9p21+markers+in+human+malignant+cell+lines&doi=10.1016%2F0165-4608%2895%2900157-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Codeletion of the genes for p16INK4, methylthioadenosine phosphorylase, interferon-α1, interferon-β1, and other 9p21 markers in human malignant cell lines</span></div><div class="casAuthors">Zhang, Hongyang; Chen, Zhi-Hao; Savarese, Todd M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Genetics and Cytogenetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-8</span>CODEN:
                <span class="NLM_cas:coden">CGCYDF</span>;
        ISSN:<span class="NLM_cas:issn">0165-4608</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">In this study, 27 malignant cell lines, including leukemias, gliomas, and lung and bladder carcinomas were screened for homozygous deletions of the putative tumor suppressor gene p16 (MTS1/CDK4I/CDKN2) and other markers within the chromosome 9p21 region; these include the genes for interferon-α1 (IFNA1), interferon-β1 (IFNB1), methylthioadenosine phosphorylase (MTAP), and two microsatellite markers, D9S171 and D9S169.  The purpose of this study was to det. the incidence of co-deletion of these markers.  Screening for homozygous deletions was carried out using direct polymerase chain reaction of genomic DNA, or, in the case of MTAP, a functional enzyme assay.  Of these cell lines, 14 (52%) were found to have homozygous deletions of the p16 gene.  Two of the 14 p16-neg. cell lines (14%) were found to have homozygous deletions within the p16 domain but no other 9p21 marker.  MTAP was co-deleted in 12 of the 14 p16-neg. cell lines (86%), whereas IFNA1 was co-deleted with p16 in eight of these lines (57%); IFNB1 was co-deleted in five (36%) of the p16-deleted cell lines.  The D9S171 marker, which may lie greater than 3 cM centromeric to p16, is co-deleted in three cell lines (21%); the D9S169 marker, which maps even further toward the centromere, was co-deleted in only one cell line (7%).  Loss of any 9p21 marker, e.g., MTAP or IFNA1, were invariable predictors of the loss of the p16 gene.  In addn., loss of IFNA1 always predicted a loss of MTAP (eight of eight cell lines), although loss of MTAP did not always predict a loss of IFNA1 (four of 12 MTAP-deleted cell lines did not have homozygous deletions of IFNA1).  Thus loss of nearby genes occurs in a high percentage of cell lines that bear homozygous deletions of the p16 locus.  Co-deletion of MTAP or IFN in p16-neg. malignant cells is of interest, as loss of these genes may influence the biol. behavior of these cells and render them susceptible to certain therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOfMShSxVr5rVg90H21EOLACvtfcHk0lgUGVpQHmcU4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XitVylsrk%253D&md5=7da196b73cc878db99a6c88e8eedd603</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0165-4608%2895%2900157-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0165-4608%252895%252900157-3%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.-H.%26aulast%3DSavarese%26aufirst%3DT.%2BM.%26atitle%3DCodeletion%2520of%2520the%2520genes%2520for%2520p16INK4%252C%2520methylthioadenosine%2520phosphorylase%252C%2520interferon-%25CE%25B11%252C%2520interferon-%25CE%25B21%252C%2520and%2520other%25209p21%2520markers%2520in%2520human%2520malignant%2520cell%2520lines%26jtitle%3DCancer%2520Genet.%2520Cytogenet.%26date%3D1996%26volume%3D86%26issue%3D1%26spage%3D22%26epage%3D28%26doi%3D10.1016%2F0165-4608%2895%2900157-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albers, E.</span></span> <span> </span><span class="NLM_article-title">Metabolic characteristics and importance of the universal methionine salvage pathway recycling methionine from 5′-methylthioadenosine</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1002/iub.278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1002%2Fiub.278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=19946895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFShtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2009&pages=1132-1142&issue=12&author=E.+Albers&title=Metabolic+characteristics+and+importance+of+the+universal+methionine+salvage+pathway+recycling+methionine+from+5%E2%80%B2-methylthioadenosine&doi=10.1002%2Fiub.278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic characteristics and importance of the universal methionine salvage pathway recycling methionine from 5'-methylthioadenosine</span></div><div class="casAuthors">Albers, Eva</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1132-1142</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  The methionine salvage pathway, also called the 5'-methylthioadenosine (MTA) cycle, recycles the S atom of MTA, which is a byproduct in the biosyntheses of polyamine and the plant hormone, ethylene.  MTA is 1st converted to 5'-methylthioribose 1-phosphate either by MTA phosphorylase or the combined action of MTA nucleosidase and 5'-methylthioribose kinase.  Subsequently, 5 addnl. enzymic steps, catalyzed by 4 or 5 proteins, will form 4-methylthio-2-oxobutyrate, the deaminated form of methionine.  The final transamination is achieved by transaminases active in the amino acid biosynthesis.  This pathway is present with some variations in all types of organisms and seems to be designed for a quick removal of MTA achieved by high affinities of the 1st enzymes.  During evolution, some enzymes attained addnl. functions, like a proposed role in nuclear mRNA processing by the aci-reductone dioxygenase.  For others, the function seems to be lost due to conditions in specific ecol. niches, such as, presence of S and/or absence of O resulting in that, e.g., Escherichia coli is lacking a functional pathway.  The pathway is regulated as response to S availability and take part in the regulation of polyamine synthesis.  Some of the enzymes in the pathway show sep. specificities in different organisms and some others are unique for groups of bacteria and parasites.  Thus, promising targets for antimicrobial agents have been identified.  Other medical topics to which this pathway has connections are cancer, apoptosis, and the inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHogIDh-oIBLVg90H21EOLACvtfcHk0lgUGVpQHmcU4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFShtLnO&md5=3f7d52b51c55763984f86e6151dad45b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fiub.278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.278%26sid%3Dliteratum%253Aachs%26aulast%3DAlbers%26aufirst%3DE.%26atitle%3DMetabolic%2520characteristics%2520and%2520importance%2520of%2520the%2520universal%2520methionine%2520salvage%2520pathway%2520recycling%2520methionine%2520from%25205%25E2%2580%25B2-methylthioadenosine%26jtitle%3DIUBMB%2520Life%26date%3D2009%26volume%3D61%26issue%3D12%26spage%3D1132%26epage%3D1142%26doi%3D10.1002%2Fiub.278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardini, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulter, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Analogues of Methionine as Substrates and Inhibitors of the Methionine Adenosyltransferase Reaction</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">499</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=4918253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADyaE3cXlt1OhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1970&pages=481-499&issue=5&author=J.+B.+Lombardiniauthor=A.+W.+Coulterauthor=P.+Talalay&title=Analogues+of+Methionine+as+Substrates+and+Inhibitors+of+the+Methionine+Adenosyltransferase+Reaction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Analogs of methionine as substrates and inhibitors of the methionine adenosyltransferase reaction.  Deductions concerning the conformation of methionine</span></div><div class="casAuthors">Lombardini, J. B.; Coulter, A. W.; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">481-99</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">Steric, electronic, and conformational requirements are described for analogs of L-methionine essential to their function as substrates or inhibitors of the methionine adenosyltransferase reaction (EC 2.4.1.13).  With the aid of partially purified transferase prepns. from Escherichia coli, bakers' yeast, and rat liver, a systematic study of substrate analogs has been undertaken.  Inhibitors of the enzyme fall into 3 categories:  (a) straight C chain amino acids, such as L-2-amino-4-hexenoic acid (trans but not the cis isomer) and L-2-amino-4-hexynoic acid, which are the most potent inhibitors; (b) cyclic amino acids, among which 1-aminocyclopentanecarboxylic acid and 1 of the 4 isomers of 1-amino-3-methylcyclopentanecarboxylic acid (either the 1R, 3R or the 1S, 3R isomer) are the most powerful; and (c) O-acetyl-L-serine, O-carbamoyl-L-serine, and S-carbamoyl-L-cysteine.  Since inhibitors belonging to groups a and b possess considerable conformational rigidity by virtue of the presence of unsatns. or cyclic structures, it has been possible to draw conclusions with respect to the conformation of L-methionine at the active site of the adenosyltransferase reaction.  A no. of the inhibitors of the methionine adenosyltransferase reaction, such as 1-aminocyclopentanecarboxylic acid and S-carbamoyl-L-cysteine, are known to be inhibitors of the growth of certain microorganisms and tumors.  The possibility is suggested that these inhibitory activities may be mediated at least in part through the inhibition of the synthesis of S-adenosyl-L-methionine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQxsk5Ck1f1rVg90H21EOLACvtfcHk0lgUGVpQHmcU4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXlt1OhsL8%253D&md5=707357bc113134ff3daccf1d49ecaf11</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLombardini%26aufirst%3DJ.%2BB.%26aulast%3DCoulter%26aufirst%3DA.%2BW.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DAnalogues%2520of%2520Methionine%2520as%2520Substrates%2520and%2520Inhibitors%2520of%2520the%2520Methionine%2520Adenosyltransferase%2520Reaction%26jtitle%3DMol.%2520Pharmacol.%26date%3D1970%26volume%3D6%26issue%3D5%26spage%3D481%26epage%3D499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardini, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sufrin, J. R.</span></span> <span> </span><span class="NLM_article-title">Chemotherapeutic potential of methionine analogue inhibitors of tumor-derived methionine adenosyltransferases</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(83)90528-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2F0006-2952%2883%2990528-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=6847699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADyaL3sXhtlOgs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1983&pages=489-495&issue=3&author=J.+B.+Lombardiniauthor=J.+R.+Sufrin&title=Chemotherapeutic+potential+of+methionine+analogue+inhibitors+of+tumor-derived+methionine+adenosyltransferases&doi=10.1016%2F0006-2952%2883%2990528-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Chemotherapeutic potential of methionine analog inhibitors of tumor-derived methionine adenosyltransferases</span></div><div class="casAuthors">Lombardini, John B.; Sufrin, Janice R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">489-95</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">Two isozymes of ATP:L-methionine S-adenosyltransferase (MAT)  [9012-52-6] were fractionated from rat Novikoff solid hepatoma.  Their Km values for L-methionine  [63-68-3] and/or their inhibition consts. for various L-methionine analogs were significantly different from the kinetic consts. obtained for three isozymes fractionated from normal rat liver.  Ki Values for cycloleucine  [52-52-8] and (±)-2-aminobicyclo[2.1.1]hexane-2-carboxylic acid  [71046-30-5], presented for each tumor and liver isozyme, indicate that (±)-2-aminobicyclo[2.1.1]hexane-2-carboxylic acid was the more potent inhibitor.  Dixon plots were also used to test a series of amino acid analogs [cycloleucine, 1-aminocyclobutanecarboxylic acid  [22264-50-2], 1-aminocyclohexanecarboxylic acid  [2756-85-6], (±)-2-aminobicyclo[2.1.1]hexane-2-carboxylic acid, L-2-amino-4-hexynoic acid  [29834-76-2], (Z)-L-2-amino-5-chloro-4-hexenoic acid  [51018-17-8], L-ethionine  [13073-35-3], S-n-propyl-DL-homocysteine  [2598-46-1], S-n-butyl-DL-homocysteine  [4378-14-7], and seleno-DL-ethionine  [2578-27-0]] of methionine for inhibitory potency.  Fixed L-methionine concns. were used to det. the concn. of inhibitor necessary to inhibit the MAT reaction by 50%.  Differential inhibitory activities of the amino acid analogs were noted between the tumor and rat liver isozymes, thus supporting the suggestion that tumor-derived MAT isozymes may provide an exploitable target for cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGporWK3_U66Q7Vg90H21EOLACvtfcHk0ljpvtlqQxidGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhtlOgs7o%253D&md5=cbb7c2624fc4e88c98f57a0a2b8ba59e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2883%2990528-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252883%252990528-2%26sid%3Dliteratum%253Aachs%26aulast%3DLombardini%26aufirst%3DJ.%2BB.%26aulast%3DSufrin%26aufirst%3DJ.%2BR.%26atitle%3DChemotherapeutic%2520potential%2520of%2520methionine%2520analogue%2520inhibitors%2520of%2520tumor-derived%2520methionine%2520adenosyltransferases%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1983%26volume%3D32%26issue%3D3%26spage%3D489%26epage%3D495%26doi%3D10.1016%2F0006-2952%2883%2990528-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinlan, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolaños, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowlin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grantner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlicek-Bryant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman-Cook, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dann, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S. K.</span></span> <span> </span><span class="NLM_article-title">Targeting S-adenosylmethionine biosynthesis with a novel allosteric inhibitor of Mat2A</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1038%2Fnchembio.2384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=28553945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=785-792&issue=7&author=C.+L.+Quinlanauthor=S.+E.+Kaiserauthor=B.+Bola%C3%B1osauthor=D.+Nowlinauthor=R.+Grantnerauthor=S.+Karlicek-Bryantauthor=J.+L.+Fengauthor=S.+Jenkinsonauthor=K.+Freeman-Cookauthor=S.+G.+Dannauthor=X.+Wangauthor=P.+A.+Wellsauthor=V.+R.+Fantinauthor=A.+E.+Stewartauthor=S.+K.+Grant&title=Targeting+S-adenosylmethionine+biosynthesis+with+a+novel+allosteric+inhibitor+of+Mat2A&doi=10.1038%2Fnchembio.2384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting S-adenosylmethionine biosynthesis with a novel allosteric inhibitor of Mat2A</span></div><div class="casAuthors">Quinlan, Casey L.; Kaiser, Stephen E.; Bolanos, Ben; Nowlin, Dawn; Grantner, Rita; Karlicek-Bryant, Shannon; Feng, Jun Li; Jenkinson, Stephen; Freeman-Cook, Kevin; Dann, Stephen G.; Wang, Xiaoli; Wells, Peter A.; Fantin, Valeria R.; Stewart, Al E.; Grant, Stephan K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">785-792</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">S-Adenosyl-L-methionine (SAM) is an enzyme cofactor used in Me transfer reactions and polyamine biosynthesis.  The biosynthesis of SAM from ATP and L-methionine is performed by the methionine adenosyltransferase enzyme family (Mat; EC 2.5.1.6).  Human methionine adenosyltransferase 2A (Mat2A), the extrahepatic isoform, is often deregulated in cancer.  We identified a Mat2A inhibitor, PF-9366, that binds an allosteric site on Mat2A that overlaps with the binding site for the Mat2A regulator, Mat2B.  Studies exploiting PF-9366 suggested a general mode of Mat2A allosteric regulation.  Allosteric binding of PF-9366 or Mat2B altered the Mat2A active site, resulting in increased substrate affinity and decreased enzyme turnover.  These data support a model whereby Mat2B functions as an inhibitor of Mat2A activity when methionine or SAM levels are high, yet functions as an activator of Mat2A when methionine or SAM levels are low.  The ramification of Mat2A activity modulation in cancer cells is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpniywgvxjL1bVg90H21EOLACvtfcHk0ljpvtlqQxidGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosFSruro%253D&md5=26be67f7977cbcccd5f967844924dc24</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2384%26sid%3Dliteratum%253Aachs%26aulast%3DQuinlan%26aufirst%3DC.%2BL.%26aulast%3DKaiser%26aufirst%3DS.%2BE.%26aulast%3DBola%25C3%25B1os%26aufirst%3DB.%26aulast%3DNowlin%26aufirst%3DD.%26aulast%3DGrantner%26aufirst%3DR.%26aulast%3DKarlicek-Bryant%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DJ.%2BL.%26aulast%3DJenkinson%26aufirst%3DS.%26aulast%3DFreeman-Cook%26aufirst%3DK.%26aulast%3DDann%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWells%26aufirst%3DP.%2BA.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DStewart%26aufirst%3DA.%2BE.%26aulast%3DGrant%26aufirst%3DS.%2BK.%26atitle%3DTargeting%2520S-adenosylmethionine%2520biosynthesis%2520with%2520a%2520novel%2520allosteric%2520inhibitor%2520of%2520Mat2A%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D7%26spage%3D785%26epage%3D792%26doi%3D10.1038%2Fnchembio.2384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonyuk, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Liempd, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mato, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasnain, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, A. L.</span></span> <span> </span><span class="NLM_article-title">Structure and function study of the complex that synthesizes S-adenosylmethionine</span>. <i>IUCrJ</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">249</span>, <span class="refDoi"> DOI: 10.1107/S2052252514012585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1107%2FS2052252514012585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=25075345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtV2jsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=240-249&author=B.+Murrayauthor=S.+V.+Antonyukauthor=A.+Marinaauthor=S.+M.+Van+Liempdauthor=S.+C.+Luauthor=J.+M.+Matoauthor=S.+S.+Hasnainauthor=A.+L.+Rojas&title=Structure+and+function+study+of+the+complex+that+synthesizes+S-adenosylmethionine&doi=10.1107%2FS2052252514012585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function study of the complex that synthesizes S-adenosylmethionine</span></div><div class="casAuthors">Murray, Ben; Antonyuk, Svetlana V.; Marina, Alberto; Van Liempd, Sebastiaan M.; Lu, Shelly C.; Mato, Jose M.; Hasnain, S. Samar; Rojas, Adriana L.</div><div class="citationInfo"><span class="NLM_cas:title">IUCrJ</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">240-249</span>CODEN:
                <span class="NLM_cas:coden">IUCRAJ</span>;
        ISSN:<span class="NLM_cas:issn">2052-2525</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">S-Adenosylmethionine (SAMe) is the principal Me donor of the cell and is synthesized via an ATP-driven process by methionine adenosyltransferase (MAT) enzymes.  It is tightly linked with cell proliferation in liver and colon cancer.  In humans, there are three genes, mat1A, mat2A and mat2B, which encode MAT enzymes. mat2A and mat2B transcribe MATα2 and MATβ enzyme subunits, resp., with catalytic and regulatory roles.  The MATα2β complex is expressed in nearly all tissues and is thought to be essential in providing the necessary SAMe flux for methylation of DNA and various proteins including histones.  In human hepatocellular carcinoma mat2A and mat2B genes are upregulated, highlighting the importance of the MATα2β complex in liver disease.  The individual subunits have been structurally characterized but the nature of the complex has remained elusive despite its existence having been postulated for more than 20 years and the observation that MATβ is often co-localized with MATα2.  Though SAMe can be produced by MAT(α2)4 alone, this paper shows that the V max of the MATα2β complex is three- to fourfold higher depending on the variants of MATβ that participate in complex formation.  Using X-ray crystallog. and soln. X-ray scattering, the first structures are provided of this 258 kDa functional complex both in crystals and soln. with an unexpected stoichiometry of 4α2 and 2βV2 subunits.  It is demonstrated that the N-terminal regulates the activity of the complex and it is shown that complex formation takes place surprisingly via the C-terminal of MATβV2 that buries itself in a tunnel created at the interface of the MAT(α2)2.  The structural data suggest a unique mechanism of regulation and provide a gateway for structure-based drug design in anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6Zyk4WShPyLVg90H21EOLACvtfcHk0ljpvtlqQxidGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtV2jsrjF&md5=b5cf7fe51c8fcf98ecc49bb290bd41e3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1107%2FS2052252514012585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052252514012585%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DAntonyuk%26aufirst%3DS.%2BV.%26aulast%3DMarina%26aufirst%3DA.%26aulast%3DVan%2BLiempd%26aufirst%3DS.%2BM.%26aulast%3DLu%26aufirst%3DS.%2BC.%26aulast%3DMato%26aufirst%3DJ.%2BM.%26aulast%3DHasnain%26aufirst%3DS.%2BS.%26aulast%3DRojas%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520and%2520function%2520study%2520of%2520the%2520complex%2520that%2520synthesizes%2520S-adenosylmethionine%26jtitle%3DIUCrJ%26date%3D2014%26volume%3D1%26spage%3D240%26epage%3D249%26doi%3D10.1107%2FS2052252514012585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z. D.</span>; <span class="NLM_string-name">Sui, Z.</span>; <span class="NLM_string-name">Travins, J. M.</span>; <span class="NLM_string-name">Ye, Z.</span></span> <span> </span><span class="NLM_article-title">Pyrazolo[1,5-a]pyrimidin-7(4H)-ones as inhibitors of cellular metabolic processes and their preparation</span>. Patent <span class="NLM_patent">US20180079753 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Z.+D.+Konteatis&author=Z.+Sui&author=J.+M.+Travins&author=Z.+Ye&title=Pyrazolo%5B1%2C5-a%5Dpyrimidin-7%284H%29-ones+as+inhibitors+of+cellular+metabolic+processes+and+their+preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKonteatis%26aufirst%3DZ.%2BD.%26atitle%3DPyrazolo%255B1%252C5-a%255Dpyrimidin-7%25284H%2529-ones%2520as%2520inhibitors%2520of%2520cellular%2520metabolic%2520processes%2520and%2520their%2520preparation%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z. D.</span>; <span class="NLM_string-name">Li, M.</span>; <span class="NLM_string-name">Reznik, S. K.</span>; <span class="NLM_string-name">Sui, Z.</span>; <span class="NLM_string-name">Travins, J. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of azaheterobicyclic inhibitors of MAT2A and methods of use for treating cancer</span>. Patent <span class="NLM_patent">WO2020139992 A1</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=Z.+D.+Konteatis&author=M.+Li&author=S.+K.+Reznik&author=Z.+Sui&author=J.+M.+Travins&title=Preparation+of+azaheterobicyclic+inhibitors+of+MAT2A+and+methods+of+use+for+treating+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKonteatis%26aufirst%3DZ.%2BD.%26atitle%3DPreparation%2520of%2520azaheterobicyclic%2520inhibitors%2520of%2520MAT2A%2520and%2520methods%2520of%2520use%2520for%2520treating%2520cancer%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z. D.</span>; <span class="NLM_string-name">Li, M.</span>; <span class="NLM_string-name">Liu, P.</span>; <span class="NLM_string-name">Medeiros, M.</span>; <span class="NLM_string-name">Reznik, S. K.</span>; <span class="NLM_string-name">Sui, Z.</span>; <span class="NLM_string-name">Travins, J. M.</span>; <span class="NLM_string-name">Popovici-Muller, J.</span>; <span class="NLM_string-name">Zhou, S.</span>; <span class="NLM_string-name">Ma, G.</span></span> <span> </span><span class="NLM_article-title">Preparation of heterobicyclic compounds as inhibitors of MAT2A and methods of use for treating cancer</span>. Patent <span class="NLM_patent">WO2019191470 A1</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Z.+D.+Konteatis&author=M.+Li&author=P.+Liu&author=M.+Medeiros&author=S.+K.+Reznik&author=Z.+Sui&author=J.+M.+Travins&author=J.+Popovici-Muller&author=S.+Zhou&author=G.+Ma&title=Preparation+of+heterobicyclic+compounds+as+inhibitors+of+MAT2A+and+methods+of+use+for+treating+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKonteatis%26aufirst%3DZ.%2BD.%26atitle%3DPreparation%2520of%2520heterobicyclic%2520compounds%2520as%2520inhibitors%2520of%2520MAT2A%2520and%2520methods%2520of%2520use%2520for%2520treating%2520cancer%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&issue=9&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lgG5RX0BiPfQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26issue%3D9%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacquemard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">A bright future for fragment-based drug discovery: what does it hold?</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1080/17460441.2019.1583643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1080%2F17460441.2019.1583643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=30793989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A280%3ADC%252BB3cfmsVCnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=413-416&issue=5&author=C.+Jacquemardauthor=E.+Kellenberger&title=A+bright+future+for+fragment-based+drug+discovery%3A+what+does+it+hold%3F&doi=10.1080%2F17460441.2019.1583643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A bright future for fragment-based drug discovery: what does it hold?</span></div><div class="casAuthors">Jacquemard Celien; Kellenberger Esther</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on drug discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">413-416</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSHl2exLyY4fhfKb6ELK8t0fW6udTcc2ebUuve8WUBUprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfmsVCnsQ%253D%253D&md5=d5ebc2b03fbf98207bd692d1e55279c1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F17460441.2019.1583643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2019.1583643%26sid%3Dliteratum%253Aachs%26aulast%3DJacquemard%26aufirst%3DC.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DA%2520bright%2520future%2520for%2520fragment-based%2520drug%2520discovery%253A%2520what%2520does%2520it%2520hold%253F%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2019%26volume%3D14%26issue%3D5%26spage%3D413%26epage%3D416%26doi%3D10.1080%2F17460441.2019.1583643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Kloe, G. E.</span></span> <span> </span><span class="NLM_article-title">Fragment library design: efficiently hunting drugs in chemical space</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e173</span>– <span class="NLM_lpage">e180</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2010.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2Fj.ddtec.2010.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVegsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e173-e180&issue=3&author=S.+M.+Boydauthor=G.+E.+de+Kloe&title=Fragment+library+design%3A+efficiently+hunting+drugs+in+chemical+space&doi=10.1016%2Fj.ddtec.2010.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment library design: efficiently hunting drugs in chemical space</span></div><div class="casAuthors">Boyd, Susan M.; de Kloe, Gerdien E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">E173-E180</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  With the growth in fragment-based drug discovery, numerous strategies have been described for the design of fragment libraries.  Key choices need to be made on both the selection criteria to be applied and the source of the fragments in the library.  Here we review some of the key trends and recent developments in the rapidly evolving field of fragment library design, providing an overview of current design strategies and surveying the characteristics of published fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVJ64DfPahqLVg90H21EOLACvtfcHk0li2zKD4Gx0IWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVegsLs%253D&md5=4ec37c1cf8b68c43b5395667313d34fa</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DBoyd%26aufirst%3DS.%2BM.%26aulast%3Dde%2BKloe%26aufirst%3DG.%2BE.%26atitle%3DFragment%2520library%2520design%253A%2520efficiently%2520hunting%2520drugs%2520in%2520chemical%2520space%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2010%26volume%3D7%26issue%3D3%26spage%3De173%26epage%3De180%26doi%3D10.1016%2Fj.ddtec.2010.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheron, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasty, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhnovich, E. I.</span></span> <span> </span><span class="NLM_article-title">OpenGrowth: An Automated and Rational Algorithm for Finding New Protein Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4171</span>– <span class="NLM_lpage">4188</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00886</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00886" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4171-4188&issue=9&author=N.+Cheronauthor=N.+Jastyauthor=E.+I.+Shakhnovich&title=OpenGrowth%3A+An+Automated+and+Rational+Algorithm+for+Finding+New+Protein+Ligands&doi=10.1021%2Facs.jmedchem.5b00886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">OpenGrowth: An Automated and Rational Algorithm for Finding New Protein Ligands</span></div><div class="casAuthors">Cheron, Nicolas; Jasty, Naveen; Shakhnovich, Eugene I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4171-4188</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new open-source software, called OpenGrowth, which aims to create de novo ligands by connecting small org. fragments in the active site of proteins.  Mol. growth is biased to produce structures that statistically resemble drugs in an input training database.  Consequently, the produced mols. have superior synthetic accessibility and pharmacokinetic properties compared with randomly grown mols.  The growth process can take into account the flexibility of the target protein and can be started from a seed to mimic R-group strategy or fragment-based drug discovery.  Primary applications of the software on the HIV-1 protease allowed us to quickly identify new inhibitors with a predicted Kd as low as 18 nM.  We also present a graphical user interface that allows a user to select easily the fragments to include in the growth process.  OpenGrowth is released under the GNU GPL license and is available free of charge on the authors' website and at http://opengrowth.sourceforge.net/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcRSoqoxTyrVg90H21EOLACvtfcHk0li2zKD4Gx0IWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI&md5=6f8895e71ca7749cc1dbe559bda1db7c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00886%26sid%3Dliteratum%253Aachs%26aulast%3DCheron%26aufirst%3DN.%26aulast%3DJasty%26aufirst%3DN.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DOpenGrowth%253A%2520An%2520Automated%2520and%2520Rational%2520Algorithm%2520for%2520Finding%2520New%2520Protein%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4171%26epage%3D4188%26doi%3D10.1021%2Facs.jmedchem.5b00886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&issue=9&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0li2zKD4Gx0IWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26issue%3D9%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&issue=3&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+molecular+recognition+exploiting+hydrophobic+enclosure+in+protein-ligand+binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0ljYtZBVNfhRlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520molecular%2520recognition%2520exploiting%2520hydrophobic%2520enclosure%2520in%2520protein-ligand%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26issue%3D3%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit20"><span>As this
manuscript was in preparation a patent application (<span class="NLM_contrib-group"><span class="NLM_string-name">Alam, M.</span>; <span class="NLM_string-name">Cleary, L.</span>; <span class="NLM_string-name">Fleury, M.</span>; <span class="NLM_string-name">Pei, Z.</span>; <span class="NLM_string-name">Steel, R.</span>; <span class="NLM_string-name">Sutton, J.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Newby, Z. E. R.</span></span> <span> </span><span class="NLM_article-title">2-Oxoquinazoline derivatives as methionine adenosyltransferase 2A inhibitors and their preparation</span>. Patent <span class="NLM_patent">WO2020123395 A1</span>, 2020) from Ideaya was published on the chemical series described in this paper.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=As+this%0Amanuscript+was+in+preparation+a+patent+application+%28Alam%2C+M.%3B+Cleary%2C+L.%3B+Fleury%2C+M.%3B+Pei%2C+Z.%3B+Steel%2C+R.%3B+Sutton%2C+J.%3B+Knox%2C+J.+E.%3B+Newby%2C+Z.+E.+R.+2-Oxoquinazoline+derivatives+as+methionine+adenosyltransferase+2A+inhibitors+and+their+preparation.+Patent+WO2020123395+A1%2C+2020%29+from+Ideaya+was+published+on+the+chemical+series+described+in+this+paper."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlam%26aufirst%3DM.%26atitle%3D2-Oxoquinazoline%2520derivatives%2520as%2520methionine%2520adenosyltransferase%25202A%2520inhibitors%2520and%2520their%2520preparation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span>; <span class="NLM_string-name">Chesworth, R.</span>; <span class="NLM_string-name">Boriack-Sjodin, P. A.</span>; <span class="NLM_string-name">Munchhof, M. J.</span>; <span class="NLM_string-name">Jin, L.</span>; <span class="NLM_string-name">Penebre, E.</span>; <span class="NLM_string-name">Barbash, O. I.</span></span> <span> </span><span class="NLM_article-title">Prmt5 Inhibitors and Uses Thereof</span>. Patent <span class="NLM_patent">WO2016022605 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=K.+W.+Duncan&author=R.+Chesworth&author=P.+A.+Boriack-Sjodin&author=M.+J.+Munchhof&author=L.+Jin&author=E.+Penebre&author=O.+I.+Barbash&title=Prmt5+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DPrmt5%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref22':['cit22'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. MAT2a is a functional homodimer with two catalytic sites and a site of allosteric regulation located at the dimer interface. (A) The two MAT2a monomers are depicted as cartoon in blue and gray. At the interface, the reaction products <i>S</i>-adenosylmethionine (SAM) and (β-γ-imido)triphosphate are captured in one of the two active sites (sticks with yellow carbons). Binding of the carboxy-terminal peptide of MAT2b (purple cartoon) to the dimer interface leads to allosteric regulation of SAM biosynthesis (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ndn">4ndn</a>, chains A, B, E). (B) Surface view of MAT2a. Gray shading depicts cavities at the dimer interface, and the largest of these harbor the active and allosteric sites. (C) Recently reported allosteric inhibitors of MAT2a.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Fragment binding induces changes in the allosteric site. (A) Fragment hits <b>1</b> and <b>2</b> and their MAT2a binding affinities as determined by SPR. (B) Crystal structure of <b>2</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs">7bhs</a>, 1.1 Å) shows two molecules of the ligand (green sticks and yellow surface) occupying the symmetric site at the dimer interface. Overlay of the MAT2b-bound state (purple sticks and turquoise surface) illustrates the cryptic pocket accessed by small-molecule ligands. (C, D) Close-up view of the binding site and interactions formed by triazinone hit <b>1</b> (cyan, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhr">7bhr</a>, 1.1 Å) and quinazoline hit <b>2</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs">7bhs</a>, 1.1 Å) with Arg313.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Merging strategy from hits <b>1</b> and <b>2</b> to quinazolinone <b>5.</b> Binding mode confirmed by crystallography (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bht">7bht</a>, 1.1 Å) shows that key interactions are maintained, notably the bidentate hydrogen bonding to Arg313 and hydrophobic stacking with Phe333. Compound <b>5</b> is visibly nonplanar as seen in the electron density (blue mesh shows 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1σ); see also <a href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf" class="ext-link">Figure S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Optimization of the substituent occupying the aromatic side chain subsite. (A) The hydroxyethyl substituent in <b>26</b> forms a hydrogen bond with the backbone carbonyl of Gly273. Two alternative conformations were observed in the crystal structure, shown as light and dark gray sticks (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhu">7bhu</a>, 1.2 Å). (B) The phenyl substitution in <b>28</b> (pink) brings an almost 400-fold potency boost compared to <b>5</b> (orange). The superposition of both crystal structures shows that the phenyl group displaces two water molecules. The side chain of Phe333 undergoes a slight rotation, maximizing a CH−π interaction (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhv">7bhv</a>, 1.2 Å). (C) <b>29</b> (yellow, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhw">7bhw</a>, 1.2 Å) fills a small hydrophobic side pocket with a methyl substituent. (D) <b>31</b> (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhx">7bhx</a>, 1.1 Å) features a pyridine to capture a water interaction. A weak CH-hydrogen bond to Gly273 is indicated by gray dashes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mouse plasma concentrations of <b>28</b> po (blue circle), <b>28</b> sc (red square), <b>31</b> po (green triangle), and <b>31</b> sc (purple triangle) at 100 mg/kg.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Subcutaneous dosing of <b>28</b> at 50 mg/kg QD: (A) reduction in tumor SAM; (B) antitumor response (left) and body weight change (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>5</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) dimethylamine, THF, rt, 2 h; (b) acetic acid, 100 °C, 12 h, 29% over two steps.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/medium/jm1c00067_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>28</b>, <b>31</b>, <b>32</b>, and <b>33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-10/acs.jmedchem.1c00067/20210520/images/large/jm1c00067_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00067&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) oxalyl chloride, DCE, 80 °C, 30 min; (b) aniline, rt, 1 h, 62–75% over two steps; (c) KHMDS, THF, −78 °C, 1 h, then rt, 3.5 h, 67–80%; (d) PyBOP, DBU, amine, MeCN, rt, 2.5 h, 50–92%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06247" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06247" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 22 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mato, J. M.</span></span> <span> </span><span class="NLM_article-title">S-adenosylmethionine in liver health, injury, and cancer</span>. <i>Physiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>92</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1515</span>– <span class="NLM_lpage">1542</span>, <span class="refDoi"> DOI: 10.1152/physrev.00047.2011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1152%2Fphysrev.00047.2011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=23073625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2012&pages=1515-1542&issue=4&author=S.+C.+Luauthor=J.+M.+Mato&title=S-adenosylmethionine+in+liver+health%2C+injury%2C+and+cancer&doi=10.1152%2Fphysrev.00047.2011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">S-adenosylmethionine in liver health, injury, and cancer</span></div><div class="casAuthors">Lu, Shelly C.; Mato, Jose M.</div><div class="citationInfo"><span class="NLM_cas:title">Physiological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1515-1542</span>CODEN:
                <span class="NLM_cas:coden">PHREA7</span>;
        ISSN:<span class="NLM_cas:issn">0031-9333</span>.
    
            (<span class="NLM_cas:orgname">American Physiological Society</span>)
        </div><div class="casAbstract">A review.  S-adenosylmethionine (AdoMet, also known as SAM and SAMe) is the principal biol. Me donor synthesized in all mammalian cells but most abundantly in the liver.  Biosynthesis of AdoMet requires the enzyme methionine adenosyltransferase (MAT).  In mammals, two genes, MAT1A that is largely expressed by normal liver and MAT2A that is expressed by all extrahepatic tissues, encode MAT.  Patients with chronic liver disease have reduced MAT activity and AdoMet levels.  Mice lacking Mat1a have reduced hepatic AdoMet levels and develop oxidative stress, steatohepatitis, and hepatocellular carcinoma (HCC).  In these mice, several signaling pathways are abnormal that can contribute to HCC formation.  However, injury and HCC also occur if hepatic AdoMet level is excessive chronically.  This can result from inactive mutation of the enzyme glycine N-methyltransferase (GNMT).  Children with GNMT mutation have elevated liver transaminases, and Gnmt knockout mice develop liver injury, fibrosis, and HCC.  Thus a normal hepatic AdoMet level is necessary to maintain liver health and prevent injury and HCC.  AdoMet is effective in cholestasis of pregnancy, and its role in other human liver diseases remains to be better defined.  In exptl. models, it is effective as a chemopreventive agent in HCC and perhaps other forms of cancer as well.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriGQKr7t0oIrVg90H21EOLACvtfcHk0lhzQtCBAUmvdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2lsbnP&md5=6479e1e0b3d84bf946747ad7e580ea22</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1152%2Fphysrev.00047.2011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1152%252Fphysrev.00047.2011%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%2BC.%26aulast%3DMato%26aufirst%3DJ.%2BM.%26atitle%3DS-adenosylmethionine%2520in%2520liver%2520health%252C%2520injury%252C%2520and%2520cancer%26jtitle%3DPhysiol.%2520Rev.%26date%3D2012%26volume%3D92%26issue%3D4%26spage%3D1515%26epage%3D1542%26doi%3D10.1152%2Fphysrev.00047.2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mato, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span> <span> </span><span class="NLM_article-title">Role of methionine adenosyltransferase genes in hepatocarcinogenesis</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">1480</span>– <span class="NLM_lpage">1497</span>, <span class="refDoi"> DOI: 10.3390/cancers3021480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.3390%2Fcancers3021480" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=24212770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvV2nsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1480-1497&issue=2&author=K.+Ramaniauthor=J.+M.+Matoauthor=S.+C.+Lu&title=Role+of+methionine+adenosyltransferase+genes+in+hepatocarcinogenesis&doi=10.3390%2Fcancers3021480"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Role of methionine adenosyltransferase genes in hepatocarcinogenesis</span></div><div class="casAuthors">Ramani, Komal; Mato, Jose M.; Lu, Shelly C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1480-1497</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Hepatocellular carcinoma (HCC) is the most common primary malignant tumor of the liver.  Detection of HCC can be difficult, as most of the patients who develop this tumor have no symptoms other than those related to their longstanding liver disease.  There is an urgent need to understand the mol. mechanisms that are responsible for the development of this disease so that appropriate therapies can be designed.  Methionine adenosyltransferase (MAT) is an essential enzyme required for the biosynthesis of S-adenosylmethionine (AdoMet), an important Me donor in the cell.  Alterations in the expression of MAT genes and a decline in AdoMet biosynthesis are known to be assocd. with liver injury, cirrhosis and HCC.  This review focuses on the role of MAT genes in HCC development and the scope for therapeutic strategies using these genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSoGTREhz3e7Vg90H21EOLACvtfcHk0lhzQtCBAUmvdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvV2nsrs%253D&md5=d37cac20dfc5e1b5d560819306a9919a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3390%2Fcancers3021480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3021480%26sid%3Dliteratum%253Aachs%26aulast%3DRamani%26aufirst%3DK.%26aulast%3DMato%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DS.%2BC.%26atitle%3DRole%2520of%2520methionine%2520adenosyltransferase%2520genes%2520in%2520hepatocarcinogenesis%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26issue%3D2%26spage%3D1480%26epage%3D1497%26doi%3D10.3390%2Fcancers3021480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, E. R.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Weck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlabach, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavrakis, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castelletti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gampa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramadan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sovath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachmann, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buxton, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caothien, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caushi, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés-Cros, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deBeaumont, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaunay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desplat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dwoske, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldridge, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farsidjani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flemming, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattenberger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurov, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jagani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapoor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mollé, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phadke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repko, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schouwey, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanahan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stamm, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiedt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varadarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatesan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitari, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzle, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">Project DRIVE: A Compendium of Cancer Dependencies and Synthetic Lethal Relationships Uncovered by Large-Scale, Deep RNAi Screening</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">592</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2Fj.cell.2017.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=28753431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=577-592&issue=3&author=E.+R.+McDonaldauthor=A.+de+Weckauthor=M.+R.+Schlabachauthor=E.+Billyauthor=K.+J.+Mavrakisauthor=G.+R.+Hoffmanauthor=D.+Belurauthor=D.+Castellettiauthor=E.+Friasauthor=K.+Gampaauthor=J.+Goljiauthor=I.+Kaoauthor=L.+Liauthor=P.+Megelauthor=T.+A.+Perkinsauthor=N.+Ramadanauthor=D.+A.+Ruddyauthor=S.+J.+Silverauthor=S.+Sovathauthor=M.+Stumpauthor=O.+Weberauthor=R.+Widmerauthor=J.+Yuauthor=K.+Yuauthor=Y.+Yueauthor=D.+Abramowskiauthor=E.+Ackleyauthor=R.+Barrettauthor=J.+Bergerauthor=J.+L.+Bernardauthor=R.+Billigauthor=S.+M.+Brachmannauthor=F.+Buxtonauthor=R.+Caothienauthor=J.+X.+Caushiauthor=F.+S.+Chungauthor=M.+Cort%C3%A9s-Crosauthor=R.+S.+deBeaumontauthor=C.+Delaunayauthor=A.+Desplatauthor=W.+Duongauthor=D.+A.+Dwoskeauthor=R.+S.+Eldridgeauthor=A.+Farsidjaniauthor=F.+Fengauthor=J.+Fengauthor=D.+Flemmingauthor=W.+Forresterauthor=G.+G.+Galliauthor=Z.+Gaoauthor=F.+Gauterauthor=V.+Gibajaauthor=K.+Haasauthor=M.+Hattenbergerauthor=T.+Hoodauthor=K.+E.+Hurovauthor=Z.+Jaganiauthor=M.+Jenalauthor=J.+A.+Johnsonauthor=M.+D.+Jonesauthor=A.+Kapoorauthor=J.+Kornauthor=J.+Liuauthor=Q.+Liuauthor=S.+Liuauthor=Y.+Liuauthor=A.+T.+Looauthor=K.+J.+Macchiauthor=T.+Martinauthor=G.+McAllisterauthor=A.+Meyerauthor=S.+Moll%C3%A9author=R.+A.+Pagliariniauthor=T.+Phadkeauthor=B.+Repkoauthor=T.+Schouweyauthor=F.+Shanahanauthor=Q.+Shenauthor=C.+Stammauthor=C.+Stephanauthor=V.+M.+Stuckeauthor=R.+Tiedtauthor=M.+Varadarajanauthor=K.+Venkatesanauthor=A.+C.+Vitariauthor=M.+Wallrothauthor=J.+Weilerauthor=J.+Zhangauthor=C.+Mickaninauthor=V.+E.+Myerauthor=J.+A.+Porterauthor=A.+Laiauthor=H.+Bitterauthor=E.+Leesauthor=N.+Keenauthor=A.+Kauffmannauthor=F.+Stegmeierauthor=F.+Hofmannauthor=T.+Schmelzleauthor=W.+R.+Sellers&title=Project+DRIVE%3A+A+Compendium+of+Cancer+Dependencies+and+Synthetic+Lethal+Relationships+Uncovered+by+Large-Scale%2C+Deep+RNAi+Screening&doi=10.1016%2Fj.cell.2017.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Project DRIVE: A compendium of cancer dependencies and synthetic lethal relationships uncovered by large-scale, deep RNAi screening</span></div><div class="casAuthors">McDonald, E. Robert, III; de Weck, Antoine; Schlabach, Michael R.; Billy, Eric; Mavrakis, Konstantinos J.; Hoffman, Gregory R.; Belur, Dhiren; Castelletti, Deborah; Frias, Elizabeth; Gampa, Kalyani; Golji, Javad; Kao, Iris; Li, Li; Megel, Philippe; Perkins, Thomas A.; Ramadan, Nadire; Ruddy, David A.; Silver, Serena J.; Sovath, Sosathya; Stump, Mark; Weber, Odile; Widmer, Roland; Yu, Jianjun; Yu, Kristine; Yue, Yingzi; Abramowski, Dorothee; Ackley, Elizabeth; Barrett, Rosemary; Berger, Joel; Bernard, Julie L.; Billig, Rebecca; Brachmann, Saskia M.; Buxton, Frank; Caothien, Roger; Caushi, Justina X.; Chung, Franklin S.; Cortes-Cros, Marta; de Beaumont, Rosalie S.; Delaunay, Clara; Desplat, Aurore; Duong, William; Dwoske, Donald A.; Eldridge, Richard S.; Farsidjani, Ali; Feng, Fei; Feng, Jia Jia; Flemming, Daisy; Forrester, William; Galli, Giorgio G.; Gao, Zhenhai; Gauter, Francois; Gibaja, Veronica; Haas, Kristy; Hattenberger, Marc; Hood, Tami; Hurov, Kristen E.; Jagani, Zainab; Jenal, Mathias; Johnson, Jennifer A.; Jones, Michael D.; Kapoor, Avnish; Korn, Joshua; Liu, Jilin; Liu, Qiumei; Liu, Shumei; Liu, Yue; Loo, Alice T.; Macchi, Kaitlin J.; Martin, Typhaine; McAllister, Gregory; Meyer, Amandine; Molle, Sandra; Pagliarini, Raymond A.; Phadke, Tanushree; Repko, Brian; Schouwey, Tanja; Shanahan, Frances; Shen, Qiong; Stamm, Christelle; Stephan, Christine; Stucke, Volker M.; Tiedt, Ralph; Varadarajan, Malini; Venkatesan, Kavitha; Vitari, Alberto C.; Wallroth, Marco; Weiler, Jan; Zhang, Jing; Mickanin, Craig; Myer, Vic E.; Porter, Jeffery A.; Lai, Albert; Bitter, Hans; Lees, Emma; Keen, Nicholas; Kauffmann, Audrey; Stegmeier, Frank; Hofmann, Francesco; Schmelzle, Tobias; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">577-592.e10</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Elucidation of the mutational landscape of human cancer has progressed rapidly and been accompanied by the development of therapeutics targeting mutant oncogenes.  However, a comprehensive mapping of cancer dependencies has lagged behind and the discovery of therapeutic targets for counteracting tumor suppressor gene loss is needed.  To identify vulnerabilities relevant to specific cancer subtypes, we conducted a large-scale RNAi screen in which viability effects of mRNA knockdown were assessed for 7837 genes using an av. of 20 shRNAs per gene in 398 cancer cell lines.  We describe findings of this screen, outlining the classes of cancer dependency genes and their relationships to genetic, expression, and lineage features.  In addn., we describe robust gene-interaction networks recapitulating both protein complexes and functional cooperation among complexes and pathways.  This dataset along with a web portal is provided to the community to assist in the discovery and translation of new therapeutic approaches for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokUs33_s2nC7Vg90H21EOLACvtfcHk0lgoBlsy2DyZjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GisLvI&md5=41592e684cc8f2cd28117cb3303f6120</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonald%26aufirst%3DE.%2BR.%26aulast%3Dde%2BWeck%26aufirst%3DA.%26aulast%3DSchlabach%26aufirst%3DM.%2BR.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DMavrakis%26aufirst%3DK.%2BJ.%26aulast%3DHoffman%26aufirst%3DG.%2BR.%26aulast%3DBelur%26aufirst%3DD.%26aulast%3DCastelletti%26aufirst%3DD.%26aulast%3DFrias%26aufirst%3DE.%26aulast%3DGampa%26aufirst%3DK.%26aulast%3DGolji%26aufirst%3DJ.%26aulast%3DKao%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMegel%26aufirst%3DP.%26aulast%3DPerkins%26aufirst%3DT.%2BA.%26aulast%3DRamadan%26aufirst%3DN.%26aulast%3DRuddy%26aufirst%3DD.%2BA.%26aulast%3DSilver%26aufirst%3DS.%2BJ.%26aulast%3DSovath%26aufirst%3DS.%26aulast%3DStump%26aufirst%3DM.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DWidmer%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DAckley%26aufirst%3DE.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DBerger%26aufirst%3DJ.%26aulast%3DBernard%26aufirst%3DJ.%2BL.%26aulast%3DBillig%26aufirst%3DR.%26aulast%3DBrachmann%26aufirst%3DS.%2BM.%26aulast%3DBuxton%26aufirst%3DF.%26aulast%3DCaothien%26aufirst%3DR.%26aulast%3DCaushi%26aufirst%3DJ.%2BX.%26aulast%3DChung%26aufirst%3DF.%2BS.%26aulast%3DCort%25C3%25A9s-Cros%26aufirst%3DM.%26aulast%3DdeBeaumont%26aufirst%3DR.%2BS.%26aulast%3DDelaunay%26aufirst%3DC.%26aulast%3DDesplat%26aufirst%3DA.%26aulast%3DDuong%26aufirst%3DW.%26aulast%3DDwoske%26aufirst%3DD.%2BA.%26aulast%3DEldridge%26aufirst%3DR.%2BS.%26aulast%3DFarsidjani%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFlemming%26aufirst%3DD.%26aulast%3DForrester%26aufirst%3DW.%26aulast%3DGalli%26aufirst%3DG.%2BG.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DGauter%26aufirst%3DF.%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DHattenberger%26aufirst%3DM.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DHurov%26aufirst%3DK.%2BE.%26aulast%3DJagani%26aufirst%3DZ.%26aulast%3DJenal%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DKapoor%26aufirst%3DA.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLoo%26aufirst%3DA.%2BT.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMartin%26aufirst%3DT.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DMeyer%26aufirst%3DA.%26aulast%3DMoll%25C3%25A9%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%2BA.%26aulast%3DPhadke%26aufirst%3DT.%26aulast%3DRepko%26aufirst%3DB.%26aulast%3DSchouwey%26aufirst%3DT.%26aulast%3DShanahan%26aufirst%3DF.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DStamm%26aufirst%3DC.%26aulast%3DStephan%26aufirst%3DC.%26aulast%3DStucke%26aufirst%3DV.%2BM.%26aulast%3DTiedt%26aufirst%3DR.%26aulast%3DVaradarajan%26aufirst%3DM.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DVitari%26aufirst%3DA.%2BC.%26aulast%3DWallroth%26aufirst%3DM.%26aulast%3DWeiler%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DLees%26aufirst%3DE.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DKauffmann%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DSchmelzle%26aufirst%3DT.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DProject%2520DRIVE%253A%2520A%2520Compendium%2520of%2520Cancer%2520Dependencies%2520and%2520Synthetic%2520Lethal%2520Relationships%2520Uncovered%2520by%2520Large-Scale%252C%2520Deep%2520RNAi%2520Screening%26jtitle%3DCell%26date%3D2017%26volume%3D170%26issue%3D3%26spage%3D577%26epage%3D592%26doi%3D10.1016%2Fj.cell.2017.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span> <span> </span><span class="NLM_article-title">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span>. <i>Science (Washington, DC, U. S.)</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span> (<span class="NLM_issue">6278</span>),  <span class="NLM_fpage">1214</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1126/science.aad5214</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1126%2Fscience.aad5214" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=26912360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=1214-8&issue=6278&author=G.+V.+Kryukovauthor=F.+H.+Wilsonauthor=J.+R.+Ruthauthor=J.+Paulkauthor=A.+Tsherniakauthor=S.+E.+Marlowauthor=F.+Vazquezauthor=B.+A.+Weirauthor=M.+E.+Fitzgeraldauthor=M.+Tanakaauthor=C.+M.+Bielskiauthor=J.+M.+Scottauthor=C.+Dennisauthor=G.+S.+Cowleyauthor=J.+S.+Boehmauthor=D.+E.+Rootauthor=T.+R.+Golubauthor=C.+B.+Clishauthor=J.+E.+Bradnerauthor=W.+C.+Hahnauthor=L.+A.+Garraway&title=MTAP+deletion+confers+enhanced+dependency+on+the+PRMT5+arginine+methyltransferase+in+cancer+cells&doi=10.1126%2Fscience.aad5214"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP deletion confers enhanced dependency on the PRMT5 arginine methyltransferase in cancer cells</span></div><div class="casAuthors">Kryukov, Gregory V.; Wilson, Frederick H.; Ruth, Jason R.; Paulk, Joshiawa; Tsherniak, Aviad; Marlow, Sara E.; Vazquez, Francisca; Weir, Barbara A.; Fitzgerald, Mark E.; Tanaka, Minoru; Bielski, Craig M.; Scott, Justin M.; Dennis, Courtney; Cowley, Glenn S.; Boehm, Jesse S.; Root, David E.; Golub, Todd R.; Clish, Clary B.; Bradner, James E.; Hahn, William C.; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6278</span>),
    <span class="NLM_cas:pages">1214-1218</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The discovery of cancer dependencies has the potential to inform therapeutic strategies and to identify putative drug targets.  Integrating data from comprehensive genomic profiling of cancer cell lines and from functional characterization of cancer cell dependencies, we discovered that loss of the enzyme methylthioadenosine phosphorylase (MTAP) confers a selective dependence on protein arginine methyltransferase 5 (PRMT5) and its binding partner WDR77.  MTAP is frequently lost due to its proximity to the commonly deleted tumor suppressor gene, CDKN2A.  We obsd. increased intracellular concns. of methylthioadenosine (MTA, the metabolite cleaved by MTAP) in cells harboring MTAP deletions.  Furthermore, MTA specifically inhibited PRMT5 enzymic activity.  Administration of either MTA or a small-mol. PRMT5 inhibitor showed a modest preferential impairment of cell viability for MTAP-null cancer cell lines compared with isogenic MTAP-expressing counterparts.  Together, our findings reveal PRMT5 as a potential vulnerability across multiple cancer lineages augmented by a common "passenger" genomic alteration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr94dQPnMgrqbVg90H21EOLACvtfcHk0lgoBlsy2DyZjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjs12gtro%253D&md5=2b017a8c54972828c9024a9d85ff2e2d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad5214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad5214%26sid%3Dliteratum%253Aachs%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DWilson%26aufirst%3DF.%2BH.%26aulast%3DRuth%26aufirst%3DJ.%2BR.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DMarlow%26aufirst%3DS.%2BE.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DTanaka%26aufirst%3DM.%26aulast%3DBielski%26aufirst%3DC.%2BM.%26aulast%3DScott%26aufirst%3DJ.%2BM.%26aulast%3DDennis%26aufirst%3DC.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DMTAP%2520deletion%2520confers%2520enhanced%2520dependency%2520on%2520the%2520PRMT5%2520arginine%2520methyltransferase%2520in%2520cancer%2520cells%26jtitle%3DScience%2520%2528Washington%252C%2520DC%252C%2520U.%2520S.%2529%26date%3D2016%26volume%3D351%26issue%3D6278%26spage%3D1214%26epage%3D8%26doi%3D10.1126%2Fscience.aad5214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marjon, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clasquin, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McVay, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kernytsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murtie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biller, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span> <span> </span><span class="NLM_article-title">MTAP Deletions in Cancer Create Vulnerability to Targeting of the MAT2A/PRMT5/RIOK1 Axis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">574</span>– <span class="NLM_lpage">587</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.03.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2Fj.celrep.2016.03.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=27068473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=574-587&issue=3&author=K.+Marjonauthor=M.+J.+Cameronauthor=P.+Quangauthor=M.+F.+Clasquinauthor=E.+Mandleyauthor=K.+Kuniiauthor=M.+McVayauthor=S.+Choeauthor=A.+Kernytskyauthor=S.+Grossauthor=Z.+Konteatisauthor=J.+Murtieauthor=M.+L.+Blakeauthor=J.+Travinsauthor=M.+Dorschauthor=S.+A.+Billerauthor=K.+M.+Marks&title=MTAP+Deletions+in+Cancer+Create+Vulnerability+to+Targeting+of+the+MAT2A%2FPRMT5%2FRIOK1+Axis&doi=10.1016%2Fj.celrep.2016.03.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">MTAP Deletions in Cancer Create Vulnerability to Targeting of the MAT2A/PRMT5/RIOK1 Axis</span></div><div class="casAuthors">Marjon, Katya; Cameron, Michael J.; Quang, Phong; Clasquin, Michelle F.; Mandley, Everton; Kunii, Kaiko; McVay, Michael; Choe, Sung; Kernytsky, Andrew; Gross, Stefan; Konteatis, Zenon; Murtie, Joshua; Blake, Michelle L.; Travins, Jeremy; Dorsch, Marion; Biller, Scott A.; Marks, Kevin M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-587</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Homozygous deletions of p16/CDKN2A are prevalent in cancer, and these mutations commonly involve co-deletion of adjacent genes, including methylthioadenosine phosphorylase (MTAP).  Here, we used shRNA screening and identified the metabolic enzyme, methionine adenosyltransferase II alpha (MAT2A), and the arginine methyltransferase, PRMT5, as vulnerable enzymes in cells with MTAP deletion.  Metabolomic and biochem. studies revealed a mechanistic basis for this synthetic lethality.  The MTAP substrate methylthioadenosine (MTA) accumulates upon MTAP loss.  Biochem. profiling of a methyltransferase enzyme panel revealed that MTA is a potent and selective inhibitor of PRMT5.  MTAP-deleted cells have reduced PRMT5 methylation activity and increased sensitivity to PRMT5 depletion.  MAT2A produces the PRMT5 substrate S-adenosylmethionine (SAM), and MAT2A depletion reduces growth and PRMT5 methylation activity selectively in MTAP-deleted cells.  Furthermore, this vulnerability extends to PRMT5 co-complex proteins such as RIOK1.  Thus, the unique biochem. features of PRMT5 create an axis of targets vulnerable in CDKN2A/MTAP-deleted cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokW739dwptcbVg90H21EOLACvtfcHk0lhW_GuNIccJNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlsleltrk%253D&md5=73a6839e63b85a71f3661dc9e0c7b4ea</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.03.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.03.043%26sid%3Dliteratum%253Aachs%26aulast%3DMarjon%26aufirst%3DK.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DQuang%26aufirst%3DP.%26aulast%3DClasquin%26aufirst%3DM.%2BF.%26aulast%3DMandley%26aufirst%3DE.%26aulast%3DKunii%26aufirst%3DK.%26aulast%3DMcVay%26aufirst%3DM.%26aulast%3DChoe%26aufirst%3DS.%26aulast%3DKernytsky%26aufirst%3DA.%26aulast%3DGross%26aufirst%3DS.%26aulast%3DKonteatis%26aufirst%3DZ.%26aulast%3DMurtie%26aufirst%3DJ.%26aulast%3DBlake%26aufirst%3DM.%2BL.%26aulast%3DTravins%26aufirst%3DJ.%26aulast%3DDorsch%26aufirst%3DM.%26aulast%3DBiller%26aufirst%3DS.%2BA.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26atitle%3DMTAP%2520Deletions%2520in%2520Cancer%2520Create%2520Vulnerability%2520to%2520Targeting%2520of%2520the%2520MAT2A%252FPRMT5%252FRIOK1%2520Axis%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D15%26issue%3D3%26spage%3D574%26epage%3D587%26doi%3D10.1016%2Fj.celrep.2016.03.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, T. M.</span></span> <span> </span><span class="NLM_article-title">Codeletion of the genes for p16INK4, methylthioadenosine phosphorylase, interferon-α1, interferon-β1, and other 9p21 markers in human malignant cell lines</span>. <i>Cancer Genet. Cytogenet.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>86</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">28</span>, <span class="refDoi"> DOI: 10.1016/0165-4608(95)00157-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2F0165-4608%2895%2900157-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=8616780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADyaK28XitVylsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1996&pages=22-28&issue=1&author=H.+Zhangauthor=Z.-H.+Chenauthor=T.+M.+Savarese&title=Codeletion+of+the+genes+for+p16INK4%2C+methylthioadenosine+phosphorylase%2C+interferon-%CE%B11%2C+interferon-%CE%B21%2C+and+other+9p21+markers+in+human+malignant+cell+lines&doi=10.1016%2F0165-4608%2895%2900157-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Codeletion of the genes for p16INK4, methylthioadenosine phosphorylase, interferon-α1, interferon-β1, and other 9p21 markers in human malignant cell lines</span></div><div class="casAuthors">Zhang, Hongyang; Chen, Zhi-Hao; Savarese, Todd M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Genetics and Cytogenetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-8</span>CODEN:
                <span class="NLM_cas:coden">CGCYDF</span>;
        ISSN:<span class="NLM_cas:issn">0165-4608</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">In this study, 27 malignant cell lines, including leukemias, gliomas, and lung and bladder carcinomas were screened for homozygous deletions of the putative tumor suppressor gene p16 (MTS1/CDK4I/CDKN2) and other markers within the chromosome 9p21 region; these include the genes for interferon-α1 (IFNA1), interferon-β1 (IFNB1), methylthioadenosine phosphorylase (MTAP), and two microsatellite markers, D9S171 and D9S169.  The purpose of this study was to det. the incidence of co-deletion of these markers.  Screening for homozygous deletions was carried out using direct polymerase chain reaction of genomic DNA, or, in the case of MTAP, a functional enzyme assay.  Of these cell lines, 14 (52%) were found to have homozygous deletions of the p16 gene.  Two of the 14 p16-neg. cell lines (14%) were found to have homozygous deletions within the p16 domain but no other 9p21 marker.  MTAP was co-deleted in 12 of the 14 p16-neg. cell lines (86%), whereas IFNA1 was co-deleted with p16 in eight of these lines (57%); IFNB1 was co-deleted in five (36%) of the p16-deleted cell lines.  The D9S171 marker, which may lie greater than 3 cM centromeric to p16, is co-deleted in three cell lines (21%); the D9S169 marker, which maps even further toward the centromere, was co-deleted in only one cell line (7%).  Loss of any 9p21 marker, e.g., MTAP or IFNA1, were invariable predictors of the loss of the p16 gene.  In addn., loss of IFNA1 always predicted a loss of MTAP (eight of eight cell lines), although loss of MTAP did not always predict a loss of IFNA1 (four of 12 MTAP-deleted cell lines did not have homozygous deletions of IFNA1).  Thus loss of nearby genes occurs in a high percentage of cell lines that bear homozygous deletions of the p16 locus.  Co-deletion of MTAP or IFN in p16-neg. malignant cells is of interest, as loss of these genes may influence the biol. behavior of these cells and render them susceptible to certain therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOfMShSxVr5rVg90H21EOLACvtfcHk0lhW_GuNIccJNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XitVylsrk%253D&md5=7da196b73cc878db99a6c88e8eedd603</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0165-4608%2895%2900157-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0165-4608%252895%252900157-3%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DZ.-H.%26aulast%3DSavarese%26aufirst%3DT.%2BM.%26atitle%3DCodeletion%2520of%2520the%2520genes%2520for%2520p16INK4%252C%2520methylthioadenosine%2520phosphorylase%252C%2520interferon-%25CE%25B11%252C%2520interferon-%25CE%25B21%252C%2520and%2520other%25209p21%2520markers%2520in%2520human%2520malignant%2520cell%2520lines%26jtitle%3DCancer%2520Genet.%2520Cytogenet.%26date%3D1996%26volume%3D86%26issue%3D1%26spage%3D22%26epage%3D28%26doi%3D10.1016%2F0165-4608%2895%2900157-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albers, E.</span></span> <span> </span><span class="NLM_article-title">Metabolic characteristics and importance of the universal methionine salvage pathway recycling methionine from 5′-methylthioadenosine</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1132</span>– <span class="NLM_lpage">1142</span>, <span class="refDoi"> DOI: 10.1002/iub.278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1002%2Fiub.278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=19946895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFShtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2009&pages=1132-1142&issue=12&author=E.+Albers&title=Metabolic+characteristics+and+importance+of+the+universal+methionine+salvage+pathway+recycling+methionine+from+5%E2%80%B2-methylthioadenosine&doi=10.1002%2Fiub.278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic characteristics and importance of the universal methionine salvage pathway recycling methionine from 5'-methylthioadenosine</span></div><div class="casAuthors">Albers, Eva</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1132-1142</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Inc.</span>)
        </div><div class="casAbstract">A review.  The methionine salvage pathway, also called the 5'-methylthioadenosine (MTA) cycle, recycles the S atom of MTA, which is a byproduct in the biosyntheses of polyamine and the plant hormone, ethylene.  MTA is 1st converted to 5'-methylthioribose 1-phosphate either by MTA phosphorylase or the combined action of MTA nucleosidase and 5'-methylthioribose kinase.  Subsequently, 5 addnl. enzymic steps, catalyzed by 4 or 5 proteins, will form 4-methylthio-2-oxobutyrate, the deaminated form of methionine.  The final transamination is achieved by transaminases active in the amino acid biosynthesis.  This pathway is present with some variations in all types of organisms and seems to be designed for a quick removal of MTA achieved by high affinities of the 1st enzymes.  During evolution, some enzymes attained addnl. functions, like a proposed role in nuclear mRNA processing by the aci-reductone dioxygenase.  For others, the function seems to be lost due to conditions in specific ecol. niches, such as, presence of S and/or absence of O resulting in that, e.g., Escherichia coli is lacking a functional pathway.  The pathway is regulated as response to S availability and take part in the regulation of polyamine synthesis.  Some of the enzymes in the pathway show sep. specificities in different organisms and some others are unique for groups of bacteria and parasites.  Thus, promising targets for antimicrobial agents have been identified.  Other medical topics to which this pathway has connections are cancer, apoptosis, and the inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHogIDh-oIBLVg90H21EOLACvtfcHk0lhW_GuNIccJNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFShtLnO&md5=3f7d52b51c55763984f86e6151dad45b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fiub.278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.278%26sid%3Dliteratum%253Aachs%26aulast%3DAlbers%26aufirst%3DE.%26atitle%3DMetabolic%2520characteristics%2520and%2520importance%2520of%2520the%2520universal%2520methionine%2520salvage%2520pathway%2520recycling%2520methionine%2520from%25205%25E2%2580%25B2-methylthioadenosine%26jtitle%3DIUBMB%2520Life%26date%3D2009%26volume%3D61%26issue%3D12%26spage%3D1132%26epage%3D1142%26doi%3D10.1002%2Fiub.278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardini, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulter, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talalay, P.</span></span> <span> </span><span class="NLM_article-title">Analogues of Methionine as Substrates and Inhibitors of the Methionine Adenosyltransferase Reaction</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">499</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=4918253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADyaE3cXlt1OhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1970&pages=481-499&issue=5&author=J.+B.+Lombardiniauthor=A.+W.+Coulterauthor=P.+Talalay&title=Analogues+of+Methionine+as+Substrates+and+Inhibitors+of+the+Methionine+Adenosyltransferase+Reaction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Analogs of methionine as substrates and inhibitors of the methionine adenosyltransferase reaction.  Deductions concerning the conformation of methionine</span></div><div class="casAuthors">Lombardini, J. B.; Coulter, A. W.; Talalay, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">481-99</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">Steric, electronic, and conformational requirements are described for analogs of L-methionine essential to their function as substrates or inhibitors of the methionine adenosyltransferase reaction (EC 2.4.1.13).  With the aid of partially purified transferase prepns. from Escherichia coli, bakers' yeast, and rat liver, a systematic study of substrate analogs has been undertaken.  Inhibitors of the enzyme fall into 3 categories:  (a) straight C chain amino acids, such as L-2-amino-4-hexenoic acid (trans but not the cis isomer) and L-2-amino-4-hexynoic acid, which are the most potent inhibitors; (b) cyclic amino acids, among which 1-aminocyclopentanecarboxylic acid and 1 of the 4 isomers of 1-amino-3-methylcyclopentanecarboxylic acid (either the 1R, 3R or the 1S, 3R isomer) are the most powerful; and (c) O-acetyl-L-serine, O-carbamoyl-L-serine, and S-carbamoyl-L-cysteine.  Since inhibitors belonging to groups a and b possess considerable conformational rigidity by virtue of the presence of unsatns. or cyclic structures, it has been possible to draw conclusions with respect to the conformation of L-methionine at the active site of the adenosyltransferase reaction.  A no. of the inhibitors of the methionine adenosyltransferase reaction, such as 1-aminocyclopentanecarboxylic acid and S-carbamoyl-L-cysteine, are known to be inhibitors of the growth of certain microorganisms and tumors.  The possibility is suggested that these inhibitory activities may be mediated at least in part through the inhibition of the synthesis of S-adenosyl-L-methionine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQxsk5Ck1f1rVg90H21EOLACvtfcHk0lgLKV0rFGFU9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3cXlt1OhsL8%253D&md5=707357bc113134ff3daccf1d49ecaf11</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLombardini%26aufirst%3DJ.%2BB.%26aulast%3DCoulter%26aufirst%3DA.%2BW.%26aulast%3DTalalay%26aufirst%3DP.%26atitle%3DAnalogues%2520of%2520Methionine%2520as%2520Substrates%2520and%2520Inhibitors%2520of%2520the%2520Methionine%2520Adenosyltransferase%2520Reaction%26jtitle%3DMol.%2520Pharmacol.%26date%3D1970%26volume%3D6%26issue%3D5%26spage%3D481%26epage%3D499" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardini, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sufrin, J. R.</span></span> <span> </span><span class="NLM_article-title">Chemotherapeutic potential of methionine analogue inhibitors of tumor-derived methionine adenosyltransferases</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>32</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">489</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(83)90528-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2F0006-2952%2883%2990528-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=6847699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADyaL3sXhtlOgs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1983&pages=489-495&issue=3&author=J.+B.+Lombardiniauthor=J.+R.+Sufrin&title=Chemotherapeutic+potential+of+methionine+analogue+inhibitors+of+tumor-derived+methionine+adenosyltransferases&doi=10.1016%2F0006-2952%2883%2990528-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Chemotherapeutic potential of methionine analog inhibitors of tumor-derived methionine adenosyltransferases</span></div><div class="casAuthors">Lombardini, John B.; Sufrin, Janice R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">489-95</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">Two isozymes of ATP:L-methionine S-adenosyltransferase (MAT)  [9012-52-6] were fractionated from rat Novikoff solid hepatoma.  Their Km values for L-methionine  [63-68-3] and/or their inhibition consts. for various L-methionine analogs were significantly different from the kinetic consts. obtained for three isozymes fractionated from normal rat liver.  Ki Values for cycloleucine  [52-52-8] and (±)-2-aminobicyclo[2.1.1]hexane-2-carboxylic acid  [71046-30-5], presented for each tumor and liver isozyme, indicate that (±)-2-aminobicyclo[2.1.1]hexane-2-carboxylic acid was the more potent inhibitor.  Dixon plots were also used to test a series of amino acid analogs [cycloleucine, 1-aminocyclobutanecarboxylic acid  [22264-50-2], 1-aminocyclohexanecarboxylic acid  [2756-85-6], (±)-2-aminobicyclo[2.1.1]hexane-2-carboxylic acid, L-2-amino-4-hexynoic acid  [29834-76-2], (Z)-L-2-amino-5-chloro-4-hexenoic acid  [51018-17-8], L-ethionine  [13073-35-3], S-n-propyl-DL-homocysteine  [2598-46-1], S-n-butyl-DL-homocysteine  [4378-14-7], and seleno-DL-ethionine  [2578-27-0]] of methionine for inhibitory potency.  Fixed L-methionine concns. were used to det. the concn. of inhibitor necessary to inhibit the MAT reaction by 50%.  Differential inhibitory activities of the amino acid analogs were noted between the tumor and rat liver isozymes, thus supporting the suggestion that tumor-derived MAT isozymes may provide an exploitable target for cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGporWK3_U66Q7Vg90H21EOLACvtfcHk0lgLKV0rFGFU9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhtlOgs7o%253D&md5=cbb7c2624fc4e88c98f57a0a2b8ba59e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2883%2990528-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252883%252990528-2%26sid%3Dliteratum%253Aachs%26aulast%3DLombardini%26aufirst%3DJ.%2BB.%26aulast%3DSufrin%26aufirst%3DJ.%2BR.%26atitle%3DChemotherapeutic%2520potential%2520of%2520methionine%2520analogue%2520inhibitors%2520of%2520tumor-derived%2520methionine%2520adenosyltransferases%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1983%26volume%3D32%26issue%3D3%26spage%3D489%26epage%3D495%26doi%3D10.1016%2F0006-2952%2883%2990528-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinlan, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolaños, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowlin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grantner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlicek-Bryant, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman-Cook, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dann, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, S. K.</span></span> <span> </span><span class="NLM_article-title">Targeting S-adenosylmethionine biosynthesis with a novel allosteric inhibitor of Mat2A</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">785</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1038%2Fnchembio.2384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=28553945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=785-792&issue=7&author=C.+L.+Quinlanauthor=S.+E.+Kaiserauthor=B.+Bola%C3%B1osauthor=D.+Nowlinauthor=R.+Grantnerauthor=S.+Karlicek-Bryantauthor=J.+L.+Fengauthor=S.+Jenkinsonauthor=K.+Freeman-Cookauthor=S.+G.+Dannauthor=X.+Wangauthor=P.+A.+Wellsauthor=V.+R.+Fantinauthor=A.+E.+Stewartauthor=S.+K.+Grant&title=Targeting+S-adenosylmethionine+biosynthesis+with+a+novel+allosteric+inhibitor+of+Mat2A&doi=10.1038%2Fnchembio.2384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting S-adenosylmethionine biosynthesis with a novel allosteric inhibitor of Mat2A</span></div><div class="casAuthors">Quinlan, Casey L.; Kaiser, Stephen E.; Bolanos, Ben; Nowlin, Dawn; Grantner, Rita; Karlicek-Bryant, Shannon; Feng, Jun Li; Jenkinson, Stephen; Freeman-Cook, Kevin; Dann, Stephen G.; Wang, Xiaoli; Wells, Peter A.; Fantin, Valeria R.; Stewart, Al E.; Grant, Stephan K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">785-792</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">S-Adenosyl-L-methionine (SAM) is an enzyme cofactor used in Me transfer reactions and polyamine biosynthesis.  The biosynthesis of SAM from ATP and L-methionine is performed by the methionine adenosyltransferase enzyme family (Mat; EC 2.5.1.6).  Human methionine adenosyltransferase 2A (Mat2A), the extrahepatic isoform, is often deregulated in cancer.  We identified a Mat2A inhibitor, PF-9366, that binds an allosteric site on Mat2A that overlaps with the binding site for the Mat2A regulator, Mat2B.  Studies exploiting PF-9366 suggested a general mode of Mat2A allosteric regulation.  Allosteric binding of PF-9366 or Mat2B altered the Mat2A active site, resulting in increased substrate affinity and decreased enzyme turnover.  These data support a model whereby Mat2B functions as an inhibitor of Mat2A activity when methionine or SAM levels are high, yet functions as an activator of Mat2A when methionine or SAM levels are low.  The ramification of Mat2A activity modulation in cancer cells is also described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpniywgvxjL1bVg90H21EOLACvtfcHk0lgLKV0rFGFU9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosFSruro%253D&md5=26be67f7977cbcccd5f967844924dc24</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2384%26sid%3Dliteratum%253Aachs%26aulast%3DQuinlan%26aufirst%3DC.%2BL.%26aulast%3DKaiser%26aufirst%3DS.%2BE.%26aulast%3DBola%25C3%25B1os%26aufirst%3DB.%26aulast%3DNowlin%26aufirst%3DD.%26aulast%3DGrantner%26aufirst%3DR.%26aulast%3DKarlicek-Bryant%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DJ.%2BL.%26aulast%3DJenkinson%26aufirst%3DS.%26aulast%3DFreeman-Cook%26aufirst%3DK.%26aulast%3DDann%26aufirst%3DS.%2BG.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWells%26aufirst%3DP.%2BA.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DStewart%26aufirst%3DA.%2BE.%26aulast%3DGrant%26aufirst%3DS.%2BK.%26atitle%3DTargeting%2520S-adenosylmethionine%2520biosynthesis%2520with%2520a%2520novel%2520allosteric%2520inhibitor%2520of%2520Mat2A%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26issue%3D7%26spage%3D785%26epage%3D792%26doi%3D10.1038%2Fnchembio.2384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonyuk, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Liempd, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mato, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasnain, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, A. L.</span></span> <span> </span><span class="NLM_article-title">Structure and function study of the complex that synthesizes S-adenosylmethionine</span>. <i>IUCrJ</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">249</span>, <span class="refDoi"> DOI: 10.1107/S2052252514012585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1107%2FS2052252514012585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=25075345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtV2jsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2014&pages=240-249&author=B.+Murrayauthor=S.+V.+Antonyukauthor=A.+Marinaauthor=S.+M.+Van+Liempdauthor=S.+C.+Luauthor=J.+M.+Matoauthor=S.+S.+Hasnainauthor=A.+L.+Rojas&title=Structure+and+function+study+of+the+complex+that+synthesizes+S-adenosylmethionine&doi=10.1107%2FS2052252514012585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function study of the complex that synthesizes S-adenosylmethionine</span></div><div class="casAuthors">Murray, Ben; Antonyuk, Svetlana V.; Marina, Alberto; Van Liempd, Sebastiaan M.; Lu, Shelly C.; Mato, Jose M.; Hasnain, S. Samar; Rojas, Adriana L.</div><div class="citationInfo"><span class="NLM_cas:title">IUCrJ</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">240-249</span>CODEN:
                <span class="NLM_cas:coden">IUCRAJ</span>;
        ISSN:<span class="NLM_cas:issn">2052-2525</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">S-Adenosylmethionine (SAMe) is the principal Me donor of the cell and is synthesized via an ATP-driven process by methionine adenosyltransferase (MAT) enzymes.  It is tightly linked with cell proliferation in liver and colon cancer.  In humans, there are three genes, mat1A, mat2A and mat2B, which encode MAT enzymes. mat2A and mat2B transcribe MATα2 and MATβ enzyme subunits, resp., with catalytic and regulatory roles.  The MATα2β complex is expressed in nearly all tissues and is thought to be essential in providing the necessary SAMe flux for methylation of DNA and various proteins including histones.  In human hepatocellular carcinoma mat2A and mat2B genes are upregulated, highlighting the importance of the MATα2β complex in liver disease.  The individual subunits have been structurally characterized but the nature of the complex has remained elusive despite its existence having been postulated for more than 20 years and the observation that MATβ is often co-localized with MATα2.  Though SAMe can be produced by MAT(α2)4 alone, this paper shows that the V max of the MATα2β complex is three- to fourfold higher depending on the variants of MATβ that participate in complex formation.  Using X-ray crystallog. and soln. X-ray scattering, the first structures are provided of this 258 kDa functional complex both in crystals and soln. with an unexpected stoichiometry of 4α2 and 2βV2 subunits.  It is demonstrated that the N-terminal regulates the activity of the complex and it is shown that complex formation takes place surprisingly via the C-terminal of MATβV2 that buries itself in a tunnel created at the interface of the MAT(α2)2.  The structural data suggest a unique mechanism of regulation and provide a gateway for structure-based drug design in anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6Zyk4WShPyLVg90H21EOLACvtfcHk0lgun0FkdX6TSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtV2jsrjF&md5=b5cf7fe51c8fcf98ecc49bb290bd41e3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1107%2FS2052252514012585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052252514012585%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DB.%26aulast%3DAntonyuk%26aufirst%3DS.%2BV.%26aulast%3DMarina%26aufirst%3DA.%26aulast%3DVan%2BLiempd%26aufirst%3DS.%2BM.%26aulast%3DLu%26aufirst%3DS.%2BC.%26aulast%3DMato%26aufirst%3DJ.%2BM.%26aulast%3DHasnain%26aufirst%3DS.%2BS.%26aulast%3DRojas%26aufirst%3DA.%2BL.%26atitle%3DStructure%2520and%2520function%2520study%2520of%2520the%2520complex%2520that%2520synthesizes%2520S-adenosylmethionine%26jtitle%3DIUCrJ%26date%3D2014%26volume%3D1%26spage%3D240%26epage%3D249%26doi%3D10.1107%2FS2052252514012585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z. D.</span>; <span class="NLM_string-name">Sui, Z.</span>; <span class="NLM_string-name">Travins, J. M.</span>; <span class="NLM_string-name">Ye, Z.</span></span> <span> </span><span class="NLM_article-title">Pyrazolo[1,5-a]pyrimidin-7(4H)-ones as inhibitors of cellular metabolic processes and their preparation</span>. Patent <span class="NLM_patent">US20180079753 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=Z.+D.+Konteatis&author=Z.+Sui&author=J.+M.+Travins&author=Z.+Ye&title=Pyrazolo%5B1%2C5-a%5Dpyrimidin-7%284H%29-ones+as+inhibitors+of+cellular+metabolic+processes+and+their+preparation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKonteatis%26aufirst%3DZ.%2BD.%26atitle%3DPyrazolo%255B1%252C5-a%255Dpyrimidin-7%25284H%2529-ones%2520as%2520inhibitors%2520of%2520cellular%2520metabolic%2520processes%2520and%2520their%2520preparation%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z. D.</span>; <span class="NLM_string-name">Li, M.</span>; <span class="NLM_string-name">Reznik, S. K.</span>; <span class="NLM_string-name">Sui, Z.</span>; <span class="NLM_string-name">Travins, J. M.</span></span> <span> </span><span class="NLM_article-title">Preparation of azaheterobicyclic inhibitors of MAT2A and methods of use for treating cancer</span>. Patent <span class="NLM_patent">WO2020139992 A1</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=Z.+D.+Konteatis&author=M.+Li&author=S.+K.+Reznik&author=Z.+Sui&author=J.+M.+Travins&title=Preparation+of+azaheterobicyclic+inhibitors+of+MAT2A+and+methods+of+use+for+treating+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKonteatis%26aufirst%3DZ.%2BD.%26atitle%3DPreparation%2520of%2520azaheterobicyclic%2520inhibitors%2520of%2520MAT2A%2520and%2520methods%2520of%2520use%2520for%2520treating%2520cancer%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Konteatis, Z. D.</span>; <span class="NLM_string-name">Li, M.</span>; <span class="NLM_string-name">Liu, P.</span>; <span class="NLM_string-name">Medeiros, M.</span>; <span class="NLM_string-name">Reznik, S. K.</span>; <span class="NLM_string-name">Sui, Z.</span>; <span class="NLM_string-name">Travins, J. M.</span>; <span class="NLM_string-name">Popovici-Muller, J.</span>; <span class="NLM_string-name">Zhou, S.</span>; <span class="NLM_string-name">Ma, G.</span></span> <span> </span><span class="NLM_article-title">Preparation of heterobicyclic compounds as inhibitors of MAT2A and methods of use for treating cancer</span>. Patent <span class="NLM_patent">WO2019191470 A1</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Z.+D.+Konteatis&author=M.+Li&author=P.+Liu&author=M.+Medeiros&author=S.+K.+Reznik&author=Z.+Sui&author=J.+M.+Travins&author=J.+Popovici-Muller&author=S.+Zhou&author=G.+Ma&title=Preparation+of+heterobicyclic+compounds+as+inhibitors+of+MAT2A+and+methods+of+use+for+treating+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKonteatis%26aufirst%3DZ.%2BD.%26atitle%3DPreparation%2520of%2520heterobicyclic%2520compounds%2520as%2520inhibitors%2520of%2520MAT2A%2520and%2520methods%2520of%2520use%2520for%2520treating%2520cancer%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erlanson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">619</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&issue=9&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lgun0FkdX6TSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26issue%3D9%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacquemard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">A bright future for fragment-based drug discovery: what does it hold?</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>14</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1080/17460441.2019.1583643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1080%2F17460441.2019.1583643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=30793989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A280%3ADC%252BB3cfmsVCnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2019&pages=413-416&issue=5&author=C.+Jacquemardauthor=E.+Kellenberger&title=A+bright+future+for+fragment-based+drug+discovery%3A+what+does+it+hold%3F&doi=10.1080%2F17460441.2019.1583643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A bright future for fragment-based drug discovery: what does it hold?</span></div><div class="casAuthors">Jacquemard Celien; Kellenberger Esther</div><div class="citationInfo"><span class="NLM_cas:title">Expert opinion on drug discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">413-416</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSHl2exLyY4fhfKb6ELK8t0fW6udTcc2eYXfC8-r6kzELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfmsVCnsQ%253D%253D&md5=d5ebc2b03fbf98207bd692d1e55279c1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1080%2F17460441.2019.1583643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2019.1583643%26sid%3Dliteratum%253Aachs%26aulast%3DJacquemard%26aufirst%3DC.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DA%2520bright%2520future%2520for%2520fragment-based%2520drug%2520discovery%253A%2520what%2520does%2520it%2520hold%253F%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2019%26volume%3D14%26issue%3D5%26spage%3D413%26epage%3D416%26doi%3D10.1080%2F17460441.2019.1583643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Kloe, G. E.</span></span> <span> </span><span class="NLM_article-title">Fragment library design: efficiently hunting drugs in chemical space</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">e173</span>– <span class="NLM_lpage">e180</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2010.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1016%2Fj.ddtec.2010.11.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVegsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=e173-e180&issue=3&author=S.+M.+Boydauthor=G.+E.+de+Kloe&title=Fragment+library+design%3A+efficiently+hunting+drugs+in+chemical+space&doi=10.1016%2Fj.ddtec.2010.11.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment library design: efficiently hunting drugs in chemical space</span></div><div class="casAuthors">Boyd, Susan M.; de Kloe, Gerdien E.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">E173-E180</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  With the growth in fragment-based drug discovery, numerous strategies have been described for the design of fragment libraries.  Key choices need to be made on both the selection criteria to be applied and the source of the fragments in the library.  Here we review some of the key trends and recent developments in the rapidly evolving field of fragment library design, providing an overview of current design strategies and surveying the characteristics of published fragment libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVJ64DfPahqLVg90H21EOLACvtfcHk0liqfgupZAvDQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVegsLs%253D&md5=4ec37c1cf8b68c43b5395667313d34fa</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2010.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2010.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DBoyd%26aufirst%3DS.%2BM.%26aulast%3Dde%2BKloe%26aufirst%3DG.%2BE.%26atitle%3DFragment%2520library%2520design%253A%2520efficiently%2520hunting%2520drugs%2520in%2520chemical%2520space%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2010%26volume%3D7%26issue%3D3%26spage%3De173%26epage%3De180%26doi%3D10.1016%2Fj.ddtec.2010.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheron, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jasty, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakhnovich, E. I.</span></span> <span> </span><span class="NLM_article-title">OpenGrowth: An Automated and Rational Algorithm for Finding New Protein Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4171</span>– <span class="NLM_lpage">4188</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00886</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00886" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4171-4188&issue=9&author=N.+Cheronauthor=N.+Jastyauthor=E.+I.+Shakhnovich&title=OpenGrowth%3A+An+Automated+and+Rational+Algorithm+for+Finding+New+Protein+Ligands&doi=10.1021%2Facs.jmedchem.5b00886"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">OpenGrowth: An Automated and Rational Algorithm for Finding New Protein Ligands</span></div><div class="casAuthors">Cheron, Nicolas; Jasty, Naveen; Shakhnovich, Eugene I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4171-4188</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a new open-source software, called OpenGrowth, which aims to create de novo ligands by connecting small org. fragments in the active site of proteins.  Mol. growth is biased to produce structures that statistically resemble drugs in an input training database.  Consequently, the produced mols. have superior synthetic accessibility and pharmacokinetic properties compared with randomly grown mols.  The growth process can take into account the flexibility of the target protein and can be started from a seed to mimic R-group strategy or fragment-based drug discovery.  Primary applications of the software on the HIV-1 protease allowed us to quickly identify new inhibitors with a predicted Kd as low as 18 nM.  We also present a graphical user interface that allows a user to select easily the fragments to include in the growth process.  OpenGrowth is released under the GNU GPL license and is available free of charge on the authors' website and at http://opengrowth.sourceforge.net/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcRSoqoxTyrVg90H21EOLACvtfcHk0liqfgupZAvDQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVygsrzI&md5=6f8895e71ca7749cc1dbe559bda1db7c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00886&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00886%26sid%3Dliteratum%253Aachs%26aulast%3DCheron%26aufirst%3DN.%26aulast%3DJasty%26aufirst%3DN.%26aulast%3DShakhnovich%26aufirst%3DE.%2BI.%26atitle%3DOpenGrowth%253A%2520An%2520Automated%2520and%2520Rational%2520Algorithm%2520for%2520Finding%2520New%2520Protein%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26issue%3D9%26spage%3D4171%26epage%3D4188%26doi%3D10.1021%2Facs.jmedchem.5b00886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&issue=9&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0liqfgupZAvDQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26issue%3D9%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&issue=3&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+molecular+recognition+exploiting+hydrophobic+enclosure+in+protein-ligand+binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0lhjdsDBqOE9Sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520molecular%2520recognition%2520exploiting%2520hydrophobic%2520enclosure%2520in%2520protein-ligand%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26issue%3D3%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit20"><span>As this
manuscript was in preparation a patent application (<span class="NLM_contrib-group"><span class="NLM_string-name">Alam, M.</span>; <span class="NLM_string-name">Cleary, L.</span>; <span class="NLM_string-name">Fleury, M.</span>; <span class="NLM_string-name">Pei, Z.</span>; <span class="NLM_string-name">Steel, R.</span>; <span class="NLM_string-name">Sutton, J.</span>; <span class="NLM_string-name">Knox, J. E.</span>; <span class="NLM_string-name">Newby, Z. E. R.</span></span> <span> </span><span class="NLM_article-title">2-Oxoquinazoline derivatives as methionine adenosyltransferase 2A inhibitors and their preparation</span>. Patent <span class="NLM_patent">WO2020123395 A1</span>, 2020) from Ideaya was published on the chemical series described in this paper.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=As+this%0Amanuscript+was+in+preparation+a+patent+application+%28Alam%2C+M.%3B+Cleary%2C+L.%3B+Fleury%2C+M.%3B+Pei%2C+Z.%3B+Steel%2C+R.%3B+Sutton%2C+J.%3B+Knox%2C+J.+E.%3B+Newby%2C+Z.+E.+R.+2-Oxoquinazoline+derivatives+as+methionine+adenosyltransferase+2A+inhibitors+and+their+preparation.+Patent+WO2020123395+A1%2C+2020%29+from+Ideaya+was+published+on+the+chemical+series+described+in+this+paper."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DAlam%26aufirst%3DM.%26atitle%3D2-Oxoquinazoline%2520derivatives%2520as%2520methionine%2520adenosyltransferase%25202A%2520inhibitors%2520and%2520their%2520preparation" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span>; <span class="NLM_string-name">Chesworth, R.</span>; <span class="NLM_string-name">Boriack-Sjodin, P. A.</span>; <span class="NLM_string-name">Munchhof, M. J.</span>; <span class="NLM_string-name">Jin, L.</span>; <span class="NLM_string-name">Penebre, E.</span>; <span class="NLM_string-name">Barbash, O. I.</span></span> <span> </span><span class="NLM_article-title">Prmt5 Inhibitors and Uses Thereof</span>. Patent <span class="NLM_patent">WO2016022605 A1</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=K.+W.+Duncan&author=R.+Chesworth&author=P.+A.+Boriack-Sjodin&author=M.+J.+Munchhof&author=L.+Jin&author=E.+Penebre&author=O.+I.+Barbash&title=Prmt5+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DPrmt5%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ndn" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ndn','PDB','4ndn'); return false;">PDB: 4ndn</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs','PDB','7bhs'); return false;">PDB: 7bhs</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhr" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhr','PDB','7bhr'); return false;">PDB: 7bhr</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bht" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bht','PDB','7bht'); return false;">PDB: 7bht</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhu','PDB','7bhu'); return false;">PDB: 7bhu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhv" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhv','PDB','7bhv'); return false;">PDB: 7bhv</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhw" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhw','PDB','7bhw'); return false;">PDB: 7bhw</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhx" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhx','PDB','7bhx'); return false;">PDB: 7bhx</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i35"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00067">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_47483"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00067?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00067</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary figures and table; extended methods (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_001.pdf">jm1c00067_si_001.pdf (1.02 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00067/suppl_file/jm1c00067_si_002.csv">jm1c00067_si_002.csv (1.22 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Crystal structures of MAT2a small molecule complexes have been deposited in the PDB under the following accession codes: <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhr">7bhr</a>), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhs">7bhs</a>), <b>5</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bht">7bht</a>), <b>26</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhu">7bhu</a>), <b>28</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhv">7bhv</a>), <b>29</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhw">7bhw</a>), <b>31</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7bhx">7bhx</a>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00067&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-10%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00067%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00067" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67998348abad21be","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
